
<html lang="en"     class="pb-page"  data-request-id="52f6d7c4-ebf7-4c9f-8e5e-3cf31ae2c36b"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;issue:issue:10.1021/jmcmar.2021.64.issue-9;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.0c02184;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Helicobacter pylori Xanthine–Guanine–Hypoxanthine Phosphoribosyltransferase—A Putative Target for Drug Discovery against Gastrointestinal Tract Infections" /></meta><meta name="dc.Creator" content="Dianne T.  Keough" /></meta><meta name="dc.Creator" content="Shun Jie  Wun" /></meta><meta name="dc.Creator" content="Ondřej  Baszczyňski" /></meta><meta name="dc.Creator" content="Wai Soon  Eng" /></meta><meta name="dc.Creator" content="Petr  Špaček" /></meta><meta name="dc.Creator" content="Santosh  Panjikar" /></meta><meta name="dc.Creator" content="Lieve  Naesens" /></meta><meta name="dc.Creator" content="Radek  Pohl" /></meta><meta name="dc.Creator" content="Dominik  Rejman" /></meta><meta name="dc.Creator" content="Dana  Hocková" /></meta><meta name="dc.Creator" content="Richard L.  Ferrero" /></meta><meta name="dc.Creator" content="Luke W.  Guddat" /></meta><meta name="dc.Description" content="Helicobacter pylori (Hp) is a human pathogen that lives in the gastric mucosa of approximately 50% of the world’s population causing gastritis, peptic ulcers, and gastric cancer. An increase in res..." /></meta><meta name="Description" content="Helicobacter pylori (Hp) is a human pathogen that lives in the gastric mucosa of approximately 50% of the world’s population causing gastritis, peptic ulcers, and gastric cancer. An increase in res..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 23, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c02184" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02184" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c02184" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02184" /></link>
        
    
    

<title>Helicobacter pylori Xanthine–Guanine–Hypoxanthine Phosphoribosyltransferase—A Putative Target for Drug Discovery against Gastrointestinal Tract Infections | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c02184" /></meta><meta property="og:title" content="Helicobacter pylori Xanthine–Guanine–Hypoxanthine Phosphoribosyltransferase—A Putative Target for Drug Discovery against Gastrointestinal Tract Infections" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0023.jpeg" /></meta><meta property="og:description" content="Helicobacter pylori (Hp) is a human pathogen that lives in the gastric mucosa of approximately 50% of the world’s population causing gastritis, peptic ulcers, and gastric cancer. An increase in resistance to current drugs has sparked the search for new Hp drug targets and therapeutics. One target is the disruption of nucleic acid production, which can be achieved by impeding the synthesis of 6-oxopurine nucleoside monophosphates, the precursors of DNA and RNA. These metabolites are synthesized by Hp xanthine–guanine–hypoxanthine phosphoribosyltransferase (XGHPRT). Here, nucleoside phosphonates have been evaluated, which inhibit the activity of this enzyme with Ki values as low as 200 nM. The prodrugs of these compounds arrest the growth of Hp at a concentration of 50 μM in cell-based assays. The kinetic properties of HpXGHPRT have been determined together with its X-ray crystal structure in the absence and presence of 9-[(N-3-phosphonopropyl)-aminomethyl-9-deazahypoxanthine, providing a basis for new antibiotic development." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c02184"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02184">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c02184&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c02184&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c02184&amp;href=/doi/10.1021/acs.jmedchem.0c02184" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 9</span><span class="cit-fg-pageRange">, 5710-5729</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/9" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c02179" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c02207" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title"><i>Helicobacter pylori</i> Xanthine–Guanine–Hypoxanthine Phosphoribosyltransferase—A Putative Target for Drug Discovery against Gastrointestinal Tract Infections</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Dianne T. Keough</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dianne T. Keough</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">The School of Chemistry and Molecular Biosciences, The University of Queensland, Brisbane 4072, Queensland, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dianne+T.++Keough">Dianne T. Keough</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shun Jie Wun</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shun Jie Wun</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">The School of Chemistry and Molecular Biosciences, The University of Queensland, Brisbane 4072, Queensland, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shun+Jie++Wun">Shun Jie Wun</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ondřej Baszczyňski</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ondřej Baszczyňski</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Organic Chemistry and Biochemistry, Academy of Sciences of the Czech Republic, Prague 6 CZ-166 10, Czech Republic</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ond%C5%99ej++Baszczy%C5%88ski">Ondřej Baszczyňski</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0195-9953" title="Orcid link">http://orcid.org/0000-0003-0195-9953</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wai Soon Eng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wai Soon Eng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">The School of Chemistry and Molecular Biosciences, The University of Queensland, Brisbane 4072, Queensland, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wai+Soon++Eng">Wai Soon Eng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Petr Špaček</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Petr Špaček</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Organic Chemistry and Biochemistry, Academy of Sciences of the Czech Republic, Prague 6 CZ-166 10, Czech Republic</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Petr++%C5%A0pa%C4%8Dek">Petr Špaček</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Santosh Panjikar</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Santosh Panjikar</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Australian Synchrotron, ANSTO, 800 Blackburn Road, Clayton 3168, Victoria, Australia</div><div class="loa-info-affiliations-info">Department of Biochemistry and Molecular Biology, Monash University, Clayton 3800, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Santosh++Panjikar">Santosh Panjikar</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lieve Naesens</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lieve Naesens</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Katholieke Universiteit, Leuven, Rega Institute for Medical Research, Leuven 3000, Belgium</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lieve++Naesens">Lieve Naesens</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9742-9302" title="Orcid link">http://orcid.org/0000-0001-9742-9302</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Radek Pohl</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Radek Pohl</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Organic Chemistry and Biochemistry, Academy of Sciences of the Czech Republic, Prague 6 CZ-166 10, Czech Republic</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Radek++Pohl">Radek Pohl</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dominik Rejman</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dominik Rejman</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Organic Chemistry and Biochemistry, Academy of Sciences of the Czech Republic, Prague 6 CZ-166 10, Czech Republic</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dominik++Rejman">Dominik Rejman</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dana Hocková</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dana Hocková</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Organic Chemistry and Biochemistry, Academy of Sciences of the Czech Republic, Prague 6 CZ-166 10, Czech Republic</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dana++Hockov%C3%A1">Dana Hocková</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Richard L. Ferrero</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Richard L. Ferrero</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Hudson Institute of Medical Research, Clayton 3800, Victoria, Australia</div><div class="loa-info-affiliations-info">Department of Molecular and Translational Sciences, Monash University, Clayton 3800, Australia</div><div class="loa-info-affiliations-info">Biomedicine Discovery Institute, Department of Microbiology, Monash University, Clayton 3800, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Richard+L.++Ferrero">Richard L. Ferrero</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Luke W. Guddat</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Luke W. Guddat</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">The School of Chemistry and Molecular Biosciences, The University of Queensland, Brisbane 4072, Queensland, Australia</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#fb978e909ed59c8e9f9f9a8fbb8e8ad59e9f8ed59a8e"><span class="__cf_email__" data-cfemail="5e322b353b70392b3a3a3f2a1e2b2f703b3a2b703f2b">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Luke+W.++Guddat">Luke W. Guddat</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8204-8408" title="Orcid link">http://orcid.org/0000-0002-8204-8408</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02184&amp;href=/doi/10.1021%2Facs.jmedchem.0c02184" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 9</span><span class="cit-pageRange">, 5710–5729</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 23, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>17 December 2020</li><li><span class="item_label"><b>Published</b> online</span>23 April 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 13 May 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c02184" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02184</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D5710%26pageCount%3D20%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DDianne%2BT.%2BKeough%252C%2BShun%2BJie%2BWun%252C%2BOnd%25C5%2599ej%2BBaszczy%25C5%2588ski%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D9%26contentID%3Dacs.jmedchem.0c02184%26title%3DHelicobacter%2Bpylori%2BXanthine%25E2%2580%2593Guanine%25E2%2580%2593Hypoxanthine%2BPhosphoribosyltransferase%25E2%2580%2594A%2BPutative%2BTarget%2Bfor%2BDrug%2BDiscovery%2Bagainst%2BGastrointestinal%2BTract%2BInfections%26numPages%3D20%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D5729%26publicationDate%3DMay%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c02184"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">576</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c02184" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Helicobacter pylori Xanthine–Guanine–Hypoxanthine Phosphoribosyltransferase—A Putative Target for Drug Discovery against Gastrointestinal Tract Infections&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Dianne&quot;,&quot;last_name&quot;:&quot;T. Keough&quot;},{&quot;first_name&quot;:&quot;Shun&quot;,&quot;last_name&quot;:&quot;Jie Wun&quot;},{&quot;first_name&quot;:&quot;Ondřej&quot;,&quot;last_name&quot;:&quot;Baszczyňski&quot;},{&quot;first_name&quot;:&quot;Wai&quot;,&quot;last_name&quot;:&quot;Soon Eng&quot;},{&quot;first_name&quot;:&quot;Petr&quot;,&quot;last_name&quot;:&quot;Špaček&quot;},{&quot;first_name&quot;:&quot;Santosh&quot;,&quot;last_name&quot;:&quot;Panjikar&quot;},{&quot;first_name&quot;:&quot;Lieve&quot;,&quot;last_name&quot;:&quot;Naesens&quot;},{&quot;first_name&quot;:&quot;Radek&quot;,&quot;last_name&quot;:&quot;Pohl&quot;},{&quot;first_name&quot;:&quot;Dominik&quot;,&quot;last_name&quot;:&quot;Rejman&quot;},{&quot;first_name&quot;:&quot;Dana&quot;,&quot;last_name&quot;:&quot;Hocková&quot;},{&quot;first_name&quot;:&quot;Richard&quot;,&quot;last_name&quot;:&quot;L. Ferrero&quot;},{&quot;first_name&quot;:&quot;Luke&quot;,&quot;last_name&quot;:&quot;W. Guddat&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;23&quot;,&quot;issue&quot;:&quot;9&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;5710-5729&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c02184&quot;},&quot;abstract&quot;:&quot;Helicobacter pylori (Hp) is a human pathogen that lives in the gastric mucosa of approximately 50% of the world’s population causing gastritis, peptic ulcers, and gastric cancer. An increase in resistance to current drugs has sparked the search for new Hp drug targets and therapeutics. One target is the disruption of nucleic acid production, which can be achieved by impeding the synthesis of 6-oxopurine nucleoside monophosphates, the precursors of DNA and RNA. These metabolites are synthesized by Hp xanthine–guanine–hypoxanthine phosphoribosyltransferase (XGHPRT). Here, nucleoside phosphonates have been evaluated, which inhibit the activity of this enzyme with Ki values as low as 200 nM. The prodrugs of these compounds arrest the growth of Hp at a concentration of 50 μM in cell-based assays. The kinetic properties of HpXGHPRT have been determined together with its X-ray crystal structure in the absence and presence of 9-[(N-3-phosphonopropyl)-aminomethyl-9-deazahypoxanthine, providing a basis for new antibiot&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02184&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02184" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02184&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02184" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02184&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02184" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02184&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02184&amp;href=/doi/10.1021/acs.jmedchem.0c02184" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c02184" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c02184" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (10 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c02184%26sid%3Dliteratum%253Aachs%26pmid%3D33891818%26genre%3Darticle%26aulast%3DKeough%26date%3D2021%26atitle%3DHelicobacter%2Bpylori%2BXanthine%25E2%2580%2593Guanine%25E2%2580%2593Hypoxanthine%2BPhosphoribosyltransferase%25E2%2580%2594A%2BPutative%2BTarget%2Bfor%2BDrug%2BDiscovery%2Bagainst%2BGastrointestinal%2BTract%2BInfections%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D9%26spage%3D5710%26epage%3D5729%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291651" title="Anions">Anions</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291070" title="Pharmaceuticals">Pharmaceuticals</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/9" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/jmcmar.2021.64.issue-9/20210513/jmcmar.2021.64.issue-9.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0023.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText"><i>Helicobacter pylori</i> (Hp) is a human pathogen that lives in the gastric mucosa of approximately 50% of the world’s population causing gastritis, peptic ulcers, and gastric cancer. An increase in resistance to current drugs has sparked the search for new Hp drug targets and therapeutics. One target is the disruption of nucleic acid production, which can be achieved by impeding the synthesis of 6-oxopurine nucleoside monophosphates, the precursors of DNA and RNA. These metabolites are synthesized by Hp xanthine–guanine–hypoxanthine phosphoribosyltransferase (XGHPRT). Here, nucleoside phosphonates have been evaluated, which inhibit the activity of this enzyme with <i>K</i><sub>i</sub> values as low as 200 nM. The prodrugs of these compounds arrest the growth of Hp at a concentration of 50 μM in cell-based assays. The kinetic properties of HpXGHPRT have been determined together with its X-ray crystal structure in the absence and presence of 9-[(<i>N</i>-3-phosphonopropyl)-aminomethyl-9-deazahypoxanthine, providing a basis for new antibiotic development.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61075" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61075" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Gastritis, peptic ulcers, two forms of cancer, gastric mucosa-associated lymphoid tissue lymphoma, and gastric adenocarcinoma have been shown to be associated with the presence of <i>Helicobacter pylori</i> (Hp) in the gastric mucosa. The finding that Hp is the etiological agent for these diseases was established in 1982 by two Australian scientists, Marshall and Warren,<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> who were subsequently awarded the Nobel Prize for Physiology and Medicine in 2005 for this discovery. Hp is found in the gastric mucosa of at least 50% of the world’s population,<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> though this number is closer to 100% in some countries.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Its presence is asymptomatic, and carriers have no prior knowledge of when, or if, life-threatening symptoms could take hold.<a onclick="showRef(event, 'ref4 ref5 ref6 ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref4 ref5 ref6 ref7 ref8 ref9">(4−9)</a> In addition to causing the devastating symptoms of gastritis and peptic ulcers, Hp is the third leading cause of cancer and the most common cause of cancer due to infectious agents, ranking it above papillomavirus and hepatitis B.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a></div><div class="NLM_p">Together with other Gram-negative infections, there has been an alarming increase in multi-drug-resistant (MDR) strains of Hp (where MDR is defined as resistance to ≥ three antibiotics of different classes) in recent years.<a onclick="showRef(event, 'ref4 ref11'); return false;" href="javascript:void(0);" class="ref ref4 ref11">(4,11)</a> This resistance has been found to be as high as 15–20% and, in some locations, this percentage is higher.<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12,13)</a> This led the WHO to include Hp in the list of top 12 pathogens for urgent drug discovery and prompted the drive to find new drug targets and new antibiotics to combat the diseases caused by this organism.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a></div><div class="NLM_p">One possible target for innovative therapies to treat Hp infections is the metabolic production of nucleic acids. If this can be interrupted, or significantly slowed, the organism will have a diminished ability to survive. All organisms need to be able to synthesize their own DNA/RNA, and this indicates that they must have a constant supply of precursor nucleoside monophosphates. There are two metabolic routes whereby an organism can obtain these compounds: (i) by the synthesis of the purine ring in a 10-step energetically demanding metabolic pathway (de novo) or (ii) by the transport of purine bases or nucleosides from their host (salvage). Hp has adopted the second of the two strategies with the full complement of genes coding for the de novo pathway being redundant.<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16,17)</a></div><div class="NLM_p">Mendz and colleagues undertook the first studies on the purine salvage pathway in Hp<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> and established the role of HpXGHPRT in the synthesis of its purine nucleotides. The fact that Hp requires exogenous purine bases/nucleosides for growth has been documented previously.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> For microbes, the rate at which they generate AMP and GMP pools has also been found to be the limiting step in their growth<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> and, hence, the need for a fully functional salvage pathway. <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> shows the enzymes available to the bacterium needed to synthesize the nucleoside monophosphates, GMP and IMP, which are then incorporated directly into its DNA and RNA.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Purine salvage pathway in Hp. The green oval indicates the cell membrane of Hp. (A) GMP synthase; (B) IMP dehydrogenase; (C) adenylosuccinate synthase; (D) adenylosuccinate lyase; (E) GMP reductase; (F) purine nucleoside phosphorylase; APRT: adenine phosphoribosyltransferase; and XGHPRT: xanthine–guanine–hypoxanthine phosphoribosyltransferase. Adapted with permission from Liechti and Goldberg.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">A diagram for the cycling of bases, nucleobases, and nucleotides within Hp is given in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Cycling of purine bases, nucleosides, and nucleotides in Hp. The gene coding for the 5′/3′-nucleotidase is <i>sur</i>E. Adapted with permission from Liechti and Goldberg.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> The red cross signifies that nucleoside monophosphates are unable to cross the cellular membrane into the bacteria.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">A central enzyme for the supply of purine nucleoside monophosphates is xanthine–guanine–hypoxanthine phosphoribosyltransferase (XGHPRT) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Thus, it is a candidate for drug intervention of Hp. XGHPRT belongs to a family of enzymes known as 6-oxopurine phosphoribosyltransferases (PRTases). The reaction catalyzed by these enzymes is given in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Reaction catalyzed by 6-oxopurine PRTases. When R = NH<sub>2</sub>, the base is guanine, when R = H, the base is hypoxanthine, and when R = OH, the base is xanthine.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">It has previously been demonstrated that inhibitors of <i>Plasmodium falciparum</i> (Pf)HGXPRT, <i>Plasmodium vivax</i> (Pv)HGPRT, or <i>Mycobacterium tuberculosis</i> (Mt)HGPRT show activity as anti-malarial<a onclick="showRef(event, 'ref21 ref22 ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref21 ref22 ref23 ref24 ref25">(21−25)</a> and anti-tuberculosis agents.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> One class of inhibitors is broadly referred to as “nucleoside phosphonates” as they contain at least one carbon–phosphorus bond, making them stable to enzymatic or chemical hydrolysis. These compounds are designed to mimic either the substrates or products of the catalytic reaction or that of the transition state. The nucleoside phosphonate inhibitors of 6-oxopurine PRTases can be categorized into six groups: the acyclic nucleoside phosphonates (ANPs);<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> the aza-ANPs;<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> the symmetric and asymmetric acyclic nucleoside bis-phosphonates;<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> the acyclic immucillin phosphonates (AIPs);<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> the pyrrolidine nucleoside phosphonates (PNPs);<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> and the pyrrolidine nucleoside bis-phosphonates (PNBPs).<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> The scaffolds for these inhibitors are shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Chemical templates of 6-oxopurine PRTase phosphonate inhibitors. (A) First ANPs. X denotes the atoms connecting the two functional groups, the base and the phosphonate group, and can be either carbon or oxygen atoms. R<sub>1</sub> denotes where further chemical attachments have been made.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> (B) aza-ANPs. Attachments at (R<sub>2</sub>) allow a functional group to reach into a third cavity in the active site, the pyrophosphate binding pocket.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> (C) AIPs containing a positively charged nitrogen in the linker mimicking the positive charge on the ribose oxygen which is predicted to occur at the transition state. Variable attachments have been made at R<sub>3</sub>.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> (D) PNPs. The length of the linker (<i>m</i>) can vary.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> (E) PNBPs. A second phosphonate group is added to the carbon atom in the pyrrolidine ring. Rotation around the five-membered ring allows either of the phosphonate groups to point to the two different areas in the active site that can bind phosphate groups.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Crystal structures of different representatives from these five classes of nucleoside phosphonate inhibitors have been obtained in complex with the 6-oxopurine PRTases from <i>Homo sapiens</i> (Hs), <i>Escherichia coli</i> (Ec), Mt, Pf, and Pv. The <i>K</i><sub>i</sub> values for the inhibitors differ for each of these enzymes. Thus, the crystal structures show not only how they bind in the active site but also provide the explanations for the range of binding constants and offer clues as to how the inhibitor can be tailor-made to fit more tightly in the active site of the 6-oxopurine PRTases from the individual organisms.</div><div class="NLM_p">The original design of these inhibitors was based on the successful antiviral drugs including tenofovir, adefovir, and cidofovir, which also contain a nucleobase (either adenine or cytosine) linked by a series of atoms to a phosphonate group.<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30,31)</a> To improve the biological activity of those drugs, the negative charges were masked by hydrophobic attachments to increase membrane transport and, thus, their biological activity.<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32,33)</a> Such compounds are prodrugs since the masking groups have to be hydrolyzed by intracellular enzymes present in the target cell to release the effective therapeutic.<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32,33)</a> This prodrug strategy was thus adopted to convert phosphonate inhibitors of PfHGXPRT and MtHGPRT into compounds that arrest the growth of Pf<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> and Mt in cell-based assays.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></div><div class="NLM_p last">Based on previous studies, HpXGHPRT is suggested as a target enzyme for the development of chemotherapeutics against Hp infections. Herein, recombinant HpXGHPRT has been expressed and purified and its kinetic properties were evaluated. Non-naturally occurring 6-oxopurines were also assayed to determine if they are substrates for the enzyme and if they possess anti-Hp activity in cell assays. Such compounds have the potential to be developed into new therapeutics. In addition, we present the first crystal of HpXGHPRT, which is used to explain substrate specificity and to show how phosphonates newly synthesized in our laboratories inhibit this enzyme. We then demonstrate that the prodrugs of these compounds arrest the growth of Hp in cell-based assays.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74116" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74116" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Catalytic Properties of the Target Enzyme, HpXGHPRT</h3><div class="NLM_p">Recombinant hexa-his tag HpXGHPRT was expressed and purified to homogeneity. The kinetic constants of this enzyme for the naturally occurring 6-oxopurine bases and 5-phospho-α-<span class="smallcaps smallerCapital">d</span>-ribosyl-1-pyrophosphate (PRib-PP) were determined. These are compared with those for the human 6-oxopurine PRTase as well as three bacterial 6-oxopurine PRTases to assess its relative catalytic efficiency (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Kinetic Constants for HpXGHPRT, Human HGPRT, and Three Bacterial 6-Oxopurine PRTases</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">enzyme</th><th class="colsep0 rowsep0" align="center" char="±"><i>k</i><sub>cat</sub> (s<sup>–1</sup>) (G)</th><th class="colsep0 rowsep0" align="center" char="±"><i>k</i><sub>cat</sub> (s<sup>–1</sup>) (Hx)</th><th class="colsep0 rowsep0" align="center"><i>k</i><sub>cat</sub> (s<sup>–1</sup>) (X)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HpXGHPRT</td><td class="colsep0 rowsep0" align="char" char="±">7.3 ± 0.2</td><td class="colsep0 rowsep0" align="char" char="±">4.5 ± 0.4</td><td class="colsep0 rowsep0" align="left">2.3 ± 0.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HsHGPRT<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a></td><td class="colsep0 rowsep0" align="char" char="±">8.2 ± 0.6</td><td class="colsep0 rowsep0" align="char" char="±">5.2 ± 0.4</td><td class="colsep0 rowsep0" align="left">NS<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">EcXGPRT<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a></td><td class="colsep0 rowsep0" align="char" char="±">28.0 ± 0.1</td><td class="colsep0 rowsep0" align="char" char="±">13.7 ± 1.1</td><td class="colsep0 rowsep0" align="left">38.5 ± 0.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">EcHPRT<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a></td><td class="colsep0 rowsep0" align="char" char="±">10.2 ± 0.7</td><td class="colsep0 rowsep0" align="char" char="±">59.0 ± 3.5</td><td class="colsep0 rowsep0" align="left">0.008 ± 0.0001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MtHGPRT<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></td><td class="colsep0 rowsep0" align="char" char="±">0.62 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.55 ± 0.01</td><td class="colsep0 rowsep0" align="left">NS<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td></tr></tbody></table><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="left" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">enzyme</th><th class="colsep0 rowsep0" align="center" char="±"><i>K</i><sub>m</sub> (μM) (G)</th><th class="colsep0 rowsep0" align="center" char="±"><i>K</i><sub>m</sub> (μM) (Hx)</th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>m</sub> (μM) (X)</th><th class="colsep0 rowsep0" align="center" char="±"><i>K</i><sub>m</sub> (μM) (PRib-PP)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HpXGHPRT</td><td class="colsep0 rowsep0" align="char" char="±">3.1 ± 0.2</td><td class="colsep0 rowsep0" align="char" char="±">8.4 ± 0.5</td><td class="colsep0 rowsep0" align="left">20 ± 2</td><td class="colsep0 rowsep0" align="char" char="±">27 ± 4<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HsHGPRT<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a></td><td class="colsep0 rowsep0" align="char" char="±">1.9 ± 0.4</td><td class="colsep0 rowsep0" align="char" char="±">3.4 ± 1.0</td><td class="colsep0 rowsep0" align="left">NS</td><td class="colsep0 rowsep0" align="char" char="±">65 ± 5<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">EcXGPRT<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a></td><td class="colsep0 rowsep0" align="char" char="±">4.3 ± 0.3</td><td class="colsep0 rowsep0" align="char" char="±">91 ± 11</td><td class="colsep0 rowsep0" align="left">30.5 ± 2.6</td><td class="colsep0 rowsep0" align="char" char="±">139 ± 16<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">EcHPRT<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a></td><td class="colsep0 rowsep0" align="char" char="±">294 ± 20</td><td class="colsep0 rowsep0" align="char" char="±">12.5 ± 2.4</td><td class="colsep0 rowsep0" align="left">25 ± 6.1</td><td class="colsep0 rowsep0" align="char" char="±">193 ± 7<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MtHGPRT<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></td><td class="colsep0 rowsep0" align="char" char="±">4.4 ± 0.4</td><td class="colsep0 rowsep0" align="char" char="±">8.3 ± 0.7</td><td class="colsep0 rowsep0" align="left">NS<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char="±">465 ± 15<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></td></tr></tbody></table><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">enzyme</th><th class="colsep0 rowsep0" align="center" char="."><i>k</i><sub>cat</sub>/<i>K</i><sub>m</sub> (μM<sup>–1</sup> s<sup>–1</sup>) (G)</th><th class="colsep0 rowsep0" align="center" char="."><i>k</i><sub>cat</sub>/<i>K</i><sub>m</sub> (μM<sup>–1</sup> s<sup>–1</sup>) (Hx)</th><th class="colsep0 rowsep0" align="center"><i>k</i><sub>cat</sub>/<i>K</i><sub>m</sub> (μM<sup>–1</sup> s<sup>–1</sup>) (X)</th><th class="colsep0 rowsep0" align="center" char="."><i>k</i><sub>cat</sub>/<i>K</i><sub>m</sub> (μM<sup>–1</sup> s<sup>–1</sup>) (PRib-PP)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HpXGHPRT</td><td class="colsep0 rowsep0" align="char" char=".">2.3</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="left">0.10</td><td class="colsep0 rowsep0" align="char" char=".">0.3<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HsHGPRT<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a></td><td class="colsep0 rowsep0" align="char" char=".">4.3</td><td class="colsep0 rowsep0" align="char" char=".">1.5</td><td class="colsep0 rowsep0" align="left">NS<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char=".">0.13<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">EcXGPRT<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a></td><td class="colsep0 rowsep0" align="char" char=".">6.5</td><td class="colsep0 rowsep0" align="char" char=".">0.15</td><td class="colsep0 rowsep0" align="left">1.3</td><td class="colsep0 rowsep0" align="char" char=".">0.2<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">EcHPRT<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a></td><td class="colsep0 rowsep0" align="char" char=".">0.03</td><td class="colsep0 rowsep0" align="char" char=".">4.7</td><td class="colsep0 rowsep0" align="left">0.03</td><td class="colsep0 rowsep0" align="char" char=".">0.3<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Guanine as a substrate.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">NS = not a substrate.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">Hx as a substrate.</p></div></div></div><div class="NLM_p">In terms of its <i>k</i><sub>cat</sub> and <i>K</i><sub>m</sub> values for hypoxanthine, guanine, and PRib-PP, HpXGHPRT is more akin to human HGPRT than to the three bacterial enzymes (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). However, HpXGHPRT differs from the human enzyme in that xanthine is not a substrate for the human enzyme, but it is a good substrate for HpXGHPRT with a <i>k</i><sub>cat</sub> value only 3 times lower than for guanine and 1.7 times lower for hypoxanthine (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The <i>K</i><sub>m</sub> value of 20 μM demonstrates that it also binds this substrate well, slightly better than EcXGPRT, which has a <i>K</i><sub>m</sub> value for xanthine of 30 μM (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><div class="NLM_p last">The <i>K</i><sub>m</sub> value of PRib-PP for HpXGHPRT is also similar to the human enzyme but is 5 times lower than for EcXGPRT, 7 times lower than for EcHPRT, and 17 times lower than for MtHGPRT. Thus, the Hp enzyme has a higher affinity for this substrate than EcXGPRT, EcHPRT, or MtHGPRT (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The two Ec enzymes have higher <i>k</i><sub>cat</sub> values compared with HpXGHPRT for the purine bases. As denoted by their name, EcXGPRT prefers xanthine (19-fold higher) and guanine (4-fold higher), while EcHPRT prefers Hx (16-fold higher). In contrast, these purine bases have much higher <i>K</i><sub>m</sub> values for the Ec enzymes, resulting in little difference in catalytic efficiency between these three enzymes (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). MtHGPRT has similar <i>K</i><sub>m</sub> values for guanine and Hx as HpXGHPRT, but the major difference lies in their respective <i>k</i><sub>cat</sub> values (approximately 10-fold lower for both bases for the Mt enzyme), making MtHGPRT the slowest of these five enzymes but still highly active (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Thus, HpXGHPRT is an efficient enzyme consistent with its central role in the synthesis of nucleoside monophosphates in vivo. Furthermore, the recombinant enzyme is suitable for screening and testing of compounds that could be developed as drug leads.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Catalytic Properties of This Enzyme Can Be Used to Arrest the Growth of Hp In Vitro</h3><div class="NLM_p">The purine base analogues, 6-mercaptopurine and 6-thioguanine, have been shown to possess anti-cancer properties,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> and they also arrest the growth of Pf in vitro.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> The mechanism of action of these two anti-cancer agents is via conversion to their toxic metabolites, the corresponding nucleoside monophosphates, by human HGPRT.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Thus, HpXGHPRT could play a similar role in Hp.</div><div class="NLM_p">6-Thioxanthine is not a substrate for human HGPRT, and it functions as a non-toxic drug in human cells. This purine base requires the introduction of the Ec <i>gpt</i> gene into the cancer cells before cell proliferation can cease or be reduced.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> It then exerts its anti-cancer activity by the action of the expressed EcXGPRT enzyme, which catalyzes the formation of the corresponding 6-thioxanthine nucleoside monophosphate. GMP synthase converts this nucleoside into the lethal 6-thioguanine monophosphate.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> This is the principle used in suicide gene therapy, whereby an innocuous compound can be transformed into an active drug.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></div><div class="NLM_p">The kinetic constants of 6-thioxanthine, 6-thioguanine, and 6-mercaptopurine for HpXGHPRT were determined with a view to ascertain if they were substrates of HpXGHPRT which would convert them into toxic metabolites (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). All three have similar kinetic properties to those of the naturally occurring bases (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The substitution of the 6-oxo group by a 6-thio group has only a small effect on <i>k</i><sub>cat</sub> and <i>K</i><sub>m</sub> values for either guanine or xanthine (cf. <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The biggest difference is a 5-fold increase in the <i>K</i><sub>m</sub> for 6-mercaptopurine when compared with hypoxanthine. The naturally occurring purine bases, guanine, hypoxanthine, and xanthine, have been shown to be transported into the Hp cells from its host cell (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), so it is assumed that these three bases would be transported in a similar manner. <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> shows that 6-thioguanine and 6-thioxanthine exhibit activity against Hp in cell culture. Thus, purine base analogues have potential as drug leads against Hp. 6-Thioxanthine should exhibit a favorable therapeutic index as it is not a substrate for the human enzyme and it is non-toxic to human cells.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Kinetic Constants of the 6-Thio-Substituted Substrates for HpXGHPRT and Their Activity in Hp Cell Assays</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0020.gif" alt="" id="gr19" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">(−) No growth at 50 μM concentration of the base in the cell culture.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">(+) Some inhibition of growth at 50 μM concentration of the base in the cell culture.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">ND = not determined.</p></div></div><div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Inhibitors of This Enzyme and Assessment of Their Prodrugs in Arresting the Growth of Hp in Cell-Based Assays</h3><div class="NLM_p">Eleven compounds were chosen from our nucleoside phosphonate inventory, and their <i>K</i><sub>i</sub> values were determined for HpXGHPRT (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). All are competitive inhibitors with respect to PRib-PP. The prodrugs of each of these inhibitors were then synthesized to assess their biological activity. The prodrug approach was adopted as the negative charges of the phosphonyl oxygens of these inhibitors severely reduces, or abolishes, their ability to traverse cell membranes.<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32,33)</a> Four different masking groups were trialed to determine the prodrug of choice for Hp. The compound number and structure of each inhibitor together with its <i>K</i><sub>i</sub> value and cell-based activity is given in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. <i>K</i><sub>i</sub> Values of HpXGHPRT Inhibitors and the Biological Activity of Their Prodrugs against Hp in Cell-Based Assays</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0021.gif" alt="" id="gr20" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0022.gif" alt="" id="gr21" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last"><b>A</b>, <b>B</b>, <b>C</b>, and <b>D</b> denote the chemical structure of the masking groups.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Synthesis in methods, unpublished compound.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">(−) indicates that there is no growth at 50 μM concentration of the compound in the cell culture.</p></div><div class="footnote" id="t3fn4"><sup><sup>d</sup></sup><p class="last">(+) indicates that there was some inhibition of growth at 50 μM concentration of the compound in the cell culture.</p></div><div class="footnote" id="t3fn5"><sup><sup>e</sup></sup><p class="last">(+++) indicates that the addition of the inhibitor to the cell culture at 50 μM concentration made no difference to the growth.</p></div></div><div></div></div><div class="NLM_p">Compound <b>I</b> has one of the lowest <i>K</i><sub>i</sub> values for HpXGHPRT in this series of compounds, and one of its prodrugs, <b>IA</b>, inhibits Hp growth (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). However, the second prodrug of this compound, <b>IB</b>, had no biological effect, even at 50 μM, suggesting that a phenyl group is the preferred option compared with a methyl group for delivery and activation in Hp. Other instances where the substitution of alanine for phenylalanine influences the biological effectiveness of the prodrug in vitro have also been reported.<a onclick="showRef(event, 'ref33 ref40'); return false;" href="javascript:void(0);" class="ref ref33 ref40">(33,40)</a> It is not known if the differences in biological activity are due to increased transport or to more efficient hydrolysis of the masking group within the cell or whether it is a combination of both.</div><div class="NLM_p">Compounds <b>II</b> and <b>III</b> have similar <i>K</i><sub>i</sub> values as <b>I</b> (∼0.2 μM), and their masking group is the same as <b>IA</b>. The prodrug of <b>IIA</b> is slightly more effective in vitro than <b>IIIA</b>. The only difference between <b>II</b> and <b>III</b> is the number of carbon atoms in the linker connecting the hydroxyl group to the nitrogen atom with <b>II</b> having one extra carbon. Though the effect on the anti-bacterial growth is not marked, this extra carbon in the linker affects either transport or hydrolysis in the Hp cells.</div><div class="NLM_p">There is little difference in the <i>K</i><sub>i</sub> values between <b>IV</b> and <b>V</b> (0.4 and 1.1 μM), and they have the same masking groups. Therefore, it is not unexpected that the prodrugs, <b>IV-A</b> and <b>V-A</b>, have a similar ability to prevent Hp growth in vivo. Compound <b>VI</b> has a slightly higher <i>K</i><sub>i</sub> value (1.9 μM) than <b>I–V</b> and two different prodrugs of this compound were synthesized (<b>VI-C</b> and <b>VI-D</b>). Surprisingly, the prodrug with 252 masking group “<b>D</b>” has better Hp activity than <b>C</b>. This masking group was originally prepared for use as an anti-malarial drug formulation as it is a mimic of primaquine. However, it works well as a prodrug for Hp.</div><div class="NLM_p">Compounds <b>VII–X</b> have <i>K</i><sub>i</sub> values between 2 and 5 μM. With a similar masking group, all possess activity against Hp though <b>VIII-A</b> and <b>X-A</b> are slightly more effective. <b>X-A</b> has good biological activity and its parent compound is a good inhibitor. Its activity against the growth of Hp in vitro could be due to the fact that there is only one phosphonate group and, therefore, only two masking groups have to be hydrolyzed as opposed to four. Furthermore, the attachment of the hydrophobic phenyl group could facilitate transport. This prodrug also possesses excellent IC<sub>50</sub> values against Pf and Mt in cell culture (1.5 and 1.1 μM), suggesting that this form of drug and prodrug as a good lead for chemotherapeutic development.</div><div class="NLM_p">Compound <b>XI</b> is the guanine analogue of <b>V</b>, but it has a higher <i>K</i><sub>i</sub> value (cf. 32 μM with 1.1 μM) (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). The masking group in both these prodrugs is the same, so the only difference is in the value of the inhibition constant. <b>V-A</b> has good biological activity in vitro but <b>XI-A</b> has no effect, suggesting that the parent compound has to be a good inhibitor of HpXGPRT to be effective in vitro.</div><div class="NLM_p last">The cytotoxicity in human cells for a number of these compounds has been determined previously. These values range from 100 to >300 μM, higher concentrations than those needed to kill Hp in culture.<a onclick="showRef(event, 'ref21 ref22 ref26 ref27 ref28 ref29 ref39 ref41'); return false;" href="javascript:void(0);" class="ref ref21 ref22 ref26 ref27 ref28 ref29 ref39 ref41">(21,22,26−29,39,41)</a> This good selectivity index is an indication that these compounds are forerunners for antibiotic development against Hp.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Subunit Size and Amino Acid Sequence of HpXGHPRT</h3><div class="NLM_p">A bioinformatic study was performed to assess the similarity and differences in the amino acid sequences between HpXGHPRT and the four 6-oxopurine PRTases (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>).<a onclick="showRef(event, 'ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref42 ref43">(42,43)</a> The subunit of HpXGHPRT consists of 153 amino acids making it, along with EcXGPRT (152 amino acids), one of the two smallest of these five enzymes (HsHGPRT contains 218 residues, EcHPRT contains 178 residues, and MtHGPRT contains 202 residues) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Sequence alignment between HpXGHPRT and 6-oxopurine PRTases from humans, Ec and Mt. The residues highlighted in cyan are identical and those in red are homologous. The “boxes” indicate the binding site of functional groups, that is, purine base, 5′-phosphate group of the nucleoside monophosphates or PRib-PP, the divalent metal ion, and PP<sub>i</sub> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). These locations are based on the known crystal structures of the human and bacterial enzymes in complex with substrates or products of the reaction or with inhibitors. HpXGHPRT (WP_00559479); EcXGPRT (POA9M5); HsHGPRT (POO492); EcHPRT (POA9M2); and MtHGPRT (POA5TO).<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">There is little amino acid sequence identity between HpXGHPRT and the other four enzymes with HpXGHPRT having 29% identity with EcXGPRT, 24% identity with human HGPRT, 36% with EcHPRT, and 29% with MtHGPRT. Overall, there are only eight completely conserved amino acid residues (G36, L43, S59, Y60, D92, G94, K121, and P136) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). Based on this sequence alignment, six of the eight conserved residues are in the expected binding site of the functional groups of the substrates/products of the reaction, while L43 and P136 do not appear to have a significant role in either catalysis or substrate/product binding.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Biological Unit of HpXGHPRT in Solution</h3><div class="NLM_p last">On gel filtration in 0.1 M Tris-HCl, 0.01 M MgCl<sub>2</sub>, pH 7.4 containing 1 mM DTT, HpXGHPRT elutes as a fully active dimer with a molecular mass of 38.6 ± 1.9 kDa. This compares with the calculated molecular mass of 36.9 kDa. It is, therefore, likely that the active biological unit is dimeric. Under identical conditions, EcXGPRT eluted from the same column with a mass of 69.3 ± 2.8 kDa, consistent with the calculated mass of a tetramer (65.6 kDa). MtHGPRT exhibited a mass of 91 ± 3.9 kDa, which is tetrameric (calculated mass of 92 kDa). EcXGPRT has previously been found to elute as an active tetramer on Superose-12 gel filtration<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> while analytical studies<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> and gel filtration of EcHPRT confirm that, in solution, this enzyme is also a tetramer, suggesting that this is its active biological unit. Active human HGPRT also exists as a tetramer in solution.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Thus, not only is HpXGHPRT a catalytically efficient member of the 6-oxopurine PRTase family but it is the smallest of these enzymes in its molecular mass in solution.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Crystal Structure of HpXGHPRT</h3><div class="NLM_p">In order to explain how inhibitors bind to HpXGHPRT and to provide a framework for rational design, the knowledge of the 3D structure of the enzyme is essential. Two crystal structures of HpXGHPRT have, therefore, been obtained. One is in the absence of added ligand and the second in the presence of 9-[(<i>N</i>-3-phosphonopropyl)-aminomethyl]-9-deazahypoxanthine (compound <b>XII</b>).<a onclick="showRef(event, 'ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref24 ref25">(24,25)</a> In both cases, the asymmetric unit (ASU) contains a dimer. Data collection, processing, and refinement statistics are given in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Data Collection, Processing, and Refinement Statistics for Two Crystal Structures of HpXGHPRT</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">no inhibitor</th><th class="colsep0 rowsep0" align="center">(+) compound <b>XII</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="3" align="center">Crystal Data</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">X-ray wavelength</td><td class="colsep0 rowsep0" align="left">0.95365</td><td class="colsep0 rowsep0" align="left">1.3776</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">unit cell length <i>a</i>, <i>b</i>, <i>c</i> (Å)</td><td class="colsep0 rowsep0" align="left">40.20, 51.54, 82.69</td><td class="colsep0 rowsep0" align="left">40.22, 51.49, 82.88</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">unit cell angle α, β, γ (deg)</td><td class="colsep0 rowsep0" align="left">90, 96.09, 90</td><td class="colsep0 rowsep0" align="left">90, 96.21, 90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">space group</td><td class="colsep0 rowsep0" align="left"><i>P</i>2<sub>1</sub></td><td class="colsep0 rowsep0" align="left"><i>P</i>2<sub>1</sub></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="3" align="center">Diffraction Data</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">temperature (K)</td><td class="colsep0 rowsep0" align="left">100</td><td class="colsep0 rowsep0" align="left">100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">resolution range (Å)</td><td class="colsep0 rowsep0" align="left">43.67–1.47 (1.49–1.47)<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">43.78–1.96 (2.08–1.96)<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">observations</td><td class="colsep0 rowsep0" align="left">389,538 (16,723)</td><td class="colsep0 rowsep0" align="left">3,335,000 (473,183)<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">unique reflections</td><td class="colsep0 rowsep0" align="left">57,196 (2566)</td><td class="colsep0 rowsep0" align="left">24,376 (2370)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">completeness (%)</td><td class="colsep0 rowsep0" align="left">99.0 (89.2)</td><td class="colsep0 rowsep0" align="left">99.5 (96.7)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>merge</sub><a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">0.080 (0.592)</td><td class="colsep0 rowsep0" align="left">0.088 (0.592)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>pim</sub><a class="ref internalNav" href="#t4fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="left">0.036 (0.264)</td><td class="colsep0 rowsep0" align="left">0.020 (0.101)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">⟨<i>I</i>⟩/⟨σ(<i>I</i>)⟩</td><td class="colsep0 rowsep0" align="left">10.6 (2.0)</td><td class="colsep0 rowsep0" align="left">74.1 (15.8)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="3" align="center">Refinement Statistics</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">resolution limits (Å)</td><td class="colsep0 rowsep0" align="left">34.54–1.47 (1.52–1.47)</td><td class="colsep0 rowsep0" align="left">19.99–1.97 (2.04–1.97)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>work</sub></td><td class="colsep0 rowsep0" align="left">17.09</td><td class="colsep0 rowsep0" align="left">15.09</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>free</sub></td><td class="colsep0 rowsep0" align="left">19.17</td><td class="colsep0 rowsep0" align="left">19.41</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">clashscore<a class="ref internalNav" href="#t4fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="left">3.12</td><td class="colsep0 rowsep0" align="left">4.38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RMSD bond lengths (Å)</td><td class="colsep0 rowsep0" align="left">0.015</td><td class="colsep0 rowsep0" align="left">0.013</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RMSD bond angles (deg)</td><td class="colsep0 rowsep0" align="left">1.36</td><td class="colsep0 rowsep0" align="left">1.06</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">subunits per ASU</td><td class="colsep0 rowsep0" align="left">2</td><td class="colsep0 rowsep0" align="left">2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">inhibitors</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">sulfate ions</td><td class="colsep0 rowsep0" align="left">4</td><td class="colsep0 rowsep0" align="left">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">water molecules</td><td class="colsep0 rowsep0" align="left">312</td><td class="colsep0 rowsep0" align="left">344</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="3" align="center">Ramachandran Statistics</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">favored</td><td class="colsep0 rowsep0" align="left">99.32</td><td class="colsep0 rowsep0" align="left">99.30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">outliers</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="3" align="center">Amino Acids with Visible Density</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">subunit A</td><td class="colsep0 rowsep0" align="left">–1 to 63; 65–153</td><td class="colsep0 rowsep0" align="left">–1 to 59; 68–153</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">subunit B</td><td class="colsep0 rowsep0" align="left">–3 to 57; 68–153</td><td class="colsep0 rowsep0" align="left">–3 to 58; 68–153</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="3" align="center"><i>B</i>-Factors (Å<sup>2</sup>)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">subunit A</td><td class="colsep0 rowsep0" align="left">23.15</td><td class="colsep0 rowsep0" align="left">21.66</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">subunit B</td><td class="colsep0 rowsep0" align="left">20.31</td><td class="colsep0 rowsep0" align="left">20.40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">sulfate</td><td class="colsep0 rowsep0" align="left">26.33</td><td class="colsep0 rowsep0" align="left">28.83</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>XII</b></td><td class="colsep0 rowsep0" align="left">not present</td><td class="colsep0 rowsep0" align="left">48.65</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">water molecules</td><td class="colsep0 rowsep0" align="left">35.91</td><td class="colsep0 rowsep0" align="left">35.95</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">The values in parentheses are for the outer resolution shell.</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">A high redundancy of data was obtained because we were anticipating the need for anomalous phasing of the data. However, the structure was ultimately solved by molecular replacement. There was strong electron density for the side chains of all the amino acids including those that are negatively charged, indicating that no radiation damage occurred during the time of data collection.</p></div><div class="footnote" id="t4fn3"><sup><sup>c</sup></sup><p class="last"><i>R</i><sub>merge</sub> = ∑<sub><i>hkl</i></sub>∑<sub><i>i</i></sub>|<i>I</i><sub><i>i</i></sub>(<i>hkl</i>) – (<i>I</i>(<i>hkl</i>))|/∑<sub><i>hkl</i></sub>∑<sub><i>i</i></sub><i>I</i><sub><i>i</i></sub>(<i>hkl</i>).</p></div><div class="footnote" id="t4fn4"><sup><sup>d</sup></sup><p class="last"><i>R</i><sub>p.i.m.</sub> = ∑<sub><i>hkl</i></sub>[1/[<i>N</i>(<i>hkl</i>) – 1]]<sup>1/2</sup>∑<sub><i>i</i></sub>|<i>I</i><sub><i>i</i></sub>(<i>hkl</i>) – (<i>I</i>(<i>hkl</i>))|/∑<sub><i>hkl</i></sub>∑<sub><i>i</i></sub><i>I</i><sub><i>i</i></sub>(<i>hkl</i>), where <i>I</i><sub><i>i</i></sub>(<i>hkl</i>) is the observed intensity and is the average intensity obtained from multiple observations of symmetry-related reflections.</p></div><div class="footnote" id="t4fn5"><sup><sup>e</sup></sup><p class="last">Clashscore is defined as the number of bad overlaps ≥0.4 Å per thousand atoms.</p></div></div></div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Subunit 3D Structure of HpXGHPRT</h3><div class="NLM_p">The subunit structure of HpXGHPRT in the absence of added ligand is shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>. The electron density map of this structure shows two tetrahedral shapes which we have ascribed to sulfate ions arising from the crystallization solution. This assignment was confirmed by an anomalous difference map showing a strong signal for sulfur in chain B at the 5′-phosphate binding pocket. There is also a weaker signal in the density in the predicted PP<sub>i</sub> binding site for sulfur but, as this is just above the background, it was less conclusive. The location of the eight completely conserved amino acids in the five 6-oxopurine PRTases in the subunit structure are also shown together with the two acidic amino acids predicted to be involved with binding a divalent metal ion.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Crystal structure of subunit A of HpXGHPRT in the absence of an inhibitor. The eight conserved residues across five 6-oxopurine PRTases are in green and the two bound sulfate ions are shown as spheres. Red indicates the location of the long flexible loop (S59–I71) which is visible in this subunit except for the main chain atoms of H64. Two highly conserved acidic amino acids, 88D and 89E in HpXGHPRT (DD in EcXGPRT and ED in the other three amino acid sequences), are in dark blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Comparison of the Subunit Structure of HpXGHPRT with the Human and Three Other Bacterial Enzymes</h3><div class="NLM_p">The secondary structure of Hp consists of four α-helices and four β-sheets and has a core structure that is similar to four other 6-oxopurine PRTases (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). The most significant structural differences between these five enzymes occur at the C-terminus (residues 122–153 in HpXGHPRT): HsHGPRT, EcHPRT, and MtHGPRT. All have two β-sheets at their C-terminal, while for EcXGPRT, the C-terminus reaches the active site of the adjoining subunit. HpXGHPRT, in contrast, has an α-helix motif of 11 amino acids at the C-terminus.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Comparison of the subunit structure of HpXGHPRT with HsHGPRT and three other bacterial 6-oxopurine PRTases. (A) HpXGHPRT (153 amino acids) and (B) human (Hs) HGPRT (218 amino acids; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RAD">4RAD</a>).<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Ligand is 9-[(<i>N</i>-phosphonoethyl-<i>N</i>-phosphonoethoxyethyl)-2-aminoethyl]-guanine; (C) EcXGPRT (152 amino acids; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1A97">1A97</a>.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> Ligand is GMP); (D) EcHPRT (178 amino acids; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1G9S">1G9S</a>.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Ligand is IMP); and (E) MtHGPRT (202 amino acids); PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RHU">4RHU</a>.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Ligand is 2-(2-(guanin-9-yl)ethyl)(2-((2-hydroxyethyl)(2-phosphonoethyl)amino)ethyl) phosphonic acid.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Quaternary Crystal Structure of HpXGHPRT</h3><div class="NLM_p">The dimeric structure of HpXGHPRT is shown in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>. The two sulfate ions are located where the phosphate groups of the two products of the reaction (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>), that is, the 5′-phosphate group of XMP/GMP/IMP and pyrophosphate, are bound.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Dimeric crystal structure of HpXGHPRT with two sulfate ions in each subunit. The red cartoon is the long flexible loop (subunit A) which is predicted to move over the active site during catalysis.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div id="sec2_9_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> Long Flexible Loop</h4><div class="NLM_p last">S59–71I in the amino acid sequence of HpXGHPRT corresponds to 105S–I117 in the human HGPRT amino acid sequence (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). This region of the human HGPRT structure is called the “long flexible loop”, which closes over the active site when the transition-state analogues are bound.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> This flexible loop is resolved and visible in one of the subunits (subunit A) of the HpXGHPRT structure (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>) but is disordered in subunit B. Thus, this structure represents two slightly different states, which are either prior to, or just after, the catalytic reaction.</div></div><div id="sec2_9_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> Two Phosphate Binding Sites</h4><div class="NLM_p">One of the sulfate ions binds in the predicted 5′-phosphate binding site and the bonding pattern that the sulfonyl oxygens make with main chain and side chain amino acids as well as water molecules is shown in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>A. The electron density for the sulfate group is unequivocal (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>A). Thus, the sulfonyl group is firmly anchored in place by a stretch of amino acids, 92-DSGNS-96. D92 and G94 are two of the completely conserved residues across these five 6-oxopurine PRTases (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a> and <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>). The second sulfate ion is found in that area of the enzyme that is predicted to bind pyrophosphate and includes R34 and G35 (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a> and <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a>B). The “RG” motif is highly conserved across 6-oxopurine PRTases, though in the human and Mt enzymes, arginine is replaced by the similarly charged lysine. R34 helps to keep a stable dimeric structure by formation of a bond between the NH1 atom of its side chain and the carbonyl of L49 in the adjoining subunit. Once again, the electron density for the sulfate ion and nearby waters is unequivocal (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>B). <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>C shows that water molecules bridge the two sulfate ions located in the active site. This is the only known structure where the 5′-phosphate binding site and a phosphate mimic in the pyrophosphate binding site are occupied.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Sulfate binding sites in HpXGHPRT (A). (Left) Interactions that the sulfonyl oxygen atoms form in the predicted 5′-phosphate binding site of HpXGHPRT. (Right) 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density for the sulfate group and four water molecules (green spheres) is overlaid as a blue mesh. These are contoured at a sigma level of 1.0. (B) (Left) The location of the second sulfate ion in HpXGHPRT. The loop in subunit A (gray; M33–G350) joins an α-helix motif to a β-sheet. Hydrogen bonding pattern when the sulfate ion binds. Subunit A is represented in blue and subunit B is represented in pink. (Right) The 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density map for the bound sulfate group and nearby waters is overlaid in blue mesh. (C) Two sulfate ions in the PP<sub>i</sub> binding site [<b>I</b>] to the sulfate ion in the 5′-phosphate binding site [<b>II</b>] and connected by a network of water molecules (green spheres).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Crystal Structure of HpXGHPRT in Complex with 9-[(<i>N</i>-3-Phosphonopropyl)-aminomethyl]-9-deazahypoxanthine (Compound <b>XII</b>)</h3><div class="NLM_p">Diffracting crystals were obtained in the presence of [(<i>N</i>-3-phosphonopropyl)-aminomethyl]-9-deazahypoxanthine, which is one of two compounds known as AIPs.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> These compounds are partial mimics of the transition-state analogues and were designed and synthesized by Schramm and his colleagues.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> They contain 9-deazahypoxathine as the purine base as well as a positively charged nitrogen in the linker connecting this base to a phosphonate group (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>A,B). The replacement of the N<sup>9</sup> atom by a carbon atom decreases the p<i>K</i><sub>a</sub> of the N<sup>7</sup> atom in the purine ring from 8.5 to 2.3<a onclick="showRef(event, 'ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref46 ref47">(46,47)</a> and, thus, this atom is protonated under physiological conditions. This protonation was designed so that a hydrogen bond could form between the carboxylate of the highly conserved aspartic acid residue (D92 in HpXGHPRT; <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>) and N<sup>7</sup> of the purine base, reflecting the prediction of the transition state. Therefore, substitution of the naturally occurring purine base by the deaza base could assist in tighter binding of the inhibitor. 9-Deazaguanine is a competitive inhibitor with guanine for HpXGHPRT having a <i>K</i><sub>i</sub> value of 3.0 ± 0.2 μM, which is similar to the <i>K</i><sub>m</sub> for guanine (3.1 μM; <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). It is likely, therefore, that 9-deazahypoxanthine would bind in the same location in the active site as the naturally occurring base.</div><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0011.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Chemical structure of two known AIPs and the proposed transition state. (A) Compound <b>XII</b>. 9-[(<i>N</i>-3-Phosphonopropyl)-aminomethyl]-9-deazahypoxanthine.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> (B) Compound <b>XIII</b>. 9-[(<i>N</i>-3-(4-Hydroxyphosphonobutyl))-aminomethyl]-9-deazahypoxanthine.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> (C) Proposed transition state of the catalytic reaction.<a onclick="showRef(event, 'ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref46 ref47">(46,47)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The positively charged nitrogen in the linker (<b>XII</b> and <b>XIII</b>) mimics that of the oxygen in the chemical structure of the transition state during the catalytic reaction (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>C). Its primary role appears to be in positioning a pyrophosphate group in the active site.</div><div class="NLM_p">There is only one published crystal structure of a 6-oxopurine PRTase in complex with an AIP. This is PfHGXPRT in complex with <b>XIII</b>.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> This complex also contains one pyrophosphate molecule and one magnesium ion per subunit.</div><div class="NLM_p">Compound <b>XII</b> is a competitive inhibitor of PfHGXPRT and HsHGPRT and is highly selective for the Pf enzyme.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> The <i>K</i><sub>i</sub> values of <b>XII</b> for HsHGPRT and PfHGXPRT are 4940 and 11 nM, respectively. The <i>K</i><sub>i</sub> value of <b>XII</b> for HpXGHPRT is 2400 ± 0.4 nM, a value closer to that of the human enzyme. In comparison, the <i>K</i><sub>i</sub> values of <b>XIII</b> for HsHGPRT and PfHGXPRT are much lower, being 385 and 0.65 nM, respectively, suggesting that the addition of the −CH<sub>2</sub>OH group plays a significant role in determining the <i>K</i><sub>i</sub> value.</div><div class="NLM_p"><a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>A shows the Connolly surface of <b>XII</b> and sulfate bound in the active site of HpXGHPRT (subunit A). <b>XII</b> was not observed in subunit B and just two sulfate ions were found in this subunit. The specific interactions that <b>XII</b> forms when bound in the active site of HpXGHPRT are shown in <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>B.</div><figure id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0012.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Crystal structure of HpXGHPRT in complex with <b>XII</b>. (A) Connolly surface of HpXGHPRT dimer in complex with <b>XII</b> and one sulfate ion with subunit B and two sulfate ions in subunit A. (B) <b>XII</b> bound in the active site of HpXGHPRT and the interactions that it forms with backbone and side-chain atoms of amino acids and also water molecules. Green dashed lines show the hydrogen bond network; red dashed lines show the distances in Å between the ligand and the side chain atoms of two highly conserved amino acids, D92 and K121.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In this structure, the side-chain atom of D92 has not moved to form a hydrogen bond with the N<sup>7</sup> atom of the purine ring as occurs when compound <b>XIII</b> (together with pyrophosphate and a magnesium ion) or the transition-state analogue (together with pyrophosphate and two magnesium ions) bind to either human HGPRT or PfHGXPRT (distance is 2.8 Å in both those cases). Further, the NZ atom of the invariant lysine residue, K121, is far away to form a hydrogen bond with the 6-oxo group of 9-deazahypoxanthine.</div><div id="sec2_10_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> Positively Charged Nitrogen in a Linker</h4><div class="NLM_p"><a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>A shows the Connolly surface of PfHGXPRT in complex with <b>XIII</b> and shows that the large flexible loop is firmly closed over the active site and the ligand is “buried” in the active site.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> The HpXGHPRT·<b>XII</b> complex has been superimposed onto this complex to compare their modes of binding in an attempt to provide a rationale for the range of inhibition constants. <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>B,C shows the interactions that the two ligands make in the active sites of their respective enzymes.</div><figure id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0013.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Interactions between two ANPs and HpXGHPRT and PfHGXPRT. (A) Connolly surface of the crystal structure of the PfHGXPRT·<b>XIII</b> complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3OZG">3OZG</a>) with the structure of the HpXGHPRT·<b>XII</b> complex superimposed. The green sphere represents a magnesium ion. (B) The interactions that the positively charged nitrogen atom of <b>XII</b> forms in the active site of HpXGHPRT. (C) The interactions that the positively charged nitrogen atom of <b>XIII</b> forms in the active site of PfHGXPRT. All distances shown are in Å. Water molecules are shown as red spheres in (B,C).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In the HpXGHPRT·<b>XII</b> complex, the positively charged nitrogen is linked, via two bridging waters, to two of the oxygens of the sulfate group located in the pyrophosphate binding site. One of these water molecules is then linked to the OD1 atom of the conserved acidic residue, E89 (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>A,B). The location of this sulfate ion mimics that of the second phosphate group of pyrophosphate, which is found in the active site of the PfHGXPRT·<b>XIII</b> complex (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>A,C).<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> In the HpXGHPRT·<b>XII</b> structure, the second water molecule fulfills the role of one of the oxygen atoms of the pyrophosphate group in the PfHGXPRT·<b>XIII</b> complex (cf. <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>B,C).</div><div class="NLM_p">Thus, the role of the positively charged nitrogen atom in both these structures is to contribute to holding the sulfate ion (or a phosphate ion which can substitute for sulfate) or PP<sub>i</sub> (PfHGXPRT). It also helps in positioning the side chain of the conserved acidic residue (cf. <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>B,C). When the transition-state analogue, ImmHp, binds to PfHGXPRT (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1CJB">1CJB</a>)<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> together with pyrophosphate and two magnesium ions, a similar role is attributed to the positively charged atom as it is also linked to the phosphoryl oxygens of pyrophosphate.</div><div class="NLM_p last">Thus, the HpXGHPRT·<b>XII</b> complex is a partial reflection of the transition state in catalysis of this enzyme. Attempts to crystallize with pyrophosphate included in the active site together with compound <b>XII</b> were unsuccessful. This is attributed to the fact that this binding site may have been already occupied by sulfate and pyrophosphate was unable to displace it.</div></div><div id="sec2_10_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> Positioning of the Purine Ring</h4><div class="NLM_p">In subunit B of the HpXGHPRT structure in the absence or presence of a ligand, the side chain of W138 forms a π-stacking arrangement with the side chain of a histidine residue (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>). This residue is two residues distal to the beginning of the authentic amino acid sequence (HYSY) and is the fifth histidine residue at the N-terminus in the added hexa-his tag. In subunit A, in the absence of the inhibitor, the same histidine residue has not formed this packing arrangement and is freely rotating. Hence, its side chain is not visible and has been modeled in as alanine (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>). The side chain of W138 is then not restricted and freely rotates (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>). The <i>B</i>-factor for this side chain is 62 Å<sup>2</sup> indicative of its flexibility. In contrast, the <i>B</i>-factor in chain A is 22 Å<sup>2</sup> as it is firmly clamped in its position.</div><figure id="fig13" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0014.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. Movement of the side chain of the N-terminal and the side chain of W138 in HpXGHPRT. (Left) In the absence of inhibitor. Subunit A is orange; subunit B is gold. In subunit B, there is no density for the side chain of His(−2) and this side chain is modeled as alanine. (Right) In the presence of <b>XII</b> in the active site. Subunit A is represented in pink; subunit B is represented in silver. (−2) signifies the number of amino acids distal to the beginning of the integral amino acid sequence (HYSY).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In all the 6-oxopurine PRTases, the purine base of the substrate or inhibitor is also “locked” into position by a π-stacking arrangement with the side chain of an aromatic amino acid. In EcXGPRT, this amino acid residue is tryptophan, while in HsHGPRT,<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> EcHPRT,<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> and MtHGPRT,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> it is phenylalanine (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). The aromatic amino acid is located on a flexible loop where it is freely rotatable and it only moves over the active site when a purine base, either free as in the PfHGXPRT·Hx·PP<sub>i</sub>·Mg<sup>2+</sup> complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3OZF">3OZF</a>)<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> or as the nucleoside monophosphate or a phosphonate inhibitor, enters the active site of these enzymes.<a onclick="showRef(event, 'ref22 ref48'); return false;" href="javascript:void(0);" class="ref ref22 ref48">(22,48)</a></div></div><div id="sec2_10_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Xanthine Is a Substrate for HpXGHPRT</h4><div class="NLM_p">Xanthine binds to HpXGHPRT and EcXGPRT with similar <i>K</i><sub>m</sub> values, 20 and 30 μM, respectively (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). However, this purine base is not a substrate for human HGPRT. In the EcXGPRT·GMP complex, the exocyclic amino group forms two hydrogen bonds, one with a bridging water to the OD1 atoms of E136 and D140 and the second to the carbonyl of I135. When xanthine binds, the exocyclic oxygen atom forms a bond with the water molecule, but it is not possible to make a bond with the carbonyl of I135, so the position of the base shifts slightly to avoid repulsion. In the HpXGHPRT·<b>XII</b> complex, there is only a proton in the 2-position of the purine ring. In this structure, the side chain of E144 points toward the active site. This makes the distance between C<sup>2</sup> and the OD oxygen of E144, 3.4 Å. Thus, there is room to accommodate an exocyclic oxo group in this area of the active site and accounts for the fact that xanthine is a good substrate for this enzyme.</div><div class="NLM_p">The Cα backbone of human HGPRT adopts a different configuration in this region (cf. <a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">14</a></a>A,B). The exocyclic amino group of, the inhibitor, compound <b>I</b> (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>) forms a hydrogen bond with the carbonyl of D193. Thus, if xanthine tried to bind to this enzyme, repulsion would exist between the exocyclic oxygen atom and the carbonyl oxygen of D193 with the result that this purine base cannot bind to the human enzyme.</div><figure id="fig14" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0015.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 14. Comparison of the position of the purine base in the HpXGHPRT, EcXGPRT, and HsHGPRT complexes. (A) Interactions that the N<sup>1</sup> atom in the purine ring makes when the exocyclic amino group at the C<sup>2</sup> position of the purine base makes when <b>XII</b> or GMP are bound in the active site of HpXGHPRT or EcXGPRT (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1A97">1A97</a>), respectively.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> Pink represents HpXGHPRT and blue represents EcXGPRT. The blue sphere is the bridging water between the exocyclic amino group and the two amino acid side chains in the EcXGPRT·GMP complex. (B) Human HGPRT in complex with the inhibitor, [3<i>R</i>,4<i>R</i>-4-guanin-9-yl-3-((<i>S</i>)-2-hydroxy-2-phosphonoethyl)oxy-1-<i>N</i>-(phosphonopropionyl)pyrrolidine (compound <b>1</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5HIA">5HIA</a>).<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig14"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_10_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> Binding Site for a Divalent Metal Ion</h4><div class="NLM_p">In 6-oxopurine PRTases, there are two highly conserved acidic amino acid residues (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). These are D88 and E89 in HpXGHPRT (<a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">15</a></a>). They occupy a similar location in subunits A and B in both of the two HpXGHPRT structures. These two corresponding amino acid residues in the 6-oxopurine PRTase structures have been found to be directly coordinated to magnesium in the absence of a ligand<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> or in the presence of a phosphonate inhibitor.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> When the transition-state analogues are bound, they are indirectly coordinated to a magnesium ion via a hydrogen bond to the iminoribitol hydroxyl groups.<a onclick="showRef(event, 'ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref46 ref47">(46,47)</a> In contrast, there is no electron density found in the two high-resolution structures of HpXGHPRT to show the presence of metal ion within the coordination distance of the side chains of these two residues and, instead, they are turned toward each other and, via water molecules, to the sulfonyl oxygens of the sulfate group located in the binding pocket of the β-phosphate pocket and the carbonyl of V82. This residue is conserved as either valine or leucine in this group of 6-oxopurine PRTases (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>).</div><figure id="fig15" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 15</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0016.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 15. Interactions that the side chains of D88 and E89 form at the active site of HpXGHPRT (subunit A, no inhibitor present). This sulfate ion is located in the pyrophosphate binding pocket.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig15"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_10_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> Role of K121</h4><div class="NLM_p">K121 in HpXGHPRT is highly conserved across 6-oxopurine PRTases (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). It has been proposed that its role is to confer the specificity for this class of enzymes by forming a hydrogen bond between the NZ atom and the exocyclic oxo group. Thus, 6-oxopurine phosphoribosyltransferases are unable to use adenine as a substrate because repulsion would occur with the exocyclic amino group.</div><div class="NLM_p">In the HpXGHPRT complex, the NZ atom of K121, which is located on a flexible loop, is 5.9 Å away from the 6-oxo atom of the purine ring and, consequently, unable to form a bond with the 6-oxo group (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>). Instead, the 6-oxo exocyclic group forms a hydrogen bond with a bridging water molecule and, via this molecule, to the backbone carbonyl of the conserved proline residue, P136 (<a data-tab="pane-pcw-Figures" href="#fig16" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig16" id="" class=" internalNav">16</a></a>). The NZ atom of K121 forms a hydrogen bond with a water molecule, which is then linked to the carbonyl backbone oxygen of A137. This network appears to be the basis for the 6-oxopurine specificity of the Hp enzyme. HpXGHPRT retains its specificity, though for the 6-oxo group, and adenine is not a substrate. HpXGHPRT (360 nM, final concentration) was added to the cuvette containing 0.1 M Tris-HCl, 0.01 M MgCl<sub>2</sub>, pH 7.4, 114 μM adenine, and 550 μM PRib-PP at 25 °C and the reaction followed spectrophotometrically at 255 nm for 600 s. The concentration of adenine is 16 times that of the <i>K</i><sub>m</sub> value for guanine, 25 times higher than for Hx, and 50 times higher than for xanthine (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) and the enzyme concentration is 10 times than when guanine was used as a substrate. However, there was no change in absorbance (<0.001) over this time period, so the enzyme is unable to convert adenine to AMP. Thus, the method adopted by HpXGHPRT to bind the 6-oxopurine base has not broadened its specificity for the purine base, and it retains its specificity for an exocyclic 6-oxo group.</div><figure id="fig16" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Figure 16</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0017.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 16. Comparison of inhibitor binding to two 6-oxopurine PRTases. (A) Superimposition of HpXGHPRT·<b>XII</b> complex, EcXGPRT·GMP complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1A97">1A97</a>), and EcXGPRT.(<i>S</i>)-3-(guanin-9-yl)pyrrolidine-<i>N</i>-ylacetylphosphonic acid complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4JIT">4JIT</a>).<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> XGPRT and GMP are represented in blue; (<i>S</i>)-3-(guanin-9-yl)pyrrolidine-<i>N</i>-ylacetylphosphonic acid is represented in green; HpXGHPRT and <b>XII</b> are represented in pink. The purine ring and the five-membered ring of the Ec complexes are located in identical locations. (B) Hydrogen bond network that K121 forms at the active site in the HpXGHPRT·<b>XII</b> complex and the distances in angstroms between the NZ atom of this amino acid. The distance between the OD atom of D92 and the N<sup>7</sup> atom of the purine ring is also shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig16"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">EcXGPRT also appears to adopt a modified network of bonds and is maybe less reliant that the other 6-oxopurine PRTases on the ubiquitous lysine residue.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> The distance between the 6-oxo group and the NZ atom is 3.6 Å when GMP is bound, but this is 3.2 Å in the presence of the inhibitor, (<i>S</i>)-3-(guanin-9-yl)pyrrolidine-<i>N</i>-ylacetylphosphonic acid (<a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">14</a></a>). This bacterial enzyme may place greater reliance on the hydrogen bond with the amide of I135.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> This residue is highly conserved (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>) and completely conserved between EcXGPRT and HpXGHPRT, but it does not perform the same role in the Hp enzyme as it is located too far away from the 6-oxogroup. Thus, though the proline amino acid residue is completely conserved across this group of the 6-oxopurine PRTases, it is only in the HpXGHPRT enzyme that it appears to have an important role to play in the enzyme function.</div></div></div><div id="sec2_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Derivative of Compound <b>XII</b></h3><div class="NLM_p">A derivative of <b>XII</b> was synthesized with the aim of positioning a second phosphonate group in the pyrophosphate binding pocket. The chemical structure of this compound (<b>XIV</b>) is given in <a data-tab="pane-pcw-Figures" href="#fig17" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig17" id="" class=" internalNav">17</a></a>.</div><figure id="fig17" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Figure 17</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0018.jpeg" id="gr17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 17. Chemical structure of <b>XIV</b>, 9-[(<i>N</i>,<i>N</i>-(bis-3-phosphonopropyl))-aminomethyl]-9-deazahypoxanthine.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig17"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">This molecule inhibited HpXGHPRT with a <i>K</i><sub>i</sub> of 4 ± 0.7 μM. Thus, though this modification did not result in the desired decrease in the inhibition constant, compound <b>XIV</b> does not inhibit human HGPRT, even at a concentration of 46 μM in the assay. Thus, like xanthine and its derivatives, it is highly selective for HpXGHPRT. This result suggests possibilities for the design of inhibitors totally specific for the bacterial enzyme. Thus, like 6-thioxanthine, this compound can constitute a lead molecule for therapeutics.</div></div><div id="sec2_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Possible Binding Modes of Compounds <b>I</b> and <b>III</b> to HpXGHPRT</h3><div class="NLM_p">In an attempt to explain the lower <i>K</i><sub>i</sub> values of compounds <b>I</b> and <b>III</b> compared with compound <b>XII</b>, two known structural complexes were superimposed onto that of the HpXGHPRT·<b>XIII</b> complex. These are the human HGPRT·<b>I</b> complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5HIA">5HIA</a>)<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> and the MtHGPRT·<b>III</b> complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RHX">4RHX</a>)<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> (<a data-tab="pane-pcw-Figures" href="#fig18" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig18" id="" class=" internalNav">18</a></a>).</div><figure id="fig18" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Figure 18</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0019.jpeg" id="gr18" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 18. Superimposition of the complexes of two known crystal structures onto that of the HpXGHPRT·<b>XII</b> complex. (A) Human HGPRT·<b>I</b> complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5HIA">5HIA</a>: green) with the HpXGHPRT·<b>XII</b> complex. (B) MtHGPRT·<b>III</b> complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RHX">4RHX</a>) with the HpXGHPRT·<b>XII</b> complex. A putative location of the pyrophosphate was obtained by superimposition of the human HGPRT structure in complex with the transition state analogue (ImmHP) and PP<sub>i</sub>. (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BZY">1BZY</a>).<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig18"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Both <b>I</b> and <b>III</b> contain a second phosphonate group covalently attached to a nitrogen atom in the pyrrolidine ring (<a data-tab="pane-pcw-Figures" href="#fig18" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig18" id="" class=" internalNav">18</a></a>A) or in the acyclic linker (<a data-tab="pane-pcw-Figures" href="#fig18" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig18" id="" class=" internalNav">18</a></a>B) (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). In the human HGPRT·<b>I</b> complex, this phosphonate group reaches further into the putative pyrophosphate binding site than that of the phosphonate group in the MtHGPRT·<b>III</b> complex. In both cases, the phosphonyl oxygens would be expected to form additional interactions in the active site compared with compound <b>XII</b>, thus contributing to the decrease in the <i>K</i><sub>i</sub> values (2.4 μM for <b>XII</b> cf. 0.2 μM for <b>I</b> and 0.3 μM for <b>III</b>; <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). These superimpositions are possible binding modes to show that these two compounds bind in the active site of HpXGHPRT, but other variations are possible given the flexible nature of the active site of the enzyme and the compounds themselves. Nonetheless, the superposition provides a basis for proposing future chemical modifications to increase the affinity.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78678" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78678" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">HpXGHPRT is one of the smallest and most compact of 6-oxopurine PRTases with a subunit mass of 17,327 Da and exists, in solution, as an active dimer. It is catalytically efficient, using all the three naturally occurring purine bases with <i>k</i><sub>cat</sub>/<i>K</i><sub>m</sub> values of 2.3, 0.5, and 0.1 μM<sup>–1</sup> s<sup>–1</sup> for guanine, hypoxanthine, and xanthine, respectively. These kinetic properties suggest that the nucleoside phosphonates, a class of inhibitors of other 6-oxopurine PRTases, may also inhibit HpXGHPRT activity. All 12 such compounds investigated here were found to inhibit this enzyme with the lowest <i>K</i><sub>i</sub> value being 200 nM. Prodrugs, designed to increase membrane transport of these inhibitors, arrested the growth of Hp in cell-based assays. The catalytic properties of HpXGHPRT can also be used to convert foreign substrates into toxic metabolites, which are shown to be effective in vitro against Hp growth. Thus, the enzymatic properties of HpXGHPRT can be used in two ways to prevent the growth of Hp. The inhibitors and the substrate analogue have low to negligible toxicity in human cells. The first crystal structures of HpXGHPRT have been obtained. These structures provide clues for chemical modifications to the inhibitors to increase their potency. The innovative therapeutics described here are specific for HpXGPRT as they are based on the chemical structure of its substrates/products. As such, it is unlikely that they could potently bind to other proteins. The enzymological properties of HpXGHPRT combined with its 3D structure thus provide a platform for the design of new chemotherapeutics aimed at the treatment of Hp infections.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34077" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34077" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Expression and Purification of HpXGHPRT</h3><div class="NLM_p last">Hexa-his tag HpXGHPRT was expressed and purified to homogeneity as for human hexa-his tag HGPRT.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> The purified enzyme was stored at −80 °C in 0.1 M Tris-HCl, 0.1 M MgCl<sub>2</sub>, pH 7.4, containing 1 mM DTT, and 400 μM PRib-PP. The concentration of protein was calculated using an <i>A</i>1cm<sup>1%</sup> of 1.38 M<sup>–1</sup> cm<sup>–1</sup> and a subunit molecular mass of 18,481.1 Da.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Determination of Kinetic Constants</h3><div class="NLM_p last">The kinetic constants, <i>k</i><sub>cat</sub> and <i>K</i><sub>m</sub>, were measured using a continuous spectrophotometric assay in 0.1 M Tris-HCl, 0.01 M MgCl<sub>2</sub>, pH 7.4, and 25 °C. The reaction was initiated by the addition of an aliquot of enzyme to the cuvette containing the assay buffer and the two substrates, the purine base and PRib-PP. The reaction was followed for 60 s over the absorbance range of 0–0.05 a.u. The reaction for each base was followed at these wavelengths: guanine (257.5 nm), hypoxanthine (245 nm), xanthine (255 nm), 6-mercaptopurine (300 nm), 6-thioguanine (355 nm), 6-thioxanthine (355 nm), and adenine (255 nm). The Δε values for the different bases are guanine (5816.5 M<sup>–1</sup> cm<sup>–1</sup>), hypoxanthine (2439 M<sup>–1</sup> cm<sup>–1</sup>), xanthine (4685 M<sup>–1</sup> cm<sup>–1</sup>), 6-mercaptopurine (3284 M<sup>–1</sup> cm<sup>–1</sup>), 6-thioguanine (4310 M<sup>–1</sup> cm<sup>–1</sup>), 6-thioxanthine (3740 M<sup>–1</sup> cm<sup>–1</sup>), and adenine (3300 M<sup>–1</sup> cm<sup>–1</sup>).<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> The <i>K</i><sub>m</sub> values for the bases were measured under saturating conditions of PRib-PP (580 μM), and the <i>K</i><sub>m</sub> values for PRib-PP under saturating conditions of guanine, 64 μM. All experiments were performed as technical replicates in triplicate. The kinetic constants were calculated using GraphPad Prism.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Determination of <i>K</i><sub>i</sub> Values</h3><div class="NLM_p last">The two substrates and the inhibitor were incubated in the assay buffer, 0.1 M Tris-HCl, 0.01 M MgCl<sub>2</sub>, pH 7.4, at 25 °C, and the reaction initiated by the addition of an aliquot of enzyme. <i>K</i><sub>m(app)</sub> was first calculated at different concentrations of inhibitor using the equation, <i>v</i><sub>o</sub> = <i>V</i><sub>max</sub>·[S]<sub>o</sub>/<i>K</i><sub>m(app)</sub> + [S]<sub>o</sub>. The concentration of inhibitor was then chosen so that the <i>K</i><sub>m(app)</sub> was at least 5 times the concentration of the variable substrate. The <i>K</i><sub>i</sub> values were determined at a constant concentration of guanine (64 μM) and a range of concentrations of PRib-PP (10–1200 μM). The range of inhibitor concentrations varied from 1 to 140 μM, depending on the degree of inhibition by that particular inhibitor. Initial velocity (<i>v</i><sub>o</sub>) was determined at the substrate saturation concentration. All experiments were performed as technical replicates in triplicate. For compound <b>XII</b>, the <i>K</i><sub>i</sub> was calculated using the Morrison equation<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> as for the inhibition of the human and Pf 6-oxopurine PRTases by this compound.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Gel Filtration</h3><div class="NLM_p last">Gel filtration was performed using size exclusion chromatography coupled to multi-angle light scattering (SEC–MALS) and with a G-75 Sephadex column equilibrated in 0.1 M Tris-HCl, 0.01 M MgCl<sub>2</sub>, pH 7.4 containing 1 mM DTT.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Crystallization and Structure Determination</h3><div class="NLM_p">The concentration of enzyme used to obtain highly diffracting crystals was 0.22 mM in terms of subunits (i.e., 4 mg mL<sup>–1</sup>). To obtain crystals for the HpXGHPRT·<b>XII</b> complex, the enzyme was incubated with or without <b>XII</b> (5.75 mM) on ice for 5 min prior to setting up the crystal trays. The crystallization conditions were found by using the crystallization screening plates prepared by the Mosquito protein crystallization robot with 100 nL of the screening condition and 100 nL of protein. The crystallization screen was JCSG Plus from Molecular Dimensions. The crystallization conditions are 0.1 M bis–Tris-propane, pH 5.5, 0.2 M (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, and 25% PEG3350. HpXGHPRT crystals, in the presence and absence of <b>XII</b>, were obtained using the hanging drop method with 2 μL of enzyme and 2 μL of well solution. The trays were incubated at 18 °C and crystals became visible after 14 days. The cryoprotectant was 10% glycerol in the well solution to which was added compound <b>XII</b> to afford a final concentration of 1 mM. Crystals were cryocooled in liquid nitrogen for transport to the Australian Synchrotron where they were robotically placed in a cryostream (100 K). X-ray data were collected remotely at the MX2 beamline.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> All data sets were processed with XDS.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a></div><div class="NLM_p last">The structure of the HpHGXPRT·<b>XII</b> complex was determined by molecular replacement with Auto-Rickshaw<a onclick="showRef(event, 'ref53 ref54'); return false;" href="javascript:void(0);" class="ref ref53 ref54">(53,54)</a> with MoRDa<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> identifying and using the PDB accession code, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5BQP">5BQP</a>, <i>Sulfolobus solfataricus</i> HGXPRT as a template model. The template model indicated about 27% sequence identity against the target molecule. The resulting model was subjected to rigid body, positional and <i>B</i>-factor refinement in CNS<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> and continued to refine with REFMAC5<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> resulting in <i>R</i>/<i>R</i><sub>free</sub> to 45.6/49.5%. At this stage, density modification was carried out with the model phase in PIRATE<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> and the improved phases were used for model building in ARP/wARP.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> The initial model contained 246 out of total 306 residues and of which 226 residues were correctly fitted in the electron density and refinement of the model resulted in <i>R</i>/<i>R</i><sub>free</sub> to 27.4/31.4%. Subsequent refinement and manual model building were performed with PHENIX 1.8<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> and COOT 0.8,<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> respectively. The liganded structure was solved by MR using the program PHASER<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> and the unliganded structure of HpXGHPRT as a starting model within PHENIX 1.8<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> and refinement and model building were carried out as described above. The data collection, processing, and refinement statistics for all of the data sets are presented in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Hp In Vitro Assays</h3><div class="NLM_p">Stock solutions of the prodrug (50 mM) were prepared in dimethyl sulfoxide (DMSO). These solutions were then diluted further to give a final concentration of 1 mM in DMSO. Aqueous stock solutions of the purine bases (approx. 2 mM) were also diluted to give a final concentration of 1 mM.</div><div class="NLM_p">The Hp strain 26695 was cultivated under microaerobic conditions (6% O<sub>2</sub>, 6% CO<sub>2</sub>, and 88% N<sub>2</sub>) at 37 °C in brain heart infusion (BHI) broth supplemented with 10% (v/v) fetal calf serum and 0.2% (v/v) Skirrow’s supplement.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> Cell suspensions were diluted with the BHI media to give a cell density of 10<sup>6</sup> bacteria/mL.</div><div class="NLM_p last">Horse blood agar (950 μL), containing 8% (v/v) horse blood and 0.2% (v/v) Skirrow’s supplement,<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> were added into each well. The compounds (50 μL) or DMSO (50 μL) was added into each well. DMSO alone was added to determine that any decrease in cell growth was not due to DMSO itself. The final concentration of DMSO was 5%. To each well, 10 μL of the cell suspension was then added. The plates were then incubated at 37 °C for three days in microaerobic condition (6% O<sub>2</sub>, 6% CO<sub>2</sub>, and 88% N<sub>2</sub>). Cell growth was then observed.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Bioinformatic Analysis</h3><div class="NLM_p last">The protein sequences for HpXGHPRT, EcXGPRT, HsHGPRT, EcHPRT, and MtHGPRT were obtained from UniProtKB.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> ClustalW2 was used to perform the sequence alignments of the enzyme.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a></div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Synthesis of the Inhibitors and Their Prodrugs</h3><div class="NLM_p">Except for the three prodrugs, <b>II-A</b>, <b>VI-C</b>, and <b>VI-D</b> (synthesis described in the paragraphs below), the inhibitors and their prodrugs (listed below) were synthesized as described previously. They were all stored at 5 °C, under which conditions, they are stable for >10 years. These compounds are >95% pure as assessed by liquid chromatography/mass spectrometry (LC/MS). [3<i>R</i>,4<i>R</i>]-4-Guanin-9-yl-3-((<i>S</i>)-2-hydroxy-2-phosphonoethyl)oxy-1-<i>N</i>-(phosphonopropionyl)pyrrolidine (<b>I</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a><a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a>); tetra-(<span class="smallcaps smallerCapital">l</span>-phenylalanine ethyl ester) prodrug of [3<i>R</i>,4<i>R</i>]-4-guanin-9-yl-3-((<i>S</i>)-2-hydroxy-2-phosphonoethyl)oxy-1-<i>N</i>-(phosphonopropionyl)pyrrolidine (<b>I-A</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a><a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a>); tetra-(<span class="smallcaps smallerCapital">l</span>-alanine ethyl ester) prodrug of [3<i>R</i>,4<i>R</i>]-4-guanin-9-yl-3-((<i>S</i>)-2-hydroxy-2-phosphonoethyl)oxy-1-<i>N</i>-(phosphonopropionyl)pyrrolidine (<b>I-B</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a><a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a>); (2-((2-(guanin-9-yl)ethyl)(2-((2-hydroxypropyl)(2-phosphonoethyl)amino)ethyl)amino)ethyl)phosphonic acid (<b>II</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a><a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a>); (2-((2-(guanin-9-yl)ethyl)(2-((2-hydroxyethyl)(2-phosphonoethyl)amino)ethyl)amino)ethyl)phosphonic acid (<b>III</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a><a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a>); tetra-(<span class="smallcaps smallerCapital">l</span>-phenylalanine ethyl ester) prodrug of (2-((2-(guanin-9-yl)ethyl)(2-((2-hydroxyethyl)(2-phosphonoethyl)amino)ethyl)-amino)ethyl)phosphonic acid (<b>III-A</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a><a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a>); sodium salt of [3-(guanine-9-yl)-2-((2-phosphonoethoxy)methyl)propoxy]methyl phosphonic acid (<b>IV</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a><a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a>); tetra-(<span class="smallcaps smallerCapital">l</span>-phenylalanine ethyl ester) prodrug of [3-(guanine-9-yl)-2-((2-phosphonoethoxy)methyl)propoxy]methyl-phosphonic acid (<b>IV-A</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a><a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a>); 9-[(<i>N</i>-phosphonoethyl-<i>N</i>-phosphonoethoxyethyl)-2-aminoethyl]hypoxanthine (<b>V</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a><a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a>); tetra-(<span class="smallcaps smallerCapital">l</span>-phenylalanine ethyl ester) prodrug of 9-[(<i>N</i>-phosphonoethyl-<i>N</i>-phosphonoethoxyethyl)-2-aminoethyl]hypoxanthine (<b>V-A</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a><a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a>); diisopropyl (<i>S</i>)-((2-(guanin-9-yl)pyrrolidine-1-yl)-2-oxoethyl)phosphonate (<b>VI</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a><a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a>); sodium salt of [(2-(guanine-9-yl)propane-1,3-diyl)bis(oxy)]bis(ethane-2,1-diyl) diphosphonic acid (<b>VII</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a><a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a>); tetra-(<span class="smallcaps smallerCapital">l</span>-phenylalanine ethyl ester) prodrug of [(2-(guanine-9-yl)propane-1,3-diyl)bis(oxy)]bis(ethane-2,1-diyl)diphosphonic acid (<b>VII-A</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a><a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a>); [(2-[(guanin-yl)methyl]propane-1,3-diyl)bis(oxy)]bis(methylene) diphosphonic acid (<b>VIII</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a><a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a>); tetra-(<span class="smallcaps smallerCapital">l</span>-phenylalanine ethyl ester) prodrug of [(2-[(guanin-9-yl)methyl]propane-1,3-diyl)bis(oxy)]bis(methylene) diphosphonic acid (<b>V-IIIA</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a><a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a>); sodium salt of [3-(8-bromoguanine-9-yl)-2-((2-phosphonoethoxy)methyl)propoxy]methyl) phosphonic acid (<b>IX</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a><a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a>); tetra-(<span class="smallcaps smallerCapital">l</span>-phenylalanine ethyl ester) prodrug of [3-(8-bromoguanine-9-yl)-2-((2-phosphonoethoxy)methyl)propoxy]methyl phosphonic acid (<b>IX-A</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a><a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a>); (2-((2-(benzyloxy)ethyl)(2-(guanin-9-yl)ethyl)amino)ethyl)phosphonate (<b>X</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a><a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a>); bis-(<span class="smallcaps smallerCapital">l</span>-phenylalanine isopropyl ester) prodrug of (2-((2-(benzyloxy)ethyl)(2-(guanin-9-yl)ethyl)amino)ethyl)phosphonate (<b>X-A</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a><a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a>); 9-[(<i>N</i>-phosphonoethyl-<i>N</i>-phosphonoethoxyethyl)-2-aminoethyl]guanine (<b>XI</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a><a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a>); tetra-(<span class="smallcaps smallerCapital">l</span>-phenylalanine ethyl ester) prodrug of 9-[(<i>N</i>-phosphonoethyl-<i>N</i>-phosphonoethoxyethyl)-2-aminoethyl]guanine (<b>XI-A</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a><a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a>); 9-[(<i>N</i>-3-phosphonopropyl)-aminomethyl]-9-deazahypoxanthine (<b>XII</b>, <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a><a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a>); and 9-[(<i>N</i>,<i>N</i>-(bis-3-phosphonopropyl))-aminomethyl]-9-deazahypoxanthine (<b>XIV</b>, <a data-tab="pane-pcw-Figures" href="#fig17" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig17" id="" class=" internalNav">17</a></a><a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a>) were prepared previously as described previously (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>).</div><div id="sec4_8_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> Synthesis of Tetra-(<span class="smallcaps smallerCapital">l</span>-phenylalanine ethyl ester) Prodrug of (2-((2-(Guanin-9-yl)ethyl)(2-((2-hydroxypropyl)(2-phosphonoethyl)amino)ethyl)amino)ethyl)phosphonic Acid (<b>II-A</b>)</h4><div class="NLM_p last">A mixture of diethyl (2-((2-(guanin-9-yl)ethyl)(2-((2-hydroxypropyl)(2-phosphonoethyl)amino)ethyl)amino)ethyl)phosphonate,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> (0.5 mmol), dry pyridine (8 mL), and BrSiMe<sub>3</sub> (1 mL) was stirred overnight at room temperature under argon. After evaporation and co-distillation with pyridine (2 × 5 mL) under an argon atmosphere, the residue was dissolved in dry pyridine (10 mL) and ethyl (<span class="smallcaps smallerCapital">l</span>)-phenylalanine hydrochloride (1.75 g, 7.1 mmol) and triethylamine (3.1 mL) were added. The mixture was heated to 70 °C under an argon atmosphere and then a solution of aldrithiol (2.31 g, 10.5 mmol) and triphenylphosphine (2.75 g, 10.5 mmol) in dry pyridine (8 mL) was added. The reaction mixture was heated at 70 °C for 3 days, the solvent was evaporated, and the residue co-evaporated with toluene (2 × 10 mL) and purified by column chromatography on silica gel (gradient CHCl<sub>3</sub>–MeOH). The crude product further purified by preparative high-performance LC (HPLC). The phosphoramidate prodrug was obtained as foam, yield 34%. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): 1.13 (m, 12H, Et); 1.63 (m, 4H, CH<sub>2</sub>–P); 2.61 (m, 8H) and 2.76 (m, 4H, CH<sub>2</sub>–N); 2.95 (m, 10H, CH<sub>2</sub>Ph and CH<sub>2</sub>); 3.48 (m, 2H, H-11′); 4.04 (m, 14H, Et, CHNH and H-1′); 4.23 (m, 4H, NH); 5.91 (s, 1H, OH); 6.21 (s, 2H, NH<sub>2</sub>); 7.24 (m, 20H, Ar); 7.60 (s, 1H, H-8); 10.31 (s, 1H, NH). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): 13.47 (m, 4C, Et); 26.43 (d, 2C, <i>J</i><sub>P,C</sub> = 111.5, C–P); 43.71 (C-1′); 47.23; 50.24, 50.36 (2C), 50.75, 52.27 and 53.39 (C–N); 53.88 (m, 4C, NHCH); 58.82 (C–OH); 60.00 (m, 4C, Et); 116.57 (C-5); 126.08 (m, 4C, Ar); 127.72 (m, 8C, Ar); 128.92 (m, 8C, Ar); 136.90 (m, 4C, Ar); 136.96 (C-8); 150.91 (C-4); 153.27 (C-2); 156.45 (C-6); 172.53 m, 4C (CO). MS (ESI): <i>m</i>/<i>z</i> = 1212 [M + H]<sup>+</sup>. This compound is >95% pure as assessed by LC/MS.</div></div><div id="sec4_8_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> Synthesis of Bis-(2-pivaloylthioethyl) Diamide Prodrug of Diisopropyl (<i>S</i>)-((2-(Guanin-9-yl)pyrrolidine-1-yl)-2-oxoethyl)phosphonate (<b>VI-C</b>)</h4><div class="NLM_p">2-Chloro-5,5-dimethyl-1,3,2-dioxaphosphorinane 2-oxide (0.6 g, 3.25 mmol) was added to the mixture of triethylammonium salt of (<i>S</i>)-3-(guanin-9-yl)-pyrrolidine-<i>N</i>-ylacetylphosphonic acid (synthesized according to the published procedure<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a>) (0.29 g, 0.65 mmol), trioctylamine (0.45 g, 1.3 mmol), and 2-pivaloylthioethanol (0.32 g, 1.96 mmol) in pyridine (10 mL). The reaction was stirred overnight under an argon atmosphere at rt. The reaction mixture was concentrated in vacuo, dissolved in chloroform (20 mL), and washed with a saturated solution of NaHCO<sub>3</sub> (20 mL) and 1% citric acid (20 mL). The organic phase was dried over Na<sub>2</sub>SO<sub>4</sub> and the desired product was obtained by column chromatography on silica gel using a linear gradient of ethanol in chloroform in 70% yield (0.29 g, 0.46 mmol) in the form of a white amorphous solid. The product was lyophilized from dioxane. According to NMR, an ∼ 1:1 mixture of A and B rotamers was obtained.</div><div class="NLM_p"><sup>1</sup>H NMR (499.8 MHz, CD<sub>3</sub>OD, 25 °C): 1.209, 1.211, 1.226, 1.231 (4 × s, 4 × 9H, (CH<sub>3</sub>)<sub>3</sub>C-A,B); 2.48 (m, 1H, H-4′b-B); 2.51–2.59 (m, 3H, H-4′a-B, H-4′-A); 3.14–3.20 (m, 8H, SC<b>H</b><sub><b>2</b></sub>CH<sub>2</sub>O-A,B); 3.21–3.28 (m, 4H, CH<sub>2</sub>P-A,B); 3.63 (bm, 1H, H-5′b-B); 3.76 (bm, 1H, H-5′a-B); 3.85 (bm, 1H, H-5′b-A); 3.92 (bdd, 1H, <i>J</i><sub>gem</sub> = 12.7, <i>J</i><sub>2′b,3′</sub> = 5.0, H-2′b-A); 3.98 (bm, 1H, H-2′a-A); 4.00 (bm, 1H, H-5′a-A); 4.09 (dd, 1H, <i>J</i><sub>gem</sub> = 10.8, <i>J</i><sub>2′b,3′</sub> = 6.0, H-2′b-B); 4.15–4.23 (m, 9H, H-2′a-B, SCH<sub>2</sub>C<b>H</b><sub><b>2</b></sub>O-A,B); 5.05 (m, 1H, H-3′-A); 5.11 (m, 1H, H-3′-B); 7.78, 7.83 (2 × s, 2 × 1H, H-8-A,B).</div><div class="NLM_p"><sup>13</sup>C NMR (125.7 MHz, CD<sub>3</sub>OD, 25 °C): 27.66, 27.68 ((<b>C</b>H<sub>3</sub>)<sub>3</sub>C-A,B); 29.67, 29.70 (d, <i>J</i><sub>C,P</sub> = 6.5, S<b>C</b>H<sub>2</sub>CH<sub>2</sub>O-A,B); 30.61 (CH<sub>2</sub>-4′-B); 32.29 (CH<sub>2</sub>-4′-A); 34.70 (d, <i>J</i><sub>C,P</sub> = 133.0, CH<sub>2</sub>P-A,B); 45.78 (CH<sub>2</sub>-5′-B); 47.18 (CH<sub>2</sub>-5′-A); 47.51 ((CH<sub>3</sub>)<sub>3</sub><b>C</b>-A,B); 51.62 (CH<sub>2</sub>-2′-A); 52.62 (CH<sub>2</sub>-2′-B); 54.47 (CH-3′-A); 55.51 (CH-3′-B); 66.40, 66.48, 66.51 (d, <i>J</i><sub>C,P</sub> = 6.4, SCH<sub>2</sub><b>C</b>H<sub>2</sub>O-A,B); 117.95, 117.98 (C-5-A,B); 137.69, 137.88 (CH-8-A,B); 153.08 (C-4-A,B); 155.14 (C-2-A,B); 159.37 (C-6-A,B); 165.41, 165.55 (d, <i>J</i><sub>C,P</sub> = 5.7, NCO-A,B); 207.05 (SCO-A,B). <sup>31</sup>P <sup>1</sup>H NMR (202.3 MHz, CD<sub>3</sub>OD, 25 °C): 23.42, 23.49.</div><div class="NLM_p last">HRMS (ESI<sup>–</sup>) for C<sub>25</sub>H<sub>39</sub>N<sub>6</sub>O<sub>7</sub>PNa (M + Na)<sup>+</sup>: calcd, 653.19515; found, 653.19519. This compound is >95% pure as assessed by LC/MS.</div></div><div id="sec4_8_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> Synthesis of <i>N</i><sup>4</sup>-(6-Methoxyquinolin-8-yl)pentane-1,4-diamine (Primaquine) Diamide Prodrug of Diisopropyl (<i>S</i>)-((2-(Guanin-9-yl)pyrrolidine-1-yl)-2-oxoethyl)phosphonate (<b>VI-D</b>)</h4><div class="NLM_p">Bromotrimethylsilane (0.3 mL, 2.2 mmol) was added to the solution of diisopropyl (<i>S</i>)-((2-(guanin-9-yl)pyrrolidine-1-yl)-2-oxoethyl)phosphonate (synthesized according to a published procedure<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a>) (0.19 g, 0.44 mmol) in pyridine (5 mL) at rt under an argon atmosphere and the reaction mixture was stirred overnight. This reaction mixture was concentrated in vacuo at the exclusion of moisture and dissolved in the same solvent (5 mL). Primaquine-free base (0.57 g, 2.2 g), triphenylphosphine (0.92 g, 3.52 mmol), DIPEA (0.3 mL, 1.76 mmol), and aldrithiol (0.77 g, 3.52 mmol) were added to the solution and the reaction mixture was stirred under an argon atmosphere at rt for 2 days. This reaction mixture was concentrated in vacuo and the desired product was obtained by column chromatography on silica gel using a linear gradient of methanol in chloroform, followed by preparative reversed-phase HPLC using a linear gradient of methanol in water in 80% yield (mg, mmol). ∼1:1 mixture of amidic rotamers (each of them is a mixture of 4 diastereoisomers) = 8 isomers. <sup>1</sup>H NMR (500.0 MHz, CD<sub>3</sub>OD): 1.19–1.29 (m, 48H, C<b>H</b><sub><b>3</b></sub>CH(NH)CH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>N); 1.48–1.72 (m, 64H, CH<sub>3</sub>CH(NH)C<b>H</b><sub><b>2</b></sub>C<b>H</b><sub><b>2</b></sub>CH<sub>2</sub>N); 2.24–2.43 (m, 16H, H-4′); 2.79–2.94 (m, 48H, CH<sub>2</sub>P, CH<sub>3</sub>CH(NH)CH<sub>2</sub>CH<sub>2</sub>C<b>H</b><sub><b>2</b></sub>N); 3.47–3.54 (m, 4H, H-5′); 3.54–3.72 (m, 24H, H-5′, CH<sub>3</sub>C<b>H</b>(NH)CH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>N); 3.73–3.82 (m, 8H, H-2′,5′); 3.820, 3.826, 3.827, 3.831, 3.835, 3.837, 3.845 (7 × s, 48H, CH<sub>3</sub>O); 3.86–3.95, 4.05–4.13 (2m, 12H, H-2′); 4.88–4.95 (m, 8H, H-3′, overlapped with water signal); 6.24–6.31 (m, 16H, H-7-quin); 6.36–6.44 (m, 16H, H-5-quin); 7.27–7.35 (m, 16H, H-3-quin); 7.69, 7.743, 7.749, 7.752, 7.760 (5 × s, 8H, H-8); 7.93–8.02 (m, 16H, H-4-quin); 8.42–8.49 (m, 16H, H-2-quin).</div><div class="NLM_p last"><sup>13</sup>C NMR (125.7 MHz, CD<sub>3</sub>OD): 20.83, 20.84 (<b>C</b>H<sub>3</sub>CH(NH)CH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>N); 29.63, 29.67 (CH<sub>3</sub>CH(NH)CH<sub>2</sub><b>C</b>H<sub>2</sub>CH<sub>2</sub>N); 30.56, 32.12 (CH<sub>2</sub>-4′); 34.72, 34.74, 34.76, 34.89 (CH<sub>3</sub>CH(NH)<b>C</b>H<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>N); 37.23, 37.71 (2 × d, <i>J</i><sub>C,P</sub> = 100.3, CH<sub>2</sub>P); 40.91–41.29 (m, CH<sub>3</sub>CH(NH)CH<sub>2</sub>CH<sub>2</sub><b>C</b>H<sub>2</sub>N); 45.49, 47.10, 47.12 (CH<sub>2</sub>-5′); 48.46 (CH<sub>3</sub><b>C</b>H(NH)CH<sub>2</sub>CH<sub>2</sub>CH<sub>2</sub>N); 51.39, 51.42, 52.34, 52.36, 52.39, 52.41 (CH<sub>2</sub>-2′); 54.24, 55.29 (CH-3′); 55.63 (CH<sub>3</sub>O); 92.89, 92.95 (CH-5-quin); 98.29, 98.31, 98.36 (CH-7-quin); 117.92, 117.95 (C-5); 122.92, 122.94 (CH-3-quin); 131.55, 131.56, 131.57 (C-4a-quin); 136.29, 136.30, 136.31, 136.33 (CH-4-quin); 136.43, 136.44, 136.45 (C-8a-quin); 137.59, 137.70, 137.72, 137.75, 137.77 (CH-8); 145.295, 145.300, 145.317, 145.325 (CH-2-quin); 146.14, 146.16 (C-8-quin); 152.988, 152.994, 153.037, 153.044 (C-4); 155.10 (C-2); 159.38, 159.39 (C-6); 160.95, 160.96, 160.97, 160.98 (C-6-quin); 167.65–167.86 (m, <b>C</b>OCH<sub>2</sub>P). <sup>31</sup>P <sup>1</sup>H NMR (202.4 MHz, CD<sub>3</sub>OD): 26.07; 26.12; 26.13; 26.15; 26.21; 26.22; 26.25. IR ν<sub>max</sub>: (CHCl<sub>3</sub>) 3360 (w, br), 2928 (m), 2860 (m, w), 2834 (sh, w), 1693 (s, ws), 1654 (sh, m), 1640 (sh, s), 1632 (sh, s), 1618 (vs), 1577 (s), 1521 (vs), 1444 (sh, m), 1389 (s), 1374 (sh), 1255 (m), 1238 (m), 1198 (s), 1166 (s), 1121 (m), 1091 (m, w), 1052 (m, w), 1032 (w), 888 (w), 821 (m), 847 (m), 641 (w). HR-ESI C<sub>41</sub>H<sub>54</sub>O<sub>5</sub>N<sub>12</sub>P (M + H)<sup>+</sup>: calcd, 825.40723; found, 825.40794. This compound is >95% pure as assessed by LC/MS.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i40"><a href="/doi/suppl/10.1021/acs.jmedchem.0c02184" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35502" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35502" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c02184?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02184</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02184/suppl_file/jm0c02184_si_001.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02184/suppl_file/jm0c02184_si_001.csv">jm0c02184_si_001.csv (3.77 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c02184" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86353" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86353" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Luke W. Guddat</span> - <span class="hlFld-Affiliation affiliation">The
School of Chemistry and Molecular Biosciences, The University of Queensland, Brisbane 4072, Queensland, Australia</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8204-8408" title="Orcid link">http://orcid.org/0000-0002-8204-8408</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#701c051b155e1705141411043005015e1514055e1105"><span class="__cf_email__" data-cfemail="137f6678763d7466777772675366623d7677663d7266">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dianne T. Keough</span> - <span class="hlFld-Affiliation affiliation">The
School of Chemistry and Molecular Biosciences, The University of Queensland, Brisbane 4072, Queensland, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shun Jie Wun</span> - <span class="hlFld-Affiliation affiliation">The
School of Chemistry and Molecular Biosciences, The University of Queensland, Brisbane 4072, Queensland, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ondřej Baszczyňski</span> - <span class="hlFld-Affiliation affiliation">Institute
of Organic Chemistry and Biochemistry, Academy
of Sciences of the Czech Republic, Prague 6 CZ-166 10, Czech
Republic</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0195-9953" title="Orcid link">http://orcid.org/0000-0003-0195-9953</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wai Soon Eng</span> - <span class="hlFld-Affiliation affiliation">The
School of Chemistry and Molecular Biosciences, The University of Queensland, Brisbane 4072, Queensland, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Petr Špaček</span> - <span class="hlFld-Affiliation affiliation">Institute
of Organic Chemistry and Biochemistry, Academy
of Sciences of the Czech Republic, Prague 6 CZ-166 10, Czech
Republic</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Santosh Panjikar</span> - <span class="hlFld-Affiliation affiliation">Australian
Synchrotron, ANSTO, 800 Blackburn Road, Clayton 3168, Victoria, Australia</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Biology, Monash
University, Clayton 3800, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lieve Naesens</span> - <span class="hlFld-Affiliation affiliation">Katholieke
Universiteit, Leuven, Rega Institute for Medical Research, Leuven 3000, Belgium</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9742-9302" title="Orcid link">http://orcid.org/0000-0001-9742-9302</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Radek Pohl</span> - <span class="hlFld-Affiliation affiliation">Institute
of Organic Chemistry and Biochemistry, Academy
of Sciences of the Czech Republic, Prague 6 CZ-166 10, Czech
Republic</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dominik Rejman</span> - <span class="hlFld-Affiliation affiliation">Institute
of Organic Chemistry and Biochemistry, Academy
of Sciences of the Czech Republic, Prague 6 CZ-166 10, Czech
Republic</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dana Hocková</span> - <span class="hlFld-Affiliation affiliation">Institute
of Organic Chemistry and Biochemistry, Academy
of Sciences of the Czech Republic, Prague 6 CZ-166 10, Czech
Republic</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Richard L. Ferrero</span> - <span class="hlFld-Affiliation affiliation">Hudson Institute
of Medical Research, Clayton 3800, Victoria, Australia</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Molecular and Translational Sciences, Monash University, Clayton 3800, Australia</span>; 
    <span class="hlFld-Affiliation affiliation">Biomedicine
Discovery Institute, Department of Microbiology, Monash University, Clayton 3800, Australia</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions from all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br><p class="inlineNote">The atomic coordinates and structure factors of HpXGHPRT in the presence (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KL6">7KL6</a>) and absence (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KL7">7KL7</a>) of 9-[(<i>N</i>-3-phosphonopropyl)-aminomethyl]-9-deazahypoxanthine (compound <b>XII</b>) have been deposited in the Protein Data Bank. Authors will release the atomic coordinates upon article publication.</p></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i43">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53401" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53401" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Initial crystallographic conditions were determined using the University of Queensland Remote-Operation Crystallization and X-ray Diffraction facility (UQROCX). X-ray measurements were performed at the Australian Synchrotron, ANSTO. This research was undertaken in part using the MX2 beamline at the Australian Synchrotron and made use of the Australian Cancer Research Foundation (ACRF) detector. The work was funded by the National Health and Medical Research Foundation, grant number 1147368, and by the Czech Science Foundation (grant no. 19-07707S) to D.H. Research at the Hudson Institute was funded by the NHMRC (Senior Research Fellowship APP1079904) and the Victorian Government’s Operational Infrastructure Support Program.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">HGXPRT</td><td class="NLM_def"><p class="first last">hypoxanthine–guanine–xanthine phosphoribosyltransferase</p></td></tr><tr><td class="NLM_term">PRTase</td><td class="NLM_def"><p class="first last">phosphoribosyltransferase</p></td></tr><tr><td class="NLM_term">PRib-PP</td><td class="NLM_def"><p class="first last">5-phospho-α-<span class="smallcaps smallerCapital">d</span>-ribosyl-1-pyrophosphate</p></td></tr><tr><td class="NLM_term">Hp</td><td class="NLM_def"><p class="first last"><i>Helicobacter pylori</i></p></td></tr><tr><td class="NLM_term">Ec</td><td class="NLM_def"><p class="first last"><i>Escherichia coli</i></p></td></tr><tr><td class="NLM_term">Mt</td><td class="NLM_def"><p class="first last"><i>Mycobacterium tuberculosis</i></p></td></tr><tr><td class="NLM_term">Hs</td><td class="NLM_def"><p class="first last"><i>Homo sapiens</i></p></td></tr><tr><td class="NLM_term">Pf</td><td class="NLM_def"><p class="first last"><i>Plasmodium falciparum</i></p></td></tr><tr><td class="NLM_term">ANP</td><td class="NLM_def"><p class="first last">acyclic nucleoside phosphonate</p></td></tr><tr><td class="NLM_term">PP<sub>i</sub></td><td class="NLM_def"><p class="first last">pyrophosphate</p></td></tr><tr><td class="NLM_term">AIP</td><td class="NLM_def"><p class="first last">acyclic immucillin phosphonate</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i45">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37427" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37427" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 64 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, J. R.</span></span> <span> </span><span class="NLM_article-title">Unidentified curved bacilli in the stomach of patients with gastritis and peptic ulceration</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>323</i></span>,  <span class="NLM_fpage">1311</span>– <span class="NLM_lpage">1315</span>, <span class="refDoi"> DOI: 10.1016/s0140-6736(84)91816-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1016%2FS0140-6736%2884%2991816-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=323&publication_year=1984&pages=1311-1315&author=B.+Marshallauthor=J.+R.+Warren&title=Unidentified+curved+bacilli+in+the+stomach+of+patients+with+gastritis+and+peptic+ulceration&doi=10.1016%2Fs0140-6736%2884%2991816-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2884%2991816-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252884%252991816-6%26sid%3Dliteratum%253Aachs%26aulast%3DMarshall%26aufirst%3DB.%26aulast%3DWarren%26aufirst%3DJ.%2BR.%26atitle%3DUnidentified%2520curved%2520bacilli%2520in%2520the%2520stomach%2520of%2520patients%2520with%2520gastritis%2520and%2520peptic%2520ulceration%26jtitle%3DLancet%26date%3D1984%26volume%3D323%26spage%3D1311%26epage%3D1315%26doi%3D10.1016%2Fs0140-6736%2884%2991816-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hooi, J. K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, W. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suen, M. M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Underwood, F. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanyingoh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malfertheiner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, V. W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J. C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, F. K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, J. J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaplan, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, S. C.</span></span> <span> </span><span class="NLM_article-title">Global prevalence of <i>Helicobacter pylori</i> infection: Systematic review and meta-analysis</span>. <i>Gastroenterology</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>153</i></span>,  <span class="NLM_fpage">420</span>– <span class="NLM_lpage">429</span>, <span class="refDoi"> DOI: 10.1053/j.gastro.2017.04.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1053%2Fj.gastro.2017.04.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=28456631" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A280%3ADC%252BC1crisFaktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=153&publication_year=2017&pages=420-429&author=J.+K.+Y.+Hooiauthor=W.+Y.+Laiauthor=W.+K.+Ngauthor=M.+M.+Y.+Suenauthor=F.+E.+Underwoodauthor=D.+Tanyingohauthor=P.+Malfertheinerauthor=D.+Y.+Grahamauthor=V.+W.+S.+Wongauthor=J.+C.+Y.+Wuauthor=F.+K.+L.+Chanauthor=J.+J.+Y.+Sungauthor=G.+G.+Kaplanauthor=S.+C.+Ng&title=Global+prevalence+of+Helicobacter+pylori+infection%3A+Systematic+review+and+meta-analysis&doi=10.1053%2Fj.gastro.2017.04.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Global Prevalence of Helicobacter pylori Infection: Systematic Review and Meta-Analysis</span></div><div class="casAuthors">Hooi James K Y; Lai Wan Ying; Suen Michael M Y; Wong Vincent W S; Wu Justin C Y; Chan Francis K L; Sung Joseph J Y; Ng Wee Khoon; Underwood Fox E; Tanyingoh Divine; Malfertheiner Peter; Graham David Y; Kaplan Gilaad G; Ng Siew C</div><div class="citationInfo"><span class="NLM_cas:title">Gastroenterology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">153</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">420-429</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND & AIMS:  The epidemiology of Helicobacter pylori infection has changed with improvements in sanitation and methods of eradication.  We performed a systematic review and meta-analysis to evaluate changes in the global prevalence of H pylori infection.  METHODS:  We performed a systematic search of the MEDLINE and EMBASE databases for studies of the prevalence of H pylori infection published from January 1, 1970 through January 1, 2016.  We analyzed data based on United Nations geoscheme regions and individual countries.  We used a random effects model to calculate pooled prevalence estimates with 95% confidence intervals (CIs), weighted by study size.  We extrapolated 2015 prevalence estimates to obtain the estimated number of individuals with H pylori infection.  RESULTS:  Among 14,006 reports screened, we identified 263 full-text articles on the prevalence of H pylori infection; 184 were included in the final analysis, comprising data from 62 countries.  Africa had the highest pooled prevalence of H pylori infection (70.1%; 95% CI, 62.6-77.7), whereas Oceania had the lowest prevalence (24.4%; 95% CI, 18.5-30.4).  Among individual countries, the prevalence of H pylori infection varied from as low as 18.9% in Switzerland (95% CI, 13.1-24.7) to 87.7% in Nigeria (95% CI, 83.1-92.2).  Based on regional prevalence estimates, there were approximately 4.4 billion individuals with H pylori infection worldwide in 2015.  CONCLUSIONS:  In a systematic review and meta-analysis to assess the prevalence of H pylori infection worldwide, we observed large amounts of variation among regions-more than half the world's population is infected.  These data can be used in development of customized strategies for the global eradication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT6iihP5eyfJaqcneEtCi92fW6udTcc2eb9M1Iu2acHXrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1crisFaktA%253D%253D&md5=612061e322f559cdbe6c3a5c86fa0b15</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1053%2Fj.gastro.2017.04.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1053%252Fj.gastro.2017.04.022%26sid%3Dliteratum%253Aachs%26aulast%3DHooi%26aufirst%3DJ.%2BK.%2BY.%26aulast%3DLai%26aufirst%3DW.%2BY.%26aulast%3DNg%26aufirst%3DW.%2BK.%26aulast%3DSuen%26aufirst%3DM.%2BM.%2BY.%26aulast%3DUnderwood%26aufirst%3DF.%2BE.%26aulast%3DTanyingoh%26aufirst%3DD.%26aulast%3DMalfertheiner%26aufirst%3DP.%26aulast%3DGraham%26aufirst%3DD.%2BY.%26aulast%3DWong%26aufirst%3DV.%2BW.%2BS.%26aulast%3DWu%26aufirst%3DJ.%2BC.%2BY.%26aulast%3DChan%26aufirst%3DF.%2BK.%2BL.%26aulast%3DSung%26aufirst%3DJ.%2BJ.%2BY.%26aulast%3DKaplan%26aufirst%3DG.%2BG.%26aulast%3DNg%26aufirst%3DS.%2BC.%26atitle%3DGlobal%2520prevalence%2520of%2520Helicobacter%2520pylori%2520infection%253A%2520Systematic%2520review%2520and%2520meta-analysis%26jtitle%3DGastroenterology%26date%3D2017%26volume%3D153%26spage%3D420%26epage%3D429%26doi%3D10.1053%2Fj.gastro.2017.04.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Megraud, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leon-Barua, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bazzoli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Merwe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaz Coelho, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fock, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedail, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malfertheiner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakil, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamid, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goh, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krabshuis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Mair, A.</span></span>; <span class="NLM_contrib-group">World Gastroenterology Organization</span> <span> </span><span class="NLM_article-title"><i>Helicobacter pylori</i> in developing countries. World gastroenterology organisation global guideline</span>. <i>J. Gastrointest. Liver Dis.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">304</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=21961099" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A280%3ADC%252BC3MfnsFyjtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2011&pages=299-304&author=R.+H.+Huntauthor=S.+D.+Xiaoauthor=F.+Megraudauthor=R.+Leon-Baruaauthor=F.+Bazzoliauthor=S.+van+der+Merweauthor=L.+G.+Vaz+Coelhoauthor=M.+Fockauthor=S.+Fedailauthor=H.+Cohenauthor=P.+Malfertheinerauthor=N.+Vakilauthor=S.+Hamidauthor=K.+L.+Gohauthor=B.+C.+Wongauthor=J.+Krabshuisauthor=A.+Le+Mairauthor=World+Gastroenterology+Organization&title=Helicobacter+pylori+in+developing+countries.+World+gastroenterology+organisation+global+guideline"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Helicobacter pylori in developing countries. World Gastroenterology Organisation Global Guideline</span></div><div class="casAuthors">Hunt R H; Xiao S D; Megraud F; Leon-Barua R; Bazzoli F; van der Merwe S; Vaz Coelho L G; Fock M; Fedail S; Cohen H; Malfertheiner P; Vakil N; Hamid S; Goh K L; Wong B C Y; Krabshuis J; Le Mair A</div><div class="citationInfo"><span class="NLM_cas:title">Journal of gastrointestinal and liver diseases : JGLD</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">299-304</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSRh26RHe2V9NenUz0lpNqnfW6udTcc2eb9M1Iu2acHXrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MfnsFyjtw%253D%253D&md5=184f5c36f135f3c20da2c25afbea1461</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHunt%26aufirst%3DR.%2BH.%26aulast%3DXiao%26aufirst%3DS.%2BD.%26aulast%3DMegraud%26aufirst%3DF.%26aulast%3DLeon-Barua%26aufirst%3DR.%26aulast%3DBazzoli%26aufirst%3DF.%26aulast%3Dvan%2Bder%2BMerwe%26aufirst%3DS.%26aulast%3DVaz%2BCoelho%26aufirst%3DL.%2BG.%26aulast%3DFock%26aufirst%3DM.%26aulast%3DFedail%26aufirst%3DS.%26aulast%3DCohen%26aufirst%3DH.%26aulast%3DMalfertheiner%26aufirst%3DP.%26aulast%3DVakil%26aufirst%3DN.%26aulast%3DHamid%26aufirst%3DS.%26aulast%3DGoh%26aufirst%3DK.%2BL.%26aulast%3DWong%26aufirst%3DB.%2BC.%26aulast%3DKrabshuis%26aufirst%3DJ.%26aulast%3DLe%2BMair%26aufirst%3DA.%26aulast%3D%26atitle%3DHelicobacter%2520pylori%2520in%2520developing%2520countries.%2520World%2520gastroenterology%2520organisation%2520global%2520guideline%26jtitle%3DJ.%2520Gastrointest.%2520Liver%2520Dis.%26date%3D2011%26volume%3D20%26spage%3D299%26epage%3D304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saleem, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howden, C. W.</span></span> <span> </span><span class="NLM_article-title">Update on the management of <i>Helicobacter pylori</i> infection</span>. <i>Curr. Treat. Options Gastroenterol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">476</span>– <span class="NLM_lpage">487</span>, <span class="refDoi"> DOI: 10.1007/s11938-020-00300-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1007%2Fs11938-020-00300-3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2020&pages=476-487&author=N.+Saleemauthor=C.+W.+Howden&title=Update+on+the+management+of+Helicobacter+pylori+infection&doi=10.1007%2Fs11938-020-00300-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1007%2Fs11938-020-00300-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11938-020-00300-3%26sid%3Dliteratum%253Aachs%26aulast%3DSaleem%26aufirst%3DN.%26aulast%3DHowden%26aufirst%3DC.%2BW.%26atitle%3DUpdate%2520on%2520the%2520management%2520of%2520Helicobacter%2520pylori%2520infection%26jtitle%3DCurr.%2520Treat.%2520Options%2520Gastroenterol.%26date%3D2020%26volume%3D18%26spage%3D476%26epage%3D487%26doi%3D10.1007%2Fs11938-020-00300-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sarajlic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuper, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vetter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klicznik, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gratz, I. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wessler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Posselt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horejs-Hoeck, J.</span></span> <span> </span><span class="NLM_article-title"><i>H. pylori</i> modulates DC functions via T4SS/TNFα/p38-dependent SOCS3 expression</span>. <i>Cell Commun. Signaling</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">160</span>, <span class="refDoi"> DOI: 10.1186/s12964-020-00655-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1186%2Fs12964-020-00655-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=33023610" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitVyiur%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2020&pages=160&author=M.+Sarajlicauthor=T.+Neuperauthor=J.+Vetterauthor=S.+Schallerauthor=M.+M.+Klicznikauthor=I.+K.+Gratzauthor=S.+Wesslerauthor=G.+Posseltauthor=J.+Horejs-Hoeck&title=H.+pylori+modulates+DC+functions+via+T4SS%2FTNF%CE%B1%2Fp38-dependent+SOCS3+expression&doi=10.1186%2Fs12964-020-00655-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">H. pylori modulates DC functions via T4SS/TNFα/p38-dependent SOCS3 expression</span></div><div class="casAuthors">Sarajlic, Muamera; Neuper, Theresa; Vetter, Julia; Schaller, Susanne; Klicznik, Maria M.; Gratz, Iris K.; Wessler, Silja; Posselt, Gernot; Horejs-Hoeck, Jutta</div><div class="citationInfo"><span class="NLM_cas:title">Cell Communication and Signaling</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">160</span>CODEN:
                <span class="NLM_cas:coden">CCSEC6</span>;
        ISSN:<span class="NLM_cas:issn">1478-811X</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Helicobacter pylori (H. pylori) is a gram-neg. bacterium that chronically infects approx. 50% of the world's human population.  While in most cases the infection remains asymptomatic, 10% of infected individuals develop gastric pathologies and 1-3% progress to gastric cancer.  Although H. pylori induces severe inflammatory responses, the host's immune system fails to clear the pathogen and H. pylori can persist in the human stomach for decades.  As suppressor of cytokine signaling (SOCS) proteins are important feedback regulators limiting inflammatory responses, we hypothesized that H. pylori could modulate the host's immune responses by inducing SOCS expression.  The phenotype of human monocyte-derived DCs (moDCs) infected with H. pylori was analyzed by flow cytometry and multiplex technol.  SOCS expression levels were monitored by qPCR and signaling studies were conducted by means of Western blot.  For functional studies, RNA interference-based silencing of SOCS1-3 and co-cultures with CD4+ T cells were performed.  We show that H. pylori pos. gastritis patients express significantly higher SOCS3, but not SOCS1 and SOCS2, levels compared to H. pylori neg. patients.  Moreover, infection of human moDCs with H. pylori rapidly induces SOCS3 expression, which requires the type IV secretion system (T4SS), release of TNFα, and signaling via the MAP kinase p38, but appears to be independent of TLR2, TLR4, MEK1/2 and STAT proteins.  Silencing of SOCS3 expression in moDCs prior to H. pylori infection resulted in increased release of both pro- and anti-inflammatory cytokines, upregulation of PD-L1, and decreased T-cell proliferation.  This study shows that H. pylori induces SOCS3 via an autocrine loop involving the T4SS and TNFα and p38 signaling.  Moreover, we demonstrate that high levels of SOCS3 in DCs dampen PD-L1 expression on DCs, which in turn drives T-cell proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpI0i4xiDIzq7Vg90H21EOLACvtfcHk0ljcPOTxYvKzkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitVyiur%252FJ&md5=338c54fb4ab02f7129dec2ec6c8ebb66</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1186%2Fs12964-020-00655-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12964-020-00655-1%26sid%3Dliteratum%253Aachs%26aulast%3DSarajlic%26aufirst%3DM.%26aulast%3DNeuper%26aufirst%3DT.%26aulast%3DVetter%26aufirst%3DJ.%26aulast%3DSchaller%26aufirst%3DS.%26aulast%3DKlicznik%26aufirst%3DM.%2BM.%26aulast%3DGratz%26aufirst%3DI.%2BK.%26aulast%3DWessler%26aufirst%3DS.%26aulast%3DPosselt%26aufirst%3DG.%26aulast%3DHorejs-Hoeck%26aufirst%3DJ.%26atitle%3DH.%2520pylori%2520modulates%2520DC%2520functions%2520via%2520T4SS%252FTNF%25CE%25B1%252Fp38-dependent%2520SOCS3%2520expression%26jtitle%3DCell%2520Commun.%2520Signaling%26date%3D2020%26volume%3D18%26spage%3D160%26doi%3D10.1186%2Fs12964-020-00655-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuipers, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israel, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusters, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerrits, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Ende, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Hulst, R. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wirth, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Höök-Nikanne, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaser, M. J.</span></span> <span> </span><span class="NLM_article-title">Quasispecies development of <i>Helicobacter pylori</i> observed in paired isolates obtained years apart from the same host</span>. <i>J. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>181</i></span>,  <span class="NLM_fpage">273</span>– <span class="NLM_lpage">282</span>, <span class="refDoi"> DOI: 10.1086/315173</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1086%2F315173" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10608776" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BD3cXoslagsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=181&publication_year=2000&pages=273-282&author=E.+J.+Kuipersauthor=D.+A.+Israelauthor=J.+G.+Kustersauthor=M.+M.+Gerritsauthor=J.+Weelauthor=A.+van+der+Endeauthor=R.+W.+M.+van+der+Hulstauthor=H.+P.+Wirthauthor=J.+H%C3%B6%C3%B6k-Nikanneauthor=S.+A.+Thompsonauthor=M.+J.+Blaser&title=Quasispecies+development+of+Helicobacter+pylori+observed+in+paired+isolates+obtained+years+apart+from+the+same+host&doi=10.1086%2F315173"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Quasispecies development of Helicobacter pylori observed in paired isolates obtained years apart from the same host</span></div><div class="casAuthors">Kuipers, Ernst J.; Israel, Dawn A.; Kusters, Johannes G.; Gerrits, Monique M.; Weel, Jan; van der Ende, Arie; van der Hulst, Rene W. M.; Wirth, Hans P.; Hook-Nikanne, Johanna; Thompson, Stuart A.; Blaser, Martin J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Infectious Diseases</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">181</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">273-282</span>CODEN:
                <span class="NLM_cas:coden">JIDIAQ</span>;
        ISSN:<span class="NLM_cas:issn">0022-1899</span>.
    
            (<span class="NLM_cas:orgname">University of Chicago Press</span>)
        </div><div class="casAbstract">Helicobacter pylori isolates show greater genetic diversity than other bacterial species studied, but the basis for this phenomenon is unknown.  Whether detectable genomic mutation appears within an H. pylori population during persistent colonization was investigated.  Paired H. pylori populations obtained across 7-10-yr intervals from 13 patients were characterized by use of PCR genotyping for cagA, vacA, iceA, recA, and IS605; RAPD-PCR and AFLP anal.; and ELISA, to det. Lewis phenotypes.  Genotyping, including recA sequence anal., revealed that initial and follow-up populations represented the same population in 11 patients (85%).  Nevertheless, distinct dissimilarities were shown within each of these 11 pairs by both RAPD-PCR and AFLP analyses.  During follow-up, Lewis-y levels, but not Lewis-x levels, decreased significantly.  The changes detected by RAPD-PCR and AFLP indicate that genetic drift occurs within H. pylori populations over the course of years of colonization of a single host.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0xi_ogykbCLVg90H21EOLACvtfcHk0ljcPOTxYvKzkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXoslagsA%253D%253D&md5=f00f66b9ebaf14ce33427c430df09303</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1086%2F315173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1086%252F315173%26sid%3Dliteratum%253Aachs%26aulast%3DKuipers%26aufirst%3DE.%2BJ.%26aulast%3DIsrael%26aufirst%3DD.%2BA.%26aulast%3DKusters%26aufirst%3DJ.%2BG.%26aulast%3DGerrits%26aufirst%3DM.%2BM.%26aulast%3DWeel%26aufirst%3DJ.%26aulast%3Dvan%2Bder%2BEnde%26aufirst%3DA.%26aulast%3Dvan%2Bder%2BHulst%26aufirst%3DR.%2BW.%2BM.%26aulast%3DWirth%26aufirst%3DH.%2BP.%26aulast%3DH%25C3%25B6%25C3%25B6k-Nikanne%26aufirst%3DJ.%26aulast%3DThompson%26aufirst%3DS.%2BA.%26aulast%3DBlaser%26aufirst%3DM.%2BJ.%26atitle%3DQuasispecies%2520development%2520of%2520Helicobacter%2520pylori%2520observed%2520in%2520paired%2520isolates%2520obtained%2520years%2520apart%2520from%2520the%2520same%2520host%26jtitle%3DJ.%2520Infect.%2520Dis.%26date%3D2000%26volume%3D181%26spage%3D273%26epage%3D282%26doi%3D10.1086%2F315173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peek, R. M.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaser, M. J.</span></span> <span> </span><span class="NLM_article-title"><i>Helicobacter pylori</i> and gastrointestinal tract adenocarcinomas</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">28</span>– <span class="NLM_lpage">37</span>, <span class="refDoi"> DOI: 10.1038/nrc703</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1038%2Fnrc703" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=11902583" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BD38XhvF2jt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2002&pages=28-37&author=R.+M.+Peekauthor=M.+J.+Blaser&title=Helicobacter+pylori+and+gastrointestinal+tract+adenocarcinomas&doi=10.1038%2Fnrc703"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Helicobacter pylori and gastrointestinal tract adenocarcinomas</span></div><div class="casAuthors">Peek, Richard M., Jr.; Blaser, Martin J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">28-37</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Although gastric adenocarcinoma is assocd. with the presence of Helicobacter pylori in the stomach, only a small fraction of colonized individuals develop this common malignancy.  H. pylori strain and host genotypes probably influence the risk of carcinogenesis by differentially affecting host inflammatory responses and epithelial-cell physiol.  Understanding the host-microbial interactions that lead to neoplasia will improve cancer-targeted therapeutics and diagnostics, and provide mechanistic insights into other malignancies that arise within the context of microbially initiated inflammatory states.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCiTckbOS5wLVg90H21EOLACvtfcHk0ljcPOTxYvKzkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhvF2jt7c%253D&md5=3dcc5fdf6d5347a2cfe82e8eed349d59</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrc703&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc703%26sid%3Dliteratum%253Aachs%26aulast%3DPeek%26aufirst%3DR.%2BM.%26aulast%3DBlaser%26aufirst%3DM.%2BJ.%26atitle%3DHelicobacter%2520pylori%2520and%2520gastrointestinal%2520tract%2520adenocarcinomas%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2002%26volume%3D2%26spage%3D28%26epage%3D37%26doi%3D10.1038%2Fnrc703" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Uemura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamakido, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taniyama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlemper, R. J.</span></span> <span> </span><span class="NLM_article-title"><i>Helicobacter pylori</i> infection and the development of gastric cancer</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>345</i></span>,  <span class="NLM_fpage">784</span>– <span class="NLM_lpage">789</span>, <span class="refDoi"> DOI: 10.1056/nejmoa001999</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1056%2FNEJMoa001999" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=11556297" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A280%3ADC%252BD3MrgslCqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=345&publication_year=2001&pages=784-789&author=N.+Uemuraauthor=S.+Okamotoauthor=S.+Yamamotoauthor=N.+Matsumuraauthor=S.+Yamaguchiauthor=M.+Yamakidoauthor=K.+Taniyamaauthor=N.+Sasakiauthor=R.+J.+Schlemper&title=Helicobacter+pylori+infection+and+the+development+of+gastric+cancer&doi=10.1056%2Fnejmoa001999"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Helicobacter pylori infection and the development of gastric cancer</span></div><div class="casAuthors">Uemura N; Okamoto S; Yamamoto S; Matsumura N; Yamaguchi S; Yamakido M; Taniyama K; Sasaki N; Schlemper R J</div><div class="citationInfo"><span class="NLM_cas:title">The New England journal of medicine</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">345</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">784-9</span>
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    </div><div class="casAbstract">BACKGROUND:  Although many studies have found an association between Helicobacter pylori infection and the development of gastric cancer, many aspects of this relation remain uncertain.  METHODS:  We prospectively studied 1526 Japanese patients who had duodenal ulcers, gastric ulcers, gastric hyperplasia, or nonulcer dyspepsia at the time of enrollment; 1246 had H. pylori infection and 280 did not.  The mean follow-up was 7.8 years (range, 1.0 to 10.6).  Patients underwent endoscopy with biopsy at enrollment and then between one and three years after enrollment.  H. pylori infection was assessed by histologic examination, serologic testing, and rapid urease tests and was defined by a positive result on any of these tests.  RESULTS:  Gastric cancers developed in 36 (2.9 percent) of the infected and none of the uninfected patients.  There were 23 intestinal-type and 13 diffuse-type cancers.  Among the patients with H. pylori infection, those with severe gastric atrophy, corpus-predominant gastritis, and intestinal metaplasia were at significantly higher risk for gastric cancer.  We detected gastric cancers in 21 (4.7 percent) of the 445 patients with nonulcer dyspepsia, 10 (3.4 percent) of the 297 with gastric ulcers, 5 (2.2 percent) of the 229 with gastric hyperplastic polyps, and none of the 275 with duodenal ulcers.  CONCLUSIONS:  Gastric cancer develops in persons infected with H. pylori but not in uninfected persons.  Those with histologic findings of severe gastric atrophy, corpus-predominant gastritis, or intestinal metaplasia are at increased risk.  Persons with H. pylori infection and nonulcer dyspepsia, gastric ulcers, or gastric hyperplastic polyps are also at risk, but those with duodenal ulcers are not.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTAlvuOt_-alMAKB7Fa-xrUfW6udTcc2eYit4iII9Nusrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3MrgslCqtA%253D%253D&md5=53ac3adf0529c87369f26614e10633fd</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa001999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa001999%26sid%3Dliteratum%253Aachs%26aulast%3DUemura%26aufirst%3DN.%26aulast%3DOkamoto%26aufirst%3DS.%26aulast%3DYamamoto%26aufirst%3DS.%26aulast%3DMatsumura%26aufirst%3DN.%26aulast%3DYamaguchi%26aufirst%3DS.%26aulast%3DYamakido%26aufirst%3DM.%26aulast%3DTaniyama%26aufirst%3DK.%26aulast%3DSasaki%26aufirst%3DN.%26aulast%3DSchlemper%26aufirst%3DR.%2BJ.%26atitle%3DHelicobacter%2520pylori%2520infection%2520and%2520the%2520development%2520of%2520gastric%2520cancer%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2001%26volume%3D345%26spage%3D784%26epage%3D789%26doi%3D10.1056%2Fnejmoa001999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kusters, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Vliet, A. H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuipers, E. J.</span></span> <span> </span><span class="NLM_article-title">Pathogenesis of <i>Helicobacter pylori</i> infection</span>. <i>Clin. Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">449</span>– <span class="NLM_lpage">490</span>, <span class="refDoi"> DOI: 10.1128/cmr.00054-05</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1128%2FCMR.00054-05" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=16847081" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BD28XosVaqsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2006&pages=449-490&author=J.+G.+Kustersauthor=A.+H.+M.+van+Vlietauthor=E.+J.+Kuipers&title=Pathogenesis+of+Helicobacter+pylori+infection&doi=10.1128%2Fcmr.00054-05"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Pathogenesis of Helicobacter pylori infection</span></div><div class="casAuthors">Kusters, Johannes G.; van Vliet, Arnoud H. M.; Kuipers, Ernst J.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Microbiology Reviews</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">449-490</span>CODEN:
                <span class="NLM_cas:coden">CMIREX</span>;
        ISSN:<span class="NLM_cas:issn">0893-8512</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A review.  Helicobacter pylori is the first formally recognized bacterial carcinogen and is one of the most successful human pathogens, as over half of the world's population is colonized with this gram-neg. bacterium.  Unless treated, colonization usually persists lifelong.  H. pylori infection represents a key factor in the etiol. of various gastrointestinal diseases, ranging from chronic active gastritis without clin. symptoms to peptic ulceration, gastric adenocarcinoma, and gastric mucosa-assocd. lymphoid tissue lymphoma.  Disease outcome is the result of the complex interplay between the host and the bacterium.  Host immune gene polymorphisms and gastric acid secretion largely det. the bacterium's ability to colonize a specific gastric niche.  Bacterial virulence factors such as the cytotoxin-assocd. gene pathogenicity island-encoded protein CagA and the vacuolating cytotoxin VacA aid in this colonization of the gastric mucosa and subsequently seem to modulate the host's immune system.  This review focuses on the microbiol., clin., immunol., and biochem. aspects of the pathogenesis of H. pylori.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRylz2V0k-XrVg90H21EOLACvtfcHk0lgTRHIXL6fpRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XosVaqsrg%253D&md5=5d97264b972343880e13499120984405</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1128%2FCMR.00054-05&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FCMR.00054-05%26sid%3Dliteratum%253Aachs%26aulast%3DKusters%26aufirst%3DJ.%2BG.%26aulast%3Dvan%2BVliet%26aufirst%3DA.%2BH.%2BM.%26aulast%3DKuipers%26aufirst%3DE.%2BJ.%26atitle%3DPathogenesis%2520of%2520Helicobacter%2520pylori%2520infection%26jtitle%3DClin.%2520Microbiol.%2520Rev.%26date%3D2006%26volume%3D19%26spage%3D449%26epage%3D490%26doi%3D10.1128%2Fcmr.00054-05" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Martel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georges, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bray, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferlay, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clifford, G. M.</span></span> <span> </span><span class="NLM_article-title">Global burden of cancer attributable to infections in 2018: a worldwide incidence analysis</span>. <i>Lancet Global Health</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">e180</span>– <span class="NLM_lpage">e190</span>, <span class="refDoi"> DOI: 10.1016/s2214-109x(19)30488-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1016%2FS2214-109X%2819%2930488-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=31862245" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2020&pages=e180-e190&author=C.+de%0AMartelauthor=D.+Georgesauthor=F.+Brayauthor=J.+Ferlayauthor=G.+M.+Clifford&title=Global+burden+of+cancer+attributable+to+infections+in+2018%3A+a+worldwide+incidence+analysis&doi=10.1016%2Fs2214-109x%2819%2930488-7"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2FS2214-109X%2819%2930488-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS2214-109X%252819%252930488-7%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BMartel%26aufirst%3DC.%26aulast%3DGeorges%26aufirst%3DD.%26aulast%3DBray%26aufirst%3DF.%26aulast%3DFerlay%26aufirst%3DJ.%26aulast%3DClifford%26aufirst%3DG.%2BM.%26atitle%3DGlobal%2520burden%2520of%2520cancer%2520attributable%2520to%2520infections%2520in%25202018%253A%2520a%2520worldwide%2520incidence%2520analysis%26jtitle%3DLancet%2520Global%2520Health%26date%3D2020%26volume%3D8%26spage%3De180%26epage%3De190%26doi%3D10.1016%2Fs2214-109x%2819%2930488-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fallone, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malfertheiner, P.</span></span> <span> </span><span class="NLM_article-title">Reconciliation of recent <i>Helicobacter pylori</i> treatment guidelines in a time of Increasing resistance to antibiotics</span>. <i>Gastroenterology</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>157</i></span>,  <span class="NLM_fpage">44</span>– <span class="NLM_lpage">53</span>, <span class="refDoi"> DOI: 10.1053/j.gastro.2019.04.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1053%2Fj.gastro.2019.04.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=30998990" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A280%3ADC%252BB3M%252FmvVanuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=2019&pages=44-53&author=C.+A.+Falloneauthor=S.+F.+Mossauthor=P.+Malfertheiner&title=Reconciliation+of+recent+Helicobacter+pylori+treatment+guidelines+in+a+time+of+Increasing+resistance+to+antibiotics&doi=10.1053%2Fj.gastro.2019.04.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Reconciliation of Recent Helicobacter pylori Treatment Guidelines in a Time of Increasing Resistance to Antibiotics</span></div><div class="casAuthors">Fallone Carlo A; Moss Steven F; Malfertheiner Peter</div><div class="citationInfo"><span class="NLM_cas:title">Gastroenterology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">44-53</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Increasing resistance to antibiotics worldwide has adverse effects on the effectiveness of standard therapies to eradicate Helicobacter pylori infection.  We reviewed guidelines developed by expert groups in Europe, Canada, and the United States for the treatment of H pylori infection.  We compared the recommendations of these guidelines, reconciled them, and addressed the increasing resistance of H pylori to antibiotic therapy regimens.  The guidelines recommend bismuth quadruple therapy for first-line treatment, replacing clarithromycin-based triple therapy.  There is consensus for concomitant 4-drug therapy as an alternative, especially when bismuth is not available.  When therapy is unsuccessful, it is likely due to resistance to clarithromycin, levofloxacin, and/or metronidazole; these drugs, if used previously, should be avoided in subsequent eradication attempts.  Second-line therapies should be bismuth quadruple therapy or levofloxacin triple therapy, depending on suspected resistance, reserving rifabutin-based triple and high-dose dual amoxicillin proton pump inhibitor therapy for subsequent treatment attempts.  The increasing resistance of H pylori to antibiotic therapy necessitates local availability of susceptibility tests for individuals, and establishment of regional and national monitoring programs to develop evidence-based locally relevant eradication strategies.  Further studies into the development of more easily accessible methods of resistance testing, such as biomarker analysis of stool samples, are required.  Options under investigation include substituting vonoprazan for proton pump inhibitors, adding probiotics, and vaccine development.  Narrow-spectrum antibiotics and new therapeutic targets could be identified based on genomic, proteomic, and metabolomic analyses of H pylori.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT6WfBfFoYhqo3yKPtYViAGfW6udTcc2eYit4iII9Nusrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M%252FmvVanuw%253D%253D&md5=43362a80e77044c93801503cafac13bc</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1053%2Fj.gastro.2019.04.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1053%252Fj.gastro.2019.04.011%26sid%3Dliteratum%253Aachs%26aulast%3DFallone%26aufirst%3DC.%2BA.%26aulast%3DMoss%26aufirst%3DS.%2BF.%26aulast%3DMalfertheiner%26aufirst%3DP.%26atitle%3DReconciliation%2520of%2520recent%2520Helicobacter%2520pylori%2520treatment%2520guidelines%2520in%2520a%2520time%2520of%2520Increasing%2520resistance%2520to%2520antibiotics%26jtitle%3DGastroenterology%26date%3D2019%26volume%3D157%26spage%3D44%26epage%3D53%26doi%3D10.1053%2Fj.gastro.2019.04.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ruiter, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wunderink, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veenendaal, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Visser, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Boer, M. G. J.</span></span> <span> </span><span class="NLM_article-title"><i>Helicobacter pylori</i> resistance in the Netherlands: a growing problem?</span>. <i>Neth. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">394</span>– <span class="NLM_lpage">398</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=29219812" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A280%3ADC%252BC1M3psVSgsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2017&pages=394-398&author=R.+Ruiterauthor=H.+F.+Wunderinkauthor=R.+A.+Veenendaalauthor=L.+G.+Visserauthor=M.+G.+J.+de+Boer&title=Helicobacter+pylori+resistance+in+the+Netherlands%3A+a+growing+problem%3F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Helicobacter pylori resistance in the Netherlands: a growing problem?</span></div><div class="casAuthors">Ruiter R; Wunderink H F; Veenendaal R A; Visser L G; de Boer M G J</div><div class="citationInfo"><span class="NLM_cas:title">The Netherlands journal of medicine</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">394-398</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Helicobacter pylori infection is clinically associated with dyspepsia, gastric and duodenal ulcers, and gastric cancer.  Increasing antimicrobial resistance in H. pylori is a worldwide problem and failure of eradication with standard triple therapy (high-dose proton pump inhibition, amoxicillin and clarithromycin) is directly related to the presence of a resistant strain.  Other treatment combinations have been investigated, but with inconsistent results.  Based on a review of the recent literature in conjunction with an analysis of the regional resistance data, we address the increasing complexity of H. pylori eradication therapy.  Culture and susceptibility results of all first H. pylori isolates of adults (> 18 years) seen in the Leiden University Medical Center, from January 2006 to December 2015, were analysed (n = 707).  An increase in clarithromycin resistance was observed from 9.8% to 18.1% (p = 0.002) in the periods from 2006-2010 and 2011-2015, respectively.  For ampicillin the resistance increased from 6.3% to 10.0% (p = 0.37), and for metronidazole from 20.7% to 23.2% (p = 0.42).  The tetracycline resistance remained low at 3.2% and 2.3%, respectively.  The treatment paradigm is shifting towards individualised treatment rather than a one-strategy-fits-all approach.  In case of treatment failure it should be strongly considered to refer a patient for endoscopy, biopsy and culture.  Thereafter, targeted antimicrobial treatment based on susceptibility results can be initiated.  Furthermore, accumulating data indicate that prolongation of treatment to 14 days, as opposed to the current standard 7 day course, contributes to a higher H. pylori eradication rate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQOV9IT3Mt4h8vLc4ARHEmzfW6udTcc2eY2D-C90iLN9Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M3psVSgsg%253D%253D&md5=d3d5dc09136da3f3bea141708b5cc56e</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRuiter%26aufirst%3DR.%26aulast%3DWunderink%26aufirst%3DH.%2BF.%26aulast%3DVeenendaal%26aufirst%3DR.%2BA.%26aulast%3DVisser%26aufirst%3DL.%2BG.%26aulast%3Dde%2BBoer%26aufirst%3DM.%2BG.%2BJ.%26atitle%3DHelicobacter%2520pylori%2520resistance%2520in%2520the%2520Netherlands%253A%2520a%2520growing%2520problem%253F%26jtitle%3DNeth.%2520J.%2520Med.%26date%3D2017%26volume%3D75%26spage%3D394%26epage%3D398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Savoldi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrara, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tacconelli, E.</span></span> <span> </span><span class="NLM_article-title">Prevalence of antibiotic resistance in <i>Helicobacter pylori</i>: A systematic review and meta-analysis in World Health Organization regions</span>. <i>Gastroenterology</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>155</i></span>,  <span class="NLM_fpage">1372</span>– <span class="NLM_lpage">1382</span>, <span class="refDoi"> DOI: 10.1053/j.gastro.2018.07.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1053%2Fj.gastro.2018.07.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=29990487" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A280%3ADC%252BB3c%252FmsFOmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2018&pages=1372-1382&author=A.+Savoldiauthor=E.+Carraraauthor=D.+Y.+Grahamauthor=M.+Contiauthor=E.+Tacconelli&title=Prevalence+of+antibiotic+resistance+in+Helicobacter+pylori%3A+A+systematic+review+and+meta-analysis+in+World+Health+Organization+regions&doi=10.1053%2Fj.gastro.2018.07.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Prevalence of Antibiotic Resistance in Helicobacter pylori: A Systematic Review and Meta-analysis in World Health Organization Regions</span></div><div class="casAuthors">Savoldi Alessia; Carrara Elena; Conti Michela; Graham David Y; Tacconelli Evelina</div><div class="citationInfo"><span class="NLM_cas:title">Gastroenterology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">155</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1372-1382.e17</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND & AIMS:  In 2017, the World Health Organization (WHO) designated clarithromycin-resistant Helicobacter pylori a high priority for antibiotic research and development.  However, there are no clear data on the global distribution of resistance or its clinical effects.  We performed a systematic review and meta-analysis to assess the distribution of H pylori resistance to commonly used antibiotics and to measure the association between antibiotic resistance and treatment failure.  METHODS:  We searched publication databases for studies that assessed rates of H pylori resistance to clarithromycin, metronidazole, levofloxacin, amoxicillin, or tetracycline.  Pooled estimates of primary and secondary resistance and 95% confidence intervals (CIs) were grouped by WHO region.  The association between antibiotic resistance and treatment failure was measured by extracting data on treatment efficacy in patients with resistant and susceptible isolates and pooling odds ratios with 95% CIs.  RESULTS:  We identified 178 studies, comprising 66,142 isolates from 65 countries.  Primary and secondary resistance rates to clarithromycin, metronidazole, and levofloxacin were ≥15% in all WHO regions, except primary clarithromycin resistance in the Americas (10%; 95% CI, 4%-16%) and South-East Asia region (10%; 95% CI, 5%-16%) and primary levofloxacin resistance in the European region (11%; 95% CI, 9%-13%).  There was considerable heterogeneity (I(2) > 75%) among all analyses-this might have resulted from the grouping of resistance rates by country.  Increasing antibiotic resistance was observed in most WHO regions.  Resistance to clarithromycin was significantly associated with failure of clarithromycin-containing regimens (odds ratio, 6.97; 95% CI, 5.23-9.28; P < .001).  CONCLUSIONS:  Resistance of H pylori to antibiotics has reached alarming levels worldwide, which has a great effect on efficacy of treatment.  Local surveillance networks are required to select appropriate eradication regimens for each region.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdMPUw-TFMkOWhZTslLrfffW6udTcc2eY2D-C90iLN9Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c%252FmsFOmsA%253D%253D&md5=196e04c7bb5b3e5b75d2825ead30c5c8</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1053%2Fj.gastro.2018.07.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1053%252Fj.gastro.2018.07.007%26sid%3Dliteratum%253Aachs%26aulast%3DSavoldi%26aufirst%3DA.%26aulast%3DCarrara%26aufirst%3DE.%26aulast%3DGraham%26aufirst%3DD.%2BY.%26aulast%3DConti%26aufirst%3DM.%26aulast%3DTacconelli%26aufirst%3DE.%26atitle%3DPrevalence%2520of%2520antibiotic%2520resistance%2520in%2520Helicobacter%2520pylori%253A%2520A%2520systematic%2520review%2520and%2520meta-analysis%2520in%2520World%2520Health%2520Organization%2520regions%26jtitle%3DGastroenterology%26date%3D2018%26volume%3D155%26spage%3D1372%26epage%3D1382%26doi%3D10.1053%2Fj.gastro.2018.07.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiotani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, D. Y.</span></span> <span> </span><span class="NLM_article-title">Current and future treatment of <i>Helicobacter pylori</i> infections</span>. <i>Adv. Exp. Med. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>1149</i></span>,  <span class="NLM_fpage">211</span>– <span class="NLM_lpage">225</span>, <span class="refDoi"> DOI: 10.1007/5584_2019_367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1007%2F5584_2019_367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=31016626" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BB3cXosFGmtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1149&publication_year=2019&pages=211-225&author=H.+Matsumotoauthor=A.+Shiotaniauthor=D.+Y.+Graham&title=Current+and+future+treatment+of+Helicobacter+pylori+infections&doi=10.1007%2F5584_2019_367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Current and Future Treatment of Helicobacter pylori Infections</span></div><div class="casAuthors">Matsumoto, Hiroshi; Shiotani, Akiko; Graham, David Y.</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Experimental Medicine and Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">1149</span>
        (<span class="NLM_cas:issue">Helicobacter pylori in Human Diseases: Advances in Microbiology, Infectious Diseases and Public Health, Volume 11</span>),
    <span class="NLM_cas:pages">211-225</span>CODEN:
                <span class="NLM_cas:coden">AEMBAP</span>;
        ISSN:<span class="NLM_cas:issn">2214-8019</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Helicobacter pylori is one of the most common human pathogens and it has been estd. that about 50% of the world's population is currently infected.  The present consensus is that, unless there are compelling reasons, all H. pylori infections should be cured.  Since the 1990s, different national and international guidelines for the management of H. pylori -related diseases have been published and periodically updated regarding indications for treatment, diagnostic procedures, and preferred treatment regimens.  Most guidelines provide sophisticated meta-analyses examg. the outcome of different regimens done in regions with variable, often high rates of resistance to antibiotics, for which the prevalence and effects of resistance was often ignored.  Although successful antimicrobial therapy must be susceptibility-based, increasing antimicrobial resistance and general unavailability of susceptibility testing have required clinicians to generally rely on empiric regimens.  Antibiotics resistance of H. pylori has reached alarming high levels worldwide, which has an effect to efficacy of treatment.  The recommendations should provide regimes for multi-resistant infections or for those where susceptibility testing is unavailable or refused.  The first rule is to use only proven locally effective therapies.  Because of patient intolerances, drug allergies, and local experiences, the clinicians should have at least two options for first-line therapy.  As with any antimicrobial therapy, a thorough review of prior antibiotic use is invaluable to identify the presence of probably resistance.  The second key is patient education regarding potential and expected side-effects and the importance of completing the course of antibiotics.  We also review here triple therapies, sequential-concomitant, hybrid therapies, bismuth therapies, dual therapy, vonoprazan, modern antibiotic treatments, probiotics and vaccination.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9WdssYI98C7Vg90H21EOLACvtfcHk0lhOhwz1RjD-mQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXosFGmtr8%253D&md5=0bb9d394c1a9c4b0073871d0094cb814</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1007%2F5584_2019_367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F5584_2019_367%26sid%3Dliteratum%253Aachs%26aulast%3DMatsumoto%26aufirst%3DH.%26aulast%3DShiotani%26aufirst%3DA.%26aulast%3DGraham%26aufirst%3DD.%2BY.%26atitle%3DCurrent%2520and%2520future%2520treatment%2520of%2520Helicobacter%2520pylori%2520infections%26jtitle%3DAdv.%2520Exp.%2520Med.%2520Biol.%26date%3D2019%26volume%3D1149%26spage%3D211%26epage%3D225%26doi%3D10.1007%2F5584_2019_367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tacconelli, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrara, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savoldi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harbarth, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendelson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monnet, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pulcini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahlmeter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kluytmans, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmeli, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouellette, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Outterson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavaleri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houchens, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grayson, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theuretzbacher, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magrini, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aboderin, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Abri, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Awang Jalil, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benzonana, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattacharya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brink, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burkert, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cars, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornaglia, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dyar, O. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedrich, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gales, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gandra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giske, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goff, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goossens, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gottlieb, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzman Blanco, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hryniewicz, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kattula, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jinks, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanj, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerr, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kieny, M.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozlov, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labarca, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laxminarayan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leder, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leibovici, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy-Hara, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Littman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malhotra-Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manchanda, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moja, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ndoye, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paterson, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramon-Pardo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez-Baño, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanguinetti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sengupta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharland, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Si-Mehand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silver, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinbakk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomsen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thwaites, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Meer, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Kinh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vega, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villegas, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wechsler-Fördös, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wertheim, H. F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wesangula, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodford, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yilmaz, F. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zorzet, A.</span></span> <span> </span><span class="NLM_article-title">Discovery, research, and development of new antibiotics: the WHO priority list of antibiotic-resistant bacteria and tuberculosis</span>. <i>Lancet Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">318</span>– <span class="NLM_lpage">327</span>, <span class="refDoi"> DOI: 10.1016/s1473-3099(17)30753-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1016%2FS1473-3099%2817%2930753-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=29276051" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A280%3ADC%252BC1MzkvFOgsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=318-327&author=E.+Tacconelliauthor=E.+Carraraauthor=A.+Savoldiauthor=S.+Harbarthauthor=M.+Mendelsonauthor=D.+L.+Monnetauthor=C.+Pulciniauthor=G.+Kahlmeterauthor=J.+Kluytmansauthor=Y.+Carmeliauthor=M.+Ouelletteauthor=K.+Outtersonauthor=J.+Patelauthor=M.+Cavaleriauthor=E.+M.+Coxauthor=C.+R.+Houchensauthor=M.+L.+Graysonauthor=P.+Hansenauthor=N.+Singhauthor=U.+Theuretzbacherauthor=N.+Magriniauthor=A.+O.+Aboderinauthor=S.+S.+Al-Abriauthor=N.+Awang+Jalilauthor=N.+Benzonanaauthor=S.+Bhattacharyaauthor=A.+J.+Brinkauthor=F.+R.+Burkertauthor=O.+Carsauthor=G.+Cornagliaauthor=O.+J.+Dyarauthor=A.+W.+Friedrichauthor=A.+C.+Galesauthor=S.+Gandraauthor=C.+G.+Giskeauthor=D.+A.+Goffauthor=H.+Goossensauthor=T.+Gottliebauthor=M.+Guzman+Blancoauthor=W.+Hryniewiczauthor=D.+Kattulaauthor=T.+Jinksauthor=S.+S.+Kanjauthor=L.+Kerrauthor=M.-P.+Kienyauthor=Y.+S.+Kimauthor=R.+S.+Kozlovauthor=J.+Labarcaauthor=R.+Laxminarayanauthor=K.+Lederauthor=L.+Leiboviciauthor=G.+Levy-Haraauthor=J.+Littmanauthor=S.+Malhotra-Kumarauthor=V.+Manchandaauthor=L.+Mojaauthor=B.+Ndoyeauthor=A.+Panauthor=D.+L.+Patersonauthor=M.+Paulauthor=H.+Qiuauthor=P.+Ramon-Pardoauthor=J.+Rodr%C3%ADguez-Ba%C3%B1oauthor=M.+Sanguinettiauthor=S.+Senguptaauthor=M.+Sharlandauthor=M.+Si-Mehandauthor=L.+L.+Silverauthor=W.+Songauthor=M.+Steinbakkauthor=J.+Thomsenauthor=G.+E.+Thwaitesauthor=J.+W.+van+der+Meerauthor=N.+Van+Kinhauthor=S.+Vegaauthor=M.+V.+Villegasauthor=A.+Wechsler-F%C3%B6rd%C3%B6sauthor=H.+F.+L.+Wertheimauthor=E.+Wesangulaauthor=N.+Woodfordauthor=F.+O.+Yilmazauthor=A.+Zorzet&title=Discovery%2C+research%2C+and+development+of+new+antibiotics%3A+the+WHO+priority+list+of+antibiotic-resistant+bacteria+and+tuberculosis&doi=10.1016%2Fs1473-3099%2817%2930753-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery, research, and development of new antibiotics: the WHO priority list of antibiotic-resistant bacteria and tuberculosis</span></div><div class="casAuthors">Tacconelli Evelina; Carrara Elena; Savoldi Alessia; Harbarth Stephan; Mendelson Marc; Monnet Dominique L; Pulcini Celine; Kahlmeter Gunnar; Kluytmans Jan; Carmeli Yehuda; Ouellette Marc; Outterson Kevin; Patel Jean; Cavaleri Marco; Cox Edward M; Houchens Chris R; Grayson M Lindsay; Hansen Paul; Singh Nalini; Theuretzbacher Ursula; Magrini Nicola</div><div class="citationInfo"><span class="NLM_cas:title">The Lancet. Infectious diseases</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">318-327</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  The spread of antibiotic-resistant bacteria poses a substantial threat to morbidity and mortality worldwide.  Due to its large public health and societal implications, multidrug-resistant tuberculosis has been long regarded by WHO as a global priority for investment in new drugs.  In 2016, WHO was requested by member states to create a priority list of other antibiotic-resistant bacteria to support research and development of effective drugs.  METHODS:  We used a multicriteria decision analysis method to prioritise antibiotic-resistant bacteria; this method involved the identification of relevant criteria to assess priority against which each antibiotic-resistant bacterium was rated.  The final priority ranking of the antibiotic-resistant bacteria was established after a preference-based survey was used to obtain expert weighting of criteria.  FINDINGS:  We selected 20 bacterial species with 25 patterns of acquired resistance and ten criteria to assess priority: mortality, health-care burden, community burden, prevalence of resistance, 10-year trend of resistance, transmissibility, preventability in the community setting, preventability in the health-care setting, treatability, and pipeline.  We stratified the priority list into three tiers (critical, high, and medium priority), using the 33rd percentile of the bacterium's total scores as the cutoff.  Critical-priority bacteria included carbapenem-resistant Acinetobacter baumannii and Pseudomonas aeruginosa, and carbapenem-resistant and third-generation cephalosporin-resistant Enterobacteriaceae.  The highest ranked Gram-positive bacteria (high priority) were vancomycin-resistant Enterococcus faecium and meticillin-resistant Staphylococcus aureus.  Of the bacteria typically responsible for community-acquired infections, clarithromycin-resistant Helicobacter pylori, and fluoroquinolone-resistant Campylobacter spp, Neisseria gonorrhoeae, and Salmonella typhi were included in the high-priority tier.  INTERPRETATION:  Future development strategies should focus on antibiotics that are active against multidrug-resistant tuberculosis and Gram-negative bacteria.  The global strategy should include antibiotic-resistant bacteria responsible for community-acquired infections such as Salmonella spp, Campylobacter spp, N gonorrhoeae, and H pylori.  FUNDING:  World Health Organization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSkARC7DR8uZLGbPKh4T9eAfW6udTcc2eZQhccH7BRhh7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MzkvFOgsA%253D%253D&md5=14021ec57670b29fec6a88226a6a1fdd</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2FS1473-3099%2817%2930753-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1473-3099%252817%252930753-3%26sid%3Dliteratum%253Aachs%26aulast%3DTacconelli%26aufirst%3DE.%26aulast%3DCarrara%26aufirst%3DE.%26aulast%3DSavoldi%26aufirst%3DA.%26aulast%3DHarbarth%26aufirst%3DS.%26aulast%3DMendelson%26aufirst%3DM.%26aulast%3DMonnet%26aufirst%3DD.%2BL.%26aulast%3DPulcini%26aufirst%3DC.%26aulast%3DKahlmeter%26aufirst%3DG.%26aulast%3DKluytmans%26aufirst%3DJ.%26aulast%3DCarmeli%26aufirst%3DY.%26aulast%3DOuellette%26aufirst%3DM.%26aulast%3DOutterson%26aufirst%3DK.%26aulast%3DPatel%26aufirst%3DJ.%26aulast%3DCavaleri%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DE.%2BM.%26aulast%3DHouchens%26aufirst%3DC.%2BR.%26aulast%3DGrayson%26aufirst%3DM.%2BL.%26aulast%3DHansen%26aufirst%3DP.%26aulast%3DSingh%26aufirst%3DN.%26aulast%3DTheuretzbacher%26aufirst%3DU.%26aulast%3DMagrini%26aufirst%3DN.%26aulast%3DAboderin%26aufirst%3DA.%2BO.%26aulast%3DAl-Abri%26aufirst%3DS.%2BS.%26aulast%3DAwang%2BJalil%26aufirst%3DN.%26aulast%3DBenzonana%26aufirst%3DN.%26aulast%3DBhattacharya%26aufirst%3DS.%26aulast%3DBrink%26aufirst%3DA.%2BJ.%26aulast%3DBurkert%26aufirst%3DF.%2BR.%26aulast%3DCars%26aufirst%3DO.%26aulast%3DCornaglia%26aufirst%3DG.%26aulast%3DDyar%26aufirst%3DO.%2BJ.%26aulast%3DFriedrich%26aufirst%3DA.%2BW.%26aulast%3DGales%26aufirst%3DA.%2BC.%26aulast%3DGandra%26aufirst%3DS.%26aulast%3DGiske%26aufirst%3DC.%2BG.%26aulast%3DGoff%26aufirst%3DD.%2BA.%26aulast%3DGoossens%26aufirst%3DH.%26aulast%3DGottlieb%26aufirst%3DT.%26aulast%3DGuzman%2BBlanco%26aufirst%3DM.%26aulast%3DHryniewicz%26aufirst%3DW.%26aulast%3DKattula%26aufirst%3DD.%26aulast%3DJinks%26aufirst%3DT.%26aulast%3DKanj%26aufirst%3DS.%2BS.%26aulast%3DKerr%26aufirst%3DL.%26aulast%3DKieny%26aufirst%3DM.-P.%26aulast%3DKim%26aufirst%3DY.%2BS.%26aulast%3DKozlov%26aufirst%3DR.%2BS.%26aulast%3DLabarca%26aufirst%3DJ.%26aulast%3DLaxminarayan%26aufirst%3DR.%26aulast%3DLeder%26aufirst%3DK.%26aulast%3DLeibovici%26aufirst%3DL.%26aulast%3DLevy-Hara%26aufirst%3DG.%26aulast%3DLittman%26aufirst%3DJ.%26aulast%3DMalhotra-Kumar%26aufirst%3DS.%26aulast%3DManchanda%26aufirst%3DV.%26aulast%3DMoja%26aufirst%3DL.%26aulast%3DNdoye%26aufirst%3DB.%26aulast%3DPan%26aufirst%3DA.%26aulast%3DPaterson%26aufirst%3DD.%2BL.%26aulast%3DPaul%26aufirst%3DM.%26aulast%3DQiu%26aufirst%3DH.%26aulast%3DRamon-Pardo%26aufirst%3DP.%26aulast%3DRodr%25C3%25ADguez-Ba%25C3%25B1o%26aufirst%3DJ.%26aulast%3DSanguinetti%26aufirst%3DM.%26aulast%3DSengupta%26aufirst%3DS.%26aulast%3DSharland%26aufirst%3DM.%26aulast%3DSi-Mehand%26aufirst%3DM.%26aulast%3DSilver%26aufirst%3DL.%2BL.%26aulast%3DSong%26aufirst%3DW.%26aulast%3DSteinbakk%26aufirst%3DM.%26aulast%3DThomsen%26aufirst%3DJ.%26aulast%3DThwaites%26aufirst%3DG.%2BE.%26aulast%3Dvan%2Bder%2BMeer%26aufirst%3DJ.%2BW.%26aulast%3DVan%2BKinh%26aufirst%3DN.%26aulast%3DVega%26aufirst%3DS.%26aulast%3DVillegas%26aufirst%3DM.%2BV.%26aulast%3DWechsler-F%25C3%25B6rd%25C3%25B6s%26aufirst%3DA.%26aulast%3DWertheim%26aufirst%3DH.%2BF.%2BL.%26aulast%3DWesangula%26aufirst%3DE.%26aulast%3DWoodford%26aufirst%3DN.%26aulast%3DYilmaz%26aufirst%3DF.%2BO.%26aulast%3DZorzet%26aufirst%3DA.%26atitle%3DDiscovery%252C%2520research%252C%2520and%2520development%2520of%2520new%2520antibiotics%253A%2520the%2520WHO%2520priority%2520list%2520of%2520antibiotic-resistant%2520bacteria%2520and%2520tuberculosis%26jtitle%3DLancet%2520Infect.%2520Dis.%26date%3D2018%26volume%3D18%26spage%3D318%26epage%3D327%26doi%3D10.1016%2Fs1473-3099%2817%2930753-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alm, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trust, T. J.</span></span> <span> </span><span class="NLM_article-title">Analysis of the genetic diversity of <i>Helicobacter pylori</i>: the tale of two genomes</span>. <i>J. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">834</span>– <span class="NLM_lpage">846</span>, <span class="refDoi"> DOI: 10.1007/s001099900067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1007%2Fs001099900067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10682319" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BD3cXht1alu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=1999&pages=834-846&author=R.+A.+Almauthor=T.+J.+Trust&title=Analysis+of+the+genetic+diversity+of+Helicobacter+pylori%3A+the+tale+of+two+genomes&doi=10.1007%2Fs001099900067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of the genetic diversity of Helicobacter pylori: the tale of two genomes</span></div><div class="casAuthors">Alm, Richard A.; Trust, Trevor J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Medicine (Berlin)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">834-846</span>CODEN:
                <span class="NLM_cas:coden">JMLME8</span>;
        ISSN:<span class="NLM_cas:issn">0946-2716</span>.
    
            (<span class="NLM_cas:orgname">Springer-Verlag</span>)
        </div><div class="casAbstract">Infection with Helicobacter pylori has been linked to numerous severe gastroduodenal diseases including peptic ulcer and gastric cancer.  Several techniques have been used to measure the genetic heterogeneity of H. pylori at several different levels and to det. whether there is any correlation with severity of disease.  The availability of two completed genome sequences from unrelated strains (J99 and 26695) has allowed an anal. of the level of diversity from a large-scale yet detailed perspective.  Although the two chromosomes are organized differently in a limited no. of discrete regions, the genome size and gene order of these two "high-virulence" (cagA+ and vacA+) H. pylori isolates was found to be highly similar.  The regions of organizational difference are assocd. with insertion sequences, DNA restriction/modification genes, repeat sequences, or a combination of the above.  A significant level of variation at the nucleotide level is seen across the genome, providing an explanation for why the nucleotide-based typing techniques have such high discriminatory power among independent H. pylori isolates.  This nucleotide variation together with the organizational rearrangements appears to have provided an over-estn. of the gene order diversity of H. pylori as assessed by pulse-field gel electrophoresis.  Functional assignments are assigned to approx. only 60% of the gene products in each strain, with one-half of the remaining gene products of unknown function having homologues in other bacteria, while the remainder appear to be H. pylori-specific.  Between 6% and 7% of the coding capacity of each strain are genes that are absent from the other strain, with almost one-half of these strain-specific genes located in a single hypervariable region called the plasticity zone.  The majority of the strain-specific genes in each strain are also H. pylori-specific, with no homologues being identified in the public databases.  Significantly, over one-half of the functionally assigned strain-specific genes in both H. pylori J99 and 26695 encode DNA restriction/modification enzymes.  Anal. of the level of conservation between orthologues from the two strains indicates that the H. pylori specific genes have a lower level of conservation than those orthologues to which a putative function can be assigned.  The plasticity zone represents one of several regions across each genome that is comprised of lower (G+C)% content DNA, some of which has been detected in self-replicating plasmids, suggesting that both horizontal transfer from other species and plasmid integration are responsible for the strain-specific diversity at this locus.  These analyses have yielded results with important implications for understanding the genetic diversity of H. pylori and its assocd. diseases, and imply a need to reassess the resp. roles of bacterial and host factors in H. pylori assocd. diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNZS65Up1rmbVg90H21EOLACvtfcHk0lgU-Fu5ohItOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXht1alu7g%253D&md5=88bed741cf5e4de688dc3ba1a4874e60</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1007%2Fs001099900067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs001099900067%26sid%3Dliteratum%253Aachs%26aulast%3DAlm%26aufirst%3DR.%2BA.%26aulast%3DTrust%26aufirst%3DT.%2BJ.%26atitle%3DAnalysis%2520of%2520the%2520genetic%2520diversity%2520of%2520Helicobacter%2520pylori%253A%2520the%2520tale%2520of%2520two%2520genomes%26jtitle%3DJ.%2520Mol.%2520Med.%26date%3D1999%26volume%3D77%26spage%3D834%26epage%3D846%26doi%3D10.1007%2Fs001099900067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doig, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jonge, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alm, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uria-Nickelsen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noonan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummino, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guild, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moir, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vovis, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trust, T. J.</span></span> <span> </span><span class="NLM_article-title"><i>Helicobacter pylori</i> physiology predicted from genomic comparison of two strains</span>. <i>Microbiol. Mol. Biol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">675</span>– <span class="NLM_lpage">707</span>, <span class="refDoi"> DOI: 10.1128/mmbr.63.3.675-707.1999</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1128%2FMMBR.63.3.675-707.1999" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10477312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADyaK1MXmt1CgsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=1999&pages=675-707&author=P.+Doigauthor=B.+L.+de+Jongeauthor=R.+A.+Almauthor=E.+D.+Brownauthor=M.+Uria-Nickelsenauthor=B.+Noonanauthor=S.+D.+Millsauthor=P.+Tumminoauthor=G.+Carmelauthor=B.+C.+Guildauthor=D.+T.+Moirauthor=G.+F.+Vovisauthor=T.+J.+Trust&title=Helicobacter+pylori+physiology+predicted+from+genomic+comparison+of+two+strains&doi=10.1128%2Fmmbr.63.3.675-707.1999"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Helicobacter pylori physiology predicted from genomic comparison of two strains</span></div><div class="casAuthors">Doig, Peter; De Jonge, Boudewijn L.; Alm, Richard A.; Brown, Eric D.; Uria-Nickelsen, Maria; Noonan, Brian; Mills, Scott D.; Tummino, Peter; Carmel, Gilles; Guild, Braydon C.; Moir, Donald T.; Vovis, Gerald F.; Trust, Trevor J.</div><div class="citationInfo"><span class="NLM_cas:title">Microbiology and Molecular Biology Reviews</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">675-707</span>CODEN:
                <span class="NLM_cas:coden">MMBRF7</span>;
        ISSN:<span class="NLM_cas:issn">1092-2172</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A review with 160 refs.  Helicobacter pylori is a gram-neg. bacteria which colonizes the gastric mucosa of humans and is implicated in a wide range of gastroduodenal diseases.  This paper reviews the physiol. of this bacterium as predicted from the sequenced genomes of two unrelated strains and reconciles these predictions with the literature.  In general, the predicted capabilities are in good agreement with reported exptl. observations.  H. pylori is limited in carbohydrate utilization and will use amino acids, for which it has transporter systems, as sources of carbon.  Energy can be generated by fermn., and the bacterium possesses components necessary for both aerobic and anaerobic respiration.  Sulfur metab. is limited, whereas nitrogen metab. is extensive.  There is active uptake of DNA via transformation and ample restriction-modification activities.  The cell contains numerous outer membrane proteins, some of which are porins or involved in iron uptake.  Some of these outer membrane proteins and the lipopolysaccharide may be regulated by a slipped-strand repair mechanism which probably results in phase variation and plays a role in colonization.  In contrast to a commonly held belief that H. pylori is a very diverse species, few differences were predicted in the physiol. of these two unrelated strains, indicating that host and environmental factors probably play a significant role in the outcome of H. pylori-related disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobMBLuOtLulbVg90H21EOLACvtfcHk0lgU-Fu5ohItOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmt1CgsLg%253D&md5=47b3a0e3eec3eaea3bf8f3ff869e84dc</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1128%2FMMBR.63.3.675-707.1999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMMBR.63.3.675-707.1999%26sid%3Dliteratum%253Aachs%26aulast%3DDoig%26aufirst%3DP.%26aulast%3Dde%2BJonge%26aufirst%3DB.%2BL.%26aulast%3DAlm%26aufirst%3DR.%2BA.%26aulast%3DBrown%26aufirst%3DE.%2BD.%26aulast%3DUria-Nickelsen%26aufirst%3DM.%26aulast%3DNoonan%26aufirst%3DB.%26aulast%3DMills%26aufirst%3DS.%2BD.%26aulast%3DTummino%26aufirst%3DP.%26aulast%3DCarmel%26aufirst%3DG.%26aulast%3DGuild%26aufirst%3DB.%2BC.%26aulast%3DMoir%26aufirst%3DD.%2BT.%26aulast%3DVovis%26aufirst%3DG.%2BF.%26aulast%3DTrust%26aufirst%3DT.%2BJ.%26atitle%3DHelicobacter%2520pylori%2520physiology%2520predicted%2520from%2520genomic%2520comparison%2520of%2520two%2520strains%26jtitle%3DMicrobiol.%2520Mol.%2520Biol.%2520Rev.%26date%3D1999%26volume%3D63%26spage%3D675%26epage%3D707%26doi%3D10.1128%2Fmmbr.63.3.675-707.1999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mendz, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jimenez, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hazell, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gero, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Sullivan, W. J.</span></span> <span> </span><span class="NLM_article-title">Salvage synthesis of purine nucleotides by <i>Helicobacter pylori</i></span>. <i>J. Appl. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">674</span>– <span class="NLM_lpage">681</span>, <span class="refDoi"> DOI: 10.1111/j.1365-2672.1994.tb02818.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1111%2Fj.1365-2672.1994.tb02818.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=7822226" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADyaK2MXjtV2nt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=1994&pages=674-681&author=G.+L.+Mendzauthor=B.+M.+Jimenezauthor=S.+L.+Hazellauthor=A.+M.+Geroauthor=W.+J.+O%E2%80%99Sullivan&title=Salvage+synthesis+of+purine+nucleotides+by+Helicobacter+pylori&doi=10.1111%2Fj.1365-2672.1994.tb02818.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Salvage synthesis of purine nucleotides by Helicobacter pylori</span></div><div class="casAuthors">Mendz, G.L.; Jimenez, B.M.; Hazell, S.L.; Gero, A.M.; O'Sullivan, W.J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Bacteriology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">674-81</span>CODEN:
                <span class="NLM_cas:coden">JABAA4</span>;
        ISSN:<span class="NLM_cas:issn">0021-8847</span>.
    </div><div class="casAbstract">The incorporation of purine nucleotide precursors into Helicobacter pylori and the activities of enzymes involved in nucleotide salvage biosynthetic pathways were investigated by radioactive tracer anal. and NMR spectroscopy.  The organism took up the nucleobases adenine, guanine and hypoxanthine, and the nucleosides adenosine, guanosine and deoxyadenosine.  Any incorporation of deoxyguanosine by the cells was below the detection limits of the methods employed.  The activities of adenine-, guanine- and hypoxanthine-phosphoribosyltransferases were established.  The bacterium showed high levels of adenosine and guanosine nucleosidase activities and lesser activity for deoxyadenosine; no hydrolysis of deoxyguanosine was detected.  Phosphorylase activities were not obsd. with any of the nucleosides.  Phosphotransferase activities with similar rates were demonstrated for adenosine, guanosine and deoxyadenosine; and a weaker activity was detected for deoxyguanosine.  No nucleoside kinase activities were obsd. with any of the nucleosides.  The presence of adenylate kinase was established, but no guanylate kinase activity was obsd.  The study provided evidence for the presence in H. pylori of salvage pathways for the biosynthesis of purine nucleotides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeDXkLLAPIVrVg90H21EOLACvtfcHk0lgU-Fu5ohItOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXjtV2nt7Y%253D&md5=b29745361f8364ede6bd1b4d8d59d81a</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2672.1994.tb02818.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2672.1994.tb02818.x%26sid%3Dliteratum%253Aachs%26aulast%3DMendz%26aufirst%3DG.%2BL.%26aulast%3DJimenez%26aufirst%3DB.%2BM.%26aulast%3DHazell%26aufirst%3DS.%2BL.%26aulast%3DGero%26aufirst%3DA.%2BM.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DW.%2BJ.%26atitle%3DSalvage%2520synthesis%2520of%2520purine%2520nucleotides%2520by%2520Helicobacter%2520pylori%26jtitle%3DJ.%2520Appl.%2520Bacteriol.%26date%3D1994%26volume%3D77%26spage%3D674%26epage%3D681%26doi%3D10.1111%2Fj.1365-2672.1994.tb02818.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liechti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, J. B.</span></span> <span> </span><span class="NLM_article-title"><i>Helicobacter pylori</i> relies primarily on the purine salvage pathway for purine nucleotide biosynthesis</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>194</i></span>,  <span class="NLM_fpage">839</span>– <span class="NLM_lpage">854</span>, <span class="refDoi"> DOI: 10.1128/jb.05757-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1128%2FJB.05757-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=22194455" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvFKhsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=194&publication_year=2012&pages=839-854&author=G.+Liechtiauthor=J.+B.+Goldberg&title=Helicobacter+pylori+relies+primarily+on+the+purine+salvage+pathway+for+purine+nucleotide+biosynthesis&doi=10.1128%2Fjb.05757-11"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Helicobacter pylori relies primarily on the purine salvage pathway for purine nucleotide biosynthesis</span></div><div class="casAuthors">Liechti, George; Goldberg, Joanna B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">194</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">839-854</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Helicobacter pylori is a chronic colonizer of the gastric epithelium and plays a major role in the development of gastritis, peptic ulcer disease, and gastric cancer.  In its coevolution with humans, the streamlining of the H. pylori genome has resulted in a significant redn. in metabolic pathways, one being purine nucleotide biosynthesis.  Bioinformatic anal. has revealed that H. pylori lacks the enzymic machinery for de novo prodn. of IMP, the first purine nucleotide formed during GTP and ATP biosynthesis.  This suggests that H. pylori must rely heavily on salvage of purines from the environment.  In this study, the authors deleted several genes putatively involved in purine salvage and processing.  The growth and survival of these mutants were analyzed in both nutrient-rich and minimal media, and the results confirmed the presence of a robust purine salvage pathway in H. pylori.  Of the two phosphoribosyltransferase genes found in the H. pylori genome, only gpt appears to be essential, and an Δapt mutant strain was still capable of growth on adenine, suggesting that adenine processing via Apt is not essential.  Deletion of the putative nucleoside phosphorylase gene deoD resulted in an inability of H. pylori to grow on purine nucleosides or the purine base adenine.  The results suggest a purine requirement for growth of H. pylori in std. media, indicating that H. pylori possesses the ability to utilize purines and nucleosides from the environment in the absence of a de novo purine nucleotide biosynthesis pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkwZ7_YPN3JbVg90H21EOLACvtfcHk0lgbOp9yB2U6AQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvFKhsrg%253D&md5=bc5df306a59c0ea85f24f22c04de04a0</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1128%2FJB.05757-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJB.05757-11%26sid%3Dliteratum%253Aachs%26aulast%3DLiechti%26aufirst%3DG.%26aulast%3DGoldberg%26aufirst%3DJ.%2BB.%26atitle%3DHelicobacter%2520pylori%2520relies%2520primarily%2520on%2520the%2520purine%2520salvage%2520pathway%2520for%2520purine%2520nucleotide%2520biosynthesis%26jtitle%3DJ.%2520Bacteriol.%26date%3D2012%26volume%3D194%26spage%3D839%26epage%3D854%26doi%3D10.1128%2Fjb.05757-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hedstrom, L.</span></span> <span> </span><span class="NLM_article-title">IMP dehydrogenase: structure, mechanism, and inhibition</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">2903</span>– <span class="NLM_lpage">2928</span>, <span class="refDoi"> DOI: 10.1021/cr900021w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr900021w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmsFamtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2009&pages=2903-2928&author=L.+Hedstrom&title=IMP+dehydrogenase%3A+structure%2C+mechanism%2C+and+inhibition&doi=10.1021%2Fcr900021w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">IMP Dehydrogenase: Structure, Mechanism, and Inhibition</span></div><div class="casAuthors">Hedstrom, Lizbeth</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2903-2928</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The discovery that IMP dehydrogenase (IMPDH) is amplified in tumors and rapidly proliferating tissues provided the foundation for design of drugs targeting this enzyme.  Though yet much success in the cancer arena has not been achieved, IMPDH inhibitors are now widely used in immunosuppressive and antiviral chemotherapy, and IMPDH may also be a target for antimicrobial drugs.  Clin. relevance aside, IMPDH is a fascinating enzyme.  It traverses several conformations while catalyzing two different chem. transformations, utilizing unusual chem. strategies to promote each reaction.  Monovalent cations such as potassium activate IMPDH, possibly by acting as a mol. lubricant to facilitate these conformational changes.  The biol. of IMPDH also displays some surprising twists.  IMPDH binds nucleic acids and is assocd. with polyribosomes, though the physiol. role of this interaction also has not yet been elucidated.  Perhaps most intriguing is the discovery that mutations in IMPDH are assocd. with hereditary retinal disease.  These mutations cluster to a subdomain that is not required for enzymic activity, and the function of this subdomain is currently under debate.  The paper reviews recent work on the biochem. of IMPDH, integrating structure, function, and inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHrolvTNRBmbVg90H21EOLACvtfcHk0lgbOp9yB2U6AQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmsFamtbo%253D&md5=f475f8b409366b48440100770242bccf</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fcr900021w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr900021w%26sid%3Dliteratum%253Aachs%26aulast%3DHedstrom%26aufirst%3DL.%26atitle%3DIMP%2520dehydrogenase%253A%2520structure%252C%2520mechanism%252C%2520and%2520inhibition%26jtitle%3DChem.%2520Rev.%26date%3D2009%26volume%3D109%26spage%3D2903%26epage%3D2928%26doi%3D10.1021%2Fcr900021w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keough, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocková, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janeba, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naesens, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edstein, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chavchich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span> <span> </span><span class="NLM_article-title">Aza-acyclic nucleoside phosphonates containing a second phosphonate group as inhibitors of the human, <i>Plasmodium falciparum</i> and <i>vivax</i> 6-oxopurine phosphoribosyltransferases and their prodrugs as antimalarial agents</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">827</span>– <span class="NLM_lpage">846</span>, <span class="refDoi"> DOI: 10.1021/jm501416t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501416t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVyhsrzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=827-846&author=D.+T.+Keoughauthor=D.+Hockov%C3%A1author=Z.+Janebaauthor=T.-H.+Wangauthor=L.+Naesensauthor=M.+D.+Edsteinauthor=M.+Chavchichauthor=L.+W.+Guddat&title=Aza-acyclic+nucleoside+phosphonates+containing+a+second+phosphonate+group+as+inhibitors+of+the+human%2C+Plasmodium+falciparum+and+vivax+6-oxopurine+phosphoribosyltransferases+and+their+prodrugs+as+antimalarial+agents&doi=10.1021%2Fjm501416t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Aza-acyclic Nucleoside Phosphonates Containing a Second Phosphonate Group As Inhibitors of the Human, Plasmodium falciparum and vivax 6-Oxopurine Phosphoribosyltransferases and Their Prodrugs As Antimalarial Agents</span></div><div class="casAuthors">Keough, Dianne T.; Hockova, Dana; Janeba, Zlatko; Wang, Tzu-Hsuan; Naesens, Lieve; Edstein, Michael D.; Chavchich, Marina; Guddat, Luke W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">827-846</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Hypoxanthine-guanine-[xanthine] phosphoribosyltransferase (HG[X]PRT) is considered an important target for antimalarial chemotherapy as it is the only pathway for the synthesis of the purine nucleoside monophosphates required for DNA/RNA prodn.  Thus, inhibition of this enzyme should result in cessation of replication.  The aza-acyclic nucleoside phosphonates (aza-ANPs) are good inhibitors of Plasmodium falciparum HGXPRT (PfHGXPRT), with Ki values as low as 0.08 and 0.01 μM for Plasmodium vivax HGPRT (PvHGPRT).  Prodrugs of these aza-ANPs exhibit antimalarial activity against Pf lines with IC50 values (0.8-6.0 μM) and have low cytotoxicity against human cells.  Crystal structures of six of these compds. in complex with human HGPRT have been detd.  These suggest that the different affinities of these aza-ANPs could be due to the flexibility of the loops surrounding the active site as well as the flexibility of the inhibitors, allowing them to adapt to fit into three binding pockets of the enzyme(s).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9eXB3S3z3v7Vg90H21EOLACvtfcHk0lgbOp9yB2U6AQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVyhsrzF&md5=c7d1dad69f99a1b4928b2a65fa411670</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm501416t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501416t%26sid%3Dliteratum%253Aachs%26aulast%3DKeough%26aufirst%3DD.%2BT.%26aulast%3DHockov%25C3%25A1%26aufirst%3DD.%26aulast%3DJaneba%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DT.-H.%26aulast%3DNaesens%26aufirst%3DL.%26aulast%3DEdstein%26aufirst%3DM.%2BD.%26aulast%3DChavchich%26aufirst%3DM.%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26atitle%3DAza-acyclic%2520nucleoside%2520phosphonates%2520containing%2520a%2520second%2520phosphonate%2520group%2520as%2520inhibitors%2520of%2520the%2520human%252C%2520Plasmodium%2520falciparum%2520and%2520vivax%25206-oxopurine%2520phosphoribosyltransferases%2520and%2520their%2520prodrugs%2520as%2520antimalarial%2520agents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D827%26epage%3D846%26doi%3D10.1021%2Fjm501416t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keough, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rejman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pohl, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zborníková, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocková, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Croll, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edstein, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birrell, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chavchich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naesens, L. M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierens, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brereton, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span> <span> </span><span class="NLM_article-title">Design of <i>Plasmodium vivax</i> hypoxanthine-guanine phosphoribosyltransferase inhibitors as potential antimalarial therapeutics</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">82</span>– <span class="NLM_lpage">90</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00916</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00916" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=82-90&author=D.+T.+Keoughauthor=D.+Rejmanauthor=R.+Pohlauthor=E.+Zborn%C3%ADkov%C3%A1author=D.+Hockov%C3%A1author=T.+Crollauthor=M.+D.+Edsteinauthor=G.+W.+Birrellauthor=M.+Chavchichauthor=L.+M.+J.+Naesensauthor=G.+K.+Pierensauthor=I.+M.+Breretonauthor=L.+W.+Guddat&title=Design+of+Plasmodium+vivax+hypoxanthine-guanine+phosphoribosyltransferase+inhibitors+as+potential+antimalarial+therapeutics&doi=10.1021%2Facschembio.7b00916"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00916&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00916%26sid%3Dliteratum%253Aachs%26aulast%3DKeough%26aufirst%3DD.%2BT.%26aulast%3DRejman%26aufirst%3DD.%26aulast%3DPohl%26aufirst%3DR.%26aulast%3DZborn%25C3%25ADkov%25C3%25A1%26aufirst%3DE.%26aulast%3DHockov%25C3%25A1%26aufirst%3DD.%26aulast%3DCroll%26aufirst%3DT.%26aulast%3DEdstein%26aufirst%3DM.%2BD.%26aulast%3DBirrell%26aufirst%3DG.%2BW.%26aulast%3DChavchich%26aufirst%3DM.%26aulast%3DNaesens%26aufirst%3DL.%2BM.%2BJ.%26aulast%3DPierens%26aufirst%3DG.%2BK.%26aulast%3DBrereton%26aufirst%3DI.%2BM.%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26atitle%3DDesign%2520of%2520Plasmodium%2520vivax%2520hypoxanthine-guanine%2520phosphoribosyltransferase%2520inhibitors%2520as%2520potential%2520antimalarial%2520therapeutics%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D82%26epage%3D90%26doi%3D10.1021%2Facschembio.7b00916" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keough, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocková, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holý, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naesens, L. M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skinner-Adams, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jersey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span> <span> </span><span class="NLM_article-title">Inhibition of hypoxanthine-guanine phosphoribosyltransferase by acyclic nucleoside phosphonates: A new class of antimalarial therapeutics</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">4391</span>– <span class="NLM_lpage">4399</span>, <span class="refDoi"> DOI: 10.1021/jm900267n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm900267n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnt1Ons7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=4391-4399&author=D.+T.+Keoughauthor=D.+Hockov%C3%A1author=A.+Hol%C3%BDauthor=L.+M.+J.+Naesensauthor=T.+S.+Skinner-Adamsauthor=J.+de+Jerseyauthor=L.+W.+Guddat&title=Inhibition+of+hypoxanthine-guanine+phosphoribosyltransferase+by+acyclic+nucleoside+phosphonates%3A+A+new+class+of+antimalarial+therapeutics&doi=10.1021%2Fjm900267n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Hypoxanthine-Guanine Phosphoribosyltransferase by Acyclic Nucleoside Phosphonates: A New Class of Antimalarial Therapeutics</span></div><div class="casAuthors">Keough, Dianne T.; Hockova, Dana; Holy, Antonin; Naesens, Lieve M. J.; Skinner-Adams, Tina S.; de Jersey, John; Guddat, Luke W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4391-4399</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The purine salvage enzyme hypoxanthine-guanine-xanthine phosphoribosyltransferase (HGXPRT) is essential for purine nucleotide and hence nucleic acid synthesis in the malaria parasite, Plasmodium falciparum.  Acyclic nucleoside phosphonates (ANPs) are analogs of the nucleotide product of the reaction, comprising a purine base joined by a linker to a phosphonate moiety.  Ki values for 19 ANPs were detd. for Pf HGXPRT and the corresponding human enzyme, HGPRT.  Values for Pf HGXPRT were as low as 100 nM, with selectivity for the parasite enzyme of up to 58.  Structures of human HGPRT in complex with three ANPs are reported.  On binding, a large mobile loop in the free enzyme moves to partly cover the active site.  For three ANPs, the IC50 values for Pf grown in cell culture were 1, 14, and 46 μM, while the cytotoxic concn. for the first compd. was 489 μM.  These results provide a basis for the design of potent and selective ANP inhibitors of Pf HGXPRT as antimalarial drug leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqn8rPcoy0EBbVg90H21EOLACvtfcHk0lhO9buWZftnrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnt1Ons7w%253D&md5=76746f78ec791a08ba2376beba86fa2b</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm900267n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm900267n%26sid%3Dliteratum%253Aachs%26aulast%3DKeough%26aufirst%3DD.%2BT.%26aulast%3DHockov%25C3%25A1%26aufirst%3DD.%26aulast%3DHol%25C3%25BD%26aufirst%3DA.%26aulast%3DNaesens%26aufirst%3DL.%2BM.%2BJ.%26aulast%3DSkinner-Adams%26aufirst%3DT.%2BS.%26aulast%3Dde%2BJersey%26aufirst%3DJ.%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26atitle%3DInhibition%2520of%2520hypoxanthine-guanine%2520phosphoribosyltransferase%2520by%2520acyclic%2520nucleoside%2520phosphonates%253A%2520A%2520new%2520class%2520of%2520antimalarial%2520therapeutics%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D4391%26epage%3D4399%26doi%3D10.1021%2Fjm900267n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hazleton, K. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cassera, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clinch, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crump, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosario, I.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merino, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almo, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyler, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schramm, V. L.</span></span> <span> </span><span class="NLM_article-title">Acyclic immucillin phosphonates: Second generation inhibitors of <i>Plasmodium falciparum</i> hypoxanthine-guanine-xanthine phosphoribosyltransferase</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">721</span>– <span class="NLM_lpage">730</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2012.04.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1016%2Fj.chembiol.2012.04.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=22726686" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC38XptVKqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=721-730&author=K.+Z.+Hazletonauthor=M.-C.+Hoauthor=M.+B.+Casseraauthor=K.+Clinchauthor=D.+R.+Crumpauthor=I.+Rosarioauthor=E.+F.+Merinoauthor=S.+C.+Almoauthor=P.+C.+Tylerauthor=V.+L.+Schramm&title=Acyclic+immucillin+phosphonates%3A+Second+generation+inhibitors+of+Plasmodium+falciparum+hypoxanthine-guanine-xanthine+phosphoribosyltransferase&doi=10.1016%2Fj.chembiol.2012.04.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Acyclic Immucillin Phosphonates: Second-Generation Inhibitors of Plasmodium falciparum Hypoxanthine- Guanine-Xanthine Phosphoribosyltransferase</span></div><div class="casAuthors">Hazleton, Keith Z.; Ho, Meng-Chiao; Cassera, Maria B.; Clinch, Keith; Crump, Douglas R.; Rosario, Irving; Merino, Emilio F.; Almo, Steve C.; Tyler, Peter C.; Schramm, Vern L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">721-730</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Plasmodium falciparum, the primary cause of deaths from malaria, is a purine auxotroph and relies on hypoxanthine salvage from the host purine pool.  Purine starvation as an antimalarial target has been validated by inhibition of purine nucleoside phosphorylase.  Hypoxanthine depletion kills Plasmodium falciparum in cell culture and in Aotus monkey infections.  Hypoxanthine-guanine-xanthine phosphoribosyltransferase (HGXPRT) from P. falciparum is required for hypoxanthine salvage by forming IMP, a branchpoint for all purine nucleotide synthesis in the parasite.  Here, we present a class of HGXPRT inhibitors, the acyclic immucillin phosphonates (AIPs), and cell permeable AIP prodrugs.  The AIPs are simple, potent, selective, and biol. stable inhibitors.  The AIP prodrugs block proliferation of cultured parasites by inhibiting the incorporation of hypoxanthine into the parasite nucleotide pool and validates HGXPRT as a target in malaria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJ-0LZKeBULLVg90H21EOLACvtfcHk0lhO9buWZftnrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptVKqsLs%253D&md5=9c72350a38b247652ca5209996a73495</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2012.04.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2012.04.012%26sid%3Dliteratum%253Aachs%26aulast%3DHazleton%26aufirst%3DK.%2BZ.%26aulast%3DHo%26aufirst%3DM.-C.%26aulast%3DCassera%26aufirst%3DM.%2BB.%26aulast%3DClinch%26aufirst%3DK.%26aulast%3DCrump%26aufirst%3DD.%2BR.%26aulast%3DRosario%26aufirst%3DI.%26aulast%3DMerino%26aufirst%3DE.%2BF.%26aulast%3DAlmo%26aufirst%3DS.%2BC.%26aulast%3DTyler%26aufirst%3DP.%2BC.%26aulast%3DSchramm%26aufirst%3DV.%2BL.%26atitle%3DAcyclic%2520immucillin%2520phosphonates%253A%2520Second%2520generation%2520inhibitors%2520of%2520Plasmodium%2520falciparum%2520hypoxanthine-guanine-xanthine%2520phosphoribosyltransferase%26jtitle%3DChem.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D721%26epage%3D730%26doi%3D10.1016%2Fj.chembiol.2012.04.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clinch, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crump, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hazleton, K. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schramm, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyler, P. C.</span></span> <span> </span><span class="NLM_article-title">Acyclic phosph(on)ate inhibitors of <i>Plasmodium falciparum</i> hypoxanthine-guanine-xanthine phosphoribosyltransferase</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">5629</span>– <span class="NLM_lpage">5646</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2013.02.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1016%2Fj.bmc.2013.02.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=23810424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVWls73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=5629-5646&author=K.+Clinchauthor=D.+R.+Crumpauthor=G.+B.+Evansauthor=K.+Z.+Hazletonauthor=J.+M.+Masonauthor=V.+L.+Schrammauthor=P.+C.+Tyler&title=Acyclic+phosph%28on%29ate+inhibitors+of+Plasmodium+falciparum+hypoxanthine-guanine-xanthine+phosphoribosyltransferase&doi=10.1016%2Fj.bmc.2013.02.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Acyclic phosph(on)ate inhibitors of Plasmodium falciparum hypoxanthine-guanine-xanthine phosphoribosyltransferase</span></div><div class="casAuthors">Clinch, Keith; Crump, Douglas R.; Evans, Gary B.; Hazleton, Keith Z.; Mason, Jennifer M.; Schramm, Vern L.; Tyler, Peter C.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5629-5646</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The pathogenic protozoa responsible for malaria lack enzymes for the de novo synthesis of purines and rely on purine salvage from the host.  In Plasmodium falciparum (Pf), hypoxanthine-guanine-xanthine phosphoribosyltransferase (HGXPRT) converts hypoxanthine to inosine monophosphate and is essential for purine salvage making the enzyme an anti-malarial drug target.  We have synthesized a no. of simple acyclic aza-C-nucleosides and shown that some are potent inhibitors of Pf HGXPRT while showing excellent selectivity for the Pf vs. the human enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUUjPr4oHetrVg90H21EOLACvtfcHk0lgaQ81a0s_OVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVWls73I&md5=1a7be25d17441e003e48c568a3ba231c</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.02.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.02.016%26sid%3Dliteratum%253Aachs%26aulast%3DClinch%26aufirst%3DK.%26aulast%3DCrump%26aufirst%3DD.%2BR.%26aulast%3DEvans%26aufirst%3DG.%2BB.%26aulast%3DHazleton%26aufirst%3DK.%2BZ.%26aulast%3DMason%26aufirst%3DJ.%2BM.%26aulast%3DSchramm%26aufirst%3DV.%2BL.%26aulast%3DTyler%26aufirst%3DP.%2BC.%26atitle%3DAcyclic%2520phosph%2528on%2529ate%2520inhibitors%2520of%2520Plasmodium%2520falciparum%2520hypoxanthine-guanine-xanthine%2520phosphoribosyltransferase%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D5629%26epage%3D5646%26doi%3D10.1016%2Fj.bmc.2013.02.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eng, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocková, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Špaček, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janeba, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woods, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naesens, L. M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keough, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span> <span> </span><span class="NLM_article-title">First crystal structures of <i>Mycobacterium tuberculosis</i> 6-oxopurine phosphoribosyltransferase: Complexes with GMP and pyrophosphate and with acyclic nucleoside phosphonates whose prodrugs have antituberculosis activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">4822</span>– <span class="NLM_lpage">4838</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00611</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00611" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC2MXntFGitL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=4822-4838&author=W.+S.+Engauthor=D.+Hockov%C3%A1author=P.+%C5%A0pa%C4%8Dekauthor=Z.+Janebaauthor=N.+P.+Westauthor=K.+Woodsauthor=L.+M.+J.+Naesensauthor=D.+T.+Keoughauthor=L.+W.+Guddat&title=First+crystal+structures+of+Mycobacterium+tuberculosis+6-oxopurine+phosphoribosyltransferase%3A+Complexes+with+GMP+and+pyrophosphate+and+with+acyclic+nucleoside+phosphonates+whose+prodrugs+have+antituberculosis+activity&doi=10.1021%2Facs.jmedchem.5b00611"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">First crystal structures of Mycobacterium tuberculosis 6-oxopurine phosphoribosyltransferase: Complexes with GMP and pyrophosphate and with acyclic nucleoside phosphonates whose prodrugs have antituberculosis activity</span></div><div class="casAuthors">Eng, Wai Soon; Hockova, Dana; Spacek, Petr; Janeba, Zlatko; West, Nicholas P.; Woods, Kyra; Naesens, Lieve M. J.; Keough, Dianne T.; Guddat, Luke W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4822-4838</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human tuberculosis is a chronic infectious disease affecting millions of lives.  Because of emerging resistance to current medications, new therapeutic drugs are needed.  One potential new target is hypoxanthine-guanine phosphoribosyltransferase (MtHGPRT), a key enzyme of the purine salvage pathway.  Here, newly synthesized acyclic nucleoside phosphonates (ANPs) have been shown to be competitive inhibitors of MtHGPRT with Ki values as low as 0.69 μM.  Prodrugs of these compds. arrest the growth of a virulent strain of M. tuberculosis with MIC50 values as low as 4.5 μM and possess low cytotoxicity in mammalian cells (CC50 values as high as >300 μM).  In addn., the first crystal structures of MtHGPRT (2.03-2.76 Å resoln.) have been detd., three of these in complex with novel ANPs and one with GMP and pyrophosphate.  These data provide a solid foundation for the further development of ANPs as selective inhibitors of MtHGPRT and as antituberculosis agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-Nicla0R_-7Vg90H21EOLACvtfcHk0lgaQ81a0s_OVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXntFGitL0%253D&md5=c36282ce6728032802af6ef43e0ecc3b</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00611%26sid%3Dliteratum%253Aachs%26aulast%3DEng%26aufirst%3DW.%2BS.%26aulast%3DHockov%25C3%25A1%26aufirst%3DD.%26aulast%3D%25C5%25A0pa%25C4%258Dek%26aufirst%3DP.%26aulast%3DJaneba%26aufirst%3DZ.%26aulast%3DWest%26aufirst%3DN.%2BP.%26aulast%3DWoods%26aufirst%3DK.%26aulast%3DNaesens%26aufirst%3DL.%2BM.%2BJ.%26aulast%3DKeough%26aufirst%3DD.%2BT.%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26atitle%3DFirst%2520crystal%2520structures%2520of%2520Mycobacterium%2520tuberculosis%25206-oxopurine%2520phosphoribosyltransferase%253A%2520Complexes%2520with%2520GMP%2520and%2520pyrophosphate%2520and%2520with%2520acyclic%2520nucleoside%2520phosphonates%2520whose%2520prodrugs%2520have%2520antituberculosis%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D4822%26epage%3D4838%26doi%3D10.1021%2Facs.jmedchem.5b00611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Špaček, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keough, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chavchich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dračínský, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janeba, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naesens, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edstein, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocková, D.</span></span> <span> </span><span class="NLM_article-title">Synthesis and evaluation of asymmetric acyclic nucleoside bisphosphonates as inhibitors of <i>Plasmodium falciparum</i> and human hypoxanthine-guanine-(xanthine) phosphoribosyltransferase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">7539</span>– <span class="NLM_lpage">7554</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00926</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00926" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlCgu7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=7539-7554&author=P.+%C5%A0pa%C4%8Dekauthor=D.+T.+Keoughauthor=M.+Chavchichauthor=M.+Dra%C4%8D%C3%ADnsk%C3%BDauthor=Z.+Janebaauthor=L.+Naesensauthor=M.+D.+Edsteinauthor=L.+W.+Guddatauthor=D.+Hockov%C3%A1&title=Synthesis+and+evaluation+of+asymmetric+acyclic+nucleoside+bisphosphonates+as+inhibitors+of+Plasmodium+falciparum+and+human+hypoxanthine-guanine-%28xanthine%29+phosphoribosyltransferase&doi=10.1021%2Facs.jmedchem.7b00926"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Evaluation of Asymmetric Acyclic Nucleoside Bisphosphonates as Inhibitors of Plasmodium falciparum and Human Hypoxanthine-Guanine-(Xanthine) Phosphoribosyltransferase</span></div><div class="casAuthors">Spacek, Petr; Keough, Dianne T.; Chavchich, Marina; Dracinsky, Martin; Janeba, Zlatko; Naesens, Lieve; Edstein, Michael D.; Guddat, Luke W.; Hockova, Dana</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7539-7554</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Acyclic nucleoside bisphosphonates (ANbPs) have previously been shown to be good inhibitors of human hypoxanthine-guanine phosphoribosyltransferase (HGPRT) and Plasmodium falciparum (Pf) hypoxanthine-guanine-xanthine phosphoribosyltransferase (PfHGXPRT).  On the basis of this scaffold, a new series of ANbPs was synthesized.  One of these new ANbPs, [3-(guanine-9-yl)-2-((2-phosphonoethoxy)methyl)propoxy]methylphosphonic acid, exhibited Ki values of 6 and 70 nM for human HGPRT and Pf HGXPRT, resp.  These low Ki values were achieved by inserting an extra carbon atom in the linker connecting the N9 atom of guanine to one of the phosphonate groups.  The crystal structure of this ANbP in complex with human HGPRT was detd. at 2.0 Å resoln. and shows that it fills three key pockets in the active site.  The most potent phosphoramidate prodrugs of these compds. have IC50 values in the low micromolar range in Pf lines and low toxicity in human A549 cells, demonstrating that these ANbPs are excellent antimalarial drug leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnSzTuy4-WwrVg90H21EOLACvtfcHk0lgaQ81a0s_OVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlCgu7rP&md5=9c47d1dfc40b240ad2c152b3cb542eec</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00926%26sid%3Dliteratum%253Aachs%26aulast%3D%25C5%25A0pa%25C4%258Dek%26aufirst%3DP.%26aulast%3DKeough%26aufirst%3DD.%2BT.%26aulast%3DChavchich%26aufirst%3DM.%26aulast%3DDra%25C4%258D%25C3%25ADnsk%25C3%25BD%26aufirst%3DM.%26aulast%3DJaneba%26aufirst%3DZ.%26aulast%3DNaesens%26aufirst%3DL.%26aulast%3DEdstein%26aufirst%3DM.%2BD.%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26aulast%3DHockov%25C3%25A1%26aufirst%3DD.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520asymmetric%2520acyclic%2520nucleoside%2520bisphosphonates%2520as%2520inhibitors%2520of%2520Plasmodium%2520falciparum%2520and%2520human%2520hypoxanthine-guanine-%2528xanthine%2529%2520phosphoribosyltransferase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D7539%26epage%3D7554%26doi%3D10.1021%2Facs.jmedchem.7b00926" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Špaček, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keough, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chavchich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dračínský, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janeba, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naesens, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edstein, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocková, D.</span></span> <span> </span><span class="NLM_article-title">Synthesis and evaluation of symmetric acyclic nucleoside bisphosphonates as inhibitors of the <i>Plasmodium falciparum, Plasmodium vivax</i> and human 6-oxopurine phosphoribosyltransferases and the antimalarial activity of their prodrugs</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">4008</span>– <span class="NLM_lpage">4030</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2017.05.048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1016%2Fj.bmc.2017.05.048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=28601510" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpslGnur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=4008-4030&author=P.+%C5%A0pa%C4%8Dekauthor=D.+T.+Keoughauthor=M.+Chavchichauthor=M.+Dra%C4%8D%C3%ADnsk%C3%BDauthor=Z.+Janebaauthor=L.+Naesensauthor=M.+D.+Edsteinauthor=L.+W.+Guddatauthor=D.+Hockov%C3%A1&title=Synthesis+and+evaluation+of+symmetric+acyclic+nucleoside+bisphosphonates+as+inhibitors+of+the+Plasmodium+falciparum%2C+Plasmodium+vivax+and+human+6-oxopurine+phosphoribosyltransferases+and+the+antimalarial+activity+of+their+prodrugs&doi=10.1016%2Fj.bmc.2017.05.048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and evaluation of symmetric acyclic nucleoside bisphosphonates as inhibitors of the Plasmodium falciparum, Plasmodium vivax and human 6-oxopurine phosphoribosyltransferases and the antimalarial activity of their prodrugs</span></div><div class="casAuthors">Spacek, Petr; Keough, Dianne T.; Chavchich, Marina; Dracinsky, Martin; Janeba, Zlatko; Naesens, Lieve; Edstein, Michael D.; Guddat, Luke W.; Hockova, Dana</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4008-4030</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Two new series of sym. acyclic nucleoside bisphosphonates (ANbPs) have been synthesized as potential inhibitors of the Plasmodium falciparum (Pf) and vivax (Pv) 6-oxopurine phosphoribosyltransferases.  The structural variability between these sym. ANbPs lies in the no. of atoms in the two acyclic linkers connecting the N9 atom of the purine base to each of two phosphonate groups and the branching point of the acyclic moiety relative to the purine base, which occurs at either the alpha or beta positions.  Within each series, six different 6-oxopurine bases have been attached.  In general, the ANbPs with either guanine or hypoxanthine have lower Ki values than for those contg. either the 8-bromo or 7-deaza 6-oxopurine bases.  The lowest Ki values obtained for the two parasite enzymes were 0.1 μM (Pf) and 0.2 μM (Pv) for this series of compds.  Two phosphoramidate prodrugs of these inhibitors exhibited antimalarial activity against Pf in infected erythrocyte cell culture with IC50 values of 0.8 and 1.5 μM.  These two compds. exhibited low cytotoxicity in human A549 cells having CC50 values of >300 μM resulting in an excellent selectivity index.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRylWYh-MpNLVg90H21EOLACvtfcHk0ljYRCcQnTxrEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpslGnur8%253D&md5=62891e50ca20c740844a44c1a7e8876b</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.05.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.05.048%26sid%3Dliteratum%253Aachs%26aulast%3D%25C5%25A0pa%25C4%258Dek%26aufirst%3DP.%26aulast%3DKeough%26aufirst%3DD.%2BT.%26aulast%3DChavchich%26aufirst%3DM.%26aulast%3DDra%25C4%258D%25C3%25ADnsk%25C3%25BD%26aufirst%3DM.%26aulast%3DJaneba%26aufirst%3DZ.%26aulast%3DNaesens%26aufirst%3DL.%26aulast%3DEdstein%26aufirst%3DM.%2BD.%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26aulast%3DHockov%25C3%25A1%26aufirst%3DD.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520symmetric%2520acyclic%2520nucleoside%2520bisphosphonates%2520as%2520inhibitors%2520of%2520the%2520Plasmodium%2520falciparum%252C%2520Plasmodium%2520vivax%2520and%2520human%25206-oxopurine%2520phosphoribosyltransferases%2520and%2520the%2520antimalarial%2520activity%2520of%2520their%2520prodrugs%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D4008%26epage%3D4030%26doi%3D10.1016%2Fj.bmc.2017.05.048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keough, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocková, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rejman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Špaček, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vrbková, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krečmerová, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eng, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jans, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naesens, L. M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de
Jersey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span> <span> </span><span class="NLM_article-title">Inhibition of the <i>Escherichia coli</i> 6-oxopurine phosphoribosyltransferases by nucleoside phosphonates: potential for new antibacterial agents</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">6967</span>– <span class="NLM_lpage">6984</span>, <span class="refDoi"> DOI: 10.1021/jm400779n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400779n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1ChsL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=6967-6984&author=D.+T.+Keoughauthor=D.+Hockov%C3%A1author=D.+Rejmanauthor=P.+%C5%A0pa%C4%8Dekauthor=S.+Vrbkov%C3%A1author=M.+Kre%C4%8Dmerov%C3%A1author=W.+S.+Engauthor=H.+Jansauthor=N.+P.+Westauthor=L.+M.+J.+Naesensauthor=J.+de%0AJerseyauthor=L.+W.+Guddat&title=Inhibition+of+the+Escherichia+coli+6-oxopurine+phosphoribosyltransferases+by+nucleoside+phosphonates%3A+potential+for+new+antibacterial+agents&doi=10.1021%2Fjm400779n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of the Escherichia coli 6-Oxopurine Phosphoribosyltransferases by Nucleoside Phosphonates: Potential for New Antibacterial Agents</span></div><div class="casAuthors">Keough, Dianne T.; Hockova, Dana; Rejman, Dominik; Spacek, Petr; Vrbkova, Silvie; Krecmerova, Marcela; Eng, Wai Soon; Jans, Harmen; West, Nicholas P.; Naesens, Lieve M. J.; de Jersey, John; Guddat, Luke W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6967-6984</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Escherichia coli (Ec) cells possess two purine salvage enzymes: xanthine-guanine phosphoribosyltransferase (XGPRT) and hypoxanthine phosphoribosyltransferase (HPRT).  EcXGPRT shares a common structural feature with other members of this family, a flexible loop that closes over the active site during catalysis.  The replacement of six of these amino acids by alanine has no effect on the Km for the two substrates.  However, the Ki for the nucleoside monophosphate increases by 27-fold, and the kcat is reduced by ∼200-fold.  Nucleoside phosphonates (NP) are good inhibitors of EcXGPRT and EcHPRT, with Ki values as low as 10 nM.  In the absence of the flexible loop, these values increase by 5- to 30-fold, indicating the importance of the loop for high-affinity inhibition.  Crystal structures of two NPs in complex with EcXGPRT explain the tight binding.  Prodrugs of NPs with low Ki values for EcXGPRT or EcHPRT exhibit IC50 values between 5 and 23 μM against Mycobacterium tuberculosis in cell-based assays, suggesting that these compds. are therapeutic leads against pathogenic bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjJGY9KeZWpbVg90H21EOLACvtfcHk0ljYRCcQnTxrEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1ChsL7M&md5=bc8016299374898032e7311d054ec883</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm400779n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400779n%26sid%3Dliteratum%253Aachs%26aulast%3DKeough%26aufirst%3DD.%2BT.%26aulast%3DHockov%25C3%25A1%26aufirst%3DD.%26aulast%3DRejman%26aufirst%3DD.%26aulast%3D%25C5%25A0pa%25C4%258Dek%26aufirst%3DP.%26aulast%3DVrbkov%25C3%25A1%26aufirst%3DS.%26aulast%3DKre%25C4%258Dmerov%25C3%25A1%26aufirst%3DM.%26aulast%3DEng%26aufirst%3DW.%2BS.%26aulast%3DJans%26aufirst%3DH.%26aulast%3DWest%26aufirst%3DN.%2BP.%26aulast%3DNaesens%26aufirst%3DL.%2BM.%2BJ.%26aulast%3Dde%2BJersey%26aufirst%3DJ.%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26atitle%3DInhibition%2520of%2520the%2520Escherichia%2520coli%25206-oxopurine%2520phosphoribosyltransferases%2520by%2520nucleoside%2520phosphonates%253A%2520potential%2520for%2520new%2520antibacterial%2520agents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D6967%26epage%3D6984%26doi%3D10.1021%2Fjm400779n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Clercq, E.</span></span> <span> </span><span class="NLM_article-title">The acyclic nucleoside phosphonates (ANPs): Antonin Holy’s legacy</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">1278</span>– <span class="NLM_lpage">1303</span>, <span class="refDoi"> DOI: 10.1002/med.21283</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1002%2Fmed.21283" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=23568857" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsF2hu7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2013&pages=1278-1303&author=E.+De%0AClercq&title=The+acyclic+nucleoside+phosphonates+%28ANPs%29%3A+Antonin+Holy%E2%80%99s+legacy&doi=10.1002%2Fmed.21283"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">The Acyclic Nucleoside Phosphonates (ANPs): Antonin Holy's Legacy</span></div><div class="casAuthors">De Clercq, Erik</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1278-1303</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  The name of Antonin Holy has become synonymous for the era of acyclic nucleoside phosphonates (ANPs), which started with (S)-HPMPA as the prototype and (S)-HPMPC (cidofovir) as the first marketed compd.  It has now evolved to a no. of compds. clin. used in the treatment of HIV and hepatitis B virus infections, either as such [tenofovir disoproxil fumarate (TDF, Viread)] or in combination [Truvada, Atripla, Complera, Stribild].  Truvada has also been approved for the prevention of HIV infections.  Forthcoming is a new formulation of tenofovir (TAF: tenofovir alafenamide).  Also forthcoming are several "quad" drug combinations contg. either TDF or TAF.  Other ANPs, based on either an alkoxy side chain or 5-azacytosine heterocycle seem highly promising and worth further pursuing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWroqgi65J4bVg90H21EOLACvtfcHk0ljYRCcQnTxrEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsF2hu7jF&md5=f921578d8a2b5eaa975704b1225dfa8e</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fmed.21283&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21283%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BClercq%26aufirst%3DE.%26atitle%3DThe%2520acyclic%2520nucleoside%2520phosphonates%2520%2528ANPs%2529%253A%2520Antonin%2520Holy%25E2%2580%2599s%2520legacy%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2013%26volume%3D33%26spage%3D1278%26epage%3D1303%26doi%3D10.1002%2Fmed.21283" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Clercq, E.</span></span> <span> </span><span class="NLM_article-title">The Holy Trinity: the acyclic nucleoside phosphonates</span>. <i>Adv. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">316</span>, <span class="refDoi"> DOI: 10.1016/b978-0-12-405880-4.00008-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1016%2FB978-0-12-405880-4.00008-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=23886004" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Kku7jN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2013&pages=293-316&author=E.+De%0AClercq&title=The+Holy+Trinity%3A+the+acyclic+nucleoside+phosphonates&doi=10.1016%2Fb978-0-12-405880-4.00008-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">The Holy Trinity: the acyclic nucleoside phosphonates</span></div><div class="casAuthors">De Clercq, Erik</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Pharmacology (San Diego, CA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">Antiviral Agents</span>),
    <span class="NLM_cas:pages">293-316</span>CODEN:
                <span class="NLM_cas:coden">ADPHEL</span>;
        ISSN:<span class="NLM_cas:issn">1054-3589</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  The Holy Trinity was named after Dr Antonin Holy (the Holy Trinity being an Unesco recognized monument in Olomouc, Czech Republic), who together with Dr John C. Martin (Gilead Sciences) and myself pioneered a new class of antiviral agents, the acyclic nucleoside phosphonates.  These compds. have revolutionized the antiviral drug field with several drugs that have been approved for the treatment of various DNA virus infections (cidofovir), hepatitis B (adefovir), and AIDS (HIV infection; tenofovir).  The latter is also available as its oral prodrug, tenofovir disoproxil fumarate, for the treatment of hepatitis B and in combination with emtricitabine ((-)FTC) for the treatment and prophylaxis of HIV infections and in combination with (-)FTC and other HIV inhibitors, i.e., efavirenz, rilpivirine, or elvitegravir (and a pharmacoenhancer thereof, cobicistat), for the treatment of AIDS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBSAiIxqMFs7Vg90H21EOLACvtfcHk0lhInrl1_1Fm8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Kku7jN&md5=563984dfb1d047b65f14d3f7b61d48db</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-405880-4.00008-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FB978-0-12-405880-4.00008-1%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BClercq%26aufirst%3DE.%26atitle%3DThe%2520Holy%2520Trinity%253A%2520the%2520acyclic%2520nucleoside%2520phosphonates%26jtitle%3DAdv.%2520Pharmacol.%26date%3D2013%26volume%3D67%26spage%3D293%26epage%3D316%26doi%3D10.1016%2Fb978-0-12-405880-4.00008-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Slusarczyk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serpi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pertusati, F.</span></span> <span> </span><span class="NLM_article-title">Phosphoramidates and phosphonamidates (ProTides) with antiviral activity</span>. <i>Antiviral Chem. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">204020661877524</span>, <span class="refDoi"> DOI: 10.1177/2040206618775243</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1177%2F2040206618775243" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2018&pages=204020661877524&author=M.+Slusarczykauthor=M.+Serpiauthor=F.+Pertusati&title=Phosphoramidates+and+phosphonamidates+%28ProTides%29+with+antiviral+activity&doi=10.1177%2F2040206618775243"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1177%2F2040206618775243&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2040206618775243%26sid%3Dliteratum%253Aachs%26aulast%3DSlusarczyk%26aufirst%3DM.%26aulast%3DSerpi%26aufirst%3DM.%26aulast%3DPertusati%26aufirst%3DF.%26atitle%3DPhosphoramidates%2520and%2520phosphonamidates%2520%2528ProTides%2529%2520with%2520antiviral%2520activity%26jtitle%3DAntiviral%2520Chem.%2520Chemother.%26date%3D2018%26volume%3D26%26spage%3D204020661877524%26doi%3D10.1177%2F2040206618775243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mehellou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rattan, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balzarini, J.</span></span> <span> </span><span class="NLM_article-title">The ProTide Prodrug Technology: From the Concept to the Clinic</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">2211</span>– <span class="NLM_lpage">2226</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00734</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00734" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlSitL3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=2211-2226&author=Y.+Mehellouauthor=H.+S.+Rattanauthor=J.+Balzarini&title=The+ProTide+Prodrug+Technology%3A+From+the+Concept+to+the+Clinic&doi=10.1021%2Facs.jmedchem.7b00734"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">The ProTide Prodrug Technology: From the Concept to the Clinic</span></div><div class="casAuthors">Mehellou, Youcef; Rattan, Hardeep S.; Balzarini, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2211-2226</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The ProTide technol. is a prodrug approach developed for the efficient intracellular delivery of nucleoside analog monophosphates and monophosphonates.  In this approach, the hydroxyls of the monophosphate or monophosphonate groups are masked by an arom. group and an amino acid ester moiety, which are enzymically cleaved-off inside cells to release the free nucleoside monophosphate and monophosphonate species.  Structurally, this represents the current end-point of an extensive medicinal chem. endeavor that spans almost three decades.  It started from the masking of nucleoside monophosphate and monophosphonate groups by simple alkyl groups and evolved into the sophisticated ProTide system as known today.  This technol. has been extensively employed in drug discovery, and it has already led to the discovery of two FDA-approved (antiviral) ProTides.  In this work, we will review the development of the ProTide technol., its application in drug discovery, and its role in the improvement of drug delivery and efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeaJFaQn4HwrVg90H21EOLACvtfcHk0lhInrl1_1Fm8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlSitL3P&md5=ab9f5805c3c9d2d80fbdd82a3ad86c73</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00734&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00734%26sid%3Dliteratum%253Aachs%26aulast%3DMehellou%26aufirst%3DY.%26aulast%3DRattan%26aufirst%3DH.%2BS.%26aulast%3DBalzarini%26aufirst%3DJ.%26atitle%3DThe%2520ProTide%2520Prodrug%2520Technology%253A%2520From%2520the%2520Concept%2520to%2520the%2520Clinic%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D2211%26epage%3D2226%26doi%3D10.1021%2Facs.jmedchem.7b00734" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keough, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skinner-Adams, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, A.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brereton, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jersey, J.</span></span> <span> </span><span class="NLM_article-title">Lead compounds for antimalarial chemotherapy: Purine base analogs discriminate between human and <i>P. falciparum</i> 6-oxopurine phosphoribosyltransferases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">7479</span>– <span class="NLM_lpage">7486</span>, <span class="refDoi"> DOI: 10.1021/jm061012j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm061012j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1WhsLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=7479-7486&author=D.+T.+Keoughauthor=T.+Skinner-Adamsauthor=M.+K.+Jonesauthor=A.-L.+Ngauthor=I.+M.+Breretonauthor=L.+W.+Guddatauthor=J.+de+Jersey&title=Lead+compounds+for+antimalarial+chemotherapy%3A+Purine+base+analogs+discriminate+between+human+and+P.+falciparum+6-oxopurine+phosphoribosyltransferases&doi=10.1021%2Fjm061012j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Lead cmpounds for antimalarial chemotherapy: purine base analogs discriminate between human and P. falciparum 6-oxopurine Phosphoribosyltransferases</span></div><div class="casAuthors">Keough, Dianne T.; Skinner-Adams, Tina; Jones, Malcolm K.; Ng, Ai-Lin; Brereton, Ian M.; Guddat, Luke W.; de Jersey, John</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">7479-7486</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The malarial parasite Plasmodium falciparum depends on the purine salvage enzyme hypoxanthine-guanine-xanthine phosphoribosyltransferase (HGXPRT) to convert purine bases from the host to nucleotides needed for DNA and RNA synthesis.  An approach to developing antimalarial drugs is to use HGXPRT to convert introduced purine base analogs to nucleotides that are toxic to the parasite.  This strategy requires that these compds. be good substrates for the parasite enzyme but poor substrates for the human counterpart, HGPRT.  Bases with a chlorine atom in the 6-position or a nitrogen in the 8-position exhibited strong discrimination between P. falciparum HGXPRT and human HGPRT.  The kcat/Km values for the Plasmodium enzyme using 6-chloroguanine and 8-azaguanine as substrates were 50-80-fold and 336-fold higher than for the human enzyme, resp.  These and other bases were effective in inhibiting the growth of the parasite in vitro, giving IC50 values as low as 1 μM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5wP9-53NDS7Vg90H21EOLACvtfcHk0lhInrl1_1Fm8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1WhsLvM&md5=525b0f86fa5d8d560945cddfcdf872c0</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm061012j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm061012j%26sid%3Dliteratum%253Aachs%26aulast%3DKeough%26aufirst%3DD.%2BT.%26aulast%3DSkinner-Adams%26aufirst%3DT.%26aulast%3DJones%26aufirst%3DM.%2BK.%26aulast%3DNg%26aufirst%3DA.-L.%26aulast%3DBrereton%26aufirst%3DI.%2BM.%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26aulast%3Dde%2BJersey%26aufirst%3DJ.%26atitle%3DLead%2520compounds%2520for%2520antimalarial%2520chemotherapy%253A%2520Purine%2520base%2520analogs%2520discriminate%2520between%2520human%2520and%2520P.%2520falciparum%25206-oxopurine%2520phosphoribosyltransferases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D7479%26epage%3D7486%26doi%3D10.1021%2Fjm061012j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jersey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, J. L.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of <i>Escherichia coli</i> xanthine phosphoribosyltransferase</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">4125</span>– <span class="NLM_lpage">4134</span>, <span class="refDoi"> DOI: 10.1021/bi962640d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi962640d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADyaK2sXisVeqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1997&pages=4125-4134&author=S.+Vosauthor=J.+de+Jerseyauthor=J.+L.+Martin&title=Crystal+structure+of+Escherichia+coli+xanthine+phosphoribosyltransferase&doi=10.1021%2Fbi962640d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure of Escherichia coli Xanthine Phosphoribosyltransferase</span></div><div class="casAuthors">Vos, Siska; de Jersey, John; Martin, Jennifer L.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4125-4134</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Xanthine phosphoribosyltransferase (XPRT; EC 2.4.2.22) from Escherichia coli is a tetrameric enzyme having 152 residues per subunit.  XPRT catalyzes the transfer of the phosphoribosyl group from 5-phospho-α-D-ribosyl 1-pyrophosphate (PRib-PP) to the 6-oxopurine bases guanine, xanthine, and hypoxanthine to form GMP, XMP, and IMP, resp.  Crystals grown in the absence of substrate or product were used to det. the structure of XPRT at a resoln. of 1.8 Å, by multiple isomorphous replacement.  The core structure of XPRT includes a five-stranded parallel β-sheet surrounded by three α-helixes, which is similar to that obsd. in other known phosphoribosyltransferase (PRTase) structures.  The XPRT structure also has several interesting features.  A glutamine residue in the purine binding site may be responsible for the altered 6-oxopurine base specificity seen in this enzyme compared to other 6-oxopurine PRTases.  Also, the authors observe both a magnesium ion and a sulfate ion bound at the PRib-PP binding site of XPRT.  The sulfate ion interacts with Arg-37 which has a cis-peptide conformation, and the magnesium ion interacts with Asp-89, a highly conserved acidic residue in the PRib-PP binding site motif.  The XPRT structure also incorporates a feature which has not been obsd. in other PRTase structures.  The C-terminal 12 residues of XPRT adopt an unusual extended conformation and make interactions with a neighboring subunit.  The very last residue, Arg-152, could form part of the active site of a symmetry-related subunit in the XPRT tetramer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojPPh9OZQ1RbVg90H21EOLACvtfcHk0ljNd2pHOkqsJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXisVeqtbk%253D&md5=b8da0a6f29db2721d2a01d1fa5f154d5</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fbi962640d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi962640d%26sid%3Dliteratum%253Aachs%26aulast%3DVos%26aufirst%3DS.%26aulast%3Dde%2BJersey%26aufirst%3DJ.%26aulast%3DMartin%26aufirst%3DJ.%2BL.%26atitle%3DCrystal%2520structure%2520of%2520Escherichia%2520coli%2520xanthine%2520phosphoribosyltransferase%26jtitle%3DBiochemistry%26date%3D1997%26volume%3D36%26spage%3D4125%26epage%3D4134%26doi%3D10.1021%2Fbi962640d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keough, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jersey, J.</span></span> <span> </span><span class="NLM_article-title">Crystal structures of free, IMP-, and GMP-bound <i>Escherichia coli</i> hypoxanthine phosphoribosyltransferase</span>. <i>Prot. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1626</span>– <span class="NLM_lpage">1638</span>, <span class="refDoi"> DOI: 10.1110/ps.0201002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1110%2Fps.0201002" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2002&pages=1626-1638&author=L.+W.+Guddatauthor=S.+Vosauthor=J.+L.+Martinauthor=D.+T.+Keoughauthor=J.+de+Jersey&title=Crystal+structures+of+free%2C+IMP-%2C+and+GMP-bound+Escherichia+coli+hypoxanthine+phosphoribosyltransferase&doi=10.1110%2Fps.0201002"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1110%2Fps.0201002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.0201002%26sid%3Dliteratum%253Aachs%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26aulast%3DVos%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DJ.%2BL.%26aulast%3DKeough%26aufirst%3DD.%2BT.%26aulast%3Dde%2BJersey%26aufirst%3DJ.%26atitle%3DCrystal%2520structures%2520of%2520free%252C%2520IMP-%252C%2520and%2520GMP-bound%2520Escherichia%2520coli%2520hypoxanthine%2520phosphoribosyltransferase%26jtitle%3DProt.%2520Sci.%26date%3D2002%26volume%3D11%26spage%3D1626%26epage%3D1638%26doi%3D10.1110%2Fps.0201002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpenter, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamson, D. J.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of action of 6-thioguanine, 6-mercaptopurine, and 8-azaguanine</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">1975</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">2872</span>– <span class="NLM_lpage">2878</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1157053" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADyaE28Xht1entw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=1975&pages=2872-2878&author=J.+A.+Nelsonauthor=J.+W.+Carpenterauthor=L.+M.+Roseauthor=D.+J.+Adamson&title=Mechanisms+of+action+of+6-thioguanine%2C+6-mercaptopurine%2C+and+8-azaguanine"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of action of 6-thioguanine, 6-mercaptopurine, and 8-azaguanine</span></div><div class="casAuthors">Nelson, J. Arly; Carpenter, Jean W.; Rose, Lucy M.; Adamson, Doris J.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">1975</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2872-8</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    </div><div class="casAbstract">The toxic effect of 6-thioguanine (I) [154-42-7] to purine biosynthesis and cell viability in H. Ep. 2 cells was not reversed by aminoimidazolecarboxamide, suggesting that inhibition of purine biosynthesis de novo is not the sole mechanism of toxicity.  Also, 6-(methylmercapto)purine ribonucleoside, a potent inhibitor of purine biosynthesis de novo, produced more marked reductions in cellular pools of purines than did 6-thioguanine without killing cells.  There was no apparent inhibition by 6- thioguanosine 5'-monophosphate of other enzymes leading to the synthesis of guanosine 5'-triphosphate as detd. in whole cells by measurements of radioactive hypoxanthine or guanine incorporation.  Inhibition of DNA synthesis by 1 mM thymidine protected cells from 6-mercaptopurine [50-44-2] or I but failed to protect cells from 8-azaguanine [134-58-7] toxicity.  On the other hand, inhibition of RNA synthesis by 6-azauridine plus deoxycytidine protected cells against 8-azaguanine but did not protect against I or 6-mercaptopurine toxicity.  In agreement with the in vitro data, arabinosylcytosine (a potent inhibitor of DNA synthesis) failed to protect mice against 8-azaguanine.  Thus, the incorporation into DNA (as I nucleotide) appears to be a mechanism of toxicity for these thiopyrines, whereas 8-azaguanine is toxic due to its incorporation into RNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobxwvtEIkV-bVg90H21EOLACvtfcHk0ljNd2pHOkqsJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE28Xht1entw%253D%253D&md5=551badd3cfc883dd9cf47c4cb5a623d5</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNelson%26aufirst%3DJ.%2BA.%26aulast%3DCarpenter%26aufirst%3DJ.%2BW.%26aulast%3DRose%26aufirst%3DL.%2BM.%26aulast%3DAdamson%26aufirst%3DD.%2BJ.%26atitle%3DMechanisms%2520of%2520action%2520of%25206-thioguanine%252C%25206-mercaptopurine%252C%2520and%25208-azaguanine%26jtitle%3DCancer%2520Res.%26date%3D1975%26volume%3D35%26spage%3D2872%26epage%3D2878" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yazawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunicardi, F. C.</span></span> <span> </span><span class="NLM_article-title">Current progress in suicide gene therapy for cancer</span>. <i>World J. Surg.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">783</span>– <span class="NLM_lpage">789</span>, <span class="refDoi"> DOI: 10.1007/s00268-002-4053-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1007%2Fs00268-002-4053-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=11948367" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A280%3ADC%252BD38zltlWnsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2002&pages=783-789&author=K.+Yazawaauthor=W.+E.+Fisherauthor=F.+C.+Brunicardi&title=Current+progress+in+suicide+gene+therapy+for+cancer&doi=10.1007%2Fs00268-002-4053-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Current progress in suicide gene therapy for cancer</span></div><div class="casAuthors">Yazawa Kazuyuki; Fisher William E; Brunicardi F Charles</div><div class="citationInfo"><span class="NLM_cas:title">World journal of surgery</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">783-9</span>
        ISSN:<span class="NLM_cas:issn">0364-2313</span>.
    </div><div class="casAbstract">Standard chemotherapeutic agents and ionizing radiation destroy dividing cells.  Because tumor cells divide more rapidly than normal cells, there is a therapeutic index in which damage to the cancer cells is maximized while keeping the toxicity to the normal host cells acceptable.  Suicide gene therapy strives to deliver genes to the cancer cells, which convert nontoxic prodrugs into active chemotherapeutic agents.  With this strategy, the systemically administered prodrug is converted to the active chemotherapeutic agent only in cancer cells, thereby allowing a maximal therapeutic effect while limiting systemic toxicity.  A literature search was conducted using the MEDLINE database from 1990 to 2001 to identify articles related to suicide gene therapy for cancer.  A number of suicide gene systems have been identified, including the herpes simplex virus thymidine kinase gene, the cytosine deaminase gene, the varicella-zoster virus thymidine kinase gene, the nitroreductase gene, the Escherichia coli gpt gene, and the E. coli Deo gene.  Various vectors, including liposomes, retroviruses, and adenoviruses, have been used to transfer these suicide genes to tumor cells.  These strategies have been effective in cell culture experiments, laboratory animals, and some early clinical trials.  Advances in tissue- and cell-specific delivery of suicide genes using specific promoters will improve the clinical utility of suicide gene therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQXdL1aW36dHZp6t8nS6Nc3fW6udTcc2eYz8FXyTji2D7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD38zltlWnsA%253D%253D&md5=81ed0f9b4b2a3448b22fd17ba1225751</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1007%2Fs00268-002-4053-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00268-002-4053-5%26sid%3Dliteratum%253Aachs%26aulast%3DYazawa%26aufirst%3DK.%26aulast%3DFisher%26aufirst%3DW.%2BE.%26aulast%3DBrunicardi%26aufirst%3DF.%2BC.%26atitle%3DCurrent%2520progress%2520in%2520suicide%2520gene%2520therapy%2520for%2520cancer%26jtitle%3DWorld%2520J.%2520Surg.%26date%3D2002%26volume%3D26%26spage%3D783%26epage%3D789%26doi%3D10.1007%2Fs00268-002-4053-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hocková, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janeba, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naesens, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edstein, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chavchich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keough, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span> <span> </span><span class="NLM_article-title">Antimalarial activity of prodrugs of N-branched acyclic nucleoside phosphonate inhibitors of 6-oxopurine phosphoribosyltransferases</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">5502</span>– <span class="NLM_lpage">5510</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2015.07.038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1016%2Fj.bmc.2015.07.038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=26275679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1CrtLrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=5502-5510&author=D.+Hockov%C3%A1author=Z.+Janebaauthor=L.+Naesensauthor=M.+D.+Edsteinauthor=M.+Chavchichauthor=D.+T.+Keoughauthor=L.+W.+Guddat&title=Antimalarial+activity+of+prodrugs+of+N-branched+acyclic+nucleoside+phosphonate+inhibitors+of+6-oxopurine+phosphoribosyltransferases&doi=10.1016%2Fj.bmc.2015.07.038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Antimalarial activity of prodrugs of N-branched acyclic nucleoside phosphonate inhibitors of 6-oxopurine phosphoribosyltransferases</span></div><div class="casAuthors">Hockova, Dana; Janeba, Zlatko; Naesens, Lieve; Edstein, Michael D.; Chavchich, Marina; Keough, Dianne T.; Guddat, Luke W.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5502-5510</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Acyclic nucleoside phosphonates that contain a 6-oxopurine base are good inhibitors of the human and Plasmodium falciparum 6-oxopurine phosphoribosyltransferases, key enzymes of the purine salvage pathway.  Chem. modifications, based on the crystal structures of several inhibitors in complex with the human PRTase, led to the design of a new class of inhibitors-the aza-ANPs.  Because of the neg. charges of the phosphonic acid moiety, their ability to cross cell membranes is, however, limited.  Thus, phosphoramidate prodrugs of the aza-ANPs were prepd. to improve permeability.  These prodrugs arrest parasitemia with IC50 values in the micromolar range against Plasmodium falciparum-infected erythrocyte cultures (both chloroquine-sensitive and chloroquine-resistant Pf strains).  The prodrugs exhibit low cytotoxicity in several human cell lines.  Thus, they fulfill two essential criteria to qualify them as promising antimalarial drug leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6zbiAVYpUqbVg90H21EOLACvtfcHk0ljNd2pHOkqsJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1CrtLrP&md5=0ec3d8eefea40295963feb4333670b02</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2015.07.038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2015.07.038%26sid%3Dliteratum%253Aachs%26aulast%3DHockov%25C3%25A1%26aufirst%3DD.%26aulast%3DJaneba%26aufirst%3DZ.%26aulast%3DNaesens%26aufirst%3DL.%26aulast%3DEdstein%26aufirst%3DM.%2BD.%26aulast%3DChavchich%26aufirst%3DM.%26aulast%3DKeough%26aufirst%3DD.%2BT.%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26atitle%3DAntimalarial%2520activity%2520of%2520prodrugs%2520of%2520N-branched%2520acyclic%2520nucleoside%2520phosphonate%2520inhibitors%2520of%25206-oxopurine%2520phosphoribosyltransferases%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2015%26volume%3D23%26spage%3D5502%26epage%3D5510%26doi%3D10.1016%2Fj.bmc.2015.07.038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGuigan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madela, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aljarah, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourdin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrica, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolykhalov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bleiman, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganguly, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorovits, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henson, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muhammad, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obikhod, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patti, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vernachio, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramamurty, C. V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battina, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlain, S.</span></span> <span> </span><span class="NLM_article-title">Phosphorodiamidates as a promising new phosphate prodrug motif for antiviral drug discovery: application to anti-HCV agents</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">8632</span>– <span class="NLM_lpage">8645</span>, <span class="refDoi"> DOI: 10.1021/jm2011673</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2011673" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A280%3ADC%252BC38%252FmtlWjsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=8632-8645&author=C.+McGuiganauthor=K.+Madelaauthor=M.+Aljarahauthor=C.+Bourdinauthor=M.+Arricaauthor=E.+Barrettauthor=S.+Jonesauthor=A.+Kolykhalovauthor=B.+Bleimanauthor=K.+D.+Bryantauthor=B.+Gangulyauthor=E.+Gorovitsauthor=G.+Hensonauthor=D.+Hunleyauthor=J.+Hutchinsauthor=J.+Muhammadauthor=A.+Obikhodauthor=J.+Pattiauthor=C.+R.+Waltersauthor=J.+Wangauthor=J.+Vernachioauthor=C.+V.+S.+Ramamurtyauthor=S.+K.+Battinaauthor=S.+Chamberlain&title=Phosphorodiamidates+as+a+promising+new+phosphate+prodrug+motif+for+antiviral+drug+discovery%3A+application+to+anti-HCV+agents&doi=10.1021%2Fjm2011673"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphorodiamidates as a promising new phosphate prodrug motif for antiviral drug discovery: application to anti-HCV agents</span></div><div class="casAuthors">McGuigan Christopher; Madela Karolina; Aljarah Mohamed; Bourdin Claire; Arrica Maria; Barrett Emma; Jones Sarah; Kolykhalov Alexander; Bleiman Blair; Bryant K Dawn; Ganguly Babita; Gorovits Elena; Henson Geoffrey; Hunley Damound; Hutchins Jeff; Muhammad Jerry; Obikhod Aleksandr; Patti Joseph; Walters C Robin; Wang Jin; Vernachio John; Ramamurty Changalvala V S; Battina Srinivas K; Chamberlain Stanley</div><div class="citationInfo"><span class="NLM_cas:title">Journal of medicinal chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8632-45</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">We herein report phosphorodiamidates as a significant new phosphate prodrug motif.  Sixty-seven phosphorodiamidates are reported of two 6-O-alkyl 2'-C-methyl guanosines, with significant variation in the diamidate structure.  Both symmetrical and asymmetric phosphorodiamidates are reported, derived from various esterified amino acids, both d and l, and also from various simple amines.  All of the compounds were evaluated versus hepatitis C virus in replicon assay, and nanomolar activity levels were observed.  Many compounds were noncytotoxic at 100 μM, leading to high antiviral selectivities.  The agents are stable in acidic, neutral, and moderately basic media and in selected biological media but show efficient processing by carboxypeptidases and efficiently yield the free nucleoside monophosphate in cells.  On the basis of in vitro data, eight leads were selected for additional in vivo evaluation, with the intent of selecting one candidate for progression toward clinical studies.  This phosphorodiamidate prodrug method may have broad application outside of HCV and antivirals as it offers many of the advantages of phosphoramidate ProTides but without the chirality issues present in most cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSQ8wY11iyNsRLDWw4HIq2sfW6udTcc2eZ6KNE5mP1aaLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38%252FmtlWjsQ%253D%253D&md5=5b7d43c2582a96a963e3927988054df7</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm2011673&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2011673%26sid%3Dliteratum%253Aachs%26aulast%3DMcGuigan%26aufirst%3DC.%26aulast%3DMadela%26aufirst%3DK.%26aulast%3DAljarah%26aufirst%3DM.%26aulast%3DBourdin%26aufirst%3DC.%26aulast%3DArrica%26aufirst%3DM.%26aulast%3DBarrett%26aufirst%3DE.%26aulast%3DJones%26aufirst%3DS.%26aulast%3DKolykhalov%26aufirst%3DA.%26aulast%3DBleiman%26aufirst%3DB.%26aulast%3DBryant%26aufirst%3DK.%2BD.%26aulast%3DGanguly%26aufirst%3DB.%26aulast%3DGorovits%26aufirst%3DE.%26aulast%3DHenson%26aufirst%3DG.%26aulast%3DHunley%26aufirst%3DD.%26aulast%3DHutchins%26aufirst%3DJ.%26aulast%3DMuhammad%26aufirst%3DJ.%26aulast%3DObikhod%26aufirst%3DA.%26aulast%3DPatti%26aufirst%3DJ.%26aulast%3DWalters%26aufirst%3DC.%2BR.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DVernachio%26aufirst%3DJ.%26aulast%3DRamamurty%26aufirst%3DC.%2BV.%2BS.%26aulast%3DBattina%26aufirst%3DS.%2BK.%26aulast%3DChamberlain%26aufirst%3DS.%26atitle%3DPhosphorodiamidates%2520as%2520a%2520promising%2520new%2520phosphate%2520prodrug%2520motif%2520for%2520antiviral%2520drug%2520discovery%253A%2520application%2520to%2520anti-HCV%2520agents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D8632%26epage%3D8645%26doi%3D10.1021%2Fjm2011673" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keough, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Špaček, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocková, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tichý, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vrbková, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slavětínská, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janeba, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naesens, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edstein, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chavchich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jersey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span> <span> </span><span class="NLM_article-title">Acyclic nucleoside phosphonates containing a second phosphonate group are potent inhibitors of 6-oxopurine phosphoribosyltransferases and have antimalarial activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">2513</span>– <span class="NLM_lpage">2526</span>, <span class="refDoi"> DOI: 10.1021/jm301893b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301893b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlWgsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2513-2526&author=D.+T.+Keoughauthor=P.+%C5%A0pa%C4%8Dekauthor=D.+Hockov%C3%A1author=T.+Tich%C3%BDauthor=S.+Vrbkov%C3%A1author=L.+Slav%C4%9Bt%C3%ADnsk%C3%A1author=Z.+Janebaauthor=L.+Naesensauthor=M.+D.+Edsteinauthor=M.+Chavchichauthor=T.-H.+Wangauthor=J.+de+Jerseyauthor=L.+W.+Guddat&title=Acyclic+nucleoside+phosphonates+containing+a+second+phosphonate+group+are+potent+inhibitors+of+6-oxopurine+phosphoribosyltransferases+and+have+antimalarial+activity&doi=10.1021%2Fjm301893b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Acyclic Nucleoside Phosphonates Containing a Second Phosphonate Group Are Potent Inhibitors of 6-Oxopurine Phosphoribosyltransferases and Have Antimalarial Activity</span></div><div class="casAuthors">Keough, Dianne T.; Spacek, Petr; Hockova, Dana; Tichy, Tomas; Vrbkova, Silvie; Slavetinska, Lenka; Janeba, Zlatko; Naesens, Lieve; Edstein, Michael D.; Chavchich, Marina; Wang, Tzu-Hsuan; de Jersey, John; Guddat, Luke W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2513-2526</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Acyclic nucleoside phosphonates (ANPs) that contain a 6-oxopurine base are good inhibitors of the Plasmodium falciparum (Pf) and Plasmodium vivax (Pv) 6-oxopurine phosphoribosyltransferases (PRTs).  Chem. modifications based on the crystal structure of 2-(phosphonoethoxy)ethylguanine (PEEG) in complex with human HGPRT have led to the design of new ANPs.  These novel compds. contain a second phosphonate group attached to the ANP scaffold.  {[(2-[(Guanine-9H-yl)methyl]propane-1,3-diyl)bis(oxy)]bis(methylene)}diphosphonic acid (I) exhibited a Ki value of 30 nM for human HGPRT and 70 nM for Pf HGXPRT.  The crystal structure of this compd. in complex with human HGPRT shows that it fills or partially fills three crit. locations in the active site: the binding sites of the purine base, the 5'-phosphate group, and pyrophosphate.  This is the first HG(X)PRT inhibitor that has been able to achieve this result.  Prodrugs have been synthesized resulting in IC50 values as low as 3.8 μM for Pf grown in cell culture, up to 25-fold lower compared to the parent compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOIyMCQRsPnrVg90H21EOLACvtfcHk0lijSmRXgI-zeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlWgsrw%253D&md5=f57c4cfd4f9bcb94e6ce93894bfd4d0f</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjm301893b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301893b%26sid%3Dliteratum%253Aachs%26aulast%3DKeough%26aufirst%3DD.%2BT.%26aulast%3D%25C5%25A0pa%25C4%258Dek%26aufirst%3DP.%26aulast%3DHockov%25C3%25A1%26aufirst%3DD.%26aulast%3DTich%25C3%25BD%26aufirst%3DT.%26aulast%3DVrbkov%25C3%25A1%26aufirst%3DS.%26aulast%3DSlav%25C4%259Bt%25C3%25ADnsk%25C3%25A1%26aufirst%3DL.%26aulast%3DJaneba%26aufirst%3DZ.%26aulast%3DNaesens%26aufirst%3DL.%26aulast%3DEdstein%26aufirst%3DM.%2BD.%26aulast%3DChavchich%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DT.-H.%26aulast%3Dde%2BJersey%26aufirst%3DJ.%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26atitle%3DAcyclic%2520nucleoside%2520phosphonates%2520containing%2520a%2520second%2520phosphonate%2520group%2520are%2520potent%2520inhibitors%2520of%25206-oxopurine%2520phosphoribosyltransferases%2520and%2520have%2520antimalarial%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2513%26epage%3D2526%26doi%3D10.1021%2Fjm301893b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Larkin, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackshields, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chenna, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGettigan, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McWilliam, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valentin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallace, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilm, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibson, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higgins, D. G.</span></span> <span> </span><span class="NLM_article-title">Clustal W and Clustal X version 2.0</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">2947</span>– <span class="NLM_lpage">2948</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btm404</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1093%2Fbioinformatics%2Fbtm404" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=17846036" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlaqsL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2007&pages=2947-2948&author=M.+A.+Larkinauthor=G.+Blackshieldsauthor=N.+P.+Brownauthor=R.+Chennaauthor=P.+A.+McGettiganauthor=H.+McWilliamauthor=F.+Valentinauthor=I.+M.+Wallaceauthor=A.+Wilmauthor=R.+Lopezauthor=J.+D.+Thompsonauthor=T.+J.+Gibsonauthor=D.+G.+Higgins&title=Clustal+W+and+Clustal+X+version+2.0&doi=10.1093%2Fbioinformatics%2Fbtm404"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Clustal W and Clustal X version 2.0</span></div><div class="casAuthors">Larkin, M. A.; Blackshields, G.; Brown, N. P.; Chenna, R.; McGettigan, P. A.; McWilliam, H.; Valentin, F.; Wallace, I. M.; Wilm, A.; Lopez, R.; Thompson, J. D.; Gibson, T. J.; Higgins, D. G.</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">2947-2948</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Clustal W and Clustal X multiple sequence alignment programs have been completely rewritten in C++.  This will facilitate the further development of the alignment algorithms in the future and has allowed proper porting of the programs to the latest versions of Linux, Macintosh and Windows operating systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWsv_FPaWsDLVg90H21EOLACvtfcHk0ljovc5sFtZXPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlaqsL%252FM&md5=14c988ecd0b5081ef79149f31a567880</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtm404&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtm404%26sid%3Dliteratum%253Aachs%26aulast%3DLarkin%26aufirst%3DM.%2BA.%26aulast%3DBlackshields%26aufirst%3DG.%26aulast%3DBrown%26aufirst%3DN.%2BP.%26aulast%3DChenna%26aufirst%3DR.%26aulast%3DMcGettigan%26aufirst%3DP.%2BA.%26aulast%3DMcWilliam%26aufirst%3DH.%26aulast%3DValentin%26aufirst%3DF.%26aulast%3DWallace%26aufirst%3DI.%2BM.%26aulast%3DWilm%26aufirst%3DA.%26aulast%3DLopez%26aufirst%3DR.%26aulast%3DThompson%26aufirst%3DJ.%2BD.%26aulast%3DGibson%26aufirst%3DT.%2BJ.%26aulast%3DHiggins%26aufirst%3DD.%2BG.%26atitle%3DClustal%2520W%2520and%2520Clustal%2520X%2520version%25202.0%26jtitle%3DBioinformatics%26date%3D2007%26volume%3D23%26spage%3D2947%26epage%3D2948%26doi%3D10.1093%2Fbioinformatics%2Fbtm404" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leinonen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diez, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binns, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleischmann, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Apweiler, R.</span></span> <span> </span><span class="NLM_article-title">UniProt archive</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">3236</span>– <span class="NLM_lpage">3237</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/bth191</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1093%2Fbioinformatics%2Fbth191" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=15044231" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVSrtbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2004&pages=3236-3237&author=R.+Leinonenauthor=F.+G.+Diezauthor=D.+Binnsauthor=W.+Fleischmannauthor=R.+Lopezauthor=R.+Apweiler&title=UniProt+archive&doi=10.1093%2Fbioinformatics%2Fbth191"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">UniProt archive</span></div><div class="casAuthors">Leinonen, Rasko; Diez, Federico Garcia; Binns, David; Fleischmann, Wolfgang; Lopez, Rodrigo; Apweiler, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3236-3237</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">UniProt Archive (UniParc) is the most comprehensive, non-redundant protein sequence database available.  Its protein sequences are retrieved from predominant, publicly accessible resources.  All new and updated protein sequences are collected and loaded daily into UniParc for full coverage.  To avoid redundancy, each unique sequence is stored only once with a stable protein identifier, which can be used later in UniParc to identify the same protein in all source databases.  When proteins are loaded into the database, database cross-refs. are created to link them to the origins of the sequences.  As a result, performing a sequence search against UniParc is equiv. to performing the same search against all databases cross-referenced by UniParc.  UniParc contains only protein sequences and database cross-refs.; all other information must be retrieved from the source databases.  UniParc is available at http://www.ebi.ac.uk/uniparc/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQIl2XMN-a77Vg90H21EOLACvtfcHk0ljovc5sFtZXPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVSrtbfK&md5=6fa7558121997bac740281fa3c2cf985</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbth191&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbth191%26sid%3Dliteratum%253Aachs%26aulast%3DLeinonen%26aufirst%3DR.%26aulast%3DDiez%26aufirst%3DF.%2BG.%26aulast%3DBinns%26aufirst%3DD.%26aulast%3DFleischmann%26aufirst%3DW.%26aulast%3DLopez%26aufirst%3DR.%26aulast%3DApweiler%26aufirst%3DR.%26atitle%3DUniProt%2520archive%26jtitle%3DBioinformatics%26date%3D2004%26volume%3D20%26spage%3D3236%26epage%3D3237%26doi%3D10.1093%2Fbioinformatics%2Fbth191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holden, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, W. N.</span></span> <span> </span><span class="NLM_article-title">Human hypoxanthine-guanine phosphoribosyltransferase. Evidence for tetrameric structure</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1978</span>,  <span class="NLM_volume"><i>253</i></span>,  <span class="NLM_fpage">4459</span>– <span class="NLM_lpage">4463</span>, <span class="refDoi"> DOI: 10.1016/s0021-9258(17)34742-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1016%2FS0021-9258%2817%2934742-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=659426" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADyaE1cXkvVKntLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=253&publication_year=1978&pages=4459-4463&author=J.+A.+Holdenauthor=W.+N.+Kelley&title=Human+hypoxanthine-guanine+phosphoribosyltransferase.+Evidence+for+tetrameric+structure&doi=10.1016%2Fs0021-9258%2817%2934742-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Human hypoxanthine-guanine phosphoribosyltransferase.  Evidence for tetrameric structure</span></div><div class="casAuthors">Holden, Joseph A.; Kelley, William N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1978</span>),
    <span class="NLM_cas:volume">253</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4459-63</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">Hypoxanthine-guanine phosphoribosyltransferase was purified 23,000-fold from normal human erythrocytes.  The purifn. included affinity chromatog. on a GMP column.  The subunit mol. wt. of the enzyme obtained from this purifn. was 24,000.  The finding of 4 protein species after crosslinkage of the highly purified enzyme with dimethylsuberimidate, dimethyladipimidate, and glutaraldehyde suggests that the enzyme may exist in the native state as a tetramer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2nH7mTT0lVLVg90H21EOLACvtfcHk0ljovc5sFtZXPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE1cXkvVKntLk%253D&md5=7f00fd4a9c39881c8da1e1c62ee85959</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2FS0021-9258%2817%2934742-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0021-9258%252817%252934742-7%26sid%3Dliteratum%253Aachs%26aulast%3DHolden%26aufirst%3DJ.%2BA.%26aulast%3DKelley%26aufirst%3DW.%2BN.%26atitle%3DHuman%2520hypoxanthine-guanine%2520phosphoribosyltransferase.%2520Evidence%2520for%2520tetrameric%2520structure%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1978%26volume%3D253%26spage%3D4459%26epage%3D4463%26doi%3D10.1016%2Fs0021-9258%2817%2934742-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parry, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jersey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, J. L.</span></span> <span> </span><span class="NLM_article-title">Structures of free and complexed forms of <i>Escherichia coli</i> xanthine-guanine phosphoribosyltransferase</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">875</span>– <span class="NLM_lpage">889</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1998.2051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1006%2Fjmbi.1998.2051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=9743633" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADyaK1cXmsFOls7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=1998&pages=875-889&author=S.+Vosauthor=R.+J.+Parryauthor=M.+R.+Burnsauthor=J.+de+Jerseyauthor=J.+L.+Martin&title=Structures+of+free+and+complexed+forms+of+Escherichia+coli+xanthine-guanine+phosphoribosyltransferase&doi=10.1006%2Fjmbi.1998.2051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Structures of free and complexed forms of Escherichia coli xanthine-guanine phosphoribosyltransferase</span></div><div class="casAuthors">Vos, Siska; Parry, Ronald J.; Burns, Mark R.; De Jersey, John; Martin, Jennifer L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">875-889</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">Structures of free, substrate-bound and product-bound forms of Escherichia coli xanthine-guanine phosphoribosyltransferase (XGPRT) have been detd. by X-ray crystallog.  These are compared with the previously detd. structure of magnesium and sulfate-bound XPRT.  The structure of free XGPRT at 2.25 Å resoln. confirms the flexibility of residues in and around a mobile loop identified in other PRTases and shows that the cis-peptide conformation of Arg37 at the active site is maintained in the absence of bound ligands.  The structures of XGPRT complexed with the purine base substrates guanine or xanthine in combination with cPRib-PP, an analog of the second substrate PRib-PP, have been solved to 2.0 Å resoln.  In these two structures the disordered phosphate-binding loop of uncomplexed XGPRT becomes ordered through interactions with the 5'-phosphate group of cPRib-PP.  The cyclopentane ring of cPRib-PP has the C3 exo pucker conformation, stabilized by the cPRib-PP-bound Mg2+.  The purine base specificity of XGPRT appears to be due to water-mediated interactions between the 2-exocyclic groups of guanine or xanthine and side-chains of Glu136 and Asp140, as well as the main-chain oxygen atom of Ile135.  Asp92, together with Lys115, could help stabilize the N7-protonated tautomer of the incoming base and could act as a general base to remove the proton from N7 when the nucleotide product is formed.  The 2.6 Å resoln. structure of XGPRT complexed with product GMP is similar to the substrate-bound complexes.  However, the ribose ring of GMP is rotated by ∼24° compared with the equiv. ring in cPRib-PP.  This rotation results in the loss of all interactions between the ribosyl group and the enzyme in the product complex.  (c) 1998 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoI3pcM6-h6bLVg90H21EOLACvtfcHk0ljovc5sFtZXPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmsFOls7g%253D&md5=243f8db6a5009ff8d10230bc5ff670f6</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1998.2051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1998.2051%26sid%3Dliteratum%253Aachs%26aulast%3DVos%26aufirst%3DS.%26aulast%3DParry%26aufirst%3DR.%2BJ.%26aulast%3DBurns%26aufirst%3DM.%2BR.%26aulast%3Dde%2BJersey%26aufirst%3DJ.%26aulast%3DMartin%26aufirst%3DJ.%2BL.%26atitle%3DStructures%2520of%2520free%2520and%2520complexed%2520forms%2520of%2520Escherichia%2520coli%2520xanthine-guanine%2520phosphoribosyltransferase%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1998%26volume%3D282%26spage%3D875%26epage%3D889%26doi%3D10.1006%2Fjmbi.1998.2051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schramm, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almo, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyler, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furneaux, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grubmeyer, C.</span></span> <span> </span><span class="NLM_article-title">The 2.0 Å structure of human hypoxanthine-guanine phosphoribosyltransferase in complex with a transition-state analog inhibitor</span>. <i>Nat. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">588</span>– <span class="NLM_lpage">593</span>, <span class="refDoi"> DOI: 10.1038/9376</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1038%2F9376" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10360366" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADyaK1MXjs1yktLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1999&pages=588-593&author=V.+L.+Schrammauthor=S.+C.+Almoauthor=W.+Shiauthor=C.+M.+Liauthor=P.+C.+Tylerauthor=R.+H.+Furneauxauthor=C.+Grubmeyer&title=The+2.0+%C3%85+structure+of+human+hypoxanthine-guanine+phosphoribosyltransferase+in+complex+with+a+transition-state+analog+inhibitor&doi=10.1038%2F9376"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">The 2.0 Å structure of human hypoxanthine-guanine phosphoribosyltransferase in complex with a transition-state analog inhibitor</span></div><div class="casAuthors">Shi, Wuxian; Li, Caroline M.; Tyler, Peter C.; Furneaux, Richard H.; Grubmeyer, Charles; Schramm, Vern L.; Almo, Steven C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">588-593</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature America</span>)
        </div><div class="casAbstract">The crystal structure of human hypoxanthine-guanine phosphoribosyltransferase (HGPRT) bound to the transition-state analog, immucillinGP, and Mg2+-pyrophosphate (MgPPi) was detd. to 2.0 Å resoln.  ImmucillinGP was designed as a stable analog with the stereoelectronic features of the transition state.  Bound inhibitor at the catalytic site indicated that the oxocarbenium ion of the transition state was stabilized by neighboring-group participation from MgPPi and 05'.  A short H-bond formed between Asp-137 and the purine ring analog.  Two Mg2+ ions sandwiched PPi and contacted both OH groups of the ribosyl analog.  The transition-state analog was shielded from bulk solvent by a catalytic loop that moved ∼25 Å to cover the active site and became an ordered antiparallel β-sheet.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPL0QhGqlEFLVg90H21EOLACvtfcHk0lhxigMk-cOWIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjs1yktLg%253D&md5=9344053a685003a4720d6be3419f09c5</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2F9376&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F9376%26sid%3Dliteratum%253Aachs%26aulast%3DSchramm%26aufirst%3DV.%2BL.%26aulast%3DAlmo%26aufirst%3DS.%2BC.%26aulast%3DShi%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DC.%2BM.%26aulast%3DTyler%26aufirst%3DP.%2BC.%26aulast%3DFurneaux%26aufirst%3DR.%2BH.%26aulast%3DGrubmeyer%26aufirst%3DC.%26atitle%3DThe%25202.0%2520%25C3%2585%2520structure%2520of%2520human%2520hypoxanthine-guanine%2520phosphoribosyltransferase%2520in%2520complex%2520with%2520a%2520transition-state%2520analog%2520inhibitor%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D1999%26volume%3D6%26spage%3D588%26epage%3D593%26doi%3D10.1038%2F9376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyler, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furneaux, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cahill, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girvin, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grubmeyer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schramm, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almo, S. C.</span></span> <span> </span><span class="NLM_article-title">The 2.0 Å structure of malarial purine phosphoribosyltransferase in complex with a transition-state analogue inhibitor</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">9872</span>– <span class="NLM_lpage">9880</span>, <span class="refDoi"> DOI: 10.1021/bi990664p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi990664p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADyaK1MXktlWjt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=9872-9880&author=W.+Shiauthor=C.+M.+Liauthor=P.+C.+Tylerauthor=R.+H.+Furneauxauthor=S.+M.+Cahillauthor=M.+E.+Girvinauthor=C.+Grubmeyerauthor=V.+L.+Schrammauthor=S.+C.+Almo&title=The+2.0+%C3%85+structure+of+malarial+purine+phosphoribosyltransferase+in+complex+with+a+transition-state+analogue+inhibitor&doi=10.1021%2Fbi990664p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">The 2.0 Å Structure of Malarial Purine Phosphoribosyltransferase in Complex with a Transition-State Analogue Inhibitor</span></div><div class="casAuthors">Shi, Wuxian; Li, Caroline M.; Tyler, Peter C.; Furneaux, Richard H.; Cahill, Sean M.; Girvin, Mark E.; Grubmeyer, Charles; Schramm, Vern L.; Almo, Steven C.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">9872-9880</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Malaria is a leading cause of worldwide mortality from infectious disease.  Plasmodium falciparum proliferation in human erythrocytes requires purine salvage by hypoxanthine-guanine-xanthine phosphoribosyltransferase (HGXPRTase).  The enzyme is a target for the development of novel antimalarials.  Design and synthesis of transition-state analog inhibitors permitted cocrystn. with the malarial enzyme and refinement of the complex to 2.0 Å resoln.  Catalytic site contacts in the malarial enzyme are similar to those of human hypoxanthine-guanine phosphoribosyltransferase (HGPRTase) despite distinct substrate specificity.  The crystal structure of malarial HGXPRTase with bound inhibitor, pyrophosphate, and two Mg2+ ions reveals features unique to the transition-state analog complex.  Substrate-assisted catalysis occurs by ribooxocarbenium stabilization from the O5' lone pair and a pyrophosphate oxygen.  A dissociative reaction coordinate path is implicated in which the primary reaction coordinate motion is the ribosyl C1' in motion between relatively immobile purine base and (Mg)2-pyrophosphate.  Several short hydrogen bonds form in the complex of the enzyme and inhibitor.  The proton NMR spectrum of the transition-state analog complex of malarial HGXPRTase contains two downfield signals at 14.3 and 15.3 ppm.  Despite the structural similarity to the human enzyme, the NMR spectra of the complexes reveal differences in hydrogen bonding between the transition-state analog complexes of the human and malarial HG(X)PRTases.  The X-ray crystal structures and NMR spectra reveal chem. and structural features that suggest a strategy for the design of malaria-specific transition-state inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6InWXezcVG7Vg90H21EOLACvtfcHk0lhxigMk-cOWIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXktlWjt7o%253D&md5=b885550fcda9e1cb936ca983f56d6567</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fbi990664p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi990664p%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DC.%2BM.%26aulast%3DTyler%26aufirst%3DP.%2BC.%26aulast%3DFurneaux%26aufirst%3DR.%2BH.%26aulast%3DCahill%26aufirst%3DS.%2BM.%26aulast%3DGirvin%26aufirst%3DM.%2BE.%26aulast%3DGrubmeyer%26aufirst%3DC.%26aulast%3DSchramm%26aufirst%3DV.%2BL.%26aulast%3DAlmo%26aufirst%3DS.%2BC.%26atitle%3DThe%25202.0%2520%25C3%2585%2520structure%2520of%2520malarial%2520purine%2520phosphoribosyltransferase%2520in%2520complex%2520with%2520a%2520transition-state%2520analogue%2520inhibitor%26jtitle%3DBiochemistry%26date%3D1999%26volume%3D38%26spage%3D9872%26epage%3D9880%26doi%3D10.1021%2Fbi990664p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eads, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scapin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grubmeyer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchettini, J. C.</span></span> <span> </span><span class="NLM_article-title">The crystal structure of human hypoxanthine-guanine phosphoribosyltransferase with bound GMP</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">325</span>– <span class="NLM_lpage">334</span>, <span class="refDoi"> DOI: 10.1016/0092-8674(94)90301-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1016%2F0092-8674%2894%2990301-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=8044844" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADyaK2cXmt1Cru70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=1994&pages=325-334&author=J.+C.+Eadsauthor=G.+Scapinauthor=Y.+Xuauthor=C.+Grubmeyerauthor=J.+C.+Sacchettini&title=The+crystal+structure+of+human+hypoxanthine-guanine+phosphoribosyltransferase+with+bound+GMP&doi=10.1016%2F0092-8674%2894%2990301-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">The crystal structure of human hypoxanthine-guanine phosphoribosyltransferase with bound GMP</span></div><div class="casAuthors">Eads, Janina C.; Scapin, Giovanna; Xu, Yiming; Grubmeyer, Charles; Sacchettini, James C.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">325-34</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    </div><div class="casAbstract">The crystal structure of HGPRTase with bound GMP has been detd. and refined to 2.5 Å resoln.  The enzyme has a core α/β structure resembling the nucleotide-binding fold of dehydrogenases, and a second lobe composed of residues from the amino and carboxy termini.  The GMP mol. binds in an anti conformation in a solvent-exposed cleft of the enzyme.  Lys-165, which forms a hydrogen bond to O6 of GMP, appears to be crit. for detg. the specificity for guanine and hypoxanthine over adenine.  The location of active site residues also provides evidence for a possible mechanism for general base-assisted HGPRTase catalysis.  A rationalization of the effects on stability and activity of naturally occurring single amino acid mutations of HGPRTase is presented, including a discussion of several mutations at the active site that lead to Lesch-Nyhan syndrome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAWyBkNXeCzLVg90H21EOLACvtfcHk0lhxigMk-cOWIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXmt1Cru70%253D&md5=8589aa21a9ebc8daba1f14e10b4b9482</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2894%2990301-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252894%252990301-8%26sid%3Dliteratum%253Aachs%26aulast%3DEads%26aufirst%3DJ.%2BC.%26aulast%3DScapin%26aufirst%3DG.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DGrubmeyer%26aufirst%3DC.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26atitle%3DThe%2520crystal%2520structure%2520of%2520human%2520hypoxanthine-guanine%2520phosphoribosyltransferase%2520with%2520bound%2520GMP%26jtitle%3DCell%26date%3D1994%26volume%3D78%26spage%3D325%26epage%3D334%26doi%3D10.1016%2F0092-8674%2894%2990301-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eng, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocková, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Špaček, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baszczyňski, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janeba, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naesens, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keough, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span> <span> </span><span class="NLM_article-title">Crystal structures of acyclic nucleoside phosphonates in complex with <i>Escherichia coli</i> hypoxanthine phosphoribosyltransferase</span>. <i>ChemistrySelect</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">6267</span>– <span class="NLM_lpage">6276</span>, <span class="refDoi"> DOI: 10.1002/slct.201601679</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1002%2Fslct.201601679" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFymtb%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2016&pages=6267-6276&author=W.+S.+Engauthor=D.+Hockov%C3%A1author=P.+%C5%A0pa%C4%8Dekauthor=O.+Baszczy%C5%88skiauthor=Z.+Janebaauthor=L.+Naesensauthor=D.+T.+Keoughauthor=L.+W.+Guddat&title=Crystal+structures+of+acyclic+nucleoside+phosphonates+in+complex+with+Escherichia+coli+hypoxanthine+phosphoribosyltransferase&doi=10.1002%2Fslct.201601679"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structures of Acyclic Nucleoside Phosphonates in Complex with Escherichia coli Hypoxanthine Phosphoribosyltransferase</span></div><div class="casAuthors">Eng, Wai Soon; Hockova, Dana; Spacek, Petr; Baszczynski, Ondrej; Janeba, Zlatko; Naesens, Lieve; Keough, Dianne T.; Guddat, Luke W.</div><div class="citationInfo"><span class="NLM_cas:title">ChemistrySelect</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6267-6276</span>CODEN:
                <span class="NLM_cas:coden">CHEMUD</span>;
        ISSN:<span class="NLM_cas:issn">2365-6549</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Drug resistant uropathogenic E. coli assocd. with urinary tract infections (UTIs) are a serious and debilitating health threat.  Therefore new drug targets to treat this disease need to be explored.  One possible approach is to block the synthesis of the nucleoside monophosphates required for DNA/RNA prodn. in these pathogens.  In E. coli, the purine salvage pathway has two 6-oxopurine phosphoribosyltransferases (PRTs), xanthine-guanine PRT (EcXGPRT) and hypoxanthine PRT (EcHPRT).  Here, we investigate acyclic nucleoside phosphonates (ANPs) as inhibitors of EcHPRT and have detd. six crystal structures of EcHPRT in complex with ANPs.  These data reveal the binding modes of these compds. and can assist in future rational structure-based design efforts.  It is suggested that a combination of inhibitors that block de novo and salvage pathways is a plausible approach to developing new antibiotics for E. coli UTIs.  In addn., we provide details of a novel approach to accelerating the crystn. of this enzyme that may be of general applicability for rational drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDiYpWpYajtLVg90H21EOLACvtfcHk0lhj6TYmQoPaMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFymtb%252FL&md5=617b757eef6150b74f2839e7b97333fb</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1002%2Fslct.201601679&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fslct.201601679%26sid%3Dliteratum%253Aachs%26aulast%3DEng%26aufirst%3DW.%2BS.%26aulast%3DHockov%25C3%25A1%26aufirst%3DD.%26aulast%3D%25C5%25A0pa%25C4%258Dek%26aufirst%3DP.%26aulast%3DBaszczy%25C5%2588ski%26aufirst%3DO.%26aulast%3DJaneba%26aufirst%3DZ.%26aulast%3DNaesens%26aufirst%3DL.%26aulast%3DKeough%26aufirst%3DD.%2BT.%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26atitle%3DCrystal%2520structures%2520of%2520acyclic%2520nucleoside%2520phosphonates%2520in%2520complex%2520with%2520Escherichia%2520coli%2520hypoxanthine%2520phosphoribosyltransferase%26jtitle%3DChemistrySelect%26date%3D2016%26volume%3D1%26spage%3D6267%26epage%3D6276%26doi%3D10.1002%2Fslct.201601679" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, J. F.</span></span> <span> </span><span class="NLM_article-title">Kinetics of the reversible inhibition of enzyme-catalysed reactions by tight-binding inhibitors</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">1969</span>,  <span class="NLM_volume"><i>185</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">286</span>, <span class="refDoi"> DOI: 10.1016/0005-2744(69)90420-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1016%2F0005-2744%2869%2990420-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=4980133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADyaF1MXkvVOqsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=185&publication_year=1969&pages=269-286&author=J.+F.+Morrison&title=Kinetics+of+the+reversible+inhibition+of+enzyme-catalysed+reactions+by+tight-binding+inhibitors&doi=10.1016%2F0005-2744%2869%2990420-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetics of the reversible inhibition of enzyme-catalyzed reactions by tight-binding inhibitors</span></div><div class="casAuthors">Morrison, John F.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Enzymology</span>
        (<span class="NLM_cas:date">1969</span>),
    <span class="NLM_cas:volume">185</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">269-86</span>CODEN:
                <span class="NLM_cas:coden">BBEZAD</span>;
        ISSN:<span class="NLM_cas:issn">0924-1086</span>.
    </div><div class="casAbstract">A theoretical kinetic study has been undertaken to det. the kinetic characteristics of enzymic reactions that are reversibly inhibited by a substrate analog. at concns. comparable to that of the enzyme.  A general rate equation, that allows for the combination of such a tight-binding inhibitor with one or more enzyme forms of any reaction mechanism, has been derived and shown to be quadratic with sq. and linear terms in υ which represents the true initial steady-state velocity.  From plots of υ against total enzyme concn. at fixed total inhibitor concn., the concn. of enzyme catalytic sites can be detd. irrespective of the nature of the inhibition and the reaction mechanism.  However, plots of υ (or I/υ) against total inhibitor concn. can be used for this purpose only if the dissocn. const. assocd. with the inhibitor is virtually zero.  Particular initial rate equations have been obtained for the cases in which the tight-binding inhibitor gives rise to competitive, noncompetitive, and uncompetitive inhibition with respect to one of the substrates of a bireactant mechanism.  From these equations it has been deduced that, in the presence of inhibitor, plots of I/υ against the reciprocals of the substrate concns. yield families of concave-down, nonrectangular hyperbolas.  The difficulties assocd. with the detn. of inhibition consts. for tight-binding inhibitors are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpj4s_N2iV3CLVg90H21EOLACvtfcHk0lhj6TYmQoPaMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF1MXkvVOqsbo%253D&md5=74d467477385488dfbb053358534ada6</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2F0005-2744%2869%2990420-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0005-2744%252869%252990420-3%26sid%3Dliteratum%253Aachs%26aulast%3DMorrison%26aufirst%3DJ.%2BF.%26atitle%3DKinetics%2520of%2520the%2520reversible%2520inhibition%2520of%2520enzyme-catalysed%2520reactions%2520by%2520tight-binding%2520inhibitors%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D1969%26volume%3D185%26spage%3D269%26epage%3D286%26doi%3D10.1016%2F0005-2744%2869%2990420-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aragão, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aishima, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherukuvada, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarken, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clift, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowieson, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ericsson, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gee, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macedo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mudie, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panjikar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riboldi-Tunnicliffe, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rostan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caradoc-Davies, T. T.</span></span> <span> </span><span class="NLM_article-title">MX2: a high-flux undulator microfocus beamline serving both the chemical and macromolecular crystallography communities at the Australian Synchrotron</span>. <i>J. Synchrotron Radiat.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">885</span>– <span class="NLM_lpage">891</span>, <span class="refDoi"> DOI: 10.1107/s1600577518003120</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1107%2FS1600577518003120" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=29714201" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC1cXovVCnsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=885-891&author=D.+Arag%C3%A3oauthor=J.+Aishimaauthor=H.+Cherukuvadaauthor=R.+Clarkenauthor=M.+Cliftauthor=N.+P.+Cowiesonauthor=D.+J.+Ericssonauthor=C.+L.+Geeauthor=S.+Macedoauthor=N.+Mudieauthor=S.+Panjikarauthor=J.+R.+Priceauthor=A.+Riboldi-Tunnicliffeauthor=R.+Rostanauthor=R.+Williamsonauthor=T.+T.+Caradoc-Davies&title=MX2%3A+a+high-flux+undulator+microfocus+beamline+serving+both+the+chemical+and+macromolecular+crystallography+communities+at+the+Australian+Synchrotron&doi=10.1107%2Fs1600577518003120"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">MX2: a high-flux undulator microfocus beamline serving both the chemical and macromolecular crystallography communities at the Australian Synchrotron</span></div><div class="casAuthors">Aragao, David; Aishima, Jun; Cherukuvada, Hima; Clarken, Robert; Clift, Mark; Cowieson, Nathan Philip; Ericsson, Daniel Jesper; Gee, Christine L.; Macedo, Sofia; Mudie, Nathan; Panjikar, Santosh; Price, Jason Roy; Riboldi-Tunnicliffe, Alan; Rostan, Robert; Williamson, Rachel; Caradoc-Davies, Thomas Tudor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">885-891</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">1600-5775</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MX2 is an in-vacuum undulator-based crystallog. beamline at the 3 GeV Australian Synchrotron.  The beamline delivers hard X-rays in the energy range 4.8-21 keV to a focal spot of 22 × 12 μm FWHM (H × V).  At 13 keV the flux at the sample is 3.4 × 1012 photons s-1.  The beamline endstation allows robotic handling of cryogenic samples via an updated SSRL SAM robot.  This beamline is ideal for weakly diffracting hard-to-crystallize proteins, virus particles, protein assemblies and nucleic acids as well as smaller mols. such as inorg. catalysts and org. drug mols.  The beamline is now mature and has enjoyed a full user program for the last nine years.  This paper describes the beamline status, plans for its future and some recent scientific highlights.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgtd7iUcXqY7Vg90H21EOLACvtfcHk0lhj6TYmQoPaMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXovVCnsr8%253D&md5=8a52a2db9f5d01958df27ae74e2496d3</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1107%2FS1600577518003120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS1600577518003120%26sid%3Dliteratum%253Aachs%26aulast%3DArag%25C3%25A3o%26aufirst%3DD.%26aulast%3DAishima%26aufirst%3DJ.%26aulast%3DCherukuvada%26aufirst%3DH.%26aulast%3DClarken%26aufirst%3DR.%26aulast%3DClift%26aufirst%3DM.%26aulast%3DCowieson%26aufirst%3DN.%2BP.%26aulast%3DEricsson%26aufirst%3DD.%2BJ.%26aulast%3DGee%26aufirst%3DC.%2BL.%26aulast%3DMacedo%26aufirst%3DS.%26aulast%3DMudie%26aufirst%3DN.%26aulast%3DPanjikar%26aufirst%3DS.%26aulast%3DPrice%26aufirst%3DJ.%2BR.%26aulast%3DRiboldi-Tunnicliffe%26aufirst%3DA.%26aulast%3DRostan%26aufirst%3DR.%26aulast%3DWilliamson%26aufirst%3DR.%26aulast%3DCaradoc-Davies%26aufirst%3DT.%2BT.%26atitle%3DMX2%253A%2520a%2520high-flux%2520undulator%2520microfocus%2520beamline%2520serving%2520both%2520the%2520chemical%2520and%2520macromolecular%2520crystallography%2520communities%2520at%2520the%2520Australian%2520Synchrotron%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2018%26volume%3D25%26spage%3D885%26epage%3D891%26doi%3D10.1107%2Fs1600577518003120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/s0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2Fs0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lhsuKMF0HKucw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2Fs0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Panjikar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parthasarathy, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamzin, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, P. A.</span></span> <span> </span><span class="NLM_article-title">Auto-Rickshaw: an automated crystal structure determination platform as an efficient tool for the validation of an X-ray diffraction experiment</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">449</span>– <span class="NLM_lpage">457</span>, <span class="refDoi"> DOI: 10.1107/s0907444905001307</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1107%2FS0907444905001307" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=15805600" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BD2MXivVeisrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2005&pages=449-457&author=S.+Panjikarauthor=V.+Parthasarathyauthor=V.+S.+Lamzinauthor=M.+S.+Weissauthor=P.+A.+Tucker&title=Auto-Rickshaw%3A+an+automated+crystal+structure+determination+platform+as+an+efficient+tool+for+the+validation+of+an+X-ray+diffraction+experiment&doi=10.1107%2Fs0907444905001307"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Auto-Rickshaw: an automated crystal structure determination platform as an efficient tool for the validation of an X-ray diffraction experiment</span></div><div class="casAuthors">Panjikar, Santosh; Parthasarathy, Venkataraman; Lamzin, Victor S.; Weiss, Manfred S.; Tucker, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">D61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">449-457</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The EMBL-Hamburg Automated Crystal Structure Detn. Platform is a system that combines a no. of existing macromol. crystallog. computer programs and several decision-makers into a software pipeline for automated and efficient crystal structure detn.  The pipeline can be invoked as soon as x-ray data from derivatized protein crystals have been collected and processed.  It is controlled by a web-based graphical user interface for data and parameter input, and for monitoring the progress of structure detn.  A large no. of possible structure-soln. paths are encoded in the system and the optimal path is selected by the decision-makers as the structure soln. evolves.  The processes have been optimized for speed so that the pipeline can be used effectively for validating the x-ray expt. at a synchrotron beamline.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZbIxeGX0aqrVg90H21EOLACvtfcHk0lhsuKMF0HKucw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXivVeisrk%253D&md5=98b93bf2e456104466b2e7b07e3ac197</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0907444905001307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444905001307%26sid%3Dliteratum%253Aachs%26aulast%3DPanjikar%26aufirst%3DS.%26aulast%3DParthasarathy%26aufirst%3DV.%26aulast%3DLamzin%26aufirst%3DV.%2BS.%26aulast%3DWeiss%26aufirst%3DM.%2BS.%26aulast%3DTucker%26aufirst%3DP.%2BA.%26atitle%3DAuto-Rickshaw%253A%2520an%2520automated%2520crystal%2520structure%2520determination%2520platform%2520as%2520an%2520efficient%2520tool%2520for%2520the%2520validation%2520of%2520an%2520X-ray%2520diffraction%2520experiment%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2005%26volume%3D61%26spage%3D449%26epage%3D457%26doi%3D10.1107%2Fs0907444905001307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Panjikar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parthasarathy, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamzin, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, P. A.</span></span> <span> </span><span class="NLM_article-title">On the combination of molecular replacement and single-wavelength anomalous diffraction phasing for automated structure determination</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">1089</span>– <span class="NLM_lpage">1097</span>, <span class="refDoi"> DOI: 10.1107/s0907444909029643</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1107%2FS0907444909029643" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=19770506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2009&pages=1089-1097&author=S.+Panjikarauthor=V.+Parthasarathyauthor=V.+S.+Lamzinauthor=M.+S.+Weissauthor=P.+A.+Tucker&title=On+the+combination+of+molecular+replacement+and+single-wavelength+anomalous+diffraction+phasing+for+automated+structure+determination&doi=10.1107%2Fs0907444909029643"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">On the combination of molecular replacement and single-wavelength anomalous diffraction phasing for automated structure determination</span></div><div class="casAuthors">Panjikar, Santosh; Parthasarathy, Venkataraman; Lamzin, Victor S.; Weiss, Manfred S.; Tucker, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1089-1097</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A combination of mol. replacement and single-wavelength anomalous diffraction phasing has been incorporated into the automated structure-detn. platform Auto-Rickshaw.  The complete MRSAD procedure includes mol. replacement, model refinement, exptl. phasing, phase improvement and automated model building.  The improvement over the std. SAD or MR approaches is illustrated by ten test cases taken from the JCSG diffraction data-set database.  Poor MR or SAD phases with phase errors larger than 70° can be improved using the described procedure and a large fraction of the model can be detd. in a purely automatic manner from X-ray data extending to better than 2.6 Å resoln.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoz21EsxIkqJLVg90H21EOLACvtfcHk0lhsuKMF0HKucw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnP&md5=e3629b21fee4dc43450e853417e6b598</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1107%2FS0907444909029643&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909029643%26sid%3Dliteratum%253Aachs%26aulast%3DPanjikar%26aufirst%3DS.%26aulast%3DParthasarathy%26aufirst%3DV.%26aulast%3DLamzin%26aufirst%3DV.%2BS.%26aulast%3DWeiss%26aufirst%3DM.%2BS.%26aulast%3DTucker%26aufirst%3DP.%2BA.%26atitle%3DOn%2520the%2520combination%2520of%2520molecular%2520replacement%2520and%2520single-wavelength%2520anomalous%2520diffraction%2520phasing%2520for%2520automated%2520structure%2520determination%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2009%26volume%3D65%26spage%3D1089%26epage%3D1097%26doi%3D10.1107%2Fs0907444909029643" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebedev, A.</span></span> <span> </span><span class="NLM_article-title">MoRDa, an automatic molecular replacement pipeline</span>. <i>Acta Crystallogr., Sect. A: Found. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">S19</span>, <span class="refDoi"> DOI: 10.1107/s2053273315099672</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1107%2FS2053273315099672" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=25567570" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2015&pages=S19&author=A.+Vaginauthor=A.+Lebedev&title=MoRDa%2C+an+automatic+molecular+replacement+pipeline&doi=10.1107%2Fs2053273315099672"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1107%2FS2053273315099672&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2053273315099672%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DLebedev%26aufirst%3DA.%26atitle%3DMoRDa%252C%2520an%2520automatic%2520molecular%2520replacement%2520pipeline%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520A%253A%2520Found.%2520Adv.%26date%3D2015%26volume%3D71%26spage%3DS19%26doi%3D10.1107%2Fs2053273315099672" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brünger, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clore, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeLano, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gros, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuszewski, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilges, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simonson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, G. L.</span></span> <span> </span><span class="NLM_article-title">Crystallography & NMR System: A New Software Suite for Macromolecular Structure Determination</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">905</span>– <span class="NLM_lpage">921</span>, <span class="refDoi"> DOI: 10.1107/s0907444998003254</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1107%2FS0907444998003254" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=9757107" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A280%3ADyaK1cvjslKhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=1998&pages=905-921&author=A.+T.+Br%C3%BCngerauthor=P.+D.+Adamsauthor=G.+M.+Cloreauthor=W.+L.+DeLanoauthor=P.+Grosauthor=R.+W.+Grosse-Kunstleveauthor=J.+S.+Jiangauthor=J.+Kuszewskiauthor=M.+Nilgesauthor=N.+S.+Pannuauthor=R.+J.+Readauthor=L.+M.+Riceauthor=T.+Simonsonauthor=G.+L.+Warren&title=Crystallography+%26+NMR+System%3A+A+New+Software+Suite+for+Macromolecular+Structure+Determination&doi=10.1107%2Fs0907444998003254"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Crystallography & NMR system: A new software suite for macromolecular structure determination</span></div><div class="casAuthors">Brunger A T; Adams P D; Clore G M; DeLano W L; Gros P; Grosse-Kunstleve R W; Jiang J S; Kuszewski J; Nilges M; Pannu N S; Read R J; Rice L M; Simonson T; Warren G L</div><div class="citationInfo"><span class="NLM_cas:title">Acta crystallographica. Section D, Biological crystallography</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">Pt 5</span>),
    <span class="NLM_cas:pages">905-21</span>
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    </div><div class="casAbstract">A new software suite, called Crystallography & NMR System (CNS), has been developed for macromolecular structure determination by X-ray crystallography or solution nuclear magnetic resonance (NMR) spectroscopy.  In contrast to existing structure-determination programs, the architecture of CNS is highly flexible, allowing for extension to other structure-determination methods, such as electron microscopy and solid-state NMR spectroscopy.  CNS has a hierarchical structure: a high-level hypertext markup language (HTML) user interface, task-oriented user input files, module files, a symbolic structure-determination language (CNS language), and low-level source code.  Each layer is accessible to the user.  The novice user may just use the HTML interface, while the more advanced user may use any of the other layers.  The source code will be distributed, thus source-code modification is possible.  The CNS language is sufficiently powerful and flexible that many new algorithms can be easily implemented in the CNS language without changes to the source code.  The CNS language allows the user to perform operations on data structures, such as structure factors, electron-density maps, and atomic properties.  The power of the CNS language has been demonstrated by the implementation of a comprehensive set of crystallographic procedures for phasing, density modification and refinement.  User-friendly task-oriented input files are available for nearly all aspects of macromolecular structure determination by X-ray crystallography and solution NMR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRAeLCFcGBdlx0LsK7uFuY7fW6udTcc2eYVUzSVMV9R0Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK1cvjslKhsw%253D%253D&md5=37afee1b07ac44d3f3730a39061b4e79</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1107%2FS0907444998003254&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444998003254%26sid%3Dliteratum%253Aachs%26aulast%3DBr%25C3%25BCnger%26aufirst%3DA.%2BT.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DClore%26aufirst%3DG.%2BM.%26aulast%3DDeLano%26aufirst%3DW.%2BL.%26aulast%3DGros%26aufirst%3DP.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DJiang%26aufirst%3DJ.%2BS.%26aulast%3DKuszewski%26aufirst%3DJ.%26aulast%3DNilges%26aufirst%3DM.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRice%26aufirst%3DL.%2BM.%26aulast%3DSimonson%26aufirst%3DT.%26aulast%3DWarren%26aufirst%3DG.%2BL.%26atitle%3DCrystallography%2520%2526%2520NMR%2520System%253A%2520A%2520New%2520Software%2520Suite%2520for%2520Macromolecular%2520Structure%2520Determination%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1998%26volume%3D54%26spage%3D905%26epage%3D921%26doi%3D10.1107%2Fs0907444998003254" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skubák, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebedev, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span> <span> </span><span class="NLM_article-title">REFMAC5 for the refinement of macromolecular crystal structures</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">355</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1107/s0907444911001314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1107%2FS0907444911001314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=21460454" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=355-367&author=G.+N.+Murshudovauthor=P.+Skub%C3%A1kauthor=A.+A.+Lebedevauthor=N.+S.+Pannuauthor=R.+A.+Steinerauthor=R.+A.+Nichollsauthor=M.+D.+Winnauthor=F.+Longauthor=A.+A.+Vagin&title=REFMAC5+for+the+refinement+of+macromolecular+crystal+structures&doi=10.1107%2Fs0907444911001314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">REFMAC5 for the refinement of macromolecular crystal structures</span></div><div class="casAuthors">Murshudov, Garib N.; Skubak, Pavol; Lebedev, Andrey A.; Pannu, Navraj S.; Steiner, Roberto A.; Nicholls, Robert A.; Winn, Martyn D.; Long, Fei; Vagin, Alexei A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">355-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">This paper describes various components of the macromol. crystallog. refinement program REFMAC5, which is distributed as part of the CCP4 suite.  REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS exptl. diffraction data.  To ensure chem. and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization.  Reliable models at resolns. at least as low as 4 Å can be achieved thanks to low-resoln. refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on at. displacement parameters (ADPs) based on the Kullback-Leibler divergence.  REFMAC5 addnl. offers TLS parameterization and, when high-resoln. data are available, fast refinement of anisotropic ADPs.  Refinement in the presence of twinning is performed in a fully automated fashion.  REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resoln. spectrum encountered in macromol. crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5Jq-i2qIq5rVg90H21EOLACvtfcHk0li2e66dsKQTpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D&md5=f8f3202d246908500057ad7c71015b7b</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1107%2FS0907444911001314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911001314%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DSkub%25C3%25A1k%26aufirst%3DP.%26aulast%3DLebedev%26aufirst%3DA.%2BA.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DSteiner%26aufirst%3DR.%2BA.%26aulast%3DNicholls%26aufirst%3DR.%2BA.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DLong%26aufirst%3DF.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26atitle%3DREFMAC5%2520for%2520the%2520refinement%2520of%2520macromolecular%2520crystal%2520structures%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D355%26epage%3D367%26doi%3D10.1107%2Fs0907444911001314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">General quadratic functions in real and reciprocal space and their application to likelihood phasing</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1612</span>– <span class="NLM_lpage">1621</span>, <span class="refDoi"> DOI: 10.1107/s0907444900013263</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1107%2FS0907444900013263" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=11092927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BD3cXosVOksL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2000&pages=1612-1621&author=K.+Cowtan&title=General+quadratic+functions+in+real+and+reciprocal+space+and+their+application+to+likelihood+phasing&doi=10.1107%2Fs0907444900013263"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">General quadratic functions in real and reciprocal space and their application to likelihood phasing</span></div><div class="casAuthors">Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">D56</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1612-1621</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">A general multivariate quadratic function of the structure factors is constructed and transformed to obtain a quadratic function of the continuous electron d.  Two special cases, where structure factors are independent and where electron-d. values are independent, were examd.  These results are related to the new likelihood-based framework of Terwilliger [Terwilliger (1999), Acta Cryst. D55, pp. 1863-1871] for employing structural information which was previously exploited by conventional d.-modification calcns.  The treatment here involves different assumptions and highlights new features of Terwilliger's calcn.  The generalization quadratic construction gave cross terms relating all reflections and electron densities.  Other applications of this approach are considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMQJNDOqn0wrVg90H21EOLACvtfcHk0li2e66dsKQTpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXosVOksL0%253D&md5=fea050d02f0400be7dc294573908ef97</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1107%2FS0907444900013263&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444900013263%26sid%3Dliteratum%253Aachs%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DGeneral%2520quadratic%2520functions%2520in%2520real%2520and%2520reciprocal%2520space%2520and%2520their%2520application%2520to%2520likelihood%2520phasing%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2000%26volume%3D56%26spage%3D1612%26epage%3D1621%26doi%3D10.1107%2Fs0907444900013263" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Langer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, S. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamzin, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrakis, A.</span></span> <span> </span><span class="NLM_article-title">Automated macromolecular model building for X-ray crystallography using ARP/wARP version 7</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">1171</span>– <span class="NLM_lpage">1179</span>, <span class="refDoi"> DOI: 10.1038/nprot.2008.91</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1038%2Fnprot.2008.91" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=18600222" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BD1cXotFaku7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=1171-1179&author=G.+Langerauthor=S.+X.+Cohenauthor=V.+S.+Lamzinauthor=A.+Perrakis&title=Automated+macromolecular+model+building+for+X-ray+crystallography+using+ARP%2FwARP+version+7&doi=10.1038%2Fnprot.2008.91"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Automated macromolecular model building for X-ray crystallography using ARP/wARP version 7</span></div><div class="casAuthors">Langer, Gerrit; Cohen, Serge X.; Lamzin, Victor S.; Perrakis, Anastassis</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1171-1179</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">ARP/wARP is a software suite to build macromol. models in x-ray crystallog. electron d. maps.  Structural genomics initiatives and the study of complex macromol. assemblies and membrane proteins all rely on advanced methods for 3D structure detn.  ARP/wARP meets these needs by providing the tools to obtain a macromol. model automatically, with a reproducible computational procedure.  ARP/wARP 7.0 tackles several tasks: iterative protein model building including a high-level decision-making control module; fast construction of the secondary structure of a protein; building flexible loops in alternate conformations; fully automated placement of ligands, including a choice of the best-fitting ligand from a 'cocktail'; and finding ordered water mols.  All protocols are easy to handle by a nonexpert user through a graphical user interface or a command line.  The time required is typically a few minutes although iterative model building may take a few hours.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcDZXxfCGFTbVg90H21EOLACvtfcHk0liDfUGN42Sk_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXotFaku7s%253D&md5=9b104e39902aee4db24be06e06707c63</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2008.91&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2008.91%26sid%3Dliteratum%253Aachs%26aulast%3DLanger%26aufirst%3DG.%26aulast%3DCohen%26aufirst%3DS.%2BX.%26aulast%3DLamzin%26aufirst%3DV.%2BS.%26aulast%3DPerrakis%26aufirst%3DA.%26atitle%3DAutomated%2520macromolecular%2520model%2520building%2520for%2520X-ray%2520crystallography%2520using%2520ARP%252FwARP%2520version%25207%26jtitle%3DNat.%2520Protoc.%26date%3D2008%26volume%3D3%26spage%3D1171%26epage%3D1179%26doi%3D10.1038%2Fnprot.2008.91" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/s0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2Fs0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0liDfUGN42Sk_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2Fs0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/s0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2Fs0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0liDfUGN42Sk_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2Fs0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/s0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2Fs0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0libJiVeTNBw9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2Fs0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrero, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Junk, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, J. W.</span></span> <span> </span><span class="NLM_article-title">Bismuth(III) complexes derived from non-steroidal anti-inflammatory drugs and their activity against <i>Helicobacter pylori</i></span>. <i>Dalton Trans.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">2861</span>– <span class="NLM_lpage">2868</span>, <span class="refDoi"> DOI: 10.1039/c000164c</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1039%2Fc000164c" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=20200713" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC3cXislaitb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2010&pages=2861-2868&author=P.+C.+Andrewsauthor=R.+L.+Ferreroauthor=P.+C.+Junkauthor=I.+Kumarauthor=Q.+Luuauthor=K.+Nguyenauthor=J.+W.+Taylor&title=Bismuth%28III%29+complexes+derived+from+non-steroidal+anti-inflammatory+drugs+and+their+activity+against+Helicobacter+pylori&doi=10.1039%2Fc000164c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Bismuth(III) complexes derived from non-steroidal anti-inflammatory drugs and their activity against Helicobacter pylori</span></div><div class="casAuthors">Andrews, Philip C.; Ferrero, Richard L.; Junk, Peter C.; Kumar, Ish; Luu, Quynh; Nguyen, Kim; Taylor, James W.</div><div class="citationInfo"><span class="NLM_cas:title">Dalton Transactions</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2861-2868</span>CODEN:
                <span class="NLM_cas:coden">DTARAF</span>;
        ISSN:<span class="NLM_cas:issn">1477-9226</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The formation of bismuth(III) complexes of carboxylates and benzoates derived from the 1 : 3 reaction of BiPh3 with the common non-steroidal anti-inflammatory drugs (NSAIDs) ketoprofen, naproxen, ibuprofen, mefenamic acid, diflunisal, 5-chlorosalicylic acid, fenbufen, sulindac, tolfenamic acid and flufenamic acid, has been achieved using both solvent-free and solvent-mediated methods.  The thermochem. profiles of the solvent-free reactions were studied using DSC-TGA.  All reactions produced the tris-substituted complexes of general formula [BiL3]n, with the complexes derived from ketoprofen and sulindac having an addnl. single bismuth bound H2O mol. in the inner coordination sphere.  The complexes are stable in air over a period of six months, do not undergo significant decompn. when suspended overnight in water, but decomp. in 1 M HCl soln. to release the free acid form of the NSAID.  All ten complexes show excellent in vitro activity against Helicobacter pylori with MIC values of ≥6.25 μg mL-1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9gUc3Zv-OerVg90H21EOLACvtfcHk0libJiVeTNBw9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXislaitb8%253D&md5=810b668a0f30c61e33f2f5144fca725c</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1039%2Fc000164c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc000164c%26sid%3Dliteratum%253Aachs%26aulast%3DAndrews%26aufirst%3DP.%2BC.%26aulast%3DFerrero%26aufirst%3DR.%2BL.%26aulast%3DJunk%26aufirst%3DP.%2BC.%26aulast%3DKumar%26aufirst%3DI.%26aulast%3DLuu%26aufirst%3DQ.%26aulast%3DNguyen%26aufirst%3DK.%26aulast%3DTaylor%26aufirst%3DJ.%2BW.%26atitle%3DBismuth%2528III%2529%2520complexes%2520derived%2520from%2520non-steroidal%2520anti-inflammatory%2520drugs%2520and%2520their%2520activity%2520against%2520Helicobacter%2520pylori%26jtitle%3DDalton%2520Trans.%26date%3D2010%26volume%3D39%26spage%3D2861%26epage%3D2868%26doi%3D10.1039%2Fc000164c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jenks, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labigne, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrero, R. L.</span></span> <span> </span><span class="NLM_article-title">Exposure to metronidazole <i>in vivo</i> readily induces resistance in <i>Helicobacter pylori</i> and reduces the efficacy of eradication therapy in mice</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">777</span>– <span class="NLM_lpage">781</span>, <span class="refDoi"> DOI: 10.1128/aac.43.4.777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1128%2FAAC.43.4.777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10103180" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADyaK1MXisVSgurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=1999&pages=777-781&author=P.+J.+Jenksauthor=A.+Labigneauthor=R.+L.+Ferrero&title=Exposure+to+metronidazole+in+vivo+readily+induces+resistance+in+Helicobacter+pylori+and+reduces+the+efficacy+of+eradication+therapy+in+mice&doi=10.1128%2Faac.43.4.777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Exposure to metronidazole in vivo readily induces resistance in Helicobacter pylori and reduces the efficacy of eradication therapy in mice</span></div><div class="casAuthors">Jenks, Peter J.; Labigne, Agnes; Ferrero, Richard L.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">777-781</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The H. pylori SS1 mouse model was used to characterize the development of resistance in H. pylori after treatment with metronidazole monotherapy and to examine the effect of prior exposure to metronidazole on the efficacy of a metronidazole-contg. eradication regimen.  Mice colonized with the metronidazole-sensitive H. pylori SS1 strain were treated for 7 days with either peptone trypsin broth or the mouse equiv. of 400 mg metronidazole once a day or 3 times per day (TID).  In a sep. expt., H. pylori-infected mice were administered either peptone trypsin broth or the mouse equiv. of 400 mg metronidazole TID for 7 days, followed 1 mo later by either peptone trypsin broth or the mouse equiv. of 20 mg omeprazole, 250 mg clarithromycin, and 400 mg metronidazole twice a day for 7 days.  At least 1 mo after the completion of treatment, the mice were sacrificed and their stomachs were cultured for H. pylori.  The susceptibilities of isolates to metronidazole were assessed by agar diln. detn. of the min. inhibitory concns. (MICs).  Mixed populations of metronidazole-resistant and -sensitive strains were isolated from 70% of the mice treated with 400 mg metronidazole TID.  The ratio of resistant to sensitive strains was 1:100, and the MICs for the resistant strains ranged 8-64 μg/mL.  In the 2nd expt., H. pylori was eradicated from 70% of the mice treated with eradication therapy alone, compared to 25% of those pretreated with metronidazole.  Mice still infected after treatment with metronidazole and eradication therapy contained mixed populations of metronidazole-resistant and -sensitive isolates in a ratio of 1:25.  These results demonstrate that H. pylori readily acquires resistance to metronidazole in vivo and that prior exposure of the organism to metronidazole is assocd. with failure of eradication therapy.  H. pylori-infected mice provide a suitable model for the study of resistance mechanisms in H. pylori and will be useful in detg. optimal regimens for the eradication of resistant strains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpSvrrQWXQxbVg90H21EOLACvtfcHk0libJiVeTNBw9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXisVSgurs%253D&md5=92f4266a99969230fcfc4d4fb6d73ae9</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1128%2FAAC.43.4.777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.43.4.777%26sid%3Dliteratum%253Aachs%26aulast%3DJenks%26aufirst%3DP.%2BJ.%26aulast%3DLabigne%26aufirst%3DA.%26aulast%3DFerrero%26aufirst%3DR.%2BL.%26atitle%3DExposure%2520to%2520metronidazole%2520in%2520vivo%2520readily%2520induces%2520resistance%2520in%2520Helicobacter%2520pylori%2520and%2520reduces%2520the%2520efficacy%2520of%2520eradication%2520therapy%2520in%2520mice%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1999%26volume%3D43%26spage%3D777%26epage%3D781%26doi%3D10.1128%2Faac.43.4.777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0023.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0023.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Purine salvage pathway in Hp. The green oval indicates the cell membrane of Hp. (A) GMP synthase; (B) IMP dehydrogenase; (C) adenylosuccinate synthase; (D) adenylosuccinate lyase; (E) GMP reductase; (F) purine nucleoside phosphorylase; APRT: adenine phosphoribosyltransferase; and XGHPRT: xanthine–guanine–hypoxanthine phosphoribosyltransferase. Adapted with permission from Liechti and Goldberg.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Cycling of purine bases, nucleosides, and nucleotides in Hp. The gene coding for the 5′/3′-nucleotidase is <i>sur</i>E. Adapted with permission from Liechti and Goldberg.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> The red cross signifies that nucleoside monophosphates are unable to cross the cellular membrane into the bacteria.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Reaction catalyzed by 6-oxopurine PRTases. When R = NH<sub>2</sub>, the base is guanine, when R = H, the base is hypoxanthine, and when R = OH, the base is xanthine.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Chemical templates of 6-oxopurine PRTase phosphonate inhibitors. (A) First ANPs. X denotes the atoms connecting the two functional groups, the base and the phosphonate group, and can be either carbon or oxygen atoms. R<sub>1</sub> denotes where further chemical attachments have been made.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> (B) aza-ANPs. Attachments at (R<sub>2</sub>) allow a functional group to reach into a third cavity in the active site, the pyrophosphate binding pocket.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> (C) AIPs containing a positively charged nitrogen in the linker mimicking the positive charge on the ribose oxygen which is predicted to occur at the transition state. Variable attachments have been made at R<sub>3</sub>.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> (D) PNPs. The length of the linker (<i>m</i>) can vary.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> (E) PNBPs. A second phosphonate group is added to the carbon atom in the pyrrolidine ring. Rotation around the five-membered ring allows either of the phosphonate groups to point to the two different areas in the active site that can bind phosphate groups.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Sequence alignment between HpXGHPRT and 6-oxopurine PRTases from humans, Ec and Mt. The residues highlighted in cyan are identical and those in red are homologous. The “boxes” indicate the binding site of functional groups, that is, purine base, 5′-phosphate group of the nucleoside monophosphates or PRib-PP, the divalent metal ion, and PP<sub>i</sub> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). These locations are based on the known crystal structures of the human and bacterial enzymes in complex with substrates or products of the reaction or with inhibitors. HpXGHPRT (WP_00559479); EcXGPRT (POA9M5); HsHGPRT (POO492); EcHPRT (POA9M2); and MtHGPRT (POA5TO).<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Crystal structure of subunit A of HpXGHPRT in the absence of an inhibitor. The eight conserved residues across five 6-oxopurine PRTases are in green and the two bound sulfate ions are shown as spheres. Red indicates the location of the long flexible loop (S59–I71) which is visible in this subunit except for the main chain atoms of H64. Two highly conserved acidic amino acids, 88D and 89E in HpXGHPRT (DD in EcXGPRT and ED in the other three amino acid sequences), are in dark blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Comparison of the subunit structure of HpXGHPRT with HsHGPRT and three other bacterial 6-oxopurine PRTases. (A) HpXGHPRT (153 amino acids) and (B) human (Hs) HGPRT (218 amino acids; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RAD">4RAD</a>).<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Ligand is 9-[(<i>N</i>-phosphonoethyl-<i>N</i>-phosphonoethoxyethyl)-2-aminoethyl]-guanine; (C) EcXGPRT (152 amino acids; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1A97">1A97</a>.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> Ligand is GMP); (D) EcHPRT (178 amino acids; PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1G9S">1G9S</a>.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Ligand is IMP); and (E) MtHGPRT (202 amino acids); PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RHU">4RHU</a>.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Ligand is 2-(2-(guanin-9-yl)ethyl)(2-((2-hydroxyethyl)(2-phosphonoethyl)amino)ethyl) phosphonic acid.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Dimeric crystal structure of HpXGHPRT with two sulfate ions in each subunit. The red cartoon is the long flexible loop (subunit A) which is predicted to move over the active site during catalysis.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Sulfate binding sites in HpXGHPRT (A). (Left) Interactions that the sulfonyl oxygen atoms form in the predicted 5′-phosphate binding site of HpXGHPRT. (Right) 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density for the sulfate group and four water molecules (green spheres) is overlaid as a blue mesh. These are contoured at a sigma level of 1.0. (B) (Left) The location of the second sulfate ion in HpXGHPRT. The loop in subunit A (gray; M33–G350) joins an α-helix motif to a β-sheet. Hydrogen bonding pattern when the sulfate ion binds. Subunit A is represented in blue and subunit B is represented in pink. (Right) The 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density map for the bound sulfate group and nearby waters is overlaid in blue mesh. (C) Two sulfate ions in the PP<sub>i</sub> binding site [<b>I</b>] to the sulfate ion in the 5′-phosphate binding site [<b>II</b>] and connected by a network of water molecules (green spheres).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0011.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Chemical structure of two known AIPs and the proposed transition state. (A) Compound <b>XII</b>. 9-[(<i>N</i>-3-Phosphonopropyl)-aminomethyl]-9-deazahypoxanthine.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> (B) Compound <b>XIII</b>. 9-[(<i>N</i>-3-(4-Hydroxyphosphonobutyl))-aminomethyl]-9-deazahypoxanthine.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> (C) Proposed transition state of the catalytic reaction.<a onclick="showRef(event, 'ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref46 ref47">(46,47)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0012.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Crystal structure of HpXGHPRT in complex with <b>XII</b>. (A) Connolly surface of HpXGHPRT dimer in complex with <b>XII</b> and one sulfate ion with subunit B and two sulfate ions in subunit A. (B) <b>XII</b> bound in the active site of HpXGHPRT and the interactions that it forms with backbone and side-chain atoms of amino acids and also water molecules. Green dashed lines show the hydrogen bond network; red dashed lines show the distances in Å between the ligand and the side chain atoms of two highly conserved amino acids, D92 and K121.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0013.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Interactions between two ANPs and HpXGHPRT and PfHGXPRT. (A) Connolly surface of the crystal structure of the PfHGXPRT·<b>XIII</b> complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3OZG">3OZG</a>) with the structure of the HpXGHPRT·<b>XII</b> complex superimposed. The green sphere represents a magnesium ion. (B) The interactions that the positively charged nitrogen atom of <b>XII</b> forms in the active site of HpXGHPRT. (C) The interactions that the positively charged nitrogen atom of <b>XIII</b> forms in the active site of PfHGXPRT. All distances shown are in Å. Water molecules are shown as red spheres in (B,C).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig13" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0014.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. Movement of the side chain of the N-terminal and the side chain of W138 in HpXGHPRT. (Left) In the absence of inhibitor. Subunit A is orange; subunit B is gold. In subunit B, there is no density for the side chain of His(−2) and this side chain is modeled as alanine. (Right) In the presence of <b>XII</b> in the active site. Subunit A is represented in pink; subunit B is represented in silver. (−2) signifies the number of amino acids distal to the beginning of the integral amino acid sequence (HYSY).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig14" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0015.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 14. Comparison of the position of the purine base in the HpXGHPRT, EcXGPRT, and HsHGPRT complexes. (A) Interactions that the N<sup>1</sup> atom in the purine ring makes when the exocyclic amino group at the C<sup>2</sup> position of the purine base makes when <b>XII</b> or GMP are bound in the active site of HpXGHPRT or EcXGPRT (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1A97">1A97</a>), respectively.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> Pink represents HpXGHPRT and blue represents EcXGPRT. The blue sphere is the bridging water between the exocyclic amino group and the two amino acid side chains in the EcXGPRT·GMP complex. (B) Human HGPRT in complex with the inhibitor, [3<i>R</i>,4<i>R</i>-4-guanin-9-yl-3-((<i>S</i>)-2-hydroxy-2-phosphonoethyl)oxy-1-<i>N</i>-(phosphonopropionyl)pyrrolidine (compound <b>1</b>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5HIA">5HIA</a>).<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig14"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig15" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 15</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0016.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 15. Interactions that the side chains of D88 and E89 form at the active site of HpXGHPRT (subunit A, no inhibitor present). This sulfate ion is located in the pyrophosphate binding pocket.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig15"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig16" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Figure 16</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0017.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 16. Comparison of inhibitor binding to two 6-oxopurine PRTases. (A) Superimposition of HpXGHPRT·<b>XII</b> complex, EcXGPRT·GMP complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1A97">1A97</a>), and EcXGPRT.(<i>S</i>)-3-(guanin-9-yl)pyrrolidine-<i>N</i>-ylacetylphosphonic acid complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4JIT">4JIT</a>).<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> XGPRT and GMP are represented in blue; (<i>S</i>)-3-(guanin-9-yl)pyrrolidine-<i>N</i>-ylacetylphosphonic acid is represented in green; HpXGHPRT and <b>XII</b> are represented in pink. The purine ring and the five-membered ring of the Ec complexes are located in identical locations. (B) Hydrogen bond network that K121 forms at the active site in the HpXGHPRT·<b>XII</b> complex and the distances in angstroms between the NZ atom of this amino acid. The distance between the OD atom of D92 and the N<sup>7</sup> atom of the purine ring is also shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig16"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig17" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Figure 17</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0018.jpeg" id="rightTab-gr17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 17. Chemical structure of <b>XIV</b>, 9-[(<i>N</i>,<i>N</i>-(bis-3-phosphonopropyl))-aminomethyl]-9-deazahypoxanthine.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig17"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig18" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Figure 18</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/medium/jm0c02184_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0019.jpeg" id="rightTab-gr18" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 18. Superimposition of the complexes of two known crystal structures onto that of the HpXGHPRT·<b>XII</b> complex. (A) Human HGPRT·<b>I</b> complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5HIA">5HIA</a>: green) with the HpXGHPRT·<b>XII</b> complex. (B) MtHGPRT·<b>III</b> complex (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RHX">4RHX</a>) with the HpXGHPRT·<b>XII</b> complex. A putative location of the pyrophosphate was obtained by superimposition of the human HGPRT structure in complex with the transition state analogue (ImmHP) and PP<sub>i</sub>. (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BZY">1BZY</a>).<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02184/20210506/images/large/jm0c02184_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02184&amp;id=fig18"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i45">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02500" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02500" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 64 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, J. R.</span></span> <span> </span><span class="NLM_article-title">Unidentified curved bacilli in the stomach of patients with gastritis and peptic ulceration</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>323</i></span>,  <span class="NLM_fpage">1311</span>– <span class="NLM_lpage">1315</span>, <span class="refDoi"> DOI: 10.1016/s0140-6736(84)91816-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1016%2FS0140-6736%2884%2991816-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=323&publication_year=1984&pages=1311-1315&author=B.+Marshallauthor=J.+R.+Warren&title=Unidentified+curved+bacilli+in+the+stomach+of+patients+with+gastritis+and+peptic+ulceration&doi=10.1016%2Fs0140-6736%2884%2991816-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2884%2991816-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252884%252991816-6%26sid%3Dliteratum%253Aachs%26aulast%3DMarshall%26aufirst%3DB.%26aulast%3DWarren%26aufirst%3DJ.%2BR.%26atitle%3DUnidentified%2520curved%2520bacilli%2520in%2520the%2520stomach%2520of%2520patients%2520with%2520gastritis%2520and%2520peptic%2520ulceration%26jtitle%3DLancet%26date%3D1984%26volume%3D323%26spage%3D1311%26epage%3D1315%26doi%3D10.1016%2Fs0140-6736%2884%2991816-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hooi, J. K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, W. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suen, M. M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Underwood, F. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanyingoh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malfertheiner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, V. W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J. C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, F. K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, J. J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaplan, G. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, S. C.</span></span> <span> </span><span class="NLM_article-title">Global prevalence of <i>Helicobacter pylori</i> infection: Systematic review and meta-analysis</span>. <i>Gastroenterology</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>153</i></span>,  <span class="NLM_fpage">420</span>– <span class="NLM_lpage">429</span>, <span class="refDoi"> DOI: 10.1053/j.gastro.2017.04.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1053%2Fj.gastro.2017.04.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=28456631" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A280%3ADC%252BC1crisFaktA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=153&publication_year=2017&pages=420-429&author=J.+K.+Y.+Hooiauthor=W.+Y.+Laiauthor=W.+K.+Ngauthor=M.+M.+Y.+Suenauthor=F.+E.+Underwoodauthor=D.+Tanyingohauthor=P.+Malfertheinerauthor=D.+Y.+Grahamauthor=V.+W.+S.+Wongauthor=J.+C.+Y.+Wuauthor=F.+K.+L.+Chanauthor=J.+J.+Y.+Sungauthor=G.+G.+Kaplanauthor=S.+C.+Ng&title=Global+prevalence+of+Helicobacter+pylori+infection%3A+Systematic+review+and+meta-analysis&doi=10.1053%2Fj.gastro.2017.04.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Global Prevalence of Helicobacter pylori Infection: Systematic Review and Meta-Analysis</span></div><div class="casAuthors">Hooi James K Y; Lai Wan Ying; Suen Michael M Y; Wong Vincent W S; Wu Justin C Y; Chan Francis K L; Sung Joseph J Y; Ng Wee Khoon; Underwood Fox E; Tanyingoh Divine; Malfertheiner Peter; Graham David Y; Kaplan Gilaad G; Ng Siew C</div><div class="citationInfo"><span class="NLM_cas:title">Gastroenterology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">153</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">420-429</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND & AIMS:  The epidemiology of Helicobacter pylori infection has changed with improvements in sanitation and methods of eradication.  We performed a systematic review and meta-analysis to evaluate changes in the global prevalence of H pylori infection.  METHODS:  We performed a systematic search of the MEDLINE and EMBASE databases for studies of the prevalence of H pylori infection published from January 1, 1970 through January 1, 2016.  We analyzed data based on United Nations geoscheme regions and individual countries.  We used a random effects model to calculate pooled prevalence estimates with 95% confidence intervals (CIs), weighted by study size.  We extrapolated 2015 prevalence estimates to obtain the estimated number of individuals with H pylori infection.  RESULTS:  Among 14,006 reports screened, we identified 263 full-text articles on the prevalence of H pylori infection; 184 were included in the final analysis, comprising data from 62 countries.  Africa had the highest pooled prevalence of H pylori infection (70.1%; 95% CI, 62.6-77.7), whereas Oceania had the lowest prevalence (24.4%; 95% CI, 18.5-30.4).  Among individual countries, the prevalence of H pylori infection varied from as low as 18.9% in Switzerland (95% CI, 13.1-24.7) to 87.7% in Nigeria (95% CI, 83.1-92.2).  Based on regional prevalence estimates, there were approximately 4.4 billion individuals with H pylori infection worldwide in 2015.  CONCLUSIONS:  In a systematic review and meta-analysis to assess the prevalence of H pylori infection worldwide, we observed large amounts of variation among regions-more than half the world's population is infected.  These data can be used in development of customized strategies for the global eradication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT6iihP5eyfJaqcneEtCi92fW6udTcc2ebzlih16lcZx7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1crisFaktA%253D%253D&md5=612061e322f559cdbe6c3a5c86fa0b15</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1053%2Fj.gastro.2017.04.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1053%252Fj.gastro.2017.04.022%26sid%3Dliteratum%253Aachs%26aulast%3DHooi%26aufirst%3DJ.%2BK.%2BY.%26aulast%3DLai%26aufirst%3DW.%2BY.%26aulast%3DNg%26aufirst%3DW.%2BK.%26aulast%3DSuen%26aufirst%3DM.%2BM.%2BY.%26aulast%3DUnderwood%26aufirst%3DF.%2BE.%26aulast%3DTanyingoh%26aufirst%3DD.%26aulast%3DMalfertheiner%26aufirst%3DP.%26aulast%3DGraham%26aufirst%3DD.%2BY.%26aulast%3DWong%26aufirst%3DV.%2BW.%2BS.%26aulast%3DWu%26aufirst%3DJ.%2BC.%2BY.%26aulast%3DChan%26aufirst%3DF.%2BK.%2BL.%26aulast%3DSung%26aufirst%3DJ.%2BJ.%2BY.%26aulast%3DKaplan%26aufirst%3DG.%2BG.%26aulast%3DNg%26aufirst%3DS.%2BC.%26atitle%3DGlobal%2520prevalence%2520of%2520Helicobacter%2520pylori%2520infection%253A%2520Systematic%2520review%2520and%2520meta-analysis%26jtitle%3DGastroenterology%26date%3D2017%26volume%3D153%26spage%3D420%26epage%3D429%26doi%3D10.1053%2Fj.gastro.2017.04.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Megraud, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leon-Barua, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bazzoli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Merwe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaz Coelho, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fock, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedail, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malfertheiner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakil, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamid, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goh, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krabshuis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Mair, A.</span></span>; <span class="NLM_contrib-group">World Gastroenterology Organization</span> <span> </span><span class="NLM_article-title"><i>Helicobacter pylori</i> in developing countries. World gastroenterology organisation global guideline</span>. <i>J. Gastrointest. Liver Dis.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">304</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=21961099" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A280%3ADC%252BC3MfnsFyjtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2011&pages=299-304&author=R.+H.+Huntauthor=S.+D.+Xiaoauthor=F.+Megraudauthor=R.+Leon-Baruaauthor=F.+Bazzoliauthor=S.+van+der+Merweauthor=L.+G.+Vaz+Coelhoauthor=M.+Fockauthor=S.+Fedailauthor=H.+Cohenauthor=P.+Malfertheinerauthor=N.+Vakilauthor=S.+Hamidauthor=K.+L.+Gohauthor=B.+C.+Wongauthor=J.+Krabshuisauthor=A.+Le+Mairauthor=World+Gastroenterology+Organization&title=Helicobacter+pylori+in+developing+countries.+World+gastroenterology+organisation+global+guideline"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Helicobacter pylori in developing countries. World Gastroenterology Organisation Global Guideline</span></div><div class="casAuthors">Hunt R H; Xiao S D; Megraud F; Leon-Barua R; Bazzoli F; van der Merwe S; Vaz Coelho L G; Fock M; Fedail S; Cohen H; Malfertheiner P; Vakil N; Hamid S; Goh K L; Wong B C Y; Krabshuis J; Le Mair A</div><div class="citationInfo"><span class="NLM_cas:title">Journal of gastrointestinal and liver diseases : JGLD</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">299-304</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSRh26RHe2V9NenUz0lpNqnfW6udTcc2ebzlih16lcZx7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MfnsFyjtw%253D%253D&md5=184f5c36f135f3c20da2c25afbea1461</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHunt%26aufirst%3DR.%2BH.%26aulast%3DXiao%26aufirst%3DS.%2BD.%26aulast%3DMegraud%26aufirst%3DF.%26aulast%3DLeon-Barua%26aufirst%3DR.%26aulast%3DBazzoli%26aufirst%3DF.%26aulast%3Dvan%2Bder%2BMerwe%26aufirst%3DS.%26aulast%3DVaz%2BCoelho%26aufirst%3DL.%2BG.%26aulast%3DFock%26aufirst%3DM.%26aulast%3DFedail%26aufirst%3DS.%26aulast%3DCohen%26aufirst%3DH.%26aulast%3DMalfertheiner%26aufirst%3DP.%26aulast%3DVakil%26aufirst%3DN.%26aulast%3DHamid%26aufirst%3DS.%26aulast%3DGoh%26aufirst%3DK.%2BL.%26aulast%3DWong%26aufirst%3DB.%2BC.%26aulast%3DKrabshuis%26aufirst%3DJ.%26aulast%3DLe%2BMair%26aufirst%3DA.%26aulast%3D%26atitle%3DHelicobacter%2520pylori%2520in%2520developing%2520countries.%2520World%2520gastroenterology%2520organisation%2520global%2520guideline%26jtitle%3DJ.%2520Gastrointest.%2520Liver%2520Dis.%26date%3D2011%26volume%3D20%26spage%3D299%26epage%3D304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saleem, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howden, C. W.</span></span> <span> </span><span class="NLM_article-title">Update on the management of <i>Helicobacter pylori</i> infection</span>. <i>Curr. Treat. Options Gastroenterol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">476</span>– <span class="NLM_lpage">487</span>, <span class="refDoi"> DOI: 10.1007/s11938-020-00300-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1007%2Fs11938-020-00300-3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2020&pages=476-487&author=N.+Saleemauthor=C.+W.+Howden&title=Update+on+the+management+of+Helicobacter+pylori+infection&doi=10.1007%2Fs11938-020-00300-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1007%2Fs11938-020-00300-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11938-020-00300-3%26sid%3Dliteratum%253Aachs%26aulast%3DSaleem%26aufirst%3DN.%26aulast%3DHowden%26aufirst%3DC.%2BW.%26atitle%3DUpdate%2520on%2520the%2520management%2520of%2520Helicobacter%2520pylori%2520infection%26jtitle%3DCurr.%2520Treat.%2520Options%2520Gastroenterol.%26date%3D2020%26volume%3D18%26spage%3D476%26epage%3D487%26doi%3D10.1007%2Fs11938-020-00300-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sarajlic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuper, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vetter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klicznik, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gratz, I. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wessler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Posselt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horejs-Hoeck, J.</span></span> <span> </span><span class="NLM_article-title"><i>H. pylori</i> modulates DC functions via T4SS/TNFα/p38-dependent SOCS3 expression</span>. <i>Cell Commun. Signaling</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">160</span>, <span class="refDoi"> DOI: 10.1186/s12964-020-00655-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1186%2Fs12964-020-00655-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=33023610" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitVyiur%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2020&pages=160&author=M.+Sarajlicauthor=T.+Neuperauthor=J.+Vetterauthor=S.+Schallerauthor=M.+M.+Klicznikauthor=I.+K.+Gratzauthor=S.+Wesslerauthor=G.+Posseltauthor=J.+Horejs-Hoeck&title=H.+pylori+modulates+DC+functions+via+T4SS%2FTNF%CE%B1%2Fp38-dependent+SOCS3+expression&doi=10.1186%2Fs12964-020-00655-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">H. pylori modulates DC functions via T4SS/TNFα/p38-dependent SOCS3 expression</span></div><div class="casAuthors">Sarajlic, Muamera; Neuper, Theresa; Vetter, Julia; Schaller, Susanne; Klicznik, Maria M.; Gratz, Iris K.; Wessler, Silja; Posselt, Gernot; Horejs-Hoeck, Jutta</div><div class="citationInfo"><span class="NLM_cas:title">Cell Communication and Signaling</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">160</span>CODEN:
                <span class="NLM_cas:coden">CCSEC6</span>;
        ISSN:<span class="NLM_cas:issn">1478-811X</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Helicobacter pylori (H. pylori) is a gram-neg. bacterium that chronically infects approx. 50% of the world's human population.  While in most cases the infection remains asymptomatic, 10% of infected individuals develop gastric pathologies and 1-3% progress to gastric cancer.  Although H. pylori induces severe inflammatory responses, the host's immune system fails to clear the pathogen and H. pylori can persist in the human stomach for decades.  As suppressor of cytokine signaling (SOCS) proteins are important feedback regulators limiting inflammatory responses, we hypothesized that H. pylori could modulate the host's immune responses by inducing SOCS expression.  The phenotype of human monocyte-derived DCs (moDCs) infected with H. pylori was analyzed by flow cytometry and multiplex technol.  SOCS expression levels were monitored by qPCR and signaling studies were conducted by means of Western blot.  For functional studies, RNA interference-based silencing of SOCS1-3 and co-cultures with CD4+ T cells were performed.  We show that H. pylori pos. gastritis patients express significantly higher SOCS3, but not SOCS1 and SOCS2, levels compared to H. pylori neg. patients.  Moreover, infection of human moDCs with H. pylori rapidly induces SOCS3 expression, which requires the type IV secretion system (T4SS), release of TNFα, and signaling via the MAP kinase p38, but appears to be independent of TLR2, TLR4, MEK1/2 and STAT proteins.  Silencing of SOCS3 expression in moDCs prior to H. pylori infection resulted in increased release of both pro- and anti-inflammatory cytokines, upregulation of PD-L1, and decreased T-cell proliferation.  This study shows that H. pylori induces SOCS3 via an autocrine loop involving the T4SS and TNFα and p38 signaling.  Moreover, we demonstrate that high levels of SOCS3 in DCs dampen PD-L1 expression on DCs, which in turn drives T-cell proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpI0i4xiDIzq7Vg90H21EOLACvtfcHk0ljdLYipfF-J9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitVyiur%252FJ&md5=338c54fb4ab02f7129dec2ec6c8ebb66</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1186%2Fs12964-020-00655-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12964-020-00655-1%26sid%3Dliteratum%253Aachs%26aulast%3DSarajlic%26aufirst%3DM.%26aulast%3DNeuper%26aufirst%3DT.%26aulast%3DVetter%26aufirst%3DJ.%26aulast%3DSchaller%26aufirst%3DS.%26aulast%3DKlicznik%26aufirst%3DM.%2BM.%26aulast%3DGratz%26aufirst%3DI.%2BK.%26aulast%3DWessler%26aufirst%3DS.%26aulast%3DPosselt%26aufirst%3DG.%26aulast%3DHorejs-Hoeck%26aufirst%3DJ.%26atitle%3DH.%2520pylori%2520modulates%2520DC%2520functions%2520via%2520T4SS%252FTNF%25CE%25B1%252Fp38-dependent%2520SOCS3%2520expression%26jtitle%3DCell%2520Commun.%2520Signaling%26date%3D2020%26volume%3D18%26spage%3D160%26doi%3D10.1186%2Fs12964-020-00655-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kuipers, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israel, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusters, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerrits, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Ende, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Hulst, R. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wirth, H. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Höök-Nikanne, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaser, M. J.</span></span> <span> </span><span class="NLM_article-title">Quasispecies development of <i>Helicobacter pylori</i> observed in paired isolates obtained years apart from the same host</span>. <i>J. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>181</i></span>,  <span class="NLM_fpage">273</span>– <span class="NLM_lpage">282</span>, <span class="refDoi"> DOI: 10.1086/315173</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1086%2F315173" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10608776" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BD3cXoslagsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=181&publication_year=2000&pages=273-282&author=E.+J.+Kuipersauthor=D.+A.+Israelauthor=J.+G.+Kustersauthor=M.+M.+Gerritsauthor=J.+Weelauthor=A.+van+der+Endeauthor=R.+W.+M.+van+der+Hulstauthor=H.+P.+Wirthauthor=J.+H%C3%B6%C3%B6k-Nikanneauthor=S.+A.+Thompsonauthor=M.+J.+Blaser&title=Quasispecies+development+of+Helicobacter+pylori+observed+in+paired+isolates+obtained+years+apart+from+the+same+host&doi=10.1086%2F315173"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Quasispecies development of Helicobacter pylori observed in paired isolates obtained years apart from the same host</span></div><div class="casAuthors">Kuipers, Ernst J.; Israel, Dawn A.; Kusters, Johannes G.; Gerrits, Monique M.; Weel, Jan; van der Ende, Arie; van der Hulst, Rene W. M.; Wirth, Hans P.; Hook-Nikanne, Johanna; Thompson, Stuart A.; Blaser, Martin J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Infectious Diseases</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">181</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">273-282</span>CODEN:
                <span class="NLM_cas:coden">JIDIAQ</span>;
        ISSN:<span class="NLM_cas:issn">0022-1899</span>.
    
            (<span class="NLM_cas:orgname">University of Chicago Press</span>)
        </div><div class="casAbstract">Helicobacter pylori isolates show greater genetic diversity than other bacterial species studied, but the basis for this phenomenon is unknown.  Whether detectable genomic mutation appears within an H. pylori population during persistent colonization was investigated.  Paired H. pylori populations obtained across 7-10-yr intervals from 13 patients were characterized by use of PCR genotyping for cagA, vacA, iceA, recA, and IS605; RAPD-PCR and AFLP anal.; and ELISA, to det. Lewis phenotypes.  Genotyping, including recA sequence anal., revealed that initial and follow-up populations represented the same population in 11 patients (85%).  Nevertheless, distinct dissimilarities were shown within each of these 11 pairs by both RAPD-PCR and AFLP analyses.  During follow-up, Lewis-y levels, but not Lewis-x levels, decreased significantly.  The changes detected by RAPD-PCR and AFLP indicate that genetic drift occurs within H. pylori populations over the course of years of colonization of a single host.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0xi_ogykbCLVg90H21EOLACvtfcHk0ljdLYipfF-J9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXoslagsA%253D%253D&md5=f00f66b9ebaf14ce33427c430df09303</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1086%2F315173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1086%252F315173%26sid%3Dliteratum%253Aachs%26aulast%3DKuipers%26aufirst%3DE.%2BJ.%26aulast%3DIsrael%26aufirst%3DD.%2BA.%26aulast%3DKusters%26aufirst%3DJ.%2BG.%26aulast%3DGerrits%26aufirst%3DM.%2BM.%26aulast%3DWeel%26aufirst%3DJ.%26aulast%3Dvan%2Bder%2BEnde%26aufirst%3DA.%26aulast%3Dvan%2Bder%2BHulst%26aufirst%3DR.%2BW.%2BM.%26aulast%3DWirth%26aufirst%3DH.%2BP.%26aulast%3DH%25C3%25B6%25C3%25B6k-Nikanne%26aufirst%3DJ.%26aulast%3DThompson%26aufirst%3DS.%2BA.%26aulast%3DBlaser%26aufirst%3DM.%2BJ.%26atitle%3DQuasispecies%2520development%2520of%2520Helicobacter%2520pylori%2520observed%2520in%2520paired%2520isolates%2520obtained%2520years%2520apart%2520from%2520the%2520same%2520host%26jtitle%3DJ.%2520Infect.%2520Dis.%26date%3D2000%26volume%3D181%26spage%3D273%26epage%3D282%26doi%3D10.1086%2F315173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peek, R. M.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaser, M. J.</span></span> <span> </span><span class="NLM_article-title"><i>Helicobacter pylori</i> and gastrointestinal tract adenocarcinomas</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">28</span>– <span class="NLM_lpage">37</span>, <span class="refDoi"> DOI: 10.1038/nrc703</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1038%2Fnrc703" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=11902583" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BD38XhvF2jt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2002&pages=28-37&author=R.+M.+Peekauthor=M.+J.+Blaser&title=Helicobacter+pylori+and+gastrointestinal+tract+adenocarcinomas&doi=10.1038%2Fnrc703"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Helicobacter pylori and gastrointestinal tract adenocarcinomas</span></div><div class="casAuthors">Peek, Richard M., Jr.; Blaser, Martin J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">28-37</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Although gastric adenocarcinoma is assocd. with the presence of Helicobacter pylori in the stomach, only a small fraction of colonized individuals develop this common malignancy.  H. pylori strain and host genotypes probably influence the risk of carcinogenesis by differentially affecting host inflammatory responses and epithelial-cell physiol.  Understanding the host-microbial interactions that lead to neoplasia will improve cancer-targeted therapeutics and diagnostics, and provide mechanistic insights into other malignancies that arise within the context of microbially initiated inflammatory states.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCiTckbOS5wLVg90H21EOLACvtfcHk0ljdLYipfF-J9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhvF2jt7c%253D&md5=3dcc5fdf6d5347a2cfe82e8eed349d59</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrc703&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc703%26sid%3Dliteratum%253Aachs%26aulast%3DPeek%26aufirst%3DR.%2BM.%26aulast%3DBlaser%26aufirst%3DM.%2BJ.%26atitle%3DHelicobacter%2520pylori%2520and%2520gastrointestinal%2520tract%2520adenocarcinomas%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2002%26volume%3D2%26spage%3D28%26epage%3D37%26doi%3D10.1038%2Fnrc703" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Uemura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamakido, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taniyama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlemper, R. J.</span></span> <span> </span><span class="NLM_article-title"><i>Helicobacter pylori</i> infection and the development of gastric cancer</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>345</i></span>,  <span class="NLM_fpage">784</span>– <span class="NLM_lpage">789</span>, <span class="refDoi"> DOI: 10.1056/nejmoa001999</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1056%2FNEJMoa001999" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=11556297" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A280%3ADC%252BD3MrgslCqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=345&publication_year=2001&pages=784-789&author=N.+Uemuraauthor=S.+Okamotoauthor=S.+Yamamotoauthor=N.+Matsumuraauthor=S.+Yamaguchiauthor=M.+Yamakidoauthor=K.+Taniyamaauthor=N.+Sasakiauthor=R.+J.+Schlemper&title=Helicobacter+pylori+infection+and+the+development+of+gastric+cancer&doi=10.1056%2Fnejmoa001999"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Helicobacter pylori infection and the development of gastric cancer</span></div><div class="casAuthors">Uemura N; Okamoto S; Yamamoto S; Matsumura N; Yamaguchi S; Yamakido M; Taniyama K; Sasaki N; Schlemper R J</div><div class="citationInfo"><span class="NLM_cas:title">The New England journal of medicine</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">345</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">784-9</span>
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    </div><div class="casAbstract">BACKGROUND:  Although many studies have found an association between Helicobacter pylori infection and the development of gastric cancer, many aspects of this relation remain uncertain.  METHODS:  We prospectively studied 1526 Japanese patients who had duodenal ulcers, gastric ulcers, gastric hyperplasia, or nonulcer dyspepsia at the time of enrollment; 1246 had H. pylori infection and 280 did not.  The mean follow-up was 7.8 years (range, 1.0 to 10.6).  Patients underwent endoscopy with biopsy at enrollment and then between one and three years after enrollment.  H. pylori infection was assessed by histologic examination, serologic testing, and rapid urease tests and was defined by a positive result on any of these tests.  RESULTS:  Gastric cancers developed in 36 (2.9 percent) of the infected and none of the uninfected patients.  There were 23 intestinal-type and 13 diffuse-type cancers.  Among the patients with H. pylori infection, those with severe gastric atrophy, corpus-predominant gastritis, and intestinal metaplasia were at significantly higher risk for gastric cancer.  We detected gastric cancers in 21 (4.7 percent) of the 445 patients with nonulcer dyspepsia, 10 (3.4 percent) of the 297 with gastric ulcers, 5 (2.2 percent) of the 229 with gastric hyperplastic polyps, and none of the 275 with duodenal ulcers.  CONCLUSIONS:  Gastric cancer develops in persons infected with H. pylori but not in uninfected persons.  Those with histologic findings of severe gastric atrophy, corpus-predominant gastritis, or intestinal metaplasia are at increased risk.  Persons with H. pylori infection and nonulcer dyspepsia, gastric ulcers, or gastric hyperplastic polyps are also at risk, but those with duodenal ulcers are not.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTAlvuOt_-alMAKB7Fa-xrUfW6udTcc2eYZWW62pUCP27ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3MrgslCqtA%253D%253D&md5=53ac3adf0529c87369f26614e10633fd</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa001999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa001999%26sid%3Dliteratum%253Aachs%26aulast%3DUemura%26aufirst%3DN.%26aulast%3DOkamoto%26aufirst%3DS.%26aulast%3DYamamoto%26aufirst%3DS.%26aulast%3DMatsumura%26aufirst%3DN.%26aulast%3DYamaguchi%26aufirst%3DS.%26aulast%3DYamakido%26aufirst%3DM.%26aulast%3DTaniyama%26aufirst%3DK.%26aulast%3DSasaki%26aufirst%3DN.%26aulast%3DSchlemper%26aufirst%3DR.%2BJ.%26atitle%3DHelicobacter%2520pylori%2520infection%2520and%2520the%2520development%2520of%2520gastric%2520cancer%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2001%26volume%3D345%26spage%3D784%26epage%3D789%26doi%3D10.1056%2Fnejmoa001999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kusters, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Vliet, A. H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuipers, E. J.</span></span> <span> </span><span class="NLM_article-title">Pathogenesis of <i>Helicobacter pylori</i> infection</span>. <i>Clin. Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">449</span>– <span class="NLM_lpage">490</span>, <span class="refDoi"> DOI: 10.1128/cmr.00054-05</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1128%2FCMR.00054-05" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=16847081" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BD28XosVaqsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2006&pages=449-490&author=J.+G.+Kustersauthor=A.+H.+M.+van+Vlietauthor=E.+J.+Kuipers&title=Pathogenesis+of+Helicobacter+pylori+infection&doi=10.1128%2Fcmr.00054-05"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Pathogenesis of Helicobacter pylori infection</span></div><div class="casAuthors">Kusters, Johannes G.; van Vliet, Arnoud H. M.; Kuipers, Ernst J.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Microbiology Reviews</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">449-490</span>CODEN:
                <span class="NLM_cas:coden">CMIREX</span>;
        ISSN:<span class="NLM_cas:issn">0893-8512</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A review.  Helicobacter pylori is the first formally recognized bacterial carcinogen and is one of the most successful human pathogens, as over half of the world's population is colonized with this gram-neg. bacterium.  Unless treated, colonization usually persists lifelong.  H. pylori infection represents a key factor in the etiol. of various gastrointestinal diseases, ranging from chronic active gastritis without clin. symptoms to peptic ulceration, gastric adenocarcinoma, and gastric mucosa-assocd. lymphoid tissue lymphoma.  Disease outcome is the result of the complex interplay between the host and the bacterium.  Host immune gene polymorphisms and gastric acid secretion largely det. the bacterium's ability to colonize a specific gastric niche.  Bacterial virulence factors such as the cytotoxin-assocd. gene pathogenicity island-encoded protein CagA and the vacuolating cytotoxin VacA aid in this colonization of the gastric mucosa and subsequently seem to modulate the host's immune system.  This review focuses on the microbiol., clin., immunol., and biochem. aspects of the pathogenesis of H. pylori.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRylz2V0k-XrVg90H21EOLACvtfcHk0lg3gaCCxH9jMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XosVaqsrg%253D&md5=5d97264b972343880e13499120984405</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1128%2FCMR.00054-05&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FCMR.00054-05%26sid%3Dliteratum%253Aachs%26aulast%3DKusters%26aufirst%3DJ.%2BG.%26aulast%3Dvan%2BVliet%26aufirst%3DA.%2BH.%2BM.%26aulast%3DKuipers%26aufirst%3DE.%2BJ.%26atitle%3DPathogenesis%2520of%2520Helicobacter%2520pylori%2520infection%26jtitle%3DClin.%2520Microbiol.%2520Rev.%26date%3D2006%26volume%3D19%26spage%3D449%26epage%3D490%26doi%3D10.1128%2Fcmr.00054-05" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de
Martel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georges, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bray, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferlay, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clifford, G. M.</span></span> <span> </span><span class="NLM_article-title">Global burden of cancer attributable to infections in 2018: a worldwide incidence analysis</span>. <i>Lancet Global Health</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">e180</span>– <span class="NLM_lpage">e190</span>, <span class="refDoi"> DOI: 10.1016/s2214-109x(19)30488-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1016%2FS2214-109X%2819%2930488-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=31862245" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2020&pages=e180-e190&author=C.+de%0AMartelauthor=D.+Georgesauthor=F.+Brayauthor=J.+Ferlayauthor=G.+M.+Clifford&title=Global+burden+of+cancer+attributable+to+infections+in+2018%3A+a+worldwide+incidence+analysis&doi=10.1016%2Fs2214-109x%2819%2930488-7"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2FS2214-109X%2819%2930488-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS2214-109X%252819%252930488-7%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BMartel%26aufirst%3DC.%26aulast%3DGeorges%26aufirst%3DD.%26aulast%3DBray%26aufirst%3DF.%26aulast%3DFerlay%26aufirst%3DJ.%26aulast%3DClifford%26aufirst%3DG.%2BM.%26atitle%3DGlobal%2520burden%2520of%2520cancer%2520attributable%2520to%2520infections%2520in%25202018%253A%2520a%2520worldwide%2520incidence%2520analysis%26jtitle%3DLancet%2520Global%2520Health%26date%3D2020%26volume%3D8%26spage%3De180%26epage%3De190%26doi%3D10.1016%2Fs2214-109x%2819%2930488-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fallone, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malfertheiner, P.</span></span> <span> </span><span class="NLM_article-title">Reconciliation of recent <i>Helicobacter pylori</i> treatment guidelines in a time of Increasing resistance to antibiotics</span>. <i>Gastroenterology</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>157</i></span>,  <span class="NLM_fpage">44</span>– <span class="NLM_lpage">53</span>, <span class="refDoi"> DOI: 10.1053/j.gastro.2019.04.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1053%2Fj.gastro.2019.04.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=30998990" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A280%3ADC%252BB3M%252FmvVanuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=2019&pages=44-53&author=C.+A.+Falloneauthor=S.+F.+Mossauthor=P.+Malfertheiner&title=Reconciliation+of+recent+Helicobacter+pylori+treatment+guidelines+in+a+time+of+Increasing+resistance+to+antibiotics&doi=10.1053%2Fj.gastro.2019.04.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Reconciliation of Recent Helicobacter pylori Treatment Guidelines in a Time of Increasing Resistance to Antibiotics</span></div><div class="casAuthors">Fallone Carlo A; Moss Steven F; Malfertheiner Peter</div><div class="citationInfo"><span class="NLM_cas:title">Gastroenterology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">44-53</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Increasing resistance to antibiotics worldwide has adverse effects on the effectiveness of standard therapies to eradicate Helicobacter pylori infection.  We reviewed guidelines developed by expert groups in Europe, Canada, and the United States for the treatment of H pylori infection.  We compared the recommendations of these guidelines, reconciled them, and addressed the increasing resistance of H pylori to antibiotic therapy regimens.  The guidelines recommend bismuth quadruple therapy for first-line treatment, replacing clarithromycin-based triple therapy.  There is consensus for concomitant 4-drug therapy as an alternative, especially when bismuth is not available.  When therapy is unsuccessful, it is likely due to resistance to clarithromycin, levofloxacin, and/or metronidazole; these drugs, if used previously, should be avoided in subsequent eradication attempts.  Second-line therapies should be bismuth quadruple therapy or levofloxacin triple therapy, depending on suspected resistance, reserving rifabutin-based triple and high-dose dual amoxicillin proton pump inhibitor therapy for subsequent treatment attempts.  The increasing resistance of H pylori to antibiotic therapy necessitates local availability of susceptibility tests for individuals, and establishment of regional and national monitoring programs to develop evidence-based locally relevant eradication strategies.  Further studies into the development of more easily accessible methods of resistance testing, such as biomarker analysis of stool samples, are required.  Options under investigation include substituting vonoprazan for proton pump inhibitors, adding probiotics, and vaccine development.  Narrow-spectrum antibiotics and new therapeutic targets could be identified based on genomic, proteomic, and metabolomic analyses of H pylori.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT6WfBfFoYhqo3yKPtYViAGfW6udTcc2eZHkyU4pA-4hrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M%252FmvVanuw%253D%253D&md5=43362a80e77044c93801503cafac13bc</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1053%2Fj.gastro.2019.04.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1053%252Fj.gastro.2019.04.011%26sid%3Dliteratum%253Aachs%26aulast%3DFallone%26aufirst%3DC.%2BA.%26aulast%3DMoss%26aufirst%3DS.%2BF.%26aulast%3DMalfertheiner%26aufirst%3DP.%26atitle%3DReconciliation%2520of%2520recent%2520Helicobacter%2520pylori%2520treatment%2520guidelines%2520in%2520a%2520time%2520of%2520Increasing%2520resistance%2520to%2520antibiotics%26jtitle%3DGastroenterology%26date%3D2019%26volume%3D157%26spage%3D44%26epage%3D53%26doi%3D10.1053%2Fj.gastro.2019.04.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ruiter, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wunderink, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veenendaal, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Visser, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Boer, M. G. J.</span></span> <span> </span><span class="NLM_article-title"><i>Helicobacter pylori</i> resistance in the Netherlands: a growing problem?</span>. <i>Neth. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">394</span>– <span class="NLM_lpage">398</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=29219812" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A280%3ADC%252BC1M3psVSgsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2017&pages=394-398&author=R.+Ruiterauthor=H.+F.+Wunderinkauthor=R.+A.+Veenendaalauthor=L.+G.+Visserauthor=M.+G.+J.+de+Boer&title=Helicobacter+pylori+resistance+in+the+Netherlands%3A+a+growing+problem%3F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Helicobacter pylori resistance in the Netherlands: a growing problem?</span></div><div class="casAuthors">Ruiter R; Wunderink H F; Veenendaal R A; Visser L G; de Boer M G J</div><div class="citationInfo"><span class="NLM_cas:title">The Netherlands journal of medicine</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">394-398</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Helicobacter pylori infection is clinically associated with dyspepsia, gastric and duodenal ulcers, and gastric cancer.  Increasing antimicrobial resistance in H. pylori is a worldwide problem and failure of eradication with standard triple therapy (high-dose proton pump inhibition, amoxicillin and clarithromycin) is directly related to the presence of a resistant strain.  Other treatment combinations have been investigated, but with inconsistent results.  Based on a review of the recent literature in conjunction with an analysis of the regional resistance data, we address the increasing complexity of H. pylori eradication therapy.  Culture and susceptibility results of all first H. pylori isolates of adults (> 18 years) seen in the Leiden University Medical Center, from January 2006 to December 2015, were analysed (n = 707).  An increase in clarithromycin resistance was observed from 9.8% to 18.1% (p = 0.002) in the periods from 2006-2010 and 2011-2015, respectively.  For ampicillin the resistance increased from 6.3% to 10.0% (p = 0.37), and for metronidazole from 20.7% to 23.2% (p = 0.42).  The tetracycline resistance remained low at 3.2% and 2.3%, respectively.  The treatment paradigm is shifting towards individualised treatment rather than a one-strategy-fits-all approach.  In case of treatment failure it should be strongly considered to refer a patient for endoscopy, biopsy and culture.  Thereafter, targeted antimicrobial treatment based on susceptibility results can be initiated.  Furthermore, accumulating data indicate that prolongation of treatment to 14 days, as opposed to the current standard 7 day course, contributes to a higher H. pylori eradication rate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQOV9IT3Mt4h8vLc4ARHEmzfW6udTcc2eYl6uQxW-xHaLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M3psVSgsg%253D%253D&md5=d3d5dc09136da3f3bea141708b5cc56e</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRuiter%26aufirst%3DR.%26aulast%3DWunderink%26aufirst%3DH.%2BF.%26aulast%3DVeenendaal%26aufirst%3DR.%2BA.%26aulast%3DVisser%26aufirst%3DL.%2BG.%26aulast%3Dde%2BBoer%26aufirst%3DM.%2BG.%2BJ.%26atitle%3DHelicobacter%2520pylori%2520resistance%2520in%2520the%2520Netherlands%253A%2520a%2520growing%2520problem%253F%26jtitle%3DNeth.%2520J.%2520Med.%26date%3D2017%26volume%3D75%26spage%3D394%26epage%3D398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Savoldi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrara, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tacconelli, E.</span></span> <span> </span><span class="NLM_article-title">Prevalence of antibiotic resistance in <i>Helicobacter pylori</i>: A systematic review and meta-analysis in World Health Organization regions</span>. <i>Gastroenterology</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>155</i></span>,  <span class="NLM_fpage">1372</span>– <span class="NLM_lpage">1382</span>, <span class="refDoi"> DOI: 10.1053/j.gastro.2018.07.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1053%2Fj.gastro.2018.07.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=29990487" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A280%3ADC%252BB3c%252FmsFOmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2018&pages=1372-1382&author=A.+Savoldiauthor=E.+Carraraauthor=D.+Y.+Grahamauthor=M.+Contiauthor=E.+Tacconelli&title=Prevalence+of+antibiotic+resistance+in+Helicobacter+pylori%3A+A+systematic+review+and+meta-analysis+in+World+Health+Organization+regions&doi=10.1053%2Fj.gastro.2018.07.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Prevalence of Antibiotic Resistance in Helicobacter pylori: A Systematic Review and Meta-analysis in World Health Organization Regions</span></div><div class="casAuthors">Savoldi Alessia; Carrara Elena; Conti Michela; Graham David Y; Tacconelli Evelina</div><div class="citationInfo"><span class="NLM_cas:title">Gastroenterology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">155</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1372-1382.e17</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND & AIMS:  In 2017, the World Health Organization (WHO) designated clarithromycin-resistant Helicobacter pylori a high priority for antibiotic research and development.  However, there are no clear data on the global distribution of resistance or its clinical effects.  We performed a systematic review and meta-analysis to assess the distribution of H pylori resistance to commonly used antibiotics and to measure the association between antibiotic resistance and treatment failure.  METHODS:  We searched publication databases for studies that assessed rates of H pylori resistance to clarithromycin, metronidazole, levofloxacin, amoxicillin, or tetracycline.  Pooled estimates of primary and secondary resistance and 95% confidence intervals (CIs) were grouped by WHO region.  The association between antibiotic resistance and treatment failure was measured by extracting data on treatment efficacy in patients with resistant and susceptible isolates and pooling odds ratios with 95% CIs.  RESULTS:  We identified 178 studies, comprising 66,142 isolates from 65 countries.  Primary and secondary resistance rates to clarithromycin, metronidazole, and levofloxacin were ≥15% in all WHO regions, except primary clarithromycin resistance in the Americas (10%; 95% CI, 4%-16%) and South-East Asia region (10%; 95% CI, 5%-16%) and primary levofloxacin resistance in the European region (11%; 95% CI, 9%-13%).  There was considerable heterogeneity (I(2) > 75%) among all analyses-this might have resulted from the grouping of resistance rates by country.  Increasing antibiotic resistance was observed in most WHO regions.  Resistance to clarithromycin was significantly associated with failure of clarithromycin-containing regimens (odds ratio, 6.97; 95% CI, 5.23-9.28; P < .001).  CONCLUSIONS:  Resistance of H pylori to antibiotics has reached alarming levels worldwide, which has a great effect on efficacy of treatment.  Local surveillance networks are required to select appropriate eradication regimens for each region.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdMPUw-TFMkOWhZTslLrfffW6udTcc2eYl6uQxW-xHaLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c%252FmsFOmsA%253D%253D&md5=196e04c7bb5b3e5b75d2825ead30c5c8</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1053%2Fj.gastro.2018.07.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1053%252Fj.gastro.2018.07.007%26sid%3Dliteratum%253Aachs%26aulast%3DSavoldi%26aufirst%3DA.%26aulast%3DCarrara%26aufirst%3DE.%26aulast%3DGraham%26aufirst%3DD.%2BY.%26aulast%3DConti%26aufirst%3DM.%26aulast%3DTacconelli%26aufirst%3DE.%26atitle%3DPrevalence%2520of%2520antibiotic%2520resistance%2520in%2520Helicobacter%2520pylori%253A%2520A%2520systematic%2520review%2520and%2520meta-analysis%2520in%2520World%2520Health%2520Organization%2520regions%26jtitle%3DGastroenterology%26date%3D2018%26volume%3D155%26spage%3D1372%26epage%3D1382%26doi%3D10.1053%2Fj.gastro.2018.07.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiotani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, D. Y.</span></span> <span> </span><span class="NLM_article-title">Current and future treatment of <i>Helicobacter pylori</i> infections</span>. <i>Adv. Exp. Med. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>1149</i></span>,  <span class="NLM_fpage">211</span>– <span class="NLM_lpage">225</span>, <span class="refDoi"> DOI: 10.1007/5584_2019_367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1007%2F5584_2019_367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=31016626" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BB3cXosFGmtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1149&publication_year=2019&pages=211-225&author=H.+Matsumotoauthor=A.+Shiotaniauthor=D.+Y.+Graham&title=Current+and+future+treatment+of+Helicobacter+pylori+infections&doi=10.1007%2F5584_2019_367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Current and Future Treatment of Helicobacter pylori Infections</span></div><div class="casAuthors">Matsumoto, Hiroshi; Shiotani, Akiko; Graham, David Y.</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Experimental Medicine and Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">1149</span>
        (<span class="NLM_cas:issue">Helicobacter pylori in Human Diseases: Advances in Microbiology, Infectious Diseases and Public Health, Volume 11</span>),
    <span class="NLM_cas:pages">211-225</span>CODEN:
                <span class="NLM_cas:coden">AEMBAP</span>;
        ISSN:<span class="NLM_cas:issn">2214-8019</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Helicobacter pylori is one of the most common human pathogens and it has been estd. that about 50% of the world's population is currently infected.  The present consensus is that, unless there are compelling reasons, all H. pylori infections should be cured.  Since the 1990s, different national and international guidelines for the management of H. pylori -related diseases have been published and periodically updated regarding indications for treatment, diagnostic procedures, and preferred treatment regimens.  Most guidelines provide sophisticated meta-analyses examg. the outcome of different regimens done in regions with variable, often high rates of resistance to antibiotics, for which the prevalence and effects of resistance was often ignored.  Although successful antimicrobial therapy must be susceptibility-based, increasing antimicrobial resistance and general unavailability of susceptibility testing have required clinicians to generally rely on empiric regimens.  Antibiotics resistance of H. pylori has reached alarming high levels worldwide, which has an effect to efficacy of treatment.  The recommendations should provide regimes for multi-resistant infections or for those where susceptibility testing is unavailable or refused.  The first rule is to use only proven locally effective therapies.  Because of patient intolerances, drug allergies, and local experiences, the clinicians should have at least two options for first-line therapy.  As with any antimicrobial therapy, a thorough review of prior antibiotic use is invaluable to identify the presence of probably resistance.  The second key is patient education regarding potential and expected side-effects and the importance of completing the course of antibiotics.  We also review here triple therapies, sequential-concomitant, hybrid therapies, bismuth therapies, dual therapy, vonoprazan, modern antibiotic treatments, probiotics and vaccination.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9WdssYI98C7Vg90H21EOLACvtfcHk0lj8i2uChDJerA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXosFGmtr8%253D&md5=0bb9d394c1a9c4b0073871d0094cb814</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1007%2F5584_2019_367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F5584_2019_367%26sid%3Dliteratum%253Aachs%26aulast%3DMatsumoto%26aufirst%3DH.%26aulast%3DShiotani%26aufirst%3DA.%26aulast%3DGraham%26aufirst%3DD.%2BY.%26atitle%3DCurrent%2520and%2520future%2520treatment%2520of%2520Helicobacter%2520pylori%2520infections%26jtitle%3DAdv.%2520Exp.%2520Med.%2520Biol.%26date%3D2019%26volume%3D1149%26spage%3D211%26epage%3D225%26doi%3D10.1007%2F5584_2019_367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tacconelli, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrara, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savoldi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harbarth, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendelson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monnet, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pulcini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahlmeter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kluytmans, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmeli, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouellette, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Outterson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavaleri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houchens, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grayson, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theuretzbacher, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magrini, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aboderin, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Abri, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Awang Jalil, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benzonana, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattacharya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brink, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burkert, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cars, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornaglia, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dyar, O. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedrich, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gales, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gandra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giske, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goff, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goossens, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gottlieb, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzman Blanco, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hryniewicz, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kattula, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jinks, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanj, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerr, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kieny, M.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozlov, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labarca, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laxminarayan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leder, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leibovici, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy-Hara, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Littman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malhotra-Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manchanda, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moja, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ndoye, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paterson, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramon-Pardo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez-Baño, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanguinetti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sengupta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharland, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Si-Mehand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silver, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinbakk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomsen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thwaites, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Meer, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Kinh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vega, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villegas, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wechsler-Fördös, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wertheim, H. F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wesangula, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodford, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yilmaz, F. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zorzet, A.</span></span> <span> </span><span class="NLM_article-title">Discovery, research, and development of new antibiotics: the WHO priority list of antibiotic-resistant bacteria and tuberculosis</span>. <i>Lancet Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">318</span>– <span class="NLM_lpage">327</span>, <span class="refDoi"> DOI: 10.1016/s1473-3099(17)30753-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1016%2FS1473-3099%2817%2930753-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=29276051" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A280%3ADC%252BC1MzkvFOgsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=318-327&author=E.+Tacconelliauthor=E.+Carraraauthor=A.+Savoldiauthor=S.+Harbarthauthor=M.+Mendelsonauthor=D.+L.+Monnetauthor=C.+Pulciniauthor=G.+Kahlmeterauthor=J.+Kluytmansauthor=Y.+Carmeliauthor=M.+Ouelletteauthor=K.+Outtersonauthor=J.+Patelauthor=M.+Cavaleriauthor=E.+M.+Coxauthor=C.+R.+Houchensauthor=M.+L.+Graysonauthor=P.+Hansenauthor=N.+Singhauthor=U.+Theuretzbacherauthor=N.+Magriniauthor=A.+O.+Aboderinauthor=S.+S.+Al-Abriauthor=N.+Awang+Jalilauthor=N.+Benzonanaauthor=S.+Bhattacharyaauthor=A.+J.+Brinkauthor=F.+R.+Burkertauthor=O.+Carsauthor=G.+Cornagliaauthor=O.+J.+Dyarauthor=A.+W.+Friedrichauthor=A.+C.+Galesauthor=S.+Gandraauthor=C.+G.+Giskeauthor=D.+A.+Goffauthor=H.+Goossensauthor=T.+Gottliebauthor=M.+Guzman+Blancoauthor=W.+Hryniewiczauthor=D.+Kattulaauthor=T.+Jinksauthor=S.+S.+Kanjauthor=L.+Kerrauthor=M.-P.+Kienyauthor=Y.+S.+Kimauthor=R.+S.+Kozlovauthor=J.+Labarcaauthor=R.+Laxminarayanauthor=K.+Lederauthor=L.+Leiboviciauthor=G.+Levy-Haraauthor=J.+Littmanauthor=S.+Malhotra-Kumarauthor=V.+Manchandaauthor=L.+Mojaauthor=B.+Ndoyeauthor=A.+Panauthor=D.+L.+Patersonauthor=M.+Paulauthor=H.+Qiuauthor=P.+Ramon-Pardoauthor=J.+Rodr%C3%ADguez-Ba%C3%B1oauthor=M.+Sanguinettiauthor=S.+Senguptaauthor=M.+Sharlandauthor=M.+Si-Mehandauthor=L.+L.+Silverauthor=W.+Songauthor=M.+Steinbakkauthor=J.+Thomsenauthor=G.+E.+Thwaitesauthor=J.+W.+van+der+Meerauthor=N.+Van+Kinhauthor=S.+Vegaauthor=M.+V.+Villegasauthor=A.+Wechsler-F%C3%B6rd%C3%B6sauthor=H.+F.+L.+Wertheimauthor=E.+Wesangulaauthor=N.+Woodfordauthor=F.+O.+Yilmazauthor=A.+Zorzet&title=Discovery%2C+research%2C+and+development+of+new+antibiotics%3A+the+WHO+priority+list+of+antibiotic-resistant+bacteria+and+tuberculosis&doi=10.1016%2Fs1473-3099%2817%2930753-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery, research, and development of new antibiotics: the WHO priority list of antibiotic-resistant bacteria and tuberculosis</span></div><div class="casAuthors">Tacconelli Evelina; Carrara Elena; Savoldi Alessia; Harbarth Stephan; Mendelson Marc; Monnet Dominique L; Pulcini Celine; Kahlmeter Gunnar; Kluytmans Jan; Carmeli Yehuda; Ouellette Marc; Outterson Kevin; Patel Jean; Cavaleri Marco; Cox Edward M; Houchens Chris R; Grayson M Lindsay; Hansen Paul; Singh Nalini; Theuretzbacher Ursula; Magrini Nicola</div><div class="citationInfo"><span class="NLM_cas:title">The Lancet. Infectious diseases</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">318-327</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  The spread of antibiotic-resistant bacteria poses a substantial threat to morbidity and mortality worldwide.  Due to its large public health and societal implications, multidrug-resistant tuberculosis has been long regarded by WHO as a global priority for investment in new drugs.  In 2016, WHO was requested by member states to create a priority list of other antibiotic-resistant bacteria to support research and development of effective drugs.  METHODS:  We used a multicriteria decision analysis method to prioritise antibiotic-resistant bacteria; this method involved the identification of relevant criteria to assess priority against which each antibiotic-resistant bacterium was rated.  The final priority ranking of the antibiotic-resistant bacteria was established after a preference-based survey was used to obtain expert weighting of criteria.  FINDINGS:  We selected 20 bacterial species with 25 patterns of acquired resistance and ten criteria to assess priority: mortality, health-care burden, community burden, prevalence of resistance, 10-year trend of resistance, transmissibility, preventability in the community setting, preventability in the health-care setting, treatability, and pipeline.  We stratified the priority list into three tiers (critical, high, and medium priority), using the 33rd percentile of the bacterium's total scores as the cutoff.  Critical-priority bacteria included carbapenem-resistant Acinetobacter baumannii and Pseudomonas aeruginosa, and carbapenem-resistant and third-generation cephalosporin-resistant Enterobacteriaceae.  The highest ranked Gram-positive bacteria (high priority) were vancomycin-resistant Enterococcus faecium and meticillin-resistant Staphylococcus aureus.  Of the bacteria typically responsible for community-acquired infections, clarithromycin-resistant Helicobacter pylori, and fluoroquinolone-resistant Campylobacter spp, Neisseria gonorrhoeae, and Salmonella typhi were included in the high-priority tier.  INTERPRETATION:  Future development strategies should focus on antibiotics that are active against multidrug-resistant tuberculosis and Gram-negative bacteria.  The global strategy should include antibiotic-resistant bacteria responsible for community-acquired infections such as Salmonella spp, Campylobacter spp, N gonorrhoeae, and H pylori.  FUNDING:  World Health Organization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSkARC7DR8uZLGbPKh4T9eAfW6udTcc2eYwHuVxhFYnr7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MzkvFOgsA%253D%253D&md5=14021ec57670b29fec6a88226a6a1fdd</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2FS1473-3099%2817%2930753-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1473-3099%252817%252930753-3%26sid%3Dliteratum%253Aachs%26aulast%3DTacconelli%26aufirst%3DE.%26aulast%3DCarrara%26aufirst%3DE.%26aulast%3DSavoldi%26aufirst%3DA.%26aulast%3DHarbarth%26aufirst%3DS.%26aulast%3DMendelson%26aufirst%3DM.%26aulast%3DMonnet%26aufirst%3DD.%2BL.%26aulast%3DPulcini%26aufirst%3DC.%26aulast%3DKahlmeter%26aufirst%3DG.%26aulast%3DKluytmans%26aufirst%3DJ.%26aulast%3DCarmeli%26aufirst%3DY.%26aulast%3DOuellette%26aufirst%3DM.%26aulast%3DOutterson%26aufirst%3DK.%26aulast%3DPatel%26aufirst%3DJ.%26aulast%3DCavaleri%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DE.%2BM.%26aulast%3DHouchens%26aufirst%3DC.%2BR.%26aulast%3DGrayson%26aufirst%3DM.%2BL.%26aulast%3DHansen%26aufirst%3DP.%26aulast%3DSingh%26aufirst%3DN.%26aulast%3DTheuretzbacher%26aufirst%3DU.%26aulast%3DMagrini%26aufirst%3DN.%26aulast%3DAboderin%26aufirst%3DA.%2BO.%26aulast%3DAl-Abri%26aufirst%3DS.%2BS.%26aulast%3DAwang%2BJalil%26aufirst%3DN.%26aulast%3DBenzonana%26aufirst%3DN.%26aulast%3DBhattacharya%26aufirst%3DS.%26aulast%3DBrink%26aufirst%3DA.%2BJ.%26aulast%3DBurkert%26aufirst%3DF.%2BR.%26aulast%3DCars%26aufirst%3DO.%26aulast%3DCornaglia%26aufirst%3DG.%26aulast%3DDyar%26aufirst%3DO.%2BJ.%26aulast%3DFriedrich%26aufirst%3DA.%2BW.%26aulast%3DGales%26aufirst%3DA.%2BC.%26aulast%3DGandra%26aufirst%3DS.%26aulast%3DGiske%26aufirst%3DC.%2BG.%26aulast%3DGoff%26aufirst%3DD.%2BA.%26aulast%3DGoossens%26aufirst%3DH.%26aulast%3DGottlieb%26aufirst%3DT.%26aulast%3DGuzman%2BBlanco%26aufirst%3DM.%26aulast%3DHryniewicz%26aufirst%3DW.%26aulast%3DKattula%26aufirst%3DD.%26aulast%3DJinks%26aufirst%3DT.%26aulast%3DKanj%26aufirst%3DS.%2BS.%26aulast%3DKerr%26aufirst%3DL.%26aulast%3DKieny%26aufirst%3DM.-P.%26aulast%3DKim%26aufirst%3DY.%2BS.%26aulast%3DKozlov%26aufirst%3DR.%2BS.%26aulast%3DLabarca%26aufirst%3DJ.%26aulast%3DLaxminarayan%26aufirst%3DR.%26aulast%3DLeder%26aufirst%3DK.%26aulast%3DLeibovici%26aufirst%3DL.%26aulast%3DLevy-Hara%26aufirst%3DG.%26aulast%3DLittman%26aufirst%3DJ.%26aulast%3DMalhotra-Kumar%26aufirst%3DS.%26aulast%3DManchanda%26aufirst%3DV.%26aulast%3DMoja%26aufirst%3DL.%26aulast%3DNdoye%26aufirst%3DB.%26aulast%3DPan%26aufirst%3DA.%26aulast%3DPaterson%26aufirst%3DD.%2BL.%26aulast%3DPaul%26aufirst%3DM.%26aulast%3DQiu%26aufirst%3DH.%26aulast%3DRamon-Pardo%26aufirst%3DP.%26aulast%3DRodr%25C3%25ADguez-Ba%25C3%25B1o%26aufirst%3DJ.%26aulast%3DSanguinetti%26aufirst%3DM.%26aulast%3DSengupta%26aufirst%3DS.%26aulast%3DSharland%26aufirst%3DM.%26aulast%3DSi-Mehand%26aufirst%3DM.%26aulast%3DSilver%26aufirst%3DL.%2BL.%26aulast%3DSong%26aufirst%3DW.%26aulast%3DSteinbakk%26aufirst%3DM.%26aulast%3DThomsen%26aufirst%3DJ.%26aulast%3DThwaites%26aufirst%3DG.%2BE.%26aulast%3Dvan%2Bder%2BMeer%26aufirst%3DJ.%2BW.%26aulast%3DVan%2BKinh%26aufirst%3DN.%26aulast%3DVega%26aufirst%3DS.%26aulast%3DVillegas%26aufirst%3DM.%2BV.%26aulast%3DWechsler-F%25C3%25B6rd%25C3%25B6s%26aufirst%3DA.%26aulast%3DWertheim%26aufirst%3DH.%2BF.%2BL.%26aulast%3DWesangula%26aufirst%3DE.%26aulast%3DWoodford%26aufirst%3DN.%26aulast%3DYilmaz%26aufirst%3DF.%2BO.%26aulast%3DZorzet%26aufirst%3DA.%26atitle%3DDiscovery%252C%2520research%252C%2520and%2520development%2520of%2520new%2520antibiotics%253A%2520the%2520WHO%2520priority%2520list%2520of%2520antibiotic-resistant%2520bacteria%2520and%2520tuberculosis%26jtitle%3DLancet%2520Infect.%2520Dis.%26date%3D2018%26volume%3D18%26spage%3D318%26epage%3D327%26doi%3D10.1016%2Fs1473-3099%2817%2930753-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alm, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trust, T. J.</span></span> <span> </span><span class="NLM_article-title">Analysis of the genetic diversity of <i>Helicobacter pylori</i>: the tale of two genomes</span>. <i>J. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">834</span>– <span class="NLM_lpage">846</span>, <span class="refDoi"> DOI: 10.1007/s001099900067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1007%2Fs001099900067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10682319" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BD3cXht1alu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=1999&pages=834-846&author=R.+A.+Almauthor=T.+J.+Trust&title=Analysis+of+the+genetic+diversity+of+Helicobacter+pylori%3A+the+tale+of+two+genomes&doi=10.1007%2Fs001099900067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of the genetic diversity of Helicobacter pylori: the tale of two genomes</span></div><div class="casAuthors">Alm, Richard A.; Trust, Trevor J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Medicine (Berlin)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">834-846</span>CODEN:
                <span class="NLM_cas:coden">JMLME8</span>;
        ISSN:<span class="NLM_cas:issn">0946-2716</span>.
    
            (<span class="NLM_cas:orgname">Springer-Verlag</span>)
        </div><div class="casAbstract">Infection with Helicobacter pylori has been linked to numerous severe gastroduodenal diseases including peptic ulcer and gastric cancer.  Several techniques have been used to measure the genetic heterogeneity of H. pylori at several different levels and to det. whether there is any correlation with severity of disease.  The availability of two completed genome sequences from unrelated strains (J99 and 26695) has allowed an anal. of the level of diversity from a large-scale yet detailed perspective.  Although the two chromosomes are organized differently in a limited no. of discrete regions, the genome size and gene order of these two "high-virulence" (cagA+ and vacA+) H. pylori isolates was found to be highly similar.  The regions of organizational difference are assocd. with insertion sequences, DNA restriction/modification genes, repeat sequences, or a combination of the above.  A significant level of variation at the nucleotide level is seen across the genome, providing an explanation for why the nucleotide-based typing techniques have such high discriminatory power among independent H. pylori isolates.  This nucleotide variation together with the organizational rearrangements appears to have provided an over-estn. of the gene order diversity of H. pylori as assessed by pulse-field gel electrophoresis.  Functional assignments are assigned to approx. only 60% of the gene products in each strain, with one-half of the remaining gene products of unknown function having homologues in other bacteria, while the remainder appear to be H. pylori-specific.  Between 6% and 7% of the coding capacity of each strain are genes that are absent from the other strain, with almost one-half of these strain-specific genes located in a single hypervariable region called the plasticity zone.  The majority of the strain-specific genes in each strain are also H. pylori-specific, with no homologues being identified in the public databases.  Significantly, over one-half of the functionally assigned strain-specific genes in both H. pylori J99 and 26695 encode DNA restriction/modification enzymes.  Anal. of the level of conservation between orthologues from the two strains indicates that the H. pylori specific genes have a lower level of conservation than those orthologues to which a putative function can be assigned.  The plasticity zone represents one of several regions across each genome that is comprised of lower (G+C)% content DNA, some of which has been detected in self-replicating plasmids, suggesting that both horizontal transfer from other species and plasmid integration are responsible for the strain-specific diversity at this locus.  These analyses have yielded results with important implications for understanding the genetic diversity of H. pylori and its assocd. diseases, and imply a need to reassess the resp. roles of bacterial and host factors in H. pylori assocd. diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNZS65Up1rmbVg90H21EOLACvtfcHk0ljukgQPAneHJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXht1alu7g%253D&md5=88bed741cf5e4de688dc3ba1a4874e60</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1007%2Fs001099900067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs001099900067%26sid%3Dliteratum%253Aachs%26aulast%3DAlm%26aufirst%3DR.%2BA.%26aulast%3DTrust%26aufirst%3DT.%2BJ.%26atitle%3DAnalysis%2520of%2520the%2520genetic%2520diversity%2520of%2520Helicobacter%2520pylori%253A%2520the%2520tale%2520of%2520two%2520genomes%26jtitle%3DJ.%2520Mol.%2520Med.%26date%3D1999%26volume%3D77%26spage%3D834%26epage%3D846%26doi%3D10.1007%2Fs001099900067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doig, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jonge, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alm, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uria-Nickelsen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noonan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummino, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guild, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moir, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vovis, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trust, T. J.</span></span> <span> </span><span class="NLM_article-title"><i>Helicobacter pylori</i> physiology predicted from genomic comparison of two strains</span>. <i>Microbiol. Mol. Biol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">675</span>– <span class="NLM_lpage">707</span>, <span class="refDoi"> DOI: 10.1128/mmbr.63.3.675-707.1999</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1128%2FMMBR.63.3.675-707.1999" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10477312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADyaK1MXmt1CgsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=1999&pages=675-707&author=P.+Doigauthor=B.+L.+de+Jongeauthor=R.+A.+Almauthor=E.+D.+Brownauthor=M.+Uria-Nickelsenauthor=B.+Noonanauthor=S.+D.+Millsauthor=P.+Tumminoauthor=G.+Carmelauthor=B.+C.+Guildauthor=D.+T.+Moirauthor=G.+F.+Vovisauthor=T.+J.+Trust&title=Helicobacter+pylori+physiology+predicted+from+genomic+comparison+of+two+strains&doi=10.1128%2Fmmbr.63.3.675-707.1999"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Helicobacter pylori physiology predicted from genomic comparison of two strains</span></div><div class="casAuthors">Doig, Peter; De Jonge, Boudewijn L.; Alm, Richard A.; Brown, Eric D.; Uria-Nickelsen, Maria; Noonan, Brian; Mills, Scott D.; Tummino, Peter; Carmel, Gilles; Guild, Braydon C.; Moir, Donald T.; Vovis, Gerald F.; Trust, Trevor J.</div><div class="citationInfo"><span class="NLM_cas:title">Microbiology and Molecular Biology Reviews</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">675-707</span>CODEN:
                <span class="NLM_cas:coden">MMBRF7</span>;
        ISSN:<span class="NLM_cas:issn">1092-2172</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A review with 160 refs.  Helicobacter pylori is a gram-neg. bacteria which colonizes the gastric mucosa of humans and is implicated in a wide range of gastroduodenal diseases.  This paper reviews the physiol. of this bacterium as predicted from the sequenced genomes of two unrelated strains and reconciles these predictions with the literature.  In general, the predicted capabilities are in good agreement with reported exptl. observations.  H. pylori is limited in carbohydrate utilization and will use amino acids, for which it has transporter systems, as sources of carbon.  Energy can be generated by fermn., and the bacterium possesses components necessary for both aerobic and anaerobic respiration.  Sulfur metab. is limited, whereas nitrogen metab. is extensive.  There is active uptake of DNA via transformation and ample restriction-modification activities.  The cell contains numerous outer membrane proteins, some of which are porins or involved in iron uptake.  Some of these outer membrane proteins and the lipopolysaccharide may be regulated by a slipped-strand repair mechanism which probably results in phase variation and plays a role in colonization.  In contrast to a commonly held belief that H. pylori is a very diverse species, few differences were predicted in the physiol. of these two unrelated strains, indicating that host and environmental factors probably play a significant role in the outcome of H. pylori-related disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobMBLuOtLulbVg90H21EOLACvtfcHk0ljf6DBBf4m7JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmt1CgsLg%253D&md5=47b3a0e3eec3eaea3bf8f3ff869e84dc</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1128%2FMMBR.63.3.675-707.1999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMMBR.63.3.675-707.1999%26sid%3Dliteratum%253Aachs%26aulast%3DDoig%26aufirst%3DP.%26aulast%3Dde%2BJonge%26aufirst%3DB.%2BL.%26aulast%3DAlm%26aufirst%3DR.%2BA.%26aulast%3DBrown%26aufirst%3DE.%2BD.%26aulast%3DUria-Nickelsen%26aufirst%3DM.%26aulast%3DNoonan%26aufirst%3DB.%26aulast%3DMills%26aufirst%3DS.%2BD.%26aulast%3DTummino%26aufirst%3DP.%26aulast%3DCarmel%26aufirst%3DG.%26aulast%3DGuild%26aufirst%3DB.%2BC.%26aulast%3DMoir%26aufirst%3DD.%2BT.%26aulast%3DVovis%26aufirst%3DG.%2BF.%26aulast%3DTrust%26aufirst%3DT.%2BJ.%26atitle%3DHelicobacter%2520pylori%2520physiology%2520predicted%2520from%2520genomic%2520comparison%2520of%2520two%2520strains%26jtitle%3DMicrobiol.%2520Mol.%2520Biol.%2520Rev.%26date%3D1999%26volume%3D63%26spage%3D675%26epage%3D707%26doi%3D10.1128%2Fmmbr.63.3.675-707.1999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mendz, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jimenez, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hazell, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gero, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Sullivan, W. J.</span></span> <span> </span><span class="NLM_article-title">Salvage synthesis of purine nucleotides by <i>Helicobacter pylori</i></span>. <i>J. Appl. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">674</span>– <span class="NLM_lpage">681</span>, <span class="refDoi"> DOI: 10.1111/j.1365-2672.1994.tb02818.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1111%2Fj.1365-2672.1994.tb02818.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=7822226" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADyaK2MXjtV2nt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=1994&pages=674-681&author=G.+L.+Mendzauthor=B.+M.+Jimenezauthor=S.+L.+Hazellauthor=A.+M.+Geroauthor=W.+J.+O%E2%80%99Sullivan&title=Salvage+synthesis+of+purine+nucleotides+by+Helicobacter+pylori&doi=10.1111%2Fj.1365-2672.1994.tb02818.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Salvage synthesis of purine nucleotides by Helicobacter pylori</span></div><div class="casAuthors">Mendz, G.L.; Jimenez, B.M.; Hazell, S.L.; Gero, A.M.; O'Sullivan, W.J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Bacteriology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">674-81</span>CODEN:
                <span class="NLM_cas:coden">JABAA4</span>;
        ISSN:<span class="NLM_cas:issn">0021-8847</span>.
    </div><div class="casAbstract">The incorporation of purine nucleotide precursors into Helicobacter pylori and the activities of enzymes involved in nucleotide salvage biosynthetic pathways were investigated by radioactive tracer anal. and NMR spectroscopy.  The organism took up the nucleobases adenine, guanine and hypoxanthine, and the nucleosides adenosine, guanosine and deoxyadenosine.  Any incorporation of deoxyguanosine by the cells was below the detection limits of the methods employed.  The activities of adenine-, guanine- and hypoxanthine-phosphoribosyltransferases were established.  The bacterium showed high levels of adenosine and guanosine nucleosidase activities and lesser activity for deoxyadenosine; no hydrolysis of deoxyguanosine was detected.  Phosphorylase activities were not obsd. with any of the nucleosides.  Phosphotransferase activities with similar rates were demonstrated for adenosine, guanosine and deoxyadenosine; and a weaker activity was detected for deoxyguanosine.  No nucleoside kinase activities were obsd. with any of the nucleosides.  The presence of adenylate kinase was established, but no guanylate kinase activity was obsd.  The study provided evidence for the presence in H. pylori of salvage pathways for the biosynthesis of purine nucleotides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeDXkLLAPIVrVg90H21EOLACvtfcHk0ljf6DBBf4m7JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXjtV2nt7Y%253D&md5=b29745361f8364ede6bd1b4d8d59d81a</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2672.1994.tb02818.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2672.1994.tb02818.x%26sid%3Dliteratum%253Aachs%26aulast%3DMendz%26aufirst%3DG.%2BL.%26aulast%3DJimenez%26aufirst%3DB.%2BM.%26aulast%3DHazell%26aufirst%3DS.%2BL.%26aulast%3DGero%26aufirst%3DA.%2BM.%26aulast%3DO%25E2%2580%2599Sullivan%26aufirst%3DW.%2BJ.%26atitle%3DSalvage%2520synthesis%2520of%2520purine%2520nucleotides%2520by%2520Helicobacter%2520pylori%26jtitle%3DJ.%2520Appl.%2520Bacteriol.%26date%3D1994%26volume%3D77%26spage%3D674%26epage%3D681%26doi%3D10.1111%2Fj.1365-2672.1994.tb02818.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liechti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, J. B.</span></span> <span> </span><span class="NLM_article-title"><i>Helicobacter pylori</i> relies primarily on the purine salvage pathway for purine nucleotide biosynthesis</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>194</i></span>,  <span class="NLM_fpage">839</span>– <span class="NLM_lpage">854</span>, <span class="refDoi"> DOI: 10.1128/jb.05757-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1128%2FJB.05757-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=22194455" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvFKhsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=194&publication_year=2012&pages=839-854&author=G.+Liechtiauthor=J.+B.+Goldberg&title=Helicobacter+pylori+relies+primarily+on+the+purine+salvage+pathway+for+purine+nucleotide+biosynthesis&doi=10.1128%2Fjb.05757-11"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Helicobacter pylori relies primarily on the purine salvage pathway for purine nucleotide biosynthesis</span></div><div class="casAuthors">Liechti, George; Goldberg, Joanna B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">194</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">839-854</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Helicobacter pylori is a chronic colonizer of the gastric epithelium and plays a major role in the development of gastritis, peptic ulcer disease, and gastric cancer.  In its coevolution with humans, the streamlining of the H. pylori genome has resulted in a significant redn. in metabolic pathways, one being purine nucleotide biosynthesis.  Bioinformatic anal. has revealed that H. pylori lacks the enzymic machinery for de novo prodn. of IMP, the first purine nucleotide formed during GTP and ATP biosynthesis.  This suggests that H. pylori must rely heavily on salvage of purines from the environment.  In this study, the authors deleted several genes putatively involved in purine salvage and processing.  The growth and survival of these mutants were analyzed in both nutrient-rich and minimal media, and the results confirmed the presence of a robust purine salvage pathway in H. pylori.  Of the two phosphoribosyltransferase genes found in the H. pylori genome, only gpt appears to be essential, and an Δapt mutant strain was still capable of growth on adenine, suggesting that adenine processing via Apt is not essential.  Deletion of the putative nucleoside phosphorylase gene deoD resulted in an inability of H. pylori to grow on purine nucleosides or the purine base adenine.  The results suggest a purine requirement for growth of H. pylori in std. media, indicating that H. pylori possesses the ability to utilize purines and nucleosides from the environment in the absence of a de novo purine nucleotide biosynthesis pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkwZ7_YPN3JbVg90H21EOLACvtfcHk0ljf6DBBf4m7JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvFKhsrg%253D&md5=bc5df306a59c0ea85f24f22c04de04a0</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1128%2FJB.05757-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJB.05757-11%26sid%3Dliteratum%253Aachs%26aulast%3DLiechti%26aufirst%3DG.%26aulast%3DGoldberg%26aufirst%3DJ.%2BB.%26atitle%3DHelicobacter%2520pylori%2520relies%2520primarily%2520on%2520the%2520purine%2520salvage%2520pathway%2520for%2520purine%2520nucleotide%2520biosynthesis%26jtitle%3DJ.%2520Bacteriol.%26date%3D2012%26volume%3D194%26spage%3D839%26epage%3D854%26doi%3D10.1128%2Fjb.05757-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hedstrom, L.</span></span> <span> </span><span class="NLM_article-title">IMP dehydrogenase: structure, mechanism, and inhibition</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">2903</span>– <span class="NLM_lpage">2928</span>, <span class="refDoi"> DOI: 10.1021/cr900021w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr900021w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmsFamtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2009&pages=2903-2928&author=L.+Hedstrom&title=IMP+dehydrogenase%3A+structure%2C+mechanism%2C+and+inhibition&doi=10.1021%2Fcr900021w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">IMP Dehydrogenase: Structure, Mechanism, and Inhibition</span></div><div class="casAuthors">Hedstrom, Lizbeth</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2903-2928</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The discovery that IMP dehydrogenase (IMPDH) is amplified in tumors and rapidly proliferating tissues provided the foundation for design of drugs targeting this enzyme.  Though yet much success in the cancer arena has not been achieved, IMPDH inhibitors are now widely used in immunosuppressive and antiviral chemotherapy, and IMPDH may also be a target for antimicrobial drugs.  Clin. relevance aside, IMPDH is a fascinating enzyme.  It traverses several conformations while catalyzing two different chem. transformations, utilizing unusual chem. strategies to promote each reaction.  Monovalent cations such as potassium activate IMPDH, possibly by acting as a mol. lubricant to facilitate these conformational changes.  The biol. of IMPDH also displays some surprising twists.  IMPDH binds nucleic acids and is assocd. with polyribosomes, though the physiol. role of this interaction also has not yet been elucidated.  Perhaps most intriguing is the discovery that mutations in IMPDH are assocd. with hereditary retinal disease.  These mutations cluster to a subdomain that is not required for enzymic activity, and the function of this subdomain is currently under debate.  The paper reviews recent work on the biochem. of IMPDH, integrating structure, function, and inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHrolvTNRBmbVg90H21EOLACvtfcHk0ljCONEEn3441A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmsFamtbo%253D&md5=f475f8b409366b48440100770242bccf</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fcr900021w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr900021w%26sid%3Dliteratum%253Aachs%26aulast%3DHedstrom%26aufirst%3DL.%26atitle%3DIMP%2520dehydrogenase%253A%2520structure%252C%2520mechanism%252C%2520and%2520inhibition%26jtitle%3DChem.%2520Rev.%26date%3D2009%26volume%3D109%26spage%3D2903%26epage%3D2928%26doi%3D10.1021%2Fcr900021w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keough, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocková, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janeba, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naesens, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edstein, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chavchich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span> <span> </span><span class="NLM_article-title">Aza-acyclic nucleoside phosphonates containing a second phosphonate group as inhibitors of the human, <i>Plasmodium falciparum</i> and <i>vivax</i> 6-oxopurine phosphoribosyltransferases and their prodrugs as antimalarial agents</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">827</span>– <span class="NLM_lpage">846</span>, <span class="refDoi"> DOI: 10.1021/jm501416t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501416t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVyhsrzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=827-846&author=D.+T.+Keoughauthor=D.+Hockov%C3%A1author=Z.+Janebaauthor=T.-H.+Wangauthor=L.+Naesensauthor=M.+D.+Edsteinauthor=M.+Chavchichauthor=L.+W.+Guddat&title=Aza-acyclic+nucleoside+phosphonates+containing+a+second+phosphonate+group+as+inhibitors+of+the+human%2C+Plasmodium+falciparum+and+vivax+6-oxopurine+phosphoribosyltransferases+and+their+prodrugs+as+antimalarial+agents&doi=10.1021%2Fjm501416t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Aza-acyclic Nucleoside Phosphonates Containing a Second Phosphonate Group As Inhibitors of the Human, Plasmodium falciparum and vivax 6-Oxopurine Phosphoribosyltransferases and Their Prodrugs As Antimalarial Agents</span></div><div class="casAuthors">Keough, Dianne T.; Hockova, Dana; Janeba, Zlatko; Wang, Tzu-Hsuan; Naesens, Lieve; Edstein, Michael D.; Chavchich, Marina; Guddat, Luke W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">827-846</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Hypoxanthine-guanine-[xanthine] phosphoribosyltransferase (HG[X]PRT) is considered an important target for antimalarial chemotherapy as it is the only pathway for the synthesis of the purine nucleoside monophosphates required for DNA/RNA prodn.  Thus, inhibition of this enzyme should result in cessation of replication.  The aza-acyclic nucleoside phosphonates (aza-ANPs) are good inhibitors of Plasmodium falciparum HGXPRT (PfHGXPRT), with Ki values as low as 0.08 and 0.01 μM for Plasmodium vivax HGPRT (PvHGPRT).  Prodrugs of these aza-ANPs exhibit antimalarial activity against Pf lines with IC50 values (0.8-6.0 μM) and have low cytotoxicity against human cells.  Crystal structures of six of these compds. in complex with human HGPRT have been detd.  These suggest that the different affinities of these aza-ANPs could be due to the flexibility of the loops surrounding the active site as well as the flexibility of the inhibitors, allowing them to adapt to fit into three binding pockets of the enzyme(s).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9eXB3S3z3v7Vg90H21EOLACvtfcHk0ljCONEEn3441A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVyhsrzF&md5=c7d1dad69f99a1b4928b2a65fa411670</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm501416t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501416t%26sid%3Dliteratum%253Aachs%26aulast%3DKeough%26aufirst%3DD.%2BT.%26aulast%3DHockov%25C3%25A1%26aufirst%3DD.%26aulast%3DJaneba%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DT.-H.%26aulast%3DNaesens%26aufirst%3DL.%26aulast%3DEdstein%26aufirst%3DM.%2BD.%26aulast%3DChavchich%26aufirst%3DM.%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26atitle%3DAza-acyclic%2520nucleoside%2520phosphonates%2520containing%2520a%2520second%2520phosphonate%2520group%2520as%2520inhibitors%2520of%2520the%2520human%252C%2520Plasmodium%2520falciparum%2520and%2520vivax%25206-oxopurine%2520phosphoribosyltransferases%2520and%2520their%2520prodrugs%2520as%2520antimalarial%2520agents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D827%26epage%3D846%26doi%3D10.1021%2Fjm501416t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keough, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rejman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pohl, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zborníková, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocková, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Croll, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edstein, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birrell, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chavchich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naesens, L. M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierens, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brereton, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span> <span> </span><span class="NLM_article-title">Design of <i>Plasmodium vivax</i> hypoxanthine-guanine phosphoribosyltransferase inhibitors as potential antimalarial therapeutics</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">82</span>– <span class="NLM_lpage">90</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00916</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00916" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=82-90&author=D.+T.+Keoughauthor=D.+Rejmanauthor=R.+Pohlauthor=E.+Zborn%C3%ADkov%C3%A1author=D.+Hockov%C3%A1author=T.+Crollauthor=M.+D.+Edsteinauthor=G.+W.+Birrellauthor=M.+Chavchichauthor=L.+M.+J.+Naesensauthor=G.+K.+Pierensauthor=I.+M.+Breretonauthor=L.+W.+Guddat&title=Design+of+Plasmodium+vivax+hypoxanthine-guanine+phosphoribosyltransferase+inhibitors+as+potential+antimalarial+therapeutics&doi=10.1021%2Facschembio.7b00916"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00916&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00916%26sid%3Dliteratum%253Aachs%26aulast%3DKeough%26aufirst%3DD.%2BT.%26aulast%3DRejman%26aufirst%3DD.%26aulast%3DPohl%26aufirst%3DR.%26aulast%3DZborn%25C3%25ADkov%25C3%25A1%26aufirst%3DE.%26aulast%3DHockov%25C3%25A1%26aufirst%3DD.%26aulast%3DCroll%26aufirst%3DT.%26aulast%3DEdstein%26aufirst%3DM.%2BD.%26aulast%3DBirrell%26aufirst%3DG.%2BW.%26aulast%3DChavchich%26aufirst%3DM.%26aulast%3DNaesens%26aufirst%3DL.%2BM.%2BJ.%26aulast%3DPierens%26aufirst%3DG.%2BK.%26aulast%3DBrereton%26aufirst%3DI.%2BM.%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26atitle%3DDesign%2520of%2520Plasmodium%2520vivax%2520hypoxanthine-guanine%2520phosphoribosyltransferase%2520inhibitors%2520as%2520potential%2520antimalarial%2520therapeutics%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D82%26epage%3D90%26doi%3D10.1021%2Facschembio.7b00916" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keough, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocková, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holý, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naesens, L. M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skinner-Adams, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jersey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span> <span> </span><span class="NLM_article-title">Inhibition of hypoxanthine-guanine phosphoribosyltransferase by acyclic nucleoside phosphonates: A new class of antimalarial therapeutics</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">4391</span>– <span class="NLM_lpage">4399</span>, <span class="refDoi"> DOI: 10.1021/jm900267n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm900267n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnt1Ons7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=4391-4399&author=D.+T.+Keoughauthor=D.+Hockov%C3%A1author=A.+Hol%C3%BDauthor=L.+M.+J.+Naesensauthor=T.+S.+Skinner-Adamsauthor=J.+de+Jerseyauthor=L.+W.+Guddat&title=Inhibition+of+hypoxanthine-guanine+phosphoribosyltransferase+by+acyclic+nucleoside+phosphonates%3A+A+new+class+of+antimalarial+therapeutics&doi=10.1021%2Fjm900267n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Hypoxanthine-Guanine Phosphoribosyltransferase by Acyclic Nucleoside Phosphonates: A New Class of Antimalarial Therapeutics</span></div><div class="casAuthors">Keough, Dianne T.; Hockova, Dana; Holy, Antonin; Naesens, Lieve M. J.; Skinner-Adams, Tina S.; de Jersey, John; Guddat, Luke W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4391-4399</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The purine salvage enzyme hypoxanthine-guanine-xanthine phosphoribosyltransferase (HGXPRT) is essential for purine nucleotide and hence nucleic acid synthesis in the malaria parasite, Plasmodium falciparum.  Acyclic nucleoside phosphonates (ANPs) are analogs of the nucleotide product of the reaction, comprising a purine base joined by a linker to a phosphonate moiety.  Ki values for 19 ANPs were detd. for Pf HGXPRT and the corresponding human enzyme, HGPRT.  Values for Pf HGXPRT were as low as 100 nM, with selectivity for the parasite enzyme of up to 58.  Structures of human HGPRT in complex with three ANPs are reported.  On binding, a large mobile loop in the free enzyme moves to partly cover the active site.  For three ANPs, the IC50 values for Pf grown in cell culture were 1, 14, and 46 μM, while the cytotoxic concn. for the first compd. was 489 μM.  These results provide a basis for the design of potent and selective ANP inhibitors of Pf HGXPRT as antimalarial drug leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqn8rPcoy0EBbVg90H21EOLACvtfcHk0ljCONEEn3441A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnt1Ons7w%253D&md5=76746f78ec791a08ba2376beba86fa2b</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm900267n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm900267n%26sid%3Dliteratum%253Aachs%26aulast%3DKeough%26aufirst%3DD.%2BT.%26aulast%3DHockov%25C3%25A1%26aufirst%3DD.%26aulast%3DHol%25C3%25BD%26aufirst%3DA.%26aulast%3DNaesens%26aufirst%3DL.%2BM.%2BJ.%26aulast%3DSkinner-Adams%26aufirst%3DT.%2BS.%26aulast%3Dde%2BJersey%26aufirst%3DJ.%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26atitle%3DInhibition%2520of%2520hypoxanthine-guanine%2520phosphoribosyltransferase%2520by%2520acyclic%2520nucleoside%2520phosphonates%253A%2520A%2520new%2520class%2520of%2520antimalarial%2520therapeutics%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D4391%26epage%3D4399%26doi%3D10.1021%2Fjm900267n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hazleton, K. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ho, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cassera, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clinch, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crump, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosario, I.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merino, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almo, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyler, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schramm, V. L.</span></span> <span> </span><span class="NLM_article-title">Acyclic immucillin phosphonates: Second generation inhibitors of <i>Plasmodium falciparum</i> hypoxanthine-guanine-xanthine phosphoribosyltransferase</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">721</span>– <span class="NLM_lpage">730</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2012.04.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1016%2Fj.chembiol.2012.04.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=22726686" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC38XptVKqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=721-730&author=K.+Z.+Hazletonauthor=M.-C.+Hoauthor=M.+B.+Casseraauthor=K.+Clinchauthor=D.+R.+Crumpauthor=I.+Rosarioauthor=E.+F.+Merinoauthor=S.+C.+Almoauthor=P.+C.+Tylerauthor=V.+L.+Schramm&title=Acyclic+immucillin+phosphonates%3A+Second+generation+inhibitors+of+Plasmodium+falciparum+hypoxanthine-guanine-xanthine+phosphoribosyltransferase&doi=10.1016%2Fj.chembiol.2012.04.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Acyclic Immucillin Phosphonates: Second-Generation Inhibitors of Plasmodium falciparum Hypoxanthine- Guanine-Xanthine Phosphoribosyltransferase</span></div><div class="casAuthors">Hazleton, Keith Z.; Ho, Meng-Chiao; Cassera, Maria B.; Clinch, Keith; Crump, Douglas R.; Rosario, Irving; Merino, Emilio F.; Almo, Steve C.; Tyler, Peter C.; Schramm, Vern L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">721-730</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Plasmodium falciparum, the primary cause of deaths from malaria, is a purine auxotroph and relies on hypoxanthine salvage from the host purine pool.  Purine starvation as an antimalarial target has been validated by inhibition of purine nucleoside phosphorylase.  Hypoxanthine depletion kills Plasmodium falciparum in cell culture and in Aotus monkey infections.  Hypoxanthine-guanine-xanthine phosphoribosyltransferase (HGXPRT) from P. falciparum is required for hypoxanthine salvage by forming IMP, a branchpoint for all purine nucleotide synthesis in the parasite.  Here, we present a class of HGXPRT inhibitors, the acyclic immucillin phosphonates (AIPs), and cell permeable AIP prodrugs.  The AIPs are simple, potent, selective, and biol. stable inhibitors.  The AIP prodrugs block proliferation of cultured parasites by inhibiting the incorporation of hypoxanthine into the parasite nucleotide pool and validates HGXPRT as a target in malaria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJ-0LZKeBULLVg90H21EOLACvtfcHk0liPn2P5JRg9ZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptVKqsLs%253D&md5=9c72350a38b247652ca5209996a73495</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2012.04.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2012.04.012%26sid%3Dliteratum%253Aachs%26aulast%3DHazleton%26aufirst%3DK.%2BZ.%26aulast%3DHo%26aufirst%3DM.-C.%26aulast%3DCassera%26aufirst%3DM.%2BB.%26aulast%3DClinch%26aufirst%3DK.%26aulast%3DCrump%26aufirst%3DD.%2BR.%26aulast%3DRosario%26aufirst%3DI.%26aulast%3DMerino%26aufirst%3DE.%2BF.%26aulast%3DAlmo%26aufirst%3DS.%2BC.%26aulast%3DTyler%26aufirst%3DP.%2BC.%26aulast%3DSchramm%26aufirst%3DV.%2BL.%26atitle%3DAcyclic%2520immucillin%2520phosphonates%253A%2520Second%2520generation%2520inhibitors%2520of%2520Plasmodium%2520falciparum%2520hypoxanthine-guanine-xanthine%2520phosphoribosyltransferase%26jtitle%3DChem.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D721%26epage%3D730%26doi%3D10.1016%2Fj.chembiol.2012.04.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clinch, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crump, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hazleton, K. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schramm, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyler, P. C.</span></span> <span> </span><span class="NLM_article-title">Acyclic phosph(on)ate inhibitors of <i>Plasmodium falciparum</i> hypoxanthine-guanine-xanthine phosphoribosyltransferase</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">5629</span>– <span class="NLM_lpage">5646</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2013.02.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1016%2Fj.bmc.2013.02.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=23810424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVWls73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=5629-5646&author=K.+Clinchauthor=D.+R.+Crumpauthor=G.+B.+Evansauthor=K.+Z.+Hazletonauthor=J.+M.+Masonauthor=V.+L.+Schrammauthor=P.+C.+Tyler&title=Acyclic+phosph%28on%29ate+inhibitors+of+Plasmodium+falciparum+hypoxanthine-guanine-xanthine+phosphoribosyltransferase&doi=10.1016%2Fj.bmc.2013.02.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Acyclic phosph(on)ate inhibitors of Plasmodium falciparum hypoxanthine-guanine-xanthine phosphoribosyltransferase</span></div><div class="casAuthors">Clinch, Keith; Crump, Douglas R.; Evans, Gary B.; Hazleton, Keith Z.; Mason, Jennifer M.; Schramm, Vern L.; Tyler, Peter C.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5629-5646</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The pathogenic protozoa responsible for malaria lack enzymes for the de novo synthesis of purines and rely on purine salvage from the host.  In Plasmodium falciparum (Pf), hypoxanthine-guanine-xanthine phosphoribosyltransferase (HGXPRT) converts hypoxanthine to inosine monophosphate and is essential for purine salvage making the enzyme an anti-malarial drug target.  We have synthesized a no. of simple acyclic aza-C-nucleosides and shown that some are potent inhibitors of Pf HGXPRT while showing excellent selectivity for the Pf vs. the human enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUUjPr4oHetrVg90H21EOLACvtfcHk0liPn2P5JRg9ZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVWls73I&md5=1a7be25d17441e003e48c568a3ba231c</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.02.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.02.016%26sid%3Dliteratum%253Aachs%26aulast%3DClinch%26aufirst%3DK.%26aulast%3DCrump%26aufirst%3DD.%2BR.%26aulast%3DEvans%26aufirst%3DG.%2BB.%26aulast%3DHazleton%26aufirst%3DK.%2BZ.%26aulast%3DMason%26aufirst%3DJ.%2BM.%26aulast%3DSchramm%26aufirst%3DV.%2BL.%26aulast%3DTyler%26aufirst%3DP.%2BC.%26atitle%3DAcyclic%2520phosph%2528on%2529ate%2520inhibitors%2520of%2520Plasmodium%2520falciparum%2520hypoxanthine-guanine-xanthine%2520phosphoribosyltransferase%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D5629%26epage%3D5646%26doi%3D10.1016%2Fj.bmc.2013.02.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eng, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocková, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Špaček, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janeba, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woods, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naesens, L. M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keough, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span> <span> </span><span class="NLM_article-title">First crystal structures of <i>Mycobacterium tuberculosis</i> 6-oxopurine phosphoribosyltransferase: Complexes with GMP and pyrophosphate and with acyclic nucleoside phosphonates whose prodrugs have antituberculosis activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">4822</span>– <span class="NLM_lpage">4838</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00611</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00611" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC2MXntFGitL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=4822-4838&author=W.+S.+Engauthor=D.+Hockov%C3%A1author=P.+%C5%A0pa%C4%8Dekauthor=Z.+Janebaauthor=N.+P.+Westauthor=K.+Woodsauthor=L.+M.+J.+Naesensauthor=D.+T.+Keoughauthor=L.+W.+Guddat&title=First+crystal+structures+of+Mycobacterium+tuberculosis+6-oxopurine+phosphoribosyltransferase%3A+Complexes+with+GMP+and+pyrophosphate+and+with+acyclic+nucleoside+phosphonates+whose+prodrugs+have+antituberculosis+activity&doi=10.1021%2Facs.jmedchem.5b00611"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">First crystal structures of Mycobacterium tuberculosis 6-oxopurine phosphoribosyltransferase: Complexes with GMP and pyrophosphate and with acyclic nucleoside phosphonates whose prodrugs have antituberculosis activity</span></div><div class="casAuthors">Eng, Wai Soon; Hockova, Dana; Spacek, Petr; Janeba, Zlatko; West, Nicholas P.; Woods, Kyra; Naesens, Lieve M. J.; Keough, Dianne T.; Guddat, Luke W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4822-4838</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human tuberculosis is a chronic infectious disease affecting millions of lives.  Because of emerging resistance to current medications, new therapeutic drugs are needed.  One potential new target is hypoxanthine-guanine phosphoribosyltransferase (MtHGPRT), a key enzyme of the purine salvage pathway.  Here, newly synthesized acyclic nucleoside phosphonates (ANPs) have been shown to be competitive inhibitors of MtHGPRT with Ki values as low as 0.69 μM.  Prodrugs of these compds. arrest the growth of a virulent strain of M. tuberculosis with MIC50 values as low as 4.5 μM and possess low cytotoxicity in mammalian cells (CC50 values as high as >300 μM).  In addn., the first crystal structures of MtHGPRT (2.03-2.76 Å resoln.) have been detd., three of these in complex with novel ANPs and one with GMP and pyrophosphate.  These data provide a solid foundation for the further development of ANPs as selective inhibitors of MtHGPRT and as antituberculosis agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-Nicla0R_-7Vg90H21EOLACvtfcHk0liPn2P5JRg9ZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXntFGitL0%253D&md5=c36282ce6728032802af6ef43e0ecc3b</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00611%26sid%3Dliteratum%253Aachs%26aulast%3DEng%26aufirst%3DW.%2BS.%26aulast%3DHockov%25C3%25A1%26aufirst%3DD.%26aulast%3D%25C5%25A0pa%25C4%258Dek%26aufirst%3DP.%26aulast%3DJaneba%26aufirst%3DZ.%26aulast%3DWest%26aufirst%3DN.%2BP.%26aulast%3DWoods%26aufirst%3DK.%26aulast%3DNaesens%26aufirst%3DL.%2BM.%2BJ.%26aulast%3DKeough%26aufirst%3DD.%2BT.%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26atitle%3DFirst%2520crystal%2520structures%2520of%2520Mycobacterium%2520tuberculosis%25206-oxopurine%2520phosphoribosyltransferase%253A%2520Complexes%2520with%2520GMP%2520and%2520pyrophosphate%2520and%2520with%2520acyclic%2520nucleoside%2520phosphonates%2520whose%2520prodrugs%2520have%2520antituberculosis%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D4822%26epage%3D4838%26doi%3D10.1021%2Facs.jmedchem.5b00611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Špaček, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keough, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chavchich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dračínský, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janeba, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naesens, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edstein, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocková, D.</span></span> <span> </span><span class="NLM_article-title">Synthesis and evaluation of asymmetric acyclic nucleoside bisphosphonates as inhibitors of <i>Plasmodium falciparum</i> and human hypoxanthine-guanine-(xanthine) phosphoribosyltransferase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">7539</span>– <span class="NLM_lpage">7554</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00926</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00926" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlCgu7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=7539-7554&author=P.+%C5%A0pa%C4%8Dekauthor=D.+T.+Keoughauthor=M.+Chavchichauthor=M.+Dra%C4%8D%C3%ADnsk%C3%BDauthor=Z.+Janebaauthor=L.+Naesensauthor=M.+D.+Edsteinauthor=L.+W.+Guddatauthor=D.+Hockov%C3%A1&title=Synthesis+and+evaluation+of+asymmetric+acyclic+nucleoside+bisphosphonates+as+inhibitors+of+Plasmodium+falciparum+and+human+hypoxanthine-guanine-%28xanthine%29+phosphoribosyltransferase&doi=10.1021%2Facs.jmedchem.7b00926"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Evaluation of Asymmetric Acyclic Nucleoside Bisphosphonates as Inhibitors of Plasmodium falciparum and Human Hypoxanthine-Guanine-(Xanthine) Phosphoribosyltransferase</span></div><div class="casAuthors">Spacek, Petr; Keough, Dianne T.; Chavchich, Marina; Dracinsky, Martin; Janeba, Zlatko; Naesens, Lieve; Edstein, Michael D.; Guddat, Luke W.; Hockova, Dana</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7539-7554</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Acyclic nucleoside bisphosphonates (ANbPs) have previously been shown to be good inhibitors of human hypoxanthine-guanine phosphoribosyltransferase (HGPRT) and Plasmodium falciparum (Pf) hypoxanthine-guanine-xanthine phosphoribosyltransferase (PfHGXPRT).  On the basis of this scaffold, a new series of ANbPs was synthesized.  One of these new ANbPs, [3-(guanine-9-yl)-2-((2-phosphonoethoxy)methyl)propoxy]methylphosphonic acid, exhibited Ki values of 6 and 70 nM for human HGPRT and Pf HGXPRT, resp.  These low Ki values were achieved by inserting an extra carbon atom in the linker connecting the N9 atom of guanine to one of the phosphonate groups.  The crystal structure of this ANbP in complex with human HGPRT was detd. at 2.0 Å resoln. and shows that it fills three key pockets in the active site.  The most potent phosphoramidate prodrugs of these compds. have IC50 values in the low micromolar range in Pf lines and low toxicity in human A549 cells, demonstrating that these ANbPs are excellent antimalarial drug leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnSzTuy4-WwrVg90H21EOLACvtfcHk0lirH0HLysaJxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlCgu7rP&md5=9c47d1dfc40b240ad2c152b3cb542eec</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00926%26sid%3Dliteratum%253Aachs%26aulast%3D%25C5%25A0pa%25C4%258Dek%26aufirst%3DP.%26aulast%3DKeough%26aufirst%3DD.%2BT.%26aulast%3DChavchich%26aufirst%3DM.%26aulast%3DDra%25C4%258D%25C3%25ADnsk%25C3%25BD%26aufirst%3DM.%26aulast%3DJaneba%26aufirst%3DZ.%26aulast%3DNaesens%26aufirst%3DL.%26aulast%3DEdstein%26aufirst%3DM.%2BD.%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26aulast%3DHockov%25C3%25A1%26aufirst%3DD.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520asymmetric%2520acyclic%2520nucleoside%2520bisphosphonates%2520as%2520inhibitors%2520of%2520Plasmodium%2520falciparum%2520and%2520human%2520hypoxanthine-guanine-%2528xanthine%2529%2520phosphoribosyltransferase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D7539%26epage%3D7554%26doi%3D10.1021%2Facs.jmedchem.7b00926" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Špaček, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keough, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chavchich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dračínský, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janeba, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naesens, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edstein, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocková, D.</span></span> <span> </span><span class="NLM_article-title">Synthesis and evaluation of symmetric acyclic nucleoside bisphosphonates as inhibitors of the <i>Plasmodium falciparum, Plasmodium vivax</i> and human 6-oxopurine phosphoribosyltransferases and the antimalarial activity of their prodrugs</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">4008</span>– <span class="NLM_lpage">4030</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2017.05.048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1016%2Fj.bmc.2017.05.048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=28601510" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpslGnur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=4008-4030&author=P.+%C5%A0pa%C4%8Dekauthor=D.+T.+Keoughauthor=M.+Chavchichauthor=M.+Dra%C4%8D%C3%ADnsk%C3%BDauthor=Z.+Janebaauthor=L.+Naesensauthor=M.+D.+Edsteinauthor=L.+W.+Guddatauthor=D.+Hockov%C3%A1&title=Synthesis+and+evaluation+of+symmetric+acyclic+nucleoside+bisphosphonates+as+inhibitors+of+the+Plasmodium+falciparum%2C+Plasmodium+vivax+and+human+6-oxopurine+phosphoribosyltransferases+and+the+antimalarial+activity+of+their+prodrugs&doi=10.1016%2Fj.bmc.2017.05.048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and evaluation of symmetric acyclic nucleoside bisphosphonates as inhibitors of the Plasmodium falciparum, Plasmodium vivax and human 6-oxopurine phosphoribosyltransferases and the antimalarial activity of their prodrugs</span></div><div class="casAuthors">Spacek, Petr; Keough, Dianne T.; Chavchich, Marina; Dracinsky, Martin; Janeba, Zlatko; Naesens, Lieve; Edstein, Michael D.; Guddat, Luke W.; Hockova, Dana</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4008-4030</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Two new series of sym. acyclic nucleoside bisphosphonates (ANbPs) have been synthesized as potential inhibitors of the Plasmodium falciparum (Pf) and vivax (Pv) 6-oxopurine phosphoribosyltransferases.  The structural variability between these sym. ANbPs lies in the no. of atoms in the two acyclic linkers connecting the N9 atom of the purine base to each of two phosphonate groups and the branching point of the acyclic moiety relative to the purine base, which occurs at either the alpha or beta positions.  Within each series, six different 6-oxopurine bases have been attached.  In general, the ANbPs with either guanine or hypoxanthine have lower Ki values than for those contg. either the 8-bromo or 7-deaza 6-oxopurine bases.  The lowest Ki values obtained for the two parasite enzymes were 0.1 μM (Pf) and 0.2 μM (Pv) for this series of compds.  Two phosphoramidate prodrugs of these inhibitors exhibited antimalarial activity against Pf in infected erythrocyte cell culture with IC50 values of 0.8 and 1.5 μM.  These two compds. exhibited low cytotoxicity in human A549 cells having CC50 values of >300 μM resulting in an excellent selectivity index.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRylWYh-MpNLVg90H21EOLACvtfcHk0lirH0HLysaJxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpslGnur8%253D&md5=62891e50ca20c740844a44c1a7e8876b</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.05.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.05.048%26sid%3Dliteratum%253Aachs%26aulast%3D%25C5%25A0pa%25C4%258Dek%26aufirst%3DP.%26aulast%3DKeough%26aufirst%3DD.%2BT.%26aulast%3DChavchich%26aufirst%3DM.%26aulast%3DDra%25C4%258D%25C3%25ADnsk%25C3%25BD%26aufirst%3DM.%26aulast%3DJaneba%26aufirst%3DZ.%26aulast%3DNaesens%26aufirst%3DL.%26aulast%3DEdstein%26aufirst%3DM.%2BD.%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26aulast%3DHockov%25C3%25A1%26aufirst%3DD.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520symmetric%2520acyclic%2520nucleoside%2520bisphosphonates%2520as%2520inhibitors%2520of%2520the%2520Plasmodium%2520falciparum%252C%2520Plasmodium%2520vivax%2520and%2520human%25206-oxopurine%2520phosphoribosyltransferases%2520and%2520the%2520antimalarial%2520activity%2520of%2520their%2520prodrugs%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D4008%26epage%3D4030%26doi%3D10.1016%2Fj.bmc.2017.05.048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keough, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocková, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rejman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Špaček, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vrbková, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krečmerová, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eng, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jans, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naesens, L. M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de
Jersey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span> <span> </span><span class="NLM_article-title">Inhibition of the <i>Escherichia coli</i> 6-oxopurine phosphoribosyltransferases by nucleoside phosphonates: potential for new antibacterial agents</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">6967</span>– <span class="NLM_lpage">6984</span>, <span class="refDoi"> DOI: 10.1021/jm400779n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400779n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1ChsL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=6967-6984&author=D.+T.+Keoughauthor=D.+Hockov%C3%A1author=D.+Rejmanauthor=P.+%C5%A0pa%C4%8Dekauthor=S.+Vrbkov%C3%A1author=M.+Kre%C4%8Dmerov%C3%A1author=W.+S.+Engauthor=H.+Jansauthor=N.+P.+Westauthor=L.+M.+J.+Naesensauthor=J.+de%0AJerseyauthor=L.+W.+Guddat&title=Inhibition+of+the+Escherichia+coli+6-oxopurine+phosphoribosyltransferases+by+nucleoside+phosphonates%3A+potential+for+new+antibacterial+agents&doi=10.1021%2Fjm400779n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of the Escherichia coli 6-Oxopurine Phosphoribosyltransferases by Nucleoside Phosphonates: Potential for New Antibacterial Agents</span></div><div class="casAuthors">Keough, Dianne T.; Hockova, Dana; Rejman, Dominik; Spacek, Petr; Vrbkova, Silvie; Krecmerova, Marcela; Eng, Wai Soon; Jans, Harmen; West, Nicholas P.; Naesens, Lieve M. J.; de Jersey, John; Guddat, Luke W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">6967-6984</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Escherichia coli (Ec) cells possess two purine salvage enzymes: xanthine-guanine phosphoribosyltransferase (XGPRT) and hypoxanthine phosphoribosyltransferase (HPRT).  EcXGPRT shares a common structural feature with other members of this family, a flexible loop that closes over the active site during catalysis.  The replacement of six of these amino acids by alanine has no effect on the Km for the two substrates.  However, the Ki for the nucleoside monophosphate increases by 27-fold, and the kcat is reduced by ∼200-fold.  Nucleoside phosphonates (NP) are good inhibitors of EcXGPRT and EcHPRT, with Ki values as low as 10 nM.  In the absence of the flexible loop, these values increase by 5- to 30-fold, indicating the importance of the loop for high-affinity inhibition.  Crystal structures of two NPs in complex with EcXGPRT explain the tight binding.  Prodrugs of NPs with low Ki values for EcXGPRT or EcHPRT exhibit IC50 values between 5 and 23 μM against Mycobacterium tuberculosis in cell-based assays, suggesting that these compds. are therapeutic leads against pathogenic bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjJGY9KeZWpbVg90H21EOLACvtfcHk0ljXjSnqWuGbIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1ChsL7M&md5=bc8016299374898032e7311d054ec883</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm400779n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400779n%26sid%3Dliteratum%253Aachs%26aulast%3DKeough%26aufirst%3DD.%2BT.%26aulast%3DHockov%25C3%25A1%26aufirst%3DD.%26aulast%3DRejman%26aufirst%3DD.%26aulast%3D%25C5%25A0pa%25C4%258Dek%26aufirst%3DP.%26aulast%3DVrbkov%25C3%25A1%26aufirst%3DS.%26aulast%3DKre%25C4%258Dmerov%25C3%25A1%26aufirst%3DM.%26aulast%3DEng%26aufirst%3DW.%2BS.%26aulast%3DJans%26aufirst%3DH.%26aulast%3DWest%26aufirst%3DN.%2BP.%26aulast%3DNaesens%26aufirst%3DL.%2BM.%2BJ.%26aulast%3Dde%2BJersey%26aufirst%3DJ.%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26atitle%3DInhibition%2520of%2520the%2520Escherichia%2520coli%25206-oxopurine%2520phosphoribosyltransferases%2520by%2520nucleoside%2520phosphonates%253A%2520potential%2520for%2520new%2520antibacterial%2520agents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D6967%26epage%3D6984%26doi%3D10.1021%2Fjm400779n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Clercq, E.</span></span> <span> </span><span class="NLM_article-title">The acyclic nucleoside phosphonates (ANPs): Antonin Holy’s legacy</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">1278</span>– <span class="NLM_lpage">1303</span>, <span class="refDoi"> DOI: 10.1002/med.21283</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1002%2Fmed.21283" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=23568857" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsF2hu7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2013&pages=1278-1303&author=E.+De%0AClercq&title=The+acyclic+nucleoside+phosphonates+%28ANPs%29%3A+Antonin+Holy%E2%80%99s+legacy&doi=10.1002%2Fmed.21283"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">The Acyclic Nucleoside Phosphonates (ANPs): Antonin Holy's Legacy</span></div><div class="casAuthors">De Clercq, Erik</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1278-1303</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  The name of Antonin Holy has become synonymous for the era of acyclic nucleoside phosphonates (ANPs), which started with (S)-HPMPA as the prototype and (S)-HPMPC (cidofovir) as the first marketed compd.  It has now evolved to a no. of compds. clin. used in the treatment of HIV and hepatitis B virus infections, either as such [tenofovir disoproxil fumarate (TDF, Viread)] or in combination [Truvada, Atripla, Complera, Stribild].  Truvada has also been approved for the prevention of HIV infections.  Forthcoming is a new formulation of tenofovir (TAF: tenofovir alafenamide).  Also forthcoming are several "quad" drug combinations contg. either TDF or TAF.  Other ANPs, based on either an alkoxy side chain or 5-azacytosine heterocycle seem highly promising and worth further pursuing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWroqgi65J4bVg90H21EOLACvtfcHk0ljXjSnqWuGbIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsF2hu7jF&md5=f921578d8a2b5eaa975704b1225dfa8e</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1002%2Fmed.21283&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21283%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BClercq%26aufirst%3DE.%26atitle%3DThe%2520acyclic%2520nucleoside%2520phosphonates%2520%2528ANPs%2529%253A%2520Antonin%2520Holy%25E2%2580%2599s%2520legacy%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2013%26volume%3D33%26spage%3D1278%26epage%3D1303%26doi%3D10.1002%2Fmed.21283" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Clercq, E.</span></span> <span> </span><span class="NLM_article-title">The Holy Trinity: the acyclic nucleoside phosphonates</span>. <i>Adv. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">316</span>, <span class="refDoi"> DOI: 10.1016/b978-0-12-405880-4.00008-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1016%2FB978-0-12-405880-4.00008-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=23886004" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1Kku7jN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2013&pages=293-316&author=E.+De%0AClercq&title=The+Holy+Trinity%3A+the+acyclic+nucleoside+phosphonates&doi=10.1016%2Fb978-0-12-405880-4.00008-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">The Holy Trinity: the acyclic nucleoside phosphonates</span></div><div class="casAuthors">De Clercq, Erik</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Pharmacology (San Diego, CA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">Antiviral Agents</span>),
    <span class="NLM_cas:pages">293-316</span>CODEN:
                <span class="NLM_cas:coden">ADPHEL</span>;
        ISSN:<span class="NLM_cas:issn">1054-3589</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  The Holy Trinity was named after Dr Antonin Holy (the Holy Trinity being an Unesco recognized monument in Olomouc, Czech Republic), who together with Dr John C. Martin (Gilead Sciences) and myself pioneered a new class of antiviral agents, the acyclic nucleoside phosphonates.  These compds. have revolutionized the antiviral drug field with several drugs that have been approved for the treatment of various DNA virus infections (cidofovir), hepatitis B (adefovir), and AIDS (HIV infection; tenofovir).  The latter is also available as its oral prodrug, tenofovir disoproxil fumarate, for the treatment of hepatitis B and in combination with emtricitabine ((-)FTC) for the treatment and prophylaxis of HIV infections and in combination with (-)FTC and other HIV inhibitors, i.e., efavirenz, rilpivirine, or elvitegravir (and a pharmacoenhancer thereof, cobicistat), for the treatment of AIDS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBSAiIxqMFs7Vg90H21EOLACvtfcHk0ljXjSnqWuGbIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1Kku7jN&md5=563984dfb1d047b65f14d3f7b61d48db</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-405880-4.00008-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FB978-0-12-405880-4.00008-1%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BClercq%26aufirst%3DE.%26atitle%3DThe%2520Holy%2520Trinity%253A%2520the%2520acyclic%2520nucleoside%2520phosphonates%26jtitle%3DAdv.%2520Pharmacol.%26date%3D2013%26volume%3D67%26spage%3D293%26epage%3D316%26doi%3D10.1016%2Fb978-0-12-405880-4.00008-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Slusarczyk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serpi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pertusati, F.</span></span> <span> </span><span class="NLM_article-title">Phosphoramidates and phosphonamidates (ProTides) with antiviral activity</span>. <i>Antiviral Chem. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">204020661877524</span>, <span class="refDoi"> DOI: 10.1177/2040206618775243</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1177%2F2040206618775243" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2018&pages=204020661877524&author=M.+Slusarczykauthor=M.+Serpiauthor=F.+Pertusati&title=Phosphoramidates+and+phosphonamidates+%28ProTides%29+with+antiviral+activity&doi=10.1177%2F2040206618775243"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1177%2F2040206618775243&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2040206618775243%26sid%3Dliteratum%253Aachs%26aulast%3DSlusarczyk%26aufirst%3DM.%26aulast%3DSerpi%26aufirst%3DM.%26aulast%3DPertusati%26aufirst%3DF.%26atitle%3DPhosphoramidates%2520and%2520phosphonamidates%2520%2528ProTides%2529%2520with%2520antiviral%2520activity%26jtitle%3DAntiviral%2520Chem.%2520Chemother.%26date%3D2018%26volume%3D26%26spage%3D204020661877524%26doi%3D10.1177%2F2040206618775243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mehellou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rattan, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balzarini, J.</span></span> <span> </span><span class="NLM_article-title">The ProTide Prodrug Technology: From the Concept to the Clinic</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">2211</span>– <span class="NLM_lpage">2226</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00734</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00734" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlSitL3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=2211-2226&author=Y.+Mehellouauthor=H.+S.+Rattanauthor=J.+Balzarini&title=The+ProTide+Prodrug+Technology%3A+From+the+Concept+to+the+Clinic&doi=10.1021%2Facs.jmedchem.7b00734"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">The ProTide Prodrug Technology: From the Concept to the Clinic</span></div><div class="casAuthors">Mehellou, Youcef; Rattan, Hardeep S.; Balzarini, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2211-2226</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The ProTide technol. is a prodrug approach developed for the efficient intracellular delivery of nucleoside analog monophosphates and monophosphonates.  In this approach, the hydroxyls of the monophosphate or monophosphonate groups are masked by an arom. group and an amino acid ester moiety, which are enzymically cleaved-off inside cells to release the free nucleoside monophosphate and monophosphonate species.  Structurally, this represents the current end-point of an extensive medicinal chem. endeavor that spans almost three decades.  It started from the masking of nucleoside monophosphate and monophosphonate groups by simple alkyl groups and evolved into the sophisticated ProTide system as known today.  This technol. has been extensively employed in drug discovery, and it has already led to the discovery of two FDA-approved (antiviral) ProTides.  In this work, we will review the development of the ProTide technol., its application in drug discovery, and its role in the improvement of drug delivery and efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeaJFaQn4HwrVg90H21EOLACvtfcHk0licNomsUaq8Rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlSitL3P&md5=ab9f5805c3c9d2d80fbdd82a3ad86c73</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00734&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00734%26sid%3Dliteratum%253Aachs%26aulast%3DMehellou%26aufirst%3DY.%26aulast%3DRattan%26aufirst%3DH.%2BS.%26aulast%3DBalzarini%26aufirst%3DJ.%26atitle%3DThe%2520ProTide%2520Prodrug%2520Technology%253A%2520From%2520the%2520Concept%2520to%2520the%2520Clinic%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D2211%26epage%3D2226%26doi%3D10.1021%2Facs.jmedchem.7b00734" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keough, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skinner-Adams, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, A.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brereton, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jersey, J.</span></span> <span> </span><span class="NLM_article-title">Lead compounds for antimalarial chemotherapy: Purine base analogs discriminate between human and <i>P. falciparum</i> 6-oxopurine phosphoribosyltransferases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">7479</span>– <span class="NLM_lpage">7486</span>, <span class="refDoi"> DOI: 10.1021/jm061012j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm061012j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1WhsLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=7479-7486&author=D.+T.+Keoughauthor=T.+Skinner-Adamsauthor=M.+K.+Jonesauthor=A.-L.+Ngauthor=I.+M.+Breretonauthor=L.+W.+Guddatauthor=J.+de+Jersey&title=Lead+compounds+for+antimalarial+chemotherapy%3A+Purine+base+analogs+discriminate+between+human+and+P.+falciparum+6-oxopurine+phosphoribosyltransferases&doi=10.1021%2Fjm061012j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Lead cmpounds for antimalarial chemotherapy: purine base analogs discriminate between human and P. falciparum 6-oxopurine Phosphoribosyltransferases</span></div><div class="casAuthors">Keough, Dianne T.; Skinner-Adams, Tina; Jones, Malcolm K.; Ng, Ai-Lin; Brereton, Ian M.; Guddat, Luke W.; de Jersey, John</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">7479-7486</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The malarial parasite Plasmodium falciparum depends on the purine salvage enzyme hypoxanthine-guanine-xanthine phosphoribosyltransferase (HGXPRT) to convert purine bases from the host to nucleotides needed for DNA and RNA synthesis.  An approach to developing antimalarial drugs is to use HGXPRT to convert introduced purine base analogs to nucleotides that are toxic to the parasite.  This strategy requires that these compds. be good substrates for the parasite enzyme but poor substrates for the human counterpart, HGPRT.  Bases with a chlorine atom in the 6-position or a nitrogen in the 8-position exhibited strong discrimination between P. falciparum HGXPRT and human HGPRT.  The kcat/Km values for the Plasmodium enzyme using 6-chloroguanine and 8-azaguanine as substrates were 50-80-fold and 336-fold higher than for the human enzyme, resp.  These and other bases were effective in inhibiting the growth of the parasite in vitro, giving IC50 values as low as 1 μM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5wP9-53NDS7Vg90H21EOLACvtfcHk0licNomsUaq8Rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1WhsLvM&md5=525b0f86fa5d8d560945cddfcdf872c0</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm061012j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm061012j%26sid%3Dliteratum%253Aachs%26aulast%3DKeough%26aufirst%3DD.%2BT.%26aulast%3DSkinner-Adams%26aufirst%3DT.%26aulast%3DJones%26aufirst%3DM.%2BK.%26aulast%3DNg%26aufirst%3DA.-L.%26aulast%3DBrereton%26aufirst%3DI.%2BM.%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26aulast%3Dde%2BJersey%26aufirst%3DJ.%26atitle%3DLead%2520compounds%2520for%2520antimalarial%2520chemotherapy%253A%2520Purine%2520base%2520analogs%2520discriminate%2520between%2520human%2520and%2520P.%2520falciparum%25206-oxopurine%2520phosphoribosyltransferases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D7479%26epage%3D7486%26doi%3D10.1021%2Fjm061012j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jersey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, J. L.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of <i>Escherichia coli</i> xanthine phosphoribosyltransferase</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">4125</span>– <span class="NLM_lpage">4134</span>, <span class="refDoi"> DOI: 10.1021/bi962640d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi962640d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADyaK2sXisVeqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1997&pages=4125-4134&author=S.+Vosauthor=J.+de+Jerseyauthor=J.+L.+Martin&title=Crystal+structure+of+Escherichia+coli+xanthine+phosphoribosyltransferase&doi=10.1021%2Fbi962640d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure of Escherichia coli Xanthine Phosphoribosyltransferase</span></div><div class="casAuthors">Vos, Siska; de Jersey, John; Martin, Jennifer L.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4125-4134</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Xanthine phosphoribosyltransferase (XPRT; EC 2.4.2.22) from Escherichia coli is a tetrameric enzyme having 152 residues per subunit.  XPRT catalyzes the transfer of the phosphoribosyl group from 5-phospho-α-D-ribosyl 1-pyrophosphate (PRib-PP) to the 6-oxopurine bases guanine, xanthine, and hypoxanthine to form GMP, XMP, and IMP, resp.  Crystals grown in the absence of substrate or product were used to det. the structure of XPRT at a resoln. of 1.8 Å, by multiple isomorphous replacement.  The core structure of XPRT includes a five-stranded parallel β-sheet surrounded by three α-helixes, which is similar to that obsd. in other known phosphoribosyltransferase (PRTase) structures.  The XPRT structure also has several interesting features.  A glutamine residue in the purine binding site may be responsible for the altered 6-oxopurine base specificity seen in this enzyme compared to other 6-oxopurine PRTases.  Also, the authors observe both a magnesium ion and a sulfate ion bound at the PRib-PP binding site of XPRT.  The sulfate ion interacts with Arg-37 which has a cis-peptide conformation, and the magnesium ion interacts with Asp-89, a highly conserved acidic residue in the PRib-PP binding site motif.  The XPRT structure also incorporates a feature which has not been obsd. in other PRTase structures.  The C-terminal 12 residues of XPRT adopt an unusual extended conformation and make interactions with a neighboring subunit.  The very last residue, Arg-152, could form part of the active site of a symmetry-related subunit in the XPRT tetramer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojPPh9OZQ1RbVg90H21EOLACvtfcHk0licNomsUaq8Rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXisVeqtbk%253D&md5=b8da0a6f29db2721d2a01d1fa5f154d5</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fbi962640d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi962640d%26sid%3Dliteratum%253Aachs%26aulast%3DVos%26aufirst%3DS.%26aulast%3Dde%2BJersey%26aufirst%3DJ.%26aulast%3DMartin%26aufirst%3DJ.%2BL.%26atitle%3DCrystal%2520structure%2520of%2520Escherichia%2520coli%2520xanthine%2520phosphoribosyltransferase%26jtitle%3DBiochemistry%26date%3D1997%26volume%3D36%26spage%3D4125%26epage%3D4134%26doi%3D10.1021%2Fbi962640d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keough, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jersey, J.</span></span> <span> </span><span class="NLM_article-title">Crystal structures of free, IMP-, and GMP-bound <i>Escherichia coli</i> hypoxanthine phosphoribosyltransferase</span>. <i>Prot. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1626</span>– <span class="NLM_lpage">1638</span>, <span class="refDoi"> DOI: 10.1110/ps.0201002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1110%2Fps.0201002" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2002&pages=1626-1638&author=L.+W.+Guddatauthor=S.+Vosauthor=J.+L.+Martinauthor=D.+T.+Keoughauthor=J.+de+Jersey&title=Crystal+structures+of+free%2C+IMP-%2C+and+GMP-bound+Escherichia+coli+hypoxanthine+phosphoribosyltransferase&doi=10.1110%2Fps.0201002"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1110%2Fps.0201002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.0201002%26sid%3Dliteratum%253Aachs%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26aulast%3DVos%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DJ.%2BL.%26aulast%3DKeough%26aufirst%3DD.%2BT.%26aulast%3Dde%2BJersey%26aufirst%3DJ.%26atitle%3DCrystal%2520structures%2520of%2520free%252C%2520IMP-%252C%2520and%2520GMP-bound%2520Escherichia%2520coli%2520hypoxanthine%2520phosphoribosyltransferase%26jtitle%3DProt.%2520Sci.%26date%3D2002%26volume%3D11%26spage%3D1626%26epage%3D1638%26doi%3D10.1110%2Fps.0201002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpenter, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamson, D. J.</span></span> <span> </span><span class="NLM_article-title">Mechanisms of action of 6-thioguanine, 6-mercaptopurine, and 8-azaguanine</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">1975</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">2872</span>– <span class="NLM_lpage">2878</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1157053" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADyaE28Xht1entw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=1975&pages=2872-2878&author=J.+A.+Nelsonauthor=J.+W.+Carpenterauthor=L.+M.+Roseauthor=D.+J.+Adamson&title=Mechanisms+of+action+of+6-thioguanine%2C+6-mercaptopurine%2C+and+8-azaguanine"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of action of 6-thioguanine, 6-mercaptopurine, and 8-azaguanine</span></div><div class="casAuthors">Nelson, J. Arly; Carpenter, Jean W.; Rose, Lucy M.; Adamson, Doris J.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">1975</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2872-8</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    </div><div class="casAbstract">The toxic effect of 6-thioguanine (I) [154-42-7] to purine biosynthesis and cell viability in H. Ep. 2 cells was not reversed by aminoimidazolecarboxamide, suggesting that inhibition of purine biosynthesis de novo is not the sole mechanism of toxicity.  Also, 6-(methylmercapto)purine ribonucleoside, a potent inhibitor of purine biosynthesis de novo, produced more marked reductions in cellular pools of purines than did 6-thioguanine without killing cells.  There was no apparent inhibition by 6- thioguanosine 5'-monophosphate of other enzymes leading to the synthesis of guanosine 5'-triphosphate as detd. in whole cells by measurements of radioactive hypoxanthine or guanine incorporation.  Inhibition of DNA synthesis by 1 mM thymidine protected cells from 6-mercaptopurine [50-44-2] or I but failed to protect cells from 8-azaguanine [134-58-7] toxicity.  On the other hand, inhibition of RNA synthesis by 6-azauridine plus deoxycytidine protected cells against 8-azaguanine but did not protect against I or 6-mercaptopurine toxicity.  In agreement with the in vitro data, arabinosylcytosine (a potent inhibitor of DNA synthesis) failed to protect mice against 8-azaguanine.  Thus, the incorporation into DNA (as I nucleotide) appears to be a mechanism of toxicity for these thiopyrines, whereas 8-azaguanine is toxic due to its incorporation into RNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobxwvtEIkV-bVg90H21EOLACvtfcHk0lj3r0ppgLZ5cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE28Xht1entw%253D%253D&md5=551badd3cfc883dd9cf47c4cb5a623d5</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNelson%26aufirst%3DJ.%2BA.%26aulast%3DCarpenter%26aufirst%3DJ.%2BW.%26aulast%3DRose%26aufirst%3DL.%2BM.%26aulast%3DAdamson%26aufirst%3DD.%2BJ.%26atitle%3DMechanisms%2520of%2520action%2520of%25206-thioguanine%252C%25206-mercaptopurine%252C%2520and%25208-azaguanine%26jtitle%3DCancer%2520Res.%26date%3D1975%26volume%3D35%26spage%3D2872%26epage%3D2878" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yazawa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunicardi, F. C.</span></span> <span> </span><span class="NLM_article-title">Current progress in suicide gene therapy for cancer</span>. <i>World J. Surg.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">783</span>– <span class="NLM_lpage">789</span>, <span class="refDoi"> DOI: 10.1007/s00268-002-4053-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1007%2Fs00268-002-4053-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=11948367" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A280%3ADC%252BD38zltlWnsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2002&pages=783-789&author=K.+Yazawaauthor=W.+E.+Fisherauthor=F.+C.+Brunicardi&title=Current+progress+in+suicide+gene+therapy+for+cancer&doi=10.1007%2Fs00268-002-4053-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Current progress in suicide gene therapy for cancer</span></div><div class="casAuthors">Yazawa Kazuyuki; Fisher William E; Brunicardi F Charles</div><div class="citationInfo"><span class="NLM_cas:title">World journal of surgery</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">783-9</span>
        ISSN:<span class="NLM_cas:issn">0364-2313</span>.
    </div><div class="casAbstract">Standard chemotherapeutic agents and ionizing radiation destroy dividing cells.  Because tumor cells divide more rapidly than normal cells, there is a therapeutic index in which damage to the cancer cells is maximized while keeping the toxicity to the normal host cells acceptable.  Suicide gene therapy strives to deliver genes to the cancer cells, which convert nontoxic prodrugs into active chemotherapeutic agents.  With this strategy, the systemically administered prodrug is converted to the active chemotherapeutic agent only in cancer cells, thereby allowing a maximal therapeutic effect while limiting systemic toxicity.  A literature search was conducted using the MEDLINE database from 1990 to 2001 to identify articles related to suicide gene therapy for cancer.  A number of suicide gene systems have been identified, including the herpes simplex virus thymidine kinase gene, the cytosine deaminase gene, the varicella-zoster virus thymidine kinase gene, the nitroreductase gene, the Escherichia coli gpt gene, and the E. coli Deo gene.  Various vectors, including liposomes, retroviruses, and adenoviruses, have been used to transfer these suicide genes to tumor cells.  These strategies have been effective in cell culture experiments, laboratory animals, and some early clinical trials.  Advances in tissue- and cell-specific delivery of suicide genes using specific promoters will improve the clinical utility of suicide gene therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQXdL1aW36dHZp6t8nS6Nc3fW6udTcc2ebqKd8DwyJwO7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD38zltlWnsA%253D%253D&md5=81ed0f9b4b2a3448b22fd17ba1225751</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1007%2Fs00268-002-4053-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00268-002-4053-5%26sid%3Dliteratum%253Aachs%26aulast%3DYazawa%26aufirst%3DK.%26aulast%3DFisher%26aufirst%3DW.%2BE.%26aulast%3DBrunicardi%26aufirst%3DF.%2BC.%26atitle%3DCurrent%2520progress%2520in%2520suicide%2520gene%2520therapy%2520for%2520cancer%26jtitle%3DWorld%2520J.%2520Surg.%26date%3D2002%26volume%3D26%26spage%3D783%26epage%3D789%26doi%3D10.1007%2Fs00268-002-4053-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hocková, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janeba, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naesens, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edstein, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chavchich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keough, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span> <span> </span><span class="NLM_article-title">Antimalarial activity of prodrugs of N-branched acyclic nucleoside phosphonate inhibitors of 6-oxopurine phosphoribosyltransferases</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">5502</span>– <span class="NLM_lpage">5510</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2015.07.038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1016%2Fj.bmc.2015.07.038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=26275679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1CrtLrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=5502-5510&author=D.+Hockov%C3%A1author=Z.+Janebaauthor=L.+Naesensauthor=M.+D.+Edsteinauthor=M.+Chavchichauthor=D.+T.+Keoughauthor=L.+W.+Guddat&title=Antimalarial+activity+of+prodrugs+of+N-branched+acyclic+nucleoside+phosphonate+inhibitors+of+6-oxopurine+phosphoribosyltransferases&doi=10.1016%2Fj.bmc.2015.07.038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Antimalarial activity of prodrugs of N-branched acyclic nucleoside phosphonate inhibitors of 6-oxopurine phosphoribosyltransferases</span></div><div class="casAuthors">Hockova, Dana; Janeba, Zlatko; Naesens, Lieve; Edstein, Michael D.; Chavchich, Marina; Keough, Dianne T.; Guddat, Luke W.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5502-5510</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Acyclic nucleoside phosphonates that contain a 6-oxopurine base are good inhibitors of the human and Plasmodium falciparum 6-oxopurine phosphoribosyltransferases, key enzymes of the purine salvage pathway.  Chem. modifications, based on the crystal structures of several inhibitors in complex with the human PRTase, led to the design of a new class of inhibitors-the aza-ANPs.  Because of the neg. charges of the phosphonic acid moiety, their ability to cross cell membranes is, however, limited.  Thus, phosphoramidate prodrugs of the aza-ANPs were prepd. to improve permeability.  These prodrugs arrest parasitemia with IC50 values in the micromolar range against Plasmodium falciparum-infected erythrocyte cultures (both chloroquine-sensitive and chloroquine-resistant Pf strains).  The prodrugs exhibit low cytotoxicity in several human cell lines.  Thus, they fulfill two essential criteria to qualify them as promising antimalarial drug leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6zbiAVYpUqbVg90H21EOLACvtfcHk0lj3r0ppgLZ5cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1CrtLrP&md5=0ec3d8eefea40295963feb4333670b02</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2015.07.038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2015.07.038%26sid%3Dliteratum%253Aachs%26aulast%3DHockov%25C3%25A1%26aufirst%3DD.%26aulast%3DJaneba%26aufirst%3DZ.%26aulast%3DNaesens%26aufirst%3DL.%26aulast%3DEdstein%26aufirst%3DM.%2BD.%26aulast%3DChavchich%26aufirst%3DM.%26aulast%3DKeough%26aufirst%3DD.%2BT.%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26atitle%3DAntimalarial%2520activity%2520of%2520prodrugs%2520of%2520N-branched%2520acyclic%2520nucleoside%2520phosphonate%2520inhibitors%2520of%25206-oxopurine%2520phosphoribosyltransferases%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2015%26volume%3D23%26spage%3D5502%26epage%3D5510%26doi%3D10.1016%2Fj.bmc.2015.07.038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGuigan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madela, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aljarah, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourdin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrica, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolykhalov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bleiman, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganguly, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorovits, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henson, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hutchins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muhammad, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obikhod, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patti, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walters, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vernachio, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramamurty, C. V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battina, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlain, S.</span></span> <span> </span><span class="NLM_article-title">Phosphorodiamidates as a promising new phosphate prodrug motif for antiviral drug discovery: application to anti-HCV agents</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">8632</span>– <span class="NLM_lpage">8645</span>, <span class="refDoi"> DOI: 10.1021/jm2011673</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2011673" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A280%3ADC%252BC38%252FmtlWjsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=8632-8645&author=C.+McGuiganauthor=K.+Madelaauthor=M.+Aljarahauthor=C.+Bourdinauthor=M.+Arricaauthor=E.+Barrettauthor=S.+Jonesauthor=A.+Kolykhalovauthor=B.+Bleimanauthor=K.+D.+Bryantauthor=B.+Gangulyauthor=E.+Gorovitsauthor=G.+Hensonauthor=D.+Hunleyauthor=J.+Hutchinsauthor=J.+Muhammadauthor=A.+Obikhodauthor=J.+Pattiauthor=C.+R.+Waltersauthor=J.+Wangauthor=J.+Vernachioauthor=C.+V.+S.+Ramamurtyauthor=S.+K.+Battinaauthor=S.+Chamberlain&title=Phosphorodiamidates+as+a+promising+new+phosphate+prodrug+motif+for+antiviral+drug+discovery%3A+application+to+anti-HCV+agents&doi=10.1021%2Fjm2011673"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphorodiamidates as a promising new phosphate prodrug motif for antiviral drug discovery: application to anti-HCV agents</span></div><div class="casAuthors">McGuigan Christopher; Madela Karolina; Aljarah Mohamed; Bourdin Claire; Arrica Maria; Barrett Emma; Jones Sarah; Kolykhalov Alexander; Bleiman Blair; Bryant K Dawn; Ganguly Babita; Gorovits Elena; Henson Geoffrey; Hunley Damound; Hutchins Jeff; Muhammad Jerry; Obikhod Aleksandr; Patti Joseph; Walters C Robin; Wang Jin; Vernachio John; Ramamurty Changalvala V S; Battina Srinivas K; Chamberlain Stanley</div><div class="citationInfo"><span class="NLM_cas:title">Journal of medicinal chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8632-45</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">We herein report phosphorodiamidates as a significant new phosphate prodrug motif.  Sixty-seven phosphorodiamidates are reported of two 6-O-alkyl 2'-C-methyl guanosines, with significant variation in the diamidate structure.  Both symmetrical and asymmetric phosphorodiamidates are reported, derived from various esterified amino acids, both d and l, and also from various simple amines.  All of the compounds were evaluated versus hepatitis C virus in replicon assay, and nanomolar activity levels were observed.  Many compounds were noncytotoxic at 100 μM, leading to high antiviral selectivities.  The agents are stable in acidic, neutral, and moderately basic media and in selected biological media but show efficient processing by carboxypeptidases and efficiently yield the free nucleoside monophosphate in cells.  On the basis of in vitro data, eight leads were selected for additional in vivo evaluation, with the intent of selecting one candidate for progression toward clinical studies.  This phosphorodiamidate prodrug method may have broad application outside of HCV and antivirals as it offers many of the advantages of phosphoramidate ProTides but without the chirality issues present in most cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSQ8wY11iyNsRLDWw4HIq2sfW6udTcc2eaLN1dLcU5fqbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38%252FmtlWjsQ%253D%253D&md5=5b7d43c2582a96a963e3927988054df7</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm2011673&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2011673%26sid%3Dliteratum%253Aachs%26aulast%3DMcGuigan%26aufirst%3DC.%26aulast%3DMadela%26aufirst%3DK.%26aulast%3DAljarah%26aufirst%3DM.%26aulast%3DBourdin%26aufirst%3DC.%26aulast%3DArrica%26aufirst%3DM.%26aulast%3DBarrett%26aufirst%3DE.%26aulast%3DJones%26aufirst%3DS.%26aulast%3DKolykhalov%26aufirst%3DA.%26aulast%3DBleiman%26aufirst%3DB.%26aulast%3DBryant%26aufirst%3DK.%2BD.%26aulast%3DGanguly%26aufirst%3DB.%26aulast%3DGorovits%26aufirst%3DE.%26aulast%3DHenson%26aufirst%3DG.%26aulast%3DHunley%26aufirst%3DD.%26aulast%3DHutchins%26aufirst%3DJ.%26aulast%3DMuhammad%26aufirst%3DJ.%26aulast%3DObikhod%26aufirst%3DA.%26aulast%3DPatti%26aufirst%3DJ.%26aulast%3DWalters%26aufirst%3DC.%2BR.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DVernachio%26aufirst%3DJ.%26aulast%3DRamamurty%26aufirst%3DC.%2BV.%2BS.%26aulast%3DBattina%26aufirst%3DS.%2BK.%26aulast%3DChamberlain%26aufirst%3DS.%26atitle%3DPhosphorodiamidates%2520as%2520a%2520promising%2520new%2520phosphate%2520prodrug%2520motif%2520for%2520antiviral%2520drug%2520discovery%253A%2520application%2520to%2520anti-HCV%2520agents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D8632%26epage%3D8645%26doi%3D10.1021%2Fjm2011673" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keough, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Špaček, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocková, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tichý, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vrbková, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slavětínská, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janeba, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naesens, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edstein, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chavchich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jersey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span> <span> </span><span class="NLM_article-title">Acyclic nucleoside phosphonates containing a second phosphonate group are potent inhibitors of 6-oxopurine phosphoribosyltransferases and have antimalarial activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">2513</span>– <span class="NLM_lpage">2526</span>, <span class="refDoi"> DOI: 10.1021/jm301893b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301893b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlWgsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2513-2526&author=D.+T.+Keoughauthor=P.+%C5%A0pa%C4%8Dekauthor=D.+Hockov%C3%A1author=T.+Tich%C3%BDauthor=S.+Vrbkov%C3%A1author=L.+Slav%C4%9Bt%C3%ADnsk%C3%A1author=Z.+Janebaauthor=L.+Naesensauthor=M.+D.+Edsteinauthor=M.+Chavchichauthor=T.-H.+Wangauthor=J.+de+Jerseyauthor=L.+W.+Guddat&title=Acyclic+nucleoside+phosphonates+containing+a+second+phosphonate+group+are+potent+inhibitors+of+6-oxopurine+phosphoribosyltransferases+and+have+antimalarial+activity&doi=10.1021%2Fjm301893b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Acyclic Nucleoside Phosphonates Containing a Second Phosphonate Group Are Potent Inhibitors of 6-Oxopurine Phosphoribosyltransferases and Have Antimalarial Activity</span></div><div class="casAuthors">Keough, Dianne T.; Spacek, Petr; Hockova, Dana; Tichy, Tomas; Vrbkova, Silvie; Slavetinska, Lenka; Janeba, Zlatko; Naesens, Lieve; Edstein, Michael D.; Chavchich, Marina; Wang, Tzu-Hsuan; de Jersey, John; Guddat, Luke W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2513-2526</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Acyclic nucleoside phosphonates (ANPs) that contain a 6-oxopurine base are good inhibitors of the Plasmodium falciparum (Pf) and Plasmodium vivax (Pv) 6-oxopurine phosphoribosyltransferases (PRTs).  Chem. modifications based on the crystal structure of 2-(phosphonoethoxy)ethylguanine (PEEG) in complex with human HGPRT have led to the design of new ANPs.  These novel compds. contain a second phosphonate group attached to the ANP scaffold.  {[(2-[(Guanine-9H-yl)methyl]propane-1,3-diyl)bis(oxy)]bis(methylene)}diphosphonic acid (I) exhibited a Ki value of 30 nM for human HGPRT and 70 nM for Pf HGXPRT.  The crystal structure of this compd. in complex with human HGPRT shows that it fills or partially fills three crit. locations in the active site: the binding sites of the purine base, the 5'-phosphate group, and pyrophosphate.  This is the first HG(X)PRT inhibitor that has been able to achieve this result.  Prodrugs have been synthesized resulting in IC50 values as low as 3.8 μM for Pf grown in cell culture, up to 25-fold lower compared to the parent compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOIyMCQRsPnrVg90H21EOLACvtfcHk0lhyfLDaarncdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlWgsrw%253D&md5=f57c4cfd4f9bcb94e6ce93894bfd4d0f</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjm301893b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301893b%26sid%3Dliteratum%253Aachs%26aulast%3DKeough%26aufirst%3DD.%2BT.%26aulast%3D%25C5%25A0pa%25C4%258Dek%26aufirst%3DP.%26aulast%3DHockov%25C3%25A1%26aufirst%3DD.%26aulast%3DTich%25C3%25BD%26aufirst%3DT.%26aulast%3DVrbkov%25C3%25A1%26aufirst%3DS.%26aulast%3DSlav%25C4%259Bt%25C3%25ADnsk%25C3%25A1%26aufirst%3DL.%26aulast%3DJaneba%26aufirst%3DZ.%26aulast%3DNaesens%26aufirst%3DL.%26aulast%3DEdstein%26aufirst%3DM.%2BD.%26aulast%3DChavchich%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DT.-H.%26aulast%3Dde%2BJersey%26aufirst%3DJ.%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26atitle%3DAcyclic%2520nucleoside%2520phosphonates%2520containing%2520a%2520second%2520phosphonate%2520group%2520are%2520potent%2520inhibitors%2520of%25206-oxopurine%2520phosphoribosyltransferases%2520and%2520have%2520antimalarial%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2513%26epage%3D2526%26doi%3D10.1021%2Fjm301893b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Larkin, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackshields, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chenna, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGettigan, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McWilliam, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valentin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallace, I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilm, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibson, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higgins, D. G.</span></span> <span> </span><span class="NLM_article-title">Clustal W and Clustal X version 2.0</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">2947</span>– <span class="NLM_lpage">2948</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btm404</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1093%2Fbioinformatics%2Fbtm404" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=17846036" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlaqsL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2007&pages=2947-2948&author=M.+A.+Larkinauthor=G.+Blackshieldsauthor=N.+P.+Brownauthor=R.+Chennaauthor=P.+A.+McGettiganauthor=H.+McWilliamauthor=F.+Valentinauthor=I.+M.+Wallaceauthor=A.+Wilmauthor=R.+Lopezauthor=J.+D.+Thompsonauthor=T.+J.+Gibsonauthor=D.+G.+Higgins&title=Clustal+W+and+Clustal+X+version+2.0&doi=10.1093%2Fbioinformatics%2Fbtm404"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Clustal W and Clustal X version 2.0</span></div><div class="casAuthors">Larkin, M. A.; Blackshields, G.; Brown, N. P.; Chenna, R.; McGettigan, P. A.; McWilliam, H.; Valentin, F.; Wallace, I. M.; Wilm, A.; Lopez, R.; Thompson, J. D.; Gibson, T. J.; Higgins, D. G.</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">2947-2948</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Clustal W and Clustal X multiple sequence alignment programs have been completely rewritten in C++.  This will facilitate the further development of the alignment algorithms in the future and has allowed proper porting of the programs to the latest versions of Linux, Macintosh and Windows operating systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWsv_FPaWsDLVg90H21EOLACvtfcHk0lhyfLDaarncdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlaqsL%252FM&md5=14c988ecd0b5081ef79149f31a567880</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtm404&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtm404%26sid%3Dliteratum%253Aachs%26aulast%3DLarkin%26aufirst%3DM.%2BA.%26aulast%3DBlackshields%26aufirst%3DG.%26aulast%3DBrown%26aufirst%3DN.%2BP.%26aulast%3DChenna%26aufirst%3DR.%26aulast%3DMcGettigan%26aufirst%3DP.%2BA.%26aulast%3DMcWilliam%26aufirst%3DH.%26aulast%3DValentin%26aufirst%3DF.%26aulast%3DWallace%26aufirst%3DI.%2BM.%26aulast%3DWilm%26aufirst%3DA.%26aulast%3DLopez%26aufirst%3DR.%26aulast%3DThompson%26aufirst%3DJ.%2BD.%26aulast%3DGibson%26aufirst%3DT.%2BJ.%26aulast%3DHiggins%26aufirst%3DD.%2BG.%26atitle%3DClustal%2520W%2520and%2520Clustal%2520X%2520version%25202.0%26jtitle%3DBioinformatics%26date%3D2007%26volume%3D23%26spage%3D2947%26epage%3D2948%26doi%3D10.1093%2Fbioinformatics%2Fbtm404" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leinonen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diez, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binns, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleischmann, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Apweiler, R.</span></span> <span> </span><span class="NLM_article-title">UniProt archive</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">3236</span>– <span class="NLM_lpage">3237</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/bth191</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1093%2Fbioinformatics%2Fbth191" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=15044231" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVSrtbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2004&pages=3236-3237&author=R.+Leinonenauthor=F.+G.+Diezauthor=D.+Binnsauthor=W.+Fleischmannauthor=R.+Lopezauthor=R.+Apweiler&title=UniProt+archive&doi=10.1093%2Fbioinformatics%2Fbth191"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">UniProt archive</span></div><div class="casAuthors">Leinonen, Rasko; Diez, Federico Garcia; Binns, David; Fleischmann, Wolfgang; Lopez, Rodrigo; Apweiler, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3236-3237</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">UniProt Archive (UniParc) is the most comprehensive, non-redundant protein sequence database available.  Its protein sequences are retrieved from predominant, publicly accessible resources.  All new and updated protein sequences are collected and loaded daily into UniParc for full coverage.  To avoid redundancy, each unique sequence is stored only once with a stable protein identifier, which can be used later in UniParc to identify the same protein in all source databases.  When proteins are loaded into the database, database cross-refs. are created to link them to the origins of the sequences.  As a result, performing a sequence search against UniParc is equiv. to performing the same search against all databases cross-referenced by UniParc.  UniParc contains only protein sequences and database cross-refs.; all other information must be retrieved from the source databases.  UniParc is available at http://www.ebi.ac.uk/uniparc/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQIl2XMN-a77Vg90H21EOLACvtfcHk0ligKhxAkVDH1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVSrtbfK&md5=6fa7558121997bac740281fa3c2cf985</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbth191&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbth191%26sid%3Dliteratum%253Aachs%26aulast%3DLeinonen%26aufirst%3DR.%26aulast%3DDiez%26aufirst%3DF.%2BG.%26aulast%3DBinns%26aufirst%3DD.%26aulast%3DFleischmann%26aufirst%3DW.%26aulast%3DLopez%26aufirst%3DR.%26aulast%3DApweiler%26aufirst%3DR.%26atitle%3DUniProt%2520archive%26jtitle%3DBioinformatics%26date%3D2004%26volume%3D20%26spage%3D3236%26epage%3D3237%26doi%3D10.1093%2Fbioinformatics%2Fbth191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holden, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelley, W. N.</span></span> <span> </span><span class="NLM_article-title">Human hypoxanthine-guanine phosphoribosyltransferase. Evidence for tetrameric structure</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1978</span>,  <span class="NLM_volume"><i>253</i></span>,  <span class="NLM_fpage">4459</span>– <span class="NLM_lpage">4463</span>, <span class="refDoi"> DOI: 10.1016/s0021-9258(17)34742-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1016%2FS0021-9258%2817%2934742-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=659426" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADyaE1cXkvVKntLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=253&publication_year=1978&pages=4459-4463&author=J.+A.+Holdenauthor=W.+N.+Kelley&title=Human+hypoxanthine-guanine+phosphoribosyltransferase.+Evidence+for+tetrameric+structure&doi=10.1016%2Fs0021-9258%2817%2934742-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Human hypoxanthine-guanine phosphoribosyltransferase.  Evidence for tetrameric structure</span></div><div class="casAuthors">Holden, Joseph A.; Kelley, William N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1978</span>),
    <span class="NLM_cas:volume">253</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4459-63</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">Hypoxanthine-guanine phosphoribosyltransferase was purified 23,000-fold from normal human erythrocytes.  The purifn. included affinity chromatog. on a GMP column.  The subunit mol. wt. of the enzyme obtained from this purifn. was 24,000.  The finding of 4 protein species after crosslinkage of the highly purified enzyme with dimethylsuberimidate, dimethyladipimidate, and glutaraldehyde suggests that the enzyme may exist in the native state as a tetramer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2nH7mTT0lVLVg90H21EOLACvtfcHk0ligKhxAkVDH1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE1cXkvVKntLk%253D&md5=7f00fd4a9c39881c8da1e1c62ee85959</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2FS0021-9258%2817%2934742-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0021-9258%252817%252934742-7%26sid%3Dliteratum%253Aachs%26aulast%3DHolden%26aufirst%3DJ.%2BA.%26aulast%3DKelley%26aufirst%3DW.%2BN.%26atitle%3DHuman%2520hypoxanthine-guanine%2520phosphoribosyltransferase.%2520Evidence%2520for%2520tetrameric%2520structure%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1978%26volume%3D253%26spage%3D4459%26epage%3D4463%26doi%3D10.1016%2Fs0021-9258%2817%2934742-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parry, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jersey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, J. L.</span></span> <span> </span><span class="NLM_article-title">Structures of free and complexed forms of <i>Escherichia coli</i> xanthine-guanine phosphoribosyltransferase</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">875</span>– <span class="NLM_lpage">889</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1998.2051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1006%2Fjmbi.1998.2051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=9743633" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADyaK1cXmsFOls7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=1998&pages=875-889&author=S.+Vosauthor=R.+J.+Parryauthor=M.+R.+Burnsauthor=J.+de+Jerseyauthor=J.+L.+Martin&title=Structures+of+free+and+complexed+forms+of+Escherichia+coli+xanthine-guanine+phosphoribosyltransferase&doi=10.1006%2Fjmbi.1998.2051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Structures of free and complexed forms of Escherichia coli xanthine-guanine phosphoribosyltransferase</span></div><div class="casAuthors">Vos, Siska; Parry, Ronald J.; Burns, Mark R.; De Jersey, John; Martin, Jennifer L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">875-889</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">Structures of free, substrate-bound and product-bound forms of Escherichia coli xanthine-guanine phosphoribosyltransferase (XGPRT) have been detd. by X-ray crystallog.  These are compared with the previously detd. structure of magnesium and sulfate-bound XPRT.  The structure of free XGPRT at 2.25 Å resoln. confirms the flexibility of residues in and around a mobile loop identified in other PRTases and shows that the cis-peptide conformation of Arg37 at the active site is maintained in the absence of bound ligands.  The structures of XGPRT complexed with the purine base substrates guanine or xanthine in combination with cPRib-PP, an analog of the second substrate PRib-PP, have been solved to 2.0 Å resoln.  In these two structures the disordered phosphate-binding loop of uncomplexed XGPRT becomes ordered through interactions with the 5'-phosphate group of cPRib-PP.  The cyclopentane ring of cPRib-PP has the C3 exo pucker conformation, stabilized by the cPRib-PP-bound Mg2+.  The purine base specificity of XGPRT appears to be due to water-mediated interactions between the 2-exocyclic groups of guanine or xanthine and side-chains of Glu136 and Asp140, as well as the main-chain oxygen atom of Ile135.  Asp92, together with Lys115, could help stabilize the N7-protonated tautomer of the incoming base and could act as a general base to remove the proton from N7 when the nucleotide product is formed.  The 2.6 Å resoln. structure of XGPRT complexed with product GMP is similar to the substrate-bound complexes.  However, the ribose ring of GMP is rotated by ∼24° compared with the equiv. ring in cPRib-PP.  This rotation results in the loss of all interactions between the ribosyl group and the enzyme in the product complex.  (c) 1998 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoI3pcM6-h6bLVg90H21EOLACvtfcHk0ligKhxAkVDH1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmsFOls7g%253D&md5=243f8db6a5009ff8d10230bc5ff670f6</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1998.2051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1998.2051%26sid%3Dliteratum%253Aachs%26aulast%3DVos%26aufirst%3DS.%26aulast%3DParry%26aufirst%3DR.%2BJ.%26aulast%3DBurns%26aufirst%3DM.%2BR.%26aulast%3Dde%2BJersey%26aufirst%3DJ.%26aulast%3DMartin%26aufirst%3DJ.%2BL.%26atitle%3DStructures%2520of%2520free%2520and%2520complexed%2520forms%2520of%2520Escherichia%2520coli%2520xanthine-guanine%2520phosphoribosyltransferase%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1998%26volume%3D282%26spage%3D875%26epage%3D889%26doi%3D10.1006%2Fjmbi.1998.2051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schramm, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almo, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyler, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furneaux, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grubmeyer, C.</span></span> <span> </span><span class="NLM_article-title">The 2.0 Å structure of human hypoxanthine-guanine phosphoribosyltransferase in complex with a transition-state analog inhibitor</span>. <i>Nat. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">588</span>– <span class="NLM_lpage">593</span>, <span class="refDoi"> DOI: 10.1038/9376</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1038%2F9376" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10360366" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADyaK1MXjs1yktLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1999&pages=588-593&author=V.+L.+Schrammauthor=S.+C.+Almoauthor=W.+Shiauthor=C.+M.+Liauthor=P.+C.+Tylerauthor=R.+H.+Furneauxauthor=C.+Grubmeyer&title=The+2.0+%C3%85+structure+of+human+hypoxanthine-guanine+phosphoribosyltransferase+in+complex+with+a+transition-state+analog+inhibitor&doi=10.1038%2F9376"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">The 2.0 Å structure of human hypoxanthine-guanine phosphoribosyltransferase in complex with a transition-state analog inhibitor</span></div><div class="casAuthors">Shi, Wuxian; Li, Caroline M.; Tyler, Peter C.; Furneaux, Richard H.; Grubmeyer, Charles; Schramm, Vern L.; Almo, Steven C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">588-593</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature America</span>)
        </div><div class="casAbstract">The crystal structure of human hypoxanthine-guanine phosphoribosyltransferase (HGPRT) bound to the transition-state analog, immucillinGP, and Mg2+-pyrophosphate (MgPPi) was detd. to 2.0 Å resoln.  ImmucillinGP was designed as a stable analog with the stereoelectronic features of the transition state.  Bound inhibitor at the catalytic site indicated that the oxocarbenium ion of the transition state was stabilized by neighboring-group participation from MgPPi and 05'.  A short H-bond formed between Asp-137 and the purine ring analog.  Two Mg2+ ions sandwiched PPi and contacted both OH groups of the ribosyl analog.  The transition-state analog was shielded from bulk solvent by a catalytic loop that moved ∼25 Å to cover the active site and became an ordered antiparallel β-sheet.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPL0QhGqlEFLVg90H21EOLACvtfcHk0lj6hYVJRhKM6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjs1yktLg%253D&md5=9344053a685003a4720d6be3419f09c5</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2F9376&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F9376%26sid%3Dliteratum%253Aachs%26aulast%3DSchramm%26aufirst%3DV.%2BL.%26aulast%3DAlmo%26aufirst%3DS.%2BC.%26aulast%3DShi%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DC.%2BM.%26aulast%3DTyler%26aufirst%3DP.%2BC.%26aulast%3DFurneaux%26aufirst%3DR.%2BH.%26aulast%3DGrubmeyer%26aufirst%3DC.%26atitle%3DThe%25202.0%2520%25C3%2585%2520structure%2520of%2520human%2520hypoxanthine-guanine%2520phosphoribosyltransferase%2520in%2520complex%2520with%2520a%2520transition-state%2520analog%2520inhibitor%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D1999%26volume%3D6%26spage%3D588%26epage%3D593%26doi%3D10.1038%2F9376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyler, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furneaux, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cahill, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girvin, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grubmeyer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schramm, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almo, S. C.</span></span> <span> </span><span class="NLM_article-title">The 2.0 Å structure of malarial purine phosphoribosyltransferase in complex with a transition-state analogue inhibitor</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">9872</span>– <span class="NLM_lpage">9880</span>, <span class="refDoi"> DOI: 10.1021/bi990664p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi990664p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADyaK1MXktlWjt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=9872-9880&author=W.+Shiauthor=C.+M.+Liauthor=P.+C.+Tylerauthor=R.+H.+Furneauxauthor=S.+M.+Cahillauthor=M.+E.+Girvinauthor=C.+Grubmeyerauthor=V.+L.+Schrammauthor=S.+C.+Almo&title=The+2.0+%C3%85+structure+of+malarial+purine+phosphoribosyltransferase+in+complex+with+a+transition-state+analogue+inhibitor&doi=10.1021%2Fbi990664p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">The 2.0 Å Structure of Malarial Purine Phosphoribosyltransferase in Complex with a Transition-State Analogue Inhibitor</span></div><div class="casAuthors">Shi, Wuxian; Li, Caroline M.; Tyler, Peter C.; Furneaux, Richard H.; Cahill, Sean M.; Girvin, Mark E.; Grubmeyer, Charles; Schramm, Vern L.; Almo, Steven C.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">9872-9880</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Malaria is a leading cause of worldwide mortality from infectious disease.  Plasmodium falciparum proliferation in human erythrocytes requires purine salvage by hypoxanthine-guanine-xanthine phosphoribosyltransferase (HGXPRTase).  The enzyme is a target for the development of novel antimalarials.  Design and synthesis of transition-state analog inhibitors permitted cocrystn. with the malarial enzyme and refinement of the complex to 2.0 Å resoln.  Catalytic site contacts in the malarial enzyme are similar to those of human hypoxanthine-guanine phosphoribosyltransferase (HGPRTase) despite distinct substrate specificity.  The crystal structure of malarial HGXPRTase with bound inhibitor, pyrophosphate, and two Mg2+ ions reveals features unique to the transition-state analog complex.  Substrate-assisted catalysis occurs by ribooxocarbenium stabilization from the O5' lone pair and a pyrophosphate oxygen.  A dissociative reaction coordinate path is implicated in which the primary reaction coordinate motion is the ribosyl C1' in motion between relatively immobile purine base and (Mg)2-pyrophosphate.  Several short hydrogen bonds form in the complex of the enzyme and inhibitor.  The proton NMR spectrum of the transition-state analog complex of malarial HGXPRTase contains two downfield signals at 14.3 and 15.3 ppm.  Despite the structural similarity to the human enzyme, the NMR spectra of the complexes reveal differences in hydrogen bonding between the transition-state analog complexes of the human and malarial HG(X)PRTases.  The X-ray crystal structures and NMR spectra reveal chem. and structural features that suggest a strategy for the design of malaria-specific transition-state inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6InWXezcVG7Vg90H21EOLACvtfcHk0lj6hYVJRhKM6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXktlWjt7o%253D&md5=b885550fcda9e1cb936ca983f56d6567</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fbi990664p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi990664p%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DC.%2BM.%26aulast%3DTyler%26aufirst%3DP.%2BC.%26aulast%3DFurneaux%26aufirst%3DR.%2BH.%26aulast%3DCahill%26aufirst%3DS.%2BM.%26aulast%3DGirvin%26aufirst%3DM.%2BE.%26aulast%3DGrubmeyer%26aufirst%3DC.%26aulast%3DSchramm%26aufirst%3DV.%2BL.%26aulast%3DAlmo%26aufirst%3DS.%2BC.%26atitle%3DThe%25202.0%2520%25C3%2585%2520structure%2520of%2520malarial%2520purine%2520phosphoribosyltransferase%2520in%2520complex%2520with%2520a%2520transition-state%2520analogue%2520inhibitor%26jtitle%3DBiochemistry%26date%3D1999%26volume%3D38%26spage%3D9872%26epage%3D9880%26doi%3D10.1021%2Fbi990664p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eads, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scapin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grubmeyer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchettini, J. C.</span></span> <span> </span><span class="NLM_article-title">The crystal structure of human hypoxanthine-guanine phosphoribosyltransferase with bound GMP</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">325</span>– <span class="NLM_lpage">334</span>, <span class="refDoi"> DOI: 10.1016/0092-8674(94)90301-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1016%2F0092-8674%2894%2990301-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=8044844" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADyaK2cXmt1Cru70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=1994&pages=325-334&author=J.+C.+Eadsauthor=G.+Scapinauthor=Y.+Xuauthor=C.+Grubmeyerauthor=J.+C.+Sacchettini&title=The+crystal+structure+of+human+hypoxanthine-guanine+phosphoribosyltransferase+with+bound+GMP&doi=10.1016%2F0092-8674%2894%2990301-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">The crystal structure of human hypoxanthine-guanine phosphoribosyltransferase with bound GMP</span></div><div class="casAuthors">Eads, Janina C.; Scapin, Giovanna; Xu, Yiming; Grubmeyer, Charles; Sacchettini, James C.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">325-34</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    </div><div class="casAbstract">The crystal structure of HGPRTase with bound GMP has been detd. and refined to 2.5 Å resoln.  The enzyme has a core α/β structure resembling the nucleotide-binding fold of dehydrogenases, and a second lobe composed of residues from the amino and carboxy termini.  The GMP mol. binds in an anti conformation in a solvent-exposed cleft of the enzyme.  Lys-165, which forms a hydrogen bond to O6 of GMP, appears to be crit. for detg. the specificity for guanine and hypoxanthine over adenine.  The location of active site residues also provides evidence for a possible mechanism for general base-assisted HGPRTase catalysis.  A rationalization of the effects on stability and activity of naturally occurring single amino acid mutations of HGPRTase is presented, including a discussion of several mutations at the active site that lead to Lesch-Nyhan syndrome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAWyBkNXeCzLVg90H21EOLACvtfcHk0lj6hYVJRhKM6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXmt1Cru70%253D&md5=8589aa21a9ebc8daba1f14e10b4b9482</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2894%2990301-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252894%252990301-8%26sid%3Dliteratum%253Aachs%26aulast%3DEads%26aufirst%3DJ.%2BC.%26aulast%3DScapin%26aufirst%3DG.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DGrubmeyer%26aufirst%3DC.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26atitle%3DThe%2520crystal%2520structure%2520of%2520human%2520hypoxanthine-guanine%2520phosphoribosyltransferase%2520with%2520bound%2520GMP%26jtitle%3DCell%26date%3D1994%26volume%3D78%26spage%3D325%26epage%3D334%26doi%3D10.1016%2F0092-8674%2894%2990301-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eng, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocková, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Špaček, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baszczyňski, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janeba, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naesens, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keough, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guddat, L. W.</span></span> <span> </span><span class="NLM_article-title">Crystal structures of acyclic nucleoside phosphonates in complex with <i>Escherichia coli</i> hypoxanthine phosphoribosyltransferase</span>. <i>ChemistrySelect</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">6267</span>– <span class="NLM_lpage">6276</span>, <span class="refDoi"> DOI: 10.1002/slct.201601679</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1002%2Fslct.201601679" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFymtb%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2016&pages=6267-6276&author=W.+S.+Engauthor=D.+Hockov%C3%A1author=P.+%C5%A0pa%C4%8Dekauthor=O.+Baszczy%C5%88skiauthor=Z.+Janebaauthor=L.+Naesensauthor=D.+T.+Keoughauthor=L.+W.+Guddat&title=Crystal+structures+of+acyclic+nucleoside+phosphonates+in+complex+with+Escherichia+coli+hypoxanthine+phosphoribosyltransferase&doi=10.1002%2Fslct.201601679"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structures of Acyclic Nucleoside Phosphonates in Complex with Escherichia coli Hypoxanthine Phosphoribosyltransferase</span></div><div class="casAuthors">Eng, Wai Soon; Hockova, Dana; Spacek, Petr; Baszczynski, Ondrej; Janeba, Zlatko; Naesens, Lieve; Keough, Dianne T.; Guddat, Luke W.</div><div class="citationInfo"><span class="NLM_cas:title">ChemistrySelect</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6267-6276</span>CODEN:
                <span class="NLM_cas:coden">CHEMUD</span>;
        ISSN:<span class="NLM_cas:issn">2365-6549</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Drug resistant uropathogenic E. coli assocd. with urinary tract infections (UTIs) are a serious and debilitating health threat.  Therefore new drug targets to treat this disease need to be explored.  One possible approach is to block the synthesis of the nucleoside monophosphates required for DNA/RNA prodn. in these pathogens.  In E. coli, the purine salvage pathway has two 6-oxopurine phosphoribosyltransferases (PRTs), xanthine-guanine PRT (EcXGPRT) and hypoxanthine PRT (EcHPRT).  Here, we investigate acyclic nucleoside phosphonates (ANPs) as inhibitors of EcHPRT and have detd. six crystal structures of EcHPRT in complex with ANPs.  These data reveal the binding modes of these compds. and can assist in future rational structure-based design efforts.  It is suggested that a combination of inhibitors that block de novo and salvage pathways is a plausible approach to developing new antibiotics for E. coli UTIs.  In addn., we provide details of a novel approach to accelerating the crystn. of this enzyme that may be of general applicability for rational drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDiYpWpYajtLVg90H21EOLACvtfcHk0lhawkqMzNVXpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFymtb%252FL&md5=617b757eef6150b74f2839e7b97333fb</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1002%2Fslct.201601679&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fslct.201601679%26sid%3Dliteratum%253Aachs%26aulast%3DEng%26aufirst%3DW.%2BS.%26aulast%3DHockov%25C3%25A1%26aufirst%3DD.%26aulast%3D%25C5%25A0pa%25C4%258Dek%26aufirst%3DP.%26aulast%3DBaszczy%25C5%2588ski%26aufirst%3DO.%26aulast%3DJaneba%26aufirst%3DZ.%26aulast%3DNaesens%26aufirst%3DL.%26aulast%3DKeough%26aufirst%3DD.%2BT.%26aulast%3DGuddat%26aufirst%3DL.%2BW.%26atitle%3DCrystal%2520structures%2520of%2520acyclic%2520nucleoside%2520phosphonates%2520in%2520complex%2520with%2520Escherichia%2520coli%2520hypoxanthine%2520phosphoribosyltransferase%26jtitle%3DChemistrySelect%26date%3D2016%26volume%3D1%26spage%3D6267%26epage%3D6276%26doi%3D10.1002%2Fslct.201601679" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, J. F.</span></span> <span> </span><span class="NLM_article-title">Kinetics of the reversible inhibition of enzyme-catalysed reactions by tight-binding inhibitors</span>. <i>Biochim. Biophys. Acta</i> <span class="NLM_year" style="font-weight: bold;">1969</span>,  <span class="NLM_volume"><i>185</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">286</span>, <span class="refDoi"> DOI: 10.1016/0005-2744(69)90420-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1016%2F0005-2744%2869%2990420-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=4980133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADyaF1MXkvVOqsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=185&publication_year=1969&pages=269-286&author=J.+F.+Morrison&title=Kinetics+of+the+reversible+inhibition+of+enzyme-catalysed+reactions+by+tight-binding+inhibitors&doi=10.1016%2F0005-2744%2869%2990420-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetics of the reversible inhibition of enzyme-catalyzed reactions by tight-binding inhibitors</span></div><div class="casAuthors">Morrison, John F.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Enzymology</span>
        (<span class="NLM_cas:date">1969</span>),
    <span class="NLM_cas:volume">185</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">269-86</span>CODEN:
                <span class="NLM_cas:coden">BBEZAD</span>;
        ISSN:<span class="NLM_cas:issn">0924-1086</span>.
    </div><div class="casAbstract">A theoretical kinetic study has been undertaken to det. the kinetic characteristics of enzymic reactions that are reversibly inhibited by a substrate analog. at concns. comparable to that of the enzyme.  A general rate equation, that allows for the combination of such a tight-binding inhibitor with one or more enzyme forms of any reaction mechanism, has been derived and shown to be quadratic with sq. and linear terms in υ which represents the true initial steady-state velocity.  From plots of υ against total enzyme concn. at fixed total inhibitor concn., the concn. of enzyme catalytic sites can be detd. irrespective of the nature of the inhibition and the reaction mechanism.  However, plots of υ (or I/υ) against total inhibitor concn. can be used for this purpose only if the dissocn. const. assocd. with the inhibitor is virtually zero.  Particular initial rate equations have been obtained for the cases in which the tight-binding inhibitor gives rise to competitive, noncompetitive, and uncompetitive inhibition with respect to one of the substrates of a bireactant mechanism.  From these equations it has been deduced that, in the presence of inhibitor, plots of I/υ against the reciprocals of the substrate concns. yield families of concave-down, nonrectangular hyperbolas.  The difficulties assocd. with the detn. of inhibition consts. for tight-binding inhibitors are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpj4s_N2iV3CLVg90H21EOLACvtfcHk0lhawkqMzNVXpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF1MXkvVOqsbo%253D&md5=74d467477385488dfbb053358534ada6</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2F0005-2744%2869%2990420-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0005-2744%252869%252990420-3%26sid%3Dliteratum%253Aachs%26aulast%3DMorrison%26aufirst%3DJ.%2BF.%26atitle%3DKinetics%2520of%2520the%2520reversible%2520inhibition%2520of%2520enzyme-catalysed%2520reactions%2520by%2520tight-binding%2520inhibitors%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D1969%26volume%3D185%26spage%3D269%26epage%3D286%26doi%3D10.1016%2F0005-2744%2869%2990420-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aragão, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aishima, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherukuvada, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarken, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clift, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowieson, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ericsson, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gee, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macedo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mudie, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panjikar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riboldi-Tunnicliffe, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rostan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caradoc-Davies, T. T.</span></span> <span> </span><span class="NLM_article-title">MX2: a high-flux undulator microfocus beamline serving both the chemical and macromolecular crystallography communities at the Australian Synchrotron</span>. <i>J. Synchrotron Radiat.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">885</span>– <span class="NLM_lpage">891</span>, <span class="refDoi"> DOI: 10.1107/s1600577518003120</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1107%2FS1600577518003120" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=29714201" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC1cXovVCnsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=885-891&author=D.+Arag%C3%A3oauthor=J.+Aishimaauthor=H.+Cherukuvadaauthor=R.+Clarkenauthor=M.+Cliftauthor=N.+P.+Cowiesonauthor=D.+J.+Ericssonauthor=C.+L.+Geeauthor=S.+Macedoauthor=N.+Mudieauthor=S.+Panjikarauthor=J.+R.+Priceauthor=A.+Riboldi-Tunnicliffeauthor=R.+Rostanauthor=R.+Williamsonauthor=T.+T.+Caradoc-Davies&title=MX2%3A+a+high-flux+undulator+microfocus+beamline+serving+both+the+chemical+and+macromolecular+crystallography+communities+at+the+Australian+Synchrotron&doi=10.1107%2Fs1600577518003120"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">MX2: a high-flux undulator microfocus beamline serving both the chemical and macromolecular crystallography communities at the Australian Synchrotron</span></div><div class="casAuthors">Aragao, David; Aishima, Jun; Cherukuvada, Hima; Clarken, Robert; Clift, Mark; Cowieson, Nathan Philip; Ericsson, Daniel Jesper; Gee, Christine L.; Macedo, Sofia; Mudie, Nathan; Panjikar, Santosh; Price, Jason Roy; Riboldi-Tunnicliffe, Alan; Rostan, Robert; Williamson, Rachel; Caradoc-Davies, Thomas Tudor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">885-891</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">1600-5775</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MX2 is an in-vacuum undulator-based crystallog. beamline at the 3 GeV Australian Synchrotron.  The beamline delivers hard X-rays in the energy range 4.8-21 keV to a focal spot of 22 × 12 μm FWHM (H × V).  At 13 keV the flux at the sample is 3.4 × 1012 photons s-1.  The beamline endstation allows robotic handling of cryogenic samples via an updated SSRL SAM robot.  This beamline is ideal for weakly diffracting hard-to-crystallize proteins, virus particles, protein assemblies and nucleic acids as well as smaller mols. such as inorg. catalysts and org. drug mols.  The beamline is now mature and has enjoyed a full user program for the last nine years.  This paper describes the beamline status, plans for its future and some recent scientific highlights.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgtd7iUcXqY7Vg90H21EOLACvtfcHk0lhawkqMzNVXpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXovVCnsr8%253D&md5=8a52a2db9f5d01958df27ae74e2496d3</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1107%2FS1600577518003120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS1600577518003120%26sid%3Dliteratum%253Aachs%26aulast%3DArag%25C3%25A3o%26aufirst%3DD.%26aulast%3DAishima%26aufirst%3DJ.%26aulast%3DCherukuvada%26aufirst%3DH.%26aulast%3DClarken%26aufirst%3DR.%26aulast%3DClift%26aufirst%3DM.%26aulast%3DCowieson%26aufirst%3DN.%2BP.%26aulast%3DEricsson%26aufirst%3DD.%2BJ.%26aulast%3DGee%26aufirst%3DC.%2BL.%26aulast%3DMacedo%26aufirst%3DS.%26aulast%3DMudie%26aufirst%3DN.%26aulast%3DPanjikar%26aufirst%3DS.%26aulast%3DPrice%26aufirst%3DJ.%2BR.%26aulast%3DRiboldi-Tunnicliffe%26aufirst%3DA.%26aulast%3DRostan%26aufirst%3DR.%26aulast%3DWilliamson%26aufirst%3DR.%26aulast%3DCaradoc-Davies%26aufirst%3DT.%2BT.%26atitle%3DMX2%253A%2520a%2520high-flux%2520undulator%2520microfocus%2520beamline%2520serving%2520both%2520the%2520chemical%2520and%2520macromolecular%2520crystallography%2520communities%2520at%2520the%2520Australian%2520Synchrotron%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2018%26volume%3D25%26spage%3D885%26epage%3D891%26doi%3D10.1107%2Fs1600577518003120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/s0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2Fs0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lhbW-e23RKDyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2Fs0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Panjikar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parthasarathy, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamzin, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, P. A.</span></span> <span> </span><span class="NLM_article-title">Auto-Rickshaw: an automated crystal structure determination platform as an efficient tool for the validation of an X-ray diffraction experiment</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">449</span>– <span class="NLM_lpage">457</span>, <span class="refDoi"> DOI: 10.1107/s0907444905001307</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1107%2FS0907444905001307" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=15805600" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BD2MXivVeisrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2005&pages=449-457&author=S.+Panjikarauthor=V.+Parthasarathyauthor=V.+S.+Lamzinauthor=M.+S.+Weissauthor=P.+A.+Tucker&title=Auto-Rickshaw%3A+an+automated+crystal+structure+determination+platform+as+an+efficient+tool+for+the+validation+of+an+X-ray+diffraction+experiment&doi=10.1107%2Fs0907444905001307"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Auto-Rickshaw: an automated crystal structure determination platform as an efficient tool for the validation of an X-ray diffraction experiment</span></div><div class="casAuthors">Panjikar, Santosh; Parthasarathy, Venkataraman; Lamzin, Victor S.; Weiss, Manfred S.; Tucker, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">D61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">449-457</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The EMBL-Hamburg Automated Crystal Structure Detn. Platform is a system that combines a no. of existing macromol. crystallog. computer programs and several decision-makers into a software pipeline for automated and efficient crystal structure detn.  The pipeline can be invoked as soon as x-ray data from derivatized protein crystals have been collected and processed.  It is controlled by a web-based graphical user interface for data and parameter input, and for monitoring the progress of structure detn.  A large no. of possible structure-soln. paths are encoded in the system and the optimal path is selected by the decision-makers as the structure soln. evolves.  The processes have been optimized for speed so that the pipeline can be used effectively for validating the x-ray expt. at a synchrotron beamline.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZbIxeGX0aqrVg90H21EOLACvtfcHk0lhbW-e23RKDyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXivVeisrk%253D&md5=98b93bf2e456104466b2e7b07e3ac197</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0907444905001307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444905001307%26sid%3Dliteratum%253Aachs%26aulast%3DPanjikar%26aufirst%3DS.%26aulast%3DParthasarathy%26aufirst%3DV.%26aulast%3DLamzin%26aufirst%3DV.%2BS.%26aulast%3DWeiss%26aufirst%3DM.%2BS.%26aulast%3DTucker%26aufirst%3DP.%2BA.%26atitle%3DAuto-Rickshaw%253A%2520an%2520automated%2520crystal%2520structure%2520determination%2520platform%2520as%2520an%2520efficient%2520tool%2520for%2520the%2520validation%2520of%2520an%2520X-ray%2520diffraction%2520experiment%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2005%26volume%3D61%26spage%3D449%26epage%3D457%26doi%3D10.1107%2Fs0907444905001307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Panjikar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parthasarathy, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamzin, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, P. A.</span></span> <span> </span><span class="NLM_article-title">On the combination of molecular replacement and single-wavelength anomalous diffraction phasing for automated structure determination</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">1089</span>– <span class="NLM_lpage">1097</span>, <span class="refDoi"> DOI: 10.1107/s0907444909029643</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1107%2FS0907444909029643" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=19770506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2009&pages=1089-1097&author=S.+Panjikarauthor=V.+Parthasarathyauthor=V.+S.+Lamzinauthor=M.+S.+Weissauthor=P.+A.+Tucker&title=On+the+combination+of+molecular+replacement+and+single-wavelength+anomalous+diffraction+phasing+for+automated+structure+determination&doi=10.1107%2Fs0907444909029643"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">On the combination of molecular replacement and single-wavelength anomalous diffraction phasing for automated structure determination</span></div><div class="casAuthors">Panjikar, Santosh; Parthasarathy, Venkataraman; Lamzin, Victor S.; Weiss, Manfred S.; Tucker, Paul A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1089-1097</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A combination of mol. replacement and single-wavelength anomalous diffraction phasing has been incorporated into the automated structure-detn. platform Auto-Rickshaw.  The complete MRSAD procedure includes mol. replacement, model refinement, exptl. phasing, phase improvement and automated model building.  The improvement over the std. SAD or MR approaches is illustrated by ten test cases taken from the JCSG diffraction data-set database.  Poor MR or SAD phases with phase errors larger than 70° can be improved using the described procedure and a large fraction of the model can be detd. in a purely automatic manner from X-ray data extending to better than 2.6 Å resoln.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoz21EsxIkqJLVg90H21EOLACvtfcHk0lhbW-e23RKDyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnP&md5=e3629b21fee4dc43450e853417e6b598</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1107%2FS0907444909029643&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909029643%26sid%3Dliteratum%253Aachs%26aulast%3DPanjikar%26aufirst%3DS.%26aulast%3DParthasarathy%26aufirst%3DV.%26aulast%3DLamzin%26aufirst%3DV.%2BS.%26aulast%3DWeiss%26aufirst%3DM.%2BS.%26aulast%3DTucker%26aufirst%3DP.%2BA.%26atitle%3DOn%2520the%2520combination%2520of%2520molecular%2520replacement%2520and%2520single-wavelength%2520anomalous%2520diffraction%2520phasing%2520for%2520automated%2520structure%2520determination%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2009%26volume%3D65%26spage%3D1089%26epage%3D1097%26doi%3D10.1107%2Fs0907444909029643" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebedev, A.</span></span> <span> </span><span class="NLM_article-title">MoRDa, an automatic molecular replacement pipeline</span>. <i>Acta Crystallogr., Sect. A: Found. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">S19</span>, <span class="refDoi"> DOI: 10.1107/s2053273315099672</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1107%2FS2053273315099672" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=25567570" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2015&pages=S19&author=A.+Vaginauthor=A.+Lebedev&title=MoRDa%2C+an+automatic+molecular+replacement+pipeline&doi=10.1107%2Fs2053273315099672"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1107%2FS2053273315099672&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2053273315099672%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DLebedev%26aufirst%3DA.%26atitle%3DMoRDa%252C%2520an%2520automatic%2520molecular%2520replacement%2520pipeline%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520A%253A%2520Found.%2520Adv.%26date%3D2015%26volume%3D71%26spage%3DS19%26doi%3D10.1107%2Fs2053273315099672" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brünger, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clore, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeLano, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gros, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuszewski, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilges, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simonson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, G. L.</span></span> <span> </span><span class="NLM_article-title">Crystallography & NMR System: A New Software Suite for Macromolecular Structure Determination</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">905</span>– <span class="NLM_lpage">921</span>, <span class="refDoi"> DOI: 10.1107/s0907444998003254</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1107%2FS0907444998003254" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=9757107" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A280%3ADyaK1cvjslKhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=1998&pages=905-921&author=A.+T.+Br%C3%BCngerauthor=P.+D.+Adamsauthor=G.+M.+Cloreauthor=W.+L.+DeLanoauthor=P.+Grosauthor=R.+W.+Grosse-Kunstleveauthor=J.+S.+Jiangauthor=J.+Kuszewskiauthor=M.+Nilgesauthor=N.+S.+Pannuauthor=R.+J.+Readauthor=L.+M.+Riceauthor=T.+Simonsonauthor=G.+L.+Warren&title=Crystallography+%26+NMR+System%3A+A+New+Software+Suite+for+Macromolecular+Structure+Determination&doi=10.1107%2Fs0907444998003254"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Crystallography & NMR system: A new software suite for macromolecular structure determination</span></div><div class="casAuthors">Brunger A T; Adams P D; Clore G M; DeLano W L; Gros P; Grosse-Kunstleve R W; Jiang J S; Kuszewski J; Nilges M; Pannu N S; Read R J; Rice L M; Simonson T; Warren G L</div><div class="citationInfo"><span class="NLM_cas:title">Acta crystallographica. Section D, Biological crystallography</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">Pt 5</span>),
    <span class="NLM_cas:pages">905-21</span>
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    </div><div class="casAbstract">A new software suite, called Crystallography & NMR System (CNS), has been developed for macromolecular structure determination by X-ray crystallography or solution nuclear magnetic resonance (NMR) spectroscopy.  In contrast to existing structure-determination programs, the architecture of CNS is highly flexible, allowing for extension to other structure-determination methods, such as electron microscopy and solid-state NMR spectroscopy.  CNS has a hierarchical structure: a high-level hypertext markup language (HTML) user interface, task-oriented user input files, module files, a symbolic structure-determination language (CNS language), and low-level source code.  Each layer is accessible to the user.  The novice user may just use the HTML interface, while the more advanced user may use any of the other layers.  The source code will be distributed, thus source-code modification is possible.  The CNS language is sufficiently powerful and flexible that many new algorithms can be easily implemented in the CNS language without changes to the source code.  The CNS language allows the user to perform operations on data structures, such as structure factors, electron-density maps, and atomic properties.  The power of the CNS language has been demonstrated by the implementation of a comprehensive set of crystallographic procedures for phasing, density modification and refinement.  User-friendly task-oriented input files are available for nearly all aspects of macromolecular structure determination by X-ray crystallography and solution NMR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRAeLCFcGBdlx0LsK7uFuY7fW6udTcc2eb8UXpaolNvZLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK1cvjslKhsw%253D%253D&md5=37afee1b07ac44d3f3730a39061b4e79</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1107%2FS0907444998003254&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444998003254%26sid%3Dliteratum%253Aachs%26aulast%3DBr%25C3%25BCnger%26aufirst%3DA.%2BT.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DClore%26aufirst%3DG.%2BM.%26aulast%3DDeLano%26aufirst%3DW.%2BL.%26aulast%3DGros%26aufirst%3DP.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DJiang%26aufirst%3DJ.%2BS.%26aulast%3DKuszewski%26aufirst%3DJ.%26aulast%3DNilges%26aufirst%3DM.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRice%26aufirst%3DL.%2BM.%26aulast%3DSimonson%26aufirst%3DT.%26aulast%3DWarren%26aufirst%3DG.%2BL.%26atitle%3DCrystallography%2520%2526%2520NMR%2520System%253A%2520A%2520New%2520Software%2520Suite%2520for%2520Macromolecular%2520Structure%2520Determination%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1998%26volume%3D54%26spage%3D905%26epage%3D921%26doi%3D10.1107%2Fs0907444998003254" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skubák, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebedev, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span> <span> </span><span class="NLM_article-title">REFMAC5 for the refinement of macromolecular crystal structures</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">355</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1107/s0907444911001314</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1107%2FS0907444911001314" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=21460454" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=355-367&author=G.+N.+Murshudovauthor=P.+Skub%C3%A1kauthor=A.+A.+Lebedevauthor=N.+S.+Pannuauthor=R.+A.+Steinerauthor=R.+A.+Nichollsauthor=M.+D.+Winnauthor=F.+Longauthor=A.+A.+Vagin&title=REFMAC5+for+the+refinement+of+macromolecular+crystal+structures&doi=10.1107%2Fs0907444911001314"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">REFMAC5 for the refinement of macromolecular crystal structures</span></div><div class="casAuthors">Murshudov, Garib N.; Skubak, Pavol; Lebedev, Andrey A.; Pannu, Navraj S.; Steiner, Roberto A.; Nicholls, Robert A.; Winn, Martyn D.; Long, Fei; Vagin, Alexei A.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">355-367</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">This paper describes various components of the macromol. crystallog. refinement program REFMAC5, which is distributed as part of the CCP4 suite.  REFMAC5 utilizes different likelihood functions depending on the diffraction data employed (amplitudes or intensities), the presence of twinning and the availability of SAD/SIRAS exptl. diffraction data.  To ensure chem. and structural integrity of the refined model, REFMAC5 offers several classes of restraints and choices of model parameterization.  Reliable models at resolns. at least as low as 4 Å can be achieved thanks to low-resoln. refinement tools such as secondary-structure restraints, restraints to known homologous structures, automatic global and local NCS restraints, 'jelly-body' restraints and the use of novel long-range restraints on at. displacement parameters (ADPs) based on the Kullback-Leibler divergence.  REFMAC5 addnl. offers TLS parameterization and, when high-resoln. data are available, fast refinement of anisotropic ADPs.  Refinement in the presence of twinning is performed in a fully automated fashion.  REFMAC5 is a flexible and highly optimized refinement package that is ideally suited for refinement across the entire resoln. spectrum encountered in macromol. crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5Jq-i2qIq5rVg90H21EOLACvtfcHk0liC6ediqrG61w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtbk%253D&md5=f8f3202d246908500057ad7c71015b7b</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1107%2FS0907444911001314&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911001314%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DSkub%25C3%25A1k%26aufirst%3DP.%26aulast%3DLebedev%26aufirst%3DA.%2BA.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DSteiner%26aufirst%3DR.%2BA.%26aulast%3DNicholls%26aufirst%3DR.%2BA.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DLong%26aufirst%3DF.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26atitle%3DREFMAC5%2520for%2520the%2520refinement%2520of%2520macromolecular%2520crystal%2520structures%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D355%26epage%3D367%26doi%3D10.1107%2Fs0907444911001314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">General quadratic functions in real and reciprocal space and their application to likelihood phasing</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1612</span>– <span class="NLM_lpage">1621</span>, <span class="refDoi"> DOI: 10.1107/s0907444900013263</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1107%2FS0907444900013263" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=11092927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BD3cXosVOksL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2000&pages=1612-1621&author=K.+Cowtan&title=General+quadratic+functions+in+real+and+reciprocal+space+and+their+application+to+likelihood+phasing&doi=10.1107%2Fs0907444900013263"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">General quadratic functions in real and reciprocal space and their application to likelihood phasing</span></div><div class="casAuthors">Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">D56</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1612-1621</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">A general multivariate quadratic function of the structure factors is constructed and transformed to obtain a quadratic function of the continuous electron d.  Two special cases, where structure factors are independent and where electron-d. values are independent, were examd.  These results are related to the new likelihood-based framework of Terwilliger [Terwilliger (1999), Acta Cryst. D55, pp. 1863-1871] for employing structural information which was previously exploited by conventional d.-modification calcns.  The treatment here involves different assumptions and highlights new features of Terwilliger's calcn.  The generalization quadratic construction gave cross terms relating all reflections and electron densities.  Other applications of this approach are considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMQJNDOqn0wrVg90H21EOLACvtfcHk0liC6ediqrG61w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXosVOksL0%253D&md5=fea050d02f0400be7dc294573908ef97</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1107%2FS0907444900013263&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444900013263%26sid%3Dliteratum%253Aachs%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DGeneral%2520quadratic%2520functions%2520in%2520real%2520and%2520reciprocal%2520space%2520and%2520their%2520application%2520to%2520likelihood%2520phasing%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2000%26volume%3D56%26spage%3D1612%26epage%3D1621%26doi%3D10.1107%2Fs0907444900013263" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Langer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, S. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamzin, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrakis, A.</span></span> <span> </span><span class="NLM_article-title">Automated macromolecular model building for X-ray crystallography using ARP/wARP version 7</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">1171</span>– <span class="NLM_lpage">1179</span>, <span class="refDoi"> DOI: 10.1038/nprot.2008.91</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1038%2Fnprot.2008.91" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=18600222" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BD1cXotFaku7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=1171-1179&author=G.+Langerauthor=S.+X.+Cohenauthor=V.+S.+Lamzinauthor=A.+Perrakis&title=Automated+macromolecular+model+building+for+X-ray+crystallography+using+ARP%2FwARP+version+7&doi=10.1038%2Fnprot.2008.91"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Automated macromolecular model building for X-ray crystallography using ARP/wARP version 7</span></div><div class="casAuthors">Langer, Gerrit; Cohen, Serge X.; Lamzin, Victor S.; Perrakis, Anastassis</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1171-1179</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">ARP/wARP is a software suite to build macromol. models in x-ray crystallog. electron d. maps.  Structural genomics initiatives and the study of complex macromol. assemblies and membrane proteins all rely on advanced methods for 3D structure detn.  ARP/wARP meets these needs by providing the tools to obtain a macromol. model automatically, with a reproducible computational procedure.  ARP/wARP 7.0 tackles several tasks: iterative protein model building including a high-level decision-making control module; fast construction of the secondary structure of a protein; building flexible loops in alternate conformations; fully automated placement of ligands, including a choice of the best-fitting ligand from a 'cocktail'; and finding ordered water mols.  All protocols are easy to handle by a nonexpert user through a graphical user interface or a command line.  The time required is typically a few minutes although iterative model building may take a few hours.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcDZXxfCGFTbVg90H21EOLACvtfcHk0lixXvsD13xM6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXotFaku7s%253D&md5=9b104e39902aee4db24be06e06707c63</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2008.91&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2008.91%26sid%3Dliteratum%253Aachs%26aulast%3DLanger%26aufirst%3DG.%26aulast%3DCohen%26aufirst%3DS.%2BX.%26aulast%3DLamzin%26aufirst%3DV.%2BS.%26aulast%3DPerrakis%26aufirst%3DA.%26atitle%3DAutomated%2520macromolecular%2520model%2520building%2520for%2520X-ray%2520crystallography%2520using%2520ARP%252FwARP%2520version%25207%26jtitle%3DNat.%2520Protoc.%26date%3D2008%26volume%3D3%26spage%3D1171%26epage%3D1179%26doi%3D10.1038%2Fnprot.2008.91" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/s0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2Fs0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lixXvsD13xM6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2Fs0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/s0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2Fs0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lixXvsD13xM6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2Fs0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/s0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2Fs0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0liYWK3PrYKGrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2Fs0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrero, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Junk, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, J. W.</span></span> <span> </span><span class="NLM_article-title">Bismuth(III) complexes derived from non-steroidal anti-inflammatory drugs and their activity against <i>Helicobacter pylori</i></span>. <i>Dalton Trans.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">2861</span>– <span class="NLM_lpage">2868</span>, <span class="refDoi"> DOI: 10.1039/c000164c</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1039%2Fc000164c" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=20200713" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADC%252BC3cXislaitb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2010&pages=2861-2868&author=P.+C.+Andrewsauthor=R.+L.+Ferreroauthor=P.+C.+Junkauthor=I.+Kumarauthor=Q.+Luuauthor=K.+Nguyenauthor=J.+W.+Taylor&title=Bismuth%28III%29+complexes+derived+from+non-steroidal+anti-inflammatory+drugs+and+their+activity+against+Helicobacter+pylori&doi=10.1039%2Fc000164c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Bismuth(III) complexes derived from non-steroidal anti-inflammatory drugs and their activity against Helicobacter pylori</span></div><div class="casAuthors">Andrews, Philip C.; Ferrero, Richard L.; Junk, Peter C.; Kumar, Ish; Luu, Quynh; Nguyen, Kim; Taylor, James W.</div><div class="citationInfo"><span class="NLM_cas:title">Dalton Transactions</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2861-2868</span>CODEN:
                <span class="NLM_cas:coden">DTARAF</span>;
        ISSN:<span class="NLM_cas:issn">1477-9226</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The formation of bismuth(III) complexes of carboxylates and benzoates derived from the 1 : 3 reaction of BiPh3 with the common non-steroidal anti-inflammatory drugs (NSAIDs) ketoprofen, naproxen, ibuprofen, mefenamic acid, diflunisal, 5-chlorosalicylic acid, fenbufen, sulindac, tolfenamic acid and flufenamic acid, has been achieved using both solvent-free and solvent-mediated methods.  The thermochem. profiles of the solvent-free reactions were studied using DSC-TGA.  All reactions produced the tris-substituted complexes of general formula [BiL3]n, with the complexes derived from ketoprofen and sulindac having an addnl. single bismuth bound H2O mol. in the inner coordination sphere.  The complexes are stable in air over a period of six months, do not undergo significant decompn. when suspended overnight in water, but decomp. in 1 M HCl soln. to release the free acid form of the NSAID.  All ten complexes show excellent in vitro activity against Helicobacter pylori with MIC values of ≥6.25 μg mL-1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9gUc3Zv-OerVg90H21EOLACvtfcHk0liYWK3PrYKGrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXislaitb8%253D&md5=810b668a0f30c61e33f2f5144fca725c</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1039%2Fc000164c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc000164c%26sid%3Dliteratum%253Aachs%26aulast%3DAndrews%26aufirst%3DP.%2BC.%26aulast%3DFerrero%26aufirst%3DR.%2BL.%26aulast%3DJunk%26aufirst%3DP.%2BC.%26aulast%3DKumar%26aufirst%3DI.%26aulast%3DLuu%26aufirst%3DQ.%26aulast%3DNguyen%26aufirst%3DK.%26aulast%3DTaylor%26aufirst%3DJ.%2BW.%26atitle%3DBismuth%2528III%2529%2520complexes%2520derived%2520from%2520non-steroidal%2520anti-inflammatory%2520drugs%2520and%2520their%2520activity%2520against%2520Helicobacter%2520pylori%26jtitle%3DDalton%2520Trans.%26date%3D2010%26volume%3D39%26spage%3D2861%26epage%3D2868%26doi%3D10.1039%2Fc000164c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jenks, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labigne, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrero, R. L.</span></span> <span> </span><span class="NLM_article-title">Exposure to metronidazole <i>in vivo</i> readily induces resistance in <i>Helicobacter pylori</i> and reduces the efficacy of eradication therapy in mice</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">777</span>– <span class="NLM_lpage">781</span>, <span class="refDoi"> DOI: 10.1128/aac.43.4.777</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10.1128%2FAAC.43.4.777" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=10103180" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;key=1%3ACAS%3A528%3ADyaK1MXisVSgurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=1999&pages=777-781&author=P.+J.+Jenksauthor=A.+Labigneauthor=R.+L.+Ferrero&title=Exposure+to+metronidazole+in+vivo+readily+induces+resistance+in+Helicobacter+pylori+and+reduces+the+efficacy+of+eradication+therapy+in+mice&doi=10.1128%2Faac.43.4.777"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Exposure to metronidazole in vivo readily induces resistance in Helicobacter pylori and reduces the efficacy of eradication therapy in mice</span></div><div class="casAuthors">Jenks, Peter J.; Labigne, Agnes; Ferrero, Richard L.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">777-781</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The H. pylori SS1 mouse model was used to characterize the development of resistance in H. pylori after treatment with metronidazole monotherapy and to examine the effect of prior exposure to metronidazole on the efficacy of a metronidazole-contg. eradication regimen.  Mice colonized with the metronidazole-sensitive H. pylori SS1 strain were treated for 7 days with either peptone trypsin broth or the mouse equiv. of 400 mg metronidazole once a day or 3 times per day (TID).  In a sep. expt., H. pylori-infected mice were administered either peptone trypsin broth or the mouse equiv. of 400 mg metronidazole TID for 7 days, followed 1 mo later by either peptone trypsin broth or the mouse equiv. of 20 mg omeprazole, 250 mg clarithromycin, and 400 mg metronidazole twice a day for 7 days.  At least 1 mo after the completion of treatment, the mice were sacrificed and their stomachs were cultured for H. pylori.  The susceptibilities of isolates to metronidazole were assessed by agar diln. detn. of the min. inhibitory concns. (MICs).  Mixed populations of metronidazole-resistant and -sensitive strains were isolated from 70% of the mice treated with 400 mg metronidazole TID.  The ratio of resistant to sensitive strains was 1:100, and the MICs for the resistant strains ranged 8-64 μg/mL.  In the 2nd expt., H. pylori was eradicated from 70% of the mice treated with eradication therapy alone, compared to 25% of those pretreated with metronidazole.  Mice still infected after treatment with metronidazole and eradication therapy contained mixed populations of metronidazole-resistant and -sensitive isolates in a ratio of 1:25.  These results demonstrate that H. pylori readily acquires resistance to metronidazole in vivo and that prior exposure of the organism to metronidazole is assocd. with failure of eradication therapy.  H. pylori-infected mice provide a suitable model for the study of resistance mechanisms in H. pylori and will be useful in detg. optimal regimens for the eradication of resistant strains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpSvrrQWXQxbVg90H21EOLACvtfcHk0liYWK3PrYKGrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXisVSgurs%253D&md5=92f4266a99969230fcfc4d4fb6d73ae9</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1128%2FAAC.43.4.777&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.43.4.777%26sid%3Dliteratum%253Aachs%26aulast%3DJenks%26aufirst%3DP.%2BJ.%26aulast%3DLabigne%26aufirst%3DA.%26aulast%3DFerrero%26aufirst%3DR.%2BL.%26atitle%3DExposure%2520to%2520metronidazole%2520in%2520vivo%2520readily%2520induces%2520resistance%2520in%2520Helicobacter%2520pylori%2520and%2520reduces%2520the%2520efficacy%2520of%2520eradication%2520therapy%2520in%2520mice%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1999%26volume%3D43%26spage%3D777%26epage%3D781%26doi%3D10.1128%2Faac.43.4.777" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RAD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RAD','PDB','4RAD'); return false;">PDB: 4RAD</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1A97" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1A97','PDB','1A97'); return false;">PDB: 1A97</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1G9S" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1G9S','PDB','1G9S'); return false;">PDB: 1G9S</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RHU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RHU','PDB','4RHU'); return false;">PDB: 4RHU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3OZG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3OZG','PDB','3OZG'); return false;">PDB: 3OZG</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1CJB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1CJB','PDB','1CJB'); return false;">PDB: 1CJB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3OZF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3OZF','PDB','3OZF'); return false;">PDB: 3OZF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5HIA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5HIA','PDB','5HIA'); return false;">PDB: 5HIA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4JIT" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4JIT','PDB','4JIT'); return false;">PDB: 4JIT</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RHX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4RHX','PDB','4RHX'); return false;">PDB: 4RHX</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BZY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BZY','PDB','1BZY'); return false;">PDB: 1BZY</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5BQP" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5BQP','PDB','5BQP'); return false;">PDB: 5BQP</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KL6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KL6','PDB','7KL6'); return false;">PDB: 7KL6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KL7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7KL7','PDB','7KL7'); return false;">PDB: 7KL7</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i40"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02184">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_60012"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c02184?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02184</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02184/suppl_file/jm0c02184_si_001.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02184/suppl_file/jm0c02184_si_001.csv">jm0c02184_si_001.csv (3.77 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c02184&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-9%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c02184%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c02184" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                10MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679974154e072390","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
