
<html lang="en"     class="pb-page"  data-request-id="3321c7c2-96c4-410a-a46d-46f7deb52dfa"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/jm900414x;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2009.52.issue-14"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="One Scaffold, Three Binding Modes: Novel and Selective Pteridine Reductase 1 Inhibitors Derived from Fragment Hits Discovered by Virtual Screening" /></meta><meta name="dc.Creator" content="Chidochangu P.  Mpamhanga" /></meta><meta name="dc.Creator" content="Daniel  Spinks" /></meta><meta name="dc.Creator" content="Lindsay B.  Tulloch" /></meta><meta name="dc.Creator" content="Emma J.  Shanks" /></meta><meta name="dc.Creator" content="David A.  Robinson" /></meta><meta name="dc.Creator" content="Iain T.  Collie" /></meta><meta name="dc.Creator" content="Alan H.  Fairlamb" /></meta><meta name="dc.Creator" content="Paul G.  Wyatt" /></meta><meta name="dc.Creator" content="Julie A.  Frearson" /></meta><meta name="dc.Creator" content="William N.  Hunter" /></meta><meta name="dc.Creator" content="Ian H.  Gilbert" /></meta><meta name="dc.Creator" content="Ruth  Brenk" /></meta><meta name="dc.Description" content="The enzyme pteridine reductase 1 (PTR1) is a potential target for new compounds to treat human African trypanosomiasis. A virtual screening campaign for fragments inhibiting PTR1 was carried out. T..." /></meta><meta name="Description" content="The enzyme pteridine reductase 1 (PTR1) is a potential target for new compounds to treat human African trypanosomiasis. A virtual screening campaign for fragments inhibiting PTR1 was carried out. T..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 15, 2009" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm900414x" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2009 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm900414x" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm900414x" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm900414x" /></link>
        
    
    

<title>One Scaffold, Three Binding Modes: Novel and Selective Pteridine Reductase 1 Inhibitors Derived from Fragment Hits Discovered by Virtual Screening | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm900414x" /></meta><meta property="og:title" content="One Scaffold, Three Binding Modes: Novel and Selective Pteridine Reductase 1 Inhibitors Derived from Fragment Hits Discovered by Virtual Screening" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0011.jpeg" /></meta><meta property="og:description" content="The enzyme pteridine reductase 1 (PTR1) is a potential target for new compounds to treat human African trypanosomiasis. A virtual screening campaign for fragments inhibiting PTR1 was carried out. Two novel chemical series were identified containing aminobenzothiazole and aminobenzimidazole scaffolds, respectively. One of the hits (2-amino-6-chloro-benzimidazole) was subjected to crystal structure analysis and a high resolution crystal structure in complex with PTR1 was obtained, confirming the predicted binding mode. However, the crystal structures of two analogues (2-amino-benzimidazole and 1-(3,4-dichloro-benzyl)-2-amino-benzimidazole) in complex with PTR1 revealed two alternative binding modes. In these complexes, previously unobserved protein movements and water-mediated protein−ligand contacts occurred, which prohibited a correct prediction of the binding modes. On the basis of the alternative binding mode of 1-(3,4-dichloro-benzyl)-2-amino-benzimidazole, derivatives were designed and selective PTR1 inhibitors with low nanomolar potency and favorable physicochemical properties were obtained." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm900414x"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm900414x">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm900414x&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm900414x&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm900414x&amp;href=/doi/10.1021/jm900414x" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2009</span><span class="cit-fg-volume">, 52</span><span class="cit-fg-issue">, 14</span><span class="cit-fg-pageRange">, 4454-4465</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/52/14" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm900367w" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm900431g" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">One Scaffold, Three Binding Modes: Novel and Selective Pteridine Reductase 1 Inhibitors Derived from Fragment Hits Discovered by Virtual Screening<a class="ref internalNav" href="#ac1" aria-label=""></a></span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Chidochangu+P.++Mpamhanga">Chidochangu P. Mpamhanga</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Daniel++Spinks">Daniel Spinks</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lindsay+B.++Tulloch">Lindsay B. Tulloch</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Emma+J.++Shanks">Emma J. Shanks</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=David+A.++Robinson">David A. Robinson</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Iain+T.++Collie">Iain T. Collie</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Alan+H.++Fairlamb">Alan H. Fairlamb</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Paul+G.++Wyatt">Paul G. Wyatt</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Julie+A.++Frearson">Julie A. Frearson</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=William+N.++Hunter">William N. Hunter</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ian+H.++Gilbert">Ian H. Gilbert</a></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ruth++Brenk">Ruth Brenk</a></span><span class="author-xref-symbol "><sup>*</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info"><span class="aff-text">Division of Biological Chemistry and Drug Discovery, College of Life Sciences, University of Dundee, Dow Street, Dundee DD1 5EH, U.K.</span></div><div class="corresp-info"><strong>*</strong>To whom correspondence should be addressed. Phone +44 1302 386230. E-mail: <a href="/cdn-cgi/l/email-protection#62104c0010070c092206170c0607074c03014c1709"><span class="__cf_email__" data-cfemail="daa8f4b8a8bfb4b19abeafb4bebfbff4bbb9f4afb1">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm900414x&amp;href=/doi/10.1021%2Fjm900414x" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2009</span></span><span class="cit-volume">, 52</span><span class="cit-issue">, 14</span><span class="cit-pageRange">, 4454–4465</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 15, 2009</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>31 March 2009</li><li><span class="item_label"><b>Published</b> online</span>15 June 2009</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 23 July 2009</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm900414x" title="DOI URL">https://doi.org/10.1021/jm900414x</a></div><div class="article_header-article-copyright"><strong>Copyright © 2009 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice_termsofuse.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm900414x"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3486</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">68</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm900414x" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;One Scaffold, Three Binding Modes: Novel and Selective Pteridine Reductase 1 Inhibitors Derived from Fragment Hits Discovered by Virtual Screening&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Chidochangu&quot;,&quot;last_name&quot;:&quot;P. Mpamhanga&quot;},{&quot;first_name&quot;:&quot;Daniel&quot;,&quot;last_name&quot;:&quot;Spinks&quot;},{&quot;first_name&quot;:&quot;Lindsay&quot;,&quot;last_name&quot;:&quot;B. Tulloch&quot;},{&quot;first_name&quot;:&quot;Emma&quot;,&quot;last_name&quot;:&quot;J. Shanks&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;A. Robinson&quot;},{&quot;first_name&quot;:&quot;Iain&quot;,&quot;last_name&quot;:&quot;T. Collie&quot;},{&quot;first_name&quot;:&quot;Alan&quot;,&quot;last_name&quot;:&quot;H. Fairlamb&quot;},{&quot;first_name&quot;:&quot;Paul&quot;,&quot;last_name&quot;:&quot;G. Wyatt&quot;},{&quot;first_name&quot;:&quot;Julie&quot;,&quot;last_name&quot;:&quot;A. Frearson&quot;},{&quot;first_name&quot;:&quot;William&quot;,&quot;last_name&quot;:&quot;N. Hunter&quot;},{&quot;first_name&quot;:&quot;Ian&quot;,&quot;last_name&quot;:&quot;H. Gilbert&quot;},{&quot;first_name&quot;:&quot;Ruth&quot;,&quot;last_name&quot;:&quot;Brenk&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2009&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;15&quot;,&quot;issue&quot;:&quot;14&quot;,&quot;volume&quot;:&quot;52&quot;,&quot;pages&quot;:&quot;4454-4465&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm900414x&quot;},&quot;abstract&quot;:&quot;The enzyme pteridine reductase 1 (PTR1) is a potential target for new compounds to treat human African trypanosomiasis. A virtual screening campaign for fragments inhibiting PTR1 was carried out. Two novel chemical series were identified containing aminobenzothiazole and aminobenzimidazole scaffolds, respectively. One of the hits (2-amino-6-chloro-benzimidazole) was subjected to crystal structure analysis and a high resolution crystal structure in complex with PTR1 was obtained, confirming the predicted binding mode. However, the crystal structures of two analogues (2-amino-benzimidazole and 1-(3,4-dichloro-benzyl)-2-amino-benzimidazole) in complex with PTR1 revealed two alternative binding modes. In these complexes, previously unobserved protein movements and water-mediated protein−ligand contacts occurred, which prohibited a correct prediction of the binding modes. On the basis of the alternative binding mode of 1-(3,4-dichloro-benzyl)-2-amino-benzimidazole, derivatives were designed and selective PTR1 inhi&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm900414x&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm900414x" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm900414x&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm900414x" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm900414x&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm900414x" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm900414x&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm900414x&amp;href=/doi/10.1021/jm900414x" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm900414x" /></input><a href="/doi/pdf/10.1021/jm900414x" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm900414x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm900414x%26sid%3Dliteratum%253Aachs%26pmid%3D19527033%26genre%3Darticle%26aulast%3DMpamhanga%26date%3D2009%26atitle%3DOne%2BScaffold%252C%2BThree%2BBinding%2BModes%253A%2BNovel%2Band%2BSelective%2BPteridine%2BReductase%2B1%2BInhibitors%2BDerived%2Bfrom%2BFragment%2BHits%2BDiscovered%2Bby%2BVirtual%2BScreening%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D52%26issue%3D14%26spage%3D4454%26epage%3D4465%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292580" title="Binding modes">Binding modes</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291880" title="Crystal structure">Crystal structure</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/52/14" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/medium/jm-2009-00414x_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm900414x&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The enzyme pteridine reductase 1 (PTR1) is a potential target for new compounds to treat human African trypanosomiasis. A virtual screening campaign for fragments inhibiting PTR1 was carried out. Two novel chemical series were identified containing aminobenzothiazole and aminobenzimidazole scaffolds, respectively. One of the hits (2-amino-6-chloro-benzimidazole) was subjected to crystal structure analysis and a high resolution crystal structure in complex with PTR1 was obtained, confirming the predicted binding mode. However, the crystal structures of two analogues (2-amino-benzimidazole and 1-(3,4-dichloro-benzyl)-2-amino-benzimidazole) in complex with PTR1 revealed two alternative binding modes. In these complexes, previously unobserved protein movements and water-mediated protein−ligand contacts occurred, which prohibited a correct prediction of the binding modes. On the basis of the alternative binding mode of 1-(3,4-dichloro-benzyl)-2-amino-benzimidazole, derivatives were designed and selective PTR1 inhibitors with low nanomolar potency and favorable physicochemical properties were obtained.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec articleNote"><h4><span>†</span>  Funding Statement</h4><p class="last">The crystal structures described in this paper have been deposited with the following PDB ID codes: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2WD7">2WD7</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2WD8">2WD8</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GN1">3GN1</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GN2">3GN2</a>.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51513" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51513" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Human African trypanosomiasis (HAT<a class="ref internalNav" href="#fn1" aria-label="a">a</a><div class="NLM_fn" id="fn1"><i>a</i><p class="last">Abbreviations: DHFR, dihydrofolate reductase; HAT, human African trypanosomiasis; <i>h</i>DHFR, human DHFR; <i>Lm</i>PTR1, <i>L. major</i> PTR1; PTR1, pteridine reductase 1; PSA, polar surface area; <i>Tb</i>DHFR, <i>T. brucei</i> DHFR; <i>Tb</i>PTR1, <i>T. brucei</i> PTR1.</p></div>), or sleeping sickness, claims the lives of at least 48000 people every year, and about 60 million people in sub-Saharan Africa are at risk of infection.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> HAT, transmitted by the tsetse fly, is caused by two subspecies of the protozoan parasite <i>Trypanosoma brucei</i>.<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2, 3)</a> The disease progresses in two stages. In the first stage, the parasites proliferate solely within the bloodstream and extracellular tissue space. In the late stage, the central nervous system (CNS) becomes infected, causing symptoms that are characteristic of the disease. If left untreated, HAT is always fatal. Current treatment is hampered by expensive and toxic drugs and emerging resistance and treatment failures complicate the treatment even further.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Accordingly, there is an urgent need for the development of safer and more efficient drugs.</div><div class="NLM_p">Promising targets for new therapeutics are enzymes involved in pterin and folate metabolism.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Both pterins and folates are essential for growth in <i>Leishmania</i> ssp. and related trypanosomatids, yet genes encoding enzymes for de novo synthesis are lacking from their respective genomes.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Therefore, trypanosomatids are required to salvage oxidized pteridines such as biopterin and folate and to subsequently reduce them to active cofactors such as tetrahydrobiopterin (H<sub>4</sub>B) and tetrahydrofolate (H<sub>4</sub>F) by means of pteridine reductase 1 (PTR1; EC 1.5.1.33) and the bifunctional enzyme dihydrofolate reductase-thymidylate synthase (DHFR-TS; EC 1.5.1.4 and 2.1.1.45, respectively).<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></div><div class="NLM_p">DHFR is a well established drug target for a range of diseases.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6, 7)</a> It is therefore surprising that antifolates commonly used as anticancer or anti-infective drugs have not shown equivalent efficacy against <i>T. brucei</i> or the related organism <i>L. major</i>.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8, 9)</a> However, for the latter organism, in which this metabolism has been more extensively investigated, it has been demonstrated that PTR1 not only reduces its principle substrates biopterin and dihydrobiopterin (H<sub>2</sub>B) but also dihydrofolate (H<sub>2</sub>F) and thus circumvents DHFR inhibition.<a onclick="showRef(event, 'ref8 ref10'); return false;" href="javascript:void(0);" class="ref ref8 ref10">(8, 10)</a> It is therefore possible that DHFR and PTR1 have to be inhibited cooperatively in order to a produce a therapeutic effect in <i>L. major.</i></div><div class="NLM_p">Recent evidence indicates that PTR1 might also be a <i>T. brucei</i> drug target in its own right. Unlike <i>L. major ptr1</i><sup><i>−</i></sup> mutants,<a onclick="showRef(event, 'ref8 ref11'); return false;" href="javascript:void(0);" class="ref ref8 ref11">(8, 11)</a> the bloodstream form of <i>T. brucei ptr1</i><sup><i>−</i></sup> mutants is no longer viable in culture medium, suggesting that PTR1 is essential for parasite survival (Sienkiewicz and Fairlamb, unpublished results). This observation prompted us to design <i>Tb</i>PTR1 inhibitors to support chemical validation of this potential target and to serve as leads for further development.</div><div class="NLM_p">Several PTR1 inhibitors are known. Most of them are derived from DHFR inhibitors and contain either a 2,4-diaminopteridine, 2,4-diaminoquinazoline, or 2,4-diaminopyrimidine core (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>).<a onclick="showRef(event, 'ref6 ref7 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref12 ref13">(6, 7, 12, 13)</a> It is therefore not surprising that many PTR1 inhibitors also inhibit human and <i>T. brucei</i> DHFR with inhibition constants in the low micromolar to nanomolar range.<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12, 13)</a> This broad spectrum activity is undesirable both for chemical tools and lead compounds: first, effects due to PTR1 inhibition cannot be distinguished from those due to <i>Tb</i>DHFR inhibition and, second, inhibition of <i>h</i>DHFR will probably lead to undesirable side effects. Moreover, these core structures have a relatively high polar surface area (PSA), ranging from 77 Å<sup>2</sup> to over 100 Å<sup>2</sup>. These types of compounds are therefore less suitable starting points for drug design, particularly when CNS active drugs are required to treat the second stage of the disease.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/medium/jm-2009-00414x_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Common core structures of PTR1 and DHFR inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm900414x&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Crystal structures of <i>Lm</i>PTR1 and <i>Tb</i>PTR1 have been determined.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15, 16)</a> The substrate binding site is a well-defined cleft (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>). In the majority of the crystal structures, the aromatic heterocycle of the ligands is sandwiched between the nicotinamide part of the cofactor NADP<sup>+</sup> and the aromatic moiety of Phe97 (<i>T. brucei</i> numbering). Further, the ligands form extensive hydrogen bonds with the cofactor and surrounding amino acids. Even relatively small compounds such as 6-methylpteridine-2,4-diamine (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>) are potent <i>Lm</i>PTR1 inhibitors with inhibition constants in the low micromolar range.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Collectively, these data suggest that the PTR1 binding pocket is well suited for a fragment-based strategy for hit discovery.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/medium/jm-2009-00414x_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Ligands and binding site residues of <i>Tb</i>PTR1-methotrexate complex (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2C7V">2C7V</a>, blue carbon atoms) superimposed with those from the triamterene complex (green carbon atoms).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm900414x&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In fragment-based hit discovery, libraries are screened that typically contain molecules with a molecular mass less than 300 Da and with less than three hydrogen-bond donors and six hydrogen-bond acceptors.<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17, 18)</a> These constraints support the identification of small ligands that bind with a high ligand efficiency and that can be readily optimized to potent drug-like inhibitors. Usually, biophysical methods such as X-ray crystallography, nuclear magnetic resonance (NMR), or surface plasmon resonance (SPR) are used to detect the initial hits, which often have binding affinities in the low millimolar to low micromolar range. In a few cases, also computational methods have been applied to predict fragment binding.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></div><div class="NLM_p last">Here, we describe our efforts to support chemical validation of <i>Tb</i>PTR1 as a drug target. We report on the discovery of novel <i>Tb</i>PTR1 inhibitors using computational fragment screening. The implications of this study for PTR1 inhibitor design and virtual fragment screening are discussed.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85526" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85526" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Virtual Screening</h3><div class="NLM_p">The following strategy was adopted for virtual screening: (1) selection of an appropriate <i>Tb</i>PTR1 structure as template for docking; (2) compilation of a fragment library; (3) docking of the fragment library into the active site of the crystal structure; (4) selection of hits using a combination of pharmacophore-filtering and visual inspection.</div><div class="NLM_p">Superposition of the available <i>Tb</i>PTR1 crystal structures revealed that the binding site is rather rigid, and the observed movements are within the crystallographic coordinate error (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>). The choice of crystal structure should therefore not significantly influence the docking results, and the crystal structure of <i>Tb</i>PTR1 in complex with triamterene and NADP<sup>+</sup> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BME">3BME</a>) was used as template for virtual screening.</div><div class="NLM_p">With one exception, in all available <i>Tb</i>PTR1 crystal structures, the ligands form a similar hydrogen-bonding network as observed in the methotrexate and triamterene complexes (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>). However, if in a pharmacophore hypothesis all hydrogen bonds were required, only compounds resembling the known inhibitors and hence with the same problematic physicochemical properties would be retrieved. Therefore, the pharmacophore criteria were relaxed to tolerate all compounds that can interact with the β-phosphate group of the cofactor and can form any of the additional hydrogen bonds.</div><div class="NLM_p">Recently, we reported on the assembly of a database containing commercially available <i>lead-like</i> compounds.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> To obtain a library of <i>fragment-like</i> compounds, this lead-like set was further filtered for compounds containing fewer than 20 heavy atoms, only one or two ring systems, at least one hydrogen-bond donor group, fewer than four rotatable bonds, and a ClogP/ClogD of less than 3.5. By applying these filters, the initial set containing more than 250000 molecules was reduced by approximately 90%.</div><div class="NLM_p last">The resulting fragment library was sequentially docked into the <i>Tb</i>PTR1 binding site using DOCK 3.5.54<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20, 21)</a> A binding mode was generated for 25386 of the 26084 compounds contained in the set. Subsequent filtering of the predicted orientations with the pharmacophore hypothesis described above resulted in 2725 compounds. Compounds containing known PTR1 inhibitor scaffolds (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>) were removed, and the remaining compounds were grouped into five different clusters based on which of the hydrogen bonding interactions previously observed in PTR1−ligand complexes (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>) they fulfilled. The clusters were visually inspected in order to identify compounds representing a diverse set of scaffolds. In addition, the PSA for the scaffold involved in forming the key hydrogen bonds should ideally be less than 70 Å<sup>2</sup>. The selected compounds were subsequently minimized in the binding site while keeping the protein rigid. The criteria for compound purchase were quality of the hydrogen-bond network after minimization and shape complementarity of the ligand and the binding site. Finally, 59 compounds were selected for testing, out of which 45 compounds were available for purchase (<a class="ref internalNav" href="#notes-1" aria-label="Table S1, Supporting Information">Table S1, Supporting Information</a>).</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Hit Evaluation</h3><div class="NLM_p last">PTR1 inhibition was measured using a novel assay platform recently developed by our group, details of which will be reported in a separate publication. Given the low molecular weight of the screening hits, inhibition was initially tested at 100 μM. Three fragments representing three different chemical scaffolds demonstrated more than 50% inhibition and a further seven compounds from three additional series showed between 30 and 50% inhibition (<a class="ref internalNav" href="#notes-1" aria-label="Table S1, Supporting Information">Table S1, Supporting Information</a>). The hit compounds from the two most potent series of this initial testing containing aminobenzothiazole or aminobenzimidazole cores were further evaluated.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Aminobenzothiazole Series</h3><div class="NLM_p">For the aminobenzothiazole series, three compounds were selected for purchase (<b>1</b>−<b>3</b>, Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>), but only two were available (<b>2</b> and <b>3</b>). They have apparent <i>K</i><sub>i</sub> (<i>K</i><sub>i</sub><sup>app</sup>) values of around 21 and 141 μM, giving ligand efficiencies<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> of 0.46 and 0.29 kcal mol<sup>−1</sup>/heavy atom, respectively. The PSA of their core scaffold is 39 Å<sup>2</sup> (60−65 Å<sup>2</sup> for the whole molecule), which is 38 to 61 Å<sup>2</sup> lower than that of the previously known PTR1 inhibitor cores (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>). The pairwise Tanimoto coefficients between previously known PTR1 and DHFR inhibitors scaffolds and these compounds are only 0.1−0.2.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Docking Results, PTR1 Inhibition Data, and PSA of Virtual Screening Hits<a class="ref internalNav" href="#tbl1-fn1" aria-label="b">b</a></div></div></div><div class="scrollable-table-wrap"><table class="table single-image" border="0"><colgroup><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><tbody><tr valign="top"><td colspan="6" align="center" valign="bottom"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/medium/jm-2009-00414x_0012.gif" alt="" id="_i7" /></img></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">The cores of compounds <b>1</b>−<b>3</b> were defined as 2-aminobenzothiazole and of compound <b>4</b> as 2-aminobenzimidazole.</p></div><div class="footnote" id="tbl1-fn1"><sup>b</sup><p class="last"><i>K</i><sub><i>i</i></sub><sup>app</sup> and Hill slopes are averaged over at least two independent measurements.</p></div></div></div><div class="NLM_p">All attempts to cocrystallize these ligands with PTR1 or to soak them into crystals were unsuccessful. This is most likely due to the limited solubility of these compounds. In the proposed binding mode, the ligand is sandwiched between the aromatic moiety of Phe97 and the nicotinamide part of NADP<sup>+</sup>. Further, the ligand forms one hydrogen bond with Ser95 and two with the cofactor (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/medium/jm-2009-00414x_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Modeled binding mode of <b>2</b>. The ligands is sandwiched between the aromatic moiety of Phe97 and the nicotinamide part of NADP<sup>+</sup>. Further, the ligand forms hydrogen bonds with Ser95 and the cofactor.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm900414x&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Aminobenzimidazole Series</h3><div class="NLM_p">In the first instance, only one compound containing an aminobenzimidazole scaffold was purchased (<b>4</b>, Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). This is the most potent compound discovered from this virtual fragment screen, with a <i>K</i><sub>i</sub><sup>app</sup> value of 10.6 μM and a ligand efficiency of 0.62 kcal mol<sup>−1</sup>/heavy atom. The compound has a low PSA of 55 Å<sup>2</sup> and Tanimoto coefficients of 0.3−0.5 when compared with previously known PTR1 and DHFR inhibitor scaffolds.</div><div class="NLM_p">In the proposed binding mode, the ligand is protonated at the nitrogen atom facing the phosphate group of NADP<sup>+</sup> and forms hydrogen bonds with Ser95 and the cofactor (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>b). To initially validate this binding mode in the absence of a crystal structure, close analogues were tested with variations in the six position (<b>5</b> and <b>6</b>) and on N1 (<b>7</b>−<b>9</b>, Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). Removing the chloro-substituent (<b>5</b>) was accompanied by a more than 20-fold drop in affinity, whereas replacing this substituent with a methylphenyl group (<b>6</b>) resulted in an equipotent compound. Consistent with the proposed binding mode addition of an ethyl group in the N1 (<b>7</b>) position diminished affinity, as this substituent prevents hydrogen-bonding interaction with the phosphate group of NADP<sup>+</sup>. In contrast, larger substituents on this position restored affinity (<b>8</b>) or even led to more than a 20-fold increase in affinity (<b>9</b>) compared to the virtual screening hit.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/medium/jm-2009-00414x_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0005.jpeg" id="_i9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (a) View in the binding site of PTR1·<b>4</b> together with <i>F</i><sub>o</sub> − <i>F</i><sub>c</sub> omit map (contoured on 2.0σ), which was calculated by omitting the ligand from the final model. The ligand adopts two distinct binding modes. In the major confirmation (green carbon atoms), the chloro-substituent packs against Leu209 and Pro210, whereas in the minor conformation (yellow carbon atoms), the chloro atom sits in the open cavity of the active site. (b) Modeled binding mode of <b>4</b> (orange carbon atoms) superimposed with the dominant binding mode determined crystallographically (green carbon atoms).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm900414x&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. PTR1 Inhibition Data of Readily Available Analogues of 2-Amino-6-chloro-benzimidazole (<b>4</b>) <a class="ref internalNav" href="#tbl2-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table single-image" border="0"><colgroup><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><tbody><tr valign="top"><td colspan="4" align="center" valign="bottom"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/medium/jm-2009-00414x_0013.gif" alt="" id="_i11" /></img></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl2-fn1"><div class="footnote" id="tbl2-fn1"><sup>a</sup><p class="last"><i>K</i><sub>i</sub><sup>app</sup> and Hill slopes are averaged over at least two independent measurements.</p></div></div></div><div class="NLM_p">The binding modes of <b>4</b>,<b> 5</b>, and <b>9</b> were determined using X-ray crystallography (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>). Ligand <b>4</b> adopts two distinct binding modes with either the N1 or N3 being protonated and forming a hydrogen bond with the phosphate group of the cofactor (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>a). The ratio of these binding modes is 70:30 as estimated by crystallographic <i>B</i> factors. In the major conformation the chloro-substituent packs against a hydrophobic surface formed by Leu209 and Pro210, whereas in the minor conformation, the chloro atom sits in the open cavity of the active site and forms no van der Waals contacts. The major binding mode resembles closely the best scoring binding mode of this ligand predicted by DOCK 3.5 (rmsd = 0.95 Å, Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>b). The tautomeric form of <b>4</b> required in the minor binding mode was present in the docking database but not stored in the final hit list because only the highest scoring version of each compound was kept. The predicted binding mode of this tautomer is within one Å rmsd of the crystallographically determined minor binding mode of <b>4</b> (data not shown).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Crystallographic Data and Refinement Statistics of <i>Tb</i>PTR1−Ligand Complexes<a class="ref internalNav" href="#tbl3-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="rowsep1"><th class="rowsep1" align="left">ligand complex</th><th class="rowsep1" align="center"><b>4</b></th><th class="rowsep1" align="center"><b>5</b></th><th class="rowsep1" align="center"><b>9</b></th><th class="rowsep1" align="center"><b>12</b></th></tr></thead><tbody><tr valign="top"><td class="colsep1" colspan="5" align="left">Details of Data Collection</td></tr><tr valign="top"><td align="left">PDB code</td><td align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2WD7">2WD7</a></td><td align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GN1">3GN1</a></td><td align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GN2">3GN2</a></td><td align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2WD8">2WD8</a></td></tr><tr valign="top"><td align="left">space group</td><td align="left"><i>P</i>2<sub>1</sub></td><td align="left"><i>P</i>2<sub>1</sub></td><td align="left"><i>P</i>2<sub>1</sub></td><td align="left"><i>P</i>2<sub>1</sub></td></tr><tr valign="top"><td align="left">unit cell dimensions (Å)</td><td align="left"><i>a</i> = 74.68</td><td align="left"><i>a</i> = 74.64</td><td align="left"><i>a</i> = 74.89</td><td align="left"><i>a</i> = 74.66</td></tr><tr valign="top"><td align="left"> </td><td align="left"><i>b</i> = 89.89</td><td align="left"><i>b</i> = 90.41</td><td align="left"><i>b</i> = 90.78</td><td align="left"><i>b</i> = 89.89</td></tr><tr valign="top"><td align="left"> </td><td align="left"><i>c</i> = 82.70</td><td align="left"><i>c</i> = 82.64</td><td align="left"><i>c</i> = 82.86</td><td align="left"><i>c</i> = 83.05</td></tr><tr valign="top"><td align="left"> </td><td align="left">β = 115.48</td><td align="left">β = 115.73</td><td align="left">β = 115.85</td><td align="left">β = 115.54</td></tr><tr valign="top"><td align="left">resolution range (Å)</td><td align="left">30.0−1.90</td><td align="left">30.66−2.00</td><td align="left">67.42−1.60</td><td align="left">30.0−2.10</td></tr><tr valign="top"><td align="left"> </td><td align="left">(2.0−1.90)</td><td align="left">(2.10−2.00)</td><td align="left">(1.69−1.60)</td><td align="left">(2.18−2.10)</td></tr><tr valign="top"><td align="left">observations</td><td align="left">147241</td><td align="left">341528</td><td align="left">522950</td><td align="left">118137</td></tr><tr valign="top"><td align="left">unique observations</td><td align="left">71041</td><td align="left">63246</td><td align="left">124512</td><td align="left">48991</td></tr><tr valign="top"><td align="left">redundancy</td><td align="left">2.1</td><td align="left">5.4</td><td align="left">4.2</td><td align="left">2.4</td></tr><tr valign="top"><td align="left">completeness (%)</td><td align="left">91.6 (63.8)</td><td align="left">94.8 (89.9)</td><td align="left">94.9 (93.7)</td><td align="left">84.7 (84.9)</td></tr><tr valign="top"><td align="left">⟨<i>I</i>/σ(<i>I</i>)⟩</td><td align="left">13.0 (2.4)</td><td align="left">9.3 (5.1)</td><td align="left">8.3 (2.0)</td><td align="left">11.5 (2.43)</td></tr><tr valign="top"><td align="left"><i>R</i><sub>merge</sub><a class="ref internalNav" href="#t3fn1" aria-label="b">b</a> (%)</td><td align="left">5.4 (31.1)</td><td align="left">5.1 (13.6)</td><td align="left">6.7 (35.9)</td><td align="left">9.4 (56.9)</td></tr><tr valign="top"><td class="rowsep1" colspan="5" align="center">Refinement Statistics</td></tr><tr valign="top"><td align="left">resolution range (Å)</td><td align="left">30.0−1.9</td><td align="left">30.66−2.00</td><td align="left">67.42−1.60</td><td align="left">30.0−2.10</td></tr><tr valign="top"><td align="left"><i>R</i> factor<a class="ref internalNav" href="#t3fn2" aria-label="c">c</a> % (<i>R</i><sub>work</sub>/<i>R</i><sub>free</sub>)</td><td align="left">15.5/20.2</td><td align="left">15.5/21.6</td><td align="left">14.6/17.8</td><td align="left">15.8/22.6</td></tr><tr valign="top"><td align="left">number of atoms<a class="ref internalNav" href="#t3fn3" aria-label="d">d</a></td><td align="left">7545/195/88/687</td><td align="left">7480/192/20/896</td><td align="left">7568/192/95/1272</td><td align="left">7453/144/100/733</td></tr><tr valign="top"><td align="left">mean <i>B</i> factor<a class="ref internalNav" href="#t3fn4" aria-label="e">e</a> (Å<sup>2</sup>)</td><td align="left">26/21/27/35</td><td align="left">12/9/26/21</td><td align="left">11/8/11/27</td><td align="left">30/30/28/41</td></tr><tr valign="top"><td align="left">rms bond length deviation (Å)</td><td align="left">0.016</td><td align="left">0.014</td><td align="left">0.009</td><td align="left">0.012</td></tr><tr valign="top"><td align="left">rms bond angle deviation (deg)</td><td align="left">1.627</td><td align="left">1.473</td><td align="left">1.293</td><td align="left">1.419</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl3-fn1"><div class="footnote" id="tbl3-fn1"><sup>a</sup><p class="last">Values between brackets are for the highest resolution shell.</p></div><div class="footnote" id="t3fn1"><sup>b</sup><p class="last"><i>R</i><sub>merge</sub> = ∑|<i>I</i> − ⟨<i>I</i>⟩|/∑⟨<i>I</i>⟩.</p></div><div class="footnote" id="t3fn2"><sup>c</sup><p class="last"><i>R</i> factor = ∑ <i>F</i><sub>o</sub>− <i>F</i><sub>c</sub>|/∑ <i>F</i><sub>o</sub>.</p></div><div class="footnote" id="t3fn3"><sup>d</sup><p class="last">Number of atoms of protein, cofactor, ligand, and water molecules, respectively.</p></div><div class="footnote" id="t3fn4"><sup>e</sup><p class="last">Mean <i>B</i> factor for protein, cofactor, ligand, and water molecules, respectively</p></div></div></div><div class="NLM_p">For the unsubstituted fragment <b>5</b> only one binding mode was detected (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>). In this binding mode, the ligand forms an edge−face interaction with Trp221 and water mediated hydrogen bonds with the cofactor. The water molecule interacting with the α-phosphate group of the cofactor occupies a similar position as a water molecule in a <i>Lm</i>PTR1 ligand complex determined previously.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> The other water molecule has not been observed in any of the previously published structures. Structural water molecules were not considered in the docking protocol and binding mode prediction of <b>5</b> failed (rmsd = 3.04 Å).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/medium/jm-2009-00414x_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Crystallographically determined binding mode of <b>5</b> together with <i>F</i><sub>o</sub> − <i>F</i><sub>c</sub> omit map (contoured on 2.0σ), which was calculated by omitting the ligand from the final model. Two water molecules mediate contacts between the phosphate groups of the cofactor and the ligand.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm900414x&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The most potent ligand <b>9</b> adopts a binding mode, which does not resemble any previously observed binding mode (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>). This ligand binds in an area of the active site perpendicular to the canonical binding mode, 3.7 Å away from the nicotinamide moiety of the cofactor. The core occupies a pocket formed by Phe97, Asp161, Met163, Cys168, Phe171, Tyr174, and Gly205. The ligand forms a bidentate interaction with the carboxyl group of Asp161 via the protonated N3 and the 2-amino group, which also hydrogen bonds to the backbone carbonyl of Gly205. The dichloro-phenyl moiety is placed in a pocket defined by the side chains of Val206, Trp221, Leu263, Cys168, and Met163. The pocket is closed by the C-terminal residues His267 and Asp268 from a neighboring subunit. The ligand does not form any hydrogen bonds with the cofactor. The phosphate group of NADP<sup>+</sup> is surrounded by water molecules, which occupy similar positions as in PTR1·<b>5</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>).</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/medium/jm-2009-00414x_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Crystallographically determined binding mode of <b>9</b> (orange carbon atoms for ligand and blue carbon atoms for protein) superimposed with the receptor conformation used for docking calculations (gray carbon atoms) together with <i>F</i><sub>o</sub> − <i>F</i><sub>c</sub> omit map (contoured on 2.0σ), which was calculated by omitting the ligand from the final model. To accommodate the ligand in the binding site, the side chain of Trp221 rotated by 10° and the sulfur atom of Cys168 shifted by 2 Å.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm900414x&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In one monomer (chain D) of the PTR·<b>9</b> complex, an additional molecule of <b>9</b> was observed in the substrate binding site at lower occupancy. In this binding mode the dichloro-phenyl moiety of the ligand is sandwiched between the nicotinamide part of the cofactor and Phe97 while the benzimidazole part is hydrogen-bonded to the phosphate groups of the cofactor displacing a near-by loop (Leu208-Ala212). This molecule was only modeled with an occupancy of 0.75. The largely hydrophobic nature of the interactions formed within the substrate binding site coupled with the conformational change required for binding suggests that the binding mode can be disregarded as a crystallographic artifact.</div><div class="NLM_p">Despite the 1-substituted analogue occupying a very different area of the active site in PTR1 in its main binding mode, the overall rmsd between the main chain atoms of the <i>Tb</i>PTR1·<b>4</b> and <i>Tb</i>PTR1·<b>9</b> complexes is only 0.25 Å (calculated for chain A), indicating that there is negligible gross conformational change. Considering residues within the active site region the rmsd is 0.43 Å for main chain atoms and 0.65 Å for all atoms, respectively, reflecting the limited nature of ligand induced conformational change. However, the torsion angle defined by CA-CB-CD-CD2 of Trp221 changes from −65° to −75° (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>). Without this change, the dichloro moiety of <b>9</b> would clash with the tryptophan side chain. In addition, the sulfur atom of Cys168 is shifted by 2 Å in order to accommodate the aminobenzimidazole moiety.</div><div class="NLM_p">Receptor flexibility was not considered during docking and the docking program was set up to place a ring system close to the phosphate group of the core factor. As a result, a low scoring binding mode was obtained for <b>9</b>, which was inconsistent with the above-described pharmacophore hypothesis and which did not resemble the experimentally detected one.</div><div class="NLM_p">All attempts to determine the binding mode of <b>6</b> failed, probably due to the limited solubility of this compound. Most likely <b>6</b> adopts a similar binding mode as <b>4</b> with the methylphenyl substituent located in the same area of the binding site as the chloro-substituent of the screening hits. Adopting one of the alternative binding modes would either lead to a steric clash of the aromatic substituent with Trp221 or Phe171.</div><div class="NLM_p">In summary, depending on the substituents the aminobenzimidazole core adopts three distinct binding modes in the PTR1 active site (Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>). Only one of them (<b>4</b>) resembles the previously observed binding modes (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>).</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/medium/jm-2009-00414x_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Superposition of the crystallographically determined binding modes of <b>4</b> (orange carbon atoms), <b>5</b> (green carbon atoms), and <b>9</b> (gray carbon atoms). Depending on the substituents, the amino-benzimidazole core adopts three different binding modes.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm900414x&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Hit Expansion of Aminobenzimidazole Series</h3><div class="NLM_p">Crystal structure analysis of <i>Tb</i>PTR1·<b>9</b> revealed two hydrophobic pockets filled with structural water molecules next to the aminobenzimidazole core of <b>9</b>: a larger pocket toward Trp221 and a smaller pocket bordered by Pro167, Tyr174, and Asn175 (Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a>). To improve affinity of the screening hits, derivatives with moieties suitable to fill the larger pocket were designed (Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>). The 7-chloro-derivative <b>10</b> has a <i>K</i><sub>i</sub><sup>app</sup> of 0.5 μM and is equipotent to <b>9</b>. Substituting the chloro-atom with a propoxy-group (<b>11</b>) led to a 10-fold increase in potency resulting in a <i>K</i><sub>i</sub><sup>app</sup> of 0.05 μM. Introducing a phenyl group in this position afforded <b>12</b>. With a <i>K</i><sub>i</sub><sup>app</sup> of 7 nM this was the most potent compound in the series.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/medium/jm-2009-00414x_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Cut-away view of solvent accessible surface of the binding pocket of PTR1·<b>9</b>. Two hydrophobic pockets filled with ordered water molecules are close the aminobenzimidazole core of <b>9</b>, a smaller pocket bordered by Pro167, Tyr174, and Asn175, and a larger one toward Trp221.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm900414x&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Hit Expansion of the Aminobenzimidazole Series<a class="ref internalNav" href="#tbl4-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table single-image" border="0"><colgroup><col align="center" /></col></colgroup><tbody><tr valign="top"><td align="center" valign="bottom"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/medium/jm-2009-00414x_0001.gif" alt="" id="_i15" /></img></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl4-fn1"><div class="footnote" id="tbl4-fn1"><sup>a</sup><p class="last"><i>K</i><sub><i>i</i></sub><sup>app</sup> and Hill slopes are averaged over at least two independent measurements.</p></div></div></div><div class="NLM_p">The crystal structure of <i>Tb</i>PTR1·<b>12</b> confirmed the modeled binding mode (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>, Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a>). As predicted, the inhibitor binds in a similar manner as the parent compound <b>9</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>). The additional phenyl ring lies between the side chains of Phe97 and Trp221, forming and edge−face interaction with the latter residue and displacing the structural water molecules, which were found in this area in <i>Tb</i>PTR1·<b>9</b>.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/medium/jm-2009-00414x_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Crystallographically determined binding mode of <b>12</b> together with <i>F</i><sub>o</sub> − <i>F</i><sub>c</sub> omit map (contoured on 2.0σ), which was calculated by omitting the ligand from the final model. The phenyl group of the ligand forms an edge−face interaction with Trp221.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm900414x&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">All compounds showed no significant inhibition of <i>T. bruce</i>i or human DHFR when tested at concentrations of up to 30 μM when their <i>K</i><sub>i</sub><sup>app</sup> values were below 1 μM and up to 500 μM for all other compounds. The most potent inhibitor <b>12</b> has favorable physicochemical properties for cell and CNS penetration with a molecular weight of 368 Da, a PSA of 55 Å, and an experimentally determined logD of 3.7 units. However, when compound <b>12</b> was assayed against <i>T. brucei</i> in cell culture, the EC<sub>50</sub> value obtained was only 10 μM, despite the compound having a <i>K</i><sub>i</sub><sup>app</sup> of 7 nM against PTR1.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71054" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71054" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p"><i>T. brucei</i> PTR1 has recently been genetically validated as a drug target for HAT (Sienkiewicz and Fairlamb, unpublished results). Here, we were interested in developing inhibitors of the enzyme to facilitate chemical validation of PTR1 as a drug target for HAT and also to provide drug leads. The leads should have suitable physicochemical properties for further optimization; in the case of stage two HAT, this includes being able to penetrate the CNS.<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2, 3)</a> One property that is generally needed for such molecules is a low polar surface area (<70 Å<sup>2</sup>).<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Published inhibitors of PTR1 suffer from low solubility and a relatively high PSA, which reduces the possibility of blood−brain barrier permeation. Furthermore, known PTR1 inhibitors also often inhibit DHFR, which may give rise to toxicity.<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12, 13)</a> Therefore we aimed to discover novel scaffolds with a low PSA and that inhibit PTR1 and are selective over DHFR. To achieve these goals, we used a virtual fragment screening strategy followed by structure-based ligand design.</div><div class="NLM_p">We wish to emphasize three results from this study: (1) Virtual fragment screening was successfully applied to discover two novel PTR1 inhibitor scaffolds. (2) X-ray crystallography could confirm the binding mode of one of the inhibitors, but unexpected binding modes were also observed. (3) A highly selective and potent <i>Tb</i>PTR1 inhibitor was designed that can now serve as a chemical tool for target validation.</div><div class="NLM_p">Our virtual fragment docking protocol identified two novel PTR1 inhibitor scaffolds (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). There is an ongoing debate in the field whether ligand-based or protein-based methods are more effective in virtual screening.<a onclick="showRef(event, 'ref24 ref25 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26 ref27">(24-27)</a> The scaffolds discovered in this study have a low Tanimoto similarity (≤0.5) to previously known PTR1 and DHFR inhibitor scaffolds, suggesting that a purely ligand-based screening approach based on 2D similarity would not have been able to identify these compounds.</div><div class="NLM_p">Fragment docking has been reported for a variety of targets.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> However, it was noted that the scoring functions that were originally developed for drug-like ligands do not perform equally well for fragment-like ligands.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> As previous studies demonstrated that pharmacophore constraints are useful tools to overcome these limitations,<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28, 29)</a> we adopted this strategy and filtered the docking poses using interaction fingerprints.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> As hypothesized, a better ranking for the putative ligands was achieved (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). However, at least one representative per chemical series ranked already among the top 100 best scoring compounds before reranking. Arguably, in this study, pharmacophore constraints facilitated visual inspection but even without filtering, the same scaffolds would have been discovered.</div><div class="NLM_p">Knowledge of the binding mode of a fragment is crucial to accelerate optimization of binding affinity.<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17, 18)</a> One of the binding modes of fragment <b>4</b> was correctly predicted by our docking protocol (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>). However, crystal structure analysis revealed that different tautomeric forms of this fragment adopt two distinct binding modes. The second tautomer was present in our database and was predicted to bind in the conformation found in the crystal structure, but because only the highest scoring version of each ligand was saved in the final hit list, this binding mode was not considered when inspecting the virtual screening hits. Removal of the chlorine atom of <b>4</b> resulted in ligand <b>5</b>, which adopts a further distinct binding mode (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>). In this orientation, two water molecules mediate contacts between the ligand and the protein. Finally, when <b>4</b> was substituted on N1 with a dichlorobenzyl moiety (<b>9</b>), the protein pocket expanded slightly and the ligand no longer formed hydrogen bonds with the cofactor but bound at a distal site interacting with Asp161 (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>). None of these binding modes was predicted correctly. This points to well recognized issues with molecular docking, which are not specific to fragment docking, such as treatment of tautomeric states, structural water molecules, and protein flexibility.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Progress has been made especially in addressing the latter two areas. However, more work remains to be done before these methods can be routinely employed in molecular docking.<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32, 33)</a></div><div class="NLM_p">The different binding modes for the aminobenzimidazole derivatives <b>4</b>,<b> 5</b>, and <b>9</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>) are also interesting regarding ongoing discussions on if, or how, the binding modes of the ligands change when going from fragment to lead and drug.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Babaoglu and Shoichet deconstructed a β-lactamase inhibitor into four fragments and found different binding modes for each of them.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> However, only one of these fragments contained the same key recognition element that was also present in the parent molecule. This might potentially explain the multiple observed binding modes in their case.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Here, all three ligands contain the same key recognition element, the benzimidazole motive, but due to the substitution patterns of <b>5</b> and <b>9</b>, their binding modes are mutually exclusive. A substituent in N1-position in the binding orientation of <b>5</b> would lead to a steric clash with the phosphate group of the cofactor (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>), while substituting the 8-position in the binding mode of <b>9</b> would lead to a steric clash with Phe174 (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>). The smaller fragment <b>4</b> could theoretically adopt either of these binding modes. Instead, the ligand binds in a third orientation with water-mediated contacts to the phosphate group of the cofactor and an edge−face interaction with the aromatic side chain of Trp221. The later interaction has been observed to be important to improve binding affinity of unrelated PTR1 inhibitors (unpublished results) and might be the driving force for this orientation.</div><div class="NLM_p">Chemical validation of <i>Tb</i>PTR1 as a drug target for HAT is still outstanding. To this end, potent and selective PTR1 inhibitors are needed. Here, we focused on optimizing one of the two scaffolds that were discovered by virtual fragment screening. Out of the three possible binding modes that the aminobenzimidazole derivatives can adopt in the binding pocket, we chose the binding mode of <b>9</b> as a start point for hit expansion. Because of pronounced differences between DHFR and PTR1 in the area of the active site where this inhibitor binds, this strategy should lead to highly selective PTR1 inhibitors.</div><div class="NLM_p last">To optimize the submicromolar activity of <b>9</b> (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>), we sought to fill the hydrophobic pocket close to Trp221 identified by the analysis of the crystal structure <i>Tb</i>PTR1·<b>9</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a>). Indeed, the subsequently synthesized phenyl-derivative <b>12</b> was more than 100-fold more active than <b>9</b> and highly selective over both human and <i>T. brucei</i> DHFR. Its measured logD is with 3.7 units in a range that suggests that this compound should be membrane permeable. In addition, this ligand has a PSA of only 55 Å<sup>2</sup>, which is well in the required range for compounds to pass the blood−brain barrier, a property required to treat the second stage of African sleeping sickness.<a onclick="showRef(event, 'ref2 ref3 ref14'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref14">(2, 3, 14)</a> Consequently, the ligand should be a valuable tool to support chemical validation of <i>Tb</i>PTR1 as drug target. Somewhat surprisingly, this ligand poorly inhibited <i>T. brucei</i> cell growth in culture, having an EC<sub>50</sub> value of only 10 μM. Similar results have been obtained with PTR1 inhibitors containing different chemical scaffolds (unpublished data). At the moment, it is unclear what the reasons are for lack of translation of potent inhibition of PTR1 to an antiproliferative activity against the parasites in culture. Studies are underway to investigate this further.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25516" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25516" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Preparation of Small Molecule Databases</h3><div class="NLM_p last">An in-house database containing over 2.2 million commercially available compounds was constructed and associated physicochemical descriptors along with flags for unwanted groups were calculated as described previously.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> For molecular docking, in house python scripts based on OpenEye’s OEToolkit (Openeye, Santa Fe, NM) were used to generate common tautomeric states and to charge small molecules, whereas for acidic and basic groups with estimated p<i>K</i><sub>a</sub> values between 5 and 9 both, the neutral and charged forms were stored. Three dimensional conformations were calculated using Corina (Molecular Networks, Germany), and low energy conformers were sampled using Omega (Openeye). Partial atomic charges and desolvation energies for the transfer from high-dielectric media (simulated as water) to low dielectric media (simulated as cyclohexane) were computed using AMSOL (<a href="http://comp.chem.umn.edu/amsol/" class="extLink">http://comp.chem.umn.edu/amsol/</a>) with the same settings as used previously.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> The generated molecules were stored in a hierarchical whereas each ring fragment of each database molecule was sequentially used as anchor to align the conformers resulting in several anchors for molecules containing multiple ring fragments.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Receptor Preparation</h3><div class="NLM_p last">The crystal structure of <i>Tb</i>PTR1 in complex with triamterene and NADP<sup>+</sup> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BME">3BME</a>) was chosen as the receptor for this study. Polar hydrogen atoms were added and their positions were minimized using the MAB force field<a onclick="showRef(event, 'ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref37 ref38">(37, 38)</a> as implemented in Moloc (Gerber Molecular Design, Switzerland) with the ligand present in order to obtain a hydrogen-bonding network optimized for ligand binding. Subsequently, all crystallographic water molecules and the inhibitor were removed. Amber charges were assigned to the protein atoms and AMSOL charges to the cofactor. Spheres were manually placed in the binding site in the area that was occupied by the pyrido[3,4-<i>b</i>]pyrazine-2,5,7-triamine part of the ligand and used as receptor matching positions for docking and to lower the effective dielectric in the bindings. Grids to store information about excluded volumes, electrostatic and van der Waals potential, and solvent occlusion were calculated as described earlier.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Molecular Docking</h3><div class="NLM_p last">DOCK 3.5.54 was used to dock small molecules flexibly in to the active site of <i>Tb</i>PTR1.<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20, 21)</a> The following settings were chosen to sample ligand orientations: ligand and receptor bins were set to 0.5 Å, overlap bins were set to 0.4 Å, the distance tolerance for matching ligand atoms to receptor matching sites ranged from 1.1 to 1.2 Å. Each docking pose which did not place any atoms in areas occupied by the receptor was scored for electrostatic and van der Waals complementarity<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> and penalized according to its estimated partial desolvation energy (B. Shoichet, unpublished). For each compound, only the best-scoring database representation (tautomer, protonation state, multiple ring alignment) was stored in the final docking hit list.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Pharmacophore-Based Filtering of Docking Poses and Visual Inspection</h3><div class="NLM_p last">After analysis of available crystal structures, key interactions in the pteridine binding site were identified as π-stacking interactions to Phe97 and the nicotinamide part of the cofactor NADP<sup>+</sup>, an essential hydrogen bond to one of the oxygen atoms of the phosphate group of the cofactor, and optional hydrogen bonds to either the hydroxyl group of Ser95, Tyr174, or the ribose part of the cofactor (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>). To filter and cluster the docking poses, the hydrogen bond interactions identified were encoded into interaction fingerprints.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> A hydrogen bond interaction was assumed when a distance of <3.5 Å was measured between the protein atoms potentially involved in hydrogen bonds described above and a heteroatom of the ligand. Compounds passing this filter step were minimized in the binding site using the MAB force field<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32, 33)</a> as implemented in Moloc. Subsequently, the obtained binding poses were visually inspected and representative compounds still maintaining the required hydrogen-bond network were short-listed for purchase.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Structure-Based Ligand Design</h3><div class="NLM_p last">Starting with the crystal structure of <i>Tb</i>PTR1·<b>9</b>, potential derivatives of the hit compound that bear moieties suitable to fill the hydrophobic pocket next to Trp221 (Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a>) were constructed using Moloc. Subsequently their binding modes were minimized using the MAB force field and promising compounds were selected for synthesis</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Calculation of Tanimoto Coefficients and Polar Surface Area</h3><div class="NLM_p last">Tanimoto coefficients were calculated using Sybyl (Tripos, St. Louis, MO) based on standard Tripos fingerprints. Topological polar surface areas were calculated using a Python script provided by Openeye based on a previously published method.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a></div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> PTR1 and DHFR Activity Assays</h3><div class="NLM_p">Compounds <b>2</b>−<b>4</b> and <b>8</b>−<b>9</b> were purchased from Asinex, compound <b>5</b> from Sigma-Aldrich, and compound <b>7</b> from Chemdiv. Purity (>90%) and identity of these compounds was confirmed by LCMS. Compound <b>6</b> was available in-house from a different chemistry program and its synthesis will be reported elsewhere. The synthesis for compounds <b>10</b>−<b>12</b> is reported below.</div><div class="NLM_p last"><i>Tb</i>PTR1 and DHFR activity was measured in 96-well microtiter plates via reduction of cytochrome <i>c</i> (cyt<i>c</i>) as a result of the enzymatic production of tetrahydrobiopterin (H<sub>4</sub>B) and terahydrofolate (H<sub>4</sub>F), respectively. In brief, <i>Tb</i>PTR1 activity was assayed in a buffer containing 20 mM sodium citrate, 1 mM EDTA, H<sub>2</sub>B (0.35 μM), cyt<i>c</i> (81 μM), and NADPH (100 μM) at pH 6.0; DHFR was assayed in 20 mM sodium citrate, 1 mM EDTA, H<sub>2</sub>F (4.4 μM), cyt<i>c</i> (81 μM), and NADPH (100 μM) at pH 7.4. Enzyme activity was monitored by reading absorbance at 550 nm within the linear phase of reaction. Full details of this method will be reported in a separate publication. <i>K</i><sub>i</sub><sup>app</sup> values were calculated using a modified Morrison equation.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Protein Crystallization, Data Collection, and Structure Solution</h3><div class="NLM_p"><i>Tb</i>PTR1 was purified and crystallized as described previously.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> In short, <i>Tb</i>PTR1 was concentrated to 6 mg/mL in 20 mM Tris-HCl pH 8.0. Ligands were solubilized to a concentration of 200 mM in DMSO. The ternary complex of <i>Tb</i>PTR1 with cofactor and ligand was prepared by incubating the protein solution (6 mg/mL) with 2 mM of the ligand, 1 mM NADP<sup>+</sup>, and 20 mM dithiothreitol in 20 mM Tris-HCl pH 8.0 on ice for 30 min prior to crystallization. Crystallization was carried out by the vapor diffusion method by mixing 2 μL of the protein solution with 2 μL of the reservoir solution and incubating the drops over 100 μL of the reservoir in sitting drop plates. The reservoir solution consisted of 1.5−3.0 M sodium acetate and 0.1 M citrate buffer pH 4.5−6.0. Diffraction quality crystals were obtained after 2−3 days at 18 °C.</div><div class="NLM_p">Diffraction data for the complexes with <b>4</b>,<b> 5</b>, and <b>12</b> were measured using a rotating anode X-ray source (Rigaku Micromax 007) and an image plate detector (Rigaku R-AXIS IV<sup>++</sup>) and for the remaining complex at beamline ID23-1 at the European Synchrotron Radiation Facility (ESRF) in Grenoble, France. Crystals were prepared for data collection by transferring them through a cryoprotection solution of mother liquor with 20% glycerol added and then flash frozen in a stream of gaseous nitrogen at −173 °C. Data were integrated and scaled using MOSFLM<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> and SCALA<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> from the CCP4<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> suite of programs, except for complex PTR1·<b>12</b>, for which the HKL suite was used.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></div><div class="NLM_p">Molecular replacement as implemented in MOLREP<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> was used to solve the structure using the protein chains from the <i>Tb</i>PTR1·methotrexate complex<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2C7V">2C7V</a>) as the starting model. After the molecular replacement step, a round of rigid-body refinement was carried out using REFMAC5.<a onclick="showRef(event, 'ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref46 ref47">(46, 47)</a> Ligand models and associated topology files were created with PRODRG<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> and were built into <i>F</i><sub>o</sub> − <i>F</i><sub>c</sub> electron density maps using COOT.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> Further rounds of restrained refinement were carried out using REFMAC5 and manual alteration of the models, including addition of solvent molecules using COOT.</div><div class="NLM_p last">Figures of protein−ligand complexes were produced using PyMOL (DeLano Scientific, Palo Alto, CA).</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Synthesis of Amino-benzimidazole Analogues</h3><div class="NLM_p"><sup>1</sup>H NMR spectra were recorded on either a Bruker Avance DPX 500 or on a Bruker Avance 300 spectrometer. Chemical shifts (δ) are expressed in ppm. Signal splitting patterns are described as singlet (s), broad singlet (bs), doublet (d), triplet (t), quartet (q), multiplet (m), or combinations thereof.</div><div class="NLM_p">Low resolution electrospray (ES) mass spectra were recorded on a Bruker MicroTof mass spectrometer, run in positive ion mode, using either methanol, methanol/water (95:5), or water/acetonitrile (1:1) and 0.1% formic acid as the mobile phase. High resolution electrospray measurements were performed on a Bruker MicroTof mass spectrometer.</div><div class="NLM_p">LC-MS analysis and chromatographic separations were conducted with a Bruker MicroTof mass spectrometer using an Agilent HPLC 1100 with a diode array detector in series. The column used was a Phenomenex Gemini C18 column, 50 mm × 3.0 mm, 5 mm particle size. The following method was used: mobile phase, water/acetonitrile +0.1% HCOOH 80:20 to 5:95 gradient over 3.5 min, and then held at for 1.5 min; flow rate 0.5 mL/min.</div><div class="NLM_p">All compounds exemplified had a measured purity of greater than 95% on this analytical HPLC-MS system (TIC and HPLC UV). HPLC retention times and M<sup>+</sup> data are given below to substantiate the purity and integrity of the compounds. <sup>1</sup>H NMR also confirmed compound identity and purity (wrt other organic components being absent).</div><div class="NLM_p">Thin layer chromatography (TLC) was carried out on Merck silica gel 60 F254 plates using UV light and/or KMnO<sub>4</sub> for visualization. TLC data are given as the <i>R</i><sub>f</sub> value with the corresponding eluent system specified in brackets. Column chromatography was performed using RediSep 4 or 12 g silica prepacked columns.</div><div class="NLM_p last">All reactions were carried out under dry and inert conditions unless otherwise stated.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> 7-Chloro-1-(3,4-dichlorobenzyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-amine (<b>10</b>)</h3><div class="NLM_p">7-Chloro-1<i>H</i>-benzo[<i>d</i>]imidazol-2-amine (<b>16</b>) (200 mg, 1.19 mmol), potassium hydroxide flakes (100 mg, 1.79 mmol), and 3,4-dichlorobenzyl chloride (244 mg, 1.25 mmol) were stirred in 15 mL ethanol at 20 °C for 48 h. The solution was then concentrated by removing the ethanol and diluted with 40 mL ethyl acetate and washed with sodium bicarbonate solution (1 × 20 mL), water (1 × 30 mL), and brine (2 × 20 mL). The organic layer was dried with magnesium sulfate and concentrated to remove the ethyl acetate to afford a cream-colored solid. Compound <b>10</b> was the more polar product obtained purified by flash column chromatography (10% methanol/ dichloromethane eluent), yielding compound <b>10</b> as a white solid (68 mg, 17% yield). <sup>1</sup>H NMR and LCMS confirmed purity at >98%.</div><div class="NLM_p">The relative <sup>1</sup>H NMR signal for the CH<sub>2</sub>-dichlorobenzyl protons at 5.54 ppm, versus 5.28 ppm for 4-isomer, agrees with the expected shift for a 7-substituted <i>N</i><sup>1</sup>-benzylated benzimidazole versus a 4-substituted benzimidazole.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a></div><div class="NLM_p"><sup>1</sup>H NMR (500 MHz, DMSO): δ = 5.54 (2H, s, CH2), 6.78 (2H, s, NH2), 6.83 (1H, m, ArH), 6.95 (1H, dd (<i>J</i> = 2.1 Hz, <i>J</i> = 8.4 Hz) ArH), 6.97 (1H, t (<i>J</i> = 7.9 Hz) ArH), 7.15 (1H, dd (<i>J</i> = 0.9 Hz, <i>J</i> = 7.8 Hz), ArH), 7.31 (1H, s, ArH), 7.61 (1H, d (<i>J</i> = 8.3 Hz), ArH).</div><div class="NLM_p">LCMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> (%) 327 (M + H)<sup>+</sup>, retention time 0.9 min.</div><div class="NLM_p last">HRMS (ES<sup>+</sup>): calcd for (Cl35) C14H11Cl3N3 [M + H]<sup>+</sup> 326.0013, found 325.9998 (4.47 ppm).</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> 7-(<i>n</i>-Propyloxy)-1-(3,4-dichlorobenzyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-amine (<b>11</b>)</h3><div class="NLM_p">7-(<i>n</i>-Propyloxy)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-amine (<b>13</b>) (250 mg, 1.31 mmol), potassium hydroxide flakes (110 mg, 6.28 mmol), and 3,4-dichlorobenzyl chloride (269 mg, 1.38 mmol) were stirred in 10 mL ethanol at 20 °C for 48 h. The reaction was worked up as described for compound <b>10</b>, and columned. Compound <b>11</b> was the more polar product obtained purified by flash column chromatography (10% methanol/ dichloromethane eluent), yielding compound <b>11</b> as a white solid (90 mg, 20% yield). <sup>1</sup>H NMR and LCMS confirmed purity at >98%.</div><div class="NLM_p">Absolute regiochemistry of compound <b>11</b> was confirmed by the relative <sup>1</sup>H NMR signal for the CH<sub>2</sub>-dichlorobenzyl protons at 5.48 ppm, versus 5.08 ppm for 4-isomer. This agrees with the expected shift for a 7-alkoxy substituted <i>N</i><sup>1</sup>-benzylated benzimidazole versus a 4-substituted benzimidazole.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a></div><div class="NLM_p"><sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ = 0.94 (3H, t (<i>J</i> = 7.4 Hz), CH<sub>3</sub>), 1.71 (2H, m, CH<sub>2</sub>), 4.03 (2H, t (<i>J</i> = 6.4 Hz), CH<sub>2</sub>), 4.30 (2H, bs, NH<sub>2</sub>), 5.48 (2H, s, CH<sub>2</sub>), 6.62 (1H, dd (<i>J</i> = 1.1 Hz, <i>J</i> = 7.7 Hz), ArH), 7.08 (2H, m, 2ArH), 7.13 (1H, m, ArH), 7.35 (1H, s, ArH), 7.42 (1H, d (<i>J</i> = 8.3 Hz), ArH).</div><div class="NLM_p">LCMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> (%) 351 (M + H)<sup>+</sup>, retention time 0.9 min.</div><div class="NLM_p last">HRMS (ES<sup>+</sup>): calcd for C<sub>17</sub>H<sub>18</sub>Cl<sub>2</sub>N<sub>3</sub>O [M + H]<sup>+</sup> 350.0821, found 350.0815 (1.75 ppm).</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> 7-Phenyl-1-(3,4-dichlorobenzyl)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-amine (<b>12</b>)</h3><div class="NLM_p">7-Phenyl-1<i>H</i>-benzo[<i>d</i>]imidazol-2-amine (<b>18</b>) (115 mg, 0.55 mmol), potassium hydroxide flakes (46 mg, 0.83 mmol), and 3,4-dichlorobenzyl chloride (113 mg, 0.58 mmol) were stirred in 8 mL of ethanol at 20 °C for 48 h. The reaction was worked up described for compound <b>10</b> and columned. Compound <b>12</b> was the more polar product obtained purified by flash column chromatography (10% methanol/dichloromethane eluent), collected as a white solid (35 mg, 18% yield). <sup>1</sup>H NMR and LCMS confirmed purity at >98%.</div><div class="NLM_p">Absolute regiochemistry of compound <b>12</b> was confirmed by NOESY NMR: a strong interaction between CH<sub>2</sub>-dichlorobenzyl protons at 4.88 ppm (2H, s, CH<sub>2</sub>) and the ortho proton signals of the pendant phenyl ring at 7.12 ppm (2H, d, CH) was observed, indicating close proximity of these substituents. Also a crystal structure of compound <b>12</b> complex with PTR1 enzyme was obtained (Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a>), confirming the absolute regiochemistry of compound <b>12</b> as 7-phenyl-1-(3,4-dichloro)-1<i>H</i>-benzo(<i>d</i>)imidazol-2-amine.</div><div class="NLM_p"><sup>1</sup>H NMR (500 MHz, DMSO): δ = 4.88 (2H, s, CH2), 6.37 (1H, dd (<i>J </i>= 2.1 Hz, <i>J</i> = 8.3 Hz), ArH), 6.50 (1H, m, ArH), 6.60 (2H, s, NH<sub>2</sub>), 6.61 (1H, dd (<i>J</i> = 1.1 Hz, <i>J</i> = 7.5 Hz), ArH), 7.00 (1H, t (<i>J</i> = 7.7 Hz), ArH), 7.12 (2H, bd (<i>J</i> = 8.3 Hz), ArH), 7.21 (1H, dd (<i>J</i> = 1.1 Hz, <i>J</i> = 7.8 Hz), ArH), 7.31 (2H, t (<i>J</i> = 7.3 Hz), 2ArH), 7.38 (2H, m, 2ArH).</div><div class="NLM_p">LCMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> (%) 369 (M + H)<sup>+</sup>, retention time 0.9 min.</div><div class="NLM_p last">HRMS (ES<sup>+</sup>): calcd for C<sub>20</sub>H<sub>16</sub>Cl<sub>2</sub>N<sub>3</sub> [M + H]<sup>+</sup> 368.0716, found 368.0708 (2.05 ppm).</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> 7-(<i>n</i>-Propyloxy)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-amine (<b>13</b>)</h3><div class="NLM_p">3-<i>n</i>-Propyloxybenzene-1,2-diamine (<b>14)</b> (410 mg, 2.47 mmol) and cyanogenbromide 2 M solution in acetonitrile (1.36 mL, 2.72 mmol) were stirred in acetonitrile (12 mL) and water (3 mL) at 20 °C for 36 h. The mixture was then concentrated to remove the acetonitrile. Ethyl acetate (40 mL) and sodium bicarbonate solution (1 × 40 mL) were added, and the biphasic solution was transferred to a separating funnel. The organic layer was washed with water (1 × 25 mL), sodium bicarbonate solution (1 × 20 mL), and brine (2 × 30 mL), before being dried with MgSO<sub>4</sub> and concentrated to remove ethyl acetate. Crude compound <b>13</b> was purified by flash column chromatography (8% methanol/dichloromethane eluent), yielding compound <b>13</b> as a white solid (300 mg, 64% yield). <sup>1</sup>H NMR and LCMS confirmed purity at >98%.</div><div class="NLM_p"><sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ = 1.01 (3H, t (<i>J</i> = 7.4 Hz), CH<sub>3</sub>), 1.72 (2H, m, CH<sub>2</sub>), 4.03 (2H, m, CH<sub>2</sub>), 5.83 (2H, bs, NH<sub>2</sub>), 6.46 (1H, m, ArH), 6.86 (2H, m, ArCH), 10.70 (1H, bs, NH).</div><div class="NLM_p last">LCMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> (%) 192 (M + H)<sup>+</sup>, retention time 0.6 min.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> 3-Propoxybenzene-1,2-diamine (<b>14</b>)</h3><div class="NLM_p">2-Nitro-6-propoxyaniline (<b>15)</b> (300 mg, 1.53 mmol) and tin(II) chloride (1.45 g, 7.65 mmol) were taken into in ethanol (15 mL). The reaction was heated in a microwave reactor at 140 °C for 10 min and then poured into NaHCO<sub>3</sub> solution (30 mL) and the product was extracted into ethyl acetate (2 × 25 mL). The organic layers were combined and washed with sodium bicarbonate solution (1 × 30 mL), water (1 × 30 mL), and brine (2 × 40 mL) before being dried with MgSO<sub>4</sub> and concentrated to remove ethyl acetate. The solution was concentrated to remove ethyl acetate, affording compound <b>14</b> as a pale-yellow gum in quantitative yield. <sup>1</sup>H NMR and LCMS confirmed purity at >95%.</div><div class="NLM_p"><sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ = 1.07 (3H, t (<i>J</i> = 7.4 Hz), CH<sub>3</sub>), 1.82 (2H, m, CH<sub>2</sub>), 3.44 (2H, m, NH<sub>2</sub>), 3.98 (2H, t (<i>J</i> = 6.5 Hz), CH<sub>2</sub>), 6.40 (2H, m, ArH), 6.68 (1H, t (<i>J</i> = 8.0 Hz), ArH).</div><div class="NLM_p last">LCMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> (%) 167 (M + H)<sup>+</sup>, retention time 0.5 min.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> 2-Nitro-6-propoxyaniline (<b>15</b>)</h3><div class="NLM_p">2-Amino-3-nitrophenol (500 mg, 3.23 mmol), K<sub>2</sub>CO<sub>3</sub> (669 mg, 4.84 mmol), and 3-bromopropane (438 mg, 3.35 mmol) were taken into DMF (35 mL). The reaction was stirred at 20 °C for 12 h and then poured into NaHCO<sub>3</sub> solution (20 mL), and the product was extracted into ethyl acetate (2 × 30 mL). The organic layers were combined then washed with water (1 × 30 mL), sodium bicarbonate solution (1 × 30 mL), and brine (2 × 40 mL) before being dried with MgSO<sub>4</sub> and concentrated to remove ethyl acetate. Crude compound <b>15</b> was purified by flash column chromatography (3% methanol/dichloromethane eluent), affording compound <b>15</b> as a pale-yellow solid (460 mg, 73% yield). <sup>1</sup>H NMR and LCMS confirmed purity at >98%.</div><div class="NLM_p"><sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ = 1.09 (3H, s, CH<sub>3</sub>), 1.91 (2H, s, CH<sub>2</sub>), 4.02 (2H, m, CH<sub>2</sub>), 6.47 (2H, m, NH<sub>2</sub>), 6.64 (1H, dd (<i>J</i> = 7.8 Hz, <i>J</i> = 8.9 Hz), ArH), 6.91 (1H, d (<i>J</i> = 7.6 Hz), ArH), 7.75 (1H, dd (<i>J</i> = 1.1 Hz, <i>J</i> = 8.9 Hz), ArH).</div><div class="NLM_p last">LCMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> (%) 197 (M + H)<sup>+</sup>, retention time 0.5 min.</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> 7-Chloro-1<i>H</i>-benzo[<i>d</i>]imidazol-2-amine (<b>16</b>)</h3><div class="NLM_p">Prepared like compound <b>13</b>, starting from 3-chlorobenzene-1,2-diamine (<b>17</b>) (624 mg, 4.38 mmol) and cyanogenbromide 2 M solution in acetonitrile (2.41 mL, 4.82 mmol), in acetonitrile (18 mL), and water (4 mL). The reaction was stirred at 20 °C for 36 h. Crude compound <b>16</b> was purified by flash column chromatography (8% methanol/dichloromethane eluent), yielding compound <b>16</b> as a white solid (510 mg, 69% yield). <sup>1</sup>H NMR and LCMS confirmed purity at >98%.</div><div class="NLM_p"><sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ = 6.40 (2H, bs, NH<sub>2</sub>), 6.79 (1H, bm, ArH), 6.90 (1H, m, ArH), 7.0 (1H, dd (<i>J</i> = 0.8 Hz, <i>J</i> = 7.8 Hz), ArH), 10.94 (1H, bs, NH).</div><div class="NLM_p last">LCMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> (%) 168.5 (M + H)<sup>+</sup>, retention time 0.6 min.</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> 3-Chlorobenzene-1,2-diamine (<b>17</b>)</h3><div class="NLM_p">Prepared like compound <b>14</b>, using 2-chloro-6-nitroaniline (1.0 g, 5.8 mmol), tin(II) chloride (5.5 g, 29 mmol) in ethanol 30 mL). Compound <b>17</b> was obtained in quantitative yield as a colorless gum. LCMS confirmed purity at >95%. Used as “crude” in example <b>16</b>.</div><div class="NLM_p last">LCMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> (%) 143 (M + H)<sup>+</sup>, retention time 0.4 min.</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> 7-Phenyl-1<i>H</i>-benzo[<i>d</i>]imidazol-2-amine (<b>18</b>)</h3><div class="NLM_p">Prepared like compound <b>13</b>, starting from biphenyl-2,3-diamine (<b>19</b>) (350 mg, 1.9 mmol) and cyanogenbromide 2 M solution in acetonitrile (1.05 mL, 2.09 mmol), in acetonitrile (18 mL), and water (4 mL). The reaction was stirred at 20 °C for 18 h and worked up as described above. Crude compound <b>18</b> was purified by flash column chromatography (8% methanol/dichloromethane eluent), yielding compound <b>18</b> as a white solid (250 mg, 63% yield). <sup>1</sup>H NMR and LCMS confirmed purity at >98%.</div><div class="NLM_p"><sup>1</sup>H NMR (500 MHz, DMSO): δ = 6.22 (2H, s, CH<sub>2</sub>), 6.95 (1H, t (<i>J</i> = 7.7 Hz), ArH), 7.08 (1H, m, ArH), 7.31 (1H, t, (<i>J</i> = 7.5 Hz), ArH), 7.44 (1H, t (<i>J</i> = 7.6 Hz), ArH), 7.90 (2H, bs, NH<sub>2</sub>), 10.90 (1H, bs, NH).</div><div class="NLM_p last">LCMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> (%) 210 (M + H)<sup>+</sup>, retention time 0.5 min.</div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Biphenyl-2,3-diamine (<b>19</b>)</h3><div class="NLM_p">Prepared like compound <b>14</b>, using 3-phenyl-2-nitroaniline (<b>20</b>) (500 mg, 2.34 mmol), tin(II) chloride (2.2 g, 11.7 mmol) in ethanol 25 mL). The reaction was heated in a microwave reactor at 140 °C for 10 min and then worked up as previously described. The solution was concentrated to remove ethyl acetate, affording compound <b>19</b> in quantitative yield, as a colorless gum. LCMS confirmed purity at >95%. Used as “crude” in example <b>18</b>.</div><div class="NLM_p last">LCMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> (%) 184 (M + H)<sup>+</sup>, retention time 0.5 min.</div></div><div id="sec4_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> 3-Phenyl-2-nitroaniline (<b>20</b>)</h3><div class="NLM_p">A solution of 2-phenylnitrobenzene (398 mg, 2 mmol) and <i>o</i>-methylhydroxylamine (118 mg, 2.5 mmol) in DMF (3 mL) was added dropwise (over 5 min) to a stirred suspension of <i><sup>t</sup></i>BuOK (672 mg, 6 mmol) and Cu(I)Cl (20 mg, 0.2 mmol) in DMF (7 mL). The reaction was stirred at 20 °C for 60 min and then quenched with ammonium chloride solution. The product was extracted into DCM, dried with MgSO<sub>4</sub>, and concentrated to remove the dichloromethane. Compound <b>20</b> was separated from the side product (2-phenyl-4-nitroaniline) and remaining starting material by flash column chromatography (10% ethyl acetate/hexane eluent), affording compound <b>20</b> as a yellow solid (86 mg, 20% yield). <sup>1</sup>H NMR and LCMS confirmed purity at >98%.</div><div class="NLM_p"><sup>1</sup>H NMR (500 MHz, CDCl3): δ = 6.5 (1H, dd (<i>J</i> = 1.3 Hz, <i>J</i> = 7.7 Hz), ArH), 6.57 (2H, s, NH2), 6.8 (1H, m, ArH), 6.89 (1H, d (<i>J</i> = 7.7 Hz), ArH), 6.95 (1H, dt (<i>J</i> = 1.2 Hz, <i>J</i> = 7.6 Hz), ArH), 7.2 (2H, m, 2ArH), 7.3 (1H, dt (<i>J</i> = 1.6 Hz, <i>J</i> = 7.7 Hz), ArH), 7.53 (1H, dd (<i>J</i> = 1.1 Hz, <i>J</i> = 7.9 Hz), ArH).</div><div class="NLM_p last">LCMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> (%) 215 (M + H)<sup>+</sup>, retention time 0.6 min.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i39"><a href="/doi/suppl/10.1021/jm900414x">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84170" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84170" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">A table listing initial screening hits with corresponding percent inhibition data is available as <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>.This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm900414x/suppl_file/jm900414x_si_001.pdf">jm900414x_si_001.pdf (1.11 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm900414x" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55002" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55002" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ruth Brenk</span> - <span class="hlFld-Affiliation affiliation">Division of Biological Chemistry and Drug Discovery, College of Life Sciences, University of Dundee, Dow Street, Dundee DD1 5EH, U.K.</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#a4d68ac6d6c1cacfe4c0d1cac0c1c18ac5c78ad1cf"><span class="__cf_email__" data-cfemail="f587db9787909b9eb591809b919090db9496db809e">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chidochangu P. Mpamhanga</span> - <span class="hlFld-Affiliation affiliation">Division of Biological Chemistry and Drug Discovery, College of Life Sciences, University of Dundee, Dow Street, Dundee DD1 5EH, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniel Spinks</span> - <span class="hlFld-Affiliation affiliation">Division of Biological Chemistry and Drug Discovery, College of Life Sciences, University of Dundee, Dow Street, Dundee DD1 5EH, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lindsay B. Tulloch</span> - <span class="hlFld-Affiliation affiliation">Division of Biological Chemistry and Drug Discovery, College of Life Sciences, University of Dundee, Dow Street, Dundee DD1 5EH, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Emma J. Shanks</span> - <span class="hlFld-Affiliation affiliation">Division of Biological Chemistry and Drug Discovery, College of Life Sciences, University of Dundee, Dow Street, Dundee DD1 5EH, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David A. Robinson</span> - <span class="hlFld-Affiliation affiliation">Division of Biological Chemistry and Drug Discovery, College of Life Sciences, University of Dundee, Dow Street, Dundee DD1 5EH, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Iain T. Collie</span> - <span class="hlFld-Affiliation affiliation">Division of Biological Chemistry and Drug Discovery, College of Life Sciences, University of Dundee, Dow Street, Dundee DD1 5EH, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alan H. Fairlamb</span> - <span class="hlFld-Affiliation affiliation">Division of Biological Chemistry and Drug Discovery, College of Life Sciences, University of Dundee, Dow Street, Dundee DD1 5EH, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paul G. Wyatt</span> - <span class="hlFld-Affiliation affiliation">Division of Biological Chemistry and Drug Discovery, College of Life Sciences, University of Dundee, Dow Street, Dundee DD1 5EH, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Julie A. Frearson</span> - <span class="hlFld-Affiliation affiliation">Division of Biological Chemistry and Drug Discovery, College of Life Sciences, University of Dundee, Dow Street, Dundee DD1 5EH, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">William N. Hunter</span> - <span class="hlFld-Affiliation affiliation">Division of Biological Chemistry and Drug Discovery, College of Life Sciences, University of Dundee, Dow Street, Dundee DD1 5EH, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ian H. Gilbert</span> - <span class="hlFld-Affiliation affiliation">Division of Biological Chemistry and Drug Discovery, College of Life Sciences, University of Dundee, Dow Street, Dundee DD1 5EH, U.K.</span></div></li></ul></li><li></li><li></li><li></li></ul></div><div class="ack"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i38">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10057" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10057" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was funded by the Welcome Trust (WT 077705, 082596, and 083481) and BBSRC (BBS/B/14434). We thank Scottish Enterprise and the Scottish Structural Proteomics Facility (SSPF) for additional financial support and Openeye for free software licenses. Further, we thank Nicholas Moir for administrating the Linux workstations and Dr. Mark Agacan for maintenance of the in-house X-ray crystallography facilities.</p></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i40">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40087" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40087" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 50 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Pink, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hudson, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mouries, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bendig, M.</span><span> </span><span class="NLM_article-title">Opportunities and challenges in antiparasitic drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug. Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">727</span><span class="NLM_x">–</span> <span class="NLM_lpage">740</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2005&pages=727-740&author=R.+Pinkauthor=A.+Hudsonauthor=M.+A.+Mouriesauthor=M.+Bendig&title=Opportunities+and+challenges+in+antiparasitic+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPink%26aufirst%3DR.%26aulast%3DHudson%26aufirst%3DA.%26aulast%3DMouries%26aufirst%3DM.%2BA.%26aulast%3DBendig%26aufirst%3DM.%26atitle%3DOpportunities%2520and%2520challenges%2520in%2520antiparasitic%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug.%2520Discovery%26date%3D2005%26volume%3D4%26spage%3D727%26epage%3D740" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Brun, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balmer, O.</span><span> </span><span class="NLM_article-title">New developments in human African trypanosomiasis</span> <span class="citation_source-journal">Curr. Opin. Infect. Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">415</span><span class="NLM_x">–</span> <span class="NLM_lpage">420</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1097%2F01.qco.0000244045.93016.b1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=16940863" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A280%3ADC%252BD28rhs1CntQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2006&pages=415-420&author=R.+Brunauthor=O.+Balmer&title=New+developments+in+human+African+trypanosomiasis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">New developments in human African trypanosomiasis</span></div><div class="casAuthors">Brun Reto; Balmer Oliver</div><div class="citationInfo"><span class="NLM_cas:title">Current opinion in infectious diseases</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">415-20</span>
        ISSN:<span class="NLM_cas:issn">0951-7375</span>.
    </div><div class="casAbstract">PURPOSE OF REVIEW:  To review recent literature on human African trypanosomiasis, focussing on genome sequencing, diagnosis and drug discovery, and typing of trypanosomes.  RECENT FINDINGS:  The most important recent development has been the completion of the Trypanosoma brucei genome which will greatly facilitate the discovery of new drug targets and genetic markers.  Correct staging of the disease is of key importance for treatment.  The analysis of sleep patterns is a promising new method to this end and has advanced enough to begin thorough clinical trials.  In terms of novel drug candidates, dicationic molecules show the most promise with one oral diamidine in phase 3 clinical trials.  New targets and classes of molecules which show in vitro trypanocidal activity are also described.  Two new methods - MGE-PCR and microsatellites - allow analyses without parasite cultivation, eliminating a major impediment to efficient sampling for population studies.  The finding that several wild animal species harbour T. b. gambiense, and that parasite transmission is efficient even from very low parasitaemias, sheds a new light on the importance of animal reservoirs.  SUMMARY:  The use of T. brucei as model system for molecular and cell biology is regularly producing new technologies exploitable for diagnosis and new drugs.  Drug discovery and development experience a revival through new public-private partnerships and initiatives.  The challenge remains to translate this progress into improvements for affected people in disease endemic areas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTXtcSNPwREGHthwrmtH4OBfW6udTcc2ea1KHIm0UZecrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD28rhs1CntQ%253D%253D&md5=7d9c553ecb478b197fab2dd572d814b2</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1097%2F01.qco.0000244045.93016.b1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252F01.qco.0000244045.93016.b1%26sid%3Dliteratum%253Aachs%26aulast%3DBrun%26aufirst%3DR.%26aulast%3DBalmer%26aufirst%3DO.%26atitle%3DNew%2520developments%2520in%2520human%2520African%2520trypanosomiasis%26jtitle%3DCurr.%2520Opin.%2520Infect.%2520Dis.%26date%3D2006%26volume%3D19%26spage%3D415%26epage%3D420" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Kennedy, P. G.</span><span> </span><span class="NLM_article-title">The continuing problem of human African trypanosomiasis (sleeping sickness)</span> <span class="citation_source-journal">Ann. Neurol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">116</span><span class="NLM_x">–</span> <span class="NLM_lpage">126</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1002%2Fana.21429" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=18756506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A280%3ADC%252BD1crnsFKrsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2008&pages=116-126&author=P.+G.+Kennedy&title=The+continuing+problem+of+human+African+trypanosomiasis+%28sleeping+sickness%29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The continuing problem of human African trypanosomiasis (sleeping sickness)</span></div><div class="casAuthors">Kennedy Peter G E</div><div class="citationInfo"><span class="NLM_cas:title">Annals of neurology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">116-26</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Human African trypanosomiasis, also known as sleeping sickness, is a neglected disease, and it continues to pose a major threat to 60 million people in 36 countries in sub-Saharan Africa.  Transmitted by the bite of the tsetse fly, the disease is caused by protozoan parasites of the genus Trypanosoma and comes in two types: East African human African trypanosomiasis caused by Trypanosoma brucei rhodesiense and the West African form caused by Trypanosoma brucei gambiense.  There is an early or hemolymphatic stage and a late or encephalitic stage, when the parasites cross the blood-brain barrier to invade the central nervous system.  Two critical current issues are disease staging and drug therapy, especially for late-stage disease.  Lumbar puncture to analyze cerebrospinal fluid will remain the only method of disease staging until reliable noninvasive methods are developed, but there is no widespread consensus as to what exactly defines biologically central nervous system disease or what specific cerebrospinal fluid findings should justify drug therapy for late-stage involvement.  All four main drugs used for human African trypanosomiasis are toxic, and melarsoprol, the only drug that is effective for both types of central nervous system disease, is so toxic that it kills 5% of patients who receive it.  Eflornithine, alone or combined with nifurtimox, is being used increasingly as first-line therapy for gambiense disease.  There is a pressing need for an effective, safe oral drug for both stages of the disease, but this will require a significant increase in investment for new drug discovery from Western governments and the pharmaceutical industry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQoak2ZgH4MMJdYoSR9eyvsfW6udTcc2ea1KHIm0UZecrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1crnsFKrsw%253D%253D&md5=7bc60606f9dab108c3b1c6911cb018a8</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1002%2Fana.21429&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fana.21429%26sid%3Dliteratum%253Aachs%26aulast%3DKennedy%26aufirst%3DP.%2BG.%26atitle%3DThe%2520continuing%2520problem%2520of%2520human%2520African%2520trypanosomiasis%2520%2528sleeping%2520sickness%2529%26jtitle%3DAnn.%2520Neurol.%26date%3D2008%26volume%3D64%26spage%3D116%26epage%3D126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Nare, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luba, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hardy, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beverley, S.</span><span> </span><span class="NLM_article-title">New approaches to <i>Leishmania</i> chemotherapy: pteridine reductase 1 (PTR1) as a target and modulator of antifolate sensitivity</span> <span class="citation_source-journal">Parasitology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">114</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl.</span><span class="NLM_x">) </span> <span class="NLM_fpage">S101</span><span class="NLM_x">–</span> <span class="NLM_lpage">S110</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=1997&pages=S101-S110&issue=Suppl.&author=B.+Nareauthor=J.+Lubaauthor=L.+W.+Hardyauthor=S.+Beverley&title=New+approaches+to+Leishmania+chemotherapy%3A+pteridine+reductase+1+%28PTR1%29+as+a+target+and+modulator+of+antifolate+sensitivity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNare%26aufirst%3DB.%26aulast%3DLuba%26aufirst%3DJ.%26aulast%3DHardy%26aufirst%3DL.%2BW.%26aulast%3DBeverley%26aufirst%3DS.%26atitle%3DNew%2520approaches%2520to%2520Leishmania%2520chemotherapy%253A%2520pteridine%2520reductase%25201%2520%2528PTR1%2529%2520as%2520a%2520target%2520and%2520modulator%2520of%2520antifolate%2520sensitivity%26jtitle%3DParasitology%26date%3D1997%26volume%3D114%26issue%3DSuppl%26spage%3DS101%26epage%3DS110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Berriman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghedin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hertz-Fowler, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blandin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renauld, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bartholomeu, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lennard, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caler, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamlin, N. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haas, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bohme, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hannick, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aslett, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shallom, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcello, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hou, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wickstead, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alsmark, U. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atkin, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barron, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bringaud, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brooks, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carrington, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherevach, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chillingworth, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Churcher, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, L. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corton, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cronin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doggett, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Djikeng, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feldblyum, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Field, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fraser, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodhead, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hance, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harper, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hauser, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hostetler, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ivens, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jagels, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kerhornou, A. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koo, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larke, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landfear, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larkin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leech, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Line, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lord, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macleod, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mooney, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moule, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morgan, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mungall, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norbertczak, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ormond, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pai, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peacock, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quail, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabbinowitsch, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rajandream, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reitter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salzberg, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanders, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schobel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmonds, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simpson, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tallon, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turner, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tait, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tivey, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Aken, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wanless, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitehead, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodward, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wortman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Embley, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gull, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ullu, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barry, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairlamb, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Opperdoes, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barrell, B. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donelson, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fraser, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melville, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">El-Sayed, N. M.</span><span> </span><span class="NLM_article-title">The genome of the African trypanosome <i>Trypanosoma brucei</i></span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">309</span><span class="NLM_x">, </span> <span class="NLM_fpage">416</span><span class="NLM_x">–</span> <span class="NLM_lpage">422</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=309&publication_year=2005&pages=416-422&author=M.+Berrimanauthor=E.+Ghedinauthor=C.+Hertz-Fowlerauthor=G.+Blandinauthor=H.+Renauldauthor=D.+C.+Bartholomeuauthor=N.+J.+Lennardauthor=E.+Calerauthor=N.+E.+Hamlinauthor=B.+Haasauthor=U.+Bohmeauthor=L.+Hannickauthor=M.+A.+Aslettauthor=J.+Shallomauthor=L.+Marcelloauthor=L.+Houauthor=B.+Wicksteadauthor=U.+C.+Alsmarkauthor=C.+Arrowsmithauthor=R.+J.+Atkinauthor=A.+J.+Barronauthor=F.+Bringaudauthor=K.+Brooksauthor=M.+Carringtonauthor=I.+Cherevachauthor=T.+J.+Chillingworthauthor=C.+Churcherauthor=L.+N.+Clarkauthor=C.+H.+Cortonauthor=A.+Croninauthor=R.+M.+Daviesauthor=J.+Doggettauthor=A.+Djikengauthor=T.+Feldblyumauthor=M.+C.+Fieldauthor=A.+Fraserauthor=I.+Goodheadauthor=Z.+Hanceauthor=D.+Harperauthor=B.+R.+Harrisauthor=H.+Hauserauthor=J.+Hostetlerauthor=A.+Ivensauthor=K.+Jagelsauthor=D.+Johnsonauthor=J.+Johnsonauthor=K.+Jonesauthor=A.+X.+Kerhornouauthor=H.+Kooauthor=N.+Larkeauthor=S.+Landfearauthor=C.+Larkinauthor=V.+Leechauthor=A.+Lineauthor=A.+Lordauthor=A.+Macleodauthor=P.+J.+Mooneyauthor=S.+Mouleauthor=D.+M.+Martinauthor=G.+W.+Morganauthor=K.+Mungallauthor=H.+Norbertczakauthor=D.+Ormondauthor=G.+Paiauthor=C.+S.+Peacockauthor=J.+Petersonauthor=M.+A.+Quailauthor=E.+Rabbinowitschauthor=M.+A.+Rajandreamauthor=C.+Reitterauthor=S.+L.+Salzbergauthor=M.+Sandersauthor=S.+Schobelauthor=S.+Sharpauthor=M.+Simmondsauthor=A.+J.+Simpsonauthor=L.+Tallonauthor=C.+M.+Turnerauthor=A.+Taitauthor=A.+R.+Tiveyauthor=S.+Van+Akenauthor=D.+Walkerauthor=D.+Wanlessauthor=S.+Wangauthor=B.+Whiteauthor=O.+Whiteauthor=S.+Whiteheadauthor=J.+Woodwardauthor=J.+Wortmanauthor=M.+D.+Adamsauthor=T.+M.+Embleyauthor=K.+Gullauthor=E.+Ulluauthor=J.+D.+Barryauthor=A.+H.+Fairlambauthor=F.+Opperdoesauthor=B.+G.+Barrellauthor=J.+E.+Donelsonauthor=N.+Hallauthor=C.+M.+Fraserauthor=S.+E.+Melvilleauthor=N.+M.+El-Sayed&title=The+genome+of+the+African+trypanosome+Trypanosoma+brucei"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBerriman%26aufirst%3DM.%26aulast%3DGhedin%26aufirst%3DE.%26aulast%3DHertz-Fowler%26aufirst%3DC.%26aulast%3DBlandin%26aufirst%3DG.%26aulast%3DRenauld%26aufirst%3DH.%26aulast%3DBartholomeu%26aufirst%3DD.%2BC.%26aulast%3DLennard%26aufirst%3DN.%2BJ.%26aulast%3DCaler%26aufirst%3DE.%26aulast%3DHamlin%26aufirst%3DN.%2BE.%26aulast%3DHaas%26aufirst%3DB.%26aulast%3DBohme%26aufirst%3DU.%26aulast%3DHannick%26aufirst%3DL.%26aulast%3DAslett%26aufirst%3DM.%2BA.%26aulast%3DShallom%26aufirst%3DJ.%26aulast%3DMarcello%26aufirst%3DL.%26aulast%3DHou%26aufirst%3DL.%26aulast%3DWickstead%26aufirst%3DB.%26aulast%3DAlsmark%26aufirst%3DU.%2BC.%26aulast%3DArrowsmith%26aufirst%3DC.%26aulast%3DAtkin%26aufirst%3DR.%2BJ.%26aulast%3DBarron%26aufirst%3DA.%2BJ.%26aulast%3DBringaud%26aufirst%3DF.%26aulast%3DBrooks%26aufirst%3DK.%26aulast%3DCarrington%26aufirst%3DM.%26aulast%3DCherevach%26aufirst%3DI.%26aulast%3DChillingworth%26aufirst%3DT.%2BJ.%26aulast%3DChurcher%26aufirst%3DC.%26aulast%3DClark%26aufirst%3DL.%2BN.%26aulast%3DCorton%26aufirst%3DC.%2BH.%26aulast%3DCronin%26aufirst%3DA.%26aulast%3DDavies%26aufirst%3DR.%2BM.%26aulast%3DDoggett%26aufirst%3DJ.%26aulast%3DDjikeng%26aufirst%3DA.%26aulast%3DFeldblyum%26aufirst%3DT.%26aulast%3DField%26aufirst%3DM.%2BC.%26aulast%3DFraser%26aufirst%3DA.%26aulast%3DGoodhead%26aufirst%3DI.%26aulast%3DHance%26aufirst%3DZ.%26aulast%3DHarper%26aufirst%3DD.%26aulast%3DHarris%26aufirst%3DB.%2BR.%26aulast%3DHauser%26aufirst%3DH.%26aulast%3DHostetler%26aufirst%3DJ.%26aulast%3DIvens%26aufirst%3DA.%26aulast%3DJagels%26aufirst%3DK.%26aulast%3DJohnson%26aufirst%3DD.%26aulast%3DJohnson%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DK.%26aulast%3DKerhornou%26aufirst%3DA.%2BX.%26aulast%3DKoo%26aufirst%3DH.%26aulast%3DLarke%26aufirst%3DN.%26aulast%3DLandfear%26aufirst%3DS.%26aulast%3DLarkin%26aufirst%3DC.%26aulast%3DLeech%26aufirst%3DV.%26aulast%3DLine%26aufirst%3DA.%26aulast%3DLord%26aufirst%3DA.%26aulast%3DMacleod%26aufirst%3DA.%26aulast%3DMooney%26aufirst%3DP.%2BJ.%26aulast%3DMoule%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DD.%2BM.%26aulast%3DMorgan%26aufirst%3DG.%2BW.%26aulast%3DMungall%26aufirst%3DK.%26aulast%3DNorbertczak%26aufirst%3DH.%26aulast%3DOrmond%26aufirst%3DD.%26aulast%3DPai%26aufirst%3DG.%26aulast%3DPeacock%26aufirst%3DC.%2BS.%26aulast%3DPeterson%26aufirst%3DJ.%26aulast%3DQuail%26aufirst%3DM.%2BA.%26aulast%3DRabbinowitsch%26aufirst%3DE.%26aulast%3DRajandream%26aufirst%3DM.%2BA.%26aulast%3DReitter%26aufirst%3DC.%26aulast%3DSalzberg%26aufirst%3DS.%2BL.%26aulast%3DSanders%26aufirst%3DM.%26aulast%3DSchobel%26aufirst%3DS.%26aulast%3DSharp%26aufirst%3DS.%26aulast%3DSimmonds%26aufirst%3DM.%26aulast%3DSimpson%26aufirst%3DA.%2BJ.%26aulast%3DTallon%26aufirst%3DL.%26aulast%3DTurner%26aufirst%3DC.%2BM.%26aulast%3DTait%26aufirst%3DA.%26aulast%3DTivey%26aufirst%3DA.%2BR.%26aulast%3DVan%2BAken%26aufirst%3DS.%26aulast%3DWalker%26aufirst%3DD.%26aulast%3DWanless%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DWhite%26aufirst%3DB.%26aulast%3DWhite%26aufirst%3DO.%26aulast%3DWhitehead%26aufirst%3DS.%26aulast%3DWoodward%26aufirst%3DJ.%26aulast%3DWortman%26aufirst%3DJ.%26aulast%3DAdams%26aufirst%3DM.%2BD.%26aulast%3DEmbley%26aufirst%3DT.%2BM.%26aulast%3DGull%26aufirst%3DK.%26aulast%3DUllu%26aufirst%3DE.%26aulast%3DBarry%26aufirst%3DJ.%2BD.%26aulast%3DFairlamb%26aufirst%3DA.%2BH.%26aulast%3DOpperdoes%26aufirst%3DF.%26aulast%3DBarrell%26aufirst%3DB.%2BG.%26aulast%3DDonelson%26aufirst%3DJ.%2BE.%26aulast%3DHall%26aufirst%3DN.%26aulast%3DFraser%26aufirst%3DC.%2BM.%26aulast%3DMelville%26aufirst%3DS.%2BE.%26aulast%3DEl-Sayed%26aufirst%3DN.%2BM.%26atitle%3DThe%2520genome%2520of%2520the%2520African%2520trypanosome%2520Trypanosoma%2520brucei%26jtitle%3DScience%26date%3D2005%26volume%3D309%26spage%3D416%26epage%3D422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Gangjee, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, H. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurup, S.</span><span> </span><span class="NLM_article-title">Recent advances in classical and non-classical antifolates as antitumor and antiopportunistic infection agents: part I</span> <span class="citation_source-journal">Anticancer Agents Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">524</span><span class="NLM_x">–</span> <span class="NLM_lpage">542</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.2174%2F187152007781668724" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=17896913" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtV2gtr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=524-542&author=A.+Gangjeeauthor=H.+D.+Jainauthor=S.+Kurup&title=Recent+advances+in+classical+and+non-classical+antifolates+as+antitumor+and+antiopportunistic+infection+agents%3A+part+I"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in classical and non-classical antifolates as antitumor and antiopportunistic infection agents: part I</span></div><div class="casAuthors">Gangjee, Aleem; Jain, Hiteshkumar D.; Kurup, Sonali</div><div class="citationInfo"><span class="NLM_cas:title">Anti-Cancer Agents in Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">524-542</span>CODEN:
                <span class="NLM_cas:coden">AAMCE4</span>;
        ISSN:<span class="NLM_cas:issn">1871-5206</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Antifolates that inhibit the key enzymes thymidylate synthase (TS) and dihydrofolate reductase (DHFR) have found clin. utility as antitumor and antiopportunistic agents.  Methotrexate {MTX, (1)} and 5-fluorouracil (5-FU) were among the first clin. useful DHFR and TS inhibitors, resp.  The development of resistance to 5-FU, its occasional unpredictable activity and toxicity resulted in the search of novel antifolates.  Pemetrexed (4) and raltitrexed (5) specifically inhibit TS, and are clin. useful as antitumor agents.  A major mechanism of tumor resistance to clin. useful antifolates is based on their need for polyglutamylation via the enzyme folypoly-γ-glutamate synthetase (FPGS).  Novel antifolates have been developed that do not need to be polyglutamylated and include plevitrexed (6) and GWI843 (7).  Nonclassical antifolates for antitumor and parasitic chemotherapy, such as nolatrexed (8), trimethoprim {TMP, (11)} and piritrexim {PTX, (12)}, can passively diffuse into cells and hence do not have to depend on FPGS or the reduced folate carrier (RFC).  Variations in the structures of antifolates have helped delineate the structural influence on the interaction with TS, DHFR, FPGS, and RFC utilization.  The differences in the active site of human and pathogen DHFR have also been exploited.  The literature contains excellent reviews on the design and synthesis of antifolates prior to 1996.  This two-part review discusses the design, synthesis and structural requirements for TS and DHFR inhibition and their relevance to antitumor and parasitic chemotherapy, since 1996.  Monocyclic and 6-5 fused bicyclic antifolates will be discussed in Part I, while 6-6 bicyclic and tricyclic antifolates will be discussed in Part II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofSNk-MZR8S7Vg90H21EOLACvtfcHk0ljPH4FFNqfmNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtV2gtr3J&md5=c5a93e7c3fa50fdf071800131d0ef1db</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.2174%2F187152007781668724&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F187152007781668724%26sid%3Dliteratum%253Aachs%26aulast%3DGangjee%26aufirst%3DA.%26aulast%3DJain%26aufirst%3DH.%2BD.%26aulast%3DKurup%26aufirst%3DS.%26atitle%3DRecent%2520advances%2520in%2520classical%2520and%2520non-classical%2520antifolates%2520as%2520antitumor%2520and%2520antiopportunistic%2520infection%2520agents%253A%2520part%2520I%26jtitle%3DAnticancer%2520Agents%2520Med.%2520Chem.%26date%3D2007%26volume%3D7%26spage%3D524%26epage%3D542" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Gangjee, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, H. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurup, S.</span><span> </span><span class="NLM_article-title">Recent advances in classical and nonclassical antifolates as antitumor and antiopportunistic infection agents: Part II</span> <span class="citation_source-journal">Anticancer Agents Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">205</span><span class="NLM_x">–</span> <span class="NLM_lpage">231</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.2174%2F187152008783497064" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=18288923" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFOmsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=205-231&author=A.+Gangjeeauthor=H.+D.+Jainauthor=S.+Kurup&title=Recent+advances+in+classical+and+nonclassical+antifolates+as+antitumor+and+antiopportunistic+infection+agents%3A+Part+II"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in classical and non-classical antifolates as antitumor and antiopportunistic infection agents: Part II</span></div><div class="casAuthors">Gangjee, Aleem; Jain, Hiteshkumar D.; Kurup, Sonali</div><div class="citationInfo"><span class="NLM_cas:title">Anti-Cancer Agents in Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">205-231</span>CODEN:
                <span class="NLM_cas:coden">AAMCE4</span>;
        ISSN:<span class="NLM_cas:issn">1871-5206</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Antifolates that inhibit the key enzymes thymidylate synthase (TS) and dihydrofolate reductase (DHFR) have found clin. utility as antitumor and antiopportunistic agents.  Methotrexate (MTX) and 5-fluorouracil (5-FU) were among the first clin. useful DHFR and TS inhibitors, resp.  The development of resistance to 5-FU, its occasional unpredictable activity and toxicity resulted in the search of novel antifolates.  Pemetrexed and raltitrexed are newer antifolates that specifically inhibit TS, and are clin. useful as antitumor agents.  A major mechanism of tumor resistance to clin. useful antifolates is based on their need for polyglutamylation via the enzyme folylpoly-γ-glutamate synthetase (FPGS).  Recently, classical antifolates that do not need to be polyglutamylated have also been developed and include plevitrexed and GW1843.  Nolatrexed, trimethoprim (TMP) and piritrexim (PTX) are nonclassical antifolates for antitumor and parasitic chemotherapy that passively diffuse into cells and hence do not have to depend on FPGS or the reduced folate carrier (RFC).  Structural requirements for inhibition with antifolates have been studied extensively and novel agents that exploit key interactions in the active site of TS, DHFR, FPGS, and RFC have been proposed.  This two-part review discusses the design, synthesis and structural requirements for TS and DHFR inhibition and their relevance to antitumor and parasitic chemotherapy, since 1996.  Monocyclic and 6-5 fused bicyclic antifolates were discussed in Part I.  The 6-6 bicyclic and tricyclic antifolates will be discussed here in Part II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo94ZvcpFPCFbVg90H21EOLACvtfcHk0ljPH4FFNqfmNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFOmsbo%253D&md5=eaa3bca180f933e57873a7b2f26303d6</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.2174%2F187152008783497064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F187152008783497064%26sid%3Dliteratum%253Aachs%26aulast%3DGangjee%26aufirst%3DA.%26aulast%3DJain%26aufirst%3DH.%2BD.%26aulast%3DKurup%26aufirst%3DS.%26atitle%3DRecent%2520advances%2520in%2520classical%2520and%2520nonclassical%2520antifolates%2520as%2520antitumor%2520and%2520antiopportunistic%2520infection%2520agents%253A%2520Part%2520II%26jtitle%3DAnticancer%2520Agents%2520Med.%2520Chem.%26date%3D2008%26volume%3D8%26spage%3D205%26epage%3D231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Bello, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nare, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freedman, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hardy, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beverley, S. M.</span><span> </span><span class="NLM_article-title">PTR1: a reductase mediating salvage of oxidized pteridines and methotrexate resistance in the protozoan parasite <i>Leishmania major</i></span> <span class="citation_source-journal">Proc. Natl. Acad. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">91</span><span class="NLM_x">, </span> <span class="NLM_fpage">11442</span><span class="NLM_x">–</span> <span class="NLM_lpage">11446</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1073%2Fpnas.91.24.11442" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=7972081" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADyaK2MXitlyrs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=1994&pages=11442-11446&author=A.+R.+Belloauthor=B.+Nareauthor=D.+Freedmanauthor=L.+Hardyauthor=S.+M.+Beverley&title=PTR1%3A+a+reductase+mediating+salvage+of+oxidized+pteridines+and+methotrexate+resistance+in+the+protozoan+parasite+Leishmania+major"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">PTR1: a reductase mediating salvage of oxidized pteridines and methotrexate resistance in the protozoan parasite Leishmania major</span></div><div class="casAuthors">Bello, Alexandre R.; Nare, Bakela; Freedman, Daniel; Hardy, Larry; Beverley, Stephen M.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11442-6</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Trypanosomatid protozoans are pterin auxotrophs, a finding noted decades ago which heralded the discovery of key metabolic roles played by pteridines in eukaryotes.  We have now identified the enzyme mediating unconjugated pteridine salvage in the human parasite Leishmania major.  PTR1 is the gene in the amplified H region responsible for methotrexate (MTX) resistance, and belongs to a large family of oxidoreductases with diverse substrates and roles.  We generated Leishmania lacking PTR1 by homologous gene targeting, and these ptr1- mutants required reduced biopterin (dihydro- or tetrahydrobiopterin) for growth.  PTR1 purified from engineered Escherichia coli exhibited a MTX-sensitive, NADPH-dependent biopterin reductase activity.  PTR1 showed good activity with folate and significant activity with dihydrofolate and dihydrobiopterin, but not with quinonoid dihydrobiopterin.  PTR1 thus differs considerably from previously reported pteridine reductases of trypanosomatids and vertebrates.  Pteridine reductase activity was diminished in ptr1- Leishmania and was evaluated in transfected parasites bearing multiple copies of PTR1; correspondingly, ptr1- was MTX-hypersensitive whereas the multicopy transfectant was MTX-resistant.  The concordance of the biochem. and genetic properties of PTR1 suggests that this is the primary enzyme mediating pteridine salvage.  These findings suggest several possible mechanisms for PTR1-mediated MTX resistance and should aid in the design of rational chemotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiIWNgbgpSOrVg90H21EOLACvtfcHk0ljZZt_3_vpvdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXitlyrs7c%253D&md5=202ac2d39e22c7e1a6fe3cf4aeb3943c</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1073%2Fpnas.91.24.11442&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.91.24.11442%26sid%3Dliteratum%253Aachs%26aulast%3DBello%26aufirst%3DA.%2BR.%26aulast%3DNare%26aufirst%3DB.%26aulast%3DFreedman%26aufirst%3DD.%26aulast%3DHardy%26aufirst%3DL.%26aulast%3DBeverley%26aufirst%3DS.%2BM.%26atitle%3DPTR1%253A%2520a%2520reductase%2520mediating%2520salvage%2520of%2520oxidized%2520pteridines%2520and%2520methotrexate%2520resistance%2520in%2520the%2520protozoan%2520parasite%2520Leishmania%2520major%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%26date%3D1994%26volume%3D91%26spage%3D11442%26epage%3D11446" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Sienkiewicz, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaroslawski, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyllie, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairlamb, A. H.</span><span> </span><span class="NLM_article-title">Chemical and genetic validation of dihydrofolate reductase-thymidylate synthase as a drug target in African trypanosomes</span> <span class="citation_source-journal">Mol. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">520</span><span class="NLM_x">–</span> <span class="NLM_lpage">533</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1111%2Fj.1365-2958.2008.06305.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=18557814" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD1cXoslyjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2008&pages=520-533&author=N.+Sienkiewiczauthor=S.+Jaroslawskiauthor=S.+Wyllieauthor=A.+H.+Fairlamb&title=Chemical+and+genetic+validation+of+dihydrofolate+reductase-thymidylate+synthase+as+a+drug+target+in+African+trypanosomes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical and genetic validation of dihydrofolate reductase-thymidylate synthase as a drug target in African trypanosomes</span></div><div class="casAuthors">Sienkiewicz, Natasha; Jaroslawski, Szymon; Wyllie, Susan; Fairlamb, Alan H.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Microbiology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">520-533</span>CODEN:
                <span class="NLM_cas:coden">MOMIEE</span>;
        ISSN:<span class="NLM_cas:issn">0950-382X</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The phenotypes of single-(SKO) and double-knockout (DKO) lines of dihydrofolate reductase-thymidylate synthase (DHFR-TS) of bloodstream Trypanosoma brucei were evaluated in vitro and in vivo.  Growth of SKO in vitro is identical to wild-type (WT) cells, whereas DKO has an abs. requirement for thymidine.  Removal of thymidine from the medium triggers growth arrest in S phase, assocd. with gross morphol. changes, followed by cell death after 60 h.  DKO is unable to infect mice, whereas the virulence of SKO is similar to WT.  Normal growth and virulence could be restored by transfection of DKO with T. brucei DHFR-TS, but not with Escherichia coli TS.  As pteridine reductase (PTR1) levels are unchanged in SKO and DKO cells, PTR1 is not able to compensate for loss of DHFR activity.  Drugs such as raltitrexed or methotrexate with structural similarity to folic acid are up to 300-fold more potent inhibitors of WT cultured in a novel low-folate medium, unlike hydrophobic antifols such as trimetrexate or pyrimethamine.  DKO trypanosomes show reduced sensitivity to these inhibitors ranging from twofold for trimetrexate to > 10 000-fold for raltitrexed.  These data demonstrate that DHFR-TS is essential for parasite survival and represents a promising target for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAtWTOrNZCBLVg90H21EOLACvtfcHk0ljZZt_3_vpvdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXoslyjt7w%253D&md5=026298043b57f45ae922ac3833100a0c</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2958.2008.06305.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2958.2008.06305.x%26sid%3Dliteratum%253Aachs%26aulast%3DSienkiewicz%26aufirst%3DN.%26aulast%3DJaroslawski%26aufirst%3DS.%26aulast%3DWyllie%26aufirst%3DS.%26aulast%3DFairlamb%26aufirst%3DA.%2BH.%26atitle%3DChemical%2520and%2520genetic%2520validation%2520of%2520dihydrofolate%2520reductase-thymidylate%2520synthase%2520as%2520a%2520drug%2520target%2520in%2520African%2520trypanosomes%26jtitle%3DMol.%2520Microbiol.%26date%3D2008%26volume%3D69%26spage%3D520%26epage%3D533" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Nare, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hardy, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beverley, S. M.</span><span> </span><span class="NLM_article-title">The roles of pteridine reductase 1 and dihydrofolate reductase-thymidylate synthase in pteridine metabolism in the protozoan parasite <i>Leishmania major</i></span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">272</span><span class="NLM_x">, </span> <span class="NLM_fpage">13883</span><span class="NLM_x">–</span> <span class="NLM_lpage">13891</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=272&publication_year=1997&pages=13883-13891&author=B.+Nareauthor=L.+W.+Hardyauthor=S.+M.+Beverley&title=The+roles+of+pteridine+reductase+1+and+dihydrofolate+reductase-thymidylate+synthase+in+pteridine+metabolism+in+the+protozoan+parasite+Leishmania+major"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNare%26aufirst%3DB.%26aulast%3DHardy%26aufirst%3DL.%2BW.%26aulast%3DBeverley%26aufirst%3DS.%2BM.%26atitle%3DThe%2520roles%2520of%2520pteridine%2520reductase%25201%2520and%2520dihydrofolate%2520reductase-thymidylate%2520synthase%2520in%2520pteridine%2520metabolism%2520in%2520the%2520protozoan%2520parasite%2520Leishmania%2520major%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1997%26volume%3D272%26spage%3D13883%26epage%3D13891" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Cunningham, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Titus, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turco, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beverley, S. M.</span><span> </span><span class="NLM_article-title">Regulation of differentiation to the infective stage of the protozoan parasite <i>Leishmania</i> major by tetrahydrobiopterin</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">292</span><span class="NLM_x">, </span> <span class="NLM_fpage">285</span><span class="NLM_x">–</span> <span class="NLM_lpage">287</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2001&pages=285-287&author=M.+L.+Cunninghamauthor=R.+G.+Titusauthor=S.+J.+Turcoauthor=S.+M.+Beverley&title=Regulation+of+differentiation+to+the+infective+stage+of+the+protozoan+parasite+Leishmania+major+by+tetrahydrobiopterin"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCunningham%26aufirst%3DM.%2BL.%26aulast%3DTitus%26aufirst%3DR.%2BG.%26aulast%3DTurco%26aufirst%3DS.%2BJ.%26aulast%3DBeverley%26aufirst%3DS.%2BM.%26atitle%3DRegulation%2520of%2520differentiation%2520to%2520the%2520infective%2520stage%2520of%2520the%2520protozoan%2520parasite%2520Leishmania%2520major%2520by%2520tetrahydrobiopterin%26jtitle%3DScience%26date%3D2001%26volume%3D292%26spage%3D285%26epage%3D287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Cavazzuti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paglietti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, W. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gamarro, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piras, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loriga, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allecca, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corona, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLuskey, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tulloch, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibellini, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costi, M. P.</span><span> </span><span class="NLM_article-title">Discovery of potent pteridine reductase inhibitors to guide antiparasite drug development</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">1448</span><span class="NLM_x">–</span> <span class="NLM_lpage">1453</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1073%2Fpnas.0704384105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=18245389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhvFCitrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=1448-1453&author=A.+Cavazzutiauthor=G.+Pagliettiauthor=W.+N.+Hunterauthor=F.+Gamarroauthor=S.+Pirasauthor=M.+Lorigaauthor=S.+Alleccaauthor=P.+Coronaauthor=K.+McLuskeyauthor=L.+Tullochauthor=F.+Gibelliniauthor=S.+Ferrariauthor=M.+P.+Costi&title=Discovery+of+potent+pteridine+reductase+inhibitors+to+guide+antiparasite+drug+development"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of potent pteridine reductase inhibitors to guide antiparasite drug development</span></div><div class="casAuthors">Cavazzuti, Antonio; Paglietti, Giuseppe; Hunter, William N.; Gamarro, Francisco; Piras, Sandra; Loriga, Mario; Alleca, Sergio; Corona, Paola; McLuskey, Karen; Tulloch, Lindsay; Gibellini, Federica; Ferrari, Stefania; Costi, Maria Paola</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1448-1453</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Pteridine reductase (PTR1) is essential for salvage of pterins by parasitic trypanosomatids and is a target for the development of improved therapies.  To identify inhibitors of Leishmania major and Trypanosoma cruzi PTR1, a rapid-screening strategy using a folate-based library was combined with structure-based design.  Assays were carried out against folate-dependent enzymes including PTR1, dihydrofolate reductase (DHFR), and thymidylate synthase.  Affinity profiling detd. selectivity and specificity of a series of quinoxaline and 2,4-diaminopteridine derivs., and nine compds. showed greater activity against parasite enzymes compared with human enzymes.  Compd. I [R = H, Me (II)] displayed a Ki of 100 nM toward LmPTR1, and the crystal structure of the LmPTR1:NADPH:I ternary complex revealed a substrate-like binding mode distinct from that previously obsd. for similar compds.  A second round of design, synthesis, and assay produced a compd. II with a significantly improved Ki (37 nM) against LmPTR1, and the structure of this complex was also detd.  Biol. evaluation of selected inhibitors was performed against the extracellular forms of T. cruzi and L. major, both wild-type and overexpressing PTR1 lines, as a model for PTR1-driven antifolate drug resistance and the intracellular form of T. cruzi.  An additive profile was obsd. when PTR1 inhibitors were used in combination with known DHFR inhibitors, and a redn. in toxicity of treatment was obsd. with respect to administration of a DHFR inhibitor alone.  The successful combination of antifolates targeting two enzymes indicates high potential for such an approach in the development of previously undescribed antiparasitic drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpclEp8-gcEyrVg90H21EOLACvtfcHk0ljRSqbLFOSsnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhvFCitrk%253D&md5=d2477ec35df976858f13b6d14d283cad</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0704384105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0704384105%26sid%3Dliteratum%253Aachs%26aulast%3DCavazzuti%26aufirst%3DA.%26aulast%3DPaglietti%26aufirst%3DG.%26aulast%3DHunter%26aufirst%3DW.%2BN.%26aulast%3DGamarro%26aufirst%3DF.%26aulast%3DPiras%26aufirst%3DS.%26aulast%3DLoriga%26aufirst%3DM.%26aulast%3DAllecca%26aufirst%3DS.%26aulast%3DCorona%26aufirst%3DP.%26aulast%3DMcLuskey%26aufirst%3DK.%26aulast%3DTulloch%26aufirst%3DL.%26aulast%3DGibellini%26aufirst%3DF.%26aulast%3DFerrari%26aufirst%3DS.%26aulast%3DCosti%26aufirst%3DM.%2BP.%26atitle%3DDiscovery%2520of%2520potent%2520pteridine%2520reductase%2520inhibitors%2520to%2520guide%2520antiparasite%2520drug%2520development%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%26date%3D2008%26volume%3D105%26spage%3D1448%26epage%3D1453" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Hardy, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matthews, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nare, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beverley, S. M.</span><span> </span><span class="NLM_article-title">Biochemical and Genetic Tests for Inhibitors of <i>Leishmania Pteridine</i> Pathways</span> <span class="citation_source-journal">Exp. Parasitol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">87</span><span class="NLM_x">, </span> <span class="NLM_fpage">158</span><span class="NLM_x">–</span> <span class="NLM_lpage">170</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1006%2Fexpr.1997.4207" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=9398595" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=1997&pages=158-170&author=L.+W.+Hardyauthor=W.+Matthewsauthor=B.+Nareauthor=S.+M.+Beverley&title=Biochemical+and+Genetic+Tests+for+Inhibitors+of+Leishmania+Pteridine+Pathways"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1006%2Fexpr.1997.4207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fexpr.1997.4207%26sid%3Dliteratum%253Aachs%26aulast%3DHardy%26aufirst%3DL.%2BW.%26aulast%3DMatthews%26aufirst%3DW.%26aulast%3DNare%26aufirst%3DB.%26aulast%3DBeverley%26aufirst%3DS.%2BM.%26atitle%3DBiochemical%2520and%2520Genetic%2520Tests%2520for%2520Inhibitors%2520of%2520Leishmania%2520Pteridine%2520Pathways%26jtitle%3DExp.%2520Parasitol.%26date%3D1997%26volume%3D87%26spage%3D158%26epage%3D170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Ecker, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Noe, C. R.</span><span> </span><span class="NLM_article-title">In silico prediction models for blood−brain barrier permeation</span> <span class="citation_source-journal">Curr. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1617</span><span class="NLM_x">–</span> <span class="NLM_lpage">1628</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=15180567" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD2cXktlKksrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2004&pages=1617-1628&author=G.+F.+Eckerauthor=C.+R.+Noe&title=In+silico+prediction+models+for+blood%E2%88%92brain+barrier+permeation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">In silico prediction models for blood-brain barrier permeation</span></div><div class="casAuthors">Ecker, Gerhard F.; Noe, Christian R.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1617-1628</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The ability to permeate across the blood brain barrier (BBB) is essential for drugs acting on the central nervous system (CNS).  Thus, for speeding up the drug discovery process in the CNS-area, it is of great importance to develop systems that allow rapid and inexpensive screening of the BBB-permeability properties of novel lead compds. or at least small subsets of combinatorial CNS-libraries.  In this field, in silico prediction methods gain increasing importance.  Starting with simple regression models based on calcn. of lipophilicity and polar surface area, the field developed via PLS methods to grid based approaches (e.g. VolSurf).  Addnl., the use of artificial neural networks gain increasing importance.  However, permeation through the BBB is also influenced by active transport systems.  For nutrients and endogenous compds., such as amino acids, monocarboxylic acids, amines, hexoses, thyroid hormones, purine bases, and nucleosides, several transport systems regulating the entry of the resp. compd. classes into the brain were identified.  The other way round there is striking evidence that expression of active efflux pumps like the multidrug transporter P-glycoprotein (P-gp) on the luminal membrane of the brain capillary endothelial cells accounts for poor BBB permeability of certain drugs.  Undoubtedly, P-gp is an important impediment for the entry of hydrophobic drugs into the brain.  Thus, proper prediction models should also take into account the active transport phenomena.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnO4MhXnN3lrVg90H21EOLACvtfcHk0ljRSqbLFOSsnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXktlKksrw%253D&md5=4bf6aced7aaa6bb593c183d135b3ad2c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEcker%26aufirst%3DG.%2BF.%26aulast%3DNoe%26aufirst%3DC.%2BR.%26atitle%3DIn%2520silico%2520prediction%2520models%2520for%2520blood%25E2%2588%2592brain%2520barrier%2520permeation%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2004%26volume%3D11%26spage%3D1617%26epage%3D1628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Dawson, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibellini, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sienkiewicz, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tulloch, L. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fyfe, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLuskey, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairlamb, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, W. N.</span><span> </span><span class="NLM_article-title">Structure and reactivity of <i>Trypanosoma brucei</i> pteridine reductase: inhibition by the archetypal antifolate methotrexate</span> <span class="citation_source-journal">Mol. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">1457</span><span class="NLM_x">–</span> <span class="NLM_lpage">1468</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1111%2Fj.1365-2958.2006.05332.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=16968221" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFensbrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2006&pages=1457-1468&author=A.+Dawsonauthor=F.+Gibelliniauthor=N.+Sienkiewiczauthor=L.+B.+Tullochauthor=P.+K.+Fyfeauthor=K.+McLuskeyauthor=A.+H.+Fairlambauthor=W.+N.+Hunter&title=Structure+and+reactivity+of+Trypanosoma+brucei+pteridine+reductase%3A+inhibition+by+the+archetypal+antifolate+methotrexate"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and reactivity of Trypanosoma brucei pteridine reductase: inhibition by the archetypal antifolate methotrexate</span></div><div class="casAuthors">Dawson, Alice; Gibellini, Federica; Sienkiewicz, Natasha; Tulloch, Lindsay B.; Fyfe, Paul K.; McLuskey, Karen; Fairlamb, Alan H.; Hunter, William N.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Microbiology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1457-1468</span>CODEN:
                <span class="NLM_cas:coden">MOMIEE</span>;
        ISSN:<span class="NLM_cas:issn">0950-382X</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The protozoan Trypanosoma brucei has a functional pteridine reductase (TbPTR1), an NADPH-dependent short-chain reductase that participates in the salvage of pterins, which are essential for parasite growth.  PTR1 displays broad-spectrum activity with pterins and folates, provides a metabolic bypass for inhibition of the trypanosomatid dihydrofolate reductase and therefore compromises the use of antifolates for treatment of trypanosomiasis.  Catalytic properties of recombinant TbPTR1 and inhibition by the archetypal antifolate methotrexate have been characterized and the crystal structure of the ternary complex with cofactor NADP+ and the inhibitor detd. at 2.2 Å resoln.  This enzyme shares 50% amino acid sequence identity with Leishmania major PTR1 (LmPTR1) and comparisons show that the architecture of the cofactor binding site, and the catalytic center are highly conserved, as are most interactions with the inhibitor.  However, specific amino acid differences, in particular the placement of Trp221 at the side of the active site, and adjustment of the β6-α6 loop and α6 helix at one side of the substrate-binding cleft significantly reduce the size of the substrate binding site of TbPTR1 and alter the chem. properties compared with LmPTR1.  A reactive Cys168, within the active site cleft, in conjunction with the C-terminus carboxyl group and His267 of a partner subunit forms a triad similar to the catalytic component of cysteine proteases.  TbPTR1 therefore offers novel structural features to exploit in the search for inhibitors of therapeutic value against African trypanosomiasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEKvWRoJekYrVg90H21EOLACvtfcHk0lj3WbSCpfKEtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFensbrF&md5=aa70764962fef14898e2b1bb22e646b3</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2958.2006.05332.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2958.2006.05332.x%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DA.%26aulast%3DGibellini%26aufirst%3DF.%26aulast%3DSienkiewicz%26aufirst%3DN.%26aulast%3DTulloch%26aufirst%3DL.%2BB.%26aulast%3DFyfe%26aufirst%3DP.%2BK.%26aulast%3DMcLuskey%26aufirst%3DK.%26aulast%3DFairlamb%26aufirst%3DA.%2BH.%26aulast%3DHunter%26aufirst%3DW.%2BN.%26atitle%3DStructure%2520and%2520reactivity%2520of%2520Trypanosoma%2520brucei%2520pteridine%2520reductase%253A%2520inhibition%2520by%2520the%2520archetypal%2520antifolate%2520methotrexate%26jtitle%3DMol.%2520Microbiol.%26date%3D2006%26volume%3D61%26spage%3D1457%26epage%3D1468" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Gourley, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuttelkopf, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leonard, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luba, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hardy, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beverley, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, W. N.</span><span> </span><span class="NLM_article-title">Pteridine reductase mechanism correlates pterin metabolism with drug resistance in trypanosomatid parasites</span> <span class="citation_source-journal">Nat. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">521</span><span class="NLM_x">–</span> <span class="NLM_lpage">525</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1038%2F88584" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=11373620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD3MXktFektLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2001&pages=521-525&author=D.+G.+Gourleyauthor=A.+W.+Schuttelkopfauthor=G.+A.+Leonardauthor=J.+Lubaauthor=L.+W.+Hardyauthor=S.+M.+Beverleyauthor=W.+N.+Hunter&title=Pteridine+reductase+mechanism+correlates+pterin+metabolism+with+drug+resistance+in+trypanosomatid+parasites"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Pteridine reductase mechanism correlates pterin metabolism with drug resistance in trypanosomatid parasites</span></div><div class="casAuthors">Gourley, David G.; Schottelkopf, Alexander W.; Leonard, Gordon A.; Luba, James; Hardy, Larry W.; Beverley, Stephen M.; Hunter, William N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">521-525</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature America Inc.</span>)
        </div><div class="casAbstract">Pteridine reductase (PTR1) is a short-chain reductase (SDR) responsible for the salvage of pterins in parasitic trypanosomatids.  PTR1 catalyzes the NADPH-dependent two-step redn. of oxidized pterins to the active tetrahydro-forms and reduces susceptibility to antifolates by alleviating dihydrofolate reductase (DHFR) inhibition.  Crystal structure of PTR1 complexed with cofactor and 7,8-dihydrobiopterin (DHB) or methotrexate (MTX) delineate the enzyme mechanism, broad spectrum of activity and inhibition by substrate or an antifolate.  PTR1 applies two distinct reductive mechanisms to substrates bound in one orientation.  The first redn. uses the generic SDR mechanism, whereas the second shares similarities with the mechanism proposed for DHFR.  Both DHB and MTX form extensive hydrogen bonding networks with NADP(H) but differ in the orientation of the pteridine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrB1KQsn-v77Vg90H21EOLACvtfcHk0lj3WbSCpfKEtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXktFektLw%253D&md5=b5be3a57375838e0a9b41429794ba37f</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2F88584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F88584%26sid%3Dliteratum%253Aachs%26aulast%3DGourley%26aufirst%3DD.%2BG.%26aulast%3DSchuttelkopf%26aufirst%3DA.%2BW.%26aulast%3DLeonard%26aufirst%3DG.%2BA.%26aulast%3DLuba%26aufirst%3DJ.%26aulast%3DHardy%26aufirst%3DL.%2BW.%26aulast%3DBeverley%26aufirst%3DS.%2BM.%26aulast%3DHunter%26aufirst%3DW.%2BN.%26atitle%3DPteridine%2520reductase%2520mechanism%2520correlates%2520pterin%2520metabolism%2520with%2520drug%2520resistance%2520in%2520trypanosomatid%2520parasites%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D2001%26volume%3D8%26spage%3D521%26epage%3D525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Congreve, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chessari, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tisi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodhead, A. J.</span><span> </span><span class="NLM_article-title">Recent Developments in Fragment-Based Drug Discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">3661</span><span class="NLM_x">–</span> <span class="NLM_lpage">3680</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm8000373" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=3661-3680&author=M.+Congreveauthor=G.+Chessariauthor=D.+Tisiauthor=A.+J.+Woodhead&title=Recent+Developments+in+Fragment-Based+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm8000373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8000373%26sid%3Dliteratum%253Aachs%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DTisi%26aufirst%3DD.%26aulast%3DWoodhead%26aufirst%3DA.%2BJ.%26atitle%3DRecent%2520Developments%2520in%2520Fragment-Based%2520Drug%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D3661%26epage%3D3680" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greer, J.</span><span> </span><span class="NLM_article-title">A decade of fragment-based drug design: strategic advances and lessons learned</span> <span class="citation_source-journal">Nat. Rev. Drug. Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">211</span><span class="NLM_x">–</span> <span class="NLM_lpage">219</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1038%2Fnrd2220" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=17290284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD2sXit12rtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=211-219&author=P.+J.+Hajdukauthor=J.+Greer&title=A+decade+of+fragment-based+drug+design%3A+strategic+advances+and+lessons+learned"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">A decade of fragment-based drug design: strategic advances and lessons learned</span></div><div class="casAuthors">Hajduk, Philip J.; Greer, Jonathan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">211-219</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Since the early 1990s, several technol. and scientific advances - such as combinatorial chem., high-throughput screening and the sequencing of the human genome - have been heralded as remedies to the problems facing the pharmaceutical industry.  The use of these technologies in some form is now well established at most pharmaceutical companies; however, the return on investment in terms of marketed products has not met expectations.  Fragment-based drug design is another tool for drug discovery that has emerged in the past decade.  Here, we describe the development and evolution of fragment-based drug design, analyze the role that this approach can have in combination with other discovery technologies and highlight the impact that fragment-based methods have made in progressing new medicines into the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOB8K3AESqTbVg90H21EOLACvtfcHk0ljWDDhtjFN6Zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXit12rtr4%253D&md5=bae574da3671475d8488615c3bfa9849</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnrd2220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2220%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DGreer%26aufirst%3DJ.%26atitle%3DA%2520decade%2520of%2520fragment-based%2520drug%2520design%253A%2520strategic%2520advances%2520and%2520lessons%2520learned%26jtitle%3DNat.%2520Rev.%2520Drug.%2520Discovery%26date%3D2007%26volume%3D6%26spage%3D211%26epage%3D219" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Brenk, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schipani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krasowski, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, I. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frearson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyatt, P. G.</span><span> </span><span class="NLM_article-title">Lessons learned from assembling screening libraries for drug discovery for neglected diseases</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">435</span><span class="NLM_x">–</span> <span class="NLM_lpage">444</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=435-444&author=R.+Brenkauthor=A.+Schipaniauthor=D.+Jamesauthor=A.+Krasowskiauthor=I.+H.+Gilbertauthor=J.+Frearsonauthor=P.+G.+Wyatt&title=Lessons+learned+from+assembling+screening+libraries+for+drug+discovery+for+neglected+diseases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBrenk%26aufirst%3DR.%26aulast%3DSchipani%26aufirst%3DA.%26aulast%3DJames%26aufirst%3DD.%26aulast%3DKrasowski%26aufirst%3DA.%26aulast%3DGilbert%26aufirst%3DI.%2BH.%26aulast%3DFrearson%26aufirst%3DJ.%26aulast%3DWyatt%26aufirst%3DP.%2BG.%26atitle%3DLessons%2520learned%2520from%2520assembling%2520screening%2520libraries%2520for%2520drug%2520discovery%2520for%2520neglected%2520diseases%26jtitle%3DChemMedChem%26date%3D2008%26volume%3D3%26spage%3D435%26epage%3D444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Lorber, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Flexible ligand docking using conformational ensembles</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">938</span><span class="NLM_x">–</span> <span class="NLM_lpage">950</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1002%2Fpro.5560070411" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=9568900" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADyaK1cXisFSnurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1998&pages=938-950&author=D.+M.+Lorberauthor=B.+K.+Shoichet&title=Flexible+ligand+docking+using+conformational+ensembles"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Flexible ligand docking using conformational ensembles</span></div><div class="casAuthors">Lorber, David M.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">938-950</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">Mol. docking algorithms suggest possible structures for mol. complexes.  They are used to model biol. function and to discover potential ligands.  A present challenge for docking algorithms is the treatment of mol. flexibility.  Here, the rigid body program, DOCK, is modified to allow it to rapidly fit multiple conformations of ligands.  Conformations of a given mol. are pre-calcd. in the same frame of ref., so that each conformer shares a common rigid fragment with all other conformations.  The ligand conformers are then docked together, as an ensemble, into a receptor binding site.  This takes advantage of the redundancy present in differing conformers of the same mol.  The algorithm was tested using three org. ligand protein systems and two protein-protein systems.  Both the bound and unbound conformations of the receptors were used.  The ligand ensemble method found conformations that resembled those detd. in X-ray crystal structures (RMS values typically less than 1.5 Å).  To test the method's usefulness for inhibitor discovery, multi-compd. and multi-conformer databases were screened for compds. known to bind to dihydrofolate reductase and compds. known to bind to thymidylate synthase.  In both cases, known inhibitors and substrates were identified in conformations resembling those obsd. exptl.  The ligand ensemble method was 100-fold faster than docking a single conformation at a time and was able to screen a database of over 34 million conformations from 117,000 mols. in one to four CPU days on a workstation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8GYuDVf7zG7Vg90H21EOLACvtfcHk0ljWDDhtjFN6Zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXisFSnurg%253D&md5=64d79dfe6a048c76aec43f8bd9f6bc62</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1002%2Fpro.5560070411&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.5560070411%26sid%3Dliteratum%253Aachs%26aulast%3DLorber%26aufirst%3DD.%2BM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DFlexible%2520ligand%2520docking%2520using%2520conformational%2520ensembles%26jtitle%3DProtein%2520Sci.%26date%3D1998%26volume%3D7%26spage%3D938%26epage%3D950" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Wei, B. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baase, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weaver, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matthews, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">A model binding site for testing scoring functions in molecular docking</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">322</span><span class="NLM_x">, </span> <span class="NLM_fpage">339</span><span class="NLM_x">–</span> <span class="NLM_lpage">355</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1016%2FS0022-2836%2802%2900777-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=12217695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD38Xms1Wqsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=322&publication_year=2002&pages=339-355&author=B.+Q.+Weiauthor=W.+A.+Baaseauthor=L.+H.+Weaverauthor=B.+W.+Matthewsauthor=B.+K.+Shoichet&title=A+model+binding+site+for+testing+scoring+functions+in+molecular+docking"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A Model Binding Site for Testing Scoring Functions in Molecular Docking</span></div><div class="casAuthors">Wei, Binqing Q.; Baase, Walter A.; Weaver, Larry H.; Matthews, Brian W.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">322</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">339-355</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Prediction of interaction energies between ligands and their receptors remains a major challenge for structure-based inhibitor discovery.  Much effort has been devoted to developing scoring schemes that can successfully rank the affinities of a diverse set of possible ligands to a binding site for which the structure is known.  To test these scoring functions, well-characterized exptl. systems can be very useful.  Here, mutation-created binding sites in T4 lysozyme were used to investigate how the quality of at. charges and solvation energies affects mol. docking.  At. charges and solvation energies were calcd. for 172,118 mols. in the Available Chems.  Directory using a semi-empirical quantum mech. approach by the program AMSOL.  The database was first screened against the apolar cavity site created by the mutation Leu99Ala (L99A).  Compared to the electronegativity-based charges that are widely used, the new charges and desolvation energies improved ranking of known apolar ligands, and better distinguished them from more polar isosteres that are not obsd. to bind.  To investigate whether the new charges had predictive value, the non-polar residue Met102, which forms part of the binding site, was changed to the polar residue glutamine.  The structure of the resulting Leu99 Ala and Met102 Gln double mutant of T4 lysozyme (L99A/M102Q) was detd. and the docking calcn. was repeated for the new site.  Seven representative polar mols. that preferentially docked to the polar vs. the apolar binding site were tested exptl.  All seven bind to the polar cavity (L99A/M102Q) but do not detectably bind to the apolar cavity (L99A).  Five ligand-bound structures of L99A/M102Q were detd. by X-ray crystallog.  Docking predictions corresponded to the crystallog. results to within 0.4 A RMSD.  Improved treatment of partial at. charges and desolvation energies in database docking appears feasible and leads to better distinction of true ligands.  Simple model binding sites, such as L99A and its more polar variants, may find broad use in the development and testing of docking algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGral4WDhJW6jbVg90H21EOLACvtfcHk0ljWDDhtjFN6Zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xms1Wqsrs%253D&md5=040e009b33f125ab4abbb322cf32d2c9</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2802%2900777-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252802%252900777-5%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DB.%2BQ.%26aulast%3DBaase%26aufirst%3DW.%2BA.%26aulast%3DWeaver%26aufirst%3DL.%2BH.%26aulast%3DMatthews%26aufirst%3DB.%2BW.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DA%2520model%2520binding%2520site%2520for%2520testing%2520scoring%2520functions%2520in%2520molecular%2520docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2002%26volume%3D322%26spage%3D339%26epage%3D355" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alex, A.</span><span> </span><span class="NLM_article-title">Ligand efficiency: a useful metric for lead selection</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">430</span><span class="NLM_x">–</span> <span class="NLM_lpage">431</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1016%2FS1359-6446%2804%2903069-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=15109945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=430-431&author=A.+L.+Hopkinsauthor=C.+R.+Groomauthor=A.+Alex&title=Ligand+efficiency%3A+a+useful+metric+for+lead+selection"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency: a useful metric for lead selection</span></div><div class="casAuthors">Hopkins Andrew L; Groom Colin R; Alex Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">430-1</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdMyKW_d6RPeIHSmpohpd0fW6udTcc2eYEqFqK2qKWNbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D&md5=eb19a1bca53247d0bd9adcb99d5817e6</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903069-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903069-7%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DAlex%26aufirst%3DA.%26atitle%3DLigand%2520efficiency%253A%2520a%2520useful%2520metric%2520for%2520lead%2520selection%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2004%26volume%3D9%26spage%3D430%26epage%3D431" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Schuttelkopf, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hardy, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beverley, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, W. N.</span><span> </span><span class="NLM_article-title">Structures of <i>Leishmania major</i> Pteridine Reductase Complexes Reveal the Active Site Features Important for Ligand Binding and to Guide Inhibitor Design</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">352</span><span class="NLM_x">, </span> <span class="NLM_fpage">105</span><span class="NLM_x">–</span> <span class="NLM_lpage">116</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=352&publication_year=2005&pages=105-116&author=A.+W.+Schuttelkopfauthor=L.+W.+Hardyauthor=S.+M.+Beverleyauthor=W.+N.+Hunter&title=Structures+of+Leishmania+major+Pteridine+Reductase+Complexes+Reveal+the+Active+Site+Features+Important+for+Ligand+Binding+and+to+Guide+Inhibitor+Design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSchuttelkopf%26aufirst%3DA.%2BW.%26aulast%3DHardy%26aufirst%3DL.%2BW.%26aulast%3DBeverley%26aufirst%3DS.%2BM.%26aulast%3DHunter%26aufirst%3DW.%2BN.%26atitle%3DStructures%2520of%2520Leishmania%2520major%2520Pteridine%2520Reductase%2520Complexes%2520Reveal%2520the%2520Active%2520Site%2520Features%2520Important%2520for%2520Ligand%2520Binding%2520and%2520to%2520Guide%2520Inhibitor%2520Design%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2005%26volume%3D352%26spage%3D105%26epage%3D116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Chen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lyne, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giordanetto, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lovell, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span> </span><span class="NLM_article-title">On evaluating molecular-docking methods for pose prediction and enrichment factors</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">401</span><span class="NLM_x">–</span> <span class="NLM_lpage">415</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci0503255" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlShtLvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=401-415&author=H.+Chenauthor=P.+D.+Lyneauthor=F.+Giordanettoauthor=T.+Lovellauthor=J.+Li&title=On+evaluating+molecular-docking+methods+for+pose+prediction+and+enrichment+factors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">On Evaluating Molecular-Docking Methods for Pose Prediction and Enrichment Factors</span></div><div class="casAuthors">Chen, Hongming; Lyne, Paul D.; Giordanetto, Fabrizio; Lovell, Timothy; Li, Jin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">401-415</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Four of the most well-known, com. available docking programs, FlexX, GOLD, GLIDE, and ICM, have been examd. for their ligand-docking and virtual-screening capabilities.  The relative performance of the programs in reproducing the native ligand conformation from starting SMILES strings for 164 high-resoln. protein-ligand complexes is presented and compared.  Applying only the native scoring functions, the latest versions of these four docking programs were also used to conduct virtual screening for 12 protein targets of therapeutic interest, involving both publicly available structures and AstraZeneca inhouse structures.  The capability of the four programs to correctly rank-order target-specific active compds. over alternative binders and nonbinders (decoys plus randomly selected compds.) and thereby enrich a small subset of a screening library is compared.  Enrichments from the virtual-screening expts. are contrasted with those obtained with alternative 3D shape-matching and 2D similarity database-search methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcdaTxux-OvLVg90H21EOLACvtfcHk0ljT4TyjZgyIxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlShtLvE&md5=2bc9b31cb2dd5926fa8d5480f4129758</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fci0503255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci0503255%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DH.%26aulast%3DLyne%26aufirst%3DP.%2BD.%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DLovell%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DJ.%26atitle%3DOn%2520evaluating%2520molecular-docking%2520methods%2520for%2520pose%2520prediction%2520and%2520enrichment%2520factors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D401%26epage%3D415" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">McGaughey, G. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheridan, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bayly, C. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Culberson, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kreatsoulas, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindsley, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maiorov, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Truchon, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cornell, W. D.</span><span> </span><span class="NLM_article-title">Comparison of Topological, Shape, and Docking Methods in Virtual Screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1504</span><span class="NLM_x">–</span> <span class="NLM_lpage">1519</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci700052x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmvVOktLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=1504-1519&author=G.+B.+McGaugheyauthor=R.+P.+Sheridanauthor=C.+I.+Baylyauthor=J.+C.+Culbersonauthor=C.+Kreatsoulasauthor=S.+Lindsleyauthor=V.+Maiorovauthor=J.+F.+Truchonauthor=W.+D.+Cornell&title=Comparison+of+Topological%2C+Shape%2C+and+Docking+Methods+in+Virtual+Screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Topological, Shape, and Docking Methods in Virtual Screening</span></div><div class="casAuthors">McGaughey, Georgia B.; Sheridan, Robert P.; Bayly, Christopher I.; Culberson, J. Chris; Kreatsoulas, Constantine; Lindsley, Stacey; Maiorov, Vladimir; Truchon, Jean-Francois; Cornell, Wendy D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1504-1519</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Virtual screening benchmarking studies were carried out on 11 targets to evaluate the performance of three commonly used approaches: 2D ligand similarity (Daylight, TOPOSIM), 3D ligand similarity (SQW, ROCS), and protein structure-based docking (FLOG, FRED, Glide).  Active and decoy compd. sets were assembled from both the MDDR and the Merck compd. databases.  Averaged over multiple targets, ligand-based methods outperformed docking algorithms.  This was true for 3D ligand-based methods only when chem. typing was included.  Using mean enrichment factor as a performance metric, Glide appears to be the best docking method among the three with FRED a close second.  Results for all virtual screening methods are database dependent and can vary greatly for particular targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozxILlzTNAErVg90H21EOLACvtfcHk0liVKh-uB-o8Cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmvVOktLk%253D&md5=d76e6a531097729350e70ffc8d1782f2</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fci700052x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci700052x%26sid%3Dliteratum%253Aachs%26aulast%3DMcGaughey%26aufirst%3DG.%2BB.%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DBayly%26aufirst%3DC.%2BI.%26aulast%3DCulberson%26aufirst%3DJ.%2BC.%26aulast%3DKreatsoulas%26aufirst%3DC.%26aulast%3DLindsley%26aufirst%3DS.%26aulast%3DMaiorov%26aufirst%3DV.%26aulast%3DTruchon%26aufirst%3DJ.%2BF.%26aulast%3DCornell%26aufirst%3DW.%2BD.%26atitle%3DComparison%2520of%2520Topological%252C%2520Shape%252C%2520and%2520Docking%2520Methods%2520in%2520Virtual%2520Screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D1504%26epage%3D1519" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Tan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geppert, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sisay, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gutschow, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Integrating Structure- and Ligand-Based Virtual Screening: Comparison of Individual, Parallel, and Fused Molecular Docking and Similarity Search Calculations on Multiple Targets</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1566</span><span class="NLM_x">–</span> <span class="NLM_lpage">1571</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1002%2Fcmdc.200800129" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=18651695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOgt7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=1566-1571&author=L.+Tanauthor=H.+Geppertauthor=M.+T.+Sisayauthor=M.+Gutschowauthor=J.+Bajorath&title=Integrating+Structure-+and+Ligand-Based+Virtual+Screening%3A+Comparison+of+Individual%2C+Parallel%2C+and+Fused+Molecular+Docking+and+Similarity+Search+Calculations+on+Multiple+Targets"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Integrating structure- and ligand-based virtual screening: comparison of individual, parallel, and fused molecular docking and similarity search calculations on multiple targets</span></div><div class="casAuthors">Tan, Lu; Geppert, Hanna; Sisay, Mihiret T.; Guetschow, Michael; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1566-1571</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Similarity searching is often used to preselect compds. for docking, thereby decreasing the size of screening databases.  However, integrated structure- and ligand-based screening schemes are rare at present.  Docking and similarity search calcns. using 2D fingerprints were carried out in a comparative manner on nine target enzymes, for which significant nos. of diverse inhibitors could be obtained.  In the absence of knowledge-based docking constraints and target-directed parameter optimization, fingerprint searching displayed a clear preference over docking calcns.  Alternative combinations of docking and similarity search results were investigated and found to further increase compd. recall of individual methods in a no. of instances.  When the results of similarity searching and docking were combined, parallel selection of candidate compds. from individual rankings was generally superior to rank fusion.  We suggest that complementary results from docking and similarity searching can be captured by integrated compd. selection schemes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIyWJxL_1eGrVg90H21EOLACvtfcHk0liVKh-uB-o8Cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOgt7rM&md5=fa4690d0059bb144b57a89c5e88be93a</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200800129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200800129%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DL.%26aulast%3DGeppert%26aufirst%3DH.%26aulast%3DSisay%26aufirst%3DM.%2BT.%26aulast%3DGutschow%26aufirst%3DM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DIntegrating%2520Structure-%2520and%2520Ligand-Based%2520Virtual%2520Screening%253A%2520Comparison%2520of%2520Individual%252C%2520Parallel%252C%2520and%2520Fused%2520Molecular%2520Docking%2520and%2520Similarity%2520Search%2520Calculations%2520on%2520Multiple%2520Targets%26jtitle%3DChemMedChem%26date%3D2008%26volume%3D3%26spage%3D1566%26epage%3D1571" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">von Korff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freyss, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sander, T.</span><span> </span><span class="NLM_article-title">Comparison of Ligand- and Structure-Based Virtual Screening on the DUD Data Set</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">209</span><span class="NLM_x">–</span> <span class="NLM_lpage">231</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci800303k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1agtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=209-231&author=M.+von+Korffauthor=J.+Freyssauthor=T.+Sander&title=Comparison+of+Ligand-+and+Structure-Based+Virtual+Screening+on+the+DUD+Data+Set"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Ligand- and Structure-Based Virtual Screening on the DUD Data Set</span></div><div class="casAuthors">von Korff, Modest; Freyss, Joel; Sander, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">209-231</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Several inhouse developed descriptors and our inhouse docking tool ActDock were compared with virtual screening on the data set of useful decoys (DUD).  The results were compared with the chem. fingerprint descriptor from ChemAxon and with the docking results of the original DUD publication.  The DUD is the first published data set providing active mols., decoys, and refs. for crystal structures of ligand-target complexes.  The DUD was designed for the purpose of evaluating docking programs.  It contains 2950 active compds. against a total of 40 target proteins.  Furthermore, for every ligand the data set contains 36 structurally dissimilar decoy compds. with similar physicochem. properties.  We extd. the ligands from the target proteins to extend the applicability of the data set to include ligand based virtual screening.  From the 40 target proteins, 37 contained ligands that we used as query mols. for virtual screening evaluation.  With this data set a large comparison was done between four different chem. fingerprints, a topol. pharmacophore descriptor, the Flexophore descriptor, and ActDock.  The Actelion docking tool relies on a MM2 force field and a pharmacophore point interaction statistic for scoring; the details are described in this publication.  In terms of enrichment rates the chem. fingerprint descriptors performed better than the Flexophore and the docking tool.  After removing mols. chem. similar to the query mols. the Flexophore descriptor outperformed the chem. descriptors and the topol. pharmacophore descriptors.  With the similarity matrix calcns. used in this study it was shown that the Flexophore is well suited to find new chem. entities via "scaffold hopping".  The Flexophore descriptor can be explored with a Java applet at http://www.cheminformatics.ch in the submenu Tools→Flexophore.  Its usage is free of charge and does not require registration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9OO4y7ZdpcbVg90H21EOLACvtfcHk0liVKh-uB-o8Cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1agtLY%253D&md5=8a530b53359d924f00f79d7d92302f8e</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fci800303k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800303k%26sid%3Dliteratum%253Aachs%26aulast%3Dvon%2BKorff%26aufirst%3DM.%26aulast%3DFreyss%26aufirst%3DJ.%26aulast%3DSander%26aufirst%3DT.%26atitle%3DComparison%2520of%2520Ligand-%2520and%2520Structure-Based%2520Virtual%2520Screening%2520on%2520the%2520DUD%2520Data%2520Set%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D209%26epage%3D231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berdini, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartshorn, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mooij, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, P.</span><span> </span><span class="NLM_article-title">Virtual screening using protein−ligand docking: avoiding artificial enrichment</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">793</span><span class="NLM_x">–</span> <span class="NLM_lpage">806</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci034289q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVGhs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2004&pages=793-806&author=M.+L.+Verdonkauthor=V.+Berdiniauthor=M.+J.+Hartshornauthor=W.+T.+Mooijauthor=C.+W.+Murrayauthor=R.+D.+Taylorauthor=P.+Watson&title=Virtual+screening+using+protein%E2%88%92ligand+docking%3A+avoiding+artificial+enrichment"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening using protein-ligand docking: Avoiding artificial enrichment</span></div><div class="casAuthors">Verdonk, Marcel L.; Berdini, Valerio; Hartshorn, Michael J.; Mooij, Wijnand T. M.; Murray, Christopher W.; Taylor, Richard D.; Watson, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">793-806</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This study addresses a no. of topical issues around the use of protein-ligand docking in virtual screening.  We show that, for the validation of such methods, it is key to use focused libraries (contg. compds. with one-dimensional properties, similar to the actives), rather than "random" or "drug-like" libraries to test the actives against.  We also show that, to obtain good enrichments, the docking program needs to produce reliable binding modes.  We demonstrate how pharmacophores can be used to guide the dockings and improve enrichments, and we compare the performance of three consensus-ranking protocols against ranking based on individual scoring functions.  Finally, we show that protein-ligand docking can be an effective aid in the screening for weak, fragment-like binders, which has rapidly become a popular strategy for hit identification.  All results presented are based on carefully constructed virtual screening expts. against four targets, using the protein-ligand docking program GOLD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRRkYiB4qsvbVg90H21EOLACvtfcHk0lg66B3rpqgRwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVGhs70%253D&md5=b0cd9e9a04ddebe401dee9744e2184cb</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fci034289q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci034289q%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DBerdini%26aufirst%3DV.%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26aulast%3DMooij%26aufirst%3DW.%2BT.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DTaylor%26aufirst%3DR.%2BD.%26aulast%3DWatson%26aufirst%3DP.%26atitle%3DVirtual%2520screening%2520using%2520protein%25E2%2588%2592ligand%2520docking%253A%2520avoiding%2520artificial%2520enrichment%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2004%26volume%3D44%26spage%3D793%26epage%3D806" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Marcou, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Optimizing fragment and scaffold docking by use of molecular interaction fingerprints</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">195</span><span class="NLM_x">–</span> <span class="NLM_lpage">207</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci600342e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht12iurzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=195-207&author=G.+Marcouauthor=D.+Rognan&title=Optimizing+fragment+and+scaffold+docking+by+use+of+molecular+interaction+fingerprints"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Optimizing Fragment and Scaffold Docking by Use of Molecular Interaction Fingerprints</span></div><div class="casAuthors">Marcou, Gilles; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">195-207</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein-ligand interaction fingerprints have been used to postprocess docking poses of three ligand data sets: a set of 40 low-mol.-wt. compds. from the Protein Data Bank, a collection of 40 scaffolds from pharmaceutically relevant protein ligands, and a database of 19 scaffolds extd. from true cdk2 inhibitors seeded in 2230 scaffold decoys.  Four popular docking tools (FlexX, Glide, Gold, and Surflex) were used to generate poses for ligands of the three data sets.  In all cases, scoring by the similarity of interaction fingerprints to a given ref. was statistically superior to conventional scoring functions in posing low-mol.-wt. fragments, predicting protein-bound scaffold coordinates according to the known binding mode of related ligands, and screening a scaffold library to enrich a hit list in true cdk2-targeted scaffolds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHLbVKxK-BfLVg90H21EOLACvtfcHk0lg66B3rpqgRwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht12iurzL&md5=0c9137a39d40fbcc83546aec17b595ba</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fci600342e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600342e%26sid%3Dliteratum%253Aachs%26aulast%3DMarcou%26aufirst%3DG.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DOptimizing%2520fragment%2520and%2520scaffold%2520docking%2520by%2520use%2520of%2520molecular%2520interaction%2520fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D195%26epage%3D207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Mpamhanga, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLay, I. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willett, P.</span><span> </span><span class="NLM_article-title">Knowledge-based interaction fingerprint scoring: a simple method for improving the effectiveness of fast scoring functions</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">686</span><span class="NLM_x">–</span> <span class="NLM_lpage">698</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci050420d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD28XpsVGjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=686-698&author=C.+P.+Mpamhangaauthor=B.+Chenauthor=I.+M.+McLayauthor=P.+Willett&title=Knowledge-based+interaction+fingerprint+scoring%3A+a+simple+method+for+improving+the+effectiveness+of+fast+scoring+functions"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Knowledge-Based Interaction Fingerprint Scoring: A Simple Method for Improving the Effectiveness of Fast Scoring Functions</span></div><div class="casAuthors">Mpamhanga, Chidochangu P.; Chen, Beining; McLay, Iain M.; Willett, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">686-698</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new method for the postprocessing of docking outputs has been developed, based on encoding putative 3D binding modes (docking solns.) as ligand-protein interactions into simple bit strings, a method analogous to the structural interaction fingerprint.  Instead of employing traditional scoring functions, the method uses a series of new, knowledge-based scores derived from the similarity of the bit strings for each docking soln. to that of a known ref. binding mode.  A GOLD docking study was carried out using the Bissantz estrogen receptor antagonist set along with the new scoring method.  Superior recovery rates, with up to 2-fold enrichments, were obsd. when the new knowledge-based scoring was compared to the GOLD fitness score.  In addn., top ranking sets of mols. (actives and potential actives or decoys) were structurally diverse with low mol. wts. and structural complexities.  Principal component anal. and clustering of the fingerprints permits the easy sepn. of active from inactive binding modes and the visualization of diverse binding modes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZfTfk88wvHLVg90H21EOLACvtfcHk0lg66B3rpqgRwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpsVGjuw%253D%253D&md5=0fecc7a1894b8bf0e53f077deefc2a85</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fci050420d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci050420d%26sid%3Dliteratum%253Aachs%26aulast%3DMpamhanga%26aufirst%3DC.%2BP.%26aulast%3DChen%26aufirst%3DB.%26aulast%3DMcLay%26aufirst%3DI.%2BM.%26aulast%3DWillett%26aufirst%3DP.%26atitle%3DKnowledge-based%2520interaction%2520fingerprint%2520scoring%253A%2520a%2520simple%2520method%2520for%2520improving%2520the%2520effectiveness%2520of%2520fast%2520scoring%2520functions%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D686%26epage%3D698" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Virtual ligand screening: strategies, perspectives and limitations</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">580</span><span class="NLM_x">–</span> <span class="NLM_lpage">594</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1016%2Fj.drudis.2006.05.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=16793526" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvFGqtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=580-594&author=G.+Klebe&title=Virtual+ligand+screening%3A+strategies%2C+perspectives+and+limitations"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual ligand screening: strategies, perspectives and limitations</span></div><div class="casAuthors">Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">13 & 14</span>),
    <span class="NLM_cas:pages">580-594</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In contrast to high-throughput screening, in virtual ligand screening (VS), compds. are selected using computer programs to predict their binding to a target receptor.  A key prerequisite is knowledge about the spatial and energetic criteria responsible for protein-ligand binding.  The concepts and prerequisites to perform VS are summarized here, and explanations are sought for the enduring limitations of the technol.  Target selection, anal. and prepn. are discussed, as well as considerations about the compilation of candidate ligand libraries.  The tools and strategies of a VS campaign, and the accuracy of scoring and ranking of the results, are also considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN3dWlP-WDY7Vg90H21EOLACvtfcHk0liYF4pvV46EqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvFGqtLo%253D&md5=5adfd125d48082238a6c5ad0e8343c59</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2006.05.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2006.05.012%26sid%3Dliteratum%253Aachs%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DVirtual%2520ligand%2520screening%253A%2520strategies%252C%2520perspectives%2520and%2520limitations%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2006%26volume%3D11%26spage%3D580%26epage%3D594" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Mancera, R. L.</span><span> </span><span class="NLM_article-title">Molecular modeling of hydration in drug design</span> <span class="citation_source-journal">Curr. Opin. Drug Discovery Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">275</span><span class="NLM_x">–</span> <span class="NLM_lpage">280</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=17554853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmtVWlsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2007&pages=275-280&author=R.+L.+Mancera&title=Molecular+modeling+of+hydration+in+drug+design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular modeling of hydration in drug design</span></div><div class="casAuthors">Mancera, Ricardo L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Drug Discovery & Development</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">275-280</span>CODEN:
                <span class="NLM_cas:coden">CODDFF</span>;
        ISSN:<span class="NLM_cas:issn">1367-6733</span>.
    
            (<span class="NLM_cas:orgname">Thomson Scientific</span>)
        </div><div class="casAbstract">A review.  The accurate prediction of the binding mode and affinity of a drug mol. to its putative receptor is crucial for successful drug design and optimization.  Modeling the role of the aq. solvent in ligand-protein interactions is one of the key components.  Three main computational techniques have been developed to model hydration: free energy methods, ligand-protein docking and scoring, and the explicit inclusion of tightly bound water mols. in modeling.  This review provides a summary of the most recent developments in these approaches used for the mol. modeling of hydration effects in the context of computational structure-based drug design methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5oaWakeqXu7Vg90H21EOLACvtfcHk0ljegjrqTJz79Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmtVWlsrg%253D&md5=00f8343ef349b19271d7199dd048d215</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMancera%26aufirst%3DR.%2BL.%26atitle%3DMolecular%2520modeling%2520of%2520hydration%2520in%2520drug%2520design%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Dev.%26date%3D2007%26volume%3D10%26spage%3D275%26epage%3D280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Sousa, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fernandes, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramos, M. J.</span><span> </span><span class="NLM_article-title">Protein-ligand docking: current status and future challenges</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">15</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2006&pages=15-26&author=S.+F.+Sousaauthor=P.+A.+Fernandesauthor=M.+J.+Ramos&title=Protein-ligand+docking%3A+current+status+and+future+challenges"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSousa%26aufirst%3DS.%2BF.%26aulast%3DFernandes%26aufirst%3DP.%2BA.%26aulast%3DRamos%26aufirst%3DM.%2BJ.%26atitle%3DProtein-ligand%2520docking%253A%2520current%2520status%2520and%2520future%2520challenges%26jtitle%3DProteins%26date%3D2006%26volume%3D65%26spage%3D15%26epage%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Babaoglu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Deconstructing fragment-based inhibitor discovery</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">720</span><span class="NLM_x">–</span> <span class="NLM_lpage">723</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1038%2Fnchembio831" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=17072304" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1WqurbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=720-723&author=K.+Babaogluauthor=B.+K.+Shoichet&title=Deconstructing+fragment-based+inhibitor+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Deconstructing fragment-based inhibitor discovery</span></div><div class="casAuthors">Babaoglu, Kerim; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">720-723</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Fragment-based screens test multiple low-mol. wt. mols. for binding to a target.  Fragments often bind with low affinities but typically have better ligand efficiencies (ΔGbind /heavy atom count) than traditional screening hits.  This efficiency, combined with accompanying at.-resoln. structures, has made fragments popular starting points for drug discovery programs.  Fragment-based design adopts a constructive strategy: affinity is enhanced either by cycles of functional-group addn. or by joining two independent fragments together.  The final inhibitor is expected to adopt the same geometry as the original fragment hit.  Here the authors consider whether the inverse, deconstructive logic also applies-can one always parse a higher-affinity inhibitor into fragments that recapitulate the binding geometry of the larger mol..  Cocrystal structures of fragments deconstructed from a known β-lactamase inhibitor suggest that this is not always the case.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVl5_M15h_rLVg90H21EOLACvtfcHk0ljegjrqTJz79Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1WqurbM&md5=de0d5632f6354314794c2f45cfdd2efd</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnchembio831&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio831%26sid%3Dliteratum%253Aachs%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DDeconstructing%2520fragment-based%2520inhibitor%2520discovery%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2006%26volume%3D2%26spage%3D720%26epage%3D723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Hajduk, P. J.</span><span> </span><span class="NLM_article-title">Puzzling through fragment-based drug design</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">–</span> <span class="NLM_lpage">659</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1038%2Fnchembio1206-658" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=17108979" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1SrtLbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=658-659&author=P.+J.+Hajduk&title=Puzzling+through+fragment-based+drug+design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Puzzling through fragment-based drug design</span></div><div class="casAuthors">Hajduk, Philip J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">658-659</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Fragment-based drug design capitalizes on the modular binding of low-mol.-wt., low-affinity ligands.  However, the deconstruction of lead-like inhibitors into putative fragments reveals the surprising complexity of dealing with low-affinity leads, thereby challenging oversimplification of these leads and highlighting the richness of their chem. diversity and mol. recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHhksf7gx20LVg90H21EOLACvtfcHk0ljegjrqTJz79Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1SrtLbI&md5=f509ade062cbb89fa77c6273eb280089</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fnchembio1206-658&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio1206-658%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26atitle%3DPuzzling%2520through%2520fragment-based%2520drug%2520design%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2006%26volume%3D2%26spage%3D658%26epage%3D659" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Brenk, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vetter, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyce, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodin, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Probing molecular docking in a charged model binding site</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">357</span><span class="NLM_x">, </span> <span class="NLM_fpage">1449</span><span class="NLM_x">–</span> <span class="NLM_lpage">1470</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=357&publication_year=2006&pages=1449-1470&author=R.+Brenkauthor=S.+W.+Vetterauthor=S.+E.+Boyceauthor=D.+B.+Goodinauthor=B.+K.+Shoichet&title=Probing+molecular+docking+in+a+charged+model+binding+site"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBrenk%26aufirst%3DR.%26aulast%3DVetter%26aufirst%3DS.%2BW.%26aulast%3DBoyce%26aufirst%3DS.%2BE.%26aulast%3DGoodin%26aufirst%3DD.%2BB.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DProbing%2520molecular%2520docking%2520in%2520a%2520charged%2520model%2520binding%2520site%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D357%26spage%3D1449%26epage%3D1470" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Gerber, P. R.</span><span> </span><span class="NLM_article-title">Charge distribution from a simple molecular orbital type calculation and nonbonding interaction terms in the force field MAB</span> <span class="citation_source-journal">J. Comput.-Aided. Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">37</span><span class="NLM_x">–</span> <span class="NLM_lpage">51</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1998&pages=37-51&author=P.+R.+Gerber&title=Charge+distribution+from+a+simple+molecular+orbital+type+calculation+and+nonbonding+interaction+terms+in+the+force+field+MAB"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGerber%26aufirst%3DP.%2BR.%26atitle%3DCharge%2520distribution%2520from%2520a%2520simple%2520molecular%2520orbital%2520type%2520calculation%2520and%2520nonbonding%2520interaction%2520terms%2520in%2520the%2520force%2520field%2520MAB%26jtitle%3DJ.%2520Comput.-Aided.%2520Mol.%2520Des.%26date%3D1998%26volume%3D12%26spage%3D37%26epage%3D51" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Gerber, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, K.</span><span> </span><span class="NLM_article-title">MAB, a generally applicable molecular force field for structure modelling in medicinal chemistry</span> <span class="citation_source-journal">J. Comput.-Aided. Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">251</span><span class="NLM_x">–</span> <span class="NLM_lpage">268</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1995&pages=251-268&author=P.+R.+Gerberauthor=K.+M%C3%BCller&title=MAB%2C+a+generally+applicable+molecular+force+field+for+structure+modelling+in+medicinal+chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGerber%26aufirst%3DP.%2BR.%26aulast%3DM%25C3%25BCller%26aufirst%3DK.%26atitle%3DMAB%252C%2520a%2520generally%2520applicable%2520molecular%2520force%2520field%2520for%2520structure%2520modelling%2520in%2520medicinal%2520chemistry%26jtitle%3DJ.%2520Comput.-Aided.%2520Mol.%2520Des.%26date%3D1995%26volume%3D9%26spage%3D251%26epage%3D268" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Ertl, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rohde, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Selzer, P.</span><span> </span><span class="NLM_article-title">Fast calculation of molecular polar surface area as a sum of fragment-based contributions and its application to the prediction of drug transport properties</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">3714</span><span class="NLM_x">–</span> <span class="NLM_lpage">3717</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm000942e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmsFeitb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=3714-3717&author=P.+Ertlauthor=B.+Rohdeauthor=P.+Selzer&title=Fast+calculation+of+molecular+polar+surface+area+as+a+sum+of+fragment-based+contributions+and+its+application+to+the+prediction+of+drug+transport+properties"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Fast Calculation of Molecular Polar Surface Area as a Sum of Fragment-Based Contributions and Its Application to the Prediction of Drug Transport Properties</span></div><div class="casAuthors">Ertl, Peter; Rohde, Bernhard; Selzer, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3714-3717</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. polar surface area (PSA), i.e., surface belonging to polar atoms, is a descriptor that was shown to correlate well with passive mol. transport through membranes and, therefore, allows prediction of transport properties of drugs.  The calcn. of PSA, however, is rather time-consuming because of the necessity to generate a reasonable 3D mol. geometry and the calcn. of the surface itself.  A new approach for the calcn. of the PSA is presented here, based on the summation of tabulated surface contributions of polar fragments.  The method, termed topol. PSA (TPSA), provides results which are practically identical with the 3D PSA (the correlation coeff. between 3D PSA and fragment-based TPSA for 34 810 mols. from the World Drug Index is 0.99), while the computation speed is 2-3 orders of magnitude faster.  The new methodol. may, therefore, be used for fast bioavailability screening of virtual libraries having millions of mols.  This article describes the new methodol. and shows the results of validation studies based on sets of published absorption data, including intestinal absorption, Caco-2 monolayer penetration, and blood-brain barrier penetration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhP6Qncx_h6LVg90H21EOLACvtfcHk0lj0vA_SSWHnwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmsFeitb4%253D&md5=23ae4e7708bd8ddeb4a24ad160a29bd6</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm000942e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm000942e%26sid%3Dliteratum%253Aachs%26aulast%3DErtl%26aufirst%3DP.%26aulast%3DRohde%26aufirst%3DB.%26aulast%3DSelzer%26aufirst%3DP.%26atitle%3DFast%2520calculation%2520of%2520molecular%2520polar%2520surface%2520area%2520as%2520a%2520sum%2520of%2520fragment-based%2520contributions%2520and%2520its%2520application%2520to%2520the%2520prediction%2520of%2520drug%2520transport%2520properties%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D3714%26epage%3D3717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Kuzmic, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sideris, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cregar, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elrod, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janc, J. W.</span><span> </span><span class="NLM_article-title">High-throughput screening of enzyme inhibitors: automatic determination of tight-binding inhibition constants</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">281</span><span class="NLM_x">, </span> <span class="NLM_fpage">62</span><span class="NLM_x">–</span> <span class="NLM_lpage">67</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2000&pages=62-67&author=P.+Kuzmicauthor=S.+Siderisauthor=L.+M.+Cregarauthor=K.+C.+Elrodauthor=K.+D.+Riceauthor=J.+W.+Janc&title=High-throughput+screening+of+enzyme+inhibitors%3A+automatic+determination+of+tight-binding+inhibition+constants"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKuzmic%26aufirst%3DP.%26aulast%3DSideris%26aufirst%3DS.%26aulast%3DCregar%26aufirst%3DL.%2BM.%26aulast%3DElrod%26aufirst%3DK.%2BC.%26aulast%3DRice%26aufirst%3DK.%2BD.%26aulast%3DJanc%26aufirst%3DJ.%2BW.%26atitle%3DHigh-throughput%2520screening%2520of%2520enzyme%2520inhibitors%253A%2520automatic%2520determination%2520of%2520tight-binding%2520inhibition%2520constants%26jtitle%3DAnal.%2520Biochem.%26date%3D2000%26volume%3D281%26spage%3D62%26epage%3D67" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Leslie, A. G.</span><span> </span><span class="NLM_article-title">The integration of macromolecular diffraction data</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">48</span><span class="NLM_x">–</span> <span class="NLM_lpage">57</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1107%2FS0907444905039107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=16369093" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlant7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=48-57&author=A.+G.+Leslie&title=The+integration+of+macromolecular+diffraction+data"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">The integration of macromolecular diffraction data</span></div><div class="casAuthors">Leslie, Andrew G. W.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">48-57</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The objective of any modern data-processing program is to produce from a set of diffraction images a set of indexes (hkls) with their assocd. intensities (and ests. of their uncertainties), together with an accurate est. of the crystal unit-cell parameters.  This procedure should not only be reliable, but should involve an abs. min. of user intervention.  The process can be conveniently divided into three stages.  The first (autoindexing) dets. the unit-cell parameters and the orientation of the crystal.  The unit-cell parameters may indicate the likely Laue group of the crystal.  The second step is to refine the initial est. of the unit-cell parameters and also the crystal mosaicity using a procedure known as post-refinement.  The third step is to integrate the images, which consists of predicting the positions of the Bragg reflections on each image and obtaining an est. of the intensity of each reflection and its uncertainty.  This is carried out while simultaneously refining various detector and crystal parameters.  Basic features of the algorithms employed for each of these three sep. steps are described, principally with ref. to the program MOSFLM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoO8D4_R2Ly97Vg90H21EOLACvtfcHk0ljYPkpDSCpUHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlant7vP&md5=70d65687f3b3426c93e96bfb706e0c5c</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1107%2FS0907444905039107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444905039107%26sid%3Dliteratum%253Aachs%26aulast%3DLeslie%26aufirst%3DA.%2BG.%26atitle%3DThe%2520integration%2520of%2520macromolecular%2520diffraction%2520data%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D48%26epage%3D57" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Evans, P.</span><span> </span><span class="NLM_article-title">Scaling and assessment of data quality</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">72</span><span class="NLM_x">–</span> <span class="NLM_lpage">82</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1107%2FS0907444905036693" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=16369096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=72-82&author=P.+Evans&title=Scaling+and+assessment+of+data+quality"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Scaling and assessment of data quality</span></div><div class="casAuthors">Evans, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">72-82</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The various phys. factors affecting measured diffraction intensities are discussed, as are the scaling models which may be used to put the data on a consistent scale.  After scaling, the intensities can be analyzed to set the real resoln. of the data set, to detect bad regions (e.g. bad images), to analyze radiation damage and to assess the overall quality of the data set.  The significance of any anomalous signal may be assessed by probability and correlation anal.  The algorithms used by the CCP4 scaling program SCALA are described.  A requirement for the scaling and merging of intensities is knowledge of the Laue group and point-group symmetries: the possible symmetry of the diffraction pattern may be detd. from scores such as correlation coeffs. between observations which might be symmetry-related.  These scoring functions are implemented in a new program POINTLESS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGravLSLU62RhLVg90H21EOLACvtfcHk0ljYPkpDSCpUHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM&md5=293d3876e534c0c57813990515bb3c76</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1107%2FS0907444905036693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444905036693%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%26atitle%3DScaling%2520and%2520assessment%2520of%2520data%2520quality%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D72%26epage%3D82" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Collaborative Computational Project Number 4</span><span> </span><span class="NLM_article-title">The CCP4 suite: programs for protein crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="NLM_x">–</span> <span class="NLM_lpage">763</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=15299374" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-763&author=Collaborative+Computational+Project+Number+4&title=The+CCP4+suite%3A+programs+for+protein+crystallography"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%2520suite%253A%2520programs%2520for%2520protein%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Otwinowski, Z.; Minor, W.</span><span> </span><span class="NLM_article-title">Processing of X-ray Diffraction Data Collected in Oscillation Mode</span>. In  <span class="citation_source-book">Methods in Enzymology</span>; <span class="NLM_contrib-group">Carter, C. W. J.; Sweet, R. M.</span>, Eds.; <span class="NLM_publisher-name">Academic Press</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">1997</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">276</span>, pp  <span class="NLM_fpage">307</span>− <span class="NLM_lpage">326</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1997&pages=307-326&author=Z.+Otwinowski&author=W.+Minorauthor=C.+W.+J.+Carter&author=R.+M.+Sweet&title=Methods+in+Enzymology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26atitle%3DProcessing%2520of%2520X-ray%2520Diffraction%2520Data%2520Collected%2520in%2520Oscillation%2520Mode%26btitle%3DMethods%2520in%2520Enzymology%26aulast%3DCarter%26aufirst%3DC.%2BW.%2BJ.%26pub%3DAcademic%2520Press%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Vagin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teplyakov, A.</span><span> </span><span class="NLM_article-title">MOLREP: an automated program for molecular replacement</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1022</span><span class="NLM_x">–</span> <span class="NLM_lpage">1025</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1107%2FS0021889897006766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1997&pages=1022-1025&author=A.+Vaginauthor=A.+Teplyakov&title=MOLREP%3A+an+automated+program+for+molecular+replacement"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">MOLREP: an automated program for molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1025</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement which uses effective new approaches in data processing and rotational and translational searching.  These include an automatic choice of all parameters, scaling by Patterson origin peaks and soft resoln. cutoff.  One of the cornerstones of the program is an original full-symmetry translation function combined with a packing function.  Information from the model already placed in the cell is incorporated in both translation and packing functions.  A no. of tests using exptl. data proved the ability of the program to find the correct soln. in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGXdZ-nTKxLVg90H21EOLACvtfcHk0ljYPkpDSCpUHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D&md5=96af64aedd8a8f018c3c18b57a6b4b21</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1107%2FS0021889897006766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889897006766%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMOLREP%253A%2520an%2520automated%2520program%2520for%2520molecular%2520replacement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1997%26volume%3D30%26spage%3D1022%26epage%3D1025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">255</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0liyEu5QrA7eYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isupov, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span> </span><span class="NLM_article-title">Use of TLS parameters to model anisotropic displacements in macromolecular refinement</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">122</span><span class="NLM_x">–</span> <span class="NLM_lpage">133</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1107%2FS0907444900014736" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=11134934" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD3MXkvVyj" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2001&pages=122-133&author=M.+D.+Winnauthor=M.+N.+Isupovauthor=G.+N.+Murshudov&title=Use+of+TLS+parameters+to+model+anisotropic+displacements+in+macromolecular+refinement"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Use of TLS parameters to model anisotropic displacements in macromolecular refinement</span></div><div class="casAuthors">Winn, M. D.; Isupov, M. N.; Murshudov, G. N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">D57</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">122-133</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">An essential step in macromol. refinement is the selection of model parameters which give as good a description of the exptl. data as possible while retaining a realistic data-to-parameter ratio.  This is particularly true of the choice of at. displacement parameters, where the move from individual isotropic to individual anisotropic refinement involves a sixfold increase in the no. of required displacement parameters.  The no. of refinement parameters can be reduced by using collective variables rather than independent at. variables and one of the simplest examples of this is the TLS parameterization for describing the translation, libration and screw-rotation displacements of a pseudo-rigid body.  This article describes the implementation of the TLS parameterization in the macromol. refinement program REFMAC.  Derivs. of the residual with respect to the TLS parameters are expanded in terms of the derivs. with respect to individual anisotropic U values, which in turn are calcd. using a fast Fourier transform technique.  TLS refinement is therefore fast and can be used routinely.  Examples of TLS refinement are given for glyceraldehyde-3-phosphate dehydrogenase (GAPDH) and a transcription activator GerE, for both of which there is data to only 2.0 Å, so that individual anisotropic refinement is not feasible.  GAPDH has been refined with between one and four TLS groups in the asym. unit and GerE with six TLS groups.  In both cases, inclusion of TLS parameters gives improved refinement statistics and in particular an improvement in R and free R values of several percent.  Furthermore, GAPDH and GerE have two and six mols. in the asym. unit, resp., and in each case the displacement parameters differ significantly between mols.  These differences are well accounted for by the TLS parameterization, leaving residual local displacements which are very similar between mols. and to which NCS restraints can be applied.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKZSHF9jjKo7Vg90H21EOLACvtfcHk0liyEu5QrA7eYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXkvVyj&md5=ee69ba99f66fcf18fd7ef49bb8d9b779</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1107%2FS0907444900014736&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444900014736%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DIsupov%26aufirst%3DM.%2BN.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DUse%2520of%2520TLS%2520parameters%2520to%2520model%2520anisotropic%2520displacements%2520in%2520macromolecular%2520refinement%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2001%26volume%3D57%26spage%3D122%26epage%3D133" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Schuttelkopf, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Aalten, D. M. F.</span><span> </span><span class="NLM_article-title">PRODRG: a tool for high-throughput crystallography of protein−ligand complexes</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">1355</span><span class="NLM_x">–</span> <span class="NLM_lpage">1363</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=1355-1363&author=A.+W.+Schuttelkopfauthor=D.+M.+F.+van+Aalten&title=PRODRG%3A+a+tool+for+high-throughput+crystallography+of+protein%E2%88%92ligand+complexes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSchuttelkopf%26aufirst%3DA.%2BW.%26aulast%3Dvan%2BAalten%26aufirst%3DD.%2BM.%2BF.%26atitle%3DPRODRG%253A%2520a%2520tool%2520for%2520high-throughput%2520crystallography%2520of%2520protein%25E2%2588%2592ligand%2520complexes%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D1355%26epage%3D1363" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0liyEu5QrA7eYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Iemura, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hori, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohtaka, H.</span><span> </span><span class="NLM_article-title">Syntheses of the Metabolites of 1-(2-Ethoxyethyl)-2-(Hexahydro-4-Methyl-1<i>h-</i>1,4-Diazepin-1-Yl)-1<i>h</i>-Benzimidazole Difumarate (kg-2413) and Related Compounds</span> <span class="citation_source-journal">Chem. Pharm. Bull.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">962</span><span class="NLM_x">–</span> <span class="NLM_lpage">966</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1989&pages=962-966&author=R.+Iemuraauthor=M.+Horiauthor=H.+Ohtaka&title=Syntheses+of+the+Metabolites+of+1-%282-Ethoxyethyl%29-2-%28Hexahydro-4-Methyl-1h-1%2C4-Diazepin-1-Yl%29-1h-Benzimidazole+Difumarate+%28kg-2413%29+and+Related+Compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DIemura%26aufirst%3DR.%26aulast%3DHori%26aufirst%3DM.%26aulast%3DOhtaka%26aufirst%3DH.%26atitle%3DSyntheses%2520of%2520the%2520Metabolites%2520of%25201-%25282-Ethoxyethyl%2529-2-%2528Hexahydro-4-Methyl-1h-1%252C4-Diazepin-1-Yl%2529-1h-Benzimidazole%2520Difumarate%2520%2528kg-2413%2529%2520and%2520Related%2520Compounds%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D1989%26volume%3D37%26spage%3D962%26epage%3D966" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 68 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Pasquale Linciano, Cecilia Pozzi, Lucia dello Iacono, Flavio di Pisa, Giacomo Landi, Alessio Bonucci, Sheraz Gul, Maria Kuzikov, Bernhard Ellinger, Gesa Witt, Nuno Santarem, Catarina Baptista, Caio Franco, Carolina B. Moraes, Wolfgang Müller, Ulrike Wittig, Rosaria Luciani, Antony Sesenna, Antonio Quotadamo, Stefania Ferrari, Ina Pöhner, Anabela Cordeiro-da-Silva, Stefano Mangani, Luca Costantino, <span class="NLM_string-name hlFld-ContribAuthor">Maria Paola Costi</span>. </span><span class="cited-content_cbyCitation_article-title">Enhancement of Benzothiazoles as Pteridine Reductase-1 Inhibitors for the Treatment of Trypanosomatidic Infections. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (8)
                                     , 3989-4012. <a href="https://doi.org/10.1021/acs.jmedchem.8b02021" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b02021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b02021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b02021%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DEnhancement%252Bof%252BBenzothiazoles%252Bas%252BPteridine%252BReductase-1%252BInhibitors%252Bfor%252Bthe%252BTreatment%252Bof%252BTrypanosomatidic%252BInfections%26aulast%3DLinciano%26aufirst%3DPasquale%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D26122018%26date%3D09042019%26date%3D25032019%26volume%3D62%26issue%3D8%26spage%3D3989%26epage%3D4012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Gydo C.
P. van Zundert, Brandi M. Hudson, Saulo H. P. de Oliveira, Daniel A. Keedy, Rasmus Fonseca, Amelie Heliou, Pooja Suresh, Kenneth Borrelli, Tyler Day, James S. Fraser, <span class="NLM_string-name hlFld-ContribAuthor">Henry van den Bedem</span>. </span><span class="cited-content_cbyCitation_article-title">qFit-ligand Reveals Widespread Conformational Heterogeneity of Drug-Like Molecules in X-Ray Electron Density Maps. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (24)
                                     , 11183-11198. <a href="https://doi.org/10.1021/acs.jmedchem.8b01292" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01292</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01292&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01292%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DqFit-ligand%252BReveals%252BWidespread%252BConformational%252BHeterogeneity%252Bof%252BDrug-Like%252BMolecules%252Bin%252BX-Ray%252BElectron%252BDensity%252BMaps%26aulast%3Dvan%2BZundert%26aufirst%3DGydo%2BC.%2BP.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D15082018%26date%3D06122018%26date%3D20112018%26volume%3D61%26issue%3D24%26spage%3D11183%26epage%3D11198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Lauren A. Webster, Michael Thomas, Michael Urbaniak, Susan Wyllie, Han Ong, Michele Tinti, Alan H. Fairlamb, Markus Boesche, Sonja Ghidelli-Disse, Gerard Drewes, <span class="NLM_string-name hlFld-ContribAuthor">Ian H. Gilbert</span>. </span><span class="cited-content_cbyCitation_article-title">Development of Chemical Proteomics for the Folateome and Analysis of the Kinetoplastid Folateome. </span><span class="cited-content_cbyCitation_journal-name">ACS Infectious Diseases</span><span> <strong>2018,</strong> <em>4 </em>
                                    (10)
                                     , 1475-1486. <a href="https://doi.org/10.1021/acsinfecdis.8b00097" title="DOI URL">https://doi.org/10.1021/acsinfecdis.8b00097</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsinfecdis.8b00097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsinfecdis.8b00097%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Infectious%2520Diseases%26atitle%3DDevelopment%252Bof%252BChemical%252BProteomics%252Bfor%252Bthe%252BFolateome%252Band%252BAnalysis%252Bof%252Bthe%252BKinetoplastid%252BFolateome%26aulast%3DWebster%26aufirst%3DLauren%2BA.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D16042018%26date%3D28092018%26volume%3D4%26issue%3D10%26spage%3D1475%26epage%3D1486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Joshua Pottel, Anat Levit, Magdalena Korczynska, Marcus Fischer, <span class="NLM_string-name hlFld-ContribAuthor">Brian K. Shoichet</span>. </span><span class="cited-content_cbyCitation_article-title">The Recognition of Unrelated Ligands by Identical Proteins. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2018,</strong> <em>13 </em>
                                    (9)
                                     , 2522-2533. <a href="https://doi.org/10.1021/acschembio.8b00443" title="DOI URL">https://doi.org/10.1021/acschembio.8b00443</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.8b00443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.8b00443%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DThe%252BRecognition%252Bof%252BUnrelated%252BLigands%252Bby%252BIdentical%252BProteins%26aulast%3DPottel%26aufirst%3DJoshua%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D10052018%26date%3D10082018%26date%3D27082018%26date%3D10082018%26volume%3D13%26issue%3D9%26spage%3D2522%26epage%3D2533" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John J.  Irwin</span> and <span class="hlFld-ContribAuthor ">Brian K.  Shoichet</span>  . </span><span class="cited-content_cbyCitation_article-title">Docking Screens for Novel Ligands Conferring New Biology. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (9)
                                     , 4103-4120. <a href="https://doi.org/10.1021/acs.jmedchem.5b02008" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b02008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b02008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b02008%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDocking%252BScreens%252Bfor%252BNovel%252BLigands%252BConferring%252BNew%252BBiology%26aulast%3DIrwin%26aufirst%3DJohn%2BJ.%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D25122015%26date%3D15032016%26date%3D12052016%26date%3D25022016%26volume%3D59%26issue%3D9%26spage%3D4103%26epage%3D4120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Albert J.  Kooistra</span>, <span class="hlFld-ContribAuthor ">Rob  Leurs</span>, <span class="hlFld-ContribAuthor ">Iwan J. P.  de Esch</span>, and <span class="hlFld-ContribAuthor ">Chris  de Graaf</span>  . </span><span class="cited-content_cbyCitation_article-title">Structure-Based Prediction of G-Protein-Coupled Receptor Ligand Function: A β-Adrenoceptor Case Study. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2015,</strong> <em>55 </em>
                                    (5)
                                     , 1045-1061. <a href="https://doi.org/10.1021/acs.jcim.5b00066" title="DOI URL">https://doi.org/10.1021/acs.jcim.5b00066</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.5b00066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.5b00066%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DStructure-Based%252BPrediction%252Bof%252BG-Protein-Coupled%252BReceptor%252BLigand%252BFunction%25253A%252BA%252B%2525CE%2525B2-Adrenoceptor%252BCase%252BStudy%26aulast%3DKooistra%26aufirst%3DAlbert%2BJ.%26date%3D2015%26date%3D2015%26date%3D2014%26date%3D20112014%26date%3D01052015%26date%3D26052015%26date%3D07042015%26volume%3D55%26issue%3D5%26spage%3D1045%26epage%3D1061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Advait S.  Nagle</span>, <span class="hlFld-ContribAuthor ">Shilpi  Khare</span>, <span class="hlFld-ContribAuthor ">Arun Babu  Kumar</span>, <span class="hlFld-ContribAuthor ">Frantisek  Supek</span>, <span class="hlFld-ContribAuthor ">Andriy  Buchynskyy</span>, <span class="hlFld-ContribAuthor ">Casey J. N.  Mathison</span>, <span class="hlFld-ContribAuthor ">Naveen Kumar  Chennamaneni</span>, <span class="hlFld-ContribAuthor ">Nagendar  Pendem</span>, <span class="hlFld-ContribAuthor ">Frederick S.  Buckner</span>, <span class="hlFld-ContribAuthor ">Michael H.  Gelb</span>, and <span class="hlFld-ContribAuthor ">Valentina  Molteni</span>  . </span><span class="cited-content_cbyCitation_article-title">Recent Developments in Drug Discovery for Leishmaniasis and Human African Trypanosomiasis. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2014,</strong> <em>114 </em>
                                    (22)
                                     , 11305-11347. <a href="https://doi.org/10.1021/cr500365f" title="DOI URL">https://doi.org/10.1021/cr500365f</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/cr500365f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fcr500365f%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DRecent%252BDevelopments%252Bin%252BDrug%252BDiscovery%252Bfor%252BLeishmaniasis%252Band%252BHuman%252BAfrican%252BTrypanosomiasis%26aulast%3DNagle%26aufirst%3DAdvait%2BS.%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D10072014%26date%3D03112014%26date%3D26112014%26volume%3D114%26issue%3D22%26spage%3D11305%26epage%3D11347" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Abedawn I.  Khalaf</span>, <span class="hlFld-ContribAuthor ">Judith K.  Huggan</span>, <span class="hlFld-ContribAuthor ">Colin J.  Suckling</span>, <span class="hlFld-ContribAuthor ">Colin L.  Gibson</span>, <span class="hlFld-ContribAuthor ">Kirsten  Stewart</span>, <span class="hlFld-ContribAuthor ">Federica  Giordani</span>, <span class="hlFld-ContribAuthor ">Michael P.  Barrett</span>, <span class="hlFld-ContribAuthor ">Pui Ee  Wong</span>, <span class="hlFld-ContribAuthor ">Keri L.  Barrack</span>, and <span class="hlFld-ContribAuthor ">William N.  Hunter</span>  . </span><span class="cited-content_cbyCitation_article-title">Structure-Based Design and Synthesis of Antiparasitic Pyrrolopyrimidines Targeting Pteridine Reductase 1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2014,</strong> <em>57 </em>
                                    (15)
                                     , 6479-6494. <a href="https://doi.org/10.1021/jm500483b" title="DOI URL">https://doi.org/10.1021/jm500483b</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm500483b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm500483b%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252BDesign%252Band%252BSynthesis%252Bof%252BAntiparasitic%252BPyrrolopyrimidines%252BTargeting%252BPteridine%252BReductase%252B1%26aulast%3DKhalaf%26aufirst%3DAbedawn%2BI.%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D27032014%26date%3D29072014%26date%3D14082014%26date%3D09072014%26volume%3D57%26issue%3D15%26spage%3D6479%26epage%3D6494" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kuldeep  Chauhan</span>, <span class="hlFld-ContribAuthor ">Moni  Sharma</span>, <span class="hlFld-ContribAuthor ">Rahul  Shivahare</span>, <span class="hlFld-ContribAuthor ">Utsab  Debnath</span>, <span class="hlFld-ContribAuthor ">Suman  Gupta</span>, <span class="hlFld-ContribAuthor ">Yenamandra S.  Prabhakar</span>, and <span class="hlFld-ContribAuthor ">Prem M. S.  Chauhan</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of Triazine Mimetics As Potent Antileishmanial Agents. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2013,</strong> <em>4 </em>
                                    (11)
                                     , 1108-1113. <a href="https://doi.org/10.1021/ml400317e" title="DOI URL">https://doi.org/10.1021/ml400317e</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ml400317e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fml400317e%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Bof%252BTriazine%252BMimetics%252BAs%252BPotent%252BAntileishmanial%252BAgents%26aulast%3DChauhan%26aufirst%3DKuldeep%26date%3D2013%26date%3D2013%26date%3D2013%26date%3D2013%26date%3D14082013%26date%3D01102013%26date%3D03102013%26date%3D14112013%26date%3D01102013%26volume%3D4%26issue%3D11%26spage%3D1108%26epage%3D1113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ian H.  Gilbert</span>  . </span><span class="cited-content_cbyCitation_article-title">Drug Discovery for Neglected Diseases: Molecular Target-Based and Phenotypic Approaches. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2013,</strong> <em>56 </em>
                                    (20)
                                     , 7719-7726. <a href="https://doi.org/10.1021/jm400362b" title="DOI URL">https://doi.org/10.1021/jm400362b</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm400362b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm400362b%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDrug%252BDiscovery%252Bfor%252BNeglected%252BDiseases%25253A%252BMolecular%252BTarget-Based%252Band%252BPhenotypic%252BApproaches%26aulast%3DGilbert%26aufirst%3DIan%2BH.%26date%3D2013%26date%3D2013%26date%3D2013%26date%3D11032013%26date%3D09092013%26date%3D24102013%26volume%3D56%26issue%3D20%26spage%3D7719%26epage%3D7726" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Robert  Urich</span>, <span class="hlFld-ContribAuthor ">Grant  Wishart</span>, <span class="hlFld-ContribAuthor ">Michael  Kiczun</span>, <span class="hlFld-ContribAuthor ">André  Richters</span>, <span class="hlFld-ContribAuthor ">Naomi  Tidten-Luksch</span>, <span class="hlFld-ContribAuthor ">Daniel  Rauh</span>, <span class="hlFld-ContribAuthor ">Brad  Sherborne</span>, <span class="hlFld-ContribAuthor ">Paul G.  Wyatt</span>, and <span class="hlFld-ContribAuthor ">Ruth  Brenk</span>  . </span><span class="cited-content_cbyCitation_article-title">De Novo Design of Protein Kinase Inhibitors by in Silico Identification of Hinge Region-Binding Fragments. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2013,</strong> <em>8 </em>
                                    (5)
                                     , 1044-1052. <a href="https://doi.org/10.1021/cb300729y" title="DOI URL">https://doi.org/10.1021/cb300729y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/cb300729y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fcb300729y%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DDe%252BNovo%252BDesign%252Bof%252BProtein%252BKinase%252BInhibitors%252Bby%252Bin%252BSilico%252BIdentification%252Bof%252BHinge%252BRegion-Binding%252BFragments%26aulast%3DUrich%26aufirst%3DRobert%26date%3D2013%26date%3D2013%26date%3D2013%26date%3D2012%26date%3D28122012%26date%3D12032013%26date%3D27032013%26date%3D17052013%26volume%3D8%26issue%3D5%26spage%3D1044%26epage%3D1052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stefania  Ferrari</span>, <span class="hlFld-ContribAuthor ">Federica  Morandi</span>, <span class="hlFld-ContribAuthor ">Domantas  Motiejunas</span>, <span class="hlFld-ContribAuthor ">Erika  Nerini</span>, <span class="hlFld-ContribAuthor ">Stefan  Henrich</span>, <span class="hlFld-ContribAuthor ">Rosaria  Luciani</span>, <span class="hlFld-ContribAuthor ">Alberto  Venturelli</span>, <span class="hlFld-ContribAuthor ">Sandra  Lazzari</span>, <span class="hlFld-ContribAuthor ">Samuele  Calò</span>, <span class="hlFld-ContribAuthor ">Shreedhara  Gupta</span>, <span class="hlFld-ContribAuthor ">Veronique  Hannaert</span>, <span class="hlFld-ContribAuthor ">Paul A. M.  Michels</span>, <span class="hlFld-ContribAuthor ">Rebecca C.  Wade</span>, and <span class="hlFld-ContribAuthor ">M. Paola  Costi</span>  . </span><span class="cited-content_cbyCitation_article-title">Virtual Screening Identification of Nonfolate Compounds, Including a CNS Drug, as Antiparasitic Agents Inhibiting Pteridine Reductase. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2011,</strong> <em>54 </em>
                                    (1)
                                     , 211-221. <a href="https://doi.org/10.1021/jm1010572" title="DOI URL">https://doi.org/10.1021/jm1010572</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm1010572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm1010572%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DVirtual%252BScreening%252BIdentification%252Bof%252BNonfolate%252BCompounds%25252C%252BIncluding%252Ba%252BCNS%252BDrug%25252C%252Bas%252BAntiparasitic%252BAgents%252BInhibiting%252BPteridine%252BReductase%26aulast%3DFerrari%26aufirst%3DStefania%26date%3D2011%26date%3D2010%26date%3D2010%26date%3D14082010%26date%3D02122010%26date%3D13012011%26volume%3D54%26issue%3D1%26spage%3D211%26epage%3D221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lindsay B.  Tulloch</span>, <span class="hlFld-ContribAuthor ">Viviane P.  Martini</span>, <span class="hlFld-ContribAuthor ">Jorge  Iulek</span>, <span class="hlFld-ContribAuthor ">Judith K.  Huggan</span>, <span class="hlFld-ContribAuthor ">Jeong Hwan  Lee</span>, <span class="hlFld-ContribAuthor ">Colin L.  Gibson</span>, <span class="hlFld-ContribAuthor ">Terry K.  Smith</span>, <span class="hlFld-ContribAuthor ">Colin J.  Suckling</span> and <span class="hlFld-ContribAuthor ">William N.  Hunter</span>  . </span><span class="cited-content_cbyCitation_article-title">Structure-Based Design of Pteridine Reductase Inhibitors Targeting African Sleeping Sickness and the Leishmaniases. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2010,</strong> <em>53 </em>
                                    (1)
                                     , 221-229. <a href="https://doi.org/10.1021/jm901059x" title="DOI URL">https://doi.org/10.1021/jm901059x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm901059x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm901059x%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Based%252BDesign%252Bof%252BPteridine%252BReductase%252BInhibitors%252BTargeting%252BAfrican%252BSleeping%252BSickness%252Band%252Bthe%252BLeishmaniases%26aulast%3DTulloch%26aufirst%3DLindsay%2BB.%26date%3D2010%26date%3D2009%26date%3D2009%26date%3D21072009%26date%3D16112009%26date%3D14012010%26volume%3D53%26issue%3D1%26spage%3D221%26epage%3D229" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maud  Bollenbach</span>, <span class="hlFld-ContribAuthor ">Simona  Nemska</span>, <span class="hlFld-ContribAuthor ">Patrick  Wagner</span>, <span class="hlFld-ContribAuthor ">Guillaume  Camelin</span>, <span class="hlFld-ContribAuthor ">François  Daubeuf</span>, <span class="hlFld-ContribAuthor ">Adeline  Obrecht</span>, <span class="hlFld-ContribAuthor ">Pascal  Villa</span>, <span class="hlFld-ContribAuthor ">Didier  Rognan</span>, <span class="hlFld-ContribAuthor ">Frédéric  Bihel</span>, <span class="hlFld-ContribAuthor ">Jean-Jacques  Bourguignon</span>, <span class="hlFld-ContribAuthor ">Martine  Schmitt</span>, <span class="hlFld-ContribAuthor ">Nelly  Frossard</span>. </span><span class="cited-content_cbyCitation_article-title">Design, Synthesis and Biological Evaluation of Arylpyridin-2-yl Guanidine Derivatives and Cyclic Mimetics as Novel MSK1 Inhibitors. An Application in an Asthma Model. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (2)
                                     , 391. <a href="https://doi.org/10.3390/molecules26020391" title="DOI URL">https://doi.org/10.3390/molecules26020391</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26020391&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26020391%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DDesign%25252C%252BSynthesis%252Band%252BBiological%252BEvaluation%252Bof%252BArylpyridin-2-yl%252BGuanidine%252BDerivatives%252Band%252BCyclic%252BMimetics%252Bas%252BNovel%252BMSK1%252BInhibitors.%252BAn%252BApplication%252Bin%252Ban%252BAsthma%252BModel%26aulast%3DBollenbach%26aufirst%3DMaud%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D2%26spage%3D391" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Amr H.  Mahmoud</span>, <span class="hlFld-ContribAuthor ">Matthew R.  Masters</span>, <span class="hlFld-ContribAuthor ">Ying  Yang</span>, <span class="hlFld-ContribAuthor ">Markus A.  Lill</span>. </span><span class="cited-content_cbyCitation_article-title">Elucidating the multiple roles of hydration for accurate protein-ligand binding prediction via deep learning. </span><span class="cited-content_cbyCitation_journal-name">Communications Chemistry</span><span> <strong>2020,</strong> <em>3 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s42004-020-0261-x" title="DOI URL">https://doi.org/10.1038/s42004-020-0261-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s42004-020-0261-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs42004-020-0261-x%26sid%3Dliteratum%253Aachs%26jtitle%3DCommunications%2520Chemistry%26atitle%3DElucidating%252Bthe%252Bmultiple%252Broles%252Bof%252Bhydration%252Bfor%252Baccurate%252Bprotein-ligand%252Bbinding%252Bprediction%252Bvia%252Bdeep%252Blearning%26aulast%3DMahmoud%26aufirst%3DAmr%2BH.%26date%3D2020%26date%3D2020%26volume%3D3%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hina  Shamshad</span>, <span class="hlFld-ContribAuthor ">Abdul  Hafiz</span>, <span class="hlFld-ContribAuthor ">Ismail I.  Althagafi</span>, <span class="hlFld-ContribAuthor ">Maria  Saeed</span>, <span class="hlFld-ContribAuthor ">Agha Zeeshan  Mirza</span>. </span><span class="cited-content_cbyCitation_article-title">Characterization of the Trypanosoma brucei Pteridine Reductase Active- Site using Computational Docking and Virtual Screening Techniques. </span><span class="cited-content_cbyCitation_journal-name">Current Computer-Aided Drug Design</span><span> <strong>2020,</strong> <em>16 </em>
                                    (5)
                                     , 583-598. <a href="https://doi.org/10.2174/1573409915666190827163327" title="DOI URL">https://doi.org/10.2174/1573409915666190827163327</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1573409915666190827163327&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1573409915666190827163327%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Computer-Aided%2520Drug%2520Design%26atitle%3DCharacterization%252Bof%252Bthe%252BTrypanosoma%252Bbrucei%252BPteridine%252BReductase%252BActive-%252BSite%252Busing%252BComputational%252BDocking%252Band%252BVirtual%252BScreening%252BTechniques%26aulast%3DShamshad%26aufirst%3DHina%26date%3D2020%26volume%3D16%26issue%3D5%26spage%3D583%26epage%3D598" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Huseyin  Istanbullu</span>, <span class="hlFld-ContribAuthor ">Gulsah  Bayraktar</span>, <span class="hlFld-ContribAuthor ">Hasan  Akbaba</span>, <span class="hlFld-ContribAuthor ">Ibrahim  Cavus</span>, <span class="hlFld-ContribAuthor ">Gunes  Coban</span>, <span class="hlFld-ContribAuthor ">Bilge  Debelec Butuner</span>, <span class="hlFld-ContribAuthor ">Ali Ahmet  Kilimcioglu</span>, <span class="hlFld-ContribAuthor ">Ahmet  Ozbilgin</span>, <span class="hlFld-ContribAuthor ">Vildan  Alptuzun</span>, <span class="hlFld-ContribAuthor ">Ercin  Erciyas</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis, and in vitro biological evaluation of novel thiazolopyrimidine derivatives as antileishmanial compounds. </span><span class="cited-content_cbyCitation_journal-name">Archiv der Pharmazie</span><span> <strong>2020,</strong> <em>353 </em>
                                    (8)
                                     , 1900325. <a href="https://doi.org/10.1002/ardp.201900325" title="DOI URL">https://doi.org/10.1002/ardp.201900325</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ardp.201900325&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fardp.201900325%26sid%3Dliteratum%253Aachs%26jtitle%3DArchiv%2520der%2520Pharmazie%26atitle%3DDesign%25252C%252Bsynthesis%25252C%252Band%252Bin%252Bvitro%252Bbiological%252Bevaluation%252Bof%252Bnovel%252Bthiazolopyrimidine%252Bderivatives%252Bas%252Bantileishmanial%252Bcompounds%26aulast%3DIstanbullu%26aufirst%3DHuseyin%26date%3D2020%26date%3D2020%26volume%3D353%26issue%3D8%26spage%3D1900325" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giacomo  Landi</span>, <span class="hlFld-ContribAuthor ">Pasquale  Linciano</span>, <span class="hlFld-ContribAuthor ">Giusy  Tassone</span>, <span class="hlFld-ContribAuthor ">Maria Paola  Costi</span>, <span class="hlFld-ContribAuthor ">Stefano  Mangani</span>, <span class="hlFld-ContribAuthor ">Cecilia  Pozzi</span>. </span><span class="cited-content_cbyCitation_article-title">High-resolution crystal structure of
              Trypanosoma brucei
              pteridine reductase 1 in complex with an innovative tricyclic-based inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Acta Crystallographica Section D Structural Biology</span><span> <strong>2020,</strong> <em>76 </em>
                                    (6)
                                     , 558-564. <a href="https://doi.org/10.1107/S2059798320004891" title="DOI URL">https://doi.org/10.1107/S2059798320004891</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1107/S2059798320004891&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1107%2FS2059798320004891%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Crystallographica%2520Section%2520D%2520Structural%2520Biology%26atitle%3DHigh-resolution%252Bcrystal%252Bstructure%252Bof%252BTrypanosoma%252Bbrucei%252Bpteridine%252Breductase%252B1%252Bin%252Bcomplex%252Bwith%252Ban%252Binnovative%252Btricyclic-based%252Binhibitor%26aulast%3DLandi%26aufirst%3DGiacomo%26date%3D2020%26date%3D2020%26volume%3D76%26issue%3D6%26spage%3D558%26epage%3D564" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Claudio A.  Pereira</span>, <span class="hlFld-ContribAuthor ">Melisa  Sayé</span>, <span class="hlFld-ContribAuthor ">Chantal  Reigada</span>, <span class="hlFld-ContribAuthor ">Ariel M.  Silber</span>, <span class="hlFld-ContribAuthor ">Guillermo R.  Labadie</span>, <span class="hlFld-ContribAuthor ">Mariana R.  Miranda</span>, <span class="hlFld-ContribAuthor ">Edward  Valera-Vera</span>. </span><span class="cited-content_cbyCitation_article-title">Computational approaches for drug discovery against trypanosomatid-caused diseases. </span><span class="cited-content_cbyCitation_journal-name">Parasitology</span><span> <strong>2020,</strong> <em>147 </em>
                                    (6)
                                     , 611-633. <a href="https://doi.org/10.1017/S0031182020000207" title="DOI URL">https://doi.org/10.1017/S0031182020000207</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1017/S0031182020000207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1017%2FS0031182020000207%26sid%3Dliteratum%253Aachs%26jtitle%3DParasitology%26atitle%3DComputational%252Bapproaches%252Bfor%252Bdrug%252Bdiscovery%252Bagainst%252Btrypanosomatid-caused%252Bdiseases%26aulast%3DPereira%26aufirst%3DClaudio%2BA.%26date%3D2020%26date%3D2020%26volume%3D147%26issue%3D6%26spage%3D611%26epage%3D633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Amit Kumar  Halder</span>, <span class="hlFld-ContribAuthor ">M. Natália  Dias Soeiro Cordeiro</span>. </span><span class="cited-content_cbyCitation_article-title">Advanced in Silico Methods for the Development of Anti- Leishmaniasis and Anti-Trypanosomiasis Agents. </span><span class="cited-content_cbyCitation_journal-name">Current Medicinal Chemistry</span><span> <strong>2020,</strong> <em>27 </em>
                                    (5)
                                     , 697-718. <a href="https://doi.org/10.2174/0929867325666181031093702" title="DOI URL">https://doi.org/10.2174/0929867325666181031093702</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/0929867325666181031093702&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F0929867325666181031093702%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Medicinal%2520Chemistry%26atitle%3DAdvanced%252Bin%252BSilico%252BMethods%252Bfor%252Bthe%252BDevelopment%252Bof%252BAnti-%252BLeishmaniasis%252Band%252BAnti-Trypanosomiasis%252BAgents%26aulast%3DHalder%26aufirst%3DAmit%2BKumar%26date%3D2020%26volume%3D27%26issue%3D5%26spage%3D697%26epage%3D718" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Miaomiao  Liu</span>, <span class="hlFld-ContribAuthor ">Ronald J.  Quinn</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-based screening with natural products for novel anti-parasitic disease drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2019,</strong> <em>14 </em>
                                    (12)
                                     , 1283-1295. <a href="https://doi.org/10.1080/17460441.2019.1653849" title="DOI URL">https://doi.org/10.1080/17460441.2019.1653849</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2019.1653849&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2019.1653849%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DFragment-based%252Bscreening%252Bwith%252Bnatural%252Bproducts%252Bfor%252Bnovel%252Banti-parasitic%252Bdisease%252Bdrug%252Bdiscovery%26aulast%3DLiu%26aufirst%3DMiaomiao%26date%3D2019%26date%3D2019%26volume%3D14%26issue%3D12%26spage%3D1283%26epage%3D1295" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anjali  Sidhu</span>, <span class="hlFld-ContribAuthor ">Sharu  Kukreja</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of novel fluorinated benzothiazol-2-yl-1,2,4-triazoles: Molecular docking, antifungal evaluation and in silico evaluation for SAR. </span><span class="cited-content_cbyCitation_journal-name">Arabian Journal of Chemistry</span><span> <strong>2019,</strong> <em>12 </em>
                                    (8)
                                     , 2118-2127. <a href="https://doi.org/10.1016/j.arabjc.2015.01.009" title="DOI URL">https://doi.org/10.1016/j.arabjc.2015.01.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.arabjc.2015.01.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.arabjc.2015.01.009%26sid%3Dliteratum%253Aachs%26jtitle%3DArabian%2520Journal%2520of%2520Chemistry%26atitle%3DSynthesis%252Bof%252Bnovel%252Bfluorinated%252Bbenzothiazol-2-yl-1%25252C2%25252C4-triazoles%25253A%252BMolecular%252Bdocking%25252C%252Bantifungal%252Bevaluation%252Band%252Bin%252Bsilico%252Bevaluation%252Bfor%252BSAR%26aulast%3DSidhu%26aufirst%3DAnjali%26date%3D2019%26volume%3D12%26issue%3D8%26spage%3D2118%26epage%3D2127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maciej  Majewski</span>, <span class="hlFld-ContribAuthor ">Sergio  Ruiz-Carmona</span>, <span class="hlFld-ContribAuthor ">Xavier  Barril</span>. </span><span class="cited-content_cbyCitation_article-title">An investigation of structural stability in protein-ligand complexes reveals the balance between order and disorder. </span><span class="cited-content_cbyCitation_journal-name">Communications Chemistry</span><span> <strong>2019,</strong> <em>2 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s42004-019-0205-5" title="DOI URL">https://doi.org/10.1038/s42004-019-0205-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s42004-019-0205-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs42004-019-0205-5%26sid%3Dliteratum%253Aachs%26jtitle%3DCommunications%2520Chemistry%26atitle%3DAn%252Binvestigation%252Bof%252Bstructural%252Bstability%252Bin%252Bprotein-ligand%252Bcomplexes%252Breveals%252Bthe%252Bbalance%252Bbetween%252Border%252Band%252Bdisorder%26aulast%3DMajewski%26aufirst%3DMaciej%26date%3D2019%26date%3D2019%26volume%3D2%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">María  Valente</span>, <span class="hlFld-ContribAuthor ">Antonio E.  Vidal</span>, <span class="hlFld-ContribAuthor ">Dolores  González-Pacanowska</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Kinetoplastid and Apicomplexan Thymidylate Biosynthesis as an Antiprotozoal Strategy. </span><span class="cited-content_cbyCitation_journal-name">Current Medicinal Chemistry</span><span> <strong>2019,</strong> <em>26 </em>
                                    (22)
                                     , 4262-4279. <a href="https://doi.org/10.2174/0929867325666180926154329" title="DOI URL">https://doi.org/10.2174/0929867325666180926154329</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/0929867325666180926154329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F0929867325666180926154329%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BKinetoplastid%252Band%252BApicomplexan%252BThymidylate%252BBiosynthesis%252Bas%252Ban%252BAntiprotozoal%252BStrategy%26aulast%3DValente%26aufirst%3DMar%25C3%25ADa%26date%3D2019%26volume%3D26%26issue%3D22%26spage%3D4262%26epage%3D4279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel A.  Erlanson</span>, <span class="hlFld-ContribAuthor ">Ben J.  Davis</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Jahnke</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-Based Drug Discovery: Advancing Fragments in the Absence of Crystal Structures. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2019,</strong> <em>26 </em>
                                    (1)
                                     , 9-15. <a href="https://doi.org/10.1016/j.chembiol.2018.10.001" title="DOI URL">https://doi.org/10.1016/j.chembiol.2018.10.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2018.10.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2018.10.001%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DFragment-Based%252BDrug%252BDiscovery%25253A%252BAdvancing%252BFragments%252Bin%252Bthe%252BAbsence%252Bof%252BCrystal%252BStructures%26aulast%3DErlanson%26aufirst%3DDaniel%2BA.%26date%3D2019%26volume%3D26%26issue%3D1%26spage%3D9%26epage%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Magambo Phillip  Kimuda</span>, <span class="hlFld-ContribAuthor ">Dustin  Laming</span>, <span class="hlFld-ContribAuthor ">Heinrich C.  Hoppe</span>, <span class="hlFld-ContribAuthor ">Özlem  Tastan Bishop</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of Novel Potential Inhibitors of Pteridine Reductase 1 in Trypanosoma brucei via Computational Structure-Based Approaches and in Vitro Inhibition Assays. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2019,</strong> <em>24 </em>
                                    (1)
                                     , 142. <a href="https://doi.org/10.3390/molecules24010142" title="DOI URL">https://doi.org/10.3390/molecules24010142</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules24010142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules24010142%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DIdentification%252Bof%252BNovel%252BPotential%252BInhibitors%252Bof%252BPteridine%252BReductase%252B1%252Bin%252BTrypanosoma%252Bbrucei%252Bvia%252BComputational%252BStructure-Based%252BApproaches%252Band%252Bin%252BVitro%252BInhibition%252BAssays%26aulast%3DKimuda%26aufirst%3DMagambo%2BPhillip%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D1%26spage%3D142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Raphael  Klein</span>, <span class="hlFld-ContribAuthor ">Pasquale  Linciano</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Celenza</span>, <span class="hlFld-ContribAuthor ">Pierangelo  Bellio</span>, <span class="hlFld-ContribAuthor ">Sofia  Papaioannou</span>, <span class="hlFld-ContribAuthor ">Jesus  Blazquez</span>, <span class="hlFld-ContribAuthor ">Laura  Cendron</span>, <span class="hlFld-ContribAuthor ">Ruth  Brenk</span>, <span class="hlFld-ContribAuthor ">Donatella  Tondi</span>, . </span><span class="cited-content_cbyCitation_article-title">In silico identification and experimental validation of hits active against KPC-2 β-lactamase. </span><span class="cited-content_cbyCitation_journal-name">PLOS ONE</span><span> <strong>2018,</strong> <em>13 </em>
                                    (11)
                                     , e0203241. <a href="https://doi.org/10.1371/journal.pone.0203241" title="DOI URL">https://doi.org/10.1371/journal.pone.0203241</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0203241&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0203241%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520ONE%26atitle%3DIn%252Bsilico%252Bidentification%252Band%252Bexperimental%252Bvalidation%252Bof%252Bhits%252Bactive%252Bagainst%252BKPC-2%252B%2525CE%2525B2-lactamase%26aulast%3DKlein%26aufirst%3DRaphael%26date%3D2018%26date%3D2018%26volume%3D13%26issue%3D11%26spage%3De0203241" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gregorio  Cullia</span>, <span class="hlFld-ContribAuthor ">Lucia  Tamborini</span>, <span class="hlFld-ContribAuthor ">Paola  Conti</span>, <span class="hlFld-ContribAuthor ">Carlo  De Micheli</span>, <span class="hlFld-ContribAuthor ">Andrea  Pinto</span>. </span><span class="cited-content_cbyCitation_article-title">Folates in
              Trypanosoma brucei
              : Achievements and Opportunities. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2018,</strong> <em>13 </em>
                                    (20)
                                     , 2150-2158. <a href="https://doi.org/10.1002/cmdc.201800500" title="DOI URL">https://doi.org/10.1002/cmdc.201800500</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201800500&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201800500%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DFolates%252Bin%252BTrypanosoma%252Bbrucei%252B%25253A%252BAchievements%252Band%252BOpportunities%26aulast%3DCullia%26aufirst%3DGregorio%26date%3D2018%26date%3D2018%26volume%3D13%26issue%3D20%26spage%3D2150%26epage%3D2158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Cecilia  Pozzi</span>, <span class="hlFld-ContribAuthor ">Giusy  Tassone</span>, <span class="hlFld-ContribAuthor ">Stefano  Mangani</span>. </span><span class="cited-content_cbyCitation_article-title">X-ray Crystallography Contributions to Drug Discovery Against Parasite. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 175-230. <a href="https://doi.org/10.1016/bs.armc.2018.08.005" title="DOI URL">https://doi.org/10.1016/bs.armc.2018.08.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.armc.2018.08.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.armc.2018.08.005%26sid%3Dliteratum%253Aachs%26atitle%3DX-ray%252BCrystallography%252BContributions%252Bto%252BDrug%252BDiscovery%252BAgainst%252BParasite%26aulast%3DPozzi%26aufirst%3DCecilia%26date%3D2018%26spage%3D175%26epage%3D230%26pub%3DElsevier%26atitle%3DNeglected%252BDiseases%25253A%252BExtensive%252BSpace%252Bfor%252BModern%252BDrug%252BDiscovery%26date%3D2018%26volume%3D51" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Joanna  Panecka-Hofman</span>, <span class="hlFld-ContribAuthor ">Ina  Pöhner</span>, <span class="hlFld-ContribAuthor ">Francesca  Spyrakis</span>, <span class="hlFld-ContribAuthor ">Talia  Zeppelin</span>, <span class="hlFld-ContribAuthor ">Flavio  Di Pisa</span>, <span class="hlFld-ContribAuthor ">Lucia  Dello Iacono</span>, <span class="hlFld-ContribAuthor ">Alessio  Bonucci</span>, <span class="hlFld-ContribAuthor ">Antonio  Quotadamo</span>, <span class="hlFld-ContribAuthor ">Alberto  Venturelli</span>, <span class="hlFld-ContribAuthor ">Stefano  Mangani</span>, <span class="hlFld-ContribAuthor ">Maria Paola  Costi</span>, <span class="hlFld-ContribAuthor ">Rebecca C.  Wade</span>. </span><span class="cited-content_cbyCitation_article-title">Comparative mapping of on-targets and off-targets for the discovery of anti-trypanosomatid folate pathway inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Biochimica et Biophysica Acta (BBA) - General Subjects</span><span> <strong>2017,</strong> <em>1861 </em>
                                    (12)
                                     , 3215-3230. <a href="https://doi.org/10.1016/j.bbagen.2017.09.012" title="DOI URL">https://doi.org/10.1016/j.bbagen.2017.09.012</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbagen.2017.09.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbagen.2017.09.012%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochimica%2520et%2520Biophysica%2520Acta%2520%2528BBA%2529%2520-%2520General%2520Subjects%26atitle%3DComparative%252Bmapping%252Bof%252Bon-targets%252Band%252Boff-targets%252Bfor%252Bthe%252Bdiscovery%252Bof%252Banti-trypanosomatid%252Bfolate%252Bpathway%252Binhibitors%26aulast%3DPanecka-Hofman%26aufirst%3DJoanna%26date%3D2017%26volume%3D1861%26issue%3D12%26spage%3D3215%26epage%3D3230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Juliana da Fonseca Rezende e  Mello</span>, <span class="hlFld-ContribAuthor ">Renan Augusto  Gomes</span>, <span class="hlFld-ContribAuthor ">Drielli Gomes  Vital-Fujii</span>, <span class="hlFld-ContribAuthor ">Glaucio Monteiro  Ferreira</span>, <span class="hlFld-ContribAuthor ">Gustavo Henrique Goulart  Trossini</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-based drug discovery as alternative strategy to the drug development for neglected diseases. </span><span class="cited-content_cbyCitation_journal-name">Chemical Biology & Drug Design</span><span> <strong>2017,</strong> <em>90 </em>
                                    (6)
                                     , 1067-1078. <a href="https://doi.org/10.1111/cbdd.13030" title="DOI URL">https://doi.org/10.1111/cbdd.13030</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/cbdd.13030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fcbdd.13030%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Biology%2520%2526%2520Drug%2520Design%26atitle%3DFragment-based%252Bdrug%252Bdiscovery%252Bas%252Balternative%252Bstrategy%252Bto%252Bthe%252Bdrug%252Bdevelopment%252Bfor%252Bneglected%252Bdiseases%26aulast%3DMello%26aufirst%3DJuliana%2Bda%2BFonseca%2BRezende%2Be%26date%3D2017%26date%3D2017%26volume%3D90%26issue%3D6%26spage%3D1067%26epage%3D1078" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bilal  Zulfiqar</span>, <span class="hlFld-ContribAuthor ">Amy  Jones</span>, <span class="hlFld-ContribAuthor ">Melissa  Sykes</span>, <span class="hlFld-ContribAuthor ">Todd  Shelper</span>, <span class="hlFld-ContribAuthor ">Rohan  Davis</span>, <span class="hlFld-ContribAuthor ">Vicky  Avery</span>. </span><span class="cited-content_cbyCitation_article-title">Screening a Natural Product-Based Library against Kinetoplastid Parasites. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2017,</strong> <em>22 </em>
                                    (10)
                                     , 1715. <a href="https://doi.org/10.3390/molecules22101715" title="DOI URL">https://doi.org/10.3390/molecules22101715</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules22101715&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules22101715%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DScreening%252Ba%252BNatural%252BProduct-Based%252BLibrary%252Bagainst%252BKinetoplastid%252BParasites%26aulast%3DZulfiqar%26aufirst%3DBilal%26date%3D2017%26date%3D2017%26volume%3D22%26issue%3D10%26spage%3D1715" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wiktoria  Jedwabny</span>, <span class="hlFld-ContribAuthor ">Joanna  Panecka-Hofman</span>, <span class="hlFld-ContribAuthor ">Edyta  Dyguda-Kazimierowicz</span>, <span class="hlFld-ContribAuthor ">Rebecca C.  Wade</span>, <span class="hlFld-ContribAuthor ">W. Andrzej  Sokalski</span>. </span><span class="cited-content_cbyCitation_article-title">Application of a simple quantum chemical approach to ligand fragment scoring for Trypanosoma brucei pteridine reductase 1 inhibition. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2017,</strong> <em>31 </em>
                                    (8)
                                     , 715-728. <a href="https://doi.org/10.1007/s10822-017-0035-4" title="DOI URL">https://doi.org/10.1007/s10822-017-0035-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-017-0035-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-017-0035-4%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DApplication%252Bof%252Ba%252Bsimple%252Bquantum%252Bchemical%252Bapproach%252Bto%252Bligand%252Bfragment%252Bscoring%252Bfor%252BTrypanosoma%252Bbrucei%252Bpteridine%252Breductase%2525C2%2525A01%252Binhibition%26aulast%3DJedwabny%26aufirst%3DWiktoria%26date%3D2017%26date%3D2017%26volume%3D31%26issue%3D8%26spage%3D715%26epage%3D728" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Leah M.  Wachsmuth</span>, <span class="hlFld-ContribAuthor ">Meredith G.  Johnson</span>, <span class="hlFld-ContribAuthor ">Jason  Gavenonis</span>, . </span><span class="cited-content_cbyCitation_article-title">Essential multimeric enzymes in kinetoplastid parasites: A host of potentially druggable protein-protein interactions. </span><span class="cited-content_cbyCitation_journal-name">PLOS Neglected Tropical Diseases</span><span> <strong>2017,</strong> <em>11 </em>
                                    (6)
                                     , e0005720. <a href="https://doi.org/10.1371/journal.pntd.0005720" title="DOI URL">https://doi.org/10.1371/journal.pntd.0005720</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pntd.0005720&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pntd.0005720%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520Neglected%2520Tropical%2520Diseases%26atitle%3DEssential%252Bmultimeric%252Benzymes%252Bin%252Bkinetoplastid%252Bparasites%25253A%252BA%252Bhost%252Bof%252Bpotentially%252Bdruggable%252Bprotein-protein%252Binteractions%26aulast%3DWachsmuth%26aufirst%3DLeah%2BM.%26date%3D2017%26date%3D2017%26volume%3D11%26issue%3D6%26spage%3De0005720" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Álvaro  Martín-Montes</span>, <span class="hlFld-ContribAuthor ">Daniel  Plano</span>, <span class="hlFld-ContribAuthor ">Rubén  Martín-Escolano</span>, <span class="hlFld-ContribAuthor ">Verónica  Alcolea</span>, <span class="hlFld-ContribAuthor ">Marta  Díaz</span>, <span class="hlFld-ContribAuthor ">Silvia  Pérez-Silanes</span>, <span class="hlFld-ContribAuthor ">Socorro  Espuelas</span>, <span class="hlFld-ContribAuthor ">Esther  Moreno</span>, <span class="hlFld-ContribAuthor ">Clotilde  Marín</span>, <span class="hlFld-ContribAuthor ">Ramón  Gutiérrez-Sánchez</span>, <span class="hlFld-ContribAuthor ">Carmen  Sanmartín</span>, <span class="hlFld-ContribAuthor ">Manuel  Sánchez-Moreno</span>. </span><span class="cited-content_cbyCitation_article-title">Library of Seleno-Compounds as Novel Agents against Leishmania Species. </span><span class="cited-content_cbyCitation_journal-name">Antimicrobial Agents and Chemotherapy</span><span> <strong>2017,</strong> <em>61 </em>
                                    (6)
                                     <a href="https://doi.org/10.1128/AAC.02546-16" title="DOI URL">https://doi.org/10.1128/AAC.02546-16</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1128/AAC.02546-16&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1128%2FAAC.02546-16%26sid%3Dliteratum%253Aachs%26jtitle%3DAntimicrobial%2520Agents%2520and%2520Chemotherapy%26atitle%3DLibrary%252Bof%252BSeleno-Compounds%252Bas%252BNovel%252BAgents%252Bagainst%252BLeishmania%252BSpecies%26aulast%3DMart%25C3%25ADn-Montes%26aufirst%3D%25C3%2581lvaro%26date%3D2017%26volume%3D61%26issue%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Flavio  Di Pisa</span>, <span class="hlFld-ContribAuthor ">Giacomo  Landi</span>, <span class="hlFld-ContribAuthor ">Lucia  Dello Iacono</span>, <span class="hlFld-ContribAuthor ">Cecilia  Pozzi</span>, <span class="hlFld-ContribAuthor ">Chiara  Borsari</span>, <span class="hlFld-ContribAuthor ">Stefania  Ferrari</span>, <span class="hlFld-ContribAuthor ">Matteo  Santucci</span>, <span class="hlFld-ContribAuthor ">Nuno  Santarem</span>, <span class="hlFld-ContribAuthor ">Anabela  Cordeiro-da-Silva</span>, <span class="hlFld-ContribAuthor ">Carolina  Moraes</span>, <span class="hlFld-ContribAuthor ">Laura  Alcantara</span>, <span class="hlFld-ContribAuthor ">Vanessa  Fontana</span>, <span class="hlFld-ContribAuthor ">Lucio  Freitas-Junior</span>, <span class="hlFld-ContribAuthor ">Sheraz  Gul</span>, <span class="hlFld-ContribAuthor ">Maria  Kuzikov</span>, <span class="hlFld-ContribAuthor ">Birte  Behrens</span>, <span class="hlFld-ContribAuthor ">Ina  Pöhner</span>, <span class="hlFld-ContribAuthor ">Rebecca  Wade</span>, <span class="hlFld-ContribAuthor ">Maria  Costi</span>, <span class="hlFld-ContribAuthor ">Stefano  Mangani</span>. </span><span class="cited-content_cbyCitation_article-title">Chroman-4-One Derivatives Targeting Pteridine Reductase 1 and Showing Anti-Parasitic Activity. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2017,</strong> <em>22 </em>
                                    (3)
                                     , 426. <a href="https://doi.org/10.3390/molecules22030426" title="DOI URL">https://doi.org/10.3390/molecules22030426</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules22030426&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules22030426%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DChroman-4-One%252BDerivatives%252BTargeting%252BPteridine%252BReductase%252B1%252Band%252BShowing%252BAnti-Parasitic%252BActivity%26aulast%3DDi%2BPisa%26aufirst%3DFlavio%26date%3D2017%26date%3D2017%26volume%3D22%26issue%3D3%26spage%3D426" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ben J.  Davis</span>, <span class="hlFld-ContribAuthor ">Stephen D.  Roughley</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-Based Lead Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 371-439. <a href="https://doi.org/10.1016/bs.armc.2017.07.002" title="DOI URL">https://doi.org/10.1016/bs.armc.2017.07.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.armc.2017.07.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.armc.2017.07.002%26sid%3Dliteratum%253Aachs%26atitle%3DFragment-Based%252BLead%252BDiscovery%26aulast%3DDavis%26aufirst%3DBen%2BJ.%26date%3D2017%26spage%3D371%26epage%3D439%26pub%3DElsevier%26atitle%3DPlatform%252BTechnologies%252Bin%252BDrug%252BDiscovery%252Band%252BValidation%26date%3D2017%26volume%3D50" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sharu  Kukreja</span>, <span class="hlFld-ContribAuthor ">Anjali  Sidhu</span>, <span class="hlFld-ContribAuthor ">Vineet K.  Sharma</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of novel 7-fluoro-3-substituted-1,2,4-triazolo[3,4-b]benzothiazoles (FTBs) as potent antifungal agents: molecular docking and in silico evaluation. </span><span class="cited-content_cbyCitation_journal-name">Research on Chemical Intermediates</span><span> <strong>2016,</strong> <em>42 </em>
                                    (12)
                                     , 8329-8344. <a href="https://doi.org/10.1007/s11164-016-2599-3" title="DOI URL">https://doi.org/10.1007/s11164-016-2599-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11164-016-2599-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11164-016-2599-3%26sid%3Dliteratum%253Aachs%26jtitle%3DResearch%2520on%2520Chemical%2520Intermediates%26atitle%3DSynthesis%252Bof%252Bnovel%252B7-fluoro-3-substituted-1%25252C2%25252C4-triazolo%25255B3%25252C4-b%25255Dbenzothiazoles%252B%252528FTBs%252529%252Bas%252Bpotent%252Bantifungal%252Bagents%25253A%252Bmolecular%252Bdocking%252Band%252Bin%252Bsilico%252Bevaluation%26aulast%3DKukreja%26aufirst%3DSharu%26date%3D2016%26date%3D2016%26volume%3D42%26issue%3D12%26spage%3D8329%26epage%3D8344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ifedayo  Ogungbe</span>, <span class="hlFld-ContribAuthor ">William  Setzer</span>. </span><span class="cited-content_cbyCitation_article-title">The Potential of Secondary Metabolites from Plants as Drugs or Leads against Protozoan Neglected Diseases—Part III: In-Silico Molecular Docking Investigations. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2016,</strong> <em>21 </em>
                                    (10)
                                     , 1389. <a href="https://doi.org/10.3390/molecules21101389" title="DOI URL">https://doi.org/10.3390/molecules21101389</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules21101389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules21101389%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DThe%252BPotential%252Bof%252BSecondary%252BMetabolites%252Bfrom%252BPlants%252Bas%252BDrugs%252Bor%252BLeads%252Bagainst%252BProtozoan%252BNeglected%252BDiseases%2525E2%252580%252594Part%252BIII%25253A%252BIn-Silico%252BMolecular%252BDocking%252BInvestigations%26aulast%3DOgungbe%26aufirst%3DIfedayo%26date%3D2016%26date%3D2016%26volume%3D21%26issue%3D10%26spage%3D1389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">María  Valente</span>, <span class="hlFld-ContribAuthor ">Antonio E.  Vidal</span>, <span class="hlFld-ContribAuthor ">Dolores  González Pacanowska</span>. </span><span class="cited-content_cbyCitation_article-title">Potential of Pyrimidine Metabolism for Antitrypanosomal Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2016,</strong>,, 147-170. <a href="https://doi.org/10.1002/9783527694082.ch6" title="DOI URL">https://doi.org/10.1002/9783527694082.ch6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527694082.ch6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527694082.ch6%26sid%3Dliteratum%253Aachs%26atitle%3DPotential%252Bof%252BPyrimidine%252BMetabolism%252Bfor%252BAntitrypanosomal%252BDrug%252BDiscovery%26aulast%3DValente%26aufirst%3DMar%25C3%25ADa%26date%3D2016%26date%3D2016%26spage%3D147%26epage%3D170%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26atitle%3DComprehensive%252BAnalysis%252Bof%252BParasite%252BBiology%25253A%252BFrom%252BMetabolism%252Bto%252BDrug%252BDiscovery%26aulast%3DM%25C3%25BCller%26aufirst%3DSylke%26date%3D2016%26volume%3D418" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Douglas R.  Houston</span>, <span class="hlFld-ContribAuthor ">Li-Hsuan  Yen</span>, <span class="hlFld-ContribAuthor ">Simon  Pettit</span>, <span class="hlFld-ContribAuthor ">Malcolm D.  Walkinshaw</span>, . </span><span class="cited-content_cbyCitation_article-title">Structure- and Ligand-Based Virtual Screening Identifies New Scaffolds for Inhibitors of the Oncoprotein MDM2. </span><span class="cited-content_cbyCitation_journal-name">PLOS ONE</span><span> <strong>2015,</strong> <em>10 </em>
                                    (4)
                                     , e0121424. <a href="https://doi.org/10.1371/journal.pone.0121424" title="DOI URL">https://doi.org/10.1371/journal.pone.0121424</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0121424&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0121424%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520ONE%26atitle%3DStructure-%252Band%252BLigand-Based%252BVirtual%252BScreening%252BIdentifies%252BNew%252BScaffolds%252Bfor%252BInhibitors%252Bof%252Bthe%252BOncoprotein%252BMDM2%26aulast%3DHouston%26aufirst%3DDouglas%2BR.%26date%3D2015%26date%3D2015%26volume%3D10%26issue%3D4%26spage%3De0121424" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maren  Kuhnert</span>, <span class="hlFld-ContribAuthor ">Helene  Köster</span>, <span class="hlFld-ContribAuthor ">Ruben  Bartholomäus</span>, <span class="hlFld-ContribAuthor ">Ah Young  Park</span>, <span class="hlFld-ContribAuthor ">Amir  Shahim</span>, <span class="hlFld-ContribAuthor ">Andreas  Heine</span>, <span class="hlFld-ContribAuthor ">Holger  Steuber</span>, <span class="hlFld-ContribAuthor ">Gerhard  Klebe</span>, <span class="hlFld-ContribAuthor ">Wibke E.  Diederich</span>. </span><span class="cited-content_cbyCitation_article-title">Chamäleon-artige Bindungsmodi in der Leitstrukturoptimierung: wechselnde Bindungsgeometrien bei Aspartylprotease-Inhibitoren. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2015,</strong> <em>127 </em>
                                    (9)
                                     , 2891-2896. <a href="https://doi.org/10.1002/ange.201411206" title="DOI URL">https://doi.org/10.1002/ange.201411206</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.201411206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.201411206%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DCham%2525C3%2525A4leon-artige%252BBindungsmodi%252Bin%252Bder%252BLeitstrukturoptimierung%25253A%252Bwechselnde%252BBindungsgeometrien%252Bbei%252BAspartylprotease-Inhibitoren%26aulast%3DKuhnert%26aufirst%3DMaren%26date%3D2015%26date%3D2015%26volume%3D127%26issue%3D9%26spage%3D2891%26epage%3D2896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maren  Kuhnert</span>, <span class="hlFld-ContribAuthor ">Helene  Köster</span>, <span class="hlFld-ContribAuthor ">Ruben  Bartholomäus</span>, <span class="hlFld-ContribAuthor ">Ah Young  Park</span>, <span class="hlFld-ContribAuthor ">Amir  Shahim</span>, <span class="hlFld-ContribAuthor ">Andreas  Heine</span>, <span class="hlFld-ContribAuthor ">Holger  Steuber</span>, <span class="hlFld-ContribAuthor ">Gerhard  Klebe</span>, <span class="hlFld-ContribAuthor ">Wibke E.  Diederich</span>. </span><span class="cited-content_cbyCitation_article-title">Tracing Binding Modes in Hit-to-Lead Optimization: Chameleon-Like Poses of Aspartic Protease Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2015,</strong> <em>54 </em>
                                    (9)
                                     , 2849-2853. <a href="https://doi.org/10.1002/anie.201411206" title="DOI URL">https://doi.org/10.1002/anie.201411206</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.201411206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.201411206%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DTracing%252BBinding%252BModes%252Bin%252BHit-to-Lead%252BOptimization%25253A%252BChameleon-Like%252BPoses%252Bof%252BAspartic%252BProtease%252BInhibitors%26aulast%3DKuhnert%26aufirst%3DMaren%26date%3D2015%26date%3D2015%26volume%3D54%26issue%3D9%26spage%3D2849%26epage%3D2853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Divya  Dube</span>, <span class="hlFld-ContribAuthor ">Sujata  Sharma</span>, <span class="hlFld-ContribAuthor ">Tej P.  Singh</span>, <span class="hlFld-ContribAuthor ">Punit  Kaur</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmacophore Mapping, In Silico Screening and Molecular Docking to Identify Selective
              Trypanosoma brucei
              Pteridine Reductase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Molecular Informatics</span><span> <strong>2014,</strong> <em>33 </em>
                                    (2)
                                     , 124-134. <a href="https://doi.org/10.1002/minf.201300023" title="DOI URL">https://doi.org/10.1002/minf.201300023</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/minf.201300023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fminf.201300023%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Informatics%26atitle%3DPharmacophore%252BMapping%25252C%252BIn%252BSilico%252BScreening%252Band%252BMolecular%252BDocking%252Bto%252BIdentify%252BSelective%252BTrypanosoma%252Bbrucei%252BPteridine%252BReductase%252BInhibitors%26aulast%3DDube%26aufirst%3DDivya%26date%3D2014%26date%3D2014%26volume%3D33%26issue%3D2%26spage%3D124%26epage%3D134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Craig  Wilkinson</span>, <span class="hlFld-ContribAuthor ">Martin J.  McPhillie</span>, <span class="hlFld-ContribAuthor ">Ying  Zhou</span>, <span class="hlFld-ContribAuthor ">Stuart  Woods</span>, <span class="hlFld-ContribAuthor ">Gustavo A.  Afanador</span>, <span class="hlFld-ContribAuthor ">Shaun  Rawson</span>, <span class="hlFld-ContribAuthor ">Farzana  Khaliq</span>, <span class="hlFld-ContribAuthor ">Sean T.  Prigge</span>, <span class="hlFld-ContribAuthor ">Craig W.  Roberts</span>, <span class="hlFld-ContribAuthor ">David W.  Rice</span>, <span class="hlFld-ContribAuthor ">Rima  McLeod</span>, <span class="hlFld-ContribAuthor ">Colin W.  Fishwick</span>, <span class="hlFld-ContribAuthor ">Stephen P.  Muench</span>. </span><span class="cited-content_cbyCitation_article-title">The benzimidazole based drugs show good activity against T. gondii but poor activity against its proposed enoyl reductase enzyme target. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2014,</strong> <em>24 </em>
                                    (3)
                                     , 911-916. <a href="https://doi.org/10.1016/j.bmcl.2013.12.066" title="DOI URL">https://doi.org/10.1016/j.bmcl.2013.12.066</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2013.12.066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2013.12.066%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DThe%252Bbenzimidazole%252Bbased%252Bdrugs%252Bshow%252Bgood%252Bactivity%252Bagainst%252BT.%252Bgondii%252Bbut%252Bpoor%252Bactivity%252Bagainst%252Bits%252Bproposed%252Benoyl%252Breductase%252Benzyme%252Btarget%26aulast%3DWilkinson%26aufirst%3DCraig%26date%3D2014%26volume%3D24%26issue%3D3%26spage%3D911%26epage%3D916" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Despina  Smirlis</span>, <span class="hlFld-ContribAuthor ">Milena Botelho Pereira  Soares</span>. </span><span class="cited-content_cbyCitation_article-title">Selection of Molecular Targets for Drug Development Against Trypanosomatids. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2014,</strong>,, 43-76. <a href="https://doi.org/10.1007/978-94-007-7305-9_2" title="DOI URL">https://doi.org/10.1007/978-94-007-7305-9_2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-94-007-7305-9_2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-94-007-7305-9_2%26sid%3Dliteratum%253Aachs%26atitle%3DSelection%252Bof%252BMolecular%252BTargets%252Bfor%252BDrug%252BDevelopment%252BAgainst%252BTrypanosomatids%26aulast%3DSmirlis%26aufirst%3DDespina%26date%3D2014%26date%3D2013%26spage%3D43%26epage%3D76%26pub%3DSpringer%2520Netherlands%26atitle%3DProteins%252Band%252BProteomics%252Bof%252BLeishmania%252Band%252BTrypanosoma%26aulast%3DSantos%26aufirst%3DAndr%25C3%25A9%2BL.S.%26date%3D2014%26volume%3D74" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ryan G.  Coleman</span>, <span class="hlFld-ContribAuthor ">Michael  Carchia</span>, <span class="hlFld-ContribAuthor ">Teague  Sterling</span>, <span class="hlFld-ContribAuthor ">John J.  Irwin</span>, <span class="hlFld-ContribAuthor ">Brian K.  Shoichet</span>, . </span><span class="cited-content_cbyCitation_article-title">Ligand Pose and Orientational Sampling in Molecular Docking. </span><span class="cited-content_cbyCitation_journal-name">PLoS ONE</span><span> <strong>2013,</strong> <em>8 </em>
                                    (10)
                                     , e75992. <a href="https://doi.org/10.1371/journal.pone.0075992" title="DOI URL">https://doi.org/10.1371/journal.pone.0075992</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0075992&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0075992%26sid%3Dliteratum%253Aachs%26jtitle%3DPLoS%2520ONE%26atitle%3DLigand%252BPose%252Band%252BOrientational%252BSampling%252Bin%252BMolecular%252BDocking%26aulast%3DColeman%26aufirst%3DRyan%2BG.%26date%3D2013%26date%3D2013%26volume%3D8%26issue%3D10%26spage%3De75992" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stefania  Ferrari</span>, <span class="hlFld-ContribAuthor ">Valeria  Losasso</span>, <span class="hlFld-ContribAuthor ">Puneet  Saxena</span>, <span class="hlFld-ContribAuthor ">Maria Paola  Costi</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting the Trypanosomatidic Enzymes Pteridine Reductase and Dihydrofolate Reductase. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2013,</strong>,, 445-472. <a href="https://doi.org/10.1002/9783527670383.ch24" title="DOI URL">https://doi.org/10.1002/9783527670383.ch24</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527670383.ch24&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527670383.ch24%26sid%3Dliteratum%253Aachs%26atitle%3DTargeting%252Bthe%252BTrypanosomatidic%252BEnzymes%252BPteridine%252BReductase%252Band%252BDihydrofolate%252BReductase%26aulast%3DFerrari%26aufirst%3DStefania%26date%3D2013%26date%3D2013%26spage%3D445%26epage%3D472%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26atitle%3DTrypanosomatid%252BDiseases%26aulast%3DJ%25C3%25A4ger%26aufirst%3DTimo%26date%3D2013%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mattia  Mori</span>, <span class="hlFld-ContribAuthor ">Maurizio  Botta</span>. </span><span class="cited-content_cbyCitation_article-title">Drug Design and Screening by
              In Silico
              Approaches. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2013,</strong>,, 57-79. <a href="https://doi.org/10.1002/9783527670383.ch4" title="DOI URL">https://doi.org/10.1002/9783527670383.ch4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527670383.ch4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527670383.ch4%26sid%3Dliteratum%253Aachs%26atitle%3DDrug%252BDesign%252Band%252BScreening%252Bby%252BIn%252BSilico%252BApproaches%26aulast%3DMori%26aufirst%3DMattia%26date%3D2013%26date%3D2013%26spage%3D57%26epage%3D79%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26atitle%3DTrypanosomatid%252BDiseases%26aulast%3DJ%25C3%25A4ger%26aufirst%3DTimo%26date%3D2013%26volume%3D329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dagmar  Fajkusova</span>, <span class="hlFld-ContribAuthor ">Matus  Pesko</span>, <span class="hlFld-ContribAuthor ">Stanislava  Keltosova</span>, <span class="hlFld-ContribAuthor ">Jiahui  Guo</span>, <span class="hlFld-ContribAuthor ">Zbynek  Oktabec</span>, <span class="hlFld-ContribAuthor ">Marcela  Vejsova</span>, <span class="hlFld-ContribAuthor ">Peter  Kollar</span>, <span class="hlFld-ContribAuthor ">Aidan  Coffey</span>, <span class="hlFld-ContribAuthor ">Jozef  Csollei</span>, <span class="hlFld-ContribAuthor ">Katarina  Kralova</span>, <span class="hlFld-ContribAuthor ">Josef  Jampilek</span>. </span><span class="cited-content_cbyCitation_article-title">Anti-infective and herbicidal activity of N-substituted 2-aminobenzothiazoles. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2012,</strong> <em>20 </em>
                                    (24)
                                     , 7059-7068. <a href="https://doi.org/10.1016/j.bmc.2012.10.007" title="DOI URL">https://doi.org/10.1016/j.bmc.2012.10.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2012.10.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2012.10.007%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DAnti-infective%252Band%252Bherbicidal%252Bactivity%252Bof%252BN-substituted%252B2-aminobenzothiazoles%26aulast%3DFajkusova%26aufirst%3DDagmar%26date%3D2012%26volume%3D20%26issue%3D24%26spage%3D7059%26epage%3D7068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Seiichi  Uchiyama</span>, <span class="hlFld-ContribAuthor ">Kohki  Kimura</span>, <span class="hlFld-ContribAuthor ">Chie  Gota</span>, <span class="hlFld-ContribAuthor ">Kohki  Okabe</span>, <span class="hlFld-ContribAuthor ">Kyoko  Kawamoto</span>, <span class="hlFld-ContribAuthor ">Noriko  Inada</span>, <span class="hlFld-ContribAuthor ">Toshitada  Yoshihara</span>, <span class="hlFld-ContribAuthor ">Seiji  Tobita</span>. </span><span class="cited-content_cbyCitation_article-title">Environment-Sensitive Fluorophores with Benzothiadiazole and Benzoselenadiazole Structures as Candidate Components of a Fluorescent Polymeric Thermometer. </span><span class="cited-content_cbyCitation_journal-name">Chemistry - A European Journal</span><span> <strong>2012,</strong> <em>18 </em>
                                    (31)
                                     , 9552-9563. <a href="https://doi.org/10.1002/chem.201200597" title="DOI URL">https://doi.org/10.1002/chem.201200597</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/chem.201200597&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fchem.201200597%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520-%2520A%2520European%2520Journal%26atitle%3DEnvironment-Sensitive%252BFluorophores%252Bwith%252BBenzothiadiazole%252Band%252BBenzoselenadiazole%252BStructures%252Bas%252BCandidate%252BComponents%252Bof%252Ba%252BFluorescent%252BPolymeric%252BThermometer%26aulast%3DUchiyama%26aufirst%3DSeiichi%26date%3D2012%26date%3D2012%26volume%3D18%26issue%3D31%26spage%3D9552%26epage%3D9563" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Divya  Dube</span>, <span class="hlFld-ContribAuthor ">Vinita  Periwal</span>, <span class="hlFld-ContribAuthor ">Mukesh  Kumar</span>, <span class="hlFld-ContribAuthor ">Sujata  Sharma</span>, <span class="hlFld-ContribAuthor ">Tej P.  Singh</span>, <span class="hlFld-ContribAuthor ">Punit  Kaur</span>. </span><span class="cited-content_cbyCitation_article-title">3D-QSAR based pharmacophore modeling and virtual screening for identification of novel pteridine reductase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Modeling</span><span> <strong>2012,</strong> <em>18 </em>
                                    (5)
                                     , 1701-1711. <a href="https://doi.org/10.1007/s00894-011-1187-0" title="DOI URL">https://doi.org/10.1007/s00894-011-1187-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00894-011-1187-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00894-011-1187-0%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Modeling%26atitle%3D3D-QSAR%252Bbased%252Bpharmacophore%252Bmodeling%252Band%252Bvirtual%252Bscreening%252Bfor%252Bidentification%252Bof%252Bnovel%252Bpteridine%252Breductase%252Binhibitors%26aulast%3DDube%26aufirst%3DDivya%26date%3D2012%26date%3D2011%26volume%3D18%26issue%3D5%26spage%3D1701%26epage%3D1711" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Naomi  Tidten-Luksch</span>, <span class="hlFld-ContribAuthor ">Raffaella  Grimaldi</span>, <span class="hlFld-ContribAuthor ">Leah S.  Torrie</span>, <span class="hlFld-ContribAuthor ">Julie A.  Frearson</span>, <span class="hlFld-ContribAuthor ">William N.  Hunter</span>, <span class="hlFld-ContribAuthor ">Ruth  Brenk</span>, . </span><span class="cited-content_cbyCitation_article-title">IspE Inhibitors Identified by a Combination of In Silico and In Vitro High-Throughput Screening. </span><span class="cited-content_cbyCitation_journal-name">PLoS ONE</span><span> <strong>2012,</strong> <em>7 </em>
                                    (4)
                                     , e35792. <a href="https://doi.org/10.1371/journal.pone.0035792" title="DOI URL">https://doi.org/10.1371/journal.pone.0035792</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0035792&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0035792%26sid%3Dliteratum%253Aachs%26jtitle%3DPLoS%2520ONE%26atitle%3DIspE%252BInhibitors%252BIdentified%252Bby%252Ba%252BCombination%252Bof%252BIn%252BSilico%252Band%252BIn%252BVitro%252BHigh-Throughput%252BScreening%26aulast%3DTidten-Luksch%26aufirst%3DNaomi%26date%3D2012%26date%3D2012%26volume%3D7%26issue%3D4%26spage%3De35792" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vivian  Cody</span>, <span class="hlFld-ContribAuthor ">Jim  Pace</span>, <span class="hlFld-ContribAuthor ">Jennifer  Piraino</span>, <span class="hlFld-ContribAuthor ">Sherry F.  Queener</span>. </span><span class="cited-content_cbyCitation_article-title">Crystallographic analysis reveals a novel second binding site for trimethoprim in active site double mutants of human dihydrofolate reductase. </span><span class="cited-content_cbyCitation_journal-name">Journal of Structural Biology</span><span> <strong>2011,</strong> <em>176 </em>
                                    (1)
                                     , 52-59. <a href="https://doi.org/10.1016/j.jsb.2011.06.001" title="DOI URL">https://doi.org/10.1016/j.jsb.2011.06.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jsb.2011.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jsb.2011.06.001%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Structural%2520Biology%26atitle%3DCrystallographic%252Banalysis%252Breveals%252Ba%252Bnovel%252Bsecond%252Bbinding%252Bsite%252Bfor%252Btrimethoprim%252Bin%252Bactive%252Bsite%252Bdouble%252Bmutants%252Bof%252Bhuman%252Bdihydrofolate%252Breductase%26aulast%3DCody%26aufirst%3DVivian%26date%3D2011%26volume%3D176%26issue%3D1%26spage%3D52%26epage%3D59" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anelia Ts.  Mavrova</span>, <span class="hlFld-ContribAuthor ">Diana  Wesselinova</span>, <span class="hlFld-ContribAuthor ">Nikolay  Vassilev</span>, <span class="hlFld-ContribAuthor ">Jordan A.  Tsenov</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis, characterization and cytotoxicity of some novel 1,3-disubstituted-2,3-dihydro-2-iminobenzimidazoles. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2011,</strong> <em>46 </em>
                                    (8)
                                     , 3362-3367. <a href="https://doi.org/10.1016/j.ejmech.2011.04.061" title="DOI URL">https://doi.org/10.1016/j.ejmech.2011.04.061</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2011.04.061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2011.04.061%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSynthesis%25252C%252Bcharacterization%252Band%252Bcytotoxicity%252Bof%252Bsome%252Bnovel%252B1%25252C3-disubstituted-2%25252C3-dihydro-2-iminobenzimidazoles%26aulast%3DMavrova%26aufirst%3DAnelia%2BTs.%26date%3D2011%26volume%3D46%26issue%3D8%26spage%3D3362%26epage%3D3367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rafaela S  Ferreira</span>, <span class="hlFld-ContribAuthor ">Rafael VC  Guido</span>, <span class="hlFld-ContribAuthor ">Adriano D  Andricopulo</span>, <span class="hlFld-ContribAuthor ">Glaucius  Oliva</span>. </span><span class="cited-content_cbyCitation_article-title">In silico
              screening strategies for novel inhibitors of parasitic diseases. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2011,</strong> <em>6 </em>
                                    (5)
                                     , 481-489. <a href="https://doi.org/10.1517/17460441.2011.563297" title="DOI URL">https://doi.org/10.1517/17460441.2011.563297</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1517/17460441.2011.563297&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1517%2F17460441.2011.563297%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DIn%252Bsilico%252Bscreening%252Bstrategies%252Bfor%252Bnovel%252Binhibitors%252Bof%252Bparasitic%252Bdiseases%26aulast%3DFerreira%26aufirst%3DRafaela%2BS%26date%3D2011%26date%3D2011%26volume%3D6%26issue%3D5%26spage%3D481%26epage%3D489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Peter  Daldrop</span>, <span class="hlFld-ContribAuthor ">Francis E.  Reyes</span>, <span class="hlFld-ContribAuthor ">David A.  Robinson</span>, <span class="hlFld-ContribAuthor ">Colin M.  Hammond</span>, <span class="hlFld-ContribAuthor ">David M.  Lilley</span>, <span class="hlFld-ContribAuthor ">Robert T.  Batey</span>, <span class="hlFld-ContribAuthor ">Ruth  Brenk</span>. </span><span class="cited-content_cbyCitation_article-title">Novel Ligands for a Purine Riboswitch Discovered by RNA-Ligand Docking. </span><span class="cited-content_cbyCitation_journal-name">Chemistry & Biology</span><span> <strong>2011,</strong> <em>18 </em>
                                    (3)
                                     , 324-335. <a href="https://doi.org/10.1016/j.chembiol.2010.12.020" title="DOI URL">https://doi.org/10.1016/j.chembiol.2010.12.020</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2010.12.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2010.12.020%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520%2526%2520Biology%26atitle%3DNovel%252BLigands%252Bfor%252Ba%252BPurine%252BRiboswitch%252BDiscovered%252Bby%252BRNA-Ligand%252BDocking%26aulast%3DDaldrop%26aufirst%3DPeter%26date%3D2011%26volume%3D18%26issue%3D3%26spage%3D324%26epage%3D335" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel  Spinks</span>, <span class="hlFld-ContribAuthor ">Han B.  Ong</span>, <span class="hlFld-ContribAuthor ">Chidochangu P.  Mpamhanga</span>, <span class="hlFld-ContribAuthor ">Emma J.  Shanks</span>, <span class="hlFld-ContribAuthor ">David A.  Robinson</span>, <span class="hlFld-ContribAuthor ">Iain T.  Collie</span>, <span class="hlFld-ContribAuthor ">Kevin D.  Read</span>, <span class="hlFld-ContribAuthor ">Julie A.  Frearson</span>, <span class="hlFld-ContribAuthor ">Paul G.  Wyatt</span>, <span class="hlFld-ContribAuthor ">Ruth  Brenk</span>, <span class="hlFld-ContribAuthor ">Alan H.  Fairlamb</span>, <span class="hlFld-ContribAuthor ">Ian H.  Gilbert</span>. </span><span class="cited-content_cbyCitation_article-title">Design, Synthesis and Biological Evaluation of Novel Inhibitors of
              Trypanosoma brucei
              Pteridine Reductase 1. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2011,</strong> <em>6 </em>
                                    (2)
                                     , 302-308. <a href="https://doi.org/10.1002/cmdc.201000450" title="DOI URL">https://doi.org/10.1002/cmdc.201000450</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201000450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201000450%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DDesign%25252C%252BSynthesis%252Band%252BBiological%252BEvaluation%252Bof%252BNovel%252BInhibitors%252Bof%252BTrypanosoma%252Bbrucei%252BPteridine%252BReductase%2525E2%252580%2525851%26aulast%3DSpinks%26aufirst%3DDaniel%26date%3D2011%26date%3D2010%26volume%3D6%26issue%3D2%26spage%3D302%26epage%3D308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel A.  Erlanson</span>. </span><span class="cited-content_cbyCitation_article-title">Introduction to Fragment-Based Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2011,</strong>,, 1-32. <a href="https://doi.org/10.1007/128_2011_180" title="DOI URL">https://doi.org/10.1007/128_2011_180</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/128_2011_180&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F128_2011_180%26sid%3Dliteratum%253Aachs%26atitle%3DIntroduction%252Bto%252BFragment-Based%252BDrug%252BDiscovery%26aulast%3DErlanson%26aufirst%3DDaniel%2BA.%26date%3D2011%26date%3D2011%26spage%3D1%26epage%3D32%26pub%3DSpringer%2520Berlin%2520Heidelberg%26atitle%3DFragment-Based%252BDrug%252BDiscovery%252Band%252BX-Ray%252BCrystallography%26aulast%3DDavies%26aufirst%3DThomas%2BG.%26date%3D2012%26volume%3D317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Darren W.  Begley</span>, <span class="hlFld-ContribAuthor ">Douglas R.  Davies</span>, <span class="hlFld-ContribAuthor ">Robert  C. Hartley</span>, <span class="hlFld-ContribAuthor ">Thomas E.  Edwards</span>, <span class="hlFld-ContribAuthor ">Bart L.  Staker</span>, <span class="hlFld-ContribAuthor ">Wesley C.  Van Voorhis</span>, <span class="hlFld-ContribAuthor ">Peter J.  Myler</span>, <span class="hlFld-ContribAuthor ">Lance J.  Stewart</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment Screening of Infectious Disease Targets in a Structural Genomics Environment. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2011,</strong>,, 533-556. <a href="https://doi.org/10.1016/B978-0-12-381274-2.00021-2" title="DOI URL">https://doi.org/10.1016/B978-0-12-381274-2.00021-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-381274-2.00021-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-381274-2.00021-2%26sid%3Dliteratum%253Aachs%26atitle%3DFragment%252BScreening%252Bof%252BInfectious%252BDisease%252BTargets%252Bin%252Ba%252BStructural%252BGenomics%252BEnvironment%26aulast%3DBegley%26aufirst%3DDarren%2BW.%26date%3D2011%26spage%3D533%26epage%3D556%26pub%3DElsevier%26atitle%3DFragment-Based%252BDrug%252BDesign%252B-%252BTools%25252C%252BPractical%252BApproaches%25252C%252Band%252BExamples%26date%3D2011%26volume%3D493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Obdulia  Rabal</span>, <span class="hlFld-ContribAuthor ">Manuel  Urbano-Cuadrado</span>, <span class="hlFld-ContribAuthor ">Julen  Oyarzabal</span>. </span><span class="cited-content_cbyCitation_article-title">Computational medicinal chemistry in fragment-based drug discovery: what, how and when. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2011,</strong> <em>3 </em>
                                    (1)
                                     , 95-134. <a href="https://doi.org/10.4155/fmc.10.277" title="DOI URL">https://doi.org/10.4155/fmc.10.277</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc.10.277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc.10.277%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DComputational%252Bmedicinal%252Bchemistry%252Bin%252Bfragment-based%252Bdrug%252Bdiscovery%25253A%252Bwhat%25252C%252Bhow%252Band%252Bwhen%26aulast%3DRabal%26aufirst%3DObdulia%26date%3D2011%26volume%3D3%26issue%3D1%26spage%3D95%26epage%3D134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christopher W  Murray</span>, <span class="hlFld-ContribAuthor ">Tom L  Blundell</span>. </span><span class="cited-content_cbyCitation_article-title">Structural biology in fragment-based drug design. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Structural Biology</span><span> <strong>2010,</strong> <em>20 </em>
                                    (4)
                                     , 497-507. <a href="https://doi.org/10.1016/j.sbi.2010.04.003" title="DOI URL">https://doi.org/10.1016/j.sbi.2010.04.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.sbi.2010.04.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.sbi.2010.04.003%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Structural%2520Biology%26atitle%3DStructural%252Bbiology%252Bin%252Bfragment-based%252Bdrug%252Bdesign%26aulast%3DMurray%26aufirst%3DChristopher%2BW%26date%3D2010%26volume%3D20%26issue%3D4%26spage%3D497%26epage%3D507" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Natasha  Sienkiewicz</span>, <span class="hlFld-ContribAuthor ">Han B.  Ong</span>, <span class="hlFld-ContribAuthor ">Alan H.  Fairlamb</span>. </span><span class="cited-content_cbyCitation_article-title">Trypanosoma brucei pteridine reductase 1 is essential for survival in vitro and for virulence in mice. </span><span class="cited-content_cbyCitation_journal-name">Molecular Microbiology</span><span> <strong>2010,</strong> <em>77 </em>
                                    (3)
                                     , 658-671. <a href="https://doi.org/10.1111/j.1365-2958.2010.07236.x" title="DOI URL">https://doi.org/10.1111/j.1365-2958.2010.07236.x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/j.1365-2958.2010.07236.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fj.1365-2958.2010.07236.x%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Microbiology%26atitle%3DTrypanosoma%252Bbrucei%252Bpteridine%252Breductase%252B1%252Bis%252Bessential%252Bfor%252Bsurvival%252Bin%252Bvitro%252Band%252Bfor%252Bvirulence%252Bin%252Bmice%26aulast%3DSienkiewicz%26aufirst%3DNatasha%26date%3D2010%26date%3D2010%26volume%3D77%26issue%3D3%26spage%3D658%26epage%3D671" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gian Filippo  Ruda</span>, <span class="hlFld-ContribAuthor ">Gordon  Campbell</span>, <span class="hlFld-ContribAuthor ">Vincent P.  Alibu</span>, <span class="hlFld-ContribAuthor ">Michael P.  Barrett</span>, <span class="hlFld-ContribAuthor ">Ruth  Brenk</span>, <span class="hlFld-ContribAuthor ">Ian H.  Gilbert</span>. </span><span class="cited-content_cbyCitation_article-title">Virtual fragment screening for novel inhibitors of 6-phosphogluconate dehydrogenase. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2010,</strong> <em>18 </em>
                                    (14)
                                     , 5056-5062. <a href="https://doi.org/10.1016/j.bmc.2010.05.077" title="DOI URL">https://doi.org/10.1016/j.bmc.2010.05.077</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2010.05.077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2010.05.077%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DVirtual%252Bfragment%252Bscreening%252Bfor%252Bnovel%252Binhibitors%252Bof%252B6-phosphogluconate%252Bdehydrogenase%26aulast%3DRuda%26aufirst%3DGian%2BFilippo%26date%3D2010%26volume%3D18%26issue%3D14%26spage%3D5056%26epage%3D5062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anthony G  Coyne</span>, <span class="hlFld-ContribAuthor ">Duncan E  Scott</span>, <span class="hlFld-ContribAuthor ">Chris  Abell</span>. </span><span class="cited-content_cbyCitation_article-title">Drugging challenging targets using fragment-based approaches. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2010,</strong> <em>14 </em>
                                    (3)
                                     , 299-307. <a href="https://doi.org/10.1016/j.cbpa.2010.02.010" title="DOI URL">https://doi.org/10.1016/j.cbpa.2010.02.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2010.02.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2010.02.010%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DDrugging%252Bchallenging%252Btargets%252Busing%252Bfragment-based%252Bapproaches%26aulast%3DCoyne%26aufirst%3DAnthony%2BG%26date%3D2010%26volume%3D14%26issue%3D3%26spage%3D299%26epage%3D307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Emma J.  Shanks</span>, <span class="hlFld-ContribAuthor ">Han B.  Ong</span>, <span class="hlFld-ContribAuthor ">David A.  Robinson</span>, <span class="hlFld-ContribAuthor ">Stephen  Thompson</span>, <span class="hlFld-ContribAuthor ">Natasha  Sienkiewicz</span>, <span class="hlFld-ContribAuthor ">Alan H.  Fairlamb</span>, <span class="hlFld-ContribAuthor ">Julie A.  Frearson</span>. </span><span class="cited-content_cbyCitation_article-title">Development and validation of a cytochrome c-coupled assay for pteridine reductase 1 and dihydrofolate reductase. </span><span class="cited-content_cbyCitation_journal-name">Analytical Biochemistry</span><span> <strong>2010,</strong> <em>396 </em>
                                    (2)
                                     , 194-203. <a href="https://doi.org/10.1016/j.ab.2009.09.003" title="DOI URL">https://doi.org/10.1016/j.ab.2009.09.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ab.2009.09.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ab.2009.09.003%26sid%3Dliteratum%253Aachs%26jtitle%3DAnalytical%2520Biochemistry%26atitle%3DDevelopment%252Band%252Bvalidation%252Bof%252Ba%252Bcytochrome%252Bc-coupled%252Bassay%252Bfor%252Bpteridine%252Breductase%252B1%252Band%252Bdihydrofolate%252Breductase%26aulast%3DShanks%26aufirst%3DEmma%2BJ.%26date%3D2010%26volume%3D396%26issue%3D2%26spage%3D194%26epage%3D203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yu  Chen</span>, <span class="hlFld-ContribAuthor ">Denise Teotico  Pohlhaus</span>. </span><span class="cited-content_cbyCitation_article-title">In silico docking and scoring of fragments. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today: Technologies</span><span> <strong>2010,</strong> <em>7 </em>
                                    (3)
                                     , e149-e156. <a href="https://doi.org/10.1016/j.ddtec.2010.11.002" title="DOI URL">https://doi.org/10.1016/j.ddtec.2010.11.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ddtec.2010.11.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ddtec.2010.11.002%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technologies%26atitle%3DIn%252Bsilico%252Bdocking%252Band%252Bscoring%252Bof%252Bfragments%26aulast%3DChen%26aufirst%3DYu%26date%3D2010%26volume%3D7%26issue%3D3%26spage%3De149%26epage%3De156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Susan M.  Boyd</span>, <span class="hlFld-ContribAuthor ">Gerdien E.  de Kloe</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment library design: efficiently hunting drugs in chemical space. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today: Technologies</span><span> <strong>2010,</strong> <em>7 </em>
                                    (3)
                                     , e173-e180. <a href="https://doi.org/10.1016/j.ddtec.2010.11.010" title="DOI URL">https://doi.org/10.1016/j.ddtec.2010.11.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ddtec.2010.11.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ddtec.2010.11.010%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technologies%26atitle%3DFragment%252Blibrary%252Bdesign%25253A%252Befficiently%252Bhunting%252Bdrugs%252Bin%252Bchemical%252Bspace%26aulast%3DBoyd%26aufirst%3DSusan%2BM.%26date%3D2010%26volume%3D7%26issue%3D3%26spage%3De173%26epage%3De180" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/medium/jm-2009-00414x_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm900414x&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/medium/jm-2009-00414x_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Common core structures of PTR1 and DHFR inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm900414x&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/medium/jm-2009-00414x_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Ligands and binding site residues of <i>Tb</i>PTR1-methotrexate complex (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2C7V">2C7V</a>, blue carbon atoms) superimposed with those from the triamterene complex (green carbon atoms).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm900414x&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/medium/jm-2009-00414x_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Modeled binding mode of <b>2</b>. The ligands is sandwiched between the aromatic moiety of Phe97 and the nicotinamide part of NADP<sup>+</sup>. Further, the ligand forms hydrogen bonds with Ser95 and the cofactor.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm900414x&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/medium/jm-2009-00414x_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0005.jpeg" id="rightTab-_i9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (a) View in the binding site of PTR1·<b>4</b> together with <i>F</i><sub>o</sub> − <i>F</i><sub>c</sub> omit map (contoured on 2.0σ), which was calculated by omitting the ligand from the final model. The ligand adopts two distinct binding modes. In the major confirmation (green carbon atoms), the chloro-substituent packs against Leu209 and Pro210, whereas in the minor conformation (yellow carbon atoms), the chloro atom sits in the open cavity of the active site. (b) Modeled binding mode of <b>4</b> (orange carbon atoms) superimposed with the dominant binding mode determined crystallographically (green carbon atoms).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm900414x&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/medium/jm-2009-00414x_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Crystallographically determined binding mode of <b>5</b> together with <i>F</i><sub>o</sub> − <i>F</i><sub>c</sub> omit map (contoured on 2.0σ), which was calculated by omitting the ligand from the final model. Two water molecules mediate contacts between the phosphate groups of the cofactor and the ligand.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm900414x&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/medium/jm-2009-00414x_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Crystallographically determined binding mode of <b>9</b> (orange carbon atoms for ligand and blue carbon atoms for protein) superimposed with the receptor conformation used for docking calculations (gray carbon atoms) together with <i>F</i><sub>o</sub> − <i>F</i><sub>c</sub> omit map (contoured on 2.0σ), which was calculated by omitting the ligand from the final model. To accommodate the ligand in the binding site, the side chain of Trp221 rotated by 10° and the sulfur atom of Cys168 shifted by 2 Å.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm900414x&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/medium/jm-2009-00414x_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Superposition of the crystallographically determined binding modes of <b>4</b> (orange carbon atoms), <b>5</b> (green carbon atoms), and <b>9</b> (gray carbon atoms). Depending on the substituents, the amino-benzimidazole core adopts three different binding modes.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm900414x&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/medium/jm-2009-00414x_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Cut-away view of solvent accessible surface of the binding pocket of PTR1·<b>9</b>. Two hydrophobic pockets filled with ordered water molecules are close the aminobenzimidazole core of <b>9</b>, a smaller pocket bordered by Pro167, Tyr174, and Asn175, and a larger one toward Trp221.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm900414x&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/medium/jm-2009-00414x_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Crystallographically determined binding mode of <b>12</b> together with <i>F</i><sub>o</sub> − <i>F</i><sub>c</sub> omit map (contoured on 2.0σ), which was calculated by omitting the ligand from the final model. The phenyl group of the ligand forms an edge−face interaction with Trp221.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-14/jm900414x/production/images/large/jm-2009-00414x_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm900414x&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i40">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24518" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24518" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 50 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Pink, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hudson, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mouries, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bendig, M.</span><span> </span><span class="NLM_article-title">Opportunities and challenges in antiparasitic drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug. Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">727</span><span class="NLM_x">–</span> <span class="NLM_lpage">740</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2005&pages=727-740&author=R.+Pinkauthor=A.+Hudsonauthor=M.+A.+Mouriesauthor=M.+Bendig&title=Opportunities+and+challenges+in+antiparasitic+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPink%26aufirst%3DR.%26aulast%3DHudson%26aufirst%3DA.%26aulast%3DMouries%26aufirst%3DM.%2BA.%26aulast%3DBendig%26aufirst%3DM.%26atitle%3DOpportunities%2520and%2520challenges%2520in%2520antiparasitic%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug.%2520Discovery%26date%3D2005%26volume%3D4%26spage%3D727%26epage%3D740" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Brun, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balmer, O.</span><span> </span><span class="NLM_article-title">New developments in human African trypanosomiasis</span> <span class="citation_source-journal">Curr. Opin. Infect. Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">415</span><span class="NLM_x">–</span> <span class="NLM_lpage">420</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1097%2F01.qco.0000244045.93016.b1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=16940863" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A280%3ADC%252BD28rhs1CntQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2006&pages=415-420&author=R.+Brunauthor=O.+Balmer&title=New+developments+in+human+African+trypanosomiasis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">New developments in human African trypanosomiasis</span></div><div class="casAuthors">Brun Reto; Balmer Oliver</div><div class="citationInfo"><span class="NLM_cas:title">Current opinion in infectious diseases</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">415-20</span>
        ISSN:<span class="NLM_cas:issn">0951-7375</span>.
    </div><div class="casAbstract">PURPOSE OF REVIEW:  To review recent literature on human African trypanosomiasis, focussing on genome sequencing, diagnosis and drug discovery, and typing of trypanosomes.  RECENT FINDINGS:  The most important recent development has been the completion of the Trypanosoma brucei genome which will greatly facilitate the discovery of new drug targets and genetic markers.  Correct staging of the disease is of key importance for treatment.  The analysis of sleep patterns is a promising new method to this end and has advanced enough to begin thorough clinical trials.  In terms of novel drug candidates, dicationic molecules show the most promise with one oral diamidine in phase 3 clinical trials.  New targets and classes of molecules which show in vitro trypanocidal activity are also described.  Two new methods - MGE-PCR and microsatellites - allow analyses without parasite cultivation, eliminating a major impediment to efficient sampling for population studies.  The finding that several wild animal species harbour T. b. gambiense, and that parasite transmission is efficient even from very low parasitaemias, sheds a new light on the importance of animal reservoirs.  SUMMARY:  The use of T. brucei as model system for molecular and cell biology is regularly producing new technologies exploitable for diagnosis and new drugs.  Drug discovery and development experience a revival through new public-private partnerships and initiatives.  The challenge remains to translate this progress into improvements for affected people in disease endemic areas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTXtcSNPwREGHthwrmtH4OBfW6udTcc2eaPK9r8oUA5vLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD28rhs1CntQ%253D%253D&md5=7d9c553ecb478b197fab2dd572d814b2</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1097%2F01.qco.0000244045.93016.b1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252F01.qco.0000244045.93016.b1%26sid%3Dliteratum%253Aachs%26aulast%3DBrun%26aufirst%3DR.%26aulast%3DBalmer%26aufirst%3DO.%26atitle%3DNew%2520developments%2520in%2520human%2520African%2520trypanosomiasis%26jtitle%3DCurr.%2520Opin.%2520Infect.%2520Dis.%26date%3D2006%26volume%3D19%26spage%3D415%26epage%3D420" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Kennedy, P. G.</span><span> </span><span class="NLM_article-title">The continuing problem of human African trypanosomiasis (sleeping sickness)</span> <span class="citation_source-journal">Ann. Neurol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">116</span><span class="NLM_x">–</span> <span class="NLM_lpage">126</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1002%2Fana.21429" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=18756506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A280%3ADC%252BD1crnsFKrsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2008&pages=116-126&author=P.+G.+Kennedy&title=The+continuing+problem+of+human+African+trypanosomiasis+%28sleeping+sickness%29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The continuing problem of human African trypanosomiasis (sleeping sickness)</span></div><div class="casAuthors">Kennedy Peter G E</div><div class="citationInfo"><span class="NLM_cas:title">Annals of neurology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">116-26</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Human African trypanosomiasis, also known as sleeping sickness, is a neglected disease, and it continues to pose a major threat to 60 million people in 36 countries in sub-Saharan Africa.  Transmitted by the bite of the tsetse fly, the disease is caused by protozoan parasites of the genus Trypanosoma and comes in two types: East African human African trypanosomiasis caused by Trypanosoma brucei rhodesiense and the West African form caused by Trypanosoma brucei gambiense.  There is an early or hemolymphatic stage and a late or encephalitic stage, when the parasites cross the blood-brain barrier to invade the central nervous system.  Two critical current issues are disease staging and drug therapy, especially for late-stage disease.  Lumbar puncture to analyze cerebrospinal fluid will remain the only method of disease staging until reliable noninvasive methods are developed, but there is no widespread consensus as to what exactly defines biologically central nervous system disease or what specific cerebrospinal fluid findings should justify drug therapy for late-stage involvement.  All four main drugs used for human African trypanosomiasis are toxic, and melarsoprol, the only drug that is effective for both types of central nervous system disease, is so toxic that it kills 5% of patients who receive it.  Eflornithine, alone or combined with nifurtimox, is being used increasingly as first-line therapy for gambiense disease.  There is a pressing need for an effective, safe oral drug for both stages of the disease, but this will require a significant increase in investment for new drug discovery from Western governments and the pharmaceutical industry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQoak2ZgH4MMJdYoSR9eyvsfW6udTcc2eaPK9r8oUA5vLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1crnsFKrsw%253D%253D&md5=7bc60606f9dab108c3b1c6911cb018a8</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1002%2Fana.21429&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fana.21429%26sid%3Dliteratum%253Aachs%26aulast%3DKennedy%26aufirst%3DP.%2BG.%26atitle%3DThe%2520continuing%2520problem%2520of%2520human%2520African%2520trypanosomiasis%2520%2528sleeping%2520sickness%2529%26jtitle%3DAnn.%2520Neurol.%26date%3D2008%26volume%3D64%26spage%3D116%26epage%3D126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Nare, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luba, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hardy, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beverley, S.</span><span> </span><span class="NLM_article-title">New approaches to <i>Leishmania</i> chemotherapy: pteridine reductase 1 (PTR1) as a target and modulator of antifolate sensitivity</span> <span class="citation_source-journal">Parasitology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">114</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl.</span><span class="NLM_x">) </span> <span class="NLM_fpage">S101</span><span class="NLM_x">–</span> <span class="NLM_lpage">S110</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=1997&pages=S101-S110&issue=Suppl.&author=B.+Nareauthor=J.+Lubaauthor=L.+W.+Hardyauthor=S.+Beverley&title=New+approaches+to+Leishmania+chemotherapy%3A+pteridine+reductase+1+%28PTR1%29+as+a+target+and+modulator+of+antifolate+sensitivity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNare%26aufirst%3DB.%26aulast%3DLuba%26aufirst%3DJ.%26aulast%3DHardy%26aufirst%3DL.%2BW.%26aulast%3DBeverley%26aufirst%3DS.%26atitle%3DNew%2520approaches%2520to%2520Leishmania%2520chemotherapy%253A%2520pteridine%2520reductase%25201%2520%2528PTR1%2529%2520as%2520a%2520target%2520and%2520modulator%2520of%2520antifolate%2520sensitivity%26jtitle%3DParasitology%26date%3D1997%26volume%3D114%26issue%3DSuppl%26spage%3DS101%26epage%3DS110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Berriman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghedin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hertz-Fowler, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blandin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renauld, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bartholomeu, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lennard, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caler, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamlin, N. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haas, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bohme, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hannick, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aslett, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shallom, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcello, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hou, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wickstead, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alsmark, U. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atkin, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barron, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bringaud, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brooks, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carrington, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherevach, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chillingworth, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Churcher, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, L. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corton, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cronin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doggett, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Djikeng, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feldblyum, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Field, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fraser, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodhead, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hance, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harper, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hauser, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hostetler, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ivens, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jagels, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kerhornou, A. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koo, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larke, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landfear, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larkin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leech, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Line, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lord, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macleod, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mooney, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moule, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morgan, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mungall, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norbertczak, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ormond, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pai, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peacock, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quail, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabbinowitsch, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rajandream, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reitter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salzberg, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanders, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schobel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmonds, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simpson, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tallon, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turner, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tait, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tivey, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Aken, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wanless, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitehead, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodward, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wortman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Embley, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gull, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ullu, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barry, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairlamb, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Opperdoes, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barrell, B. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donelson, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hall, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fraser, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melville, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">El-Sayed, N. M.</span><span> </span><span class="NLM_article-title">The genome of the African trypanosome <i>Trypanosoma brucei</i></span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">309</span><span class="NLM_x">, </span> <span class="NLM_fpage">416</span><span class="NLM_x">–</span> <span class="NLM_lpage">422</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=309&publication_year=2005&pages=416-422&author=M.+Berrimanauthor=E.+Ghedinauthor=C.+Hertz-Fowlerauthor=G.+Blandinauthor=H.+Renauldauthor=D.+C.+Bartholomeuauthor=N.+J.+Lennardauthor=E.+Calerauthor=N.+E.+Hamlinauthor=B.+Haasauthor=U.+Bohmeauthor=L.+Hannickauthor=M.+A.+Aslettauthor=J.+Shallomauthor=L.+Marcelloauthor=L.+Houauthor=B.+Wicksteadauthor=U.+C.+Alsmarkauthor=C.+Arrowsmithauthor=R.+J.+Atkinauthor=A.+J.+Barronauthor=F.+Bringaudauthor=K.+Brooksauthor=M.+Carringtonauthor=I.+Cherevachauthor=T.+J.+Chillingworthauthor=C.+Churcherauthor=L.+N.+Clarkauthor=C.+H.+Cortonauthor=A.+Croninauthor=R.+M.+Daviesauthor=J.+Doggettauthor=A.+Djikengauthor=T.+Feldblyumauthor=M.+C.+Fieldauthor=A.+Fraserauthor=I.+Goodheadauthor=Z.+Hanceauthor=D.+Harperauthor=B.+R.+Harrisauthor=H.+Hauserauthor=J.+Hostetlerauthor=A.+Ivensauthor=K.+Jagelsauthor=D.+Johnsonauthor=J.+Johnsonauthor=K.+Jonesauthor=A.+X.+Kerhornouauthor=H.+Kooauthor=N.+Larkeauthor=S.+Landfearauthor=C.+Larkinauthor=V.+Leechauthor=A.+Lineauthor=A.+Lordauthor=A.+Macleodauthor=P.+J.+Mooneyauthor=S.+Mouleauthor=D.+M.+Martinauthor=G.+W.+Morganauthor=K.+Mungallauthor=H.+Norbertczakauthor=D.+Ormondauthor=G.+Paiauthor=C.+S.+Peacockauthor=J.+Petersonauthor=M.+A.+Quailauthor=E.+Rabbinowitschauthor=M.+A.+Rajandreamauthor=C.+Reitterauthor=S.+L.+Salzbergauthor=M.+Sandersauthor=S.+Schobelauthor=S.+Sharpauthor=M.+Simmondsauthor=A.+J.+Simpsonauthor=L.+Tallonauthor=C.+M.+Turnerauthor=A.+Taitauthor=A.+R.+Tiveyauthor=S.+Van+Akenauthor=D.+Walkerauthor=D.+Wanlessauthor=S.+Wangauthor=B.+Whiteauthor=O.+Whiteauthor=S.+Whiteheadauthor=J.+Woodwardauthor=J.+Wortmanauthor=M.+D.+Adamsauthor=T.+M.+Embleyauthor=K.+Gullauthor=E.+Ulluauthor=J.+D.+Barryauthor=A.+H.+Fairlambauthor=F.+Opperdoesauthor=B.+G.+Barrellauthor=J.+E.+Donelsonauthor=N.+Hallauthor=C.+M.+Fraserauthor=S.+E.+Melvilleauthor=N.+M.+El-Sayed&title=The+genome+of+the+African+trypanosome+Trypanosoma+brucei"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBerriman%26aufirst%3DM.%26aulast%3DGhedin%26aufirst%3DE.%26aulast%3DHertz-Fowler%26aufirst%3DC.%26aulast%3DBlandin%26aufirst%3DG.%26aulast%3DRenauld%26aufirst%3DH.%26aulast%3DBartholomeu%26aufirst%3DD.%2BC.%26aulast%3DLennard%26aufirst%3DN.%2BJ.%26aulast%3DCaler%26aufirst%3DE.%26aulast%3DHamlin%26aufirst%3DN.%2BE.%26aulast%3DHaas%26aufirst%3DB.%26aulast%3DBohme%26aufirst%3DU.%26aulast%3DHannick%26aufirst%3DL.%26aulast%3DAslett%26aufirst%3DM.%2BA.%26aulast%3DShallom%26aufirst%3DJ.%26aulast%3DMarcello%26aufirst%3DL.%26aulast%3DHou%26aufirst%3DL.%26aulast%3DWickstead%26aufirst%3DB.%26aulast%3DAlsmark%26aufirst%3DU.%2BC.%26aulast%3DArrowsmith%26aufirst%3DC.%26aulast%3DAtkin%26aufirst%3DR.%2BJ.%26aulast%3DBarron%26aufirst%3DA.%2BJ.%26aulast%3DBringaud%26aufirst%3DF.%26aulast%3DBrooks%26aufirst%3DK.%26aulast%3DCarrington%26aufirst%3DM.%26aulast%3DCherevach%26aufirst%3DI.%26aulast%3DChillingworth%26aufirst%3DT.%2BJ.%26aulast%3DChurcher%26aufirst%3DC.%26aulast%3DClark%26aufirst%3DL.%2BN.%26aulast%3DCorton%26aufirst%3DC.%2BH.%26aulast%3DCronin%26aufirst%3DA.%26aulast%3DDavies%26aufirst%3DR.%2BM.%26aulast%3DDoggett%26aufirst%3DJ.%26aulast%3DDjikeng%26aufirst%3DA.%26aulast%3DFeldblyum%26aufirst%3DT.%26aulast%3DField%26aufirst%3DM.%2BC.%26aulast%3DFraser%26aufirst%3DA.%26aulast%3DGoodhead%26aufirst%3DI.%26aulast%3DHance%26aufirst%3DZ.%26aulast%3DHarper%26aufirst%3DD.%26aulast%3DHarris%26aufirst%3DB.%2BR.%26aulast%3DHauser%26aufirst%3DH.%26aulast%3DHostetler%26aufirst%3DJ.%26aulast%3DIvens%26aufirst%3DA.%26aulast%3DJagels%26aufirst%3DK.%26aulast%3DJohnson%26aufirst%3DD.%26aulast%3DJohnson%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DK.%26aulast%3DKerhornou%26aufirst%3DA.%2BX.%26aulast%3DKoo%26aufirst%3DH.%26aulast%3DLarke%26aufirst%3DN.%26aulast%3DLandfear%26aufirst%3DS.%26aulast%3DLarkin%26aufirst%3DC.%26aulast%3DLeech%26aufirst%3DV.%26aulast%3DLine%26aufirst%3DA.%26aulast%3DLord%26aufirst%3DA.%26aulast%3DMacleod%26aufirst%3DA.%26aulast%3DMooney%26aufirst%3DP.%2BJ.%26aulast%3DMoule%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DD.%2BM.%26aulast%3DMorgan%26aufirst%3DG.%2BW.%26aulast%3DMungall%26aufirst%3DK.%26aulast%3DNorbertczak%26aufirst%3DH.%26aulast%3DOrmond%26aufirst%3DD.%26aulast%3DPai%26aufirst%3DG.%26aulast%3DPeacock%26aufirst%3DC.%2BS.%26aulast%3DPeterson%26aufirst%3DJ.%26aulast%3DQuail%26aufirst%3DM.%2BA.%26aulast%3DRabbinowitsch%26aufirst%3DE.%26aulast%3DRajandream%26aufirst%3DM.%2BA.%26aulast%3DReitter%26aufirst%3DC.%26aulast%3DSalzberg%26aufirst%3DS.%2BL.%26aulast%3DSanders%26aufirst%3DM.%26aulast%3DSchobel%26aufirst%3DS.%26aulast%3DSharp%26aufirst%3DS.%26aulast%3DSimmonds%26aufirst%3DM.%26aulast%3DSimpson%26aufirst%3DA.%2BJ.%26aulast%3DTallon%26aufirst%3DL.%26aulast%3DTurner%26aufirst%3DC.%2BM.%26aulast%3DTait%26aufirst%3DA.%26aulast%3DTivey%26aufirst%3DA.%2BR.%26aulast%3DVan%2BAken%26aufirst%3DS.%26aulast%3DWalker%26aufirst%3DD.%26aulast%3DWanless%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DWhite%26aufirst%3DB.%26aulast%3DWhite%26aufirst%3DO.%26aulast%3DWhitehead%26aufirst%3DS.%26aulast%3DWoodward%26aufirst%3DJ.%26aulast%3DWortman%26aufirst%3DJ.%26aulast%3DAdams%26aufirst%3DM.%2BD.%26aulast%3DEmbley%26aufirst%3DT.%2BM.%26aulast%3DGull%26aufirst%3DK.%26aulast%3DUllu%26aufirst%3DE.%26aulast%3DBarry%26aufirst%3DJ.%2BD.%26aulast%3DFairlamb%26aufirst%3DA.%2BH.%26aulast%3DOpperdoes%26aufirst%3DF.%26aulast%3DBarrell%26aufirst%3DB.%2BG.%26aulast%3DDonelson%26aufirst%3DJ.%2BE.%26aulast%3DHall%26aufirst%3DN.%26aulast%3DFraser%26aufirst%3DC.%2BM.%26aulast%3DMelville%26aufirst%3DS.%2BE.%26aulast%3DEl-Sayed%26aufirst%3DN.%2BM.%26atitle%3DThe%2520genome%2520of%2520the%2520African%2520trypanosome%2520Trypanosoma%2520brucei%26jtitle%3DScience%26date%3D2005%26volume%3D309%26spage%3D416%26epage%3D422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Gangjee, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, H. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurup, S.</span><span> </span><span class="NLM_article-title">Recent advances in classical and non-classical antifolates as antitumor and antiopportunistic infection agents: part I</span> <span class="citation_source-journal">Anticancer Agents Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">524</span><span class="NLM_x">–</span> <span class="NLM_lpage">542</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.2174%2F187152007781668724" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=17896913" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtV2gtr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=524-542&author=A.+Gangjeeauthor=H.+D.+Jainauthor=S.+Kurup&title=Recent+advances+in+classical+and+non-classical+antifolates+as+antitumor+and+antiopportunistic+infection+agents%3A+part+I"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in classical and non-classical antifolates as antitumor and antiopportunistic infection agents: part I</span></div><div class="casAuthors">Gangjee, Aleem; Jain, Hiteshkumar D.; Kurup, Sonali</div><div class="citationInfo"><span class="NLM_cas:title">Anti-Cancer Agents in Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">524-542</span>CODEN:
                <span class="NLM_cas:coden">AAMCE4</span>;
        ISSN:<span class="NLM_cas:issn">1871-5206</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Antifolates that inhibit the key enzymes thymidylate synthase (TS) and dihydrofolate reductase (DHFR) have found clin. utility as antitumor and antiopportunistic agents.  Methotrexate {MTX, (1)} and 5-fluorouracil (5-FU) were among the first clin. useful DHFR and TS inhibitors, resp.  The development of resistance to 5-FU, its occasional unpredictable activity and toxicity resulted in the search of novel antifolates.  Pemetrexed (4) and raltitrexed (5) specifically inhibit TS, and are clin. useful as antitumor agents.  A major mechanism of tumor resistance to clin. useful antifolates is based on their need for polyglutamylation via the enzyme folypoly-γ-glutamate synthetase (FPGS).  Novel antifolates have been developed that do not need to be polyglutamylated and include plevitrexed (6) and GWI843 (7).  Nonclassical antifolates for antitumor and parasitic chemotherapy, such as nolatrexed (8), trimethoprim {TMP, (11)} and piritrexim {PTX, (12)}, can passively diffuse into cells and hence do not have to depend on FPGS or the reduced folate carrier (RFC).  Variations in the structures of antifolates have helped delineate the structural influence on the interaction with TS, DHFR, FPGS, and RFC utilization.  The differences in the active site of human and pathogen DHFR have also been exploited.  The literature contains excellent reviews on the design and synthesis of antifolates prior to 1996.  This two-part review discusses the design, synthesis and structural requirements for TS and DHFR inhibition and their relevance to antitumor and parasitic chemotherapy, since 1996.  Monocyclic and 6-5 fused bicyclic antifolates will be discussed in Part I, while 6-6 bicyclic and tricyclic antifolates will be discussed in Part II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofSNk-MZR8S7Vg90H21EOLACvtfcHk0lgS9rQRjS7Oaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtV2gtr3J&md5=c5a93e7c3fa50fdf071800131d0ef1db</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.2174%2F187152007781668724&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F187152007781668724%26sid%3Dliteratum%253Aachs%26aulast%3DGangjee%26aufirst%3DA.%26aulast%3DJain%26aufirst%3DH.%2BD.%26aulast%3DKurup%26aufirst%3DS.%26atitle%3DRecent%2520advances%2520in%2520classical%2520and%2520non-classical%2520antifolates%2520as%2520antitumor%2520and%2520antiopportunistic%2520infection%2520agents%253A%2520part%2520I%26jtitle%3DAnticancer%2520Agents%2520Med.%2520Chem.%26date%3D2007%26volume%3D7%26spage%3D524%26epage%3D542" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Gangjee, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, H. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurup, S.</span><span> </span><span class="NLM_article-title">Recent advances in classical and nonclassical antifolates as antitumor and antiopportunistic infection agents: Part II</span> <span class="citation_source-journal">Anticancer Agents Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">205</span><span class="NLM_x">–</span> <span class="NLM_lpage">231</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.2174%2F187152008783497064" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=18288923" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFOmsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2008&pages=205-231&author=A.+Gangjeeauthor=H.+D.+Jainauthor=S.+Kurup&title=Recent+advances+in+classical+and+nonclassical+antifolates+as+antitumor+and+antiopportunistic+infection+agents%3A+Part+II"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in classical and non-classical antifolates as antitumor and antiopportunistic infection agents: Part II</span></div><div class="casAuthors">Gangjee, Aleem; Jain, Hiteshkumar D.; Kurup, Sonali</div><div class="citationInfo"><span class="NLM_cas:title">Anti-Cancer Agents in Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">205-231</span>CODEN:
                <span class="NLM_cas:coden">AAMCE4</span>;
        ISSN:<span class="NLM_cas:issn">1871-5206</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Antifolates that inhibit the key enzymes thymidylate synthase (TS) and dihydrofolate reductase (DHFR) have found clin. utility as antitumor and antiopportunistic agents.  Methotrexate (MTX) and 5-fluorouracil (5-FU) were among the first clin. useful DHFR and TS inhibitors, resp.  The development of resistance to 5-FU, its occasional unpredictable activity and toxicity resulted in the search of novel antifolates.  Pemetrexed and raltitrexed are newer antifolates that specifically inhibit TS, and are clin. useful as antitumor agents.  A major mechanism of tumor resistance to clin. useful antifolates is based on their need for polyglutamylation via the enzyme folylpoly-γ-glutamate synthetase (FPGS).  Recently, classical antifolates that do not need to be polyglutamylated have also been developed and include plevitrexed and GW1843.  Nolatrexed, trimethoprim (TMP) and piritrexim (PTX) are nonclassical antifolates for antitumor and parasitic chemotherapy that passively diffuse into cells and hence do not have to depend on FPGS or the reduced folate carrier (RFC).  Structural requirements for inhibition with antifolates have been studied extensively and novel agents that exploit key interactions in the active site of TS, DHFR, FPGS, and RFC have been proposed.  This two-part review discusses the design, synthesis and structural requirements for TS and DHFR inhibition and their relevance to antitumor and parasitic chemotherapy, since 1996.  Monocyclic and 6-5 fused bicyclic antifolates were discussed in Part I.  The 6-6 bicyclic and tricyclic antifolates will be discussed here in Part II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo94ZvcpFPCFbVg90H21EOLACvtfcHk0lhTlGsDRxJQNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFOmsbo%253D&md5=eaa3bca180f933e57873a7b2f26303d6</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.2174%2F187152008783497064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F187152008783497064%26sid%3Dliteratum%253Aachs%26aulast%3DGangjee%26aufirst%3DA.%26aulast%3DJain%26aufirst%3DH.%2BD.%26aulast%3DKurup%26aufirst%3DS.%26atitle%3DRecent%2520advances%2520in%2520classical%2520and%2520nonclassical%2520antifolates%2520as%2520antitumor%2520and%2520antiopportunistic%2520infection%2520agents%253A%2520Part%2520II%26jtitle%3DAnticancer%2520Agents%2520Med.%2520Chem.%26date%3D2008%26volume%3D8%26spage%3D205%26epage%3D231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Bello, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nare, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freedman, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hardy, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beverley, S. M.</span><span> </span><span class="NLM_article-title">PTR1: a reductase mediating salvage of oxidized pteridines and methotrexate resistance in the protozoan parasite <i>Leishmania major</i></span> <span class="citation_source-journal">Proc. Natl. Acad. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">91</span><span class="NLM_x">, </span> <span class="NLM_fpage">11442</span><span class="NLM_x">–</span> <span class="NLM_lpage">11446</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1073%2Fpnas.91.24.11442" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=7972081" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADyaK2MXitlyrs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=1994&pages=11442-11446&author=A.+R.+Belloauthor=B.+Nareauthor=D.+Freedmanauthor=L.+Hardyauthor=S.+M.+Beverley&title=PTR1%3A+a+reductase+mediating+salvage+of+oxidized+pteridines+and+methotrexate+resistance+in+the+protozoan+parasite+Leishmania+major"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">PTR1: a reductase mediating salvage of oxidized pteridines and methotrexate resistance in the protozoan parasite Leishmania major</span></div><div class="casAuthors">Bello, Alexandre R.; Nare, Bakela; Freedman, Daniel; Hardy, Larry; Beverley, Stephen M.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11442-6</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Trypanosomatid protozoans are pterin auxotrophs, a finding noted decades ago which heralded the discovery of key metabolic roles played by pteridines in eukaryotes.  We have now identified the enzyme mediating unconjugated pteridine salvage in the human parasite Leishmania major.  PTR1 is the gene in the amplified H region responsible for methotrexate (MTX) resistance, and belongs to a large family of oxidoreductases with diverse substrates and roles.  We generated Leishmania lacking PTR1 by homologous gene targeting, and these ptr1- mutants required reduced biopterin (dihydro- or tetrahydrobiopterin) for growth.  PTR1 purified from engineered Escherichia coli exhibited a MTX-sensitive, NADPH-dependent biopterin reductase activity.  PTR1 showed good activity with folate and significant activity with dihydrofolate and dihydrobiopterin, but not with quinonoid dihydrobiopterin.  PTR1 thus differs considerably from previously reported pteridine reductases of trypanosomatids and vertebrates.  Pteridine reductase activity was diminished in ptr1- Leishmania and was evaluated in transfected parasites bearing multiple copies of PTR1; correspondingly, ptr1- was MTX-hypersensitive whereas the multicopy transfectant was MTX-resistant.  The concordance of the biochem. and genetic properties of PTR1 suggests that this is the primary enzyme mediating pteridine salvage.  These findings suggest several possible mechanisms for PTR1-mediated MTX resistance and should aid in the design of rational chemotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiIWNgbgpSOrVg90H21EOLACvtfcHk0lhTlGsDRxJQNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXitlyrs7c%253D&md5=202ac2d39e22c7e1a6fe3cf4aeb3943c</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1073%2Fpnas.91.24.11442&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.91.24.11442%26sid%3Dliteratum%253Aachs%26aulast%3DBello%26aufirst%3DA.%2BR.%26aulast%3DNare%26aufirst%3DB.%26aulast%3DFreedman%26aufirst%3DD.%26aulast%3DHardy%26aufirst%3DL.%26aulast%3DBeverley%26aufirst%3DS.%2BM.%26atitle%3DPTR1%253A%2520a%2520reductase%2520mediating%2520salvage%2520of%2520oxidized%2520pteridines%2520and%2520methotrexate%2520resistance%2520in%2520the%2520protozoan%2520parasite%2520Leishmania%2520major%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%26date%3D1994%26volume%3D91%26spage%3D11442%26epage%3D11446" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Sienkiewicz, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaroslawski, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyllie, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairlamb, A. H.</span><span> </span><span class="NLM_article-title">Chemical and genetic validation of dihydrofolate reductase-thymidylate synthase as a drug target in African trypanosomes</span> <span class="citation_source-journal">Mol. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">520</span><span class="NLM_x">–</span> <span class="NLM_lpage">533</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1111%2Fj.1365-2958.2008.06305.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=18557814" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD1cXoslyjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2008&pages=520-533&author=N.+Sienkiewiczauthor=S.+Jaroslawskiauthor=S.+Wyllieauthor=A.+H.+Fairlamb&title=Chemical+and+genetic+validation+of+dihydrofolate+reductase-thymidylate+synthase+as+a+drug+target+in+African+trypanosomes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical and genetic validation of dihydrofolate reductase-thymidylate synthase as a drug target in African trypanosomes</span></div><div class="casAuthors">Sienkiewicz, Natasha; Jaroslawski, Szymon; Wyllie, Susan; Fairlamb, Alan H.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Microbiology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">520-533</span>CODEN:
                <span class="NLM_cas:coden">MOMIEE</span>;
        ISSN:<span class="NLM_cas:issn">0950-382X</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The phenotypes of single-(SKO) and double-knockout (DKO) lines of dihydrofolate reductase-thymidylate synthase (DHFR-TS) of bloodstream Trypanosoma brucei were evaluated in vitro and in vivo.  Growth of SKO in vitro is identical to wild-type (WT) cells, whereas DKO has an abs. requirement for thymidine.  Removal of thymidine from the medium triggers growth arrest in S phase, assocd. with gross morphol. changes, followed by cell death after 60 h.  DKO is unable to infect mice, whereas the virulence of SKO is similar to WT.  Normal growth and virulence could be restored by transfection of DKO with T. brucei DHFR-TS, but not with Escherichia coli TS.  As pteridine reductase (PTR1) levels are unchanged in SKO and DKO cells, PTR1 is not able to compensate for loss of DHFR activity.  Drugs such as raltitrexed or methotrexate with structural similarity to folic acid are up to 300-fold more potent inhibitors of WT cultured in a novel low-folate medium, unlike hydrophobic antifols such as trimetrexate or pyrimethamine.  DKO trypanosomes show reduced sensitivity to these inhibitors ranging from twofold for trimetrexate to > 10 000-fold for raltitrexed.  These data demonstrate that DHFR-TS is essential for parasite survival and represents a promising target for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAtWTOrNZCBLVg90H21EOLACvtfcHk0lhTlGsDRxJQNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXoslyjt7w%253D&md5=026298043b57f45ae922ac3833100a0c</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2958.2008.06305.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2958.2008.06305.x%26sid%3Dliteratum%253Aachs%26aulast%3DSienkiewicz%26aufirst%3DN.%26aulast%3DJaroslawski%26aufirst%3DS.%26aulast%3DWyllie%26aufirst%3DS.%26aulast%3DFairlamb%26aufirst%3DA.%2BH.%26atitle%3DChemical%2520and%2520genetic%2520validation%2520of%2520dihydrofolate%2520reductase-thymidylate%2520synthase%2520as%2520a%2520drug%2520target%2520in%2520African%2520trypanosomes%26jtitle%3DMol.%2520Microbiol.%26date%3D2008%26volume%3D69%26spage%3D520%26epage%3D533" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Nare, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hardy, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beverley, S. M.</span><span> </span><span class="NLM_article-title">The roles of pteridine reductase 1 and dihydrofolate reductase-thymidylate synthase in pteridine metabolism in the protozoan parasite <i>Leishmania major</i></span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">272</span><span class="NLM_x">, </span> <span class="NLM_fpage">13883</span><span class="NLM_x">–</span> <span class="NLM_lpage">13891</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=272&publication_year=1997&pages=13883-13891&author=B.+Nareauthor=L.+W.+Hardyauthor=S.+M.+Beverley&title=The+roles+of+pteridine+reductase+1+and+dihydrofolate+reductase-thymidylate+synthase+in+pteridine+metabolism+in+the+protozoan+parasite+Leishmania+major"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNare%26aufirst%3DB.%26aulast%3DHardy%26aufirst%3DL.%2BW.%26aulast%3DBeverley%26aufirst%3DS.%2BM.%26atitle%3DThe%2520roles%2520of%2520pteridine%2520reductase%25201%2520and%2520dihydrofolate%2520reductase-thymidylate%2520synthase%2520in%2520pteridine%2520metabolism%2520in%2520the%2520protozoan%2520parasite%2520Leishmania%2520major%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1997%26volume%3D272%26spage%3D13883%26epage%3D13891" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Cunningham, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Titus, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turco, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beverley, S. M.</span><span> </span><span class="NLM_article-title">Regulation of differentiation to the infective stage of the protozoan parasite <i>Leishmania</i> major by tetrahydrobiopterin</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">292</span><span class="NLM_x">, </span> <span class="NLM_fpage">285</span><span class="NLM_x">–</span> <span class="NLM_lpage">287</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=292&publication_year=2001&pages=285-287&author=M.+L.+Cunninghamauthor=R.+G.+Titusauthor=S.+J.+Turcoauthor=S.+M.+Beverley&title=Regulation+of+differentiation+to+the+infective+stage+of+the+protozoan+parasite+Leishmania+major+by+tetrahydrobiopterin"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCunningham%26aufirst%3DM.%2BL.%26aulast%3DTitus%26aufirst%3DR.%2BG.%26aulast%3DTurco%26aufirst%3DS.%2BJ.%26aulast%3DBeverley%26aufirst%3DS.%2BM.%26atitle%3DRegulation%2520of%2520differentiation%2520to%2520the%2520infective%2520stage%2520of%2520the%2520protozoan%2520parasite%2520Leishmania%2520major%2520by%2520tetrahydrobiopterin%26jtitle%3DScience%26date%3D2001%26volume%3D292%26spage%3D285%26epage%3D287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Cavazzuti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paglietti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, W. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gamarro, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piras, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loriga, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allecca, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corona, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLuskey, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tulloch, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibellini, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costi, M. P.</span><span> </span><span class="NLM_article-title">Discovery of potent pteridine reductase inhibitors to guide antiparasite drug development</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">1448</span><span class="NLM_x">–</span> <span class="NLM_lpage">1453</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1073%2Fpnas.0704384105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=18245389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhvFCitrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=1448-1453&author=A.+Cavazzutiauthor=G.+Pagliettiauthor=W.+N.+Hunterauthor=F.+Gamarroauthor=S.+Pirasauthor=M.+Lorigaauthor=S.+Alleccaauthor=P.+Coronaauthor=K.+McLuskeyauthor=L.+Tullochauthor=F.+Gibelliniauthor=S.+Ferrariauthor=M.+P.+Costi&title=Discovery+of+potent+pteridine+reductase+inhibitors+to+guide+antiparasite+drug+development"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of potent pteridine reductase inhibitors to guide antiparasite drug development</span></div><div class="casAuthors">Cavazzuti, Antonio; Paglietti, Giuseppe; Hunter, William N.; Gamarro, Francisco; Piras, Sandra; Loriga, Mario; Alleca, Sergio; Corona, Paola; McLuskey, Karen; Tulloch, Lindsay; Gibellini, Federica; Ferrari, Stefania; Costi, Maria Paola</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1448-1453</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Pteridine reductase (PTR1) is essential for salvage of pterins by parasitic trypanosomatids and is a target for the development of improved therapies.  To identify inhibitors of Leishmania major and Trypanosoma cruzi PTR1, a rapid-screening strategy using a folate-based library was combined with structure-based design.  Assays were carried out against folate-dependent enzymes including PTR1, dihydrofolate reductase (DHFR), and thymidylate synthase.  Affinity profiling detd. selectivity and specificity of a series of quinoxaline and 2,4-diaminopteridine derivs., and nine compds. showed greater activity against parasite enzymes compared with human enzymes.  Compd. I [R = H, Me (II)] displayed a Ki of 100 nM toward LmPTR1, and the crystal structure of the LmPTR1:NADPH:I ternary complex revealed a substrate-like binding mode distinct from that previously obsd. for similar compds.  A second round of design, synthesis, and assay produced a compd. II with a significantly improved Ki (37 nM) against LmPTR1, and the structure of this complex was also detd.  Biol. evaluation of selected inhibitors was performed against the extracellular forms of T. cruzi and L. major, both wild-type and overexpressing PTR1 lines, as a model for PTR1-driven antifolate drug resistance and the intracellular form of T. cruzi.  An additive profile was obsd. when PTR1 inhibitors were used in combination with known DHFR inhibitors, and a redn. in toxicity of treatment was obsd. with respect to administration of a DHFR inhibitor alone.  The successful combination of antifolates targeting two enzymes indicates high potential for such an approach in the development of previously undescribed antiparasitic drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpclEp8-gcEyrVg90H21EOLACvtfcHk0lij89HeK-L3CQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhvFCitrk%253D&md5=d2477ec35df976858f13b6d14d283cad</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0704384105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0704384105%26sid%3Dliteratum%253Aachs%26aulast%3DCavazzuti%26aufirst%3DA.%26aulast%3DPaglietti%26aufirst%3DG.%26aulast%3DHunter%26aufirst%3DW.%2BN.%26aulast%3DGamarro%26aufirst%3DF.%26aulast%3DPiras%26aufirst%3DS.%26aulast%3DLoriga%26aufirst%3DM.%26aulast%3DAllecca%26aufirst%3DS.%26aulast%3DCorona%26aufirst%3DP.%26aulast%3DMcLuskey%26aufirst%3DK.%26aulast%3DTulloch%26aufirst%3DL.%26aulast%3DGibellini%26aufirst%3DF.%26aulast%3DFerrari%26aufirst%3DS.%26aulast%3DCosti%26aufirst%3DM.%2BP.%26atitle%3DDiscovery%2520of%2520potent%2520pteridine%2520reductase%2520inhibitors%2520to%2520guide%2520antiparasite%2520drug%2520development%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%26date%3D2008%26volume%3D105%26spage%3D1448%26epage%3D1453" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Hardy, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matthews, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nare, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beverley, S. M.</span><span> </span><span class="NLM_article-title">Biochemical and Genetic Tests for Inhibitors of <i>Leishmania Pteridine</i> Pathways</span> <span class="citation_source-journal">Exp. Parasitol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">87</span><span class="NLM_x">, </span> <span class="NLM_fpage">158</span><span class="NLM_x">–</span> <span class="NLM_lpage">170</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1006%2Fexpr.1997.4207" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=9398595" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=1997&pages=158-170&author=L.+W.+Hardyauthor=W.+Matthewsauthor=B.+Nareauthor=S.+M.+Beverley&title=Biochemical+and+Genetic+Tests+for+Inhibitors+of+Leishmania+Pteridine+Pathways"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1006%2Fexpr.1997.4207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fexpr.1997.4207%26sid%3Dliteratum%253Aachs%26aulast%3DHardy%26aufirst%3DL.%2BW.%26aulast%3DMatthews%26aufirst%3DW.%26aulast%3DNare%26aufirst%3DB.%26aulast%3DBeverley%26aufirst%3DS.%2BM.%26atitle%3DBiochemical%2520and%2520Genetic%2520Tests%2520for%2520Inhibitors%2520of%2520Leishmania%2520Pteridine%2520Pathways%26jtitle%3DExp.%2520Parasitol.%26date%3D1997%26volume%3D87%26spage%3D158%26epage%3D170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Ecker, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Noe, C. R.</span><span> </span><span class="NLM_article-title">In silico prediction models for blood−brain barrier permeation</span> <span class="citation_source-journal">Curr. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1617</span><span class="NLM_x">–</span> <span class="NLM_lpage">1628</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=15180567" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD2cXktlKksrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2004&pages=1617-1628&author=G.+F.+Eckerauthor=C.+R.+Noe&title=In+silico+prediction+models+for+blood%E2%88%92brain+barrier+permeation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">In silico prediction models for blood-brain barrier permeation</span></div><div class="casAuthors">Ecker, Gerhard F.; Noe, Christian R.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1617-1628</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The ability to permeate across the blood brain barrier (BBB) is essential for drugs acting on the central nervous system (CNS).  Thus, for speeding up the drug discovery process in the CNS-area, it is of great importance to develop systems that allow rapid and inexpensive screening of the BBB-permeability properties of novel lead compds. or at least small subsets of combinatorial CNS-libraries.  In this field, in silico prediction methods gain increasing importance.  Starting with simple regression models based on calcn. of lipophilicity and polar surface area, the field developed via PLS methods to grid based approaches (e.g. VolSurf).  Addnl., the use of artificial neural networks gain increasing importance.  However, permeation through the BBB is also influenced by active transport systems.  For nutrients and endogenous compds., such as amino acids, monocarboxylic acids, amines, hexoses, thyroid hormones, purine bases, and nucleosides, several transport systems regulating the entry of the resp. compd. classes into the brain were identified.  The other way round there is striking evidence that expression of active efflux pumps like the multidrug transporter P-glycoprotein (P-gp) on the luminal membrane of the brain capillary endothelial cells accounts for poor BBB permeability of certain drugs.  Undoubtedly, P-gp is an important impediment for the entry of hydrophobic drugs into the brain.  Thus, proper prediction models should also take into account the active transport phenomena.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnO4MhXnN3lrVg90H21EOLACvtfcHk0lij89HeK-L3CQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXktlKksrw%253D&md5=4bf6aced7aaa6bb593c183d135b3ad2c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEcker%26aufirst%3DG.%2BF.%26aulast%3DNoe%26aufirst%3DC.%2BR.%26atitle%3DIn%2520silico%2520prediction%2520models%2520for%2520blood%25E2%2588%2592brain%2520barrier%2520permeation%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2004%26volume%3D11%26spage%3D1617%26epage%3D1628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Dawson, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibellini, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sienkiewicz, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tulloch, L. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fyfe, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLuskey, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairlamb, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, W. N.</span><span> </span><span class="NLM_article-title">Structure and reactivity of <i>Trypanosoma brucei</i> pteridine reductase: inhibition by the archetypal antifolate methotrexate</span> <span class="citation_source-journal">Mol. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">61</span><span class="NLM_x">, </span> <span class="NLM_fpage">1457</span><span class="NLM_x">–</span> <span class="NLM_lpage">1468</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1111%2Fj.1365-2958.2006.05332.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=16968221" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtFensbrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2006&pages=1457-1468&author=A.+Dawsonauthor=F.+Gibelliniauthor=N.+Sienkiewiczauthor=L.+B.+Tullochauthor=P.+K.+Fyfeauthor=K.+McLuskeyauthor=A.+H.+Fairlambauthor=W.+N.+Hunter&title=Structure+and+reactivity+of+Trypanosoma+brucei+pteridine+reductase%3A+inhibition+by+the+archetypal+antifolate+methotrexate"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and reactivity of Trypanosoma brucei pteridine reductase: inhibition by the archetypal antifolate methotrexate</span></div><div class="casAuthors">Dawson, Alice; Gibellini, Federica; Sienkiewicz, Natasha; Tulloch, Lindsay B.; Fyfe, Paul K.; McLuskey, Karen; Fairlamb, Alan H.; Hunter, William N.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Microbiology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1457-1468</span>CODEN:
                <span class="NLM_cas:coden">MOMIEE</span>;
        ISSN:<span class="NLM_cas:issn">0950-382X</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The protozoan Trypanosoma brucei has a functional pteridine reductase (TbPTR1), an NADPH-dependent short-chain reductase that participates in the salvage of pterins, which are essential for parasite growth.  PTR1 displays broad-spectrum activity with pterins and folates, provides a metabolic bypass for inhibition of the trypanosomatid dihydrofolate reductase and therefore compromises the use of antifolates for treatment of trypanosomiasis.  Catalytic properties of recombinant TbPTR1 and inhibition by the archetypal antifolate methotrexate have been characterized and the crystal structure of the ternary complex with cofactor NADP+ and the inhibitor detd. at 2.2 Å resoln.  This enzyme shares 50% amino acid sequence identity with Leishmania major PTR1 (LmPTR1) and comparisons show that the architecture of the cofactor binding site, and the catalytic center are highly conserved, as are most interactions with the inhibitor.  However, specific amino acid differences, in particular the placement of Trp221 at the side of the active site, and adjustment of the β6-α6 loop and α6 helix at one side of the substrate-binding cleft significantly reduce the size of the substrate binding site of TbPTR1 and alter the chem. properties compared with LmPTR1.  A reactive Cys168, within the active site cleft, in conjunction with the C-terminus carboxyl group and His267 of a partner subunit forms a triad similar to the catalytic component of cysteine proteases.  TbPTR1 therefore offers novel structural features to exploit in the search for inhibitors of therapeutic value against African trypanosomiasis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEKvWRoJekYrVg90H21EOLACvtfcHk0lgKrsisvPzoxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtFensbrF&md5=aa70764962fef14898e2b1bb22e646b3</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2958.2006.05332.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2958.2006.05332.x%26sid%3Dliteratum%253Aachs%26aulast%3DDawson%26aufirst%3DA.%26aulast%3DGibellini%26aufirst%3DF.%26aulast%3DSienkiewicz%26aufirst%3DN.%26aulast%3DTulloch%26aufirst%3DL.%2BB.%26aulast%3DFyfe%26aufirst%3DP.%2BK.%26aulast%3DMcLuskey%26aufirst%3DK.%26aulast%3DFairlamb%26aufirst%3DA.%2BH.%26aulast%3DHunter%26aufirst%3DW.%2BN.%26atitle%3DStructure%2520and%2520reactivity%2520of%2520Trypanosoma%2520brucei%2520pteridine%2520reductase%253A%2520inhibition%2520by%2520the%2520archetypal%2520antifolate%2520methotrexate%26jtitle%3DMol.%2520Microbiol.%26date%3D2006%26volume%3D61%26spage%3D1457%26epage%3D1468" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Gourley, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuttelkopf, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leonard, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luba, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hardy, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beverley, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, W. N.</span><span> </span><span class="NLM_article-title">Pteridine reductase mechanism correlates pterin metabolism with drug resistance in trypanosomatid parasites</span> <span class="citation_source-journal">Nat. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">521</span><span class="NLM_x">–</span> <span class="NLM_lpage">525</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1038%2F88584" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=11373620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD3MXktFektLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2001&pages=521-525&author=D.+G.+Gourleyauthor=A.+W.+Schuttelkopfauthor=G.+A.+Leonardauthor=J.+Lubaauthor=L.+W.+Hardyauthor=S.+M.+Beverleyauthor=W.+N.+Hunter&title=Pteridine+reductase+mechanism+correlates+pterin+metabolism+with+drug+resistance+in+trypanosomatid+parasites"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Pteridine reductase mechanism correlates pterin metabolism with drug resistance in trypanosomatid parasites</span></div><div class="casAuthors">Gourley, David G.; Schottelkopf, Alexander W.; Leonard, Gordon A.; Luba, James; Hardy, Larry W.; Beverley, Stephen M.; Hunter, William N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">521-525</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature America Inc.</span>)
        </div><div class="casAbstract">Pteridine reductase (PTR1) is a short-chain reductase (SDR) responsible for the salvage of pterins in parasitic trypanosomatids.  PTR1 catalyzes the NADPH-dependent two-step redn. of oxidized pterins to the active tetrahydro-forms and reduces susceptibility to antifolates by alleviating dihydrofolate reductase (DHFR) inhibition.  Crystal structure of PTR1 complexed with cofactor and 7,8-dihydrobiopterin (DHB) or methotrexate (MTX) delineate the enzyme mechanism, broad spectrum of activity and inhibition by substrate or an antifolate.  PTR1 applies two distinct reductive mechanisms to substrates bound in one orientation.  The first redn. uses the generic SDR mechanism, whereas the second shares similarities with the mechanism proposed for DHFR.  Both DHB and MTX form extensive hydrogen bonding networks with NADP(H) but differ in the orientation of the pteridine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrB1KQsn-v77Vg90H21EOLACvtfcHk0lg_ABKOIh9V1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXktFektLw%253D&md5=b5be3a57375838e0a9b41429794ba37f</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2F88584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F88584%26sid%3Dliteratum%253Aachs%26aulast%3DGourley%26aufirst%3DD.%2BG.%26aulast%3DSchuttelkopf%26aufirst%3DA.%2BW.%26aulast%3DLeonard%26aufirst%3DG.%2BA.%26aulast%3DLuba%26aufirst%3DJ.%26aulast%3DHardy%26aufirst%3DL.%2BW.%26aulast%3DBeverley%26aufirst%3DS.%2BM.%26aulast%3DHunter%26aufirst%3DW.%2BN.%26atitle%3DPteridine%2520reductase%2520mechanism%2520correlates%2520pterin%2520metabolism%2520with%2520drug%2520resistance%2520in%2520trypanosomatid%2520parasites%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D2001%26volume%3D8%26spage%3D521%26epage%3D525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Congreve, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chessari, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tisi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodhead, A. J.</span><span> </span><span class="NLM_article-title">Recent Developments in Fragment-Based Drug Discovery</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">3661</span><span class="NLM_x">–</span> <span class="NLM_lpage">3680</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm8000373" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=3661-3680&author=M.+Congreveauthor=G.+Chessariauthor=D.+Tisiauthor=A.+J.+Woodhead&title=Recent+Developments+in+Fragment-Based+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm8000373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8000373%26sid%3Dliteratum%253Aachs%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DChessari%26aufirst%3DG.%26aulast%3DTisi%26aufirst%3DD.%26aulast%3DWoodhead%26aufirst%3DA.%2BJ.%26atitle%3DRecent%2520Developments%2520in%2520Fragment-Based%2520Drug%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D3661%26epage%3D3680" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greer, J.</span><span> </span><span class="NLM_article-title">A decade of fragment-based drug design: strategic advances and lessons learned</span> <span class="citation_source-journal">Nat. Rev. Drug. Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">211</span><span class="NLM_x">–</span> <span class="NLM_lpage">219</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1038%2Fnrd2220" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=17290284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD2sXit12rtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=211-219&author=P.+J.+Hajdukauthor=J.+Greer&title=A+decade+of+fragment-based+drug+design%3A+strategic+advances+and+lessons+learned"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">A decade of fragment-based drug design: strategic advances and lessons learned</span></div><div class="casAuthors">Hajduk, Philip J.; Greer, Jonathan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">211-219</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Since the early 1990s, several technol. and scientific advances - such as combinatorial chem., high-throughput screening and the sequencing of the human genome - have been heralded as remedies to the problems facing the pharmaceutical industry.  The use of these technologies in some form is now well established at most pharmaceutical companies; however, the return on investment in terms of marketed products has not met expectations.  Fragment-based drug design is another tool for drug discovery that has emerged in the past decade.  Here, we describe the development and evolution of fragment-based drug design, analyze the role that this approach can have in combination with other discovery technologies and highlight the impact that fragment-based methods have made in progressing new medicines into the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOB8K3AESqTbVg90H21EOLACvtfcHk0lg_ABKOIh9V1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXit12rtr4%253D&md5=bae574da3671475d8488615c3bfa9849</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnrd2220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2220%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DGreer%26aufirst%3DJ.%26atitle%3DA%2520decade%2520of%2520fragment-based%2520drug%2520design%253A%2520strategic%2520advances%2520and%2520lessons%2520learned%26jtitle%3DNat.%2520Rev.%2520Drug.%2520Discovery%26date%3D2007%26volume%3D6%26spage%3D211%26epage%3D219" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Brenk, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schipani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krasowski, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, I. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frearson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyatt, P. G.</span><span> </span><span class="NLM_article-title">Lessons learned from assembling screening libraries for drug discovery for neglected diseases</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">435</span><span class="NLM_x">–</span> <span class="NLM_lpage">444</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=435-444&author=R.+Brenkauthor=A.+Schipaniauthor=D.+Jamesauthor=A.+Krasowskiauthor=I.+H.+Gilbertauthor=J.+Frearsonauthor=P.+G.+Wyatt&title=Lessons+learned+from+assembling+screening+libraries+for+drug+discovery+for+neglected+diseases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBrenk%26aufirst%3DR.%26aulast%3DSchipani%26aufirst%3DA.%26aulast%3DJames%26aufirst%3DD.%26aulast%3DKrasowski%26aufirst%3DA.%26aulast%3DGilbert%26aufirst%3DI.%2BH.%26aulast%3DFrearson%26aufirst%3DJ.%26aulast%3DWyatt%26aufirst%3DP.%2BG.%26atitle%3DLessons%2520learned%2520from%2520assembling%2520screening%2520libraries%2520for%2520drug%2520discovery%2520for%2520neglected%2520diseases%26jtitle%3DChemMedChem%26date%3D2008%26volume%3D3%26spage%3D435%26epage%3D444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Lorber, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Flexible ligand docking using conformational ensembles</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">938</span><span class="NLM_x">–</span> <span class="NLM_lpage">950</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1002%2Fpro.5560070411" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=9568900" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADyaK1cXisFSnurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1998&pages=938-950&author=D.+M.+Lorberauthor=B.+K.+Shoichet&title=Flexible+ligand+docking+using+conformational+ensembles"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Flexible ligand docking using conformational ensembles</span></div><div class="casAuthors">Lorber, David M.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">938-950</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">Mol. docking algorithms suggest possible structures for mol. complexes.  They are used to model biol. function and to discover potential ligands.  A present challenge for docking algorithms is the treatment of mol. flexibility.  Here, the rigid body program, DOCK, is modified to allow it to rapidly fit multiple conformations of ligands.  Conformations of a given mol. are pre-calcd. in the same frame of ref., so that each conformer shares a common rigid fragment with all other conformations.  The ligand conformers are then docked together, as an ensemble, into a receptor binding site.  This takes advantage of the redundancy present in differing conformers of the same mol.  The algorithm was tested using three org. ligand protein systems and two protein-protein systems.  Both the bound and unbound conformations of the receptors were used.  The ligand ensemble method found conformations that resembled those detd. in X-ray crystal structures (RMS values typically less than 1.5 Å).  To test the method's usefulness for inhibitor discovery, multi-compd. and multi-conformer databases were screened for compds. known to bind to dihydrofolate reductase and compds. known to bind to thymidylate synthase.  In both cases, known inhibitors and substrates were identified in conformations resembling those obsd. exptl.  The ligand ensemble method was 100-fold faster than docking a single conformation at a time and was able to screen a database of over 34 million conformations from 117,000 mols. in one to four CPU days on a workstation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8GYuDVf7zG7Vg90H21EOLACvtfcHk0lg_ABKOIh9V1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXisFSnurg%253D&md5=64d79dfe6a048c76aec43f8bd9f6bc62</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1002%2Fpro.5560070411&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.5560070411%26sid%3Dliteratum%253Aachs%26aulast%3DLorber%26aufirst%3DD.%2BM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DFlexible%2520ligand%2520docking%2520using%2520conformational%2520ensembles%26jtitle%3DProtein%2520Sci.%26date%3D1998%26volume%3D7%26spage%3D938%26epage%3D950" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Wei, B. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baase, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weaver, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matthews, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">A model binding site for testing scoring functions in molecular docking</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">322</span><span class="NLM_x">, </span> <span class="NLM_fpage">339</span><span class="NLM_x">–</span> <span class="NLM_lpage">355</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1016%2FS0022-2836%2802%2900777-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=12217695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD38Xms1Wqsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=322&publication_year=2002&pages=339-355&author=B.+Q.+Weiauthor=W.+A.+Baaseauthor=L.+H.+Weaverauthor=B.+W.+Matthewsauthor=B.+K.+Shoichet&title=A+model+binding+site+for+testing+scoring+functions+in+molecular+docking"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A Model Binding Site for Testing Scoring Functions in Molecular Docking</span></div><div class="casAuthors">Wei, Binqing Q.; Baase, Walter A.; Weaver, Larry H.; Matthews, Brian W.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">322</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">339-355</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Prediction of interaction energies between ligands and their receptors remains a major challenge for structure-based inhibitor discovery.  Much effort has been devoted to developing scoring schemes that can successfully rank the affinities of a diverse set of possible ligands to a binding site for which the structure is known.  To test these scoring functions, well-characterized exptl. systems can be very useful.  Here, mutation-created binding sites in T4 lysozyme were used to investigate how the quality of at. charges and solvation energies affects mol. docking.  At. charges and solvation energies were calcd. for 172,118 mols. in the Available Chems.  Directory using a semi-empirical quantum mech. approach by the program AMSOL.  The database was first screened against the apolar cavity site created by the mutation Leu99Ala (L99A).  Compared to the electronegativity-based charges that are widely used, the new charges and desolvation energies improved ranking of known apolar ligands, and better distinguished them from more polar isosteres that are not obsd. to bind.  To investigate whether the new charges had predictive value, the non-polar residue Met102, which forms part of the binding site, was changed to the polar residue glutamine.  The structure of the resulting Leu99 Ala and Met102 Gln double mutant of T4 lysozyme (L99A/M102Q) was detd. and the docking calcn. was repeated for the new site.  Seven representative polar mols. that preferentially docked to the polar vs. the apolar binding site were tested exptl.  All seven bind to the polar cavity (L99A/M102Q) but do not detectably bind to the apolar cavity (L99A).  Five ligand-bound structures of L99A/M102Q were detd. by X-ray crystallog.  Docking predictions corresponded to the crystallog. results to within 0.4 A RMSD.  Improved treatment of partial at. charges and desolvation energies in database docking appears feasible and leads to better distinction of true ligands.  Simple model binding sites, such as L99A and its more polar variants, may find broad use in the development and testing of docking algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGral4WDhJW6jbVg90H21EOLACvtfcHk0lgKPwjXBffpIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xms1Wqsrs%253D&md5=040e009b33f125ab4abbb322cf32d2c9</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2802%2900777-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252802%252900777-5%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DB.%2BQ.%26aulast%3DBaase%26aufirst%3DW.%2BA.%26aulast%3DWeaver%26aufirst%3DL.%2BH.%26aulast%3DMatthews%26aufirst%3DB.%2BW.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DA%2520model%2520binding%2520site%2520for%2520testing%2520scoring%2520functions%2520in%2520molecular%2520docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2002%26volume%3D322%26spage%3D339%26epage%3D355" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Hopkins, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groom, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alex, A.</span><span> </span><span class="NLM_article-title">Ligand efficiency: a useful metric for lead selection</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">430</span><span class="NLM_x">–</span> <span class="NLM_lpage">431</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1016%2FS1359-6446%2804%2903069-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=15109945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2004&pages=430-431&author=A.+L.+Hopkinsauthor=C.+R.+Groomauthor=A.+Alex&title=Ligand+efficiency%3A+a+useful+metric+for+lead+selection"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Ligand efficiency: a useful metric for lead selection</span></div><div class="casAuthors">Hopkins Andrew L; Groom Colin R; Alex Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">430-1</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdMyKW_d6RPeIHSmpohpd0fW6udTcc2ebHwhJB87sNCrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2c3ht1ygtw%253D%253D&md5=eb19a1bca53247d0bd9adcb99d5817e6</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2804%2903069-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252804%252903069-7%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DGroom%26aufirst%3DC.%2BR.%26aulast%3DAlex%26aufirst%3DA.%26atitle%3DLigand%2520efficiency%253A%2520a%2520useful%2520metric%2520for%2520lead%2520selection%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2004%26volume%3D9%26spage%3D430%26epage%3D431" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Schuttelkopf, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hardy, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beverley, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, W. N.</span><span> </span><span class="NLM_article-title">Structures of <i>Leishmania major</i> Pteridine Reductase Complexes Reveal the Active Site Features Important for Ligand Binding and to Guide Inhibitor Design</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">352</span><span class="NLM_x">, </span> <span class="NLM_fpage">105</span><span class="NLM_x">–</span> <span class="NLM_lpage">116</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=352&publication_year=2005&pages=105-116&author=A.+W.+Schuttelkopfauthor=L.+W.+Hardyauthor=S.+M.+Beverleyauthor=W.+N.+Hunter&title=Structures+of+Leishmania+major+Pteridine+Reductase+Complexes+Reveal+the+Active+Site+Features+Important+for+Ligand+Binding+and+to+Guide+Inhibitor+Design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSchuttelkopf%26aufirst%3DA.%2BW.%26aulast%3DHardy%26aufirst%3DL.%2BW.%26aulast%3DBeverley%26aufirst%3DS.%2BM.%26aulast%3DHunter%26aufirst%3DW.%2BN.%26atitle%3DStructures%2520of%2520Leishmania%2520major%2520Pteridine%2520Reductase%2520Complexes%2520Reveal%2520the%2520Active%2520Site%2520Features%2520Important%2520for%2520Ligand%2520Binding%2520and%2520to%2520Guide%2520Inhibitor%2520Design%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2005%26volume%3D352%26spage%3D105%26epage%3D116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Chen, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lyne, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giordanetto, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lovell, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span> </span><span class="NLM_article-title">On evaluating molecular-docking methods for pose prediction and enrichment factors</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">401</span><span class="NLM_x">–</span> <span class="NLM_lpage">415</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci0503255" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlShtLvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=401-415&author=H.+Chenauthor=P.+D.+Lyneauthor=F.+Giordanettoauthor=T.+Lovellauthor=J.+Li&title=On+evaluating+molecular-docking+methods+for+pose+prediction+and+enrichment+factors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">On Evaluating Molecular-Docking Methods for Pose Prediction and Enrichment Factors</span></div><div class="casAuthors">Chen, Hongming; Lyne, Paul D.; Giordanetto, Fabrizio; Lovell, Timothy; Li, Jin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">401-415</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Four of the most well-known, com. available docking programs, FlexX, GOLD, GLIDE, and ICM, have been examd. for their ligand-docking and virtual-screening capabilities.  The relative performance of the programs in reproducing the native ligand conformation from starting SMILES strings for 164 high-resoln. protein-ligand complexes is presented and compared.  Applying only the native scoring functions, the latest versions of these four docking programs were also used to conduct virtual screening for 12 protein targets of therapeutic interest, involving both publicly available structures and AstraZeneca inhouse structures.  The capability of the four programs to correctly rank-order target-specific active compds. over alternative binders and nonbinders (decoys plus randomly selected compds.) and thereby enrich a small subset of a screening library is compared.  Enrichments from the virtual-screening expts. are contrasted with those obtained with alternative 3D shape-matching and 2D similarity database-search methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcdaTxux-OvLVg90H21EOLACvtfcHk0liUHawRV1X9pQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlShtLvE&md5=2bc9b31cb2dd5926fa8d5480f4129758</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fci0503255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci0503255%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DH.%26aulast%3DLyne%26aufirst%3DP.%2BD.%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DLovell%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DJ.%26atitle%3DOn%2520evaluating%2520molecular-docking%2520methods%2520for%2520pose%2520prediction%2520and%2520enrichment%2520factors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D401%26epage%3D415" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">McGaughey, G. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheridan, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bayly, C. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Culberson, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kreatsoulas, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindsley, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maiorov, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Truchon, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cornell, W. D.</span><span> </span><span class="NLM_article-title">Comparison of Topological, Shape, and Docking Methods in Virtual Screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1504</span><span class="NLM_x">–</span> <span class="NLM_lpage">1519</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci700052x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmvVOktLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=1504-1519&author=G.+B.+McGaugheyauthor=R.+P.+Sheridanauthor=C.+I.+Baylyauthor=J.+C.+Culbersonauthor=C.+Kreatsoulasauthor=S.+Lindsleyauthor=V.+Maiorovauthor=J.+F.+Truchonauthor=W.+D.+Cornell&title=Comparison+of+Topological%2C+Shape%2C+and+Docking+Methods+in+Virtual+Screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Topological, Shape, and Docking Methods in Virtual Screening</span></div><div class="casAuthors">McGaughey, Georgia B.; Sheridan, Robert P.; Bayly, Christopher I.; Culberson, J. Chris; Kreatsoulas, Constantine; Lindsley, Stacey; Maiorov, Vladimir; Truchon, Jean-Francois; Cornell, Wendy D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1504-1519</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Virtual screening benchmarking studies were carried out on 11 targets to evaluate the performance of three commonly used approaches: 2D ligand similarity (Daylight, TOPOSIM), 3D ligand similarity (SQW, ROCS), and protein structure-based docking (FLOG, FRED, Glide).  Active and decoy compd. sets were assembled from both the MDDR and the Merck compd. databases.  Averaged over multiple targets, ligand-based methods outperformed docking algorithms.  This was true for 3D ligand-based methods only when chem. typing was included.  Using mean enrichment factor as a performance metric, Glide appears to be the best docking method among the three with FRED a close second.  Results for all virtual screening methods are database dependent and can vary greatly for particular targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozxILlzTNAErVg90H21EOLACvtfcHk0liUHawRV1X9pQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmvVOktLk%253D&md5=d76e6a531097729350e70ffc8d1782f2</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fci700052x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci700052x%26sid%3Dliteratum%253Aachs%26aulast%3DMcGaughey%26aufirst%3DG.%2BB.%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DBayly%26aufirst%3DC.%2BI.%26aulast%3DCulberson%26aufirst%3DJ.%2BC.%26aulast%3DKreatsoulas%26aufirst%3DC.%26aulast%3DLindsley%26aufirst%3DS.%26aulast%3DMaiorov%26aufirst%3DV.%26aulast%3DTruchon%26aufirst%3DJ.%2BF.%26aulast%3DCornell%26aufirst%3DW.%2BD.%26atitle%3DComparison%2520of%2520Topological%252C%2520Shape%252C%2520and%2520Docking%2520Methods%2520in%2520Virtual%2520Screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D1504%26epage%3D1519" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Tan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geppert, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sisay, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gutschow, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajorath, J.</span><span> </span><span class="NLM_article-title">Integrating Structure- and Ligand-Based Virtual Screening: Comparison of Individual, Parallel, and Fused Molecular Docking and Similarity Search Calculations on Multiple Targets</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1566</span><span class="NLM_x">–</span> <span class="NLM_lpage">1571</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1002%2Fcmdc.200800129" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=18651695" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOgt7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=1566-1571&author=L.+Tanauthor=H.+Geppertauthor=M.+T.+Sisayauthor=M.+Gutschowauthor=J.+Bajorath&title=Integrating+Structure-+and+Ligand-Based+Virtual+Screening%3A+Comparison+of+Individual%2C+Parallel%2C+and+Fused+Molecular+Docking+and+Similarity+Search+Calculations+on+Multiple+Targets"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Integrating structure- and ligand-based virtual screening: comparison of individual, parallel, and fused molecular docking and similarity search calculations on multiple targets</span></div><div class="casAuthors">Tan, Lu; Geppert, Hanna; Sisay, Mihiret T.; Guetschow, Michael; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1566-1571</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Similarity searching is often used to preselect compds. for docking, thereby decreasing the size of screening databases.  However, integrated structure- and ligand-based screening schemes are rare at present.  Docking and similarity search calcns. using 2D fingerprints were carried out in a comparative manner on nine target enzymes, for which significant nos. of diverse inhibitors could be obtained.  In the absence of knowledge-based docking constraints and target-directed parameter optimization, fingerprint searching displayed a clear preference over docking calcns.  Alternative combinations of docking and similarity search results were investigated and found to further increase compd. recall of individual methods in a no. of instances.  When the results of similarity searching and docking were combined, parallel selection of candidate compds. from individual rankings was generally superior to rank fusion.  We suggest that complementary results from docking and similarity searching can be captured by integrated compd. selection schemes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIyWJxL_1eGrVg90H21EOLACvtfcHk0liUHawRV1X9pQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOgt7rM&md5=fa4690d0059bb144b57a89c5e88be93a</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200800129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200800129%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DL.%26aulast%3DGeppert%26aufirst%3DH.%26aulast%3DSisay%26aufirst%3DM.%2BT.%26aulast%3DGutschow%26aufirst%3DM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DIntegrating%2520Structure-%2520and%2520Ligand-Based%2520Virtual%2520Screening%253A%2520Comparison%2520of%2520Individual%252C%2520Parallel%252C%2520and%2520Fused%2520Molecular%2520Docking%2520and%2520Similarity%2520Search%2520Calculations%2520on%2520Multiple%2520Targets%26jtitle%3DChemMedChem%26date%3D2008%26volume%3D3%26spage%3D1566%26epage%3D1571" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">von Korff, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freyss, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sander, T.</span><span> </span><span class="NLM_article-title">Comparison of Ligand- and Structure-Based Virtual Screening on the DUD Data Set</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">209</span><span class="NLM_x">–</span> <span class="NLM_lpage">231</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci800303k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1agtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=209-231&author=M.+von+Korffauthor=J.+Freyssauthor=T.+Sander&title=Comparison+of+Ligand-+and+Structure-Based+Virtual+Screening+on+the+DUD+Data+Set"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of Ligand- and Structure-Based Virtual Screening on the DUD Data Set</span></div><div class="casAuthors">von Korff, Modest; Freyss, Joel; Sander, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">209-231</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Several inhouse developed descriptors and our inhouse docking tool ActDock were compared with virtual screening on the data set of useful decoys (DUD).  The results were compared with the chem. fingerprint descriptor from ChemAxon and with the docking results of the original DUD publication.  The DUD is the first published data set providing active mols., decoys, and refs. for crystal structures of ligand-target complexes.  The DUD was designed for the purpose of evaluating docking programs.  It contains 2950 active compds. against a total of 40 target proteins.  Furthermore, for every ligand the data set contains 36 structurally dissimilar decoy compds. with similar physicochem. properties.  We extd. the ligands from the target proteins to extend the applicability of the data set to include ligand based virtual screening.  From the 40 target proteins, 37 contained ligands that we used as query mols. for virtual screening evaluation.  With this data set a large comparison was done between four different chem. fingerprints, a topol. pharmacophore descriptor, the Flexophore descriptor, and ActDock.  The Actelion docking tool relies on a MM2 force field and a pharmacophore point interaction statistic for scoring; the details are described in this publication.  In terms of enrichment rates the chem. fingerprint descriptors performed better than the Flexophore and the docking tool.  After removing mols. chem. similar to the query mols. the Flexophore descriptor outperformed the chem. descriptors and the topol. pharmacophore descriptors.  With the similarity matrix calcns. used in this study it was shown that the Flexophore is well suited to find new chem. entities via "scaffold hopping".  The Flexophore descriptor can be explored with a Java applet at http://www.cheminformatics.ch in the submenu Tools→Flexophore.  Its usage is free of charge and does not require registration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9OO4y7ZdpcbVg90H21EOLACvtfcHk0liumniMN_ap5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1agtLY%253D&md5=8a530b53359d924f00f79d7d92302f8e</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fci800303k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800303k%26sid%3Dliteratum%253Aachs%26aulast%3Dvon%2BKorff%26aufirst%3DM.%26aulast%3DFreyss%26aufirst%3DJ.%26aulast%3DSander%26aufirst%3DT.%26atitle%3DComparison%2520of%2520Ligand-%2520and%2520Structure-Based%2520Virtual%2520Screening%2520on%2520the%2520DUD%2520Data%2520Set%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D209%26epage%3D231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Verdonk, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berdini, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartshorn, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mooij, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, P.</span><span> </span><span class="NLM_article-title">Virtual screening using protein−ligand docking: avoiding artificial enrichment</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">793</span><span class="NLM_x">–</span> <span class="NLM_lpage">806</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci034289q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVGhs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2004&pages=793-806&author=M.+L.+Verdonkauthor=V.+Berdiniauthor=M.+J.+Hartshornauthor=W.+T.+Mooijauthor=C.+W.+Murrayauthor=R.+D.+Taylorauthor=P.+Watson&title=Virtual+screening+using+protein%E2%88%92ligand+docking%3A+avoiding+artificial+enrichment"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening using protein-ligand docking: Avoiding artificial enrichment</span></div><div class="casAuthors">Verdonk, Marcel L.; Berdini, Valerio; Hartshorn, Michael J.; Mooij, Wijnand T. M.; Murray, Christopher W.; Taylor, Richard D.; Watson, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">793-806</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This study addresses a no. of topical issues around the use of protein-ligand docking in virtual screening.  We show that, for the validation of such methods, it is key to use focused libraries (contg. compds. with one-dimensional properties, similar to the actives), rather than "random" or "drug-like" libraries to test the actives against.  We also show that, to obtain good enrichments, the docking program needs to produce reliable binding modes.  We demonstrate how pharmacophores can be used to guide the dockings and improve enrichments, and we compare the performance of three consensus-ranking protocols against ranking based on individual scoring functions.  Finally, we show that protein-ligand docking can be an effective aid in the screening for weak, fragment-like binders, which has rapidly become a popular strategy for hit identification.  All results presented are based on carefully constructed virtual screening expts. against four targets, using the protein-ligand docking program GOLD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRRkYiB4qsvbVg90H21EOLACvtfcHk0liumniMN_ap5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVGhs70%253D&md5=b0cd9e9a04ddebe401dee9744e2184cb</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fci034289q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci034289q%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DBerdini%26aufirst%3DV.%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26aulast%3DMooij%26aufirst%3DW.%2BT.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DTaylor%26aufirst%3DR.%2BD.%26aulast%3DWatson%26aufirst%3DP.%26atitle%3DVirtual%2520screening%2520using%2520protein%25E2%2588%2592ligand%2520docking%253A%2520avoiding%2520artificial%2520enrichment%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2004%26volume%3D44%26spage%3D793%26epage%3D806" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Marcou, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Optimizing fragment and scaffold docking by use of molecular interaction fingerprints</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">195</span><span class="NLM_x">–</span> <span class="NLM_lpage">207</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci600342e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht12iurzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=195-207&author=G.+Marcouauthor=D.+Rognan&title=Optimizing+fragment+and+scaffold+docking+by+use+of+molecular+interaction+fingerprints"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Optimizing Fragment and Scaffold Docking by Use of Molecular Interaction Fingerprints</span></div><div class="casAuthors">Marcou, Gilles; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">195-207</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein-ligand interaction fingerprints have been used to postprocess docking poses of three ligand data sets: a set of 40 low-mol.-wt. compds. from the Protein Data Bank, a collection of 40 scaffolds from pharmaceutically relevant protein ligands, and a database of 19 scaffolds extd. from true cdk2 inhibitors seeded in 2230 scaffold decoys.  Four popular docking tools (FlexX, Glide, Gold, and Surflex) were used to generate poses for ligands of the three data sets.  In all cases, scoring by the similarity of interaction fingerprints to a given ref. was statistically superior to conventional scoring functions in posing low-mol.-wt. fragments, predicting protein-bound scaffold coordinates according to the known binding mode of related ligands, and screening a scaffold library to enrich a hit list in true cdk2-targeted scaffolds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHLbVKxK-BfLVg90H21EOLACvtfcHk0liumniMN_ap5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht12iurzL&md5=0c9137a39d40fbcc83546aec17b595ba</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fci600342e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600342e%26sid%3Dliteratum%253Aachs%26aulast%3DMarcou%26aufirst%3DG.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DOptimizing%2520fragment%2520and%2520scaffold%2520docking%2520by%2520use%2520of%2520molecular%2520interaction%2520fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D195%26epage%3D207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Mpamhanga, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLay, I. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willett, P.</span><span> </span><span class="NLM_article-title">Knowledge-based interaction fingerprint scoring: a simple method for improving the effectiveness of fast scoring functions</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">686</span><span class="NLM_x">–</span> <span class="NLM_lpage">698</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci050420d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD28XpsVGjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=686-698&author=C.+P.+Mpamhangaauthor=B.+Chenauthor=I.+M.+McLayauthor=P.+Willett&title=Knowledge-based+interaction+fingerprint+scoring%3A+a+simple+method+for+improving+the+effectiveness+of+fast+scoring+functions"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Knowledge-Based Interaction Fingerprint Scoring: A Simple Method for Improving the Effectiveness of Fast Scoring Functions</span></div><div class="casAuthors">Mpamhanga, Chidochangu P.; Chen, Beining; McLay, Iain M.; Willett, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">686-698</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new method for the postprocessing of docking outputs has been developed, based on encoding putative 3D binding modes (docking solns.) as ligand-protein interactions into simple bit strings, a method analogous to the structural interaction fingerprint.  Instead of employing traditional scoring functions, the method uses a series of new, knowledge-based scores derived from the similarity of the bit strings for each docking soln. to that of a known ref. binding mode.  A GOLD docking study was carried out using the Bissantz estrogen receptor antagonist set along with the new scoring method.  Superior recovery rates, with up to 2-fold enrichments, were obsd. when the new knowledge-based scoring was compared to the GOLD fitness score.  In addn., top ranking sets of mols. (actives and potential actives or decoys) were structurally diverse with low mol. wts. and structural complexities.  Principal component anal. and clustering of the fingerprints permits the easy sepn. of active from inactive binding modes and the visualization of diverse binding modes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZfTfk88wvHLVg90H21EOLACvtfcHk0ljqLGooHEZm5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpsVGjuw%253D%253D&md5=0fecc7a1894b8bf0e53f077deefc2a85</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fci050420d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci050420d%26sid%3Dliteratum%253Aachs%26aulast%3DMpamhanga%26aufirst%3DC.%2BP.%26aulast%3DChen%26aufirst%3DB.%26aulast%3DMcLay%26aufirst%3DI.%2BM.%26aulast%3DWillett%26aufirst%3DP.%26atitle%3DKnowledge-based%2520interaction%2520fingerprint%2520scoring%253A%2520a%2520simple%2520method%2520for%2520improving%2520the%2520effectiveness%2520of%2520fast%2520scoring%2520functions%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D686%26epage%3D698" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Virtual ligand screening: strategies, perspectives and limitations</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">580</span><span class="NLM_x">–</span> <span class="NLM_lpage">594</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1016%2Fj.drudis.2006.05.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=16793526" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvFGqtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=580-594&author=G.+Klebe&title=Virtual+ligand+screening%3A+strategies%2C+perspectives+and+limitations"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual ligand screening: strategies, perspectives and limitations</span></div><div class="casAuthors">Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">13 & 14</span>),
    <span class="NLM_cas:pages">580-594</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In contrast to high-throughput screening, in virtual ligand screening (VS), compds. are selected using computer programs to predict their binding to a target receptor.  A key prerequisite is knowledge about the spatial and energetic criteria responsible for protein-ligand binding.  The concepts and prerequisites to perform VS are summarized here, and explanations are sought for the enduring limitations of the technol.  Target selection, anal. and prepn. are discussed, as well as considerations about the compilation of candidate ligand libraries.  The tools and strategies of a VS campaign, and the accuracy of scoring and ranking of the results, are also considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN3dWlP-WDY7Vg90H21EOLACvtfcHk0ljqLGooHEZm5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvFGqtLo%253D&md5=5adfd125d48082238a6c5ad0e8343c59</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2006.05.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2006.05.012%26sid%3Dliteratum%253Aachs%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DVirtual%2520ligand%2520screening%253A%2520strategies%252C%2520perspectives%2520and%2520limitations%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2006%26volume%3D11%26spage%3D580%26epage%3D594" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Mancera, R. L.</span><span> </span><span class="NLM_article-title">Molecular modeling of hydration in drug design</span> <span class="citation_source-journal">Curr. Opin. Drug Discovery Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">275</span><span class="NLM_x">–</span> <span class="NLM_lpage">280</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=17554853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmtVWlsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2007&pages=275-280&author=R.+L.+Mancera&title=Molecular+modeling+of+hydration+in+drug+design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular modeling of hydration in drug design</span></div><div class="casAuthors">Mancera, Ricardo L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Drug Discovery & Development</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">275-280</span>CODEN:
                <span class="NLM_cas:coden">CODDFF</span>;
        ISSN:<span class="NLM_cas:issn">1367-6733</span>.
    
            (<span class="NLM_cas:orgname">Thomson Scientific</span>)
        </div><div class="casAbstract">A review.  The accurate prediction of the binding mode and affinity of a drug mol. to its putative receptor is crucial for successful drug design and optimization.  Modeling the role of the aq. solvent in ligand-protein interactions is one of the key components.  Three main computational techniques have been developed to model hydration: free energy methods, ligand-protein docking and scoring, and the explicit inclusion of tightly bound water mols. in modeling.  This review provides a summary of the most recent developments in these approaches used for the mol. modeling of hydration effects in the context of computational structure-based drug design methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5oaWakeqXu7Vg90H21EOLACvtfcHk0ljqLGooHEZm5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmtVWlsrg%253D&md5=00f8343ef349b19271d7199dd048d215</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMancera%26aufirst%3DR.%2BL.%26atitle%3DMolecular%2520modeling%2520of%2520hydration%2520in%2520drug%2520design%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Dev.%26date%3D2007%26volume%3D10%26spage%3D275%26epage%3D280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Sousa, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fernandes, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramos, M. J.</span><span> </span><span class="NLM_article-title">Protein-ligand docking: current status and future challenges</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">15</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2006&pages=15-26&author=S.+F.+Sousaauthor=P.+A.+Fernandesauthor=M.+J.+Ramos&title=Protein-ligand+docking%3A+current+status+and+future+challenges"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSousa%26aufirst%3DS.%2BF.%26aulast%3DFernandes%26aufirst%3DP.%2BA.%26aulast%3DRamos%26aufirst%3DM.%2BJ.%26atitle%3DProtein-ligand%2520docking%253A%2520current%2520status%2520and%2520future%2520challenges%26jtitle%3DProteins%26date%3D2006%26volume%3D65%26spage%3D15%26epage%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Babaoglu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Deconstructing fragment-based inhibitor discovery</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">720</span><span class="NLM_x">–</span> <span class="NLM_lpage">723</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1038%2Fnchembio831" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=17072304" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1WqurbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=720-723&author=K.+Babaogluauthor=B.+K.+Shoichet&title=Deconstructing+fragment-based+inhibitor+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Deconstructing fragment-based inhibitor discovery</span></div><div class="casAuthors">Babaoglu, Kerim; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">720-723</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Fragment-based screens test multiple low-mol. wt. mols. for binding to a target.  Fragments often bind with low affinities but typically have better ligand efficiencies (ΔGbind /heavy atom count) than traditional screening hits.  This efficiency, combined with accompanying at.-resoln. structures, has made fragments popular starting points for drug discovery programs.  Fragment-based design adopts a constructive strategy: affinity is enhanced either by cycles of functional-group addn. or by joining two independent fragments together.  The final inhibitor is expected to adopt the same geometry as the original fragment hit.  Here the authors consider whether the inverse, deconstructive logic also applies-can one always parse a higher-affinity inhibitor into fragments that recapitulate the binding geometry of the larger mol..  Cocrystal structures of fragments deconstructed from a known β-lactamase inhibitor suggest that this is not always the case.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVl5_M15h_rLVg90H21EOLACvtfcHk0ljIl71js_Eb_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1WqurbM&md5=de0d5632f6354314794c2f45cfdd2efd</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnchembio831&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio831%26sid%3Dliteratum%253Aachs%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DDeconstructing%2520fragment-based%2520inhibitor%2520discovery%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2006%26volume%3D2%26spage%3D720%26epage%3D723" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Hajduk, P. J.</span><span> </span><span class="NLM_article-title">Puzzling through fragment-based drug design</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">–</span> <span class="NLM_lpage">659</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1038%2Fnchembio1206-658" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=17108979" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1SrtLbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=658-659&author=P.+J.+Hajduk&title=Puzzling+through+fragment-based+drug+design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Puzzling through fragment-based drug design</span></div><div class="casAuthors">Hajduk, Philip J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">658-659</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Fragment-based drug design capitalizes on the modular binding of low-mol.-wt., low-affinity ligands.  However, the deconstruction of lead-like inhibitors into putative fragments reveals the surprising complexity of dealing with low-affinity leads, thereby challenging oversimplification of these leads and highlighting the richness of their chem. diversity and mol. recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHhksf7gx20LVg90H21EOLACvtfcHk0ljIl71js_Eb_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1SrtLbI&md5=f509ade062cbb89fa77c6273eb280089</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fnchembio1206-658&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio1206-658%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26atitle%3DPuzzling%2520through%2520fragment-based%2520drug%2520design%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2006%26volume%3D2%26spage%3D658%26epage%3D659" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Brenk, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vetter, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyce, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodin, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Probing molecular docking in a charged model binding site</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">357</span><span class="NLM_x">, </span> <span class="NLM_fpage">1449</span><span class="NLM_x">–</span> <span class="NLM_lpage">1470</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=357&publication_year=2006&pages=1449-1470&author=R.+Brenkauthor=S.+W.+Vetterauthor=S.+E.+Boyceauthor=D.+B.+Goodinauthor=B.+K.+Shoichet&title=Probing+molecular+docking+in+a+charged+model+binding+site"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBrenk%26aufirst%3DR.%26aulast%3DVetter%26aufirst%3DS.%2BW.%26aulast%3DBoyce%26aufirst%3DS.%2BE.%26aulast%3DGoodin%26aufirst%3DD.%2BB.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DProbing%2520molecular%2520docking%2520in%2520a%2520charged%2520model%2520binding%2520site%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D357%26spage%3D1449%26epage%3D1470" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Gerber, P. R.</span><span> </span><span class="NLM_article-title">Charge distribution from a simple molecular orbital type calculation and nonbonding interaction terms in the force field MAB</span> <span class="citation_source-journal">J. Comput.-Aided. Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">37</span><span class="NLM_x">–</span> <span class="NLM_lpage">51</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1998&pages=37-51&author=P.+R.+Gerber&title=Charge+distribution+from+a+simple+molecular+orbital+type+calculation+and+nonbonding+interaction+terms+in+the+force+field+MAB"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGerber%26aufirst%3DP.%2BR.%26atitle%3DCharge%2520distribution%2520from%2520a%2520simple%2520molecular%2520orbital%2520type%2520calculation%2520and%2520nonbonding%2520interaction%2520terms%2520in%2520the%2520force%2520field%2520MAB%26jtitle%3DJ.%2520Comput.-Aided.%2520Mol.%2520Des.%26date%3D1998%26volume%3D12%26spage%3D37%26epage%3D51" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Gerber, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, K.</span><span> </span><span class="NLM_article-title">MAB, a generally applicable molecular force field for structure modelling in medicinal chemistry</span> <span class="citation_source-journal">J. Comput.-Aided. Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">251</span><span class="NLM_x">–</span> <span class="NLM_lpage">268</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1995&pages=251-268&author=P.+R.+Gerberauthor=K.+M%C3%BCller&title=MAB%2C+a+generally+applicable+molecular+force+field+for+structure+modelling+in+medicinal+chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGerber%26aufirst%3DP.%2BR.%26aulast%3DM%25C3%25BCller%26aufirst%3DK.%26atitle%3DMAB%252C%2520a%2520generally%2520applicable%2520molecular%2520force%2520field%2520for%2520structure%2520modelling%2520in%2520medicinal%2520chemistry%26jtitle%3DJ.%2520Comput.-Aided.%2520Mol.%2520Des.%26date%3D1995%26volume%3D9%26spage%3D251%26epage%3D268" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Ertl, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rohde, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Selzer, P.</span><span> </span><span class="NLM_article-title">Fast calculation of molecular polar surface area as a sum of fragment-based contributions and its application to the prediction of drug transport properties</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">3714</span><span class="NLM_x">–</span> <span class="NLM_lpage">3717</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm000942e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmsFeitb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=3714-3717&author=P.+Ertlauthor=B.+Rohdeauthor=P.+Selzer&title=Fast+calculation+of+molecular+polar+surface+area+as+a+sum+of+fragment-based+contributions+and+its+application+to+the+prediction+of+drug+transport+properties"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Fast Calculation of Molecular Polar Surface Area as a Sum of Fragment-Based Contributions and Its Application to the Prediction of Drug Transport Properties</span></div><div class="casAuthors">Ertl, Peter; Rohde, Bernhard; Selzer, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3714-3717</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. polar surface area (PSA), i.e., surface belonging to polar atoms, is a descriptor that was shown to correlate well with passive mol. transport through membranes and, therefore, allows prediction of transport properties of drugs.  The calcn. of PSA, however, is rather time-consuming because of the necessity to generate a reasonable 3D mol. geometry and the calcn. of the surface itself.  A new approach for the calcn. of the PSA is presented here, based on the summation of tabulated surface contributions of polar fragments.  The method, termed topol. PSA (TPSA), provides results which are practically identical with the 3D PSA (the correlation coeff. between 3D PSA and fragment-based TPSA for 34 810 mols. from the World Drug Index is 0.99), while the computation speed is 2-3 orders of magnitude faster.  The new methodol. may, therefore, be used for fast bioavailability screening of virtual libraries having millions of mols.  This article describes the new methodol. and shows the results of validation studies based on sets of published absorption data, including intestinal absorption, Caco-2 monolayer penetration, and blood-brain barrier penetration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhP6Qncx_h6LVg90H21EOLACvtfcHk0liVGpYX1G1gbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmsFeitb4%253D&md5=23ae4e7708bd8ddeb4a24ad160a29bd6</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm000942e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm000942e%26sid%3Dliteratum%253Aachs%26aulast%3DErtl%26aufirst%3DP.%26aulast%3DRohde%26aufirst%3DB.%26aulast%3DSelzer%26aufirst%3DP.%26atitle%3DFast%2520calculation%2520of%2520molecular%2520polar%2520surface%2520area%2520as%2520a%2520sum%2520of%2520fragment-based%2520contributions%2520and%2520its%2520application%2520to%2520the%2520prediction%2520of%2520drug%2520transport%2520properties%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D3714%26epage%3D3717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Kuzmic, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sideris, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cregar, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elrod, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janc, J. W.</span><span> </span><span class="NLM_article-title">High-throughput screening of enzyme inhibitors: automatic determination of tight-binding inhibition constants</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">281</span><span class="NLM_x">, </span> <span class="NLM_fpage">62</span><span class="NLM_x">–</span> <span class="NLM_lpage">67</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2000&pages=62-67&author=P.+Kuzmicauthor=S.+Siderisauthor=L.+M.+Cregarauthor=K.+C.+Elrodauthor=K.+D.+Riceauthor=J.+W.+Janc&title=High-throughput+screening+of+enzyme+inhibitors%3A+automatic+determination+of+tight-binding+inhibition+constants"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKuzmic%26aufirst%3DP.%26aulast%3DSideris%26aufirst%3DS.%26aulast%3DCregar%26aufirst%3DL.%2BM.%26aulast%3DElrod%26aufirst%3DK.%2BC.%26aulast%3DRice%26aufirst%3DK.%2BD.%26aulast%3DJanc%26aufirst%3DJ.%2BW.%26atitle%3DHigh-throughput%2520screening%2520of%2520enzyme%2520inhibitors%253A%2520automatic%2520determination%2520of%2520tight-binding%2520inhibition%2520constants%26jtitle%3DAnal.%2520Biochem.%26date%3D2000%26volume%3D281%26spage%3D62%26epage%3D67" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Leslie, A. G.</span><span> </span><span class="NLM_article-title">The integration of macromolecular diffraction data</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">48</span><span class="NLM_x">–</span> <span class="NLM_lpage">57</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1107%2FS0907444905039107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=16369093" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlant7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=48-57&author=A.+G.+Leslie&title=The+integration+of+macromolecular+diffraction+data"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">The integration of macromolecular diffraction data</span></div><div class="casAuthors">Leslie, Andrew G. W.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">48-57</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The objective of any modern data-processing program is to produce from a set of diffraction images a set of indexes (hkls) with their assocd. intensities (and ests. of their uncertainties), together with an accurate est. of the crystal unit-cell parameters.  This procedure should not only be reliable, but should involve an abs. min. of user intervention.  The process can be conveniently divided into three stages.  The first (autoindexing) dets. the unit-cell parameters and the orientation of the crystal.  The unit-cell parameters may indicate the likely Laue group of the crystal.  The second step is to refine the initial est. of the unit-cell parameters and also the crystal mosaicity using a procedure known as post-refinement.  The third step is to integrate the images, which consists of predicting the positions of the Bragg reflections on each image and obtaining an est. of the intensity of each reflection and its uncertainty.  This is carried out while simultaneously refining various detector and crystal parameters.  Basic features of the algorithms employed for each of these three sep. steps are described, principally with ref. to the program MOSFLM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoO8D4_R2Ly97Vg90H21EOLACvtfcHk0liVGpYX1G1gbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlant7vP&md5=70d65687f3b3426c93e96bfb706e0c5c</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1107%2FS0907444905039107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444905039107%26sid%3Dliteratum%253Aachs%26aulast%3DLeslie%26aufirst%3DA.%2BG.%26atitle%3DThe%2520integration%2520of%2520macromolecular%2520diffraction%2520data%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D48%26epage%3D57" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Evans, P.</span><span> </span><span class="NLM_article-title">Scaling and assessment of data quality</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">72</span><span class="NLM_x">–</span> <span class="NLM_lpage">82</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1107%2FS0907444905036693" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=16369096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=72-82&author=P.+Evans&title=Scaling+and+assessment+of+data+quality"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Scaling and assessment of data quality</span></div><div class="casAuthors">Evans, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">72-82</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">The various phys. factors affecting measured diffraction intensities are discussed, as are the scaling models which may be used to put the data on a consistent scale.  After scaling, the intensities can be analyzed to set the real resoln. of the data set, to detect bad regions (e.g. bad images), to analyze radiation damage and to assess the overall quality of the data set.  The significance of any anomalous signal may be assessed by probability and correlation anal.  The algorithms used by the CCP4 scaling program SCALA are described.  A requirement for the scaling and merging of intensities is knowledge of the Laue group and point-group symmetries: the possible symmetry of the diffraction pattern may be detd. from scores such as correlation coeffs. between observations which might be symmetry-related.  These scoring functions are implemented in a new program POINTLESS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGravLSLU62RhLVg90H21EOLACvtfcHk0liVGpYX1G1gbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlant7jM&md5=293d3876e534c0c57813990515bb3c76</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1107%2FS0907444905036693&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444905036693%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%26atitle%3DScaling%2520and%2520assessment%2520of%2520data%2520quality%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D72%26epage%3D82" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Collaborative Computational Project Number 4</span><span> </span><span class="NLM_article-title">The CCP4 suite: programs for protein crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">760</span><span class="NLM_x">–</span> <span class="NLM_lpage">763</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=15299374" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-763&author=Collaborative+Computational+Project+Number+4&title=The+CCP4+suite%3A+programs+for+protein+crystallography"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%2520suite%253A%2520programs%2520for%2520protein%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Otwinowski, Z.; Minor, W.</span><span> </span><span class="NLM_article-title">Processing of X-ray Diffraction Data Collected in Oscillation Mode</span>. In  <span class="citation_source-book">Methods in Enzymology</span>; <span class="NLM_contrib-group">Carter, C. W. J.; Sweet, R. M.</span>, Eds.; <span class="NLM_publisher-name">Academic Press</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">1997</span><span class="NLM_x">; </span>Vol.  <span class="NLM_volume">276</span>, pp  <span class="NLM_fpage">307</span>− <span class="NLM_lpage">326</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1997&pages=307-326&author=Z.+Otwinowski&author=W.+Minorauthor=C.+W.+J.+Carter&author=R.+M.+Sweet&title=Methods+in+Enzymology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26atitle%3DProcessing%2520of%2520X-ray%2520Diffraction%2520Data%2520Collected%2520in%2520Oscillation%2520Mode%26btitle%3DMethods%2520in%2520Enzymology%26aulast%3DCarter%26aufirst%3DC.%2BW.%2BJ.%26pub%3DAcademic%2520Press%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Vagin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teplyakov, A.</span><span> </span><span class="NLM_article-title">MOLREP: an automated program for molecular replacement</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1022</span><span class="NLM_x">–</span> <span class="NLM_lpage">1025</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1107%2FS0021889897006766" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1997&pages=1022-1025&author=A.+Vaginauthor=A.+Teplyakov&title=MOLREP%3A+an+automated+program+for+molecular+replacement"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">MOLREP: an automated program for molecular replacement</span></div><div class="casAuthors">Vagin, Alexei; Teplyakov, Alexei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1022-1025</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">MOLREP is an automated program for mol. replacement which uses effective new approaches in data processing and rotational and translational searching.  These include an automatic choice of all parameters, scaling by Patterson origin peaks and soft resoln. cutoff.  One of the cornerstones of the program is an original full-symmetry translation function combined with a packing function.  Information from the model already placed in the cell is incorporated in both translation and packing functions.  A no. of tests using exptl. data proved the ability of the program to find the correct soln. in difficult cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkGXdZ-nTKxLVg90H21EOLACvtfcHk0ljVXrcTyrRjNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhslehu7Y%253D&md5=96af64aedd8a8f018c3c18b57a6b4b21</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1107%2FS0021889897006766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889897006766%26sid%3Dliteratum%253Aachs%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DTeplyakov%26aufirst%3DA.%26atitle%3DMOLREP%253A%2520an%2520automated%2520program%2520for%2520molecular%2520replacement%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D1997%26volume%3D30%26spage%3D1022%26epage%3D1025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">255</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0ljVXrcTyrRjNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isupov, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span> </span><span class="NLM_article-title">Use of TLS parameters to model anisotropic displacements in macromolecular refinement</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">122</span><span class="NLM_x">–</span> <span class="NLM_lpage">133</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1107%2FS0907444900014736" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=11134934" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD3MXkvVyj" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2001&pages=122-133&author=M.+D.+Winnauthor=M.+N.+Isupovauthor=G.+N.+Murshudov&title=Use+of+TLS+parameters+to+model+anisotropic+displacements+in+macromolecular+refinement"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Use of TLS parameters to model anisotropic displacements in macromolecular refinement</span></div><div class="casAuthors">Winn, M. D.; Isupov, M. N.; Murshudov, G. N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">D57</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">122-133</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard International Publishers Ltd.</span>)
        </div><div class="casAbstract">An essential step in macromol. refinement is the selection of model parameters which give as good a description of the exptl. data as possible while retaining a realistic data-to-parameter ratio.  This is particularly true of the choice of at. displacement parameters, where the move from individual isotropic to individual anisotropic refinement involves a sixfold increase in the no. of required displacement parameters.  The no. of refinement parameters can be reduced by using collective variables rather than independent at. variables and one of the simplest examples of this is the TLS parameterization for describing the translation, libration and screw-rotation displacements of a pseudo-rigid body.  This article describes the implementation of the TLS parameterization in the macromol. refinement program REFMAC.  Derivs. of the residual with respect to the TLS parameters are expanded in terms of the derivs. with respect to individual anisotropic U values, which in turn are calcd. using a fast Fourier transform technique.  TLS refinement is therefore fast and can be used routinely.  Examples of TLS refinement are given for glyceraldehyde-3-phosphate dehydrogenase (GAPDH) and a transcription activator GerE, for both of which there is data to only 2.0 Å, so that individual anisotropic refinement is not feasible.  GAPDH has been refined with between one and four TLS groups in the asym. unit and GerE with six TLS groups.  In both cases, inclusion of TLS parameters gives improved refinement statistics and in particular an improvement in R and free R values of several percent.  Furthermore, GAPDH and GerE have two and six mols. in the asym. unit, resp., and in each case the displacement parameters differ significantly between mols.  These differences are well accounted for by the TLS parameterization, leaving residual local displacements which are very similar between mols. and to which NCS restraints can be applied.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKZSHF9jjKo7Vg90H21EOLACvtfcHk0ljVXrcTyrRjNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXkvVyj&md5=ee69ba99f66fcf18fd7ef49bb8d9b779</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1107%2FS0907444900014736&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444900014736%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DIsupov%26aufirst%3DM.%2BN.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DUse%2520of%2520TLS%2520parameters%2520to%2520model%2520anisotropic%2520displacements%2520in%2520macromolecular%2520refinement%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2001%26volume%3D57%26spage%3D122%26epage%3D133" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Schuttelkopf, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Aalten, D. M. F.</span><span> </span><span class="NLM_article-title">PRODRG: a tool for high-throughput crystallography of protein−ligand complexes</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">1355</span><span class="NLM_x">–</span> <span class="NLM_lpage">1363</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=1355-1363&author=A.+W.+Schuttelkopfauthor=D.+M.+F.+van+Aalten&title=PRODRG%3A+a+tool+for+high-throughput+crystallography+of+protein%E2%88%92ligand+complexes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSchuttelkopf%26aufirst%3DA.%2BW.%26aulast%3Dvan%2BAalten%26aufirst%3DD.%2BM.%2BF.%26atitle%3DPRODRG%253A%2520a%2520tool%2520for%2520high-throughput%2520crystallography%2520of%2520protein%25E2%2588%2592ligand%2520complexes%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D1355%26epage%3D1363" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Fjm900414x&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Fjm900414x&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Fjm900414x&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0li50l06Ju1Gfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Iemura, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hori, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohtaka, H.</span><span> </span><span class="NLM_article-title">Syntheses of the Metabolites of 1-(2-Ethoxyethyl)-2-(Hexahydro-4-Methyl-1<i>h-</i>1,4-Diazepin-1-Yl)-1<i>h</i>-Benzimidazole Difumarate (kg-2413) and Related Compounds</span> <span class="citation_source-journal">Chem. Pharm. Bull.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">962</span><span class="NLM_x">–</span> <span class="NLM_lpage">966</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1989&pages=962-966&author=R.+Iemuraauthor=M.+Horiauthor=H.+Ohtaka&title=Syntheses+of+the+Metabolites+of+1-%282-Ethoxyethyl%29-2-%28Hexahydro-4-Methyl-1h-1%2C4-Diazepin-1-Yl%29-1h-Benzimidazole+Difumarate+%28kg-2413%29+and+Related+Compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DIemura%26aufirst%3DR.%26aulast%3DHori%26aufirst%3DM.%26aulast%3DOhtaka%26aufirst%3DH.%26atitle%3DSyntheses%2520of%2520the%2520Metabolites%2520of%25201-%25282-Ethoxyethyl%2529-2-%2528Hexahydro-4-Methyl-1h-1%252C4-Diazepin-1-Yl%2529-1h-Benzimidazole%2520Difumarate%2520%2528kg-2413%2529%2520and%2520Related%2520Compounds%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D1989%26volume%3D37%26spage%3D962%26epage%3D966" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2WD7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2WD7','PDB','2WD7'); return false;">PDB: 2WD7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2WD8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2WD8','PDB','2WD8'); return false;">PDB: 2WD8</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GN1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GN1','PDB','3GN1'); return false;">PDB: 3GN1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GN2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GN2','PDB','3GN2'); return false;">PDB: 3GN2</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2C7V" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2C7V','PDB','2C7V'); return false;">PDB: 2C7V</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BME" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3BME','PDB','3BME'); return false;">PDB: 3BME</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i39"><a href="/doi/suppl/10.1021/jm900414x">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_18381"></div></div></div></div></div><hr /></hr><p class="last">A table listing initial screening hits with corresponding percent inhibition data is available as <a class="ref internalNav" href="#notes-1">Supporting Information</a>.This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm900414x/suppl_file/jm900414x_si_001.pdf">jm900414x_si_001.pdf (1.11 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm900414x&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Fjm900414x%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2009.52.issue-14" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm900414x" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67994a4bad4b3c0a","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
