
<html lang="en"     class="pb-page"  data-request-id="267d5579-8fe1-4c87-9007-9c7c3c404e3f"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;article:article:10.1021/acs.jmedchem.9b02042;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2020.63.issue-6"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Detection of Active Granzyme A in NK92 Cells with Fluorescent Activity-Based Probe" /></meta><meta name="dc.Creator" content="Sonia  Kołt" /></meta><meta name="dc.Creator" content="Tomasz  Janiszewski" /></meta><meta name="dc.Creator" content="Dion  Kaiserman" /></meta><meta name="dc.Creator" content="Sylwia  Modrzycka" /></meta><meta name="dc.Creator" content="Scott J.  Snipas" /></meta><meta name="dc.Creator" content="Guy  Salvesen" /></meta><meta name="dc.Creator" content="Marcin  Dra̧g" /></meta><meta name="dc.Creator" content="Phillip I.  Bird" /></meta><meta name="dc.Creator" content="Paulina  Kasperkiewicz" /></meta><meta name="dc.Description" content="Cytotoxic T-lymphocytes (CTLs) and natural killer cells (NKs) kill compromised cells to defend against tumor and viral infections. Both effector cell types use multiple strategies to induce target ..." /></meta><meta name="Description" content="Cytotoxic T-lymphocytes (CTLs) and natural killer cells (NKs) kill compromised cells to defend against tumor and viral infections. Both effector cell types use multiple strategies to induce target ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 6, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b02042" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b02042" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b02042" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b02042" /></link>
        
    
    

<title>Detection of Active Granzyme A in NK92 Cells with Fluorescent Activity-Based Probe | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b02042" /></meta><meta property="og:title" content="Detection of Active Granzyme A in NK92 Cells with Fluorescent Activity-Based Probe" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/large/jm9b02042_0006.jpeg" /></meta><meta property="og:description" content="Cytotoxic T-lymphocytes (CTLs) and natural killer cells (NKs) kill compromised cells to defend against tumor and viral infections. Both effector cell types use multiple strategies to induce target cell death including Fas/CD95 activation and the release of perforin and a group of lymphocyte granule serine proteases called granzymes. Granzymes have relatively broad and overlapping substrate specificities and may hydrolyze a wide range of peptidic epitopes; it is therefore challenging to identify their natural and synthetic substrates and to distinguish their localization and functions. Here, we present a specific and potent substrate, an inhibitor, and an activity-based probe of Granzyme A (GrA) that can be used to follow functional GrA in cells." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b02042"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b02042">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b02042&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b02042&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b02042&amp;href=/doi/10.1021/acs.jmedchem.9b02042" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 6</span><span class="cit-fg-pageRange">, 3359-3369</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/6" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b02123" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00060" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Detection of Active Granzyme A in NK92 Cells with Fluorescent Activity-Based Probe</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Sonia Kołt</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sonia Kołt</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Wrocław University of Science and Technology, Department of Chemical Biology and Bioimaging, Wyb. Wyspiańskiego 29, 50-370 Wroclaw, Poland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sonia++Ko%C5%82t">Sonia Kołt</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4076-0445" title="Orcid link">http://orcid.org/0000-0003-4076-0445</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tomasz Janiszewski</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tomasz Janiszewski</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Wrocław University of Science and Technology, Department of Chemical Biology and Bioimaging, Wyb. Wyspiańskiego 29, 50-370 Wroclaw, Poland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tomasz++Janiszewski">Tomasz Janiszewski</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dion Kaiserman</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dion Kaiserman</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Monash University, Monash Biomedicine Discovery Institute, Department of Biochemistry and Molecular Biology, 23 Innovation Walk, Clayton, VIC 3800, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dion++Kaiserman">Dion Kaiserman</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sylwia Modrzycka</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sylwia Modrzycka</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Wrocław University of Science and Technology, Department of Chemical Biology and Bioimaging, Wyb. Wyspiańskiego 29, 50-370 Wroclaw, Poland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sylwia++Modrzycka">Sylwia Modrzycka</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Scott J. Snipas</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Scott J. Snipas</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">NCI-designated Cancer Center, Sanford-Burnham Prebys Medical Discovery Institute, La Jolla, California 92037, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Scott+J.++Snipas">Scott J. Snipas</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Guy Salvesen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Guy Salvesen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">NCI-designated Cancer Center, Sanford-Burnham Prebys Medical Discovery Institute, La Jolla, California 92037, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Guy++Salvesen">Guy Salvesen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Marcin Dra̧g</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marcin Dra̧g</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Wrocław University of Science and Technology, Department of Chemical Biology and Bioimaging, Wyb. Wyspiańskiego 29, 50-370 Wroclaw, Poland</div><div class="loa-info-affiliations-info">NCI-designated Cancer Center, Sanford-Burnham Prebys Medical Discovery Institute, La Jolla, California 92037, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marcin++Dra%CC%A7g">Marcin Dra̧g</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8510-1967" title="Orcid link">http://orcid.org/0000-0001-8510-1967</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Phillip I. Bird</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Phillip I. Bird</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Monash University, Monash Biomedicine Discovery Institute, Department of Biochemistry and Molecular Biology, 23 Innovation Walk, Clayton, VIC 3800, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Phillip+I.++Bird">Phillip I. Bird</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Paulina Kasperkiewicz</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paulina Kasperkiewicz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Wrocław University of Science and Technology, Department of Chemical Biology and Bioimaging, Wyb. Wyspiańskiego 29, 50-370 Wroclaw, Poland</div></div><span class="conrtib-corresp"><strong>*</strong>P.K.: email, <a href="/cdn-cgi/l/email-protection#8bfbeafee7e2e5eaa5e0eaf8fbeef9e0e2eefce2e8f1cbfbfcf9a5eeeffea5fbe7"><span class="__cf_email__" data-cfemail="aedecfdbc2c7c0cf80c5cfdddecbdcc5c7cbd9c7cdd4eeded9dc80cbcadb80dec2">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paulina++Kasperkiewicz">Paulina Kasperkiewicz</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1291-047X" title="Orcid link">http://orcid.org/0000-0002-1291-047X</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b02042&amp;href=/doi/10.1021%2Facs.jmedchem.9b02042" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 6</span><span class="cit-pageRange">, 3359–3369</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 6, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>6 January 2020</li><li><span class="item_label"><b>Published</b> online</span>6 March 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 26 March 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b02042" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b02042</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice/index.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a><span class="article_header-cc-by text">with CC-BY<a href="/page/policy/authorchoice_ccby_termsofuse.html" title="License">license</a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b02042"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1239</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">2</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b02042" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Detection of Active Granzyme A in NK92 Cells with Fluorescent Activity-Based Probe&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Sonia&quot;,&quot;last_name&quot;:&quot;Kołt&quot;},{&quot;first_name&quot;:&quot;Tomasz&quot;,&quot;last_name&quot;:&quot;Janiszewski&quot;},{&quot;first_name&quot;:&quot;Dion&quot;,&quot;last_name&quot;:&quot;Kaiserman&quot;},{&quot;first_name&quot;:&quot;Sylwia&quot;,&quot;last_name&quot;:&quot;Modrzycka&quot;},{&quot;first_name&quot;:&quot;Scott&quot;,&quot;last_name&quot;:&quot;J. Snipas&quot;},{&quot;first_name&quot;:&quot;Guy&quot;,&quot;last_name&quot;:&quot;Salvesen&quot;},{&quot;first_name&quot;:&quot;Marcin&quot;,&quot;last_name&quot;:&quot;Dra̧g&quot;},{&quot;first_name&quot;:&quot;Phillip&quot;,&quot;last_name&quot;:&quot;I. Bird&quot;},{&quot;first_name&quot;:&quot;Paulina&quot;,&quot;last_name&quot;:&quot;Kasperkiewicz&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;06&quot;,&quot;issue&quot;:&quot;6&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;3359-3369&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b02042&quot;},&quot;abstract&quot;:&quot;Cytotoxic T-lymphocytes (CTLs) and natural killer cells (NKs) kill compromised cells to defend against tumor and viral infections. Both effector cell types use multiple strategies to induce target cell death including Fas/CD95 activation and the release of perforin and a group of lymphocyte granule serine proteases called granzymes. Granzymes have relatively broad and overlapping substrate specificities and may hydrolyze a wide range of peptidic epitopes; it is therefore challenging to identify their natural and synthetic substrates and to distinguish their localization and functions. Here, we present a specific and potent substrate, an inhibitor, and an activity-based probe of Granzyme A (GrA) that can be used to follow functional GrA in cells.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b02042&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b02042" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b02042&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b02042" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b02042&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b02042" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b02042&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b02042&amp;href=/doi/10.1021/acs.jmedchem.9b02042" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b02042" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b02042" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b02042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b02042%26sid%3Dliteratum%253Aachs%26pmid%3D32142286%26genre%3Darticle%26aulast%3DKo%25C5%2582t%26date%3D2020%26atitle%3DDetection%2Bof%2BActive%2BGranzyme%2BA%2Bin%2BNK92%2BCells%2Bwith%2BFluorescent%2BActivity-Based%2BProbe%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D6%26spage%3D3359%26epage%3D3369%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=291503" title="Organic polymers">Organic polymers</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292601" title="Probes">Probes</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/6" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/jmcmar.2020.63.issue-6/20200326/jmcmar.2020.63.issue-6.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/medium/jm9b02042_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/large/jm9b02042_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02042&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Cytotoxic T-lymphocytes (CTLs) and natural killer cells (NKs) kill compromised cells to defend against tumor and viral infections. Both effector cell types use multiple strategies to induce target cell death including Fas/CD95 activation and the release of perforin and a group of lymphocyte granule serine proteases called granzymes. Granzymes have relatively broad and overlapping substrate specificities and may hydrolyze a wide range of peptidic epitopes; it is therefore challenging to identify their natural and synthetic substrates and to distinguish their localization and functions. Here, we present a specific and potent substrate, an inhibitor, and an activity-based probe of Granzyme A (GrA) that can be used to follow functional GrA in cells.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75747" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75747" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Granule-associated serine proteases (Grs) play a pivotal role in the immune system and are released by cytotoxic natural killer cells (NKs) and cytotoxic T-lymphocytes (CTLs) to eliminate abnormal cells, such as those infected with bacteria or viruses, or cancer cells.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> Grs are stored in granules within the killer cell which, on engagement with a target cell, fuse with the killer cell membrane releasing their contents into the synaptic space. Besides Grs, granules contain the pore forming cytolytic protein (perforin). Perforin polymerizes to form a pore within the target cell membrane, allowing Grs to enter the target cell cytoplasm, which leads to dramatic changes in its morphology: nuclear changes, chromatin decondensation, membrane blebbing, and, finally, cell death by lysis.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a></div><div class="NLM_p">There are five Grs in humans, namely, Gr A, B, H, K, and M. Grs share similar structural features and use a catalytic triad of His-Asp-Ser to accomplish substrate protein cleavage. Granzyme A (GrA) is the only Gr that forms a dimer and is located at the same branch of an evolutionary tree as Granzyme K (GrK), found in the tryptase cluster at chromosome 5. These enzymes share 39.7% similarity (with 89% certainty), representing one of the most complementary pairs of Grs. Both enzymes are trypsin-like proteases and cleave after basic amino acid residues. Surprisingly, GrA and GrB share some substrates, and among them, 22 were cleaved in the same region but not at the same site,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> while GrB antibodies were found to label GrA, providing strong evidence of their similarity.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></div><div class="NLM_p">Grs are stored in an inactive form within granules to prevent effector (host cell) suicide, and Grs inadvertently released from granules into the host cell cytosol can be inhibited by endogenous inhibitors. The best-known examples of these endogenous inhibitors are serpins (serine protease inhibitors), which protect cells from misdirected Grs. SerpinB9<a onclick="showRef(event, 'ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9">(7−9)</a> protects against GrB activity, while no intracellular inhibitor has yet been identified for GrA;<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> however, serpinC1 prevents GrA activity in plasma.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a></div><div class="NLM_p">Despite intense research, the precise role of each Gr has not yet been determined. This is mainly due to the lack of specific tools for the in-depth investigation of Grs. Antibodies, which are commonly used, do not discriminate between an active and inactive form of the enzyme and therefore cannot be used to evaluate the control of activation and the subsequent activity of Grs. One solution to this problem involves activity-based probes (ABPs) that target only the active version of the enzyme. To date, only a few groups have tried to make chemical functional ABPs for Grs, but the major issue was poor selectivity of these probes for individual enzymes.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> The first ABP for GrB and GrA designed using combinatorial chemistry methods was reported by the Craik group; however, the kinetic inhibition constant rates of the designed ABPs were quite low (<i>k</i><sub>obs</sub>/I = 460 M<sup>–1</sup> s<sup>–1</sup> for GrB and 2000 M<sup>–1</sup> s<sup>–1</sup> for GrA), making them problematic for use in biological assays.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a></div><div class="NLM_p last">Here, we describe a novel ABP that will allow us to distinguish GrA from other Grs and allow for GrA detection in biological samples. Our probe is cell permeable and detects GrA in the complex environment of NK92 cell lysates; it can be applied in future research on GrA in straightforward and convenient assays.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81634" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81634" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> GrA Substrate Specificity</h3><div class="NLM_p">GrA possesses a broad substrate specificity and can hydrolyze many physiological substrates;<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> as well as collagens and fibronectin, GrA hydrolyzes apurinic/apyrimidinic endonuclease-1 (Ape1),<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> high mobility group protein 2 (HMG2),<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> histone H1,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> nuclear lamins,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> poly ADP-ribose polymerase (PARP-1),<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> endoplasmic reticulum-associated SET complex, and three prime repair exonuclease 1 (TREX1).<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19,20)</a> We started our study by establishing an extensive GrA substrate specificity profile using a set of natural and unnatural amino acids. First, we synthesized a tailored library of fluorogenic substrates that contain the same P4–P2 motif, designed based on existing data, and at P1, we incorporated various amino acid residues (the overall structure of the library was Ac-Ile-Ser-Pro-P1-ACC, where ACC is 7-amino-4-carbamoylmethylcoumarin, Ac is an acetyl group, and P1 is an individual amino acid).</div><div class="NLM_p">Next, we tested the activity of human GrA against individual P1 library components, and our results were in accordance with existing data concerning natural amino acids. We found that GrA interacts almost exclusively with positively charged amino acid residues, with the highest activity toward ones that contain a guanidine group (<span class="smallcaps smallerCapital">l</span>-Arg, 92% and <span class="smallcaps smallerCapital">l</span>-Phe(guan), 100%). However, our data suggested that the GrA S1 pocket is deep, since amino acids with long side chains containing basic residues (<span class="smallcaps smallerCapital">l</span>-Lys, 21%) were accommodated much better than those with shorter side chains and the same chemical character such as <span class="smallcaps smallerCapital">l</span>-Dap and <span class="smallcaps smallerCapital">l</span>-Dab. Additionally, <span class="smallcaps smallerCapital">l</span>-Abu(Bth) was recognized better than <span class="smallcaps smallerCapital">l</span>-Ala(Bth), which is one methylene group shorter. Interestingly, a substrate containing a 3-aminobenzoic acid residue (3-Abz, 47%) was cleaved by GrA better than the basic <span class="smallcaps smallerCapital">l</span>-Lys residue, 4-aminobenzoic acid residue (4-Abz, 17.2%), or 2-aminobenzoic acid residue (2-Abz, 1.8%). In 2010, the Gevaert group, using N-terminal peptide-centric proteomic technology, characterized the substrate specificity of GrA.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Their results demonstrated that GrA almost exclusively hydrolyzes substrates after basic residues at the P1 position (arginine and lysine), confirming previous findings. Overall, the GrA molecule is positively charged with a pI of > 9, but the negative charge of Asp189 in the S1 pocket forces the binding of positively charged residues. The S1 pocket is deep and wide and is formed by Asp189-Ser195, Gly224-Val227, Lys219-Gly221, and Ser214-Leu217,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> reflecting its catalytic preferences for long, large, and basic amino acids. We found that GrA can also accommodate acidic amino acids in this pocket; however, there is 5 times lower activity (<span class="smallcaps smallerCapital">l</span>-Asp, 17.8%) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A, B and <a class="ref internalNav" href="#notes3" aria-label="Supporting Information Figure S1">Supporting Information Figure S1</a>).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/medium/jm9b02042_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/large/jm9b02042_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Design and characteristics of GrA substrates. (A) GrA P4–P1 catalytic preferences. The activity of GrA was tested using the HyCoSuL strategy or individual substrates for P1 examination. Substrates were placed into the wells of the plate at the same concentration, followed by the addition of the enzyme in the reaction buffer. The increase in fluorescence was measured with 355 nm/460 nm excitation/emission wavelengths. Data are presented as a violin diagram of all events. For detailed analysis, see <a class="ref internalNav" href="#notes3" aria-label="Supporting Information Figure S1">Supporting Information Figure S1</a>. (B) Amino acid structures selected for new substrate synthesis based on library screening. The best recognized sequences were chosen for individual substrate design.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/large/jm9b02042_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02042&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To test GrA specificity at the P4–P2 positions, we utilized the HyCoSuL strategy and a library containing Arg at the P1 position.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> We observed that the GrA S2 pocket can accommodate many structurally different amino acids with different chemical properties, including polar, nonpolar, basic, and acidic residues, proline derivatives, and <span class="smallcaps smallerCapital">d</span>-amino acids. A strong preference was evident for the proline derivative <span class="smallcaps smallerCapital">l</span>-octahydroindole-2-carboxylic acid (<span class="smallcaps smallerCapital">l</span>-Oic, 100%): substrates containing this derivative were approximately 40% more potently hydrolyzed than the other amino acids. Oic is a proline derivative equipped with an additional flexible cyclohexane residue. Cyclohexane can form few three-dimensional shapes and easily switch between conformations; therefore, we speculate that this ring, attached to the proline, a cyclic secondary amine, improves substrate accommodation at the S2 pocket. This pocket is formed by Asp95-Arg99 and Ser214-Leu217 loops.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Because of the guanidine group of Arg99, this pocket can bind aliphatic amino acids better than basic amino acids.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Previous studies reported the preferences of GrA to different amino acids, such as hydrophobic <span class="smallcaps smallerCapital">l</span>-Phe or <span class="smallcaps smallerCapital">l</span>-Asn (based on PS-SCL analysis)<a onclick="showRef(event, 'ref13 ref21'); return false;" href="javascript:void(0);" class="ref ref13 ref21">(13,21)</a> at the S2 pocket, which is in agreement with our studies; however, previous studies were carried out on the library that contained only natural amino acids<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A, B and <a class="ref internalNav" href="#notes3" aria-label="Supporting Information Figure S1">Supporting Information Figure S1</a>).</div><div class="NLM_p">The P3 position, similar to the P4 position, has broad substrate preferences. Among a wide range of unnatural amino acids with different chemical properties, all of them, excluding <span class="smallcaps smallerCapital">d</span>-alanine, were well recognized by GrA (>30%), whereas library substrates containing some natural residues, such as <span class="smallcaps smallerCapital">l</span>-Arg, <span class="smallcaps smallerCapital">l</span>-Asn, <span class="smallcaps smallerCapital">l</span>-Glu, <span class="smallcaps smallerCapital">l</span>-Leu, <span class="smallcaps smallerCapital">l</span>-Lys, <span class="smallcaps smallerCapital">l</span>-Phe, <span class="smallcaps smallerCapital">l</span>-Thr, and <span class="smallcaps smallerCapital">l</span>-Val, were not hydrolyzed (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A, B and <a class="ref internalNav" href="#notes3" aria-label="Supporting Information Figure S1">Supporting Information Figure S1</a>). However, the most preferred amino acid was Gly (100%), a natural, nonpolar amino acid, without a side chain. This finding is in agreement with the previous results obtained by the Gevaert group;<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> however, by extending the classic PS-SCL library with a set of unnatural amino acids, we observed that in contrast to natural sequences, all unnatural residues were recognized in this pocket. In accordance with the results of the Gevaert group, the S3 pocket of GrA is narrower than that of trypsin but is still capable of binding bulky amino acids such as <span class="smallcaps smallerCapital">l</span>-hSer(O-Bzl) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A, B and <a class="ref internalNav" href="#notes3" aria-label="Supporting Information  Figure S1">Supporting Information  Figure S1</a>).</div><div class="NLM_p last">The GrA S4 pocket is capable of binding various amino acids with preferences for the <span class="smallcaps smallerCapital">l</span>-Ile from natural amino acids and for bridged analogs of <span class="smallcaps smallerCapital">l</span>-phenylalanine, 1,2,3,4-tetrahydroisoquinoline-3-carboxylic acid (<span class="smallcaps smallerCapital">l</span>-Tic, 100%), mono-oxidized methionine (<span class="smallcaps smallerCapital">l</span>-Met(O), 77%), and threonine with a benzyl group (<span class="smallcaps smallerCapital">l</span>-Thr(Bzl), 63%) from unnatural residues (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A, B). Methionine oxidation is a naturally occurring posttranslational modification of endogenous substrates that occurs within the organism; therefore, we speculate that GrA may be involved in the hydrolysis of posttranslationally modified substrates with <span class="smallcaps smallerCapital">l</span>-Met(O).</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> GrA Fluorogenic Substrate Design, Synthesis, and Kinetic Evaluation</h3><div class="NLM_p">Two commercially available GrA substrates exist: Z-GPR-AFC/MNA (where <b>AFC</b> is 7-amido-4-trifluoromethylcoumarin trifluoroacetate, <b>MNA</b> is 4-methoxy-2-naphthylamide trifluoroacetate, and <b>Z</b> is a Cbz-protecting group). These substrates are tripeptides and are characterized by low kinetic constants. We therefore decided to change the peptidic leading sequence in GrA substrates. To change the sequence, based on the results from HyCoSuL library screening, we designed and synthesized 18 fluorogenic tetrapeptides that contain ACC as a fluorophore and an acetylated N-terminus (named SK8-SK28), and one reference substrate based on natural amino acids (named SK7). In these sequences, we applied the combination of selected amino acids at <b>P4</b> (<span class="smallcaps smallerCapital">l</span>-Tic, <span class="smallcaps smallerCapital">l</span>-Ile, and <span class="smallcaps smallerCapital">l</span>-Met(O)), <b>P3</b> (Gly and <span class="smallcaps smallerCapital">l</span>-Abu(Bth)), <b>P2</b> (<span class="smallcaps smallerCapital">l</span>-Oic, <span class="smallcaps smallerCapital">l</span>-Hyp(Bzl), <span class="smallcaps smallerCapital">l</span>-Phe(4-F), and <span class="smallcaps smallerCapital">l</span>-Tyr(2,6-Cl<sub>2</sub>–Bzl)), and <b>P1</b> (<span class="smallcaps smallerCapital">l</span>-Asp, 3-Abz, <span class="smallcaps smallerCapital">l</span>-Arg, <span class="smallcaps smallerCapital">l</span>-Phe(guan)) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B).</div><div class="NLM_p">We measured the activity of GrA against these new substrates and selected the top substrate, SK15 (Ac-Tic-Gly-Oic-Arg-ACC), as the leading candidate to be used in our inhibitor and ABP. GrA was approximately 40-fold more active against the SK15 substrate than the reference sequence (SK7), with a <i>k</i><sub>cat</sub>/<i>K</i><sub>M</sub> value of 3100 M<sup>–1</sup> s<sup>–1</sup> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Surprisingly, substrates that contain 3-Abz at the P1 position and different, defined amino acid residues at P2, P3, and P4 were not recognized by GrA, although library screening revealed that this amino acid was only two times less active than arginine. Our kinetic analyses have shown enzyme pockets cooperativity: that the binding of particular substrate residues at a GrA S1 pocket can influence the binding of particular residues at other subsites.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Steady-State Michaelis–Menten Parameters for GrA, GrK, and GrB Determined for SK15 and SK7 Substrates<a class="ref internalNav" href="#tbl1-fn1" aria-label="*">*</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center"><b>GrA</b></th><th class="rowsep1 colsep0" align="center"><b>GrB</b></th><th class="rowsep1 colsep0" align="center"><b>GrK</b></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"><b>SK15</b></th><th class="colsep0 rowsep0" align="center"><b>SK7</b></th><th class="colsep0 rowsep0" align="center"><b>SK15</b></th><th class="colsep0 rowsep0" align="center"><b>SK15</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>K</i><sub>M</sub> [μM]</td><td class="colsep0 rowsep0" align="left">87.4 ± 1.3</td><td class="colsep0 rowsep0" align="left">450 ± 1.3</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#tbl1-fn1" aria-label="*">*</a></td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#tbl1-fn1" aria-label="*">*</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>k</i><sub>cat</sub> [s<sup>−1</sup>]</td><td class="colsep0 rowsep0" align="left">0.27 ± 0.0029</td><td class="colsep0 rowsep0" align="left">0.035 ± 0.00076</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#tbl1-fn1" aria-label="*">*</a></td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#tbl1-fn1" aria-label="*">*</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>k</i><sub>cat</sub>/<i>K</i><sub>M</sub> [s<sup>–1</sup> M<sup>–1</sup>]</td><td class="colsep0 rowsep0" align="left">3100 ± 80</td><td class="colsep0 rowsep0" align="left">78 ± 6.60</td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#tbl1-fn1" aria-label="*">*</a></td><td class="colsep0 rowsep0" align="left">ND<a class="ref internalNav" href="#tbl1-fn1" aria-label="*">*</a></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup><sup>*</sup></sup><p class="last">ND – no activity detected.</p></div></div></div><div class="NLM_p">Among the substrates designed by our group, sequences with <span class="smallcaps smallerCapital">l</span>-Arg at P1 (substrates SK15-SK21) possessed the highest activity, and substrates containing <span class="smallcaps smallerCapital">l</span>-Phe(guan) at P1 (SK22-SK28) were less active than P1-Arg substrates and more active than P1-3-Abz sequences (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). The exchange of only one amino acid at P4 (<span class="smallcaps smallerCapital">l</span>-Tic to <span class="smallcaps smallerCapital">l</span>-Ile) resulted in a decrease of the hydrolysis rate (3.5-fold), while the exchange of one amino acid at P3 (Gly to L-Abu(Bth)) resulted in a marked decrease in activity (63-fold), suggesting that this pocket has a crucial role in substrate interactions and that the amino acid that occupies S3 should be small and aliphatic. SK15 was the most active substrate hydrolyzed by GrA, as reflected by low <i>K</i><sub>M</sub> values (87.4 ± 1.3 μM) and high <i>k</i><sub>cat</sub> rates (0.27 ± 0.0029 s<sup>–1</sup>) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/medium/jm9b02042_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/large/jm9b02042_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Selection of the most active GrA substrate. (A) Substrates SK7–SK28 were placed into the wells of the plate at the same concentration (1 μM), followed by the addition of 89 nM enzyme in the reaction buffer. The increase in fluorescence was measured with 355 nm/460 nm excitation/emission wavelengths, accordingly. Data present the average of 2 replicates as adjusted RFU/s. (B) Comparison of SK7 (blue) and SK15 (red) structures.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/large/jm9b02042_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02042&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Selectivity of GrA Substrates (SK8–28)</h3><div class="NLM_p">Since GrA and GrK are the most structurally similar Grs, they may recognize and hydrolyze the same natural and synthetic substrates. Therefore, we tested whether our new GrA substrates are also recognized by GrK or GrB (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). For this experiment, we screened substrates SK8 to SK28 and a GrK<sub>ref</sub> substrate (Ac-Tyr-Arg-Phe-Lys-ACC) or GrB<sub>ref</sub> substrate (Ac-Ile-Glu-Pro-Asp-ACC) at the same concentration using a plate reader, and we monitored the increase in fluorescence over time. The linear portions of the diagram were analyzed, and as we expected, the majority of the substrates were well recognized by GrK, even better than their recognition by GrA. However, the most potent substrate for GrA, SK15, was weakly hydrolyzed by GrK (20%) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A and <a class="ref internalNav" href="#notes3" aria-label="Figure S2">Figure S2</a>). Additionally, we tested substrates SK8 to SK28 with GrB and noticed that GrB significantly hydrolyzed only two GrA substrates (SK13 and SK17) but was inactive on the SK15 substrate. Additional selectivity testing of SK15 on other serine proteases (granzyme M (GrM), one of the most potent proteases proteinase 3 (PR3) and human neutrophil elastase (hNE)) can be found in <a class="ref internalNav" href="#notes3" aria-label="Supporting Information Figure S2">Supporting Information Figure S2</a>. Based on that, we concluded that Tic-Gly-Oic-Arg is a leading sequence for a specific potent inhibitor and ABP for GrA.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/medium/jm9b02042_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/large/jm9b02042_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) GrA substrate selectivity testing on GrK and GrB. Substrates SK8–SK28 and the reference GrK and GrB substrates were placed in a 96-well plate at a concentration of 200 μM and treated with GrK or GrB. The fluorescence increase was measured with 355 nm/460 nm excitation and emission wavelengths. Data present the average of 2 replicates as adjusted RFU/s. (B) Structures of SK15I and SK15.5 and kinetic evaluation of the inhibitor and probe. (C) Selectivity of SK15I and SK15.5 toward GrK, GrB, GrM, PR3, and hNE.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/large/jm9b02042_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02042&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> GrA Inhibitor Design and Characteristics</h3><div class="NLM_p">Currently, biological studies on GrA activity use inhibitors characterized by low potency or cells lacking GrA derived using gene knockout or suppression strategies. The latter strategies are time-consuming and require many resources; therefore, we decided to obtain a potent and selective inhibitor that will allow us to perform so-called chemical knockdown of GrA. Based on the structure of our selective and potent substrate SK15, we designed an inhibitor with two main fragments: a recognition sequence and a warhead (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). Using classical techniques (solid phase peptide synthesis, peptide synthesis in solution), we synthesized an inhibitor by replacing the ACC group with an arginine derivative of diphenyl phosphonate. First, we synthesized a recognition element, based on the structure of the best substrate, by elongating the peptide on chlorotrityl resin. The N-terminus was protected by incorporating a methoxysuccinyl group to obtain MeOSuc-Tic-Gly-Oic-COOH (<b>1</b>). Second, we selected a diphenyl phosphonate as a reactive group that binds covalently in the active site of serine proteases. This type of inhibitor mimics the peptide structure and therefore fits perfectly within the active site of serine proteases. Using an Oleksyszyn reaction, we obtained a warhead <sup>+</sup>H<sub>3</sub>N-Arg<sup>P</sup>(OPh)<sub>2</sub> (<b>2</b>). Finally, we coupled <b>1</b> and <b>2</b> in a solution, and after the coupling reaction using classic coupling reagents, we obtained MeOSuc-Tic-Gly-Oic-Arg<sup>P</sup>(OPh)<sub>2</sub> (<b>3</b>).</div><div class="NLM_p last">We calculated the apparent second-order rate constants for inhibition (<i>k</i><sub>obs(app)</sub>/I) of GrA-mediated SK15I under pseudo first-order conditions by serially diluting the new inhibitor in the range of 714–333 nM with a constant optimal substrate (SK15) concentration of 10 μM (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). We demonstrated that the inhibitor binds with the kinetic constant of <i>k</i><sub>obs</sub>/I = 171 000 ± 24 000 M<sup>–1</sup> s<sup>–1</sup>, which is approximately 85-fold better than the reference inhibitor (2000 M<sup>–1</sup> s<sup>–1</sup>) and is the most potent GrA inhibitor described to date.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Fluorescent ABP Design and Kinetic Evaluation</h3><div class="NLM_p last">Antibody-related techniques allow evaluation of the total amount of enzyme present in a sample but do not distinguish between active and inactive (e.g., zymogen) forms. To date, an ABP for GrA <i>in-gel</i> detection has not been reported. This type of analysis allows detection of the active enzyme in biological samples via SDS-PAGE without the need for protein transfer and immunoblotting. We therefore designed a new probe that will allow for the convenient and straightforward <i>in-gel</i> detection of GrA in cell, tissue, or fluid extracts. The probe differs from SK15I only at the <i>N-</i>terminus. The new molecule, named SK15.5 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B), was equipped with a fluorogenic moiety, the cyanine derivative Cy5, which allows the direct detection of the enzyme in samples in SDS-PAGE gels, without the need for membrane transfer. Additionally, the fluorogenic moiety was separated from the recognition sequence with an aminohexanoic acid linker, which reduces the undesirable binding of fluorophore in the active site pockets of GrA. For this purpose, first, on the chlorotrityl resin, we synthesized Cy5-Ahx-Tic-Gly-Oic-COOH and purified it with HPLC. After purification, Cy5-Ahx-Tic-Gly-Oic-COOH was coupled with the warhead molecule (<b>2</b>) to obtain Cy5-Ahx-Tic-Gly-Oic-Arg<sup>P</sup>(OPh)<sub>2</sub>. The application of a cyanine derivative at the N-terminus dramatically reduced <i>k</i><sub>obs</sub>/I to 3521 ± 296 M<sup>–1</sup> s<sup>–1</sup>, which may reflect the catalytic preferences of GrA in enzyme pockets more distant from the active site. However, the <i>k</i><sub>obs</sub>/I values for SK15.5 against GrA were still very high, and this probe may be employed for detection of the active enzyme in cells, plasma, supernatants, or lysates. Therefore, we tested this hypothesis in our next experiments.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Selectivity of SK15I and SK15.5</h3><div class="NLM_p last">We wondered whether the selectivity of our SK15-based molecules for GrA would be maintained by substrate structure modifications, such as replacing ACC with a diphenyl phosphonate warhead and changing the N-terminus from acetyl to the cyanine derivative Cy5. GrK is the only enzyme from the Grs family that hydrolyzes after basic amino acids, and GrB is one of the most potent Grs; therefore, we selected these enzymes as some of our selectivity controls, as well as other serine proteases: GrM, very potent PR3 and hNE. We tested the residual activity of all of the above enzymes preinhibited with SK15I and SK15.5 (separately) using substrates dedicated to each enzyme (SK15 for GrA, GrK<sub>ref</sub> for GrK, GrB<sub>ref</sub> for GrB, Ac-Lys-Val-Pro-Leu-ACC for GrM, Ac-Glu(O-Bzl)-Lys(Ac)-Hyp(Bzl)-Nva-ACC for PR3 and Ac-Arg(NO)<sub>2</sub>-Glu(Bzl)-Oic-Abu-ACC for hNE (<a class="ref internalNav" href="#notes3" aria-label="Figure.S2">Figure.S2</a>)). As indicated in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C, we can see a minimal inhibition of all five remaining enzymes (less than 20%) with SK15I and almost no inhibition effect with SK15.5 on any of these enzymes (except for minimal 3% inhibition for GrK). Our inhibitor and probe are equipped with diphenyl phosphonate warhead, which is known to covalently bind in the active site of serine proteases;<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> hence, we speculate that this caused observed slight reactivity in inhibition for all investigated serine proteases. As we expected, the activity of GrA was completely blocked by these inhibitors. These results were promising in terms of using our molecules, especially SK15.5 in biological samples.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> <i>In-Gel</i> GrA Detection</h3><div class="NLM_p">Next, we tested the utility of our fluorescent probe for the <i>in-gel</i> detection of active GrA in test samples. First, we labeled purified GrA with SK15.5 at equal concentrations (the enzyme to probe ratio was 1:1) for different lengths of time in the range of 0 min to 2 h. We then ran a simple SDS-PAGE assay to detect labeled GrA (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). SK15.5 is a potent probe with a <i>k</i><sub>obs</sub>/I value of 3520 ± 296 M<sup>–1</sup> s<sup>–1</sup>; therefore, we were expecting to detect the enzyme after a short incubation time with the probe. In our hands, the probe bound to the enzyme, as indicated by the fluorescent 35 kDa species, after the minimal incubation time (5 min). After 30 min incubation, we did not observe an increase in the labeled species; therefore, we concluded that the GrA was maximally labeled with the probe. Therefore, in the next experiment, we tested the detection limit, i.e. the minimum concentration of SK15.5 needed for GrA labeling, and we estimated it to be approximately 50 nM. The optimal concentration was 500 nM (strong signal) (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/medium/jm9b02042_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/large/jm9b02042_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. In-gel GrA detection with SK15.5 using recombinant enzyme and NK92 cell line lysates. (A) Incubation time of SK15.5 with enzyme optimization. GrA (250 nM) was incubated with 250 nM SK15.5 for the indicated time in the range 0–120 min at 37 °C, followed by a reduction with 3× SDS loading buffer and SDS-PAGE analysis. (B) SK15.5 concentration optimization. GrA (250 nM) was incubated with SK15.5 at various concentrations ranging from 10 to 500 nM for 30 min at 37 °C, followed by reduction with 3× SDS loading buffer. (C) SK15.5 binds exclusively with the active GrA. Probe binding was tested on both the active enzyme and the proenzyme at the same concentration and incubation time. The active site GrA inhibitor and the presence of prodipeptide abolished probe binding. (D) GrA in-gel detection in NK92 cell lysates. NK92 cells in a log growth phase were lysed (1 × 10<sup>7</sup> cells/mL) in lysis buffer and sonicated. Lysates were incubated with SK15.5 in a concentration range of 250 nM to 2 μM, as indicated, for 1 h at 37 °C. Samples were run on SDS-PAGE gel, transferred to a membrane, immunostained with anti-GrA, and imaged with 488 and 658 nm lasers. The results are representative of at least 2 replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/large/jm9b02042_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02042&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> SK15.5 Binds Exclusively to the Active Form of GrA</h3><div class="NLM_p last">In contrast to the closely related GrB, GrH, GrM, and GrK, GrA is a homodimer formed by stable disulfide-linked subunits.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Since the subunits are oriented in opposite directions, the surrounding area may form an exosite.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Therefore, we aimed to verify whether the zymogen form of GrA containing the activation prodomain peptide Glu27-Lys28<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> or the active GrA form with the processed N-terminus would bind the SK15.5 probe. After both GrA forms were incubated with SK15.5 followed by Western blot analysis, we noticed only a sharp single species corresponding to the processed form of the enzyme, and no signal was detected in the zymogen. The preinhibition of either GrA form with the active site-directed inhibitor SK15I also prevented probe binding (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C, <a class="ref internalNav" href="#notes3" aria-label="Figure S4">Figure S4</a>). Therefore, we concluded that GrA must be fully processed to demonstrate catalytic activity and that our probe binds exclusively within the GrA active site.</div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> GrA Labeling in NK92 Cell Lysates</h3><div class="NLM_p">Our ultimate goal was to produce a probe that not only can detect purified, active GrA but also detects GrA in more complex biological samples. To test whether our probe binds selectively to GrA in whole cell lysates, we selected NK92 cells, in which Grs are abundant. We lysed the cells in the exponential growth phase, and to detect GrA activity, we treated lysates with SK15.5 for 1 h. We doubled the optimal time of GrA:SK15.5 incubation to verify whether the elongation of incubation time results in nonspecific binding.</div><div class="NLM_p">We analyzed the presence of a Cy5 signal from the SK15.5 probe and observed a strong signal from a protein between 26 and 35 kDa, corresponding to a monomer of GrA (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D). As a control, we ran lysates without a probe, and we did not observe a signal, meaning that the observed signal was indeed from SK15.5. Additionally, an overlay of a signal from the probe and GrA antibody confirmed the binding of SK15.5 with only GrA. An additional signal between 35 and 55 kDa was noticed upon antibody labeling, and we speculate that it was a dimer form of GrA.</div><div class="NLM_p last">We did not observe any additional bands, even at a high probe concentration (2 μM), supporting our previous finding that SK15.5 is GrA selective. We concluded that our probe can be easily employed for in-gel GrA detection in cell extracts.</div></div><div id="sec2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> GrA Labeling in Live NK92 Cells</h3><div class="NLM_p">NK92 is an NK-like cell line that contains abundant GrA. The main goal of this research was to obtain the first ABP for GrA <i>in-gel</i> analysis. As we demonstrated in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>, our probe selectively binds with GrA in a complex system of cellular lysates. To test whether our probe can be used for GrA detection in living cells we first checked the cytotoxicity of the SK15.5 probe on MDA-MB-231 cells using a classic MTS assay and demonstrated that after a very long incubation time of 24 h, our probe was cytotoxic only at a very high concentration, which was higher than the GrA amount within the cells. Therefore, we established that SK15.5 is harmless to these cells at a concentration of 1 μM. We also tested the cytotoxicity of SK15I inhibitor, for future application in cell-based assays, and no harmful effect on cells was observed even at the very high inhibitor concentration of 10 μM (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/medium/jm9b02042_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/large/jm9b02042_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. SK15.5 probe utility in living cells. (A) SK15.5 probe and SK15I inhibitor cytotoxicity. MDA-MB-231 cells were treated with different SK15.5 probe or SK15I inhibitor concentrations in growth media for 24 h, and the cell viability was verified with an MTS assay at 490 nm. As indicated, the probe is cytotoxic only at very high concentrations, higher than 2.5 μM, while the inhibitor shows no cytotoxic effect even at 10 μM. The experiment was performed in duplicate, and the data are presented as an average of these repetitions with standard deviation. (B) SK15.5 probe cell permeability. NK92 cells in culture media were treated with SK15.5 (1 μM) for the indicated time up to 8 h, followed by cell washing, lysis, and SDS-PAGE analysis followed by protein transfer to a nitrocellulose membrane. Where indicated, the cells were preincubated with SK15I to prevent probe binding at the GrA active site. The results represent two biological replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/large/jm9b02042_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02042&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Next, we tested whether our SK15.5 probe is cell permeable. Thus, we treated NK92 cells with 1 μM SK15.5 for up to 8 h and prepared cell extracts for SDS-PAGE. We noticed a strongly labeled species within 30 min, and the intensity increased until 2 h (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). This result led to the conclusion that the optimal time to detect the maximum amount of labeled GrA within NK92 cells is 2 h, while for nonquantitative detection, 30 min can be used. Additionally, we utilized the GrA covalent inhibitor (10 μM, 1 h) prior to the SK15.5 probe (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B, lane 7), and we observed a substantial decrease in probe binding. As an additional control, immunolabeling was performed and confirmed the selective binding of the probe with GrA (for full size blots, please see <a class="ref internalNav" href="#notes3" aria-label="Figure S3">Figure S3</a>).</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68663" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68663" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Grs are a family of incompletely understood serine proteases that are involved in inflammatory and apoptotic processes.<a onclick="showRef(event, 'ref1 ref28'); return false;" href="javascript:void(0);" class="ref ref1 ref28">(1,28)</a> GrA and GrK recognize the same substrate recognition motifs, and both enzymes preferentially cleave after basic amino acid residues such as arginine or lysine; however, GrA has stronger affinity for Arg, while GrK has a stronger affinity for Lys. GrA, in contrast to GrK, is a heterodimer linked by disulfide bonds, and this feature distinguishes GrA from other Grs but has no influence on GrA/K substrate recognition motifs. These features make the design and evaluation of new chemical tools for GrA investigation challenging, and until this study, only one low potency chemical probe for GrA had been successfully designed.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Therefore, the goal of our studies was to develop a potent and fluorescent chemical marker for easy and straightforward labeling of GrA in its active form. Hence, we constructed the first fluorescent ABP for in-gel GrA detection and validated its utility for labeling GrA in living cells.</div><div class="NLM_p">GrA substrate specificity has been previously examined using PS-SCL and correlated poorly with the recognition motif in physiological GrA substrates.<a onclick="showRef(event, 'ref10 ref13'); return false;" href="javascript:void(0);" class="ref ref10 ref13">(10,13)</a> More effective searches for substrate sequence motifs are necessary; thus, in our study, we employed a HyCoSuL strategy that incorporates a broad range of unnatural amino acids to find an optimal protease substrate sequence.<a onclick="showRef(event, 'ref22 ref25 ref29'); return false;" href="javascript:void(0);" class="ref ref22 ref25 ref29">(22,25,29)</a></div><div class="NLM_p">Our findings on GrA substrate preferences correlate well with previous studies that reported natural peptidic substrates of GrA. The comparison of P1 GrA motifs determined via PS-SCL and phage display to the HyCoSuL data revealed that in addition to natural basic amino acids such as <span class="smallcaps smallerCapital">l</span>-Arg or <span class="smallcaps smallerCapital">l</span>-Lys, this enzyme also recognizes nonphysiological residues such as phenylalanine with a guanidine group. Moreover, the enzyme recognizes these nonphysiological residues 5-fold better than it recognizes <span class="smallcaps smallerCapital">l</span>-Lys, reflecting the ability of GrA to recognize bulky and positively charged amino acids. This is due to the shape and character of the S1 pocket forced by the amino acid residues forming it.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Despite basic amino acids, we have also observed the hydrolysis occurring after the acidic amino acid <span class="smallcaps smallerCapital">l</span>-Asp, at a similar level as basic <span class="smallcaps smallerCapital">l</span>-Lys; therefore, we speculate that the S1 shape of the pocket is very flexible and can adapt different amino acid residues from basic to negatively charged residues.</div><div class="NLM_p">Previous studies reported that GrA prefers <span class="smallcaps smallerCapital">l</span>-Asp, <span class="smallcaps smallerCapital">l</span>-Asn, <span class="smallcaps smallerCapital">l</span>-Phe, or <span class="smallcaps smallerCapital">l</span>-Pro P2 at the P2 position. However, when we increased the diversity of the PS-SCL library by incorporating unnatural amino acids, we noticed a dramatic increase in GrA activity on substrates with the proline derivative L-Oic. This moiety is an L-octahydroindole, and its cyclohexyl ring is flexible and tends to assume multiple nonplanar conformations. This may indicate the presence of an additional pocket close to S2 or that the shape of S2 can accommodate more complex structures than the shapes of natural amino acids. In addition to P2, our results confirm previously reported P3 and P4 specificity.<a onclick="showRef(event, 'ref5 ref13'); return false;" href="javascript:void(0);" class="ref ref5 ref13">(5,13)</a> Surprisingly, among the unnatural amino acids, incorporating bulky <span class="smallcaps smallerCapital">l</span>-Abu(Bth) at P3 and <span class="smallcaps smallerCapital">l</span>-Tic at P4 resulted in efficiently cleaved substrates; however, after extensive validation, substrates containing the Tic-Gly-Oic-Arg recognition motif were found to be most potently processed by GrA. We speculate that this may be due to enzyme pocket cooperativity:<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> because two neighboring residues, <span class="smallcaps smallerCapital">l</span>-Tic and <span class="smallcaps smallerCapital">l</span>-Abu(Bth), are bulky, there may not be enough room for both. Therefore, the application of Gly at P3 resulted in the optimal sequence design.</div><div class="NLM_p">All synthesized GrA substrates were screened with GrK and both enzymes hydrolyzed the majority of the substrates, confirming the similarity of the core substrate recognition motif in these enzymes. The structure of our optimal substrate (SK15) is more sophisticated, and <span class="smallcaps smallerCapital">l</span>-Oic at P2 and <span class="smallcaps smallerCapital">l</span>-Tic at P4 allow SK15 to be well recognized by GrA and barely hydrolyzed by GrK and not recognized by other investigated serine proteases such as GrB, GrM, PR3, and hNE. Therefore, we applied this recognition motif in a novel inhibitor and a fluorescent ABP and demonstrated the first specific, potent, and fluorescent marker for GrA in-gel detection.</div><div class="NLM_p">GrA is the only member of the Gr family that occurs as a dimer. The Jenne group demonstrated the existence of the GrA dimer and the active sites orientation in opposite directions. This group also suggested the existence of exosites in the GrA homodimer near the active sites, which serve as assistants for the active site.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Being aware that our probe can bind to sites other than catalytic ones, we tested our fluorescent probe binding with active GrA and zymogen (N-terminal-processed GrA or N-terminal-containing prodipeptide GrA). This revealed that our activity-based probe did not label the zymogen form of GrA and preinhibition of active GrA with SK15I did not result in unspecific probe binding. Thus, we demonstrated that our covalent probe binds exclusively within the GrA active site.</div><div class="NLM_p">Cells and cell lysates are more complex mixtures of proteases (mainly serine and cysteine proteases). To test the sensitivity and selectivity of SK15.5 for GrA, we incubated it with NK92 cell lysates, and we observed only one species, which was confirmed by antibody staining for GrA. We also demonstrated that SK15.5 is cell permeable and can label GrA within cells. Importantly we found that SK15.5 has very low cytotoxicity: thus, it has the potential for use in future GrA in vivo investigations.</div><div class="NLM_p last">Although Grs play a crucial role in the elimination of infected cells, their activity is also associated with disorders such as pulmonary disease, cardiovascular disease, and cancer. Furthermore, increased Grs activity correlates with patient age, since aging leads to an increase in inflammatory mediator release and to the activation of immune cells, thereby increasing Grs secretion. Therefore, in this work we have developed a novel fluorescent ABP for the fast, direct, and convenient detection of GrA. In the future, this type of molecule can be used to monitor active enzyme levels within cells in targeted cell killing or cell death assays. Additionally, since SK15.5 is not cytotoxic at high concentrations and has the properties of a covalent inhibitor, it can be applied for the so-called chemical knock down of GrA (inhibition of its activity) to investigate the role and functions of GrA within cells. Finally, the fluorescent properties of SK15.5 may allow the visualization of GrA within NKs, CTLs, or other cells under both physiological and pathophysiological conditions.</div></div><div id="sec3_1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26134" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26134" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Reagents</h3><div class="NLM_p">All chemical and biological reagents were purchased from commercial suppliers and used without further purification. The following reagents for solid phase peptide synthesis, Rink Amide RA resin (particle size 200–300 mesh, loading 0.74 mmol/g), 2-chlorotrityl chloride resin, Fmoc-protected amino acids, biotin, 1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>] pyridinium 3-oxid hexafluorophosphate (HATU), piperidine, diisopropylcarbodiimide (DICI), and trifluoroacetic acid (TFA), were purchased from Iris Biotech GmbH (Marktredwitz, Germany); anhydrous <i>N</i>-hydroxybenzotriazole (HOBt, purity >98%) was purchased from Creosauls; 2,4,6-collidine (2,4,6-trimethylpyridine), HPLC pure acetonitrile, and triisopropylsilane (TIPS) were purchased from Sigma-Aldrich (Poznan, Poland); and <i>N,N</i>-diisopropylethylamie (DIPEA) was purchased from VWR International (Gdansk, Poland). Solvents such as <i>N</i>,<i>N</i>-dimethylformamide (DMF), dichloromethane (DCM), methanol (MeOH), diethyl ether (Et<sub>2</sub>O), and acetic acid (AcOH) were purchased from Avantor (Gliwice, Poland). Cy5 was purchased from Lumiprobe GmbH (Hannover, Germany). Human neutrophil elastase was purchased from BioCentrum (Kraków, Poland) and proteinase 3 from Athens Research Technology (Athens, USA). Anti-GrA antibody (rabbit, monoclonal, ab209205) was obtained from Abcam (Cambridge, Great Britain). All cell lines were purchased from DSMZ (Braunschweig, Germany) or ATCC (Manassas, USA). All kinetic assays were performed using a Spectra Max Gemini XPS spectrofluorometer (Molecular Devices) on 96-well Corning plates. The parameters of the process were as follows: 37 °C; excitation/emission wavelength, 355/460 nm; cutoff, 455 nm; varying concentrations of substrates and enzyme. All measurements were performed in duplicate, and the data represent the average of these repetitions</div><div class="NLM_p">Peptide substrates, inhibitors, and ABPs were purified by HPLC (Waters M600 solvent delivery module, Waters M2489 detector system, semipreparative Wide Pore C8 Discovery column, Waters sp z.o.o., Warszawa, Poland). The solvent components were as follows: water/0.1% TFA for phase A and acetonitrile/0.1% TFA for phase B.</div><div class="NLM_p">The purity of each substrate was determined by an analytical HPLC system using a Discovery Bio Wide Pore C8 analytical column (flow rate of 1.5 mL/min). The solvent composition was as follows: phase A (water/0.1% TFA) and phase B (acetonitrile/0.1% TFA); gradient, from 100% A to 0% A over a period of 20 min. The purity of all compounds was ≥95%. The molecular weight of each substrate was confirmed by high-resolution mass spectrometry on a WATERS LCT premier XE high-resolution mass spectrometer with electrospray ionization (ESI) and a time-of-flight (TOF) module (the mass spectra and HPLC diagrams are in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b02042/suppl_file/jm9b02042_si_001.pdf" class="ext-link">Supporting Information</a>).</div><div id="sec3_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18">1.  Library and Substrate Synthesis</h4><div class="NLM_p last">The P1-Arg HyCoSuL library was synthesized according to the protocols described before.<a onclick="showRef(event, 'ref25 ref30 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref25 ref30 ref31 ref32">(25,30−32)</a> Individual substrates were synthesized using the solid phase peptide synthesis method on Rink amide resin loaded with 0.74 mmol/g. First, 3 g of resin was placed in a glass peptide synthesis vessel and swollen in DCM for 30 min. The N-terminal Fmoc-protecting group was removed with 20% piperidine in DMF solution with 5, 5, and 20 min cycles, and deprotection was confirmed with a ninhydrin test. Afterward, the resin was washed thoroughly 6 times with DMF. Next, Fmoc-ACC (2.5 equiv, 2.44 g) preactivated with DICI (2.5 equiv, 716 μL) and HOBt (2.5 equiv, 832 mg) was added to the resin, and the reaction was carried out for 24 h. Next, the reaction was repeated with 1.5 equiv of all of the above reagents to improve the yield of Fmoc-ACC coupling. Then, the Fmoc-protecting groups were removed in the same manner as above. The resin was washed 6 times with DMF, 3 times with DCM, and 3 times with MeOH and dried over P<sub>2</sub>O<sub>5</sub> overnight. Dried resin was divided into 100 mg portions and placed in a multiwell peptide synthesis vessel. To attach the P1 amino acid, 3 equiv of Fmoc-P1 was preactivated with 3 equiv of HATU (85 mg) and 3 equiv of collidine (29 μL) and was poured onto the resin and gently agitated for 24 h. After the deprotection cycle, the P2, P3, and P4 amino acids were coupled to the H<sub>2</sub>N-P1-ACC resin with DICI (2.5 equiv) and HOBt (2.5 equiv) as coupling reagents in coupling/deprotection cycles, with each time preceded with a ninhydrin test. The N-terminus was protected with an acetyl group using 5 equiv of AcOH, 5 equiv of HBTU, and 5 equiv of DIPEA in DMF. After solvent removal, the resin was washed six times with DMF to remove excess reagents, three times with DCM, and three times with MeOH, dried over P<sub>2</sub>O<sub>5</sub>, and cleaved from the resin with a mixture of TFA/TIPS/H2O (%, v/v/v, 95:2.5:2.5). The crude product was purified by HPLC and lyophilized. The purity of each substrate was confirmed by analytical HPLC and analyzed using HRMS. Substrates were dissolved in peptide-grade DMSO to a 20 mM concentration and stored at −80 °C until use.</div></div><div id="sec3_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19">2.  Production of Recombinant Grs</h4><div class="NLM_p last">Production and purification of recombinant proteases from <i>Pichia pastoris</i> was as previously described for GrB.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Proteases were confirmed >95% active by interaction with inhibitors SERPINB9 (for GrB) or SERPINB6 (for GrA and GrK).</div></div><div id="sec3_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20">3.  Substrate Kinetic Assays</h4><div class="NLM_p last">All kinetic assays were carried out in assay buffer (50 mM Tris, 100 mM NaCl, 5 mM CaCl<sub>2</sub>, 0.01% (v/v) Tween-20, pH 7.5). P1-Arg HyCoSuL library screening was performed on 384-well plates. Then, 0.75 μL of each library substrate was placed in one well, and preincubated (30 min, 37 °C) enzyme solution was added. The total reaction volume was 30 μL, the final library concentration was 250 μM, and the enzyme concentration was 777 nM. Individual substrates SK8–SK28 were also tested on 384-well plates with a total substrate concentration of 67 μM and an enzyme concentration of 583 nM for GrA, 488 nM for GrK and 18 nM for GrB. Briefly, substrates with an initial concentration of 20 mM were diluted 10 times in DMSO, and 1 μL of each substrate dilution was placed in separate wells on the plate, followed by 29 μL of enzyme in assay buffer. The increase in fluorescence over time was measured at excitation/emission wavelengths of 355/460 nm and a cutoff of 455 nm. The kinetic parameters of the best substrate SK15 were determined at a GrA concentration of 93 nM. Substrate concentrations ranged from 1000 μM to 7.81 μM. In the first well of the plate, 3 μL of SK15 was added to 30 μL of assay buffer, and half of this solution was transferred to a second well containing 15 μL of assay buffer. The substrate was serially diluted until eight well, in which the final substrate concentration reached 7.81 μM. Afterward, 15 μL of preincubated (30 min, 37 °C) enzyme was added to each well, and the fluorescence readout was measured. The <i>K</i><sub>M</sub>, <i>V</i><sub>max</sub>, and <i>k</i><sub>cat</sub>/<i>K</i><sub>M</sub> parameters were calculated in Excel software and GraphPad Prism using Michaelis–Menten nonlinear regression.</div></div><div id="sec3_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21">4.  Inhibitor and ABP Synthesis</h4><div class="NLM_p last">Inhibitor and ABP synthesis was divided into two main parts. (1) The first step involved the synthesis of the peptide fragment (MeOSuc/Cy5-Tic-Gly-Oic–OH). 200 mg of 2-chlorotrityl resin in the peptide synthesis vessel was swollen in 10 mL of DCM for 20 min, and then, the resin was washed once with DCM. Next, the first amino acid, Fmoc-Oic-OH (2.5 equiv, 312 mg), was dissolved in 3 mL of DCM (the mixture was cloudy) in a Falcon tube, activated for 3 min with 2.5 equiv of DIPEA (139 μL), poured onto the resin under an argon atmosphere, and stirred overnight on a platform rocker at room temperature. Then, the resin was washed 3 times with DCM, and the remaining active sites on the CTC resin were deactivated with 5 mL of methanol solution, DCM/MeOH/DIPEA (80:15:5 v/v/v), for 60 min. Next, the resin was washed 6 times with DMF, and the Fmoc-protecting group was removed with 20% piperidine solution in 5, 5, and 20 min cycles. Next, Fmoc-Gly-OH (2.5 equiv, 237 mg) was coupled to the resin with HOBt (2.5 equiv, 120 mg) and DICI (2.5 equiv, 100 μL). Afterward, the resin was deprotected as above, and Fmoc-Tic-OH (2.5 equiv, 319 mg) was coupled in the same manner. In the case of SK15I inhibitor synthesis, the last step included the coupling of the methoxy-succinyl group (2.5 equiv, 108 mg) using 2.5 equiv of HOBt and 2.5 equiv of DICI to protect the N-terminus of the peptide. The last step in the synthesis of the SK15.5 probe was the coupling of the Cy5 fluorophore to the N-terminus. First, the amount of chlorotrityl resin was reduced to 50 mg. Cy5 NHS ester (0.3 equiv, 15 mg) was dissolved in 1 mL of DMF, preactivated with DIPEA (0.45 equiv, 6.25 μL), and stirred with the resin overnight. Next, the resin was washed with DMF and dried with DCM and MeOH. The peptide was cleaved from the resin with 5 mL of the cleavage cocktail DCM:TFE:AcOH (8:1:1, v/v/v). The supernatant was collected, 20 mL of hexane was added, and the crude product was evaporated and lyophilized. The reactive warhead molecule Cbz-Arg<sup>P</sup>(Boc)<sub>2</sub>-(OPh)<sub>2</sub> was synthesized according to the protocol described elsewhere.<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> The Cbz-protecting group was removed using hydrogen and palladium on carbon. The obtained H<sub>2</sub>N-Arg<sup>P</sup>(Boc)<sub>2</sub>-(OPh)<sub>2</sub> was coupled with a peptide fragment (Cy5-Ahx-Tic-Gly-Oic-COOH) with HATU (1.2 equiv) and collidine (4 equiv), and the reaction was monitored with analytical HPLC. When the reaction was completed, the product was extracted to ethyl acetate with 5% NaHCO<sub>3</sub>, 5% citric acid, and brine and dried over P<sub>2</sub>O<sub>5</sub>. Boc-protecting groups on the guanidine group of arginine were removed with a mixture of TFA:DCM:TIPS (80:15:5, v/v/v) for 30 min at room temperature, and TFA was removed with argon flow. The crude product was dissolved in 2 mL of DMSO, purified on semipreparative HPLC on a Wide Pore C8 column (Discovery), lyophilized, dissolved to 10 mM in DMSO, and stored at −80 °C until use.</div></div><div id="sec3_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22">5.  Inhibition Kinetic Assays and <i>k</i><sub>obs</sub>/I Determination</h4><div class="NLM_p">The inhibitor (SK15I) and probe (SK15.5) kinetic experiments were performed in 50 mM Tris, 100 mM NaCl, 5 mM CaCl<sub>2</sub>, and 0.01% (v/v) Tween-20, pH 7.5, assay buffer on a Spectra Max Gemini XPS spectrofluorometer (Molecular Devices) in 96-well plates (Corning Costar opaque) at 37 °C. The second-order inhibition rate constant (<i>k</i><sub>obs</sub>/I expressed in M<sup>–1</sup> s<sup>–1</sup>) toward GrA was measured under pseudo-first-order kinetic conditions. Ten microliters of substrate (SK15) was mixed with varying inhibitor or probe concentrations and incubated at 37 °C for 15 min. Afterward, 20 μL of active GrA (350 nM) preincubated at 37 °C was added, and the fluorescence increase over time was measured at the excitation/emission wavelengths of 355/460 nm and a cutoff of 455 nm. The <i>k</i><sub>obs</sub>/I parameters were calculated in Excel software and GraphPad Prism. Experiments are presented as the average of at least three experiments with standard deviation.</div><div class="NLM_p last">For residual activity assay, 488 nM of GrK, 18 nM of GrB, 74 nM of GrM, 3.3 nM of hNE, and 3.4 nM of PR3 were preincubated with 10 μM of SK15I and SK15.5 (separately) for 30 min at 37 °C and then added to 200 μM of GrK<sub>ref</sub> or GrB<sub>ref</sub>, respectively. As a control, enzymes without inhibitor and probe were tested. The activity was measured on a spectrofluorometer with excitation/emission wavelengths of 355/460 nm and a cutoff of 455 nm.</div></div><div id="sec3_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23">6.  In-Gel Detection of GrA Using SK15.5</h4><div class="NLM_p last">GrA and SK15.5 concentrations and incubation times were varied depending on the analysis performed. In the incubation time optimization assay, the enzyme and probe concentrations were constant (250 nM), and the incubation time ranged from 0 to 120 min. In probe concentration optimization, enzyme (250 nM) was incubated with SK15.5 at varying concentrations ranging from 10 to 500 nM. Incubation was carried out in assay buffer (50 mM Tris, 100 mM NaCl, 5 mM CaCl<sub>2</sub>, 0.01% (v/v) Tween-20, pH 7.5) at 37 °C. Enzyme was incubated with probe in a volume of 80 μL, followed by the reduction with 40 μL of 3 × SDS/DTT for 5 min at 95 °C. Samples were centrifuged (3 min, 4500<i>g</i>), and 40 μL of each sample was run on 4–12% Bis-Tris 10-well gels (Invitrogen). The first well was loaded with 2 μL of the protein marker PageRuler Plus Prestained Protein Ladder (Thermo Scientific). Electrophoresis was conducted at 165 V for 37 min at room temperature. Proteins were then transferred to a nitrocellulose membrane (0.2 μm, Bio-Rad) for 60 min at 10 V. Labeled proteins were detected at 658 nm (red channel) using a Sapphire Biomolecular Imager (Azure Biosystems). When testing the exclusive binding of SK15.5 with active enzyme, GrA forms (active and inactive) were diluted in assay buffer (50 mM Tris, 100 mM NaCl, 5 mM CaCl<sub>2</sub>, 0.01% (v/v) Tween-20, pH 7.5) to final concentration of 250 nM and preincubated with SK15I (1 μM) for 1 h at 37 °C, where indicated. Next, SK15.5 probe was added at final concentration 500 nM and incubated for another 1 h. Total sample volume was 80 μL, to which 40 μL of 3× SDS/DTT was added and boiled for 5 min at 95 °C. Samples were centrifuged (3 min, 4500<i>g</i>), and 40 μL of each sample was run on 4–12% Bis-Tris 10-well gels (Invitrogen). Each form of the enzyme without SK15.5 and probe itself were run as controls. The first well was loaded with 2 μL of protein marker PageRuler Plus Prestained Protein Ladder (Thermo Scientific). Electrophoresis and membrane transfer were conducted in the same manner as above. Membrane was blocked with 5% BSA in Tris-buffered saline with 0.1% Tween 20 (TBS-T) for 60 min at room temperature and immunostained overnight with rabbit antihuman monoclonal GrA antibody (Abcam, ab209205, 1:1000) and Alexa Fluor 488 goat antirabbit secondary antibody (Invitrogen, A32731, 1:2000) for 1 h. Membrane was imaged with 488 nm (green channel) and 658 nm (red channel) using Sapphire Biomolecular Imager (Azure Biosystems).</div></div><div id="sec3_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24">7.  GrA Labeling in Cell Lysates</h4><div class="NLM_p last">To label active GrA in the NK92 cell line (DSMZ, ACC 488), proper cell culture media had to be applied. Initially, we used three different combinations of media and supplements, according to the literature and supplier information. Cells were cultured in (1) alpha MEM (Sigma-Aldrich), supplemented with 12.5% FBS (Sigma-Aldrich), 12.5% horse serum (HS; Sigma-Aldrich), 2 mM glutamine (Sigma-Aldrich), and 50 U/mL IL-2 (PeproTech); (2) alpha MEM with 12.5% FBS, 12.5% HS, 2 mM glutamine, 0.1 mM β-mercaptoethanol (Gibco), 100 U/mL penicillin/streptomycin (Biowest), and 100 U/mL Il-2; (3) RPMI 1640 media (Sigma-Aldrich) with 10% FBS, 2 mM glutamine, 0.1 mM β-mercaptoethanol, 100 U/mL penicillin/streptomycin, and 100 U/mL Il-2. For further experiments, we chose medium (1) because it resulted in the highest viability of NK92 cells. Cells in the log growth phase were counted with Trypan Blue, and 10<sup>7</sup> cells were collected, centrifuged, washed with DBPS (Corning), and resuspended in 1 mL of lysis buffer (25 mM KCl, 5 mM MgCl<sub>2</sub>, 10 mM Tris-HCl, and 0.5% v/v Tween-20, pH 8.0). Cells were lysed by sonication (20 kHz for 10 s), and lysates were aliquoted and stored at −20 °C until use. Samples were loaded onto 4–12% bis-Tris 10-well gels (Invitrogen), and SDS-PAGE was carried out at 165 V for 37 min at room temperature. Afterward, the membrane was blocked with 5% BSA in Tris-buffered saline with 0.1% Tween 20 (TBS-T). Then, the membrane was treated with rabbit antihuman monoclonal GrA antibody (Abcam, ab209205, 1:1000) and Alexa Fluor 488 goat antirabbit secondary antibody (Invitrogen, A32731, 1:2000) and imaged at 488 nm (green channel) and 658 nm (red channel) using a Sapphire Biomolecular Imager (Azure Biosystems).</div></div><div id="sec3_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25">8.  SK15.5 and SK15I Cytotoxicity Assay</h4><div class="NLM_p last">A cytotoxicity assay was performed with a commercially available MTS kit to determine the number of viable cells (Promega, G3580). MDA-MB-231 (ATCC, HTB-26) cells were cultured in DMEM (Biowest) supplemented with 10% FBS and penicillin/streptomycin at a concentration of 100 IU/mL and 100 μg/mL, respectively. Cells at a density of 5 × 10<sup>5</sup> cells/mL and a final volume of 90 μL were seeded on a clear 96-well plate 24 h prior to the assay for proper attachment. SK15.5 probe or SK15I inhibitor at concentrations ranging from 1 nM to 10 μM diluted in 10 μL of DPBS was added to cells and incubated for 24 h at 37 °C with 5% CO<sub>2</sub>. Afterward, 20 μL of MTS reagent was added to each well and incubated at 37 °C with 5% CO<sub>2</sub> for 2 h. The absorbance was measured on a spectrophotometer (Molecular Devices) at 490 nm, and the results were analyzed with SoftMax and GraphPad Prism software.</div></div><div id="sec3_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26">9.  SK15.5 Cell Membrane Permeability Assay</h4><div class="NLM_p last">NK92 cells (DSMZ, ACC 488) in the log phase were counted using Trypan Blue, and 10<sup>6</sup> cells were collected in separate vials, centrifuged, and resuspended in 1 mL of fresh culture media (alpha MEM supplemented with 12.5% FBS, 12.5% HS, 2 mM glutamine, and 100 U/mL IL-2). To each vial containing 10<sup>6</sup> cells, SK15.5 probe at a final concentration of 1 μM was added, and the vial was incubated for the indicated time (between 0 and 8 h). As a control, GrA was preinhibited with 10 μM (final concentration) SK15I and incubated for 1 h at 37 °C, and untreated samples were simultaneously incubated. At the end of every time point, the cells were centrifuged, washed twice with DPBS, centrifuged and treated with 100 μL of 1× SDS/DTT solution. Afterward, the samples were sonicated (10 kHz for 10 s) and loaded onto a 4–12% Bis-Tris 10-well gel (Invitrogen). SDS-PAGE was run at 165 V for 37 min, and membrane transfer (10 V, 60 min) was subsequently performed. Then, the membrane was blocked with 5% BSA in TBS-T for 60 min at room temperature and immunostained with rabbit antihuman monoclonal GrA antibody (Abcam, ab209205, 1:1000) and Alexa Fluor 488 goat antirabbit secondary antibody (Invitrogen, A32731, 1:2000) and imaged at 488 nm (green channel) and 658 nm (red channel) using a Sapphire Biomolecular Imager (Azure Biosystems).</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27"><a href="/doi/suppl/10.1021/acs.jmedchem.9b02042" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10621" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10621" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b02042?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b02042</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Detailed screening profiles; full-size blots; substrate selectivity testing; amino acid structures used in libraries and individual compounds; HPLC and MS analyses for substrates, inhibitor, and activity-based probe (<a href="/doi/suppl/10.1021/acs.jmedchem.9b02042/suppl_file/jm9b02042_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b02042/suppl_file/jm9b02042_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b02042/suppl_file/jm9b02042_si_001.pdf">jm9b02042_si_001.pdf (2.7 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b02042/suppl_file/jm9b02042_si_002.csv">jm9b02042_si_002.csv (3.02 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b02042" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62284" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62284" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paulina Kasperkiewicz</span> - <span class="hlFld-Affiliation affiliation">Wrocław
University of Science and Technology, Department of Chemical Biology and Bioimaging, Wyb. Wyspiańskiego 29, 50-370 Wroclaw, Poland</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1291-047X" title="Orcid link">http://orcid.org/0000-0002-1291-047X</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#9dedfce8f1f4f3fcb3f6fceeedf8eff6f4f8eaf4fee7ddedeaefb3f8f9e8b3edf1"><span class="__cf_email__" data-cfemail="017160746d686f602f6a60727164736a68647668627b417176732f6465742f716d">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sonia Kołt</span> - <span class="hlFld-Affiliation affiliation">Wrocław
University of Science and Technology, Department of Chemical Biology and Bioimaging, Wyb. Wyspiańskiego 29, 50-370 Wroclaw, Poland</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4076-0445" title="Orcid link">http://orcid.org/0000-0003-4076-0445</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tomasz Janiszewski</span> - <span class="hlFld-Affiliation affiliation">Wrocław
University of Science and Technology, Department of Chemical Biology and Bioimaging, Wyb. Wyspiańskiego 29, 50-370 Wroclaw, Poland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dion Kaiserman</span> - <span class="hlFld-Affiliation affiliation">Monash
University, Monash Biomedicine Discovery
Institute, Department
of Biochemistry and Molecular Biology, 23 Innovation Walk, Clayton, VIC 3800, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sylwia Modrzycka</span> - <span class="hlFld-Affiliation affiliation">Wrocław
University of Science and Technology, Department of Chemical Biology and Bioimaging, Wyb. Wyspiańskiego 29, 50-370 Wroclaw, Poland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Scott J. Snipas</span> - <span class="hlFld-Affiliation affiliation">NCI-designated
Cancer Center, Sanford-Burnham Prebys Medical
Discovery Institute, La Jolla, California 92037, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guy Salvesen</span> - <span class="hlFld-Affiliation affiliation">NCI-designated
Cancer Center, Sanford-Burnham Prebys Medical
Discovery Institute, La Jolla, California 92037, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marcin Dra̧g</span> - <span class="hlFld-Affiliation affiliation">Wrocław
University of Science and Technology, Department of Chemical Biology and Bioimaging, Wyb. Wyspiańskiego 29, 50-370 Wroclaw, Poland</span>; 
    <span class="hlFld-Affiliation affiliation">NCI-designated
Cancer Center, Sanford-Burnham Prebys Medical
Discovery Institute, La Jolla, California 92037, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8510-1967" title="Orcid link">http://orcid.org/0000-0001-8510-1967</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Phillip I. Bird</span> - <span class="hlFld-Affiliation affiliation">Monash
University, Monash Biomedicine Discovery
Institute, Department
of Biochemistry and Molecular Biology, 23 Innovation Walk, Clayton, VIC 3800, Australia</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>S.K. and P.K. designed the experiments and analyzed the results. S.K., P.K., T.J., and S.M. performed experiments. S.K. and P.K. wrote the manuscript. S.K., T.J., S.M., S.J.S., G.S., D.K., M.D., P.I.B., and P.K. reviewed the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11286" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11286" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the HOMING Programme, a Grant Project of the Foundation for Polish Science, funded by the European Union under agreement No. 2016-3/24. P.K. is the beneficiary of L’Oreal Poland and Polish Ministry of Science and Higher Education scholarships.</p></div><table summary="" class="NLM_def-list" id="para"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">ABP</td><td class="NLM_def"><p class="first last">activity-based probe</p></td></tr><tr><td class="NLM_term">ACC</td><td class="NLM_def"><p class="first last">7-amino-4-carbamoylmethylcoumarin</p></td></tr><tr><td class="NLM_term">AFC</td><td class="NLM_def"><p class="first last">7-amido-4-trifluoromethylcoumarin trifluoroacetate</p></td></tr><tr><td class="NLM_term">MNA</td><td class="NLM_def"><p class="first last">4-methoxy-2-naphthylamide trifluoroacetate</p></td></tr><tr><td class="NLM_term">HyCoSuL</td><td class="NLM_def"><p class="first last">hybrid combinatorial substrate library</p></td></tr><tr><td class="NLM_term">RFU</td><td class="NLM_def"><p class="first last">relative fluorescence unit</p></td></tr><tr><td class="NLM_term">GrA</td><td class="NLM_def"><p class="first last">granzyme A</p></td></tr><tr><td class="NLM_term">GrB</td><td class="NLM_def"><p class="first last">granzyme B</p></td></tr><tr><td class="NLM_term">GrK</td><td class="NLM_def"><p class="first last">granzyme K</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i31">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82054" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82054" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 35 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Metkar, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menaa, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallich, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freudenberg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raja, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Froelich, C. J.</span></span> <span> </span><span class="NLM_article-title">Human and Mouse Granzyme A Induce a Proinflammatory Cytokine Response</span>. <i>Immunity</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">720</span>– <span class="NLM_lpage">733</span>, <span class="refDoi"> DOI: 10.1016/j.immuni.2008.08.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1016%2Fj.immuni.2008.08.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=18951048" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVaqsL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2008&pages=720-733&author=S.+S.+Metkarauthor=C.+Menaaauthor=J.+Pardoauthor=B.+Wangauthor=R.+Wallichauthor=M.+Freudenbergauthor=S.+Kimauthor=S.+M.+Rajaauthor=L.+Shiauthor=M.+M.+Simonauthor=C.+J.+Froelich&title=Human+and+Mouse+Granzyme+A+Induce+a+Proinflammatory+Cytokine+Response&doi=10.1016%2Fj.immuni.2008.08.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Human and mouse granzyme A induce a proinflammatory cytokine response</span></div><div class="casAuthors">Metkar, Sunil S.; Menaa, Cheikh; Pardo, Julian; Wang, Baikun; Wallich, Reinhard; Freudenberg, Marina; Kim, Stephen; Raja, Srikumar M.; Shi, Lianfa; Simon, Markus M.; Froelich, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Immunity</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">720-733</span>CODEN:
                <span class="NLM_cas:coden">IUNIEH</span>;
        ISSN:<span class="NLM_cas:issn">1074-7613</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Granzyme A (GzmA) is considered a major proapoptotic protease.  The authors have discovered that GzmA-induced cell death involves rapid membrane damage that depends on the synergy between micromolar concns. of GzmA and sublytic perforin (PFN).  Ironically, GzmA and GzmB, independent of their catalytic activity, both mediated this swift necrosis.  Even without PFN, lower concns. of human GzmA stimulated monocytes to secrete proinflammatory cytokines [interleukin-1β (IL-1β), TNFα, and IL-6] that were blocked by a caspase-1 inhibitor.  Moreover, murine GzmA and GzmA+ cytotoxic T lymphocytes (CTLs) induce IL-1β from primary mouse macrophages, and GzmA-/- mice resist lipopolysaccharide-induced toxicity.  Thus, the granule secretory pathway plays an unexpected role in inflammation, with GzmA acting as an endogenous modulator.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcOH7m4-hE5bVg90H21EOLACvtfcHk0lihlKHobzhGJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVaqsL7M&md5=0fd5c17d7f7a7cba24940c03d305851c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.immuni.2008.08.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.immuni.2008.08.014%26sid%3Dliteratum%253Aachs%26aulast%3DMetkar%26aufirst%3DS.%2BS.%26aulast%3DMenaa%26aufirst%3DC.%26aulast%3DPardo%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DWallich%26aufirst%3DR.%26aulast%3DFreudenberg%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DRaja%26aufirst%3DS.%2BM.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DSimon%26aufirst%3DM.%2BM.%26aulast%3DFroelich%26aufirst%3DC.%2BJ.%26atitle%3DHuman%2520and%2520Mouse%2520Granzyme%2520A%2520Induce%2520a%2520Proinflammatory%2520Cytokine%2520Response%26jtitle%3DImmunity%26date%3D2008%26volume%3D29%26spage%3D720%26epage%3D733%26doi%3D10.1016%2Fj.immuni.2008.08.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinvalet, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chowdhury, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlesinger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lieberman, J.</span></span> <span> </span><span class="NLM_article-title">The Cytotoxic T Lymphocyte Protease Granzyme A cleaves and Inactivates Poly(adenosine 5′-diphosphate-ribose) Polymerase-1</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">1205</span>– <span class="NLM_lpage">1216</span>, <span class="refDoi"> DOI: 10.1182/blood-2008-12-195768</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1182%2Fblood-2008-12-195768" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=19506301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVSgsbfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2009&pages=1205-1216&author=P.+Zhuauthor=D.+Martinvaletauthor=D.+Chowdhuryauthor=D.+Zhangauthor=A.+Schlesingerauthor=J.+Lieberman&title=The+Cytotoxic+T+Lymphocyte+Protease+Granzyme+A+cleaves+and+Inactivates+Poly%28adenosine+5%E2%80%B2-diphosphate-ribose%29+Polymerase-1&doi=10.1182%2Fblood-2008-12-195768"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The cytotoxic T lymphocyte protease granzyme A cleaves and inactivates poly(adenosine 5'-diphosphate-ribose) polymerase-1</span></div><div class="casAuthors">Zhu, Pengcheng; Martinvalet, Denis; Chowdhury, Dipanjan; Zhang, Dong; Schlesinger, Ann; Lieberman, Judy</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1205-1216</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Granzyme A (GzmA) in killer cells induces caspase-independent programmed cell death.  In this study, we show that GzmA cleaves the DNA damage sensor poly(ADP-ribose) polymerase-1 (PARP-1) after Lys498 in its automodification domain, sepg. the DNA binding domain from the catalytic domain, which interferes with repair of GzmA-induced DNA damage and enhances susceptibility to GzmA-mediated death.  Overexpressing K498A PARP-1 reduces GzmA-mediated death and drives dying cells to necrosis rather than apoptosis.  Conversely, inhibiting or genetically disrupting PARP-1 enhances cell vulnerability.  The N-terminal GzmA cleavage fragment of PARP-1 acts as a PARP-1 dominant neg., binding to DNA and blocking DNA repair.  Disrupting PARP-1, which is also a caspase target, is therefore required for efficient apoptosis by both caspase-independent and caspase-dependent pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7GDXiodn5VrVg90H21EOLACvtfcHk0lhuyGNBv0y0hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVSgsbfJ&md5=5f94645fd597047c79d10a1e81001daa</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1182%2Fblood-2008-12-195768&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2008-12-195768%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DP.%26aulast%3DMartinvalet%26aufirst%3DD.%26aulast%3DChowdhury%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DSchlesinger%26aufirst%3DA.%26aulast%3DLieberman%26aufirst%3DJ.%26atitle%3DThe%2520Cytotoxic%2520T%2520Lymphocyte%2520Protease%2520Granzyme%2520A%2520cleaves%2520and%2520Inactivates%2520Poly%2528adenosine%25205%25E2%2580%25B2-diphosphate-ribose%2529%2520Polymerase-1%26jtitle%3DBlood%26date%3D2009%26volume%3D114%26spage%3D1205%26epage%3D1216%26doi%3D10.1182%2Fblood-2008-12-195768" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Law, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukoyanova, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voskoboinik, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caradoc-Davies, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baran, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunstone, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Angelo, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orlova, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coulibaly, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verschoor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Browne, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciccone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuiper, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bird, P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trapani, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saibil, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whisstock, J. C.</span></span> <span> </span><span class="NLM_article-title">The Structural Basis for Membrane Binding and Pore Formation by Lymphocyte Perforin</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">447</span>– <span class="NLM_lpage">451</span>, <span class="refDoi"> DOI: 10.1038/nature09518</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1038%2Fnature09518" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=21037563" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlGrsLjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=447-451&author=R.+H.+Lawauthor=N.+Lukoyanovaauthor=I.+Voskoboinikauthor=T.+T.+Caradoc-Daviesauthor=K.+Baranauthor=M.+A.+Dunstoneauthor=M.+E.+D%E2%80%99Angeloauthor=E.+V.+Orlovaauthor=F.+Coulibalyauthor=S.+Verschoorauthor=K.+A.+Browneauthor=A.+Cicconeauthor=M.+J.+Kuiperauthor=P.+I.+Birdauthor=J.+A.+Trapaniauthor=H.+R.+Saibilauthor=J.+C.+Whisstock&title=The+Structural+Basis+for+Membrane+Binding+and+Pore+Formation+by+Lymphocyte+Perforin&doi=10.1038%2Fnature09518"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The structural basis for membrane binding and pore formation by lymphocyte perforin</span></div><div class="casAuthors">Law, Ruby H. P.; Lukoyanova, Natalya; Voskoboinik, Ilia; Caradoc-Davies, Tom T.; Baran, Katherine; Dunstone, Michelle A.; D'Angelo, Michael E.; Orlova, Elena V.; Coulibaly, Fasseli; Verschoor, Sandra; Browne, Kylie A.; Ciccone, Annette; Kuiper, Michael J.; Bird, Phillip I.; Trapani, Joseph A.; Saibil, Helen R.; Whisstock, James C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7322</span>),
    <span class="NLM_cas:pages">447-451</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Natural killer cells and cytotoxic T lymphocytes accomplish the critically important function of killing virus-infected and neoplastic cells.  They do this by releasing the pore-forming protein perforin and granzyme proteases from cytoplasmic granules into the cleft formed between the abutting killer and target cell membranes.  Perforin, a 67-kilodalton multidomain protein, oligomerizes to form pores that deliver the pro-apoptopic granzymes into the cytosol of the target cell.  The importance of perforin is highlighted by the fatal consequences of congenital perforin deficiency, with more than 50 different perforin mutations linked to familial haemophagocytic lymphohistiocytosis (type 2 FHL).  Here we elucidate the mechanism of perforin pore formation by detg. the X-ray crystal structure of monomeric murine perforin, together with a cryo-electron microscopy reconstruction of the entire perforin pore.  Perforin is a thin 'key-shaped' mol., contg. an amino-terminal membrane attack complex perforin-like (MACPF)/cholesterol dependent cytolysin (CDC) domain followed by an epidermal growth factor (EGF) domain that together with the extreme carboxy-terminal sequence, forms a central shelf-like structure.  A C-terminal C2 domain mediates initial, Ca2+-dependent membrane binding.  Most unexpectedly, however, electron microscopy reveals that the orientation of the perforin MACPF domain in the pore is inside-out relative to the subunit arrangement in CDCs.  These data reveal remarkable flexibility in the mechanism of action of the conserved MACPF/CDC fold and provide new insights into how related immune defense mols. such as complement proteins assemble into pores.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvdSkxAx6O_bVg90H21EOLACvtfcHk0lhuyGNBv0y0hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlGrsLjP&md5=39539b6f65789792a95773391b6a3a21</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnature09518&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09518%26sid%3Dliteratum%253Aachs%26aulast%3DLaw%26aufirst%3DR.%2BH.%26aulast%3DLukoyanova%26aufirst%3DN.%26aulast%3DVoskoboinik%26aufirst%3DI.%26aulast%3DCaradoc-Davies%26aufirst%3DT.%2BT.%26aulast%3DBaran%26aufirst%3DK.%26aulast%3DDunstone%26aufirst%3DM.%2BA.%26aulast%3DD%25E2%2580%2599Angelo%26aufirst%3DM.%2BE.%26aulast%3DOrlova%26aufirst%3DE.%2BV.%26aulast%3DCoulibaly%26aufirst%3DF.%26aulast%3DVerschoor%26aufirst%3DS.%26aulast%3DBrowne%26aufirst%3DK.%2BA.%26aulast%3DCiccone%26aufirst%3DA.%26aulast%3DKuiper%26aufirst%3DM.%2BJ.%26aulast%3DBird%26aufirst%3DP.%2BI.%26aulast%3DTrapani%26aufirst%3DJ.%2BA.%26aulast%3DSaibil%26aufirst%3DH.%2BR.%26aulast%3DWhisstock%26aufirst%3DJ.%2BC.%26atitle%3DThe%2520Structural%2520Basis%2520for%2520Membrane%2520Binding%2520and%2520Pore%2520Formation%2520by%2520Lymphocyte%2520Perforin%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D447%26epage%3D451%26doi%3D10.1038%2Fnature09518" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whisstock, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voskoboinik, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trapani, J. A.</span></span> <span> </span><span class="NLM_article-title">Protecting a Serial Killer: Pathways for Perforin Trafficking and Self-Defence Ensure Sequential Target Cell Death</span>. <i>Trends Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">406</span>– <span class="NLM_lpage">412</span>, <span class="refDoi"> DOI: 10.1016/j.it.2012.04.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1016%2Fj.it.2012.04.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=22608996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVymtrnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=406-412&author=J.+A.+Lopezauthor=A.+J.+Brennanauthor=J.+C.+Whisstockauthor=I.+Voskoboinikauthor=J.+A.+Trapani&title=Protecting+a+Serial+Killer%3A+Pathways+for+Perforin+Trafficking+and+Self-Defence+Ensure+Sequential+Target+Cell+Death&doi=10.1016%2Fj.it.2012.04.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Protecting a serial killer: pathways for perforin trafficking and self-defence ensure sequential target cell death</span></div><div class="casAuthors">Lopez, Jamie A.; Brennan, Amelia J.; Whisstock, James C.; Voskoboinik, Ilia; Trapani, Joseph A.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Immunology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">406-412</span>CODEN:
                <span class="NLM_cas:coden">TIRMAE</span>;
        ISSN:<span class="NLM_cas:issn">1471-4906</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Considerable progress has been made in understanding how cytotoxic lymphocytes use the highly toxic pore-forming protein perforin to eliminate dangerous cells, while remaining refractory to lysis.  At least two mechanisms jointly preserve the killer cell: the C-terminal residues of perforin dictate its rapid export from the endoplasmic reticulum (ER), whose milieu otherwise favors pore formation; perforin is then stored in secretory granules whose acidity prevent its oligomerization.  Following exocytosis, perforin delivers the proapoptotic protease, granzyme B, into the target cell by disrupting its plasma membrane.  Although the precise mechanism of perforin/granzyme synergy remains controversial, the recently defined crystal structure of the perforin monomer and cryo-electron microscopy (EM) of the entire pore suggest that passive transmembrane granzyme diffusion is the dominant proapoptotic mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0hU-DsmbO5rVg90H21EOLACvtfcHk0liC2C75c1CETQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVymtrnE&md5=fab538509cf3aba68fdb78bab97e1b31</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.it.2012.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.it.2012.04.001%26sid%3Dliteratum%253Aachs%26aulast%3DLopez%26aufirst%3DJ.%2BA.%26aulast%3DBrennan%26aufirst%3DA.%2BJ.%26aulast%3DWhisstock%26aufirst%3DJ.%2BC.%26aulast%3DVoskoboinik%26aufirst%3DI.%26aulast%3DTrapani%26aufirst%3DJ.%2BA.%26atitle%3DProtecting%2520a%2520Serial%2520Killer%253A%2520Pathways%2520for%2520Perforin%2520Trafficking%2520and%2520Self-Defence%2520Ensure%2520Sequential%2520Target%2520Cell%2520Death%26jtitle%3DTrends%2520Immunol.%26date%3D2012%26volume%3D33%26spage%3D406%26epage%3D412%26doi%3D10.1016%2Fj.it.2012.04.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
Damme, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maurer-Stroh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colaert, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timmerman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenhaber, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandekerckhove, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gevaert, K.</span></span> <span> </span><span class="NLM_article-title">The Substrate Specificity Profile of Human Granzyme A</span>. <i>Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>391</i></span>,  <span class="NLM_fpage">983</span>– <span class="NLM_lpage">997</span>, <span class="refDoi"> DOI: 10.1515/bc.2010.096</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1515%2Fbc.2010.096" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=20536382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFyhur3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=391&publication_year=2010&pages=983-997&author=P.+Van%0ADammeauthor=S.+Maurer-Strohauthor=H.+Haoauthor=N.+Colaertauthor=E.+Timmermanauthor=F.+Eisenhaberauthor=J.+Vandekerckhoveauthor=K.+Gevaert&title=The+Substrate+Specificity+Profile+of+Human+Granzyme+A&doi=10.1515%2Fbc.2010.096"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The substrate specificity profile of human granzyme A</span></div><div class="casAuthors">Van Damme, Petra; Maurer-Stroh, Sebastian; Hao, Han; Colaert, Niklaas; Timmerman, Evy; Eisenhaber, Frank; Vandekerckhove, Joel; Gevaert, Kris</div><div class="citationInfo"><span class="NLM_cas:title">Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">391</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">983-997</span>CODEN:
                <span class="NLM_cas:coden">BICHF3</span>;
        ISSN:<span class="NLM_cas:issn">1431-6730</span>.
    
            (<span class="NLM_cas:orgname">Walter de Gruyter GmbH & Co. KG</span>)
        </div><div class="casAbstract">The exact biol. function of granzyme A, a granule-assocd. serine protease belonging to the tryptase family of proteases, is still a matter of debate because conflicting roles have been suggested, such as initiation of caspase-independent apoptosis-like cell death and endogenous modulation of inflammatory processes.  In contrast to its well-studied family member, granzyme B, far less is known about the physiol. targets of granzyme A.  Using an N-terminal peptide-centric proteomics technol., the substrate specificity of human granzyme A was extensively characterized at the level of macromol. protein substrates.  Overall, more than 260 cleavage sites, almost exclusively favoring basic residues at the P1 position, in approx. 200 unique protein substrates, including the well-known in vitro substrates APEX-endonuclease 1 and different histones, were identified.  Further substrate characterization was used to delineate phys. properties in the substrate specificity profiles, which further highlights important aspects in protease/substrate biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIzGI0uDl6ybVg90H21EOLACvtfcHk0liC2C75c1CETQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFyhur3M&md5=9b5fa0c047066e1683e4ada5781f298e</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1515%2Fbc.2010.096&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252Fbc.2010.096%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BDamme%26aufirst%3DP.%26aulast%3DMaurer-Stroh%26aufirst%3DS.%26aulast%3DHao%26aufirst%3DH.%26aulast%3DColaert%26aufirst%3DN.%26aulast%3DTimmerman%26aufirst%3DE.%26aulast%3DEisenhaber%26aufirst%3DF.%26aulast%3DVandekerckhove%26aufirst%3DJ.%26aulast%3DGevaert%26aufirst%3DK.%26atitle%3DThe%2520Substrate%2520Specificity%2520Profile%2520of%2520Human%2520Granzyme%2520A%26jtitle%3DBiol.%2520Chem.%26date%3D2010%26volume%3D391%26spage%3D983%26epage%3D997%26doi%3D10.1515%2Fbc.2010.096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Masson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tschopp, J.</span></span> <span> </span><span class="NLM_article-title">A Family of Serine Esterases in Lytic Granules of Cytolytic T Lymphocytes</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1987</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">679</span>– <span class="NLM_lpage">685</span>, <span class="refDoi"> DOI: 10.1016/0092-8674(87)90544-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1016%2F0092-8674%2887%2990544-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=3555842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADyaL1cXht1OhtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=1987&pages=679-685&author=D.+Massonauthor=J.+Tschopp&title=A+Family+of+Serine+Esterases+in+Lytic+Granules+of+Cytolytic+T+Lymphocytes&doi=10.1016%2F0092-8674%2887%2990544-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">A family of serine esterases in lytic granules of cytolytic T lymphocytes</span></div><div class="casAuthors">Masson, Daniele; Tschopp, Juerg</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">679-85</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    </div><div class="casAbstract">Cytoplasmic granules of cytolytic T lymphocytes (CTLs) contain, in addn. to the pore-forming protein perforin, a family of highly homologous serine esterases, granzymes A-H.  The serine esterase affinity label diisopropyl fluorophosphate reacts strongly with granzymes A and D, to a lesser extent with B, E, G, and H, and not at all with C and F.  For granzymes A and D, synthetic substrates have been found.  Antibodies raised against granzyme B strongly crossreact with A, G, and H, and antibodies to granzyme D recognize C, E, and F.  These antigenic relationships correlate with similarities in the N-terminal amino acid sequences.  At least 60% homol. is obsd. between the 8 proteins, and all are similar to rat mast cell protease 2.  Sequence anal. suggests the identity of granzyme A with a protease predicted from a CTL-specific cDNA clone (H factor) and of granzyme B, G, or H with a protein encoded by the CTL-specific cDNA clone CTLA 1/CCP 1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4eATGzgITYrVg90H21EOLACvtfcHk0liC2C75c1CETQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXht1OhtLk%253D&md5=bb34a562c3d9ea9c83c28714673ab9a8</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2887%2990544-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252887%252990544-7%26sid%3Dliteratum%253Aachs%26aulast%3DMasson%26aufirst%3DD.%26aulast%3DTschopp%26aufirst%3DJ.%26atitle%3DA%2520Family%2520of%2520Serine%2520Esterases%2520in%2520Lytic%2520Granules%2520of%2520Cytolytic%2520T%2520Lymphocytes%26jtitle%3DCell%26date%3D1987%26volume%3D49%26spage%3D679%26epage%3D685%26doi%3D10.1016%2F0092-8674%2887%2990544-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bots, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Bostelen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rademaker, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Offringa, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medema, J. P.</span></span> <span> </span><span class="NLM_article-title">Serpins Prevent Granzyme-Induced Death in a Species-Specific Manner</span>. <i>Immunol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">79</span>– <span class="NLM_lpage">86</span>, <span class="refDoi"> DOI: 10.1111/j.1440-1711.2005.01417.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1111%2Fj.1440-1711.2005.01417.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=16405655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BD28Xhs1CqsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2006&pages=79-86&author=M.+Botsauthor=L.+Van+Bostelenauthor=M.+T.+Rademakerauthor=R.+Offringaauthor=J.+P.+Medema&title=Serpins+Prevent+Granzyme-Induced+Death+in+a+Species-Specific+Manner&doi=10.1111%2Fj.1440-1711.2005.01417.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Serpins prevent granzyme-induced death in a species-specific manner</span></div><div class="casAuthors">Bots, Michael; Van Bostelen, Liesbeth; Rademaker, Mirjam Tga; Offringa, Rienk; Medema, Jan Paul</div><div class="citationInfo"><span class="NLM_cas:title">Immunology and Cell Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">79-86</span>CODEN:
                <span class="NLM_cas:coden">ICBIEZ</span>;
        ISSN:<span class="NLM_cas:issn">0818-9641</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Asia Pty Ltd.</span>)
        </div><div class="casAbstract">Expression of serine protease inhibitors (serpins) is one of the mechanisms used by tumor cells to escape immune surveillance.  Previously, we have shown that expression of serpins SPI-6 and SPI-CI, resp., renders tumor cells resistant to granzyme B (GrB)-mediated death and granzyme M (GrM)-mediated death.  To obtain better insight into the interaction between serpins and their target proteases, we investigated the roles of protease inhibitor (PI)-9 and SPI-6 in the resistance to GrB-mediated and CD95-mediated death in further detail.  Neither human PI-9 nor its murine orthologue SPI-6 was capable of preventing CD95-induced apoptosis in murine or human cells, indicating that these serpins do not inhibit the activation of apical caspases in this pathway.  High expression of PI-9 or SPI-6 did prevent apoptosis induced by human GrB.  Strikingly, only SPI-6, and not PI-9, was capable of inhibiting murine GrB, suggesting that a difference in enzymic specificity exists between the mouse and the human granzymes.  In agreement with this suggestion, murine GrB was clearly less effective in inducing apoptosis in human cells.  Similar species specificity was also obsd. for SPI-CI and GrM when either their capacity to assoc. or the effectiveness of GrM-induced cytotoxicity was analyzed.  Our findings therefore indicate a species diversity that has a clear effect on mixed in vitro effector target settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdxJrk6vU67bVg90H21EOLACvtfcHk0ljifY8hn5ebkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xhs1CqsLg%253D&md5=5acd0431f4c9c6b6532e7641ea1ecfa6</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1111%2Fj.1440-1711.2005.01417.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1440-1711.2005.01417.x%26sid%3Dliteratum%253Aachs%26aulast%3DBots%26aufirst%3DM.%26aulast%3DVan%2BBostelen%26aufirst%3DL.%26aulast%3DRademaker%26aufirst%3DM.%2BT.%26aulast%3DOffringa%26aufirst%3DR.%26aulast%3DMedema%26aufirst%3DJ.%2BP.%26atitle%3DSerpins%2520Prevent%2520Granzyme-Induced%2520Death%2520in%2520a%2520Species-Specific%2520Manner%26jtitle%3DImmunol.%2520Cell%2520Biol.%26date%3D2006%26volume%3D84%26spage%3D79%26epage%3D86%26doi%3D10.1111%2Fj.1440-1711.2005.01417.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bird, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutton, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coughlin, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Jong, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trapani, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bird, P. I.</span></span> <span> </span><span class="NLM_article-title">A Cytosolic Granzyme B Inhibitor Related to the Viral Apoptotic Regulator Cytokine Response Modifier A Is Present in Cytotoxic Lymphocytes</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>271</i></span>,  <span class="NLM_fpage">27802</span>– <span class="NLM_lpage">27809</span>, <span class="refDoi"> DOI: 10.1074/jbc.271.44.27802</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1074%2Fjbc.271.44.27802" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=8910377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADyaK28XmvVWrs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=271&publication_year=1996&pages=27802-27809&author=J.+Sunauthor=C.+H.+Birdauthor=V.+Suttonauthor=L.+McDonaldauthor=P.+B.+Coughlinauthor=T.+A.+De+Jongauthor=J.+A.+Trapaniauthor=P.+I.+Bird&title=A+Cytosolic+Granzyme+B+Inhibitor+Related+to+the+Viral+Apoptotic+Regulator+Cytokine+Response+Modifier+A+Is+Present+in+Cytotoxic+Lymphocytes&doi=10.1074%2Fjbc.271.44.27802"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">A cytosolic Granzyme B inhibitor related to the viral apoptotic regulator cytokine response modifier A is present in cytotoxic lymphocytes</span></div><div class="casAuthors">Sun, Jiuru; Bird, Catherina H.; Sutton, Vivien; McDonald, Lisa; Coughlin, Paul B.; De Jong, Tanya A.; Trapani, Joseph A.; Bird, Phillip I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">271</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">27802-27809</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Using a polymerase chain reaction strategy we identified a serine proteinase inhibitor (serpin) in human bone marrow that is related to the cellular serpin proteinase inhibitor 6 (PI-6) and the viral serpin cytokine response modifier A (CrmA).  This serpin, proteinase inhibitor 9 (PI-9), has an unusual reactive center P1(Glu)-P1'(Cys), which suggests that it inhibits serine proteinases that cleave after acidic residues.  The only known serine proteinase with this specificity is granzyme B, a granule cytotoxin produced by cytotoxic lymphocytes.  To test the interaction of PI-9 with granzyme B we prepd. recombinant hexa-histidine tagged PI-9 in a yeast expression system.  Addn. of the recombinant protein to native granzyme B resulted in an SDS-resistant complex typical of serpin-serine proteinase interactions.  Further anal. showed that complex formation followed bimol. kinetics with a second order rate const. of 1.7 × 106 M-1s-1, which is in the range for a physiol. significant serpin-proteinase interaction.  Recombinant PI-9 also completely abrogated granzyme B and perforin-mediated cytotoxicity in vitro.  Examn. of PI-9 mRNA distribution demonstrated that it is expressed in immune tissue, primarily in lymphocytes.  The highest levels of PI-9 mRNA and protein were obsd. in natural killer cell leukemia cell lines and in interleukin-2 stimulated peripheral blood mono-nuclear cells, which also produce granzyme B.  Like PI-6, PI-9 was shown to be a cytosolic protein that is not secreted.  Fractionation of natural killer cells and stimulated peripheral blood mononuclear cells demonstrated that PI-9 is in a sep. subcellular compartment to granzyme B.  These results suggest that PI-9 serves to inactivate misdirected granzyme B following cytotoxic cell degranulation.  This may explain why cytotoxic cells are not damaged by their own granzyme B during destruction of abnormal cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNfbHp6EO9D7Vg90H21EOLACvtfcHk0ljifY8hn5ebkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmvVWrs7g%253D&md5=6d268203aab57b62ccd40354900bf05f</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1074%2Fjbc.271.44.27802&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.271.44.27802%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DBird%26aufirst%3DC.%2BH.%26aulast%3DSutton%26aufirst%3DV.%26aulast%3DMcDonald%26aufirst%3DL.%26aulast%3DCoughlin%26aufirst%3DP.%2BB.%26aulast%3DDe%2BJong%26aufirst%3DT.%2BA.%26aulast%3DTrapani%26aufirst%3DJ.%2BA.%26aulast%3DBird%26aufirst%3DP.%2BI.%26atitle%3DA%2520Cytosolic%2520Granzyme%2520B%2520Inhibitor%2520Related%2520to%2520the%2520Viral%2520Apoptotic%2520Regulator%2520Cytokine%2520Response%2520Modifier%2520A%2520Is%2520Present%2520in%2520Cytotoxic%2520Lymphocytes%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1996%26volume%3D271%26spage%3D27802%26epage%3D27809%26doi%3D10.1074%2Fjbc.271.44.27802" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bird, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutton, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirst, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trapani, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bird, P. I.</span></span> <span> </span><span class="NLM_article-title">Selective Regulation of Apoptosis: the Cytotoxic Lymphocyte Serpin Proteinase Inhibitor 9 Protects Against Granzyme B-Mediated Apoptosis Without Perturbing the Fas Cell Death Pathway</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">6387</span>– <span class="NLM_lpage">6398</span>, <span class="refDoi"> DOI: 10.1128/MCB.18.11.6387</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1128%2FMCB.18.11.6387" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=9774654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADyaK1cXmvF2gu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1998&pages=6387-6398&author=C.+H.+Birdauthor=V.+R.+Suttonauthor=J.+Sunauthor=C.+E.+Hirstauthor=A.+Novakauthor=S.+Kumarauthor=J.+A.+Trapaniauthor=P.+I.+Bird&title=Selective+Regulation+of+Apoptosis%3A+the+Cytotoxic+Lymphocyte+Serpin+Proteinase+Inhibitor+9+Protects+Against+Granzyme+B-Mediated+Apoptosis+Without+Perturbing+the+Fas+Cell+Death+Pathway&doi=10.1128%2FMCB.18.11.6387"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Selective regulation of apoptosis: the cytotoxic lymphocyte serpin proteinase inhibitor 9 protects against granzyme B-mediated apoptosis without perturbing the Fas cell death pathway</span></div><div class="casAuthors">Bird, Catherina H.; Sutton, Vivien R.; Sun, Jiuru; Hirst, Claire E.; Novak, Andrea; Kumar, Sharad; Trapani, Joseph A.; Bird, Phillip I.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6387-6398</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Cytotoxic lymphocytes (CLs) induce caspase activation and apoptosis of target cells either through Fas activation or through release of granule cytotoxins, particularly granzyme B.  CLs themselves resist granule-mediated apoptosis but are eventually cleared via Fas-mediated apoptosis.  Here we show that the CL cytoplasmic serpin proteinase inhibitor 9 (PI-9) can protect transfected cells against apoptosis induced by either purified granzyme B and perforin or intact CLs.  A PI-9 P1 mutant (Glu to Asp) is a 100-fold-less-efficient granzyme B inhibitor that no longer protects against granzyme B-mediated apoptosis.  PI-9 is highly specific for granzyme B because it does not inhibit eight of the nine caspases tested or protect transfected cells against Fas-mediated apoptosis.  In contrast, the P1(Asp) mutant is an effective caspase inhibitor that protects against Fas-mediated apoptosis.  We propose that PI-9 shields CLs specifically against misdirected granzyme B to prevent autolysis or fratricide, but it does not interfere with homeostatic deletion via Fas-mediated apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqG5qYadKK0ibVg90H21EOLACvtfcHk0ljifY8hn5ebkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmvF2gu78%253D&md5=1a0ec08419e9661d697bdacea4b1a1b9</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1128%2FMCB.18.11.6387&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.18.11.6387%26sid%3Dliteratum%253Aachs%26aulast%3DBird%26aufirst%3DC.%2BH.%26aulast%3DSutton%26aufirst%3DV.%2BR.%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DHirst%26aufirst%3DC.%2BE.%26aulast%3DNovak%26aufirst%3DA.%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DTrapani%26aufirst%3DJ.%2BA.%26aulast%3DBird%26aufirst%3DP.%2BI.%26atitle%3DSelective%2520Regulation%2520of%2520Apoptosis%253A%2520the%2520Cytotoxic%2520Lymphocyte%2520Serpin%2520Proteinase%2520Inhibitor%25209%2520Protects%2520Against%2520Granzyme%2520B-Mediated%2520Apoptosis%2520Without%2520Perturbing%2520the%2520Fas%2520Cell%2520Death%2520Pathway%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1998%26volume%3D18%26spage%3D6387%26epage%3D6398%26doi%3D10.1128%2FMCB.18.11.6387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaiserman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plasman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gevaert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Damme, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bird, P. I.</span></span> <span> </span><span class="NLM_article-title">Identification of Serpinb6b as a Species-Specific Mouse Granzyme A Inhibitor Suggests Functional Divergence Between Human and Mouse Granzyme A</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>289</i></span>,  <span class="NLM_fpage">9408</span>– <span class="NLM_lpage">9417</span>, <span class="refDoi"> DOI: 10.1074/jbc.M113.525808</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1074%2Fjbc.M113.525808" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=24505135" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltVSrsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=9408-9417&author=D.+Kaisermanauthor=S.+E.+Stewartauthor=K.+Plasmanauthor=K.+Gevaertauthor=P.+Van+Dammeauthor=P.+I.+Bird&title=Identification+of+Serpinb6b+as+a+Species-Specific+Mouse+Granzyme+A+Inhibitor+Suggests+Functional+Divergence+Between+Human+and+Mouse+Granzyme+A&doi=10.1074%2Fjbc.M113.525808"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Serpinb6b as a Species-specific Mouse Granzyme A Inhibitor Suggests Functional Divergence between Human and Mouse Granzyme A</span></div><div class="casAuthors">Kaiserman, Dion; Stewart, Sarah E.; Plasman, Kim; Gevaert, Kris; Van Damme, Petra; Bird, Phillip I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">9408-9417</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The granzyme family serine proteases are key effector mols. expressed by cytotoxic lymphocytes.  The physiol. role of granzyme (Gzm) A is controversial, with significant debate over its ability to induce death in target cells.  Here, we investigate the natural inhibitors of GzmA.  We employed substrate phage display and positional proteomics to compare substrate specificities of mouse (m) and human (h) GzmA at the peptide and proteome-wide levels and we used the resulting substrate specificity profiles to search for potential inhibitors from the intracellular serpin family.  We identified Serpinb6b as a potent inhibitor of mGzmA.  Serpinb6b interacts with mGzmA, but not hGzmA, with an assocn. const. of 1.9 ± 0.8 × 105 m-1 s-1 and a stoichiometry of inhibition of 1.8.  Mouse GzmA is over five times more cytotoxic than hGzmA when delivered into P815 target cells with streptolysin O, whereas transfection of target cells with a Serpinb6b cDNA increases the EC50 value of mGzmA 13-fold, without affecting hGzmA cytotoxicity.  Unexpectedly, we also found that Serpinb6b employs an exosite to specifically inhibit dimeric but not monomeric mGzmA.  The identification of an intracellular inhibitor specific for mGzmA only indicates that a lineage-specific increase in GzmA cytotoxic potential has driven cognate inhibitor evolution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2Tt5NBdjb6rVg90H21EOLACvtfcHk0ljirB9X7BXOKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltVSrsbY%253D&md5=53a28002b6b0d7146d4f8aa2e46b5fad</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.525808&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.525808%26sid%3Dliteratum%253Aachs%26aulast%3DKaiserman%26aufirst%3DD.%26aulast%3DStewart%26aufirst%3DS.%2BE.%26aulast%3DPlasman%26aufirst%3DK.%26aulast%3DGevaert%26aufirst%3DK.%26aulast%3DVan%2BDamme%26aufirst%3DP.%26aulast%3DBird%26aufirst%3DP.%2BI.%26atitle%3DIdentification%2520of%2520Serpinb6b%2520as%2520a%2520Species-Specific%2520Mouse%2520Granzyme%2520A%2520Inhibitor%2520Suggests%2520Functional%2520Divergence%2520Between%2520Human%2520and%2520Mouse%2520Granzyme%2520A%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D9408%26epage%3D9417%26doi%3D10.1074%2Fjbc.M113.525808" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Masson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tschopp, J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Lymphocyte Protease Granzyme A by Antithrombin III</span>. <i>Mol. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1283</span>– <span class="NLM_lpage">1289</span>, <span class="refDoi"> DOI: 10.1016/0161-5890(88)90043-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1016%2F0161-5890%2888%2990043-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=3266293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADyaL1MXhsFSgtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1988&pages=1283-1289&author=D.+Massonauthor=J.+Tschopp&title=Inhibition+of+Lymphocyte+Protease+Granzyme+A+by+Antithrombin+III&doi=10.1016%2F0161-5890%2888%2990043-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of lymphocyte protease granzyme A by antithrombin III</span></div><div class="casAuthors">Masson, Daniele; Tschopp, Juerg</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Immunology</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1283-9</span>CODEN:
                <span class="NLM_cas:coden">MOIMD5</span>;
        ISSN:<span class="NLM_cas:issn">0161-5890</span>.
    </div><div class="casAbstract">T-lymphocytes contain a cytoplasmic granule assocd. homo-dimeric protease designated granzyme A.  Upon T-cell-target cell interaction, the granules undergo exocytosis and granzyme A, and other granule constituents, are released.  Granzyme A secreted into plasma is immediately inactivated by antithrombin III.  The rate of complex formation is enhanced 400-fold in the presence of heparin.  Two different complexes are generated: granzyme A-antithrombin III and granzyme A-(antithrombin III)2, resp., indicating that both active centers of granzyme A are functional.  Thus, the proteolytic activity of lymphocyte protease granzyme A, whose physiol. relevant function is unknown, is well regulated in plasma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopFdXHadn9I7Vg90H21EOLACvtfcHk0ljirB9X7BXOKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXhsFSgtr4%253D&md5=884f8720d9344b4587c2a5c0641e0a4b</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2F0161-5890%2888%2990043-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0161-5890%252888%252990043-0%26sid%3Dliteratum%253Aachs%26aulast%3DMasson%26aufirst%3DD.%26aulast%3DTschopp%26aufirst%3DJ.%26atitle%3DInhibition%2520of%2520Lymphocyte%2520Protease%2520Granzyme%2520A%2520by%2520Antithrombin%2520III%26jtitle%3DMol.%2520Immunol.%26date%3D1988%26volume%3D25%26spage%3D1283%26epage%3D1289%26doi%3D10.1016%2F0161-5890%2888%2990043-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allison, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodard, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gault, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ewoldt, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kam, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abuelyaman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudig, D.</span></span> <span> </span><span class="NLM_article-title">Characterization, Application and Potential Uses of Biotin-Tagged Inhibitors for Lymphocyte Serine Proteases (granzymes)</span>. <i>Mol. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">615</span>– <span class="NLM_lpage">623</span>, <span class="refDoi"> DOI: 10.1016/0161-5890(96)00025-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1016%2F0161-5890%2896%2900025-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=8760273" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADyaK28XkvVOntLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=1996&pages=615-623&author=U.+Winklerauthor=N.+J.+Allisonauthor=S.+L.+Woodardauthor=R.+A.+Gaultauthor=G.+R.+Ewoldtauthor=C.+M.+Kamauthor=A.+Abuelyamanauthor=J.+C.+Powersauthor=D.+Hudig&title=Characterization%2C+Application+and+Potential+Uses+of+Biotin-Tagged+Inhibitors+for+Lymphocyte+Serine+Proteases+%28granzymes%29&doi=10.1016%2F0161-5890%2896%2900025-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization, application and potential uses of biotin-tagged inhibitors for lymphocyte serine proteases (granzymes)</span></div><div class="casAuthors">Winkler, Ulrike; Allison, N. Janine; Woodard, Susan L.; Gault, Ruth A.; Ewoldt, Gerald R.; Kam, Chih-Min; Abuelyaman, Ahmed; Powers, James C.; Hudig, Dorothy</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Immunology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">7/8</span>),
    <span class="NLM_cas:pages">615-623</span>CODEN:
                <span class="NLM_cas:coden">MOIMD5</span>;
        ISSN:<span class="NLM_cas:issn">0161-5890</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Cytotoxic lymphocytes and natural killer cells are able to kill their target cells in minutes.  The death of the target cell occurs after the release of cytoplasmic granules from the effector cell.  These granules contain the pore-forming protein perforin and serine proteases (granzymes).  To date 10 genes encoding lymphocyte granzymes have been discovered; of these only four have been purified and characterized for their substrate specificity.  Several are predicted to have a common chymase-like specificity which is found in the granule exts.  Others may need to be enriched as active enzymes before they can be evaluated for substrate hydrolysis.  Due to the limitations of detection by substrate hydrolysis, a more sensitive method for the detection of dil. granules was needed.  We report the differing reactivities of seven biotin (Bi)-tagged isocoumarin (IC) inhibitors for Asp-ase, chymase, tryptase and Met-ase granzymes.  The inhibitors contained different substituents at their no. 3 position: methoxy (OMe), ethoxy (OEt), propoxy (OPr) or 2-phenylethoxy (OEtPh) groups.  The OMe group conferred general reactivity, whereas the OEtPh group conferred selective reactivity with chymase granzymes.  The inhibitors that contained the longest aminocaproyl (Aca) spacers between the biotin-tag and the isocoumarin ring mediated the most stable granzyme inactivation.  These inhibitors were the most effective at blocking lysis of red blood cells by the granule exts.  The inhibitors were used in protein blotting expts. where the biotin was detected with an avidin-enzyme complex.  Over 10 granzymes were labeled by the inhibitor Bi-Aca-Aca-IC-OMe.  The inhibitors detected granzymes when they were not readily detected by substrate hydrolysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfZi3CZ2KeK7Vg90H21EOLACvtfcHk0ljirB9X7BXOKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XkvVOntLs%253D&md5=7dc2064eacd8abff24b5ac8d49b09817</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2F0161-5890%2896%2900025-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0161-5890%252896%252900025-9%26sid%3Dliteratum%253Aachs%26aulast%3DWinkler%26aufirst%3DU.%26aulast%3DAllison%26aufirst%3DN.%2BJ.%26aulast%3DWoodard%26aufirst%3DS.%2BL.%26aulast%3DGault%26aufirst%3DR.%2BA.%26aulast%3DEwoldt%26aufirst%3DG.%2BR.%26aulast%3DKam%26aufirst%3DC.%2BM.%26aulast%3DAbuelyaman%26aufirst%3DA.%26aulast%3DPowers%26aufirst%3DJ.%2BC.%26aulast%3DHudig%26aufirst%3DD.%26atitle%3DCharacterization%252C%2520Application%2520and%2520Potential%2520Uses%2520of%2520Biotin-Tagged%2520Inhibitors%2520for%2520Lymphocyte%2520Serine%2520Proteases%2520%2528granzymes%2529%26jtitle%3DMol.%2520Immunol.%26date%3D1996%26volume%3D33%26spage%3D615%26epage%3D623%26doi%3D10.1016%2F0161-5890%2896%2900025-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mahrus, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, C. S.</span></span> <span> </span><span class="NLM_article-title">Selective Chemical Functional Probes of Granzymes A and B Reveal Granzyme B is a Major Effector of Natural Killer Cell-Mediated Lysis of Target Cells</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">567</span>– <span class="NLM_lpage">577</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2005.03.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1016%2Fj.chembiol.2005.03.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=15911377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BD2MXks1aku7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2005&pages=567-577&author=S.+Mahrusauthor=C.+S.+Craik&title=Selective+Chemical+Functional+Probes+of+Granzymes+A+and+B+Reveal+Granzyme+B+is+a+Major+Effector+of+Natural+Killer+Cell-Mediated+Lysis+of+Target+Cells&doi=10.1016%2Fj.chembiol.2005.03.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Selective chemical functional probes of granzymes A and B reveal granzyme B Is a major effector of natural killer cell-mediated lysis of target cells</span></div><div class="casAuthors">Mahrus, Sami; Craik, Charles S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">567-577</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The mechanism of target cell lysis in cytotoxic lymphocyte-mediated death is not well understood, and the role of granzymes in this process is unclear.  Chem. functional probes were thus prepd. for the major granzymes A and B to deconvolute their role in natural killer cell-mediated lysis of target cells.  These biotinylated and substrate specificity-based di-Ph phosphonates allowed facile evaluation of selectivity through activity-based profiling in cell lysates and intact cells.  Both inhibitors were extremely selective in vitro and in cells.  Use of these inhibitors in cell-based assays revealed granzyme A to be a minor effector and granzyme B to be a major effector of target cell lysis by natural killer cells.  These studies indicate that the proapoptotic granzyme B functions also as a pronecrotic effector of target cell death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBTIufZM9xDrVg90H21EOLACvtfcHk0ljirB9X7BXOKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXks1aku7Y%253D&md5=07dbb4ea5f6e2186a0ab880280247391</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2005.03.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2005.03.006%26sid%3Dliteratum%253Aachs%26aulast%3DMahrus%26aufirst%3DS.%26aulast%3DCraik%26aufirst%3DC.%2BS.%26atitle%3DSelective%2520Chemical%2520Functional%2520Probes%2520of%2520Granzymes%2520A%2520and%2520B%2520Reveal%2520Granzyme%2520B%2520is%2520a%2520Major%2520Effector%2520of%2520Natural%2520Killer%2520Cell-Mediated%2520Lysis%2520of%2520Target%2520Cells%26jtitle%3DChem.%2520Biol.%26date%3D2005%26volume%3D12%26spage%3D567%26epage%3D577%26doi%3D10.1016%2Fj.chembiol.2005.03.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buzza, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bird, P. I.</span></span> <span> </span><span class="NLM_article-title">Extracellular Granzymes: Current Perspectives</span>. <i>Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>387</i></span>,  <span class="NLM_fpage">827</span>– <span class="NLM_lpage">837</span>, <span class="refDoi"> DOI: 10.1515/BC.2006.106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1515%2FBC.2006.106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=16913832" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BD28XnvFanurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=387&publication_year=2006&pages=827-837&author=M.+S.+Buzzaauthor=P.+I.+Bird&title=Extracellular+Granzymes%3A+Current+Perspectives&doi=10.1515%2FBC.2006.106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Extracellular granzymes: current perspectives</span></div><div class="casAuthors">Buzza, Marguerite S.; Bird, Phillip I.</div><div class="citationInfo"><span class="NLM_cas:title">Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">387</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">827-837</span>CODEN:
                <span class="NLM_cas:coden">BICHF3</span>;
        ISSN:<span class="NLM_cas:issn">1431-6730</span>.
    
            (<span class="NLM_cas:orgname">Walter de Gruyter GmbH & Co. KG</span>)
        </div><div class="casAbstract">A review.  Granzyme A (GrA) and granzyme B (GrB) play key roles in the induction of target cell death induced by cytotoxic lymphocytes.  While these roles have been extensively studied, it has become apparent that both granzymes also possess extracellular activities.  Sol. granzymes are found extracellularly in normal plasma and are elevated in a no. of diseases, ranging from viral and bacterial infections to autoimmune diseases.  Here, the authors discuss current knowledge of extracellular granzyme substrates, inhibitors, and functions, and the pathol. consequences of extracellular granzymes in disease.  In addn., the authors provide new evidence for the role of glycosaminoglycan-binding sites of granzymes in extracellular matrix remodeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkEnSXuKAZTLVg90H21EOLACvtfcHk0ljt-OOAqGKLew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XnvFanurY%253D&md5=fffabaab6d38fb53098a05ab8a53d851</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1515%2FBC.2006.106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252FBC.2006.106%26sid%3Dliteratum%253Aachs%26aulast%3DBuzza%26aufirst%3DM.%2BS.%26aulast%3DBird%26aufirst%3DP.%2BI.%26atitle%3DExtracellular%2520Granzymes%253A%2520Current%2520Perspectives%26jtitle%3DBiol.%2520Chem.%26date%3D2006%26volume%3D387%26spage%3D827%26epage%3D837%26doi%3D10.1515%2FBC.2006.106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresford, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novina, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pommier, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lieberman, J.</span></span> <span> </span><span class="NLM_article-title">Cleaving the Oxidative Repair Protein Ape1 Enhances Cell Death Mediated by Granzyme A</span>. <i>Nat. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">145</span>– <span class="NLM_lpage">153</span>, <span class="refDoi"> DOI: 10.1038/ni885</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1038%2Fni885" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=12524539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmslKisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2003&pages=145-153&author=Z.+Fanauthor=P.+J.+Beresfordauthor=D.+Zhangauthor=Z.+Xuauthor=C.+D.+Novinaauthor=A.+Yoshidaauthor=Y.+Pommierauthor=J.+Lieberman&title=Cleaving+the+Oxidative+Repair+Protein+Ape1+Enhances+Cell+Death+Mediated+by+Granzyme+A&doi=10.1038%2Fni885"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Cleaving the oxidative repair protein Ape1 enhances cell death mediated by granzyme A</span></div><div class="casAuthors">Fan, Zusen; Beresford, Paul J.; Zhang, Dong; Xu, Zhan; Novina, Carl D.; Yoshida, Akira; Pommier, Yves; Lieberman, Judy</div><div class="citationInfo"><span class="NLM_cas:title">Nature Immunology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">145-153</span>CODEN:
                <span class="NLM_cas:coden">NIAMCZ</span>;
        ISSN:<span class="NLM_cas:issn">1529-2908</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The cytolytic T lymphocyte protease granzyme A (GzmA) initiates a caspase-independent cell death pathway.  Here the authors report that the rate-limiting enzyme of DNA base excision repair, apurinic endonuclease-1 (Ape1), which is also known as redox factor-1 (Ref-1), or APEX, binds to GzmA and is contained in the SET complex, a macromol. complex of 270-420 kDa that is assocd. with the endoplasmic reticulum and is targeted by GzmA during cell-mediated death.  GzmA cleaves Ape1 after Lys31 and destroys its known oxidative repair functions.  In so doing, GzmA may block cellular repair and force apoptosis.  In support of this, cells with silenced Ape1 expression are more sensitive, whereas cells overexpressing noncleavable Ape1 are more resistant, to GzmA-mediated death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZLqGUNSZPx7Vg90H21EOLACvtfcHk0ljt-OOAqGKLew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmslKisw%253D%253D&md5=552ae476ca917c2ac5cb52b8ab094e9a</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fni885&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fni885%26sid%3Dliteratum%253Aachs%26aulast%3DFan%26aufirst%3DZ.%26aulast%3DBeresford%26aufirst%3DP.%2BJ.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DNovina%26aufirst%3DC.%2BD.%26aulast%3DYoshida%26aufirst%3DA.%26aulast%3DPommier%26aufirst%3DY.%26aulast%3DLieberman%26aufirst%3DJ.%26atitle%3DCleaving%2520the%2520Oxidative%2520Repair%2520Protein%2520Ape1%2520Enhances%2520Cell%2520Death%2520Mediated%2520by%2520Granzyme%2520A%26jtitle%3DNat.%2520Immunol.%26date%3D2003%26volume%3D4%26spage%3D145%26epage%3D153%26doi%3D10.1038%2Fni885" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresford, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lieberman, J.</span></span> <span> </span><span class="NLM_article-title">HMG2 Interacts with the Nucleosome Assembly Protein SET and is a Target of the Cytotoxic T-lymphocyte Protease Granzyme A</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">2810</span>– <span class="NLM_lpage">2820</span>, <span class="refDoi"> DOI: 10.1128/MCB.22.8.2810-2820.2002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1128%2FMCB.22.8.2810-2820.2002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=11909973" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BD38XislWnt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2002&pages=2810-2820&author=Z.+Fanauthor=P.+J.+Beresfordauthor=D.+Zhangauthor=J.+Lieberman&title=HMG2+Interacts+with+the+Nucleosome+Assembly+Protein+SET+and+is+a+Target+of+the+Cytotoxic+T-lymphocyte+Protease+Granzyme+A&doi=10.1128%2FMCB.22.8.2810-2820.2002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">HMG2 interacts with the nucleosome assembly protein SET and is a target of the cytotoxic T-lymphocyte protease granzyme A</span></div><div class="casAuthors">Fan, Zusen; Beresford, Paul J.; Zhang, Dong; Lieberman, Judy</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2810-2820</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The cytotoxic T-lymphocyte protease granzyme A induces caspase-independent cell death in which DNA single-stranded nicking is obsd. instead of oligonucleosomal fragmentation.  A 270- to 420-kDa endoplasmic reticulum-assocd. complex (SET complex) contg. the nucleosome assembly protein SET, the tumor suppressor pp32, and the base excision repair enzyme APE can induce single-stranded DNA damage in isolated nuclei in a granzyme A-dependent manner.  The normal functions of the SET complex are unknown, but the functions of its components suggest that it is involved in activating transcription and DNA repair.  The authors now find that the SET complex contains DNA binding and bending activities mediated by the chromatin-assocd. protein HMG2.  HMG2 facilitates assembly of nucleoprotein higher-order structures by bending and looping DNA or by stabilizing under-wound DNA.  HMG2 is in the SET complex and coppts. with SET.  By confocal microscopy, it is obsd. that cytoplasmic HMG2 colocalizes with SET in assocn. with the endoplasmic reticulum, but most nuclear HMG2 is unassocd. with SET.  This phys. assocn. suggests that HMG2 may facilitate the nucleosome assembly, transcriptional activation, and DNA repair functions of SET and/or APE.  HMG2, like SET and APE, is a physiol. relevant granzyme A substrate in targeted cells.  HMG1, however, is not a substrate.  Granzyme A cleavage after Lys65 in the midst of HMG box A destroys HMG2-mediated DNA binding and bending functions.  Granzyme A cleavage and functional disruption of key nuclear substrates, including HMG2, SET, APE, lamins, and histones, are likely to cripple the cellular repair response to promote cell death in this novel caspase-independent death pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxOx2JgQyVQrVg90H21EOLACvtfcHk0ljt-OOAqGKLew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XislWnt78%253D&md5=cd5f8b8e666b5b12f45bdce1887908b3</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1128%2FMCB.22.8.2810-2820.2002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.22.8.2810-2820.2002%26sid%3Dliteratum%253Aachs%26aulast%3DFan%26aufirst%3DZ.%26aulast%3DBeresford%26aufirst%3DP.%2BJ.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DLieberman%26aufirst%3DJ.%26atitle%3DHMG2%2520Interacts%2520with%2520the%2520Nucleosome%2520Assembly%2520Protein%2520SET%2520and%2520is%2520a%2520Target%2520of%2520the%2520Cytotoxic%2520T-lymphocyte%2520Protease%2520Granzyme%2520A%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2002%26volume%3D22%26spage%3D2810%26epage%3D2820%26doi%3D10.1128%2FMCB.22.8.2810-2820.2002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasternack, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresford, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenberg, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lieberman, J.</span></span> <span> </span><span class="NLM_article-title">Induction of Rapid Histone Degradation by the Cytotoxic T Lymphocyte Protease Granzyme A</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">3683</span>– <span class="NLM_lpage">3690</span>, <span class="refDoi"> DOI: 10.1074/jbc.M005390200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1074%2Fjbc.M005390200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=11060286" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhtFWitrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2001&pages=3683-3690&author=D.+Zhangauthor=M.+S.+Pasternackauthor=P.+J.+Beresfordauthor=L.+Wagnerauthor=A.+H.+Greenbergauthor=J.+Lieberman&title=Induction+of+Rapid+Histone+Degradation+by+the+Cytotoxic+T+Lymphocyte+Protease+Granzyme+A&doi=10.1074%2Fjbc.M005390200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Induction of rapid histone degradation by the cytotoxic T lymphocyte protease granzyme A</span></div><div class="casAuthors">Zhang, Dong; Pasternack, Mark S.; Beresford, Paul J.; Wagner, Ludwig; Greenberg, Arnold H.; Lieberman, Judy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">3683-3690</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The cytotoxic T lymphocyte protease granzyme A induces caspase-independent cell death in which DNA single-strand nicking is obsd. instead of oligonucleosomal fragmentation.  Granzyme A is a specific tryptase that concs. in the nucleus of targeted cells and synergistically enhances DNA fragmentation induced by the caspase activator granzyme B.  Here we show that granzyme A treatment of isolated nuclei enhances DNA accessibility to exogenous endonucleases.  In vitro and after cell loading with perforin, GrnA completely degrades histone H1 and cleaves core histones into ∼16-kDa fragments.  Histone digestion provides a mechanism for unfolding compacted chromatin and facilitating endogenous DNase access to DNA during T cell and natural killer cell granule-mediated apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKVCFhmAR0cbVg90H21EOLACvtfcHk0li5yUx5ozPDUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhtFWitrw%253D&md5=ca2b82889f1ab1ed9b85ac48e6e15eaf</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M005390200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M005390200%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DPasternack%26aufirst%3DM.%2BS.%26aulast%3DBeresford%26aufirst%3DP.%2BJ.%26aulast%3DWagner%26aufirst%3DL.%26aulast%3DGreenberg%26aufirst%3DA.%2BH.%26aulast%3DLieberman%26aufirst%3DJ.%26atitle%3DInduction%2520of%2520Rapid%2520Histone%2520Degradation%2520by%2520the%2520Cytotoxic%2520T%2520Lymphocyte%2520Protease%2520Granzyme%2520A%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2001%26volume%3D276%26spage%3D3683%26epage%3D3690%26doi%3D10.1074%2Fjbc.M005390200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresford, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenberg, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lieberman, J.</span></span> <span> </span><span class="NLM_article-title">Granzymes A and B Directly Cleave Lamins and Disrupt the Nuclear Lamina During Granule-Mediated Cytolysis</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">5746</span>– <span class="NLM_lpage">5751</span>, <span class="refDoi"> DOI: 10.1073/pnas.101329598</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1073%2Fpnas.101329598" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=11331782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjs1Wnsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=5746-5751&author=D.+Zhangauthor=P.+J.+Beresfordauthor=A.+H.+Greenbergauthor=J.+Lieberman&title=Granzymes+A+and+B+Directly+Cleave+Lamins+and+Disrupt+the+Nuclear+Lamina+During+Granule-Mediated+Cytolysis&doi=10.1073%2Fpnas.101329598"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Granzymes A and B directly cleave lamins and disrupt the nuclear lamina during granule-mediated cytolysis</span></div><div class="casAuthors">Zhang, Dong; Beresford, Paul J.; Greenberg, Arnold H.; Lieberman, Judy</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5746-5751</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Cytotoxic T lymphocytes (CTL) induce apoptosis by engaging death receptors or by exocytosis of cytolytic granules contg. granzyme (Gzm) proteases and perforin.  The lamins, which maintain the structural integrity of the nuclear envelope, are cleaved by caspases during caspase-mediated apoptosis.  Although death receptor engagement and GzmB activate caspases, CTL also induce apoptosis during caspase blockade.  Both GzmA and GzmB directly and efficiently cleave lamin B in vitro, in situ in isolated nuclei and in cells loaded with perforin and Gzms, even in the presence of caspase inhibitors.  Lamin-B is cleaved by GzmA at concns. of 3 nM, but GzmB is 50 times less active.  GzmA cuts lamin B at R392; GzmB cuts at the caspase VEVD231 site.  Characteristic lamin B fragments generated by Gzm proteolysis also are obsd. during CTL lysis, even in the presence of caspase inhibitors or in cells overexpressing bcl-2.  Lamins A/C are direct substrates of GzmA, but not GzmB.  GzmA and GzmB therefore directly target crit. caspase substrates in caspase-resistant cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGyCLT712Qj7Vg90H21EOLACvtfcHk0li5yUx5ozPDUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjs1Wnsbo%253D&md5=21fdb3a99cbfb762955a2f30583d4ccd</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1073%2Fpnas.101329598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.101329598%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DBeresford%26aufirst%3DP.%2BJ.%26aulast%3DGreenberg%26aufirst%3DA.%2BH.%26aulast%3DLieberman%26aufirst%3DJ.%26atitle%3DGranzymes%2520A%2520and%2520B%2520Directly%2520Cleave%2520Lamins%2520and%2520Disrupt%2520the%2520Nuclear%2520Lamina%2520During%2520Granule-Mediated%2520Cytolysis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2001%26volume%3D98%26spage%3D5746%26epage%3D5751%26doi%3D10.1073%2Fpnas.101329598" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresford, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lieberman, J.</span></span> <span> </span><span class="NLM_article-title">Tumor Suppressor NM23-H1 is a Granzyme A-activated DNase During CTL-mediated Apoptosis, and the Nucleosome Assembly Protein SET is its Inhibitor</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">659</span>– <span class="NLM_lpage">672</span>, <span class="refDoi"> DOI: 10.1016/S0092-8674(03)00150-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1016%2FS0092-8674%2803%2900150-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=12628186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BD3sXitFykurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2003&pages=659-672&author=Z.+Fanauthor=P.+J.+Beresfordauthor=D.+Y.+Ohauthor=D.+Zhangauthor=J.+Lieberman&title=Tumor+Suppressor+NM23-H1+is+a+Granzyme+A-activated+DNase+During+CTL-mediated+Apoptosis%2C+and+the+Nucleosome+Assembly+Protein+SET+is+its+Inhibitor&doi=10.1016%2FS0092-8674%2803%2900150-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Tumor suppressor NM23-H1 is a granzyme A-activated dnase during CTL-mediated apoptosis, and the nucleosome assembly protein SET is its inhibitor</span></div><div class="casAuthors">Fan, Zusen; Beresford, Paul J.; Oh, David Y.; Zhang, Dong; Lieberman, Judy</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">659-672</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Granzyme A (GzmA) induces a caspase-independent cell death pathway characterized by single-stranded DNA nicks and other features of apoptosis.  A GzmA-activated DNase (GAAD) is in an ER assocd. complex contg. pp32 and the GzmA substrates SET, HMG-2, and Ape1.  The authors show that GAAD is NM23-H1, a nucleoside diphosphate kinase implicated in suppression of tumor metastasis, and its specific inhibitor (IGAAD) is SET.  NM23-H1 binds to SET and is released from inhibition by GzmA cleavage of SET.  After GzmA loading or CTL attack, SET and NM23-H1 translocate to the nucleus and SET is degraded, allowing NM23-H1 to nick chromosomal DNA.  GzmA-treated cells with silenced NM23-H1 expression are resistant to GzmA-mediated DNA damage and cytolysis, while cells over-expressing NM23-H1 are more sensitive.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovlpoLT-sBjbVg90H21EOLACvtfcHk0li5yUx5ozPDUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXitFykurc%253D&md5=f3bbb5b994d3a27eb9d2864d33d55f55</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2803%2900150-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252803%252900150-8%26sid%3Dliteratum%253Aachs%26aulast%3DFan%26aufirst%3DZ.%26aulast%3DBeresford%26aufirst%3DP.%2BJ.%26aulast%3DOh%26aufirst%3DD.%2BY.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DLieberman%26aufirst%3DJ.%26atitle%3DTumor%2520Suppressor%2520NM23-H1%2520is%2520a%2520Granzyme%2520A-activated%2520DNase%2520During%2520CTL-mediated%2520Apoptosis%252C%2520and%2520the%2520Nucleosome%2520Assembly%2520Protein%2520SET%2520is%2520its%2520Inhibitor%26jtitle%3DCell%26date%3D2003%26volume%3D112%26spage%3D659%26epage%3D672%26doi%3D10.1016%2FS0092-8674%2803%2900150-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chowdhury, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresford, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demple, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrino, F. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lieberman, J.</span></span> <span> </span><span class="NLM_article-title">The Exonuclease TREX1 is in the SET Complex and Acts in Concert with NM23-H1 to Degrade DNA during Granzyme A-mediated Cell Death</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">142</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2006.06.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1016%2Fj.molcel.2006.06.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=16818237" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BD28XnsVyjs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2006&pages=133-142&author=D.+Chowdhuryauthor=P.+J.+Beresfordauthor=P.+Zhuauthor=D.+Zhangauthor=J.+S.+Sungauthor=B.+Dempleauthor=F.+W.+Perrinoauthor=J.+Lieberman&title=The+Exonuclease+TREX1+is+in+the+SET+Complex+and+Acts+in+Concert+with+NM23-H1+to+Degrade+DNA+during+Granzyme+A-mediated+Cell+Death&doi=10.1016%2Fj.molcel.2006.06.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The exonuclease TREX1 is in the SET complex and acts in concert with NM23-H1 to degrade DNA during granzyme A-mediated cell death</span></div><div class="casAuthors">Chowdhury, Dipanjan; Beresford, Paul J.; Zhu, Pengcheng; Zhang, Dong; Sung, Jung-Suk; Demple, Bruce; Perrino, Fred W.; Lieberman, Judy</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">133-142</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Granzyme A (GzmA) activates a caspase-independent cell death pathway with morphol. features of apoptosis.  Single-stranded DNA damage is initiated when the endonuclease NM23-H1 becomes activated to nick DNA after granzyme A cleaves its inhibitor, SET.  SET and NM23-H1 reside in an endoplasmic reticulum-assocd. complex (the SET complex) that translocates to the nucleus in response to superoxide generation by granzyme A.  We now find the 3'-to-5' exonuclease TREX1, but not its close homolog TREX2, in the SET complex.  TREX1 binds to SET and colocalizes and translocates with the SET complex.  NM23-H1 and TREX1 work in concert to degrade DNA.  Silencing NM23-H1 or TREX1 inhibits DNA damage and death of cells treated with perforin (PFN) and granzyme A, but not of cells treated with perforin and granzyme B (GzmB).  After granzyme A activates NM23-H1 to make single-stranded nicks, TREX1 removes nucleotides from the nicked 3' end to reduce the possibility of repair by rejoining the nicked ends.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrse4Cx7pIswrVg90H21EOLACvtfcHk0ljws-uBnkuoIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XnsVyjs7s%253D&md5=06fe4768669c11d53537668777960eec</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2006.06.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2006.06.005%26sid%3Dliteratum%253Aachs%26aulast%3DChowdhury%26aufirst%3DD.%26aulast%3DBeresford%26aufirst%3DP.%2BJ.%26aulast%3DZhu%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DSung%26aufirst%3DJ.%2BS.%26aulast%3DDemple%26aufirst%3DB.%26aulast%3DPerrino%26aufirst%3DF.%2BW.%26aulast%3DLieberman%26aufirst%3DJ.%26atitle%3DThe%2520Exonuclease%2520TREX1%2520is%2520in%2520the%2520SET%2520Complex%2520and%2520Acts%2520in%2520Concert%2520with%2520NM23-H1%2520to%2520Degrade%2520DNA%2520during%2520Granzyme%2520A-mediated%2520Cell%2520Death%26jtitle%3DMol.%2520Cell%26date%3D2006%26volume%3D23%26spage%3D133%26epage%3D142%26doi%3D10.1016%2Fj.molcel.2006.06.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bell, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goetz, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahrus, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletterick, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, C. S.</span></span> <span> </span><span class="NLM_article-title">The Oligomeric Structure of Human Granzyme A is a Determinant of its Extended Substrate Specificity</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">527</span>– <span class="NLM_lpage">534</span>, <span class="refDoi"> DOI: 10.1038/nsb944</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1038%2Fnsb944" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2003&pages=527-534&author=J.+K.+Bellauthor=D.+H.+Goetzauthor=S.+Mahrusauthor=J.+L.+Harrisauthor=R.+J.+Fletterickauthor=C.+S.+Craik&title=The+Oligomeric+Structure+of+Human+Granzyme+A+is+a+Determinant+of+its+Extended+Substrate+Specificity&doi=10.1038%2Fnsb944"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnsb944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsb944%26sid%3Dliteratum%253Aachs%26aulast%3DBell%26aufirst%3DJ.%2BK.%26aulast%3DGoetz%26aufirst%3DD.%2BH.%26aulast%3DMahrus%26aufirst%3DS.%26aulast%3DHarris%26aufirst%3DJ.%2BL.%26aulast%3DFletterick%26aufirst%3DR.%2BJ.%26aulast%3DCraik%26aufirst%3DC.%2BS.%26atitle%3DThe%2520Oligomeric%2520Structure%2520of%2520Human%2520Granzyme%2520A%2520is%2520a%2520Determinant%2520of%2520its%2520Extended%2520Substrate%2520Specificity%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2003%26volume%3D10%26spage%3D527%26epage%3D534%26doi%3D10.1038%2Fnsb944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kasperkiewicz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poreba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snipas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirchhofer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvesen, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span> <span> </span><span class="NLM_article-title">Design of a Selective Substrate and Activity Based Probe for Human Neutrophil Serine Protease 4</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">e0132818</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0132818</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1371%2Fjournal.pone.0132818" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=26172376" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVSrtLnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0132818&author=P.+Kasperkiewiczauthor=M.+Porebaauthor=S.+J.+Snipasauthor=S.+J.+Linauthor=D.+Kirchhoferauthor=G.+S.+Salvesenauthor=M.+Drag&title=Design+of+a+Selective+Substrate+and+Activity+Based+Probe+for+Human+Neutrophil+Serine+Protease+4&doi=10.1371%2Fjournal.pone.0132818"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Design of a selective substrate and activity based probe for human neutrophil serine protease 4</span></div><div class="casAuthors">Kasperkiewicz, Paulina; Poreba, Marcin; Snipas, Scott J.; Jack, Lin S.; Kirchhofer, Daniel; Salvesen, Guy S.; Drag, Marcin</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e0132818/1-e0132818/12</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Human neutrophil serine protease 4 (NSP4), also known as PRSS57, is a recently discovered fourth member of the neutrophil serine proteases family.  Although its biol. function is not precisely defined, it is suggested to regulate neutrophil response and innate immune reactions.  To create optimal substrates and visualization probes for NSP4 that distinguish it from other NSPs we have employed a Hybrid Combinatorial Substrate Library approach that utilizes natural and unnatural amino acids to explore protease subsite preferences.  Library results were validated by synthesizing individual substrates, leading to the identification of an optimal substrate peptide.  This substrate was converted to a covalent di-Ph phosphonate probe with an embedded biotin tag.  This probe demonstrated high inhibitory activity and stringent specificity and may be suitable for visualizing NSP4 in the background of other NSPs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7Dosrmw75eLVg90H21EOLACvtfcHk0ljws-uBnkuoIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVSrtLnE&md5=b2323901d336008609b238a1bc344eca</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0132818&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0132818%26sid%3Dliteratum%253Aachs%26aulast%3DKasperkiewicz%26aufirst%3DP.%26aulast%3DPoreba%26aufirst%3DM.%26aulast%3DSnipas%26aufirst%3DS.%2BJ.%26aulast%3DLin%26aufirst%3DS.%2BJ.%26aulast%3DKirchhofer%26aufirst%3DD.%26aulast%3DSalvesen%26aufirst%3DG.%2BS.%26aulast%3DDrag%26aufirst%3DM.%26atitle%3DDesign%2520of%2520a%2520Selective%2520Substrate%2520and%2520Activity%2520Based%2520Probe%2520for%2520Human%2520Neutrophil%2520Serine%2520Protease%25204%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26spage%3De0132818%26doi%3D10.1371%2Fjournal.pone.0132818" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Poe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biddison, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blake, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodkey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigal, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zweerink, H. J.</span></span> <span> </span><span class="NLM_article-title">Human Cytotoxic Lymphocyte Tryptase. Its Purification from Granules and the Characterization of Inhibitor and Substrate Specificity</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>263</i></span>,  <span class="NLM_fpage">13215</span>– <span class="NLM_lpage">13222</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=3047119" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADyaL1MXjtlOq" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=263&publication_year=1988&pages=13215-13222&author=M.+Poeauthor=C.+D.+Bennettauthor=W.+E.+Biddisonauthor=J.+T.+Blakeauthor=G.+P.+Nortonauthor=J.+A.+Rodkeyauthor=N.+H.+Sigalauthor=R.+V.+Turnerauthor=J.+K.+Wuauthor=H.+J.+Zweerink&title=Human+Cytotoxic+Lymphocyte+Tryptase.+Its+Purification+from+Granules+and+the+Characterization+of+Inhibitor+and+Substrate+Specificity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Human cytotoxic lymphocyte tryptase.  Its purification from granules and the characterization of inhibitor and substrate specificity</span></div><div class="casAuthors">Poe, Martin; Bennett, Carl D.; Biddison, William E.; Blake, Joseph T.; Norton, Gerard P.; Rodkey, John A.; Sigal, Nolan H.; Turner, Richard V.; Wu, Joseph K.; Zweerink, Hans J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">263</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">13215-22</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">A trypsin-like enzyme (tryptase) (I) was purified to homogeneity from the granules of a human cytolytic lymphocyte (CTL) line, Q31, by a 3-step procedure.  By including 0.3% Triton X-100 and 1 mg/mL heparin in purifn. buffers, near total yields of I activity were obtained during the purifn.  I migrated in SDS-PAGE at a position corresponding to 28 kilodaltons with and to 45 kilodaltons without 2-mercaptoethanol.  It had an N-terminal sequence identical to a previously reported for human CTL I at 20 of 22 positions identified.  I hydrolyzed Nα-carbobenzoxy-L-lysine thiobenzyl ester (BLT), and this BLT esterase activity was most efficient at slightly alk. pH and was relatively more active near neutral pH than mouse CTL I.  Human α1-protease inhibitor, human antithrombin III, phenylmethanesulfonyl fluoride, and p-aminobenzamidine inhibited Q31 I.  The inhibition by human antithrombin III was rapid enough to be of physiol. significance.  A survey of oligopeptide p-nitroanilides found that the best substrate for human Q31 I was H-D-(ε-carbobenzyloxy)Lys-L-Pro-L-Arg-p-nitroanilide.  Q31 I appeared to have broad specificity for amino acid residues at P2 and P3, i.e., at 2 and 3 residues N-terminal to the scissile bond.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD3Ki-3s0xV7Vg90H21EOLACvtfcHk0ljws-uBnkuoIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXjtlOq&md5=3bbbbcac76ef4a5416a3cca6a58ee304</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPoe%26aufirst%3DM.%26aulast%3DBennett%26aufirst%3DC.%2BD.%26aulast%3DBiddison%26aufirst%3DW.%2BE.%26aulast%3DBlake%26aufirst%3DJ.%2BT.%26aulast%3DNorton%26aufirst%3DG.%2BP.%26aulast%3DRodkey%26aufirst%3DJ.%2BA.%26aulast%3DSigal%26aufirst%3DN.%2BH.%26aulast%3DTurner%26aufirst%3DR.%2BV.%26aulast%3DWu%26aufirst%3DJ.%2BK.%26aulast%3DZweerink%26aufirst%3DH.%2BJ.%26atitle%3DHuman%2520Cytotoxic%2520Lymphocyte%2520Tryptase.%2520Its%2520Purification%2520from%2520Granules%2520and%2520the%2520Characterization%2520of%2520Inhibitor%2520and%2520Substrate%2520Specificity%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1988%26volume%3D263%26spage%3D13215%26epage%3D13222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ng, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pike, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, S. E.</span></span> <span> </span><span class="NLM_article-title">Subsite Cooperativity in Protease Specificity</span>. <i>Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>390</i></span>,  <span class="NLM_fpage">401</span>– <span class="NLM_lpage">407</span>, <span class="refDoi"> DOI: 10.1515/BC.2009.065</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1515%2FBC.2009.065" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=19361286" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmvVWqtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=390&publication_year=2009&pages=401-407&author=N.+M.+Ngauthor=R.+N.+Pikeauthor=S.+E.+Boyd&title=Subsite+Cooperativity+in+Protease+Specificity&doi=10.1515%2FBC.2009.065"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Subsite cooperativity in protease specificity</span></div><div class="casAuthors">Ng, Natasha M.; Pike, Robert N.; Boyd, Sarah E.</div><div class="citationInfo"><span class="NLM_cas:title">Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">390</span>
        (<span class="NLM_cas:issue">5/6</span>),
    <span class="NLM_cas:pages">401-407</span>CODEN:
                <span class="NLM_cas:coden">BICHF3</span>;
        ISSN:<span class="NLM_cas:issn">1431-6730</span>.
    
            (<span class="NLM_cas:orgname">Walter de Gruyter GmbH & Co. KG</span>)
        </div><div class="casAbstract">A review.  Proteases play vital roles in a range of biol. processes, such as cell cycle, cell growth and differentiation, apoptosis, hemostasis and signaling.  Fundamental to our knowledge of protease action is an understanding of how the active site operates; this has been examd. through extensive studies of the substrate specificity of the enzymes.  Kinetic and structural analyses have shown that the binding of a particular substrate residue at a protease subsite can have either a pos. or neg. influence on the binding of particular residues at other subsites.  This phenomenon has been termed subsite cooperativity and has been obsd. in a wide range of proteases, often between non-adjacent subsites.  This review aims to highlight studies where subsite cooperativity has been obsd., exptl. techniques used in the past and potential methods that can be employed to comprehensively examine this phenomenon.  Further understanding of how the protease active site recognizes and chooses its substrates for cleavage will have a significant impact on the development of pharmaceuticals that target these enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK8MrK5j6zybVg90H21EOLACvtfcHk0ljLGGmv_PfS3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmvVWqtro%253D&md5=68b6d056b3e5da0255703f2360d3d6d5</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1515%2FBC.2009.065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252FBC.2009.065%26sid%3Dliteratum%253Aachs%26aulast%3DNg%26aufirst%3DN.%2BM.%26aulast%3DPike%26aufirst%3DR.%2BN.%26aulast%3DBoyd%26aufirst%3DS.%2BE.%26atitle%3DSubsite%2520Cooperativity%2520in%2520Protease%2520Specificity%26jtitle%3DBiol.%2520Chem.%26date%3D2009%26volume%3D390%26spage%3D401%26epage%3D407%26doi%3D10.1515%2FBC.2009.065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kasperkiewicz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altman, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Angelo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvesen, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span> <span> </span><span class="NLM_article-title">Toolbox of Fluorescent Probes for Parallel Imaging Reveals Uneven Location of Serine Proteases in Neutrophils</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">10115</span>– <span class="NLM_lpage">10125</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b04394</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b04394" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtV2ru7zM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=10115-10125&author=P.+Kasperkiewiczauthor=Y.+Altmanauthor=M.+D%E2%80%99Angeloauthor=G.+S.+Salvesenauthor=M.+Drag&title=Toolbox+of+Fluorescent+Probes+for+Parallel+Imaging+Reveals+Uneven+Location+of+Serine+Proteases+in+Neutrophils&doi=10.1021%2Fjacs.7b04394"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Toolbox of Fluorescent Probes for Parallel Imaging Reveals Uneven Location of Serine Proteases in Neutrophils</span></div><div class="casAuthors">Kasperkiewicz, Paulina; Altman, Yoav; D'Angelo, Maximiliano; Salvesen, Guy S.; Drag, Marcin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">10115-10125</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Neutrophils, the front line defenders against infection, express four serine proteases (NSPs) that play roles in the control of cell-signaling pathways and defense against pathogens and whose imbalance leads to pathol. conditions.  Dissecting the roles of individual NSPs in humans is problematic because neutrophils are end-stage cells with a short half-life and minimal ongoing protein synthesis.  To gain insight into the regulation of NSP activity we have generated a small-mol. chem. toolbox consisting of activity-based probes with different fluorophore-detecting groups with minimal wavelength overlap and highly selective natural and unnatural amino acid recognition sequences.  The key feature of these activity-based probes is the ability to use them for simultaneous observation and detection of all four individual NSPs by fluorescence microscopy, a feature never achieved in previous studies.  Using these probes we demonstrate uneven distribution of NSPs in neutrophil azurophil granules, such that they seem to be mutually excluded from each other, suggesting the existence of unknown granule-targeting mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCQyFRKT-WtrVg90H21EOLACvtfcHk0ljLGGmv_PfS3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtV2ru7zM&md5=72b3e9e59a26a034fa66aa9cbd714b61</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b04394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b04394%26sid%3Dliteratum%253Aachs%26aulast%3DKasperkiewicz%26aufirst%3DP.%26aulast%3DAltman%26aufirst%3DY.%26aulast%3DD%25E2%2580%2599Angelo%26aufirst%3DM.%26aulast%3DSalvesen%26aufirst%3DG.%2BS.%26aulast%3DDrag%26aufirst%3DM.%26atitle%3DToolbox%2520of%2520Fluorescent%2520Probes%2520for%2520Parallel%2520Imaging%2520Reveals%2520Uneven%2520Location%2520of%2520Serine%2520Proteases%2520in%2520Neutrophils%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D10115%26epage%3D10125%26doi%3D10.1021%2Fjacs.7b04394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hink-Schauer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estebanez-Perpina, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurschus, F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bode, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenne, D. E.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of the Apoptosis-Inducing Human Granzyme A Dimer</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">535</span>– <span class="NLM_lpage">540</span>, <span class="refDoi"> DOI: 10.1038/nsb945</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1038%2Fnsb945" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2003&pages=535-540&author=C.+Hink-Schauerauthor=E.+Estebanez-Perpinaauthor=F.+C.+Kurschusauthor=W.+Bodeauthor=D.+E.+Jenne&title=Crystal+Structure+of+the+Apoptosis-Inducing+Human+Granzyme+A+Dimer&doi=10.1038%2Fnsb945"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnsb945&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsb945%26sid%3Dliteratum%253Aachs%26aulast%3DHink-Schauer%26aufirst%3DC.%26aulast%3DEstebanez-Perpina%26aufirst%3DE.%26aulast%3DKurschus%26aufirst%3DF.%2BC.%26aulast%3DBode%26aufirst%3DW.%26aulast%3DJenne%26aufirst%3DD.%2BE.%26atitle%3DCrystal%2520Structure%2520of%2520the%2520Apoptosis-Inducing%2520Human%2520Granzyme%2520A%2520Dimer%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2003%26volume%3D10%26spage%3D535%26epage%3D540%26doi%3D10.1038%2Fnsb945" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krahenbuhl, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rey, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenne, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanzavecchia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groscurth, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tschopp, J.</span></span> <span> </span><span class="NLM_article-title">Characterization of Granzymes A and B Isolated from Granules of Cloned Human Cytotoxic T Lymphocytes</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">3471</span>– <span class="NLM_lpage">3477</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=3263427" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A280%3ADyaL1M%252FjsFClug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=1988&pages=3471-3477&author=O.+Krahenbuhlauthor=C.+Reyauthor=D.+Jenneauthor=A.+Lanzavecchiaauthor=P.+Groscurthauthor=S.+Carrelauthor=J.+Tschopp&title=Characterization+of+Granzymes+A+and+B+Isolated+from+Granules+of+Cloned+Human+Cytotoxic+T+Lymphocytes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of granzymes A and B isolated from granules of cloned human cytotoxic T lymphocytes</span></div><div class="casAuthors">Krahenbuhl O; Rey C; Jenne D; Lanzavecchia A; Groscurth P; Carrel S; Tschopp J</div><div class="citationInfo"><span class="NLM_cas:title">Journal of immunology (Baltimore, Md. : 1950)</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3471-7</span>
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    </div><div class="casAbstract">A human CD8+ CTL clone with cytolytic potential was shown to express two serine proteases, a 50-kDa homodimer and a 27-kDa monomer, which were purified from cytoplasmic granules.  N-terminal sequencing of the purified proteins revealed that the 50-kDa homodimer is the gene product of the human Hanukah factor cDNA clone and that it represents the human homologue to granzyme A.  Similarly, the 27-kDa protein was shown to be the serine esterase encoded by the human lymphocyte protease cDNA clone and corresponds to granzyme B.  There was no evidence for the presence of other granzymes, in particular for the human homologues to murine granzymes C, D, E, and F.  The substrate best cleaved by granzyme A was Gly-Pro-Arg-amido-4-methyl-coumarin after the Arg residue and the pH optimum was 8 to 8.5.  Upon triggering of the TCR-CD3 complex with an anti-CD3 mAb, granzyme A was released into the culture medium.  Furthermore, a granule-associated hemolytic activity was detected after salt extraction and partial purification of granule proteins.  This suggests that hemolytically active human perforin can be obtained from inactive granules.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSndU-TK3UV58jl5L8hwNX0fW6udTcc2ebyAAvN0k5vRbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaL1M%252FjsFClug%253D%253D&md5=fc03478f25e9e0714a0282cc822f0f11</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKrahenbuhl%26aufirst%3DO.%26aulast%3DRey%26aufirst%3DC.%26aulast%3DJenne%26aufirst%3DD.%26aulast%3DLanzavecchia%26aufirst%3DA.%26aulast%3DGroscurth%26aufirst%3DP.%26aulast%3DCarrel%26aufirst%3DS.%26aulast%3DTschopp%26aufirst%3DJ.%26atitle%3DCharacterization%2520of%2520Granzymes%2520A%2520and%2520B%2520Isolated%2520from%2520Granules%2520of%2520Cloned%2520Human%2520Cytotoxic%2520T%2520Lymphocytes%26jtitle%3DJ.%2520Immunol.%26date%3D1988%26volume%3D141%26spage%3D3471%26epage%3D3477" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lieberman, J.</span></span> <span> </span><span class="NLM_article-title">Granzyme A Activates Another Way to Die</span>. <i>Immunol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>235</i></span>,  <span class="NLM_fpage">93</span>– <span class="NLM_lpage">104</span>, <span class="refDoi"> DOI: 10.1111/j.0105-2896.2010.00902.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1111%2Fj.0105-2896.2010.00902.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=20536557" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnt1Kjsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=235&publication_year=2010&pages=93-104&author=J.+Lieberman&title=Granzyme+A+Activates+Another+Way+to+Die&doi=10.1111%2Fj.0105-2896.2010.00902.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Granzyme A activates another way to die</span></div><div class="casAuthors">Lieberman, Judy</div><div class="citationInfo"><span class="NLM_cas:title">Immunological Reviews</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">235</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">93-104</span>CODEN:
                <span class="NLM_cas:coden">IMRED2</span>;
        ISSN:<span class="NLM_cas:issn">1600-065X</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Granzyme A (GzmA) is the most abundant serine protease in killer cell cytotoxic granules.  GzmA activates a novel programed cell death pathway that begins in the mitochondrion, where cleavage of NDUFS3 in electron transport complex 1 disrupts mitochondrial metab. and generates reactive oxygen species (ROS).  ROS drives the endoplasmic reticulum-assocd. SET complex into the nucleus, where it activates single-stranded DNA damage.  GzmA also targets other important nuclear proteins for degrdn., including histones, the lamins that maintain the nuclear envelope, and several key DNA damage repair proteins (Ku70, PARP-1).  Cells that are resistant to the caspases or GzmB by overexpressing bcl-2 family anti-apoptotic proteins or caspase or GzmB protease inhibitors are sensitive to GzmA.  By activating multiple cell death pathways, killer cells provide better protection against a variety of intracellular pathogens and tumors.  GzmA also has proinflammatory activity; it activates pro-interleukin-1β and may also have other proinflammatory effects that remain to be elucidated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrE7K84--EYWbVg90H21EOLACvtfcHk0li9aB1LJ_TgLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnt1Kjsbc%253D&md5=75a9b3e75d7d20bc50f9b85dd31aec38</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1111%2Fj.0105-2896.2010.00902.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.0105-2896.2010.00902.x%26sid%3Dliteratum%253Aachs%26aulast%3DLieberman%26aufirst%3DJ.%26atitle%3DGranzyme%2520A%2520Activates%2520Another%2520Way%2520to%2520Die%26jtitle%3DImmunol.%2520Rev.%26date%3D2010%26volume%3D235%26spage%3D93%26epage%3D104%26doi%3D10.1111%2Fj.0105-2896.2010.00902.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kasperkiewicz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poreba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snipas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winterbourn, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvesen, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span> <span> </span><span class="NLM_article-title">Design of Ultrasensitive Probes for Human Neutrophil Elastase Through Hybrid Combinatorial Substrate Library Profiling</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">2518</span>– <span class="NLM_lpage">2523</span>, <span class="refDoi"> DOI: 10.1073/pnas.1318548111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1073%2Fpnas.1318548111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=24550277" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BC2cXivFaht7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=2518-2523&author=P.+Kasperkiewiczauthor=M.+Porebaauthor=S.+J.+Snipasauthor=H.+Parkerauthor=C.+C.+Winterbournauthor=G.+S.+Salvesenauthor=M.+Drag&title=Design+of+Ultrasensitive+Probes+for+Human+Neutrophil+Elastase+Through+Hybrid+Combinatorial+Substrate+Library+Profiling&doi=10.1073%2Fpnas.1318548111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Design of ultrasensitive probes for human neutrophil elastase through hybrid combinatorial substrate library profiling</span></div><div class="casAuthors">Kasperkiewicz, Paulina; Poreba, Marcin; Snipas, Scott J.; Parker, Heather; Winterbourn, Christine C.; Salvesen, Guy S.; Drag, Marcin</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2518-2523</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The exploration of protease substrate specificity is generally restricted to naturally occurring amino acids, limiting the degree of conformational space that can be surveyed.  We substantially enhanced this by incorporating 102 unnatural amino acids to explore the S1-S4 pockets of human neutrophil elastase (NE).  This approach provides hybrid natural and unnatural amino acid sequences, and thus we termed it the Hybrid Combinatorial Substrate Library (HyCoSuL).  Library results were validated by the synthesis of individual tetrapeptide substrates, with the optimal substrate demonstrating more than three orders of magnitude higher catalytic efficiency than commonly used substrates of elastase.  This optimal substrate was converted to an activity-based probe that demonstrated high selectivity and revealed the specific presence of active elastase during the process of neutrophil extracellular trap formation.  We propose that this approach can be successfully used for any type of endopeptidase to deliver high activity and selectivity in substrates and probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMwdujTniBZbVg90H21EOLACvtfcHk0li9aB1LJ_TgLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXivFaht7o%253D&md5=048b0dd58455cd95a3def76ee8942337</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1318548111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1318548111%26sid%3Dliteratum%253Aachs%26aulast%3DKasperkiewicz%26aufirst%3DP.%26aulast%3DPoreba%26aufirst%3DM.%26aulast%3DSnipas%26aufirst%3DS.%2BJ.%26aulast%3DParker%26aufirst%3DH.%26aulast%3DWinterbourn%26aufirst%3DC.%2BC.%26aulast%3DSalvesen%26aufirst%3DG.%2BS.%26aulast%3DDrag%26aufirst%3DM.%26atitle%3DDesign%2520of%2520Ultrasensitive%2520Probes%2520for%2520Human%2520Neutrophil%2520Elastase%2520Through%2520Hybrid%2520Combinatorial%2520Substrate%2520Library%2520Profiling%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D2518%26epage%3D2523%26doi%3D10.1073%2Fpnas.1318548111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Poreba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvesen, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span> <span> </span><span class="NLM_article-title">Synthesis of a HyCoSuL Peptide Substrate Library to Dissect Protease Substrate Specificity</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2189</span>– <span class="NLM_lpage">2214</span>, <span class="refDoi"> DOI: 10.1038/nprot.2017.091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1038%2Fnprot.2017.091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=28933778" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFehtrjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=2189-2214&author=M.+Porebaauthor=G.+S.+Salvesenauthor=M.+Drag&title=Synthesis+of+a+HyCoSuL+Peptide+Substrate+Library+to+Dissect+Protease+Substrate+Specificity&doi=10.1038%2Fnprot.2017.091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of a HyCoSuL peptide substrate library to dissect protease substrate specificity</span></div><div class="casAuthors">Poreba, Marcin; Salvesen, Guy S.; Drag, Marcin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2189-2214</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Many biol. and chem. based approaches have been developed to design highly active and selective protease substrates and probes.  It is, however, difficult to find substrate sequences that are truly selective for any given protease, as different proteases can demonstrate a great deal of overlap in substrate specificities.  In some cases, better enzyme selectivity can be achieved using peptide libraries contg. unnatural amino acids such as the hybrid combinatorial substrate library (HyCoSuL), which uses both natural and unnatural amino acids.  HyCoSuL is a combinatorial library of tetrapeptides contg. amino acid mixts. at the P4-P2 positions, a fixed amino acid at the P1 position, and an ACC (7-amino-4-carbamoylmethylcoumarin) fluorescent tag occupying the P1' position.  Once the peptide is recognized and cleaved by a protease, the ACC is released and produces a readable fluorescence signal.  Here, we describe the synthesis and screening of HyCoSuL for human caspases and legumain.  We also discuss possible modifications and adaptations of this approach that make it a useful tool for developing highly active and selective reagents for a wide variety of proteolytic enzymes.  The protocol can be divided into three major parts: (i) solid-phase synthesis of the fluorescence-labeled HyCoSuL, (ii) screening of protease P4-P2 preferences, and (iii) synthesis of the optimized activity probes equipped with an AOMK (acyloxymethyl ketone) reactive group and a biotin label for easy detection.  Beginning with the library design, the entire protocol can be completed in 4-8 wk (HyCoSuL synthesis: 3-5 wk; HyCoSuL screening per enzyme: 4-8 d; and activity-based probe synthesis: 1-2 wk).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYBvdwWr2XhLVg90H21EOLACvtfcHk0li9aB1LJ_TgLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFehtrjF&md5=5ece1429e76feedd3eb864f07da7f8b7</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2017.091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2017.091%26sid%3Dliteratum%253Aachs%26aulast%3DPoreba%26aufirst%3DM.%26aulast%3DSalvesen%26aufirst%3DG.%2BS.%26aulast%3DDrag%26aufirst%3DM.%26atitle%3DSynthesis%2520of%2520a%2520HyCoSuL%2520Peptide%2520Substrate%2520Library%2520to%2520Dissect%2520Protease%2520Substrate%2520Specificity%26jtitle%3DNat.%2520Protoc.%26date%3D2017%26volume%3D12%26spage%3D2189%26epage%3D2214%26doi%3D10.1038%2Fnprot.2017.091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Poreba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasperkiewicz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snipas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fasci, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvesen, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span> <span> </span><span class="NLM_article-title">Unnatural Amino Acids Increase Sensitivity and Provide for the Design of Highly Selective Caspase Substrates</span>. <i>Cell Death Differ.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1482</span>– <span class="NLM_lpage">1492</span>, <span class="refDoi"> DOI: 10.1038/cdd.2014.64</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1038%2Fcdd.2014.64" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=24832467" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BC2cXot1ymsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=1482-1492&author=M.+Porebaauthor=P.+Kasperkiewiczauthor=S.+J.+Snipasauthor=D.+Fasciauthor=G.+S.+Salvesenauthor=M.+Drag&title=Unnatural+Amino+Acids+Increase+Sensitivity+and+Provide+for+the+Design+of+Highly+Selective+Caspase+Substrates&doi=10.1038%2Fcdd.2014.64"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Unnatural amino acids increase sensitivity and provide for the design of highly selective caspase substrates</span></div><div class="casAuthors">Poreba, M.; Kasperkiewicz, P.; Snipas, S. J.; Fasci, D.; Salvesen, G. S.; Drag, M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death & Differentiation</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1482-1492</span>CODEN:
                <span class="NLM_cas:coden">CDDIEK</span>;
        ISSN:<span class="NLM_cas:issn">1350-9047</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Traditional combinatorial peptidyl substrate library approaches generally utilize natural amino acids, limiting the usefulness of this tool in generating selective substrates for proteases that share similar substrate specificity profiles.  To address this limitation, we synthesized a Hybrid Combinatorial Substrate Library (HyCoSuL) with the general formula of Ac-P4-P3-P2-Asp-ACC, testing the approach on a family of closely related proteases - the human caspases.  The power of this library for caspase discrimination extends far beyond traditional PS-SCL approach, as in addn. to 19 natural amino acids we also used 110 diverse unnatural amino acids that can more extensively explore the chem. space represented by caspase-active sites.  Using this approach we identified and employed peptide-based substrates that provided excellent discrimination between individual caspases, allowing us to simultaneously resolve the individual contribution of the apical caspase-9 and the executioner caspase-3 and caspase-7 in the development of cytochrome-c-dependent apoptosis for the first time.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSDqSSP6nnUrVg90H21EOLACvtfcHk0ljGOYR0Zt6L2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXot1ymsLs%253D&md5=319049bbe5679cb416e1698a900d2a47</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fcdd.2014.64&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcdd.2014.64%26sid%3Dliteratum%253Aachs%26aulast%3DPoreba%26aufirst%3DM.%26aulast%3DKasperkiewicz%26aufirst%3DP.%26aulast%3DSnipas%26aufirst%3DS.%2BJ.%26aulast%3DFasci%26aufirst%3DD.%26aulast%3DSalvesen%26aufirst%3DG.%2BS.%26aulast%3DDrag%26aufirst%3DM.%26atitle%3DUnnatural%2520Amino%2520Acids%2520Increase%2520Sensitivity%2520and%2520Provide%2520for%2520the%2520Design%2520of%2520Highly%2520Selective%2520Caspase%2520Substrates%26jtitle%3DCell%2520Death%2520Differ.%26date%3D2014%26volume%3D21%26spage%3D1482%26epage%3D1492%26doi%3D10.1038%2Fcdd.2014.64" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rut, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poreba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasperkiewicz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snipas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span> <span> </span><span class="NLM_article-title">Selective Substrates and Activity-Based Probes for Imaging of the Human Constitutive 20S Proteasome in Cells and Blood Samples</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">5222</span>– <span class="NLM_lpage">5234</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00026</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00026" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVags7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=5222-5234&author=W.+Rutauthor=M.+Porebaauthor=P.+Kasperkiewiczauthor=S.+J.+Snipasauthor=M.+Drag&title=Selective+Substrates+and+Activity-Based+Probes+for+Imaging+of+the+Human+Constitutive+20S+Proteasome+in+Cells+and+Blood+Samples&doi=10.1021%2Facs.jmedchem.8b00026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Substrates and Activity-Based Probes for Imaging of the Human Constitutive 20S Proteasome in Cells and Blood Samples</span></div><div class="casAuthors">Rut, Wioletta; Poreba, Marcin; Kasperkiewicz, Paulina; Snipas, Scott J.; Drag, Marcin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5222-5234</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The proteasome is an enzyme complex crit. for maintaining protein homeostasis.  Perturbed proteasome function leads to pathologies including cancer and autoimmune and neurodegenerative disease.  Therefore, the proteasome constitutes an excellent mol. target for pharmaceutical development.  Here, using the HyCoSuL approach, the authors designed and synthesized novel and selective fluorogenic substrates for each of these three constitutive 20S proteasome activities and applied them to assess inhibition of proteasome subunits by MG-132 and a clin. used inhibitor bortezomib.  The authors' results confirm the utility of designed substrates in biochem. assays.  Furthermore, selective peptide sequences obtained in this manner were used to construct fluorophore-labeled activity-based probes and then used to detect each constitutive 20S proteasome subunit simultaneously in lysates of HEK-293F cells and red blood cells.  Overall, the authors describe a simple and rapid method useful to measure constitutive 20S proteasome activity in whole human blood samples that could enable early diagnosis of pathol. states assocd. with aberrantly upregulated proteasome activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrd0FSTAesirVg90H21EOLACvtfcHk0ljGOYR0Zt6L2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVags7vO&md5=1fb97b492a7f3f25d6659035f5bf4b14</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00026%26sid%3Dliteratum%253Aachs%26aulast%3DRut%26aufirst%3DW.%26aulast%3DPoreba%26aufirst%3DM.%26aulast%3DKasperkiewicz%26aufirst%3DP.%26aulast%3DSnipas%26aufirst%3DS.%2BJ.%26aulast%3DDrag%26aufirst%3DM.%26atitle%3DSelective%2520Substrates%2520and%2520Activity-Based%2520Probes%2520for%2520Imaging%2520of%2520the%2520Human%2520Constitutive%252020S%2520Proteasome%2520in%2520Cells%2520and%2520Blood%2520Samples%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D5222%26epage%3D5234%26doi%3D10.1021%2Facs.jmedchem.8b00026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bird, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buzza, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKee, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whisstock, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bird, P. I.</span></span> <span> </span><span class="NLM_article-title">Expression and Purification of Recombinant Human Granzyme B from Pichia Pastoris</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>261</i></span>,  <span class="NLM_fpage">251</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1006/bbrc.1999.0989</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1006%2Fbbrc.1999.0989" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10425174" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADyaK1MXks1Okurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=261&publication_year=1999&pages=251-255&author=J.+Sunauthor=C.+H.+Birdauthor=M.+S.+Buzzaauthor=K.+E.+McKeeauthor=J.+C.+Whisstockauthor=P.+I.+Bird&title=Expression+and+Purification+of+Recombinant+Human+Granzyme+B+from+Pichia+Pastoris&doi=10.1006%2Fbbrc.1999.0989"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Expression and purification of recombinant human granzyme B from Pichia pastoris</span></div><div class="casAuthors">Sun, Jiuru; Bird, Catherina H.; Buzza, Marguerite S.; McKee, Katherine E.; Whisstock, James C.; Bird, Phillip I.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">261</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">251-255</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">Granzyme B (I) is a cytotoxic lymphocyte granule serine proteinase that is pivotal in the induction of target cell apoptosis.  Here, the authors describe the expression of recombinant human I in Pichia pastoris as a chimeric zymogen comprising the α-factor signal sequence, a prodomain including an enterokinase cleavage site, and the mature I sequence followed by a hexa-histidine tag.  Inactive zymogen was purified from the medium by immobilized Co2+ chelate affinity chromatog. and then activated by enterokinase (final yield was ∼1 mg/L).  The recombinant enzyme resembled native I in size and glycosylation, hydrolyzed the substrate, Boc-Ala-Ala-Asp-thiobenzyl ester, with equiv. efficiency (Km = 82 μM; kcat = 12 s-1), processed procaspase-3 to its subunit form, and was inhibited by the cognate serpin PI-9.  It efficiently induced DNA degrdn. and apoptosis of human cells.  The availability of recombinant human I will facilitate further investigation of its structure and role in immune effector cells.  (c) 1999 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRjLDtcYtVKLVg90H21EOLACvtfcHk0ljDg4HiYHGIlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXks1Okurk%253D&md5=531feb4c527d01411891554ebe16886e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1006%2Fbbrc.1999.0989&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fbbrc.1999.0989%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DBird%26aufirst%3DC.%2BH.%26aulast%3DBuzza%26aufirst%3DM.%2BS.%26aulast%3DMcKee%26aufirst%3DK.%2BE.%26aulast%3DWhisstock%26aufirst%3DJ.%2BC.%26aulast%3DBird%26aufirst%3DP.%2BI.%26atitle%3DExpression%2520and%2520Purification%2520of%2520Recombinant%2520Human%2520Granzyme%2520B%2520from%2520Pichia%2520Pastoris%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D1999%26volume%3D261%26spage%3D251%26epage%3D255%26doi%3D10.1006%2Fbbrc.1999.0989" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oleksyszyn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, J. C.</span></span> <span> </span><span class="NLM_article-title">Amino Acid and Peptide Phosphonate Derivatives as Specific Inhibitors of Serine Peptidases</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>244</i></span>,  <span class="NLM_fpage">423</span>– <span class="NLM_lpage">441</span>, <span class="refDoi"> DOI: 10.1016/0076-6879(94)44032-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1016%2F0076-6879%2894%2944032-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=7845224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADyaK2MXjvFWrtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=244&publication_year=1994&pages=423-441&author=J.+Oleksyszynauthor=J.+C.+Powers&title=Amino+Acid+and+Peptide+Phosphonate+Derivatives+as+Specific+Inhibitors+of+Serine+Peptidases&doi=10.1016%2F0076-6879%2894%2944032-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Amino acid and peptide phosphonate derivatives as specific inhibitors of serine peptidases</span></div><div class="casAuthors">Oleksyszyn, Jozef; Powers, James C.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">244</span>
        (<span class="NLM_cas:issue">Proteolytic Enzymes: Serine and Cysteine Peptidases</span>),
    <span class="NLM_cas:pages">423-41</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    </div><div class="casAbstract">Peptidyl derivs. of α-aminoalkyl phosphonate di-Ph esters are highly specific irreversible inhibitors of serine peptidases and are chem. stable and stable in plasma.  They offer a no. of advantages over other types of inhibitors currently in use in biol. expts.  After reaction with the enzyme, they form very stable enzyme-inhibitor complexes, making them interesting tools for x-ray studies on the active site structure of new serine peptidases.  Synthetic methods, synthetic examples, and inhibition kinetics and methods are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaaiAF4Y5HxbVg90H21EOLACvtfcHk0ljDg4HiYHGIlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXjvFWrtrs%253D&md5=78dc438666d1816ccbc9551d95b6c281</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2F0076-6879%2894%2944032-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0076-6879%252894%252944032-8%26sid%3Dliteratum%253Aachs%26aulast%3DOleksyszyn%26aufirst%3DJ.%26aulast%3DPowers%26aufirst%3DJ.%2BC.%26atitle%3DAmino%2520Acid%2520and%2520Peptide%2520Phosphonate%2520Derivatives%2520as%2520Specific%2520Inhibitors%2520of%2520Serine%2520Peptidases%26jtitle%3DMethods%2520Enzymol.%26date%3D1994%26volume%3D244%26spage%3D423%26epage%3D441%26doi%3D10.1016%2F0076-6879%2894%2944032-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oleksyszyn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subotkowska, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mastalerz, P.</span></span> <span> </span><span class="NLM_article-title">Diphenyl 1-Aminoalkanephosphonates</span>. <i>Synthesis</i> <span class="NLM_year" style="font-weight: bold;">1979</span>,  <span class="NLM_volume"><i>1979</i></span>,  <span class="NLM_fpage">985</span>– <span class="NLM_lpage">986</span>, <span class="refDoi"> DOI: 10.1055/s-1979-28903</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1055%2Fs-1979-28903" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1979&publication_year=1979&pages=985-986&author=J.+Oleksyszynauthor=L.+Subotkowskaauthor=P.+Mastalerz&title=Diphenyl+1-Aminoalkanephosphonates&doi=10.1055%2Fs-1979-28903"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1055%2Fs-1979-28903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-1979-28903%26sid%3Dliteratum%253Aachs%26aulast%3DOleksyszyn%26aufirst%3DJ.%26aulast%3DSubotkowska%26aufirst%3DL.%26aulast%3DMastalerz%26aufirst%3DP.%26atitle%3DDiphenyl%25201-Aminoalkanephosphonates%26jtitle%3DSynthesis%26date%3D1979%26volume%3D1979%26spage%3D985%26epage%3D986%26doi%3D10.1055%2Fs-1979-28903" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 2 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Hong-Rae Kim, Ravichandra Tagirasa, <span class="NLM_string-name hlFld-ContribAuthor">Euna Yoo</span>. </span><span class="cited-content_cbyCitation_article-title">Covalent Small Molecule Immunomodulators Targeting the Protease Active Site. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (9)
                                     , 5291-5322. <a href="https://doi.org/10.1021/acs.jmedchem.1c00172" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00172</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00172&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00172%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DCovalent%252BSmall%252BMolecule%252BImmunomodulators%252BTargeting%252Bthe%252BProtease%252BActive%252BSite%26aulast%3DKim%26aufirst%3DHong-Rae%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D30012021%26date%3D27042021%26volume%3D64%26issue%3D9%26spage%3D5291%26epage%3D5322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Paulina  Kasperkiewicz</span>. </span><span class="cited-content_cbyCitation_article-title">Peptidyl Activity-Based Probes for Imaging Serine Proteases. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2021,</strong> <em>9 </em><a href="https://doi.org/10.3389/fchem.2021.639410" title="DOI URL">https://doi.org/10.3389/fchem.2021.639410</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2021.639410&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2021.639410%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DPeptidyl%252BActivity-Based%252BProbes%252Bfor%252BImaging%252BSerine%252BProteases%26aulast%3DKasperkiewicz%26aufirst%3DPaulina%26date%3D2021%26date%3D2021%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/medium/jm9b02042_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/large/jm9b02042_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02042&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/medium/jm9b02042_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/large/jm9b02042_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Design and characteristics of GrA substrates. (A) GrA P4–P1 catalytic preferences. The activity of GrA was tested using the HyCoSuL strategy or individual substrates for P1 examination. Substrates were placed into the wells of the plate at the same concentration, followed by the addition of the enzyme in the reaction buffer. The increase in fluorescence was measured with 355 nm/460 nm excitation/emission wavelengths. Data are presented as a violin diagram of all events. For detailed analysis, see <a class="ref internalNav" href="#notes3" aria-label="Supporting Information Figure S1">Supporting Information Figure S1</a>. (B) Amino acid structures selected for new substrate synthesis based on library screening. The best recognized sequences were chosen for individual substrate design.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/large/jm9b02042_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02042&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/medium/jm9b02042_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/large/jm9b02042_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Selection of the most active GrA substrate. (A) Substrates SK7–SK28 were placed into the wells of the plate at the same concentration (1 μM), followed by the addition of 89 nM enzyme in the reaction buffer. The increase in fluorescence was measured with 355 nm/460 nm excitation/emission wavelengths, accordingly. Data present the average of 2 replicates as adjusted RFU/s. (B) Comparison of SK7 (blue) and SK15 (red) structures.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/large/jm9b02042_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02042&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/medium/jm9b02042_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/large/jm9b02042_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) GrA substrate selectivity testing on GrK and GrB. Substrates SK8–SK28 and the reference GrK and GrB substrates were placed in a 96-well plate at a concentration of 200 μM and treated with GrK or GrB. The fluorescence increase was measured with 355 nm/460 nm excitation and emission wavelengths. Data present the average of 2 replicates as adjusted RFU/s. (B) Structures of SK15I and SK15.5 and kinetic evaluation of the inhibitor and probe. (C) Selectivity of SK15I and SK15.5 toward GrK, GrB, GrM, PR3, and hNE.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/large/jm9b02042_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02042&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/medium/jm9b02042_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/large/jm9b02042_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. In-gel GrA detection with SK15.5 using recombinant enzyme and NK92 cell line lysates. (A) Incubation time of SK15.5 with enzyme optimization. GrA (250 nM) was incubated with 250 nM SK15.5 for the indicated time in the range 0–120 min at 37 °C, followed by a reduction with 3× SDS loading buffer and SDS-PAGE analysis. (B) SK15.5 concentration optimization. GrA (250 nM) was incubated with SK15.5 at various concentrations ranging from 10 to 500 nM for 30 min at 37 °C, followed by reduction with 3× SDS loading buffer. (C) SK15.5 binds exclusively with the active GrA. Probe binding was tested on both the active enzyme and the proenzyme at the same concentration and incubation time. The active site GrA inhibitor and the presence of prodipeptide abolished probe binding. (D) GrA in-gel detection in NK92 cell lysates. NK92 cells in a log growth phase were lysed (1 × 10<sup>7</sup> cells/mL) in lysis buffer and sonicated. Lysates were incubated with SK15.5 in a concentration range of 250 nM to 2 μM, as indicated, for 1 h at 37 °C. Samples were run on SDS-PAGE gel, transferred to a membrane, immunostained with anti-GrA, and imaged with 488 and 658 nm lasers. The results are representative of at least 2 replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/large/jm9b02042_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02042&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/medium/jm9b02042_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/large/jm9b02042_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. SK15.5 probe utility in living cells. (A) SK15.5 probe and SK15I inhibitor cytotoxicity. MDA-MB-231 cells were treated with different SK15.5 probe or SK15I inhibitor concentrations in growth media for 24 h, and the cell viability was verified with an MTS assay at 490 nm. As indicated, the probe is cytotoxic only at very high concentrations, higher than 2.5 μM, while the inhibitor shows no cytotoxic effect even at 10 μM. The experiment was performed in duplicate, and the data are presented as an average of these repetitions with standard deviation. (B) SK15.5 probe cell permeability. NK92 cells in culture media were treated with SK15.5 (1 μM) for the indicated time up to 8 h, followed by cell washing, lysis, and SDS-PAGE analysis followed by protein transfer to a nitrocellulose membrane. Where indicated, the cells were preincubated with SK15I to prevent probe binding at the GrA active site. The results represent two biological replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-6/acs.jmedchem.9b02042/20200928/images/large/jm9b02042_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b02042&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i31">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78423" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78423" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 35 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Metkar, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menaa, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallich, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freudenberg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raja, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Froelich, C. J.</span></span> <span> </span><span class="NLM_article-title">Human and Mouse Granzyme A Induce a Proinflammatory Cytokine Response</span>. <i>Immunity</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">720</span>– <span class="NLM_lpage">733</span>, <span class="refDoi"> DOI: 10.1016/j.immuni.2008.08.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1016%2Fj.immuni.2008.08.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=18951048" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVaqsL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2008&pages=720-733&author=S.+S.+Metkarauthor=C.+Menaaauthor=J.+Pardoauthor=B.+Wangauthor=R.+Wallichauthor=M.+Freudenbergauthor=S.+Kimauthor=S.+M.+Rajaauthor=L.+Shiauthor=M.+M.+Simonauthor=C.+J.+Froelich&title=Human+and+Mouse+Granzyme+A+Induce+a+Proinflammatory+Cytokine+Response&doi=10.1016%2Fj.immuni.2008.08.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Human and mouse granzyme A induce a proinflammatory cytokine response</span></div><div class="casAuthors">Metkar, Sunil S.; Menaa, Cheikh; Pardo, Julian; Wang, Baikun; Wallich, Reinhard; Freudenberg, Marina; Kim, Stephen; Raja, Srikumar M.; Shi, Lianfa; Simon, Markus M.; Froelich, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Immunity</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">720-733</span>CODEN:
                <span class="NLM_cas:coden">IUNIEH</span>;
        ISSN:<span class="NLM_cas:issn">1074-7613</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Granzyme A (GzmA) is considered a major proapoptotic protease.  The authors have discovered that GzmA-induced cell death involves rapid membrane damage that depends on the synergy between micromolar concns. of GzmA and sublytic perforin (PFN).  Ironically, GzmA and GzmB, independent of their catalytic activity, both mediated this swift necrosis.  Even without PFN, lower concns. of human GzmA stimulated monocytes to secrete proinflammatory cytokines [interleukin-1β (IL-1β), TNFα, and IL-6] that were blocked by a caspase-1 inhibitor.  Moreover, murine GzmA and GzmA+ cytotoxic T lymphocytes (CTLs) induce IL-1β from primary mouse macrophages, and GzmA-/- mice resist lipopolysaccharide-induced toxicity.  Thus, the granule secretory pathway plays an unexpected role in inflammation, with GzmA acting as an endogenous modulator.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcOH7m4-hE5bVg90H21EOLACvtfcHk0lhHL-NwLNDQOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVaqsL7M&md5=0fd5c17d7f7a7cba24940c03d305851c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.immuni.2008.08.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.immuni.2008.08.014%26sid%3Dliteratum%253Aachs%26aulast%3DMetkar%26aufirst%3DS.%2BS.%26aulast%3DMenaa%26aufirst%3DC.%26aulast%3DPardo%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DWallich%26aufirst%3DR.%26aulast%3DFreudenberg%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DRaja%26aufirst%3DS.%2BM.%26aulast%3DShi%26aufirst%3DL.%26aulast%3DSimon%26aufirst%3DM.%2BM.%26aulast%3DFroelich%26aufirst%3DC.%2BJ.%26atitle%3DHuman%2520and%2520Mouse%2520Granzyme%2520A%2520Induce%2520a%2520Proinflammatory%2520Cytokine%2520Response%26jtitle%3DImmunity%26date%3D2008%26volume%3D29%26spage%3D720%26epage%3D733%26doi%3D10.1016%2Fj.immuni.2008.08.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinvalet, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chowdhury, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlesinger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lieberman, J.</span></span> <span> </span><span class="NLM_article-title">The Cytotoxic T Lymphocyte Protease Granzyme A cleaves and Inactivates Poly(adenosine 5′-diphosphate-ribose) Polymerase-1</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">1205</span>– <span class="NLM_lpage">1216</span>, <span class="refDoi"> DOI: 10.1182/blood-2008-12-195768</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1182%2Fblood-2008-12-195768" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=19506301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVSgsbfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2009&pages=1205-1216&author=P.+Zhuauthor=D.+Martinvaletauthor=D.+Chowdhuryauthor=D.+Zhangauthor=A.+Schlesingerauthor=J.+Lieberman&title=The+Cytotoxic+T+Lymphocyte+Protease+Granzyme+A+cleaves+and+Inactivates+Poly%28adenosine+5%E2%80%B2-diphosphate-ribose%29+Polymerase-1&doi=10.1182%2Fblood-2008-12-195768"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The cytotoxic T lymphocyte protease granzyme A cleaves and inactivates poly(adenosine 5'-diphosphate-ribose) polymerase-1</span></div><div class="casAuthors">Zhu, Pengcheng; Martinvalet, Denis; Chowdhury, Dipanjan; Zhang, Dong; Schlesinger, Ann; Lieberman, Judy</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1205-1216</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Granzyme A (GzmA) in killer cells induces caspase-independent programmed cell death.  In this study, we show that GzmA cleaves the DNA damage sensor poly(ADP-ribose) polymerase-1 (PARP-1) after Lys498 in its automodification domain, sepg. the DNA binding domain from the catalytic domain, which interferes with repair of GzmA-induced DNA damage and enhances susceptibility to GzmA-mediated death.  Overexpressing K498A PARP-1 reduces GzmA-mediated death and drives dying cells to necrosis rather than apoptosis.  Conversely, inhibiting or genetically disrupting PARP-1 enhances cell vulnerability.  The N-terminal GzmA cleavage fragment of PARP-1 acts as a PARP-1 dominant neg., binding to DNA and blocking DNA repair.  Disrupting PARP-1, which is also a caspase target, is therefore required for efficient apoptosis by both caspase-independent and caspase-dependent pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7GDXiodn5VrVg90H21EOLACvtfcHk0lhHL-NwLNDQOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVSgsbfJ&md5=5f94645fd597047c79d10a1e81001daa</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1182%2Fblood-2008-12-195768&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2008-12-195768%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DP.%26aulast%3DMartinvalet%26aufirst%3DD.%26aulast%3DChowdhury%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DSchlesinger%26aufirst%3DA.%26aulast%3DLieberman%26aufirst%3DJ.%26atitle%3DThe%2520Cytotoxic%2520T%2520Lymphocyte%2520Protease%2520Granzyme%2520A%2520cleaves%2520and%2520Inactivates%2520Poly%2528adenosine%25205%25E2%2580%25B2-diphosphate-ribose%2529%2520Polymerase-1%26jtitle%3DBlood%26date%3D2009%26volume%3D114%26spage%3D1205%26epage%3D1216%26doi%3D10.1182%2Fblood-2008-12-195768" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Law, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukoyanova, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voskoboinik, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caradoc-Davies, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baran, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunstone, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Angelo, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orlova, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coulibaly, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verschoor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Browne, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciccone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuiper, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bird, P. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trapani, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saibil, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whisstock, J. C.</span></span> <span> </span><span class="NLM_article-title">The Structural Basis for Membrane Binding and Pore Formation by Lymphocyte Perforin</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>468</i></span>,  <span class="NLM_fpage">447</span>– <span class="NLM_lpage">451</span>, <span class="refDoi"> DOI: 10.1038/nature09518</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1038%2Fnature09518" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=21037563" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlGrsLjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=468&publication_year=2010&pages=447-451&author=R.+H.+Lawauthor=N.+Lukoyanovaauthor=I.+Voskoboinikauthor=T.+T.+Caradoc-Daviesauthor=K.+Baranauthor=M.+A.+Dunstoneauthor=M.+E.+D%E2%80%99Angeloauthor=E.+V.+Orlovaauthor=F.+Coulibalyauthor=S.+Verschoorauthor=K.+A.+Browneauthor=A.+Cicconeauthor=M.+J.+Kuiperauthor=P.+I.+Birdauthor=J.+A.+Trapaniauthor=H.+R.+Saibilauthor=J.+C.+Whisstock&title=The+Structural+Basis+for+Membrane+Binding+and+Pore+Formation+by+Lymphocyte+Perforin&doi=10.1038%2Fnature09518"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The structural basis for membrane binding and pore formation by lymphocyte perforin</span></div><div class="casAuthors">Law, Ruby H. P.; Lukoyanova, Natalya; Voskoboinik, Ilia; Caradoc-Davies, Tom T.; Baran, Katherine; Dunstone, Michelle A.; D'Angelo, Michael E.; Orlova, Elena V.; Coulibaly, Fasseli; Verschoor, Sandra; Browne, Kylie A.; Ciccone, Annette; Kuiper, Michael J.; Bird, Phillip I.; Trapani, Joseph A.; Saibil, Helen R.; Whisstock, James C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">468</span>
        (<span class="NLM_cas:issue">7322</span>),
    <span class="NLM_cas:pages">447-451</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Natural killer cells and cytotoxic T lymphocytes accomplish the critically important function of killing virus-infected and neoplastic cells.  They do this by releasing the pore-forming protein perforin and granzyme proteases from cytoplasmic granules into the cleft formed between the abutting killer and target cell membranes.  Perforin, a 67-kilodalton multidomain protein, oligomerizes to form pores that deliver the pro-apoptopic granzymes into the cytosol of the target cell.  The importance of perforin is highlighted by the fatal consequences of congenital perforin deficiency, with more than 50 different perforin mutations linked to familial haemophagocytic lymphohistiocytosis (type 2 FHL).  Here we elucidate the mechanism of perforin pore formation by detg. the X-ray crystal structure of monomeric murine perforin, together with a cryo-electron microscopy reconstruction of the entire perforin pore.  Perforin is a thin 'key-shaped' mol., contg. an amino-terminal membrane attack complex perforin-like (MACPF)/cholesterol dependent cytolysin (CDC) domain followed by an epidermal growth factor (EGF) domain that together with the extreme carboxy-terminal sequence, forms a central shelf-like structure.  A C-terminal C2 domain mediates initial, Ca2+-dependent membrane binding.  Most unexpectedly, however, electron microscopy reveals that the orientation of the perforin MACPF domain in the pore is inside-out relative to the subunit arrangement in CDCs.  These data reveal remarkable flexibility in the mechanism of action of the conserved MACPF/CDC fold and provide new insights into how related immune defense mols. such as complement proteins assemble into pores.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvdSkxAx6O_bVg90H21EOLACvtfcHk0lhHL-NwLNDQOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlGrsLjP&md5=39539b6f65789792a95773391b6a3a21</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnature09518&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09518%26sid%3Dliteratum%253Aachs%26aulast%3DLaw%26aufirst%3DR.%2BH.%26aulast%3DLukoyanova%26aufirst%3DN.%26aulast%3DVoskoboinik%26aufirst%3DI.%26aulast%3DCaradoc-Davies%26aufirst%3DT.%2BT.%26aulast%3DBaran%26aufirst%3DK.%26aulast%3DDunstone%26aufirst%3DM.%2BA.%26aulast%3DD%25E2%2580%2599Angelo%26aufirst%3DM.%2BE.%26aulast%3DOrlova%26aufirst%3DE.%2BV.%26aulast%3DCoulibaly%26aufirst%3DF.%26aulast%3DVerschoor%26aufirst%3DS.%26aulast%3DBrowne%26aufirst%3DK.%2BA.%26aulast%3DCiccone%26aufirst%3DA.%26aulast%3DKuiper%26aufirst%3DM.%2BJ.%26aulast%3DBird%26aufirst%3DP.%2BI.%26aulast%3DTrapani%26aufirst%3DJ.%2BA.%26aulast%3DSaibil%26aufirst%3DH.%2BR.%26aulast%3DWhisstock%26aufirst%3DJ.%2BC.%26atitle%3DThe%2520Structural%2520Basis%2520for%2520Membrane%2520Binding%2520and%2520Pore%2520Formation%2520by%2520Lymphocyte%2520Perforin%26jtitle%3DNature%26date%3D2010%26volume%3D468%26spage%3D447%26epage%3D451%26doi%3D10.1038%2Fnature09518" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whisstock, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voskoboinik, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trapani, J. A.</span></span> <span> </span><span class="NLM_article-title">Protecting a Serial Killer: Pathways for Perforin Trafficking and Self-Defence Ensure Sequential Target Cell Death</span>. <i>Trends Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">406</span>– <span class="NLM_lpage">412</span>, <span class="refDoi"> DOI: 10.1016/j.it.2012.04.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1016%2Fj.it.2012.04.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=22608996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVymtrnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=406-412&author=J.+A.+Lopezauthor=A.+J.+Brennanauthor=J.+C.+Whisstockauthor=I.+Voskoboinikauthor=J.+A.+Trapani&title=Protecting+a+Serial+Killer%3A+Pathways+for+Perforin+Trafficking+and+Self-Defence+Ensure+Sequential+Target+Cell+Death&doi=10.1016%2Fj.it.2012.04.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Protecting a serial killer: pathways for perforin trafficking and self-defence ensure sequential target cell death</span></div><div class="casAuthors">Lopez, Jamie A.; Brennan, Amelia J.; Whisstock, James C.; Voskoboinik, Ilia; Trapani, Joseph A.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Immunology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">406-412</span>CODEN:
                <span class="NLM_cas:coden">TIRMAE</span>;
        ISSN:<span class="NLM_cas:issn">1471-4906</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Considerable progress has been made in understanding how cytotoxic lymphocytes use the highly toxic pore-forming protein perforin to eliminate dangerous cells, while remaining refractory to lysis.  At least two mechanisms jointly preserve the killer cell: the C-terminal residues of perforin dictate its rapid export from the endoplasmic reticulum (ER), whose milieu otherwise favors pore formation; perforin is then stored in secretory granules whose acidity prevent its oligomerization.  Following exocytosis, perforin delivers the proapoptotic protease, granzyme B, into the target cell by disrupting its plasma membrane.  Although the precise mechanism of perforin/granzyme synergy remains controversial, the recently defined crystal structure of the perforin monomer and cryo-electron microscopy (EM) of the entire pore suggest that passive transmembrane granzyme diffusion is the dominant proapoptotic mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0hU-DsmbO5rVg90H21EOLACvtfcHk0lj3S-CgwbsIqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVymtrnE&md5=fab538509cf3aba68fdb78bab97e1b31</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.it.2012.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.it.2012.04.001%26sid%3Dliteratum%253Aachs%26aulast%3DLopez%26aufirst%3DJ.%2BA.%26aulast%3DBrennan%26aufirst%3DA.%2BJ.%26aulast%3DWhisstock%26aufirst%3DJ.%2BC.%26aulast%3DVoskoboinik%26aufirst%3DI.%26aulast%3DTrapani%26aufirst%3DJ.%2BA.%26atitle%3DProtecting%2520a%2520Serial%2520Killer%253A%2520Pathways%2520for%2520Perforin%2520Trafficking%2520and%2520Self-Defence%2520Ensure%2520Sequential%2520Target%2520Cell%2520Death%26jtitle%3DTrends%2520Immunol.%26date%3D2012%26volume%3D33%26spage%3D406%26epage%3D412%26doi%3D10.1016%2Fj.it.2012.04.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
Damme, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maurer-Stroh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colaert, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timmerman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenhaber, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandekerckhove, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gevaert, K.</span></span> <span> </span><span class="NLM_article-title">The Substrate Specificity Profile of Human Granzyme A</span>. <i>Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>391</i></span>,  <span class="NLM_fpage">983</span>– <span class="NLM_lpage">997</span>, <span class="refDoi"> DOI: 10.1515/bc.2010.096</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1515%2Fbc.2010.096" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=20536382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFyhur3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=391&publication_year=2010&pages=983-997&author=P.+Van%0ADammeauthor=S.+Maurer-Strohauthor=H.+Haoauthor=N.+Colaertauthor=E.+Timmermanauthor=F.+Eisenhaberauthor=J.+Vandekerckhoveauthor=K.+Gevaert&title=The+Substrate+Specificity+Profile+of+Human+Granzyme+A&doi=10.1515%2Fbc.2010.096"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The substrate specificity profile of human granzyme A</span></div><div class="casAuthors">Van Damme, Petra; Maurer-Stroh, Sebastian; Hao, Han; Colaert, Niklaas; Timmerman, Evy; Eisenhaber, Frank; Vandekerckhove, Joel; Gevaert, Kris</div><div class="citationInfo"><span class="NLM_cas:title">Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">391</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">983-997</span>CODEN:
                <span class="NLM_cas:coden">BICHF3</span>;
        ISSN:<span class="NLM_cas:issn">1431-6730</span>.
    
            (<span class="NLM_cas:orgname">Walter de Gruyter GmbH & Co. KG</span>)
        </div><div class="casAbstract">The exact biol. function of granzyme A, a granule-assocd. serine protease belonging to the tryptase family of proteases, is still a matter of debate because conflicting roles have been suggested, such as initiation of caspase-independent apoptosis-like cell death and endogenous modulation of inflammatory processes.  In contrast to its well-studied family member, granzyme B, far less is known about the physiol. targets of granzyme A.  Using an N-terminal peptide-centric proteomics technol., the substrate specificity of human granzyme A was extensively characterized at the level of macromol. protein substrates.  Overall, more than 260 cleavage sites, almost exclusively favoring basic residues at the P1 position, in approx. 200 unique protein substrates, including the well-known in vitro substrates APEX-endonuclease 1 and different histones, were identified.  Further substrate characterization was used to delineate phys. properties in the substrate specificity profiles, which further highlights important aspects in protease/substrate biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIzGI0uDl6ybVg90H21EOLACvtfcHk0lj3S-CgwbsIqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFyhur3M&md5=9b5fa0c047066e1683e4ada5781f298e</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1515%2Fbc.2010.096&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252Fbc.2010.096%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BDamme%26aufirst%3DP.%26aulast%3DMaurer-Stroh%26aufirst%3DS.%26aulast%3DHao%26aufirst%3DH.%26aulast%3DColaert%26aufirst%3DN.%26aulast%3DTimmerman%26aufirst%3DE.%26aulast%3DEisenhaber%26aufirst%3DF.%26aulast%3DVandekerckhove%26aufirst%3DJ.%26aulast%3DGevaert%26aufirst%3DK.%26atitle%3DThe%2520Substrate%2520Specificity%2520Profile%2520of%2520Human%2520Granzyme%2520A%26jtitle%3DBiol.%2520Chem.%26date%3D2010%26volume%3D391%26spage%3D983%26epage%3D997%26doi%3D10.1515%2Fbc.2010.096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Masson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tschopp, J.</span></span> <span> </span><span class="NLM_article-title">A Family of Serine Esterases in Lytic Granules of Cytolytic T Lymphocytes</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1987</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">679</span>– <span class="NLM_lpage">685</span>, <span class="refDoi"> DOI: 10.1016/0092-8674(87)90544-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1016%2F0092-8674%2887%2990544-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=3555842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADyaL1cXht1OhtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=1987&pages=679-685&author=D.+Massonauthor=J.+Tschopp&title=A+Family+of+Serine+Esterases+in+Lytic+Granules+of+Cytolytic+T+Lymphocytes&doi=10.1016%2F0092-8674%2887%2990544-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">A family of serine esterases in lytic granules of cytolytic T lymphocytes</span></div><div class="casAuthors">Masson, Daniele; Tschopp, Juerg</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">1987</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">679-85</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    </div><div class="casAbstract">Cytoplasmic granules of cytolytic T lymphocytes (CTLs) contain, in addn. to the pore-forming protein perforin, a family of highly homologous serine esterases, granzymes A-H.  The serine esterase affinity label diisopropyl fluorophosphate reacts strongly with granzymes A and D, to a lesser extent with B, E, G, and H, and not at all with C and F.  For granzymes A and D, synthetic substrates have been found.  Antibodies raised against granzyme B strongly crossreact with A, G, and H, and antibodies to granzyme D recognize C, E, and F.  These antigenic relationships correlate with similarities in the N-terminal amino acid sequences.  At least 60% homol. is obsd. between the 8 proteins, and all are similar to rat mast cell protease 2.  Sequence anal. suggests the identity of granzyme A with a protease predicted from a CTL-specific cDNA clone (H factor) and of granzyme B, G, or H with a protein encoded by the CTL-specific cDNA clone CTLA 1/CCP 1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4eATGzgITYrVg90H21EOLACvtfcHk0lg5rWWqjssqKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXht1OhtLk%253D&md5=bb34a562c3d9ea9c83c28714673ab9a8</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2887%2990544-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252887%252990544-7%26sid%3Dliteratum%253Aachs%26aulast%3DMasson%26aufirst%3DD.%26aulast%3DTschopp%26aufirst%3DJ.%26atitle%3DA%2520Family%2520of%2520Serine%2520Esterases%2520in%2520Lytic%2520Granules%2520of%2520Cytolytic%2520T%2520Lymphocytes%26jtitle%3DCell%26date%3D1987%26volume%3D49%26spage%3D679%26epage%3D685%26doi%3D10.1016%2F0092-8674%2887%2990544-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bots, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Bostelen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rademaker, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Offringa, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medema, J. P.</span></span> <span> </span><span class="NLM_article-title">Serpins Prevent Granzyme-Induced Death in a Species-Specific Manner</span>. <i>Immunol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">79</span>– <span class="NLM_lpage">86</span>, <span class="refDoi"> DOI: 10.1111/j.1440-1711.2005.01417.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1111%2Fj.1440-1711.2005.01417.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=16405655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BD28Xhs1CqsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2006&pages=79-86&author=M.+Botsauthor=L.+Van+Bostelenauthor=M.+T.+Rademakerauthor=R.+Offringaauthor=J.+P.+Medema&title=Serpins+Prevent+Granzyme-Induced+Death+in+a+Species-Specific+Manner&doi=10.1111%2Fj.1440-1711.2005.01417.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Serpins prevent granzyme-induced death in a species-specific manner</span></div><div class="casAuthors">Bots, Michael; Van Bostelen, Liesbeth; Rademaker, Mirjam Tga; Offringa, Rienk; Medema, Jan Paul</div><div class="citationInfo"><span class="NLM_cas:title">Immunology and Cell Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">79-86</span>CODEN:
                <span class="NLM_cas:coden">ICBIEZ</span>;
        ISSN:<span class="NLM_cas:issn">0818-9641</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Asia Pty Ltd.</span>)
        </div><div class="casAbstract">Expression of serine protease inhibitors (serpins) is one of the mechanisms used by tumor cells to escape immune surveillance.  Previously, we have shown that expression of serpins SPI-6 and SPI-CI, resp., renders tumor cells resistant to granzyme B (GrB)-mediated death and granzyme M (GrM)-mediated death.  To obtain better insight into the interaction between serpins and their target proteases, we investigated the roles of protease inhibitor (PI)-9 and SPI-6 in the resistance to GrB-mediated and CD95-mediated death in further detail.  Neither human PI-9 nor its murine orthologue SPI-6 was capable of preventing CD95-induced apoptosis in murine or human cells, indicating that these serpins do not inhibit the activation of apical caspases in this pathway.  High expression of PI-9 or SPI-6 did prevent apoptosis induced by human GrB.  Strikingly, only SPI-6, and not PI-9, was capable of inhibiting murine GrB, suggesting that a difference in enzymic specificity exists between the mouse and the human granzymes.  In agreement with this suggestion, murine GrB was clearly less effective in inducing apoptosis in human cells.  Similar species specificity was also obsd. for SPI-CI and GrM when either their capacity to assoc. or the effectiveness of GrM-induced cytotoxicity was analyzed.  Our findings therefore indicate a species diversity that has a clear effect on mixed in vitro effector target settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdxJrk6vU67bVg90H21EOLACvtfcHk0lg5rWWqjssqKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xhs1CqsLg%253D&md5=5acd0431f4c9c6b6532e7641ea1ecfa6</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1111%2Fj.1440-1711.2005.01417.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1440-1711.2005.01417.x%26sid%3Dliteratum%253Aachs%26aulast%3DBots%26aufirst%3DM.%26aulast%3DVan%2BBostelen%26aufirst%3DL.%26aulast%3DRademaker%26aufirst%3DM.%2BT.%26aulast%3DOffringa%26aufirst%3DR.%26aulast%3DMedema%26aufirst%3DJ.%2BP.%26atitle%3DSerpins%2520Prevent%2520Granzyme-Induced%2520Death%2520in%2520a%2520Species-Specific%2520Manner%26jtitle%3DImmunol.%2520Cell%2520Biol.%26date%3D2006%26volume%3D84%26spage%3D79%26epage%3D86%26doi%3D10.1111%2Fj.1440-1711.2005.01417.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bird, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutton, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coughlin, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Jong, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trapani, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bird, P. I.</span></span> <span> </span><span class="NLM_article-title">A Cytosolic Granzyme B Inhibitor Related to the Viral Apoptotic Regulator Cytokine Response Modifier A Is Present in Cytotoxic Lymphocytes</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>271</i></span>,  <span class="NLM_fpage">27802</span>– <span class="NLM_lpage">27809</span>, <span class="refDoi"> DOI: 10.1074/jbc.271.44.27802</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1074%2Fjbc.271.44.27802" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=8910377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADyaK28XmvVWrs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=271&publication_year=1996&pages=27802-27809&author=J.+Sunauthor=C.+H.+Birdauthor=V.+Suttonauthor=L.+McDonaldauthor=P.+B.+Coughlinauthor=T.+A.+De+Jongauthor=J.+A.+Trapaniauthor=P.+I.+Bird&title=A+Cytosolic+Granzyme+B+Inhibitor+Related+to+the+Viral+Apoptotic+Regulator+Cytokine+Response+Modifier+A+Is+Present+in+Cytotoxic+Lymphocytes&doi=10.1074%2Fjbc.271.44.27802"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">A cytosolic Granzyme B inhibitor related to the viral apoptotic regulator cytokine response modifier A is present in cytotoxic lymphocytes</span></div><div class="casAuthors">Sun, Jiuru; Bird, Catherina H.; Sutton, Vivien; McDonald, Lisa; Coughlin, Paul B.; De Jong, Tanya A.; Trapani, Joseph A.; Bird, Phillip I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">271</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">27802-27809</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Using a polymerase chain reaction strategy we identified a serine proteinase inhibitor (serpin) in human bone marrow that is related to the cellular serpin proteinase inhibitor 6 (PI-6) and the viral serpin cytokine response modifier A (CrmA).  This serpin, proteinase inhibitor 9 (PI-9), has an unusual reactive center P1(Glu)-P1'(Cys), which suggests that it inhibits serine proteinases that cleave after acidic residues.  The only known serine proteinase with this specificity is granzyme B, a granule cytotoxin produced by cytotoxic lymphocytes.  To test the interaction of PI-9 with granzyme B we prepd. recombinant hexa-histidine tagged PI-9 in a yeast expression system.  Addn. of the recombinant protein to native granzyme B resulted in an SDS-resistant complex typical of serpin-serine proteinase interactions.  Further anal. showed that complex formation followed bimol. kinetics with a second order rate const. of 1.7 × 106 M-1s-1, which is in the range for a physiol. significant serpin-proteinase interaction.  Recombinant PI-9 also completely abrogated granzyme B and perforin-mediated cytotoxicity in vitro.  Examn. of PI-9 mRNA distribution demonstrated that it is expressed in immune tissue, primarily in lymphocytes.  The highest levels of PI-9 mRNA and protein were obsd. in natural killer cell leukemia cell lines and in interleukin-2 stimulated peripheral blood mono-nuclear cells, which also produce granzyme B.  Like PI-6, PI-9 was shown to be a cytosolic protein that is not secreted.  Fractionation of natural killer cells and stimulated peripheral blood mononuclear cells demonstrated that PI-9 is in a sep. subcellular compartment to granzyme B.  These results suggest that PI-9 serves to inactivate misdirected granzyme B following cytotoxic cell degranulation.  This may explain why cytotoxic cells are not damaged by their own granzyme B during destruction of abnormal cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNfbHp6EO9D7Vg90H21EOLACvtfcHk0lg5rWWqjssqKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmvVWrs7g%253D&md5=6d268203aab57b62ccd40354900bf05f</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1074%2Fjbc.271.44.27802&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.271.44.27802%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DBird%26aufirst%3DC.%2BH.%26aulast%3DSutton%26aufirst%3DV.%26aulast%3DMcDonald%26aufirst%3DL.%26aulast%3DCoughlin%26aufirst%3DP.%2BB.%26aulast%3DDe%2BJong%26aufirst%3DT.%2BA.%26aulast%3DTrapani%26aufirst%3DJ.%2BA.%26aulast%3DBird%26aufirst%3DP.%2BI.%26atitle%3DA%2520Cytosolic%2520Granzyme%2520B%2520Inhibitor%2520Related%2520to%2520the%2520Viral%2520Apoptotic%2520Regulator%2520Cytokine%2520Response%2520Modifier%2520A%2520Is%2520Present%2520in%2520Cytotoxic%2520Lymphocytes%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1996%26volume%3D271%26spage%3D27802%26epage%3D27809%26doi%3D10.1074%2Fjbc.271.44.27802" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bird, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutton, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirst, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trapani, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bird, P. I.</span></span> <span> </span><span class="NLM_article-title">Selective Regulation of Apoptosis: the Cytotoxic Lymphocyte Serpin Proteinase Inhibitor 9 Protects Against Granzyme B-Mediated Apoptosis Without Perturbing the Fas Cell Death Pathway</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">6387</span>– <span class="NLM_lpage">6398</span>, <span class="refDoi"> DOI: 10.1128/MCB.18.11.6387</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1128%2FMCB.18.11.6387" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=9774654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADyaK1cXmvF2gu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1998&pages=6387-6398&author=C.+H.+Birdauthor=V.+R.+Suttonauthor=J.+Sunauthor=C.+E.+Hirstauthor=A.+Novakauthor=S.+Kumarauthor=J.+A.+Trapaniauthor=P.+I.+Bird&title=Selective+Regulation+of+Apoptosis%3A+the+Cytotoxic+Lymphocyte+Serpin+Proteinase+Inhibitor+9+Protects+Against+Granzyme+B-Mediated+Apoptosis+Without+Perturbing+the+Fas+Cell+Death+Pathway&doi=10.1128%2FMCB.18.11.6387"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Selective regulation of apoptosis: the cytotoxic lymphocyte serpin proteinase inhibitor 9 protects against granzyme B-mediated apoptosis without perturbing the Fas cell death pathway</span></div><div class="casAuthors">Bird, Catherina H.; Sutton, Vivien R.; Sun, Jiuru; Hirst, Claire E.; Novak, Andrea; Kumar, Sharad; Trapani, Joseph A.; Bird, Phillip I.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6387-6398</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Cytotoxic lymphocytes (CLs) induce caspase activation and apoptosis of target cells either through Fas activation or through release of granule cytotoxins, particularly granzyme B.  CLs themselves resist granule-mediated apoptosis but are eventually cleared via Fas-mediated apoptosis.  Here we show that the CL cytoplasmic serpin proteinase inhibitor 9 (PI-9) can protect transfected cells against apoptosis induced by either purified granzyme B and perforin or intact CLs.  A PI-9 P1 mutant (Glu to Asp) is a 100-fold-less-efficient granzyme B inhibitor that no longer protects against granzyme B-mediated apoptosis.  PI-9 is highly specific for granzyme B because it does not inhibit eight of the nine caspases tested or protect transfected cells against Fas-mediated apoptosis.  In contrast, the P1(Asp) mutant is an effective caspase inhibitor that protects against Fas-mediated apoptosis.  We propose that PI-9 shields CLs specifically against misdirected granzyme B to prevent autolysis or fratricide, but it does not interfere with homeostatic deletion via Fas-mediated apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqG5qYadKK0ibVg90H21EOLACvtfcHk0lgqkL_Wf-JYhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmvF2gu78%253D&md5=1a0ec08419e9661d697bdacea4b1a1b9</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1128%2FMCB.18.11.6387&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.18.11.6387%26sid%3Dliteratum%253Aachs%26aulast%3DBird%26aufirst%3DC.%2BH.%26aulast%3DSutton%26aufirst%3DV.%2BR.%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DHirst%26aufirst%3DC.%2BE.%26aulast%3DNovak%26aufirst%3DA.%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DTrapani%26aufirst%3DJ.%2BA.%26aulast%3DBird%26aufirst%3DP.%2BI.%26atitle%3DSelective%2520Regulation%2520of%2520Apoptosis%253A%2520the%2520Cytotoxic%2520Lymphocyte%2520Serpin%2520Proteinase%2520Inhibitor%25209%2520Protects%2520Against%2520Granzyme%2520B-Mediated%2520Apoptosis%2520Without%2520Perturbing%2520the%2520Fas%2520Cell%2520Death%2520Pathway%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1998%26volume%3D18%26spage%3D6387%26epage%3D6398%26doi%3D10.1128%2FMCB.18.11.6387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaiserman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plasman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gevaert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Damme, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bird, P. I.</span></span> <span> </span><span class="NLM_article-title">Identification of Serpinb6b as a Species-Specific Mouse Granzyme A Inhibitor Suggests Functional Divergence Between Human and Mouse Granzyme A</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>289</i></span>,  <span class="NLM_fpage">9408</span>– <span class="NLM_lpage">9417</span>, <span class="refDoi"> DOI: 10.1074/jbc.M113.525808</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1074%2Fjbc.M113.525808" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=24505135" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BC2cXltVSrsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=9408-9417&author=D.+Kaisermanauthor=S.+E.+Stewartauthor=K.+Plasmanauthor=K.+Gevaertauthor=P.+Van+Dammeauthor=P.+I.+Bird&title=Identification+of+Serpinb6b+as+a+Species-Specific+Mouse+Granzyme+A+Inhibitor+Suggests+Functional+Divergence+Between+Human+and+Mouse+Granzyme+A&doi=10.1074%2Fjbc.M113.525808"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Serpinb6b as a Species-specific Mouse Granzyme A Inhibitor Suggests Functional Divergence between Human and Mouse Granzyme A</span></div><div class="casAuthors">Kaiserman, Dion; Stewart, Sarah E.; Plasman, Kim; Gevaert, Kris; Van Damme, Petra; Bird, Phillip I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">9408-9417</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The granzyme family serine proteases are key effector mols. expressed by cytotoxic lymphocytes.  The physiol. role of granzyme (Gzm) A is controversial, with significant debate over its ability to induce death in target cells.  Here, we investigate the natural inhibitors of GzmA.  We employed substrate phage display and positional proteomics to compare substrate specificities of mouse (m) and human (h) GzmA at the peptide and proteome-wide levels and we used the resulting substrate specificity profiles to search for potential inhibitors from the intracellular serpin family.  We identified Serpinb6b as a potent inhibitor of mGzmA.  Serpinb6b interacts with mGzmA, but not hGzmA, with an assocn. const. of 1.9 ± 0.8 × 105 m-1 s-1 and a stoichiometry of inhibition of 1.8.  Mouse GzmA is over five times more cytotoxic than hGzmA when delivered into P815 target cells with streptolysin O, whereas transfection of target cells with a Serpinb6b cDNA increases the EC50 value of mGzmA 13-fold, without affecting hGzmA cytotoxicity.  Unexpectedly, we also found that Serpinb6b employs an exosite to specifically inhibit dimeric but not monomeric mGzmA.  The identification of an intracellular inhibitor specific for mGzmA only indicates that a lineage-specific increase in GzmA cytotoxic potential has driven cognate inhibitor evolution.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2Tt5NBdjb6rVg90H21EOLACvtfcHk0lgqkL_Wf-JYhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXltVSrsbY%253D&md5=53a28002b6b0d7146d4f8aa2e46b5fad</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.525808&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.525808%26sid%3Dliteratum%253Aachs%26aulast%3DKaiserman%26aufirst%3DD.%26aulast%3DStewart%26aufirst%3DS.%2BE.%26aulast%3DPlasman%26aufirst%3DK.%26aulast%3DGevaert%26aufirst%3DK.%26aulast%3DVan%2BDamme%26aufirst%3DP.%26aulast%3DBird%26aufirst%3DP.%2BI.%26atitle%3DIdentification%2520of%2520Serpinb6b%2520as%2520a%2520Species-Specific%2520Mouse%2520Granzyme%2520A%2520Inhibitor%2520Suggests%2520Functional%2520Divergence%2520Between%2520Human%2520and%2520Mouse%2520Granzyme%2520A%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D9408%26epage%3D9417%26doi%3D10.1074%2Fjbc.M113.525808" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Masson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tschopp, J.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Lymphocyte Protease Granzyme A by Antithrombin III</span>. <i>Mol. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1283</span>– <span class="NLM_lpage">1289</span>, <span class="refDoi"> DOI: 10.1016/0161-5890(88)90043-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1016%2F0161-5890%2888%2990043-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=3266293" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADyaL1MXhsFSgtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1988&pages=1283-1289&author=D.+Massonauthor=J.+Tschopp&title=Inhibition+of+Lymphocyte+Protease+Granzyme+A+by+Antithrombin+III&doi=10.1016%2F0161-5890%2888%2990043-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of lymphocyte protease granzyme A by antithrombin III</span></div><div class="casAuthors">Masson, Daniele; Tschopp, Juerg</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Immunology</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1283-9</span>CODEN:
                <span class="NLM_cas:coden">MOIMD5</span>;
        ISSN:<span class="NLM_cas:issn">0161-5890</span>.
    </div><div class="casAbstract">T-lymphocytes contain a cytoplasmic granule assocd. homo-dimeric protease designated granzyme A.  Upon T-cell-target cell interaction, the granules undergo exocytosis and granzyme A, and other granule constituents, are released.  Granzyme A secreted into plasma is immediately inactivated by antithrombin III.  The rate of complex formation is enhanced 400-fold in the presence of heparin.  Two different complexes are generated: granzyme A-antithrombin III and granzyme A-(antithrombin III)2, resp., indicating that both active centers of granzyme A are functional.  Thus, the proteolytic activity of lymphocyte protease granzyme A, whose physiol. relevant function is unknown, is well regulated in plasma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopFdXHadn9I7Vg90H21EOLACvtfcHk0lgqkL_Wf-JYhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXhsFSgtr4%253D&md5=884f8720d9344b4587c2a5c0641e0a4b</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2F0161-5890%2888%2990043-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0161-5890%252888%252990043-0%26sid%3Dliteratum%253Aachs%26aulast%3DMasson%26aufirst%3DD.%26aulast%3DTschopp%26aufirst%3DJ.%26atitle%3DInhibition%2520of%2520Lymphocyte%2520Protease%2520Granzyme%2520A%2520by%2520Antithrombin%2520III%26jtitle%3DMol.%2520Immunol.%26date%3D1988%26volume%3D25%26spage%3D1283%26epage%3D1289%26doi%3D10.1016%2F0161-5890%2888%2990043-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allison, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodard, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gault, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ewoldt, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kam, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abuelyaman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudig, D.</span></span> <span> </span><span class="NLM_article-title">Characterization, Application and Potential Uses of Biotin-Tagged Inhibitors for Lymphocyte Serine Proteases (granzymes)</span>. <i>Mol. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">615</span>– <span class="NLM_lpage">623</span>, <span class="refDoi"> DOI: 10.1016/0161-5890(96)00025-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1016%2F0161-5890%2896%2900025-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=8760273" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADyaK28XkvVOntLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=1996&pages=615-623&author=U.+Winklerauthor=N.+J.+Allisonauthor=S.+L.+Woodardauthor=R.+A.+Gaultauthor=G.+R.+Ewoldtauthor=C.+M.+Kamauthor=A.+Abuelyamanauthor=J.+C.+Powersauthor=D.+Hudig&title=Characterization%2C+Application+and+Potential+Uses+of+Biotin-Tagged+Inhibitors+for+Lymphocyte+Serine+Proteases+%28granzymes%29&doi=10.1016%2F0161-5890%2896%2900025-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization, application and potential uses of biotin-tagged inhibitors for lymphocyte serine proteases (granzymes)</span></div><div class="casAuthors">Winkler, Ulrike; Allison, N. Janine; Woodard, Susan L.; Gault, Ruth A.; Ewoldt, Gerald R.; Kam, Chih-Min; Abuelyaman, Ahmed; Powers, James C.; Hudig, Dorothy</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Immunology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">7/8</span>),
    <span class="NLM_cas:pages">615-623</span>CODEN:
                <span class="NLM_cas:coden">MOIMD5</span>;
        ISSN:<span class="NLM_cas:issn">0161-5890</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Cytotoxic lymphocytes and natural killer cells are able to kill their target cells in minutes.  The death of the target cell occurs after the release of cytoplasmic granules from the effector cell.  These granules contain the pore-forming protein perforin and serine proteases (granzymes).  To date 10 genes encoding lymphocyte granzymes have been discovered; of these only four have been purified and characterized for their substrate specificity.  Several are predicted to have a common chymase-like specificity which is found in the granule exts.  Others may need to be enriched as active enzymes before they can be evaluated for substrate hydrolysis.  Due to the limitations of detection by substrate hydrolysis, a more sensitive method for the detection of dil. granules was needed.  We report the differing reactivities of seven biotin (Bi)-tagged isocoumarin (IC) inhibitors for Asp-ase, chymase, tryptase and Met-ase granzymes.  The inhibitors contained different substituents at their no. 3 position: methoxy (OMe), ethoxy (OEt), propoxy (OPr) or 2-phenylethoxy (OEtPh) groups.  The OMe group conferred general reactivity, whereas the OEtPh group conferred selective reactivity with chymase granzymes.  The inhibitors that contained the longest aminocaproyl (Aca) spacers between the biotin-tag and the isocoumarin ring mediated the most stable granzyme inactivation.  These inhibitors were the most effective at blocking lysis of red blood cells by the granule exts.  The inhibitors were used in protein blotting expts. where the biotin was detected with an avidin-enzyme complex.  Over 10 granzymes were labeled by the inhibitor Bi-Aca-Aca-IC-OMe.  The inhibitors detected granzymes when they were not readily detected by substrate hydrolysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfZi3CZ2KeK7Vg90H21EOLACvtfcHk0lgf2OCKKnzEhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XkvVOntLs%253D&md5=7dc2064eacd8abff24b5ac8d49b09817</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2F0161-5890%2896%2900025-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0161-5890%252896%252900025-9%26sid%3Dliteratum%253Aachs%26aulast%3DWinkler%26aufirst%3DU.%26aulast%3DAllison%26aufirst%3DN.%2BJ.%26aulast%3DWoodard%26aufirst%3DS.%2BL.%26aulast%3DGault%26aufirst%3DR.%2BA.%26aulast%3DEwoldt%26aufirst%3DG.%2BR.%26aulast%3DKam%26aufirst%3DC.%2BM.%26aulast%3DAbuelyaman%26aufirst%3DA.%26aulast%3DPowers%26aufirst%3DJ.%2BC.%26aulast%3DHudig%26aufirst%3DD.%26atitle%3DCharacterization%252C%2520Application%2520and%2520Potential%2520Uses%2520of%2520Biotin-Tagged%2520Inhibitors%2520for%2520Lymphocyte%2520Serine%2520Proteases%2520%2528granzymes%2529%26jtitle%3DMol.%2520Immunol.%26date%3D1996%26volume%3D33%26spage%3D615%26epage%3D623%26doi%3D10.1016%2F0161-5890%2896%2900025-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mahrus, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, C. S.</span></span> <span> </span><span class="NLM_article-title">Selective Chemical Functional Probes of Granzymes A and B Reveal Granzyme B is a Major Effector of Natural Killer Cell-Mediated Lysis of Target Cells</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">567</span>– <span class="NLM_lpage">577</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2005.03.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1016%2Fj.chembiol.2005.03.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=15911377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BD2MXks1aku7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2005&pages=567-577&author=S.+Mahrusauthor=C.+S.+Craik&title=Selective+Chemical+Functional+Probes+of+Granzymes+A+and+B+Reveal+Granzyme+B+is+a+Major+Effector+of+Natural+Killer+Cell-Mediated+Lysis+of+Target+Cells&doi=10.1016%2Fj.chembiol.2005.03.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Selective chemical functional probes of granzymes A and B reveal granzyme B Is a major effector of natural killer cell-mediated lysis of target cells</span></div><div class="casAuthors">Mahrus, Sami; Craik, Charles S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">567-577</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The mechanism of target cell lysis in cytotoxic lymphocyte-mediated death is not well understood, and the role of granzymes in this process is unclear.  Chem. functional probes were thus prepd. for the major granzymes A and B to deconvolute their role in natural killer cell-mediated lysis of target cells.  These biotinylated and substrate specificity-based di-Ph phosphonates allowed facile evaluation of selectivity through activity-based profiling in cell lysates and intact cells.  Both inhibitors were extremely selective in vitro and in cells.  Use of these inhibitors in cell-based assays revealed granzyme A to be a minor effector and granzyme B to be a major effector of target cell lysis by natural killer cells.  These studies indicate that the proapoptotic granzyme B functions also as a pronecrotic effector of target cell death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBTIufZM9xDrVg90H21EOLACvtfcHk0lgf2OCKKnzEhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXks1aku7Y%253D&md5=07dbb4ea5f6e2186a0ab880280247391</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2005.03.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2005.03.006%26sid%3Dliteratum%253Aachs%26aulast%3DMahrus%26aufirst%3DS.%26aulast%3DCraik%26aufirst%3DC.%2BS.%26atitle%3DSelective%2520Chemical%2520Functional%2520Probes%2520of%2520Granzymes%2520A%2520and%2520B%2520Reveal%2520Granzyme%2520B%2520is%2520a%2520Major%2520Effector%2520of%2520Natural%2520Killer%2520Cell-Mediated%2520Lysis%2520of%2520Target%2520Cells%26jtitle%3DChem.%2520Biol.%26date%3D2005%26volume%3D12%26spage%3D567%26epage%3D577%26doi%3D10.1016%2Fj.chembiol.2005.03.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buzza, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bird, P. I.</span></span> <span> </span><span class="NLM_article-title">Extracellular Granzymes: Current Perspectives</span>. <i>Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>387</i></span>,  <span class="NLM_fpage">827</span>– <span class="NLM_lpage">837</span>, <span class="refDoi"> DOI: 10.1515/BC.2006.106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1515%2FBC.2006.106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=16913832" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BD28XnvFanurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=387&publication_year=2006&pages=827-837&author=M.+S.+Buzzaauthor=P.+I.+Bird&title=Extracellular+Granzymes%3A+Current+Perspectives&doi=10.1515%2FBC.2006.106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Extracellular granzymes: current perspectives</span></div><div class="casAuthors">Buzza, Marguerite S.; Bird, Phillip I.</div><div class="citationInfo"><span class="NLM_cas:title">Biological Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">387</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">827-837</span>CODEN:
                <span class="NLM_cas:coden">BICHF3</span>;
        ISSN:<span class="NLM_cas:issn">1431-6730</span>.
    
            (<span class="NLM_cas:orgname">Walter de Gruyter GmbH & Co. KG</span>)
        </div><div class="casAbstract">A review.  Granzyme A (GrA) and granzyme B (GrB) play key roles in the induction of target cell death induced by cytotoxic lymphocytes.  While these roles have been extensively studied, it has become apparent that both granzymes also possess extracellular activities.  Sol. granzymes are found extracellularly in normal plasma and are elevated in a no. of diseases, ranging from viral and bacterial infections to autoimmune diseases.  Here, the authors discuss current knowledge of extracellular granzyme substrates, inhibitors, and functions, and the pathol. consequences of extracellular granzymes in disease.  In addn., the authors provide new evidence for the role of glycosaminoglycan-binding sites of granzymes in extracellular matrix remodeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkEnSXuKAZTLVg90H21EOLACvtfcHk0lgf2OCKKnzEhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XnvFanurY%253D&md5=fffabaab6d38fb53098a05ab8a53d851</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1515%2FBC.2006.106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252FBC.2006.106%26sid%3Dliteratum%253Aachs%26aulast%3DBuzza%26aufirst%3DM.%2BS.%26aulast%3DBird%26aufirst%3DP.%2BI.%26atitle%3DExtracellular%2520Granzymes%253A%2520Current%2520Perspectives%26jtitle%3DBiol.%2520Chem.%26date%3D2006%26volume%3D387%26spage%3D827%26epage%3D837%26doi%3D10.1515%2FBC.2006.106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresford, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novina, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshida, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pommier, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lieberman, J.</span></span> <span> </span><span class="NLM_article-title">Cleaving the Oxidative Repair Protein Ape1 Enhances Cell Death Mediated by Granzyme A</span>. <i>Nat. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">145</span>– <span class="NLM_lpage">153</span>, <span class="refDoi"> DOI: 10.1038/ni885</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1038%2Fni885" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=12524539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmslKisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2003&pages=145-153&author=Z.+Fanauthor=P.+J.+Beresfordauthor=D.+Zhangauthor=Z.+Xuauthor=C.+D.+Novinaauthor=A.+Yoshidaauthor=Y.+Pommierauthor=J.+Lieberman&title=Cleaving+the+Oxidative+Repair+Protein+Ape1+Enhances+Cell+Death+Mediated+by+Granzyme+A&doi=10.1038%2Fni885"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Cleaving the oxidative repair protein Ape1 enhances cell death mediated by granzyme A</span></div><div class="casAuthors">Fan, Zusen; Beresford, Paul J.; Zhang, Dong; Xu, Zhan; Novina, Carl D.; Yoshida, Akira; Pommier, Yves; Lieberman, Judy</div><div class="citationInfo"><span class="NLM_cas:title">Nature Immunology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">145-153</span>CODEN:
                <span class="NLM_cas:coden">NIAMCZ</span>;
        ISSN:<span class="NLM_cas:issn">1529-2908</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The cytolytic T lymphocyte protease granzyme A (GzmA) initiates a caspase-independent cell death pathway.  Here the authors report that the rate-limiting enzyme of DNA base excision repair, apurinic endonuclease-1 (Ape1), which is also known as redox factor-1 (Ref-1), or APEX, binds to GzmA and is contained in the SET complex, a macromol. complex of 270-420 kDa that is assocd. with the endoplasmic reticulum and is targeted by GzmA during cell-mediated death.  GzmA cleaves Ape1 after Lys31 and destroys its known oxidative repair functions.  In so doing, GzmA may block cellular repair and force apoptosis.  In support of this, cells with silenced Ape1 expression are more sensitive, whereas cells overexpressing noncleavable Ape1 are more resistant, to GzmA-mediated death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZLqGUNSZPx7Vg90H21EOLACvtfcHk0lha_go2sOwAkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmslKisw%253D%253D&md5=552ae476ca917c2ac5cb52b8ab094e9a</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fni885&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fni885%26sid%3Dliteratum%253Aachs%26aulast%3DFan%26aufirst%3DZ.%26aulast%3DBeresford%26aufirst%3DP.%2BJ.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DNovina%26aufirst%3DC.%2BD.%26aulast%3DYoshida%26aufirst%3DA.%26aulast%3DPommier%26aufirst%3DY.%26aulast%3DLieberman%26aufirst%3DJ.%26atitle%3DCleaving%2520the%2520Oxidative%2520Repair%2520Protein%2520Ape1%2520Enhances%2520Cell%2520Death%2520Mediated%2520by%2520Granzyme%2520A%26jtitle%3DNat.%2520Immunol.%26date%3D2003%26volume%3D4%26spage%3D145%26epage%3D153%26doi%3D10.1038%2Fni885" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresford, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lieberman, J.</span></span> <span> </span><span class="NLM_article-title">HMG2 Interacts with the Nucleosome Assembly Protein SET and is a Target of the Cytotoxic T-lymphocyte Protease Granzyme A</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">2810</span>– <span class="NLM_lpage">2820</span>, <span class="refDoi"> DOI: 10.1128/MCB.22.8.2810-2820.2002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1128%2FMCB.22.8.2810-2820.2002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=11909973" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BD38XislWnt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2002&pages=2810-2820&author=Z.+Fanauthor=P.+J.+Beresfordauthor=D.+Zhangauthor=J.+Lieberman&title=HMG2+Interacts+with+the+Nucleosome+Assembly+Protein+SET+and+is+a+Target+of+the+Cytotoxic+T-lymphocyte+Protease+Granzyme+A&doi=10.1128%2FMCB.22.8.2810-2820.2002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">HMG2 interacts with the nucleosome assembly protein SET and is a target of the cytotoxic T-lymphocyte protease granzyme A</span></div><div class="casAuthors">Fan, Zusen; Beresford, Paul J.; Zhang, Dong; Lieberman, Judy</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2810-2820</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The cytotoxic T-lymphocyte protease granzyme A induces caspase-independent cell death in which DNA single-stranded nicking is obsd. instead of oligonucleosomal fragmentation.  A 270- to 420-kDa endoplasmic reticulum-assocd. complex (SET complex) contg. the nucleosome assembly protein SET, the tumor suppressor pp32, and the base excision repair enzyme APE can induce single-stranded DNA damage in isolated nuclei in a granzyme A-dependent manner.  The normal functions of the SET complex are unknown, but the functions of its components suggest that it is involved in activating transcription and DNA repair.  The authors now find that the SET complex contains DNA binding and bending activities mediated by the chromatin-assocd. protein HMG2.  HMG2 facilitates assembly of nucleoprotein higher-order structures by bending and looping DNA or by stabilizing under-wound DNA.  HMG2 is in the SET complex and coppts. with SET.  By confocal microscopy, it is obsd. that cytoplasmic HMG2 colocalizes with SET in assocn. with the endoplasmic reticulum, but most nuclear HMG2 is unassocd. with SET.  This phys. assocn. suggests that HMG2 may facilitate the nucleosome assembly, transcriptional activation, and DNA repair functions of SET and/or APE.  HMG2, like SET and APE, is a physiol. relevant granzyme A substrate in targeted cells.  HMG1, however, is not a substrate.  Granzyme A cleavage after Lys65 in the midst of HMG box A destroys HMG2-mediated DNA binding and bending functions.  Granzyme A cleavage and functional disruption of key nuclear substrates, including HMG2, SET, APE, lamins, and histones, are likely to cripple the cellular repair response to promote cell death in this novel caspase-independent death pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxOx2JgQyVQrVg90H21EOLACvtfcHk0lha_go2sOwAkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XislWnt78%253D&md5=cd5f8b8e666b5b12f45bdce1887908b3</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1128%2FMCB.22.8.2810-2820.2002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.22.8.2810-2820.2002%26sid%3Dliteratum%253Aachs%26aulast%3DFan%26aufirst%3DZ.%26aulast%3DBeresford%26aufirst%3DP.%2BJ.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DLieberman%26aufirst%3DJ.%26atitle%3DHMG2%2520Interacts%2520with%2520the%2520Nucleosome%2520Assembly%2520Protein%2520SET%2520and%2520is%2520a%2520Target%2520of%2520the%2520Cytotoxic%2520T-lymphocyte%2520Protease%2520Granzyme%2520A%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2002%26volume%3D22%26spage%3D2810%26epage%3D2820%26doi%3D10.1128%2FMCB.22.8.2810-2820.2002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasternack, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresford, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenberg, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lieberman, J.</span></span> <span> </span><span class="NLM_article-title">Induction of Rapid Histone Degradation by the Cytotoxic T Lymphocyte Protease Granzyme A</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">3683</span>– <span class="NLM_lpage">3690</span>, <span class="refDoi"> DOI: 10.1074/jbc.M005390200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1074%2Fjbc.M005390200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=11060286" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhtFWitrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2001&pages=3683-3690&author=D.+Zhangauthor=M.+S.+Pasternackauthor=P.+J.+Beresfordauthor=L.+Wagnerauthor=A.+H.+Greenbergauthor=J.+Lieberman&title=Induction+of+Rapid+Histone+Degradation+by+the+Cytotoxic+T+Lymphocyte+Protease+Granzyme+A&doi=10.1074%2Fjbc.M005390200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Induction of rapid histone degradation by the cytotoxic T lymphocyte protease granzyme A</span></div><div class="casAuthors">Zhang, Dong; Pasternack, Mark S.; Beresford, Paul J.; Wagner, Ludwig; Greenberg, Arnold H.; Lieberman, Judy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">3683-3690</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The cytotoxic T lymphocyte protease granzyme A induces caspase-independent cell death in which DNA single-strand nicking is obsd. instead of oligonucleosomal fragmentation.  Granzyme A is a specific tryptase that concs. in the nucleus of targeted cells and synergistically enhances DNA fragmentation induced by the caspase activator granzyme B.  Here we show that granzyme A treatment of isolated nuclei enhances DNA accessibility to exogenous endonucleases.  In vitro and after cell loading with perforin, GrnA completely degrades histone H1 and cleaves core histones into ∼16-kDa fragments.  Histone digestion provides a mechanism for unfolding compacted chromatin and facilitating endogenous DNase access to DNA during T cell and natural killer cell granule-mediated apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKVCFhmAR0cbVg90H21EOLACvtfcHk0lha_go2sOwAkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhtFWitrw%253D&md5=ca2b82889f1ab1ed9b85ac48e6e15eaf</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M005390200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M005390200%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DPasternack%26aufirst%3DM.%2BS.%26aulast%3DBeresford%26aufirst%3DP.%2BJ.%26aulast%3DWagner%26aufirst%3DL.%26aulast%3DGreenberg%26aufirst%3DA.%2BH.%26aulast%3DLieberman%26aufirst%3DJ.%26atitle%3DInduction%2520of%2520Rapid%2520Histone%2520Degradation%2520by%2520the%2520Cytotoxic%2520T%2520Lymphocyte%2520Protease%2520Granzyme%2520A%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2001%26volume%3D276%26spage%3D3683%26epage%3D3690%26doi%3D10.1074%2Fjbc.M005390200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresford, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenberg, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lieberman, J.</span></span> <span> </span><span class="NLM_article-title">Granzymes A and B Directly Cleave Lamins and Disrupt the Nuclear Lamina During Granule-Mediated Cytolysis</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">5746</span>– <span class="NLM_lpage">5751</span>, <span class="refDoi"> DOI: 10.1073/pnas.101329598</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1073%2Fpnas.101329598" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=11331782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjs1Wnsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=5746-5751&author=D.+Zhangauthor=P.+J.+Beresfordauthor=A.+H.+Greenbergauthor=J.+Lieberman&title=Granzymes+A+and+B+Directly+Cleave+Lamins+and+Disrupt+the+Nuclear+Lamina+During+Granule-Mediated+Cytolysis&doi=10.1073%2Fpnas.101329598"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Granzymes A and B directly cleave lamins and disrupt the nuclear lamina during granule-mediated cytolysis</span></div><div class="casAuthors">Zhang, Dong; Beresford, Paul J.; Greenberg, Arnold H.; Lieberman, Judy</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5746-5751</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Cytotoxic T lymphocytes (CTL) induce apoptosis by engaging death receptors or by exocytosis of cytolytic granules contg. granzyme (Gzm) proteases and perforin.  The lamins, which maintain the structural integrity of the nuclear envelope, are cleaved by caspases during caspase-mediated apoptosis.  Although death receptor engagement and GzmB activate caspases, CTL also induce apoptosis during caspase blockade.  Both GzmA and GzmB directly and efficiently cleave lamin B in vitro, in situ in isolated nuclei and in cells loaded with perforin and Gzms, even in the presence of caspase inhibitors.  Lamin-B is cleaved by GzmA at concns. of 3 nM, but GzmB is 50 times less active.  GzmA cuts lamin B at R392; GzmB cuts at the caspase VEVD231 site.  Characteristic lamin B fragments generated by Gzm proteolysis also are obsd. during CTL lysis, even in the presence of caspase inhibitors or in cells overexpressing bcl-2.  Lamins A/C are direct substrates of GzmA, but not GzmB.  GzmA and GzmB therefore directly target crit. caspase substrates in caspase-resistant cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGyCLT712Qj7Vg90H21EOLACvtfcHk0lgFc71YLyb_-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjs1Wnsbo%253D&md5=21fdb3a99cbfb762955a2f30583d4ccd</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1073%2Fpnas.101329598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.101329598%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DBeresford%26aufirst%3DP.%2BJ.%26aulast%3DGreenberg%26aufirst%3DA.%2BH.%26aulast%3DLieberman%26aufirst%3DJ.%26atitle%3DGranzymes%2520A%2520and%2520B%2520Directly%2520Cleave%2520Lamins%2520and%2520Disrupt%2520the%2520Nuclear%2520Lamina%2520During%2520Granule-Mediated%2520Cytolysis%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2001%26volume%3D98%26spage%3D5746%26epage%3D5751%26doi%3D10.1073%2Fpnas.101329598" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresford, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lieberman, J.</span></span> <span> </span><span class="NLM_article-title">Tumor Suppressor NM23-H1 is a Granzyme A-activated DNase During CTL-mediated Apoptosis, and the Nucleosome Assembly Protein SET is its Inhibitor</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">659</span>– <span class="NLM_lpage">672</span>, <span class="refDoi"> DOI: 10.1016/S0092-8674(03)00150-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1016%2FS0092-8674%2803%2900150-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=12628186" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BD3sXitFykurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2003&pages=659-672&author=Z.+Fanauthor=P.+J.+Beresfordauthor=D.+Y.+Ohauthor=D.+Zhangauthor=J.+Lieberman&title=Tumor+Suppressor+NM23-H1+is+a+Granzyme+A-activated+DNase+During+CTL-mediated+Apoptosis%2C+and+the+Nucleosome+Assembly+Protein+SET+is+its+Inhibitor&doi=10.1016%2FS0092-8674%2803%2900150-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Tumor suppressor NM23-H1 is a granzyme A-activated dnase during CTL-mediated apoptosis, and the nucleosome assembly protein SET is its inhibitor</span></div><div class="casAuthors">Fan, Zusen; Beresford, Paul J.; Oh, David Y.; Zhang, Dong; Lieberman, Judy</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">659-672</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Granzyme A (GzmA) induces a caspase-independent cell death pathway characterized by single-stranded DNA nicks and other features of apoptosis.  A GzmA-activated DNase (GAAD) is in an ER assocd. complex contg. pp32 and the GzmA substrates SET, HMG-2, and Ape1.  The authors show that GAAD is NM23-H1, a nucleoside diphosphate kinase implicated in suppression of tumor metastasis, and its specific inhibitor (IGAAD) is SET.  NM23-H1 binds to SET and is released from inhibition by GzmA cleavage of SET.  After GzmA loading or CTL attack, SET and NM23-H1 translocate to the nucleus and SET is degraded, allowing NM23-H1 to nick chromosomal DNA.  GzmA-treated cells with silenced NM23-H1 expression are resistant to GzmA-mediated DNA damage and cytolysis, while cells over-expressing NM23-H1 are more sensitive.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovlpoLT-sBjbVg90H21EOLACvtfcHk0lgFc71YLyb_-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXitFykurc%253D&md5=f3bbb5b994d3a27eb9d2864d33d55f55</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2803%2900150-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252803%252900150-8%26sid%3Dliteratum%253Aachs%26aulast%3DFan%26aufirst%3DZ.%26aulast%3DBeresford%26aufirst%3DP.%2BJ.%26aulast%3DOh%26aufirst%3DD.%2BY.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DLieberman%26aufirst%3DJ.%26atitle%3DTumor%2520Suppressor%2520NM23-H1%2520is%2520a%2520Granzyme%2520A-activated%2520DNase%2520During%2520CTL-mediated%2520Apoptosis%252C%2520and%2520the%2520Nucleosome%2520Assembly%2520Protein%2520SET%2520is%2520its%2520Inhibitor%26jtitle%3DCell%26date%3D2003%26volume%3D112%26spage%3D659%26epage%3D672%26doi%3D10.1016%2FS0092-8674%2803%2900150-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chowdhury, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beresford, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demple, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrino, F. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lieberman, J.</span></span> <span> </span><span class="NLM_article-title">The Exonuclease TREX1 is in the SET Complex and Acts in Concert with NM23-H1 to Degrade DNA during Granzyme A-mediated Cell Death</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">142</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2006.06.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1016%2Fj.molcel.2006.06.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=16818237" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BD28XnsVyjs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2006&pages=133-142&author=D.+Chowdhuryauthor=P.+J.+Beresfordauthor=P.+Zhuauthor=D.+Zhangauthor=J.+S.+Sungauthor=B.+Dempleauthor=F.+W.+Perrinoauthor=J.+Lieberman&title=The+Exonuclease+TREX1+is+in+the+SET+Complex+and+Acts+in+Concert+with+NM23-H1+to+Degrade+DNA+during+Granzyme+A-mediated+Cell+Death&doi=10.1016%2Fj.molcel.2006.06.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">The exonuclease TREX1 is in the SET complex and acts in concert with NM23-H1 to degrade DNA during granzyme A-mediated cell death</span></div><div class="casAuthors">Chowdhury, Dipanjan; Beresford, Paul J.; Zhu, Pengcheng; Zhang, Dong; Sung, Jung-Suk; Demple, Bruce; Perrino, Fred W.; Lieberman, Judy</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">133-142</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Granzyme A (GzmA) activates a caspase-independent cell death pathway with morphol. features of apoptosis.  Single-stranded DNA damage is initiated when the endonuclease NM23-H1 becomes activated to nick DNA after granzyme A cleaves its inhibitor, SET.  SET and NM23-H1 reside in an endoplasmic reticulum-assocd. complex (the SET complex) that translocates to the nucleus in response to superoxide generation by granzyme A.  We now find the 3'-to-5' exonuclease TREX1, but not its close homolog TREX2, in the SET complex.  TREX1 binds to SET and colocalizes and translocates with the SET complex.  NM23-H1 and TREX1 work in concert to degrade DNA.  Silencing NM23-H1 or TREX1 inhibits DNA damage and death of cells treated with perforin (PFN) and granzyme A, but not of cells treated with perforin and granzyme B (GzmB).  After granzyme A activates NM23-H1 to make single-stranded nicks, TREX1 removes nucleotides from the nicked 3' end to reduce the possibility of repair by rejoining the nicked ends.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrse4Cx7pIswrVg90H21EOLACvtfcHk0lh29ZFBV8y7fw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XnsVyjs7s%253D&md5=06fe4768669c11d53537668777960eec</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2006.06.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2006.06.005%26sid%3Dliteratum%253Aachs%26aulast%3DChowdhury%26aufirst%3DD.%26aulast%3DBeresford%26aufirst%3DP.%2BJ.%26aulast%3DZhu%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DSung%26aufirst%3DJ.%2BS.%26aulast%3DDemple%26aufirst%3DB.%26aulast%3DPerrino%26aufirst%3DF.%2BW.%26aulast%3DLieberman%26aufirst%3DJ.%26atitle%3DThe%2520Exonuclease%2520TREX1%2520is%2520in%2520the%2520SET%2520Complex%2520and%2520Acts%2520in%2520Concert%2520with%2520NM23-H1%2520to%2520Degrade%2520DNA%2520during%2520Granzyme%2520A-mediated%2520Cell%2520Death%26jtitle%3DMol.%2520Cell%26date%3D2006%26volume%3D23%26spage%3D133%26epage%3D142%26doi%3D10.1016%2Fj.molcel.2006.06.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bell, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goetz, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahrus, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletterick, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, C. S.</span></span> <span> </span><span class="NLM_article-title">The Oligomeric Structure of Human Granzyme A is a Determinant of its Extended Substrate Specificity</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">527</span>– <span class="NLM_lpage">534</span>, <span class="refDoi"> DOI: 10.1038/nsb944</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1038%2Fnsb944" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2003&pages=527-534&author=J.+K.+Bellauthor=D.+H.+Goetzauthor=S.+Mahrusauthor=J.+L.+Harrisauthor=R.+J.+Fletterickauthor=C.+S.+Craik&title=The+Oligomeric+Structure+of+Human+Granzyme+A+is+a+Determinant+of+its+Extended+Substrate+Specificity&doi=10.1038%2Fnsb944"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnsb944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsb944%26sid%3Dliteratum%253Aachs%26aulast%3DBell%26aufirst%3DJ.%2BK.%26aulast%3DGoetz%26aufirst%3DD.%2BH.%26aulast%3DMahrus%26aufirst%3DS.%26aulast%3DHarris%26aufirst%3DJ.%2BL.%26aulast%3DFletterick%26aufirst%3DR.%2BJ.%26aulast%3DCraik%26aufirst%3DC.%2BS.%26atitle%3DThe%2520Oligomeric%2520Structure%2520of%2520Human%2520Granzyme%2520A%2520is%2520a%2520Determinant%2520of%2520its%2520Extended%2520Substrate%2520Specificity%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2003%26volume%3D10%26spage%3D527%26epage%3D534%26doi%3D10.1038%2Fnsb944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kasperkiewicz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poreba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snipas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirchhofer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvesen, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span> <span> </span><span class="NLM_article-title">Design of a Selective Substrate and Activity Based Probe for Human Neutrophil Serine Protease 4</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">e0132818</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0132818</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1371%2Fjournal.pone.0132818" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=26172376" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVSrtLnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=e0132818&author=P.+Kasperkiewiczauthor=M.+Porebaauthor=S.+J.+Snipasauthor=S.+J.+Linauthor=D.+Kirchhoferauthor=G.+S.+Salvesenauthor=M.+Drag&title=Design+of+a+Selective+Substrate+and+Activity+Based+Probe+for+Human+Neutrophil+Serine+Protease+4&doi=10.1371%2Fjournal.pone.0132818"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Design of a selective substrate and activity based probe for human neutrophil serine protease 4</span></div><div class="casAuthors">Kasperkiewicz, Paulina; Poreba, Marcin; Snipas, Scott J.; Jack, Lin S.; Kirchhofer, Daniel; Salvesen, Guy S.; Drag, Marcin</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e0132818/1-e0132818/12</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Human neutrophil serine protease 4 (NSP4), also known as PRSS57, is a recently discovered fourth member of the neutrophil serine proteases family.  Although its biol. function is not precisely defined, it is suggested to regulate neutrophil response and innate immune reactions.  To create optimal substrates and visualization probes for NSP4 that distinguish it from other NSPs we have employed a Hybrid Combinatorial Substrate Library approach that utilizes natural and unnatural amino acids to explore protease subsite preferences.  Library results were validated by synthesizing individual substrates, leading to the identification of an optimal substrate peptide.  This substrate was converted to a covalent di-Ph phosphonate probe with an embedded biotin tag.  This probe demonstrated high inhibitory activity and stringent specificity and may be suitable for visualizing NSP4 in the background of other NSPs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7Dosrmw75eLVg90H21EOLACvtfcHk0lh29ZFBV8y7fw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVSrtLnE&md5=b2323901d336008609b238a1bc344eca</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0132818&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0132818%26sid%3Dliteratum%253Aachs%26aulast%3DKasperkiewicz%26aufirst%3DP.%26aulast%3DPoreba%26aufirst%3DM.%26aulast%3DSnipas%26aufirst%3DS.%2BJ.%26aulast%3DLin%26aufirst%3DS.%2BJ.%26aulast%3DKirchhofer%26aufirst%3DD.%26aulast%3DSalvesen%26aufirst%3DG.%2BS.%26aulast%3DDrag%26aufirst%3DM.%26atitle%3DDesign%2520of%2520a%2520Selective%2520Substrate%2520and%2520Activity%2520Based%2520Probe%2520for%2520Human%2520Neutrophil%2520Serine%2520Protease%25204%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26spage%3De0132818%26doi%3D10.1371%2Fjournal.pone.0132818" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Poe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biddison, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blake, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodkey, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigal, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zweerink, H. J.</span></span> <span> </span><span class="NLM_article-title">Human Cytotoxic Lymphocyte Tryptase. Its Purification from Granules and the Characterization of Inhibitor and Substrate Specificity</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>263</i></span>,  <span class="NLM_fpage">13215</span>– <span class="NLM_lpage">13222</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=3047119" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADyaL1MXjtlOq" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=263&publication_year=1988&pages=13215-13222&author=M.+Poeauthor=C.+D.+Bennettauthor=W.+E.+Biddisonauthor=J.+T.+Blakeauthor=G.+P.+Nortonauthor=J.+A.+Rodkeyauthor=N.+H.+Sigalauthor=R.+V.+Turnerauthor=J.+K.+Wuauthor=H.+J.+Zweerink&title=Human+Cytotoxic+Lymphocyte+Tryptase.+Its+Purification+from+Granules+and+the+Characterization+of+Inhibitor+and+Substrate+Specificity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Human cytotoxic lymphocyte tryptase.  Its purification from granules and the characterization of inhibitor and substrate specificity</span></div><div class="casAuthors">Poe, Martin; Bennett, Carl D.; Biddison, William E.; Blake, Joseph T.; Norton, Gerard P.; Rodkey, John A.; Sigal, Nolan H.; Turner, Richard V.; Wu, Joseph K.; Zweerink, Hans J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">263</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">13215-22</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">A trypsin-like enzyme (tryptase) (I) was purified to homogeneity from the granules of a human cytolytic lymphocyte (CTL) line, Q31, by a 3-step procedure.  By including 0.3% Triton X-100 and 1 mg/mL heparin in purifn. buffers, near total yields of I activity were obtained during the purifn.  I migrated in SDS-PAGE at a position corresponding to 28 kilodaltons with and to 45 kilodaltons without 2-mercaptoethanol.  It had an N-terminal sequence identical to a previously reported for human CTL I at 20 of 22 positions identified.  I hydrolyzed Nα-carbobenzoxy-L-lysine thiobenzyl ester (BLT), and this BLT esterase activity was most efficient at slightly alk. pH and was relatively more active near neutral pH than mouse CTL I.  Human α1-protease inhibitor, human antithrombin III, phenylmethanesulfonyl fluoride, and p-aminobenzamidine inhibited Q31 I.  The inhibition by human antithrombin III was rapid enough to be of physiol. significance.  A survey of oligopeptide p-nitroanilides found that the best substrate for human Q31 I was H-D-(ε-carbobenzyloxy)Lys-L-Pro-L-Arg-p-nitroanilide.  Q31 I appeared to have broad specificity for amino acid residues at P2 and P3, i.e., at 2 and 3 residues N-terminal to the scissile bond.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD3Ki-3s0xV7Vg90H21EOLACvtfcHk0lh29ZFBV8y7fw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXjtlOq&md5=3bbbbcac76ef4a5416a3cca6a58ee304</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPoe%26aufirst%3DM.%26aulast%3DBennett%26aufirst%3DC.%2BD.%26aulast%3DBiddison%26aufirst%3DW.%2BE.%26aulast%3DBlake%26aufirst%3DJ.%2BT.%26aulast%3DNorton%26aufirst%3DG.%2BP.%26aulast%3DRodkey%26aufirst%3DJ.%2BA.%26aulast%3DSigal%26aufirst%3DN.%2BH.%26aulast%3DTurner%26aufirst%3DR.%2BV.%26aulast%3DWu%26aufirst%3DJ.%2BK.%26aulast%3DZweerink%26aufirst%3DH.%2BJ.%26atitle%3DHuman%2520Cytotoxic%2520Lymphocyte%2520Tryptase.%2520Its%2520Purification%2520from%2520Granules%2520and%2520the%2520Characterization%2520of%2520Inhibitor%2520and%2520Substrate%2520Specificity%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1988%26volume%3D263%26spage%3D13215%26epage%3D13222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ng, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pike, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, S. E.</span></span> <span> </span><span class="NLM_article-title">Subsite Cooperativity in Protease Specificity</span>. <i>Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>390</i></span>,  <span class="NLM_fpage">401</span>– <span class="NLM_lpage">407</span>, <span class="refDoi"> DOI: 10.1515/BC.2009.065</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1515%2FBC.2009.065" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=19361286" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmvVWqtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=390&publication_year=2009&pages=401-407&author=N.+M.+Ngauthor=R.+N.+Pikeauthor=S.+E.+Boyd&title=Subsite+Cooperativity+in+Protease+Specificity&doi=10.1515%2FBC.2009.065"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Subsite cooperativity in protease specificity</span></div><div class="casAuthors">Ng, Natasha M.; Pike, Robert N.; Boyd, Sarah E.</div><div class="citationInfo"><span class="NLM_cas:title">Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">390</span>
        (<span class="NLM_cas:issue">5/6</span>),
    <span class="NLM_cas:pages">401-407</span>CODEN:
                <span class="NLM_cas:coden">BICHF3</span>;
        ISSN:<span class="NLM_cas:issn">1431-6730</span>.
    
            (<span class="NLM_cas:orgname">Walter de Gruyter GmbH & Co. KG</span>)
        </div><div class="casAbstract">A review.  Proteases play vital roles in a range of biol. processes, such as cell cycle, cell growth and differentiation, apoptosis, hemostasis and signaling.  Fundamental to our knowledge of protease action is an understanding of how the active site operates; this has been examd. through extensive studies of the substrate specificity of the enzymes.  Kinetic and structural analyses have shown that the binding of a particular substrate residue at a protease subsite can have either a pos. or neg. influence on the binding of particular residues at other subsites.  This phenomenon has been termed subsite cooperativity and has been obsd. in a wide range of proteases, often between non-adjacent subsites.  This review aims to highlight studies where subsite cooperativity has been obsd., exptl. techniques used in the past and potential methods that can be employed to comprehensively examine this phenomenon.  Further understanding of how the protease active site recognizes and chooses its substrates for cleavage will have a significant impact on the development of pharmaceuticals that target these enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK8MrK5j6zybVg90H21EOLACvtfcHk0lgQOjPJMvClJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmvVWqtro%253D&md5=68b6d056b3e5da0255703f2360d3d6d5</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1515%2FBC.2009.065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252FBC.2009.065%26sid%3Dliteratum%253Aachs%26aulast%3DNg%26aufirst%3DN.%2BM.%26aulast%3DPike%26aufirst%3DR.%2BN.%26aulast%3DBoyd%26aufirst%3DS.%2BE.%26atitle%3DSubsite%2520Cooperativity%2520in%2520Protease%2520Specificity%26jtitle%3DBiol.%2520Chem.%26date%3D2009%26volume%3D390%26spage%3D401%26epage%3D407%26doi%3D10.1515%2FBC.2009.065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kasperkiewicz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altman, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Angelo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvesen, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span> <span> </span><span class="NLM_article-title">Toolbox of Fluorescent Probes for Parallel Imaging Reveals Uneven Location of Serine Proteases in Neutrophils</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">10115</span>– <span class="NLM_lpage">10125</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b04394</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b04394" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtV2ru7zM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=10115-10125&author=P.+Kasperkiewiczauthor=Y.+Altmanauthor=M.+D%E2%80%99Angeloauthor=G.+S.+Salvesenauthor=M.+Drag&title=Toolbox+of+Fluorescent+Probes+for+Parallel+Imaging+Reveals+Uneven+Location+of+Serine+Proteases+in+Neutrophils&doi=10.1021%2Fjacs.7b04394"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Toolbox of Fluorescent Probes for Parallel Imaging Reveals Uneven Location of Serine Proteases in Neutrophils</span></div><div class="casAuthors">Kasperkiewicz, Paulina; Altman, Yoav; D'Angelo, Maximiliano; Salvesen, Guy S.; Drag, Marcin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">10115-10125</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Neutrophils, the front line defenders against infection, express four serine proteases (NSPs) that play roles in the control of cell-signaling pathways and defense against pathogens and whose imbalance leads to pathol. conditions.  Dissecting the roles of individual NSPs in humans is problematic because neutrophils are end-stage cells with a short half-life and minimal ongoing protein synthesis.  To gain insight into the regulation of NSP activity we have generated a small-mol. chem. toolbox consisting of activity-based probes with different fluorophore-detecting groups with minimal wavelength overlap and highly selective natural and unnatural amino acid recognition sequences.  The key feature of these activity-based probes is the ability to use them for simultaneous observation and detection of all four individual NSPs by fluorescence microscopy, a feature never achieved in previous studies.  Using these probes we demonstrate uneven distribution of NSPs in neutrophil azurophil granules, such that they seem to be mutually excluded from each other, suggesting the existence of unknown granule-targeting mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCQyFRKT-WtrVg90H21EOLACvtfcHk0lgQOjPJMvClJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtV2ru7zM&md5=72b3e9e59a26a034fa66aa9cbd714b61</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b04394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b04394%26sid%3Dliteratum%253Aachs%26aulast%3DKasperkiewicz%26aufirst%3DP.%26aulast%3DAltman%26aufirst%3DY.%26aulast%3DD%25E2%2580%2599Angelo%26aufirst%3DM.%26aulast%3DSalvesen%26aufirst%3DG.%2BS.%26aulast%3DDrag%26aufirst%3DM.%26atitle%3DToolbox%2520of%2520Fluorescent%2520Probes%2520for%2520Parallel%2520Imaging%2520Reveals%2520Uneven%2520Location%2520of%2520Serine%2520Proteases%2520in%2520Neutrophils%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D10115%26epage%3D10125%26doi%3D10.1021%2Fjacs.7b04394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hink-Schauer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estebanez-Perpina, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurschus, F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bode, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenne, D. E.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of the Apoptosis-Inducing Human Granzyme A Dimer</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">535</span>– <span class="NLM_lpage">540</span>, <span class="refDoi"> DOI: 10.1038/nsb945</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1038%2Fnsb945" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2003&pages=535-540&author=C.+Hink-Schauerauthor=E.+Estebanez-Perpinaauthor=F.+C.+Kurschusauthor=W.+Bodeauthor=D.+E.+Jenne&title=Crystal+Structure+of+the+Apoptosis-Inducing+Human+Granzyme+A+Dimer&doi=10.1038%2Fnsb945"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnsb945&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsb945%26sid%3Dliteratum%253Aachs%26aulast%3DHink-Schauer%26aufirst%3DC.%26aulast%3DEstebanez-Perpina%26aufirst%3DE.%26aulast%3DKurschus%26aufirst%3DF.%2BC.%26aulast%3DBode%26aufirst%3DW.%26aulast%3DJenne%26aufirst%3DD.%2BE.%26atitle%3DCrystal%2520Structure%2520of%2520the%2520Apoptosis-Inducing%2520Human%2520Granzyme%2520A%2520Dimer%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2003%26volume%3D10%26spage%3D535%26epage%3D540%26doi%3D10.1038%2Fnsb945" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krahenbuhl, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rey, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenne, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanzavecchia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groscurth, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tschopp, J.</span></span> <span> </span><span class="NLM_article-title">Characterization of Granzymes A and B Isolated from Granules of Cloned Human Cytotoxic T Lymphocytes</span>. <i>J. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>141</i></span>,  <span class="NLM_fpage">3471</span>– <span class="NLM_lpage">3477</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=3263427" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A280%3ADyaL1M%252FjsFClug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=1988&pages=3471-3477&author=O.+Krahenbuhlauthor=C.+Reyauthor=D.+Jenneauthor=A.+Lanzavecchiaauthor=P.+Groscurthauthor=S.+Carrelauthor=J.+Tschopp&title=Characterization+of+Granzymes+A+and+B+Isolated+from+Granules+of+Cloned+Human+Cytotoxic+T+Lymphocytes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of granzymes A and B isolated from granules of cloned human cytotoxic T lymphocytes</span></div><div class="casAuthors">Krahenbuhl O; Rey C; Jenne D; Lanzavecchia A; Groscurth P; Carrel S; Tschopp J</div><div class="citationInfo"><span class="NLM_cas:title">Journal of immunology (Baltimore, Md. : 1950)</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">141</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3471-7</span>
        ISSN:<span class="NLM_cas:issn">0022-1767</span>.
    </div><div class="casAbstract">A human CD8+ CTL clone with cytolytic potential was shown to express two serine proteases, a 50-kDa homodimer and a 27-kDa monomer, which were purified from cytoplasmic granules.  N-terminal sequencing of the purified proteins revealed that the 50-kDa homodimer is the gene product of the human Hanukah factor cDNA clone and that it represents the human homologue to granzyme A.  Similarly, the 27-kDa protein was shown to be the serine esterase encoded by the human lymphocyte protease cDNA clone and corresponds to granzyme B.  There was no evidence for the presence of other granzymes, in particular for the human homologues to murine granzymes C, D, E, and F.  The substrate best cleaved by granzyme A was Gly-Pro-Arg-amido-4-methyl-coumarin after the Arg residue and the pH optimum was 8 to 8.5.  Upon triggering of the TCR-CD3 complex with an anti-CD3 mAb, granzyme A was released into the culture medium.  Furthermore, a granule-associated hemolytic activity was detected after salt extraction and partial purification of granule proteins.  This suggests that hemolytically active human perforin can be obtained from inactive granules.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSndU-TK3UV58jl5L8hwNX0fW6udTcc2eYRk02T_MXRzbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaL1M%252FjsFClug%253D%253D&md5=fc03478f25e9e0714a0282cc822f0f11</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKrahenbuhl%26aufirst%3DO.%26aulast%3DRey%26aufirst%3DC.%26aulast%3DJenne%26aufirst%3DD.%26aulast%3DLanzavecchia%26aufirst%3DA.%26aulast%3DGroscurth%26aufirst%3DP.%26aulast%3DCarrel%26aufirst%3DS.%26aulast%3DTschopp%26aufirst%3DJ.%26atitle%3DCharacterization%2520of%2520Granzymes%2520A%2520and%2520B%2520Isolated%2520from%2520Granules%2520of%2520Cloned%2520Human%2520Cytotoxic%2520T%2520Lymphocytes%26jtitle%3DJ.%2520Immunol.%26date%3D1988%26volume%3D141%26spage%3D3471%26epage%3D3477" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lieberman, J.</span></span> <span> </span><span class="NLM_article-title">Granzyme A Activates Another Way to Die</span>. <i>Immunol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>235</i></span>,  <span class="NLM_fpage">93</span>– <span class="NLM_lpage">104</span>, <span class="refDoi"> DOI: 10.1111/j.0105-2896.2010.00902.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1111%2Fj.0105-2896.2010.00902.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=20536557" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnt1Kjsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=235&publication_year=2010&pages=93-104&author=J.+Lieberman&title=Granzyme+A+Activates+Another+Way+to+Die&doi=10.1111%2Fj.0105-2896.2010.00902.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Granzyme A activates another way to die</span></div><div class="casAuthors">Lieberman, Judy</div><div class="citationInfo"><span class="NLM_cas:title">Immunological Reviews</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">235</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">93-104</span>CODEN:
                <span class="NLM_cas:coden">IMRED2</span>;
        ISSN:<span class="NLM_cas:issn">1600-065X</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Granzyme A (GzmA) is the most abundant serine protease in killer cell cytotoxic granules.  GzmA activates a novel programed cell death pathway that begins in the mitochondrion, where cleavage of NDUFS3 in electron transport complex 1 disrupts mitochondrial metab. and generates reactive oxygen species (ROS).  ROS drives the endoplasmic reticulum-assocd. SET complex into the nucleus, where it activates single-stranded DNA damage.  GzmA also targets other important nuclear proteins for degrdn., including histones, the lamins that maintain the nuclear envelope, and several key DNA damage repair proteins (Ku70, PARP-1).  Cells that are resistant to the caspases or GzmB by overexpressing bcl-2 family anti-apoptotic proteins or caspase or GzmB protease inhibitors are sensitive to GzmA.  By activating multiple cell death pathways, killer cells provide better protection against a variety of intracellular pathogens and tumors.  GzmA also has proinflammatory activity; it activates pro-interleukin-1β and may also have other proinflammatory effects that remain to be elucidated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrE7K84--EYWbVg90H21EOLACvtfcHk0lil3U0IS5nVSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnt1Kjsbc%253D&md5=75a9b3e75d7d20bc50f9b85dd31aec38</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1111%2Fj.0105-2896.2010.00902.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.0105-2896.2010.00902.x%26sid%3Dliteratum%253Aachs%26aulast%3DLieberman%26aufirst%3DJ.%26atitle%3DGranzyme%2520A%2520Activates%2520Another%2520Way%2520to%2520Die%26jtitle%3DImmunol.%2520Rev.%26date%3D2010%26volume%3D235%26spage%3D93%26epage%3D104%26doi%3D10.1111%2Fj.0105-2896.2010.00902.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kasperkiewicz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poreba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snipas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winterbourn, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvesen, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span> <span> </span><span class="NLM_article-title">Design of Ultrasensitive Probes for Human Neutrophil Elastase Through Hybrid Combinatorial Substrate Library Profiling</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">2518</span>– <span class="NLM_lpage">2523</span>, <span class="refDoi"> DOI: 10.1073/pnas.1318548111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1073%2Fpnas.1318548111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=24550277" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BC2cXivFaht7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=2518-2523&author=P.+Kasperkiewiczauthor=M.+Porebaauthor=S.+J.+Snipasauthor=H.+Parkerauthor=C.+C.+Winterbournauthor=G.+S.+Salvesenauthor=M.+Drag&title=Design+of+Ultrasensitive+Probes+for+Human+Neutrophil+Elastase+Through+Hybrid+Combinatorial+Substrate+Library+Profiling&doi=10.1073%2Fpnas.1318548111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Design of ultrasensitive probes for human neutrophil elastase through hybrid combinatorial substrate library profiling</span></div><div class="casAuthors">Kasperkiewicz, Paulina; Poreba, Marcin; Snipas, Scott J.; Parker, Heather; Winterbourn, Christine C.; Salvesen, Guy S.; Drag, Marcin</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2518-2523</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The exploration of protease substrate specificity is generally restricted to naturally occurring amino acids, limiting the degree of conformational space that can be surveyed.  We substantially enhanced this by incorporating 102 unnatural amino acids to explore the S1-S4 pockets of human neutrophil elastase (NE).  This approach provides hybrid natural and unnatural amino acid sequences, and thus we termed it the Hybrid Combinatorial Substrate Library (HyCoSuL).  Library results were validated by the synthesis of individual tetrapeptide substrates, with the optimal substrate demonstrating more than three orders of magnitude higher catalytic efficiency than commonly used substrates of elastase.  This optimal substrate was converted to an activity-based probe that demonstrated high selectivity and revealed the specific presence of active elastase during the process of neutrophil extracellular trap formation.  We propose that this approach can be successfully used for any type of endopeptidase to deliver high activity and selectivity in substrates and probes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMwdujTniBZbVg90H21EOLACvtfcHk0lil3U0IS5nVSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXivFaht7o%253D&md5=048b0dd58455cd95a3def76ee8942337</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1318548111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1318548111%26sid%3Dliteratum%253Aachs%26aulast%3DKasperkiewicz%26aufirst%3DP.%26aulast%3DPoreba%26aufirst%3DM.%26aulast%3DSnipas%26aufirst%3DS.%2BJ.%26aulast%3DParker%26aufirst%3DH.%26aulast%3DWinterbourn%26aufirst%3DC.%2BC.%26aulast%3DSalvesen%26aufirst%3DG.%2BS.%26aulast%3DDrag%26aufirst%3DM.%26atitle%3DDesign%2520of%2520Ultrasensitive%2520Probes%2520for%2520Human%2520Neutrophil%2520Elastase%2520Through%2520Hybrid%2520Combinatorial%2520Substrate%2520Library%2520Profiling%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D2518%26epage%3D2523%26doi%3D10.1073%2Fpnas.1318548111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Poreba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvesen, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span> <span> </span><span class="NLM_article-title">Synthesis of a HyCoSuL Peptide Substrate Library to Dissect Protease Substrate Specificity</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2189</span>– <span class="NLM_lpage">2214</span>, <span class="refDoi"> DOI: 10.1038/nprot.2017.091</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1038%2Fnprot.2017.091" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=28933778" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFehtrjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=2189-2214&author=M.+Porebaauthor=G.+S.+Salvesenauthor=M.+Drag&title=Synthesis+of+a+HyCoSuL+Peptide+Substrate+Library+to+Dissect+Protease+Substrate+Specificity&doi=10.1038%2Fnprot.2017.091"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of a HyCoSuL peptide substrate library to dissect protease substrate specificity</span></div><div class="casAuthors">Poreba, Marcin; Salvesen, Guy S.; Drag, Marcin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2189-2214</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Many biol. and chem. based approaches have been developed to design highly active and selective protease substrates and probes.  It is, however, difficult to find substrate sequences that are truly selective for any given protease, as different proteases can demonstrate a great deal of overlap in substrate specificities.  In some cases, better enzyme selectivity can be achieved using peptide libraries contg. unnatural amino acids such as the hybrid combinatorial substrate library (HyCoSuL), which uses both natural and unnatural amino acids.  HyCoSuL is a combinatorial library of tetrapeptides contg. amino acid mixts. at the P4-P2 positions, a fixed amino acid at the P1 position, and an ACC (7-amino-4-carbamoylmethylcoumarin) fluorescent tag occupying the P1' position.  Once the peptide is recognized and cleaved by a protease, the ACC is released and produces a readable fluorescence signal.  Here, we describe the synthesis and screening of HyCoSuL for human caspases and legumain.  We also discuss possible modifications and adaptations of this approach that make it a useful tool for developing highly active and selective reagents for a wide variety of proteolytic enzymes.  The protocol can be divided into three major parts: (i) solid-phase synthesis of the fluorescence-labeled HyCoSuL, (ii) screening of protease P4-P2 preferences, and (iii) synthesis of the optimized activity probes equipped with an AOMK (acyloxymethyl ketone) reactive group and a biotin label for easy detection.  Beginning with the library design, the entire protocol can be completed in 4-8 wk (HyCoSuL synthesis: 3-5 wk; HyCoSuL screening per enzyme: 4-8 d; and activity-based probe synthesis: 1-2 wk).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYBvdwWr2XhLVg90H21EOLACvtfcHk0lgDB5U8CUtc2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFehtrjF&md5=5ece1429e76feedd3eb864f07da7f8b7</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2017.091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2017.091%26sid%3Dliteratum%253Aachs%26aulast%3DPoreba%26aufirst%3DM.%26aulast%3DSalvesen%26aufirst%3DG.%2BS.%26aulast%3DDrag%26aufirst%3DM.%26atitle%3DSynthesis%2520of%2520a%2520HyCoSuL%2520Peptide%2520Substrate%2520Library%2520to%2520Dissect%2520Protease%2520Substrate%2520Specificity%26jtitle%3DNat.%2520Protoc.%26date%3D2017%26volume%3D12%26spage%3D2189%26epage%3D2214%26doi%3D10.1038%2Fnprot.2017.091" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Poreba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasperkiewicz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snipas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fasci, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvesen, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span> <span> </span><span class="NLM_article-title">Unnatural Amino Acids Increase Sensitivity and Provide for the Design of Highly Selective Caspase Substrates</span>. <i>Cell Death Differ.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1482</span>– <span class="NLM_lpage">1492</span>, <span class="refDoi"> DOI: 10.1038/cdd.2014.64</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1038%2Fcdd.2014.64" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=24832467" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BC2cXot1ymsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2014&pages=1482-1492&author=M.+Porebaauthor=P.+Kasperkiewiczauthor=S.+J.+Snipasauthor=D.+Fasciauthor=G.+S.+Salvesenauthor=M.+Drag&title=Unnatural+Amino+Acids+Increase+Sensitivity+and+Provide+for+the+Design+of+Highly+Selective+Caspase+Substrates&doi=10.1038%2Fcdd.2014.64"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Unnatural amino acids increase sensitivity and provide for the design of highly selective caspase substrates</span></div><div class="casAuthors">Poreba, M.; Kasperkiewicz, P.; Snipas, S. J.; Fasci, D.; Salvesen, G. S.; Drag, M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death & Differentiation</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1482-1492</span>CODEN:
                <span class="NLM_cas:coden">CDDIEK</span>;
        ISSN:<span class="NLM_cas:issn">1350-9047</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Traditional combinatorial peptidyl substrate library approaches generally utilize natural amino acids, limiting the usefulness of this tool in generating selective substrates for proteases that share similar substrate specificity profiles.  To address this limitation, we synthesized a Hybrid Combinatorial Substrate Library (HyCoSuL) with the general formula of Ac-P4-P3-P2-Asp-ACC, testing the approach on a family of closely related proteases - the human caspases.  The power of this library for caspase discrimination extends far beyond traditional PS-SCL approach, as in addn. to 19 natural amino acids we also used 110 diverse unnatural amino acids that can more extensively explore the chem. space represented by caspase-active sites.  Using this approach we identified and employed peptide-based substrates that provided excellent discrimination between individual caspases, allowing us to simultaneously resolve the individual contribution of the apical caspase-9 and the executioner caspase-3 and caspase-7 in the development of cytochrome-c-dependent apoptosis for the first time.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSDqSSP6nnUrVg90H21EOLACvtfcHk0lgDB5U8CUtc2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXot1ymsLs%253D&md5=319049bbe5679cb416e1698a900d2a47</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fcdd.2014.64&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcdd.2014.64%26sid%3Dliteratum%253Aachs%26aulast%3DPoreba%26aufirst%3DM.%26aulast%3DKasperkiewicz%26aufirst%3DP.%26aulast%3DSnipas%26aufirst%3DS.%2BJ.%26aulast%3DFasci%26aufirst%3DD.%26aulast%3DSalvesen%26aufirst%3DG.%2BS.%26aulast%3DDrag%26aufirst%3DM.%26atitle%3DUnnatural%2520Amino%2520Acids%2520Increase%2520Sensitivity%2520and%2520Provide%2520for%2520the%2520Design%2520of%2520Highly%2520Selective%2520Caspase%2520Substrates%26jtitle%3DCell%2520Death%2520Differ.%26date%3D2014%26volume%3D21%26spage%3D1482%26epage%3D1492%26doi%3D10.1038%2Fcdd.2014.64" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rut, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poreba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasperkiewicz, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snipas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span> <span> </span><span class="NLM_article-title">Selective Substrates and Activity-Based Probes for Imaging of the Human Constitutive 20S Proteasome in Cells and Blood Samples</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">5222</span>– <span class="NLM_lpage">5234</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00026</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00026" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVags7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=5222-5234&author=W.+Rutauthor=M.+Porebaauthor=P.+Kasperkiewiczauthor=S.+J.+Snipasauthor=M.+Drag&title=Selective+Substrates+and+Activity-Based+Probes+for+Imaging+of+the+Human+Constitutive+20S+Proteasome+in+Cells+and+Blood+Samples&doi=10.1021%2Facs.jmedchem.8b00026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Substrates and Activity-Based Probes for Imaging of the Human Constitutive 20S Proteasome in Cells and Blood Samples</span></div><div class="casAuthors">Rut, Wioletta; Poreba, Marcin; Kasperkiewicz, Paulina; Snipas, Scott J.; Drag, Marcin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5222-5234</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The proteasome is an enzyme complex crit. for maintaining protein homeostasis.  Perturbed proteasome function leads to pathologies including cancer and autoimmune and neurodegenerative disease.  Therefore, the proteasome constitutes an excellent mol. target for pharmaceutical development.  Here, using the HyCoSuL approach, the authors designed and synthesized novel and selective fluorogenic substrates for each of these three constitutive 20S proteasome activities and applied them to assess inhibition of proteasome subunits by MG-132 and a clin. used inhibitor bortezomib.  The authors' results confirm the utility of designed substrates in biochem. assays.  Furthermore, selective peptide sequences obtained in this manner were used to construct fluorophore-labeled activity-based probes and then used to detect each constitutive 20S proteasome subunit simultaneously in lysates of HEK-293F cells and red blood cells.  Overall, the authors describe a simple and rapid method useful to measure constitutive 20S proteasome activity in whole human blood samples that could enable early diagnosis of pathol. states assocd. with aberrantly upregulated proteasome activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrd0FSTAesirVg90H21EOLACvtfcHk0lgDB5U8CUtc2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVags7vO&md5=1fb97b492a7f3f25d6659035f5bf4b14</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00026%26sid%3Dliteratum%253Aachs%26aulast%3DRut%26aufirst%3DW.%26aulast%3DPoreba%26aufirst%3DM.%26aulast%3DKasperkiewicz%26aufirst%3DP.%26aulast%3DSnipas%26aufirst%3DS.%2BJ.%26aulast%3DDrag%26aufirst%3DM.%26atitle%3DSelective%2520Substrates%2520and%2520Activity-Based%2520Probes%2520for%2520Imaging%2520of%2520the%2520Human%2520Constitutive%252020S%2520Proteasome%2520in%2520Cells%2520and%2520Blood%2520Samples%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D5222%26epage%3D5234%26doi%3D10.1021%2Facs.jmedchem.8b00026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bird, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buzza, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKee, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whisstock, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bird, P. I.</span></span> <span> </span><span class="NLM_article-title">Expression and Purification of Recombinant Human Granzyme B from Pichia Pastoris</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>261</i></span>,  <span class="NLM_fpage">251</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1006/bbrc.1999.0989</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1006%2Fbbrc.1999.0989" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10425174" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADyaK1MXks1Okurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=261&publication_year=1999&pages=251-255&author=J.+Sunauthor=C.+H.+Birdauthor=M.+S.+Buzzaauthor=K.+E.+McKeeauthor=J.+C.+Whisstockauthor=P.+I.+Bird&title=Expression+and+Purification+of+Recombinant+Human+Granzyme+B+from+Pichia+Pastoris&doi=10.1006%2Fbbrc.1999.0989"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Expression and purification of recombinant human granzyme B from Pichia pastoris</span></div><div class="casAuthors">Sun, Jiuru; Bird, Catherina H.; Buzza, Marguerite S.; McKee, Katherine E.; Whisstock, James C.; Bird, Phillip I.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">261</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">251-255</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">Granzyme B (I) is a cytotoxic lymphocyte granule serine proteinase that is pivotal in the induction of target cell apoptosis.  Here, the authors describe the expression of recombinant human I in Pichia pastoris as a chimeric zymogen comprising the α-factor signal sequence, a prodomain including an enterokinase cleavage site, and the mature I sequence followed by a hexa-histidine tag.  Inactive zymogen was purified from the medium by immobilized Co2+ chelate affinity chromatog. and then activated by enterokinase (final yield was ∼1 mg/L).  The recombinant enzyme resembled native I in size and glycosylation, hydrolyzed the substrate, Boc-Ala-Ala-Asp-thiobenzyl ester, with equiv. efficiency (Km = 82 μM; kcat = 12 s-1), processed procaspase-3 to its subunit form, and was inhibited by the cognate serpin PI-9.  It efficiently induced DNA degrdn. and apoptosis of human cells.  The availability of recombinant human I will facilitate further investigation of its structure and role in immune effector cells.  (c) 1999 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRjLDtcYtVKLVg90H21EOLACvtfcHk0lgcMFKWpToEqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXks1Okurk%253D&md5=531feb4c527d01411891554ebe16886e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1006%2Fbbrc.1999.0989&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fbbrc.1999.0989%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DBird%26aufirst%3DC.%2BH.%26aulast%3DBuzza%26aufirst%3DM.%2BS.%26aulast%3DMcKee%26aufirst%3DK.%2BE.%26aulast%3DWhisstock%26aufirst%3DJ.%2BC.%26aulast%3DBird%26aufirst%3DP.%2BI.%26atitle%3DExpression%2520and%2520Purification%2520of%2520Recombinant%2520Human%2520Granzyme%2520B%2520from%2520Pichia%2520Pastoris%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D1999%26volume%3D261%26spage%3D251%26epage%3D255%26doi%3D10.1006%2Fbbrc.1999.0989" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oleksyszyn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, J. C.</span></span> <span> </span><span class="NLM_article-title">Amino Acid and Peptide Phosphonate Derivatives as Specific Inhibitors of Serine Peptidases</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>244</i></span>,  <span class="NLM_fpage">423</span>– <span class="NLM_lpage">441</span>, <span class="refDoi"> DOI: 10.1016/0076-6879(94)44032-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1016%2F0076-6879%2894%2944032-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=7845224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=1%3ACAS%3A528%3ADyaK2MXjvFWrtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=244&publication_year=1994&pages=423-441&author=J.+Oleksyszynauthor=J.+C.+Powers&title=Amino+Acid+and+Peptide+Phosphonate+Derivatives+as+Specific+Inhibitors+of+Serine+Peptidases&doi=10.1016%2F0076-6879%2894%2944032-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Amino acid and peptide phosphonate derivatives as specific inhibitors of serine peptidases</span></div><div class="casAuthors">Oleksyszyn, Jozef; Powers, James C.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">244</span>
        (<span class="NLM_cas:issue">Proteolytic Enzymes: Serine and Cysteine Peptidases</span>),
    <span class="NLM_cas:pages">423-41</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    </div><div class="casAbstract">Peptidyl derivs. of α-aminoalkyl phosphonate di-Ph esters are highly specific irreversible inhibitors of serine peptidases and are chem. stable and stable in plasma.  They offer a no. of advantages over other types of inhibitors currently in use in biol. expts.  After reaction with the enzyme, they form very stable enzyme-inhibitor complexes, making them interesting tools for x-ray studies on the active site structure of new serine peptidases.  Synthetic methods, synthetic examples, and inhibition kinetics and methods are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaaiAF4Y5HxbVg90H21EOLACvtfcHk0lgcMFKWpToEqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXjvFWrtrs%253D&md5=78dc438666d1816ccbc9551d95b6c281</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2F0076-6879%2894%2944032-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0076-6879%252894%252944032-8%26sid%3Dliteratum%253Aachs%26aulast%3DOleksyszyn%26aufirst%3DJ.%26aulast%3DPowers%26aufirst%3DJ.%2BC.%26atitle%3DAmino%2520Acid%2520and%2520Peptide%2520Phosphonate%2520Derivatives%2520as%2520Specific%2520Inhibitors%2520of%2520Serine%2520Peptidases%26jtitle%3DMethods%2520Enzymol.%26date%3D1994%26volume%3D244%26spage%3D423%26epage%3D441%26doi%3D10.1016%2F0076-6879%2894%2944032-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oleksyszyn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subotkowska, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mastalerz, P.</span></span> <span> </span><span class="NLM_article-title">Diphenyl 1-Aminoalkanephosphonates</span>. <i>Synthesis</i> <span class="NLM_year" style="font-weight: bold;">1979</span>,  <span class="NLM_volume"><i>1979</i></span>,  <span class="NLM_fpage">985</span>– <span class="NLM_lpage">986</span>, <span class="refDoi"> DOI: 10.1055/s-1979-28903</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;key=10.1055%2Fs-1979-28903" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1979&publication_year=1979&pages=985-986&author=J.+Oleksyszynauthor=L.+Subotkowskaauthor=P.+Mastalerz&title=Diphenyl+1-Aminoalkanephosphonates&doi=10.1055%2Fs-1979-28903"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1055%2Fs-1979-28903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-1979-28903%26sid%3Dliteratum%253Aachs%26aulast%3DOleksyszyn%26aufirst%3DJ.%26aulast%3DSubotkowska%26aufirst%3DL.%26aulast%3DMastalerz%26aufirst%3DP.%26atitle%3DDiphenyl%25201-Aminoalkanephosphonates%26jtitle%3DSynthesis%26date%3D1979%26volume%3D1979%26spage%3D985%26epage%3D986%26doi%3D10.1055%2Fs-1979-28903" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i27"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b02042">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_53300"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b02042?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b02042</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Detailed screening profiles; full-size blots; substrate selectivity testing; amino acid structures used in libraries and individual compounds; HPLC and MS analyses for substrates, inhibitor, and activity-based probe (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b02042/suppl_file/jm9b02042_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b02042/suppl_file/jm9b02042_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b02042/suppl_file/jm9b02042_si_001.pdf">jm9b02042_si_001.pdf (2.7 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b02042/suppl_file/jm9b02042_si_002.csv">jm9b02042_si_002.csv (3.02 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b02042&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b02042%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-6" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b02042" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799a8ba39643dbe","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
