
<html lang="en"     class="pb-page"  data-request-id="36a6c3b6-efcf-4fb5-9397-2ab633f718ac"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2021.64.issue-5;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.0c01779;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Fragment-like Chloroquinolineamines Activate the Orphan Nuclear Receptor Nurr1 and Elucidate Activation Mechanisms" /></meta><meta name="dc.Creator" content="Sabine  Willems" /></meta><meta name="dc.Creator" content="Julia  Ohrndorf" /></meta><meta name="dc.Creator" content="Whitney  Kilu" /></meta><meta name="dc.Creator" content="Jan  Heering" /></meta><meta name="dc.Creator" content="Daniel  Merk" /></meta><meta name="dc.Description" content="The ligand-activated transcription factor nuclear receptor related-1 (Nurr1) exhibits great potential for neurodegenerative disease treatment, but potent Nurr1 modulators to further probe and valid..." /></meta><meta name="Description" content="The ligand-activated transcription factor nuclear receptor related-1 (Nurr1) exhibits great potential for neurodegenerative disease treatment, but potent Nurr1 modulators to further probe and valid..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 25, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01779" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 The Authors. Published byAmerican Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01779" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01779" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01779" /></link>
        
    
    

<title>Fragment-like Chloroquinolineamines Activate the Orphan Nuclear Receptor Nurr1 and Elucidate Activation Mechanisms | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01779" /></meta><meta property="og:title" content="Fragment-like Chloroquinolineamines Activate the Orphan Nuclear Receptor Nurr1 and Elucidate Activation Mechanisms" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/large/jm0c01779_0011.jpeg" /></meta><meta property="og:description" content="The ligand-activated transcription factor nuclear receptor related-1 (Nurr1) exhibits great potential for neurodegenerative disease treatment, but potent Nurr1 modulators to further probe and validate the nuclear receptor as a therapeutic target are lacking. We have systematically studied the structure–activity relationship of the 4-amino-7-chloroquinoline scaffold contained in Nurr1 activators amodiaquine and chloroquine and discovered fragment-like analogues that activated Nurr1 in several cellular settings. The most active descendants promoted the transcriptional activity of Nurr1 on human response elements as monomer, homodimer, and heterodimer and markedly enhanced Nurr1-dependent gene expression in human astrocytes. As a tool to elucidate mechanisms involving in Nurr1 activation, these Nurr1 agonists induced robust recruitment of NCoR1 and NCoR2 co-regulators to the Nurr1 ligand binding domain and promoted Nurr1 dimerization. These findings provide important insights in Nurr1 regulation. The fragment-sized Nurr1 agonists are appealing starting points for medicinal chemistry and valuable early Nurr1 agonist tools for pharmacology and chemical biology." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01779"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01779">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01779&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01779&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01779&amp;href=/doi/10.1021/acs.jmedchem.0c01779" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 5</span><span class="cit-fg-pageRange">, 2659-2668</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/5" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01660" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01715" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Fragment-like Chloroquinolineamines Activate the Orphan Nuclear Receptor Nurr1 and Elucidate Activation Mechanisms</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Sabine Willems</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sabine Willems</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Chemistry, Goethe University, Frankfurt, Max-von-Laue-Str. 9, D-60438 Frankfurt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sabine++Willems">Sabine Willems</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Julia Ohrndorf</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Julia Ohrndorf</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Chemistry, Goethe University, Frankfurt, Max-von-Laue-Str. 9, D-60438 Frankfurt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Julia++Ohrndorf">Julia Ohrndorf</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Whitney Kilu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Whitney Kilu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Chemistry, Goethe University, Frankfurt, Max-von-Laue-Str. 9, D-60438 Frankfurt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Whitney++Kilu">Whitney Kilu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jan Heering</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jan Heering</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Fraunhofer Institute for Translational Medicine and Pharmacology ITMP, Theodor-Stern-Kai 7, D-60596 Frankfurt am Main, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jan++Heering">Jan Heering</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Daniel Merk</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Daniel Merk</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Chemistry, Goethe University, Frankfurt, Max-von-Laue-Str. 9, D-60438 Frankfurt, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#9df0f8eff6ddedf5fceff0fef5f8f0b3e8f3f4b0fbeffcf3f6fbe8efe9b3f9f8"><span class="__cf_email__" data-cfemail="3c51594e577c4c545d4e515f54595112495255115a4e5d52575a494e48125859">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Daniel++Merk">Daniel Merk</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5359-8128" title="Orcid link">http://orcid.org/0000-0002-5359-8128</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01779&amp;href=/doi/10.1021%2Facs.jmedchem.0c01779" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 5</span><span class="cit-pageRange">, 2659–2668</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 25, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>19 October 2020</li><li><span class="item_label"><b>Published</b> online</span>25 February 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 11 March 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01779" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01779</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 The Authors. Published byAmerican Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D2659%26pageCount%3D10%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DSabine%2BWillems%252C%2BJulia%2BOhrndorf%252C%2BWhitney%2BKilu%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D5%26contentID%3Dacs.jmedchem.0c01779%26title%3DFragment-like%2BChloroquinolineamines%2BActivate%2Bthe%2BOrphan%2BNuclear%2BReceptor%2BNurr1%2Band%2BElucidate%2BActivation%2BMechanisms%26numPages%3D10%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D2668%26publicationDate%3DMarch%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01779"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">702</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">2</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01779" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Fragment-like Chloroquinolineamines Activate the Orphan Nuclear Receptor Nurr1 and Elucidate Activation Mechanisms&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Sabine&quot;,&quot;last_name&quot;:&quot;Willems&quot;},{&quot;first_name&quot;:&quot;Julia&quot;,&quot;last_name&quot;:&quot;Ohrndorf&quot;},{&quot;first_name&quot;:&quot;Whitney&quot;,&quot;last_name&quot;:&quot;Kilu&quot;},{&quot;first_name&quot;:&quot;Jan&quot;,&quot;last_name&quot;:&quot;Heering&quot;},{&quot;first_name&quot;:&quot;Daniel&quot;,&quot;last_name&quot;:&quot;Merk&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;25&quot;,&quot;issue&quot;:&quot;5&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;2659-2668&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01779&quot;},&quot;abstract&quot;:&quot;The ligand-activated transcription factor nuclear receptor related-1 (Nurr1) exhibits great potential for neurodegenerative disease treatment, but potent Nurr1 modulators to further probe and validate the nuclear receptor as a therapeutic target are lacking. We have systematically studied the structure–activity relationship of the 4-amino-7-chloroquinoline scaffold contained in Nurr1 activators amodiaquine and chloroquine and discovered fragment-like analogues that activated Nurr1 in several cellular settings. The most active descendants promoted the transcriptional activity of Nurr1 on human response elements as monomer, homodimer, and heterodimer and markedly enhanced Nurr1-dependent gene expression in human astrocytes. As a tool to elucidate mechanisms involving in Nurr1 activation, these Nurr1 agonists induced robust recruitment of NCoR1 and NCoR2 co-regulators to the Nurr1 ligand binding domain and promoted Nurr1 dimerization. These findings provide important insights in Nurr1 regulation. The fragment-si&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01779&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01779" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01779&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01779" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01779&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01779" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01779&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01779&amp;href=/doi/10.1021/acs.jmedchem.0c01779" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01779" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01779" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01779&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01779%26sid%3Dliteratum%253Aachs%26pmid%3D33629841%26genre%3Darticle%26aulast%3DWillems%26date%3D2021%26atitle%3DFragment-like%2BChloroquinolineamines%2BActivate%2Bthe%2BOrphan%2BNuclear%2BReceptor%2BNurr1%2Band%2BElucidate%2BActivation%2BMechanisms%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D5%26spage%3D2659%26epage%3D2668%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292589" title="Agonists">Agonists</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/5" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/jmcmar.2021.64.issue-5/20210311/jmcmar.2021.64.issue-5.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/medium/jm0c01779_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/large/jm0c01779_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01779&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The ligand-activated transcription factor nuclear receptor related-1 (Nurr1) exhibits great potential for neurodegenerative disease treatment, but potent Nurr1 modulators to further probe and validate the nuclear receptor as a therapeutic target are lacking. We have systematically studied the structure–activity relationship of the 4-amino-7-chloroquinoline scaffold contained in Nurr1 activators amodiaquine and chloroquine and discovered fragment-like analogues that activated Nurr1 in several cellular settings. The most active descendants promoted the transcriptional activity of Nurr1 on human response elements as monomer, homodimer, and heterodimer and markedly enhanced Nurr1-dependent gene expression in human astrocytes. As a tool to elucidate mechanisms involving in Nurr1 activation, these Nurr1 agonists induced robust recruitment of NCoR1 and NCoR2 co-regulators to the Nurr1 ligand binding domain and promoted Nurr1 dimerization. These findings provide important insights in Nurr1 regulation. The fragment-sized Nurr1 agonists are appealing starting points for medicinal chemistry and valuable early Nurr1 agonist tools for pharmacology and chemical biology.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02474" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02474" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The ligand-sensing transcription factor nuclear receptor related-1 (Nurr1, NR4A2)<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> is an orphan nuclear receptor with neuroprotective properties. Nurr1 is found in several neuronal cells with particularly high expression in dopaminergic neurons.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> It regulates the expression of various genes in dopamine metabolism and transport, which turned out to be crucial for the protection and survival of dopaminergic neurons.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Altered expression of Nurr1 in patients of Parkinson’s Disease (PD) and the observation that neuronal Nurr1 knockout in mice causes a phenotype resembling PD further point to high therapeutic potential of Nurr1 modulation in PD and other neurodegenerative pathologies.<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2,3)</a> The prostaglandins PGA1 and PGE1 were recently discovered as endogenous Nurr1 ligands with intermediate micromolar potency and neuroprotective effects.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> However, the lack of potent Nurr1 modulators as tools hinders further evaluation and validation of Nurr1 as a therapeutic target.</div><div class="NLM_p">Amodiaquine (AQ, <b>1</b>), chloroquine (CQ, <b>2</b>), and glafenine (<b>3</b>) have been reported as first-in-class activators of Nurr1<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> with micromolar activity (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). These compounds importantly demonstrate that Nurr1 can be activated with small molecules. However, their limited potency and their effects on various other proteins and signaling pathways<a onclick="showRef(event, 'ref5 ref6 ref7 ref8 ref9 ref10 ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7 ref8 ref9 ref10 ref11 ref12 ref13">(5−13)</a> render them insufficient as tools for pharmacology and chemical biology. Optimized Nurr1 activators are required to study the role of the orphan nuclear receptor in health and disease.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SAR of Side-Chain Motifs in the AQ/CQ Chemotype. Biological Activity of 1–10 on Nurr1 In Vitro<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/medium/jm0c01779_0006.gif" alt="" id="gr5" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Activity was determined in a Gal4-Nurr1 hybrid reporter gene assay. EC<sub>50</sub> and IC<sub>50</sub> values are the mean ± SD; <i>n</i> ≥ 3. Max. fold activation refers to fold reporter activity compared to DMSO (0.1%)-treated cells.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Inactive: no significant effect on reporter activity (≥1.5-fold activation or compared to Gal4-VP16 at the highest nontoxic concentration as indicated).</p></div></div><div></div></div><div class="NLM_p">Munoz-Tello et al.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> have recently demonstrated that among the putative Nurr1 modulators reported in the literature, only AQ (<b>1</b>), CQ (<b>2</b>), and cytosporone B act as direct ligands of the nuclear receptor to control its transcriptional activity. Moreover, this recent study confirms previous observations that AQ (<b>1</b>) and CQ (<b>2</b>) likely target the canonical ligand binding site of the Nurr1 ligand binding domain (LBD)<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> in contrast to PGA1 and PGE1, which bind to a noncanonical site between helices H5, H11, and H12.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> These important findings<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> make the AQ/CQ chemotype of Nurr1 modulators an attractive starting point for the development of Nurr1 targeting tool compounds.</div><div class="NLM_p last">Using <b>1</b>–<b>3</b> as the starting matter, we have assessed the structure–activity relationship (SAR) of this AQ chemotype of Nurr1 ligands by a rapid fragment-based strategy. We have discovered that the 4-amino-7-chloroquinoline (<b>4</b>) scaffold alone is sufficient for Nurr1 activation despite low potency. By systematically varying its substitution pattern, we discovered potent, fragment-like Nurr1 agonists (<b>24</b> and <b>26</b>), which activated Nurr1 in cellular and cell-free settings. <b>26</b> evolves as a valuable chemical tool to probe molecular mechanisms of Nurr1 activation. In contrast to AQ and CQ, whose tool compound applicability is hindered by nonspecific effects on transcriptional activity,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a><b>26</b> overcomes this limitation. Using <b>26</b> for functional studies, we observed robust recruitment of the nuclear receptor co-regulators NCoR1 and NCoR2 to Nurr1 by <b>26</b> and a promoting effect on Nurr1 homodimerization, which provide improved understanding of molecular Nurr1 activation mechanisms.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31364" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31364" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Chemistry</h3><div class="NLM_p">Nurr1 modulators <b>4</b>–<b>33</b> were synthesized according to <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a> or commercially available. <b>6</b> and <b>29</b>–<b>33</b> were prepared from the respective 4-chloroquinolines <b>12</b> and <b>34</b> and the respective amines <b>35</b>–<b>40</b> by nucleophilic aromatic substitution according to a previously reported procedure to <b>6</b><a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> with suitable adaptions (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). <b>27</b> and <b>28</b> were generated by reductive amination from 5-chloronapthalen-1-amine (<b>24</b>) and the respective ketones <b>41</b> and <b>42</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).</div><figure id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/medium/jm0c01779_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/large/jm0c01779_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <b>6</b> and <b>29</b>–<b>33</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/large/jm0c01779_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01779&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) KI, EtOH, 2 N HCl, 90 °C, 14–20 h; (b) EtOH, <i>μw</i>, 140 °C, 36–48 h.</p></p></figure><figure id="sch2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/medium/jm0c01779_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/large/jm0c01779_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of 27 and 28<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/large/jm0c01779_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01779&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) HOAc, DCE, room temperature, 30 min; then NaB(OAc)<sub>3</sub>H, DCE, 50 °C, 24 h.</p></p></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Biological Evaluation</h3><div class="NLM_p">A cellular hybrid reporter gene assay in HEK293 cells served as a primary test system to determine Nurr1 modulation by <b>1</b>–<b>33</b>. This assay is based on a hybrid receptor construct composed of the human Nurr1 LBD and the Gal4 DNA binding domain from yeast. A Gal4-sensitive firefly luciferase construct served as the reporter gene, and constitutively expressed renilla luciferase (SV40 promoter) was used to normalize for transfection efficiency and to monitor test compound toxicity. In agreement with the constitutively active nature of Nurr1,<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> the chimeric Gal4-Nurr1 receptor displays strong intrinsic transcriptional inducer activity also in the absence of a ligand. As a control experiment, all tested compounds were assessed for unspecific effects on reporter activity in an analogous setting with the potent transcriptional inducer Gal4-VP16<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> replacing Gal4-Nurr1 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01779/suppl_file/jm0c01779_si_001.pdf" class="ext-link">Figure S1</a>). By providing insights into the type of activity (agonist or inverse agonist), potency, and efficacy of the tested compounds, this hybrid reporter gene assay appeared as the most suitable primary test system, especially since knowledge on Nurr1-interacting co-regulators and their response to ligands as the basis for cell-free recruitment assays is still limited.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a></div><div class="NLM_p">For further insights into cellular Nurr1 modulation in more physiological settings, selected compounds were profiled for activation of full-length human Nurr1 as the monomer, homodimer, or RXR-heterodimer. For this, firefly reporter constructs comprising a single repeat of the respective human response elements of monomeric Nurr1 (NBRE), the Nurr1 homodimer (NurRE), or the RXR-Nurr1 heterodimer (DR5) in front of the reporter gene were used. Nurr1 and, in the case of DR5, also RXRα were overexpressed using CMV promoter-dependent expression constructs. As for the hybrid Gal4-Nurr1 assay, constitutively expressed renilla luciferase served for normalization purposes. In addition, effects of selected compounds on Nurr1-regulated gene expression were evaluated in Nurr1 expressing<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> T98G glioblastoma cells on the mRNA level by quantitative real-time polymerase chain reaction (qRT-PCR).</div><div class="NLM_p last">Nurr1 modulation was also studied in cell-free, homogeneous time-resolved fluorescence resonance energy transfer (HTRF)-based settings using Tb<sup>3+</sup>-cryptate-labeled co-regulator peptides derived from NCoR1 and NCoR2 as FRET donors and recombinant, GFP-labeled Nurr1 LBD protein as the FRET acceptor.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Structure–Activity Relationship</h3><div class="NLM_p">All three reported Nurr1 activators AQ (<b>1</b>), CQ (<b>2</b>), and glafenine (<b>3</b>) share an identical 4-amino-7-chloroquinoline scaffold prompting the hypothesis that this shared structural feature strongly contributes to their biological activity on Nurr1. To preliminarily test this assumption, we determined the Nurr1 modulatory activity of a small series of further 4-amino-7-chloroquinoline derivatives (<b>4</b>–<b>10</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><div class="NLM_p">AQ (<b>1</b>) and CQ (<b>2</b>) activated Gal4-Nurr1 with intermediate micromolar EC<sub>50</sub> values of 36 ± 4 μM (3.6 ± 0.1 max. fold activation) and 47 ± 5 μM (2.0 ± 0.1 max. fold activation), respectively, which agreed with their reported activities<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> and validated our in vitro test system. Glafenine (<b>3</b>) turned out to be too toxic to be reasonably characterized in the cellular setting. The minimal shared structure 4-amino-7-chloroquinoline (<b>4</b>) of AQ (<b>1</b>) and CQ (<b>2</b>) was indeed sufficient to activate Nurr1 despite lower potency in a high micromolar range. AQ analogue <b>5</b> lacking the phenolic hydroxyl group retained reduced Nurr1 agonistic potency, too, while removal of the diethylaminomethyl motif (<b>6</b>) resulted in full loss of activity. Replacement of the basic side chain of AQ (<b>1</b>) and CQ (<b>2</b>) by a butyric acid motif of similar size in <b>7</b> produced an inverse agonist that markedly reduced the transcriptional activity of Nurr1. A small lipophilic isopentyl side chain residue (<b>8</b>) recovered Nurr1 agonism with a remarkable 1.8 μM EC<sub>50</sub> value but low activation efficacy. The bulkier 4-methylcyclohexyl- (<b>9</b>) and benzyl- (<b>10</b>) derivatives failed to modulate Nurr1 activity. These preliminary SAR observations indicate that the side chain motif contributes to Nurr1 modulation but characterize the 4-amino-7-chloroquinoline (<b>4</b>) as the key structural feature for Nurr1 modulation. Intrigued by the observation that the fragment-sized structure <b>4</b> was sufficient to activate Nurr1, we studied the SAR of the isolated chloroquinoline-amine motif individually.</div><div class="NLM_p">First, we evaluated the individual contributions of structural features of <b>4</b> to Nurr1 activation by their systematic removal (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). 7-Chloroquinoline (<b>11</b>) lacking the 4-amino group revealed inverse Nurr1 agonism with moderate repressor efficacy, while 4,7-dichloroquinoline (<b>12</b>) was inactive pointing to an important contribution of the amino group to Nurr1 activation. 4-Aminoquinoline (<b>13</b>) lacking the 7-chlorine substituent was inactive, too, indicating the chlorine as another important feature for Nurr1 agonism. Introduction of an additional methyl group in the 2-position of the 4-amino-7-chloroquinoline (<b>14</b>) was accompanied with a remarkable gain in potency by almost a factor of 10 compared to <b>4</b>.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Contribution of Chlorine and Amine Residues in the Chloroquinolineamine Scaffold. Biological Activity of 11–14 on Nurr1 In Vitro<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/medium/jm0c01779_0007.gif" alt="" id="gr6" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Activity was determined in a Gal4-Nurr1 hybrid reporter gene assay. EC<sub>50</sub> and IC<sub>50</sub> values are the mean ± SD; n ≥ 3. Max. fold activation refers to fold reporter activity compared to DMSO (0.1%)-treated cells.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">Inactive: no significant effect on reporter activity (≥1.5-fold activation or compared to Gal4-VP16 at the highest nontoxic concentration as indicated).</p></div></div><div></div></div><div class="NLM_p">We then systematically varied the regiochemistry of the essential chlorine and amine substituents (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Shifting the chlorine atom from the 7- (<b>4</b>) to the 6- (<b>15</b>) or 8-position (<b>16</b>) strongly promoted potency on Nurr1, with 8-chloro-4-aminoquinoline (<b>16</b>) as the most favored isomer. For the amine substituent, agonism on Nurr1 was lost when the amino group was moved from the 4-position in <b>4</b> to the 2- (<b>17</b>) or 3-position (<b>18</b>), while shifting the amine to the benzoid ring in the 5-position (<b>19</b>) was favored by Nurr1. However, this structural modification turned out to be incompatible with the favored 6- (<b>15</b>) or 8-position (<b>16</b>) of the chlorine substituent since 6-chloro-5-aminoquinoline (<b>20</b>) and 8-chloro-5-aminoquinoline (<b>21</b>) were inactive. When we replaced the 8-chlorine substituent of the preferred chloroquinolineamine isomer <b>16</b> by a bulkier trifluoromethyl group (<b>22</b>) or by a smaller fluorine atom (<b>23</b>), we observed a drastic loss in potency, suggesting that the chlorine atom was highly favored in this position. Eventually, we also addressed the contribution of the quinoline nitrogen atom in the favored regioisomer <b>16</b> whose removal in naphthalene <b>24</b> was favored and promoted potency by a factor of 5, whereas shifting the nitrogen by one position to isoquinoline <b>25</b> resulted in inactivity.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. SAR and Biological Activity of Chloroquinoline-Amine Regioisomers 15–25 on Nurr1 In Vitro<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/medium/jm0c01779_0008.gif" alt="" id="gr7" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Activity was determined in a Gal4-Nurr1 hybrid reporter gene assay. EC<sub>50</sub> values are the mean ± SD; n ≥ 3. Max. fold activation refers to fold reporter activity compared to DMSO (0.1%)-treated cells.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Inactive: no significant effect on reporter activity (≥1.5-fold activation or compared to Gal4-VP16 at the highest nontoxic concentration as indicated).</p></div></div><div></div></div><div class="NLM_p">The in vitro activities of <b>11</b>–<b>25</b> demonstrated the presence and regiochemistry of the amine and chlorine substituents as crucial contributing factors for potency on Nurr1. Additionally, we observed an increase in potency for a methyl group in the 2-position. Combination of this favorable methyl substituent (<b>14</b>) with a preferred regiochemistry of the chlorine and amine substituents (<b>16</b>) in <b>26</b> further enhanced potency to a low micromolar EC<sub>50</sub> value (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Fused SAR in the Chloroquinolineamine Fragment. Biological Activity of 14, 16, and 26 on Nurr1 In Vitro<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/medium/jm0c01779_0009.gif" alt="" id="gr8" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Activity was determined in a Gal4-Nurr1 hybrid reporter gene assay. EC<sub>50</sub> values are the mean ± SD; n ≥ 3. Max. fold activation refers to fold reporter activity compared to DMSO (0.1%)-treated cells.</p></div></div><div></div></div><div class="NLM_p">Our systematic SAR analysis of <b>4</b> and analogues as Nurr1 agonists rendered <b>24</b> and <b>26</b> as the most favorable derivatives. With EC<sub>50</sub> values of 7 and 17 μM, respectively, both fragment-like molecules <b>24</b> and <b>26</b> possess slightly higher potencies on Nurr1 than the template drugs AQ (<b>1</b>) and CQ (<b>2</b>) while comprising markedly lower molecular weights. Accordingly, <b>24</b> and <b>26</b> are superior in terms of ligand efficiency (LE), lipophilic ligand efficiency (LLE), and size-independent ligand efficiency (SILE)<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> compared to AQ (<b>1</b>) and CQ (<b>2</b>), which were employed as leads (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>).</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Efficiency Metrics of Nurr1 Agonists<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">ID</th><th class="colsep0 rowsep0" align="center" char="±">EC<sub>50</sub> (Nurr1)</th><th class="colsep0 rowsep0" align="center" char=".">LE</th><th class="colsep0 rowsep0" align="center" char=".">LLE<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char=".">SILE</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="char" char="±">36 ± 4 μM</td><td class="colsep0 rowsep0" align="char" char=".">0.24</td><td class="colsep0 rowsep0" align="char" char=".">–0.39</td><td class="colsep0 rowsep0" align="char" char=".">1.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="char" char="±">47 ± 5 μM</td><td class="colsep0 rowsep0" align="char" char=".">0.27</td><td class="colsep0 rowsep0" align="char" char=".">–0.95</td><td class="colsep0 rowsep0" align="char" char=".">1.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="char" char="±">7.3 ± 0.5 μM</td><td class="colsep0 rowsep0" align="char" char=".">0.59</td><td class="colsep0 rowsep0" align="char" char=".">1.85</td><td class="colsep0 rowsep0" align="char" char=".">2.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="char" char="±">17 ± 6 μM</td><td class="colsep0 rowsep0" align="char" char=".">0.50</td><td class="colsep0 rowsep0" align="char" char=".">1.83</td><td class="colsep0 rowsep0" align="char" char=".">2.2</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">Metrics were calculated as described in ref <a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a>.</p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">alogP for LLE calculations was retrieved from the ALOGPS 2.1 resource.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a></p></div></div></div><div class="NLM_p">In an attempt to recombine <b>24</b> and <b>26</b> with the substituents of the template drugs, we prepared and characterized the respective CQ analogues (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). Compound <b>27</b> resulting from fusion of CQ (<b>2</b>) and fragment <b>24</b> was active but too toxic for full dose–response characterization on Gal4-Nurr1. Its potency was inferior to fragment <b>24</b> but might be a minor improvement over CQ (<b>2</b>). To exclude steric hindrance as a reason for the surprisingly low potency of <b>27</b>, we studied its smaller dimethyl analogue <b>28</b>, which was inactive up to 10 μM and toxic at higher concentrations. The fusion of <b>26</b> and <b>2</b> in <b>29</b> was nontoxic but inactive on Gal4-Nurr1 up to high 100 μM concentration. Despite the undesirable nonspecific activities of AQ (<b>1</b>),<a onclick="showRef(event, 'ref5 ref6 ref7 ref8 ref9 ref10 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7 ref8 ref9 ref10 ref20 ref21">(5−10,20,21)</a> we also fused <b>26</b> with the AQ side chain, but the resulting compound <b>30</b> was inactive, too. These results suggest that recombination of the optimized fragment with the AQ (<b>1</b>) or CQ (<b>2</b>) side chains was not a constructive strategy to achieve further optimization.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Biological Activity of Fused Structures 27–33 on Nurr1 In Vitro<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/medium/jm0c01779_0010.gif" alt="" id="gr9" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">Activity was determined in a Gal4-Nurr1 hybrid reporter gene assay. EC<sub>50</sub> values are the mean ± SD; n ≥ 3. Max. fold activation refers to fold reporter activity compared to DMSO (0.1%)-treated cells.</p></div><div class="footnote" id="t6fn2"><sup><sup>b</sup></sup><p class="last">Inactive: no significant effect on reporter activity (≥1.5-fold activation or compared to Gal4-VP16 at the highest nontoxic concentration as indicated).</p></div></div><div></div></div><div class="NLM_p">The early SAR analysis had also revealed a branched lipophilic side chain (<b>8</b>) as favored for Nurr1 activation. Hence, we additionally probed fusion of this isopentyl motif and related small lipophilic groups with the optimized chloroquinoline fragment <b>26</b> (<b>31</b>–<b>33</b>, <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). All three derivatives comprising an isopropyl (<b>31</b>), <i>n</i>-butyl (<b>32</b>), or isopentyl (<b>33</b>) substituent activated Nurr1 with micromolar potencies, but neither modification provided an improvement in potency compared to the fragment <b>26</b>. Together, the weak Nurr1 modulatory activities of the fused derivatives <b>27</b>–<b>33</b>, hence, indicate that the SAR of the chloroquinoline core and the side chain motif is not additive and that systematic efforts are needed for further optimization.</div><div class="NLM_p">While compound <b>24</b> evolved as the most potent Nurr1 agonist in this study, it also exhibited nonspecific effects on the control gene (renilla luciferase) in our reporter gene assays (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01779/suppl_file/jm0c01779_si_001.pdf" class="ext-link">Figures S2a and S3</a>). Its efficacy in Nurr1 activation may therefore be overestimated. Control experiments on Gal4-VP16 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01779/suppl_file/jm0c01779_si_001.pdf" class="ext-link">Figure S1</a>) and a pronounced increase in Nurr1-induced reporter activity demonstrate, however, that <b>24</b> activates Nurr1 despite nonspecific effects. Of note, the previously reported Nurr1 agonists CQ (<b>2</b>) and especially AQ (<b>1</b>) cause opposite nonspecific effects on control gene activity (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01779/suppl_file/jm0c01779_si_001.pdf" class="ext-link">Figure S2a</a>), indicating nonspecific transcriptional activation as already observed by Munoz-Tello et al.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Compound <b>26</b>, in contrast, did not affect renilla activity (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01779/suppl_file/jm0c01779_si_001.pdf" class="ext-link">Figure S2a</a>) and VP16-dependent transcriptional activity (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01779/suppl_file/jm0c01779_si_001.pdf" class="ext-link">Figure S1</a>), suggesting it as an improved Nurr1 agonist tool with reduced nonspecific effects.</div><div class="NLM_p">Based on these considerations, <b>26</b> evolved as the most attractive fragment descendant of AQ (<b>1</b>) and CQ (<b>2</b>) for further studies on Nurr1 agonist characteristics. In addition, fragment <b>24</b> presents as an attractive starting point for further optimization despite nonspecific activity that will require attention (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01779/suppl_file/jm0c01779_si_001.pdf" class="ext-link">Figure S2</a>). Hence, we studied the effects of <b>24</b> and <b>26</b> on Nurr1 activity in more physiological settings involving the full-length human nuclear receptor (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a and <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>b; <a href="/doi/suppl/10.1021/acs.jmedchem.0c01779/suppl_file/jm0c01779_si_001.pdf" class="ext-link">Figure S4</a>). Both compounds activated the Nurr1 monomer, homodimer, and RXR-heterodimer on the human Nurr1 response elements NBRE, NurRE, and DR5. In line with the hybrid reporter gene assay data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01779/suppl_file/jm0c01779_si_001.pdf" class="ext-link">Figure S5</a>), <b>24</b> revealed higher potency compared to <b>26</b> and CQ (<b>2</b>),<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> while <b>26</b> activated the human Nurr1 response elements with higher efficacy. To confirm Nurr1 modulation also in an orthogonal native cellular setting, we determined the effects of <b>24</b> and <b>26</b> on Nurr1-regulated gene expression in Nurr1 expressing<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> human astrocytes (T98G) on the mRNA level (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>c</a>). Both <b>24</b> and <b>26</b> caused a marked induction of vesicular monoamine transporter 2 (VMAT2; SLC18A2) and tyrosine hydroxylase (TH) in a dose-dependent fashion. These results demonstrate Nurr1 activation by <b>24</b> and <b>26</b> in various cellular settings, and hence, fully characterize the fragments <b>24</b> and <b>26</b> as Nurr1 agonists useful as lead compounds for medicinal chemistry and as early tools to study Nurr1 biology.</div><figure id="fig1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/medium/jm0c01779_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/large/jm0c01779_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Nurr1 agonism of <b>24</b> and <b>26</b> in cellular settings. (a, b) Nurr1 agonist activity of (a) <b>24</b> and (b) <b>26</b> on full-length human Nurr1 as the monomer (NBRE), homodimer (NurRE), and RXR-heterodimer (DR5). Data are the mean ± S.E.M.; n ≥ 3. Individual curves are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01779/suppl_file/jm0c01779_si_001.pdf" class="ext-link">Figure S4</a>, and corresponding EC<sub>50</sub> values are listed in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01779/suppl_file/jm0c01779_si_001.pdf" class="ext-link">Table S1</a>. (c) Effects of <b>24</b> and <b>26</b> on mRNA expression of vesicular monoamine transporter 2 (VMAT2) and tyrosine hydroxylase (TH) in human astrocytes (T98G). CQ (<b>2</b>) for comparison. Nurr1 agonists <b>24</b> and <b>26</b> strongly promoted Nurr1-regulated VMAT2 and TH expression in a dose-dependent manner. Data are the mean ± S.E.M.; <i>n</i> = 4. mRNA levels were determined by qRT-PCR and analyzed by the 2<sup>-ΔΔCt</sup> method. * p < 0.05, ** p < 0.01, *** p < 0.001 vs. DMSO control (t-test).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/large/jm0c01779_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01779&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The chloroquinoline <b>26</b> comprises a consistent Nurr1 agonist profile in multiple orthogonal cellular settings including hybrid and full-length Nurr1 reporter gene assays and gene expression studies in native cells. In addition, the compound is characterized by reduced nonspecific effects compared to AQ (<b>1</b>) and CQ (<b>2</b>) and thus seemed suitable as a tool to study regulatory interactions involving in Nurr1 activation in HTRF-based systems. Using inverse Nurr1 agonists, we have previously discovered a ligand-sensitive interaction of Nurr1 with the nuclear receptor co-repressors NCoR1 and NCoR2.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> However, the previously available Nurr1 agonists AQ (<b>1</b>) and CQ (<b>2</b>) exhibited very weak (CQ) or no (AQ) effects at all on the binding of these co-regulators to Nurr1.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> In addition, their nonspecific transcriptional effects put the applicability of AQ (<b>1</b>) and CQ (<b>2</b>) as tools into question. The regulatory contributions of the bona fide co-repressors NCoR1 and NCoR2 to Nurr1 activation, thus, remained elusive. Initial experiments (not shown) suggested enhanced recruitment of NCoR1 and NCoR2 upon binding of <b>26</b> to the Nurr1 LBD, which aligned with our previous finding that these co-regulators are displaced from Nurr1 by inverse agonists. To avoid any photophysical effects of <b>26</b> in studying co-regulator recruitment to Nurr1, we determined NCoR1 and NCoR2 binding by titrating the GFP-labeled Nurr1 LBD in the presence of a fixed concentration of <b>26</b> and a fixed concentration of the FRET donor-labeled co-regulator peptide. The FRET acceptor concentration was also kept constant throughout the titration by adding respective amounts of free GFP. This setting ensured that any photophysical effects of <b>26</b> were constant over the full curve and that an increase in the HTRF signal only resulted from binding of the labeled co-regulator to the labeled Nurr1 LBD. Using this assay setup, we observed a dose-dependent increase in NCoR1 and NCoR2 recruitment to the Nurr1 LBD in the presence of <b>26</b> (20 and 100 μM, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a and <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>b) or <b>24</b> (100 μM, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01779/suppl_file/jm0c01779_si_001.pdf" class="ext-link">Figure S6</a>). In line with the previously discovered co-regulator displacement by inverse agonists, these results indicate involvement of NCoR1 and NCoR2 binding in Nurr1 activation. This is further supported by the fact that Nurr1 acts as a constitutive transcriptional activator and markedly recruits NCoR1 and NCoR2 also in the apo state (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a and <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>b, DMSO). In addition to effects on NCoR1 and NCoR2 recruitment, we have previously observed a ligand-sensitive dimerization of the Nurr1 LBD,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> wherein inverse Nurr1 agonists strongly counteracted Nurr1 homodimer formation. In line with this, the Nurr1 activators <b>24</b> and <b>26</b> enhanced Nurr1 homodimerization (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>c</a>; <a href="/doi/suppl/10.1021/acs.jmedchem.0c01779/suppl_file/jm0c01779_si_001.pdf" class="ext-link">Figure S6</a>) as another factor likely involving in Nurr1 activation. This observation aligns with the higher agonist efficacy of <b>26</b> on the human Nurr1 homodimer response element NurRE (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>b</a>). Hence, the Nurr1 agonist <b>26</b> as a tool in HTRF settings revealed increased recruitment of NCoR1 and NCoR2 to the Nurr1 LBD as well as enhanced Nurr1 homodimerization as contributing factors of Nurr1 activation.</div><figure id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/medium/jm0c01779_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/large/jm0c01779_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Effects of Nurr1 agonist <b>26</b> (20 and 100 μM) on co-regulator recruitment to the Nurr1 LBD in homogeneous time-resolved fluorescence resonance energy transfer (HTRF) assays. (a) Recruitment of Tb<sup>3+</sup>-cryptate-labeled NCoR1 to the sGFP-labeled Nurr1 LBD. (b) Recruitment of Tb<sup>3+</sup>-cryptate-labeled NCoR2 to the sGFP-labeled Nurr1 LBD. (c) Homodimerization between the Tb<sup>3+</sup>-cryptate-labeled and sGFP-labeled Nurr1 LBD. Data are the mean ± SD; <i>N</i> = 3. Compound <b>24</b> exhibits comparable effects (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01779/suppl_file/jm0c01779_si_001.pdf" class="ext-link">Figure S6</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/large/jm0c01779_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01779&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50881" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50881" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The available collection of direct Nurr1 modulating small molecules is scarce, limiting further efforts to validate Nurr1 as a drug target. Munoz-Tello et al.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> have demonstrated that the majority of the putative Nurr1 ligands does not directly interact with the nuclear receptor and thereby characterized the AQ/CQ chemotype as the most suitable starting point for further tool compound development. They have also shown that despite directly activating Nurr1, AQ and CQ have nonspecific effects on transcriptional activity, limiting their applicability as tools to study Nurr1. This aligns with several previously reported activities of the antimalarials<a onclick="showRef(event, 'ref5 ref6 ref7 ref8 ref9 ref10 ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7 ref8 ref9 ref10 ref11 ref12 ref13">(5−13)</a> and with our observation of elevated control gene transcription upon AQ and CQ treatment.</div><div class="NLM_p last">By fragmentation of the AQ/CQ chemotype of Nurr1 activators and systematic SAR elucidation of the fragment-sized chloroquinoline scaffold, we discovered <b>24</b> and <b>26</b> as novel Nurr1 agonists. Despite providing only moderately increased potency compared to AQ and CQ, these compounds have remarkably lower size and molecular weight, which make them attractive lead compounds for further expansion and optimization by medicinal chemistry. In addition, while <b>24</b> also exhibited nonspecific (potentially cytotoxic) effects on transcriptional activity, <b>26</b> had no such activity. Thereby, <b>26</b> overcomes one limitation of AQ and CQ as Nurr1 ligands, and hence, is a valuable early Nurr1 agonist tool to study the receptor’s molecular and cellular function. Using <b>26</b> for extended functional studies, we confirmed the hypothesis that the inverse Nurr1 agonist responsive<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> co-regulators NCoR1 and NCoR2 are robustly recruited to Nurr1 in the presence of agonists, which together with strengthened Nurr1 homodimerization upon binding of <b>26</b> provides insights in the activation mechanism of Nurr1.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18584" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18584" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Chemistry</h3><div class="NLM_p"><i>General</i>. All chemicals and solvents for synthesis were obtained from commercial sources in reagent grade and used without further purification. TLC was performed using TLC plates (silica gel 60 F254, 0.2 mm, Merck or Alugram Xtra Sil G/UV 0.2 mm, Macherey-Nagel) with detection under UV light (254 and 366 nm). Preparative column chromatography was performed using Silicagel 60 (Macherey-Nagel) and solvents of technical grade. Reactions with air- or moisture-sensitive compounds were carried out under an argon atmosphere and in anhydrous solvents. NMR spectra were recorded on Bruker AV 500 and AV 600 spectrometers (Bruker Corporation, Billerica, MA, USA). Chemical shifts (δ) are reported in ppm relative to TMS and coupling constants (<i>J</i>) in Hz. Multiplicity of signals is indicated as s for singlet, d for doublet, t for triplet, q for quartet, and m for multiplet. High-resolution mass spectra were recorded on a MALDI LTQ ORBITRAP XL instrument (Thermo Fisher Scientific) or on a Bruker maXis ESI-Qq-TOF-MS instrument (Bruker). Compound purity was analyzed using a Varian ProStar HPLC (SpectraLab Scientific Inc., Markham, ON, Canada) equipped with a MultoHigh100 Phenyl-5 μ 240 + 4 mm column (CS-Chromatographie Service GmbH, Langerwehe, Germany) using a gradient (H<sub>2</sub>O/MeOH 80:20 + 0.1% formic acid isocratic for 5 min to MeOH + 0.1% formic acid after additional 45 min and MeOH + 0.1% formic acid for additional 10 min) at a flow rate of 1 mL/min and UV detection at 245 and 280 nm. All final compounds for biological evaluation had a purity >95% according to the AUC at 245 and 280 nm UV detection. Final compounds from commercial sources were obtained from Sigma Aldrich, Enamine, TCI, Fluorochem, abcr, Apollo Scientific or Life Chemicals and had a purity >95% according to the supplier’s certificate.</div><div class="NLM_p">4-((7-Chloroquinolin-4-yl)amino)phenol (<b>6</b>). 4,7-Dichloroquinoline (<b>12</b>, 0.99 g, 5.0 mmol, 1.0 equiv), 4-aminophenol (<b>35</b>, 0.60 g, 5.5 mmol, 1.1 equiv), and a catalytic amount of potassium iodide were dissolved in ethanol (EtOH, 20 mL), aqueous hydrochloric acid (1.0 mL, 2 N) was added dropwise, and the mixture was stirred under reflux for 14 h. After cooling to room temperature, the crude product was filtered off and washed with EtOH to obtain <b>6</b> as a yellow solid in 98% yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d<sub>6</sub></i>): δ = 10.99 (s, 1H), 9.94 (s, 1H), 8.80 (d, <i>J</i> = 9.1 Hz, 1H), 8.45 (d, <i>J</i> = 7.1 Hz, 1H), 8.14 (d, <i>J</i> = 2.0 Hz, 1H), 7.84 (dd, <i>J</i> = 9.1, 2.1 Hz, 1H), 7.30–7.19 (m, 2H), 7.00–6.93 (m, 2H), 6.62 (d, <i>J</i> = 7.0 Hz, 1H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d<sub>6</sub></i>): δ = 157.09, 155.43, 143.10, 139.02, 138.31, 127.69, 127.23, 127.19 (2C), 125.98, 119.19, 116.49 (2C), 115.66, 99.99. HRMS (MALDI): <i>m/z</i> calculated 271.06327 for C<sub>15</sub>H<sub>12</sub>ClN<sub>2</sub>O, found 271.06403 ([M + H]<sup>+</sup>).</div><div class="NLM_p"><i>N</i><sup>4</sup>-(5-Chloronaphthalen-1-yl)-<i>N</i><sup>1</sup>,<i>N</i><sup>1</sup>-diethylpentane-1,4-diamine (<b>27</b>). 5-Chloronaphthalen-1-amine (<b>24</b>, 89 mg, 0.50 mmol, 1.0 equiv) and 5-(diethylamino)pentan-2-one (<b>41</b>, 73 μL, 0.55 mmol, 1.1 equiv) were dissolved in 1,2-dichloroethane (10 mL), molecular sieves (4 Å) were added, and the mixture was stirred at room temperature for 2 h. Then, acetic acid (0.75 mL) was added, and the mixture was stirred at 50 °C for another 2 h. NaB(OAc)<sub>3</sub>H (0.32 g, 1.5 mmol, 3.0 equiv) was then added, and the mixture was stirred at 50 °C for 24 h. After quenching with aqueous sodium hydroxide solution (100 mL, 1 M), phases were separated, and the aqueous layer was extracted twice with EtOAc (2x 100 mL). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, and solvents were removed under reduced pressure. The crude product was purified by column chromatography in hexane/EtOAc (10:1) + 2% triethylamine to obtain <b>27</b> as a brown oil in 45% yield. <sup>1</sup>H NMR (500 MHz, acetone-<i>d</i><sub>6</sub>): δ = 8.14 (dt, <i>J</i> = 8.6, 1.0 Hz, 1H), 7.56 (dd, <i>J</i> = 7.4, 1.0 Hz, 1H), 7.51–7.40 (m, 2H), 7.32 (dd, <i>J</i> = 8.6, 7.3 Hz, 1H), 6.74 (dt, <i>J</i> = 7.3, 1.0 Hz, 1H), 5.43 (s, 1H), 3.80–3.71 (m, 1H), 2.54–2.45 (m, 6H), 1.86–1.78 (m, 1H), 1.69–1.58 (m, 3H), 1.30 (d, <i>J</i> = 6.3 Hz, 3H), 0.98 (t, <i>J</i> = 7.1 Hz, 6H). <sup>13</sup>C NMR (126 MHz, acetone-<i>d<sub>6</sub></i>): δ = 144.93, 132.72, 132.44, 129.17, 126.86, 125.65, 124.43, 121.54, 111.98, 105.72, 53.51, 49.24, 47.55 (2C), 35.01, 24.70, 20.63, 12.12 (2C). HRMS (ESI+): <i>m/z</i> calculated 320.1973 for C<sub>19</sub>H<sub>28</sub>ClN<sub>2</sub>, found 320.1982 ([M + H]<sup>+</sup>).</div><div class="NLM_p"><i>N</i><sup>4</sup>-(5-Chloronaphthalen-1-yl)-<i>N</i><sup>1</sup>,<i>N</i><sup>1</sup>-dimethylpentane-1,4-diamine (<b>28</b>). 5-Chloronaphthalen-1-amine (<b>24</b>, 44 mg, 0.25 mmol, 1.0 equiv) and 5-(dimethylamino)pentan-2-one (<b>42</b>, 50 mg, 0.33 mmol, 1.3 equiv) were dissolved in 1,2-dichloroethane (10 mL), molecular sieves (4 Å) were added, and the mixture was stirred at room temperature for 2 h. Then, acetic acid (0.75 mL) was added, and the mixture was stirred at 50 °C for another 2 h. NaB(OAc)<sub>3</sub>H (0.16 g, 0.75 mmol, 3.0 equiv) was then added, and the mixture was stirred at 50 °C for 24 h. After quenching with aqueous sodium hydroxide solution (50 mL, 1 M), phases were separated, and the aqueous layer was extracted twice with EtOAc (2x 25 mL). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, and solvents were removed under reduced pressure. The crude product was purified by column chromatography with hexane/EtOAc (1:1) and acetone/toluene (20:1) + 2% triethylamine to obtain <b>28</b> as a pale brown solid in 25% yield. <sup>1</sup>H NMR (500 MHz, acetone-<i>d<sub>6</sub></i>): δ = 8.16 (dt, <i>J</i> = 8.6, 0.9 Hz, 1H), 7.56 (dd, <i>J</i> = 7.4, 1.0 Hz, 1H), 7.51–7.41 (m, 2H), 7.33 (dd, <i>J</i> = 8.6, 7.4 Hz, 1H), 6.73 (dt, <i>J</i> = 7.2, 1.1 Hz, 1H), 3.79–3.73 (m, 1H), 2.50 (t, <i>J</i> = 7.0 Hz, 2H), 2.31 (s, 6H), 1.90–1.61 (m, 4H), 1.30 (d, <i>J</i> = 6.3 Hz, 3H). <sup>13</sup>C NMR (126 MHz, acetone-<i>d<sub>6</sub></i>) δ 144.93, 132.71, 132.39, 129.17, 126.88, 125.68, 124.46, 121.67, 111.98, 105.63, 59.56, 49.15, 44.76 (2C), 34.70, 24.32, 20.76. HRMS (ESI+): <i>m/z</i> calculated 291.1628 for C<sub>17</sub>H<sub>24</sub>ClN<sub>2</sub>, found 291.1626 ([M + H]<sup>+</sup>).</div><div class="NLM_p"><i>N</i><sup>4</sup>-(8-Chloro-2-methylquinolin-4-yl)-<i>N</i><sup>1</sup>,<i>N</i><sup>1</sup>-diethylpentane-1,4-diamine (<b>29</b>). 4,8-Dichloro-2-methylquinoline (<b>34</b>, 106 mg, 0.5 mmol, 1.0 equiv) and <i>N</i><sup>1</sup>,<i>N</i><sup>1</sup>-diethylpentane-1,4-diamine (<b>37</b>, 0.12 mL, 0.6 mmol, 1.2 equiv) were dissolved in EtOH (6 mL). The mixture was stirred under microwave irradiation at 140 °C for 36 h. After cooling to room temperature, the solvent was removed under vacuum. The crude product was purified by column chromatography using a gradient of hexane/EtOAc (5:1) + 2% triethylamine to hexane/EtOAc (1:1) + 2% triethylamine to obtain <b>29</b> as a pale yellow solid in 54% yield. <sup>1</sup>H NMR (600 MHz, deuterium oxide): δ = 7.90 (d, <i>J</i> = 8.5 Hz, 1H), 7.75 (d, <i>J</i> = 7.7 Hz, 1H), 7.30 (t, <i>J</i> = 8.0 Hz, 1H), 6.58 (s, 1H), 3.90–3.82 (m, 1H), 3.17 (q, <i>J</i> = 7.2 Hz, 4H), 2.96–2.91 (m, 2H), 2.49 (s, 3H), 1.23 (d, <i>J</i> = 6.3 Hz, 3H), 1.15 (t, <i>J</i> = 7.3 Hz, 6H), 1.06–1.01 (m, 4H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d<sub>6</sub></i>): δ = 160.90, 150.68, 144.75, 131.99, 130.01, 123.74, 121.57, 119.50, 99.90, 52.34, 49.22, 48.27 (2C), 33.93, 25.84, 23.08, 20.47, 11.44 (2C). HRMS (ESI+): <i>m/z</i> calculated 334.2050 for C<sub>19</sub>H<sub>29</sub>ClN<sub>3</sub>, found 334.2055 ([M + H]<sup>+</sup>).</div><div class="NLM_p">4-((8-Chloro-2-methylquinolin-4-yl)amino)-2-((diethylamino)methyl)phenol (<b>30</b>). 4,8-Dichloro-2-methylquinoline (<b>34</b>, 0.21 g, 1.0 mmol, 1.0 equiv), 4-amino-2-(diethylamino)methyl)phenol dihydrochloride (<b>36</b>, 293 mg, 1.1 mmol, 1.1 equiv), and a catalytic amount of potassium iodide were dissolved in EtOH (30 mL), aqueous hydrochloric acid (0.2 mL, 2 N) was added dropwise, and the mixture was stirred under reflux for 20 h. After cooling to room temperature, the mixture was filtered, and the filtrate was concentrated under reduced pressure. The crude product was washed with cold EtOH to obtain <b>30</b> as a yellow solid in 51% yield. <sup>1</sup>H NMR (500 MHz, deuterium oxide): δ = 8.18 (dd, <i>J</i> = 8.6, 1.2 Hz, 1H), 7.99 (dd, <i>J</i> = 7.7, 1.2 Hz, 1H), 7.60 (t, 1H), 7.46–7.41 (m, 2H), 7.19–7.14 (m, 1H), 6.67 (s, 1H), 4.37 (s, 2H), 3.30 (dq, <i>J</i> = 10.1, 7.2 Hz, 4H), 2.62 (s, 3H), 1.38 (t, <i>J</i> = 7.3 Hz, 6H). <sup>13</sup>C NMR (126 MHz, Deuterium Oxide): δ = 156.20, 156.16, 155.95, 135.40, 134.52, 130.28, 129.90, 129.18, 127.36, 124.08, 121.94, 118.78, 117.82, 117.66, 101.72, 52.20, 48.36 (2C), 20.68, 8.83 (2C). HRMS (MALDI): <i>m/z</i> calculated 370.16807 for C<sub>21</sub>H<sub>25</sub>ClN<sub>3</sub>O, found 370.16737 ([M + H]<sup>+</sup>).</div><div class="NLM_p">8-Chloro-<i>N</i>-isopropyl-2-methylquinolin-4-amine (<b>31</b>). 4,8-Dichloro-2-methylquinoline (<b>34</b>, 0.11 g, 0.50 mmol, 1.0 equiv) and propan-2-amine (<b>38</b>, 51 μL, 0.60 mmol, 1.2 equiv) were dissolved in EtOH (6.0 mL), and the mixture was stirred for 36 h at 140 °C under microwave irradiation. The solvent was evaporated in vacuum, aqueous sodium hydroxide solution (20 mL) and ethyl acetate (20 mL) were added, phases were separated, and the aqueous layer was extracted two times with ethyl acetate (2x 20 mL). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, and the solvent was evaporated in vacuum. The crude product was purified by column chromatography (hexane/ethyl acetate 5:1 + 2% NEt<sub>3</sub> and hexane/ethyl acetate 1:1 + 2% NEt<sub>3</sub>) to obtain the title compound as a yellow oil (15 mg, 13%). <sup>1</sup>H NMR (500 MHz, Acetone-<i>d<sub>6</sub></i>): δ = 8.05 (dd, <i>J</i> = 8.5, 1.3 Hz, 1H), 7.70 (dd, <i>J</i> = 7.4, 1.3 Hz, 1H), 7.29–7.22 (m, 1H), 6.54 (s, 1H), 6.14 (d, <i>J</i> = 6.9 Hz, 1H), 4.01–3.90 (m, 1H), 2.54 (s, 3H), 1.34 (d, <i>J</i> = 6.4 Hz, 6H). <sup>13</sup>C NMR (126 MHz, acetone-<i>d<sub>6</sub></i>): δ = 160.73, 150.50, 145.82, 133.84, 129.73, 123.57, 120.86, 120.12, 100.33, 44.81, 25.92, 22.31 (2C). HRMS (MALDI): <i>m/z</i> calculated 235.09965 for C<sub>13</sub>H<sub>16</sub>ClN<sub>2</sub>, found 235.10002 ([M + H]<sup>+</sup>).</div><div class="NLM_p"><i>N</i>-Butyl-8-chloro-2-methylquinolin-4-amine (<b>32</b>). 4,8-Dichloro-2-methylquinoline (<b>34</b>, 0.11 g, 0.50 mmol, 1.0 equiv) and butan-1-amine (<b>39</b>, 73 mg, 1.0 mmol, 2.0 equiv) were dissolved in EtOH (6.0 mL), and the mixture was stirred for 48 h at 140 °C under microwave irradiation. The solvent was evaporated in vacuum, and the crude product was purified by column chromatography (hexane/ethyl acetate 5:1 + 2% NEt<sub>3</sub>) to obtain the title compound as a yellow oil (30 mg, 24%). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ = 7.71–7.64 (m, 2H), 7.20 (t, <i>J</i> = 8.0 Hz, 1H), 6.26 (s, 1H), 3.27–3.22 (m, 2H), 2.62 (s, 3H), 1.72–1.65 (m, 2H), 1.48–1.40 (m, 2H), 0.94 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ = 159.86, 150.46, 129.63, 123.57, 118.83, 118.61, 99.63, 43.18, 30.92, 25.76, 20.34, 13.85. HRMS (MALDI): <i>m/z</i> calculated 249.11530 for C<sub>14</sub>H<sub>18</sub>ClN<sub>2</sub>, found 249.11583 ([M + H]<sup>+</sup>).</div><div class="NLM_p last">8-Chloro-2-methyl-<i>N</i>-(3-methylbutan-2-yl)quinolin-4-amine (<b>33</b>). 4,8-Dichloro-2-methylquinoline (<b>34</b>, 0.11 g, 0.50 mmol, 1.0 equiv) and 3-methylbutan-2-amine (<b>40</b>, 87 mg, 1.0 mmol, 2.0 equiv) were dissolved in EtOH (6.0 mL), and the mixture was stirred for 48 h at 140 °C under microwave irradiation. The solvent was evaporated in vacuum, aqueous sodium hydroxide solution (20 mL) and ethyl acetate (20 mL) were added, phases were separated, and the aqueous layer was extracted two times with ethyl acetate (2x 20 mL). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, and the solvent was evaporated in vacuum. The crude product was purified by column chromatography (hexane/ethyl acetate 5:1 + 2% NEt<sub>3</sub> and hexane/ethyl acetate 1:1 + 2% NEt<sub>3</sub>) to obtain the title compound as a green oil (6 mg, 4%). <sup>1</sup>H NMR (500 MHz, MeOD): δ = 8.08 (dd, <i>J</i> = 8.5, 1.2 Hz, 1H), 7.70 (dd, <i>J</i> = 7.5, 1.2 Hz, 1H), 7.31–7.26 (m, 1H), 6.49 (s, 1H), 3.57 (quint., <i>J</i> = 6.7 Hz, 1H), 2.53 (s, 3H), 1.95–1.84 (m, <i>J</i> = 6.8, 1H), 1.19 (d, <i>J</i> = 6.6, 3H), 0.94–0.91 (m, 6H). <sup>13</sup>C NMR (126 MHz, MeOD): δ = 159.03, 152.19, 130.57, 123.81, 120.37, 118.63, 99.28, 54.29, 32.47, 22.35, 18.62, 17.36, 15.44. HRMS (MALDI): <i>m/z</i> calculated 263.13095 for C<sub>15</sub>H<sub>20</sub>ClN<sub>2</sub>, found 263.13157 ([M + H]<sup>+</sup>).</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Hybrid Gal4-Nurr1 Reporter Gene Assay</h3><div class="NLM_p last"><i>Plasmids.</i> The Gal4-fusion receptor plasmid pFA-CMV-hNURR1-LBD<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> coding for the hinge region and LBD of the canonical isoform of human Nurr1 has been reported previously. The Gal4-VP16<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> fusion protein expressed from plasmid pECE-SV40-Gal4-VP16<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> (Addgene, entry 71728, Watertown, MA, USA) served as a ligand-independent transcriptional inducer for control experiments. pFR-Luc (Stratagene, La Jolla, CA, USA) was used as a reporter plasmid and pRL-SV40 (Promega, Madison, WI, USA) for normalization of transfection efficiency and test compound toxicity. <i>Assay procedure.</i> HEK293T cells were grown in DMEM high glucose, supplemented with 10% fetal calf serum (FCS), sodium pyruvate (1 mM), penicillin (100 U/mL), and streptomycin (100 μg/mL) at 37 °C and 5% CO<sub>2</sub>. The day before transfection, HEK293T cells were seeded in 96-well plates (3 × 10<sup>4</sup> cells/well). The medium was changed to Opti-MEM without supplements right before transfection. Transient transfection was performed using the Lipofectamine LTX reagent (Invitrogen, Carlsbad, CA, USA) according to the manufacturer’s protocol with pFR-Luc (Stratagene), pRL-SV40 (Promega), and the corresponding Gal4-fusion nuclear receptor plasmid pFA-CMV-hNR-LBD. Five hours after transfection, the medium was changed to Opti-MEM supplemented with penicillin (100 U/mL) and streptomycin (100 μg/mL), now additionally containing 0.1% DMSO and the respective test compound or 0.1% DMSO alone as untreated control. Each concentration was tested in duplicates, and each experiment was performed independently at least three times. The Gal4-VP16 control experiments were carried out in duplicates as well, with at least four independent repeats. Following overnight (12–14 h) incubation with the test compounds, the cells were assayed for luciferase activity using the Dual-Glo<sup>TM</sup> Luciferase Assay System (Promega) according to the manufacturer’s protocol. Luminescence was measured with a Spark 10 M luminometer (Tecan Group AG, Männedorf, Switzerland). Normalization of transfection efficiency and cell growth were done by division of firefly luciferase data by renilla luciferase data and multiplying the value by 1000 resulting in relative light units (RLU). Fold activation was obtained by dividing the mean RLU of a test compound at a respective concentration by the mean RLU of untreated control. The hybrid assay was validated with amodiaquine and chloroquine as reference agonists, which yielded EC<sub>50</sub> values in agreement with the literature. For dose–response curve fitting and calculation of EC<sub>50</sub>/IC<sub>50</sub> values, the equations “[Agonist]/[Inhibitor] vs. response – variable slope (four parameters)” were performed with mean fold activations ± SD using GraphPad Prism (version 7.00, GraphPad Software, La Jolla, CA, USA).</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Full-Length Nurr1 Reporter Gene Assays</h3><div class="NLM_p last"><i>Plasmids:</i> The reporter plasmids pFR-Luc-NBRE,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> pFR-Luc-NurRE,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> and pFR-Luc-DR5<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> each containing one copy of the respective human Nurr1 response element NBRE Nl3 (TGATATCGAAAACA<u class="uu">AAAGGTCA</u>), NurRE (from POMC; <u class="uu">TGATATTTACCTCCAAATGCCA</u>), or DR5 (TGATA<u class="uu">GGTTCACCGAAAGGTCA</u>) were described previously. The full length human nuclear receptor Nurr1 (pcDNA3.1-hNurr1-NE; Addgene, entry 102363) and, for DR5, RXRα (pSG5-hRXR<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a>) were overexpressed. pFL-SV40 (Promega) was used for normalization of transfection efficacy and evaluation of compound toxicity. <i>Assay procedure</i>: HEK293T cells were grown in DMEM high glucose, supplemented with 10% FCS, sodium pyruvate (1 mM), penicillin (100 U/mL), and streptomycin (100 μg/mL) at 37 °C and 5% CO<sub>2</sub>. The day before transfection, HEK293T cells were seeded in 96-well plates (3 × 10<sup>4</sup> cells/well). The medium was changed to Opti-MEM without supplements right before transfection. Transient transfection was performed using the Lipofectamine LTX reagent (Invitrogen) according to the manufacturer’s protocol with pFR-Luc-NBRE,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> pFR-Luc-NurRE<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> or pFR-Luc-DR5,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> pRL-SV40 (Promega), the human full length receptor plasmid pcDNA3.1-hNurr1-NE, and, for DR5, also pSG5-hRXR.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Five hours after transfection, the medium was changed to Opti-MEM supplemented with penicillin (100 U/mL) and streptomycin (100 μg/mL), now additionally containing 0.1% DMSO and the respective test compound or 0.1% DMSO alone as untreated control. For full dose–response characterization, each concentration was tested in duplicates and each experiment was performed independently at least three times. Following overnight (12–14 h) incubation with the test compounds, the cells were assayed for luciferase activity using a Dual-Glo<sup>TM</sup> Luciferase Assay System (Promega) according to the manufacturer’s protocol. Luminescence was measured with a Spark 10 M luminometer (Tecan Group AG). Normalization of transfection efficiency and cell growth were done by division of firefly luciferase data by renilla luciferase data and multiplying the value by 1000 resulting in RLU. Fold activation was obtained by dividing the mean RLU of a test compound at a respective concentration by the mean RLU of untreated control. The full length Nurr1 reporter gene assays were validated with amodiaquine and chloroquine as reference agonists.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Nurr1 Co-Regulator Recruitment Assays</h3><div class="NLM_p last">Interaction of co-regulator peptides to the Nurr1-LBD was studied in a homogeneous time-resolved fluorescence resonance energy transfer (HT-FRET) assay system. Terbium cryptate as streptavidin conjugate (Tb-SA; Cisbio Bioassays, Codolet, France) was used as FRET donor for stable coupling to biotinylated co-regulator peptides NCoR1 or NCoR2. As FRET acceptor, recombinant Nurr1-LBD protein<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> fused to sGFP was used, which has been reported previously. GFP-Nurr1-LBD was titrated against biotinylated NCoR1 (1 nM) or NCoR2 (18 nM) copeptide and Tb-SA (2 or 12 nM, respectively) in the presence of a fixed concentration (20 or 100 μM, in assay buffer containing 1% DMSO) of the respective ligand or 1% DMSO. To maintain a constant GFP concentration, free GFP protein was added to the dilution series. The experiments were performed in HTRF assay buffer (150 mM KF, 25 mM HEPES pH 7.5 (KOH), 5% (w/v) Glycerol, supplemented with 0.1% (w/v) CHAPS and 5 mM DTT) with 1% DMSO in an assay volume of 20 μL. All HTRF experiments were carried out in a 384-well format using white flat bottom polystyrol microtiter plates (Greiner Bio-One, Frickenhausen, Germany), and each concentration was tested in technical triplicates. After 1 h incubation at room temperature, fluorescence intensities after excitation at 340 nm were recorded at 520 nm for GFP acceptor fluorescence and 620 nm for Tb-SA donor fluorescence on a SPARK plate reader (Tecan Group Ltd.). FI520nm was divided by FI620nm and multiplied with 10,000 to give a dimensionless HTRF signal. ΔHTRFs were calculated as differences between each individual HTFR value and the corresponding untreated control (1% DMSO) of the same dissolution series. For the dose–response curve fitting, the equation “[Agonist] vs. response – variable slope (four parameters)” was performed with three replicate values of ΔHTRF using GraphPad Prism (version 7.00, GraphPad Software). The co-regulator peptides were purchased from Eurogentec (Seraing, Belgium), and sequences were the following: nuclear receptor co-repressor 1 (NCoR1) nuclear receptor-interaction domain 1 (ID1), Biotin-GMGQVPRTHRLITLADHICQIITQDFARN-COOH; and NCoR2 ID2, Biotin-SQAVQEHASTNMGLEAIIRKALMGKYDQW-COOH.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Nurr1 Homodimerization Assay</h3><div class="NLM_p last">Modulation of Nurr1 LBD homodimerization was studied in an HT-FRET assay setup using the biotinylated recombinant Nurr1 LBD<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> and GFP-Nurr1 LBD. Assay solutions were prepared in HTRF assay buffer supplemented with 0.1% (w/v) CHAPS and 5 mM DTT as well as 1% DMSO with test compounds <b>24</b> (100 μM) and <b>26</b> (20 or 100 μM) or DMSO alone as negative control. The biotinylated Nurr1 LBD (0.375 nM) and Tb-SA (0.75 nM) served as the FRET donor complex, which was kept constant, while the GFP-coupled protein as the FRET acceptor was varied in concentration. Titration of the GFP-Nurr1 LBD started at 500 nM, and each concentration was tested in technical triplicates. Accordingly, free GFP was added to keep the total GFP content stable throughout the entire series in order to suppress artifacts from changes in the degree of diffusion-enhanced FRET. The samples were equilibrated at room temperature for 2 h before FI520nm and FI620nm were recorded after excitation at 340 nm, and the HTRF signal, ΔHTRF, and dose–response curves were calculated as described above.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Quantification of Nurr1-Regulated mRNA Expression in T98G Cells by qRT-PCR</h3><div class="NLM_p last">T98G cells were grown in 6-well plates (2.5 × 10<sup>5</sup> cells/well) in DMEM high glucose, supplemented with 10% FCS, sodium pyruvate (1 mM), penicillin (100 U/mL), and streptomycin (100 μg/mL) at 37 °C and 5% CO<sub>2</sub>. Before incubation with test compounds, the medium was changed to DMEM supplemented with 1% charcoal-stripped FCS, sodium pyruvate (1 mM), penicillin (100 U/mL), and streptomycin (100 μg/mL) for 24 h. For gene expression analysis, the cells were incubated with <b>2</b> (50 μM), <b>24</b> (10 or 30 μM), <b>26</b> (10 or 30 μM), or 0.1% DMSO as untreated control for 8 h. The cells were then harvested and directly used for RNA extraction. Total RNA (3 μg) was extracted from T98G cells by the E.Z.N.A. Total RNA Kit I (R6834–02, Omega Bio-Tek, Inc., Norcross, GA). RNA was reverse-transcribed into cDNA using the High-Capacity RNA-to-cDNA Kit (4387406, Thermo Fischer Scientific, Inc.) according to the manufacturer’s protocol. Nurr1 target gene expression was evaluated by qRT-PCR analysis with a StepOnePlus System (Life Technologies, Carlsbad, CA) using Power SYBR Green (Life Technologies; 12.5 μL/well). Each sample was set up in duplicates and repeated in four independent experiments. The expression was quantified by the comparative 2<sup>-ΔΔCt</sup> method, and glyceraldehyde 3-phosphate dehydrogenase (GAPDH) served as the reference gene. Primer sequences for Nurr1 target genes vesicular monoamine transporter 2 (VMAT2) and tyrosine hydroxylase (TH) were obtained from OriGene (OriGene Technologies Inc., Rockville, MD, USA). The following PCR primers were used: hGAPDH: 5′-ATA TGA TTC CAC CCA TGG CA (fw), 5′-GAT GAT GAC CCT TTT GGC TC (rev), hVMAT2: 5′-GCT ATG CCT TCC TGC TGA TTG C (fw), 5′-CCA AGG CGA TTC CCA TGA CGT T (rev), and hTH: 5′-GCT GGA CAA GTG TCA TCA CCT G (fw), and 5′-CCT GTA CTG GAA GGC GAT CTC A (rev).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01779" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87213" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87213" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01779?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01779</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Control experiments; nonspecific effects and cytotoxicity; raw luminescence data; effects of Nurr1 agonist 24; summarized cellular Nurr1 modulatory activities; and WST-1 toxicity assay (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01779/suppl_file/jm0c01779_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">HPLC traces of compounds synthesized in this study (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01779/suppl_file/jm0c01779_si_002.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline"><b>Molecular formula strings</b> with structures and activity data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01779/suppl_file/jm0c01779_si_003.csv" class="ext-link">csv</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01779/suppl_file/jm0c01779_si_001.pdf">jm0c01779_si_001.pdf (916.63 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01779/suppl_file/jm0c01779_si_002.pdf">jm0c01779_si_002.pdf (893.9 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01779/suppl_file/jm0c01779_si_003.csv">jm0c01779_si_003.csv (1.76 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c01779" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17443" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17443" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniel Merk</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Chemistry, Goethe University, Frankfurt, Max-von-Laue-Str. 9, D-60438 Frankfurt, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5359-8128" title="Orcid link">http://orcid.org/0000-0002-5359-8128</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#92fff7e0f9d2e2faf3e0fff1faf7ffbce7fcfbbff4e0f3fcf9f4e7e0e6bcf6f7"><span class="__cf_email__" data-cfemail="87eae2f5ecc7f7efe6f5eae4efe2eaa9f2e9eeaae1f5e6e9ece1f2f5f3a9e3e2">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sabine Willems</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Chemistry, Goethe University, Frankfurt, Max-von-Laue-Str. 9, D-60438 Frankfurt, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Julia Ohrndorf</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Chemistry, Goethe University, Frankfurt, Max-von-Laue-Str. 9, D-60438 Frankfurt, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Whitney Kilu</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Chemistry, Goethe University, Frankfurt, Max-von-Laue-Str. 9, D-60438 Frankfurt, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jan Heering</span> - <span class="hlFld-Affiliation affiliation">Fraunhofer
Institute for Translational Medicine and Pharmacology ITMP, Theodor-Stern-Kai 7, D-60596 Frankfurt am Main, Germany</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18157" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18157" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This research was financially supported by the Aventis Foundation (Life Science Bridge Award to D.M.) and the LOEWE center Translational Medicine and Pharmacology (TMP). Gal4-VP16 was a gift from Lea Sistonen (Addgene plasmid # 71728 ; http://n2t.net/addgene:71728 ; RRID:Addgene_71728). pcDNA3.1-hNurr1-NE was a gift from Shu Leong Ho (Addgene plasmid # 102363 ; http://n2t.net/addgene:102363 ; RRID:Addgene_102363).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">AQ</td><td class="NLM_def"><p class="first last">amodiaquine</p></td></tr><tr><td class="NLM_term">CQ</td><td class="NLM_def"><p class="first last">chloroquine</p></td></tr><tr><td class="NLM_term">DR5</td><td class="NLM_def"><p class="first last">direct repeats spaced by five nucleotides</p></td></tr><tr><td class="NLM_term">FRET</td><td class="NLM_def"><p class="first last">fluorescence resonance energy transfer</p></td></tr><tr><td class="NLM_term">GFP</td><td class="NLM_def"><p class="first last">green fluorescent protein</p></td></tr><tr><td class="NLM_term">HTRF</td><td class="NLM_def"><p class="first last">homogeneous time-resolved fluorescence resonance energy transfer</p></td></tr><tr><td class="NLM_term">LBD</td><td class="NLM_def"><p class="first last">ligand binding domain</p></td></tr><tr><td class="NLM_term">LE</td><td class="NLM_def"><p class="first last">ligand efficiency</p></td></tr><tr><td class="NLM_term">LLE</td><td class="NLM_def"><p class="first last">lipophilic ligand efficiency</p></td></tr><tr><td class="NLM_term">NBRE</td><td class="NLM_def"><p class="first last">NGFI-B response element</p></td></tr><tr><td class="NLM_term">NCoR1</td><td class="NLM_def"><p class="first last">nuclear receptor co-repressor 1</p></td></tr><tr><td class="NLM_term">NCoR2</td><td class="NLM_def"><p class="first last">nuclear receptor co-repressor 2</p></td></tr><tr><td class="NLM_term">Nurr1</td><td class="NLM_def"><p class="first last">nuclear receptor related 1</p></td></tr><tr><td class="NLM_term">NurRE</td><td class="NLM_def"><p class="first last">Nur-response element</p></td></tr><tr><td class="NLM_term">PD</td><td class="NLM_def"><p class="first last">Parkinson’s Disease</p></td></tr><tr><td class="NLM_term">qRT-PCR</td><td class="NLM_def"><p class="first last">quantitative real-time polymerase chain reaction</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">SILE</td><td class="NLM_def"><p class="first last">size-independent ligand efficiency</p></td></tr><tr><td class="NLM_term">TH</td><td class="NLM_def"><p class="first last">tyrosine hydroxylase</p></td></tr><tr><td class="NLM_term">VMAT2</td><td class="NLM_def"><p class="first last">vesicular monoamine transporter 2</p></td></tr><tr><td class="NLM_term">VP16</td><td class="NLM_def"><p class="first last">Herpes simplex virus protein vmw65</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57548" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57548" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 23 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benoit, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aarnisalo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perlmann, T.</span></span> <span> </span><span class="NLM_article-title">Structure and Function of Nurr 1 Identifies a Class of Ligand-Independent Nuclear Receptors</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>423</i></span>,  <span class="NLM_fpage">555</span>– <span class="NLM_lpage">560</span>, <span class="refDoi"> DOI: 10.1038/nature01645</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1038%2Fnature01645" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=12774125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BD3sXktVygtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=423&publication_year=2003&pages=555-560&author=Z.+Wangauthor=G.+Benoitauthor=J.+Liuauthor=S.+Prasadauthor=P.+Aarnisaloauthor=X.+Liuauthor=H.+Xuauthor=N.+P.+Walkerauthor=T.+Perlmann&title=Structure+and+Function+of+Nurr+1+Identifies+a+Class+of+Ligand-Independent+Nuclear+Receptors&doi=10.1038%2Fnature01645"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and function of Nurr1 identifies a class of ligand-independent nuclear receptors</span></div><div class="casAuthors">Wang, Zhulun; Benoit, Gerard; Liu, Jinsong; Prasad, Srividya; Aarnisalo, Piia; Liu, Xiaohong; Xu, Haoda; Walker, Nigel P. C.; Perlmann, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">423</span>
        (<span class="NLM_cas:issue">6939</span>),
    <span class="NLM_cas:pages">555-560</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Members of the nuclear receptor (NR) superfamily of transcription factors modulate gene transcription in response to small lipophilic mols.  Transcriptional activity is regulated by ligands binding to the carboxy-terminal ligand-binding domains (LBDs) of cognate NRs.  A subgroup of NRs referred to as 'orphan receptors' lack identified ligands, however, raising issues about the function of their LBDs.  Here we report the crystal structure of the LBD of the orphan receptor Nurr1 at 2.2 Å resoln.  The Nurr1 LBD adopts a canonical protein fold resembling that of agonist-bound, transcriptionally active LBDs in NRs, but the structure has two distinctive features.  First, the Nurr1 LBD contains no cavity as a result of the tight packing of side chains from several bulky hydrophobic residues in the region normally occupied by ligands.  Second, Nurr1 lacks a 'classical' binding site for coactivators.  Despite these differences, the Nurr1 LBD can be regulated in mammalian cells.  Notably, transcriptional activity is correlated with the Nurr1 LBD adopting a more stable conformation.  Our findings highlight a unique structural class of NRs and define a model for ligand-independent NR function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrf_Dsl30OicrVg90H21EOLACvtfcHk0ljhdmuKfNi83A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXktVygtb4%253D&md5=d19448447ffb618404ae84e4aa37c505</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnature01645&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature01645%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DBenoit%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DPrasad%26aufirst%3DS.%26aulast%3DAarnisalo%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DWalker%26aufirst%3DN.%2BP.%26aulast%3DPerlmann%26aufirst%3DT.%26atitle%3DStructure%2520and%2520Function%2520of%2520Nurr%25201%2520Identifies%2520a%2520Class%2520of%2520Ligand-Independent%2520Nuclear%2520Receptors%26jtitle%3DNature%26date%3D2003%26volume%3D423%26spage%3D555%26epage%3D560%26doi%3D10.1038%2Fnature01645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Decressac, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volakakis, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Björklund, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perlmann, T.</span></span> <span> </span><span class="NLM_article-title">NURR1 in Parkinson Disease—From Pathogenesis to Therapeutic Potential</span>. <i>Nat. Rev. Neurol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">629</span>– <span class="NLM_lpage">636</span>, <span class="refDoi"> DOI: 10.1038/nrneurol.2013.209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1038%2Fnrneurol.2013.209" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=24126627" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ChsrjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=629-636&author=M.+Decressacauthor=N.+Volakakisauthor=A.+Bj%C3%B6rklundauthor=T.+Perlmann&title=NURR1+in+Parkinson+Disease%E2%80%94From+Pathogenesis+to+Therapeutic+Potential&doi=10.1038%2Fnrneurol.2013.209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">NURR1 in Parkinson disease-from pathogenesis to therapeutic potential</span></div><div class="casAuthors">Decressac, Mickael; Volakakis, Nikolaos; Bjoerklund, Anders; Perlmann, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Neurology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">629-636</span>CODEN:
                <span class="NLM_cas:coden">NRNACP</span>;
        ISSN:<span class="NLM_cas:issn">1759-4758</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In Parkinson disease (PD), affected midbrain dopamine (DA) neurons lose specific dopaminergic properties before the neurons die.  How the phenotype of DA neurons is normally established and the ways in which pathol. affects the maintenance of cell identity are, therefore, important considerations.  Orphan nuclear receptor NURR1 (NURR1, also known as NR4A2) is involved in the differentiation of midbrain DA neurons, but also has an important role in the adult brain.  Emerging evidence indicates that impaired NURR1 function might contribute to the pathogenesis of PD: NURR1 and its transcriptional targets are downregulated in midbrain DA neurons that express high levels of the disease-causing protein α-synuclein.  Clin. and exptl. data indicate that disrupted NURR1 function contributes to induction of DA neuron dysfunction, which is seen in early stages of PD.  The likely involvement of NURR1 in the development and progression of PD makes this protein a potentially interesting target for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKj9zkT4BEiLVg90H21EOLACvtfcHk0ljhdmuKfNi83A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ChsrjJ&md5=0ccc0530d53c1d19ef38e81307b99691</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrneurol.2013.209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrneurol.2013.209%26sid%3Dliteratum%253Aachs%26aulast%3DDecressac%26aufirst%3DM.%26aulast%3DVolakakis%26aufirst%3DN.%26aulast%3DBj%25C3%25B6rklund%26aufirst%3DA.%26aulast%3DPerlmann%26aufirst%3DT.%26atitle%3DNURR1%2520in%2520Parkinson%2520Disease%25E2%2580%2594From%2520Pathogenesis%2520to%2520Therapeutic%2520Potential%26jtitle%3DNat.%2520Rev.%2520Neurol.%26date%3D2013%26volume%3D9%26spage%3D629%26epage%3D636%26doi%3D10.1038%2Fnrneurol.2013.209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, B.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, Q. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sohn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, W.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, E.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naffin-Olivos, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ringe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petsko, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K.-S.</span></span> <span> </span><span class="NLM_article-title">Nuclear Receptor Nurr1 Agonists Enhance Its Dual Functions and Improve Behavioral Deficits in an Animal Model of Parkinson’s Disease</span>. <i>Proc. Natl. Acad. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">8756</span>– <span class="NLM_lpage">8761</span>, <span class="refDoi"> DOI: 10.1073/pnas.1509742112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1073%2Fpnas.1509742112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=26124091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVOrs7%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=8756-8761&author=C.-H.+Kimauthor=B.-S.+Hanauthor=J.+Moonauthor=D.-J.+Kimauthor=J.+Shinauthor=S.+Rajanauthor=Q.+T.+Nguyenauthor=M.+Sohnauthor=W.-G.+Kimauthor=M.+Hanauthor=I.+Jeongauthor=K.-S.+Kimauthor=E.-H.+Leeauthor=Y.+Tuauthor=J.+L.+Naffin-Olivosauthor=C.-H.+Parkauthor=D.+Ringeauthor=H.+S.+Yoonauthor=G.+A.+Petskoauthor=K.-S.+Kim&title=Nuclear+Receptor+Nurr1+Agonists+Enhance+Its+Dual+Functions+and+Improve+Behavioral+Deficits+in+an+Animal+Model+of+Parkinson%E2%80%99s+Disease&doi=10.1073%2Fpnas.1509742112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Nuclear receptor Nurr1 agonists enhance its dual functions and improve behavioral deficits in an animal model of Parkinson's disease</span></div><div class="casAuthors">Kim, Chun-Hyung; Han, Baek-Soo; Moon, Jisook; Kim, Deog-Joong; Shin, Joon; Rajan, Sreekanth; Nguyen, Quoc Toan; Sohn, Mijin; Kim, Won-Gon; Han, Minjoon; Jeong, Inhye; Kim, Kyoung-Shim; Lee, Eun-Hye; Tu, Yupeng; Naffin-Olivos, Jacqueline L.; Park, Chang-Hwan; Ringe, Dagmar; Yoon, Ho Sup; Petsko, Gregory A.; Kim, Kwang-Soo</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">8756-8761</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Parkinson's disease (PD), primarily caused by selective degeneration of midbrain dopamine (mDA) neurons, is the most prevalent movement disorder, affecting 1-2% of the global population over the age of 65.  Currently available pharmacol. treatments are largely symptomatic and lose their efficacy over time with accompanying severe side effects such as dyskinesia.  Thus, there is an unmet clin. need to develop mechanism-based and/or disease-modifying treatments.  Based on the unique dual role of the nuclear orphan receptor Nurr1 for development and maintenance of mDA neurons and their protection from inflammation-induced death, we hypothesize that Nurr1 can be a mol. target for neuroprotective therapeutic development for PD.  Here we show successful identification of Nurr1 agonists sharing an identical chem. scaffold, 4-amino-7-chloroquinoline, suggesting a crit. structure-activity relationship.  In particular, we found that two antimalarial drugs, amodiaquine and chloroquine stimulate the transcriptional function of Nurr1 through phys. interaction with its ligand binding domain (LBD).  Remarkably, these compds. were able to enhance the contrasting dual functions of Nurr1 by further increasing transcriptional activation of mDA-specific genes and further enhancing transrepression of neurotoxic proinflammatory gene expression in microglia.  Importantly, these compds. significantly improved behavioral deficits in 6-hydroxydopamine lesioned rat model of PD without any detectable signs of dyskinesia-like behavior.  These findings offer proof of principle that small mols. targeting the Nurr1 LBD can be used as a mechanism-based and neuroprotective strategy for PD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaWD4RLxZ8nbVg90H21EOLACvtfcHk0lgdjDM5-Pipcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVOrs7%252FL&md5=715bbb21e39675989c84aa2ab51447dc</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1509742112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1509742112%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DC.-H.%26aulast%3DHan%26aufirst%3DB.-S.%26aulast%3DMoon%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DD.-J.%26aulast%3DShin%26aufirst%3DJ.%26aulast%3DRajan%26aufirst%3DS.%26aulast%3DNguyen%26aufirst%3DQ.%2BT.%26aulast%3DSohn%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DW.-G.%26aulast%3DHan%26aufirst%3DM.%26aulast%3DJeong%26aufirst%3DI.%26aulast%3DKim%26aufirst%3DK.-S.%26aulast%3DLee%26aufirst%3DE.-H.%26aulast%3DTu%26aufirst%3DY.%26aulast%3DNaffin-Olivos%26aufirst%3DJ.%2BL.%26aulast%3DPark%26aufirst%3DC.-H.%26aulast%3DRinge%26aufirst%3DD.%26aulast%3DYoon%26aufirst%3DH.%2BS.%26aulast%3DPetsko%26aufirst%3DG.%2BA.%26aulast%3DKim%26aufirst%3DK.-S.%26atitle%3DNuclear%2520Receptor%2520Nurr1%2520Agonists%2520Enhance%2520Its%2520Dual%2520Functions%2520and%2520Improve%2520Behavioral%2520Deficits%2520in%2520an%2520Animal%2520Model%2520of%2520Parkinson%25E2%2580%2599s%2520Disease%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%26date%3D2015%26volume%3D112%26spage%3D8756%26epage%3D8761%26doi%3D10.1073%2Fpnas.1509742112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rajan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toh, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lescar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoo, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beldar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feitosa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petsko, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K.-S.</span></span> <span> </span><span class="NLM_article-title">PGE1 and PGA1 Bind to Nurr1 and Activate Its Transcriptional Function</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">876</span>– <span class="NLM_lpage">886</span>, <span class="refDoi"> DOI: 10.1038/s41589-020-0553-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1038%2Fs41589-020-0553-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=32451509" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVSgsLzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2020&pages=876-886&author=S.+Rajanauthor=Y.+Jangauthor=C.+H.+Kimauthor=W.+Kimauthor=H.+T.+Tohauthor=J.+Jeonauthor=B.+Songauthor=A.+Serraauthor=J.+Lescarauthor=J.+Y.+Yooauthor=S.+Beldarauthor=H.+Yeauthor=C.+Kangauthor=X.+W.+Liuauthor=M.+Feitosaauthor=Y.+Kimauthor=D.+Hwangauthor=G.+Gohauthor=K.+L.+Limauthor=H.+M.+Parkauthor=C.+H.+Leeauthor=S.+F.+Ohauthor=G.+A.+Petskoauthor=H.+S.+Yoonauthor=K.-S.+Kim&title=PGE1+and+PGA1+Bind+to+Nurr1+and+Activate+Its+Transcriptional+Function&doi=10.1038%2Fs41589-020-0553-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">PGE1 and PGA1 bind to Nurr1 and activate its transcriptional function</span></div><div class="casAuthors">Rajan, Sreekanth; Jang, Yongwoo; Kim, Chun-Hyung; Kim, Woori; Toh, Hui Ting; Jeon, Jeha; Song, Bin; Serra, Aida; Lescar, Julien; Yoo, Jun Yeob; Beldar, Serap; Ye, Hong; Kang, Congbao; Liu, Xue-Wei; Feitosa, Melissa; Kim, Yeahan; Hwang, Dabin; Goh, Geraldine; Lim, Kah-Leong; Park, Hye Min; Lee, Choong Hwan; Oh, Sungwhan F.; Petsko, Gregory A.; Yoon, Ho Sup; Kim, Kwang-Soo</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">876-886</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The orphan nuclear receptor Nurr1 is crit. for the development, maintenance and protection of midbrain dopaminergic (mDA) neurons.  Here we show that prostaglandin E1 (PGE1) and its dehydrated metabolite, PGA1, directly interact with the ligand-binding domain (LBD) of Nurr1 and stimulate its transcriptional function.  We also report the crystallog. structure of Nurr1-LBD bound to PGA1 at 2.05 Å resoln.  PGA1 couples covalently to Nurr1-LBD by forming a Michael adduct with Cys566, and induces notable conformational changes, including a 21° shift of the activation function-2 helix (H12) away from the protein core.  Furthermore, PGE1/PGA1 exhibit neuroprotective effects in a Nurr1-dependent manner, prominently enhance expression of Nurr1 target genes in mDA neurons and improve motor deficits in 1-methyl-4-phenyl-1,2,3,6-tetrahydropyridine-lesioned mouse models of Parkinson's disease.  Based on these results, we propose that PGE1/PGA1 represent native ligands of Nurr1 and can exert neuroprotective effects on mDA neurons, via activation of Nurr1's transcriptional function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMia6BJvwVdLVg90H21EOLACvtfcHk0lgdjDM5-Pipcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVSgsLzI&md5=5e726ca7eff0e5ed8216d4d4bc99b4e9</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fs41589-020-0553-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-020-0553-6%26sid%3Dliteratum%253Aachs%26aulast%3DRajan%26aufirst%3DS.%26aulast%3DJang%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DC.%2BH.%26aulast%3DKim%26aufirst%3DW.%26aulast%3DToh%26aufirst%3DH.%2BT.%26aulast%3DJeon%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DB.%26aulast%3DSerra%26aufirst%3DA.%26aulast%3DLescar%26aufirst%3DJ.%26aulast%3DYoo%26aufirst%3DJ.%2BY.%26aulast%3DBeldar%26aufirst%3DS.%26aulast%3DYe%26aufirst%3DH.%26aulast%3DKang%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DX.%2BW.%26aulast%3DFeitosa%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DHwang%26aufirst%3DD.%26aulast%3DGoh%26aufirst%3DG.%26aulast%3DLim%26aufirst%3DK.%2BL.%26aulast%3DPark%26aufirst%3DH.%2BM.%26aulast%3DLee%26aufirst%3DC.%2BH.%26aulast%3DOh%26aufirst%3DS.%2BF.%26aulast%3DPetsko%26aufirst%3DG.%2BA.%26aulast%3DYoon%26aufirst%3DH.%2BS.%26aulast%3DKim%26aufirst%3DK.-S.%26atitle%3DPGE1%2520and%2520PGA1%2520Bind%2520to%2520Nurr1%2520and%2520Activate%2520Its%2520Transcriptional%2520Function%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2020%26volume%3D16%26spage%3D876%26epage%3D886%26doi%3D10.1038%2Fs41589-020-0553-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Bruyn, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Westen, G. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">IJzerman, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stieger, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Witte, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Augustijns, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annaert, P. P.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Identification of Oatp1b1/3 Inhibitors</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">1257</span>– <span class="NLM_lpage">1267</span>, <span class="refDoi"> DOI: 10.1124/mol.112.084152</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1124%2Fmol.112.084152" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=23571415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptlOgtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2013&pages=1257-1267&author=T.+De%0ABruynauthor=G.+J.+P.+Van+Westenauthor=A.+P.+IJzermanauthor=B.+Stiegerauthor=P.+De+Witteauthor=P.+F.+Augustijnsauthor=P.+P.+Annaert&title=Structure-Based+Identification+of+Oatp1b1%2F3+Inhibitors&doi=10.1124%2Fmol.112.084152"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based identification of OATP1B1/3 inhibitors</span></div><div class="casAuthors">De Bruyn, Tom; van Western, Gerard J. P.; IJzerman, Adriaan P.; Stieger, Bruno; de Witte, Peter; Augustijns, Patrick F.; Annaert, Pieter P.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1257-1267</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Several recent studies show that inhibition of the hepatic transport proteins org. anion-transporting polypeptide 1B1 (OATP1B1) and 1B3 (OATP1B3) can result in clin. relevant drug-drug interactions (DDI).  To avoid late-stage development drug failures due to OATP1B-mediated DDI, predictive in vitro and in silico methods should be implemented at an early stage of the drug candidate evaluation process.  In the present study, we first developed a high-throughput in vitro transporter inhibition assay for the OATP1B subfamily.  A total of 2000 compds. were tested as potential modulators of the uptake of the OATP1B substrate sodium fluorescein, in OATP1B1- or 1B3-transfected Chinese hamster ovary cells.  At an equimolar substrate-inhibitor concn. of 10 μM, 212 and 139 mols. were identified as OATP1B1 and OATP1B3 inhibitors, resp. (min. 50% inhibition).  For 69 compds., previously not identified as OATP1B inhibitors, concn.-dependent inhibition was also detd., yielding Ki values ranging from 0.06 to 6.5 μM.  Based on these in vitro data, we subsequently developed a proteochemometrics-based in silico model, which predicted OATP1B inhibitors in the test group (20% of the dataset) with high specificity (86%) and sensitivity (78%).  Moreover, several physicochem. compd. properties and substructures related to OATP1B1/1B3 inhibition or inactivity were identified.  Finally, model performance was prospectively verified with a set of 54 compds. not included in the original dataset.  This validation indicated that 80 and 74% of the compds. were correctly classified for OATP1B1 and OATP1B3 inhibition, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoktLtqKIqWh7Vg90H21EOLACvtfcHk0ljJ0UoARz0Xfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptlOgtr8%253D&md5=bbf1617e8efba061f65b8f643baa5b98</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1124%2Fmol.112.084152&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.112.084152%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BBruyn%26aufirst%3DT.%26aulast%3DVan%2BWesten%26aufirst%3DG.%2BJ.%2BP.%26aulast%3DIJzerman%26aufirst%3DA.%2BP.%26aulast%3DStieger%26aufirst%3DB.%26aulast%3DDe%2BWitte%26aufirst%3DP.%26aulast%3DAugustijns%26aufirst%3DP.%2BF.%26aulast%3DAnnaert%26aufirst%3DP.%2BP.%26atitle%3DStructure-Based%2520Identification%2520of%2520Oatp1b1%252F3%2520Inhibitors%26jtitle%3DMol.%2520Pharmacol.%26date%3D2013%26volume%3D83%26spage%3D1257%26epage%3D1267%26doi%3D10.1124%2Fmol.112.084152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alpert, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jez, J. M.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of Phosphoethanolamine Methyltransferase from Plasmodium Falciparum in Complex with Amodiaquine</span>. <i>Bioorganic Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">4990</span>– <span class="NLM_lpage">4993</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.06.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1016%2Fj.bmcl.2012.06.032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=22771008" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvVSlt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=4990-4993&author=S.+G.+Leeauthor=T.+D.+Alpertauthor=J.+M.+Jez&title=Crystal+Structure+of+Phosphoethanolamine+Methyltransferase+from+Plasmodium+Falciparum+in+Complex+with+Amodiaquine&doi=10.1016%2Fj.bmcl.2012.06.032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of phosphoethanolamine methyltransferase from Plasmodium falciparum in complex with amodiaquine</span></div><div class="casAuthors">Lee, Soon Goo; Alpert, Tara D.; Jez, Joseph M.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4990-4993</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Phosphoethanolamine N-methyltransferase (PMT) is essential for phospholipid biogenesis in the malarial parasite Plasmodium falciparum.  PfPMT catalyzes the triple methylation of phosphoethanolamine to produce phosphocholine, which is then used for phosphatidylcholine synthesis.  Here we describe the 2.0 Å resoln. X-ray crystal structure of PfPMT in complex with amodiaquine.  To better characterize inhibition of PfPMT by amodiaquine, we detd. the IC50 values of a series of aminoquinolines using a direct radiochem. assay.  Both structural and functional analyses provide a possible approach for the development of new small mol. inhibitors of PfPMT.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzsPoLH-UZ2bVg90H21EOLACvtfcHk0ljJ0UoARz0Xfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvVSlt7s%253D&md5=aa252f8ce91a293c83bfb07cdce77670</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.06.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.06.032%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DS.%2BG.%26aulast%3DAlpert%26aufirst%3DT.%2BD.%26aulast%3DJez%26aufirst%3DJ.%2BM.%26atitle%3DCrystal%2520Structure%2520of%2520Phosphoethanolamine%2520Methyltransferase%2520from%2520Plasmodium%2520Falciparum%2520in%2520Complex%2520with%2520Amodiaquine%26jtitle%3DBioorganic%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D4990%26epage%3D4993%26doi%3D10.1016%2Fj.bmcl.2012.06.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ren, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span> <span> </span><span class="NLM_article-title">Discovery and Characterization of Novel, Potent, and Selective Cytochrome P450 2J2 Inhibitors</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">60</span>– <span class="NLM_lpage">71</span>, <span class="refDoi"> DOI: 10.1124/dmd.112.048264</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1124%2Fdmd.112.048264" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=23033255" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2012&pages=60-71&author=S.+Renauthor=J.+Zengauthor=Y.+Meiauthor=J.+Z.+H.+Zhangauthor=S.+F.+Yanauthor=J.+Feiauthor=L.+Chen&title=Discovery+and+Characterization+of+Novel%2C+Potent%2C+and+Selective+Cytochrome+P450+2J2+Inhibitors&doi=10.1124%2Fdmd.112.048264"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1124%2Fdmd.112.048264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.112.048264%26sid%3Dliteratum%253Aachs%26aulast%3DRen%26aufirst%3DS.%26aulast%3DZeng%26aufirst%3DJ.%26aulast%3DMei%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%2BZ.%2BH.%26aulast%3DYan%26aufirst%3DS.%2BF.%26aulast%3DFei%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DL.%26atitle%3DDiscovery%2520and%2520Characterization%2520of%2520Novel%252C%2520Potent%252C%2520and%2520Selective%2520Cytochrome%2520P450%25202J2%2520Inhibitors%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2012%26volume%3D41%26spage%3D60%26epage%3D71%26doi%3D10.1124%2Fdmd.112.048264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Neill, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shone, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maggs, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stocks, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biagini, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bray, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibbons, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berry, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winstanley, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukhtar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonar-Law, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hindley, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bambal, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bates, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gresham, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brigandi, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez-delas-Heras, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gargallo, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, S. A.</span></span> <span> </span><span class="NLM_article-title">Candidate Selection and Preclinical Evaluation of N-Tert-Butyl Isoquine (GSK369796), an Affordable and Effective 4-Aminoquinoline Antimalarial for the 21st Century</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1408</span>– <span class="NLM_lpage">1415</span>, <span class="refDoi"> DOI: 10.1021/jm8012618</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm8012618" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhvFShsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=1408-1415&author=P.+M.+O%E2%80%99Neillauthor=B.+K.+Parkauthor=A.+E.+Shoneauthor=J.+L.+Maggsauthor=P.+Robertsauthor=P.+A.+Stocksauthor=G.+A.+Biaginiauthor=P.+G.+Brayauthor=P.+Gibbonsauthor=N.+Berryauthor=P.+A.+Winstanleyauthor=A.+Mukhtarauthor=R.+Bonar-Lawauthor=S.+Hindleyauthor=R.+B.+Bambalauthor=C.+B.+Davisauthor=M.+Batesauthor=T.+K.+Hartauthor=S.+L.+Greshamauthor=R.+M.+Lawrenceauthor=R.+A.+Brigandiauthor=F.+M.+Gomez-delas-Herasauthor=D.+V.+Gargalloauthor=S.+A.+Ward&title=Candidate+Selection+and+Preclinical+Evaluation+of+N-Tert-Butyl+Isoquine+%28GSK369796%29%2C+an+Affordable+and+Effective+4-Aminoquinoline+Antimalarial+for+the+21st+Century&doi=10.1021%2Fjm8012618"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Candidate Selection and Preclinical Evaluation of N-tert-Butyl Isoquine (GSK369796), An Affordable and Effective 4-Aminoquinoline Antimalarial for the 21st Century</span></div><div class="casAuthors">O'Neill, Paul M.; Park, B. Kevin; Shone, Alison E.; Maggs, James L.; Roberts, Phillip; Stocks, Paul A.; Biagini, Giancarlo A.; Bray, Patrick G.; Gibbons, Peter; Berry, Neil; Winstanley, Peter A.; Mukhtar, Amira; Bonar-Law, Richard; Hindley, Stephen; Bambal, Ramesh B.; Davis, Charles B.; Bates, Martin; Hart, Timothy K.; Gresham, Stephanie L.; Lawrence, Ron M.; Brigandi, Richard A.; Gomez-delas-Heras, Federico M.; Gargallo, Domingo V.; Ward, Stephen A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1408-1415</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">N-tert-Bu isoquine (4) (GSK369796) is a 4-aminoquinoline drug candidate selected and developed as part of a public-private partnership between academics at Liverpool, MMV, and GSK pharmaceuticals.  This mol. was rationally designed based on chem., toxicol., pharmacokinetic, and pharmacodynamic considerations and was selected based on excellent activity against Plasmodium falciparum in vitro and rodent malaria parasites in vivo.  The optimized chem. delivered this novel synthetic quinoline in a two-step procedure from cheap and readily available starting materials.  The mol. has a full industry std. preclin. development program allowing first into humans to proceed.  Employing chloroquine (1) and amodiaquine (2) as comparator mols. in the preclin. plan, the first preclin. dossier of pharmacokinetic, toxicity, and safety pharmacol. has also been established for the 4-aminoquinoline antimalarial class.  These studies have revealed preclin. liabilities that have never translated into the human experience.  This has resulted in the availability of crit. information to other drug development teams interested in developing antimalarials within this class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpD8p09GaCH5bVg90H21EOLACvtfcHk0ljJ_j4mMZ7VvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhvFShsbo%253D&md5=d08b37c74a7393eaefb5b86fd5a265dc</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm8012618&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8012618%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Neill%26aufirst%3DP.%2BM.%26aulast%3DPark%26aufirst%3DB.%2BK.%26aulast%3DShone%26aufirst%3DA.%2BE.%26aulast%3DMaggs%26aufirst%3DJ.%2BL.%26aulast%3DRoberts%26aufirst%3DP.%26aulast%3DStocks%26aufirst%3DP.%2BA.%26aulast%3DBiagini%26aufirst%3DG.%2BA.%26aulast%3DBray%26aufirst%3DP.%2BG.%26aulast%3DGibbons%26aufirst%3DP.%26aulast%3DBerry%26aufirst%3DN.%26aulast%3DWinstanley%26aufirst%3DP.%2BA.%26aulast%3DMukhtar%26aufirst%3DA.%26aulast%3DBonar-Law%26aufirst%3DR.%26aulast%3DHindley%26aufirst%3DS.%26aulast%3DBambal%26aufirst%3DR.%2BB.%26aulast%3DDavis%26aufirst%3DC.%2BB.%26aulast%3DBates%26aufirst%3DM.%26aulast%3DHart%26aufirst%3DT.%2BK.%26aulast%3DGresham%26aufirst%3DS.%2BL.%26aulast%3DLawrence%26aufirst%3DR.%2BM.%26aulast%3DBrigandi%26aufirst%3DR.%2BA.%26aulast%3DGomez-delas-Heras%26aufirst%3DF.%2BM.%26aulast%3DGargallo%26aufirst%3DD.%2BV.%26aulast%3DWard%26aufirst%3DS.%2BA.%26atitle%3DCandidate%2520Selection%2520and%2520Preclinical%2520Evaluation%2520of%2520N-Tert-Butyl%2520Isoquine%2520%2528GSK369796%2529%252C%2520an%2520Affordable%2520and%2520Effective%25204-Aminoquinoline%2520Antimalarial%2520for%2520the%252021st%2520Century%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D1408%26epage%3D1415%26doi%3D10.1021%2Fjm8012618" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, E. S.</span></span> <span> </span><span class="NLM_article-title">Novel Anti-adipogenic Activity of Anti-malarial Amodiaquine Through Suppression of PPARγ Activity</span>. <i>Arch. Pharmacal Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">1336</span>– <span class="NLM_lpage">1343</span>, <span class="refDoi"> DOI: 10.1007/s12272-017-0965-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1007%2Fs12272-017-0965-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=29071567" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslWrtrnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2017&pages=1336-1343&author=T.+H.+Kimauthor=H.+K.+Kimauthor=E.+S.+Hwang&title=Novel+Anti-adipogenic+Activity+of+Anti-malarial+Amodiaquine+Through+Suppression+of+PPAR%CE%B3+Activity&doi=10.1007%2Fs12272-017-0965-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Novel anti-adipogenic activity of anti-malarial amodiaquine through suppression of PPARγ activity</span></div><div class="casAuthors">Kim, Tae Hee; Kim, Hyo Kyeong; Hwang, Eun Sook</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Pharmacal Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1336-1343</span>CODEN:
                <span class="NLM_cas:coden">APHRDQ</span>;
        ISSN:<span class="NLM_cas:issn">0253-6269</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Korea</span>)
        </div><div class="casAbstract">Amodiaquine (AQ) was developed as a selective drug against Plasmodium falciparum malaria infection and has received increasing attention as a therapeutic agent for the treatment of rheumatoid arthritis, Parkinson's disease, and cancer due to its anti-inflammatory, anti-proliferative, and autophagic-lysosomal blockade properties.  As autophagy activation is involved in promoting adipogenic differentiation, the authors examd. whether anti-autophagic AQ affected adipocyte differentiation of 3T3-L1 pre-adipocytes.  AQ dose-dependently and significantly suppressed adipocyte differentiation in conjunction with decreases in lipid droplet formation and expression of adipogenic markers including adiponectin, adipocyte fatty acid-binding protein 2 (aP2), resistin, and leptin.  Although peroxisome proliferator-activated receptor γ (PPARγ) decreases by inhibition of autophagy, AQ treatment did not induce PPARγ degrdn. despite the suppression of autophagolysosomal degrdn.  Instead, AQ suppressed the PPARγ activity to transcriptionally activate the aP2 gene transcription through the selective prevention of nuclear localization of PPARγ.  These results demonstrated the novel anti-adipogenic activity of AQ and identified its underlying mechanism that AQ suppressed adipogenic gene expression and lipid formation by inhibiting nuclear localization of PPARγ in an autophagy-independent manner.  AQ is recommended as a safe and effective anti-obesity drug for controlling overweight and obesity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpX9uBChrseGLVg90H21EOLACvtfcHk0ljJ_j4mMZ7VvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslWrtrnK&md5=886cf5f5dc483d57a46b6a3067d06d82</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1007%2Fs12272-017-0965-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12272-017-0965-3%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DT.%2BH.%26aulast%3DKim%26aufirst%3DH.%2BK.%26aulast%3DHwang%26aufirst%3DE.%2BS.%26atitle%3DNovel%2520Anti-adipogenic%2520Activity%2520of%2520Anti-malarial%2520Amodiaquine%2520Through%2520Suppression%2520of%2520PPAR%25CE%25B3%2520Activity%26jtitle%3DArch.%2520Pharmacal%2520Res.%26date%3D2017%26volume%3D40%26spage%3D1336%26epage%3D1343%26doi%3D10.1007%2Fs12272-017-0965-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beland, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, J. L.</span></span> <span> </span><span class="NLM_article-title">Apoptosis Contributes to the Cytotoxicity Induced by Amodiaquine and Its Major Metabolite N-Desethylamodiaquine in Hepatic Cells</span>. <i>Toxicol. Vitr.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>62</i></span>, <span class="NLM_elocation-id">104669</span> <span class="refDoi"> DOI: 10.1016/j.tiv.2019.104669</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1016%2Fj.tiv.2019.104669" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=31629065" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A280%3ADC%252BB3MnpsFWnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2020&author=Y.+Tangauthor=Q.+Wuauthor=F.+A.+Belandauthor=S.+Chenauthor=J.+L.+Fang&title=Apoptosis+Contributes+to+the+Cytotoxicity+Induced+by+Amodiaquine+and+Its+Major+Metabolite+N-Desethylamodiaquine+in+Hepatic+Cells&doi=10.1016%2Fj.tiv.2019.104669"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Apoptosis contributes to the cytotoxicity induced by amodiaquine and its major metabolite N-desethylamodiaquine in hepatic cells</span></div><div class="casAuthors">Tang Yangshun; Wu Qiangen; Beland Frederick A; Chen Si; Fang Jia-Long</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology in vitro : an international journal published in association with BIBRA</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">104669</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Amodiaquine (ADQ), an antimalarial drug used in endemic areas, has been reported to be associated with liver toxicity; however, the mechanism underlying its hepatoxicity remains unclear.  In this study, we examined the cytotoxicity of ADQ and its major metabolite N-desethylamodiaquine (NADQ) and the effect of cytochrome P450 (CYP)-mediated metabolism on ADQ-induced cytotoxicity.  After a 48-h treatment, ADQ and NADQ caused cytotoxicity and induced apoptosis in HepG2 cells; NADQ was slightly more toxic than ADQ.  ADQ treatment decreased the levels of anti-apoptotic Bcl-2 family proteins, which was accompanied by an increase in the levels of pro-apoptotic Bcl-2 family proteins, indicating that ADQ-induced apoptosis was mediated by the Bcl-2 family.  NADQ treatment markedly increased the phosphorylation of JNK, extracellular signal-regulated kinase (ERK1/2), and p38, indicating that NADQ-induced apoptosis was mediated by MAPK signaling pathways.  Metabolic studies using microsomes obtained from HepG2 cell lines overexpressing human CYPs demonstrated that CYP1A1, 2C8, and 3A4 were the major enzymes that metabolized ADQ to NADQ and that CYP1A2, 1B1, 2C19, and 3A5 also metabolized ADQ, but to a lesser extent.  The cytotoxicity of ADQ was increased in CYP2C8 and 3A4 overexpressing HepG2 cells compared to HepG2/CYP vector cells, confirming that NADQ was more toxic than ADQ.  Moreover, treatment of CYP2C8 and 3A4 overexpressing HepG2 cells with ADQ increased the phosphorylation of JNK, ERK1/2, and p38, but not the expression of Bcl-2 family proteins.  Our findings indicate that ADQ and its major metabolite NADQ induce apoptosis, which is mediated by members of the Bcl-2 family and the activation of MAPK signaling pathways, respectively.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTuv5_RWP5HmyCEBeYuoi7AfW6udTcc2ea53hUbEFpLtLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MnpsFWnuw%253D%253D&md5=733ef335aa42e9a8ec6a8497d0c1d5fd</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.tiv.2019.104669&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tiv.2019.104669%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DQ.%26aulast%3DBeland%26aufirst%3DF.%2BA.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DFang%26aufirst%3DJ.%2BL.%26atitle%3DApoptosis%2520Contributes%2520to%2520the%2520Cytotoxicity%2520Induced%2520by%2520Amodiaquine%2520and%2520Its%2520Major%2520Metabolite%2520N-Desethylamodiaquine%2520in%2520Hepatic%2520Cells%26jtitle%3DToxicol.%2520Vitr.%26date%3D2020%26volume%3D62%26doi%3D10.1016%2Fj.tiv.2019.104669" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouhaddou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obernier, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Meara, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezelj, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaney, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummino, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hüttenhain, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaake, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tutuncuoglu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foussard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batra, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modak, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polacco, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braberg, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fabius, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eckhardt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soucheray, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cakir, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGregor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roesch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vallet, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mac Kain, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miorin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreno, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naing, Z. Z. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirby, I. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnyk, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chorba, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrio-Hernandez, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Memon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernandez-Armenta, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathy, C. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perica, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilla, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganesan, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saltzberg, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rakesh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenthal, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calviello, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkataramanan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liboy-Lugo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wankowicz, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Safari, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ugur, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koh, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savar, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, Q. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shengjuler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Neal, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broadhurst, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klippsten, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharp, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wenzell, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuzuoglu-Ozturk, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trenker, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavero, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hiatt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rathore, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noack, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroud, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frankel, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, O. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verba, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agard, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emerman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Zastrow, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">d’Enfert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malik, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujimori, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ideker, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floor, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sali, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruggero, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taunton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kortemme, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beltrao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vignuzzi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Sastre, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krogan, N. J.</span></span> <span> </span><span class="NLM_article-title">A SARS-CoV-2 Protein Interaction Map Reveals Targets for Drug Repurposing</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>583</i></span>,  <span class="NLM_fpage">459</span>– <span class="NLM_lpage">468</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2286-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1038%2Fs41586-020-2286-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=32353859" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlCqtbrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=583&publication_year=2020&pages=459-468&author=D.+E.+Gordonauthor=G.+M.+Jangauthor=M.+Bouhaddouauthor=J.+Xuauthor=K.+Obernierauthor=K.+M.+Whiteauthor=M.+J.+O%E2%80%99Mearaauthor=V.+V.+Rezeljauthor=J.+Z.+Guoauthor=D.+L.+Swaneyauthor=T.+A.+Tumminoauthor=R.+H%C3%BCttenhainauthor=R.+M.+Kaakeauthor=A.+L.+Richardsauthor=B.+Tutuncuogluauthor=H.+Foussardauthor=J.+Batraauthor=K.+Haasauthor=M.+Modakauthor=M.+Kimauthor=P.+Haasauthor=B.+J.+Polaccoauthor=H.+Brabergauthor=J.+M.+Fabiusauthor=M.+Eckhardtauthor=M.+Soucherayauthor=M.+J.+Bennettauthor=M.+Cakirauthor=M.+J.+McGregorauthor=Q.+Liauthor=B.+Meyerauthor=F.+Roeschauthor=T.+Valletauthor=A.+Mac+Kainauthor=L.+Miorinauthor=E.+Morenoauthor=Z.+Z.+C.+Naingauthor=Y.+Zhouauthor=S.+Pengauthor=Y.+Shiauthor=Z.+Zhangauthor=W.+Shenauthor=I.+T.+Kirbyauthor=J.+E.+Melnykauthor=J.+S.+Chorbaauthor=K.+Louauthor=S.+A.+Daiauthor=I.+Barrio-Hernandezauthor=D.+Memonauthor=C.+Hernandez-Armentaauthor=J.+Lyuauthor=C.+J.+P.+Mathyauthor=T.+Pericaauthor=K.+B.+Pillaauthor=S.+J.+Ganesanauthor=D.+J.+Saltzbergauthor=R.+Rakeshauthor=X.+Liuauthor=S.+B.+Rosenthalauthor=L.+Calvielloauthor=S.+Venkataramananauthor=J.+Liboy-Lugoauthor=Y.+Linauthor=X.+P.+Huangauthor=Y.+Liuauthor=S.+A.+Wankowiczauthor=M.+Bohnauthor=M.+Safariauthor=F.+S.+Ugurauthor=C.+Kohauthor=N.+S.+Savarauthor=Q.+D.+Tranauthor=D.+Shengjulerauthor=S.+J.+Fletcherauthor=M.+C.+O%E2%80%99Nealauthor=Y.+Caiauthor=J.+C.+J.+Changauthor=D.+J.+Broadhurstauthor=S.+Klippstenauthor=P.+P.+Sharpauthor=N.+A.+Wenzellauthor=D.+Kuzuoglu-Ozturkauthor=H.+Y.+Wangauthor=R.+Trenkerauthor=J.+M.+Youngauthor=D.+A.+Caveroauthor=J.+Hiattauthor=T.+L.+Rothauthor=U.+Rathoreauthor=A.+Subramanianauthor=J.+Noackauthor=M.+Hubertauthor=R.+M.+Stroudauthor=A.+D.+Frankelauthor=O.+S.+Rosenbergauthor=K.+A.+Verbaauthor=D.+A.+Agardauthor=M.+Ottauthor=M.+Emermanauthor=N.+Juraauthor=M.+von+Zastrowauthor=E.+Verdinauthor=A.+Ashworthauthor=O.+Schwartzauthor=C.+d%E2%80%99Enfertauthor=S.+Mukherjeeauthor=M.+Jacobsonauthor=H.+S.+Malikauthor=D.+G.+Fujimoriauthor=T.+Idekerauthor=C.+S.+Craikauthor=S.+N.+Floorauthor=J.+S.+Fraserauthor=J.+D.+Grossauthor=A.+Saliauthor=B.+L.+Rothauthor=D.+Ruggeroauthor=J.+Tauntonauthor=T.+Kortemmeauthor=P.+Beltraoauthor=M.+Vignuzziauthor=A.+Garc%C3%ADa-Sastreauthor=K.+M.+Shokatauthor=B.+K.+Shoichetauthor=N.+J.+Krogan&title=A+SARS-CoV-2+Protein+Interaction+Map+Reveals+Targets+for+Drug+Repurposing&doi=10.1038%2Fs41586-020-2286-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">A SARS-CoV-2 protein interaction map reveals targets for drug repurposing</span></div><div class="casAuthors">Gordon, David E.; Jang, Gwendolyn M.; Bouhaddou, Mehdi; Xu, Jiewei; Obernier, Kirsten; White, Kris M.; O'Meara, Matthew J.; Rezelj, Veronica V.; Guo, Jeffrey Z.; Swaney, Danielle L.; Tummino, Tia A.; Huttenhain, Ruth; Kaake, Robyn M.; Richards, Alicia L.; Tutuncuoglu, Beril; Foussard, Helene; Batra, Jyoti; Haas, Kelsey; Modak, Maya; Kim, Minkyu; Haas, Paige; Polacco, Benjamin J.; Braberg, Hannes; Fabius, Jacqueline M.; Eckhardt, Manon; Soucheray, Margaret; Bennett, Melanie J.; Cakir, Merve; McGregor, Michael J.; Li, Qiongyu; Meyer, Bjoern; Roesch, Ferdinand; Vallet, Thomas; Mac Kain, Alice; Miorin, Lisa; Moreno, Elena; Naing, Zun Zar Chi; Zhou, Yuan; Peng, Shiming; Shi, Ying; Zhang, Ziyang; Shen, Wenqi; Kirby, Ilsa T.; Melnyk, James E.; Chorba, John S.; Lou, Kevin; Dai, Shizhong A.; Barrio-Hernandez, Inigo; Memon, Danish; Hernandez-Armenta, Claudia; Lyu, Jiankun; Mathy, Christopher J. P.; Perica, Tina; Pilla, Kala Bharath; Ganesan, Sai J.; Saltzberg, Daniel J.; Rakesh, Ramachandran; Liu, Xi; Rosenthal, Sara B.; Calviello, Lorenzo; Venkataramanan, Srivats; Liboy-Lugo, Jose; Lin, Yizhu; Huang, Xi-Ping; Liu, YongFeng; Wankowicz, Stephanie A.; Bohn, Markus; Safari, Maliheh; Ugur, Fatima S.; Koh, Cassandra; Savar, Nastaran Sadat; Tran, Quang Dinh; Shengjuler, Djoshkun; Fletcher, Sabrina J.; O'Neal, Michael C.; Cai, Yiming; Chang, Jason C. J.; Broadhurst, David J.; Klippsten, Saker; Sharp, Phillip P.; Wenzell, Nicole A.; Kuzuoglu-Ozturk, Duygu; Wang, Hao-Yuan; Trenker, Raphael; Young, Janet M.; Cavero, Devin A.; Hiatt, Joseph; Roth, Theodore L.; Rathore, Ujjwal; Subramanian, Advait; Noack, Julia; Hubert, Mathieu; Stroud, Robert M.; Frankel, Alan D.; Rosenberg, Oren S.; Verba, Kliment A.; Agard, David A.; Ott, Melanie; Emerman, Michael; Jura, Natalia; von Zastrow, Mark; Verdin, Eric; Ashworth, Alan; Schwartz, Olivier; d'Enfert, Christophe; Mukherjee, Shaeri; Jacobson, Matt; Malik, Harmit S.; Fujimori, Danica G.; Ideker, Trey; Craik, Charles S.; Floor, Stephen N.; Fraser, James S.; Gross, John D.; Sali, Andrej; Roth, Bryan L.; Ruggero, Davide; Taunton, Jack; Kortemme, Tanja; Beltrao, Pedro; Vignuzzi, Marco; Garcia-Sastre, Adolfo; Shokat, Kevan M.; Shoichet, Brian K.; Krogan, Nevan J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">583</span>
        (<span class="NLM_cas:issue">7816</span>),
    <span class="NLM_cas:pages">459-468</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: A newly described coronavirus named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which is the causative agent of coronavirus disease 2019 (COVID-19), has infected over 2.3 million people, led to the death of more than 160,000 individuals and caused worldwide social and economic disruption1,2.  There are no antiviral drugs with proven clin. efficacy for the treatment of COVID-19, nor are there any vaccines that prevent infection with SARS-CoV-2, and efforts to develop drugs and vaccines are hampered by the limited knowledge of the mol. details of how SARS-CoV-2 infects cells.  Here we cloned, tagged and expressed 26 of the 29 SARS-CoV-2 proteins in human cells and identified the human proteins that phys. assocd. with each of the SARS-CoV-2 proteins using affinity-purifn. mass spectrometry, identifying 332 high-confidence protein-protein interactions between SARS-CoV-2 and human proteins.  Among these, we identify 66 druggable human proteins or host factors targeted by 69 compds. (of which, 29 drugs are approved by the US Food and Drug Administration, 12 are in clin. trials and 28 are preclin. compds.).  We screened a subset of these in multiple viral assays and found two sets of pharmacol. agents that displayed antiviral activity: inhibitors of mRNA translation and predicted regulators of the sigma-1 and sigma-2 receptors.  Further studies of these host-factor-targeting agents, including their combination with drugs that directly target viral enzymes, could lead to a therapeutic regimen to treat COVID-19.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDvJUIvi-Ws7Vg90H21EOLACvtfcHk0lhxKoDYfA_fyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlCqtbrI&md5=ce8b172ae73aacd3b3d8bb64877f4435</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2286-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2286-9%26sid%3Dliteratum%253Aachs%26aulast%3DGordon%26aufirst%3DD.%2BE.%26aulast%3DJang%26aufirst%3DG.%2BM.%26aulast%3DBouhaddou%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DObernier%26aufirst%3DK.%26aulast%3DWhite%26aufirst%3DK.%2BM.%26aulast%3DO%25E2%2580%2599Meara%26aufirst%3DM.%2BJ.%26aulast%3DRezelj%26aufirst%3DV.%2BV.%26aulast%3DGuo%26aufirst%3DJ.%2BZ.%26aulast%3DSwaney%26aufirst%3DD.%2BL.%26aulast%3DTummino%26aufirst%3DT.%2BA.%26aulast%3DH%25C3%25BCttenhain%26aufirst%3DR.%26aulast%3DKaake%26aufirst%3DR.%2BM.%26aulast%3DRichards%26aufirst%3DA.%2BL.%26aulast%3DTutuncuoglu%26aufirst%3DB.%26aulast%3DFoussard%26aufirst%3DH.%26aulast%3DBatra%26aufirst%3DJ.%26aulast%3DHaas%26aufirst%3DK.%26aulast%3DModak%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DM.%26aulast%3DHaas%26aufirst%3DP.%26aulast%3DPolacco%26aufirst%3DB.%2BJ.%26aulast%3DBraberg%26aufirst%3DH.%26aulast%3DFabius%26aufirst%3DJ.%2BM.%26aulast%3DEckhardt%26aufirst%3DM.%26aulast%3DSoucheray%26aufirst%3DM.%26aulast%3DBennett%26aufirst%3DM.%2BJ.%26aulast%3DCakir%26aufirst%3DM.%26aulast%3DMcGregor%26aufirst%3DM.%2BJ.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DMeyer%26aufirst%3DB.%26aulast%3DRoesch%26aufirst%3DF.%26aulast%3DVallet%26aufirst%3DT.%26aulast%3DMac%2BKain%26aufirst%3DA.%26aulast%3DMiorin%26aufirst%3DL.%26aulast%3DMoreno%26aufirst%3DE.%26aulast%3DNaing%26aufirst%3DZ.%2BZ.%2BC.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DPeng%26aufirst%3DS.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DShen%26aufirst%3DW.%26aulast%3DKirby%26aufirst%3DI.%2BT.%26aulast%3DMelnyk%26aufirst%3DJ.%2BE.%26aulast%3DChorba%26aufirst%3DJ.%2BS.%26aulast%3DLou%26aufirst%3DK.%26aulast%3DDai%26aufirst%3DS.%2BA.%26aulast%3DBarrio-Hernandez%26aufirst%3DI.%26aulast%3DMemon%26aufirst%3DD.%26aulast%3DHernandez-Armenta%26aufirst%3DC.%26aulast%3DLyu%26aufirst%3DJ.%26aulast%3DMathy%26aufirst%3DC.%2BJ.%2BP.%26aulast%3DPerica%26aufirst%3DT.%26aulast%3DPilla%26aufirst%3DK.%2BB.%26aulast%3DGanesan%26aufirst%3DS.%2BJ.%26aulast%3DSaltzberg%26aufirst%3DD.%2BJ.%26aulast%3DRakesh%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DRosenthal%26aufirst%3DS.%2BB.%26aulast%3DCalviello%26aufirst%3DL.%26aulast%3DVenkataramanan%26aufirst%3DS.%26aulast%3DLiboy-Lugo%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWankowicz%26aufirst%3DS.%2BA.%26aulast%3DBohn%26aufirst%3DM.%26aulast%3DSafari%26aufirst%3DM.%26aulast%3DUgur%26aufirst%3DF.%2BS.%26aulast%3DKoh%26aufirst%3DC.%26aulast%3DSavar%26aufirst%3DN.%2BS.%26aulast%3DTran%26aufirst%3DQ.%2BD.%26aulast%3DShengjuler%26aufirst%3DD.%26aulast%3DFletcher%26aufirst%3DS.%2BJ.%26aulast%3DO%25E2%2580%2599Neal%26aufirst%3DM.%2BC.%26aulast%3DCai%26aufirst%3DY.%26aulast%3DChang%26aufirst%3DJ.%2BC.%2BJ.%26aulast%3DBroadhurst%26aufirst%3DD.%2BJ.%26aulast%3DKlippsten%26aufirst%3DS.%26aulast%3DSharp%26aufirst%3DP.%2BP.%26aulast%3DWenzell%26aufirst%3DN.%2BA.%26aulast%3DKuzuoglu-Ozturk%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DH.%2BY.%26aulast%3DTrenker%26aufirst%3DR.%26aulast%3DYoung%26aufirst%3DJ.%2BM.%26aulast%3DCavero%26aufirst%3DD.%2BA.%26aulast%3DHiatt%26aufirst%3DJ.%26aulast%3DRoth%26aufirst%3DT.%2BL.%26aulast%3DRathore%26aufirst%3DU.%26aulast%3DSubramanian%26aufirst%3DA.%26aulast%3DNoack%26aufirst%3DJ.%26aulast%3DHubert%26aufirst%3DM.%26aulast%3DStroud%26aufirst%3DR.%2BM.%26aulast%3DFrankel%26aufirst%3DA.%2BD.%26aulast%3DRosenberg%26aufirst%3DO.%2BS.%26aulast%3DVerba%26aufirst%3DK.%2BA.%26aulast%3DAgard%26aufirst%3DD.%2BA.%26aulast%3DOtt%26aufirst%3DM.%26aulast%3DEmerman%26aufirst%3DM.%26aulast%3DJura%26aufirst%3DN.%26aulast%3Dvon%2BZastrow%26aufirst%3DM.%26aulast%3DVerdin%26aufirst%3DE.%26aulast%3DAshworth%26aufirst%3DA.%26aulast%3DSchwartz%26aufirst%3DO.%26aulast%3Dd%25E2%2580%2599Enfert%26aufirst%3DC.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DJacobson%26aufirst%3DM.%26aulast%3DMalik%26aufirst%3DH.%2BS.%26aulast%3DFujimori%26aufirst%3DD.%2BG.%26aulast%3DIdeker%26aufirst%3DT.%26aulast%3DCraik%26aufirst%3DC.%2BS.%26aulast%3DFloor%26aufirst%3DS.%2BN.%26aulast%3DFraser%26aufirst%3DJ.%2BS.%26aulast%3DGross%26aufirst%3DJ.%2BD.%26aulast%3DSali%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DRuggero%26aufirst%3DD.%26aulast%3DTaunton%26aufirst%3DJ.%26aulast%3DKortemme%26aufirst%3DT.%26aulast%3DBeltrao%26aufirst%3DP.%26aulast%3DVignuzzi%26aufirst%3DM.%26aulast%3DGarc%25C3%25ADa-Sastre%26aufirst%3DA.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DKrogan%26aufirst%3DN.%2BJ.%26atitle%3DA%2520SARS-CoV-2%2520Protein%2520Interaction%2520Map%2520Reveals%2520Targets%2520for%2520Drug%2520Repurposing%26jtitle%3DNature%26date%3D2020%26volume%3D583%26spage%3D459%26epage%3D468%26doi%3D10.1038%2Fs41586-020-2286-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nolan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caraher, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphries, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bettley, H. A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryce, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stratford, I. J.</span></span> <span> </span><span class="NLM_article-title">In Silico Identification and Biochemical Evaluation of Novel Inhibitors of NRH:Quinone Oxidoreductase 2 (NQO2)</span>. <i>Bioorganic Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">7331</span>– <span class="NLM_lpage">7336</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2010.10.070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1016%2Fj.bmcl.2010.10.070" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=21074425" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVKjtb%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=7331-7336&author=K.+A.+Nolanauthor=M.+C.+Caraherauthor=M.+P.+Humphriesauthor=H.+A.+A.+Bettleyauthor=R.+A.+Bryceauthor=I.+J.+Stratford&title=In+Silico+Identification+and+Biochemical+Evaluation+of+Novel+Inhibitors+of+NRH%3AQuinone+Oxidoreductase+2+%28NQO2%29&doi=10.1016%2Fj.bmcl.2010.10.070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">In silico identification and biochemical evaluation of novel inhibitors of NRH:quinone oxidoreductase 2 (NQO2)</span></div><div class="casAuthors">Nolan, Karen A.; Caraher, Mary C.; Humphries, Matthew P.; Abdel-Aal Bettley, Hoda; Bryce, Richard A.; Stratford, Ian J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7331-7336</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The NCI chem. database has been screened using in silico docking to identify novel inhibitors of NRH:quinone oxidoreductase 2 (NQO2).  Compds. identified from the screen exhibit a diverse range of scaffolds and inhibitory potencies are generally in the micromolar range.  Some of the compds. also have the ability to inhibit NQO1.  The modes of binding of the different compds. to the two enzymes are illustrated and discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_GDVj_qQdRLVg90H21EOLACvtfcHk0liDUJD9AnnFNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVKjtb%252FI&md5=62687f38eb146c58c3c8c085fab0d24f</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2010.10.070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2010.10.070%26sid%3Dliteratum%253Aachs%26aulast%3DNolan%26aufirst%3DK.%2BA.%26aulast%3DCaraher%26aufirst%3DM.%2BC.%26aulast%3DHumphries%26aufirst%3DM.%2BP.%26aulast%3DBettley%26aufirst%3DH.%2BA.%2BA.%26aulast%3DBryce%26aufirst%3DR.%2BA.%26aulast%3DStratford%26aufirst%3DI.%2BJ.%26atitle%3DIn%2520Silico%2520Identification%2520and%2520Biochemical%2520Evaluation%2520of%2520Novel%2520Inhibitors%2520of%2520NRH%253AQuinone%2520Oxidoreductase%25202%2520%2528NQO2%2529%26jtitle%3DBioorganic%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D20%26spage%3D7331%26epage%3D7336%26doi%3D10.1016%2Fj.bmcl.2010.10.070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Munoz-Tello, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Vera, I. M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamenecka, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kojetin, D. J.</span></span> <span> </span><span class="NLM_article-title">Assessment of NR4A Ligands That Directly Bind and Modulate the Orphan Nuclear Receptor Nurr1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">15639</span>– <span class="NLM_lpage">15654</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00894</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00894" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisFShsrzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=15639-15654&author=P.+Munoz-Telloauthor=H.+Linauthor=P.+Khanauthor=I.+M.+S.+De+Veraauthor=T.+M.+Kameneckaauthor=D.+J.+Kojetin&title=Assessment+of+NR4A+Ligands+That+Directly+Bind+and+Modulate+the+Orphan+Nuclear+Receptor+Nurr1&doi=10.1021%2Facs.jmedchem.0c00894"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Assessment of NR4A Ligands That Directly Bind and Modulate the Orphan Nuclear Receptor Nurr1</span></div><div class="casAuthors">Munoz-Tello, Paola; Lin, Hua; Khan, Pasha; de Vera, Ian Mitchelle S.; Kamenecka, Theodore M.; Kojetin, Douglas J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">15639-15654</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nurr1/NR4A2 is an orphan nuclear receptor transcription factor implicated as a drug target for neurol. disorders including Alzheimer's and Parkinson's diseases.  Previous studies identified small-mol. NR4A nuclear receptor modulators, but it remains unclear if these ligands affect transcription via direct binding to Nurr1.  We assessed 12 ligands reported to affect NR4A activity for Nurr1-dependent and Nurr1-independent transcriptional effects and the ability to bind the Nurr1 ligand-binding domain (LBD).  Protein NMR structural footprinting data show that amodiaquine, chloroquine, and cytosporone B bind the Nurr1 LBD; ligands that do not bind include C-DIM12, celastrol, camptothecin, IP7e, isoalantolactone, Et 2-[2,3,4-trimethoxy-6-(1-octanoyl)phenyl]acetate (TMPA), and three high-throughput screening hit derivs.  Importantly, ligands that modulate Nurr1 transcription also show Nurr1-independent effects on transcription in a cell type-specific manner, indicating that care should be taken when interpreting the functional response of these ligands in transcriptional assays.  These findings should help focus medicinal chem. efforts that desire to optimize Nurr1-binding ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooBB2VAxtBXrVg90H21EOLACvtfcHk0liDUJD9AnnFNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisFShsrzL&md5=6f76ae9c9cb563d7e4ed80989366acc4</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00894&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00894%26sid%3Dliteratum%253Aachs%26aulast%3DMunoz-Tello%26aufirst%3DP.%26aulast%3DLin%26aufirst%3DH.%26aulast%3DKhan%26aufirst%3DP.%26aulast%3DDe%2BVera%26aufirst%3DI.%2BM.%2BS.%26aulast%3DKamenecka%26aufirst%3DT.%2BM.%26aulast%3DKojetin%26aufirst%3DD.%2BJ.%26atitle%3DAssessment%2520of%2520NR4A%2520Ligands%2520That%2520Directly%2520Bind%2520and%2520Modulate%2520the%2520Orphan%2520Nuclear%2520Receptor%2520Nurr1%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D15639%26epage%3D15654%26doi%3D10.1021%2Facs.jmedchem.0c00894" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Apelt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ligneau, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pertz, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganellin, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schunack, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stark, H.</span></span> <span> </span><span class="NLM_article-title">Development of a New Class of Nonimidazole Histamine H 3 Receptor Ligands with Combined Inhibitory Histamine N-Methyltransferase Activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">1128</span>– <span class="NLM_lpage">1141</span>, <span class="refDoi"> DOI: 10.1021/jm0110845</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0110845" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BD38Xps1aitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=1128-1141&author=J.+Apeltauthor=X.+Ligneauauthor=H.+H.+Pertzauthor=J.+Arrangauthor=C.+R.+Ganellinauthor=J.-C.+Schwartzauthor=W.+Schunackauthor=H.+Stark&title=Development+of+a+New+Class+of+Nonimidazole+Histamine+H+3+Receptor+Ligands+with+Combined+Inhibitory+Histamine+N-Methyltransferase+Activity&doi=10.1021%2Fjm0110845"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a New Class of Nonimidazole Histamine H3 Receptor Ligands with Combined Inhibitory Histamine N-Methyltransferase Activity</span></div><div class="casAuthors">Apelt, Joachim; Ligneau, Xavier; Pertz, Heinz H.; Arrang, Jean-Michel; Ganellin, C. Robin; Schwartz, Jean-Charles; Schunack, Walter; Stark, Holger</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1128-1141</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In search of novel ways to enhance histaminergic neurotransmission in the central nervous system, a new class of nonimidazole histamine H3 receptor ligands were developed that simultaneously possess strong inhibitory activity on the main histamine metabolizing enzyme, histamine N-methyltransferase (HMT).  The novel compds. contain an aminoquinoline moiety, which is an important structural feature for HMT inhibitory activity, connected by different spacers to a piperidino group (for H3 receptor antagonism).  Variation of the spacer structure provides two different series of compds.  One series, having only an alkylene spacer between the basic centers, led to highly potent HMT inhibitors with moderate to high affinity at human histamine H3 receptors.  The second series possesses a p-phenoxypropyl spacer, which may be extended by another alkylene chain.  This latter series also showed strong inhibitory activity on HMT, and in most cases, the H3 receptor affinity even surpassed that of the first series.  One of the most potent compds. with this dual mode of action is 4-(4-(3-piperidinopropoxy)phenylamino)quinoline (hH3, Ki = 0.09 nM; HMT, IC50 = 51 nM).  This class of compds. showed high antagonist potency and good H3 receptor selectivity in functional assays in guinea pig on H1, H2, and H3 receptors.  Because of low or missing in vivo activity of two selected compds., the proof of concept of these valuable pharmacol. tools for the supposed superior overall enhancing effect on histaminergic neurotransmission failed to appear hitherto.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbq6iENBrXJLVg90H21EOLACvtfcHk0lj-X7BUqPxyRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xps1aitA%253D%253D&md5=987730d70810f674eb0173a09591fd44</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjm0110845&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0110845%26sid%3Dliteratum%253Aachs%26aulast%3DApelt%26aufirst%3DJ.%26aulast%3DLigneau%26aufirst%3DX.%26aulast%3DPertz%26aufirst%3DH.%2BH.%26aulast%3DArrang%26aufirst%3DJ.%26aulast%3DGanellin%26aufirst%3DC.%2BR.%26aulast%3DSchwartz%26aufirst%3DJ.-C.%26aulast%3DSchunack%26aufirst%3DW.%26aulast%3DStark%26aufirst%3DH.%26atitle%3DDevelopment%2520of%2520a%2520New%2520Class%2520of%2520Nonimidazole%2520Histamine%2520H%25203%2520Receptor%2520Ligands%2520with%2520Combined%2520Inhibitory%2520Histamine%2520N-Methyltransferase%2520Activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D1128%26epage%3D1141%26doi%3D10.1021%2Fjm0110845" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sadowski, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Triezenberg, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ptashne, M.</span></span> <span> </span><span class="NLM_article-title">GAL4-VP16 Is an Unusually Potent Transcriptional Activator</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>335</i></span>,  <span class="NLM_fpage">563</span>– <span class="NLM_lpage">564</span>, <span class="refDoi"> DOI: 10.1038/335563a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1038%2F335563a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=3047590" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADyaL1MXitVSgsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=335&publication_year=1988&pages=563-564&author=I.+Sadowskiauthor=J.+Maauthor=S.+Triezenbergauthor=M.+Ptashne&title=GAL4-VP16+Is+an+Unusually+Potent+Transcriptional+Activator&doi=10.1038%2F335563a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">GAL4-VP16 is an unusually potent transcriptional activator</span></div><div class="casAuthors">Sadowski, Ivan; Ma, Jun; Triezenberg, Steve; Ptashne, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">335</span>
        (<span class="NLM_cas:issue">6190</span>),
    <span class="NLM_cas:pages">563-4</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">A new transcriptional activator is described which was formed by fusing a DNA-binding fragment of the yeast activator GAL4 to a highly acidic portion of the herpes simplex virus protein VP16.  VP16 activates transcription of immediate early viral genes by using its N-terminal sequences to attach to one or more host-encoded proteins that recognize DNA sequences in their promoters.  The hybrid protein (GAL4-VP16) activates transcription unusually efficiently in mammalian cells when bound close to, or at large distances from the gene.  The activating region of VP16 may be near-maximally potent, and it may not be coincidental that such a strong activator is encoded by a virus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCRLe_4KX8rrVg90H21EOLACvtfcHk0lj-X7BUqPxyRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXitVSgsg%253D%253D&md5=5ff658f48cec708ce35a3d687adb8b3f</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2F335563a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F335563a0%26sid%3Dliteratum%253Aachs%26aulast%3DSadowski%26aufirst%3DI.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DTriezenberg%26aufirst%3DS.%26aulast%3DPtashne%26aufirst%3DM.%26atitle%3DGAL4-VP16%2520Is%2520an%2520Unusually%2520Potent%2520Transcriptional%2520Activator%26jtitle%3DNature%26date%3D1988%26volume%3D335%26spage%3D563%26epage%3D564%26doi%3D10.1038%2F335563a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Willems, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heering, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merk, D.</span></span> <span> </span><span class="NLM_article-title">The Orphan Nuclear Receptor Nurr1 Is Responsive to Non-steroidal Anti-inflammatory Drugs</span>. <i>Commun. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">85</span>, <span class="refDoi"> DOI: 10.1038/s42004-020-0331-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1038%2Fs42004-020-0331-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitVKksrvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2020&pages=85&author=S.+Willemsauthor=W.+Kiluauthor=X.+Niauthor=A.+Chaikuadauthor=S.+Knappauthor=J.+Heeringauthor=D.+Merk&title=The+Orphan+Nuclear+Receptor+Nurr1+Is+Responsive+to+Non-steroidal+Anti-inflammatory+Drugs&doi=10.1038%2Fs42004-020-0331-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">The orphan nuclear receptor Nurr1 is responsive to non-steroidal anti-inflammatory drugs</span></div><div class="casAuthors">Willems, Sabine; Kilu, Whitney; Ni, Xiaomin; Chaikuad, Apirat; Knapp, Stefan; Heering, Jan; Merk, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Communications Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">85</span>CODEN:
                <span class="NLM_cas:coden">CCOHCT</span>;
        ISSN:<span class="NLM_cas:issn">2399-3669</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Nuclear receptor related 1 (Nurr1) is an orphan ligand-activated transcription factor and considered as neuroprotective transcriptional regulator with great potential as therapeutic target for neurodegenerative diseases.  However, the collection of available Nurr1 modulators and mechanistic understanding of Nurr1 are limited.  Here, we report the discovery of several structurally diverse non-steroidal anti-inflammatory drugs as inverse Nurr1 agonists demonstrating that Nurr1 activity can be regulated bidirectionally.  As chem. tools, these ligands enable unraveling the co-regulatory network of Nurr1 and the mode of action distinguishing agonists from inverse agonists.  In addn. to its ability to dimerize, we observe an ability of Nurr1 to recruit several canonical nuclear receptor co-regulators in a ligand-dependent fashion.  Distinct dimerization states and co-regulator interaction patterns arise as discriminating factors of Nurr1 agonists and inverse agonists.  Our results contribute a valuable collection of Nurr1 modulators and relevant mechanistic insights for future Nurr1 target validation and drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4xAONvjGyx7Vg90H21EOLACvtfcHk0lj-X7BUqPxyRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitVKksrvE&md5=780001615544ae3f93a28007f8f492be</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fs42004-020-0331-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs42004-020-0331-0%26sid%3Dliteratum%253Aachs%26aulast%3DWillems%26aufirst%3DS.%26aulast%3DKilu%26aufirst%3DW.%26aulast%3DNi%26aufirst%3DX.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHeering%26aufirst%3DJ.%26aulast%3DMerk%26aufirst%3DD.%26atitle%3DThe%2520Orphan%2520Nuclear%2520Receptor%2520Nurr1%2520Is%2520Responsive%2520to%2520Non-steroidal%2520Anti-inflammatory%2520Drugs%26jtitle%3DCommun.%2520Chem.%26date%3D2020%26volume%3D3%26spage%3D85%26doi%3D10.1038%2Fs42004-020-0331-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, U. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohankumar, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michelhaugh, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mittal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tjalkens, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Safe, S.</span></span> <span> </span><span class="NLM_article-title">Nuclear Receptor 4A2 (NR4A2) Is a Druggable Target for Glioblastomas</span>. <i>J. Neurooncol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>146</i></span>,  <span class="NLM_fpage">25</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1007/s11060-019-03349-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1007%2Fs11060-019-03349-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=31754919" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A280%3ADC%252BB3MfhvFaqsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146&publication_year=2020&pages=25-39&author=K.+Karkiauthor=X.+Liauthor=U.+H.+Jinauthor=K.+Mohankumarauthor=M.+Zareiauthor=S.+K.+Michelhaughauthor=S.+Mittalauthor=R.+Tjalkensauthor=S.+Safe&title=Nuclear+Receptor+4A2+%28NR4A2%29+Is+a+Druggable+Target+for+Glioblastomas&doi=10.1007%2Fs11060-019-03349-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Nuclear receptor 4A2 (NR4A2) is a druggable target for glioblastomas</span></div><div class="casAuthors">Karki Keshav; Li Xi; Jin Un-Ho; Mohankumar Kumaravel; Zarei Mahsa; Safe Stephen; Michelhaugh Sharon K; Mittal Sandeep; Tjalkens Ronald; Safe Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of neuro-oncology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">146</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">25-39</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">INTRODUCTION:  The orphan nuclear receptor 4A2 (NR4A2) has been extensively characterized in subcellular regions of the brain and is necessary for the function of dopaminergic neurons.  The NR4A2 ligand, 1,1-bis (3(1)-indoly1)-1-(p-chlorophenyl)methane (DIM-C-pPhCl) inhibits markers of neuroinflammation and degeneration in mouse models and in this study we investigated expression and function of NR4A2 in glioblastoma (GBM).  METHODS:  Established and patient-derived cell lines were used as models and the expression and functions of NR4A2 were determined by western blots and NR4A2 gene silencing by antisense oligonucleotides respectively.  Effects of NR4A2 knockdown and DIM-C-pPhCl on cell growth, induction of apoptosis (Annexin V Staining) and migration/invasion (Boyden chamber and spheroid invasion assay) and transactivation of NR4A2-regulated reporter genes were determined.  Tumor growth was investigated in athymic nude mice bearing U87-MG cells as xenografts.  RESULTS:  NR4A2 knockdown and DIM-C-pPhCl inhibited GBM cell and tumor growth, induced apoptosis and inhibited migration and invasion of GBM cells.  DIM-C-pPhCl and related analogs also inhibited NR4A2-regulated transactivation (luciferase activity) confirming that DIM-C-pPhCl acts as an NR4A2 antagonist and blocks NR4A2-dependent pro-oncogenic responses in GBM.  CONCLUSION:  We demonstrate for the first time that NR4A2 is pro-oncogenic in GBM and thus a potential druggable target for patients with tumors expressing this receptor.  Moreover, our bis-indole-derived NR4A2 antagonists represent a novel class of anti-cancer agents with potential future clinical applications for treating GBM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSjWs8jIOORwOkdrW1FkXG-fW6udTcc2eZ2bJeHBAcRKLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MfhvFaqsA%253D%253D&md5=8cce444908b7254c35e5092cbefe88d0</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1007%2Fs11060-019-03349-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11060-019-03349-y%26sid%3Dliteratum%253Aachs%26aulast%3DKarki%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DJin%26aufirst%3DU.%2BH.%26aulast%3DMohankumar%26aufirst%3DK.%26aulast%3DZarei%26aufirst%3DM.%26aulast%3DMichelhaugh%26aufirst%3DS.%2BK.%26aulast%3DMittal%26aufirst%3DS.%26aulast%3DTjalkens%26aufirst%3DR.%26aulast%3DSafe%26aufirst%3DS.%26atitle%3DNuclear%2520Receptor%25204A2%2520%2528NR4A2%2529%2520Is%2520a%2520Druggable%2520Target%2520for%2520Glioblastomas%26jtitle%3DJ.%2520Neurooncol.%26date%3D2020%26volume%3D146%26spage%3D25%26epage%3D39%26doi%3D10.1007%2Fs11060-019-03349-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keserü, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, C. H.</span></span> <span> </span><span class="NLM_article-title">The Role of Ligand Efficiency Metrics in Drug Discovery</span>. <i>Nat. Rev. Drug Discov.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">121</span>, <span class="refDoi"> DOI: 10.1038/nrd4163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1038%2Fnrd4163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=24481311" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=105-121&author=A.+L.+Hopkinsauthor=G.+M.+Keser%C3%BCauthor=P.+D.+Leesonauthor=D.+C.+Reesauthor=C.+H.+Reynolds&title=The+Role+of+Ligand+Efficiency+Metrics+in+Drug+Discovery&doi=10.1038%2Fnrd4163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The role of ligand efficiency metrics in drug discovery</span></div><div class="casAuthors">Hopkins, Andrew L.; Keserue, Gyoergy M.; Leeson, Paul D.; Rees, David C.; Reynolds, Charles H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">105-121</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The judicious application of ligand or binding efficiency metrics, which quantify the mol. properties required to obtain binding affinity for a drug target, is gaining traction in the selection and optimization of fragments, hits and leads.  Retrospective anal. of recently marketed oral drugs shows that they frequently have highly optimized ligand efficiency values for their targets.  Optimizing ligand efficiency metrics based on both mol. mass and lipophilicity, when set in the context of the specific target, has the potential to ameliorate the inflation of these properties that has been obsd. in current medicinal chem. practice, and to increase the quality of drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq96uhvE1OZ5LVg90H21EOLACvtfcHk0lh18KvZXprtGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D&md5=344030e59f499180f79d311302a27532</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnrd4163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4163%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DKeser%25C3%25BC%26aufirst%3DG.%2BM.%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DReynolds%26aufirst%3DC.%2BH.%26atitle%3DThe%2520Role%2520of%2520Ligand%2520Efficiency%2520Metrics%2520in%2520Drug%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discov.%26date%3D2014%26volume%3D13%26spage%3D105%26epage%3D121%26doi%3D10.1038%2Fnrd4163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tetko, I. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanchuk, V. Y.</span></span> <span> </span><span class="NLM_article-title">Application of Associative Neural Networks for Prediction of Lipophilicity in ALOGPS 2.1 Program</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">1136</span>– <span class="NLM_lpage">1145</span>, <span class="refDoi"> DOI: 10.1021/ci025515j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci025515j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BD38XlvFSmtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2002&pages=1136-1145&author=I.+V.+Tetkoauthor=V.+Y.+Tanchuk&title=Application+of+Associative+Neural+Networks+for+Prediction+of+Lipophilicity+in+ALOGPS+2.1+Program&doi=10.1021%2Fci025515j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Application of associative neural networks for prediction of lipophilicity in ALOGPS 2.1 program</span></div><div class="casAuthors">Tetko, Igor V.; Tanchuk, Vsevolod Yu.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1136-1145</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This article provides a systematic study of several important parameters of the Associative Neural Network (ASNN), such as the no. of networks in the ensemble, distance measures, neighbor functions, selection of smoothing parameters, and strategies for the user-training feature of the algorithm.  The performance of the different methods is assessed with several training/test sets used to predict lipophilicity of chem. compds.  The Spearman rank-order correlation coeff. and Parzen-window regression methods provide the best performance of the algorithm.  If addnl. user data is available, an improved prediction of lipophilicity of chems. up to 2-5 times can be calcd. when the appropriate smoothing parameters for the neural network are selected.  The detected best combinations of parameters and strategies are implemented in the ALOGPS 2.1 program that is publicly available at http://www.vcclab.org/lab/alogps.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9Nz8kkyB1NbVg90H21EOLACvtfcHk0lh18KvZXprtGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XlvFSmtrY%253D&md5=45d412977a5171d4e8b03ada28c3632c</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fci025515j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci025515j%26sid%3Dliteratum%253Aachs%26aulast%3DTetko%26aufirst%3DI.%2BV.%26aulast%3DTanchuk%26aufirst%3DV.%2BY.%26atitle%3DApplication%2520of%2520Associative%2520Neural%2520Networks%2520for%2520Prediction%2520of%2520Lipophilicity%2520in%2520ALOGPS%25202.1%2520Program%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2002%26volume%3D42%26spage%3D1136%26epage%3D1145%26doi%3D10.1021%2Fci025515j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niu, Y. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jing, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, T. Z.</span></span> <span> </span><span class="NLM_article-title">Amodiaquine-Induced Reproductive Toxicity in Adult Male Rats</span>. <i>Mol. Reprod. Dev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">174</span>– <span class="NLM_lpage">182</span>, <span class="refDoi"> DOI: 10.1002/mrd.22603</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1002%2Fmrd.22603" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=26647924" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVansbjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2016&pages=174-182&author=Y.+R.+Niuauthor=B.+Weiauthor=B.+Chenauthor=L.+H.+Xuauthor=X.+Jingauthor=C.+L.+Pengauthor=T.+Z.+Ma&title=Amodiaquine-Induced+Reproductive+Toxicity+in+Adult+Male+Rats&doi=10.1002%2Fmrd.22603"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Amodiaquine-induced reproductive toxicity in adult male rats</span></div><div class="casAuthors">Niu, Yan-Ru; Wei, Bing; Chen, Bi; Xu, Li-Hua; Jing, Xia; Peng, Cai-Ling; Ma, Tian-Zhong</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Reproduction & Development</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">174-182</span>CODEN:
                <span class="NLM_cas:coden">MREDEE</span>;
        ISSN:<span class="NLM_cas:issn">1040-452X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Amodiaquine (AQ) is routinely prescribed as an anti-malarial drug.  Here, we evaluated AQ-induced toxicity in the male reproductive system.  Eighty adult male Sprague-Dawley rats were randomly divided into four groups that received distd. water (control) or daily doses of 5 mg/kg body wt., 10 mg/kg, or 15 mg/kg AQ for 2 wk.  Testes morphol. was analyzed using hematoxylin-and-eosin staining, terminal dUTP nicked-end labeling (TUNEL), and immunostaining whereas protein expression was detd. by Western blotting.  AQ dose-dependently led to abnormal spermatogenesis.  Disruption of the blood-testis barrier and increased germ cell apoptosis were obsd. in all three AQ-treated groups.  Interestingly, AQ-induced damage of spermatogenesis recovered over time, based on the survival of promyelocytic leukemia zinc-finger (PLZF)-pos., undifferentiated spermatogonia.  Serum levels of LH and testosterone, as well as testicular testosterone levels, were not significantly altered in AQ-treated groups compared with controls.  Collectively, our study suggests that AQ exerts substantial acute side effects on the reproductive systems of adult male rats by inducing the apoptosis of differentiating spermatogenic cells and disruption of blood-testis barrier function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoOsPZ7QEDXbVg90H21EOLACvtfcHk0lh18KvZXprtGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVansbjF&md5=c8105eb83d6fed1172ff71a000d655f8</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Fmrd.22603&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmrd.22603%26sid%3Dliteratum%253Aachs%26aulast%3DNiu%26aufirst%3DY.%2BR.%26aulast%3DWei%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DB.%26aulast%3DXu%26aufirst%3DL.%2BH.%26aulast%3DJing%26aufirst%3DX.%26aulast%3DPeng%26aufirst%3DC.%2BL.%26aulast%3DMa%26aufirst%3DT.%2BZ.%26atitle%3DAmodiaquine-Induced%2520Reproductive%2520Toxicity%2520in%2520Adult%2520Male%2520Rats%26jtitle%3DMol.%2520Reprod.%2520Dev.%26date%3D2016%26volume%3D83%26spage%3D174%26epage%3D182%26doi%3D10.1002%2Fmrd.22603" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bernuau, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larrey, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campillo, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degott, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdier, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rueff, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pessayre, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benhamou, J. P.</span></span> <span> </span><span class="NLM_article-title">Amodiaquine-Induced Fulminant Hepatitis</span>. <i>J. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">109</span>– <span class="NLM_lpage">112</span>, <span class="refDoi"> DOI: 10.1016/S0168-8278(88)80469-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1016%2FS0168-8278%2888%2980469-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=3346531" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A280%3ADyaL1c7lsVeluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1988&pages=109-112&author=J.+Bernuauauthor=D.+Larreyauthor=B.+Campilloauthor=C.+Degottauthor=F.+Verdierauthor=B.+Rueffauthor=D.+Pessayreauthor=J.+P.+Benhamou&title=Amodiaquine-Induced+Fulminant+Hepatitis&doi=10.1016%2FS0168-8278%2888%2980469-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Amodiaquine-induced fulminant hepatitis</span></div><div class="casAuthors">Bernuau J; Larrey D; Campillo B; Degott C; Verdier F; Rueff B; Pessayre D; Benhamou J P</div><div class="citationInfo"><span class="NLM_cas:title">Journal of hepatology</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">109-12</span>
        ISSN:<span class="NLM_cas:issn">0168-8278</span>.
    </div><div class="casAbstract">Three patients suffered from fulminant hepatitis within 23, 59 and 22 weeks after having ingested a total dose of 16, 26 and 15 g, respectively, of amodiaquine for the prophylaxis of malaria.  Amodiaquine administration was continued for 44, 21 and 25 days after the onset of jaundice, respectively.  One patient underwent emergency orthotopic liver transplantation and survived.  The other two died.  Fulminant hepatitis threatens patients in whom amodiaquine administration is protracted for several months and not interrupted when jaundice occurs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSW1Unx4VGTl4IRAdBLKY41fW6udTcc2ebsZ16afWZHHLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaL1c7lsVeluw%253D%253D&md5=137e3d15331d0204ca4e8ab4eed16004</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2FS0168-8278%2888%2980469-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0168-8278%252888%252980469-0%26sid%3Dliteratum%253Aachs%26aulast%3DBernuau%26aufirst%3DJ.%26aulast%3DLarrey%26aufirst%3DD.%26aulast%3DCampillo%26aufirst%3DB.%26aulast%3DDegott%26aufirst%3DC.%26aulast%3DVerdier%26aufirst%3DF.%26aulast%3DRueff%26aufirst%3DB.%26aulast%3DPessayre%26aufirst%3DD.%26aulast%3DBenhamou%26aufirst%3DJ.%2BP.%26atitle%3DAmodiaquine-Induced%2520Fulminant%2520Hepatitis%26jtitle%3DJ.%2520Hepatol.%26date%3D1988%26volume%3D6%26spage%3D109%26epage%3D112%26doi%3D10.1016%2FS0168-8278%2888%2980469-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Budzyński, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puustinen, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joutsen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sistonen, L.</span></span> <span> </span><span class="NLM_article-title">Uncoupling Stress-Inducible Phosphorylation of Heat Shock Factor 1 from Its Activation</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">2530</span>– <span class="NLM_lpage">2540</span>, <span class="refDoi"> DOI: 10.1128/MCB.00816-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1128%2FMCB.00816-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=25963659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFamurrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=2530-2540&author=M.+A.+Budzy%C5%84skiauthor=M.+C.+Puustinenauthor=J.+Joutsenauthor=L.+Sistonen&title=Uncoupling+Stress-Inducible+Phosphorylation+of+Heat+Shock+Factor+1+from+Its+Activation&doi=10.1128%2FMCB.00816-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Uncoupling stress-inducible phosphorylation of heat shock factor 1 from its activation</span></div><div class="casAuthors">Budzynski, Marek A.; Puustinen, Mikael C.; Joutsen, Jenny; Sistonen, Lea</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2530-2540</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">1098-5549</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">In mammals the stress-inducible expression of genes encoding heat shock proteins is under the control of the heat shock transcription factor 1 (HSF1).  Activation of HSF1 is a multistep process, involving trimerization, acquisition of DNA-binding and transcriptional activities, which coincide with several posttranslational modifications.  Stress-inducible phosphorylation of HSF1, or hyperphosphorylation, which occurs mainly within the regulatory domain (RD), has been proposed as a requirement for HSF-driven transcription and is widely used for assessing HSF1 activation.  Nonetheless, the contribution of hyperphosphorylation to the activity of HSF1 remains unknown.  In this study, we generated a phosphorylation-deficient HSF1 mutant (HSF1Δ∼PRD), where the 15 known phosphorylation sites within the RD were disrupted.  Our results show that the phosphorylation status of the RD does not affect the subcellular localization and DNA-binding activity of HSF1.  Surprisingly, under stress conditions, HSF1Δ∼PRD is a potent transactivator of both endogenous targets and a reporter gene, and HSF1Δ∼PRD has a reduced activation threshold.  Our results provide the first direct evidence for uncoupling stress-inducible phosphorylation of HSF1 from its activation, and we propose that the phosphorylation signature alone is not an appropriate marker for HSF1 activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGri3_8rGXXWrrVg90H21EOLACvtfcHk0lh_SNb5TCgMQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFamurrP&md5=baff5411c24b2d2167af35caf17480c5</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1128%2FMCB.00816-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00816-14%26sid%3Dliteratum%253Aachs%26aulast%3DBudzy%25C5%2584ski%26aufirst%3DM.%2BA.%26aulast%3DPuustinen%26aufirst%3DM.%2BC.%26aulast%3DJoutsen%26aufirst%3DJ.%26aulast%3DSistonen%26aufirst%3DL.%26atitle%3DUncoupling%2520Stress-Inducible%2520Phosphorylation%2520of%2520Heat%2520Shock%2520Factor%25201%2520from%2520Its%2520Activation%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2015%26volume%3D35%26spage%3D2530%26epage%3D2540%26doi%3D10.1128%2FMCB.00816-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Merk, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmad, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrasco Gomez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinhilber, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schubert-Zsilavecz, M.</span></span> <span> </span><span class="NLM_article-title">Extending the Structure-Activity Relationship of Anthranilic Acid Derivatives as Farnesoid x Receptor Modulators: Development of a Highly Potent Partial Farnesoid X Receptor Agonist</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">8035</span>– <span class="NLM_lpage">8055</span>, <span class="refDoi"> DOI: 10.1021/jm500937v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500937v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFCjt7bE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8035-8055&author=D.+Merkauthor=C.+Lamersauthor=K.+Ahmadauthor=R.+Carrasco+Gomezauthor=G.+Schneiderauthor=D.+Steinhilberauthor=M.+Schubert-Zsilavecz&title=Extending+the+Structure-Activity+Relationship+of+Anthranilic+Acid+Derivatives+as+Farnesoid+x+Receptor+Modulators%3A+Development+of+a+Highly+Potent+Partial+Farnesoid+X+Receptor+Agonist&doi=10.1021%2Fjm500937v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Extending the Structure-Activity Relationship of Anthranilic Acid Derivatives As Farnesoid X Receptor Modulators: Development of a Highly Potent Partial Farnesoid X Receptor Agonist</span></div><div class="casAuthors">Merk, Daniel; Lamers, Christina; Ahmad, Khalil; Carrasco Gomez, Roberto; Schneider, Gisbert; Steinhilber, Dieter; Schubert-Zsilavecz, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8035-8055</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The ligand activated transcription factor nuclear farnesoid X receptor (FXR) is involved as a regulator in many metabolic pathways including bile acid and glucose homeostasis.  Therefore, pharmacol. activation of FXR seems a valuable therapeutic approach for several conditions including metabolic diseases linked to insulin resistance, liver disorders such as primary biliary cirrhosis or nonalcoholic steatohepatitis, and certain forms of cancer.  The available FXR agonists, however, activate the receptor to the full extent which might be disadvantageous over a longer time period.  Hence, partial FXR activators are required for long-term treatment of metabolic disorders.  Reported here are the SAR of anthranilic acid derivs. as FXR modulators and development, synthesis, and characterization of compd. (I), which is a highly potent partial FXR agonist in a reporter gene assay with an EC50 value of 8±3 nM and on mRNA level in liver cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxZMafrXUE1LVg90H21EOLACvtfcHk0lh_SNb5TCgMQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFCjt7bE&md5=1b0b248cf689ad910d91a078823a0001</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm500937v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500937v%26sid%3Dliteratum%253Aachs%26aulast%3DMerk%26aufirst%3DD.%26aulast%3DLamers%26aufirst%3DC.%26aulast%3DAhmad%26aufirst%3DK.%26aulast%3DCarrasco%2BGomez%26aufirst%3DR.%26aulast%3DSchneider%26aufirst%3DG.%26aulast%3DSteinhilber%26aufirst%3DD.%26aulast%3DSchubert-Zsilavecz%26aufirst%3DM.%26atitle%3DExtending%2520the%2520Structure-Activity%2520Relationship%2520of%2520Anthranilic%2520Acid%2520Derivatives%2520as%2520Farnesoid%2520x%2520Receptor%2520Modulators%253A%2520Development%2520of%2520a%2520Highly%2520Potent%2520Partial%2520Farnesoid%2520X%2520Receptor%2520Agonist%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8035%26epage%3D8055%26doi%3D10.1021%2Fjm500937v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit2'],'ref2':['cit3'],'ref3':['cit4'],'ref4':['cit5'],'ref5':['cit6'],'ref6':['cit7'],'ref7':['cit8'],'ref8':['cit9'],'ref9':['cit10'],'ref10':['cit11'],'ref11':['cit12'],'ref12':['cit13'],'ref13':['cit14'],'ref14':['cit15'],'ref15':['cit16'],'ref16':['cit17'],'ref17':['cit18'],'ref18':['cit19'],'ref19':['cit20'],'ref20':['cit21'],'ref21':['cit22'],'ref22':['cit23'],'ref23':['cit24']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 2 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Sabine Willems, Daniel Zaienne, <span class="NLM_string-name hlFld-ContribAuthor">Daniel Merk</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Nuclear Receptors in Neurodegeneration and Neuroinflammation. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (14)
                                     , 9592-9638. <a href="https://doi.org/10.1021/acs.jmedchem.1c00186" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00186</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00186&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00186%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BNuclear%252BReceptors%252Bin%252BNeurodegeneration%252Band%252BNeuroinflammation%26aulast%3DWillems%26aufirst%3DSabine%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D01022021%26date%3D12072021%26volume%3D64%26issue%3D14%26spage%3D9592%26epage%3D9638" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Svetlana A. Kholodar, Geoffrey Lang, Wilian A. Cortopassi, Yoshie Iizuka, Harman S. Brah, Matthew P. Jacobson, <span class="NLM_string-name hlFld-ContribAuthor">Pamela M. England</span>. </span><span class="cited-content_cbyCitation_article-title">Analogs of the Dopamine Metabolite 5,6-Dihydroxyindole Bind Directly to and Activate the Nuclear Receptor Nurr1. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2021,</strong> <em>16 </em>
                                    (7)
                                     , 1159-1163. <a href="https://doi.org/10.1021/acschembio.1c00326" title="DOI URL">https://doi.org/10.1021/acschembio.1c00326</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.1c00326&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.1c00326%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DAnalogs%252Bof%252Bthe%252BDopamine%252BMetabolite%252B5%25252C6-Dihydroxyindole%252BBind%252BDirectly%252Bto%252Band%252BActivate%252Bthe%252BNuclear%252BReceptor%252BNurr1%26aulast%3DKholodar%26aufirst%3DSvetlana%2BA.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D04052021%26date%3D11062021%26date%3D24062021%26volume%3D16%26issue%3D7%26spage%3D1159%26epage%3D1163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/medium/jm0c01779_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/large/jm0c01779_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01779&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/medium/jm0c01779_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/large/jm0c01779_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <b>6</b> and <b>29</b>–<b>33</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/large/jm0c01779_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01779&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) KI, EtOH, 2 N HCl, 90 °C, 14–20 h; (b) EtOH, <i>μw</i>, 140 °C, 36–48 h.</p></p></figure><figure data-id="sch2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/medium/jm0c01779_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/large/jm0c01779_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of 27 and 28<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/large/jm0c01779_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01779&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) HOAc, DCE, room temperature, 30 min; then NaB(OAc)<sub>3</sub>H, DCE, 50 °C, 24 h.</p></p></figure><figure data-id="fig1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/medium/jm0c01779_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/large/jm0c01779_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Nurr1 agonism of <b>24</b> and <b>26</b> in cellular settings. (a, b) Nurr1 agonist activity of (a) <b>24</b> and (b) <b>26</b> on full-length human Nurr1 as the monomer (NBRE), homodimer (NurRE), and RXR-heterodimer (DR5). Data are the mean ± S.E.M.; n ≥ 3. Individual curves are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01779/suppl_file/jm0c01779_si_001.pdf" class="ext-link">Figure S4</a>, and corresponding EC<sub>50</sub> values are listed in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01779/suppl_file/jm0c01779_si_001.pdf" class="ext-link">Table S1</a>. (c) Effects of <b>24</b> and <b>26</b> on mRNA expression of vesicular monoamine transporter 2 (VMAT2) and tyrosine hydroxylase (TH) in human astrocytes (T98G). CQ (<b>2</b>) for comparison. Nurr1 agonists <b>24</b> and <b>26</b> strongly promoted Nurr1-regulated VMAT2 and TH expression in a dose-dependent manner. Data are the mean ± S.E.M.; <i>n</i> = 4. mRNA levels were determined by qRT-PCR and analyzed by the 2<sup>-ΔΔCt</sup> method. * p < 0.05, ** p < 0.01, *** p < 0.001 vs. DMSO control (t-test).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/large/jm0c01779_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01779&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/medium/jm0c01779_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/large/jm0c01779_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Effects of Nurr1 agonist <b>26</b> (20 and 100 μM) on co-regulator recruitment to the Nurr1 LBD in homogeneous time-resolved fluorescence resonance energy transfer (HTRF) assays. (a) Recruitment of Tb<sup>3+</sup>-cryptate-labeled NCoR1 to the sGFP-labeled Nurr1 LBD. (b) Recruitment of Tb<sup>3+</sup>-cryptate-labeled NCoR2 to the sGFP-labeled Nurr1 LBD. (c) Homodimerization between the Tb<sup>3+</sup>-cryptate-labeled and sGFP-labeled Nurr1 LBD. Data are the mean ± SD; <i>N</i> = 3. Compound <b>24</b> exhibits comparable effects (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01779/suppl_file/jm0c01779_si_001.pdf" class="ext-link">Figure S6</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01779/20210305/images/large/jm0c01779_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01779&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i27">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27651" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27651" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 23 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benoit, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aarnisalo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perlmann, T.</span></span> <span> </span><span class="NLM_article-title">Structure and Function of Nurr 1 Identifies a Class of Ligand-Independent Nuclear Receptors</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>423</i></span>,  <span class="NLM_fpage">555</span>– <span class="NLM_lpage">560</span>, <span class="refDoi"> DOI: 10.1038/nature01645</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1038%2Fnature01645" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=12774125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BD3sXktVygtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=423&publication_year=2003&pages=555-560&author=Z.+Wangauthor=G.+Benoitauthor=J.+Liuauthor=S.+Prasadauthor=P.+Aarnisaloauthor=X.+Liuauthor=H.+Xuauthor=N.+P.+Walkerauthor=T.+Perlmann&title=Structure+and+Function+of+Nurr+1+Identifies+a+Class+of+Ligand-Independent+Nuclear+Receptors&doi=10.1038%2Fnature01645"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and function of Nurr1 identifies a class of ligand-independent nuclear receptors</span></div><div class="casAuthors">Wang, Zhulun; Benoit, Gerard; Liu, Jinsong; Prasad, Srividya; Aarnisalo, Piia; Liu, Xiaohong; Xu, Haoda; Walker, Nigel P. C.; Perlmann, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">423</span>
        (<span class="NLM_cas:issue">6939</span>),
    <span class="NLM_cas:pages">555-560</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Members of the nuclear receptor (NR) superfamily of transcription factors modulate gene transcription in response to small lipophilic mols.  Transcriptional activity is regulated by ligands binding to the carboxy-terminal ligand-binding domains (LBDs) of cognate NRs.  A subgroup of NRs referred to as 'orphan receptors' lack identified ligands, however, raising issues about the function of their LBDs.  Here we report the crystal structure of the LBD of the orphan receptor Nurr1 at 2.2 Å resoln.  The Nurr1 LBD adopts a canonical protein fold resembling that of agonist-bound, transcriptionally active LBDs in NRs, but the structure has two distinctive features.  First, the Nurr1 LBD contains no cavity as a result of the tight packing of side chains from several bulky hydrophobic residues in the region normally occupied by ligands.  Second, Nurr1 lacks a 'classical' binding site for coactivators.  Despite these differences, the Nurr1 LBD can be regulated in mammalian cells.  Notably, transcriptional activity is correlated with the Nurr1 LBD adopting a more stable conformation.  Our findings highlight a unique structural class of NRs and define a model for ligand-independent NR function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrf_Dsl30OicrVg90H21EOLACvtfcHk0ljx207DyRLJ1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXktVygtb4%253D&md5=d19448447ffb618404ae84e4aa37c505</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnature01645&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature01645%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DBenoit%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DPrasad%26aufirst%3DS.%26aulast%3DAarnisalo%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DWalker%26aufirst%3DN.%2BP.%26aulast%3DPerlmann%26aufirst%3DT.%26atitle%3DStructure%2520and%2520Function%2520of%2520Nurr%25201%2520Identifies%2520a%2520Class%2520of%2520Ligand-Independent%2520Nuclear%2520Receptors%26jtitle%3DNature%26date%3D2003%26volume%3D423%26spage%3D555%26epage%3D560%26doi%3D10.1038%2Fnature01645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Decressac, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volakakis, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Björklund, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perlmann, T.</span></span> <span> </span><span class="NLM_article-title">NURR1 in Parkinson Disease—From Pathogenesis to Therapeutic Potential</span>. <i>Nat. Rev. Neurol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">629</span>– <span class="NLM_lpage">636</span>, <span class="refDoi"> DOI: 10.1038/nrneurol.2013.209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1038%2Fnrneurol.2013.209" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=24126627" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1ChsrjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=629-636&author=M.+Decressacauthor=N.+Volakakisauthor=A.+Bj%C3%B6rklundauthor=T.+Perlmann&title=NURR1+in+Parkinson+Disease%E2%80%94From+Pathogenesis+to+Therapeutic+Potential&doi=10.1038%2Fnrneurol.2013.209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">NURR1 in Parkinson disease-from pathogenesis to therapeutic potential</span></div><div class="casAuthors">Decressac, Mickael; Volakakis, Nikolaos; Bjoerklund, Anders; Perlmann, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Neurology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">629-636</span>CODEN:
                <span class="NLM_cas:coden">NRNACP</span>;
        ISSN:<span class="NLM_cas:issn">1759-4758</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In Parkinson disease (PD), affected midbrain dopamine (DA) neurons lose specific dopaminergic properties before the neurons die.  How the phenotype of DA neurons is normally established and the ways in which pathol. affects the maintenance of cell identity are, therefore, important considerations.  Orphan nuclear receptor NURR1 (NURR1, also known as NR4A2) is involved in the differentiation of midbrain DA neurons, but also has an important role in the adult brain.  Emerging evidence indicates that impaired NURR1 function might contribute to the pathogenesis of PD: NURR1 and its transcriptional targets are downregulated in midbrain DA neurons that express high levels of the disease-causing protein α-synuclein.  Clin. and exptl. data indicate that disrupted NURR1 function contributes to induction of DA neuron dysfunction, which is seen in early stages of PD.  The likely involvement of NURR1 in the development and progression of PD makes this protein a potentially interesting target for therapeutic intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKj9zkT4BEiLVg90H21EOLACvtfcHk0lgoZtL8uPn_qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1ChsrjJ&md5=0ccc0530d53c1d19ef38e81307b99691</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrneurol.2013.209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrneurol.2013.209%26sid%3Dliteratum%253Aachs%26aulast%3DDecressac%26aufirst%3DM.%26aulast%3DVolakakis%26aufirst%3DN.%26aulast%3DBj%25C3%25B6rklund%26aufirst%3DA.%26aulast%3DPerlmann%26aufirst%3DT.%26atitle%3DNURR1%2520in%2520Parkinson%2520Disease%25E2%2580%2594From%2520Pathogenesis%2520to%2520Therapeutic%2520Potential%26jtitle%3DNat.%2520Rev.%2520Neurol.%26date%3D2013%26volume%3D9%26spage%3D629%26epage%3D636%26doi%3D10.1038%2Fnrneurol.2013.209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, B.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, Q. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sohn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, W.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, E.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naffin-Olivos, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ringe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petsko, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K.-S.</span></span> <span> </span><span class="NLM_article-title">Nuclear Receptor Nurr1 Agonists Enhance Its Dual Functions and Improve Behavioral Deficits in an Animal Model of Parkinson’s Disease</span>. <i>Proc. Natl. Acad. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">8756</span>– <span class="NLM_lpage">8761</span>, <span class="refDoi"> DOI: 10.1073/pnas.1509742112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1073%2Fpnas.1509742112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=26124091" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVOrs7%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=8756-8761&author=C.-H.+Kimauthor=B.-S.+Hanauthor=J.+Moonauthor=D.-J.+Kimauthor=J.+Shinauthor=S.+Rajanauthor=Q.+T.+Nguyenauthor=M.+Sohnauthor=W.-G.+Kimauthor=M.+Hanauthor=I.+Jeongauthor=K.-S.+Kimauthor=E.-H.+Leeauthor=Y.+Tuauthor=J.+L.+Naffin-Olivosauthor=C.-H.+Parkauthor=D.+Ringeauthor=H.+S.+Yoonauthor=G.+A.+Petskoauthor=K.-S.+Kim&title=Nuclear+Receptor+Nurr1+Agonists+Enhance+Its+Dual+Functions+and+Improve+Behavioral+Deficits+in+an+Animal+Model+of+Parkinson%E2%80%99s+Disease&doi=10.1073%2Fpnas.1509742112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Nuclear receptor Nurr1 agonists enhance its dual functions and improve behavioral deficits in an animal model of Parkinson's disease</span></div><div class="casAuthors">Kim, Chun-Hyung; Han, Baek-Soo; Moon, Jisook; Kim, Deog-Joong; Shin, Joon; Rajan, Sreekanth; Nguyen, Quoc Toan; Sohn, Mijin; Kim, Won-Gon; Han, Minjoon; Jeong, Inhye; Kim, Kyoung-Shim; Lee, Eun-Hye; Tu, Yupeng; Naffin-Olivos, Jacqueline L.; Park, Chang-Hwan; Ringe, Dagmar; Yoon, Ho Sup; Petsko, Gregory A.; Kim, Kwang-Soo</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">8756-8761</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Parkinson's disease (PD), primarily caused by selective degeneration of midbrain dopamine (mDA) neurons, is the most prevalent movement disorder, affecting 1-2% of the global population over the age of 65.  Currently available pharmacol. treatments are largely symptomatic and lose their efficacy over time with accompanying severe side effects such as dyskinesia.  Thus, there is an unmet clin. need to develop mechanism-based and/or disease-modifying treatments.  Based on the unique dual role of the nuclear orphan receptor Nurr1 for development and maintenance of mDA neurons and their protection from inflammation-induced death, we hypothesize that Nurr1 can be a mol. target for neuroprotective therapeutic development for PD.  Here we show successful identification of Nurr1 agonists sharing an identical chem. scaffold, 4-amino-7-chloroquinoline, suggesting a crit. structure-activity relationship.  In particular, we found that two antimalarial drugs, amodiaquine and chloroquine stimulate the transcriptional function of Nurr1 through phys. interaction with its ligand binding domain (LBD).  Remarkably, these compds. were able to enhance the contrasting dual functions of Nurr1 by further increasing transcriptional activation of mDA-specific genes and further enhancing transrepression of neurotoxic proinflammatory gene expression in microglia.  Importantly, these compds. significantly improved behavioral deficits in 6-hydroxydopamine lesioned rat model of PD without any detectable signs of dyskinesia-like behavior.  These findings offer proof of principle that small mols. targeting the Nurr1 LBD can be used as a mechanism-based and neuroprotective strategy for PD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaWD4RLxZ8nbVg90H21EOLACvtfcHk0lgzX9lEg1FJDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVOrs7%252FL&md5=715bbb21e39675989c84aa2ab51447dc</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1509742112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1509742112%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DC.-H.%26aulast%3DHan%26aufirst%3DB.-S.%26aulast%3DMoon%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DD.-J.%26aulast%3DShin%26aufirst%3DJ.%26aulast%3DRajan%26aufirst%3DS.%26aulast%3DNguyen%26aufirst%3DQ.%2BT.%26aulast%3DSohn%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DW.-G.%26aulast%3DHan%26aufirst%3DM.%26aulast%3DJeong%26aufirst%3DI.%26aulast%3DKim%26aufirst%3DK.-S.%26aulast%3DLee%26aufirst%3DE.-H.%26aulast%3DTu%26aufirst%3DY.%26aulast%3DNaffin-Olivos%26aufirst%3DJ.%2BL.%26aulast%3DPark%26aufirst%3DC.-H.%26aulast%3DRinge%26aufirst%3DD.%26aulast%3DYoon%26aufirst%3DH.%2BS.%26aulast%3DPetsko%26aufirst%3DG.%2BA.%26aulast%3DKim%26aufirst%3DK.-S.%26atitle%3DNuclear%2520Receptor%2520Nurr1%2520Agonists%2520Enhance%2520Its%2520Dual%2520Functions%2520and%2520Improve%2520Behavioral%2520Deficits%2520in%2520an%2520Animal%2520Model%2520of%2520Parkinson%25E2%2580%2599s%2520Disease%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%26date%3D2015%26volume%3D112%26spage%3D8756%26epage%3D8761%26doi%3D10.1073%2Fpnas.1509742112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rajan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toh, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lescar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoo, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beldar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feitosa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petsko, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K.-S.</span></span> <span> </span><span class="NLM_article-title">PGE1 and PGA1 Bind to Nurr1 and Activate Its Transcriptional Function</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">876</span>– <span class="NLM_lpage">886</span>, <span class="refDoi"> DOI: 10.1038/s41589-020-0553-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1038%2Fs41589-020-0553-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=32451509" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVSgsLzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2020&pages=876-886&author=S.+Rajanauthor=Y.+Jangauthor=C.+H.+Kimauthor=W.+Kimauthor=H.+T.+Tohauthor=J.+Jeonauthor=B.+Songauthor=A.+Serraauthor=J.+Lescarauthor=J.+Y.+Yooauthor=S.+Beldarauthor=H.+Yeauthor=C.+Kangauthor=X.+W.+Liuauthor=M.+Feitosaauthor=Y.+Kimauthor=D.+Hwangauthor=G.+Gohauthor=K.+L.+Limauthor=H.+M.+Parkauthor=C.+H.+Leeauthor=S.+F.+Ohauthor=G.+A.+Petskoauthor=H.+S.+Yoonauthor=K.-S.+Kim&title=PGE1+and+PGA1+Bind+to+Nurr1+and+Activate+Its+Transcriptional+Function&doi=10.1038%2Fs41589-020-0553-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">PGE1 and PGA1 bind to Nurr1 and activate its transcriptional function</span></div><div class="casAuthors">Rajan, Sreekanth; Jang, Yongwoo; Kim, Chun-Hyung; Kim, Woori; Toh, Hui Ting; Jeon, Jeha; Song, Bin; Serra, Aida; Lescar, Julien; Yoo, Jun Yeob; Beldar, Serap; Ye, Hong; Kang, Congbao; Liu, Xue-Wei; Feitosa, Melissa; Kim, Yeahan; Hwang, Dabin; Goh, Geraldine; Lim, Kah-Leong; Park, Hye Min; Lee, Choong Hwan; Oh, Sungwhan F.; Petsko, Gregory A.; Yoon, Ho Sup; Kim, Kwang-Soo</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">876-886</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The orphan nuclear receptor Nurr1 is crit. for the development, maintenance and protection of midbrain dopaminergic (mDA) neurons.  Here we show that prostaglandin E1 (PGE1) and its dehydrated metabolite, PGA1, directly interact with the ligand-binding domain (LBD) of Nurr1 and stimulate its transcriptional function.  We also report the crystallog. structure of Nurr1-LBD bound to PGA1 at 2.05 Å resoln.  PGA1 couples covalently to Nurr1-LBD by forming a Michael adduct with Cys566, and induces notable conformational changes, including a 21° shift of the activation function-2 helix (H12) away from the protein core.  Furthermore, PGE1/PGA1 exhibit neuroprotective effects in a Nurr1-dependent manner, prominently enhance expression of Nurr1 target genes in mDA neurons and improve motor deficits in 1-methyl-4-phenyl-1,2,3,6-tetrahydropyridine-lesioned mouse models of Parkinson's disease.  Based on these results, we propose that PGE1/PGA1 represent native ligands of Nurr1 and can exert neuroprotective effects on mDA neurons, via activation of Nurr1's transcriptional function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqMia6BJvwVdLVg90H21EOLACvtfcHk0lgzX9lEg1FJDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVSgsLzI&md5=5e726ca7eff0e5ed8216d4d4bc99b4e9</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fs41589-020-0553-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-020-0553-6%26sid%3Dliteratum%253Aachs%26aulast%3DRajan%26aufirst%3DS.%26aulast%3DJang%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DC.%2BH.%26aulast%3DKim%26aufirst%3DW.%26aulast%3DToh%26aufirst%3DH.%2BT.%26aulast%3DJeon%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DB.%26aulast%3DSerra%26aufirst%3DA.%26aulast%3DLescar%26aufirst%3DJ.%26aulast%3DYoo%26aufirst%3DJ.%2BY.%26aulast%3DBeldar%26aufirst%3DS.%26aulast%3DYe%26aufirst%3DH.%26aulast%3DKang%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DX.%2BW.%26aulast%3DFeitosa%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DY.%26aulast%3DHwang%26aufirst%3DD.%26aulast%3DGoh%26aufirst%3DG.%26aulast%3DLim%26aufirst%3DK.%2BL.%26aulast%3DPark%26aufirst%3DH.%2BM.%26aulast%3DLee%26aufirst%3DC.%2BH.%26aulast%3DOh%26aufirst%3DS.%2BF.%26aulast%3DPetsko%26aufirst%3DG.%2BA.%26aulast%3DYoon%26aufirst%3DH.%2BS.%26aulast%3DKim%26aufirst%3DK.-S.%26atitle%3DPGE1%2520and%2520PGA1%2520Bind%2520to%2520Nurr1%2520and%2520Activate%2520Its%2520Transcriptional%2520Function%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2020%26volume%3D16%26spage%3D876%26epage%3D886%26doi%3D10.1038%2Fs41589-020-0553-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Bruyn, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Westen, G. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">IJzerman, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stieger, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Witte, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Augustijns, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annaert, P. P.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Identification of Oatp1b1/3 Inhibitors</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">1257</span>– <span class="NLM_lpage">1267</span>, <span class="refDoi"> DOI: 10.1124/mol.112.084152</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1124%2Fmol.112.084152" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=23571415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptlOgtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2013&pages=1257-1267&author=T.+De%0ABruynauthor=G.+J.+P.+Van+Westenauthor=A.+P.+IJzermanauthor=B.+Stiegerauthor=P.+De+Witteauthor=P.+F.+Augustijnsauthor=P.+P.+Annaert&title=Structure-Based+Identification+of+Oatp1b1%2F3+Inhibitors&doi=10.1124%2Fmol.112.084152"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based identification of OATP1B1/3 inhibitors</span></div><div class="casAuthors">De Bruyn, Tom; van Western, Gerard J. P.; IJzerman, Adriaan P.; Stieger, Bruno; de Witte, Peter; Augustijns, Patrick F.; Annaert, Pieter P.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1257-1267</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">1521-0111</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Several recent studies show that inhibition of the hepatic transport proteins org. anion-transporting polypeptide 1B1 (OATP1B1) and 1B3 (OATP1B3) can result in clin. relevant drug-drug interactions (DDI).  To avoid late-stage development drug failures due to OATP1B-mediated DDI, predictive in vitro and in silico methods should be implemented at an early stage of the drug candidate evaluation process.  In the present study, we first developed a high-throughput in vitro transporter inhibition assay for the OATP1B subfamily.  A total of 2000 compds. were tested as potential modulators of the uptake of the OATP1B substrate sodium fluorescein, in OATP1B1- or 1B3-transfected Chinese hamster ovary cells.  At an equimolar substrate-inhibitor concn. of 10 μM, 212 and 139 mols. were identified as OATP1B1 and OATP1B3 inhibitors, resp. (min. 50% inhibition).  For 69 compds., previously not identified as OATP1B inhibitors, concn.-dependent inhibition was also detd., yielding Ki values ranging from 0.06 to 6.5 μM.  Based on these in vitro data, we subsequently developed a proteochemometrics-based in silico model, which predicted OATP1B inhibitors in the test group (20% of the dataset) with high specificity (86%) and sensitivity (78%).  Moreover, several physicochem. compd. properties and substructures related to OATP1B1/1B3 inhibition or inactivity were identified.  Finally, model performance was prospectively verified with a set of 54 compds. not included in the original dataset.  This validation indicated that 80 and 74% of the compds. were correctly classified for OATP1B1 and OATP1B3 inhibition, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoktLtqKIqWh7Vg90H21EOLACvtfcHk0lgzX9lEg1FJDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptlOgtr8%253D&md5=bbf1617e8efba061f65b8f643baa5b98</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1124%2Fmol.112.084152&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.112.084152%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BBruyn%26aufirst%3DT.%26aulast%3DVan%2BWesten%26aufirst%3DG.%2BJ.%2BP.%26aulast%3DIJzerman%26aufirst%3DA.%2BP.%26aulast%3DStieger%26aufirst%3DB.%26aulast%3DDe%2BWitte%26aufirst%3DP.%26aulast%3DAugustijns%26aufirst%3DP.%2BF.%26aulast%3DAnnaert%26aufirst%3DP.%2BP.%26atitle%3DStructure-Based%2520Identification%2520of%2520Oatp1b1%252F3%2520Inhibitors%26jtitle%3DMol.%2520Pharmacol.%26date%3D2013%26volume%3D83%26spage%3D1257%26epage%3D1267%26doi%3D10.1124%2Fmol.112.084152" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alpert, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jez, J. M.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of Phosphoethanolamine Methyltransferase from Plasmodium Falciparum in Complex with Amodiaquine</span>. <i>Bioorganic Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">4990</span>– <span class="NLM_lpage">4993</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2012.06.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1016%2Fj.bmcl.2012.06.032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=22771008" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvVSlt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2012&pages=4990-4993&author=S.+G.+Leeauthor=T.+D.+Alpertauthor=J.+M.+Jez&title=Crystal+Structure+of+Phosphoethanolamine+Methyltransferase+from+Plasmodium+Falciparum+in+Complex+with+Amodiaquine&doi=10.1016%2Fj.bmcl.2012.06.032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of phosphoethanolamine methyltransferase from Plasmodium falciparum in complex with amodiaquine</span></div><div class="casAuthors">Lee, Soon Goo; Alpert, Tara D.; Jez, Joseph M.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4990-4993</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Phosphoethanolamine N-methyltransferase (PMT) is essential for phospholipid biogenesis in the malarial parasite Plasmodium falciparum.  PfPMT catalyzes the triple methylation of phosphoethanolamine to produce phosphocholine, which is then used for phosphatidylcholine synthesis.  Here we describe the 2.0 Å resoln. X-ray crystal structure of PfPMT in complex with amodiaquine.  To better characterize inhibition of PfPMT by amodiaquine, we detd. the IC50 values of a series of aminoquinolines using a direct radiochem. assay.  Both structural and functional analyses provide a possible approach for the development of new small mol. inhibitors of PfPMT.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzsPoLH-UZ2bVg90H21EOLACvtfcHk0ljRSyDhWDCxzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvVSlt7s%253D&md5=aa252f8ce91a293c83bfb07cdce77670</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2012.06.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2012.06.032%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DS.%2BG.%26aulast%3DAlpert%26aufirst%3DT.%2BD.%26aulast%3DJez%26aufirst%3DJ.%2BM.%26atitle%3DCrystal%2520Structure%2520of%2520Phosphoethanolamine%2520Methyltransferase%2520from%2520Plasmodium%2520Falciparum%2520in%2520Complex%2520with%2520Amodiaquine%26jtitle%3DBioorganic%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D22%26spage%3D4990%26epage%3D4993%26doi%3D10.1016%2Fj.bmcl.2012.06.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ren, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span> <span> </span><span class="NLM_article-title">Discovery and Characterization of Novel, Potent, and Selective Cytochrome P450 2J2 Inhibitors</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">60</span>– <span class="NLM_lpage">71</span>, <span class="refDoi"> DOI: 10.1124/dmd.112.048264</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1124%2Fdmd.112.048264" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=23033255" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2012&pages=60-71&author=S.+Renauthor=J.+Zengauthor=Y.+Meiauthor=J.+Z.+H.+Zhangauthor=S.+F.+Yanauthor=J.+Feiauthor=L.+Chen&title=Discovery+and+Characterization+of+Novel%2C+Potent%2C+and+Selective+Cytochrome+P450+2J2+Inhibitors&doi=10.1124%2Fdmd.112.048264"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1124%2Fdmd.112.048264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.112.048264%26sid%3Dliteratum%253Aachs%26aulast%3DRen%26aufirst%3DS.%26aulast%3DZeng%26aufirst%3DJ.%26aulast%3DMei%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%2BZ.%2BH.%26aulast%3DYan%26aufirst%3DS.%2BF.%26aulast%3DFei%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DL.%26atitle%3DDiscovery%2520and%2520Characterization%2520of%2520Novel%252C%2520Potent%252C%2520and%2520Selective%2520Cytochrome%2520P450%25202J2%2520Inhibitors%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2012%26volume%3D41%26spage%3D60%26epage%3D71%26doi%3D10.1124%2Fdmd.112.048264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Neill, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shone, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maggs, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stocks, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biagini, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bray, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibbons, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berry, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winstanley, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukhtar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonar-Law, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hindley, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bambal, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bates, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gresham, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brigandi, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez-delas-Heras, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gargallo, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, S. A.</span></span> <span> </span><span class="NLM_article-title">Candidate Selection and Preclinical Evaluation of N-Tert-Butyl Isoquine (GSK369796), an Affordable and Effective 4-Aminoquinoline Antimalarial for the 21st Century</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1408</span>– <span class="NLM_lpage">1415</span>, <span class="refDoi"> DOI: 10.1021/jm8012618</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm8012618" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhvFShsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=1408-1415&author=P.+M.+O%E2%80%99Neillauthor=B.+K.+Parkauthor=A.+E.+Shoneauthor=J.+L.+Maggsauthor=P.+Robertsauthor=P.+A.+Stocksauthor=G.+A.+Biaginiauthor=P.+G.+Brayauthor=P.+Gibbonsauthor=N.+Berryauthor=P.+A.+Winstanleyauthor=A.+Mukhtarauthor=R.+Bonar-Lawauthor=S.+Hindleyauthor=R.+B.+Bambalauthor=C.+B.+Davisauthor=M.+Batesauthor=T.+K.+Hartauthor=S.+L.+Greshamauthor=R.+M.+Lawrenceauthor=R.+A.+Brigandiauthor=F.+M.+Gomez-delas-Herasauthor=D.+V.+Gargalloauthor=S.+A.+Ward&title=Candidate+Selection+and+Preclinical+Evaluation+of+N-Tert-Butyl+Isoquine+%28GSK369796%29%2C+an+Affordable+and+Effective+4-Aminoquinoline+Antimalarial+for+the+21st+Century&doi=10.1021%2Fjm8012618"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Candidate Selection and Preclinical Evaluation of N-tert-Butyl Isoquine (GSK369796), An Affordable and Effective 4-Aminoquinoline Antimalarial for the 21st Century</span></div><div class="casAuthors">O'Neill, Paul M.; Park, B. Kevin; Shone, Alison E.; Maggs, James L.; Roberts, Phillip; Stocks, Paul A.; Biagini, Giancarlo A.; Bray, Patrick G.; Gibbons, Peter; Berry, Neil; Winstanley, Peter A.; Mukhtar, Amira; Bonar-Law, Richard; Hindley, Stephen; Bambal, Ramesh B.; Davis, Charles B.; Bates, Martin; Hart, Timothy K.; Gresham, Stephanie L.; Lawrence, Ron M.; Brigandi, Richard A.; Gomez-delas-Heras, Federico M.; Gargallo, Domingo V.; Ward, Stephen A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1408-1415</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">N-tert-Bu isoquine (4) (GSK369796) is a 4-aminoquinoline drug candidate selected and developed as part of a public-private partnership between academics at Liverpool, MMV, and GSK pharmaceuticals.  This mol. was rationally designed based on chem., toxicol., pharmacokinetic, and pharmacodynamic considerations and was selected based on excellent activity against Plasmodium falciparum in vitro and rodent malaria parasites in vivo.  The optimized chem. delivered this novel synthetic quinoline in a two-step procedure from cheap and readily available starting materials.  The mol. has a full industry std. preclin. development program allowing first into humans to proceed.  Employing chloroquine (1) and amodiaquine (2) as comparator mols. in the preclin. plan, the first preclin. dossier of pharmacokinetic, toxicity, and safety pharmacol. has also been established for the 4-aminoquinoline antimalarial class.  These studies have revealed preclin. liabilities that have never translated into the human experience.  This has resulted in the availability of crit. information to other drug development teams interested in developing antimalarials within this class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpD8p09GaCH5bVg90H21EOLACvtfcHk0ljRSyDhWDCxzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhvFShsbo%253D&md5=d08b37c74a7393eaefb5b86fd5a265dc</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Fjm8012618&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm8012618%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Neill%26aufirst%3DP.%2BM.%26aulast%3DPark%26aufirst%3DB.%2BK.%26aulast%3DShone%26aufirst%3DA.%2BE.%26aulast%3DMaggs%26aufirst%3DJ.%2BL.%26aulast%3DRoberts%26aufirst%3DP.%26aulast%3DStocks%26aufirst%3DP.%2BA.%26aulast%3DBiagini%26aufirst%3DG.%2BA.%26aulast%3DBray%26aufirst%3DP.%2BG.%26aulast%3DGibbons%26aufirst%3DP.%26aulast%3DBerry%26aufirst%3DN.%26aulast%3DWinstanley%26aufirst%3DP.%2BA.%26aulast%3DMukhtar%26aufirst%3DA.%26aulast%3DBonar-Law%26aufirst%3DR.%26aulast%3DHindley%26aufirst%3DS.%26aulast%3DBambal%26aufirst%3DR.%2BB.%26aulast%3DDavis%26aufirst%3DC.%2BB.%26aulast%3DBates%26aufirst%3DM.%26aulast%3DHart%26aufirst%3DT.%2BK.%26aulast%3DGresham%26aufirst%3DS.%2BL.%26aulast%3DLawrence%26aufirst%3DR.%2BM.%26aulast%3DBrigandi%26aufirst%3DR.%2BA.%26aulast%3DGomez-delas-Heras%26aufirst%3DF.%2BM.%26aulast%3DGargallo%26aufirst%3DD.%2BV.%26aulast%3DWard%26aufirst%3DS.%2BA.%26atitle%3DCandidate%2520Selection%2520and%2520Preclinical%2520Evaluation%2520of%2520N-Tert-Butyl%2520Isoquine%2520%2528GSK369796%2529%252C%2520an%2520Affordable%2520and%2520Effective%25204-Aminoquinoline%2520Antimalarial%2520for%2520the%252021st%2520Century%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D1408%26epage%3D1415%26doi%3D10.1021%2Fjm8012618" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, E. S.</span></span> <span> </span><span class="NLM_article-title">Novel Anti-adipogenic Activity of Anti-malarial Amodiaquine Through Suppression of PPARγ Activity</span>. <i>Arch. Pharmacal Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">1336</span>– <span class="NLM_lpage">1343</span>, <span class="refDoi"> DOI: 10.1007/s12272-017-0965-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1007%2Fs12272-017-0965-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=29071567" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslWrtrnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2017&pages=1336-1343&author=T.+H.+Kimauthor=H.+K.+Kimauthor=E.+S.+Hwang&title=Novel+Anti-adipogenic+Activity+of+Anti-malarial+Amodiaquine+Through+Suppression+of+PPAR%CE%B3+Activity&doi=10.1007%2Fs12272-017-0965-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Novel anti-adipogenic activity of anti-malarial amodiaquine through suppression of PPARγ activity</span></div><div class="casAuthors">Kim, Tae Hee; Kim, Hyo Kyeong; Hwang, Eun Sook</div><div class="citationInfo"><span class="NLM_cas:title">Archives of Pharmacal Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1336-1343</span>CODEN:
                <span class="NLM_cas:coden">APHRDQ</span>;
        ISSN:<span class="NLM_cas:issn">0253-6269</span>.
    
            (<span class="NLM_cas:orgname">Pharmaceutical Society of Korea</span>)
        </div><div class="casAbstract">Amodiaquine (AQ) was developed as a selective drug against Plasmodium falciparum malaria infection and has received increasing attention as a therapeutic agent for the treatment of rheumatoid arthritis, Parkinson's disease, and cancer due to its anti-inflammatory, anti-proliferative, and autophagic-lysosomal blockade properties.  As autophagy activation is involved in promoting adipogenic differentiation, the authors examd. whether anti-autophagic AQ affected adipocyte differentiation of 3T3-L1 pre-adipocytes.  AQ dose-dependently and significantly suppressed adipocyte differentiation in conjunction with decreases in lipid droplet formation and expression of adipogenic markers including adiponectin, adipocyte fatty acid-binding protein 2 (aP2), resistin, and leptin.  Although peroxisome proliferator-activated receptor γ (PPARγ) decreases by inhibition of autophagy, AQ treatment did not induce PPARγ degrdn. despite the suppression of autophagolysosomal degrdn.  Instead, AQ suppressed the PPARγ activity to transcriptionally activate the aP2 gene transcription through the selective prevention of nuclear localization of PPARγ.  These results demonstrated the novel anti-adipogenic activity of AQ and identified its underlying mechanism that AQ suppressed adipogenic gene expression and lipid formation by inhibiting nuclear localization of PPARγ in an autophagy-independent manner.  AQ is recommended as a safe and effective anti-obesity drug for controlling overweight and obesity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpX9uBChrseGLVg90H21EOLACvtfcHk0lhfYpCa-eHwAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslWrtrnK&md5=886cf5f5dc483d57a46b6a3067d06d82</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1007%2Fs12272-017-0965-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs12272-017-0965-3%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DT.%2BH.%26aulast%3DKim%26aufirst%3DH.%2BK.%26aulast%3DHwang%26aufirst%3DE.%2BS.%26atitle%3DNovel%2520Anti-adipogenic%2520Activity%2520of%2520Anti-malarial%2520Amodiaquine%2520Through%2520Suppression%2520of%2520PPAR%25CE%25B3%2520Activity%26jtitle%3DArch.%2520Pharmacal%2520Res.%26date%3D2017%26volume%3D40%26spage%3D1336%26epage%3D1343%26doi%3D10.1007%2Fs12272-017-0965-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beland, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, J. L.</span></span> <span> </span><span class="NLM_article-title">Apoptosis Contributes to the Cytotoxicity Induced by Amodiaquine and Its Major Metabolite N-Desethylamodiaquine in Hepatic Cells</span>. <i>Toxicol. Vitr.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>62</i></span>, <span class="NLM_elocation-id">104669</span> <span class="refDoi"> DOI: 10.1016/j.tiv.2019.104669</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1016%2Fj.tiv.2019.104669" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=31629065" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A280%3ADC%252BB3MnpsFWnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2020&author=Y.+Tangauthor=Q.+Wuauthor=F.+A.+Belandauthor=S.+Chenauthor=J.+L.+Fang&title=Apoptosis+Contributes+to+the+Cytotoxicity+Induced+by+Amodiaquine+and+Its+Major+Metabolite+N-Desethylamodiaquine+in+Hepatic+Cells&doi=10.1016%2Fj.tiv.2019.104669"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Apoptosis contributes to the cytotoxicity induced by amodiaquine and its major metabolite N-desethylamodiaquine in hepatic cells</span></div><div class="casAuthors">Tang Yangshun; Wu Qiangen; Beland Frederick A; Chen Si; Fang Jia-Long</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology in vitro : an international journal published in association with BIBRA</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">104669</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Amodiaquine (ADQ), an antimalarial drug used in endemic areas, has been reported to be associated with liver toxicity; however, the mechanism underlying its hepatoxicity remains unclear.  In this study, we examined the cytotoxicity of ADQ and its major metabolite N-desethylamodiaquine (NADQ) and the effect of cytochrome P450 (CYP)-mediated metabolism on ADQ-induced cytotoxicity.  After a 48-h treatment, ADQ and NADQ caused cytotoxicity and induced apoptosis in HepG2 cells; NADQ was slightly more toxic than ADQ.  ADQ treatment decreased the levels of anti-apoptotic Bcl-2 family proteins, which was accompanied by an increase in the levels of pro-apoptotic Bcl-2 family proteins, indicating that ADQ-induced apoptosis was mediated by the Bcl-2 family.  NADQ treatment markedly increased the phosphorylation of JNK, extracellular signal-regulated kinase (ERK1/2), and p38, indicating that NADQ-induced apoptosis was mediated by MAPK signaling pathways.  Metabolic studies using microsomes obtained from HepG2 cell lines overexpressing human CYPs demonstrated that CYP1A1, 2C8, and 3A4 were the major enzymes that metabolized ADQ to NADQ and that CYP1A2, 1B1, 2C19, and 3A5 also metabolized ADQ, but to a lesser extent.  The cytotoxicity of ADQ was increased in CYP2C8 and 3A4 overexpressing HepG2 cells compared to HepG2/CYP vector cells, confirming that NADQ was more toxic than ADQ.  Moreover, treatment of CYP2C8 and 3A4 overexpressing HepG2 cells with ADQ increased the phosphorylation of JNK, ERK1/2, and p38, but not the expression of Bcl-2 family proteins.  Our findings indicate that ADQ and its major metabolite NADQ induce apoptosis, which is mediated by members of the Bcl-2 family and the activation of MAPK signaling pathways, respectively.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTuv5_RWP5HmyCEBeYuoi7AfW6udTcc2eYkeBjtYYE8Hrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MnpsFWnuw%253D%253D&md5=733ef335aa42e9a8ec6a8497d0c1d5fd</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.tiv.2019.104669&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tiv.2019.104669%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DQ.%26aulast%3DBeland%26aufirst%3DF.%2BA.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DFang%26aufirst%3DJ.%2BL.%26atitle%3DApoptosis%2520Contributes%2520to%2520the%2520Cytotoxicity%2520Induced%2520by%2520Amodiaquine%2520and%2520Its%2520Major%2520Metabolite%2520N-Desethylamodiaquine%2520in%2520Hepatic%2520Cells%26jtitle%3DToxicol.%2520Vitr.%26date%3D2020%26volume%3D62%26doi%3D10.1016%2Fj.tiv.2019.104669" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouhaddou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obernier, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Meara, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezelj, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swaney, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tummino, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hüttenhain, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaake, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tutuncuoglu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foussard, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batra, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modak, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polacco, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braberg, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fabius, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eckhardt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soucheray, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cakir, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGregor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roesch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vallet, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mac Kain, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miorin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreno, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naing, Z. Z. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirby, I. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnyk, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chorba, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrio-Hernandez, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Memon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernandez-Armenta, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathy, C. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perica, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilla, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganesan, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saltzberg, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rakesh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenthal, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calviello, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkataramanan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liboy-Lugo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wankowicz, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Safari, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ugur, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koh, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savar, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, Q. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shengjuler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Neal, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J. C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broadhurst, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klippsten, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharp, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wenzell, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuzuoglu-Ozturk, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trenker, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavero, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hiatt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rathore, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noack, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroud, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frankel, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, O. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verba, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agard, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emerman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Zastrow, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">d’Enfert, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malik, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujimori, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ideker, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floor, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gross, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sali, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruggero, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taunton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kortemme, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beltrao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vignuzzi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-Sastre, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krogan, N. J.</span></span> <span> </span><span class="NLM_article-title">A SARS-CoV-2 Protein Interaction Map Reveals Targets for Drug Repurposing</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>583</i></span>,  <span class="NLM_fpage">459</span>– <span class="NLM_lpage">468</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2286-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1038%2Fs41586-020-2286-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=32353859" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtlCqtbrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=583&publication_year=2020&pages=459-468&author=D.+E.+Gordonauthor=G.+M.+Jangauthor=M.+Bouhaddouauthor=J.+Xuauthor=K.+Obernierauthor=K.+M.+Whiteauthor=M.+J.+O%E2%80%99Mearaauthor=V.+V.+Rezeljauthor=J.+Z.+Guoauthor=D.+L.+Swaneyauthor=T.+A.+Tumminoauthor=R.+H%C3%BCttenhainauthor=R.+M.+Kaakeauthor=A.+L.+Richardsauthor=B.+Tutuncuogluauthor=H.+Foussardauthor=J.+Batraauthor=K.+Haasauthor=M.+Modakauthor=M.+Kimauthor=P.+Haasauthor=B.+J.+Polaccoauthor=H.+Brabergauthor=J.+M.+Fabiusauthor=M.+Eckhardtauthor=M.+Soucherayauthor=M.+J.+Bennettauthor=M.+Cakirauthor=M.+J.+McGregorauthor=Q.+Liauthor=B.+Meyerauthor=F.+Roeschauthor=T.+Valletauthor=A.+Mac+Kainauthor=L.+Miorinauthor=E.+Morenoauthor=Z.+Z.+C.+Naingauthor=Y.+Zhouauthor=S.+Pengauthor=Y.+Shiauthor=Z.+Zhangauthor=W.+Shenauthor=I.+T.+Kirbyauthor=J.+E.+Melnykauthor=J.+S.+Chorbaauthor=K.+Louauthor=S.+A.+Daiauthor=I.+Barrio-Hernandezauthor=D.+Memonauthor=C.+Hernandez-Armentaauthor=J.+Lyuauthor=C.+J.+P.+Mathyauthor=T.+Pericaauthor=K.+B.+Pillaauthor=S.+J.+Ganesanauthor=D.+J.+Saltzbergauthor=R.+Rakeshauthor=X.+Liuauthor=S.+B.+Rosenthalauthor=L.+Calvielloauthor=S.+Venkataramananauthor=J.+Liboy-Lugoauthor=Y.+Linauthor=X.+P.+Huangauthor=Y.+Liuauthor=S.+A.+Wankowiczauthor=M.+Bohnauthor=M.+Safariauthor=F.+S.+Ugurauthor=C.+Kohauthor=N.+S.+Savarauthor=Q.+D.+Tranauthor=D.+Shengjulerauthor=S.+J.+Fletcherauthor=M.+C.+O%E2%80%99Nealauthor=Y.+Caiauthor=J.+C.+J.+Changauthor=D.+J.+Broadhurstauthor=S.+Klippstenauthor=P.+P.+Sharpauthor=N.+A.+Wenzellauthor=D.+Kuzuoglu-Ozturkauthor=H.+Y.+Wangauthor=R.+Trenkerauthor=J.+M.+Youngauthor=D.+A.+Caveroauthor=J.+Hiattauthor=T.+L.+Rothauthor=U.+Rathoreauthor=A.+Subramanianauthor=J.+Noackauthor=M.+Hubertauthor=R.+M.+Stroudauthor=A.+D.+Frankelauthor=O.+S.+Rosenbergauthor=K.+A.+Verbaauthor=D.+A.+Agardauthor=M.+Ottauthor=M.+Emermanauthor=N.+Juraauthor=M.+von+Zastrowauthor=E.+Verdinauthor=A.+Ashworthauthor=O.+Schwartzauthor=C.+d%E2%80%99Enfertauthor=S.+Mukherjeeauthor=M.+Jacobsonauthor=H.+S.+Malikauthor=D.+G.+Fujimoriauthor=T.+Idekerauthor=C.+S.+Craikauthor=S.+N.+Floorauthor=J.+S.+Fraserauthor=J.+D.+Grossauthor=A.+Saliauthor=B.+L.+Rothauthor=D.+Ruggeroauthor=J.+Tauntonauthor=T.+Kortemmeauthor=P.+Beltraoauthor=M.+Vignuzziauthor=A.+Garc%C3%ADa-Sastreauthor=K.+M.+Shokatauthor=B.+K.+Shoichetauthor=N.+J.+Krogan&title=A+SARS-CoV-2+Protein+Interaction+Map+Reveals+Targets+for+Drug+Repurposing&doi=10.1038%2Fs41586-020-2286-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">A SARS-CoV-2 protein interaction map reveals targets for drug repurposing</span></div><div class="casAuthors">Gordon, David E.; Jang, Gwendolyn M.; Bouhaddou, Mehdi; Xu, Jiewei; Obernier, Kirsten; White, Kris M.; O'Meara, Matthew J.; Rezelj, Veronica V.; Guo, Jeffrey Z.; Swaney, Danielle L.; Tummino, Tia A.; Huttenhain, Ruth; Kaake, Robyn M.; Richards, Alicia L.; Tutuncuoglu, Beril; Foussard, Helene; Batra, Jyoti; Haas, Kelsey; Modak, Maya; Kim, Minkyu; Haas, Paige; Polacco, Benjamin J.; Braberg, Hannes; Fabius, Jacqueline M.; Eckhardt, Manon; Soucheray, Margaret; Bennett, Melanie J.; Cakir, Merve; McGregor, Michael J.; Li, Qiongyu; Meyer, Bjoern; Roesch, Ferdinand; Vallet, Thomas; Mac Kain, Alice; Miorin, Lisa; Moreno, Elena; Naing, Zun Zar Chi; Zhou, Yuan; Peng, Shiming; Shi, Ying; Zhang, Ziyang; Shen, Wenqi; Kirby, Ilsa T.; Melnyk, James E.; Chorba, John S.; Lou, Kevin; Dai, Shizhong A.; Barrio-Hernandez, Inigo; Memon, Danish; Hernandez-Armenta, Claudia; Lyu, Jiankun; Mathy, Christopher J. P.; Perica, Tina; Pilla, Kala Bharath; Ganesan, Sai J.; Saltzberg, Daniel J.; Rakesh, Ramachandran; Liu, Xi; Rosenthal, Sara B.; Calviello, Lorenzo; Venkataramanan, Srivats; Liboy-Lugo, Jose; Lin, Yizhu; Huang, Xi-Ping; Liu, YongFeng; Wankowicz, Stephanie A.; Bohn, Markus; Safari, Maliheh; Ugur, Fatima S.; Koh, Cassandra; Savar, Nastaran Sadat; Tran, Quang Dinh; Shengjuler, Djoshkun; Fletcher, Sabrina J.; O'Neal, Michael C.; Cai, Yiming; Chang, Jason C. J.; Broadhurst, David J.; Klippsten, Saker; Sharp, Phillip P.; Wenzell, Nicole A.; Kuzuoglu-Ozturk, Duygu; Wang, Hao-Yuan; Trenker, Raphael; Young, Janet M.; Cavero, Devin A.; Hiatt, Joseph; Roth, Theodore L.; Rathore, Ujjwal; Subramanian, Advait; Noack, Julia; Hubert, Mathieu; Stroud, Robert M.; Frankel, Alan D.; Rosenberg, Oren S.; Verba, Kliment A.; Agard, David A.; Ott, Melanie; Emerman, Michael; Jura, Natalia; von Zastrow, Mark; Verdin, Eric; Ashworth, Alan; Schwartz, Olivier; d'Enfert, Christophe; Mukherjee, Shaeri; Jacobson, Matt; Malik, Harmit S.; Fujimori, Danica G.; Ideker, Trey; Craik, Charles S.; Floor, Stephen N.; Fraser, James S.; Gross, John D.; Sali, Andrej; Roth, Bryan L.; Ruggero, Davide; Taunton, Jack; Kortemme, Tanja; Beltrao, Pedro; Vignuzzi, Marco; Garcia-Sastre, Adolfo; Shokat, Kevan M.; Shoichet, Brian K.; Krogan, Nevan J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">583</span>
        (<span class="NLM_cas:issue">7816</span>),
    <span class="NLM_cas:pages">459-468</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: A newly described coronavirus named severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), which is the causative agent of coronavirus disease 2019 (COVID-19), has infected over 2.3 million people, led to the death of more than 160,000 individuals and caused worldwide social and economic disruption1,2.  There are no antiviral drugs with proven clin. efficacy for the treatment of COVID-19, nor are there any vaccines that prevent infection with SARS-CoV-2, and efforts to develop drugs and vaccines are hampered by the limited knowledge of the mol. details of how SARS-CoV-2 infects cells.  Here we cloned, tagged and expressed 26 of the 29 SARS-CoV-2 proteins in human cells and identified the human proteins that phys. assocd. with each of the SARS-CoV-2 proteins using affinity-purifn. mass spectrometry, identifying 332 high-confidence protein-protein interactions between SARS-CoV-2 and human proteins.  Among these, we identify 66 druggable human proteins or host factors targeted by 69 compds. (of which, 29 drugs are approved by the US Food and Drug Administration, 12 are in clin. trials and 28 are preclin. compds.).  We screened a subset of these in multiple viral assays and found two sets of pharmacol. agents that displayed antiviral activity: inhibitors of mRNA translation and predicted regulators of the sigma-1 and sigma-2 receptors.  Further studies of these host-factor-targeting agents, including their combination with drugs that directly target viral enzymes, could lead to a therapeutic regimen to treat COVID-19.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDvJUIvi-Ws7Vg90H21EOLACvtfcHk0ljomIH4PX3j_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtlCqtbrI&md5=ce8b172ae73aacd3b3d8bb64877f4435</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2286-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2286-9%26sid%3Dliteratum%253Aachs%26aulast%3DGordon%26aufirst%3DD.%2BE.%26aulast%3DJang%26aufirst%3DG.%2BM.%26aulast%3DBouhaddou%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DObernier%26aufirst%3DK.%26aulast%3DWhite%26aufirst%3DK.%2BM.%26aulast%3DO%25E2%2580%2599Meara%26aufirst%3DM.%2BJ.%26aulast%3DRezelj%26aufirst%3DV.%2BV.%26aulast%3DGuo%26aufirst%3DJ.%2BZ.%26aulast%3DSwaney%26aufirst%3DD.%2BL.%26aulast%3DTummino%26aufirst%3DT.%2BA.%26aulast%3DH%25C3%25BCttenhain%26aufirst%3DR.%26aulast%3DKaake%26aufirst%3DR.%2BM.%26aulast%3DRichards%26aufirst%3DA.%2BL.%26aulast%3DTutuncuoglu%26aufirst%3DB.%26aulast%3DFoussard%26aufirst%3DH.%26aulast%3DBatra%26aufirst%3DJ.%26aulast%3DHaas%26aufirst%3DK.%26aulast%3DModak%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DM.%26aulast%3DHaas%26aufirst%3DP.%26aulast%3DPolacco%26aufirst%3DB.%2BJ.%26aulast%3DBraberg%26aufirst%3DH.%26aulast%3DFabius%26aufirst%3DJ.%2BM.%26aulast%3DEckhardt%26aufirst%3DM.%26aulast%3DSoucheray%26aufirst%3DM.%26aulast%3DBennett%26aufirst%3DM.%2BJ.%26aulast%3DCakir%26aufirst%3DM.%26aulast%3DMcGregor%26aufirst%3DM.%2BJ.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DMeyer%26aufirst%3DB.%26aulast%3DRoesch%26aufirst%3DF.%26aulast%3DVallet%26aufirst%3DT.%26aulast%3DMac%2BKain%26aufirst%3DA.%26aulast%3DMiorin%26aufirst%3DL.%26aulast%3DMoreno%26aufirst%3DE.%26aulast%3DNaing%26aufirst%3DZ.%2BZ.%2BC.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DPeng%26aufirst%3DS.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DShen%26aufirst%3DW.%26aulast%3DKirby%26aufirst%3DI.%2BT.%26aulast%3DMelnyk%26aufirst%3DJ.%2BE.%26aulast%3DChorba%26aufirst%3DJ.%2BS.%26aulast%3DLou%26aufirst%3DK.%26aulast%3DDai%26aufirst%3DS.%2BA.%26aulast%3DBarrio-Hernandez%26aufirst%3DI.%26aulast%3DMemon%26aufirst%3DD.%26aulast%3DHernandez-Armenta%26aufirst%3DC.%26aulast%3DLyu%26aufirst%3DJ.%26aulast%3DMathy%26aufirst%3DC.%2BJ.%2BP.%26aulast%3DPerica%26aufirst%3DT.%26aulast%3DPilla%26aufirst%3DK.%2BB.%26aulast%3DGanesan%26aufirst%3DS.%2BJ.%26aulast%3DSaltzberg%26aufirst%3DD.%2BJ.%26aulast%3DRakesh%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DRosenthal%26aufirst%3DS.%2BB.%26aulast%3DCalviello%26aufirst%3DL.%26aulast%3DVenkataramanan%26aufirst%3DS.%26aulast%3DLiboy-Lugo%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DX.%2BP.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWankowicz%26aufirst%3DS.%2BA.%26aulast%3DBohn%26aufirst%3DM.%26aulast%3DSafari%26aufirst%3DM.%26aulast%3DUgur%26aufirst%3DF.%2BS.%26aulast%3DKoh%26aufirst%3DC.%26aulast%3DSavar%26aufirst%3DN.%2BS.%26aulast%3DTran%26aufirst%3DQ.%2BD.%26aulast%3DShengjuler%26aufirst%3DD.%26aulast%3DFletcher%26aufirst%3DS.%2BJ.%26aulast%3DO%25E2%2580%2599Neal%26aufirst%3DM.%2BC.%26aulast%3DCai%26aufirst%3DY.%26aulast%3DChang%26aufirst%3DJ.%2BC.%2BJ.%26aulast%3DBroadhurst%26aufirst%3DD.%2BJ.%26aulast%3DKlippsten%26aufirst%3DS.%26aulast%3DSharp%26aufirst%3DP.%2BP.%26aulast%3DWenzell%26aufirst%3DN.%2BA.%26aulast%3DKuzuoglu-Ozturk%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DH.%2BY.%26aulast%3DTrenker%26aufirst%3DR.%26aulast%3DYoung%26aufirst%3DJ.%2BM.%26aulast%3DCavero%26aufirst%3DD.%2BA.%26aulast%3DHiatt%26aufirst%3DJ.%26aulast%3DRoth%26aufirst%3DT.%2BL.%26aulast%3DRathore%26aufirst%3DU.%26aulast%3DSubramanian%26aufirst%3DA.%26aulast%3DNoack%26aufirst%3DJ.%26aulast%3DHubert%26aufirst%3DM.%26aulast%3DStroud%26aufirst%3DR.%2BM.%26aulast%3DFrankel%26aufirst%3DA.%2BD.%26aulast%3DRosenberg%26aufirst%3DO.%2BS.%26aulast%3DVerba%26aufirst%3DK.%2BA.%26aulast%3DAgard%26aufirst%3DD.%2BA.%26aulast%3DOtt%26aufirst%3DM.%26aulast%3DEmerman%26aufirst%3DM.%26aulast%3DJura%26aufirst%3DN.%26aulast%3Dvon%2BZastrow%26aufirst%3DM.%26aulast%3DVerdin%26aufirst%3DE.%26aulast%3DAshworth%26aufirst%3DA.%26aulast%3DSchwartz%26aufirst%3DO.%26aulast%3Dd%25E2%2580%2599Enfert%26aufirst%3DC.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DJacobson%26aufirst%3DM.%26aulast%3DMalik%26aufirst%3DH.%2BS.%26aulast%3DFujimori%26aufirst%3DD.%2BG.%26aulast%3DIdeker%26aufirst%3DT.%26aulast%3DCraik%26aufirst%3DC.%2BS.%26aulast%3DFloor%26aufirst%3DS.%2BN.%26aulast%3DFraser%26aufirst%3DJ.%2BS.%26aulast%3DGross%26aufirst%3DJ.%2BD.%26aulast%3DSali%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DRuggero%26aufirst%3DD.%26aulast%3DTaunton%26aufirst%3DJ.%26aulast%3DKortemme%26aufirst%3DT.%26aulast%3DBeltrao%26aufirst%3DP.%26aulast%3DVignuzzi%26aufirst%3DM.%26aulast%3DGarc%25C3%25ADa-Sastre%26aufirst%3DA.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26aulast%3DKrogan%26aufirst%3DN.%2BJ.%26atitle%3DA%2520SARS-CoV-2%2520Protein%2520Interaction%2520Map%2520Reveals%2520Targets%2520for%2520Drug%2520Repurposing%26jtitle%3DNature%26date%3D2020%26volume%3D583%26spage%3D459%26epage%3D468%26doi%3D10.1038%2Fs41586-020-2286-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nolan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caraher, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humphries, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bettley, H. A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryce, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stratford, I. J.</span></span> <span> </span><span class="NLM_article-title">In Silico Identification and Biochemical Evaluation of Novel Inhibitors of NRH:Quinone Oxidoreductase 2 (NQO2)</span>. <i>Bioorganic Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">7331</span>– <span class="NLM_lpage">7336</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2010.10.070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1016%2Fj.bmcl.2010.10.070" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=21074425" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVKjtb%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=7331-7336&author=K.+A.+Nolanauthor=M.+C.+Caraherauthor=M.+P.+Humphriesauthor=H.+A.+A.+Bettleyauthor=R.+A.+Bryceauthor=I.+J.+Stratford&title=In+Silico+Identification+and+Biochemical+Evaluation+of+Novel+Inhibitors+of+NRH%3AQuinone+Oxidoreductase+2+%28NQO2%29&doi=10.1016%2Fj.bmcl.2010.10.070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">In silico identification and biochemical evaluation of novel inhibitors of NRH:quinone oxidoreductase 2 (NQO2)</span></div><div class="casAuthors">Nolan, Karen A.; Caraher, Mary C.; Humphries, Matthew P.; Abdel-Aal Bettley, Hoda; Bryce, Richard A.; Stratford, Ian J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7331-7336</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The NCI chem. database has been screened using in silico docking to identify novel inhibitors of NRH:quinone oxidoreductase 2 (NQO2).  Compds. identified from the screen exhibit a diverse range of scaffolds and inhibitory potencies are generally in the micromolar range.  Some of the compds. also have the ability to inhibit NQO1.  The modes of binding of the different compds. to the two enzymes are illustrated and discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_GDVj_qQdRLVg90H21EOLACvtfcHk0ljomIH4PX3j_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVKjtb%252FI&md5=62687f38eb146c58c3c8c085fab0d24f</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2010.10.070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2010.10.070%26sid%3Dliteratum%253Aachs%26aulast%3DNolan%26aufirst%3DK.%2BA.%26aulast%3DCaraher%26aufirst%3DM.%2BC.%26aulast%3DHumphries%26aufirst%3DM.%2BP.%26aulast%3DBettley%26aufirst%3DH.%2BA.%2BA.%26aulast%3DBryce%26aufirst%3DR.%2BA.%26aulast%3DStratford%26aufirst%3DI.%2BJ.%26atitle%3DIn%2520Silico%2520Identification%2520and%2520Biochemical%2520Evaluation%2520of%2520Novel%2520Inhibitors%2520of%2520NRH%253AQuinone%2520Oxidoreductase%25202%2520%2528NQO2%2529%26jtitle%3DBioorganic%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D20%26spage%3D7331%26epage%3D7336%26doi%3D10.1016%2Fj.bmcl.2010.10.070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Munoz-Tello, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Vera, I. M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamenecka, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kojetin, D. J.</span></span> <span> </span><span class="NLM_article-title">Assessment of NR4A Ligands That Directly Bind and Modulate the Orphan Nuclear Receptor Nurr1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">15639</span>– <span class="NLM_lpage">15654</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00894</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00894" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisFShsrzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=15639-15654&author=P.+Munoz-Telloauthor=H.+Linauthor=P.+Khanauthor=I.+M.+S.+De+Veraauthor=T.+M.+Kameneckaauthor=D.+J.+Kojetin&title=Assessment+of+NR4A+Ligands+That+Directly+Bind+and+Modulate+the+Orphan+Nuclear+Receptor+Nurr1&doi=10.1021%2Facs.jmedchem.0c00894"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Assessment of NR4A Ligands That Directly Bind and Modulate the Orphan Nuclear Receptor Nurr1</span></div><div class="casAuthors">Munoz-Tello, Paola; Lin, Hua; Khan, Pasha; de Vera, Ian Mitchelle S.; Kamenecka, Theodore M.; Kojetin, Douglas J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">15639-15654</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nurr1/NR4A2 is an orphan nuclear receptor transcription factor implicated as a drug target for neurol. disorders including Alzheimer's and Parkinson's diseases.  Previous studies identified small-mol. NR4A nuclear receptor modulators, but it remains unclear if these ligands affect transcription via direct binding to Nurr1.  We assessed 12 ligands reported to affect NR4A activity for Nurr1-dependent and Nurr1-independent transcriptional effects and the ability to bind the Nurr1 ligand-binding domain (LBD).  Protein NMR structural footprinting data show that amodiaquine, chloroquine, and cytosporone B bind the Nurr1 LBD; ligands that do not bind include C-DIM12, celastrol, camptothecin, IP7e, isoalantolactone, Et 2-[2,3,4-trimethoxy-6-(1-octanoyl)phenyl]acetate (TMPA), and three high-throughput screening hit derivs.  Importantly, ligands that modulate Nurr1 transcription also show Nurr1-independent effects on transcription in a cell type-specific manner, indicating that care should be taken when interpreting the functional response of these ligands in transcriptional assays.  These findings should help focus medicinal chem. efforts that desire to optimize Nurr1-binding ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooBB2VAxtBXrVg90H21EOLACvtfcHk0liNsvYI-CctgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisFShsrzL&md5=6f76ae9c9cb563d7e4ed80989366acc4</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00894&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00894%26sid%3Dliteratum%253Aachs%26aulast%3DMunoz-Tello%26aufirst%3DP.%26aulast%3DLin%26aufirst%3DH.%26aulast%3DKhan%26aufirst%3DP.%26aulast%3DDe%2BVera%26aufirst%3DI.%2BM.%2BS.%26aulast%3DKamenecka%26aufirst%3DT.%2BM.%26aulast%3DKojetin%26aufirst%3DD.%2BJ.%26atitle%3DAssessment%2520of%2520NR4A%2520Ligands%2520That%2520Directly%2520Bind%2520and%2520Modulate%2520the%2520Orphan%2520Nuclear%2520Receptor%2520Nurr1%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D15639%26epage%3D15654%26doi%3D10.1021%2Facs.jmedchem.0c00894" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Apelt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ligneau, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pertz, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganellin, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schunack, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stark, H.</span></span> <span> </span><span class="NLM_article-title">Development of a New Class of Nonimidazole Histamine H 3 Receptor Ligands with Combined Inhibitory Histamine N-Methyltransferase Activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">1128</span>– <span class="NLM_lpage">1141</span>, <span class="refDoi"> DOI: 10.1021/jm0110845</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0110845" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BD38Xps1aitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=1128-1141&author=J.+Apeltauthor=X.+Ligneauauthor=H.+H.+Pertzauthor=J.+Arrangauthor=C.+R.+Ganellinauthor=J.-C.+Schwartzauthor=W.+Schunackauthor=H.+Stark&title=Development+of+a+New+Class+of+Nonimidazole+Histamine+H+3+Receptor+Ligands+with+Combined+Inhibitory+Histamine+N-Methyltransferase+Activity&doi=10.1021%2Fjm0110845"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a New Class of Nonimidazole Histamine H3 Receptor Ligands with Combined Inhibitory Histamine N-Methyltransferase Activity</span></div><div class="casAuthors">Apelt, Joachim; Ligneau, Xavier; Pertz, Heinz H.; Arrang, Jean-Michel; Ganellin, C. Robin; Schwartz, Jean-Charles; Schunack, Walter; Stark, Holger</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1128-1141</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In search of novel ways to enhance histaminergic neurotransmission in the central nervous system, a new class of nonimidazole histamine H3 receptor ligands were developed that simultaneously possess strong inhibitory activity on the main histamine metabolizing enzyme, histamine N-methyltransferase (HMT).  The novel compds. contain an aminoquinoline moiety, which is an important structural feature for HMT inhibitory activity, connected by different spacers to a piperidino group (for H3 receptor antagonism).  Variation of the spacer structure provides two different series of compds.  One series, having only an alkylene spacer between the basic centers, led to highly potent HMT inhibitors with moderate to high affinity at human histamine H3 receptors.  The second series possesses a p-phenoxypropyl spacer, which may be extended by another alkylene chain.  This latter series also showed strong inhibitory activity on HMT, and in most cases, the H3 receptor affinity even surpassed that of the first series.  One of the most potent compds. with this dual mode of action is 4-(4-(3-piperidinopropoxy)phenylamino)quinoline (hH3, Ki = 0.09 nM; HMT, IC50 = 51 nM).  This class of compds. showed high antagonist potency and good H3 receptor selectivity in functional assays in guinea pig on H1, H2, and H3 receptors.  Because of low or missing in vivo activity of two selected compds., the proof of concept of these valuable pharmacol. tools for the supposed superior overall enhancing effect on histaminergic neurotransmission failed to appear hitherto.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbq6iENBrXJLVg90H21EOLACvtfcHk0liNsvYI-CctgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xps1aitA%253D%253D&md5=987730d70810f674eb0173a09591fd44</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjm0110845&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0110845%26sid%3Dliteratum%253Aachs%26aulast%3DApelt%26aufirst%3DJ.%26aulast%3DLigneau%26aufirst%3DX.%26aulast%3DPertz%26aufirst%3DH.%2BH.%26aulast%3DArrang%26aufirst%3DJ.%26aulast%3DGanellin%26aufirst%3DC.%2BR.%26aulast%3DSchwartz%26aufirst%3DJ.-C.%26aulast%3DSchunack%26aufirst%3DW.%26aulast%3DStark%26aufirst%3DH.%26atitle%3DDevelopment%2520of%2520a%2520New%2520Class%2520of%2520Nonimidazole%2520Histamine%2520H%25203%2520Receptor%2520Ligands%2520with%2520Combined%2520Inhibitory%2520Histamine%2520N-Methyltransferase%2520Activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D1128%26epage%3D1141%26doi%3D10.1021%2Fjm0110845" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sadowski, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Triezenberg, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ptashne, M.</span></span> <span> </span><span class="NLM_article-title">GAL4-VP16 Is an Unusually Potent Transcriptional Activator</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>335</i></span>,  <span class="NLM_fpage">563</span>– <span class="NLM_lpage">564</span>, <span class="refDoi"> DOI: 10.1038/335563a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1038%2F335563a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=3047590" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADyaL1MXitVSgsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=335&publication_year=1988&pages=563-564&author=I.+Sadowskiauthor=J.+Maauthor=S.+Triezenbergauthor=M.+Ptashne&title=GAL4-VP16+Is+an+Unusually+Potent+Transcriptional+Activator&doi=10.1038%2F335563a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">GAL4-VP16 is an unusually potent transcriptional activator</span></div><div class="casAuthors">Sadowski, Ivan; Ma, Jun; Triezenberg, Steve; Ptashne, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">335</span>
        (<span class="NLM_cas:issue">6190</span>),
    <span class="NLM_cas:pages">563-4</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">A new transcriptional activator is described which was formed by fusing a DNA-binding fragment of the yeast activator GAL4 to a highly acidic portion of the herpes simplex virus protein VP16.  VP16 activates transcription of immediate early viral genes by using its N-terminal sequences to attach to one or more host-encoded proteins that recognize DNA sequences in their promoters.  The hybrid protein (GAL4-VP16) activates transcription unusually efficiently in mammalian cells when bound close to, or at large distances from the gene.  The activating region of VP16 may be near-maximally potent, and it may not be coincidental that such a strong activator is encoded by a virus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCRLe_4KX8rrVg90H21EOLACvtfcHk0liNsvYI-CctgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXitVSgsg%253D%253D&md5=5ff658f48cec708ce35a3d687adb8b3f</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2F335563a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F335563a0%26sid%3Dliteratum%253Aachs%26aulast%3DSadowski%26aufirst%3DI.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DTriezenberg%26aufirst%3DS.%26aulast%3DPtashne%26aufirst%3DM.%26atitle%3DGAL4-VP16%2520Is%2520an%2520Unusually%2520Potent%2520Transcriptional%2520Activator%26jtitle%3DNature%26date%3D1988%26volume%3D335%26spage%3D563%26epage%3D564%26doi%3D10.1038%2F335563a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Willems, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heering, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merk, D.</span></span> <span> </span><span class="NLM_article-title">The Orphan Nuclear Receptor Nurr1 Is Responsive to Non-steroidal Anti-inflammatory Drugs</span>. <i>Commun. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">85</span>, <span class="refDoi"> DOI: 10.1038/s42004-020-0331-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1038%2Fs42004-020-0331-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitVKksrvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2020&pages=85&author=S.+Willemsauthor=W.+Kiluauthor=X.+Niauthor=A.+Chaikuadauthor=S.+Knappauthor=J.+Heeringauthor=D.+Merk&title=The+Orphan+Nuclear+Receptor+Nurr1+Is+Responsive+to+Non-steroidal+Anti-inflammatory+Drugs&doi=10.1038%2Fs42004-020-0331-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">The orphan nuclear receptor Nurr1 is responsive to non-steroidal anti-inflammatory drugs</span></div><div class="casAuthors">Willems, Sabine; Kilu, Whitney; Ni, Xiaomin; Chaikuad, Apirat; Knapp, Stefan; Heering, Jan; Merk, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Communications Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">85</span>CODEN:
                <span class="NLM_cas:coden">CCOHCT</span>;
        ISSN:<span class="NLM_cas:issn">2399-3669</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Nuclear receptor related 1 (Nurr1) is an orphan ligand-activated transcription factor and considered as neuroprotective transcriptional regulator with great potential as therapeutic target for neurodegenerative diseases.  However, the collection of available Nurr1 modulators and mechanistic understanding of Nurr1 are limited.  Here, we report the discovery of several structurally diverse non-steroidal anti-inflammatory drugs as inverse Nurr1 agonists demonstrating that Nurr1 activity can be regulated bidirectionally.  As chem. tools, these ligands enable unraveling the co-regulatory network of Nurr1 and the mode of action distinguishing agonists from inverse agonists.  In addn. to its ability to dimerize, we observe an ability of Nurr1 to recruit several canonical nuclear receptor co-regulators in a ligand-dependent fashion.  Distinct dimerization states and co-regulator interaction patterns arise as discriminating factors of Nurr1 agonists and inverse agonists.  Our results contribute a valuable collection of Nurr1 modulators and relevant mechanistic insights for future Nurr1 target validation and drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4xAONvjGyx7Vg90H21EOLACvtfcHk0lg1zSU-VwUTaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitVKksrvE&md5=780001615544ae3f93a28007f8f492be</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fs42004-020-0331-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs42004-020-0331-0%26sid%3Dliteratum%253Aachs%26aulast%3DWillems%26aufirst%3DS.%26aulast%3DKilu%26aufirst%3DW.%26aulast%3DNi%26aufirst%3DX.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHeering%26aufirst%3DJ.%26aulast%3DMerk%26aufirst%3DD.%26atitle%3DThe%2520Orphan%2520Nuclear%2520Receptor%2520Nurr1%2520Is%2520Responsive%2520to%2520Non-steroidal%2520Anti-inflammatory%2520Drugs%26jtitle%3DCommun.%2520Chem.%26date%3D2020%26volume%3D3%26spage%3D85%26doi%3D10.1038%2Fs42004-020-0331-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, U. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohankumar, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michelhaugh, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mittal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tjalkens, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Safe, S.</span></span> <span> </span><span class="NLM_article-title">Nuclear Receptor 4A2 (NR4A2) Is a Druggable Target for Glioblastomas</span>. <i>J. Neurooncol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>146</i></span>,  <span class="NLM_fpage">25</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1007/s11060-019-03349-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1007%2Fs11060-019-03349-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=31754919" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A280%3ADC%252BB3MfhvFaqsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146&publication_year=2020&pages=25-39&author=K.+Karkiauthor=X.+Liauthor=U.+H.+Jinauthor=K.+Mohankumarauthor=M.+Zareiauthor=S.+K.+Michelhaughauthor=S.+Mittalauthor=R.+Tjalkensauthor=S.+Safe&title=Nuclear+Receptor+4A2+%28NR4A2%29+Is+a+Druggable+Target+for+Glioblastomas&doi=10.1007%2Fs11060-019-03349-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Nuclear receptor 4A2 (NR4A2) is a druggable target for glioblastomas</span></div><div class="casAuthors">Karki Keshav; Li Xi; Jin Un-Ho; Mohankumar Kumaravel; Zarei Mahsa; Safe Stephen; Michelhaugh Sharon K; Mittal Sandeep; Tjalkens Ronald; Safe Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of neuro-oncology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">146</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">25-39</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">INTRODUCTION:  The orphan nuclear receptor 4A2 (NR4A2) has been extensively characterized in subcellular regions of the brain and is necessary for the function of dopaminergic neurons.  The NR4A2 ligand, 1,1-bis (3(1)-indoly1)-1-(p-chlorophenyl)methane (DIM-C-pPhCl) inhibits markers of neuroinflammation and degeneration in mouse models and in this study we investigated expression and function of NR4A2 in glioblastoma (GBM).  METHODS:  Established and patient-derived cell lines were used as models and the expression and functions of NR4A2 were determined by western blots and NR4A2 gene silencing by antisense oligonucleotides respectively.  Effects of NR4A2 knockdown and DIM-C-pPhCl on cell growth, induction of apoptosis (Annexin V Staining) and migration/invasion (Boyden chamber and spheroid invasion assay) and transactivation of NR4A2-regulated reporter genes were determined.  Tumor growth was investigated in athymic nude mice bearing U87-MG cells as xenografts.  RESULTS:  NR4A2 knockdown and DIM-C-pPhCl inhibited GBM cell and tumor growth, induced apoptosis and inhibited migration and invasion of GBM cells.  DIM-C-pPhCl and related analogs also inhibited NR4A2-regulated transactivation (luciferase activity) confirming that DIM-C-pPhCl acts as an NR4A2 antagonist and blocks NR4A2-dependent pro-oncogenic responses in GBM.  CONCLUSION:  We demonstrate for the first time that NR4A2 is pro-oncogenic in GBM and thus a potential druggable target for patients with tumors expressing this receptor.  Moreover, our bis-indole-derived NR4A2 antagonists represent a novel class of anti-cancer agents with potential future clinical applications for treating GBM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSjWs8jIOORwOkdrW1FkXG-fW6udTcc2eZwvo62ER11lrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MfhvFaqsA%253D%253D&md5=8cce444908b7254c35e5092cbefe88d0</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1007%2Fs11060-019-03349-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11060-019-03349-y%26sid%3Dliteratum%253Aachs%26aulast%3DKarki%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DJin%26aufirst%3DU.%2BH.%26aulast%3DMohankumar%26aufirst%3DK.%26aulast%3DZarei%26aufirst%3DM.%26aulast%3DMichelhaugh%26aufirst%3DS.%2BK.%26aulast%3DMittal%26aufirst%3DS.%26aulast%3DTjalkens%26aufirst%3DR.%26aulast%3DSafe%26aufirst%3DS.%26atitle%3DNuclear%2520Receptor%25204A2%2520%2528NR4A2%2529%2520Is%2520a%2520Druggable%2520Target%2520for%2520Glioblastomas%26jtitle%3DJ.%2520Neurooncol.%26date%3D2020%26volume%3D146%26spage%3D25%26epage%3D39%26doi%3D10.1007%2Fs11060-019-03349-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keserü, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, C. H.</span></span> <span> </span><span class="NLM_article-title">The Role of Ligand Efficiency Metrics in Drug Discovery</span>. <i>Nat. Rev. Drug Discov.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">105</span>– <span class="NLM_lpage">121</span>, <span class="refDoi"> DOI: 10.1038/nrd4163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1038%2Fnrd4163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=24481311" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=105-121&author=A.+L.+Hopkinsauthor=G.+M.+Keser%C3%BCauthor=P.+D.+Leesonauthor=D.+C.+Reesauthor=C.+H.+Reynolds&title=The+Role+of+Ligand+Efficiency+Metrics+in+Drug+Discovery&doi=10.1038%2Fnrd4163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">The role of ligand efficiency metrics in drug discovery</span></div><div class="casAuthors">Hopkins, Andrew L.; Keserue, Gyoergy M.; Leeson, Paul D.; Rees, David C.; Reynolds, Charles H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">105-121</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The judicious application of ligand or binding efficiency metrics, which quantify the mol. properties required to obtain binding affinity for a drug target, is gaining traction in the selection and optimization of fragments, hits and leads.  Retrospective anal. of recently marketed oral drugs shows that they frequently have highly optimized ligand efficiency values for their targets.  Optimizing ligand efficiency metrics based on both mol. mass and lipophilicity, when set in the context of the specific target, has the potential to ameliorate the inflation of these properties that has been obsd. in current medicinal chem. practice, and to increase the quality of drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq96uhvE1OZ5LVg90H21EOLACvtfcHk0lg1zSU-VwUTaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1SmtLk%253D&md5=344030e59f499180f79d311302a27532</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnrd4163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4163%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DKeser%25C3%25BC%26aufirst%3DG.%2BM.%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DReynolds%26aufirst%3DC.%2BH.%26atitle%3DThe%2520Role%2520of%2520Ligand%2520Efficiency%2520Metrics%2520in%2520Drug%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discov.%26date%3D2014%26volume%3D13%26spage%3D105%26epage%3D121%26doi%3D10.1038%2Fnrd4163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tetko, I. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanchuk, V. Y.</span></span> <span> </span><span class="NLM_article-title">Application of Associative Neural Networks for Prediction of Lipophilicity in ALOGPS 2.1 Program</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">1136</span>– <span class="NLM_lpage">1145</span>, <span class="refDoi"> DOI: 10.1021/ci025515j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci025515j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BD38XlvFSmtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2002&pages=1136-1145&author=I.+V.+Tetkoauthor=V.+Y.+Tanchuk&title=Application+of+Associative+Neural+Networks+for+Prediction+of+Lipophilicity+in+ALOGPS+2.1+Program&doi=10.1021%2Fci025515j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Application of associative neural networks for prediction of lipophilicity in ALOGPS 2.1 program</span></div><div class="casAuthors">Tetko, Igor V.; Tanchuk, Vsevolod Yu.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1136-1145</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This article provides a systematic study of several important parameters of the Associative Neural Network (ASNN), such as the no. of networks in the ensemble, distance measures, neighbor functions, selection of smoothing parameters, and strategies for the user-training feature of the algorithm.  The performance of the different methods is assessed with several training/test sets used to predict lipophilicity of chem. compds.  The Spearman rank-order correlation coeff. and Parzen-window regression methods provide the best performance of the algorithm.  If addnl. user data is available, an improved prediction of lipophilicity of chems. up to 2-5 times can be calcd. when the appropriate smoothing parameters for the neural network are selected.  The detected best combinations of parameters and strategies are implemented in the ALOGPS 2.1 program that is publicly available at http://www.vcclab.org/lab/alogps.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9Nz8kkyB1NbVg90H21EOLACvtfcHk0lg1zSU-VwUTaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XlvFSmtrY%253D&md5=45d412977a5171d4e8b03ada28c3632c</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fci025515j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci025515j%26sid%3Dliteratum%253Aachs%26aulast%3DTetko%26aufirst%3DI.%2BV.%26aulast%3DTanchuk%26aufirst%3DV.%2BY.%26atitle%3DApplication%2520of%2520Associative%2520Neural%2520Networks%2520for%2520Prediction%2520of%2520Lipophilicity%2520in%2520ALOGPS%25202.1%2520Program%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2002%26volume%3D42%26spage%3D1136%26epage%3D1145%26doi%3D10.1021%2Fci025515j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niu, Y. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jing, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, T. Z.</span></span> <span> </span><span class="NLM_article-title">Amodiaquine-Induced Reproductive Toxicity in Adult Male Rats</span>. <i>Mol. Reprod. Dev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">174</span>– <span class="NLM_lpage">182</span>, <span class="refDoi"> DOI: 10.1002/mrd.22603</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1002%2Fmrd.22603" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=26647924" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVansbjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2016&pages=174-182&author=Y.+R.+Niuauthor=B.+Weiauthor=B.+Chenauthor=L.+H.+Xuauthor=X.+Jingauthor=C.+L.+Pengauthor=T.+Z.+Ma&title=Amodiaquine-Induced+Reproductive+Toxicity+in+Adult+Male+Rats&doi=10.1002%2Fmrd.22603"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Amodiaquine-induced reproductive toxicity in adult male rats</span></div><div class="casAuthors">Niu, Yan-Ru; Wei, Bing; Chen, Bi; Xu, Li-Hua; Jing, Xia; Peng, Cai-Ling; Ma, Tian-Zhong</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Reproduction & Development</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">174-182</span>CODEN:
                <span class="NLM_cas:coden">MREDEE</span>;
        ISSN:<span class="NLM_cas:issn">1040-452X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Amodiaquine (AQ) is routinely prescribed as an anti-malarial drug.  Here, we evaluated AQ-induced toxicity in the male reproductive system.  Eighty adult male Sprague-Dawley rats were randomly divided into four groups that received distd. water (control) or daily doses of 5 mg/kg body wt., 10 mg/kg, or 15 mg/kg AQ for 2 wk.  Testes morphol. was analyzed using hematoxylin-and-eosin staining, terminal dUTP nicked-end labeling (TUNEL), and immunostaining whereas protein expression was detd. by Western blotting.  AQ dose-dependently led to abnormal spermatogenesis.  Disruption of the blood-testis barrier and increased germ cell apoptosis were obsd. in all three AQ-treated groups.  Interestingly, AQ-induced damage of spermatogenesis recovered over time, based on the survival of promyelocytic leukemia zinc-finger (PLZF)-pos., undifferentiated spermatogonia.  Serum levels of LH and testosterone, as well as testicular testosterone levels, were not significantly altered in AQ-treated groups compared with controls.  Collectively, our study suggests that AQ exerts substantial acute side effects on the reproductive systems of adult male rats by inducing the apoptosis of differentiating spermatogenic cells and disruption of blood-testis barrier function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqoOsPZ7QEDXbVg90H21EOLACvtfcHk0ljpCmp-_7l5DQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVansbjF&md5=c8105eb83d6fed1172ff71a000d655f8</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Fmrd.22603&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmrd.22603%26sid%3Dliteratum%253Aachs%26aulast%3DNiu%26aufirst%3DY.%2BR.%26aulast%3DWei%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DB.%26aulast%3DXu%26aufirst%3DL.%2BH.%26aulast%3DJing%26aufirst%3DX.%26aulast%3DPeng%26aufirst%3DC.%2BL.%26aulast%3DMa%26aufirst%3DT.%2BZ.%26atitle%3DAmodiaquine-Induced%2520Reproductive%2520Toxicity%2520in%2520Adult%2520Male%2520Rats%26jtitle%3DMol.%2520Reprod.%2520Dev.%26date%3D2016%26volume%3D83%26spage%3D174%26epage%3D182%26doi%3D10.1002%2Fmrd.22603" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bernuau, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larrey, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campillo, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degott, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdier, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rueff, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pessayre, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benhamou, J. P.</span></span> <span> </span><span class="NLM_article-title">Amodiaquine-Induced Fulminant Hepatitis</span>. <i>J. Hepatol.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">109</span>– <span class="NLM_lpage">112</span>, <span class="refDoi"> DOI: 10.1016/S0168-8278(88)80469-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1016%2FS0168-8278%2888%2980469-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=3346531" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A280%3ADyaL1c7lsVeluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1988&pages=109-112&author=J.+Bernuauauthor=D.+Larreyauthor=B.+Campilloauthor=C.+Degottauthor=F.+Verdierauthor=B.+Rueffauthor=D.+Pessayreauthor=J.+P.+Benhamou&title=Amodiaquine-Induced+Fulminant+Hepatitis&doi=10.1016%2FS0168-8278%2888%2980469-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Amodiaquine-induced fulminant hepatitis</span></div><div class="casAuthors">Bernuau J; Larrey D; Campillo B; Degott C; Verdier F; Rueff B; Pessayre D; Benhamou J P</div><div class="citationInfo"><span class="NLM_cas:title">Journal of hepatology</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">109-12</span>
        ISSN:<span class="NLM_cas:issn">0168-8278</span>.
    </div><div class="casAbstract">Three patients suffered from fulminant hepatitis within 23, 59 and 22 weeks after having ingested a total dose of 16, 26 and 15 g, respectively, of amodiaquine for the prophylaxis of malaria.  Amodiaquine administration was continued for 44, 21 and 25 days after the onset of jaundice, respectively.  One patient underwent emergency orthotopic liver transplantation and survived.  The other two died.  Fulminant hepatitis threatens patients in whom amodiaquine administration is protracted for several months and not interrupted when jaundice occurs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSW1Unx4VGTl4IRAdBLKY41fW6udTcc2eaQOuaUve_XHLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaL1c7lsVeluw%253D%253D&md5=137e3d15331d0204ca4e8ab4eed16004</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2FS0168-8278%2888%2980469-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0168-8278%252888%252980469-0%26sid%3Dliteratum%253Aachs%26aulast%3DBernuau%26aufirst%3DJ.%26aulast%3DLarrey%26aufirst%3DD.%26aulast%3DCampillo%26aufirst%3DB.%26aulast%3DDegott%26aufirst%3DC.%26aulast%3DVerdier%26aufirst%3DF.%26aulast%3DRueff%26aufirst%3DB.%26aulast%3DPessayre%26aufirst%3DD.%26aulast%3DBenhamou%26aufirst%3DJ.%2BP.%26atitle%3DAmodiaquine-Induced%2520Fulminant%2520Hepatitis%26jtitle%3DJ.%2520Hepatol.%26date%3D1988%26volume%3D6%26spage%3D109%26epage%3D112%26doi%3D10.1016%2FS0168-8278%2888%2980469-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Budzyński, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puustinen, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joutsen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sistonen, L.</span></span> <span> </span><span class="NLM_article-title">Uncoupling Stress-Inducible Phosphorylation of Heat Shock Factor 1 from Its Activation</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">2530</span>– <span class="NLM_lpage">2540</span>, <span class="refDoi"> DOI: 10.1128/MCB.00816-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=10.1128%2FMCB.00816-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=25963659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFamurrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=2530-2540&author=M.+A.+Budzy%C5%84skiauthor=M.+C.+Puustinenauthor=J.+Joutsenauthor=L.+Sistonen&title=Uncoupling+Stress-Inducible+Phosphorylation+of+Heat+Shock+Factor+1+from+Its+Activation&doi=10.1128%2FMCB.00816-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Uncoupling stress-inducible phosphorylation of heat shock factor 1 from its activation</span></div><div class="casAuthors">Budzynski, Marek A.; Puustinen, Mikael C.; Joutsen, Jenny; Sistonen, Lea</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2530-2540</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">1098-5549</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">In mammals the stress-inducible expression of genes encoding heat shock proteins is under the control of the heat shock transcription factor 1 (HSF1).  Activation of HSF1 is a multistep process, involving trimerization, acquisition of DNA-binding and transcriptional activities, which coincide with several posttranslational modifications.  Stress-inducible phosphorylation of HSF1, or hyperphosphorylation, which occurs mainly within the regulatory domain (RD), has been proposed as a requirement for HSF-driven transcription and is widely used for assessing HSF1 activation.  Nonetheless, the contribution of hyperphosphorylation to the activity of HSF1 remains unknown.  In this study, we generated a phosphorylation-deficient HSF1 mutant (HSF1Δ∼PRD), where the 15 known phosphorylation sites within the RD were disrupted.  Our results show that the phosphorylation status of the RD does not affect the subcellular localization and DNA-binding activity of HSF1.  Surprisingly, under stress conditions, HSF1Δ∼PRD is a potent transactivator of both endogenous targets and a reporter gene, and HSF1Δ∼PRD has a reduced activation threshold.  Our results provide the first direct evidence for uncoupling stress-inducible phosphorylation of HSF1 from its activation, and we propose that the phosphorylation signature alone is not an appropriate marker for HSF1 activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGri3_8rGXXWrrVg90H21EOLACvtfcHk0ljpCmp-_7l5DQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFamurrP&md5=baff5411c24b2d2167af35caf17480c5</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1128%2FMCB.00816-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00816-14%26sid%3Dliteratum%253Aachs%26aulast%3DBudzy%25C5%2584ski%26aufirst%3DM.%2BA.%26aulast%3DPuustinen%26aufirst%3DM.%2BC.%26aulast%3DJoutsen%26aufirst%3DJ.%26aulast%3DSistonen%26aufirst%3DL.%26atitle%3DUncoupling%2520Stress-Inducible%2520Phosphorylation%2520of%2520Heat%2520Shock%2520Factor%25201%2520from%2520Its%2520Activation%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2015%26volume%3D35%26spage%3D2530%26epage%3D2540%26doi%3D10.1128%2FMCB.00816-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Merk, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmad, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrasco Gomez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinhilber, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schubert-Zsilavecz, M.</span></span> <span> </span><span class="NLM_article-title">Extending the Structure-Activity Relationship of Anthranilic Acid Derivatives as Farnesoid x Receptor Modulators: Development of a Highly Potent Partial Farnesoid X Receptor Agonist</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">8035</span>– <span class="NLM_lpage">8055</span>, <span class="refDoi"> DOI: 10.1021/jm500937v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500937v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFCjt7bE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8035-8055&author=D.+Merkauthor=C.+Lamersauthor=K.+Ahmadauthor=R.+Carrasco+Gomezauthor=G.+Schneiderauthor=D.+Steinhilberauthor=M.+Schubert-Zsilavecz&title=Extending+the+Structure-Activity+Relationship+of+Anthranilic+Acid+Derivatives+as+Farnesoid+x+Receptor+Modulators%3A+Development+of+a+Highly+Potent+Partial+Farnesoid+X+Receptor+Agonist&doi=10.1021%2Fjm500937v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Extending the Structure-Activity Relationship of Anthranilic Acid Derivatives As Farnesoid X Receptor Modulators: Development of a Highly Potent Partial Farnesoid X Receptor Agonist</span></div><div class="casAuthors">Merk, Daniel; Lamers, Christina; Ahmad, Khalil; Carrasco Gomez, Roberto; Schneider, Gisbert; Steinhilber, Dieter; Schubert-Zsilavecz, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8035-8055</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The ligand activated transcription factor nuclear farnesoid X receptor (FXR) is involved as a regulator in many metabolic pathways including bile acid and glucose homeostasis.  Therefore, pharmacol. activation of FXR seems a valuable therapeutic approach for several conditions including metabolic diseases linked to insulin resistance, liver disorders such as primary biliary cirrhosis or nonalcoholic steatohepatitis, and certain forms of cancer.  The available FXR agonists, however, activate the receptor to the full extent which might be disadvantageous over a longer time period.  Hence, partial FXR activators are required for long-term treatment of metabolic disorders.  Reported here are the SAR of anthranilic acid derivs. as FXR modulators and development, synthesis, and characterization of compd. (I), which is a highly potent partial FXR agonist in a reporter gene assay with an EC50 value of 8±3 nM and on mRNA level in liver cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxZMafrXUE1LVg90H21EOLACvtfcHk0lhEouOdBJAgLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFCjt7bE&md5=1b0b248cf689ad910d91a078823a0001</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm500937v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500937v%26sid%3Dliteratum%253Aachs%26aulast%3DMerk%26aufirst%3DD.%26aulast%3DLamers%26aufirst%3DC.%26aulast%3DAhmad%26aufirst%3DK.%26aulast%3DCarrasco%2BGomez%26aufirst%3DR.%26aulast%3DSchneider%26aufirst%3DG.%26aulast%3DSteinhilber%26aufirst%3DD.%26aulast%3DSchubert-Zsilavecz%26aufirst%3DM.%26atitle%3DExtending%2520the%2520Structure-Activity%2520Relationship%2520of%2520Anthranilic%2520Acid%2520Derivatives%2520as%2520Farnesoid%2520x%2520Receptor%2520Modulators%253A%2520Development%2520of%2520a%2520Highly%2520Potent%2520Partial%2520Farnesoid%2520X%2520Receptor%2520Agonist%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8035%26epage%3D8055%26doi%3D10.1021%2Fjm500937v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i23"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01779">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_00838"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01779?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01779</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Control experiments; nonspecific effects and cytotoxicity; raw luminescence data; effects of Nurr1 agonist 24; summarized cellular Nurr1 modulatory activities; and WST-1 toxicity assay (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01779/suppl_file/jm0c01779_si_001.pdf">PDF</a>)</p></li><li><p class="inline">HPLC traces of compounds synthesized in this study (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01779/suppl_file/jm0c01779_si_002.pdf">PDF</a>)</p></li><li><p class="inline"><b>Molecular formula strings</b> with structures and activity data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01779/suppl_file/jm0c01779_si_003.csv">csv</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01779/suppl_file/jm0c01779_si_001.pdf">jm0c01779_si_001.pdf (916.63 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01779/suppl_file/jm0c01779_si_002.pdf">jm0c01779_si_002.pdf (893.9 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01779/suppl_file/jm0c01779_si_003.csv">jm0c01779_si_003.csv (1.76 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01779&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-5%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01779%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01779" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799eebdca1bd203","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
