
<html lang="en"     class="pb-page"  data-request-id="34a2cad1-88df-4fb1-92e8-890f95a7be1b"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/jm3011806;issue:issue:10.1021/jmcmar.2012.55.issue-22;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Discovery of Selective and Potent Inhibitors of Gram-Positive Bacterial Thymidylate Kinase (TMK)" /></meta><meta name="dc.Creator" content="Gabriel  Martínez-Botella" /></meta><meta name="dc.Creator" content="John N.  Breen" /></meta><meta name="dc.Creator" content="James E. S.  Duffy" /></meta><meta name="dc.Creator" content="Jacques  Dumas" /></meta><meta name="dc.Creator" content="Bolin  Geng" /></meta><meta name="dc.Creator" content="Ian K.  Gowers" /></meta><meta name="dc.Creator" content="Oluyinka M.  Green" /></meta><meta name="dc.Creator" content="Satenig  Guler" /></meta><meta name="dc.Creator" content="Martin F.  Hentemann" /></meta><meta name="dc.Creator" content="Felix A.  Hernandez-Juan" /></meta><meta name="dc.Creator" content="Diane  Joseph-McCarthy" /></meta><meta name="dc.Creator" content="Sameer  Kawatkar" /></meta><meta name="dc.Creator" content="Nicholas A.  Larsen" /></meta><meta name="dc.Creator" content="Ovadia  Lazari" /></meta><meta name="dc.Creator" content="James T.  Loch" /></meta><meta name="dc.Creator" content="Jacqueline A.  Macritchie" /></meta><meta name="dc.Creator" content="Andrew R.  McKenzie" /></meta><meta name="dc.Creator" content="Joseph V.  Newman" /></meta><meta name="dc.Creator" content="Nelson B.  Olivier" /></meta><meta name="dc.Creator" content="Linda G.  Otterson" /></meta><meta name="dc.Creator" content="Andrew P.  Owens" /></meta><meta name="dc.Creator" content="Jon  Read" /></meta><meta name="dc.Creator" content="David W.  Sheppard" /></meta><meta name="dc.Creator" content="Thomas A.  Keating" /></meta><meta name="dc.Description" content="Thymidylate kinase (TMK) is an essential enzyme in bacterial DNA synthesis. The deoxythymidine monophosphate (dTMP) substrate binding pocket was targeted in a rational-design, structure-supported e..." /></meta><meta name="Description" content="Thymidylate kinase (TMK) is an essential enzyme in bacterial DNA synthesis. The deoxythymidine monophosphate (dTMP) substrate binding pocket was targeted in a rational-design, structure-supported e..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="October 24, 2012" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm3011806" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2012 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm3011806" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm3011806" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm3011806" /></link>
        
    
    

<title>Discovery of Selective and Potent Inhibitors of Gram-Positive Bacterial Thymidylate Kinase (TMK) | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm3011806" /></meta><meta property="og:title" content="Discovery of Selective and Potent Inhibitors of Gram-Positive Bacterial Thymidylate Kinase (TMK)" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0018.jpeg" /></meta><meta property="og:description" content="Thymidylate kinase (TMK) is an essential enzyme in bacterial DNA synthesis. The deoxythymidine monophosphate (dTMP) substrate binding pocket was targeted in a rational-design, structure-supported effort, yielding a unique series of antibacterial agents showing a novel, induced-fit binding mode. Lead optimization, aided by X-ray crystallography, led to picomolar inhibitors of both Streptococcus pneumoniae and Staphylococcus aureus TMK. MICs &lt; 1 μg/mL were achieved against methicillin-resistant S. aureus (MRSA), S. pneumoniae, and vancomycin-resistant Enterococcus (VRE). Log D adjustments yielded single diastereomers 14 (TK-666) and 46, showing a broad antibacterial spectrum against Gram-positive bacteria and excellent selectivity against the human thymidylate kinase ortholog." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm3011806"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm3011806">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm3011806&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm3011806&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm3011806&amp;href=/doi/10.1021/jm3011806" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2012</span><span class="cit-fg-volume">, 55</span><span class="cit-fg-issue">, 22</span><span class="cit-fg-pageRange">, 10010-10021</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/55/22" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm301177j" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm301181r" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Selective and Potent Inhibitors of Gram-Positive Bacterial Thymidylate Kinase (TMK)</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Gabriel++Mart%C3%ADnez-Botella">Gabriel Martínez-Botella</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=John+N.++Breen">John N. Breen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=James+E.+S.++Duffy">James E. S. Duffy</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jacques++Dumas">Jacques Dumas</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Bolin++Geng">Bolin Geng</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ian+K.++Gowers">Ian K. Gowers</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Oluyinka+M.++Green">Oluyinka M. Green</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Satenig++Guler">Satenig Guler</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Martin+F.++Hentemann">Martin F. Hentemann</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Felix+A.++Hernandez-Juan">Felix A. Hernandez-Juan</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Diane++Joseph-McCarthy">Diane Joseph-McCarthy</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sameer++Kawatkar">Sameer Kawatkar</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Nicholas+A.++Larsen">Nicholas A. Larsen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ovadia++Lazari">Ovadia Lazari</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=James+T.++Loch">James T. Loch</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jacqueline+A.++Macritchie">Jacqueline A. Macritchie</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Andrew+R.++McKenzie">Andrew R. McKenzie</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Joseph+V.++Newman">Joseph V. Newman</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Nelson+B.++Olivier">Nelson B. Olivier</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Linda+G.++Otterson">Linda G. Otterson</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Andrew+P.++Owens">Andrew P. Owens</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jon++Read">Jon Read</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=David+W.++Sheppard">David W. Sheppard</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Thomas+A.++Keating">Thomas A. Keating</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">AstraZeneca Infection Innovative Medicines, 35 Gatehouse Drive, Waltham, Massachusetts 02451, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">AstraZeneca Discovery Sciences, 35 Gatehouse Drive, Waltham, Massachusetts 02451, United States</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">BioFocus, Chesterford Research Park, Saffron Walden CB10 1XL, U.K.</span></div><div class="corresp-info"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#dcbbbdbeaeb5b9b09cafbdbbb9aea4f2bfb3b1"><span class="__cf_email__" data-cfemail="f6919794849f939ab685979193848ed895999b">[email protected]</span></a>. Phone: +1.617.299.8375.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm3011806&amp;href=/doi/10.1021%2Fjm3011806" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2012</span></span><span class="cit-volume">, 55</span><span class="cit-issue">, 22</span><span class="cit-pageRange">, 10010–10021</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">October 8, 2012</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>10 August 2012</li><li><span class="item_label"><b>Published</b> online</span>24 October 2012</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 26 November 2012</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm3011806" title="DOI URL">https://doi.org/10.1021/jm3011806</a></div><div class="article_header-article-copyright"><strong>Copyright © 2012 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D10010%26pageCount%3D12%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DGabriel%2BMart%25C3%25ADnez-Botella%252C%2BJohn%2BN.%2BBreen%252C%2BJames%2BE.%2BS.%2BDuffy%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D55%26issueNum%3D22%26contentID%3Djm3011806%26title%3DDiscovery%2Bof%2BSelective%2Band%2BPotent%2BInhibitors%2Bof%2BGram-Positive%2BBacterial%2BThymidylate%2BKinase%2B%2528TMK%2529%26numPages%3D12%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D10021%26publicationDate%3DNovember%2B2012">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm3011806"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1940</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">27</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm3011806" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Selective and Potent Inhibitors of Gram-Positive Bacterial Thymidylate Kinase (TMK)&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Gabriel&quot;,&quot;last_name&quot;:&quot;Martínez-Botella&quot;},{&quot;first_name&quot;:&quot;John&quot;,&quot;last_name&quot;:&quot;N. Breen&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;E. S. Duffy&quot;},{&quot;first_name&quot;:&quot;Jacques&quot;,&quot;last_name&quot;:&quot;Dumas&quot;},{&quot;first_name&quot;:&quot;Bolin&quot;,&quot;last_name&quot;:&quot;Geng&quot;},{&quot;first_name&quot;:&quot;Ian&quot;,&quot;last_name&quot;:&quot;K. Gowers&quot;},{&quot;first_name&quot;:&quot;Oluyinka&quot;,&quot;last_name&quot;:&quot;M. Green&quot;},{&quot;first_name&quot;:&quot;Satenig&quot;,&quot;last_name&quot;:&quot;Guler&quot;},{&quot;first_name&quot;:&quot;Martin&quot;,&quot;last_name&quot;:&quot;F. Hentemann&quot;},{&quot;first_name&quot;:&quot;Felix&quot;,&quot;last_name&quot;:&quot;A. Hernandez-Juan&quot;},{&quot;first_name&quot;:&quot;Diane&quot;,&quot;last_name&quot;:&quot;Joseph-McCarthy&quot;},{&quot;first_name&quot;:&quot;Sameer&quot;,&quot;last_name&quot;:&quot;Kawatkar&quot;},{&quot;first_name&quot;:&quot;Nicholas&quot;,&quot;last_name&quot;:&quot;A. Larsen&quot;},{&quot;first_name&quot;:&quot;Ovadia&quot;,&quot;last_name&quot;:&quot;Lazari&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;T. Loch&quot;},{&quot;first_name&quot;:&quot;Jacqueline&quot;,&quot;last_name&quot;:&quot;A. Macritchie&quot;},{&quot;first_name&quot;:&quot;Andrew&quot;,&quot;last_name&quot;:&quot;R. McKenzie&quot;},{&quot;first_name&quot;:&quot;Joseph&quot;,&quot;last_name&quot;:&quot;V. Newman&quot;},{&quot;first_name&quot;:&quot;Nelson&quot;,&quot;last_name&quot;:&quot;B. Olivier&quot;},{&quot;first_name&quot;:&quot;Linda&quot;,&quot;last_name&quot;:&quot;G. Otterson&quot;},{&quot;first_name&quot;:&quot;Andrew&quot;,&quot;last_name&quot;:&quot;P. Owens&quot;},{&quot;first_name&quot;:&quot;Jon&quot;,&quot;last_name&quot;:&quot;Read&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;W. Sheppard&quot;},{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;A. Keating&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2012&quot;,&quot;month&quot;:&quot;10&quot;,&quot;day&quot;:&quot;24&quot;,&quot;issue&quot;:&quot;22&quot;,&quot;volume&quot;:&quot;55&quot;,&quot;pages&quot;:&quot;10010-10021&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm3011806&quot;},&quot;abstract&quot;:&quot;Thymidylate kinase (TMK) is an essential enzyme in bacterial DNA synthesis. The deoxythymidine monophosphate (dTMP) substrate binding pocket was targeted in a rational-design, structure-supported effort, yielding a unique series of antibacterial agents showing a novel, induced-fit binding mode. Lead optimization, aided by X-ray crystallography, led to picomolar inhibitors of both Streptococcus pneumoniae and Staphylococcus aureus TMK. MICs &lt; 1 μg/mL were achieved against methicillin-resistant S. aureus (MRSA), S. pneumoniae, and vancomycin-resistant Enterococcus (VRE). Log D adjustments yielded single diastereomers 14 (TK-666) and 46, showing a broad antibacterial spectrum against Gram-positive bacteria and excellent selectivity against the human thymidylate kinase ortholog.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm3011806&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm3011806" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm3011806&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm3011806" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm3011806&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm3011806" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm3011806&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm3011806&amp;href=/doi/10.1021/jm3011806" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm3011806" /></input><a href="/doi/pdf/10.1021/jm3011806" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm3011806&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm3011806%26sid%3Dliteratum%253Aachs%26pmid%3D23043329%26genre%3Darticle%26aulast%3DMart%25C3%25ADnez-Botella%26date%3D2012%26atitle%3DDiscovery%2Bof%2BSelective%2Band%2BPotent%2BInhibitors%2Bof%2BGram-Positive%2BBacterial%2BThymidylate%2BKinase%2B%2528TMK%2529%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D55%26issue%3D22%26spage%3D10010%26epage%3D10021%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292288" title="X-rays">X-rays</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291880" title="Crystal structure">Crystal structure</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292444" title="Molecular structure">Molecular structure</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/55/22" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jmcmar.2012.55.issue-22/production/jmcmar.2012.55.issue-22.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/medium/jm-2012-011806_0018.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011806&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Thymidylate kinase (TMK) is an essential enzyme in bacterial DNA synthesis. The deoxythymidine monophosphate (dTMP) substrate binding pocket was targeted in a rational-design, structure-supported effort, yielding a unique series of antibacterial agents showing a novel, induced-fit binding mode. Lead optimization, aided by X-ray crystallography, led to picomolar inhibitors of both <i>Streptococcus pneumoniae</i> and <i>Staphylococcus aureus</i> TMK. MICs < 1 μg/mL were achieved against methicillin-resistant <i>S. aureus</i> (MRSA), <i>S. pneumoniae</i>, and vancomycin-resistant <i>Enterococcus</i> (VRE). Log <i>D</i> adjustments yielded single diastereomers <b>14</b> (TK-666) and <b>46</b>, showing a broad antibacterial spectrum against Gram-positive bacteria and excellent selectivity against the human thymidylate kinase ortholog.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec articleNote"><h4><span>∥</span> Author Present Address</h4><p class="last">Sage Therapeutics, 215 First Street, Cambridge, MA 02141, United States.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35688" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35688" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">The discovery of novel antibacterial classes has proven particularly challenging for the research community in both industry and academia in the last few decades.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The onset of the genomic era and the promise of the discovery of novel mechanisms have not translated into marketed drugs.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> The need for new antibiotics to treat drug-resistant Gram-positive infections is acute, especially against methicillin-resistant <i>Staphylococcus aureus</i> and <i>Staphylococcus epidermidis</i> and vancomycin-resistant <i>Enterococcus faecium</i> and <i>Enterococcus faecalis</i>.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> In this context, structure-guided design approaches<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> have yielded some initial success in addressing the resistance of existing classes, for example the drug candidate Iclaprim,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> and novel chemical classes of DNA gyrase inhibitors.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> We have recently reported the first <i>in vivo</i> efficacious inhibitor of thymidylate kinase (TMK) targeting Gram-positive bacteria.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> TMK is a nucleotide kinase that catalyzes the phosphorylation of deoxythymidine monophosphate (dTMP) to deoxythymidine diphosphate (dTDP) using ATP as a cosubstrate. This is a necessary step in the biosynthesis of deoxythymidine triphosphate (dTTP) for DNA synthesis. While dTMP can be produced both <i>de novo</i> (from dUMP) and <i>via</i> salvage (from thymidine), neither it nor dTDP can be imported into the cell nor biosynthesized through an alternate path. This makes TMK an essential enzyme and a very attractive target for therapeutic intervention.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> This report describes the discovery of the first high-quality inhibitors of Gram-positive bacterial TMK employing a structure-guided approach.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04731" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04731" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Piperidinylthymine analogs were synthesized as described in Schemes <a class="ref internalNav" href="#sch1" aria-label="1">1</a>, <a class="ref internalNav" href="#sch2" aria-label="2">2</a>, and <a class="ref internalNav" href="#sch3" aria-label="3">3</a>. The key intermediate, piperidinylthymine <b>1</b>, was prepared using commercially available racemic- or (<i>R</i>)- or (<i>S</i>)-<i>tert</i>-butyl 3-aminopiperidine-1-carboxylate (<b>3</b>) and 3-methoxyacrylic acid (<b>2</b>) (Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a>). Standard reductive amination conditions to couple <b>1</b> with benzaldehydes afforded the analogs described in Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>. Scheme <a class="ref internalNav" href="#sch2" aria-label="2">2</a> shows the synthesis for compounds in Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>. 2-Fluoro-4-formylbenzonitrile (<b>5</b>) and <b>1</b> afforded intermediate <b>6</b> under reductive amination conditions using polymer-supported cyanoborohydride. The addition of phenols, followed by the hydrolysis of the cyano group in <b>7</b> gave the desired compounds in moderate yields. In order to install an aliphatic side chain in the benzylic carbon bridging the piperidine and the central phenyl ring, addition of commercially available zincate <b>9</b> to acid chlorides was performed routinely and in high yields (Scheme <a class="ref internalNav" href="#sch3" aria-label="3">3</a>). The resulting ketone <b>11a</b> was transformed into mesylate <b>11b</b> and then coupled to <b>1</b> in moderate yields, though, in some instances, this reaction led to elimination products. The ketone <b>11a</b> and protected 3-aminopiperidine could alternately be coupled under standard reductive amination conditions and the thymine ring built as described in Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a>. Installation of the phenoxy ring was carried out under microwave conditions, followed by subsequent hydrolysis of the benzonitrile under aqueous basic conditions (Scheme <a class="ref internalNav" href="#sch3" aria-label="3">3</a>). The diastereomeric mixture <b>13</b> was then separated, affording pure diastereomers (Scheme <a class="ref internalNav" href="#sch3" aria-label="3">3</a>).</div><figure id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/medium/jm-2012-011806_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of TMK Inhibitors in Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011806&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) (i) oxalyl chloride, AgOCN, 95 °C, 3 h; (ii) H<sub>2</sub>SO<sub>4</sub>, MeOH, 95 °C, 16 h, 40% overall; (b) 3-phenoxybenzaldehyde, trimethylammonium cyanoborohydride, 22 °C, 16 h, 19%.</p></p></figure><figure id="sch2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/medium/jm-2012-011806_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of TMK Inhibitors in Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011806&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) DMF, AcOH, polymer-BCN(H)<sub>3</sub>, 22 °C, 16 h, 28%; (b) 3-chlorophenol, K<sub>2</sub>CO<sub>3</sub>, NMP, 150 °C, 15 min, 45%; (c) H<sub>2</sub>SO<sub>4</sub>, 150 °C, 30 min, 9%.</p></p></figure><figure id="sch3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/medium/jm-2012-011806_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of TMK Inhibitors in Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011806&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Pd(PPh<sub>3</sub>)<sub>4</sub>, THF; 22 °C, 16 h, 85%; (b) (i) NaBH<sub>4</sub>, MeOH, 22 °C, 2 h, 29%; (ii) MsCl, TEA, THF, 0 °C, 4 h; (c) (S)-<b>1</b>, <b>11b</b>, DIEA, CH<sub>3</sub>CN, 70 °C, 20 h, 13%; (d) (i) 3-bromophenol, K<sub>2</sub>CO<sub>3</sub>, NMP, 150 °C, 30 min, 67%; (ii) NaOH, H<sub>2</sub>O, EtOH, 100 °C, 16 h, 77%; (e) purification/separation by chiral chromatography, 15%.</p></p></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. TMK Inhibition Activity of Analogues of <b>15</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/medium/jm-2012-011806_0011.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/medium/jm-2012-011806_0012.gif" alt="" id="fx2" /></img><div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. TMK Inhibition Activity of Analogues of <b>4</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/medium/jm-2012-011806_0013.gif" alt="" id="fx3" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/medium/jm-2012-011806_0014.gif" alt="" id="fx4" /></img><div></div></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Activity of Analogues of <b>34</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/medium/jm-2012-011806_0015.gif" alt="" id="fx5" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">MIC (μg/mL)<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center" char=".">Spn</th><th class="colsep0 rowsep0" align="center" char=".">Sau</th><th class="colsep0 rowsep0" align="center" char=".">log <i>D</i></th><th class="colsep0 rowsep0" align="center" char=".">Spn</th><th class="colsep0 rowsep0" align="center">SauMSQS/SauMRQR</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="char" char=".">1.9</td><td class="colsep0 rowsep0" align="char" char=".">73</td><td class="colsep0 rowsep0" align="char" char=".">–0.55</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="left">32/32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="left">Br</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">27</td><td class="colsep0 rowsep0" align="char" char=".">–0.51</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="left">16/32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">114</td><td class="colsep0 rowsep0" align="char" char=".">–0.82</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">64/128</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33</b></td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="char" char=".">28</td><td class="colsep0 rowsep0" align="char" char=".">339</td><td class="colsep0 rowsep0" align="char" char=".">–1</td><td class="colsep0 rowsep0" align="char" char=".">32</td><td class="colsep0 rowsep0" align="left">128/128</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34</b></td><td class="colsep0 rowsep0" align="left">CF<sub>3</sub></td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">120</td><td class="colsep0 rowsep0" align="char" char=".">–0.38</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="left">64/64</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>a</sup><p class="last">Bacterial strains from the AstraZeneca collection. MSQS: methicillin- and quinolone-sensitive. MRQR: methicillin- and quinolone-resistant.</p></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41377" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41377" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The X-ray structure of dTMP bound to <i>S. aureus</i> TMK<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> provided the starting point for the rational design of thymine-containing scaffolds (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>). A set of deoxyribose-replacement ring systems were designed from docking models, and small libraries of each scaffold were synthesized by coupling available R groups. These compounds were screened for inhibition against the <i>S. aureus</i> and <i>S. pneumoniae</i> TMK enzymes. While nearly all of the compounds synthesized were inactive, as a result of this effort, thymine-based racemic compound <b>15</b> was discovered (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>). Compound <b>15</b> has moderate affinity against <i>S. pneumoniae</i> TMK (IC<sub>50</sub> = 3.4 μM) and low affinity against <i>S. aureus</i> TMK (IC<sub>50</sub> = 156 μM), but given the dearth of other leads, it was pursued in a medicinal chemistry program. An early survey of the phenoxy ring (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>) led to racemic compound <b>16</b>, which was suitable for protein crystallography, yielding the first structure of an inhibitor bound to <i>S. aureus</i> and a distinct binding mode when compared to dTMP<a onclick="showRef(event, 'ref7 ref9'); return false;" href="javascript:void(0);" class="ref ref7 ref9">(7, 9)</a> (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>). The X-ray structure showed the thymine ring (ring A) retaining its position from dTMP, and anchoring the inhibitor by creating multiple hydrogen bonds with residues Arg70, Ser97, and Gln101, and π-stacking with Phe66. Strikingly, a single enantiomer (<i>S</i>)-<b>16</b> was found in the crystal structure, which positions the compound to bend <i>away</i> from the substrate binding channel rather than following it as the dTMP substrate does. The piperidine (ring B) in this enantiomer provided the correct geometry to enable the turn. The middle phenyl ring (ring C) appeared to act as a spacer, placing the phenoxy ring (ring D) into a newly revealed hydrophobic pocket, created by movement of a helix and a loop at the left-hand side of the binding pocket.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> This pocket is not evident in the apo or dTMP structures.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> To verify the insight gained from the crystal structure, several enantiomeric pairs (<b>27</b>–<b>30</b>) were synthesized (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>), starting with enantiomerically pure <i>tert</i>-butyl 3-aminopiperidine-1-carboxylate (<b>2</b>) (Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a>). In all cases, the (<i>S</i>)-enantiomer was 12- to 100-fold more potent against both <i>S. pneumoniae</i> and <i>S. aureus</i> TMK enzymes than the (<i>R</i>)-enantiomer.</div><figure id="fig1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/medium/jm-2012-011806_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. The crystal structure of the <i>Staphylococcus aureus</i> thymidylate kinase (TMK) with bound dTMP (left)<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> was the starting point for design of lead inhibitor scaffolds (right). dTMP is depicted in orange, with the terminal phosphate located at the right-hand side of the panel. This phosphate points in the direction of the substrate channel leading to the ATP binding site (not shown).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011806&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/medium/jm-2012-011806_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structure of thymidylate kinase (TMK) racemic lead inhibitor <b>15</b>. The rings are lettered A–D for discussion in the text.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011806&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/medium/jm-2012-011806_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Crystal structure of (<i>S</i>)-<b>16</b> bound to <i>S. aureus</i> TMK at 2.0 Å. The green mesh represents the 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density map contoured to 1 σ. Racemic <b>16</b> was soaked with protein crystals, but only the single enantiomer was found to be bound. Compared to the binding mode of dTMP (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>), the position and contacts of thymine ring A are extremely well-conserved. However, piperidinyl ring B, instead of executing a turn toward the substrate channel (off right-hand side of panel) instead bends the molecule to the left, into a newly formed, induced-fit binding pocket for ring D. This pocket is not evident in the Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a> structure. Major changes in protein structure to create this cryptic pocket include rotation of the Arg48 side chain, movement of the α2 helix (leftmost in figure), and movement of the connecting loop at the top left of the figure. Phe66 forms a π-stack with rings A and D.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011806&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">With the X-ray structure of (<i>S</i>)-<b>16</b> in hand, binding was improved by first modifying ring D. Two approaches were employed: first, a virtual library of compounds with substituted phenoxy rings was docked into the binding pocket of <i>S. aureus</i> TMK.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Results suggested that a great variety of substitutions could be tolerated in this pocket. Based on this, a set of compounds were synthesized and tested. However, little improvement was observed in the activity compared to the case of <b>16</b>. Second, a more traditional Topliss method was also employed in parallel to the above-mentioned structure-based approach. The Topliss method<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> is based on the assumption that any substitution will modify the biological activity relative to the parent compound as a result of changes in hydrophobic, electronic, and steric effects. This approach resulted in compounds with significant improvements in enzyme inhibition. In particular, the 3-Cl and 3-Br analogs (<b>22</b>, <b>23</b>) achieved more than a 100-fold improvement against both <i>S. pneumoniae</i> TMK and <i>S. aureus</i> TMK. Such an improvement most likely resulted from a combination of desolvation and favorable hydrophobic interactions. These inhibitors do, however, show a higher affinity, ∼10-fold, against <i>S. pneumoniae</i> TMK versus <i>S. aureus</i> TMK. The difference in compound potency between these two species may be attributed to the nature of the hydrogen bond network formed between protein and compound. The scaffold presented herein interacts with the binding site such that rings A and D form a π-stacking interaction with an aromatic residue: Tyr71 and Phe66 in <i>S. pneumoniae</i> and <i>S. aureus</i> TMK, respectively.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> We hypothesize that the hydroxyl group of <i>S. pneumoniae</i> Tyr71 forms a hydrogen bond to the thymine carbonyl group. As the hydrogen bonding energy between two uncharged groups is estimated to be 0.5–1.8 kcal/mol,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> we ascribe the roughly 10-fold difference in IC<sub>50</sub> between the orthologs to the additional interaction.</div><div class="NLM_p">In our search for conserved residues across Gram-positive bacteria, Arg48 was identified as an attractive side chain to target. By the use of X-ray cocrystal structures and models, we predicted that placing an acid group at the C-4 of ring C would be optimal (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>A). When synthesized and assayed, these acid derivatives showed more than a 20-fold improvement in IC<sub>50</sub> (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>). The X-ray cocrystal structure of compound <b>34</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>B) showed the formation of two clear hydrogen bonds between the carboxylate of <b>34</b> and the guanidinium of Arg48. As shown in Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>, these compounds showed excellent IC<sub>50</sub> and MIC values against <i>S. pneumoniae</i>. The enzymatic IC<sub>50</sub> and MIC values for <i>S. aureus</i> were improved, but they remained more than 10-fold weaker than those of <i>S. pneumoniae</i>.</div><figure id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/medium/jm-2012-011806_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Structural characterization of the binding of <b>34</b> in S. <i>aureus</i> TMK. The addition of a carboxylate group to ring C was designed to specifically interact with the side chain of conserved Arg48. This carboxylate drove significant improvements in potency and physical properties. (A) Computational model of the carboxylate-Arg48 interaction predicted a single hydrogen bond. (B) Crystal structure, resolved to 1.7 Å. The carboxylic acid interaction with Arg48 is revealed to be bidentate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011806&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In order to improve the binding affinity against both <i>S. pneumoniae</i> and <i>S. aureus</i> TMK, the next area targeted was the methylene linker between ring B and ring C. Based on the X-ray structures and models, we hypothesized that adding a substituent to this carbon should have a positive effect by stabilizing the U-shaped binding conformation. The addition of a small group, such as a methyl (<b>35</b>), led to an improvement in the affinity for <i>S. aureus</i> TMK of more than 10-fold (Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>). Although the enzyme affinities for both <i>S. pneumoniae</i> and <i>S. aureus</i> TMK were now comparable, the MICs for <i>S. aureus</i> were relatively weaker than those for <i>S. pneumoniae</i>. At this point, we established the link between enzyme inhibition and observed MICs through resistant mutant generation, radioactive-precursor incorporation assays, and <i>tmk</i> overexpression strains.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> The compounds were found to exert their growth-inhibitory effect cleanly in both <i>S. aureus</i> and <i>S. pneumoniae</i> through TMK inhibition.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Activity of Single Diastereomer Analogues of <b>41</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/medium/jm-2012-011806_0016.gif" alt="" id="fx6" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/medium/jm-2012-011806_0017.gif" alt="" id="fx7" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>Table a</sup><p class="last">Bacterial strains from the AstraZeneca collection. MSQS: methicillin- and quinolone-sensitive. MRQR: methicillin- and quinolone-resistant.</p></div></div><div></div></div><div class="NLM_p">In an effort to improve whole-cell activity, we examined the physical properties of the series. <i>S. aureus</i> membrane permeability appears to be highly dependent on the physical properties of the molecules. In our experience, there is a good correlation between log <i>D</i> and the <i>S. aureus</i> MICs, with optimal whole-cell activity observed for log <i>D</i> values between 1 and 2.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Profiling of compounds <b>35</b>–<b>40</b> (Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>) showed log <i>D</i> values between −0.3 and 0.5. In order to increase the log <i>D</i>, a series of compounds with substituents of increasing size and lipophilicity at the carbon linker between rings B and C were designed using calculated log <i>D</i> values and then synthesized (Scheme <a class="ref internalNav" href="#sch3" aria-label="3">3</a>). As the size of the substituent increased, the (<i>R</i>) configuration of the new stereocenter became increasingly favored (Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>, <b>39</b>–<b>43</b>). The X-ray structure of compound <b>41</b> showed that the binding mode was unchanged compared to our first X-ray cocrystal structure with all major interactions maintained (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>). The source of affinity difference between the (<i>R</i>,<i>S</i>) and (<i>S</i>,<i>S</i>) diastereomers may be explained by the ability of the linker side chain in the (<i>R</i>) configuration to establish a hydrophobic interaction with Val51 and Leu52. Under the crystallization conditions, the enzyme also selectively bound the (<i>R</i>,<i>S</i>)-diastereomer when the diastereomeric mixture of <b>41/42</b> was used.</div><figure id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/medium/jm-2012-011806_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Co-crystal structure of a single diastereomer (<i>R</i>,<i>S</i>)-<b>41</b> and <i>S. aureus</i> TMK at 2.0 Å, depicting optimized inhibitor binding to the enzyme (IC<sub>50</sub> = 1.2 nM). Key features are retention of thymine binding interactions (ring A), hydrophobic packing of the isobutyl group against conserved hydrophobic residues Val51/Leu52 (linker between rings B and C), the bidentate salt bridge of the ring C carboxylate to Arg48, and the cryptic hydrophobic pocket hosting the chlorophenyl ring D. The π-stack of rings A and D with Phe66 is maintained.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011806&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As shown in Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>, the MICs against <i>S. aureus</i> improved as the log <i>D</i> increased from 0 to 2. For example, compound <b>47</b> exhibited excellent MICs against <i>S. aureus</i> (0.1–0.03 μg/mL) and a log <i>D</i> value of 2. On the other hand, compound <b>51</b> exhibited only a modest improvement in <i>S. aureus</i> MIC (0.5 μg/mL) in spite of the highest log <i>D</i> (2.5), indicating a limit to lipophilicity-induced gains in whole-cell activity for this series (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>).</div><figure id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/medium/jm-2012-011806_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. pMIC vs log <i>D</i> relationship for piperidinylthymine analogs in Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>. <i>S. aureus</i> appears to be more sensitive to log <i>D</i> variation than <i>S. pneumoniae</i>, but both reach an MIC optimum with log D between 1.5 and 1.8.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011806&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Compounds <b>14</b> (TK-666)<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> and <b>46</b> in Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a> were profiled against pathogens <i>S. aureus</i>, <i>S. pneumoniae</i>, <i>Streptococcus pyogenes</i>, <i>Staphylococcus epidermidis</i>, and <i>Enterococcus</i> spp., showing excellent activity and demonstrating the value of a novel antibacterial target and mechanism for addressing serious exisiting clinical resistance mechanisms. Finally, for any novel antibacterial target with an essential eukaryotic ortholog, it is critical that selectivity be robust to avoid potential toxicity. Several compounds were thus tested in the human TMK IC<sub>50</sub> assay and showed exquisite (10<sup>5</sup>–10<sup>6</sup>) selectivity against the human homologue as well as very low potential for cytotoxicity in both human (A459) and yeast whole-cell assays (Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a>). Compound <b>14</b> was further dosed in a murine <i>S. aureus</i>-infected thigh model and demonstrated both efficacy (infection stasis at 150 mg/kg single dose, versus 40 mg/kg for comparator levofloxacin) and tolerability (no observed adverse effects up to and including the highest dose of 800 mg/kg/day).<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a></div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Gram-Positive Antibacterial Spectrum of Analogues <b>14</b> and <b>46</b></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">MIC (μg/mL)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">bacterial strain<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char="."><b>(</b><i><b>R,S</b></i><b>)</b><i><b>-</b></i><b>14</b></th><th class="colsep0 rowsep0" align="center" char="."><b>(</b><i><b>R,S</b></i><b>)</b><i><b>-</b></i><b>46</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>S. pneumoniae</i></td><td class="colsep0 rowsep0" align="char" char=".">0.02</td><td class="colsep0 rowsep0" align="char" char=".">0.01</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>Streptococcus pyogenes</i></td><td class="colsep0 rowsep0" align="char" char=".">0.08</td><td class="colsep0 rowsep0" align="char" char=".">0.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>S. aureus</i> MSQS<i>/S. aureus</i> MRQR</td><td class="colsep0 rowsep0" align="char" char=".">0.25/0.25</td><td class="colsep0 rowsep0" align="char" char=".">0.25/0.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>Staphylococcus epidermidis</i></td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>Enterococcus faecium</i> (LRE)</td><td class="colsep0 rowsep0" align="char" char=".">0.25</td><td class="colsep0 rowsep0" align="char" char=".">0.25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>Enterococcus faecalis</i> (VRE)</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">0.06</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup>a</sup><p class="last">Bacterial strains from AstraZeneca collection. MSQS: methicillin- and quinolone-sensitive; MRQR: methicillin- and quinolone-resistant; LRE: linezolid-resistant <i>Enterococcus</i>; VRE: vancomycin-resistant <i>Enterococcus</i>.</p></div></div></div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Eukaryotic Selectivity for Selected Analogues</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" align="center" char=".">IC<sub>50</sub> (nM)</th><th class="rowsep1 colsep0" colspan="2" align="center" char=".">MIC (μg/mL)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char=".">Human TMK</th><th class="colsep0 rowsep0" align="center" char=".">Human A549<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char="."><i>Candida albicans</i></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34</b></td><td class="colsep0 rowsep0" align="char" char=".">>200000</td><td class="colsep0 rowsep0" align="char" char=".">>64</td><td class="colsep0 rowsep0" align="char" char=".">>64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>(</b><i><b>R,S</b></i><b>)</b><i><b>-</b></i><b>35</b></td><td class="colsep0 rowsep0" align="char" char=".">150000</td><td class="colsep0 rowsep0" align="char" char=".">>64</td><td class="colsep0 rowsep0" align="char" char=".">>64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>(</b><i><b>R,S</b></i><b>)</b><i><b>-</b></i><b>39</b></td><td class="colsep0 rowsep0" align="char" char=".">150000</td><td class="colsep0 rowsep0" align="char" char=".">>64</td><td class="colsep0 rowsep0" align="char" char=".">>64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>(</b><i><b>R,S</b></i><b>)</b><i><b>-</b></i><b>14</b></td><td class="colsep0 rowsep0" align="char" char=".">62000</td><td class="colsep0 rowsep0" align="char" char=".">>64</td><td class="colsep0 rowsep0" align="char" char=".">>64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>(</b><i><b>R,S</b></i><b>)</b><i><b>-</b></i><b>44</b></td><td class="colsep0 rowsep0" align="char" char=".">43400</td><td class="colsep0 rowsep0" align="char" char=".">>64</td><td class="colsep0 rowsep0" align="char" char=".">>64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>(</b><i><b>R,S</b></i><b>)</b><i><b>-</b></i><b>46</b></td><td class="colsep0 rowsep0" align="char" char=".">32000</td><td class="colsep0 rowsep0" align="char" char=".">>64</td><td class="colsep0 rowsep0" align="char" char=".">>64</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup>a</sup><p class="last">Human A549: Human lung epithelial adenocarcinoma cell line.</p></div></div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25030" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25030" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In this report we have illustrated the use of X-ray crystallography, computational methods, and traditional medicinal chemistry approaches to design the first potent, efficacious inhibitors of Gram-positive bacterial thymidylate kinase. After a small number of design cycles, the weak lead inhibitor <b>16</b> was rapidly optimized into the highly selective picomolar inhibitor <b>14</b>. While excellent cellular potency was easily obtained against <i>S. pneumoniae</i>, improving cellular activity against <i>S. aureus</i> proved to be more challenging due to the sensitivity of this species to the physical properties of the inhibitor. By utilizing the bridge carbon between rings B and C, log <i>D</i> adjustments were made which allowed us to improve some enzymatic potency but, more importantly, led to on-target, potent cellular activity against <i>S. aureus</i>. Compounds <b>14</b> and <b>46</b> showed excellent antibacterial spectrum against Gram-positive bacteria and 10<sup>5</sup>–10<sup>6</sup> selectivity versus the human thymidylate kinase homologue. The <i>in vivo</i> target validation provided by compound <b>14</b><a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> against <i>S. aureus</i> provides great impetus for the development of clinical candidates both in the series and for the TMK target.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52541" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52541" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> General Experimental Details</h3><div class="NLM_p">All commercial reagents and anhydrous solvents were obtained from commercial sources and were used without further purification, unless otherwise specified.</div><div class="NLM_p">LC-MS conditions: <i>Method 1:</i> Samples were analyzed by reversed phase LC-MS using Waters Xterra C18 MS 100 mm × 4.6 mm, 5 μm particle size columns; linear gradient from 5% to 95% acetonitrile in water (10 mM ammonium hydrogen carbonate) over 5.5 min; flow rate 2 mL/min; injection volume was 2–7 μL; UV detection via HP or Waters DAD (210–400 nm range). Detection was based on ESCI in positive and negative polarity using a Micromass ZQ single quadrapole LC-MS or Quattro Micro LC-MS-MS, diode-array UV detector from 210 to 400 nm via HP or Waters DAD. <i>Method 2:</i> Samples were analyzed by reversed phase LC-MS using Varian Polaris C18A, 2 mm × 50 mm, 3 μm particle size columns. An Agilent HP1100 (Wimington, DE, USA) LC system was used with a gradient elusion profile of 5–95% B over 4.5 min at 1 mL/min, then re-equilibration at initial conditions to 6 min. Injection volume was 2 μL and column temperature 30 °C. Mobile phase A was 0.1% formic acid in water and mobile phase B 0.1% formic acid in acetonitrile. Detection was based on electrospray ionization (ESI) in positive and negative polarity using Waters ZQ mass spectrometer (Milford, MA, USA), diode-array UV detector from 210 to 400 nm, and evaporative light-scattering detector (Sedex 75, Sedere, Alfortville Cedex, France). <i>Method 3:</i> As Method 2, except A was 10 mM ammonium acetate in (5/95 acetonitrile/water) and B was acetonitrile.</div><div class="NLM_p">Accurate mass was done using a hybrid quadrupole time-of-flight mass spectrometer (microTOFQ, Bruker Daltonics) in ESI+ mode. Method: 5–95% mobile phase B from 0.0 to 5.0 min, hold at 95% mobile phase B to 5.1 min. Mobile phase A: 0.1% formic acid in water, B: 0.1% formic acid in acetonitrile. 1.0 mL/min flow, column Varian Polaris C18-A 2.0 mm × 50 mm, 3.0 μm particle size. One μL of sample was injected onto an HPLC using an externally calibrated instrument for accurate mass measurement. 50 μL/min eluent from LC and 50 μL/min make up solution (CH<sub>3</sub>CN/H<sub>2</sub>O 50/50) went into MS.</div><div class="NLM_p"><sup>1</sup>H NMR spectra (δ, ppm) were recorded using Bruker Advance Ultrashield 300 MHz or Bruker DPX 400 MHz instruments.</div><div class="NLM_p">Column chromatography was performed using Silcycle FLH-R10030B Silisep cartridges (12–330 g). Preparative reversed phase HPLC chromatography was carried out using a Waters Atlantis T3-C18 column, 19 mm × 100 mm, 5 μm, a linear gradient from 10% to 90% CH<sub>3</sub>CN in H<sub>2</sub>O over 12 min (0.1% trifluoroacetic acid), and a flow rate of 20 mL/min. Preparative chiral chromatography was carried out as follows. <i>HPLC:</i> Chiralpak IC column, 30 mm × 250 mm, 5 μm; hexane (60%), methanol/ethanol (1:1) (40%), 0.5% diethylamine; flow rate 40 mL/min. <i>SFC</i>: Chiralpak AD column, 30 mm × 250 mm, 5 μm; carbon dioxide (60%), isopropanol (40%); flow rate 120 mL/min. Chiral analytical HPLC: Chiralpak IC column, 4.6 mm × 250 mm, 5 μm; hexane (60%), methanol/ethanol (1:1) (40%), 0.5% diethylamine; flow rate 1 mL/min. Chiral analytical SFC: Chiralpak AD column, 4.6 mm × 250 mm, 5 μm; carbon dioxide (60%), isopropanol (40%); flow rate 2.8 mL/min.</div><div class="NLM_p last">The purity of the final compounds was assessed on the basis of analytical LC-MS, and the results were greater than 95% unless specified otherwise.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Molecular Modeling</h3><div class="NLM_p last">Molecular docking was performed using Glide version 5.0<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> in standard flexible docking mode. X-ray structures of Sau-TMK protein in complex with related ligands were used as a template for docking studies. The protein structures were prepared using protein preparation wizard, and the ligands were prepared using Ligprep utility in Maestro 8.5 (Schrodinger, LLC 2008, New York, NY). Docked poses were minimized in OPLS 2001<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> force field as implemented in Maestro 8.5.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Experimental Procedures and Characterization Data for Analogues in Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></h3><div id="sec5_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> 5-Methyl-1-(piperidin-3-yl)pyrimidine-2,4(1<i>H</i>,3<i>H</i>)-dione (<b>1</b>)<a onclick="showRef(event, 'cit15c'); return false;" href="javascript:void(0);" class="ref cit15c">(15c)</a></h4><div class="NLM_p last">Oxalyl chloride (24 mL) was added to a mixture of (<i>E</i>)-3-methoxy-2-methylacrylic acid<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> (<b>2</b>, 42 g, 0.25 mol, 1.0 equiv) in dichloromethane (250 mL) at 0 °C. The mixture was stirred at room temperature for 1 h. The solvent was removed under reduced pressure. The resultant residue was redissolved in toluene (750 mL), and silver cyanate (48.75 g, 0.325 mol, 1.3 equiv) was added. The reaction mixture was stirred at 95 °C for 3 h and allowed to cool to RT, and the inorganics were filtered. Filtrate was added slowly to a solution of (<i>S</i>)-<i>tert</i>-butyl 3-aminopiperidine-1-carboxylate (<b>3</b>, 50 g, 0.25 mol, 1.0 equiv) in dimethyl formamide (100 mL). The mixture was stirred at room temperature for 16 h. The solvent was removed under reduced pressure. The residue was dissolved in diethyl ether (750 mL) and washed with water (2 × 250 mL), 10% aqueous NaHCO<sub>3</sub> (1 × 250 mL), and brine (1 × 100 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and filtered, and the solvent was removed under reduced pressure to afford (<i>S</i>)-<i>tert</i>-butyl (3-3-(((2<i>E</i>)-3-methoxy-2-methylprop-2-enoyl)carbamoyl)amino)piperidine-1-carboxylate, as a pale yellow solid (53 g). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.40 (s, 9H), 1.40–165 (m, 3H), 1.65 (s, 3H), 1.83 (m, 1H), 3.24 (m, 2H), 3.43 (m, 2H), 3.70 (m, 1H), 3.83 (s, 3H), 7.50 (s, 1H), 8.81 (d, 1H), 9.78 (s, 1H) ppm. A 1.69 N aqueous solution of sulfuric acid (800 mL) was added to a mixture of (<i>S</i>)-(<i>E</i>)-<i>tert</i>-butyl 3-(3-(3-methoxy-2-methylacryloyl)ureido)piperidine-1-carboxylate (53 g, 0.155 mol) and 1,4-dioxane (300 mL). The mixture was stirred at 95 °C for 16 h. The reaction mixture was allowed to cool to room temperature and slowly neutralized with addition of solid sodium hydrogen carbonate (200 g). The reaction mixture was concentrated to dryness, and the residue was extracted with hot methanol. The methanol was concentrated, and the residue was purified by column chromatography to afford <b>1</b> as a pale yellow solid (20.7 g, 40% overall yield from 2 steps). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 1.84–1.96 (m, 6H), 2.54 (dt, <i>J =</i> 2.60, 12.66 Hz, 1H), 2.74 (t, <i>J =</i> 11.60 Hz, 1H), 2.80–3.10 (m, 3H), 4.43–4.48 (m, 1H), 7.53 (s, 1H). <sup>1</sup>H NMR (400 MHz, DMSO-d<sub>6</sub>): δ 1.60–1.68 (m, 1H), 1.76 (s, 3H), 1.78–1.85 (m, 3H), 2.54–2.60 (m, 1H), 2.85–2.94 (m, 3H), 2.98–3.06 (m,1H), 4.46–4.54 (m, 1H), 7.66 (s, 1H), 11.20 (br s, 1H) ppm.</div></div><div id="sec5_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> 5-Methyl-1-(1-(3-phenoxybenzyl)piperidin-3-yl)pyrimidine-2,4(1<i>H</i>,3<i>H</i>)-dione (<b>4</b>)</h4><div class="NLM_p last">A mixture of 5-methyl-1-(piperidin-3-yl)pyrimidine-2,4(1H,3H)-dione (<b>1</b>, 100 mg, 0.48 mmol), 3-(phenoxy)benzaldehyde (285 mg, 1.44 mmol), and (polystyrylmethyl)trimethylammonium cyanoborohydride (200 mg, 0.8 mmol) in dichloromethane/acetic acid (90:10 v/v, 4 mL) was stirred at room temperature for 16 h. The reaction mixture was concentrated under reduced pressure, and the crude product was purified by reverse phase preparative HPLC to afford the title compound as a white solid (18 mg, 19%). LC-MS (Method 1) <i>t</i><sub>R</sub> = 3.43 min, <i>m</i>/<i>z</i> = 392 (M + 1). HRMS calcd for C<sub>23</sub>H<sub>26</sub>N<sub>3</sub>O<sub>3</sub> 392.1969, found 392.1969. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.55 (m, 1H), 1.68–1.80 (s, 6H), 2.05 (t, 1H), 2.20 (t, 1H), 2.70 (m, 2H), 3.55 (m, 2H), 4.45 (m, 1H), 6.95 (d, 1H), 7.00–7.20 (m, 5H), 7.35–7.42 (m, 3H), 7.74 (m, 1H), 11.23 (s, 1H) ppm.</div></div><div id="sec5_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> 5-Methyl-1-(1-(3-phenoxybenzyl)piperidin-3-yl)pyrimidine-2,4(1<i>H</i>,3<i>H</i>)-dione (<b>15</b>)</h4><div class="NLM_p last">LC-MS (Method 1) <i>t</i><sub>R</sub> = 3.43 min, <i>m</i>/<i>z</i> = 392 (M + 1). HRMS calcd for C<sub>23</sub>H<sub>26</sub>N<sub>3</sub>O<sub>3</sub> 392.1969, found 392.1969. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.55 (m, 1H), 1.68–1.80 (s, 6H), 2.05 (t, 1H), 2.20 (t, 1H), 2.70 (m, 2H), 3.55 (m, 2H), 4.45 (m, 1H), 6.95 (d, 1H), 7.00–7.20 (m, 5H), 7.35–7.42 (m, 3H), 7.74 (s, 1H), 11.23 (s, 1H) ppm.</div></div><div id="sec5_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> 5-Methyl-1-(1-(3-(3-(trifluoromethyl)phenoxy)benzyl)piperidin-3-yl)pyrimidine-2,4(1<i>H</i>,3<i>H</i>)-dione (<b>16</b>)</h4><div class="NLM_p last">LC-MS (Method 1) <i>t</i><sub>R</sub> = 3.86 min, <i>m</i>/<i>z</i> = 460 (M + 1), 458 (M – 1). HRMS calcd for C<sub>24</sub>H<sub>25</sub>F<sub>3</sub>N<sub>3</sub>O<sub>3</sub> 460.1843, found 460.1842. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.57 (m, 1H), 1.66–1.83 (m, 3H), 1.83 (s, 3H), 2.08 (t, 1H), 2.24 (t, 1H), 2.69–2.83 (m, 2H), 3.57 (s, 2H), 4.38–4.46 (m, 1H), 7.02 (dd, 1H), 7.08 (s, 1H), 7.19 (d, 1H), 7.32 (d, 2H), 7.43 (t, 1H), 7.51 (d, 1H), 7.65 (t, 1H), 7.74 (s, 1H), 11.22 (s, 1H) ppm.</div></div><div id="sec5_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 5-Methyl-1-{1-[3-(4-methylphenoxy)benzyl]piperidin-3-yl}pyrimidine-2,4(1<i>H</i>,3<i>H</i>)-dione (<b>17</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 2.64 min, <i>m</i>/<i>z</i> = 406.4 (M + 1). HRMS calcd for C<sub>24</sub>H<sub>28</sub>N<sub>3</sub>O<sub>3</sub> 406.2125, found 406.2130. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 1.68–1.74 (m, 3H), 1.80–1.82 (m, 1H), 1.90 (s, 3H), 2.34 (m, 5H), 2.58 (m, 1H), 2.77–2.79 (m, 1H), 3.49 (q, <i>J =</i> 13.20 Hz, 2H), 4.58–4.60 (m, 1H), 6.86–7.02 (m, 5H), 7.14 (d, <i>J =</i> 8.40 Hz, 2H), 7.23–7.28 (m, 1H), 7.69 (s, 1H), 8.12 (s, 1H) ppm.</div></div><div id="sec5_3_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> 1-(1-(3-(4-Chlorophenoxy)benzyl)piperidin-3-yl)-5-methylpyrimidine-2,4(1<i>H</i>,3<i>H</i>)-dione (<b>18</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 2.71 min, <i>m</i>/<i>z</i> = 426, 428 (M + 1, M + 2). HRMS calcd for C<sub>23</sub>H<sub>25</sub>ClN<sub>3</sub>O<sub>3</sub> 426.1583, found 426.1577. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 1.69–1.86 (m, 4H), 1.91 (s, 3H), 2.35 (m, 2H), 2.62 (m, 1H), 2.79–2.81 (m, 1H), 3.51 (q, <i>J =</i>13.20 Hz, 2H), 4.58–4.60 (m, 1H), 6.89–6.90 (m, 3H), 7.01 (s, 1H), 7.06–7.08 (m, 1H), 7.28–7.32 (m, 3H), 7.62 (m, 1H), 8.13 (s, 1H) ppm.</div></div><div id="sec5_3_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 1-(1-(3-(4-Methoxyphenoxy)benzyl)piperidin-3-yl)-5-methylpyrimidine-2,4(1<i>H</i>,3<i>H</i>)-dione (<b>19</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 2.50 min, <i>m</i>/<i>z</i> = 422.1 (M + 1). HRMS calcd for C<sub>24</sub>H<sub>28</sub>N<sub>3</sub>O<sub>4</sub> 422.2074, found 422.2072. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 1.66–1.75 (m, 3H), 1.81–1.83 (m, 1H), 1.91 (s, 3H), 2.37–2.42 (m, 2H), 2.59 (m, 1H), 2.78 (dd, <i>J =</i>3.68, 11.34 Hz, 1H), 3.49 (q, <i>J =</i>13.20 Hz, 2H), 3.82 (s, 3H), 4.57–4.60 (m, 1H), 6.82 (dd, <i>J =</i> 1.84, 8.12 Hz, 1H), 6.88–6.93 (m, 3H), 6.96–7.00 (m, 3H), 7.22–7.25 (m, 1H), 7.67 (s, 1H), 8.05 (s, 1H) ppm.</div></div><div id="sec5_3_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> (<i>S</i>)-5-Methyl-1-(1-(3-(<i>m</i>-tolyloxy)benzyl)piperidin-3-yl)pyrimidine-2,4(1<i>H</i>,3<i>H</i>)-dione (<b>20</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 2.70 min, <i>m</i>/<i>z</i> = 406.2 (M). HRMS calcd for C<sub>24</sub>H<sub>28</sub>N<sub>3</sub>O<sub>3</sub> 484.1634, found 406.2105. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.60 (m, 1H), 1.65–1.80 (s and m, 6H), 2.05 (t, 1H), 2.20 (t, 1H), 2.54 (m, 2H), 3.50 (s, 2H), 4.40 (m, 1H), 6.95 (m, 1H), 7.01 (m’s, 2H), 7.15 (m’s, 2H), 7.33 (m, 2H), 7.37 (t, 1H), 7.71 (s, 1H), 11.2 (br s, 1H) ppm.</div></div><div id="sec5_3_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> (<i>S</i>)-1-(1-(3-(3-Fluorophenoxy)benzyl)piperidin-3-yl)-5-methylpyrimidine-2,4(1<i>H</i>,3<i>H</i>)-dione (<b>21</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 2.77 min, <i>m</i>/<i>z</i> = 410.1 (M + 1). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 1.60–1.88 (s and m, 7H), 2.23 (t, 1H), 2.35 (t, 1H), 2.75 (m, 1H), 2.83 (m, 1H), 3.57 (s, 2H), 4.57 (m, 1H), 6.78 (m, 1H), 6.83 (m, 1H), 6.96 (m, 1H), 7.06 (m, 1H), 7.16 (d, 1H), 7.34 (m, 2H), 7.74 (s, 1H) ppm.</div></div><div id="sec5_3_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> (<i>S</i>)-1-(1-(3-(3-Chlorophenoxy)benzyl)piperidin-3-yl)-5-methylpyrimidine-2,4(1<i>H</i>,3<i>H</i>)-dione (<b>22</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 1.90 min, <i>m</i>/<i>z</i> = 426.1 (M + 1). <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 1.47–1.87 (m, 7H) 2.00–2.34 (m, 2H), 2.63 (br s, 1H), 2.68–2.84 (m, 1H), 3.47 (s, 2H), 4.45 (br s, 1H), 6.70–6.88 (m, 3H), 6.88–7.13 (m, 3H), 7.13–7.32 (m, 2H), 7.64 (br s, 1H) ppm.</div></div><div id="sec5_3_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> (<i>S</i>)-1-(1-(3-(3-Bromophenoxy)benzyl)piperidin-3-yl)-5-methylpyrimidine-2,4(1<i>H</i>,3<i>H</i>)-dione (<b>23</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 3.04 min, <i>m/z =</i> 469.8 471.7 (M + 1, M + 3). HRMS calcd for C<sub>23</sub>H<sub>25</sub>BrN<sub>3</sub>O<sub>3</sub> 470.1074, found 470.1057. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.60 (m, 1H), 1.65–1.80 (s and m, 6H), 2.05 (t, 1H), 2.20 (t, 1H), 2.54 (m, 2H), 3.50 (s, 2H), 4.40 (m, 1H), 6.95 (m, 1H), 7.01 (m’s 2H), 7.15 (m’s, 2H), 7.33 (m, 2H), 7.37 (t, 1H), 7.71 (s, 1H), 11.2 (br s, 1H) ppm.</div></div><div id="sec5_3_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> (<i>S</i>)-1-(1-(3-(3-Chloro-4-methylphenoxy)benzyl)piperidin-3-yl)-5-methylpyrimidine-2,4(1<i>H</i>,3<i>H</i>)-dione (<b>24</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 3.19 min, <i>m</i>/<i>z</i> = 439.95, 441.88 (M + 1, M + 3). HRMS calcd for C<sub>24</sub>H<sub>27</sub>ClN<sub>3</sub>O<sub>3</sub> 440.1735, found 440.1723. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 1.66–1.90 (s and m, 7H), 2.23 (m, 1H), 2.33 (m, 4H), 2.73 (m, 1H), 2.85 (m, 1H), 3.55 (s, 2H), 4.55 (m, 1H), 6.83 (m, 1H), 6.83 (m, 1H), 6.91 (m, 1H), 6.94 (d, 1H), 7.00 (m, 1H), 7.11 (d, <i>J =</i> 7.83, 1H), 7.24 (d, <i>J =</i> 8.34, 1H), 7.33 (d, <i>J =</i> 7.83, 1H) 7.75 (s, 1H) ppm.</div></div><div id="sec5_3_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> (<i>S</i>)-1-(1-(3-(3-Chloro-5-fluorophenoxy)benzyl)piperidin-3-yl)-5-methylpyrimidine-2,4(1<i>H</i>,3<i>H</i>)-dione (<b>25</b>)</h4><div class="NLM_p last">LC-MS (Method 1) <i>t</i><sub>R</sub> = 1.86 min, <i>m</i>/<i>z</i> = 444.1 (M + 1), 442.3 (M – 1). HRMS calcd for C<sub>23</sub>H<sub>24</sub>ClFN<sub>3</sub>O<sub>3</sub> 444.1485, found 444.1487. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 1.48–1.81 (m, 7H), 2.13 (br s, 1H), 2.24 (t, <i>J =</i> 10.17 Hz, 1H), 2.65 (d, <i>J =</i> 11.30 Hz, 1H), 2.76 (dd, <i>J =</i> 10.55, 3.77 Hz, 1H), 3.49 (s, 2H), 4.36–4.55 (m, 1H), 6.56 (d, <i>J =</i> 10.55 Hz, 1 H), 6.66 (s, 1H), 6.81 (d, <i>J =</i> 8.29 Hz, 1H), 6.90 (d, <i>J =</i> 8.29 Hz, 1H), 6.99 (s, 1H), 7.11 (d, <i>J =</i> 7.54 Hz, 1H), 7.30 (t, <i>J</i> = 7.91 Hz, 1H), 7.65 (s, 1H) ppm.</div></div><div id="sec5_3_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> (<i>S</i>)-1-(1-(3-(3,5-Dichlorophenoxy)benzyl)piperidin-3-yl)-5-methylpyrimidine-2,4(1<i>H</i>,3<i>H</i>)-dione (<b>26</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 1.92 min, <i>m</i>/<i>z</i> = 460.1, 462.5 (M, M + 2). HRMS calcd for C<sub>23</sub>H<sub>24</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>3</sub> 460.1189, found 460.1184. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 1.83 (m, 7H), 2.28 (dt, 2H), 2.8 (dm, 2H), 3.58 (s, 2H), 4.55 (m, 1H), 6.89 (m, 2H), 6.93 (d, 1H), 7.08 (s, 1H), 7.15 (s, 1H), 7.28 (m, 1H), 7.39 (m, 1H), 7.74 (s, 1H) ppm.</div></div><div id="sec5_3_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> (<i>S</i>)-1-(1-(3-(4-Chlorophenoxy)benzyl)piperidin-3-yl)-5-methylpyrimidine-2,4(1<i>H</i>,3<i>H</i>)-dione (<b>27</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 1.96 min, <i>m</i>/<i>z</i> = 426.1 (M + 1). HRMS calcd for C<sub>23</sub>H<sub>25</sub>ClN<sub>3</sub>O<sub>3</sub> 426.1579, found 426.1570. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.79 (m, 7H), 2.82 (s, 1H), 3.17 (m, 1H), 3.42 (t, 2H), 4.32 (m, 2H), 4.72 (br t, 1H), 7.08 (s, 1H), 7.10 (s, 1H), 7.20 (d, 1H), 7.24 (s, 1H), 7.37 (d, 1H), 7.43–7.55 (m, 4H), 9.9 (br s, 1H), 11.4 (s, 1H) ppm.</div></div><div id="sec5_3_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> (<i>R</i>)-1-(1-(3-(4-Chlorophenoxy)benzyl)piperidin-3-yl)-5-methylpyrimidine-2,4(1<i>H</i>,3<i>H</i>)-dione (<b>28</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 1.82 min, <i>m</i>/<i>z</i> = 428.3 (M + 1). HRMS calcd for C<sub>23</sub>H<sub>25</sub>ClN<sub>3</sub>O<sub>3</sub> 426.1579, found 426.1583. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 1.54–1.93 (m, 7H), 2.05–2.42 (m, 2H), 2.63–2.96 (m, 2H), 3.55 (br s, 2H), 4.55 (d, <i>J =</i> 3.77 Hz, 1H), 6.78–7.05 (m, 4H), 7.10 (d, <i>J =</i> 7.54 Hz, 1H), 7.33 (d, <i>J =</i> 4.52 Hz, 3H), 7.72 (br s, 1H) ppm.</div></div><div id="sec5_3_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> (<i>S</i>)-1-(1-(3-(3,4-Dichlorophenoxy)benzyl)piperidin-3-yl)-5-methylpyrimidine-2,4(1<i>H</i>,3<i>H</i>)-dione (<b>29</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 2.29 min, <i>m</i>/<i>z</i> = 460.0, 461.9 (M + 1 M + 3). HRMS calcd for C<sub>23</sub>H<sub>24</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>3</sub> 460.1189, found 460.1190. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.75 (s, 3H), 1.90 (m, 4H), 2.85 (m, 1H), 3.20 (m, 1H), 3.45 (t, 2H), 4.30 (m, 2H), 4.75 (t, 1H), 7.07 (m, 1H), 7.20 (d, 1H), 7.35 (m’s, 3H), 7.55 (m’s, 2H), 7.85 (d, 1H), 10.6 (br s, 1H), 11.35 (s, 1H) ppm.</div></div><div id="sec5_3_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> (<i>R</i>)-1-(1-(3-(3,4-Dichlorophenoxy)benzyl)piperidin-3-yl)-5-methylpyrimidine-2,4(1<i>H</i>,3<i>H</i>)-dione (<b>30</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 2.04 min, <i>m</i>/<i>z</i> = 461.9 (M + 1), 458.3 (M – 1). HRMS calcd for C<sub>23</sub>H<sub>24</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>3</sub> 460.1189, found 460.1187. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 1.54–1.94 (m, 7H), 2.10–2.40 (m, 2H), 2.75 (d, <i>J =</i> 11.30 Hz, 1H), 2.80–2.92 (m, 1H), 3.20–3.45 (m, 2H), 3.45–3.67 (m, 2H), 4.54 (br s, 1H), 6.76–7.00 (m, 2H), 7.00–7.26 (m, 3H), 7.26–7.58 (m, 2H), 7.72 (br s, 1H) ppm.</div></div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> Experimental Procedures and Characterization Data for Analogues in Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></h3><div id="sec5_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> (<i>S</i>)-2-Fluoro-4-((3-(5-methyl-2,4-dioxo-3,4-dihydropyrimidin-1(2<i>H</i>)-yl)piperidin-1-yl)methyl)benzonitrile (<b>6</b>)</h4><div class="NLM_p last">(<i>S</i>)-5-Methyl-1-(piperidin-3-yl)pyrimidine-2,4(1<i>H</i>,3<i>H</i>)-dione (<b>1</b>, 1052 mg, 5.03 mmol), cyanoborohydride on resin (2515 mg, 5.03 mmol), and 2-fluoro-4-formylbenzonitrile (<b>5</b>, 500 mg, 3.35 mmol) were suspended in DMF (10 mL). Acetic acid (0.500 mL) was added, and the mixture was stirred at RT overnight. LCMS showed product. The resin was removed by filtration, and the crude was then diluted in ethyl acetate, washed with saturated aqueous sodium bicarbonate and water, and then dried over anhydrous sodium sulfate. After the solvent was removed under reduced pressure, the crude oil was purified by reverse phase chromatography HPLC (acetonitrile, water, 0.1% TFA) (5–50% acetonitrile). After the solvent was removed, a solid was isolated and identified as <b>6</b> (323 mg, 28%). LC-MS (Method 1) <i>t</i><sub>R</sub> = 0.88 min, <i>m</i>/<i>z</i> = 342.0, 344.0 (M, M + 2). <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 1.82 (overlap m and s, 7H), 2.19 (m, 1H), 2.33 (t, <i>J</i> = 10.5 Hz, 1H), 2.78 (d, <i>J</i> = 10.5 Hz, 1H), 2.89 (dd, <i>J</i> = 10.5, 4.5 Hz, 1H), 3.67 (s, 2H), 4.59 (m, 1H), 7.39 (m, 2H), 7.71 (m, 2H) ppm.</div></div><div id="sec5_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> (<i>S</i>)-2-(3-Chlorophenoxy)-4-((3-(5-methyl-2,4-dioxo-3,4-dihydropyrimidin-1(<i>2H</i>)-yl)piperidin-1-yl)methyl)benzonitrile (<b>7</b>)</h4><div class="NLM_p last">3-Chlorophenol (28.2 mg, 0.22 mmol), <b>6</b> (50 mg, 0.15 mmol), and anhydrous potassium carbonate (20.18 mg, 0.15 mmol) were suspended in <i>N</i>-methyl-2-pyrrolidinone (1 mL). The mixture was heated using a microwave oven at 150 °C for 15 min. The reaction mixture was then purified by preparative HPLC, eluting with mixtures of acetonitrile/water/0.1% TFA. The fractions containing clean product were mixed, and the solvent was removed under reduced pressure, yielding <b>7</b> (30.0 mg, 45%) as a solid. LC-MS (Method 1) <i>t</i><sub>R</sub> = 1.79 min, <i>m</i>/<i>z</i> = 451.1 (M + 1), 449.2 (M – 1).</div></div><div id="sec5_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> (<i>S</i>)-2-(3-Chlorophenoxy)-4-((3-(5-methyl-2,4-dioxo-3,4-dihydropyrimidin-1(2<i>H</i>)-yl)piperidin-1-yl)methyl)benzoic Acid (<b>8</b>)</h4><div class="NLM_p last"><b>7</b> (30 mg, 0.07 mmol) was suspended in water (1 mL). Sulfuric acid (0.100 mL) was added, and the sample was heated using microwave irradiation for 30 min at 150 °C. The crude was then purified using preparative HPLC, eluting with mixtures of acetonitrile/water/TFA 0.1%. The recovered clean fractions were combined, and the solvent was removed using the lyophilizer, yielding <b>8</b> (3.40 mg, 9%) as an off white solid. LC-MS (Method 1) <i>t</i><sub>R</sub> = 1.42 min, <i>m</i>/<i>z</i> = 470.1, 468.2 (M + 1, M – 1). HRMS calcd for C<sub>24</sub>H<sub>25</sub>ClN<sub>3</sub>O<sub>5</sub> 470.1477, found 470.1469. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 1.89 (m, 4H), 2.08 (m, 3H), 3.0 (br t, 1H), 3.23 (apparent t, overlap with solvent signal, 1H), 3.5 (m, 2H), 4.38 (m, 2H), 4.67 (m, 1H), 6.9 (dd, <i>J =</i> 8.3, 1.1 Hz, 1H), 7.0 (t, <i>J =</i> 2.2 Hz, 1H), 7.13 (m, 1H), 7.24 (d, <i>J =</i> 1.5 Hz, 1H), 7.33 (m, 1H), 7.43 (m, 1H), 8.04 (d, <i>J =</i> 7.9 Hz, 1H) ppm.</div></div><div id="sec5_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> (<i>S</i>)-2-(3-Bromophenoxy)-4-((3-(5-methyl-2,4-dioxo-3,4-dihydropyrimidin-1(2<i>H</i>)-yl)piperidin-1-yl)methyl)benzoic Acid (<b>31</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 1.45 min, <i>m</i>/<i>z</i> = 514.0, 516.0 (M + 1, M + 3), 512.3 (M – 1). HRMS calcd for C<sub>24</sub>H<sub>25</sub>BrN<sub>3</sub>O<sub>5</sub> 514.0972, found 514.0947. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 1.78–2.05 (m, 6H), 2.84 (t, 1H), 3.06 (t, 1H), 3.33 (m, 2H), 4.21 (m, 2H), 4.57 (br s, 1H), 6.85 (m, 1H), 7.03 (s, 1H), 7.15 (m, 1H), 7.34 (d, 1H), 7.39 (m, 1H), 7.92 (d, 1H) ppm.</div></div><div id="sec5_4_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> (<i>S</i>)-4-((3-(5-Methyl-2,4-dioxo-3,4-dihydropyrimidin-1(2<i>H</i>)-yl)piperidin-1-yl)methyl)-2-(<i>m</i>-tolyloxy)benzoic Acid (<b>32</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 1.38 min, <i>m</i>/<i>z</i> = 450.3 (M + 1), 448.34 (M – 1). HRMS calcd for C<sub>25</sub>H<sub>28</sub>N<sub>3</sub>O<sub>5</sub> 450.2023, found 450.2004. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 1.78 (s, 3H), 1.82–2.14 (m, 3H), 2.21 (s, 2H), 2.87 (br s, 1H), 3.10 (t, <i>J =</i> 11.68 Hz, 1H), 3.29–3.46 (m, 2H), 4.11–4.34 (m, 2H), 4.55 (dd, <i>J =</i> 10.55, 5.27 Hz, 1H), 6.68 (d, <i>J =</i> 7.54 Hz, 1H), 6.76 (s, 1H), 6.87 (d, <i>J =</i> 7.54 Hz, 1H), 7.02 (s, 1H), 7.13 (t, <i>J =</i> 7.54 Hz, 1H), 7.23 (d, <i>J =</i> 9.80 Hz, 1H), 7.32 (s, 1H), 7.88 (d, <i>J =</i> 8.29 Hz, 1H) ppm.</div></div><div id="sec5_4_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> (<i>S</i>)-2-(3-Fluorophenoxy)-4-((3-(5-methyl-2,4-dioxo-3,4-dihydropyrimidin-1(2<i>H</i>)-yl)piperidin-1-yl)methyl)benzoic Acid (<b>33</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 1.3 min, <i>m</i>/<i>z</i> = 454.3 (M + 1), 452.2 (M – 1). HRMS calcd for C<sub>24</sub>H<sub>25</sub>FN<sub>3</sub>O<sub>5</sub> 454.1773, found 454.1759. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 1.47–1.81 (m, 7 H), 2.15 (br s, 1H), 2.28 (t, <i>J =</i> 10.17 Hz, 1H), 2.68 (d, <i>J =</i> 10.55 Hz, 1H), 2.79 (dd, <i>J =</i> 10.55, 3.77 Hz, 1H), 3.54 (s, 2H), 4.44 (br s, 1H), 6.49–6.76 (m, 3H), 6.97 (s, 1H), 7.17 (dd, <i>J =</i> 14.69, 7.16 Hz, 2H), 7.58 (s, 1H), 7.74 (d, <i>J =</i> 8.29 Hz, 1H) ppm.</div></div><div id="sec5_4_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 4-((3-(5-Methyl-2,4-dioxo-3,4-dihydropyrimidin-1(2<i>H</i>)-yl)piperidin-1-yl)methyl)-2-(3-(trifluoromethyl)phenoxy)benzoic Acid (<b>34</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 2.46 min, <i>m</i>/<i>z</i> = 504.2 (M + 1). HRMS calcd for C<sub>25</sub>H<sub>25</sub>FN<sub>3</sub>O<sub>5</sub> 504.1741, found 504.1742. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 1.48–1.53 (m, 1H), 1.65–1.71 (m, 3H), 1.73 (s, 3H), 2.03 (t, <i>J =</i> 10.04 Hz, 1H), 2.20 (t, <i>J =</i> 10.16 Hz, 1H), 2.66–2.76 (m, 2H), 3.57 (s, 2H), 4.35–4.39 (m, 1H), 7.09–7.14 (m, 3H), 7.29 (d, <i>J =</i> 8.08 Hz, 1H), 7.40 (d, <i>J =</i> 7.72 Hz, 1H), 7.56 (t, <i>J =</i> 7.88 Hz, 1H), 7.68 (s, 1H), 7.85 (d, <i>J =</i> 7.92 Hz, 1H), 11.19 (s, 1H), 12.88 (br s, 1H) ppm.</div></div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> Experimental Procedures and Characterization Data for Analogs in Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></h3><div id="sec5_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 1-(4-Cyano-3-fluorophenyl)-(3,3-dimethylbutyl)methanesulfonate (<b>11b</b>)</h4><div class="NLM_p last">(4-Cyano-3-fluorophenyl)zinc(II) bromide (<b>9</b>, 163 mL, 81.72 mmol) was added dropwise (over 20 min) to a suspension of 3,3-dimethylbutanoyl chloride (<b>10</b>, 10.37 mL, 74.29 mmol) and tetrakis(triphenylphosphine)palladium(0) (2.146 g, 1.86 mmol) in tetrahydrofuran (200 mL) at room temperature and under an atmosphere of nitrogen. The solution turned from pale orange to dark red overnight. The crude was diluted in ethyl acetate (200 mL) and washed with aqueous saturated sodium bicarbonate, water, and aqueous 1 N HCl. The organic extract was dried over anhydrous sodium sulfate, and the solvent was removed under reduced pressure, yielding a thick reddish oil containing a shiny solid. The crude oil was suspended in diethyl ether, and the solids were removed by filtration. The crude was then purified by flash chromatography on silica gel (330 g) eluting with mixtures of hexanes/ethyl acetate (from 0 to 30% ethyl acetate in 53 min). The main fraction was combined and the solvents were removed under reduced pressure, yielding 4-(3,3-dimethylbutanoyl)-2-fluorobenzonitrile (<b>11a</b>, 13.80 g, 85%) as a semisolid (pale yellow). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.09 (s, 9H), 2.87 (s, 2H), 7.57–7.93 (m, 3H) ppm. <b>11a</b> (1.5 g, 6.84 mmol) was dissolved in MeOH (10 mL). NaBH<sub>4</sub> (0.227 g, 5.99 mmol) was added in small portions (gas evolution). The reaction was stirred for 2 h. The crude was dissolved in ethyl acetate and washed with 1 N HCl, saturated aqueous sodium bicarbonate, and brine. The organic extract was dried over anhydrous sodium sulfate, and the solvent was then removed under reduced pressure. The crude oil was purified by flash chromatography on silica gel (40 g), eluting with mixtures of hexanes/ethyl acetate (from 0 to 70% ethyl acetate in 25 min). The cleanest fractions were combined, and the solvent was removed under reduced pressure, yielding 2-fluoro-4-(1-hydroxy-3,3-dimethylbutyl)benzonitrile (0.433 g, 29%) as a colorless oil. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.05 (s, 9H), 1.53 (dd, <i>J =</i> 14.8, 2.7 Hz, 1H), 1.7 (dd, <i>J =</i> 14.8, 8.9 Hz, 1H), 4.9 (dd, <i>J =</i> 8.8, 2.2 Hz, 1H), 7.2 (m, 2H, obscured by CHCl<sub>3</sub> signal), 7.6 (m, 1H) ppm. 2-Fluoro-4-(1-hydroxy-3,3-dimethylbutyl)benzonitrile (433 mg, 1.9 mmol), and TEA (0.552 mL, 3.96 mmol) were dissolved in dichloromethane (5 mL) at 0 °C. Methanesulfonyl chloride (0.169 mL, 2.18 mmol) was added, and the reaction was stirred for 4 h. The reaction mixture was diluted with dichloromethane, washed with 0.5 N HCl, saturated aqueous sodium bicarbonate, and brine. The organic extract was dried over anhydrous sodium sulfate, and the solvent was removed under reduced pressure, yielding <b>11b</b> as an oil which was used in the next step without further purification. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.01 (s, 9H), 1.61 (m, overlap with water, 1H), 2.05 (dd, <i>J =</i> 15.1, 8.85 Hz, 1H), 2.84 (s, 3H), 5.71 (dd, <i>J =</i> 8.85, 3.58 Hz 1H), 7.29 (m, 2H, obscured by CHCl<sub>3</sub> signal), 7.68 (dd, <i>J =</i> 8.10, 6.64 Hz, 1H) ppm.</div></div><div id="sec5_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 4-(3,3-Dimethyl-1-((<i>S</i>)-3-(5-methyl-2,4-dioxo-3,4-dihydropyrimidin-1-(<i>2H</i>)-yl)piperidin-1-yl)butyl)-2-fluorobenzonitrile (<b>12</b>)</h4><div class="NLM_p last"><b>11b</b> (4.1 g, 13.70 mmol), (<i>S</i>)-5-methyl-1-(piperidin-3-yl)pyrimidine-2,4(1H,3H)-dione (<b>1</b>, 7.16 g, 34.24 mmol), and DIEA (7.18 mL, 41.09 mmol) were dissolved in acetonitrile (40 mL). The mixture was heated at 70 °C for 20 h. LCMS showed the desired product. The reaction was diluted in ethyl acetate and washed with brine. The organic extract was dried over anhydrous sodium sulfate. After removing the solvent under reduced pressure, the crude was purified by flash chromatography on silica gel (80 g), eluting with mixtures of hexanes/ethyl acetate (from 0 to 100% ethyl acetate in 25 min). The cleanest fractions were combined, and the solvent was removed under reduced pressure, yielding <b>12</b> (0.710 g, 13%) as an off white solid. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 0.9 (bs, 9H), 1.56 (bt, 2H), 1.8 (m, 9H), 2.89 (dd, 1H), 2.95 (m, 1H), 3.88 (m, 1H), 4.53 (m, 1H), 7.3 (m, 2H), 7.52 (d, <i>J</i> = 18.6 Hz, 1H), 7.73 (m, 1H) ppm.</div></div><div id="sec5_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 2-(3-Bromophenoxy)-4-(3,3-dimethyl-1-((<i>S</i>)-(5-methyl-2,4-dioxo-3,4-dihydropyrimidin-1-(<i>2H</i>)-yl)piperidin-1-yl)butyl)benzoic Acid (<b>13</b>)</h4><div class="NLM_p last"><b>12</b> (2.7 g, 6.55 mmol) was dissolved in NMP (20 mL). 3-Bromophenol (1.699 g, 9.82 mmol) and K<sub>2</sub>CO<sub>3</sub> (2.71 g, 19.64 mmol) were added, and the mixture was split in two and heated in the microwave at 150 °C for 30 min (two vials). The crudes were combined and diluted with ethyl acetate and then washed with aqueous saturated potassium bicarbonate and water. After drying over anhydrous sodium sulfate, the solvent was removed under reduced pressure, yielding a crude oil. The crude was then purified by flash chromatography (220 g) eluting with a mixture of hexanes/ethyl acetate (0–100% in 30 min). The fractions containing product were combined and the solvent removed under reduced pressure, yielding 2-(3-bromophenoxy)-4-(3,3-dimethyl-1-((<i>S</i>)-3-(5-methyl-2,4-dioxo-3,4-dihydropyrimidin-1(2<i>H</i>)-yl)piperidin-1-yl)butyl)benzonitrile (2.50 g, 67%) as a yellow foam/solid. <i>t</i><sub>R</sub> = 2.58 min, <i>m</i>/<i>z</i> = 565.0, 567.0 (M + 1, M + 3). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 0.86 (m, 9H), 1.76 (m, 9H), 2.58 (d, <i>J =</i> 10.9 Hz, 1H), 2.76 (d, <i>J =</i> 10.4 Hz, 1H), 2.8 (m, 1H), 3.64 (m, 1H), 4.56 (m, 1H), 6.76 (m, 1H), 7.03 (m, 2H), 7.17 (m, 1H), 7.31 (m, 2H), 7.66 (dd, <i>J =</i> 7.9, 2.3 Hz, 2H), 9.0 (d, <i>J =</i> 5.1 Hz, 1H) ppm. 2-(3-Bromophenoxy)-4-(3,3-dimethyl-1-((<i>S</i>)-3-(5-methyl-2,4-dioxo-3,4-dihydropyrimidin-1(2<i>H</i>)-yl)piperidin-1-yl)butyl)benzonitrile (2.5 g, 4.42 mmol) was then dissolved in ethanol (15 mL) and water (15 mL). Sodium hydroxide (2.65 g, 66.31 mmol) was added, and the mixture was heated at 100 °C for 16 h. The LCMS of the crude showed only acid. The ethanol was removed under a stream of nitrogen. The aqueous solution was treated with concentrated aqueous HCl, and a solid was formed and isolated via filtration. The solid was suspended in ethyl acetate and washed with water. The organic fraction was isolated and dried over anhydrous sodium sulfate, and the solvent was then removed under reduced pressure, yielding the title compound <b>13</b> as an off white solid (2 g, 77%). LC-MS (Method 2) <i>t</i><sub>R</sub> = 2.07 min, <i>m</i>/<i>z</i> = 584.1, 586.1 (M + 1, M + 3). <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 0.85 (s, 9H), 2.01 (m, 9H), 2.36 (br t, 1H), 2.8 (m, 1H), 3.06 (m, 1H), 3.55 (d, <i>J =</i> 11.3 Hz, 1H), 3.73 (d, <i>J =</i> 11.3 Hz, 1H), 4.61 (dd, <i>J =</i> 10.5, 7.9 Hz, 1H), 6.93 (m, 1H), 7.07 (m, 1H), 7.27 (m, 2H), 7.40 (m, 1H), 7.47 (d, <i>J =</i> 4.71 Hz, 1H), 7.60 (d, <i>J =</i> 8.1 Hz, 1H), 8.09 (d, <i>J =</i> 8.1 Hz, 1H) ppm.</div></div><div id="sec5_5_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 2-(3-Bromophenoxy)-4-((<i>R</i>)-3,3-dimethyl-1-((<i>S</i>)-(5-methyl-2,4-dioxo-3,4-dihydropyrimidin-1-(<i>2H</i>)-yl)piperidin-1-yl)butyl)benzoic Acid (<b>14</b>)</h4><div class="NLM_p last">Diastereomeric mixture <b>13</b> was subjected to HPLC preparative chiral separation eluting with a mixture of hexanes (60%), methanol/ethanol (1:1) (40%), and diethylamine (0.5%). The title compound eluted second. After removing the solvent, the sample (825 mg) contained residual diethyl amine that was removed by preparative HPLC (XBridge C18 OBD, 19 mm × 150 mm, 5 μm; flow rate 20 mL/min) eluting with a mixture of a linear gradient from 40 to 69% in 7.5 min methanol in H<sub>2</sub>O aqueous (0.2% NH<sub>4</sub>OH). After lyophilizing the sample, the title compound <b>14</b> was obtained as an off white solid (400 mg, 15%). LC-MS (Method 2) <i>t</i><sub>R</sub> = 2.2 min, <i>m</i>/<i>z</i> = 584.0, 586.0 (M + 1, M + 3) HRMS calcd for C<sub>29</sub>H<sub>35</sub>BrN<sub>3</sub>O<sub>5</sub> 584.1755, found 584.1746. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 0.87 (s, 9H), 1.76 (m, 9H), 2.12 (m, 2H), 2.82 (d, <i>J =</i> 10.7 Hz, 1H), 3.0 (d, <i>J =</i> 7.2 Hz, 1H), 3.84 (t, <i>J =</i> 6.3 Hz, 1H), 4.49 (br s, 1H), 6.9 (m, 2H), 7.04 (d, <i>J =</i> 1.9 Hz, 1H), 7.18 (m, 3H), 7.53 (s, 1H), 7.79 (d, <i>J =</i> 7.91 Hz, 1H) ppm. Diastereomeric ratio: >98:1. Anal. Calcd for C<sub>29</sub>H<sub>34</sub>BrN<sub>3</sub>O<sub>5</sub>·H<sub>2</sub>O·0.05C<sub>3</sub>H<sub>8</sub>O: C, 58.51; H, 6.09; N, 6.8. Found: C, 58.41 ; H, 5.92; N, 6.90.</div></div><div id="sec5_5_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 2-(3-Chlorophenoxy)-4-((<i>R</i>)-1-((<i>S</i>)-3-(5-methyl-2,4-dioxo-3,4-dihydropyrimidin-1(2<i>H</i>)-yl)piperidin-1-yl)ethyl)benzoic Acid (<b>35</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 1.46 min, <i>m</i>/<i>z</i> = 484.0 (M + 1), 482.2 (M – 1). HRMS calcd for C<sub>25</sub>H<sub>27</sub>ClN<sub>3</sub>O<sub>5</sub> 484.1634, found 484.1618. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 1.31 (d, <i>J =</i> 6.03 Hz, 3H), 1.47–1.68 (m, 2H), 1.68–1.86 (m, 5H), 1.99–2.35 (m, 2H), 2.82 (t, <i>J =</i> 10.55 Hz, 2H), 3.61 (d, <i>J =</i> 6.78 Hz, 1H), 4.40 (br s, 1H), 6.73 (d, <i>J =</i> 8.29 Hz, 1H), 6.82 (s, 1H), 6.88–7.05 (m, 2H), 7.19 (t, <i>J =</i> 8.29 Hz, 2 H), 7.49 (s, 1H), 7.78 (d, <i>J =</i> 8.29 Hz, 1H) ppm.</div></div><div id="sec5_5_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> 2-(3-Chlorophenoxy)-4-((<i>S</i>)-1-((<i>S</i>)-3-(5-methyl-2,4-dioxo-3,4-dihydropyrimidin-1(2<i>H</i>)-yl)piperidin-1-yl)ethyl)benzoic Acid (<b>36</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 1.47 min, <i>m</i>/<i>z</i> = 484.0 (M + 1). HRMS calcd for C<sub>25</sub>H<sub>27</sub>ClN<sub>3</sub>O<sub>5</sub> 484.1634, found 484.1620. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 1.30 (d, <i>J =</i> 6.78 Hz, 3H), 1.44–1.83 (m, 7H), 2.08 (d, <i>J =</i> 12.06 Hz, 1H), 2.24 (t, <i>J =</i> 10.17 Hz, 1H), 2.68 (d, <i>J =</i> 12.06 Hz, 1H), 2.96 (d, <i>J =</i> 9.04 Hz, 1H), 3.57 (t, <i>J =</i> 6.40 Hz, 1H), 4.45 (br s, 1H), 6.65–6.86 (m, 2H), 6.87–7.01 (m, 2H), 7.06–7.26 (m, 2H), 7.55 (s, 1H), 7.78 (d, <i>J =</i> 8.29 Hz, 1H) ppm.</div></div><div id="sec5_5_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> (3-Chlorophenoxy)-4-((<i>R</i>)-1-((<i>S</i>)-3-(5-methyl-2,4-dioxo-3,4-dihydropyrimidin-1(2<i>H</i>)-yl)piperidin-1-yl)propyl)benzoic Acid (<b>37</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 1.6 min, <i>m</i>/<i>z</i> = 500.0 (M + 1), 496.2 (M – 1). HRMS calcd for C<sub>26</sub>H<sub>29</sub>ClN<sub>3</sub>O<sub>5</sub> 498.1790, found 498.1790. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 0.83 (t, <i>J =</i> 7.16 Hz, 3H), 1.88 (s, 3H), 1.93–2.11 (m, 3H), 2.11–2.43 (m, 3H), 2.89 (br s, 1H), 3.09 (br s, 1H), 3.46 (d, <i>J =</i> 10.55 Hz, 1H), 3.56 (br s, 1H), 4.38 (d, <i>J =</i> 7.54 Hz, 1H), 4.68 (br s, 1H), 6.88 (d, <i>J =</i> 8.29 Hz, 1H), 7.00 (s, 1H), 7.13 (d, <i>J =</i> 6.78 Hz, 1H), 7.26 (s, 1 H), 7.34 (t, <i>J =</i> 8.29 Hz, 1H), 7.39–7.56 (m, 2H), 8.09 (d, <i>J =</i> 8.29 Hz, 1H) ppm.</div></div><div id="sec5_5_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> (3-Chlorophenoxy)-4-((<i>S</i>)-1-((<i>S</i>)-3-(5-methyl-2,4-dioxo-3,4-dihydropyrimidin-1(2<i>H</i>)-yl)piperidin-1-yl)propyl)benzoic Acid (<b>38</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 1.68 min, <i>m</i>/<i>z</i> = 498.1 (M + 1), 496.2 (M – 1). <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 0.82 (t, 3H), 1.9 (s and m, 6H), 2.2 (br s, 1H), 3.08 (br s, 1H), 3.48 (br s, 1H), 3.70 (br s, 1H), 4.31 (m, 1H), 4.75 (m overlap with d-solvent signal, 1H), 6.9 (d, 1H), 7.01 (d, 1H), 7.15 (d, 1H), 7.23 (s, 1H), 7.34 (t, 1H), 7.45 (br s, 1H), 8.1 (d, 1H) ppm.</div></div><div id="sec5_5_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 2-(3-Chlorophenoxy)-4-((<i>R</i>)-1-((<i>S</i>)-3-(5-methyl-2,4-dioxo-3,4-dihydropyrimidin-1-(<i>2H</i>)-yl)piperidin-1-yl)butyl)benzoic Acid (<b>39</b>)</h4><div class="NLM_p last">LC-MS (Method 3) <i>t</i><sub>R</sub> = 1.8 min, <i>m</i>/<i>z</i> = 512.2, 510.1 (M + 1, M – 1). HRMS calcd for C<sub>29</sub>H<sub>31</sub>ClN<sub>3</sub>O<sub>5</sub> 512.1947, found 512.1936. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ. 0.81 (t, <i>J =</i> 7.16 Hz, 3 H), 0.98–1.30 (m, 2 H), 1.51–1.90 (m, 9 H), 2.11 (t, <i>J =</i> 10.17 Hz, 2 H), 2.70–2.90 (m, 2 H), 3.53 (dd, <i>J =</i> 9.04, 5.27 Hz, 1 H), 4.37 (d, <i>J =</i> 9.04 Hz, 1 H), 6.75 (d, <i>J =</i> 8.29, Hz, 1 H), 6.84 (d, <i>J =</i> 12.81 Hz, 2 H), 6.96 (d, <i>J =</i> 6.03 Hz, 1 H), 7.10 (d, <i>J =</i> 8.29 Hz, 1 H), 7.19 (t, <i>J =</i> 8.29 Hz, 1 H), 7.52 (s, 1 H), 7.77 (d, <i>J =</i> 7.54 Hz, 1 H) ppm. Diastereomeric ratio: >99:1.</div></div><div id="sec5_5_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> 2-(3-Chlorophenoxy)-4-((<i>S</i>)-1-((<i>S</i>)-3-(5-methyl-2,4-dioxo-3,4-dihydropyrimidin-1(2<i>H</i>)-yl)piperidin-1-yl)butyl)benzoic Acid (<b>40</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 1.86 min, <i>m</i>/<i>z</i> = 512.3 (M + 1), 510.2 (M – 1). HRMS calcd for C<sub>27</sub>H<sub>31</sub>ClN<sub>3</sub>O<sub>5</sub> 512.1947, found 512.1936. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 0.81–1.01 (m, 3H), 1.08–1.29 (m, 2H), 1.57–1.97 (m, 9H), 2.12 (t, <i>J =</i> 10.17 Hz, 1H), 2.32 (t, <i>J =</i> 10.17 Hz, 1H), 2.86 (d, <i>J =</i> 11.30 Hz, 1H), 3.07 (d, <i>J =</i> 10.55 Hz, 1H), 3.61 (dd, <i>J =</i> 9.04, 5.27 Hz, 1H), 4.47–4.68 (m, 1H), 6.78–6.96 (m, 2 H), 6.99 (s, 1H), 7.07 (d, <i>J =</i> 7.54 Hz, 1H), 7.16–7.35 (m, 2H), 7.61 (s, 1H), 7.91 (d, <i>J =</i> 8.29 Hz, 1H) ppm. Diastereomeric ratio: >98:1.</div></div><div id="sec5_5_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> 2-(3-Chlorophenoxy)-4-((<i>R</i>)-3-methyl-1-((<i>S</i>)-3-(5-methyl-2,4-dioxo-3,4-dihydropyrimidin-1(2<i>H</i>)-yl)piperidin-1-yl)butyl)benzoic Acid (<b>41</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 1.9 min, <i>m</i>/<i>z</i> = 526.0 (M + 1). HRMS calcd for C<sub>28</sub>H<sub>33</sub>ClN<sub>3</sub>O<sub>5</sub> 526.2103, found 526.2100. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 0.79 (app t, 6H), 1.19 (app t, 1H), 1.31 (m, 1H), 1.66 (m, 6H), 1.75 (s, 3H), 2.11 (d, 2H), 2.86 (m, 3H), 3.64 (m, 1H), 4.38 (m, 1H), 6.75 (dd, 1H), 6.83 (m, 2H), 6.94 (m, 1H), 7.09 (d, 1H), 7.17 (app t, 1H), 7.47 (br s, 1H), 7.71 (d, 1H) ppm.</div></div><div id="sec5_5_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 2-(3-Chlorophenoxy)-4-((<i>S</i>)-3-methyl-1-((<i>S</i>)-3-(5-methyl-2,4-dioxo-3,4-dihydropyrimidin-1(2<i>H</i>)-yl)piperidin-1-yl)butyl)benzoic Acid (<b>42</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 1.9 min, <i>m</i>/<i>z</i> = 526.0 (M + 1). HRMS calcd for C<sub>28</sub>H<sub>33</sub>ClN<sub>3</sub>O<sub>5</sub> 526.2103, found 526.2091. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 0.76 (app t, 6H), 1.19 (app t, 1H), 1.25 (m, 1H), 1.60 (m, 9H), 1.92 (m, 1H), 2.10 (m, 1H), 2.70 (m, 1H), 2.86 (m, 1H), 3.47 (m, 1H), 4.46 (m, 1H), 6.79 (m, 2H), 6.83 (m, 1H), 6.91 (m, 1H), 7.01 (d, 1H), 7.14 (app t, 1H), 7.49 (br s, 1H), 7.57 (d, 1H) ppm.</div></div><div id="sec5_5_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> 2-(3-Bromophenoxy)-4-((<i>S</i>)-3,3-dimethyl-1-((<i>S</i>)-3-(5-methyl-2,4-dioxo-3,4-dihydropyrimidin-1(2<i>H</i>)-yl)piperidin-1-yl)butyl)benzoic Acid (<b>43</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 2.0 min, <i>m</i>/<i>z</i> = 586.3 (M + 1), 584.3 (M – 1). HRMS calcd for C<sub>29</sub>H<sub>35</sub>BrN<sub>3</sub>O<sub>5</sub> 584.1755, found 584.1742. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 0.75 (s, 9H), 1.47–1.62 (m, 2H), 1.62–1.90 (m, 7H), 1.96 (br s, 1H), 2.16 (t, <i>J =</i> 10.55 Hz, 1H), 2.93 (dd, <i>J =</i> 16.77, 11.68 Hz, 2H), 3.80 (dd, <i>J =</i> 8.29, 3.96 Hz, 1H), 4.48 (br s, 1H), 6.73–6.84 (m, 1H), 6.84–7.00 (m, 2H), 7.02–7.22 (m, 3H), 7.38 (s, 1H), 7.79 (d, <i>J =</i> 7.91 Hz, 1H) ppm. Diastereomeric ratio: >99:1.</div></div><div id="sec5_5_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> 2-(3-Chlorophenoxy)-4-((<i>R</i>)-3,3-dimethyl-1-((<i>S</i>)-3-(5-methyl-2,4-dioxo-3,4-dihydropyrimidin-1(2<i>H</i>)-yl)piperidin-1-yl)pentyl)benzoic Acid (<b>44</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 2.1 min, <i>m</i>/<i>z</i> = 554.2, 552.2 (M + 1, M + 3). HRMS calcd for C<sub>30</sub>H<sub>37</sub>ClN<sub>3</sub>O<sub>5</sub> 554.2416, found 554.2410. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 0.74 (s, 3H), 0.77–0.83 (m, 3 H), 0.85 (s, 3H), 1.16–1.28 (m, 2H), 1.86–2.00 (m, 7H), 2.12 (br s, 1H), 2.24–2.37 (m, 1H), 2.67 (s, 1H), 2.84 (br s, 1H), 3.43–3.64 (m, 3 H), 4.50 (d, <i>J =</i> 11.11 Hz, 1H), 6.88 (dd, <i>J =</i> 8.10, 1.13 Hz, 1H), 6.90–6.94 (m, 1H), 7.12 (d, <i>J</i> = 7.91 Hz, 1H), 7.30–7.38 (m, 2H), 7.42 (s, 1H), 7.52 (d, <i>J =</i> 6.41 Hz, 1H), 8.07 (d, <i>J =</i> 8.10 Hz, 1H) ppm.</div></div><div id="sec5_5_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> 2-(3-Chlorophenoxy)-4-((<i>R</i>)-3-ethyl-1-((<i>S</i>)-3-(5-methyl-2,4-dioxo-3,4-dihydropyrimidin-1(2<i>H</i>)-yl)piperidin-1-yl)pentyl)benzoic Acid (<b>45</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 2.13 min, <i>m</i>/<i>z</i> = 554.5 (M + 1), 552.5 (M – 1). HRMS calcd for C<sub>30</sub>H<sub>37</sub>ClN<sub>3</sub>O<sub>5</sub> 554.2416, found 554.2415. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 0.59–0.82 (m, 6H), 0.93 (br s, 1H), 1.03–1.33 (m, 7H), 1.40–1.79 (m, 9H), 1.94–2.16 (m, 2H), 2.70–2.88 (m, 2H), 2.94 (q, <i>J =</i> 7.28 Hz, 2H), 3.48–3.66 (m, 1H), 4.38 (br s, 1H), 6.67–6.86 (m, 3H), 6.93 (d, <i>J =</i> 7.16 Hz, 1H), 7.06 (d, <i>J</i> = 7.91 Hz, 1H), 7.11–7.25 (m, 1H), 7.47 (s, 1H), 7.68 (d, <i>J =</i> 7.72 Hz, 1H) ppm.</div></div><div id="sec5_5_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> 2-(3-Bromophenoxy)-4-((<i>R</i>)-1-((<i>S</i>)-3-(5-methyl-2,4-dioxo-3,4-dihydropyrimidin-1(2<i>H</i>)-yl)piperidin-1-yl)hexyl)benzoic Acid (<b>46</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 2.07 min, <i>m</i>/<i>z</i> = 584.2, 586.2 (M + 1, M + 3). HRMS calcd for C<sub>29</sub>H<sub>35</sub>BrN<sub>3</sub>O<sub>5</sub> 584.1755, found 584.1754. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 0.70–0.78 (m, 3H), 0.95–1.05 (m, 1H), 1.06–1.21 (m, 5H), 1.56 (d, <i>J =</i> 10.55 Hz, 1H), 1.63–1.84 (m, 7H), 1.98–2.07 (m, 1H), 2.17–2.27 (m, 2H), 2.76 (d, <i>J =</i> 12.06 Hz, 1H), 2.93–2.99 (m, 1H), 3.49 (dd, <i>J =</i> 8.67, 5.65 Hz, 1H), 4.47 (br s, 1H), 6.78–6.83 (m, 1H), 6.87 (d, <i>J =</i> 1.51 Hz, 1H), 6.92–6.95 (m, 1H), 7.09–7.14 (m, 3H), 7.49 (s, 1H), 7.80 (d, <i>J =</i> 8.10 Hz, 1H) ppm.</div></div><div id="sec5_5_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> (2-(3-Bromophenoxy)-4-((<i>R</i>)-3-ethyl-3-methyl-1-((<i>S</i>)-3-(5-methyl-2,4-dioxo-3,4-dihydropyrimidin-1(2<i>H</i>)-yl)piperidin-1-yl)pentyl)benzoic Acid (<b>47</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 2.26 min, <i>m</i>/<i>z</i> = 612.1, 614.1 (M + 1, M + 3). HRMS calcd for C<sub>31</sub>H<sub>39</sub>BrN<sub>3</sub>O<sub>5</sub> 612.2068, found 612.2075. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 0.69–0.81 (m, 9 H), 1.13–1.34 (m, 4H), 1.61 (d, <i>J =</i> 8.67 Hz, 2H), 1.74–1.89 (m, 7H), 1.90–2.00 (m, 1H), 2.17 (t, <i>J =</i> 10.46 Hz, 1H), 2.94 (br s, 2H), 3.76 (t, <i>J =</i> 6.03 Hz, 1H), 4.58 (br s, 1H), 6.83–6.99 (m, 2H), 7.03–7.10 (m, 1H), 7.14–7.26 (m, 3H), 7.54 (s, 1H), 7.81 (d, <i>J =</i> 7.91 Hz, 1H) ppm.</div></div><div id="sec5_5_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 2-(3-Chlorophenoxy)-4-((<i>R</i>)-3,3-dimethyl-1-((<i>S</i>)-3-(5-methyl-2,4-dioxo-3,4-dihydropyrimidin-1(2<i>H</i>)-yl)piperidin-1-yl)hexyl)benzoic Acid (<b>48</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 2.16 min, <i>m</i>/<i>z</i> = 568.1 (M + 1), 566.2 (M – 1). HRMS calcd for C<sub>31</sub>H<sub>39</sub>ClN<sub>3</sub>O<sub>5</sub> 568.2573, found 568.2562. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 0.64 (s, 3H), 0.69 (t, <i>J =</i> 6.78 Hz, 3H), 0.75 (s, 3H), 0.92–1.20 (m, 4H), 1.72–1.91 (m, 7H), 2.02 (d, <i>J =</i> 9.23 Hz, 1 H), 2.18–2.31 (m, 1H), 2.60 (br s, 1H), 2.85 (br s, 1H), 3.44 (d, <i>J</i> = 7.72 Hz, 1H), 3.52 (d, <i>J</i> = 10.74 Hz, 1H), 4.45 (d, <i>J</i> = 10.55 Hz, 1H), 4.55 (d, <i>J =</i> 7.54 Hz, 1H), 6.77 (ddd, <i>J =</i> 8.29, 2.45, 0.75 Hz, 1H), 6.80–6.84 (m, 1H), 7.01 (ddd, <i>J</i> = 8.01, 1.88, 0.85 Hz, 1H), 7.18–7.26 (m, 2 H), 7.29 (d, <i>J</i> = 1.13 Hz, 1H), 7.44 (dd, <i>J =</i> 8.01, 1.60 Hz, 1H), 7.97 (d, <i>J =</i> 8.10 Hz, 1H) ppm. Diastereomeric ratio: >99:1</div></div><div id="sec5_5_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> 2-(3-Chlorophenoxy)-4-((<i>R</i>)-2-cyclohexyl-1-((<i>S</i>)-3-(5-methyl-2,4-dioxo-3,4-dihydropyrimidin-1(2<i>H</i>)-yl)piperidin-1-yl)ethyl)benzoic Acid (<b>49</b>)</h4><div class="NLM_p last">LC-MS (Method 1) <i>t</i><sub>R</sub> = 2.12 min, <i>m</i>/<i>z</i> = 566.3 (M + 1), 564.3 (M – 1). HRMS calcd for C<sub>31</sub>H<sub>37</sub>ClN<sub>3</sub>O<sub>5</sub> 566.2416, found 566.2412. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 0.70–0.93 (m, 3H), 1.03 (d, <i>J =</i> 7.72 Hz, 4H), 1.19 (t, <i>J =</i> 7.35 Hz, 4H), 1.36–1.70 (m, 10H), 1.70–1.81 (m, 3H), 1.88–2.15 (m, 2H), 2.78 (dd, <i>J =</i> 17.43, 11.77 Hz, 2H), 2.94 (q, <i>J =</i> 7.41 Hz, 2H), 3.57 (dd, <i>J =</i> 9.42, 5.84 Hz, 1H), 4.37 (br s, 1H), 6.70–6.86 (m, 3H), 6.92 (dd, <i>J =</i> 8.01, 1.04 Hz, 1H), 6.97–7.07 (m, 1H), 7.07–7.22 (m, 1H), 7.45 (s, 1H), 7.59 (d, <i>J =</i> 7.91 Hz, 1H) ppm.</div></div><div id="sec5_5_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> 2-(3-Chlorophenoxy)-4-((<i>R</i>)-1-((<i>S</i>)-3-(5-methyl-2,4-dioxo-3,4-dihydropyrimidin-1(2<i>H</i>)-yl)piperidin-1-yl)-2-(tetrahydro-2H-pyran-4-yl)ethyl)benzoic Acid (<b>50</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 1.60 min, <i>m</i>/<i>z</i> = 568.2 (M + 1), 566.3 (M – 1). HRMS calcd for C<sub>30</sub>H<sub>35</sub>ClN<sub>3</sub>O<sub>6</sub> 568.2209, found 568.2203. <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 1.08–1.33 (m, 3H), 1.41 (d, <i>J =</i> 12.62 Hz, 1H), 1.55 (d, <i>J =</i> 11.11 Hz, 3H), 1.61 (br s, 1H), 1.65–1.83 (m, 6H), 1.83–1.99 (m, 1H), 2.14 (t, <i>J =</i> 10.27 Hz, 1H), 2.76 (d, <i>J =</i> 10.93 Hz, 1H), 2.90 (d, <i>J =</i> 9.98 Hz, 1H), 3.55–3.68 (m, 1H), 3.76 (d, <i>J =</i> 11.87 Hz, 2H), 4.45 (br s, 1H), 6.69–6.81 (m, 2H), 6.86 (s, 1H), 6.89–7.01 (m, 1H), 7.06–7.28 (m, 2 H), 7.44 (s, 1H), 7.79 (d, <i>J =</i> 7.91 Hz, 1H) ppm. Diastereomeric ratio: >99:1.</div></div><div id="sec5_5_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> (2-(3-Bromophenoxy)-4-(<i>R</i>)-(2-(1-adamantane)-1-((<i>S</i>)-3-(5-methyl-2,4-dioxo-3,4-dihydropyrimidin-1(2<i>H</i>)-yl)piperidin-1-yl)ethyl)benzoic Acid (<b>51</b>)</h4><div class="NLM_p last">LC-MS (Method 2) <i>t</i><sub>R</sub> = 2.45 min, <i>m</i>/<i>z</i> = 662, 664 (M + 1, M + 3). HRMS calcd for C<sub>35</sub>H<sub>41</sub>BrN<sub>3</sub>O<sub>5</sub> 662.2224, found 662.2230. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.1–1.8 (m, 25H), 2.62 (d, 1H), 2.78 (m, 1H), 3.69 (m, 2H), 4.26 (m, 1H), 6.63 (s, 1H), 6.72 (m, 2H), 6.90 (d, 1H), 6.97 (d, 1H), 7.18 (t, 2H), 7.39 (d, 1H), 7.52 (s, 1H) ppm. Diastereomeric ratio: 99:1.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i67"><a href="/doi/suppl/10.1021/jm3011806">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88487" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88487" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Enzymatic, MIC, and log <i>D</i> assay protocols as well as protein crystallography methods and statistics tables.  This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>. PDB accession codes: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HEJ">4HEJ</a> (compound (<i>S</i>)-<b>16</b>); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GSY">4GSY</a> (<b>34</b>); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HDC">4HDC</a> (<b>41</b>).</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm3011806/suppl_file/jm3011806_si_001.pdf">jm3011806_si_001.pdf (175.38 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm3011806" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68046" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68046" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gabriel Martínez-Botella</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca Infection
Innovative Medicines, 35 Gatehouse Drive, Waltham, Massachusetts
02451, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#0b6c6a6979626e674b786a6c6e797325686466"><span class="__cf_email__" data-cfemail="03646261716a666f4370626466717b2d606c6e">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">John N. Breen</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca Infection
Innovative Medicines, 35 Gatehouse Drive, Waltham, Massachusetts
02451, United States</span>; 
    <span class="hlFld-Affiliation affiliation">AstraZeneca Discovery
Sciences, 35 Gatehouse Drive, Waltham, Massachusetts
02451, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James E. S. Duffy</span> - <span class="hlFld-Affiliation affiliation">BioFocus, Chesterford Research Park, Saffron Walden
CB10 1XL, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jacques Dumas</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca Infection
Innovative Medicines, 35 Gatehouse Drive, Waltham, Massachusetts
02451, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bolin Geng</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca Infection
Innovative Medicines, 35 Gatehouse Drive, Waltham, Massachusetts
02451, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ian K. Gowers</span> - <span class="hlFld-Affiliation affiliation">BioFocus, Chesterford Research Park, Saffron Walden
CB10 1XL, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Oluyinka M. Green</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca Infection
Innovative Medicines, 35 Gatehouse Drive, Waltham, Massachusetts
02451, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Satenig Guler</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca Infection
Innovative Medicines, 35 Gatehouse Drive, Waltham, Massachusetts
02451, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Martin F. Hentemann</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca Infection
Innovative Medicines, 35 Gatehouse Drive, Waltham, Massachusetts
02451, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Felix A. Hernandez-Juan</span> - <span class="hlFld-Affiliation affiliation">BioFocus, Chesterford Research Park, Saffron Walden
CB10 1XL, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Diane Joseph-McCarthy</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca Infection
Innovative Medicines, 35 Gatehouse Drive, Waltham, Massachusetts
02451, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sameer Kawatkar</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca Infection
Innovative Medicines, 35 Gatehouse Drive, Waltham, Massachusetts
02451, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicholas A. Larsen</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca Discovery
Sciences, 35 Gatehouse Drive, Waltham, Massachusetts
02451, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ovadia Lazari</span> - <span class="hlFld-Affiliation affiliation">BioFocus, Chesterford Research Park, Saffron Walden
CB10 1XL, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James T. Loch</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca Infection
Innovative Medicines, 35 Gatehouse Drive, Waltham, Massachusetts
02451, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jacqueline A. Macritchie</span> - <span class="hlFld-Affiliation affiliation">BioFocus, Chesterford Research Park, Saffron Walden
CB10 1XL, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrew R. McKenzie</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca Infection
Innovative Medicines, 35 Gatehouse Drive, Waltham, Massachusetts
02451, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Joseph V. Newman</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca Infection
Innovative Medicines, 35 Gatehouse Drive, Waltham, Massachusetts
02451, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nelson B. Olivier</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca Discovery
Sciences, 35 Gatehouse Drive, Waltham, Massachusetts
02451, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Linda G. Otterson</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca Infection
Innovative Medicines, 35 Gatehouse Drive, Waltham, Massachusetts
02451, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrew P. Owens</span> - <span class="hlFld-Affiliation affiliation">BioFocus, Chesterford Research Park, Saffron Walden
CB10 1XL, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jon Read</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca Discovery
Sciences, 35 Gatehouse Drive, Waltham, Massachusetts
02451, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David W. Sheppard</span> - <span class="hlFld-Affiliation affiliation">BioFocus, Chesterford Research Park, Saffron Walden
CB10 1XL, U.K.</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas A. Keating</span> - <span class="hlFld-Affiliation affiliation">AstraZeneca Infection
Innovative Medicines, 35 Gatehouse Drive, Waltham, Massachusetts
02451, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d813e4406-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i68">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02840" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02840" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Mahalingam Kannan and the team at Syngene (Bangalore, India) for chemistry support. We thank AstraZeneca Infection colleagues Nancy DeGrace and Natascha Bezdenejnih-Snyder for analytical support, Seth Ribe for synthetic scale-up, Hongming Wang for modeling support, Ann Boriack-Sjodin for structural biology support, Swati Prasad for human TMK IC<sub>50</sub>s, and Selvi Pradeepan for coordinating log <i>D</i> measurements.</p></div><div class="postAsapNotesSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="postAsapNotesSection">Note Added After ASAP Publication</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11430" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11430" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">After this paper was published online October 24, 2012, reference <a onclick="showRef(event, 'cit15c'); return false;" href="javascript:void(0);" class="ref cit15c">15c</a> was added to the reference list. The corrected version was reposted November 7, 2012.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i69" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i69"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i70" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i70"> Abbreviations</h2><tr><td class="NLM_term">TMK</td><td class="NLM_def"><p class="first last">thymidylate kinase</p></td></tr><tr><td class="NLM_term">Spn</td><td class="NLM_def"><p class="first last"><i>Streptococcus pneumoniae</i></p></td></tr><tr><td class="NLM_term">Sau</td><td class="NLM_def"><p class="first last"><i>Staphylococcus aureus</i></p></td></tr><tr><td class="NLM_term">MIC</td><td class="NLM_def"><p class="first last">minimum inhibitory concentration</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i71">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13571" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13571" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 15 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Silver, L. L.</span><span> </span><span class="NLM_article-title">Challenges of Antibacterial Discovery</span> <span class="citation_source-journal">Clin. Microbiol. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="NLM_x">–</span> <span class="NLM_lpage">109</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm3011806&amp;key=10.1128%2FCMR.00030-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm3011806&amp;key=21233508" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm3011806&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjtFSgu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2011&pages=71-109&author=L.+L.+Silver&title=Challenges+of+Antibacterial+Discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Challenges of antibacterial discovery</span></div><div class="casAuthors">Silver, Lynn L.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Microbiology Reviews</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">71-109</span>CODEN:
                <span class="NLM_cas:coden">CMIREX</span>;
        ISSN:<span class="NLM_cas:issn">0893-8512</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A review.  The discovery of novel small-mol. antibacterial drugs has been stalled for many years.  The purpose of this review is to underscore and illustrate those scientific problems unique to the discovery and optimization of novel antibacterial agents that have adversely affected the output of the effort.  The major challenges fall into two areas: (i) proper target selection, particularly the necessity of pursuing mol. targets that are not prone to rapid resistance development, and (ii) improvement of chem. libraries to overcome limitations of diversity, esp. that which is necessary to overcome barriers to bacterial entry and proclivity to be effluxed, esp. in Gram-neg. organisms.  Failure to address these problems has led to a great deal of misdirected effort.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsTDYwtK6vHbVg90H21EOLACvtfcHk0lhRMSA5LYOZoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjtFSgu7s%253D&md5=a77b7a1f2763b73d4bd63daf67f78cc7</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1128%2FCMR.00030-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FCMR.00030-10%26sid%3Dliteratum%253Aachs%26aulast%3DSilver%26aufirst%3DL.%2BL.%26atitle%3DChallenges%2520of%2520Antibacterial%2520Discovery%26jtitle%3DClin.%2520Microbiol.%2520Rev.%26date%3D2011%26volume%3D24%26spage%3D71%26epage%3D109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Payne, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gwynn, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holmes, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pompliano, D.</span><span> </span><span class="NLM_article-title">Drugs for Bad Bugs: Confronting the Challenges of Antibacterial Discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">29</span><span class="NLM_x">–</span> <span class="NLM_lpage">40</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=16&amp;doi=10.1021%2Fjm3011806&amp;key=10.1038%2Fnrd2201" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=8&amp;doi=10.1021%2Fjm3011806&amp;key=17159923" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=32&amp;doi=10.1021%2Fjm3011806&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlGktbfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=29-40&author=D.+J.+Payneauthor=M.+N.+Gwynnauthor=D.+J.+Holmesauthor=D.+Pompliano&title=Drugs+for+Bad+Bugs%3A+Confronting+the+Challenges+of+Antibacterial+Discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2aR"><div class="casContent"><span class="casTitleNuber">2a</span><div class="casTitle"><span class="NLM_cas:atitle">Drugs for bad bugs: confronting the challenges of antibacterial discovery</span></div><div class="casAuthors">Payne, David J.; Gwynn, Michael N.; Holmes, David J.; Pompliano, David L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-40</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The sequencing of the first complete bacterial genome in 1995 heralded a new era of hope for antibacterial drug discoverers, who now had the tools to search entire genomes for new antibacterial targets.  Several companies, including GlaxoSmithKline, moved back into the antibacterials area and embraced a genomics-derived, target-based approach to screen for new classes of drugs with novel modes of action.  Here, we share our experience of evaluating more than 300 genes and 70 high-throughput screening campaigns over a period of 7 years, and look at what we learned and how that has influenced GlaxoSmithKline's antibacterials strategy going forward.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH1tM2BSZbWLVg90H21EOLACvtfcHk0lhruh5kcr4acA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlGktbfM&md5=18100253f9868af578ea621ab98a5c86</span></div><a href="/servlet/linkout?suffix=cit2a&amp;dbid=16384&amp;doi=10.1038%2Fnrd2201&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2201%26sid%3Dliteratum%253Aachs%26aulast%3DPayne%26aufirst%3DD.%2BJ.%26aulast%3DGwynn%26aufirst%3DM.%2BN.%26aulast%3DHolmes%26aufirst%3DD.%2BJ.%26aulast%3DPompliano%26aufirst%3DD.%26atitle%3DDrugs%2520for%2520Bad%2520Bugs%253A%2520Confronting%2520the%2520Challenges%2520of%2520Antibacterial%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D6%26spage%3D29%26epage%3D40" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit2b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Barrett, J. F.</span><span> </span><span class="NLM_article-title">Impact of Genomics-Emerging Targets for Antibacterial Therapy</span> <span class="citation_source-journal">Comprehensive Med. Chem. II</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">731</span><span class="NLM_x">–</span> <span class="NLM_lpage">748</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=32&amp;doi=10.1021%2Fjm3011806&amp;key=1%3ACAS%3A528%3ADC%252BD2sXptFChs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&pages=731-748&author=J.+F.+Barrett&title=Impact+of+Genomics-Emerging+Targets+for+Antibacterial+Therapy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2bR"><div class="casContent"><span class="casTitleNuber">2b</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of genomics-emerging targets for antibacterial therapy</span></div><div class="casAuthors">Barrett, J. F.</div><div class="citationInfo"><span class="NLM_cas:title">Comprehensive Medicinal Chemistry II</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">731-748</span>CODEN:
                <span class="NLM_cas:coden">69JQHZ</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review on the impact of genomics-emerging targets for antibacterial therapy.  The disease state of bacterial infection is discussed, along with genomics targets, genomics as a scientific discipline, genomics tools, exptl. proof of essentiality and quality of targets, genes to screen approach, gene expression levels, and proteomics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaln0105NqjLVg90H21EOLACvtfcHk0lhruh5kcr4acA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXptFChs7Y%253D&md5=05470126c3a5e0901b69d7419fd3feab</span></div><a href="/servlet/linkout?suffix=cit2b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBarrett%26aufirst%3DJ.%2BF.%26atitle%3DImpact%2520of%2520Genomics-Emerging%2520Targets%2520for%2520Antibacterial%2520Therapy%26jtitle%3DComprehensive%2520Med.%2520Chem.%2520II%26date%3D2006%26volume%3D7%26spage%3D731%26epage%3D748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Fischbach, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walsh, C. T.</span><span> </span><span class="NLM_article-title">Antibiotics for Emerging Pathogens</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">325</span><span class="NLM_x">, </span> <span class="NLM_fpage">1089</span><span class="NLM_x">–</span> <span class="NLM_lpage">1093</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm3011806&amp;key=10.1126%2Fscience.1176667" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm3011806&amp;key=19713519" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm3011806&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVersrzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=325&publication_year=2009&pages=1089-1093&author=M.+A.+Fischbachauthor=C.+T.+Walsh&title=Antibiotics+for+Emerging+Pathogens"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Antibiotics for Emerging Pathogens</span></div><div class="casAuthors">Fischbach, Michael A.; Walsh, Christopher T.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">325</span>
        (<span class="NLM_cas:issue">5944</span>),
    <span class="NLM_cas:pages">1089-1093</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Antibiotic-resistant strains of pathogenic bacteria are increasingly prevalent in hospitals and the community.  New antibiotics are needed to combat these bacterial pathogens, but progress in developing them has been slow.  Historically, most antibiotics have come from a small set of mol. scaffolds whose functional lifetimes have been extended by generations of synthetic tailoring.  The emergence of multidrug resistance among the latest generation of pathogens suggests that the discovery of new scaffolds should be a priority.  Promising approaches to scaffold discovery are emerging; they include mining underexplored microbial niches for natural products, designing screens that avoid rediscovering old scaffolds, and repurposing libraries of synthetic mols. for use as antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZw1IG5S90IbVg90H21EOLACvtfcHk0lhruh5kcr4acA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVersrzK&md5=96d902c55376951663e06dcddf0d571d</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1126%2Fscience.1176667&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1176667%26sid%3Dliteratum%253Aachs%26aulast%3DFischbach%26aufirst%3DM.%2BA.%26aulast%3DWalsh%26aufirst%3DC.%2BT.%26atitle%3DAntibiotics%2520for%2520Emerging%2520Pathogens%26jtitle%3DScience%26date%3D2009%26volume%3D325%26spage%3D1089%26epage%3D1093" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Charifson, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grossman, T. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mueller, P.</span><span> </span><span class="NLM_article-title">The Use of Structure-Guided Design to Discover New Anti-Microbial Agents: Focus on Antibacterial Resistance</span> <span class="citation_source-journal">Anti-Infect. Agents Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">73</span><span class="NLM_x">–</span> <span class="NLM_lpage">86</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=73-86&author=P.+S.+Charifsonauthor=T.+H.+Grossmanauthor=P.+Mueller&title=The+Use+of+Structure-Guided+Design+to+Discover+New+Anti-Microbial+Agents%3A+Focus+on+Antibacterial+Resistance"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCharifson%26aufirst%3DP.%2BS.%26aulast%3DGrossman%26aufirst%3DT.%2BH.%26aulast%3DMueller%26aufirst%3DP.%26atitle%3DThe%2520Use%2520of%2520Structure-Guided%2520Design%2520to%2520Discover%2520New%2520Anti-Microbial%2520Agents%253A%2520Focus%2520on%2520Antibacterial%2520Resistance%26jtitle%3DAnti-Infect.%2520Agents%2520Med.%2520Chem.%26date%3D2009%26volume%3D8%26spage%3D73%26epage%3D86" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Schneider, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hawser, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Islam, K.</span><span> </span><span class="NLM_article-title">Iclaprim, a Novel Diaminopyrimidine with Potent Activity on Trimethoprim Sensitive and Resistant Bacteria</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">4217</span><span class="NLM_x">–</span> <span class="NLM_lpage">4221</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=4217-4221&author=P.+Schneiderauthor=S.+Hawserauthor=K.+Islam&title=Iclaprim%2C+a+Novel+Diaminopyrimidine+with+Potent+Activity+on+Trimethoprim+Sensitive+and+Resistant+Bacteria"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DP.%26aulast%3DHawser%26aufirst%3DS.%26aulast%3DIslam%26aufirst%3DK.%26atitle%3DIclaprim%252C%2520a%2520Novel%2520Diaminopyrimidine%2520with%2520Potent%2520Activity%2520on%2520Trimethoprim%2520Sensitive%2520and%2520Resistant%2520Bacteria%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2003%26volume%3D13%26spage%3D4217%26epage%3D4221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Charifson, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anne-Laure Grillot, A.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grossman, T. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parsons, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Badia, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deininger, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drumm, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gross, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LeTiran, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mani, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicolau, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perola, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ronkin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shannon, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swenson, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tessier, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, S.-K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trudeau, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stamos, D.</span><span> </span><span class="NLM_article-title">Novel Dual-Targeting Benzimidazole Urea Inhibitors of DNA Gyrase and Topoisomerase IV Possessing Potent Antibacterial Activity: Intelligent Design and Evolution through the Judicious Use of Structure-Guided Design and Stucture–Activity Relationships</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">5243</span><span class="NLM_x">–</span> <span class="NLM_lpage">5263</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm800318d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=5243-5263&author=P.+S.+Charifsonauthor=A.-L.+Anne-Laure+Grillotauthor=T.+H.+Grossmanauthor=J.+D.+Parsonsauthor=M.+Badiaauthor=S.+Bellonauthor=D.+D.+Deiningerauthor=J.+E.+Drummauthor=C.+H.+Grossauthor=A.+LeTiranauthor=Y.+Liaoauthor=N.+Maniauthor=D.+P.+Nicolauauthor=E.+Perolaauthor=S.+Ronkinauthor=D.+Shannonauthor=L.+L.+Swensonauthor=Q.+Tangauthor=P.+R.+Tessierauthor=S.-K.+Tianauthor=M.+Trudeauauthor=T.+Wangauthor=Y.+Weiauthor=H.+Zhangauthor=D.+Stamos&title=Novel+Dual-Targeting+Benzimidazole+Urea+Inhibitors+of+DNA+Gyrase+and+Topoisomerase+IV+Possessing+Potent+Antibacterial+Activity%3A+Intelligent+Design+and+Evolution+through+the+Judicious+Use+of+Structure-Guided+Design+and+Stucture%E2%80%93Activity+Relationships"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6a&amp;dbid=16384&amp;doi=10.1021%2Fjm800318d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800318d%26sid%3Dliteratum%253Aachs%26aulast%3DCharifson%26aufirst%3DP.%2BS.%26aulast%3DAnne-Laure%2BGrillot%26aufirst%3DA.-L.%26aulast%3DGrossman%26aufirst%3DT.%2BH.%26aulast%3DParsons%26aufirst%3DJ.%2BD.%26aulast%3DBadia%26aufirst%3DM.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DDeininger%26aufirst%3DD.%2BD.%26aulast%3DDrumm%26aufirst%3DJ.%2BE.%26aulast%3DGross%26aufirst%3DC.%2BH.%26aulast%3DLeTiran%26aufirst%3DA.%26aulast%3DLiao%26aufirst%3DY.%26aulast%3DMani%26aufirst%3DN.%26aulast%3DNicolau%26aufirst%3DD.%2BP.%26aulast%3DPerola%26aufirst%3DE.%26aulast%3DRonkin%26aufirst%3DS.%26aulast%3DShannon%26aufirst%3DD.%26aulast%3DSwenson%26aufirst%3DL.%2BL.%26aulast%3DTang%26aufirst%3DQ.%26aulast%3DTessier%26aufirst%3DP.%2BR.%26aulast%3DTian%26aufirst%3DS.-K.%26aulast%3DTrudeau%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DWei%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DStamos%26aufirst%3DD.%26atitle%3DNovel%2520Dual-Targeting%2520Benzimidazole%2520Urea%2520Inhibitors%2520of%2520DNA%2520Gyrase%2520and%2520Topoisomerase%2520IV%2520Possessing%2520Potent%2520Antibacterial%2520Activity%253A%2520Intelligent%2520Design%2520and%2520Evolution%2520through%2520the%2520Judicious%2520Use%2520of%2520Structure-Guided%2520Design%2520and%2520Stucture%25E2%2580%2593Activity%2520Relationships%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D5243%26epage%3D5263" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit6b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Eakin, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hales, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walkup, G. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bist, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mullen, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bryant, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Embrey, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breeze, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Timms, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrews, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uria-Nickelsen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Demeritt, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loch, J. T.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hull, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blodgett, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Illingworth, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prince, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boriack-Sjodin, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hauck, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacPherson, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ni, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherer, B.</span><span> </span><span class="NLM_article-title">Pyrrolamide DNA gyrase inhibitors: fragment-based nuclear magnetic resonance screening to identify antibacterial agents</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">1240</span><span class="NLM_x">–</span> <span class="NLM_lpage">1246</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=16&amp;doi=10.1021%2Fjm3011806&amp;key=10.1128%2FAAC.05485-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=8&amp;doi=10.1021%2Fjm3011806&amp;key=22183167" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=32&amp;doi=10.1021%2Fjm3011806&amp;key=1%3ACAS%3A528%3ADC%252BC38XjsVKls7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2012&pages=1240-1246&author=A.+E.+Eakinauthor=O.+Greenauthor=N.+Halesauthor=G.+K.+Walkupauthor=S.+Bistauthor=A.+Singhauthor=G.+Mullenauthor=J.+Bryantauthor=K.+Embreyauthor=N.+Gaoauthor=A.+Breezeauthor=D.+Timmsauthor=B.+Andrewsauthor=M.+Uria-Nickelsenauthor=J.+Demerittauthor=J.+T.+Lochauthor=K.+Hullauthor=A.+Blodgettauthor=R.+N.+Illingworthauthor=B.+Princeauthor=P.+A.+Boriack-Sjodinauthor=S.+Hauckauthor=L.+J.+MacPhersonauthor=H.+Niauthor=B.+Sherer&title=Pyrrolamide+DNA+gyrase+inhibitors%3A+fragment-based+nuclear+magnetic+resonance+screening+to+identify+antibacterial+agents"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6bR"><div class="casContent"><span class="casTitleNuber">6b</span><div class="casTitle"><span class="NLM_cas:atitle">Pyrrolamide DNA gyrase inhibitors: fragment-based nuclear magnetic resonance screening to identify antibacterial agents</span></div><div class="casAuthors">Eakin, Ann E.; Green, Oluyinka; Hales, Neil; Walkup, Grant K.; Bist, Shanta; Singh, Alok; Mullen, George; Bryant, Joanna; Embrey, Kevin; Gao, Ning; Breeze, Alex; Timms, Dave; Andrews, Beth; Uria-Nickelsen, Maria; Demeritt, Julie; Loch, James T., III; Hull, Ken; Blodgett, April; Illingworth, Ruth N.; Prince, Bryan; Boriack-Sjodin, P. Ann; Hauck, Sheila; MacPherson, Lawrence J.; Ni, Haihong; Sherer, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1240-1246</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">DNA gyrase is an essential enzyme in bacteria, and its inhibition results in the disruption of DNA synthesis and, subsequently, cell death.  The pyrrolamides are a novel class of antibacterial agents targeting DNA gyrase.  These compds. were identified by a fragment-based lead generation (FBLG) approach using NMR (NMR) screening to identify low-mol.-wt. compds. that bind to the ATP pocket of DNA gyrase.  A pyrrole hit with a binding const. of 1 mM formed the basis of the design and synthesis of a focused library of compds. that resulted in the rapid identification of a lead compd. that inhibited DNA gyrase with a 50% inhibitory concn. (IC50) of 3 μM.  The potency of the lead compd. was further optimized by utilizing iterative X-ray crystallog. to yield DNA gyrase inhibitors that also displayed antibacterial activity.  Spontaneous mutants were isolated in Staphylococcus aureus by plating on agar plates contg. pyrrolamide 4 at the MIC.  The resistant variants displayed 4- to 8-fold-increased MIC values relative to the parent strain.  DNA sequencing revealed two independent point mutations in the pyrrolamide binding region of the gyrB genes from these variants, supporting the hypothesis that the mode of action of these compds. was inhibition of DNA gyrase.  Efficacy of a representative pyrrolamide was demonstrated against Streptococcus pneumoniae in a mouse lung infection model.  These data demonstrate that the pyrrolamides are a novel class of DNA gyrase inhibitors with the potential to deliver future antibacterial agents targeting multiple clin. indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgukeglwN38rVg90H21EOLACvtfcHk0lhPP8kk_C8biw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjsVKls7g%253D&md5=81f04321981eef4258f2982e0c71cfbc</span></div><a href="/servlet/linkout?suffix=cit6b&amp;dbid=16384&amp;doi=10.1128%2FAAC.05485-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.05485-11%26sid%3Dliteratum%253Aachs%26aulast%3DEakin%26aufirst%3DA.%2BE.%26aulast%3DGreen%26aufirst%3DO.%26aulast%3DHales%26aufirst%3DN.%26aulast%3DWalkup%26aufirst%3DG.%2BK.%26aulast%3DBist%26aufirst%3DS.%26aulast%3DSingh%26aufirst%3DA.%26aulast%3DMullen%26aufirst%3DG.%26aulast%3DBryant%26aufirst%3DJ.%26aulast%3DEmbrey%26aufirst%3DK.%26aulast%3DGao%26aufirst%3DN.%26aulast%3DBreeze%26aufirst%3DA.%26aulast%3DTimms%26aufirst%3DD.%26aulast%3DAndrews%26aufirst%3DB.%26aulast%3DUria-Nickelsen%26aufirst%3DM.%26aulast%3DDemeritt%26aufirst%3DJ.%26aulast%3DLoch%26aufirst%3DJ.%2BT.%26aulast%3DHull%26aufirst%3DK.%26aulast%3DBlodgett%26aufirst%3DA.%26aulast%3DIllingworth%26aufirst%3DR.%2BN.%26aulast%3DPrince%26aufirst%3DB.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DHauck%26aufirst%3DS.%26aulast%3DMacPherson%26aufirst%3DL.%2BJ.%26aulast%3DNi%26aufirst%3DH.%26aulast%3DSherer%26aufirst%3DB.%26atitle%3DPyrrolamide%2520DNA%2520gyrase%2520inhibitors%253A%2520fragment-based%2520nuclear%2520magnetic%2520resonance%2520screening%2520to%2520identify%2520antibacterial%2520agents%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2012%26volume%3D56%26spage%3D1240%26epage%3D1246" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Keating, T. A.; Newman, J. V.; Olivier, N. B.; Otterson, L. G.; Andrews, B.; Boriack-Sjodin, P. A.; Breen, J. N.; Doig, P.; Dumas, J.; Gangl, E.; Green, O. M.; Guler, S. Y.; Hentemann, M. F.; Joseph-McCarthy, D.; Kawatkar, S.; Kutschke, A.; Loch, J. T.; McKenzie, A. R.; Pradeepan, S.; Prasad, S.; Martínez-Botella, G.</span><span> </span><span class="NLM_article-title"><i>In Vivo</i> Validation of Thymidylate Kinase (TMK) with a Rationally-Designed, Selective Antibacterial Compound</span>.  <span class="citation_source-journal">ACS Chem. Biol.</span> DOI: <span class="refDoi"> DOI: 10.1021/cb300316n</span> . Published online: Aug 21,<span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span>.</span><div class="citationLinks">[<a href="/doi/10.1021/cb300316n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=T.+A.+Keating&author=J.+V.+Newman&author=N.+B.+Olivier&author=L.+G.+Otterson&author=B.+Andrews&author=P.+A.+Boriack-Sjodin&author=J.+N.+Breen&author=P.+Doig&author=J.+Dumas&author=E.+Gangl&author=O.+M.+Green&author=S.+Y.+Guler&author=M.+F.+Hentemann&author=D.+Joseph-McCarthy&author=S.+Kawatkar&author=A.+Kutschke&author=J.+T.+Loch&author=A.+R.+McKenzie&author=S.+Pradeepan&author=S.+Prasad&author=G.+Mart%C3%ADnez-Botella&title=In+Vivo+Validation+of+Thymidylate+Kinase+%28TMK%29+with+a+Rationally-Designed%2C+Selective+Antibacterial+Compound&doi=10.1021%2Fcb300316n"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fcb300316n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300316n%26sid%3Dliteratum%253Aachs%26aulast%3DKeating%26aufirst%3DT.%2BA.%26atitle%3DIn%2520Vivo%2520Validation%2520of%2520Thymidylate%2520Kinase%2520%2528TMK%2529%2520with%2520a%2520Rationally-Designed%252C%2520Selective%2520Antibacterial%2520Compound%26jtitle%3DACS%2520Chem.%2520Biol.%26doi%3D10.1021%2Fcb300316n%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Vanheusden, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Rompaey, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munier-Lehmann, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pochet, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herdewijn, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Calenbergh, S.</span><span> </span><span class="NLM_article-title">Thymidine and thymidine-5′-O-monophosphate analogues as inhibitors of <i>Mycobacterium tuberculosis</i> thymidylate kinase</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">3045</span><span class="NLM_x">–</span> <span class="NLM_lpage">3048</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=3045-3048&author=V.+Vanheusdenauthor=P.+Van+Rompaeyauthor=H.+Munier-Lehmannauthor=S.+Pochetauthor=P.+Herdewijnauthor=S.+Van+Calenbergh&title=Thymidine+and+thymidine-5%E2%80%B2-O-monophosphate+analogues+as+inhibitors+of+Mycobacterium+tuberculosis+thymidylate+kinase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVanheusden%26aufirst%3DV.%26aulast%3DVan%2BRompaey%26aufirst%3DP.%26aulast%3DMunier-Lehmann%26aufirst%3DH.%26aulast%3DPochet%26aufirst%3DS.%26aulast%3DHerdewijn%26aufirst%3DP.%26aulast%3DVan%2BCalenbergh%26aufirst%3DS.%26atitle%3DThymidine%2520and%2520thymidine-5%25E2%2580%25B2-O-monophosphate%2520analogues%2520as%2520inhibitors%2520of%2520Mycobacterium%2520tuberculosis%2520thymidylate%2520kinase%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2003%26volume%3D13%26spage%3D3045%26epage%3D3048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit8b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Choi, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plummer, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Starr, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desbonnet, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soutter, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dillman, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roush, W. R.</span><span> </span><span class="NLM_article-title">Structure Guided Development of Novel Thymidine Mimetics Targeting <i>Pseudomonas aeruginosa</i> Thymidylate Kinase: From Hit to Lead Generation</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">852</span><span class="NLM_x">–</span> <span class="NLM_lpage">870</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm201349f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=852-870&author=J.+Y.+Choiauthor=M.+S.+Plummerauthor=J.+Starrauthor=C.+R.+Desbonnetauthor=H.+Soutterauthor=J.+Changauthor=J.+R.+Millerauthor=K.+Dillmanauthor=A.+A.+Millerauthor=W.+R.+Roush&title=Structure+Guided+Development+of+Novel+Thymidine+Mimetics+Targeting+Pseudomonas+aeruginosa+Thymidylate+Kinase%3A+From+Hit+to+Lead+Generation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8b&amp;dbid=16384&amp;doi=10.1021%2Fjm201349f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201349f%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DJ.%2BY.%26aulast%3DPlummer%26aufirst%3DM.%2BS.%26aulast%3DStarr%26aufirst%3DJ.%26aulast%3DDesbonnet%26aufirst%3DC.%2BR.%26aulast%3DSoutter%26aufirst%3DH.%26aulast%3DChang%26aufirst%3DJ.%26aulast%3DMiller%26aufirst%3DJ.%2BR.%26aulast%3DDillman%26aufirst%3DK.%26aulast%3DMiller%26aufirst%3DA.%2BA.%26aulast%3DRoush%26aufirst%3DW.%2BR.%26atitle%3DStructure%2520Guided%2520Development%2520of%2520Novel%2520Thymidine%2520Mimetics%2520Targeting%2520Pseudomonas%2520aeruginosa%2520Thymidylate%2520Kinase%253A%2520From%2520Hit%2520to%2520Lead%2520Generation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D852%26epage%3D870" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Kotaka, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhaliwal, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nichols, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Angell, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockyer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hawkins, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stammers, D. K.</span><span> </span><span class="NLM_article-title">Structures of <i>S. aureus</i> thymidylate kinase reveal an atypical active site configuration and an intermediate conformational state upon substrate binding</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">774</span><span class="NLM_x">–</span> <span class="NLM_lpage">784</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm3011806&amp;key=10.1110%2Fps.052002406" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm3011806&amp;key=16522804" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm3011806&amp;key=1%3ACAS%3A528%3ADC%252BD28XjsFCqsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2006&pages=774-784&author=M.+Kotakaauthor=B.+Dhaliwalauthor=J.+Renauthor=C.+E.+Nicholsauthor=R.+Angellauthor=M.+Lockyerauthor=A.+R.+Hawkinsauthor=D.+K.+Stammers&title=Structures+of+S.+aureus+thymidylate+kinase+reveal+an+atypical+active+site+configuration+and+an+intermediate+conformational+state+upon+substrate+binding"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Structures of S. aureus thymidylate kinase reveal an atypical active site configuration and an intermediate conformational state upon substrate binding</span></div><div class="casAuthors">Kotaka, Masayo; Dhaliwal, Balvinder; Ren, Jingshan; Nichols, Charles E.; Angell, Richard; Lockyer, Michael; Hawkins, Alastair R.; Stammers, David K.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">774-784</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Methicillin-resistant Staphylococcus aureus (MRSA) poses a major threat to human health, particularly through hospital-acquired infection.  The spread of MRSA means that novel targets are required to develop potential inhibitors to combat infections caused by such drug-resistant bacteria.  Thymidylate kinase (TMK) is attractive as an antibacterial target as it is essential for providing components for DNA synthesis.  Here, we report crystal structures of unliganded and thymidylate-bound forms of S. aureus thymidylate kinase (SaTMK).  His-tagged and untagged SaTMK crystallize with differing lattice packing and show variations in conformational states for unliganded and thymidylate (TMP)-bound forms.  In addn. to open and closed forms of SaTMK, an intermediate conformation in TMP binding is obsd., in which the site is partially closed.  Anal. of these structures indicates a sequence of events upon TMP binding, with helix α3 shifting position initially, followed by movement of α2 to close the substrate site.  In addn., we observe significant conformational differences in the TMP-binding site in SaTMK as compared to available TMK structures from other bacterial species, Escherichia coli and Mycobacterium tuberculosis as well as human TMK.  In SaTMK, Arg 48 is situated at the base of the TMP-binding site, close to the thymine ring, whereas a cis-proline occupies the equiv. position in other TMKs.  The obsd. TMK structural differences mean that design of compds. highly specific for the S. aureus enzyme looks possible; such inhibitors could minimize the transfer of drug resistance between different bacterial species.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGe5Sj8j_9ObVg90H21EOLACvtfcHk0li0ONWjgXr6PQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjsFCqsbc%253D&md5=a0ec45f5ad70b08a5e22f100ee98e443</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1110%2Fps.052002406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.052002406%26sid%3Dliteratum%253Aachs%26aulast%3DKotaka%26aufirst%3DM.%26aulast%3DDhaliwal%26aufirst%3DB.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DNichols%26aufirst%3DC.%2BE.%26aulast%3DAngell%26aufirst%3DR.%26aulast%3DLockyer%26aufirst%3DM.%26aulast%3DHawkins%26aufirst%3DA.%2BR.%26aulast%3DStammers%26aufirst%3DD.%2BK.%26atitle%3DStructures%2520of%2520S.%2520aureus%2520thymidylate%2520kinase%2520reveal%2520an%2520atypical%2520active%2520site%2520configuration%2520and%2520an%2520intermediate%2520conformational%2520state%2520upon%2520substrate%2520binding%26jtitle%3DProtein%2520Sci.%26date%3D2006%26volume%3D15%26spage%3D774%26epage%3D784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Martínez-Botella, G.; Breen, J. N.; Duffy, J. E. S.; Dumas, J.; Geng, B.; Gowers, I. K.; Green, O. M.; Guler, S.; Hentemann, M. F.; Hernandez-Juan, F. A.; Joseph-McCarthy, D.; Kawatkar, S.; Larsen, N. A.; Lazari, O.; Loch, J. T.; Macritchie, J. A.; McKenzie, A. R.; Newman, J. V.; Olivier, N. B.; Otterson, L. G.; Owens, A. P.; Read, J.; Sheppard, D. W.; Keating, T. A.</span> Unpublished results.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Mart%C3%ADnez-Botella%2C+G.%3B+Breen%2C+J.+N.%3B+Duffy%2C+J.+E.+S.%3B+Dumas%2C+J.%3B+Geng%2C+B.%3B+Gowers%2C+I.+K.%3B+Green%2C+O.+M.%3B+Guler%2C+S.%3B+Hentemann%2C+M.+F.%3B+Hernandez-Juan%2C+F.+A.%3B+Joseph-McCarthy%2C+D.%3B+Kawatkar%2C+S.%3B+Larsen%2C+N.+A.%3B+Lazari%2C+O.%3B+Loch%2C+J.+T.%3B+Macritchie%2C+J.+A.%3B+McKenzie%2C+A.+R.%3B+Newman%2C+J.+V.%3B+Olivier%2C+N.+B.%3B+Otterson%2C+L.+G.%3B+Owens%2C+A.+P.%3B+Read%2C+J.%3B+Sheppard%2C+D.+W.%3B+Keating%2C+T.+A.+Unpublished+results."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DMart%25C3%25ADnez-Botella%26aufirst%3DG." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Topliss, J. G.</span><span> </span><span class="NLM_article-title">Utilization of Operational Schemes for Analog Synthesis in Drug Design</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1972</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">1006</span><span class="NLM_x">–</span> <span class="NLM_lpage">1011</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm00280a002" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm3011806&amp;key=1%3ACAS%3A528%3ADyaE3sXhtFChug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1972&pages=1006-1011&author=J.+G.+Topliss&title=Utilization+of+Operational+Schemes+for+Analog+Synthesis+in+Drug+Design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Utilization of operational schemes for analog synthesis in drug design</span></div><div class="casAuthors">Topliss, John G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1972</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1006-11</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">Proposals were presented for stepwise selection for synthesis of new analogs of a biol. active compd. which were designed to maximize the chances of synthesizing the most potent compds. in the series as early as possible.  The schemes were based on a fundamental assumption of the Hansch method that a particular substituent may modify activity relative to the parent compd. by virtue of resulting changes in hydrophobic, electronic, and steric effects.  The effectiveness of the method was illustrated by retrospective application to several existing series of compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-zymvXigwSbVg90H21EOLACvtfcHk0lgIreOD966ijQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3sXhtFChug%253D%253D&md5=acf62a05dbdb3d1d17f91097f66ab8c1</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjm00280a002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00280a002%26sid%3Dliteratum%253Aachs%26aulast%3DTopliss%26aufirst%3DJ.%2BG.%26atitle%3DUtilization%2520of%2520Operational%2520Schemes%2520for%2520Analog%2520Synthesis%2520in%2520Drug%2520Design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1972%26volume%3D15%26spage%3D1006%26epage%3D1011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Fersht, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, J.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knill-Jones, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lowe, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilkinson, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blow, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brick, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waye, M. M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winter, G.</span><span> </span><span class="NLM_article-title">Hydrogen bonding and biological specificity analysed by protein engineering</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1985</span><span class="NLM_x">, </span> <span class="NLM_volume">314</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">238</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Fjm3011806&amp;key=10.1038%2F314235a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Fjm3011806&amp;key=3845322" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm3011806&amp;key=1%3ACAS%3A528%3ADyaL2MXhvVWmsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=314&publication_year=1985&pages=235-238&author=A.+R.+Fershtauthor=J.-P.+Shiauthor=J.+Knill-Jonesauthor=D.+M.+Loweauthor=A.+J.+Wilkinsonauthor=D.+M.+Blowauthor=P.+Brickauthor=P.+Carterauthor=M.+M.+Y.+Wayeauthor=G.+Winter&title=Hydrogen+bonding+and+biological+specificity+analysed+by+protein+engineering"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrogen bonding and biological specificity analyzed by protein engineering</span></div><div class="casAuthors">Fersht, Alan R.; Shi, Jian Ping; Knill-Jones, Jack; Lowe, Denise M.; Wilkinson, Anthony J.; Blow, David M.; Brick, Peter; Carter, Paul; Waye, Mary M. Y.; Winter, Greg</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">314</span>
        (<span class="NLM_cas:issue">6008</span>),
    <span class="NLM_cas:pages">235-8</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">The energetics of H bonding were examd. by using tyrosyl-tRNA synthetase variants produced by site-specific mutagenesis in Bacillus stearothermophilus.  The mutant enzymes varied in their ability to form H bonds.  The contributions of enzyme amino acid side chains to binding of tyrosine (in the transition state) were calcd. from kcat/Km (where kcat is the reaction rate const.).  Deletion of a side chain which forms a H bond with an uncharged group on the substrate decreased binding energy by only 0.5-1.5 kcal/mol, whereas deletion of an uncharged enzyme side chain which formed a H bond with a charged group on the substrate weakened binding by ∼3.5-4.5 kcal/mol.  Deletion of a group which forms an unusually long H bond actually improved binding.  Changes in other side chains (e.g., deletion of Me groups within the protein) caused changes in substrate binding of ≤1 kcal/mol.  The results are discussed with respect to determinants of specificity in biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9Mpcnb5fP9LVg90H21EOLACvtfcHk0lgIreOD966ijQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXhvVWmsrw%253D&md5=667009772868f87a06dbeab388d06012</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2F314235a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F314235a0%26sid%3Dliteratum%253Aachs%26aulast%3DFersht%26aufirst%3DA.%2BR.%26aulast%3DShi%26aufirst%3DJ.-P.%26aulast%3DKnill-Jones%26aufirst%3DJ.%26aulast%3DLowe%26aufirst%3DD.%2BM.%26aulast%3DWilkinson%26aufirst%3DA.%2BJ.%26aulast%3DBlow%26aufirst%3DD.%2BM.%26aulast%3DBrick%26aufirst%3DP.%26aulast%3DCarter%26aufirst%3DP.%26aulast%3DWaye%26aufirst%3DM.%2BM.%2BY.%26aulast%3DWinter%26aufirst%3DG.%26atitle%3DHydrogen%2520bonding%2520and%2520biological%2520specificity%2520analysed%2520by%2520protein%2520engineering%26jtitle%3DNature%26date%3D1985%26volume%3D314%26spage%3D235%26epage%3D238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klicic, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knoll, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelley, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Francis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shenkin, P. S.</span><span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 1. Method and assessment of docking accuracy</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1739</span><span class="NLM_x">–</span> <span class="NLM_lpage">1749</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=32&amp;doi=10.1021%2Fjm3011806&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+1.+Method+and+assessment+of+docking+accuracy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13aR"><div class="casContent"><span class="casTitleNuber">13a</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0lgIreOD966ijQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit13a&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25201.%2520Method%2520and%2520assessment%2520of%2520docking%2520accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit13b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beard, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollard, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1750</span><span class="NLM_x">–</span> <span class="NLM_lpage">1759</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=32&amp;doi=10.1021%2Fjm3011806&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+2.+Enrichment+factors+in+database+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13bR"><div class="casContent"><span class="casTitleNuber">13b</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0lgIreOD966ijQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit13b&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25202.%2520Enrichment%2520factors%2520in%2520database%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Kaminski, G. A</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tirado-Rives, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Evaluation and Reparametrization of the OPLS-AA Force Field for Proteins via Comparison with Accurate Quantum Chemical Calculations on Peptides</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">6474</span><span class="NLM_x">–</span> <span class="NLM_lpage">6487</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2001&pages=6474-6487&author=G.+A+Kaminskiauthor=R.+A.+Friesnerauthor=J.+Tirado-Rivesauthor=W.+L.+Jorgensen&title=Evaluation+and+Reparametrization+of+the+OPLS-AA+Force+Field+for+Proteins+via+Comparison+with+Accurate+Quantum+Chemical+Calculations+on+Peptides"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKaminski%26aufirst%3DG.%2BA%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DEvaluation%2520and%2520Reparametrization%2520of%2520the%2520OPLS-AA%2520Force%2520Field%2520for%2520Proteins%2520via%2520Comparison%2520with%2520Accurate%2520Quantum%2520Chemical%2520Calculations%2520on%2520Peptides%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2001%26volume%3D5%26spage%3D6474%26epage%3D6487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Bieber, P.</span> <span class="citation_source-journal">Comptes Rendus</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1951</span><span class="NLM_x">, </span> <span class="NLM_volume">233</span><span class="NLM_x">, </span> <span class="NLM_fpage">655</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=32&amp;doi=10.1021%2Fjm3011806&amp;key=1%3ACAS%3A528%3ADyaG38XksFajsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Bieber%2C+P.Comptes+Rendus+1951%2C+233%2C+655"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15aR"><div class="casContent"><span class="casTitleNuber">15a</span><div class="casTitle"><span class="NLM_cas:atitle">Action of certain alcoholates on methyl α,β-dihaloisobutyrates</span></div><div class="casAuthors">Bieber, Paul</div><div class="citationInfo"><span class="NLM_cas:title"></span>
        (<span class="NLM_cas:date">1951</span>),
    <span class="NLM_cas:volume">233</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">655-7</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">CH2:CMeCO2Me and Br at 10° give 95% CH2BrCBrMeCO2Me (I), b. 203°, b15 86°, nD20.5 1.505, d420.5 1.805.  The di Cl analog of I b. 170-1°, nD17.5 1.4545, d417.5 1.253.  I and 1 mol. MeONa give CHBr: CMeCO2Me, b. 164°, nD20 1.488, d420 1.4877 (acid chloride, b20 62-4°; amide, m. 109°; anilide, m. 102-3°).  I and excess MeONa give an equil. mixt. of (MeO)2CHCHMeCO2Me, isolated as the amide, m. 90°, and MeOCH: CMeCO2Me (II), b. 175-6°, nD18 1.460, d418 1.070 (acid, m. 105.5-6°).  II and Cl give MeOCHCl-CClMeCO2Me (III), b15 88-91°, nD16 1.4545; II and 2,4-(O2N)2C6H3NHNH2 give 2,4-(O2N)2C6H3NHN:CHCHMeCO2Me, m. 99.5-100°.  The Et ester of the β-EtO analog of III b18 100-3°, nD22 1.445, d422 1.1645.  I and Me3CONa give Me3COCH:CMeCO2Me (IV), b14 86-7°, nD16 1.4555, d416 0.983, sapond. to the acid, m. 117-17.5°.  IV and NaHSO4 give Me2C:CH2 and OHCCHMeCO2Me, which is sapond. and decompd. to EtCHO, MeOH, and CO2.  I and PhONa give only PhOCH: CMeCO2Me (V), b4 121°, nD19 1.540, d419 1.111, and some CH2:CMeCO2Me.  V is sapond. to the acid, m. 109° (acid chloride, b14 142-3°; amide, m. 100°; anilide, m. 152-3°), which with Cu and quinoline gives PhOCH: CHMe, b15 71°, nD19 1.525, d419 0.9713.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoc5N-IkNv-erVg90H21EOLACvtfcHk0ljcDjwqh0GxXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaG38XksFajsw%253D%253D&md5=ede619ae3bcf298fb58ad8d6cf090180</span></div><a href="/servlet/linkout?suffix=cit15a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBieber%26aufirst%3DP.%26jtitle%3DComptes%2520Rendus%26date%3D1951%26volume%3D233%26spage%3D655" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit15b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Csuk, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">von Scholz, Y.</span><span> </span><span class="NLM_article-title">Synthesis of racemic carbocyclic cyclopropanoid nucleoside analogues</span> <span class="citation_source-journal">Tetrahedron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x"> (</span><span class="NLM_issue">26</span><span class="NLM_x">) </span> <span class="NLM_fpage">7193</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=1995&pages=7193&issue=26&author=R.+Csukauthor=Y.+von+Scholz&title=Synthesis+of+racemic+carbocyclic+cyclopropanoid+nucleoside+analogues"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCsuk%26aufirst%3DR.%26aulast%3Dvon%2BScholz%26aufirst%3DY.%26atitle%3DSynthesis%2520of%2520racemic%2520carbocyclic%2520cyclopropanoid%2520nucleoside%2520analogues%26jtitle%3DTetrahedron%26date%3D1995%26volume%3D51%26issue%3D26%26spage%3D7193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit15c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Kovačková, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dračínský, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rejman, D.</span><span> </span><span class="NLM_article-title">The synthesis of piperidine nucleoside analogs  - a comparison of several methods to access the introduction of nucleobases</span> <span class="citation_source-journal">Tetrahedron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">1485</span><span class="NLM_x">–</span> <span class="NLM_lpage">1500</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=1485-1500&author=S.+Kova%C4%8Dkov%C3%A1author=M.+Dra%C4%8D%C3%ADnsk%C3%BDauthor=D.+Rejman&title=The+synthesis+of+piperidine+nucleoside+analogs++-+a+comparison+of+several+methods+to+access+the+introduction+of+nucleobases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15c&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKova%25C4%258Dkov%25C3%25A1%26aufirst%3DS.%26aulast%3DDra%25C4%258D%25C3%25ADnsk%25C3%25BD%26aufirst%3DM.%26aulast%3DRejman%26aufirst%3DD.%26atitle%3DThe%2520synthesis%2520of%2520piperidine%2520nucleoside%2520analogs%2520%2520-%2520a%2520comparison%2520of%2520several%2520methods%2520to%2520access%2520the%2520introduction%2520of%2520nucleobases%26jtitle%3DTetrahedron%26date%3D2011%26volume%3D67%26spage%3D1485%26epage%3D1500" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2a','cit2b'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6a','cit6b'],'ref7':['cit7'],'ref8':['cit8a','cit8b'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13a','cit13b'],'ref14':['cit14'],'ref15':['cit15a','cit15b','cit15c']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 27 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Zhouling Xie, Xiaoxiao Yang, Yajun Duan, Jihong Han, <span class="NLM_string-name hlFld-ContribAuthor">Chenzhong Liao</span>. </span><span class="cited-content_cbyCitation_article-title">Small-Molecule Kinase Inhibitors for the Treatment of Nononcologic Diseases. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (3)
                                     , 1283-1345. <a href="https://doi.org/10.1021/acs.jmedchem.0c01511" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01511</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01511&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01511%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSmall-Molecule%252BKinase%252BInhibitors%252Bfor%252Bthe%252BTreatment%252Bof%252BNononcologic%252BDiseases%26aulast%3DXie%26aufirst%3DZhouling%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D29082020%26date%3D22012021%26volume%3D64%26issue%3D3%26spage%3D1283%26epage%3D1345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mengshen David Chen, Ian J. Fucci, Kaustubh Sinha, <span class="NLM_string-name hlFld-ContribAuthor">Gordon S. Rule</span>. </span><span class="cited-content_cbyCitation_article-title">dGMP Binding to Thymidylate Kinase from Plasmodium falciparum Shows Half-Site Binding and Induces Protein Dynamics at the Dimer Interface. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2020,</strong> <em>59 </em>
                                    (5)
                                     , 694-703. <a href="https://doi.org/10.1021/acs.biochem.9b00898" title="DOI URL">https://doi.org/10.1021/acs.biochem.9b00898</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.9b00898&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.9b00898%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DdGMP%252BBinding%252Bto%252BThymidylate%252BKinase%252Bfrom%252BPlasmodium%252Bfalciparum%252BShows%252BHalf-Site%252BBinding%252Band%252BInduces%252BProtein%252BDynamics%252Bat%252Bthe%252BDimer%252BInterface%26aulast%3DChen%26aufirst%3DMengshen%2BDavid%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D04102019%26date%3D29122019%26date%3D14012020%26volume%3D59%26issue%3D5%26spage%3D694%26epage%3D703" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mark W. Campbell, Jordan S. Compton, Christopher B. Kelly, <span class="NLM_string-name hlFld-ContribAuthor">Gary A. Molander</span>. </span><span class="cited-content_cbyCitation_article-title">Three-Component Olefin Dicarbofunctionalization Enabled by Nickel/Photoredox Dual Catalysis. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2019,</strong> <em>141 </em>
                                    (51)
                                     , 20069-20078. <a href="https://doi.org/10.1021/jacs.9b08282" title="DOI URL">https://doi.org/10.1021/jacs.9b08282</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.9b08282&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.9b08282%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DThree-Component%252BOlefin%252BDicarbofunctionalization%252BEnabled%252Bby%252BNickel%25252FPhotoredox%252BDual%252BCatalysis%26aulast%3DCampbell%26aufirst%3DMark%2BW.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D07082019%26date%3D13122019%26volume%3D141%26issue%3D51%26spage%3D20069%26epage%3D20078" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Mengshen
“David” Chen, Kaustubh Sinha, Gordon S. Rule, <span class="NLM_string-name hlFld-ContribAuthor">Danith H. Ly</span>. </span><span class="cited-content_cbyCitation_article-title">Interaction of α-Thymidine Inhibitors with Thymidylate Kinase from Plasmodium falciparum. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2018,</strong> <em>57 </em>
                                    (19)
                                     , 2868-2875. <a href="https://doi.org/10.1021/acs.biochem.8b00162" title="DOI URL">https://doi.org/10.1021/acs.biochem.8b00162</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.8b00162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.8b00162%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DInteraction%252Bof%252B%2525CE%2525B1-Thymidine%252BInhibitors%252Bwith%252BThymidylate%252BKinase%252Bfrom%252BPlasmodium%252Bfalciparum%26aulast%3DChen%26aufirst%3DMengshen%2B%25E2%2580%259CDavid%25E2%2580%259D%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D09022018%26date%3D13042018%26date%3D02052018%26date%3D23042018%26volume%3D57%26issue%3D19%26spage%3D2868%26epage%3D2875" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Lijun Song, Romain Merceron, Begoña Gracia, Ainhoa Lucía Quintana, Martijn D. P. Risseeuw, Fabian Hulpia, Paul Cos, José A. Aínsa, Hélène Munier-Lehmann, Savvas N. Savvides, <span class="NLM_string-name hlFld-ContribAuthor">Serge Van Calenbergh</span>. </span><span class="cited-content_cbyCitation_article-title">Structure Guided Lead Generation toward Nonchiral M. tuberculosis Thymidylate Kinase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (7)
                                     , 2753-2775. <a href="https://doi.org/10.1021/acs.jmedchem.7b01570" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01570</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01570&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b01570%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure%252BGuided%252BLead%252BGeneration%252Btoward%252BNonchiral%252BM.%252Btuberculosis%252BThymidylate%252BKinase%252BInhibitors%26aulast%3DSong%26aufirst%3DLijun%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D24102017%26date%3D15032018%26date%3D06032018%26volume%3D61%26issue%3D7%26spage%3D2753%26epage%3D2775" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kaustubh  Sinha</span> and <span class="hlFld-ContribAuthor ">Gordon S.  Rule</span>  . </span><span class="cited-content_cbyCitation_article-title">The Structure of Thymidylate Kinase from Candida albicans Reveals a Unique Structural Element. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2017,</strong> <em>56 </em>
                                    (33)
                                     , 4360-4370. <a href="https://doi.org/10.1021/acs.biochem.7b00498" title="DOI URL">https://doi.org/10.1021/acs.biochem.7b00498</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.7b00498&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.7b00498%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DThe%252BStructure%252Bof%252BThymidylate%252BKinase%252Bfrom%252BCandida%252Balbicans%252BReveals%252Ba%252BUnique%252BStructural%252BElement%26aulast%3DSinha%26aufirst%3DKaustubh%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D23052017%26date%3D24072017%26date%3D09082017%26date%3D22082017%26date%3D25072017%26volume%3D56%26issue%3D33%26spage%3D4360%26epage%3D4370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maruti  Naik</span>, <span class="hlFld-ContribAuthor ">Anandkumar  Raichurkar</span>, <span class="hlFld-ContribAuthor ">Balachandra S.  Bandodkar</span>, <span class="hlFld-ContribAuthor ">Begur V.  Varun</span>, <span class="hlFld-ContribAuthor ">Shantika  Bhat</span>, <span class="hlFld-ContribAuthor ">Rajesh  Kalkhambkar</span>, <span class="hlFld-ContribAuthor ">Kannan  Murugan</span>, <span class="hlFld-ContribAuthor ">Rani  Menon</span>, <span class="hlFld-ContribAuthor ">Jyothi  Bhat</span>, <span class="hlFld-ContribAuthor ">Beena  Paul</span>, <span class="hlFld-ContribAuthor ">Harini  Iyer</span>, <span class="hlFld-ContribAuthor ">Syeed  Hussein</span>, <span class="hlFld-ContribAuthor ">Julie A.  Tucker</span>, <span class="hlFld-ContribAuthor ">Martin  Vogtherr</span>, <span class="hlFld-ContribAuthor ">Kevin J.  Embrey</span>, <span class="hlFld-ContribAuthor ">Helen  McMiken</span>, <span class="hlFld-ContribAuthor ">Swati  Prasad</span>, <span class="hlFld-ContribAuthor ">Adrian  Gill</span>, <span class="hlFld-ContribAuthor ">Bheemarao G.  Ugarkar</span>, <span class="hlFld-ContribAuthor ">Janani  Venkatraman</span>, <span class="hlFld-ContribAuthor ">Jon  Read</span>, and <span class="hlFld-ContribAuthor ">Manoranjan  Panda</span>  . </span><span class="cited-content_cbyCitation_article-title">Structure Guided Lead Generation for M. tuberculosis Thymidylate Kinase (Mtb TMK): Discovery of 3-Cyanopyridone and 1,6-Naphthyridin-2-one as Potent Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2015,</strong> <em>58 </em>
                                    (2)
                                     , 753-766. <a href="https://doi.org/10.1021/jm5012947" title="DOI URL">https://doi.org/10.1021/jm5012947</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm5012947&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm5012947%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure%252BGuided%252BLead%252BGeneration%252Bfor%252BM.%252Btuberculosis%252BThymidylate%252BKinase%252B%252528Mtb%252BTMK%252529%25253A%252BDiscovery%252Bof%252B3-Cyanopyridone%252Band%252B1%25252C6-Naphthyridin-2-one%252Bas%252BPotent%252BInhibitors%26aulast%3DNaik%26aufirst%3DMaruti%26date%3D2015%26date%3D2014%26date%3D2014%26date%3D25082014%26date%3D23122014%26date%3D22012015%26date%3D08122014%26volume%3D58%26issue%3D2%26spage%3D753%26epage%3D766" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sameer P.  Kawatkar</span>, <span class="hlFld-ContribAuthor ">Thomas A.  Keating</span>, <span class="hlFld-ContribAuthor ">Nelson B.  Olivier</span>, <span class="hlFld-ContribAuthor ">John N.  Breen</span>, <span class="hlFld-ContribAuthor ">Oluyinka M.  Green</span>, <span class="hlFld-ContribAuthor ">Satenig Y.  Guler</span>, <span class="hlFld-ContribAuthor ">Martin F.  Hentemann</span>, <span class="hlFld-ContribAuthor ">James T.  Loch</span>, <span class="hlFld-ContribAuthor ">Andrew R.  McKenzie</span>, <span class="hlFld-ContribAuthor ">Joseph V.  Newman</span>, <span class="hlFld-ContribAuthor ">Linda G.  Otterson</span>, and <span class="hlFld-ContribAuthor ">Gabriel  Martínez-Botella</span>  . </span><span class="cited-content_cbyCitation_article-title">Antibacterial Inhibitors of Gram-Positive Thymidylate Kinase: Structure–Activity Relationships and Chiral Preference of a New Hydrophobic Binding Region. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2014,</strong> <em>57 </em>
                                    (11)
                                     , 4584-4597. <a href="https://doi.org/10.1021/jm500463c" title="DOI URL">https://doi.org/10.1021/jm500463c</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm500463c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm500463c%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAntibacterial%252BInhibitors%252Bof%252BGram-Positive%252BThymidylate%252BKinase%25253A%252BStructure%2525E2%252580%252593Activity%252BRelationships%252Band%252BChiral%252BPreference%252Bof%252Ba%252BNew%252BHydrophobic%252BBinding%252BRegion%26aulast%3DKawatkar%26aufirst%3DSameer%2BP.%26date%3D2014%26date%3D2014%26date%3D2013%26date%3D19122013%26date%3D03062014%26date%3D12062014%26date%3D14052014%26volume%3D57%26issue%3D11%26spage%3D4584%26epage%3D4597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kaustubh  Sinha</span>, <span class="hlFld-ContribAuthor ">Gordon S.  Rule</span>. </span><span class="cited-content_cbyCitation_article-title">Conformational diversity defines substrate specificity of thymidylate/uridylate kinase from
              Candida albicans. </span><span class="cited-content_cbyCitation_journal-name">Proteins: Structure, Function, and Bioinformatics</span><span> <strong>2021,</strong> <em>89 </em>
                                    (8)
                                     , 937-944. <a href="https://doi.org/10.1002/prot.26071" title="DOI URL">https://doi.org/10.1002/prot.26071</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/prot.26071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fprot.26071%26sid%3Dliteratum%253Aachs%26jtitle%3DProteins%253A%2520Structure%252C%2520Function%252C%2520and%2520Bioinformatics%26atitle%3DConformational%252Bdiversity%252Bdefines%252Bsubstrate%252Bspecificity%252Bof%252Bthymidylate%25252Furidylate%252Bkinase%252Bfrom%252BCandida%252Balbicans%26aulast%3DSinha%26aufirst%3DKaustubh%26date%3D2021%26date%3D2021%26volume%3D89%26issue%3D8%26spage%3D937%26epage%3D944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jing  Gao</span>, <span class="hlFld-ContribAuthor ">Yunhui  Tang</span>, <span class="hlFld-ContribAuthor ">Xinyi  Sun</span>, <span class="hlFld-ContribAuthor ">Qiujing  Chen</span>, <span class="hlFld-ContribAuthor ">Yiqian  Peng</span>, <span class="hlFld-ContribAuthor ">Catherine Jia-Yun  Tsai</span>, <span class="hlFld-ContribAuthor ">Qi  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Downregulation of Ribosomal Contents and Kinase Activities Is Associated with the Inhibitive Effect on the Growth of Group B Streptococcus Induced by Placental Extracellular Vesicles. </span><span class="cited-content_cbyCitation_journal-name">Biology</span><span> <strong>2021,</strong> <em>10 </em>
                                    (7)
                                     , 664. <a href="https://doi.org/10.3390/biology10070664" title="DOI URL">https://doi.org/10.3390/biology10070664</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/biology10070664&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fbiology10070664%26sid%3Dliteratum%253Aachs%26jtitle%3DBiology%26atitle%3DDownregulation%252Bof%252BRibosomal%252BContents%252Band%252BKinase%252BActivities%252BIs%252BAssociated%252Bwith%252Bthe%252BInhibitive%252BEffect%252Bon%252Bthe%252BGrowth%252Bof%252BGroup%252BB%252BStreptococcus%252BInduced%252Bby%252BPlacental%252BExtracellular%252BVesicles%26aulast%3DGao%26aufirst%3DJing%26date%3D2021%26date%3D2021%26volume%3D10%26issue%3D7%26spage%3D664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">F  Cools</span>, <span class="hlFld-ContribAuthor ">P  Delputte</span>, <span class="hlFld-ContribAuthor ">P  Cos</span>. </span><span class="cited-content_cbyCitation_article-title">The search for novel treatment strategies for
              Streptococcus pneumoniae
              infections. </span><span class="cited-content_cbyCitation_journal-name">FEMS Microbiology Reviews</span><span> <strong>2021,</strong> <em>70 </em><a href="https://doi.org/10.1093/femsre/fuaa072" title="DOI URL">https://doi.org/10.1093/femsre/fuaa072</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/femsre/fuaa072&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Ffemsre%2Ffuaa072%26sid%3Dliteratum%253Aachs%26jtitle%3DFEMS%2520Microbiology%2520Reviews%26atitle%3DThe%252Bsearch%252Bfor%252Bnovel%252Btreatment%252Bstrategies%252Bfor%252BStreptococcus%252Bpneumoniae%252Binfections%26aulast%3DCools%26aufirst%3DF%26date%3D2021%26date%3D2021%26volume%3D70" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yanlin  Jian</span>, <span class="hlFld-ContribAuthor ">Fabian  Hulpia</span>, <span class="hlFld-ContribAuthor ">Martijn  D. P. Risseeuw</span>, <span class="hlFld-ContribAuthor ">He Eun  Forbes</span>, <span class="hlFld-ContribAuthor ">Guy  Caljon</span>, <span class="hlFld-ContribAuthor ">Hélène  Munier-Lehmann</span>, <span class="hlFld-ContribAuthor ">Helena  I. M. Boshoff</span>, <span class="hlFld-ContribAuthor ">Serge  Van Calenbergh</span>. </span><span class="cited-content_cbyCitation_article-title">1-(1-Arylethylpiperidin-4-yl)thymine Analogs as Antimycobacterial TMPK Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2020,</strong> <em>25 </em>
                                    (12)
                                     , 2805. <a href="https://doi.org/10.3390/molecules25122805" title="DOI URL">https://doi.org/10.3390/molecules25122805</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules25122805&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules25122805%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3D1-%2525281-Arylethylpiperidin-4-yl%252529thymine%252BAnalogs%252Bas%252BAntimycobacterial%252BTMPK%252BInhibitors%26aulast%3DJian%26aufirst%3DYanlin%26date%3D2020%26date%3D2020%26volume%3D25%26issue%3D12%26spage%3D2805" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Katharigatta N  Venugopala</span>, <span class="hlFld-ContribAuthor ">Christophe  Tratrat</span>, <span class="hlFld-ContribAuthor ">Melendhran  Pillay</span>, <span class="hlFld-ContribAuthor ">Sandeep  Chandrashekharappa</span>, <span class="hlFld-ContribAuthor ">Omar Husham Ahmed  Al-Attraqchi</span>, <span class="hlFld-ContribAuthor ">Bandar E  Aldhubiab</span>, <span class="hlFld-ContribAuthor ">Mahesh  Attimarad</span>, <span class="hlFld-ContribAuthor ">Osama I  Alwassil</span>, <span class="hlFld-ContribAuthor ">Anroop B  Nair</span>, <span class="hlFld-ContribAuthor ">Nagaraja  Sreeharsha</span>, <span class="hlFld-ContribAuthor ">Rashmi  Venugopala</span>, <span class="hlFld-ContribAuthor ">Mohamed A  Morsy</span>, <span class="hlFld-ContribAuthor ">Michelyne  Haroun</span>, <span class="hlFld-ContribAuthor ">Hezekiel M  Kumalo</span>, <span class="hlFld-ContribAuthor ">Bharti  Odhav</span>, <span class="hlFld-ContribAuthor ">Koleka  Mlisana</span>. </span><span class="cited-content_cbyCitation_article-title"><p>In silico Design and Synthesis of Tetrahydropyrimidinones and Tetrahydropyrimidinethiones as Potential Thymidylate Kinase Inhibitors Exerting Anti-TB Activity Against <em>Mycobacterium tuberculosis</em></p>. </span><span class="cited-content_cbyCitation_journal-name">Drug Design, Development and Therapy</span><span> <strong>2020,</strong> <em>Volume 14 </em>, 1027-1039. <a href="https://doi.org/10.2147/DDDT.S228381" title="DOI URL">https://doi.org/10.2147/DDDT.S228381</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2147/DDDT.S228381&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2147%2FDDDT.S228381%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Design%252C%2520Development%2520and%2520Therapy%26atitle%3D%25253Cp%25253EIn%252Bsilico%252BDesign%252Band%252BSynthesis%252Bof%252BTetrahydropyrimidinones%252Band%252BTetrahydropyrimidinethiones%252Bas%252BPotential%252BThymidylate%252BKinase%252BInhibitors%252BExerting%252BAnti-TB%252BActivity%252BAgainst%252B%25253Cem%25253EMycobacterium%252Btuberculosis%25253C%25252Fem%25253E%25253C%25252Fp%25253E%26aulast%3DVenugopala%26aufirst%3DKatharigatta%2BN%26date%3D2020%26date%3D2020%26volume%3DVolume%252014%26spage%3D1027%26epage%3D1039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mahmoud  Kandeel</span>, <span class="hlFld-ContribAuthor ">Yukio  Kitade</span>, <span class="hlFld-ContribAuthor ">Abdulla  Al-Taher</span>, <span class="hlFld-ContribAuthor ">Mohammed  Al-Nazawi</span>, . </span><span class="cited-content_cbyCitation_article-title">The structural basis of unique substrate recognition by Plasmodium thymidylate kinase: Molecular dynamics simulation and inhibitory studies. </span><span class="cited-content_cbyCitation_journal-name">PLOS ONE</span><span> <strong>2019,</strong> <em>14 </em>
                                    (2)
                                     , e0212065. <a href="https://doi.org/10.1371/journal.pone.0212065" title="DOI URL">https://doi.org/10.1371/journal.pone.0212065</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0212065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0212065%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520ONE%26atitle%3DThe%252Bstructural%252Bbasis%252Bof%252Bunique%252Bsubstrate%252Brecognition%252Bby%252BPlasmodium%252Bthymidylate%252Bkinase%25253A%252BMolecular%252Bdynamics%252Bsimulation%252Band%252Binhibitory%252Bstudies%26aulast%3DKandeel%26aufirst%3DMahmoud%26date%3D2019%26date%3D2019%26volume%3D14%26issue%3D2%26spage%3De0212065" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yanlin  Jian</span>, <span class="hlFld-ContribAuthor ">Martijn D. P.  Risseeuw</span>, <span class="hlFld-ContribAuthor ">Mathy  Froeyen</span>, <span class="hlFld-ContribAuthor ">Lijun  Song</span>, <span class="hlFld-ContribAuthor ">Davie  Cappoen</span>, <span class="hlFld-ContribAuthor ">Paul  Cos</span>, <span class="hlFld-ContribAuthor ">Hélène  Munier-Lehmann</span>, <span class="hlFld-ContribAuthor ">Serge  van Calenbergh</span>. </span><span class="cited-content_cbyCitation_article-title">1-(Piperidin-3-yl)thymine amides as inhibitors of
              M. tuberculosis
              thymidylate kinase. </span><span class="cited-content_cbyCitation_journal-name">Journal of Enzyme Inhibition and Medicinal Chemistry</span><span> <strong>2019,</strong> <em>34 </em>
                                    (1)
                                     , 1730-1739. <a href="https://doi.org/10.1080/14756366.2019.1662790" title="DOI URL">https://doi.org/10.1080/14756366.2019.1662790</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/14756366.2019.1662790&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F14756366.2019.1662790%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Enzyme%2520Inhibition%2520and%2520Medicinal%2520Chemistry%26atitle%3D1-%252528Piperidin-3-yl%252529thymine%252Bamides%252Bas%252Binhibitors%252Bof%252BM.%252Btuberculosis%252Bthymidylate%252Bkinase%26aulast%3DJian%26aufirst%3DYanlin%26date%3D2019%26date%3D2019%26volume%3D34%26issue%3D1%26spage%3D1730%26epage%3D1739" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sherine E.  Thomas</span>, <span class="hlFld-ContribAuthor ">Vitor  Mendes</span>, <span class="hlFld-ContribAuthor ">So Yeon  Kim</span>, <span class="hlFld-ContribAuthor ">Sony  Malhotra</span>, <span class="hlFld-ContribAuthor ">Bernardo  Ochoa-Montaño</span>, <span class="hlFld-ContribAuthor ">Michal  Blaszczyk</span>, <span class="hlFld-ContribAuthor ">Tom L.  Blundell</span>. </span><span class="cited-content_cbyCitation_article-title">Structural Biology and the Design of New Therapeutics: From HIV and Cancer to Mycobacterial Infections. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Biology</span><span> <strong>2017,</strong> <em>429 </em>
                                    (17)
                                     , 2677-2693. <a href="https://doi.org/10.1016/j.jmb.2017.06.014" title="DOI URL">https://doi.org/10.1016/j.jmb.2017.06.014</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmb.2017.06.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmb.2017.06.014%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Biology%26atitle%3DStructural%252BBiology%252Band%252Bthe%252BDesign%252Bof%252BNew%252BTherapeutics%25253A%252BFrom%252BHIV%252Band%252BCancer%252Bto%252BMycobacterial%252BInfections%26aulast%3DThomas%26aufirst%3DSherine%2BE.%26date%3D2017%26volume%3D429%26issue%3D17%26spage%3D2677%26epage%3D2693" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kinga  Nyíri</span>, <span class="hlFld-ContribAuthor ">Beáta G.  Vértessy</span>. </span><span class="cited-content_cbyCitation_article-title">Perturbation of genome integrity to fight pathogenic microorganisms. </span><span class="cited-content_cbyCitation_journal-name">Biochimica et Biophysica Acta (BBA) - General Subjects</span><span> <strong>2017,</strong> <em>1861 </em>
                                    (1)
                                     , 3593-3612. <a href="https://doi.org/10.1016/j.bbagen.2016.05.024" title="DOI URL">https://doi.org/10.1016/j.bbagen.2016.05.024</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbagen.2016.05.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbagen.2016.05.024%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochimica%2520et%2520Biophysica%2520Acta%2520%2528BBA%2529%2520-%2520General%2520Subjects%26atitle%3DPerturbation%252Bof%252Bgenome%252Bintegrity%252Bto%252Bfight%252Bpathogenic%252Bmicroorganisms%26aulast%3DNy%25C3%25ADri%26aufirst%3DKinga%26date%3D2017%26volume%3D1861%26issue%3D1%26spage%3D3593%26epage%3D3612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lijun  Song</span>, <span class="hlFld-ContribAuthor ">Martijn D.P.  Risseeuw</span>, <span class="hlFld-ContribAuthor ">Matheus  Froeyen</span>, <span class="hlFld-ContribAuthor ">Izet  Karalic</span>, <span class="hlFld-ContribAuthor ">Jan  Goeman</span>, <span class="hlFld-ContribAuthor ">Davie  Cappoen</span>, <span class="hlFld-ContribAuthor ">Johan  Van der Eycken</span>, <span class="hlFld-ContribAuthor ">Paul  Cos</span>, <span class="hlFld-ContribAuthor ">Hélène  Munier-Lehmann</span>, <span class="hlFld-ContribAuthor ">Serge  Van Calenbergh</span>. </span><span class="cited-content_cbyCitation_article-title">Elaboration of a proprietary thymidylate kinase inhibitor motif towards anti-tuberculosis agents. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2016,</strong> <em>24 </em>
                                    (21)
                                     , 5172-5182. <a href="https://doi.org/10.1016/j.bmc.2016.08.041" title="DOI URL">https://doi.org/10.1016/j.bmc.2016.08.041</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2016.08.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2016.08.041%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DElaboration%252Bof%252Ba%252Bproprietary%252Bthymidylate%252Bkinase%252Binhibitor%252Bmotif%252Btowards%252Banti-tuberculosis%252Bagents%26aulast%3DSong%26aufirst%3DLijun%26date%3D2016%26volume%3D24%26issue%3D21%26spage%3D5172%26epage%3D5182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">. </span><span class="cited-content_cbyCitation_article-title">References. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2015,</strong>,, 421-454. <a href="https://doi.org/10.1128/9781555819316.refs" title="DOI URL">https://doi.org/10.1128/9781555819316.refs</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1128/9781555819316.refs&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1128%2F9781555819316.refs%26sid%3Dliteratum%253Aachs%26atitle%3DReferences%26date%3D2015%26date%3D2015%26spage%3D421%26epage%3D454%26pub%3DASM%2520Press%26atitle%3DAntibiotics%26aulast%3DWalsh%26aufirst%3DChristopher%26date%3D2016%26volume%3D5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Denis  Kazakiewicz</span>, <span class="hlFld-ContribAuthor ">Jonathan R.  Karr</span>, <span class="hlFld-ContribAuthor ">Karol M.  Langner</span>, <span class="hlFld-ContribAuthor ">Dariusz  Plewczynski</span>. </span><span class="cited-content_cbyCitation_article-title">A combined systems and structural modeling approach repositions antibiotics for Mycoplasma genitalium. </span><span class="cited-content_cbyCitation_journal-name">Computational Biology and Chemistry</span><span> <strong>2015,</strong> <em>59 </em>, 91-97. <a href="https://doi.org/10.1016/j.compbiolchem.2015.07.007" title="DOI URL">https://doi.org/10.1016/j.compbiolchem.2015.07.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.compbiolchem.2015.07.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.compbiolchem.2015.07.007%26sid%3Dliteratum%253Aachs%26jtitle%3DComputational%2520Biology%2520and%2520Chemistry%26atitle%3DA%252Bcombined%252Bsystems%252Band%252Bstructural%252Bmodeling%252Bapproach%252Brepositions%252Bantibiotics%252Bfor%252BMycoplasma%252Bgenitalium%26aulast%3DKazakiewicz%26aufirst%3DDenis%26date%3D2015%26volume%3D59%26spage%3D91%26epage%3D97" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhi-Hua  Guo</span>, <span class="hlFld-ContribAuthor ">Yong  Yin</span>, <span class="hlFld-ContribAuthor ">Cong  Wang</span>, <span class="hlFld-ContribAuthor ">Peng-Fei  Wang</span>, <span class="hlFld-ContribAuthor ">Xing-Tao  Zhang</span>, <span class="hlFld-ContribAuthor ">Zhong-Chang  Wang</span>, <span class="hlFld-ContribAuthor ">Hai-Liang  Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and molecular docking of salicylic acid derivatives containing metronidazole as a new class of antimicrobial agents. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2015,</strong> <em>23 </em>
                                    (18)
                                     , 6148-6156. <a href="https://doi.org/10.1016/j.bmc.2015.07.075" title="DOI URL">https://doi.org/10.1016/j.bmc.2015.07.075</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2015.07.075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2015.07.075%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bmolecular%252Bdocking%252Bof%252Bsalicylic%252Bacid%252Bderivatives%252Bcontaining%252Bmetronidazole%252Bas%252Ba%252Bnew%252Bclass%252Bof%252Bantimicrobial%252Bagents%26aulast%3DGuo%26aufirst%3DZhi-Hua%26date%3D2015%26volume%3D23%26issue%3D18%26spage%3D6148%26epage%3D6156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gregory S.  Basarab</span>, <span class="hlFld-ContribAuthor ">Ann E.  Eakin</span>, <span class="hlFld-ContribAuthor ">Wright W.  Nichols</span>. </span><span class="cited-content_cbyCitation_article-title">Design of Antibacterial Agents. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2015,</strong>,, 611-626. <a href="https://doi.org/10.1016/B978-0-12-397169-2.00034-2" title="DOI URL">https://doi.org/10.1016/B978-0-12-397169-2.00034-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-397169-2.00034-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-397169-2.00034-2%26sid%3Dliteratum%253Aachs%26atitle%3DDesign%252Bof%252BAntibacterial%252BAgents%26aulast%3DBasarab%26aufirst%3DGregory%2BS.%26date%3D2015%26spage%3D611%26epage%3D626%26pub%3DElsevier%26atitle%3DMolecular%252BMedical%252BMicrobiology%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jun  Fan</span>, <span class="hlFld-ContribAuthor ">Boudewijn L. M.  de Jonge</span>, <span class="hlFld-ContribAuthor ">Kathy  MacCormack</span>, <span class="hlFld-ContribAuthor ">Shubha  Sriram</span>, <span class="hlFld-ContribAuthor ">Robert E.  McLaughlin</span>, <span class="hlFld-ContribAuthor ">Helen  Plant</span>, <span class="hlFld-ContribAuthor ">Marian  Preston</span>, <span class="hlFld-ContribAuthor ">Paul R.  Fleming</span>, <span class="hlFld-ContribAuthor ">Robert  Albert</span>, <span class="hlFld-ContribAuthor ">Melinda  Foulk</span>, <span class="hlFld-ContribAuthor ">Scott D.  Mills</span>. </span><span class="cited-content_cbyCitation_article-title">A Novel High-Throughput Cell-Based Assay Aimed at Identifying Inhibitors of DNA Metabolism in Bacteria. </span><span class="cited-content_cbyCitation_journal-name">Antimicrobial Agents and Chemotherapy</span><span> <strong>2014,</strong> <em>58 </em>
                                    (12)
                                     , 7264-7272. <a href="https://doi.org/10.1128/AAC.03475-14" title="DOI URL">https://doi.org/10.1128/AAC.03475-14</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1128/AAC.03475-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1128%2FAAC.03475-14%26sid%3Dliteratum%253Aachs%26jtitle%3DAntimicrobial%2520Agents%2520and%2520Chemotherapy%26atitle%3DA%252BNovel%252BHigh-Throughput%252BCell-Based%252BAssay%252BAimed%252Bat%252BIdentifying%252BInhibitors%252Bof%252BDNA%252BMetabolism%252Bin%252BBacteria%26aulast%3DFan%26aufirst%3DJun%26date%3D2014%26date%3D2014%26volume%3D58%26issue%3D12%26spage%3D7264%26epage%3D7272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pawan Kumar  Doharey</span>, <span class="hlFld-ContribAuthor ">Manish Kumar  Suthar</span>, <span class="hlFld-ContribAuthor ">Anita  Verma</span>, <span class="hlFld-ContribAuthor ">Vikash  Kumar</span>, <span class="hlFld-ContribAuthor ">Sunita  Yadav</span>, <span class="hlFld-ContribAuthor ">Vishal M.  Balaramnavar</span>, <span class="hlFld-ContribAuthor ">Sushma  Rathaur</span>, <span class="hlFld-ContribAuthor ">Anil Kumar  Saxena</span>, <span class="hlFld-ContribAuthor ">Mohammad Imran  Siddiqi</span>, <span class="hlFld-ContribAuthor ">Jitendra Kumar  Saxena</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular cloning and characterization of Brugia malayi thymidylate kinase. </span><span class="cited-content_cbyCitation_journal-name">Acta Tropica</span><span> <strong>2014,</strong> <em>133 </em>, 83-92. <a href="https://doi.org/10.1016/j.actatropica.2014.02.003" title="DOI URL">https://doi.org/10.1016/j.actatropica.2014.02.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.actatropica.2014.02.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.actatropica.2014.02.003%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Tropica%26atitle%3DMolecular%252Bcloning%252Band%252Bcharacterization%252Bof%252BBrugia%252Bmalayi%252Bthymidylate%252Bkinase%26aulast%3DDoharey%26aufirst%3DPawan%2BKumar%26date%3D2014%26volume%3D133%26spage%3D83%26epage%3D92" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christopher T  Walsh</span>, <span class="hlFld-ContribAuthor ">Timothy A  Wencewicz</span>. </span><span class="cited-content_cbyCitation_article-title">Prospects for new antibiotics: a molecule-centered perspective. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Antibiotics</span><span> <strong>2014,</strong> <em>67 </em>
                                    (1)
                                     , 7-22. <a href="https://doi.org/10.1038/ja.2013.49" title="DOI URL">https://doi.org/10.1038/ja.2013.49</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/ja.2013.49&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fja.2013.49%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Antibiotics%26atitle%3DProspects%252Bfor%252Bnew%252Bantibiotics%25253A%252Ba%252Bmolecule-centered%252Bperspective%26aulast%3DWalsh%26aufirst%3DChristopher%2BT%26date%3D2014%26date%3D2013%26volume%3D67%26issue%3D1%26spage%3D7%26epage%3D22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gabriel  Martínez-Botella</span>, <span class="hlFld-ContribAuthor ">James T.  Loch</span>, <span class="hlFld-ContribAuthor ">Oluyinka M.  Green</span>, <span class="hlFld-ContribAuthor ">Sameer P.  Kawatkar</span>, <span class="hlFld-ContribAuthor ">Nelson B.  Olivier</span>, <span class="hlFld-ContribAuthor ">P. Ann  Boriack-Sjodin</span>, <span class="hlFld-ContribAuthor ">Thomas A.  Keating</span>. </span><span class="cited-content_cbyCitation_article-title">Sulfonylpiperidines as novel, antibacterial inhibitors of Gram-positive thymidylate kinase (TMK). </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2013,</strong> <em>23 </em>
                                    (1)
                                     , 169-173. <a href="https://doi.org/10.1016/j.bmcl.2012.10.128" title="DOI URL">https://doi.org/10.1016/j.bmcl.2012.10.128</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2012.10.128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2012.10.128%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DSulfonylpiperidines%252Bas%252Bnovel%25252C%252Bantibacterial%252Binhibitors%252Bof%252BGram-positive%252Bthymidylate%252Bkinase%252B%252528TMK%252529%26aulast%3DMart%25C3%25ADnez-Botella%26aufirst%3DGabriel%26date%3D2013%26volume%3D23%26issue%3D1%26spage%3D169%26epage%3D173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ren  Sun</span>, <span class="hlFld-ContribAuthor ">Liya  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibition of Mycoplasma pneumoniae growth by FDA-approved anticancer and antiviral nucleoside and nucleobase analogs. </span><span class="cited-content_cbyCitation_journal-name">BMC Microbiology</span><span> <strong>2013,</strong> <em>13 </em>
                                    (1)
                                     , 184. <a href="https://doi.org/10.1186/1471-2180-13-184" title="DOI URL">https://doi.org/10.1186/1471-2180-13-184</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/1471-2180-13-184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2F1471-2180-13-184%26sid%3Dliteratum%253Aachs%26jtitle%3DBMC%2520Microbiology%26atitle%3DInhibition%252Bof%252BMycoplasma%252Bpneumoniae%252Bgrowth%252Bby%252BFDA-approved%252Banticancer%252Band%252Bantiviral%252Bnucleoside%252Band%252Bnucleobase%252Banalogs%26aulast%3DSun%26aufirst%3DRen%26date%3D2013%26volume%3D13%26issue%3D1%26spage%3D184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/medium/jm-2012-011806_0018.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011806&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/medium/jm-2012-011806_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of TMK Inhibitors in Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011806&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) (i) oxalyl chloride, AgOCN, 95 °C, 3 h; (ii) H<sub>2</sub>SO<sub>4</sub>, MeOH, 95 °C, 16 h, 40% overall; (b) 3-phenoxybenzaldehyde, trimethylammonium cyanoborohydride, 22 °C, 16 h, 19%.</p></p></figure><figure data-id="sch2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/medium/jm-2012-011806_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of TMK Inhibitors in Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011806&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) DMF, AcOH, polymer-BCN(H)<sub>3</sub>, 22 °C, 16 h, 28%; (b) 3-chlorophenol, K<sub>2</sub>CO<sub>3</sub>, NMP, 150 °C, 15 min, 45%; (c) H<sub>2</sub>SO<sub>4</sub>, 150 °C, 30 min, 9%.</p></p></figure><figure data-id="sch3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/medium/jm-2012-011806_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of TMK Inhibitors in Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011806&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Pd(PPh<sub>3</sub>)<sub>4</sub>, THF; 22 °C, 16 h, 85%; (b) (i) NaBH<sub>4</sub>, MeOH, 22 °C, 2 h, 29%; (ii) MsCl, TEA, THF, 0 °C, 4 h; (c) (S)-<b>1</b>, <b>11b</b>, DIEA, CH<sub>3</sub>CN, 70 °C, 20 h, 13%; (d) (i) 3-bromophenol, K<sub>2</sub>CO<sub>3</sub>, NMP, 150 °C, 30 min, 67%; (ii) NaOH, H<sub>2</sub>O, EtOH, 100 °C, 16 h, 77%; (e) purification/separation by chiral chromatography, 15%.</p></p></figure><figure data-id="fig1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/medium/jm-2012-011806_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. The crystal structure of the <i>Staphylococcus aureus</i> thymidylate kinase (TMK) with bound dTMP (left)<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> was the starting point for design of lead inhibitor scaffolds (right). dTMP is depicted in orange, with the terminal phosphate located at the right-hand side of the panel. This phosphate points in the direction of the substrate channel leading to the ATP binding site (not shown).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011806&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/medium/jm-2012-011806_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Structure of thymidylate kinase (TMK) racemic lead inhibitor <b>15</b>. The rings are lettered A–D for discussion in the text.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011806&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/medium/jm-2012-011806_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Crystal structure of (<i>S</i>)-<b>16</b> bound to <i>S. aureus</i> TMK at 2.0 Å. The green mesh represents the 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density map contoured to 1 σ. Racemic <b>16</b> was soaked with protein crystals, but only the single enantiomer was found to be bound. Compared to the binding mode of dTMP (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>), the position and contacts of thymine ring A are extremely well-conserved. However, piperidinyl ring B, instead of executing a turn toward the substrate channel (off right-hand side of panel) instead bends the molecule to the left, into a newly formed, induced-fit binding pocket for ring D. This pocket is not evident in the Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a> structure. Major changes in protein structure to create this cryptic pocket include rotation of the Arg48 side chain, movement of the α2 helix (leftmost in figure), and movement of the connecting loop at the top left of the figure. Phe66 forms a π-stack with rings A and D.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011806&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/medium/jm-2012-011806_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Structural characterization of the binding of <b>34</b> in S. <i>aureus</i> TMK. The addition of a carboxylate group to ring C was designed to specifically interact with the side chain of conserved Arg48. This carboxylate drove significant improvements in potency and physical properties. (A) Computational model of the carboxylate-Arg48 interaction predicted a single hydrogen bond. (B) Crystal structure, resolved to 1.7 Å. The carboxylic acid interaction with Arg48 is revealed to be bidentate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011806&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/medium/jm-2012-011806_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Co-crystal structure of a single diastereomer (<i>R</i>,<i>S</i>)-<b>41</b> and <i>S. aureus</i> TMK at 2.0 Å, depicting optimized inhibitor binding to the enzyme (IC<sub>50</sub> = 1.2 nM). Key features are retention of thymine binding interactions (ring A), hydrophobic packing of the isobutyl group against conserved hydrophobic residues Val51/Leu52 (linker between rings B and C), the bidentate salt bridge of the ring C carboxylate to Arg48, and the cryptic hydrophobic pocket hosting the chlorophenyl ring D. The π-stack of rings A and D with Phe66 is maintained.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011806&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/medium/jm-2012-011806_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. pMIC vs log <i>D</i> relationship for piperidinylthymine analogs in Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>. <i>S. aureus</i> appears to be more sensitive to log <i>D</i> variation than <i>S. pneumoniae</i>, but both reach an MIC optimum with log D between 1.5 and 1.8.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2012/jmcmar.2012.55.issue-22/jm3011806/production/images/large/jm-2012-011806_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm3011806&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i71">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21377" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21377" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 15 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Silver, L. L.</span><span> </span><span class="NLM_article-title">Challenges of Antibacterial Discovery</span> <span class="citation_source-journal">Clin. Microbiol. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">71</span><span class="NLM_x">–</span> <span class="NLM_lpage">109</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm3011806&amp;key=10.1128%2FCMR.00030-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm3011806&amp;key=21233508" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm3011806&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjtFSgu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2011&pages=71-109&author=L.+L.+Silver&title=Challenges+of+Antibacterial+Discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Challenges of antibacterial discovery</span></div><div class="casAuthors">Silver, Lynn L.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Microbiology Reviews</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">71-109</span>CODEN:
                <span class="NLM_cas:coden">CMIREX</span>;
        ISSN:<span class="NLM_cas:issn">0893-8512</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A review.  The discovery of novel small-mol. antibacterial drugs has been stalled for many years.  The purpose of this review is to underscore and illustrate those scientific problems unique to the discovery and optimization of novel antibacterial agents that have adversely affected the output of the effort.  The major challenges fall into two areas: (i) proper target selection, particularly the necessity of pursuing mol. targets that are not prone to rapid resistance development, and (ii) improvement of chem. libraries to overcome limitations of diversity, esp. that which is necessary to overcome barriers to bacterial entry and proclivity to be effluxed, esp. in Gram-neg. organisms.  Failure to address these problems has led to a great deal of misdirected effort.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsTDYwtK6vHbVg90H21EOLACvtfcHk0lj9Akpy-Ue_fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjtFSgu7s%253D&md5=a77b7a1f2763b73d4bd63daf67f78cc7</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1128%2FCMR.00030-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FCMR.00030-10%26sid%3Dliteratum%253Aachs%26aulast%3DSilver%26aufirst%3DL.%2BL.%26atitle%3DChallenges%2520of%2520Antibacterial%2520Discovery%26jtitle%3DClin.%2520Microbiol.%2520Rev.%26date%3D2011%26volume%3D24%26spage%3D71%26epage%3D109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Payne, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gwynn, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holmes, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pompliano, D.</span><span> </span><span class="NLM_article-title">Drugs for Bad Bugs: Confronting the Challenges of Antibacterial Discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">29</span><span class="NLM_x">–</span> <span class="NLM_lpage">40</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=16&amp;doi=10.1021%2Fjm3011806&amp;key=10.1038%2Fnrd2201" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=8&amp;doi=10.1021%2Fjm3011806&amp;key=17159923" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=32&amp;doi=10.1021%2Fjm3011806&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlGktbfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=29-40&author=D.+J.+Payneauthor=M.+N.+Gwynnauthor=D.+J.+Holmesauthor=D.+Pompliano&title=Drugs+for+Bad+Bugs%3A+Confronting+the+Challenges+of+Antibacterial+Discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2aR"><div class="casContent"><span class="casTitleNuber">2a</span><div class="casTitle"><span class="NLM_cas:atitle">Drugs for bad bugs: confronting the challenges of antibacterial discovery</span></div><div class="casAuthors">Payne, David J.; Gwynn, Michael N.; Holmes, David J.; Pompliano, David L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-40</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The sequencing of the first complete bacterial genome in 1995 heralded a new era of hope for antibacterial drug discoverers, who now had the tools to search entire genomes for new antibacterial targets.  Several companies, including GlaxoSmithKline, moved back into the antibacterials area and embraced a genomics-derived, target-based approach to screen for new classes of drugs with novel modes of action.  Here, we share our experience of evaluating more than 300 genes and 70 high-throughput screening campaigns over a period of 7 years, and look at what we learned and how that has influenced GlaxoSmithKline's antibacterials strategy going forward.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqH1tM2BSZbWLVg90H21EOLACvtfcHk0lhpS21AVyQR4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlGktbfM&md5=18100253f9868af578ea621ab98a5c86</span></div><a href="/servlet/linkout?suffix=cit2a&amp;dbid=16384&amp;doi=10.1038%2Fnrd2201&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2201%26sid%3Dliteratum%253Aachs%26aulast%3DPayne%26aufirst%3DD.%2BJ.%26aulast%3DGwynn%26aufirst%3DM.%2BN.%26aulast%3DHolmes%26aufirst%3DD.%2BJ.%26aulast%3DPompliano%26aufirst%3DD.%26atitle%3DDrugs%2520for%2520Bad%2520Bugs%253A%2520Confronting%2520the%2520Challenges%2520of%2520Antibacterial%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D6%26spage%3D29%26epage%3D40" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit2b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Barrett, J. F.</span><span> </span><span class="NLM_article-title">Impact of Genomics-Emerging Targets for Antibacterial Therapy</span> <span class="citation_source-journal">Comprehensive Med. Chem. II</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">731</span><span class="NLM_x">–</span> <span class="NLM_lpage">748</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=32&amp;doi=10.1021%2Fjm3011806&amp;key=1%3ACAS%3A528%3ADC%252BD2sXptFChs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&pages=731-748&author=J.+F.+Barrett&title=Impact+of+Genomics-Emerging+Targets+for+Antibacterial+Therapy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2bR"><div class="casContent"><span class="casTitleNuber">2b</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of genomics-emerging targets for antibacterial therapy</span></div><div class="casAuthors">Barrett, J. F.</div><div class="citationInfo"><span class="NLM_cas:title">Comprehensive Medicinal Chemistry II</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">731-748</span>CODEN:
                <span class="NLM_cas:coden">69JQHZ</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review on the impact of genomics-emerging targets for antibacterial therapy.  The disease state of bacterial infection is discussed, along with genomics targets, genomics as a scientific discipline, genomics tools, exptl. proof of essentiality and quality of targets, genes to screen approach, gene expression levels, and proteomics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaln0105NqjLVg90H21EOLACvtfcHk0lhpS21AVyQR4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXptFChs7Y%253D&md5=05470126c3a5e0901b69d7419fd3feab</span></div><a href="/servlet/linkout?suffix=cit2b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBarrett%26aufirst%3DJ.%2BF.%26atitle%3DImpact%2520of%2520Genomics-Emerging%2520Targets%2520for%2520Antibacterial%2520Therapy%26jtitle%3DComprehensive%2520Med.%2520Chem.%2520II%26date%3D2006%26volume%3D7%26spage%3D731%26epage%3D748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Fischbach, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walsh, C. T.</span><span> </span><span class="NLM_article-title">Antibiotics for Emerging Pathogens</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">325</span><span class="NLM_x">, </span> <span class="NLM_fpage">1089</span><span class="NLM_x">–</span> <span class="NLM_lpage">1093</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm3011806&amp;key=10.1126%2Fscience.1176667" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm3011806&amp;key=19713519" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm3011806&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVersrzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=325&publication_year=2009&pages=1089-1093&author=M.+A.+Fischbachauthor=C.+T.+Walsh&title=Antibiotics+for+Emerging+Pathogens"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Antibiotics for Emerging Pathogens</span></div><div class="casAuthors">Fischbach, Michael A.; Walsh, Christopher T.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">325</span>
        (<span class="NLM_cas:issue">5944</span>),
    <span class="NLM_cas:pages">1089-1093</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Antibiotic-resistant strains of pathogenic bacteria are increasingly prevalent in hospitals and the community.  New antibiotics are needed to combat these bacterial pathogens, but progress in developing them has been slow.  Historically, most antibiotics have come from a small set of mol. scaffolds whose functional lifetimes have been extended by generations of synthetic tailoring.  The emergence of multidrug resistance among the latest generation of pathogens suggests that the discovery of new scaffolds should be a priority.  Promising approaches to scaffold discovery are emerging; they include mining underexplored microbial niches for natural products, designing screens that avoid rediscovering old scaffolds, and repurposing libraries of synthetic mols. for use as antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZw1IG5S90IbVg90H21EOLACvtfcHk0lhpS21AVyQR4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVersrzK&md5=96d902c55376951663e06dcddf0d571d</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1126%2Fscience.1176667&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1176667%26sid%3Dliteratum%253Aachs%26aulast%3DFischbach%26aufirst%3DM.%2BA.%26aulast%3DWalsh%26aufirst%3DC.%2BT.%26atitle%3DAntibiotics%2520for%2520Emerging%2520Pathogens%26jtitle%3DScience%26date%3D2009%26volume%3D325%26spage%3D1089%26epage%3D1093" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Charifson, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grossman, T. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mueller, P.</span><span> </span><span class="NLM_article-title">The Use of Structure-Guided Design to Discover New Anti-Microbial Agents: Focus on Antibacterial Resistance</span> <span class="citation_source-journal">Anti-Infect. Agents Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">73</span><span class="NLM_x">–</span> <span class="NLM_lpage">86</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=73-86&author=P.+S.+Charifsonauthor=T.+H.+Grossmanauthor=P.+Mueller&title=The+Use+of+Structure-Guided+Design+to+Discover+New+Anti-Microbial+Agents%3A+Focus+on+Antibacterial+Resistance"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCharifson%26aufirst%3DP.%2BS.%26aulast%3DGrossman%26aufirst%3DT.%2BH.%26aulast%3DMueller%26aufirst%3DP.%26atitle%3DThe%2520Use%2520of%2520Structure-Guided%2520Design%2520to%2520Discover%2520New%2520Anti-Microbial%2520Agents%253A%2520Focus%2520on%2520Antibacterial%2520Resistance%26jtitle%3DAnti-Infect.%2520Agents%2520Med.%2520Chem.%26date%3D2009%26volume%3D8%26spage%3D73%26epage%3D86" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Schneider, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hawser, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Islam, K.</span><span> </span><span class="NLM_article-title">Iclaprim, a Novel Diaminopyrimidine with Potent Activity on Trimethoprim Sensitive and Resistant Bacteria</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">4217</span><span class="NLM_x">–</span> <span class="NLM_lpage">4221</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=4217-4221&author=P.+Schneiderauthor=S.+Hawserauthor=K.+Islam&title=Iclaprim%2C+a+Novel+Diaminopyrimidine+with+Potent+Activity+on+Trimethoprim+Sensitive+and+Resistant+Bacteria"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DP.%26aulast%3DHawser%26aufirst%3DS.%26aulast%3DIslam%26aufirst%3DK.%26atitle%3DIclaprim%252C%2520a%2520Novel%2520Diaminopyrimidine%2520with%2520Potent%2520Activity%2520on%2520Trimethoprim%2520Sensitive%2520and%2520Resistant%2520Bacteria%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2003%26volume%3D13%26spage%3D4217%26epage%3D4221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Charifson, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anne-Laure Grillot, A.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grossman, T. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parsons, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Badia, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deininger, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drumm, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gross, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LeTiran, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mani, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicolau, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perola, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ronkin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shannon, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swenson, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tessier, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, S.-K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trudeau, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stamos, D.</span><span> </span><span class="NLM_article-title">Novel Dual-Targeting Benzimidazole Urea Inhibitors of DNA Gyrase and Topoisomerase IV Possessing Potent Antibacterial Activity: Intelligent Design and Evolution through the Judicious Use of Structure-Guided Design and Stucture–Activity Relationships</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">5243</span><span class="NLM_x">–</span> <span class="NLM_lpage">5263</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm800318d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=5243-5263&author=P.+S.+Charifsonauthor=A.-L.+Anne-Laure+Grillotauthor=T.+H.+Grossmanauthor=J.+D.+Parsonsauthor=M.+Badiaauthor=S.+Bellonauthor=D.+D.+Deiningerauthor=J.+E.+Drummauthor=C.+H.+Grossauthor=A.+LeTiranauthor=Y.+Liaoauthor=N.+Maniauthor=D.+P.+Nicolauauthor=E.+Perolaauthor=S.+Ronkinauthor=D.+Shannonauthor=L.+L.+Swensonauthor=Q.+Tangauthor=P.+R.+Tessierauthor=S.-K.+Tianauthor=M.+Trudeauauthor=T.+Wangauthor=Y.+Weiauthor=H.+Zhangauthor=D.+Stamos&title=Novel+Dual-Targeting+Benzimidazole+Urea+Inhibitors+of+DNA+Gyrase+and+Topoisomerase+IV+Possessing+Potent+Antibacterial+Activity%3A+Intelligent+Design+and+Evolution+through+the+Judicious+Use+of+Structure-Guided+Design+and+Stucture%E2%80%93Activity+Relationships"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6a&amp;dbid=16384&amp;doi=10.1021%2Fjm800318d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800318d%26sid%3Dliteratum%253Aachs%26aulast%3DCharifson%26aufirst%3DP.%2BS.%26aulast%3DAnne-Laure%2BGrillot%26aufirst%3DA.-L.%26aulast%3DGrossman%26aufirst%3DT.%2BH.%26aulast%3DParsons%26aufirst%3DJ.%2BD.%26aulast%3DBadia%26aufirst%3DM.%26aulast%3DBellon%26aufirst%3DS.%26aulast%3DDeininger%26aufirst%3DD.%2BD.%26aulast%3DDrumm%26aufirst%3DJ.%2BE.%26aulast%3DGross%26aufirst%3DC.%2BH.%26aulast%3DLeTiran%26aufirst%3DA.%26aulast%3DLiao%26aufirst%3DY.%26aulast%3DMani%26aufirst%3DN.%26aulast%3DNicolau%26aufirst%3DD.%2BP.%26aulast%3DPerola%26aufirst%3DE.%26aulast%3DRonkin%26aufirst%3DS.%26aulast%3DShannon%26aufirst%3DD.%26aulast%3DSwenson%26aufirst%3DL.%2BL.%26aulast%3DTang%26aufirst%3DQ.%26aulast%3DTessier%26aufirst%3DP.%2BR.%26aulast%3DTian%26aufirst%3DS.-K.%26aulast%3DTrudeau%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DWei%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DStamos%26aufirst%3DD.%26atitle%3DNovel%2520Dual-Targeting%2520Benzimidazole%2520Urea%2520Inhibitors%2520of%2520DNA%2520Gyrase%2520and%2520Topoisomerase%2520IV%2520Possessing%2520Potent%2520Antibacterial%2520Activity%253A%2520Intelligent%2520Design%2520and%2520Evolution%2520through%2520the%2520Judicious%2520Use%2520of%2520Structure-Guided%2520Design%2520and%2520Stucture%25E2%2580%2593Activity%2520Relationships%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D5243%26epage%3D5263" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit6b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Eakin, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hales, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walkup, G. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bist, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mullen, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bryant, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Embrey, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breeze, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Timms, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrews, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uria-Nickelsen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Demeritt, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loch, J. T.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hull, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blodgett, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Illingworth, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prince, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boriack-Sjodin, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hauck, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacPherson, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ni, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sherer, B.</span><span> </span><span class="NLM_article-title">Pyrrolamide DNA gyrase inhibitors: fragment-based nuclear magnetic resonance screening to identify antibacterial agents</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">1240</span><span class="NLM_x">–</span> <span class="NLM_lpage">1246</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=16&amp;doi=10.1021%2Fjm3011806&amp;key=10.1128%2FAAC.05485-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=8&amp;doi=10.1021%2Fjm3011806&amp;key=22183167" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=32&amp;doi=10.1021%2Fjm3011806&amp;key=1%3ACAS%3A528%3ADC%252BC38XjsVKls7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2012&pages=1240-1246&author=A.+E.+Eakinauthor=O.+Greenauthor=N.+Halesauthor=G.+K.+Walkupauthor=S.+Bistauthor=A.+Singhauthor=G.+Mullenauthor=J.+Bryantauthor=K.+Embreyauthor=N.+Gaoauthor=A.+Breezeauthor=D.+Timmsauthor=B.+Andrewsauthor=M.+Uria-Nickelsenauthor=J.+Demerittauthor=J.+T.+Lochauthor=K.+Hullauthor=A.+Blodgettauthor=R.+N.+Illingworthauthor=B.+Princeauthor=P.+A.+Boriack-Sjodinauthor=S.+Hauckauthor=L.+J.+MacPhersonauthor=H.+Niauthor=B.+Sherer&title=Pyrrolamide+DNA+gyrase+inhibitors%3A+fragment-based+nuclear+magnetic+resonance+screening+to+identify+antibacterial+agents"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6bR"><div class="casContent"><span class="casTitleNuber">6b</span><div class="casTitle"><span class="NLM_cas:atitle">Pyrrolamide DNA gyrase inhibitors: fragment-based nuclear magnetic resonance screening to identify antibacterial agents</span></div><div class="casAuthors">Eakin, Ann E.; Green, Oluyinka; Hales, Neil; Walkup, Grant K.; Bist, Shanta; Singh, Alok; Mullen, George; Bryant, Joanna; Embrey, Kevin; Gao, Ning; Breeze, Alex; Timms, Dave; Andrews, Beth; Uria-Nickelsen, Maria; Demeritt, Julie; Loch, James T., III; Hull, Ken; Blodgett, April; Illingworth, Ruth N.; Prince, Bryan; Boriack-Sjodin, P. Ann; Hauck, Sheila; MacPherson, Lawrence J.; Ni, Haihong; Sherer, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1240-1246</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">DNA gyrase is an essential enzyme in bacteria, and its inhibition results in the disruption of DNA synthesis and, subsequently, cell death.  The pyrrolamides are a novel class of antibacterial agents targeting DNA gyrase.  These compds. were identified by a fragment-based lead generation (FBLG) approach using NMR (NMR) screening to identify low-mol.-wt. compds. that bind to the ATP pocket of DNA gyrase.  A pyrrole hit with a binding const. of 1 mM formed the basis of the design and synthesis of a focused library of compds. that resulted in the rapid identification of a lead compd. that inhibited DNA gyrase with a 50% inhibitory concn. (IC50) of 3 μM.  The potency of the lead compd. was further optimized by utilizing iterative X-ray crystallog. to yield DNA gyrase inhibitors that also displayed antibacterial activity.  Spontaneous mutants were isolated in Staphylococcus aureus by plating on agar plates contg. pyrrolamide 4 at the MIC.  The resistant variants displayed 4- to 8-fold-increased MIC values relative to the parent strain.  DNA sequencing revealed two independent point mutations in the pyrrolamide binding region of the gyrB genes from these variants, supporting the hypothesis that the mode of action of these compds. was inhibition of DNA gyrase.  Efficacy of a representative pyrrolamide was demonstrated against Streptococcus pneumoniae in a mouse lung infection model.  These data demonstrate that the pyrrolamides are a novel class of DNA gyrase inhibitors with the potential to deliver future antibacterial agents targeting multiple clin. indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgukeglwN38rVg90H21EOLACvtfcHk0lhwcXAo2a1DBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjsVKls7g%253D&md5=81f04321981eef4258f2982e0c71cfbc</span></div><a href="/servlet/linkout?suffix=cit6b&amp;dbid=16384&amp;doi=10.1128%2FAAC.05485-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.05485-11%26sid%3Dliteratum%253Aachs%26aulast%3DEakin%26aufirst%3DA.%2BE.%26aulast%3DGreen%26aufirst%3DO.%26aulast%3DHales%26aufirst%3DN.%26aulast%3DWalkup%26aufirst%3DG.%2BK.%26aulast%3DBist%26aufirst%3DS.%26aulast%3DSingh%26aufirst%3DA.%26aulast%3DMullen%26aufirst%3DG.%26aulast%3DBryant%26aufirst%3DJ.%26aulast%3DEmbrey%26aufirst%3DK.%26aulast%3DGao%26aufirst%3DN.%26aulast%3DBreeze%26aufirst%3DA.%26aulast%3DTimms%26aufirst%3DD.%26aulast%3DAndrews%26aufirst%3DB.%26aulast%3DUria-Nickelsen%26aufirst%3DM.%26aulast%3DDemeritt%26aufirst%3DJ.%26aulast%3DLoch%26aufirst%3DJ.%2BT.%26aulast%3DHull%26aufirst%3DK.%26aulast%3DBlodgett%26aufirst%3DA.%26aulast%3DIllingworth%26aufirst%3DR.%2BN.%26aulast%3DPrince%26aufirst%3DB.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DHauck%26aufirst%3DS.%26aulast%3DMacPherson%26aufirst%3DL.%2BJ.%26aulast%3DNi%26aufirst%3DH.%26aulast%3DSherer%26aufirst%3DB.%26atitle%3DPyrrolamide%2520DNA%2520gyrase%2520inhibitors%253A%2520fragment-based%2520nuclear%2520magnetic%2520resonance%2520screening%2520to%2520identify%2520antibacterial%2520agents%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2012%26volume%3D56%26spage%3D1240%26epage%3D1246" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Keating, T. A.; Newman, J. V.; Olivier, N. B.; Otterson, L. G.; Andrews, B.; Boriack-Sjodin, P. A.; Breen, J. N.; Doig, P.; Dumas, J.; Gangl, E.; Green, O. M.; Guler, S. Y.; Hentemann, M. F.; Joseph-McCarthy, D.; Kawatkar, S.; Kutschke, A.; Loch, J. T.; McKenzie, A. R.; Pradeepan, S.; Prasad, S.; Martínez-Botella, G.</span><span> </span><span class="NLM_article-title"><i>In Vivo</i> Validation of Thymidylate Kinase (TMK) with a Rationally-Designed, Selective Antibacterial Compound</span>.  <span class="citation_source-journal">ACS Chem. Biol.</span> DOI: <span class="refDoi"> DOI: 10.1021/cb300316n</span> . Published online: Aug 21,<span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span>.</span><div class="citationLinks">[<a href="/doi/10.1021/cb300316n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=T.+A.+Keating&author=J.+V.+Newman&author=N.+B.+Olivier&author=L.+G.+Otterson&author=B.+Andrews&author=P.+A.+Boriack-Sjodin&author=J.+N.+Breen&author=P.+Doig&author=J.+Dumas&author=E.+Gangl&author=O.+M.+Green&author=S.+Y.+Guler&author=M.+F.+Hentemann&author=D.+Joseph-McCarthy&author=S.+Kawatkar&author=A.+Kutschke&author=J.+T.+Loch&author=A.+R.+McKenzie&author=S.+Pradeepan&author=S.+Prasad&author=G.+Mart%C3%ADnez-Botella&title=In+Vivo+Validation+of+Thymidylate+Kinase+%28TMK%29+with+a+Rationally-Designed%2C+Selective+Antibacterial+Compound&doi=10.1021%2Fcb300316n"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fcb300316n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300316n%26sid%3Dliteratum%253Aachs%26aulast%3DKeating%26aufirst%3DT.%2BA.%26atitle%3DIn%2520Vivo%2520Validation%2520of%2520Thymidylate%2520Kinase%2520%2528TMK%2529%2520with%2520a%2520Rationally-Designed%252C%2520Selective%2520Antibacterial%2520Compound%26jtitle%3DACS%2520Chem.%2520Biol.%26doi%3D10.1021%2Fcb300316n%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Vanheusden, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Rompaey, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munier-Lehmann, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pochet, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herdewijn, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Calenbergh, S.</span><span> </span><span class="NLM_article-title">Thymidine and thymidine-5′-O-monophosphate analogues as inhibitors of <i>Mycobacterium tuberculosis</i> thymidylate kinase</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">3045</span><span class="NLM_x">–</span> <span class="NLM_lpage">3048</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=3045-3048&author=V.+Vanheusdenauthor=P.+Van+Rompaeyauthor=H.+Munier-Lehmannauthor=S.+Pochetauthor=P.+Herdewijnauthor=S.+Van+Calenbergh&title=Thymidine+and+thymidine-5%E2%80%B2-O-monophosphate+analogues+as+inhibitors+of+Mycobacterium+tuberculosis+thymidylate+kinase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVanheusden%26aufirst%3DV.%26aulast%3DVan%2BRompaey%26aufirst%3DP.%26aulast%3DMunier-Lehmann%26aufirst%3DH.%26aulast%3DPochet%26aufirst%3DS.%26aulast%3DHerdewijn%26aufirst%3DP.%26aulast%3DVan%2BCalenbergh%26aufirst%3DS.%26atitle%3DThymidine%2520and%2520thymidine-5%25E2%2580%25B2-O-monophosphate%2520analogues%2520as%2520inhibitors%2520of%2520Mycobacterium%2520tuberculosis%2520thymidylate%2520kinase%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2003%26volume%3D13%26spage%3D3045%26epage%3D3048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit8b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Choi, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plummer, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Starr, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desbonnet, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soutter, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dillman, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roush, W. R.</span><span> </span><span class="NLM_article-title">Structure Guided Development of Novel Thymidine Mimetics Targeting <i>Pseudomonas aeruginosa</i> Thymidylate Kinase: From Hit to Lead Generation</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">852</span><span class="NLM_x">–</span> <span class="NLM_lpage">870</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm201349f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=852-870&author=J.+Y.+Choiauthor=M.+S.+Plummerauthor=J.+Starrauthor=C.+R.+Desbonnetauthor=H.+Soutterauthor=J.+Changauthor=J.+R.+Millerauthor=K.+Dillmanauthor=A.+A.+Millerauthor=W.+R.+Roush&title=Structure+Guided+Development+of+Novel+Thymidine+Mimetics+Targeting+Pseudomonas+aeruginosa+Thymidylate+Kinase%3A+From+Hit+to+Lead+Generation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8b&amp;dbid=16384&amp;doi=10.1021%2Fjm201349f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201349f%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DJ.%2BY.%26aulast%3DPlummer%26aufirst%3DM.%2BS.%26aulast%3DStarr%26aufirst%3DJ.%26aulast%3DDesbonnet%26aufirst%3DC.%2BR.%26aulast%3DSoutter%26aufirst%3DH.%26aulast%3DChang%26aufirst%3DJ.%26aulast%3DMiller%26aufirst%3DJ.%2BR.%26aulast%3DDillman%26aufirst%3DK.%26aulast%3DMiller%26aufirst%3DA.%2BA.%26aulast%3DRoush%26aufirst%3DW.%2BR.%26atitle%3DStructure%2520Guided%2520Development%2520of%2520Novel%2520Thymidine%2520Mimetics%2520Targeting%2520Pseudomonas%2520aeruginosa%2520Thymidylate%2520Kinase%253A%2520From%2520Hit%2520to%2520Lead%2520Generation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D852%26epage%3D870" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Kotaka, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhaliwal, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nichols, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Angell, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lockyer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hawkins, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stammers, D. K.</span><span> </span><span class="NLM_article-title">Structures of <i>S. aureus</i> thymidylate kinase reveal an atypical active site configuration and an intermediate conformational state upon substrate binding</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">774</span><span class="NLM_x">–</span> <span class="NLM_lpage">784</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Fjm3011806&amp;key=10.1110%2Fps.052002406" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Fjm3011806&amp;key=16522804" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Fjm3011806&amp;key=1%3ACAS%3A528%3ADC%252BD28XjsFCqsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2006&pages=774-784&author=M.+Kotakaauthor=B.+Dhaliwalauthor=J.+Renauthor=C.+E.+Nicholsauthor=R.+Angellauthor=M.+Lockyerauthor=A.+R.+Hawkinsauthor=D.+K.+Stammers&title=Structures+of+S.+aureus+thymidylate+kinase+reveal+an+atypical+active+site+configuration+and+an+intermediate+conformational+state+upon+substrate+binding"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Structures of S. aureus thymidylate kinase reveal an atypical active site configuration and an intermediate conformational state upon substrate binding</span></div><div class="casAuthors">Kotaka, Masayo; Dhaliwal, Balvinder; Ren, Jingshan; Nichols, Charles E.; Angell, Richard; Lockyer, Michael; Hawkins, Alastair R.; Stammers, David K.</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">774-784</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Methicillin-resistant Staphylococcus aureus (MRSA) poses a major threat to human health, particularly through hospital-acquired infection.  The spread of MRSA means that novel targets are required to develop potential inhibitors to combat infections caused by such drug-resistant bacteria.  Thymidylate kinase (TMK) is attractive as an antibacterial target as it is essential for providing components for DNA synthesis.  Here, we report crystal structures of unliganded and thymidylate-bound forms of S. aureus thymidylate kinase (SaTMK).  His-tagged and untagged SaTMK crystallize with differing lattice packing and show variations in conformational states for unliganded and thymidylate (TMP)-bound forms.  In addn. to open and closed forms of SaTMK, an intermediate conformation in TMP binding is obsd., in which the site is partially closed.  Anal. of these structures indicates a sequence of events upon TMP binding, with helix α3 shifting position initially, followed by movement of α2 to close the substrate site.  In addn., we observe significant conformational differences in the TMP-binding site in SaTMK as compared to available TMK structures from other bacterial species, Escherichia coli and Mycobacterium tuberculosis as well as human TMK.  In SaTMK, Arg 48 is situated at the base of the TMP-binding site, close to the thymine ring, whereas a cis-proline occupies the equiv. position in other TMKs.  The obsd. TMK structural differences mean that design of compds. highly specific for the S. aureus enzyme looks possible; such inhibitors could minimize the transfer of drug resistance between different bacterial species.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGe5Sj8j_9ObVg90H21EOLACvtfcHk0liCyyGwxHV51g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjsFCqsbc%253D&md5=a0ec45f5ad70b08a5e22f100ee98e443</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1110%2Fps.052002406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.052002406%26sid%3Dliteratum%253Aachs%26aulast%3DKotaka%26aufirst%3DM.%26aulast%3DDhaliwal%26aufirst%3DB.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DNichols%26aufirst%3DC.%2BE.%26aulast%3DAngell%26aufirst%3DR.%26aulast%3DLockyer%26aufirst%3DM.%26aulast%3DHawkins%26aufirst%3DA.%2BR.%26aulast%3DStammers%26aufirst%3DD.%2BK.%26atitle%3DStructures%2520of%2520S.%2520aureus%2520thymidylate%2520kinase%2520reveal%2520an%2520atypical%2520active%2520site%2520configuration%2520and%2520an%2520intermediate%2520conformational%2520state%2520upon%2520substrate%2520binding%26jtitle%3DProtein%2520Sci.%26date%3D2006%26volume%3D15%26spage%3D774%26epage%3D784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Martínez-Botella, G.; Breen, J. N.; Duffy, J. E. S.; Dumas, J.; Geng, B.; Gowers, I. K.; Green, O. M.; Guler, S.; Hentemann, M. F.; Hernandez-Juan, F. A.; Joseph-McCarthy, D.; Kawatkar, S.; Larsen, N. A.; Lazari, O.; Loch, J. T.; Macritchie, J. A.; McKenzie, A. R.; Newman, J. V.; Olivier, N. B.; Otterson, L. G.; Owens, A. P.; Read, J.; Sheppard, D. W.; Keating, T. A.</span> Unpublished results.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Mart%C3%ADnez-Botella%2C+G.%3B+Breen%2C+J.+N.%3B+Duffy%2C+J.+E.+S.%3B+Dumas%2C+J.%3B+Geng%2C+B.%3B+Gowers%2C+I.+K.%3B+Green%2C+O.+M.%3B+Guler%2C+S.%3B+Hentemann%2C+M.+F.%3B+Hernandez-Juan%2C+F.+A.%3B+Joseph-McCarthy%2C+D.%3B+Kawatkar%2C+S.%3B+Larsen%2C+N.+A.%3B+Lazari%2C+O.%3B+Loch%2C+J.+T.%3B+Macritchie%2C+J.+A.%3B+McKenzie%2C+A.+R.%3B+Newman%2C+J.+V.%3B+Olivier%2C+N.+B.%3B+Otterson%2C+L.+G.%3B+Owens%2C+A.+P.%3B+Read%2C+J.%3B+Sheppard%2C+D.+W.%3B+Keating%2C+T.+A.+Unpublished+results."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DMart%25C3%25ADnez-Botella%26aufirst%3DG." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Topliss, J. G.</span><span> </span><span class="NLM_article-title">Utilization of Operational Schemes for Analog Synthesis in Drug Design</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1972</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">1006</span><span class="NLM_x">–</span> <span class="NLM_lpage">1011</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm00280a002" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm3011806&amp;key=1%3ACAS%3A528%3ADyaE3sXhtFChug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1972&pages=1006-1011&author=J.+G.+Topliss&title=Utilization+of+Operational+Schemes+for+Analog+Synthesis+in+Drug+Design"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Utilization of operational schemes for analog synthesis in drug design</span></div><div class="casAuthors">Topliss, John G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1972</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1006-11</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">Proposals were presented for stepwise selection for synthesis of new analogs of a biol. active compd. which were designed to maximize the chances of synthesizing the most potent compds. in the series as early as possible.  The schemes were based on a fundamental assumption of the Hansch method that a particular substituent may modify activity relative to the parent compd. by virtue of resulting changes in hydrophobic, electronic, and steric effects.  The effectiveness of the method was illustrated by retrospective application to several existing series of compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-zymvXigwSbVg90H21EOLACvtfcHk0lhxhtwekRgPNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE3sXhtFChug%253D%253D&md5=acf62a05dbdb3d1d17f91097f66ab8c1</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjm00280a002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00280a002%26sid%3Dliteratum%253Aachs%26aulast%3DTopliss%26aufirst%3DJ.%2BG.%26atitle%3DUtilization%2520of%2520Operational%2520Schemes%2520for%2520Analog%2520Synthesis%2520in%2520Drug%2520Design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1972%26volume%3D15%26spage%3D1006%26epage%3D1011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Fersht, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, J.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knill-Jones, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lowe, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilkinson, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blow, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brick, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waye, M. M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winter, G.</span><span> </span><span class="NLM_article-title">Hydrogen bonding and biological specificity analysed by protein engineering</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1985</span><span class="NLM_x">, </span> <span class="NLM_volume">314</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">238</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Fjm3011806&amp;key=10.1038%2F314235a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Fjm3011806&amp;key=3845322" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm3011806&amp;key=1%3ACAS%3A528%3ADyaL2MXhvVWmsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=314&publication_year=1985&pages=235-238&author=A.+R.+Fershtauthor=J.-P.+Shiauthor=J.+Knill-Jonesauthor=D.+M.+Loweauthor=A.+J.+Wilkinsonauthor=D.+M.+Blowauthor=P.+Brickauthor=P.+Carterauthor=M.+M.+Y.+Wayeauthor=G.+Winter&title=Hydrogen+bonding+and+biological+specificity+analysed+by+protein+engineering"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Hydrogen bonding and biological specificity analyzed by protein engineering</span></div><div class="casAuthors">Fersht, Alan R.; Shi, Jian Ping; Knill-Jones, Jack; Lowe, Denise M.; Wilkinson, Anthony J.; Blow, David M.; Brick, Peter; Carter, Paul; Waye, Mary M. Y.; Winter, Greg</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">314</span>
        (<span class="NLM_cas:issue">6008</span>),
    <span class="NLM_cas:pages">235-8</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">The energetics of H bonding were examd. by using tyrosyl-tRNA synthetase variants produced by site-specific mutagenesis in Bacillus stearothermophilus.  The mutant enzymes varied in their ability to form H bonds.  The contributions of enzyme amino acid side chains to binding of tyrosine (in the transition state) were calcd. from kcat/Km (where kcat is the reaction rate const.).  Deletion of a side chain which forms a H bond with an uncharged group on the substrate decreased binding energy by only 0.5-1.5 kcal/mol, whereas deletion of an uncharged enzyme side chain which formed a H bond with a charged group on the substrate weakened binding by ∼3.5-4.5 kcal/mol.  Deletion of a group which forms an unusually long H bond actually improved binding.  Changes in other side chains (e.g., deletion of Me groups within the protein) caused changes in substrate binding of ≤1 kcal/mol.  The results are discussed with respect to determinants of specificity in biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9Mpcnb5fP9LVg90H21EOLACvtfcHk0lhxhtwekRgPNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXhvVWmsrw%253D&md5=667009772868f87a06dbeab388d06012</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2F314235a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F314235a0%26sid%3Dliteratum%253Aachs%26aulast%3DFersht%26aufirst%3DA.%2BR.%26aulast%3DShi%26aufirst%3DJ.-P.%26aulast%3DKnill-Jones%26aufirst%3DJ.%26aulast%3DLowe%26aufirst%3DD.%2BM.%26aulast%3DWilkinson%26aufirst%3DA.%2BJ.%26aulast%3DBlow%26aufirst%3DD.%2BM.%26aulast%3DBrick%26aufirst%3DP.%26aulast%3DCarter%26aufirst%3DP.%26aulast%3DWaye%26aufirst%3DM.%2BM.%2BY.%26aulast%3DWinter%26aufirst%3DG.%26atitle%3DHydrogen%2520bonding%2520and%2520biological%2520specificity%2520analysed%2520by%2520protein%2520engineering%26jtitle%3DNature%26date%3D1985%26volume%3D314%26spage%3D235%26epage%3D238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klicic, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knoll, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelley, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Francis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shenkin, P. S.</span><span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 1. Method and assessment of docking accuracy</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1739</span><span class="NLM_x">–</span> <span class="NLM_lpage">1749</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13a&amp;dbid=32&amp;doi=10.1021%2Fjm3011806&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+1.+Method+and+assessment+of+docking+accuracy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13aR"><div class="casContent"><span class="casTitleNuber">13a</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0lhxhtwekRgPNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit13a&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25201.%2520Method%2520and%2520assessment%2520of%2520docking%2520accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit13b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beard, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollard, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1750</span><span class="NLM_x">–</span> <span class="NLM_lpage">1759</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13b&amp;dbid=32&amp;doi=10.1021%2Fjm3011806&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+2.+Enrichment+factors+in+database+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13bR"><div class="casContent"><span class="casTitleNuber">13b</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0lhxhtwekRgPNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit13b&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25202.%2520Enrichment%2520factors%2520in%2520database%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Kaminski, G. A</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tirado-Rives, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Evaluation and Reparametrization of the OPLS-AA Force Field for Proteins via Comparison with Accurate Quantum Chemical Calculations on Peptides</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">6474</span><span class="NLM_x">–</span> <span class="NLM_lpage">6487</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2001&pages=6474-6487&author=G.+A+Kaminskiauthor=R.+A.+Friesnerauthor=J.+Tirado-Rivesauthor=W.+L.+Jorgensen&title=Evaluation+and+Reparametrization+of+the+OPLS-AA+Force+Field+for+Proteins+via+Comparison+with+Accurate+Quantum+Chemical+Calculations+on+Peptides"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKaminski%26aufirst%3DG.%2BA%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DEvaluation%2520and%2520Reparametrization%2520of%2520the%2520OPLS-AA%2520Force%2520Field%2520for%2520Proteins%2520via%2520Comparison%2520with%2520Accurate%2520Quantum%2520Chemical%2520Calculations%2520on%2520Peptides%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2001%26volume%3D5%26spage%3D6474%26epage%3D6487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Bieber, P.</span> <span class="citation_source-journal">Comptes Rendus</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1951</span><span class="NLM_x">, </span> <span class="NLM_volume">233</span><span class="NLM_x">, </span> <span class="NLM_fpage">655</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15a&amp;dbid=32&amp;doi=10.1021%2Fjm3011806&amp;key=1%3ACAS%3A528%3ADyaG38XksFajsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Bieber%2C+P.Comptes+Rendus+1951%2C+233%2C+655"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15aR"><div class="casContent"><span class="casTitleNuber">15a</span><div class="casTitle"><span class="NLM_cas:atitle">Action of certain alcoholates on methyl α,β-dihaloisobutyrates</span></div><div class="casAuthors">Bieber, Paul</div><div class="citationInfo"><span class="NLM_cas:title"></span>
        (<span class="NLM_cas:date">1951</span>),
    <span class="NLM_cas:volume">233</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">655-7</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">CH2:CMeCO2Me and Br at 10° give 95% CH2BrCBrMeCO2Me (I), b. 203°, b15 86°, nD20.5 1.505, d420.5 1.805.  The di Cl analog of I b. 170-1°, nD17.5 1.4545, d417.5 1.253.  I and 1 mol. MeONa give CHBr: CMeCO2Me, b. 164°, nD20 1.488, d420 1.4877 (acid chloride, b20 62-4°; amide, m. 109°; anilide, m. 102-3°).  I and excess MeONa give an equil. mixt. of (MeO)2CHCHMeCO2Me, isolated as the amide, m. 90°, and MeOCH: CMeCO2Me (II), b. 175-6°, nD18 1.460, d418 1.070 (acid, m. 105.5-6°).  II and Cl give MeOCHCl-CClMeCO2Me (III), b15 88-91°, nD16 1.4545; II and 2,4-(O2N)2C6H3NHNH2 give 2,4-(O2N)2C6H3NHN:CHCHMeCO2Me, m. 99.5-100°.  The Et ester of the β-EtO analog of III b18 100-3°, nD22 1.445, d422 1.1645.  I and Me3CONa give Me3COCH:CMeCO2Me (IV), b14 86-7°, nD16 1.4555, d416 0.983, sapond. to the acid, m. 117-17.5°.  IV and NaHSO4 give Me2C:CH2 and OHCCHMeCO2Me, which is sapond. and decompd. to EtCHO, MeOH, and CO2.  I and PhONa give only PhOCH: CMeCO2Me (V), b4 121°, nD19 1.540, d419 1.111, and some CH2:CMeCO2Me.  V is sapond. to the acid, m. 109° (acid chloride, b14 142-3°; amide, m. 100°; anilide, m. 152-3°), which with Cu and quinoline gives PhOCH: CHMe, b15 71°, nD19 1.525, d419 0.9713.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoc5N-IkNv-erVg90H21EOLACvtfcHk0lilwHtIUhkYIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaG38XksFajsw%253D%253D&md5=ede619ae3bcf298fb58ad8d6cf090180</span></div><a href="/servlet/linkout?suffix=cit15a&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBieber%26aufirst%3DP.%26jtitle%3DComptes%2520Rendus%26date%3D1951%26volume%3D233%26spage%3D655" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit15b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Csuk, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">von Scholz, Y.</span><span> </span><span class="NLM_article-title">Synthesis of racemic carbocyclic cyclopropanoid nucleoside analogues</span> <span class="citation_source-journal">Tetrahedron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x"> (</span><span class="NLM_issue">26</span><span class="NLM_x">) </span> <span class="NLM_fpage">7193</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=1995&pages=7193&issue=26&author=R.+Csukauthor=Y.+von+Scholz&title=Synthesis+of+racemic+carbocyclic+cyclopropanoid+nucleoside+analogues"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15b&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCsuk%26aufirst%3DR.%26aulast%3Dvon%2BScholz%26aufirst%3DY.%26atitle%3DSynthesis%2520of%2520racemic%2520carbocyclic%2520cyclopropanoid%2520nucleoside%2520analogues%26jtitle%3DTetrahedron%26date%3D1995%26volume%3D51%26issue%3D26%26spage%3D7193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit15c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Kovačková, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dračínský, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rejman, D.</span><span> </span><span class="NLM_article-title">The synthesis of piperidine nucleoside analogs  - a comparison of several methods to access the introduction of nucleobases</span> <span class="citation_source-journal">Tetrahedron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">1485</span><span class="NLM_x">–</span> <span class="NLM_lpage">1500</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=1485-1500&author=S.+Kova%C4%8Dkov%C3%A1author=M.+Dra%C4%8D%C3%ADnsk%C3%BDauthor=D.+Rejman&title=The+synthesis+of+piperidine+nucleoside+analogs++-+a+comparison+of+several+methods+to+access+the+introduction+of+nucleobases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15c&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKova%25C4%258Dkov%25C3%25A1%26aufirst%3DS.%26aulast%3DDra%25C4%258D%25C3%25ADnsk%25C3%25BD%26aufirst%3DM.%26aulast%3DRejman%26aufirst%3DD.%26atitle%3DThe%2520synthesis%2520of%2520piperidine%2520nucleoside%2520analogs%2520%2520-%2520a%2520comparison%2520of%2520several%2520methods%2520to%2520access%2520the%2520introduction%2520of%2520nucleobases%26jtitle%3DTetrahedron%26date%3D2011%26volume%3D67%26spage%3D1485%26epage%3D1500" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HEJ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HEJ','PDB','4HEJ'); return false;">PDB: 4HEJ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GSY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GSY','PDB','4GSY'); return false;">PDB: 4GSY</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HDC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HDC','PDB','4HDC'); return false;">PDB: 4HDC</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i67"><a href="/doi/suppl/10.1021/jm3011806">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_28846"></div></div></div></div></div><hr /></hr><p class="last">Enzymatic, MIC, and log <i>D</i> assay protocols as well as protein crystallography methods and statistics tables.  This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>. PDB accession codes: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HEJ">4HEJ</a> (compound (<i>S</i>)-<b>16</b>); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GSY">4GSY</a> (<b>34</b>); <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4HDC">4HDC</a> (<b>41</b>).</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm3011806/suppl_file/jm3011806_si_001.pdf">jm3011806_si_001.pdf (175.38 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm3011806&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Fjm3011806%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2012.55.issue-22%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm3011806" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679952183993d1df","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
