
<html lang="en"     class="pb-page"  data-request-id="e003663d-bea4-4c3c-9c6a-74cd11e4ecbd"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2021.64.issue-12;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.1c00141"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Novel Bisubstrate Inhibitors for Protein N-Terminal Acetyltransferase D" /></meta><meta name="dc.Creator" content="Youchao  Deng" /></meta><meta name="dc.Creator" content="Sunbin  Deng" /></meta><meta name="dc.Creator" content="Yi-Hsun  Ho" /></meta><meta name="dc.Creator" content="Sarah M.  Gardner" /></meta><meta name="dc.Creator" content="Zhi  Huang" /></meta><meta name="dc.Creator" content="Ronen  Marmorstein" /></meta><meta name="dc.Creator" content="Rong  Huang" /></meta><meta name="dc.Description" content="Protein N-terminal acetyltransferase D (NatD, NAA40) that specifically acetylates the alpha-N-terminus of histone H4 and H2A has been implicated in various diseases, but no inhibitor has been repor..." /></meta><meta name="Description" content="Protein N-terminal acetyltransferase D (NatD, NAA40) that specifically acetylates the alpha-N-terminus of histone H4 and H2A has been implicated in various diseases, but no inhibitor has been repor..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 10, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00141" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00141" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00141" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00141" /></link>
        
    
    

<title>Novel Bisubstrate Inhibitors for Protein N-Terminal Acetyltransferase D | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00141" /></meta><meta property="og:title" content="Novel Bisubstrate Inhibitors for Protein N-Terminal Acetyltransferase D" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/large/jm1c00141_0007.jpeg" /></meta><meta property="og:description" content="Protein N-terminal acetyltransferase D (NatD, NAA40) that specifically acetylates the alpha-N-terminus of histone H4 and H2A has been implicated in various diseases, but no inhibitor has been reported for this important enzyme. Based on the acetyl transfer mechanism of NatD, we designed and prepared a series of highly potent NatD bisubstrate inhibitors by covalently linking coenzyme A to different peptide substrates via an acetyl or propionyl spacer. The most potent bisubstrate inhibitor displayed an apparent Ki value of 1.0 nM. Biochemical studies indicated that bisubstrate inhibitors are competitive to the peptide substrate and noncompetitive to the cofactor, suggesting that NatD undergoes an ordered Bi–Bi mechanism. We also demonstrated that these inhibitors are highly specific toward NatD, displaying about 1000-fold selectivity over other closely related acetyltransferases. High-resolution crystal structures of NatD bound to two of these inhibitors revealed the molecular basis for their selectivity and inhibition mechanism, providing a rational path for future inhibitor development." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00141"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00141">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00141&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00141&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00141&amp;href=/doi/10.1021/acs.jmedchem.1c00141" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 12</span><span class="cit-fg-pageRange">, 8263-8271</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/12" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00914" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c00211" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Novel Bisubstrate Inhibitors for Protein N-Terminal Acetyltransferase D</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Youchao Deng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Youchao Deng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry and Molecular Pharmacology, Purdue Institute for Drug Discovery, Purdue University Center for Cancer Research, Purdue University, West Lafayette, Indiana 47907, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Youchao++Deng">Youchao Deng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sunbin Deng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sunbin Deng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of Pennsylvania, Philadelphia, Pennsylvania 19104, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sunbin++Deng">Sunbin Deng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yi-Hsun Ho</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yi-Hsun Ho</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry and Molecular Pharmacology, Purdue Institute for Drug Discovery, Purdue University Center for Cancer Research, Purdue University, West Lafayette, Indiana 47907, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yi-Hsun++Ho">Yi-Hsun Ho</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sarah M. Gardner</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sarah M. Gardner</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Biophysics, Abramson Family Cancer Research Institute, Graduate Group in Biochemistry and Molecular Biophysics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania 19104, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sarah+M.++Gardner">Sarah M. Gardner</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zhi Huang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zhi Huang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry and Molecular Pharmacology, Purdue Institute for Drug Discovery, Purdue University Center for Cancer Research, Purdue University, West Lafayette, Indiana 47907, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zhi++Huang">Zhi Huang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ronen Marmorstein</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ronen Marmorstein</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University of Pennsylvania, Philadelphia, Pennsylvania 19104, United States</div><div class="loa-info-affiliations-info">Department of Biochemistry and Biophysics, Abramson Family Cancer Research Institute, Graduate Group in Biochemistry and Molecular Biophysics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania 19104, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ronen++Marmorstein">Ronen Marmorstein</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Rong Huang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rong Huang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry and Molecular Pharmacology, Purdue Institute for Drug Discovery, Purdue University Center for Cancer Research, Purdue University, West Lafayette, Indiana 47907, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#375f425659501a457747424553425219525342"><span class="__cf_email__" data-cfemail="dbb3aebab5bcf6a99babaea9bfaebef5bebfae">[email protected]</span></a>. Tel.: (765) 494 3426.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rong++Huang">Rong Huang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-1477-3165" title="Orcid link">https://orcid.org/0000-0002-1477-3165</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00141&amp;href=/doi/10.1021%2Facs.jmedchem.1c00141" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 12</span><span class="cit-pageRange">, 8263–8271</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 10, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>25 January 2021</li><li><span class="item_label"><b>Published</b> online</span>10 June 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 June 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00141" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00141</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D8263%26pageCount%3D9%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DYouchao%2BDeng%252C%2BSunbin%2BDeng%252C%2BYi-Hsun%2BHo%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D12%26contentID%3Dacs.jmedchem.1c00141%26title%3DNovel%2BBisubstrate%2BInhibitors%2Bfor%2BProtein%2BN-Terminal%2BAcetyltransferase%2BD%26numPages%3D9%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D8271%26publicationDate%3DJune%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00141"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">511</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00141" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Novel Bisubstrate Inhibitors for Protein N-Terminal Acetyltransferase D&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Youchao&quot;,&quot;last_name&quot;:&quot;Deng&quot;},{&quot;first_name&quot;:&quot;Sunbin&quot;,&quot;last_name&quot;:&quot;Deng&quot;},{&quot;first_name&quot;:&quot;Yi-Hsun&quot;,&quot;last_name&quot;:&quot;Ho&quot;},{&quot;first_name&quot;:&quot;Sarah&quot;,&quot;last_name&quot;:&quot;M. Gardner&quot;},{&quot;first_name&quot;:&quot;Zhi&quot;,&quot;last_name&quot;:&quot;Huang&quot;},{&quot;first_name&quot;:&quot;Ronen&quot;,&quot;last_name&quot;:&quot;Marmorstein&quot;},{&quot;first_name&quot;:&quot;Rong&quot;,&quot;last_name&quot;:&quot;Huang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;10&quot;,&quot;issue&quot;:&quot;12&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;8263-8271&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00141&quot;},&quot;abstract&quot;:&quot;Protein N-terminal acetyltransferase D (NatD, NAA40) that specifically acetylates the alpha-N-terminus of histone H4 and H2A has been implicated in various diseases, but no inhibitor has been reported for this important enzyme. Based on the acetyl transfer mechanism of NatD, we designed and prepared a series of highly potent NatD bisubstrate inhibitors by covalently linking coenzyme A to different peptide substrates via an acetyl or propionyl spacer. The most potent bisubstrate inhibitor displayed an apparent Ki value of 1.0 nM. Biochemical studies indicated that bisubstrate inhibitors are competitive to the peptide substrate and noncompetitive to the cofactor, suggesting that NatD undergoes an ordered Bi–Bi mechanism. We also demonstrated that these inhibitors are highly specific toward NatD, displaying about 1000-fold selectivity over other closely related acetyltransferases. High-resolution crystal structures of NatD bound to two of these inhibitors revealed the molecular basis for their selectivity and in&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00141&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00141" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00141&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00141" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00141&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00141" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00141&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00141&amp;href=/doi/10.1021/acs.jmedchem.1c00141" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00141" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00141" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00141%26sid%3Dliteratum%253Aachs%26pmid%3D34110812%26genre%3Darticle%26aulast%3DDeng%26date%3D2021%26atitle%3DNovel%2BBisubstrate%2BInhibitors%2Bfor%2BProtein%2BN-Terminal%2BAcetyltransferase%2BD%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D12%26spage%3D8263%26epage%3D8271%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=291490" title="Mass spectrometry">Mass spectrometry</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/12" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/jmcmar.2021.64.issue-12/20210624/jmcmar.2021.64.issue-12.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/medium/jm1c00141_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/large/jm1c00141_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00141&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Protein N-terminal acetyltransferase D (NatD, NAA40) that specifically acetylates the alpha-N-terminus of histone H4 and H2A has been implicated in various diseases, but no inhibitor has been reported for this important enzyme. Based on the acetyl transfer mechanism of NatD, we designed and prepared a series of highly potent NatD bisubstrate inhibitors by covalently linking coenzyme A to different peptide substrates via an acetyl or propionyl spacer. The most potent bisubstrate inhibitor displayed an apparent <i>K</i><sub>i</sub> value of 1.0 nM. Biochemical studies indicated that bisubstrate inhibitors are competitive to the peptide substrate and noncompetitive to the cofactor, suggesting that NatD undergoes an ordered Bi–Bi mechanism. We also demonstrated that these inhibitors are highly specific toward NatD, displaying about 1000-fold selectivity over other closely related acetyltransferases. High-resolution crystal structures of NatD bound to two of these inhibitors revealed the molecular basis for their selectivity and inhibition mechanism, providing a rational path for future inhibitor development.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47587" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47587" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">α-N-terminal acetylation (Nα-acetylation) is a ubiquitous protein modification that occurs on 80–90% of human proteins.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> It is essential for various biological functions, including protein–protein interactions, protein complex formation, cellular apoptosis, rDNA transcriptional regulation, protein subcellular localization, and degradation.<a onclick="showRef(event, 'ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4">(2−4)</a> This modification is catalyzed by protein N-terminal acetyltransferases (NATs) that transfer an acetyl group from the donor acetyl coenzyme A (AcCoA) onto the Nα-amino group of protein substrates. To date, eight members of eukaryotic NATs (NatA–NatH) have been reported.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Among them, NatA and NatD acetylate the nascent chain after the initiator methionine is cleaved.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> NatA is a multisubunit enzyme acetylating ∼40% of human proteins that contain small and uncharged first residues.<a onclick="showRef(event, 'ref1 ref6'); return false;" href="javascript:void(0);" class="ref ref1 ref6">(1,6)</a> In contrast, NatD (NAA40) is a monomeric protein showing extremely high substrate specificity for histone proteins H2A and H4 that have the same N-terminal sequence SGRGK.<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a></div><div class="NLM_p">Nα-acetylation on H4 has diverse biological functions and implications in tumorigenesis.<a onclick="showRef(event, 'ref2 ref9 ref7'); return false;" href="javascript:void(0);" class="ref ref2 ref9 ref7">(2,9,7)</a> Depletion of NatD induces apoptosis through the mitochondrial pathway in colorectal cancer cells.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> In addition, NatD is downregulated in hepatocellular carcinoma tissues and upregulated in primary human lung cancer tissues.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> As the function of histone H4 Nα-acetylation has recently come to light, it has been shown to regulate crosstalk with arginine methylation, lysine acetylation, and serine phosphorylation on H4.<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12,13)</a> For example, Nα-acetylation of H4 stimulates ribosomal DNA expression by inhibiting asymmetric dimethylation of Arg3 on H4 (H4R3me2a), consistent with its critical role in cell growth.<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12,13)</a> Nα-acetylation of H4 suppresses Ser1 phosphorylation and induces the expression of Slug transcription to promote the epithelial-to-mesenchymal transition in lung cancer, suggesting that the acetyltransferase activity of NatD is critical for Slug regulation.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> In addition, Nα-acetylation of H4 promotes the expression of oncogenes through the upregulation of protein arginine methyltransferase 5 in colorectal cancer cells.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Based on these disease connections, NatD has surfaced as a new therapeutic target. Hence, potent and selective NatD inhibitors would be valuable probes to interrogate its functions and therapeutical potential. However, there is no NatD inhibitor reported to date.</div><div class="NLM_p last">The cocrystal structure of the ternary NatD/CoA/SGRGK complex (PDB: 4U9W) revealed that its substrate peptide (SGRGK) is inserted into a highly acidic binding pocket of NatD, and biochemical studies supported the importance of the first four residues (SGRG) of the cognate substrate for specific recognition.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Although the kinetic mechanism of NatD has not yet been elucidated, both NatA and NatE have been shown to follow an ordered Bi–Bi kinetic mechanism.<a onclick="showRef(event, 'ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16">(14−16)</a> Structural alignment of NatD with NatA suggests a similar kinetic mechanism as NatA.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Furthermore, upon mutation of an active-site C137, a previously suggested catalytic residue that forms an acetyl–NatD intermediate, hNatD remained largely active, supporting a direct transfer of the acetyl group through a Bi–Bi kinetic mechanism.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Based on these observations, we hypothesized that bisubstrate analogues could be potent inhibitors for NatD. Bisubstrate analogues have previously been used to help elucidate the catalytic mechanism of other NATs and to develop valuable tool compounds, usually displaying IC<sub>50</sub> values within a low micromolar to high nanomolar range against <i>Sp</i>NatA,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> hNatB,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> hNatF,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> and NatH.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> In the study presented here, we developed highly potent and selective NatD bisubstrate inhibitors. Moreover, the X-ray crystal structures of NatD bound to two bisubstrate inhibitors have been determined to elucidate the molecular interactions, thus paving the way for the structure-based development of more druglike NatD inhibitors.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81727" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81727" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Inhibitor Design</h3><div class="NLM_p">Bisubstrate analogues that covalently connect cofactors and short peptide substrates with a linker have been reported for targeting a variety of transferases, such as methyltransferases and acetyltransferases.<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20,21)</a> Based on the cocrystal structure of the NatD/CoA/SGRGK ternary complex (PDB: 4U9W), the linear distance between the α-nitrogen atom of the first Ser residue and the sulfur atom of CoA was measured to be about 3.1 Å (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Therefore, we proposed to connect the Nα-amines of different peptide substrates and the thiol group of CoA through either an acetyl or propionyl linker to prepare bisubstrate analogues to mimic the transition state, which would be predicted to provide potent and selective inhibitors for NatD.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/medium/jm1c00141_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/large/jm1c00141_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Inhibitor design strategy. The distance between the sulfur atom of CoA (magenta stick) and the Nα nitrogen atom of the peptide (magenta stick) is 3.1 Å in the ternary complex of NatD/CoA/SGRGK (PDB: 4U9W), where NatD is in surface view (gray). The designed NatD bisubstrate analogues <b>1</b>–<b>9</b> are illustrated.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/large/jm1c00141_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00141&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Inhibitor Synthesis</h3><div class="NLM_p">Peptides were prepared using the Rink amide resin following a standard Fmoc solid-phase peptide synthesis protocol. 2-Bromoacetic acid or 3-bromopropionic acid was coupled with the free N-terminal amine of the peptide on the resin. Subsequent cleavage with a cocktail consisting of trifluoroacetic acid (TFA)/water (H<sub>2</sub>O)/triisopropylsilane (TIPS) (95:2.5:2.5) and purification by high-performance liquid chromatography (HPLC) provided the purified bromopeptides, which were then reacted with coenzyme A trilithium salt dihydrate in a triethylammonium bicarbonate buffer.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> The resulting mixture was purified by HPLC to afford the desired bisubstrate analogues <b>1</b>–<b>9</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/medium/jm1c00141_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/large/jm1c00141_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Route for the Bisubstrate Analogues</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/large/jm1c00141_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00141&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last">Reagents and conditions: (a) 2-bromoacetic acid or 3-bromopropionic acid, <i>N</i>,<i>N</i>′-diisopropylcarbodiimide (DIC), hydroxybenzotriazole (HOBt), dimethylformamide (DMF), r.t., overnight; (b) TFA/TIPS/water (95:2.5:2.5), r.t., 4 h, 34–56% in two steps; and (c) CoASH, triethylammonium bicarbonate buffer, pH 8.4 ± 0.1, r.t., 48 h, 33–55%.</p></p></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Structure–Activity Relationship Studies</h3><div class="NLM_p">All synthesized bisubstrate analogues were first evaluated using an established fluorescence assay.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Initial testing was performed in the presence of both AcCoA and an H4-8 peptide (SGRGKGGK-CONH<sub>2</sub>) at their respective Michaelis constant (<i>K</i><sub>m</sub>) values. However, the resulting IC<sub>50</sub> values of compounds <b>1</b>–<b>8</b> were close to or lower than the concentration of NatD (45 nM) in the assay (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00141/suppl_file/jm1c00141_si_001.pdf" class="ext-link">Figure S3a,b</a>), suggesting that they are tight-binding inhibitors.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23,24)</a> To understand the contribution of each component of the bisubstrate analogue to the NatD inhibition, we designed and prepared two control compounds <b>9</b> and <b>10</b>, where <b>9</b> connected CoA via only the Ser residue and <b>10</b> was without the CoA moiety. Compound <b>9</b> displayed an IC<sub>50</sub> value of 218 nM, which showed an over fourfold reduction compared to those analogues containing 3–5 mer peptides. This result corroborated that the first three amino acids contribute significantly to the interaction with NatD as suggested by the ternary complex of NatD/CoA/SGRGK,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> whereas the inhibitory activity of <b>10</b> was reduced over 1000-fold compared to <b>9</b>, supporting the importance of each moiety for potent inhibition of NatD. As the fluorescence assay exhibited a narrow dynamic range (less than twofold) to fully characterize these tight-binding inhibitors <b>1</b>–<b>8</b>, we applied an orthogonal radioactive assay to directly monitor the production of acetylated peptides under a similar condition with both AcCoA and H4-19 peptide substrate (SGRGKGGKGLGKGGAKRHR–COOH; GenScript) at their 4×<i>K</i><sub>m</sub> values as the radioactive assay is generally more sensitive.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> The difference among compounds <b>1</b>–<b>8</b> was more salient in the radioactive assay compared to the fluorescence assay as a 10-fold difference was observed among the IC<sub>50</sub> values of compounds <b>1</b>–<b>8</b> ranging from 4 to 41 nM for NatD (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.1c00141/suppl_file/jm1c00141_si_001.pdf" class="ext-link">Figure S3c,d</a>). The inhibitory activity was reduced when only Ser was incorporated in the substrate moiety, displaying an IC<sub>50</sub> value over 370 nM and supporting the importance of the first three amino acids. Meanwhile, removal of the CoA moiety also reduced the inhibitory activity as propionyl-SGRGK <b>10</b> exhibited an IC<sub>50</sub> value of over 500 μM. Even under this saturated condition, compounds <b>1</b>–<b>8</b> showed tight-binding inhibition as their IC<sub>50</sub> values were below the concentration of NatD, and Morrison’s quadratic equation was applied to analyze the concentration–response data to define their apparent <i>K</i><sub>i</sub> values.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> The most potent inhibitor <b>6</b> that links CoA and the SGRGK peptide through a propionyl linker exhibited a <i>K</i><sub>i,app</sub> value of 1.0 nM (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a,b), which is 2-fold and 8.4-fold higher than those of <b>2</b> and <b>8</b>, respectively. For bisubstrate analogues containing either a tetrapeptide SGRG or pentapeptide SGRGK, a propionyl linker was more favorable than an acetyl linker as the inhibitory activity was two- to fourfold better. However, an acetyl linker was better than a propionyl linker for the bisubstrate analogues that only contained a tripeptide SGR as <b>7</b> was twofold more potent than <b>8</b>. For the bisubstrate analogues containing a pentapeptide SGRGA, both linkers were acceptable and showed more similar potency. When three to five amino acids were incorporated in the bisubstrate analogues, both acetyl and propionyl linkers within the bisubstrate analogues can be accommodated by NatD.</div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/medium/jm1c00141_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/large/jm1c00141_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Biochemical characterization of bisubstrate analogues using the radioactive assay. (a) Concentration–response plots of bisubstrate analogues with an acetyl linker under 4×<i>K</i><sub>m</sub> of the H4-19 peptide and AcCoA condition fitted to Morrison’s quadratic equation (<i>n</i> = 3). (b) Concentration–response plots of bisubstrate analogues with a propionyl linker under 4×<i>K</i><sub>m</sub> of the H4-19 peptide and AcCoA condition (<i>n</i> = 3). (c) CoA-C3-SGRGK (<b>6</b>) is noncompetitive with respect to AcCoA. (d) <b>6</b> is competitive with respect to H4. (e,f) Selectivity study of <b>6</b> and <b>7</b> against a panel of protein acetyltransferases in triplicate (<i>n</i> = 3).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/large/jm1c00141_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00141&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Inhibition Activities of the Bisubstrate Inhibitors</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">ID</th><th class="colsep0 rowsep0" align="center">structure</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">Morrison <i>K</i><sub>i,app</sub> (nM)<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="left">CoA-C2-SGRG</td><td class="colsep0 rowsep0" align="left">48 ± 2.5</td><td class="colsep0 rowsep0" align="left">27 ± 3.8</td><td class="colsep0 rowsep0" align="left">4.4 ± 0.85</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="left">CoA-C3-SGRG</td><td class="colsep0 rowsep0" align="left">48 ± 8.4</td><td class="colsep0 rowsep0" align="left">7.1 ± 1.8</td><td class="colsep0 rowsep0" align="left">2.1 ± 0.32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="left">CoA-C2-SGRGA</td><td class="colsep0 rowsep0" align="left">41 ± 3.8</td><td class="colsep0 rowsep0" align="left">25 ± 3.4</td><td class="colsep0 rowsep0" align="left">6.2 ± 1.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="left">CoA-C3-SGRGA</td><td class="colsep0 rowsep0" align="left">32 ± 2.5</td><td class="colsep0 rowsep0" align="left">35 ± 5.4</td><td class="colsep0 rowsep0" align="left">8.7 ± 1.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left">CoA-C2-SGRGK</td><td class="colsep0 rowsep0" align="left">35 ± 6.0</td><td class="colsep0 rowsep0" align="left">20 ± 3.2</td><td class="colsep0 rowsep0" align="left">4.4 ± 0.58</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="left">CoA-C3-SGRGK</td><td class="colsep0 rowsep0" align="left">40 ± 7.7</td><td class="colsep0 rowsep0" align="left">4.1 ± 0.40</td><td class="colsep0 rowsep0" align="left">1.0 ± 0.11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">CoA-C2-SGR</td><td class="colsep0 rowsep0" align="left">29 ± 5.6</td><td class="colsep0 rowsep0" align="left">16 ± 4.9</td><td class="colsep0 rowsep0" align="left">3.7 ± 0.71</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">CoA-C3-SGR</td><td class="colsep0 rowsep0" align="left">40 ± 3.9</td><td class="colsep0 rowsep0" align="left">41 ± 6.7</td><td class="colsep0 rowsep0" align="left">8.4 ± 1.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="left">CoA-C2-S</td><td class="colsep0 rowsep0" align="left">218 ± 28</td><td class="colsep0 rowsep0" align="left">>370</td><td class="colsep0 rowsep0" align="left">NA</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="left">propionyl-SGRGK</td><td class="colsep0 rowsep0" align="left">330 ± 35 μM</td><td class="colsep0 rowsep0" align="left">>500 μM</td><td class="colsep0 rowsep0" align="left">NA</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Note: Inhibition activities were measured in duplicate (<i>n</i> = 2) in the fluorescence assay with both AcCoA and peptide substrate at their respective <i>K</i><sub>m</sub> values.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">IC<sub>50</sub> values were determined in triplicate (<i>n</i> = 3) in the radioactive assay with both AcCoA and peptide substrate at their respective 4×<i>K</i><sub>m</sub> values.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">Apparent <i>K</i><sub>i</sub> values were analyzed by fitting the concentration–response data obtained in the radioactive assay to Morrison’s quadratic equation for compounds <b>1</b>–<b>8</b>. IC<sub>50</sub> and <i>K</i><sub>i,app</sub> values were presented as mean ± standard deviation (SD). NA: not applied.</p></div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Inhibition Mechanism</h3><div class="NLM_p last">The most potent compound <b>6</b> was selected to investigate the inhibition mechanism. As shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>c, compound <b>6</b> exhibited a noncompetitive inhibition pattern with AcCoA. The data were globally fit to the noncompetitive inhibition equation as a preferred model, producing the best fit parameters, α<i>K</i><sub>i</sub> = 2.1 ± 0.38 nM and <i>K</i><sub>m</sub> = 0.64 ± 0.07 nM. In contrast, compound <b>6</b> exhibited a competitive inhibition pattern with the peptide substrate (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>d). The data were globally fit to the competitive inhibition equation as a preferred model, producing the best fit parameters, <i>K</i><sub>i</sub> = 0.61 ± 0.22 nM and <i>K</i><sub>m</sub> = 0.16 ± 0.03 nM. For a two-substrate enzyme with an ordered binding mechanism where substrate A preferentially binds before substrate B, it is expected that a bisubstrate analogue will be competitive with substrate A and noncompetitive with substrate B.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Thus, our results suggested that NatD undergoes an ordered Bi–Bi mechanism in which the peptide substrate preferentially binds NatD first, followed by AcCoA.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Selectivity Study</h3><div class="NLM_p last">Two of the most potent bisubstrate analogues <b>6</b> and <b>7</b> were chosen to examine their selectivity against a panel of closely related NATs, including NatA, NatB, NatC, and NatE and protein lysine acetyltransferases PCAF and hMOF, which all harbor an Ac-CoA binding motif. Both <b>6</b> and <b>7</b> did not show any inhibition toward NatB, NatC, NatE, and PCAF at 10 μM (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>e,f), indicating over 10,000-fold selectivity for NatD. Compound <b>6</b> demonstrated 10–40% of inhibition against both NatA and hMOF at 1.0 μM, indicating over 1000-fold selectivity for NatD over NatA and hMOF. Compound <b>7</b> displayed IC<sub>50</sub> values over 0.1 μM against both NatA and hMOF, suggesting that it was less selective than <b>6</b>. As NatA is a founding member of the NAT family that prefers to acetylate substrates starting with Ser or Ala, the decreased selectivity of <b>7</b> compared to <b>6</b> is informative in the future development of selective NatD inhibitors.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Cocrystal Structures of <b>5</b> and <b>6</b> with NatD</h3><div class="NLM_p">To understand the molecular interactions between the bisubstrate inhibitors and NatD, the X-ray cocrystal structures of NatD complexed with <b>5</b> (PDB ID: 7KD7) and <b>6</b> (PDB ID: 7KPU) were obtained at 1.44 Å and 1.43 Å resolution, respectively (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a,b). The structure refinement statistics for these structures is provided in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00141/suppl_file/jm1c00141_si_001.pdf" class="ext-link">Table S1</a>. Overall, both structures showed high similarity to the previously reported NatD/CoA/SGRGK ternary complex (PDB ID: 4U9W).<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Structural alignment of the <b>5-NatD</b> and <b>6-NatD</b> binary complexes with the NatD/CoA/SGRGK ternary complex gave RMSD values of 0.255 and 0.217 Å, respectively. Interestingly, despite the different lengths of the linker region within the two bisubstrate inhibitors, the sulfur atom of CoA and the nitrogen atom of the peptide Nα-amino group remained in the same position (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>c). Thus, such similar positioning of these two atoms explains the comparable potency of the inhibitors with two different linkers as described above. Also it is worth noting that CoA-C2-SGRGK and CoA-C3-SGRGK adopt slightly different CoA binding modes, especially with respect to the β-mercaptoethylamine group (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>c). Thus, we speculate that the flexibility of this CoA binding region endows NatD with the ability to accept a bisubstrate inhibitor CoA-C3-SGRGK with a relatively long linker. The structures also reveal why <b>9</b> and <b>10</b> poorly inhibit NatD since both the peptide and CoA portion of the inhibitor make important contributions to inhibitor binding. The tight binding of bisubstrate analogues further supports the Bi–Bi mechanism of NatD.</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/medium/jm1c00141_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/large/jm1c00141_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structures of hNatD with inhibitors. (a) Structure of hNatD with <b>5</b> (CoA-C2-SGRGK) bound. hNatD is shown in cyan as a cartoon and CoA-C2-SGRGK as magenta sticks. The 2mF<sub>obs</sub>-DF<sub>cal</sub> electron density around the bisubstrate is shown at a contour level of 1σ (gray mesh). (b) Structure of hNatD with <b>6</b> (CoA-C3-SGRGK) bound. hNatD is shown in yellow as a cartoon and CoA-C3-SGRGK as blue sticks. The 2mF<sub>obs</sub>-DF<sub>cal</sub> electron density around the bisubstrate is shown at a contour level of 1σ (gray mesh). (c) Overlay of the structure in (a) and (b). The zoom-in view shows the positions of the sulfur atom of CoA and the nitrogen atom of the peptide amino group. (d) Interaction between CoA and hNatD residues is generated with LIGPLOT. Hydrogen bonds are indicated by dashed green lines, and van der Waals interactions are indicated with red semicircles. Water molecules are indicated with cyan spheres. (e) Interaction between SGRGK and hNatD residues is generated with LIGPLOT. Hydrogen bonds are indicated by dashed green lines, and van der Waals interactions are indicated with red semicircles. Water molecules are indicated with cyan spheres.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/large/jm1c00141_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00141&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Given the similarity between both NatD and inhibitor structures, we focused on the structure of NatD with CoA-C2-Ser<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a>-Gly<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a>-Arg<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a>-Gly<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a>-Lys<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> to further examine the molecular basis of NatD inhibition. An extensive H-bonding network is observed for CoA recognition involving several water molecules and NatD residues Asn80, Val140, Leu142, Val146, Arg148, Lys149, Gly150, Leu151, Gly152, Lys153, Phe176, and Asn179 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>d). Hydrophobic interactions are also observed in NatD residues Met81, Gln141, Arg 147, Leu173, Gly181, Ala182, Phe185, Phe186, and Ala189. These extensive interactions explain why the inhibitory activity drops significantly when the CoA portion is removed (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, compare <b>6</b> and <b>10</b>). NatD interacts with the peptide portion of the bisubstrate inhibitor through hydrogen bonding to the backbone of residues <b>1–5</b> and the side chains of Ser<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> and Arg.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> These interactions involve several water molecules and NatD residues Tyr85, Asp127, Val128, Glu129, Tyr136, Cys137, Tyr138, Gln139, Thr174, and Tyr121 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>e). In contrast, van der Waals interactions are significantly more limited, involving only a few residues, including Met80, Trp90, and Ile213. The extensive NatD interaction with the first three residues is consistent with the biochemical inhibitory results: the inhibition potency depends on the inclusion of Ser<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> through Arg<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, compare <b>9</b> with <b>1–8</b>). The NatD/inhibitor structures also reveal that the side chain of Lys<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> does not mediate any interactions with the enzyme, which is consistent with the biochemical findings that a minimal effect on inhibitor potency was observed when Lys<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> was replaced with Ala (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, compare <b>3–4</b> with <b>5–6</b>). Taking these structures together with the binding data, we propose that compounds that mimic at least the mercapto group of acetyl-CoA and the SGR sequence of the peptide with some toleration for flexibility linking these regions could represent suitable druglike lead molecules for NatD inhibition.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Cellular Target Engagement</h3><div class="NLM_p">Although the inherent properties of bisubstrate inhibitors preclude their direct use in cellular studies as they are not able to directly penetrate the cell membrane, we took advantage of digitonin permeabilization to promote cell uptake of compounds <b>2</b> and <b>6</b> for a preliminary investigation on cellular target engagement.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> At 100 μM concentration of compounds <b>2</b> and <b>6</b> with 2 h of incubation, we were able to introduce the compounds into digitonin (5 μg/mL)-treated HCT116 cells (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00141/suppl_file/jm1c00141_si_001.pdf" class="ext-link">Figure S4</a>).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Although the amount of <b>2</b> and <b>6</b> that could enter the cells could not be determined, both <b>2</b> and <b>6</b> induced a higher thermal stabilization (Δ<i>T</i><sub>agg</sub> = +1.3 °C and +1.7 °C, respectively) using the cellular thermal shift assay (CETSA) compared to the control (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>).<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> This result demonstrated that <b>2</b> and <b>6</b> could engage NatD in cells. Future efforts will focus on the introduction of cell-permeable peptides at the C-terminal end of the bisubstrate inhibitors to improve the cell penetration, together with a comprehensive SAR study of <b>6</b> for future optimization.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/medium/jm1c00141_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/large/jm1c00141_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. CETSA. (a) Thermal stability of endogenous NatD by immunoblotting using the anti-NatD antibody (Invitrogen PIPA520882) when treated with <b>2</b> or <b>6</b> or a phosphate-buffered saline (PBS) solution for 2 h at 100 μM after digitonin (5 μg/mL) permeabilization for 20 min. (b) Quantified relative NatD band intensities of the immunoblot in duplicate. <i>T</i><sub>agg:</sub> apparent aggregation temperature.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/large/jm1c00141_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00141&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82247" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82247" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In this work, we designed and synthesized the first series of potent and selective NatD bisubstrate inhibitors <b>1–8</b> exhibiting <i>K</i><sub>i,app</sub> values ranging from 1.0 to 8.7 nM in the radioactive assay. The most potent inhibitor CoA-C3-SGRGK showed a <i>K</i><sub>i,app</sub> value of 1.0 ± 0.11 nM, exhibiting about 1000-fold selectivity for NatD over other acetyltransferases, including NatA-C, NatE, hMOF, and PCAF. Compared to the reported bisubstrate inhibitors for other NATs, including NatA and NatE using a similar bisubstrate strategy,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> the high potency of NatD bisubstrate inhibitors further establishes the uniqueness of NatD and strengthens the possibility to develop potent and specific inhibitors for NatD. Furthermore, the cocrystal structures of NatD in complex with CoA-C3-SGRGK and CoA-C2-SGR clearly demonstrate that the bisubstrate inhibitors engaged both substrate and cofactor AcCoA binding sites. The structural observations reveal that the NatD active site is specifically tailored for its histone substrate, explaining the selectivity of the NatD bisubstrate inhibitors. Comparable inhibitory activities and minor differences in the AcCoA binding site for bisubstrate analogues with either an acetyl or propionyl linker suggest active-site plasticity, which may result from a subtle conformational change within the cofactor binding site of NatD. Moreover, these cocrystal structures of NatD–inhibitor binary complexes provide a structural foundation to guide the future development of druglike NatD inhibitors.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75362" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75362" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> General Chemistry Procedures</h3><div class="NLM_p last">All chemicals and solvents were purchased from commercial suppliers and used without further purification unless stated otherwise. Preparative reversed-phase high-pressure liquid chromatography (RP-HPLC) was performed using an Agilent 1260 series system. Systems were run with 0–95% methanol/water gradient with a 0.1% TFA modifier. High-resolution matrix-assisted laser desorption/ionization (MALDI) spectra were recorded using a 4800 MALDI-TOF/TOF mass spectrometer (Sciex) at the Mass Spectrometry and Purdue Proteomics Facility (PPF), Purdue University. Peptides were synthesized using the CEM Liberty Blue peptide synthesizer. Compounds were also characterized and confirmed by TLC–MS or MALDI–MS. The purity of the final compounds was confirmed using a Waters LC–MS system and/or an Agilent 1260 series system. Systems were run with 0–40% methanol/water gradient with 0.1% TFA. The purity of all target compounds obtained was >95%.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> General Procedure A for Solid-Phase Peptide Synthesis</h3><div class="NLM_p last">The peptides were synthesized using a Liberty Blue automated microwave peptide synthesizer (CEM Corp., Matthews, NC, USA) following a standard Fmoc protocol.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> A Rink amide MBHA resin (0.05 mmol) was used as a solid support and placed in a microwave tube. Standard couplings of amino acids were carried out in 0.2 M DMF and external amino acids in 0.1 M DMF using 0.5 M DIC and 1.0 M Oxyma in DMF for activation and 20% piperidine in DMF for deprotection. The resin was transferred to a filter-equipped syringe, washed with CH<sub>2</sub>Cl<sub>2</sub> (3 mL) and MeOH (3 mL) three times, and dried under air.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> General Procedure B for the Synthesis of the Bromoacetylated Peptide<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a></h3><div class="NLM_p last">To a suspension of peptide on resin (0.05 mmol, 1.0 equiv) in DMF (2 mL) were added the corresponding acid (0.1 mmol, 2 equiv), DIC (15.5 μL, 0.1 mmol, 2 equiv), and HOBt (13.5 mg, 0.1 mmol, 2 equiv). The suspension was shaken at room temperature for 20 h. The solvent was filtered, and the resulting resin was washed with CH<sub>2</sub>Cl<sub>2</sub> (3 mL) and MeOH (3 mL) three times and dried under air. The dried peptide was cleaved from the resin using a cleavage cocktail (TFA/TIPS/ddH2O = 95/2.5/2.5 v/v) (5 mL) for 0.05 mmol scale of resin for 4 h. The suspension was filtered, the filtrate was washed with TFA (2 mL), and the volatiles of the filtrate were removed under N<sub>2</sub>. The peptide solution was precipitated with cold anhydrous ether (10 mL) and centrifuged at 4200 rpm for 10 min. The supernatant was discarded, and the pellet was washed with cold ether, centrifuged, supernatant was removed, and air-dried. The dried peptide was dissolved in ddH2O (5 mL) and filtered using a 0.2 μM filter membrane. The filtered sample solution was purified by preparative RP-HPLC using an Agilent 1260 series system with a C18 column (5 μm, 10 mm × 250 mm) at a flow rate of 4.0 mL/min. Two mobile phases (mobile phase A consisting of 0.1% TFA in ddH2O and mobile phase B consisting of methanol) were used and monitored at 214 and 256 nm. An injection volume of 400 μL of the solution was injected into the column. The desired fractions were evaporated, and the resulting solution was frozen (−85 °C) and lyophilized.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> General Procedure C for the Synthesis of the Bisubstrate Analogues<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a></h3><div class="NLM_p">The bromoacetylated peptide (1 equiv) was dissolved in 1 mL of triethylammonium bicarbonate buffer, pH 8.4 ± 0.1. The mixture was added to coenzyme A trilithium salt dihydrate (2 equiv), and the solution was allowed to react for 4 h at room temperature and then overnight at 4 °C. The reaction solution (100 μL) was purified by HPLC to obtain the desired product, which was detected by MALDI.</div><div class="NLM_p">Compound <b>1</b> (5.1 mg, 43% yield) was synthesized by following the general procedure C as a foamy white solid. MALDI-TOF (positive) <i>m</i>/<i>z</i>: calcd for C<sub>36</sub>H<sub>63</sub>N<sub>15</sub>O<sub>22</sub>P<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup><i>m</i>/<i>z</i> 1182.3200, found <i>m</i>/<i>z</i> 1182.3802.</div><div class="NLM_p">Compound <b>2</b> (4.3 mg, 36% yield) was synthesized by following the general procedure C as a foamy white solid. MALDI-TOF (positive) <i>m</i>/<i>z</i>: calcd for C<sub>37</sub>H<sub>65</sub>N<sub>15</sub>O<sub>22</sub>P<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup><i>m</i>/<i>z</i> 1196.3357, found <i>m</i>/<i>z</i> 1196.3725.</div><div class="NLM_p">Compound <b>3</b> (5.6 mg, 45% yield) was synthesized by following the general procedure C as a foamy white solid. MALDI-TOF (positive) <i>m</i>/<i>z</i>: calcd for C<sub>39</sub>H<sub>68</sub>N<sub>16</sub>O<sub>23</sub>P<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup><i>m</i>/<i>z</i> 1253.3571, found <i>m</i>/<i>z</i> 1253.5187.</div><div class="NLM_p">Compound <b>4</b> (4.8 mg, 38% yield) was synthesized by following the general procedure C as a foamy white solid. MALDI-TOF (positive) <i>m</i>/<i>z</i>: calcd for C<sub>40</sub>H<sub>70</sub>N<sub>16</sub>O<sub>23</sub>P<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup><i>m</i>/<i>z</i> 1267.3728, found <i>m</i>/<i>z</i> 1267.3970.</div><div class="NLM_p">Compound <b>5</b> (6.2 mg, 47% yield) was synthesized by following the general procedure C as a foamy white solid. MALDI-TOF (positive) <i>m</i>/<i>z</i>: calcd for C<sub>42</sub>H<sub>75</sub>N<sub>17</sub>O<sub>23</sub>P<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup><i>m</i>/<i>z</i> 1310.4150, found <i>m</i>/<i>z</i> 1310.4417.</div><div class="NLM_p">Compound <b>6</b> (4.8 mg, 36% yield) was synthesized by following the general procedure C as a foamy white solid. MALDI-TOF (positive) <i>m</i>/<i>z</i>: calcd for C<sub>43</sub>H<sub>77</sub>N<sub>17</sub>O<sub>23</sub>P<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup><i>m</i>/<i>z</i> 1324.4306, found <i>m</i>/<i>z</i> 1324.4917.</div><div class="NLM_p">Compound <b>7</b> (4.7 mg, 42% yield) was synthesized by following the general procedure C as a foamy white solid. MALDI-TOF (positive) <i>m</i>/<i>z</i>: calcd for C<sub>34</sub>H<sub>60</sub>N<sub>14</sub>O<sub>23</sub>P<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup><i>m</i>/<i>z</i> 1125.2986, found <i>m</i>/<i>z</i> 1125.2587.</div><div class="NLM_p">Compound <b>8</b> (3.8 mg, 33% yield) was synthesized by following the general procedure C as a foamy white solid. MALDI-TOF (positive) <i>m</i>/<i>z</i>: calcd for C<sub>35</sub>H<sub>62</sub>N<sub>14</sub>O<sub>23</sub>P<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup><i>m</i>/<i>z</i> 1139.3142, found <i>m</i>/<i>z</i> 1139.2743.</div><div class="NLM_p">Compound <b>9</b> (4.7 mg, 51% yield) was synthesized by following the general procedure C as a foamy white solid. MALDI-TOF (positive) <i>m</i>/<i>z</i>: calcd for C<sub>26</sub>H<sub>45</sub>N<sub>9</sub>O<sub>19</sub>P<sub>3</sub>S<sup>+</sup> [M + H]<sup>+</sup><i>m</i>/<i>z</i> 912.1760, found <i>m</i>/<i>z</i> 912.1420.</div><div class="NLM_p">Compound <b>10</b> (3.1 mg, 55% yield) was synthesized by following the general procedure B as a foamy white solid. MALDI-TOF (positive) <i>m</i>/<i>z</i>: calcd for C<sub>22</sub>H<sub>43</sub>N<sub>10</sub>O<sub>7</sub><sup>+</sup> [M + H]<sup>+</sup><i>m</i>/<i>z</i> 559.3311, found <i>m</i>/<i>z</i> 559.3385.</div><div class="NLM_p">Protein Expression and Purification</div><div class="NLM_p last">Expression and purification of human PCAF, hMOF, and NatA-E except for NatC were performed as previously described.<a onclick="showRef(event, 'ref8 ref17 ref31 ref32 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref8 ref17 ref31 ref32 ref33 ref34">(8,17,31−34)</a></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> NatC</h3><div class="NLM_p last">Ternary <i>Schizosaccharomyces pombe</i> NatC encoding NAA30<sup>FL</sup>, NAA35<sup>31–708</sup>, and NAA38<sup>48–116</sup> was expressed in <i>Escherichia coli</i> cells and purified as follows. Cells were harvested by centrifugation, resuspended, and lysed by sonication in a lysis buffer containing 25 mM Tris, pH 8.0, 300 mM NaCl, and 10 mg/mL PMSF. The lysate was clarified by centrifugation and passed over a nickel resin (Thermo Scientific), which was subsequently washed with ∼10 column volumes of a wash buffer containing 25 mM Tris, pH 8.0, 300 mM NaCl, 20 mM imidazole, and 10 mM 2-mercaptoethanol. The protein was eluted with a buffer containing 25 mM Tris, pH 8.0, 300 mM NaCl, 200 mM imidazole, and 10 mM βME. After elution, His-tagged Ulp1 protease was added to the eluent to cleave the SUMO tag. The eluent was further dialyzed with a buffer containing 25 mM sodium citrate monobasic, pH 5.5, 10 mM NaCl, and 10 mM 2-mercaptoethanol. The protein was purified with a 5 mL HiTrap SP ion-exchange column and eluted with a salt gradient (10–1000 mM NaCl). Peak fractions were concentrated to ∼0.5 mL with a 50 kDa concentrator (Amicon Ultra, Millipore) and loaded into an S200 gel-filtration column (GE Healthcare) with a buffer containing 25 mM HEPES, pH 7.0, 200 mM NaCl, and 1 mM dithiothreitol (DTT). Peak fractions were concentrated to ∼15 mg/mL as measured by UV280 and flash-frozen for storage at −80 °C until further use.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Fluorescence Assay</h3><div class="NLM_p last">A fluorescence-based assay was conducted to study the IC<sub>50</sub> values for all the compounds. The assay was performed under the following conditions in a final well volume of 40 μL: 25 mM HEPES (pH = 7.5), 150 mM NaCl, 0.01% Triton X-100, 45 nM NatD, 0.5 μM AcCoA, and 15 μM ThioGlo4. The inhibitors were added at concentrations ranging from 0.15 nM to 10 μM. After 30 min of incubation, reactions were initiated by the addition of 5.0 μM H4-8 peptide. The fluorescence was monitored using a BMG CLARIOstar microplate reader with excitation at 400 nm and emission at 465 nm. Data were processed using GraphPad Prism software 7.0.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Radioisotopic Acetyltransferase Assay</h3><div class="NLM_p last">FL NatD was used for the activity assays. NatD acetyltransferase assays were carried out in 25 mM HEPES, pH 7.5, 150 mM NaCl, and 1 mM DTT. The H4 substrate peptide used in the assay corresponds to the first 19 residues of human H4 (NH<sub>2</sub>-SGRGKGGKGLGKGGAKRHR-COOH; GenScript). In the assay, 50 nM hNatD was mixed with 2 μM radiolabeled [<sup>14</sup>C]acetyl-CoA (4 mCi/mmol; PerkinElmer Life Sciences), 20 μM of the peptide, and inhibitors of concentrations ranging from 0.15 nM to 10 μM for a reaction of 30 min at room temperature. To quench the reaction, the reaction solution was applied onto negatively charged P81 paper discs (SVI, St Vincent’s Institute of Medical Research) to trap the peptides, and the paper discs were immediately placed in a wash buffer (10 mM HEPES, pH 7.5). The paper discs were washed three times, at 5 min per wash, to remove unreacted acetyl-CoA. The papers discs were then dried with acetone and 4 mL of a scintillation fluid was added, and the signal was measured with a Packard Tri-Carb 1500 liquid scintillation analyzer. Each reaction was performed in triplicate. The IC<sub>50</sub> statistic values were determined with GraphPad Prism software 7.0.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a></div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Inhibition Mechanism Study</h3><div class="NLM_p last">The experiment procedures are the same as described for the radioactive assay. In the AcCoA concentration fixed assay, 50 nM hNatD was mixed with 2 μM radiolabeled [<sup>14</sup>C]acetyl-CoA, H4 peptide (0.25, 1.0, 10, and 20 μM), and inhibitors (0, 0.45, and 1.4 nM) for a reaction of 13 min at room temperature. In the H4 peptide concentration fixed assay, 50 nM hNatD was mixed with 20 μM H4 peptide, [<sup>14</sup>C]acetyl-CoA (0.15, 0.3, 0.6, 1.5, 3, and 6 μM), and inhibitors (0, 0.45, 1.4, and 4.1 nM) for a reaction of 13 min at room temperature.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Selectivity Assays</h3><div class="NLM_p">The selectivity studies of NatA, NatB, NatC, NatE, PCAF, and hMOF were performed as follows. hNatA (100 nM) was mixed with 30 μM [<sup>14</sup>C]acetyl-CoA and 30 μM of either H4 peptide or SASE peptide (NH<sub>2</sub>-SASEAGVRWGRPVGRRRRP-COOH; GenScript) with 0, 0.1, 1, or 10 μM inhibitors for a 12 min reaction at room temperature in a buffer containing 75 mM HEPES, pH 7.5, 120 mM NaCl, and 1 mM DTT.</div><div class="NLM_p">hNatB (100 nM) was mixed with 50 μM [<sup>14</sup>C]acetyl-CoA and 50 μM MDVF peptide (NH<sub>2</sub>-MDVFMKGRWGRPVGRRRRP-COOH, GenScript) with 0, 0.1, 1, or 10 μM inhibitors for a 10 min reaction at room temperature in a buffer containing 75 mM HEPES, pH 7.5, 120 mM NaCl, and 1 mM DTT.</div><div class="NLM_p"><i>Sp</i>NatC (50 nM) was mixed with 30 μM [<sup>14</sup>C]acetyl-CoA and 10 μM MLRF peptide (NH<sub>2</sub>-ML RFVTKRWGRPVGRRRRPCOOH, GenScript), with 0, 0.1, 1, or 10 μM inhibitors, for a 5 min reaction at room temperature in a buffer containing 75 mM HEPES, pH 7.0, 120 mM NaCl, and 1 mM DTT.</div><div class="NLM_p">hNatE (300 nM) was mixed with 50 μM [<sup>14</sup>C]acetyl-CoA and 100 μM MLGP peptide (NH<sub>2</sub>-MLGPEGGRWGRPVGRRRRP-COOH, GenScript), with 0, 0.1, 1, or 10 μM inhibitors for a 40 min reaction at room temperature in a buffer containing 75 mM HEPES, pH 7.0, 120 mM NaCl, and 1 mM DTT.</div><div class="NLM_p">PCAF (100 nM) was mixed with 50 μM [<sup>14</sup>C]acetyl-CoA and 400 μM H4 peptide with 0, 0.1, 1, or 10 μM inhibitors for a 20 min reaction at room temperature in a buffer containing 40 mM Tris, pH 8.0, 100 mM NaCl, 1 mM DTT, and 2 mg/mL BSA.</div><div class="NLM_p last">hMOF (50 nM) was mixed with 50 μM [<sup>14</sup>C]acetyl-CoA and 400 μM H4 peptide with 0, 0.1, 1, or 10 μM inhibitors for a 20 min reaction at room temperature in a buffer containing 100 mM Tris, pH 8.0, 50 mM NaCl, 1 mM DTT, 800 μM cysteine, and 0.25 mg/mL BSA.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Cocrystallization and Structure Determination</h3><div class="NLM_p last">For cocrystallization and structure determination of <b>5</b> and <b>6</b> with NatD, a truncation construct of hNatD<sup>17–220</sup> was prepared and purified similarly as described above. The purified hNatD<sup>17–220</sup> (10 mg/mL) was incubated with 1 mM of either <b>5</b> or <b>6</b> for 30 min in ice before the crystallization. The best crystals of hNatD<sup>17–220</sup> with <b>5</b> bound was obtained by hanging drop vapor diffusion at 20 °C in a well containing 0.1 M Bis–Tris, pH 5.5, and 2 M ammonium sulfate, in a drop containing a 1:1.5 mixture of protein to well solution. The best crystals of hNatD<sup>17–220</sup> with <b>6</b> bound was obtained by hanging drop vapor diffusion at 20 °C in a well solution containing 0.1 M Bis–Tris, pH 5.5, and 2 M ammonium sulfate, in a drop containing a 1.25:1 mixture of protein to well solution. All crystals were cryoprotected by transferring them to their respective well solutions supplemented with 20% glycerol before being flash-frozen in liquid nitrogen. Data were collected at the Advanced Photon Source (beamline 24-ID-E) and processed using HKL2000.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a></div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Structure Determination and Refinement</h3><div class="NLM_p last">Both structures were determined by molecular replacement using the structure of NatD/CoA/SGRGK (PDB: 4U9W) with ligands and solvent molecules removed from the search model. Molecular replacement was carried out using Phaser in Phenix.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Initial manual model building was performed in Coot<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> and all subsequent rounds of refinement were performed using phenix.refine and Coot interchangeably. Refinement statistics are provided in <a href="/doi/suppl/10.1021/acs.jmedchem.1c00141/suppl_file/jm1c00141_si_001.pdf" class="ext-link">Table S1</a>. The final model and structure factors were submitted to the Protein Data Bank. Distance calculations as well as three-dimensional alignment r.m.s. deviations and graphics were generated in PyMOL (http://www.pymol.org/).</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> CETSA</h3><div class="NLM_p last">The assay was performed based on a modified protocol for the cell lysate CETSA experiments.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> In brief, cultured HCT116 cells (5.0 × 10<sup>7</sup>) were incubated with 5 μg/mL digitonin at 37 °C for 20 min.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> The medium was aspirated and replenished with a fresh medium with PBS or inhibitors of 100 μM. After 2 h, the cells were harvested and washed with PBS and resuspended in 1 mL of PBS. The cell solution (300 μL) was directly lysed by three freeze–thaw cycles with liquid nitrogen and clarified by centrifugation at 20,000 g for 20 min at 4 °C. The supernatant was analyzed with MALDI–MS. Aliquots of 50 μL from each condition were distributed into PCR strip tubes and heated at 37, 40, 45, 50, 55, 58, 60, 62, 65, 69, 73, and 75 °C for 3 min before being cooled to room temperature in 3 min. The cells were lysed by three freeze–thaw cycles with liquid nitrogen and clarified by centrifugation at 20,000 g for 20 min at 4 °C. The soluble fractions (lysate) were then analyzed by immunoblotting using the anti-NatD antibody (Invitrogen PIPA520882) and quantified using ImageJ.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00141" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78877" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78877" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00141?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00141</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">HRMS and HPLC spectra of compounds <b>1</b>–<b>10 (</b>Figures S1,S2<b>)</b>; IC<sub>50</sub> fitting curves of compounds <b>1</b>–<b>10</b> (Figure S3); cell permeability study (Figure S4); and data statistics for NatD crystal structures in complex with inhibitors (Table S1) (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00141/suppl_file/jm1c00141_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings for all synthesized inhibitors (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00141/suppl_file/jm1c00141_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00141/suppl_file/jm1c00141_si_001.pdf">jm1c00141_si_001.pdf (1.06 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00141/suppl_file/jm1c00141_si_002.csv">jm1c00141_si_002.csv (1.61 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates for the structure of human NatD in complex with compound <b>5 (</b>PDB ID: 7KD7<b>) and 6 (</b>PDB ID: 7KPU<b>)</b> have been deposited in the Protein Data Bank. Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.1c00141" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63125" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63125" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rong Huang</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and Molecular Pharmacology, Purdue Institute
for Drug Discovery, Purdue University Center for Cancer Research, Purdue University, West Lafayette, Indiana 47907, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-1477-3165" title="Orcid link">https://orcid.org/0000-0002-1477-3165</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#49213c28272e643b09393c3b2d3c2c672c2d3c"><span class="__cf_email__" data-cfemail="7f170a1e1118520d3f0f0a0d1b0a1a511a1b0a">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Youchao Deng</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and Molecular Pharmacology, Purdue Institute
for Drug Discovery, Purdue University Center for Cancer Research, Purdue University, West Lafayette, Indiana 47907, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sunbin Deng</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Pennsylvania, Philadelphia, Pennsylvania 19104, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yi-Hsun Ho</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and Molecular Pharmacology, Purdue Institute
for Drug Discovery, Purdue University Center for Cancer Research, Purdue University, West Lafayette, Indiana 47907, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sarah M. Gardner</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Biophysics, Abramson Family Cancer Research Institute,
Graduate Group in Biochemistry and Molecular Biophysics, Perelman
School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania 19104, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhi Huang</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and Molecular Pharmacology, Purdue Institute
for Drug Discovery, Purdue University Center for Cancer Research, Purdue University, West Lafayette, Indiana 47907, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ronen Marmorstein</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University of Pennsylvania, Philadelphia, Pennsylvania 19104, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Biophysics, Abramson Family Cancer Research Institute,
Graduate Group in Biochemistry and Molecular Biophysics, Perelman
School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania 19104, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>Y.D. synthesized and characterized all compounds described in the manuscript. Y.H. and Y.D. performed the fluorescence assay and cellular thermal shift assay. Z.H. contributed to the synthesis. Y.D. and S.D. prepared manuscript figures and text. S.D. performed the radioactivity assay and selectivity study. S.D. determined  the cocrystal structures.  R.M. designed and supervised the experiments by S.D. and S.M.G. and prepared the manuscript text. R.H. developed the concept, designed and supervised the experiments by Y.D., Y.H., and Z.H., and prepared manuscript text and figures. All authors have read and approved the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i31">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_86602" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_86602" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors acknowledge the support from the NIH grant R35GM118090 (R.M.) and the proteomics core facility at the Purdue University Center for Cancer Research (P30 CA023168). We also thank the support from the Department of Medicinal Chemistry and Molecular Pharmacology (R.H.) and Leah Gottlieb for her assistance in obtaining proteins of PCAF and hMOF for the inhibitor selectivity studies.</p></div><div class="postAsapNotesSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="postAsapNotesSection">Note Added After ASAP Publication</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61773" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61773" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Originally published ASAP on June 10, 2021; abstract graphic updated on June 16, 2021.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">NatA</td><td class="NLM_def"><p class="first last">protein N-terminal acetyltransferase A</p></td></tr><tr><td class="NLM_term">CoA</td><td class="NLM_def"><p class="first last">coenzyme A</p></td></tr><tr><td class="NLM_term">AcCoA</td><td class="NLM_def"><p class="first last">acetyl coenzyme A</p></td></tr><tr><td class="NLM_term">r.t.</td><td class="NLM_def"><p class="first last">room temperature</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i33">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28662" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28662" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 37 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arnesen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Damme, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polevoda, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helsens, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evjenth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colaert, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varhaug, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandekerckhove, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lillehaug, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gevaert, K.</span></span> <span> </span><span class="NLM_article-title">Proteomics Analyses Reveal the Evolutionary Conservation and Divergence of N-Terminal Acetyltransferases from Yeast and Humans</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">8157</span>– <span class="NLM_lpage">8162</span>, <span class="refDoi"> DOI: 10.1073/pnas.0901931106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1073%2Fpnas.0901931106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=19420222" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmvVWmsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=8157-8162&author=T.+Arnesenauthor=P.+Van+Dammeauthor=B.+Polevodaauthor=K.+Helsensauthor=R.+Evjenthauthor=N.+Colaertauthor=J.+E.+Varhaugauthor=J.+Vandekerckhoveauthor=J.+R.+Lillehaugauthor=F.+Shermanauthor=K.+Gevaert&title=Proteomics+Analyses+Reveal+the+Evolutionary+Conservation+and+Divergence+of+N-Terminal+Acetyltransferases+from+Yeast+and+Humans&doi=10.1073%2Fpnas.0901931106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomics analyses reveal the evolutionary conservation and divergence of N-terminal acetyltransferases from yeast and humans</span></div><div class="casAuthors">Arnesen, Thomas; Van Damme, Petra; Polevoda, Bogdan; Helsens, Kenny; Evjenth, Rune; Colaert, Niklaas; Varhaug, Jan Erik; Vandekerckhove, Joel; Lillehaug, Johan R.; Sherman, Fred; Gevaert, Kris</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8157-8162, S8157/1-S8157/13</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Nα-terminal acetylation is one of the most common protein modifications in eukaryotes.  The COmbined FRActional Diagonal Chromatog. (COFRADIC) proteomics technol. that can be specifically used to isolate N-terminal peptides was used to det. the N-terminal acetylation status of 742 human and 379 yeast protein N termini, representing the largest eukaryotic dataset of N-terminal acetylation.  The major N-terminal acetyltransferase (NAT), NatA, acts on subclasses of proteins with Ser-, Ala-, Thr-, Gly-, Cys-, and Val- N termini.  NatA is composed of subunits encoded by yARD1 and yNAT1 in yeast and hARD1 and hNAT1 in humans.  A yeast ard1-Δ nat1-Δ strain was phenotypically complemented by hARD1 hNAT1, suggesting that yNatA and hNatA are similar.  However, heterologous combinations, hARD1 yNAT1 and yARD1 hNAT1, were not functional in yeast, suggesting significant structural subunit differences between the species.  Proteomics of a yeast ard1-Δ nat1-Δ strain expressing hNatA demonstrated that hNatA acts on nearly the same set of yeast proteins as yNatA, further revealing that NatA from humans and yeast have identical or nearly identical specificities.  Nevertheless, all NatA substrates in yeast were only partially N-acetylated, whereas the corresponding NatA substrates in HeLa cells were mainly completely N-acetylated.  Overall, we obsd. a higher proportion of N-terminally acetylated proteins in humans (84%) as compared with yeast (57%).  N-acetylation occurred on approx. one-half of the human proteins with Met-Lys-termini, but did not occur on yeast proteins with such termini.  Thus, although we revealed different N-acetylation patterns in yeast and humans, the major NAT, NatA, acetylates the same substrates in both species.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroDhOHLppIB7Vg90H21EOLACvtfcHk0liA-iXQXa3cfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmvVWmsb8%253D&md5=c4fe67e8da4d45102d8d4c79fc4d7d19</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0901931106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0901931106%26sid%3Dliteratum%253Aachs%26aulast%3DArnesen%26aufirst%3DT.%26aulast%3DVan%2BDamme%26aufirst%3DP.%26aulast%3DPolevoda%26aufirst%3DB.%26aulast%3DHelsens%26aufirst%3DK.%26aulast%3DEvjenth%26aufirst%3DR.%26aulast%3DColaert%26aufirst%3DN.%26aulast%3DVarhaug%26aufirst%3DJ.%2BE.%26aulast%3DVandekerckhove%26aufirst%3DJ.%26aulast%3DLillehaug%26aufirst%3DJ.%2BR.%26aulast%3DSherman%26aufirst%3DF.%26aulast%3DGevaert%26aufirst%3DK.%26atitle%3DProteomics%2520Analyses%2520Reveal%2520the%2520Evolutionary%2520Conservation%2520and%2520Divergence%2520of%2520N-Terminal%2520Acetyltransferases%2520from%2520Yeast%2520and%2520Humans%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D8157%26epage%3D8162%26doi%3D10.1073%2Fpnas.0901931106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monda, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulman, B. A.</span></span> <span> </span><span class="NLM_article-title">N-Terminal Acetylation Acts as an Avidity Enhancer within an Interconnected Multiprotein Complex</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>334</i></span>,  <span class="NLM_fpage">674</span>– <span class="NLM_lpage">678</span>, <span class="refDoi"> DOI: 10.1126/science.1209307</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1126%2Fscience.1209307" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=21940857" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyqu7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=334&publication_year=2011&pages=674-678&author=D.+C.+Scottauthor=J.+K.+Mondaauthor=E.+J.+Bennettauthor=J.+W.+Harperauthor=B.+A.+Schulman&title=N-Terminal+Acetylation+Acts+as+an+Avidity+Enhancer+within+an+Interconnected+Multiprotein+Complex&doi=10.1126%2Fscience.1209307"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">N-Terminal Acetylation Acts as an Avidity Enhancer Within an Interconnected Multiprotein Complex</span></div><div class="casAuthors">Scott, Daniel C.; Monda, Julie K.; Bennett, Eric J.; Harper, J. Wade; Schulman, Brenda A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">334</span>
        (<span class="NLM_cas:issue">6056</span>),
    <span class="NLM_cas:pages">674-678</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Although many eukaryotic proteins are amino (N)-terminally acetylated, structural mechanisms by which N-terminal acetylation mediates protein interactions are largely unknown.  Here, N-terminal acetylation of the E2 enzyme, Ubc12, dictates distinctive E3-dependent ligation of the ubiquitin-like protein Nedd8 to Cul1.  Structural, biochem., biophys., and genetic analyses revealed how complete burial of Ubc12's N-acetyl-methionine in a hydrophobic pocket in the E3, Dcn1, promotes cullin neddylation.  The results suggest that the N-terminal acetyl both directs Ubc12's interactions with Dcn1 and prevents repulsion of a charged N terminus.  The authors' data provide a link between acetylation and ubiquitin-like protein conjugation and define a mechanism for N-terminal acetylation-dependent recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2YZdYRdU_fbVg90H21EOLACvtfcHk0liA-iXQXa3cfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyqu7jL&md5=1654bea16702633a9baf611827f60450</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1126%2Fscience.1209307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1209307%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BC.%26aulast%3DMonda%26aufirst%3DJ.%2BK.%26aulast%3DBennett%26aufirst%3DE.%2BJ.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DSchulman%26aufirst%3DB.%2BA.%26atitle%3DN-Terminal%2520Acetylation%2520Acts%2520as%2520an%2520Avidity%2520Enhancer%2520within%2520an%2520Interconnected%2520Multiprotein%2520Complex%26jtitle%3DScience%26date%3D2011%26volume%3D334%26spage%3D674%26epage%3D678%26doi%3D10.1126%2Fscience.1209307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Behnia, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panic, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whyte, J. R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span> <span> </span><span class="NLM_article-title">Targeting of the Arf-like GTPase Arl3p to the Golgi Requires N-Terminal Acetylation and the Membrane Protein Sys1p</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">405</span>– <span class="NLM_lpage">413</span>, <span class="refDoi"> DOI: 10.1038/ncb1120</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1038%2Fncb1120" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=15077113" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjsFSltrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2004&pages=405-413&author=R.+Behniaauthor=B.+Panicauthor=J.+R.+C.+Whyteauthor=S.+Munro&title=Targeting+of+the+Arf-like+GTPase+Arl3p+to+the+Golgi+Requires+N-Terminal+Acetylation+and+the+Membrane+Protein+Sys1p&doi=10.1038%2Fncb1120"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting of the Arf-like GTPase Arl3p to the Golgi requires N-terminal acetylation and the membrane protein Sys1p</span></div><div class="casAuthors">Behnia, Rudy; Panic, Bojana; Whyte, James R. C.; Munro, Sean</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">405-413</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The GTPase Arl3p is required to recruit a second GTPase, Arl1p, to the Golgi in Saccharomyces cerevisiae.  Arl1p binds to the GRIP domain, which is present in a no. of long coiled-coil proteins or 'golgins'.  Here we show that Arl3p is not myristoylated like most members of the Arf family, but is instead amino-terminally acetylated by the NatC complex.  Targeting of Arl3p also requires a Golgi membrane protein Sys1p.  The human homologs of Arl3p (Arf-related protein 1 (ARFRP1)) and Sys1p (hSys1) can be isolated in a complex after chem. crosslinking.  This suggests that the targeting of ARFRP1/Arl3p to the Golgi is mediated by a direct interaction between its acetylated N terminus and Sys1p/hSys1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohQbxTiIsb2bVg90H21EOLACvtfcHk0lgs1Aw218hG7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjsFSltrY%253D&md5=99e48d738c8771f7cd23771d8700c5a1</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fncb1120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb1120%26sid%3Dliteratum%253Aachs%26aulast%3DBehnia%26aufirst%3DR.%26aulast%3DPanic%26aufirst%3DB.%26aulast%3DWhyte%26aufirst%3DJ.%2BR.%2BC.%26aulast%3DMunro%26aufirst%3DS.%26atitle%3DTargeting%2520of%2520the%2520Arf-like%2520GTPase%2520Arl3p%2520to%2520the%2520Golgi%2520Requires%2520N-Terminal%2520Acetylation%2520and%2520the%2520Membrane%2520Protein%2520Sys1p%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2004%26volume%3D6%26spage%3D405%26epage%3D413%26doi%3D10.1038%2Fncb1120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aksnes, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ree, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnesen, T.</span></span> <span> </span><span class="NLM_article-title">Co-Translational, Post-Translational, and Non-Catalytic Roles of N-Terminal Acetyltransferases</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">1097</span>– <span class="NLM_lpage">1114</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2019.02.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1016%2Fj.molcel.2019.02.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=30878283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkvVertbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2019&pages=1097-1114&author=H.+Aksnesauthor=R.+Reeauthor=T.+Arnesen&title=Co-Translational%2C+Post-Translational%2C+and+Non-Catalytic+Roles+of+N-Terminal+Acetyltransferases&doi=10.1016%2Fj.molcel.2019.02.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Co-translational, Post-translational, and Non-catalytic Roles of N-Terminal Acetyltransferases</span></div><div class="casAuthors">Aksnes, Henriette; Ree, Rasmus; Arnesen, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1097-1114</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Recent studies of N-terminal acetylation have identified new N-terminal acetyltransferases (NATs) and expanded the known functions of these enzymes beyond their roles as ribosome-assocd. co-translational modifiers.  For instance, the identification of Golgi- and chloroplast-assocd. NATs shows that acetylation of N termini also happens post-translationally.  In addn., we now appreciate that some NATs are highly specific; for example, a dedicated NAT responsible for post-translational N-terminal acetylation of actin was recently revealed.  Other studies have extended NAT function beyond Nt acetylation, including functions as lysine acetyltransferases (KATs) and non-catalytic roles.  Finally, emerging studies emphasize the physiol. relevance of N-terminal acetylation, including roles in calorie-restriction-induced longevity and pathol. α-synuclein aggregation in Parkinson's disease.  Combined, the NATs rise as multifunctional proteins, and N-terminal acetylation is gaining recognition as a major cellular regulator.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZokGZxIsfBLVg90H21EOLACvtfcHk0lgs1Aw218hG7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkvVertbc%253D&md5=8f4282b0ff189e0b3bbc7d6643398e90</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2019.02.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2019.02.007%26sid%3Dliteratum%253Aachs%26aulast%3DAksnes%26aufirst%3DH.%26aulast%3DRee%26aufirst%3DR.%26aulast%3DArnesen%26aufirst%3DT.%26atitle%3DCo-Translational%252C%2520Post-Translational%252C%2520and%2520Non-Catalytic%2520Roles%2520of%2520N-Terminal%2520Acetyltransferases%26jtitle%3DMol.%2520Cell%26date%3D2019%26volume%3D73%26spage%3D1097%26epage%3D1114%26doi%3D10.1016%2Fj.molcel.2019.02.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span> <span> </span><span class="NLM_article-title">Protein N-Terminal Acetylation: Structural Basis, Mechanism, Versatility, and Regulation</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2020.08.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1016%2Fj.tibs.2020.08.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=32912665" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1Oltb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2021&pages=15-27&author=S.+Dengauthor=R.+Marmorstein&title=Protein+N-Terminal+Acetylation%3A+Structural+Basis%2C+Mechanism%2C+Versatility%2C+and+Regulation&doi=10.1016%2Fj.tibs.2020.08.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Protein N-Terminal Acetylation: Structural Basis, Mechanism, Versatility, and Regulation</span></div><div class="casAuthors">Deng, Sunbin; Marmorstein, Ronen</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">15-27</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">N-terminal acetylation (NTA) is one of the most widespread protein modifications, which occurs on most eukaryotic proteins, but is significantly less common on bacterial and archaea proteins.  This modification is carried out by a family of enzymes called N-terminal acetyltransferases (NATs).  To date, 12 NATs have been identified, harboring different compn., substrate specificity, and in some cases, modes of regulation.  Recent structural and biochem. anal. of NAT proteins allows for a comparison of their mol. mechanisms and modes of regulation, which are described here.  Although sharing an evolutionarily conserved fold and related catalytic mechanism, each catalytic subunit uses unique elements to mediate substrate-specific activity, and use NAT-type specific auxiliary and regulatory subunits, for their cellular functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOQcv07MEVNbVg90H21EOLACvtfcHk0lgs1Aw218hG7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1Oltb7N&md5=fcec4efee655278b2184deb9557ea627</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2020.08.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2020.08.005%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DS.%26aulast%3DMarmorstein%26aufirst%3DR.%26atitle%3DProtein%2520N-Terminal%2520Acetylation%253A%2520Structural%2520Basis%252C%2520Mechanism%252C%2520Versatility%252C%2520and%2520Regulation%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2021%26volume%3D46%26spage%3D15%26epage%3D27%26doi%3D10.1016%2Fj.tibs.2020.08.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ree, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varland, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnesen, T.</span></span> <span> </span><span class="NLM_article-title">Spotlight on Protein N-Terminal Acetylation</span>. <i>Exp. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">13</span>, <span class="refDoi"> DOI: 10.1038/s12276-018-0116-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1038%2Fs12276-018-0116-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=30054468" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVSqs77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2018&pages=1-13&author=R.+Reeauthor=S.+Varlandauthor=T.+Arnesen&title=Spotlight+on+Protein+N-Terminal+Acetylation&doi=10.1038%2Fs12276-018-0116-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Spotlight on protein N-terminal acetylation</span></div><div class="casAuthors">Ree, Rasmus; Varland, Sylvia; Arnesen, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Experimental & Molecular Medicine</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1-13</span>CODEN:
                <span class="NLM_cas:coden">EMMEF3</span>;
        ISSN:<span class="NLM_cas:issn">2092-6413</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  N-terminal acetylation (Nt-acetylation) is a widespread protein modification among eukaryotes and prokaryotes alike.  By appending an acetyl group to the N-terminal amino group, the charge, hydrophobicity, and size of the N-terminus is altered in an irreversible manner.  This alteration has implications for the lifespan, folding characteristics and binding properties of the acetylated protein.  The enzymic machinery responsible for Nt-acetylation has been largely described, but significant knowledge gaps remain.  In this review, we provide an overview of eukaryotic N-terminal acetyltransferases (NATs) and the impact of Nt-acetylation.  We also discuss other functions of known NATs and outline methods for studying Nt-acetylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAsJi2UstR57Vg90H21EOLACvtfcHk0liwDkMp8oQyOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVSqs77F&md5=f45b6c761774a8302c340356032b442f</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fs12276-018-0116-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs12276-018-0116-z%26sid%3Dliteratum%253Aachs%26aulast%3DRee%26aufirst%3DR.%26aulast%3DVarland%26aufirst%3DS.%26aulast%3DArnesen%26aufirst%3DT.%26atitle%3DSpotlight%2520on%2520Protein%2520N-Terminal%2520Acetylation%26jtitle%3DExp.%2520Mol.%2520Med.%26date%3D2018%26volume%3D50%26spage%3D1%26epage%3D13%26doi%3D10.1038%2Fs12276-018-0116-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hole, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Damme, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalva, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aksnes, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glomnes, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varhaug, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lillehaug, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gevaert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnesen, T.</span></span> <span> </span><span class="NLM_article-title">The Human N-Alpha-Acetyltransferase 40 (HNaa40p/HNatD) Is Conserved from Yeast and N-Terminally Acetylates Histones H2A and H4</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e24713</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0024713</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1371%2Fjournal.pone.0024713" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=21935442" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1Oms77L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&author=K.+Holeauthor=P.+van+Dammeauthor=M.+Dalvaauthor=H.+Aksnesauthor=N.+Glomnesauthor=J.+E.+Varhaugauthor=J.+R.+Lillehaugauthor=K.+Gevaertauthor=T.+Arnesen&title=The+Human+N-Alpha-Acetyltransferase+40+%28HNaa40p%2FHNatD%29+Is+Conserved+from+Yeast+and+N-Terminally+Acetylates+Histones+H2A+and+H4&doi=10.1371%2Fjournal.pone.0024713"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The human N-alpha-acetyltransferase 40 (hNaa40p/hNatD) is conserved from yeast and N-terminally acetylates histones H2A and H4</span></div><div class="casAuthors">Hole, Kristine; Van Damme, Petra; Dalva, Monica; Aksnes, Henriette; Glomnes, Nina; Varhaug, Jan Erik; Lillehaug, Johan R.; Gevaert, Kris; Arnesen, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e24713</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Protein Nα-terminal acetylation (Nt-acetylation) is considered one of the most common protein modification in eukaryotes, and 80-90 % of all sol. human proteins are modified in this way, with functional implications ranging from altered protein function and stability to translocation potency amongst others.  Nt-acetylation is catalyzed by N-terminal acetyltransferases (NATs), and in yeast five NAT types are identified and denoted NatA-NatE.  Higher eukaryotes addnl. express NatF.  Except for NatD, human orthologs for all yeast NATs are identified.  YNatD is defined as the catalytic unit Naa40p (Nat4) which co-translationally Nt-acetylates histones H2A and H4.  In this study we identified and characterized hNaa40p/hNatD, the human ortholog of the yeast Naa40p.  An in vitro proteome-derived peptide library Nt-acetylation assay indicated that recombinant hNaa40p acetylates N-termini starting with the consensus sequence Ser-Gly-Gly-Gly-Lys-, strongly resembling the N-termini of the human histones H2A and H4.  This was confirmed as recombinant hNaa40p Nt-acetylated the oligopeptides derived from the N-termini of both histones.  In contrast, a synthetically Nt-acetylated H4 N-terminal peptide with all lysines being non-acetylated, was not significantly acetylated by hNaa40p, indicating that hNaa40p catalyzed H4 Nα-acetylation and not H4 lysine N.vepsiln.-acetylation.  Also, immunopptd. hNaa40p specifically Nt-acetylated H4 in vitro.  Heterologous expression of hNaa40p in a yeast naa40-Δ strain restored Nt-acetylation of yeast histone H4, but not H2A in vivo, probably reflecting the fact that the N-terminal sequences of human H2A and H4 are highly similar to each other and to yeast H4 while the N-terminal sequence of yeast H2A differs.  Thus, Naa40p seems to have co-evolved with the human H2A sequence.  Finally, a partial co-sedimentation with ribosomes indicates that hNaa40p co-translationally acetylates H2A and H4.  Combined, our results strongly suggest that human Naa40p/NatD is conserved from yeast.  Thus, the NATs of all classes of N-terminally acetylated proteins in humans now appear to be accounted for.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrd6pv_cgdTgLVg90H21EOLACvtfcHk0liwDkMp8oQyOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1Oms77L&md5=e2a58a3364d8547f35dad47edfb2c971</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0024713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0024713%26sid%3Dliteratum%253Aachs%26aulast%3DHole%26aufirst%3DK.%26aulast%3Dvan%2BDamme%26aufirst%3DP.%26aulast%3DDalva%26aufirst%3DM.%26aulast%3DAksnes%26aufirst%3DH.%26aulast%3DGlomnes%26aufirst%3DN.%26aulast%3DVarhaug%26aufirst%3DJ.%2BE.%26aulast%3DLillehaug%26aufirst%3DJ.%2BR.%26aulast%3DGevaert%26aufirst%3DK.%26aulast%3DArnesen%26aufirst%3DT.%26atitle%3DThe%2520Human%2520N-Alpha-Acetyltransferase%252040%2520%2528HNaa40p%252FHNatD%2529%2520Is%2520Conserved%2520from%2520Yeast%2520and%2520N-Terminally%2520Acetylates%2520Histones%2520H2A%2520and%2520H4%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26doi%3D10.1371%2Fjournal.pone.0024713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Magin, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liszczak, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span> <span> </span><span class="NLM_article-title">The Molecular Basis for Histone H4- and H2A-Specific Amino-Terminal Acetylation by NatD</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">332</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1016/j.str.2014.10.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1016%2Fj.str.2014.10.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=25619998" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVaru7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=332-341&author=R.+S.+Maginauthor=G.+P.+Liszczakauthor=R.+Marmorstein&title=The+Molecular+Basis+for+Histone+H4-+and+H2A-Specific+Amino-Terminal+Acetylation+by+NatD&doi=10.1016%2Fj.str.2014.10.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The Molecular Basis for Histone H4- and H2A-Specific Amino-Terminal Acetylation by NatD</span></div><div class="casAuthors">Magin, Robert S.; Liszczak, Glen P.; Marmorstein, Ronen</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">332-341</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">N-terminal acetylation is among the most common protein modifications in eukaryotes and is mediated by evolutionarily conserved N-terminal acetyltransferases (NATs).  NatD is among the most selective NATs; its only known substrates are histones H4 and H2A, contg. the N-terminal sequence SGRGK in humans.  Here we characterize the mol. basis for substrate-specific acetylation by NatD by reporting its crystal structure bound to cognate substrates and performing related biochem. studies.  A novel N-terminal segment wraps around the catalytic core domain to make stabilizing interactions, and the α1-α2 and β6-β7 loops adopt novel conformations to properly orient the histone N termini in the binding site.  Ser1 and Arg3 of the histone make extensive contacts to highly conserved NatD residues in the substrate binding pocket, and flanking glycine residues also appear to contribute to substrate-specific binding by NatD, together defining a Ser-Gly-Arg-Gly (SGRG) recognition sequence.  These studies have implications for understanding substrate-specific acetylation by NAT enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqe7cTJj5emoLVg90H21EOLACvtfcHk0ljA0EXGfb5h-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVaru7c%253D&md5=54c7a1111ff735db3ab546b0fc9eb8aa</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2014.10.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2014.10.025%26sid%3Dliteratum%253Aachs%26aulast%3DMagin%26aufirst%3DR.%2BS.%26aulast%3DLiszczak%26aufirst%3DG.%2BP.%26aulast%3DMarmorstein%26aufirst%3DR.%26atitle%3DThe%2520Molecular%2520Basis%2520for%2520Histone%2520H4-%2520and%2520H2A-Specific%2520Amino-Terminal%2520Acetylation%2520by%2520NatD%26jtitle%3DStructure%26date%3D2015%26volume%3D23%26spage%3D332%26epage%3D341%26doi%3D10.1016%2Fj.str.2014.10.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Polevoda, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norbeck, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takakura, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blomberg, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, F.</span></span> <span> </span><span class="NLM_article-title">Identification and Specificities of N-Terminal Acetyltransferases from Saccharomyces Cerevisiae</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">6155</span>– <span class="NLM_lpage">6168</span>, <span class="refDoi"> DOI: 10.1093/emboj/18.21.6155</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1093%2Femboj%2F18.21.6155" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10545125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADyaK1MXns1ent7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1999&pages=6155-6168&author=B.+Polevodaauthor=J.+Norbeckauthor=H.+Takakuraauthor=A.+Blombergauthor=F.+Sherman&title=Identification+and+Specificities+of+N-Terminal+Acetyltransferases+from+Saccharomyces+Cerevisiae&doi=10.1093%2Femboj%2F18.21.6155"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and specificities of N-terminal acetyltransferases from Saccharomyces cerevisiae</span></div><div class="casAuthors">Polevoda, Bogdan; Norbeck, Joakim; Takakura, Hikaru; Blomberg, Anders; Sherman, Fred</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6155-6168</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">N-terminal acetylation can occur cotranslationally on the initiator methionine residue or on the penultimate residue if the methionine is cleaved.  We investigated the three N-terminal acetyltransferases (NATs), Ard1p/Nat1p, Nat3p and Mak3p.  Ard1p and Mak3p are significantly related to each other by amino acid sequence, as is Nat3p, which was uncovered in this study using programming alignment procedures.  Mutants deleted in any one of these NAT genes were viable, but some exhibited diminished mating efficiency and reduced growth at 37°C, and on glycerol and NaCl-contg. media.  The three NATs had the following substrate specificities as detd. in vivo by examg. acetylation of 14 altered forms of iso-1-cytochrome c and 55 abundant normal proteins in each of the deleted strains: Ard1p/Nat1p, subclasses with Ser-, Ala-, Gly- and Thr-termini; Nat3p, Met-Glu- and Met- Asp- and a subclass of Met-Asn-termini; and Mak3p subclasses with Met-Ile- and Met-Leu-termini.  In addn., a special subclass of substrates with Ser-Glu-Phe-, Ala-Glu-Phe- and Gly-Glu-Phe-termini required all three NATs for acetylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-zatcFLGJ8rVg90H21EOLACvtfcHk0ljA0EXGfb5h-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXns1ent7s%253D&md5=16d7691c8fb2459187c5d5d28f0b6e37</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1093%2Femboj%2F18.21.6155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252F18.21.6155%26sid%3Dliteratum%253Aachs%26aulast%3DPolevoda%26aufirst%3DB.%26aulast%3DNorbeck%26aufirst%3DJ.%26aulast%3DTakakura%26aufirst%3DH.%26aulast%3DBlomberg%26aufirst%3DA.%26aulast%3DSherman%26aufirst%3DF.%26atitle%3DIdentification%2520and%2520Specificities%2520of%2520N-Terminal%2520Acetyltransferases%2520from%2520Saccharomyces%2520Cerevisiae%26jtitle%3DEMBO%2520J.%26date%3D1999%26volume%3D18%26spage%3D6155%26epage%3D6168%26doi%3D10.1093%2Femboj%2F18.21.6155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pavlou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirmizis, A.</span></span> <span> </span><span class="NLM_article-title">Depletion of Histone N-Terminal-Acetyltransferase Naa40 Induces P53-Independent Apoptosis in Colorectal Cancer Cells via the Mitochondrial Pathway</span>. <i>Apoptosis</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">298</span>– <span class="NLM_lpage">311</span>, <span class="refDoi"> DOI: 10.1007/s10495-015-1207-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1007%2Fs10495-015-1207-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=26666750" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVOjs7vN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=298-311&author=D.+Pavlouauthor=A.+Kirmizis&title=Depletion+of+Histone+N-Terminal-Acetyltransferase+Naa40+Induces+P53-Independent+Apoptosis+in+Colorectal+Cancer+Cells+via+the+Mitochondrial+Pathway&doi=10.1007%2Fs10495-015-1207-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Depletion of histone N-terminal-acetyltransferase Naa40 induces p53-independent apoptosis in colorectal cancer cells via the mitochondrial pathway</span></div><div class="casAuthors">Pavlou, Demetria; Kirmizis, Antonis</div><div class="citationInfo"><span class="NLM_cas:title">Apoptosis</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">298-311</span>CODEN:
                <span class="NLM_cas:coden">APOPFN</span>;
        ISSN:<span class="NLM_cas:issn">1360-8185</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Protein N-terminal acetylation is an abundant post-translational modification in eukaryotes implicated in various fundamental cellular and biochem. processes.  This modification is catalyzed by evolutionarily conserved N-terminal acetyltransferases (NATs) whose deregulation has been linked to cancer development and thus, are emerging as useful diagnostic and therapeutic targets.  Naa40 is a highly selective NAT that acetylates the amino-termini of histones H4 and H2A and acts as a sensor of cell growth in yeast.  In the present study, we examine the role of Naa40 in cancer cell survival.  We demonstrate that depletion of Naa40 in HCT116 and HT-29 colorectal cancer cells decreases cell survival by enhancing apoptosis, whereas Naa40 redn. in non-cancerous mouse embryonic fibroblasts has no effect on cell viability.  Specifically, Naa40 knockdown in colon cancer cells activates the mitochondrial caspase-9-mediated apoptotic cascade.  Consistent with this, we show that caspase-9 activation is required for the induced apoptosis because treatment of cells with an irreversible caspase-9 inhibitor impedes apoptosis when Naa40 is depleted.  Furthermore, the effect of Naa40-depletion on cell-death is mediated through a p53-independent mechanism since p53-null HCT116 cells still undergo apoptosis upon redn. of the acetyltransferase.  Altogether, these findings reveal an anti-apoptotic role for Naa40 and exhibit its potential as a therapeutic target in colorectal cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqS_QZjAJOfpbVg90H21EOLACvtfcHk0ljA0EXGfb5h-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVOjs7vN&md5=80bc057a86e6bf5620baa6ed12971aaf</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1007%2Fs10495-015-1207-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10495-015-1207-0%26sid%3Dliteratum%253Aachs%26aulast%3DPavlou%26aufirst%3DD.%26aulast%3DKirmizis%26aufirst%3DA.%26atitle%3DDepletion%2520of%2520Histone%2520N-Terminal-Acetyltransferase%2520Naa40%2520Induces%2520P53-Independent%2520Apoptosis%2520in%2520Colorectal%2520Cancer%2520Cells%2520via%2520the%2520Mitochondrial%2520Pathway%26jtitle%3DApoptosis%26date%3D2016%26volume%3D21%26spage%3D298%26epage%3D311%26doi%3D10.1007%2Fs10495-015-1207-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Man, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, Q.</span></span> <span> </span><span class="NLM_article-title">Patt1, a Novel Protein Acetyltransferase That Is Highly Expressed in Liver and Downregulated in Hepatocellular Carcinoma, Enhances Apoptosis of Hepatoma Cells</span>. <i>Int. J. Biochem. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">2528</span>– <span class="NLM_lpage">2537</span>, <span class="refDoi"> DOI: 10.1016/j.biocel.2009.08.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1016%2Fj.biocel.2009.08.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=19695338" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlGhsrrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2009&pages=2528-2537&author=Z.+Liuauthor=Y.+Liuauthor=H.+Wangauthor=X.+Geauthor=Q.+Jinauthor=G.+Dingauthor=Y.+Huauthor=B.+Zhouauthor=Z.+Chenauthor=X.+Geauthor=B.+Zhangauthor=X.+Manauthor=Q.+Zhai&title=Patt1%2C+a+Novel+Protein+Acetyltransferase+That+Is+Highly+Expressed+in+Liver+and+Downregulated+in+Hepatocellular+Carcinoma%2C+Enhances+Apoptosis+of+Hepatoma+Cells&doi=10.1016%2Fj.biocel.2009.08.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Patt1, a novel protein acetyltransferase that is highly expressed in liver and downregulated in hepatocellular carcinoma, enhances apoptosis of hepatoma cells</span></div><div class="casAuthors">Liu, Zhen; Liu, Yang; Wang, Huiqiang; Ge, Xinjian; Jin, Qihuang; Ding, Guanghui; Hu, Yanan; Zhou, Ben; Chen, Zhongjian; Ge, Xuemei; Zhang, Baohua; Man, Xiaobo; Zhai, Qiwei</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Biochemistry & Cell Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2528-2537</span>CODEN:
                <span class="NLM_cas:coden">IJBBFU</span>;
        ISSN:<span class="NLM_cas:issn">1357-2725</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Protein acetylation is increasingly recognized as an important post-translational modification.  Although a lot of protein acetyltransferases have been identified, a few putative acetyltransferases are yet to be studied.  In this study, the authors identified a novel protein acetyltransferase, Patt1, which belongs to GNAT family.  Patt1 exhibited histone acetyltransferase activity and auto-acetylation activity.  Deletion and mutation anal. of the predicted acetyltransferase domain in Patt1 showed that the conserved Glu139 was an important residue for its protein acetyltransferase activity.  Furthermore, the authors found that Patt1 was highly expressed in liver and significantly down-regulated in hepatocellular carcinoma tissues.  In addn., the authors showed that overexpression of Patt1 enhanced the apoptosis of hepatoma cells dependent on its acetyltransferase activity, whereas knockdown of Patt1 significantly protected Chang liver cells from apoptosis.  These data suggest that Patt1 might be involved in the development of hepatocellular carcinoma, and could be served as a potential therapy target for hepatocellular carcinoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7oW5hR1AxrLVg90H21EOLACvtfcHk0lglLnMoYjG9tQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlGhsrrL&md5=d49149cb399f2eeccbe26abf6b5526a3</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.biocel.2009.08.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biocel.2009.08.009%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DGe%26aufirst%3DX.%26aulast%3DJin%26aufirst%3DQ.%26aulast%3DDing%26aufirst%3DG.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DGe%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DMan%26aufirst%3DX.%26aulast%3DZhai%26aufirst%3DQ.%26atitle%3DPatt1%252C%2520a%2520Novel%2520Protein%2520Acetyltransferase%2520That%2520Is%2520Highly%2520Expressed%2520in%2520Liver%2520and%2520Downregulated%2520in%2520Hepatocellular%2520Carcinoma%252C%2520Enhances%2520Apoptosis%2520of%2520Hepatoma%2520Cells%26jtitle%3DInt.%2520J.%2520Biochem.%2520Cell%2520Biol.%26date%3D2009%26volume%3D41%26spage%3D2528%26epage%3D2537%26doi%3D10.1016%2Fj.biocel.2009.08.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schiza, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molina-Serrano, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kyriakou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadjiantoniou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirmizis, A.</span></span> <span> </span><span class="NLM_article-title">N-Alpha-Terminal Acetylation of Histone H4 Regulates Arginine Methylation and Ribosomal DNA Silencing</span>. <i>PLoS Genet.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">e1003805</span> <span class="refDoi"> DOI: 10.1371/journal.pgen.1003805</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1371%2Fjournal.pgen.1003805" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=24068969" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFKitbrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&author=V.+Schizaauthor=D.+Molina-Serranoauthor=D.+Kyriakouauthor=A.+Hadjiantoniouauthor=A.+Kirmizis&title=N-Alpha-Terminal+Acetylation+of+Histone+H4+Regulates+Arginine+Methylation+and+Ribosomal+DNA+Silencing&doi=10.1371%2Fjournal.pgen.1003805"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">N-alpha-terminal acetylation of histone H4 regulates arginine methylation and ribosomal DNA silencing</span></div><div class="casAuthors">Schiza, Vassia; Molina-Serrano, Diego; Kyriakou, Dimitris; Hadjiantoniou, Antonia; Kirmizis, Antonis</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Genetics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e1003805</span>CODEN:
                <span class="NLM_cas:coden">PGLEB5</span>;
        ISSN:<span class="NLM_cas:issn">1553-7404</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Post-translational modifications of histones play a key role in DNA-based processes, like transcription, by modulating chromatin structure.  N-terminal acetylation is unique among the numerous histone modifications because it is deposited on the N-alpha amino group of the first residue instead of the side-chain of amino acids.  The function of this modification and its interplay with other internal histone marks has not been previously addressed.  Here, we identified N-terminal acetylation of H4 (N-acH4) as a novel regulator of arginine methylation and chromatin silencing in Saccharomyces cerevisiae.  Lack of the H4 N-alpha acetyltransferase (Nat4) activity results specifically in increased deposition of asym. dimethylation of histone H4 arginine 3 (H4R3me2a) and in enhanced ribosomal-DNA silencing.  Consistent with this, H4 N-terminal acetylation impairs the activity of the Hmt1 methyltransferase towards H4R3 in vitro.  Furthermore, combinatorial loss of N-acH4 with internal histone acetylation at lysines 5, 8 and 12 has a synergistic induction of H4R3me2a deposition and rDNA silencing that leads to a severe growth defect.  This defect is completely rescued by mutating arginine 3 to lysine (H4R3K), suggesting that abnormal deposition of a single histone modification, H4R3me2a, can impact on cell growth.  Notably, the cross-talk between N-acH4 and H4R3me2a, which regulates rDNA silencing, is induced under calorie restriction conditions.  Collectively, these findings unveil a mol. and biol. function for H4 N-terminal acetylation, identify its interplay with internal histone modifications, and provide general mechanistic implications for N-alpha-terminal acetylation, one of the most common protein modifications in eukaryotes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhb1VM3trLc7Vg90H21EOLACvtfcHk0lglLnMoYjG9tQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFKitbrL&md5=d0b5fe7abb3fc3eba1d6c9a39132e7e8</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pgen.1003805&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pgen.1003805%26sid%3Dliteratum%253Aachs%26aulast%3DSchiza%26aufirst%3DV.%26aulast%3DMolina-Serrano%26aufirst%3DD.%26aulast%3DKyriakou%26aufirst%3DD.%26aulast%3DHadjiantoniou%26aufirst%3DA.%26aulast%3DKirmizis%26aufirst%3DA.%26atitle%3DN-Alpha-Terminal%2520Acetylation%2520of%2520Histone%2520H4%2520Regulates%2520Arginine%2520Methylation%2520and%2520Ribosomal%2520DNA%2520Silencing%26jtitle%3DPLoS%2520Genet.%26date%3D2013%26volume%3D9%26doi%3D10.1371%2Fjournal.pgen.1003805" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ju, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gui, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kvansakul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, D. C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span> <span> </span><span class="NLM_article-title">NatD Promotes Lung Cancer Progression by Preventing Histone H4 Serine Phosphorylation to Activate Slug Expression</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">14</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-00988-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1038%2Fs41467-017-00988-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=28232747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtV2ntrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1-14&author=J.+Juauthor=A.+Chenauthor=Y.+Dengauthor=M.+Liuauthor=Y.+Wangauthor=Y.+Wangauthor=M.+Nieauthor=C.+Wangauthor=H.+Dingauthor=B.+Yaoauthor=T.+Guiauthor=X.+Liauthor=Z.+Xuauthor=C.+Maauthor=Y.+Songauthor=M.+Kvansakulauthor=K.+Zenauthor=C.+Y.+Zhangauthor=C.+Luoauthor=M.+Fangauthor=D.+C.+S.+Huangauthor=C.+D.+Allisauthor=R.+Tanauthor=C.+K.+Zengauthor=J.+Weiauthor=Q.+Zhao&title=NatD+Promotes+Lung+Cancer+Progression+by+Preventing+Histone+H4+Serine+Phosphorylation+to+Activate+Slug+Expression&doi=10.1038%2Fs41467-017-00988-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">ORAI channels are critical for receptor-mediated endocytosis of albumin</span></div><div class="casAuthors">Zeng, Bo; Chen, Gui-Lan; Garcia-Vaz, Eliana; Bhandari, Sunil; Daskoulidou, Nikoleta; Berglund, Lisa M.; Jiang, Hongni; Hallett, Thomas; Zhou, Lu-Ping; Huang, Li; Xu, Zi-Hao; Nair, Viji; Nelson, Robert G.; Ju, Wenjun; Kretzler, Matthias; Atkin, Stephen L.; Gomez, Maria F.; Xu, Shang-Zhong</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-12</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Impaired albumin reabsorption by proximal tubular epithelial cells (PTECs) has been highlighted in diabetic nephropathy (DN), but little is known about the underlying mol. mechanisms.  Here we find that ORAI1-3, are preferentially expressed in PTECs and downregulated in patients with DN.  Hyperglycemia or blockade of insulin signaling reduces the expression of ORAI1-3.  Inhibition of ORAI channels by BTP2 and diethylstilbestrol or silencing of ORAI expression impairs albumin uptake.  Transgenic mice expressing a dominant-neg. Orai1 mutant (E108Q) increases albuminuria, and in vivo injection of BTP2 exacerbates albuminuria in streptozotocin-induced and Akita diabetic mice.  The albumin endocytosis is Ca2+-dependent and accompanied by ORAI1 internalization.  Amnionless (AMN) assocs. with ORAIs and forms STIM/ORAI/AMN complexes after Ca2+ store depletion.  STIM1/ORAI1 colocalizes with clathrin, but not with caveolin, at the apical membrane of PTECs, which dets. clathrin-mediated endocytosis.  These findings provide insights into the mechanisms of protein reabsorption and potential targets for treating diabetic proteinuria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvu0IP8NBiH7Vg90H21EOLACvtfcHk0ljMw0_zlxsNKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtV2ntrrJ&md5=ef4e199dafb36bc703a8894408936db5</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-00988-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-00988-5%26sid%3Dliteratum%253Aachs%26aulast%3DJu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DA.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DNie%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DYao%26aufirst%3DB.%26aulast%3DGui%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DMa%26aufirst%3DC.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DKvansakul%26aufirst%3DM.%26aulast%3DZen%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DC.%2BY.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DFang%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DD.%2BC.%2BS.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DTan%26aufirst%3DR.%26aulast%3DZeng%26aufirst%3DC.%2BK.%26aulast%3DWei%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DQ.%26atitle%3DNatD%2520Promotes%2520Lung%2520Cancer%2520Progression%2520by%2520Preventing%2520Histone%2520H4%2520Serine%2520Phosphorylation%2520to%2520Activate%2520Slug%2520Expression%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D1%26epage%3D14%26doi%3D10.1038%2Fs41467-017-00988-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liszczak, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foyn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petersson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnesen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span> <span> </span><span class="NLM_article-title">Molecular Basis for N-Terminal Acetylation by the Heterodimeric NatA Complex</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1098</span>, <span class="refDoi"> DOI: 10.1038/nsmb.2636</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1038%2Fnsmb.2636" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=23912279" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1WhtLvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=1098&author=G.+Liszczakauthor=J.+M.+Goldbergauthor=H.+Foynauthor=E.+J.+Peterssonauthor=T.+Arnesenauthor=R.+Marmorstein&title=Molecular+Basis+for+N-Terminal+Acetylation+by+the+Heterodimeric+NatA+Complex&doi=10.1038%2Fnsmb.2636"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis for N-terminal acetylation by the heterodimeric NatA complex</span></div><div class="casAuthors">Liszczak, Glen; Goldberg, Jacob M.; Foyn, Havard; Petersson, E. James; Arnesen, Thomas; Marmorstein, Ronen</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1098-1105</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">N-terminal acetylation is ubiquitous among eukaryotic proteins and controls a myriad of biol. processes.  Of the N-terminal acetyltransferases (NATs) that facilitate this cotranslational modification, the heterodimeric NatA complex has the most diversity for substrate selection and modifies the majority of all N-terminally acetylated proteins.  Here, the authors report the x-ray crystal structure of the 100-kDa holo-NatA complex from Schizosaccharomyces pombe, in the absence and presence of a bisubstrate peptide-CoA-conjugate inhibitor, as well as the structure of the uncomplexed Naa10p catalytic subunit.  The NatA-Naa15p auxiliary subunit contains 13 tetratricopeptide motifs and adopts a ring-like topol. that wraps around the NatA-Naa10p subunit, an interaction that alters the Naa10p active site for substrate-specific acetylation.  These studies have implications for understanding the mechanistic details of other NAT complexes and how regulatory subunits modulate the activity of the broader family of protein acetyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrrjoCe0eahrVg90H21EOLACvtfcHk0ljMw0_zlxsNKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1WhtLvJ&md5=d95329e33f31932afaddcb81e491c9cc</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2636&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2636%26sid%3Dliteratum%253Aachs%26aulast%3DLiszczak%26aufirst%3DG.%26aulast%3DGoldberg%26aufirst%3DJ.%2BM.%26aulast%3DFoyn%26aufirst%3DH.%26aulast%3DPetersson%26aufirst%3DE.%2BJ.%26aulast%3DArnesen%26aufirst%3DT.%26aulast%3DMarmorstein%26aufirst%3DR.%26atitle%3DMolecular%2520Basis%2520for%2520N-Terminal%2520Acetylation%2520by%2520the%2520Heterodimeric%2520NatA%2520Complex%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D1098%26doi%3D10.1038%2Fnsmb.2636" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evjenth, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenner, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnesen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frøystein, N. Å.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lillehaug, J. R.</span></span> <span> </span><span class="NLM_article-title">Human Protein N-Terminal Acetyltransferase HNaa50p (HNAT5/HSAN) Follows Ordered Sequential Catalytic Mechanism: Combined Kinetic and NMR Study</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">10081</span>– <span class="NLM_lpage">10088</span>, <span class="refDoi"> DOI: 10.1074/jbc.M111.326587</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1074%2Fjbc.M111.326587" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=22311970" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC38XktlKku7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=10081-10088&author=R.+H.+Evjenthauthor=A.+K.+Brennerauthor=P.+R.+Thompsonauthor=T.+Arnesenauthor=N.+%C3%85.+Fr%C3%B8ysteinauthor=J.+R.+Lillehaug&title=Human+Protein+N-Terminal+Acetyltransferase+HNaa50p+%28HNAT5%2FHSAN%29+Follows+Ordered+Sequential+Catalytic+Mechanism%3A+Combined+Kinetic+and+NMR+Study&doi=10.1074%2Fjbc.M111.326587"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Human Protein N-terminal Acetyltransferase hNaa50p (hNAT5/hSAN) Follows Ordered Sequential Catalytic Mechanism: Combined Kinetic and NMR Study</span></div><div class="casAuthors">Evjenth, Rune H.; Brenner, Annette K.; Thompson, Paul R.; Arnesen, Thomas; Froystein, Nils Aage; Lillehaug, Johan R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">10081-10088</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Nα-Acetylation is a common protein modification catalyzed by different N-terminal acetyltransferases (NATs).  Their essential role in the biogenesis and degrdn. of proteins is becoming increasingly evident.  The NAT hNaa50p preferentially modifies peptides starting with methionine followed by a hydrophobic amino acid. hNaa50p also possesses Nε-autoacetylation activity.  So far, no eukaryotic NAT has been mechanistically investigated.  In this study, we used NMR spectroscopy, bisubstrate kinetic assays, and product inhibition expts. to demonstrate that hNaa50p utilizes an ordered Bi Bi reaction of the Theorell-Chance type.  The NMR results, both the substrate binding study and the dynamic data, further indicate that the binding of acetyl-CoA induces a conformational change that is required for the peptide to bind to the active site.  In support of an ordered Bi Bi reaction mechanism, addn. of peptide in the absence of acetyl-CoA did not alter the structure of the protein.  This model is further strengthened by the NMR results using a catalytically inactive hNaa50p mutant.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojMOeRuLDeibVg90H21EOLACvtfcHk0ljiUqRKXHYYow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XktlKku7w%253D&md5=15290fa41ab6d9589f8c47a913960207</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.326587&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.326587%26sid%3Dliteratum%253Aachs%26aulast%3DEvjenth%26aufirst%3DR.%2BH.%26aulast%3DBrenner%26aufirst%3DA.%2BK.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26aulast%3DArnesen%26aufirst%3DT.%26aulast%3DFr%25C3%25B8ystein%26aufirst%3DN.%2B%25C3%2585.%26aulast%3DLillehaug%26aufirst%3DJ.%2BR.%26atitle%3DHuman%2520Protein%2520N-Terminal%2520Acetyltransferase%2520HNaa50p%2520%2528HNAT5%252FHSAN%2529%2520Follows%2520Ordered%2520Sequential%2520Catalytic%2520Mechanism%253A%2520Combined%2520Kinetic%2520and%2520NMR%2520Study%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D10081%26epage%3D10088%26doi%3D10.1074%2Fjbc.M111.326587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weyer, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gumiero, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lapouge, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bange, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopp, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinning, I.</span></span> <span> </span><span class="NLM_article-title">Structural Basis of HypK Regulating N-Terminal Acetylation by the NatA Complex</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">10</span>, <span class="refDoi"> DOI: 10.1038/ncomms15726</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1038%2Fncomms15726" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=28232747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC1cXptVCmsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1-10&author=F.+A.+Weyerauthor=A.+Gumieroauthor=K.+Lapougeauthor=G.+Bangeauthor=J.+Koppauthor=I.+Sinning&title=Structural+Basis+of+HypK+Regulating+N-Terminal+Acetylation+by+the+NatA+Complex&doi=10.1038%2Fncomms15726"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Heparan sulfate proteoglycans present PCSK9 to the LDL receptor</span></div><div class="casAuthors">Gustafsen, Camilla; Olsen, Ditte; Vilstrup, Joachim; Lund, Signe; Reinhardt, Anika; Wellner, Niels; Larsen, Torben; Andersen, Christian B. F.; Weyer, Kathrin; Li, Jin-ping; Seeberger, Peter H.; Thirup, Soeren; Madsen, Peder; Glerup, Simon</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-14</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Coronary artery disease is the main cause of death worldwide and accelerated by increased plasma levels of cholesterol-rich low-d. lipoprotein particles (LDL).  Circulating PCSK9 contributes to coronary artery disease by inducing lysosomal degrdn. of the LDL receptor (LDLR) in the liver and thereby reducing LDL clearance.  Here, we show that liver heparan sulfate proteoglycans are PCSK9 receptors and essential for PCSK9-induced LDLR degrdn.  The heparan sulfate-binding site is located in the PCSK9 prodomain and formed by surface-exposed basic residues interacting with trisulfated heparan sulfate disaccharide repeats.  Accordingly, heparan sulfate mimetics and monoclonal antibodies directed against the heparan sulfate-binding site are potent PCSK9 inhibitors.  We propose that heparan sulfate proteoglycans lining the hepatocyte surface capture PCSK9 and facilitates subsequent PCSK9:LDLR complex formation.  Our findings provide new insights into LDL biol. and show that targeting PCSK9 using heparan sulfate mimetics is a potential therapeutic strategy in coronary artery disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKbZYmSf5-8rVg90H21EOLACvtfcHk0ljiUqRKXHYYow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXptVCmsLk%253D&md5=bf1a00ff0a960d5f23c8c3f4e87a3e1e</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fncomms15726&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms15726%26sid%3Dliteratum%253Aachs%26aulast%3DWeyer%26aufirst%3DF.%2BA.%26aulast%3DGumiero%26aufirst%3DA.%26aulast%3DLapouge%26aufirst%3DK.%26aulast%3DBange%26aufirst%3DG.%26aulast%3DKopp%26aufirst%3DJ.%26aulast%3DSinning%26aufirst%3DI.%26atitle%3DStructural%2520Basis%2520of%2520HypK%2520Regulating%2520N-Terminal%2520Acetylation%2520by%2520the%2520NatA%2520Complex%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D1%26epage%3D10%26doi%3D10.1038%2Fncomms15726" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gottlieb, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petersson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span> <span> </span><span class="NLM_article-title">Molecular Basis for N-Terminal Alpha-Synuclein Acetylation by Human Natb</span>. <i>Elife</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">e57491</span> <span class="refDoi"> DOI: 10.7554/eLife.57491</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.7554%2FeLife.57491" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=32885784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisFKms77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2020&author=S.+Dengauthor=B.+Panauthor=L.+Gottliebauthor=E.+J.+Peterssonauthor=R.+Marmorstein&title=Molecular+Basis+for+N-Terminal+Alpha-Synuclein+Acetylation+by+Human+Natb&doi=10.7554%2FeLife.57491"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis for N-terminal alpha-synuclein acetylation by human NatB</span></div><div class="casAuthors">Deng, Sunbin; Pan, Buyan; Gottlieb, Leah; James Petersson, E.; Marmorstein, Ronen</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e57491</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">NatB is one of three major N-terminal acetyltransferase (NAT) complexes (NatA-NatC), which co-translationally acetylate the N-termini of eukaryotic proteins.  Its substrates account for about 21% of the human proteome, including well known proteins such as actin, tropomyosin, CDK2, and α-synuclein (αSyn).  Human NatB (hNatB) mediated N-terminal acetylation of αSyn has been demonstrated to play key roles in the pathogenesis of Parkinson's disease and as a potential therapeutic target for hepatocellular carcinoma.  Here we report the cryo-EM structure of hNatB bound to a CoA-αSyn conjugate, together with structure-guided anal. of mutational effects on catalysis.  This anal. reveals functionally important differences with human NatA and Candida albicans NatB, resolves key hNatB protein determinants for αSyn N-terminal acetylation, and identifies important residues for substrate-specific recognition and acetylation by NatB enzymes.  These studies have implications for developing small mol. NatB probes and for understanding the mode of substrate selection by NAT enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNXZN29IscJbVg90H21EOLACvtfcHk0ljiUqRKXHYYow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisFKms77F&md5=1047ab779ce9685e625367de6e3dd18e</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.7554%2FeLife.57491&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.57491%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DS.%26aulast%3DPan%26aufirst%3DB.%26aulast%3DGottlieb%26aufirst%3DL.%26aulast%3DPetersson%26aufirst%3DE.%2BJ.%26aulast%3DMarmorstein%26aufirst%3DR.%26atitle%3DMolecular%2520Basis%2520for%2520N-Terminal%2520Alpha-Synuclein%2520Acetylation%2520by%2520Human%2520Natb%26jtitle%3DElife%26date%3D2020%26volume%3D9%26doi%3D10.7554%2FeLife.57491" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Støve, S. I. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magin, R. S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foyn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haug, B. E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnesen, T.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of the Golgi-Associated Human Nα-Acetyltransferase 60 Reveals the Molecular Determinants for Substrate-Specific Acetylation</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1044</span>– <span class="NLM_lpage">1056</span>, <span class="refDoi"> DOI: 10.1016/j.str.2016.04.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1016%2Fj.str.2016.04.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=27320834" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A280%3ADC%252BC2s7hsFOrsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=1044-1056&author=S.+I.+I.+St%C3%B8veauthor=R.+S.+S.+Maginauthor=H.+Foynauthor=B.+E.+E.+Haugauthor=R.+Marmorsteinauthor=T.+Arnesen&title=Crystal+Structure+of+the+Golgi-Associated+Human+N%CE%B1-Acetyltransferase+60+Reveals+the+Molecular+Determinants+for+Substrate-Specific+Acetylation&doi=10.1016%2Fj.str.2016.04.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure of the Golgi-Associated Human Nα-Acetyltransferase 60 Reveals the Molecular Determinants for Substrate-Specific Acetylation</span></div><div class="casAuthors">Stove Svein Isungset; Magin Robert S; Foyn Havard; Haug Bengt Erik; Marmorstein Ronen; Arnesen Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Structure (London, England : 1993)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1044-56</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">N-Terminal acetylation is a common and important protein modification catalyzed by N-terminal acetyltransferases (NATs).  Six human NATs (NatA-NatF) contain one catalytic subunit each, Naa10 to Naa60, respectively.  In contrast to the ribosome-associated NatA to NatE, NatF/Naa60 specifically associates with Golgi membranes and acetylates transmembrane proteins.  To gain insight into the molecular basis for the function of Naa60, we developed an Naa60 bisubstrate CoA-peptide conjugate inhibitor, determined its X-ray structure when bound to CoA and inhibitor, and carried out biochemical experiments.  We show that Naa60 adapts an overall fold similar to that of the catalytic subunits of ribosome-associated NATs, but with the addition of two novel elongated loops that play important roles in substrate-specific binding.  One of these loops mediates a dimer to monomer transition upon substrate-specific binding.  Naa60 employs a catalytic mechanism most similar to Naa50.  Collectively, these data reveal the molecular basis for Naa60-specific acetyltransferase activity with implications for its Golgi-specific functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS6VBO9-n6Q1WhHGZei4OW9fW6udTcc2eaZcDRJmgBwZrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s7hsFOrsA%253D%253D&md5=f7e2b98f1d865fe8b8ca3bea30ff62f8</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2016.04.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2016.04.020%26sid%3Dliteratum%253Aachs%26aulast%3DSt%25C3%25B8ve%26aufirst%3DS.%2BI.%2BI.%26aulast%3DMagin%26aufirst%3DR.%2BS.%2BS.%26aulast%3DFoyn%26aufirst%3DH.%26aulast%3DHaug%26aufirst%3DB.%2BE.%2BE.%26aulast%3DMarmorstein%26aufirst%3DR.%26aulast%3DArnesen%26aufirst%3DT.%26atitle%3DCrystal%2520Structure%2520of%2520the%2520Golgi-Associated%2520Human%2520N%25CE%25B1-Acetyltransferase%252060%2520Reveals%2520the%2520Molecular%2520Determinants%2520for%2520Substrate-Specific%2520Acetylation%26jtitle%3DStructure%26date%3D2016%26volume%3D24%26spage%3D1044%26epage%3D1056%26doi%3D10.1016%2Fj.str.2016.04.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goris, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magin, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foyn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myklebust, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varland, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ree, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drazic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhambra, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Støve, S. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haug, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnesen, T.</span></span> <span> </span><span class="NLM_article-title">Structural Determinants and Cellular Environment Define Processed Actin as the Sole Substrate of the N-Terminal Acetyltransferase NAA80</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">4405</span>– <span class="NLM_lpage">4410</span>, <span class="refDoi"> DOI: 10.1073/pnas.1719251115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1073%2Fpnas.1719251115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=29581307" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVKjtrnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2018&pages=4405-4410&author=M.+Gorisauthor=R.+S.+Maginauthor=H.+Foynauthor=L.+M.+Myklebustauthor=S.+Varlandauthor=R.+Reeauthor=A.+Drazicauthor=P.+Bhambraauthor=S.+I.+St%C3%B8veauthor=M.+Baumannauthor=B.+E.+Haugauthor=R.+Marmorsteinauthor=T.+Arnesen&title=Structural+Determinants+and+Cellular+Environment+Define+Processed+Actin+as+the+Sole+Substrate+of+the+N-Terminal+Acetyltransferase+NAA80&doi=10.1073%2Fpnas.1719251115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Structural determinants and cellular environment define processed actin as the sole substrate of the N-terminal acetyltransferase NAA80</span></div><div class="casAuthors">Goris, Marianne; Magin, Robert S.; Foyn, Haavard; Myklebust, Line M.; Varland, Sylvia; Ree, Rasmus; Drazic, Adrian; Bhambra, Parminder; Stoeve, Svein I.; Baumann, Markus; Haug, Bengt Erik; Marmorstein, Ronen; Arnesen, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4405-4410</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">N-terminal (Nt) acetylation is a major protein modification catalyzed by N-terminal acetyltransferases (NATs).  Methionine acidic N termini, including actin, are cotranslationally Nt acetylated by NatB in all eukaryotes, but animal actins contg. acidic N termini, are addnl. post-translationally Nt acetylated by NAA80.  Actin Nt acetylation was found to regulate cytoskeletal dynamics and motility, thus making NAA80 a potential target for cell migration regulation.  In this work, we developed potent and selective bisubstrate inhibitors for NAA80 and detd. the crystal structure of NAA80 in complex with such an inhibitor, revealing that NAA80 adopts a fold similar to other NAT enzymes but with a more open substrate binding region.  Furthermore, in contrast to most other NATs, the substrate specificity of NAA80 is mainly derived through interactions between the enzyme and the acidic amino acids at positions 2 and 3 of the actin substrate and not residues 1 and 2.  A yeast model revealed that ectopic expression of NAA80 in a strain lacking NatB activity partially restored Nt acetylation of NatB substrates, including yeast actin.  Thus, NAA80 holds intrinsic capacity to posttranslationally Nt acetylate NatB-type substrates in vivo.  In sum, the presence of a dominant cotranslational NatB in all eukaryotes, the specific post-translational actin methionine removal in animals, and finally, the unique structural features of NAA80 leave only the processed actins as in vivo substrates of NAA80.  Together, this study reveals the mol. and cellular basis of NAA80 Nt acetylation and provides a scaffold for development of inhibitors for the regulation of cytoskeletal properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGVMeywrLUl7Vg90H21EOLACvtfcHk0ljOi05HFiPSAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVKjtrnJ&md5=3e3de1e9f289d208dd3eb2633335c15a</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1719251115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1719251115%26sid%3Dliteratum%253Aachs%26aulast%3DGoris%26aufirst%3DM.%26aulast%3DMagin%26aufirst%3DR.%2BS.%26aulast%3DFoyn%26aufirst%3DH.%26aulast%3DMyklebust%26aufirst%3DL.%2BM.%26aulast%3DVarland%26aufirst%3DS.%26aulast%3DRee%26aufirst%3DR.%26aulast%3DDrazic%26aufirst%3DA.%26aulast%3DBhambra%26aufirst%3DP.%26aulast%3DSt%25C3%25B8ve%26aufirst%3DS.%2BI.%26aulast%3DBaumann%26aufirst%3DM.%26aulast%3DHaug%26aufirst%3DB.%2BE.%26aulast%3DMarmorstein%26aufirst%3DR.%26aulast%3DArnesen%26aufirst%3DT.%26atitle%3DStructural%2520Determinants%2520and%2520Cellular%2520Environment%2520Define%2520Processed%2520Actin%2520as%2520the%2520Sole%2520Substrate%2520of%2520the%2520N-Terminal%2520Acetyltransferase%2520NAA80%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2018%26volume%3D115%26spage%3D4405%26epage%3D4410%26doi%3D10.1073%2Fpnas.1719251115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Foyn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewallen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narawane, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varhaug, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnesen, T.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Kinetic Characterization of Protein N-Terminal Acetyltransferase Inhibitors</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1121</span>– <span class="NLM_lpage">1127</span>, <span class="refDoi"> DOI: 10.1021/cb400136s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb400136s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlt1Whsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1121-1127&author=H.+Foynauthor=J.+E.+Jonesauthor=D.+Lewallenauthor=R.+Narawaneauthor=J.+E.+Varhaugauthor=P.+R.+Thompsonauthor=T.+Arnesen&title=Design%2C+Synthesis%2C+and+Kinetic+Characterization+of+Protein+N-Terminal+Acetyltransferase+Inhibitors&doi=10.1021%2Fcb400136s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Kinetic Characterization of Protein N-Terminal Acetyltransferase Inhibitors</span></div><div class="casAuthors">Foyn, Haavard; Jones, Justin E.; Lewallen, Dan; Narawane, Rashmi; Varhaug, Jan Erik; Thompson, Paul R.; Arnesen, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1121-1127</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The N-termini of 80-90% of human proteins are acetylated by the N-terminal actransferases (NATs), NatA-NatF.  The major NAT complex, NatA, and particularly the catalytic subunit hNaa10 (ARD1) has been implicated in cancer development.  For example, knockdown of hNaa10 results in cancer cell death and the arrest of cell proliferation.  It also sensitized cancer cells to drug-induced cytotoxicity.  Human NatE has a distinct substrate specificity and is essential for normal chromosome segregation.  Thus, NAT inhibitors may potentially be valuable anticancer therapeutics, either directly or as adjuvants.  Herein, the authors report the design and synthesis of the first inhibitors targeting these enzymes.  Using the substrate specificity of the enzymes as a guide, the authors synthesized three bisubstrate analogs that potently and selectively inhibit the NatA complex (CoA-Ac-SES4; IC50 = 15.1 μM), hNaa10, the catalytic subunit of NatA (CoA-Ac-EEE4; Ki = 1.6 μM), and NatE/hNaa50 (CoA-Ac-MLG7; Ki* = 8 nM); CoA-Ac-EEE4 is a reversible competitive inhibitor of hNaa10, and CoA-Ac-MLG7 is a slow tight binding inhibitor of hNaa50.  The authors' demonstration that it is possible to develop NAT selective inhibitors should assist future efforts to develop NAT inhibitors with more drug-like properties that can be used to chem. interrogate in vivo NAT function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2gkDQnZDQarVg90H21EOLACvtfcHk0liZdpmN3_L06Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlt1Whsrg%253D&md5=4c84fb13c4bcb9b7b0fe3db0b69e5eef</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fcb400136s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb400136s%26sid%3Dliteratum%253Aachs%26aulast%3DFoyn%26aufirst%3DH.%26aulast%3DJones%26aufirst%3DJ.%2BE.%26aulast%3DLewallen%26aufirst%3DD.%26aulast%3DNarawane%26aufirst%3DR.%26aulast%3DVarhaug%26aufirst%3DJ.%2BE.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26aulast%3DArnesen%26aufirst%3DT.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Kinetic%2520Characterization%2520of%2520Protein%2520N-Terminal%2520Acetyltransferase%2520Inhibitors%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D1121%26epage%3D1127%26doi%3D10.1021%2Fcb400136s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noinaj, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Discovery of Bisubstrate Inhibitors for Protein N-Terminal Methyltransferase 1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">3773</span>– <span class="NLM_lpage">3779</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00206</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00206" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC1MXltFequ7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=3773-3779&author=D.+Chenauthor=G.+Dongauthor=N.+Noinajauthor=R.+Huang&title=Discovery+of+Bisubstrate+Inhibitors+for+Protein+N-Terminal+Methyltransferase+1&doi=10.1021%2Facs.jmedchem.9b00206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of bisubstrate inhibitors for protein N-terminal methyltransferase 1</span></div><div class="casAuthors">Chen, Dongxing; Dong, Guangping; Noinaj, Nicholas; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3773-3779</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein N-terminal methyltransferase 1 (NTMT1) plays an important role in regulating mitosis and DNA repair.  Here, we describe the discovery of a potent NTMT1 bisubstrate inhibitor 4 (IC50 = 35 ± 2 nM) that exhibits greater than 100-fold selectivity against a panel of methyltransferases.  We also report the first crystal structure of NTMT1 in complex with an inhibitor, which revealed that 4 occupies substrate and cofactor binding sites of NTMT1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqM1ZcSF10eY7Vg90H21EOLACvtfcHk0liZdpmN3_L06Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXltFequ7o%253D&md5=34782952c04423e217abf6cd3d228f94</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00206%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DD.%26aulast%3DDong%26aufirst%3DG.%26aulast%3DNoinaj%26aufirst%3DN.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DDiscovery%2520of%2520Bisubstrate%2520Inhibitors%2520for%2520Protein%2520N-Terminal%2520Methyltransferase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D3773%26epage%3D3779%26doi%3D10.1021%2Facs.jmedchem.9b00206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ho, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Development of a Continuous Fluorescence-Based Assay for n-Terminal Acetyltransferase D</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">12</span>, <span class="refDoi"> DOI: 10.3390/ijms22020594</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.3390%2Fijms22020594" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2021&pages=1-12&author=Y.+H.+Hoauthor=L.+Chenauthor=R.+Huang&title=Development+of+a+Continuous+Fluorescence-Based+Assay+for+n-Terminal+Acetyltransferase+D&doi=10.3390%2Fijms22020594"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.3390%2Fijms22020594&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms22020594%26sid%3Dliteratum%253Aachs%26aulast%3DHo%26aufirst%3DY.%2BH.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DDevelopment%2520of%2520a%2520Continuous%2520Fluorescence-Based%2520Assay%2520for%2520n-Terminal%2520Acetyltransferase%2520D%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2021%26volume%3D22%26spage%3D1%26epage%3D12%26doi%3D10.3390%2Fijms22020594" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <span> </span><span class="NLM_article-title">Tight Binding Inhibition</span>. In  <i>Evaluation of Enzyme Inhibitors in Drug Discovery: A Guide for Medicinal Chemists and Pharmacologists</i>; <span class="NLM_edition">2</span>nd Ed., <span class="NLM_publisher-name">John Wiley & Sons</span>: <span class="NLM_publisher-loc">New Jersey</span>, <span class="NLM_year">2013</span>,  <span class="NLM_fpage">245</span>– <span class="NLM_lpage">285</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1002%2F9781118540398.ch7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=245-285&author=R.+A.+Copeland&title=Evaluation+of+Enzyme+Inhibitors+in+Drug+Discovery%3A+A+Guide+for+Medicinal+Chemists+and+Pharmacologists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1002%2F9781118540398.ch7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781118540398.ch7%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DTight%2520Binding%2520Inhibition%26btitle%3DEvaluation%2520of%2520Enzyme%2520Inhibitors%2520in%2520Drug%2520Discovery%253A%2520A%2520Guide%2520for%2520Medicinal%2520Chemists%2520and%2520Pharmacologists%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%26date%3D2013%26spage%3D245%26epage%3D285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, J. F.</span></span> <span> </span><span class="NLM_article-title">The Slow-Binding and Slow, Tight-Binding Inhibition of Enzyme-Catalysed Reactions</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">102</span>– <span class="NLM_lpage">105</span>, <span class="refDoi"> DOI: 10.1016/0968-0004(82)90157-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1016%2F0968-0004%2882%2990157-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADyaL38XhslKrtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1982&pages=102-105&author=J.+F.+Morrison&title=The+Slow-Binding+and+Slow%2C+Tight-Binding+Inhibition+of+Enzyme-Catalysed+Reactions&doi=10.1016%2F0968-0004%2882%2990157-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The slow-binding and slow, tight-binding inhibition of enzyme-catalyzed reactions</span></div><div class="casAuthors">Morrison, John F.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">102-5</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    </div><div class="casAbstract">The mechanisms of enzyme inhibition by slow-binding and slow, tight-binding reversible inhibitors are discussed, as well as the kinetics of these processes as opposed to classical and tight-binding inhibitor mechanisms.  Examples of these types of inhibitors in the enzyme literature are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoun0ayjodbYrVg90H21EOLACvtfcHk0liPlx5yVYumWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL38XhslKrtrw%253D&md5=1740627c1ad2adf182f4c0ae355bb4de</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2F0968-0004%2882%2990157-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0968-0004%252882%252990157-8%26sid%3Dliteratum%253Aachs%26aulast%3DMorrison%26aufirst%3DJ.%2BF.%26atitle%3DThe%2520Slow-Binding%2520and%2520Slow%252C%2520Tight-Binding%2520Inhibition%2520of%2520Enzyme-Catalysed%2520Reactions%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D1982%26volume%3D7%26spage%3D102%26epage%3D105%26doi%3D10.1016%2F0968-0004%2882%2990157-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Khalil, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">A potent inhibitor of the melatonin rhythm enzyme</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">6195</span>– <span class="NLM_lpage">6196</span>, <span class="refDoi"> DOI: 10.1021/ja981365a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja981365a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADyaK1cXjslehsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=1998&pages=6195-6196&author=E.+M.+Khalilauthor=P.+A.+Cole&title=A+potent+inhibitor+of+the+melatonin+rhythm+enzyme&doi=10.1021%2Fja981365a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">A Potent Inhibitor of the Melatonin Rhythm Enzyme</span></div><div class="casAuthors">Khalil, Ehab M.; Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">6195-6196</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibitors of melatonin biosynthesis could not only help improve our understanding of circadian rhythm but they might also have therapeutic roles in mood and sleep disorders.  The key enzyme to be targeted in this regard is serotonin N-acetyltransferase (arylalkylamine N-acetyltransferase, AANAT), also called the melatonin rhythm enzyme.  This paper reports the synthesis and activity of the first potent and specific inhibitor of AANAT.  This inhibitor is a bisubstrate analog which contains covalently linked components of serotonin and acetyl-CoA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3jN1ADqSw8rVg90H21EOLACvtfcHk0liPlx5yVYumWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXjslehsbs%253D&md5=a7e66d9c02d47c5eb94b92c221dfa431</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fja981365a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja981365a%26sid%3Dliteratum%253Aachs%26aulast%3DKhalil%26aufirst%3DE.%2BM.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DA%2520potent%2520inhibitor%2520of%2520the%2520melatonin%2520rhythm%2520enzyme%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1998%26volume%3D120%26spage%3D6195%26epage%3D6196%26doi%3D10.1021%2Fja981365a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaudet, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, C. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molinaro, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kottwitz, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rohde, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dangeard, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrieumerlou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girardin, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray-Owen, S. D.</span></span> <span> </span><span class="NLM_article-title">Innate recognition of intracellular bacterial growth is driven by the TIFA-dependent cytosolic surveillance pathway</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1418</span>– <span class="NLM_lpage">1430</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2017.04.063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1016%2Fj.celrep.2017.04.063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=28514661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnvF2jsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2017&pages=1418-1430&author=R.+G.+Gaudetauthor=C.+X.+Guoauthor=R.+Molinaroauthor=H.+Kottwitzauthor=J.+R.+Rohdeauthor=A.+S.+Dangeardauthor=C.+Arrieumerlouauthor=S.+E.+Girardinauthor=S.+D.+Gray-Owen&title=Innate+recognition+of+intracellular+bacterial+growth+is+driven+by+the+TIFA-dependent+cytosolic+surveillance+pathway&doi=10.1016%2Fj.celrep.2017.04.063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Innate Recognition of Intracellular Bacterial Growth Is Driven by the TIFA-Dependent Cytosolic Surveillance Pathway</span></div><div class="casAuthors">Gaudet, Ryan G.; Guo, Cynthia X.; Molinaro, Raphael; Kottwitz, Haila; Rohde, John R.; Dangeard, Anne-Sophie; Arrieumerlou, Cecile; Girardin, Stephen E.; Gray-Owen, Scott D.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1418-1430</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Intestinal epithelial cells (IECs) act as sentinels for incoming pathogens.  Cytosol-invasive bacteria, such as Shigella flexneri, trigger a robust pro-inflammatory nuclear factor κB (NF-κB) response from IECs that is believed to depend entirely on the peptidoglycan sensor NOD1.  We found that, during Shigella infection, the TRAF-interacting forkhead-assocd. protein A (TIFA)-dependent cytosolic surveillance pathway, which senses the bacterial metabolite heptose-1,7-bisphosphate (HBP), functions after NOD1 to detect bacteria replicating free in the host cytosol.  Whereas NOD1 mediated a transient burst of NF-κB activation during bacterial entry, TIFA sensed HBP released during bacterial replication, assembling into large signaling complexes to drive a dynamic inflammatory response that reflected the rate of intracellular bacterial proliferation.  Strikingly, IECs lacking TIFA were unable to discriminate between proliferating and stagnant intracellular bacteria, despite the NOD1/2 pathways being intact.  Our results define TIFA as a rheostat for intracellular bacterial replication, escalating the immune response to invasive Gram-neg. bacteria that exploit the host cytosol for growth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofl1BqukOlLLVg90H21EOLACvtfcHk0liPlx5yVYumWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnvF2jsr4%253D&md5=3287157c992c75ba1dd93ac59c5a47ba</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.04.063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.04.063%26sid%3Dliteratum%253Aachs%26aulast%3DGaudet%26aufirst%3DR.%2BG.%26aulast%3DGuo%26aufirst%3DC.%2BX.%26aulast%3DMolinaro%26aufirst%3DR.%26aulast%3DKottwitz%26aufirst%3DH.%26aulast%3DRohde%26aufirst%3DJ.%2BR.%26aulast%3DDangeard%26aufirst%3DA.%2BS.%26aulast%3DArrieumerlou%26aufirst%3DC.%26aulast%3DGirardin%26aufirst%3DS.%2BE.%26aulast%3DGray-Owen%26aufirst%3DS.%2BD.%26atitle%3DInnate%2520recognition%2520of%2520intracellular%2520bacterial%2520growth%2520is%2520driven%2520by%2520the%2520TIFA-dependent%2520cytosolic%2520surveillance%2520pathway%26jtitle%3DCell%2520Rep.%26date%3D2017%26volume%3D19%26spage%3D1418%26epage%3D1430%26doi%3D10.1016%2Fj.celrep.2017.04.063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noinaj, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Structure-based discovery of cell-potent peptidomimetic inhibitors for protein N-terminal methyltransferase 1</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">485</span>– <span class="NLM_lpage">493</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.1c00012</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.1c00012" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BB3MXlt1Krtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2021&pages=485-493&author=D.+Chenauthor=G.+Dongauthor=Y.+Dengauthor=N.+Noinajauthor=R.+Huang&title=Structure-based+discovery+of+cell-potent+peptidomimetic+inhibitors+for+protein+N-terminal+methyltransferase+1&doi=10.1021%2Facsmedchemlett.1c00012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of cell-potent peptidomimetic inhibitors for protein N-terminal methyltransferase 1</span></div><div class="casAuthors">Chen, Dongxing; Dong, Guangping; Deng, Youchao; Noinaj, Nicholas; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">485-493</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein N-terminal methyltransferases (NTMTs) catalyze the methylation of the α-N-terminal amines of proteins starting with an X-P-K/R motif.  NTMT1 has been implicated in various cancers and in aging, implying its role as a potential therapeutic target.  Through structural modifications of a lead NTMT1 inhibitor, BM30, we designed and synthesized a diverse set of inhibitors to probe the NTMT1 active site.  The incorporation of a naphthyl group at the N-terminal region and an ortho-aminobenzoic amide at the C-terminal region of BM30 generates the top cell-potent inhibitor DC541, demonstrating increased activity on both purified NTMT1 (IC50 of 0.34 ± 0.02μM) and the cellular α-N-terminal methylation level of regulator of chromosome condensation 1 (RCC1, IC50 value of 30μM) in human colorectal cancer HT29 cells.  Furthermore, DC541 exhibits over 300-fold selectivity to several methyltransferases.  This study points out the direction for the development of more cell-potent inhibitors for NTMT1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrr2eVAYKlcULVg90H21EOLACvtfcHk0lgJJTXkuelojA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXlt1Krtr8%253D&md5=4ce642d17391e555deb2b65994b24794</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.1c00012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.1c00012%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DD.%26aulast%3DDong%26aufirst%3DG.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DNoinaj%26aufirst%3DN.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DStructure-based%2520discovery%2520of%2520cell-potent%2520peptidomimetic%2520inhibitors%2520for%2520protein%2520N-terminal%2520methyltransferase%25201%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2021%26volume%3D12%26spage%3D485%26epage%3D493%26doi%3D10.1021%2Facsmedchemlett.1c00012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jafari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almqvist, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axelsson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ignatushchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundback, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez
Molina, D.</span></span> <span> </span><span class="NLM_article-title">The cellular thermal shift assay for evaluating drug target interactions in cells</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2100</span>– <span class="NLM_lpage">2122</span>, <span class="refDoi"> DOI: 10.1038/nprot.2014.138</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1038%2Fnprot.2014.138" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=25101824" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12msrfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=2100-2122&author=R.+Jafariauthor=H.+Almqvistauthor=H.+Axelssonauthor=M.+Ignatushchenkoauthor=T.+Lundbackauthor=P.+Nordlundauthor=D.+Martinez%0AMolina&title=The+cellular+thermal+shift+assay+for+evaluating+drug+target+interactions+in+cells&doi=10.1038%2Fnprot.2014.138"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">The cellular thermal shift assay for evaluating drug target interactions in cells</span></div><div class="casAuthors">Jafari, Rozbeh; Almqvist, Helena; Axelsson, Hanna; Ignatushchenko, Marina; Lundbaeck, Thomas; Nordlund, Paer; Molina, Daniel Martinez</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2100-2122</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Thermal shift assays are used to study thermal stabilization of proteins upon ligand binding.  Such assays have been used extensively on purified proteins in the drug discovery industry and in academia to detect interactions.  Recently, we published a proof-of-principle study describing the implementation of thermal shift assays in a cellular format, which we call the cellular thermal shift assay (CETSA).  The method allows studies of target engagement of drug candidates in a cellular context, herein exemplified with exptl. data on the human kinases p38α and ERK1/2.  The assay involves treatment of cells with a compd. of interest, heating to denature and ppt. proteins, cell lysis, and the sepn. of cell debris and aggregates from the sol. protein fraction.  Whereas unbound proteins denature and ppt. at elevated temps., ligand-bound proteins remain in soln.  We describe two procedures for detecting the stabilized protein in the sol. fraction of the samples.  One approach involves sample workup and detection using quant. western blotting, whereas the second is performed directly in soln. and relies on the induced proximity of two target-directed antibodies upon binding to sol. protein.  The latter protocol has been optimized to allow an increased throughput, as potential applications require large nos. of samples.  Both approaches can be completed in a day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzarqomAK6eLVg90H21EOLACvtfcHk0lgJJTXkuelojA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12msrfJ&md5=e7852e1a306bca1698a8e8463ae566b1</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2014.138&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2014.138%26sid%3Dliteratum%253Aachs%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DAlmqvist%26aufirst%3DH.%26aulast%3DAxelsson%26aufirst%3DH.%26aulast%3DIgnatushchenko%26aufirst%3DM.%26aulast%3DLundback%26aufirst%3DT.%26aulast%3DNordlund%26aufirst%3DP.%26aulast%3DMartinez%2BMolina%26aufirst%3DD.%26atitle%3DThe%2520cellular%2520thermal%2520shift%2520assay%2520for%2520evaluating%2520drug%2520target%2520interactions%2520in%2520cells%26jtitle%3DNat.%2520Protoc.%26date%3D2014%26volume%3D9%26spage%3D2100%26epage%3D2122%26doi%3D10.1038%2Fnprot.2014.138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasband, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eliceiri, K. W.</span></span> <span> </span><span class="NLM_article-title">NIH Image to ImageJ: 25 years of image analysis</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">671</span>– <span class="NLM_lpage">675</span>, <span class="refDoi"> DOI: 10.1038/nmeth.2089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1038%2Fnmeth.2089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=22930834" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVKntb7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2012&pages=671-675&author=C.+A.+Schneiderauthor=W.+S.+Rasbandauthor=K.+W.+Eliceiri&title=NIH+Image+to+ImageJ%3A+25+years+of+image+analysis&doi=10.1038%2Fnmeth.2089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">NIH Image to ImageJ: 25 years of image analysis</span></div><div class="casAuthors">Schneider, Caroline A.; Rasband, Wayne S.; Eliceiri, Kevin W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7_part1</span>),
    <span class="NLM_cas:pages">671-675</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">For the past 25 years NIH Image and ImageJ software have been pioneers as open tools for the anal. of scientific images.  We discuss the origins, challenges and solns. of these two programs, and how their history can serve to advise and inform other software projects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofzHXrdLwDn7Vg90H21EOLACvtfcHk0lgJJTXkuelojA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVKntb7P&md5=85ab928cd79f1e2f2351c834c0c600f0</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.2089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.2089%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DC.%2BA.%26aulast%3DRasband%26aufirst%3DW.%2BS.%26aulast%3DEliceiri%26aufirst%3DK.%2BW.%26atitle%3DNIH%2520Image%2520to%2520ImageJ%253A%252025%2520years%2520of%2520image%2520analysis%26jtitle%3DNat.%2520Methods%26date%3D2012%26volume%3D9%26spage%3D671%26epage%3D675%26doi%3D10.1038%2Fnmeth.2089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Collins, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanier, G. S.</span></span> <span> </span><span class="NLM_article-title">High-Efficiency Solid Phase Peptide Synthesis (He-Spps)</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">940</span>– <span class="NLM_lpage">943</span>, <span class="refDoi"> DOI: 10.1021/ol4036825</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol4036825" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1eiu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2014&pages=940-943&author=J.+M.+Collinsauthor=K.+A.+Porterauthor=S.+K.+Singhauthor=G.+S.+Vanier&title=High-Efficiency+Solid+Phase+Peptide+Synthesis+%28He-Spps%29&doi=10.1021%2Fol4036825"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">High-Efficiency Solid Phase Peptide Synthesis (HE-SPPS)</span></div><div class="casAuthors">Collins, Jonathan M.; Porter, Keith A.; Singh, Sandeep K.; Vanier, Grace S.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">940-943</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of improvements to the std. solid-phase peptide synthesis (SPPS) process allowing for significant gains in product purity along with only a 4 min std. cycle time and a 90% redn. in total waste produced is reported.  For example, syntheses of the well-known 65-74acyl carrier protein (ACP) and 1-42β-amyloid peptides were accomplished with 93 and 72% purity (UPLC-MS) in only 44 and 229 min, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqY1pWgm1oVPLVg90H21EOLACvtfcHk0lj1UN-r_1ZOXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1eiu7o%253D&md5=4338beecbc9e347d93cbbdd8e4543094</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fol4036825&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol4036825%26sid%3Dliteratum%253Aachs%26aulast%3DCollins%26aufirst%3DJ.%2BM.%26aulast%3DPorter%26aufirst%3DK.%2BA.%26aulast%3DSingh%26aufirst%3DS.%2BK.%26aulast%3DVanier%26aufirst%3DG.%2BS.%26atitle%3DHigh-Efficiency%2520Solid%2520Phase%2520Peptide%2520Synthesis%2520%2528He-Spps%2529%26jtitle%3DOrg.%2520Lett.%26date%3D2014%26volume%3D16%26spage%3D940%26epage%3D943%26doi%3D10.1021%2Fol4036825" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gottlieb, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span> <span> </span><span class="NLM_article-title">Structure of Human NatA and Its Regulation by the Huntingtin Interacting Protein HYPK</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">925</span>– <span class="NLM_lpage">935.e8</span>, <span class="refDoi"> DOI: 10.1016/j.str.2018.04.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1016%2Fj.str.2018.04.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=29754825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC1cXptlyrsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2018&pages=925-935.e8&author=L.+Gottliebauthor=R.+Marmorstein&title=Structure+of+Human+NatA+and+Its+Regulation+by+the+Huntingtin+Interacting+Protein+HYPK&doi=10.1016%2Fj.str.2018.04.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of Human NatA and Its Regulation by the Huntingtin Interacting Protein HYPK</span></div><div class="casAuthors">Gottlieb, Leah; Marmorstein, Ronen</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">925-935.e8</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Co-translational N-terminal protein acetylation regulates many protein functions including degrdn., folding, interprotein interactions, and targeting.  Human NatA (hNatA), one of six conserved metazoan N-terminal acetyltransferases, contains Naa10 catalytic and Naa15 auxiliary subunits, and assocs. with the intrinsically disordered Huntingtin yeast two-hybrid protein K (HYPK).  We report on the crystal structures of hNatA and hNatA/HYPK, and assocd. biochem. and enzymic analyses.  We demonstrate that hNatA contains unique features: a stabilizing inositol hexaphosphate (IP6) mol. and a metazoan-specific Naa15 domain that mediates high-affinity HYPK binding.  We find that HYPK harbors intrinsic hNatA-specific inhibitory activity through a bipartite structure: a ubiquitin-assocd. domain that binds a hNaa15 metazoan-specific region and an N-terminal loop-helix region that distorts the hNaa10 active site.  We show that HYPK binding blocks hNaa50 targeting to hNatA, likely limiting Naa50 ribosome localization in vivo.  These studies provide a model for metazoan NAT activity and HYPK regulation of N-terminal acetylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4MVEFDfG3brVg90H21EOLACvtfcHk0lj1UN-r_1ZOXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXptlyrsb0%253D&md5=64af1f850cc8f207c5f0fa17ed804a5d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2018.04.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2018.04.003%26sid%3Dliteratum%253Aachs%26aulast%3DGottlieb%26aufirst%3DL.%26aulast%3DMarmorstein%26aufirst%3DR.%26atitle%3DStructure%2520of%2520Human%2520NatA%2520and%2520Its%2520Regulation%2520by%2520the%2520Huntingtin%2520Interacting%2520Protein%2520HYPK%26jtitle%3DStructure%26date%3D2018%26volume%3D26%26spage%3D925%26epage%3D935.e8%26doi%3D10.1016%2Fj.str.2018.04.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magin, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petersson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span> <span> </span><span class="NLM_article-title">Structure and Mechanism of Acetylation by the N-Terminal Dual Enzyme NatA/Naa50 Complex</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1057</span>– <span class="NLM_lpage">1070.e4</span>, <span class="refDoi"> DOI: 10.1016/j.str.2019.04.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1016%2Fj.str.2019.04.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=31155310" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVyju7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2019&pages=1057-1070.e4&author=S.+Dengauthor=R.+S.+Maginauthor=X.+Weiauthor=B.+Panauthor=E.+J.+Peterssonauthor=R.+Marmorstein&title=Structure+and+Mechanism+of+Acetylation+by+the+N-Terminal+Dual+Enzyme+NatA%2FNaa50+Complex&doi=10.1016%2Fj.str.2019.04.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and Mechanism of Acetylation by the N-Terminal Dual Enzyme NatA/Naa50 Complex</span></div><div class="casAuthors">Deng, Sunbin; Magin, Robert S.; Wei, Xuepeng; Pan, Buyan; Petersson, E. James; Marmorstein, Ronen</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1057-1070.e4</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">NatA co-translationally acetylates the N termini of over 40% of eukaryotic proteins and can assoc. with another catalytic subunit, Naa50, to form a ternary NatA/Naa50 dual enzyme complex (also called NatE).  The mol. basis of assocn. between Naa50 and NatA and the mechanism for how their assocn. affects their catalytic activities in yeast and human are poorly understood.  Here, we detd. the X-ray crystal structure of yeast NatA/Naa50 as a scaffold to understand coregulation of NatA/Naa50 activity in both yeast and human.  We find that Naa50 makes evolutionarily conserved contacts to both the Naa10 and Naa15 subunits of NatA.  These interactions promote catalytic crosstalk within the human complex, but do so to a lesser extent in the yeast complex, where Naa50 activity is compromised.  These studies have implications for understanding the role of the NatA/Naa50 complex in modulating the majority of the N-terminal acetylome in diverse species.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2COdqCfNww7Vg90H21EOLACvtfcHk0lj1UN-r_1ZOXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVyju7fL&md5=0320692226fac3f4148f4d409f699187</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2019.04.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2019.04.014%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DS.%26aulast%3DMagin%26aufirst%3DR.%2BS.%26aulast%3DWei%26aufirst%3DX.%26aulast%3DPan%26aufirst%3DB.%26aulast%3DPetersson%26aufirst%3DE.%2BJ.%26aulast%3DMarmorstein%26aufirst%3DR.%26atitle%3DStructure%2520and%2520Mechanism%2520of%2520Acetylation%2520by%2520the%2520N-Terminal%2520Dual%2520Enzyme%2520NatA%252FNaa50%2520Complex%26jtitle%3DStructure%26date%3D2019%26volume%3D27%26spage%3D1057%26epage%3D1070.e4%26doi%3D10.1016%2Fj.str.2019.04.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McTiernan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnesen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span> <span> </span><span class="NLM_article-title">Molecular Basis for N-Terminal Acetylation by Human NatE and Its Modulation by HYPK</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">14</span>, <span class="refDoi"> DOI: 10.1038/s41467-020-14584-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1038%2Fs41467-020-14584-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=31911652" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=1-14&author=S.+Dengauthor=N.+McTiernanauthor=X.+Weiauthor=T.+Arnesenauthor=R.+Marmorstein&title=Molecular+Basis+for+N-Terminal+Acetylation+by+Human+NatE+and+Its+Modulation+by+HYPK&doi=10.1038%2Fs41467-020-14584-7"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fs41467-020-14584-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-020-14584-7%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DS.%26aulast%3DMcTiernan%26aufirst%3DN.%26aulast%3DWei%26aufirst%3DX.%26aulast%3DArnesen%26aufirst%3DT.%26aulast%3DMarmorstein%26aufirst%3DR.%26atitle%3DMolecular%2520Basis%2520for%2520N-Terminal%2520Acetylation%2520by%2520Human%2520NatE%2520and%2520Its%2520Modulation%2520by%2520HYPK%26jtitle%3DNat.%2520Commun.%26date%3D2020%26volume%3D11%26spage%3D1%26epage%3D14%26doi%3D10.1038%2Fs41467-020-14584-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCullough, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, F. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span> <span> </span><span class="NLM_article-title">Structural and Functional Role of Acetyltransferase HMOF K274 Autoacetylation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>291</i></span>,  <span class="NLM_fpage">18190</span>– <span class="NLM_lpage">18198</span>, <span class="refDoi"> DOI: 10.1074/jbc.M116.736264</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1074%2Fjbc.M116.736264" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=27382063" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVWjtrfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2016&pages=18190-18198&author=C.+E.+McCulloughauthor=S.+Songauthor=M.+H.+Shinauthor=F.+B.+Johnsonauthor=R.+Marmorstein&title=Structural+and+Functional+Role+of+Acetyltransferase+HMOF+K274+Autoacetylation&doi=10.1074%2Fjbc.M116.736264"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and Functional Role of Acetyltransferase hMOF K274 Autoacetylation</span></div><div class="casAuthors">McCullough, Cheryl E.; Song, Shufei; Shin, Michael H.; Johnson, F. Brad; Marmorstein, Ronen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">291</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">18190-18198</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Many histone acetyltransferases undergo autoacetylation, either through chem. or enzymic means, to potentiate enzymic cognate substrate lysine acetylation, although the mode and mol. role of such autoacetylation is poorly understood.  The MYST family of histone acetyltransferases is autoacetylated at an active site lysine residue to facilitate cognate substrate lysine binding and acetylation.  Here, we report on a detailed mol. investigation of Lys-274 autoacetylation of the human MYST protein Males Absent on the First (hMOF).  A mutational scan of hMOF Lys-274 reveals that all amino acid substitutions of this residue are able to bind cofactor but are significantly destabilized, both in vitro and in cells, and are catalytically inactive for cognate histone H4 peptide lysine acetylation.  The x-ray crystal structure of a hMOF K274P mutant suggests that the reduced stability and catalytic activity stems from a disordering of the residue 274-harboring a α2-β7 loop.  We also provide structural evidence that a C316S/E350Q mutant, which is defective for cognate substrate lysine acetylation; and biochem. evidence that a K268M mutant, which is defective for Lys-274 chem. acetylation in the context of a K274-peptide, can still undergo quant. K274 autoacetylation.  Together, these studies point to the crit. and specific role of hMOF Lys-274 autoacetylation in hMOF stability and cognate substrate acetylation and argues that binding of Ac-CoA to hMOF likely drives Lys-274 autoacetylation for subsequent cognate substrate acetylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJlS94p61LA7Vg90H21EOLACvtfcHk0li9CqUTTSA40A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVWjtrfP&md5=08cdceca89e76d9a44389fc3022380b8</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.736264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.736264%26sid%3Dliteratum%253Aachs%26aulast%3DMcCullough%26aufirst%3DC.%2BE.%26aulast%3DSong%26aufirst%3DS.%26aulast%3DShin%26aufirst%3DM.%2BH.%26aulast%3DJohnson%26aufirst%3DF.%2BB.%26aulast%3DMarmorstein%26aufirst%3DR.%26atitle%3DStructural%2520and%2520Functional%2520Role%2520of%2520Acetyltransferase%2520HMOF%2520K274%2520Autoacetylation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2016%26volume%3D291%26spage%3D18190%26epage%3D18198%26doi%3D10.1074%2Fjbc.M116.736264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">Processing of X-Ray Diffraction Data Collected in Oscillation Mode</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">326</span>, <span class="refDoi"> DOI: 10.1016/S0076-6879(97)76066-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+X-Ray+Diffraction+Data+Collected+in+Oscillation+Mode&doi=10.1016%2FS0076-6879%2897%2976066-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0lgv-3Bm_vPr3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520X-Ray%2520Diffraction%2520Data%2520Collected%2520in%2520Oscillation%2520Mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2FS0076-6879%2897%2976066-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: A Comprehensive Python-Based System for Macromolecular Structure Solution</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+A+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lgv-3Bm_vPr3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520A%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: Model-Building Tools for Molecular Graphics</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+Model-Building+Tools+for+Molecular+Graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lgv-3Bm_vPr3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520Model-Building%2520Tools%2520for%2520Molecular%2520Graphics%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/medium/jm1c00141_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/large/jm1c00141_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00141&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/medium/jm1c00141_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/large/jm1c00141_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Inhibitor design strategy. The distance between the sulfur atom of CoA (magenta stick) and the Nα nitrogen atom of the peptide (magenta stick) is 3.1 Å in the ternary complex of NatD/CoA/SGRGK (PDB: 4U9W), where NatD is in surface view (gray). The designed NatD bisubstrate analogues <b>1</b>–<b>9</b> are illustrated.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/large/jm1c00141_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00141&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/medium/jm1c00141_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/large/jm1c00141_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Route for the Bisubstrate Analogues</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/large/jm1c00141_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00141&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last">Reagents and conditions: (a) 2-bromoacetic acid or 3-bromopropionic acid, <i>N</i>,<i>N</i>′-diisopropylcarbodiimide (DIC), hydroxybenzotriazole (HOBt), dimethylformamide (DMF), r.t., overnight; (b) TFA/TIPS/water (95:2.5:2.5), r.t., 4 h, 34–56% in two steps; and (c) CoASH, triethylammonium bicarbonate buffer, pH 8.4 ± 0.1, r.t., 48 h, 33–55%.</p></p></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/medium/jm1c00141_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/large/jm1c00141_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Biochemical characterization of bisubstrate analogues using the radioactive assay. (a) Concentration–response plots of bisubstrate analogues with an acetyl linker under 4×<i>K</i><sub>m</sub> of the H4-19 peptide and AcCoA condition fitted to Morrison’s quadratic equation (<i>n</i> = 3). (b) Concentration–response plots of bisubstrate analogues with a propionyl linker under 4×<i>K</i><sub>m</sub> of the H4-19 peptide and AcCoA condition (<i>n</i> = 3). (c) CoA-C3-SGRGK (<b>6</b>) is noncompetitive with respect to AcCoA. (d) <b>6</b> is competitive with respect to H4. (e,f) Selectivity study of <b>6</b> and <b>7</b> against a panel of protein acetyltransferases in triplicate (<i>n</i> = 3).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/large/jm1c00141_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00141&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/medium/jm1c00141_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/large/jm1c00141_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structures of hNatD with inhibitors. (a) Structure of hNatD with <b>5</b> (CoA-C2-SGRGK) bound. hNatD is shown in cyan as a cartoon and CoA-C2-SGRGK as magenta sticks. The 2mF<sub>obs</sub>-DF<sub>cal</sub> electron density around the bisubstrate is shown at a contour level of 1σ (gray mesh). (b) Structure of hNatD with <b>6</b> (CoA-C3-SGRGK) bound. hNatD is shown in yellow as a cartoon and CoA-C3-SGRGK as blue sticks. The 2mF<sub>obs</sub>-DF<sub>cal</sub> electron density around the bisubstrate is shown at a contour level of 1σ (gray mesh). (c) Overlay of the structure in (a) and (b). The zoom-in view shows the positions of the sulfur atom of CoA and the nitrogen atom of the peptide amino group. (d) Interaction between CoA and hNatD residues is generated with LIGPLOT. Hydrogen bonds are indicated by dashed green lines, and van der Waals interactions are indicated with red semicircles. Water molecules are indicated with cyan spheres. (e) Interaction between SGRGK and hNatD residues is generated with LIGPLOT. Hydrogen bonds are indicated by dashed green lines, and van der Waals interactions are indicated with red semicircles. Water molecules are indicated with cyan spheres.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/large/jm1c00141_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00141&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/medium/jm1c00141_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/large/jm1c00141_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. CETSA. (a) Thermal stability of endogenous NatD by immunoblotting using the anti-NatD antibody (Invitrogen PIPA520882) when treated with <b>2</b> or <b>6</b> or a phosphate-buffered saline (PBS) solution for 2 h at 100 μM after digitonin (5 μg/mL) permeabilization for 20 min. (b) Quantified relative NatD band intensities of the immunoblot in duplicate. <i>T</i><sub>agg:</sub> apparent aggregation temperature.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00141/20210616/images/large/jm1c00141_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00141&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i33">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47704" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47704" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 37 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arnesen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Damme, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polevoda, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helsens, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evjenth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colaert, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varhaug, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandekerckhove, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lillehaug, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gevaert, K.</span></span> <span> </span><span class="NLM_article-title">Proteomics Analyses Reveal the Evolutionary Conservation and Divergence of N-Terminal Acetyltransferases from Yeast and Humans</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">8157</span>– <span class="NLM_lpage">8162</span>, <span class="refDoi"> DOI: 10.1073/pnas.0901931106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1073%2Fpnas.0901931106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=19420222" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmvVWmsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=8157-8162&author=T.+Arnesenauthor=P.+Van+Dammeauthor=B.+Polevodaauthor=K.+Helsensauthor=R.+Evjenthauthor=N.+Colaertauthor=J.+E.+Varhaugauthor=J.+Vandekerckhoveauthor=J.+R.+Lillehaugauthor=F.+Shermanauthor=K.+Gevaert&title=Proteomics+Analyses+Reveal+the+Evolutionary+Conservation+and+Divergence+of+N-Terminal+Acetyltransferases+from+Yeast+and+Humans&doi=10.1073%2Fpnas.0901931106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Proteomics analyses reveal the evolutionary conservation and divergence of N-terminal acetyltransferases from yeast and humans</span></div><div class="casAuthors">Arnesen, Thomas; Van Damme, Petra; Polevoda, Bogdan; Helsens, Kenny; Evjenth, Rune; Colaert, Niklaas; Varhaug, Jan Erik; Vandekerckhove, Joel; Lillehaug, Johan R.; Sherman, Fred; Gevaert, Kris</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8157-8162, S8157/1-S8157/13</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Nα-terminal acetylation is one of the most common protein modifications in eukaryotes.  The COmbined FRActional Diagonal Chromatog. (COFRADIC) proteomics technol. that can be specifically used to isolate N-terminal peptides was used to det. the N-terminal acetylation status of 742 human and 379 yeast protein N termini, representing the largest eukaryotic dataset of N-terminal acetylation.  The major N-terminal acetyltransferase (NAT), NatA, acts on subclasses of proteins with Ser-, Ala-, Thr-, Gly-, Cys-, and Val- N termini.  NatA is composed of subunits encoded by yARD1 and yNAT1 in yeast and hARD1 and hNAT1 in humans.  A yeast ard1-Δ nat1-Δ strain was phenotypically complemented by hARD1 hNAT1, suggesting that yNatA and hNatA are similar.  However, heterologous combinations, hARD1 yNAT1 and yARD1 hNAT1, were not functional in yeast, suggesting significant structural subunit differences between the species.  Proteomics of a yeast ard1-Δ nat1-Δ strain expressing hNatA demonstrated that hNatA acts on nearly the same set of yeast proteins as yNatA, further revealing that NatA from humans and yeast have identical or nearly identical specificities.  Nevertheless, all NatA substrates in yeast were only partially N-acetylated, whereas the corresponding NatA substrates in HeLa cells were mainly completely N-acetylated.  Overall, we obsd. a higher proportion of N-terminally acetylated proteins in humans (84%) as compared with yeast (57%).  N-acetylation occurred on approx. one-half of the human proteins with Met-Lys-termini, but did not occur on yeast proteins with such termini.  Thus, although we revealed different N-acetylation patterns in yeast and humans, the major NAT, NatA, acetylates the same substrates in both species.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroDhOHLppIB7Vg90H21EOLACvtfcHk0lj5ampHgwhUgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmvVWmsb8%253D&md5=c4fe67e8da4d45102d8d4c79fc4d7d19</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0901931106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0901931106%26sid%3Dliteratum%253Aachs%26aulast%3DArnesen%26aufirst%3DT.%26aulast%3DVan%2BDamme%26aufirst%3DP.%26aulast%3DPolevoda%26aufirst%3DB.%26aulast%3DHelsens%26aufirst%3DK.%26aulast%3DEvjenth%26aufirst%3DR.%26aulast%3DColaert%26aufirst%3DN.%26aulast%3DVarhaug%26aufirst%3DJ.%2BE.%26aulast%3DVandekerckhove%26aufirst%3DJ.%26aulast%3DLillehaug%26aufirst%3DJ.%2BR.%26aulast%3DSherman%26aufirst%3DF.%26aulast%3DGevaert%26aufirst%3DK.%26atitle%3DProteomics%2520Analyses%2520Reveal%2520the%2520Evolutionary%2520Conservation%2520and%2520Divergence%2520of%2520N-Terminal%2520Acetyltransferases%2520from%2520Yeast%2520and%2520Humans%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D8157%26epage%3D8162%26doi%3D10.1073%2Fpnas.0901931106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monda, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulman, B. A.</span></span> <span> </span><span class="NLM_article-title">N-Terminal Acetylation Acts as an Avidity Enhancer within an Interconnected Multiprotein Complex</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>334</i></span>,  <span class="NLM_fpage">674</span>– <span class="NLM_lpage">678</span>, <span class="refDoi"> DOI: 10.1126/science.1209307</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1126%2Fscience.1209307" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=21940857" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyqu7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=334&publication_year=2011&pages=674-678&author=D.+C.+Scottauthor=J.+K.+Mondaauthor=E.+J.+Bennettauthor=J.+W.+Harperauthor=B.+A.+Schulman&title=N-Terminal+Acetylation+Acts+as+an+Avidity+Enhancer+within+an+Interconnected+Multiprotein+Complex&doi=10.1126%2Fscience.1209307"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">N-Terminal Acetylation Acts as an Avidity Enhancer Within an Interconnected Multiprotein Complex</span></div><div class="casAuthors">Scott, Daniel C.; Monda, Julie K.; Bennett, Eric J.; Harper, J. Wade; Schulman, Brenda A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">334</span>
        (<span class="NLM_cas:issue">6056</span>),
    <span class="NLM_cas:pages">674-678</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Although many eukaryotic proteins are amino (N)-terminally acetylated, structural mechanisms by which N-terminal acetylation mediates protein interactions are largely unknown.  Here, N-terminal acetylation of the E2 enzyme, Ubc12, dictates distinctive E3-dependent ligation of the ubiquitin-like protein Nedd8 to Cul1.  Structural, biochem., biophys., and genetic analyses revealed how complete burial of Ubc12's N-acetyl-methionine in a hydrophobic pocket in the E3, Dcn1, promotes cullin neddylation.  The results suggest that the N-terminal acetyl both directs Ubc12's interactions with Dcn1 and prevents repulsion of a charged N terminus.  The authors' data provide a link between acetylation and ubiquitin-like protein conjugation and define a mechanism for N-terminal acetylation-dependent recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2YZdYRdU_fbVg90H21EOLACvtfcHk0lj5ampHgwhUgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyqu7jL&md5=1654bea16702633a9baf611827f60450</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1126%2Fscience.1209307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1209307%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BC.%26aulast%3DMonda%26aufirst%3DJ.%2BK.%26aulast%3DBennett%26aufirst%3DE.%2BJ.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DSchulman%26aufirst%3DB.%2BA.%26atitle%3DN-Terminal%2520Acetylation%2520Acts%2520as%2520an%2520Avidity%2520Enhancer%2520within%2520an%2520Interconnected%2520Multiprotein%2520Complex%26jtitle%3DScience%26date%3D2011%26volume%3D334%26spage%3D674%26epage%3D678%26doi%3D10.1126%2Fscience.1209307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Behnia, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panic, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whyte, J. R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munro, S.</span></span> <span> </span><span class="NLM_article-title">Targeting of the Arf-like GTPase Arl3p to the Golgi Requires N-Terminal Acetylation and the Membrane Protein Sys1p</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">405</span>– <span class="NLM_lpage">413</span>, <span class="refDoi"> DOI: 10.1038/ncb1120</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1038%2Fncb1120" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=15077113" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjsFSltrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2004&pages=405-413&author=R.+Behniaauthor=B.+Panicauthor=J.+R.+C.+Whyteauthor=S.+Munro&title=Targeting+of+the+Arf-like+GTPase+Arl3p+to+the+Golgi+Requires+N-Terminal+Acetylation+and+the+Membrane+Protein+Sys1p&doi=10.1038%2Fncb1120"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting of the Arf-like GTPase Arl3p to the Golgi requires N-terminal acetylation and the membrane protein Sys1p</span></div><div class="casAuthors">Behnia, Rudy; Panic, Bojana; Whyte, James R. C.; Munro, Sean</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">405-413</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The GTPase Arl3p is required to recruit a second GTPase, Arl1p, to the Golgi in Saccharomyces cerevisiae.  Arl1p binds to the GRIP domain, which is present in a no. of long coiled-coil proteins or 'golgins'.  Here we show that Arl3p is not myristoylated like most members of the Arf family, but is instead amino-terminally acetylated by the NatC complex.  Targeting of Arl3p also requires a Golgi membrane protein Sys1p.  The human homologs of Arl3p (Arf-related protein 1 (ARFRP1)) and Sys1p (hSys1) can be isolated in a complex after chem. crosslinking.  This suggests that the targeting of ARFRP1/Arl3p to the Golgi is mediated by a direct interaction between its acetylated N terminus and Sys1p/hSys1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohQbxTiIsb2bVg90H21EOLACvtfcHk0lj5psdj38QYZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjsFSltrY%253D&md5=99e48d738c8771f7cd23771d8700c5a1</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fncb1120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb1120%26sid%3Dliteratum%253Aachs%26aulast%3DBehnia%26aufirst%3DR.%26aulast%3DPanic%26aufirst%3DB.%26aulast%3DWhyte%26aufirst%3DJ.%2BR.%2BC.%26aulast%3DMunro%26aufirst%3DS.%26atitle%3DTargeting%2520of%2520the%2520Arf-like%2520GTPase%2520Arl3p%2520to%2520the%2520Golgi%2520Requires%2520N-Terminal%2520Acetylation%2520and%2520the%2520Membrane%2520Protein%2520Sys1p%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2004%26volume%3D6%26spage%3D405%26epage%3D413%26doi%3D10.1038%2Fncb1120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aksnes, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ree, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnesen, T.</span></span> <span> </span><span class="NLM_article-title">Co-Translational, Post-Translational, and Non-Catalytic Roles of N-Terminal Acetyltransferases</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">1097</span>– <span class="NLM_lpage">1114</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2019.02.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1016%2Fj.molcel.2019.02.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=30878283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkvVertbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2019&pages=1097-1114&author=H.+Aksnesauthor=R.+Reeauthor=T.+Arnesen&title=Co-Translational%2C+Post-Translational%2C+and+Non-Catalytic+Roles+of+N-Terminal+Acetyltransferases&doi=10.1016%2Fj.molcel.2019.02.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Co-translational, Post-translational, and Non-catalytic Roles of N-Terminal Acetyltransferases</span></div><div class="casAuthors">Aksnes, Henriette; Ree, Rasmus; Arnesen, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1097-1114</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Recent studies of N-terminal acetylation have identified new N-terminal acetyltransferases (NATs) and expanded the known functions of these enzymes beyond their roles as ribosome-assocd. co-translational modifiers.  For instance, the identification of Golgi- and chloroplast-assocd. NATs shows that acetylation of N termini also happens post-translationally.  In addn., we now appreciate that some NATs are highly specific; for example, a dedicated NAT responsible for post-translational N-terminal acetylation of actin was recently revealed.  Other studies have extended NAT function beyond Nt acetylation, including functions as lysine acetyltransferases (KATs) and non-catalytic roles.  Finally, emerging studies emphasize the physiol. relevance of N-terminal acetylation, including roles in calorie-restriction-induced longevity and pathol. α-synuclein aggregation in Parkinson's disease.  Combined, the NATs rise as multifunctional proteins, and N-terminal acetylation is gaining recognition as a major cellular regulator.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZokGZxIsfBLVg90H21EOLACvtfcHk0lj5psdj38QYZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkvVertbc%253D&md5=8f4282b0ff189e0b3bbc7d6643398e90</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2019.02.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2019.02.007%26sid%3Dliteratum%253Aachs%26aulast%3DAksnes%26aufirst%3DH.%26aulast%3DRee%26aufirst%3DR.%26aulast%3DArnesen%26aufirst%3DT.%26atitle%3DCo-Translational%252C%2520Post-Translational%252C%2520and%2520Non-Catalytic%2520Roles%2520of%2520N-Terminal%2520Acetyltransferases%26jtitle%3DMol.%2520Cell%26date%3D2019%26volume%3D73%26spage%3D1097%26epage%3D1114%26doi%3D10.1016%2Fj.molcel.2019.02.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span> <span> </span><span class="NLM_article-title">Protein N-Terminal Acetylation: Structural Basis, Mechanism, Versatility, and Regulation</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">27</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2020.08.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1016%2Fj.tibs.2020.08.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=32912665" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1Oltb7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2021&pages=15-27&author=S.+Dengauthor=R.+Marmorstein&title=Protein+N-Terminal+Acetylation%3A+Structural+Basis%2C+Mechanism%2C+Versatility%2C+and+Regulation&doi=10.1016%2Fj.tibs.2020.08.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Protein N-Terminal Acetylation: Structural Basis, Mechanism, Versatility, and Regulation</span></div><div class="casAuthors">Deng, Sunbin; Marmorstein, Ronen</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">15-27</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">N-terminal acetylation (NTA) is one of the most widespread protein modifications, which occurs on most eukaryotic proteins, but is significantly less common on bacterial and archaea proteins.  This modification is carried out by a family of enzymes called N-terminal acetyltransferases (NATs).  To date, 12 NATs have been identified, harboring different compn., substrate specificity, and in some cases, modes of regulation.  Recent structural and biochem. anal. of NAT proteins allows for a comparison of their mol. mechanisms and modes of regulation, which are described here.  Although sharing an evolutionarily conserved fold and related catalytic mechanism, each catalytic subunit uses unique elements to mediate substrate-specific activity, and use NAT-type specific auxiliary and regulatory subunits, for their cellular functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOQcv07MEVNbVg90H21EOLACvtfcHk0lh3EvzEIloggQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1Oltb7N&md5=fcec4efee655278b2184deb9557ea627</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2020.08.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2020.08.005%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DS.%26aulast%3DMarmorstein%26aufirst%3DR.%26atitle%3DProtein%2520N-Terminal%2520Acetylation%253A%2520Structural%2520Basis%252C%2520Mechanism%252C%2520Versatility%252C%2520and%2520Regulation%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2021%26volume%3D46%26spage%3D15%26epage%3D27%26doi%3D10.1016%2Fj.tibs.2020.08.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ree, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varland, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnesen, T.</span></span> <span> </span><span class="NLM_article-title">Spotlight on Protein N-Terminal Acetylation</span>. <i>Exp. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">13</span>, <span class="refDoi"> DOI: 10.1038/s12276-018-0116-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1038%2Fs12276-018-0116-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=30054468" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVSqs77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2018&pages=1-13&author=R.+Reeauthor=S.+Varlandauthor=T.+Arnesen&title=Spotlight+on+Protein+N-Terminal+Acetylation&doi=10.1038%2Fs12276-018-0116-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Spotlight on protein N-terminal acetylation</span></div><div class="casAuthors">Ree, Rasmus; Varland, Sylvia; Arnesen, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Experimental & Molecular Medicine</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1-13</span>CODEN:
                <span class="NLM_cas:coden">EMMEF3</span>;
        ISSN:<span class="NLM_cas:issn">2092-6413</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  N-terminal acetylation (Nt-acetylation) is a widespread protein modification among eukaryotes and prokaryotes alike.  By appending an acetyl group to the N-terminal amino group, the charge, hydrophobicity, and size of the N-terminus is altered in an irreversible manner.  This alteration has implications for the lifespan, folding characteristics and binding properties of the acetylated protein.  The enzymic machinery responsible for Nt-acetylation has been largely described, but significant knowledge gaps remain.  In this review, we provide an overview of eukaryotic N-terminal acetyltransferases (NATs) and the impact of Nt-acetylation.  We also discuss other functions of known NATs and outline methods for studying Nt-acetylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAsJi2UstR57Vg90H21EOLACvtfcHk0lh3EvzEIloggQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVSqs77F&md5=f45b6c761774a8302c340356032b442f</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fs12276-018-0116-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs12276-018-0116-z%26sid%3Dliteratum%253Aachs%26aulast%3DRee%26aufirst%3DR.%26aulast%3DVarland%26aufirst%3DS.%26aulast%3DArnesen%26aufirst%3DT.%26atitle%3DSpotlight%2520on%2520Protein%2520N-Terminal%2520Acetylation%26jtitle%3DExp.%2520Mol.%2520Med.%26date%3D2018%26volume%3D50%26spage%3D1%26epage%3D13%26doi%3D10.1038%2Fs12276-018-0116-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hole, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Damme, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalva, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aksnes, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glomnes, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varhaug, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lillehaug, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gevaert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnesen, T.</span></span> <span> </span><span class="NLM_article-title">The Human N-Alpha-Acetyltransferase 40 (HNaa40p/HNatD) Is Conserved from Yeast and N-Terminally Acetylates Histones H2A and H4</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e24713</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0024713</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1371%2Fjournal.pone.0024713" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=21935442" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1Oms77L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&author=K.+Holeauthor=P.+van+Dammeauthor=M.+Dalvaauthor=H.+Aksnesauthor=N.+Glomnesauthor=J.+E.+Varhaugauthor=J.+R.+Lillehaugauthor=K.+Gevaertauthor=T.+Arnesen&title=The+Human+N-Alpha-Acetyltransferase+40+%28HNaa40p%2FHNatD%29+Is+Conserved+from+Yeast+and+N-Terminally+Acetylates+Histones+H2A+and+H4&doi=10.1371%2Fjournal.pone.0024713"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The human N-alpha-acetyltransferase 40 (hNaa40p/hNatD) is conserved from yeast and N-terminally acetylates histones H2A and H4</span></div><div class="casAuthors">Hole, Kristine; Van Damme, Petra; Dalva, Monica; Aksnes, Henriette; Glomnes, Nina; Varhaug, Jan Erik; Lillehaug, Johan R.; Gevaert, Kris; Arnesen, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e24713</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Protein Nα-terminal acetylation (Nt-acetylation) is considered one of the most common protein modification in eukaryotes, and 80-90 % of all sol. human proteins are modified in this way, with functional implications ranging from altered protein function and stability to translocation potency amongst others.  Nt-acetylation is catalyzed by N-terminal acetyltransferases (NATs), and in yeast five NAT types are identified and denoted NatA-NatE.  Higher eukaryotes addnl. express NatF.  Except for NatD, human orthologs for all yeast NATs are identified.  YNatD is defined as the catalytic unit Naa40p (Nat4) which co-translationally Nt-acetylates histones H2A and H4.  In this study we identified and characterized hNaa40p/hNatD, the human ortholog of the yeast Naa40p.  An in vitro proteome-derived peptide library Nt-acetylation assay indicated that recombinant hNaa40p acetylates N-termini starting with the consensus sequence Ser-Gly-Gly-Gly-Lys-, strongly resembling the N-termini of the human histones H2A and H4.  This was confirmed as recombinant hNaa40p Nt-acetylated the oligopeptides derived from the N-termini of both histones.  In contrast, a synthetically Nt-acetylated H4 N-terminal peptide with all lysines being non-acetylated, was not significantly acetylated by hNaa40p, indicating that hNaa40p catalyzed H4 Nα-acetylation and not H4 lysine N.vepsiln.-acetylation.  Also, immunopptd. hNaa40p specifically Nt-acetylated H4 in vitro.  Heterologous expression of hNaa40p in a yeast naa40-Δ strain restored Nt-acetylation of yeast histone H4, but not H2A in vivo, probably reflecting the fact that the N-terminal sequences of human H2A and H4 are highly similar to each other and to yeast H4 while the N-terminal sequence of yeast H2A differs.  Thus, Naa40p seems to have co-evolved with the human H2A sequence.  Finally, a partial co-sedimentation with ribosomes indicates that hNaa40p co-translationally acetylates H2A and H4.  Combined, our results strongly suggest that human Naa40p/NatD is conserved from yeast.  Thus, the NATs of all classes of N-terminally acetylated proteins in humans now appear to be accounted for.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrd6pv_cgdTgLVg90H21EOLACvtfcHk0lh3EvzEIloggQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1Oms77L&md5=e2a58a3364d8547f35dad47edfb2c971</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0024713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0024713%26sid%3Dliteratum%253Aachs%26aulast%3DHole%26aufirst%3DK.%26aulast%3Dvan%2BDamme%26aufirst%3DP.%26aulast%3DDalva%26aufirst%3DM.%26aulast%3DAksnes%26aufirst%3DH.%26aulast%3DGlomnes%26aufirst%3DN.%26aulast%3DVarhaug%26aufirst%3DJ.%2BE.%26aulast%3DLillehaug%26aufirst%3DJ.%2BR.%26aulast%3DGevaert%26aufirst%3DK.%26aulast%3DArnesen%26aufirst%3DT.%26atitle%3DThe%2520Human%2520N-Alpha-Acetyltransferase%252040%2520%2528HNaa40p%252FHNatD%2529%2520Is%2520Conserved%2520from%2520Yeast%2520and%2520N-Terminally%2520Acetylates%2520Histones%2520H2A%2520and%2520H4%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26doi%3D10.1371%2Fjournal.pone.0024713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Magin, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liszczak, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span> <span> </span><span class="NLM_article-title">The Molecular Basis for Histone H4- and H2A-Specific Amino-Terminal Acetylation by NatD</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">332</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1016/j.str.2014.10.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1016%2Fj.str.2014.10.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=25619998" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVaru7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=332-341&author=R.+S.+Maginauthor=G.+P.+Liszczakauthor=R.+Marmorstein&title=The+Molecular+Basis+for+Histone+H4-+and+H2A-Specific+Amino-Terminal+Acetylation+by+NatD&doi=10.1016%2Fj.str.2014.10.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The Molecular Basis for Histone H4- and H2A-Specific Amino-Terminal Acetylation by NatD</span></div><div class="casAuthors">Magin, Robert S.; Liszczak, Glen P.; Marmorstein, Ronen</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">332-341</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">N-terminal acetylation is among the most common protein modifications in eukaryotes and is mediated by evolutionarily conserved N-terminal acetyltransferases (NATs).  NatD is among the most selective NATs; its only known substrates are histones H4 and H2A, contg. the N-terminal sequence SGRGK in humans.  Here we characterize the mol. basis for substrate-specific acetylation by NatD by reporting its crystal structure bound to cognate substrates and performing related biochem. studies.  A novel N-terminal segment wraps around the catalytic core domain to make stabilizing interactions, and the α1-α2 and β6-β7 loops adopt novel conformations to properly orient the histone N termini in the binding site.  Ser1 and Arg3 of the histone make extensive contacts to highly conserved NatD residues in the substrate binding pocket, and flanking glycine residues also appear to contribute to substrate-specific binding by NatD, together defining a Ser-Gly-Arg-Gly (SGRG) recognition sequence.  These studies have implications for understanding substrate-specific acetylation by NAT enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqe7cTJj5emoLVg90H21EOLACvtfcHk0lh3EvzEIloggQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVaru7c%253D&md5=54c7a1111ff735db3ab546b0fc9eb8aa</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2014.10.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2014.10.025%26sid%3Dliteratum%253Aachs%26aulast%3DMagin%26aufirst%3DR.%2BS.%26aulast%3DLiszczak%26aufirst%3DG.%2BP.%26aulast%3DMarmorstein%26aufirst%3DR.%26atitle%3DThe%2520Molecular%2520Basis%2520for%2520Histone%2520H4-%2520and%2520H2A-Specific%2520Amino-Terminal%2520Acetylation%2520by%2520NatD%26jtitle%3DStructure%26date%3D2015%26volume%3D23%26spage%3D332%26epage%3D341%26doi%3D10.1016%2Fj.str.2014.10.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Polevoda, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norbeck, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takakura, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blomberg, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, F.</span></span> <span> </span><span class="NLM_article-title">Identification and Specificities of N-Terminal Acetyltransferases from Saccharomyces Cerevisiae</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">6155</span>– <span class="NLM_lpage">6168</span>, <span class="refDoi"> DOI: 10.1093/emboj/18.21.6155</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1093%2Femboj%2F18.21.6155" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10545125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADyaK1MXns1ent7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1999&pages=6155-6168&author=B.+Polevodaauthor=J.+Norbeckauthor=H.+Takakuraauthor=A.+Blombergauthor=F.+Sherman&title=Identification+and+Specificities+of+N-Terminal+Acetyltransferases+from+Saccharomyces+Cerevisiae&doi=10.1093%2Femboj%2F18.21.6155"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and specificities of N-terminal acetyltransferases from Saccharomyces cerevisiae</span></div><div class="casAuthors">Polevoda, Bogdan; Norbeck, Joakim; Takakura, Hikaru; Blomberg, Anders; Sherman, Fred</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6155-6168</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">N-terminal acetylation can occur cotranslationally on the initiator methionine residue or on the penultimate residue if the methionine is cleaved.  We investigated the three N-terminal acetyltransferases (NATs), Ard1p/Nat1p, Nat3p and Mak3p.  Ard1p and Mak3p are significantly related to each other by amino acid sequence, as is Nat3p, which was uncovered in this study using programming alignment procedures.  Mutants deleted in any one of these NAT genes were viable, but some exhibited diminished mating efficiency and reduced growth at 37°C, and on glycerol and NaCl-contg. media.  The three NATs had the following substrate specificities as detd. in vivo by examg. acetylation of 14 altered forms of iso-1-cytochrome c and 55 abundant normal proteins in each of the deleted strains: Ard1p/Nat1p, subclasses with Ser-, Ala-, Gly- and Thr-termini; Nat3p, Met-Glu- and Met- Asp- and a subclass of Met-Asn-termini; and Mak3p subclasses with Met-Ile- and Met-Leu-termini.  In addn., a special subclass of substrates with Ser-Glu-Phe-, Ala-Glu-Phe- and Gly-Glu-Phe-termini required all three NATs for acetylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-zatcFLGJ8rVg90H21EOLACvtfcHk0lhg-QG3xUEM6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXns1ent7s%253D&md5=16d7691c8fb2459187c5d5d28f0b6e37</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1093%2Femboj%2F18.21.6155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252F18.21.6155%26sid%3Dliteratum%253Aachs%26aulast%3DPolevoda%26aufirst%3DB.%26aulast%3DNorbeck%26aufirst%3DJ.%26aulast%3DTakakura%26aufirst%3DH.%26aulast%3DBlomberg%26aufirst%3DA.%26aulast%3DSherman%26aufirst%3DF.%26atitle%3DIdentification%2520and%2520Specificities%2520of%2520N-Terminal%2520Acetyltransferases%2520from%2520Saccharomyces%2520Cerevisiae%26jtitle%3DEMBO%2520J.%26date%3D1999%26volume%3D18%26spage%3D6155%26epage%3D6168%26doi%3D10.1093%2Femboj%2F18.21.6155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pavlou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirmizis, A.</span></span> <span> </span><span class="NLM_article-title">Depletion of Histone N-Terminal-Acetyltransferase Naa40 Induces P53-Independent Apoptosis in Colorectal Cancer Cells via the Mitochondrial Pathway</span>. <i>Apoptosis</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">298</span>– <span class="NLM_lpage">311</span>, <span class="refDoi"> DOI: 10.1007/s10495-015-1207-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1007%2Fs10495-015-1207-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=26666750" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVOjs7vN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=298-311&author=D.+Pavlouauthor=A.+Kirmizis&title=Depletion+of+Histone+N-Terminal-Acetyltransferase+Naa40+Induces+P53-Independent+Apoptosis+in+Colorectal+Cancer+Cells+via+the+Mitochondrial+Pathway&doi=10.1007%2Fs10495-015-1207-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Depletion of histone N-terminal-acetyltransferase Naa40 induces p53-independent apoptosis in colorectal cancer cells via the mitochondrial pathway</span></div><div class="casAuthors">Pavlou, Demetria; Kirmizis, Antonis</div><div class="citationInfo"><span class="NLM_cas:title">Apoptosis</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">298-311</span>CODEN:
                <span class="NLM_cas:coden">APOPFN</span>;
        ISSN:<span class="NLM_cas:issn">1360-8185</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Protein N-terminal acetylation is an abundant post-translational modification in eukaryotes implicated in various fundamental cellular and biochem. processes.  This modification is catalyzed by evolutionarily conserved N-terminal acetyltransferases (NATs) whose deregulation has been linked to cancer development and thus, are emerging as useful diagnostic and therapeutic targets.  Naa40 is a highly selective NAT that acetylates the amino-termini of histones H4 and H2A and acts as a sensor of cell growth in yeast.  In the present study, we examine the role of Naa40 in cancer cell survival.  We demonstrate that depletion of Naa40 in HCT116 and HT-29 colorectal cancer cells decreases cell survival by enhancing apoptosis, whereas Naa40 redn. in non-cancerous mouse embryonic fibroblasts has no effect on cell viability.  Specifically, Naa40 knockdown in colon cancer cells activates the mitochondrial caspase-9-mediated apoptotic cascade.  Consistent with this, we show that caspase-9 activation is required for the induced apoptosis because treatment of cells with an irreversible caspase-9 inhibitor impedes apoptosis when Naa40 is depleted.  Furthermore, the effect of Naa40-depletion on cell-death is mediated through a p53-independent mechanism since p53-null HCT116 cells still undergo apoptosis upon redn. of the acetyltransferase.  Altogether, these findings reveal an anti-apoptotic role for Naa40 and exhibit its potential as a therapeutic target in colorectal cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqS_QZjAJOfpbVg90H21EOLACvtfcHk0lhg-QG3xUEM6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVOjs7vN&md5=80bc057a86e6bf5620baa6ed12971aaf</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1007%2Fs10495-015-1207-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10495-015-1207-0%26sid%3Dliteratum%253Aachs%26aulast%3DPavlou%26aufirst%3DD.%26aulast%3DKirmizis%26aufirst%3DA.%26atitle%3DDepletion%2520of%2520Histone%2520N-Terminal-Acetyltransferase%2520Naa40%2520Induces%2520P53-Independent%2520Apoptosis%2520in%2520Colorectal%2520Cancer%2520Cells%2520via%2520the%2520Mitochondrial%2520Pathway%26jtitle%3DApoptosis%26date%3D2016%26volume%3D21%26spage%3D298%26epage%3D311%26doi%3D10.1007%2Fs10495-015-1207-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Man, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, Q.</span></span> <span> </span><span class="NLM_article-title">Patt1, a Novel Protein Acetyltransferase That Is Highly Expressed in Liver and Downregulated in Hepatocellular Carcinoma, Enhances Apoptosis of Hepatoma Cells</span>. <i>Int. J. Biochem. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">2528</span>– <span class="NLM_lpage">2537</span>, <span class="refDoi"> DOI: 10.1016/j.biocel.2009.08.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1016%2Fj.biocel.2009.08.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=19695338" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlGhsrrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2009&pages=2528-2537&author=Z.+Liuauthor=Y.+Liuauthor=H.+Wangauthor=X.+Geauthor=Q.+Jinauthor=G.+Dingauthor=Y.+Huauthor=B.+Zhouauthor=Z.+Chenauthor=X.+Geauthor=B.+Zhangauthor=X.+Manauthor=Q.+Zhai&title=Patt1%2C+a+Novel+Protein+Acetyltransferase+That+Is+Highly+Expressed+in+Liver+and+Downregulated+in+Hepatocellular+Carcinoma%2C+Enhances+Apoptosis+of+Hepatoma+Cells&doi=10.1016%2Fj.biocel.2009.08.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Patt1, a novel protein acetyltransferase that is highly expressed in liver and downregulated in hepatocellular carcinoma, enhances apoptosis of hepatoma cells</span></div><div class="casAuthors">Liu, Zhen; Liu, Yang; Wang, Huiqiang; Ge, Xinjian; Jin, Qihuang; Ding, Guanghui; Hu, Yanan; Zhou, Ben; Chen, Zhongjian; Ge, Xuemei; Zhang, Baohua; Man, Xiaobo; Zhai, Qiwei</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Biochemistry & Cell Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2528-2537</span>CODEN:
                <span class="NLM_cas:coden">IJBBFU</span>;
        ISSN:<span class="NLM_cas:issn">1357-2725</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Protein acetylation is increasingly recognized as an important post-translational modification.  Although a lot of protein acetyltransferases have been identified, a few putative acetyltransferases are yet to be studied.  In this study, the authors identified a novel protein acetyltransferase, Patt1, which belongs to GNAT family.  Patt1 exhibited histone acetyltransferase activity and auto-acetylation activity.  Deletion and mutation anal. of the predicted acetyltransferase domain in Patt1 showed that the conserved Glu139 was an important residue for its protein acetyltransferase activity.  Furthermore, the authors found that Patt1 was highly expressed in liver and significantly down-regulated in hepatocellular carcinoma tissues.  In addn., the authors showed that overexpression of Patt1 enhanced the apoptosis of hepatoma cells dependent on its acetyltransferase activity, whereas knockdown of Patt1 significantly protected Chang liver cells from apoptosis.  These data suggest that Patt1 might be involved in the development of hepatocellular carcinoma, and could be served as a potential therapy target for hepatocellular carcinoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7oW5hR1AxrLVg90H21EOLACvtfcHk0lhg-QG3xUEM6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlGhsrrL&md5=d49149cb399f2eeccbe26abf6b5526a3</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.biocel.2009.08.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biocel.2009.08.009%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DGe%26aufirst%3DX.%26aulast%3DJin%26aufirst%3DQ.%26aulast%3DDing%26aufirst%3DG.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DGe%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DMan%26aufirst%3DX.%26aulast%3DZhai%26aufirst%3DQ.%26atitle%3DPatt1%252C%2520a%2520Novel%2520Protein%2520Acetyltransferase%2520That%2520Is%2520Highly%2520Expressed%2520in%2520Liver%2520and%2520Downregulated%2520in%2520Hepatocellular%2520Carcinoma%252C%2520Enhances%2520Apoptosis%2520of%2520Hepatoma%2520Cells%26jtitle%3DInt.%2520J.%2520Biochem.%2520Cell%2520Biol.%26date%3D2009%26volume%3D41%26spage%3D2528%26epage%3D2537%26doi%3D10.1016%2Fj.biocel.2009.08.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schiza, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molina-Serrano, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kyriakou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadjiantoniou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirmizis, A.</span></span> <span> </span><span class="NLM_article-title">N-Alpha-Terminal Acetylation of Histone H4 Regulates Arginine Methylation and Ribosomal DNA Silencing</span>. <i>PLoS Genet.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">e1003805</span> <span class="refDoi"> DOI: 10.1371/journal.pgen.1003805</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1371%2Fjournal.pgen.1003805" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=24068969" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFKitbrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&author=V.+Schizaauthor=D.+Molina-Serranoauthor=D.+Kyriakouauthor=A.+Hadjiantoniouauthor=A.+Kirmizis&title=N-Alpha-Terminal+Acetylation+of+Histone+H4+Regulates+Arginine+Methylation+and+Ribosomal+DNA+Silencing&doi=10.1371%2Fjournal.pgen.1003805"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">N-alpha-terminal acetylation of histone H4 regulates arginine methylation and ribosomal DNA silencing</span></div><div class="casAuthors">Schiza, Vassia; Molina-Serrano, Diego; Kyriakou, Dimitris; Hadjiantoniou, Antonia; Kirmizis, Antonis</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Genetics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e1003805</span>CODEN:
                <span class="NLM_cas:coden">PGLEB5</span>;
        ISSN:<span class="NLM_cas:issn">1553-7404</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Post-translational modifications of histones play a key role in DNA-based processes, like transcription, by modulating chromatin structure.  N-terminal acetylation is unique among the numerous histone modifications because it is deposited on the N-alpha amino group of the first residue instead of the side-chain of amino acids.  The function of this modification and its interplay with other internal histone marks has not been previously addressed.  Here, we identified N-terminal acetylation of H4 (N-acH4) as a novel regulator of arginine methylation and chromatin silencing in Saccharomyces cerevisiae.  Lack of the H4 N-alpha acetyltransferase (Nat4) activity results specifically in increased deposition of asym. dimethylation of histone H4 arginine 3 (H4R3me2a) and in enhanced ribosomal-DNA silencing.  Consistent with this, H4 N-terminal acetylation impairs the activity of the Hmt1 methyltransferase towards H4R3 in vitro.  Furthermore, combinatorial loss of N-acH4 with internal histone acetylation at lysines 5, 8 and 12 has a synergistic induction of H4R3me2a deposition and rDNA silencing that leads to a severe growth defect.  This defect is completely rescued by mutating arginine 3 to lysine (H4R3K), suggesting that abnormal deposition of a single histone modification, H4R3me2a, can impact on cell growth.  Notably, the cross-talk between N-acH4 and H4R3me2a, which regulates rDNA silencing, is induced under calorie restriction conditions.  Collectively, these findings unveil a mol. and biol. function for H4 N-terminal acetylation, identify its interplay with internal histone modifications, and provide general mechanistic implications for N-alpha-terminal acetylation, one of the most common protein modifications in eukaryotes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhb1VM3trLc7Vg90H21EOLACvtfcHk0lifuCvYHIxXqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFKitbrL&md5=d0b5fe7abb3fc3eba1d6c9a39132e7e8</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pgen.1003805&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pgen.1003805%26sid%3Dliteratum%253Aachs%26aulast%3DSchiza%26aufirst%3DV.%26aulast%3DMolina-Serrano%26aufirst%3DD.%26aulast%3DKyriakou%26aufirst%3DD.%26aulast%3DHadjiantoniou%26aufirst%3DA.%26aulast%3DKirmizis%26aufirst%3DA.%26atitle%3DN-Alpha-Terminal%2520Acetylation%2520of%2520Histone%2520H4%2520Regulates%2520Arginine%2520Methylation%2520and%2520Ribosomal%2520DNA%2520Silencing%26jtitle%3DPLoS%2520Genet.%26date%3D2013%26volume%3D9%26doi%3D10.1371%2Fjournal.pgen.1003805" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ju, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gui, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kvansakul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, D. C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span> <span> </span><span class="NLM_article-title">NatD Promotes Lung Cancer Progression by Preventing Histone H4 Serine Phosphorylation to Activate Slug Expression</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">14</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-00988-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1038%2Fs41467-017-00988-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=28232747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtV2ntrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1-14&author=J.+Juauthor=A.+Chenauthor=Y.+Dengauthor=M.+Liuauthor=Y.+Wangauthor=Y.+Wangauthor=M.+Nieauthor=C.+Wangauthor=H.+Dingauthor=B.+Yaoauthor=T.+Guiauthor=X.+Liauthor=Z.+Xuauthor=C.+Maauthor=Y.+Songauthor=M.+Kvansakulauthor=K.+Zenauthor=C.+Y.+Zhangauthor=C.+Luoauthor=M.+Fangauthor=D.+C.+S.+Huangauthor=C.+D.+Allisauthor=R.+Tanauthor=C.+K.+Zengauthor=J.+Weiauthor=Q.+Zhao&title=NatD+Promotes+Lung+Cancer+Progression+by+Preventing+Histone+H4+Serine+Phosphorylation+to+Activate+Slug+Expression&doi=10.1038%2Fs41467-017-00988-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">ORAI channels are critical for receptor-mediated endocytosis of albumin</span></div><div class="casAuthors">Zeng, Bo; Chen, Gui-Lan; Garcia-Vaz, Eliana; Bhandari, Sunil; Daskoulidou, Nikoleta; Berglund, Lisa M.; Jiang, Hongni; Hallett, Thomas; Zhou, Lu-Ping; Huang, Li; Xu, Zi-Hao; Nair, Viji; Nelson, Robert G.; Ju, Wenjun; Kretzler, Matthias; Atkin, Stephen L.; Gomez, Maria F.; Xu, Shang-Zhong</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-12</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Impaired albumin reabsorption by proximal tubular epithelial cells (PTECs) has been highlighted in diabetic nephropathy (DN), but little is known about the underlying mol. mechanisms.  Here we find that ORAI1-3, are preferentially expressed in PTECs and downregulated in patients with DN.  Hyperglycemia or blockade of insulin signaling reduces the expression of ORAI1-3.  Inhibition of ORAI channels by BTP2 and diethylstilbestrol or silencing of ORAI expression impairs albumin uptake.  Transgenic mice expressing a dominant-neg. Orai1 mutant (E108Q) increases albuminuria, and in vivo injection of BTP2 exacerbates albuminuria in streptozotocin-induced and Akita diabetic mice.  The albumin endocytosis is Ca2+-dependent and accompanied by ORAI1 internalization.  Amnionless (AMN) assocs. with ORAIs and forms STIM/ORAI/AMN complexes after Ca2+ store depletion.  STIM1/ORAI1 colocalizes with clathrin, but not with caveolin, at the apical membrane of PTECs, which dets. clathrin-mediated endocytosis.  These findings provide insights into the mechanisms of protein reabsorption and potential targets for treating diabetic proteinuria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvu0IP8NBiH7Vg90H21EOLACvtfcHk0liSFebCNNSZHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtV2ntrrJ&md5=ef4e199dafb36bc703a8894408936db5</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-00988-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-00988-5%26sid%3Dliteratum%253Aachs%26aulast%3DJu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DA.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DNie%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DYao%26aufirst%3DB.%26aulast%3DGui%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DMa%26aufirst%3DC.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DKvansakul%26aufirst%3DM.%26aulast%3DZen%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DC.%2BY.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DFang%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DD.%2BC.%2BS.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DTan%26aufirst%3DR.%26aulast%3DZeng%26aufirst%3DC.%2BK.%26aulast%3DWei%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DQ.%26atitle%3DNatD%2520Promotes%2520Lung%2520Cancer%2520Progression%2520by%2520Preventing%2520Histone%2520H4%2520Serine%2520Phosphorylation%2520to%2520Activate%2520Slug%2520Expression%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D1%26epage%3D14%26doi%3D10.1038%2Fs41467-017-00988-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liszczak, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foyn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petersson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnesen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span> <span> </span><span class="NLM_article-title">Molecular Basis for N-Terminal Acetylation by the Heterodimeric NatA Complex</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1098</span>, <span class="refDoi"> DOI: 10.1038/nsmb.2636</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1038%2Fnsmb.2636" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=23912279" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1WhtLvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=1098&author=G.+Liszczakauthor=J.+M.+Goldbergauthor=H.+Foynauthor=E.+J.+Peterssonauthor=T.+Arnesenauthor=R.+Marmorstein&title=Molecular+Basis+for+N-Terminal+Acetylation+by+the+Heterodimeric+NatA+Complex&doi=10.1038%2Fnsmb.2636"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis for N-terminal acetylation by the heterodimeric NatA complex</span></div><div class="casAuthors">Liszczak, Glen; Goldberg, Jacob M.; Foyn, Havard; Petersson, E. James; Arnesen, Thomas; Marmorstein, Ronen</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1098-1105</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">N-terminal acetylation is ubiquitous among eukaryotic proteins and controls a myriad of biol. processes.  Of the N-terminal acetyltransferases (NATs) that facilitate this cotranslational modification, the heterodimeric NatA complex has the most diversity for substrate selection and modifies the majority of all N-terminally acetylated proteins.  Here, the authors report the x-ray crystal structure of the 100-kDa holo-NatA complex from Schizosaccharomyces pombe, in the absence and presence of a bisubstrate peptide-CoA-conjugate inhibitor, as well as the structure of the uncomplexed Naa10p catalytic subunit.  The NatA-Naa15p auxiliary subunit contains 13 tetratricopeptide motifs and adopts a ring-like topol. that wraps around the NatA-Naa10p subunit, an interaction that alters the Naa10p active site for substrate-specific acetylation.  These studies have implications for understanding the mechanistic details of other NAT complexes and how regulatory subunits modulate the activity of the broader family of protein acetyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrrjoCe0eahrVg90H21EOLACvtfcHk0liSFebCNNSZHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1WhtLvJ&md5=d95329e33f31932afaddcb81e491c9cc</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2636&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2636%26sid%3Dliteratum%253Aachs%26aulast%3DLiszczak%26aufirst%3DG.%26aulast%3DGoldberg%26aufirst%3DJ.%2BM.%26aulast%3DFoyn%26aufirst%3DH.%26aulast%3DPetersson%26aufirst%3DE.%2BJ.%26aulast%3DArnesen%26aufirst%3DT.%26aulast%3DMarmorstein%26aufirst%3DR.%26atitle%3DMolecular%2520Basis%2520for%2520N-Terminal%2520Acetylation%2520by%2520the%2520Heterodimeric%2520NatA%2520Complex%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D1098%26doi%3D10.1038%2Fnsmb.2636" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evjenth, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brenner, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnesen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frøystein, N. Å.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lillehaug, J. R.</span></span> <span> </span><span class="NLM_article-title">Human Protein N-Terminal Acetyltransferase HNaa50p (HNAT5/HSAN) Follows Ordered Sequential Catalytic Mechanism: Combined Kinetic and NMR Study</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">10081</span>– <span class="NLM_lpage">10088</span>, <span class="refDoi"> DOI: 10.1074/jbc.M111.326587</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1074%2Fjbc.M111.326587" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=22311970" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC38XktlKku7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=10081-10088&author=R.+H.+Evjenthauthor=A.+K.+Brennerauthor=P.+R.+Thompsonauthor=T.+Arnesenauthor=N.+%C3%85.+Fr%C3%B8ysteinauthor=J.+R.+Lillehaug&title=Human+Protein+N-Terminal+Acetyltransferase+HNaa50p+%28HNAT5%2FHSAN%29+Follows+Ordered+Sequential+Catalytic+Mechanism%3A+Combined+Kinetic+and+NMR+Study&doi=10.1074%2Fjbc.M111.326587"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Human Protein N-terminal Acetyltransferase hNaa50p (hNAT5/hSAN) Follows Ordered Sequential Catalytic Mechanism: Combined Kinetic and NMR Study</span></div><div class="casAuthors">Evjenth, Rune H.; Brenner, Annette K.; Thompson, Paul R.; Arnesen, Thomas; Froystein, Nils Aage; Lillehaug, Johan R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">10081-10088</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Nα-Acetylation is a common protein modification catalyzed by different N-terminal acetyltransferases (NATs).  Their essential role in the biogenesis and degrdn. of proteins is becoming increasingly evident.  The NAT hNaa50p preferentially modifies peptides starting with methionine followed by a hydrophobic amino acid. hNaa50p also possesses Nε-autoacetylation activity.  So far, no eukaryotic NAT has been mechanistically investigated.  In this study, we used NMR spectroscopy, bisubstrate kinetic assays, and product inhibition expts. to demonstrate that hNaa50p utilizes an ordered Bi Bi reaction of the Theorell-Chance type.  The NMR results, both the substrate binding study and the dynamic data, further indicate that the binding of acetyl-CoA induces a conformational change that is required for the peptide to bind to the active site.  In support of an ordered Bi Bi reaction mechanism, addn. of peptide in the absence of acetyl-CoA did not alter the structure of the protein.  This model is further strengthened by the NMR results using a catalytically inactive hNaa50p mutant.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojMOeRuLDeibVg90H21EOLACvtfcHk0liSFebCNNSZHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XktlKku7w%253D&md5=15290fa41ab6d9589f8c47a913960207</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.326587&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.326587%26sid%3Dliteratum%253Aachs%26aulast%3DEvjenth%26aufirst%3DR.%2BH.%26aulast%3DBrenner%26aufirst%3DA.%2BK.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26aulast%3DArnesen%26aufirst%3DT.%26aulast%3DFr%25C3%25B8ystein%26aufirst%3DN.%2B%25C3%2585.%26aulast%3DLillehaug%26aufirst%3DJ.%2BR.%26atitle%3DHuman%2520Protein%2520N-Terminal%2520Acetyltransferase%2520HNaa50p%2520%2528HNAT5%252FHSAN%2529%2520Follows%2520Ordered%2520Sequential%2520Catalytic%2520Mechanism%253A%2520Combined%2520Kinetic%2520and%2520NMR%2520Study%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D10081%26epage%3D10088%26doi%3D10.1074%2Fjbc.M111.326587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weyer, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gumiero, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lapouge, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bange, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopp, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinning, I.</span></span> <span> </span><span class="NLM_article-title">Structural Basis of HypK Regulating N-Terminal Acetylation by the NatA Complex</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">10</span>, <span class="refDoi"> DOI: 10.1038/ncomms15726</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1038%2Fncomms15726" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=28232747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC1cXptVCmsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1-10&author=F.+A.+Weyerauthor=A.+Gumieroauthor=K.+Lapougeauthor=G.+Bangeauthor=J.+Koppauthor=I.+Sinning&title=Structural+Basis+of+HypK+Regulating+N-Terminal+Acetylation+by+the+NatA+Complex&doi=10.1038%2Fncomms15726"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Heparan sulfate proteoglycans present PCSK9 to the LDL receptor</span></div><div class="casAuthors">Gustafsen, Camilla; Olsen, Ditte; Vilstrup, Joachim; Lund, Signe; Reinhardt, Anika; Wellner, Niels; Larsen, Torben; Andersen, Christian B. F.; Weyer, Kathrin; Li, Jin-ping; Seeberger, Peter H.; Thirup, Soeren; Madsen, Peder; Glerup, Simon</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-14</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Coronary artery disease is the main cause of death worldwide and accelerated by increased plasma levels of cholesterol-rich low-d. lipoprotein particles (LDL).  Circulating PCSK9 contributes to coronary artery disease by inducing lysosomal degrdn. of the LDL receptor (LDLR) in the liver and thereby reducing LDL clearance.  Here, we show that liver heparan sulfate proteoglycans are PCSK9 receptors and essential for PCSK9-induced LDLR degrdn.  The heparan sulfate-binding site is located in the PCSK9 prodomain and formed by surface-exposed basic residues interacting with trisulfated heparan sulfate disaccharide repeats.  Accordingly, heparan sulfate mimetics and monoclonal antibodies directed against the heparan sulfate-binding site are potent PCSK9 inhibitors.  We propose that heparan sulfate proteoglycans lining the hepatocyte surface capture PCSK9 and facilitates subsequent PCSK9:LDLR complex formation.  Our findings provide new insights into LDL biol. and show that targeting PCSK9 using heparan sulfate mimetics is a potential therapeutic strategy in coronary artery disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKbZYmSf5-8rVg90H21EOLACvtfcHk0lgWvfIwvw5-6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXptVCmsLk%253D&md5=bf1a00ff0a960d5f23c8c3f4e87a3e1e</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fncomms15726&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms15726%26sid%3Dliteratum%253Aachs%26aulast%3DWeyer%26aufirst%3DF.%2BA.%26aulast%3DGumiero%26aufirst%3DA.%26aulast%3DLapouge%26aufirst%3DK.%26aulast%3DBange%26aufirst%3DG.%26aulast%3DKopp%26aufirst%3DJ.%26aulast%3DSinning%26aufirst%3DI.%26atitle%3DStructural%2520Basis%2520of%2520HypK%2520Regulating%2520N-Terminal%2520Acetylation%2520by%2520the%2520NatA%2520Complex%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D1%26epage%3D10%26doi%3D10.1038%2Fncomms15726" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gottlieb, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petersson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span> <span> </span><span class="NLM_article-title">Molecular Basis for N-Terminal Alpha-Synuclein Acetylation by Human Natb</span>. <i>Elife</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">e57491</span> <span class="refDoi"> DOI: 10.7554/eLife.57491</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.7554%2FeLife.57491" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=32885784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisFKms77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2020&author=S.+Dengauthor=B.+Panauthor=L.+Gottliebauthor=E.+J.+Peterssonauthor=R.+Marmorstein&title=Molecular+Basis+for+N-Terminal+Alpha-Synuclein+Acetylation+by+Human+Natb&doi=10.7554%2FeLife.57491"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis for N-terminal alpha-synuclein acetylation by human NatB</span></div><div class="casAuthors">Deng, Sunbin; Pan, Buyan; Gottlieb, Leah; James Petersson, E.; Marmorstein, Ronen</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e57491</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">NatB is one of three major N-terminal acetyltransferase (NAT) complexes (NatA-NatC), which co-translationally acetylate the N-termini of eukaryotic proteins.  Its substrates account for about 21% of the human proteome, including well known proteins such as actin, tropomyosin, CDK2, and α-synuclein (αSyn).  Human NatB (hNatB) mediated N-terminal acetylation of αSyn has been demonstrated to play key roles in the pathogenesis of Parkinson's disease and as a potential therapeutic target for hepatocellular carcinoma.  Here we report the cryo-EM structure of hNatB bound to a CoA-αSyn conjugate, together with structure-guided anal. of mutational effects on catalysis.  This anal. reveals functionally important differences with human NatA and Candida albicans NatB, resolves key hNatB protein determinants for αSyn N-terminal acetylation, and identifies important residues for substrate-specific recognition and acetylation by NatB enzymes.  These studies have implications for developing small mol. NatB probes and for understanding the mode of substrate selection by NAT enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNXZN29IscJbVg90H21EOLACvtfcHk0lgWvfIwvw5-6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisFKms77F&md5=1047ab779ce9685e625367de6e3dd18e</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.7554%2FeLife.57491&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.57491%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DS.%26aulast%3DPan%26aufirst%3DB.%26aulast%3DGottlieb%26aufirst%3DL.%26aulast%3DPetersson%26aufirst%3DE.%2BJ.%26aulast%3DMarmorstein%26aufirst%3DR.%26atitle%3DMolecular%2520Basis%2520for%2520N-Terminal%2520Alpha-Synuclein%2520Acetylation%2520by%2520Human%2520Natb%26jtitle%3DElife%26date%3D2020%26volume%3D9%26doi%3D10.7554%2FeLife.57491" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Støve, S. I. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magin, R. S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foyn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haug, B. E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnesen, T.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of the Golgi-Associated Human Nα-Acetyltransferase 60 Reveals the Molecular Determinants for Substrate-Specific Acetylation</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1044</span>– <span class="NLM_lpage">1056</span>, <span class="refDoi"> DOI: 10.1016/j.str.2016.04.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1016%2Fj.str.2016.04.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=27320834" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A280%3ADC%252BC2s7hsFOrsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=1044-1056&author=S.+I.+I.+St%C3%B8veauthor=R.+S.+S.+Maginauthor=H.+Foynauthor=B.+E.+E.+Haugauthor=R.+Marmorsteinauthor=T.+Arnesen&title=Crystal+Structure+of+the+Golgi-Associated+Human+N%CE%B1-Acetyltransferase+60+Reveals+the+Molecular+Determinants+for+Substrate-Specific+Acetylation&doi=10.1016%2Fj.str.2016.04.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure of the Golgi-Associated Human Nα-Acetyltransferase 60 Reveals the Molecular Determinants for Substrate-Specific Acetylation</span></div><div class="casAuthors">Stove Svein Isungset; Magin Robert S; Foyn Havard; Haug Bengt Erik; Marmorstein Ronen; Arnesen Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Structure (London, England : 1993)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1044-56</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">N-Terminal acetylation is a common and important protein modification catalyzed by N-terminal acetyltransferases (NATs).  Six human NATs (NatA-NatF) contain one catalytic subunit each, Naa10 to Naa60, respectively.  In contrast to the ribosome-associated NatA to NatE, NatF/Naa60 specifically associates with Golgi membranes and acetylates transmembrane proteins.  To gain insight into the molecular basis for the function of Naa60, we developed an Naa60 bisubstrate CoA-peptide conjugate inhibitor, determined its X-ray structure when bound to CoA and inhibitor, and carried out biochemical experiments.  We show that Naa60 adapts an overall fold similar to that of the catalytic subunits of ribosome-associated NATs, but with the addition of two novel elongated loops that play important roles in substrate-specific binding.  One of these loops mediates a dimer to monomer transition upon substrate-specific binding.  Naa60 employs a catalytic mechanism most similar to Naa50.  Collectively, these data reveal the molecular basis for Naa60-specific acetyltransferase activity with implications for its Golgi-specific functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS6VBO9-n6Q1WhHGZei4OW9fW6udTcc2eY-a0QBEtyW5rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s7hsFOrsA%253D%253D&md5=f7e2b98f1d865fe8b8ca3bea30ff62f8</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2016.04.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2016.04.020%26sid%3Dliteratum%253Aachs%26aulast%3DSt%25C3%25B8ve%26aufirst%3DS.%2BI.%2BI.%26aulast%3DMagin%26aufirst%3DR.%2BS.%2BS.%26aulast%3DFoyn%26aufirst%3DH.%26aulast%3DHaug%26aufirst%3DB.%2BE.%2BE.%26aulast%3DMarmorstein%26aufirst%3DR.%26aulast%3DArnesen%26aufirst%3DT.%26atitle%3DCrystal%2520Structure%2520of%2520the%2520Golgi-Associated%2520Human%2520N%25CE%25B1-Acetyltransferase%252060%2520Reveals%2520the%2520Molecular%2520Determinants%2520for%2520Substrate-Specific%2520Acetylation%26jtitle%3DStructure%26date%3D2016%26volume%3D24%26spage%3D1044%26epage%3D1056%26doi%3D10.1016%2Fj.str.2016.04.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goris, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magin, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foyn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myklebust, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varland, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ree, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drazic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhambra, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Støve, S. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haug, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnesen, T.</span></span> <span> </span><span class="NLM_article-title">Structural Determinants and Cellular Environment Define Processed Actin as the Sole Substrate of the N-Terminal Acetyltransferase NAA80</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">4405</span>– <span class="NLM_lpage">4410</span>, <span class="refDoi"> DOI: 10.1073/pnas.1719251115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1073%2Fpnas.1719251115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=29581307" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVKjtrnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2018&pages=4405-4410&author=M.+Gorisauthor=R.+S.+Maginauthor=H.+Foynauthor=L.+M.+Myklebustauthor=S.+Varlandauthor=R.+Reeauthor=A.+Drazicauthor=P.+Bhambraauthor=S.+I.+St%C3%B8veauthor=M.+Baumannauthor=B.+E.+Haugauthor=R.+Marmorsteinauthor=T.+Arnesen&title=Structural+Determinants+and+Cellular+Environment+Define+Processed+Actin+as+the+Sole+Substrate+of+the+N-Terminal+Acetyltransferase+NAA80&doi=10.1073%2Fpnas.1719251115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Structural determinants and cellular environment define processed actin as the sole substrate of the N-terminal acetyltransferase NAA80</span></div><div class="casAuthors">Goris, Marianne; Magin, Robert S.; Foyn, Haavard; Myklebust, Line M.; Varland, Sylvia; Ree, Rasmus; Drazic, Adrian; Bhambra, Parminder; Stoeve, Svein I.; Baumann, Markus; Haug, Bengt Erik; Marmorstein, Ronen; Arnesen, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4405-4410</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">N-terminal (Nt) acetylation is a major protein modification catalyzed by N-terminal acetyltransferases (NATs).  Methionine acidic N termini, including actin, are cotranslationally Nt acetylated by NatB in all eukaryotes, but animal actins contg. acidic N termini, are addnl. post-translationally Nt acetylated by NAA80.  Actin Nt acetylation was found to regulate cytoskeletal dynamics and motility, thus making NAA80 a potential target for cell migration regulation.  In this work, we developed potent and selective bisubstrate inhibitors for NAA80 and detd. the crystal structure of NAA80 in complex with such an inhibitor, revealing that NAA80 adopts a fold similar to other NAT enzymes but with a more open substrate binding region.  Furthermore, in contrast to most other NATs, the substrate specificity of NAA80 is mainly derived through interactions between the enzyme and the acidic amino acids at positions 2 and 3 of the actin substrate and not residues 1 and 2.  A yeast model revealed that ectopic expression of NAA80 in a strain lacking NatB activity partially restored Nt acetylation of NatB substrates, including yeast actin.  Thus, NAA80 holds intrinsic capacity to posttranslationally Nt acetylate NatB-type substrates in vivo.  In sum, the presence of a dominant cotranslational NatB in all eukaryotes, the specific post-translational actin methionine removal in animals, and finally, the unique structural features of NAA80 leave only the processed actins as in vivo substrates of NAA80.  Together, this study reveals the mol. and cellular basis of NAA80 Nt acetylation and provides a scaffold for development of inhibitors for the regulation of cytoskeletal properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGVMeywrLUl7Vg90H21EOLACvtfcHk0lgEjz5NOT8_Xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVKjtrnJ&md5=3e3de1e9f289d208dd3eb2633335c15a</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1719251115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1719251115%26sid%3Dliteratum%253Aachs%26aulast%3DGoris%26aufirst%3DM.%26aulast%3DMagin%26aufirst%3DR.%2BS.%26aulast%3DFoyn%26aufirst%3DH.%26aulast%3DMyklebust%26aufirst%3DL.%2BM.%26aulast%3DVarland%26aufirst%3DS.%26aulast%3DRee%26aufirst%3DR.%26aulast%3DDrazic%26aufirst%3DA.%26aulast%3DBhambra%26aufirst%3DP.%26aulast%3DSt%25C3%25B8ve%26aufirst%3DS.%2BI.%26aulast%3DBaumann%26aufirst%3DM.%26aulast%3DHaug%26aufirst%3DB.%2BE.%26aulast%3DMarmorstein%26aufirst%3DR.%26aulast%3DArnesen%26aufirst%3DT.%26atitle%3DStructural%2520Determinants%2520and%2520Cellular%2520Environment%2520Define%2520Processed%2520Actin%2520as%2520the%2520Sole%2520Substrate%2520of%2520the%2520N-Terminal%2520Acetyltransferase%2520NAA80%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2018%26volume%3D115%26spage%3D4405%26epage%3D4410%26doi%3D10.1073%2Fpnas.1719251115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Foyn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewallen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narawane, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varhaug, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnesen, T.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Kinetic Characterization of Protein N-Terminal Acetyltransferase Inhibitors</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1121</span>– <span class="NLM_lpage">1127</span>, <span class="refDoi"> DOI: 10.1021/cb400136s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb400136s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlt1Whsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1121-1127&author=H.+Foynauthor=J.+E.+Jonesauthor=D.+Lewallenauthor=R.+Narawaneauthor=J.+E.+Varhaugauthor=P.+R.+Thompsonauthor=T.+Arnesen&title=Design%2C+Synthesis%2C+and+Kinetic+Characterization+of+Protein+N-Terminal+Acetyltransferase+Inhibitors&doi=10.1021%2Fcb400136s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Kinetic Characterization of Protein N-Terminal Acetyltransferase Inhibitors</span></div><div class="casAuthors">Foyn, Haavard; Jones, Justin E.; Lewallen, Dan; Narawane, Rashmi; Varhaug, Jan Erik; Thompson, Paul R.; Arnesen, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1121-1127</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The N-termini of 80-90% of human proteins are acetylated by the N-terminal actransferases (NATs), NatA-NatF.  The major NAT complex, NatA, and particularly the catalytic subunit hNaa10 (ARD1) has been implicated in cancer development.  For example, knockdown of hNaa10 results in cancer cell death and the arrest of cell proliferation.  It also sensitized cancer cells to drug-induced cytotoxicity.  Human NatE has a distinct substrate specificity and is essential for normal chromosome segregation.  Thus, NAT inhibitors may potentially be valuable anticancer therapeutics, either directly or as adjuvants.  Herein, the authors report the design and synthesis of the first inhibitors targeting these enzymes.  Using the substrate specificity of the enzymes as a guide, the authors synthesized three bisubstrate analogs that potently and selectively inhibit the NatA complex (CoA-Ac-SES4; IC50 = 15.1 μM), hNaa10, the catalytic subunit of NatA (CoA-Ac-EEE4; Ki = 1.6 μM), and NatE/hNaa50 (CoA-Ac-MLG7; Ki* = 8 nM); CoA-Ac-EEE4 is a reversible competitive inhibitor of hNaa10, and CoA-Ac-MLG7 is a slow tight binding inhibitor of hNaa50.  The authors' demonstration that it is possible to develop NAT selective inhibitors should assist future efforts to develop NAT inhibitors with more drug-like properties that can be used to chem. interrogate in vivo NAT function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2gkDQnZDQarVg90H21EOLACvtfcHk0lgEjz5NOT8_Xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlt1Whsrg%253D&md5=4c84fb13c4bcb9b7b0fe3db0b69e5eef</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fcb400136s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb400136s%26sid%3Dliteratum%253Aachs%26aulast%3DFoyn%26aufirst%3DH.%26aulast%3DJones%26aufirst%3DJ.%2BE.%26aulast%3DLewallen%26aufirst%3DD.%26aulast%3DNarawane%26aufirst%3DR.%26aulast%3DVarhaug%26aufirst%3DJ.%2BE.%26aulast%3DThompson%26aufirst%3DP.%2BR.%26aulast%3DArnesen%26aufirst%3DT.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Kinetic%2520Characterization%2520of%2520Protein%2520N-Terminal%2520Acetyltransferase%2520Inhibitors%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D1121%26epage%3D1127%26doi%3D10.1021%2Fcb400136s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noinaj, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Discovery of Bisubstrate Inhibitors for Protein N-Terminal Methyltransferase 1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">3773</span>– <span class="NLM_lpage">3779</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00206</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00206" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC1MXltFequ7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=3773-3779&author=D.+Chenauthor=G.+Dongauthor=N.+Noinajauthor=R.+Huang&title=Discovery+of+Bisubstrate+Inhibitors+for+Protein+N-Terminal+Methyltransferase+1&doi=10.1021%2Facs.jmedchem.9b00206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of bisubstrate inhibitors for protein N-terminal methyltransferase 1</span></div><div class="casAuthors">Chen, Dongxing; Dong, Guangping; Noinaj, Nicholas; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3773-3779</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein N-terminal methyltransferase 1 (NTMT1) plays an important role in regulating mitosis and DNA repair.  Here, we describe the discovery of a potent NTMT1 bisubstrate inhibitor 4 (IC50 = 35 ± 2 nM) that exhibits greater than 100-fold selectivity against a panel of methyltransferases.  We also report the first crystal structure of NTMT1 in complex with an inhibitor, which revealed that 4 occupies substrate and cofactor binding sites of NTMT1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqM1ZcSF10eY7Vg90H21EOLACvtfcHk0lgEjz5NOT8_Xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXltFequ7o%253D&md5=34782952c04423e217abf6cd3d228f94</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00206%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DD.%26aulast%3DDong%26aufirst%3DG.%26aulast%3DNoinaj%26aufirst%3DN.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DDiscovery%2520of%2520Bisubstrate%2520Inhibitors%2520for%2520Protein%2520N-Terminal%2520Methyltransferase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D3773%26epage%3D3779%26doi%3D10.1021%2Facs.jmedchem.9b00206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ho, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Development of a Continuous Fluorescence-Based Assay for n-Terminal Acetyltransferase D</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">12</span>, <span class="refDoi"> DOI: 10.3390/ijms22020594</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.3390%2Fijms22020594" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2021&pages=1-12&author=Y.+H.+Hoauthor=L.+Chenauthor=R.+Huang&title=Development+of+a+Continuous+Fluorescence-Based+Assay+for+n-Terminal+Acetyltransferase+D&doi=10.3390%2Fijms22020594"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.3390%2Fijms22020594&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms22020594%26sid%3Dliteratum%253Aachs%26aulast%3DHo%26aufirst%3DY.%2BH.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DDevelopment%2520of%2520a%2520Continuous%2520Fluorescence-Based%2520Assay%2520for%2520n-Terminal%2520Acetyltransferase%2520D%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2021%26volume%3D22%26spage%3D1%26epage%3D12%26doi%3D10.3390%2Fijms22020594" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span> <span> </span><span class="NLM_article-title">Tight Binding Inhibition</span>. In  <i>Evaluation of Enzyme Inhibitors in Drug Discovery: A Guide for Medicinal Chemists and Pharmacologists</i>; <span class="NLM_edition">2</span>nd Ed., <span class="NLM_publisher-name">John Wiley & Sons</span>: <span class="NLM_publisher-loc">New Jersey</span>, <span class="NLM_year">2013</span>,  <span class="NLM_fpage">245</span>– <span class="NLM_lpage">285</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1002%2F9781118540398.ch7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=245-285&author=R.+A.+Copeland&title=Evaluation+of+Enzyme+Inhibitors+in+Drug+Discovery%3A+A+Guide+for+Medicinal+Chemists+and+Pharmacologists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1002%2F9781118540398.ch7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781118540398.ch7%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26atitle%3DTight%2520Binding%2520Inhibition%26btitle%3DEvaluation%2520of%2520Enzyme%2520Inhibitors%2520in%2520Drug%2520Discovery%253A%2520A%2520Guide%2520for%2520Medicinal%2520Chemists%2520and%2520Pharmacologists%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%26date%3D2013%26spage%3D245%26epage%3D285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, J. F.</span></span> <span> </span><span class="NLM_article-title">The Slow-Binding and Slow, Tight-Binding Inhibition of Enzyme-Catalysed Reactions</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">102</span>– <span class="NLM_lpage">105</span>, <span class="refDoi"> DOI: 10.1016/0968-0004(82)90157-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1016%2F0968-0004%2882%2990157-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADyaL38XhslKrtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1982&pages=102-105&author=J.+F.+Morrison&title=The+Slow-Binding+and+Slow%2C+Tight-Binding+Inhibition+of+Enzyme-Catalysed+Reactions&doi=10.1016%2F0968-0004%2882%2990157-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The slow-binding and slow, tight-binding inhibition of enzyme-catalyzed reactions</span></div><div class="casAuthors">Morrison, John F.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">102-5</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    </div><div class="casAbstract">The mechanisms of enzyme inhibition by slow-binding and slow, tight-binding reversible inhibitors are discussed, as well as the kinetics of these processes as opposed to classical and tight-binding inhibitor mechanisms.  Examples of these types of inhibitors in the enzyme literature are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoun0ayjodbYrVg90H21EOLACvtfcHk0lhRkI9mcAl0Qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL38XhslKrtrw%253D&md5=1740627c1ad2adf182f4c0ae355bb4de</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2F0968-0004%2882%2990157-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0968-0004%252882%252990157-8%26sid%3Dliteratum%253Aachs%26aulast%3DMorrison%26aufirst%3DJ.%2BF.%26atitle%3DThe%2520Slow-Binding%2520and%2520Slow%252C%2520Tight-Binding%2520Inhibition%2520of%2520Enzyme-Catalysed%2520Reactions%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D1982%26volume%3D7%26spage%3D102%26epage%3D105%26doi%3D10.1016%2F0968-0004%2882%2990157-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Khalil, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, P. A.</span></span> <span> </span><span class="NLM_article-title">A potent inhibitor of the melatonin rhythm enzyme</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">6195</span>– <span class="NLM_lpage">6196</span>, <span class="refDoi"> DOI: 10.1021/ja981365a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja981365a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADyaK1cXjslehsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=1998&pages=6195-6196&author=E.+M.+Khalilauthor=P.+A.+Cole&title=A+potent+inhibitor+of+the+melatonin+rhythm+enzyme&doi=10.1021%2Fja981365a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">A Potent Inhibitor of the Melatonin Rhythm Enzyme</span></div><div class="casAuthors">Khalil, Ehab M.; Cole, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">6195-6196</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibitors of melatonin biosynthesis could not only help improve our understanding of circadian rhythm but they might also have therapeutic roles in mood and sleep disorders.  The key enzyme to be targeted in this regard is serotonin N-acetyltransferase (arylalkylamine N-acetyltransferase, AANAT), also called the melatonin rhythm enzyme.  This paper reports the synthesis and activity of the first potent and specific inhibitor of AANAT.  This inhibitor is a bisubstrate analog which contains covalently linked components of serotonin and acetyl-CoA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3jN1ADqSw8rVg90H21EOLACvtfcHk0lhRkI9mcAl0Qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXjslehsbs%253D&md5=a7e66d9c02d47c5eb94b92c221dfa431</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fja981365a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja981365a%26sid%3Dliteratum%253Aachs%26aulast%3DKhalil%26aufirst%3DE.%2BM.%26aulast%3DCole%26aufirst%3DP.%2BA.%26atitle%3DA%2520potent%2520inhibitor%2520of%2520the%2520melatonin%2520rhythm%2520enzyme%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1998%26volume%3D120%26spage%3D6195%26epage%3D6196%26doi%3D10.1021%2Fja981365a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaudet, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, C. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molinaro, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kottwitz, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rohde, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dangeard, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrieumerlou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girardin, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray-Owen, S. D.</span></span> <span> </span><span class="NLM_article-title">Innate recognition of intracellular bacterial growth is driven by the TIFA-dependent cytosolic surveillance pathway</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1418</span>– <span class="NLM_lpage">1430</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2017.04.063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1016%2Fj.celrep.2017.04.063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=28514661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnvF2jsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2017&pages=1418-1430&author=R.+G.+Gaudetauthor=C.+X.+Guoauthor=R.+Molinaroauthor=H.+Kottwitzauthor=J.+R.+Rohdeauthor=A.+S.+Dangeardauthor=C.+Arrieumerlouauthor=S.+E.+Girardinauthor=S.+D.+Gray-Owen&title=Innate+recognition+of+intracellular+bacterial+growth+is+driven+by+the+TIFA-dependent+cytosolic+surveillance+pathway&doi=10.1016%2Fj.celrep.2017.04.063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Innate Recognition of Intracellular Bacterial Growth Is Driven by the TIFA-Dependent Cytosolic Surveillance Pathway</span></div><div class="casAuthors">Gaudet, Ryan G.; Guo, Cynthia X.; Molinaro, Raphael; Kottwitz, Haila; Rohde, John R.; Dangeard, Anne-Sophie; Arrieumerlou, Cecile; Girardin, Stephen E.; Gray-Owen, Scott D.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1418-1430</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Intestinal epithelial cells (IECs) act as sentinels for incoming pathogens.  Cytosol-invasive bacteria, such as Shigella flexneri, trigger a robust pro-inflammatory nuclear factor κB (NF-κB) response from IECs that is believed to depend entirely on the peptidoglycan sensor NOD1.  We found that, during Shigella infection, the TRAF-interacting forkhead-assocd. protein A (TIFA)-dependent cytosolic surveillance pathway, which senses the bacterial metabolite heptose-1,7-bisphosphate (HBP), functions after NOD1 to detect bacteria replicating free in the host cytosol.  Whereas NOD1 mediated a transient burst of NF-κB activation during bacterial entry, TIFA sensed HBP released during bacterial replication, assembling into large signaling complexes to drive a dynamic inflammatory response that reflected the rate of intracellular bacterial proliferation.  Strikingly, IECs lacking TIFA were unable to discriminate between proliferating and stagnant intracellular bacteria, despite the NOD1/2 pathways being intact.  Our results define TIFA as a rheostat for intracellular bacterial replication, escalating the immune response to invasive Gram-neg. bacteria that exploit the host cytosol for growth.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofl1BqukOlLLVg90H21EOLACvtfcHk0lhRkI9mcAl0Qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnvF2jsr4%253D&md5=3287157c992c75ba1dd93ac59c5a47ba</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.04.063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.04.063%26sid%3Dliteratum%253Aachs%26aulast%3DGaudet%26aufirst%3DR.%2BG.%26aulast%3DGuo%26aufirst%3DC.%2BX.%26aulast%3DMolinaro%26aufirst%3DR.%26aulast%3DKottwitz%26aufirst%3DH.%26aulast%3DRohde%26aufirst%3DJ.%2BR.%26aulast%3DDangeard%26aufirst%3DA.%2BS.%26aulast%3DArrieumerlou%26aufirst%3DC.%26aulast%3DGirardin%26aufirst%3DS.%2BE.%26aulast%3DGray-Owen%26aufirst%3DS.%2BD.%26atitle%3DInnate%2520recognition%2520of%2520intracellular%2520bacterial%2520growth%2520is%2520driven%2520by%2520the%2520TIFA-dependent%2520cytosolic%2520surveillance%2520pathway%26jtitle%3DCell%2520Rep.%26date%3D2017%26volume%3D19%26spage%3D1418%26epage%3D1430%26doi%3D10.1016%2Fj.celrep.2017.04.063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noinaj, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Structure-based discovery of cell-potent peptidomimetic inhibitors for protein N-terminal methyltransferase 1</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">485</span>– <span class="NLM_lpage">493</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.1c00012</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.1c00012" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BB3MXlt1Krtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2021&pages=485-493&author=D.+Chenauthor=G.+Dongauthor=Y.+Dengauthor=N.+Noinajauthor=R.+Huang&title=Structure-based+discovery+of+cell-potent+peptidomimetic+inhibitors+for+protein+N-terminal+methyltransferase+1&doi=10.1021%2Facsmedchemlett.1c00012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of cell-potent peptidomimetic inhibitors for protein N-terminal methyltransferase 1</span></div><div class="casAuthors">Chen, Dongxing; Dong, Guangping; Deng, Youchao; Noinaj, Nicholas; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">485-493</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein N-terminal methyltransferases (NTMTs) catalyze the methylation of the α-N-terminal amines of proteins starting with an X-P-K/R motif.  NTMT1 has been implicated in various cancers and in aging, implying its role as a potential therapeutic target.  Through structural modifications of a lead NTMT1 inhibitor, BM30, we designed and synthesized a diverse set of inhibitors to probe the NTMT1 active site.  The incorporation of a naphthyl group at the N-terminal region and an ortho-aminobenzoic amide at the C-terminal region of BM30 generates the top cell-potent inhibitor DC541, demonstrating increased activity on both purified NTMT1 (IC50 of 0.34 ± 0.02μM) and the cellular α-N-terminal methylation level of regulator of chromosome condensation 1 (RCC1, IC50 value of 30μM) in human colorectal cancer HT29 cells.  Furthermore, DC541 exhibits over 300-fold selectivity to several methyltransferases.  This study points out the direction for the development of more cell-potent inhibitors for NTMT1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrr2eVAYKlcULVg90H21EOLACvtfcHk0lgtNVmJwXXYLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXlt1Krtr8%253D&md5=4ce642d17391e555deb2b65994b24794</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.1c00012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.1c00012%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DD.%26aulast%3DDong%26aufirst%3DG.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DNoinaj%26aufirst%3DN.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DStructure-based%2520discovery%2520of%2520cell-potent%2520peptidomimetic%2520inhibitors%2520for%2520protein%2520N-terminal%2520methyltransferase%25201%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2021%26volume%3D12%26spage%3D485%26epage%3D493%26doi%3D10.1021%2Facsmedchemlett.1c00012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jafari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almqvist, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axelsson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ignatushchenko, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lundback, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordlund, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez
Molina, D.</span></span> <span> </span><span class="NLM_article-title">The cellular thermal shift assay for evaluating drug target interactions in cells</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2100</span>– <span class="NLM_lpage">2122</span>, <span class="refDoi"> DOI: 10.1038/nprot.2014.138</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1038%2Fnprot.2014.138" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=25101824" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12msrfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=2100-2122&author=R.+Jafariauthor=H.+Almqvistauthor=H.+Axelssonauthor=M.+Ignatushchenkoauthor=T.+Lundbackauthor=P.+Nordlundauthor=D.+Martinez%0AMolina&title=The+cellular+thermal+shift+assay+for+evaluating+drug+target+interactions+in+cells&doi=10.1038%2Fnprot.2014.138"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">The cellular thermal shift assay for evaluating drug target interactions in cells</span></div><div class="casAuthors">Jafari, Rozbeh; Almqvist, Helena; Axelsson, Hanna; Ignatushchenko, Marina; Lundbaeck, Thomas; Nordlund, Paer; Molina, Daniel Martinez</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2100-2122</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Thermal shift assays are used to study thermal stabilization of proteins upon ligand binding.  Such assays have been used extensively on purified proteins in the drug discovery industry and in academia to detect interactions.  Recently, we published a proof-of-principle study describing the implementation of thermal shift assays in a cellular format, which we call the cellular thermal shift assay (CETSA).  The method allows studies of target engagement of drug candidates in a cellular context, herein exemplified with exptl. data on the human kinases p38α and ERK1/2.  The assay involves treatment of cells with a compd. of interest, heating to denature and ppt. proteins, cell lysis, and the sepn. of cell debris and aggregates from the sol. protein fraction.  Whereas unbound proteins denature and ppt. at elevated temps., ligand-bound proteins remain in soln.  We describe two procedures for detecting the stabilized protein in the sol. fraction of the samples.  One approach involves sample workup and detection using quant. western blotting, whereas the second is performed directly in soln. and relies on the induced proximity of two target-directed antibodies upon binding to sol. protein.  The latter protocol has been optimized to allow an increased throughput, as potential applications require large nos. of samples.  Both approaches can be completed in a day.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzarqomAK6eLVg90H21EOLACvtfcHk0lgtNVmJwXXYLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12msrfJ&md5=e7852e1a306bca1698a8e8463ae566b1</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2014.138&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2014.138%26sid%3Dliteratum%253Aachs%26aulast%3DJafari%26aufirst%3DR.%26aulast%3DAlmqvist%26aufirst%3DH.%26aulast%3DAxelsson%26aufirst%3DH.%26aulast%3DIgnatushchenko%26aufirst%3DM.%26aulast%3DLundback%26aufirst%3DT.%26aulast%3DNordlund%26aufirst%3DP.%26aulast%3DMartinez%2BMolina%26aufirst%3DD.%26atitle%3DThe%2520cellular%2520thermal%2520shift%2520assay%2520for%2520evaluating%2520drug%2520target%2520interactions%2520in%2520cells%26jtitle%3DNat.%2520Protoc.%26date%3D2014%26volume%3D9%26spage%3D2100%26epage%3D2122%26doi%3D10.1038%2Fnprot.2014.138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rasband, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eliceiri, K. W.</span></span> <span> </span><span class="NLM_article-title">NIH Image to ImageJ: 25 years of image analysis</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">671</span>– <span class="NLM_lpage">675</span>, <span class="refDoi"> DOI: 10.1038/nmeth.2089</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1038%2Fnmeth.2089" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=22930834" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVKntb7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2012&pages=671-675&author=C.+A.+Schneiderauthor=W.+S.+Rasbandauthor=K.+W.+Eliceiri&title=NIH+Image+to+ImageJ%3A+25+years+of+image+analysis&doi=10.1038%2Fnmeth.2089"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">NIH Image to ImageJ: 25 years of image analysis</span></div><div class="casAuthors">Schneider, Caroline A.; Rasband, Wayne S.; Eliceiri, Kevin W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7_part1</span>),
    <span class="NLM_cas:pages">671-675</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">For the past 25 years NIH Image and ImageJ software have been pioneers as open tools for the anal. of scientific images.  We discuss the origins, challenges and solns. of these two programs, and how their history can serve to advise and inform other software projects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofzHXrdLwDn7Vg90H21EOLACvtfcHk0lgtNVmJwXXYLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVKntb7P&md5=85ab928cd79f1e2f2351c834c0c600f0</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnmeth.2089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnmeth.2089%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DC.%2BA.%26aulast%3DRasband%26aufirst%3DW.%2BS.%26aulast%3DEliceiri%26aufirst%3DK.%2BW.%26atitle%3DNIH%2520Image%2520to%2520ImageJ%253A%252025%2520years%2520of%2520image%2520analysis%26jtitle%3DNat.%2520Methods%26date%3D2012%26volume%3D9%26spage%3D671%26epage%3D675%26doi%3D10.1038%2Fnmeth.2089" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Collins, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanier, G. S.</span></span> <span> </span><span class="NLM_article-title">High-Efficiency Solid Phase Peptide Synthesis (He-Spps)</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">940</span>– <span class="NLM_lpage">943</span>, <span class="refDoi"> DOI: 10.1021/ol4036825</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol4036825" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1eiu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2014&pages=940-943&author=J.+M.+Collinsauthor=K.+A.+Porterauthor=S.+K.+Singhauthor=G.+S.+Vanier&title=High-Efficiency+Solid+Phase+Peptide+Synthesis+%28He-Spps%29&doi=10.1021%2Fol4036825"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">High-Efficiency Solid Phase Peptide Synthesis (HE-SPPS)</span></div><div class="casAuthors">Collins, Jonathan M.; Porter, Keith A.; Singh, Sandeep K.; Vanier, Grace S.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">940-943</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of improvements to the std. solid-phase peptide synthesis (SPPS) process allowing for significant gains in product purity along with only a 4 min std. cycle time and a 90% redn. in total waste produced is reported.  For example, syntheses of the well-known 65-74acyl carrier protein (ACP) and 1-42β-amyloid peptides were accomplished with 93 and 72% purity (UPLC-MS) in only 44 and 229 min, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqY1pWgm1oVPLVg90H21EOLACvtfcHk0lgtNVmJwXXYLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1eiu7o%253D&md5=4338beecbc9e347d93cbbdd8e4543094</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fol4036825&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol4036825%26sid%3Dliteratum%253Aachs%26aulast%3DCollins%26aufirst%3DJ.%2BM.%26aulast%3DPorter%26aufirst%3DK.%2BA.%26aulast%3DSingh%26aufirst%3DS.%2BK.%26aulast%3DVanier%26aufirst%3DG.%2BS.%26atitle%3DHigh-Efficiency%2520Solid%2520Phase%2520Peptide%2520Synthesis%2520%2528He-Spps%2529%26jtitle%3DOrg.%2520Lett.%26date%3D2014%26volume%3D16%26spage%3D940%26epage%3D943%26doi%3D10.1021%2Fol4036825" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gottlieb, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span> <span> </span><span class="NLM_article-title">Structure of Human NatA and Its Regulation by the Huntingtin Interacting Protein HYPK</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">925</span>– <span class="NLM_lpage">935.e8</span>, <span class="refDoi"> DOI: 10.1016/j.str.2018.04.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1016%2Fj.str.2018.04.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=29754825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC1cXptlyrsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2018&pages=925-935.e8&author=L.+Gottliebauthor=R.+Marmorstein&title=Structure+of+Human+NatA+and+Its+Regulation+by+the+Huntingtin+Interacting+Protein+HYPK&doi=10.1016%2Fj.str.2018.04.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of Human NatA and Its Regulation by the Huntingtin Interacting Protein HYPK</span></div><div class="casAuthors">Gottlieb, Leah; Marmorstein, Ronen</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">925-935.e8</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Co-translational N-terminal protein acetylation regulates many protein functions including degrdn., folding, interprotein interactions, and targeting.  Human NatA (hNatA), one of six conserved metazoan N-terminal acetyltransferases, contains Naa10 catalytic and Naa15 auxiliary subunits, and assocs. with the intrinsically disordered Huntingtin yeast two-hybrid protein K (HYPK).  We report on the crystal structures of hNatA and hNatA/HYPK, and assocd. biochem. and enzymic analyses.  We demonstrate that hNatA contains unique features: a stabilizing inositol hexaphosphate (IP6) mol. and a metazoan-specific Naa15 domain that mediates high-affinity HYPK binding.  We find that HYPK harbors intrinsic hNatA-specific inhibitory activity through a bipartite structure: a ubiquitin-assocd. domain that binds a hNaa15 metazoan-specific region and an N-terminal loop-helix region that distorts the hNaa10 active site.  We show that HYPK binding blocks hNaa50 targeting to hNatA, likely limiting Naa50 ribosome localization in vivo.  These studies provide a model for metazoan NAT activity and HYPK regulation of N-terminal acetylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4MVEFDfG3brVg90H21EOLACvtfcHk0lgoeMzh4DCesQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXptlyrsb0%253D&md5=64af1f850cc8f207c5f0fa17ed804a5d</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2018.04.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2018.04.003%26sid%3Dliteratum%253Aachs%26aulast%3DGottlieb%26aufirst%3DL.%26aulast%3DMarmorstein%26aufirst%3DR.%26atitle%3DStructure%2520of%2520Human%2520NatA%2520and%2520Its%2520Regulation%2520by%2520the%2520Huntingtin%2520Interacting%2520Protein%2520HYPK%26jtitle%3DStructure%26date%3D2018%26volume%3D26%26spage%3D925%26epage%3D935.e8%26doi%3D10.1016%2Fj.str.2018.04.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magin, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petersson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span> <span> </span><span class="NLM_article-title">Structure and Mechanism of Acetylation by the N-Terminal Dual Enzyme NatA/Naa50 Complex</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1057</span>– <span class="NLM_lpage">1070.e4</span>, <span class="refDoi"> DOI: 10.1016/j.str.2019.04.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1016%2Fj.str.2019.04.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=31155310" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVyju7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2019&pages=1057-1070.e4&author=S.+Dengauthor=R.+S.+Maginauthor=X.+Weiauthor=B.+Panauthor=E.+J.+Peterssonauthor=R.+Marmorstein&title=Structure+and+Mechanism+of+Acetylation+by+the+N-Terminal+Dual+Enzyme+NatA%2FNaa50+Complex&doi=10.1016%2Fj.str.2019.04.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and Mechanism of Acetylation by the N-Terminal Dual Enzyme NatA/Naa50 Complex</span></div><div class="casAuthors">Deng, Sunbin; Magin, Robert S.; Wei, Xuepeng; Pan, Buyan; Petersson, E. James; Marmorstein, Ronen</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1057-1070.e4</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">NatA co-translationally acetylates the N termini of over 40% of eukaryotic proteins and can assoc. with another catalytic subunit, Naa50, to form a ternary NatA/Naa50 dual enzyme complex (also called NatE).  The mol. basis of assocn. between Naa50 and NatA and the mechanism for how their assocn. affects their catalytic activities in yeast and human are poorly understood.  Here, we detd. the X-ray crystal structure of yeast NatA/Naa50 as a scaffold to understand coregulation of NatA/Naa50 activity in both yeast and human.  We find that Naa50 makes evolutionarily conserved contacts to both the Naa10 and Naa15 subunits of NatA.  These interactions promote catalytic crosstalk within the human complex, but do so to a lesser extent in the yeast complex, where Naa50 activity is compromised.  These studies have implications for understanding the role of the NatA/Naa50 complex in modulating the majority of the N-terminal acetylome in diverse species.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2COdqCfNww7Vg90H21EOLACvtfcHk0lgoeMzh4DCesQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVyju7fL&md5=0320692226fac3f4148f4d409f699187</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2019.04.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2019.04.014%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DS.%26aulast%3DMagin%26aufirst%3DR.%2BS.%26aulast%3DWei%26aufirst%3DX.%26aulast%3DPan%26aufirst%3DB.%26aulast%3DPetersson%26aufirst%3DE.%2BJ.%26aulast%3DMarmorstein%26aufirst%3DR.%26atitle%3DStructure%2520and%2520Mechanism%2520of%2520Acetylation%2520by%2520the%2520N-Terminal%2520Dual%2520Enzyme%2520NatA%252FNaa50%2520Complex%26jtitle%3DStructure%26date%3D2019%26volume%3D27%26spage%3D1057%26epage%3D1070.e4%26doi%3D10.1016%2Fj.str.2019.04.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deng, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McTiernan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnesen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span> <span> </span><span class="NLM_article-title">Molecular Basis for N-Terminal Acetylation by Human NatE and Its Modulation by HYPK</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">14</span>, <span class="refDoi"> DOI: 10.1038/s41467-020-14584-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1038%2Fs41467-020-14584-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=31911652" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=1-14&author=S.+Dengauthor=N.+McTiernanauthor=X.+Weiauthor=T.+Arnesenauthor=R.+Marmorstein&title=Molecular+Basis+for+N-Terminal+Acetylation+by+Human+NatE+and+Its+Modulation+by+HYPK&doi=10.1038%2Fs41467-020-14584-7"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fs41467-020-14584-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-020-14584-7%26sid%3Dliteratum%253Aachs%26aulast%3DDeng%26aufirst%3DS.%26aulast%3DMcTiernan%26aufirst%3DN.%26aulast%3DWei%26aufirst%3DX.%26aulast%3DArnesen%26aufirst%3DT.%26aulast%3DMarmorstein%26aufirst%3DR.%26atitle%3DMolecular%2520Basis%2520for%2520N-Terminal%2520Acetylation%2520by%2520Human%2520NatE%2520and%2520Its%2520Modulation%2520by%2520HYPK%26jtitle%3DNat.%2520Commun.%26date%3D2020%26volume%3D11%26spage%3D1%26epage%3D14%26doi%3D10.1038%2Fs41467-020-14584-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCullough, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, F. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span> <span> </span><span class="NLM_article-title">Structural and Functional Role of Acetyltransferase HMOF K274 Autoacetylation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>291</i></span>,  <span class="NLM_fpage">18190</span>– <span class="NLM_lpage">18198</span>, <span class="refDoi"> DOI: 10.1074/jbc.M116.736264</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1074%2Fjbc.M116.736264" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=27382063" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVWjtrfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2016&pages=18190-18198&author=C.+E.+McCulloughauthor=S.+Songauthor=M.+H.+Shinauthor=F.+B.+Johnsonauthor=R.+Marmorstein&title=Structural+and+Functional+Role+of+Acetyltransferase+HMOF+K274+Autoacetylation&doi=10.1074%2Fjbc.M116.736264"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and Functional Role of Acetyltransferase hMOF K274 Autoacetylation</span></div><div class="casAuthors">McCullough, Cheryl E.; Song, Shufei; Shin, Michael H.; Johnson, F. Brad; Marmorstein, Ronen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">291</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">18190-18198</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Many histone acetyltransferases undergo autoacetylation, either through chem. or enzymic means, to potentiate enzymic cognate substrate lysine acetylation, although the mode and mol. role of such autoacetylation is poorly understood.  The MYST family of histone acetyltransferases is autoacetylated at an active site lysine residue to facilitate cognate substrate lysine binding and acetylation.  Here, we report on a detailed mol. investigation of Lys-274 autoacetylation of the human MYST protein Males Absent on the First (hMOF).  A mutational scan of hMOF Lys-274 reveals that all amino acid substitutions of this residue are able to bind cofactor but are significantly destabilized, both in vitro and in cells, and are catalytically inactive for cognate histone H4 peptide lysine acetylation.  The x-ray crystal structure of a hMOF K274P mutant suggests that the reduced stability and catalytic activity stems from a disordering of the residue 274-harboring a α2-β7 loop.  We also provide structural evidence that a C316S/E350Q mutant, which is defective for cognate substrate lysine acetylation; and biochem. evidence that a K268M mutant, which is defective for Lys-274 chem. acetylation in the context of a K274-peptide, can still undergo quant. K274 autoacetylation.  Together, these studies point to the crit. and specific role of hMOF Lys-274 autoacetylation in hMOF stability and cognate substrate acetylation and argues that binding of Ac-CoA to hMOF likely drives Lys-274 autoacetylation for subsequent cognate substrate acetylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJlS94p61LA7Vg90H21EOLACvtfcHk0lgoeMzh4DCesQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVWjtrfP&md5=08cdceca89e76d9a44389fc3022380b8</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M116.736264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M116.736264%26sid%3Dliteratum%253Aachs%26aulast%3DMcCullough%26aufirst%3DC.%2BE.%26aulast%3DSong%26aufirst%3DS.%26aulast%3DShin%26aufirst%3DM.%2BH.%26aulast%3DJohnson%26aufirst%3DF.%2BB.%26aulast%3DMarmorstein%26aufirst%3DR.%26atitle%3DStructural%2520and%2520Functional%2520Role%2520of%2520Acetyltransferase%2520HMOF%2520K274%2520Autoacetylation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2016%26volume%3D291%26spage%3D18190%26epage%3D18198%26doi%3D10.1074%2Fjbc.M116.736264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">Processing of X-Ray Diffraction Data Collected in Oscillation Mode</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>276</i></span>,  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">326</span>, <span class="refDoi"> DOI: 10.1016/S0076-6879(97)76066-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=27754618" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=1997&pages=307-326&author=Z.+Otwinowskiauthor=W.+Minor&title=Processing+of+X-Ray+Diffraction+Data+Collected+in+Oscillation+Mode&doi=10.1016%2FS0076-6879%2897%2976066-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Processing of x-ray diffraction data collected in oscillation mode</span></div><div class="casAuthors">Otwinowski, Zbyszek; Minor, Wladek</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">276</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part A</span>),
    <span class="NLM_cas:pages">307-326</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Macromol. crystallog. is an iterative process.  Rarely do the first crystals provide all the necessary data to solve the biol. problem being studied.  Each step benefits from experience learned in previous steps.  To monitor the progress, the HKL package provides 2 tools: (1) statistics, both weighted (χ2) and unweighted (R-merge), are provided, and the Bayesian reasoning and multicomponent error model facilitates obtaining the proper error ests. and (2) visualization of the process plays a double role by helping the operator to confirm that the process of data redn., including the resulting statistics, is correct, and allowing one to evaluate problems for which there are no good statistical criteria.  Visualization also provides confidence that the point of diminishing returns in data collection and redn. has been reached.  At that point, the effort should be directed to solving the structure.  The methods presented here have been applied to solve a large variety of problems, from inorg. mols. with 5 Å unit cell to rotavirus of 700 Å diam. crystd. in 700 × 1000 × 1400 Å cell.  Overall quality of the method was tested by many researchers by successful application of the programs to MAD structure detns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrA3fvBOHqhhrVg90H21EOLACvtfcHk0liBOeGItdrRuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivFehsbw%253D&md5=c9536971d4e32cc35352c40fb9368131</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DMinor%26aufirst%3DW.%26atitle%3DProcessing%2520of%2520X-Ray%2520Diffraction%2520Data%2520Collected%2520in%2520Oscillation%2520Mode%26jtitle%3DMethods%2520Enzymol.%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326%26doi%3D10.1016%2FS0076-6879%2897%2976066-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkóczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: A Comprehensive Python-Based System for Macromolecular Structure Solution</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+A+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0liBOeGItdrRuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520A%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: Model-Building Tools for Molecular Graphics</span>. <i>Acta Crystallogr. Sect. D Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+Model-Building+Tools+for+Molecular+Graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0liBOeGItdrRuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520Model-Building%2520Tools%2520for%2520Molecular%2520Graphics%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i28"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00141">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_34521"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00141?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00141</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">HRMS and HPLC spectra of compounds <b>1</b>–<b>10 (</b>Figures S1,S2<b>)</b>; IC<sub>50</sub> fitting curves of compounds <b>1</b>–<b>10</b> (Figure S3); cell permeability study (Figure S4); and data statistics for NatD crystal structures in complex with inhibitors (Table S1) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00141/suppl_file/jm1c00141_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings for all synthesized inhibitors (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00141/suppl_file/jm1c00141_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00141/suppl_file/jm1c00141_si_001.pdf">jm1c00141_si_001.pdf (1.06 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00141/suppl_file/jm1c00141_si_002.csv">jm1c00141_si_002.csv (1.61 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates for the structure of human NatD in complex with compound <b>5 (</b>PDB ID: 7KD7<b>) and 6 (</b>PDB ID: 7KPU<b>)</b> have been deposited in the Protein Data Bank. Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00141&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-12%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00141" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00141" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67994cc48ea23d94","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
