
<html lang="en"     class="pb-page"  data-request-id="5bf40f85-40a2-4375-bc17-a400c7ab2d41"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.0c01063;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2020.63.issue-21"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of Ketone-Based Covalent Inhibitors of Coronavirus 3CL Proteases for the Potential Therapeutic Treatment of COVID-19" /></meta><meta name="dc.Creator" content="Robert L.  Hoffman" /></meta><meta name="dc.Creator" content="Robert S.  Kania" /></meta><meta name="dc.Creator" content="Mary A.  Brothers" /></meta><meta name="dc.Creator" content="Jay F.  Davies" /></meta><meta name="dc.Creator" content="Rose A.  Ferre" /></meta><meta name="dc.Creator" content="Ketan S.  Gajiwala" /></meta><meta name="dc.Creator" content="Mingying  He" /></meta><meta name="dc.Creator" content="Robert J.  Hogan" /></meta><meta name="dc.Creator" content="Kirk  Kozminski" /></meta><meta name="dc.Creator" content="Lilian Y.  Li" /></meta><meta name="dc.Creator" content="Jonathan W.  Lockner" /></meta><meta name="dc.Creator" content="Jihong  Lou" /></meta><meta name="dc.Creator" content="Michelle T.  Marra" /></meta><meta name="dc.Creator" content="Lennert J.  Mitchell, Jr." /></meta><meta name="dc.Creator" content="Brion W.  Murray" /></meta><meta name="dc.Creator" content="James A.  Nieman" /></meta><meta name="dc.Creator" content="Stephen  Noell" /></meta><meta name="dc.Creator" content="Simon P.  Planken" /></meta><meta name="dc.Creator" content="Thomas  Rowe" /></meta><meta name="dc.Creator" content="Kevin  Ryan" /></meta><meta name="dc.Creator" content="George J.  Smith, III" /></meta><meta name="dc.Creator" content="James E.  Solowiej" /></meta><meta name="dc.Creator" content="Claire M.  Steppan" /></meta><meta name="dc.Creator" content="Barbara  Taggart" /></meta><meta name="dc.Description" content="The novel coronavirus disease COVID-19 that emerged in 2019 is caused by the virus SARS CoV-2 and named for its close genetic similarity to SARS CoV-1 that caused severe acute respiratory syndrome ..." /></meta><meta name="Description" content="The novel coronavirus disease COVID-19 that emerged in 2019 is caused by the virus SARS CoV-2 and named for its close genetic similarity to SARS CoV-1 that caused severe acute respiratory syndrome ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="October 15, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01063" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2020 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01063" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01063" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01063" /></link>
        
    
    

<title>Discovery of Ketone-Based Covalent Inhibitors of Coronavirus 3CL Proteases for the Potential Therapeutic Treatment of COVID-19 | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01063" /></meta><meta property="og:title" content="Discovery of Ketone-Based Covalent Inhibitors of Coronavirus 3CL Proteases for the Potential Therapeutic Treatment of COVID-19" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0020.jpeg" /></meta><meta property="og:description" content="The novel coronavirus disease COVID-19 that emerged in 2019 is caused by the virus SARS CoV-2 and named for its close genetic similarity to SARS CoV-1 that caused severe acute respiratory syndrome (SARS) in 2002. Both SARS coronavirus genomes encode two overlapping large polyproteins, which are cleaved at specific sites by a 3C-like cysteine protease (3CLpro) in a post-translational processing step that is critical for coronavirus replication. The 3CLpro sequences for CoV-1 and CoV-2 viruses are 100% identical in the catalytic domain that carries out protein cleavage. A research effort that focused on the discovery of reversible and irreversible ketone-based inhibitors of SARS CoV-1 3CLpro employing ligand-protease structures solved by X-ray crystallography led to the identification of 3 and 4. Preclinical experiments reveal 4 (PF-00835231) as a potent inhibitor of CoV-2 3CLpro with suitable pharmaceutical properties to warrant further development as an intravenous treatment for COVID-19." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01063"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01063">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01063&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01063&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01063&amp;href=/doi/10.1021/acs.jmedchem.0c01063" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 21</span><span class="cit-fg-pageRange">, 12725-12747</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/21" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01059" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01065" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Ketone-Based Covalent Inhibitors of Coronavirus 3CL Proteases for the Potential Therapeutic Treatment of COVID-19</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Robert L. Hoffman</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Robert L. Hoffman</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pfizer Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#77051815120503591b591f1811111a16193707111e0d12055914181a"><span class="__cf_email__" data-cfemail="b8cad7daddcacc96d496d0d7deded5d9d6f8c8ded1c2ddca96dbd7d5">[email protected]</span></a>. Tel.: +1-858-622-3000.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Robert+L.++Hoffman">Robert L. Hoffman</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2030-9094" title="Orcid link">http://orcid.org/0000-0002-2030-9094</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Robert S. Kania</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Robert S. Kania</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pfizer Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Robert+S.++Kania">Robert S. Kania</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mary A. Brothers</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mary A. Brothers</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pfizer Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mary+A.++Brothers">Mary A. Brothers</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jay F. Davies</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jay F. Davies</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pfizer Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jay+F.++Davies">Jay F. Davies</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rose A. Ferre</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rose A. Ferre</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pfizer Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rose+A.++Ferre">Rose A. Ferre</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ketan S. Gajiwala</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ketan S. Gajiwala</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pfizer Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ketan+S.++Gajiwala">Ketan S. Gajiwala</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mingying He</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mingying He</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pfizer Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mingying++He">Mingying He</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Robert J. Hogan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Robert J. Hogan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Southern Research Institute, 2000 9th Avenue South, Birmingham, Alabama 35205 United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Robert+J.++Hogan">Robert J. Hogan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kirk Kozminski</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kirk Kozminski</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pfizer Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kirk++Kozminski">Kirk Kozminski</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lilian Y. Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lilian Y. Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pfizer Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lilian+Y.++Li">Lilian Y. Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jonathan W. Lockner</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jonathan W. Lockner</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pfizer Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jonathan+W.++Lockner">Jonathan W. Lockner</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6148-0579" title="Orcid link">http://orcid.org/0000-0001-6148-0579</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jihong Lou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jihong Lou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pfizer Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jihong++Lou">Jihong Lou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michelle T. Marra</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michelle T. Marra</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pfizer Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michelle+T.++Marra">Michelle T. Marra</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lennert J. Mitchell Jr.</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lennert J. Mitchell, Jr.</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pfizer Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lennert+J.++Mitchell%2C++Jr.">Lennert J. Mitchell, Jr.</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Brion W. Murray</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Brion W. Murray</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pfizer Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Brion+W.++Murray">Brion W. Murray</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">James A. Nieman</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">James A. Nieman</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pfizer Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=James+A.++Nieman">James A. Nieman</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5087-0819" title="Orcid link">http://orcid.org/0000-0002-5087-0819</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stephen Noell</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stephen Noell</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pfizer Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stephen++Noell">Stephen Noell</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Simon P. Planken</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Simon P. Planken</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pfizer Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Simon+P.++Planken">Simon P. Planken</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8901-8992" title="Orcid link">http://orcid.org/0000-0001-8901-8992</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Thomas Rowe</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Thomas Rowe</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Southern Research Institute, 2000 9th Avenue South, Birmingham, Alabama 35205 United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Thomas++Rowe">Thomas Rowe</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kevin Ryan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kevin Ryan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pfizer Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kevin++Ryan">Kevin Ryan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">George J. Smith III</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">George J. Smith, III</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pfizer Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=George+J.++Smith%2C++III">George J. Smith, III</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">James E. Solowiej</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">James E. Solowiej</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pfizer Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=James+E.++Solowiej">James E. Solowiej</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Claire M. Steppan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Claire M. Steppan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Pfizer Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Claire+M.++Steppan">Claire M. Steppan</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Barbara Taggart</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Barbara Taggart</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Southern Research Institute, 2000 9th Avenue South, Birmingham, Alabama 35205 United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Barbara++Taggart">Barbara Taggart</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01063&amp;href=/doi/10.1021%2Facs.jmedchem.0c01063" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 21</span><span class="cit-pageRange">, 12725–12747</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">October 15, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>19 June 2020</li><li><span class="item_label"><b>Published</b> online</span>15 October 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 12 November 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01063" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01063</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01063"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">12707</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">46</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01063" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Ketone-Based Covalent Inhibitors of Coronavirus 3CL Proteases for the Potential Therapeutic Treatment of COVID-19&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;L. Hoffman&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;S. Kania&quot;},{&quot;first_name&quot;:&quot;Mary&quot;,&quot;last_name&quot;:&quot;A. Brothers&quot;},{&quot;first_name&quot;:&quot;Jay&quot;,&quot;last_name&quot;:&quot;F. Davies&quot;},{&quot;first_name&quot;:&quot;Rose&quot;,&quot;last_name&quot;:&quot;A. Ferre&quot;},{&quot;first_name&quot;:&quot;Ketan&quot;,&quot;last_name&quot;:&quot;S. Gajiwala&quot;},{&quot;first_name&quot;:&quot;Mingying&quot;,&quot;last_name&quot;:&quot;He&quot;},{&quot;first_name&quot;:&quot;Robert&quot;,&quot;last_name&quot;:&quot;J. Hogan&quot;},{&quot;first_name&quot;:&quot;Kirk&quot;,&quot;last_name&quot;:&quot;Kozminski&quot;},{&quot;first_name&quot;:&quot;Lilian&quot;,&quot;last_name&quot;:&quot;Y. Li&quot;},{&quot;first_name&quot;:&quot;Jonathan&quot;,&quot;last_name&quot;:&quot;W. Lockner&quot;},{&quot;first_name&quot;:&quot;Jihong&quot;,&quot;last_name&quot;:&quot;Lou&quot;},{&quot;first_name&quot;:&quot;Michelle&quot;,&quot;last_name&quot;:&quot;T. Marra&quot;},{&quot;first_name&quot;:&quot;Lennert&quot;,&quot;last_name&quot;:&quot;J. Mitchell&quot;},{&quot;first_name&quot;:&quot;Brion&quot;,&quot;last_name&quot;:&quot;W. Murray&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;A. Nieman&quot;},{&quot;first_name&quot;:&quot;Stephen&quot;,&quot;last_name&quot;:&quot;Noell&quot;},{&quot;first_name&quot;:&quot;Simon&quot;,&quot;last_name&quot;:&quot;P. Planken&quot;},{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;Rowe&quot;},{&quot;first_name&quot;:&quot;Kevin&quot;,&quot;last_name&quot;:&quot;Ryan&quot;},{&quot;first_name&quot;:&quot;George&quot;,&quot;last_name&quot;:&quot;J. Smith&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;E. Solowiej&quot;},{&quot;first_name&quot;:&quot;Claire&quot;,&quot;last_name&quot;:&quot;M. Steppan&quot;},{&quot;first_name&quot;:&quot;Barbara&quot;,&quot;last_name&quot;:&quot;Taggart&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;10&quot;,&quot;day&quot;:&quot;15&quot;,&quot;issue&quot;:&quot;21&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;12725-12747&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01063&quot;},&quot;abstract&quot;:&quot;The novel coronavirus disease COVID-19 that emerged in 2019 is caused by the virus SARS CoV-2 and named for its close genetic similarity to SARS CoV-1 that caused severe acute respiratory syndrome (SARS) in 2002. Both SARS coronavirus genomes encode two overlapping large polyproteins, which are cleaved at specific sites by a 3C-like cysteine protease (3CLpro) in a post-translational processing step that is critical for coronavirus replication. The 3CLpro sequences for CoV-1 and CoV-2 viruses are 100% identical in the catalytic domain that carries out protein cleavage. A research effort that focused on the discovery of reversible and irreversible ketone-based inhibitors of SARS CoV-1 3CLpro employing ligand-protease structures solved by X-ray crystallography led to the identification of 3 and 4. Preclinical experiments reveal 4 (PF-00835231) as a potent inhibitor of CoV-2 3CLpro with suitable pharmaceutical properties to warrant further development as an intravenous treatment for COVID-19.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01063&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01063" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01063&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01063" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01063&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01063" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01063&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01063&amp;href=/doi/10.1021/acs.jmedchem.0c01063" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01063" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01063" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01063%26sid%3Dliteratum%253Aachs%26pmid%3D33054210%26genre%3Darticle%26aulast%3DHoffman%26date%3D2020%26atitle%3DDiscovery%2Bof%2BKetone-Based%2BCovalent%2BInhibitors%2Bof%2BCoronavirus%2B3CL%2BProteases%2Bfor%2Bthe%2BPotential%2BTherapeutic%2BTreatment%2Bof%2BCOVID-19%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D21%26spage%3D12725%26epage%3D12747%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292645" title="SARS">SARS</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291465" title="Structure activity relationship">Structure activity relationship</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=291763" title="Noncovalent interactions">Noncovalent interactions</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/21" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/jmcmar.2020.63.issue-21/20201112/jmcmar.2020.63.issue-21.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/medium/jm0c01063_0020.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01063&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The novel coronavirus disease COVID-19 that emerged in 2019 is caused by the virus SARS CoV-2 and named for its close genetic similarity to SARS CoV-1 that caused severe acute respiratory syndrome (SARS) in 2002. Both SARS coronavirus genomes encode two overlapping large polyproteins, which are cleaved at specific sites by a 3C-like cysteine protease (3CL<sup>pro</sup>) in a post-translational processing step that is critical for coronavirus replication. The 3CL<sup>pro</sup> sequences for CoV-1 and CoV-2 viruses are 100% identical in the catalytic domain that carries out protein cleavage. A research effort that focused on the discovery of reversible and irreversible ketone-based inhibitors of SARS CoV-1 3CL<sup>pro</sup> employing ligand-protease structures solved by X-ray crystallography led to the identification of <b>3</b> and <b>4</b>. Preclinical experiments reveal <b>4</b> (<b>PF-00835231</b>) as a potent inhibitor of CoV-2 3CL<sup>pro</sup> with suitable pharmaceutical properties to warrant further development as an intravenous treatment for COVID-19.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec articleNote"><h4><span></span>  Note</h4><p class="first last">This article is made available via the <a href="/page/vi/chemistry_coronavirus_research" class="ext-link">ACS COVID-19 subset</a> for unrestricted RESEARCH re-use and analyses in any form or by any means with acknowledgement of the original source. These permissions are granted for the duration of the World Health Organization (WHO) declaration of COVID-19 as a global pandemic.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74808" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74808" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In late 2019, cases of an unknown respiratory tract infection were first reported in Wuhan, China. By February 2020, the novel coronavirus SARS CoV-2 was identified as the causative agent for COVID-19.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> Genome analysis of this virus revealed a high similarity to SARS CoV-1, the coronavirus that caused severe acute respiratory syndrome (SARS) in 2002–2003.<a onclick="showRef(event, 'ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4">(2−4)</a> Like SARS CoV-1, which resulted in 799 deaths among the 8464 probable cases,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> SARS CoV-2 can induce fever, coughing, and difficulty breathing that rapidly becomes more serious in some cases. The spread of SARS CoV-2 has been more extensive than that of SARS CoV-1, causing a global pandemic with the current number of worldwide infections surpassing eight million and deaths surpassing 400,000.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> The SARS CoV-1 genome encodes for two large polyproteins pp1a (∼450 kDa) and pp1ab (∼750 kDa) that contain overlapping sequences and include a 3C-like cysteine protease (3CL<sup>pro</sup>). The function of this internally encoded 3CL<sup>pro</sup> is integral to the processing of these proteins and critical for viral replication.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> The SARS CoV-1 3CL<sup>pro</sup> shares a high degree of structural homology and similar substrate specificity with the coronavirus 3C-like cysteine proteases of hCoV 229E and TGEV<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> but is most similar to the SARS CoV-2 3CL<sup>pro</sup>. Specifically, the SARS CoV-1 and SARS CoV-2 share 96% identity between their respective 3CL<sup>pro</sup> sequences and 100% identity in the active site.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> A recent report by Dai et al. demonstrates that crystallographic information and structure–activity relationships obtained with the SARS CoV-1 3CL<sup>pro</sup> could facilitate the design of potent SARS CoV-2 3CL<sup>pro</sup> inhibitors with antiviral potency.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a></div><div class="NLM_p">There are numerous reports of reversible cysteine protease inhibitors, which include aldehydes,<a onclick="showRef(event, 'ref10 ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref10 ref11 ref12 ref13">(10−13)</a> thio- or oxymethylketones,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> cyclic ketones,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> amidomethylketones,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> nitriles,<a onclick="showRef(event, 'ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref17 ref18">(17,18)</a> or various 1,2-dicarbonyl motifs.<a onclick="showRef(event, 'ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref19 ref20">(19,20)</a> The electrophilic carbon of these chemotypes reacts reversibly with the sulfur atom of an active-site cysteine forming a covalently bound tetrahedral complex. Stabilization of this charged protein–ligand transition state by an “oxyanion hole” present in the active site has been observed with certain inhibitor classes by X-ray crystallography and NMR.<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a> A recent report describes the optimization of a series of peptidomimetics designed with an α-ketoamide warhead, which achieves broad-spectrum inhibition against the M<sup>pro</sup> for several coronaviruses and enteroviruses. Importantly, the reported protease potencies correlate to antiviral potencies, and the authors suggest that the reported SARS CoV-1 3CL<sup>pro</sup> activity of their lead could be predictive of the SARS CoV-2 3CL<sup>pro</sup> activity.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></div><div class="NLM_p">Alternative modes of enzyme inhibition are covalently bound irreversible inhibitors. While chemically reactive affinity labels such as chloromethylketones (CMKs) have demonstrated potent protease inhibition and in certain instances <i>in vivo</i> activity, their inherent chemical reactivity precludes development as clinical agents due to safety concerns.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> However, acyloxymethylketones first reported by Krantz as “quiescent affinity labels” for cathepsin B have demonstrated potent levels of enzyme inhibition with relatively low chemical reactivity.<a onclick="showRef(event, 'ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref25 ref26">(25,26)</a> Mechanistic studies of this class of inhibitors with the cysteine protease caspase-1 support the intermediacy of a thiohemiketal arising from the initial 1,2-carbonyl attack. Orientation and stabilization of the acyloxy leaving group within the S<sub>1</sub>′ domain (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) of the protease can facilitate S<sub>N</sub>2 displacement <i>via</i> sulfur migration, resulting in irreversible or bimodal (reversible inhibition followed by slow inactivation) inhibition.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/medium/jm0c01063_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0002.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Subsite nomenclature for proteolytic enzymes is shown. Amino acid residues to the left of the polypeptide scissile amide bond are numbered sequentially, beginning with P<sub>1</sub> and increasing toward the N-terminus. Amino acid residues to the right of the scissile bond are numbered sequentially, beginning with P<sub>1</sub>′ and increasing toward the C-terminus. Complimentary regions of the protease active site employ the corresponding S numbering.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01063&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Following the SARS outbreak, a CoV-1 3CL<sup>pro</sup> homology model was published in 2003 with comparisons made to the human rhinovirus (HRV) 3C<sup>pro</sup>.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Although these two proteases share very little sequence conservation, the substrate consensus sequences have a Gln residue in common at P<sub>1</sub>. Analysis of the HRV 3C<sup>pro</sup> crystal structure with <b>1</b> (rupintrivir; <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) revealed binding interactions that are like those observed for <b>1</b> bound to TGEV 3CL<sup>pro</sup>, which was used in the construction of their homology model. The authors concluded that Michael acceptor <b>1</b> would serve as a starting point to expedite the identification of a potent SARS CoV-1 3CL<sup>pro</sup> inhibitor.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/medium/jm0c01063_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0003.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. HRV clinical candidate <b>rupintrivir</b> and SARS CoV-1 3CL<sup>pro</sup> inhibitor <b>2</b> employing Michael acceptor-based warheads along with optimized SARS CoV-1 3CL<sup>pro</sup> inhibitors <b>3</b> and <b>4</b> containing a carbonyl designed for attack by the catalytic cysteine residue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01063&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Performing a similar analysis, we tested <b>1</b> and additional HRV 3C<sup>pro</sup> inhibitors that displayed very weak to unmeasurable SARS CoV-1 3CL<sup>pro</sup> inhibition. Concurrently, new Michael acceptor derivatives were designed that tested P<sub>1</sub>/P<sub>2</sub> alterations and truncation of the P<sub>3</sub>/P<sub>4</sub> binding motif to optimize SARS CoV-1 3CL<sup>pro</sup> inhibition. Compound <b>2</b>, a derivative that contains an indole capping group at P<sub>2</sub>, displayed modest levels of irreversible inhibition (<i>k</i><sub>obs</sub>/<i>I</i> = 586 ± 11 M<sup>–1</sup> s<sup>–1</sup>) that enabled cocrystallization in complex with SARS CoV-1 3CL<sup>pro</sup> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XHO">6XHO</a>).<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Comparison of this structure with the complex of <b>1</b> in HRV 3C<sup>pro</sup> helped to rationalize the poor performance of the warhead, which had been successful in HRV. The structure of the SARS CoV-1 3CL<sup>pro</sup> complex revealed an eclipsed torsion about the resulting sp<sup>3</sup> α,β-carbons from the Michael acceptor (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). Furthermore, the carbonyl oxygen sp<sup>2</sup> lone pair electrons are not aligned well with either 3CL<sup>pro</sup> hydrogen bond donor of the oxyanion hole. Specifically, the hydrogen bond between the carbonyl oxygen and the Gly143 NH in the SARS CoV-1 3CL<sup>pro</sup> complex is 3.4 Å, while the corresponding hydrogen bond distance is a much more favorable 2.8 Å in the complex of <b>1</b> with HRV 3C<sup>pro</sup>. In contrast to these unfavorable interactions, the constrained lactam<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> at the P<sub>1</sub> site, which was designed to be isosteric with the highly conserved P<sub>1</sub> glutamine present in all SARS CoV substrates, is well positioned in the S<sub>1</sub> pocket making a favorable hydrogen bond to His163. The NH of the lactam is within the hydrogen bond distance to Glu166 and the backbone oxygen of Phe140 in the 3CL<sup>pro</sup>. Additionally, the NH of the indole P<sub>2</sub> capping moiety makes a hydrogen bond with the protein backbone while it extends across an otherwise lipophilic surface over the P<sub>3</sub> pocket.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/medium/jm0c01063_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0004.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cocrystal structure of the covalent adduct of <b>2</b> bound to SARS CoV-1 3CL<sup>pro</sup> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XHO">6XHO</a>). The Connolly surface for the inhibitor binding pocket is shown in gray. The bonds are represented as the dashed lines, with the bond length measured between heavy atoms. The views are centered on warhead with active-site interactions and of complete inhibitor–protein interactions depicted.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01063&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Although the structure of <b>2</b> in complex with SARS CoV-1 3CL<sup>pro</sup> is the final product, the above analysis is expected to apply similarly to the more relevant and structurally close transition state. Accordingly, a focused design strategy aimed at replacing the warhead while retaining the P<sub>1</sub> lactam and utilizing the methoxy indole capping group in the early rounds of optimization was initiated. Specifically, an ideal electrophilic warhead would be bioisosteric, with the scissile amide carbonyl of peptidyl substrates ensuring proper alignment within the oxyanion hole of the protease. To better mimic the tetrahedral intermediate generated in the amide bond cleavage, we designed ketone-based covalent reversible and irreversible inhibitors of SARS CoV-1 3CL<sup>pro</sup>, as illustrated by <b>3</b> and <b>4</b>. Compound <b>4</b> was selected as a development candidate for SARS CoV-1 but with the successful public health response that ended the 2003 pandemic, the clinical advancement of <b>4</b> was suspended. Following the COVID-19 outbreak, testing has demonstrated that <b>4</b> is a potent inhibitor of the SARS CoV-2 3CL<sup>pro</sup> (<i>K</i><sub>i</sub> = 0.27 ± 0.1 nM) and a cocrystal structure with <b>4</b> bound in the active site has been solved. In addition to the potent inhibition of the 3CL<sup>pro</sup> and viral replication of several coronaviruses, <b>4</b> possesses solubility as well as metabolic and chemical stability characteristics, which are consistent with a continuous infusion IV therapeutic treatment for COVID-19. We report the research focused on the discovery of reversible and irreversible ketone-based inhibitors of SARS CoV-1 3CL<sup>pro</sup> employing ligand-protease structures solved by X-ray crystallography, which led to the identification of <b>4</b> as a molecule warranting further evaluation for its potential to treat coronaviruses, including COVID-19.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87143" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87143" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Chemistry</h3><div class="NLM_p">Central to our strategy of examining ketone-based thiophiles was the preparation of an array of intermediates containing diverse methylketone moieties and protecting groups. The preparation of halomethylketone intermediates was accomplished in a two-step procedure by the generation of a diazoketone intermediate<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> or directly by a modified Kowalski–Haque reaction,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> as depicted in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>.</div><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/medium/jm0c01063_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0007.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Preparation of P<sub>1</sub> Synthetic Intermediates</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01063&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(i) 3 M NaOH, MeOH (95%), (ii) isobutylchloroformate (IBCF), triethylamine (TEA), tetrahydrofuran (THF), (iii) CH<sub>2</sub>N<sub>2</sub>/ether (92%), (iv) 4 M HCl/dioxane (83%), (v) 1.0 equiv 48% HBr (83%), (vi) HNCH<sub>3</sub>(OCH<sub>3</sub>), EDC, HOBt, NMM, dichloromethane (DCM) (79%), (vii) Mg, HgCl<sub>2</sub>, BOM-Cl, THF, −78°C (50%), and (viii) lithium diisopropylamide (LDA), ClCH<sub>2</sub>I, THF (54%).</p></p></figure><div class="NLM_p">Saponification of amino ester <b>5</b><a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> afforded the corresponding amino acid <b>6</b> in nearly quantitative yield. Acid <b>6</b> was treated with isobutylchloroformate and triethylamine, and the resulting mixed anhydride was reacted with diazomethane<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> to provide high isolated yields of <b>7</b>. Diazoketone <b>7</b> was treated with hydrochloric acid for simultaneous nitrogen deprotection and conversion to the chloromethylketone (CMK) intermediate <b>10</b>. Alternatively, the <i>N</i>-Boc-protected bromomethylketone was prepared by treatment of <b>7</b> with stoichiometric quantities of 48% HBr in dichloromethane to provide <b>11</b>. Another preparation of CMK intermediates that avoids the use of diazomethane was accomplished by the reaction of <b>5</b> with excess LDA and chloroiodomethane to afford <b>8</b> in moderate yields. A direct approach to oxymethylketone intermediate <b>9</b> was achieved by conversion of acid <b>6</b> to the corresponding Weinreb amide followed by treatment with the Grignard generated from benzyl chloromethyl ether.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a></div><div class="NLM_p">A key element for rapid access to substituted methylketone derivatives was the generation of the fully elaborated CMK <b>12</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). The utility of the CMK moiety to prepare acyloxy- and hydroxy-substituted methylketones is well established. Elaboration of <b>10</b> in a sequence of deprotection/1-[<i>bis</i>(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxid hexafluorophosphate (HATU)-mediated peptide couplings provided CMK derivative <b>12</b> in moderate yield with no epimerization observed. Reaction of <b>12</b> with various carboxylic acids in the presence of cesium fluoride at 60 °C provided acyloxymethylketones <b>13</b>–<b>28</b>. Using this synthetic approach with substitution of P<sub>2</sub> and amine capping structural elements provided inhibitors <b>3</b> and <b>29</b>–<b>31</b>.</div><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/medium/jm0c01063_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0008.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Preparation of Substituted Acyloxymethylketones and Aminomethylketones</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01063&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(i) Boc-Leu-OH, HATU, NMM, <i>N</i>,<i>N</i>-dimethylformamide (DMF), 0 °C (64%), (ii) 4 M HCl/dioxane, (iii) HATU, 4-MeO-2-indolecarboxylic acid, NMM, DMF, 0 °C (61%), and (iv) CsF, DMF, carboxylic acid, 60 °C (43–91%).</p></p></figure><div class="NLM_p">Hydroxymethylketone (HMK) derivatives were accessed by two complimentary approaches illustrated in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>. Elaboration of the P<sub>1</sub>′-protected oxygen intermediate <b>9</b> by sequential amide bond coupling reactions provided benzylether derivatives such as <b>34</b> in moderate yields with a low propensity for epimerization at the P<sub>1</sub> center. Hydrogenolytic debenzylation readily afforded the final HMK inhibitors exemplified by <b>36</b>. Alternatively, fully elaborated CMK derivatives such as <b>35</b> were reacted with benzoylformic acid followed by methanolysis to generate HMK final products such as <b>36</b>.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Alkylation of the terminal hydroxyl was achieved under microwave-assisted conditions in the presence of silver (I) oxide to afford alkoxymethylketone inhibitors, as illustrated by <b>37</b>. Using these two synthetic approaches with substitution of P<sub>2</sub> and amine capping structural elements provided inhibitors <b>4</b> and <b>38</b>–<b>50</b>.</div><figure id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/medium/jm0c01063_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0009.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Preparation of Hydroxymethylketone Derivatives</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01063&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(i) 4 M HCl/dioxane, (ii) Boc-Leu-OH, HATU, NMM, DMF, 0 °C, (iii) HATU, 4-MeO-2-indolecarboxylic acid, NMM, 0 °C, (iv) Pd/C, H<sub>2</sub>, EtOH (77%), (v) CsF, DMF, benzoylformic acid, 60 °C, and then MeOH, cat. K<sub>2</sub>CO<sub>3</sub> (53%), and (vi) CH<sub>3</sub>I, Ag<sub>2</sub>O, DCE (9%).</p></p></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Biological Evaluation</h3><div id="sec2_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i9" class="anchor-spacer"></div><h4 class="article-section__title" id="_i9"> Irreversible Inhibitors of SARS CoV-1 3CL<sup>pro</sup></h4><div class="NLM_p">The warhead reactivity and binding affinity to P<sub>1</sub>′ are both important design considerations to ensure specificity of irreversible inhibitors. Krantz has demonstrated that the inherent chemical reactivity of acyloxymethylketones can be tuned to inhibit cysteine proteases through the modulation of substituent effects on the carboxylate leaving group.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> The importance of leaving group “strength” was confirmed by the strong dependence between the p<i>K</i><sub>a</sub> of the leaving carboxylate and cathepsin B enzyme inhibition. These otherwise weak electrophiles are elegant “quiescent” inhibitors that harness the very same interactions with catalytic residues that lead to proteolysis rate acceleration. Molecular modeling of a 2,6-dichlorobenzoate design with SARS CoV-1 3CL<sup>pro</sup> indicated that a low strain conformation of the ketone carbonyl was aligned in the oxyanion hole and the substituted benzoate is accommodated within the P<sub>1</sub>′ site. A series of acyloxymethylketone derivatives were prepared, as depicted in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SARS CoV-1 3CL<sup>pro</sup> Inhibition Data for Acyloxymethylketone Inhibitors</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/medium/jm0c01063_0010.gif" alt="" id="fx2" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="left"> </th><th class="colsep0 rowsep0" align="left"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char="±">SARS CoV-1 3CL<sup>pro</sup> FRET<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">entry</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center" char="±"><i>k</i><sub>obs</sub>/<i>I</i> (M<sup>–1</sup> s<sup>–1</sup>)</th><th class="colsep0 rowsep0" align="center" char="±">IC<sub>50</sub> (nM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±">283,039 ± 22,586</td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±">220 ± 0.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="left"><i>cyc</i>-propyl</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±">182 ± 6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="left"><i>tert</i>-butyl</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±">230 ± 5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±">86 ± 3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="left">4-OMe-Ph</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±">79 ± 3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="left">4-Me-Ph</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±">87 ± 2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="left">4-CN-Ph</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±">53 ± 1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="left">4-F-Ph</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±">82 ± 3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="left">4-Cl-Ph</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±">97 ± 3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="left">2,6-(Cl)<sub>2</sub>-Ph</td><td class="colsep0 rowsep0" align="char" char="±">62,993 ± 2,501</td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="left">2,6-(F)<sub>2</sub>-Ph</td><td class="colsep0 rowsep0" align="char" char="±">12,776 ± 594</td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="left">2-OH-4-Cl-Ph</td><td class="colsep0 rowsep0" align="char" char="±">11,525 ± 40</td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="left">2-F, 4-CN-Ph</td><td class="colsep0 rowsep0" align="char" char="±">13,321 ± 2,309</td><td class="colsep0 rowsep0" align="char" char="±"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="left">2,6-(Me)<sub>2</sub>-Ph</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±">74 ± 4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="left">2,6-(MeO)<sub>2</sub>-Ph</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±">205 ± 2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="left">2-CN-Ph</td><td class="colsep0 rowsep0" align="char" char="±"> </td><td class="colsep0 rowsep0" align="char" char="±">17 ± 2</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">See the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for details on assay methods; the values were calculated from at least eight data points with at least two independent determinations.</p></div></div></div><div class="NLM_p">As expected, chloromethylketone <b>12</b> is a potent irreversible inhibitor of SARS CoV-1 3CL<sup>pro</sup>. Compounds <b>22</b>–<b>25</b> possess the most electron-deficient benzoate leaving groups and display kinetics consistent with irreversible inhibition. The remaining entries appear to be reversible inhibitors in the time scale of the assay, with <b>28</b> possessing the most potent IC<sub>50</sub>. A crystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XHN">6XHN</a>)<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> of <b>28</b> in complex with SARS CoV-1 3CL<sup>pro</sup> at 2.25 Å shows a covalent adduct, which demonstrates the bimodal activity of certain acyloxy inhibitors (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). In this structure, the electron density for the 2-cyanobenzoate moiety is absent and the 3CL<sup>pro</sup> active-site cysteine (Cys145) sulfur forms a covalent bond to the methylene carbon (1.8 Å C–S bond length). The ketone carbonyl is positioned within the oxyanion hole and as such engaging in hydrogen bonds with backbone NH groups of Gly143 and Cys145. A detailed analysis of an extended hydrogen bond network from catalytic His41 reveals that the side-chain imidazole serves as a hydrogen bond donor to the interior of the protease. Specifically, the imidazole hydrogen is directed toward a lone pair electron acceptor from a structural water. We can conclude that this water must be an acceptor to His41 as one of the water hydrogens is engaged with acceptor electrons on the side chain of Asp187, and the second hydrogen engages a terminating acceptor backbone carbonyl on Asp176 through a short network that includes the side chains of an internal, neutral His164 and Thr175.</div><figure id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/medium/jm0c01063_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0005.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Cocrystal structure of the covalent adduct of <b>28</b> bound to SARS CoV-1 3CL<sup>pro</sup> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XHN">6XHN</a>). The Connolly surface for the inhibitor binding pocket is shown in gray. The bonds are represented as dashed lines, with the bond length between heavy atoms depicted.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01063&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The 2,6-dichlorobenzoate derivative <b>22</b> displays the highest levels of potency in the SARS CoV-1 3CL<sup>pro</sup> and in antiviral cytopathic effect assays in Vero 76 cells (EC<sub>50</sub> = 0.29 ± 0.19 μM).<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> We note here that the assay in Vero 76 cells provided data establishing antiviral activity, but the translation of potency measured in Vero cells, which have high efflux potential, to the potency achievable in human lung cells was unknown (vide infra). As expected, <b>22</b> possesses low levels of reactivity toward endogenous nucleophiles such as glutathione (<i>t</i><sub>1/2</sub> > 60 min)<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> and exhibits high levels of stability in human plasma (<i>t</i><sub>1/2</sub> > 240 min).<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Although <b>22</b> displays a promising activity profile, it possesses very poor solubility in clinically relevant IV vehicle formulations, thus limiting its development as an IV clinical agent.</div><div class="NLM_p">To improve the solubility characteristics of inhibitors incorporating the lipophilic 2,6-dichlorobenzoate, a limited set of inhibitors containing smaller or more polar amine capping fragments were prepared (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Replacement of the 4-methoxy indole cap present in <b>22</b> with a benzimidazole provided <b>31</b>. This inhibitor displayed potent irreversible inhibition kinetics but did not improve solubility compared to <b>22</b>. Reduction in the size of the P<sub>2</sub> capping element as represented by <b>29</b> and <b>30</b> provided derivatives that possessed weak reversible SARS CoV-1 3CL<sup>pro</sup> inhibition.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. SARS CoV-1 3CL<sup>pro</sup> Inhibition Data for P<sub>3</sub>-Modified Acyloxymethylketone Inhibitors</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/medium/jm0c01063_0011.gif" alt="" id="fx3" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/medium/jm0c01063_0012.gif" alt="" id="fx4" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">See the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for details on assay methods; the values were calculated from at least eight data points with at least two independent determinations.</p></div></div><div></div></div><div class="NLM_p">Another design to replace the 4-methoxy indole was the (2<i>R</i>)-tetrahydrofuran-2-carboxylate, which molecular modeling suggested could make additional favorable protein interactions with a Gln189 side chain and led to the preparation of <b>3</b>. Initially, this derivative appeared to display reversible binding kinetics. However, further kinetic studies of compound <b>3</b> with the SARS CoV-1 3CL<sup>pro</sup> protein demonstrated irreversible time-dependent inactivation of this enzyme (<i>k</i><sub>obs</sub>/<i>I</i> = 5834 ± 151 M<sup>–1</sup> s<sup>–1</sup>). Additionally, <b>3</b> possessed potent antiviral activity in Vero cells (EC<sub>50</sub> = 2.4 μM) and solubility in vehicle formulations almost 20-fold higher than <b>22</b>. Evaluation of the pharmacokinetic profile of <b>3</b> in rat suggested that this inhibitor possessed properties potentially sufficient for an IV continuous infusion dosing paradigm.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a></div><div class="NLM_p last">Although irreversible inhibition can provide an extended pharmacodynamic effect when protein resynthesis rate is slow compared to drug clearance, such extended effects for 3CL<sup>pro</sup> inhibition were not necessarily expected. The 3CL<sup>pro</sup>-mediated proteolysis of newly expressed viral polyproteins is essential to virus replication; however, this activity only occurs during a single step in the virus life cycle that closely follows each cell infection. Viral particles themselves are not reliant on 3CL<sup>pro</sup> activity nor are the remaining coronavirus life cycle steps. Each event of cell infection initiates newly synthesized 3CL<sup>pro</sup>. Because detailed kinetics of this process are not understood, reversible and irreversible inhibitors were investigated equally.</div></div><div id="sec2_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i12" class="anchor-spacer"></div><h4 class="article-section__title" id="_i12"> Reversible Inhibitors of SARS CoV-1 3CL<sup>pro</sup></h4><div class="NLM_p">Peptidyl HMK inhibitors and their corresponding ethers have been designed as potent reversible cysteine protease inhibitors of cathepsin K<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> and cruzain.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> High-resolution crystal structures of two such inhibitors complexed with cruzain highlighted the presence of a strong hydrogen bond between the catalytic His and the α-hydroxyl moiety of the inhibitor but interestingly no covalent bond between the active-site Cys and the ketone pharmacophore. Utilizing the crystal structure of <b>28</b> in complex with SARS CoV-1 3CL<sup>pro</sup>, molecular modeling of an HMK-based design indicated the potential for establishing a covalent bond between the sulfur of the active-site cysteine (Cys145) and the HMK carbonyl as well as hydrogen bond interactions between catalytic His41 and the terminal hydroxy moiety. Additionally, the structure of <b>28</b> suggested the possibility to increase the lipophilic filling of the deep hydrophobic S<sub>2</sub> pocket as a strategy to increase binding affinity. To evaluate these structure-based observations, a series of reversible HMK inhibitors containing P<sub>2</sub> diversity were prepared as depicted in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. SARS 3CL<sup>pro</sup> Inhibition Data for P<sub>2</sub>-Modified HMK Inhibitors</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/medium/jm0c01063_0013.gif" alt="" id="fx5" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/medium/jm0c01063_0014.gif" alt="" id="fx6" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a&b</sup></sup><p class="last">See the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for details on assay methods; the values were calculated from at least eight data points with at least two independent determinations.</p></div><div class="footnote" id="t3fn2"><sup><sup>c</sup></sup><p class="last">This value is a <i>K</i><sub>i</sub> measurement.</p></div><div class="footnote" id="t3fn3"><sup><sup>d</sup></sup><p class="last">These values are from a single determination.</p></div></div><div></div></div><div class="NLM_p">As suggested by molecular modeling, the S<sub>2</sub> site appears to accommodate a variety of linear, branched, and cyclic alkyl moieties (entries <b>36</b> and <b>39</b>–<b>41</b>). Interestingly, the P<sub>2</sub> Phe derivative <b>42</b> displayed less potent inhibition of the SARS CoV-1 3CL<sup>pro</sup> than the corresponding saturated analogue <b>41</b>. Also noteworthy is the large attenuation in enzyme inhibition seen with P<sub>2</sub>-<i>N</i>-methyl-Leu inhibitor <b>38</b> (IC<sub>50</sub> = 83 nM) as compared to <b>4</b> (<i>K</i><sub>i</sub> = 4 nM). This loss of potency is consistent with the inhibitor ligand interactions observed with <b>2</b> and <b>4</b> that show the P<sub>2</sub> NH involved in a hydrogen bond with the side-chain amide of Gln189. Additionally, the methyl substitution in <b>38</b> would be expected to alter the conformation of the 4-methoxy indole cap and perturb the observed ligand–enzyme hydrogen bond network present in <b>2</b> and <b>4</b>.</div><div class="NLM_p">The results for <b>4</b> and the other examples in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> reveal a high antiviral EC<sub>50</sub>/enzyme IC<sub>50</sub> ratio, which may arise from poor cell permeability. Indeed, these HMK inhibitors exhibit very low permeability and high levels of efflux beyond the sensitivity of the Caco-2 <i>in vitro</i> assay. However, the impact of high efflux on antiviral potencies from Vero 76 cells (derived from monkey kidney) versus disease-relevant human lung epithelial cells is unknown. To better understand the observed high antiviral EC<sub>50</sub>/enzyme IC<sub>50</sub> ratio for <b>4</b>, we evaluated the role of efflux in the Vero 76 cell line by the <i>in vitro</i> experimental design discussed further below. Concurrently, a strategy to design molecules reducing efflux by active transporters, such as P-glycoprotein, was pursued to decrease high antiviral EC<sub>50</sub>/enzyme IC<sub>50</sub> ratios. An analysis of the physicochemical properties of <b>4</b> suggested that increasing logP, reducing polar surface area (PSA), and reducing the number of hydrogen bond donors/acceptors were design strategies with the potential to improve cellular permeability and reduce efflux.</div><div class="NLM_p">An examination of the crystal structure of <b>2</b> in complex with SARS CoV-1 3CL<sup>pro</sup> suggested a limited functional role for the 4-methoxy substituent contained in the P<sub>3</sub> indole cap. Additionally, the terminal α-hydroxyl moiety located at P<sub>1</sub>′ appeared to provide a potential opportunity to remove a hydrogen bond donor. Although it was anticipated that the α-hydroxyl moiety would form a hydrogen bond to the catalytic His41, the directionality of this interaction was uncertain. As noted earlier, His41 presents a hydrogen bond donor to the interior of the protease, leaving an acceptor to participate in the proton transfer cycling from neutral to cationic in the putative catalytic mechanism<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> of proteolysis. If His41 is protonated in the inhibitor-bound complex, a lone pair from the α-hydroxyl moiety would function as a hydrogen bond acceptor from the proton on the His residue. On the other hand, if His41 is deprotonated, the hydrogen of the α-hydroxyl moiety would act as a hydrogen bond donor to the nitrogen lone pair of the His residue. Consistent with the strategy to decrease PSA and hydrogen bond donors, a series of inhibitors were prepared that contained alterations to the P<sub>3</sub> cap and substitution of the α-hydroxyl group.</div><div class="NLM_p"><a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a> summarizes this effort where two of the optimal P<sub>2</sub> residues (Leu and β-<i>tert</i>-butyl-Ala) were conserved. Removal of the methoxy group from the indole generally led to slightly weaker potency in both the enzymatic and antiviral assays across pairs (<b>43</b> vs <b>36</b>, <b>44</b> vs <b>37</b>, <b>50</b> vs <b>48</b> and <b>49</b> vs <b>4</b>) and therefore no improvement in the antiviral/protease inhibition ratio. The impact of more significant changes to the P<sub>3</sub> cap is illustrated by tetrahydrofuranyl derivatives <b>46</b> and <b>47</b>. These derivatives lack the lipophilic aryl ring and NH hydrogen bond donor present in the indole P<sub>3</sub> cap and exhibit a pronounced reduction in enzymatic and antiviral potency. A noteworthy reduction in biochemical and antiviral potency was observed in comparing each hydroxymethylketone derivative with its corresponding ether counterpart. These results suggest that modest structural and physicochemical alterations of <b>4</b> fail to significantly decrease the high EC<sub>50</sub>/IC<sub>50</sub> ratios for this class of peptidomimetics. However, <i>in vitro</i> analysis of <b>4</b> revealed this derivative to possess high levels of metabolic stability in human liver microsomes (<i>t</i><sub>1/2</sub> = 107 min).<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> Further evaluation of this breakthrough derivative in several aqueous-based formulations suitable for IV administration indicated high levels of solubility.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a></div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. SARS CoV-1 3CL<sup>pro</sup> Inhibition Data for P<sub>3</sub>- and P<sub>2</sub>-Modified Hydroxymethyl and Alkoxymethylketone Inhibitors</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/medium/jm0c01063_0015.gif" alt="" id="fx7" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/medium/jm0c01063_0016.gif" alt="" id="fx8" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a&b</sup></sup><p class="last">See the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for details on assay methods; the values were calculated from at least eight data points with at least two independent determinations.</p></div><div class="footnote" id="t4fn2"><sup><sup>c</sup></sup><p class="last">This value is a <i>K</i><sub>i</sub> measurement.</p></div><div class="footnote" id="t4fn3"><sup><sup>d</sup></sup><p class="last">These values are from a single determination.</p></div></div><div></div></div><div class="NLM_p">Cocrystal structures of <b>4</b> bound to the 3CL<sup>pro</sup> of both SARS CoV-1 and CoV-2 were solved at 1.47 and 1.26 Å resolutions, respectively (PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XHL">6XHL</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XHM">6XHM</a>).<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> As expected, the ligand binding sites of the main protease from SARS CoV-1 and SARS CoV-2 are conserved in sequence and are nearly identical structurally. The schematic diagram in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a> is representative of the covalent adduct between <b>4</b> and the 3CL<sup>pro</sup> from both CoV-1 and CoV-2. The warhead HMK carbonyl carbon of <b>4</b> forms a covalent bond to the 3CL<sup>pro</sup> active-site cysteine (Cys145) sulfur generating a tetrahedral carbinol complex (1.8 Å C–S bond length). This carbinol hydroxyl forms hydrogen bonds with the backbone NH of Cys145 and with the amide NH of Gly143 via a bridging water molecule. Another key active-site interaction is the hydrogen bond between the primary alcohol moiety of <b>4</b> and the catalytic His41. Similar interactions between catalytic His residues and the OH moiety of HMK in other protease–inhibitor complexes have been reported.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a></div><figure id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/medium/jm0c01063_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0006.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Cocrystal structure of the covalent adduct of <b>4</b> bound to SARS CoV-2 3CL<sup>pro</sup> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XHM">6XHM</a>). The Connolly surface for the inhibitor binding pocket is shown in gray. The bonds are represented as dashed lines, with the bond length between heavy atoms depicted. The schematic rendering of the active site with dashed lines represented as hydrogen bonds with key residues and curved lines to show S<sub>1</sub> and S<sub>2</sub> binding pockets is depicted.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01063&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In S<sub>1</sub>, the lactam carbonyl of <b>4</b> forms a strong hydrogen bond with the side chain of His163, while the lactam NH is within the hydrogen-bonding distance but poorly aligned with the side-chain oxygen of Glu166 and the backbone oxygen of Phe140. These three residues partially define the bottom and edge of the S<sub>1</sub> pocket. Additionally, the backbone carbonyl and NH of Glu166 form β-sheet-like hydrogen-bonding interactions with the NH and C2-carbonyl of the indole fragment of <b>4</b>. The NH of the P<sub>2</sub> Leu of inhibitor <b>4</b> and the side chain of Gln189 form a hydrogen bond, while the carbonyl of P<sub>2</sub> Leu is exposed to the solvent. The inhibitor P<sub>1</sub> NH forms a strong hydrogen bond with the backbone carbonyl of His164. The inhibitor’s lipophilic leucine residue is bound to the SARS 3CL<sup>pro</sup> in the hydrophobic S<sub>2</sub> pocket formed by residues Asp187, Arg188, Gln189, Met49, and His41. The indole portion of <b>4</b> does not protrude into the S<sub>3</sub> pocket but rather rests across a partially collapsed S<sub>3</sub>, making extended van der Waals interactions across the 189–191 residue backbone atoms. The closed S<sub>3</sub> is the only significant conformational difference compared to a recently reported structure of SARS CoV-2 3CL<sup>pro</sup> inhibited by α-ketoamides.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> The energetics of this extended interaction, the associated protein conformation stabilities, or warhead difference may contribute to the 3-order-of-magnitude increased SARS CoV-2 3CL<sup>pro</sup> potency of HMK <b>4</b> (see below) compared to that of the referenced ketoamides.</div><div class="NLM_p">Hydroxymethylketone <b>4</b> was evaluated against a panel of other viral and human proteases (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). In general, <b>4</b> is highly selective for 3CL<sup>pro</sup> inhibition, displaying IC<sub>50</sub> values of >10 μM against many of the other proteases and possessing modest levels of inhibition of cathepsin B (IC<sub>50</sub> = 1.3 μM) and rhinovirus 3C<sup>pro</sup> (IC<sub>50</sub> = 1.7 μM). High levels of similarity between the catalytic sites of 3CL<sup>pro</sup> proteases suggest the potential of <b>4</b> as a pan-coronavirus inhibitor. Testing revealed that <b>4</b> demonstrates potent inhibition of the related hCoV 229E protease (IC<sub>50</sub> = 0.004 μM) but more importantly, given the current global pandemic, potently inhibits SARS CoV-2 3CL<sup>pro</sup> with a <i>K</i><sub>i</sub> of 0.27 nM.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Protease Inhibition by <b>4</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/medium/jm0c01063_0017.gif" alt="" id="fx9" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">protease</th><th class="colsep0 rowsep0" align="center">source</th><th class="colsep0 rowsep0" align="center" char=".">% inhibition @ 10 μM<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3CL<sup>pro</sup></td><td class="colsep0 rowsep0" align="left">SARS CoV-1</td><td class="colsep0 rowsep0" rowspan="2" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">0.004 ± 0.0003<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">3CL<sup>pro</sup></td><td class="colsep0 rowsep0" align="left">SARS CoV-2</td><td class="colsep0 rowsep0" align="left">0.00027 ± 0.0001<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">cathepsin B</td><td class="colsep0 rowsep0" align="left">human</td><td class="colsep0 rowsep0" align="char" char=".">93</td><td class="colsep0 rowsep0" align="left">1.3 ± 0.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">cathepsin D</td><td class="colsep0 rowsep0" align="left">human</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">leukocyte elastase</td><td class="colsep0 rowsep0" align="left">human</td><td class="colsep0 rowsep0" align="char" char=".">–31</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">chymotrypsin</td><td class="colsep0 rowsep0" align="left">human</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">thrombin</td><td class="colsep0 rowsep0" align="left">human</td><td class="colsep0 rowsep0" align="char" char=".">–28</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">pepsin</td><td class="colsep0 rowsep0" align="left">human</td><td class="colsep0 rowsep0" align="char" char=".">–2</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">caspase 2</td><td class="colsep0 rowsep0" align="left">human</td><td class="colsep0 rowsep0" align="char" char=".">–20</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HIV-1</td><td class="colsep0 rowsep0" align="left">HIV-1</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HCV</td><td class="colsep0 rowsep0" align="left">HCV</td><td class="colsep0 rowsep0" align="char" char=".">16</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HRV</td><td class="colsep0 rowsep0" align="left">HRV</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">1.7 ± 1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">hCoV 229E</td><td class="colsep0 rowsep0" align="left">human</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">0.004 ± 0.0004</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">See the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for details on assay methods; the values were calculated from at least eight data points with at least two independent determinations.</p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">For SARS CoV-1 and CoV-2 3CL<sup>pro</sup>, <i>K</i><sub>i</sub> values are reported here.</p></div></div></div><div class="NLM_p">The antiviral specificity of <b>4</b> was evaluated against a panel of viruses (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). No antiviral activity was observed when <b>4</b> was tested against two human rhinovirus strains (HRV-14 and HRV-16), human immunodeficiency virus-1 (HIV-1), or human cytomegalovirus (HCMV) in cell culture. This compound also does not inhibit the HCV replicon. As anticipated by the potent hCoV 229E protease activity, <b>4</b> is a potent inhibitor of the human coronavirus hCoV 229E in MRC-5 cells (human lung-derived) with an EC<sub>50</sub> of 0.090 μM.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Viral Inhibition by <b>4</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/medium/jm0c01063_0018.gif" alt="" id="fx10" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">virus</th><th class="colsep0 rowsep0" align="center">EC<sub>50</sub> (μM)<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">CC<sub>50</sub> (μM)<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SARS CoV-1</td><td class="colsep0 rowsep0" align="left">4.8 ± 2.2</td><td class="colsep0 rowsep0" align="left">452</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">hCoV 229E</td><td class="colsep0 rowsep0" align="left">0.09 ± 0.04</td><td class="colsep0 rowsep0" align="left">>40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HRV-14</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>1,260</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HRV-16</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>1,260</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HIV-1 RF</td><td class="colsep0 rowsep0" align="left">>32</td><td class="colsep0 rowsep0" align="left">>32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HCV replicon</td><td class="colsep0 rowsep0" align="left">>320</td><td class="colsep0 rowsep0" align="left">>320</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HCMV <b>RC256</b></td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>100</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">See the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for details on assay methods; the values were calculated from at least eight data points with at least two independent determinations.</p></div></div></div><div class="NLM_p last">The hCoV 229E antiviral potency is more than 50-fold greater than that for SARS CoV-1 in the Vero 76 line, even though <b>4</b> possesses nearly equivalent 3CL<sup>pro</sup> biochemical potency for the hCoV 229E and CoV-1 viruses. While undefined differences between hCoV 229E and SARS CoV-1 could account for these large differences in antiviral EC<sub>50</sub>/enzyme IC<sub>50</sub> ratios, different active transporter expression levels between MRC-5 and Vero 76 cell lines may also be a contributing factor. Consistent with the previously stated goal to characterize the impact of efflux on potency measured in the SARS CoV-1 antiviral assay, we co-dosed a known P-glycoprotein transport inhibitor at a fixed concentration (0.5 μM CP-100356)<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> in a full dose–response of inhibitor <b>4</b>. This resulted in a pronounced potency shift of SARS CoV-1 antiviral activity (EC<sub>50</sub> = 0.11 μM) that now more closely matched the antiviral potency seen for hCoV 229E. As a control, the same co-dosing of efflux inhibitor led to no such shift in the hCoV 229E antiviral potency of <b>4</b>, demonstrating that the human lung-derived MRC-5 cells show less P-glycoprotein-based transporter under similar assay conditions. These results suggest that the high antiviral EC<sub>50</sub>/enzyme IC<sub>50</sub> ratio observed in Vero 76 cells is an artifact of the high efflux potential of that assay cell line and may underestimate the antiviral potency in human lung cells, the relevant tissue for SARS and COVID-19. Assays to directly measure the antiviral potency of <b>4</b> against SARS CoV-2 in clinically relevant human lung cells are currently under investigation.</div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63075" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63075" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">This report details the efforts to identify irreversible and reversible peptidomimetic inhibitors of the SARS CoV-1 3CL<sup>pro</sup> as potential IV agents for the treatment of SARS and other coronavirus diseases. Two classes, acyloxymethylketones and hydroxymethylketones, were discovered as potent inhibitors of the SARS 3CL<sup>pro</sup>. The hydroxymethylketone derivative <b>4</b> demonstrated potent SARS CoV-1 inhibition in 3CL<sup>pro</sup> and antiviral assays. Additionally, <b>4</b> possesses acceptable solubility, stability in plasma, and low <i>in vitro</i> and <i>in vivo</i> clearances suitable for further development as a coronavirus treatment.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> Importantly, the potent inhibition of SARS CoV-2 3CL<sup>pro</sup> has prompted further preclinical investigations of <b>4</b> as a potential therapeutic treatment for patients with COVID-19.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64661" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64661" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> General Procedures</h3><div class="NLM_p">Unless otherwise noted, materials were obtained from commercial suppliers and used without further purification. The removal of solvent under reduced pressure or concentration refers to distillation using a Büchi rotary evaporator attached to a vacuum pump (3 mmHg). Products obtained as solids or high-boiling oils were dried under vacuum (1 mmHg). Silica gel chromatography was performed on either a CombiFlash (Teledyne ISCO), SP4, or Isolera (Biotage) purification system. All reactions were performed under a positive pressure of nitrogen or argon or with a drying tube at an ambient temperature (unless otherwise stated) in anhydrous solvents, unless otherwise indicated. Analytical thin-layer chromatography (TLC) was performed on glass-backed silica gel 60 F254 plates (Analtech, 0.25 mm), eluting with the appropriate solvent ratio (v/v). The reactions were assayed by high-performance liquid chromatography–mass spectrometry (LCMS) or TLC and terminated as judged by the consumption of the starting material. LCMS utilized wavelengths of 254 and 220 nm and either electrospray ionization (ESI) in positive mode or atmospheric-pressure chemical ionization (APCI) in positive mode. The TLC plates were visualized using UV light, <i>p</i>-anisaldehyde, phosphomolybdic acid, or iodine staining. Microwave-assisted reactions were run in a Biotage Initiator. <sup>1</sup>H NMR spectra were recorded on a Bruker XWIN-NMR (400 MHz) spectrometer. Proton chemical shifts are reported in parts per million downfield from tetramethylsilane (TMS). <sup>1</sup>H NMR data are reported as chemical shift (multiplicity, coupling constant, integration). Multiplicities are denoted as follows: s, singlet; d, doublet; t, triplet; q, quartet; dd, doublet of doublets; dt, doublet of triplets; and brs, broad singlet. For spectra obtained in CDCl<sub>3</sub>, DMSO-<i>d</i><sub>6</sub>, and CD<sub>3</sub>OD, the residual protons (7.27, 2.50, and 3.31 ppm, respectively) were used as the internal references. The purity of the final products is >95% as determined by HPLC and/or <sup>1</sup>H NMR analyses.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> <i>N</i>-(<i>tert</i>-Butoxycarbonyl)-3-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]-<span class="smallcaps smallerCapital">l</span>-alanine (<b>6</b>)</h4><div class="NLM_p last">A 3 L multineck flask equipped with an overhead stirrer and internal thermometer was charged with methyl <i>N</i>-(<i>tert</i>-butoxycarbonyl)-3-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]-<span class="smallcaps smallerCapital">l</span>-alaninate (compound <b>5</b>)<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> (205 g, 716 mmol) followed by methanol (1 L), and the solution was cooled to 0 °C using ice/NaCl bath. NaOH (115 g in 950 mL water, 2.9 mol) solution was precooled to 0 °C and then added to the flask <i>via</i> a pressure-equalizing dropping funnel at such a rate to maintain the internal temperature below 5 °C. The resulting solution was stirred at 0 °C for 1 h before neutralizing with conc. hydrochloric acid (keeping the internal temperature below 10 °C) and then removing the methanol <i>in vacuo</i>. The residue was diluted with ethyl acetate (400 mL), acidified to pH 3 with conc. hydrochloric acid, and then the mixture was transferred to a sep funnel, and the organics were removed. The aqueous was extracted with ethyl acetate (2 × 400 mL), and the combined organics were washed with brine, dried over MgSO<sub>4</sub>, filtered, and concentrated to yield the title compound as a white foam, 95%. <sup>1</sup>H NMR (400 MHz, MeOD) δ 3.98–4.28 (m, 1H), 3.25–3.41 (m, 2H), 2.44–2.57 (m, 1H), 2.29–2.41 (m, 1H), 2.03–2.14 (m, 1H), 1.73–1.90 (m, 2H), 1.44 (s, 9H); MS (APCI−) for C<sub>12</sub>H<sub>20</sub>N<sub>2</sub>O<sub>5</sub><i>m</i>/<i>z</i> 272.3 (M – H)<sup>−</sup>.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> <i>tert</i>-Butyl ((1<i>S</i>)-3-Diazo-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)carbamate (<b>7</b>)</h4><div class="NLM_p last">A solution of <i>N</i>-(<i>tert</i>-butoxycarbonyl)-3-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]-<span class="smallcaps smallerCapital">l</span>-alanine (2.72 g, 10.0 mmol) in THF (100 mL) was placed under an atmosphere of N<sub>2</sub> and cooled to −23 °C. The resulting clear colorless solution was successively treated with triethylamine (2.1 mL, 15.0 mmol) followed by isobutylchloroformate (1.6 mL, 12.0 mmol). The reaction mixture gradually became opaque with a fine white precipitate and after 1 h was filtered. The colorless filtrate was transferred to a nonground joint flask, cooled to 0 °C, and slowly treated with a solution of diazomethane (∼35 mL, ∼16.6 mmol) in diethyl ether. Note: The diazomethane was generated employing a Diazald kit according to the procedure described in the Aldrich Technical Bulletin AL-180. The resulting yellow clear solution was gradually warmed to room temperature (RT) over 16 h. At this time, N<sub>2</sub> was bubbled through the reaction to remove excess diazomethane followed by <i>in vacuo</i> concentration. The resulting residue was diluted with ethyl acetate (100 mL), washed once with sat. NaHCO<sub>3</sub> (50 mL), once with brine (50 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated to give a crude yellow foam. This material was purified by LC (150 g 230–400 SiO<sub>2</sub>, 3–4% methanol/chloroform) to afford 2.72 g (92%) of the title compound as a light yellow foam. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 7.63 (bs, 1H), 7.42 (d, <i>J</i> = 8 Hz, 1H), 6.06 (bs, 1H), 3.96 (m, 1H), 3.13 (m, 2H), 2.21 (m, 1H), 2.01 (m, 1H), 1.86 (m, 1H), 1.63–1.52 (m, 2H), 1.38 (s, 9H); MS (ESI+) for C<sub>13</sub>H<sub>20</sub>N<sub>4</sub>O<sub>4</sub><i>m</i>/<i>z</i> 319.0 (M + Na)<sup>+</sup>.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> <i>tert</i>-Butyl ((1<i>S</i>)-3-Chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)carbamate (<b>8</b>)</h4><div class="NLM_p last">A three-neck flame-dried flask equipped with a nitrogen inlet and internal thermometer was charged with methyl <i>N</i>-(<i>tert</i>-butoxycarbonyl)-3-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]-<span class="smallcaps smallerCapital">l</span>-alaninate (10 g, 35 mmol), THF (200 mL), and chloroiodomethane (10.2 mL, 140 mmol), and the solution was cooled to −77 °C. LDA (140 mL, 210 mmol, 1.5 M mono-THF complex in cyclohexane) was added <i>via</i> a pressure-equalizing dropping funnel at such a rate to keep the internal temperature below −70 °C. After complete addition, the reaction was stirred for an additional hour and quenched with a mixture of AcOH (33 mL) and THF (200 mL) with a rate of addition regulated to maintain the internal temperature below −65 °C. After complete addition, the dark suspension was stirred for 10 min and then warmed to an ambient temperature. The reaction was diluted with ethyl acetate (500 mL), and the organics were washed with water (250 mL), satd. NaHCO<sub>3</sub> (250 mL), and brine (250 mL), dried over MgSO<sub>4</sub>, filtered, and the solvents were removed <i>in vacuo</i> to yield the crude product as a dark oil, which was purified by flash chromatography eluting with ethyl acetate. The resulting solid was triturated with diethyl ether to afford the title compound as a pale yellow solid, 5.7 g, 54%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.88 (s, 1H), 7.75 (d, <i>J</i> = 7.6 Hz, 1H), 4.73–4.94 (m, 2H), 4.37 (m, 1H), 3.28–3.43 (m, 2H), 2.46 (m, 1H), 2.30–2.40 (m, 1H), 2.02–2.14 (m, 1H), 1.77–1.95 (m, 2H), 1.60 (s, 9H); MS (API-ES+) for C<sub>13</sub>H<sub>21</sub>N<sub>2</sub>O<sub>4</sub>Cl <i>m</i>/<i>z</i> 327.1 (M + Na)<sup>+</sup>; anal. calcd for C<sub>13</sub>H<sub>21</sub>ClN<sub>2</sub>O<sub>4</sub>: C, 51.23; H, 6.95; N, 9.19. Found: C, 51.03; H, 6.93; N, 9.03.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> <i>tert</i>-Butyl((1<i>S</i>)-3-(benzyloxy)-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)carbamate (<b>9</b>)</h4><div class="NLM_p">To a 3 L multinecked flask equipped with an overhead stirrer, nitrogen inlet, and internal thermometer was charged <i>N</i>-(<i>tert</i>-butoxycarbonyl)-3-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]-<span class="smallcaps smallerCapital">l</span>-alanine (190 g, 698 mmol) followed by dichloromethane (1100 mL) and the solution was cooled to 0 °C employing an ice/NaCl bath. <i>N</i>,<i>O</i>-Dimethylhydroxylamine hydrochloric acid salt (68 g, 698 mmol) was added followed by <i>N</i>-methylmorpholine (230 mL, 2.09 mol), HOBt.hydrate (106 g, 698 mmol), and EDCI (147 g, 768 mmol), and the mixture was stirred at 0 °C under nitrogen for 6 h before quenching with water (500 mL). The biphasic mixture was transferred to a sep funnel, and the organics were isolated, washed with 1 M hydrochloric acid (2 × 500 mL), water (400 mL), satd. NaHCO<sub>3</sub> (2 × 700 mL), and brine (300 mL) and then dried over MgSO<sub>4</sub>, filtered, and the solvents were evaporated in vacuo to yield <i>tert</i>-butyl ((1<i>S</i>)-2-[methoxy(methyl)amino]-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}ethyl)carbamate as a pale yellow solid, 173 g, 79%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 5.92 (s, 1H), 5.39 (d, <i>J</i> = 8.8 Hz, 1H), 4.66 (td, <i>J</i> = 9.6, 2.8 Hz, 1H), 3.77 (s, 3H), 3.33 (dd, <i>J</i> = 9.3, 4.0 Hz, 2H), 3.19 (s, 2H), 2.41–2.62 (m, 2H), 2.05–2.16 (m, 1H), 1.75–1.95 (m, 1H), 1.67 (m, 1H), 1.41 (s, 9H).</div><div class="NLM_p last">To a 100 mL multinecked flask equipped with a stirrer bar, nitrogen inlet, and internal thermometer were charged magnesium turnings (dried in oven at 100 °C overnight, 1.04 g, 43 mmol) and HgCl<sub>2</sub> (774 mg, 2.85 mmol), and the flask was purged with nitrogen for 10 min. THF (50 mL) was added, and the suspension was cooled to −45 °C before adding BOM-Cl (5.94 mL, 43 mmol), and the resulting suspension was stirred for 5 h, the temperature returning to 5 °C. The suspension was recooled to −50 °C, and <i>tert</i>-butyl ((1<i>S</i>)-2-[methoxy(methyl)amino]-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}ethyl)carbamate (1.5 g, 4.76 mmol) was added, and the thick suspension was stirred for 48 h under nitrogen and was allowed to come to an ambient temperature. The reaction was quenched by the careful addition of satd. NH<sub>4</sub>Cl solution (25 mL), and the mixture was stirred until effervescence ceased and then extracted with ethyl acetate (3 × 120 mL). The combined organics were dried over MgSO<sub>4</sub>, filtered, and the solvents were evaporated <i>in vacuo</i> to yield a crude orange gum, which was purified by flash chromatography, eluting with 1–3% methanol/dichloromethane to afford the title compound as a clear glass, 900 mg, 50%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.21–7.33 (m, 5H), 5.63–5.73 (m, 2H), 4.52 (m, 3H), 4.19 (q, <i>J</i> = 17.4 Hz, 2H), 3.19–3.25 (m, 2H), 2.29–2.42 (m, 2H), 1.68–1.93 (m, 3H), 1.35 (s, 9H); MS (APCI+) for C<sub>20</sub>H<sub>28</sub>N<sub>2</sub>O<sub>5</sub><i>m</i>/<i>z</i> 378.1 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> (3<i>S</i>)-3-[(2<i>S</i>)-2-Amino-4-chloro-3-oxobutyl]pyrrolidin-2-one, Hydrochloride Salt (<b>10</b>)</h4><div class="NLM_p last">A solution of <i>tert</i>-butyl ((1<i>S</i>)-3-diazo-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl) carbamate (592 mg, 2.0 mmol) in 1,4-dioxane (7 mL) was placed under an atmosphere of N<sub>2</sub> and cooled to 0 °C. This clear pale yellow solution was dropwise treated with a solution of 4 M hydrochloric acid in 1,4-dioxane (5 mL, 20 mmol) with copious gas evolution observed. Upon complete addition, the reaction was warmed to RT over 1 h with the formation of a white precipitate. The solid was collected by filtration, washed with diethyl ether, and dried to give 401 mg (83%) of the title compound as a white powder. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 8.76 (bs, 3H), 7.96 (s, 1H), 4.93 (d, <i>J</i> = 16 Hz, 1H), 4.80 (d, <i>J</i> = 16 Hz, 1H), 4.28 (m, 1H), 3.18 (m, 2H), 2.61 (m, 1H), 2.30 (m, 1H), 1.93 (m, 2H), 1.70 (m, 1H); MS (ESI+) for C<sub>8</sub>H<sub>13</sub>ClN<sub>2</sub>O<sub>2</sub><i>m</i>/<i>z</i> 205.0 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> <i>tert</i>-Butyl ((1<i>S</i>)-3-Bromo-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)carbamate (<b>11</b>)</h4><div class="NLM_p last">A solution of <i>tert</i>-butyl ((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)carbamate (1.26 g, 4.3 mmol) in dichloromethane (107 mL) at 0 °C under nitrogen was treated with 48% hydrobromic acid (0.48 mL, 4.3 mmol) with effervescence observed. The reaction was stirred at 0 °C for 1 h, washed once with water (30 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated to afford 1.23 g (83%) of the title compound as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.64 (s, 1H), 7.51 (d, <i>J</i> = 8 Hz, 1H), 4.46 (d, <i>J</i> = 16 Hz, 1H), 4.41 (d, <i>J</i> = 16 Hz, 1H), 4.19 (m, 1H), 3.13 (m, 2H), 2.26 (m, 1H), 2.13 (m, 1H), 1.87 (m, 1H), 1.63 (m, 2H), 1.38 (s, 9H); MS (ESI+) for C<sub>13</sub>H<sub>21</sub>BrN<sub>2</sub>O<sub>4</sub><i>m</i>/<i>z</i> 371.0 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-Chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3-methylbutyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide (<b>12</b>)</h4><div class="NLM_p">A solution of <b>29</b>, <i>N</i><sup>2</sup>-(<i>tert</i>-butoxycarbonyl)-<i>N</i><sup>1</sup>-((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-<span class="smallcaps smallerCapital">l</span>-leucinamide (244 mg, 0.59 mmol) in 1,4-dioxane (1.0 mL) was placed under an atmosphere of N<sub>2</sub>. This clear colorless solution was treated with a solution of 4 M hydrochloric acid in 1,4-dioxane with no observable change. The reaction gradually became opaque with the formation of a gummy precipitate. After 2 h, the volatiles were removed <i>in vacuo</i>, diluted with 1:1 ethanol/1,4-dioxane, concentrated, and high-vacuum-dried to give <i>N</i><sup>1</sup>-((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-<span class="smallcaps smallerCapital">l</span>-leucinamide hydrochloride as an off-white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 9.18 (d, <i>J</i> = 8 Hz, 1H), 8.40 (bs, 3H), 7.69 (s, 1H), 4.68 (s, 2H), 4.49 (m, 1H), 3.81 (m, 1H), 3.15 (m, 2H), 2.37 (m, 1H), 2.17 (m, 1H), 1.92 (m, 1H), 1.67 (m, 3H), 1.55 (m, 2H), 0.92 (d, <i>J</i> = 4 Hz, 3H), 0.88 (d, <i>J</i> = 4 Hz, 3H); MS (ESI+) for C<sub>14</sub>H<sub>24</sub>ClN<sub>3</sub>O<sub>3</sub><i>m</i>/<i>z</i> 318.1 (M + H)<sup>+</sup>.</div><div class="NLM_p last">A solution of the crude hydrochloride salt and 4-methoxy-1<i>H</i>-indole-2-carboxylic acid (123 mg, 0.64 mmol) in DMF (2.5 mL) was placed under an atmosphere of N<sub>2</sub> and cooled to 0 °C. This pale yellow solution was successively treated with HATU (245 mg, 0.64 mmol) and <i>N</i>-methylmorpholine (0.14 mL, 1.29 mmol) turning a brighter color. After 30 min, the reaction was quenched with 1:1 ice/sat NaHCO<sub>3</sub> (25 mL) and extracted three times with ethyl acetate (20 mL). The combined organics were washed once with brine (30 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated to give a yellow syrup. This material was purified by LC (30 g 230–400 SiO<sub>2</sub>, 4% methanol/chloroform) to afford 167 mg (58%) of the title compound as an off-white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 11.59 (s, 1H), 8.62 (d, <i>J</i> = 8 Hz, 1H), 8.44 (d, <i>J</i> = 4 Hz, 1H), 7.65 (s, 1H), 7.38 (s, 1H), 7.10 (t, <i>J</i> = 8 Hz, 1H), 7.02 (d, <i>J</i> = 8 Hz, 1H), 6.51 (d, <i>J</i> = 8 Hz, 1H), 4.60 (d, <i>J</i> = 16 Hz, 1H), 4.58 (d, <i>J</i> = 16 Hz, 1H), 4.46 (m, 2H), 3.89 (s, 3H), 3.11 (m, 2H), 2.29 (m, 1H), 2.11 (m, 1H), 1.99 (m, 1H), 1.76–1.54 (m, 5H), 0.95 (d, <i>J</i> = 8 Hz, 3H), 0.9 (d, <i>J</i> = 8 Hz, 3H); MS (ESI+) for C<sub>24</sub>H<sub>31</sub>ClN<sub>4</sub>O<sub>5</sub><i>m</i>/<i>z</i> 491.1 (M + H)<sup>+</sup>; HRMS (ESI+) calcd for C<sub>24</sub>H<sub>31</sub>ClN<sub>4</sub>O<sub>5</sub>+H1 491.2056, found 491.2058.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> (3<i>S</i>)-3-({<i>N</i>-[(4-Methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl Acetate (<b>13</b>)</h4><div class="NLM_p last">An oven-dried 40 mL scintillation vial with a spinbar was charged with acetic acid (27 mg, 0.46 mmol) followed by a solution of <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3-methylbutyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide (172 mg, 0.35 mmol) in DMF (3.5 mL) and purged with N<sub>2</sub>. This pale yellow solution was then treated with CsF (122 mg, 0.81 mmol), sealed with a Teflon-lined screwcap, and heated at 65 °C on a reaction block with vigorous stirring. After 3 h, the reaction was cooled to RT, diluted with water (30 mL), and extracted with dichloromethane (4 × 7 mL). The combined organic layers were washed with water (2 × 20 mL), brine (20 mL), and concentrated <i>in vacuo</i>. This material was purified by Biotage MPLC (25 M, 2.5–4.5% methanol/dichloromethane) to afford 78 mg (45%) of the title compound as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.57 (s, 1H), 8.57 (d, <i>J</i> = 7.8 Hz, 1H), 8.43 (d, <i>J</i> = 7.6 Hz, 1H), 7.64 (s, 1H), 7.36 (d, <i>J</i> = 1.8 Hz, 1H), 7.08 (t, <i>J</i> = 8.0 Hz, 1H), 6.99 (d, <i>J</i> = 8.1 Hz, 1H), 6.49 (d, <i>J</i> = 7.6 Hz, 1H), 4.83 (d, <i>J</i> = 3.0 Hz, 1H), 4.76–4.95 (m, 1H), 4.35–4.50 (m, 2H), 3.87 (s, 3H), 3.03–3.17 (m, 2H), 2.22–2.35 (m, 1H), 2.09–2.22 (m, 1H), 2.07 (s, 3H), 1.90–2.04 (m, 1H), 1.65–1.77 (m, 2H), 1.48–1.65 (m, 3H), 0.94 (d, <i>J</i> = 6.3 Hz, 3H), 0.89 (d, <i>J</i> = 6.3 Hz, 3H); MS (ESI+) for C<sub>26</sub>H<sub>34</sub>N<sub>4</sub>O<sub>7</sub><i>m</i>/<i>z</i> 515.2 (M + H)<sup>+</sup>. Anal. calcd for C<sub>26</sub>H<sub>34</sub>N<sub>4</sub>O<sub>7</sub>·0.65 H<sub>2</sub>O: C, 59.34; H, 6.76; N, 10.65. Found: C, 59.41; H, 6.63; N, 10.68.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> (3<i>S</i>)-3-({<i>N</i>-[(4-Methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl Cyclopropanecarboxylate (<b>14</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of (3<i>S</i>)-3-({<i>N</i>-[(4-methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl acetate but substituting cyclopropanecarboxylic acid and making noncritical variations provided a crude product. This material was purified by Biotage MPLC (25 M, 2.5–4.5% methanol/dichloromethane) to afford 82 mg (43%) of the title compound. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.57 (d, <i>J</i> = 2.0 Hz, 1H), 8.56 (d, <i>J</i> = 7.8 Hz, 1H), 8.43 (d, <i>J</i> = 7.6 Hz, 1H), 7.63 (s, 1H), 7.36 (d, <i>J</i> = 1.5 Hz, 1H), 7.08 (t, <i>J</i> = 8.0 Hz, 1H), 6.97–7.02 (m, 1H), 6.49 (d, <i>J</i> = 7.6 Hz, 1H), 4.85 (d, 1H), 4.78–4.96 (m, 1H), 4.33–4.51 (m, 2H), 3.87 (s, 3H), 3.02–3.16 (m, 2H), 2.22–2.35 (m, 1H), 2.01–2.11 (m, 1H), 1.89–2.00 (m, 1H), 1.65–1.77 (m, 3H), 1.46–1.65 (m, 3H), 0.81–0.98 (m, 10 H); MS (ESI+) for C<sub>28</sub>H<sub>36</sub>N<sub>4</sub>O<sub>7</sub><i>m</i>/<i>z</i> 541.2 (M + H)<sup>+</sup>. Anal. calcd for C<sub>28</sub>H<sub>36</sub>N<sub>4</sub>O<sub>7</sub>·0.59 H<sub>2</sub>O: C, 61.01; H, 6.80; N, 10.16. Found: C, 61.04; H, 6.72; N, 10.00.</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> (3<i>S</i>)-3-({<i>N</i>-[(4-Methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl Pivalate (<b>15</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of (3<i>S</i>)-3-({<i>N</i>-[(4-methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl acetate but substituting pivalic acid and making noncritical variations provided a crude product. This material was purified by Biotage MPLC (25 M, 2.5–4.5% methanol/dichloromethane) to afford 152 mg (78%) of the title compound. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.58 (s, 1H), 8.56 (d, <i>J</i> = 7.8 Hz, 1H), 8.44 (d, <i>J</i> = 7.8 Hz, 1H), 7.64 (s, 1H), 7.36 (s, 1H), 7.08 (t, <i>J</i> = 8.0 Hz, 1H), 6.99 (d, 1H), 6.49 (d, <i>J</i> = 7.6 Hz, 1H), 4.84 (s, 1H), 4.77–4.94 (m, 1H), 4.34–4.51 (m, 2H), 3.87 (s, 3H), 3.02–3.16 (m, 2H), 1.91–2.36 (m, 3H), 1.48–1.78 (m, 5H), 1.16 (s, 9H), 0.94 (d, <i>J</i> = 6.3 Hz, 3H), 0.89 (d, <i>J</i> = 6.3 Hz, 3H); MS (ESI+) for C<sub>29</sub>H<sub>40</sub>N<sub>4</sub>O<sub>7</sub><i>m</i>/<i>z</i> 557.2 (M + H)<sup>+</sup>. Anal. calcd for C<sub>29</sub>H<sub>40</sub>N<sub>4</sub>O<sub>7</sub>·0.76 H<sub>2</sub>O: C, 61.07; H, 7.34; N, 9.82. Found: C, 61.14; H, 7.63; N, 9.59.</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> (3<i>S</i>)-3-({<i>N</i>-[(4-Methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl Benzoate (<b>16</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of (3<i>S</i>)-3-({<i>N</i>-[(4-methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl acetate but substituting benzoic acid and making noncritical variations provided a crude product. This material was purified by Biotage MPLC (25 M, 2.5–4.5% methanol/dichloromethane) to afford 183 mg (91%) of the title compound. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.59 (s, 1H), 8.64 (d, <i>J</i> = 8.1 Hz, 1H), 8.46 (d, <i>J</i> = 7.6 Hz, 1H), 7.98 (d, <i>J</i> = 8.1 Hz, 2H), 7.91–7.95 (m, 1H), 7.66–7.71 (m, 1H), 7.66 (s, 1H), 7.52–7.57 (m, 1H), 7.37 (d, <i>J</i> = 1.8 Hz, 1H), 7.08 (t, <i>J</i> = 8.0 Hz, 1H), 6.96–7.02 (m, 1H), 6.49 (d, <i>J</i> = 7.6 Hz, 1H), 5.13 (s, 1H), 5.06–5.24 (m, 1H), 4.44–4.53 (m, 2H), 3.87 (s, 3H), 3.04–3.15 (m, 2H), 2.34 (m, 1H), 2.07–2.27 (m, 1H), 1.98–2.07 (m, 1H), 1.52–1.79 (m, 5H), 0.94 (d, <i>J</i> = 6.3 Hz, 3H), 0.89 (d, <i>J</i> = 6.3 Hz, 3H); MS (ESI+) for C<sub>31</sub>H<sub>36</sub>N<sub>4</sub>O<sub>7</sub><i>m</i>/<i>z</i> 577.2 (M + H)<sup>+</sup>. Anal. calcd for C<sub>31</sub>H<sub>36</sub>N<sub>4</sub>O<sub>7</sub>·1.04 H<sub>2</sub>O: C, 62.54; H, 6.45; N, 9.41. Found: C, 62.49; H, 6.54; N, 9.31.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> (3<i>S</i>)-3-({<i>N</i>-[(4-Methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl 4-Methoxybenzoate (<b>17</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of (3<i>S</i>)-3-({<i>N</i>-[(4-methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl acetate but substituting 4-methoxybenzoic acid and making noncritical variations provided a crude product. This material was purified by Biotage MPLC (25 M, 2.5–4.5% methanol/dichloromethane) to afford 145 mg (68%) of the title compound. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.59 (d, <i>J</i> = 1.8 Hz, 1H), 8.62 (d, <i>J</i> = 7.8 Hz, 1H), 8.46 (d, <i>J</i> = 7.6 Hz, 1H), 7.93 (d, <i>J</i> = 9.1 Hz, 2H), 7.65 (s, 1H), 7.37 (d, <i>J</i> = 2.0 Hz, 1H), 7.04–7.11 (m, 2H), 7.00 (dd, <i>J</i> = 8.6, 3.0 Hz, 2H), 6.50 (d, <i>J</i> = 7.6 Hz, 1H), 5.07 (s, 1H), 5.01–5.18 (m, 1H), 4.44–4.51 (m, 2H), 3.87 (s, 3H), 3.83 (s, 3H), 3.04–3.17 (m, 2H), 2.28–2.38 (m, 1H), 2.07–2.25 (m, 1H), 1.97–2.06 (m, 1H), 1.50–1.79 (m, 5H), 0.94 (d, <i>J</i> = 6.3 Hz, 3H), 0.89 (d, <i>J</i> = 6.3 Hz, 3H); MS (ESI+) for C<sub>32</sub>H<sub>38</sub>N<sub>4</sub>O<sub>8</sub><i>m</i>/<i>z</i> 607.2 (M + H)<sup>+</sup>. Anal. calcd for C<sub>32</sub>H<sub>38</sub>N<sub>4</sub>O<sub>8</sub>·0.80 H<sub>2</sub>O·0.05 DCM: C, 61.56; H, 6.40; N, 8.96. Found: C, 61.84; H, 6.36; N, 8.58.</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> (3<i>S</i>)-3-({<i>N</i>-[(4-Methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl 4-Methylbenzoate (<b>18</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of (3<i>S</i>)-3-({<i>N</i>-[(4-methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl acetate but substituting 4-methylbenzoic acid and making noncritical variations provided a crude product. This material was purified by Biotage MPLC (25 M, 2.5–4.5% methanol/dichloromethane) to afford 92 mg (44%) of the title compound. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.59 (d, <i>J</i> = 1.8 Hz, 1H), 8.63 (d, <i>J</i> = 8.1 Hz, 1H), 8.46 (d, <i>J</i> = 7.6 Hz, 1H), 7.87 (d, <i>J</i> = 8.1 Hz, 2H), 7.65 (s, 1H), 7.37 (d, <i>J</i> = 1.8 Hz, 1H), 7.34 (d, <i>J</i> = 8.3 Hz, 2H), 7.08 (t, <i>J</i> = 8.0 Hz, 1H), 6.97–7.01 (m, 1H), 6.50 (d, <i>J</i> = 7.8 Hz, 1H), 5.09 (s, 1H), 5.03–5.20 (m, 1H), 4.43–4.52 (m, 2H), 3.87 (s, 3H), 3.04–3.17 (m, 2H), 2.38 (s, 3H), 2.28–2.34 (m, 1H), 2.06–2.14 (m, 1H), 1.96–2.06 (m, 1H), 1.51–1.78 (m, 5H), 0.94 (d, <i>J</i> = 6.3 Hz, 3H), 0.89 (d, <i>J</i> = 6.3 Hz, 3H); MS (ESI+) for C<sub>32</sub>H<sub>38</sub>N<sub>4</sub>O<sub>7</sub><i>m</i>/<i>z</i> 591.2 (M + H)<sup>+</sup>. Anal. calcd for C<sub>32</sub>H<sub>38</sub>N<sub>4</sub>O<sub>7</sub>·0.10 H<sub>2</sub>O·0.11 DCM: C, 64.08; H, 6.43; N, 9.31. Found: C, 64.36; H, 6.41; N, 8.92.</div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> (3<i>S</i>)-3-({<i>N</i>-[(4-Methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl 4-Cyanobenzoate (<b>19</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of (3<i>S</i>)-3-({<i>N</i>-[(4-methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl acetate but substituting 4-cyanobenzoic acid and making noncritical variations provided a crude product. This material was purified by Biotage MPLC (25 M, 2.5–4.5% methanol/dichloromethane) to afford 159 mg (75%) of the title compound. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.58 (d, <i>J</i> = 2.0 Hz, 1H), 8.64 (d, <i>J</i> = 8.1 Hz, 1H), 8.46 (d, <i>J</i> = 7.6 Hz, 1H), 8.09–8.14 (m, 2H), 7.99–8.05 (m, 2H), 7.66 (s, 1H), 7.37 (d, <i>J</i> = 1.5 Hz, 1H), 7.08 (t, <i>J</i> = 8.0 Hz, 1H), 6.99 (d, <i>J</i> = 8.3 Hz, 1H), 6.50 (d, <i>J</i> = 7.6 Hz, 1H), 5.17 (d, <i>J</i> = 2.8 Hz, 1H), 5.11–5.27 (m, 1H), 4.44–4.53 (m, 2H), 3.87 (s, 3H), 3.04–3.17 (m, 2H), 2.28–2.38 (m, 1H), 2.07–2.26 (m, 1H), 1.97–2.06 (m, 1H), 1.51–1.78 (m, 5H), 0.94 (d, <i>J</i> = 6.3 Hz, 3H), 0.89 (d, <i>J</i> = 6.3 Hz, 3H); MS (ESI+) for C<sub>32</sub>H<sub>35</sub>N<sub>5</sub>O<sub>7</sub><i>m</i>/<i>z</i> 602.2 (M + H)<sup>+</sup>. Anal. calcd for C<sub>32</sub>H<sub>35</sub>N<sub>5</sub>O<sub>7</sub>·0.90 H<sub>2</sub>O: C, 62.21; H, 6.00; N, 11.33. Found: C, 62.23; H, 6.06; N, 11.58.</div></div><div id="sec4_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> (3<i>S</i>)-3-({<i>N</i>-[(4-Methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl 4-Fluorobenzoate (<b>20</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of (3<i>S</i>)-3-({<i>N</i>-[(4-methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl acetate but substituting 4-fluorobenzoic acid and making noncritical variations provided a crude product. This material was purified by Biotage MPLC (25 M, 2.5–4.5% methanol/dichloromethane) to afford 136 mg (65%) of the title compound. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.58 (d, <i>J</i> = 2.0 Hz, 1H), 8.63 (d, <i>J</i> = 7.8 Hz, 1H), 8.46 (d, <i>J</i> = 7.6 Hz, 1H), 8.04 (dd, <i>J</i> = 8.6, 5.6 Hz, 2H), 7.65 (s, 1H), 7.35–7.41 (m, 3H), 7.08 (t, <i>J</i> = 8.0 Hz, 1H), 6.99 (d, <i>J</i> = 8.1 Hz, 1H), 6.50 (d, <i>J</i> = 7.6 Hz, 1H), 5.12 (d, <i>J</i> = 1.3 Hz, 1H), 5.06–5.23 (m, 1H), 4.44–4.52 (m, 2H), 3.87 (s, 3H), 3.04–3.17 (m, 2H), 2.28–2.38 (m, 1H), 2.07–2.26 (m, 1H), 1.98–2.06 (m, 1H), 1.52–1.79 (m, 5H), 0.94 (d, <i>J</i> = 6.3 Hz, 3H), 0.89 (d, <i>J</i> = 6.3 Hz, 3H); MS (ESI+) for C<sub>31</sub>H<sub>35</sub>FN<sub>4</sub>O<sub>7</sub><i>m</i>/<i>z</i> 595.2 (M + H)<sup>+</sup>. Anal. calcd for C<sub>31</sub>H<sub>35</sub>FN<sub>4</sub>O<sub>7</sub>·0.59 H<sub>2</sub>O·0.06 DCM: C, 61.28; H, 5.98; N, 9.20. Found: C, 61.27; H, 5.98; N, 9.11.</div></div><div id="sec4_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> (3<i>S</i>)-3-({<i>N</i>-[(4-Methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl 4-Chlorobenzoate (<b>21</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of (3<i>S</i>)-3-({<i>N</i>-[(4-methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl acetate but substituting 4-chlorobenzoic acid and making noncritical variations provided a crude product. This material was purified by Biotage MPLC (25 M, 2.5–4.5% methanol/dichloromethane) to afford 172 mg (80%) of the title compound. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.58 (d, <i>J</i> = 1.8 Hz, 1H), 8.63 (d, <i>J</i> = 7.8 Hz, 1H), 8.46 (d, <i>J</i> = 7.6 Hz, 1H), 7.98 (d, <i>J</i> = 8.6 Hz, 2H), 7.65 (s, 1H), 7.62 (d, <i>J</i> = 8.6 Hz, 2H), 7.37 (d, <i>J</i> = 1.8 Hz, 1H), 7.08 (t, <i>J</i> = 8.0 Hz, 1H), 6.99 (d, <i>J</i> = 8.3 Hz, 1H), 6.50 (d, <i>J</i> = 7.6 Hz, 1H), 5.13 (d, <i>J</i> = 1.5 Hz, 1H), 5.07–5.23 (m, 1H), 4.44–4.52 (m, 2H), 3.87 (s, 3H), 3.04–3.14 (m, 2H), 2.28–2.38 (m, 1H), 2.06–2.26 (m, 1H), 1.97–2.06 (m, 1H), 1.51–1.78 (m, 5H), 0.94 (d, <i>J</i> = 6.3 Hz, 3H), 0.89 (d, <i>J</i> = 6.3 Hz, 3H); MS (ESI+) for C<sub>31</sub>H<sub>35</sub>ClN<sub>4</sub>O<sub>7</sub><i>m</i>/<i>z</i> 611.1 (M + H)<sup>+</sup>. Anal. calcd for C<sub>31</sub>H<sub>35</sub>ClN<sub>4</sub>O<sub>7</sub>·0.72 H<sub>2</sub>O·0.05 DCM: C, 59.36; H, 5.86; N, 8.92. Found: C, 59.31; H, 5.99; N, 9.23.</div></div><div id="sec4_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> (3<i>S</i>)-3-({<i>N</i>-[(4-Methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl 2,6-Dichlorobenzoate (<b>22</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of (3<i>S</i>)-3-({<i>N</i>-[(4-methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl acetate but substituting 2,6-dichlorobenzoic acid and making noncritical variations provided a crude product. This material was purified by LC (20 g 230–400 SiO<sub>2</sub>, 3% methanol/chloroform) to afford 114 mg (65%) of the title compound as a white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 11.57 (d, <i>J</i> = 2 Hz, 1H), 8.62 (d, <i>J</i> = 8 Hz, 1H), 8.46 (d, <i>J</i> = 4 Hz, 1H), 7.65–7.53 (m, 4H), 7.36 (s, 1H), 7.08 (t, <i>J</i> = 8 Hz, 1H), 6.99 (d, <i>J</i> = 8 Hz, 1H), 6.49 (d, <i>J</i> = 8 Hz, 1H), 5.19 (d, <i>J</i> = 20 Hz, 1H), 5.15 (d, <i>J</i> = 20 Hz, 1H), 4.51 (m, 2H), 3.87 (s, 3H), 3.11 (m, 2H), 2.30 (m, 1H), 2.06 (m, 2H), 1.76–1.51 (m, 5H), 0.94 (d, <i>J</i> = 8 Hz, 3H), 0.89 (d, <i>J</i> = 8 Hz, 3H); MS (ESI+) for C<sub>31</sub>H<sub>34</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>7</sub><i>m</i>/<i>z</i> 645.1 (M + H)<sup>+</sup>; Anal. Calcd for C<sub>31</sub>H<sub>34</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>7</sub>·0.2 H<sub>2</sub>O: C, 57.36; H, 5.34; N, 8.63. Found: C, 57.23; H, 5.55; N, 8.46. HRMS (ESI+) calcd for C<sub>31</sub>H<sub>34</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>7</sub>+H1 645.1877, found 645.1871.</div></div><div id="sec4_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> (3<i>S</i>)-3-({<i>N</i>-[(4-Methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl 2,6-Difluorobenzoate (<b>23</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of (3<i>S</i>)-3-({<i>N</i>-[(4-methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl acetate but substituting 2,6-difluorobenzoic acid and making noncritical variations provided a crude brown oily solid. This material was triturated with chloroform/ethyl acetate to afford 180 mg (75%) of the title compound as a white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 11.65 (d, <i>J</i> = 2 Hz, 1H), 8.68 (d, <i>J</i> = 8 Hz, 1H), 8.54 (d, <i>J</i> = 8 Hz, 1H), 7.73–7.65 (m, 2H), 7.36 (s, 1H), 7.27 (t, <i>J</i> = 8 Hz, 1H), 7.08 (t, <i>J</i> = 8 Hz, 1H), 6.99 (d, <i>J</i> = 8 Hz, 1H), 6.49 (d, <i>J</i> = 8 Hz, 1H), 5.19 (d, <i>J</i> = 16 Hz, 1H), 5.14 (d, <i>J</i> = 16 Hz, 1H), 4.48 (m, 2H), 3.87 (s, 3H), 3.11 (m, 2H), 2.32 (m, 1H), 2.11–1.91 (m, 2H), 1.81–1.56 (m, 5H), 0.94 (d, <i>J</i> = 4 Hz, 3H), 0.89 (d, <i>J</i> = 4 Hz, 3H); MS (ESI+) for C<sub>31</sub>H<sub>34</sub>F<sub>2</sub>N<sub>4</sub>O<sub>7</sub><i>m</i>/<i>z</i> 613.2 (M + H)<sup>+</sup>; HRMS (ESI+) calcd for C<sub>31</sub>H<sub>34</sub>F<sub>2</sub>N<sub>4</sub>O<sub>7</sub>+H1 613.2469, found 613.2476.</div></div><div id="sec4_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> (3<i>S</i>)-3-({<i>N</i>-[(4-Methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl 4-Chloro-2-hydroxybenzoate (<b>24</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of (3<i>S</i>)-3-({<i>N</i>-[(4-methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl acetate but substituting 4-chlorosalicylic acid and making noncritical variations provided a crude brown syrup. This material was purified by Biotage MPLC (25 M, 2–3% methanol/dichloromethane) to afford 64 mg (25%) of the title compound as a tan solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 11.58 (s, 1H), 10.50 (bs, 1H), 8.65 (d, <i>J</i> = 8 Hz, 1H), 8.46 (d, <i>J</i> = 8 Hz, 1H), 7.81 (d, <i>J</i> = 8 Hz, 1H), 7.63 (m, 1H), 7.37 (s, 1H), 7.08 (m, 2H), 7.00 (m, 2H), 6.50 (d, <i>J</i> = 8 Hz, 1H), 5.16 (d, <i>J</i> = 16 Hz, 2H), 5.11 (d, <i>J</i> = 16 Hz, 2H), 4.49 (m, 2H), 3.87 (s, 3H), 3.13 (m, 2H), 2.32 (m, 1H), 2.11–1.98 (m, 2H), 1.76–1.55 (m, 5H), 0.94 (d, <i>J</i> = 6 Hz, 3H), 0.89 (d, <i>J</i> = 6 Hz, 3H); MS (ESI-) for C<sub>31</sub>H<sub>35</sub>ClN<sub>4</sub>O<sub>8</sub><i>m</i>/<i>z</i> 625.1 (M – H)<sup>−</sup>; anal. calcd for C<sub>31</sub>H<sub>35</sub>ClN<sub>4</sub>O<sub>8</sub>·0.1 H<sub>2</sub>O·0.14 CHCl<sub>3</sub>: C, 57.93; H, 5.52; N, 8.68. Found: C, 58.04; H, 5.78; N, 8.74; HRMS (ESI+) calcd for C<sub>31</sub>H<sub>35</sub>ClN<sub>4</sub>O<sub>8</sub>+H1 627.2216, found 627.2219.</div></div><div id="sec4_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> (3<i>S</i>)-3-({<i>N</i>-[(4-Methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl 4-Cyano-2-fluorobenzoate (<b>25</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of (3<i>S</i>)-3-({<i>N</i>-[(4-methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl acetate but substituting 4-cyano-2-fluorobenzoic acid and making noncritical variations provided a crude oily brown solid. This material was purified by Biotage MPLC (25 M, 2.5–3.5% methanol/dichloromethane) to afford 119 mg (48%) of the title compound as an off-white powder. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 11.58 (d, <i>J</i> = 2 Hz, 1H), 8.63 (d, <i>J</i> = 8 Hz, 1H), 8.46 (d, <i>J</i> = 8 Hz, 1H), 8.06 (m, 2H), 7.83 (d, <i>J</i> = 8 Hz, 1H), 7.65 (s, 1H), 7.36 (d, <i>J</i> = 2 Hz, 1H), 7.08 (t, <i>J</i> = 8 Hz, 1H), 6.99 (d, <i>J</i> = 8 Hz, 1H), 6.50 (d, <i>J</i> = 8 Hz, 1H), 5.16 (d, <i>J</i> = 16 Hz, 1H), 5.11 (d, <i>J</i> = 16 Hz, 1H), 4.49 (m, 2H), 3.87 (s, 3H), 3.11 (m, 2H), 2.32 (m, 1H), 2.10–1.98 (m, 2H), 1.77–1.53 (m, 5H), 0.94 (d, <i>J</i> = 6 Hz, 3H), 0.89 (d, <i>J</i> = 6 Hz, 3H); MS (ESI+) for C<sub>32</sub>H<sub>34</sub>FN<sub>5</sub>O<sub>7</sub><i>m</i>/<i>z</i> 620.1 (M + H)<sup>+</sup>; anal. calcd for C<sub>32</sub>H<sub>34</sub>FN<sub>5</sub>O<sub>7</sub>·0.3 H<sub>2</sub>O: C, 61.49; H, 5.58; N, 11.20. Found: C, 61.47; H, 5.61; N, 10.98; HRMS (ESI+) calcd for C<sub>32</sub>H<sub>34</sub>FN<sub>5</sub>O<sub>7</sub>+H1 620.2515, found 620.2532.</div></div><div id="sec4_1_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> (3<i>S</i>)-3-({<i>N</i>-[(4-Methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl 2,6-Dimethylbenzoate (<b>26</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of (3<i>S</i>)-3-({<i>N</i>-[(4-methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl acetate but substituting 2,6-dimethylbenzoic acid and making noncritical variations provided a crude brown foam. This material was purified by Biotage MPLC (25 M, 3–4% methanol/dichloromethane) to afford 177 mg (73%) of the title compound as an off-white glass. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 11.58 (d, <i>J</i> = 2 Hz, 1H), 8.63 (d, <i>J</i> = 8 Hz, 1H), 8.46 (d, <i>J</i> = 8 Hz, 1H), 7.65 (s, 1H), 7.36 (s, 1H), 7.25 (t, <i>J</i> = 8 Hz, 1H), 7.08 (m, 3H), 7.00 (d, <i>J</i> = 8 Hz, 1H), 6.49 (d, <i>J</i> = 8 Hz, 1H), 5.11 (s, 2H), 4.51 (m, 2H), 3.87 (s, 3H), 3.10 (m, 2H), 2.32–2.21 (m, 7H), 2.07 (m, 2H), 1.78–1.53 (m, 5H), 0.94 (d, <i>J</i> = 8 Hz, 3H), 0.89 (d, <i>J</i> = 8 Hz, 3H); MS (ESI+) for C<sub>33</sub>H<sub>40</sub>N<sub>4</sub>O<sub>7</sub><i>m</i>/<i>z</i> 605.2 (M + H)<sup>+</sup>; anal. calcd for C<sub>33</sub>H<sub>40</sub>N<sub>4</sub>O<sub>7</sub>·0.3 H<sub>2</sub>O·0.2 CHCl<sub>3</sub>: C, 62.90; H, 6.49; N, 8.84. Found: C, 62.95; H, 6.42; N, 8.72. HRMS (ESI+) calcd for C<sub>33</sub>H<sub>40</sub>N<sub>4</sub>O<sub>7</sub>+H1 605.2970, found 605.2985.</div></div><div id="sec4_1_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> (3<i>S</i>)-3-({<i>N</i>-[(4-Methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl 2,6-Dimethoxybenzoate (<b>27</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of (3<i>S</i>)-3-({<i>N</i>-[(4-methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl acetate but substituting 2,6-dimethoxybenzoic acid and making noncritical variations provided a crude brown foam. This material was purified by Biotage MPLC (25 M, 3–4% methanol/dichloromethane) to afford 169 mg (66%) of the title compound as an off-white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 11.58 (d, <i>J</i> = 2 Hz, 1H), 8.55 (d, <i>J</i> = 8 Hz, 1H), 8.45 (d, <i>J</i> = 8 Hz, 1H), 7.64 (s, 1H), 7.40–7.35 (m, 2H), 7.08 (t, <i>J</i> = 8 Hz, 1H), 7.00 (d, <i>J</i> = 8 Hz, 1H), 6.71 (d, <i>J</i> = 8 Hz, 1H), 6.50 (d, <i>J</i> = 8 Hz, 1H), 5.01 (d, <i>J</i> = 16 Hz, 1H), 4.95 (d, <i>J</i> = 16 Hz, 1H), 4.53–4.48 (m, 2H), 3.87 (s, 3H), 3.75 (s, 6H), 3.08 (m, 2H), 2.31 (m, 1H), 2.11–1.90 (m, 2H), 1.75–1.55 (m, 5H), 0.94 (d, <i>J</i> = 8 Hz, 3H), 0.89 (d, <i>J</i> = 8 Hz, 3H); MS (ESI+) for C<sub>33</sub>H<sub>40</sub>N<sub>4</sub>O<sub>9</sub><i>m</i>/<i>z</i> 637.2 (M + H)<sup>+</sup>; anal. calcd for C<sub>33</sub>H<sub>40</sub>N<sub>4</sub>O<sub>9</sub>·0.25 H<sub>2</sub>O·0.25 CHCl<sub>3</sub>: C, 59.51; H, 6.12; N, 8.35. Found: C, 59.49; H, 6.08; N, 8.42.</div></div><div id="sec4_1_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> (3<i>S</i>)-3-({<i>N</i>-[(4-Methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl 2-Cyanobenzoate (<b>28</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of (3<i>S</i>)-3-({<i>N</i>-[(4-methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl acetate but substituting 2-cyanobenzoic acid and making noncritical variations provided a crude brown oily solid. This material was triturated with chloroform/ethyl acetate to afford 91 mg (38%) of the title compound as a white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 11.58 (s, 1H), 8.65 (d, <i>J</i> = 8 Hz, 1H), 8.46 (d, <i>J</i> = 8 Hz, 1H), 8.17 (m, 1H), 8.04 (m, 1H), 7.88 (m, 1H), 7.65 (s, 1H), 7.37 (s, 1H), 7.08 (t, <i>J</i> = 8 Hz, 1H), 6.99 (d, <i>J</i> = 8 Hz, 1H), 6.49 (d, <i>J</i> = 8 Hz, 1H), 5.19 (m, 2H), 4.50 (m, 2H), 3.87 (s, 3H), 3.11 (m, 2H), 2.32 (m, 1H), 2.07–1.97 (m, 2H), 1.76–1.56 (m, 5H), 0.94 (d, <i>J</i> = 6 Hz, 3H), 0.89 (d, <i>J</i> = 6 Hz, 3H); MS (ESI+) for C<sub>32</sub>H<sub>35</sub>N<sub>5</sub>O<sub>7</sub><i>m</i>/<i>z</i> 602.2 (M + H)<sup>+</sup>; anal. calcd for C<sub>32</sub>H<sub>35</sub>N<sub>5</sub>O<sub>7</sub>·0.4 H<sub>2</sub>O: C, 63.12, H, 5.93; N, 11.50. Found: C, 63.16; H, 5.96; N, 11.43; HRMS (ESI+) calcd for C<sub>32</sub>H<sub>35</sub>N<sub>5</sub>O<sub>7</sub>+H1 602.2609, found 602.2610.</div></div><div id="sec4_1_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> (3<i>S</i>)-3-[(<i>tert</i>-Butoxycarbonyl)amino]-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl 2,6-Dichlorobenzoate</h4><div class="NLM_p last">A solution of <i>tert</i>-butyl ((1<i>S</i>)-3-bromo-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)carbamate (1.17 g, 3.2 mmol) in DMF (16 mL) was treated with 2,6-dichlorobenzoic acid (794 mg, 4.2 mmol) followed by cesium fluoride (1.18 g, 7.4 mmol). The resulting suspension was placed in a preheated oil bath at 65 °C for 2 h. The reaction was cooled to an ambient temperature, diluted with ethyl acetate (100 mL), washed once with water (40 mL) and with brine (40 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated to give a crude yellow syrup. This material was purified by Biotage MPLC (25 M, 4.5% methanol/dichloromethane) to afford 1.19 mg (80%) of the title compound as an off-white glass. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.60 (m, 5H), 5.35 (s, 2H), 4.32 (m, 1H), 3.23 (m, 2H), 2.24 (m, 1H), 2.11 (m, 1H), 1.86 (m, 1H), 1.67 (m, 2H), 1.35 (s, 9H); MS (ESI+) for C<sub>20</sub>H<sub>24</sub>Cl<sub>2</sub>N<sub>2</sub>O<sub>6</sub><i>m</i>/<i>z</i> 481.0 (M + Na)<sup>+</sup>; HRMS (ESI+) calcd for C<sub>20</sub>H<sub>24</sub>Cl<sub>2</sub>N<sub>2</sub>O<sub>6</sub>+H 481.0903, found 481.0890.</div></div><div id="sec4_1_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> <i>N</i><sup>2</sup>-(<i>tert</i>-Butoxycarbonyl)-<i>N</i><sup>1</sup>-((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-<span class="smallcaps smallerCapital">l</span>-leucinamide</h4><div class="NLM_p last">A solution of (3<i>S</i>)-3-[(2<i>S</i>)-2-amino-4-chloro-3-oxobutyl]pyrrolidin-2-one hydrochloride (391 mg, 1.6 mmol) and Boc-Leu-OH (412 mg, 1.8 mmol) in DMF (9 mL) was placed under an atmosphere of N<sub>2</sub> and cooled to 0 °C. This clear pale yellow solution was successively treated with HATU (678 mg, 1.8 mmol) followed by <i>N</i>-methylmorpholine (0.41 mL, 3.7 mmol). The reaction mixture gradually became opaque and after 1 h was quenched with 1:1 ice/sat aqueous NaHCO<sub>3</sub> (40 mL) and extracted three times with ethyl acetate (40 mL). The combined organics were washed once with brine (30 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated to give a yellow syrup. This material was purified by LC (50 g 230–400 SiO<sub>2</sub>, 3–5% methanol/chloroform) to afford 636 mg (40%) of <i>N</i><sup>2</sup>-(<i>tert</i>-butoxycarbonyl)-<i>N</i><sup>1</sup>-((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-<span class="smallcaps smallerCapital">l</span>-leucinamide as a white foam. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 8.47 (d, <i>J</i> = 8 Hz, 1H), 7.64 (s, 1H), 7.04 (d, <i>J</i> = 8 Hz, 1H), 4.60 (d, <i>J</i> = 16 Hz, 1H), 4.53 (d, <i>J</i> = 16 Hz, 1H), 4.40 (m, 1H), 3.90 (m, 1H), 3.16 (m, 1H), 3.08 (m, 1H), 2.24 (m, 1H), 2.10 (m, 1H), 1.98 (m, 1H), 1.63 (m, 2H), 1.45–1.37 (m, 11 H), 0.89 (d, <i>J</i> = 4 Hz, 3H), 0.85 (d, <i>J</i> = 4 Hz, 3H); MS (ESI+) for C<sub>19</sub>H<sub>32</sub>ClN<sub>3</sub>O<sub>5</sub><i>m</i>/<i>z</i> 418.1 (M + H)<sup>+</sup>. Anal. calcd for C<sub>19</sub>H<sub>32</sub>ClN<sub>3</sub>O<sub>5</sub>·0.6 H<sub>2</sub>O: C, 53.22; H, 7.81; N, 9.80. Found: C, 53.00; H, 7.65; N, 9.54. HRMS (ESI+) calcd for C<sub>19</sub>H<sub>32</sub>ClN<sub>3</sub>O<sub>5</sub>+H1 418.2103, found 418.2091.</div></div><div id="sec4_1_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> (3<i>S</i>)-3-{[<i>N</i>-(<i>tert</i>-Butoxycarbonyl)-<span class="smallcaps smallerCapital">l</span>-leucyl]amino}-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl 2,6-Dichlorobenzoate (<b>29</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of <i>N</i><sup>2</sup>-(<i>tert</i>-butoxycarbonyl)-<i>N</i><sup>1</sup>-((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-<span class="smallcaps smallerCapital">l</span>-leucinamide but substituting (3<i>S</i>)-3-[(<i>tert</i>-butoxycarbonyl)amino]-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl 2,6-dichlorobenzoate and making noncritical variations provided a brown foam. This material was purified by LC (50 g 230–400 SiO<sub>2</sub>, 2.5–3.5% methanol/chloroform) to afford 413 mg (61%) of the title compound as a light yellow foam. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 8.47 (d, <i>J</i> = 8 Hz, 1H), 7.60 (m, 4H), 7.05 (d, <i>J</i> = 8 Hz, 1H), 5.19 (d, <i>J</i> = 16 Hz, 1H), 5.13 (d, <i>J</i> = 16 Hz, 1H), 4.47 (m, 1H), 3.95 (m, 1H), 3.16 (m, 1H), 3.08 (m, 1H), 2.27 (m, 1H), 2.10 (m, 1H), 2.03 (m, 1H), 1.65 (m, 2H), 1.47–1.37 (m, 11 H), 0.89 (d, <i>J</i> = 4 Hz, 3H), 0.85 (d, <i>J</i> = 4 Hz, 3H); MS (ESI+) for C<sub>26</sub>H<sub>35</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>7</sub><i>m</i>/<i>z</i> 572.1 (M + H)<sup>+</sup>. Anal. calcd for C<sub>26</sub>H<sub>35</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>7</sub>·0.5 H<sub>2</sub>O: C, 53.70; H, 6.24; N, 7.23. Found: C, 53.74; H, 6.31; N, 7.31; HRMS (ESI+) calcd for C<sub>26</sub>H<sub>35</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>7</sub>+Na 594.1744, found 594.1729.</div></div><div id="sec4_1_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> (3<i>S</i>)-3-[(<i>N</i>-Acetyl-<span class="smallcaps smallerCapital">l</span>-leucyl)amino]-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl 2,6-Dichlorobenzoate (<b>30</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of <i>N</i><sup>2</sup>-acetyl-<i>N</i><sup>1</sup>-[(1<i>S</i>)-1-(chloroacetyl)-4-(dimethylamino)-4-oxobutyl]-<i>N</i><sup>2</sup>-methyl-<span class="smallcaps smallerCapital">l</span>-leucinamide but substituting (3<i>S</i>)-3-{[<i>N</i>-(<i>tert</i>-butoxycarbonyl)-<span class="smallcaps smallerCapital">l</span>-leucyl]amino}-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl 2,6-dichlorobenzoate and making noncritical variations provided a pale yellow oil. This material was purified by Biotage MPLC (25 M, 3.5–4.5% methanol/dichloromethane) to afford 161 mg (78%) of the title compound as a white foam. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 8.56 (d, <i>J</i> = 8 Hz, 1H), 8.06 (d, <i>J</i> = 8 Hz, 1H), 7.65–7.53 (m, 4H), 5.17 (d, <i>J</i> = 16 Hz, 1H), 5.10 (d, <i>J</i> = 16 Hz, 1H), 4.44 (m, 1H), 4.25 (m, 1H), 3.12 (m, 2H), 2.25 (m, 1H), 2.06 (m, 1H), 1.97 (m, 1H), 1.84 (s, 3H), 1.63 (m, 3H), 1.45 (m, 2H), 0.90 (d, <i>J</i> = 8 Hz, 3H), 0.85 (d, <i>J</i> = 8 Hz, 3H); MS (ESI+) for C<sub>23</sub>H<sub>29</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>6</sub><i>m</i>/<i>z</i> 514.0 (M + H)<sup>+</sup>; anal. calcd for C<sub>23</sub>H<sub>29</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>6</sub>·0.25 H<sub>2</sub>O·0.1 EtOAc: C, 53.26; H, 5.79; N, 7.96. Found: C, 53.27; H, 5.82; N, 7.84. HRMS (ESI+) calcd for C<sub>23</sub>H<sub>29</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>6</sub>+H1 514.1506, found 514.1508.</div></div><div id="sec4_1_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> (3<i>S</i>)-3-{[<i>N</i>-(1<i>H</i>-Benzimidazol-2-ylcarbonyl)-<span class="smallcaps smallerCapital">l</span>-leucyl]amino}-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl 2,6-Dichlorobenzoate (<b>31</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3-methylbutyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide but substituting (3<i>S</i>)-3-{[<i>N</i>-(<i>tert</i>-butoxycarbonyl)-<span class="smallcaps smallerCapital">l</span>-leucyl]amino}-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl 2,6-dichlorobenzoate and 1H-benzimidazole-2-carboxylic acid and making noncritical variations provided a crude yellow oil. This material was purified by Biotage MPLC (25 M, 3–4% methanol/dichloromethane) to afford 95 mg (72%) of the title compound as an off-white foam. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 13.32 (s, 1H), 8.89 (d, <i>J</i> = 8 Hz, 1H), 8.60 (d, <i>J</i> = 8 Hz, 1H), 7.74 (d, <i>J</i> = 8 Hz, 1H), 7.59 (m, 5H), 7.29 (m, 2H), 5.20 (d, <i>J</i> = 20 Hz, 1H), 5.15 (d, <i>J</i> = 20 Hz, 1H), 4.56 (m, 2H), 3.12 (m, 2H), 2.30 (m, 1H), 2.01 (m, 2H), 1.83 (m, 1H), 1.65 (m, 4H), 0.92 (d, <i>J</i> = 8 Hz, 3H), 0.89 (d, <i>J</i> = 8 Hz, 3H); MS (ESI+) for C<sub>29</sub>H<sub>31</sub>Cl<sub>2</sub>N<sub>5</sub>O<sub>6</sub><i>m</i>/<i>z</i> 616.0 (M + H)<sup>+</sup>. Anal. calcd for C<sub>29</sub>H<sub>31</sub>Cl<sub>2</sub>N<sub>5</sub>O<sub>6</sub>: C, 56.50; H, 5.07; N, 11.36. Found: C, 57.28; H, 5.32; N, 11.35. HRMS (ESI+) calcd for C<sub>29</sub>H<sub>31</sub>Cl<sub>2</sub>N<sub>5</sub>O<sub>6</sub>+H1 616.1724, found 616.1729.</div></div><div id="sec4_1_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> (3<i>S</i>)-3-({4-Methyl-<i>N</i>-[(2<i>R</i>)-tetrahydrofuran-2-ylcarbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl 2,6-Dichlorobenzoate (<b>3</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of (3<i>S</i>)-3-({<i>N</i>-[(4-methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl acetate but substituting <i>N</i><sup>1</sup>-((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-4-methyl-<i>N</i><sup>2</sup>-[(2<i>R</i>)-tetrahydrofuran-2-ylcarbonyl]-<span class="smallcaps smallerCapital">l</span>-leucinamide and 2,6-dichlorobenzoic acid and making noncritical variations provided a light amber residue. The residue was purified by preparative HPLC (Luna 10μ C18) eluting with a gradient of MeCN containing 0.1% AcOH in water containing 0.1% AcOH to give 0.155 g (54%) of the title compound as a cream colored solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.52 (d, <i>J</i> = 8 Hz, 1H), 7.79 (d, <i>J</i> = 8 Hz, 1H), 7.71 (s, 1H), 7.66–7.55 (m, 3H), 5.18 (s, 2H), 4.54–4.40 (m, 1H), 4.37–4.35 (m, 1H), 4.25 (m, 1H), 3.98–3.91 (m, 1H), 3.84–3.72 (m, 1H), 3.23–3.07 (m, 2H), 2.33–2.23 (m, 1H), 2.16–2.05 (m, 2H), 1.86–1.74 (m, 3H), 1.72–1.62 (m, 5H), 0.89 (s, 9H); MS (ESI+) for C<sub>27</sub>H<sub>35</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>7</sub><i>m</i>/<i>z</i> 584 (M + H). Anal. calcd for C<sub>27</sub>H<sub>35</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>7</sub>·0.5 H<sub>2</sub>O: C, 54.64; H, 6.11; N, 7.08. Found: C, 54.26; H, 6.00; N, 6.87. HRMS (ESI+) calcd for C<sub>27</sub>H<sub>35</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>7</sub>+H1 584.1925, found 584.1921.</div></div><div id="sec4_1_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> <i>N</i><sup>2</sup>-(<i>tert</i>-Butoxycarbonyl)-<i>N</i><sup>1</sup>-((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-4-methyl-<span class="smallcaps smallerCapital">l</span>-leucinamide (<b>32</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of <i>N</i><sup>2</sup>-(<i>tert</i>-butoxycarbonyl)-<i>N</i><sup>1</sup>-((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-<span class="smallcaps smallerCapital">l</span>-leucinamide but substituting Boc-(β t-butyl)-Ala-OH and making noncritical variations provided a golden syrup. This material was purified by Biotage MPLC (65i column, 2.5–3.5% methanol/chloroform) to afford 3.41 g (40%) of the title compound as a white foam. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 8.39 (d, <i>J</i> = 8 Hz, 1H), 7.64 (s, 1H), 7.00 (d, <i>J</i> = 8 Hz, 1H), 4.58 (d, <i>J</i> = 16 Hz, 1H), 4.52 (d, <i>J</i> = 16 Hz, 1H), 4.38 (m, 1H), 3.92 (m, 1H), 3.15 (t, <i>J</i> = 8 Hz, 1H), 3.07 (q, <i>J</i> = 8 Hz, 1H), 2.22 (m, 1H), 2.09 (m, 1H), 1.97 (m, 1H), 1.62 (m, 2H), 1.50 (m, 2H), 1.36 (s, 9H), 0.88 (s, 9H); MS (ESI+) for C<sub>20</sub>H<sub>34</sub>ClN<sub>3</sub>O<sub>6</sub><i>m</i>/<i>z</i> 432.1 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> <i>N</i><sup>1</sup>-((1<i>S</i>)-3-(Benzyloxy)-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-<i>N</i><sup>2</sup>-(<i>tert</i>-butoxycarbonyl)-4-methyl-<span class="smallcaps smallerCapital">l</span>-leucinamide (<b>33</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of <i>tert</i>-butyl [(1<i>S</i>)-2-[((1<i>S</i>)-3-(benzyloxy)-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]-1-(cyclohexylmethyl)-2-oxoethyl]carbamate but substituting Boc-(β t-butyl)-Ala-OH and making noncritical variations provided a brown syrup. This material was purified by LC (33 g 230–400 SiO<sub>2</sub>, 1–2.5% methanol/chloroform) to afford 258 mg (55%) of the title compound as a light yellow foam. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 8.23 (d, <i>J</i> = 8 Hz, 1H), 7.62 (s, 1H), 7.32 (m, 5H), 6.96 (d, <i>J</i> = 8 Hz, 1H), 4.46 (s, 2H), 4.35–4.33 (m, 2H), 4.21 (d, <i>J</i> = 16 Hz, 1H), 3.95 (m, 1H), 3.09 (m, 2H), 2.24 (m, 1H), 2.07 (m, 1H), 1.92 (m, 1H), 1.58 (m, 2H), 1.46 (m, 2H), 1.35 (s, 9H), 0.87 (m, 9H); MS (ESI+) for C<sub>27</sub>H<sub>41</sub>N<sub>3</sub>O<sub>6</sub><i>m</i>/<i>z</i> 504.2 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-(Benzyloxy)-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3,3-dimethylbutyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide (<b>34</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of <i>N</i>-[(1<i>S</i>)-2-[((1<i>S</i>)-3-(benzyloxy)-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]-1-(cyclohexylmethyl)-2-oxoethyl]-4-methoxy-1<i>H</i>-indole-2-carboxamide but substituting <i>N</i><sup>1</sup>-((1<i>S</i>)-3-(benzyloxy)-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-<i>N</i><sup>2</sup>-(<i>tert</i>-butoxycarbonyl)-4-methyl-<span class="smallcaps smallerCapital">l</span>-leucinamide and making noncritical variations provided a golden syrup. This material was purified by Biotage MPLC (25 M column, 3% methanol/chloroform) to afford 173 mg (60%) of the title compound as an off-white foam. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.58 (s, 1H), 8.43 (app t, <i>J</i> = 8 Hz, 2H), 7.62 (s, 1H), 7.32 (m, 6H), 7.08 (t, <i>J</i> = 8 Hz, 1H), 7.00 (d, <i>J</i> = 8 Hz, 1H), 6.49 (d, <i>J</i> = 8 Hz, 1H), 4.52 (m, 1H), 4.47–4.35 (m, 4H), 4.22 (d, <i>J</i> = 16 Hz, 1H), 3.87 (s, 3H), 3.05 (m, 2H), 2.28 (m, 1H), 2.06 (m, 1H), 1.93 (m, 1H), 1.64–1.57 (m, 3H), 0.92 (s, 9H); MS (ESI+) for C<sub>32</sub>H<sub>40</sub>N<sub>4</sub>O<sub>6</sub><i>m</i>/<i>z</i> 577.2 (M + H)<sup>+</sup>; anal. calcd for C<sub>32</sub>H<sub>40</sub>N<sub>4</sub>O<sub>6</sub>·0.33 methanol·0.25 H<sub>2</sub>O: C, 65.62; H, 7.13; N, 9.47. Found: C, 65.65; H, 6.90; N, 9.58; HRMS (ESI+) calcd for C<sub>32</sub>H<sub>40</sub>N<sub>4</sub>O<sub>6</sub> 577.3021, found 577.3001.</div></div><div id="sec4_1_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-Chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3,3-dimethylbutyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide (<b>35</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3-methylbutyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide but substituting <i>N</i><sup>2</sup>-(<i>tert</i>-butoxycarbonyl)-<i>N</i><sup>1</sup>-((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-4-methyl-<span class="smallcaps smallerCapital">l</span>-leucinamide and making noncritical variations provided a pale yellow solid. This material was purified by Biotage MPLC (40 M column, 2–4.5% methanol/chloroform) to afford 920 g (60%) of the title compound as an off-white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.59 (d, <i>J</i> = 1.8 Hz, 1H), 8.54 (d, <i>J</i> = 7.8 Hz, 1H), 8.45 (d, <i>J</i> = 8.1 Hz, 1H), 7.64 (s, 1H), 7.32 (d, <i>J</i> = 1.8 Hz, 1H), 7.08 (t, <i>J</i> = 8.0 Hz, 1H), 7.00 (d, <i>J</i> = 8.3 Hz, 1H), 6.49 (d, <i>J</i> = 7.6 Hz, 1H), 4.52–4.62 (m, 1H), 4.40–4.51 (m, 2H), 3.87 (s, 3H), 3.01–3.16 (m, 2H), 2.19–2.31 (m, 1H), 2.03–2.12 (m, 1H), 1.91–2.01 (m, 1H), 1.81 (dd, <i>J</i> = 14.1, 9.9 Hz, 1H), 1.54–1.72 (m, 3H), 0.93 (s, 9H); MS (ESI+) for C<sub>25</sub>H<sub>33</sub>ClN<sub>4</sub>O<sub>5</sub><i>m</i>/<i>z</i> 505.1 (M + H)<sup>+</sup>; HRMS (ESI+) calcd for (M + H)<sup>+</sup> 505.2212, found 505.2204.</div></div><div id="sec4_1_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-Hydroxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3,3-dimethylbutyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide (<b>36</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-hydroxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3-methylbutyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide but substituting <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3,3-dimethylbutyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide and making noncritical variations provided a crude brown glass. This material was purified by Biotage MPLC (40 M column, 2–10% methanol/chloroform) to afford 415 mg (53%) of the title compound as a white solid. MS (ESI+) for C<sub>25</sub>H<sub>34</sub>N<sub>4</sub>O<sub>6</sub><i>m</i>/<i>z</i> 487.2 (M + H)<sup>+</sup>. Alternatively, following the procedure described for the preparation of <i>N</i>-{(1<i>S</i>)-1-(cyclohexylmethyl)-2-[((1<i>S</i>)-3-hydroxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]-2-oxoethyl}-4-methoxy-1<i>H</i>-indole-2-carboxamide but substituting <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-(benzyloxy)-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3,3-dimethylbutyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide and making noncritical variations provided a tan solid. This material was purified by Biotage MPLC (25 M column, 6–7% methanol/chloroform) to afford 100 mg (77%) of the title compound as a white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 11.58 (s, 1H), 8.42 (d, <i>J</i> = 8 Hz, 1H), 8.37 (d, <i>J</i> = 8 Hz, 1H), 7.62 (s, 1H), 7.32 (s, 1H), 7.08 (t, <i>J</i> = 8 Hz, 1H), 6.99 (d, <i>J</i> = 8 Hz, 1H), 6.49 (d, <i>J</i> = 8 Hz, 1H), 5.04 (t, <i>J</i> = 8 Hz, 1H), 4.52 (m, 1H), 4.42 (m, 1H), 4.24 (dd, <i>J</i> = 8, 20 Hz, 1H), 4.12 (dd, <i>J</i> = 8, 20 Hz, 1H), 3.87 (s, 3H), 3.09 (m, 2H), 2.25 (m, 1H), 2.06 (m, 1H), 1.93 (m, 1H), 1.80 (dd, <i>J</i> = 8, 16 Hz, 1H), 0.93 (s, 9H); MS (ESI+) for C<sub>25</sub>H<sub>34</sub>N<sub>4</sub>O<sub>6</sub><i>m</i>/<i>z</i> 487.2 (M + H)<sup>+</sup>; anal. calcd for C<sub>25</sub>H<sub>34</sub>N<sub>4</sub>O<sub>6</sub>·0.3 H<sub>2</sub>O: C, 61.03; H, 7.09; N, 11.39. Found: C, 61.07; H, 7.09; N, 11.22; HRMS (ESI+) calcd for C<sub>25</sub>H<sub>34</sub>N<sub>4</sub>O<sub>6</sub> 487.2551, found 487.2541.</div></div><div id="sec4_1_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 4-Methoxy-<i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-methoxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3,3-dimethylbutyl)-1<i>H</i>-indole-2-carboxamide (<b>37</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of 4-methoxy-<i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-methoxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-4-methylpentyl)-1<i>H</i>-indole-2-carboxamide but substituting <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-hydroxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3,3-dimethylbutyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide and making noncritical variations provided a crude product. This material was purified by Biotage MPLC (40S column, 2–5% methanol/chloroform) to afford 21 mg (9%) of the title compound as a pale tan solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.58 (s, 1H), 8.42 (d, <i>J</i> = 6.8 Hz, 1H), 7.62 (s, 1H), 7.32 (s, 1H), 7.08 (t, <i>J</i> = 7.7 Hz, 1H), 7.00 (d, <i>J</i> = 7.8 Hz, 1H), 6.49 (d, <i>J</i> = 7.3 Hz, 1H), 4.49–4.55 (m, 1H), 4.32–4.41 (m, 1H), 4.21–4.30 (m, 1H), 4.12 (t, <i>J</i> = 16.9 Hz, 1H), 3.87 (s, 3H), 3.23 (s, 3H), 3.01–3.16 (m, 2H), 2.20–2.32 (m, 1H), 2.03–2.15 (m, 1H), 1.87–1.98 (m, 1H), 1.73–1.85 (m, 1H), 1.54–1.70 (m, 3H), 0.93 (s, 9H); MS (ESI+) for C<sub>26</sub>H<sub>36</sub>N<sub>4</sub>O<sub>6</sub><i>m</i>/<i>z</i> 501.2 (M + H)<sup>+</sup>; HRMS (ESI+) calcd for (M + H)<sup>+</sup> 501.2708, found 501.2701.</div></div><div id="sec4_1_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-Hydroxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3-methylbutyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide (<b>4</b>)</h4><div class="NLM_p last">A solution of <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3-methylbutyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide (488 mg, 0.99 mmol) and benzoylformic acid (195 mg, 1.3 mmol) in DMF (6.5 mL) was placed under an atmosphere of N<sub>2</sub>. This clear pale yellow solution was treated with cesium fluoride (350 mg, 2.3 mmol) followed by heating to 65 °C. After 4 h, the now yellow suspension was cooled to RT, diluted with ethyl acetate (60 mL), washed three times with water (30 mL) and once with brine (30 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated to give (3<i>S</i>)-3-({<i>N</i>-[(4-methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl oxo(phenyl)acetate as a crude yellow foam. MS (ESI+) for C<sub>32</sub>H<sub>36</sub>N<sub>4</sub>O<sub>8</sub><i>m</i>/<i>z</i> 605.2 (M + H)<sup>+</sup>. A solution of the crude (3<i>S</i>)-3-({<i>N</i>-[(4-methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-leucyl}amino)-2-oxo-4-[(3<i>S</i>)-2-oxopyrrolidin-3-yl]butyl oxo(phenyl)acetate in methanol (40 mL) was placed under an atmosphere of N<sub>2</sub> and treated with potassium carbonate (7 mg, 0.05 mmol) with vigorous stirring. After 1 h, the volatiles were removed <i>in vacuo</i> (bath < 30 °C) to give a crude yellow glass. This material was purified by Biotage MPLC (25 M column, 6% methanol/chloroform) to afford 346 mg (73%) of the title compound as an off-white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 11.56 (s, 1H), 8.44 (d, <i>J</i> = 8 Hz, 1H), 8.39 (d, <i>J</i> = 8 Hz, 1H), 7.61 (s, 1H), 7.35 (s, 1H), 7.08 (t, <i>J</i> = 8 Hz, 1H), 6.99 (d, <i>J</i> = 8 Hz, 1H), 6.49 (d, <i>J</i> = 8 Hz, 1H), 5.04 (t, <i>J</i> = 8 Hz, 1H), 4.46 (m, 2H), 4.25 (dd, <i>J</i> = 8, 20 Hz, 1H), 4.13 (dd, <i>J</i> = 8, 20 Hz, 1H), 3.87 (s, 3H), 3.10 (m, 2H), 2.28 (m, 1H), 2.08 (m, 1H), 1.92 (m, 1H), 1.70–1.53 (m, 5H), 0.93 (d, <i>J</i> = 8 Hz, 3H), 0.89 (d, <i>J</i> = 8 Hz, 3H); MS (ESI+) for C<sub>24</sub>H<sub>32</sub>N<sub>4</sub>O<sub>6</sub><i>m</i>/<i>z</i> 473.2 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> <i>N</i><sup>1</sup>-((1<i>S</i>)-3-(Benzyloxy)-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-<i>N</i><sup>2</sup>-(<i>tert</i>-butoxycarbonyl)-<i>N</i><sup>2</sup>-methyl-<span class="smallcaps smallerCapital">l</span>-leucinamide</h4><div class="NLM_p last">Following the procedure described for the preparation of <i>tert</i>-butyl [(1<i>S</i>)-2-[((1<i>S</i>)-3-(benzyloxy)-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]-1-(cyclohexylmethyl)-2-oxoethyl]carbamate but substituting Boc-<i>N</i>-methyl-Leu and making noncritical variations provided a crude brown oil. This material was purified by Biotage flash chromatography, eluting with methanol/dichloromethane to afford the title compound (123 mg, 47%) as a yellow gum. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.22–7.33 (m, 5H), 5.76 (bd, 1H), 4.73 (s, 1H), 4.46–4.59 (m, 2H), 4.08–4.27 (m, 2H), 3.14–3.28 (m, 2H), 2.68 (s, 3H), 2.16–2.38 (m, 2H), 1.88–2.00 (m, 1H), 1.56–1.86 (m, 5H), 1.41 (s, 9H), 0.94 (d, <i>J</i> = 8 Hz, 3H), 0.89 (d, <i>J</i> = 8 Hz, 3H).</div></div><div id="sec4_1_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-(Benzyloxy)-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3-methylbutyl)-4-methoxy-<i>N</i>-methyl-1<i>H</i>-indole-2-carboxamide</h4><div class="NLM_p last">Following the procedure described for the preparation of <i>N</i>-[(1<i>S</i>)-2-[((1<i>S</i>)-3-(benzyloxy)-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]-1-(cyclohexylmethyl)-2-oxoethyl]-4-methoxy-1<i>H</i>-indole-2-carboxamide but substituting <i>N</i><sup>1</sup>-((1<i>S</i>)-3-(benzyloxy)-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-<i>N</i><sup>2</sup>-(<i>tert</i>-butoxycarbonyl)-<i>N</i><sup>2</sup>-methyl-<span class="smallcaps smallerCapital">l</span>-leucinamide and making noncritical variations provided a crude brown oil. This material was purified by Biotage flash chromatography, eluting with methanol/dichloromethane to afford the title compound as a clear oil, 67 mg, 48%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 9.83 (s, 1H), 7.95 (s, 1H), 7.26–7.36 (m, 5H), 7.18 (t, <i>J</i> = 8.0 Hz, 1H), 7.04 (d, <i>J</i> = 8.3 Hz, 1H), 7.00 (s, 1H), 6.48 (d, <i>J</i> = 7.8 Hz, 1H), 5.87 (s, 1H), 5.18–5.27 (m, 1H), 4.70 (s, 1H), 4.50–4.63 (m, 2H), 4.16–4.36 (m, 2H), 3.93 (s, 3H), 3.32 (s, 3H), 3.03–3.18 (m, 2H), 2.22 (s, 2H), 1.91–2.02 (m, 1H), 1.81 (t, <i>J</i> = 7.3 Hz, 3H), 1.66–1.72 (m, 1H), 1.55 (s, 1H), 0.87–1.01 (m, 6H); MS (API-ES−) for C<sub>32</sub>H<sub>40</sub>N<sub>4</sub>O<sub>6</sub><i>m</i>/<i>z</i> 576.7 (M – H)<sup>−</sup>.</div></div><div id="sec4_1_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-Hydroxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3-methylbutyl)-4-methoxy-<i>N</i>-methyl-1<i>H</i>-indole-2-carboxamide (<b>38</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of <i>N</i>-{(1<i>S</i>)-1-(cyclohexylmethyl)-2-[((1<i>S</i>)-3-hydroxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]-2-oxoethyl}-4-methoxy-1<i>H</i>-indole-2-carboxamide but substituting N-((1<i>S</i>)-1-{[((1<i>S</i>)-3-(benzyloxy)-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3-methylbutyl)-4-methoxy-<i>N</i>-methyl-1<i>H</i>-indole-2-carboxamide and making noncritical variations provided a crude clear glass. This material was purified by Biotage flash chromatography, eluting with methanol/dichloromethane to afford the title compound as a white foam, 42 mg, 82%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 10.16 (s, 1H), 8.26 (s, 1H), 7.18 (t, <i>J</i> = 8.0 Hz, 1H), 7.05 (d, <i>J</i> = 8.1 Hz, 1H), 6.99 (d, <i>J</i> = 2.3 Hz, 1H), 6.48 (d, <i>J</i> = 7.6 Hz, 1H), 6.09 (s, 1H), 5.14–5.27 (m, 1H), 4.51–4.65 (m, 1H), 4.25–4.50 (m, 2H), 3.94 (s, 3H), 3.26–3.46 (m, 3H), 3.11–3.24 (m, 1H), 3.01–3.11 (m, 1H), 1.48–2.39 (m, 9H), 0.88–1.02 (m, 6H); MS (API-ES–-) for C<sub>25</sub>H<sub>34</sub>N<sub>4</sub>O<sub>6</sub><i>m</i>/<i>z</i> 485.3 (M – H)<sup>−</sup>.</div></div><div id="sec4_1_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> <i>N</i><sup>2</sup>-(<i>tert</i>-Butoxycarbonyl)-<i>N</i><sup>1</sup>-((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-<span class="smallcaps smallerCapital">l</span>-norleucinamide</h4><div class="NLM_p last">Following the procedure described for the preparation of <i>N</i><sup>2</sup>-(<i>tert</i>-butoxycarbonyl)-<i>N</i><sup>1</sup>-((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-<span class="smallcaps smallerCapital">l</span>-leucinamide but substituting Boc-NorLeu-OH and making noncritical variations provided a golden syrup. This material was purified by LC (100 g 230–400 SiO<sub>2</sub>, 2.5–3.5% methanol/chloroform) to afford 727 mg (42%) of the title compound as a light yellow foam. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 8.45 (d, <i>J</i> = 8 Hz, 1H), 7.62 (s, 1H), 7.00 (d, <i>J</i> = 8 Hz, 1H), 4.59 (d, <i>J</i> = 16 Hz, 1H), 4.53 (d, <i>J</i> = 16 foam in a 42% isolated yield. <sup>1</sup>H Hz, 1H), 4.38 (m, 1H), 3.80 (m, 1H), 3.15 (m, 1H), 3.06 (m, 1H), 2.22 (m, 1H), 2.07 (m, 1H), 1.98 (m, 1H), 1.63–1.51 (m, 4H), 1.36 (m, 9H), 1.24 (m, 4H), 0.83 (t, <i>J</i> = 8 Hz, 3H); MS (ESI+) for C<sub>19</sub>H<sub>32</sub>ClN<sub>3</sub>O<sub>5</sub><i>m</i>/<i>z</i> 418.1 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-Chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}pentyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide</h4><div class="NLM_p last">Following the procedure described for the preparation of <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3-methylbutyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide but substituting <i>N</i><sup>2</sup>-(<i>tert</i>-butoxycarbonyl)-<i>N</i><sup>1</sup>-((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-<span class="smallcaps smallerCapital">l</span>-norleucinamide and making no other critical variations provided a crude yellow foam. This material was purified by Biotage MPLC (25 M column, 3% methanol/chloroform) to afford 254 mg (63%) of the title compound as an off-white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 11.56 (s, 1H), 8.57 (d, <i>J</i> = 8 Hz, 1H), 8.39 (d, <i>J</i> = 4 Hz, 1H), 7.62 (s, 1H), 7.36 (s, 1H), 7.08 (t, <i>J</i> = 8 Hz, 1H), 6.99 (d, <i>J</i> = 8 Hz, 1H), 6.49 (d, <i>J</i> = 8 Hz, 1H), 4.58 (d, <i>J</i> = 16 Hz, 1H), 4.56 (d, <i>J</i> = 16 Hz, 1H), 4.43 (m, 1H), 4.35 (m, 1H), 3.87 (s, 3H), 3.09 (m, 2H), 2.27 (m, 1H), 2.08 (m, 1H), 1.97 (m, 1H), 1.74–1.54 (m, 4H), 1.30 (m, 4H), 0.86 (t, <i>J</i> = 8 Hz, 3H); MS (ESI+) for C<sub>24</sub>H<sub>31</sub>N<sub>4</sub>O<sub>5</sub>Cl <i>m</i>/<i>z</i> 491.1 (M + H)<sup>+</sup>; anal. calcd for C<sub>24</sub>H<sub>31</sub>ClN<sub>4</sub>O<sub>5</sub>: C, 58.71; H, 6.36; N, 11.41. Found: C, 58.66; H, 6.45; N, 11.22.</div></div><div id="sec4_1_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-Hydroxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}pentyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide (<b>39</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-hydroxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3-methylbutyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide but substituting of <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}pentyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide and making noncritical variations provided a crude yellow foam. This material was purified by Biotage MPLC (25 M column, 5–6% methanol/chloroform) to afford 82 mg (35%) of the title compound as a white foam. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 11.57 (s, 1H), 8.42 (d, <i>J</i> = 8 Hz, 1H), 8.38 (d, <i>J</i> = 8 Hz, 1H), 7.62 (s, 1H), 7.35 (s, 1H), 7.09 (t, <i>J</i> = 8 Hz, 1H), 6.99 (d, <i>J</i> = 8 Hz, 1H), 6.49 (d, <i>J</i> = 8 Hz, 1H), 5.06 (t, <i>J</i> = 8 Hz, 1H), 4.44 (m, 2H), 4.25 (dd, <i>J</i> = 8, 20 Hz, 1H), 4.14 (dd, <i>J</i> = 8, 20 Hz, 1H), 3.87 (s, 3H), 3.08 (m, 2H), 2.28 (m, 1H), 2.10 (m, 1H), 1.91 (m, 1H), 1.74–1.54 (m, 4H), 1.32 (m, 4H), 0.87 (t, <i>J</i> = 8 Hz, 3H); MS (ESI+) for C<sub>24</sub>H<sub>32</sub>N<sub>4</sub>O<sub>6</sub><i>m</i>/<i>z</i> 473.2 (M + H)<sup>+</sup>; anal. calcd for C<sub>24</sub>H<sub>32</sub>N<sub>4</sub>O<sub>6</sub>·0.6 H<sub>2</sub>O·0.2 ethyl acetate: C, 59.46; H, 7.01; N, 11.18. Found: C, 53.37; H, 6.94; N, 11.23; HRMS (ESI+) calcd for C<sub>24</sub>H<sub>32</sub>N<sub>4</sub>O<sub>6</sub>+H1 473.2395, found 473.2382.</div></div><div id="sec4_1_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> Methyl <i>N</i>-[(9<i>H</i>-Fluoren-9-ylmethoxy)carbonyl]-5-methyl-<span class="smallcaps smallerCapital">l</span>-norleucinate</h4><div class="NLM_p last">To a solution of <i>N</i>-[(9<i>H</i>-fluoren-9-ylmethoxy)carbonyl]-5-methyl-<span class="smallcaps smallerCapital">l</span>-norleucine (2.14 g, 5.8 mmol) in methanol (15 mL) is added toluene (30 mL) followed by the dropwise addition of TMS-diazomethane (2.9 mL, 2 M in hexane, 5.8 mmol). TLC analysis indicated incomplete reaction, and TMS-diazomethane was added dropwise until a yellow color persisted. At this time, the reaction was quenched by the addition of AcOH (1 mL) followed by concentration <i>in vacuo</i>. The residue was purified by Biotage flash chromatography, eluting with ethyl acetate/hexane to afford the title compound as a white solid, 2.18 g, 98%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.76 (d, <i>J</i> = 7.6 Hz, 2H), 7.60 (dd, <i>J</i> = 7.2, 3.9 Hz, 2H), 7.40 (t, <i>J</i> = 7.2 Hz, 2H), 7.31 (t, <i>J</i> = 7.5 Hz, 2H), 5.26 (d, <i>J</i> = 8.6 Hz, 1H), 4.31–4.51 (m, 3H), 4.23 (t, <i>J</i> = 7.1 Hz, 1H), 3.75 (s, 3H), 1.78–1.93 (m, 1H), 1.60–1.76 (m, 1H), 1.45–1.60 (m, 1H), 1.05–1.34 (m, 2H), 0.88 (d, <i>J</i> = 4 Hz, 3H), 0.86 (d, <i>J</i> = 4 Hz, 3H); MS (APCI+) for C<sub>23</sub>H<sub>27</sub>NO<sub>4</sub><i>m</i>/<i>z</i> 160.1 (M-Fmoc+H)<sup>+</sup>.</div></div><div id="sec4_1_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> Methyl <i>N</i>-(<i>tert</i>-Butoxycarbonyl)-5-methyl-<span class="smallcaps smallerCapital">l</span>-norleucinate</h4><div class="NLM_p last">To a solution of methyl <i>N</i>-[(9<i>H</i>-fluoren-9-ylmethoxy)carbonyl]-5-methyl-<span class="smallcaps smallerCapital">l</span>-norleucinate (2.18 g, 5.72 mmol) in DMF (50 mL) was added KF (2.33 g, 40.04 mmol) followed by triethylamine (1.70 mL, 12.24 mmol) and di-<i>tert-</i>butyl dicarbonate (7.39 mmol), and the mixture was stirred at an ambient temperature. After 4 h, TLC analysis indicated incomplete reaction and the reaction mixture was treated with a second portion of KF (2.7 g, 46.55 mmol) and BOC<sub>2</sub>O (800 mg, 3.67 mmol). After 16 h, the mixture was diluted with diethyl ether (300 mL), washed with satd. NaHCO<sub>3</sub> (2 × 50 mL), 1 M hydrochloric acid (2 × 50 mL), NaHCO<sub>3</sub> (50 mL), and brine (50 mL), dried over MgSO<sub>4</sub>, filtered, and the solvents were evaporated <i>in vacuo</i> to yield the crude product, which was purified by Biotage flash chromatography eluting with dichloromethane/hexane to afford the title compound as a clear oil, 980 mg, 66%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 4.96 (d, <i>J</i> = 6.8 Hz, 1H), 4.21–4.32 (m, 1H), 3.72 (s, 3H), 1.72–1.85 (m, 1H), 1.46–1.66 (m, 2H), 1.43 (s, 9H), 1.11–1.29 (m, 2H), 0.88 (d, <i>J</i> = 4 Hz, 3H), 0.86 (d, <i>J</i> = 4 Hz, 3H); MS (API-ES+) for C<sub>13</sub>H<sub>25</sub>NO<sub>4</sub><i>m</i>/<i>z</i> 282.2 (M + Na)<sup>+</sup>.</div></div><div id="sec4_1_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> <i>N</i>-(<i>tert</i>-Butoxycarbonyl)-5-methyl-<span class="smallcaps smallerCapital">l</span>-norleucine</h4><div class="NLM_p last">To a solution of methyl <i>N</i>-(<i>tert</i>-butoxycarbonyl)-5-methyl-<span class="smallcaps smallerCapital">l</span>-norleucinate (980 mg, 3.78 mmol) in THF (30 mL) at 0 °C was added a solution (precooled to 5 °C) of LiOH (1 M, 11.3 mL, 11.33 mmol), and the resulting mixture was stirred at 0 °C for 1 h and then allowed to warm to an ambient temperature. The reaction was acidified to <i>p</i>H 2 with 1 M hydrochloric acid and extracted with ethyl acetate (3 × 60 mL). The combined organics were washed with brine (100 mL), dried over MgSO<sub>4</sub>, filtered, and the solvent was removed <i>in vacuo</i> to yield the title compound as a clear oil, 990 mg, 99%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 4.96 (d, <i>J</i> = 7.8 Hz, 1H), 4.23–4.34 (m, 1H), 1.75–1.93 (m, 2H), 1.60–1.72 (m, 1H), 1.50–1.59 (m, 1H), 1.44 (s, 9H), 1.19–1.30 (m, 1H), 0.88 (d, <i>J</i> = 4 Hz, 3H), 0.86 (d, <i>J</i> = 4 Hz, 3H); MS (API-ES+) for C<sub>12</sub>H<sub>23</sub>NO<sub>4</sub><i>m</i>/<i>z</i> 268.1 (M + Na)<sup>+</sup>.</div></div><div id="sec4_1_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> <i>N</i><sup>2</sup>-(<i>tert</i>-Butoxycarbonyl)-<i>N</i><sup>1</sup>-((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-5-methyl-<span class="smallcaps smallerCapital">l</span>-norleucinamide</h4><div class="NLM_p last">Following the procedure described for the preparation of <i>N</i><sup>2</sup>-(<i>tert</i>-butoxycarbonyl)-<i>N</i><sup>1</sup>-((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-<span class="smallcaps smallerCapital">l</span>-leucinamide but substituting <i>N</i>-(<i>tert</i>-butoxycarbonyl)-5-methyl-<span class="smallcaps smallerCapital">l</span>-norleucine and making noncritical variations provided a crude golden oil. This material was purified by Biotage flash chromatography, eluting with methanol/dichloromethane to afford the title compound as an off-white solid, 360 mg, 41%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.45 (d, <i>J</i> = 8.1 Hz, 1H), 7.62 (s, 1H), 7.02 (d, <i>J</i> = 7.1 Hz, 1H), 4.49–4.62 (m, 2H), 4.33–4.44 (m, 1H), 3.78 (m, 1H), 3.15 (t, <i>J</i> = 8.7 Hz, 1H), 3.00–3.10 (m, 1H), 2.18–2.30 (m, 1H), 2.04–2.14 (m, 1H), 1.92–2.02 (m, 1H), 1.40–1.68 (m, 5H), 1.36 (s, 9H), 1.05–1.25 (m, <i>J</i> = 7.3 Hz, 2H), 0.83 (d, <i>J</i> = 1.52 Hz, 3H), 0.82 (d, <i>J</i> = 1.52 Hz, 3H); MS (API-ES+) for C<sub>20</sub>H<sub>34</sub>N<sub>3</sub>O<sub>5</sub>Cl <i>m</i>/<i>z</i> 454.2 (M + Na)<sup>+</sup>.</div></div><div id="sec4_1_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-Chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-4-methylpentyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide</h4><div class="NLM_p last">Following the procedure described for the preparation of <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3-methylbutyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide but substituting <i>N</i><sup>2</sup>-(<i>tert</i>-butoxycarbonyl)-<i>N</i><sup>1</sup>-((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-5-methyl-<span class="smallcaps smallerCapital">l</span>-norleucinamide and making no other critical variations provided a crude yellow foam. This material was purified by Biotage MPLC (25 M column, 2.5–3.5% methanol/chloroform) to afford 307 mg (73%) of the title compound as a white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 11.57 (s, 1H), 8.59 (d, <i>J</i> = 8 Hz, 1H), 8.41 (d, <i>J</i> = 4 Hz, 1H), 7.64 (s, 1H), 7.37 (s, 1H), 7.09 (t, <i>J</i> = 8 Hz, 1H), 7.00 (d, <i>J</i> = 8 Hz, 1H), 6.49 (d, <i>J</i> = 8 Hz, 1H), 4.59 (s, 2H), 4.44 (m, 1H), 4.35 (m, 1H), 3.87 (s, 3H), 3.08 (m, 2H), 2.26 (m, 1H), 2.07 (m, 1H), 1.98 (m, 1H), 1.70–1.51 (m, 5H), 1.25 (m, 2H), 0.88 (d, <i>J</i> = 4 Hz, 3H), 0.86 (d, <i>J</i> = 4 Hz, 3H); MS (ESI+) for C<sub>25</sub>H<sub>33</sub>ClN<sub>4</sub>O<sub>5</sub><i>m</i>/<i>z</i> 505.2 (M + H)<sup>+</sup>; HRMS (ESI+) calcd for C<sub>25</sub>H<sub>33</sub>ClN<sub>4</sub>O<sub>5</sub>+H1 505.2212, found 505.2204.</div></div><div id="sec4_1_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-Hydroxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-4-methylpentyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide (<b>40</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-hydroxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3-methylbutyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide but substituting <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-4-methylpentyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide and making noncritical variations provided a crude yellow foam. This material was purified by Biotage MPLC (25 M column, 5–5.5% methanol/chloroform) to afford 135 mg (50%) of the title compound as a white foam. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 11.57 (s, 1H), 8.42 (d, <i>J</i> = 8 Hz, 1H), 8.38 (d, <i>J</i> = 8 Hz, 1H), 7.62 (s, 1H), 7.35 (s, 1H), 7.09 (t, <i>J</i> = 8 Hz, 1H), 6.99 (d, <i>J</i> = 8 Hz, 1H), 6.49 (d, <i>J</i> = 8 Hz, 1H), 5.06 (t, <i>J</i> = 8 Hz, 1H), 4.47–4.30 (m, 2H), 4.25 (dd, <i>J</i> = 8, 20 Hz, 1H), 4.14 (dd, <i>J</i> = 8, 20 Hz, 1H), 3.87 (s, 3H), 3.09 (m, 2H), 2.30 (m, 1H), 2.08 (m, 1H), 1.92 (m, 1H), 1.72–1.51 (m, 5H), 1.25 (m, 2H), 0.87 (d, <i>J</i> = 4 Hz, 3H), 0.86 (d, <i>J</i> = 4 Hz, 3H); MS (ESI+) for C<sub>25</sub>H<sub>34</sub>N<sub>4</sub>O<sub>6</sub><i>m</i>/<i>z</i> 487.1 (M + H)<sup>+</sup>; HRMS (ESI+) calcd for C<sub>25</sub>H<sub>34</sub>N<sub>4</sub>O<sub>6</sub> 487.2551, found 487.2541.</div></div><div id="sec4_1_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> <i>tert</i>-Butyl [(1<i>S</i>)-2-[((1<i>S</i>)-3-(Benzyloxy)-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]-1-(cyclohexylmethyl)-2-oxoethyl]carbamate</h4><div class="NLM_p last">To a solution of <i>tert</i>-butyl ((1<i>S</i>)-3-(benzyloxy)-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)carbamate (200 mg, 0.53 mmol) in dioxane (5 mL) was added 4 M hydrochloric acid/dioxane (5 mL), and the solution was stirred at an ambient temperature for 4 h before removing the solvents <i>in vacuo</i>, azeotroping the residue with toluene (2 × 10 mL), and drying <i>in vacuo</i> for 1 h. The crude hydrochloride salt was taken into DMF (3 mL), and the solution was cooled to 0 °C before adding <i>N</i>-Boc-cyclohexylalanine-OH (139 mg, 0.53 mmol), collidine (156 μL, 1.22 mmol), and HATU (194 mg, 0.53 mmol) in order, and the resulting suspension was stirred at 0 °C for 5 h. The reaction was quenched by the addition of water (30 mL), and the mixture was extracted with diethyl ether (3 × 75 mL). The combined organics were dried over MgSO<sub>4</sub>, filtered, and the solvents were removed <i>in vacuo</i> to yield the crude product, which was purified by flash chromatography, eluting with 1–3% methanol/dichloromethane to afford the title compound as a pale brown gum, 205 mg, 76%. The product was contaminated with ∼20% of another diastereoisomer. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.69 (d, <i>J</i> = 5.6 Hz, 1H), 7.26–7.42 (m, 5H), 5.93 (s, 1H), 4.94 (d, <i>J</i> = 7.6 Hz, 1H), 4.72 (m, 1H), 4.60 (d, <i>J</i> = 11.6 Hz, 1H), 4.55 (d, <i>J</i> = 11.6 Hz, 1H), 4.14–4.34 (m, 3H), 3.27 (m, 2H), 2.23–2.51 (m, 3H), 1.56–2.05 (m, 9H), 1.33–1.48 (m, 9H), 1.05–1.29 (m, 4H), 0.80–1.03 (m, 2H).</div></div><div id="sec4_1_50" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> <i>N</i>-[(1<i>S</i>)-2-[((1<i>S</i>)-3-(Benzyloxy)-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]-1-(cyclohexylmethyl)-2-oxoethyl]-4-methoxy-1<i>H</i>-indole-2-carboxamide</h4><div class="NLM_p last">To a solution of <i>tert</i>-butyl [(1<i>S</i>)-2-[((1<i>S</i>)-3-(benzyloxy)-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]-1-(cyclohexylmethyl)-2-oxoethyl]carbamate (200 mg, 0.38 mmol) in dioxane (4 mL) was added 4 M hydrochloric acid/dioxane (4 mL), and the solution was stirred at an ambient temperature for 2 h before removing the solvents <i>in vacuo</i>, azeotroping the residue with toluene (2 × 10 mL), and drying <i>in vacuo</i> for 1 h. The crude hydrochloride salt was taken into DMF (3 mL), and the solution was cooled to 0 °C before adding 4-methoxy-indole-2-carboxylic acid (73 mg, 0.38 mmol), collidine (125 μL, 0.95 mmol), and HATU (144 mg, 0.38 mmol) in order, and the resulting suspension was stirred at 0 °C for 6 h. The reaction was quenched by the addition of water (20 mL), and the mixture was extracted with diethyl ether (3 × 50 mL). The combined organics were washed with water (20 mL) and brine (20 mL), dried over MgSO<sub>4</sub>, filtered, and the solvents were removed <i>in vacuo</i> to yield the crude product, which was purified by Biotage flash chromatography, eluting with 1–3% methanol/dichloromethane to afford the title compound as a pale brown gum, 90 mg, 39%. The product was contaminated with ∼20% of another diastereomer from the previous step. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 9.59 (s, 1H), 8.13 (d, <i>J</i> = 6.6 Hz, 1H), 7.21–7.32 (m, 5H), 7.11 (t, <i>J</i> = 8.0 Hz, 1H), 7.02 (d, <i>J</i> = 1.5 Hz, 1H), 6.94 (d, <i>J</i> = 8.3 Hz, 1H), 6.72–6.78 (m, 1H), 6.42 (d, <i>J</i> = 7.8 Hz, 1H), 5.97 (s, 1H), 4.73 (m, 1H), 4.64 (m, 1H), 4.53 (d, <i>J</i> = 11.6 Hz, 1H), 4.46 (d, <i>J</i> = 11.6 Hz, 1H), 4.26 (d, <i>J</i> = 17.2 Hz, 1H), 4.15 (d, <i>J</i> = 17.4 Hz, 1H), 3.87 (s, 3H), 3.08–3.16 (m, 2H), 2.27–2.38 (m, 1H), 1.92 (m, 1H), 1.49–1.84 (m, 9H), 1.26–1.42 (m, 1H), 0.98–1.23 (m, 4H), 0.78–0.98 (m, 2H); MS (APCI+) for C<sub>34</sub>H<sub>42</sub>N<sub>4</sub>O<sub>6</sub><i>m</i>/<i>z</i> 603.2 (M + H)<sup>+</sup>.</div></div><div id="sec4_1_51" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> <i>N</i>-{(1<i>S</i>)-1-(Cyclohexylmethyl)-2-[((1<i>S</i>)-3-hydroxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]-2-oxoethyl}-4-methoxy-1<i>H</i>-indole-2-carboxamide (<b>41</b>)</h4><div class="NLM_p last">To a solution of <i>N</i>-[(1<i>S</i>)-2-[((1<i>S</i>)-3-(benzyloxy)-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]-1-(cyclohexylmethyl)-2-oxoethyl]-4-methoxy-1<i>H</i>-indole-2-carboxamide (80 mg, 0.13 mmol) in EtOH (3 mL) was added 10% Pd/C (50 mg), and the suspension was hydrogenated at an ambient temperature under H<sub>2</sub> (1 atm balloon) for 5 h. The catalyst was removed by filtration, and the solvents were evaporated <i>in vacuo</i> to yield the crude product, which was purified by Biotage flash chromatography, eluting with 2–10% methanol/dichloromethane to afford the title compound, 37 mg, 55% as a white solid. <sup>1</sup>H NMR (400 MHz, MeOD) δ 7.19 (s, 1H), 7.05 (t, <i>J</i> = 8.0 Hz, 1H), 6.93 (d, <i>J</i> = 8.1 Hz, 1H), 6.41 (d, <i>J</i> = 7.8 Hz, 1H), 4.52–4.59 (m, 2H), 4.27 (m, 2H), 3.83 (s, 3H), 3.08–3.21 (m, 2H), 2.47 (m, 1H), 2.13–2.24 (m, 1H), 1.92–2.03 (m, 1H), 1.53–1.79 (m, 9H), 1.31–1.45 (m, 1H), 1.04–1.29 (m, 3H), 0.81–1.02 (m, 2H); MS (APCI+) for C<sub>27</sub>H<sub>36</sub>N<sub>4</sub>O<sub>6</sub><i>m</i>/<i>z</i> 513.2. (M + H)<sup>+</sup>; anal. calcd for C<sub>27</sub>H<sub>36</sub>N<sub>4</sub>O<sub>6</sub><b>·</b> 0.8 H<sub>2</sub>O: C, 61.53; H, 7.19; N, 10.63. Found: C, 61.83; H, 7.12; N, 10.27.</div></div><div id="sec4_1_52" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> <i>N</i>-(<i>tert</i>-Butoxycarbonyl)-<i>N</i>-((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-<span class="smallcaps smallerCapital">l</span>-phenylalaninamide</h4><div class="NLM_p last">Following the procedure described for the preparation of <i>N</i><sup>2</sup>-(<i>tert</i>-butoxycarbonyl)-<i>N</i><sup>1</sup>-((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-<span class="smallcaps smallerCapital">l</span>-leucinamide but substituting Boc-Phe-OH and making noncritical variations provided a crude brown oil. This material was purified by Biotage flash chromatography, eluting with methanol/dichloromethane to afford the title compound as a white solid, 351 mg, 36%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.02 (s, 1H), 7.19–7.31 (m, 5H), 5.77 (s, 1H), 5.10 (d, <i>J</i> = 6.1 Hz, 1H), 4.52–4.57 (m, 1H), 4.44 (m, 1H), 4.01–4.12 (m, 2H), 3.26–3.37 (m, 2H), 3.01–3.08 (m, 2H), 2.29–2.37 (m, 1H), 2.15–2.25 (m, 1H), 1.98–2.06 (m, 1H), 1.75–1.91 (m, 2H), 1.40 (s, 9H); MS (API-ES−) for C<sub>22</sub>H<sub>30</sub>N<sub>3</sub>O<sub>5</sub>Cl <i>m</i>/<i>z</i> 450.2 (M – H)<sup>−</sup>.</div></div><div id="sec4_1_53" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> <i>N</i>-((1<i>S</i>)-3-Chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-<i>N</i>-[(4-methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-phenylalaninamide</h4><div class="NLM_p last">Following the procedure described for the preparation of <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3-methylbutyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide but substituting <i>N</i>-(<i>tert</i>-butoxycarbonyl)-<i>N</i>-((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-<span class="smallcaps smallerCapital">l</span>-phenylalaninamide and making noncritical variations provided a crude green solid. This material was purified by Biotage flash chromatography, eluting with methanol/dichloromethane to afford the title compound as a white solid, 300 mg, 91%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.52 (d, <i>J</i> = 2.0 Hz, 1H), 8.68 (d, <i>J</i> = 7.8 Hz, 1H), 8.61 (d, <i>J</i> = 7.8 Hz, 1H), 7.61 (s, 1H), 7.37 (d, <i>J</i> = 7.1 Hz, 2H), 7.31 (d, <i>J</i> = 1.8 Hz, 1H), 7.26 (t, <i>J</i> = 7.6 Hz, 2H), 7.16 (t, <i>J</i> = 7.2 Hz, 1H), 7.07 (t, 1H), 6.97 (d, <i>J</i> = 8.1 Hz, 1H), 6.49 (d, <i>J</i> = 7.6 Hz, 1H), 4.68 (m, 1H), 4.38–4.49 (m, 3H), 3.88 (s, 3H), 2.97–3.17 (m, 4H), 2.20–2.32 (m, 1H), 2.02–2.14 (m, 1H), 1.93–2.02 (m, 1H), 1.52–1.69 (m, 2H); MS (APCI±) for C<sub>27</sub>H<sub>29</sub>N<sub>4</sub>O<sub>5</sub>Cl <i>m</i>/<i>z</i> 526.0 (M + H)<sup>−</sup>.</div></div><div id="sec4_1_54" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> <i>N</i>-((1<i>S</i>)-3-Hydroxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-<i>N</i>-[(4-methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-phenylalaninamide (<b>42</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-hydroxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3-methylbutyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide but substituting <i>N</i>-((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-<i>N</i>-[(4-methoxy-1<i>H</i>-indol-2-yl)carbonyl]-<span class="smallcaps smallerCapital">l</span>-phenylalaninamide and making noncritical variations provided a crude greenish gum. This material was purified by Biotage flash chromatography, eluting with methanol/dichloromethane to afford the title compound as an off-white solid, 123 mg, 44%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.50 (d, <i>J</i> = 2.0 Hz, 1H), 8.58 (dd, <i>J</i> = 8.2, 3.9 Hz, 2H), 7.63 (s, 1H), 7.35–7.43 (m, 2H), 7.31 (d, <i>J</i> = 1.8 Hz, 1H), 7.27 (t, <i>J</i> = 7.6 Hz, 2H), 7.17 (t, <i>J</i> = 7.3 Hz, 1H), 7.08 (t, <i>J</i> = 8.0 Hz, 1H), 6.98 (d, <i>J</i> = 8.1 Hz, 1H), 6.48 (d, <i>J</i> = 7.8 Hz, 1H), 5.06 (t, <i>J</i> = 6.1 Hz, 1H), 4.72 (m, 1H), 4.48 (m, 1H), 4.16 (m, 2H), 3.89 (s, 3H), 2.97–3.18 (m, 4H), 2.24–2.36 (m, 1H), 2.04–2.18 (m, 1H), 1.88–2.01 (m, 1H), 1.55–1.76 (m, 2H); MS (APCI+) for C<sub>27</sub>H<sub>30</sub>N<sub>4</sub>O<sub>6</sub><i>m</i>/<i>z</i> 507.1 (M + H)<sup>+</sup>; anal. calcd for C<sub>27</sub>H<sub>30</sub>N<sub>4</sub>O<sub>6</sub><b>·</b> 1.25 H<sub>2</sub>O: C, 61.29; H, 6.19; N, 10.59. Found: C, 61.37; H, 6.06; N, 10.49.</div></div><div id="sec4_1_55" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-Chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3,3-dimethylbutyl)-1<i>H</i>-indole-2-carboxamide</h4><div class="NLM_p last">Following the procedure described for the preparation of <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3-methylbutyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide but substituting <i>N</i><sup>2</sup>-(<i>tert</i>-butoxycarbonyl)-<i>N</i><sup>1</sup>-((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-4-methyl-<span class="smallcaps smallerCapital">l</span>-leucinamide and indole-2-carboxylic acid and making no other critical variations provided a crude yellow foam. This material was purified by Biotage MPLC (40 M column, 2.5–3.5% methanol/chloroform) to afford 1.06 g (62%) of the title compound as a light yellow foam. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 11.59 (s, 1H), 8.58 (d, <i>J</i> = 8 Hz, 1H), 8.50 (d, <i>J</i> = 8 Hz, 1H), 7.64 (s, 1H), 7.61 (d, <i>J</i> = 8 Hz, 1H), 7.41 (d, <i>J</i> = 8 Hz, 1H), 7.23 (s, 1H), 7.17 (t, <i>J</i> = 8 Hz, 1H), 7.02 (t, <i>J</i> = 8 Hz, 1H), 4.62–4.50 (m, 3H), 4.45 (m, 1H), 3.10 (m, 2H), 2.25 (m, 1H), 2.08 (m, 1H), 1.96 (m, 1H), 1.80 (m, 1H), 1.72–1.58 (m, 3H), 0.94 (s, 9H); MS (ESI+) for C<sub>24</sub>H<sub>31</sub>ClN<sub>4</sub>O<sub>4</sub><i>m</i>/<i>z</i> 475.1 (M + H)<sup>+</sup>; anal. calcd for C<sub>24</sub>H<sub>31</sub>ClN<sub>4</sub>O<sub>4</sub>·0.35 CHCl<sub>3</sub>: C, 56.59; H, 6.12; N, 10.84. Found: C, 56.38; H, 6.18; N, 10.75; HRMS (ESI+) calcd for C<sub>24</sub>H<sub>31</sub>ClN<sub>4</sub>O<sub>4</sub> 475.2107, found 475.2122.</div></div><div id="sec4_1_56" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-Hydroxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3,3-dimethylbutyl)-1<i>H</i>-indole-2-carboxamide (<b>43</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-hydroxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3-methylbutyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide but substituting <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3,3-dimethylbutyl)-1<i>H</i>-indole-2-carboxamide and making noncritical variations provided a crude yellow foam. This material was purified by Biotage MPLC (40 M column, 4.5–5.5% methanol/chloroform) to afford 730 mg (72%) of the title compound as a white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 11.59 (s, 1H), 8.49 (d, <i>J</i> = 8 Hz, 1H), 8.43 (d, <i>J</i> = 8 Hz, 1H), 7.62 (s, 1H), 7.60 (s, 1H), 7.41 (d, <i>J</i> = 8 Hz, 1H), 7.23 (s, 1H), 7.17 (t, <i>J</i> = 8 Hz, 1H), 7.02 (t, <i>J</i> = 8 Hz, 1H), 5.05 (t, <i>J</i> = 8 Hz, 1H), 4.56 (m, 1H), 4.43 (m, 1H), 4.25 (dd, <i>J</i> = 8, 20 Hz, 1H), 4.13 (dd, <i>J</i> = 8, 20 Hz, 1H), 3.10 (m, 2H), 2.25 (m, 1H), 2.07 (m, 1H), 1.93 (m, 1H), 1.80 (m, 1H), 1.64 (m, 3H), 0.94 (s, 9H); MS (ESI+) for C<sub>24</sub>H<sub>32</sub>N<sub>4</sub>O<sub>5</sub><i>m</i>/<i>z</i> 457.1 (M + H)<sup>+</sup>; anal. calcd for C<sub>24</sub>H<sub>32</sub>N<sub>4</sub>O<sub>5</sub>·0.2 CHCl<sub>3</sub>·0.2 ethyl acetate·0.25 H<sub>2</sub>O: C, 59.75; H, 6.88; N, 11.15. Found: C, 59.67; H, 6.72; N, 11.03; HRMS (ESI+) calcd for C<sub>24</sub>H<sub>32</sub>N<sub>4</sub>O<sub>5</sub> 457.2446, found 457.2439.</div></div><div id="sec4_1_57" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-Methoxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3,3-dimethylbutyl)-1<i>H</i>-indole-2-carboxamide (<b>44</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of 4-methoxy-<i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-methoxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-4-methylpentyl)-1<i>H</i>-indole-2-carboxamide but substituting <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-hydroxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3,3-dimethylbutyl)-1<i>H</i>-indole-2-carboxamide and making noncritical variations provided a crude tan foam. This material was purified by Biotage MPLC (25 M column, 3.5–4.5% methanol/chloroform) to afford 43 mg (15%) of the title compound as a white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 11.58 (s, 1H), 8.48 (apar t, <i>J</i> = 8 Hz, 2H), 7.61 (apar d, <i>J</i> = 8 Hz, 2H), 7.41 (d, <i>J</i> = 8 Hz, 1H), 7.23 (s, 1H), 7.17 (t, <i>J</i> = 8 Hz, 1H), 7.02 (t, <i>J</i> = 8 Hz, 1H), 4.55 (m, 1H), 4.37 (m, 1H), 4.25 (d, <i>J</i> = 20 Hz, 1H), 4.10 (d, <i>J</i> = 20 Hz, 1H), 3.23 (s, 3H), 3.06 (m, 2H), 2.27 (m, 1H), 2.07 (m, 1H), 1.91 (m, 1H), 1.79 (m, 1H), 1.69 (m, 3H), 0.94 (s, 9H); MS (ESI+) for C<sub>25</sub>H<sub>34</sub>N<sub>4</sub>O<sub>5</sub><i>m</i>/<i>z</i> 471.2 (M + H)<sup>+</sup>; anal. calcd for C<sub>25</sub>H<sub>34</sub>N<sub>4</sub>O<sub>5</sub>·0.2 ethyl acetate·0.75 H<sub>2</sub>O: C, 61.76; H, 7.46; N, 11.17. Found: C, 61.85; H, 7.15; N, 11.02; HRMS (ESI+) calcd for C<sub>25</sub>H<sub>34</sub>N<sub>4</sub>O<sub>5</sub> 471.2602, found 471.2595.</div></div><div id="sec4_1_58" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-Ethoxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3,3-dimethylbutyl)-1<i>H</i>-indole-2-carboxamide (<b>45</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-methoxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3,3-dimethylbutyl)-1<i>H</i>-indole-2-carboxamide but substituting iodoethane and making noncritical variations provided a crude tan foam. This material was purified by Biotage MPLC (25 M column, 3–5% methanol/chloroform) to afford 19 mg (6%) of the title compound as an off-white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 11.58 (s, 1H), 8.48 (d, <i>J</i> = 8 Hz, 1H), 8.45 (d, <i>J</i> = 8 Hz, 1H), 7.61 (apar d, <i>J</i> = 8 Hz, 2H), 7.41 (d, <i>J</i> = 8 Hz, 1H), 7.23 (s, 1H), 7.17 (t, <i>J</i> = 8 Hz, 1H), 7.02 (t, <i>J</i> = 8 Hz, 1H), 4.56 (m, 1H), 4.40 (m, 1H), 4.28 (d, <i>J</i> = 16 Hz, 1H), 4.13 (d, <i>J</i> = 16 Hz, 1H), 3.41 (m, 2H), 3.06 (m, 2H), 2.26 (m, 1H), 2.09 (m, 1H), 1.91 (m, 1H), 1.82 (m, 1H), 1.63 (m, 3H), 1.08 (t, <i>J</i> = 8 Hz, 3H), 0.94 (s, 9H); MS (ESI+) for C<sub>26</sub>H<sub>36</sub>N<sub>4</sub>O<sub>5</sub><i>m</i>/<i>z</i> 485.2 (M + H)<sup>+</sup>; HRMS (ESI+) calcd for C<sub>26</sub>H<sub>36</sub>N<sub>4</sub>O<sub>5</sub> 485.2759, found 485.2756.</div></div><div id="sec4_1_59" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> <i>N</i><sup>1</sup>-((1<i>S</i>)-3-Chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-4-methyl-<i>N</i><sup>2</sup>-[(2<i>R</i>)-tetrahydrofuran-2-ylcarbonyl]-<span class="smallcaps smallerCapital">l</span>-leucinamide</h4><div class="NLM_p last">Following the procedure described for the preparation of <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3-methylbutyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide but substituting <i>N</i><sup>2</sup>-(<i>tert</i>-butoxycarbonyl)-<i>N</i><sup>1</sup>-((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-4-methyl-<span class="smallcaps smallerCapital">l</span>-leucinamide and (2<i>R</i>)-tetrahydrofuran-2-carboxylic acid and making noncritical variations provided a crude product. This crude material was purified by Biotage MPLC (40 M cartridge, chloroform mobile phase with 2% methanol followed by 3% methanol, sample loaded in chloroform) resulting in the isolation of 1.13 g (61%) of the title compound as a light yellow foam. <i>R<sub>f</sub></i> = 0.27 (95:5 dichloromethane/methanol); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.47 (d, <i>J</i> = 8 Hz, 1H), 7.72 (d, <i>J</i> = 8 Hz, 1H), 7.68 (s, 1H), 4.55 (s, 2H), 4.44–4.36 (m, 1H), 4.35–4.27 (m, 1H), 4.21 (dd, <i>J</i> = 8, 5 Hz, 1H), 3.95–3.86 (m, 1H), 3.79–3.71 (m, 1H), 3.18–3.07 (m, 2H), 2.21 (td, <i>J</i> = 9, 4 Hz, 1H), 2.13–2.03 (m, 2H), 1.97–1.88 (m, 1H), 1.85–1.74 (m, 3H), 1.68–1.56 (m, 4H), 0.88 (s, 9H); MS (ESI+) for C<sub>20</sub>H<sub>32</sub>ClN<sub>3</sub>O<sub>5</sub><i>m</i>/<i>z</i> 430 (M + H).</div></div><div id="sec4_1_60" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> <i>N</i><sup>1</sup>-((1<i>S</i>)-3-Hydroxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-4-methyl-<i>N</i><sup>2</sup>-[(2<i>R</i>)-tetrahydrofuran-2-ylcarbonyl]-<span class="smallcaps smallerCapital">l</span>-leucinamide (<b>46</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-hydroxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3-methylbutyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide but substituting <i>N</i><sup>1</sup>-((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-4-methyl-<i>N</i><sup>2</sup>-[(2<i>R</i>)-tetrahydrofuran-2-ylcarbonyl]-<span class="smallcaps smallerCapital">l</span>-leucinamide and making noncritical variations provided a crude product. This material was purified by a series of two radial chromatographies (first −2 mm plate, 90:10 dichloromethane/methanol to 90:20, sample loaded in 90:10) (second −1 mm plates, 90:10 dichloromethane/methanol to 95:5, sample loaded in dichloromethane) to provide 0.402 g (46%) of the title compound as a light yellow foam. <i>R<sub>f</sub></i> = 0.44 (90:10 dichloromethane/methanol); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.27 (d, <i>J</i> = 8 Hz, 1H), 7.71 (d, <i>J</i> = 9 Hz, 1H), 7.65 (s, 1H), 5.11 (t, <i>J</i> = 6 Hz, 1H), 4.46–4.38 (m, 1H), 4.37–4.30 (m, 1H), 4.24–4.15 (m, 2H), 4.14–4.08 (m, 1H), 3.96–3.87 (m, 1H), 3.79–3.70 (m, 1H), 3.18–3.06 (m, 2H), 2.25–2.16 (m, 1H), 2.13–2.02 (m, 2H), 1.87–1.75 (m, 4H), 1.66–1.54 (m, 4H), 0.88 (s, 9H); MS (ESI+) for C<sub>20</sub>H<sub>33</sub>N<sub>3</sub>O<sub>6</sub><i>m</i>/<i>z</i> 412 (M + H). Anal. calcd for C<sub>20</sub>H<sub>33</sub>N<sub>3</sub>O<sub>6</sub>·0.5H<sub>2</sub>O: C, 57.12; H, 8.16; N, 9.99. Found: C, 57.25; H, 7.93; N, 9.68. HRMS (ESI+) calcd for C<sub>20</sub>H<sub>33</sub>N<sub>3</sub>O<sub>6</sub>+H 412.2442, found 412.2447.</div></div><div id="sec4_1_61" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> <i>N</i><sup>1</sup>-((1<i>S</i>)-3-Methoxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-4-methyl-<i>N</i><sup>2</sup>-[(2<i>R</i>)-tetrahydrofuran-2-ylcarbonyl]-<span class="smallcaps smallerCapital">l</span>-leucinamide (<b>47</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of 4-methoxy-<i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-methoxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-4-methylpentyl)-1<i>H</i>-indole-2-carboxamide but substituting <i>N</i><sup>1</sup>-((1<i>S</i>)-3-hydroxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)-4-methyl-<i>N</i><sup>2</sup>-[(2<i>R</i>)-tetrahydrofuran-2-ylcarbonyl]-<span class="smallcaps smallerCapital">l</span>-leucinamide and making noncritical variations provided a crude product. This material was purified by radial chromatography (1 mm plate, 95:5 dichloromethane/methanol, sample loaded in dichloromethane) resulting in the isolation of 45.6 mg (22%) of the title compound as a light orange gum and as a mixture of diastereomers. <i>R<sub>f</sub></i> = 0.30 (95:5 dichloromethane/methanol); <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, major diastereomer) δ 8.33 (d, <i>J</i> = 8 Hz, 1H), 7.70 (d, <i>J</i> = 9 Hz, 1H), 7.66 (s, 1H), 4.39–4.28 (m, 2H), 4.24–4.07 (m, 3H), 3.94–3.86 (m, 1H), 3.75 (q, <i>J</i> = 7 Hz, 1H), 3.24 (s, 3H), 3.18–3.06 (m, 2H), 2.27–2.15 (m, 1H), 2.14–2.03 (m, 2H), 1.92–1.73 (m, 4H), 1.67–1.53 (m, 4H), 0.88 (s, 9H); MS (ESI+) for C<sub>21</sub>H<sub>35</sub>N<sub>3</sub>O<sub>6</sub><i>m</i>/<i>z</i> 426 (M + H). HRMS (ESI+) calcd for C<sub>21</sub>H<sub>35</sub>N<sub>3</sub>O<sub>6</sub>+H1 426.2599, found 426.2604.</div></div><div id="sec4_1_62" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> 4-Methoxy-<i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-methoxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3-methylbutyl)-1<i>H</i>-indole-2-carboxamide (<b>48</b>)</h4><div class="NLM_p last">A solution of <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-hydroxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3-methylbutyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide (185 mg, 0.39 mmol), iodomethane (0.12 mL, 2.0 mmol), and silver(I) oxide (182 mg, 0.79 mmol) in dichloromethane (12 mL) was placed under an atmosphere of N<sub>2</sub>. The resulting black thick suspension was heated to reflux for 18 h, treated with a second portion of iodomethane (0.12 mL, 2.0 mmol), and returned to reflux for an additional 24 h. The reaction was cooled to RT, diluted with dichloromethane (20 mL), washed once with water (20 mL), once with brine (20 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated to give a crude tan solid. This material was purified by Biotage MPLC (25 M column, 3.5–4.5% methanol/chloroform) to afford 19 mg (10%) of the title compound as an off-white solid. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ 11.57 (s, 1H), 8.48 (d, <i>J</i> = 8 Hz, 1H), 8.40 (d, <i>J</i> = 8 Hz, 1H), 7.63 (s, 1H), 7.35 (s, 1H), 7.08 (t, <i>J</i> = 8 Hz, 1H), 6.99 (d, <i>J</i> = 8 Hz, 1H), 6.49 (d, <i>J</i> = 8 Hz, 1H), 4.47 (m, 1H), 4.38 (m, 1H), 4.25 (d, <i>J</i> = 16 Hz, 1H), 4.11 (d, <i>J</i> = 16 Hz, 1H), 3.87 (s, 3H), 3.23 (s, 3H), 3.08 (m, 2H), 2.31 (m, 1H), 2.08 (m, 1H), 1.92 (m, 1H), 1.72–1.51 (m, 5H), 0.93 (d, <i>J</i> = 8 Hz, 3H), 0.88 (d, <i>J</i> = 8 Hz, 3H); MS (ESI+) for C<sub>25</sub>H<sub>34</sub>N<sub>4</sub>O<sub>6</sub><i>m</i>/<i>z</i> 487.2 (M + H)<sup>+</sup>; HRMS (ESI+) calcd for C<sub>25</sub>H<sub>34</sub>N<sub>4</sub>O<sub>6</sub>+H1 487.2551, found 487.2563.</div></div><div id="sec4_1_63" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-Chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3-methylbutyl)-1<i>H</i>-indole-2-carboxamide</h4><div class="NLM_p last">Following the procedure described for the preparation of <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3-methylbutyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide but substituting indole-2-carboxylic acid and making noncritical variations provided a crude orange foam. This material was purified by Biotage flash chromatography, eluting with methanol/dichloromethane to afford the title compound as an off-white solid, 1.75 g, 82%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.58 (s, 1H), 8.64 (d, <i>J</i> = 7.8 Hz, 1H), 8.50 (d, <i>J</i> = 7.8 Hz, 1H), 7.55–7.69 (m, 2H), 7.42 (d, <i>J</i> = 8.3 Hz, 1H), 7.26 (d, <i>J</i> = 1.5 Hz, 1H), 7.16 (m, 1H), 7.02 (t, <i>J</i> = 7.5 Hz, 1H), 4.53–4.66 (m, 2H), 4.39–4.52 (m, 2H), 2.94–3.18 (m, 2H), 2.22–2.35 (m, 1H), 2.03–2.16 (m, 1H), 1.89–2.02 (m, 1H), 1.45–1.82 (m, 5H), 0.93 (d, <i>J</i> = 6.3 Hz, 3H), 0.88 (d, <i>J</i> = 6.3 Hz, 3H); MS (APCI−) for C<sub>23</sub>H<sub>29</sub>N<sub>4</sub>O<sub>4</sub>Cl <i>m</i>/<i>z</i> 459.1 (M – H)<sup>−</sup>.</div></div><div id="sec4_1_64" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-Hydroxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3-methylbutyl)-1<i>H</i>-indole-2-carboxamide (<b>49</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-hydroxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3-methylbutyl)-4-methoxy-1<i>H</i>-indole-2-carboxamide but substituting <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-chloro-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3-methylbutyl)-1<i>H</i>-indole-2-carboxamide and making noncritical variations provided a crude black gum. This material was purified by Biotage flash chromatography, eluting with methanol/dichloromethane to afford the title compound as an off-white solid, 218 mg, 18%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.58 (s, 1H), 8.42–8.61 (m, 2H), 7.61 (d, <i>J</i> = 8.6 Hz, 2H), 7.41 (d, <i>J</i> = 8.1 Hz, 1H), 7.26 (s, 1H), 7.17 (t, <i>J</i> = 7.6 Hz, 1H), 7.02 (t, <i>J</i> = 7.5 Hz, 1H), 5.02 (t, 1H), 4.38–4.58 (m, 2H), 4.09–4.32 (m, 2H), 3.01–3.17 (m, 2H), 2.24–2.38 (m, 1H), 2.03–2.22 (m, 2H), 1.92 (m, 1H), 1.46–1.77 (m, 4H), 0.85–0.99 (m, 6H); MS (APCI+) for C<sub>23</sub>H<sub>30</sub>N<sub>4</sub>O<sub>5</sub><i>m</i>/<i>z</i> 443.1 (M + H)<sup>+</sup>; anal. calcd for C<sub>23</sub>H<sub>30</sub>N<sub>4</sub>O<sub>5</sub>·0.55 H<sub>2</sub>O·0.05 DCM: C, 60.62; H, 6.89; N, 12.27. Found: C, 60.83; H, 6.90; N,11.93.</div></div><div id="sec4_1_65" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-Methoxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3-methylbutyl)-1<i>H</i>-indole-2-carboxamide (<b>50</b>)</h4><div class="NLM_p last">Following the procedure described for the preparation of 4-methoxy-<i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-methoxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-4-methylpentyl)-1<i>H</i>-indole-2-carboxamide but substituting <i>N</i>-((1<i>S</i>)-1-{[((1<i>S</i>)-3-hydroxy-2-oxo-1-{[(3<i>S</i>)-2-oxopyrrolidin-3-yl]methyl}propyl)amino]carbonyl}-3-methylbutyl)-1<i>H</i>-indole-2-carboxamide and making noncritical variations provided a crude pale brown oil. This material was purified by Biotage flash chromatography, eluting with methanol/dichloromethane to afford the title compound as a white foam, 15 mg, 7%. <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 9.52 (s, 1H), 8.38 (d, <i>J</i> = 6.1 Hz, 1H), 7.62 (d, <i>J</i> = 8.1 Hz, 1H), 7.40 (dd, <i>J</i> = 8.1, 3.8 Hz, 1H), 7.20–7.29 (m, 1H), 7.11 (t, <i>J</i> = 7.5 Hz, 1H), 6.98 (dd, <i>J</i> = 11.4, 1.5 Hz, 1H), 6.87 (d, <i>J</i> = 8.3 Hz, 1H), 5.98 (d, <i>J</i> = 13.1 Hz, 1H), 4.71–4.88 (m, 1H), 4.63 (m, 1H), 4.08–4.33 (m, 2H), 3.38–3.44 (s, 3H), 3.08–3.33 (m, 2H), 2.28–2.63 (m, 2H), 2.13–2.25 (m, 1H), 1.55–2.10 (m, 4H), 0.85–1.03 (m, 6H); MS (APCI−) for C<sub>24</sub>H<sub>32</sub>N<sub>4</sub>O<sub>5</sub><i>m</i>/<i>z</i> 455.2 (M – H)<sup>−</sup>.</div></div></div><div id="sec4_1_65_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i85" class="anchor-spacer"></div><h3 class="article-section__title" id="_i85"> SARS CoV-1 Protease FRET Assay and Analysis</h3><div class="NLM_p last">Proteolytic activity of coronavirus 3CL protease is measured using a continuous fluorescence resonance energy-transfer assay. The SARS 3CL<sup>pro</sup> FRET assay measures the protease-catalyzed cleavage of TAMRA-SITSAVLQSGFRKMK-(DABCYL)-OH to TAMRA-SITSAVLQ and SGFRKMK(DABCYL)-OH. The fluorescence of the cleaved TAMRA (ex. 558 nm/em. 581 nm) peptide was measured using a TECAN SAFIRE fluorescence plate reader over the course of 10 min. Typical reaction solutions contained 20 mM HEPES (pH 7.0), 1 mM EDTA, 4.0 FRET substrate, 4% DMSO, and 0.005% Tween-20. Assays were initiated with the addition of 25 nM SARS CoV-1 3CL<sup>pro</sup> nucleotide sequence 9985–10902 of the Urbani strain of SARS coronavirus complete genome sequence (NCBI accession number AY278741). Percent inhibition was determined in duplicate at a 0.001 mM level of inhibitor. Data was analyzed with the nonlinear regression analysis program KaleidaGraph using the equation<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/medium/jm0c01063_m001.gif" alt="" id="_i86" /></img></span>where offset equals the fluorescence signal of the uncleaved peptide substrate, and limit equals the fluorescence of the fully cleaved peptide substrate. The <i>k</i><sub>obs</sub> is the first-order rate constant for this reaction and, in the absence of any inhibitor, represents the utilization of a substrate. In an enzyme start reaction, which contains an irreversible inhibitor, and where the calculated limit is less than 20% of the theoretical maximum limit, the calculated <i>k</i><sub>obs</sub> represents the rate of inactivation of coronavirus 3CL protease. The slope (<i>k</i><sub>obs</sub>/<i>I</i>) of a plot of <i>k</i><sub>obs</sub> vs [<i>I</i>] is a measure of the avidity of the inhibitor for an enzyme. For very fast irreversible inhibitors, <i>k</i><sub>obs</sub>/<i>I</i> is calculated from observations at only one or two [<i>I</i>] rather than as a slope. For the determination of IC<sub>50</sub> values for reversible inhibitors, the data was analyzed with the nonlinear regression analysis programs Xlfit3.05 (IDBS, Guildford, U.K.) or GraphPad Prism 8.02 (GraphPad Software San Diego, CA).</div></div><div id="sec4_1_65_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i87" class="anchor-spacer"></div><h3 class="article-section__title" id="_i87"> SARS CoV-2 Protease FRET Assay and Analysis</h3><div class="NLM_p last">The proteolytic activity of the main protease, 3CL<sup>pro</sup>, of SARS CoV-2 was monitored using a continuous fluorescence resonance energy-transfer (FRET) assay. The SARS CoV-2 3CL<sup>pro</sup> assay measures the activity of full-length SARS CoV-2 3CL protease to cleave a synthetic fluorogenic substrate peptide with the following sequence DABCYL-KTSAVLQ-SGFRKME-EDANS modeled on a consensus peptide.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> The fluorescence of the cleaved EDANS peptide (excitation 340 nm/emission 490 nm) is measured using a fluorescence intensity protocol on a Flexstation reader (Molecular Devices). The fluorescent signal is reduced in the presence of PF-00835231, a potent inhibitor of SARS CoV-2 3CL<sup>pro</sup>. The assay reaction buffer contained 20 mM Tris–HCl (pH 7.3), 100 nM NaCl, 1 mM EDTA, 5 mM TCEP, and 25 μM peptide substrate. Enzyme reactions were initiated with the addition of 15 nM SARS CoV-2 3CL protease and allowed to proceed for 60 min at 23 °C. Percent inhibition or activity was calculated based on control wells containing no compound (0% inhibition/100% activity) and a control compound (100% inhibition/0% activity). IC<sub>50</sub> values were generated using a four-parameter fit model using ABASE software (IDBS). <i>K</i><sub>i</sub> values were fit to the Morrison equation with the enzyme concentration parameter fixed to 15 nM, the <i>K</i><sub>m</sub> parameter fixed to 14 μM, and the substrate concentration parameter fixed to 25 μM using Activity Base software (IDBS).</div></div><div id="sec4_1_65_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i88" class="anchor-spacer"></div><h3 class="article-section__title" id="_i88"> SARS CoV-1 Antiviral Assays</h3><div class="NLM_p last">All assays were performed in BSL3 containment. Vero 76 cells were plated at 10 000 cells per well using phenol red-free DMEM or IMEM (Gibco). After adhering for 2 h at 37 °C, various concentrations of compound (320, 100, 33, 10, 3.3, 1.0, 0.3, or 0.1 μM) were added and the cells were infected with 2.6 × 10<sup>3</sup> PFU/well SARS CoV Toronto-2 (provided as a gift from Dr. Heinz Feldman (NIAID, Hamilton, MT)) or mock-infected with the medium only. After 66 h, cell viability was determined using either the neutral red method or the CellTiter-Glo Luminescent Cell Viability Assay (Promega, Madison, WI). For the neutral red method, the cells were washed twice with PBS, and 100 μL of DMEM pH 4.5 containing 0.066% neutral red was added to the cells for 2 h at 37 °C. The cells were again washed twice with PBS, 100 μL of buffer solution (50% EtOH, 1% acetic acid) was added, and the OD was read at 540 nM after a 10 min incubation at 37 °C with shaking. Data are expressed as the percent of neutral red or luminescent signal in wells of compound-treated cells compared to the signal in wells of uninfected, compound-free cells. The 50% effective concentration (EC<sub>50</sub>) is calculated as the concentration of the compound that increases the percent of the neutral red or luminescent signal in infected, compound-treated cells to 50% of that produced by uninfected, compound-free cells. The 50% cytotoxicity concentration (CC<sub>50</sub>) is calculated as the concentration of the compound that decreases the percent of the neutral red or luminescent signal in uninfected, compound-treated cells to 50% of that produced in uninfected, compound-free cells.<a onclick="showRef(event, 'ref51 ref52'); return false;" href="javascript:void(0);" class="ref ref51 ref52">(51,52)</a></div></div><div id="sec4_1_65_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i89" class="anchor-spacer"></div><h3 class="article-section__title" id="_i89"> hCOV 229E Antiviral assays</h3><div class="NLM_p last">MRC-5 cells were plated at 10 000 cells per well using phenol red-free MEM (Gibco). Various concentrations of compound (320, 100, 33, 10, 3.3, 1.0, 0.3, or 0.1 μM) were added, and the cells were infected with an amount of hCoV 229E, which caused 80% cell death in 4 days or mock-infected with the medium only. After 4 days, cell viability was determined using the XTT dye reduction method.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> Data are expressed as the percent of neutral red or luminescent signal in wells of compound-treated cells compared to the signal in wells of uninfected, compound-free cells. The 50% effective concentration (EC<sub>50</sub>) is calculated as the concentration of the compound that increases the percent of the neutral red or luminescent signal in infected, compound-treated cells to 50% of that produced by uninfected, compound-free cells. The 50% cytotoxicity concentration (CC<sub>50</sub>) is calculated as the concentration of the compound that decreases the percent of the neutral red or luminescent signal in uninfected, compound-treated cells to 50% of that produced in uninfected, compound-free cells.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i90" class="anchor-spacer"></div><h3 class="article-section__title" id="_i90"> Other Antiviral Assays</h3><div class="NLM_p last">The following antiviral assays for HCMV,<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> HIV-RF,<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> HRV-14, and HRV-16,<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> as well as the HCV replicon,<a onclick="showRef(event, 'ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref56 ref57">(56,57)</a> were performed as described in the above references.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i91" class="anchor-spacer"></div><h3 class="article-section__title" id="_i91"> Expression and Purification of CoV-2 3CL Protease</h3><div class="NLM_p last">The expression and purification strategy followed previous studies on SARS CoV-1 3CL<sup>pro</sup>.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> Genes encoding SARS CoV-1 3CL<sup>pro</sup> and SARS CoV-2 M<sup>pro</sup> MN908947.3 were synthesized with codon usage optimized for <i>Escherichia coli</i> expression (Genscript and IDT). Genes were cloned into a modified pET24a vector to produce a TEV-cleavable N-terminal 6xHis tag under the T7 promoter control. BL21(DE3) cells (LifeTech) were grown in Terrific Broth (Teknova) at 37 °C until an OD<sub>600</sub> = 0.6–0.8 and induced with 0.4 M IPTG for 5 h at 30 °C. Cell pellets were stored at −80 °C until purification. Cell pellets were resuspended and lysed by microfluidization in Buffer A (20 mM Tris pH 8.0 + 150 mM NaCl). Lysates were clarified at 25,000 × g for 1 h at 4 °C. The soluble lysate was loaded onto a nickel affinity column (Probond) and washed sequentially with Buffer A + 5 mM and 20 mM imidazole. The proteases were eluted with Buffer A + 300 mM imidazole. The His-tag was removed by TEV protease (1:40) during an overnight dialysis step in Buffer A + 1 mM DTT. The CoV M<sup>pro</sup> proteins were further purified on a HiTrap Q hp (GE Healthcare), and the Q-flow-through material was concentrated and loaded onto a 26/60 Superdex-75 (GE Healthcare) gel filtration column equilibrated with Buffer B (20 mM Tris pH 7.8 + 150 mM NaCl + 1 mM EDTA + 1 mM DTT). Final proteins were concentrated from 5 to 25 mg/mL and were directly moved to crystallization experiments or snap-frozen in liquid nitrogen for storage.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i92" class="anchor-spacer"></div><h3 class="article-section__title" id="_i92"> Crystallization/Soak Protocols for SARS1 with <b>2</b></h3><div class="NLM_p last">Crystals of compound <b>2</b> bound to SARS1/CoV-1 DC2 protease were produced <i>via</i> cocrystallization. SARS1/CoV-1 DC2 protease at 10.00 mg/mL was incubated with a 3-fold molar excess of compound <b>2</b> for 18 h at 4 °C. The complex was then passed through a 0.45 μM cellulose acetate spin filter and set up for crystallization using an NT-8 crystallization robot (Formulatrix). Using MRC-2 crystallization plates, wells containing 40 μL of 4% w/v PEG 6000, 0.1 M MES pH 6.0, and 2.5 mM DTT were dispensed, and then sitting drops consisting of 0.3 μL protein were set up against a 0.3 μL well buffer. Crystallization plates were incubated at 21 °C, and rectangular crystals measuring 0.1 × 0.25 × 0.25 mm<sup>3</sup> grew within 24 h. Cocrystals were flash-frozen in liquid nitrogen after being passed through a cryo consisting of a well buffer containing 20% glycerol.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i93" class="anchor-spacer"></div><h3 class="article-section__title" id="_i93"> Crystallization/Soak Protocols for SARS1 with <b>28</b></h3><div class="NLM_p last">Crystals of compound 28 bound to SARS1/CoV-1 DC2 protease were produced <i>via</i> cocrystallization. SARS1/CoV-1 DC2 protease at 10.00 mg/mL was incubated with a 3-fold molar excess of compound 28 for 18 h at 4 °C. The complex was then passed through a 0.45 μM cellulose acetate spin filter and set up for crystallization using an NT-8 crystallization robot (Formulatrix). Using MRC-2 crystallization plates, wells containing 40 μL of 10% w/v PEG 8000, 0.2 M sodium chloride, 0.1 M sodium potassium phosphate pH 6.2, and 10 mM TCEP were dispensed, and then sitting drops consisting of 0.3 μL protein were set up against a 0.3 μL well buffer. Crystallization plates were incubated at 21 °C, and rectangular crystals measuring 0.1 × 0.25 × 0.25 mm<sup>3</sup> grew within 24 h. Cocrystals were flash-frozen in liquid nitrogen after being passed through a cryo consisting of a well buffer containing 20% glycerol.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i94" class="anchor-spacer"></div><h3 class="article-section__title" id="_i94"> Crystallization/Soak Protocols for SARS1 and SARS2 with <b>4</b> (PF-00835231)</h3><div id="sec4_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i95" class="anchor-spacer"></div><h4 class="article-section__title" id="_i95"> LJEC2520 + <b>PF-00835231</b></h4><div class="NLM_p last">Crystals of <b>PF-00835231</b> bound to SARS1/CoV-1 DC2 protease <i>via</i> cocrystallization: Freshly prepared SARS1/CoV-1 DC2 protease at 10.00 mg/mL was incubated with a 3-fold molar excess of <b>PF-00835231</b> for 18 h at 4 °C. The complex was then passed through a 0.45 μM cellulose acetate spin filter and set up for crystallization using an NT-8 crystallization robot (Formulatrix). Using MRC-2 crystallization plates, wells containing 40 μL of 10% w/v PEG 6000, 0.1 M MES pH 6.0, and 10% glycerol were dispensed, and then sitting drops consisting of 0.15 μL protein were set up against a 0.15 μL well buffer. Crystallization plates were incubated at 21 °C, and rectangular crystals measuring 0.2 × 0.25 × 0.15 mm<sup>3</sup> grew overnight. Crystals were flash-frozen in liquid nitrogen after being passed through a cryo consisting of a well buffer containing 20% glycerol.</div></div><div id="sec4_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i96" class="anchor-spacer"></div><h4 class="article-section__title" id="_i96"> LJEC2521 + <b>PF-00835231</b></h4><div class="NLM_p last">Crystals of <b>PF-00835231</b> bound to SARS2/CoV-2 mature protease <i>via</i> cocrystallization: Freshly prepared SARS2/CoV-2 mature protease at 11.3 mg/mL was incubated with a 3-fold molar excess of <b>PF-00835231</b> for 18 h at 4 °C. The complex was then passed through a 0.45 μM cellulose acetate spin filter and set up for crystallization using an NT-8 crystallization robot (Formulatrix). Using MRC-2 crystallization plates, wells containing 40 μL of 15% v/v 2-propanol, 0.1 M citric acid pH 5.0, and 10% w/v PEG 10 000 were dispensed, and then sitting drops consisting of 0.3 μL protein and a 0.3 μL well buffer were set up. Crystallization plates were incubated at 13 °C, and crystals shaped as rectangular blocks measuring 0.1 × 0.2 × 0.2 mm<sup>3</sup> grew overnight. Crystals were flash-frozen in liquid nitrogen after being passed through a cryo consisting of a well buffer containing 20% glycerol.</div></div><div id="sec4_6_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i97" class="anchor-spacer"></div><h4 class="article-section__title" id="_i97"> LJEC2522 + <b>PF-00835231</b></h4><div class="NLM_p">Crystals of <b>PF-00835231</b> bound to SARS2/CoV-2 DC2 protease <i>via</i> soaking: Freshly prepared SARS2/CoV-2 DC2 protease at 25.8 mg/mL was passed through a 0.45 mM cellulose acetate spin filter and set up for crystallization using an NT-8 crystallization robot (Formulatrix). Using MRC-2 crystallization plates, wells containing 40 mL of 0.2 M potassium sodium tartrate tetrahydrate and 20% w/v PEG 3350 were dispensed, and then sitting drops consisting of 0.3 μL protein were set up in a 1:1 ratio with a well buffer. Crystallization plates were incubated at 13 °C, and large crystals shaped as beveled plates measuring 0.35 × 0.2 × 0.025 mm<sup>3</sup> grew after 48 h. <b>PF-00835231</b> (in 100% DMSO solution) was introduced to the drop (in situ), at a final concentration of 1 mM, and then incubated at 13 °C for 24 h. Soaked crystals were flash-frozen in liquid nitrogen after being passed through a cryo consisting of a well buffer containing 20% glycerol. Structure determination.</div><div class="NLM_p">X-ray diffraction data were collected at the IMCA-CAT 17-ID beamline of Advanced Photon Source at Argonne National Labs and processed using autoPROC. Structure of the SARS CoV main protease in complex with <b>PF-835231</b> was determined by molecular replacement using the published protein structure (PDBID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1Q2W">1Q2W</a>) as the starting model in program Phaser and refined iteratively using autoBUSTER followed by model building in Coot.</div><div class="NLM_p last">Structure of the SARS CoV-2 main protease in complex with <b>PF-835231</b> was determined similarly by rigid body refinement using the SARS CoV main protease structure in complex with the same ligand as the starting model.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i98"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01063" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75587" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75587" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01063?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01063</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Molecular formula strings for all compounds with biological data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01063/suppl_file/jm0c01063_si_001.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">HPLC purity for compound <b>4</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01063/suppl_file/jm0c01063_si_002.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01063/suppl_file/jm0c01063_si_001.csv">jm0c01063_si_001.csv (4.06 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01063/suppl_file/jm0c01063_si_002.pdf">jm0c01063_si_002.pdf (95.19 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c01063" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52501" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52501" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert L. Hoffman</span> - <span class="hlFld-Affiliation affiliation">Pfizer
Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2030-9094" title="Orcid link">http://orcid.org/0000-0002-2030-9094</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#bbc9d4d9dec9cf95d795d3d4ddddd6dad5fbcbddd2c1dec995d8d4d6"><span class="__cf_email__" data-cfemail="22504d404750560c4e0c4a4d44444f434c6252444b5847500c414d4f">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert S. Kania</span> - <span class="hlFld-Affiliation affiliation">Pfizer
Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mary A. Brothers</span> - <span class="hlFld-Affiliation affiliation">Pfizer
Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jay F. Davies</span> - <span class="hlFld-Affiliation affiliation">Pfizer
Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rose A. Ferre</span> - <span class="hlFld-Affiliation affiliation">Pfizer
Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ketan S. Gajiwala</span> - <span class="hlFld-Affiliation affiliation">Pfizer
Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mingying He</span> - <span class="hlFld-Affiliation affiliation">Pfizer
Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Robert J. Hogan</span> - <span class="hlFld-Affiliation affiliation">Southern
Research Institute, 2000 9th Avenue South, Birmingham, Alabama 35205 United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kirk Kozminski</span> - <span class="hlFld-Affiliation affiliation">Pfizer
Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lilian Y. Li</span> - <span class="hlFld-Affiliation affiliation">Pfizer
Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jonathan W. Lockner</span> - <span class="hlFld-Affiliation affiliation">Pfizer
Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6148-0579" title="Orcid link">http://orcid.org/0000-0001-6148-0579</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jihong Lou</span> - <span class="hlFld-Affiliation affiliation">Pfizer
Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michelle T. Marra</span> - <span class="hlFld-Affiliation affiliation">Pfizer
Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lennert J. Mitchell</span> - <span class="hlFld-Affiliation affiliation">Pfizer
Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Brion W. Murray</span> - <span class="hlFld-Affiliation affiliation">Pfizer
Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James A. Nieman</span> - <span class="hlFld-Affiliation affiliation">Pfizer
Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5087-0819" title="Orcid link">http://orcid.org/0000-0002-5087-0819</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephen Noell</span> - <span class="hlFld-Affiliation affiliation">Pfizer
Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Simon P. Planken</span> - <span class="hlFld-Affiliation affiliation">Pfizer
Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8901-8992" title="Orcid link">http://orcid.org/0000-0001-8901-8992</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas Rowe</span> - <span class="hlFld-Affiliation affiliation">Southern
Research Institute, 2000 9th Avenue South, Birmingham, Alabama 35205 United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kevin Ryan</span> - <span class="hlFld-Affiliation affiliation">Pfizer
Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">George J. Smith</span> - <span class="hlFld-Affiliation affiliation">Pfizer
Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James E. Solowiej</span> - <span class="hlFld-Affiliation affiliation">Pfizer
Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Claire M. Steppan</span> - <span class="hlFld-Affiliation affiliation">Pfizer
Worldwide Research and Development, 10770 Science Center Drive, San Diego, California 92121 United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Barbara Taggart</span> - <span class="hlFld-Affiliation affiliation">Southern
Research Institute, 2000 9th Avenue South, Birmingham, Alabama 35205 United
States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through the contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li><span class="author-information-subsection-header">Funding</span><p>Funding for the antiviral testing was provided by NIAID under a cooperative agreement between NIAID and Pfizer.</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e10281-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i100">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83356" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83356" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors would like to thank the Pfizer La Jolla Discovery Technology group who were responsible for all HRMS, preparative HPLC, and SFC separations. Antiviral testing was performed at the Southern Research Institute (Birmingham, AL). The authors wish to thank Dr. Heinz Feldman (NIAID, Hamilton, MT) for the kind gift of the SARS CoV Toronto-2 cell line enabling this testing. The authors wish to thank Cathy Laughlin, Christopher Tseng, and Jack Secrist for their roles in this collaboration. The authors would also like to thank Kevin Freeman-Cook, Charlotte Allerton, and Gina M. Yanochko-Hoffman for critically reading the manuscript and offering valued editing suggestions.</p></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i101">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78320" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78320" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 58 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Z.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, Z.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, M.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, F.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.-Z.</span></span> <span> </span><span class="NLM_article-title">A new coronavirus associated with human respiratory disease in China</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>579</i></span>,  <span class="NLM_fpage">265</span>– <span class="NLM_lpage">269</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2008-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1038%2Fs41586-020-2008-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=32015508" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFKlsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=579&publication_year=2020&pages=265-269&author=F.+Wuauthor=S.+Zhaoauthor=B.+Yuauthor=Y.-M.+Chenauthor=W.+Wangauthor=Z.-G.+Songauthor=Y.+Huauthor=Z.-W.+Taoauthor=J.-H.+Tianauthor=Y.-Y.+Peiauthor=M.-L.+Yuanauthor=Y.-L.+Zhangauthor=F.-H.+Daiauthor=Y.+Liuauthor=Q.-M.+Wangauthor=J.-J.+Zhengauthor=L.+Xuauthor=E.+C.+Holmesauthor=Y.-Z.+Zhang&title=A+new+coronavirus+associated+with+human+respiratory+disease+in+China&doi=10.1038%2Fs41586-020-2008-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">A new coronavirus associated with human respiratory disease in China</span></div><div class="casAuthors">Wu, Fan; Zhao, Su; Yu, Bin; Chen, Yan-Mei; Wang, Wen; Song, Zhi-Gang; Hu, Yi; Tao, Zhao-Wu; Tian, Jun-Hua; Pei, Yuan-Yuan; Yuan, Ming-Li; Zhang, Yu-Ling; Dai, Fa-Hui; Liu, Yi; Wang, Qi-Min; Zheng, Jiao-Jiao; Xu, Lin; Holmes, Edward C.; Zhang, Yong-Zhen</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">579</span>
        (<span class="NLM_cas:issue">7798</span>),
    <span class="NLM_cas:pages">265-269</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Emerging infectious diseases, such as severe acute respiratory syndrome (SARS) and Zika virus disease, present a major threat to public health.  Despite intense research efforts, how, when and where new diseases appear are still a source of considerable uncertainty.  A severe respiratory disease was recently reported in Wuhan, Hubei province, China.  As of 25 Jan. 2020, at least 1,975 cases had been reported since the first patient was hospitalized on 12 Dec. 2019.  Epidemiol. investigations have suggested that the outbreak was assocd. with a seafood market in Wuhan.  Here we study a single patient who was a worker at the market and who was admitted to the Central Hospital of Wuhan on 26 Dec. 2019 while experiencing a severe respiratory syndrome that included fever, dizziness and a cough.  Metagenomic RNA sequencing of a sample of bronchoalveolar lavage fluid from the patient identified a new RNA virus strain from the family Coronaviridae, which is designated here 'WH-Human 1' coronavirus (and has also been referred to as '2019-nCoV').  Phylogenetic anal. of the complete viral genome (29,903 nucleotides) revealed that the virus was most closely related (89.1% nucleotide similarity) to a group of SARS-like coronaviruses (genus Betacoronavirus, subgenus Sarbecovirus) that had previously been found in bats in China.  This outbreak highlights the ongoing ability of viral spill-over from animals to cause severe disease in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5h_VHfuDuTbVg90H21EOLACvtfcHk0lidaIdQEnpsVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFKlsLc%253D&md5=0163a684829e880a0c3347e19f0ce52a</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2008-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2008-3%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DY.-M.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DSong%26aufirst%3DZ.-G.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DTao%26aufirst%3DZ.-W.%26aulast%3DTian%26aufirst%3DJ.-H.%26aulast%3DPei%26aufirst%3DY.-Y.%26aulast%3DYuan%26aufirst%3DM.-L.%26aulast%3DZhang%26aufirst%3DY.-L.%26aulast%3DDai%26aufirst%3DF.-H.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DQ.-M.%26aulast%3DZheng%26aufirst%3DJ.-J.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DHolmes%26aufirst%3DE.%2BC.%26aulast%3DZhang%26aufirst%3DY.-Z.%26atitle%3DA%2520new%2520coronavirus%2520associated%2520with%2520human%2520respiratory%2520disease%2520in%2520China%26jtitle%3DNature%26date%3D2020%26volume%3D579%26spage%3D265%26epage%3D269%26doi%3D10.1038%2Fs41586-020-2008-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Si, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, R.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, X.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, F.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, G.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.-L.</span></span> <span> </span><span class="NLM_article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>579</i></span>,  <span class="NLM_fpage">270</span>– <span class="NLM_lpage">273</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2012-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1038%2Fs41586-020-2012-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=32015507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFKlsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=579&publication_year=2020&pages=270-273&author=P.+Zhouauthor=X.-L.+Yangauthor=X.-G.+Wangauthor=B.+Huauthor=L.+Zhangauthor=W.+Zhangauthor=H.-R.+Siauthor=Y.+Zhuauthor=B.+Liauthor=C.-L.+Huangauthor=H.-D.+Chenauthor=J.+Chenauthor=Y.+Luoauthor=H.+Guoauthor=R.-D.+Jiangauthor=M.-Q.+Liuauthor=Y.+Chenauthor=X.-R.+Shenauthor=X.+Wangauthor=X.-S.+Zhengauthor=K.+Zhaoauthor=Q.-J.+Chenauthor=F.+Dengauthor=L.-L.+Liuauthor=B.+Yanauthor=F.-X.+Zhanauthor=Y.-Y.+Wangauthor=G.-F.+Xiaoauthor=Z.-L.+Shi&title=A+pneumonia+outbreak+associated+with+a+new+coronavirus+of+probable+bat+origin&doi=10.1038%2Fs41586-020-2012-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span></div><div class="casAuthors">Zhou, Peng; Yang, Xing-Lou; Wang, Xian-Guang; Hu, Ben; Zhang, Lei; Zhang, Wei; Si, Hao-Rui; Zhu, Yan; Li, Bei; Huang, Chao-Lin; Chen, Hui-Dong; Chen, Jing; Luo, Yun; Guo, Hua; Jiang, Ren-Di; Liu, Mei-Qin; Chen, Ying; Shen, Xu-Rui; Wang, Xi; Zheng, Xiao-Shuang; Zhao, Kai; Chen, Quan-Jiao; Deng, Fei; Liu, Lin-Lin; Yan, Bing; Zhan, Fa-Xian; Wang, Yan-Yi; Xiao, Geng-Fu; Shi, Zheng-Li</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">579</span>
        (<span class="NLM_cas:issue">7798</span>),
    <span class="NLM_cas:pages">270-273</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Since the outbreak of severe acute respiratory syndrome (SARS) 18 years ago, a large no. of SARS-related coronaviruses (SARSr-CoVs) have been discovered in their natural reservoir host, bats1-4.  Previous studies have shown that some bat SARSr-CoVs have the potential to infect humans5-7.  Here we report the identification and characterization of a new coronavirus (2019-nCoV), which caused an epidemic of acute respiratory syndrome in humans in Wuhan, China.  The epidemic, which started on 12 Dec. 2019, had caused 2,794 lab.-confirmed infections including 80 deaths by 26 Jan. 2020.  Full-length genome sequences were obtained from five patients at an early stage of the outbreak.  The sequences are almost identical and share 79.6% sequence identity to SARS-CoV.  Furthermore, we show that 2019-nCoV is 96% identical at the whole-genome level to a bat coronavirus.  Pairwise protein sequence anal. of seven conserved non-structural proteins domains show that this virus belongs to the species of SARSr-CoV.  In addn., 2019-nCoV virus isolated from the bronchoalveolar lavage fluid of a critically ill patient could be neutralized by sera from several patients.  Notably, we confirmed that 2019-nCoV uses the same cell entry receptor-angiotensin converting enzyme II (ACE2)-as SARS-CoV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrV_Wb_iUsbFrVg90H21EOLACvtfcHk0lidaIdQEnpsVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFKlsLg%253D&md5=236f17d4d3c7978d72513e5e0258f1b3</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2012-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2012-7%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DP.%26aulast%3DYang%26aufirst%3DX.-L.%26aulast%3DWang%26aufirst%3DX.-G.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DSi%26aufirst%3DH.-R.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DHuang%26aufirst%3DC.-L.%26aulast%3DChen%26aufirst%3DH.-D.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DR.-D.%26aulast%3DLiu%26aufirst%3DM.-Q.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DX.-R.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DX.-S.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DQ.-J.%26aulast%3DDeng%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DL.-L.%26aulast%3DYan%26aufirst%3DB.%26aulast%3DZhan%26aufirst%3DF.-X.%26aulast%3DWang%26aufirst%3DY.-Y.%26aulast%3DXiao%26aufirst%3DG.-F.%26aulast%3DShi%26aufirst%3DZ.-L.%26atitle%3DA%2520pneumonia%2520outbreak%2520associated%2520with%2520a%2520new%2520coronavirus%2520of%2520probable%2520bat%2520origin%26jtitle%3DNature%26date%3D2020%26volume%3D579%26spage%3D270%26epage%3D273%26doi%3D10.1038%2Fs41586-020-2012-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marra, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Astell, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holt, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks-Wilson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butterfield, Y. S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khattra, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asano, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barber, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cloutier, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coughlin, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girn, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffith, O. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandoh, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrescu, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schein, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddiqui, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smailus, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stott, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plummer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artsob, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bastien, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Booth, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowness, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czub, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drebot, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernando, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flick, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garbutt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grolla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyers, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Normand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroher, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tipples, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogrig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunham, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krajden, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petric, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skowronski, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upton, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roper, R. L.</span></span> <span> </span><span class="NLM_article-title">The genome sequence of the SARS-associated coronavirus</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>300</i></span>,  <span class="NLM_fpage">1399</span>– <span class="NLM_lpage">1404</span>, <span class="refDoi"> DOI: 10.1126/science.1085953</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1126%2Fscience.1085953" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=12730501" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BD3sXktFGkt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=300&publication_year=2003&pages=1399-1404&author=M.+A.+Marraauthor=J.+M.+Jonesauthor=C.+R.+Astellauthor=R.+A.+Holtauthor=A.+Brooks-Wilsonauthor=Y.+S.+N.+Butterfieldauthor=J.+Khattraauthor=J.+K.+Asanoauthor=S.+A.+Barberauthor=S.+Y.+Chanauthor=A.+Cloutierauthor=S.+M.+Coughlinauthor=D.+Freemanauthor=N.+Girnauthor=O.+L.+Griffithauthor=S.+R.+Leachauthor=M.+Mayoauthor=H.+McDonaldauthor=S.+B.+Montgomeryauthor=P.+K.+Pandohauthor=A.+S.+Petrescuauthor=A.+G.+Robertsonauthor=J.+E.+Scheinauthor=A.+Siddiquiauthor=D.+E.+Smailusauthor=J.+M.+Stottauthor=G.+S.+Yangauthor=F.+Plummerauthor=A.+Andonovauthor=H.+Artsobauthor=N.+Bastienauthor=K.+Bernardauthor=T.+F.+Boothauthor=D.+Bownessauthor=M.+Czubauthor=M.+Drebotauthor=L.+Fernandoauthor=R.+Flickauthor=M.+Garbuttauthor=M.+Grayauthor=A.+Grollaauthor=S.+Jonesauthor=H.+Feldmannauthor=A.+Meyersauthor=A.+Kabaniauthor=Y.+Liauthor=S.+Normandauthor=U.+Stroherauthor=G.+A.+Tipplesauthor=S.+Tylerauthor=R.+Vogrigauthor=D.+Wardauthor=B.+Watsonauthor=R.+C.+Brunhamauthor=M.+Krajdenauthor=M.+Petricauthor=D.+M.+Skowronskiauthor=C.+Uptonauthor=R.+L.+Roper&title=The+genome+sequence+of+the+SARS-associated+coronavirus&doi=10.1126%2Fscience.1085953"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The genome sequence of the SARS-associated coronavirus</span></div><div class="casAuthors">Marra, Marco A.; Jones, Steven J. M.; Astell, Caroline R.; Holt, Robert A.; Brooks-Wilson, Angela; Butterfield, Yaron S. N.; Khattra, Jaswinder; Asano, Jennifer K.; Barber, Sarah A.; Chan, Susanna Y.; Cloutier, Alison; Coughlin, Shaun M.; Freeman, Doug; Girn, Noreen; Griffith, Obi L.; Leach, Stephen R.; Mayo, Michael; McDonald, Helen; Montgomery, Stephen B.; Pandoh, Pawan K.; Petrescu, Anca S.; Robertson, A. Gordon; Schein, Jacqueline E.; Siddiqui, Asim; Smailus, Duane E.; Stott, Jeff M.; Yang, George S.; Plummer, Francis; Andonov, Anton; Artsob, Harvey; Bastien, Nathalie; Bernard, Kathy; Booth, Timothy F.; Bowness, Donnie; Czub, Martin; Drebot, Michael; Fernando, Lisa; Flick, Ramon; Garbutt, Michael; Gray, Michael; Grolla, Allen; Jones, Steven; Feldmann, Heinz; Meyers, Adrienne; Kabani, Amin; Li, Yan; Normand, Susan; Stroher, Ute; Tipples, Graham A.; Tyler, Shaun; Vogrig, Robert; Ward, Diane; Watson, Brynn; Brunham, Robert C.; Krajden, Mel; Petric, Martin; Skowronski, Danuta M.; Upton, Chris; Roper, Rachel L.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">300</span>
        (<span class="NLM_cas:issue">5624</span>),
    <span class="NLM_cas:pages">1399-1404</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">We sequenced the 29,751-base genome of the severe acute respiratory syndrome (SARS)-assocd. coronavirus known as the Tor2 isolate.  The genome sequence reveals that this coronavirus is only moderately related to other known coronaviruses, including two human coronaviruses, HCoV-OC43 and HCoV-229E.  Phylogenetic anal. of the predicted viral proteins indicates that the virus does not closely resemble any of the three previously known groups of coronaviruses.  The genome sequence will aid in the diagnosis of SARS virus infection in humans and potential animal hosts (using polymerase chain reaction and immunol. tests), in the development of antivirals (including neutralizing antibodies), and in the identification of putative epitopes for vaccine development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqo2-FA4hirLLVg90H21EOLACvtfcHk0ljO5NyJVoKDwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXktFGkt7c%253D&md5=d639d786543b32039b250134134d653b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1126%2Fscience.1085953&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1085953%26sid%3Dliteratum%253Aachs%26aulast%3DMarra%26aufirst%3DM.%2BA.%26aulast%3DJones%26aufirst%3DJ.%2BM.%26aulast%3DAstell%26aufirst%3DC.%2BR.%26aulast%3DHolt%26aufirst%3DR.%2BA.%26aulast%3DBrooks-Wilson%26aufirst%3DA.%26aulast%3DButterfield%26aufirst%3DY.%2BS.%2BN.%26aulast%3DKhattra%26aufirst%3DJ.%26aulast%3DAsano%26aufirst%3DJ.%2BK.%26aulast%3DBarber%26aufirst%3DS.%2BA.%26aulast%3DChan%26aufirst%3DS.%2BY.%26aulast%3DCloutier%26aufirst%3DA.%26aulast%3DCoughlin%26aufirst%3DS.%2BM.%26aulast%3DFreeman%26aufirst%3DD.%26aulast%3DGirn%26aufirst%3DN.%26aulast%3DGriffith%26aufirst%3DO.%2BL.%26aulast%3DLeach%26aufirst%3DS.%2BR.%26aulast%3DMayo%26aufirst%3DM.%26aulast%3DMcDonald%26aufirst%3DH.%26aulast%3DMontgomery%26aufirst%3DS.%2BB.%26aulast%3DPandoh%26aufirst%3DP.%2BK.%26aulast%3DPetrescu%26aufirst%3DA.%2BS.%26aulast%3DRobertson%26aufirst%3DA.%2BG.%26aulast%3DSchein%26aufirst%3DJ.%2BE.%26aulast%3DSiddiqui%26aufirst%3DA.%26aulast%3DSmailus%26aufirst%3DD.%2BE.%26aulast%3DStott%26aufirst%3DJ.%2BM.%26aulast%3DYang%26aufirst%3DG.%2BS.%26aulast%3DPlummer%26aufirst%3DF.%26aulast%3DAndonov%26aufirst%3DA.%26aulast%3DArtsob%26aufirst%3DH.%26aulast%3DBastien%26aufirst%3DN.%26aulast%3DBernard%26aufirst%3DK.%26aulast%3DBooth%26aufirst%3DT.%2BF.%26aulast%3DBowness%26aufirst%3DD.%26aulast%3DCzub%26aufirst%3DM.%26aulast%3DDrebot%26aufirst%3DM.%26aulast%3DFernando%26aufirst%3DL.%26aulast%3DFlick%26aufirst%3DR.%26aulast%3DGarbutt%26aufirst%3DM.%26aulast%3DGray%26aufirst%3DM.%26aulast%3DGrolla%26aufirst%3DA.%26aulast%3DJones%26aufirst%3DS.%26aulast%3DFeldmann%26aufirst%3DH.%26aulast%3DMeyers%26aufirst%3DA.%26aulast%3DKabani%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DNormand%26aufirst%3DS.%26aulast%3DStroher%26aufirst%3DU.%26aulast%3DTipples%26aufirst%3DG.%2BA.%26aulast%3DTyler%26aufirst%3DS.%26aulast%3DVogrig%26aufirst%3DR.%26aulast%3DWard%26aufirst%3DD.%26aulast%3DWatson%26aufirst%3DB.%26aulast%3DBrunham%26aufirst%3DR.%2BC.%26aulast%3DKrajden%26aufirst%3DM.%26aulast%3DPetric%26aufirst%3DM.%26aulast%3DSkowronski%26aufirst%3DD.%2BM.%26aulast%3DUpton%26aufirst%3DC.%26aulast%3DRoper%26aufirst%3DR.%2BL.%26atitle%3DThe%2520genome%2520sequence%2520of%2520the%2520SARS-associated%2520coronavirus%26jtitle%3DScience%26date%3D2003%26volume%3D300%26spage%3D1399%26epage%3D1404%26doi%3D10.1126%2Fscience.1085953" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rota, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oberste, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monroe, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nix, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campagnoli, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Icenogle, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penaranda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bankamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maher, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frace, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeRisi, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdman, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peret, T. C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ksiazek, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rollin, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liffick, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Limor, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCaustland, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen-Rasmussen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fouchier, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunther, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osterhaus, A. D. M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drosten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallansch, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellini, W. J.</span></span> <span> </span><span class="NLM_article-title">The genome sequence of the SARS-associated coronavirus. Characterization of a novel coronavirus associated with severe acute respiratory syndrome</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>300</i></span>,  <span class="NLM_fpage">1394</span>– <span class="NLM_lpage">1399</span>, <span class="refDoi"> DOI: 10.1126/science.1085952</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1126%2Fscience.1085952" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=12730500" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BD3sXktFGkt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=300&publication_year=2003&pages=1394-1399&author=P.+A.+Rotaauthor=M.+S.+Obersteauthor=S.+S.+Monroeauthor=W.+A.+Nixauthor=R.+Campagnoliauthor=J.+P.+Icenogleauthor=S.+Penarandaauthor=B.+Bankampauthor=K.+Maherauthor=M.-H.+Chenauthor=S.+Tongauthor=A.+Taminauthor=L.+Loweauthor=M.+Fraceauthor=J.+L.+DeRisiauthor=Q.+Chenauthor=D.+Wangauthor=D.+D.+Erdmanauthor=T.+C.+T.+Peretauthor=C.+Burnsauthor=T.+G.+Ksiazekauthor=P.+E.+Rollinauthor=A.+Sanchezauthor=S.+Liffickauthor=B.+Hollowayauthor=J.+Limorauthor=K.+McCaustlandauthor=M.+Olsen-Rasmussenauthor=R.+Fouchierauthor=S.+Guntherauthor=A.+D.+M.+E.+Osterhausauthor=C.+Drostenauthor=M.+A.+Pallanschauthor=L.+J.+Andersonauthor=W.+J.+Bellini&title=The+genome+sequence+of+the+SARS-associated+coronavirus.+Characterization+of+a+novel+coronavirus+associated+with+severe+acute+respiratory+syndrome&doi=10.1126%2Fscience.1085952"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of a novel coronavirus associated with severe acute respiratory syndrome</span></div><div class="casAuthors">Rota, Paul A.; Oberste, M. Steven; Monroe, Stephan S.; Nix, W. Allan; Campagnoli, Ray; Icenogle, Joseph P.; Penaranda, Silvia; Bankamp, Bettina; Maher, Kaija; Chen, Min-hsin; Tong, Suxiong; Tamin, Azaibi; Lowe, Luis; Frace, Michael; DeRisi, Joseph L.; Chen, Qi; Wang, David; Erdman, Dean D.; Peret, Teresa C. T.; Burns, Cara; Ksiazek, Thomas G.; Rollin, Pierre E.; Sanchez, Anthony; Liffick, Stephanie; Holloway, Brian; Limor, Josef; McCaustland, Karen; Olsen-Rasmussen, Melissa; Fouchier, Ron; Guenther, Stephan; Osterhaus, Albert D. M. E.; Drosten, Christian; Pallansch, Mark A.; Anderson, Larry J.; Bellini, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">300</span>
        (<span class="NLM_cas:issue">5624</span>),
    <span class="NLM_cas:pages">1394-1399</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">In Mar. 2003, a novel coronavirus (SARS-CoV) was discovered in assocn. with cases of severe acute respiratory syndrome (SARS).  The sequence of the complete genome of SARS-CoV was detd., and the initial characterization of the viral genome is presented in this report.  The genome of SARS-CoV is 29,727 nucleotides in length and has 11 open reading frames, and its genome organization is similar to that of other coronaviruses.  Phylogenetic analyses and sequence comparisons showed that SARS-CoV is not closely related to any of the previously characterized coronaviruses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGmSeCD00yh7Vg90H21EOLACvtfcHk0ljO5NyJVoKDwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXktFGkt7Y%253D&md5=eb8d2156d170566242ebd78824e5d30b</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1126%2Fscience.1085952&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1085952%26sid%3Dliteratum%253Aachs%26aulast%3DRota%26aufirst%3DP.%2BA.%26aulast%3DOberste%26aufirst%3DM.%2BS.%26aulast%3DMonroe%26aufirst%3DS.%2BS.%26aulast%3DNix%26aufirst%3DW.%2BA.%26aulast%3DCampagnoli%26aufirst%3DR.%26aulast%3DIcenogle%26aufirst%3DJ.%2BP.%26aulast%3DPenaranda%26aufirst%3DS.%26aulast%3DBankamp%26aufirst%3DB.%26aulast%3DMaher%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DM.-H.%26aulast%3DTong%26aufirst%3DS.%26aulast%3DTamin%26aufirst%3DA.%26aulast%3DLowe%26aufirst%3DL.%26aulast%3DFrace%26aufirst%3DM.%26aulast%3DDeRisi%26aufirst%3DJ.%2BL.%26aulast%3DChen%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DErdman%26aufirst%3DD.%2BD.%26aulast%3DPeret%26aufirst%3DT.%2BC.%2BT.%26aulast%3DBurns%26aufirst%3DC.%26aulast%3DKsiazek%26aufirst%3DT.%2BG.%26aulast%3DRollin%26aufirst%3DP.%2BE.%26aulast%3DSanchez%26aufirst%3DA.%26aulast%3DLiffick%26aufirst%3DS.%26aulast%3DHolloway%26aufirst%3DB.%26aulast%3DLimor%26aufirst%3DJ.%26aulast%3DMcCaustland%26aufirst%3DK.%26aulast%3DOlsen-Rasmussen%26aufirst%3DM.%26aulast%3DFouchier%26aufirst%3DR.%26aulast%3DGunther%26aufirst%3DS.%26aulast%3DOsterhaus%26aufirst%3DA.%2BD.%2BM.%2BE.%26aulast%3DDrosten%26aufirst%3DC.%26aulast%3DPallansch%26aufirst%3DM.%2BA.%26aulast%3DAnderson%26aufirst%3DL.%2BJ.%26aulast%3DBellini%26aufirst%3DW.%2BJ.%26atitle%3DThe%2520genome%2520sequence%2520of%2520the%2520SARS-associated%2520coronavirus.%2520Characterization%2520of%2520a%2520novel%2520coronavirus%2520associated%2520with%2520severe%2520acute%2520respiratory%2520syndrome%26jtitle%3DScience%26date%3D2003%26volume%3D300%26spage%3D1394%26epage%3D1399%26doi%3D10.1126%2Fscience.1085952" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">World Health Organization</span>. <span> </span><span class="NLM_article-title">Cumulative Number of Reported Probable
Cases</span>, <a href="http://www.who.int/csr/sars/country/2003_06_17/en/index.html" class="extLink">http://www.who.int/csr/sars/country/2003_06_17/en/index.html</a> (accessed June 17, <span class="NLM_year">2003</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2003&author=World+Health+Organization&title=Cumulative+Number+of+Reported+Probable%0ACases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DCumulative%2520Number%2520of%2520Reported%2520Probable%250ACases%26date%3D2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">John Hopkins University</span>. <span> </span><span class="NLM_article-title">COVID-19
Dashboard by the Center for Systems Science and Engineering
(CSSE) at Johns Hopkins University (JHU)</span>, <a href="https://coronavirus.jhu.edu/map.html" class="extLink">https://coronavirus.jhu.edu/map.html</a> (accessed June 16, <span class="NLM_year">2020</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=John+Hopkins+University&title=COVID-19%0ADashboard+by+the+Center+for+Systems+Science+and+Engineering%0A%28CSSE%29+at+Johns+Hopkins+University+%28JHU%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DCOVID-19%250ADashboard%2520by%2520the%2520Center%2520for%2520Systems%2520Science%2520and%2520Engineering%250A%2528CSSE%2529%2520at%2520Johns%2520Hopkins%2520University%2520%2528JHU%2529%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thiel, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivanov, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Putics, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hertzig, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chelle, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">WeiBbrich, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snijder, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabenau, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doerr, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorbalenya, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziebuhr, J.</span></span> <span> </span><span class="NLM_article-title">Mechanisms and enzymes involved in SARS coronavirus genome expression</span>. <i>J. Gen. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">2305</span>– <span class="NLM_lpage">2315</span>, <span class="refDoi"> DOI: 10.1099/vir.0.19424-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1099%2Fvir.0.19424-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=12917450" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BD3sXntFaqsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2003&pages=2305-2315&author=V.+Thielauthor=K.+A.+Ivanovauthor=A.+Puticsauthor=T.+Hertzigauthor=B.+Chelleauthor=S.+Bayerauthor=B.+WeiBbrichauthor=E.+J.+Snijderauthor=H.+Rabenauauthor=H.+W.+Doerrauthor=A.+E.+Gorbalenyaauthor=J.+Ziebuhr&title=Mechanisms+and+enzymes+involved+in+SARS+coronavirus+genome+expression&doi=10.1099%2Fvir.0.19424-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms and enzymes involved in SARS coronavirus genome expression</span></div><div class="casAuthors">Thiel, Volker; Ivanov, Konstantin A.; Putics, Akos; Hertzig, Tobias; Schelle, Barbara; Bayer, Sonja; Weissbrich, Benedikt; Snijder, Eric J.; Rabenau, Holger; Doerr, Hans Wilhelm; Gorbalenya, Alexander E.; Ziebuhr, John</div><div class="citationInfo"><span class="NLM_cas:title">Journal of General Virology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2305-2315</span>CODEN:
                <span class="NLM_cas:coden">JGVIAY</span>;
        ISSN:<span class="NLM_cas:issn">0022-1317</span>.
    
            (<span class="NLM_cas:orgname">Society for General Microbiology</span>)
        </div><div class="casAbstract">A novel coronavirus is the causative agent of the current epidemic of severe acute respiratory syndrome (SARS).  Coronaviruses are exceptionally large RNA viruses and employ complex regulatory mechanisms to express their genomes.  Here, we detd. the sequence of SARS coronavirus (SARS-CoV), isolate Frankfurt 1, and characterized key RNA elements and protein functions involved in viral genome expression.  Important regulatory mechanisms, such as the (discontinuous) synthesis of eight subgenomic mRNAs, ribosomal frameshifting and post-translational proteolytic processing, were addressed.  Activities of three SARS coronavirus enzymes, the helicase and two cysteine proteinases, which are known to be critically involved in replication, transcription and/or post-translational polyprotein processing, were characterized.  The availability of recombinant forms of key replicative enzymes of SARS coronavirus should pave the way for high-throughput screening approaches to identify candidate inhibitors in compd. libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwV_DQUJ6EFbVg90H21EOLACvtfcHk0li75Mynvpj6bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXntFaqsb4%253D&md5=c0e344cdde7ce697dee5a6aaeea9feb7</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1099%2Fvir.0.19424-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1099%252Fvir.0.19424-0%26sid%3Dliteratum%253Aachs%26aulast%3DThiel%26aufirst%3DV.%26aulast%3DIvanov%26aufirst%3DK.%2BA.%26aulast%3DPutics%26aufirst%3DA.%26aulast%3DHertzig%26aufirst%3DT.%26aulast%3DChelle%26aufirst%3DB.%26aulast%3DBayer%26aufirst%3DS.%26aulast%3DWeiBbrich%26aufirst%3DB.%26aulast%3DSnijder%26aufirst%3DE.%2BJ.%26aulast%3DRabenau%26aufirst%3DH.%26aulast%3DDoerr%26aufirst%3DH.%2BW.%26aulast%3DGorbalenya%26aufirst%3DA.%2BE.%26aulast%3DZiebuhr%26aufirst%3DJ.%26atitle%3DMechanisms%2520and%2520enzymes%2520involved%2520in%2520SARS%2520coronavirus%2520genome%2520expression%26jtitle%3DJ.%2520Gen.%2520Virol.%26date%3D2003%26volume%3D84%26spage%3D2305%26epage%3D2315%26doi%3D10.1099%2Fvir.0.19424-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curth, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drosten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauerhering, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rox, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved α-ketoamide inhibitors</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>, <span class="NLM_elocation-id">eabb3405</span> <span class="refDoi"> DOI: 10.1126/science.abb3405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1126%2Fscience.abb3405" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&author=L.+Zhangauthor=D.+Linauthor=X.+Sunauthor=U.+Curthauthor=C.+Drostenauthor=L.+Sauerheringauthor=S.+Beckerauthor=K.+Roxauthor=R.+Hilgenfeld&title=Crystal+structure+of+SARS-CoV-2+main+protease+provides+a+basis+for+design+of+improved+%CE%B1-ketoamide+inhibitors&doi=10.1126%2Fscience.abb3405"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1126%2Fscience.abb3405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abb3405%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DD.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DCurth%26aufirst%3DU.%26aulast%3DDrosten%26aufirst%3DC.%26aulast%3DSauerhering%26aufirst%3DL.%26aulast%3DBecker%26aufirst%3DS.%26aulast%3DRox%26aufirst%3DK.%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3DCrystal%2520structure%2520of%2520SARS-CoV-2%2520main%2520protease%2520provides%2520a%2520basis%2520for%2520design%2520of%2520improved%2520%25CE%25B1-ketoamide%2520inhibitors%26jtitle%3DScience%26date%3D2020%26volume%3D368%26doi%3D10.1126%2Fscience.abb3405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of antiviral drug candidates targeting the SARS-CoV-2 main protease</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">1331</span>– <span class="NLM_lpage">1335</span>, <span class="refDoi"> DOI: 10.1126/science.abb4489</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1126%2Fscience.abb4489" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=32321856" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1alsrzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=1331-1335&author=W.+Daiauthor=B.+Zhangauthor=X.-M.+Jiangauthor=H.+Suauthor=J.+Liauthor=Y.+Zhaoauthor=X.+Xieauthor=Z.+Jinauthor=J.+Pengauthor=F.+Liuauthor=C.+Liauthor=Y.+Liauthor=F.+Baiauthor=H.+Wangauthor=X.+Chengauthor=X.+Cenauthor=S.+Huauthor=X.+Yangauthor=J.+Wangauthor=X.+Liuauthor=G.+Xiaoauthor=H.+Jiangauthor=Z.+Raoauthor=L.-K.+Zhangauthor=Y.+Xuauthor=H.+Yangauthor=H.+Liu&title=Structure-based+design+of+antiviral+drug+candidates+targeting+the+SARS-CoV-2+main+protease&doi=10.1126%2Fscience.abb4489"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design of antiviral drug candidates targeting the SARS-CoV-2 main protease</span></div><div class="casAuthors">Dai, Wenhao; Zhang, Bing; Jiang, Xia-Ming; Su, Haixia; Li, Jian; Zhao, Yao; Xie, Xiong; Jin, Zhenming; Peng, Jingjing; Liu, Fengjiang; Li, Chunpu; Li, You; Bai, Fang; Wang, Haofeng; Cheng, Xi; Cen, Xiaobo; Hu, Shulei; Yang, Xiuna; Wang, Jiang; Liu, Xiang; Xiao, Gengfu; Jiang, Hualiang; Rao, Zihe; Zhang, Lei-Ke; Xu, Yechun; Yang, Haitao; Liu, Hong</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6497</span>),
    <span class="NLM_cas:pages">1331-1335</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) is the etiol. agent responsible for the global COVID-19 (coronavirus disease 2019) outbreak.  The main protease of SARS-CoV-2, Mpro, is a key enzyme that plays a pivotal role in mediating viral replication and transcription.  We designed and synthesized two lead compds. (11a and 11b) targeting Mpro.  Both exhibited excellent inhibitory activity and potent anti-SARS-CoV-2 infection activity.  The x-ray crystal structures of SARS-CoV-2 Mpro in complex with 11a or 11b, both detd. at a resoln. of 1.5 angstroms, showed that the aldehyde groups of 11a and 11b are covalently bound to cysteine 145 of Mpro.  Both compds. showed good pharmacokinetic properties in vivo, and 11a also exhibited low toxicity, which suggests that these compds. are promising drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFFvzOwCOpZrVg90H21EOLACvtfcHk0li75Mynvpj6bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1alsrzL&md5=ec87c340410edb919d2584a2b4d33e1a</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1126%2Fscience.abb4489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abb4489%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DJiang%26aufirst%3DX.-M.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DJin%26aufirst%3DZ.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DCen%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DXiao%26aufirst%3DG.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DRao%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DL.-K.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DH.%26atitle%3DStructure-based%2520design%2520of%2520antiviral%2520drug%2520candidates%2520targeting%2520the%2520SARS-CoV-2%2520main%2520protease%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3D1331%26epage%3D1335%26doi%3D10.1126%2Fscience.abb4489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Webber, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okano, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Little, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reich, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuhrman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Love, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendrickson, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patrick, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meador, J. W.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferre, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binford, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Worland, S. T.</span></span> <span> </span><span class="NLM_article-title">Tripeptide aldehyde inhibitors of human rhinovirus 3C protease: design, synthesis, biological evaluation, and cocrystal structure solution of P1 glutamine isosteric replacements</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">2786</span>– <span class="NLM_lpage">2805</span>, <span class="refDoi"> DOI: 10.1021/jm980071x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm980071x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK1cXksVKrtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=2786-2805&author=S.+E.+Webberauthor=K.+Okanoauthor=T.+L.+Littleauthor=S.+H.+Reichauthor=Y.+Xinauthor=S.+A.+Fuhrmanauthor=D.+A.+Matthewsauthor=R.+A.+Loveauthor=T.+F.+Hendricksonauthor=A.+K.+Patrickauthor=J.+W.+Meadorauthor=R.+A.+Ferreauthor=E.+L.+Brownauthor=C.+E.+Fordauthor=S.+L.+Binfordauthor=S.+T.+Worland&title=Tripeptide+aldehyde+inhibitors+of+human+rhinovirus+3C+protease%3A+design%2C+synthesis%2C+biological+evaluation%2C+and+cocrystal+structure+solution+of+P1+glutamine+isosteric+replacements&doi=10.1021%2Fjm980071x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Tripeptide Aldehyde Inhibitors of Human Rhinovirus 3C Protease: Design, Synthesis, Biological Evaluation, and Cocrystal Structure Solution of P1 Glutamine Isosteric Replacements</span></div><div class="casAuthors">Webber, Stephen E.; Okano, Koji; Little, Thomas L.; Reich, Siegfried H.; Xin, Yue; Fuhrman, Shella A.; Matthews, David A.; Love, Robert A.; Hendrickson, Thomas F.; Patick, Amy K.; Meador, James W., III; Ferre, Rose Ann; Brown, Edward L.; Ford, Clifford E.; Binford, Susan L.; Worland, Stephen T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2786-2805</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The investigation of tripeptide aldehydes I [R = CH2CONH2, NHCHO, NHAc, NHCOEt, NHCOCHMe2, NHCOCF3, NHBz, NMeAc, 2-oxo-1-pyrrolidinyl, 5-isoxazolylcarbonylamino, NHCO2Me, NHBoc, NHSO2Me, NHCONMe2, CH2CONMe2, CH2S(O)Me, CH2CONHPh3, CH2CN] as reversible covalent inhibitors of human rhinovirus (HRV) 3C protease (3CP) is reported.  Mol. models based on the apo crystal structure of HRV-14 3CP and other trypsin-like serine proteases were constructed to approx. the binding of peptide substrates, generate transition state models of P1-P1' amide cleavage, and propose novel tripeptide aldehydes.  Glutaminal derivs. have limitations since they exist predominantly in the cyclic hemiaminal form.  Therefore, several isosteric replacements for the P1 carboxamide side chain were designed and incorporated into the tripeptide aldehydes.  These compds. were found to be potent inhibitors of purified HRV-14 3CP with Kis ranging from 0.005 to 0.64 μM.  Several have low micromolar antiviral activity when tested against HRV-14-infected H1-HeLa cells.  The N-acetyl deriv. I (R = NHAc) was also shown to be active against HRV serotypes 2, 16, and 89.  High-resoln. cocrystal structures of HRV-2 3CP, covalently bound to I [R = NHAc, CH2CONMe2, CH2S(O)Me] were solved.  These cocrystal structures were analyzed and compared with the author's original HRV-14 3CP-substrate and inhibitor models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvPwUsYQ057bVg90H21EOLACvtfcHk0lhm5NHqqZH_oA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXksVKrtrg%253D&md5=4ac8f340209d300aa1dbee0005e3c717</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm980071x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm980071x%26sid%3Dliteratum%253Aachs%26aulast%3DWebber%26aufirst%3DS.%2BE.%26aulast%3DOkano%26aufirst%3DK.%26aulast%3DLittle%26aufirst%3DT.%2BL.%26aulast%3DReich%26aufirst%3DS.%2BH.%26aulast%3DXin%26aufirst%3DY.%26aulast%3DFuhrman%26aufirst%3DS.%2BA.%26aulast%3DMatthews%26aufirst%3DD.%2BA.%26aulast%3DLove%26aufirst%3DR.%2BA.%26aulast%3DHendrickson%26aufirst%3DT.%2BF.%26aulast%3DPatrick%26aufirst%3DA.%2BK.%26aulast%3DMeador%26aufirst%3DJ.%2BW.%26aulast%3DFerre%26aufirst%3DR.%2BA.%26aulast%3DBrown%26aufirst%3DE.%2BL.%26aulast%3DFord%26aufirst%3DC.%2BE.%26aulast%3DBinford%26aufirst%3DS.%2BL.%26aulast%3DWorland%26aufirst%3DS.%2BT.%26atitle%3DTripeptide%2520aldehyde%2520inhibitors%2520of%2520human%2520rhinovirus%25203C%2520protease%253A%2520design%252C%2520synthesis%252C%2520biological%2520evaluation%252C%2520and%2520cocrystal%2520structure%2520solution%2520of%2520P1%2520glutamine%2520isosteric%2520replacements%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26spage%3D2786%26epage%3D2805%26doi%3D10.1021%2Fjm980071x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shepherd, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKinney, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakulchik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villarreal, E. C.</span></span> <span> </span><span class="NLM_article-title">Small peptidic aldehyde inhibitors of human rhinovirus 3C protease</span>. <i>Biorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">2893</span>– <span class="NLM_lpage">2896</span>, <span class="refDoi"> DOI: 10.1016/S0960-894X(96)00537-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1016%2FS0960-894X%2896%2900537-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK2sXhtleksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1996&pages=2893-2896&author=T.+A.+Shepherdauthor=G.+A.+Coxauthor=E.+McKinneyauthor=J.+Tangauthor=M.+Wakulchikauthor=R.+E.+Zimmermanauthor=E.+C.+Villarreal&title=Small+peptidic+aldehyde+inhibitors+of+human+rhinovirus+3C+protease&doi=10.1016%2FS0960-894X%2896%2900537-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Small peptidic aldehyde inhibitors of human rhinovirus 3C protease</span></div><div class="casAuthors">Shepherd, Timothy A.; Cox, Gregory A.; McKinney, Emma; Tang, Joseph; Wakulchik, Mark; Zimmerman, Ronald E.; Villarreal, Elcira C.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">2893-2896</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Small peptide aldehydes were designed to mimic the preferred substrate requirements for the human rhinovirus 3C protease.  Di- and tripeptide aldehydes contg. a methionine sulfone as a P1 surrogate for glutamine show low micromol. enzyme inhibitory and antiviral tissue culture activity.  LY338387, obtained in a short and efficient synthesis, appears to validate the protease as a therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVNHDYhXYZnrVg90H21EOLACvtfcHk0lhm5NHqqZH_oA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXhtleksQ%253D%253D&md5=320b47228b1b730493851bf35eed9b3c</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2896%2900537-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252896%252900537-9%26sid%3Dliteratum%253Aachs%26aulast%3DShepherd%26aufirst%3DT.%2BA.%26aulast%3DCox%26aufirst%3DG.%2BA.%26aulast%3DMcKinney%26aufirst%3DE.%26aulast%3DTang%26aufirst%3DJ.%26aulast%3DWakulchik%26aufirst%3DM.%26aulast%3DZimmerman%26aufirst%3DR.%2BE.%26aulast%3DVillarreal%26aufirst%3DE.%2BC.%26atitle%3DSmall%2520peptidic%2520aldehyde%2520inhibitors%2520of%2520human%2520rhinovirus%25203C%2520protease%26jtitle%3DBiorg.%2520Med.%2520Chem.%2520Lett.%26date%3D1996%26volume%3D6%26spage%3D2893%26epage%3D2896%26doi%3D10.1016%2FS0960-894X%2896%2900537-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaldor, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammond, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dressman, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labus, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chadwell, F. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kline, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinz, B. A.</span></span> <span> </span><span class="NLM_article-title">Glutamine-derived aldehydes for the inhibition of human rhinovirus 3C protease</span>. <i>Biorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">2021</span>– <span class="NLM_lpage">2026</span>, <span class="refDoi"> DOI: 10.1016/0960-894X(95)00345-T</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1016%2F0960-894X%2895%2900345-T" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK2MXnvFKgsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1995&pages=2021-2026&author=S.+W.+Kaldorauthor=M.+Hammondauthor=B.+A.+Dressmanauthor=J.+M.+Labusauthor=F.+W.+Chadwellauthor=A.+D.+Klineauthor=B.+A.+Heinz&title=Glutamine-derived+aldehydes+for+the+inhibition+of+human+rhinovirus+3C+protease&doi=10.1016%2F0960-894X%2895%2900345-T"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Glutamine-derived aldehydes for the inhibition of human rhinovirus 3C protease</span></div><div class="casAuthors">Kaldor, Stephen W.; Hammond, Marlys; Dressman, Bruce A.; Labus, Jean M.; Chadwell, Frederick W.; Kline, Allen D.; Heinz, Beverly A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2021-6</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Peptide aldehydes have been synthesized and evaluated as inhibitors of human rhinovirus 3C protease.  Those inhibitors contg. a C-terminal glutamine aldehyde were prepd. using newly developed methodol. involving redn. of the corresponding glutarimide, which can be easily prepd. from N-protected glutamine.  Low mol. wt. (<500) compds. with low to submicromolar inhibitory activity in both isolated enzyme and in vitro translation assays have been identified.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq70iEZSn1YkrVg90H21EOLACvtfcHk0lizUPJpZOgXdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXnvFKgsrs%253D&md5=79f602bff4045ffa3a223758830d4d32</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2F0960-894X%2895%2900345-T&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0960-894X%252895%252900345-T%26sid%3Dliteratum%253Aachs%26aulast%3DKaldor%26aufirst%3DS.%2BW.%26aulast%3DHammond%26aufirst%3DM.%26aulast%3DDressman%26aufirst%3DB.%2BA.%26aulast%3DLabus%26aufirst%3DJ.%2BM.%26aulast%3DChadwell%26aufirst%3DF.%2BW.%26aulast%3DKline%26aufirst%3DA.%2BD.%26aulast%3DHeinz%26aufirst%3DB.%2BA.%26atitle%3DGlutamine-derived%2520aldehydes%2520for%2520the%2520inhibition%2520of%2520human%2520rhinovirus%25203C%2520protease%26jtitle%3DBiorg.%2520Med.%2520Chem.%2520Lett.%26date%3D1995%26volume%3D5%26spage%3D2021%26epage%3D2026%26doi%3D10.1016%2F0960-894X%2895%2900345-T" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malcolm, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shechosky, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKay, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vederas, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sant, D. V.</span></span> <span> </span><span class="NLM_article-title">Peptide aldehyde inhibitors of hepatitis A virus 3C proteinase</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">8172</span>– <span class="NLM_lpage">8179</span>, <span class="refDoi"> DOI: 10.1021/bi00025a024</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00025a024" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK2MXmtVyit7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1995&pages=8172-8179&author=B.+A.+Malcolmauthor=C.+Loweauthor=S.+Shechoskyauthor=R.+T.+McKayauthor=C.+C.+Yangauthor=V.+J.+Shahauthor=R.+J.+Simonauthor=J.+C.+Vederasauthor=D.+V.+Sant&title=Peptide+aldehyde+inhibitors+of+hepatitis+A+virus+3C+proteinase&doi=10.1021%2Fbi00025a024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide Aldehyde Inhibitors of Hepatitis A Virus 3C Proteinase</span></div><div class="casAuthors">Malcolm, Bruce A.; Lowe, Christopher; Shechosky, Shirley; McKay, Ryan T.; Yang, Chi Ching; Shah, Vibhakar J.; Simon, Reyna J.; Vederas, John C.; Smith, Daniel V.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">8172-9</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Picornaviral 3C proteinases are a group of closely related thiol proteinases responsible for processing of the viral polyprotein into its component proteins.  These proteinases adopt a chymotrypsin-like fold [Allaire et al. (1994) Nature 369, 72-77; Matthews et al. (1994) Cell 77, 761-771] and a display an active-site configuration like those of the serine proteinases.  Peptide-aldehydes based on the preferred peptide substrates for hepatitis A virus (HAV) 3C proteinase (picornain 3C) were synthesized by redn. of a thioester precursor.  Acetyl-Leu-Ala-Ala-(N,N'-dimethylglutaminal) was found to be a reversible, slow-binding inhibitor for HAV 3C with a Ki* of 4.2 × 10-8 M.  This inhibitor showed 50-fold less activity against the highly homologous human rhinovirus (strain 14) 3C proteinase, whose peptide substrate specificity is slightly different, suggesting a high degree of selectivity.  NMR spectrometry of the adduct of the 13C-labeled inhibitor with the HAV-3C proteinase indicate that a thiohemiacetal is formed between the enzyme and the aldehyde carbon as previously noted for peptide-aldehyde inhibitors of papain [Lewis & Wolfenden (1977) Biochem. 16, 4890-4894; Gamcsik et al. (1983) J. Am. Chem. Soc. 105, 6324-6325].  The adduct can also be obsd. by electrospray mass spectrometry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyqMCwJj1TxLVg90H21EOLACvtfcHk0lizUPJpZOgXdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmtVyit7c%253D&md5=8ed831816e5289ceae4879cc67769b46</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fbi00025a024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00025a024%26sid%3Dliteratum%253Aachs%26aulast%3DMalcolm%26aufirst%3DB.%2BA.%26aulast%3DLowe%26aufirst%3DC.%26aulast%3DShechosky%26aufirst%3DS.%26aulast%3DMcKay%26aufirst%3DR.%2BT.%26aulast%3DYang%26aufirst%3DC.%2BC.%26aulast%3DShah%26aufirst%3DV.%2BJ.%26aulast%3DSimon%26aufirst%3DR.%2BJ.%26aulast%3DVederas%26aufirst%3DJ.%2BC.%26aulast%3DSant%26aufirst%3DD.%2BV.%26atitle%3DPeptide%2520aldehyde%2520inhibitors%2520of%2520hepatitis%2520A%2520virus%25203C%2520proteinase%26jtitle%3DBiochemistry%26date%3D1995%26volume%3D34%26spage%3D8172%26epage%3D8179%26doi%3D10.1021%2Fbi00025a024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marquis, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ru, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamashita, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomaszek, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ijames, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janson, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Meguid, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Alessio, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McQueney, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veber, D. F.</span></span> <span> </span><span class="NLM_article-title">Potent dipeptidylketone inhibitors of the cysteine protease cathepsin K</span>. <i>Biorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">581</span>– <span class="NLM_lpage">588</span>, <span class="refDoi"> DOI: 10.1016/S0968-0896(99)00011-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1016%2FS0968-0896%2899%2900011-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10353637" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK1MXis12mt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1999&pages=581-588&author=R.+W.+Marquisauthor=Y.+Ruauthor=D.+S.+Yamashitaauthor=H.-J.+Ohauthor=J.+Yenauthor=S.+K.+Thompsonauthor=T.+J.+Carrauthor=M.+A.+Levyauthor=T.+A.+Tomaszekauthor=C.+F.+Ijamesauthor=W.+W.+Smithauthor=B.+Zhaoauthor=C.+A.+Jansonauthor=S.+S.+Abdel-Meguidauthor=K.+J.+D%E2%80%99Alessioauthor=M.+S.+McQueneyauthor=D.+F.+Veber&title=Potent+dipeptidylketone+inhibitors+of+the+cysteine+protease+cathepsin+K&doi=10.1016%2FS0968-0896%2899%2900011-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Potent dipeptidylketone inhibitors of the cysteine protease cathepsin K</span></div><div class="casAuthors">Marquis, Robert W.; Ru, Yu; Yamashita, Dennis S.; Oh, Hye-Ja; Yen, Jack; Thompson, Scott K.; Carr, Thomas J.; Levy, Mark A.; Tomaszek, Thaddeus A.; Ijames, Carl F.; Smith, Ward W.; Zhao, Baoguang; Janson, Cheryl A.; Abdel-Meguid, Sherin S.; D'Alessio, Karla J.; McQueney, M. S.; Veber, Daniel F.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">581-588</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Cathepsin K (EC 3.4.22.38) is a cysteine protease of the papain superfamily which is selectively expressed within the osteoclast.  Several lines of evidence have pointed to the fact that this protease may play an important role in the degrdn. of the bone matrix.  Potent and selective inhibitors of cathepsin K could be important therapeutic agents for the control of excessive bone resorption.  Recently a series of peptide aldehydes have been shown to be potent inhibitors of cathepsin K.  In an effort to design more selective and metabolically stable inhibitors of cathepsin K, a series of electronically attenuated alkoxymethylketones and thiomethylketones inhibitors, e.g. I [R = iso-Bu, (CH2)4NH2, Me, CH2OH; X = OMe, OCH2Ph-4-Ph, SPh, SCH2Ph] have been synthesized.  The X-ray co-crystal structure of one of these analogs in complex with cathepsin K shows the inhibitor binding in the primed side of the enzyme active site with a covalent interaction between the active site cysteine 25 and the carbonyl carbon of the inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqScu6Z7mFzxrVg90H21EOLACvtfcHk0lizUPJpZOgXdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXis12mt7o%253D&md5=592914f0ef29bf154bd29e1a057def38</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2FS0968-0896%2899%2900011-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0968-0896%252899%252900011-5%26sid%3Dliteratum%253Aachs%26aulast%3DMarquis%26aufirst%3DR.%2BW.%26aulast%3DRu%26aufirst%3DY.%26aulast%3DYamashita%26aufirst%3DD.%2BS.%26aulast%3DOh%26aufirst%3DH.-J.%26aulast%3DYen%26aufirst%3DJ.%26aulast%3DThompson%26aufirst%3DS.%2BK.%26aulast%3DCarr%26aufirst%3DT.%2BJ.%26aulast%3DLevy%26aufirst%3DM.%2BA.%26aulast%3DTomaszek%26aufirst%3DT.%2BA.%26aulast%3DIjames%26aufirst%3DC.%2BF.%26aulast%3DSmith%26aufirst%3DW.%2BW.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DJanson%26aufirst%3DC.%2BA.%26aulast%3DAbdel-Meguid%26aufirst%3DS.%2BS.%26aulast%3DD%25E2%2580%2599Alessio%26aufirst%3DK.%2BJ.%26aulast%3DMcQueney%26aufirst%3DM.%2BS.%26aulast%3DVeber%26aufirst%3DD.%2BF.%26atitle%3DPotent%2520dipeptidylketone%2520inhibitors%2520of%2520the%2520cysteine%2520protease%2520cathepsin%2520K%26jtitle%3DBiorg.%2520Med.%2520Chem.%26date%3D1999%26volume%3D7%26spage%3D581%26epage%3D588%26doi%3D10.1016%2FS0968-0896%2899%2900011-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellman, J. A.</span></span> <span> </span><span class="NLM_article-title">General solid-phase method to prepare novel cyclic ketone inhibitors of the cysteine protease cruzain</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2993</span>, <span class="refDoi"> DOI: 10.1016/S0960-894X(02)00619-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1016%2FS0960-894X%2802%2900619-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=12270191" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BD38Xntlansr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2002&pages=2993&author=L.+Huangauthor=J.+A.+Ellman&title=General+solid-phase+method+to+prepare+novel+cyclic+ketone+inhibitors+of+the+cysteine+protease+cruzain&doi=10.1016%2FS0960-894X%2802%2900619-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">General solid-phase method to prepare novel cyclic ketone inhibitors of the cysteine protease cruzain</span></div><div class="casAuthors">Huang, Lily; Ellman, Jonathan A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2993-2996</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A series of constrained ketone-based inhibitors has been developed that show low nanomolar Ki values.  These ketone inhibitors showed promising activity towards cruzain, the cysteine protease implicated in Chagas' disease.  This series of constrained inhibitors, which can be accessed quickly and efficiently using a solid-phase combinatorial strategy, should be applicable to other members of the cysteine protease class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPoZFsUb1mHrVg90H21EOLACvtfcHk0lir2qdenNhe5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xntlansr4%253D&md5=0f84ccaeaca3bbd862ac577f6ee83c7b</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2802%2900619-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252802%252900619-4%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DEllman%26aufirst%3DJ.%2BA.%26atitle%3DGeneral%2520solid-phase%2520method%2520to%2520prepare%2520novel%2520cyclic%2520ketone%2520inhibitors%2520of%2520the%2520cysteine%2520protease%2520cruzain%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2002%26volume%3D12%26spage%3D2993%26doi%3D10.1016%2FS0960-894X%2802%2900619-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamashita, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janson, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomaszek, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bossard, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McQueney, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Alessio, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amegadzie, B. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanning, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Meguid, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DesJarlais, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gleason, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veber, D. F.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Structure and design of potent and selective cathepsin K inhibitors</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">11351</span>– <span class="NLM_lpage">11352</span>, <span class="refDoi"> DOI: 10.1021/ja972204u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja972204u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK2sXnsVSlt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=1997&pages=11351-11352&author=D.+S.+Yamashitaauthor=W.+W.+Smithauthor=B.+Zhaoauthor=C.+A.+Jansonauthor=T.+A.+Tomaszekauthor=C.+A.+Bossardauthor=M.+A.+Levyauthor=H.-J.+Ohauthor=S.+A.+Carrauthor=M.+McQueneyauthor=K.+J.+D%E2%80%99Alessioauthor=B.+Y.+Amegadzieauthor=C.+R.+Hanningauthor=S.+Abdel-Meguidauthor=R.+L.+DesJarlaisauthor=J.+G.+Gleasonauthor=D.+F.+Veber&title=Structure+and+design+of+potent+and+selective+cathepsin+K+inhibitors&doi=10.1021%2Fja972204u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and Design of Potent and Selective Cathepsin K Inhibitors</span></div><div class="casAuthors">Yamashita, Dennis S.; Smith, Ward W.; Zhao, Baoguang; Janson, Cheryl A.; Tomaszek, Thaddeus A.; Bossard, Mary J.; Levy, Mark A.; Oh, Hye-Ja; Carr, Thomas J.; Thompson, Scott K.; Ijames, Carl F.; Carr, Steven A.; McQueney, Michael; D'Alessio, Karla J.; Amegadzie, Bernard Y.; Hanning, Charles R.; Abdel-Meguid, Sherin; DesJarlais, Renee L.; Gleason, John G.; Veber, Daniel F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">11351-11352</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The sym. diacylaminomethyl ketone, Cbz-Leu-NHCH2COCH2NH-(Cbz-Leu) (I), is a reversible, time-independent, selective inhibitor of the papain-related, osteoclast-specific cysteine protease, cathepsin K (Ki,app cathepsin K = 22 nM, Ki,app papain, >10 μM, Ki,app cathepsin L = 0.34 μM, cathepsin B = 1.3 μM, cathepsin S = 0.89 μM, Ki,app trypsin >50 μM, Ki,app chymotrypsin >50 μM).  An X-ray co-crystal structure of I bound to cathepsin K shows addn. of the active site thiol to the ketone and specific binding on both sides of the active site.  The effects of C- and N-methylation of the I on potency against cathepsin K were examd.  Also, the diacylaminomethyl ketone scaffold was incorporated into a conformationally constraining pyrrolidinone ring and the binding orientation of this ketone on cathepsin K was detd. by X-ray co-crystallog.  The 4-phenoxy Ph sulfonamide was found to be a Cbz-Leucine amide mimetic.  An X-ray co-crystal structure of this mimetic compd. bound to cathepsin K was solved.  The α,α'-diaminomethyl ketone nucleus may be a general scaffold for inhibition of cysteine proteases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9nqPi1iGo47Vg90H21EOLACvtfcHk0lir2qdenNhe5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXnsVSlt78%253D&md5=2c459df27f8d7fabdad2b1df913c1020</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fja972204u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja972204u%26sid%3Dliteratum%253Aachs%26aulast%3DYamashita%26aufirst%3DD.%2BS.%26aulast%3DSmith%26aufirst%3DW.%2BW.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DJanson%26aufirst%3DC.%2BA.%26aulast%3DTomaszek%26aufirst%3DT.%2BA.%26aulast%3DBossard%26aufirst%3DC.%2BA.%26aulast%3DLevy%26aufirst%3DM.%2BA.%26aulast%3DOh%26aufirst%3DH.-J.%26aulast%3DCarr%26aufirst%3DS.%2BA.%26aulast%3DMcQueney%26aufirst%3DM.%26aulast%3DD%25E2%2580%2599Alessio%26aufirst%3DK.%2BJ.%26aulast%3DAmegadzie%26aufirst%3DB.%2BY.%26aulast%3DHanning%26aufirst%3DC.%2BR.%26aulast%3DAbdel-Meguid%26aufirst%3DS.%26aulast%3DDesJarlais%26aufirst%3DR.%2BL.%26aulast%3DGleason%26aufirst%3DJ.%2BG.%26aulast%3DVeber%26aufirst%3DD.%2BF.%26atitle%3DStructure%2520and%2520design%2520of%2520potent%2520and%2520selective%2520cathepsin%2520K%2520inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1997%26volume%3D119%26spage%3D11351%26epage%3D11352%26doi%3D10.1021%2Fja972204u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moon, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanzlik, R. P.</span></span> <span> </span><span class="NLM_article-title">Reversible covalent inhibition of papain by a peptide nitrile. Carbon-13 NMR evidence for a thioimidate ester adduct</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1986</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">1350</span>– <span class="NLM_lpage">1351</span>, <span class="refDoi"> DOI: 10.1021/ja00266a066</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00266a066" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaL28Xht1Gmtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=1986&pages=1350-1351&author=J.+B.+Moonauthor=R.+S.+Colemanauthor=R.+P.+Hanzlik&title=Reversible+covalent+inhibition+of+papain+by+a+peptide+nitrile.+Carbon-13+NMR+evidence+for+a+thioimidate+ester+adduct&doi=10.1021%2Fja00266a066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible covalent inhibition of papain by a peptide nitrile.  Carbon-13 NMR evidence for a thioimidate ester adduct</span></div><div class="casAuthors">Moon, Joseph B.; Coleman, Robert S.; Hanzlik, Robert P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1350-1</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Acetyl-L-phenylalaninylglcyine nitrile (I) is a powerful competitive inhibitor of papain (Ki = 1 μM).  [Nitrile-13C]-I was synthesized, and its interaction with papain was obsd. by 13C NMR spectroscopy.  The 13C NMR spectrum of the free ligand shows a single resonance at 116.61 ppm.  In the presence of papain (∼0.65 molar equivalents), a new signal at 182.08 ppm is seen; this signal disappears immediately when the soln. is acidified and treated with 2,2'-dipyridyl disulfide, which reacts irreversibly with the active-site SH of papain.  Thus, assocn. of I with papain involves reversible formation of a covalent thioimidate bond with the active-site SH of the enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCxv9ppgcPL7Vg90H21EOLACvtfcHk0lir2qdenNhe5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL28Xht1Gmtbk%253D&md5=94a9e892576d62c31a51b030f1cf7a17</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fja00266a066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00266a066%26sid%3Dliteratum%253Aachs%26aulast%3DMoon%26aufirst%3DJ.%2BB.%26aulast%3DColeman%26aufirst%3DR.%2BS.%26aulast%3DHanzlik%26aufirst%3DR.%2BP.%26atitle%3DReversible%2520covalent%2520inhibition%2520of%2520papain%2520by%2520a%2520peptide%2520nitrile.%2520Carbon-13%2520NMR%2520evidence%2520for%2520a%2520thioimidate%2520ester%2520adduct%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1986%26volume%3D108%26spage%3D1350%26epage%3D1351%26doi%3D10.1021%2Fja00266a066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cowen, S. D.</span>; <span class="NLM_string-name">Greenspan, D. P.</span>; <span class="NLM_string-name">McQuire, L. W.</span>; <span class="NLM_string-name">Tommasi, A. R.</span>; <span class="NLM_string-name">Van Duzer, J. H.</span></span> <span> </span><span class="NLM_article-title">N-substituted peptidyl nitriles as cysteine cathepsin inhibitors</span>. <span class="NLM_patent">WO01/87828 A1</span>, publication date Nov 22, <span class="NLM_year">2001</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2001&author=S.+D.+Cowen&author=D.+P.+Greenspan&author=L.+W.+McQuire&author=A.+R.+Tommasi&author=J.+H.+Van+Duzer&title=N-substituted+peptidyl+nitriles+as+cysteine+cathepsin+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DCowen%26aufirst%3DS.%2BD.%26atitle%3DN-substituted%2520peptidyl%2520nitriles%2520as%2520cysteine%2520cathepsin%2520inhibitors%26date%3D2001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patil, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golubski, Z. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hori, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tehrani, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foreman, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eveleth, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartus, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, J. C.</span></span> <span> </span><span class="NLM_article-title">Peptide alpha-keto ester, alpha-keto amide, and alpha-keto acid inhibitors of calpains and other cysteine proteases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">3472</span>– <span class="NLM_lpage">3480</span>, <span class="refDoi"> DOI: 10.1021/jm00074a031</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00074a031" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK3sXmsVCitbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1993&pages=3472-3480&author=Z.+Liauthor=G.+S.+Patilauthor=Z.+E.+Golubskiauthor=H.+Horiauthor=K.+Tehraniauthor=J.+E.+Foremanauthor=D.+D.+Evelethauthor=R.+T.+Bartusauthor=J.+C.+Powers&title=Peptide+alpha-keto+ester%2C+alpha-keto+amide%2C+and+alpha-keto+acid+inhibitors+of+calpains+and+other+cysteine+proteases&doi=10.1021%2Fjm00074a031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide α-keto ester, α-keto amide, and α-keto acid inhibitors of calpains and other cysteine proteases</span></div><div class="casAuthors">Li, Zhaozhao; Patil, Girish S.; Golubski, Zbigniew E.; Hori, Hitoshi; Tehrani, Kamin; Foreman, J. E.; Eveleth, David D.; Bartus, Raymond T.; Powers, James C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">3472-80</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">A series of dipeptidyl and tripeptidyl α-keto esters, α-keto amides, and α-keto acids having leucine in the P2 position were synthesized and evaluated as inhibitors for the cysteine proteases calpain I, calpain II, cathepsin B, and papain.  In general, peptidyl α-keto acids, were more inhibitory toward calpain I and II than α-keto amides, which in turn were more effective than α-keto esters.  In the series Z-Leu-AA-COOEt, the inhibitory potency decreased in the order: Met (lowest KI) > Nva > Phe > 4-Cl-Phe > Abu > Nle (highest KI) with calpain I, while almost the reverse order was obsd. for calpain II.  Extending the dipeptide α-keto ester to a tripeptide α-keto ester yielded significant enhancement in the inhibitory potency toward cathepsin B, but smaller changes toward the calpains.  Changing the ester group in the α-keto esters did not substantially decrease KI values for calpain I and calpain II.  N-monosubstituted α-keto amides were better inhibitors than the corresponding α-keto esters.  α-Keto amides with hydrophobic alkyl groups or alkyl groups with an attached Ph group had the lower KI values.  N,N-disubstituted α-keto amides were much less potent inhibitors than the corresponding N-monosubstituted peptide α-keto amides.  The peptide α-keto acid Z-Leu-Phe-COOH was the best inhibitor for calpain I (KI = 0.0085 μM) and calpain II (KI = 0.0057 μM) discovered in this study.  It is likely that the inhibitors are transition-state analogs and form tetrahedral adducts with the active site cysteine of cysteine proteases and form hydrogen bonds with the active site histidine and possibly another hydrogen bond donor in the case of monosubstituted amides.  Several inhibitors prevented spectrin degrdn. in a platelet membrane permeability assay and may be useful for the treatment of diseases which involve neurodegeneration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5VQ5w26OSKrVg90H21EOLACvtfcHk0liyVS_ErMh28w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXmsVCitbo%253D&md5=f73638cf2385179be60d4d22ce091190</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm00074a031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00074a031%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DPatil%26aufirst%3DG.%2BS.%26aulast%3DGolubski%26aufirst%3DZ.%2BE.%26aulast%3DHori%26aufirst%3DH.%26aulast%3DTehrani%26aufirst%3DK.%26aulast%3DForeman%26aufirst%3DJ.%2BE.%26aulast%3DEveleth%26aufirst%3DD.%2BD.%26aulast%3DBartus%26aufirst%3DR.%2BT.%26aulast%3DPowers%26aufirst%3DJ.%2BC.%26atitle%3DPeptide%2520alpha-keto%2520ester%252C%2520alpha-keto%2520amide%252C%2520and%2520alpha-keto%2520acid%2520inhibitors%2520of%2520calpains%2520and%2520other%2520cysteine%2520proteases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1993%26volume%3D36%26spage%3D3472%26epage%3D3480%26doi%3D10.1021%2Fjm00074a031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Victor, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinz, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakulchik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q. M.</span></span> <span> </span><span class="NLM_article-title">Synthesis and evaluation of tripeptidyl alpha-ketoamides as human rhinovirus 3C protease inhibitors</span>. <i>Biorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">3531</span>– <span class="NLM_lpage">3536</span>, <span class="refDoi"> DOI: 10.1016/S0960-894X(03)00753-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1016%2FS0960-894X%2803%2900753-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=14505664" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnsVKgtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=3531-3536&author=S.-H.+Chenauthor=J.+Lamarauthor=F.+Victorauthor=N.+Snyderauthor=R.+Johnsonauthor=B.+A.+Heinzauthor=M.+Wakulchikauthor=Q.+M.+Wang&title=Synthesis+and+evaluation+of+tripeptidyl+alpha-ketoamides+as+human+rhinovirus+3C+protease+inhibitors&doi=10.1016%2FS0960-894X%2803%2900753-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and evaluation of tripeptidyl α-ketoamides as human rhinovirus 3C protease inhibitors</span></div><div class="casAuthors">Chen, Shu-Hui; Lamar, Jason; Victor, Frantz; Snyder, Nancy; Johnson, Robert; Heinz, Beverly A.; Wakulchik, Mark; Wang, Q. May</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3531-3536</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">We describe the synthesis and biol. evaluation of a series of tripeptidyl α-ketoamides as human rhinovirus (HRV) 3C protease inhibitors.  The most potent inhibitor Me2CHO2C-L-Val-L-Phe-L-NVal-COCHMePh-(S) exhibited impressive enzyme inhibitory activity as well as antiviral activity against HRV-14.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGYlHVZR46lrVg90H21EOLACvtfcHk0liyVS_ErMh28w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnsVKgtrs%253D&md5=ce59b2bcb3e78ffa458b3746386294d3</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2803%2900753-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252803%252900753-4%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.-H.%26aulast%3DLamar%26aufirst%3DJ.%26aulast%3DVictor%26aufirst%3DF.%26aulast%3DSnyder%26aufirst%3DN.%26aulast%3DJohnson%26aufirst%3DR.%26aulast%3DHeinz%26aufirst%3DB.%2BA.%26aulast%3DWakulchik%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DQ.%2BM.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520tripeptidyl%2520alpha-ketoamides%2520as%2520human%2520rhinovirus%25203C%2520protease%2520inhibitors%26jtitle%3DBiorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2003%26volume%3D13%26spage%3D3531%26epage%3D3536%26doi%3D10.1016%2FS0960-894X%2803%2900753-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otto, H.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirmeister, T.</span></span> <span> </span><span class="NLM_article-title">Cysteine proteases and their inhibitors</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">172</span>, <span class="refDoi"> DOI: 10.1021/cr950025u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr950025u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK2sXktFOhsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=1997&pages=133-172&author=H.-H.+Ottoauthor=T.+Schirmeister&title=Cysteine+proteases+and+their+inhibitors&doi=10.1021%2Fcr950025u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Cysteine Proteases and Their Inhibitors</span></div><div class="casAuthors">Otto, Hans-Hartwig; Schirmeister, Tanja</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">133-171</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with 599 refs.  Some typical examples of cysteine proteases from different sources are characterized in terms of property, structure and specificity.  Possible functions of human and mammalian cysteine proteases, lysosomal cathepsins and cytoplasmic calpains are discussed, as well as their role in pathol. processes.  In addn. to the endogenous inhibitors, the cystatins and calpastatins, the currently available synthetic inhibitors are also described together with their structures, reaction mechanisms, activities and selectivities.  Potential medical applications of cysteine protease inhibitors are discussed.  Particular emphasis is given to comparison of cysteine proteases with serine proteases, due to their similar proteolytic mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBgmD-cFH7IrVg90H21EOLACvtfcHk0liyVS_ErMh28w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXktFOhsA%253D%253D&md5=0c38c0f09f14ad664b6ca707ce08a5b9</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fcr950025u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr950025u%26sid%3Dliteratum%253Aachs%26aulast%3DOtto%26aufirst%3DH.-H.%26aulast%3DSchirmeister%26aufirst%3DT.%26atitle%3DCysteine%2520proteases%2520and%2520their%2520inhibitors%26jtitle%3DChem.%2520Rev.%26date%3D1997%26volume%3D97%26spage%3D133%26epage%3D172%26doi%3D10.1021%2Fcr950025u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gamcsik, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malthouse, J. P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Primrose, W. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackenzie, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, A. S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, A. I.</span></span> <span> </span><span class="NLM_article-title">Structure and stereochemistry of tetrahedral inhibitor complexes of papain by direct NMR observation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">6324</span>– <span class="NLM_lpage">6325</span>, <span class="refDoi"> DOI: 10.1021/ja00358a030</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00358a030" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaL3sXltlKjs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=1983&pages=6324-6325&author=M.+P.+Gamcsikauthor=J.+P.+G.+Malthouseauthor=W.+U.+Primroseauthor=N.+E.+Mackenzieauthor=A.+S.+F.+Boydauthor=R.+A.+Russellauthor=A.+I.+Scott&title=Structure+and+stereochemistry+of+tetrahedral+inhibitor+complexes+of+papain+by+direct+NMR+observation&doi=10.1021%2Fja00358a030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and stereochemistry of tetrahedral inhibitor complexes of papain by direct NMR observation</span></div><div class="casAuthors">Gamcsik, Michael P.; Malthouse, J. Paul G.; Primrose, William U.; Mackenzie, Neil E.; Boyd, Alan S. F.; Russell, Richard A.; Scott, A. Ian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6324-5</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">The site-specific inhibitor of papain, N-acetylphenylalanylglycinal, was synthesized with the aldehyde C atom enriched (90%) with 13C.  The resultant inhibitor complex had the structure of a tetrahedral hemithioacetal which, under high-resoln. 13C NMR observation, could be resolved into species corresponding to diastereoisomers of the acetal, showing that covalent complex formation proceeded with lack of stereospecificity at the active site of the enzyme.  Differentiation of the relative stabilities of the 2 isomers was possible by titrn. with 2,2'-dipyridyl disulfide and a tentative assignment of abs. stereochem. to each isomer could thus be made.  These expts. provide a novel probe for mapping the active site of cysteine (and hence serine) proteases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsRAF-1roQt7Vg90H21EOLACvtfcHk0liyVS_ErMh28w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXltlKjs78%253D&md5=1f13f3c27eb8027009369c694c9f0c1b</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fja00358a030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00358a030%26sid%3Dliteratum%253Aachs%26aulast%3DGamcsik%26aufirst%3DM.%2BP.%26aulast%3DMalthouse%26aufirst%3DJ.%2BP.%2BG.%26aulast%3DPrimrose%26aufirst%3DW.%2BU.%26aulast%3DMackenzie%26aufirst%3DN.%2BE.%26aulast%3DBoyd%26aufirst%3DA.%2BS.%2BF.%26aulast%3DRussell%26aufirst%3DR.%2BA.%26aulast%3DScott%26aufirst%3DA.%2BI.%26atitle%3DStructure%2520and%2520stereochemistry%2520of%2520tetrahedral%2520inhibitor%2520complexes%2520of%2520papain%2520by%2520direct%2520NMR%2520observation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1983%26volume%3D105%26spage%3D6324%26epage%3D6325%26doi%3D10.1021%2Fja00358a030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusov, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Brunn, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leyssen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanko, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neyts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Wilde, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snijder, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span> <span> </span><span class="NLM_article-title">α-Ketoamides as broad-spectrum inhibitors of coronavirus and enterovirus replication: Structure-based design, synthesis, and activity assessment</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">4562</span>– <span class="NLM_lpage">4578</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01828</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01828" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BB3cXivValt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=4562-4578&author=L.+Zhangauthor=D.+Linauthor=Y.+Kusovauthor=Y.+Nianauthor=Q.+Maauthor=J.+Wangauthor=A.+von+Brunnauthor=P.+Leyssenauthor=K.+Lankoauthor=J.+Neytsauthor=A.+de+Wildeauthor=E.+Snijderauthor=H.+Liuauthor=R.+Hilgenfeld&title=%CE%B1-Ketoamides+as+broad-spectrum+inhibitors+of+coronavirus+and+enterovirus+replication%3A+Structure-based+design%2C+synthesis%2C+and+activity+assessment&doi=10.1021%2Facs.jmedchem.9b01828"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">α-Ketoamides as Broad-Spectrum Inhibitors of Coronavirus and Enterovirus Replication: Structure-Based Design, Synthesis, and Activity Assessment</span></div><div class="casAuthors">Zhang, Linlin; Lin, Daizong; Kusov, Yuri; Nian, Yong; Ma, Qingjun; Wang, Jiang; von Brunn, Albrecht; Leyssen, Pieter; Lanko, Kristina; Neyts, Johan; de Wilde, Adriaan; Snijder, Eric J.; Liu, Hong; Hilgenfeld, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4562-4578</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The main protease of coronaviruses and the 3C protease of enteroviruses share a similar active-site architecture and a unique requirement for glutamine in the P1 position of the substrate.  Because of their unique specificity and essential role in viral polyprotein processing, these proteases are suitable targets for the development of antiviral drugs.  In order to obtain near-equipotent, broad-spectrum antivirals against alphacoronaviruses, betacoronaviruses, and enteroviruses, we pursued structure-based design of peptidomimetic α-ketoamides as inhibitors of main and 3C proteases.  Six crystal structures of protease:inhibitor complexes were detd. as part of this study.  Compds. synthesized were tested against the recombinant proteases as well as in viral replicons and virus-infected cell cultures; most of them were not cell-toxic.  Optimization of the P2 substituent of the α-ketoamides proved crucial for achieving near-equipotency against the three virus genera.  The best near-equipotent inhibitors, (I) [R1 = cyclopentylmethyl (P2) and R1 = cyclohexylmethyl (P2)], display low-micromolar EC50 values against enteroviruses, alphacoronaviruses, and betacoronaviruses in cell cultures.  In Huh7 cells, I [R1 = cyclohexylmethyl (P2)] exhibits three-digit picomolar activity against Middle East Respiratory Syndrome coronavirus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWpcfl4wTRLLVg90H21EOLACvtfcHk0lgA39a_nI7l9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXivValt7o%253D&md5=60d91a8ecf916856a844d835ecf84d5f</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01828&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01828%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DD.%26aulast%3DKusov%26aufirst%3DY.%26aulast%3DNian%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3Dvon%2BBrunn%26aufirst%3DA.%26aulast%3DLeyssen%26aufirst%3DP.%26aulast%3DLanko%26aufirst%3DK.%26aulast%3DNeyts%26aufirst%3DJ.%26aulast%3Dde%2BWilde%26aufirst%3DA.%26aulast%3DSnijder%26aufirst%3DE.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3D%25CE%25B1-Ketoamides%2520as%2520broad-spectrum%2520inhibitors%2520of%2520coronavirus%2520and%2520enterovirus%2520replication%253A%2520Structure-based%2520design%252C%2520synthesis%252C%2520and%2520activity%2520assessment%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D4562%26epage%3D4578%26doi%3D10.1021%2Facs.jmedchem.9b01828" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Powers, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asgian, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ekici, O. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, K. E.</span></span> <span> </span><span class="NLM_article-title">Irreversible inhibitors of serine, cysteine, and threonine proteases</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">4639</span>– <span class="NLM_lpage">4750</span>, <span class="refDoi"> DOI: 10.1021/cr010182v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr010182v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BD38XosFSnsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2002&pages=4639-4750&author=J.+C.+Powersauthor=J.+L.+Asgianauthor=O.+D.+Ekiciauthor=K.+E.+James&title=Irreversible+inhibitors+of+serine%2C+cysteine%2C+and+threonine+proteases&doi=10.1021%2Fcr010182v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Irreversible inhibitors of serine, cysteine, and threonine proteases</span></div><div class="casAuthors">Powers, James C.; Asgian, Juliana L.; Ekici, Oezlem Dogan; James, Karen Ellis</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4639-4750</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Topics discussed include alkylating, acylating, phosphonylating, and sulfonylating agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorivaxGhmhirVg90H21EOLACvtfcHk0lgA39a_nI7l9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XosFSnsr4%253D&md5=eb21ff0297b87537a7d459c92f13c75d</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fcr010182v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr010182v%26sid%3Dliteratum%253Aachs%26aulast%3DPowers%26aufirst%3DJ.%2BC.%26aulast%3DAsgian%26aufirst%3DJ.%2BL.%26aulast%3DEkici%26aufirst%3DO.%2BD.%26aulast%3DJames%26aufirst%3DK.%2BE.%26atitle%3DIrreversible%2520inhibitors%2520of%2520serine%252C%2520cysteine%252C%2520and%2520threonine%2520proteases%26jtitle%3DChem.%2520Rev.%26date%3D2002%26volume%3D102%26spage%3D4639%26epage%3D4750%26doi%3D10.1021%2Fcr010182v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copp, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coles, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pauls, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spencer, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heard, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krantz, A.</span></span> <span> </span><span class="NLM_article-title">New inhibitors of cysteine proteinases. Peptidyl acyloxymethyl ketones and the quiescent nucleofuge strategy</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">4429</span>– <span class="NLM_lpage">4431</span>, <span class="refDoi"> DOI: 10.1021/ja00221a062</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00221a062" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaL1cXktlCguro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=1988&pages=4429-4431&author=R.+A.+Smithauthor=L.+J.+Coppauthor=P.+J.+Colesauthor=H.+W.+Paulsauthor=V.+J.+Robinsonauthor=R.+W.+Spencerauthor=S.+B.+Heardauthor=A.+Krantz&title=New+inhibitors+of+cysteine+proteinases.+Peptidyl+acyloxymethyl+ketones+and+the+quiescent+nucleofuge+strategy&doi=10.1021%2Fja00221a062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">New inhibitors of cysteine proteinases.  Peptidyl acyloxymethyl ketones and the quiescent nucleofuge strategy</span></div><div class="casAuthors">Smith, Roger A.; Copp, Leslie J.; Coles, Peter J.; Pauls, Henry W.; Robinson, Valerie J.; Spencer, Robin W.; Heard, Stephen B.; Krantz, Allen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4429-31</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Peptidyl acyloxymethyl ketones are potent and specific inhibitors of the cysteine proteinase, cathepsin B.  These inhibitors are affinity labels in which the peptide moiety serves to transport an aryl carboxylate nucleofuge that is uniquely reactive toward the active-site cysteine thiol of the target enzyme.  Heretofore, in constructing affinity labels the choice of leaving group was severely restricted.  The aryl carboxylate group offers considerable variation as a design element in that its binding affinity and reactivity can be controlled by substituent effects.  The apparent 2nd-order rate const. which characterizes the inhibition of bovine spleen cathepsin B by this series spans several orders of magnitude and in certain cases exceeds 106 M-1 s-1.  The inhibition is time-dependent, active-site directed, and irreversible.  The inactivation requires that the peptide component contain high affinity recognition elements for this enzyme and that the aryl carboxylate leaving group has a pKa <4.  Impressive inhibitory activity is also obsd. for certain peptidyl aryloxymethyl ketones.  NMR resonances of the inactivated adduct resulting from the action of representative 13C-labeled acyloxymethyl and aryloxymethyl ketone inhibitors on the model cysteine proteinase papain are consistent with the proposed mechanism of inactivation, involving displacement of the nucleofuge by the active-site cysteine to give a (cysteine-25)-thiomethyl ketone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0tvSCUqAffrVg90H21EOLACvtfcHk0lgA39a_nI7l9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXktlCguro%253D&md5=12b38defc0f9731ff6adeebac78b0711</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fja00221a062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00221a062%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DR.%2BA.%26aulast%3DCopp%26aufirst%3DL.%2BJ.%26aulast%3DColes%26aufirst%3DP.%2BJ.%26aulast%3DPauls%26aufirst%3DH.%2BW.%26aulast%3DRobinson%26aufirst%3DV.%2BJ.%26aulast%3DSpencer%26aufirst%3DR.%2BW.%26aulast%3DHeard%26aufirst%3DS.%2BB.%26aulast%3DKrantz%26aufirst%3DA.%26atitle%3DNew%2520inhibitors%2520of%2520cysteine%2520proteinases.%2520Peptidyl%2520acyloxymethyl%2520ketones%2520and%2520the%2520quiescent%2520nucleofuge%2520strategy%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1988%26volume%3D110%26spage%3D4429%26epage%3D4431%26doi%3D10.1021%2Fja00221a062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dolle, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoyer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, C. V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helaszek, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ator, M. A.</span></span> <span> </span><span class="NLM_article-title">P1 aspartate-based peptide alpha-((2,6-dichlorobenzoyl)oxy)methyl ketones as potent time-dependent inhibitors of interleukin-1 beta-converting enzyme</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">563</span>– <span class="NLM_lpage">564</span>, <span class="refDoi"> DOI: 10.1021/jm00031a003</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00031a003" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK2cXhsFaqu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1994&pages=563-564&author=R.+E.+Dolleauthor=D.+Hoyerauthor=C.+V.+C.+Prasadauthor=S.+J.+Schmidtauthor=C.+T.+Helaszekauthor=R.+E.+Millerauthor=M.+A.+Ator&title=P1+aspartate-based+peptide+alpha-%28%282%2C6-dichlorobenzoyl%29oxy%29methyl+ketones+as+potent+time-dependent+inhibitors+of+interleukin-1+beta-converting+enzyme&doi=10.1021%2Fjm00031a003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">P1 Aspartate-Based Peptide α-((2,6-Dichlorobenzoyl)oxy)methyl Ketones as Potent Time-Dependent Inhibitors of Interleukin-1β-Converting Enzyme</span></div><div class="casAuthors">Dolle, Roland E.; Hoyer, Denton; Prasad, C. V. C.; Schmidt, Stanley J.; Helaszek, Carla T.; Miller, Robert E.; Ator, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">563-4</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">Peptide-based α-((2,6-dichlorobenzoyl)oxy)methyl ketones which contain a P1L-aspartic acid residue were synthesized and evaluated as time-dependent inhibitors of interleukin-1β-converting enzyme.  Structure-activity relations are discussed.  The ((arylacyl)oxy)methyl ketone displayed the highest selectivity for the enzyme vs. cathepsin B, whereas the more potent Me ketones demonstrated less preference for interleukin-1β-converting enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpi_XwuRN-QKLVg90H21EOLACvtfcHk0lhRIir9ozk6qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXhsFaqu7s%253D&md5=007ff60fdfd21b80393083e93bfeb6fb</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm00031a003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00031a003%26sid%3Dliteratum%253Aachs%26aulast%3DDolle%26aufirst%3DR.%2BE.%26aulast%3DHoyer%26aufirst%3DD.%26aulast%3DPrasad%26aufirst%3DC.%2BV.%2BC.%26aulast%3DSchmidt%26aufirst%3DS.%2BJ.%26aulast%3DHelaszek%26aufirst%3DC.%2BT.%26aulast%3DMiller%26aufirst%3DR.%2BE.%26aulast%3DAtor%26aufirst%3DM.%2BA.%26atitle%3DP1%2520aspartate-based%2520peptide%2520alpha-%2528%25282%252C6-dichlorobenzoyl%2529oxy%2529methyl%2520ketones%2520as%2520potent%2520time-dependent%2520inhibitors%2520of%2520interleukin-1%2520beta-converting%2520enzyme%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1994%26volume%3D37%26spage%3D563%26epage%3D564%26doi%3D10.1021%2Fjm00031a003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brady, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giegel, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grinnell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lunney, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talanian, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, N.</span></span> <span> </span><span class="NLM_article-title">A catalytic mechanism for caspase-1 and for bimodal inhibition of caspase-1 by activated aspartic ketones</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">621</span>– <span class="NLM_lpage">631</span>, <span class="refDoi"> DOI: 10.1016/S0968-0896(99)00009-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1016%2FS0968-0896%2899%2900009-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10353641" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK1MXis12mt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1999&pages=621-631&author=K.+D.+Bradyauthor=D.+A.+Giegelauthor=C.+Grinnellauthor=E.+Lunneyauthor=R.+V.+Talanianauthor=W.+Wongauthor=N.+Walker&title=A+catalytic+mechanism+for+caspase-1+and+for+bimodal+inhibition+of+caspase-1+by+activated+aspartic+ketones&doi=10.1016%2FS0968-0896%2899%2900009-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">A catalytic mechanism for caspase-1 and for bimodal inhibition of caspase-1 by activated aspartic ketones</span></div><div class="casAuthors">Brady, Kenneth D.; Giegel, David A.; Grinnell, Christine; Lunney, Elizabeth; Talanian, Robert V.; Wong, Winnie; Walker, Nigel</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">621-631</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">We have evaluated 619 aspartic ketones with 9 different types of prime-side groups (acyloxymethyl, aryloxymethyl, arylthiomethyl, alkylthiomethyl, acylamino-oxymethyl, sulfonylaminomethyl, α-ketoamide, α-(1-phenyl-3-trifluoromethyl-pyrazol-5-yl)oxymethyl (PTP), and aliph. ketones) as inhibitors of caspase-1.  The inhibitory behaviors could be classified as reversible, inactivating, or bimodal (i.e. reversible inhibition followed by slow inactivation) based on the kinetically obsd. formation of reversible thiohemiketal complexes and conversion to an irreversible thioether adduct, and the mechanism of any given ketone was only poorly predictable on the basis of leaving group structure and chem.  Among 201 bimodal inhibitors, the rate of conversion of the reversible thiohemiketal complex to the inactive thioether (ki) was strictly first-order, consistent with direct conversion of the thiohemiketal to the thioether with no intermediate collapse to free ketone and thiolate.  We have examd. 22 crystallog. structures of caspase-1 complexed as a thiohemiketal with the inhibitors from 8 different ketone classes, and found the Cys285S-C-Cα'-leaving group dihedral angle to be near either to 60° or to 180°.  Only the 180° conformation was permissive for SN2 displacement of the leaving group and, furthermore, positioned His237Nδ to stabilize developing charge on the leaving group.  Among these structures and 19 addnl. complexes, all showed a strong interaction between His237Nδ and the ketone or thiohemiketal oxygen.  We therefore propose a proteolytic mechanism for caspase-1 involving polarization of the scissile carbonyl by the His237 imidazolium group.  During thiohemiketal/thioether conversion (but probably not during peptide hydrolysis), the leaving group is stabilized by the His237 imidazolium.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGN4AAx-wiarVg90H21EOLACvtfcHk0lhRIir9ozk6qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXis12mt7Y%253D&md5=61c58cf618b855f3573cb3669a7e9224</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2FS0968-0896%2899%2900009-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0968-0896%252899%252900009-7%26sid%3Dliteratum%253Aachs%26aulast%3DBrady%26aufirst%3DK.%2BD.%26aulast%3DGiegel%26aufirst%3DD.%2BA.%26aulast%3DGrinnell%26aufirst%3DC.%26aulast%3DLunney%26aufirst%3DE.%26aulast%3DTalanian%26aufirst%3DR.%2BV.%26aulast%3DWong%26aufirst%3DW.%26aulast%3DWalker%26aufirst%3DN.%26atitle%3DA%2520catalytic%2520mechanism%2520for%2520caspase-1%2520and%2520for%2520bimodal%2520inhibition%2520of%2520caspase-1%2520by%2520activated%2520aspartic%2520ketones%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D1999%26volume%3D7%26spage%3D621%26epage%3D631%26doi%3D10.1016%2FS0968-0896%2899%2900009-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anand, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziebuhr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wadhwani, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesters, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span> <span> </span><span class="NLM_article-title">Coronavirus main proteinase (3CL<sup>pro</sup>) structure: basis for design of anti-SARS drugs</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>300</i></span>,  <span class="NLM_fpage">1763</span>– <span class="NLM_lpage">1767</span>, <span class="refDoi"> DOI: 10.1126/science.1085658</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1126%2Fscience.1085658" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=12746549" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BD3sXksVKisLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=300&publication_year=2003&pages=1763-1767&author=K.+Anandauthor=J.+Ziebuhrauthor=P.+Wadhwaniauthor=J.+R.+Mestersauthor=R.+Hilgenfeld&title=Coronavirus+main+proteinase+%283CLpro%29+structure%3A+basis+for+design+of+anti-SARS+drugs&doi=10.1126%2Fscience.1085658"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Coronavirus Main Proteinase (3CLpro) Structure: Basis for Design of Anti-SARS Drugs</span></div><div class="casAuthors">Anand, Kanchan; Ziebuhr, John; Wadhwani, Parvesh; Mesters, Jeroen R.; Hilgenfeld, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">300</span>
        (<span class="NLM_cas:issue">5626</span>),
    <span class="NLM_cas:pages">1763-1767</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A novel coronavirus has been identified as the causative agent of severe acute respiratory syndrome (SARS).  The viral main proteinase (Mpro, also called 3CLpro), which controls the activities of the coronavirus replication complex, is an attractive target for therapy.  The authors detd. crystal structures for human coronavirus (strain 229E) Mpro and for an inhibitor complex of porcine coronavirus [transmissible gastroenteritis virus (TGEV)] Mpro, and the authors constructed a homol. model for SARS coronavirus (SARS-CoV) Mpro.  The structures reveal a remarkable degree of conservation of the substrate-binding sites, which is further supported by recombinant SARS-CoV Mpro-mediated cleavage of a TGEV Mpro substrate.  Mol. modeling suggests that available rhinovirus 3Cpro inhibitors may be modified to make them useful for treating SARS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEmONTL1SzDrVg90H21EOLACvtfcHk0lhRIir9ozk6qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXksVKisLk%253D&md5=7a8d8d39a783cb3960f6f22931f35569</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1126%2Fscience.1085658&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1085658%26sid%3Dliteratum%253Aachs%26aulast%3DAnand%26aufirst%3DK.%26aulast%3DZiebuhr%26aufirst%3DJ.%26aulast%3DWadhwani%26aufirst%3DP.%26aulast%3DMesters%26aufirst%3DJ.%2BR.%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3DCoronavirus%2520main%2520proteinase%2520%25283CLpro%2529%2520structure%253A%2520basis%2520for%2520design%2520of%2520anti-SARS%2520drugs%26jtitle%3DScience%26date%3D2003%26volume%3D300%26spage%3D1763%26epage%3D1767%26doi%3D10.1126%2Fscience.1085658" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span> </span><span class="NLM_article-title">The X-ray crystallographic coordinates for the SARS CoV-1 3CL<sup>pro</sup> covalently bound with inhibitor <b>2</b> is deposited
in the Protein Data Bank (PDB)</span>, <a href="http://rcsb.rutgers.edu/pdb/index" class="extLink">http://rcsb.rutgers.edu/pdb/index</a> (PDB code 6XHO).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=The+X-ray+crystallographic+coordinates+for+the+SARS+CoV-1+3CLpro+covalently+bound+with+inhibitor+2+is+deposited%0Ain+the+Protein+Data+Bank+%28PDB%29%2C+http%3A%2F%2Frcsb.rutgers.edu%2Fpdb%2Findex+%28PDB+code+6XHO%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DThe%2520X-ray%2520crystallographic%2520coordinates%2520for%2520the%2520SARS%2520CoV-1%25203CLpro%2520covalently%2520bound%2520with%2520inhibitor%25202%2520is%2520deposited%250Ain%2520the%2520Protein%2520Data%2520Bank%2520%2528PDB%2529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prins, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webber, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marakovits, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuhrman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patick, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathews, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rejto, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendrickson, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuntland, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meador, J. W.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferre, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harr, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosa, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Worland, S. L.</span></span> <span> </span><span class="NLM_article-title">Structure-based design, synthesis, and biological evaluation of irreversible human rhinovirus 3C protease inhibitors. 4. Incorporation of P1 lactam moieties as L-glutamine replacements</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">1213</span>, <span class="refDoi"> DOI: 10.1021/jm9805384</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9805384" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK1MXhvVeisLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=1213&author=P.+S.+Dragovichauthor=T.+J.+Prinsauthor=R.+Zhouauthor=S.+E.+Webberauthor=J.+T.+Marakovitsauthor=S.+A.+Fuhrmanauthor=A.+K.+Patickauthor=D.+A.+Mathewsauthor=C.+A.+Leeauthor=C.+E.+Fordauthor=B.+J.+Burkeauthor=P.+A.+Rejtoauthor=T.+F.+Hendricksonauthor=T.+Tuntlandauthor=E.+L.+Brownauthor=J.+W.+Meadorauthor=R.+A.+Ferreauthor=J.+E.+Harrauthor=M.+B.+Kosaauthor=S.+L.+Worland&title=Structure-based+design%2C+synthesis%2C+and+biological+evaluation+of+irreversible+human+rhinovirus+3C+protease+inhibitors.+4.+Incorporation+of+P1+lactam+moieties+as+L-glutamine+replacements&doi=10.1021%2Fjm9805384"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design, Synthesis, and Biological Evaluation of Irreversible Human Rhinovirus 3C Protease Inhibitors. 4. Incorporation of P1 Lactam Moieties as L-Glutamine Replacements</span></div><div class="casAuthors">Dragovich, Peter S.; Prins, Thomas J.; Zhou, Ru; Webber, Stephen E.; Marakovits, Joseph T.; Fuhrman, Shella A.; Patick, Amy K.; Matthews, David A.; Lee, Caroline A.; Ford, Clifford E.; Burke, Benjamin J.; Rejto, Paul A.; Hendrickson, Thomas F.; Tuntland, Tove; Brown, Edward L.; Meador, James W., III; Ferre, Rose Ann; Harr, James E. V.; Kosa, Maha B.; Worland, Stephen T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1213-1224</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The structure-based design, chem. synthesis, and biol. evaluation of various human rhinovirus (HRV) 3C protease (3CP) inhibitors, e.g. I [R = CH2CHMe2, CHMe2; X = (S)-CH, (R)-CH, N; n = 1, 2] and II (R = CHMe2, CMe3; Y = NH, CH2) which incorporate P1 lactam moieties in lieu of an L-glutamine residue are described.  These compds. are comprised of a tripeptidyl or peptidomimetic binding determinant and an Et propenoate Michael acceptor moiety which forms an irreversible covalent adduct with the active site cysteine residue of the 3C enzyme.  The P1-lactam-contg. inhibitors display significantly increased 3CP inhibition activity along with improved antirhinoviral properties relative to corresponding L-glutamine-derived mols.  In addn., several lactam-contg. compds. exhibit excellent selectivity for HRV 3CP over several other serine and cysteine proteases and are not appreciably degraded by a variety of biol. agents.  Lactam II (R = CHMe2, Y = CH2, n = 1) (AG7088) was one of the most potent inhibitors (mean antirhinoviral EC90 ≈ 0.10 μM, n = 46 serotypes), and is shown to warrant addnl. preclin. development to explore its potential for use as an antirhinoviral agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlBuxFPwMr97Vg90H21EOLACvtfcHk0ljM5vd_bRuMWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXhvVeisLk%253D&md5=0e2bf5a03d17ea371663a252e7b97b20</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm9805384&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9805384%26sid%3Dliteratum%253Aachs%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DPrins%26aufirst%3DT.%2BJ.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DWebber%26aufirst%3DS.%2BE.%26aulast%3DMarakovits%26aufirst%3DJ.%2BT.%26aulast%3DFuhrman%26aufirst%3DS.%2BA.%26aulast%3DPatick%26aufirst%3DA.%2BK.%26aulast%3DMathews%26aufirst%3DD.%2BA.%26aulast%3DLee%26aufirst%3DC.%2BA.%26aulast%3DFord%26aufirst%3DC.%2BE.%26aulast%3DBurke%26aufirst%3DB.%2BJ.%26aulast%3DRejto%26aufirst%3DP.%2BA.%26aulast%3DHendrickson%26aufirst%3DT.%2BF.%26aulast%3DTuntland%26aufirst%3DT.%26aulast%3DBrown%26aufirst%3DE.%2BL.%26aulast%3DMeador%26aufirst%3DJ.%2BW.%26aulast%3DFerre%26aufirst%3DR.%2BA.%26aulast%3DHarr%26aufirst%3DJ.%2BE.%26aulast%3DKosa%26aufirst%3DM.%2BB.%26aulast%3DWorland%26aufirst%3DS.%2BL.%26atitle%3DStructure-based%2520design%252C%2520synthesis%252C%2520and%2520biological%2520evaluation%2520of%2520irreversible%2520human%2520rhinovirus%25203C%2520protease%2520inhibitors.%25204.%2520Incorporation%2520of%2520P1%2520lactam%2520moieties%2520as%2520L-glutamine%2520replacements%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D1213%26doi%3D10.1021%2Fjm9805384" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rich, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, J. V. N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houseman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKenzie, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malkovsky, M.</span></span> <span> </span><span class="NLM_article-title">New hydroxyethylamine HIV protease inhibitors that suppress viral replication</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">3803</span>– <span class="NLM_lpage">3812</span>, <span class="refDoi"> DOI: 10.1021/jm00099a008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00099a008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK38XlvFags7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=1992&pages=3803-3812&author=D.+H.+Richauthor=J.+V.+N.+V.+Prasadauthor=C.-Q.+Sunauthor=J.+Greenauthor=R.+Muellerauthor=K.+Housemanauthor=D.+MacKenzieauthor=M.+Malkovsky&title=New+hydroxyethylamine+HIV+protease+inhibitors+that+suppress+viral+replication&doi=10.1021%2Fjm00099a008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">New hydroxyethylamine HIV protease inhibitors that suppress viral replication</span></div><div class="casAuthors">Rich, Daniel H.; Prasad, J. V. N. Vara; Sun, Chong Qing; Green, Jeremy; Mueller, Richard; Houseman, Kathryn; MacKenzie, Debra; Malkovsky, Miroslav</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">3803-12</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">The synthesis of analogs of Ac-Ser-Leu-Asn-[Phe-HEA-Pro]-Ile-Val-OMe (JG-365; Phe-HEA-Pro = hydroxyethylamine transition state analog I), a tight-binding inhibitor of human immunodeficiency virus protease (HIVP), are reported.  Systematic modification of the P3 and P3' regions of the inhibitors has led to smaller HIVP inhibitors that inhibit viral replication in HIV-infected and simian immunodeficiency virus (SIV)-infected cell cultures.  Six aliph. and arom. derivs. were prepd. by replacing residues in the P3 regions of Boc-Leu-Asn-[Phe-HEA-Pro]-Ile-Val-OMe (Boc = Me3CO2C).  Arom. side chains at P3 gave better inhibitors than aliph. side chains.  The better inhibitors in this series contained a β-naphthylalanine or a biphenyl unit at P3.  A second series of HIVP inhibitors were obtained by converting the P3 group into acyl groups.  R-Asn-[Phe-HEA-Pro]-Ile-Phe-OMe (R = PhCH2O2C, 2-quinolinylcarbonyl) are potent HIVP inhibitors with Ki values equal to 1.0 and 0.1 nM, resp.  The inhibition consts. were det. by using the continuous fluorometric assay developed by M. V. Toth and G. R. Marshall (1990).  The activities of the protease inhibitors for inhibition of SIV replication were detd. in vitro using CEMx174 cells.  Inhibition of HIV infection was detd. essentially as reported by R. Pauwels, et. al. (1988).  The anti-HIV assay was carried out in culture using CEM cells (a CD4+ lymphocyte line) infected with virus strain HTLV-IIIb with a multiplicity of infection of 0.1.  Several analogs inhibited the cytopathic effect at concns. of 0.1-0.8 μg/mL.  These results establish that good inhibitors of HIV protease that inhibit viral replication in infected lymphocytes in in vitro cell assays can be obtained from JG-365 when the Ac-Ser-Leu unit is replaced by arom. acyl derivs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFgN6BYEYXb7Vg90H21EOLACvtfcHk0ljM5vd_bRuMWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XlvFags7Y%253D&md5=4fd3643c608b8ebdff13563757868d23</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm00099a008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00099a008%26sid%3Dliteratum%253Aachs%26aulast%3DRich%26aufirst%3DD.%2BH.%26aulast%3DPrasad%26aufirst%3DJ.%2BV.%2BN.%2BV.%26aulast%3DSun%26aufirst%3DC.-Q.%26aulast%3DGreen%26aufirst%3DJ.%26aulast%3DMueller%26aufirst%3DR.%26aulast%3DHouseman%26aufirst%3DK.%26aulast%3DMacKenzie%26aufirst%3DD.%26aulast%3DMalkovsky%26aufirst%3DM.%26atitle%3DNew%2520hydroxyethylamine%2520HIV%2520protease%2520inhibitors%2520that%2520suppress%2520viral%2520replication%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1992%26volume%3D35%26spage%3D3803%26epage%3D3812%26doi%3D10.1021%2Fjm00099a008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamashita, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brook, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomaszek, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczuk, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tew, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veber, D. F.</span></span> <span> </span><span class="NLM_article-title">Solid-phase synthesis of a combinatorial array of 1,3-bis(acylamino)-2-butanones, inhibitors of the cysteine proteases cathepsins K and L</span>. <i>J. Comb. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">207</span>– <span class="NLM_lpage">215</span>, <span class="refDoi"> DOI: 10.1021/cc9800374</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cc9800374" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK1MXisFSqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=1999&pages=207-215&author=D.+S.+Yamashitaauthor=X.+Dongauthor=H.-J.+Ohauthor=C.+S.+Brookauthor=T.+A.+Tomaszekauthor=L.+Szewczukauthor=D.+G.+Tewauthor=D.+F.+Veber&title=Solid-phase+synthesis+of+a+combinatorial+array+of+1%2C3-bis%28acylamino%29-2-butanones%2C+inhibitors+of+the+cysteine+proteases+cathepsins+K+and+L&doi=10.1021%2Fcc9800374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Solid-Phase Synthesis of a Combinatorial Array of 1,3-Bis(acylamino)-2-butanones, Inhibitors of the Cysteine Proteases Cathepsins K and L</span></div><div class="casAuthors">Yamashita, Dennis S.; Dong, Xiaoyang; Oh, Hye-Ja; Brook, Christopher S.; Tomaszek, Thaddeus A.; Szewczuk, Lawrence; Tew, David G.; Veber, Daniel F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Combinatorial Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">207-215</span>CODEN:
                <span class="NLM_cas:coden">JCCHFF</span>;
        ISSN:<span class="NLM_cas:issn">1520-4766</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To more rapidly prep. members of the 1,3-bis(acylamino)-2-butanone class of cysteine protease inhibitors, a solid-phase synthesis was developed.  1-Azido-3-amino-2,2-dimethoxybutane, which has two amino groups differentiated and the ketone protected as a ketal, served as a surrogate for the 1,3-diamino-2-butanone core.  Thus, 1-azido-3-amino-2,2-dimethoxybutane was coupled to the BAL-resin-linked carboxylic acids derived from α-amino acid esters.  Evaluation of a small combinatorial array by measuring inhibition consts. (Ki,apps) against cathepsins K, L, and B provided some structure-activity relationship trends with respect to selectivity and potency. Novel, potent inhibitors of cathepsins K and L were identified.  Novel, potent inhibitors of cathepsins K and L were identified.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5W8F450f8KrVg90H21EOLACvtfcHk0ljM5vd_bRuMWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXisFSqsLk%253D&md5=96a8b824edfbe519b2afac34c3d9262d</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fcc9800374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcc9800374%26sid%3Dliteratum%253Aachs%26aulast%3DYamashita%26aufirst%3DD.%2BS.%26aulast%3DDong%26aufirst%3DX.%26aulast%3DOh%26aufirst%3DH.-J.%26aulast%3DBrook%26aufirst%3DC.%2BS.%26aulast%3DTomaszek%26aufirst%3DT.%2BA.%26aulast%3DSzewczuk%26aufirst%3DL.%26aulast%3DTew%26aufirst%3DD.%2BG.%26aulast%3DVeber%26aufirst%3DD.%2BF.%26atitle%3DSolid-phase%2520synthesis%2520of%2520a%2520combinatorial%2520array%2520of%25201%252C3-bis%2528acylamino%2529-2-butanones%252C%2520inhibitors%2520of%2520the%2520cysteine%2520proteases%2520cathepsins%2520K%2520and%2520L%26jtitle%3DJ.%2520Comb.%2520Chem.%26date%3D1999%26volume%3D1%26spage%3D207%26epage%3D215%26doi%3D10.1021%2Fcc9800374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nayyar, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tibbetts, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, T. P.</span></span> <span> </span><span class="NLM_article-title">An efficient synthesis of a key intermediate for the preparation of the rhinovirus protease inhibitor AG7088 via asymmetric dianionic cyanomethylation of N-Boc-L-(+)-glutamic acid dimethyl ester</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">6807</span>– <span class="NLM_lpage">6809</span>, <span class="refDoi"> DOI: 10.1016/S0040-4039(01)01416-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1016%2FS0040-4039%2801%2901416-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmslCms7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2001&pages=6807-6809&author=Q.+Tianauthor=N.+K.+Nayyarauthor=S.+Babuauthor=L.+Chenauthor=J.+Taoauthor=S.+Leeauthor=A.+Tibbettsauthor=T.+Moranauthor=J.+Liouauthor=M.+Guoauthor=T.+P.+Kennedy&title=An+efficient+synthesis+of+a+key+intermediate+for+the+preparation+of+the+rhinovirus+protease+inhibitor+AG7088+via+asymmetric+dianionic+cyanomethylation+of+N-Boc-L-%28%2B%29-glutamic+acid+dimethyl+ester&doi=10.1016%2FS0040-4039%2801%2901416-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">An efficient synthesis of a key intermediate for the preparation of the rhinovirus protease inhibitor AG7088 via asymmetric dianionic cyanomethylation of N-Boc-L-(+)-glutamic acid dimethyl ester</span></div><div class="casAuthors">Tian, Q.; Nayyar, N. K.; Babu, S.; Chen, L.; Tao, J.; Lee, S.; Tibbetts, A.; Moran, T.; Liou, J.; Guo, M.; Kennedy, T. P.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">6807-6809</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">An efficient synthetic route to a key intermediate [(2E,4S)-4-(N-Boc-amino)-5-[(3S)-2-oxo-3-pyrrolidinyl]-2-pentenoic acid Et ester (Boc = Me3CO2C)] for the prepn. of the rhinovirus protease inhibitor AG7088 has been developed employing a key asym. dianionic cyanomethylation of N-Boc-L-(+)-glutamic acid di-Me ester.  This methodol. enables the prepn. of this compd. in kilogram quantities with an overall yield of 30%.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfwuKJVB5h-bVg90H21EOLACvtfcHk0lgTRs9w6bvv-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmslCms7w%253D&md5=ffb96a4ab738a521993ce0551d9d4223</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2FS0040-4039%2801%2901416-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4039%252801%252901416-2%26sid%3Dliteratum%253Aachs%26aulast%3DTian%26aufirst%3DQ.%26aulast%3DNayyar%26aufirst%3DN.%2BK.%26aulast%3DBabu%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DTao%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DTibbetts%26aufirst%3DA.%26aulast%3DMoran%26aufirst%3DT.%26aulast%3DLiou%26aufirst%3DJ.%26aulast%3DGuo%26aufirst%3DM.%26aulast%3DKennedy%26aufirst%3DT.%2BP.%26atitle%3DAn%2520efficient%2520synthesis%2520of%2520a%2520key%2520intermediate%2520for%2520the%2520preparation%2520of%2520the%2520rhinovirus%2520protease%2520inhibitor%2520AG7088%2520via%2520asymmetric%2520dianionic%2520cyanomethylation%2520of%2520N-Boc-L-%2528%252B%2529-glutamic%2520acid%2520dimethyl%2520ester%26jtitle%3DTetrahedron%2520Lett.%26date%3D2001%26volume%3D42%26spage%3D6807%26epage%3D6809%26doi%3D10.1016%2FS0040-4039%2801%2901416-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="note"><p class="first last">Generation of diazomethane followed Aldrich technical bulletin AL-180 employing Diazald kit.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mendonca, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatraman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, J. T.</span></span> <span> </span><span class="NLM_article-title">Novel route to the synthesis of peptides containing 2-amino-1′-hydroxymethyl ketones and their application as cathepsin K inhibitors</span>. <i>Biorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2887</span>– <span class="NLM_lpage">2891</span>, <span class="refDoi"> DOI: 10.1016/S0960-894X(02)00611-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1016%2FS0960-894X%2802%2900611-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=12270170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BD38Xntlagu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2002&pages=2887-2891&author=R.+V.+Mendoncaauthor=S.+Venkatramanauthor=J.+T.+Palmer&title=Novel+route+to+the+synthesis+of+peptides+containing+2-amino-1%E2%80%B2-hydroxymethyl+ketones+and+their+application+as+cathepsin+K+inhibitors&doi=10.1016%2FS0960-894X%2802%2900611-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Novel route to the synthesis of peptides containing 2-amino-1'-hydroxymethyl ketones and their application as cathepsin K inhibitors</span></div><div class="casAuthors">Mendonca, Rohan V.; Venkatraman, Shankar; Palmer, James T.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2887-2891</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Cathepsin K is highly expressed in human osteoclasts and is implicated in bone resorption.  This makes it an attractive target for the treatment of osteoporosis.  Peptides contg. 2-amino-1'-hydroxymethyl ketones and 2-amino-1'-alkoxymethyl ketones were discovered as potent inhibitors of cathepsin K.  A novel synthetic route was devised to facilitate rapid elucidation of the SAR of these inhibitors.  The synthesis and SAR of hydroxymethyl ketones are presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBSBV5awQSv7Vg90H21EOLACvtfcHk0lgTRs9w6bvv-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xntlagu70%253D&md5=318ea46f2a2aee28f8daeacd808d492e</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2802%2900611-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252802%252900611-X%26sid%3Dliteratum%253Aachs%26aulast%3DMendonca%26aufirst%3DR.%2BV.%26aulast%3DVenkatraman%26aufirst%3DS.%26aulast%3DPalmer%26aufirst%3DJ.%2BT.%26atitle%3DNovel%2520route%2520to%2520the%2520synthesis%2520of%2520peptides%2520containing%25202-amino-1%25E2%2580%25B2-hydroxymethyl%2520ketones%2520and%2520their%2520application%2520as%2520cathepsin%2520K%2520inhibitors%26jtitle%3DBiorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2002%26volume%3D12%26spage%3D2887%26epage%3D2891%26doi%3D10.1016%2FS0960-894X%2802%2900611-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="note"><p class="first last">These conditions result in detectable levels of epimerization at the P<sub>1</sub> site. The resulting diastereomer has been observed by HPLC, NMR, and LCMS and is readily removed by fractional recrystallization.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krantz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copp, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coles, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heard, S. B.</span></span> <span> </span><span class="NLM_article-title">Peptidyl (acyloxy)methyl ketones and the quiescent affinity label concept: the departing group as a variable structural element in the design of inactivators of cysteine proteinases</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">4678</span>– <span class="NLM_lpage">4687</span>, <span class="refDoi"> DOI: 10.1021/bi00233a007</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00233a007" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK3MXit1CgsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1991&pages=4678-4687&author=A.+Krantzauthor=L.+J.+Coppauthor=P.+J.+Colesauthor=R.+A.+Smithauthor=S.+B.+Heard&title=Peptidyl+%28acyloxy%29methyl+ketones+and+the+quiescent+affinity+label+concept%3A+the+departing+group+as+a+variable+structural+element+in+the+design+of+inactivators+of+cysteine+proteinases&doi=10.1021%2Fbi00233a007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Peptidyl (acyloxy)methyl ketones and the quiescent affinity label concept:  the departing group as a variable structural element in the design of inactivators of cysteine proteinases</span></div><div class="casAuthors">Krantz, Allen; Copp, Leslie J.; Coles, Peter J.; Smith, Roger A.; Heard, Stephen B.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">4678-87</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">(Acyloxy)methyl ketones, of general structure Z-[AA2]-[AA1]-CH2OCOAr (Z = benzyloxycarbonyl; AA = amino acid; Ar = aryl), were potent inactivators of the cysteine proteinase, cathepsin B.  These reagents were designed as affinity labels in which the dipeptidyl moiety serves as an affinity group (complementary to the S1 and S2 sites of the enzyme), while the (acyloxy)methyl ketone unit (-COCH2OCOR), contg. a weak leaving group in the form of a carboxylate nucleofuge, functions as the potentially reactive entity that labels the enzyme.  The inhibition was time-dependent, active-site directed, and irreversible.  The apparent 2nd-order rate const. (kinact/Kinact), which characterizes the inhibition of cathepsin B by this series, spanned several orders of magnitude and in certain cases exceeded 106 M-1 s-1.  The activity of this series of inhibitors was found to be exquisitely sensitive to the nature of the carboxylate leaving group as well as the affinity group.  A strong dependence of 2nd-order inactivation rate on leaving group pKa was uncovered for Z-Phe-Ala (acyloxy)methyl ketones [log(k/K) = -1.1 × pKa + 7.2.  Heretofore, in constructing affinity labels the choice of leaving group was quite restricted.  The aryl carboxylate group thus offers considerable variation as a design element in that both its binding affinity and reactivity can be controlled by substituent effects.  Specific peptidyl (acyloxy)methyl ketones thus represent prime examples of highly potent, chem. stable enzyme inhibitors with variable structural elements in both the affinity and departing groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprQo4CgIuJM7Vg90H21EOLACvtfcHk0lgTRs9w6bvv-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXit1CgsLY%253D&md5=887882366c35b5cba8021d1d14eb47be</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fbi00233a007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00233a007%26sid%3Dliteratum%253Aachs%26aulast%3DKrantz%26aufirst%3DA.%26aulast%3DCopp%26aufirst%3DL.%2BJ.%26aulast%3DColes%26aufirst%3DP.%2BJ.%26aulast%3DSmith%26aufirst%3DR.%2BA.%26aulast%3DHeard%26aufirst%3DS.%2BB.%26atitle%3DPeptidyl%2520%2528acyloxy%2529methyl%2520ketones%2520and%2520the%2520quiescent%2520affinity%2520label%2520concept%253A%2520the%2520departing%2520group%2520as%2520a%2520variable%2520structural%2520element%2520in%2520the%2520design%2520of%2520inactivators%2520of%2520cysteine%2520proteinases%26jtitle%3DBiochemistry%26date%3D1991%26volume%3D30%26spage%3D4678%26epage%3D4687%26doi%3D10.1021%2Fbi00233a007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span> </span><span class="NLM_article-title">The X-ray crystallographic coordinates for the SARS
CoV-1 and SARS
CoV-2 3CL<sup>pro</sup> each covalently bound inhibitor 28 are deposited
in the Protein Data Bank (PDB)</span>, <a href="http://rcsb.rutgers.edu/pdb/index" class="extLink">http://rcsb.rutgers.edu/pdb/index</a> (PDB Code 6XHN).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=The+X-ray+crystallographic+coordinates+for+the+SARS%0ACoV-1+and+SARS%0ACoV-2+3CLpro+each+covalently+bound+inhibitor+28+are+deposited%0Ain+the+Protein+Data+Bank+%28PDB%29%2C+http%3A%2F%2Frcsb.rutgers.edu%2Fpdb%2Findex+%28PDB+Code+6XHN%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DThe%2520X-ray%2520crystallographic%2520coordinates%2520for%2520the%2520SARS%250ACoV-1%2520and%2520SARS%250ACoV-2%25203CLpro%2520each%2520covalently%2520bound%2520inhibitor%252028%2520are%2520deposited%250Ain%2520the%2520Protein%2520Data%2520Bank%2520%2528PDB%2529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, R. L.</span>; <span class="NLM_string-name">Kania, R. S.</span>; <span class="NLM_string-name">Nieman, J. A.</span>; <span class="NLM_string-name">Planken, S. P.</span>; <span class="NLM_string-name">Smith, G. J.,  III.</span></span> <span> </span><span class="NLM_article-title">SARS antiviral data for all described inhibitors are disclosed</span>. Preparation of peptides as anticoronaviral agents. <span class="NLM_patent">WO2005/113580 A1</span>, Dec 1, <span class="NLM_year">2005</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&author=R.+L.+Hoffman&author=R.+S.+Kania&author=J.+A.+Nieman&author=S.+P.+Planken&author=G.+J.+Smith&title=SARS+antiviral+data+for+all+described+inhibitors+are+disclosed"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DHoffman%26aufirst%3DR.%2BL.%26atitle%3DSARS%2520antiviral%2520data%2520for%2520all%2520described%2520inhibitors%2520are%2520disclosed%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="note"><p class="first last">Compound is incubated at 1 μM substrate concentration and 5 mM GSH in pH 7.8 0.1 M TRIS buffer at 37 °C. Timepoints are taken over 30 min and terminated with 250 mM iodoacetamide to quench the excess GSH. Samples were then diluted in acetonitrile for LC/MS analysis. Data are reported as half-life.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="note"><p class="first last">Compound is spiked at 1 μM substrate concentration in pH 7.4 adjusted (with HCL or NaOH) plasma and incubated at 37 °C. 50 μL plasma samples were taken at multiple timepoints during a 60 minute incubation and added to 200 μL acetonitrile to stop enzymatic degradation. Samples were centrifuged at 3000 rpm and 150 μL supernatant was transferred to a 96 well block for analysis by LC/MS. Data are reported as half-life.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="note"><p class="first last">The solubility of <b>3</b> is 0.377 mg/mL (2.5% w/v SBECD, 5 mM phosphate buffer, pH 7.4, made isotonic with 4.25% dextrose). Rat <i>in vivo</i> parameters for <b>3</b> were from IV dosing used a 50% PEG 200/10% EtOH/40% saline formulation. For 2 mg/kg dose the following values were obtained: 26.2 mL/min/kg CL; 1.0 L/kg <i>V</i><sub>dss</sub>; 0.9 <i>t</i><sub>1/2</sub>; and 1.3 h·μg/mL AUC.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brinen, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellman, J. A.</span></span> <span> </span><span class="NLM_article-title">Crystal structures of reversible ketone-Based inhibitors of the cysteine protease cruzain</span>. <i>Biorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">21</span>– <span class="NLM_lpage">29</span>, <span class="refDoi"> DOI: 10.1016/S0968-0896(02)00427-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1016%2FS0968-0896%2802%2900427-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=12467703" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BD38XptFGltrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2003&pages=21-29&author=L.+Huangauthor=L.+S.+Brinenauthor=J.+A.+Ellman&title=Crystal+structures+of+reversible+ketone-Based+inhibitors+of+the+cysteine+protease+cruzain&doi=10.1016%2FS0968-0896%2802%2900427-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of reversible ketone-based inhibitors of the cysteine protease cruzain</span></div><div class="casAuthors">Huang, Lily; Brinen, Linda S.; Ellman, Jonathan A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">21-29</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">The crystal structures of two hydroxymethyl ketone inhibitors complexed to the cysteine protease cruzain have been detd. at 1.1 and 1.2 A resoln., resp.  These high resoln. crystal structures provide the first structures of non-covalent inhibitors bound to cruzain.  A series of compds. were prepd. and tested based upon the structures providing further insight into the key binding interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodxjvcaYN6GLVg90H21EOLACvtfcHk0litErQ6pswhYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XptFGltrw%253D&md5=a96042e8c37352c548f90082bfe6ee7a</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2FS0968-0896%2802%2900427-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0968-0896%252802%252900427-3%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DBrinen%26aufirst%3DL.%2BS.%26aulast%3DEllman%26aufirst%3DJ.%2BA.%26atitle%3DCrystal%2520structures%2520of%2520reversible%2520ketone-Based%2520inhibitors%2520of%2520the%2520cysteine%2520protease%2520cruzain%26jtitle%3DBiorg.%2520Med.%2520Chem.%26date%3D2003%26volume%3D11%26spage%3D21%26epage%3D29%26doi%3D10.1016%2FS0968-0896%2802%2900427-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="note"><p class="first last">Interestingly, this reference reports no covalent bond observed between the active site cysteine and the ketone pharmacophore.</p></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fuentes-Prior, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvesen, G. S.</span></span> <span> </span><span class="NLM_article-title">The protein structures that shape caspase activity, specificity, activation and inhibition</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>384</i></span>,  <span class="NLM_fpage">201</span>– <span class="NLM_lpage">232</span>, <span class="refDoi"> DOI: 10.1042/BJ20041142</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1042%2FBJ20041142" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=15450003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVSqs7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=384&publication_year=2004&pages=201-232&author=P.+Fuentes-Priorauthor=G.+S.+Salvesen&title=The+protein+structures+that+shape+caspase+activity%2C+specificity%2C+activation+and+inhibition&doi=10.1042%2FBJ20041142"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">The protein structures that shape caspase activity, specificity, activation and inhibition</span></div><div class="casAuthors">Fuentes-Prior, Pablo; Salvesen, Guy S.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">384</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">201-232</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">A review.  The death morphol. commonly known as apoptosis results from a post-translational pathway driven largely by specific limited proteolysis.  In the last decade the structural basis for apoptosis regulation has moved from nothing to 'quite good', and we now know the fundamental structures of examples from the initiator phase, the pre-mitochondrial regulator phase, the executioner phase, inhibitors and their antagonists, and even the structures of some substrates.  The field is as well advanced as the best known of proteolytic pathways, the coagulation cascade.  Fundamentally new mechanisms in protease regulation have been disclosed.  Structural evidence suggests that caspases have an unusual catalytic mechanism, and that they are activated by apparently unrelated events, depending on which position in the apoptotic pathway they occupy.  Some naturally occurring caspase inhibitors have adopted classic inhibition strategies, but other have revealed completely novel mechanisms.  All of the structural and mechanistic information can, and is, being applied to drive therapeutic strategies to combat overactivation of apoptosis in degenerative disease, and underactivation in neoplasia.  We present a comprehensive review of the caspases, their regulators and inhibitors from a structural and mechanistic point of view, and with an aim to consolidate the many threads that define the rapid growth of this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowJ7v8n99EJrVg90H21EOLACvtfcHk0litErQ6pswhYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVSqs7rP&md5=b3c9b22933c76e89ee5fa993e9ec2247</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1042%2FBJ20041142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20041142%26sid%3Dliteratum%253Aachs%26aulast%3DFuentes-Prior%26aufirst%3DP.%26aulast%3DSalvesen%26aufirst%3DG.%2BS.%26atitle%3DThe%2520protein%2520structures%2520that%2520shape%2520caspase%2520activity%252C%2520specificity%252C%2520activation%2520and%2520inhibition%26jtitle%3DBiochem.%2520J.%26date%3D2004%26volume%3D384%26spage%3D201%26epage%3D232%26doi%3D10.1042%2FBJ20041142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Obach, R. S.</span></span> <span> </span><span class="NLM_article-title">Prediction of human clearance of twenty-nine drugs from hepatic microsomal intrinsic clearance data: an examination of in vitro half-life approach and nonspecific binding to microsomes</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1350</span>– <span class="NLM_lpage">1359</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10534321" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK1MXntFSht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1999&pages=1350-1359&author=R.+S.+Obach&title=Prediction+of+human+clearance+of+twenty-nine+drugs+from+hepatic+microsomal+intrinsic+clearance+data%3A+an+examination+of+in+vitro+half-life+approach+and+nonspecific+binding+to+microsomes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of human clearance of twenty-nine drugs from hepatic microsomal intrinsic clearance data: an examination of in vitro half-life approach and nonspecific binding to microsomes</span></div><div class="casAuthors">Obach, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1350-1359</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Twenty-nine drugs of disparate structures and physicochem. properties were used in an examn. of the capability of human liver microsomal lability data ("in vitro T1/2" approach) to be useful in the prediction of human clearance.  Addnl., the potential importance of nonspecific binding to microsomes in the in vitro incubation milieu for the accurate prediction of human clearance was investigated.  The compds. examd. demonstrated a wide range of microsomal metabolic labilities with scaled intrinsic clearance values ranging from less than 0.5 mL/min/kg to 189 mL/min/kg.  Microsomal binding was detd. at microsomal protein concns. used in the lability incubations.  For the 29 compds. studied, unbound fractions in microsomes ranged from 0.11 to 1.0.  Generally, basic compds. demonstrated the greatest extent of binding and neutral and acidic compds. the least extent of binding.  In the projection of human clearance values, basic and neutral compds. were well predicted when all binding considerations (blood and microsome) were disregarded, however, including both binding considerations also yielded reasonable predictions.  Including only blood binding yielded very poor projections of human clearance for these two types of compds.  However, for acidic compds., disregarding all binding considerations yielded poor predictions of human clearance.  It was generally most difficult to accurately predict clearance for this class of compds.; however the accuracy was best when all binding considerations were included.  Overall, inclusion of both blood and microsome binding values gave the best agreement between in vivo clearance values and clearance values projected from in vitro intrinsic clearance data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWp2hUTOxyubVg90H21EOLACvtfcHk0litErQ6pswhYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXntFSht7c%253D&md5=7804cd4dfb131caac45aff9b7a1df901</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DObach%26aufirst%3DR.%2BS.%26atitle%3DPrediction%2520of%2520human%2520clearance%2520of%2520twenty-nine%2520drugs%2520from%2520hepatic%2520microsomal%2520intrinsic%2520clearance%2520data%253A%2520an%2520examination%2520of%2520in%2520vitro%2520half-life%2520approach%2520and%2520nonspecific%2520binding%2520to%2520microsomes%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D1999%26volume%3D27%26spage%3D1350%26epage%3D1359" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="note"><p class="first last">Solubility of <b>4</b> in several formulation vehicles: 4.6 mg/mL (2.5% w/v SBECD, 5 mM phosphate buffer, pH 7.4, made isotonic with 4.25% dextrose); 4.6 mg/mL (2.5% w/v SBECD, 5 mM citrate buffer, pH 7.4, made isotonic with 4.25% dextrose); 6.6 mg/mL (2.5% w/v HPCD).</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span> </span><span class="NLM_article-title">The X-ray crystallographic
coordinates for the SARS CoV-1 3CL<sup>pro</sup> (PDB Code 6XHL) and
SARS CoV-2 3CL<sup>pro</sup> (6XHM)
each covalently bound inhibitor <b>4</b> are deposited in the
Protein Data Bank (PDB)</span>, <a href="http://rcsb.rutgers.edu/pdb/index" class="extLink">http://rcsb.rutgers.edu/pdb/index</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=The+X-ray+crystallographic%0Acoordinates+for+the+SARS+CoV-1+3CLpro+%28PDB+Code+6XHL%29+and%0ASARS+CoV-2+3CLpro+%286XHM%29%0Aeach+covalently+bound+inhibitor+4+are+deposited+in+the%0AProtein+Data+Bank+%28PDB%29%2C+http%3A%2F%2Frcsb.rutgers.edu%2Fpdb%2Findex."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DThe%2520X-ray%2520crystallographic%250Acoordinates%2520for%2520the%2520SARS%2520CoV-1%25203CLpro%2520%2528PDB%2520Code%25206XHL%2529%2520and%250ASARS%2520CoV-2%25203CLpro%2520%25286XHM%2529%250Aeach%2520covalently%2520bound%2520inhibitor%25204%2520are%2520deposited%2520in%2520the%250AProtein%2520Data%2520Bank%2520%2528PDB%2529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wandel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kajiji, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guengerich, F. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, A. A. J.</span></span> <span> </span><span class="NLM_article-title">P-glycoprotein and cytochrome P-450 3A inhibition: dissociation of inhibitory potencies</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3944</span>– <span class="NLM_lpage">3948</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10463589" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK1MXlsVOltL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=1999&pages=3944-3948&author=C.+Wandelauthor=R.+B.+Kimauthor=S.+Kajijiauthor=F.+P.+Guengerichauthor=G.+R.+Wilkinsonauthor=A.+A.+J.+Wood&title=P-glycoprotein+and+cytochrome+P-450+3A+inhibition%3A+dissociation+of+inhibitory+potencies"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">P-glycoprotein and cytochrome P-450 3A inhibition: dissociation of inhibitory potencies</span></div><div class="casAuthors">Wandel, Christoph; Kim, Richard B.; Kajiji, Shama; Guengerich, F. Peter; Wilkinson, Grant R.; Wood, Alastair J. J.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3944-3948</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">AACR Subscription Office</span>)
        </div><div class="casAbstract">Many P-glycoprotein (P-gp) inhibitors studied in vitro and in vivo are also known or suspected to be substrates and/or inhibitors of cytochrome P 450 3A (CYP3A).  Such overlap raises the question of whether CYP3A inhibition is an intrinsic characteristic of P-gp inhibitors, a matter of concern in the development and rational use of such agents.  Thus, the purpose of the present study was to det. whether the ability to inhibit P-gp and CYP3A is, in fact, linked and whether specific P-gp inhibitors with limited ability to inhibit CYP3A can be identified.  Therefore, the potency of a series of 14 P-gp inhibitors was assessed by measuring their inhibition of the transepithelial flux across Caco-2 cells of digoxin, a prototypical P-gp substrate.  CYP3A inhibition was detd. from the impairment of nifedipine oxidn. by human liver microsomes.  Detn. of the apparent Ki values for CYP3A inhibition and the IC50s for P-gp and CYP3A inhibition allowed comparison of the relative inhibitory potency of the compds. on the two proteins' function.  The IC50s for P-gp inhibition ranged from 0.04 to 3.8 μM.  All compds. inhibited CYP3A with apparent Ki values of between 0.3 and 76 μM and IC50s between 1.5 and 50 μM.  However, no correlation was found between the extent of P-gp inhibition and CYP3A inhibition, and the ratio of the IC50 for CYP3A inhibition to the IC50 for P-gp inhibition varied from 1.1 to 125.  These results demonstrate that, although many P-gp inhibitors are potent inhibitors of CYP3A, a varying degree of selectivity is present.  The development and use of P-gp inhibitors with minimal or absent CYP3A inhibitory effects should decrease the impact of drug interactions on the therapeutic use of such compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCV7MbrQ_Rn7Vg90H21EOLACvtfcHk0liNMsZnfd0seQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXlsVOltL4%253D&md5=2993e6b2b3cd2b4c00dd4ca71b020ee2</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWandel%26aufirst%3DC.%26aulast%3DKim%26aufirst%3DR.%2BB.%26aulast%3DKajiji%26aufirst%3DS.%26aulast%3DGuengerich%26aufirst%3DF.%2BP.%26aulast%3DWilkinson%26aufirst%3DG.%2BR.%26aulast%3DWood%26aufirst%3DA.%2BA.%2BJ.%26atitle%3DP-glycoprotein%2520and%2520cytochrome%2520P-450%25203A%2520inhibition%253A%2520dissociation%2520of%2520inhibitory%2520potencies%26jtitle%3DCancer%2520Res.%26date%3D1999%26volume%3D59%26spage%3D3944%26epage%3D3948" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="note"><p class="first last"><i>In vivo</i> parameters for <b>4</b> were from IV dosing using a 50% PEG 200/10% EtOH/40% saline formulation. For rat (2 mg/kg) the following values were obtained: 28.6 mL/min/kg CL; 1.2 L/kg Vdss; 1.0 <i>t</i><sub>1/2</sub>; and 1.2 h·μg/mL AUC. For dog (2 mg/kg) the following values were obtained: 10.5 mL/min/kg CL; 0.4 L/kg Vdss; 1.3 <i>t</i><sub>1/2</sub>; and 3.8 h·μg/mL AUC. For monkey (4 mg/kg) the following values were obtained: 20.6 mL/min/kg CL; 0.6 L/kg <i>V</i><sub>dss</sub>; 1.6 <i>t</i><sub>1/2</sub>; and 3.8 h·μg/mL AUC.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grum-Tokars, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratia, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Begaye, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesecar, A.</span></span> <span> </span><span class="NLM_article-title">Evaluating the 3C-like protease activity of SARS-Coronavirus: recommendations for standardized assays for drug discovery</span>. <i>Virus Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">63</span>– <span class="NLM_lpage">73</span>, <span class="refDoi"> DOI: 10.1016/j.virusres.2007.02.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1016%2Fj.virusres.2007.02.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=17397958" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtlWksb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2008&pages=63-73&author=V.+Grum-Tokarsauthor=K.+Ratiaauthor=A.+Begayeauthor=S.+Bakerauthor=A.+Mesecar&title=Evaluating+the+3C-like+protease+activity+of+SARS-Coronavirus%3A+recommendations+for+standardized+assays+for+drug+discovery&doi=10.1016%2Fj.virusres.2007.02.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluating the 3C-like protease activity of SARS-Coronavirus: Recommendations for standardized assays for drug discovery</span></div><div class="casAuthors">Grum-Tokars, Valerie; Ratia, Kiira; Begaye, Adrian; Baker, Susan C.; Mesecar, Andrew D.</div><div class="citationInfo"><span class="NLM_cas:title">Virus Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">63-73</span>CODEN:
                <span class="NLM_cas:coden">VIREDF</span>;
        ISSN:<span class="NLM_cas:issn">0168-1702</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Although the initial outbreaks of the deadly coronavirus that causes severe acute respiratory syndrome (SARS-CoV) were controlled by public health measures, the development of vaccines and antiviral agents for SARS-CoV is essential for improving control and treatment of future outbreaks.  One potential target for SARS-CoV antiviral drug development is the 3C-like protease (3CLpro).  This enzyme is an attractive target since it is essential for viral replication, and since there are now a no. of high resoln. X-ray structures of SARS-CoV 3CLpro available making structure-based drug-design possible.  As a result, SARS-CoV 3CLpro has become the focus of numerous drug discovery efforts worldwide, but as a consequence, a variety of different 3CLpro expression constructs and kinetic assays have been independently developed making evaluation and comparison between potential inhibitors problematic.  Here, we review the literature focusing on different SARS-CoV 3CLpro expression constructs and assays used to measure enzymic activity.  Moreover, we provide exptl. evidence showing that the activity of 3CLpro enzyme is significantly reduced when non-native sequences or affinity-tags are added to the N- or C-termini of the enzyme, or when the enzyme used in assays is at concns. below the equil. dissocn. const. of the 3CLpro dimer.  We demonstrate for the first time the utility of a highly sensitive and novel Alexa488-QSY7 FRET-based peptide substrate designed for routine anal. and high-throughput screening, and show that kinetic consts. detd. from FRET-based assays that are uncorrected for inner-filter effects can lead to artifacts.  Finally, we evaluated the effects of common assay components including DTT, NaCl, EDTA and DMSO on enzymic activity, and we recommend standardized assay conditions and constructs for routine SARS-CoV 3CLpro assays to facilitate direct comparisons between SARS-CoV 3CLpro inhibitors under development worldwide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpev2bSotX3dLVg90H21EOLACvtfcHk0liNMsZnfd0seQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtlWksb8%253D&md5=4895fe12a3e04a2b157cd0317ec2a3da</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.virusres.2007.02.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.virusres.2007.02.015%26sid%3Dliteratum%253Aachs%26aulast%3DGrum-Tokars%26aufirst%3DV.%26aulast%3DRatia%26aufirst%3DK.%26aulast%3DBegaye%26aufirst%3DA.%26aulast%3DBaker%26aufirst%3DS.%26aulast%3DMesecar%26aufirst%3DA.%26atitle%3DEvaluating%2520the%25203C-like%2520protease%2520activity%2520of%2520SARS-Coronavirus%253A%2520recommendations%2520for%2520standardized%2520assays%2520for%2520drug%2520discovery%26jtitle%3DVirus%2520Res.%26date%3D2008%26volume%3D133%26spage%3D63%26epage%3D73%26doi%3D10.1016%2Fj.virusres.2007.02.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borenfreund, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puerner, J.</span></span> <span> </span><span class="NLM_article-title">Toxicity determined in vitro by morphological alterations and neutral red absorption</span>. <i>Toxicol. Lett.</i> <span class="NLM_year" style="font-weight: bold;">1985</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">119</span>– <span class="NLM_lpage">124</span>, <span class="refDoi"> DOI: 10.1016/0378-4274(85)90046-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1016%2F0378-4274%2885%2990046-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=3983963" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaL2MXhvVShu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=1985&pages=119-124&author=E.+Borenfreundauthor=J.+Puerner&title=Toxicity+determined+in+vitro+by+morphological+alterations+and+neutral+red+absorption&doi=10.1016%2F0378-4274%2885%2990046-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Toxicity determined in vitro by morphological alterations and neutral red absorption</span></div><div class="casAuthors">Borenfreund, Ellen; Puerner, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology Letters</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">2-3</span>),
    <span class="NLM_cas:pages">119-24</span>CODEN:
                <span class="NLM_cas:coden">TOLED5</span>;
        ISSN:<span class="NLM_cas:issn">0378-4274</span>.
    </div><div class="casAbstract">A method is described which combines the use of a visual morphol. cytotoxicity assay with a quant. neutral red  [553-24-2] spectrophotometric test, for the assessment of the effect of toxic agents on mouse fibroblast 3T3 cells in culture.  These sensitive and reproducible assays lend themselves to a screening procedure of potential toxicants which can help reduce the use of animals for toxicity testing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocpD9P9r7fAbVg90H21EOLACvtfcHk0liNMsZnfd0seQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXhvVShu7s%253D&md5=8bb53b9abc203b72c07e75668fac401f</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2F0378-4274%2885%2990046-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0378-4274%252885%252990046-3%26sid%3Dliteratum%253Aachs%26aulast%3DBorenfreund%26aufirst%3DE.%26aulast%3DPuerner%26aufirst%3DJ.%26atitle%3DToxicity%2520determined%2520in%2520vitro%2520by%2520morphological%2520alterations%2520and%2520neutral%2520red%2520absorption%26jtitle%3DToxicol.%2520Lett.%26date%3D1985%26volume%3D24%26spage%3D119%26epage%3D124%26doi%3D10.1016%2F0378-4274%2885%2990046-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smee, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurst, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clyde, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, C.</span></span> <span> </span><span class="NLM_article-title">Evaluation of cell viability dyes in antiviral assays with RNA viruses that exhibit different cytopathogenic properties</span>. <i>J. Virol. Methods</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>246</i></span>,  <span class="NLM_fpage">51</span>– <span class="NLM_lpage">57</span>, <span class="refDoi"> DOI: 10.1016/j.jviromet.2017.03.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1016%2Fj.jviromet.2017.03.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=28359770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvVKns70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=246&publication_year=2017&pages=51-57&author=D.+F.+Smeeauthor=B.+Hurstauthor=W.+Evansauthor=N.+Clydeauthor=S.+Wrightauthor=C.+Petersonauthor=K.-H.+Jungauthor=C.+Day&title=Evaluation+of+cell+viability+dyes+in+antiviral+assays+with+RNA+viruses+that+exhibit+different+cytopathogenic+properties&doi=10.1016%2Fj.jviromet.2017.03.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of cell viability dyes in antiviral assays with RNA viruses that exhibit different cytopathogenic properties</span></div><div class="casAuthors">Smee, Donald F.; Hurst, Brett L.; Evans, W. Joseph; Clyde, Nathan; Wright, Sean; Peterson, Christopher; Jung, Kie-Hoon; Day, Craig W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virological Methods</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">246</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">51-57</span>CODEN:
                <span class="NLM_cas:coden">JVMEDH</span>;
        ISSN:<span class="NLM_cas:issn">0166-0934</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Studies were conducted to det. the performance of four dyes in assessing antiviral activities of compds. against three RNA viruses with differing cytopathogenic properties.  Dyes included alamarBlue measured by absorbance (ALB-A) and fluorescence (ALB-F), neutral red (NR), Viral ToxGlo (VTG), and WST-1.  Viruses were chikungunya, dengue type 2, and Junin, which generally cause 100, 80-90, and 50% maximal cytopathic effect (CPE), resp., in Vero or Vero 76 cells Compds. evaluated were 6-azauridine, BCX-4430, 3-deazaguanine, EICAR, favipiravir, infergen, mycophenolic acid (MPA), ribavirin, and tiazofurin.  The 50% virus-inhibitory (EC50) values for each inhibitor and virus combination did not vary significantly based on the dye used.  However, dyes varied in distinguishing the vitality of virus-infected cultures when not all cells were killed by virus infection.  For example, VTG uptake into dengue-infected cells was nearly 50% when visual examn. showed only 10-20% cell survival.  ALB-A measured infected cell viability differently than ALB-F as follows: 16% vs. 32% (dengue-infected), resp., and 51% vs. 72% (Junin-infected), resp.  Cytotoxicity (CC50) assays with dyes in uninfected proliferating cells produced similar CC50 values for EICAR (1.5-8.9 μM) and MPA (0.8-2.5 μM). 6-Azauridine toxicity was 6.1-17.5 μM with NR, VTG, and WST-1, compared to 48-92 μM with ALB-A and ALB-F (P < 0.001).  Curiously, the CC50 values for 3-deazaguanine were 83-93 μM with ALB-F vs. 2.4-7.0 μM with all other dyes including ALB-A (P < 0.001).  Overall, ALB minimized the toxicities detected with these two inhibitors.  Because the choice of dyes affected CC50 values, this impacted on the resulting in vitro selectivity indexes (calcd. as CC50/EC50 ratio).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0QJD6MLX54bVg90H21EOLACvtfcHk0lhw0OIofmd-Fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmvVKns70%253D&md5=2e19131fe61c56db0d6e7a19c105f99c</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.jviromet.2017.03.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jviromet.2017.03.012%26sid%3Dliteratum%253Aachs%26aulast%3DSmee%26aufirst%3DD.%2BF.%26aulast%3DHurst%26aufirst%3DB.%26aulast%3DEvans%26aufirst%3DW.%26aulast%3DClyde%26aufirst%3DN.%26aulast%3DWright%26aufirst%3DS.%26aulast%3DPeterson%26aufirst%3DC.%26aulast%3DJung%26aufirst%3DK.-H.%26aulast%3DDay%26aufirst%3DC.%26atitle%3DEvaluation%2520of%2520cell%2520viability%2520dyes%2520in%2520antiviral%2520assays%2520with%2520RNA%2520viruses%2520that%2520exhibit%2520different%2520cytopathogenic%2520properties%26jtitle%3DJ.%2520Virol.%2520Methods%26date%3D2017%26volume%3D246%26spage%3D51%26epage%3D57%26doi%3D10.1016%2Fj.jviromet.2017.03.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scudiero, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoemaker, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paull, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monks, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tierney, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nofziger, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Currens, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seniff, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, M. R.</span></span> <span> </span><span class="NLM_article-title">Evaluation of a soluble tetrazolium/formazan assay for cell growth and drug sensitivity in culture using human and other tumor cell lines</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">4827</span>– <span class="NLM_lpage">4833</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=3409223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaL1cXls1arsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=1988&pages=4827-4833&author=D.+A.+Scudieroauthor=R.+H.+Shoemakerauthor=K.+D.+Paullauthor=A.+Monksauthor=S.+Tierneyauthor=T.+H.+Nofzigerauthor=M.+J.+Currensauthor=D.+Seniffauthor=M.+R.+Boyd&title=Evaluation+of+a+soluble+tetrazolium%2Fformazan+assay+for+cell+growth+and+drug+sensitivity+in+culture+using+human+and+other+tumor+cell+lines"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of a soluble tetrazolium/formazan assay for cell growth and drug sensitivity in culture using human and other tumor cell lines</span></div><div class="casAuthors">Scudiero, Dominic A.; Shoemaker, Robert H.; Paull, Kenneth D.; Monks, Anne; Tierney, Siobhan; Nofziger, Thomas H.; Currens, Michael J.; Seniff, Donna; Boyd, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4827-33</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    </div><div class="casAbstract">An automated microculture tetrazolium assay (MTA) involving DMSO solubilization of cellular-generated 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide (MTT)-formazan had several inherent disadvantages.  Possible alternative MTAs were developed utilizing a different tetrazolium reagent, 2,3-bis(2-methoxy4-nitro-5-sulfophenyl)-5-[(phenylamino)carbonyl]-2H-tetrazolium hydroxide (XTT), which is metabolically reduced in viable cells to a water-sol. formazan product.  This reagent allowed direct absorbance readings, therefore eliminating a solubilization step and shortening the microculture growth assay procedure.  Most human tumor cell lines examd. metabolized XTT less efficiently than MTT; however, the addn. of phenazine methosulfate (PMS) markedly enhanced cellular redn. of XTT.  In the presence of PMS, the XTT reagent yielded usable absorbance values for growth and drug sensitivity evaluations with a variety of cell lines.  Depending on the metabolic reductive capacity of a given cell line, the optimal conditions for a 4-h XTT incubation assay were 50 μg of XTT and 0.15-0.4 μg of PMS per well.  Drug profiles obtained with representative human tumor cell lines for several std. compds. utilizing the XTT-PMS methodol. were similar to the profiles obtained with MTT.  Addn. of PMS appeared to have little effect on the metab. of MTT.  The new XTT reagent thus provides for a simplified, in vitro cell growth assay with possible applicability to a variety of problems in cellular pharmacol. and biol.  However, the MTA using the XTT reagent still shares many of the limitations and potential pitfalls of MTT or other tetrazolium-based assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNCG5C-wpmkrVg90H21EOLACvtfcHk0lhw0OIofmd-Fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXls1arsrk%253D&md5=b88dc8255a07af053a91897fd6c451cc</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DScudiero%26aufirst%3DD.%2BA.%26aulast%3DShoemaker%26aufirst%3DR.%2BH.%26aulast%3DPaull%26aufirst%3DK.%2BD.%26aulast%3DMonks%26aufirst%3DA.%26aulast%3DTierney%26aufirst%3DS.%26aulast%3DNofziger%26aufirst%3DT.%2BH.%26aulast%3DCurrens%26aufirst%3DM.%2BJ.%26aulast%3DSeniff%26aufirst%3DD.%26aulast%3DBoyd%26aufirst%3DM.%2BR.%26atitle%3DEvaluation%2520of%2520a%2520soluble%2520tetrazolium%252Fformazan%2520assay%2520for%2520cell%2520growth%2520and%2520drug%2520sensitivity%2520in%2520culture%2520using%2520human%2520and%2520other%2520tumor%2520cell%2520lines%26jtitle%3DCancer%2520Res.%26date%3D1988%26volume%3D48%26spage%3D4827%26epage%3D4833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weislow, O. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiser, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fine, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoemaker, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, M. R.</span></span> <span> </span><span class="NLM_article-title">New soluble-formazan assay for HIV-1 cytopathic effects: application to high-flux screening of synthetic and natural products for AIDS-antiviral activity</span>. <i>J. Natl. Cancer Inst.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">577</span>– <span class="NLM_lpage">586</span>, <span class="refDoi"> DOI: 10.1093/jnci/81.8.577</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1093%2Fjnci%2F81.8.577" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=2495366" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaL1MXkvV2lt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=1989&pages=577-586&author=O.+S.+Weislowauthor=R.+Kiserauthor=D.+L.+Fineauthor=J.+Baderauthor=R.+H.+Shoemakerauthor=M.+R.+Boyd&title=New+soluble-formazan+assay+for+HIV-1+cytopathic+effects%3A+application+to+high-flux+screening+of+synthetic+and+natural+products+for+AIDS-antiviral+activity&doi=10.1093%2Fjnci%2F81.8.577"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">New soluble-formazan assay for HIV-1 cytopathic effects:  application to high-flux screening of synthetic and natural products for AIDS-antiviral activity</span></div><div class="casAuthors">Weislow, Owen S.; Kiser, Rebecca; Fine, Donald L.; Bader, John; Shoemaker, Robert H.; Boyd, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">577-86</span>CODEN:
                <span class="NLM_cas:coden">JNCIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    </div><div class="casAbstract">An effective and optimally safe microculture method was developed for the rapid and convenient assay of the in vitro cytopathic effects of human immunodeficiency virus (HIV-1) on human lymphoblastoid or other suitable host cells.  The assay procedure is applicable to the evaluation of drug effects on in vitro infections induced directly in cultured host cells by cell-free HIV-1 or by coculture with H9 cells chronically infected with HIV-1.  The assay uses a newly developed tetrazolium reagent that is metabolically reduced by viable cells to yield a sol., colored formazan product measureable by conventional colorimetric techniques.  This simple microassay minimizes the no. of plate manipulations typically required with other assay methods and, coupled with computerized data collection and anal., facilitates large-scale screening of agents for potential antiviral activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDJZMdwKZfgbVg90H21EOLACvtfcHk0lhw0OIofmd-Fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXkvV2lt78%253D&md5=c39a027f4ddc6bb5d7b1108c1b128be6</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2F81.8.577&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252F81.8.577%26sid%3Dliteratum%253Aachs%26aulast%3DWeislow%26aufirst%3DO.%2BS.%26aulast%3DKiser%26aufirst%3DR.%26aulast%3DFine%26aufirst%3DD.%2BL.%26aulast%3DBader%26aufirst%3DJ.%26aulast%3DShoemaker%26aufirst%3DR.%2BH.%26aulast%3DBoyd%26aufirst%3DM.%2BR.%26atitle%3DNew%2520soluble-formazan%2520assay%2520for%2520HIV-1%2520cytopathic%2520effects%253A%2520application%2520to%2520high-flux%2520screening%2520of%2520synthetic%2520and%2520natural%2520products%2520for%2520AIDS-antiviral%2520activity%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D1989%26volume%3D81%26spage%3D577%26epage%3D586%26doi%3D10.1093%2Fjnci%2F81.8.577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hippenmeyer, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dilworth, V. M.</span></span> <span> </span><span class="NLM_article-title">A rapid assay for determination of antiviral activity against human cytomegalovirus</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">35</span>– <span class="NLM_lpage">42</span>, <span class="refDoi"> DOI: 10.1016/0166-3542(95)00976-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1016%2F0166-3542%2895%2900976-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=8863994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK28XksF2htLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=1996&pages=35-42&author=P.+J.+Hippenmeyerauthor=V.+M.+Dilworth&title=A+rapid+assay+for+determination+of+antiviral+activity+against+human+cytomegalovirus&doi=10.1016%2F0166-3542%2895%2900976-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">A rapid assay for determination of antiviral activity against human cytomegalovirus</span></div><div class="casAuthors">Hippenmeyer, Paul J.; Dilworth, Vickie M.</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">35-42</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">RC256, the recombinant human cytomegalovirus (HCMV) which expresses β-galactosidase, was used as a tool for rapid screening of compds. for antiviral activity.  The effective concn. of antiviral compd. needed to inhibit RC256 was identical to the concns. needed to inhibit other strains of HCMV as measured by plaque redn. assay or virus yield redn. assay.  Measurement of β-galactosidase activity in infected cell lysates allowed detn. of effective concns. 48 h postinfection with results comparable to the longer, more laborious assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDFUA4a6va1bVg90H21EOLACvtfcHk0lh7m-ju6ZnkWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XksF2htLk%253D&md5=33af6dd34fd78e65c6a9ab78813c5dcf</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2F0166-3542%2895%2900976-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0166-3542%252895%252900976-0%26sid%3Dliteratum%253Aachs%26aulast%3DHippenmeyer%26aufirst%3DP.%2BJ.%26aulast%3DDilworth%26aufirst%3DV.%2BM.%26atitle%3DA%2520rapid%2520assay%2520for%2520determination%2520of%2520antiviral%2520activity%2520against%2520human%2520cytomegalovirus%26jtitle%3DAntiviral%2520Res.%26date%3D1996%26volume%3D32%26spage%3D35%26epage%3D42%26doi%3D10.1016%2F0166-3542%2895%2900976-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patick, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binford, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brothers, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diem, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maldonado, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prins, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuhrman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meador, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zalman, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Worland, S. T.</span></span> <span> </span><span class="NLM_article-title">In vitro antiviral activity of AG7088, a potent inhibitor of human rhinovirus 3C protease</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">2444</span>– <span class="NLM_lpage">2450</span>, <span class="refDoi"> DOI: 10.1128/AAC.43.10.2444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1128%2FAAC.43.10.2444" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10508022" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK1MXms1KqsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=1999&pages=2444-2450&author=A.+K.+Patickauthor=S.+L.+Binfordauthor=M.+A.+Brothersauthor=R.+L.+Jacksonauthor=C.+E.+Fordauthor=M.+D.+Diemauthor=F.+Maldonadoauthor=P.+S.+Dragovichauthor=R.+Zhouauthor=T.+J.+Prinsauthor=S.+A.+Fuhrmanauthor=J.+W.+Meadorauthor=L.+S.+Zalmanauthor=D.+A.+Matthewsauthor=S.+T.+Worland&title=In+vitro+antiviral+activity+of+AG7088%2C+a+potent+inhibitor+of+human+rhinovirus+3C+protease&doi=10.1128%2FAAC.43.10.2444"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro antiviral activity of AG7088, a potent inhibitor of human rhinovirus 3C protease</span></div><div class="casAuthors">Patick, A. K.; Binford, S. L.; Brothers, M. A.; Jackson, R. L.; Ford, C. E.; Diem, M. D.; Maldonado, F.; Dragovich, P. S.; Zhou, R.; Prins, T. J.; Fuhrman, S. A.; Meador, J. W.; Zalman, L. S.; Matthews, D. A.; Worland, S. T.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2444-2450</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">AG-7088 is a potent, irreversible inhibitor of human rhinovirus (HRV) 3C protease (inactivation rate const. kobs/[I] = 1,470,000±440,000 M-1 s-1 for HRV 14) that was discovered by protein structure-based drug design methodologies.  In H1-HeLa and MRC-5 cell protection assays, AG-7088 inhibited the replication of all HRV serotypes (48 of 48) tested with a mean 50% EC50 of 0.023 μM (range 0.003-0.081 μM) and a mean EC90 of 0.082 μM (range 0.018-0.261 μM), as well as that of related picornaviruses including coxsackieviruses A21 and B3, enterovirus 70, and echovirus 11.  No decreases in the antiviral activity of AG-7088 were obsd. when assays were performed in the presence of α1-acid glycoprotein or mucin proteins found in nasal secretions.  The 50% cytotoxic concn. of AG-7088 was >1000 μM, yielding a therapeutic index of >12,346 to >333,333.  In a single-cycle, time-of-addn. assay, AG-7088 had antiviral activity when added up to 6 h after infection.  A compd. targeting viral attachment and/or uncoating (WIN-51711) was effective only when added at the initiation of virus infection.  Direct inhibition of 3C proteolytic activity in infected cells treated with AG-7088 was demonstrated by SDS-polyacrylamide gel electrophoresis anal. of radiolabeled proteins, which showed a dose-dependent accumulation of viral precursor polyproteins and a decrease of processed protein products.  The broad spectrum of antiviral activity of AG-7088, combined with its efficacy even when added late in the virus life cycle, highlights the advantages of 3C protease as a target and suggests that AG7088 is a promising clin. candidate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojpMqAUz2M3LVg90H21EOLACvtfcHk0lh7m-ju6ZnkWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXms1KqsL8%253D&md5=70e98480d38e133c96866005b0578f75</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1128%2FAAC.43.10.2444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.43.10.2444%26sid%3Dliteratum%253Aachs%26aulast%3DPatick%26aufirst%3DA.%2BK.%26aulast%3DBinford%26aufirst%3DS.%2BL.%26aulast%3DBrothers%26aufirst%3DM.%2BA.%26aulast%3DJackson%26aufirst%3DR.%2BL.%26aulast%3DFord%26aufirst%3DC.%2BE.%26aulast%3DDiem%26aufirst%3DM.%2BD.%26aulast%3DMaldonado%26aufirst%3DF.%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DPrins%26aufirst%3DT.%2BJ.%26aulast%3DFuhrman%26aufirst%3DS.%2BA.%26aulast%3DMeador%26aufirst%3DJ.%2BW.%26aulast%3DZalman%26aufirst%3DL.%2BS.%26aulast%3DMatthews%26aufirst%3DD.%2BA.%26aulast%3DWorland%26aufirst%3DS.%2BT.%26atitle%3DIn%2520vitro%2520antiviral%2520activity%2520of%2520AG7088%252C%2520a%2520potent%2520inhibitor%2520of%2520human%2520rhinovirus%25203C%2520protease%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1999%26volume%3D43%26spage%3D2444%26epage%3D2450%26doi%3D10.1128%2FAAC.43.10.2444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blight, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolykhalov, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, C. M.</span></span> <span> </span><span class="NLM_article-title">Efficient initiation of HCV RNA replication in cell culture</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">1972</span>– <span class="NLM_lpage">1974</span>, <span class="refDoi"> DOI: 10.1126/science.290.5498.1972</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1126%2Fscience.290.5498.1972" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=11110665" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BD3cXoslKgsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2000&pages=1972-1974&author=K.+J.+Blightauthor=A.+A.+Kolykhalovauthor=C.+M.+Rice&title=Efficient+initiation+of+HCV+RNA+replication+in+cell+culture&doi=10.1126%2Fscience.290.5498.1972"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient initiation of HCV RNA replication in cell culture</span></div><div class="casAuthors">Blight, Keril J.; Kolykhalov, Alexander A.; Rice, Charles M.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">5498</span>),
    <span class="NLM_cas:pages">1972-1974</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Hepatitis C virus (HCV) infection is a global health problem affecting an estd. 170 million individuals worldwide.  We report the identification of multiple independent adaptive mutations that cluster in the HCV nonstructural protein NS5A and confer increased replicative ability in vitro.  Among these adaptive mutations were a single amino acid substitution that allowed HCV RNA replication in 10% of transfected hepatoma cells and a deletion of 47 amino acids encompassing the interferon (IFN) sensitivity detg. region (ISDR).  Independent of the ISDR, IFN-α rapidly inhibited HCV RNA replication in vitro.  This work establishes a robust, cell-based system for genetic and functional analyses of HCV replication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPYOuqOjy-ZbVg90H21EOLACvtfcHk0lh7m-ju6ZnkWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXoslKgsL4%253D&md5=67eb65d15efc36854faf3940db9a5d91</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1126%2Fscience.290.5498.1972&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.290.5498.1972%26sid%3Dliteratum%253Aachs%26aulast%3DBlight%26aufirst%3DK.%2BJ.%26aulast%3DKolykhalov%26aufirst%3DA.%2BA.%26aulast%3DRice%26aufirst%3DC.%2BM.%26atitle%3DEfficient%2520initiation%2520of%2520HCV%2520RNA%2520replication%2520in%2520cell%2520culture%26jtitle%3DScience%26date%3D2000%26volume%3D290%26spage%3D1972%26epage%3D1974%26doi%3D10.1126%2Fscience.290.5498.1972" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Solowiej, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, B.</span></span> <span> </span><span class="NLM_article-title">Steady-state and pre-steady-state kinetic evaluation of severe acute respiratory syndrome coronavirus (SARS-CoV) 3CL<sup>pro</sup> cysteine protease: development of an ion-pair model for catalysis</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">2617</span>, <span class="refDoi"> DOI: 10.1021/bi702107v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi702107v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1OmtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=2617&author=J.+Solowiejauthor=J.+Thomsonauthor=K.+Ryanauthor=C.+Luoauthor=M.+Heauthor=J.+Louauthor=B.+Murray&title=Steady-state+and+pre-steady-state+kinetic+evaluation+of+severe+acute+respiratory+syndrome+coronavirus+%28SARS-CoV%29+3CLpro+cysteine+protease%3A+development+of+an+ion-pair+model+for+catalysis&doi=10.1021%2Fbi702107v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Steady-State and Pre-Steady-State Kinetic Evaluation of Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) 3CLpro Cysteine Protease: Development of an Ion-Pair Model for Catalysis</span></div><div class="casAuthors">Solowiej, James; Thomson, James A.; Ryan, Kevin; Luo, Chun; He, Mingying; Lou, Jihong; Murray, Brion W.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2617-2630</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Severe acute respiratory syndrome (SARS) was a worldwide epidemic caused by a coronavirus that has a cysteine protease (3CLpro) essential to its life cycle.  Steady-state and pre-steady-state kinetic methods were used with highly active 3CLpro to characterize the reaction mechanism.  We show that 3CLpro has mechanistic features common and disparate to the archetypical proteases papain and chymotrypsin.  The kinetic mechanism for 3CLpro-mediated ester hydrolysis, including the individual rate consts., is consistent with a simple double displacement mechanism.  The pre-steady-state burst rate was independent of ester substrate concn. indicating a high commitment to catalysis.  When homologous peptidic amide and ester substrates were compared, a series of interesting observations emerged.  Despite a 2000-fold difference in nonenzymic reactivity, highly related amide and ester substrates were found to have similar kinetic parameters in both the steady-state and pre-steady-state.  Steady-state solvent isotope effect (SIE) studies showed an inverse SIE for the amide but not ester substrates.  Evaluation of the SIE in the pre-steady-state revealed normal SIEs for both amide and ester burst rates.  Proton inventory (PI) studies on amide peptide hydrolysis were consistent with two proton-transfer reactions in the transition state while the ester data was consistent with a single proton-transfer reaction.  Finally, the pH-inactivation profile of 3CLpro with iodoacetamide is indicative of an ion-pair mechanism.  Taken together, the data are consistent with a 3CLpro mechanism that utilizes an "electrostatic" trigger to initiate the acylation reaction, a cysteine-histidine catalytic dyad ion pair, an enzyme-facilitated release of P1, and a general base-catalyzed deacylation reaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpO92WZdvoiorVg90H21EOLACvtfcHk0lh7m-ju6ZnkWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1OmtrY%253D&md5=77d755ed4262558b065ea6246a1f57ae</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fbi702107v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi702107v%26sid%3Dliteratum%253Aachs%26aulast%3DSolowiej%26aufirst%3DJ.%26aulast%3DThomson%26aufirst%3DJ.%26aulast%3DRyan%26aufirst%3DK.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DLou%26aufirst%3DJ.%26aulast%3DMurray%26aufirst%3DB.%26atitle%3DSteady-state%2520and%2520pre-steady-state%2520kinetic%2520evaluation%2520of%2520severe%2520acute%2520respiratory%2520syndrome%2520coronavirus%2520%2528SARS-CoV%2529%25203CLpro%2520cysteine%2520protease%253A%2520development%2520of%2520an%2520ion-pair%2520model%2520for%2520catalysis%26jtitle%3DBiochemistry%26date%3D2008%26volume%3D47%26spage%3D2617%26doi%3D10.1021%2Fbi702107v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':[],'ref35':['cit35'],'ref36':[],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':[],'ref41':[],'ref42':[],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':[],'ref47':['cit47'],'ref48':['cit48'],'ref49':[],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 46 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Chamandi S. Dampalla, Yunjeong Kim, Naemi Bickmeier, Athri D. Rathnayake, Harry Nhat Nguyen, Jian Zheng, Maithri M. Kashipathy, Matthew A. Baird, Kevin P. Battaile, Scott Lovell, Stanley Perlman, Kyeong-Ok Chang, <span class="NLM_string-name hlFld-ContribAuthor">William C. Groutas</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Guided Design of Conformationally Constrained Cyclohexane Inhibitors of Severe Acute Respiratory Syndrome Coronavirus-2 3CL Protease. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (14)
                                     , 10047-10058. <a href="https://doi.org/10.1021/acs.jmedchem.1c00319" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00319</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00319&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00319%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Guided%252BDesign%252Bof%252BConformationally%252BConstrained%252BCyclohexane%252BInhibitors%252Bof%252BSevere%252BAcute%252BRespiratory%252BSyndrome%252BCoronavirus-2%252B3CL%252BProtease%26aulast%3DDampalla%26aufirst%3DChamandi%2BS.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D20022021%26date%3D02072021%26volume%3D64%26issue%3D14%26spage%3D10047%26epage%3D10058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Bing Bai, Alexandr Belovodskiy, Mostofa Hena, Appan Srinivas Kandadai, Michael A. Joyce, Holly A. Saffran, Justin A. Shields, Muhammad Bashir Khan, Elena Arutyunova, Jimmy Lu, Sardeev K. Bajwa, Darren Hockman, Conrad Fischer, Tess Lamer, Wayne Vuong, Marco J. van Belkum, Zhengxian Gu, Fusen Lin, Yanhua Du, Jia Xu, Mohammad Rahim, Howard S. Young, John C. Vederas, D. Lorne Tyrrell, M. Joanne Lemieux, <span class="NLM_string-name hlFld-ContribAuthor">James A. Nieman</span>. </span><span class="cited-content_cbyCitation_article-title">Peptidomimetic α-Acyloxymethylketone Warheads with Six-Membered Lactam P1 Glutamine Mimic: SARS-CoV-2 3CL Protease Inhibition, Coronavirus Antiviral Activity, and in Vitro Biological Stability. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00616&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00616%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DPeptidomimetic%252B%2525CE%2525B1-Acyloxymethylketone%252BWarheads%252Bwith%252BSix-Membered%252BLactam%252BP1%252BGlutamine%252BMimic%25253A%252BSARS-CoV-2%252B3CL%252BProtease%252BInhibition%25252C%252BCoronavirus%252BAntiviral%252BActivity%25252C%252Band%252Bin%252BVitro%252BBiological%252BStability%26aulast%3DBai%26aufirst%3DBing%26date%3D2021%26date%3D2021%26date%3D04042021%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Daniel W. Kneller, Gwyndalyn Phillips, Kevin L. Weiss, Qiu Zhang, Leighton Coates, <span class="NLM_string-name hlFld-ContribAuthor">Andrey Kovalevsky</span>. </span><span class="cited-content_cbyCitation_article-title">Direct Observation of Protonation State Modulation in SARS-CoV-2 Main Protease upon Inhibitor Binding with Neutron Crystallography. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (8)
                                     , 4991-5000. <a href="https://doi.org/10.1021/acs.jmedchem.1c00058" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00058</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00058%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDirect%252BObservation%252Bof%252BProtonation%252BState%252BModulation%252Bin%252BSARS-CoV-2%252BMain%252BProtease%252Bupon%252BInhibitor%252BBinding%252Bwith%252BNeutron%252BCrystallography%26aulast%3DKneller%26aufirst%3DDaniel%2BW.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D13012021%26date%3D23032021%26volume%3D64%26issue%3D8%26spage%3D4991%26epage%3D5000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Wenhao Dai, Dirk Jochmans, Hang Xie, Hang Yang, Jian Li, Haixia Su, Di Chang, Jiang Wang, Jingjing Peng, Lili Zhu, Yong Nian, Rolf Hilgenfeld, Hualiang Jiang, Kaixian Chen, Leike Zhang, Yechun Xu, Johan Neyts, <span class="NLM_string-name hlFld-ContribAuthor">Hong Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Design, Synthesis, and Biological Evaluation of Peptidomimetic Aldehydes as Broad-Spectrum Inhibitors against Enterovirus and SARS-CoV-2. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02258%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252BSynthesis%25252C%252Band%252BBiological%252BEvaluation%252Bof%252BPeptidomimetic%252BAldehydes%252Bas%252BBroad-Spectrum%252BInhibitors%252Bagainst%252BEnterovirus%252Band%252BSARS-CoV-2%26aulast%3DDai%26aufirst%3DWenhao%26date%3D2021%26date%3D2020%26date%3D29122020%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Carlos A. Ramos-Guzmán, J. Javier Ruiz-Pernía, <span class="NLM_string-name hlFld-ContribAuthor">Iñaki Tuñón</span>. </span><span class="cited-content_cbyCitation_article-title">Multiscale Simulations of SARS-CoV-2 3CL Protease Inhibition with Aldehyde Derivatives. Role of Protein and Inhibitor Conformational Changes in the Reaction Mechanism. </span><span class="cited-content_cbyCitation_journal-name">ACS Catalysis</span><span> <strong>2021,</strong> <em>11 </em>
                                    (7)
                                     , 4157-4168. <a href="https://doi.org/10.1021/acscatal.0c05522" title="DOI URL">https://doi.org/10.1021/acscatal.0c05522</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acscatal.0c05522&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facscatal.0c05522%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Catalysis%26atitle%3DMultiscale%252BSimulations%252Bof%252BSARS-CoV-2%252B3CL%252BProtease%252BInhibition%252Bwith%252BAldehyde%252BDerivatives.%252BRole%252Bof%252BProtein%252Band%252BInhibitor%252BConformational%252BChanges%252Bin%252Bthe%252BReaction%252BMechanism%26aulast%3DRamos-Guzm%25C3%25A1n%26aufirst%3DCarlos%2BA.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D2021%26date%3D16122020%26date%3D15022021%26date%3D25032021%26volume%3D11%26issue%3D7%26spage%3D4157%26epage%3D4168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Stephen Pelly, <span class="NLM_string-name hlFld-ContribAuthor">Dennis Liotta</span>. </span><span class="cited-content_cbyCitation_article-title">Potent SARS-CoV-2 Direct-Acting Antivirals Provide an Important Complement to COVID-19 Vaccines. </span><span class="cited-content_cbyCitation_journal-name">ACS Central Science</span><span> <strong>2021,</strong> <em>7 </em>
                                    (3)
                                     , 396-399. <a href="https://doi.org/10.1021/acscentsci.1c00258" title="DOI URL">https://doi.org/10.1021/acscentsci.1c00258</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acscentsci.1c00258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facscentsci.1c00258%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Central%2520Science%26atitle%3DPotent%252BSARS-CoV-2%252BDirect-Acting%252BAntivirals%252BProvide%252Ban%252BImportant%252BComplement%252Bto%252BCOVID-19%252BVaccines%26aulast%3DPelly%26aufirst%3DStephen%26date%3D2021%26date%3D2021%26date%3D03032021%26volume%3D7%26issue%3D3%26spage%3D396%26epage%3D399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Neha Verma, Jack A. Henderson, <span class="NLM_string-name hlFld-ContribAuthor">Jana Shen</span>. </span><span class="cited-content_cbyCitation_article-title">Proton-Coupled Conformational Activation of SARS Coronavirus Main Proteases and Opportunity for Designing Small-Molecule Broad-Spectrum Targeted Covalent Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2020,</strong> <em>142 </em>
                                    (52)
                                     , 21883-21890. <a href="https://doi.org/10.1021/jacs.0c10770" title="DOI URL">https://doi.org/10.1021/jacs.0c10770</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.0c10770&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.0c10770%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DProton-Coupled%252BConformational%252BActivation%252Bof%252BSARS%252BCoronavirus%252BMain%252BProteases%252Band%252BOpportunity%252Bfor%252BDesigning%252BSmall-Molecule%252BBroad-Spectrum%252BTargeted%252BCovalent%252BInhibitors%26aulast%3DVerma%26aufirst%3DNeha%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D11102020%26date%3D15122020%26volume%3D142%26issue%3D52%26spage%3D21883%26epage%3D21890" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Rolando Cannalire, Carmen Cerchia, Andrea R. Beccari, Francesco Saverio Di Leva, <span class="NLM_string-name hlFld-ContribAuthor">Vincenzo Summa</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting SARS-CoV-2 Proteases and Polymerase for COVID-19 Treatment: State of the Art and Future Opportunities. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01140%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BSARS-CoV-2%252BProteases%252Band%252BPolymerase%252Bfor%252BCOVID-19%252BTreatment%25253A%252BState%252Bof%252Bthe%252BArt%252Band%252BFuture%252BOpportunities%26aulast%3DCannalire%26aufirst%3DRolando%26date%3D2020%26date%3D2020%26date%3D02072020%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wayne  Vuong</span>, <span class="hlFld-ContribAuthor ">Conrad  Fischer</span>, <span class="hlFld-ContribAuthor ">Muhammad Bashir  Khan</span>, <span class="hlFld-ContribAuthor ">Marco J.  van Belkum</span>, <span class="hlFld-ContribAuthor ">Tess  Lamer</span>, <span class="hlFld-ContribAuthor ">Kurtis D.  Willoughby</span>, <span class="hlFld-ContribAuthor ">Jimmy  Lu</span>, <span class="hlFld-ContribAuthor ">Elena  Arutyunova</span>, <span class="hlFld-ContribAuthor ">Michael A.  Joyce</span>, <span class="hlFld-ContribAuthor ">Holly A.  Saffran</span>, <span class="hlFld-ContribAuthor ">Justin A.  Shields</span>, <span class="hlFld-ContribAuthor ">Howard S.  Young</span>, <span class="hlFld-ContribAuthor ">James A.  Nieman</span>, <span class="hlFld-ContribAuthor ">D. Lorne  Tyrrell</span>, <span class="hlFld-ContribAuthor ">M. Joanne  Lemieux</span>, <span class="hlFld-ContribAuthor ">John C.  Vederas</span>. </span><span class="cited-content_cbyCitation_article-title">Improved SARS-CoV-2 Mpro inhibitors based on feline antiviral drug GC376: Structural enhancements, increased solubility, and micellar studies. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>222 </em>, 113584. <a href="https://doi.org/10.1016/j.ejmech.2021.113584" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113584</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113584%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DImproved%252BSARS-CoV-2%252BMpro%252Binhibitors%252Bbased%252Bon%252Bfeline%252Bantiviral%252Bdrug%252BGC376%25253A%252BStructural%252Benhancements%25252C%252Bincreased%252Bsolubility%25252C%252Band%252Bmicellar%252Bstudies%26aulast%3DVuong%26aufirst%3DWayne%26date%3D2021%26volume%3D222%26spage%3D113584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Colleen B  Jonsson</span>, <span class="hlFld-ContribAuthor ">Jennifer E  Golden</span>, <span class="hlFld-ContribAuthor ">Bernd  Meibohm</span>. </span><span class="cited-content_cbyCitation_article-title">Time to ‘Mind the Gap’ in novel small molecule drug discovery for direct-acting antivirals for SARS-CoV-2. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Virology</span><span> <strong>2021,</strong> <em>50 </em>, 1-7. <a href="https://doi.org/10.1016/j.coviro.2021.06.008" title="DOI URL">https://doi.org/10.1016/j.coviro.2021.06.008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.coviro.2021.06.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.coviro.2021.06.008%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Virology%26atitle%3DTime%252Bto%252B%2525E2%252580%252598Mind%252Bthe%252BGap%2525E2%252580%252599%252Bin%252Bnovel%252Bsmall%252Bmolecule%252Bdrug%252Bdiscovery%252Bfor%252Bdirect-acting%252Bantivirals%252Bfor%252BSARS-CoV-2%26aulast%3DJonsson%26aufirst%3DColleen%2BB%26date%3D2021%26volume%3D50%26spage%3D1%26epage%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Robert M  Cox</span>, <span class="hlFld-ContribAuthor ">Richard K  Plemper</span>. </span><span class="cited-content_cbyCitation_article-title">The impact of high-resolution structural data on stemming the COVID-19 pandemic. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Virology</span><span> <strong>2021,</strong> <em>49 </em>, 127-138. <a href="https://doi.org/10.1016/j.coviro.2021.05.005" title="DOI URL">https://doi.org/10.1016/j.coviro.2021.05.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.coviro.2021.05.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.coviro.2021.05.005%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Virology%26atitle%3DThe%252Bimpact%252Bof%252Bhigh-resolution%252Bstructural%252Bdata%252Bon%252Bstemming%252Bthe%252BCOVID-19%252Bpandemic%26aulast%3DCox%26aufirst%3DRobert%2BM%26date%3D2021%26volume%3D49%26spage%3D127%26epage%3D138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Varpu  Marjomäki</span>, <span class="hlFld-ContribAuthor ">Kerttu  Kalander</span>, <span class="hlFld-ContribAuthor ">Maarit  Hellman</span>, <span class="hlFld-ContribAuthor ">Perttu  Permi</span>. </span><span class="cited-content_cbyCitation_article-title">Enteroviruses and coronaviruses: similarities and therapeutic targets. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Targets</span><span> <strong>2021,</strong> <em>7 </em>, 1-11. <a href="https://doi.org/10.1080/14728222.2021.1952985" title="DOI URL">https://doi.org/10.1080/14728222.2021.1952985</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/14728222.2021.1952985&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F14728222.2021.1952985%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Targets%26atitle%3DEnteroviruses%252Band%252Bcoronaviruses%25253A%252Bsimilarities%252Band%252Btherapeutic%252Btargets%26aulast%3DMarjom%25C3%25A4ki%26aufirst%3DVarpu%26date%3D2021%26date%3D2021%26volume%3D7%26spage%3D1%26epage%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kaiming  Tao</span>, <span class="hlFld-ContribAuthor ">Philip L.  Tzou</span>, <span class="hlFld-ContribAuthor ">Janin  Nouhin</span>, <span class="hlFld-ContribAuthor ">Hector  Bonilla</span>, <span class="hlFld-ContribAuthor ">Prasanna  Jagannathan</span>, <span class="hlFld-ContribAuthor ">Robert W.  Shafer</span>. </span><span class="cited-content_cbyCitation_article-title">SARS-CoV-2 Antiviral Therapy. </span><span class="cited-content_cbyCitation_journal-name">Clinical Microbiology Reviews</span><span> <strong>2021,</strong> <em>7 </em><a href="https://doi.org/10.1128/CMR.00109-21" title="DOI URL">https://doi.org/10.1128/CMR.00109-21</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1128/CMR.00109-21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1128%2FCMR.00109-21%26sid%3Dliteratum%253Aachs%26jtitle%3DClinical%2520Microbiology%2520Reviews%26atitle%3DSARS-CoV-2%252BAntiviral%252BTherapy%26aulast%3DTao%26aufirst%3DKaiming%26date%3D2021%26date%3D2021%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Huihui  Yang</span>, <span class="hlFld-ContribAuthor ">Jinfei  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">A review of the latest research on M
              pro
              targeting SARS-COV inhibitors. </span><span class="cited-content_cbyCitation_journal-name">RSC Medicinal Chemistry</span><span> <strong>2021,</strong> <em>12 </em>
                                    (7)
                                     , 1026-1036. <a href="https://doi.org/10.1039/D1MD00066G" title="DOI URL">https://doi.org/10.1039/D1MD00066G</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1MD00066G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1MD00066G%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Medicinal%2520Chemistry%26atitle%3DA%252Breview%252Bof%252Bthe%252Blatest%252Bresearch%252Bon%252BM%252Bpro%252Btargeting%252BSARS-COV%252Binhibitors%26aulast%3DYang%26aufirst%3DHuihui%26date%3D2021%26date%3D2021%26volume%3D12%26issue%3D7%26spage%3D1026%26epage%3D1036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chamandi S.  Dampalla</span>, <span class="hlFld-ContribAuthor ">Jian  Zheng</span>, <span class="hlFld-ContribAuthor ">Krishani Dinali  Perera</span>, <span class="hlFld-ContribAuthor ">Lok-Yin Roy  Wong</span>, <span class="hlFld-ContribAuthor ">David K.  Meyerholz</span>, <span class="hlFld-ContribAuthor ">Harry Nhat  Nguyen</span>, <span class="hlFld-ContribAuthor ">Maithri M.  Kashipathy</span>, <span class="hlFld-ContribAuthor ">Kevin P.  Battaile</span>, <span class="hlFld-ContribAuthor ">Scott  Lovell</span>, <span class="hlFld-ContribAuthor ">Yunjeong  Kim</span>, <span class="hlFld-ContribAuthor ">Stanley  Perlman</span>, <span class="hlFld-ContribAuthor ">William C.  Groutas</span>, <span class="hlFld-ContribAuthor ">Kyeong-Ok  Chang</span>. </span><span class="cited-content_cbyCitation_article-title">Postinfection treatment with a protease inhibitor increases survival of mice with a fatal SARS-CoV-2 infection. </span><span class="cited-content_cbyCitation_journal-name">Proceedings of the National Academy of Sciences</span><span> <strong>2021,</strong> <em>118 </em>
                                    (29)
                                     , e2101555118. <a href="https://doi.org/10.1073/pnas.2101555118" title="DOI URL">https://doi.org/10.1073/pnas.2101555118</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1073/pnas.2101555118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1073%2Fpnas.2101555118%26sid%3Dliteratum%253Aachs%26jtitle%3DProceedings%2520of%2520the%2520National%2520Academy%2520of%2520Sciences%26atitle%3DPostinfection%252Btreatment%252Bwith%252Ba%252Bprotease%252Binhibitor%252Bincreases%252Bsurvival%252Bof%252Bmice%252Bwith%252Ba%252Bfatal%252BSARS-CoV-2%252Binfection%26aulast%3DDampalla%26aufirst%3DChamandi%2BS.%26date%3D2021%26date%3D2021%26volume%3D118%26issue%3D29%26spage%3De2101555118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jennifer C.  Milligan</span>, <span class="hlFld-ContribAuthor ">Theresa U.  Zeisner</span>, <span class="hlFld-ContribAuthor ">George  Papageorgiou</span>, <span class="hlFld-ContribAuthor ">Dhira  Joshi</span>, <span class="hlFld-ContribAuthor ">Christelle  Soudy</span>, <span class="hlFld-ContribAuthor ">Rachel  Ulferts</span>, <span class="hlFld-ContribAuthor ">Mary  Wu</span>, <span class="hlFld-ContribAuthor ">Chew Theng  Lim</span>, <span class="hlFld-ContribAuthor ">Kang Wei  Tan</span>, <span class="hlFld-ContribAuthor ">Florian  Weissmann</span>, <span class="hlFld-ContribAuthor ">Berta  Canal</span>, <span class="hlFld-ContribAuthor ">Ryo  Fujisawa</span>, <span class="hlFld-ContribAuthor ">Tom  Deegan</span>, <span class="hlFld-ContribAuthor ">Hema  Nagaraj</span>, <span class="hlFld-ContribAuthor ">Ganka  Bineva-Todd</span>, <span class="hlFld-ContribAuthor ">Clovis  Basier</span>, <span class="hlFld-ContribAuthor ">Joseph F.  Curran</span>, <span class="hlFld-ContribAuthor ">Michael  Howell</span>, <span class="hlFld-ContribAuthor ">Rupert  Beale</span>, <span class="hlFld-ContribAuthor ">Karim  Labib</span>, <span class="hlFld-ContribAuthor ">Nicola  O'Reilly</span>, <span class="hlFld-ContribAuthor ">John F.X.  Diffley</span>. </span><span class="cited-content_cbyCitation_article-title">Identifying SARS-CoV-2 antiviral compounds by screening for small molecule inhibitors of Nsp5 main protease. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Journal</span><span> <strong>2021,</strong> <em>478 </em>
                                    (13)
                                     , 2499-2515. <a href="https://doi.org/10.1042/BCJ20210197" title="DOI URL">https://doi.org/10.1042/BCJ20210197</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/BCJ20210197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FBCJ20210197%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Journal%26atitle%3DIdentifying%252BSARS-CoV-2%252Bantiviral%252Bcompounds%252Bby%252Bscreening%252Bfor%252Bsmall%252Bmolecule%252Binhibitors%252Bof%252BNsp5%252Bmain%252Bprotease%26aulast%3DMilligan%26aufirst%3DJennifer%2BC.%26date%3D2021%26date%3D2021%26volume%3D478%26issue%3D13%26spage%3D2499%26epage%3D2515" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthew D  Hall</span>, <span class="hlFld-ContribAuthor ">James M  Anderson</span>, <span class="hlFld-ContribAuthor ">Annaliesa  Anderson</span>, <span class="hlFld-ContribAuthor ">David  Baker</span>, <span class="hlFld-ContribAuthor ">Jay  Bradner</span>, <span class="hlFld-ContribAuthor ">Kyle R  Brimacombe</span>, <span class="hlFld-ContribAuthor ">Elizabeth A  Campbell</span>, <span class="hlFld-ContribAuthor ">Kizzmekia S  Corbett</span>, <span class="hlFld-ContribAuthor ">Kara  Carter</span>, <span class="hlFld-ContribAuthor ">Sara  Cherry</span>, <span class="hlFld-ContribAuthor ">Lillian  Chiang</span>, <span class="hlFld-ContribAuthor ">Tomas  Cihlar</span>, <span class="hlFld-ContribAuthor ">Emmie  de Wit</span>, <span class="hlFld-ContribAuthor ">Mark  Denison</span>, <span class="hlFld-ContribAuthor ">Matthew  Disney</span>, <span class="hlFld-ContribAuthor ">Courtney V  Fletcher</span>, <span class="hlFld-ContribAuthor ">Stephanie L  Ford-Scheimer</span>, <span class="hlFld-ContribAuthor ">Matthias  Götte</span>, <span class="hlFld-ContribAuthor ">Abigail C  Grossman</span>, <span class="hlFld-ContribAuthor ">Frederick G  Hayden</span>, <span class="hlFld-ContribAuthor ">Daria J  Hazuda</span>, <span class="hlFld-ContribAuthor ">Charlotte A  Lanteri</span>, <span class="hlFld-ContribAuthor ">Hilary  Marston</span>, <span class="hlFld-ContribAuthor ">Andrew D  Mesecar</span>, <span class="hlFld-ContribAuthor ">Stephanie  Moore</span>, <span class="hlFld-ContribAuthor ">Jennifer O  Nwankwo</span>, <span class="hlFld-ContribAuthor ">Jules  O’Rear</span>, <span class="hlFld-ContribAuthor ">George  Painter</span>, <span class="hlFld-ContribAuthor ">Kumar  Singh Saikatendu</span>, <span class="hlFld-ContribAuthor ">Celia A  Schiffer</span>, <span class="hlFld-ContribAuthor ">Timothy P  Sheahan</span>, <span class="hlFld-ContribAuthor ">Pei-Yong  Shi</span>, <span class="hlFld-ContribAuthor ">Hugh D  Smyth</span>, <span class="hlFld-ContribAuthor ">Michael J  Sofia</span>, <span class="hlFld-ContribAuthor ">Marla  Weetall</span>, <span class="hlFld-ContribAuthor ">Sandra K  Weller</span>, <span class="hlFld-ContribAuthor ">Richard  Whitley</span>, <span class="hlFld-ContribAuthor ">Anthony S  Fauci</span>, <span class="hlFld-ContribAuthor ">Christopher P  Austin</span>, <span class="hlFld-ContribAuthor ">Francis S  Collins</span>, <span class="hlFld-ContribAuthor ">Anthony J  Conley</span>, <span class="hlFld-ContribAuthor ">Mindy I  Davis</span>. </span><span class="cited-content_cbyCitation_article-title">Report of the National Institutes of Health SARS-CoV-2 Antiviral Therapeutics Summit. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Infectious Diseases</span><span> <strong>2021,</strong> <em>224 </em>
                                    (Supplement_1)
                                     , S1-S21. <a href="https://doi.org/10.1093/infdis/jiab305" title="DOI URL">https://doi.org/10.1093/infdis/jiab305</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/infdis/jiab305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Finfdis%2Fjiab305%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Infectious%2520Diseases%26atitle%3DReport%252Bof%252Bthe%252BNational%252BInstitutes%252Bof%252BHealth%252BSARS-CoV-2%252BAntiviral%252BTherapeutics%252BSummit%26aulast%3DHall%26aufirst%3DMatthew%2BD%26date%3D2021%26date%3D2021%26volume%3D224%26issue%3DSupplement_1%26spage%3DS1%26epage%3DS21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ulises  Martínez‐Ortega</span>, <span class="hlFld-ContribAuthor ">Diego I.  Figueroa‐Figueroa</span>, <span class="hlFld-ContribAuthor ">Francisco  Hernández‐Luis</span>, <span class="hlFld-ContribAuthor ">Rodrigo  Aguayo‐Ortiz</span>. </span><span class="cited-content_cbyCitation_article-title">In Silico Characterization of Masitinib Interaction with SARS‐CoV‐2 Main Protease. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2021,</strong> <em>21 </em><a href="https://doi.org/10.1002/cmdc.202100375" title="DOI URL">https://doi.org/10.1002/cmdc.202100375</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202100375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202100375%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DIn%252BSilico%252BCharacterization%252Bof%252BMasitinib%252BInteraction%252Bwith%252BSARS%2525E2%252580%252590CoV%2525E2%252580%2525902%252BMain%252BProtease%26aulast%3DMart%25C3%25ADnez%25E2%2580%2590Ortega%26aufirst%3DUlises%26date%3D2021%26date%3D2021%26volume%3D21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jian-Bo  Tong</span>, <span class="hlFld-ContribAuthor ">Ding  Luo</span>, <span class="hlFld-ContribAuthor ">Hai-Yin  Xu</span>, <span class="hlFld-ContribAuthor ">Shuai  Bian</span>, <span class="hlFld-ContribAuthor ">Xing  Zhang</span>, <span class="hlFld-ContribAuthor ">Xue-Chun  Xiao</span>, <span class="hlFld-ContribAuthor ">Jie  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">A computational approach for designing novel SARS-CoV-2 M
              pro
              inhibitors: combined QSAR, molecular docking, and molecular dynamics simulation techniques. </span><span class="cited-content_cbyCitation_journal-name">New Journal of Chemistry</span><span> <strong>2021,</strong> <em>45 </em>
                                    (26)
                                     , 11512-11529. <a href="https://doi.org/10.1039/D1NJ02127C" title="DOI URL">https://doi.org/10.1039/D1NJ02127C</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1NJ02127C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1NJ02127C%26sid%3Dliteratum%253Aachs%26jtitle%3DNew%2520Journal%2520of%2520Chemistry%26atitle%3DA%252Bcomputational%252Bapproach%252Bfor%252Bdesigning%252Bnovel%252BSARS-CoV-2%252BM%252Bpro%252Binhibitors%25253A%252Bcombined%252BQSAR%25252C%252Bmolecular%252Bdocking%25252C%252Band%252Bmolecular%252Bdynamics%252Bsimulation%252Btechniques%26aulast%3DTong%26aufirst%3DJian-Bo%26date%3D2021%26date%3D2021%26volume%3D45%26issue%3D26%26spage%3D11512%26epage%3D11529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Salvatore Vincenzo  Giofrè</span>, <span class="hlFld-ContribAuthor ">Edoardo  Napoli</span>, <span class="hlFld-ContribAuthor ">Nunzio  Iraci</span>, <span class="hlFld-ContribAuthor ">Antonio  Speciale</span>, <span class="hlFld-ContribAuthor ">Francesco  Cimino</span>, <span class="hlFld-ContribAuthor ">Claudia  Muscarà</span>, <span class="hlFld-ContribAuthor ">Maria Sofia  Molonia</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Ruberto</span>, <span class="hlFld-ContribAuthor ">Antonella  Saija</span>. </span><span class="cited-content_cbyCitation_article-title">Interaction of selected terpenoids with two SARS-CoV-2 key therapeutic targets: An in silico study through molecular docking and dynamics simulations. </span><span class="cited-content_cbyCitation_journal-name">Computers in Biology and Medicine</span><span> <strong>2021,</strong> <em>134 </em>, 104538. <a href="https://doi.org/10.1016/j.compbiomed.2021.104538" title="DOI URL">https://doi.org/10.1016/j.compbiomed.2021.104538</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.compbiomed.2021.104538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.compbiomed.2021.104538%26sid%3Dliteratum%253Aachs%26jtitle%3DComputers%2520in%2520Biology%2520and%2520Medicine%26atitle%3DInteraction%252Bof%252Bselected%252Bterpenoids%252Bwith%252Btwo%252BSARS-CoV-2%252Bkey%252Btherapeutic%252Btargets%25253A%252BAn%252Bin%252Bsilico%252Bstudy%252Bthrough%252Bmolecular%252Bdocking%252Band%252Bdynamics%252Bsimulations%26aulast%3DGiofr%25C3%25A8%26aufirst%3DSalvatore%2BVincenzo%26date%3D2021%26volume%3D134%26spage%3D104538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Abhik  Paul</span>, <span class="hlFld-ContribAuthor ">Arnab  Sarkar</span>, <span class="hlFld-ContribAuthor ">Sanjukta  Saha</span>, <span class="hlFld-ContribAuthor ">Avik  Maji</span>, <span class="hlFld-ContribAuthor ">Pritha  Janah</span>, <span class="hlFld-ContribAuthor ">Tapan  Kumar Maity</span>. </span><span class="cited-content_cbyCitation_article-title">Synthetic and computational efforts towards the development of peptidomimetics and small-molecule SARS-CoV 3CLpro inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2021,</strong> <em>362 </em>, 116301. <a href="https://doi.org/10.1016/j.bmc.2021.116301" title="DOI URL">https://doi.org/10.1016/j.bmc.2021.116301</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2021.116301&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2021.116301%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DSynthetic%252Band%252Bcomputational%252Befforts%252Btowards%252Bthe%252Bdevelopment%252Bof%252Bpeptidomimetics%252Band%252Bsmall-molecule%252BSARS-CoV%252B3CLpro%252Binhibitors%26aulast%3DPaul%26aufirst%3DAbhik%26date%3D2021%26volume%3D362%26spage%3D116301" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Subramanyam  Vankadara</span>, <span class="hlFld-ContribAuthor ">Yun  Xuan Wong</span>, <span class="hlFld-ContribAuthor ">Boping  Liu</span>, <span class="hlFld-ContribAuthor ">Yi  Yang See</span>, <span class="hlFld-ContribAuthor ">Li  Hong Tan</span>, <span class="hlFld-ContribAuthor ">Qian  Wen Tan</span>, <span class="hlFld-ContribAuthor ">Gang  Wang</span>, <span class="hlFld-ContribAuthor ">Ratna  Karuna</span>, <span class="hlFld-ContribAuthor ">Xue  Guo</span>, <span class="hlFld-ContribAuthor ">Shu  Ting Tan</span>, <span class="hlFld-ContribAuthor ">Jia  Yi Fong</span>, <span class="hlFld-ContribAuthor ">Joma  Joy</span>, <span class="hlFld-ContribAuthor ">C.S.  Brian Chia</span>. </span><span class="cited-content_cbyCitation_article-title">A head-to-head comparison of the inhibitory activities of 15 peptidomimetic SARS-CoV-2 3CLpro inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>382 </em>, 128263. <a href="https://doi.org/10.1016/j.bmcl.2021.128263" title="DOI URL">https://doi.org/10.1016/j.bmcl.2021.128263</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2021.128263&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2021.128263%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DA%252Bhead-to-head%252Bcomparison%252Bof%252Bthe%252Binhibitory%252Bactivities%252Bof%252B15%252Bpeptidomimetic%252BSARS-CoV-2%252B3CLpro%252Binhibitors%26aulast%3DVankadara%26aufirst%3DSubramanyam%26date%3D2021%26volume%3D382%26spage%3D128263" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tonko  Dražić</span>, <span class="hlFld-ContribAuthor ">Nikos  Kühl</span>, <span class="hlFld-ContribAuthor ">Mila M.  Leuthold</span>, <span class="hlFld-ContribAuthor ">Mira A. M.  Behnam</span>, <span class="hlFld-ContribAuthor ">Christian D.  Klein</span>. </span><span class="cited-content_cbyCitation_article-title">Efficiency Improvements and Discovery of New Substrates for a SARS-CoV-2 Main Protease FRET Assay. </span><span class="cited-content_cbyCitation_journal-name">SLAS DISCOVERY: Advancing the Science of Drug Discovery</span><span> <strong>2021,</strong> <em>371 </em>, 247255522110206. <a href="https://doi.org/10.1177/24725552211020681" title="DOI URL">https://doi.org/10.1177/24725552211020681</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/24725552211020681&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F24725552211020681%26sid%3Dliteratum%253Aachs%26jtitle%3DSLAS%2520DISCOVERY%253A%2520Advancing%2520the%2520Science%2520of%2520Drug%2520Discovery%26atitle%3DEfficiency%252BImprovements%252Band%252BDiscovery%252Bof%252BNew%252BSubstrates%252Bfor%252Ba%252BSARS-CoV-2%252BMain%252BProtease%252BFRET%252BAssay%26aulast%3DDra%25C5%25BEi%25C4%2587%26aufirst%3DTonko%26date%3D2021%26date%3D2021%26volume%3D371%26spage%3D247255522110206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">N.R.  Jena</span>. </span><span class="cited-content_cbyCitation_article-title">Drug targets, mechanisms of drug action, and therapeutics against SARS-CoV-2. </span><span class="cited-content_cbyCitation_journal-name">Chemical Physics Impact</span><span> <strong>2021,</strong> <em>2 </em>, 100011. <a href="https://doi.org/10.1016/j.chphi.2021.100011" title="DOI URL">https://doi.org/10.1016/j.chphi.2021.100011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chphi.2021.100011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chphi.2021.100011%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Physics%2520Impact%26atitle%3DDrug%252Btargets%25252C%252Bmechanisms%252Bof%252Bdrug%252Baction%25252C%252Band%252Btherapeutics%252Bagainst%252BSARS-CoV-2%26aulast%3DJena%26aufirst%3DN.R.%26date%3D2021%26volume%3D2%26spage%3D100011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maya G.  Deshmukh</span>, <span class="hlFld-ContribAuthor ">Joseph A.  Ippolito</span>, <span class="hlFld-ContribAuthor ">Chun-Hui  Zhang</span>, <span class="hlFld-ContribAuthor ">Elizabeth A.  Stone</span>, <span class="hlFld-ContribAuthor ">Raquel A.  Reilly</span>, <span class="hlFld-ContribAuthor ">Scott J.  Miller</span>, <span class="hlFld-ContribAuthor ">William L.  Jorgensen</span>, <span class="hlFld-ContribAuthor ">Karen S.  Anderson</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-guided design of a perampanel-derived pharmacophore targeting the SARS-CoV-2 main protease. </span><span class="cited-content_cbyCitation_journal-name">Structure</span><span> <strong>2021,</strong> <em>66 </em><a href="https://doi.org/10.1016/j.str.2021.06.002" title="DOI URL">https://doi.org/10.1016/j.str.2021.06.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.str.2021.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.str.2021.06.002%26sid%3Dliteratum%253Aachs%26jtitle%3DStructure%26atitle%3DStructure-guided%252Bdesign%252Bof%252Ba%252Bperampanel-derived%252Bpharmacophore%252Btargeting%252Bthe%252BSARS-CoV-2%252Bmain%252Bprotease%26aulast%3DDeshmukh%26aufirst%3DMaya%2BG.%26date%3D2021%26volume%3D66" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Koen  Vandyck</span>, <span class="hlFld-ContribAuthor ">Rana  Abdelnabi</span>, <span class="hlFld-ContribAuthor ">Kusum  Gupta</span>, <span class="hlFld-ContribAuthor ">Dirk  Jochmans</span>, <span class="hlFld-ContribAuthor ">Andreas  Jekle</span>, <span class="hlFld-ContribAuthor ">Jerome  Deval</span>, <span class="hlFld-ContribAuthor ">Dinah  Misner</span>, <span class="hlFld-ContribAuthor ">Dorothée  Bardiot</span>, <span class="hlFld-ContribAuthor ">Caroline S.  Foo</span>, <span class="hlFld-ContribAuthor ">Cheng  Liu</span>, <span class="hlFld-ContribAuthor ">Suping  Ren</span>, <span class="hlFld-ContribAuthor ">Leonid  Beigelman</span>, <span class="hlFld-ContribAuthor ">Lawrence M.  Blatt</span>, <span class="hlFld-ContribAuthor ">Sandro  Boland</span>, <span class="hlFld-ContribAuthor ">Laura  Vangeel</span>, <span class="hlFld-ContribAuthor ">Steven  Dejonghe</span>, <span class="hlFld-ContribAuthor ">Patrick  Chaltin</span>, <span class="hlFld-ContribAuthor ">Arnaud  Marchand</span>, <span class="hlFld-ContribAuthor ">Vladimir  Serebryany</span>, <span class="hlFld-ContribAuthor ">Antitsa  Stoycheva</span>, <span class="hlFld-ContribAuthor ">Sushmita  Chanda</span>, <span class="hlFld-ContribAuthor ">Julian A.  Symons</span>, <span class="hlFld-ContribAuthor ">Pierre  Raboisson</span>, <span class="hlFld-ContribAuthor ">Johan  Neyts</span>. </span><span class="cited-content_cbyCitation_article-title">ALG-097111, a potent and selective SARS-CoV-2 3-chymotrypsin-like cysteine protease inhibitor exhibits in vivo efficacy in a Syrian Hamster model. </span><span class="cited-content_cbyCitation_journal-name">Biochemical and Biophysical Research Communications</span><span> <strong>2021,</strong> <em>555 </em>, 134-139. <a href="https://doi.org/10.1016/j.bbrc.2021.03.096" title="DOI URL">https://doi.org/10.1016/j.bbrc.2021.03.096</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbrc.2021.03.096&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbrc.2021.03.096%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520and%2520Biophysical%2520Research%2520Communications%26atitle%3DALG-097111%25252C%252Ba%252Bpotent%252Band%252Bselective%252BSARS-CoV-2%252B3-chymotrypsin-like%252Bcysteine%252Bprotease%252Binhibitor%252Bexhibits%252Bin%2525C2%2525A0vivo%252Befficacy%252Bin%252Ba%252BSyrian%252BHamster%252Bmodel%26aulast%3DVandyck%26aufirst%3DKoen%26date%3D2021%26volume%3D555%26spage%3D134%26epage%3D139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alexander I.  Denesyuk</span>, <span class="hlFld-ContribAuthor ">Sergei E.  Permyakov</span>, <span class="hlFld-ContribAuthor ">Mark S.  Johnson</span>, <span class="hlFld-ContribAuthor ">Eugene A.  Permyakov</span>, <span class="hlFld-ContribAuthor ">Vladimir N.  Uversky</span>, <span class="hlFld-ContribAuthor ">Konstantin  Denessiouk</span>. </span><span class="cited-content_cbyCitation_article-title">Structural leitmotif and functional variations of the structural catalytic core in (chymo)trypsin-like serine/cysteine fold proteinases. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Biological Macromolecules</span><span> <strong>2021,</strong> <em>179 </em>, 601-609. <a href="https://doi.org/10.1016/j.ijbiomac.2021.03.042" title="DOI URL">https://doi.org/10.1016/j.ijbiomac.2021.03.042</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ijbiomac.2021.03.042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ijbiomac.2021.03.042%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Biological%2520Macromolecules%26atitle%3DStructural%252Bleitmotif%252Band%252Bfunctional%252Bvariations%252Bof%252Bthe%252Bstructural%252Bcatalytic%252Bcore%252Bin%252B%252528chymo%252529trypsin-like%252Bserine%25252Fcysteine%252Bfold%252Bproteinases%26aulast%3DDenesyuk%26aufirst%3DAlexander%2BI.%26date%3D2021%26volume%3D179%26spage%3D601%26epage%3D609" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Varada  Anirudhan</span>, <span class="hlFld-ContribAuthor ">Hyun  Lee</span>, <span class="hlFld-ContribAuthor ">Han  Cheng</span>, <span class="hlFld-ContribAuthor ">Laura  Cooper</span>, <span class="hlFld-ContribAuthor ">Lijun  Rong</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting SARS‐CoV‐2 viral proteases as a therapeutic strategy to treat COVID‐19. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medical Virology</span><span> <strong>2021,</strong> <em>93 </em>
                                    (5)
                                     , 2722-2734. <a href="https://doi.org/10.1002/jmv.26814" title="DOI URL">https://doi.org/10.1002/jmv.26814</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jmv.26814&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjmv.26814%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medical%2520Virology%26atitle%3DTargeting%252BSARS%2525E2%252580%252590CoV%2525E2%252580%2525902%252Bviral%252Bproteases%252Bas%252Ba%252Btherapeutic%252Bstrategy%252Bto%252Btreat%252BCOVID%2525E2%252580%25259019%26aulast%3DAnirudhan%26aufirst%3DVarada%26date%3D2021%26date%3D2021%26volume%3D93%26issue%3D5%26spage%3D2722%26epage%3D2734" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Julian  Breidenbach</span>, <span class="hlFld-ContribAuthor ">Carina  Lemke</span>, <span class="hlFld-ContribAuthor ">Thanigaimalai  Pillaiyar</span>, <span class="hlFld-ContribAuthor ">Laura  Schäkel</span>, <span class="hlFld-ContribAuthor ">Ghazl  Al Hamwi</span>, <span class="hlFld-ContribAuthor ">Miriam  Diett</span>, <span class="hlFld-ContribAuthor ">Robin  Gedschold</span>, <span class="hlFld-ContribAuthor ">Nina  Geiger</span>, <span class="hlFld-ContribAuthor ">Vittoria  Lopez</span>, <span class="hlFld-ContribAuthor ">Salahuddin  Mirza</span>, <span class="hlFld-ContribAuthor ">Vigneshwaran  Namasivayam</span>, <span class="hlFld-ContribAuthor ">Anke C.  Schiedel</span>, <span class="hlFld-ContribAuthor ">Katharina  Sylvester</span>, <span class="hlFld-ContribAuthor ">Dominik  Thimm</span>, <span class="hlFld-ContribAuthor ">Christin  Vielmuth</span>, <span class="hlFld-ContribAuthor ">Lan  Phuong Vu</span>, <span class="hlFld-ContribAuthor ">Maria  Zyulina</span>, <span class="hlFld-ContribAuthor ">Jochen  Bodem</span>, <span class="hlFld-ContribAuthor ">Michael  Gütschow</span>, <span class="hlFld-ContribAuthor ">Christa E.  Müller</span>. </span><span class="cited-content_cbyCitation_article-title">Die Hauptprotease von SARS‐CoV‐2 als Zielstruktur: Von der Etablierung eines Hochdurchsatz‐Screenings zum Design maßgeschneiderter Inhibitoren. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2021,</strong> <em>133 </em>
                                    (18)
                                     , 10515-10521. <a href="https://doi.org/10.1002/ange.202016961" title="DOI URL">https://doi.org/10.1002/ange.202016961</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.202016961&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.202016961%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DDie%252BHauptprotease%252Bvon%252BSARS%2525E2%252580%252590CoV%2525E2%252580%2525902%252Bals%252BZielstruktur%25253A%252BVon%252Bder%252BEtablierung%252Beines%252BHochdurchsatz%2525E2%252580%252590Screenings%252Bzum%252BDesign%252Bma%2525C3%25259Fgeschneiderter%252BInhibitoren%26aulast%3DBreidenbach%26aufirst%3DJulian%26date%3D2021%26date%3D2021%26volume%3D133%26issue%3D18%26spage%3D10515%26epage%3D10521" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Julian  Breidenbach</span>, <span class="hlFld-ContribAuthor ">Carina  Lemke</span>, <span class="hlFld-ContribAuthor ">Thanigaimalai  Pillaiyar</span>, <span class="hlFld-ContribAuthor ">Laura  Schäkel</span>, <span class="hlFld-ContribAuthor ">Ghazl  Al Hamwi</span>, <span class="hlFld-ContribAuthor ">Miriam  Diett</span>, <span class="hlFld-ContribAuthor ">Robin  Gedschold</span>, <span class="hlFld-ContribAuthor ">Nina  Geiger</span>, <span class="hlFld-ContribAuthor ">Vittoria  Lopez</span>, <span class="hlFld-ContribAuthor ">Salahuddin  Mirza</span>, <span class="hlFld-ContribAuthor ">Vigneshwaran  Namasivayam</span>, <span class="hlFld-ContribAuthor ">Anke C.  Schiedel</span>, <span class="hlFld-ContribAuthor ">Katharina  Sylvester</span>, <span class="hlFld-ContribAuthor ">Dominik  Thimm</span>, <span class="hlFld-ContribAuthor ">Christin  Vielmuth</span>, <span class="hlFld-ContribAuthor ">Lan  Phuong Vu</span>, <span class="hlFld-ContribAuthor ">Maria  Zyulina</span>, <span class="hlFld-ContribAuthor ">Jochen  Bodem</span>, <span class="hlFld-ContribAuthor ">Michael  Gütschow</span>, <span class="hlFld-ContribAuthor ">Christa E.  Müller</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting the Main Protease of SARS‐CoV‐2: From the Establishment of High Throughput Screening to the Design of Tailored Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2021,</strong> <em>60 </em>
                                    (18)
                                     , 10423-10429. <a href="https://doi.org/10.1002/anie.202016961" title="DOI URL">https://doi.org/10.1002/anie.202016961</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.202016961&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.202016961%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DTargeting%252Bthe%252BMain%252BProtease%252Bof%252BSARS%2525E2%252580%252590CoV%2525E2%252580%2525902%25253A%252BFrom%252Bthe%252BEstablishment%252Bof%252BHigh%252BThroughput%252BScreening%252Bto%252Bthe%252BDesign%252Bof%252BTailored%252BInhibitors%26aulast%3DBreidenbach%26aufirst%3DJulian%26date%3D2021%26date%3D2021%26volume%3D60%26issue%3D18%26spage%3D10423%26epage%3D10429" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maren  de Vries</span>, <span class="hlFld-ContribAuthor ">Adil S.  Mohamed</span>, <span class="hlFld-ContribAuthor ">Rachel A.  Prescott</span>, <span class="hlFld-ContribAuthor ">Ana M.  Valero-Jimenez</span>, <span class="hlFld-ContribAuthor ">Ludovic  Desvignes</span>, <span class="hlFld-ContribAuthor ">Rebecca  O’Connor</span>, <span class="hlFld-ContribAuthor ">Claire  Steppan</span>, <span class="hlFld-ContribAuthor ">Joseph C.  Devlin</span>, <span class="hlFld-ContribAuthor ">Ellie  Ivanova</span>, <span class="hlFld-ContribAuthor ">Alberto  Herrera</span>, <span class="hlFld-ContribAuthor ">Austin  Schinlever</span>, <span class="hlFld-ContribAuthor ">Paige  Loose</span>, <span class="hlFld-ContribAuthor ">Kelly  Ruggles</span>, <span class="hlFld-ContribAuthor ">Sergei B.  Koralov</span>, <span class="hlFld-ContribAuthor ">Annaliesa S.  Anderson</span>, <span class="hlFld-ContribAuthor ">Joseph  Binder</span>, <span class="hlFld-ContribAuthor ">Meike  Dittmann</span>, . </span><span class="cited-content_cbyCitation_article-title">A Comparative Analysis of SARS-CoV-2 Antivirals Characterizes 3CL
              pro
              Inhibitor PF-00835231 as a Potential New Treatment for COVID-19. </span><span class="cited-content_cbyCitation_journal-name">Journal of Virology</span><span> <strong>2021,</strong> <em>95 </em>
                                    (10)
                                     <a href="https://doi.org/10.1128/JVI.01819-20" title="DOI URL">https://doi.org/10.1128/JVI.01819-20</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1128/JVI.01819-20&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1128%2FJVI.01819-20%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Virology%26atitle%3DA%252BComparative%252BAnalysis%252Bof%252BSARS-CoV-2%252BAntivirals%252BCharacterizes%252B3CL%252Bpro%252BInhibitor%252BPF-00835231%252Bas%252Ba%252BPotential%252BNew%252BTreatment%252Bfor%252BCOVID-19%26aulast%3Dde%2BVries%26aufirst%3DMaren%26date%3D2021%26volume%3D95%26issue%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elie  Dolgin</span>. </span><span class="cited-content_cbyCitation_article-title">The race for antiviral drugs to beat COVID — and the next pandemic. </span><span class="cited-content_cbyCitation_journal-name">Nature</span><span> <strong>2021,</strong> <em>592 </em>
                                    (7854)
                                     , 340-343. <a href="https://doi.org/10.1038/d41586-021-00958-4" title="DOI URL">https://doi.org/10.1038/d41586-021-00958-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/d41586-021-00958-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fd41586-021-00958-4%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%26atitle%3DThe%252Brace%252Bfor%252Bantiviral%252Bdrugs%252Bto%252Bbeat%252BCOVID%252B%2525E2%252580%252594%252Band%252Bthe%252Bnext%252Bpandemic%26aulast%3DDolgin%26aufirst%3DElie%26date%3D2021%26date%3D2021%26volume%3D592%26issue%3D7854%26spage%3D340%26epage%3D343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrea  Citarella</span>, <span class="hlFld-ContribAuthor ">Angela  Scala</span>, <span class="hlFld-ContribAuthor ">Anna  Piperno</span>, <span class="hlFld-ContribAuthor ">Nicola  Micale</span>. </span><span class="cited-content_cbyCitation_article-title">SARS-CoV-2 Mpro: A Potential Target for Peptidomimetics and Small-Molecule Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Biomolecules</span><span> <strong>2021,</strong> <em>11 </em>
                                    (4)
                                     , 607. <a href="https://doi.org/10.3390/biom11040607" title="DOI URL">https://doi.org/10.3390/biom11040607</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/biom11040607&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fbiom11040607%26sid%3Dliteratum%253Aachs%26jtitle%3DBiomolecules%26atitle%3DSARS-CoV-2%252BMpro%25253A%252BA%252BPotential%252BTarget%252Bfor%252BPeptidomimetics%252Band%252BSmall-Molecule%252BInhibitors%26aulast%3DCitarella%26aufirst%3DAndrea%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D4%26spage%3D607" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elena  Arutyunova</span>, <span class="hlFld-ContribAuthor ">Muhammad  Bashir Khan</span>, <span class="hlFld-ContribAuthor ">Conrad  Fischer</span>, <span class="hlFld-ContribAuthor ">Jimmy  Lu</span>, <span class="hlFld-ContribAuthor ">Tess  Lamer</span>, <span class="hlFld-ContribAuthor ">Wayne  Vuong</span>, <span class="hlFld-ContribAuthor ">Marco J.  van Belkum</span>, <span class="hlFld-ContribAuthor ">Ryan T.  McKay</span>, <span class="hlFld-ContribAuthor ">D.  Lorne Tyrrell</span>, <span class="hlFld-ContribAuthor ">John C.  Vederas</span>, <span class="hlFld-ContribAuthor ">Howard S.  Young</span>, <span class="hlFld-ContribAuthor ">M.  Joanne Lemieux</span>. </span><span class="cited-content_cbyCitation_article-title">N-Terminal finger stabilizes the S1 pocket for the reversible feline drug GC376 in the SARS-CoV-2 Mpro dimer. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Biology</span><span> <strong>2021,</strong> <em> </em>, 167003. <a href="https://doi.org/10.1016/j.jmb.2021.167003" title="DOI URL">https://doi.org/10.1016/j.jmb.2021.167003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmb.2021.167003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmb.2021.167003%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Biology%26atitle%3DN-Terminal%252Bfinger%252Bstabilizes%252Bthe%252BS1%252Bpocket%252Bfor%252Bthe%252Breversible%252Bfeline%252Bdrug%252BGC376%252Bin%252Bthe%252BSARS-CoV-2%252BMpro%252Bdimer%26aulast%3DArutyunova%26aufirst%3DElena%26date%3D2021%26spage%3D167003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jingxin  Qiao</span>, <span class="hlFld-ContribAuthor ">Yue-Shan  Li</span>, <span class="hlFld-ContribAuthor ">Rui  Zeng</span>, <span class="hlFld-ContribAuthor ">Feng-Liang  Liu</span>, <span class="hlFld-ContribAuthor ">Rong-Hua  Luo</span>, <span class="hlFld-ContribAuthor ">Chong  Huang</span>, <span class="hlFld-ContribAuthor ">Yi-Fei  Wang</span>, <span class="hlFld-ContribAuthor ">Jie  Zhang</span>, <span class="hlFld-ContribAuthor ">Baoxue  Quan</span>, <span class="hlFld-ContribAuthor ">Chenjian  Shen</span>, <span class="hlFld-ContribAuthor ">Xin  Mao</span>, <span class="hlFld-ContribAuthor ">Xinlei  Liu</span>, <span class="hlFld-ContribAuthor ">Weining  Sun</span>, <span class="hlFld-ContribAuthor ">Wei  Yang</span>, <span class="hlFld-ContribAuthor ">Xincheng  Ni</span>, <span class="hlFld-ContribAuthor ">Kai  Wang</span>, <span class="hlFld-ContribAuthor ">Ling  Xu</span>, <span class="hlFld-ContribAuthor ">Zi-Lei  Duan</span>, <span class="hlFld-ContribAuthor ">Qing-Cui  Zou</span>, <span class="hlFld-ContribAuthor ">Hai-Lin  Zhang</span>, <span class="hlFld-ContribAuthor ">Wang  Qu</span>, <span class="hlFld-ContribAuthor ">Yang-Hao-Peng  Long</span>, <span class="hlFld-ContribAuthor ">Ming-Hua  Li</span>, <span class="hlFld-ContribAuthor ">Rui-Cheng  Yang</span>, <span class="hlFld-ContribAuthor ">Xiaolong  Liu</span>, <span class="hlFld-ContribAuthor ">Jing  You</span>, <span class="hlFld-ContribAuthor ">Yangli  Zhou</span>, <span class="hlFld-ContribAuthor ">Rui  Yao</span>, <span class="hlFld-ContribAuthor ">Wen-Pei  Li</span>, <span class="hlFld-ContribAuthor ">Jing-Ming  Liu</span>, <span class="hlFld-ContribAuthor ">Pei  Chen</span>, <span class="hlFld-ContribAuthor ">Yang  Liu</span>, <span class="hlFld-ContribAuthor ">Gui-Feng  Lin</span>, <span class="hlFld-ContribAuthor ">Xin  Yang</span>, <span class="hlFld-ContribAuthor ">Jun  Zou</span>, <span class="hlFld-ContribAuthor ">Linli  Li</span>, <span class="hlFld-ContribAuthor ">Yiguo  Hu</span>, <span class="hlFld-ContribAuthor ">Guang-Wen  Lu</span>, <span class="hlFld-ContribAuthor ">Wei-Min  Li</span>, <span class="hlFld-ContribAuthor ">Yu-Quan  Wei</span>, <span class="hlFld-ContribAuthor ">Yong-Tang  Zheng</span>, <span class="hlFld-ContribAuthor ">Jian  Lei</span>, <span class="hlFld-ContribAuthor ">Shengyong  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">SARS-CoV-2 M
              pro
              inhibitors with antiviral activity in a transgenic mouse model. </span><span class="cited-content_cbyCitation_journal-name">Science</span><span> <strong>2021,</strong> <em>371 </em>
                                    (6536)
                                     , 1374-1378. <a href="https://doi.org/10.1126/science.abf1611" title="DOI URL">https://doi.org/10.1126/science.abf1611</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1126/science.abf1611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1126%2Fscience.abf1611%26sid%3Dliteratum%253Aachs%26jtitle%3DScience%26atitle%3DSARS-CoV-2%252BM%252Bpro%252Binhibitors%252Bwith%252Bantiviral%252Bactivity%252Bin%252Ba%252Btransgenic%252Bmouse%252Bmodel%26aulast%3DQiao%26aufirst%3DJingxin%26date%3D2021%26date%3D2021%26volume%3D371%26issue%3D6536%26spage%3D1374%26epage%3D1378" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ernest  Awoonor-Williams</span>, <span class="hlFld-ContribAuthor ">Abd Al-Aziz A.  Abu-Saleh</span>. </span><span class="cited-content_cbyCitation_article-title">Covalent and non-covalent binding free energy calculations for peptidomimetic inhibitors of SARS-CoV-2 main protease. </span><span class="cited-content_cbyCitation_journal-name">Physical Chemistry Chemical Physics</span><span> <strong>2021,</strong> <em>23 </em>
                                    (11)
                                     , 6746-6757. <a href="https://doi.org/10.1039/D1CP00266J" title="DOI URL">https://doi.org/10.1039/D1CP00266J</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1CP00266J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1CP00266J%26sid%3Dliteratum%253Aachs%26jtitle%3DPhysical%2520Chemistry%2520Chemical%2520Physics%26atitle%3DCovalent%252Band%252Bnon-covalent%252Bbinding%252Bfree%252Benergy%252Bcalculations%252Bfor%252Bpeptidomimetic%252Binhibitors%252Bof%252BSARS-CoV-2%252Bmain%252Bprotease%26aulast%3DAwoonor-Williams%26aufirst%3DErnest%26date%3D2021%26date%3D2021%26volume%3D23%26issue%3D11%26spage%3D6746%26epage%3D6757" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Carlos A.  Ramos-Guzmán</span>, <span class="hlFld-ContribAuthor ">J. Javier  Ruiz-Pernía</span>, <span class="hlFld-ContribAuthor ">Iñaki  Tuñón</span>. </span><span class="cited-content_cbyCitation_article-title">A microscopic description of SARS-CoV-2 main protease inhibition with Michael acceptors. Strategies for improving inhibitor design. </span><span class="cited-content_cbyCitation_journal-name">Chemical Science</span><span> <strong>2021,</strong> <em>12 </em>
                                    (10)
                                     , 3489-3496. <a href="https://doi.org/10.1039/D0SC04978F" title="DOI URL">https://doi.org/10.1039/D0SC04978F</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0SC04978F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0SC04978F%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Science%26atitle%3DA%252Bmicroscopic%252Bdescription%252Bof%252BSARS-CoV-2%252Bmain%252Bprotease%252Binhibition%252Bwith%252BMichael%252Bacceptors.%252BStrategies%252Bfor%252Bimproving%252Binhibitor%252Bdesign%26aulast%3DRamos-Guzm%25C3%25A1n%26aufirst%3DCarlos%2BA.%26date%3D2021%26date%3D2021%26volume%3D12%26issue%3D10%26spage%3D3489%26epage%3D3496" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Omkar  Indari</span>, <span class="hlFld-ContribAuthor ">Shweta  Jakhmola</span>, <span class="hlFld-ContribAuthor ">Elangovan  Manivannan</span>, <span class="hlFld-ContribAuthor ">Hem Chandra  Jha</span>. </span><span class="cited-content_cbyCitation_article-title">An Update on Antiviral Therapy Against SARS-CoV-2: How Far Have We Come?. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pharmacology</span><span> <strong>2021,</strong> <em>12 </em><a href="https://doi.org/10.3389/fphar.2021.632677" title="DOI URL">https://doi.org/10.3389/fphar.2021.632677</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphar.2021.632677&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphar.2021.632677%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pharmacology%26atitle%3DAn%252BUpdate%252Bon%252BAntiviral%252BTherapy%252BAgainst%252BSARS-CoV-2%25253A%252BHow%252BFar%252BHave%252BWe%252BCome%25253F%26aulast%3DIndari%26aufirst%3DOmkar%26date%3D2021%26date%3D2021%26volume%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Cheng  Liu</span>, <span class="hlFld-ContribAuthor ">Sandro  Boland</span>, <span class="hlFld-ContribAuthor ">Michael D.  Scholle</span>, <span class="hlFld-ContribAuthor ">Dorothee  Bardiot</span>, <span class="hlFld-ContribAuthor ">Arnaud  Marchand</span>, <span class="hlFld-ContribAuthor ">Patrick  Chaltin</span>, <span class="hlFld-ContribAuthor ">Lawrence M.  Blatt</span>, <span class="hlFld-ContribAuthor ">Leonid  Beigelman</span>, <span class="hlFld-ContribAuthor ">Julian A.  Symons</span>, <span class="hlFld-ContribAuthor ">Pierre  Raboisson</span>, <span class="hlFld-ContribAuthor ">Zachary A.  Gurard-Levin</span>, <span class="hlFld-ContribAuthor ">Koen  Vandyck</span>, <span class="hlFld-ContribAuthor ">Jerome  Deval</span>. </span><span class="cited-content_cbyCitation_article-title">Dual inhibition of SARS-CoV-2 and human rhinovirus with protease inhibitors in clinical development. </span><span class="cited-content_cbyCitation_journal-name">Antiviral Research</span><span> <strong>2021,</strong> <em>187 </em>, 105020. <a href="https://doi.org/10.1016/j.antiviral.2021.105020" title="DOI URL">https://doi.org/10.1016/j.antiviral.2021.105020</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.antiviral.2021.105020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.antiviral.2021.105020%26sid%3Dliteratum%253Aachs%26jtitle%3DAntiviral%2520Research%26atitle%3DDual%252Binhibition%252Bof%252BSARS-CoV-2%252Band%252Bhuman%252Brhinovirus%252Bwith%252Bprotease%252Binhibitors%252Bin%252Bclinical%252Bdevelopment%26aulast%3DLiu%26aufirst%3DCheng%26date%3D2021%26volume%3D187%26spage%3D105020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mariusz  Jaskolski</span>, <span class="hlFld-ContribAuthor ">Zbigniew  Dauter</span>, <span class="hlFld-ContribAuthor ">Ivan G.  Shabalin</span>, <span class="hlFld-ContribAuthor ">Miroslaw  Gilski</span>, <span class="hlFld-ContribAuthor ">Dariusz  Brzezinski</span>, <span class="hlFld-ContribAuthor ">Marcin  Kowiel</span>, <span class="hlFld-ContribAuthor ">Bernhard  Rupp</span>, <span class="hlFld-ContribAuthor ">Alexander  Wlodawer</span>. </span><span class="cited-content_cbyCitation_article-title">Crystallographic models of SARS-CoV-2 3CL
              pro
              : in-depth assessment of structure quality and validation. </span><span class="cited-content_cbyCitation_journal-name">IUCrJ</span><span> <strong>2021,</strong> <em>8 </em>
                                    (2)
                                     , 238-256. <a href="https://doi.org/10.1107/S2052252521001159" title="DOI URL">https://doi.org/10.1107/S2052252521001159</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1107/S2052252521001159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1107%2FS2052252521001159%26sid%3Dliteratum%253Aachs%26jtitle%3DIUCrJ%26atitle%3DCrystallographic%252Bmodels%252Bof%252BSARS-CoV-2%252B3CL%252Bpro%252B%25253A%252Bin-depth%252Bassessment%252Bof%252Bstructure%252Bquality%252Band%252Bvalidation%26aulast%3DJaskolski%26aufirst%3DMariusz%26date%3D2021%26date%3D2021%26volume%3D8%26issue%3D2%26spage%3D238%26epage%3D256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mohammad Hassan  Baig</span>, <span class="hlFld-ContribAuthor ">Tanuj  Sharma</span>, <span class="hlFld-ContribAuthor ">Irfan  Ahmad</span>, <span class="hlFld-ContribAuthor ">Mohammed  Abohashrh</span>, <span class="hlFld-ContribAuthor ">Mohammad Mahtab  Alam</span>, <span class="hlFld-ContribAuthor ">Jae-June  Dong</span>. </span><span class="cited-content_cbyCitation_article-title">Is PF-00835231 a Pan-SARS-CoV-2 Mpro Inhibitor? A Comparative Study. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (6)
                                     , 1678. <a href="https://doi.org/10.3390/molecules26061678" title="DOI URL">https://doi.org/10.3390/molecules26061678</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26061678&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26061678%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DIs%252BPF-00835231%252Ba%252BPan-SARS-CoV-2%252BMpro%252BInhibitor%25253F%252BA%252BComparative%252BStudy%26aulast%3DBaig%26aufirst%3DMohammad%2BHassan%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D6%26spage%3D1678" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Haixia  Su</span>, <span class="hlFld-ContribAuthor ">Yechun  Xu</span>, <span class="hlFld-ContribAuthor ">Hualiang  Jiang</span>. </span><span class="cited-content_cbyCitation_article-title">Drug discovery and development targeting the life cycle of SARS-CoV-2. </span><span class="cited-content_cbyCitation_journal-name">Fundamental Research</span><span> <strong>2021,</strong> <em>1 </em>
                                    (2)
                                     , 151-165. <a href="https://doi.org/10.1016/j.fmre.2021.01.013" title="DOI URL">https://doi.org/10.1016/j.fmre.2021.01.013</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.fmre.2021.01.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.fmre.2021.01.013%26sid%3Dliteratum%253Aachs%26jtitle%3DFundamental%2520Research%26atitle%3DDrug%252Bdiscovery%252Band%252Bdevelopment%252Btargeting%252Bthe%252Blife%252Bcycle%252Bof%252BSARS-CoV-2%26aulast%3DSu%26aufirst%3DHaixia%26date%3D2021%26volume%3D1%26issue%3D2%26spage%3D151%26epage%3D165" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">. </span><span class="cited-content_cbyCitation_article-title">Synthesis of PF-00835231. </span><span class="cited-content_cbyCitation_journal-name">Synfacts</span><span> <strong>2021,</strong>,, 0131. <a href="https://doi.org/10.1055/s-0040-1719322" title="DOI URL">https://doi.org/10.1055/s-0040-1719322</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1055/s-0040-1719322&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1055%2Fs-0040-1719322%26sid%3Dliteratum%253Aachs%26jtitle%3DSynfacts%26atitle%3DSynthesis%252Bof%252BPF-00835231%26date%3D2021%26date%3D2021%26volume%3D17%26issue%3D02%26spage%3D0131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gordon J.  Lockbaum</span>, <span class="hlFld-ContribAuthor ">Archie C.  Reyes</span>, <span class="hlFld-ContribAuthor ">Jeong Min  Lee</span>, <span class="hlFld-ContribAuthor ">Ronak  Tilvawala</span>, <span class="hlFld-ContribAuthor ">Ellen A.  Nalivaika</span>, <span class="hlFld-ContribAuthor ">Akbar  Ali</span>, <span class="hlFld-ContribAuthor ">Nese  Kurt Yilmaz</span>, <span class="hlFld-ContribAuthor ">Paul R.  Thompson</span>, <span class="hlFld-ContribAuthor ">Celia A.  Schiffer</span>. </span><span class="cited-content_cbyCitation_article-title">Crystal Structure of SARS-CoV-2 Main Protease in Complex with the Non-Covalent Inhibitor ML188. </span><span class="cited-content_cbyCitation_journal-name">Viruses</span><span> <strong>2021,</strong> <em>13 </em>
                                    (2)
                                     , 174. <a href="https://doi.org/10.3390/v13020174" title="DOI URL">https://doi.org/10.3390/v13020174</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/v13020174&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fv13020174%26sid%3Dliteratum%253Aachs%26jtitle%3DViruses%26atitle%3DCrystal%252BStructure%252Bof%252BSARS-CoV-2%252BMain%252BProtease%252Bin%252BComplex%252Bwith%252Bthe%252BNon-Covalent%252BInhibitor%252BML188%26aulast%3DLockbaum%26aufirst%3DGordon%2BJ.%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D2%26spage%3D174" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Erik  De Clercq</span>. </span><span class="cited-content_cbyCitation_article-title">Perspectives for antivirals to limit SARS-CoV-2 infection (COVID-19). </span><span class="cited-content_cbyCitation_journal-name">Microbiology Australia</span><span> <strong>2021,</strong> <em>42 </em>
                                    (1)
                                     , 47. <a href="https://doi.org/10.1071/MA21013" title="DOI URL">https://doi.org/10.1071/MA21013</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1071/MA21013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1071%2FMA21013%26sid%3Dliteratum%253Aachs%26jtitle%3DMicrobiology%2520Australia%26atitle%3DPerspectives%252Bfor%252Bantivirals%252Bto%252Blimit%252BSARS-CoV-2%252Binfection%252B%252528COVID-19%252529%26aulast%3DDe%2BClercq%26aufirst%3DErik%26date%3D2021%26volume%3D42%26issue%3D1%26spage%3D47" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bruno Silva  Andrade</span>, <span class="hlFld-ContribAuthor ">Fernanda de Souza  Rangel</span>, <span class="hlFld-ContribAuthor ">Naiane Oliveira  Santos</span>, <span class="hlFld-ContribAuthor ">Andria dos Santos  Freitas</span>, <span class="hlFld-ContribAuthor ">Wagner Rodrigues de Assis  Soares</span>, <span class="hlFld-ContribAuthor ">Sérgio  Siqueira</span>, <span class="hlFld-ContribAuthor ">Debmalya  Barh</span>, <span class="hlFld-ContribAuthor ">Aristóteles  Góes-Neto</span>, <span class="hlFld-ContribAuthor ">Alexander  Birbrair</span>, <span class="hlFld-ContribAuthor ">Vasco Ariston de Carvalho  Azevedo</span>. </span><span class="cited-content_cbyCitation_article-title">Repurposing Approved Drugs for Guiding COVID-19 Prophylaxis: A Systematic Review. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pharmacology</span><span> <strong>2020,</strong> <em>11 </em><a href="https://doi.org/10.3389/fphar.2020.590598" title="DOI URL">https://doi.org/10.3389/fphar.2020.590598</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphar.2020.590598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphar.2020.590598%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pharmacology%26atitle%3DRepurposing%252BApproved%252BDrugs%252Bfor%252BGuiding%252BCOVID-19%252BProphylaxis%25253A%252BA%252BSystematic%252BReview%26aulast%3DAndrade%26aufirst%3DBruno%2BSilva%26date%3D2020%26date%3D2020%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/medium/jm0c01063_0020.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01063&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/medium/jm0c01063_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0002.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Subsite nomenclature for proteolytic enzymes is shown. Amino acid residues to the left of the polypeptide scissile amide bond are numbered sequentially, beginning with P<sub>1</sub> and increasing toward the N-terminus. Amino acid residues to the right of the scissile bond are numbered sequentially, beginning with P<sub>1</sub>′ and increasing toward the C-terminus. Complimentary regions of the protease active site employ the corresponding S numbering.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01063&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/medium/jm0c01063_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0003.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. HRV clinical candidate <b>rupintrivir</b> and SARS CoV-1 3CL<sup>pro</sup> inhibitor <b>2</b> employing Michael acceptor-based warheads along with optimized SARS CoV-1 3CL<sup>pro</sup> inhibitors <b>3</b> and <b>4</b> containing a carbonyl designed for attack by the catalytic cysteine residue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01063&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/medium/jm0c01063_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0004.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cocrystal structure of the covalent adduct of <b>2</b> bound to SARS CoV-1 3CL<sup>pro</sup> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XHO">6XHO</a>). The Connolly surface for the inhibitor binding pocket is shown in gray. The bonds are represented as the dashed lines, with the bond length measured between heavy atoms. The views are centered on warhead with active-site interactions and of complete inhibitor–protein interactions depicted.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01063&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/medium/jm0c01063_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0007.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Preparation of P<sub>1</sub> Synthetic Intermediates</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01063&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(i) 3 M NaOH, MeOH (95%), (ii) isobutylchloroformate (IBCF), triethylamine (TEA), tetrahydrofuran (THF), (iii) CH<sub>2</sub>N<sub>2</sub>/ether (92%), (iv) 4 M HCl/dioxane (83%), (v) 1.0 equiv 48% HBr (83%), (vi) HNCH<sub>3</sub>(OCH<sub>3</sub>), EDC, HOBt, NMM, dichloromethane (DCM) (79%), (vii) Mg, HgCl<sub>2</sub>, BOM-Cl, THF, −78°C (50%), and (viii) lithium diisopropylamide (LDA), ClCH<sub>2</sub>I, THF (54%).</p></p></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/medium/jm0c01063_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0008.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Preparation of Substituted Acyloxymethylketones and Aminomethylketones</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01063&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(i) Boc-Leu-OH, HATU, NMM, <i>N</i>,<i>N</i>-dimethylformamide (DMF), 0 °C (64%), (ii) 4 M HCl/dioxane, (iii) HATU, 4-MeO-2-indolecarboxylic acid, NMM, DMF, 0 °C (61%), and (iv) CsF, DMF, carboxylic acid, 60 °C (43–91%).</p></p></figure><figure data-id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/medium/jm0c01063_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0009.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Preparation of Hydroxymethylketone Derivatives</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01063&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>(i) 4 M HCl/dioxane, (ii) Boc-Leu-OH, HATU, NMM, DMF, 0 °C, (iii) HATU, 4-MeO-2-indolecarboxylic acid, NMM, 0 °C, (iv) Pd/C, H<sub>2</sub>, EtOH (77%), (v) CsF, DMF, benzoylformic acid, 60 °C, and then MeOH, cat. K<sub>2</sub>CO<sub>3</sub> (53%), and (vi) CH<sub>3</sub>I, Ag<sub>2</sub>O, DCE (9%).</p></p></figure><figure data-id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/medium/jm0c01063_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0005.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Cocrystal structure of the covalent adduct of <b>28</b> bound to SARS CoV-1 3CL<sup>pro</sup> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XHN">6XHN</a>). The Connolly surface for the inhibitor binding pocket is shown in gray. The bonds are represented as dashed lines, with the bond length between heavy atoms depicted.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01063&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/medium/jm0c01063_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0006.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Cocrystal structure of the covalent adduct of <b>4</b> bound to SARS CoV-2 3CL<sup>pro</sup> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XHM">6XHM</a>). The Connolly surface for the inhibitor binding pocket is shown in gray. The bonds are represented as dashed lines, with the bond length between heavy atoms depicted. The schematic rendering of the active site with dashed lines represented as hydrogen bonds with key residues and curved lines to show S<sub>1</sub> and S<sub>2</sub> binding pockets is depicted.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-21/acs.jmedchem.0c01063/20201113/images/large/jm0c01063_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01063&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i101">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82565" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82565" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 58 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Z.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, Z.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, M.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, F.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.-Z.</span></span> <span> </span><span class="NLM_article-title">A new coronavirus associated with human respiratory disease in China</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>579</i></span>,  <span class="NLM_fpage">265</span>– <span class="NLM_lpage">269</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2008-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1038%2Fs41586-020-2008-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=32015508" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFKlsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=579&publication_year=2020&pages=265-269&author=F.+Wuauthor=S.+Zhaoauthor=B.+Yuauthor=Y.-M.+Chenauthor=W.+Wangauthor=Z.-G.+Songauthor=Y.+Huauthor=Z.-W.+Taoauthor=J.-H.+Tianauthor=Y.-Y.+Peiauthor=M.-L.+Yuanauthor=Y.-L.+Zhangauthor=F.-H.+Daiauthor=Y.+Liuauthor=Q.-M.+Wangauthor=J.-J.+Zhengauthor=L.+Xuauthor=E.+C.+Holmesauthor=Y.-Z.+Zhang&title=A+new+coronavirus+associated+with+human+respiratory+disease+in+China&doi=10.1038%2Fs41586-020-2008-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">A new coronavirus associated with human respiratory disease in China</span></div><div class="casAuthors">Wu, Fan; Zhao, Su; Yu, Bin; Chen, Yan-Mei; Wang, Wen; Song, Zhi-Gang; Hu, Yi; Tao, Zhao-Wu; Tian, Jun-Hua; Pei, Yuan-Yuan; Yuan, Ming-Li; Zhang, Yu-Ling; Dai, Fa-Hui; Liu, Yi; Wang, Qi-Min; Zheng, Jiao-Jiao; Xu, Lin; Holmes, Edward C.; Zhang, Yong-Zhen</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">579</span>
        (<span class="NLM_cas:issue">7798</span>),
    <span class="NLM_cas:pages">265-269</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Emerging infectious diseases, such as severe acute respiratory syndrome (SARS) and Zika virus disease, present a major threat to public health.  Despite intense research efforts, how, when and where new diseases appear are still a source of considerable uncertainty.  A severe respiratory disease was recently reported in Wuhan, Hubei province, China.  As of 25 Jan. 2020, at least 1,975 cases had been reported since the first patient was hospitalized on 12 Dec. 2019.  Epidemiol. investigations have suggested that the outbreak was assocd. with a seafood market in Wuhan.  Here we study a single patient who was a worker at the market and who was admitted to the Central Hospital of Wuhan on 26 Dec. 2019 while experiencing a severe respiratory syndrome that included fever, dizziness and a cough.  Metagenomic RNA sequencing of a sample of bronchoalveolar lavage fluid from the patient identified a new RNA virus strain from the family Coronaviridae, which is designated here 'WH-Human 1' coronavirus (and has also been referred to as '2019-nCoV').  Phylogenetic anal. of the complete viral genome (29,903 nucleotides) revealed that the virus was most closely related (89.1% nucleotide similarity) to a group of SARS-like coronaviruses (genus Betacoronavirus, subgenus Sarbecovirus) that had previously been found in bats in China.  This outbreak highlights the ongoing ability of viral spill-over from animals to cause severe disease in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5h_VHfuDuTbVg90H21EOLACvtfcHk0ljqWI1JhSRJ6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFKlsLc%253D&md5=0163a684829e880a0c3347e19f0ce52a</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2008-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2008-3%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DY.-M.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DSong%26aufirst%3DZ.-G.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DTao%26aufirst%3DZ.-W.%26aulast%3DTian%26aufirst%3DJ.-H.%26aulast%3DPei%26aufirst%3DY.-Y.%26aulast%3DYuan%26aufirst%3DM.-L.%26aulast%3DZhang%26aufirst%3DY.-L.%26aulast%3DDai%26aufirst%3DF.-H.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DQ.-M.%26aulast%3DZheng%26aufirst%3DJ.-J.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DHolmes%26aufirst%3DE.%2BC.%26aulast%3DZhang%26aufirst%3DY.-Z.%26atitle%3DA%2520new%2520coronavirus%2520associated%2520with%2520human%2520respiratory%2520disease%2520in%2520China%26jtitle%3DNature%26date%3D2020%26volume%3D579%26spage%3D265%26epage%3D269%26doi%3D10.1038%2Fs41586-020-2008-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Si, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, R.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, X.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, F.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, G.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Z.-L.</span></span> <span> </span><span class="NLM_article-title">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>579</i></span>,  <span class="NLM_fpage">270</span>– <span class="NLM_lpage">273</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-2012-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1038%2Fs41586-020-2012-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=32015507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFKlsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=579&publication_year=2020&pages=270-273&author=P.+Zhouauthor=X.-L.+Yangauthor=X.-G.+Wangauthor=B.+Huauthor=L.+Zhangauthor=W.+Zhangauthor=H.-R.+Siauthor=Y.+Zhuauthor=B.+Liauthor=C.-L.+Huangauthor=H.-D.+Chenauthor=J.+Chenauthor=Y.+Luoauthor=H.+Guoauthor=R.-D.+Jiangauthor=M.-Q.+Liuauthor=Y.+Chenauthor=X.-R.+Shenauthor=X.+Wangauthor=X.-S.+Zhengauthor=K.+Zhaoauthor=Q.-J.+Chenauthor=F.+Dengauthor=L.-L.+Liuauthor=B.+Yanauthor=F.-X.+Zhanauthor=Y.-Y.+Wangauthor=G.-F.+Xiaoauthor=Z.-L.+Shi&title=A+pneumonia+outbreak+associated+with+a+new+coronavirus+of+probable+bat+origin&doi=10.1038%2Fs41586-020-2012-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">A pneumonia outbreak associated with a new coronavirus of probable bat origin</span></div><div class="casAuthors">Zhou, Peng; Yang, Xing-Lou; Wang, Xian-Guang; Hu, Ben; Zhang, Lei; Zhang, Wei; Si, Hao-Rui; Zhu, Yan; Li, Bei; Huang, Chao-Lin; Chen, Hui-Dong; Chen, Jing; Luo, Yun; Guo, Hua; Jiang, Ren-Di; Liu, Mei-Qin; Chen, Ying; Shen, Xu-Rui; Wang, Xi; Zheng, Xiao-Shuang; Zhao, Kai; Chen, Quan-Jiao; Deng, Fei; Liu, Lin-Lin; Yan, Bing; Zhan, Fa-Xian; Wang, Yan-Yi; Xiao, Geng-Fu; Shi, Zheng-Li</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">579</span>
        (<span class="NLM_cas:issue">7798</span>),
    <span class="NLM_cas:pages">270-273</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Since the outbreak of severe acute respiratory syndrome (SARS) 18 years ago, a large no. of SARS-related coronaviruses (SARSr-CoVs) have been discovered in their natural reservoir host, bats1-4.  Previous studies have shown that some bat SARSr-CoVs have the potential to infect humans5-7.  Here we report the identification and characterization of a new coronavirus (2019-nCoV), which caused an epidemic of acute respiratory syndrome in humans in Wuhan, China.  The epidemic, which started on 12 Dec. 2019, had caused 2,794 lab.-confirmed infections including 80 deaths by 26 Jan. 2020.  Full-length genome sequences were obtained from five patients at an early stage of the outbreak.  The sequences are almost identical and share 79.6% sequence identity to SARS-CoV.  Furthermore, we show that 2019-nCoV is 96% identical at the whole-genome level to a bat coronavirus.  Pairwise protein sequence anal. of seven conserved non-structural proteins domains show that this virus belongs to the species of SARSr-CoV.  In addn., 2019-nCoV virus isolated from the bronchoalveolar lavage fluid of a critically ill patient could be neutralized by sera from several patients.  Notably, we confirmed that 2019-nCoV uses the same cell entry receptor-angiotensin converting enzyme II (ACE2)-as SARS-CoV.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrV_Wb_iUsbFrVg90H21EOLACvtfcHk0ljqWI1JhSRJ6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFKlsLg%253D&md5=236f17d4d3c7978d72513e5e0258f1b3</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-2012-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-2012-7%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DP.%26aulast%3DYang%26aufirst%3DX.-L.%26aulast%3DWang%26aufirst%3DX.-G.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DSi%26aufirst%3DH.-R.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DHuang%26aufirst%3DC.-L.%26aulast%3DChen%26aufirst%3DH.-D.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DR.-D.%26aulast%3DLiu%26aufirst%3DM.-Q.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DX.-R.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DX.-S.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DQ.-J.%26aulast%3DDeng%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DL.-L.%26aulast%3DYan%26aufirst%3DB.%26aulast%3DZhan%26aufirst%3DF.-X.%26aulast%3DWang%26aufirst%3DY.-Y.%26aulast%3DXiao%26aufirst%3DG.-F.%26aulast%3DShi%26aufirst%3DZ.-L.%26atitle%3DA%2520pneumonia%2520outbreak%2520associated%2520with%2520a%2520new%2520coronavirus%2520of%2520probable%2520bat%2520origin%26jtitle%3DNature%26date%3D2020%26volume%3D579%26spage%3D270%26epage%3D273%26doi%3D10.1038%2Fs41586-020-2012-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marra, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Astell, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holt, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks-Wilson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butterfield, Y. S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khattra, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asano, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barber, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cloutier, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coughlin, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girn, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffith, O. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montgomery, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandoh, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrescu, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schein, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddiqui, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smailus, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stott, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plummer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artsob, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bastien, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Booth, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowness, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czub, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drebot, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernando, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flick, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garbutt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grolla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyers, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kabani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Normand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroher, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tipples, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogrig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunham, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krajden, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petric, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skowronski, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upton, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roper, R. L.</span></span> <span> </span><span class="NLM_article-title">The genome sequence of the SARS-associated coronavirus</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>300</i></span>,  <span class="NLM_fpage">1399</span>– <span class="NLM_lpage">1404</span>, <span class="refDoi"> DOI: 10.1126/science.1085953</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1126%2Fscience.1085953" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=12730501" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BD3sXktFGkt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=300&publication_year=2003&pages=1399-1404&author=M.+A.+Marraauthor=J.+M.+Jonesauthor=C.+R.+Astellauthor=R.+A.+Holtauthor=A.+Brooks-Wilsonauthor=Y.+S.+N.+Butterfieldauthor=J.+Khattraauthor=J.+K.+Asanoauthor=S.+A.+Barberauthor=S.+Y.+Chanauthor=A.+Cloutierauthor=S.+M.+Coughlinauthor=D.+Freemanauthor=N.+Girnauthor=O.+L.+Griffithauthor=S.+R.+Leachauthor=M.+Mayoauthor=H.+McDonaldauthor=S.+B.+Montgomeryauthor=P.+K.+Pandohauthor=A.+S.+Petrescuauthor=A.+G.+Robertsonauthor=J.+E.+Scheinauthor=A.+Siddiquiauthor=D.+E.+Smailusauthor=J.+M.+Stottauthor=G.+S.+Yangauthor=F.+Plummerauthor=A.+Andonovauthor=H.+Artsobauthor=N.+Bastienauthor=K.+Bernardauthor=T.+F.+Boothauthor=D.+Bownessauthor=M.+Czubauthor=M.+Drebotauthor=L.+Fernandoauthor=R.+Flickauthor=M.+Garbuttauthor=M.+Grayauthor=A.+Grollaauthor=S.+Jonesauthor=H.+Feldmannauthor=A.+Meyersauthor=A.+Kabaniauthor=Y.+Liauthor=S.+Normandauthor=U.+Stroherauthor=G.+A.+Tipplesauthor=S.+Tylerauthor=R.+Vogrigauthor=D.+Wardauthor=B.+Watsonauthor=R.+C.+Brunhamauthor=M.+Krajdenauthor=M.+Petricauthor=D.+M.+Skowronskiauthor=C.+Uptonauthor=R.+L.+Roper&title=The+genome+sequence+of+the+SARS-associated+coronavirus&doi=10.1126%2Fscience.1085953"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The genome sequence of the SARS-associated coronavirus</span></div><div class="casAuthors">Marra, Marco A.; Jones, Steven J. M.; Astell, Caroline R.; Holt, Robert A.; Brooks-Wilson, Angela; Butterfield, Yaron S. N.; Khattra, Jaswinder; Asano, Jennifer K.; Barber, Sarah A.; Chan, Susanna Y.; Cloutier, Alison; Coughlin, Shaun M.; Freeman, Doug; Girn, Noreen; Griffith, Obi L.; Leach, Stephen R.; Mayo, Michael; McDonald, Helen; Montgomery, Stephen B.; Pandoh, Pawan K.; Petrescu, Anca S.; Robertson, A. Gordon; Schein, Jacqueline E.; Siddiqui, Asim; Smailus, Duane E.; Stott, Jeff M.; Yang, George S.; Plummer, Francis; Andonov, Anton; Artsob, Harvey; Bastien, Nathalie; Bernard, Kathy; Booth, Timothy F.; Bowness, Donnie; Czub, Martin; Drebot, Michael; Fernando, Lisa; Flick, Ramon; Garbutt, Michael; Gray, Michael; Grolla, Allen; Jones, Steven; Feldmann, Heinz; Meyers, Adrienne; Kabani, Amin; Li, Yan; Normand, Susan; Stroher, Ute; Tipples, Graham A.; Tyler, Shaun; Vogrig, Robert; Ward, Diane; Watson, Brynn; Brunham, Robert C.; Krajden, Mel; Petric, Martin; Skowronski, Danuta M.; Upton, Chris; Roper, Rachel L.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">300</span>
        (<span class="NLM_cas:issue">5624</span>),
    <span class="NLM_cas:pages">1399-1404</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">We sequenced the 29,751-base genome of the severe acute respiratory syndrome (SARS)-assocd. coronavirus known as the Tor2 isolate.  The genome sequence reveals that this coronavirus is only moderately related to other known coronaviruses, including two human coronaviruses, HCoV-OC43 and HCoV-229E.  Phylogenetic anal. of the predicted viral proteins indicates that the virus does not closely resemble any of the three previously known groups of coronaviruses.  The genome sequence will aid in the diagnosis of SARS virus infection in humans and potential animal hosts (using polymerase chain reaction and immunol. tests), in the development of antivirals (including neutralizing antibodies), and in the identification of putative epitopes for vaccine development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqo2-FA4hirLLVg90H21EOLACvtfcHk0ljjW2Ug2Np3Qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXktFGkt7c%253D&md5=d639d786543b32039b250134134d653b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1126%2Fscience.1085953&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1085953%26sid%3Dliteratum%253Aachs%26aulast%3DMarra%26aufirst%3DM.%2BA.%26aulast%3DJones%26aufirst%3DJ.%2BM.%26aulast%3DAstell%26aufirst%3DC.%2BR.%26aulast%3DHolt%26aufirst%3DR.%2BA.%26aulast%3DBrooks-Wilson%26aufirst%3DA.%26aulast%3DButterfield%26aufirst%3DY.%2BS.%2BN.%26aulast%3DKhattra%26aufirst%3DJ.%26aulast%3DAsano%26aufirst%3DJ.%2BK.%26aulast%3DBarber%26aufirst%3DS.%2BA.%26aulast%3DChan%26aufirst%3DS.%2BY.%26aulast%3DCloutier%26aufirst%3DA.%26aulast%3DCoughlin%26aufirst%3DS.%2BM.%26aulast%3DFreeman%26aufirst%3DD.%26aulast%3DGirn%26aufirst%3DN.%26aulast%3DGriffith%26aufirst%3DO.%2BL.%26aulast%3DLeach%26aufirst%3DS.%2BR.%26aulast%3DMayo%26aufirst%3DM.%26aulast%3DMcDonald%26aufirst%3DH.%26aulast%3DMontgomery%26aufirst%3DS.%2BB.%26aulast%3DPandoh%26aufirst%3DP.%2BK.%26aulast%3DPetrescu%26aufirst%3DA.%2BS.%26aulast%3DRobertson%26aufirst%3DA.%2BG.%26aulast%3DSchein%26aufirst%3DJ.%2BE.%26aulast%3DSiddiqui%26aufirst%3DA.%26aulast%3DSmailus%26aufirst%3DD.%2BE.%26aulast%3DStott%26aufirst%3DJ.%2BM.%26aulast%3DYang%26aufirst%3DG.%2BS.%26aulast%3DPlummer%26aufirst%3DF.%26aulast%3DAndonov%26aufirst%3DA.%26aulast%3DArtsob%26aufirst%3DH.%26aulast%3DBastien%26aufirst%3DN.%26aulast%3DBernard%26aufirst%3DK.%26aulast%3DBooth%26aufirst%3DT.%2BF.%26aulast%3DBowness%26aufirst%3DD.%26aulast%3DCzub%26aufirst%3DM.%26aulast%3DDrebot%26aufirst%3DM.%26aulast%3DFernando%26aufirst%3DL.%26aulast%3DFlick%26aufirst%3DR.%26aulast%3DGarbutt%26aufirst%3DM.%26aulast%3DGray%26aufirst%3DM.%26aulast%3DGrolla%26aufirst%3DA.%26aulast%3DJones%26aufirst%3DS.%26aulast%3DFeldmann%26aufirst%3DH.%26aulast%3DMeyers%26aufirst%3DA.%26aulast%3DKabani%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DNormand%26aufirst%3DS.%26aulast%3DStroher%26aufirst%3DU.%26aulast%3DTipples%26aufirst%3DG.%2BA.%26aulast%3DTyler%26aufirst%3DS.%26aulast%3DVogrig%26aufirst%3DR.%26aulast%3DWard%26aufirst%3DD.%26aulast%3DWatson%26aufirst%3DB.%26aulast%3DBrunham%26aufirst%3DR.%2BC.%26aulast%3DKrajden%26aufirst%3DM.%26aulast%3DPetric%26aufirst%3DM.%26aulast%3DSkowronski%26aufirst%3DD.%2BM.%26aulast%3DUpton%26aufirst%3DC.%26aulast%3DRoper%26aufirst%3DR.%2BL.%26atitle%3DThe%2520genome%2520sequence%2520of%2520the%2520SARS-associated%2520coronavirus%26jtitle%3DScience%26date%3D2003%26volume%3D300%26spage%3D1399%26epage%3D1404%26doi%3D10.1126%2Fscience.1085953" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rota, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oberste, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monroe, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nix, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campagnoli, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Icenogle, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Penaranda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bankamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maher, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frace, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeRisi, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdman, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peret, T. C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ksiazek, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rollin, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liffick, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Limor, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCaustland, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen-Rasmussen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fouchier, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunther, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osterhaus, A. D. M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drosten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallansch, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellini, W. J.</span></span> <span> </span><span class="NLM_article-title">The genome sequence of the SARS-associated coronavirus. Characterization of a novel coronavirus associated with severe acute respiratory syndrome</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>300</i></span>,  <span class="NLM_fpage">1394</span>– <span class="NLM_lpage">1399</span>, <span class="refDoi"> DOI: 10.1126/science.1085952</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1126%2Fscience.1085952" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=12730500" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BD3sXktFGkt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=300&publication_year=2003&pages=1394-1399&author=P.+A.+Rotaauthor=M.+S.+Obersteauthor=S.+S.+Monroeauthor=W.+A.+Nixauthor=R.+Campagnoliauthor=J.+P.+Icenogleauthor=S.+Penarandaauthor=B.+Bankampauthor=K.+Maherauthor=M.-H.+Chenauthor=S.+Tongauthor=A.+Taminauthor=L.+Loweauthor=M.+Fraceauthor=J.+L.+DeRisiauthor=Q.+Chenauthor=D.+Wangauthor=D.+D.+Erdmanauthor=T.+C.+T.+Peretauthor=C.+Burnsauthor=T.+G.+Ksiazekauthor=P.+E.+Rollinauthor=A.+Sanchezauthor=S.+Liffickauthor=B.+Hollowayauthor=J.+Limorauthor=K.+McCaustlandauthor=M.+Olsen-Rasmussenauthor=R.+Fouchierauthor=S.+Guntherauthor=A.+D.+M.+E.+Osterhausauthor=C.+Drostenauthor=M.+A.+Pallanschauthor=L.+J.+Andersonauthor=W.+J.+Bellini&title=The+genome+sequence+of+the+SARS-associated+coronavirus.+Characterization+of+a+novel+coronavirus+associated+with+severe+acute+respiratory+syndrome&doi=10.1126%2Fscience.1085952"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of a novel coronavirus associated with severe acute respiratory syndrome</span></div><div class="casAuthors">Rota, Paul A.; Oberste, M. Steven; Monroe, Stephan S.; Nix, W. Allan; Campagnoli, Ray; Icenogle, Joseph P.; Penaranda, Silvia; Bankamp, Bettina; Maher, Kaija; Chen, Min-hsin; Tong, Suxiong; Tamin, Azaibi; Lowe, Luis; Frace, Michael; DeRisi, Joseph L.; Chen, Qi; Wang, David; Erdman, Dean D.; Peret, Teresa C. T.; Burns, Cara; Ksiazek, Thomas G.; Rollin, Pierre E.; Sanchez, Anthony; Liffick, Stephanie; Holloway, Brian; Limor, Josef; McCaustland, Karen; Olsen-Rasmussen, Melissa; Fouchier, Ron; Guenther, Stephan; Osterhaus, Albert D. M. E.; Drosten, Christian; Pallansch, Mark A.; Anderson, Larry J.; Bellini, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">300</span>
        (<span class="NLM_cas:issue">5624</span>),
    <span class="NLM_cas:pages">1394-1399</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">In Mar. 2003, a novel coronavirus (SARS-CoV) was discovered in assocn. with cases of severe acute respiratory syndrome (SARS).  The sequence of the complete genome of SARS-CoV was detd., and the initial characterization of the viral genome is presented in this report.  The genome of SARS-CoV is 29,727 nucleotides in length and has 11 open reading frames, and its genome organization is similar to that of other coronaviruses.  Phylogenetic analyses and sequence comparisons showed that SARS-CoV is not closely related to any of the previously characterized coronaviruses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGmSeCD00yh7Vg90H21EOLACvtfcHk0ljjW2Ug2Np3Qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXktFGkt7Y%253D&md5=eb8d2156d170566242ebd78824e5d30b</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1126%2Fscience.1085952&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1085952%26sid%3Dliteratum%253Aachs%26aulast%3DRota%26aufirst%3DP.%2BA.%26aulast%3DOberste%26aufirst%3DM.%2BS.%26aulast%3DMonroe%26aufirst%3DS.%2BS.%26aulast%3DNix%26aufirst%3DW.%2BA.%26aulast%3DCampagnoli%26aufirst%3DR.%26aulast%3DIcenogle%26aufirst%3DJ.%2BP.%26aulast%3DPenaranda%26aufirst%3DS.%26aulast%3DBankamp%26aufirst%3DB.%26aulast%3DMaher%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DM.-H.%26aulast%3DTong%26aufirst%3DS.%26aulast%3DTamin%26aufirst%3DA.%26aulast%3DLowe%26aufirst%3DL.%26aulast%3DFrace%26aufirst%3DM.%26aulast%3DDeRisi%26aufirst%3DJ.%2BL.%26aulast%3DChen%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DErdman%26aufirst%3DD.%2BD.%26aulast%3DPeret%26aufirst%3DT.%2BC.%2BT.%26aulast%3DBurns%26aufirst%3DC.%26aulast%3DKsiazek%26aufirst%3DT.%2BG.%26aulast%3DRollin%26aufirst%3DP.%2BE.%26aulast%3DSanchez%26aufirst%3DA.%26aulast%3DLiffick%26aufirst%3DS.%26aulast%3DHolloway%26aufirst%3DB.%26aulast%3DLimor%26aufirst%3DJ.%26aulast%3DMcCaustland%26aufirst%3DK.%26aulast%3DOlsen-Rasmussen%26aufirst%3DM.%26aulast%3DFouchier%26aufirst%3DR.%26aulast%3DGunther%26aufirst%3DS.%26aulast%3DOsterhaus%26aufirst%3DA.%2BD.%2BM.%2BE.%26aulast%3DDrosten%26aufirst%3DC.%26aulast%3DPallansch%26aufirst%3DM.%2BA.%26aulast%3DAnderson%26aufirst%3DL.%2BJ.%26aulast%3DBellini%26aufirst%3DW.%2BJ.%26atitle%3DThe%2520genome%2520sequence%2520of%2520the%2520SARS-associated%2520coronavirus.%2520Characterization%2520of%2520a%2520novel%2520coronavirus%2520associated%2520with%2520severe%2520acute%2520respiratory%2520syndrome%26jtitle%3DScience%26date%3D2003%26volume%3D300%26spage%3D1394%26epage%3D1399%26doi%3D10.1126%2Fscience.1085952" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">World Health Organization</span>. <span> </span><span class="NLM_article-title">Cumulative Number of Reported Probable
Cases</span>, <a href="http://www.who.int/csr/sars/country/2003_06_17/en/index.html" class="extLink">http://www.who.int/csr/sars/country/2003_06_17/en/index.html</a> (accessed June 17, <span class="NLM_year">2003</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2003&author=World+Health+Organization&title=Cumulative+Number+of+Reported+Probable%0ACases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DCumulative%2520Number%2520of%2520Reported%2520Probable%250ACases%26date%3D2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">John Hopkins University</span>. <span> </span><span class="NLM_article-title">COVID-19
Dashboard by the Center for Systems Science and Engineering
(CSSE) at Johns Hopkins University (JHU)</span>, <a href="https://coronavirus.jhu.edu/map.html" class="extLink">https://coronavirus.jhu.edu/map.html</a> (accessed June 16, <span class="NLM_year">2020</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=John+Hopkins+University&title=COVID-19%0ADashboard+by+the+Center+for+Systems+Science+and+Engineering%0A%28CSSE%29+at+Johns+Hopkins+University+%28JHU%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DCOVID-19%250ADashboard%2520by%2520the%2520Center%2520for%2520Systems%2520Science%2520and%2520Engineering%250A%2528CSSE%2529%2520at%2520Johns%2520Hopkins%2520University%2520%2528JHU%2529%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thiel, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ivanov, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Putics, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hertzig, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chelle, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">WeiBbrich, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snijder, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabenau, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doerr, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorbalenya, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziebuhr, J.</span></span> <span> </span><span class="NLM_article-title">Mechanisms and enzymes involved in SARS coronavirus genome expression</span>. <i>J. Gen. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">2305</span>– <span class="NLM_lpage">2315</span>, <span class="refDoi"> DOI: 10.1099/vir.0.19424-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1099%2Fvir.0.19424-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=12917450" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BD3sXntFaqsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2003&pages=2305-2315&author=V.+Thielauthor=K.+A.+Ivanovauthor=A.+Puticsauthor=T.+Hertzigauthor=B.+Chelleauthor=S.+Bayerauthor=B.+WeiBbrichauthor=E.+J.+Snijderauthor=H.+Rabenauauthor=H.+W.+Doerrauthor=A.+E.+Gorbalenyaauthor=J.+Ziebuhr&title=Mechanisms+and+enzymes+involved+in+SARS+coronavirus+genome+expression&doi=10.1099%2Fvir.0.19424-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms and enzymes involved in SARS coronavirus genome expression</span></div><div class="casAuthors">Thiel, Volker; Ivanov, Konstantin A.; Putics, Akos; Hertzig, Tobias; Schelle, Barbara; Bayer, Sonja; Weissbrich, Benedikt; Snijder, Eric J.; Rabenau, Holger; Doerr, Hans Wilhelm; Gorbalenya, Alexander E.; Ziebuhr, John</div><div class="citationInfo"><span class="NLM_cas:title">Journal of General Virology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2305-2315</span>CODEN:
                <span class="NLM_cas:coden">JGVIAY</span>;
        ISSN:<span class="NLM_cas:issn">0022-1317</span>.
    
            (<span class="NLM_cas:orgname">Society for General Microbiology</span>)
        </div><div class="casAbstract">A novel coronavirus is the causative agent of the current epidemic of severe acute respiratory syndrome (SARS).  Coronaviruses are exceptionally large RNA viruses and employ complex regulatory mechanisms to express their genomes.  Here, we detd. the sequence of SARS coronavirus (SARS-CoV), isolate Frankfurt 1, and characterized key RNA elements and protein functions involved in viral genome expression.  Important regulatory mechanisms, such as the (discontinuous) synthesis of eight subgenomic mRNAs, ribosomal frameshifting and post-translational proteolytic processing, were addressed.  Activities of three SARS coronavirus enzymes, the helicase and two cysteine proteinases, which are known to be critically involved in replication, transcription and/or post-translational polyprotein processing, were characterized.  The availability of recombinant forms of key replicative enzymes of SARS coronavirus should pave the way for high-throughput screening approaches to identify candidate inhibitors in compd. libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwV_DQUJ6EFbVg90H21EOLACvtfcHk0likcYd3hz2Y6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXntFaqsb4%253D&md5=c0e344cdde7ce697dee5a6aaeea9feb7</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1099%2Fvir.0.19424-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1099%252Fvir.0.19424-0%26sid%3Dliteratum%253Aachs%26aulast%3DThiel%26aufirst%3DV.%26aulast%3DIvanov%26aufirst%3DK.%2BA.%26aulast%3DPutics%26aufirst%3DA.%26aulast%3DHertzig%26aufirst%3DT.%26aulast%3DChelle%26aufirst%3DB.%26aulast%3DBayer%26aufirst%3DS.%26aulast%3DWeiBbrich%26aufirst%3DB.%26aulast%3DSnijder%26aufirst%3DE.%2BJ.%26aulast%3DRabenau%26aufirst%3DH.%26aulast%3DDoerr%26aufirst%3DH.%2BW.%26aulast%3DGorbalenya%26aufirst%3DA.%2BE.%26aulast%3DZiebuhr%26aufirst%3DJ.%26atitle%3DMechanisms%2520and%2520enzymes%2520involved%2520in%2520SARS%2520coronavirus%2520genome%2520expression%26jtitle%3DJ.%2520Gen.%2520Virol.%26date%3D2003%26volume%3D84%26spage%3D2305%26epage%3D2315%26doi%3D10.1099%2Fvir.0.19424-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curth, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drosten, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauerhering, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rox, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span> <span> </span><span class="NLM_article-title">Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved α-ketoamide inhibitors</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>, <span class="NLM_elocation-id">eabb3405</span> <span class="refDoi"> DOI: 10.1126/science.abb3405</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1126%2Fscience.abb3405" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&author=L.+Zhangauthor=D.+Linauthor=X.+Sunauthor=U.+Curthauthor=C.+Drostenauthor=L.+Sauerheringauthor=S.+Beckerauthor=K.+Roxauthor=R.+Hilgenfeld&title=Crystal+structure+of+SARS-CoV-2+main+protease+provides+a+basis+for+design+of+improved+%CE%B1-ketoamide+inhibitors&doi=10.1126%2Fscience.abb3405"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1126%2Fscience.abb3405&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abb3405%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DD.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DCurth%26aufirst%3DU.%26aulast%3DDrosten%26aufirst%3DC.%26aulast%3DSauerhering%26aufirst%3DL.%26aulast%3DBecker%26aufirst%3DS.%26aulast%3DRox%26aufirst%3DK.%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3DCrystal%2520structure%2520of%2520SARS-CoV-2%2520main%2520protease%2520provides%2520a%2520basis%2520for%2520design%2520of%2520improved%2520%25CE%25B1-ketoamide%2520inhibitors%26jtitle%3DScience%26date%3D2020%26volume%3D368%26doi%3D10.1126%2Fscience.abb3405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of antiviral drug candidates targeting the SARS-CoV-2 main protease</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">1331</span>– <span class="NLM_lpage">1335</span>, <span class="refDoi"> DOI: 10.1126/science.abb4489</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1126%2Fscience.abb4489" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=32321856" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1alsrzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=1331-1335&author=W.+Daiauthor=B.+Zhangauthor=X.-M.+Jiangauthor=H.+Suauthor=J.+Liauthor=Y.+Zhaoauthor=X.+Xieauthor=Z.+Jinauthor=J.+Pengauthor=F.+Liuauthor=C.+Liauthor=Y.+Liauthor=F.+Baiauthor=H.+Wangauthor=X.+Chengauthor=X.+Cenauthor=S.+Huauthor=X.+Yangauthor=J.+Wangauthor=X.+Liuauthor=G.+Xiaoauthor=H.+Jiangauthor=Z.+Raoauthor=L.-K.+Zhangauthor=Y.+Xuauthor=H.+Yangauthor=H.+Liu&title=Structure-based+design+of+antiviral+drug+candidates+targeting+the+SARS-CoV-2+main+protease&doi=10.1126%2Fscience.abb4489"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design of antiviral drug candidates targeting the SARS-CoV-2 main protease</span></div><div class="casAuthors">Dai, Wenhao; Zhang, Bing; Jiang, Xia-Ming; Su, Haixia; Li, Jian; Zhao, Yao; Xie, Xiong; Jin, Zhenming; Peng, Jingjing; Liu, Fengjiang; Li, Chunpu; Li, You; Bai, Fang; Wang, Haofeng; Cheng, Xi; Cen, Xiaobo; Hu, Shulei; Yang, Xiuna; Wang, Jiang; Liu, Xiang; Xiao, Gengfu; Jiang, Hualiang; Rao, Zihe; Zhang, Lei-Ke; Xu, Yechun; Yang, Haitao; Liu, Hong</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6497</span>),
    <span class="NLM_cas:pages">1331-1335</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">SARS-CoV-2 (severe acute respiratory syndrome coronavirus 2) is the etiol. agent responsible for the global COVID-19 (coronavirus disease 2019) outbreak.  The main protease of SARS-CoV-2, Mpro, is a key enzyme that plays a pivotal role in mediating viral replication and transcription.  We designed and synthesized two lead compds. (11a and 11b) targeting Mpro.  Both exhibited excellent inhibitory activity and potent anti-SARS-CoV-2 infection activity.  The x-ray crystal structures of SARS-CoV-2 Mpro in complex with 11a or 11b, both detd. at a resoln. of 1.5 angstroms, showed that the aldehyde groups of 11a and 11b are covalently bound to cysteine 145 of Mpro.  Both compds. showed good pharmacokinetic properties in vivo, and 11a also exhibited low toxicity, which suggests that these compds. are promising drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFFvzOwCOpZrVg90H21EOLACvtfcHk0likcYd3hz2Y6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1alsrzL&md5=ec87c340410edb919d2584a2b4d33e1a</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1126%2Fscience.abb4489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.abb4489%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DJiang%26aufirst%3DX.-M.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DJin%26aufirst%3DZ.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DBai%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DCen%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DXiao%26aufirst%3DG.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DRao%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DL.-K.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DH.%26atitle%3DStructure-based%2520design%2520of%2520antiviral%2520drug%2520candidates%2520targeting%2520the%2520SARS-CoV-2%2520main%2520protease%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3D1331%26epage%3D1335%26doi%3D10.1126%2Fscience.abb4489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Webber, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okano, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Little, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reich, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuhrman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Love, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendrickson, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patrick, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meador, J. W.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferre, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binford, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Worland, S. T.</span></span> <span> </span><span class="NLM_article-title">Tripeptide aldehyde inhibitors of human rhinovirus 3C protease: design, synthesis, biological evaluation, and cocrystal structure solution of P1 glutamine isosteric replacements</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">2786</span>– <span class="NLM_lpage">2805</span>, <span class="refDoi"> DOI: 10.1021/jm980071x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm980071x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK1cXksVKrtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=2786-2805&author=S.+E.+Webberauthor=K.+Okanoauthor=T.+L.+Littleauthor=S.+H.+Reichauthor=Y.+Xinauthor=S.+A.+Fuhrmanauthor=D.+A.+Matthewsauthor=R.+A.+Loveauthor=T.+F.+Hendricksonauthor=A.+K.+Patrickauthor=J.+W.+Meadorauthor=R.+A.+Ferreauthor=E.+L.+Brownauthor=C.+E.+Fordauthor=S.+L.+Binfordauthor=S.+T.+Worland&title=Tripeptide+aldehyde+inhibitors+of+human+rhinovirus+3C+protease%3A+design%2C+synthesis%2C+biological+evaluation%2C+and+cocrystal+structure+solution+of+P1+glutamine+isosteric+replacements&doi=10.1021%2Fjm980071x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Tripeptide Aldehyde Inhibitors of Human Rhinovirus 3C Protease: Design, Synthesis, Biological Evaluation, and Cocrystal Structure Solution of P1 Glutamine Isosteric Replacements</span></div><div class="casAuthors">Webber, Stephen E.; Okano, Koji; Little, Thomas L.; Reich, Siegfried H.; Xin, Yue; Fuhrman, Shella A.; Matthews, David A.; Love, Robert A.; Hendrickson, Thomas F.; Patick, Amy K.; Meador, James W., III; Ferre, Rose Ann; Brown, Edward L.; Ford, Clifford E.; Binford, Susan L.; Worland, Stephen T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2786-2805</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The investigation of tripeptide aldehydes I [R = CH2CONH2, NHCHO, NHAc, NHCOEt, NHCOCHMe2, NHCOCF3, NHBz, NMeAc, 2-oxo-1-pyrrolidinyl, 5-isoxazolylcarbonylamino, NHCO2Me, NHBoc, NHSO2Me, NHCONMe2, CH2CONMe2, CH2S(O)Me, CH2CONHPh3, CH2CN] as reversible covalent inhibitors of human rhinovirus (HRV) 3C protease (3CP) is reported.  Mol. models based on the apo crystal structure of HRV-14 3CP and other trypsin-like serine proteases were constructed to approx. the binding of peptide substrates, generate transition state models of P1-P1' amide cleavage, and propose novel tripeptide aldehydes.  Glutaminal derivs. have limitations since they exist predominantly in the cyclic hemiaminal form.  Therefore, several isosteric replacements for the P1 carboxamide side chain were designed and incorporated into the tripeptide aldehydes.  These compds. were found to be potent inhibitors of purified HRV-14 3CP with Kis ranging from 0.005 to 0.64 μM.  Several have low micromolar antiviral activity when tested against HRV-14-infected H1-HeLa cells.  The N-acetyl deriv. I (R = NHAc) was also shown to be active against HRV serotypes 2, 16, and 89.  High-resoln. cocrystal structures of HRV-2 3CP, covalently bound to I [R = NHAc, CH2CONMe2, CH2S(O)Me] were solved.  These cocrystal structures were analyzed and compared with the author's original HRV-14 3CP-substrate and inhibitor models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvPwUsYQ057bVg90H21EOLACvtfcHk0liiUXmFQQfqJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXksVKrtrg%253D&md5=4ac8f340209d300aa1dbee0005e3c717</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fjm980071x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm980071x%26sid%3Dliteratum%253Aachs%26aulast%3DWebber%26aufirst%3DS.%2BE.%26aulast%3DOkano%26aufirst%3DK.%26aulast%3DLittle%26aufirst%3DT.%2BL.%26aulast%3DReich%26aufirst%3DS.%2BH.%26aulast%3DXin%26aufirst%3DY.%26aulast%3DFuhrman%26aufirst%3DS.%2BA.%26aulast%3DMatthews%26aufirst%3DD.%2BA.%26aulast%3DLove%26aufirst%3DR.%2BA.%26aulast%3DHendrickson%26aufirst%3DT.%2BF.%26aulast%3DPatrick%26aufirst%3DA.%2BK.%26aulast%3DMeador%26aufirst%3DJ.%2BW.%26aulast%3DFerre%26aufirst%3DR.%2BA.%26aulast%3DBrown%26aufirst%3DE.%2BL.%26aulast%3DFord%26aufirst%3DC.%2BE.%26aulast%3DBinford%26aufirst%3DS.%2BL.%26aulast%3DWorland%26aufirst%3DS.%2BT.%26atitle%3DTripeptide%2520aldehyde%2520inhibitors%2520of%2520human%2520rhinovirus%25203C%2520protease%253A%2520design%252C%2520synthesis%252C%2520biological%2520evaluation%252C%2520and%2520cocrystal%2520structure%2520solution%2520of%2520P1%2520glutamine%2520isosteric%2520replacements%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26spage%3D2786%26epage%3D2805%26doi%3D10.1021%2Fjm980071x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shepherd, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKinney, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakulchik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villarreal, E. C.</span></span> <span> </span><span class="NLM_article-title">Small peptidic aldehyde inhibitors of human rhinovirus 3C protease</span>. <i>Biorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">2893</span>– <span class="NLM_lpage">2896</span>, <span class="refDoi"> DOI: 10.1016/S0960-894X(96)00537-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1016%2FS0960-894X%2896%2900537-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK2sXhtleksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1996&pages=2893-2896&author=T.+A.+Shepherdauthor=G.+A.+Coxauthor=E.+McKinneyauthor=J.+Tangauthor=M.+Wakulchikauthor=R.+E.+Zimmermanauthor=E.+C.+Villarreal&title=Small+peptidic+aldehyde+inhibitors+of+human+rhinovirus+3C+protease&doi=10.1016%2FS0960-894X%2896%2900537-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Small peptidic aldehyde inhibitors of human rhinovirus 3C protease</span></div><div class="casAuthors">Shepherd, Timothy A.; Cox, Gregory A.; McKinney, Emma; Tang, Joseph; Wakulchik, Mark; Zimmerman, Ronald E.; Villarreal, Elcira C.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">2893-2896</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Small peptide aldehydes were designed to mimic the preferred substrate requirements for the human rhinovirus 3C protease.  Di- and tripeptide aldehydes contg. a methionine sulfone as a P1 surrogate for glutamine show low micromol. enzyme inhibitory and antiviral tissue culture activity.  LY338387, obtained in a short and efficient synthesis, appears to validate the protease as a therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVNHDYhXYZnrVg90H21EOLACvtfcHk0liiUXmFQQfqJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXhtleksQ%253D%253D&md5=320b47228b1b730493851bf35eed9b3c</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2896%2900537-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252896%252900537-9%26sid%3Dliteratum%253Aachs%26aulast%3DShepherd%26aufirst%3DT.%2BA.%26aulast%3DCox%26aufirst%3DG.%2BA.%26aulast%3DMcKinney%26aufirst%3DE.%26aulast%3DTang%26aufirst%3DJ.%26aulast%3DWakulchik%26aufirst%3DM.%26aulast%3DZimmerman%26aufirst%3DR.%2BE.%26aulast%3DVillarreal%26aufirst%3DE.%2BC.%26atitle%3DSmall%2520peptidic%2520aldehyde%2520inhibitors%2520of%2520human%2520rhinovirus%25203C%2520protease%26jtitle%3DBiorg.%2520Med.%2520Chem.%2520Lett.%26date%3D1996%26volume%3D6%26spage%3D2893%26epage%3D2896%26doi%3D10.1016%2FS0960-894X%2896%2900537-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaldor, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammond, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dressman, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labus, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chadwell, F. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kline, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinz, B. A.</span></span> <span> </span><span class="NLM_article-title">Glutamine-derived aldehydes for the inhibition of human rhinovirus 3C protease</span>. <i>Biorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">2021</span>– <span class="NLM_lpage">2026</span>, <span class="refDoi"> DOI: 10.1016/0960-894X(95)00345-T</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1016%2F0960-894X%2895%2900345-T" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK2MXnvFKgsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1995&pages=2021-2026&author=S.+W.+Kaldorauthor=M.+Hammondauthor=B.+A.+Dressmanauthor=J.+M.+Labusauthor=F.+W.+Chadwellauthor=A.+D.+Klineauthor=B.+A.+Heinz&title=Glutamine-derived+aldehydes+for+the+inhibition+of+human+rhinovirus+3C+protease&doi=10.1016%2F0960-894X%2895%2900345-T"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Glutamine-derived aldehydes for the inhibition of human rhinovirus 3C protease</span></div><div class="casAuthors">Kaldor, Stephen W.; Hammond, Marlys; Dressman, Bruce A.; Labus, Jean M.; Chadwell, Frederick W.; Kline, Allen D.; Heinz, Beverly A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2021-6</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Peptide aldehydes have been synthesized and evaluated as inhibitors of human rhinovirus 3C protease.  Those inhibitors contg. a C-terminal glutamine aldehyde were prepd. using newly developed methodol. involving redn. of the corresponding glutarimide, which can be easily prepd. from N-protected glutamine.  Low mol. wt. (<500) compds. with low to submicromolar inhibitory activity in both isolated enzyme and in vitro translation assays have been identified.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq70iEZSn1YkrVg90H21EOLACvtfcHk0liiUXmFQQfqJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXnvFKgsrs%253D&md5=79f602bff4045ffa3a223758830d4d32</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2F0960-894X%2895%2900345-T&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0960-894X%252895%252900345-T%26sid%3Dliteratum%253Aachs%26aulast%3DKaldor%26aufirst%3DS.%2BW.%26aulast%3DHammond%26aufirst%3DM.%26aulast%3DDressman%26aufirst%3DB.%2BA.%26aulast%3DLabus%26aufirst%3DJ.%2BM.%26aulast%3DChadwell%26aufirst%3DF.%2BW.%26aulast%3DKline%26aufirst%3DA.%2BD.%26aulast%3DHeinz%26aufirst%3DB.%2BA.%26atitle%3DGlutamine-derived%2520aldehydes%2520for%2520the%2520inhibition%2520of%2520human%2520rhinovirus%25203C%2520protease%26jtitle%3DBiorg.%2520Med.%2520Chem.%2520Lett.%26date%3D1995%26volume%3D5%26spage%3D2021%26epage%3D2026%26doi%3D10.1016%2F0960-894X%2895%2900345-T" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malcolm, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shechosky, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKay, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vederas, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sant, D. V.</span></span> <span> </span><span class="NLM_article-title">Peptide aldehyde inhibitors of hepatitis A virus 3C proteinase</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">8172</span>– <span class="NLM_lpage">8179</span>, <span class="refDoi"> DOI: 10.1021/bi00025a024</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00025a024" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK2MXmtVyit7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1995&pages=8172-8179&author=B.+A.+Malcolmauthor=C.+Loweauthor=S.+Shechoskyauthor=R.+T.+McKayauthor=C.+C.+Yangauthor=V.+J.+Shahauthor=R.+J.+Simonauthor=J.+C.+Vederasauthor=D.+V.+Sant&title=Peptide+aldehyde+inhibitors+of+hepatitis+A+virus+3C+proteinase&doi=10.1021%2Fbi00025a024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide Aldehyde Inhibitors of Hepatitis A Virus 3C Proteinase</span></div><div class="casAuthors">Malcolm, Bruce A.; Lowe, Christopher; Shechosky, Shirley; McKay, Ryan T.; Yang, Chi Ching; Shah, Vibhakar J.; Simon, Reyna J.; Vederas, John C.; Smith, Daniel V.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">8172-9</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Picornaviral 3C proteinases are a group of closely related thiol proteinases responsible for processing of the viral polyprotein into its component proteins.  These proteinases adopt a chymotrypsin-like fold [Allaire et al. (1994) Nature 369, 72-77; Matthews et al. (1994) Cell 77, 761-771] and a display an active-site configuration like those of the serine proteinases.  Peptide-aldehydes based on the preferred peptide substrates for hepatitis A virus (HAV) 3C proteinase (picornain 3C) were synthesized by redn. of a thioester precursor.  Acetyl-Leu-Ala-Ala-(N,N'-dimethylglutaminal) was found to be a reversible, slow-binding inhibitor for HAV 3C with a Ki* of 4.2 × 10-8 M.  This inhibitor showed 50-fold less activity against the highly homologous human rhinovirus (strain 14) 3C proteinase, whose peptide substrate specificity is slightly different, suggesting a high degree of selectivity.  NMR spectrometry of the adduct of the 13C-labeled inhibitor with the HAV-3C proteinase indicate that a thiohemiacetal is formed between the enzyme and the aldehyde carbon as previously noted for peptide-aldehyde inhibitors of papain [Lewis & Wolfenden (1977) Biochem. 16, 4890-4894; Gamcsik et al. (1983) J. Am. Chem. Soc. 105, 6324-6325].  The adduct can also be obsd. by electrospray mass spectrometry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyqMCwJj1TxLVg90H21EOLACvtfcHk0ljxQ9XBdhVxQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmtVyit7c%253D&md5=8ed831816e5289ceae4879cc67769b46</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fbi00025a024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00025a024%26sid%3Dliteratum%253Aachs%26aulast%3DMalcolm%26aufirst%3DB.%2BA.%26aulast%3DLowe%26aufirst%3DC.%26aulast%3DShechosky%26aufirst%3DS.%26aulast%3DMcKay%26aufirst%3DR.%2BT.%26aulast%3DYang%26aufirst%3DC.%2BC.%26aulast%3DShah%26aufirst%3DV.%2BJ.%26aulast%3DSimon%26aufirst%3DR.%2BJ.%26aulast%3DVederas%26aufirst%3DJ.%2BC.%26aulast%3DSant%26aufirst%3DD.%2BV.%26atitle%3DPeptide%2520aldehyde%2520inhibitors%2520of%2520hepatitis%2520A%2520virus%25203C%2520proteinase%26jtitle%3DBiochemistry%26date%3D1995%26volume%3D34%26spage%3D8172%26epage%3D8179%26doi%3D10.1021%2Fbi00025a024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marquis, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ru, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamashita, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomaszek, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ijames, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janson, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Meguid, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Alessio, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McQueney, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veber, D. F.</span></span> <span> </span><span class="NLM_article-title">Potent dipeptidylketone inhibitors of the cysteine protease cathepsin K</span>. <i>Biorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">581</span>– <span class="NLM_lpage">588</span>, <span class="refDoi"> DOI: 10.1016/S0968-0896(99)00011-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1016%2FS0968-0896%2899%2900011-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10353637" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK1MXis12mt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1999&pages=581-588&author=R.+W.+Marquisauthor=Y.+Ruauthor=D.+S.+Yamashitaauthor=H.-J.+Ohauthor=J.+Yenauthor=S.+K.+Thompsonauthor=T.+J.+Carrauthor=M.+A.+Levyauthor=T.+A.+Tomaszekauthor=C.+F.+Ijamesauthor=W.+W.+Smithauthor=B.+Zhaoauthor=C.+A.+Jansonauthor=S.+S.+Abdel-Meguidauthor=K.+J.+D%E2%80%99Alessioauthor=M.+S.+McQueneyauthor=D.+F.+Veber&title=Potent+dipeptidylketone+inhibitors+of+the+cysteine+protease+cathepsin+K&doi=10.1016%2FS0968-0896%2899%2900011-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Potent dipeptidylketone inhibitors of the cysteine protease cathepsin K</span></div><div class="casAuthors">Marquis, Robert W.; Ru, Yu; Yamashita, Dennis S.; Oh, Hye-Ja; Yen, Jack; Thompson, Scott K.; Carr, Thomas J.; Levy, Mark A.; Tomaszek, Thaddeus A.; Ijames, Carl F.; Smith, Ward W.; Zhao, Baoguang; Janson, Cheryl A.; Abdel-Meguid, Sherin S.; D'Alessio, Karla J.; McQueney, M. S.; Veber, Daniel F.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">581-588</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Cathepsin K (EC 3.4.22.38) is a cysteine protease of the papain superfamily which is selectively expressed within the osteoclast.  Several lines of evidence have pointed to the fact that this protease may play an important role in the degrdn. of the bone matrix.  Potent and selective inhibitors of cathepsin K could be important therapeutic agents for the control of excessive bone resorption.  Recently a series of peptide aldehydes have been shown to be potent inhibitors of cathepsin K.  In an effort to design more selective and metabolically stable inhibitors of cathepsin K, a series of electronically attenuated alkoxymethylketones and thiomethylketones inhibitors, e.g. I [R = iso-Bu, (CH2)4NH2, Me, CH2OH; X = OMe, OCH2Ph-4-Ph, SPh, SCH2Ph] have been synthesized.  The X-ray co-crystal structure of one of these analogs in complex with cathepsin K shows the inhibitor binding in the primed side of the enzyme active site with a covalent interaction between the active site cysteine 25 and the carbonyl carbon of the inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqScu6Z7mFzxrVg90H21EOLACvtfcHk0ljxQ9XBdhVxQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXis12mt7o%253D&md5=592914f0ef29bf154bd29e1a057def38</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2FS0968-0896%2899%2900011-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0968-0896%252899%252900011-5%26sid%3Dliteratum%253Aachs%26aulast%3DMarquis%26aufirst%3DR.%2BW.%26aulast%3DRu%26aufirst%3DY.%26aulast%3DYamashita%26aufirst%3DD.%2BS.%26aulast%3DOh%26aufirst%3DH.-J.%26aulast%3DYen%26aufirst%3DJ.%26aulast%3DThompson%26aufirst%3DS.%2BK.%26aulast%3DCarr%26aufirst%3DT.%2BJ.%26aulast%3DLevy%26aufirst%3DM.%2BA.%26aulast%3DTomaszek%26aufirst%3DT.%2BA.%26aulast%3DIjames%26aufirst%3DC.%2BF.%26aulast%3DSmith%26aufirst%3DW.%2BW.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DJanson%26aufirst%3DC.%2BA.%26aulast%3DAbdel-Meguid%26aufirst%3DS.%2BS.%26aulast%3DD%25E2%2580%2599Alessio%26aufirst%3DK.%2BJ.%26aulast%3DMcQueney%26aufirst%3DM.%2BS.%26aulast%3DVeber%26aufirst%3DD.%2BF.%26atitle%3DPotent%2520dipeptidylketone%2520inhibitors%2520of%2520the%2520cysteine%2520protease%2520cathepsin%2520K%26jtitle%3DBiorg.%2520Med.%2520Chem.%26date%3D1999%26volume%3D7%26spage%3D581%26epage%3D588%26doi%3D10.1016%2FS0968-0896%2899%2900011-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellman, J. A.</span></span> <span> </span><span class="NLM_article-title">General solid-phase method to prepare novel cyclic ketone inhibitors of the cysteine protease cruzain</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2993</span>, <span class="refDoi"> DOI: 10.1016/S0960-894X(02)00619-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1016%2FS0960-894X%2802%2900619-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=12270191" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BD38Xntlansr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2002&pages=2993&author=L.+Huangauthor=J.+A.+Ellman&title=General+solid-phase+method+to+prepare+novel+cyclic+ketone+inhibitors+of+the+cysteine+protease+cruzain&doi=10.1016%2FS0960-894X%2802%2900619-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">General solid-phase method to prepare novel cyclic ketone inhibitors of the cysteine protease cruzain</span></div><div class="casAuthors">Huang, Lily; Ellman, Jonathan A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2993-2996</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A series of constrained ketone-based inhibitors has been developed that show low nanomolar Ki values.  These ketone inhibitors showed promising activity towards cruzain, the cysteine protease implicated in Chagas' disease.  This series of constrained inhibitors, which can be accessed quickly and efficiently using a solid-phase combinatorial strategy, should be applicable to other members of the cysteine protease class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPoZFsUb1mHrVg90H21EOLACvtfcHk0ljxQ9XBdhVxQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xntlansr4%253D&md5=0f84ccaeaca3bbd862ac577f6ee83c7b</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2802%2900619-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252802%252900619-4%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DEllman%26aufirst%3DJ.%2BA.%26atitle%3DGeneral%2520solid-phase%2520method%2520to%2520prepare%2520novel%2520cyclic%2520ketone%2520inhibitors%2520of%2520the%2520cysteine%2520protease%2520cruzain%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2002%26volume%3D12%26spage%3D2993%26doi%3D10.1016%2FS0960-894X%2802%2900619-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamashita, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janson, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomaszek, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bossard, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McQueney, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Alessio, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amegadzie, B. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanning, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Meguid, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DesJarlais, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gleason, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veber, D. F.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Structure and design of potent and selective cathepsin K inhibitors</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">11351</span>– <span class="NLM_lpage">11352</span>, <span class="refDoi"> DOI: 10.1021/ja972204u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja972204u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK2sXnsVSlt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=1997&pages=11351-11352&author=D.+S.+Yamashitaauthor=W.+W.+Smithauthor=B.+Zhaoauthor=C.+A.+Jansonauthor=T.+A.+Tomaszekauthor=C.+A.+Bossardauthor=M.+A.+Levyauthor=H.-J.+Ohauthor=S.+A.+Carrauthor=M.+McQueneyauthor=K.+J.+D%E2%80%99Alessioauthor=B.+Y.+Amegadzieauthor=C.+R.+Hanningauthor=S.+Abdel-Meguidauthor=R.+L.+DesJarlaisauthor=J.+G.+Gleasonauthor=D.+F.+Veber&title=Structure+and+design+of+potent+and+selective+cathepsin+K+inhibitors&doi=10.1021%2Fja972204u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and Design of Potent and Selective Cathepsin K Inhibitors</span></div><div class="casAuthors">Yamashita, Dennis S.; Smith, Ward W.; Zhao, Baoguang; Janson, Cheryl A.; Tomaszek, Thaddeus A.; Bossard, Mary J.; Levy, Mark A.; Oh, Hye-Ja; Carr, Thomas J.; Thompson, Scott K.; Ijames, Carl F.; Carr, Steven A.; McQueney, Michael; D'Alessio, Karla J.; Amegadzie, Bernard Y.; Hanning, Charles R.; Abdel-Meguid, Sherin; DesJarlais, Renee L.; Gleason, John G.; Veber, Daniel F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">11351-11352</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The sym. diacylaminomethyl ketone, Cbz-Leu-NHCH2COCH2NH-(Cbz-Leu) (I), is a reversible, time-independent, selective inhibitor of the papain-related, osteoclast-specific cysteine protease, cathepsin K (Ki,app cathepsin K = 22 nM, Ki,app papain, >10 μM, Ki,app cathepsin L = 0.34 μM, cathepsin B = 1.3 μM, cathepsin S = 0.89 μM, Ki,app trypsin >50 μM, Ki,app chymotrypsin >50 μM).  An X-ray co-crystal structure of I bound to cathepsin K shows addn. of the active site thiol to the ketone and specific binding on both sides of the active site.  The effects of C- and N-methylation of the I on potency against cathepsin K were examd.  Also, the diacylaminomethyl ketone scaffold was incorporated into a conformationally constraining pyrrolidinone ring and the binding orientation of this ketone on cathepsin K was detd. by X-ray co-crystallog.  The 4-phenoxy Ph sulfonamide was found to be a Cbz-Leucine amide mimetic.  An X-ray co-crystal structure of this mimetic compd. bound to cathepsin K was solved.  The α,α'-diaminomethyl ketone nucleus may be a general scaffold for inhibition of cysteine proteases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9nqPi1iGo47Vg90H21EOLACvtfcHk0lil6m83tNCErw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXnsVSlt78%253D&md5=2c459df27f8d7fabdad2b1df913c1020</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fja972204u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja972204u%26sid%3Dliteratum%253Aachs%26aulast%3DYamashita%26aufirst%3DD.%2BS.%26aulast%3DSmith%26aufirst%3DW.%2BW.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DJanson%26aufirst%3DC.%2BA.%26aulast%3DTomaszek%26aufirst%3DT.%2BA.%26aulast%3DBossard%26aufirst%3DC.%2BA.%26aulast%3DLevy%26aufirst%3DM.%2BA.%26aulast%3DOh%26aufirst%3DH.-J.%26aulast%3DCarr%26aufirst%3DS.%2BA.%26aulast%3DMcQueney%26aufirst%3DM.%26aulast%3DD%25E2%2580%2599Alessio%26aufirst%3DK.%2BJ.%26aulast%3DAmegadzie%26aufirst%3DB.%2BY.%26aulast%3DHanning%26aufirst%3DC.%2BR.%26aulast%3DAbdel-Meguid%26aufirst%3DS.%26aulast%3DDesJarlais%26aufirst%3DR.%2BL.%26aulast%3DGleason%26aufirst%3DJ.%2BG.%26aulast%3DVeber%26aufirst%3DD.%2BF.%26atitle%3DStructure%2520and%2520design%2520of%2520potent%2520and%2520selective%2520cathepsin%2520K%2520inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1997%26volume%3D119%26spage%3D11351%26epage%3D11352%26doi%3D10.1021%2Fja972204u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moon, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanzlik, R. P.</span></span> <span> </span><span class="NLM_article-title">Reversible covalent inhibition of papain by a peptide nitrile. Carbon-13 NMR evidence for a thioimidate ester adduct</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1986</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">1350</span>– <span class="NLM_lpage">1351</span>, <span class="refDoi"> DOI: 10.1021/ja00266a066</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00266a066" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaL28Xht1Gmtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=1986&pages=1350-1351&author=J.+B.+Moonauthor=R.+S.+Colemanauthor=R.+P.+Hanzlik&title=Reversible+covalent+inhibition+of+papain+by+a+peptide+nitrile.+Carbon-13+NMR+evidence+for+a+thioimidate+ester+adduct&doi=10.1021%2Fja00266a066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible covalent inhibition of papain by a peptide nitrile.  Carbon-13 NMR evidence for a thioimidate ester adduct</span></div><div class="casAuthors">Moon, Joseph B.; Coleman, Robert S.; Hanzlik, Robert P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1350-1</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Acetyl-L-phenylalaninylglcyine nitrile (I) is a powerful competitive inhibitor of papain (Ki = 1 μM).  [Nitrile-13C]-I was synthesized, and its interaction with papain was obsd. by 13C NMR spectroscopy.  The 13C NMR spectrum of the free ligand shows a single resonance at 116.61 ppm.  In the presence of papain (∼0.65 molar equivalents), a new signal at 182.08 ppm is seen; this signal disappears immediately when the soln. is acidified and treated with 2,2'-dipyridyl disulfide, which reacts irreversibly with the active-site SH of papain.  Thus, assocn. of I with papain involves reversible formation of a covalent thioimidate bond with the active-site SH of the enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCxv9ppgcPL7Vg90H21EOLACvtfcHk0lil6m83tNCErw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL28Xht1Gmtbk%253D&md5=94a9e892576d62c31a51b030f1cf7a17</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fja00266a066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00266a066%26sid%3Dliteratum%253Aachs%26aulast%3DMoon%26aufirst%3DJ.%2BB.%26aulast%3DColeman%26aufirst%3DR.%2BS.%26aulast%3DHanzlik%26aufirst%3DR.%2BP.%26atitle%3DReversible%2520covalent%2520inhibition%2520of%2520papain%2520by%2520a%2520peptide%2520nitrile.%2520Carbon-13%2520NMR%2520evidence%2520for%2520a%2520thioimidate%2520ester%2520adduct%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1986%26volume%3D108%26spage%3D1350%26epage%3D1351%26doi%3D10.1021%2Fja00266a066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cowen, S. D.</span>; <span class="NLM_string-name">Greenspan, D. P.</span>; <span class="NLM_string-name">McQuire, L. W.</span>; <span class="NLM_string-name">Tommasi, A. R.</span>; <span class="NLM_string-name">Van Duzer, J. H.</span></span> <span> </span><span class="NLM_article-title">N-substituted peptidyl nitriles as cysteine cathepsin inhibitors</span>. <span class="NLM_patent">WO01/87828 A1</span>, publication date Nov 22, <span class="NLM_year">2001</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2001&author=S.+D.+Cowen&author=D.+P.+Greenspan&author=L.+W.+McQuire&author=A.+R.+Tommasi&author=J.+H.+Van+Duzer&title=N-substituted+peptidyl+nitriles+as+cysteine+cathepsin+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DCowen%26aufirst%3DS.%2BD.%26atitle%3DN-substituted%2520peptidyl%2520nitriles%2520as%2520cysteine%2520cathepsin%2520inhibitors%26date%3D2001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patil, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golubski, Z. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hori, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tehrani, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foreman, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eveleth, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartus, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, J. C.</span></span> <span> </span><span class="NLM_article-title">Peptide alpha-keto ester, alpha-keto amide, and alpha-keto acid inhibitors of calpains and other cysteine proteases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">3472</span>– <span class="NLM_lpage">3480</span>, <span class="refDoi"> DOI: 10.1021/jm00074a031</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00074a031" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK3sXmsVCitbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1993&pages=3472-3480&author=Z.+Liauthor=G.+S.+Patilauthor=Z.+E.+Golubskiauthor=H.+Horiauthor=K.+Tehraniauthor=J.+E.+Foremanauthor=D.+D.+Evelethauthor=R.+T.+Bartusauthor=J.+C.+Powers&title=Peptide+alpha-keto+ester%2C+alpha-keto+amide%2C+and+alpha-keto+acid+inhibitors+of+calpains+and+other+cysteine+proteases&doi=10.1021%2Fjm00074a031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide α-keto ester, α-keto amide, and α-keto acid inhibitors of calpains and other cysteine proteases</span></div><div class="casAuthors">Li, Zhaozhao; Patil, Girish S.; Golubski, Zbigniew E.; Hori, Hitoshi; Tehrani, Kamin; Foreman, J. E.; Eveleth, David D.; Bartus, Raymond T.; Powers, James C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">3472-80</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">A series of dipeptidyl and tripeptidyl α-keto esters, α-keto amides, and α-keto acids having leucine in the P2 position were synthesized and evaluated as inhibitors for the cysteine proteases calpain I, calpain II, cathepsin B, and papain.  In general, peptidyl α-keto acids, were more inhibitory toward calpain I and II than α-keto amides, which in turn were more effective than α-keto esters.  In the series Z-Leu-AA-COOEt, the inhibitory potency decreased in the order: Met (lowest KI) > Nva > Phe > 4-Cl-Phe > Abu > Nle (highest KI) with calpain I, while almost the reverse order was obsd. for calpain II.  Extending the dipeptide α-keto ester to a tripeptide α-keto ester yielded significant enhancement in the inhibitory potency toward cathepsin B, but smaller changes toward the calpains.  Changing the ester group in the α-keto esters did not substantially decrease KI values for calpain I and calpain II.  N-monosubstituted α-keto amides were better inhibitors than the corresponding α-keto esters.  α-Keto amides with hydrophobic alkyl groups or alkyl groups with an attached Ph group had the lower KI values.  N,N-disubstituted α-keto amides were much less potent inhibitors than the corresponding N-monosubstituted peptide α-keto amides.  The peptide α-keto acid Z-Leu-Phe-COOH was the best inhibitor for calpain I (KI = 0.0085 μM) and calpain II (KI = 0.0057 μM) discovered in this study.  It is likely that the inhibitors are transition-state analogs and form tetrahedral adducts with the active site cysteine of cysteine proteases and form hydrogen bonds with the active site histidine and possibly another hydrogen bond donor in the case of monosubstituted amides.  Several inhibitors prevented spectrin degrdn. in a platelet membrane permeability assay and may be useful for the treatment of diseases which involve neurodegeneration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5VQ5w26OSKrVg90H21EOLACvtfcHk0lil6m83tNCErw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXmsVCitbo%253D&md5=f73638cf2385179be60d4d22ce091190</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm00074a031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00074a031%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DPatil%26aufirst%3DG.%2BS.%26aulast%3DGolubski%26aufirst%3DZ.%2BE.%26aulast%3DHori%26aufirst%3DH.%26aulast%3DTehrani%26aufirst%3DK.%26aulast%3DForeman%26aufirst%3DJ.%2BE.%26aulast%3DEveleth%26aufirst%3DD.%2BD.%26aulast%3DBartus%26aufirst%3DR.%2BT.%26aulast%3DPowers%26aufirst%3DJ.%2BC.%26atitle%3DPeptide%2520alpha-keto%2520ester%252C%2520alpha-keto%2520amide%252C%2520and%2520alpha-keto%2520acid%2520inhibitors%2520of%2520calpains%2520and%2520other%2520cysteine%2520proteases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1993%26volume%3D36%26spage%3D3472%26epage%3D3480%26doi%3D10.1021%2Fjm00074a031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Victor, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinz, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakulchik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q. M.</span></span> <span> </span><span class="NLM_article-title">Synthesis and evaluation of tripeptidyl alpha-ketoamides as human rhinovirus 3C protease inhibitors</span>. <i>Biorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">3531</span>– <span class="NLM_lpage">3536</span>, <span class="refDoi"> DOI: 10.1016/S0960-894X(03)00753-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1016%2FS0960-894X%2803%2900753-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=14505664" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnsVKgtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=3531-3536&author=S.-H.+Chenauthor=J.+Lamarauthor=F.+Victorauthor=N.+Snyderauthor=R.+Johnsonauthor=B.+A.+Heinzauthor=M.+Wakulchikauthor=Q.+M.+Wang&title=Synthesis+and+evaluation+of+tripeptidyl+alpha-ketoamides+as+human+rhinovirus+3C+protease+inhibitors&doi=10.1016%2FS0960-894X%2803%2900753-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and evaluation of tripeptidyl α-ketoamides as human rhinovirus 3C protease inhibitors</span></div><div class="casAuthors">Chen, Shu-Hui; Lamar, Jason; Victor, Frantz; Snyder, Nancy; Johnson, Robert; Heinz, Beverly A.; Wakulchik, Mark; Wang, Q. May</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3531-3536</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">We describe the synthesis and biol. evaluation of a series of tripeptidyl α-ketoamides as human rhinovirus (HRV) 3C protease inhibitors.  The most potent inhibitor Me2CHO2C-L-Val-L-Phe-L-NVal-COCHMePh-(S) exhibited impressive enzyme inhibitory activity as well as antiviral activity against HRV-14.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGYlHVZR46lrVg90H21EOLACvtfcHk0lhLLrC4JcJ8wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnsVKgtrs%253D&md5=ce59b2bcb3e78ffa458b3746386294d3</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2803%2900753-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252803%252900753-4%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.-H.%26aulast%3DLamar%26aufirst%3DJ.%26aulast%3DVictor%26aufirst%3DF.%26aulast%3DSnyder%26aufirst%3DN.%26aulast%3DJohnson%26aufirst%3DR.%26aulast%3DHeinz%26aufirst%3DB.%2BA.%26aulast%3DWakulchik%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DQ.%2BM.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520tripeptidyl%2520alpha-ketoamides%2520as%2520human%2520rhinovirus%25203C%2520protease%2520inhibitors%26jtitle%3DBiorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2003%26volume%3D13%26spage%3D3531%26epage%3D3536%26doi%3D10.1016%2FS0960-894X%2803%2900753-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otto, H.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirmeister, T.</span></span> <span> </span><span class="NLM_article-title">Cysteine proteases and their inhibitors</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">172</span>, <span class="refDoi"> DOI: 10.1021/cr950025u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr950025u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK2sXktFOhsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=1997&pages=133-172&author=H.-H.+Ottoauthor=T.+Schirmeister&title=Cysteine+proteases+and+their+inhibitors&doi=10.1021%2Fcr950025u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Cysteine Proteases and Their Inhibitors</span></div><div class="casAuthors">Otto, Hans-Hartwig; Schirmeister, Tanja</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">133-171</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with 599 refs.  Some typical examples of cysteine proteases from different sources are characterized in terms of property, structure and specificity.  Possible functions of human and mammalian cysteine proteases, lysosomal cathepsins and cytoplasmic calpains are discussed, as well as their role in pathol. processes.  In addn. to the endogenous inhibitors, the cystatins and calpastatins, the currently available synthetic inhibitors are also described together with their structures, reaction mechanisms, activities and selectivities.  Potential medical applications of cysteine protease inhibitors are discussed.  Particular emphasis is given to comparison of cysteine proteases with serine proteases, due to their similar proteolytic mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBgmD-cFH7IrVg90H21EOLACvtfcHk0lhLLrC4JcJ8wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXktFOhsA%253D%253D&md5=0c38c0f09f14ad664b6ca707ce08a5b9</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fcr950025u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr950025u%26sid%3Dliteratum%253Aachs%26aulast%3DOtto%26aufirst%3DH.-H.%26aulast%3DSchirmeister%26aufirst%3DT.%26atitle%3DCysteine%2520proteases%2520and%2520their%2520inhibitors%26jtitle%3DChem.%2520Rev.%26date%3D1997%26volume%3D97%26spage%3D133%26epage%3D172%26doi%3D10.1021%2Fcr950025u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gamcsik, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malthouse, J. P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Primrose, W. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackenzie, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, A. S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, A. I.</span></span> <span> </span><span class="NLM_article-title">Structure and stereochemistry of tetrahedral inhibitor complexes of papain by direct NMR observation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">6324</span>– <span class="NLM_lpage">6325</span>, <span class="refDoi"> DOI: 10.1021/ja00358a030</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00358a030" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaL3sXltlKjs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=1983&pages=6324-6325&author=M.+P.+Gamcsikauthor=J.+P.+G.+Malthouseauthor=W.+U.+Primroseauthor=N.+E.+Mackenzieauthor=A.+S.+F.+Boydauthor=R.+A.+Russellauthor=A.+I.+Scott&title=Structure+and+stereochemistry+of+tetrahedral+inhibitor+complexes+of+papain+by+direct+NMR+observation&doi=10.1021%2Fja00358a030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and stereochemistry of tetrahedral inhibitor complexes of papain by direct NMR observation</span></div><div class="casAuthors">Gamcsik, Michael P.; Malthouse, J. Paul G.; Primrose, William U.; Mackenzie, Neil E.; Boyd, Alan S. F.; Russell, Richard A.; Scott, A. Ian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6324-5</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">The site-specific inhibitor of papain, N-acetylphenylalanylglycinal, was synthesized with the aldehyde C atom enriched (90%) with 13C.  The resultant inhibitor complex had the structure of a tetrahedral hemithioacetal which, under high-resoln. 13C NMR observation, could be resolved into species corresponding to diastereoisomers of the acetal, showing that covalent complex formation proceeded with lack of stereospecificity at the active site of the enzyme.  Differentiation of the relative stabilities of the 2 isomers was possible by titrn. with 2,2'-dipyridyl disulfide and a tentative assignment of abs. stereochem. to each isomer could thus be made.  These expts. provide a novel probe for mapping the active site of cysteine (and hence serine) proteases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsRAF-1roQt7Vg90H21EOLACvtfcHk0lhLLrC4JcJ8wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXltlKjs78%253D&md5=1f13f3c27eb8027009369c694c9f0c1b</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fja00358a030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00358a030%26sid%3Dliteratum%253Aachs%26aulast%3DGamcsik%26aufirst%3DM.%2BP.%26aulast%3DMalthouse%26aufirst%3DJ.%2BP.%2BG.%26aulast%3DPrimrose%26aufirst%3DW.%2BU.%26aulast%3DMackenzie%26aufirst%3DN.%2BE.%26aulast%3DBoyd%26aufirst%3DA.%2BS.%2BF.%26aulast%3DRussell%26aufirst%3DR.%2BA.%26aulast%3DScott%26aufirst%3DA.%2BI.%26atitle%3DStructure%2520and%2520stereochemistry%2520of%2520tetrahedral%2520inhibitor%2520complexes%2520of%2520papain%2520by%2520direct%2520NMR%2520observation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1983%26volume%3D105%26spage%3D6324%26epage%3D6325%26doi%3D10.1021%2Fja00358a030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusov, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Brunn, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leyssen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanko, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neyts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Wilde, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snijder, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span> <span> </span><span class="NLM_article-title">α-Ketoamides as broad-spectrum inhibitors of coronavirus and enterovirus replication: Structure-based design, synthesis, and activity assessment</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">4562</span>– <span class="NLM_lpage">4578</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01828</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01828" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BB3cXivValt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=4562-4578&author=L.+Zhangauthor=D.+Linauthor=Y.+Kusovauthor=Y.+Nianauthor=Q.+Maauthor=J.+Wangauthor=A.+von+Brunnauthor=P.+Leyssenauthor=K.+Lankoauthor=J.+Neytsauthor=A.+de+Wildeauthor=E.+Snijderauthor=H.+Liuauthor=R.+Hilgenfeld&title=%CE%B1-Ketoamides+as+broad-spectrum+inhibitors+of+coronavirus+and+enterovirus+replication%3A+Structure-based+design%2C+synthesis%2C+and+activity+assessment&doi=10.1021%2Facs.jmedchem.9b01828"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">α-Ketoamides as Broad-Spectrum Inhibitors of Coronavirus and Enterovirus Replication: Structure-Based Design, Synthesis, and Activity Assessment</span></div><div class="casAuthors">Zhang, Linlin; Lin, Daizong; Kusov, Yuri; Nian, Yong; Ma, Qingjun; Wang, Jiang; von Brunn, Albrecht; Leyssen, Pieter; Lanko, Kristina; Neyts, Johan; de Wilde, Adriaan; Snijder, Eric J.; Liu, Hong; Hilgenfeld, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4562-4578</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The main protease of coronaviruses and the 3C protease of enteroviruses share a similar active-site architecture and a unique requirement for glutamine in the P1 position of the substrate.  Because of their unique specificity and essential role in viral polyprotein processing, these proteases are suitable targets for the development of antiviral drugs.  In order to obtain near-equipotent, broad-spectrum antivirals against alphacoronaviruses, betacoronaviruses, and enteroviruses, we pursued structure-based design of peptidomimetic α-ketoamides as inhibitors of main and 3C proteases.  Six crystal structures of protease:inhibitor complexes were detd. as part of this study.  Compds. synthesized were tested against the recombinant proteases as well as in viral replicons and virus-infected cell cultures; most of them were not cell-toxic.  Optimization of the P2 substituent of the α-ketoamides proved crucial for achieving near-equipotency against the three virus genera.  The best near-equipotent inhibitors, (I) [R1 = cyclopentylmethyl (P2) and R1 = cyclohexylmethyl (P2)], display low-micromolar EC50 values against enteroviruses, alphacoronaviruses, and betacoronaviruses in cell cultures.  In Huh7 cells, I [R1 = cyclohexylmethyl (P2)] exhibits three-digit picomolar activity against Middle East Respiratory Syndrome coronavirus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWpcfl4wTRLLVg90H21EOLACvtfcHk0lhtif5PLUHLQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXivValt7o%253D&md5=60d91a8ecf916856a844d835ecf84d5f</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01828&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01828%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DD.%26aulast%3DKusov%26aufirst%3DY.%26aulast%3DNian%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3Dvon%2BBrunn%26aufirst%3DA.%26aulast%3DLeyssen%26aufirst%3DP.%26aulast%3DLanko%26aufirst%3DK.%26aulast%3DNeyts%26aufirst%3DJ.%26aulast%3Dde%2BWilde%26aufirst%3DA.%26aulast%3DSnijder%26aufirst%3DE.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3D%25CE%25B1-Ketoamides%2520as%2520broad-spectrum%2520inhibitors%2520of%2520coronavirus%2520and%2520enterovirus%2520replication%253A%2520Structure-based%2520design%252C%2520synthesis%252C%2520and%2520activity%2520assessment%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D4562%26epage%3D4578%26doi%3D10.1021%2Facs.jmedchem.9b01828" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Powers, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asgian, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ekici, O. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, K. E.</span></span> <span> </span><span class="NLM_article-title">Irreversible inhibitors of serine, cysteine, and threonine proteases</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">4639</span>– <span class="NLM_lpage">4750</span>, <span class="refDoi"> DOI: 10.1021/cr010182v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr010182v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BD38XosFSnsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2002&pages=4639-4750&author=J.+C.+Powersauthor=J.+L.+Asgianauthor=O.+D.+Ekiciauthor=K.+E.+James&title=Irreversible+inhibitors+of+serine%2C+cysteine%2C+and+threonine+proteases&doi=10.1021%2Fcr010182v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Irreversible inhibitors of serine, cysteine, and threonine proteases</span></div><div class="casAuthors">Powers, James C.; Asgian, Juliana L.; Ekici, Oezlem Dogan; James, Karen Ellis</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4639-4750</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Topics discussed include alkylating, acylating, phosphonylating, and sulfonylating agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorivaxGhmhirVg90H21EOLACvtfcHk0lhtif5PLUHLQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XosFSnsr4%253D&md5=eb21ff0297b87537a7d459c92f13c75d</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fcr010182v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr010182v%26sid%3Dliteratum%253Aachs%26aulast%3DPowers%26aufirst%3DJ.%2BC.%26aulast%3DAsgian%26aufirst%3DJ.%2BL.%26aulast%3DEkici%26aufirst%3DO.%2BD.%26aulast%3DJames%26aufirst%3DK.%2BE.%26atitle%3DIrreversible%2520inhibitors%2520of%2520serine%252C%2520cysteine%252C%2520and%2520threonine%2520proteases%26jtitle%3DChem.%2520Rev.%26date%3D2002%26volume%3D102%26spage%3D4639%26epage%3D4750%26doi%3D10.1021%2Fcr010182v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copp, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coles, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pauls, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spencer, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heard, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krantz, A.</span></span> <span> </span><span class="NLM_article-title">New inhibitors of cysteine proteinases. Peptidyl acyloxymethyl ketones and the quiescent nucleofuge strategy</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">4429</span>– <span class="NLM_lpage">4431</span>, <span class="refDoi"> DOI: 10.1021/ja00221a062</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja00221a062" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaL1cXktlCguro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=1988&pages=4429-4431&author=R.+A.+Smithauthor=L.+J.+Coppauthor=P.+J.+Colesauthor=H.+W.+Paulsauthor=V.+J.+Robinsonauthor=R.+W.+Spencerauthor=S.+B.+Heardauthor=A.+Krantz&title=New+inhibitors+of+cysteine+proteinases.+Peptidyl+acyloxymethyl+ketones+and+the+quiescent+nucleofuge+strategy&doi=10.1021%2Fja00221a062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">New inhibitors of cysteine proteinases.  Peptidyl acyloxymethyl ketones and the quiescent nucleofuge strategy</span></div><div class="casAuthors">Smith, Roger A.; Copp, Leslie J.; Coles, Peter J.; Pauls, Henry W.; Robinson, Valerie J.; Spencer, Robin W.; Heard, Stephen B.; Krantz, Allen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4429-31</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Peptidyl acyloxymethyl ketones are potent and specific inhibitors of the cysteine proteinase, cathepsin B.  These inhibitors are affinity labels in which the peptide moiety serves to transport an aryl carboxylate nucleofuge that is uniquely reactive toward the active-site cysteine thiol of the target enzyme.  Heretofore, in constructing affinity labels the choice of leaving group was severely restricted.  The aryl carboxylate group offers considerable variation as a design element in that its binding affinity and reactivity can be controlled by substituent effects.  The apparent 2nd-order rate const. which characterizes the inhibition of bovine spleen cathepsin B by this series spans several orders of magnitude and in certain cases exceeds 106 M-1 s-1.  The inhibition is time-dependent, active-site directed, and irreversible.  The inactivation requires that the peptide component contain high affinity recognition elements for this enzyme and that the aryl carboxylate leaving group has a pKa <4.  Impressive inhibitory activity is also obsd. for certain peptidyl aryloxymethyl ketones.  NMR resonances of the inactivated adduct resulting from the action of representative 13C-labeled acyloxymethyl and aryloxymethyl ketone inhibitors on the model cysteine proteinase papain are consistent with the proposed mechanism of inactivation, involving displacement of the nucleofuge by the active-site cysteine to give a (cysteine-25)-thiomethyl ketone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0tvSCUqAffrVg90H21EOLACvtfcHk0lhtif5PLUHLQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXktlCguro%253D&md5=12b38defc0f9731ff6adeebac78b0711</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fja00221a062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja00221a062%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DR.%2BA.%26aulast%3DCopp%26aufirst%3DL.%2BJ.%26aulast%3DColes%26aufirst%3DP.%2BJ.%26aulast%3DPauls%26aufirst%3DH.%2BW.%26aulast%3DRobinson%26aufirst%3DV.%2BJ.%26aulast%3DSpencer%26aufirst%3DR.%2BW.%26aulast%3DHeard%26aufirst%3DS.%2BB.%26aulast%3DKrantz%26aufirst%3DA.%26atitle%3DNew%2520inhibitors%2520of%2520cysteine%2520proteinases.%2520Peptidyl%2520acyloxymethyl%2520ketones%2520and%2520the%2520quiescent%2520nucleofuge%2520strategy%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1988%26volume%3D110%26spage%3D4429%26epage%3D4431%26doi%3D10.1021%2Fja00221a062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dolle, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoyer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, C. V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helaszek, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ator, M. A.</span></span> <span> </span><span class="NLM_article-title">P1 aspartate-based peptide alpha-((2,6-dichlorobenzoyl)oxy)methyl ketones as potent time-dependent inhibitors of interleukin-1 beta-converting enzyme</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">563</span>– <span class="NLM_lpage">564</span>, <span class="refDoi"> DOI: 10.1021/jm00031a003</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00031a003" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK2cXhsFaqu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1994&pages=563-564&author=R.+E.+Dolleauthor=D.+Hoyerauthor=C.+V.+C.+Prasadauthor=S.+J.+Schmidtauthor=C.+T.+Helaszekauthor=R.+E.+Millerauthor=M.+A.+Ator&title=P1+aspartate-based+peptide+alpha-%28%282%2C6-dichlorobenzoyl%29oxy%29methyl+ketones+as+potent+time-dependent+inhibitors+of+interleukin-1+beta-converting+enzyme&doi=10.1021%2Fjm00031a003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">P1 Aspartate-Based Peptide α-((2,6-Dichlorobenzoyl)oxy)methyl Ketones as Potent Time-Dependent Inhibitors of Interleukin-1β-Converting Enzyme</span></div><div class="casAuthors">Dolle, Roland E.; Hoyer, Denton; Prasad, C. V. C.; Schmidt, Stanley J.; Helaszek, Carla T.; Miller, Robert E.; Ator, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">563-4</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">Peptide-based α-((2,6-dichlorobenzoyl)oxy)methyl ketones which contain a P1L-aspartic acid residue were synthesized and evaluated as time-dependent inhibitors of interleukin-1β-converting enzyme.  Structure-activity relations are discussed.  The ((arylacyl)oxy)methyl ketone displayed the highest selectivity for the enzyme vs. cathepsin B, whereas the more potent Me ketones demonstrated less preference for interleukin-1β-converting enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpi_XwuRN-QKLVg90H21EOLACvtfcHk0lggObpVEo68Rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXhsFaqu7s%253D&md5=007ff60fdfd21b80393083e93bfeb6fb</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm00031a003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00031a003%26sid%3Dliteratum%253Aachs%26aulast%3DDolle%26aufirst%3DR.%2BE.%26aulast%3DHoyer%26aufirst%3DD.%26aulast%3DPrasad%26aufirst%3DC.%2BV.%2BC.%26aulast%3DSchmidt%26aufirst%3DS.%2BJ.%26aulast%3DHelaszek%26aufirst%3DC.%2BT.%26aulast%3DMiller%26aufirst%3DR.%2BE.%26aulast%3DAtor%26aufirst%3DM.%2BA.%26atitle%3DP1%2520aspartate-based%2520peptide%2520alpha-%2528%25282%252C6-dichlorobenzoyl%2529oxy%2529methyl%2520ketones%2520as%2520potent%2520time-dependent%2520inhibitors%2520of%2520interleukin-1%2520beta-converting%2520enzyme%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1994%26volume%3D37%26spage%3D563%26epage%3D564%26doi%3D10.1021%2Fjm00031a003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brady, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giegel, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grinnell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lunney, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talanian, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, N.</span></span> <span> </span><span class="NLM_article-title">A catalytic mechanism for caspase-1 and for bimodal inhibition of caspase-1 by activated aspartic ketones</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">621</span>– <span class="NLM_lpage">631</span>, <span class="refDoi"> DOI: 10.1016/S0968-0896(99)00009-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1016%2FS0968-0896%2899%2900009-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10353641" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK1MXis12mt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1999&pages=621-631&author=K.+D.+Bradyauthor=D.+A.+Giegelauthor=C.+Grinnellauthor=E.+Lunneyauthor=R.+V.+Talanianauthor=W.+Wongauthor=N.+Walker&title=A+catalytic+mechanism+for+caspase-1+and+for+bimodal+inhibition+of+caspase-1+by+activated+aspartic+ketones&doi=10.1016%2FS0968-0896%2899%2900009-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">A catalytic mechanism for caspase-1 and for bimodal inhibition of caspase-1 by activated aspartic ketones</span></div><div class="casAuthors">Brady, Kenneth D.; Giegel, David A.; Grinnell, Christine; Lunney, Elizabeth; Talanian, Robert V.; Wong, Winnie; Walker, Nigel</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">621-631</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">We have evaluated 619 aspartic ketones with 9 different types of prime-side groups (acyloxymethyl, aryloxymethyl, arylthiomethyl, alkylthiomethyl, acylamino-oxymethyl, sulfonylaminomethyl, α-ketoamide, α-(1-phenyl-3-trifluoromethyl-pyrazol-5-yl)oxymethyl (PTP), and aliph. ketones) as inhibitors of caspase-1.  The inhibitory behaviors could be classified as reversible, inactivating, or bimodal (i.e. reversible inhibition followed by slow inactivation) based on the kinetically obsd. formation of reversible thiohemiketal complexes and conversion to an irreversible thioether adduct, and the mechanism of any given ketone was only poorly predictable on the basis of leaving group structure and chem.  Among 201 bimodal inhibitors, the rate of conversion of the reversible thiohemiketal complex to the inactive thioether (ki) was strictly first-order, consistent with direct conversion of the thiohemiketal to the thioether with no intermediate collapse to free ketone and thiolate.  We have examd. 22 crystallog. structures of caspase-1 complexed as a thiohemiketal with the inhibitors from 8 different ketone classes, and found the Cys285S-C-Cα'-leaving group dihedral angle to be near either to 60° or to 180°.  Only the 180° conformation was permissive for SN2 displacement of the leaving group and, furthermore, positioned His237Nδ to stabilize developing charge on the leaving group.  Among these structures and 19 addnl. complexes, all showed a strong interaction between His237Nδ and the ketone or thiohemiketal oxygen.  We therefore propose a proteolytic mechanism for caspase-1 involving polarization of the scissile carbonyl by the His237 imidazolium group.  During thiohemiketal/thioether conversion (but probably not during peptide hydrolysis), the leaving group is stabilized by the His237 imidazolium.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGN4AAx-wiarVg90H21EOLACvtfcHk0lggObpVEo68Rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXis12mt7Y%253D&md5=61c58cf618b855f3573cb3669a7e9224</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2FS0968-0896%2899%2900009-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0968-0896%252899%252900009-7%26sid%3Dliteratum%253Aachs%26aulast%3DBrady%26aufirst%3DK.%2BD.%26aulast%3DGiegel%26aufirst%3DD.%2BA.%26aulast%3DGrinnell%26aufirst%3DC.%26aulast%3DLunney%26aufirst%3DE.%26aulast%3DTalanian%26aufirst%3DR.%2BV.%26aulast%3DWong%26aufirst%3DW.%26aulast%3DWalker%26aufirst%3DN.%26atitle%3DA%2520catalytic%2520mechanism%2520for%2520caspase-1%2520and%2520for%2520bimodal%2520inhibition%2520of%2520caspase-1%2520by%2520activated%2520aspartic%2520ketones%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D1999%26volume%3D7%26spage%3D621%26epage%3D631%26doi%3D10.1016%2FS0968-0896%2899%2900009-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anand, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziebuhr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wadhwani, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesters, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgenfeld, R.</span></span> <span> </span><span class="NLM_article-title">Coronavirus main proteinase (3CL<sup>pro</sup>) structure: basis for design of anti-SARS drugs</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>300</i></span>,  <span class="NLM_fpage">1763</span>– <span class="NLM_lpage">1767</span>, <span class="refDoi"> DOI: 10.1126/science.1085658</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1126%2Fscience.1085658" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=12746549" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BD3sXksVKisLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=300&publication_year=2003&pages=1763-1767&author=K.+Anandauthor=J.+Ziebuhrauthor=P.+Wadhwaniauthor=J.+R.+Mestersauthor=R.+Hilgenfeld&title=Coronavirus+main+proteinase+%283CLpro%29+structure%3A+basis+for+design+of+anti-SARS+drugs&doi=10.1126%2Fscience.1085658"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Coronavirus Main Proteinase (3CLpro) Structure: Basis for Design of Anti-SARS Drugs</span></div><div class="casAuthors">Anand, Kanchan; Ziebuhr, John; Wadhwani, Parvesh; Mesters, Jeroen R.; Hilgenfeld, Rolf</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">300</span>
        (<span class="NLM_cas:issue">5626</span>),
    <span class="NLM_cas:pages">1763-1767</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A novel coronavirus has been identified as the causative agent of severe acute respiratory syndrome (SARS).  The viral main proteinase (Mpro, also called 3CLpro), which controls the activities of the coronavirus replication complex, is an attractive target for therapy.  The authors detd. crystal structures for human coronavirus (strain 229E) Mpro and for an inhibitor complex of porcine coronavirus [transmissible gastroenteritis virus (TGEV)] Mpro, and the authors constructed a homol. model for SARS coronavirus (SARS-CoV) Mpro.  The structures reveal a remarkable degree of conservation of the substrate-binding sites, which is further supported by recombinant SARS-CoV Mpro-mediated cleavage of a TGEV Mpro substrate.  Mol. modeling suggests that available rhinovirus 3Cpro inhibitors may be modified to make them useful for treating SARS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEmONTL1SzDrVg90H21EOLACvtfcHk0lggObpVEo68Rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXksVKisLk%253D&md5=7a8d8d39a783cb3960f6f22931f35569</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1126%2Fscience.1085658&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1085658%26sid%3Dliteratum%253Aachs%26aulast%3DAnand%26aufirst%3DK.%26aulast%3DZiebuhr%26aufirst%3DJ.%26aulast%3DWadhwani%26aufirst%3DP.%26aulast%3DMesters%26aufirst%3DJ.%2BR.%26aulast%3DHilgenfeld%26aufirst%3DR.%26atitle%3DCoronavirus%2520main%2520proteinase%2520%25283CLpro%2529%2520structure%253A%2520basis%2520for%2520design%2520of%2520anti-SARS%2520drugs%26jtitle%3DScience%26date%3D2003%26volume%3D300%26spage%3D1763%26epage%3D1767%26doi%3D10.1126%2Fscience.1085658" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span> </span><span class="NLM_article-title">The X-ray crystallographic coordinates for the SARS CoV-1 3CL<sup>pro</sup> covalently bound with inhibitor <b>2</b> is deposited
in the Protein Data Bank (PDB)</span>, <a href="http://rcsb.rutgers.edu/pdb/index" class="extLink">http://rcsb.rutgers.edu/pdb/index</a> (PDB code 6XHO).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=The+X-ray+crystallographic+coordinates+for+the+SARS+CoV-1+3CLpro+covalently+bound+with+inhibitor+2+is+deposited%0Ain+the+Protein+Data+Bank+%28PDB%29%2C+http%3A%2F%2Frcsb.rutgers.edu%2Fpdb%2Findex+%28PDB+code+6XHO%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DThe%2520X-ray%2520crystallographic%2520coordinates%2520for%2520the%2520SARS%2520CoV-1%25203CLpro%2520covalently%2520bound%2520with%2520inhibitor%25202%2520is%2520deposited%250Ain%2520the%2520Protein%2520Data%2520Bank%2520%2528PDB%2529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prins, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webber, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marakovits, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuhrman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patick, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathews, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rejto, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hendrickson, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuntland, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meador, J. W.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferre, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harr, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kosa, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Worland, S. L.</span></span> <span> </span><span class="NLM_article-title">Structure-based design, synthesis, and biological evaluation of irreversible human rhinovirus 3C protease inhibitors. 4. Incorporation of P1 lactam moieties as L-glutamine replacements</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">1213</span>, <span class="refDoi"> DOI: 10.1021/jm9805384</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm9805384" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK1MXhvVeisLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=1213&author=P.+S.+Dragovichauthor=T.+J.+Prinsauthor=R.+Zhouauthor=S.+E.+Webberauthor=J.+T.+Marakovitsauthor=S.+A.+Fuhrmanauthor=A.+K.+Patickauthor=D.+A.+Mathewsauthor=C.+A.+Leeauthor=C.+E.+Fordauthor=B.+J.+Burkeauthor=P.+A.+Rejtoauthor=T.+F.+Hendricksonauthor=T.+Tuntlandauthor=E.+L.+Brownauthor=J.+W.+Meadorauthor=R.+A.+Ferreauthor=J.+E.+Harrauthor=M.+B.+Kosaauthor=S.+L.+Worland&title=Structure-based+design%2C+synthesis%2C+and+biological+evaluation+of+irreversible+human+rhinovirus+3C+protease+inhibitors.+4.+Incorporation+of+P1+lactam+moieties+as+L-glutamine+replacements&doi=10.1021%2Fjm9805384"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design, Synthesis, and Biological Evaluation of Irreversible Human Rhinovirus 3C Protease Inhibitors. 4. Incorporation of P1 Lactam Moieties as L-Glutamine Replacements</span></div><div class="casAuthors">Dragovich, Peter S.; Prins, Thomas J.; Zhou, Ru; Webber, Stephen E.; Marakovits, Joseph T.; Fuhrman, Shella A.; Patick, Amy K.; Matthews, David A.; Lee, Caroline A.; Ford, Clifford E.; Burke, Benjamin J.; Rejto, Paul A.; Hendrickson, Thomas F.; Tuntland, Tove; Brown, Edward L.; Meador, James W., III; Ferre, Rose Ann; Harr, James E. V.; Kosa, Maha B.; Worland, Stephen T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1213-1224</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The structure-based design, chem. synthesis, and biol. evaluation of various human rhinovirus (HRV) 3C protease (3CP) inhibitors, e.g. I [R = CH2CHMe2, CHMe2; X = (S)-CH, (R)-CH, N; n = 1, 2] and II (R = CHMe2, CMe3; Y = NH, CH2) which incorporate P1 lactam moieties in lieu of an L-glutamine residue are described.  These compds. are comprised of a tripeptidyl or peptidomimetic binding determinant and an Et propenoate Michael acceptor moiety which forms an irreversible covalent adduct with the active site cysteine residue of the 3C enzyme.  The P1-lactam-contg. inhibitors display significantly increased 3CP inhibition activity along with improved antirhinoviral properties relative to corresponding L-glutamine-derived mols.  In addn., several lactam-contg. compds. exhibit excellent selectivity for HRV 3CP over several other serine and cysteine proteases and are not appreciably degraded by a variety of biol. agents.  Lactam II (R = CHMe2, Y = CH2, n = 1) (AG7088) was one of the most potent inhibitors (mean antirhinoviral EC90 ≈ 0.10 μM, n = 46 serotypes), and is shown to warrant addnl. preclin. development to explore its potential for use as an antirhinoviral agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlBuxFPwMr97Vg90H21EOLACvtfcHk0lh-nt9skZ7ZFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXhvVeisLk%253D&md5=0e2bf5a03d17ea371663a252e7b97b20</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm9805384&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9805384%26sid%3Dliteratum%253Aachs%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DPrins%26aufirst%3DT.%2BJ.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DWebber%26aufirst%3DS.%2BE.%26aulast%3DMarakovits%26aufirst%3DJ.%2BT.%26aulast%3DFuhrman%26aufirst%3DS.%2BA.%26aulast%3DPatick%26aufirst%3DA.%2BK.%26aulast%3DMathews%26aufirst%3DD.%2BA.%26aulast%3DLee%26aufirst%3DC.%2BA.%26aulast%3DFord%26aufirst%3DC.%2BE.%26aulast%3DBurke%26aufirst%3DB.%2BJ.%26aulast%3DRejto%26aufirst%3DP.%2BA.%26aulast%3DHendrickson%26aufirst%3DT.%2BF.%26aulast%3DTuntland%26aufirst%3DT.%26aulast%3DBrown%26aufirst%3DE.%2BL.%26aulast%3DMeador%26aufirst%3DJ.%2BW.%26aulast%3DFerre%26aufirst%3DR.%2BA.%26aulast%3DHarr%26aufirst%3DJ.%2BE.%26aulast%3DKosa%26aufirst%3DM.%2BB.%26aulast%3DWorland%26aufirst%3DS.%2BL.%26atitle%3DStructure-based%2520design%252C%2520synthesis%252C%2520and%2520biological%2520evaluation%2520of%2520irreversible%2520human%2520rhinovirus%25203C%2520protease%2520inhibitors.%25204.%2520Incorporation%2520of%2520P1%2520lactam%2520moieties%2520as%2520L-glutamine%2520replacements%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D1213%26doi%3D10.1021%2Fjm9805384" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rich, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prasad, J. V. N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houseman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKenzie, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malkovsky, M.</span></span> <span> </span><span class="NLM_article-title">New hydroxyethylamine HIV protease inhibitors that suppress viral replication</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">3803</span>– <span class="NLM_lpage">3812</span>, <span class="refDoi"> DOI: 10.1021/jm00099a008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm00099a008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK38XlvFags7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=1992&pages=3803-3812&author=D.+H.+Richauthor=J.+V.+N.+V.+Prasadauthor=C.-Q.+Sunauthor=J.+Greenauthor=R.+Muellerauthor=K.+Housemanauthor=D.+MacKenzieauthor=M.+Malkovsky&title=New+hydroxyethylamine+HIV+protease+inhibitors+that+suppress+viral+replication&doi=10.1021%2Fjm00099a008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">New hydroxyethylamine HIV protease inhibitors that suppress viral replication</span></div><div class="casAuthors">Rich, Daniel H.; Prasad, J. V. N. Vara; Sun, Chong Qing; Green, Jeremy; Mueller, Richard; Houseman, Kathryn; MacKenzie, Debra; Malkovsky, Miroslav</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">3803-12</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    </div><div class="casAbstract">The synthesis of analogs of Ac-Ser-Leu-Asn-[Phe-HEA-Pro]-Ile-Val-OMe (JG-365; Phe-HEA-Pro = hydroxyethylamine transition state analog I), a tight-binding inhibitor of human immunodeficiency virus protease (HIVP), are reported.  Systematic modification of the P3 and P3' regions of the inhibitors has led to smaller HIVP inhibitors that inhibit viral replication in HIV-infected and simian immunodeficiency virus (SIV)-infected cell cultures.  Six aliph. and arom. derivs. were prepd. by replacing residues in the P3 regions of Boc-Leu-Asn-[Phe-HEA-Pro]-Ile-Val-OMe (Boc = Me3CO2C).  Arom. side chains at P3 gave better inhibitors than aliph. side chains.  The better inhibitors in this series contained a β-naphthylalanine or a biphenyl unit at P3.  A second series of HIVP inhibitors were obtained by converting the P3 group into acyl groups.  R-Asn-[Phe-HEA-Pro]-Ile-Phe-OMe (R = PhCH2O2C, 2-quinolinylcarbonyl) are potent HIVP inhibitors with Ki values equal to 1.0 and 0.1 nM, resp.  The inhibition consts. were det. by using the continuous fluorometric assay developed by M. V. Toth and G. R. Marshall (1990).  The activities of the protease inhibitors for inhibition of SIV replication were detd. in vitro using CEMx174 cells.  Inhibition of HIV infection was detd. essentially as reported by R. Pauwels, et. al. (1988).  The anti-HIV assay was carried out in culture using CEM cells (a CD4+ lymphocyte line) infected with virus strain HTLV-IIIb with a multiplicity of infection of 0.1.  Several analogs inhibited the cytopathic effect at concns. of 0.1-0.8 μg/mL.  These results establish that good inhibitors of HIV protease that inhibit viral replication in infected lymphocytes in in vitro cell assays can be obtained from JG-365 when the Ac-Ser-Leu unit is replaced by arom. acyl derivs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFgN6BYEYXb7Vg90H21EOLACvtfcHk0lh-nt9skZ7ZFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XlvFags7Y%253D&md5=4fd3643c608b8ebdff13563757868d23</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm00099a008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm00099a008%26sid%3Dliteratum%253Aachs%26aulast%3DRich%26aufirst%3DD.%2BH.%26aulast%3DPrasad%26aufirst%3DJ.%2BV.%2BN.%2BV.%26aulast%3DSun%26aufirst%3DC.-Q.%26aulast%3DGreen%26aufirst%3DJ.%26aulast%3DMueller%26aufirst%3DR.%26aulast%3DHouseman%26aufirst%3DK.%26aulast%3DMacKenzie%26aufirst%3DD.%26aulast%3DMalkovsky%26aufirst%3DM.%26atitle%3DNew%2520hydroxyethylamine%2520HIV%2520protease%2520inhibitors%2520that%2520suppress%2520viral%2520replication%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1992%26volume%3D35%26spage%3D3803%26epage%3D3812%26doi%3D10.1021%2Fjm00099a008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamashita, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brook, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomaszek, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczuk, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tew, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veber, D. F.</span></span> <span> </span><span class="NLM_article-title">Solid-phase synthesis of a combinatorial array of 1,3-bis(acylamino)-2-butanones, inhibitors of the cysteine proteases cathepsins K and L</span>. <i>J. Comb. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">207</span>– <span class="NLM_lpage">215</span>, <span class="refDoi"> DOI: 10.1021/cc9800374</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cc9800374" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK1MXisFSqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=1999&pages=207-215&author=D.+S.+Yamashitaauthor=X.+Dongauthor=H.-J.+Ohauthor=C.+S.+Brookauthor=T.+A.+Tomaszekauthor=L.+Szewczukauthor=D.+G.+Tewauthor=D.+F.+Veber&title=Solid-phase+synthesis+of+a+combinatorial+array+of+1%2C3-bis%28acylamino%29-2-butanones%2C+inhibitors+of+the+cysteine+proteases+cathepsins+K+and+L&doi=10.1021%2Fcc9800374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Solid-Phase Synthesis of a Combinatorial Array of 1,3-Bis(acylamino)-2-butanones, Inhibitors of the Cysteine Proteases Cathepsins K and L</span></div><div class="casAuthors">Yamashita, Dennis S.; Dong, Xiaoyang; Oh, Hye-Ja; Brook, Christopher S.; Tomaszek, Thaddeus A.; Szewczuk, Lawrence; Tew, David G.; Veber, Daniel F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Combinatorial Chemistry</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">207-215</span>CODEN:
                <span class="NLM_cas:coden">JCCHFF</span>;
        ISSN:<span class="NLM_cas:issn">1520-4766</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">To more rapidly prep. members of the 1,3-bis(acylamino)-2-butanone class of cysteine protease inhibitors, a solid-phase synthesis was developed.  1-Azido-3-amino-2,2-dimethoxybutane, which has two amino groups differentiated and the ketone protected as a ketal, served as a surrogate for the 1,3-diamino-2-butanone core.  Thus, 1-azido-3-amino-2,2-dimethoxybutane was coupled to the BAL-resin-linked carboxylic acids derived from α-amino acid esters.  Evaluation of a small combinatorial array by measuring inhibition consts. (Ki,apps) against cathepsins K, L, and B provided some structure-activity relationship trends with respect to selectivity and potency. Novel, potent inhibitors of cathepsins K and L were identified.  Novel, potent inhibitors of cathepsins K and L were identified.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5W8F450f8KrVg90H21EOLACvtfcHk0lh-nt9skZ7ZFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXisFSqsLk%253D&md5=96a8b824edfbe519b2afac34c3d9262d</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fcc9800374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcc9800374%26sid%3Dliteratum%253Aachs%26aulast%3DYamashita%26aufirst%3DD.%2BS.%26aulast%3DDong%26aufirst%3DX.%26aulast%3DOh%26aufirst%3DH.-J.%26aulast%3DBrook%26aufirst%3DC.%2BS.%26aulast%3DTomaszek%26aufirst%3DT.%2BA.%26aulast%3DSzewczuk%26aufirst%3DL.%26aulast%3DTew%26aufirst%3DD.%2BG.%26aulast%3DVeber%26aufirst%3DD.%2BF.%26atitle%3DSolid-phase%2520synthesis%2520of%2520a%2520combinatorial%2520array%2520of%25201%252C3-bis%2528acylamino%2529-2-butanones%252C%2520inhibitors%2520of%2520the%2520cysteine%2520proteases%2520cathepsins%2520K%2520and%2520L%26jtitle%3DJ.%2520Comb.%2520Chem.%26date%3D1999%26volume%3D1%26spage%3D207%26epage%3D215%26doi%3D10.1021%2Fcc9800374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nayyar, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tibbetts, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, T. P.</span></span> <span> </span><span class="NLM_article-title">An efficient synthesis of a key intermediate for the preparation of the rhinovirus protease inhibitor AG7088 via asymmetric dianionic cyanomethylation of N-Boc-L-(+)-glutamic acid dimethyl ester</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">6807</span>– <span class="NLM_lpage">6809</span>, <span class="refDoi"> DOI: 10.1016/S0040-4039(01)01416-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1016%2FS0040-4039%2801%2901416-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmslCms7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2001&pages=6807-6809&author=Q.+Tianauthor=N.+K.+Nayyarauthor=S.+Babuauthor=L.+Chenauthor=J.+Taoauthor=S.+Leeauthor=A.+Tibbettsauthor=T.+Moranauthor=J.+Liouauthor=M.+Guoauthor=T.+P.+Kennedy&title=An+efficient+synthesis+of+a+key+intermediate+for+the+preparation+of+the+rhinovirus+protease+inhibitor+AG7088+via+asymmetric+dianionic+cyanomethylation+of+N-Boc-L-%28%2B%29-glutamic+acid+dimethyl+ester&doi=10.1016%2FS0040-4039%2801%2901416-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">An efficient synthesis of a key intermediate for the preparation of the rhinovirus protease inhibitor AG7088 via asymmetric dianionic cyanomethylation of N-Boc-L-(+)-glutamic acid dimethyl ester</span></div><div class="casAuthors">Tian, Q.; Nayyar, N. K.; Babu, S.; Chen, L.; Tao, J.; Lee, S.; Tibbetts, A.; Moran, T.; Liou, J.; Guo, M.; Kennedy, T. P.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">6807-6809</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">An efficient synthetic route to a key intermediate [(2E,4S)-4-(N-Boc-amino)-5-[(3S)-2-oxo-3-pyrrolidinyl]-2-pentenoic acid Et ester (Boc = Me3CO2C)] for the prepn. of the rhinovirus protease inhibitor AG7088 has been developed employing a key asym. dianionic cyanomethylation of N-Boc-L-(+)-glutamic acid di-Me ester.  This methodol. enables the prepn. of this compd. in kilogram quantities with an overall yield of 30%.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfwuKJVB5h-bVg90H21EOLACvtfcHk0li1z8ag0TMfgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmslCms7w%253D&md5=ffb96a4ab738a521993ce0551d9d4223</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2FS0040-4039%2801%2901416-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4039%252801%252901416-2%26sid%3Dliteratum%253Aachs%26aulast%3DTian%26aufirst%3DQ.%26aulast%3DNayyar%26aufirst%3DN.%2BK.%26aulast%3DBabu%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DTao%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DTibbetts%26aufirst%3DA.%26aulast%3DMoran%26aufirst%3DT.%26aulast%3DLiou%26aufirst%3DJ.%26aulast%3DGuo%26aufirst%3DM.%26aulast%3DKennedy%26aufirst%3DT.%2BP.%26atitle%3DAn%2520efficient%2520synthesis%2520of%2520a%2520key%2520intermediate%2520for%2520the%2520preparation%2520of%2520the%2520rhinovirus%2520protease%2520inhibitor%2520AG7088%2520via%2520asymmetric%2520dianionic%2520cyanomethylation%2520of%2520N-Boc-L-%2528%252B%2529-glutamic%2520acid%2520dimethyl%2520ester%26jtitle%3DTetrahedron%2520Lett.%26date%3D2001%26volume%3D42%26spage%3D6807%26epage%3D6809%26doi%3D10.1016%2FS0040-4039%2801%2901416-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="note"><p class="first last">Generation of diazomethane followed Aldrich technical bulletin AL-180 employing Diazald kit.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mendonca, R. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkatraman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, J. T.</span></span> <span> </span><span class="NLM_article-title">Novel route to the synthesis of peptides containing 2-amino-1′-hydroxymethyl ketones and their application as cathepsin K inhibitors</span>. <i>Biorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2887</span>– <span class="NLM_lpage">2891</span>, <span class="refDoi"> DOI: 10.1016/S0960-894X(02)00611-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1016%2FS0960-894X%2802%2900611-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=12270170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BD38Xntlagu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2002&pages=2887-2891&author=R.+V.+Mendoncaauthor=S.+Venkatramanauthor=J.+T.+Palmer&title=Novel+route+to+the+synthesis+of+peptides+containing+2-amino-1%E2%80%B2-hydroxymethyl+ketones+and+their+application+as+cathepsin+K+inhibitors&doi=10.1016%2FS0960-894X%2802%2900611-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Novel route to the synthesis of peptides containing 2-amino-1'-hydroxymethyl ketones and their application as cathepsin K inhibitors</span></div><div class="casAuthors">Mendonca, Rohan V.; Venkatraman, Shankar; Palmer, James T.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">2887-2891</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">Cathepsin K is highly expressed in human osteoclasts and is implicated in bone resorption.  This makes it an attractive target for the treatment of osteoporosis.  Peptides contg. 2-amino-1'-hydroxymethyl ketones and 2-amino-1'-alkoxymethyl ketones were discovered as potent inhibitors of cathepsin K.  A novel synthetic route was devised to facilitate rapid elucidation of the SAR of these inhibitors.  The synthesis and SAR of hydroxymethyl ketones are presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBSBV5awQSv7Vg90H21EOLACvtfcHk0li1z8ag0TMfgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xntlagu70%253D&md5=318ea46f2a2aee28f8daeacd808d492e</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2802%2900611-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252802%252900611-X%26sid%3Dliteratum%253Aachs%26aulast%3DMendonca%26aufirst%3DR.%2BV.%26aulast%3DVenkatraman%26aufirst%3DS.%26aulast%3DPalmer%26aufirst%3DJ.%2BT.%26atitle%3DNovel%2520route%2520to%2520the%2520synthesis%2520of%2520peptides%2520containing%25202-amino-1%25E2%2580%25B2-hydroxymethyl%2520ketones%2520and%2520their%2520application%2520as%2520cathepsin%2520K%2520inhibitors%26jtitle%3DBiorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2002%26volume%3D12%26spage%3D2887%26epage%3D2891%26doi%3D10.1016%2FS0960-894X%2802%2900611-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="note"><p class="first last">These conditions result in detectable levels of epimerization at the P<sub>1</sub> site. The resulting diastereomer has been observed by HPLC, NMR, and LCMS and is readily removed by fractional recrystallization.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krantz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copp, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coles, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heard, S. B.</span></span> <span> </span><span class="NLM_article-title">Peptidyl (acyloxy)methyl ketones and the quiescent affinity label concept: the departing group as a variable structural element in the design of inactivators of cysteine proteinases</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">4678</span>– <span class="NLM_lpage">4687</span>, <span class="refDoi"> DOI: 10.1021/bi00233a007</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00233a007" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK3MXit1CgsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1991&pages=4678-4687&author=A.+Krantzauthor=L.+J.+Coppauthor=P.+J.+Colesauthor=R.+A.+Smithauthor=S.+B.+Heard&title=Peptidyl+%28acyloxy%29methyl+ketones+and+the+quiescent+affinity+label+concept%3A+the+departing+group+as+a+variable+structural+element+in+the+design+of+inactivators+of+cysteine+proteinases&doi=10.1021%2Fbi00233a007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Peptidyl (acyloxy)methyl ketones and the quiescent affinity label concept:  the departing group as a variable structural element in the design of inactivators of cysteine proteinases</span></div><div class="casAuthors">Krantz, Allen; Copp, Leslie J.; Coles, Peter J.; Smith, Roger A.; Heard, Stephen B.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">4678-87</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">(Acyloxy)methyl ketones, of general structure Z-[AA2]-[AA1]-CH2OCOAr (Z = benzyloxycarbonyl; AA = amino acid; Ar = aryl), were potent inactivators of the cysteine proteinase, cathepsin B.  These reagents were designed as affinity labels in which the dipeptidyl moiety serves as an affinity group (complementary to the S1 and S2 sites of the enzyme), while the (acyloxy)methyl ketone unit (-COCH2OCOR), contg. a weak leaving group in the form of a carboxylate nucleofuge, functions as the potentially reactive entity that labels the enzyme.  The inhibition was time-dependent, active-site directed, and irreversible.  The apparent 2nd-order rate const. (kinact/Kinact), which characterizes the inhibition of cathepsin B by this series, spanned several orders of magnitude and in certain cases exceeded 106 M-1 s-1.  The activity of this series of inhibitors was found to be exquisitely sensitive to the nature of the carboxylate leaving group as well as the affinity group.  A strong dependence of 2nd-order inactivation rate on leaving group pKa was uncovered for Z-Phe-Ala (acyloxy)methyl ketones [log(k/K) = -1.1 × pKa + 7.2.  Heretofore, in constructing affinity labels the choice of leaving group was quite restricted.  The aryl carboxylate group thus offers considerable variation as a design element in that both its binding affinity and reactivity can be controlled by substituent effects.  Specific peptidyl (acyloxy)methyl ketones thus represent prime examples of highly potent, chem. stable enzyme inhibitors with variable structural elements in both the affinity and departing groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprQo4CgIuJM7Vg90H21EOLACvtfcHk0li1z8ag0TMfgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXit1CgsLY%253D&md5=887882366c35b5cba8021d1d14eb47be</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fbi00233a007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00233a007%26sid%3Dliteratum%253Aachs%26aulast%3DKrantz%26aufirst%3DA.%26aulast%3DCopp%26aufirst%3DL.%2BJ.%26aulast%3DColes%26aufirst%3DP.%2BJ.%26aulast%3DSmith%26aufirst%3DR.%2BA.%26aulast%3DHeard%26aufirst%3DS.%2BB.%26atitle%3DPeptidyl%2520%2528acyloxy%2529methyl%2520ketones%2520and%2520the%2520quiescent%2520affinity%2520label%2520concept%253A%2520the%2520departing%2520group%2520as%2520a%2520variable%2520structural%2520element%2520in%2520the%2520design%2520of%2520inactivators%2520of%2520cysteine%2520proteinases%26jtitle%3DBiochemistry%26date%3D1991%26volume%3D30%26spage%3D4678%26epage%3D4687%26doi%3D10.1021%2Fbi00233a007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span> </span><span class="NLM_article-title">The X-ray crystallographic coordinates for the SARS
CoV-1 and SARS
CoV-2 3CL<sup>pro</sup> each covalently bound inhibitor 28 are deposited
in the Protein Data Bank (PDB)</span>, <a href="http://rcsb.rutgers.edu/pdb/index" class="extLink">http://rcsb.rutgers.edu/pdb/index</a> (PDB Code 6XHN).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=The+X-ray+crystallographic+coordinates+for+the+SARS%0ACoV-1+and+SARS%0ACoV-2+3CLpro+each+covalently+bound+inhibitor+28+are+deposited%0Ain+the+Protein+Data+Bank+%28PDB%29%2C+http%3A%2F%2Frcsb.rutgers.edu%2Fpdb%2Findex+%28PDB+Code+6XHN%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DThe%2520X-ray%2520crystallographic%2520coordinates%2520for%2520the%2520SARS%250ACoV-1%2520and%2520SARS%250ACoV-2%25203CLpro%2520each%2520covalently%2520bound%2520inhibitor%252028%2520are%2520deposited%250Ain%2520the%2520Protein%2520Data%2520Bank%2520%2528PDB%2529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, R. L.</span>; <span class="NLM_string-name">Kania, R. S.</span>; <span class="NLM_string-name">Nieman, J. A.</span>; <span class="NLM_string-name">Planken, S. P.</span>; <span class="NLM_string-name">Smith, G. J.,  III.</span></span> <span> </span><span class="NLM_article-title">SARS antiviral data for all described inhibitors are disclosed</span>. Preparation of peptides as anticoronaviral agents. <span class="NLM_patent">WO2005/113580 A1</span>, Dec 1, <span class="NLM_year">2005</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&author=R.+L.+Hoffman&author=R.+S.+Kania&author=J.+A.+Nieman&author=S.+P.+Planken&author=G.+J.+Smith&title=SARS+antiviral+data+for+all+described+inhibitors+are+disclosed"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DHoffman%26aufirst%3DR.%2BL.%26atitle%3DSARS%2520antiviral%2520data%2520for%2520all%2520described%2520inhibitors%2520are%2520disclosed%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="note"><p class="first last">Compound is incubated at 1 μM substrate concentration and 5 mM GSH in pH 7.8 0.1 M TRIS buffer at 37 °C. Timepoints are taken over 30 min and terminated with 250 mM iodoacetamide to quench the excess GSH. Samples were then diluted in acetonitrile for LC/MS analysis. Data are reported as half-life.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="note"><p class="first last">Compound is spiked at 1 μM substrate concentration in pH 7.4 adjusted (with HCL or NaOH) plasma and incubated at 37 °C. 50 μL plasma samples were taken at multiple timepoints during a 60 minute incubation and added to 200 μL acetonitrile to stop enzymatic degradation. Samples were centrifuged at 3000 rpm and 150 μL supernatant was transferred to a 96 well block for analysis by LC/MS. Data are reported as half-life.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="note"><p class="first last">The solubility of <b>3</b> is 0.377 mg/mL (2.5% w/v SBECD, 5 mM phosphate buffer, pH 7.4, made isotonic with 4.25% dextrose). Rat <i>in vivo</i> parameters for <b>3</b> were from IV dosing used a 50% PEG 200/10% EtOH/40% saline formulation. For 2 mg/kg dose the following values were obtained: 26.2 mL/min/kg CL; 1.0 L/kg <i>V</i><sub>dss</sub>; 0.9 <i>t</i><sub>1/2</sub>; and 1.3 h·μg/mL AUC.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brinen, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellman, J. A.</span></span> <span> </span><span class="NLM_article-title">Crystal structures of reversible ketone-Based inhibitors of the cysteine protease cruzain</span>. <i>Biorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">21</span>– <span class="NLM_lpage">29</span>, <span class="refDoi"> DOI: 10.1016/S0968-0896(02)00427-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1016%2FS0968-0896%2802%2900427-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=12467703" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BD38XptFGltrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2003&pages=21-29&author=L.+Huangauthor=L.+S.+Brinenauthor=J.+A.+Ellman&title=Crystal+structures+of+reversible+ketone-Based+inhibitors+of+the+cysteine+protease+cruzain&doi=10.1016%2FS0968-0896%2802%2900427-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of reversible ketone-based inhibitors of the cysteine protease cruzain</span></div><div class="casAuthors">Huang, Lily; Brinen, Linda S.; Ellman, Jonathan A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">21-29</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">The crystal structures of two hydroxymethyl ketone inhibitors complexed to the cysteine protease cruzain have been detd. at 1.1 and 1.2 A resoln., resp.  These high resoln. crystal structures provide the first structures of non-covalent inhibitors bound to cruzain.  A series of compds. were prepd. and tested based upon the structures providing further insight into the key binding interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodxjvcaYN6GLVg90H21EOLACvtfcHk0lhwEc0tdrPxmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XptFGltrw%253D&md5=a96042e8c37352c548f90082bfe6ee7a</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2FS0968-0896%2802%2900427-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0968-0896%252802%252900427-3%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DBrinen%26aufirst%3DL.%2BS.%26aulast%3DEllman%26aufirst%3DJ.%2BA.%26atitle%3DCrystal%2520structures%2520of%2520reversible%2520ketone-Based%2520inhibitors%2520of%2520the%2520cysteine%2520protease%2520cruzain%26jtitle%3DBiorg.%2520Med.%2520Chem.%26date%3D2003%26volume%3D11%26spage%3D21%26epage%3D29%26doi%3D10.1016%2FS0968-0896%2802%2900427-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="note"><p class="first last">Interestingly, this reference reports no covalent bond observed between the active site cysteine and the ketone pharmacophore.</p></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fuentes-Prior, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvesen, G. S.</span></span> <span> </span><span class="NLM_article-title">The protein structures that shape caspase activity, specificity, activation and inhibition</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>384</i></span>,  <span class="NLM_fpage">201</span>– <span class="NLM_lpage">232</span>, <span class="refDoi"> DOI: 10.1042/BJ20041142</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1042%2FBJ20041142" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=15450003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVSqs7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=384&publication_year=2004&pages=201-232&author=P.+Fuentes-Priorauthor=G.+S.+Salvesen&title=The+protein+structures+that+shape+caspase+activity%2C+specificity%2C+activation+and+inhibition&doi=10.1042%2FBJ20041142"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">The protein structures that shape caspase activity, specificity, activation and inhibition</span></div><div class="casAuthors">Fuentes-Prior, Pablo; Salvesen, Guy S.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">384</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">201-232</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">A review.  The death morphol. commonly known as apoptosis results from a post-translational pathway driven largely by specific limited proteolysis.  In the last decade the structural basis for apoptosis regulation has moved from nothing to 'quite good', and we now know the fundamental structures of examples from the initiator phase, the pre-mitochondrial regulator phase, the executioner phase, inhibitors and their antagonists, and even the structures of some substrates.  The field is as well advanced as the best known of proteolytic pathways, the coagulation cascade.  Fundamentally new mechanisms in protease regulation have been disclosed.  Structural evidence suggests that caspases have an unusual catalytic mechanism, and that they are activated by apparently unrelated events, depending on which position in the apoptotic pathway they occupy.  Some naturally occurring caspase inhibitors have adopted classic inhibition strategies, but other have revealed completely novel mechanisms.  All of the structural and mechanistic information can, and is, being applied to drive therapeutic strategies to combat overactivation of apoptosis in degenerative disease, and underactivation in neoplasia.  We present a comprehensive review of the caspases, their regulators and inhibitors from a structural and mechanistic point of view, and with an aim to consolidate the many threads that define the rapid growth of this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowJ7v8n99EJrVg90H21EOLACvtfcHk0lhwEc0tdrPxmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVSqs7rP&md5=b3c9b22933c76e89ee5fa993e9ec2247</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1042%2FBJ20041142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20041142%26sid%3Dliteratum%253Aachs%26aulast%3DFuentes-Prior%26aufirst%3DP.%26aulast%3DSalvesen%26aufirst%3DG.%2BS.%26atitle%3DThe%2520protein%2520structures%2520that%2520shape%2520caspase%2520activity%252C%2520specificity%252C%2520activation%2520and%2520inhibition%26jtitle%3DBiochem.%2520J.%26date%3D2004%26volume%3D384%26spage%3D201%26epage%3D232%26doi%3D10.1042%2FBJ20041142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Obach, R. S.</span></span> <span> </span><span class="NLM_article-title">Prediction of human clearance of twenty-nine drugs from hepatic microsomal intrinsic clearance data: an examination of in vitro half-life approach and nonspecific binding to microsomes</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1350</span>– <span class="NLM_lpage">1359</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10534321" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK1MXntFSht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1999&pages=1350-1359&author=R.+S.+Obach&title=Prediction+of+human+clearance+of+twenty-nine+drugs+from+hepatic+microsomal+intrinsic+clearance+data%3A+an+examination+of+in+vitro+half-life+approach+and+nonspecific+binding+to+microsomes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of human clearance of twenty-nine drugs from hepatic microsomal intrinsic clearance data: an examination of in vitro half-life approach and nonspecific binding to microsomes</span></div><div class="casAuthors">Obach, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1350-1359</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Twenty-nine drugs of disparate structures and physicochem. properties were used in an examn. of the capability of human liver microsomal lability data ("in vitro T1/2" approach) to be useful in the prediction of human clearance.  Addnl., the potential importance of nonspecific binding to microsomes in the in vitro incubation milieu for the accurate prediction of human clearance was investigated.  The compds. examd. demonstrated a wide range of microsomal metabolic labilities with scaled intrinsic clearance values ranging from less than 0.5 mL/min/kg to 189 mL/min/kg.  Microsomal binding was detd. at microsomal protein concns. used in the lability incubations.  For the 29 compds. studied, unbound fractions in microsomes ranged from 0.11 to 1.0.  Generally, basic compds. demonstrated the greatest extent of binding and neutral and acidic compds. the least extent of binding.  In the projection of human clearance values, basic and neutral compds. were well predicted when all binding considerations (blood and microsome) were disregarded, however, including both binding considerations also yielded reasonable predictions.  Including only blood binding yielded very poor projections of human clearance for these two types of compds.  However, for acidic compds., disregarding all binding considerations yielded poor predictions of human clearance.  It was generally most difficult to accurately predict clearance for this class of compds.; however the accuracy was best when all binding considerations were included.  Overall, inclusion of both blood and microsome binding values gave the best agreement between in vivo clearance values and clearance values projected from in vitro intrinsic clearance data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWp2hUTOxyubVg90H21EOLACvtfcHk0lhwEc0tdrPxmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXntFSht7c%253D&md5=7804cd4dfb131caac45aff9b7a1df901</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DObach%26aufirst%3DR.%2BS.%26atitle%3DPrediction%2520of%2520human%2520clearance%2520of%2520twenty-nine%2520drugs%2520from%2520hepatic%2520microsomal%2520intrinsic%2520clearance%2520data%253A%2520an%2520examination%2520of%2520in%2520vitro%2520half-life%2520approach%2520and%2520nonspecific%2520binding%2520to%2520microsomes%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D1999%26volume%3D27%26spage%3D1350%26epage%3D1359" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="note"><p class="first last">Solubility of <b>4</b> in several formulation vehicles: 4.6 mg/mL (2.5% w/v SBECD, 5 mM phosphate buffer, pH 7.4, made isotonic with 4.25% dextrose); 4.6 mg/mL (2.5% w/v SBECD, 5 mM citrate buffer, pH 7.4, made isotonic with 4.25% dextrose); 6.6 mg/mL (2.5% w/v HPCD).</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span> </span><span class="NLM_article-title">The X-ray crystallographic
coordinates for the SARS CoV-1 3CL<sup>pro</sup> (PDB Code 6XHL) and
SARS CoV-2 3CL<sup>pro</sup> (6XHM)
each covalently bound inhibitor <b>4</b> are deposited in the
Protein Data Bank (PDB)</span>, <a href="http://rcsb.rutgers.edu/pdb/index" class="extLink">http://rcsb.rutgers.edu/pdb/index</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=The+X-ray+crystallographic%0Acoordinates+for+the+SARS+CoV-1+3CLpro+%28PDB+Code+6XHL%29+and%0ASARS+CoV-2+3CLpro+%286XHM%29%0Aeach+covalently+bound+inhibitor+4+are+deposited+in+the%0AProtein+Data+Bank+%28PDB%29%2C+http%3A%2F%2Frcsb.rutgers.edu%2Fpdb%2Findex."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DThe%2520X-ray%2520crystallographic%250Acoordinates%2520for%2520the%2520SARS%2520CoV-1%25203CLpro%2520%2528PDB%2520Code%25206XHL%2529%2520and%250ASARS%2520CoV-2%25203CLpro%2520%25286XHM%2529%250Aeach%2520covalently%2520bound%2520inhibitor%25204%2520are%2520deposited%2520in%2520the%250AProtein%2520Data%2520Bank%2520%2528PDB%2529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wandel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kajiji, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guengerich, F. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, A. A. J.</span></span> <span> </span><span class="NLM_article-title">P-glycoprotein and cytochrome P-450 3A inhibition: dissociation of inhibitory potencies</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3944</span>– <span class="NLM_lpage">3948</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10463589" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK1MXlsVOltL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=1999&pages=3944-3948&author=C.+Wandelauthor=R.+B.+Kimauthor=S.+Kajijiauthor=F.+P.+Guengerichauthor=G.+R.+Wilkinsonauthor=A.+A.+J.+Wood&title=P-glycoprotein+and+cytochrome+P-450+3A+inhibition%3A+dissociation+of+inhibitory+potencies"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">P-glycoprotein and cytochrome P-450 3A inhibition: dissociation of inhibitory potencies</span></div><div class="casAuthors">Wandel, Christoph; Kim, Richard B.; Kajiji, Shama; Guengerich, F. Peter; Wilkinson, Grant R.; Wood, Alastair J. J.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3944-3948</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">AACR Subscription Office</span>)
        </div><div class="casAbstract">Many P-glycoprotein (P-gp) inhibitors studied in vitro and in vivo are also known or suspected to be substrates and/or inhibitors of cytochrome P 450 3A (CYP3A).  Such overlap raises the question of whether CYP3A inhibition is an intrinsic characteristic of P-gp inhibitors, a matter of concern in the development and rational use of such agents.  Thus, the purpose of the present study was to det. whether the ability to inhibit P-gp and CYP3A is, in fact, linked and whether specific P-gp inhibitors with limited ability to inhibit CYP3A can be identified.  Therefore, the potency of a series of 14 P-gp inhibitors was assessed by measuring their inhibition of the transepithelial flux across Caco-2 cells of digoxin, a prototypical P-gp substrate.  CYP3A inhibition was detd. from the impairment of nifedipine oxidn. by human liver microsomes.  Detn. of the apparent Ki values for CYP3A inhibition and the IC50s for P-gp and CYP3A inhibition allowed comparison of the relative inhibitory potency of the compds. on the two proteins' function.  The IC50s for P-gp inhibition ranged from 0.04 to 3.8 μM.  All compds. inhibited CYP3A with apparent Ki values of between 0.3 and 76 μM and IC50s between 1.5 and 50 μM.  However, no correlation was found between the extent of P-gp inhibition and CYP3A inhibition, and the ratio of the IC50 for CYP3A inhibition to the IC50 for P-gp inhibition varied from 1.1 to 125.  These results demonstrate that, although many P-gp inhibitors are potent inhibitors of CYP3A, a varying degree of selectivity is present.  The development and use of P-gp inhibitors with minimal or absent CYP3A inhibitory effects should decrease the impact of drug interactions on the therapeutic use of such compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCV7MbrQ_Rn7Vg90H21EOLACvtfcHk0lj4q-tCBIuHrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXlsVOltL4%253D&md5=2993e6b2b3cd2b4c00dd4ca71b020ee2</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWandel%26aufirst%3DC.%26aulast%3DKim%26aufirst%3DR.%2BB.%26aulast%3DKajiji%26aufirst%3DS.%26aulast%3DGuengerich%26aufirst%3DF.%2BP.%26aulast%3DWilkinson%26aufirst%3DG.%2BR.%26aulast%3DWood%26aufirst%3DA.%2BA.%2BJ.%26atitle%3DP-glycoprotein%2520and%2520cytochrome%2520P-450%25203A%2520inhibition%253A%2520dissociation%2520of%2520inhibitory%2520potencies%26jtitle%3DCancer%2520Res.%26date%3D1999%26volume%3D59%26spage%3D3944%26epage%3D3948" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="note"><p class="first last"><i>In vivo</i> parameters for <b>4</b> were from IV dosing using a 50% PEG 200/10% EtOH/40% saline formulation. For rat (2 mg/kg) the following values were obtained: 28.6 mL/min/kg CL; 1.2 L/kg Vdss; 1.0 <i>t</i><sub>1/2</sub>; and 1.2 h·μg/mL AUC. For dog (2 mg/kg) the following values were obtained: 10.5 mL/min/kg CL; 0.4 L/kg Vdss; 1.3 <i>t</i><sub>1/2</sub>; and 3.8 h·μg/mL AUC. For monkey (4 mg/kg) the following values were obtained: 20.6 mL/min/kg CL; 0.6 L/kg <i>V</i><sub>dss</sub>; 1.6 <i>t</i><sub>1/2</sub>; and 3.8 h·μg/mL AUC.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grum-Tokars, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratia, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Begaye, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesecar, A.</span></span> <span> </span><span class="NLM_article-title">Evaluating the 3C-like protease activity of SARS-Coronavirus: recommendations for standardized assays for drug discovery</span>. <i>Virus Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">63</span>– <span class="NLM_lpage">73</span>, <span class="refDoi"> DOI: 10.1016/j.virusres.2007.02.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1016%2Fj.virusres.2007.02.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=17397958" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjtlWksb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2008&pages=63-73&author=V.+Grum-Tokarsauthor=K.+Ratiaauthor=A.+Begayeauthor=S.+Bakerauthor=A.+Mesecar&title=Evaluating+the+3C-like+protease+activity+of+SARS-Coronavirus%3A+recommendations+for+standardized+assays+for+drug+discovery&doi=10.1016%2Fj.virusres.2007.02.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluating the 3C-like protease activity of SARS-Coronavirus: Recommendations for standardized assays for drug discovery</span></div><div class="casAuthors">Grum-Tokars, Valerie; Ratia, Kiira; Begaye, Adrian; Baker, Susan C.; Mesecar, Andrew D.</div><div class="citationInfo"><span class="NLM_cas:title">Virus Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">63-73</span>CODEN:
                <span class="NLM_cas:coden">VIREDF</span>;
        ISSN:<span class="NLM_cas:issn">0168-1702</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Although the initial outbreaks of the deadly coronavirus that causes severe acute respiratory syndrome (SARS-CoV) were controlled by public health measures, the development of vaccines and antiviral agents for SARS-CoV is essential for improving control and treatment of future outbreaks.  One potential target for SARS-CoV antiviral drug development is the 3C-like protease (3CLpro).  This enzyme is an attractive target since it is essential for viral replication, and since there are now a no. of high resoln. X-ray structures of SARS-CoV 3CLpro available making structure-based drug-design possible.  As a result, SARS-CoV 3CLpro has become the focus of numerous drug discovery efforts worldwide, but as a consequence, a variety of different 3CLpro expression constructs and kinetic assays have been independently developed making evaluation and comparison between potential inhibitors problematic.  Here, we review the literature focusing on different SARS-CoV 3CLpro expression constructs and assays used to measure enzymic activity.  Moreover, we provide exptl. evidence showing that the activity of 3CLpro enzyme is significantly reduced when non-native sequences or affinity-tags are added to the N- or C-termini of the enzyme, or when the enzyme used in assays is at concns. below the equil. dissocn. const. of the 3CLpro dimer.  We demonstrate for the first time the utility of a highly sensitive and novel Alexa488-QSY7 FRET-based peptide substrate designed for routine anal. and high-throughput screening, and show that kinetic consts. detd. from FRET-based assays that are uncorrected for inner-filter effects can lead to artifacts.  Finally, we evaluated the effects of common assay components including DTT, NaCl, EDTA and DMSO on enzymic activity, and we recommend standardized assay conditions and constructs for routine SARS-CoV 3CLpro assays to facilitate direct comparisons between SARS-CoV 3CLpro inhibitors under development worldwide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpev2bSotX3dLVg90H21EOLACvtfcHk0lj4q-tCBIuHrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjtlWksb8%253D&md5=4895fe12a3e04a2b157cd0317ec2a3da</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.virusres.2007.02.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.virusres.2007.02.015%26sid%3Dliteratum%253Aachs%26aulast%3DGrum-Tokars%26aufirst%3DV.%26aulast%3DRatia%26aufirst%3DK.%26aulast%3DBegaye%26aufirst%3DA.%26aulast%3DBaker%26aufirst%3DS.%26aulast%3DMesecar%26aufirst%3DA.%26atitle%3DEvaluating%2520the%25203C-like%2520protease%2520activity%2520of%2520SARS-Coronavirus%253A%2520recommendations%2520for%2520standardized%2520assays%2520for%2520drug%2520discovery%26jtitle%3DVirus%2520Res.%26date%3D2008%26volume%3D133%26spage%3D63%26epage%3D73%26doi%3D10.1016%2Fj.virusres.2007.02.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borenfreund, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puerner, J.</span></span> <span> </span><span class="NLM_article-title">Toxicity determined in vitro by morphological alterations and neutral red absorption</span>. <i>Toxicol. Lett.</i> <span class="NLM_year" style="font-weight: bold;">1985</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">119</span>– <span class="NLM_lpage">124</span>, <span class="refDoi"> DOI: 10.1016/0378-4274(85)90046-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1016%2F0378-4274%2885%2990046-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=3983963" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaL2MXhvVShu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=1985&pages=119-124&author=E.+Borenfreundauthor=J.+Puerner&title=Toxicity+determined+in+vitro+by+morphological+alterations+and+neutral+red+absorption&doi=10.1016%2F0378-4274%2885%2990046-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Toxicity determined in vitro by morphological alterations and neutral red absorption</span></div><div class="casAuthors">Borenfreund, Ellen; Puerner, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology Letters</span>
        (<span class="NLM_cas:date">1985</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">2-3</span>),
    <span class="NLM_cas:pages">119-24</span>CODEN:
                <span class="NLM_cas:coden">TOLED5</span>;
        ISSN:<span class="NLM_cas:issn">0378-4274</span>.
    </div><div class="casAbstract">A method is described which combines the use of a visual morphol. cytotoxicity assay with a quant. neutral red  [553-24-2] spectrophotometric test, for the assessment of the effect of toxic agents on mouse fibroblast 3T3 cells in culture.  These sensitive and reproducible assays lend themselves to a screening procedure of potential toxicants which can help reduce the use of animals for toxicity testing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocpD9P9r7fAbVg90H21EOLACvtfcHk0lj4q-tCBIuHrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXhvVShu7s%253D&md5=8bb53b9abc203b72c07e75668fac401f</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2F0378-4274%2885%2990046-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0378-4274%252885%252990046-3%26sid%3Dliteratum%253Aachs%26aulast%3DBorenfreund%26aufirst%3DE.%26aulast%3DPuerner%26aufirst%3DJ.%26atitle%3DToxicity%2520determined%2520in%2520vitro%2520by%2520morphological%2520alterations%2520and%2520neutral%2520red%2520absorption%26jtitle%3DToxicol.%2520Lett.%26date%3D1985%26volume%3D24%26spage%3D119%26epage%3D124%26doi%3D10.1016%2F0378-4274%2885%2990046-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smee, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurst, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clyde, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, C.</span></span> <span> </span><span class="NLM_article-title">Evaluation of cell viability dyes in antiviral assays with RNA viruses that exhibit different cytopathogenic properties</span>. <i>J. Virol. Methods</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>246</i></span>,  <span class="NLM_fpage">51</span>– <span class="NLM_lpage">57</span>, <span class="refDoi"> DOI: 10.1016/j.jviromet.2017.03.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1016%2Fj.jviromet.2017.03.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=28359770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvVKns70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=246&publication_year=2017&pages=51-57&author=D.+F.+Smeeauthor=B.+Hurstauthor=W.+Evansauthor=N.+Clydeauthor=S.+Wrightauthor=C.+Petersonauthor=K.-H.+Jungauthor=C.+Day&title=Evaluation+of+cell+viability+dyes+in+antiviral+assays+with+RNA+viruses+that+exhibit+different+cytopathogenic+properties&doi=10.1016%2Fj.jviromet.2017.03.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of cell viability dyes in antiviral assays with RNA viruses that exhibit different cytopathogenic properties</span></div><div class="casAuthors">Smee, Donald F.; Hurst, Brett L.; Evans, W. Joseph; Clyde, Nathan; Wright, Sean; Peterson, Christopher; Jung, Kie-Hoon; Day, Craig W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virological Methods</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">246</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">51-57</span>CODEN:
                <span class="NLM_cas:coden">JVMEDH</span>;
        ISSN:<span class="NLM_cas:issn">0166-0934</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Studies were conducted to det. the performance of four dyes in assessing antiviral activities of compds. against three RNA viruses with differing cytopathogenic properties.  Dyes included alamarBlue measured by absorbance (ALB-A) and fluorescence (ALB-F), neutral red (NR), Viral ToxGlo (VTG), and WST-1.  Viruses were chikungunya, dengue type 2, and Junin, which generally cause 100, 80-90, and 50% maximal cytopathic effect (CPE), resp., in Vero or Vero 76 cells Compds. evaluated were 6-azauridine, BCX-4430, 3-deazaguanine, EICAR, favipiravir, infergen, mycophenolic acid (MPA), ribavirin, and tiazofurin.  The 50% virus-inhibitory (EC50) values for each inhibitor and virus combination did not vary significantly based on the dye used.  However, dyes varied in distinguishing the vitality of virus-infected cultures when not all cells were killed by virus infection.  For example, VTG uptake into dengue-infected cells was nearly 50% when visual examn. showed only 10-20% cell survival.  ALB-A measured infected cell viability differently than ALB-F as follows: 16% vs. 32% (dengue-infected), resp., and 51% vs. 72% (Junin-infected), resp.  Cytotoxicity (CC50) assays with dyes in uninfected proliferating cells produced similar CC50 values for EICAR (1.5-8.9 μM) and MPA (0.8-2.5 μM). 6-Azauridine toxicity was 6.1-17.5 μM with NR, VTG, and WST-1, compared to 48-92 μM with ALB-A and ALB-F (P < 0.001).  Curiously, the CC50 values for 3-deazaguanine were 83-93 μM with ALB-F vs. 2.4-7.0 μM with all other dyes including ALB-A (P < 0.001).  Overall, ALB minimized the toxicities detected with these two inhibitors.  Because the choice of dyes affected CC50 values, this impacted on the resulting in vitro selectivity indexes (calcd. as CC50/EC50 ratio).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0QJD6MLX54bVg90H21EOLACvtfcHk0ljW5pAQ3cxEgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmvVKns70%253D&md5=2e19131fe61c56db0d6e7a19c105f99c</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.jviromet.2017.03.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jviromet.2017.03.012%26sid%3Dliteratum%253Aachs%26aulast%3DSmee%26aufirst%3DD.%2BF.%26aulast%3DHurst%26aufirst%3DB.%26aulast%3DEvans%26aufirst%3DW.%26aulast%3DClyde%26aufirst%3DN.%26aulast%3DWright%26aufirst%3DS.%26aulast%3DPeterson%26aufirst%3DC.%26aulast%3DJung%26aufirst%3DK.-H.%26aulast%3DDay%26aufirst%3DC.%26atitle%3DEvaluation%2520of%2520cell%2520viability%2520dyes%2520in%2520antiviral%2520assays%2520with%2520RNA%2520viruses%2520that%2520exhibit%2520different%2520cytopathogenic%2520properties%26jtitle%3DJ.%2520Virol.%2520Methods%26date%3D2017%26volume%3D246%26spage%3D51%26epage%3D57%26doi%3D10.1016%2Fj.jviromet.2017.03.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scudiero, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoemaker, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paull, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monks, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tierney, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nofziger, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Currens, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seniff, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, M. R.</span></span> <span> </span><span class="NLM_article-title">Evaluation of a soluble tetrazolium/formazan assay for cell growth and drug sensitivity in culture using human and other tumor cell lines</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">4827</span>– <span class="NLM_lpage">4833</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=3409223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaL1cXls1arsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=1988&pages=4827-4833&author=D.+A.+Scudieroauthor=R.+H.+Shoemakerauthor=K.+D.+Paullauthor=A.+Monksauthor=S.+Tierneyauthor=T.+H.+Nofzigerauthor=M.+J.+Currensauthor=D.+Seniffauthor=M.+R.+Boyd&title=Evaluation+of+a+soluble+tetrazolium%2Fformazan+assay+for+cell+growth+and+drug+sensitivity+in+culture+using+human+and+other+tumor+cell+lines"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of a soluble tetrazolium/formazan assay for cell growth and drug sensitivity in culture using human and other tumor cell lines</span></div><div class="casAuthors">Scudiero, Dominic A.; Shoemaker, Robert H.; Paull, Kenneth D.; Monks, Anne; Tierney, Siobhan; Nofziger, Thomas H.; Currens, Michael J.; Seniff, Donna; Boyd, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4827-33</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    </div><div class="casAbstract">An automated microculture tetrazolium assay (MTA) involving DMSO solubilization of cellular-generated 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide (MTT)-formazan had several inherent disadvantages.  Possible alternative MTAs were developed utilizing a different tetrazolium reagent, 2,3-bis(2-methoxy4-nitro-5-sulfophenyl)-5-[(phenylamino)carbonyl]-2H-tetrazolium hydroxide (XTT), which is metabolically reduced in viable cells to a water-sol. formazan product.  This reagent allowed direct absorbance readings, therefore eliminating a solubilization step and shortening the microculture growth assay procedure.  Most human tumor cell lines examd. metabolized XTT less efficiently than MTT; however, the addn. of phenazine methosulfate (PMS) markedly enhanced cellular redn. of XTT.  In the presence of PMS, the XTT reagent yielded usable absorbance values for growth and drug sensitivity evaluations with a variety of cell lines.  Depending on the metabolic reductive capacity of a given cell line, the optimal conditions for a 4-h XTT incubation assay were 50 μg of XTT and 0.15-0.4 μg of PMS per well.  Drug profiles obtained with representative human tumor cell lines for several std. compds. utilizing the XTT-PMS methodol. were similar to the profiles obtained with MTT.  Addn. of PMS appeared to have little effect on the metab. of MTT.  The new XTT reagent thus provides for a simplified, in vitro cell growth assay with possible applicability to a variety of problems in cellular pharmacol. and biol.  However, the MTA using the XTT reagent still shares many of the limitations and potential pitfalls of MTT or other tetrazolium-based assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNCG5C-wpmkrVg90H21EOLACvtfcHk0ljW5pAQ3cxEgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXls1arsrk%253D&md5=b88dc8255a07af053a91897fd6c451cc</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DScudiero%26aufirst%3DD.%2BA.%26aulast%3DShoemaker%26aufirst%3DR.%2BH.%26aulast%3DPaull%26aufirst%3DK.%2BD.%26aulast%3DMonks%26aufirst%3DA.%26aulast%3DTierney%26aufirst%3DS.%26aulast%3DNofziger%26aufirst%3DT.%2BH.%26aulast%3DCurrens%26aufirst%3DM.%2BJ.%26aulast%3DSeniff%26aufirst%3DD.%26aulast%3DBoyd%26aufirst%3DM.%2BR.%26atitle%3DEvaluation%2520of%2520a%2520soluble%2520tetrazolium%252Fformazan%2520assay%2520for%2520cell%2520growth%2520and%2520drug%2520sensitivity%2520in%2520culture%2520using%2520human%2520and%2520other%2520tumor%2520cell%2520lines%26jtitle%3DCancer%2520Res.%26date%3D1988%26volume%3D48%26spage%3D4827%26epage%3D4833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weislow, O. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiser, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fine, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoemaker, R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, M. R.</span></span> <span> </span><span class="NLM_article-title">New soluble-formazan assay for HIV-1 cytopathic effects: application to high-flux screening of synthetic and natural products for AIDS-antiviral activity</span>. <i>J. Natl. Cancer Inst.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">577</span>– <span class="NLM_lpage">586</span>, <span class="refDoi"> DOI: 10.1093/jnci/81.8.577</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1093%2Fjnci%2F81.8.577" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=2495366" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaL1MXkvV2lt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=1989&pages=577-586&author=O.+S.+Weislowauthor=R.+Kiserauthor=D.+L.+Fineauthor=J.+Baderauthor=R.+H.+Shoemakerauthor=M.+R.+Boyd&title=New+soluble-formazan+assay+for+HIV-1+cytopathic+effects%3A+application+to+high-flux+screening+of+synthetic+and+natural+products+for+AIDS-antiviral+activity&doi=10.1093%2Fjnci%2F81.8.577"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">New soluble-formazan assay for HIV-1 cytopathic effects:  application to high-flux screening of synthetic and natural products for AIDS-antiviral activity</span></div><div class="casAuthors">Weislow, Owen S.; Kiser, Rebecca; Fine, Donald L.; Bader, John; Shoemaker, Robert H.; Boyd, Michael R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">577-86</span>CODEN:
                <span class="NLM_cas:coden">JNCIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    </div><div class="casAbstract">An effective and optimally safe microculture method was developed for the rapid and convenient assay of the in vitro cytopathic effects of human immunodeficiency virus (HIV-1) on human lymphoblastoid or other suitable host cells.  The assay procedure is applicable to the evaluation of drug effects on in vitro infections induced directly in cultured host cells by cell-free HIV-1 or by coculture with H9 cells chronically infected with HIV-1.  The assay uses a newly developed tetrazolium reagent that is metabolically reduced by viable cells to yield a sol., colored formazan product measureable by conventional colorimetric techniques.  This simple microassay minimizes the no. of plate manipulations typically required with other assay methods and, coupled with computerized data collection and anal., facilitates large-scale screening of agents for potential antiviral activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDJZMdwKZfgbVg90H21EOLACvtfcHk0ljW5pAQ3cxEgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXkvV2lt78%253D&md5=c39a027f4ddc6bb5d7b1108c1b128be6</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2F81.8.577&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252F81.8.577%26sid%3Dliteratum%253Aachs%26aulast%3DWeislow%26aufirst%3DO.%2BS.%26aulast%3DKiser%26aufirst%3DR.%26aulast%3DFine%26aufirst%3DD.%2BL.%26aulast%3DBader%26aufirst%3DJ.%26aulast%3DShoemaker%26aufirst%3DR.%2BH.%26aulast%3DBoyd%26aufirst%3DM.%2BR.%26atitle%3DNew%2520soluble-formazan%2520assay%2520for%2520HIV-1%2520cytopathic%2520effects%253A%2520application%2520to%2520high-flux%2520screening%2520of%2520synthetic%2520and%2520natural%2520products%2520for%2520AIDS-antiviral%2520activity%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D1989%26volume%3D81%26spage%3D577%26epage%3D586%26doi%3D10.1093%2Fjnci%2F81.8.577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hippenmeyer, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dilworth, V. M.</span></span> <span> </span><span class="NLM_article-title">A rapid assay for determination of antiviral activity against human cytomegalovirus</span>. <i>Antiviral Res.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">35</span>– <span class="NLM_lpage">42</span>, <span class="refDoi"> DOI: 10.1016/0166-3542(95)00976-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1016%2F0166-3542%2895%2900976-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=8863994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK28XksF2htLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=1996&pages=35-42&author=P.+J.+Hippenmeyerauthor=V.+M.+Dilworth&title=A+rapid+assay+for+determination+of+antiviral+activity+against+human+cytomegalovirus&doi=10.1016%2F0166-3542%2895%2900976-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">A rapid assay for determination of antiviral activity against human cytomegalovirus</span></div><div class="casAuthors">Hippenmeyer, Paul J.; Dilworth, Vickie M.</div><div class="citationInfo"><span class="NLM_cas:title">Antiviral Research</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">35-42</span>CODEN:
                <span class="NLM_cas:coden">ARSRDR</span>;
        ISSN:<span class="NLM_cas:issn">0166-3542</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">RC256, the recombinant human cytomegalovirus (HCMV) which expresses β-galactosidase, was used as a tool for rapid screening of compds. for antiviral activity.  The effective concn. of antiviral compd. needed to inhibit RC256 was identical to the concns. needed to inhibit other strains of HCMV as measured by plaque redn. assay or virus yield redn. assay.  Measurement of β-galactosidase activity in infected cell lysates allowed detn. of effective concns. 48 h postinfection with results comparable to the longer, more laborious assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDFUA4a6va1bVg90H21EOLACvtfcHk0liELEAygvZIxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XksF2htLk%253D&md5=33af6dd34fd78e65c6a9ab78813c5dcf</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2F0166-3542%2895%2900976-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0166-3542%252895%252900976-0%26sid%3Dliteratum%253Aachs%26aulast%3DHippenmeyer%26aufirst%3DP.%2BJ.%26aulast%3DDilworth%26aufirst%3DV.%2BM.%26atitle%3DA%2520rapid%2520assay%2520for%2520determination%2520of%2520antiviral%2520activity%2520against%2520human%2520cytomegalovirus%26jtitle%3DAntiviral%2520Res.%26date%3D1996%26volume%3D32%26spage%3D35%26epage%3D42%26doi%3D10.1016%2F0166-3542%2895%2900976-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patick, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Binford, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brothers, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diem, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maldonado, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dragovich, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prins, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuhrman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meador, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zalman, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Worland, S. T.</span></span> <span> </span><span class="NLM_article-title">In vitro antiviral activity of AG7088, a potent inhibitor of human rhinovirus 3C protease</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">2444</span>– <span class="NLM_lpage">2450</span>, <span class="refDoi"> DOI: 10.1128/AAC.43.10.2444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1128%2FAAC.43.10.2444" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10508022" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADyaK1MXms1KqsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=1999&pages=2444-2450&author=A.+K.+Patickauthor=S.+L.+Binfordauthor=M.+A.+Brothersauthor=R.+L.+Jacksonauthor=C.+E.+Fordauthor=M.+D.+Diemauthor=F.+Maldonadoauthor=P.+S.+Dragovichauthor=R.+Zhouauthor=T.+J.+Prinsauthor=S.+A.+Fuhrmanauthor=J.+W.+Meadorauthor=L.+S.+Zalmanauthor=D.+A.+Matthewsauthor=S.+T.+Worland&title=In+vitro+antiviral+activity+of+AG7088%2C+a+potent+inhibitor+of+human+rhinovirus+3C+protease&doi=10.1128%2FAAC.43.10.2444"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro antiviral activity of AG7088, a potent inhibitor of human rhinovirus 3C protease</span></div><div class="casAuthors">Patick, A. K.; Binford, S. L.; Brothers, M. A.; Jackson, R. L.; Ford, C. E.; Diem, M. D.; Maldonado, F.; Dragovich, P. S.; Zhou, R.; Prins, T. J.; Fuhrman, S. A.; Meador, J. W.; Zalman, L. S.; Matthews, D. A.; Worland, S. T.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2444-2450</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">AG-7088 is a potent, irreversible inhibitor of human rhinovirus (HRV) 3C protease (inactivation rate const. kobs/[I] = 1,470,000±440,000 M-1 s-1 for HRV 14) that was discovered by protein structure-based drug design methodologies.  In H1-HeLa and MRC-5 cell protection assays, AG-7088 inhibited the replication of all HRV serotypes (48 of 48) tested with a mean 50% EC50 of 0.023 μM (range 0.003-0.081 μM) and a mean EC90 of 0.082 μM (range 0.018-0.261 μM), as well as that of related picornaviruses including coxsackieviruses A21 and B3, enterovirus 70, and echovirus 11.  No decreases in the antiviral activity of AG-7088 were obsd. when assays were performed in the presence of α1-acid glycoprotein or mucin proteins found in nasal secretions.  The 50% cytotoxic concn. of AG-7088 was >1000 μM, yielding a therapeutic index of >12,346 to >333,333.  In a single-cycle, time-of-addn. assay, AG-7088 had antiviral activity when added up to 6 h after infection.  A compd. targeting viral attachment and/or uncoating (WIN-51711) was effective only when added at the initiation of virus infection.  Direct inhibition of 3C proteolytic activity in infected cells treated with AG-7088 was demonstrated by SDS-polyacrylamide gel electrophoresis anal. of radiolabeled proteins, which showed a dose-dependent accumulation of viral precursor polyproteins and a decrease of processed protein products.  The broad spectrum of antiviral activity of AG-7088, combined with its efficacy even when added late in the virus life cycle, highlights the advantages of 3C protease as a target and suggests that AG7088 is a promising clin. candidate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojpMqAUz2M3LVg90H21EOLACvtfcHk0liELEAygvZIxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXms1KqsL8%253D&md5=70e98480d38e133c96866005b0578f75</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1128%2FAAC.43.10.2444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.43.10.2444%26sid%3Dliteratum%253Aachs%26aulast%3DPatick%26aufirst%3DA.%2BK.%26aulast%3DBinford%26aufirst%3DS.%2BL.%26aulast%3DBrothers%26aufirst%3DM.%2BA.%26aulast%3DJackson%26aufirst%3DR.%2BL.%26aulast%3DFord%26aufirst%3DC.%2BE.%26aulast%3DDiem%26aufirst%3DM.%2BD.%26aulast%3DMaldonado%26aufirst%3DF.%26aulast%3DDragovich%26aufirst%3DP.%2BS.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DPrins%26aufirst%3DT.%2BJ.%26aulast%3DFuhrman%26aufirst%3DS.%2BA.%26aulast%3DMeador%26aufirst%3DJ.%2BW.%26aulast%3DZalman%26aufirst%3DL.%2BS.%26aulast%3DMatthews%26aufirst%3DD.%2BA.%26aulast%3DWorland%26aufirst%3DS.%2BT.%26atitle%3DIn%2520vitro%2520antiviral%2520activity%2520of%2520AG7088%252C%2520a%2520potent%2520inhibitor%2520of%2520human%2520rhinovirus%25203C%2520protease%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1999%26volume%3D43%26spage%3D2444%26epage%3D2450%26doi%3D10.1128%2FAAC.43.10.2444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blight, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolykhalov, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rice, C. M.</span></span> <span> </span><span class="NLM_article-title">Efficient initiation of HCV RNA replication in cell culture</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">1972</span>– <span class="NLM_lpage">1974</span>, <span class="refDoi"> DOI: 10.1126/science.290.5498.1972</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=10.1126%2Fscience.290.5498.1972" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=11110665" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BD3cXoslKgsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2000&pages=1972-1974&author=K.+J.+Blightauthor=A.+A.+Kolykhalovauthor=C.+M.+Rice&title=Efficient+initiation+of+HCV+RNA+replication+in+cell+culture&doi=10.1126%2Fscience.290.5498.1972"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient initiation of HCV RNA replication in cell culture</span></div><div class="casAuthors">Blight, Keril J.; Kolykhalov, Alexander A.; Rice, Charles M.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">5498</span>),
    <span class="NLM_cas:pages">1972-1974</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Hepatitis C virus (HCV) infection is a global health problem affecting an estd. 170 million individuals worldwide.  We report the identification of multiple independent adaptive mutations that cluster in the HCV nonstructural protein NS5A and confer increased replicative ability in vitro.  Among these adaptive mutations were a single amino acid substitution that allowed HCV RNA replication in 10% of transfected hepatoma cells and a deletion of 47 amino acids encompassing the interferon (IFN) sensitivity detg. region (ISDR).  Independent of the ISDR, IFN-α rapidly inhibited HCV RNA replication in vitro.  This work establishes a robust, cell-based system for genetic and functional analyses of HCV replication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPYOuqOjy-ZbVg90H21EOLACvtfcHk0liELEAygvZIxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXoslKgsL4%253D&md5=67eb65d15efc36854faf3940db9a5d91</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1126%2Fscience.290.5498.1972&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.290.5498.1972%26sid%3Dliteratum%253Aachs%26aulast%3DBlight%26aufirst%3DK.%2BJ.%26aulast%3DKolykhalov%26aufirst%3DA.%2BA.%26aulast%3DRice%26aufirst%3DC.%2BM.%26atitle%3DEfficient%2520initiation%2520of%2520HCV%2520RNA%2520replication%2520in%2520cell%2520culture%26jtitle%3DScience%26date%3D2000%26volume%3D290%26spage%3D1972%26epage%3D1974%26doi%3D10.1126%2Fscience.290.5498.1972" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Solowiej, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, B.</span></span> <span> </span><span class="NLM_article-title">Steady-state and pre-steady-state kinetic evaluation of severe acute respiratory syndrome coronavirus (SARS-CoV) 3CL<sup>pro</sup> cysteine protease: development of an ion-pair model for catalysis</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">2617</span>, <span class="refDoi"> DOI: 10.1021/bi702107v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi702107v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1OmtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=2617&author=J.+Solowiejauthor=J.+Thomsonauthor=K.+Ryanauthor=C.+Luoauthor=M.+Heauthor=J.+Louauthor=B.+Murray&title=Steady-state+and+pre-steady-state+kinetic+evaluation+of+severe+acute+respiratory+syndrome+coronavirus+%28SARS-CoV%29+3CLpro+cysteine+protease%3A+development+of+an+ion-pair+model+for+catalysis&doi=10.1021%2Fbi702107v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Steady-State and Pre-Steady-State Kinetic Evaluation of Severe Acute Respiratory Syndrome Coronavirus (SARS-CoV) 3CLpro Cysteine Protease: Development of an Ion-Pair Model for Catalysis</span></div><div class="casAuthors">Solowiej, James; Thomson, James A.; Ryan, Kevin; Luo, Chun; He, Mingying; Lou, Jihong; Murray, Brion W.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2617-2630</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Severe acute respiratory syndrome (SARS) was a worldwide epidemic caused by a coronavirus that has a cysteine protease (3CLpro) essential to its life cycle.  Steady-state and pre-steady-state kinetic methods were used with highly active 3CLpro to characterize the reaction mechanism.  We show that 3CLpro has mechanistic features common and disparate to the archetypical proteases papain and chymotrypsin.  The kinetic mechanism for 3CLpro-mediated ester hydrolysis, including the individual rate consts., is consistent with a simple double displacement mechanism.  The pre-steady-state burst rate was independent of ester substrate concn. indicating a high commitment to catalysis.  When homologous peptidic amide and ester substrates were compared, a series of interesting observations emerged.  Despite a 2000-fold difference in nonenzymic reactivity, highly related amide and ester substrates were found to have similar kinetic parameters in both the steady-state and pre-steady-state.  Steady-state solvent isotope effect (SIE) studies showed an inverse SIE for the amide but not ester substrates.  Evaluation of the SIE in the pre-steady-state revealed normal SIEs for both amide and ester burst rates.  Proton inventory (PI) studies on amide peptide hydrolysis were consistent with two proton-transfer reactions in the transition state while the ester data was consistent with a single proton-transfer reaction.  Finally, the pH-inactivation profile of 3CLpro with iodoacetamide is indicative of an ion-pair mechanism.  Taken together, the data are consistent with a 3CLpro mechanism that utilizes an "electrostatic" trigger to initiate the acylation reaction, a cysteine-histidine catalytic dyad ion pair, an enzyme-facilitated release of P1, and a general base-catalyzed deacylation reaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpO92WZdvoiorVg90H21EOLACvtfcHk0lhz4zvSaebSrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1OmtrY%253D&md5=77d755ed4262558b065ea6246a1f57ae</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fbi702107v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi702107v%26sid%3Dliteratum%253Aachs%26aulast%3DSolowiej%26aufirst%3DJ.%26aulast%3DThomson%26aufirst%3DJ.%26aulast%3DRyan%26aufirst%3DK.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DLou%26aufirst%3DJ.%26aulast%3DMurray%26aufirst%3DB.%26atitle%3DSteady-state%2520and%2520pre-steady-state%2520kinetic%2520evaluation%2520of%2520severe%2520acute%2520respiratory%2520syndrome%2520coronavirus%2520%2528SARS-CoV%2529%25203CLpro%2520cysteine%2520protease%253A%2520development%2520of%2520an%2520ion-pair%2520model%2520for%2520catalysis%26jtitle%3DBiochemistry%26date%3D2008%26volume%3D47%26spage%3D2617%26doi%3D10.1021%2Fbi702107v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XHO" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XHO','PDB','6XHO'); return false;">PDB: 6XHO</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XHN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XHN','PDB','6XHN'); return false;">PDB: 6XHN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XHL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XHL','PDB','6XHL'); return false;">PDB: 6XHL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XHM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6XHM','PDB','6XHM'); return false;">PDB: 6XHM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1Q2W" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1Q2W','PDB','1Q2W'); return false;">PDB: 1Q2W</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i98"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01063">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_78745"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01063?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01063</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Molecular formula strings for all compounds with biological data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01063/suppl_file/jm0c01063_si_001.csv">CSV</a>)</p></li><li><p class="inline">HPLC purity for compound <b>4</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01063/suppl_file/jm0c01063_si_002.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01063/suppl_file/jm0c01063_si_001.csv">jm0c01063_si_001.csv (4.06 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01063/suppl_file/jm0c01063_si_002.pdf">jm0c01063_si_002.pdf (95.19 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01063&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01063%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-21" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01063" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799dce80b303d30","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
