
<html lang="en"     class="pb-page"  data-request-id="5f85306b-dade-45c7-a8b7-bf4463926d0a"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.9b01989;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;issue:issue:10.1021/jmcmar.2020.63.issue-16;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Drug Research Meets Network Science: Where Are We?" /></meta><meta name="dc.Creator" content="Maurizio  Recanatini" /></meta><meta name="dc.Creator" content="Chiara  Cabrelle" /></meta><meta name="dc.Description" content="Network theory provides one of the most potent analysis tools for the study of complex systems. In this paper, we illustrate the network-based perspective in drug research and how it is coherent wi..." /></meta><meta name="Description" content="Network theory provides one of the most potent analysis tools for the study of complex systems. In this paper, we illustrate the network-based perspective in drug research and how it is coherent wi..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 27, 2020" /></meta><meta name="dc.Type" content="review-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01989" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2020 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01989" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01989" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01989" /></link>
        
    
    

<title>Drug Research Meets Network Science: Where Are We? | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01989" /></meta><meta property="og:title" content="Drug Research Meets Network Science: Where Are We?" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01989/20210220/images/large/jm9b01989_0005.jpeg" /></meta><meta property="og:description" content="Network theory provides one of the most potent analysis tools for the study of complex systems. In this paper, we illustrate the network-based perspective in drug research and how it is coherent with the new paradigm of drug discovery. We first present data sources from which networks are built, then show some examples of how the networks can be used to investigate drug-related systems. A section is devoted to network-based inference applications, i.e., prediction methods based on interactomes, that can be used to identify putative drug–target interactions without resorting to 3D modeling. Finally, we present some aspects of Boolean networks dynamics, anticipating that it might become a very potent modeling framework to develop in silico screening protocols able to simulate phenotypic screening experiments. We conclude that network applications integrated with machine learning and 3D modeling methods will become an indispensable tool for computational drug discovery in the next years." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01989"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01989">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01989&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01989&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01989&amp;href=/doi/10.1021/acs.jmedchem.9b01989" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 16</span><span class="cit-fg-pageRange">, 8653-8666</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/16" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01077" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Perspective</span><a href="/doi/10.1021/acs.jmedchem.9b02120" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Drug Research Meets Network Science: Where Are We?</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Maurizio Recanatini</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Maurizio Recanatini</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacy and Biotechnology, Alma Mater Studiorum—University of Bologna, Via Belmeloro 6, I-40126 Bologna, Italy</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#2b464a5e594251424405594e484a454a5f4245426b5e4542494405425f"><span class="__cf_email__" data-cfemail="93fef2e6e1fae9fafcbde1f6f0f2fdf2e7fafdfad3e6fdfaf1fcbdfae7">[email protected]</span></a></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Maurizio++Recanatini">Maurizio Recanatini</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath1">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0039-0518" title="Orcid link">http://orcid.org/0000-0002-0039-0518</a></div></div></span></li><span class="comma-separator"> and </span><li><span><span class="hlFld-ContribAuthor">Chiara Cabrelle</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chiara Cabrelle</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacy and Biotechnology, Alma Mater Studiorum—University of Bologna, Via Belmeloro 6, I-40126 Bologna, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chiara++Cabrelle">Chiara Cabrelle</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath2">View Biography</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01989&amp;href=/doi/10.1021%2Facs.jmedchem.9b01989" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 16</span><span class="cit-pageRange">, 8653–8666</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 27, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>29 November 2019</li><li><span class="item_label"><b>Published</b> online</span>27 April 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 27 August 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01989" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01989</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="https://acsopenscience.org/open-access/licensing-options/"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a><span class="article_header-cc-by"><a href="https://creativecommons.org/licenses/by/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_cc-1612971093887.svg" alt="CC: Creative Commons" /></img><span><strong>CC: </strong>Creative Commons</span></a><a href="https://creativecommons.org/licenses/by/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_by-1612971093257.svg" alt="BY: Credit must be given to the creator" /></img><span><strong>BY: </strong>Credit must be given to the creator</span></a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01989"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2778</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">4</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01989" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Drug Research Meets Network Science: Where Are We?&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Maurizio&quot;,&quot;last_name&quot;:&quot;Recanatini&quot;},{&quot;first_name&quot;:&quot;Chiara&quot;,&quot;last_name&quot;:&quot;Cabrelle&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;27&quot;,&quot;issue&quot;:&quot;16&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;8653-8666&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01989&quot;},&quot;abstract&quot;:&quot;Network theory provides one of the most potent analysis tools for the study of complex systems. In this paper, we illustrate the network-based perspective in drug research and how it is coherent with the new paradigm of drug discovery. We first present data sources from which networks are built, then show some examples of how the networks can be used to investigate drug-related systems. A section is devoted to network-based inference applications, i.e., prediction methods based on interactomes, that can be used to identify putative drug–target interactions without resorting to 3D modeling. Finally, we present some aspects of Boolean networks dynamics, anticipating that it might become a very potent modeling framework to develop in silico screening protocols able to simulate phenotypic screening experiments. We conclude that network applications integrated with machine learning and 3D modeling methods will become an indispensable tool for computational drug discovery in the next years.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01989&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01989" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01989&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01989" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01989&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01989" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01989&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01989&amp;href=/doi/10.1021/acs.jmedchem.9b01989" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01989" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01989" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01989&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01989%26sid%3Dliteratum%253Aachs%26pmid%3D32338900%26genre%3Darticle%26aulast%3DRecanatini%26date%3D2020%26atitle%3DDrug%2BResearch%2BMeets%2BNetwork%2BScience%253A%2BWhere%2BAre%2BWe%253F%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D16%26spage%3D8653%26epage%3D8666%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291070" title="Pharmaceuticals">Pharmaceuticals</a>,</li><li><a href="/action/doSearch?ConceptID=291583" title="Mathematical methods">Mathematical methods</a>,</li><li><a href="/action/doSearch?ConceptID=292202" title="Drug discovery">Drug discovery</a>,</li><li><a href="/action/doSearch?ConceptID=290731" title="Biological databases">Biological databases</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/16" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/jmcmar.2020.63.issue-16/20200827/jmcmar.2020.63.issue-16.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01989/20210220/images/medium/jm9b01989_0005.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01989/20210220/images/large/jm9b01989_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01989&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Network theory provides one of the most potent analysis tools for the study of complex systems. In this paper, we illustrate the network-based perspective in drug research and how it is coherent with the new paradigm of drug discovery. We first present data sources from which networks are built, then show some examples of how the networks can be used to investigate drug-related systems. A section is devoted to network-based inference applications, i.e., prediction methods based on interactomes, that can be used to identify putative drug–target interactions without resorting to 3D modeling. Finally, we present some aspects of Boolean networks dynamics, anticipating that it might become a very potent modeling framework to develop <i>in silico</i> screening protocols able to simulate phenotypic screening experiments. We conclude that network applications integrated with machine learning and 3D modeling methods will become an indispensable tool for computational drug discovery in the next years.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="specialIssueNotice" class="extra-info-sec articleNote"><h4>SPECIAL ISSUE</h4><p class="last">This article is part of the <a href="/toc/jmcmar/63/16"><issue-title>Artificial Intelligence in Drug Discovery</issue-title></a> special issue.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">1.  Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31625" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31625" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The first decade of the 2000s has seen a consistent modification of the drug research landscape, due, among other aspects, to a rethinking of the drug discovery paradigm<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> and to the entrance into the era of Big Data.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> The paradigm change in drug discovery is derived from both the need to improve the success rate of pharmaceutical industry and the diffusion across the drug research community of discoveries and concepts related to systems biology.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> In fact, driven by the initially debated and finally accepted proposal of multitarget drugs,<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> the conventional one-disease–one-target idea was abandoned to embrace a less reductionist view. The inspired concept of network pharmacology elaborated by Andrew L. Hopkins<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> allowed us to move toward a new system-based paradigm.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> On the other hand, the availability of relevant amounts of data, from molecular descriptors to bioassay results, from -omic information to clinical records, provides the starting material for the elaboration of multilevel integrated models that take full advantage of the constantly growing informatics technology for model building, analysis, interpretation, and exploitation.</div><div class="NLM_p">Complexity is a distinctive trait of living systems, and recently researchers have started to investigate natural complex systems with adequate theoretical tools.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> In particular, network science permits catching of the behavior of a system as a whole, especially regarding its emergent properties, that is, the features that arise from the interaction of the systems’ parts and not as a mere sum of them. In the field of medicine, the point of view proposed by Albert L. Barabási is illuminating: if disease phenotypes can be viewed as emergent properties deriving from the interconnection of pathobiological processes, in turn arising from the cross-talk of molecular, metabolic, and regulatory networks at cellular level, a framework of “network medicine” might help in exploring causes and finding therapies at a global integrated level.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a></div><div class="NLM_p">Networks rely on the idea of modeling a real system as a map of interconnected dots and lines representing the set of elements and the set of relationships between them, respectively. The mathematical description of a network is addressed by the graph theory such that graph stands for network, and often, even though inaccurately, these terms are used as synonyms. Elements of the network are called nodes, connections between them, links or edges. The start of graph theory is commonly traced back to Leonhard Euler’s paper <i>Solutio problematis ad geometriam situs pertinentis</i> that appeared in 1741 on the <i>Commentarii Academiae Scientiarum Petropolitanae</i>.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> The problem to be solved was to find a pathway going through all seven bridges of the city of Koenigsberg without crossing the same bridge twice. Euler demonstrated the inexistence of such a pathway, based on an abstract representation of the four areas of the city (nodes) linked by the seven bridges (links). In other words, he, for the first time, used a graph to represent and solve a mathematical problem. From then on, graph theory and network science developed mainly in the fields of mathematics and physics, respectively, and today they form a sound body of science and provide formidable tools to deal with the architecture and properties of complex systems.<a onclick="showRef(event, 'ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13">(11−13)</a></div><div class="NLM_p last">Now the question is to what extent the network-based approach is influencing or has already influenced the way new therapies are looked for. Actually, the ways network theory is applied to drug discovery are numerous and aimed at different purposes. Limiting ourselves to the medicinal chemistry/drug design area, the main fields where network-based methods are employed are target identification and drug repurposing, both eventually in the context of polypharmacology. Further fields of application are the analysis of chemical spaces and the prediction of adverse drug reactions or toxicity. Several examples of these applications of network theory have appeared in the literature in the past years, and they have been reviewed in a number of papers. It is worth mentioning here the comprehensive review on the argument published in 2013 by Peter Csermely et al.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> that laid the foundations of a network-based drug science. In the present Perspective article, we recapitulate the basis of the approach and see how far we have gone with it by illustrating some recently published results. We also discuss some possible advancement and points of attention in the field.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">2.  Data, Databases, and Networks</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24827" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24827" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4">2.1.  Data and Databases</h3><div class="NLM_p">The first and most important issue to consider when dealing with the construction of networks to investigate pathobiological processes and drug effects on them concerns the material that we use to build these models, that is, what we usually call “data”. By this term we mean a wide variety of items, expressed in numerical, alphabetical, or digital form and collected in databases that often are publicly available. Although in the drug discovery community we make use of data sets at least since the early days of QSAR, the information made available by the high-throughput experimental technologies is growing at an unprecedented pace. Nowadays, we can access data sources on compounds, targets, and diseases that cover millions of molecules and thousands of proteins and genes in almost every therapeutic field.</div><div id="sec2_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i5" class="anchor-spacer"></div><h4 class="article-section__title" id="_i5">2.1.1.  Chemical Databases</h4><div class="NLM_p">On the chemical compounds side, the number and variety of freely accessible databases increase continuously, and each of these data sets contains information for numbers of molecules that range from 10<sup>3</sup> to 10<sup>7</sup>. Recently, Yang et al. attempted to classify the public chemical databases into six categories based on their content, namely, (1) chemical information, (2) bioactivity, (3) drugs, (4) natural products, (5) commercial availability, and (6) fragments.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> All types of data contained in the chemical databases can be useful for drug design purposes in general, but for what concerns network applications, data on bioactivity, drugs, and natural products are most interesting. With this respect, the most popular databases are CHEMBL<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> and PubChem,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> which provide knowledge on bioactive compounds, particularly data from activity assays and target information. DrugBank<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> contains data on approved and experimental drugs and can be an important source of information in target identification and drug repurposing studies. On the purely chemical side, ChemSpider<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> is a very rich source of physicochemical and spectral data, as well as of names, synonyms, and identifiers. In <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, the main features of the mentioned databases are summarized.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Chemical Databases</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">database name</th><th class="colsep0 rowsep0" align="center">description</th><th class="colsep0 rowsep0" align="center">url</th><th class="colsep0 rowsep0" align="center">ref.</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ChemBL</td><td class="colsep0 rowsep0" align="left">Collection of bioactive drug-like small molecules with 2D structures, calculated chemical properties and bioactivities.</td><td class="colsep0 rowsep0" align="left"><a href="https://www.ebi.ac.uk/chembl/" class="extLink">https://www.ebi.ac.uk/chembl/</a></td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PubChem</td><td class="colsep0 rowsep0" align="left">Open chemistry database of mostly small molecules that collects information on chemical structures, chemical and physical properties, and biological activities. It is structured into three linked databases: substance, compound, and bioassay.</td><td class="colsep0 rowsep0" align="left"><a href="https://pubchem.ncbi.nlm.nih.gov/" class="extLink">https://pubchem.ncbi.nlm.nih.gov/</a></td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DrugBank</td><td class="colsep0 rowsep0" align="left">Freely accessible data on small molecules and biotechnological drugs (with chemical, pharmaceutical, and pharmacological profiles), and drug targets (sequence and functions of target/enzyme/transporter/carrier) intended as drugs encyclopedia.</td><td class="colsep0 rowsep0" align="left"><a href="https://www.drugbank.ca/" class="extLink">https://www.drugbank.ca/</a></td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ChemSpider</td><td class="colsep0 rowsep0" align="left">Free chemical structures database collecting structures and related information, such as physicochemical properties and interactive spectra, made accessible through a fast search engine allowing search by name, structure, or advanced options.</td><td class="colsep0 rowsep0" align="left"><a href="http://www.chemspider.com/" class="extLink">http://www.chemspider.com/</a></td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a></td></tr></tbody></table></div></div><div class="NLM_p last">The most important issue to consider when using a chemical data set is the need of preprocessing its content, a crucial operation that may take a long time but indispensable to lower the probability of obtaining misleading results and/or building erroneous models. Tropsha and co-workers have addressed this argument in the context of QSAR modeling,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> but their conclusions and suggestions fit perfectly here. In particular, the curation of a data set should consider chemical, biological, and item identification aspects, as, for example, representation of chemical structure and presence in the data set of nonstandardized structures (salts, ions, etc.), variability of biological data for the same compound, reproducibility of results through different laboratories, presence of activity cliffs, misspelled, or mislabeled compounds, and incorrect identifiers. The same authors provided thoughtful guidelines for chemical and biological data curation to avoid errors and to ensure the stability and reliability of the models.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a></div></div><div id="sec2_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i7" class="anchor-spacer"></div><h4 class="article-section__title" id="_i7">2.1.2.  Biological Databases</h4><div class="NLM_p">When dealing with target identification or drug repurposing studies in a system-wide perspective, it is mandatory to include in the network the knowledge that is increasingly generated by the huge array of biological techniques. Fortunately, the availability of free access biological databases is even bigger than that of chemical substances. The journal <i>Nucleic Acid Research</i> (NAR) in 2019 published the 26th annual issue of the Molecular Biology Database Collection,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> presenting an impressive list of 1613 databases, shortly describing the new entries, and eventually updating old ones. Also in this case, databases are grouped, reflecting the categorization of the databases in the NAR online database collection, that is, (1) nucleic acid sequence and structure and transcriptional regulation; (2) protein sequence and structure; (3) metabolic and signaling pathways, enzymes, and networks; (4) genomics of viruses, bacteria, protozoa, and fungi; (5) genomics of human and model organisms plus comparative genomics; (6) human genomic variation, diseases, and drugs; (7) plants; (8) other topics. It is worth noting that the above-mentioned data sources are mostly genotype-based, leveraging the fast development of the -omic technologies. Among the conspicuous number of available databases, some can be of particular interest for the drug design field, like those containing information regarding proteins, as either general sequence (e.g., SMART,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> UniProt<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a>) or sequence of individual protein families (e.g., GPCRdb,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Kinomer<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a>), protein structure or protein–protein interactions (e.g., PDB,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> STRING<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a>), metabolic and signaling pathways (e.g., Reactome<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a>), human genes and diseases (e.g., DisGeNET<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a>). A special mention is deserved here of two important resources that provide drug researchers with invaluable information on targets and mechanisms of action of small molecules: the ConnectivityMap<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> and LINCS<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> platforms that give access to gene transcriptional profiles in response to perturbation by drugs or other chemical compounds. In <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, the main features of these databases are summarized.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Biological Databases</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">database name</th><th class="colsep0 rowsep0" align="center">description</th><th class="colsep0 rowsep0" align="center">url</th><th class="colsep0 rowsep0" align="center">ref</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SMART</td><td class="colsep0 rowsep0" align="left">Simple modular architecture research tool is a Web tool for identification of protein domains and exploration of domain architectures.</td><td class="colsep0 rowsep0" align="left"><a href="http://smart.embl-heidelberg.de/" class="extLink">http://smart.embl-heidelberg.de/</a></td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">UniProtKB</td><td class="colsep0 rowsep0" align="left">UniProtKB consists of two databases: SwissProt (manually annotated) and TrEMBL (automatically annotated and unreviewed). UniProtKB collects protein sequences providing rich annotations.</td><td class="colsep0 rowsep0" align="left"><a href="https://www.uniprot.org/" class="extLink">https://www.uniprot.org/</a></td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GPCRdb</td><td class="colsep0 rowsep0" align="left">GPCRdb stores reference data on GPCRs sequence, structures, mutations, and ligand interactions. GPCRdb also provides a suite of Web tools and interactive diagrams for illustrative purposes.</td><td class="colsep0 rowsep0" align="left"><a href="https://gpcrdb.org/" class="extLink">https://gpcrdb.org/</a></td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Kinomer</td><td class="colsep0 rowsep0" align="left">Protein kinases database contains annotated classification of 43 eukaryotic genomes. Kinomer is accessible through a Web interface, which enables the retrieval of sequences and the classification of arbitrary sequences.</td><td class="colsep0 rowsep0" align="left"><a href="https://www.compbio.dundee.ac.uk/kinomer" class="extLink">https://www.compbio.dundee.ac.uk/kinomer</a></td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDB</td><td class="colsep0 rowsep0" align="left">Protein Data Bank is a repository of experimentally determined 3D structures of proteins, nucleic acids, and complexes with metal ions, drugs, and small molecules.</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/" class="extLink">https://www.rcsb.org/</a></td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">STRING</td><td class="colsep0 rowsep0" align="left">Search tool for retrieval of interacting genes/proteins (STRING) database of known and predicted protein–protein interactions, including physical and functional associations for a large number of organisms. STRING allows users to visualize interactions network and to make analysis.</td><td class="colsep0 rowsep0" align="left"><a href="https://string-db.org/" class="extLink">https://string-db.org/</a></td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Reactome</td><td class="colsep0 rowsep0" align="left">Reactome is a relational pathway database in which signaling and metabolic molecules are organized into biological pathways and processes.</td><td class="colsep0 rowsep0" align="left"><a href="https://reactome.org/" class="extLink">https://reactome.org/</a></td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DisGeNET</td><td class="colsep0 rowsep0" align="left">DisGeNET is a platform that integrates and standardizes data about disease associated genes and variants (628 685 gene-disease associations, GDAs, and 210 498 variant-disease associations, VDAs).</td><td class="colsep0 rowsep0" align="left"><a href="https://www.disgenet.org/" class="extLink">https://www.disgenet.org/</a></td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ConnectivityMap</td><td class="colsep0 rowsep0" align="left">CMap database contains gene-expression profiles from cultured human cells treated with perturbagens (bioactive small molecules).</td><td class="colsep0 rowsep0" align="left"><a href="https://clue.io/cmap" class="extLink">https://clue.io/cmap</a></td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">LINCS</td><td class="colsep0 rowsep0" align="left">Library of integrated network-based cellular signatures catalogues perturbation-response signatures employing a various set of perturbations, model systems, and assay types.</td><td class="colsep0 rowsep0" align="left"><a href="http://www.lincsproject.org/" class="extLink">http://www.lincsproject.org/</a></td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a></td></tr></tbody></table></div></div></div><div id="sec2_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i9" class="anchor-spacer"></div><h4 class="article-section__title" id="_i9">2.1.3.  Phenotypic Data</h4><div class="NLM_p last">Moving to the field of human phenotypic data, we leave the territory of drug discovery to enter into the precision medicine arena. In this context, the pervasive digitalization of healthcare is providing quantitatively very important sources of phenotypic data, like primarily those contained in the electronic health records (EHRs) but also those generated by wearable devices or apps.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Limiting to EHRs, the information embedded in these documents includes the description of the health/disease status of individuals, clinical test results, drug prescriptions, and eventual adverse effects.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Of course, privacy issues limit the availability of this kind of data, and we cannot find publicly accessible databases yet. Nevertheless, several initiatives already exist to collect EHRs and related data and make them available for the biomedical research, as, for example, the “All of Us” initiative.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> As regards the contribution of this kind of information to the drug research, we observe that integration of phenotypic and genotypic data might be a necessary step toward a deeper understanding of the biological processes at the base of the onset and progression of diseases.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Even though it is not yet clear how this integration will be translated to the discovery of <i>new</i> drugs, this approach is being applied for drug repurposing, as shown by the recent work by Khosravi et al.,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> who proposed a list of repurposed drug candidates for melanoma treatment after the analysis of genome- and phenome-wide association studies.</div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10">2.2.  Building the Network</h3><div class="NLM_p">Given the wide availability of data on molecules, genes, proteins, cells, tissues, and diseases and the assumption that these entities are connected and representative of a more or less complex system, one needs to build and visualize the network (actually, the graph representing the network) in view of the subsequent analysis. The computational tools available for network visualization and analysis are countless and range in complexity depending on the dimensions of the data set and of the task to be executed. There are several popular and efficient software that can work on personal computer/workstation and allow one to perform all the basic operations on the network, from visualization to analysis of the basic parameters. Cytoscape<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> is the most popular tool, but also Gephi,<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Pajek,<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> and NetworkX<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> are rather widespread in the biological community (see <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> for details on the main features of the softwares). A recent review by Pavlopoulos et al.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> analyzes and compares the performance of different software tools for the visualization of even large networks up to the order of magnitude of 10<sup>6</sup> nodes and edges. However, when the amount of data increases, the memory requirements to load large matrices become prohibitive even for powerful workstations, and higher performance computing is needed to analyze the network. A solution can be to distribute data and processes on high numbers of cores by means of frameworks, like Hadoop<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> or Apache Spark<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> (see <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). In this way, computations are partitioned across clusters of machines that work in parallel and carry out the jobs in reasonable time and with high efficiency.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Network Building and Visualization Systems</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">database name</th><th class="colsep0 rowsep0" align="center">description</th><th class="colsep0 rowsep0" align="center">url</th><th class="colsep0 rowsep0" align="center">ref</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Cytoscape</td><td class="colsep0 rowsep0" align="left">Cytoscape is a platform for visualization, analysis, and integration of networks via basic functionalities or through apps; conceived mainly for biological research.</td><td class="colsep0 rowsep0" align="left"><a href="https://cytoscape.org/" class="extLink">https://cytoscape.org/</a></td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Gephi</td><td class="colsep0 rowsep0" align="left">Gephi allows the visualization and exploration of all types of large graphs in real-time through a 3D render engine.</td><td class="colsep0 rowsep0" align="left"><a href="https://gephi.org/" class="extLink">https://gephi.org/</a></td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Pajek</td><td class="colsep0 rowsep0" align="left">Pajek enables analysis and visualization of large networks having some thousands or millions of vertices.</td><td class="colsep0 rowsep0" align="left"><a href="http://mrvar.fdv.uni-lj.si/pajek/" class="extLink">http://mrvar.fdv.uni-lj.si/pajek/</a></td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NetworkX</td><td class="colsep0 rowsep0" align="left">NetworkX is a Phyton package designed for the creation and analysis of structure, dynamics, and functions of networks.</td><td class="colsep0 rowsep0" align="left"><a href="https://networkx.github.io/" class="extLink">https://networkx.github.io/</a></td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Apache Hadoop</td><td class="colsep0 rowsep0" align="left">Open source framework for storing and processing large data sets across clusters of computers in a distributed environment through simple programming models.</td><td class="colsep0 rowsep0" align="left"><a href="https://hadoop.apache.org/" class="extLink">https://hadoop.apache.org/</a></td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Apache Spark</td><td class="colsep0 rowsep0" align="left">A fast cluster computing system for large-scale data processing powering different libraries (SQL, MLlib, GraphX), and easy to use interactively from the Scala, Python, R, and SQL shells.</td><td class="colsep0 rowsep0" align="left"><a href="https://spark.apache.org/" class="extLink">https://spark.apache.org/</a></td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></td></tr></tbody></table></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">3.  Networks to Study Systems of Pharmaceutical Interest</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61806" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61806" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">As a first step to illustrate the use of networks in drug research, we can consider studies aimed at visualizing and analyzing systems of biomedical/pharmacological interest. A short explanation of basic concepts of network theory and of the terminology used in this field is provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01989/suppl_file/jm9b01989_si_001.pdf" class="ext-link">Supporting Information</a>.</div><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13">3.1.  Networks for the Analysis of Molecules Data Sets</h3><div class="NLM_p">A relevant example of the use of network analysis in the small organic molecules context is that of chemical space networks (CSN), a framework proposed and developed by G. Maggiora and J. Bajorath.<a onclick="showRef(event, 'ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref45 ref46">(45,46)</a> In their first review on the argument, these authors stated that “‘molecular networks’ are thought to provide an alternative way to represent and navigate chemical space” and that “chemical space exploration [...] is often motivated by the need to better understand structure–property relationships of small organic compounds”.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> In the initial applications of the CSN formalism, the aim was simply to obtain a new coordinate-free representation of the chemical space, based upon pairwise compound similarities, instead of the more commonly used coordinate-based representations referring to the descriptor space. In subsequent papers, Bajorath and co-workers showed how well network representations allowed highlighting of the properties of a chemical space viewed as a complex system to whom emergent properties like biological activity could be associated.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Different similarity metrics were introduced and validated,<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> and together with the analysis of the network topology parameters, they were shown to be a powerful tool to visualize and analyze the structure–activity relationships (SARs) of moderately sized compound sets.<a onclick="showRef(event, 'ref46 ref48'); return false;" href="javascript:void(0);" class="ref ref46 ref48">(46,48)</a> In this context, the analysis of the CSN through adequate metrics and algorithms can reveal the presence of communities (clusters) of compounds sharing latent characteristics not immediately evident from the common table format.</div><div class="NLM_p">To illustrate a simple CSN application, in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> a network of 62 poly(ADP-ribose)polymerase (PARP) inhibitors is shown. The network accounts for relationships between the compounds, and the links among them were derived from pairwise similarity values calculated based on fingerprints. The inhibitors are represented by nodes (62) that are connected by edges (188) if their structure similarity exceeds a threshold (see the legend to <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). The nodes are colored according to potency. This visualization of the chemical space based on similarity calculation facilitates the identification of the different structural families of PARP inhibitors (the main connected components of the network), and the color-coding allows one to grasp immediately the SAR of the set of compounds.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01989/20210220/images/medium/jm9b01989_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01989/20210220/images/large/jm9b01989_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Exemplary CSN of PARP inhibitors. PARPs 1, 2, and 3 family inhibitors with a measured EC<sub>50</sub> were retrieved from CHEMBL.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> The pairwise chemical similarities between compounds were assessed by means of Tanimoto coefficient (Tc) values calculated for the ECFP4 fingerprints<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> of the molecules generated by Canvas<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> (Schrödinger, LLC, New York, NY, 2019). Pairs of inhibitors were connected only if their calculated Tc value exceeded the threshold value of 0.55. The chemical structures of the inhibitors are shown inside the nodes that are colored according to pEC<sub>50</sub> values ranging from red (lowest potency) to green (highest potency) and sized based on node degree from small (low degree) to large (high degree). Edges are weighted by Tc values from thin (Tc = 0.55) to thick (Tc = 1) width. The network was generated by means of Cytoscape<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> version 3.7.2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01989/20210220/images/large/jm9b01989_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01989&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The usefulness of the CSN framework was also recently shown in the field of natural product extraction and characterization, as reported in a paper by Nothias-Esposito et al.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> Here, the authors analyzed the extracts from two species of <i>Euphorbia</i> plants by means of tandem mass spectrometry and generated the network representing the isolated compounds using the mass spectra as nodes linked by similarity. In particular, to give rise to the network, the spectra were converted into vectors that were used to calculate a similarity score between each pair of spectra.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> The network then allowed visualization of structurally related molecules (based on their spectra, irrespective of their structure) that were subsequently identified by performing a search against a public reference spectral library.</div><div class="NLM_p last">The above examples deal with relatively small data sets, but the versatility of network analysis can be exploited even when the amount of data becomes much greater. This is the case of a work recently published by Miho et al., who performed a large-scale analysis on networks built from antibody repertoires.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> To appreciate the dimension of the problem, it is enough to note that to build the network of a repertoire of, for example, 10<sup>6</sup> clones (the nodes), one needs a similarity matrix of 10<sup>12</sup> elements to define the edges. This is a relevant computational task that the authors tackled by leveraging the power of parallel distributed computing through the Apache Spark framework.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> The analysis of the network of complete antibody repertoires allowed overcoming of the limitations of using only portions of the network that are not <i>a priori</i> statistically representative of the properties (parameters) of the whole system and led the authors to derive some general principles on immune repertoire architecture.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a></div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14">3.2.  Protein Structure Networks</h3><div class="NLM_p last">If we consider that a protein, like any other molecule, is an ensemble of interacting elements (the amino acid residues, in this case), it is immediately derived that it represents a complex system in which structure, dynamics, and eventually function can be viewed as emergent properties stemming from the relationships among the residues. In this context, protein structure networks (PSN) are widely studied, as the network approach is considered quite suitable to deal with the structure–function relationships, also in the light of the fast growth of analytical/biophysical technologies applicable to protein structure determination. PSNs are built by considering the amino acids (usually the Cα atoms) as nodes that are connected by a link if the distance between them falls within a cutoff value. The analysis of the parameters describing the properties of the PSNs lends itself to the study of the protein’s 3D architecture and its implications in issues like allosteric communication, folding, and model validation.<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> In a recent review, Di Paola et al. discussed several applications of graph theory to the description of protein properties in terms of network parameters and advanced the hypothesis that the protein contact graph (representing the PSN) might in the future be considered as the structural formula of proteins.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> It is interesting to note that the PSN analysis can be carried out also in combination with other computational techniques to investigate the protein structural/conformational behavior. In particular, molecular dynamics simulations and eventually binding free energy calculations can be synergistically applied to studies of pharmaceutical interest. For example, Verkhivker showed the usefulness of this combined approach to interpret how allosteric effects act on the modulation of protein functions in the presence of inhibitors<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> or cancer driver mutations.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> Recently, Amusengeri et al.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> applied a computational protocol including a residue network analysis to identify allosteric inhibitors of <i>Plasmodium falciparum</i> 70 kDa heat shock proteins as new antimalarial drug candidates.</div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15">3.3.  Human Disease Network and Drug Discovery</h3><div class="NLM_p">Considering drug research from a system-wide, broad perspective, we cannot help but face the studies on the human disease network carried out by Barabási and his co-workers. In 2007, this author proposed the term “diseasome”<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> to indicate the network of all genetic disorders systematically linked to all disease genes: it is an example of a bipartite network (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b01989/suppl_file/jm9b01989_si_001.pdf" class="ext-link">Supporting Information</a>), where diseases are connected to the genes known to cause or effect them.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> This approach provides a wider view on diseases with respect to the usual classification, revealing possible connections between some of them, and eventually allowing for new possibilities of treatment. The same authors further elaborated on the diseasome concept by taking into consideration molecular interaction networks, i.e., the “interactomes”. The latter can be gene regulatory networks (GRNs), protein–protein interaction (PPI) networks, or metabolic networks and are indispensable further elements to consider to build a system view of the cellular mechanisms underlying the phenotype–genotype relationships in human diseases.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> It is evident here the effort to hold together different layers of a complex system, by leveraging the systems biology approach,<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> in order to escape the simplifications of reductionism.</div><div class="NLM_p">Focusing more specifically on the relevance of the above considerations in drug research, first of all, we must recall that, however, as far as drug action is concerned, the drug–target interaction remains a key event that maintains its centrality across paradigms, from the old “magic bullet” idea to the present polypharmacology, systems-based approach. Having said that, it appears clear how crucial can be the modeling of the drug–target interactomes. This goal has been pursued for a decade or more, from the early representations due to the work of Hopkins<a onclick="showRef(event, 'ref5 ref63'); return false;" href="javascript:void(0);" class="ref ref5 ref63">(5,63)</a> to the important contributions from the Barabási group.<a onclick="showRef(event, 'ref64 ref65 ref66'); return false;" href="javascript:void(0);" class="ref ref64 ref65 ref66">(64−66)</a> In essence, from the data sets of experimental information (as, for example, DrugBank, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), a bipartite drug–target network (DTN) is built from which projections on the drug or the target component are obtained. This provides both the drug network (DN), and the target network (TN), each made by nodes of the same type (drugs or targets, respectively) connected to each other if they share at least one target (the DN) or one drug (the TN). Analysis of the topology of DTNs, DNs, and TNs can reveal interesting and sometimes unexpected details on how and why drugs act on certain targets or target communities within the diseasome. It can also provide useful information for the design of drug combinations, the drug repurposing, or the interpretation of toxic effects. Applications of this kind of analysis are increasingly appearing in the literature and are reviewed in recent articles (e.g., see refs <a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> and <a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a>).</div><div class="NLM_p">As an example of a complex interactome, in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>, a DTN generated from DrugBank data sets is shown. The network displays the interactions between 1636 approved small molecule drugs and 1991 human protein targets; the edges represent 7521 unique interactions. Drugs (circles) and targets (diamonds) are connected if an interaction between them is reported in the database. As it is evident from the picture, the network includes a large connected component constituted by 3368 nodes, 1510 of which are drugs. It may be noted that a relevant number of them cluster in a tightly interconnected community comprising mostly neurological (light pink), cardiovascular (brown), and respiratory system (aquamarine) drugs. The analysis of this kind of network provides a global picture of the molecular pharmacological space and might help to identify trends or possible areas of future development in drug research. An example thereof can be found in the paper by Yildirim et al.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> Clearly, these models cannot be definitive, and they evolve as new knowledge adds to the database. Starting from this consideration, in the next section, a further step into the use of networks in drug discovery will be illustrated.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01989/20210220/images/medium/jm9b01989_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01989/20210220/images/large/jm9b01989_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Drug–target network. The DTN was built from DrugBank<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> version 5.1.5 retrieving the drug–target interactions between approved small molecule drugs and human protein targets. Drugs are represented as circle-shaped nodes, and protein targets are represented as diamond-shaped nodes. As shown in the inset, drugs are color-coded according to the first level anatomical therapeutic chemical (ATC) codes as reported in DrugBank. The nodes size accounts for the node degree from small (low degree) to large (high degree). Edges connect only drugs and targets nodes. The network was generated by means of Cytoscape<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> version 3.7.2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01989/20210220/images/large/jm9b01989_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01989&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">4.  Network-Based Inferences</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30783" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30783" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Going back to the idea that networks are simplified representations of complex systems, it is plausible to recognize that they are not definitive models, but they can evolve as new information becomes available. In other words, when dealing with networks, one has to do with the problem of missing information, a common situation in the study of biological systems, where the difficulty of obtaining experimental evidence of interactions makes the network inherently incomplete but where, on the other hand, new knowledge is continuously added. This issue is quite basic in network theory and pertains to the network reconstruction scope<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> in the general context of the network statistical physics.<a onclick="showRef(event, 'ref11 ref70'); return false;" href="javascript:void(0);" class="ref ref11 ref70">(11,70)</a> In a tighter perspective, however, the possibility of inferring missing links in a network, which is the prediction of a new link between two yet unconnected nodes, is of more practical and immediate interest. Actually, a number of methods of link prediction are available that find wide application, e.g., in social or information networks, where predicting the possible association between individuals or documents, respectively, can be very useful.<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a></div><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17">4.1.  Link Prediction Methods</h3><div class="NLM_p last">Focusing on drugs, one of the hottest issues in medicine today is the identification of disease-related genes, so much so that considerable experimental efforts are carried out to pursue this goal. Applying network modeling in this field has produced a relevant amount of knowledge so far, and many examples thereof regard the network-based identification of targets (see, for example, ref <a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> in the framework of precision medicine) for the elucidation of mechanisms of action, for drug repurposing, or for the discovery of new drugs. In practice, one attempts to predict potential drug–target interactions (DTIs) using one of the numerous available prediction methods, often borrowed from such distant fields as social sciences, communication networking, economy and finance, and so on. The goal is to produce a list of potential DTIs and rank them based on some predefined metrics. The starting point is the construction of a <i>heterogeneous</i> network on which to run a link prediction algorithm. Generally, in these cases, the heterogeneous network integrates available information on drugs, targets, and drug–target interactions obtained from different databases. Given the availability of the data, the key steps in these methods are (1) the calculation of the drug–drug and target–target similarities and (2) the application of a drug–target association inference method. As regards the former, after the initial simple use of fingerprints and primary sequences to compare drug molecules and proteins, respectively, more complex and information rich similarity metrics<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> have been devised in order to take into account also the information coming from known drug–target interactions,<a onclick="showRef(event, 'ref74'); return false;" href="javascript:void(0);" class="ref ref74">(74)</a> as well as protein and network topological information,<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> eventually elaborated through ML.<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> On the methods side, network-based prediction approaches for DTIs vary broadly, often depending on the user’s preference or expertise, but the most popular ones are derived from either recommendation<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a> or network propagation<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a> algorithms, both belonging to the class of the so-called similarity-based algorithms.<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> The methods based on recommendation algorithms aim at predicting a node’s preferences for unconnected nodes based on previously calculated similarity scores (a technique also called collaborative filtering). This approach was applied by Cheng et al.,<a onclick="showRef(event, 'ref74'); return false;" href="javascript:void(0);" class="ref ref74">(74)</a> who developed the network-based inference (NBI) method for the prediction of novel DTIs based on a bipartite drug–target graph built from an adjacency matrix obtained from known drug–target interactions. Alaimo et al.<a onclick="showRef(event, 'ref79'); return false;" href="javascript:void(0);" class="ref ref79">(79)</a> enhanced this method by integrating into the model further domain-dependent biological knowledge, which is drug–drug and target–target similarity measures. On the other hand, under the framework of network propagation algorithms several methods are included that work by simulating the spread of information across the network starting from seed nodes. The most famous one is the Google page rank algorithm<a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a> that uses the random walk through Web pages to calculate their importance. In DTI prediction, the random walk with restart (RWR) variant has been developed and successfully applied to drug–target heterogeneous networks.<a onclick="showRef(event, 'ref81 ref82'); return false;" href="javascript:void(0);" class="ref ref81 ref82">(81,82)</a> The output of these calculations is a ranked list of probabilities of drug–target association. Further methods based on network propagation have recently been proposed, among which those developed by Sharan and co-workers deserve mention, as they are particularly versatile and applicable also in the personalized medicine setting.<a onclick="showRef(event, 'ref83 ref84'); return false;" href="javascript:void(0);" class="ref ref83 ref84">(83,84)</a></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18">4.2.  Applications to Drug Repurposing</h3><div class="NLM_p">As an illustrative example of the impact a network-based approach can have on drug repurposing, here we briefly describe a study called Project Repethio (<a href="https://think-lab.github.io/p/rephetio/" class="extLink">https://think-lab.github.io/p/rephetio/</a>) recently published as a research article on <i>eLife</i>.<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a> In this work, the authors report the construction of a heterogeneous network to capture the connections among drugs and diseases (Hetionet version 1.0, <a href="https://neo4j.het.io/browser/" class="extLink">https://neo4j.het.io/browser/</a>) and its use to predict new drug/disease associations as probabilities of treatment for the repositioning of known drugs. Hetionet integrates data from public sources and is composed of ∼50 000 nodes of 11 types linked by ∼2.25 million of edges of 24 types. In <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a, the conceptual scheme of the network is shown (a metagraph, i.e., a graph in which the nodes are sets of objects, and the edges connect the sets<a onclick="showRef(event, 'ref86'); return false;" href="javascript:void(0);" class="ref ref86">(86)</a>), indicating types of nodes and links, and in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>b, the whole network is visualized with the nodes grouped by type within circles, and the links are color-coded by type.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01989/20210220/images/medium/jm9b01989_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01989/20210220/images/large/jm9b01989_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Hetionet version 1.0. (a) Metagraph showing the types of nodes used to build the network and the types of links defined to connect the nodes. A detailed description of the meaning of each link type as well as the sources of information used to collect the nodes and to draw the edges is reported in ref <a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a>. (b) Visualization of the whole heterogeneous network. Nodes of the same type are grouped within circles, and links are colored by type. This Figure 3 is reproduced from Figure 1 of <contrib-group><span class="NLM_string-name">Himmelstein, D. S.</span>; <span class="NLM_string-name">Lizee, A.</span>; <span class="NLM_string-name">Hessler, C.</span>; <span class="NLM_string-name">Brueggeman, L.</span>; <span class="NLM_string-name">Chen, S. L.</span>; <span class="NLM_string-name">Hadley, D.</span>; <span class="NLM_string-name">Green, A.</span>; <span class="NLM_string-name">Khankhanian, P.</span>; <span class="NLM_string-name">Baranzini, S. E.</span></contrib-group>; <span class="NLM_year">2017</span>;   <cite><i>eLife</i></cite> (ref <a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a>), published under the Creative Commons Attribution 4.0 International Public License (CC BY 4.0; <a href="https://creativecommons.org/licenses/by/4.0/" class="extLink">https://creativecommons.org/licenses/by/4.0/</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01989/20210220/images/large/jm9b01989_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01989&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In order to extract disease treatment predictions from the drug–disease connections in the network, the authors implemented a ML procedure (logistic regression) trained on a previously compiled gold standard of 755 disease-modifying indications. These known treatments were the positive data, while ∼30 000 nontreatments were used as negatives. In short, the algorithm learns which types of compound-disease paths (metapaths) discriminate between treatments and nontreatments, and on this basis, it predicts new drug–disease associations. For 1538 compounds and 136 diseases, 209 168 associations were prioritized, and the authors provided illustrative validation through literature data and output analysis for two of them regarding nicotine dependence and epilepsy. In our opinion, the Repethio project gives a clear idea of how network-based data analysis can impact drug research, also considering the suitability of this kind of approaches to be interfaced with the powerful ML methods for features selection and prediction. Moreover, it is an example of the use of publicly available data integrated into an online platform that in turn is open to users who can access it and take advantage for their local purposes of a time- and resource-intensive assembly and integration work. We envisage that the more research data become available for the public domain, the more frequent initiatives of this kind develop, leveraging in full the combined potential of big data and network science.</div><div class="NLM_p last">Noteworthy, the variety of network-based DTI prediction methods is growing constantly in these years, and the integration with ML-based tools is frequent, as shown also by the case illustrated above. Interested readers can refer to the numerous recent reviews on the argument, e.g., like ref <a onclick="showRef(event, 'ref87'); return false;" href="javascript:void(0);" class="ref ref87">(87)</a> (see also <a class="ref internalNav" href="#sec6" aria-label="Discussion">Discussion</a>). Another direction of development is the consideration of a further source of information in the construction of the heterogeneous network, namely, diseases. This results in the building of tripartite networks, wherein drug–disease and target (gene)–disease associations are included, thus adding a further layer to the knowledge base on which predictions are calculated.<a onclick="showRef(event, 'ref88 ref89 ref90'); return false;" href="javascript:void(0);" class="ref ref88 ref89 ref90">(88−90)</a> For instance, Zong et al.<a onclick="showRef(event, 'ref89'); return false;" href="javascript:void(0);" class="ref ref89">(89)</a> proposed a DTIs prediction method based on the use of a tripartite network called linked tripartite network (LTN), where drugs, targets, and diseases are nodes connected by drug–target, drug–disease, and target–disease associations. In practice, they merged a bipartite network built from the drug–target associations from DrugBank (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) with the drug–disease and gene–disease associations from the diseasome published by Goh et al.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> to obtain the LTN containing 1452 diseases, 8201 drug–disease associations, and 1684 target–disease associations. The method showed high efficiency in the prediction of the associations between drugs and targets within the network, but it was not able to extend the predictivity to new drugs or targets not included in the model.</div></div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">5.  Network Dynamics</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04850" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04850" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">A step forward in the application of network science in drug research can be taken if we consider the possibility of modeling the time evolution of networks, that is, network dynamics. To appreciate the prospective importance of this field for future drug discovery, it is necessary to briefly introduce Boolean networks that were proposed in 1969 by S. Kauffman in the context of a general hypothesis aimed at accounting for the regulatory circuits controlling homeostasis and differentiation in cells.<a onclick="showRef(event, 'ref91 ref92'); return false;" href="javascript:void(0);" class="ref ref91 ref92">(91,92)</a> Interested readers might refer to the thoughtful review by B. Drossel.<a onclick="showRef(event, 'ref93'); return false;" href="javascript:void(0);" class="ref ref93">(93)</a></div><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20">5.1.  Boolean Networks</h3><div class="NLM_p">Boolean networks are directed networks, built in such a way where nodes are genes and links represent the functional connections between them; each gene can be “on” or “off”, and a set of rules or update functions are associated with each node to define the state of the gene at subsequent time steps. The dynamics of the system is calculated starting from input gene(s) by updating simultaneously at each discrete time step all gene states based on the predefined rules. Given the way in which it was built, a Boolean system is deterministic and has a finite number of initial network states (2<sup><i>N</i></sup>, where <i>N</i> is the number of genes, and 2 refers to the two states on and off). After a number of iterations (time steps), it will reach a stable situation that can be a fixed point or a self-looping circle: such a network state can be accessed following different network state sequences (trajectories), and it seems to attract the system to it, being therefore defined an <i>attractor</i> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>a). Several attractors may exist for a single Boolean network, and the ensemble of them together with the trajectories from the initial to the attractor states gives rise to the <i>attractor landscape</i> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b). It may be noted that a Boolean network can simulate the dynamical state of a system in normal conditions, but it might also be perturbed by modifying the update rules, i.e., setting on or off some nodes. It has been shown that the attractor states correspond to the cellular phenotypes in response to external stimuli (see below), and consequently, exploring the attractor landscape leads to the definition of the <i>phenotype landscape</i> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>c), where a phenotype includes all the network states that lead the system to the same attractor(s).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01989/20210220/images/medium/jm9b01989_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01989/20210220/images/large/jm9b01989_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Boolean network dynamics. (a) Nodes are colored red or gray based on their “on” or “off” state, respectively. The stepwise evolution of the interactions between nodes determines some sequential steps that are calculated based on a set of rules. The stable state of the network at time step <i>t</i> = S represents an attractor. (b) Attractor landscape. Gray circles represent network states, colored circles represent attractor states. The landscape contains all the possible network states. The sets of states that converge toward an attractor form the basin of that attractor (colored areas). (c) The basins of attractors can be associated with cell phenotypes, and the gene states of the attractors determine the nature of the phenotype.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01989/20210220/images/large/jm9b01989_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01989&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">As a final observation on this kind of models, we point out a claim made by Kauffman in a recent paper, where he revisited his initial hypothesis: answering the question “What is a cell type?”, he wrote “We conclude that there is good evidence that cell types are high dimensional attractors in the dynamics of the genetic regulatory network.”<a onclick="showRef(event, 'ref94'); return false;" href="javascript:void(0);" class="ref ref94">(94)</a> This means that Boolean networks can be considered as simple cell models able to simulate the dynamic cell behavior in response to both physiological and external stimuli and to predict the cell fate in terms of phenotypic output. In other words, we might predict cellular responses to gene activation/inactivation (such as proliferation, apoptosis, cell cycle arrest, and so on) by calculating the attractors characterizing the attractor landscape (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>c). By the way, Huang et al. in a beautiful paper were the first to experimentally demonstrate that an attractor state corresponds to a unique cell phenotype that can be reached via different trajectories (gene expression patterns) in response to different perturbations.<a onclick="showRef(event, 'ref95'); return false;" href="javascript:void(0);" class="ref ref95">(95)</a></div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21">5.2.  Network Dynamics and Drug Discovery</h3><div class="NLM_p">The possible contributions of this network dynamics modeling approach to the field of drug discovery are manifold, as illustrated in the review by Bloomingdale et al.<a onclick="showRef(event, 'ref96'); return false;" href="javascript:void(0);" class="ref ref96">(96)</a> First of all, simulating transcriptional regulation networks can be useful in view of the possibility of postulating new drug targets. For example, De Anda-Jauregui et al.<a onclick="showRef(event, 'ref97'); return false;" href="javascript:void(0);" class="ref ref97">(97)</a> built a simplified Boolean model of the estrogen receptor regulatory network and ran dynamics simulations in both unperturbed and perturbed mode. To simulate the perturbations of the network, the authors systematically overexpressed (on) and knocked out (off) all genes, both singularly and in combination. The results of the attractor landscape analysis pointed out some known gene expression regulators as the proteins chiefly involved in the altered proliferative state, thus demonstrating the suitability of the approach for the exploration of cancer-related systems in the search for new drug targets.</div><div class="NLM_p">Incidentally, it must be noted that to obtain the attractor landscape, the state space of the system should in principle be fully explored by carrying out dynamics simulations starting from all the possible initial states (2<sup><i>N</i></sup>; see above). This is a computationally expensive task, and some methods have been developed to reduce the size of the network while preserving the properties of the system.<a onclick="showRef(event, 'ref96'); return false;" href="javascript:void(0);" class="ref ref96">(96)</a></div><div class="NLM_p">As a further example of the applications of network dynamics, we cite here the work by Choi et al.,<a onclick="showRef(event, 'ref98'); return false;" href="javascript:void(0);" class="ref ref98">(98)</a> who studied the attractor landscape of the p53 regulatory network and used their analysis to identify druggable targets and putative drug combinations. The simulations allowed them to identify critical nodes in the p53 network able to drive normal and breast cancer MCF7 cells toward proliferation, cell cycle arrest, or cell death (the attractors). Then, on the basis of this information, the authors tested different putative therapeutic interventions (simulated as node or link deletions) aimed at pushing MCF7 cells toward apoptosis. They found and experimentally confirmed that the combined treatment with nutlin-3 (Mdm2-p53 PPI disruptor) and Wip1 inhibitor resulted in enhanced cell death, even in the absence of DNA damage. In a subsequent paper, the same group extended their approach to a panel of 83 human cancer cell lines, for which they mapped the genomic alterations of the p53 network. This allowed them to define 45 differently wired p53 networks that were then submitted to the attractor landscape analysis to identify cancer-specific therapeutic interventions.<a onclick="showRef(event, 'ref99'); return false;" href="javascript:void(0);" class="ref ref99">(99)</a></div><div class="NLM_p last">Despite the above-mentioned cases, network dynamics is still an underexplored field in the context of drug research, but its promises make it worthy of great attention, particularly if we consider its suitability to the simulation of complex GRNs, e.g., like those involved in carcinogenesis.<a onclick="showRef(event, 'ref100'); return false;" href="javascript:void(0);" class="ref ref100">(100)</a> In this regard, it is worth mentioning Enrico Capobianco’s observation that “cell state dynamics could be studied by discrete-time Markov models”,<a onclick="showRef(event, 'ref100'); return false;" href="javascript:void(0);" class="ref ref100">(100)</a> a statement that foreshadows a bridge toward some advanced computational approaches in drug design. Actually, in the molecular simulations setting, Markov state models (MSMs) are currently used to investigate the kinetic aspects of protein molecular dynamics,<a onclick="showRef(event, 'ref101 ref102'); return false;" href="javascript:void(0);" class="ref ref101 ref102">(101,102)</a> and their applications in GRN dynamics studies are also known.<a onclick="showRef(event, 'ref103'); return false;" href="javascript:void(0);" class="ref ref103">(103)</a> Indeed, the possibility to use a common mathematical formalism to describe the dynamics of such different systems as a protein and a GRN opens a formidable perspective in view of the multiscale modeling of biological systems for drug discovery.</div></div></div><div id="sec6" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22">6.  Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48468" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48468" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Networks and network analysis tools are rather widespread in biology/chemical biology, not so in chemistry/medicinal chemistry. In any case, if we consider the broad field of drug research, we see an increased use of this kind of <i>in silico</i> modeling approach. In our view, medicinal chemists should be aware of this and take every opportunity to enter this fascinating field, the same way as when the LFER/QSAR paradigm<a onclick="showRef(event, 'ref104'); return false;" href="javascript:void(0);" class="ref ref104">(104)</a> was introduced by Corwin Hansch.</div><div class="NLM_p">In the design of a new drug, the scenario depicted by a system-based network model can be very useful and illuminating, for both practical and theoretical reasons. As regards the former, in perspective, the process of identification and selection of new drug candidates based on a network representation of the target biosystem (“network-driven drug discovery”<a onclick="showRef(event, 'ref105'); return false;" href="javascript:void(0);" class="ref ref105">(105)</a>) might be viewed as a kind of “<i>in silico</i> integrated screening”, in some way simulating and expanding the experimental phenotypic screening (e.g., see Turbine, an artificial intelligence platform for studying cancer, <a href="https://turbine.ai" class="extLink">https://turbine.ai</a>). In fact, on the basis of the available knowledge, one can build the network of interactions governing the cell behavior (an <i>in silico</i> cell) and identify those interventions able to drive the cell toward the desired fate. The network might include several layers of description, from the molecular to the <i>in vivo</i> ones and even above. Considering the already mentioned growing production of molecular, -omic, clinical, etc. data, it seems reasonable to foresee the possibility of realizing the “vertical model integration” proposed by Xie et al.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> through the construction of multipartite networks. This “<i>in silico</i> pharmacology/systems biology continuum”<a onclick="showRef(event, 'ref106'); return false;" href="javascript:void(0);" class="ref ref106">(106)</a> might allow researchers to take into consideration simultaneously most of the effects determining a drug’s action process, thus increasing the reliability of predictions and lowering the costs of drug candidates selection.</div><div class="NLM_p">On another side, from a theoretical point of view, taking a network approach to investigate human diseases means considering the intrinsic complexity of living organisms, which is nowadays accepted as a mandatory standpoint to confront pathologies.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> Coming to drugs and limiting ourselves to the pharmacological way of tackling the disease problem, the holistic view of the target system might help to devise new strategies of design. Peter Csermely gave an interesting example of this by proposing two strategies of network-based drug identification: the central hit strategy (CHS) and the network influence strategy (NIS).<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> In the first one (not new, actually), the aim is to damage a cellular network by hitting a critical node (as in antiinfectious or anticancer therapies), while in the second one, drug(s) should hit influential nodes in order to rewire the diseased network toward its normal state. Of course, key points in the application of these strategies are the analysis of the network topology, and, in the case of NIS, also the network dynamics. Moreover, Csermely convincingly described how such a way of tackling drug-related issues, while taking into account the complexity of the system, might well incorporate current operative concepts, e.g., like PPI inhibitors, multitarget drugs, allosteric drugs, and hit/lead development. In this sense, a network-based view allows one to expand both the starting point and the landscape from which the drug discovery process is considered. Not being merely a technological improvement, it could eventually lead to devising alternative paradigms of pharmacological intervention.</div><div class="NLM_p">A further theme worthy of discussion is how the network modeling approach can be integrated with the well-developed computational techniques currently employed in drug design/discovery, namely, molecular modeling and simulations, and ML. Indeed, ML and deep learning methods are already widely used in computational drug discovery,<a onclick="showRef(event, 'ref107 ref108'); return false;" href="javascript:void(0);" class="ref ref107 ref108">(107,108)</a> and they are perfectly suited to integrate in both the network construction techniques and the network-based prediction approaches. Examples thereof can be found, for example, in the works of Yamanishi et al.<a onclick="showRef(event, 'ref109'); return false;" href="javascript:void(0);" class="ref ref109">(109)</a> (application of a kernel regression method to build a pharmacological space by integrating a chemical space and a genomic space) or Mei et al.<a onclick="showRef(event, 'ref110'); return false;" href="javascript:void(0);" class="ref ref110">(110)</a> (use of a supervised bipartite model incorporating additional training from neighbors in order to predict DTIs for drugs and targets not included in the network). In a recent paper, Zhou et al.<a onclick="showRef(event, 'ref111'); return false;" href="javascript:void(0);" class="ref ref111">(111)</a> proposed a classification for the computational models used for DTI prediction comprising both network-based methods (i.e., the similarity-based algorithms outlined in <a class="ref internalNav" href="#sec4_1" aria-label="section 4.1">section 4.1</a>) that employ algorithms derived from network theory, and ML-based methods that belong to the realm of statistical learning.<a onclick="showRef(event, 'ref112'); return false;" href="javascript:void(0);" class="ref ref112">(112)</a> Taking this point of view, we can depict a drug discovery scenario where, given a complex system to be interpreted or on which predictions have to be made, one can rely on a rich toolbox of methods that are purely network-based (<a class="ref internalNav" href="#sec4_1" aria-label="section 4.1">section 4.1</a>) or ML-based<a onclick="showRef(event, 'ref112'); return false;" href="javascript:void(0);" class="ref ref112">(112)</a> or a merge of both.<a onclick="showRef(event, 'ref87'); return false;" href="javascript:void(0);" class="ref ref87">(87)</a></div><div class="NLM_p">Regarding the inclusion in networks of the atomic-level 3D information obtained by molecular modeling and simulations, this issue has to be carefully considered in order to appreciate its potential in drug discovery. In fact, if we had to do with a drug–target network or a PPI network, why not consider the protein nodes as conformational ensembles? In this way, it might become straightforward to expand each node into the set of protein conformations. In such a case, this would lead to the obtainment of a 3D interactome that could take into account the possibility of a protein binding different partners with different conformations. In an analogous way, in a bipartite drug–target network, different conformations of the same protein could bind different drugs, and should the network describe this feature, it would increase substantially the informative content of the model and its ability to predict potential DTIs. If we imagine such a “3D network model” to be used for the selection of drug candidates, it might constitute a bridge between 3D agnostic network-based prediction and classical target-based molecular design, with a great synergistic potential in terms of efficiency. In a recent commentary,<a onclick="showRef(event, 'ref113'); return false;" href="javascript:void(0);" class="ref ref113">(113)</a> Mih and Palsson discussed the perspectives of a similar scenario in systems biology. They presented several studies where structural information on ligands and proteins were included in genome-scale metabolic network models, thus allowing a more detailed level of description of such systems.</div><div class="NLM_p last">Finally, limitations and open challenges of the network-based approach to drug discovery have to be highlighted. The first problem has already been pointed out in the section on data and databases and pertains to the quality of the data of whatever type they are and whatever source they come from. Moreover, biological data may be incomplete, biased, or sparse, and the languages used to build the databases may be different and/or incompatible. All this limits severely the possibility of even building a network. The second challenge has to do with the dimensions of the data and consequently of the network. Depending on number of nodes and node degrees, the number of links to calculate can increase enormously, and this again limits the possibility of building or analyzing the network, even though some tools allow one to deal with up to millions of nodes and edges (see the review by Pavlopoulos et al.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a>). However, as the analysis becomes more detailed or complex (e.g., network dynamics), the computational demand becomes prohibitive. A way to overcome the computational problems is to distribute the workload on the cloud, a choice that seems the best technological option presently available, while waiting for the effective accessibility of quantum computers.</div></div><div id="sec7" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">7.  Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75463" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75463" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In his “General Systems Theory” book, in 1969, Ludwig von Bertalanffy wrote “In one way or another, we are forced to deal with complexities, with ‘wholes’ or ‘systems’, in all fields of knowledge. This implies a basic re-orientation in scientific thinking.”<a onclick="showRef(event, 'ref114'); return false;" href="javascript:void(0);" class="ref ref114">(114)</a> This statement fits perfectly the idea at the basis of this paper about the way drug research should be approached. Nowadays, we are well aware that the research paradigm in the pharmaceutical field has changed in this sense, and what is presented here is an <i>in silico</i> framework coherent with the new way of thinking drug discovery. By no means does this imply that the classical computational approaches to drug design should be replaced by network theory but on the contrary that they might be more efficiently employed if integrated in a network context. With a careful awareness to the open challenges outlined above, network-based discovery methods will be key players in the next decades drug research.</div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01989" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47231" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47231" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b01989?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01989</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Short presentation of the basic concepts of network theory (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01989/suppl_file/jm9b01989_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01989/suppl_file/jm9b01989_si_001.pdf">jm9b01989_si_001.pdf (864.87 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b01989" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04060" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04060" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Maurizio Recanatini</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmacy and
Biotechnology, Alma Mater Studiorum—University of Bologna, Via Belmeloro 6, I-40126 Bologna, Italy</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0039-0518" title="Orcid link">http://orcid.org/0000-0002-0039-0518</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#fd909c888f94879492d38f989e9c939c89949394bd8893949f92d39489"><span class="__cf_email__" data-cfemail="513c302423382b383e7f233432303f3025383f3811243f38333e7f3825">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chiara Cabrelle</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmacy and
Biotechnology, Alma Mater Studiorum—University of Bologna, Via Belmeloro 6, I-40126 Bologna, Italy</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="bio" rid="ath1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="biographies">Biographies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20502" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20502" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><figure id="ath1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Maurizio Recanatini</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01989&amp;id=BIO-d7e1167-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Maurizio Recanatini</b> is a Professor of Medicinal Chemistry since year 2000, now at the Department of Pharmacy and Biotechnology of the University of Bologna (Italy). His research fields are the design/identification of biologically active molecules and the study of target biological systems through the application of computational tools ranging from QSAR (today, we’d rather say statistical learning) to molecular modeling and simulations. The main therapeutic areas of interest are neurodegeneration, cancer, and channelopathies.</p></figure></div><div class="bio" rid="ath2"><figure id="ath2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Chiara Cabrelle</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01989&amp;id=BIO-d7e1172-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Chiara Cabrelle</b> received her M.S. in Pharmaceutical Chemistry and Technologies from University of Padova, Italy (2017). At present, she is a student in the Ph.D. program of Biotechnological, Biocomputational, Pharmaceutical and Pharmacological Sciences at the University of Bologna (Italy). Her current research interests focus on the application of network approaches in drug discovery in the field of computational medicinal chemistry.</p></figure></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">QSAR</td><td class="NLM_def"><p class="first last">quantitative structure–activity relationship</p></td></tr><tr><td class="NLM_term">ML</td><td class="NLM_def"><p class="first last">machine learning</p></td></tr><tr><td class="NLM_term">NAR</td><td class="NLM_def"><p class="first last"><i>Nucleic Acid Research</i> (journal)</p></td></tr><tr><td class="NLM_term">EHR</td><td class="NLM_def"><p class="first last">electronic health record</p></td></tr><tr><td class="NLM_term">CSN</td><td class="NLM_def"><p class="first last">chemical space network</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">PSN</td><td class="NLM_def"><p class="first last">protein structure network</p></td></tr><tr><td class="NLM_term">GRN</td><td class="NLM_def"><p class="first last">gene regulatory network</p></td></tr><tr><td class="NLM_term">PPI</td><td class="NLM_def"><p class="first last">protein–protein interaction</p></td></tr><tr><td class="NLM_term">DTI</td><td class="NLM_def"><p class="first last">drug–target interaction</p></td></tr><tr><td class="NLM_term">NBI</td><td class="NLM_def"><p class="first last">network-based inference</p></td></tr><tr><td class="NLM_term">RWR</td><td class="NLM_def"><p class="first last">random walk with restart</p></td></tr><tr><td class="NLM_term">MSM</td><td class="NLM_def"><p class="first last">Markov state model</p></td></tr><tr><td class="NLM_term">LFER</td><td class="NLM_def"><p class="first last">linear free-energy relationship</p></td></tr><tr><td class="NLM_term">CHS</td><td class="NLM_def"><p class="first last">central hit strategy</p></td></tr><tr><td class="NLM_term">NIS</td><td class="NLM_def"><p class="first last">network influence strategy</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30211" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30211" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 114 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dugger, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Platt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, D. B.</span></span> <span> </span><span class="NLM_article-title">Drug Development in the Era of Precision Medicine</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">183</span>– <span class="NLM_lpage">196</span>, <span class="refDoi"> DOI: 10.1038/nrd.2017.226</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1038%2Fnrd.2017.226" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=29217837" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFCjur%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=183-196&author=S.+A.+Duggerauthor=A.+Plattauthor=D.+B.+Goldstein&title=Drug+Development+in+the+Era+of+Precision+Medicine&doi=10.1038%2Fnrd.2017.226"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Drug development in the era of precision medicine</span></div><div class="casAuthors">Dugger, Sarah A.; Platt, Adam; Goldstein, David B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">183-196</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  For the past three decades, the use of genomics to inform drug discovery and development pipelines has generated both excitement and scepticism.  Although earlier efforts successfully identified some new drug targets, the overall clin. efficacy of developed drugs has remained unimpressive, owing in large part to the heterogeneous causes of disease.  Recent technol. and anal. advances in genomics, however, have now made it possible to rapidly identify and interpret the genetic variation underlying a single patient's disease, thereby providing a window into patient-specific mechanisms that cause or contribute to disease, which could ultimately enable the 'precise' targeting of these mechanisms.  Here, we first examine and highlight the successes and limitations of the earlier phases of genomics in drug discovery and development.  We then review the current major efforts in precision medicine and discuss the potential broader utility of mechanistically guided treatments going forward.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpshWBfUfoWf7Vg90H21EOLACvtfcHk0lim3QVobpWJOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFCjur%252FM&md5=e06762106391d7646c686e6ef773398a</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2017.226&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2017.226%26sid%3Dliteratum%253Aachs%26aulast%3DDugger%26aufirst%3DS.%2BA.%26aulast%3DPlatt%26aufirst%3DA.%26aulast%3DGoldstein%26aufirst%3DD.%2BB.%26atitle%3DDrug%2520Development%2520in%2520the%2520Era%2520of%2520Precision%2520Medicine%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D183%26epage%3D196%26doi%3D10.1038%2Fnrd.2017.226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span> <span> </span><span class="NLM_article-title">Big Data and Artificial Intelligence Modeling for Drug Discovery</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">573</span>– <span class="NLM_lpage">589</span>, <span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010919-023324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1146%2Fannurev-pharmtox-010919-023324" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=31518513" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslygtL%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2020&pages=573-589&author=H.+Zhu&title=Big+Data+and+Artificial+Intelligence+Modeling+for+Drug+Discovery&doi=10.1146%2Fannurev-pharmtox-010919-023324"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Big Data and Artificial Intelligence Modeling for Drug Discovery</span></div><div class="casAuthors">Zhu, Hao</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">573-589</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">Due to the massive data sets available for drug candidates, modern drug discovery has advanced to the big data era.  Central to this shift is the development of artificial intelligence approaches to implementing innovative modeling based on the dynamic, heterogeneous, and large nature of drug data sets.  As a result, recently developed artificial intelligence approaches such as deep learning and relevant modeling studies provide new solns. to efficacy and safety evaluations of drug candidates based on big data modeling and anal.  The resulting models provided deep insights into the continuum from chem. structure to in vitro, in vivo, and clin. outcomes.  The relevant novel data mining, curation, and management techniques provided crit. support to recent modeling studies.  In summary, the new advancement of artificial intelligence in the big data era has paved the road to future rational drug development and optimization, which will have a significant impact on drug discovery procedures and, eventually, public health.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonOMb5S3akSLVg90H21EOLACvtfcHk0ljNRBMNpF7q5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslygtL%252FI&md5=fa114093fe962c67655d4d7b31d502e9</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010919-023324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010919-023324%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DH.%26atitle%3DBig%2520Data%2520and%2520Artificial%2520Intelligence%2520Modeling%2520for%2520Drug%2520Discovery%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2020%26volume%3D60%26spage%3D573%26epage%3D589%26doi%3D10.1146%2Fannurev-pharmtox-010919-023324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
Der Greef, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mcburney, R. N.</span></span> <span> </span><span class="NLM_article-title">Rescuing Drug Discovery: In Vivo Systems Pathology and Systems Pharmacology</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">961</span>– <span class="NLM_lpage">967</span>, <span class="refDoi"> DOI: 10.1038/nrd1904</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1038%2Fnrd1904" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=16341061" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1yktbjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2005&pages=961-967&author=J.+Van%0ADer+Greefauthor=R.+N.+Mcburney&title=Rescuing+Drug+Discovery%3A+In+Vivo+Systems+Pathology+and+Systems+Pharmacology&doi=10.1038%2Fnrd1904"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Innovation: Rescuing drug discovery: in vivo systems pathology and systems pharmacology</span></div><div class="casAuthors">van der Greef, Jan; McBurney, Robert N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">961-967</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The pharmaceutical industry is currently beleaguered by close scrutiny from the financial community, regulators and the general public.  Productivity, in terms of new drug approvals, has generally been falling for almost a decade and the safety of a no. of highly successful drugs has recently been brought into question.  Here, we discuss whether taking an in vivo systems approach to drug discovery and development could be the paradigm shift that rescues the industry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjvid8S6CQM7Vg90H21EOLACvtfcHk0ljNRBMNpF7q5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1yktbjK&md5=ebebfc6108b47eaa3225a9ec2f2fd317</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrd1904&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1904%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BDer%2BGreef%26aufirst%3DJ.%26aulast%3DMcburney%26aufirst%3DR.%2BN.%26atitle%3DRescuing%2520Drug%2520Discovery%253A%2520In%2520Vivo%2520Systems%2520Pathology%2520and%2520Systems%2520Pharmacology%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2005%26volume%3D4%26spage%3D961%26epage%3D967%26doi%3D10.1038%2Fnrd1904" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolognesi, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minarini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumiatti, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Recanatini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melchiorre, C.</span></span> <span> </span><span class="NLM_article-title">Multi-Target-Directed Ligands to Combat Neurodegenerative Diseases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">347</span>– <span class="NLM_lpage">372</span>, <span class="refDoi"> DOI: 10.1021/jm7009364</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm7009364" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsl2qsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=347-372&author=A.+Cavalliauthor=M.+L.+Bolognesiauthor=A.+Minariniauthor=M.+Rosiniauthor=V.+Tumiattiauthor=M.+Recanatiniauthor=C.+Melchiorre&title=Multi-Target-Directed+Ligands+to+Combat+Neurodegenerative+Diseases&doi=10.1021%2Fjm7009364"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Multi-target-Directed Ligands to Combat Neurodegenerative Diseases</span></div><div class="casAuthors">Cavalli, Andrea; Bolognesi, Maria Laura; Minarini, Anna; Rosini, Michela; Tumiatti, Vincenzo; Recanatini, Maurizio; Melchiorre, Carlo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">347-372</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The authors review the investigations reported in the past decade that illustrate the ability of medicinal chemists to design multi-target-directed ligands (MTDLs) to confront neurodegenerative diseases.  The authors briefly illustrate the multifactorial nature of these diseases before discussing the possible development of MTDLs in light of the known mol. bases of the main neurodegenerative pathologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYY0I0UHEir7Vg90H21EOLACvtfcHk0lgttnXbhhlavA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsl2qsbk%253D&md5=c682235151489316d043cf97f78e56f3</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fjm7009364&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm7009364%26sid%3Dliteratum%253Aachs%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DBolognesi%26aufirst%3DM.%2BL.%26aulast%3DMinarini%26aufirst%3DA.%26aulast%3DRosini%26aufirst%3DM.%26aulast%3DTumiatti%26aufirst%3DV.%26aulast%3DRecanatini%26aufirst%3DM.%26aulast%3DMelchiorre%26aufirst%3DC.%26atitle%3DMulti-Target-Directed%2520Ligands%2520to%2520Combat%2520Neurodegenerative%2520Diseases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D347%26epage%3D372%26doi%3D10.1021%2Fjm7009364" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span> <span> </span><span class="NLM_article-title">Network Pharmacology: The next Paradigm in Drug Discovery</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">682</span>– <span class="NLM_lpage">690</span>, <span class="refDoi"> DOI: 10.1038/nchembio.118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1038%2Fnchembio.118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=18936753" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Kgs7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=682-690&author=A.+L.+Hopkins&title=Network+Pharmacology%3A+The+next+Paradigm+in+Drug+Discovery&doi=10.1038%2Fnchembio.118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Network pharmacology: the next paradigm in drug discovery</span></div><div class="casAuthors">Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">682-690</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The dominant paradigm in drug discovery is the concept of designing maximally selective ligands to act on individual drug targets.  However, many effective drugs act via modulation of multiple proteins rather than single targets.  Advances in systems biol. are revealing a phenotypic robustness and a network structure that strongly suggests that exquisitely selective compds., compared with multi-target drugs, may exhibit lower than desired clin. efficacy.  This new appreciation of the role of polypharmacol. has significant implications for tackling the two major sources of attrition in drug development, efficacy and toxicity.  Integrating network biol. and polypharmacol. holds the promise of expanding the current opportunity space for druggable targets.  However, the rational design of polypharmacol. faces considerable challenges in the need for new methods to validate target combinations and optimize multiple structure-activity relationships while maintaining drug-like properties.  Advances in these areas are creating the foundation of the next paradigm in drug discovery: network pharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqHy4FqkK-h7Vg90H21EOLACvtfcHk0lgttnXbhhlavA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Kgs7fK&md5=03b8be3ddbcab7dc65afd17df7ee8395</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.118%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DNetwork%2520Pharmacology%253A%2520The%2520next%2520Paradigm%2520in%2520Drug%2520Discovery%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2008%26volume%3D4%26spage%3D682%26epage%3D690%26doi%3D10.1038%2Fnchembio.118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hart, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span> <span> </span><span class="NLM_article-title">Providing Data Science Support for Systems Pharmacology and Its Implications to Drug Discovery</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">241</span>– <span class="NLM_lpage">256</span>, <span class="refDoi"> DOI: 10.1517/17460441.2016.1135126</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1517%2F17460441.2016.1135126" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=26689499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtV2rtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=241-256&author=T.+Hartauthor=L.+Xie&title=Providing+Data+Science+Support+for+Systems+Pharmacology+and+Its+Implications+to+Drug+Discovery&doi=10.1517%2F17460441.2016.1135126"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Providing data science support for systems pharmacology and its implications to drug discovery</span></div><div class="casAuthors">Hart, Thomas; Xie, Lei</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">241-256</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">The conventional one-drug-one-target-one-disease drug discovery process has been less successful in tracking multi-genic, multi-faceted complex diseases.  Systems pharmacol. has emerged as a new discipline to tackle the current challenges in drug discovery.  The goal of systems pharmacol. is to transform huge, heterogeneous, and dynamic biol. and clin. data into interpretable and actionable mechanistic models for decision making in drug discovery and patient treatment.  Thus, big data technol. and data science will play an essential role in systems pharmacol.  This paper critically reviews the impact of three fundamental concepts of data science on systems pharmacol.: similarity inference, overfitting avoidance, and disentangling causality from correlation.  The authors then discuss recent advances and future directions in applying the three concepts of data science to drug discovery, with a focus on proteome-wide context-specific quant. drug target deconvolution and personalized adverse drug reaction prediction.  Data science will facilitate reducing the complexity of systems pharmacol. modeling, detecting hidden correlations between complex data sets, and distinguishing causation from correlation.  The power of data science can only be fully realized when integrated with mechanism-based multi-scale modeling that explicitly takes into account the hierarchical organization of biol. systems from nucleic acid to proteins, to mol. interaction networks, to cells, to tissues, to patients, and to populations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEOsRequFdVLVg90H21EOLACvtfcHk0ljFEvBKnZNjPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtV2rtb0%253D&md5=c18c63b8c93fb06e2d7f19dd0dc50a2d</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1517%2F17460441.2016.1135126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2016.1135126%26sid%3Dliteratum%253Aachs%26aulast%3DHart%26aufirst%3DT.%26aulast%3DXie%26aufirst%3DL.%26atitle%3DProviding%2520Data%2520Science%2520Support%2520for%2520Systems%2520Pharmacology%2520and%2520Its%2520Implications%2520to%2520Drug%2520Discovery%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2016%26volume%3D11%26spage%3D241%26epage%3D256%26doi%3D10.1517%2F17460441.2016.1135126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Draizen, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, P. E.</span></span> <span> </span><span class="NLM_article-title">Harnessing Big Data for Systems Pharmacology</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">245</span>– <span class="NLM_lpage">262</span>, <span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010716-104659</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1146%2Fannurev-pharmtox-010716-104659" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=27814027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhslyksb7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=245-262&author=L.+Xieauthor=E.+J.+Draizenauthor=P.+E.+Bourne&title=Harnessing+Big+Data+for+Systems+Pharmacology&doi=10.1146%2Fannurev-pharmtox-010716-104659"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Harnessing Big Data for Systems Pharmacology</span></div><div class="casAuthors">Xie, Lei; Draizen, Eli J.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">245-262</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">Systems pharmacol. aims to holistically understand mechanisms of drug actions to support drug discovery and clin. practice.  Systems pharmacol. modeling (SPM) is data driven.  It integrates an exponentially growing amt. of data at multiple scales (genetic, mol., cellular, organismal, and environmental).  The goal of SPM is to develop mechanistic or predictive multiscale models that are interpretable and actionable.  The current explosions in genomics and other omics data, as well as the tremendous advances in big data technologies, have already enabled biologists to generate novel hypotheses and gain new knowledge through computational models of genome-wide, heterogeneous, and dynamic data sets.  More work is needed to interpret and predict a drug response phenotype, which is dependent on many known and unknown factors.  To gain a comprehensive understanding of drug actions, SPM requires close collaborations between domain experts from diverse fields and integration of heterogeneous models from biophysics, mathematics, statistics, machine learning, and semantic webs.  This creates challenges in model management, model integration, model translation, and knowledge integration.  In this review, we discuss several emergent issues in SPM and potential solns. using big data technol. and analytics.  The concurrent development of high-throughput techniques, cloud computing, data science, and the semantic web will likely allow SPM to be findable, accessible, interoperable, reusable, reliable, interpretable, and actionable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrojceq7LgsT7Vg90H21EOLACvtfcHk0ljFEvBKnZNjPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhslyksb7M&md5=120756181391dce7785bfcf0485ff10b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010716-104659&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010716-104659%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DDraizen%26aufirst%3DE.%2BJ.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DHarnessing%2520Big%2520Data%2520for%2520Systems%2520Pharmacology%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2017%26volume%3D57%26spage%3D245%26epage%3D262%26doi%3D10.1146%2Fannurev-pharmtox-010716-104659" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gentili, P. L.</span></span> <i>Untangling Complex Systems. A Grand Challenge for Science</i>; <span class="NLM_publisher-name">CRC Press</span>: <span class="NLM_publisher-loc">Boca Raton, FL, U.S.</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=P.+L.+Gentili&title=Untangling+Complex+Systems.+A+Grand+Challenge+for+Science"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DGentili%26aufirst%3DP.%2BL.%26btitle%3DUntangling%2520Complex%2520Systems.%2520A%2520Grand%2520Challenge%2520for%2520Science%26pub%3DCRC%2520Press%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Loscalzo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barabasi, A. L.</span></span> <span> </span><span class="NLM_article-title">Systems Biology and the Future of Medicine</span>. <i>Wiley Interdiscip. Rev. Syst. Biol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">619</span>– <span class="NLM_lpage">627</span>, <span class="refDoi"> DOI: 10.1002/wsbm.144</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1002%2Fwsbm.144" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=21928407" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A280%3ADC%252BC3MfovVShuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=619-627&author=J.+Loscalzoauthor=A.+L.+Barabasi&title=Systems+Biology+and+the+Future+of+Medicine&doi=10.1002%2Fwsbm.144"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Systems biology and the future of medicine</span></div><div class="casAuthors">Loscalzo Joseph; Barabasi Albert-Laszlo</div><div class="citationInfo"><span class="NLM_cas:title">Wiley interdisciplinary reviews. Systems biology and medicine</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">619-27</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Contemporary views of human disease are based on simple correlation between clinical syndromes and pathological analysis dating from the late 19th century.  Although this approach to disease diagnosis, prognosis, and treatment has served the medical establishment and society well for many years, it has serious shortcomings for the modern era of the genomic medicine that stem from its reliance on reductionist principles of experimentation and analysis.  Quantitative, holistic systems biology applied to human disease offers a unique approach for diagnosing established disease, defining disease predilection, and developing individualized (personalized) treatment strategies that can take full advantage of modern molecular pathobiology and the comprehensive data sets that are rapidly becoming available for populations and individuals.  In this way, systems pathobiology offers the promise of redefining our approach to disease and the field of medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTkNpHR0NjEf825P7XSLhESfW6udTcc2eaLR-3RPdVMrLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MfovVShuw%253D%253D&md5=f9c57e7f65de8529def4b2aec3692ab3</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1002%2Fwsbm.144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fwsbm.144%26sid%3Dliteratum%253Aachs%26aulast%3DLoscalzo%26aufirst%3DJ.%26aulast%3DBarabasi%26aufirst%3DA.%2BL.%26atitle%3DSystems%2520Biology%2520and%2520the%2520Future%2520of%2520Medicine%26jtitle%3DWiley%2520Interdiscip.%2520Rev.%2520Syst.%2520Biol.%2520Med.%26date%3D2011%26volume%3D3%26spage%3D619%26epage%3D627%26doi%3D10.1002%2Fwsbm.144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Euler, L.</span></span> <span> </span><span class="NLM_article-title">Solutio Problematis Ad Geometriam Situs Pertinentis. (1741)</span>.  <i>Euler Archive—All Works</i>; No. 53, <a href="https://scholarlycommons.pacific.edu/euler-works/53" class="extLink">https://scholarlycommons.pacific.edu/euler-works/53</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Euler%2C+L.+Solutio+Problematis+Ad+Geometriam+Situs+Pertinentis.+%281741%29.+Euler+Archive%E2%80%94All+Works%3B+No.+53%2C+https%3A%2F%2Fscholarlycommons.pacific.edu%2Feuler-works%2F53."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DEuler%26aufirst%3DL.%26atitle%3DSolutio%2520Problematis%2520Ad%2520Geometriam%2520Situs%2520Pertinentis.%2520%25281741%2529%26btitle%3DEuler%2520Archive%25E2%2580%2594All%2520Works" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Newman, M. E. J.</span></span> <span> </span><span class="NLM_article-title">The Structure and Function of Complex Networks</span>. <i>SIAM Rev.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">167</span>– <span class="NLM_lpage">256</span>, <span class="refDoi"> DOI: 10.1137/S003614450342480</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1137%2FS003614450342480" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2003&pages=167-256&author=M.+E.+J.+Newman&title=The+Structure+and+Function+of+Complex+Networks&doi=10.1137%2FS003614450342480"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1137%2FS003614450342480&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1137%252FS003614450342480%26sid%3Dliteratum%253Aachs%26aulast%3DNewman%26aufirst%3DM.%2BE.%2BJ.%26atitle%3DThe%2520Structure%2520and%2520Function%2520of%2520Complex%2520Networks%26jtitle%3DSIAM%2520Rev.%26date%3D2003%26volume%3D45%26spage%3D167%26epage%3D256%26doi%3D10.1137%2FS003614450342480" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Estrada, E.</span></span> <span> </span><span class="NLM_article-title">Graph and Network Theory in Physics</span>. <i>arXiv</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_fpage">1302.4378</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=1302.4378&author=E.+Estrada&title=Graph+and+Network+Theory+in+Physics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEstrada%26aufirst%3DE.%26atitle%3DGraph%2520and%2520Network%2520Theory%2520in%2520Physics%26jtitle%3DarXiv%26date%3D2013%26spage%3D1302.4378" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barabási, A.-L.</span></span> <i>Network Science</i>; <span class="NLM_publisher-name">Cambridge University Press</span>: <span class="NLM_publisher-loc">Cambridge, U.K.</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=A.-L.+Barab%C3%A1si&title=Network+Science"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DBarab%25C3%25A1si%26aufirst%3DA.-L.%26btitle%3DNetwork%2520Science%26pub%3DCambridge%2520University%2520Press%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Csermely, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korcsmáros, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiss, H. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">London, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span> <span> </span><span class="NLM_article-title">Structure and Dynamics of Molecular Networks: A Novel Paradigm of Drug Discovery: A Comprehensive Review</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">333</span>– <span class="NLM_lpage">408</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2013.01.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1016%2Fj.pharmthera.2013.01.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=23384594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC3sXls1CgtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2013&pages=333-408&author=P.+Csermelyauthor=T.+Korcsm%C3%A1rosauthor=H.+J.+M.+Kissauthor=G.+Londonauthor=R.+Nussinov&title=Structure+and+Dynamics+of+Molecular+Networks%3A+A+Novel+Paradigm+of+Drug+Discovery%3A+A+Comprehensive+Review&doi=10.1016%2Fj.pharmthera.2013.01.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and dynamics of molecular networks: A novel paradigm of drug discovery</span></div><div class="casAuthors">Csermely, Peter; Korcsmaros, Tamas; Kiss, Huba J. M.; London, Gabor; Nussinov, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">333-408</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Despite considerable progress in genome- and proteome-based high-throughput screening methods and in rational drug design, the increase in approved drugs in the past decade did not match the increase of drug development costs.  Network description and anal. not only give a systems-level understanding of drug action and disease complexity, but can also help to improve the efficiency of drug design.  We give a comprehensive assessment of the anal. tools of network topol. and dynamics.  The state-of-the-art use of chem. similarity, protein structure, protein-protein interaction, signaling, genetic interaction and metabolic networks in the discovery of drug targets is summarized.  We propose that network targeting follows two basic strategies.  The "central hit strategy" selectively targets central nodes/edges of the flexible networks of infectious agents or cancer cells to kill them.  The "network influence strategy" works against other diseases, where an efficient reconfiguration of rigid networks needs to be achieved by targeting the neighbors of central nodes/edges.  It is shown how network techniques can help in the identification of single-target, edgetic, multi-target and allo-network drug target candidates.  We review the recent boom in network methods helping hit identification, lead selection optimizing drug efficacy, as well as minimizing side-effects and drug toxicity.  Successful network-based drug development strategies are shown through the examples of infections, cancer, metabolic diseases, neurodegenerative diseases and aging.  Summarizing >1200 refs. we suggest an optimized protocol of network-aided drug development, and provide a list of systems-level hallmarks of drug quality.  Finally, we highlight network-related drug development trends helping to achieve these hallmarks by a cohesive, global approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFqNviuDk7kbVg90H21EOLACvtfcHk0lgpm06IAKdvSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXls1CgtLk%253D&md5=d70c48a1572815c1b4905cc6e8cdfaa6</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2013.01.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2013.01.016%26sid%3Dliteratum%253Aachs%26aulast%3DCsermely%26aufirst%3DP.%26aulast%3DKorcsm%25C3%25A1ros%26aufirst%3DT.%26aulast%3DKiss%26aufirst%3DH.%2BJ.%2BM.%26aulast%3DLondon%26aufirst%3DG.%26aulast%3DNussinov%26aufirst%3DR.%26atitle%3DStructure%2520and%2520Dynamics%2520of%2520Molecular%2520Networks%253A%2520A%2520Novel%2520Paradigm%2520of%2520Drug%2520Discovery%253A%2520A%2520Comprehensive%2520Review%26jtitle%3DPharmacol.%2520Ther.%26date%3D2013%26volume%3D138%26spage%3D333%26epage%3D408%26doi%3D10.1016%2Fj.pharmthera.2013.01.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span> <span> </span><span class="NLM_article-title">Freely Accessible Chemical Database Resources of Compounds for In Silico Drug Discovery</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">7581</span>– <span class="NLM_lpage">7597</span>, <span class="refDoi"> DOI: 10.2174/0929867325666180508100436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.2174%2F0929867325666180508100436" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2020&pages=7581-7597&author=J.+Yangauthor=D.+Wangauthor=C.+Jiaauthor=M.+Wangauthor=G.+Haoauthor=G.+Yang&title=Freely+Accessible+Chemical+Database+Resources+of+Compounds+for+In+Silico+Drug+Discovery&doi=10.2174%2F0929867325666180508100436"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.2174%2F0929867325666180508100436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F0929867325666180508100436%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DJia%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DHao%26aufirst%3DG.%26aulast%3DYang%26aufirst%3DG.%26atitle%3DFreely%2520Accessible%2520Chemical%2520Database%2520Resources%2520of%2520Compounds%2520for%2520In%2520Silico%2520Drug%2520Discovery%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2020%26volume%3D26%26spage%3D7581%26epage%3D7597%26doi%3D10.2174%2F0929867325666180508100436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mendez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bento, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Veij, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Félix, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magariños, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosquera, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mutowo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowotka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordillo-Marañón, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Junco, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mugumbate, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Lopez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosc, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radoux, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segura-Cabrera, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hersey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span> <span> </span><span class="NLM_article-title">ChEMBL: Towards Direct Deposition of Bioassay Data</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D930</span>– <span class="NLM_lpage">D940</span>, <span class="refDoi"> DOI: 10.1093/nar/gky1075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fnar%2Fgky1075" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=30398643" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1Gqtr%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D930-D940&issue=D1&author=D.+Mendezauthor=A.+Gaultonauthor=A.+P.+Bentoauthor=J.+Chambersauthor=M.+De+Veijauthor=E.+F%C3%A9lixauthor=M.+P.+Magari%C3%B1osauthor=J.+F.+Mosqueraauthor=P.+Mutowoauthor=M.+Nowotkaauthor=M.+Gordillo-Mara%C3%B1%C3%B3nauthor=F.+Hunterauthor=L.+Juncoauthor=G.+Mugumbateauthor=M.+Rodriguez-Lopezauthor=F.+Atkinsonauthor=N.+Boscauthor=C.+J.+Radouxauthor=A.+Segura-Cabreraauthor=A.+Herseyauthor=A.+R.+Leach&title=ChEMBL%3A+Towards+Direct+Deposition+of+Bioassay+Data&doi=10.1093%2Fnar%2Fgky1075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">ChEMBL: towards direct deposition of bioassay data</span></div><div class="casAuthors">Mendez, David; Gaulton, Anna; Bento, A. Patricia; Chambers, Jon; De Veij, Marleen; Felix, Eloy; Magarinos, Maria Paula; Mosquera, Juan F.; Mutowo, Prudence; Nowotka, Michal; Gordillo-Maranon, Maria; Hunter, Fiona; Junco, Laura; Mugumbate, Grace; Rodriguez-Lopez, Milagros; Atkinson, Francis; Bosc, Nicolas; Radoux, Chris J.; Cabrera, Aldo Segura; Hersey, Anne; Leach, Andrew R.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D930-D940</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is a large, open-access bioactivity database, previously described in the 2012, 2014 and 2017 Nucleic Acids Research Database Issues.  In the last two years, several important improvements have been made to the database and are described here.  These include more robust capture and representation of assay details; a new data deposition system, allowing updating of data sets and deposition of supplementary data; and a completely redesigned web interface, with enhanced search and filtering capabilities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqM7KYSXc4jerVg90H21EOLACvtfcHk0liiuS1Kfsu6FA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1Gqtr%252FF&md5=d80660ce87d5b3e0773594e4fddec046</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky1075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky1075%26sid%3Dliteratum%253Aachs%26aulast%3DMendez%26aufirst%3DD.%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDe%2BVeij%26aufirst%3DM.%26aulast%3DF%25C3%25A9lix%26aufirst%3DE.%26aulast%3DMagari%25C3%25B1os%26aufirst%3DM.%2BP.%26aulast%3DMosquera%26aufirst%3DJ.%2BF.%26aulast%3DMutowo%26aufirst%3DP.%26aulast%3DNowotka%26aufirst%3DM.%26aulast%3DGordillo-Mara%25C3%25B1%25C3%25B3n%26aufirst%3DM.%26aulast%3DHunter%26aufirst%3DF.%26aulast%3DJunco%26aufirst%3DL.%26aulast%3DMugumbate%26aufirst%3DG.%26aulast%3DRodriguez-Lopez%26aufirst%3DM.%26aulast%3DAtkinson%26aufirst%3DF.%26aulast%3DBosc%26aufirst%3DN.%26aulast%3DRadoux%26aufirst%3DC.%2BJ.%26aulast%3DSegura-Cabrera%26aufirst%3DA.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26atitle%3DChEMBL%253A%2520Towards%2520Direct%2520Deposition%2520of%2520Bioassay%2520Data%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2019%26volume%3D47%26issue%3DD1%26spage%3DD930%26epage%3DD940%26doi%3D10.1093%2Fnar%2Fgky1075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gindulyte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoemaker, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiessen, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaslavsky, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolton, E. E.</span></span> <span> </span><span class="NLM_article-title">PubChem 2019 Update: Improved Access to Chemical Data</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D1102</span>– <span class="NLM_lpage">D1109</span>, <span class="refDoi"> DOI: 10.1093/nar/gky1033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fnar%2Fgky1033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=30371825" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D1102-D1109&issue=D1&author=S.+Kimauthor=J.+Chenauthor=T.+Chengauthor=A.+Gindulyteauthor=J.+Heauthor=S.+Heauthor=Q.+Liauthor=B.+A.+Shoemakerauthor=P.+A.+Thiessenauthor=B.+Yuauthor=L.+Zaslavskyauthor=J.+Zhangauthor=E.+E.+Bolton&title=PubChem+2019+Update%3A+Improved+Access+to+Chemical+Data&doi=10.1093%2Fnar%2Fgky1033"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky1033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky1033%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DT.%26aulast%3DGindulyte%26aufirst%3DA.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DShoemaker%26aufirst%3DB.%2BA.%26aulast%3DThiessen%26aufirst%3DP.%2BA.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DZaslavsky%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DBolton%26aufirst%3DE.%2BE.%26atitle%3DPubChem%25202019%2520Update%253A%2520Improved%2520Access%2520to%2520Chemical%2520Data%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2019%26volume%3D47%26issue%3DD1%26spage%3DD1102%26epage%3DD1109%26doi%3D10.1093%2Fnar%2Fgky1033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wishart, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feunang, Y. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sajed, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayeeda, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Assempour, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iynkkaran, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maciejewski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gale, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, M.</span></span> <span> </span><span class="NLM_article-title">DrugBank 5.0: A Major Update to the DrugBank Database for 2018</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D1074</span>– <span class="NLM_lpage">D1082</span>, <span class="refDoi"> DOI: 10.1093/nar/gkx1037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fnar%2Fgkx1037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=29126136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlGisbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D1074-D1082&issue=D1&author=D.+S.+Wishartauthor=Y.+D.+Feunangauthor=A.+C.+Guoauthor=E.+J.+Loauthor=A.+Marcuauthor=J.+R.+Grantauthor=T.+Sajedauthor=D.+Johnsonauthor=C.+Liauthor=Z.+Sayeedaauthor=N.+Assempourauthor=I.+Iynkkaranauthor=Y.+Liuauthor=A.+Maciejewskiauthor=N.+Galeauthor=A.+Wilsonauthor=L.+Chinauthor=R.+Cummingsauthor=D.+Leauthor=A.+Ponauthor=C.+Knoxauthor=M.+Wilson&title=DrugBank+5.0%3A+A+Major+Update+to+the+DrugBank+Database+for+2018&doi=10.1093%2Fnar%2Fgkx1037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">DrugBank 5.0: a major update to the DrugBank database for 2018</span></div><div class="casAuthors">Wishart, David S.; Feunang, Yannick D.; Guo, An C.; Lo, Elvis J.; Marcu, Ana; Grant, Jason R.; Sajed, Tanvir; Johnson, Daniel; Li, Carin; Sayeeda, Zinat; Assempour, Nazanin; Iynkkaran, Ithayavani; Liu, Yifeng; Maciejewski, Adam; Gale, Nicola; Wilson, Alex; Chin, Lucy; Cummings, Ryan; Le, Diana; Pon, Allison; Knox, Craig; Wilson, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1074-D1082</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">DrugBank is a web-enabled database contg. comprehensivemol. information about drugs, their mechanisms, their interactions and their targets.  First described in 2006, Drug- Bank has continued to evolve over the past 12 years in response to marked improvements to web stds. and changing needs for drug research and development.  This year's update, DrugBank 5.0, represents the most significant upgrade to the database in more than 10 years.  In many cases, existing data content has grown by 100% or more over the last update.  For instance, the total no. of investigational drugs in the database has grown by almost 300%, the no. of drug-drug interactions has grown by nearly 600% and the no. of SNP-assocd. drug effects has grown more than 3000%.  Significant improvements have been made to the quantity, quality and consistency of drug indications, drug binding data as well as drug-drug and drug-food interactions.  A great deal of brand new data have also been added to DrugBank 5.0.  This includes information on the influence of hundreds of drugs on metabolite levels (pharmacometabolomics), gene expression levels (pharmacotranscriptomics) and protein expression levels (pharmacoproteomics).  New data have also been added on the status of hundreds of newdrug clin. trials and existing drug repurposing trials.  Many other important improvements in the content, interface and performance of the DrugBank website have been made and these should greatly enhance its ease of use, utility and potential applications in many areas of pharmacol. research, pharmaceutical science and drug education.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrc8rfBoMOOyLVg90H21EOLACvtfcHk0limyEg0uKYH0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlGisbvI&md5=986b28c7ea546596a26dd3ba38f05fee</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx1037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx1037%26sid%3Dliteratum%253Aachs%26aulast%3DWishart%26aufirst%3DD.%2BS.%26aulast%3DFeunang%26aufirst%3DY.%2BD.%26aulast%3DGuo%26aufirst%3DA.%2BC.%26aulast%3DLo%26aufirst%3DE.%2BJ.%26aulast%3DMarcu%26aufirst%3DA.%26aulast%3DGrant%26aufirst%3DJ.%2BR.%26aulast%3DSajed%26aufirst%3DT.%26aulast%3DJohnson%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DSayeeda%26aufirst%3DZ.%26aulast%3DAssempour%26aufirst%3DN.%26aulast%3DIynkkaran%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DMaciejewski%26aufirst%3DA.%26aulast%3DGale%26aufirst%3DN.%26aulast%3DWilson%26aufirst%3DA.%26aulast%3DChin%26aufirst%3DL.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DLe%26aufirst%3DD.%26aulast%3DPon%26aufirst%3DA.%26aulast%3DKnox%26aufirst%3DC.%26aulast%3DWilson%26aufirst%3DM.%26atitle%3DDrugBank%25205.0%253A%2520A%2520Major%2520Update%2520to%2520the%2520DrugBank%2520Database%2520for%25202018%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26issue%3DD1%26spage%3DD1074%26epage%3DD1082%26doi%3D10.1093%2Fnar%2Fgkx1037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pence, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, A.</span></span> <span> </span><span class="NLM_article-title">ChemSpider: An Online Chemical Information Resource</span>. <i>J. Chem. Educ.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">1123</span>– <span class="NLM_lpage">1124</span>, <span class="refDoi"> DOI: 10.1021/ed100697w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ed100697w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtV2jtr%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2010&pages=1123-1124&author=H.+E.+Penceauthor=A.+Williams&title=ChemSpider%3A+An+Online+Chemical+Information+Resource&doi=10.1021%2Fed100697w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">ChemSpider: An Online Chemical Information Resource</span></div><div class="casAuthors">Pence, Harry E.; Williams, Antony</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Education</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1123-1124</span>CODEN:
                <span class="NLM_cas:coden">JCEDA8</span>;
        ISSN:<span class="NLM_cas:issn">0021-9584</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society and Division of Chemical Education, Inc.</span>)
        </div><div class="casAbstract">ChemSpider is a free, online chem. database offering access to phys. and chem. properties, mol. structure, spectral data, synthetic methods, safety information, and nomenclature for almost 25 million unique chem. compds. sourced and linked to almost 400 sep. data sources on the Web.  ChemSpider is quickly becoming the primary chem. Internet portal and it can be very useful for both chem. teaching and research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVUvg_k0Vg6bVg90H21EOLACvtfcHk0lhuKHjw5mEGPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtV2jtr%252FF&md5=c7c75704f57cf699598c0250001ebb48</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fed100697w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fed100697w%26sid%3Dliteratum%253Aachs%26aulast%3DPence%26aufirst%3DH.%2BE.%26aulast%3DWilliams%26aufirst%3DA.%26atitle%3DChemSpider%253A%2520An%2520Online%2520Chemical%2520Information%2520Resource%26jtitle%3DJ.%2520Chem.%2520Educ.%26date%3D2010%26volume%3D87%26spage%3D1123%26epage%3D1124%26doi%3D10.1021%2Fed100697w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fourches, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratov, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tropsha, A.</span></span> <span> </span><span class="NLM_article-title">Trust, but Verify: On the Importance of Chemical Structure Curation in Cheminformatics and QSAR Modeling Research</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1189</span>– <span class="NLM_lpage">1204</span>, <span class="refDoi"> DOI: 10.1021/ci100176x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100176x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvVeitLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1189-1204&author=D.+Fourchesauthor=E.+Muratovauthor=A.+Tropsha&title=Trust%2C+but+Verify%3A+On+the+Importance+of+Chemical+Structure+Curation+in+Cheminformatics+and+QSAR+Modeling+Research&doi=10.1021%2Fci100176x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Trust, But Verify: On the Importance of Chemical Structure Curation in Cheminformatics and QSAR Modeling Research</span></div><div class="casAuthors">Fourches, Denis; Muratov, Eugene; Tropsha, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1189-1204</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">With the recent advent of high-throughput technologies for both compd. synthesis and biol. screening, there is no shortage of publicly or com. available data sets and databases that can be used for computational drug discovery applications.  Rapid growth of large, publicly available databases (such as PubChem or ChemSpider contg. more than 20 million mol. records each) enabled by exptl. projects such as NIH's Mol. Libraries and Imaging Initiative provides new opportunities for the development of chemoinformatics methodologies and their application to knowledge discovery in mol. databases.  A fundamental assumption of any chemoinformatics study is the correctness of the input data generated by exptl. scientists and available in various data sets.  In another recent study, the authors investigated several public and com. databases to calc. their error rates: the latter were ranging from 0.1 to 3.4% depending on the database.  How significant is the problem of accurate structure representation (given that the error rates in current databases may appear relatively low) since it concerns exploratory chemoinformatics and mol. modeling research.  Recent investigations by a large group of collaborators from six labs. have clearly demonstrated that the type of chem. descriptors has much greater influence on the prediction performance of QSAR models than the nature of model optimization techniques.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqpd35PIjh-7Vg90H21EOLACvtfcHk0lhuKHjw5mEGPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvVeitLk%253D&md5=4c19d67a67094bc595f5940157ff9a2d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fci100176x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100176x%26sid%3Dliteratum%253Aachs%26aulast%3DFourches%26aufirst%3DD.%26aulast%3DMuratov%26aufirst%3DE.%26aulast%3DTropsha%26aufirst%3DA.%26atitle%3DTrust%252C%2520but%2520Verify%253A%2520On%2520the%2520Importance%2520of%2520Chemical%2520Structure%2520Curation%2520in%2520Cheminformatics%2520and%2520QSAR%2520Modeling%2520Research%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1189%26epage%3D1204%26doi%3D10.1021%2Fci100176x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fourches, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratov, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tropsha, A.</span></span> <span> </span><span class="NLM_article-title">Curation of Chemogenomics Data</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">535</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1881</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1038%2Fnchembio.1881" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=26196763" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=535&author=D.+Fourchesauthor=E.+Muratovauthor=A.+Tropsha&title=Curation+of+Chemogenomics+Data&doi=10.1038%2Fnchembio.1881"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Curation of chemogenomics data</span></div><div class="casAuthors">Fourches, Denis; Muratov, Eugene; Tropsha, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">535</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">With the rapid accumulation of data in all areas of chem. biol. research, scientists rely increasingly on historical chemogenomics data and computational models to guide small mol. bioactivity screens and chem. probe development.  However, there is a growing public concern about the frequent irreproducibility of exptl. data reported in peer-reviewed scientific publications.  As one means of addressing the data quality problem, we propose a general chem. and biol. data curation workflow that relies on existing cheminformatics approaches to flag, and in some cases correct, possibly erroneous entries in large chemogenomics datasets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPsQj89bRYDbVg90H21EOLACvtfcHk0liQDQeqGF5gbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zL&md5=c45d4b8fb2f3570696be7818025cc4c7</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1881&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1881%26sid%3Dliteratum%253Aachs%26aulast%3DFourches%26aufirst%3DD.%26aulast%3DMuratov%26aufirst%3DE.%26aulast%3DTropsha%26aufirst%3DA.%26atitle%3DCuration%2520of%2520Chemogenomics%2520Data%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D535%26doi%3D10.1038%2Fnchembio.1881" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rigden, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernández, X. M.</span></span> <span> </span><span class="NLM_article-title">The 26th Annual Nucleic Acids Research Database Issue and Molecular Biology Database Collection</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D1</span>– <span class="NLM_lpage">D7</span>, <span class="refDoi"> DOI: 10.1093/nar/gky1267</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fnar%2Fgky1267" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=30626175" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1GqtrjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D1-D7&issue=D1&author=D.+J.+Rigdenauthor=X.+M.+Fern%C3%A1ndez&title=The+26th+Annual+Nucleic+Acids+Research+Database+Issue+and+Molecular+Biology+Database+Collection&doi=10.1093%2Fnar%2Fgky1267"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The 26th annual nucleic acids research database issue and molecular biology database collection</span></div><div class="casAuthors">Rigden, Daniel J.; Fernandez, Xose M.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1-D7</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  The 2019 Nucleic Acids Research (NAR) Database Issue contains 168 papers spanning mol. biol.  Among them, 64 are new and another 92 are updates describing resources that appeared in the Issue previously.  The remaining 12 are updates on databases most recently published elsewhere.  This Issue contains two Breakthrough articles, on the Virtual Metabolic Human (VMH) database which links human and gut microbiota metab. with diet and disease, and Vibrism DB, a database of mouse brain anatomy and gene (co-)expression with sophisticated visualization and session sharing.  Major returning nucleic acid databases include RNAcentral, miRBase and LncRNA2Target.  Protein sequence databases include UniProtKB, InterPro and Pfam, while wwPDB and RCSB cover protein structure.  STRING and KEGG update in the section on metab. and pathways.  Microbial genomes are covered by IMG/M and resources for human and model organism genomics include Ensembl, UCSC Genome Browser, GENCODE and Flybase.  Genomic variation and disease are well-covered by GWAS Catalog, PopHumanScan, OMIM and COSMIC, CADD being another major newcomer.  Major new proteomics resources reporting here include iProX and jPOSTdb.  The entire database issue is freely available online on the NAR website.  The NAR online Mol. Biol. Database Collection has been updated, reviewing 506 entries, adding 66 new resources and eliminating 147 discontinued URLs, bringing the current total to 1613 databases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoc8TUMtAaxirVg90H21EOLACvtfcHk0liQDQeqGF5gbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1GqtrjF&md5=696524ae4195a18ef6eb493310d4f828</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky1267&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky1267%26sid%3Dliteratum%253Aachs%26aulast%3DRigden%26aufirst%3DD.%2BJ.%26aulast%3DFern%25C3%25A1ndez%26aufirst%3DX.%2BM.%26atitle%3DThe%252026th%2520Annual%2520Nucleic%2520Acids%2520Research%2520Database%2520Issue%2520and%2520Molecular%2520Biology%2520Database%2520Collection%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2019%26volume%3D47%26issue%3DD1%26spage%3DD1%26epage%3DD7%26doi%3D10.1093%2Fnar%2Fgky1267" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Letunic, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bork, P.</span></span> <span> </span><span class="NLM_article-title">20 Years of the SMART Protein Domain Annotation Resource</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D493</span>– <span class="NLM_lpage">D496</span>, <span class="refDoi"> DOI: 10.1093/nar/gkx922</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fnar%2Fgkx922" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=29040681" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlGju7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D493-D496&issue=D1&author=I.+Letunicauthor=P.+Bork&title=20+Years+of+the+SMART+Protein+Domain+Annotation+Resource&doi=10.1093%2Fnar%2Fgkx922"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">20 years of the SMART protein domain annotation resource</span></div><div class="casAuthors">Letunic, Ivica; Bork, Peer</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D493-D496</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  SMART (Simple Modular Architecture Research Tool) is a web resource for the identification and annotation of protein domains and the anal. of protein domain architectures.  SMART version 8 contains manually curated models for more than 1300 protein domains, with approx. 100 new models added since our last update article (1).  The underlying protein databases were synchronized with UniProt (2), Ensembl (3) and STRING (4), doubling the total no. of annotated domains and other protein features to more than 200 million.  In its 20th year, the SMART anal. results pages have been streamlined again and its information sources have been updated.  SMART's vector based display engine has been extended to all protein schematics in SMART and rewritten to use the latest web technologies.  The internal full text search engine has been redesigned and updated, resulting in greatly increased search speed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouFalEXQtj7LVg90H21EOLACvtfcHk0lh1B13YtjZEUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlGju7rN&md5=13be1a91226804c04b077b22e3b7d622</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx922&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx922%26sid%3Dliteratum%253Aachs%26aulast%3DLetunic%26aufirst%3DI.%26aulast%3DBork%26aufirst%3DP.%26atitle%3D20%2520Years%2520of%2520the%2520SMART%2520Protein%2520Domain%2520Annotation%2520Resource%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26issue%3DD1%26spage%3DD493%26epage%3DD496%26doi%3D10.1093%2Fnar%2Fgkx922" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">The
UniProt Consortium.</span> <span> </span><span class="NLM_article-title">UniProt:
A Worldwide Hub of Protein Knowledge</span>. <i>Nucleic
Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D506</span>– <span class="NLM_lpage">D515</span>, <span class="refDoi"> DOI: 10.1093/nar/gky1049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fnar%2Fgky1049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=30395287" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D506-D515&issue=D1&author=The%0AUniProt+Consortium.&title=UniProt%3A%0AA+Worldwide+Hub+of+Protein+Knowledge&doi=10.1093%2Fnar%2Fgky1049"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky1049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky1049%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DUniProt%253A%250AA%2520Worldwide%2520Hub%2520of%2520Protein%2520Knowledge%26jtitle%3DNucleic%250AAcids%2520Res.%26date%3D2019%26volume%3D47%26issue%3DD1%26spage%3DD506%26epage%3DD515%26doi%3D10.1093%2Fnar%2Fgky1049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pándy-Szekeres, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munk, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsonkov, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mordalski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harpsøe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauser, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bojarski, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gloriam, D. E.</span></span> <span> </span><span class="NLM_article-title">GPCRdb in 2018: Adding GPCR Structure Models and Ligands</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D440</span>– <span class="NLM_lpage">D446</span>, <span class="refDoi"> DOI: 10.1093/nar/gkx1109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fnar%2Fgkx1109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=29155946" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlGisLbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D440-D446&issue=D1&author=G.+P%C3%A1ndy-Szekeresauthor=C.+Munkauthor=T.+M.+Tsonkovauthor=S.+Mordalskiauthor=K.+Harps%C3%B8eauthor=A.+S.+Hauserauthor=A.+J.+Bojarskiauthor=D.+E.+Gloriam&title=GPCRdb+in+2018%3A+Adding+GPCR+Structure+Models+and+Ligands&doi=10.1093%2Fnar%2Fgkx1109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">GPCRdb in 2018: adding GPCR structure models and ligands</span></div><div class="casAuthors">Pandy-Szekeres, Gaspar; Munk, Christian; Tsonkov, Tsonko M.; Mordalski, Stefan; Harpsoee, Kasper; Hauser, Alexander S.; Bojarski, Andrzej J.; Gloriam, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D440-D446</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">G protein-coupled receptors are the most abundant mediators of both human signaling processes and therapeutic effects.  Herein, we report GPCRomewide homol. models of unprecedented quality, and roughly 150 000 GPCR ligands with data on biol. activities and com. availability.  Based on the strategy of 'Less model - more Xtal', each model exploits both a main template and alternative local templates.  This achieved higher similarity to new structures than any of the existing resources, and refined crystal structures with missing or distorted regions.  Models are provided for inactive, intermediate and active states-except for classes C and F that so far only have inactive templates.  The ligand database has sep. browsers for: (i) target selection by receptor, family or class, (ii) ligand filtering based on cross-expt. activities (min, max and mean) or chem. properties, (iii) ligand source data and (iv) com. availability.  SMILES structures and activity spreadsheets can be downloaded for further processing.  Furthermore, three recent landmark publications on GPCR drugs, G protein selectivity and genetic variants have been accompanied with resources that now let readers view and analyze the findings themselves in GPCRdb.  Altogether, this update will enable scientific investigation for the wider GPCR community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptWfqUUDwXB7Vg90H21EOLACvtfcHk0lh1B13YtjZEUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlGisLbE&md5=0cf77dd5ea934548a2d3a0ca69991f60</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx1109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx1109%26sid%3Dliteratum%253Aachs%26aulast%3DP%25C3%25A1ndy-Szekeres%26aufirst%3DG.%26aulast%3DMunk%26aufirst%3DC.%26aulast%3DTsonkov%26aufirst%3DT.%2BM.%26aulast%3DMordalski%26aufirst%3DS.%26aulast%3DHarps%25C3%25B8e%26aufirst%3DK.%26aulast%3DHauser%26aufirst%3DA.%2BS.%26aulast%3DBojarski%26aufirst%3DA.%2BJ.%26aulast%3DGloriam%26aufirst%3DD.%2BE.%26atitle%3DGPCRdb%2520in%25202018%253A%2520Adding%2520GPCR%2520Structure%2520Models%2520and%2520Ligands%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26issue%3DD1%26spage%3DD440%26epage%3DD446%26doi%3D10.1093%2Fnar%2Fgkx1109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, D. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda-Saavedra, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, G. J.</span></span> <span> </span><span class="NLM_article-title">Kinomer v. 1.0: A Database of Systematically Classified Eukaryotic Protein Kinases</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>37</i></span> (<span class="NLM_issue">Suppl. 1</span>),  <span class="NLM_fpage">D244</span>– <span class="NLM_lpage">D250</span>, <span class="refDoi"> DOI: 10.1093/nar/gkn834</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fnar%2Fgkn834" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=18974176" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFejtLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=D244-D250&issue=Suppl.+1&author=D.+M.+A.+Martinauthor=D.+Miranda-Saavedraauthor=G.+J.+Barton&title=Kinomer+v.+1.0%3A+A+Database+of+Systematically+Classified+Eukaryotic+Protein+Kinases&doi=10.1093%2Fnar%2Fgkn834"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Kinomer v. 1.0: a database of systematically classified eukaryotic protein kinases</span></div><div class="casAuthors">Martin, David M. A.; Miranda-Saavedra, Diego; Barton, Geoffrey J.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">Database Iss</span>),
    <span class="NLM_cas:pages">D244-D250</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The regulation of protein function through reversible phosphorylation by protein kinases and phosphatases is a general mechanism controlling virtually every cellular activity.  Eukaryotic protein kinases can be classified into distinct, well-characterized groups based on amino acid sequence similarity and function.  We recently reported a highly sensitive and accurate hidden Markov model-based method for the automatic detection and classification of protein kinases into these specific groups.  The Kinomer v. 1.0 database presented here contains annotated classifications for the protein kinase complements of 43 eukaryotic genomes.  These span the taxonomic range and include fungi (16 species), plants (6), diatoms (1), amoebas (2), protists (1) and animals (17).  The kinomes are stored in a relational database and are accessible through a web interface on the basis of species, kinase group or a combination of both.  In addn., the Kinomer v. 1.0 HMM library is made available for users to perform classification on arbitrary sequences.  The Kinomer v. 1.0 database is a continually updated resource where direct comparison of kinase sequences across kinase groups and across species can give insights into kinase function and evolution.  Kinomer v. 1.0 is available at http://www.compbio.dundee.ac.uk/kinomer/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTZbp12ZxpR7Vg90H21EOLACvtfcHk0lgijxVO9Kmhmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFejtLvI&md5=07e84dc6450c15a04582a8973d0aa930</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkn834&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkn834%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DD.%2BM.%2BA.%26aulast%3DMiranda-Saavedra%26aufirst%3DD.%26aulast%3DBarton%26aufirst%3DG.%2BJ.%26atitle%3DKinomer%2520v.%25201.0%253A%2520A%2520Database%2520of%2520Systematically%2520Classified%2520Eukaryotic%2520Protein%2520Kinases%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2009%26volume%3D37%26issue%3DSuppl.%25201%26spage%3DD244%26epage%3DD250%26doi%3D10.1093%2Fnar%2Fgkn834" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burley, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berman, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhikadiya, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Costanzo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalenberg, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duarte, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodsell, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guranović, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzenko, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalro, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namkoong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peisach, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Periskova, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prlić, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Randle, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sala, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sekharan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, Y. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valasatava, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voigt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westbrook, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuravleva, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zardecki, C.</span></span> <span> </span><span class="NLM_article-title">RCSB Protein Data Bank: Biological Macromolecular Structures Enabling Research and Education in Fundamental Biology, Biomedicine, Biotechnology and Energy</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D464</span>– <span class="NLM_lpage">D474</span>, <span class="refDoi"> DOI: 10.1093/nar/gky1004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fnar%2Fgky1004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=30357411" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1Cgt7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D464-D474&issue=D1&author=S.+K.+Burleyauthor=H.+M.+Bermanauthor=C.+Bhikadiyaauthor=C.+Biauthor=L.+Chenauthor=L.+Di+Costanzoauthor=C.+Christieauthor=K.+Dalenbergauthor=J.+M.+Duarteauthor=S.+Duttaauthor=Z.+Fengauthor=S.+Ghoshauthor=D.+S.+Goodsellauthor=R.+K.+Greenauthor=V.+Guranovi%C4%87author=D.+Guzenkoauthor=B.+P.+Hudsonauthor=T.+Kalroauthor=Y.+Liangauthor=R.+Loweauthor=H.+Namkoongauthor=E.+Peisachauthor=I.+Periskovaauthor=A.+Prli%C4%87author=C.+Randleauthor=A.+Roseauthor=P.+Roseauthor=R.+Salaauthor=M.+Sekharanauthor=C.+Shaoauthor=L.+Tanauthor=Y.+P.+Taoauthor=Y.+Valasatavaauthor=M.+Voigtauthor=J.+Westbrookauthor=J.+Wooauthor=H.+Yangauthor=J.+Youngauthor=M.+Zhuravlevaauthor=C.+Zardecki&title=RCSB+Protein+Data+Bank%3A+Biological+Macromolecular+Structures+Enabling+Research+and+Education+in+Fundamental+Biology%2C+Biomedicine%2C+Biotechnology+and+Energy&doi=10.1093%2Fnar%2Fgky1004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">RCSB Protein Data Bank: biological macromolecular structures enabling research and education in fundamental biology, biomedicine, biotechnology and energy</span></div><div class="casAuthors">Burley, Stephen K.; Berman, Helen M.; Bhikadiya, Charmi; Bi, Chunxiao; Chen, Li; Di Costanzo, Luigi; Christie, Cole; Dalenberg, Ken; Duarte, Jose M.; Dutta, Shuchismita; Feng, Zukang; Ghosh, Sutapa; Goodsell, David S.; Green, Rachel K.; Guranovic, Vladimir; Guzenko, Dmytro; Hudson, Brian P.; Kalro, Tara; Liang, Yuhe; Lowe, Robert; Namkoong, Harry; Peisach, Ezra; Periskova, Irina; Prlic, Andreas; Randle, Chris; Rose, Alexander; Rose, Peter; Sala, Raul; Sekharan, Monica; Shao, Chenghua; Tan, Lihua; Tao, Yi-Ping; Valasatava, Yana; Voigt, Maria; Westbrook, John; Woo, Jesse; Yang, Huanwang; Young, Jasmine; Zhuravleva, Marina; Zardecki, Christine</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D464-D474</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  The Research Collab. for Structural Bioinformatics Protein Data Bank (RCSB PDB, rcsb.org), the US data center for the global PDB archive, serves thousands of Data Depositors in the Americas and Oceania and makes 3D macromol. structure data available at no charge and without usage restrictions to more than 1 million rcsb.org Users worldwide and 600 000 pdb101.rcsb.org education-focused Users around the globe.  PDB Data Depositors include structural biologists using macromol. crystallog., NMR spectroscopy and 3D electron microscopy.  PDB Data Consumers include researchers, educators and students studying Fundamental Biol., Biomedicine, Biotechnol. and Energy.  Recent reorganization of RCSB PDB activities into four integrated, interdependent services is described in detail, together with tools and resources added over the past 2 years to RCSB PDB web portals in support of a 'Structural View of Biol.'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqbbw8LsGXrrVg90H21EOLACvtfcHk0lgijxVO9Kmhmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1Cgt7jO&md5=190361b23d7665a66208618c0ba86b5a</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky1004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky1004%26sid%3Dliteratum%253Aachs%26aulast%3DBurley%26aufirst%3DS.%2BK.%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DBhikadiya%26aufirst%3DC.%26aulast%3DBi%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DDi%2BCostanzo%26aufirst%3DL.%26aulast%3DChristie%26aufirst%3DC.%26aulast%3DDalenberg%26aufirst%3DK.%26aulast%3DDuarte%26aufirst%3DJ.%2BM.%26aulast%3DDutta%26aufirst%3DS.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGhosh%26aufirst%3DS.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DGreen%26aufirst%3DR.%2BK.%26aulast%3DGuranovi%25C4%2587%26aufirst%3DV.%26aulast%3DGuzenko%26aufirst%3DD.%26aulast%3DHudson%26aufirst%3DB.%2BP.%26aulast%3DKalro%26aufirst%3DT.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DLowe%26aufirst%3DR.%26aulast%3DNamkoong%26aufirst%3DH.%26aulast%3DPeisach%26aufirst%3DE.%26aulast%3DPeriskova%26aufirst%3DI.%26aulast%3DPrli%25C4%2587%26aufirst%3DA.%26aulast%3DRandle%26aufirst%3DC.%26aulast%3DRose%26aufirst%3DA.%26aulast%3DRose%26aufirst%3DP.%26aulast%3DSala%26aufirst%3DR.%26aulast%3DSekharan%26aufirst%3DM.%26aulast%3DShao%26aufirst%3DC.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DTao%26aufirst%3DY.%2BP.%26aulast%3DValasatava%26aufirst%3DY.%26aulast%3DVoigt%26aufirst%3DM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DWoo%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DYoung%26aufirst%3DJ.%26aulast%3DZhuravleva%26aufirst%3DM.%26aulast%3DZardecki%26aufirst%3DC.%26atitle%3DRCSB%2520Protein%2520Data%2520Bank%253A%2520Biological%2520Macromolecular%2520Structures%2520Enabling%2520Research%2520and%2520Education%2520in%2520Fundamental%2520Biology%252C%2520Biomedicine%252C%2520Biotechnology%2520and%2520Energy%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2019%26volume%3D47%26issue%3DD1%26spage%3DD464%26epage%3DD474%26doi%3D10.1093%2Fnar%2Fgky1004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Szklarczyk, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gable, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Junge, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wyder, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huerta-Cepas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simonovic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doncheva, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bork, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mering, C. V.</span></span> <span> </span><span class="NLM_article-title">STRING V11: Protein–Protein Association Networks with Increased Coverage, Supporting Functional Discovery in Genome-Wide Experimental Datasets</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D607</span>– <span class="NLM_lpage">D613</span>, <span class="refDoi"> DOI: 10.1093/nar/gky1131</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fnar%2Fgky1131" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=30476243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1Gqtr%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D607-D613&issue=D1&author=D.+Szklarczykauthor=A.+L.+Gableauthor=D.+Lyonauthor=A.+Jungeauthor=S.+Wyderauthor=J.+Huerta-Cepasauthor=M.+Simonovicauthor=N.+T.+Donchevaauthor=J.+H.+Morrisauthor=P.+Borkauthor=L.+J.+Jensenauthor=C.+V.+Mering&title=STRING+V11%3A+Protein%E2%80%93Protein+Association+Networks+with+Increased+Coverage%2C+Supporting+Functional+Discovery+in+Genome-Wide+Experimental+Datasets&doi=10.1093%2Fnar%2Fgky1131"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">STRING v11: protein-protein association networks with increased coverage supporting functional discovery in genome-wide experimental datasets</span></div><div class="casAuthors">Szklarczyk, Damian; Gable, Annika L.; Lyon, David; Junge, Alexander; Wyder, Stefan; Huerta-Cepas, Jaime; Simonovic, Milan; Doncheva, Nadezhda T.; Morris, John H.; Bork, Peer; Jensen, Lars J.; von Mering, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D607-D613</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Proteins and their functional interactions form the backbone of the cellular machinery.  Their connectivity network needs to be considered for the full understanding of biol. phenomena, but the available information on protein-protein assocns. is incomplete and exhibits varying levels of annotation granularity and reliability.  The STRING database aims to collect, score and integrate all publicly available sources of protein-protein interaction information, and to complement these with computational predictions.  Its goal is to achieve a comprehensive and objective global network, including direct (phys.) as well as indirect (functional) interactions.  The latest version of STRING (11.0) more than doubles the no. of organisms it covers, to 5090.  The most important new feature is an option to upload entire, genome-wide datasets as input, allowing users to visualize subsets as interaction networks and to perform gene-set enrichment anal. on the entire input.  For the enrichment anal., STRING implements well-known classification systems such as Gene Ontol. and KEGG, but also offers addnl., new classification systems based on high-throughput text-mining as well as on a hierarchical clustering of the assocn. network itself.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7UX25KcxhhbVg90H21EOLACvtfcHk0lgJ5h4DP6cMng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1Gqtr%252FM&md5=43df05a2de505f1df210fb4aaa200f58</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky1131&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky1131%26sid%3Dliteratum%253Aachs%26aulast%3DSzklarczyk%26aufirst%3DD.%26aulast%3DGable%26aufirst%3DA.%2BL.%26aulast%3DLyon%26aufirst%3DD.%26aulast%3DJunge%26aufirst%3DA.%26aulast%3DWyder%26aufirst%3DS.%26aulast%3DHuerta-Cepas%26aufirst%3DJ.%26aulast%3DSimonovic%26aufirst%3DM.%26aulast%3DDoncheva%26aufirst%3DN.%2BT.%26aulast%3DMorris%26aufirst%3DJ.%2BH.%26aulast%3DBork%26aufirst%3DP.%26aulast%3DJensen%26aufirst%3DL.%2BJ.%26aulast%3DMering%26aufirst%3DC.%2BV.%26atitle%3DSTRING%2520V11%253A%2520Protein%25E2%2580%2593Protein%2520Association%2520Networks%2520with%2520Increased%2520Coverage%252C%2520Supporting%2520Functional%2520Discovery%2520in%2520Genome-Wide%2520Experimental%2520Datasets%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2019%26volume%3D47%26issue%3DD1%26spage%3DD607%26epage%3DD613%26doi%3D10.1093%2Fnar%2Fgky1131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jassal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viteri, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorente, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fabregat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sidiropoulos, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillespie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haw, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loney, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">May, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milacic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothfels, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sevilla, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shamovsky, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shorser, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varusai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiser, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermjakob, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Eustachio, P.</span></span> <span> </span><span class="NLM_article-title">The Reactome Pathway Knowledgebase</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>48</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D498</span>– <span class="NLM_lpage">D503</span>, <span class="refDoi"> DOI: 10.1093/nar/gkz1031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fnar%2Fgkz1031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=31691815" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1GltrjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2020&pages=D498-D503&issue=D1&author=B.+Jassalauthor=L.+Matthewsauthor=G.+Viteriauthor=C.+Gongauthor=P.+Lorenteauthor=A.+Fabregatauthor=K.+Sidiropoulosauthor=J.+Cookauthor=M.+Gillespieauthor=R.+Hawauthor=F.+Loneyauthor=B.+Mayauthor=M.+Milacicauthor=K.+Rothfelsauthor=C.+Sevillaauthor=V.+Shamovskyauthor=S.+Shorserauthor=T.+Varusaiauthor=J.+Weiserauthor=G.+Wuauthor=L.+Steinauthor=H.+Hermjakobauthor=P.+D%E2%80%99Eustachio&title=The+Reactome+Pathway+Knowledgebase&doi=10.1093%2Fnar%2Fgkz1031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The reactome pathway knowledgebase</span></div><div class="casAuthors">Jassal, Bijay; Matthews, Lisa; Viteri, Guilherme; Gong, Chuqiao; Lorente, Pascual; Fabregat, Antonio; Sidiropoulos, Konstantinos; Cook, Justin; Gillespie, Marc; Haw, Robin; Loney, Fred; May, Bruce; Milacic, Marija; Rothfels, Karen; Sevilla, Cristoffer; Shamovsky, Veronica; Shorser, Solomon; Varusai, Thawfeek; Weiser, Joel; Wu, Guanming; Stein, Lincoln; Hermjakob, Henning; D'Eustachio, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D498-D503</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  The Reactome Knowledgebase provides mol. details of signal transduction, transport, DNA replication, metab. and other cellular processes as an ordered network of mol. transformations in a single consistent data model, an extended version of a classic metabolic map.  Reactome functions both as an archive of biol. processes and as a tool for discovering functional relationships in data such as gene expression profiles or somatic mutation catalogs from tumor cells.  To extend our ability to annotate human disease processes, we have implemented a new drug class and have used it initially to annotate drugs relevant to cardiovascular disease.  Our annotation model depends on external domain experts to identify new areas for annotation and to review new content.  New web pages facilitate recruitment of community experts and allow those who have contributed to Reactome to identify their contributions and link them to their ORCID records.  To improve visualization of our content, we have implemented a new tool to automatically lay out the components of individual reactions with multiple options for downloading the reaction diagrams and assocd. data, and a new display of our event hierarchy that will facilitate visual interpretation of pathway anal. results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8-pLjgJKfiLVg90H21EOLACvtfcHk0liYBklEok0Wrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1GltrjP&md5=0ba27eea4957e270b8a16c0159ffcd5d</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkz1031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkz1031%26sid%3Dliteratum%253Aachs%26aulast%3DJassal%26aufirst%3DB.%26aulast%3DMatthews%26aufirst%3DL.%26aulast%3DViteri%26aufirst%3DG.%26aulast%3DGong%26aufirst%3DC.%26aulast%3DLorente%26aufirst%3DP.%26aulast%3DFabregat%26aufirst%3DA.%26aulast%3DSidiropoulos%26aufirst%3DK.%26aulast%3DCook%26aufirst%3DJ.%26aulast%3DGillespie%26aufirst%3DM.%26aulast%3DHaw%26aufirst%3DR.%26aulast%3DLoney%26aufirst%3DF.%26aulast%3DMay%26aufirst%3DB.%26aulast%3DMilacic%26aufirst%3DM.%26aulast%3DRothfels%26aufirst%3DK.%26aulast%3DSevilla%26aufirst%3DC.%26aulast%3DShamovsky%26aufirst%3DV.%26aulast%3DShorser%26aufirst%3DS.%26aulast%3DVarusai%26aufirst%3DT.%26aulast%3DWeiser%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DG.%26aulast%3DStein%26aufirst%3DL.%26aulast%3DHermjakob%26aufirst%3DH.%26aulast%3DD%25E2%2580%2599Eustachio%26aufirst%3DP.%26atitle%3DThe%2520Reactome%2520Pathway%2520Knowledgebase%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2020%26volume%3D48%26issue%3DD1%26spage%3DD498%26epage%3DD503%26doi%3D10.1093%2Fnar%2Fgkz1031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Piñero, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bravo, À.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Queralt-Rosinach, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutiérrez-Sacristán, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deu-Pons, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centeno, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-García, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanz, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furlong, L. I.</span></span> <span> </span><span class="NLM_article-title">DisGeNET: A Comprehensive Platform Integrating Information on Human Disease-Associated Genes and Variants</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D833</span>– <span class="NLM_lpage">D839</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw943</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fnar%2Fgkw943" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=27924018" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslWhtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D833-D839&issue=D1&author=J.+Pi%C3%B1eroauthor=%C3%80.+Bravoauthor=N.+Queralt-Rosinachauthor=A.+Guti%C3%A9rrez-Sacrist%C3%A1nauthor=J.+Deu-Ponsauthor=E.+Centenoauthor=J.+Garc%C3%ADa-Garc%C3%ADaauthor=F.+Sanzauthor=L.+I.+Furlong&title=DisGeNET%3A+A+Comprehensive+Platform+Integrating+Information+on+Human+Disease-Associated+Genes+and+Variants&doi=10.1093%2Fnar%2Fgkw943"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">DisGeNET: a comprehensive platform integrating information on human disease-associated genes and variants</span></div><div class="casAuthors">Pinero, Janet; Bravo, Alex; Queralt-Rosinach, Nuria; Gutierrez-Sacristan, Alba; Deu-Pons, Jordi; Centeno, Emilio; Garcia-Garcia, Javier; Sanz, Ferran; Furlong, Laura I.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D833-D839</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The information about the genetic basis of human diseases lies at the heart of precision medicine and drug discovery.  However, to realize its full potential to support these goals, several problems, such as fragmentation, heterogeneity, availability and different conceptualization of the data must be overcome.  To provide the community with a resource free of these hurdles, we have developed DisGeNET (http://www.disgenet.org), one of the largest available collections of genes and variants involved in human diseases.  DisGeNET integrates data from expert curated repositories, GWAS catalogs, animal models and the scientific literature.  DisGeNET data are homogeneously annotated with controlled vocabularies and community-driven ontologies.  Addnl., several original metrics are provided to assist the prioritization of genotype-phenotype relationships.  The information is accessible through a web interface, a Cytoscape App, an RDF SPARQL endpoint, scripts in several programming languages and an R package.  DisGeNET is a versatile platform that can be used for different research purposes including the investigation of the mol. underpinnings of specific human diseases and their comorbidities, the anal. of the properties of disease genes, the generation of hypothesis on drug therapeutic action and drug adverse effects, the validation of computationally predicted disease genes and the evaluation of text-mining methods performance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqve0GG5MK1G7Vg90H21EOLACvtfcHk0liYBklEok0Wrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslWhtL4%253D&md5=b00a79e146f48ddd5d67e85597ac89c2</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw943&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw943%26sid%3Dliteratum%253Aachs%26aulast%3DPi%25C3%25B1ero%26aufirst%3DJ.%26aulast%3DBravo%26aufirst%3D%25C3%2580.%26aulast%3DQueralt-Rosinach%26aufirst%3DN.%26aulast%3DGuti%25C3%25A9rrez-Sacrist%25C3%25A1n%26aufirst%3DA.%26aulast%3DDeu-Pons%26aufirst%3DJ.%26aulast%3DCenteno%26aufirst%3DE.%26aulast%3DGarc%25C3%25ADa-Garc%25C3%25ADa%26aufirst%3DJ.%26aulast%3DSanz%26aufirst%3DF.%26aulast%3DFurlong%26aufirst%3DL.%2BI.%26atitle%3DDisGeNET%253A%2520A%2520Comprehensive%2520Platform%2520Integrating%2520Information%2520on%2520Human%2520Disease-Associated%2520Genes%2520and%2520Variants%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26issue%3DD1%26spage%3DD833%26epage%3DD839%26doi%3D10.1093%2Fnar%2Fgkw943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corsello, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peck, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natoli, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gould, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tubelli, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asiedu, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lahr, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirschman, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donahue, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Julian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wadden, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liberzon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toder, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orzechowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enache, O. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piccioni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyons, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shamji, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vrcic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosains, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeda, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davison, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ardlie, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hogstrom, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenside, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clemons, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silver, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read-Button, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haggarty, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ronco, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehm, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doench, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bittker, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Root, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span> <span> </span><span class="NLM_article-title">A Next Generation Connectivity Map: L1000 Platform and the First 1,000,000 Profiles</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>171</i></span>,  <span class="NLM_fpage">1437</span>– <span class="NLM_lpage">1452</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2017.10.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1016%2Fj.cell.2017.10.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=29195078" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFWmsbrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2017&pages=1437-1452&author=A.+Subramanianauthor=R.+Narayanauthor=S.+M.+Corselloauthor=D.+D.+Peckauthor=T.+E.+Natoliauthor=X.+Luauthor=J.+Gouldauthor=J.+F.+Davisauthor=A.+A.+Tubelliauthor=J.+K.+Asieduauthor=D.+L.+Lahrauthor=J.+E.+Hirschmanauthor=Z.+Liuauthor=M.+Donahueauthor=B.+Julianauthor=M.+Khanauthor=D.+Waddenauthor=I.+C.+Smithauthor=D.+Lamauthor=A.+Liberzonauthor=C.+Toderauthor=M.+Bagulauthor=M.+Orzechowskiauthor=O.+M.+Enacheauthor=F.+Piccioniauthor=S.+A.+Johnsonauthor=N.+J.+Lyonsauthor=A.+H.+Bergerauthor=A.+F.+Shamjiauthor=A.+N.+Brooksauthor=A.+Vrcicauthor=C.+Flynnauthor=J.+Rosainsauthor=D.+Y.+Takedaauthor=R.+Huauthor=D.+Davisonauthor=J.+Lambauthor=K.+Ardlieauthor=L.+Hogstromauthor=P.+Greensideauthor=N.+S.+Grayauthor=P.+A.+Clemonsauthor=S.+Silverauthor=X.+Wuauthor=W.+N.+Zhaoauthor=W.+Read-Buttonauthor=X.+Wuauthor=S.+J.+Haggartyauthor=L.+V.+Roncoauthor=J.+S.+Boehmauthor=S.+L.+Schreiberauthor=J.+G.+Doenchauthor=J.+A.+Bittkerauthor=D.+E.+Rootauthor=B.+Wongauthor=T.+R.+Golub&title=A+Next+Generation+Connectivity+Map%3A+L1000+Platform+and+the+First+1%2C000%2C000+Profiles&doi=10.1016%2Fj.cell.2017.10.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">A next generation connectivity map: L1000 platform and the first 1,000,000 profiles</span></div><div class="casAuthors">Subramanian, Aravind; Narayan, Rajiv; Corsello, Steven M.; Peck, David D.; Natoli, Ted E.; Lu, Xiaodong; Gould, Joshua; Davis, John F.; Tubelli, Andrew A.; Asiedu, Jacob K.; Lahr, David L.; Hirschman, Jodi E.; Liu, Zihan; Donahue, Melanie; Julian, Bina; Khan, Mariya; Wadden, David; Smith, Ian C.; Lam, Daniel; Liberzon, Arthur; Toder, Courtney; Bagul, Mukta; Orzechowski, Marek; Enache, Oana M.; Piccioni, Federica; Johnson, Sarah A.; Lyons, Nicholas J.; Berger, Alice H.; Shamji, Alykhan F.; Brooks, Angela N.; Vrcic, Anita; Flynn, Corey; Rosains, Jacqueline; Takeda, David Y.; Hu, Roger; Davison, Desiree; Lamb, Justin; Ardlie, Kristin; Hogstrom, Larson; Greenside, Peyton; Gray, Nathanael S.; Clemons, Paul A.; Silver, Serena; Wu, Xiaoyun; Zhao, Wen-Ning; Read-Button, Willis; Wu, Xiaohua; Haggarty, Stephen J.; Ronco, Lucienne V.; Boehm, Jesse S.; Schreiber, Stuart L.; Doench, John G.; Bittker, Joshua A.; Root, David E.; Wong, Bang; Golub, Todd R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">171</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1437-1452.e17</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">We previously piloted the concept of a Connectivity Map (CMap), whereby genes, drugs, and disease states are connected by virtue of common gene-expression signatures.  Here, we report more than a 1000-fold scale-up of the CMap as part of the NIH LINCS Consortium, made possible by a new, low-cost, high-throughput reduced representation expression profiling method that we term L1000.  We show that L1000 is highly reproducible, comparable to RNA sequencing, and suitable for computational inference of the expression levels of 81% of non-measured transcripts.  We further show that the expanded CMap can be used to discover mechanism of action of small mols., functionally annotate genetic variants of disease genes, and inform clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIWrWy5IxY-rVg90H21EOLACvtfcHk0li--fSQBY6USQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFWmsbrE&md5=d2b13f2b80fe94021ac03f6a8c859a4a</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.10.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.10.049%26sid%3Dliteratum%253Aachs%26aulast%3DSubramanian%26aufirst%3DA.%26aulast%3DNarayan%26aufirst%3DR.%26aulast%3DCorsello%26aufirst%3DS.%2BM.%26aulast%3DPeck%26aufirst%3DD.%2BD.%26aulast%3DNatoli%26aufirst%3DT.%2BE.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DGould%26aufirst%3DJ.%26aulast%3DDavis%26aufirst%3DJ.%2BF.%26aulast%3DTubelli%26aufirst%3DA.%2BA.%26aulast%3DAsiedu%26aufirst%3DJ.%2BK.%26aulast%3DLahr%26aufirst%3DD.%2BL.%26aulast%3DHirschman%26aufirst%3DJ.%2BE.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DDonahue%26aufirst%3DM.%26aulast%3DJulian%26aufirst%3DB.%26aulast%3DKhan%26aufirst%3DM.%26aulast%3DWadden%26aufirst%3DD.%26aulast%3DSmith%26aufirst%3DI.%2BC.%26aulast%3DLam%26aufirst%3DD.%26aulast%3DLiberzon%26aufirst%3DA.%26aulast%3DToder%26aufirst%3DC.%26aulast%3DBagul%26aufirst%3DM.%26aulast%3DOrzechowski%26aufirst%3DM.%26aulast%3DEnache%26aufirst%3DO.%2BM.%26aulast%3DPiccioni%26aufirst%3DF.%26aulast%3DJohnson%26aufirst%3DS.%2BA.%26aulast%3DLyons%26aufirst%3DN.%2BJ.%26aulast%3DBerger%26aufirst%3DA.%2BH.%26aulast%3DShamji%26aufirst%3DA.%2BF.%26aulast%3DBrooks%26aufirst%3DA.%2BN.%26aulast%3DVrcic%26aufirst%3DA.%26aulast%3DFlynn%26aufirst%3DC.%26aulast%3DRosains%26aufirst%3DJ.%26aulast%3DTakeda%26aufirst%3DD.%2BY.%26aulast%3DHu%26aufirst%3DR.%26aulast%3DDavison%26aufirst%3DD.%26aulast%3DLamb%26aufirst%3DJ.%26aulast%3DArdlie%26aufirst%3DK.%26aulast%3DHogstrom%26aufirst%3DL.%26aulast%3DGreenside%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DClemons%26aufirst%3DP.%2BA.%26aulast%3DSilver%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DW.%2BN.%26aulast%3DRead-Button%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DRonco%26aufirst%3DL.%2BV.%26aulast%3DBoehm%26aufirst%3DJ.%2BS.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DDoench%26aufirst%3DJ.%2BG.%26aulast%3DBittker%26aufirst%3DJ.%2BA.%26aulast%3DRoot%26aufirst%3DD.%2BE.%26aulast%3DWong%26aufirst%3DB.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26atitle%3DA%2520Next%2520Generation%2520Connectivity%2520Map%253A%2520L1000%2520Platform%2520and%2520the%2520First%25201%252C000%252C000%2520Profiles%26jtitle%3DCell%26date%3D2017%26volume%3D171%26spage%3D1437%26epage%3D1452%26doi%3D10.1016%2Fj.cell.2017.10.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stathias, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koleti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidovic, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fazel-Najafabadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilarczyk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terryn, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Umeano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, D. J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lachmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evangelista, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma’ayan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medvedovic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schürer, S. C.</span></span> <span> </span><span class="NLM_article-title">LINCS Data Portal 2.0: Next Generation Access Point for Perturbation-Response Signatures</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>48</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D431</span>– <span class="NLM_lpage">D439</span>, <span class="refDoi"> DOI: 10.1093/nar/gkz1023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fnar%2Fgkz1023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=31701147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1GltrbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2020&pages=D431-D439&issue=D1&author=V.+Stathiasauthor=J.+Turnerauthor=A.+Koletiauthor=D.+Vidovicauthor=D.+Cooperauthor=M.+Fazel-Najafabadiauthor=M.+Pilarczykauthor=R.+Terrynauthor=C.+Chungauthor=A.+Umeanoauthor=D.+J.+B.+Clarkeauthor=A.+Lachmannauthor=J.+E.+Evangelistaauthor=A.+Ma%E2%80%99ayanauthor=M.+Medvedovicauthor=S.+C.+Sch%C3%BCrer&title=LINCS+Data+Portal+2.0%3A+Next+Generation+Access+Point+for+Perturbation-Response+Signatures&doi=10.1093%2Fnar%2Fgkz1023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">LINCS Data Portal 2.0: next generation access point for perturbation-response signatures</span></div><div class="casAuthors">Stathias, Vasileios; Turner, John; Koleti, Amar; Vidovic, Dusica; Cooper, Daniel; Fazel-Najafabadi, Mehdi; Pilarczyk, Marcin; Terryn, Raymond; Chung, Caty; Umeano, Afoma; Clarke, Daniel J. B.; Lachmann, Alexander; Evangelista, John Erol; Ma'ayan, Avi; Medvedovic, Mario; Schurer, Stephan C.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D431-D439</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  The Library of Integrated Network-Based Cellular Signatures (LINCS) is an NIH Common Fund program with the goal of generating a large-scale and comprehensive catalog of perturbation-response signatures by utilizing a diverse collection of perturbations across many model systems and assay types.  The LINCS Data Portal (LDP) has been the primary access point for the compendium of LINCS data and has been widely utilized.  Here, we report the first major update of LDP with substantial changes in the data architecture and APIs, a completely redesigned user interface, and enhanced curated metadata annotations to support more advanced, intuitive and deeper querying, exploration and anal. capabilities.  The cornerstone of this update has been the decision to reprocess all high-level LINCS datasets and make them accessible at the data point level enabling users to directly access and download any subset of signatures across the entire library independent from the originating source, project or assay.  Access to the individual signatures also enables the newly implemented signature search functionality, which utilizes the iLINCS platform to identify conditions that mimic or reverse gene set queries.  A newly designed query interface enables global metadata search with autosuggest across all annotations assocd. with perturbations, model systems, and signatures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzW9FiCBXXC7Vg90H21EOLACvtfcHk0li--fSQBY6USQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1GltrbE&md5=2c39a40d51764dd9a68ce181923fd87a</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkz1023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkz1023%26sid%3Dliteratum%253Aachs%26aulast%3DStathias%26aufirst%3DV.%26aulast%3DTurner%26aufirst%3DJ.%26aulast%3DKoleti%26aufirst%3DA.%26aulast%3DVidovic%26aufirst%3DD.%26aulast%3DCooper%26aufirst%3DD.%26aulast%3DFazel-Najafabadi%26aufirst%3DM.%26aulast%3DPilarczyk%26aufirst%3DM.%26aulast%3DTerryn%26aufirst%3DR.%26aulast%3DChung%26aufirst%3DC.%26aulast%3DUmeano%26aufirst%3DA.%26aulast%3DClarke%26aufirst%3DD.%2BJ.%2BB.%26aulast%3DLachmann%26aufirst%3DA.%26aulast%3DEvangelista%26aufirst%3DJ.%2BE.%26aulast%3DMa%25E2%2580%2599ayan%26aufirst%3DA.%26aulast%3DMedvedovic%26aufirst%3DM.%26aulast%3DSch%25C3%25BCrer%26aufirst%3DS.%2BC.%26atitle%3DLINCS%2520Data%2520Portal%25202.0%253A%2520Next%2520Generation%2520Access%2520Point%2520for%2520Perturbation-Response%2520Signatures%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2020%26volume%3D48%26issue%3DD1%26spage%3DD431%26epage%3DD439%26doi%3D10.1093%2Fnar%2Fgkz1023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dash, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shakyawar, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaushik, S.</span></span> <span> </span><span class="NLM_article-title">Big Data in Healthcare: Management, Analysis and Future Prospects</span>. <i>J. Big Data</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">54</span>, <span class="refDoi"> DOI: 10.1186/s40537-019-0217-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1186%2Fs40537-019-0217-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2019&pages=54&author=S.+Dashauthor=S.+K.+Shakyawarauthor=M.+Sharmaauthor=S.+Kaushik&title=Big+Data+in+Healthcare%3A+Management%2C+Analysis+and+Future+Prospects&doi=10.1186%2Fs40537-019-0217-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1186%2Fs40537-019-0217-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs40537-019-0217-0%26sid%3Dliteratum%253Aachs%26aulast%3DDash%26aufirst%3DS.%26aulast%3DShakyawar%26aufirst%3DS.%2BK.%26aulast%3DSharma%26aufirst%3DM.%26aulast%3DKaushik%26aufirst%3DS.%26atitle%3DBig%2520Data%2520in%2520Healthcare%253A%2520Management%252C%2520Analysis%2520and%2520Future%2520Prospects%26jtitle%3DJ.%2520Big%2520Data%26date%3D2019%26volume%3D6%26spage%3D54%26doi%3D10.1186%2Fs40537-019-0217-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Häyrinen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saranto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nykänen, P.</span></span> <span> </span><span class="NLM_article-title">Definition, Structure, Content, Use and Impacts of Electronic Health Records: A Review of the Research Literature</span>. <i>Int. J. Med. Inform.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">291</span>– <span class="NLM_lpage">304</span>, <span class="refDoi"> DOI: 10.1016/j.ijmedinf.2007.09.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1016%2Fj.ijmedinf.2007.09.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=17951106" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A280%3ADC%252BD1c3itVKqtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2008&pages=291-304&author=K.+H%C3%A4yrinenauthor=K.+Sarantoauthor=P.+Nyk%C3%A4nen&title=Definition%2C+Structure%2C+Content%2C+Use+and+Impacts+of+Electronic+Health+Records%3A+A+Review+of+the+Research+Literature&doi=10.1016%2Fj.ijmedinf.2007.09.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Definition, structure, content, use and impacts of electronic health records: a review of the research literature</span></div><div class="casAuthors">Hayrinen Kristiina; Saranto Kaija; Nykanen Pirkko</div><div class="citationInfo"><span class="NLM_cas:title">International journal of medical informatics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">291-304</span>
        ISSN:<span class="NLM_cas:issn">1386-5056</span>.
    </div><div class="casAbstract">PURPOSE:  This paper reviews the research literature on electronic health record (EHR) systems.  The aim is to find out (1) how electronic health records are defined, (2) how the structure of these records is described, (3) in what contexts EHRs are used, (4) who has access to EHRs, (5) which data components of the EHRs are used and studied, (6) what is the purpose of research in this field, (7) what methods of data collection have been used in the studies reviewed and (8) what are the results of these studies.  METHODS:  A systematic review was carried out of the research dealing with the content of EHRs.  A literature search was conducted on four electronic databases: Pubmed/Medline, Cinalh, Eval and Cochrane.  RESULTS:  The concept of EHR comprised a wide range of information systems, from files compiled in single departments to longitudinal collections of patient data.  Only very few papers offered descriptions of the structure of EHRs or the terminologies used.  EHRs were used in primary, secondary and tertiary care.  Data were recorded in EHRs by different groups of health care professionals.  Secretarial staff also recorded data from dictation or nurses' or physicians' manual notes.  Some information was also recorded by patients themselves; this information is validated by physicians.  It is important that the needs and requirements of different users are taken into account in the future development of information systems.  Several data components were documented in EHRs: daily charting, medication administration, physical assessment, admission nursing note, nursing care plan, referral, present complaint (e.g. symptoms), past medical history, life style, physical examination, diagnoses, tests, procedures, treatment, medication, discharge, history, diaries, problems, findings and immunization.  In the future it will be necessary to incorporate different kinds of standardized instruments, electronic interviews and nursing documentation systems in EHR systems.  The aspects of information quality most often explored in the studies reviewed were the completeness and accuracy of different data components.  It has been shown in several studies that the use of an information system was conducive to more complete and accurate documentation by health care professionals.  The quality of information is particularly important in patient care, but EHRs also provide important information for secondary purposes, such as health policy planning.  CONCLUSION:  Studies focusing on the content of EHRs are needed, especially studies of nursing documentation or patient self-documentation.  One future research area is to compare the documentation of different health care professionals with the core information about EHRs which has been determined in national health projects.  The challenge for ongoing national health record projects around the world is to take into account all the different types of EHRs and the needs and requirements of different health care professionals and consumers in the development of EHRs.  A further challenge is the use of international terminologies in order to achieve semantic interoperability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQWpV3pW8yFvA20zIbnCCI3fW6udTcc2eaTC9hRSO1FObntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c3itVKqtg%253D%253D&md5=1ed67df7fff261a430e3cd0bc9dbeb5d</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.ijmedinf.2007.09.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijmedinf.2007.09.001%26sid%3Dliteratum%253Aachs%26aulast%3DH%25C3%25A4yrinen%26aufirst%3DK.%26aulast%3DSaranto%26aufirst%3DK.%26aulast%3DNyk%25C3%25A4nen%26aufirst%3DP.%26atitle%3DDefinition%252C%2520Structure%252C%2520Content%252C%2520Use%2520and%2520Impacts%2520of%2520Electronic%2520Health%2520Records%253A%2520A%2520Review%2520of%2520the%2520Research%2520Literature%26jtitle%3DInt.%2520J.%2520Med.%2520Inform.%26date%3D2008%26volume%3D77%26spage%3D291%26epage%3D304%26doi%3D10.1016%2Fj.ijmedinf.2007.09.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Denny, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devaney, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutter, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boerwinkle, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brilliant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butts, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaffins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Channamsetty, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinea, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cotler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daviglus, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falcón, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funk, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabriel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gebo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rico, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glazer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grand, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenland, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hochfel, B. I.</span></span> <span> </span><span class="NLM_article-title">The “All of Us” Research Program</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>381</i></span>,  <span class="NLM_fpage">668</span>– <span class="NLM_lpage">676</span>, <span class="refDoi"> DOI: 10.1056/NEJMsr1809937</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1056%2FNEJMsr1809937" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=31412182" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A280%3ADC%252BB3MvnvFemsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=381&publication_year=2019&pages=668-676&author=J.+C.+Dennyauthor=S.+Devaneyauthor=J.+L.+Rutterauthor=E.+Boerwinkleauthor=M.+Brilliantauthor=P.+Buttsauthor=M.+Castroauthor=J.+Chaffinsauthor=K.+Changauthor=V.+Channamsettyauthor=C.+Chineaauthor=A.+Cotlerauthor=M.+Daviglusauthor=A.+Falc%C3%B3nauthor=G.+Funkauthor=S.+Gabrielauthor=K.+Geboauthor=E.+G.+Ricoauthor=D.+Glazerauthor=D.+B.+Goldsteinauthor=J.+M.+Grandauthor=P.+Greenlandauthor=K.+Hallauthor=B.+I.+Hochfel&title=The+%E2%80%9CAll+of+Us%E2%80%9D+Research+Program&doi=10.1056%2FNEJMsr1809937"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">The "All of Us" Research Program</span></div><div class="casAuthors">Denny Joshua C; Rutter Joni L; Goldstein David B; Philippakis Anthony; Smoller Jordan W; Jenkins Gwynne; Dishman Eric</div><div class="citationInfo"><span class="NLM_cas:title">The New England journal of medicine</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">381</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">668-676</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Knowledge gained from observational cohort studies has dramatically advanced the prevention and treatment of diseases.  Many of these cohorts, however, are small, lack diversity, or do not provide comprehensive phenotype data.  The All of Us Research Program plans to enroll a diverse group of at least 1 million persons in the United States in order to accelerate biomedical research and improve health.  The program aims to make the research results accessible to participants, and it is developing new approaches to generate, access, and make data broadly available to approved researchers.  All of Us opened for enrollment in May 2018 and currently enrolls participants 18 years of age or older from a network of more than 340 recruitment sites.  Elements of the program protocol include health questionnaires, electronic health records (EHRs), physical measurements, the use of digital health technology, and the collection and analysis of biospecimens.  As of July 2019, more than 175,000 participants had contributed biospecimens.  More than 80% of these participants are from groups that have been historically underrepresented in biomedical research.  EHR data on more than 112,000 participants from 34 sites have been collected.  The All of Us data repository should permit researchers to take into account individual differences in lifestyle, socioeconomic factors, environment, and biologic characteristics in order to advance precision diagnosis, prevention, and treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSktYPjqdU6JpHZzFRCXxVKfW6udTcc2ebvB-LcsarQR7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MvnvFemsA%253D%253D&md5=a756df47643a8eec8c7d9eb2931537b0</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1056%2FNEJMsr1809937&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMsr1809937%26sid%3Dliteratum%253Aachs%26aulast%3DDenny%26aufirst%3DJ.%2BC.%26aulast%3DDevaney%26aufirst%3DS.%26aulast%3DRutter%26aufirst%3DJ.%2BL.%26aulast%3DBoerwinkle%26aufirst%3DE.%26aulast%3DBrilliant%26aufirst%3DM.%26aulast%3DButts%26aufirst%3DP.%26aulast%3DCastro%26aufirst%3DM.%26aulast%3DChaffins%26aufirst%3DJ.%26aulast%3DChang%26aufirst%3DK.%26aulast%3DChannamsetty%26aufirst%3DV.%26aulast%3DChinea%26aufirst%3DC.%26aulast%3DCotler%26aufirst%3DA.%26aulast%3DDaviglus%26aufirst%3DM.%26aulast%3DFalc%25C3%25B3n%26aufirst%3DA.%26aulast%3DFunk%26aufirst%3DG.%26aulast%3DGabriel%26aufirst%3DS.%26aulast%3DGebo%26aufirst%3DK.%26aulast%3DRico%26aufirst%3DE.%2BG.%26aulast%3DGlazer%26aufirst%3DD.%26aulast%3DGoldstein%26aufirst%3DD.%2BB.%26aulast%3DGrand%26aufirst%3DJ.%2BM.%26aulast%3DGreenland%26aufirst%3DP.%26aulast%3DHall%26aufirst%3DK.%26aulast%3DHochfel%26aufirst%3DB.%2BI.%26atitle%3DThe%2520%25E2%2580%259CAll%2520of%2520Us%25E2%2580%259D%2520Research%2520Program%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2019%26volume%3D381%26spage%3D668%26epage%3D676%26doi%3D10.1056%2FNEJMsr1809937" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, P. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaddi, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venugopalan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M. D.</span></span> <span> </span><span class="NLM_article-title">-Omic and Electronic Health Record Big Data Analytics for Precision Medicine</span>. <i>IEEE Trans. Biomed. Eng.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">263</span>– <span class="NLM_lpage">273</span>, <span class="refDoi"> DOI: 10.1109/TBME.2016.2573285</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1109%2FTBME.2016.2573285" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=27740470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A280%3ADC%252BC2srgt12itw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2017&pages=263-273&author=P.+Y.+Wuauthor=C.+W.+Chengauthor=C.+D.+Kaddiauthor=J.+Venugopalanauthor=R.+Hoffmanauthor=M.+D.+Wang&title=-Omic+and+Electronic+Health+Record+Big+Data+Analytics+for+Precision+Medicine&doi=10.1109%2FTBME.2016.2573285"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">-Omic and Electronic Health Record Big Data Analytics for Precision Medicine</span></div><div class="casAuthors">Wu Po-Yen; Cheng Chih-Wen; Kaddi Chanchala D; Venugopalan Janani; Hoffman Ryan; Wang May D</div><div class="citationInfo"><span class="NLM_cas:title">IEEE transactions on bio-medical engineering</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">263-273</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">OBJECTIVE:  Rapid advances of high-throughput technologies and wide adoption of electronic health records (EHRs) have led to fast accumulation of -omic and EHR data.  These voluminous complex data contain abundant information for precision medicine, and big data analytics can extract such knowledge to improve the quality of healthcare.  METHODS:  In this paper, we present -omic and EHR data characteristics, associated challenges, and data analytics including data preprocessing, mining, and modeling.  RESULTS:  To demonstrate how big data analytics enables precision medicine, we provide two case studies, including identifying disease biomarkers from multi-omic data and incorporating -omic information into EHR.  CONCLUSION:  Big data analytics is able to address -omic and EHR data challenges for paradigm shift toward precision medicine.  SIGNIFICANCE:  Big data analytics makes sense of -omic and EHR data to improve healthcare outcome.  It has long lasting societal impact.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS1QO5cfWToYj69zv1FsJFMfW6udTcc2ebvB-LcsarQR7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srgt12itw%253D%253D&md5=d86fcc963fd2e0d7defaf2b8042ed5b4</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1109%2FTBME.2016.2573285&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1109%252FTBME.2016.2573285%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%2BY.%26aulast%3DCheng%26aufirst%3DC.%2BW.%26aulast%3DKaddi%26aufirst%3DC.%2BD.%26aulast%3DVenugopalan%26aufirst%3DJ.%26aulast%3DHoffman%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DM.%2BD.%26atitle%3D-Omic%2520and%2520Electronic%2520Health%2520Record%2520Big%2520Data%2520Analytics%2520for%2520Precision%2520Medicine%26jtitle%3DIEEE%2520Trans.%2520Biomed.%2520Eng.%26date%3D2017%26volume%3D64%26spage%3D263%26epage%3D273%26doi%3D10.1109%2FTBME.2016.2573285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Khosravi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goliaei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masoudi-Nejad, A.</span></span> <span> </span><span class="NLM_article-title">Active Repurposing of Drug Candidates for Melanoma Based on GWAS, PheWAS and a Wide Range of Omics Data</span>. <i>Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">30</span>, <span class="refDoi"> DOI: 10.1186/s10020-019-0098-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1186%2Fs10020-019-0098-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=31221082" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A280%3ADC%252BB3M3ovFejsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2019&pages=30&author=A.+Khosraviauthor=B.+Jayaramauthor=B.+Goliaeiauthor=A.+Masoudi-Nejad&title=Active+Repurposing+of+Drug+Candidates+for+Melanoma+Based+on+GWAS%2C+PheWAS+and+a+Wide+Range+of+Omics+Data&doi=10.1186%2Fs10020-019-0098-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Active repurposing of drug candidates for melanoma based on GWAS, PheWAS and a wide range of omics data</span></div><div class="casAuthors">Khosravi Ali; Masoudi-Nejad Ali; Jayaram B; Goliaei Bahram</div><div class="citationInfo"><span class="NLM_cas:title">Molecular medicine (Cambridge, Mass.)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">30</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Drug repurposing is a swift, safe, and cheap drug discovery method.  Melanoma disorders present low survival and high mortality rates and are challenging to diagnose and treat.  Moreover, there is a high volume of worldwide investigations that are attempting to find melanoma-related genes of influence, which can be identified as responsive targets for reliable treatment.  METHOD:  In this study, we used a wide range of data analyses to analyze over 1100 genes and proteins of influence with respect to cutaneous malignant melanoma.  Our analysis included various investigational results from genome- and phenome-wide association studies (GWAS and PheWAS, respectively), biomedical, transcriptomic, and metabolomic datasets.  We then researched the DrugBank for potential melanoma targets from the selected list.  We excluded known melanoma targets to obtain a list of druggable proteins.  We performed a precise analysis of the drugs' pathogenesis and checked the expression profiles of the selected drugs having high associations with known anti-melanoma drugs.  RESULT:  We found 35 drugs that interacted with 20 unique targets.  These drugs appear to have high melanoma treatment potentials.  We confirmed our results with previous studies and found supporting references for 30 of these drugs.  In conclusion, this investigation can be applied to various diseases for the efficient and economical repurposing of various drug compounds.  For further validation, the results may be applicable for in vivo tests and clinical trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRegUNtOaqiXMkU0YCbOU3EfW6udTcc2eZv0hjwVDBelrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3ovFejsw%253D%253D&md5=acdf7ca4dae0cf3610a7a0a60ca7a97c</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1186%2Fs10020-019-0098-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs10020-019-0098-x%26sid%3Dliteratum%253Aachs%26aulast%3DKhosravi%26aufirst%3DA.%26aulast%3DJayaram%26aufirst%3DB.%26aulast%3DGoliaei%26aufirst%3DB.%26aulast%3DMasoudi-Nejad%26aufirst%3DA.%26atitle%3DActive%2520Repurposing%2520of%2520Drug%2520Candidates%2520for%2520Melanoma%2520Based%2520on%2520GWAS%252C%2520PheWAS%2520and%2520a%2520Wide%2520Range%2520of%2520Omics%2520Data%26jtitle%3DMol.%2520Med.%26date%3D2019%26volume%3D25%26spage%3D30%26doi%3D10.1186%2Fs10020-019-0098-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shannon, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markiel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozier, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baliga, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramage, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwikowski, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ideker, T.</span></span> <span> </span><span class="NLM_article-title">Cytoscape: A Software Environment for Integrated Models of Biomolecular Interaction Networks</span>. <i>Genome Res.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2498</span>– <span class="NLM_lpage">2504</span>, <span class="refDoi"> DOI: 10.1101/gr.1239303</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1101%2Fgr.1239303" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=14597658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BD3sXovFWrtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=2498-2504&author=P.+Shannonauthor=A.+Markielauthor=O.+Ozierauthor=N.+S.+Baligaauthor=J.+T.+Wangauthor=D.+Ramageauthor=N.+Aminauthor=B.+Schwikowskiauthor=T.+Ideker&title=Cytoscape%3A+A+Software+Environment+for+Integrated+Models+of+Biomolecular+Interaction+Networks&doi=10.1101%2Fgr.1239303"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Cytoscape: A software environment for integrated models of biomolecular interaction networks</span></div><div class="casAuthors">Shannon, Paul; Markiel, Andrew; Ozier, Owen; Baliga, Nitin S.; Wang, Jonathan T.; Ramage, Daniel; Amin, Nada; Schwikowski, Benno; Ideker, Trey</div><div class="citationInfo"><span class="NLM_cas:title">Genome Research</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2498-2504</span>CODEN:
                <span class="NLM_cas:coden">GEREFS</span>;
        ISSN:<span class="NLM_cas:issn">1088-9051</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Cytoscape is an open source software project for integrating biomol. interaction networks with high-throughput expression data and other mol. states into a unified conceptual framework.  Although applicable to any system of mol. components and interactions, Cytoscape is most powerful when used in conjunction with large databases of protein-protein, protein-DNA, and genetic interactions that are increasingly available for humans and model organisms.  Cytoscape's software Core provides basic functionality to layout and query the network; to visually integrate the network with expression profiles, phenotypes, and other mol. states; and to link the network to databases of functional annotations.  The Core is extensible through a straightforward plug-in architecture, allowing rapid development of addnl. computational analyses and features.  Several case studies of Cytoscape plug-ins are surveyed, including a search for interaction pathways correlating with changes in gene expression, a study of protein complexes involved in cellular recovery to DNA damage, inference of a combined phys./functional interaction network for Halobacterium, and an interface to detailed stochastic/kinetic gene regulatory models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpz_cu7mgf2RbVg90H21EOLACvtfcHk0li0Qml0lLvQ0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXovFWrtr4%253D&md5=2bcbca9a3bd04717761f0424c0209e43</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1101%2Fgr.1239303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgr.1239303%26sid%3Dliteratum%253Aachs%26aulast%3DShannon%26aufirst%3DP.%26aulast%3DMarkiel%26aufirst%3DA.%26aulast%3DOzier%26aufirst%3DO.%26aulast%3DBaliga%26aufirst%3DN.%2BS.%26aulast%3DWang%26aufirst%3DJ.%2BT.%26aulast%3DRamage%26aufirst%3DD.%26aulast%3DAmin%26aufirst%3DN.%26aulast%3DSchwikowski%26aufirst%3DB.%26aulast%3DIdeker%26aufirst%3DT.%26atitle%3DCytoscape%253A%2520A%2520Software%2520Environment%2520for%2520Integrated%2520Models%2520of%2520Biomolecular%2520Interaction%2520Networks%26jtitle%3DGenome%2520Res.%26date%3D2003%26volume%3D13%26spage%3D2498%26epage%3D2504%26doi%3D10.1101%2Fgr.1239303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bastian, M.</span>; <span class="NLM_string-name">Heymann, S.</span>; <span class="NLM_string-name">Jacomy, M.</span></span> <span> </span><span class="NLM_article-title">Gephi: An Open Source Software for Exploring and Manipulating Networks</span>. Presented at the Third International AAAI Conference on Weblogs and Social Media, <span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=M.+Bastian&author=S.+Heymann&author=M.+Jacomy&title=Gephi%3A+An+Open+Source+Software+for+Exploring+and+Manipulating+Networks"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DBastian%26aufirst%3DM.%26atitle%3DGephi%253A%2520An%2520Open%2520Source%2520Software%2520for%2520Exploring%2520and%2520Manipulating%2520Networks%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de Nooy, W.</span>; <span class="NLM_string-name">Mrvar, A.</span>; <span class="NLM_string-name">Batagelj, V.</span></span> <i>Exploratory Social Network Analysis with Pajek: Revised and Expanded Edition for Updated Software</i>; <span class="NLM_publisher-name">Cambridge University Press</span>: <span class="NLM_publisher-loc">Cambridge, U.K.</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1017%2F9781108565691" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=W.+de+Nooy&author=A.+Mrvar&author=V.+Batagelj&title=Exploratory+Social+Network+Analysis+with+Pajek%3A+Revised+and+Expanded+Edition+for+Updated+Software"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1017%2F9781108565691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1017%252F9781108565691%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BNooy%26aufirst%3DW.%26btitle%3DExploratory%2520Social%2520Network%2520Analysis%2520with%2520Pajek%253A%2520Revised%2520and%2520Expanded%2520Edition%2520for%2520Updated%2520Software%26pub%3DCambridge%2520University%2520Press%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hagberg, A. A.</span>; <span class="NLM_string-name">Schult, D. A.</span>; <span class="NLM_string-name">Swart, P. J.</span></span> <span> </span><span class="NLM_article-title">Exploring Network Structure, Dynamics, and Function Using NetworkX</span>. In  <i>Proceedings of the 7th Python in Science Conference (SciPy 2008)</i>, Pasadena, CA, U.S.; <span class="NLM_year">2008</span>; pp  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">15</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&pages=11-15&author=A.+A.+Hagberg&author=D.+A.+Schult&author=P.+J.+Swart&title=Proceedings+of+the+7th+Python+in+Science+Conference+%28SciPy+2008%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DHagberg%26aufirst%3DA.%2BA.%26atitle%3DExploring%2520Network%2520Structure%252C%2520Dynamics%252C%2520and%2520Function%2520Using%2520NetworkX%26btitle%3DProceedings%2520of%2520the%25207th%2520Python%2520in%2520Science%2520Conference%2520%2528SciPy%25202008%2529%26date%3D2008%26spage%3D11%26epage%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pavlopoulos, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paez-Espino, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kyrpides, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iliopoulos, I.</span></span> <span> </span><span class="NLM_article-title">Empirical Comparison of Visualization Tools for Larger-Scale Network Analysis</span>. <i>Adv. Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>2017</i></span>,  <span class="NLM_fpage">1278932</span>, <span class="refDoi"> DOI: 10.1155/2017/1278932</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1155%2F2017%2F1278932" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=28804499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A280%3ADC%252BC1cfnt1WntQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2017&publication_year=2017&pages=1278932&author=G.+A.+Pavlopoulosauthor=D.+Paez-Espinoauthor=N.+C.+Kyrpidesauthor=I.+Iliopoulos&title=Empirical+Comparison+of+Visualization+Tools+for+Larger-Scale+Network+Analysis&doi=10.1155%2F2017%2F1278932"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Empirical Comparison of Visualization Tools for Larger-Scale Network Analysis</span></div><div class="casAuthors">Pavlopoulos Georgios A; Paez-Espino David; Kyrpides Nikos C; Iliopoulos Ioannis</div><div class="citationInfo"><span class="NLM_cas:title">Advances in bioinformatics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">2017</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1278932</span>
        ISSN:<span class="NLM_cas:issn">1687-8027</span>.
    </div><div class="casAbstract">Gene expression, signal transduction, protein/chemical interactions, biomedical literature cooccurrences, and other concepts are often captured in biological network representations where nodes represent a certain bioentity and edges the connections between them.  While many tools to manipulate, visualize, and interactively explore such networks already exist, only few of them can scale up and follow today's indisputable information growth.  In this review, we shortly list a catalog of available network visualization tools and, from a user-experience point of view, we identify four candidate tools suitable for larger-scale network analysis, visualization, and exploration.  We comment on their strengths and their weaknesses and empirically discuss their scalability, user friendliness, and postvisualization capabilities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSSEd2tJNwnQjDnmmyUf5qOfW6udTcc2ebwZ4kYtB41v7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cfnt1WntQ%253D%253D&md5=2615dcef0c9fad3aea83f4305aebe5e3</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1155%2F2017%2F1278932&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2017%252F1278932%26sid%3Dliteratum%253Aachs%26aulast%3DPavlopoulos%26aufirst%3DG.%2BA.%26aulast%3DPaez-Espino%26aufirst%3DD.%26aulast%3DKyrpides%26aufirst%3DN.%2BC.%26aulast%3DIliopoulos%26aufirst%3DI.%26atitle%3DEmpirical%2520Comparison%2520of%2520Visualization%2520Tools%2520for%2520Larger-Scale%2520Network%2520Analysis%26jtitle%3DAdv.%2520Bioinf.%26date%3D2017%26volume%3D2017%26spage%3D1278932%26doi%3D10.1155%2F2017%2F1278932" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manikandan, S. G.</span>; <span class="NLM_string-name">Ravi, S.</span></span> <span> </span><span class="NLM_article-title">Big Data Analysis Using Apache Hadoop</span>.  <i>Proceedings</i>, 2014 International Conference on IT Convergence and Security (ICITCS), Beijing, China, 2014; <span class="NLM_publisher-name">IEEE</span>, <span class="NLM_year">2014</span>; <span class="refDoi"> DOI: 10.1109/ICITCS.2014.7021746</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1109%2FICITCS.2014.7021746" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=S.+G.+Manikandan&author=S.+Ravi&title=Proceedings"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1109%2FICITCS.2014.7021746&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1109%252FICITCS.2014.7021746%26sid%3Dliteratum%253Aachs%26aulast%3DManikandan%26aufirst%3DS.%2BG.%26atitle%3DBig%2520Data%2520Analysis%2520Using%2520Apache%2520Hadoop%26btitle%3DProceedings%26pub%3DIEEE%26date%3D2014%26doi%3D10.1109%2FICITCS.2014.7021746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zaharia, M.</span>; <span class="NLM_string-name">Chowdhury, M.</span>; <span class="NLM_string-name">Franklin, M. J.</span>; <span class="NLM_string-name">Shenker, S.</span>; <span class="NLM_string-name">Stoica, I.</span></span> <span> </span><span class="NLM_article-title">Spark: Cluster Computing with Working Sets</span>.  <i>HotCloud ’10: Proceedings of the 2nd USENIX Conference on Hot Topics in Cloud Computing</i>; <span class="NLM_publisher-name">USENIX Association</span>, <span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=M.+Zaharia&author=M.+Chowdhury&author=M.+J.+Franklin&author=S.+Shenker&author=I.+Stoica&title=HotCloud+%E2%80%9910%3A+Proceedings+of+the+2nd+USENIX+Conference+on+Hot+Topics+in+Cloud+Computing"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DZaharia%26aufirst%3DM.%26atitle%3DSpark%253A%2520Cluster%2520Computing%2520with%2520Working%2520Sets%26btitle%3DHotCloud%2520%25E2%2580%259910%253A%2520Proceedings%2520of%2520the%25202nd%2520USENIX%2520Conference%2520on%2520Hot%2520Topics%2520in%2520Cloud%2520Computing%26pub%3DUSENIX%2520Association%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maggiora, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Chemical Space Networks: A Powerful New Paradigm for the Description of Chemical Space</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">795</span>– <span class="NLM_lpage">802</span>, <span class="refDoi"> DOI: 10.1007/s10822-014-9760-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1007%2Fs10822-014-9760-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=24925682" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpvVeisbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=795-802&author=G.+M.+Maggioraauthor=J.+Bajorath&title=Chemical+Space+Networks%3A+A+Powerful+New+Paradigm+for+the+Description+of+Chemical+Space&doi=10.1007%2Fs10822-014-9760-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical space networks: a powerful new paradigm for the description of chemical space</span></div><div class="casAuthors">Maggiora, Gerald M.; Bajorath, Jurgen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">795-802</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  The concept of chem. space is playing an increasingly important role in many areas of chem. research, esp. medicinal chem. and chem. biol.  It is generally conceived as consisting of numerous compd. clusters of varying sizes scattered throughout the space in much the same way as galaxies of stars inhabit our universe.  A no. of issues assocd. with this coordinate-based representation are discussed.  Not the least of which is the continuous nature of the space, a feature not entirely compatible with the inherently discrete nature of chem. space.  Cell-based representations, which are derived from coordinate-based spaces, have also been developed that facilitate a no. of chem. informatic activities (e.g., diverse subset selection, filling 'diversity voids', and comparing compd. collections).  These representations generally suffer the 'curse of dimensionality'.  In this work, networks are proposed as an attractive paradigm for representing chem. space since they circumvent many of the issues assocd. with coordinate- and cell-based representations, including the curse of dimensionality.  In addn., their relational structure is entirely compatible with the intrinsic nature of chem. space.  A description of the features of these chem. space networks is presented that emphasizes their statistical characteristics and indicates how they are related to various types of network topologies that exhibit random, scale-free, and/or 'small world' properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqtDD1mJ2GSLVg90H21EOLACvtfcHk0liVhfjsxc4_Dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpvVeisbY%253D&md5=fed9b69e5aef335c4e0670120e652848</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1007%2Fs10822-014-9760-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-014-9760-0%26sid%3Dliteratum%253Aachs%26aulast%3DMaggiora%26aufirst%3DG.%2BM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DChemical%2520Space%2520Networks%253A%2520A%2520Powerful%2520New%2520Paradigm%2520for%2520the%2520Description%2520of%2520Chemical%2520Space%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2014%26volume%3D28%26spage%3D795%26epage%3D802%26doi%3D10.1007%2Fs10822-014-9760-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stumpfe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maggiora, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Lessons Learned from the Design of Chemical Space Networks and Opportunities for New Applications</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">191</span>– <span class="NLM_lpage">208</span>, <span class="refDoi"> DOI: 10.1007/s10822-016-9906-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1007%2Fs10822-016-9906-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=26945865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC28XjvVGhsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=191-208&author=M.+Vogtauthor=D.+Stumpfeauthor=G.+M.+Maggioraauthor=J.+Bajorath&title=Lessons+Learned+from+the+Design+of+Chemical+Space+Networks+and+Opportunities+for+New+Applications&doi=10.1007%2Fs10822-016-9906-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons learned from the design of chemical space networks and opportunities for new applications</span></div><div class="casAuthors">Vogt, Martin; Stumpfe, Dagmar; Maggiora, Gerald M.; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">191-208</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The concept of chem. space is of fundamental relevance in chem. informatics and computer-aided drug discovery.  In a series of articles published in the Journal of Computer-Aided Mol. Design, principles of chem. space design were evaluated, mol. networks proposed as an alternative to conventional coordinate-based chem. ref. spaces, and different types of chem. space networks (CSNs) constructed and analyzed.  Central to the generation of CSNs was the way in which mol. similarity relationships were assessed and a primary focal point was the network-based representation of biol. relevant chem. space.  The design and comparison of CSNs based upon alternative similarity measures can be viewed as an evolutionary path with interesting lessons learned along the way.  CSN design has matured to the point that such chem. space representations can be used in practice.  In this contribution, highlights from the sequence of CSN design efforts are discussed in context, providing a perspective for future practical applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA5JT9f_RfpbVg90H21EOLACvtfcHk0liqUyhC1sJ7gQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjvVGhsro%253D&md5=e5607c1fe6b9a5e74a55e8e97464bb22</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1007%2Fs10822-016-9906-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-016-9906-3%26sid%3Dliteratum%253Aachs%26aulast%3DVogt%26aufirst%3DM.%26aulast%3DStumpfe%26aufirst%3DD.%26aulast%3DMaggiora%26aufirst%3DG.%2BM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DLessons%2520Learned%2520from%2520the%2520Design%2520of%2520Chemical%2520Space%2520Networks%2520and%2520Opportunities%2520for%2520New%2520Applications%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2016%26volume%3D30%26spage%3D191%26epage%3D208%26doi%3D10.1007%2Fs10822-016-9906-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maggiora, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Comparison of Bioactive Chemical Space Networks Generated Using Substructure- and Fingerprint-Based Measures of Molecular Similarity</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">595</span>– <span class="NLM_lpage">608</span>, <span class="refDoi"> DOI: 10.1007/s10822-015-9852-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1007%2Fs10822-015-9852-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=26049785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVSgtLjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=595-608&author=B.+Zhangauthor=M.+Vogtauthor=G.+M.+Maggioraauthor=J.+Bajorath&title=Comparison+of+Bioactive+Chemical+Space+Networks+Generated+Using+Substructure-+and+Fingerprint-Based+Measures+of+Molecular+Similarity&doi=10.1007%2Fs10822-015-9852-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of bioactive chemical space networks generated using substructure- and fingerprint-based measures of molecular similarity</span></div><div class="casAuthors">Zhang, Bijun; Vogt, Martin; Maggiora, Gerald M.; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">595-608</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Chem. space networks (CSNs) have recently been introduced as a conceptual alternative to coordinate-based representations of chem. space.  CSNs were initially designed as threshold networks using the Tanimoto coeff. as a continuous similarity measure.  The anal. of CSNs generated from sets of bioactive compds. revealed that many statistical properties were strongly dependent on their edge d.  While it was difficult to compare CSNs at pre-defined similarity threshold values, CSNs with const. edge d. were directly comparable.  In the current study, alternative CSN representations were constructed by applying the matched mol. pair (MMP) formalism as a substructure-based similarity criterion.  For more than 150 compd. activity classes, MMP-based CSNs (MMP-CSNs) were compared to corresponding threshold CSNs (THR-CSNs) at a const. edge d. by applying different parameters from network science, measures of community structure distributions, and indicators of structure-activity relationship (SAR) information content.  MMP-CSNs were found to be an attractive alternative to THR-CSNs, yielding low edge densities and well-resolved topologies.  MMP-CSNs and corresponding THR-CSNs often had similar topol. and closely corresponding community structures, although there was only limited overlap in similarity relationships.  The homophily principle from network science was shown to affect MMP-CSNs and THR-CSNs in different ways, despite the presence of conserved topol. features.  Moreover, activity cliff distributions in alternative CSN designs markedly differed, which has important implications for SAR anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQAfI8eb4WjbVg90H21EOLACvtfcHk0liqUyhC1sJ7gQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVSgtLjK&md5=057da71a8b5462862a850f3f484c3050</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1007%2Fs10822-015-9852-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-015-9852-5%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DVogt%26aufirst%3DM.%26aulast%3DMaggiora%26aufirst%3DG.%2BM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DComparison%2520of%2520Bioactive%2520Chemical%2520Space%2520Networks%2520Generated%2520Using%2520Substructure-%2520and%2520Fingerprint-Based%2520Measures%2520of%2520Molecular%2520Similarity%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26spage%3D595%26epage%3D608%26doi%3D10.1007%2Fs10822-015-9852-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zwierzyna, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maggiora, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Design and Characterization of Chemical Space Networks for Different Compound Data Sets</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">113</span>– <span class="NLM_lpage">125</span>, <span class="refDoi"> DOI: 10.1007/s10822-014-9821-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1007%2Fs10822-014-9821-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=25465052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVWgt7nM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=113-125&author=M.+Zwierzynaauthor=M.+Vogtauthor=G.+M.+Maggioraauthor=J.+Bajorath&title=Design+and+Characterization+of+Chemical+Space+Networks+for+Different+Compound+Data+Sets&doi=10.1007%2Fs10822-014-9821-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Design and characterization of chemical space networks for different compound data sets</span></div><div class="casAuthors">Zwierzyna, Magdalena; Vogt, Martin; Maggiora, Gerald M.; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">113-125</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Chem. Space Networks (CSNs) are generated for different compd. data sets on the basis of pairwise similarity relationships.  Such networks are thought to complement and further extend traditional coordinate-based views of chem. space.  Our proof-of-concept study focuses on CSNs based upon fingerprint similarity relationships calcd. using the conventional Tanimoto similarity metric.  The resulting CSNs are characterized with statistical measures from network science and compared in different ways.  We show that the homophily principle, which is widely considered in the context of social networks, is a major determinant of the topol. of CSNs of bioactive compds., designed as threshold networks, typically giving rise to community structures.  Many properties of CSNs are influenced by numerical features of the conventional Tanimoto similarity metric and largely dominated by the edge d. of the networks, which depends on chosen similarity threshold values.  However, properties of different CSNs with const. edge d. can be directly compared, revealing systematic differences between CSNs generated from randomly collected or bioactive compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1O5MgBoa4W7Vg90H21EOLACvtfcHk0liqUyhC1sJ7gQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVWgt7nM&md5=db2c790b6d162e7f0c14ec0f42ac374d</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1007%2Fs10822-014-9821-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-014-9821-4%26sid%3Dliteratum%253Aachs%26aulast%3DZwierzyna%26aufirst%3DM.%26aulast%3DVogt%26aufirst%3DM.%26aulast%3DMaggiora%26aufirst%3DG.%2BM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DDesign%2520and%2520Characterization%2520of%2520Chemical%2520Space%2520Networks%2520for%2520Different%2520Compound%2520Data%2520Sets%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26spage%3D113%26epage%3D125%26doi%3D10.1007%2Fs10822-014-9821-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, M.</span></span> <span> </span><span class="NLM_article-title">Extended-Connectivity Fingerprints</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">742</span>– <span class="NLM_lpage">754</span>, <span class="refDoi"> DOI: 10.1021/ci100050t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100050t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=742-754&author=D.+Rogersauthor=M.+Hahn&title=Extended-Connectivity+Fingerprints&doi=10.1021%2Fci100050t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Extended-Connectivity Fingerprints</span></div><div class="casAuthors">Rogers, David; Hahn, Mathew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">742-754</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Extended-connectivity fingerprints (ECFPs) are a novel class of topol. fingerprints for mol. characterization.  Historically, topol. fingerprints were developed for substructure and similarity searching.  ECFPs were developed specifically for structure-activity modeling.  ECFPs are circular fingerprints with a no. of useful qualities: they can be very rapidly calcd.; they are not predefined and can represent an essentially infinite no. of different mol. features (including stereochem. information); their features represent the presence of particular substructures, allowing easier interpretation of anal. results; and the ECFP algorithm can be tailored to generate different types of circular fingerprints, optimized for different uses.  While the use of ECFPs has been widely adopted and validated, a description of their implementation has not previously been presented in the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra43orI-Y7lbVg90H21EOLACvtfcHk0lh-BZgzJ-MskQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D&md5=cd6c736cd7a3d280b67f5316acce8006</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fci100050t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100050t%26sid%3Dliteratum%253Aachs%26aulast%3DRogers%26aufirst%3DD.%26aulast%3DHahn%26aufirst%3DM.%26atitle%3DExtended-Connectivity%2520Fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D742%26epage%3D754%26doi%3D10.1021%2Fci100050t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowrie, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Analysis and Comparison of 2D Fingerprints: Insights into Database Screening Performance Using Eight Fingerprint Methods</span>. <i>J. Mol. Graphics Modell.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">157</span>– <span class="NLM_lpage">170</span>, <span class="refDoi"> DOI: 10.1016/j.jmgm.2010.05.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1016%2Fj.jmgm.2010.05.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=20579912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFamtLrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2010&pages=157-170&author=J.+Duanauthor=S.+L.+Dixonauthor=J.+F.+Lowrieauthor=W.+Sherman&title=Analysis+and+Comparison+of+2D+Fingerprints%3A+Insights+into+Database+Screening+Performance+Using+Eight+Fingerprint+Methods&doi=10.1016%2Fj.jmgm.2010.05.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis and comparison of 2D fingerprints: Insights into database screening performance using eight fingerprint methods</span></div><div class="casAuthors">Duan, Jian-Xin; Dixon, Steven L.; Lowrie, Jeffrey F.; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">157-170</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Virtual screening is a widely used strategy in modern drug discovery and 2D fingerprint similarity is an important tool that has been successfully applied to retrieve active compds. from large datasets.  However, it is not always straightforward to select an appropriate fingerprint method and assocd. settings for a given problem.  Here, we applied eight different fingerprint methods, as implemented in the new cheminformatics package Canvas, on a well-validated dataset covering five targets.  The fingerprint methods include Linear, Dendritic, Radial, MACCS, MOLPRINT2D, Pairwise, Triplet, and Torsion.  We find that most fingerprints have similar retrieval rates on av.; however, each has special characteristics that distinguish its performance on different query mols. and ligand sets.  For example, some fingerprints exhibit a significant ligand size dependency whereas others are more robust with respect to variations in the query or active compds.  In cases where little information is known about the active ligands, MOLPRINT2D fingerprints produce the highest av. retrieval actives.  When multiple queries are available, we find that a fingerprint averaged over all query mols. is generally superior to fingerprints derived from single queries.  Finally, a complementarity metric is proposed to det. which fingerprint methods can be combined to improve screening results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWCcy4qE10YbVg90H21EOLACvtfcHk0lh-BZgzJ-MskQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFamtLrM&md5=f72d91018346b1e5ef97fc83f648bd86</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2010.05.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2010.05.008%26sid%3Dliteratum%253Aachs%26aulast%3DDuan%26aufirst%3DJ.%26aulast%3DDixon%26aufirst%3DS.%2BL.%26aulast%3DLowrie%26aufirst%3DJ.%2BF.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DAnalysis%2520and%2520Comparison%2520of%25202D%2520Fingerprints%253A%2520Insights%2520into%2520Database%2520Screening%2520Performance%2520Using%2520Eight%2520Fingerprint%2520Methods%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2010%26volume%3D29%26spage%3D157%26epage%3D170%26doi%3D10.1016%2Fj.jmgm.2010.05.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nothias-Esposito, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nothias, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Da Silva, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Retailleau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leyssen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roussi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Touboul, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paolini, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorrestein, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Litaudon, M.</span></span> <span> </span><span class="NLM_article-title">Investigation of Premyrsinane and Myrsinane Esters in <i>Euphorbia Cupanii</i> and <i>Euphobia Pithyusa</i> with <i>MS2LDA</i> and Combinatorial Molecular Network Annotation Propagation</span>. <i>J. Nat. Prod.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>82</i></span>,  <span class="NLM_fpage">1459</span>– <span class="NLM_lpage">1470</span>, <span class="refDoi"> DOI: 10.1021/acs.jnatprod.8b00916</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jnatprod.8b00916" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtV2htr7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2019&pages=1459-1470&author=M.+Nothias-Espositoauthor=L.+F.+Nothiasauthor=R.+R.+Da+Silvaauthor=P.+Retailleauauthor=Z.+Zhangauthor=P.+Leyssenauthor=F.+Roussiauthor=D.+Touboulauthor=J.+Paoliniauthor=P.+C.+Dorresteinauthor=M.+Litaudon&title=Investigation+of+Premyrsinane+and+Myrsinane+Esters+in+Euphorbia+Cupanii+and+Euphobia+Pithyusa+with+MS2LDA+and+Combinatorial+Molecular+Network+Annotation+Propagation&doi=10.1021%2Facs.jnatprod.8b00916"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Investigation of Premyrsinane and Myrsinane Esters in Euphorbia cupanii and Euphobia pithyusa with MS2LDA and Combinatorial Molecular Network Annotation Propagation</span></div><div class="casAuthors">Nothias-Esposito, Melissa; Nothias, Louis Felix; Da Silva, Ricardo R.; Retailleau, Pascal; Zhang, Zheng; Leyssen, Pieter; Roussi, Fanny; Touboul, David; Paolini, Julien; Dorrestein, Pieter C.; Litaudon, Marc</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Natural Products</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1459-1470</span>CODEN:
                <span class="NLM_cas:coden">JNPRDF</span>;
        ISSN:<span class="NLM_cas:issn">0163-3864</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society-American Society of Pharmacognosy</span>)
        </div><div class="casAbstract">The species Euphorbia pithyusa and Euphorbia cupanii are two closely related Mediterranean spurges for which their taxonomic relationships are still being debated.  Herein, the diterpene ester content of E. cupanii was investigated using liq. chromatog. coupled to tandem mass spectrometry.  The use of mol. networking coupled to unsupervised substructure annotation (MS2LDA) indicated the presence of new premyrsinane/myrsinane diterpene esters in the E. cupanii fractions.  A structure-guided isolation procedure yielded 16 myrsinane (11a-h, 12, and 13) and premyrsinane esters (14a-c and 15a-c), along with four 4β-phorbol esters (16a-c and 17) that showed inhibitory activity against chikungunya virus replication.  The structures of the 16 new compds. (11a-c, 11h, 12, 13, 14a-c, 15a-c, 16a-c, and 17) were characterized by NMR spectroscopy and X-ray crystallog.  To further uncover the diterpene ester content of these two species, the concept of combinatorial network annotation propagation (C-NAP) was developed.  By leveraging the fact that the diterpene esters of Euphorbia species are made up of limited building blocks, a combinatorial database of theor. structures was created and used for C-NAP that made possible the annotation of 123 premyrsinane or myrsinane esters, from which 74% are not found in any compd. database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-7jM0k3FODrVg90H21EOLACvtfcHk0lgKSpSnBF7yAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtV2htr7F&md5=2865494a678bcfd49e2c2ce4005932c5</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facs.jnatprod.8b00916&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jnatprod.8b00916%26sid%3Dliteratum%253Aachs%26aulast%3DNothias-Esposito%26aufirst%3DM.%26aulast%3DNothias%26aufirst%3DL.%2BF.%26aulast%3DDa%2BSilva%26aufirst%3DR.%2BR.%26aulast%3DRetailleau%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DLeyssen%26aufirst%3DP.%26aulast%3DRoussi%26aufirst%3DF.%26aulast%3DTouboul%26aufirst%3DD.%26aulast%3DPaolini%26aufirst%3DJ.%26aulast%3DDorrestein%26aufirst%3DP.%2BC.%26aulast%3DLitaudon%26aufirst%3DM.%26atitle%3DInvestigation%2520of%2520Premyrsinane%2520and%2520Myrsinane%2520Esters%2520in%2520Euphorbia%2520Cupanii%2520and%2520Euphobia%2520Pithyusa%2520with%2520MS2LDA%2520and%2520Combinatorial%2520Molecular%2520Network%2520Annotation%2520Propagation%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D2019%26volume%3D82%26spage%3D1459%26epage%3D1470%26doi%3D10.1021%2Facs.jnatprod.8b00916" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nothias, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vining, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meehan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esquenazi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorrestein, P. C.</span></span> <span> </span><span class="NLM_article-title">Molecular Networking As a Drug Discovery, Drug Metabolism, and Precision Medicine Strategy</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">143</span>– <span class="NLM_lpage">154</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2016.10.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1016%2Fj.tips.2016.10.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=27842887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslGru7vJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=143-154&author=R.+A.+Quinnauthor=L.+F.+Nothiasauthor=O.+Viningauthor=M.+Meehanauthor=E.+Esquenaziauthor=P.+C.+Dorrestein&title=Molecular+Networking+As+a+Drug+Discovery%2C+Drug+Metabolism%2C+and+Precision+Medicine+Strategy&doi=10.1016%2Fj.tips.2016.10.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Networking As a Drug Discovery, Drug Metabolism, and Precision Medicine Strategy</span></div><div class="casAuthors">Quinn, Robert A.; Nothias, Louis-Felix; Vining, Oliver; Meehan, Michael; Esquenazi, Eduardo; Dorrestein, Pieter C.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">143-154</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Mol. networking is a tandem mass spectrometry (MS/MS) data organizational approach that has been recently introduced in the drug discovery, metabolomics, and medical fields.  The chem. of mols. dictates how they will be fragmented by MS/MS in the gas phase and, therefore, two related mols. are likely to display similar fragment ion spectra.  Mol. networking organizes the MS/MS data as a relational spectral network thereby mapping the chem. that was detected in an MS/MS-based metabolomics expt.  Although the wider utility of mol. networking is just beginning to be recognized, in this review we highlight the principles behind mol. networking and its use for the discovery of therapeutic leads, monitoring drug metab., clin. diagnostics, and emerging applications in precision medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmWfd56pPGA7Vg90H21EOLACvtfcHk0lgKSpSnBF7yAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslGru7vJ&md5=6459b0d087969f208dc5321732619916</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2016.10.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2016.10.011%26sid%3Dliteratum%253Aachs%26aulast%3DQuinn%26aufirst%3DR.%2BA.%26aulast%3DNothias%26aufirst%3DL.%2BF.%26aulast%3DVining%26aufirst%3DO.%26aulast%3DMeehan%26aufirst%3DM.%26aulast%3DEsquenazi%26aufirst%3DE.%26aulast%3DDorrestein%26aufirst%3DP.%2BC.%26atitle%3DMolecular%2520Networking%2520As%2520a%2520Drug%2520Discovery%252C%2520Drug%2520Metabolism%252C%2520and%2520Precision%2520Medicine%2520Strategy%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2017%26volume%3D38%26spage%3D143%26epage%3D154%26doi%3D10.1016%2Fj.tips.2016.10.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miho, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roškar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greiff, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, S. T.</span></span> <span> </span><span class="NLM_article-title">Large-Scale Network Analysis Reveals the Sequence Space Architecture of Antibody Repertoires</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1321</span>, <span class="refDoi"> DOI: 10.1038/s41467-019-09278-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1038%2Fs41467-019-09278-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=30899025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A280%3ADC%252BB3cbmvVyntA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1321&author=E.+Mihoauthor=R.+Ro%C5%A1karauthor=V.+Greiffauthor=S.+T.+Reddy&title=Large-Scale+Network+Analysis+Reveals+the+Sequence+Space+Architecture+of+Antibody+Repertoires&doi=10.1038%2Fs41467-019-09278-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Large-scale network analysis reveals the sequence space architecture of antibody repertoires</span></div><div class="casAuthors">Miho Enkelejda; Reddy Sai T; Miho Enkelejda; Miho Enkelejda; Roskar Rok; Greiff Victor</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1321</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The architecture of mouse and human antibody repertoires is defined by the sequence similarity networks of the clones that compose them.  The major principles that define the architecture of antibody repertoires have remained largely unknown.  Here, we establish a high-performance computing platform to construct large-scale networks from comprehensive human and murine antibody repertoire sequencing datasets (>100,000 unique sequences).  Leveraging a network-based statistical framework, we identify three fundamental principles of antibody repertoire architecture: reproducibility, robustness and redundancy.  Antibody repertoire networks are highly reproducible across individuals despite high antibody sequence dissimilarity.  The architecture of antibody repertoires is robust to the removal of up to 50-90% of randomly selected clones, but fragile to the removal of public clones shared among individuals.  Finally, repertoire architecture is intrinsically redundant.  Our analysis provides guidelines for the large-scale network analysis of immune repertoires and may be used in the future to define disease-associated and synthetic repertoires.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS-XIX5oF9w5O98rul4nhSdfW6udTcc2ebgJ07HBxwwybntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cbmvVyntA%253D%253D&md5=700f40420ee30f58ee7a776bded3ff0e</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1038%2Fs41467-019-09278-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-019-09278-8%26sid%3Dliteratum%253Aachs%26aulast%3DMiho%26aufirst%3DE.%26aulast%3DRo%25C5%25A1kar%26aufirst%3DR.%26aulast%3DGreiff%26aufirst%3DV.%26aulast%3DReddy%26aufirst%3DS.%2BT.%26atitle%3DLarge-Scale%2520Network%2520Analysis%2520Reveals%2520the%2520Sequence%2520Space%2520Architecture%2520of%2520Antibody%2520Repertoires%26jtitle%3DNat.%2520Commun.%26date%3D2019%26volume%3D10%26spage%3D1321%26doi%3D10.1038%2Fs41467-019-09278-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhattacharyya, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vishveshwara, S.</span></span> <span> </span><span class="NLM_article-title">Protein Structure and Function: Looking through the Network of Side-Chain Interactions</span>. <i>Curr. Protein Pept. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">4</span>– <span class="NLM_lpage">25</span>, <span class="refDoi"> DOI: 10.2174/1389203716666150923105727</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.2174%2F1389203716666150923105727" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=4-25&author=M.+Bhattacharyyaauthor=S.+Ghoshauthor=S.+Vishveshwara&title=Protein+Structure+and+Function%3A+Looking+through+the+Network+of+Side-Chain+Interactions&doi=10.2174%2F1389203716666150923105727"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.2174%2F1389203716666150923105727&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1389203716666150923105727%26sid%3Dliteratum%253Aachs%26aulast%3DBhattacharyya%26aufirst%3DM.%26aulast%3DGhosh%26aufirst%3DS.%26aulast%3DVishveshwara%26aufirst%3DS.%26atitle%3DProtein%2520Structure%2520and%2520Function%253A%2520Looking%2520through%2520the%2520Network%2520of%2520Side-Chain%2520Interactions%26jtitle%3DCurr.%2520Protein%2520Pept.%2520Sci.%26date%3D2015%26volume%3D17%26spage%3D4%26epage%3D25%26doi%3D10.2174%2F1389203716666150923105727" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di
Paola, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Ruvo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paci, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santoni, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giuliani, A.</span></span> <span> </span><span class="NLM_article-title">Protein Contact Networks: An Emerging Paradigm in Chemistry</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">1598</span>– <span class="NLM_lpage">1613</span>, <span class="refDoi"> DOI: 10.1021/cr3002356</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr3002356" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslCgsLbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2013&pages=1598-1613&author=L.+Di%0APaolaauthor=M.+De+Ruvoauthor=P.+Paciauthor=D.+Santoniauthor=A.+Giuliani&title=Protein+Contact+Networks%3A+An+Emerging+Paradigm+in+Chemistry&doi=10.1021%2Fcr3002356"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Contact Networks: An Emerging Paradigm in Chemistry</span></div><div class="casAuthors">Di Paola, L.; De Ruvo, M.; Paci, P.; Santoni, D.; Giuliani, A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1598-1613</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Nowadays, many different fields of investigation ranging from systems biol. to elec. engineering, sociol., and statistical mechanics converge into the shared operational paradigm of complex network anal.  A massive advancement in the elucidation of general behavior of network systems made possible the generation of brand new graph theor. descriptors, at both single node and entire graph level, that could be useful in many fields of chem.  In this review we will deal with the protein 3D structures in terms of contact networks between amino acid residues.  This case allows for a straightforward formalization in topol. terms: the role of nodes (residues) and edges (contacts) is devoid of any ambiguity and the introduction of van der Waals radii of amino acids allows us to assign a motivated threshold for assigning contacts and building the network.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYqMg8w9WKQrVg90H21EOLACvtfcHk0ljMKUErtrNRWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslCgsLbJ&md5=2a4dbdea166906a22af0ee05ef521ab0</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fcr3002356&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr3002356%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BPaola%26aufirst%3DL.%26aulast%3DDe%2BRuvo%26aufirst%3DM.%26aulast%3DPaci%26aufirst%3DP.%26aulast%3DSantoni%26aufirst%3DD.%26aulast%3DGiuliani%26aufirst%3DA.%26atitle%3DProtein%2520Contact%2520Networks%253A%2520An%2520Emerging%2520Paradigm%2520in%2520Chemistry%26jtitle%3DChem.%2520Rev.%26date%3D2013%26volume%3D113%26spage%3D1598%26epage%3D1613%26doi%3D10.1021%2Fcr3002356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verkhivker, G. M.</span></span> <span> </span><span class="NLM_article-title">Molecular Dynamics Simulations and Modelling of the Residue Interaction Networks in the BRAF Kinase Complexes with Small Molecule Inhibitors: Probing the Allosteric Effects of Ligand-Induced Kinase Dimerization and Paradoxical Activation</span>. <i>Mol. BioSyst.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">3146</span>– <span class="NLM_lpage">3165</span>, <span class="refDoi"> DOI: 10.1039/C6MB00298F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1039%2FC6MB00298F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=27481329" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1CrsLrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=3146-3165&author=G.+M.+Verkhivker&title=Molecular+Dynamics+Simulations+and+Modelling+of+the+Residue+Interaction+Networks+in+the+BRAF+Kinase+Complexes+with+Small+Molecule+Inhibitors%3A+Probing+the+Allosteric+Effects+of+Ligand-Induced+Kinase+Dimerization+and+Paradoxical+Activation&doi=10.1039%2FC6MB00298F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics simulations and modelling of the residue interaction networks in the BRAF kinase complexes with small molecule inhibitors: probing the allosteric effects of ligand-induced kinase dimerization and paradoxical activation</span></div><div class="casAuthors">Verkhivker, G. M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3146-3165</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Protein kinases are central to proper functioning of cellular networks and are an integral part of many signal transduction pathways.  The family of protein kinases represents by far the largest and most important class of therapeutic targets in oncol.  Dimerization-induced activation has emerged as a common mechanism of allosteric regulation in BRAF kinases, which play an important role in growth factor signalling and human diseases.  Recent studies have revealed that most of the BRAF inhibitors can induce dimerization and paradoxically stimulate enzyme transactivation by conferring an active conformation in the second monomer of the kinase dimer.  The emerging connections between inhibitor binding and BRAF kinase domain dimerization have suggested a mol. basis of the activation mechanism in which BRAF inhibitors may allosterically modulate the stability of the dimerization interface and affect the organization of residue interaction networks in BRAF kinase dimers.  In this work, we integrated structural bioinformatics anal., mol. dynamics and binding free energy simulations with the protein structure network anal. of the BRAF crystal structures to det. dynamic signatures of BRAF conformations in complexes with different types of inhibitors and probe the mechanisms of the inhibitor-induced dimerization and paradoxical activation.  The results of this study highlight previously unexplored relationships between types of BRAF inhibitors, inhibitor-induced changes in the residue interaction networks and allosteric modulation of the kinase activity.  This study suggests a mechanism by which BRAF inhibitors could promote or interfere with the paradoxical activation of BRAF kinases, which may be useful in informing discovery efforts to minimize the unanticipated adverse biol. consequences of these therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVab0e2fdU27Vg90H21EOLACvtfcHk0ljMKUErtrNRWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1CrsLrF&md5=b2f2d7406801f07b4a3425da843b21b7</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1039%2FC6MB00298F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MB00298F%26sid%3Dliteratum%253Aachs%26aulast%3DVerkhivker%26aufirst%3DG.%2BM.%26atitle%3DMolecular%2520Dynamics%2520Simulations%2520and%2520Modelling%2520of%2520the%2520Residue%2520Interaction%2520Networks%2520in%2520the%2520BRAF%2520Kinase%2520Complexes%2520with%2520Small%2520Molecule%2520Inhibitors%253A%2520Probing%2520the%2520Allosteric%2520Effects%2520of%2520Ligand-Induced%2520Kinase%2520Dimerization%2520and%2520Paradoxical%2520Activation%26jtitle%3DMol.%2520BioSyst.%26date%3D2016%26volume%3D12%26spage%3D3146%26epage%3D3165%26doi%3D10.1039%2FC6MB00298F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verkhivker, G. M.</span></span> <span> </span><span class="NLM_article-title">Biophysical Simulations and Structure-Based Modeling of Residue Interaction Networks in the Tumor Suppressor Proteins Reveal Functional Role of Cancer Mutation Hotspots in Molecular Communication</span>. <i>Biochim. Biophys. Acta, Gen. Subj.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>1863</i></span>,  <span class="NLM_fpage">210</span>– <span class="NLM_lpage">225</span>, <span class="refDoi"> DOI: 10.1016/j.bbagen.2018.10.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1016%2Fj.bbagen.2018.10.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=30339916" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVSisrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1863&publication_year=2019&pages=210-225&author=G.+M.+Verkhivker&title=Biophysical+Simulations+and+Structure-Based+Modeling+of+Residue+Interaction+Networks+in+the+Tumor+Suppressor+Proteins+Reveal+Functional+Role+of+Cancer+Mutation+Hotspots+in+Molecular+Communication&doi=10.1016%2Fj.bbagen.2018.10.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Biophysical simulations and structure-based modeling of residue interaction networks in the tumor suppressor proteins reveal functional role of cancer mutation hotspots in molecular communication</span></div><div class="casAuthors">Verkhivker, Gennady M.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, General Subjects</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">1863</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">210-225</span>CODEN:
                <span class="NLM_cas:coden">BBGSB3</span>;
        ISSN:<span class="NLM_cas:issn">0304-4165</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In the current study, we have combined mol. simulations and energetic anal. with dynamics-based network modeling and perturbation response scanning to det. mol. signatures of mutational hotspot residues in the p53, PTEN, and SMAD4 tumor suppressor proteins.  By examg. structure, energetics and dynamics of these proteins, we have shown that inactivating mutations preferentially target a group of structurally stable residues that play a fundamental role in global propagation of dynamic fluctuations and mediating allosteric interaction networks.  Through integration of long-range perturbation dynamics and network-based approaches, we have quantified allosteric potential of residues in the studied proteins.  The results have revealed that mutational hotspot sites often correspond to high centrality mediating centers of the residue interaction networks that are responsible for coordination of global dynamic changes and allosteric signaling.  Our findings have also suggested that structurally stable mutational hotpots can act as major effectors of allosteric interactions and mutations in these positions are typically assocd. with severe phenotype.  Modeling of shortest inter-residue pathways has shown that mutational hotspot sites can also serve as key mediating bridges of allosteric communication in the p53 and PTEN protein structures.  Multiple regression models have indicated that functional significance of mutational hotspots can be strongly assocd. with the network signatures serving as robust predictors of crit. regulatory positions responsible for loss-of-function phenotype.  The results of this computational investigation are compared with the exptl. studies and reveal mol. signatures of mutational hotspots, providing a plausible rationale for explaining and localizing disease-causing mutations in tumor suppressor genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUXmMaHQLEnrVg90H21EOLACvtfcHk0ljmxktGhbdWiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVSisrvP&md5=8f260103f43d8bd3787de45f77d1feba</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2018.10.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2018.10.009%26sid%3Dliteratum%253Aachs%26aulast%3DVerkhivker%26aufirst%3DG.%2BM.%26atitle%3DBiophysical%2520Simulations%2520and%2520Structure-Based%2520Modeling%2520of%2520Residue%2520Interaction%2520Networks%2520in%2520the%2520Tumor%2520Suppressor%2520Proteins%2520Reveal%2520Functional%2520Role%2520of%2520Cancer%2520Mutation%2520Hotspots%2520in%2520Molecular%2520Communication%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gen.%2520Subj.%26date%3D2019%26volume%3D1863%26spage%3D210%26epage%3D225%26doi%3D10.1016%2Fj.bbagen.2018.10.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amusengeri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Astl, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lobb, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verkhivker, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bishop, Ö. T.</span></span> <span> </span><span class="NLM_article-title">Establishing Computational Approaches towards Identifying Malarial Allosteric Modulators: A Case Study of Plasmodium falciparum Hsp70s</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">5574</span>, <span class="refDoi"> DOI: 10.3390/ijms20225574</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.3390%2Fijms20225574" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BB3cXptV2qsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=5574&author=A.+Amusengeriauthor=L.+Astlauthor=K.+Lobbauthor=G.+M.+Verkhivkerauthor=%C3%96.+T.+Bishop&title=Establishing+Computational+Approaches+towards+Identifying+Malarial+Allosteric+Modulators%3A+A+Case+Study+of+Plasmodium+falciparum+Hsp70s&doi=10.3390%2Fijms20225574"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Establishing computational approaches towards identifying malarial allosteric modulators: a case study of Plasmodium falciparum Hsp70s</span></div><div class="casAuthors">Amusengeri, Arnold; Astl, Lindy; Lobb, Kevin; Verkhivker, Gennady M.; Bishop, Ozlem Tastan</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5574</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Combating malaria is almost a never-ending battle, as Plasmodium parasites develop resistance to the drugs used against them, as obsd. recently in artemisinin-based combination therapies.  The main concern now is if the resistant parasite strains spread from Southeast Asia to Africa, the continent hosting most malaria cases.  To prevent catastrophic results, we need to find non-conventional approaches.  Allosteric drug targeting sites and modulators might be a new hope for malarial treatments.  Heat shock proteins (HSPs) are potential malarial drug targets and have complex allosteric control mechanisms.  Yet, studies on designing allosteric modulators against them are limited.  Here, we identified allosteric modulators (SANC190 and SANC651) against P. falciparum Hsp70-1 and Hsp70-x, affecting the conformational dynamics of the proteins, delicately balanced by the endogenous ligands.  Previously, we established a pipeline to identify allosteric sites and modulators.  This study also further investigated alternative approaches to speed up the process by comparing all atom mol. dynamics simulations and dynamic residue network anal. with the coarse-grained (CG) versions of the calcns.  Betweenness centrality (BC) profiles for PfHsp70-1 and PfHsp70-x derived from CG simulations not only revealed similar trends but also pointed to the same functional regions and specific residues corresponding to BC profile peaks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCObllYOgg5bVg90H21EOLACvtfcHk0ljmxktGhbdWiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXptV2qsrY%253D&md5=111587e894a0e6163c60bd49471b6db8</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.3390%2Fijms20225574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms20225574%26sid%3Dliteratum%253Aachs%26aulast%3DAmusengeri%26aufirst%3DA.%26aulast%3DAstl%26aufirst%3DL.%26aulast%3DLobb%26aufirst%3DK.%26aulast%3DVerkhivker%26aufirst%3DG.%2BM.%26aulast%3DBishop%26aufirst%3D%25C3%2596.%2BT.%26atitle%3DEstablishing%2520Computational%2520Approaches%2520towards%2520Identifying%2520Malarial%2520Allosteric%2520Modulators%253A%2520A%2520Case%2520Study%2520of%2520Plasmodium%2520falciparum%2520Hsp70s%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2019%26volume%3D20%26spage%3D5574%26doi%3D10.3390%2Fijms20225574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barabási, A.-L.</span></span> <span> </span><span class="NLM_article-title">Network Medicine — From Obesity to the “Diseasome”</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>357</i></span>,  <span class="NLM_fpage">404</span>– <span class="NLM_lpage">407</span>, <span class="refDoi"> DOI: 10.1056/NEJMe078114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1056%2FNEJMe078114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=17652657" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BD2sXotFCqu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=357&publication_year=2007&pages=404-407&author=A.-L.+Barab%C3%A1si&title=Network+Medicine+%E2%80%94+From+Obesity+to+the+%E2%80%9CDiseasome%E2%80%9D&doi=10.1056%2FNEJMe078114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Network medicine - from obesity to the "diseasome"</span></div><div class="casAuthors">Barabasi, Albert-Laszlo</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">357</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">404-407</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfYNygODU5XbVg90H21EOLACvtfcHk0ljmxktGhbdWiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXotFCqu7w%253D&md5=d752566c157722c1e2e5b0e7e1fe20bf</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1056%2FNEJMe078114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMe078114%26sid%3Dliteratum%253Aachs%26aulast%3DBarab%25C3%25A1si%26aufirst%3DA.-L.%26atitle%3DNetwork%2520Medicine%2520%25E2%2580%2594%2520From%2520Obesity%2520to%2520the%2520%25E2%2580%259CDiseasome%25E2%2580%259D%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2007%26volume%3D357%26spage%3D404%26epage%3D407%26doi%3D10.1056%2FNEJMe078114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goh, K.,  II</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cusick, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valle, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barabási, A. L.</span></span> <span> </span><span class="NLM_article-title">The Human Disease Network</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">8685</span>– <span class="NLM_lpage">8690</span>, <span class="refDoi"> DOI: 10.1073/pnas.0701361104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1073%2Fpnas.0701361104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=17502601" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmt1Wisbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=8685-8690&author=K.+Gohauthor=M.+E.+Cusickauthor=D.+Valleauthor=B.+Childsauthor=M.+Vidalauthor=A.+L.+Barab%C3%A1si&title=The+Human+Disease+Network&doi=10.1073%2Fpnas.0701361104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">The human disease network</span></div><div class="casAuthors">Goh, Kwang-Il; Cusick, Michael E.; Valle, David; Childs, Barton; Vidal, Marc; Barabasi, Albert-Laszlo</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8685-8690</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A network of disorders and disease genes linked by known disorder-gene assocns. offers a platform to explore in a single graph-theoretic framework all known phenotype and disease gene assocns., indicating the common genetic origin of many diseases.  Genes assocd. with similar disorders show both higher likelihood of phys. interactions between their products and higher expression profiling similarity for their transcripts, supporting the existence of distinct disease-specific functional modules.  Essential human genes are found to likely encode hub proteins and are expressed widely in most tissues.  This suggests that disease genes also would play a central role in the human interactome.  In contrast, the vast majority of disease genes are nonessential and show no tendency to encode hub proteins, and their expression pattern indicates that they are localized in the functional periphery of the network.  A selection-based model explains the obsd. difference between essential and disease genes and also suggests that diseases caused by somatic mutations should not be peripheral, a prediction confirmed for cancer genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwE7npsmfFXbVg90H21EOLACvtfcHk0liDNyGr0OfI5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmt1Wisbg%253D&md5=58e78ecce834988a70c44c73c548a59b</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0701361104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0701361104%26sid%3Dliteratum%253Aachs%26aulast%3DGoh%26aufirst%3DK.%26aulast%3DCusick%26aufirst%3DM.%2BE.%26aulast%3DValle%26aufirst%3DD.%26aulast%3DChilds%26aufirst%3DB.%26aulast%3DVidal%26aufirst%3DM.%26aulast%3DBarab%25C3%25A1si%26aufirst%3DA.%2BL.%26atitle%3DThe%2520Human%2520Disease%2520Network%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D8685%26epage%3D8690%26doi%3D10.1073%2Fpnas.0701361104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vidal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cusick, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barabási, A. L.</span></span> <span> </span><span class="NLM_article-title">Interactome Networks and Human Disease</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">986</span>– <span class="NLM_lpage">998</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2011.02.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1016%2Fj.cell.2011.02.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=21414488" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsFCrtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2011&pages=986-998&author=M.+Vidalauthor=M.+E.+Cusickauthor=A.+L.+Barab%C3%A1si&title=Interactome+Networks+and+Human+Disease&doi=10.1016%2Fj.cell.2011.02.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Interactome Networks and Human Disease</span></div><div class="casAuthors">Vidal, Marc; Cusick, Michael E.; Barabasi, Albert-Laszlo</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">986-998</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Complex biol. systems and cellular networks may underlie most genotype to phenotype relationships.  Here, we review basic concepts in network biol., discussing different types of interactome networks and the insights that can come from analyzing them.  We elaborate on why interactome networks are important to consider in biol., how they can be mapped and integrated with each other, what global properties are starting to emerge from interactome network models, and how these properties may relate to human disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7qTUCJpaslbVg90H21EOLACvtfcHk0liDNyGr0OfI5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsFCrtrs%253D&md5=19cb64f69574390843eb2cc969f4a0cb</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.02.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.02.016%26sid%3Dliteratum%253Aachs%26aulast%3DVidal%26aufirst%3DM.%26aulast%3DCusick%26aufirst%3DM.%2BE.%26aulast%3DBarab%25C3%25A1si%26aufirst%3DA.%2BL.%26atitle%3DInteractome%2520Networks%2520and%2520Human%2520Disease%26jtitle%3DCell%26date%3D2011%26volume%3D144%26spage%3D986%26epage%3D998%26doi%3D10.1016%2Fj.cell.2011.02.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vidal, M.</span></span> <span> </span><span class="NLM_article-title">A Unifying View of 21st Century Systems Biology</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>583</i></span>,  <span class="NLM_fpage">3891</span>– <span class="NLM_lpage">3894</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2009.11.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1016%2Fj.febslet.2009.11.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=19913537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFams7nJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=583&publication_year=2009&pages=3891-3894&author=M.+Vidal&title=A+Unifying+View+of+21st+Century+Systems+Biology&doi=10.1016%2Fj.febslet.2009.11.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">A unifying view of 21st century systems biology</span></div><div class="casAuthors">Vidal, Marc</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">583</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">3891-3894</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The idea that multi-scale dynamic complex systems formed by interacting macromols. and metabolites, cells, organs and organisms underlie some of the most fundamental aspects of life was proposed by a few visionaries half a century ago.  Scientists are witnessing a powerful resurgence of this idea made possible by the availability of nearly complete genome sequences, ever improving gene annotations and interactome network maps, the development of sophisticated informatic and imaging tools, and importantly, and the use of engineering and physics concepts such as control and graph theory.  Alongside 4 other fundamental "great ideas" as suggested by Sir Paul Nurse (P. Nurse, 2003), namely, the gene, the cell, the role of chem. in biol. processes, and evolution by natural selection, systems-level understanding of "what is life" may materialize as one of the major ideas of biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpc-z3XLH_jWbVg90H21EOLACvtfcHk0ljhXoHYx4dJtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFams7nJ&md5=44fb47f2892d2eafc357b8efc0fba2f9</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2009.11.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2009.11.024%26sid%3Dliteratum%253Aachs%26aulast%3DVidal%26aufirst%3DM.%26atitle%3DA%2520Unifying%2520View%2520of%252021st%2520Century%2520Systems%2520Biology%26jtitle%3DFEBS%2520Lett.%26date%3D2009%26volume%3D583%26spage%3D3891%26epage%3D3894%26doi%3D10.1016%2Fj.febslet.2009.11.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paolini, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapland, R. H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van
Hoorn, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span> <span> </span><span class="NLM_article-title">Global Mapping of Pharmacological Space</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">805</span>– <span class="NLM_lpage">815</span>, <span class="refDoi"> DOI: 10.1038/nbt1228</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1038%2Fnbt1228" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=16841068" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BD28XmvFansb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2006&pages=805-815&author=G.+V.+Paoliniauthor=R.+H.+B.+Shaplandauthor=W.+P.+Van%0AHoornauthor=J.+S.+Masonauthor=A.+L.+Hopkins&title=Global+Mapping+of+Pharmacological+Space&doi=10.1038%2Fnbt1228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Global mapping of pharmacological space</span></div><div class="casAuthors">Paolini, Gaia V.; Shapland, Richard H. B.; van Hoorn, Willem P.; Mason, Jonathan S.; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">805-815</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We present the global mapping of pharmacol. space by the integration of several vast sources of medicinal chem. structure-activity relationships (SAR) data.  Our comprehensive mapping of pharmacol. space enables us to identify confidently the human targets for which chem. tools and drugs have been discovered to date.  The integration of SAR data from diverse sources by unique canonical chem. structure, protein sequence and disease indication enables the construction of a ligand-target matrix to explore the global relationships between chem. structure and biol. targets.  Using the data matrix, we are able to catalog the links between proteins in chem. space as a polypharmacol. interaction network.  We demonstrate that probabilistic models can be used to predict pharmacol. from a large knowledge base.  The relationships between proteins, chem. structures and drug-like properties provide a framework for developing a probabilistic approach to drug discovery that can be exploited to increase research productivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoV_CE47zZq8LVg90H21EOLACvtfcHk0ljhXoHYx4dJtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmvFansb8%253D&md5=f559b34692cc903a1b503deb07030c5d</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1038%2Fnbt1228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1228%26sid%3Dliteratum%253Aachs%26aulast%3DPaolini%26aufirst%3DG.%2BV.%26aulast%3DShapland%26aufirst%3DR.%2BH.%2BB.%26aulast%3DVan%2BHoorn%26aufirst%3DW.%2BP.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DGlobal%2520Mapping%2520of%2520Pharmacological%2520Space%26jtitle%3DNat.%2520Biotechnol.%26date%3D2006%26volume%3D24%26spage%3D805%26epage%3D815%26doi%3D10.1038%2Fnbt1228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yildirim, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goh, K.,  II</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cusick, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barabási, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidal, M.</span></span> <span> </span><span class="NLM_article-title">Drug-Target Network</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1119</span>– <span class="NLM_lpage">1126</span>, <span class="refDoi"> DOI: 10.1038/nbt1338</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1038%2Fnbt1338" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=17921997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFagt7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=1119-1126&author=M.+A.+Yildirimauthor=K.+Gohauthor=M.+E.+Cusickauthor=A.+L.+Barab%C3%A1siauthor=M.+Vidal&title=Drug-Target+Network&doi=10.1038%2Fnbt1338"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-target network</span></div><div class="casAuthors">Yildirim, Muhammed A.; Goh, Kwang-Il; Cusick, Michael E.; Barabasi, Albert-Laszlo; Vidal, Marc</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1119-1126</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The global set of relationships between protein targets of all drugs and all disease-gene products in the human protein-protein interaction or 'interactome' network remains uncharacterized.  We built a bipartite graph composed of US Food and Drug Administration-approved drugs and proteins linked by drug-target binary assocns.  The resulting network connects most drugs into a highly interlinked giant component, with strong local clustering of drugs of similar types according to Anatomical Therapeutic Chem. classification.  Topol. analyses of this network quant. showed an overabundance of 'follow-on' drugs, i.e., drugs that target already targeted proteins.  By including drugs currently under investigation, we identified a trend toward more functionally diverse targets improving polypharmacol.  To analyze the relationships between drug targets and disease-gene products, we measured the shortest distance between both sets of proteins in current models of the human interactome network.  Significant differences in distance were found between etiol. and palliative drugs.  A recent trend toward more rational drug design was obsd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2KaaiDmAQ0LVg90H21EOLACvtfcHk0lgfTrsDHJYJuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFagt7rO&md5=d669b6a7afed3d9fb9d173e53b4ebead</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1038%2Fnbt1338&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1338%26sid%3Dliteratum%253Aachs%26aulast%3DYildirim%26aufirst%3DM.%2BA.%26aulast%3DGoh%26aufirst%3DK.%26aulast%3DCusick%26aufirst%3DM.%2BE.%26aulast%3DBarab%25C3%25A1si%26aufirst%3DA.%2BL.%26aulast%3DVidal%26aufirst%3DM.%26atitle%3DDrug-Target%2520Network%26jtitle%3DNat.%2520Biotechnol.%26date%3D2007%26volume%3D25%26spage%3D1119%26epage%3D1126%26doi%3D10.1038%2Fnbt1338" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guney, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menche, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barábasi, A. L.</span></span> <span> </span><span class="NLM_article-title">Network-Based in Silico Drug Efficacy Screening</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">10331</span>, <span class="refDoi"> DOI: 10.1038/ncomms10331</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1038%2Fncomms10331" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=26831545" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslOrsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=10331&author=E.+Guneyauthor=J.+Mencheauthor=M.+Vidalauthor=A.+L.+Bar%C3%A1basi&title=Network-Based+in+Silico+Drug+Efficacy+Screening&doi=10.1038%2Fncomms10331"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Network-based in silico drug efficacy screening</span></div><div class="casAuthors">Guney, Emre; Menche, Jorg; Vidal, Marc; Barabasi, Albert-Laszlo</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10331</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The increasing cost of drug development together with a significant drop in the no. of new drug approvals raises the need for innovative approaches for target identification and efficacy prediction.  Here, we take advantage of our increasing understanding of the network-based origins of diseases to introduce a drug-disease proximity measure that quantifies the interplay between drugs targets and diseases.  By correcting for the known biases of the interactome, proximity helps us uncover the therapeutic effect of drugs, as well as to distinguish palliative from effective treatments.  Our anal. of 238 drugs used in 78 diseases indicates that the therapeutic effect of drugs is localized in a small network neighborhood of the disease genes and highlights efficacy issues for drugs used in Parkinson and several inflammatory disorders.  Finally, network-based proximity allows us to predict novel drug-disease assocns. that offer unprecedented opportunities for drug repurposing and the detection of adverse effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqV7iKm7E1M9rVg90H21EOLACvtfcHk0lgfTrsDHJYJuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslOrsL4%253D&md5=125a8074aa7a953e35882d509731b49d</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1038%2Fncomms10331&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms10331%26sid%3Dliteratum%253Aachs%26aulast%3DGuney%26aufirst%3DE.%26aulast%3DMenche%26aufirst%3DJ.%26aulast%3DVidal%26aufirst%3DM.%26aulast%3DBar%25C3%25A1basi%26aufirst%3DA.%2BL.%26atitle%3DNetwork-Based%2520in%2520Silico%2520Drug%2520Efficacy%2520Screening%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D10331%26doi%3D10.1038%2Fncomms10331" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovács, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barabási, A. L.</span></span> <span> </span><span class="NLM_article-title">Network-Based Prediction of Drug Combinations</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1197</span>, <span class="refDoi"> DOI: 10.1038/s41467-019-09186-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1038%2Fs41467-019-09186-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=30867426" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A280%3ADC%252BB3cbjslelsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1197&author=F.+Chengauthor=I.+A.+Kov%C3%A1csauthor=A.+L.+Barab%C3%A1si&title=Network-Based+Prediction+of+Drug+Combinations&doi=10.1038%2Fs41467-019-09186-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Network-based prediction of drug combinations</span></div><div class="casAuthors">Cheng Feixiong; Kovacs Istvan A; Barabasi Albert-Laszlo; Cheng Feixiong; Kovacs Istvan A; Barabasi Albert-Laszlo; Cheng Feixiong; Cheng Feixiong; Cheng Feixiong; Barabasi Albert-Laszlo; Barabasi Albert-Laszlo</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1197</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Drug combinations, offering increased therapeutic efficacy and reduced toxicity, play an important role in treating multiple complex diseases.  Yet, our ability to identify and validate effective combinations is limited by a combinatorial explosion, driven by both the large number of drug pairs as well as dosage combinations.  Here we propose a network-based methodology to identify clinically efficacious drug combinations for specific diseases.  By quantifying the network-based relationship between drug targets and disease proteins in the human protein-protein interactome, we show the existence of six distinct classes of drug-drug-disease combinations.  Relying on approved drug combinations for hypertension and cancer, we find that only one of the six classes correlates with therapeutic effects: if the targets of the drugs both hit disease module, but target separate neighborhoods.  This finding allows us to identify and validate antihypertensive combinations, offering a generic, powerful network methodology to identify efficacious combination therapies in drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT3gy6XkP0VYS6Ufuk_1xoTfW6udTcc2ebBdf9awOzIWLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cbjslelsg%253D%253D&md5=bb39b64bbd05af1c25f99263a61f622f</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1038%2Fs41467-019-09186-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-019-09186-x%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DF.%26aulast%3DKov%25C3%25A1cs%26aufirst%3DI.%2BA.%26aulast%3DBarab%25C3%25A1si%26aufirst%3DA.%2BL.%26atitle%3DNetwork-Based%2520Prediction%2520of%2520Drug%2520Combinations%26jtitle%3DNat.%2520Commun.%26date%3D2019%26volume%3D10%26spage%3D1197%26doi%3D10.1038%2Fs41467-019-09186-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Conte, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiscon, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Licursi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bizzarri, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Antò, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farina, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paci, P.</span></span> <span> </span><span class="NLM_article-title">A Paradigm Shift in Medicine: A Comprehensive Review of Network-Based Approaches</span>. <i>Biochim. Biophys. Acta, Gene Regul. Mech.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>1863</i></span>,  <span class="NLM_fpage">194416</span>, <span class="refDoi"> DOI: 10.1016/j.bbagrm.2019.194416</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1016%2Fj.bbagrm.2019.194416" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=31382052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFGnsbzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1863&publication_year=2020&pages=194416&author=F.+Conteauthor=G.+Fisconauthor=V.+Licursiauthor=D.+Bizzarriauthor=T.+D%E2%80%99Ant%C3%B2author=L.+Farinaauthor=P.+Paci&title=A+Paradigm+Shift+in+Medicine%3A+A+Comprehensive+Review+of+Network-Based+Approaches&doi=10.1016%2Fj.bbagrm.2019.194416"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">A paradigm shift in medicine: A comprehensive review of network-based approaches</span></div><div class="casAuthors">Conte, Federica; Fiscon, Giulia; Licursi, Valerio; Bizzarri, Daniele; D'Anto, Tommaso; Farina, Lorenzo; Paci, Paola</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">1863</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">194416</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Network medicine is a rapidly evolving new field of medical research, which combines principles and approaches of systems biol. and network science, holding the promise to uncovering the causes and to revolutionize the diagnosis and treatments of human diseases.  This new paradigm reflects the fact that human diseases are not caused by single mol. defects, but driven by complex interactions among a variety of mol. mediators.  The complexity of these interactions embraces different types of information: from the cellular-mol. level of protein-protein interactions to correlational studies of gene expression and regulation, to metabolic and disease pathways up to drug-disease relationships.  The anal. of these complex networks can reveal new disease genes and/or disease pathways and identify possible targets for new drug development, as well as new uses for existing drugs.  In this review, we offer a comprehensive overview of network types and algorithms used in the framework of network medicine.  This article is part of a Special Issue entitled: Transcriptional Profiles and Regulatory Gene Networks edited by Dr.  Dr. Federico Manuel Giorgi and Dr.  Shaun Mahony.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrg_nwVOWC0-rVg90H21EOLACvtfcHk0lh5vslEHIoNVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFGnsbzO&md5=22ba9de49696544d6cbd8fbcb3ceafde</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2019.194416&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2019.194416%26sid%3Dliteratum%253Aachs%26aulast%3DConte%26aufirst%3DF.%26aulast%3DFiscon%26aufirst%3DG.%26aulast%3DLicursi%26aufirst%3DV.%26aulast%3DBizzarri%26aufirst%3DD.%26aulast%3DD%25E2%2580%2599Ant%25C3%25B2%26aufirst%3DT.%26aulast%3DFarina%26aufirst%3DL.%26aulast%3DPaci%26aufirst%3DP.%26atitle%3DA%2520Paradigm%2520Shift%2520in%2520Medicine%253A%2520A%2520Comprehensive%2520Review%2520of%2520Network-Based%2520Approaches%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gene%2520Regul.%2520Mech.%26date%3D2020%26volume%3D1863%26spage%3D194416%26doi%3D10.1016%2Fj.bbagrm.2019.194416" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sonawane, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glass, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, A.</span></span> <span> </span><span class="NLM_article-title">Network Medicine in the Age of Biomedical Big Data</span>. <i>Front. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">294</span>, <span class="refDoi"> DOI: 10.3389/fgene.2019.00294</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.3389%2Ffgene.2019.00294" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=31031797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlOnurbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=294&author=A.+R.+Sonawaneauthor=S.+T.+Weissauthor=K.+Glassauthor=A.+Sharma&title=Network+Medicine+in+the+Age+of+Biomedical+Big+Data&doi=10.3389%2Ffgene.2019.00294"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Network medicine in the age of biomedical big data</span></div><div class="casAuthors">Sonawane, Abhijeet R.; Weiss, Scott T.; Glass, Kimberly; Sharma, Amitabh</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Genetics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">294</span>CODEN:
                <span class="NLM_cas:coden">FGREBR</span>;
        ISSN:<span class="NLM_cas:issn">1664-8021</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">Network medicine is an emerging area of research dealing with mol. and genetic interactions, network biomarkers of disease, and therapeutic target discovery.  Largescale biomedical data generation offers a unique opportunity to assess the effect and impact of cellular heterogeneity and environmental perturbations on the obsd. phenotype.  Marrying the two, network medicine with biomedical data provides a framework to build meaningful models and ext. impactful results at a network level.  In this review, we survey existing network types and biomedical data sources.  More importantly, we delve into ways in which the network medicine approach, aided by phenotype-specific biomedical data, can be gainfully applied.  We provide three paradigms, mainly dealing with three major biol. network archetypes: proteinprotein interaction, expression-based, and gene regulatory networks.  For each of these paradigms, we discuss a broad overview of philosophies under which various network methods work.  We also provide a few examples in each paradigm as a test case of its successful application.  Finally, we delineate several opportunities and challenges in the field of network medicine.  We hope this review provides a lexicon for researchers from biol. sciences and network theory to come on the same page to work on research areas that require interdisciplinary expertise.  Taken together, the understanding gained from combining biomedical data with networks can be useful for characterizing disease etiologies and identifying therapeutic targets, which, in turn, will lead to better preventive medicine with translational impact on personalized healthcare.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2V3pM6h97y7Vg90H21EOLACvtfcHk0lh5vslEHIoNVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlOnurbK&md5=175633a8375ad95cef9c7c7cb9ee2f81</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.3389%2Ffgene.2019.00294&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffgene.2019.00294%26sid%3Dliteratum%253Aachs%26aulast%3DSonawane%26aufirst%3DA.%2BR.%26aulast%3DWeiss%26aufirst%3DS.%2BT.%26aulast%3DGlass%26aufirst%3DK.%26aulast%3DSharma%26aufirst%3DA.%26atitle%3DNetwork%2520Medicine%2520in%2520the%2520Age%2520of%2520Biomedical%2520Big%2520Data%26jtitle%3DFront.%2520Genet.%26date%3D2019%26volume%3D10%26spage%3D294%26doi%3D10.3389%2Ffgene.2019.00294" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Squartini, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldarelli, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cimini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabrielli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garlaschelli, D.</span></span> <span> </span><span class="NLM_article-title">Reconstruction Methods for Networks: The Case of Economic and Financial Systems</span>. <i>Phys. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>757</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">47</span>, <span class="refDoi"> DOI: 10.1016/j.physrep.2018.06.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1016%2Fj.physrep.2018.06.008" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=757&publication_year=2018&pages=1-47&author=T.+Squartiniauthor=G.+Caldarelliauthor=G.+Ciminiauthor=A.+Gabrielliauthor=D.+Garlaschelli&title=Reconstruction+Methods+for+Networks%3A+The+Case+of+Economic+and+Financial+Systems&doi=10.1016%2Fj.physrep.2018.06.008"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1016%2Fj.physrep.2018.06.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.physrep.2018.06.008%26sid%3Dliteratum%253Aachs%26aulast%3DSquartini%26aufirst%3DT.%26aulast%3DCaldarelli%26aufirst%3DG.%26aulast%3DCimini%26aufirst%3DG.%26aulast%3DGabrielli%26aufirst%3DA.%26aulast%3DGarlaschelli%26aufirst%3DD.%26atitle%3DReconstruction%2520Methods%2520for%2520Networks%253A%2520The%2520Case%2520of%2520Economic%2520and%2520Financial%2520Systems%26jtitle%3DPhys.%2520Rep.%26date%3D2018%26volume%3D757%26spage%3D1%26epage%3D47%26doi%3D10.1016%2Fj.physrep.2018.06.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Albert, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barabási, A. L.</span></span> <span> </span><span class="NLM_article-title">Statistical Mechanics of Complex Networks</span>. <i>Rev. Mod. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">97</span>, <span class="refDoi"> DOI: 10.1103/RevModPhys.74.47</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1103%2FRevModPhys.74.47" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2002&pages=47-97&author=R.+Albertauthor=A.+L.+Barab%C3%A1si&title=Statistical+Mechanics+of+Complex+Networks&doi=10.1103%2FRevModPhys.74.47"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1103%2FRevModPhys.74.47&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FRevModPhys.74.47%26sid%3Dliteratum%253Aachs%26aulast%3DAlbert%26aufirst%3DR.%26aulast%3DBarab%25C3%25A1si%26aufirst%3DA.%2BL.%26atitle%3DStatistical%2520Mechanics%2520of%2520Complex%2520Networks%26jtitle%3DRev.%2520Mod.%2520Phys.%26date%3D2002%26volume%3D74%26spage%3D47%26epage%3D97%26doi%3D10.1103%2FRevModPhys.74.47" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lü, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, T.</span></span> <span> </span><span class="NLM_article-title">Link Prediction in Complex Networks: A Survey</span>. <i>Phys. A</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>390</i></span>,  <span class="NLM_fpage">1150</span>– <span class="NLM_lpage">1170</span>, <span class="refDoi"> DOI: 10.1016/j.physa.2010.11.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1016%2Fj.physa.2010.11.027" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=390&publication_year=2011&pages=1150-1170&author=L.+L%C3%BCauthor=T.+Zhou&title=Link+Prediction+in+Complex+Networks%3A+A+Survey&doi=10.1016%2Fj.physa.2010.11.027"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1016%2Fj.physa.2010.11.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.physa.2010.11.027%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25BC%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DT.%26atitle%3DLink%2520Prediction%2520in%2520Complex%2520Networks%253A%2520A%2520Survey%26jtitle%3DPhys.%2520A%26date%3D2011%26volume%3D390%26spage%3D1150%26epage%3D1170%26doi%3D10.1016%2Fj.physa.2010.11.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ozturk, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dow, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlin, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bejar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, H.</span></span> <span> </span><span class="NLM_article-title">The Emerging Potential for Network Analysis to Inform Precision Cancer Medicine</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>430</i></span>,  <span class="NLM_fpage">2875</span>– <span class="NLM_lpage">2899</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2018.06.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1016%2Fj.jmb.2018.06.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=29908887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFOku77O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=430&publication_year=2018&pages=2875-2899&author=K.+Ozturkauthor=M.+Dowauthor=D.+E.+Carlinauthor=R.+Bejarauthor=H.+Carter&title=The+Emerging+Potential+for+Network+Analysis+to+Inform+Precision+Cancer+Medicine&doi=10.1016%2Fj.jmb.2018.06.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">The Emerging Potential for Network Analysis to Inform Precision Cancer Medicine</span></div><div class="casAuthors">Ozturk, Kivilcim; Dow, Michelle; Carlin, Daniel E.; Bejar, Rafael; Carter, Hannah</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">430</span>
        (<span class="NLM_cas:issue">18_Part_A</span>),
    <span class="NLM_cas:pages">2875-2899</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Precision cancer medicine promises to tailor clin. decisions to patients using genomic information.  Indeed, successes of drugs targeting genetic alterations in tumors, such as imatinib that targets BCR-ABL in chronic myelogenous leukemia, have demonstrated the power of this approach.  However, biol. systems are complex, and patients may differ not only by the specific genetic alterations in their tumor, but also by more subtle interactions among such alterations.  Systems biol. and more specifically, network anal., provides a framework for advancing precision medicine beyond clin. actionability of individual mutations.  Here we discuss applications of network anal. to study tumor biol., early methods for N-of-1 tumor genome anal., and the path for such tools to the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSIwj-eGOs9LVg90H21EOLACvtfcHk0lgMhf7nplY2rQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFOku77O&md5=44ce88c86a6879a23b8a37f42a966c36</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2018.06.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2018.06.016%26sid%3Dliteratum%253Aachs%26aulast%3DOzturk%26aufirst%3DK.%26aulast%3DDow%26aufirst%3DM.%26aulast%3DCarlin%26aufirst%3DD.%2BE.%26aulast%3DBejar%26aufirst%3DR.%26aulast%3DCarter%26aufirst%3DH.%26atitle%3DThe%2520Emerging%2520Potential%2520for%2520Network%2520Analysis%2520to%2520Inform%2520Precision%2520Cancer%2520Medicine%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2018%26volume%3D430%26spage%3D2875%26epage%3D2899%26doi%3D10.1016%2Fj.jmb.2018.06.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vilar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hripcsak, G.</span></span> <span> </span><span class="NLM_article-title">The Role of Drug Profiles as Similarity Metrics: Applications to Repurposing, Adverse Effects Detection and Drug – Drug Interactions</span>. <i>Briefings Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">670</span>– <span class="NLM_lpage">681</span>, <span class="refDoi"> DOI: 10.1093/bib/bbw048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fbib%2Fbbw048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=27273288" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjsVyqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=670-681&author=S.+Vilarauthor=G.+Hripcsak&title=The+Role+of+Drug+Profiles+as+Similarity+Metrics%3A+Applications+to+Repurposing%2C+Adverse+Effects+Detection+and+Drug+%E2%80%93+Drug+Interactions&doi=10.1093%2Fbib%2Fbbw048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">The role of drug profiles as similarity metrics: applications to repurposing, adverse effects detection and drug-drug interactions</span></div><div class="casAuthors">Vilar, Santiago; Hripcsak, George</div><div class="citationInfo"><span class="NLM_cas:title">Briefings in Bioinformatics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">670-681</span>CODEN:
                <span class="NLM_cas:coden">BBIMFX</span>;
        ISSN:<span class="NLM_cas:issn">1477-4054</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Explosion of the availability of big data sources along with the development in computational methods provides a useful framework to study drugs' actions, such as interactions with pharmacol. targets and off-targets.  Databases related to protein interactions, adverse effects and genomic profiles are available to be used for the construction of computational models.  In this article, we focus on the description of biol. profiles for drugs that can be used as a system to compare similarity and create methods to predict and analyze drugs' actions.  We highlight profiles constructed with different biol. data, such as target-protein interactions, gene expression measurements, adverse effects and disease profiles.  We focus on the discovery of new targets or pathways for drugs already in the pharmaceutical market, also called drug repurposing, in the interaction with off-targets responsible for adverse reactions and in drug-drug interaction anal.  The current and future applications, strengths and challenges facing all these methods are also discussed.  Biol. profiles or signatures are an important source of data generation to deeply analyze biol. actions with important implications in drug-related studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoY7SlptWRX_7Vg90H21EOLACvtfcHk0lgMhf7nplY2rQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjsVyqsLs%253D&md5=53a1cda1967ba1da99b30494215b5db5</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1093%2Fbib%2Fbbw048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbib%252Fbbw048%26sid%3Dliteratum%253Aachs%26aulast%3DVilar%26aufirst%3DS.%26aulast%3DHripcsak%26aufirst%3DG.%26atitle%3DThe%2520Role%2520of%2520Drug%2520Profiles%2520as%2520Similarity%2520Metrics%253A%2520Applications%2520to%2520Repurposing%252C%2520Adverse%2520Effects%2520Detection%2520and%2520Drug%2520%25E2%2580%2593%2520Drug%2520Interactions%26jtitle%3DBriefings%2520Bioinf.%26date%3D2017%26volume%3D18%26spage%3D670%26epage%3D681%26doi%3D10.1093%2Fbib%2Fbbw048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span> <span> </span><span class="NLM_article-title">Prediction of Drug-Target Interactions and Drug Repositioning via Network-Based Inference</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e1002503</span> <span class="refDoi"> DOI: 10.1371/journal.pcbi.1002503</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1371%2Fjournal.pcbi.1002503" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=22589709" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC38Xns1GntbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&author=F.+Chengauthor=C.+Liuauthor=J.+Jiangauthor=W.+Luauthor=W.+Liauthor=G.+Liuauthor=W.+Zhouauthor=J.+Huangauthor=Y.+Tang&title=Prediction+of+Drug-Target+Interactions+and+Drug+Repositioning+via+Network-Based+Inference&doi=10.1371%2Fjournal.pcbi.1002503"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of drug-target interactions and drug repositioning via network-based inference</span></div><div class="casAuthors">Cheng, Feixiong; Liu, Chuang; Jiang, Jing; Lu, Weiqiang; Li, Weihua; Liu, Guixia; Zhou, Weixing; Huang, Jin; Tang, Yun</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e1002503</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Drug-target interaction (DTI) is the basis of drug discovery and design.  It is time consuming and costly to det. DTI exptl.  Hence, it is necessary to develop computational methods for the prediction of potential DTI.  Based on complex network theory, three supervised inference methods were developed here to predict DTI and used for drug repositioning, namely drug-based similarity inference (DBSI), target-based similarity inference (TBSI) and network-based inference (NBI).  Among them, NBI performed best on four benchmark data sets.  Then a drug-target network was created with NBI based on 12,483 FDA-approved and exptl. drug-target binary links, and some new DTIs were further predicted.  In vitro assays confirmed that five old drugs, namely montelukast, diclofenac, simvastatin, ketoconazole, and itraconazole, showed polypharmacol. features on estrogen receptors or dipeptidyl peptidase-IV with half maximal inhibitory or effective concn. ranged from 0.2 to 10 μM.  Moreover, simvastatin and ketoconazole showed potent antiproliferative activities on human MDA-MB-231 breast cancer cell line in MTT assays.  The results indicated that these methods could be powerful tools in prediction of DTIs and drug repositioning.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvYLlyS_-BBLVg90H21EOLACvtfcHk0lglSKosRQvNig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xns1GntbY%253D&md5=d98a7828bebbdf981ebb14d45cc4102a</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1002503&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1002503%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DJiang%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DTang%26aufirst%3DY.%26atitle%3DPrediction%2520of%2520Drug-Target%2520Interactions%2520and%2520Drug%2520Repositioning%2520via%2520Network-Based%2520Inference%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2012%26volume%3D8%26doi%3D10.1371%2Fjournal.pcbi.1002503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, W. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, X. Y.</span></span> <span> </span><span class="NLM_article-title">Identification of Drug-Target Interaction from Interactome Network with “guilt-by-Association” Principle and Topology Features</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">1057</span>– <span class="NLM_lpage">1064</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btv695</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fbioinformatics%2Fbtv695" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=26614126" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlWiurvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2016&pages=1057-1064&author=Z.+C.+Liauthor=M.+H.+Huangauthor=W.+Q.+Zhongauthor=Z.+Q.+Liuauthor=Y.+Xieauthor=Z.+Daiauthor=X.+Y.+Zou&title=Identification+of+Drug-Target+Interaction+from+Interactome+Network+with+%E2%80%9Cguilt-by-Association%E2%80%9D+Principle+and+Topology+Features&doi=10.1093%2Fbioinformatics%2Fbtv695"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of drug-target interaction from interactome network with 'guilt-by-association' principle and topology features</span></div><div class="casAuthors">Li, Zhan-Chao; Huang, Meng-Hua; Zhong, Wen-Qian; Liu, Zhi-Qing; Xie, Yun; Dai, Zong; Zou, Xiao-Yong</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1057-1064</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: Identifying drug-target protein interaction is a crucial step in the process of drug research and development.  Wet-lab expt. are laborious, time-consuming and expensive.  Hence, there is a strong demand for the development of a novel theor. method to identify potential interaction between drug and target protein.  Results: We use all known proteins and drugs to construct a nodes- and edges-weighted biol. relevant interactome network.  On the basis of the 'guilt-by-assocn.' principle, novel network topol. features are proposed to characterize interaction pairs and random forest algorithm is employed to identify potential drug-protein interaction.  Accuracy of 92.53% derived from the 10-fold cross-validation is about 10% higher than that of the existing method.  We identify 2272 potential drug-target interactions, some of which are assocd. with diseases, such as Torg-Winchester syndrome and rhabdomyosarcoma.  The proposed method can not only accurately predict the interaction between drug mol. and target protein, but also help disease treatment and drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqy9I_thc-a57Vg90H21EOLACvtfcHk0lglSKosRQvNig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlWiurvK&md5=5f91b24a29299c9e26be83e3275d6f3a</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtv695&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtv695%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%2BC.%26aulast%3DHuang%26aufirst%3DM.%2BH.%26aulast%3DZhong%26aufirst%3DW.%2BQ.%26aulast%3DLiu%26aufirst%3DZ.%2BQ.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DDai%26aufirst%3DZ.%26aulast%3DZou%26aufirst%3DX.%2BY.%26atitle%3DIdentification%2520of%2520Drug-Target%2520Interaction%2520from%2520Interactome%2520Network%2520with%2520%25E2%2580%259Cguilt-by-Association%25E2%2580%259D%2520Principle%2520and%2520Topology%2520Features%26jtitle%3DBioinformatics%26date%3D2016%26volume%3D32%26spage%3D1057%26epage%3D1064%26doi%3D10.1093%2Fbioinformatics%2Fbtv695" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nam, H.</span></span> <span> </span><span class="NLM_article-title">Identification of Drug-Target Interaction by a Random Walk with Restart Method on an Interactome Network</span>. <i>BMC Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">208</span>, <span class="refDoi"> DOI: 10.1186/s12859-018-2199-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1186%2Fs12859-018-2199-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=29897326" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtlGlt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2018&pages=208&author=I.+Leeauthor=H.+Nam&title=Identification+of+Drug-Target+Interaction+by+a+Random+Walk+with+Restart+Method+on+an+Interactome+Network&doi=10.1186%2Fs12859-018-2199-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of drug-target interaction by a random walk with restart method on an interactome network</span></div><div class="casAuthors">Lee, Ingoo; Nam, Hojung</div><div class="citationInfo"><span class="NLM_cas:title">BMC Bioinformatics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">Suppl.8</span>),
    <span class="NLM_cas:pages">208/1-208/10</span>CODEN:
                <span class="NLM_cas:coden">BBMIC4</span>;
        ISSN:<span class="NLM_cas:issn">1471-2105</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Identification of drug-target interactions acts as a key role in drug discovery.  However, identifying drug-target interactions via in-vitro, in-vivo expts. are very laborious, time-consuming.  Thus, predicting drug-target interactions by using computational approaches is a good alternative.  In recent studies, many feature-based and similarity-based machine learning approaches have shown promising results in drug-target interaction predictions.  A previous study showed that accounting connectivity information of drug-drug and protein-protein interactions increase performances of prediction by the concept of 'guilt-by-assocn.'.  However, the approach that only considers directly connected nodes often misses the information that could be derived from distance nodes.  Therefore, in this study, we yield global network topol. information by using a random walk with restart algorithm and apply the global topol. information to the prediction model.  Results: As a result, our prediction model demonstrates increased prediction performance compare to the 'guilt-by-assocn.' approach (AUC 0.89 and 0.67 in the training and independent test, resp.).  In addn., we show how weighted features by a random walk with restart yields better performances than original features.  Also, we confirmed that drugs and proteins that have high-degree of connectivity on the interactome network yield better performance in our model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgyLrIPIGCCLVg90H21EOLACvtfcHk0li7gpKaSwXoLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtlGlt7s%253D&md5=278cbc057e2b921e3068d01f912e75d1</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1186%2Fs12859-018-2199-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12859-018-2199-x%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DI.%26aulast%3DNam%26aufirst%3DH.%26atitle%3DIdentification%2520of%2520Drug-Target%2520Interaction%2520by%2520a%2520Random%2520Walk%2520with%2520Restart%2520Method%2520on%2520an%2520Interactome%2520Network%26jtitle%3DBMC%2520Bioinf.%26date%3D2018%26volume%3D19%26spage%3D208%26doi%3D10.1186%2Fs12859-018-2199-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alaimo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giugno, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pulvirenti, A.</span></span> <span> </span><span class="NLM_article-title">Recommendation Techniques for Drug–Target Interaction Prediction and Drug Repositioning</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>1415</i></span>,  <span class="NLM_fpage">441</span>– <span class="NLM_lpage">462</span>, <span class="refDoi"> DOI: 10.1007/978-1-4939-3572-7_23</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1007%2F978-1-4939-3572-7_23" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=27115647" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpsFWktrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1415&publication_year=2016&pages=441-462&author=S.+Alaimoauthor=R.+Giugnoauthor=A.+Pulvirenti&title=Recommendation+Techniques+for+Drug%E2%80%93Target+Interaction+Prediction+and+Drug+Repositioning&doi=10.1007%2F978-1-4939-3572-7_23"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Recommendation Techniques for Drug-Target Interaction Prediction and Drug Repositioning</span></div><div class="casAuthors">Alaimo, Salvatore; Giugno, Rosalba; Pulvirenti, Alfredo</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">1415</span>
        (<span class="NLM_cas:issue">Data Mining Techniques for the Life Sciences</span>),
    <span class="NLM_cas:pages">441-462</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The usage of computational methods in drug discovery is a common practice.  More recently, by exploiting the wealth of biol. knowledge bases, a novel approach called drug repositioning has raised.  Several computational methods are available, and these try to make a high-level integration of all the knowledge in order to discover unknown mechanisms.  In this chapter, we review drug-target interaction prediction methods based on a recommendation system.  We also give some extensions which go beyond the bipartite network case.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXY4Pjf5JWirVg90H21EOLACvtfcHk0li7gpKaSwXoLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpsFWktrc%253D&md5=e55bd214ca4ed2406679847c396c852c</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-3572-7_23&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-3572-7_23%26sid%3Dliteratum%253Aachs%26aulast%3DAlaimo%26aufirst%3DS.%26aulast%3DGiugno%26aufirst%3DR.%26aulast%3DPulvirenti%26aufirst%3DA.%26atitle%3DRecommendation%2520Techniques%2520for%2520Drug%25E2%2580%2593Target%2520Interaction%2520Prediction%2520and%2520Drug%2520Repositioning%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2016%26volume%3D1415%26spage%3D441%26epage%3D462%26doi%3D10.1007%2F978-1-4939-3572-7_23" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cowen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ideker, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raphael, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharan, R.</span></span> <span> </span><span class="NLM_article-title">Network Propagation: A Universal Amplifier of Genetic Associations</span>. <i>Nat. Rev. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">551</span>– <span class="NLM_lpage">562</span>, <span class="refDoi"> DOI: 10.1038/nrg.2017.38</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1038%2Fnrg.2017.38" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=28607512" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC2sXps1Kgur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=551-562&author=L.+Cowenauthor=T.+Idekerauthor=B.+J.+Raphaelauthor=R.+Sharan&title=Network+Propagation%3A+A+Universal+Amplifier+of+Genetic+Associations&doi=10.1038%2Fnrg.2017.38"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Network propagation: a universal amplifier of genetic associations</span></div><div class="casAuthors">Cowen, Lenore; Ideker, Trey; Raphael, Benjamin J.; Sharan, Roded</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">551-562</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Biol. networks are powerful resources for the discovery of genes and genetic modules that drive disease.  Fundamental to network anal. is the concept that genes underlying the same phenotype tend to interact; this principle can be used to combine and to amplify signals from individual genes.  Recently, numerous bioinformatic techniques have been proposed for genetic anal. using networks, based on random walks, information diffusion and elec. resistance.  These approaches have been applied successfully to identify disease genes, genetic modules and drug targets.  In fact, all these approaches are variations of a unifying math. machinery - network propagation - suggesting that it is a powerful data transformation method of broad utility in genetic research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohVG3Mzv2G17Vg90H21EOLACvtfcHk0liSXdIJGOHPGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXps1Kgur4%253D&md5=554a1303016619aaae6e7a76a4722c26</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1038%2Fnrg.2017.38&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg.2017.38%26sid%3Dliteratum%253Aachs%26aulast%3DCowen%26aufirst%3DL.%26aulast%3DIdeker%26aufirst%3DT.%26aulast%3DRaphael%26aufirst%3DB.%2BJ.%26aulast%3DSharan%26aufirst%3DR.%26atitle%3DNetwork%2520Propagation%253A%2520A%2520Universal%2520Amplifier%2520of%2520Genetic%2520Associations%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2017%26volume%3D18%26spage%3D551%26epage%3D562%26doi%3D10.1038%2Fnrg.2017.38" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alaimo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pulvirenti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giugno, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferro, A.</span></span> <span> </span><span class="NLM_article-title">Drug-Target Interaction Prediction through Domain-Tuned Network-Based Inference</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">2004</span>– <span class="NLM_lpage">2008</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btt307</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fbioinformatics%2Fbtt307" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=23720490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFyhtLvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2013&pages=2004-2008&author=S.+Alaimoauthor=A.+Pulvirentiauthor=R.+Giugnoauthor=A.+Ferro&title=Drug-Target+Interaction+Prediction+through+Domain-Tuned+Network-Based+Inference&doi=10.1093%2Fbioinformatics%2Fbtt307"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-target interaction prediction through domain-tuned network-based inference</span></div><div class="casAuthors">Alaimo, Salvatore; Pulvirenti, Alfredo; Giugno, Rosalba; Ferro, Alfredo</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2004-2008</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: The identification of drug-target interaction (DTI) represents a costly and time-consuming step in drug discovery and design.  Computational methods capable of predicting reliable DTI play an important role in the field.  Recently, recommendation methods relying on network-based inference (NBI) have been proposed.  However, such approaches implement naive topol.-based inference and do not take into account important features within the drug-target domain.  Results: In this article, we present a new NBI method, called domain tuned-hybrid (DT-Hybrid), which extends a well-established recommendation technique by domain-based knowledge including drug and target similarity.  DT-Hybrid has been extensively tested using the last version of an exptl. validated DTI database obtained from DrugBank.  Comparison with other recently proposed NBI methods clearly shows that DT-Hybrid is capable of predicting more reliable DTIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohyIWe7-Sz5bVg90H21EOLACvtfcHk0liSXdIJGOHPGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFyhtLvL&md5=cba3f4535f2edc49440b50f219f33df4</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtt307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtt307%26sid%3Dliteratum%253Aachs%26aulast%3DAlaimo%26aufirst%3DS.%26aulast%3DPulvirenti%26aufirst%3DA.%26aulast%3DGiugno%26aufirst%3DR.%26aulast%3DFerro%26aufirst%3DA.%26atitle%3DDrug-Target%2520Interaction%2520Prediction%2520through%2520Domain-Tuned%2520Network-Based%2520Inference%26jtitle%3DBioinformatics%26date%3D2013%26volume%3D29%26spage%3D2004%26epage%3D2008%26doi%3D10.1093%2Fbioinformatics%2Fbtt307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Page, L.</span></span> <span> </span><span class="NLM_article-title">The Anatomy of a Large-Scale Hypertextual Web Search Engine</span>. <i>Computer Networks and ISDN Systems</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">107</span>– <span class="NLM_lpage">117</span>, <span class="refDoi"> DOI: 10.1016/S0169-7552(98)00110-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1016%2FS0169-7552%2898%2900110-X" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1998&pages=107-117&author=S.+Brinauthor=L.+Page&title=The+Anatomy+of+a+Large-Scale+Hypertextual+Web+Search+Engine&doi=10.1016%2FS0169-7552%2898%2900110-X"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1016%2FS0169-7552%2898%2900110-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-7552%252898%252900110-X%26sid%3Dliteratum%253Aachs%26aulast%3DBrin%26aufirst%3DS.%26aulast%3DPage%26aufirst%3DL.%26atitle%3DThe%2520Anatomy%2520of%2520a%2520Large-Scale%2520Hypertextual%2520Web%2520Search%2520Engine%26jtitle%3DComputer%2520Networks%2520and%2520ISDN%2520Systems%26date%3D1998%26volume%3D30%26spage%3D107%26epage%3D117%26doi%3D10.1016%2FS0169-7552%2898%2900110-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, G. Y.</span></span> <span> </span><span class="NLM_article-title">Drug-Target Interaction Prediction by Random Walk on the Heterogeneous Network</span>. <i>Mol. BioSyst.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1970</span>– <span class="NLM_lpage">1978</span>, <span class="refDoi"> DOI: 10.1039/c2mb00002d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1039%2Fc2mb00002d" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=22538619" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC38Xot1aqt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=1970-1978&author=X.+Chenauthor=M.+X.+Liuauthor=G.+Y.+Yan&title=Drug-Target+Interaction+Prediction+by+Random+Walk+on+the+Heterogeneous+Network&doi=10.1039%2Fc2mb00002d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-target interaction prediction by random walk on the heterogeneous network</span></div><div class="casAuthors">Chen, Xing; Liu, Ming-Xi; Yan, Gui-Ying</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1970-1978</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Predicting potential drug-target interactions from heterogeneous biol. data is crit. not only for better understanding of the various interactions and biol. processes, but also for the development of novel drugs and the improvement of human medicines.  In this paper, the method of Network-based Random Walk with Restart on the Heterogeneous network (NRWRH) is developed to predict potential drug-target interactions on a large scale under the hypothesis that similar drugs often target similar target proteins and the framework of Random Walk.  Compared with traditional supervised or semi-supervised methods, NRWRH makes full use of the tool of the network for data integration to predict drug-target assocns.  It integrates three different networks (protein-protein similarity network, drug-drug similarity network, and known drug-target interaction networks) into a heterogeneous network by known drug-target interactions and implements the random walk on this heterogeneous network.  When applied to four classes of important drug-target interactions including enzymes, ion channels, GPCRs and nuclear receptors, NRWRH significantly improves previous methods in terms of cross-validation and potential drug-target interaction prediction.  Excellent performance enables us to suggest a no. of new potential drug-target interactions for drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreLCn4HSwEX7Vg90H21EOLACvtfcHk0li2xUfA_GTYOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xot1aqt7k%253D&md5=c880d83fbb20fb6b81606849c8e37533</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1039%2Fc2mb00002d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2mb00002d%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DM.%2BX.%26aulast%3DYan%26aufirst%3DG.%2BY.%26atitle%3DDrug-Target%2520Interaction%2520Prediction%2520by%2520Random%2520Walk%2520on%2520the%2520Heterogeneous%2520Network%26jtitle%3DMol.%2520BioSyst.%26date%3D2012%26volume%3D8%26spage%3D1970%26epage%3D1978%26doi%3D10.1039%2Fc2mb00002d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wild, D. J.</span></span> <span> </span><span class="NLM_article-title">Optimizing Drug–Target Interaction Prediction Based on Random Walk on Heterogeneous Networks</span>. <i>J. Cheminf.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">40</span>, <span class="refDoi"> DOI: 10.1186/s13321-015-0089-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1186%2Fs13321-015-0089-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVertL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=40&author=A.+Sealauthor=Y.-Y.+Ahnauthor=D.+J.+Wild&title=Optimizing+Drug%E2%80%93Target+Interaction+Prediction+Based+on+Random+Walk+on+Heterogeneous+Networks&doi=10.1186%2Fs13321-015-0089-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Optimizing drug-target interaction prediction based on random walk on heterogeneous networks</span></div><div class="casAuthors">Seal, Abhik; Ahn, Yong-Yeol; Wild, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cheminformatics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">40/1-40/12</span>CODEN:
                <span class="NLM_cas:coden">JCOHB3</span>;
        ISSN:<span class="NLM_cas:issn">1758-2946</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">Background: Predicting novel drug-target assocns. is important not only for developing new drugs, but also for furthering biol. knowledge by understanding how drugs work and their modes of action.  As more data about drugs, targets, and their interactions becomes available, computational approaches have become an indispensible part of drug target assocn. discovery.  In this paper we apply random walk with restart (RWR) method to a heterogeneous network of drugs and targets compiled from DrugBank database and investigate the performance of the method under parameter variation and choice of chem. fingerprint methods.  Results: We show that choice of chem. fingerprint does not affect the performance of the method when the parameters are tuned to optimal values.  We use a subset of the ChEMBL15 dataset that contains 2,763 assocns. between 544 drugs and 467 target proteins to evaluate our method, and we extd. datasets of bioactivity ≤1 and ≤10 μM activity cutoff.  For 1 μM bioactivity cutoff, we find that our method can correctly predict nearly 47, 55, 60% of the given drug-target interactions in the test dataset having more than 0, 1, 2 drug target relations for ChEMBL 1 μM dataset in top 50 rank positions.  For 10 μM bioactivity cutoff, we find that our method can correctly predict nearly 32.4, 34.8, 35.3% of the given drug-target interactions in the test dataset having more than 0, 1, 2 drug target relations for ChEMBL 1 μM dataset in top 50 rank positions.  We further examine the assocns. between 110 popular top selling drugs in 2012 and 3,519 targets and find the top ten targets for each drug.  Conclusions: We demonstrate the effectiveness and promise of the approach- RWR on heterogeneous networks using chem. features- for identifying novel drug target interactions and investigate the performance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodpTBwrBYHCrVg90H21EOLACvtfcHk0li2xUfA_GTYOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVertL%252FN&md5=41b613b3bd38d78894d4c362b9293873</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1186%2Fs13321-015-0089-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13321-015-0089-z%26sid%3Dliteratum%253Aachs%26aulast%3DSeal%26aufirst%3DA.%26aulast%3DAhn%26aufirst%3DY.-Y.%26aulast%3DWild%26aufirst%3DD.%2BJ.%26atitle%3DOptimizing%2520Drug%25E2%2580%2593Target%2520Interaction%2520Prediction%2520Based%2520on%2520Random%2520Walk%2520on%2520Heterogeneous%2520Networks%26jtitle%3DJ.%2520Cheminf.%26date%3D2015%26volume%3D7%26spage%3D40%26doi%3D10.1186%2Fs13321-015-0089-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vanunu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magger, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruppin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shlomi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharan, R.</span></span> <span> </span><span class="NLM_article-title">Associating Genes and Protein Complexes with Disease via Network Propagation</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e1000641</span> <span class="refDoi"> DOI: 10.1371/journal.pcbi.1000641</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1371%2Fjournal.pcbi.1000641" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=20090828" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&author=O.+Vanunuauthor=O.+Maggerauthor=E.+Ruppinauthor=T.+Shlomiauthor=R.+Sharan&title=Associating+Genes+and+Protein+Complexes+with+Disease+via+Network+Propagation&doi=10.1371%2Fjournal.pcbi.1000641"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1000641&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1000641%26sid%3Dliteratum%253Aachs%26aulast%3DVanunu%26aufirst%3DO.%26aulast%3DMagger%26aufirst%3DO.%26aulast%3DRuppin%26aufirst%3DE.%26aulast%3DShlomi%26aufirst%3DT.%26aulast%3DSharan%26aufirst%3DR.%26atitle%3DAssociating%2520Genes%2520and%2520Protein%2520Complexes%2520with%2520Disease%2520via%2520Network%2520Propagation%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2010%26volume%3D6%26doi%3D10.1371%2Fjournal.pcbi.1000641" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shnaps, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silverbush, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharan, R.</span></span> <span> </span><span class="NLM_article-title">Inference of Personalized Drug Targets via Network Propagation</span>. <i>Pacific Symp. Biocomput.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_fpage">156</span>– <span class="NLM_lpage">167</span>, <span class="refDoi"> DOI: 10.1142/9789814749411_0015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1142%2F9789814749411_0015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=26776182" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A280%3ADC%252BC28nhsVKktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&pages=156-167&author=O.+Shnapsauthor=E.+Perryauthor=D.+Silverbushauthor=R.+Sharan&title=Inference+of+Personalized+Drug+Targets+via+Network+Propagation&doi=10.1142%2F9789814749411_0015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">INFERENCE OF PERSONALIZED DRUG TARGETS VIA NETWORK PROPAGATION</span></div><div class="casAuthors">Shnaps Ortal; Perry Eyal; Silverbush Dana; Sharan Roded</div><div class="citationInfo"><span class="NLM_cas:title">Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">156-67</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">We present a computational strategy to simulate drug treatment in a personalized setting.  The method is based on integrating patient mutation and differential expression data with a protein-protein interaction network.  We test the impact of in-silico deletions of different proteins on the flow of information in the network and use the results to infer potential drug targets.  We apply our method to AML data from TCGA and validate the predicted drug targets using known targets.  To benchmark our patient-specific approach, we compare the personalized setting predictions to those of the conventional setting.  Our predicted drug targets are highly enriched with known targets from DrugBank and COSMIC (p < 10(-5) outperforming the non-personalized predictions.  Finally, we focus on the largest AML patient subgroup (~30%) which is characterized by an FLT3 mutation, and utilize our prediction score to rank patient sensitivity to inhibition of each predicted target, reproducing previous findings of in-vitro experiments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRJlV_JdjB3j77NkxioyDXTfW6udTcc2ebOreX1LEINx7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28nhsVKktw%253D%253D&md5=b71b20ab2c69490a634306548e971b1e</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1142%2F9789814749411_0015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1142%252F9789814749411_0015%26sid%3Dliteratum%253Aachs%26aulast%3DShnaps%26aufirst%3DO.%26aulast%3DPerry%26aufirst%3DE.%26aulast%3DSilverbush%26aufirst%3DD.%26aulast%3DSharan%26aufirst%3DR.%26atitle%3DInference%2520of%2520Personalized%2520Drug%2520Targets%2520via%2520Network%2520Propagation%26jtitle%3DPacific%2520Symp.%2520Biocomput.%26date%3D2016%26spage%3D156%26epage%3D167%26doi%3D10.1142%2F9789814749411_0015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Himmelstein, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lizee, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hessler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brueggeman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khankhanian, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baranzini, S. E.</span></span> <span> </span><span class="NLM_article-title">Systematic Integration of Biomedical Knowledge Prioritizes Drugs for Repurposing</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e26726</span> <span class="refDoi"> DOI: 10.7554/eLife.26726</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.7554%2FeLife.26726" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=28936969" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlOitLzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2017&author=D.+S.+Himmelsteinauthor=A.+Lizeeauthor=C.+Hesslerauthor=L.+Brueggemanauthor=S.+L.+Chenauthor=D.+Hadleyauthor=A.+Greenauthor=P.+Khankhanianauthor=S.+E.+Baranzini&title=Systematic+Integration+of+Biomedical+Knowledge+Prioritizes+Drugs+for+Repurposing&doi=10.7554%2FeLife.26726"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic integration of biomedical knowledge prioritizes drugs for repurposing</span></div><div class="casAuthors">Himmelstein, Daniel Scott; Lizee, Antoine; Hessler, Christine; Brueggeman, Leo; Chen, Sabrina L.; Hadley, Dexter; Green, Ari; Khankhanian, Pouya; Baranzini, Sergio E.</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e26726/1-e26726/35</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">The ability to computationally predict whether a compd. treats a disease would improve the economy and success rate of drug approval.  This study describes Project Rephetio to systematically model drug efficacy based on 755 existing treatments.  First, we constructed Hetionet (neo4j.het.io), an integrative network encoding knowledge from millions of biomedical studies.  Hetionet v1.0 consists of 47,031 nodes of 11 types and 2,250,197 relationships of 24 types.  Data were integrated from 29 public resources to connect compds., diseases, genes, anatomies, pathways, biol. processes, mol. functions, cellular components, pharmacol. classes, side effects, and symptoms.  Next, we identified network patterns that distinguish treatments from non-treatments.  Then, we predicted the probability of treatment for 209,168 compd.-disease pairs (het.io/repurpose).  Our predictions validated on two external sets of treatment and provided pharmacol. insights on epilepsy, suggesting they will help prioritize drug repurposing candidates.  This study was entirely open and received realtime feedback from 40 community members.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXC1uVD5EnNrVg90H21EOLACvtfcHk0lh3Wi-8rFUktA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlOitLzJ&md5=885b93e215fe5c564f973f7b25dd6d32</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.7554%2FeLife.26726&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.26726%26sid%3Dliteratum%253Aachs%26aulast%3DHimmelstein%26aufirst%3DD.%2BS.%26aulast%3DLizee%26aufirst%3DA.%26aulast%3DHessler%26aufirst%3DC.%26aulast%3DBrueggeman%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DS.%2BL.%26aulast%3DHadley%26aufirst%3DD.%26aulast%3DGreen%26aufirst%3DA.%26aulast%3DKhankhanian%26aufirst%3DP.%26aulast%3DBaranzini%26aufirst%3DS.%2BE.%26atitle%3DSystematic%2520Integration%2520of%2520Biomedical%2520Knowledge%2520Prioritizes%2520Drugs%2520for%2520Repurposing%26jtitle%3DeLife%26date%3D2017%26volume%3D6%26doi%3D10.7554%2FeLife.26726" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaur, D.</span>; <span class="NLM_string-name">Shastri, A.</span>; <span class="NLM_string-name">Biswas, R.</span></span> <span> </span><span class="NLM_article-title">Metagraph: A New Model of Data Structure</span>.  <i>Proceedings</i>, 2008 International Conference on Computer Science and Information Technology, ICCSIT 2008, Singapore, 2008; <span class="NLM_publisher-name">IEEE</span>, <span class="NLM_year">2008</span>; pp  <span class="NLM_fpage">729</span>– <span class="NLM_lpage">733</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&pages=729-733&author=D.+Gaur&author=A.+Shastri&author=R.+Biswas&title=Proceedings"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DGaur%26aufirst%3DD.%26atitle%3DMetagraph%253A%2520A%2520New%2520Model%2520of%2520Data%2520Structure%26btitle%3DProceedings%26pub%3DIEEE%26date%3D2008%26spage%3D729%26epage%3D733" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chien, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuang, R.</span></span> <span> </span><span class="NLM_article-title">Network-Based Machine Learning and Graph Theory Algorithms for Precision Oncology</span>. <i>npj Precis. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">25</span>, <span class="refDoi"> DOI: 10.1038/s41698-017-0029-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1038%2Fs41698-017-0029-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=29872707" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A280%3ADC%252BC1MbktlGnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2017&pages=25&author=W.+Zhangauthor=J.+Chienauthor=J.+Yongauthor=R.+Kuang&title=Network-Based+Machine+Learning+and+Graph+Theory+Algorithms+for+Precision+Oncology&doi=10.1038%2Fs41698-017-0029-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Network-based machine learning and graph theory algorithms for precision oncology</span></div><div class="casAuthors">Zhang Wei; Kuang Rui; Chien Jeremy; Yong Jeongsik</div><div class="citationInfo"><span class="NLM_cas:title">NPJ precision oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">25</span>
        ISSN:<span class="NLM_cas:issn">2397-768X</span>.
    </div><div class="casAbstract">Network-based analytics plays an increasingly important role in precision oncology.  Growing evidence in recent studies suggests that cancer can be better understood through mutated or dysregulated pathways or networks rather than individual mutations and that the efficacy of repositioned drugs can be inferred from disease modules in molecular networks.  This article reviews network-based machine learning and graph theory algorithms for integrative analysis of personal genomic data and biomedical knowledge bases to identify tumor-specific molecular mechanisms, candidate targets and repositioned drugs for personalized treatment.  The review focuses on the algorithmic design and mathematical formulation of these methods to facilitate applications and implementations of network-based analysis in the practice of precision oncology.  We review the methods applied in three scenarios to integrate genomic data and network models in different analysis pipelines, and we examine three categories of network-based approaches for repositioning drugs in drug-disease-gene networks.  In addition, we perform a comprehensive subnetwork/pathway analysis of mutations in 31 cancer genome projects in the Cancer Genome Atlas and present a detailed case study on ovarian cancer.  Finally, we discuss interesting observations, potential pitfalls and future directions in network-based precision oncology.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT5YHmp9KDVO2gRQsDqrjW_fW6udTcc2eZyNgqfVaOC3rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MbktlGnuw%253D%253D&md5=5f85ff628227c45c42dea2f702fac0be</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1038%2Fs41698-017-0029-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41698-017-0029-7%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DChien%26aufirst%3DJ.%26aulast%3DYong%26aufirst%3DJ.%26aulast%3DKuang%26aufirst%3DR.%26atitle%3DNetwork-Based%2520Machine%2520Learning%2520and%2520Graph%2520Theory%2520Algorithms%2520for%2520Precision%2520Oncology%26jtitle%3Dnpj%2520Precis.%2520Oncol.%26date%3D2017%26volume%3D1%26spage%3D25%26doi%3D10.1038%2Fs41698-017-0029-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref88"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref88'); return false;" data-citation="" class="refNumLink">88</a></strong><div class="NLM_citation" id="cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span> <span> </span><span class="NLM_article-title">Prediction of Drug-Disease Associations for Drug Repositioning through Drug-MiRNA-Disease Heterogeneous Network</span>. <i>IEEE Access</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">45281</span>– <span class="NLM_lpage">45287</span>, <span class="refDoi"> DOI: 10.1109/ACCESS.2018.2860632</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1109%2FACCESS.2018.2860632" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2018&pages=45281-45287&author=H.+Chenauthor=Z.+Zhang&title=Prediction+of+Drug-Disease+Associations+for+Drug+Repositioning+through+Drug-MiRNA-Disease+Heterogeneous+Network&doi=10.1109%2FACCESS.2018.2860632"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1109%2FACCESS.2018.2860632&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1109%252FACCESS.2018.2860632%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DZ.%26atitle%3DPrediction%2520of%2520Drug-Disease%2520Associations%2520for%2520Drug%2520Repositioning%2520through%2520Drug-MiRNA-Disease%2520Heterogeneous%2520Network%26jtitle%3DIEEE%2520Access%26date%3D2018%26volume%3D6%26spage%3D45281%26epage%3D45287%26doi%3D10.1109%2FACCESS.2018.2860632" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref89"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref89'); return false;" data-citation="" class="refNumLink">89</a></strong><div class="NLM_citation" id="cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zong, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ngo, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harismendy, O.</span></span> <span> </span><span class="NLM_article-title">Deep Mining Heterogeneous Networks of Biomedical Linked Data to Predict Novel Drug-Target Associations</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">2337</span>– <span class="NLM_lpage">2344</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btx160</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fbioinformatics%2Fbtx160" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=28430977" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A280%3ADC%252BC1cvpvFKlsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2017&pages=2337-2344&author=N.+Zongauthor=H.+Kimauthor=V.+Ngoauthor=O.+Harismendy&title=Deep+Mining+Heterogeneous+Networks+of+Biomedical+Linked+Data+to+Predict+Novel+Drug-Target+Associations&doi=10.1093%2Fbioinformatics%2Fbtx160"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Deep mining heterogeneous networks of biomedical linked data to predict novel drug-target associations</span></div><div class="casAuthors">Zong Nansu; Kim Hyeoneui; Harismendy Olivier; Ngo Victoria; Harismendy Olivier</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics (Oxford, England)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2337-2344</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Motivation:  A heterogeneous network topology possessing abundant interactions between biomedical entities has yet to be utilized in similarity-based methods for predicting drug-target associations based on the array of varying features of drugs and their targets.  Deep learning reveals features of vertices of a large network that can be adapted in accommodating the similarity-based solutions to provide a flexible method of drug-target prediction.  Results:  We propose a similarity-based drug-target prediction method that enhances existing association discovery methods by using a topology-based similarity measure.  DeepWalk, a deep learning method, is adopted in this study to calculate the similarities within Linked Tripartite Network (LTN), a heterogeneous network generated from biomedical linked datasets.  This proposed method shows promising results for drug-target association prediction: 98.96% AUC ROC score with a 10-fold cross-validation and 99.25% AUC ROC score with a Monte Carlo cross-validation with LTN.  By utilizing DeepWalk, we demonstrate that: (i) this method outperforms other existing topology-based similarity computation methods, (ii) the performance is better for tripartite than with bipartite networks and (iii) the measure of similarity using network topology outperforms the ones derived from chemical structure (drugs) or genomic sequence (targets).  Our proposed methodology proves to be capable of providing a promising solution for drug-target prediction based on topological similarity with a heterogeneous network, and may be readily re-purposed and adapted in the existing of similarity-based methodologies.  Availability and Implementation:  The proposed method has been developed in JAVA and it is available, along with the data at the following URL: https://github.com/zongnansu1982/drug-target-prediction .  Contact:  <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="2d434c5742434a6d584e5e490348495803">[email protected]</a>  Supplementary information:  Supplementary data are available at Bioinformatics online.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSnJQmQ93XAz91QT_1O5WBYfW6udTcc2eYmWi75TqNtzrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cvpvFKlsw%253D%253D&md5=92bb2d32e82bd9ac613e9966912997a0</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtx160&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtx160%26sid%3Dliteratum%253Aachs%26aulast%3DZong%26aufirst%3DN.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DNgo%26aufirst%3DV.%26aulast%3DHarismendy%26aufirst%3DO.%26atitle%3DDeep%2520Mining%2520Heterogeneous%2520Networks%2520of%2520Biomedical%2520Linked%2520Data%2520to%2520Predict%2520Novel%2520Drug-Target%2520Associations%26jtitle%3DBioinformatics%26date%3D2017%26volume%3D33%26spage%3D2337%26epage%3D2344%26doi%3D10.1093%2Fbioinformatics%2Fbtx160" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref90"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref90'); return false;" data-citation="" class="refNumLink">90</a></strong><div class="NLM_citation" id="cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luo, H.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Li, M.</span>; <span class="NLM_string-name">Luo, J.</span>; <span class="NLM_string-name">Ni, P.</span>; <span class="NLM_string-name">Zhao, K.</span>; <span class="NLM_string-name">Wu, F.</span>; <span class="NLM_string-name">Pan, Y.</span></span> <span> </span><span class="NLM_article-title">Computational Drug Repositioning with Random Walk on a Heterogeneous Network</span>.  <i>IEEE/ACM Transactions on Computational Biology and Bioinformatics</i>; <span class="NLM_publisher-name">IEEE</span>, <span class="NLM_year">2018</span>; <span class="refDoi"> DOI: 10.1109/TCBB.2018.2832078</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1109%2FTCBB.2018.2832078" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=H.+Luo&author=J.+Wang&author=M.+Li&author=J.+Luo&author=P.+Ni&author=K.+Zhao&author=F.+Wu&author=Y.+Pan&title=IEEE%2FACM+Transactions+on+Computational+Biology+and+Bioinformatics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1109%2FTCBB.2018.2832078&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1109%252FTCBB.2018.2832078%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DH.%26atitle%3DComputational%2520Drug%2520Repositioning%2520with%2520Random%2520Walk%2520on%2520a%2520Heterogeneous%2520Network%26btitle%3DIEEE%252FACM%2520Transactions%2520on%2520Computational%2520Biology%2520and%2520Bioinformatics%26pub%3DIEEE%26date%3D2018%26doi%3D10.1109%2FTCBB.2018.2832078" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref91"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref91'); return false;" data-citation="" class="refNumLink">91</a></strong><div class="NLM_citation" id="cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kauffman, S. A.</span></span> <span> </span><span class="NLM_article-title">Metabolic Stability and Epigenesis in Randomly Constructed Genetic Nets</span>. <i>J. Theor. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1969</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">437</span>– <span class="NLM_lpage">467</span>, <span class="refDoi"> DOI: 10.1016/0022-5193(69)90015-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1016%2F0022-5193%2869%2990015-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=5803332" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A280%3ADyaF1M3msVOhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1969&pages=437-467&author=S.+A.+Kauffman&title=Metabolic+Stability+and+Epigenesis+in+Randomly+Constructed+Genetic+Nets&doi=10.1016%2F0022-5193%2869%2990015-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolic stability and epigenesis in randomly constructed genetic nets</span></div><div class="casAuthors">Kauffman S A</div><div class="citationInfo"><span class="NLM_cas:title">Journal of theoretical biology</span>
        (<span class="NLM_cas:date">1969</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">437-67</span>
        ISSN:<span class="NLM_cas:issn">0022-5193</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTHZToYWQRjGOvW160gIxP_fW6udTcc2eZWQvXAFv7T77ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaF1M3msVOhsw%253D%253D&md5=98f4e4aea59ba4a321ffe3f6be718c4a</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1016%2F0022-5193%2869%2990015-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-5193%252869%252990015-0%26sid%3Dliteratum%253Aachs%26aulast%3DKauffman%26aufirst%3DS.%2BA.%26atitle%3DMetabolic%2520Stability%2520and%2520Epigenesis%2520in%2520Randomly%2520Constructed%2520Genetic%2520Nets%26jtitle%3DJ.%2520Theor.%2520Biol.%26date%3D1969%26volume%3D22%26spage%3D437%26epage%3D467%26doi%3D10.1016%2F0022-5193%2869%2990015-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref92"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref92'); return false;" data-citation="" class="refNumLink">92</a></strong><div class="NLM_citation" id="cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kauffman, S.</span></span> <span> </span><span class="NLM_article-title">Homeostasis and Differentiation in Random Genetic Control Networks</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1969</span>,  <span class="NLM_volume"><i>224</i></span>,  <span class="NLM_fpage">177</span>– <span class="NLM_lpage">178</span>, <span class="refDoi"> DOI: 10.1038/224177a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1038%2F224177a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=5343519" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A280%3ADyaE3c%252FgsFyrtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=224&publication_year=1969&pages=177-178&author=S.+Kauffman&title=Homeostasis+and+Differentiation+in+Random+Genetic+Control+Networks&doi=10.1038%2F224177a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">Homeostasis and differentiation in random genetic control networks</span></div><div class="casAuthors">Kauffman S</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">1969</span>),
    <span class="NLM_cas:volume">224</span>
        (<span class="NLM_cas:issue">5215</span>),
    <span class="NLM_cas:pages">177-8</span>
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQed00svzJ7TBLDWw4HIq2sfW6udTcc2eZWQvXAFv7T77ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaE3c%252FgsFyrtQ%253D%253D&md5=5af5af5ffa244a7b73d92b0f1e8c34bb</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1038%2F224177a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F224177a0%26sid%3Dliteratum%253Aachs%26aulast%3DKauffman%26aufirst%3DS.%26atitle%3DHomeostasis%2520and%2520Differentiation%2520in%2520Random%2520Genetic%2520Control%2520Networks%26jtitle%3DNature%26date%3D1969%26volume%3D224%26spage%3D177%26epage%3D178%26doi%3D10.1038%2F224177a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref93"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref93'); return false;" data-citation="" class="refNumLink">93</a></strong><div class="NLM_citation" id="cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drossel, B.</span></span> <span> </span><span class="NLM_article-title">Random Boolean Networks</span>. In  <i>Reviews of Nonlinear Dynamics and Complexity</i>; <span class="NLM_publisher-name">Wiley-VCH</span>: <span class="NLM_publisher-loc">Weinheim, Germany</span>, <span class="NLM_year">2008</span>; pp  <span class="NLM_fpage">69</span>– <span class="NLM_lpage">110</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1002%2F9783527626359.ch3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&pages=69-110&author=B.+Drossel&title=Reviews+of+Nonlinear+Dynamics+and+Complexity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1002%2F9783527626359.ch3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9783527626359.ch3%26sid%3Dliteratum%253Aachs%26aulast%3DDrossel%26aufirst%3DB.%26atitle%3DRandom%2520Boolean%2520Networks%26btitle%3DReviews%2520of%2520Nonlinear%2520Dynamics%2520and%2520Complexity%26pub%3DWiley-VCH%26date%3D2008%26spage%3D69%26epage%3D110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref94"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref94'); return false;" data-citation="" class="refNumLink">94</a></strong><div class="NLM_citation" id="cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bornholdt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kauffman, S.</span></span> <span> </span><span class="NLM_article-title">Ensembles, Dynamics, and Cell Types: Revisiting the Statistical Mechanics Perspective on Cellular Regulation</span>. <i>J. Theor. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>467</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">22</span>, <span class="refDoi"> DOI: 10.1016/j.jtbi.2019.01.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1016%2Fj.jtbi.2019.01.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=30711453" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A280%3ADC%252BB3cjot1Glsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=467&publication_year=2019&pages=15-22&author=S.+Bornholdtauthor=S.+Kauffman&title=Ensembles%2C+Dynamics%2C+and+Cell+Types%3A+Revisiting+the+Statistical+Mechanics+Perspective+on+Cellular+Regulation&doi=10.1016%2Fj.jtbi.2019.01.036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Ensembles, dynamics, and cell types: Revisiting the statistical mechanics perspective on cellular regulation</span></div><div class="casAuthors">Bornholdt Stefan; Kauffman Stuart</div><div class="citationInfo"><span class="NLM_cas:title">Journal of theoretical biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">467</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15-22</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Genetic regulatory networks control ontogeny.  For fifty years Boolean networks have served as models of such systems, ranging from ensembles of random Boolean networks as models for generic properties of gene regulation to working dynamical models of a growing number of sub-networks of real cells.  At the same time, their statistical mechanics has been thoroughly studied.  Here we recapitulate their original motivation in the context of current theoretical and empirical research.  We discuss ensembles of random Boolean networks whose dynamical attractors model cell types.  A sub-ensemble is the critical ensemble.  There is now strong evidence that genetic regulatory networks are dynamically critical, and that evolution is exploring the critical sub-ensemble.  The generic properties of this sub-ensemble predict essential features of cell differentiation.  In particular, the number of attractors in such networks scales as the DNA content raised to the 0.63 power.  Data on the number of cell types as a function of the DNA content per cell shows a scaling relationship of 0.88.  Thus, the theory correctly predicts a power law relationship between the number of cell types and the DNA contents per cell, and a comparable slope.  We discuss these new scaling values and show prospects for new research lines for Boolean networks as a base model for systems biology.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcReJOPEu9JaVAWRJ3OJdH_qfW6udTcc2eZWQvXAFv7T77ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cjot1Glsw%253D%253D&md5=7bacdf0ff77bd9a84fae76ec7f5a75b1</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1016%2Fj.jtbi.2019.01.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jtbi.2019.01.036%26sid%3Dliteratum%253Aachs%26aulast%3DBornholdt%26aufirst%3DS.%26aulast%3DKauffman%26aufirst%3DS.%26atitle%3DEnsembles%252C%2520Dynamics%252C%2520and%2520Cell%2520Types%253A%2520Revisiting%2520the%2520Statistical%2520Mechanics%2520Perspective%2520on%2520Cellular%2520Regulation%26jtitle%3DJ.%2520Theor.%2520Biol.%26date%3D2019%26volume%3D467%26spage%3D15%26epage%3D22%26doi%3D10.1016%2Fj.jtbi.2019.01.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref95"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref95'); return false;" data-citation="" class="refNumLink">95</a></strong><div class="NLM_citation" id="cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eichler, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bar-Yam, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingber, D. E.</span></span> <span> </span><span class="NLM_article-title">Cell Fates as High-Dimensional Attractor States of a Complex Gene Regulatory Network</span>. <i>Phys. Rev. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">128701</span>, <span class="refDoi"> DOI: 10.1103/PhysRevLett.94.128701</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1103%2FPhysRevLett.94.128701" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=15903968" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BD2MXivFSrs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2005&pages=128701&author=S.+Huangauthor=G.+Eichlerauthor=Y.+Bar-Yamauthor=D.+E.+Ingber&title=Cell+Fates+as+High-Dimensional+Attractor+States+of+a+Complex+Gene+Regulatory+Network&doi=10.1103%2FPhysRevLett.94.128701"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Cell Fates as High-Dimensional Attractor States of a Complex Gene Regulatory Network</span></div><div class="casAuthors">Huang, Sui; Eichler, Gabriel; Bar-Yam, Yaneer; Ingber, Donald E.</div><div class="citationInfo"><span class="NLM_cas:title">Physical Review Letters</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">128701/1-128701/4</span>CODEN:
                <span class="NLM_cas:coden">PRLTAO</span>;
        ISSN:<span class="NLM_cas:issn">0031-9007</span>.
    
            (<span class="NLM_cas:orgname">American Physical Society</span>)
        </div><div class="casAbstract">Cells in multicellular organisms switch between distinct cell fates, such as proliferation or differentiation into specialized cell types.  Genome-wide gene regulatory networks govern this behavior.  Theor. studies of complex networks suggest that they can exhibit ordered (stable) dynamics, raising the possibility that cell fates may represent high-dimensional attractor states.  We used gene expression profiling to show that trajectories of neutrophil differentiation converge to a common state from different directions of a 2773-dimensional gene expression state space, providing the first exptl. evidence for a high-dimensional stable attractor that represents a distinct cellular phenotype.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3MRBJUfK8WrVg90H21EOLACvtfcHk0lgk8BQcR9B0Ag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXivFSrs7k%253D&md5=afa34e2bedf468935ac7042b5fa28774</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1103%2FPhysRevLett.94.128701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevLett.94.128701%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DS.%26aulast%3DEichler%26aufirst%3DG.%26aulast%3DBar-Yam%26aufirst%3DY.%26aulast%3DIngber%26aufirst%3DD.%2BE.%26atitle%3DCell%2520Fates%2520as%2520High-Dimensional%2520Attractor%2520States%2520of%2520a%2520Complex%2520Gene%2520Regulatory%2520Network%26jtitle%3DPhys.%2520Rev.%2520Lett.%26date%3D2005%26volume%3D94%26spage%3D128701%26doi%3D10.1103%2FPhysRevLett.94.128701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref96"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref96'); return false;" data-citation="" class="refNumLink">96</a></strong><div class="NLM_citation" id="cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bloomingdale, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mager, D. E.</span></span> <span> </span><span class="NLM_article-title">Boolean Network Modeling in Systems Pharmacology</span>. <i>J. Pharmacokinet. Pharmacodyn.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">159</span>– <span class="NLM_lpage">180</span>, <span class="refDoi"> DOI: 10.1007/s10928-017-9567-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1007%2Fs10928-017-9567-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=29307099" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1cXlvVyhtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2018&pages=159-180&author=P.+Bloomingdaleauthor=V.+A.+Nguyenauthor=J.+Niuauthor=D.+E.+Mager&title=Boolean+Network+Modeling+in+Systems+Pharmacology&doi=10.1007%2Fs10928-017-9567-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">Boolean network modeling in systems pharmacology</span></div><div class="casAuthors">Bloomingdale, Peter; Nguyen, Van Anh; Niu, Jin; Mager, Donald E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacokinetics and Pharmacodynamics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">159-180</span>CODEN:
                <span class="NLM_cas:coden">JPPOAH</span>;
        ISSN:<span class="NLM_cas:issn">1567-567X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Quant. systems pharmacol. (QSP) is an emerging discipline that aims to discover how drugs modulate the dynamics of biol. components in mol. and cellular networks and the impact of those perturbations on human pathophysiol.  The integration of systems-based exptl. and computational approaches is required to facilitate the advancement of this field.  QSP models typically consist of a series of ordinary differential equations (ODE).  However, this math. framework requires extensive knowledge of parameters pertaining to biol. processes, which is often unavailable.  An alternative framework that does not require knowledge of system-specific parameters, such as Boolean network modeling, could serve as an initial foundation prior to the development of an ODE-based model.  Boolean network models have been shown to efficiently describe, in a qual. manner, the complex behavior of signal transduction and gene/protein regulatory processes.  In addn. to providing a starting point prior to quant. modeling, Boolean network models can also be utilized to discover novel therapeutic targets and combinatorial treatment strategies.  Identifying drug targets using a network-based approach could supplement current drug discovery methodologies and help to fill the innovation gap across the pharmaceutical industry.  In this review, we discuss the process of developing Boolean network models and the various analyses that can be performed to identify novel drug targets and combinatorial approaches.  An example for each of these analyses is provided using a previously developed Boolean network of signaling pathways in multiple myeloma.  Selected examples of Boolean network models of human (patho-)physiol. systems are also reviewed in brief.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLhhqodM6mfLVg90H21EOLACvtfcHk0lhu8xcXr7vBEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXlvVyhtg%253D%253D&md5=9fce87d27de012fa22ce15efeaa13861</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1007%2Fs10928-017-9567-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10928-017-9567-4%26sid%3Dliteratum%253Aachs%26aulast%3DBloomingdale%26aufirst%3DP.%26aulast%3DNguyen%26aufirst%3DV.%2BA.%26aulast%3DNiu%26aufirst%3DJ.%26aulast%3DMager%26aufirst%3DD.%2BE.%26atitle%3DBoolean%2520Network%2520Modeling%2520in%2520Systems%2520Pharmacology%26jtitle%3DJ.%2520Pharmacokinet.%2520Pharmacodyn.%26date%3D2018%26volume%3D45%26spage%3D159%26epage%3D180%26doi%3D10.1007%2Fs10928-017-9567-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref97"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref97'); return false;" data-citation="" class="refNumLink">97</a></strong><div class="NLM_citation" id="cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Anda-Jáuregui, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espinal-Enríquez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandoval-Motta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernández-Lemus, E.</span></span> <span> </span><span class="NLM_article-title">A Boolean Network Approach to Estrogen Transcriptional Regulation</span>. <i>Complexity</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>2019</i></span>,  <span class="NLM_fpage">8740279</span>, <span class="refDoi"> DOI: 10.1155/2019/8740279</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1155%2F2019%2F8740279" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2019&publication_year=2019&pages=8740279&author=G.+De%0AAnda-J%C3%A1ureguiauthor=J.+Espinal-Enr%C3%ADquezauthor=S.+Sandoval-Mottaauthor=E.+Hern%C3%A1ndez-Lemus&title=A+Boolean+Network+Approach+to+Estrogen+Transcriptional+Regulation&doi=10.1155%2F2019%2F8740279"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1155%2F2019%2F8740279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2019%252F8740279%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BAnda-J%25C3%25A1uregui%26aufirst%3DG.%26aulast%3DEspinal-Enr%25C3%25ADquez%26aufirst%3DJ.%26aulast%3DSandoval-Motta%26aufirst%3DS.%26aulast%3DHern%25C3%25A1ndez-Lemus%26aufirst%3DE.%26atitle%3DA%2520Boolean%2520Network%2520Approach%2520to%2520Estrogen%2520Transcriptional%2520Regulation%26jtitle%3DComplexity%26date%3D2019%26volume%3D2019%26spage%3D8740279%26doi%3D10.1155%2F2019%2F8740279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref98"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref98'); return false;" data-citation="" class="refNumLink">98</a></strong><div class="NLM_citation" id="cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, K. H.</span></span> <span> </span><span class="NLM_article-title">Attractor Landscape Analysis Reveals Feedback Loops in the P53 Network That Control the Cellular Response to DNA Damage</span>. <i>Sci. Signaling</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">ra83</span>, <span class="refDoi"> DOI: 10.1126/scisignal.2003363</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1126%2Fscisignal.2003363" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=23169817" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2012&pages=ra83&author=M.+Choiauthor=J.+Shiauthor=S.+H.+Jungauthor=X.+Chenauthor=K.+H.+Cho&title=Attractor+Landscape+Analysis+Reveals+Feedback+Loops+in+the+P53+Network+That+Control+the+Cellular+Response+to+DNA+Damage&doi=10.1126%2Fscisignal.2003363"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1126%2Fscisignal.2003363&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscisignal.2003363%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DJung%26aufirst%3DS.%2BH.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DCho%26aufirst%3DK.%2BH.%26atitle%3DAttractor%2520Landscape%2520Analysis%2520Reveals%2520Feedback%2520Loops%2520in%2520the%2520P53%2520Network%2520That%2520Control%2520the%2520Cellular%2520Response%2520to%2520DNA%2520Damage%26jtitle%3DSci.%2520Signaling%26date%3D2012%26volume%3D5%26spage%3Dra83%26doi%3D10.1126%2Fscisignal.2003363" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref99"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref99'); return false;" data-citation="" class="refNumLink">99</a></strong><div class="NLM_citation" id="cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, K. H.</span></span> <span> </span><span class="NLM_article-title">Network Dynamics-Based Cancer Panel Stratification for Systemic Prediction of Anticancer Drug Response</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1940</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-02160-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1038%2Fs41467-017-02160-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=29208897" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A280%3ADC%252BC1M3otlWgsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1940&author=M.+Choiauthor=J.+Shiauthor=Y.+Zhuauthor=R.+Yangauthor=K.+H.+Cho&title=Network+Dynamics-Based+Cancer+Panel+Stratification+for+Systemic+Prediction+of+Anticancer+Drug+Response&doi=10.1038%2Fs41467-017-02160-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Network dynamics-based cancer panel stratification for systemic prediction of anticancer drug response</span></div><div class="casAuthors">Choi Minsoo; Cho Kwang-Hyun; Shi Jue; Zhu Yanting; Yang Ruizhen</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1940</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cancer is a complex disease involving multiple genomic alterations that disrupt the dynamic response of signaling networks.  The heterogeneous nature of cancer, which results in highly variable drug response, is a major obstacle to developing effective cancer therapy.  Previous studies of cancer therapeutic response mostly focus on static analysis of genome-wide alterations, thus they are unable to unravel the dynamic, network-specific origin of variation.  Here we present a network dynamics-based approach to integrate cancer genomics with dynamics of biological network for drug response prediction and design of drug combination.  We select the p53 network as an example and analyze its cancer-specific state transition dynamics under distinct anticancer drug treatments by attractor landscape analysis.  Our results not only enable stratification of cancer into distinct drug response groups, but also reveal network-specific drug targets that maximize p53 network-mediated cell death, providing a basis to design combinatorial therapeutic strategies for distinct cancer genomic subtypes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT6sxPcQa39gcScOqQTSkOffW6udTcc2eYLE6aR5kYjBrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M3otlWgsQ%253D%253D&md5=5a210fe74d025fc4d143243637cab5ae</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-02160-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-02160-5%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DR.%26aulast%3DCho%26aufirst%3DK.%2BH.%26atitle%3DNetwork%2520Dynamics-Based%2520Cancer%2520Panel%2520Stratification%2520for%2520Systemic%2520Prediction%2520of%2520Anticancer%2520Drug%2520Response%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D1940%26doi%3D10.1038%2Fs41467-017-02160-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref100"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref100'); return false;" data-citation="" class="refNumLink">100</a></strong><div class="NLM_citation" id="cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Capobianco, E.</span></span> <span> </span><span class="NLM_article-title">Next Generation Networks: Featuring the Potential Role of Emerging Applications in Translational Oncology</span>. <i>J. Clin. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">664</span>, <span class="refDoi"> DOI: 10.3390/jcm8050664</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.3390%2Fjcm8050664" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2019&pages=664&author=E.+Capobianco&title=Next+Generation+Networks%3A+Featuring+the+Potential+Role+of+Emerging+Applications+in+Translational+Oncology&doi=10.3390%2Fjcm8050664"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.3390%2Fjcm8050664&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fjcm8050664%26sid%3Dliteratum%253Aachs%26aulast%3DCapobianco%26aufirst%3DE.%26atitle%3DNext%2520Generation%2520Networks%253A%2520Featuring%2520the%2520Potential%2520Role%2520of%2520Emerging%2520Applications%2520in%2520Translational%2520Oncology%26jtitle%3DJ.%2520Clin.%2520Med.%26date%3D2019%26volume%3D8%26spage%3D664%26doi%3D10.3390%2Fjcm8050664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref101"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref101'); return false;" data-citation="" class="refNumLink">101</a></strong><div class="NLM_citation" id="cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Husic, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pande, V. S.</span></span> <span> </span><span class="NLM_article-title">Markov State Models: From an Art to a Science</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">2386</span>– <span class="NLM_lpage">2396</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b12191</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b12191" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotV2huw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=2386-2396&author=B.+E.+Husicauthor=V.+S.+Pande&title=Markov+State+Models%3A+From+an+Art+to+a+Science&doi=10.1021%2Fjacs.7b12191"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Markov State Models: From an Art to a Science</span></div><div class="casAuthors">Husic, Brooke E.; Pande, Vijay S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2386-2396</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Markov state models (MSMs) are a powerful framework for analyzing dynamical systems, such as mol. dynamics (MD) simulations, that have gained widespread use over the past several decades.  This perspective offers an overview of the MSM field to date, presented for a general audience as a timeline of key developments in the field.  We sequentially address early studies that motivated the method, canonical papers that established the use of MSMs for MD anal., and subsequent advances in software and anal. protocols.  The derivation of a variational principle for MSMs in 2013 signified a turning point from expertise-driving MSM building to a systematic, objective protocol.  The variational approach, combined with best practices for model selection and open-source software, enabled a wide range of MSM anal. for applications such as protein folding and allostery, ligand binding, and protein-protein assocn.  To conclude, the current frontiers of methods development are highlighted, as well as exciting applications in exptl. design and drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo43eE2dHXoHrVg90H21EOLACvtfcHk0ljVDcQOar9ytQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotV2huw%253D%253D&md5=f00943acedec985c4d21abd86ecfce4a</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b12191&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b12191%26sid%3Dliteratum%253Aachs%26aulast%3DHusic%26aufirst%3DB.%2BE.%26aulast%3DPande%26aufirst%3DV.%2BS.%26atitle%3DMarkov%2520State%2520Models%253A%2520From%2520an%2520Art%2520to%2520a%2520Science%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D2386%26epage%3D2396%26doi%3D10.1021%2Fjacs.7b12191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref102"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref102'); return false;" data-citation="" class="refNumLink">102</a></strong><div class="NLM_citation" id="cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bernetti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masetti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Recanatini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amaro, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span> <span> </span><span class="NLM_article-title">An Integrated Markov State Model and Path Metadynamics Approach to Characterize Drug Binding Processes</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">5689</span>– <span class="NLM_lpage">5702</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.9b00450</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.9b00450" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1egurjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=5689-5702&author=M.+Bernettiauthor=M.+Masettiauthor=M.+Recanatiniauthor=R.+E.+Amaroauthor=A.+Cavalli&title=An+Integrated+Markov+State+Model+and+Path+Metadynamics+Approach+to+Characterize+Drug+Binding+Processes&doi=10.1021%2Facs.jctc.9b00450"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">An Integrated Markov State Model and Path Metadynamics Approach To Characterize Drug Binding Processes</span></div><div class="casAuthors">Bernetti, Mattia; Masetti, Matteo; Recanatini, Maurizio; Amaro, Rommie E.; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5689-5702</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Unveiling the mechanistic features of drug-target binding is of central interest in biophysics and drug discovery.  Herein, the authors address this challenge by combining two major computational approaches, namely Mol. Dynamics (MD) simulations and Markov State Models (MSM), with a Path Collective Variables (PCVs) description coupled with metadynamics.  The authors apply the methodol. to reconstruct the binding process of the antagonist alprenolol to the β2-adrenergic receptor, a well-established pharmaceutical target.  The devised protocol allowed us to est. the binding free energy and identify the min. free energy path leading to the protein-ligand complex.  In summary, the authors show that MSM and PCVs can be efficiently integrated to shed light upon mechanistic and energetic details underlying complex recognition processes in biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbe1nT6X31cLVg90H21EOLACvtfcHk0ljVDcQOar9ytQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1egurjP&md5=f3f4bfadf4a14e2beb74764fa8dae955</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.9b00450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.9b00450%26sid%3Dliteratum%253Aachs%26aulast%3DBernetti%26aufirst%3DM.%26aulast%3DMasetti%26aufirst%3DM.%26aulast%3DRecanatini%26aufirst%3DM.%26aulast%3DAmaro%26aufirst%3DR.%2BE.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DAn%2520Integrated%2520Markov%2520State%2520Model%2520and%2520Path%2520Metadynamics%2520Approach%2520to%2520Characterize%2520Drug%2520Binding%2520Processes%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2019%26volume%3D15%26spage%3D5689%26epage%3D5702%26doi%3D10.1021%2Facs.jctc.9b00450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref103"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref103'); return false;" data-citation="" class="refNumLink">103</a></strong><div class="NLM_citation" id="cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chu, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tse, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, E. L.</span></span> <span> </span><span class="NLM_article-title">Markov State Models of Gene Regulatory Networks</span>. <i>BMC Syst. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">14</span>, <span class="refDoi"> DOI: 10.1186/s12918-017-0394-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1186%2Fs12918-017-0394-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=28166778" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjt1SkurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2017&pages=14&author=B.+K.+Chuauthor=M.+J.+Tseauthor=R.+R.+Satoauthor=E.+L.+Read&title=Markov+State+Models+of+Gene+Regulatory+Networks&doi=10.1186%2Fs12918-017-0394-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Markov State Models of gene regulatory networks</span></div><div class="casAuthors">Chu, Brian K.; Tse, Margaret J.; Sato, Royce R.; Read, Elizabeth L.</div><div class="citationInfo"><span class="NLM_cas:title">BMC Systems Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">14/1-14/17</span>CODEN:
                <span class="NLM_cas:coden">BSBMCC</span>;
        ISSN:<span class="NLM_cas:issn">1752-0509</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Gene regulatory networks with dynamics characterized by multiple stable states underlie cell fate-decisions.  Quant. models that can link mol.-level knowledge of gene regulation to a global understanding of network dynamics have the potential to guide cell-reprogramming strategies.  Networks are often modeled by the stochastic Chem. Master Equation, but methods for systematic identification of key properties of the global dynamics are currently lacking.  Results: The method identifies the no., phenotypes, and lifetimes of long-lived states for a set of common gene regulatory network models.  Application of transition path theory to the constructed Markov State Model decomps. global dynamics into a set of dominant transition paths and assocd. relative probabilities for stochastic state-switching.  Conclusions: In this proof-of-concept study, we found that the Markov State Model provides a general framework for analyzing and visualizing stochastic multistability and state-transitions in gene networks.  Our results suggest that this framework-adopted from the field of atomistic Mol. Dynamics-can be a useful tool for quant. Systems Biol. at the network scale.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1h_HrYHhRhbVg90H21EOLACvtfcHk0ljVDcQOar9ytQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjt1SkurY%253D&md5=58df2525842dd65cd630fa4704a4a16c</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1186%2Fs12918-017-0394-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12918-017-0394-4%26sid%3Dliteratum%253Aachs%26aulast%3DChu%26aufirst%3DB.%2BK.%26aulast%3DTse%26aufirst%3DM.%2BJ.%26aulast%3DSato%26aufirst%3DR.%2BR.%26aulast%3DRead%26aufirst%3DE.%2BL.%26atitle%3DMarkov%2520State%2520Models%2520of%2520Gene%2520Regulatory%2520Networks%26jtitle%3DBMC%2520Syst.%2520Biol.%26date%3D2017%26volume%3D11%26spage%3D14%26doi%3D10.1186%2Fs12918-017-0394-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref104"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref104'); return false;" data-citation="" class="refNumLink">104</a></strong><div class="NLM_citation" id="cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hansch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujita, T.</span></span> <span> </span><span class="NLM_article-title">ρ-σ-π Analysis. A Method for the Correlation of Biological Activity and Chemical Structure</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1964</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">1616</span>– <span class="NLM_lpage">1626</span>, <span class="refDoi"> DOI: 10.1021/ja01062a035</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja01062a035" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADyaF2cXptF2gsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=1964&pages=1616-1626&author=C.+Hanschauthor=T.+Fujita&title=%CF%81-%CF%83-%CF%80+Analysis.+A+Method+for+the+Correlation+of+Biological+Activity+and+Chemical+Structure&doi=10.1021%2Fja01062a035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">ρ-σ-πAnalysis; method for the correlation of biological activity and chemical structure</span></div><div class="casAuthors">Hansch, Corwin; Fujita, Toshio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1964</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1616-26</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Using the substituent constant, σ, and a substituent constant, π, defined as σ = log Px - log PH(PH is the partition coeff. of a parent compd. and Px that of a deriv.), regression analyses have been made of the effect of substituents on the biol. activity of benzoic acids on mosquito larvae, phenols on gram-pos. and gram-neg. bacteria, phenyl ethyl phosphate insecticides on houseflies, thyroxine derivs. on rodents, diethylaminoethyl benzoates on guinea pigs, and carcinogenic compds. on mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoMZAWYjkBhLVg90H21EOLACvtfcHk0ljqrYPn5noZMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF2cXptF2gsQ%253D%253D&md5=190de0722f5c3a1d98a2419df7d28c58</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1021%2Fja01062a035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja01062a035%26sid%3Dliteratum%253Aachs%26aulast%3DHansch%26aufirst%3DC.%26aulast%3DFujita%26aufirst%3DT.%26atitle%3D%25CF%2581-%25CF%2583-%25CF%2580%2520Analysis.%2520A%2520Method%2520for%2520the%2520Correlation%2520of%2520Biological%2520Activity%2520and%2520Chemical%2520Structure%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1964%26volume%3D86%26spage%3D1616%26epage%3D1626%26doi%3D10.1021%2Fja01062a035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref105"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref105'); return false;" data-citation="" class="refNumLink">105</a></strong><div class="NLM_citation" id="cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whitmore, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wray, J.</span></span> <span> </span><span class="NLM_article-title">Networks for Better Drug Discovery: Making Optimal Use of Existing Chemical Space</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2497</span>– <span class="NLM_lpage">2499</span>, <span class="refDoi"> DOI: 10.4155/fmc-2018-0265</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.4155%2Ffmc-2018-0265" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVKjtLnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=2497-2499&author=A.+V.+Whitmoreauthor=J.+Wray&title=Networks+for+Better+Drug+Discovery%3A+Making+Optimal+Use+of+Existing+Chemical+Space&doi=10.4155%2Ffmc-2018-0265"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">Networks for better drug discovery: making optimal use of existing chemical space</span></div><div class="casAuthors">Whitmore, Alan Victor; Wray, Jonny</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">2497-2499</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVJe5nC7HA0rVg90H21EOLACvtfcHk0ljqrYPn5noZMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVKjtLnM&md5=991600275a08702cfd88cfdc17e38c01</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2018-0265&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2018-0265%26sid%3Dliteratum%253Aachs%26aulast%3DWhitmore%26aufirst%3DA.%2BV.%26aulast%3DWray%26aufirst%3DJ.%26atitle%3DNetworks%2520for%2520Better%2520Drug%2520Discovery%253A%2520Making%2520Optimal%2520Use%2520of%2520Existing%2520Chemical%2520Space%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2018%26volume%3D10%26spage%3D2497%26epage%3D2499%26doi%3D10.4155%2Ffmc-2018-0265" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref106"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref106'); return false;" data-citation="" class="refNumLink">106</a></strong><div class="NLM_citation" id="cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oprea, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">May, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leitão, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tropsha, A.</span></span> <span> </span><span class="NLM_article-title">Computational Systems Chemical Biology</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>672</i></span>,  <span class="NLM_fpage">459</span>– <span class="NLM_lpage">488</span>, <span class="refDoi"> DOI: 10.1007/978-1-60761-839-3_18</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1007%2F978-1-60761-839-3_18" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=20838980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlejsr%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=672&publication_year=2011&pages=459-488&author=T.+I.+Opreaauthor=E.+E.+Mayauthor=A.+Leit%C3%A3oauthor=A.+Tropsha&title=Computational+Systems+Chemical+Biology&doi=10.1007%2F978-1-60761-839-3_18"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">Computational systems chemical biology</span></div><div class="casAuthors">Oprea, Tudor I.; May, Elebeoba E.; Leitao, Andrei; Tropsha, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">672</span>
        (<span class="NLM_cas:issue">Chemoinformatics and Computational Chemical Biology</span>),
    <span class="NLM_cas:pages">459-488</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  There is a crit. need for improving the level of chem. awareness in systems biol.  The data and information related to modulation of genes and proteins by small mols. continue to accumulate at the same time as simulation tools in systems biol. and whole body physiol. based pharmacokinetics (PBPK) continue to evolve.  We called this emerging area at the interface between chem. biol. and systems biol. systems chem. biol. (SCB).  The overarching goal of computational SCB is to develop tools for integrated chem.-biol. data acquisition, filtering and processing, by taking into account relevant information related to interactions between proteins and small mols., possible metabolic transformations of small mols., as well as assocd. information related to genes, networks, small mols., and, where applicable, mutants and variants of those proteins.  There is yet an unmet need to develop an integrated in silico pharmacol./systems biol. continuum that embeds drug-target-clin. outcome (DTCO) triplets, a capability that is vital to the future of chem. biol., pharmacol., and systems biol.  Through the development of the SCB approach, scientists will be able to start addressing, in an integrated simulation environment, questions that make the best use of our ever-growing chem. and biol. data repositories at the system-wide level.  This chapter reviews some of the major research concepts and describes key components that constitute the emerging area of computational systems chem. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHfdr3pD_PjrVg90H21EOLACvtfcHk0ljqrYPn5noZMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlejsr%252FP&md5=b31b3427ddc5ad4b7a6550787e7a717d</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1007%2F978-1-60761-839-3_18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-60761-839-3_18%26sid%3Dliteratum%253Aachs%26aulast%3DOprea%26aufirst%3DT.%2BI.%26aulast%3DMay%26aufirst%3DE.%2BE.%26aulast%3DLeit%25C3%25A3o%26aufirst%3DA.%26aulast%3DTropsha%26aufirst%3DA.%26atitle%3DComputational%2520Systems%2520Chemical%2520Biology%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2011%26volume%3D672%26spage%3D459%26epage%3D488%26doi%3D10.1007%2F978-1-60761-839-3_18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref107"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref107'); return false;" data-citation="" class="refNumLink">107</a></strong><div class="NLM_citation" id="cit107"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vamathevan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czodrowski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunham, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferran, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madabhushi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spitzer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span> <span> </span><span class="NLM_article-title">Applications of Machine Learning in Drug Discovery and Development</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">463</span>– <span class="NLM_lpage">477</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0024-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1038%2Fs41573-019-0024-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=30976107" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosF2rtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=463-477&author=J.+Vamathevanauthor=D.+Clarkauthor=P.+Czodrowskiauthor=I.+Dunhamauthor=E.+Ferranauthor=G.+Leeauthor=B.+Liauthor=A.+Madabhushiauthor=P.+Shahauthor=M.+Spitzerauthor=S.+Zhao&title=Applications+of+Machine+Learning+in+Drug+Discovery+and+Development&doi=10.1038%2Fs41573-019-0024-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">Applications of machine learning in drug discovery and development</span></div><div class="casAuthors">Vamathevan, Jessica; Clark, Dominic; Czodrowski, Paul; Dunham, Ian; Ferran, Edgardo; Lee, George; Li, Bin; Madabhushi, Anant; Shah, Parantu; Spitzer, Michaela; Zhao, Shanrong</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">463-477</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Drug discovery and development pipelines are long, complex and depend on numerous factors.  Machine learning (ML) approaches provide a set of tools that can improve discovery and decision making for well-specified questions with abundant, high-quality data.  Opportunities to apply ML occur in all stages of drug discovery.  Examples include target validation, identification of prognostic biomarkers and anal. of digital pathol. data in clin. trials.  Applications have ranged in context and methodol., with some approaches yielding accurate predictions and insights.  The challenges of applying ML lie primarily with the lack of interpretability and repeatability of ML-generated results, which may limit their application.  In all areas, systematic and comprehensive high-dimensional data still need to be generated.  With ongoing efforts to tackle these issues, as well as increasing awareness of the factors needed to validate ML approaches, the application of ML can promote data-driven decision making and has the potential to speed up the process and reduce failure rates in drug discovery and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreREm2gmVwV7Vg90H21EOLACvtfcHk0lhwxpBaQwwnOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosF2rtrY%253D&md5=211782aeea3d8b9f50368f89177a70d2</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0024-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0024-5%26sid%3Dliteratum%253Aachs%26aulast%3DVamathevan%26aufirst%3DJ.%26aulast%3DClark%26aufirst%3DD.%26aulast%3DCzodrowski%26aufirst%3DP.%26aulast%3DDunham%26aufirst%3DI.%26aulast%3DFerran%26aufirst%3DE.%26aulast%3DLee%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DMadabhushi%26aufirst%3DA.%26aulast%3DShah%26aufirst%3DP.%26aulast%3DSpitzer%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DS.%26atitle%3DApplications%2520of%2520Machine%2520Learning%2520in%2520Drug%2520Discovery%2520and%2520Development%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D463%26epage%3D477%26doi%3D10.1038%2Fs41573-019-0024-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref108"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref108'); return false;" data-citation="" class="refNumLink">108</a></strong><div class="NLM_citation" id="cit108"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrne, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span> <span> </span><span class="NLM_article-title">Concepts of Artificial Intelligence for Computer-Assisted Drug Discovery</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">10520</span>– <span class="NLM_lpage">10594</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.8b00728</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.8b00728" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlCgsbzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2019&pages=10520-10594&author=X.+Yangauthor=Y.+Wangauthor=R.+Byrneauthor=G.+Schneiderauthor=S.+Yang&title=Concepts+of+Artificial+Intelligence+for+Computer-Assisted+Drug+Discovery&doi=10.1021%2Facs.chemrev.8b00728"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Concepts of Artificial Intelligence for Computer-Assisted Drug Discovery</span></div><div class="casAuthors">Yang, Xin; Wang, Yifei; Byrne, Ryan; Schneider, Gisbert; Yang, Shengyong</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">10520-10594</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Artificial intelligence (AI), and, in particular, deep learning as a subcategory of AI, provides opportunities for the discovery and development of innovative drugs.  Various machine learning approaches have recently (re)emerged, some of which may be considered instances of domain-specific AI which have been successfully employed for drug discovery and design.  This review provides a comprehensive portrayal of these machine learning techniques and of their applications in medicinal chem.  After introducing the basic principles, alongside some application notes, of the various machine learning algorithms, the current state-of-the art of AI-assisted pharmaceutical discovery is discussed, including applications in structure- and ligand-based virtual screening, de novo drug design, physicochem. and pharmacokinetic property prediction, drug repurposing, and related aspects.  Finally, several challenges and limitations of the current methods are summarized, with a view to potential future directions for AI-assisted drug discovery and design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBUfBzNv587rVg90H21EOLACvtfcHk0lhwxpBaQwwnOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlCgsbzN&md5=6e9816e30251cd4f15c831e2dd20f100</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.8b00728&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.8b00728%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DByrne%26aufirst%3DR.%26aulast%3DSchneider%26aufirst%3DG.%26aulast%3DYang%26aufirst%3DS.%26atitle%3DConcepts%2520of%2520Artificial%2520Intelligence%2520for%2520Computer-Assisted%2520Drug%2520Discovery%26jtitle%3DChem.%2520Rev.%26date%3D2019%26volume%3D119%26spage%3D10520%26epage%3D10594%26doi%3D10.1021%2Facs.chemrev.8b00728" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref109"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref109'); return false;" data-citation="" class="refNumLink">109</a></strong><div class="NLM_citation" id="cit109"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamanishi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Araki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutteridge, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honda, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanehisa, M.</span></span> <span> </span><span class="NLM_article-title">Prediction of Drug-Target Interaction Networks from the Integration of Chemical and Genomic Spaces</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">i232</span>– <span class="NLM_lpage">240</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btn162</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fbioinformatics%2Fbtn162" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=18586719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvV2rurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2008&pages=i232-240&author=Y.+Yamanishiauthor=M.+Arakiauthor=A.+Gutteridgeauthor=W.+Hondaauthor=M.+Kanehisa&title=Prediction+of+Drug-Target+Interaction+Networks+from+the+Integration+of+Chemical+and+Genomic+Spaces&doi=10.1093%2Fbioinformatics%2Fbtn162"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of drug-target interaction networks from the integration of chemical and genomic spaces</span></div><div class="casAuthors">Yamanishi, Yoshihiro; Araki, Michihiro; Gutteridge, Alex; Honda, Wataru; Kanehisa, Minoru</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">i232-i240</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: The identification of interactions between drugs and target proteins is a key area in genomic drug discovery.  Therefore, there is a strong incentive to develop new methods capable of detecting these potential drug-target interactions efficiently.  Results: In this article, we characterize four classes of drug-target interaction networks in humans involving enzymes, ion channels, G-protein-coupled receptors (GPCRs) and nuclear receptors, and reveal significant correlations between drug structure similarity, target sequence similarity and the drug-target interaction network topol.  We then develop new statistical methods to predict unknown drug-target interaction networks from chem. structure and genomic sequence information simultaneously on a large scale.  The originality of the proposed method lies in the formalization of the drug-target interaction inference as a supervised learning problem for a bipartite graph, the lack of need for 3D structure information of the target proteins, and in the integration of chem. and genomic spaces into a unified space that we call pharmacol. space'.  In the results, we demonstrate the usefulness of our proposed method for the prediction of the four classes of drug-target interaction networks.  Our comprehensively predicted drug-target interaction networks enable us to suggest many potential drug-target interactions and to increase research productivity toward genomic drug discovery.  Availability: Softwares are available upon request.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpO8mG-TwM7v7Vg90H21EOLACvtfcHk0lhwxpBaQwwnOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvV2rurg%253D&md5=3fcd40a826851fc6a5d3758f00552e60</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtn162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtn162%26sid%3Dliteratum%253Aachs%26aulast%3DYamanishi%26aufirst%3DY.%26aulast%3DAraki%26aufirst%3DM.%26aulast%3DGutteridge%26aufirst%3DA.%26aulast%3DHonda%26aufirst%3DW.%26aulast%3DKanehisa%26aufirst%3DM.%26atitle%3DPrediction%2520of%2520Drug-Target%2520Interaction%2520Networks%2520from%2520the%2520Integration%2520of%2520Chemical%2520and%2520Genomic%2520Spaces%26jtitle%3DBioinformatics%26date%3D2008%26volume%3D24%26spage%3Di232%26epage%3D240%26doi%3D10.1093%2Fbioinformatics%2Fbtn162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref110"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref110'); return false;" data-citation="" class="refNumLink">110</a></strong><div class="NLM_citation" id="cit110"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mei, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwoh, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span> <span> </span><span class="NLM_article-title">Drug-Target Interaction Prediction by Learning from Local Information and Neighbors</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">238</span>– <span class="NLM_lpage">245</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/bts670</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fbioinformatics%2Fbts670" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=23162055" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFygsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2013&pages=238-245&author=J.+P.+Meiauthor=C.+K.+Kwohauthor=P.+Yangauthor=X.+L.+Liauthor=J.+Zheng&title=Drug-Target+Interaction+Prediction+by+Learning+from+Local+Information+and+Neighbors&doi=10.1093%2Fbioinformatics%2Fbts670"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-target interaction prediction by learning from local information and neighbors</span></div><div class="casAuthors">Mei, Jian-Ping; Kwoh, Chee-Keong; Yang, Peng; Li, Xiao-Li; Zheng, Jie</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">238-245</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: In silico methods provide efficient ways to predict possible interactions between drugs and targets.  Supervised learning approach, bipartite local model (BLM), has recently been shown to be effective in prediction of drug-target interactions.  However, for drug-candidate compds. or target-candidate proteins that currently have no known interactions available, its pure  local' model is not able to be learned and hence BLM may fail to make correct prediction when involving such kind of new candidates.  Results: We present a simple procedure called neighbor-based interaction-profile inferring (NII) and integrate it into the existing BLM method to handle the new candidate problem.  Specifically, the inferred interaction profile is treated as label information and is used for model learning of new candidates.  This functionality is particularly important in practice to find targets for new drug-candidate compds. and identify targeting drugs for new target-candidate proteins.  Consistent good performance of the new BLM-NII approach has been obsd. in the expt. for the prediction of interactions between drugs and four categories of target proteins.  Esp. for nuclear receptors, BLM-NII achieves the most significant improvement as this dataset contains many drugs/targets with no interactions in the cross-validation.  This demonstrates the effectiveness of the NII strategy and also shows the great potential of BLM-NII for prediction of compd.-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqv1_LYslIEi7Vg90H21EOLACvtfcHk0lhwxpBaQwwnOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFygsbg%253D&md5=bcc7440053bb5716fedf0a4fbbc6dc9c</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbts670&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbts670%26sid%3Dliteratum%253Aachs%26aulast%3DMei%26aufirst%3DJ.%2BP.%26aulast%3DKwoh%26aufirst%3DC.%2BK.%26aulast%3DYang%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DX.%2BL.%26aulast%3DZheng%26aufirst%3DJ.%26atitle%3DDrug-Target%2520Interaction%2520Prediction%2520by%2520Learning%2520from%2520Local%2520Information%2520and%2520Neighbors%26jtitle%3DBioinformatics%26date%3D2013%26volume%3D29%26spage%3D238%26epage%3D245%26doi%3D10.1093%2Fbioinformatics%2Fbts670" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref111"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref111'); return false;" data-citation="" class="refNumLink">111</a></strong><div class="NLM_citation" id="cit111"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, L.</span></span> <span> </span><span class="NLM_article-title">Revealing Drug-Target Interactions with Computational Models and Algorithms</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1714</span>, <span class="refDoi"> DOI: 10.3390/molecules24091714</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.3390%2Fmolecules24091714" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpvF2mt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2019&pages=1714&author=L.+Zhouauthor=Z.+Liauthor=J.+Yangauthor=G.+Tianauthor=F.+Liuauthor=H.+Wenauthor=L.+Pengauthor=M.+Chenauthor=J.+Xiangauthor=L.+Peng&title=Revealing+Drug-Target+Interactions+with+Computational+Models+and+Algorithms&doi=10.3390%2Fmolecules24091714"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">Revealing drug-target interactions with computational models and algorithms</span></div><div class="casAuthors">Zhou, Liqian; Li, Zejun; Yang, Jialiang; Tian, Geng; Liu, Fuxing; Wen, Hong; Peng, Li; Chen, Min; Xiang, Ju; Peng, Lihong</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1714</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Background: Identifying possible drug-target interactions (DTIs) has become an important task in drug research and development.  Although high-throughput screening is becoming available, exptl. methods narrow down the validation space because of extremely high cost, low success rate, and time consumption.  Therefore, various computational models have been exploited to infer DTI candidates.  Methods: We introduced relevant databases and packages, mainly provided a comprehensive review of computational models for DTI identification, including network-based algorithms and machine learning-based methods.  Specially, machine learning-based methods mainly include bipartite local model, matrix factorization, regularized least squares, and deep learning.  Results: Although computational methods have obtained significant improvement in the process of DTI prediction, these models have their limitations.  We discussed potential avenues for boosting DTI prediction accuracy as well as further directions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpopTa4kc7VZrVg90H21EOLACvtfcHk0liGJduSZllGkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpvF2mt7w%253D&md5=f2f063cbcd310a0535cb6f75a2b96de9</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.3390%2Fmolecules24091714&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules24091714%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DTian%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DWen%26aufirst%3DH.%26aulast%3DPeng%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DM.%26aulast%3DXiang%26aufirst%3DJ.%26aulast%3DPeng%26aufirst%3DL.%26atitle%3DRevealing%2520Drug-Target%2520Interactions%2520with%2520Computational%2520Models%2520and%2520Algorithms%26jtitle%3DMolecules%26date%3D2019%26volume%3D24%26spage%3D1714%26doi%3D10.3390%2Fmolecules24091714" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref112"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref112'); return false;" data-citation="" class="refNumLink">112</a></strong><div class="NLM_citation" id="cit112"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hastie, T.</span>; <span class="NLM_string-name">Tibshirani, R.</span>; <span class="NLM_string-name">Friedman, J.</span></span> <i>The Elements of Statistical Learning: Data Mining, Inference, and Prediction</i>, <span class="NLM_edition">2</span>nd ed.; <span class="NLM_publisher-name">Springer</span>, <span class="NLM_year">2009</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1007%2F978-0-387-84858-7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=T.+Hastie&author=R.+Tibshirani&author=J.+Friedman&title=The+Elements+of+Statistical+Learning%3A+Data+Mining%2C+Inference%2C+and+Prediction"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1007%2F978-0-387-84858-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-0-387-84858-7%26sid%3Dliteratum%253Aachs%26aulast%3DHastie%26aufirst%3DT.%26btitle%3DThe%2520Elements%2520of%2520Statistical%2520Learning%253A%2520Data%2520Mining%252C%2520Inference%252C%2520and%2520Prediction%26pub%3DSpringer%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref113"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref113'); return false;" data-citation="" class="refNumLink">113</a></strong><div class="NLM_citation" id="cit113"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mih, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palsson, B. O.</span></span> <span> </span><span class="NLM_article-title">Expanding the Uses of Genome-scale Models with Protein Structures</span>. <i>Mol. Syst. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>, <span class="NLM_elocation-id">e8601</span> <span class="refDoi"> DOI: 10.15252/msb.20188601</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.15252%2Fmsb.20188601" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=31777175" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVGht7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&author=N.+Mihauthor=B.+O.+Palsson&title=Expanding+the+Uses+of+Genome-scale+Models+with+Protein+Structures&doi=10.15252%2Fmsb.20188601"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit113R"><div class="casContent"><span class="casTitleNuber">113</span><div class="casTitle"><span class="NLM_cas:atitle">Expanding the uses of genome-scale models with protein structures</span></div><div class="casAuthors">Mih, Nathan; Palsson, Bernhard O.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Systems Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e8601</span>CODEN:
                <span class="NLM_cas:coden">MSBOC3</span>;
        ISSN:<span class="NLM_cas:issn">1744-4292</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Biol. is reaching a convergence point of its historic reductionist and modern holistic approaches to understanding the living system.  Structural biol. has historically taken the reductionist approach to deeply probe the inner workings of complex mol. machines.  In contrast, systems biol. and genome-scale modeling have organically grown out of the wealth of data now being generated by diverse omics measurements.  In the late 2000s, a proposed interdisciplinary field of structural systems biol. pitched the merger of these two approaches, with widespread applications in pharmacol., disease modeling, protein engineering, and evolutionary studies.  In this commentary, we highlight the challenges of integrating these two fields, with a focus on genome-scale metabolic modeling, and the novel findings that are made possible from such a merger.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlMY-Y1wr5f7Vg90H21EOLACvtfcHk0liGJduSZllGkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVGht7zP&md5=2115cd81c3f187d5c0bbf65e7c17e50d</span></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.15252%2Fmsb.20188601&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.15252%252Fmsb.20188601%26sid%3Dliteratum%253Aachs%26aulast%3DMih%26aufirst%3DN.%26aulast%3DPalsson%26aufirst%3DB.%2BO.%26atitle%3DExpanding%2520the%2520Uses%2520of%2520Genome-scale%2520Models%2520with%2520Protein%2520Structures%26jtitle%3DMol.%2520Syst.%2520Biol.%26date%3D2019%26volume%3D15%26doi%3D10.15252%2Fmsb.20188601" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref114"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref114'); return false;" data-citation="" class="refNumLink">114</a></strong><div class="NLM_citation" id="cit114"><span><span class="NLM_contrib-group"><span class="NLM_string-name">von Bertalanffy, L.</span></span> <i>General System Theory. Foundations, Development, Applications</i>; <span class="NLM_publisher-name">George Braziller, Inc.</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">1969</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1969&author=L.+von+Bertalanffy&title=General+System+Theory.+Foundations%2C+Development%2C+Applications"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3Dvon%2BBertalanffy%26aufirst%3DL.%26btitle%3DGeneral%2520System%2520Theory.%2520Foundations%252C%2520Development%252C%2520Applications%26pub%3DGeorge%2520Braziller%252C%2520Inc%26date%3D1969" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87'],'ref88':['cit88'],'ref89':['cit89'],'ref90':['cit90'],'ref91':['cit91'],'ref92':['cit92'],'ref93':['cit93'],'ref94':['cit94'],'ref95':['cit95'],'ref96':['cit96'],'ref97':['cit97'],'ref98':['cit98'],'ref99':['cit99'],'ref100':['cit100'],'ref101':['cit101'],'ref102':['cit102'],'ref103':['cit103'],'ref104':['cit104'],'ref105':['cit105'],'ref106':['cit106'],'ref107':['cit107'],'ref108':['cit108'],'ref109':['cit109'],'ref110':['cit110'],'ref111':['cit111'],'ref112':['cit112'],'ref113':['cit113'],'ref114':['cit114']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 4 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yoshihisa  Tanaka</span>, <span class="hlFld-ContribAuthor ">Kako  Higashihara</span>, <span class="hlFld-ContribAuthor ">Mai Adachi  Nakazawa</span>, <span class="hlFld-ContribAuthor ">Fumiyoshi  Yamashita</span>, <span class="hlFld-ContribAuthor ">Yoshinori  Tamada</span>, <span class="hlFld-ContribAuthor ">Yasushi  Okuno</span>. </span><span class="cited-content_cbyCitation_article-title">Dynamic changes in gene-to-gene regulatory networks in response to SARS-CoV-2 infection. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2021,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-021-90556-1" title="DOI URL">https://doi.org/10.1038/s41598-021-90556-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-021-90556-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-021-90556-1%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DDynamic%252Bchanges%252Bin%252Bgene-to-gene%252Bregulatory%252Bnetworks%252Bin%252Bresponse%252Bto%252BSARS-CoV-2%252Binfection%26aulast%3DTanaka%26aufirst%3DYoshihisa%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mateus Sá Magalhães  Serafim</span>, <span class="hlFld-ContribAuthor ">Valtair Severino  dos Santos Júnior</span>, <span class="hlFld-ContribAuthor ">Jadson Castro  Gertrudes</span>, <span class="hlFld-ContribAuthor ">Vinícius Gonçalves  Maltarollo</span>, <span class="hlFld-ContribAuthor ">Kathia Maria  Honorio</span>. </span><span class="cited-content_cbyCitation_article-title">Machine learning techniques applied to the drug design and discovery of new antivirals: a brief look over the past decade. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2021,</strong> <em>11 </em>, 1-15. <a href="https://doi.org/10.1080/17460441.2021.1918098" title="DOI URL">https://doi.org/10.1080/17460441.2021.1918098</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2021.1918098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2021.1918098%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DMachine%252Blearning%252Btechniques%252Bapplied%252Bto%252Bthe%252Bdrug%252Bdesign%252Band%252Bdiscovery%252Bof%252Bnew%252Bantivirals%25253A%252Ba%252Bbrief%252Blook%252Bover%252Bthe%252Bpast%252Bdecade%26aulast%3DSerafim%26aufirst%3DMateus%2BS%25C3%25A1%2BMagalh%25C3%25A3es%26date%3D2021%26date%3D2021%26volume%3D11%26spage%3D1%26epage%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">馨  钟</span>. </span><span class="cited-content_cbyCitation_article-title">Explore the Mechanism of TiantaiWuyao Powder in Treating Depression Based on Network Pharmacology and Bioinformatics. </span><span class="cited-content_cbyCitation_journal-name">Advances in Clinical Medicine</span><span> <strong>2021,</strong> <em>11 </em>
                                    (03)
                                     , 1033-1044. <a href="https://doi.org/10.12677/ACM.2021.113148" title="DOI URL">https://doi.org/10.12677/ACM.2021.113148</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.12677/ACM.2021.113148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.12677%2FACM.2021.113148%26sid%3Dliteratum%253Aachs%26jtitle%3DAdvances%2520in%2520Clinical%2520Medicine%26atitle%3DExplore%252Bthe%252BMechanism%252Bof%252BTiantaiWuyao%252BPowder%252Bin%252BTreating%252BDepression%252BBased%252Bon%252BNetwork%252BPharmacology%252Band%252BBioinformatics%26aulast%3D%25E9%2592%259F%26aufirst%3D%25E9%25A6%25A8%26date%3D2021%26volume%3D11%26issue%3D03%26spage%3D1033%26epage%3D1044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Longendri  Aguilera-Mendoza</span>, <span class="hlFld-ContribAuthor ">Yovani  Marrero-Ponce</span>, <span class="hlFld-ContribAuthor ">César R.  García-Jacas</span>, <span class="hlFld-ContribAuthor ">Edgar  Chavez</span>, <span class="hlFld-ContribAuthor ">Jesus A.  Beltran</span>, <span class="hlFld-ContribAuthor ">Hugo A.  Guillen-Ramirez</span>, <span class="hlFld-ContribAuthor ">Carlos A.  Brizuela</span>. </span><span class="cited-content_cbyCitation_article-title">Automatic construction of molecular similarity networks for visual graph mining in chemical space of bioactive peptides: an unsupervised learning approach. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2020,</strong> <em>10 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-020-75029-1" title="DOI URL">https://doi.org/10.1038/s41598-020-75029-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-020-75029-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-020-75029-1%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DAutomatic%252Bconstruction%252Bof%252Bmolecular%252Bsimilarity%252Bnetworks%252Bfor%252Bvisual%252Bgraph%252Bmining%252Bin%252Bchemical%252Bspace%252Bof%252Bbioactive%252Bpeptides%25253A%252Ban%252Bunsupervised%252Blearning%252Bapproach%26aulast%3DAguilera-Mendoza%26aufirst%3DLongendri%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01989/20210220/images/medium/jm9b01989_0005.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01989/20210220/images/large/jm9b01989_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01989&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01989/20210220/images/medium/jm9b01989_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01989/20210220/images/large/jm9b01989_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Exemplary CSN of PARP inhibitors. PARPs 1, 2, and 3 family inhibitors with a measured EC<sub>50</sub> were retrieved from CHEMBL.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> The pairwise chemical similarities between compounds were assessed by means of Tanimoto coefficient (Tc) values calculated for the ECFP4 fingerprints<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> of the molecules generated by Canvas<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> (Schrödinger, LLC, New York, NY, 2019). Pairs of inhibitors were connected only if their calculated Tc value exceeded the threshold value of 0.55. The chemical structures of the inhibitors are shown inside the nodes that are colored according to pEC<sub>50</sub> values ranging from red (lowest potency) to green (highest potency) and sized based on node degree from small (low degree) to large (high degree). Edges are weighted by Tc values from thin (Tc = 0.55) to thick (Tc = 1) width. The network was generated by means of Cytoscape<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> version 3.7.2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01989/20210220/images/large/jm9b01989_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01989&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01989/20210220/images/medium/jm9b01989_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01989/20210220/images/large/jm9b01989_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Drug–target network. The DTN was built from DrugBank<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> version 5.1.5 retrieving the drug–target interactions between approved small molecule drugs and human protein targets. Drugs are represented as circle-shaped nodes, and protein targets are represented as diamond-shaped nodes. As shown in the inset, drugs are color-coded according to the first level anatomical therapeutic chemical (ATC) codes as reported in DrugBank. The nodes size accounts for the node degree from small (low degree) to large (high degree). Edges connect only drugs and targets nodes. The network was generated by means of Cytoscape<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> version 3.7.2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01989/20210220/images/large/jm9b01989_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01989&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01989/20210220/images/medium/jm9b01989_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01989/20210220/images/large/jm9b01989_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Hetionet version 1.0. (a) Metagraph showing the types of nodes used to build the network and the types of links defined to connect the nodes. A detailed description of the meaning of each link type as well as the sources of information used to collect the nodes and to draw the edges is reported in ref <a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a>. (b) Visualization of the whole heterogeneous network. Nodes of the same type are grouped within circles, and links are colored by type. This Figure 3 is reproduced from Figure 1 of <contrib-group><span class="NLM_string-name">Himmelstein, D. S.</span>; <span class="NLM_string-name">Lizee, A.</span>; <span class="NLM_string-name">Hessler, C.</span>; <span class="NLM_string-name">Brueggeman, L.</span>; <span class="NLM_string-name">Chen, S. L.</span>; <span class="NLM_string-name">Hadley, D.</span>; <span class="NLM_string-name">Green, A.</span>; <span class="NLM_string-name">Khankhanian, P.</span>; <span class="NLM_string-name">Baranzini, S. E.</span></contrib-group>; <span class="NLM_year">2017</span>;   <cite><i>eLife</i></cite> (ref <a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a>), published under the Creative Commons Attribution 4.0 International Public License (CC BY 4.0; <a href="https://creativecommons.org/licenses/by/4.0/" class="extLink">https://creativecommons.org/licenses/by/4.0/</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01989/20210220/images/large/jm9b01989_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01989&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01989/20210220/images/medium/jm9b01989_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01989/20210220/images/large/jm9b01989_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Boolean network dynamics. (a) Nodes are colored red or gray based on their “on” or “off” state, respectively. The stepwise evolution of the interactions between nodes determines some sequential steps that are calculated based on a set of rules. The stable state of the network at time step <i>t</i> = S represents an attractor. (b) Attractor landscape. Gray circles represent network states, colored circles represent attractor states. The landscape contains all the possible network states. The sets of states that converge toward an attractor form the basin of that attractor (colored areas). (c) The basins of attractors can be associated with cell phenotypes, and the gene states of the attractors determine the nature of the phenotype.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b01989/20210220/images/large/jm9b01989_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01989&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i28">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88112" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88112" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 114 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dugger, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Platt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, D. B.</span></span> <span> </span><span class="NLM_article-title">Drug Development in the Era of Precision Medicine</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">183</span>– <span class="NLM_lpage">196</span>, <span class="refDoi"> DOI: 10.1038/nrd.2017.226</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1038%2Fnrd.2017.226" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=29217837" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFCjur%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=183-196&author=S.+A.+Duggerauthor=A.+Plattauthor=D.+B.+Goldstein&title=Drug+Development+in+the+Era+of+Precision+Medicine&doi=10.1038%2Fnrd.2017.226"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Drug development in the era of precision medicine</span></div><div class="casAuthors">Dugger, Sarah A.; Platt, Adam; Goldstein, David B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">183-196</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  For the past three decades, the use of genomics to inform drug discovery and development pipelines has generated both excitement and scepticism.  Although earlier efforts successfully identified some new drug targets, the overall clin. efficacy of developed drugs has remained unimpressive, owing in large part to the heterogeneous causes of disease.  Recent technol. and anal. advances in genomics, however, have now made it possible to rapidly identify and interpret the genetic variation underlying a single patient's disease, thereby providing a window into patient-specific mechanisms that cause or contribute to disease, which could ultimately enable the 'precise' targeting of these mechanisms.  Here, we first examine and highlight the successes and limitations of the earlier phases of genomics in drug discovery and development.  We then review the current major efforts in precision medicine and discuss the potential broader utility of mechanistically guided treatments going forward.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpshWBfUfoWf7Vg90H21EOLACvtfcHk0lhiLkOAPXLjMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFCjur%252FM&md5=e06762106391d7646c686e6ef773398a</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2017.226&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2017.226%26sid%3Dliteratum%253Aachs%26aulast%3DDugger%26aufirst%3DS.%2BA.%26aulast%3DPlatt%26aufirst%3DA.%26aulast%3DGoldstein%26aufirst%3DD.%2BB.%26atitle%3DDrug%2520Development%2520in%2520the%2520Era%2520of%2520Precision%2520Medicine%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D183%26epage%3D196%26doi%3D10.1038%2Fnrd.2017.226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span> <span> </span><span class="NLM_article-title">Big Data and Artificial Intelligence Modeling for Drug Discovery</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">573</span>– <span class="NLM_lpage">589</span>, <span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010919-023324</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1146%2Fannurev-pharmtox-010919-023324" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=31518513" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslygtL%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2020&pages=573-589&author=H.+Zhu&title=Big+Data+and+Artificial+Intelligence+Modeling+for+Drug+Discovery&doi=10.1146%2Fannurev-pharmtox-010919-023324"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Big Data and Artificial Intelligence Modeling for Drug Discovery</span></div><div class="casAuthors">Zhu, Hao</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">573-589</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">Due to the massive data sets available for drug candidates, modern drug discovery has advanced to the big data era.  Central to this shift is the development of artificial intelligence approaches to implementing innovative modeling based on the dynamic, heterogeneous, and large nature of drug data sets.  As a result, recently developed artificial intelligence approaches such as deep learning and relevant modeling studies provide new solns. to efficacy and safety evaluations of drug candidates based on big data modeling and anal.  The resulting models provided deep insights into the continuum from chem. structure to in vitro, in vivo, and clin. outcomes.  The relevant novel data mining, curation, and management techniques provided crit. support to recent modeling studies.  In summary, the new advancement of artificial intelligence in the big data era has paved the road to future rational drug development and optimization, which will have a significant impact on drug discovery procedures and, eventually, public health.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonOMb5S3akSLVg90H21EOLACvtfcHk0lhOLwhGyeRZuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslygtL%252FI&md5=fa114093fe962c67655d4d7b31d502e9</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010919-023324&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010919-023324%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DH.%26atitle%3DBig%2520Data%2520and%2520Artificial%2520Intelligence%2520Modeling%2520for%2520Drug%2520Discovery%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2020%26volume%3D60%26spage%3D573%26epage%3D589%26doi%3D10.1146%2Fannurev-pharmtox-010919-023324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
Der Greef, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mcburney, R. N.</span></span> <span> </span><span class="NLM_article-title">Rescuing Drug Discovery: In Vivo Systems Pathology and Systems Pharmacology</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">961</span>– <span class="NLM_lpage">967</span>, <span class="refDoi"> DOI: 10.1038/nrd1904</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1038%2Fnrd1904" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=16341061" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1yktbjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2005&pages=961-967&author=J.+Van%0ADer+Greefauthor=R.+N.+Mcburney&title=Rescuing+Drug+Discovery%3A+In+Vivo+Systems+Pathology+and+Systems+Pharmacology&doi=10.1038%2Fnrd1904"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Innovation: Rescuing drug discovery: in vivo systems pathology and systems pharmacology</span></div><div class="casAuthors">van der Greef, Jan; McBurney, Robert N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">961-967</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The pharmaceutical industry is currently beleaguered by close scrutiny from the financial community, regulators and the general public.  Productivity, in terms of new drug approvals, has generally been falling for almost a decade and the safety of a no. of highly successful drugs has recently been brought into question.  Here, we discuss whether taking an in vivo systems approach to drug discovery and development could be the paradigm shift that rescues the industry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjvid8S6CQM7Vg90H21EOLACvtfcHk0lhOLwhGyeRZuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1yktbjK&md5=ebebfc6108b47eaa3225a9ec2f2fd317</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrd1904&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1904%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BDer%2BGreef%26aufirst%3DJ.%26aulast%3DMcburney%26aufirst%3DR.%2BN.%26atitle%3DRescuing%2520Drug%2520Discovery%253A%2520In%2520Vivo%2520Systems%2520Pathology%2520and%2520Systems%2520Pharmacology%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2005%26volume%3D4%26spage%3D961%26epage%3D967%26doi%3D10.1038%2Fnrd1904" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolognesi, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minarini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumiatti, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Recanatini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melchiorre, C.</span></span> <span> </span><span class="NLM_article-title">Multi-Target-Directed Ligands to Combat Neurodegenerative Diseases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">347</span>– <span class="NLM_lpage">372</span>, <span class="refDoi"> DOI: 10.1021/jm7009364</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm7009364" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsl2qsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=347-372&author=A.+Cavalliauthor=M.+L.+Bolognesiauthor=A.+Minariniauthor=M.+Rosiniauthor=V.+Tumiattiauthor=M.+Recanatiniauthor=C.+Melchiorre&title=Multi-Target-Directed+Ligands+to+Combat+Neurodegenerative+Diseases&doi=10.1021%2Fjm7009364"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Multi-target-Directed Ligands to Combat Neurodegenerative Diseases</span></div><div class="casAuthors">Cavalli, Andrea; Bolognesi, Maria Laura; Minarini, Anna; Rosini, Michela; Tumiatti, Vincenzo; Recanatini, Maurizio; Melchiorre, Carlo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">347-372</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The authors review the investigations reported in the past decade that illustrate the ability of medicinal chemists to design multi-target-directed ligands (MTDLs) to confront neurodegenerative diseases.  The authors briefly illustrate the multifactorial nature of these diseases before discussing the possible development of MTDLs in light of the known mol. bases of the main neurodegenerative pathologies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYY0I0UHEir7Vg90H21EOLACvtfcHk0lhOLwhGyeRZuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsl2qsbk%253D&md5=c682235151489316d043cf97f78e56f3</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fjm7009364&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm7009364%26sid%3Dliteratum%253Aachs%26aulast%3DCavalli%26aufirst%3DA.%26aulast%3DBolognesi%26aufirst%3DM.%2BL.%26aulast%3DMinarini%26aufirst%3DA.%26aulast%3DRosini%26aufirst%3DM.%26aulast%3DTumiatti%26aufirst%3DV.%26aulast%3DRecanatini%26aufirst%3DM.%26aulast%3DMelchiorre%26aufirst%3DC.%26atitle%3DMulti-Target-Directed%2520Ligands%2520to%2520Combat%2520Neurodegenerative%2520Diseases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D347%26epage%3D372%26doi%3D10.1021%2Fjm7009364" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span> <span> </span><span class="NLM_article-title">Network Pharmacology: The next Paradigm in Drug Discovery</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">682</span>– <span class="NLM_lpage">690</span>, <span class="refDoi"> DOI: 10.1038/nchembio.118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1038%2Fnchembio.118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=18936753" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Kgs7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=682-690&author=A.+L.+Hopkins&title=Network+Pharmacology%3A+The+next+Paradigm+in+Drug+Discovery&doi=10.1038%2Fnchembio.118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Network pharmacology: the next paradigm in drug discovery</span></div><div class="casAuthors">Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">682-690</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The dominant paradigm in drug discovery is the concept of designing maximally selective ligands to act on individual drug targets.  However, many effective drugs act via modulation of multiple proteins rather than single targets.  Advances in systems biol. are revealing a phenotypic robustness and a network structure that strongly suggests that exquisitely selective compds., compared with multi-target drugs, may exhibit lower than desired clin. efficacy.  This new appreciation of the role of polypharmacol. has significant implications for tackling the two major sources of attrition in drug development, efficacy and toxicity.  Integrating network biol. and polypharmacol. holds the promise of expanding the current opportunity space for druggable targets.  However, the rational design of polypharmacol. faces considerable challenges in the need for new methods to validate target combinations and optimize multiple structure-activity relationships while maintaining drug-like properties.  Advances in these areas are creating the foundation of the next paradigm in drug discovery: network pharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqHy4FqkK-h7Vg90H21EOLACvtfcHk0ljEeCF-oCAvzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Kgs7fK&md5=03b8be3ddbcab7dc65afd17df7ee8395</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.118%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DNetwork%2520Pharmacology%253A%2520The%2520next%2520Paradigm%2520in%2520Drug%2520Discovery%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2008%26volume%3D4%26spage%3D682%26epage%3D690%26doi%3D10.1038%2Fnchembio.118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hart, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span> <span> </span><span class="NLM_article-title">Providing Data Science Support for Systems Pharmacology and Its Implications to Drug Discovery</span>. <i>Expert Opin. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">241</span>– <span class="NLM_lpage">256</span>, <span class="refDoi"> DOI: 10.1517/17460441.2016.1135126</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1517%2F17460441.2016.1135126" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=26689499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtV2rtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=241-256&author=T.+Hartauthor=L.+Xie&title=Providing+Data+Science+Support+for+Systems+Pharmacology+and+Its+Implications+to+Drug+Discovery&doi=10.1517%2F17460441.2016.1135126"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Providing data science support for systems pharmacology and its implications to drug discovery</span></div><div class="casAuthors">Hart, Thomas; Xie, Lei</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">241-256</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">The conventional one-drug-one-target-one-disease drug discovery process has been less successful in tracking multi-genic, multi-faceted complex diseases.  Systems pharmacol. has emerged as a new discipline to tackle the current challenges in drug discovery.  The goal of systems pharmacol. is to transform huge, heterogeneous, and dynamic biol. and clin. data into interpretable and actionable mechanistic models for decision making in drug discovery and patient treatment.  Thus, big data technol. and data science will play an essential role in systems pharmacol.  This paper critically reviews the impact of three fundamental concepts of data science on systems pharmacol.: similarity inference, overfitting avoidance, and disentangling causality from correlation.  The authors then discuss recent advances and future directions in applying the three concepts of data science to drug discovery, with a focus on proteome-wide context-specific quant. drug target deconvolution and personalized adverse drug reaction prediction.  Data science will facilitate reducing the complexity of systems pharmacol. modeling, detecting hidden correlations between complex data sets, and distinguishing causation from correlation.  The power of data science can only be fully realized when integrated with mechanism-based multi-scale modeling that explicitly takes into account the hierarchical organization of biol. systems from nucleic acid to proteins, to mol. interaction networks, to cells, to tissues, to patients, and to populations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEOsRequFdVLVg90H21EOLACvtfcHk0ljEeCF-oCAvzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtV2rtb0%253D&md5=c18c63b8c93fb06e2d7f19dd0dc50a2d</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1517%2F17460441.2016.1135126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2016.1135126%26sid%3Dliteratum%253Aachs%26aulast%3DHart%26aufirst%3DT.%26aulast%3DXie%26aufirst%3DL.%26atitle%3DProviding%2520Data%2520Science%2520Support%2520for%2520Systems%2520Pharmacology%2520and%2520Its%2520Implications%2520to%2520Drug%2520Discovery%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2016%26volume%3D11%26spage%3D241%26epage%3D256%26doi%3D10.1517%2F17460441.2016.1135126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Draizen, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, P. E.</span></span> <span> </span><span class="NLM_article-title">Harnessing Big Data for Systems Pharmacology</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">245</span>– <span class="NLM_lpage">262</span>, <span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010716-104659</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1146%2Fannurev-pharmtox-010716-104659" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=27814027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhslyksb7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=245-262&author=L.+Xieauthor=E.+J.+Draizenauthor=P.+E.+Bourne&title=Harnessing+Big+Data+for+Systems+Pharmacology&doi=10.1146%2Fannurev-pharmtox-010716-104659"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Harnessing Big Data for Systems Pharmacology</span></div><div class="casAuthors">Xie, Lei; Draizen, Eli J.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">245-262</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">Systems pharmacol. aims to holistically understand mechanisms of drug actions to support drug discovery and clin. practice.  Systems pharmacol. modeling (SPM) is data driven.  It integrates an exponentially growing amt. of data at multiple scales (genetic, mol., cellular, organismal, and environmental).  The goal of SPM is to develop mechanistic or predictive multiscale models that are interpretable and actionable.  The current explosions in genomics and other omics data, as well as the tremendous advances in big data technologies, have already enabled biologists to generate novel hypotheses and gain new knowledge through computational models of genome-wide, heterogeneous, and dynamic data sets.  More work is needed to interpret and predict a drug response phenotype, which is dependent on many known and unknown factors.  To gain a comprehensive understanding of drug actions, SPM requires close collaborations between domain experts from diverse fields and integration of heterogeneous models from biophysics, mathematics, statistics, machine learning, and semantic webs.  This creates challenges in model management, model integration, model translation, and knowledge integration.  In this review, we discuss several emergent issues in SPM and potential solns. using big data technol. and analytics.  The concurrent development of high-throughput techniques, cloud computing, data science, and the semantic web will likely allow SPM to be findable, accessible, interoperable, reusable, reliable, interpretable, and actionable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrojceq7LgsT7Vg90H21EOLACvtfcHk0ljEeCF-oCAvzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhslyksb7M&md5=120756181391dce7785bfcf0485ff10b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010716-104659&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010716-104659%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DDraizen%26aufirst%3DE.%2BJ.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DHarnessing%2520Big%2520Data%2520for%2520Systems%2520Pharmacology%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2017%26volume%3D57%26spage%3D245%26epage%3D262%26doi%3D10.1146%2Fannurev-pharmtox-010716-104659" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gentili, P. L.</span></span> <i>Untangling Complex Systems. A Grand Challenge for Science</i>; <span class="NLM_publisher-name">CRC Press</span>: <span class="NLM_publisher-loc">Boca Raton, FL, U.S.</span>, <span class="NLM_year">2019</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=P.+L.+Gentili&title=Untangling+Complex+Systems.+A+Grand+Challenge+for+Science"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DGentili%26aufirst%3DP.%2BL.%26btitle%3DUntangling%2520Complex%2520Systems.%2520A%2520Grand%2520Challenge%2520for%2520Science%26pub%3DCRC%2520Press%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Loscalzo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barabasi, A. L.</span></span> <span> </span><span class="NLM_article-title">Systems Biology and the Future of Medicine</span>. <i>Wiley Interdiscip. Rev. Syst. Biol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">619</span>– <span class="NLM_lpage">627</span>, <span class="refDoi"> DOI: 10.1002/wsbm.144</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1002%2Fwsbm.144" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=21928407" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A280%3ADC%252BC3MfovVShuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=619-627&author=J.+Loscalzoauthor=A.+L.+Barabasi&title=Systems+Biology+and+the+Future+of+Medicine&doi=10.1002%2Fwsbm.144"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Systems biology and the future of medicine</span></div><div class="casAuthors">Loscalzo Joseph; Barabasi Albert-Laszlo</div><div class="citationInfo"><span class="NLM_cas:title">Wiley interdisciplinary reviews. Systems biology and medicine</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">619-27</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Contemporary views of human disease are based on simple correlation between clinical syndromes and pathological analysis dating from the late 19th century.  Although this approach to disease diagnosis, prognosis, and treatment has served the medical establishment and society well for many years, it has serious shortcomings for the modern era of the genomic medicine that stem from its reliance on reductionist principles of experimentation and analysis.  Quantitative, holistic systems biology applied to human disease offers a unique approach for diagnosing established disease, defining disease predilection, and developing individualized (personalized) treatment strategies that can take full advantage of modern molecular pathobiology and the comprehensive data sets that are rapidly becoming available for populations and individuals.  In this way, systems pathobiology offers the promise of redefining our approach to disease and the field of medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTkNpHR0NjEf825P7XSLhESfW6udTcc2eZ5s4JprfiAv7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MfovVShuw%253D%253D&md5=f9c57e7f65de8529def4b2aec3692ab3</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1002%2Fwsbm.144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fwsbm.144%26sid%3Dliteratum%253Aachs%26aulast%3DLoscalzo%26aufirst%3DJ.%26aulast%3DBarabasi%26aufirst%3DA.%2BL.%26atitle%3DSystems%2520Biology%2520and%2520the%2520Future%2520of%2520Medicine%26jtitle%3DWiley%2520Interdiscip.%2520Rev.%2520Syst.%2520Biol.%2520Med.%26date%3D2011%26volume%3D3%26spage%3D619%26epage%3D627%26doi%3D10.1002%2Fwsbm.144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Euler, L.</span></span> <span> </span><span class="NLM_article-title">Solutio Problematis Ad Geometriam Situs Pertinentis. (1741)</span>.  <i>Euler Archive—All Works</i>; No. 53, <a href="https://scholarlycommons.pacific.edu/euler-works/53" class="extLink">https://scholarlycommons.pacific.edu/euler-works/53</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Euler%2C+L.+Solutio+Problematis+Ad+Geometriam+Situs+Pertinentis.+%281741%29.+Euler+Archive%E2%80%94All+Works%3B+No.+53%2C+https%3A%2F%2Fscholarlycommons.pacific.edu%2Feuler-works%2F53."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DEuler%26aufirst%3DL.%26atitle%3DSolutio%2520Problematis%2520Ad%2520Geometriam%2520Situs%2520Pertinentis.%2520%25281741%2529%26btitle%3DEuler%2520Archive%25E2%2580%2594All%2520Works" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Newman, M. E. J.</span></span> <span> </span><span class="NLM_article-title">The Structure and Function of Complex Networks</span>. <i>SIAM Rev.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">167</span>– <span class="NLM_lpage">256</span>, <span class="refDoi"> DOI: 10.1137/S003614450342480</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1137%2FS003614450342480" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2003&pages=167-256&author=M.+E.+J.+Newman&title=The+Structure+and+Function+of+Complex+Networks&doi=10.1137%2FS003614450342480"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1137%2FS003614450342480&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1137%252FS003614450342480%26sid%3Dliteratum%253Aachs%26aulast%3DNewman%26aufirst%3DM.%2BE.%2BJ.%26atitle%3DThe%2520Structure%2520and%2520Function%2520of%2520Complex%2520Networks%26jtitle%3DSIAM%2520Rev.%26date%3D2003%26volume%3D45%26spage%3D167%26epage%3D256%26doi%3D10.1137%2FS003614450342480" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Estrada, E.</span></span> <span> </span><span class="NLM_article-title">Graph and Network Theory in Physics</span>. <i>arXiv</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_fpage">1302.4378</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=1302.4378&author=E.+Estrada&title=Graph+and+Network+Theory+in+Physics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEstrada%26aufirst%3DE.%26atitle%3DGraph%2520and%2520Network%2520Theory%2520in%2520Physics%26jtitle%3DarXiv%26date%3D2013%26spage%3D1302.4378" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barabási, A.-L.</span></span> <i>Network Science</i>; <span class="NLM_publisher-name">Cambridge University Press</span>: <span class="NLM_publisher-loc">Cambridge, U.K.</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=A.-L.+Barab%C3%A1si&title=Network+Science"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DBarab%25C3%25A1si%26aufirst%3DA.-L.%26btitle%3DNetwork%2520Science%26pub%3DCambridge%2520University%2520Press%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Csermely, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korcsmáros, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiss, H. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">London, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span> <span> </span><span class="NLM_article-title">Structure and Dynamics of Molecular Networks: A Novel Paradigm of Drug Discovery: A Comprehensive Review</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">333</span>– <span class="NLM_lpage">408</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2013.01.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1016%2Fj.pharmthera.2013.01.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=23384594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC3sXls1CgtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2013&pages=333-408&author=P.+Csermelyauthor=T.+Korcsm%C3%A1rosauthor=H.+J.+M.+Kissauthor=G.+Londonauthor=R.+Nussinov&title=Structure+and+Dynamics+of+Molecular+Networks%3A+A+Novel+Paradigm+of+Drug+Discovery%3A+A+Comprehensive+Review&doi=10.1016%2Fj.pharmthera.2013.01.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and dynamics of molecular networks: A novel paradigm of drug discovery</span></div><div class="casAuthors">Csermely, Peter; Korcsmaros, Tamas; Kiss, Huba J. M.; London, Gabor; Nussinov, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">333-408</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Despite considerable progress in genome- and proteome-based high-throughput screening methods and in rational drug design, the increase in approved drugs in the past decade did not match the increase of drug development costs.  Network description and anal. not only give a systems-level understanding of drug action and disease complexity, but can also help to improve the efficiency of drug design.  We give a comprehensive assessment of the anal. tools of network topol. and dynamics.  The state-of-the-art use of chem. similarity, protein structure, protein-protein interaction, signaling, genetic interaction and metabolic networks in the discovery of drug targets is summarized.  We propose that network targeting follows two basic strategies.  The "central hit strategy" selectively targets central nodes/edges of the flexible networks of infectious agents or cancer cells to kill them.  The "network influence strategy" works against other diseases, where an efficient reconfiguration of rigid networks needs to be achieved by targeting the neighbors of central nodes/edges.  It is shown how network techniques can help in the identification of single-target, edgetic, multi-target and allo-network drug target candidates.  We review the recent boom in network methods helping hit identification, lead selection optimizing drug efficacy, as well as minimizing side-effects and drug toxicity.  Successful network-based drug development strategies are shown through the examples of infections, cancer, metabolic diseases, neurodegenerative diseases and aging.  Summarizing >1200 refs. we suggest an optimized protocol of network-aided drug development, and provide a list of systems-level hallmarks of drug quality.  Finally, we highlight network-related drug development trends helping to achieve these hallmarks by a cohesive, global approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFqNviuDk7kbVg90H21EOLACvtfcHk0li9FHe2pR--Yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXls1CgtLk%253D&md5=d70c48a1572815c1b4905cc6e8cdfaa6</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2013.01.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2013.01.016%26sid%3Dliteratum%253Aachs%26aulast%3DCsermely%26aufirst%3DP.%26aulast%3DKorcsm%25C3%25A1ros%26aufirst%3DT.%26aulast%3DKiss%26aufirst%3DH.%2BJ.%2BM.%26aulast%3DLondon%26aufirst%3DG.%26aulast%3DNussinov%26aufirst%3DR.%26atitle%3DStructure%2520and%2520Dynamics%2520of%2520Molecular%2520Networks%253A%2520A%2520Novel%2520Paradigm%2520of%2520Drug%2520Discovery%253A%2520A%2520Comprehensive%2520Review%26jtitle%3DPharmacol.%2520Ther.%26date%3D2013%26volume%3D138%26spage%3D333%26epage%3D408%26doi%3D10.1016%2Fj.pharmthera.2013.01.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span> <span> </span><span class="NLM_article-title">Freely Accessible Chemical Database Resources of Compounds for In Silico Drug Discovery</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">7581</span>– <span class="NLM_lpage">7597</span>, <span class="refDoi"> DOI: 10.2174/0929867325666180508100436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.2174%2F0929867325666180508100436" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2020&pages=7581-7597&author=J.+Yangauthor=D.+Wangauthor=C.+Jiaauthor=M.+Wangauthor=G.+Haoauthor=G.+Yang&title=Freely+Accessible+Chemical+Database+Resources+of+Compounds+for+In+Silico+Drug+Discovery&doi=10.2174%2F0929867325666180508100436"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.2174%2F0929867325666180508100436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F0929867325666180508100436%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DJia%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DHao%26aufirst%3DG.%26aulast%3DYang%26aufirst%3DG.%26atitle%3DFreely%2520Accessible%2520Chemical%2520Database%2520Resources%2520of%2520Compounds%2520for%2520In%2520Silico%2520Drug%2520Discovery%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2020%26volume%3D26%26spage%3D7581%26epage%3D7597%26doi%3D10.2174%2F0929867325666180508100436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mendez, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bento, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chambers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Veij, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Félix, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magariños, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mosquera, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mutowo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowotka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordillo-Marañón, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Junco, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mugumbate, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Lopez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosc, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radoux, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Segura-Cabrera, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hersey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leach, A. R.</span></span> <span> </span><span class="NLM_article-title">ChEMBL: Towards Direct Deposition of Bioassay Data</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D930</span>– <span class="NLM_lpage">D940</span>, <span class="refDoi"> DOI: 10.1093/nar/gky1075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fnar%2Fgky1075" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=30398643" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1Gqtr%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D930-D940&issue=D1&author=D.+Mendezauthor=A.+Gaultonauthor=A.+P.+Bentoauthor=J.+Chambersauthor=M.+De+Veijauthor=E.+F%C3%A9lixauthor=M.+P.+Magari%C3%B1osauthor=J.+F.+Mosqueraauthor=P.+Mutowoauthor=M.+Nowotkaauthor=M.+Gordillo-Mara%C3%B1%C3%B3nauthor=F.+Hunterauthor=L.+Juncoauthor=G.+Mugumbateauthor=M.+Rodriguez-Lopezauthor=F.+Atkinsonauthor=N.+Boscauthor=C.+J.+Radouxauthor=A.+Segura-Cabreraauthor=A.+Herseyauthor=A.+R.+Leach&title=ChEMBL%3A+Towards+Direct+Deposition+of+Bioassay+Data&doi=10.1093%2Fnar%2Fgky1075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">ChEMBL: towards direct deposition of bioassay data</span></div><div class="casAuthors">Mendez, David; Gaulton, Anna; Bento, A. Patricia; Chambers, Jon; De Veij, Marleen; Felix, Eloy; Magarinos, Maria Paula; Mosquera, Juan F.; Mutowo, Prudence; Nowotka, Michal; Gordillo-Maranon, Maria; Hunter, Fiona; Junco, Laura; Mugumbate, Grace; Rodriguez-Lopez, Milagros; Atkinson, Francis; Bosc, Nicolas; Radoux, Chris J.; Cabrera, Aldo Segura; Hersey, Anne; Leach, Andrew R.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D930-D940</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">ChEMBL is a large, open-access bioactivity database, previously described in the 2012, 2014 and 2017 Nucleic Acids Research Database Issues.  In the last two years, several important improvements have been made to the database and are described here.  These include more robust capture and representation of assay details; a new data deposition system, allowing updating of data sets and deposition of supplementary data; and a completely redesigned web interface, with enhanced search and filtering capabilities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqM7KYSXc4jerVg90H21EOLACvtfcHk0ljwkc1Em0A6yA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1Gqtr%252FF&md5=d80660ce87d5b3e0773594e4fddec046</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky1075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky1075%26sid%3Dliteratum%253Aachs%26aulast%3DMendez%26aufirst%3DD.%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DChambers%26aufirst%3DJ.%26aulast%3DDe%2BVeij%26aufirst%3DM.%26aulast%3DF%25C3%25A9lix%26aufirst%3DE.%26aulast%3DMagari%25C3%25B1os%26aufirst%3DM.%2BP.%26aulast%3DMosquera%26aufirst%3DJ.%2BF.%26aulast%3DMutowo%26aufirst%3DP.%26aulast%3DNowotka%26aufirst%3DM.%26aulast%3DGordillo-Mara%25C3%25B1%25C3%25B3n%26aufirst%3DM.%26aulast%3DHunter%26aufirst%3DF.%26aulast%3DJunco%26aufirst%3DL.%26aulast%3DMugumbate%26aufirst%3DG.%26aulast%3DRodriguez-Lopez%26aufirst%3DM.%26aulast%3DAtkinson%26aufirst%3DF.%26aulast%3DBosc%26aufirst%3DN.%26aulast%3DRadoux%26aufirst%3DC.%2BJ.%26aulast%3DSegura-Cabrera%26aufirst%3DA.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26atitle%3DChEMBL%253A%2520Towards%2520Direct%2520Deposition%2520of%2520Bioassay%2520Data%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2019%26volume%3D47%26issue%3DD1%26spage%3DD930%26epage%3DD940%26doi%3D10.1093%2Fnar%2Fgky1075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gindulyte, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoemaker, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiessen, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaslavsky, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolton, E. E.</span></span> <span> </span><span class="NLM_article-title">PubChem 2019 Update: Improved Access to Chemical Data</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D1102</span>– <span class="NLM_lpage">D1109</span>, <span class="refDoi"> DOI: 10.1093/nar/gky1033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fnar%2Fgky1033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=30371825" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D1102-D1109&issue=D1&author=S.+Kimauthor=J.+Chenauthor=T.+Chengauthor=A.+Gindulyteauthor=J.+Heauthor=S.+Heauthor=Q.+Liauthor=B.+A.+Shoemakerauthor=P.+A.+Thiessenauthor=B.+Yuauthor=L.+Zaslavskyauthor=J.+Zhangauthor=E.+E.+Bolton&title=PubChem+2019+Update%3A+Improved+Access+to+Chemical+Data&doi=10.1093%2Fnar%2Fgky1033"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky1033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky1033%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DT.%26aulast%3DGindulyte%26aufirst%3DA.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DShoemaker%26aufirst%3DB.%2BA.%26aulast%3DThiessen%26aufirst%3DP.%2BA.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DZaslavsky%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DBolton%26aufirst%3DE.%2BE.%26atitle%3DPubChem%25202019%2520Update%253A%2520Improved%2520Access%2520to%2520Chemical%2520Data%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2019%26volume%3D47%26issue%3DD1%26spage%3DD1102%26epage%3DD1109%26doi%3D10.1093%2Fnar%2Fgky1033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wishart, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feunang, Y. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grant, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sajed, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayeeda, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Assempour, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iynkkaran, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maciejewski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gale, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, M.</span></span> <span> </span><span class="NLM_article-title">DrugBank 5.0: A Major Update to the DrugBank Database for 2018</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D1074</span>– <span class="NLM_lpage">D1082</span>, <span class="refDoi"> DOI: 10.1093/nar/gkx1037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fnar%2Fgkx1037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=29126136" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlGisbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D1074-D1082&issue=D1&author=D.+S.+Wishartauthor=Y.+D.+Feunangauthor=A.+C.+Guoauthor=E.+J.+Loauthor=A.+Marcuauthor=J.+R.+Grantauthor=T.+Sajedauthor=D.+Johnsonauthor=C.+Liauthor=Z.+Sayeedaauthor=N.+Assempourauthor=I.+Iynkkaranauthor=Y.+Liuauthor=A.+Maciejewskiauthor=N.+Galeauthor=A.+Wilsonauthor=L.+Chinauthor=R.+Cummingsauthor=D.+Leauthor=A.+Ponauthor=C.+Knoxauthor=M.+Wilson&title=DrugBank+5.0%3A+A+Major+Update+to+the+DrugBank+Database+for+2018&doi=10.1093%2Fnar%2Fgkx1037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">DrugBank 5.0: a major update to the DrugBank database for 2018</span></div><div class="casAuthors">Wishart, David S.; Feunang, Yannick D.; Guo, An C.; Lo, Elvis J.; Marcu, Ana; Grant, Jason R.; Sajed, Tanvir; Johnson, Daniel; Li, Carin; Sayeeda, Zinat; Assempour, Nazanin; Iynkkaran, Ithayavani; Liu, Yifeng; Maciejewski, Adam; Gale, Nicola; Wilson, Alex; Chin, Lucy; Cummings, Ryan; Le, Diana; Pon, Allison; Knox, Craig; Wilson, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1074-D1082</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">DrugBank is a web-enabled database contg. comprehensivemol. information about drugs, their mechanisms, their interactions and their targets.  First described in 2006, Drug- Bank has continued to evolve over the past 12 years in response to marked improvements to web stds. and changing needs for drug research and development.  This year's update, DrugBank 5.0, represents the most significant upgrade to the database in more than 10 years.  In many cases, existing data content has grown by 100% or more over the last update.  For instance, the total no. of investigational drugs in the database has grown by almost 300%, the no. of drug-drug interactions has grown by nearly 600% and the no. of SNP-assocd. drug effects has grown more than 3000%.  Significant improvements have been made to the quantity, quality and consistency of drug indications, drug binding data as well as drug-drug and drug-food interactions.  A great deal of brand new data have also been added to DrugBank 5.0.  This includes information on the influence of hundreds of drugs on metabolite levels (pharmacometabolomics), gene expression levels (pharmacotranscriptomics) and protein expression levels (pharmacoproteomics).  New data have also been added on the status of hundreds of newdrug clin. trials and existing drug repurposing trials.  Many other important improvements in the content, interface and performance of the DrugBank website have been made and these should greatly enhance its ease of use, utility and potential applications in many areas of pharmacol. research, pharmaceutical science and drug education.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrc8rfBoMOOyLVg90H21EOLACvtfcHk0lghj_DSIYYs5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlGisbvI&md5=986b28c7ea546596a26dd3ba38f05fee</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx1037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx1037%26sid%3Dliteratum%253Aachs%26aulast%3DWishart%26aufirst%3DD.%2BS.%26aulast%3DFeunang%26aufirst%3DY.%2BD.%26aulast%3DGuo%26aufirst%3DA.%2BC.%26aulast%3DLo%26aufirst%3DE.%2BJ.%26aulast%3DMarcu%26aufirst%3DA.%26aulast%3DGrant%26aufirst%3DJ.%2BR.%26aulast%3DSajed%26aufirst%3DT.%26aulast%3DJohnson%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DSayeeda%26aufirst%3DZ.%26aulast%3DAssempour%26aufirst%3DN.%26aulast%3DIynkkaran%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DMaciejewski%26aufirst%3DA.%26aulast%3DGale%26aufirst%3DN.%26aulast%3DWilson%26aufirst%3DA.%26aulast%3DChin%26aufirst%3DL.%26aulast%3DCummings%26aufirst%3DR.%26aulast%3DLe%26aufirst%3DD.%26aulast%3DPon%26aufirst%3DA.%26aulast%3DKnox%26aufirst%3DC.%26aulast%3DWilson%26aufirst%3DM.%26atitle%3DDrugBank%25205.0%253A%2520A%2520Major%2520Update%2520to%2520the%2520DrugBank%2520Database%2520for%25202018%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26issue%3DD1%26spage%3DD1074%26epage%3DD1082%26doi%3D10.1093%2Fnar%2Fgkx1037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pence, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, A.</span></span> <span> </span><span class="NLM_article-title">ChemSpider: An Online Chemical Information Resource</span>. <i>J. Chem. Educ.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">1123</span>– <span class="NLM_lpage">1124</span>, <span class="refDoi"> DOI: 10.1021/ed100697w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ed100697w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtV2jtr%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2010&pages=1123-1124&author=H.+E.+Penceauthor=A.+Williams&title=ChemSpider%3A+An+Online+Chemical+Information+Resource&doi=10.1021%2Fed100697w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">ChemSpider: An Online Chemical Information Resource</span></div><div class="casAuthors">Pence, Harry E.; Williams, Antony</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Education</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1123-1124</span>CODEN:
                <span class="NLM_cas:coden">JCEDA8</span>;
        ISSN:<span class="NLM_cas:issn">0021-9584</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society and Division of Chemical Education, Inc.</span>)
        </div><div class="casAbstract">ChemSpider is a free, online chem. database offering access to phys. and chem. properties, mol. structure, spectral data, synthetic methods, safety information, and nomenclature for almost 25 million unique chem. compds. sourced and linked to almost 400 sep. data sources on the Web.  ChemSpider is quickly becoming the primary chem. Internet portal and it can be very useful for both chem. teaching and research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVUvg_k0Vg6bVg90H21EOLACvtfcHk0lghj_DSIYYs5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtV2jtr%252FF&md5=c7c75704f57cf699598c0250001ebb48</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fed100697w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fed100697w%26sid%3Dliteratum%253Aachs%26aulast%3DPence%26aufirst%3DH.%2BE.%26aulast%3DWilliams%26aufirst%3DA.%26atitle%3DChemSpider%253A%2520An%2520Online%2520Chemical%2520Information%2520Resource%26jtitle%3DJ.%2520Chem.%2520Educ.%26date%3D2010%26volume%3D87%26spage%3D1123%26epage%3D1124%26doi%3D10.1021%2Fed100697w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fourches, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratov, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tropsha, A.</span></span> <span> </span><span class="NLM_article-title">Trust, but Verify: On the Importance of Chemical Structure Curation in Cheminformatics and QSAR Modeling Research</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1189</span>– <span class="NLM_lpage">1204</span>, <span class="refDoi"> DOI: 10.1021/ci100176x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100176x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvVeitLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1189-1204&author=D.+Fourchesauthor=E.+Muratovauthor=A.+Tropsha&title=Trust%2C+but+Verify%3A+On+the+Importance+of+Chemical+Structure+Curation+in+Cheminformatics+and+QSAR+Modeling+Research&doi=10.1021%2Fci100176x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Trust, But Verify: On the Importance of Chemical Structure Curation in Cheminformatics and QSAR Modeling Research</span></div><div class="casAuthors">Fourches, Denis; Muratov, Eugene; Tropsha, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1189-1204</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">With the recent advent of high-throughput technologies for both compd. synthesis and biol. screening, there is no shortage of publicly or com. available data sets and databases that can be used for computational drug discovery applications.  Rapid growth of large, publicly available databases (such as PubChem or ChemSpider contg. more than 20 million mol. records each) enabled by exptl. projects such as NIH's Mol. Libraries and Imaging Initiative provides new opportunities for the development of chemoinformatics methodologies and their application to knowledge discovery in mol. databases.  A fundamental assumption of any chemoinformatics study is the correctness of the input data generated by exptl. scientists and available in various data sets.  In another recent study, the authors investigated several public and com. databases to calc. their error rates: the latter were ranging from 0.1 to 3.4% depending on the database.  How significant is the problem of accurate structure representation (given that the error rates in current databases may appear relatively low) since it concerns exploratory chemoinformatics and mol. modeling research.  Recent investigations by a large group of collaborators from six labs. have clearly demonstrated that the type of chem. descriptors has much greater influence on the prediction performance of QSAR models than the nature of model optimization techniques.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqpd35PIjh-7Vg90H21EOLACvtfcHk0ljizLkDl4-EDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvVeitLk%253D&md5=4c19d67a67094bc595f5940157ff9a2d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fci100176x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100176x%26sid%3Dliteratum%253Aachs%26aulast%3DFourches%26aufirst%3DD.%26aulast%3DMuratov%26aufirst%3DE.%26aulast%3DTropsha%26aufirst%3DA.%26atitle%3DTrust%252C%2520but%2520Verify%253A%2520On%2520the%2520Importance%2520of%2520Chemical%2520Structure%2520Curation%2520in%2520Cheminformatics%2520and%2520QSAR%2520Modeling%2520Research%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D1189%26epage%3D1204%26doi%3D10.1021%2Fci100176x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fourches, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratov, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tropsha, A.</span></span> <span> </span><span class="NLM_article-title">Curation of Chemogenomics Data</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">535</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1881</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1038%2Fnchembio.1881" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=26196763" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=535&author=D.+Fourchesauthor=E.+Muratovauthor=A.+Tropsha&title=Curation+of+Chemogenomics+Data&doi=10.1038%2Fnchembio.1881"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Curation of chemogenomics data</span></div><div class="casAuthors">Fourches, Denis; Muratov, Eugene; Tropsha, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">535</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">With the rapid accumulation of data in all areas of chem. biol. research, scientists rely increasingly on historical chemogenomics data and computational models to guide small mol. bioactivity screens and chem. probe development.  However, there is a growing public concern about the frequent irreproducibility of exptl. data reported in peer-reviewed scientific publications.  As one means of addressing the data quality problem, we propose a general chem. and biol. data curation workflow that relies on existing cheminformatics approaches to flag, and in some cases correct, possibly erroneous entries in large chemogenomics datasets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPsQj89bRYDbVg90H21EOLACvtfcHk0ljizLkDl4-EDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Wlu7zL&md5=c45d4b8fb2f3570696be7818025cc4c7</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1881&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1881%26sid%3Dliteratum%253Aachs%26aulast%3DFourches%26aufirst%3DD.%26aulast%3DMuratov%26aufirst%3DE.%26aulast%3DTropsha%26aufirst%3DA.%26atitle%3DCuration%2520of%2520Chemogenomics%2520Data%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D535%26doi%3D10.1038%2Fnchembio.1881" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rigden, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernández, X. M.</span></span> <span> </span><span class="NLM_article-title">The 26th Annual Nucleic Acids Research Database Issue and Molecular Biology Database Collection</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D1</span>– <span class="NLM_lpage">D7</span>, <span class="refDoi"> DOI: 10.1093/nar/gky1267</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fnar%2Fgky1267" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=30626175" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1GqtrjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D1-D7&issue=D1&author=D.+J.+Rigdenauthor=X.+M.+Fern%C3%A1ndez&title=The+26th+Annual+Nucleic+Acids+Research+Database+Issue+and+Molecular+Biology+Database+Collection&doi=10.1093%2Fnar%2Fgky1267"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The 26th annual nucleic acids research database issue and molecular biology database collection</span></div><div class="casAuthors">Rigden, Daniel J.; Fernandez, Xose M.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1-D7</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  The 2019 Nucleic Acids Research (NAR) Database Issue contains 168 papers spanning mol. biol.  Among them, 64 are new and another 92 are updates describing resources that appeared in the Issue previously.  The remaining 12 are updates on databases most recently published elsewhere.  This Issue contains two Breakthrough articles, on the Virtual Metabolic Human (VMH) database which links human and gut microbiota metab. with diet and disease, and Vibrism DB, a database of mouse brain anatomy and gene (co-)expression with sophisticated visualization and session sharing.  Major returning nucleic acid databases include RNAcentral, miRBase and LncRNA2Target.  Protein sequence databases include UniProtKB, InterPro and Pfam, while wwPDB and RCSB cover protein structure.  STRING and KEGG update in the section on metab. and pathways.  Microbial genomes are covered by IMG/M and resources for human and model organism genomics include Ensembl, UCSC Genome Browser, GENCODE and Flybase.  Genomic variation and disease are well-covered by GWAS Catalog, PopHumanScan, OMIM and COSMIC, CADD being another major newcomer.  Major new proteomics resources reporting here include iProX and jPOSTdb.  The entire database issue is freely available online on the NAR website.  The NAR online Mol. Biol. Database Collection has been updated, reviewing 506 entries, adding 66 new resources and eliminating 147 discontinued URLs, bringing the current total to 1613 databases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoc8TUMtAaxirVg90H21EOLACvtfcHk0liZmSRUcCG91Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1GqtrjF&md5=696524ae4195a18ef6eb493310d4f828</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky1267&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky1267%26sid%3Dliteratum%253Aachs%26aulast%3DRigden%26aufirst%3DD.%2BJ.%26aulast%3DFern%25C3%25A1ndez%26aufirst%3DX.%2BM.%26atitle%3DThe%252026th%2520Annual%2520Nucleic%2520Acids%2520Research%2520Database%2520Issue%2520and%2520Molecular%2520Biology%2520Database%2520Collection%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2019%26volume%3D47%26issue%3DD1%26spage%3DD1%26epage%3DD7%26doi%3D10.1093%2Fnar%2Fgky1267" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Letunic, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bork, P.</span></span> <span> </span><span class="NLM_article-title">20 Years of the SMART Protein Domain Annotation Resource</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D493</span>– <span class="NLM_lpage">D496</span>, <span class="refDoi"> DOI: 10.1093/nar/gkx922</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fnar%2Fgkx922" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=29040681" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlGju7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D493-D496&issue=D1&author=I.+Letunicauthor=P.+Bork&title=20+Years+of+the+SMART+Protein+Domain+Annotation+Resource&doi=10.1093%2Fnar%2Fgkx922"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">20 years of the SMART protein domain annotation resource</span></div><div class="casAuthors">Letunic, Ivica; Bork, Peer</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D493-D496</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  SMART (Simple Modular Architecture Research Tool) is a web resource for the identification and annotation of protein domains and the anal. of protein domain architectures.  SMART version 8 contains manually curated models for more than 1300 protein domains, with approx. 100 new models added since our last update article (1).  The underlying protein databases were synchronized with UniProt (2), Ensembl (3) and STRING (4), doubling the total no. of annotated domains and other protein features to more than 200 million.  In its 20th year, the SMART anal. results pages have been streamlined again and its information sources have been updated.  SMART's vector based display engine has been extended to all protein schematics in SMART and rewritten to use the latest web technologies.  The internal full text search engine has been redesigned and updated, resulting in greatly increased search speed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouFalEXQtj7LVg90H21EOLACvtfcHk0liZmSRUcCG91Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlGju7rN&md5=13be1a91226804c04b077b22e3b7d622</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx922&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx922%26sid%3Dliteratum%253Aachs%26aulast%3DLetunic%26aufirst%3DI.%26aulast%3DBork%26aufirst%3DP.%26atitle%3D20%2520Years%2520of%2520the%2520SMART%2520Protein%2520Domain%2520Annotation%2520Resource%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26issue%3DD1%26spage%3DD493%26epage%3DD496%26doi%3D10.1093%2Fnar%2Fgkx922" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">The
UniProt Consortium.</span> <span> </span><span class="NLM_article-title">UniProt:
A Worldwide Hub of Protein Knowledge</span>. <i>Nucleic
Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D506</span>– <span class="NLM_lpage">D515</span>, <span class="refDoi"> DOI: 10.1093/nar/gky1049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fnar%2Fgky1049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=30395287" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D506-D515&issue=D1&author=The%0AUniProt+Consortium.&title=UniProt%3A%0AA+Worldwide+Hub+of+Protein+Knowledge&doi=10.1093%2Fnar%2Fgky1049"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky1049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky1049%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DUniProt%253A%250AA%2520Worldwide%2520Hub%2520of%2520Protein%2520Knowledge%26jtitle%3DNucleic%250AAcids%2520Res.%26date%3D2019%26volume%3D47%26issue%3DD1%26spage%3DD506%26epage%3DD515%26doi%3D10.1093%2Fnar%2Fgky1049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pándy-Szekeres, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munk, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsonkov, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mordalski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harpsøe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauser, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bojarski, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gloriam, D. E.</span></span> <span> </span><span class="NLM_article-title">GPCRdb in 2018: Adding GPCR Structure Models and Ligands</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D440</span>– <span class="NLM_lpage">D446</span>, <span class="refDoi"> DOI: 10.1093/nar/gkx1109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fnar%2Fgkx1109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=29155946" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlGisLbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=D440-D446&issue=D1&author=G.+P%C3%A1ndy-Szekeresauthor=C.+Munkauthor=T.+M.+Tsonkovauthor=S.+Mordalskiauthor=K.+Harps%C3%B8eauthor=A.+S.+Hauserauthor=A.+J.+Bojarskiauthor=D.+E.+Gloriam&title=GPCRdb+in+2018%3A+Adding+GPCR+Structure+Models+and+Ligands&doi=10.1093%2Fnar%2Fgkx1109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">GPCRdb in 2018: adding GPCR structure models and ligands</span></div><div class="casAuthors">Pandy-Szekeres, Gaspar; Munk, Christian; Tsonkov, Tsonko M.; Mordalski, Stefan; Harpsoee, Kasper; Hauser, Alexander S.; Bojarski, Andrzej J.; Gloriam, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D440-D446</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">G protein-coupled receptors are the most abundant mediators of both human signaling processes and therapeutic effects.  Herein, we report GPCRomewide homol. models of unprecedented quality, and roughly 150 000 GPCR ligands with data on biol. activities and com. availability.  Based on the strategy of 'Less model - more Xtal', each model exploits both a main template and alternative local templates.  This achieved higher similarity to new structures than any of the existing resources, and refined crystal structures with missing or distorted regions.  Models are provided for inactive, intermediate and active states-except for classes C and F that so far only have inactive templates.  The ligand database has sep. browsers for: (i) target selection by receptor, family or class, (ii) ligand filtering based on cross-expt. activities (min, max and mean) or chem. properties, (iii) ligand source data and (iv) com. availability.  SMILES structures and activity spreadsheets can be downloaded for further processing.  Furthermore, three recent landmark publications on GPCR drugs, G protein selectivity and genetic variants have been accompanied with resources that now let readers view and analyze the findings themselves in GPCRdb.  Altogether, this update will enable scientific investigation for the wider GPCR community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptWfqUUDwXB7Vg90H21EOLACvtfcHk0ljFGaPVh2f7HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlGisLbE&md5=0cf77dd5ea934548a2d3a0ca69991f60</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx1109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx1109%26sid%3Dliteratum%253Aachs%26aulast%3DP%25C3%25A1ndy-Szekeres%26aufirst%3DG.%26aulast%3DMunk%26aufirst%3DC.%26aulast%3DTsonkov%26aufirst%3DT.%2BM.%26aulast%3DMordalski%26aufirst%3DS.%26aulast%3DHarps%25C3%25B8e%26aufirst%3DK.%26aulast%3DHauser%26aufirst%3DA.%2BS.%26aulast%3DBojarski%26aufirst%3DA.%2BJ.%26aulast%3DGloriam%26aufirst%3DD.%2BE.%26atitle%3DGPCRdb%2520in%25202018%253A%2520Adding%2520GPCR%2520Structure%2520Models%2520and%2520Ligands%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26issue%3DD1%26spage%3DD440%26epage%3DD446%26doi%3D10.1093%2Fnar%2Fgkx1109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, D. M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda-Saavedra, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, G. J.</span></span> <span> </span><span class="NLM_article-title">Kinomer v. 1.0: A Database of Systematically Classified Eukaryotic Protein Kinases</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>37</i></span> (<span class="NLM_issue">Suppl. 1</span>),  <span class="NLM_fpage">D244</span>– <span class="NLM_lpage">D250</span>, <span class="refDoi"> DOI: 10.1093/nar/gkn834</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fnar%2Fgkn834" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=18974176" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFejtLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=D244-D250&issue=Suppl.+1&author=D.+M.+A.+Martinauthor=D.+Miranda-Saavedraauthor=G.+J.+Barton&title=Kinomer+v.+1.0%3A+A+Database+of+Systematically+Classified+Eukaryotic+Protein+Kinases&doi=10.1093%2Fnar%2Fgkn834"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Kinomer v. 1.0: a database of systematically classified eukaryotic protein kinases</span></div><div class="casAuthors">Martin, David M. A.; Miranda-Saavedra, Diego; Barton, Geoffrey J.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">Database Iss</span>),
    <span class="NLM_cas:pages">D244-D250</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The regulation of protein function through reversible phosphorylation by protein kinases and phosphatases is a general mechanism controlling virtually every cellular activity.  Eukaryotic protein kinases can be classified into distinct, well-characterized groups based on amino acid sequence similarity and function.  We recently reported a highly sensitive and accurate hidden Markov model-based method for the automatic detection and classification of protein kinases into these specific groups.  The Kinomer v. 1.0 database presented here contains annotated classifications for the protein kinase complements of 43 eukaryotic genomes.  These span the taxonomic range and include fungi (16 species), plants (6), diatoms (1), amoebas (2), protists (1) and animals (17).  The kinomes are stored in a relational database and are accessible through a web interface on the basis of species, kinase group or a combination of both.  In addn., the Kinomer v. 1.0 HMM library is made available for users to perform classification on arbitrary sequences.  The Kinomer v. 1.0 database is a continually updated resource where direct comparison of kinase sequences across kinase groups and across species can give insights into kinase function and evolution.  Kinomer v. 1.0 is available at http://www.compbio.dundee.ac.uk/kinomer/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTZbp12ZxpR7Vg90H21EOLACvtfcHk0ljFGaPVh2f7HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFejtLvI&md5=07e84dc6450c15a04582a8973d0aa930</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkn834&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkn834%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DD.%2BM.%2BA.%26aulast%3DMiranda-Saavedra%26aufirst%3DD.%26aulast%3DBarton%26aufirst%3DG.%2BJ.%26atitle%3DKinomer%2520v.%25201.0%253A%2520A%2520Database%2520of%2520Systematically%2520Classified%2520Eukaryotic%2520Protein%2520Kinases%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2009%26volume%3D37%26issue%3DSuppl.%25201%26spage%3DD244%26epage%3DD250%26doi%3D10.1093%2Fnar%2Fgkn834" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burley, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berman, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhikadiya, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Costanzo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christie, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalenberg, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duarte, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodsell, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guranović, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzenko, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalro, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Namkoong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peisach, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Periskova, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prlić, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Randle, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sala, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sekharan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, Y. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valasatava, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voigt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westbrook, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhuravleva, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zardecki, C.</span></span> <span> </span><span class="NLM_article-title">RCSB Protein Data Bank: Biological Macromolecular Structures Enabling Research and Education in Fundamental Biology, Biomedicine, Biotechnology and Energy</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D464</span>– <span class="NLM_lpage">D474</span>, <span class="refDoi"> DOI: 10.1093/nar/gky1004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fnar%2Fgky1004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=30357411" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1Cgt7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D464-D474&issue=D1&author=S.+K.+Burleyauthor=H.+M.+Bermanauthor=C.+Bhikadiyaauthor=C.+Biauthor=L.+Chenauthor=L.+Di+Costanzoauthor=C.+Christieauthor=K.+Dalenbergauthor=J.+M.+Duarteauthor=S.+Duttaauthor=Z.+Fengauthor=S.+Ghoshauthor=D.+S.+Goodsellauthor=R.+K.+Greenauthor=V.+Guranovi%C4%87author=D.+Guzenkoauthor=B.+P.+Hudsonauthor=T.+Kalroauthor=Y.+Liangauthor=R.+Loweauthor=H.+Namkoongauthor=E.+Peisachauthor=I.+Periskovaauthor=A.+Prli%C4%87author=C.+Randleauthor=A.+Roseauthor=P.+Roseauthor=R.+Salaauthor=M.+Sekharanauthor=C.+Shaoauthor=L.+Tanauthor=Y.+P.+Taoauthor=Y.+Valasatavaauthor=M.+Voigtauthor=J.+Westbrookauthor=J.+Wooauthor=H.+Yangauthor=J.+Youngauthor=M.+Zhuravlevaauthor=C.+Zardecki&title=RCSB+Protein+Data+Bank%3A+Biological+Macromolecular+Structures+Enabling+Research+and+Education+in+Fundamental+Biology%2C+Biomedicine%2C+Biotechnology+and+Energy&doi=10.1093%2Fnar%2Fgky1004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">RCSB Protein Data Bank: biological macromolecular structures enabling research and education in fundamental biology, biomedicine, biotechnology and energy</span></div><div class="casAuthors">Burley, Stephen K.; Berman, Helen M.; Bhikadiya, Charmi; Bi, Chunxiao; Chen, Li; Di Costanzo, Luigi; Christie, Cole; Dalenberg, Ken; Duarte, Jose M.; Dutta, Shuchismita; Feng, Zukang; Ghosh, Sutapa; Goodsell, David S.; Green, Rachel K.; Guranovic, Vladimir; Guzenko, Dmytro; Hudson, Brian P.; Kalro, Tara; Liang, Yuhe; Lowe, Robert; Namkoong, Harry; Peisach, Ezra; Periskova, Irina; Prlic, Andreas; Randle, Chris; Rose, Alexander; Rose, Peter; Sala, Raul; Sekharan, Monica; Shao, Chenghua; Tan, Lihua; Tao, Yi-Ping; Valasatava, Yana; Voigt, Maria; Westbrook, John; Woo, Jesse; Yang, Huanwang; Young, Jasmine; Zhuravleva, Marina; Zardecki, Christine</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D464-D474</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  The Research Collab. for Structural Bioinformatics Protein Data Bank (RCSB PDB, rcsb.org), the US data center for the global PDB archive, serves thousands of Data Depositors in the Americas and Oceania and makes 3D macromol. structure data available at no charge and without usage restrictions to more than 1 million rcsb.org Users worldwide and 600 000 pdb101.rcsb.org education-focused Users around the globe.  PDB Data Depositors include structural biologists using macromol. crystallog., NMR spectroscopy and 3D electron microscopy.  PDB Data Consumers include researchers, educators and students studying Fundamental Biol., Biomedicine, Biotechnol. and Energy.  Recent reorganization of RCSB PDB activities into four integrated, interdependent services is described in detail, together with tools and resources added over the past 2 years to RCSB PDB web portals in support of a 'Structural View of Biol.'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqbbw8LsGXrrVg90H21EOLACvtfcHk0lj1bWvGvat6Kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1Cgt7jO&md5=190361b23d7665a66208618c0ba86b5a</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky1004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky1004%26sid%3Dliteratum%253Aachs%26aulast%3DBurley%26aufirst%3DS.%2BK.%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DBhikadiya%26aufirst%3DC.%26aulast%3DBi%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DDi%2BCostanzo%26aufirst%3DL.%26aulast%3DChristie%26aufirst%3DC.%26aulast%3DDalenberg%26aufirst%3DK.%26aulast%3DDuarte%26aufirst%3DJ.%2BM.%26aulast%3DDutta%26aufirst%3DS.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGhosh%26aufirst%3DS.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26aulast%3DGreen%26aufirst%3DR.%2BK.%26aulast%3DGuranovi%25C4%2587%26aufirst%3DV.%26aulast%3DGuzenko%26aufirst%3DD.%26aulast%3DHudson%26aufirst%3DB.%2BP.%26aulast%3DKalro%26aufirst%3DT.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DLowe%26aufirst%3DR.%26aulast%3DNamkoong%26aufirst%3DH.%26aulast%3DPeisach%26aufirst%3DE.%26aulast%3DPeriskova%26aufirst%3DI.%26aulast%3DPrli%25C4%2587%26aufirst%3DA.%26aulast%3DRandle%26aufirst%3DC.%26aulast%3DRose%26aufirst%3DA.%26aulast%3DRose%26aufirst%3DP.%26aulast%3DSala%26aufirst%3DR.%26aulast%3DSekharan%26aufirst%3DM.%26aulast%3DShao%26aufirst%3DC.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DTao%26aufirst%3DY.%2BP.%26aulast%3DValasatava%26aufirst%3DY.%26aulast%3DVoigt%26aufirst%3DM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DWoo%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DYoung%26aufirst%3DJ.%26aulast%3DZhuravleva%26aufirst%3DM.%26aulast%3DZardecki%26aufirst%3DC.%26atitle%3DRCSB%2520Protein%2520Data%2520Bank%253A%2520Biological%2520Macromolecular%2520Structures%2520Enabling%2520Research%2520and%2520Education%2520in%2520Fundamental%2520Biology%252C%2520Biomedicine%252C%2520Biotechnology%2520and%2520Energy%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2019%26volume%3D47%26issue%3DD1%26spage%3DD464%26epage%3DD474%26doi%3D10.1093%2Fnar%2Fgky1004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Szklarczyk, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gable, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Junge, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wyder, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huerta-Cepas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simonovic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doncheva, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bork, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mering, C. V.</span></span> <span> </span><span class="NLM_article-title">STRING V11: Protein–Protein Association Networks with Increased Coverage, Supporting Functional Discovery in Genome-Wide Experimental Datasets</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D607</span>– <span class="NLM_lpage">D613</span>, <span class="refDoi"> DOI: 10.1093/nar/gky1131</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fnar%2Fgky1131" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=30476243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1Gqtr%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D607-D613&issue=D1&author=D.+Szklarczykauthor=A.+L.+Gableauthor=D.+Lyonauthor=A.+Jungeauthor=S.+Wyderauthor=J.+Huerta-Cepasauthor=M.+Simonovicauthor=N.+T.+Donchevaauthor=J.+H.+Morrisauthor=P.+Borkauthor=L.+J.+Jensenauthor=C.+V.+Mering&title=STRING+V11%3A+Protein%E2%80%93Protein+Association+Networks+with+Increased+Coverage%2C+Supporting+Functional+Discovery+in+Genome-Wide+Experimental+Datasets&doi=10.1093%2Fnar%2Fgky1131"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">STRING v11: protein-protein association networks with increased coverage supporting functional discovery in genome-wide experimental datasets</span></div><div class="casAuthors">Szklarczyk, Damian; Gable, Annika L.; Lyon, David; Junge, Alexander; Wyder, Stefan; Huerta-Cepas, Jaime; Simonovic, Milan; Doncheva, Nadezhda T.; Morris, John H.; Bork, Peer; Jensen, Lars J.; von Mering, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D607-D613</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Proteins and their functional interactions form the backbone of the cellular machinery.  Their connectivity network needs to be considered for the full understanding of biol. phenomena, but the available information on protein-protein assocns. is incomplete and exhibits varying levels of annotation granularity and reliability.  The STRING database aims to collect, score and integrate all publicly available sources of protein-protein interaction information, and to complement these with computational predictions.  Its goal is to achieve a comprehensive and objective global network, including direct (phys.) as well as indirect (functional) interactions.  The latest version of STRING (11.0) more than doubles the no. of organisms it covers, to 5090.  The most important new feature is an option to upload entire, genome-wide datasets as input, allowing users to visualize subsets as interaction networks and to perform gene-set enrichment anal. on the entire input.  For the enrichment anal., STRING implements well-known classification systems such as Gene Ontol. and KEGG, but also offers addnl., new classification systems based on high-throughput text-mining as well as on a hierarchical clustering of the assocn. network itself.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7UX25KcxhhbVg90H21EOLACvtfcHk0lj1bWvGvat6Kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1Gqtr%252FM&md5=43df05a2de505f1df210fb4aaa200f58</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky1131&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky1131%26sid%3Dliteratum%253Aachs%26aulast%3DSzklarczyk%26aufirst%3DD.%26aulast%3DGable%26aufirst%3DA.%2BL.%26aulast%3DLyon%26aufirst%3DD.%26aulast%3DJunge%26aufirst%3DA.%26aulast%3DWyder%26aufirst%3DS.%26aulast%3DHuerta-Cepas%26aufirst%3DJ.%26aulast%3DSimonovic%26aufirst%3DM.%26aulast%3DDoncheva%26aufirst%3DN.%2BT.%26aulast%3DMorris%26aufirst%3DJ.%2BH.%26aulast%3DBork%26aufirst%3DP.%26aulast%3DJensen%26aufirst%3DL.%2BJ.%26aulast%3DMering%26aufirst%3DC.%2BV.%26atitle%3DSTRING%2520V11%253A%2520Protein%25E2%2580%2593Protein%2520Association%2520Networks%2520with%2520Increased%2520Coverage%252C%2520Supporting%2520Functional%2520Discovery%2520in%2520Genome-Wide%2520Experimental%2520Datasets%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2019%26volume%3D47%26issue%3DD1%26spage%3DD607%26epage%3DD613%26doi%3D10.1093%2Fnar%2Fgky1131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jassal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viteri, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorente, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fabregat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sidiropoulos, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gillespie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haw, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loney, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">May, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milacic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothfels, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sevilla, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shamovsky, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shorser, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varusai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiser, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stein, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermjakob, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Eustachio, P.</span></span> <span> </span><span class="NLM_article-title">The Reactome Pathway Knowledgebase</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>48</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D498</span>– <span class="NLM_lpage">D503</span>, <span class="refDoi"> DOI: 10.1093/nar/gkz1031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fnar%2Fgkz1031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=31691815" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1GltrjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2020&pages=D498-D503&issue=D1&author=B.+Jassalauthor=L.+Matthewsauthor=G.+Viteriauthor=C.+Gongauthor=P.+Lorenteauthor=A.+Fabregatauthor=K.+Sidiropoulosauthor=J.+Cookauthor=M.+Gillespieauthor=R.+Hawauthor=F.+Loneyauthor=B.+Mayauthor=M.+Milacicauthor=K.+Rothfelsauthor=C.+Sevillaauthor=V.+Shamovskyauthor=S.+Shorserauthor=T.+Varusaiauthor=J.+Weiserauthor=G.+Wuauthor=L.+Steinauthor=H.+Hermjakobauthor=P.+D%E2%80%99Eustachio&title=The+Reactome+Pathway+Knowledgebase&doi=10.1093%2Fnar%2Fgkz1031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The reactome pathway knowledgebase</span></div><div class="casAuthors">Jassal, Bijay; Matthews, Lisa; Viteri, Guilherme; Gong, Chuqiao; Lorente, Pascual; Fabregat, Antonio; Sidiropoulos, Konstantinos; Cook, Justin; Gillespie, Marc; Haw, Robin; Loney, Fred; May, Bruce; Milacic, Marija; Rothfels, Karen; Sevilla, Cristoffer; Shamovsky, Veronica; Shorser, Solomon; Varusai, Thawfeek; Weiser, Joel; Wu, Guanming; Stein, Lincoln; Hermjakob, Henning; D'Eustachio, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D498-D503</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  The Reactome Knowledgebase provides mol. details of signal transduction, transport, DNA replication, metab. and other cellular processes as an ordered network of mol. transformations in a single consistent data model, an extended version of a classic metabolic map.  Reactome functions both as an archive of biol. processes and as a tool for discovering functional relationships in data such as gene expression profiles or somatic mutation catalogs from tumor cells.  To extend our ability to annotate human disease processes, we have implemented a new drug class and have used it initially to annotate drugs relevant to cardiovascular disease.  Our annotation model depends on external domain experts to identify new areas for annotation and to review new content.  New web pages facilitate recruitment of community experts and allow those who have contributed to Reactome to identify their contributions and link them to their ORCID records.  To improve visualization of our content, we have implemented a new tool to automatically lay out the components of individual reactions with multiple options for downloading the reaction diagrams and assocd. data, and a new display of our event hierarchy that will facilitate visual interpretation of pathway anal. results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8-pLjgJKfiLVg90H21EOLACvtfcHk0lj1bWvGvat6Kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1GltrjP&md5=0ba27eea4957e270b8a16c0159ffcd5d</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkz1031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkz1031%26sid%3Dliteratum%253Aachs%26aulast%3DJassal%26aufirst%3DB.%26aulast%3DMatthews%26aufirst%3DL.%26aulast%3DViteri%26aufirst%3DG.%26aulast%3DGong%26aufirst%3DC.%26aulast%3DLorente%26aufirst%3DP.%26aulast%3DFabregat%26aufirst%3DA.%26aulast%3DSidiropoulos%26aufirst%3DK.%26aulast%3DCook%26aufirst%3DJ.%26aulast%3DGillespie%26aufirst%3DM.%26aulast%3DHaw%26aufirst%3DR.%26aulast%3DLoney%26aufirst%3DF.%26aulast%3DMay%26aufirst%3DB.%26aulast%3DMilacic%26aufirst%3DM.%26aulast%3DRothfels%26aufirst%3DK.%26aulast%3DSevilla%26aufirst%3DC.%26aulast%3DShamovsky%26aufirst%3DV.%26aulast%3DShorser%26aufirst%3DS.%26aulast%3DVarusai%26aufirst%3DT.%26aulast%3DWeiser%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DG.%26aulast%3DStein%26aufirst%3DL.%26aulast%3DHermjakob%26aufirst%3DH.%26aulast%3DD%25E2%2580%2599Eustachio%26aufirst%3DP.%26atitle%3DThe%2520Reactome%2520Pathway%2520Knowledgebase%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2020%26volume%3D48%26issue%3DD1%26spage%3DD498%26epage%3DD503%26doi%3D10.1093%2Fnar%2Fgkz1031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Piñero, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bravo, À.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Queralt-Rosinach, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutiérrez-Sacristán, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deu-Pons, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centeno, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">García-García, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanz, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furlong, L. I.</span></span> <span> </span><span class="NLM_article-title">DisGeNET: A Comprehensive Platform Integrating Information on Human Disease-Associated Genes and Variants</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D833</span>– <span class="NLM_lpage">D839</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw943</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fnar%2Fgkw943" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=27924018" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslWhtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D833-D839&issue=D1&author=J.+Pi%C3%B1eroauthor=%C3%80.+Bravoauthor=N.+Queralt-Rosinachauthor=A.+Guti%C3%A9rrez-Sacrist%C3%A1nauthor=J.+Deu-Ponsauthor=E.+Centenoauthor=J.+Garc%C3%ADa-Garc%C3%ADaauthor=F.+Sanzauthor=L.+I.+Furlong&title=DisGeNET%3A+A+Comprehensive+Platform+Integrating+Information+on+Human+Disease-Associated+Genes+and+Variants&doi=10.1093%2Fnar%2Fgkw943"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">DisGeNET: a comprehensive platform integrating information on human disease-associated genes and variants</span></div><div class="casAuthors">Pinero, Janet; Bravo, Alex; Queralt-Rosinach, Nuria; Gutierrez-Sacristan, Alba; Deu-Pons, Jordi; Centeno, Emilio; Garcia-Garcia, Javier; Sanz, Ferran; Furlong, Laura I.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D833-D839</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The information about the genetic basis of human diseases lies at the heart of precision medicine and drug discovery.  However, to realize its full potential to support these goals, several problems, such as fragmentation, heterogeneity, availability and different conceptualization of the data must be overcome.  To provide the community with a resource free of these hurdles, we have developed DisGeNET (http://www.disgenet.org), one of the largest available collections of genes and variants involved in human diseases.  DisGeNET integrates data from expert curated repositories, GWAS catalogs, animal models and the scientific literature.  DisGeNET data are homogeneously annotated with controlled vocabularies and community-driven ontologies.  Addnl., several original metrics are provided to assist the prioritization of genotype-phenotype relationships.  The information is accessible through a web interface, a Cytoscape App, an RDF SPARQL endpoint, scripts in several programming languages and an R package.  DisGeNET is a versatile platform that can be used for different research purposes including the investigation of the mol. underpinnings of specific human diseases and their comorbidities, the anal. of the properties of disease genes, the generation of hypothesis on drug therapeutic action and drug adverse effects, the validation of computationally predicted disease genes and the evaluation of text-mining methods performance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqve0GG5MK1G7Vg90H21EOLACvtfcHk0li2CDz8JzEagg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslWhtL4%253D&md5=b00a79e146f48ddd5d67e85597ac89c2</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw943&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw943%26sid%3Dliteratum%253Aachs%26aulast%3DPi%25C3%25B1ero%26aufirst%3DJ.%26aulast%3DBravo%26aufirst%3D%25C3%2580.%26aulast%3DQueralt-Rosinach%26aufirst%3DN.%26aulast%3DGuti%25C3%25A9rrez-Sacrist%25C3%25A1n%26aufirst%3DA.%26aulast%3DDeu-Pons%26aufirst%3DJ.%26aulast%3DCenteno%26aufirst%3DE.%26aulast%3DGarc%25C3%25ADa-Garc%25C3%25ADa%26aufirst%3DJ.%26aulast%3DSanz%26aufirst%3DF.%26aulast%3DFurlong%26aufirst%3DL.%2BI.%26atitle%3DDisGeNET%253A%2520A%2520Comprehensive%2520Platform%2520Integrating%2520Information%2520on%2520Human%2520Disease-Associated%2520Genes%2520and%2520Variants%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26issue%3DD1%26spage%3DD833%26epage%3DD839%26doi%3D10.1093%2Fnar%2Fgkw943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corsello, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peck, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natoli, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gould, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tubelli, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asiedu, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lahr, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirschman, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donahue, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Julian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wadden, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liberzon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toder, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orzechowski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enache, O. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piccioni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyons, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shamji, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brooks, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vrcic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flynn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosains, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeda, D. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davison, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ardlie, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hogstrom, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenside, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clemons, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silver, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read-Button, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haggarty, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ronco, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehm, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doench, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bittker, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Root, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golub, T. R.</span></span> <span> </span><span class="NLM_article-title">A Next Generation Connectivity Map: L1000 Platform and the First 1,000,000 Profiles</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>171</i></span>,  <span class="NLM_fpage">1437</span>– <span class="NLM_lpage">1452</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2017.10.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1016%2Fj.cell.2017.10.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=29195078" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFWmsbrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=171&publication_year=2017&pages=1437-1452&author=A.+Subramanianauthor=R.+Narayanauthor=S.+M.+Corselloauthor=D.+D.+Peckauthor=T.+E.+Natoliauthor=X.+Luauthor=J.+Gouldauthor=J.+F.+Davisauthor=A.+A.+Tubelliauthor=J.+K.+Asieduauthor=D.+L.+Lahrauthor=J.+E.+Hirschmanauthor=Z.+Liuauthor=M.+Donahueauthor=B.+Julianauthor=M.+Khanauthor=D.+Waddenauthor=I.+C.+Smithauthor=D.+Lamauthor=A.+Liberzonauthor=C.+Toderauthor=M.+Bagulauthor=M.+Orzechowskiauthor=O.+M.+Enacheauthor=F.+Piccioniauthor=S.+A.+Johnsonauthor=N.+J.+Lyonsauthor=A.+H.+Bergerauthor=A.+F.+Shamjiauthor=A.+N.+Brooksauthor=A.+Vrcicauthor=C.+Flynnauthor=J.+Rosainsauthor=D.+Y.+Takedaauthor=R.+Huauthor=D.+Davisonauthor=J.+Lambauthor=K.+Ardlieauthor=L.+Hogstromauthor=P.+Greensideauthor=N.+S.+Grayauthor=P.+A.+Clemonsauthor=S.+Silverauthor=X.+Wuauthor=W.+N.+Zhaoauthor=W.+Read-Buttonauthor=X.+Wuauthor=S.+J.+Haggartyauthor=L.+V.+Roncoauthor=J.+S.+Boehmauthor=S.+L.+Schreiberauthor=J.+G.+Doenchauthor=J.+A.+Bittkerauthor=D.+E.+Rootauthor=B.+Wongauthor=T.+R.+Golub&title=A+Next+Generation+Connectivity+Map%3A+L1000+Platform+and+the+First+1%2C000%2C000+Profiles&doi=10.1016%2Fj.cell.2017.10.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">A next generation connectivity map: L1000 platform and the first 1,000,000 profiles</span></div><div class="casAuthors">Subramanian, Aravind; Narayan, Rajiv; Corsello, Steven M.; Peck, David D.; Natoli, Ted E.; Lu, Xiaodong; Gould, Joshua; Davis, John F.; Tubelli, Andrew A.; Asiedu, Jacob K.; Lahr, David L.; Hirschman, Jodi E.; Liu, Zihan; Donahue, Melanie; Julian, Bina; Khan, Mariya; Wadden, David; Smith, Ian C.; Lam, Daniel; Liberzon, Arthur; Toder, Courtney; Bagul, Mukta; Orzechowski, Marek; Enache, Oana M.; Piccioni, Federica; Johnson, Sarah A.; Lyons, Nicholas J.; Berger, Alice H.; Shamji, Alykhan F.; Brooks, Angela N.; Vrcic, Anita; Flynn, Corey; Rosains, Jacqueline; Takeda, David Y.; Hu, Roger; Davison, Desiree; Lamb, Justin; Ardlie, Kristin; Hogstrom, Larson; Greenside, Peyton; Gray, Nathanael S.; Clemons, Paul A.; Silver, Serena; Wu, Xiaoyun; Zhao, Wen-Ning; Read-Button, Willis; Wu, Xiaohua; Haggarty, Stephen J.; Ronco, Lucienne V.; Boehm, Jesse S.; Schreiber, Stuart L.; Doench, John G.; Bittker, Joshua A.; Root, David E.; Wong, Bang; Golub, Todd R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">171</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1437-1452.e17</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">We previously piloted the concept of a Connectivity Map (CMap), whereby genes, drugs, and disease states are connected by virtue of common gene-expression signatures.  Here, we report more than a 1000-fold scale-up of the CMap as part of the NIH LINCS Consortium, made possible by a new, low-cost, high-throughput reduced representation expression profiling method that we term L1000.  We show that L1000 is highly reproducible, comparable to RNA sequencing, and suitable for computational inference of the expression levels of 81% of non-measured transcripts.  We further show that the expanded CMap can be used to discover mechanism of action of small mols., functionally annotate genetic variants of disease genes, and inform clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIWrWy5IxY-rVg90H21EOLACvtfcHk0lgDFotw4oNiJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFWmsbrE&md5=d2b13f2b80fe94021ac03f6a8c859a4a</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.10.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.10.049%26sid%3Dliteratum%253Aachs%26aulast%3DSubramanian%26aufirst%3DA.%26aulast%3DNarayan%26aufirst%3DR.%26aulast%3DCorsello%26aufirst%3DS.%2BM.%26aulast%3DPeck%26aufirst%3DD.%2BD.%26aulast%3DNatoli%26aufirst%3DT.%2BE.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DGould%26aufirst%3DJ.%26aulast%3DDavis%26aufirst%3DJ.%2BF.%26aulast%3DTubelli%26aufirst%3DA.%2BA.%26aulast%3DAsiedu%26aufirst%3DJ.%2BK.%26aulast%3DLahr%26aufirst%3DD.%2BL.%26aulast%3DHirschman%26aufirst%3DJ.%2BE.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DDonahue%26aufirst%3DM.%26aulast%3DJulian%26aufirst%3DB.%26aulast%3DKhan%26aufirst%3DM.%26aulast%3DWadden%26aufirst%3DD.%26aulast%3DSmith%26aufirst%3DI.%2BC.%26aulast%3DLam%26aufirst%3DD.%26aulast%3DLiberzon%26aufirst%3DA.%26aulast%3DToder%26aufirst%3DC.%26aulast%3DBagul%26aufirst%3DM.%26aulast%3DOrzechowski%26aufirst%3DM.%26aulast%3DEnache%26aufirst%3DO.%2BM.%26aulast%3DPiccioni%26aufirst%3DF.%26aulast%3DJohnson%26aufirst%3DS.%2BA.%26aulast%3DLyons%26aufirst%3DN.%2BJ.%26aulast%3DBerger%26aufirst%3DA.%2BH.%26aulast%3DShamji%26aufirst%3DA.%2BF.%26aulast%3DBrooks%26aufirst%3DA.%2BN.%26aulast%3DVrcic%26aufirst%3DA.%26aulast%3DFlynn%26aufirst%3DC.%26aulast%3DRosains%26aufirst%3DJ.%26aulast%3DTakeda%26aufirst%3DD.%2BY.%26aulast%3DHu%26aufirst%3DR.%26aulast%3DDavison%26aufirst%3DD.%26aulast%3DLamb%26aufirst%3DJ.%26aulast%3DArdlie%26aufirst%3DK.%26aulast%3DHogstrom%26aufirst%3DL.%26aulast%3DGreenside%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DClemons%26aufirst%3DP.%2BA.%26aulast%3DSilver%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DW.%2BN.%26aulast%3DRead-Button%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DHaggarty%26aufirst%3DS.%2BJ.%26aulast%3DRonco%26aufirst%3DL.%2BV.%26aulast%3DBoehm%26aufirst%3DJ.%2BS.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DDoench%26aufirst%3DJ.%2BG.%26aulast%3DBittker%26aufirst%3DJ.%2BA.%26aulast%3DRoot%26aufirst%3DD.%2BE.%26aulast%3DWong%26aufirst%3DB.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26atitle%3DA%2520Next%2520Generation%2520Connectivity%2520Map%253A%2520L1000%2520Platform%2520and%2520the%2520First%25201%252C000%252C000%2520Profiles%26jtitle%3DCell%26date%3D2017%26volume%3D171%26spage%3D1437%26epage%3D1452%26doi%3D10.1016%2Fj.cell.2017.10.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stathias, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koleti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidovic, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fazel-Najafabadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilarczyk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terryn, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Umeano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, D. J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lachmann, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evangelista, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma’ayan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medvedovic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schürer, S. C.</span></span> <span> </span><span class="NLM_article-title">LINCS Data Portal 2.0: Next Generation Access Point for Perturbation-Response Signatures</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>48</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D431</span>– <span class="NLM_lpage">D439</span>, <span class="refDoi"> DOI: 10.1093/nar/gkz1023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fnar%2Fgkz1023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=31701147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1GltrbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2020&pages=D431-D439&issue=D1&author=V.+Stathiasauthor=J.+Turnerauthor=A.+Koletiauthor=D.+Vidovicauthor=D.+Cooperauthor=M.+Fazel-Najafabadiauthor=M.+Pilarczykauthor=R.+Terrynauthor=C.+Chungauthor=A.+Umeanoauthor=D.+J.+B.+Clarkeauthor=A.+Lachmannauthor=J.+E.+Evangelistaauthor=A.+Ma%E2%80%99ayanauthor=M.+Medvedovicauthor=S.+C.+Sch%C3%BCrer&title=LINCS+Data+Portal+2.0%3A+Next+Generation+Access+Point+for+Perturbation-Response+Signatures&doi=10.1093%2Fnar%2Fgkz1023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">LINCS Data Portal 2.0: next generation access point for perturbation-response signatures</span></div><div class="casAuthors">Stathias, Vasileios; Turner, John; Koleti, Amar; Vidovic, Dusica; Cooper, Daniel; Fazel-Najafabadi, Mehdi; Pilarczyk, Marcin; Terryn, Raymond; Chung, Caty; Umeano, Afoma; Clarke, Daniel J. B.; Lachmann, Alexander; Evangelista, John Erol; Ma'ayan, Avi; Medvedovic, Mario; Schurer, Stephan C.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D431-D439</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  The Library of Integrated Network-Based Cellular Signatures (LINCS) is an NIH Common Fund program with the goal of generating a large-scale and comprehensive catalog of perturbation-response signatures by utilizing a diverse collection of perturbations across many model systems and assay types.  The LINCS Data Portal (LDP) has been the primary access point for the compendium of LINCS data and has been widely utilized.  Here, we report the first major update of LDP with substantial changes in the data architecture and APIs, a completely redesigned user interface, and enhanced curated metadata annotations to support more advanced, intuitive and deeper querying, exploration and anal. capabilities.  The cornerstone of this update has been the decision to reprocess all high-level LINCS datasets and make them accessible at the data point level enabling users to directly access and download any subset of signatures across the entire library independent from the originating source, project or assay.  Access to the individual signatures also enables the newly implemented signature search functionality, which utilizes the iLINCS platform to identify conditions that mimic or reverse gene set queries.  A newly designed query interface enables global metadata search with autosuggest across all annotations assocd. with perturbations, model systems, and signatures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzW9FiCBXXC7Vg90H21EOLACvtfcHk0lgDFotw4oNiJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1GltrbE&md5=2c39a40d51764dd9a68ce181923fd87a</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkz1023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkz1023%26sid%3Dliteratum%253Aachs%26aulast%3DStathias%26aufirst%3DV.%26aulast%3DTurner%26aufirst%3DJ.%26aulast%3DKoleti%26aufirst%3DA.%26aulast%3DVidovic%26aufirst%3DD.%26aulast%3DCooper%26aufirst%3DD.%26aulast%3DFazel-Najafabadi%26aufirst%3DM.%26aulast%3DPilarczyk%26aufirst%3DM.%26aulast%3DTerryn%26aufirst%3DR.%26aulast%3DChung%26aufirst%3DC.%26aulast%3DUmeano%26aufirst%3DA.%26aulast%3DClarke%26aufirst%3DD.%2BJ.%2BB.%26aulast%3DLachmann%26aufirst%3DA.%26aulast%3DEvangelista%26aufirst%3DJ.%2BE.%26aulast%3DMa%25E2%2580%2599ayan%26aufirst%3DA.%26aulast%3DMedvedovic%26aufirst%3DM.%26aulast%3DSch%25C3%25BCrer%26aufirst%3DS.%2BC.%26atitle%3DLINCS%2520Data%2520Portal%25202.0%253A%2520Next%2520Generation%2520Access%2520Point%2520for%2520Perturbation-Response%2520Signatures%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2020%26volume%3D48%26issue%3DD1%26spage%3DD431%26epage%3DD439%26doi%3D10.1093%2Fnar%2Fgkz1023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dash, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shakyawar, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaushik, S.</span></span> <span> </span><span class="NLM_article-title">Big Data in Healthcare: Management, Analysis and Future Prospects</span>. <i>J. Big Data</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">54</span>, <span class="refDoi"> DOI: 10.1186/s40537-019-0217-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1186%2Fs40537-019-0217-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2019&pages=54&author=S.+Dashauthor=S.+K.+Shakyawarauthor=M.+Sharmaauthor=S.+Kaushik&title=Big+Data+in+Healthcare%3A+Management%2C+Analysis+and+Future+Prospects&doi=10.1186%2Fs40537-019-0217-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1186%2Fs40537-019-0217-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs40537-019-0217-0%26sid%3Dliteratum%253Aachs%26aulast%3DDash%26aufirst%3DS.%26aulast%3DShakyawar%26aufirst%3DS.%2BK.%26aulast%3DSharma%26aufirst%3DM.%26aulast%3DKaushik%26aufirst%3DS.%26atitle%3DBig%2520Data%2520in%2520Healthcare%253A%2520Management%252C%2520Analysis%2520and%2520Future%2520Prospects%26jtitle%3DJ.%2520Big%2520Data%26date%3D2019%26volume%3D6%26spage%3D54%26doi%3D10.1186%2Fs40537-019-0217-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Häyrinen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saranto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nykänen, P.</span></span> <span> </span><span class="NLM_article-title">Definition, Structure, Content, Use and Impacts of Electronic Health Records: A Review of the Research Literature</span>. <i>Int. J. Med. Inform.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">291</span>– <span class="NLM_lpage">304</span>, <span class="refDoi"> DOI: 10.1016/j.ijmedinf.2007.09.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1016%2Fj.ijmedinf.2007.09.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=17951106" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A280%3ADC%252BD1c3itVKqtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2008&pages=291-304&author=K.+H%C3%A4yrinenauthor=K.+Sarantoauthor=P.+Nyk%C3%A4nen&title=Definition%2C+Structure%2C+Content%2C+Use+and+Impacts+of+Electronic+Health+Records%3A+A+Review+of+the+Research+Literature&doi=10.1016%2Fj.ijmedinf.2007.09.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Definition, structure, content, use and impacts of electronic health records: a review of the research literature</span></div><div class="casAuthors">Hayrinen Kristiina; Saranto Kaija; Nykanen Pirkko</div><div class="citationInfo"><span class="NLM_cas:title">International journal of medical informatics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">291-304</span>
        ISSN:<span class="NLM_cas:issn">1386-5056</span>.
    </div><div class="casAbstract">PURPOSE:  This paper reviews the research literature on electronic health record (EHR) systems.  The aim is to find out (1) how electronic health records are defined, (2) how the structure of these records is described, (3) in what contexts EHRs are used, (4) who has access to EHRs, (5) which data components of the EHRs are used and studied, (6) what is the purpose of research in this field, (7) what methods of data collection have been used in the studies reviewed and (8) what are the results of these studies.  METHODS:  A systematic review was carried out of the research dealing with the content of EHRs.  A literature search was conducted on four electronic databases: Pubmed/Medline, Cinalh, Eval and Cochrane.  RESULTS:  The concept of EHR comprised a wide range of information systems, from files compiled in single departments to longitudinal collections of patient data.  Only very few papers offered descriptions of the structure of EHRs or the terminologies used.  EHRs were used in primary, secondary and tertiary care.  Data were recorded in EHRs by different groups of health care professionals.  Secretarial staff also recorded data from dictation or nurses' or physicians' manual notes.  Some information was also recorded by patients themselves; this information is validated by physicians.  It is important that the needs and requirements of different users are taken into account in the future development of information systems.  Several data components were documented in EHRs: daily charting, medication administration, physical assessment, admission nursing note, nursing care plan, referral, present complaint (e.g. symptoms), past medical history, life style, physical examination, diagnoses, tests, procedures, treatment, medication, discharge, history, diaries, problems, findings and immunization.  In the future it will be necessary to incorporate different kinds of standardized instruments, electronic interviews and nursing documentation systems in EHR systems.  The aspects of information quality most often explored in the studies reviewed were the completeness and accuracy of different data components.  It has been shown in several studies that the use of an information system was conducive to more complete and accurate documentation by health care professionals.  The quality of information is particularly important in patient care, but EHRs also provide important information for secondary purposes, such as health policy planning.  CONCLUSION:  Studies focusing on the content of EHRs are needed, especially studies of nursing documentation or patient self-documentation.  One future research area is to compare the documentation of different health care professionals with the core information about EHRs which has been determined in national health projects.  The challenge for ongoing national health record projects around the world is to take into account all the different types of EHRs and the needs and requirements of different health care professionals and consumers in the development of EHRs.  A further challenge is the use of international terminologies in order to achieve semantic interoperability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQWpV3pW8yFvA20zIbnCCI3fW6udTcc2eZSA2M2N-yW67ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c3itVKqtg%253D%253D&md5=1ed67df7fff261a430e3cd0bc9dbeb5d</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.ijmedinf.2007.09.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ijmedinf.2007.09.001%26sid%3Dliteratum%253Aachs%26aulast%3DH%25C3%25A4yrinen%26aufirst%3DK.%26aulast%3DSaranto%26aufirst%3DK.%26aulast%3DNyk%25C3%25A4nen%26aufirst%3DP.%26atitle%3DDefinition%252C%2520Structure%252C%2520Content%252C%2520Use%2520and%2520Impacts%2520of%2520Electronic%2520Health%2520Records%253A%2520A%2520Review%2520of%2520the%2520Research%2520Literature%26jtitle%3DInt.%2520J.%2520Med.%2520Inform.%26date%3D2008%26volume%3D77%26spage%3D291%26epage%3D304%26doi%3D10.1016%2Fj.ijmedinf.2007.09.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Denny, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devaney, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutter, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boerwinkle, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brilliant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butts, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaffins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Channamsetty, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinea, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cotler, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daviglus, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falcón, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funk, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabriel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gebo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rico, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glazer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grand, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenland, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hochfel, B. I.</span></span> <span> </span><span class="NLM_article-title">The “All of Us” Research Program</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>381</i></span>,  <span class="NLM_fpage">668</span>– <span class="NLM_lpage">676</span>, <span class="refDoi"> DOI: 10.1056/NEJMsr1809937</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1056%2FNEJMsr1809937" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=31412182" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A280%3ADC%252BB3MvnvFemsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=381&publication_year=2019&pages=668-676&author=J.+C.+Dennyauthor=S.+Devaneyauthor=J.+L.+Rutterauthor=E.+Boerwinkleauthor=M.+Brilliantauthor=P.+Buttsauthor=M.+Castroauthor=J.+Chaffinsauthor=K.+Changauthor=V.+Channamsettyauthor=C.+Chineaauthor=A.+Cotlerauthor=M.+Daviglusauthor=A.+Falc%C3%B3nauthor=G.+Funkauthor=S.+Gabrielauthor=K.+Geboauthor=E.+G.+Ricoauthor=D.+Glazerauthor=D.+B.+Goldsteinauthor=J.+M.+Grandauthor=P.+Greenlandauthor=K.+Hallauthor=B.+I.+Hochfel&title=The+%E2%80%9CAll+of+Us%E2%80%9D+Research+Program&doi=10.1056%2FNEJMsr1809937"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">The "All of Us" Research Program</span></div><div class="casAuthors">Denny Joshua C; Rutter Joni L; Goldstein David B; Philippakis Anthony; Smoller Jordan W; Jenkins Gwynne; Dishman Eric</div><div class="citationInfo"><span class="NLM_cas:title">The New England journal of medicine</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">381</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">668-676</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Knowledge gained from observational cohort studies has dramatically advanced the prevention and treatment of diseases.  Many of these cohorts, however, are small, lack diversity, or do not provide comprehensive phenotype data.  The All of Us Research Program plans to enroll a diverse group of at least 1 million persons in the United States in order to accelerate biomedical research and improve health.  The program aims to make the research results accessible to participants, and it is developing new approaches to generate, access, and make data broadly available to approved researchers.  All of Us opened for enrollment in May 2018 and currently enrolls participants 18 years of age or older from a network of more than 340 recruitment sites.  Elements of the program protocol include health questionnaires, electronic health records (EHRs), physical measurements, the use of digital health technology, and the collection and analysis of biospecimens.  As of July 2019, more than 175,000 participants had contributed biospecimens.  More than 80% of these participants are from groups that have been historically underrepresented in biomedical research.  EHR data on more than 112,000 participants from 34 sites have been collected.  The All of Us data repository should permit researchers to take into account individual differences in lifestyle, socioeconomic factors, environment, and biologic characteristics in order to advance precision diagnosis, prevention, and treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSktYPjqdU6JpHZzFRCXxVKfW6udTcc2eYHFSkLOszM4bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MvnvFemsA%253D%253D&md5=a756df47643a8eec8c7d9eb2931537b0</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1056%2FNEJMsr1809937&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMsr1809937%26sid%3Dliteratum%253Aachs%26aulast%3DDenny%26aufirst%3DJ.%2BC.%26aulast%3DDevaney%26aufirst%3DS.%26aulast%3DRutter%26aufirst%3DJ.%2BL.%26aulast%3DBoerwinkle%26aufirst%3DE.%26aulast%3DBrilliant%26aufirst%3DM.%26aulast%3DButts%26aufirst%3DP.%26aulast%3DCastro%26aufirst%3DM.%26aulast%3DChaffins%26aufirst%3DJ.%26aulast%3DChang%26aufirst%3DK.%26aulast%3DChannamsetty%26aufirst%3DV.%26aulast%3DChinea%26aufirst%3DC.%26aulast%3DCotler%26aufirst%3DA.%26aulast%3DDaviglus%26aufirst%3DM.%26aulast%3DFalc%25C3%25B3n%26aufirst%3DA.%26aulast%3DFunk%26aufirst%3DG.%26aulast%3DGabriel%26aufirst%3DS.%26aulast%3DGebo%26aufirst%3DK.%26aulast%3DRico%26aufirst%3DE.%2BG.%26aulast%3DGlazer%26aufirst%3DD.%26aulast%3DGoldstein%26aufirst%3DD.%2BB.%26aulast%3DGrand%26aufirst%3DJ.%2BM.%26aulast%3DGreenland%26aufirst%3DP.%26aulast%3DHall%26aufirst%3DK.%26aulast%3DHochfel%26aufirst%3DB.%2BI.%26atitle%3DThe%2520%25E2%2580%259CAll%2520of%2520Us%25E2%2580%259D%2520Research%2520Program%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2019%26volume%3D381%26spage%3D668%26epage%3D676%26doi%3D10.1056%2FNEJMsr1809937" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, P. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaddi, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venugopalan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M. D.</span></span> <span> </span><span class="NLM_article-title">-Omic and Electronic Health Record Big Data Analytics for Precision Medicine</span>. <i>IEEE Trans. Biomed. Eng.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">263</span>– <span class="NLM_lpage">273</span>, <span class="refDoi"> DOI: 10.1109/TBME.2016.2573285</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1109%2FTBME.2016.2573285" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=27740470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A280%3ADC%252BC2srgt12itw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2017&pages=263-273&author=P.+Y.+Wuauthor=C.+W.+Chengauthor=C.+D.+Kaddiauthor=J.+Venugopalanauthor=R.+Hoffmanauthor=M.+D.+Wang&title=-Omic+and+Electronic+Health+Record+Big+Data+Analytics+for+Precision+Medicine&doi=10.1109%2FTBME.2016.2573285"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">-Omic and Electronic Health Record Big Data Analytics for Precision Medicine</span></div><div class="casAuthors">Wu Po-Yen; Cheng Chih-Wen; Kaddi Chanchala D; Venugopalan Janani; Hoffman Ryan; Wang May D</div><div class="citationInfo"><span class="NLM_cas:title">IEEE transactions on bio-medical engineering</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">263-273</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">OBJECTIVE:  Rapid advances of high-throughput technologies and wide adoption of electronic health records (EHRs) have led to fast accumulation of -omic and EHR data.  These voluminous complex data contain abundant information for precision medicine, and big data analytics can extract such knowledge to improve the quality of healthcare.  METHODS:  In this paper, we present -omic and EHR data characteristics, associated challenges, and data analytics including data preprocessing, mining, and modeling.  RESULTS:  To demonstrate how big data analytics enables precision medicine, we provide two case studies, including identifying disease biomarkers from multi-omic data and incorporating -omic information into EHR.  CONCLUSION:  Big data analytics is able to address -omic and EHR data challenges for paradigm shift toward precision medicine.  SIGNIFICANCE:  Big data analytics makes sense of -omic and EHR data to improve healthcare outcome.  It has long lasting societal impact.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS1QO5cfWToYj69zv1FsJFMfW6udTcc2eYHFSkLOszM4bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srgt12itw%253D%253D&md5=d86fcc963fd2e0d7defaf2b8042ed5b4</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1109%2FTBME.2016.2573285&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1109%252FTBME.2016.2573285%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%2BY.%26aulast%3DCheng%26aufirst%3DC.%2BW.%26aulast%3DKaddi%26aufirst%3DC.%2BD.%26aulast%3DVenugopalan%26aufirst%3DJ.%26aulast%3DHoffman%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DM.%2BD.%26atitle%3D-Omic%2520and%2520Electronic%2520Health%2520Record%2520Big%2520Data%2520Analytics%2520for%2520Precision%2520Medicine%26jtitle%3DIEEE%2520Trans.%2520Biomed.%2520Eng.%26date%3D2017%26volume%3D64%26spage%3D263%26epage%3D273%26doi%3D10.1109%2FTBME.2016.2573285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Khosravi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jayaram, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goliaei, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masoudi-Nejad, A.</span></span> <span> </span><span class="NLM_article-title">Active Repurposing of Drug Candidates for Melanoma Based on GWAS, PheWAS and a Wide Range of Omics Data</span>. <i>Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">30</span>, <span class="refDoi"> DOI: 10.1186/s10020-019-0098-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1186%2Fs10020-019-0098-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=31221082" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A280%3ADC%252BB3M3ovFejsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2019&pages=30&author=A.+Khosraviauthor=B.+Jayaramauthor=B.+Goliaeiauthor=A.+Masoudi-Nejad&title=Active+Repurposing+of+Drug+Candidates+for+Melanoma+Based+on+GWAS%2C+PheWAS+and+a+Wide+Range+of+Omics+Data&doi=10.1186%2Fs10020-019-0098-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Active repurposing of drug candidates for melanoma based on GWAS, PheWAS and a wide range of omics data</span></div><div class="casAuthors">Khosravi Ali; Masoudi-Nejad Ali; Jayaram B; Goliaei Bahram</div><div class="citationInfo"><span class="NLM_cas:title">Molecular medicine (Cambridge, Mass.)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">30</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Drug repurposing is a swift, safe, and cheap drug discovery method.  Melanoma disorders present low survival and high mortality rates and are challenging to diagnose and treat.  Moreover, there is a high volume of worldwide investigations that are attempting to find melanoma-related genes of influence, which can be identified as responsive targets for reliable treatment.  METHOD:  In this study, we used a wide range of data analyses to analyze over 1100 genes and proteins of influence with respect to cutaneous malignant melanoma.  Our analysis included various investigational results from genome- and phenome-wide association studies (GWAS and PheWAS, respectively), biomedical, transcriptomic, and metabolomic datasets.  We then researched the DrugBank for potential melanoma targets from the selected list.  We excluded known melanoma targets to obtain a list of druggable proteins.  We performed a precise analysis of the drugs' pathogenesis and checked the expression profiles of the selected drugs having high associations with known anti-melanoma drugs.  RESULT:  We found 35 drugs that interacted with 20 unique targets.  These drugs appear to have high melanoma treatment potentials.  We confirmed our results with previous studies and found supporting references for 30 of these drugs.  In conclusion, this investigation can be applied to various diseases for the efficient and economical repurposing of various drug compounds.  For further validation, the results may be applicable for in vivo tests and clinical trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRegUNtOaqiXMkU0YCbOU3EfW6udTcc2eajSDxIAC_wnbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3ovFejsw%253D%253D&md5=acdf7ca4dae0cf3610a7a0a60ca7a97c</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1186%2Fs10020-019-0098-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs10020-019-0098-x%26sid%3Dliteratum%253Aachs%26aulast%3DKhosravi%26aufirst%3DA.%26aulast%3DJayaram%26aufirst%3DB.%26aulast%3DGoliaei%26aufirst%3DB.%26aulast%3DMasoudi-Nejad%26aufirst%3DA.%26atitle%3DActive%2520Repurposing%2520of%2520Drug%2520Candidates%2520for%2520Melanoma%2520Based%2520on%2520GWAS%252C%2520PheWAS%2520and%2520a%2520Wide%2520Range%2520of%2520Omics%2520Data%26jtitle%3DMol.%2520Med.%26date%3D2019%26volume%3D25%26spage%3D30%26doi%3D10.1186%2Fs10020-019-0098-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shannon, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markiel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozier, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baliga, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramage, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwikowski, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ideker, T.</span></span> <span> </span><span class="NLM_article-title">Cytoscape: A Software Environment for Integrated Models of Biomolecular Interaction Networks</span>. <i>Genome Res.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2498</span>– <span class="NLM_lpage">2504</span>, <span class="refDoi"> DOI: 10.1101/gr.1239303</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1101%2Fgr.1239303" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=14597658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BD3sXovFWrtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=2498-2504&author=P.+Shannonauthor=A.+Markielauthor=O.+Ozierauthor=N.+S.+Baligaauthor=J.+T.+Wangauthor=D.+Ramageauthor=N.+Aminauthor=B.+Schwikowskiauthor=T.+Ideker&title=Cytoscape%3A+A+Software+Environment+for+Integrated+Models+of+Biomolecular+Interaction+Networks&doi=10.1101%2Fgr.1239303"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Cytoscape: A software environment for integrated models of biomolecular interaction networks</span></div><div class="casAuthors">Shannon, Paul; Markiel, Andrew; Ozier, Owen; Baliga, Nitin S.; Wang, Jonathan T.; Ramage, Daniel; Amin, Nada; Schwikowski, Benno; Ideker, Trey</div><div class="citationInfo"><span class="NLM_cas:title">Genome Research</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2498-2504</span>CODEN:
                <span class="NLM_cas:coden">GEREFS</span>;
        ISSN:<span class="NLM_cas:issn">1088-9051</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Cytoscape is an open source software project for integrating biomol. interaction networks with high-throughput expression data and other mol. states into a unified conceptual framework.  Although applicable to any system of mol. components and interactions, Cytoscape is most powerful when used in conjunction with large databases of protein-protein, protein-DNA, and genetic interactions that are increasingly available for humans and model organisms.  Cytoscape's software Core provides basic functionality to layout and query the network; to visually integrate the network with expression profiles, phenotypes, and other mol. states; and to link the network to databases of functional annotations.  The Core is extensible through a straightforward plug-in architecture, allowing rapid development of addnl. computational analyses and features.  Several case studies of Cytoscape plug-ins are surveyed, including a search for interaction pathways correlating with changes in gene expression, a study of protein complexes involved in cellular recovery to DNA damage, inference of a combined phys./functional interaction network for Halobacterium, and an interface to detailed stochastic/kinetic gene regulatory models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpz_cu7mgf2RbVg90H21EOLACvtfcHk0lhbigWFAQImAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXovFWrtr4%253D&md5=2bcbca9a3bd04717761f0424c0209e43</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1101%2Fgr.1239303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgr.1239303%26sid%3Dliteratum%253Aachs%26aulast%3DShannon%26aufirst%3DP.%26aulast%3DMarkiel%26aufirst%3DA.%26aulast%3DOzier%26aufirst%3DO.%26aulast%3DBaliga%26aufirst%3DN.%2BS.%26aulast%3DWang%26aufirst%3DJ.%2BT.%26aulast%3DRamage%26aufirst%3DD.%26aulast%3DAmin%26aufirst%3DN.%26aulast%3DSchwikowski%26aufirst%3DB.%26aulast%3DIdeker%26aufirst%3DT.%26atitle%3DCytoscape%253A%2520A%2520Software%2520Environment%2520for%2520Integrated%2520Models%2520of%2520Biomolecular%2520Interaction%2520Networks%26jtitle%3DGenome%2520Res.%26date%3D2003%26volume%3D13%26spage%3D2498%26epage%3D2504%26doi%3D10.1101%2Fgr.1239303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bastian, M.</span>; <span class="NLM_string-name">Heymann, S.</span>; <span class="NLM_string-name">Jacomy, M.</span></span> <span> </span><span class="NLM_article-title">Gephi: An Open Source Software for Exploring and Manipulating Networks</span>. Presented at the Third International AAAI Conference on Weblogs and Social Media, <span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=M.+Bastian&author=S.+Heymann&author=M.+Jacomy&title=Gephi%3A+An+Open+Source+Software+for+Exploring+and+Manipulating+Networks"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DBastian%26aufirst%3DM.%26atitle%3DGephi%253A%2520An%2520Open%2520Source%2520Software%2520for%2520Exploring%2520and%2520Manipulating%2520Networks%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de Nooy, W.</span>; <span class="NLM_string-name">Mrvar, A.</span>; <span class="NLM_string-name">Batagelj, V.</span></span> <i>Exploratory Social Network Analysis with Pajek: Revised and Expanded Edition for Updated Software</i>; <span class="NLM_publisher-name">Cambridge University Press</span>: <span class="NLM_publisher-loc">Cambridge, U.K.</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1017%2F9781108565691" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=W.+de+Nooy&author=A.+Mrvar&author=V.+Batagelj&title=Exploratory+Social+Network+Analysis+with+Pajek%3A+Revised+and+Expanded+Edition+for+Updated+Software"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1017%2F9781108565691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1017%252F9781108565691%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BNooy%26aufirst%3DW.%26btitle%3DExploratory%2520Social%2520Network%2520Analysis%2520with%2520Pajek%253A%2520Revised%2520and%2520Expanded%2520Edition%2520for%2520Updated%2520Software%26pub%3DCambridge%2520University%2520Press%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hagberg, A. A.</span>; <span class="NLM_string-name">Schult, D. A.</span>; <span class="NLM_string-name">Swart, P. J.</span></span> <span> </span><span class="NLM_article-title">Exploring Network Structure, Dynamics, and Function Using NetworkX</span>. In  <i>Proceedings of the 7th Python in Science Conference (SciPy 2008)</i>, Pasadena, CA, U.S.; <span class="NLM_year">2008</span>; pp  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">15</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&pages=11-15&author=A.+A.+Hagberg&author=D.+A.+Schult&author=P.+J.+Swart&title=Proceedings+of+the+7th+Python+in+Science+Conference+%28SciPy+2008%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DHagberg%26aufirst%3DA.%2BA.%26atitle%3DExploring%2520Network%2520Structure%252C%2520Dynamics%252C%2520and%2520Function%2520Using%2520NetworkX%26btitle%3DProceedings%2520of%2520the%25207th%2520Python%2520in%2520Science%2520Conference%2520%2528SciPy%25202008%2529%26date%3D2008%26spage%3D11%26epage%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pavlopoulos, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paez-Espino, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kyrpides, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iliopoulos, I.</span></span> <span> </span><span class="NLM_article-title">Empirical Comparison of Visualization Tools for Larger-Scale Network Analysis</span>. <i>Adv. Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>2017</i></span>,  <span class="NLM_fpage">1278932</span>, <span class="refDoi"> DOI: 10.1155/2017/1278932</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1155%2F2017%2F1278932" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=28804499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A280%3ADC%252BC1cfnt1WntQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2017&publication_year=2017&pages=1278932&author=G.+A.+Pavlopoulosauthor=D.+Paez-Espinoauthor=N.+C.+Kyrpidesauthor=I.+Iliopoulos&title=Empirical+Comparison+of+Visualization+Tools+for+Larger-Scale+Network+Analysis&doi=10.1155%2F2017%2F1278932"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Empirical Comparison of Visualization Tools for Larger-Scale Network Analysis</span></div><div class="casAuthors">Pavlopoulos Georgios A; Paez-Espino David; Kyrpides Nikos C; Iliopoulos Ioannis</div><div class="citationInfo"><span class="NLM_cas:title">Advances in bioinformatics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">2017</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1278932</span>
        ISSN:<span class="NLM_cas:issn">1687-8027</span>.
    </div><div class="casAbstract">Gene expression, signal transduction, protein/chemical interactions, biomedical literature cooccurrences, and other concepts are often captured in biological network representations where nodes represent a certain bioentity and edges the connections between them.  While many tools to manipulate, visualize, and interactively explore such networks already exist, only few of them can scale up and follow today's indisputable information growth.  In this review, we shortly list a catalog of available network visualization tools and, from a user-experience point of view, we identify four candidate tools suitable for larger-scale network analysis, visualization, and exploration.  We comment on their strengths and their weaknesses and empirically discuss their scalability, user friendliness, and postvisualization capabilities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSSEd2tJNwnQjDnmmyUf5qOfW6udTcc2eZWwBscu9D_M7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cfnt1WntQ%253D%253D&md5=2615dcef0c9fad3aea83f4305aebe5e3</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1155%2F2017%2F1278932&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2017%252F1278932%26sid%3Dliteratum%253Aachs%26aulast%3DPavlopoulos%26aufirst%3DG.%2BA.%26aulast%3DPaez-Espino%26aufirst%3DD.%26aulast%3DKyrpides%26aufirst%3DN.%2BC.%26aulast%3DIliopoulos%26aufirst%3DI.%26atitle%3DEmpirical%2520Comparison%2520of%2520Visualization%2520Tools%2520for%2520Larger-Scale%2520Network%2520Analysis%26jtitle%3DAdv.%2520Bioinf.%26date%3D2017%26volume%3D2017%26spage%3D1278932%26doi%3D10.1155%2F2017%2F1278932" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manikandan, S. G.</span>; <span class="NLM_string-name">Ravi, S.</span></span> <span> </span><span class="NLM_article-title">Big Data Analysis Using Apache Hadoop</span>.  <i>Proceedings</i>, 2014 International Conference on IT Convergence and Security (ICITCS), Beijing, China, 2014; <span class="NLM_publisher-name">IEEE</span>, <span class="NLM_year">2014</span>; <span class="refDoi"> DOI: 10.1109/ICITCS.2014.7021746</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1109%2FICITCS.2014.7021746" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=S.+G.+Manikandan&author=S.+Ravi&title=Proceedings"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1109%2FICITCS.2014.7021746&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1109%252FICITCS.2014.7021746%26sid%3Dliteratum%253Aachs%26aulast%3DManikandan%26aufirst%3DS.%2BG.%26atitle%3DBig%2520Data%2520Analysis%2520Using%2520Apache%2520Hadoop%26btitle%3DProceedings%26pub%3DIEEE%26date%3D2014%26doi%3D10.1109%2FICITCS.2014.7021746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zaharia, M.</span>; <span class="NLM_string-name">Chowdhury, M.</span>; <span class="NLM_string-name">Franklin, M. J.</span>; <span class="NLM_string-name">Shenker, S.</span>; <span class="NLM_string-name">Stoica, I.</span></span> <span> </span><span class="NLM_article-title">Spark: Cluster Computing with Working Sets</span>.  <i>HotCloud ’10: Proceedings of the 2nd USENIX Conference on Hot Topics in Cloud Computing</i>; <span class="NLM_publisher-name">USENIX Association</span>, <span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=M.+Zaharia&author=M.+Chowdhury&author=M.+J.+Franklin&author=S.+Shenker&author=I.+Stoica&title=HotCloud+%E2%80%9910%3A+Proceedings+of+the+2nd+USENIX+Conference+on+Hot+Topics+in+Cloud+Computing"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DZaharia%26aufirst%3DM.%26atitle%3DSpark%253A%2520Cluster%2520Computing%2520with%2520Working%2520Sets%26btitle%3DHotCloud%2520%25E2%2580%259910%253A%2520Proceedings%2520of%2520the%25202nd%2520USENIX%2520Conference%2520on%2520Hot%2520Topics%2520in%2520Cloud%2520Computing%26pub%3DUSENIX%2520Association%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maggiora, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Chemical Space Networks: A Powerful New Paradigm for the Description of Chemical Space</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">795</span>– <span class="NLM_lpage">802</span>, <span class="refDoi"> DOI: 10.1007/s10822-014-9760-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1007%2Fs10822-014-9760-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=24925682" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpvVeisbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2014&pages=795-802&author=G.+M.+Maggioraauthor=J.+Bajorath&title=Chemical+Space+Networks%3A+A+Powerful+New+Paradigm+for+the+Description+of+Chemical+Space&doi=10.1007%2Fs10822-014-9760-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical space networks: a powerful new paradigm for the description of chemical space</span></div><div class="casAuthors">Maggiora, Gerald M.; Bajorath, Jurgen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">795-802</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  The concept of chem. space is playing an increasingly important role in many areas of chem. research, esp. medicinal chem. and chem. biol.  It is generally conceived as consisting of numerous compd. clusters of varying sizes scattered throughout the space in much the same way as galaxies of stars inhabit our universe.  A no. of issues assocd. with this coordinate-based representation are discussed.  Not the least of which is the continuous nature of the space, a feature not entirely compatible with the inherently discrete nature of chem. space.  Cell-based representations, which are derived from coordinate-based spaces, have also been developed that facilitate a no. of chem. informatic activities (e.g., diverse subset selection, filling 'diversity voids', and comparing compd. collections).  These representations generally suffer the 'curse of dimensionality'.  In this work, networks are proposed as an attractive paradigm for representing chem. space since they circumvent many of the issues assocd. with coordinate- and cell-based representations, including the curse of dimensionality.  In addn., their relational structure is entirely compatible with the intrinsic nature of chem. space.  A description of the features of these chem. space networks is presented that emphasizes their statistical characteristics and indicates how they are related to various types of network topologies that exhibit random, scale-free, and/or 'small world' properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqtDD1mJ2GSLVg90H21EOLACvtfcHk0ljX3UbrM_L0zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpvVeisbY%253D&md5=fed9b69e5aef335c4e0670120e652848</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1007%2Fs10822-014-9760-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-014-9760-0%26sid%3Dliteratum%253Aachs%26aulast%3DMaggiora%26aufirst%3DG.%2BM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DChemical%2520Space%2520Networks%253A%2520A%2520Powerful%2520New%2520Paradigm%2520for%2520the%2520Description%2520of%2520Chemical%2520Space%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2014%26volume%3D28%26spage%3D795%26epage%3D802%26doi%3D10.1007%2Fs10822-014-9760-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stumpfe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maggiora, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Lessons Learned from the Design of Chemical Space Networks and Opportunities for New Applications</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">191</span>– <span class="NLM_lpage">208</span>, <span class="refDoi"> DOI: 10.1007/s10822-016-9906-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1007%2Fs10822-016-9906-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=26945865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC28XjvVGhsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=191-208&author=M.+Vogtauthor=D.+Stumpfeauthor=G.+M.+Maggioraauthor=J.+Bajorath&title=Lessons+Learned+from+the+Design+of+Chemical+Space+Networks+and+Opportunities+for+New+Applications&doi=10.1007%2Fs10822-016-9906-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons learned from the design of chemical space networks and opportunities for new applications</span></div><div class="casAuthors">Vogt, Martin; Stumpfe, Dagmar; Maggiora, Gerald M.; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">191-208</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The concept of chem. space is of fundamental relevance in chem. informatics and computer-aided drug discovery.  In a series of articles published in the Journal of Computer-Aided Mol. Design, principles of chem. space design were evaluated, mol. networks proposed as an alternative to conventional coordinate-based chem. ref. spaces, and different types of chem. space networks (CSNs) constructed and analyzed.  Central to the generation of CSNs was the way in which mol. similarity relationships were assessed and a primary focal point was the network-based representation of biol. relevant chem. space.  The design and comparison of CSNs based upon alternative similarity measures can be viewed as an evolutionary path with interesting lessons learned along the way.  CSN design has matured to the point that such chem. space representations can be used in practice.  In this contribution, highlights from the sequence of CSN design efforts are discussed in context, providing a perspective for future practical applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA5JT9f_RfpbVg90H21EOLACvtfcHk0ljX3UbrM_L0zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjvVGhsro%253D&md5=e5607c1fe6b9a5e74a55e8e97464bb22</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1007%2Fs10822-016-9906-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-016-9906-3%26sid%3Dliteratum%253Aachs%26aulast%3DVogt%26aufirst%3DM.%26aulast%3DStumpfe%26aufirst%3DD.%26aulast%3DMaggiora%26aufirst%3DG.%2BM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DLessons%2520Learned%2520from%2520the%2520Design%2520of%2520Chemical%2520Space%2520Networks%2520and%2520Opportunities%2520for%2520New%2520Applications%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2016%26volume%3D30%26spage%3D191%26epage%3D208%26doi%3D10.1007%2Fs10822-016-9906-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maggiora, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Comparison of Bioactive Chemical Space Networks Generated Using Substructure- and Fingerprint-Based Measures of Molecular Similarity</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">595</span>– <span class="NLM_lpage">608</span>, <span class="refDoi"> DOI: 10.1007/s10822-015-9852-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1007%2Fs10822-015-9852-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=26049785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVSgtLjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=595-608&author=B.+Zhangauthor=M.+Vogtauthor=G.+M.+Maggioraauthor=J.+Bajorath&title=Comparison+of+Bioactive+Chemical+Space+Networks+Generated+Using+Substructure-+and+Fingerprint-Based+Measures+of+Molecular+Similarity&doi=10.1007%2Fs10822-015-9852-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of bioactive chemical space networks generated using substructure- and fingerprint-based measures of molecular similarity</span></div><div class="casAuthors">Zhang, Bijun; Vogt, Martin; Maggiora, Gerald M.; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">595-608</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Chem. space networks (CSNs) have recently been introduced as a conceptual alternative to coordinate-based representations of chem. space.  CSNs were initially designed as threshold networks using the Tanimoto coeff. as a continuous similarity measure.  The anal. of CSNs generated from sets of bioactive compds. revealed that many statistical properties were strongly dependent on their edge d.  While it was difficult to compare CSNs at pre-defined similarity threshold values, CSNs with const. edge d. were directly comparable.  In the current study, alternative CSN representations were constructed by applying the matched mol. pair (MMP) formalism as a substructure-based similarity criterion.  For more than 150 compd. activity classes, MMP-based CSNs (MMP-CSNs) were compared to corresponding threshold CSNs (THR-CSNs) at a const. edge d. by applying different parameters from network science, measures of community structure distributions, and indicators of structure-activity relationship (SAR) information content.  MMP-CSNs were found to be an attractive alternative to THR-CSNs, yielding low edge densities and well-resolved topologies.  MMP-CSNs and corresponding THR-CSNs often had similar topol. and closely corresponding community structures, although there was only limited overlap in similarity relationships.  The homophily principle from network science was shown to affect MMP-CSNs and THR-CSNs in different ways, despite the presence of conserved topol. features.  Moreover, activity cliff distributions in alternative CSN designs markedly differed, which has important implications for SAR anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQAfI8eb4WjbVg90H21EOLACvtfcHk0ljzNir2rVBfKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVSgtLjK&md5=057da71a8b5462862a850f3f484c3050</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1007%2Fs10822-015-9852-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-015-9852-5%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DVogt%26aufirst%3DM.%26aulast%3DMaggiora%26aufirst%3DG.%2BM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DComparison%2520of%2520Bioactive%2520Chemical%2520Space%2520Networks%2520Generated%2520Using%2520Substructure-%2520and%2520Fingerprint-Based%2520Measures%2520of%2520Molecular%2520Similarity%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26spage%3D595%26epage%3D608%26doi%3D10.1007%2Fs10822-015-9852-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zwierzyna, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maggiora, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">Design and Characterization of Chemical Space Networks for Different Compound Data Sets</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">113</span>– <span class="NLM_lpage">125</span>, <span class="refDoi"> DOI: 10.1007/s10822-014-9821-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1007%2Fs10822-014-9821-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=25465052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVWgt7nM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=113-125&author=M.+Zwierzynaauthor=M.+Vogtauthor=G.+M.+Maggioraauthor=J.+Bajorath&title=Design+and+Characterization+of+Chemical+Space+Networks+for+Different+Compound+Data+Sets&doi=10.1007%2Fs10822-014-9821-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Design and characterization of chemical space networks for different compound data sets</span></div><div class="casAuthors">Zwierzyna, Magdalena; Vogt, Martin; Maggiora, Gerald M.; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">113-125</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Chem. Space Networks (CSNs) are generated for different compd. data sets on the basis of pairwise similarity relationships.  Such networks are thought to complement and further extend traditional coordinate-based views of chem. space.  Our proof-of-concept study focuses on CSNs based upon fingerprint similarity relationships calcd. using the conventional Tanimoto similarity metric.  The resulting CSNs are characterized with statistical measures from network science and compared in different ways.  We show that the homophily principle, which is widely considered in the context of social networks, is a major determinant of the topol. of CSNs of bioactive compds., designed as threshold networks, typically giving rise to community structures.  Many properties of CSNs are influenced by numerical features of the conventional Tanimoto similarity metric and largely dominated by the edge d. of the networks, which depends on chosen similarity threshold values.  However, properties of different CSNs with const. edge d. can be directly compared, revealing systematic differences between CSNs generated from randomly collected or bioactive compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1O5MgBoa4W7Vg90H21EOLACvtfcHk0ljzNir2rVBfKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVWgt7nM&md5=db2c790b6d162e7f0c14ec0f42ac374d</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1007%2Fs10822-014-9821-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-014-9821-4%26sid%3Dliteratum%253Aachs%26aulast%3DZwierzyna%26aufirst%3DM.%26aulast%3DVogt%26aufirst%3DM.%26aulast%3DMaggiora%26aufirst%3DG.%2BM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DDesign%2520and%2520Characterization%2520of%2520Chemical%2520Space%2520Networks%2520for%2520Different%2520Compound%2520Data%2520Sets%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2015%26volume%3D29%26spage%3D113%26epage%3D125%26doi%3D10.1007%2Fs10822-014-9821-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, M.</span></span> <span> </span><span class="NLM_article-title">Extended-Connectivity Fingerprints</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">742</span>– <span class="NLM_lpage">754</span>, <span class="refDoi"> DOI: 10.1021/ci100050t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100050t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=742-754&author=D.+Rogersauthor=M.+Hahn&title=Extended-Connectivity+Fingerprints&doi=10.1021%2Fci100050t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Extended-Connectivity Fingerprints</span></div><div class="casAuthors">Rogers, David; Hahn, Mathew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">742-754</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Extended-connectivity fingerprints (ECFPs) are a novel class of topol. fingerprints for mol. characterization.  Historically, topol. fingerprints were developed for substructure and similarity searching.  ECFPs were developed specifically for structure-activity modeling.  ECFPs are circular fingerprints with a no. of useful qualities: they can be very rapidly calcd.; they are not predefined and can represent an essentially infinite no. of different mol. features (including stereochem. information); their features represent the presence of particular substructures, allowing easier interpretation of anal. results; and the ECFP algorithm can be tailored to generate different types of circular fingerprints, optimized for different uses.  While the use of ECFPs has been widely adopted and validated, a description of their implementation has not previously been presented in the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra43orI-Y7lbVg90H21EOLACvtfcHk0li74sucAwmU-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D&md5=cd6c736cd7a3d280b67f5316acce8006</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fci100050t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100050t%26sid%3Dliteratum%253Aachs%26aulast%3DRogers%26aufirst%3DD.%26aulast%3DHahn%26aufirst%3DM.%26atitle%3DExtended-Connectivity%2520Fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D742%26epage%3D754%26doi%3D10.1021%2Fci100050t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Duan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowrie, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Analysis and Comparison of 2D Fingerprints: Insights into Database Screening Performance Using Eight Fingerprint Methods</span>. <i>J. Mol. Graphics Modell.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">157</span>– <span class="NLM_lpage">170</span>, <span class="refDoi"> DOI: 10.1016/j.jmgm.2010.05.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1016%2Fj.jmgm.2010.05.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=20579912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFamtLrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2010&pages=157-170&author=J.+Duanauthor=S.+L.+Dixonauthor=J.+F.+Lowrieauthor=W.+Sherman&title=Analysis+and+Comparison+of+2D+Fingerprints%3A+Insights+into+Database+Screening+Performance+Using+Eight+Fingerprint+Methods&doi=10.1016%2Fj.jmgm.2010.05.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis and comparison of 2D fingerprints: Insights into database screening performance using eight fingerprint methods</span></div><div class="casAuthors">Duan, Jian-Xin; Dixon, Steven L.; Lowrie, Jeffrey F.; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">157-170</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Virtual screening is a widely used strategy in modern drug discovery and 2D fingerprint similarity is an important tool that has been successfully applied to retrieve active compds. from large datasets.  However, it is not always straightforward to select an appropriate fingerprint method and assocd. settings for a given problem.  Here, we applied eight different fingerprint methods, as implemented in the new cheminformatics package Canvas, on a well-validated dataset covering five targets.  The fingerprint methods include Linear, Dendritic, Radial, MACCS, MOLPRINT2D, Pairwise, Triplet, and Torsion.  We find that most fingerprints have similar retrieval rates on av.; however, each has special characteristics that distinguish its performance on different query mols. and ligand sets.  For example, some fingerprints exhibit a significant ligand size dependency whereas others are more robust with respect to variations in the query or active compds.  In cases where little information is known about the active ligands, MOLPRINT2D fingerprints produce the highest av. retrieval actives.  When multiple queries are available, we find that a fingerprint averaged over all query mols. is generally superior to fingerprints derived from single queries.  Finally, a complementarity metric is proposed to det. which fingerprint methods can be combined to improve screening results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWCcy4qE10YbVg90H21EOLACvtfcHk0li74sucAwmU-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFamtLrM&md5=f72d91018346b1e5ef97fc83f648bd86</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2010.05.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2010.05.008%26sid%3Dliteratum%253Aachs%26aulast%3DDuan%26aufirst%3DJ.%26aulast%3DDixon%26aufirst%3DS.%2BL.%26aulast%3DLowrie%26aufirst%3DJ.%2BF.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DAnalysis%2520and%2520Comparison%2520of%25202D%2520Fingerprints%253A%2520Insights%2520into%2520Database%2520Screening%2520Performance%2520Using%2520Eight%2520Fingerprint%2520Methods%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2010%26volume%3D29%26spage%3D157%26epage%3D170%26doi%3D10.1016%2Fj.jmgm.2010.05.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nothias-Esposito, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nothias, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Da Silva, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Retailleau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leyssen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roussi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Touboul, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paolini, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorrestein, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Litaudon, M.</span></span> <span> </span><span class="NLM_article-title">Investigation of Premyrsinane and Myrsinane Esters in <i>Euphorbia Cupanii</i> and <i>Euphobia Pithyusa</i> with <i>MS2LDA</i> and Combinatorial Molecular Network Annotation Propagation</span>. <i>J. Nat. Prod.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>82</i></span>,  <span class="NLM_fpage">1459</span>– <span class="NLM_lpage">1470</span>, <span class="refDoi"> DOI: 10.1021/acs.jnatprod.8b00916</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jnatprod.8b00916" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtV2htr7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=82&publication_year=2019&pages=1459-1470&author=M.+Nothias-Espositoauthor=L.+F.+Nothiasauthor=R.+R.+Da+Silvaauthor=P.+Retailleauauthor=Z.+Zhangauthor=P.+Leyssenauthor=F.+Roussiauthor=D.+Touboulauthor=J.+Paoliniauthor=P.+C.+Dorresteinauthor=M.+Litaudon&title=Investigation+of+Premyrsinane+and+Myrsinane+Esters+in+Euphorbia+Cupanii+and+Euphobia+Pithyusa+with+MS2LDA+and+Combinatorial+Molecular+Network+Annotation+Propagation&doi=10.1021%2Facs.jnatprod.8b00916"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Investigation of Premyrsinane and Myrsinane Esters in Euphorbia cupanii and Euphobia pithyusa with MS2LDA and Combinatorial Molecular Network Annotation Propagation</span></div><div class="casAuthors">Nothias-Esposito, Melissa; Nothias, Louis Felix; Da Silva, Ricardo R.; Retailleau, Pascal; Zhang, Zheng; Leyssen, Pieter; Roussi, Fanny; Touboul, David; Paolini, Julien; Dorrestein, Pieter C.; Litaudon, Marc</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Natural Products</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">82</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1459-1470</span>CODEN:
                <span class="NLM_cas:coden">JNPRDF</span>;
        ISSN:<span class="NLM_cas:issn">0163-3864</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society-American Society of Pharmacognosy</span>)
        </div><div class="casAbstract">The species Euphorbia pithyusa and Euphorbia cupanii are two closely related Mediterranean spurges for which their taxonomic relationships are still being debated.  Herein, the diterpene ester content of E. cupanii was investigated using liq. chromatog. coupled to tandem mass spectrometry.  The use of mol. networking coupled to unsupervised substructure annotation (MS2LDA) indicated the presence of new premyrsinane/myrsinane diterpene esters in the E. cupanii fractions.  A structure-guided isolation procedure yielded 16 myrsinane (11a-h, 12, and 13) and premyrsinane esters (14a-c and 15a-c), along with four 4β-phorbol esters (16a-c and 17) that showed inhibitory activity against chikungunya virus replication.  The structures of the 16 new compds. (11a-c, 11h, 12, 13, 14a-c, 15a-c, 16a-c, and 17) were characterized by NMR spectroscopy and X-ray crystallog.  To further uncover the diterpene ester content of these two species, the concept of combinatorial network annotation propagation (C-NAP) was developed.  By leveraging the fact that the diterpene esters of Euphorbia species are made up of limited building blocks, a combinatorial database of theor. structures was created and used for C-NAP that made possible the annotation of 123 premyrsinane or myrsinane esters, from which 74% are not found in any compd. database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-7jM0k3FODrVg90H21EOLACvtfcHk0lh34lvUeEorig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtV2htr7F&md5=2865494a678bcfd49e2c2ce4005932c5</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facs.jnatprod.8b00916&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jnatprod.8b00916%26sid%3Dliteratum%253Aachs%26aulast%3DNothias-Esposito%26aufirst%3DM.%26aulast%3DNothias%26aufirst%3DL.%2BF.%26aulast%3DDa%2BSilva%26aufirst%3DR.%2BR.%26aulast%3DRetailleau%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DLeyssen%26aufirst%3DP.%26aulast%3DRoussi%26aufirst%3DF.%26aulast%3DTouboul%26aufirst%3DD.%26aulast%3DPaolini%26aufirst%3DJ.%26aulast%3DDorrestein%26aufirst%3DP.%2BC.%26aulast%3DLitaudon%26aufirst%3DM.%26atitle%3DInvestigation%2520of%2520Premyrsinane%2520and%2520Myrsinane%2520Esters%2520in%2520Euphorbia%2520Cupanii%2520and%2520Euphobia%2520Pithyusa%2520with%2520MS2LDA%2520and%2520Combinatorial%2520Molecular%2520Network%2520Annotation%2520Propagation%26jtitle%3DJ.%2520Nat.%2520Prod.%26date%3D2019%26volume%3D82%26spage%3D1459%26epage%3D1470%26doi%3D10.1021%2Facs.jnatprod.8b00916" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nothias, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vining, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meehan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esquenazi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dorrestein, P. C.</span></span> <span> </span><span class="NLM_article-title">Molecular Networking As a Drug Discovery, Drug Metabolism, and Precision Medicine Strategy</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">143</span>– <span class="NLM_lpage">154</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2016.10.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1016%2Fj.tips.2016.10.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=27842887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslGru7vJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=143-154&author=R.+A.+Quinnauthor=L.+F.+Nothiasauthor=O.+Viningauthor=M.+Meehanauthor=E.+Esquenaziauthor=P.+C.+Dorrestein&title=Molecular+Networking+As+a+Drug+Discovery%2C+Drug+Metabolism%2C+and+Precision+Medicine+Strategy&doi=10.1016%2Fj.tips.2016.10.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Networking As a Drug Discovery, Drug Metabolism, and Precision Medicine Strategy</span></div><div class="casAuthors">Quinn, Robert A.; Nothias, Louis-Felix; Vining, Oliver; Meehan, Michael; Esquenazi, Eduardo; Dorrestein, Pieter C.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">143-154</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Mol. networking is a tandem mass spectrometry (MS/MS) data organizational approach that has been recently introduced in the drug discovery, metabolomics, and medical fields.  The chem. of mols. dictates how they will be fragmented by MS/MS in the gas phase and, therefore, two related mols. are likely to display similar fragment ion spectra.  Mol. networking organizes the MS/MS data as a relational spectral network thereby mapping the chem. that was detected in an MS/MS-based metabolomics expt.  Although the wider utility of mol. networking is just beginning to be recognized, in this review we highlight the principles behind mol. networking and its use for the discovery of therapeutic leads, monitoring drug metab., clin. diagnostics, and emerging applications in precision medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmWfd56pPGA7Vg90H21EOLACvtfcHk0lh34lvUeEorig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslGru7vJ&md5=6459b0d087969f208dc5321732619916</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2016.10.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2016.10.011%26sid%3Dliteratum%253Aachs%26aulast%3DQuinn%26aufirst%3DR.%2BA.%26aulast%3DNothias%26aufirst%3DL.%2BF.%26aulast%3DVining%26aufirst%3DO.%26aulast%3DMeehan%26aufirst%3DM.%26aulast%3DEsquenazi%26aufirst%3DE.%26aulast%3DDorrestein%26aufirst%3DP.%2BC.%26atitle%3DMolecular%2520Networking%2520As%2520a%2520Drug%2520Discovery%252C%2520Drug%2520Metabolism%252C%2520and%2520Precision%2520Medicine%2520Strategy%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2017%26volume%3D38%26spage%3D143%26epage%3D154%26doi%3D10.1016%2Fj.tips.2016.10.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miho, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roškar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greiff, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, S. T.</span></span> <span> </span><span class="NLM_article-title">Large-Scale Network Analysis Reveals the Sequence Space Architecture of Antibody Repertoires</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1321</span>, <span class="refDoi"> DOI: 10.1038/s41467-019-09278-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1038%2Fs41467-019-09278-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=30899025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A280%3ADC%252BB3cbmvVyntA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1321&author=E.+Mihoauthor=R.+Ro%C5%A1karauthor=V.+Greiffauthor=S.+T.+Reddy&title=Large-Scale+Network+Analysis+Reveals+the+Sequence+Space+Architecture+of+Antibody+Repertoires&doi=10.1038%2Fs41467-019-09278-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Large-scale network analysis reveals the sequence space architecture of antibody repertoires</span></div><div class="casAuthors">Miho Enkelejda; Reddy Sai T; Miho Enkelejda; Miho Enkelejda; Roskar Rok; Greiff Victor</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1321</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The architecture of mouse and human antibody repertoires is defined by the sequence similarity networks of the clones that compose them.  The major principles that define the architecture of antibody repertoires have remained largely unknown.  Here, we establish a high-performance computing platform to construct large-scale networks from comprehensive human and murine antibody repertoire sequencing datasets (>100,000 unique sequences).  Leveraging a network-based statistical framework, we identify three fundamental principles of antibody repertoire architecture: reproducibility, robustness and redundancy.  Antibody repertoire networks are highly reproducible across individuals despite high antibody sequence dissimilarity.  The architecture of antibody repertoires is robust to the removal of up to 50-90% of randomly selected clones, but fragile to the removal of public clones shared among individuals.  Finally, repertoire architecture is intrinsically redundant.  Our analysis provides guidelines for the large-scale network analysis of immune repertoires and may be used in the future to define disease-associated and synthetic repertoires.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS-XIX5oF9w5O98rul4nhSdfW6udTcc2eb3ZQ9NUppdj7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cbmvVyntA%253D%253D&md5=700f40420ee30f58ee7a776bded3ff0e</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1038%2Fs41467-019-09278-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-019-09278-8%26sid%3Dliteratum%253Aachs%26aulast%3DMiho%26aufirst%3DE.%26aulast%3DRo%25C5%25A1kar%26aufirst%3DR.%26aulast%3DGreiff%26aufirst%3DV.%26aulast%3DReddy%26aufirst%3DS.%2BT.%26atitle%3DLarge-Scale%2520Network%2520Analysis%2520Reveals%2520the%2520Sequence%2520Space%2520Architecture%2520of%2520Antibody%2520Repertoires%26jtitle%3DNat.%2520Commun.%26date%3D2019%26volume%3D10%26spage%3D1321%26doi%3D10.1038%2Fs41467-019-09278-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhattacharyya, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vishveshwara, S.</span></span> <span> </span><span class="NLM_article-title">Protein Structure and Function: Looking through the Network of Side-Chain Interactions</span>. <i>Curr. Protein Pept. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">4</span>– <span class="NLM_lpage">25</span>, <span class="refDoi"> DOI: 10.2174/1389203716666150923105727</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.2174%2F1389203716666150923105727" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=4-25&author=M.+Bhattacharyyaauthor=S.+Ghoshauthor=S.+Vishveshwara&title=Protein+Structure+and+Function%3A+Looking+through+the+Network+of+Side-Chain+Interactions&doi=10.2174%2F1389203716666150923105727"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.2174%2F1389203716666150923105727&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1389203716666150923105727%26sid%3Dliteratum%253Aachs%26aulast%3DBhattacharyya%26aufirst%3DM.%26aulast%3DGhosh%26aufirst%3DS.%26aulast%3DVishveshwara%26aufirst%3DS.%26atitle%3DProtein%2520Structure%2520and%2520Function%253A%2520Looking%2520through%2520the%2520Network%2520of%2520Side-Chain%2520Interactions%26jtitle%3DCurr.%2520Protein%2520Pept.%2520Sci.%26date%3D2015%26volume%3D17%26spage%3D4%26epage%3D25%26doi%3D10.2174%2F1389203716666150923105727" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di
Paola, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Ruvo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paci, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santoni, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giuliani, A.</span></span> <span> </span><span class="NLM_article-title">Protein Contact Networks: An Emerging Paradigm in Chemistry</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">1598</span>– <span class="NLM_lpage">1613</span>, <span class="refDoi"> DOI: 10.1021/cr3002356</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr3002356" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslCgsLbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2013&pages=1598-1613&author=L.+Di%0APaolaauthor=M.+De+Ruvoauthor=P.+Paciauthor=D.+Santoniauthor=A.+Giuliani&title=Protein+Contact+Networks%3A+An+Emerging+Paradigm+in+Chemistry&doi=10.1021%2Fcr3002356"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Contact Networks: An Emerging Paradigm in Chemistry</span></div><div class="casAuthors">Di Paola, L.; De Ruvo, M.; Paci, P.; Santoni, D.; Giuliani, A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1598-1613</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Nowadays, many different fields of investigation ranging from systems biol. to elec. engineering, sociol., and statistical mechanics converge into the shared operational paradigm of complex network anal.  A massive advancement in the elucidation of general behavior of network systems made possible the generation of brand new graph theor. descriptors, at both single node and entire graph level, that could be useful in many fields of chem.  In this review we will deal with the protein 3D structures in terms of contact networks between amino acid residues.  This case allows for a straightforward formalization in topol. terms: the role of nodes (residues) and edges (contacts) is devoid of any ambiguity and the introduction of van der Waals radii of amino acids allows us to assign a motivated threshold for assigning contacts and building the network.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYqMg8w9WKQrVg90H21EOLACvtfcHk0lhjA18faC6ryg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslCgsLbJ&md5=2a4dbdea166906a22af0ee05ef521ab0</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Fcr3002356&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr3002356%26sid%3Dliteratum%253Aachs%26aulast%3DDi%2BPaola%26aufirst%3DL.%26aulast%3DDe%2BRuvo%26aufirst%3DM.%26aulast%3DPaci%26aufirst%3DP.%26aulast%3DSantoni%26aufirst%3DD.%26aulast%3DGiuliani%26aufirst%3DA.%26atitle%3DProtein%2520Contact%2520Networks%253A%2520An%2520Emerging%2520Paradigm%2520in%2520Chemistry%26jtitle%3DChem.%2520Rev.%26date%3D2013%26volume%3D113%26spage%3D1598%26epage%3D1613%26doi%3D10.1021%2Fcr3002356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verkhivker, G. M.</span></span> <span> </span><span class="NLM_article-title">Molecular Dynamics Simulations and Modelling of the Residue Interaction Networks in the BRAF Kinase Complexes with Small Molecule Inhibitors: Probing the Allosteric Effects of Ligand-Induced Kinase Dimerization and Paradoxical Activation</span>. <i>Mol. BioSyst.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">3146</span>– <span class="NLM_lpage">3165</span>, <span class="refDoi"> DOI: 10.1039/C6MB00298F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1039%2FC6MB00298F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=27481329" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1CrsLrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=3146-3165&author=G.+M.+Verkhivker&title=Molecular+Dynamics+Simulations+and+Modelling+of+the+Residue+Interaction+Networks+in+the+BRAF+Kinase+Complexes+with+Small+Molecule+Inhibitors%3A+Probing+the+Allosteric+Effects+of+Ligand-Induced+Kinase+Dimerization+and+Paradoxical+Activation&doi=10.1039%2FC6MB00298F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics simulations and modelling of the residue interaction networks in the BRAF kinase complexes with small molecule inhibitors: probing the allosteric effects of ligand-induced kinase dimerization and paradoxical activation</span></div><div class="casAuthors">Verkhivker, G. M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3146-3165</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Protein kinases are central to proper functioning of cellular networks and are an integral part of many signal transduction pathways.  The family of protein kinases represents by far the largest and most important class of therapeutic targets in oncol.  Dimerization-induced activation has emerged as a common mechanism of allosteric regulation in BRAF kinases, which play an important role in growth factor signalling and human diseases.  Recent studies have revealed that most of the BRAF inhibitors can induce dimerization and paradoxically stimulate enzyme transactivation by conferring an active conformation in the second monomer of the kinase dimer.  The emerging connections between inhibitor binding and BRAF kinase domain dimerization have suggested a mol. basis of the activation mechanism in which BRAF inhibitors may allosterically modulate the stability of the dimerization interface and affect the organization of residue interaction networks in BRAF kinase dimers.  In this work, we integrated structural bioinformatics anal., mol. dynamics and binding free energy simulations with the protein structure network anal. of the BRAF crystal structures to det. dynamic signatures of BRAF conformations in complexes with different types of inhibitors and probe the mechanisms of the inhibitor-induced dimerization and paradoxical activation.  The results of this study highlight previously unexplored relationships between types of BRAF inhibitors, inhibitor-induced changes in the residue interaction networks and allosteric modulation of the kinase activity.  This study suggests a mechanism by which BRAF inhibitors could promote or interfere with the paradoxical activation of BRAF kinases, which may be useful in informing discovery efforts to minimize the unanticipated adverse biol. consequences of these therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVab0e2fdU27Vg90H21EOLACvtfcHk0liyL1fwOFHmyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1CrsLrF&md5=b2f2d7406801f07b4a3425da843b21b7</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1039%2FC6MB00298F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MB00298F%26sid%3Dliteratum%253Aachs%26aulast%3DVerkhivker%26aufirst%3DG.%2BM.%26atitle%3DMolecular%2520Dynamics%2520Simulations%2520and%2520Modelling%2520of%2520the%2520Residue%2520Interaction%2520Networks%2520in%2520the%2520BRAF%2520Kinase%2520Complexes%2520with%2520Small%2520Molecule%2520Inhibitors%253A%2520Probing%2520the%2520Allosteric%2520Effects%2520of%2520Ligand-Induced%2520Kinase%2520Dimerization%2520and%2520Paradoxical%2520Activation%26jtitle%3DMol.%2520BioSyst.%26date%3D2016%26volume%3D12%26spage%3D3146%26epage%3D3165%26doi%3D10.1039%2FC6MB00298F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verkhivker, G. M.</span></span> <span> </span><span class="NLM_article-title">Biophysical Simulations and Structure-Based Modeling of Residue Interaction Networks in the Tumor Suppressor Proteins Reveal Functional Role of Cancer Mutation Hotspots in Molecular Communication</span>. <i>Biochim. Biophys. Acta, Gen. Subj.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>1863</i></span>,  <span class="NLM_fpage">210</span>– <span class="NLM_lpage">225</span>, <span class="refDoi"> DOI: 10.1016/j.bbagen.2018.10.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1016%2Fj.bbagen.2018.10.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=30339916" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVSisrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1863&publication_year=2019&pages=210-225&author=G.+M.+Verkhivker&title=Biophysical+Simulations+and+Structure-Based+Modeling+of+Residue+Interaction+Networks+in+the+Tumor+Suppressor+Proteins+Reveal+Functional+Role+of+Cancer+Mutation+Hotspots+in+Molecular+Communication&doi=10.1016%2Fj.bbagen.2018.10.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Biophysical simulations and structure-based modeling of residue interaction networks in the tumor suppressor proteins reveal functional role of cancer mutation hotspots in molecular communication</span></div><div class="casAuthors">Verkhivker, Gennady M.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, General Subjects</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">1863</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">210-225</span>CODEN:
                <span class="NLM_cas:coden">BBGSB3</span>;
        ISSN:<span class="NLM_cas:issn">0304-4165</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In the current study, we have combined mol. simulations and energetic anal. with dynamics-based network modeling and perturbation response scanning to det. mol. signatures of mutational hotspot residues in the p53, PTEN, and SMAD4 tumor suppressor proteins.  By examg. structure, energetics and dynamics of these proteins, we have shown that inactivating mutations preferentially target a group of structurally stable residues that play a fundamental role in global propagation of dynamic fluctuations and mediating allosteric interaction networks.  Through integration of long-range perturbation dynamics and network-based approaches, we have quantified allosteric potential of residues in the studied proteins.  The results have revealed that mutational hotspot sites often correspond to high centrality mediating centers of the residue interaction networks that are responsible for coordination of global dynamic changes and allosteric signaling.  Our findings have also suggested that structurally stable mutational hotpots can act as major effectors of allosteric interactions and mutations in these positions are typically assocd. with severe phenotype.  Modeling of shortest inter-residue pathways has shown that mutational hotspot sites can also serve as key mediating bridges of allosteric communication in the p53 and PTEN protein structures.  Multiple regression models have indicated that functional significance of mutational hotspots can be strongly assocd. with the network signatures serving as robust predictors of crit. regulatory positions responsible for loss-of-function phenotype.  The results of this computational investigation are compared with the exptl. studies and reveal mol. signatures of mutational hotspots, providing a plausible rationale for explaining and localizing disease-causing mutations in tumor suppressor genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUXmMaHQLEnrVg90H21EOLACvtfcHk0liyL1fwOFHmyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVSisrvP&md5=8f260103f43d8bd3787de45f77d1feba</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagen.2018.10.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagen.2018.10.009%26sid%3Dliteratum%253Aachs%26aulast%3DVerkhivker%26aufirst%3DG.%2BM.%26atitle%3DBiophysical%2520Simulations%2520and%2520Structure-Based%2520Modeling%2520of%2520Residue%2520Interaction%2520Networks%2520in%2520the%2520Tumor%2520Suppressor%2520Proteins%2520Reveal%2520Functional%2520Role%2520of%2520Cancer%2520Mutation%2520Hotspots%2520in%2520Molecular%2520Communication%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gen.%2520Subj.%26date%3D2019%26volume%3D1863%26spage%3D210%26epage%3D225%26doi%3D10.1016%2Fj.bbagen.2018.10.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amusengeri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Astl, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lobb, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verkhivker, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bishop, Ö. T.</span></span> <span> </span><span class="NLM_article-title">Establishing Computational Approaches towards Identifying Malarial Allosteric Modulators: A Case Study of Plasmodium falciparum Hsp70s</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">5574</span>, <span class="refDoi"> DOI: 10.3390/ijms20225574</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.3390%2Fijms20225574" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BB3cXptV2qsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=5574&author=A.+Amusengeriauthor=L.+Astlauthor=K.+Lobbauthor=G.+M.+Verkhivkerauthor=%C3%96.+T.+Bishop&title=Establishing+Computational+Approaches+towards+Identifying+Malarial+Allosteric+Modulators%3A+A+Case+Study+of+Plasmodium+falciparum+Hsp70s&doi=10.3390%2Fijms20225574"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Establishing computational approaches towards identifying malarial allosteric modulators: a case study of Plasmodium falciparum Hsp70s</span></div><div class="casAuthors">Amusengeri, Arnold; Astl, Lindy; Lobb, Kevin; Verkhivker, Gennady M.; Bishop, Ozlem Tastan</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5574</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Combating malaria is almost a never-ending battle, as Plasmodium parasites develop resistance to the drugs used against them, as obsd. recently in artemisinin-based combination therapies.  The main concern now is if the resistant parasite strains spread from Southeast Asia to Africa, the continent hosting most malaria cases.  To prevent catastrophic results, we need to find non-conventional approaches.  Allosteric drug targeting sites and modulators might be a new hope for malarial treatments.  Heat shock proteins (HSPs) are potential malarial drug targets and have complex allosteric control mechanisms.  Yet, studies on designing allosteric modulators against them are limited.  Here, we identified allosteric modulators (SANC190 and SANC651) against P. falciparum Hsp70-1 and Hsp70-x, affecting the conformational dynamics of the proteins, delicately balanced by the endogenous ligands.  Previously, we established a pipeline to identify allosteric sites and modulators.  This study also further investigated alternative approaches to speed up the process by comparing all atom mol. dynamics simulations and dynamic residue network anal. with the coarse-grained (CG) versions of the calcns.  Betweenness centrality (BC) profiles for PfHsp70-1 and PfHsp70-x derived from CG simulations not only revealed similar trends but also pointed to the same functional regions and specific residues corresponding to BC profile peaks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCObllYOgg5bVg90H21EOLACvtfcHk0liyL1fwOFHmyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXptV2qsrY%253D&md5=111587e894a0e6163c60bd49471b6db8</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.3390%2Fijms20225574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms20225574%26sid%3Dliteratum%253Aachs%26aulast%3DAmusengeri%26aufirst%3DA.%26aulast%3DAstl%26aufirst%3DL.%26aulast%3DLobb%26aufirst%3DK.%26aulast%3DVerkhivker%26aufirst%3DG.%2BM.%26aulast%3DBishop%26aufirst%3D%25C3%2596.%2BT.%26atitle%3DEstablishing%2520Computational%2520Approaches%2520towards%2520Identifying%2520Malarial%2520Allosteric%2520Modulators%253A%2520A%2520Case%2520Study%2520of%2520Plasmodium%2520falciparum%2520Hsp70s%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2019%26volume%3D20%26spage%3D5574%26doi%3D10.3390%2Fijms20225574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barabási, A.-L.</span></span> <span> </span><span class="NLM_article-title">Network Medicine — From Obesity to the “Diseasome”</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>357</i></span>,  <span class="NLM_fpage">404</span>– <span class="NLM_lpage">407</span>, <span class="refDoi"> DOI: 10.1056/NEJMe078114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1056%2FNEJMe078114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=17652657" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BD2sXotFCqu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=357&publication_year=2007&pages=404-407&author=A.-L.+Barab%C3%A1si&title=Network+Medicine+%E2%80%94+From+Obesity+to+the+%E2%80%9CDiseasome%E2%80%9D&doi=10.1056%2FNEJMe078114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Network medicine - from obesity to the "diseasome"</span></div><div class="casAuthors">Barabasi, Albert-Laszlo</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">357</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">404-407</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfYNygODU5XbVg90H21EOLACvtfcHk0licSCCtb27Csg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXotFCqu7w%253D&md5=d752566c157722c1e2e5b0e7e1fe20bf</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1056%2FNEJMe078114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMe078114%26sid%3Dliteratum%253Aachs%26aulast%3DBarab%25C3%25A1si%26aufirst%3DA.-L.%26atitle%3DNetwork%2520Medicine%2520%25E2%2580%2594%2520From%2520Obesity%2520to%2520the%2520%25E2%2580%259CDiseasome%25E2%2580%259D%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2007%26volume%3D357%26spage%3D404%26epage%3D407%26doi%3D10.1056%2FNEJMe078114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goh, K.,  II</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cusick, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valle, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barabási, A. L.</span></span> <span> </span><span class="NLM_article-title">The Human Disease Network</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">8685</span>– <span class="NLM_lpage">8690</span>, <span class="refDoi"> DOI: 10.1073/pnas.0701361104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1073%2Fpnas.0701361104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=17502601" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmt1Wisbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=8685-8690&author=K.+Gohauthor=M.+E.+Cusickauthor=D.+Valleauthor=B.+Childsauthor=M.+Vidalauthor=A.+L.+Barab%C3%A1si&title=The+Human+Disease+Network&doi=10.1073%2Fpnas.0701361104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">The human disease network</span></div><div class="casAuthors">Goh, Kwang-Il; Cusick, Michael E.; Valle, David; Childs, Barton; Vidal, Marc; Barabasi, Albert-Laszlo</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8685-8690</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">A network of disorders and disease genes linked by known disorder-gene assocns. offers a platform to explore in a single graph-theoretic framework all known phenotype and disease gene assocns., indicating the common genetic origin of many diseases.  Genes assocd. with similar disorders show both higher likelihood of phys. interactions between their products and higher expression profiling similarity for their transcripts, supporting the existence of distinct disease-specific functional modules.  Essential human genes are found to likely encode hub proteins and are expressed widely in most tissues.  This suggests that disease genes also would play a central role in the human interactome.  In contrast, the vast majority of disease genes are nonessential and show no tendency to encode hub proteins, and their expression pattern indicates that they are localized in the functional periphery of the network.  A selection-based model explains the obsd. difference between essential and disease genes and also suggests that diseases caused by somatic mutations should not be peripheral, a prediction confirmed for cancer genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwE7npsmfFXbVg90H21EOLACvtfcHk0licSCCtb27Csg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmt1Wisbg%253D&md5=58e78ecce834988a70c44c73c548a59b</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0701361104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0701361104%26sid%3Dliteratum%253Aachs%26aulast%3DGoh%26aufirst%3DK.%26aulast%3DCusick%26aufirst%3DM.%2BE.%26aulast%3DValle%26aufirst%3DD.%26aulast%3DChilds%26aufirst%3DB.%26aulast%3DVidal%26aufirst%3DM.%26aulast%3DBarab%25C3%25A1si%26aufirst%3DA.%2BL.%26atitle%3DThe%2520Human%2520Disease%2520Network%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D8685%26epage%3D8690%26doi%3D10.1073%2Fpnas.0701361104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vidal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cusick, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barabási, A. L.</span></span> <span> </span><span class="NLM_article-title">Interactome Networks and Human Disease</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">986</span>– <span class="NLM_lpage">998</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2011.02.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1016%2Fj.cell.2011.02.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=21414488" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsFCrtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2011&pages=986-998&author=M.+Vidalauthor=M.+E.+Cusickauthor=A.+L.+Barab%C3%A1si&title=Interactome+Networks+and+Human+Disease&doi=10.1016%2Fj.cell.2011.02.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Interactome Networks and Human Disease</span></div><div class="casAuthors">Vidal, Marc; Cusick, Michael E.; Barabasi, Albert-Laszlo</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">986-998</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Complex biol. systems and cellular networks may underlie most genotype to phenotype relationships.  Here, we review basic concepts in network biol., discussing different types of interactome networks and the insights that can come from analyzing them.  We elaborate on why interactome networks are important to consider in biol., how they can be mapped and integrated with each other, what global properties are starting to emerge from interactome network models, and how these properties may relate to human disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7qTUCJpaslbVg90H21EOLACvtfcHk0licSCCtb27Csg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsFCrtrs%253D&md5=19cb64f69574390843eb2cc969f4a0cb</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.02.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.02.016%26sid%3Dliteratum%253Aachs%26aulast%3DVidal%26aufirst%3DM.%26aulast%3DCusick%26aufirst%3DM.%2BE.%26aulast%3DBarab%25C3%25A1si%26aufirst%3DA.%2BL.%26atitle%3DInteractome%2520Networks%2520and%2520Human%2520Disease%26jtitle%3DCell%26date%3D2011%26volume%3D144%26spage%3D986%26epage%3D998%26doi%3D10.1016%2Fj.cell.2011.02.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vidal, M.</span></span> <span> </span><span class="NLM_article-title">A Unifying View of 21st Century Systems Biology</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>583</i></span>,  <span class="NLM_fpage">3891</span>– <span class="NLM_lpage">3894</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2009.11.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1016%2Fj.febslet.2009.11.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=19913537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFams7nJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=583&publication_year=2009&pages=3891-3894&author=M.+Vidal&title=A+Unifying+View+of+21st+Century+Systems+Biology&doi=10.1016%2Fj.febslet.2009.11.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">A unifying view of 21st century systems biology</span></div><div class="casAuthors">Vidal, Marc</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">583</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">3891-3894</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The idea that multi-scale dynamic complex systems formed by interacting macromols. and metabolites, cells, organs and organisms underlie some of the most fundamental aspects of life was proposed by a few visionaries half a century ago.  Scientists are witnessing a powerful resurgence of this idea made possible by the availability of nearly complete genome sequences, ever improving gene annotations and interactome network maps, the development of sophisticated informatic and imaging tools, and importantly, and the use of engineering and physics concepts such as control and graph theory.  Alongside 4 other fundamental "great ideas" as suggested by Sir Paul Nurse (P. Nurse, 2003), namely, the gene, the cell, the role of chem. in biol. processes, and evolution by natural selection, systems-level understanding of "what is life" may materialize as one of the major ideas of biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpc-z3XLH_jWbVg90H21EOLACvtfcHk0ljHcnhyHt_BrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFams7nJ&md5=44fb47f2892d2eafc357b8efc0fba2f9</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2009.11.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2009.11.024%26sid%3Dliteratum%253Aachs%26aulast%3DVidal%26aufirst%3DM.%26atitle%3DA%2520Unifying%2520View%2520of%252021st%2520Century%2520Systems%2520Biology%26jtitle%3DFEBS%2520Lett.%26date%3D2009%26volume%3D583%26spage%3D3891%26epage%3D3894%26doi%3D10.1016%2Fj.febslet.2009.11.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paolini, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapland, R. H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van
Hoorn, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span> <span> </span><span class="NLM_article-title">Global Mapping of Pharmacological Space</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">805</span>– <span class="NLM_lpage">815</span>, <span class="refDoi"> DOI: 10.1038/nbt1228</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1038%2Fnbt1228" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=16841068" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BD28XmvFansb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2006&pages=805-815&author=G.+V.+Paoliniauthor=R.+H.+B.+Shaplandauthor=W.+P.+Van%0AHoornauthor=J.+S.+Masonauthor=A.+L.+Hopkins&title=Global+Mapping+of+Pharmacological+Space&doi=10.1038%2Fnbt1228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Global mapping of pharmacological space</span></div><div class="casAuthors">Paolini, Gaia V.; Shapland, Richard H. B.; van Hoorn, Willem P.; Mason, Jonathan S.; Hopkins, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">805-815</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We present the global mapping of pharmacol. space by the integration of several vast sources of medicinal chem. structure-activity relationships (SAR) data.  Our comprehensive mapping of pharmacol. space enables us to identify confidently the human targets for which chem. tools and drugs have been discovered to date.  The integration of SAR data from diverse sources by unique canonical chem. structure, protein sequence and disease indication enables the construction of a ligand-target matrix to explore the global relationships between chem. structure and biol. targets.  Using the data matrix, we are able to catalog the links between proteins in chem. space as a polypharmacol. interaction network.  We demonstrate that probabilistic models can be used to predict pharmacol. from a large knowledge base.  The relationships between proteins, chem. structures and drug-like properties provide a framework for developing a probabilistic approach to drug discovery that can be exploited to increase research productivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoV_CE47zZq8LVg90H21EOLACvtfcHk0ljHcnhyHt_BrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmvFansb8%253D&md5=f559b34692cc903a1b503deb07030c5d</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1038%2Fnbt1228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1228%26sid%3Dliteratum%253Aachs%26aulast%3DPaolini%26aufirst%3DG.%2BV.%26aulast%3DShapland%26aufirst%3DR.%2BH.%2BB.%26aulast%3DVan%2BHoorn%26aufirst%3DW.%2BP.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DGlobal%2520Mapping%2520of%2520Pharmacological%2520Space%26jtitle%3DNat.%2520Biotechnol.%26date%3D2006%26volume%3D24%26spage%3D805%26epage%3D815%26doi%3D10.1038%2Fnbt1228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yildirim, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goh, K.,  II</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cusick, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barabási, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidal, M.</span></span> <span> </span><span class="NLM_article-title">Drug-Target Network</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1119</span>– <span class="NLM_lpage">1126</span>, <span class="refDoi"> DOI: 10.1038/nbt1338</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1038%2Fnbt1338" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=17921997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFagt7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=1119-1126&author=M.+A.+Yildirimauthor=K.+Gohauthor=M.+E.+Cusickauthor=A.+L.+Barab%C3%A1siauthor=M.+Vidal&title=Drug-Target+Network&doi=10.1038%2Fnbt1338"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-target network</span></div><div class="casAuthors">Yildirim, Muhammed A.; Goh, Kwang-Il; Cusick, Michael E.; Barabasi, Albert-Laszlo; Vidal, Marc</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1119-1126</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The global set of relationships between protein targets of all drugs and all disease-gene products in the human protein-protein interaction or 'interactome' network remains uncharacterized.  We built a bipartite graph composed of US Food and Drug Administration-approved drugs and proteins linked by drug-target binary assocns.  The resulting network connects most drugs into a highly interlinked giant component, with strong local clustering of drugs of similar types according to Anatomical Therapeutic Chem. classification.  Topol. analyses of this network quant. showed an overabundance of 'follow-on' drugs, i.e., drugs that target already targeted proteins.  By including drugs currently under investigation, we identified a trend toward more functionally diverse targets improving polypharmacol.  To analyze the relationships between drug targets and disease-gene products, we measured the shortest distance between both sets of proteins in current models of the human interactome network.  Significant differences in distance were found between etiol. and palliative drugs.  A recent trend toward more rational drug design was obsd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2KaaiDmAQ0LVg90H21EOLACvtfcHk0ljHcnhyHt_BrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFagt7rO&md5=d669b6a7afed3d9fb9d173e53b4ebead</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1038%2Fnbt1338&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1338%26sid%3Dliteratum%253Aachs%26aulast%3DYildirim%26aufirst%3DM.%2BA.%26aulast%3DGoh%26aufirst%3DK.%26aulast%3DCusick%26aufirst%3DM.%2BE.%26aulast%3DBarab%25C3%25A1si%26aufirst%3DA.%2BL.%26aulast%3DVidal%26aufirst%3DM.%26atitle%3DDrug-Target%2520Network%26jtitle%3DNat.%2520Biotechnol.%26date%3D2007%26volume%3D25%26spage%3D1119%26epage%3D1126%26doi%3D10.1038%2Fnbt1338" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guney, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menche, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barábasi, A. L.</span></span> <span> </span><span class="NLM_article-title">Network-Based in Silico Drug Efficacy Screening</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">10331</span>, <span class="refDoi"> DOI: 10.1038/ncomms10331</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1038%2Fncomms10331" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=26831545" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslOrsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=10331&author=E.+Guneyauthor=J.+Mencheauthor=M.+Vidalauthor=A.+L.+Bar%C3%A1basi&title=Network-Based+in+Silico+Drug+Efficacy+Screening&doi=10.1038%2Fncomms10331"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Network-based in silico drug efficacy screening</span></div><div class="casAuthors">Guney, Emre; Menche, Jorg; Vidal, Marc; Barabasi, Albert-Laszlo</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10331</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The increasing cost of drug development together with a significant drop in the no. of new drug approvals raises the need for innovative approaches for target identification and efficacy prediction.  Here, we take advantage of our increasing understanding of the network-based origins of diseases to introduce a drug-disease proximity measure that quantifies the interplay between drugs targets and diseases.  By correcting for the known biases of the interactome, proximity helps us uncover the therapeutic effect of drugs, as well as to distinguish palliative from effective treatments.  Our anal. of 238 drugs used in 78 diseases indicates that the therapeutic effect of drugs is localized in a small network neighborhood of the disease genes and highlights efficacy issues for drugs used in Parkinson and several inflammatory disorders.  Finally, network-based proximity allows us to predict novel drug-disease assocns. that offer unprecedented opportunities for drug repurposing and the detection of adverse effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqV7iKm7E1M9rVg90H21EOLACvtfcHk0ljHcnhyHt_BrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslOrsL4%253D&md5=125a8074aa7a953e35882d509731b49d</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1038%2Fncomms10331&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms10331%26sid%3Dliteratum%253Aachs%26aulast%3DGuney%26aufirst%3DE.%26aulast%3DMenche%26aufirst%3DJ.%26aulast%3DVidal%26aufirst%3DM.%26aulast%3DBar%25C3%25A1basi%26aufirst%3DA.%2BL.%26atitle%3DNetwork-Based%2520in%2520Silico%2520Drug%2520Efficacy%2520Screening%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D10331%26doi%3D10.1038%2Fncomms10331" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovács, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barabási, A. L.</span></span> <span> </span><span class="NLM_article-title">Network-Based Prediction of Drug Combinations</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1197</span>, <span class="refDoi"> DOI: 10.1038/s41467-019-09186-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1038%2Fs41467-019-09186-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=30867426" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A280%3ADC%252BB3cbjslelsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1197&author=F.+Chengauthor=I.+A.+Kov%C3%A1csauthor=A.+L.+Barab%C3%A1si&title=Network-Based+Prediction+of+Drug+Combinations&doi=10.1038%2Fs41467-019-09186-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Network-based prediction of drug combinations</span></div><div class="casAuthors">Cheng Feixiong; Kovacs Istvan A; Barabasi Albert-Laszlo; Cheng Feixiong; Kovacs Istvan A; Barabasi Albert-Laszlo; Cheng Feixiong; Cheng Feixiong; Cheng Feixiong; Barabasi Albert-Laszlo; Barabasi Albert-Laszlo</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1197</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Drug combinations, offering increased therapeutic efficacy and reduced toxicity, play an important role in treating multiple complex diseases.  Yet, our ability to identify and validate effective combinations is limited by a combinatorial explosion, driven by both the large number of drug pairs as well as dosage combinations.  Here we propose a network-based methodology to identify clinically efficacious drug combinations for specific diseases.  By quantifying the network-based relationship between drug targets and disease proteins in the human protein-protein interactome, we show the existence of six distinct classes of drug-drug-disease combinations.  Relying on approved drug combinations for hypertension and cancer, we find that only one of the six classes correlates with therapeutic effects: if the targets of the drugs both hit disease module, but target separate neighborhoods.  This finding allows us to identify and validate antihypertensive combinations, offering a generic, powerful network methodology to identify efficacious combination therapies in drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT3gy6XkP0VYS6Ufuk_1xoTfW6udTcc2eYhox3UwSa7Brntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cbjslelsg%253D%253D&md5=bb39b64bbd05af1c25f99263a61f622f</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1038%2Fs41467-019-09186-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-019-09186-x%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DF.%26aulast%3DKov%25C3%25A1cs%26aufirst%3DI.%2BA.%26aulast%3DBarab%25C3%25A1si%26aufirst%3DA.%2BL.%26atitle%3DNetwork-Based%2520Prediction%2520of%2520Drug%2520Combinations%26jtitle%3DNat.%2520Commun.%26date%3D2019%26volume%3D10%26spage%3D1197%26doi%3D10.1038%2Fs41467-019-09186-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Conte, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiscon, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Licursi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bizzarri, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Antò, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farina, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paci, P.</span></span> <span> </span><span class="NLM_article-title">A Paradigm Shift in Medicine: A Comprehensive Review of Network-Based Approaches</span>. <i>Biochim. Biophys. Acta, Gene Regul. Mech.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>1863</i></span>,  <span class="NLM_fpage">194416</span>, <span class="refDoi"> DOI: 10.1016/j.bbagrm.2019.194416</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1016%2Fj.bbagrm.2019.194416" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=31382052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFGnsbzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1863&publication_year=2020&pages=194416&author=F.+Conteauthor=G.+Fisconauthor=V.+Licursiauthor=D.+Bizzarriauthor=T.+D%E2%80%99Ant%C3%B2author=L.+Farinaauthor=P.+Paci&title=A+Paradigm+Shift+in+Medicine%3A+A+Comprehensive+Review+of+Network-Based+Approaches&doi=10.1016%2Fj.bbagrm.2019.194416"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">A paradigm shift in medicine: A comprehensive review of network-based approaches</span></div><div class="casAuthors">Conte, Federica; Fiscon, Giulia; Licursi, Valerio; Bizzarri, Daniele; D'Anto, Tommaso; Farina, Lorenzo; Paci, Paola</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">1863</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">194416</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Network medicine is a rapidly evolving new field of medical research, which combines principles and approaches of systems biol. and network science, holding the promise to uncovering the causes and to revolutionize the diagnosis and treatments of human diseases.  This new paradigm reflects the fact that human diseases are not caused by single mol. defects, but driven by complex interactions among a variety of mol. mediators.  The complexity of these interactions embraces different types of information: from the cellular-mol. level of protein-protein interactions to correlational studies of gene expression and regulation, to metabolic and disease pathways up to drug-disease relationships.  The anal. of these complex networks can reveal new disease genes and/or disease pathways and identify possible targets for new drug development, as well as new uses for existing drugs.  In this review, we offer a comprehensive overview of network types and algorithms used in the framework of network medicine.  This article is part of a Special Issue entitled: Transcriptional Profiles and Regulatory Gene Networks edited by Dr.  Dr. Federico Manuel Giorgi and Dr.  Shaun Mahony.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrg_nwVOWC0-rVg90H21EOLACvtfcHk0liR227tMrPgCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFGnsbzO&md5=22ba9de49696544d6cbd8fbcb3ceafde</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2019.194416&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2019.194416%26sid%3Dliteratum%253Aachs%26aulast%3DConte%26aufirst%3DF.%26aulast%3DFiscon%26aufirst%3DG.%26aulast%3DLicursi%26aufirst%3DV.%26aulast%3DBizzarri%26aufirst%3DD.%26aulast%3DD%25E2%2580%2599Ant%25C3%25B2%26aufirst%3DT.%26aulast%3DFarina%26aufirst%3DL.%26aulast%3DPaci%26aufirst%3DP.%26atitle%3DA%2520Paradigm%2520Shift%2520in%2520Medicine%253A%2520A%2520Comprehensive%2520Review%2520of%2520Network-Based%2520Approaches%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gene%2520Regul.%2520Mech.%26date%3D2020%26volume%3D1863%26spage%3D194416%26doi%3D10.1016%2Fj.bbagrm.2019.194416" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sonawane, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glass, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, A.</span></span> <span> </span><span class="NLM_article-title">Network Medicine in the Age of Biomedical Big Data</span>. <i>Front. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">294</span>, <span class="refDoi"> DOI: 10.3389/fgene.2019.00294</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.3389%2Ffgene.2019.00294" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=31031797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlOnurbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=294&author=A.+R.+Sonawaneauthor=S.+T.+Weissauthor=K.+Glassauthor=A.+Sharma&title=Network+Medicine+in+the+Age+of+Biomedical+Big+Data&doi=10.3389%2Ffgene.2019.00294"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Network medicine in the age of biomedical big data</span></div><div class="casAuthors">Sonawane, Abhijeet R.; Weiss, Scott T.; Glass, Kimberly; Sharma, Amitabh</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Genetics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">294</span>CODEN:
                <span class="NLM_cas:coden">FGREBR</span>;
        ISSN:<span class="NLM_cas:issn">1664-8021</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">Network medicine is an emerging area of research dealing with mol. and genetic interactions, network biomarkers of disease, and therapeutic target discovery.  Largescale biomedical data generation offers a unique opportunity to assess the effect and impact of cellular heterogeneity and environmental perturbations on the obsd. phenotype.  Marrying the two, network medicine with biomedical data provides a framework to build meaningful models and ext. impactful results at a network level.  In this review, we survey existing network types and biomedical data sources.  More importantly, we delve into ways in which the network medicine approach, aided by phenotype-specific biomedical data, can be gainfully applied.  We provide three paradigms, mainly dealing with three major biol. network archetypes: proteinprotein interaction, expression-based, and gene regulatory networks.  For each of these paradigms, we discuss a broad overview of philosophies under which various network methods work.  We also provide a few examples in each paradigm as a test case of its successful application.  Finally, we delineate several opportunities and challenges in the field of network medicine.  We hope this review provides a lexicon for researchers from biol. sciences and network theory to come on the same page to work on research areas that require interdisciplinary expertise.  Taken together, the understanding gained from combining biomedical data with networks can be useful for characterizing disease etiologies and identifying therapeutic targets, which, in turn, will lead to better preventive medicine with translational impact on personalized healthcare.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2V3pM6h97y7Vg90H21EOLACvtfcHk0liR227tMrPgCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlOnurbK&md5=175633a8375ad95cef9c7c7cb9ee2f81</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.3389%2Ffgene.2019.00294&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffgene.2019.00294%26sid%3Dliteratum%253Aachs%26aulast%3DSonawane%26aufirst%3DA.%2BR.%26aulast%3DWeiss%26aufirst%3DS.%2BT.%26aulast%3DGlass%26aufirst%3DK.%26aulast%3DSharma%26aufirst%3DA.%26atitle%3DNetwork%2520Medicine%2520in%2520the%2520Age%2520of%2520Biomedical%2520Big%2520Data%26jtitle%3DFront.%2520Genet.%26date%3D2019%26volume%3D10%26spage%3D294%26doi%3D10.3389%2Ffgene.2019.00294" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Squartini, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldarelli, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cimini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabrielli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garlaschelli, D.</span></span> <span> </span><span class="NLM_article-title">Reconstruction Methods for Networks: The Case of Economic and Financial Systems</span>. <i>Phys. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>757</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">47</span>, <span class="refDoi"> DOI: 10.1016/j.physrep.2018.06.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1016%2Fj.physrep.2018.06.008" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=757&publication_year=2018&pages=1-47&author=T.+Squartiniauthor=G.+Caldarelliauthor=G.+Ciminiauthor=A.+Gabrielliauthor=D.+Garlaschelli&title=Reconstruction+Methods+for+Networks%3A+The+Case+of+Economic+and+Financial+Systems&doi=10.1016%2Fj.physrep.2018.06.008"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1016%2Fj.physrep.2018.06.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.physrep.2018.06.008%26sid%3Dliteratum%253Aachs%26aulast%3DSquartini%26aufirst%3DT.%26aulast%3DCaldarelli%26aufirst%3DG.%26aulast%3DCimini%26aufirst%3DG.%26aulast%3DGabrielli%26aufirst%3DA.%26aulast%3DGarlaschelli%26aufirst%3DD.%26atitle%3DReconstruction%2520Methods%2520for%2520Networks%253A%2520The%2520Case%2520of%2520Economic%2520and%2520Financial%2520Systems%26jtitle%3DPhys.%2520Rep.%26date%3D2018%26volume%3D757%26spage%3D1%26epage%3D47%26doi%3D10.1016%2Fj.physrep.2018.06.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Albert, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barabási, A. L.</span></span> <span> </span><span class="NLM_article-title">Statistical Mechanics of Complex Networks</span>. <i>Rev. Mod. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>74</i></span>,  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">97</span>, <span class="refDoi"> DOI: 10.1103/RevModPhys.74.47</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1103%2FRevModPhys.74.47" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2002&pages=47-97&author=R.+Albertauthor=A.+L.+Barab%C3%A1si&title=Statistical+Mechanics+of+Complex+Networks&doi=10.1103%2FRevModPhys.74.47"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1103%2FRevModPhys.74.47&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FRevModPhys.74.47%26sid%3Dliteratum%253Aachs%26aulast%3DAlbert%26aufirst%3DR.%26aulast%3DBarab%25C3%25A1si%26aufirst%3DA.%2BL.%26atitle%3DStatistical%2520Mechanics%2520of%2520Complex%2520Networks%26jtitle%3DRev.%2520Mod.%2520Phys.%26date%3D2002%26volume%3D74%26spage%3D47%26epage%3D97%26doi%3D10.1103%2FRevModPhys.74.47" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lü, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, T.</span></span> <span> </span><span class="NLM_article-title">Link Prediction in Complex Networks: A Survey</span>. <i>Phys. A</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>390</i></span>,  <span class="NLM_fpage">1150</span>– <span class="NLM_lpage">1170</span>, <span class="refDoi"> DOI: 10.1016/j.physa.2010.11.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1016%2Fj.physa.2010.11.027" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=390&publication_year=2011&pages=1150-1170&author=L.+L%C3%BCauthor=T.+Zhou&title=Link+Prediction+in+Complex+Networks%3A+A+Survey&doi=10.1016%2Fj.physa.2010.11.027"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1016%2Fj.physa.2010.11.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.physa.2010.11.027%26sid%3Dliteratum%253Aachs%26aulast%3DL%25C3%25BC%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DT.%26atitle%3DLink%2520Prediction%2520in%2520Complex%2520Networks%253A%2520A%2520Survey%26jtitle%3DPhys.%2520A%26date%3D2011%26volume%3D390%26spage%3D1150%26epage%3D1170%26doi%3D10.1016%2Fj.physa.2010.11.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ozturk, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dow, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlin, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bejar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, H.</span></span> <span> </span><span class="NLM_article-title">The Emerging Potential for Network Analysis to Inform Precision Cancer Medicine</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>430</i></span>,  <span class="NLM_fpage">2875</span>– <span class="NLM_lpage">2899</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2018.06.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1016%2Fj.jmb.2018.06.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=29908887" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFOku77O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=430&publication_year=2018&pages=2875-2899&author=K.+Ozturkauthor=M.+Dowauthor=D.+E.+Carlinauthor=R.+Bejarauthor=H.+Carter&title=The+Emerging+Potential+for+Network+Analysis+to+Inform+Precision+Cancer+Medicine&doi=10.1016%2Fj.jmb.2018.06.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">The Emerging Potential for Network Analysis to Inform Precision Cancer Medicine</span></div><div class="casAuthors">Ozturk, Kivilcim; Dow, Michelle; Carlin, Daniel E.; Bejar, Rafael; Carter, Hannah</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">430</span>
        (<span class="NLM_cas:issue">18_Part_A</span>),
    <span class="NLM_cas:pages">2875-2899</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Precision cancer medicine promises to tailor clin. decisions to patients using genomic information.  Indeed, successes of drugs targeting genetic alterations in tumors, such as imatinib that targets BCR-ABL in chronic myelogenous leukemia, have demonstrated the power of this approach.  However, biol. systems are complex, and patients may differ not only by the specific genetic alterations in their tumor, but also by more subtle interactions among such alterations.  Systems biol. and more specifically, network anal., provides a framework for advancing precision medicine beyond clin. actionability of individual mutations.  Here we discuss applications of network anal. to study tumor biol., early methods for N-of-1 tumor genome anal., and the path for such tools to the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSIwj-eGOs9LVg90H21EOLACvtfcHk0lgdyl1nWvKIjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFOku77O&md5=44ce88c86a6879a23b8a37f42a966c36</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2018.06.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2018.06.016%26sid%3Dliteratum%253Aachs%26aulast%3DOzturk%26aufirst%3DK.%26aulast%3DDow%26aufirst%3DM.%26aulast%3DCarlin%26aufirst%3DD.%2BE.%26aulast%3DBejar%26aufirst%3DR.%26aulast%3DCarter%26aufirst%3DH.%26atitle%3DThe%2520Emerging%2520Potential%2520for%2520Network%2520Analysis%2520to%2520Inform%2520Precision%2520Cancer%2520Medicine%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2018%26volume%3D430%26spage%3D2875%26epage%3D2899%26doi%3D10.1016%2Fj.jmb.2018.06.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vilar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hripcsak, G.</span></span> <span> </span><span class="NLM_article-title">The Role of Drug Profiles as Similarity Metrics: Applications to Repurposing, Adverse Effects Detection and Drug – Drug Interactions</span>. <i>Briefings Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">670</span>– <span class="NLM_lpage">681</span>, <span class="refDoi"> DOI: 10.1093/bib/bbw048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fbib%2Fbbw048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=27273288" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjsVyqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=670-681&author=S.+Vilarauthor=G.+Hripcsak&title=The+Role+of+Drug+Profiles+as+Similarity+Metrics%3A+Applications+to+Repurposing%2C+Adverse+Effects+Detection+and+Drug+%E2%80%93+Drug+Interactions&doi=10.1093%2Fbib%2Fbbw048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">The role of drug profiles as similarity metrics: applications to repurposing, adverse effects detection and drug-drug interactions</span></div><div class="casAuthors">Vilar, Santiago; Hripcsak, George</div><div class="citationInfo"><span class="NLM_cas:title">Briefings in Bioinformatics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">670-681</span>CODEN:
                <span class="NLM_cas:coden">BBIMFX</span>;
        ISSN:<span class="NLM_cas:issn">1477-4054</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Explosion of the availability of big data sources along with the development in computational methods provides a useful framework to study drugs' actions, such as interactions with pharmacol. targets and off-targets.  Databases related to protein interactions, adverse effects and genomic profiles are available to be used for the construction of computational models.  In this article, we focus on the description of biol. profiles for drugs that can be used as a system to compare similarity and create methods to predict and analyze drugs' actions.  We highlight profiles constructed with different biol. data, such as target-protein interactions, gene expression measurements, adverse effects and disease profiles.  We focus on the discovery of new targets or pathways for drugs already in the pharmaceutical market, also called drug repurposing, in the interaction with off-targets responsible for adverse reactions and in drug-drug interaction anal.  The current and future applications, strengths and challenges facing all these methods are also discussed.  Biol. profiles or signatures are an important source of data generation to deeply analyze biol. actions with important implications in drug-related studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoY7SlptWRX_7Vg90H21EOLACvtfcHk0lgdyl1nWvKIjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjsVyqsLs%253D&md5=53a1cda1967ba1da99b30494215b5db5</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1093%2Fbib%2Fbbw048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbib%252Fbbw048%26sid%3Dliteratum%253Aachs%26aulast%3DVilar%26aufirst%3DS.%26aulast%3DHripcsak%26aufirst%3DG.%26atitle%3DThe%2520Role%2520of%2520Drug%2520Profiles%2520as%2520Similarity%2520Metrics%253A%2520Applications%2520to%2520Repurposing%252C%2520Adverse%2520Effects%2520Detection%2520and%2520Drug%2520%25E2%2580%2593%2520Drug%2520Interactions%26jtitle%3DBriefings%2520Bioinf.%26date%3D2017%26volume%3D18%26spage%3D670%26epage%3D681%26doi%3D10.1093%2Fbib%2Fbbw048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Y.</span></span> <span> </span><span class="NLM_article-title">Prediction of Drug-Target Interactions and Drug Repositioning via Network-Based Inference</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e1002503</span> <span class="refDoi"> DOI: 10.1371/journal.pcbi.1002503</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1371%2Fjournal.pcbi.1002503" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=22589709" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC38Xns1GntbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&author=F.+Chengauthor=C.+Liuauthor=J.+Jiangauthor=W.+Luauthor=W.+Liauthor=G.+Liuauthor=W.+Zhouauthor=J.+Huangauthor=Y.+Tang&title=Prediction+of+Drug-Target+Interactions+and+Drug+Repositioning+via+Network-Based+Inference&doi=10.1371%2Fjournal.pcbi.1002503"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of drug-target interactions and drug repositioning via network-based inference</span></div><div class="casAuthors">Cheng, Feixiong; Liu, Chuang; Jiang, Jing; Lu, Weiqiang; Li, Weihua; Liu, Guixia; Zhou, Weixing; Huang, Jin; Tang, Yun</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e1002503</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Drug-target interaction (DTI) is the basis of drug discovery and design.  It is time consuming and costly to det. DTI exptl.  Hence, it is necessary to develop computational methods for the prediction of potential DTI.  Based on complex network theory, three supervised inference methods were developed here to predict DTI and used for drug repositioning, namely drug-based similarity inference (DBSI), target-based similarity inference (TBSI) and network-based inference (NBI).  Among them, NBI performed best on four benchmark data sets.  Then a drug-target network was created with NBI based on 12,483 FDA-approved and exptl. drug-target binary links, and some new DTIs were further predicted.  In vitro assays confirmed that five old drugs, namely montelukast, diclofenac, simvastatin, ketoconazole, and itraconazole, showed polypharmacol. features on estrogen receptors or dipeptidyl peptidase-IV with half maximal inhibitory or effective concn. ranged from 0.2 to 10 μM.  Moreover, simvastatin and ketoconazole showed potent antiproliferative activities on human MDA-MB-231 breast cancer cell line in MTT assays.  The results indicated that these methods could be powerful tools in prediction of DTIs and drug repositioning.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvYLlyS_-BBLVg90H21EOLACvtfcHk0lhzAWw9FM05Og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xns1GntbY%253D&md5=d98a7828bebbdf981ebb14d45cc4102a</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1002503&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1002503%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DJiang%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DTang%26aufirst%3DY.%26atitle%3DPrediction%2520of%2520Drug-Target%2520Interactions%2520and%2520Drug%2520Repositioning%2520via%2520Network-Based%2520Inference%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2012%26volume%3D8%26doi%3D10.1371%2Fjournal.pcbi.1002503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, W. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, X. Y.</span></span> <span> </span><span class="NLM_article-title">Identification of Drug-Target Interaction from Interactome Network with “guilt-by-Association” Principle and Topology Features</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">1057</span>– <span class="NLM_lpage">1064</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btv695</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fbioinformatics%2Fbtv695" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=26614126" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlWiurvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2016&pages=1057-1064&author=Z.+C.+Liauthor=M.+H.+Huangauthor=W.+Q.+Zhongauthor=Z.+Q.+Liuauthor=Y.+Xieauthor=Z.+Daiauthor=X.+Y.+Zou&title=Identification+of+Drug-Target+Interaction+from+Interactome+Network+with+%E2%80%9Cguilt-by-Association%E2%80%9D+Principle+and+Topology+Features&doi=10.1093%2Fbioinformatics%2Fbtv695"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of drug-target interaction from interactome network with 'guilt-by-association' principle and topology features</span></div><div class="casAuthors">Li, Zhan-Chao; Huang, Meng-Hua; Zhong, Wen-Qian; Liu, Zhi-Qing; Xie, Yun; Dai, Zong; Zou, Xiao-Yong</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1057-1064</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: Identifying drug-target protein interaction is a crucial step in the process of drug research and development.  Wet-lab expt. are laborious, time-consuming and expensive.  Hence, there is a strong demand for the development of a novel theor. method to identify potential interaction between drug and target protein.  Results: We use all known proteins and drugs to construct a nodes- and edges-weighted biol. relevant interactome network.  On the basis of the 'guilt-by-assocn.' principle, novel network topol. features are proposed to characterize interaction pairs and random forest algorithm is employed to identify potential drug-protein interaction.  Accuracy of 92.53% derived from the 10-fold cross-validation is about 10% higher than that of the existing method.  We identify 2272 potential drug-target interactions, some of which are assocd. with diseases, such as Torg-Winchester syndrome and rhabdomyosarcoma.  The proposed method can not only accurately predict the interaction between drug mol. and target protein, but also help disease treatment and drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqy9I_thc-a57Vg90H21EOLACvtfcHk0lhzAWw9FM05Og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlWiurvK&md5=5f91b24a29299c9e26be83e3275d6f3a</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtv695&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtv695%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DZ.%2BC.%26aulast%3DHuang%26aufirst%3DM.%2BH.%26aulast%3DZhong%26aufirst%3DW.%2BQ.%26aulast%3DLiu%26aufirst%3DZ.%2BQ.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DDai%26aufirst%3DZ.%26aulast%3DZou%26aufirst%3DX.%2BY.%26atitle%3DIdentification%2520of%2520Drug-Target%2520Interaction%2520from%2520Interactome%2520Network%2520with%2520%25E2%2580%259Cguilt-by-Association%25E2%2580%259D%2520Principle%2520and%2520Topology%2520Features%26jtitle%3DBioinformatics%26date%3D2016%26volume%3D32%26spage%3D1057%26epage%3D1064%26doi%3D10.1093%2Fbioinformatics%2Fbtv695" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nam, H.</span></span> <span> </span><span class="NLM_article-title">Identification of Drug-Target Interaction by a Random Walk with Restart Method on an Interactome Network</span>. <i>BMC Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">208</span>, <span class="refDoi"> DOI: 10.1186/s12859-018-2199-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1186%2Fs12859-018-2199-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=29897326" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtlGlt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2018&pages=208&author=I.+Leeauthor=H.+Nam&title=Identification+of+Drug-Target+Interaction+by+a+Random+Walk+with+Restart+Method+on+an+Interactome+Network&doi=10.1186%2Fs12859-018-2199-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of drug-target interaction by a random walk with restart method on an interactome network</span></div><div class="casAuthors">Lee, Ingoo; Nam, Hojung</div><div class="citationInfo"><span class="NLM_cas:title">BMC Bioinformatics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">Suppl.8</span>),
    <span class="NLM_cas:pages">208/1-208/10</span>CODEN:
                <span class="NLM_cas:coden">BBMIC4</span>;
        ISSN:<span class="NLM_cas:issn">1471-2105</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Identification of drug-target interactions acts as a key role in drug discovery.  However, identifying drug-target interactions via in-vitro, in-vivo expts. are very laborious, time-consuming.  Thus, predicting drug-target interactions by using computational approaches is a good alternative.  In recent studies, many feature-based and similarity-based machine learning approaches have shown promising results in drug-target interaction predictions.  A previous study showed that accounting connectivity information of drug-drug and protein-protein interactions increase performances of prediction by the concept of 'guilt-by-assocn.'.  However, the approach that only considers directly connected nodes often misses the information that could be derived from distance nodes.  Therefore, in this study, we yield global network topol. information by using a random walk with restart algorithm and apply the global topol. information to the prediction model.  Results: As a result, our prediction model demonstrates increased prediction performance compare to the 'guilt-by-assocn.' approach (AUC 0.89 and 0.67 in the training and independent test, resp.).  In addn., we show how weighted features by a random walk with restart yields better performances than original features.  Also, we confirmed that drugs and proteins that have high-degree of connectivity on the interactome network yield better performance in our model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgyLrIPIGCCLVg90H21EOLACvtfcHk0lhzAWw9FM05Og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtlGlt7s%253D&md5=278cbc057e2b921e3068d01f912e75d1</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1186%2Fs12859-018-2199-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12859-018-2199-x%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DI.%26aulast%3DNam%26aufirst%3DH.%26atitle%3DIdentification%2520of%2520Drug-Target%2520Interaction%2520by%2520a%2520Random%2520Walk%2520with%2520Restart%2520Method%2520on%2520an%2520Interactome%2520Network%26jtitle%3DBMC%2520Bioinf.%26date%3D2018%26volume%3D19%26spage%3D208%26doi%3D10.1186%2Fs12859-018-2199-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alaimo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giugno, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pulvirenti, A.</span></span> <span> </span><span class="NLM_article-title">Recommendation Techniques for Drug–Target Interaction Prediction and Drug Repositioning</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>1415</i></span>,  <span class="NLM_fpage">441</span>– <span class="NLM_lpage">462</span>, <span class="refDoi"> DOI: 10.1007/978-1-4939-3572-7_23</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1007%2F978-1-4939-3572-7_23" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=27115647" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpsFWktrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1415&publication_year=2016&pages=441-462&author=S.+Alaimoauthor=R.+Giugnoauthor=A.+Pulvirenti&title=Recommendation+Techniques+for+Drug%E2%80%93Target+Interaction+Prediction+and+Drug+Repositioning&doi=10.1007%2F978-1-4939-3572-7_23"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Recommendation Techniques for Drug-Target Interaction Prediction and Drug Repositioning</span></div><div class="casAuthors">Alaimo, Salvatore; Giugno, Rosalba; Pulvirenti, Alfredo</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">1415</span>
        (<span class="NLM_cas:issue">Data Mining Techniques for the Life Sciences</span>),
    <span class="NLM_cas:pages">441-462</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1940-6029</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The usage of computational methods in drug discovery is a common practice.  More recently, by exploiting the wealth of biol. knowledge bases, a novel approach called drug repositioning has raised.  Several computational methods are available, and these try to make a high-level integration of all the knowledge in order to discover unknown mechanisms.  In this chapter, we review drug-target interaction prediction methods based on a recommendation system.  We also give some extensions which go beyond the bipartite network case.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXY4Pjf5JWirVg90H21EOLACvtfcHk0lg9RLieS-2qvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpsFWktrc%253D&md5=e55bd214ca4ed2406679847c396c852c</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-3572-7_23&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4939-3572-7_23%26sid%3Dliteratum%253Aachs%26aulast%3DAlaimo%26aufirst%3DS.%26aulast%3DGiugno%26aufirst%3DR.%26aulast%3DPulvirenti%26aufirst%3DA.%26atitle%3DRecommendation%2520Techniques%2520for%2520Drug%25E2%2580%2593Target%2520Interaction%2520Prediction%2520and%2520Drug%2520Repositioning%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2016%26volume%3D1415%26spage%3D441%26epage%3D462%26doi%3D10.1007%2F978-1-4939-3572-7_23" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cowen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ideker, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raphael, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharan, R.</span></span> <span> </span><span class="NLM_article-title">Network Propagation: A Universal Amplifier of Genetic Associations</span>. <i>Nat. Rev. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">551</span>– <span class="NLM_lpage">562</span>, <span class="refDoi"> DOI: 10.1038/nrg.2017.38</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1038%2Fnrg.2017.38" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=28607512" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC2sXps1Kgur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=551-562&author=L.+Cowenauthor=T.+Idekerauthor=B.+J.+Raphaelauthor=R.+Sharan&title=Network+Propagation%3A+A+Universal+Amplifier+of+Genetic+Associations&doi=10.1038%2Fnrg.2017.38"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Network propagation: a universal amplifier of genetic associations</span></div><div class="casAuthors">Cowen, Lenore; Ideker, Trey; Raphael, Benjamin J.; Sharan, Roded</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">551-562</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Biol. networks are powerful resources for the discovery of genes and genetic modules that drive disease.  Fundamental to network anal. is the concept that genes underlying the same phenotype tend to interact; this principle can be used to combine and to amplify signals from individual genes.  Recently, numerous bioinformatic techniques have been proposed for genetic anal. using networks, based on random walks, information diffusion and elec. resistance.  These approaches have been applied successfully to identify disease genes, genetic modules and drug targets.  In fact, all these approaches are variations of a unifying math. machinery - network propagation - suggesting that it is a powerful data transformation method of broad utility in genetic research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohVG3Mzv2G17Vg90H21EOLACvtfcHk0lg9RLieS-2qvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXps1Kgur4%253D&md5=554a1303016619aaae6e7a76a4722c26</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1038%2Fnrg.2017.38&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg.2017.38%26sid%3Dliteratum%253Aachs%26aulast%3DCowen%26aufirst%3DL.%26aulast%3DIdeker%26aufirst%3DT.%26aulast%3DRaphael%26aufirst%3DB.%2BJ.%26aulast%3DSharan%26aufirst%3DR.%26atitle%3DNetwork%2520Propagation%253A%2520A%2520Universal%2520Amplifier%2520of%2520Genetic%2520Associations%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2017%26volume%3D18%26spage%3D551%26epage%3D562%26doi%3D10.1038%2Fnrg.2017.38" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alaimo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pulvirenti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giugno, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferro, A.</span></span> <span> </span><span class="NLM_article-title">Drug-Target Interaction Prediction through Domain-Tuned Network-Based Inference</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">2004</span>– <span class="NLM_lpage">2008</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btt307</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fbioinformatics%2Fbtt307" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=23720490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFyhtLvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2013&pages=2004-2008&author=S.+Alaimoauthor=A.+Pulvirentiauthor=R.+Giugnoauthor=A.+Ferro&title=Drug-Target+Interaction+Prediction+through+Domain-Tuned+Network-Based+Inference&doi=10.1093%2Fbioinformatics%2Fbtt307"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-target interaction prediction through domain-tuned network-based inference</span></div><div class="casAuthors">Alaimo, Salvatore; Pulvirenti, Alfredo; Giugno, Rosalba; Ferro, Alfredo</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2004-2008</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: The identification of drug-target interaction (DTI) represents a costly and time-consuming step in drug discovery and design.  Computational methods capable of predicting reliable DTI play an important role in the field.  Recently, recommendation methods relying on network-based inference (NBI) have been proposed.  However, such approaches implement naive topol.-based inference and do not take into account important features within the drug-target domain.  Results: In this article, we present a new NBI method, called domain tuned-hybrid (DT-Hybrid), which extends a well-established recommendation technique by domain-based knowledge including drug and target similarity.  DT-Hybrid has been extensively tested using the last version of an exptl. validated DTI database obtained from DrugBank.  Comparison with other recently proposed NBI methods clearly shows that DT-Hybrid is capable of predicting more reliable DTIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohyIWe7-Sz5bVg90H21EOLACvtfcHk0lg9RLieS-2qvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFyhtLvL&md5=cba3f4535f2edc49440b50f219f33df4</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtt307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtt307%26sid%3Dliteratum%253Aachs%26aulast%3DAlaimo%26aufirst%3DS.%26aulast%3DPulvirenti%26aufirst%3DA.%26aulast%3DGiugno%26aufirst%3DR.%26aulast%3DFerro%26aufirst%3DA.%26atitle%3DDrug-Target%2520Interaction%2520Prediction%2520through%2520Domain-Tuned%2520Network-Based%2520Inference%26jtitle%3DBioinformatics%26date%3D2013%26volume%3D29%26spage%3D2004%26epage%3D2008%26doi%3D10.1093%2Fbioinformatics%2Fbtt307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Page, L.</span></span> <span> </span><span class="NLM_article-title">The Anatomy of a Large-Scale Hypertextual Web Search Engine</span>. <i>Computer Networks and ISDN Systems</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">107</span>– <span class="NLM_lpage">117</span>, <span class="refDoi"> DOI: 10.1016/S0169-7552(98)00110-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1016%2FS0169-7552%2898%2900110-X" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1998&pages=107-117&author=S.+Brinauthor=L.+Page&title=The+Anatomy+of+a+Large-Scale+Hypertextual+Web+Search+Engine&doi=10.1016%2FS0169-7552%2898%2900110-X"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1016%2FS0169-7552%2898%2900110-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-7552%252898%252900110-X%26sid%3Dliteratum%253Aachs%26aulast%3DBrin%26aufirst%3DS.%26aulast%3DPage%26aufirst%3DL.%26atitle%3DThe%2520Anatomy%2520of%2520a%2520Large-Scale%2520Hypertextual%2520Web%2520Search%2520Engine%26jtitle%3DComputer%2520Networks%2520and%2520ISDN%2520Systems%26date%3D1998%26volume%3D30%26spage%3D107%26epage%3D117%26doi%3D10.1016%2FS0169-7552%2898%2900110-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, G. Y.</span></span> <span> </span><span class="NLM_article-title">Drug-Target Interaction Prediction by Random Walk on the Heterogeneous Network</span>. <i>Mol. BioSyst.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1970</span>– <span class="NLM_lpage">1978</span>, <span class="refDoi"> DOI: 10.1039/c2mb00002d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1039%2Fc2mb00002d" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=22538619" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC38Xot1aqt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=1970-1978&author=X.+Chenauthor=M.+X.+Liuauthor=G.+Y.+Yan&title=Drug-Target+Interaction+Prediction+by+Random+Walk+on+the+Heterogeneous+Network&doi=10.1039%2Fc2mb00002d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-target interaction prediction by random walk on the heterogeneous network</span></div><div class="casAuthors">Chen, Xing; Liu, Ming-Xi; Yan, Gui-Ying</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1970-1978</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Predicting potential drug-target interactions from heterogeneous biol. data is crit. not only for better understanding of the various interactions and biol. processes, but also for the development of novel drugs and the improvement of human medicines.  In this paper, the method of Network-based Random Walk with Restart on the Heterogeneous network (NRWRH) is developed to predict potential drug-target interactions on a large scale under the hypothesis that similar drugs often target similar target proteins and the framework of Random Walk.  Compared with traditional supervised or semi-supervised methods, NRWRH makes full use of the tool of the network for data integration to predict drug-target assocns.  It integrates three different networks (protein-protein similarity network, drug-drug similarity network, and known drug-target interaction networks) into a heterogeneous network by known drug-target interactions and implements the random walk on this heterogeneous network.  When applied to four classes of important drug-target interactions including enzymes, ion channels, GPCRs and nuclear receptors, NRWRH significantly improves previous methods in terms of cross-validation and potential drug-target interaction prediction.  Excellent performance enables us to suggest a no. of new potential drug-target interactions for drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreLCn4HSwEX7Vg90H21EOLACvtfcHk0lhEB59mDghHog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xot1aqt7k%253D&md5=c880d83fbb20fb6b81606849c8e37533</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1039%2Fc2mb00002d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2mb00002d%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DM.%2BX.%26aulast%3DYan%26aufirst%3DG.%2BY.%26atitle%3DDrug-Target%2520Interaction%2520Prediction%2520by%2520Random%2520Walk%2520on%2520the%2520Heterogeneous%2520Network%26jtitle%3DMol.%2520BioSyst.%26date%3D2012%26volume%3D8%26spage%3D1970%26epage%3D1978%26doi%3D10.1039%2Fc2mb00002d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wild, D. J.</span></span> <span> </span><span class="NLM_article-title">Optimizing Drug–Target Interaction Prediction Based on Random Walk on Heterogeneous Networks</span>. <i>J. Cheminf.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">40</span>, <span class="refDoi"> DOI: 10.1186/s13321-015-0089-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1186%2Fs13321-015-0089-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVertL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=40&author=A.+Sealauthor=Y.-Y.+Ahnauthor=D.+J.+Wild&title=Optimizing+Drug%E2%80%93Target+Interaction+Prediction+Based+on+Random+Walk+on+Heterogeneous+Networks&doi=10.1186%2Fs13321-015-0089-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Optimizing drug-target interaction prediction based on random walk on heterogeneous networks</span></div><div class="casAuthors">Seal, Abhik; Ahn, Yong-Yeol; Wild, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cheminformatics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">40/1-40/12</span>CODEN:
                <span class="NLM_cas:coden">JCOHB3</span>;
        ISSN:<span class="NLM_cas:issn">1758-2946</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">Background: Predicting novel drug-target assocns. is important not only for developing new drugs, but also for furthering biol. knowledge by understanding how drugs work and their modes of action.  As more data about drugs, targets, and their interactions becomes available, computational approaches have become an indispensible part of drug target assocn. discovery.  In this paper we apply random walk with restart (RWR) method to a heterogeneous network of drugs and targets compiled from DrugBank database and investigate the performance of the method under parameter variation and choice of chem. fingerprint methods.  Results: We show that choice of chem. fingerprint does not affect the performance of the method when the parameters are tuned to optimal values.  We use a subset of the ChEMBL15 dataset that contains 2,763 assocns. between 544 drugs and 467 target proteins to evaluate our method, and we extd. datasets of bioactivity ≤1 and ≤10 μM activity cutoff.  For 1 μM bioactivity cutoff, we find that our method can correctly predict nearly 47, 55, 60% of the given drug-target interactions in the test dataset having more than 0, 1, 2 drug target relations for ChEMBL 1 μM dataset in top 50 rank positions.  For 10 μM bioactivity cutoff, we find that our method can correctly predict nearly 32.4, 34.8, 35.3% of the given drug-target interactions in the test dataset having more than 0, 1, 2 drug target relations for ChEMBL 1 μM dataset in top 50 rank positions.  We further examine the assocns. between 110 popular top selling drugs in 2012 and 3,519 targets and find the top ten targets for each drug.  Conclusions: We demonstrate the effectiveness and promise of the approach- RWR on heterogeneous networks using chem. features- for identifying novel drug target interactions and investigate the performance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodpTBwrBYHCrVg90H21EOLACvtfcHk0lhyWAXQpWmTRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVertL%252FN&md5=41b613b3bd38d78894d4c362b9293873</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1186%2Fs13321-015-0089-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13321-015-0089-z%26sid%3Dliteratum%253Aachs%26aulast%3DSeal%26aufirst%3DA.%26aulast%3DAhn%26aufirst%3DY.-Y.%26aulast%3DWild%26aufirst%3DD.%2BJ.%26atitle%3DOptimizing%2520Drug%25E2%2580%2593Target%2520Interaction%2520Prediction%2520Based%2520on%2520Random%2520Walk%2520on%2520Heterogeneous%2520Networks%26jtitle%3DJ.%2520Cheminf.%26date%3D2015%26volume%3D7%26spage%3D40%26doi%3D10.1186%2Fs13321-015-0089-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vanunu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magger, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruppin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shlomi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharan, R.</span></span> <span> </span><span class="NLM_article-title">Associating Genes and Protein Complexes with Disease via Network Propagation</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e1000641</span> <span class="refDoi"> DOI: 10.1371/journal.pcbi.1000641</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1371%2Fjournal.pcbi.1000641" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=20090828" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&author=O.+Vanunuauthor=O.+Maggerauthor=E.+Ruppinauthor=T.+Shlomiauthor=R.+Sharan&title=Associating+Genes+and+Protein+Complexes+with+Disease+via+Network+Propagation&doi=10.1371%2Fjournal.pcbi.1000641"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1000641&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1000641%26sid%3Dliteratum%253Aachs%26aulast%3DVanunu%26aufirst%3DO.%26aulast%3DMagger%26aufirst%3DO.%26aulast%3DRuppin%26aufirst%3DE.%26aulast%3DShlomi%26aufirst%3DT.%26aulast%3DSharan%26aufirst%3DR.%26atitle%3DAssociating%2520Genes%2520and%2520Protein%2520Complexes%2520with%2520Disease%2520via%2520Network%2520Propagation%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2010%26volume%3D6%26doi%3D10.1371%2Fjournal.pcbi.1000641" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shnaps, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silverbush, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharan, R.</span></span> <span> </span><span class="NLM_article-title">Inference of Personalized Drug Targets via Network Propagation</span>. <i>Pacific Symp. Biocomput.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_fpage">156</span>– <span class="NLM_lpage">167</span>, <span class="refDoi"> DOI: 10.1142/9789814749411_0015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1142%2F9789814749411_0015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=26776182" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A280%3ADC%252BC28nhsVKktw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&pages=156-167&author=O.+Shnapsauthor=E.+Perryauthor=D.+Silverbushauthor=R.+Sharan&title=Inference+of+Personalized+Drug+Targets+via+Network+Propagation&doi=10.1142%2F9789814749411_0015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">INFERENCE OF PERSONALIZED DRUG TARGETS VIA NETWORK PROPAGATION</span></div><div class="casAuthors">Shnaps Ortal; Perry Eyal; Silverbush Dana; Sharan Roded</div><div class="citationInfo"><span class="NLM_cas:title">Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">156-67</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">We present a computational strategy to simulate drug treatment in a personalized setting.  The method is based on integrating patient mutation and differential expression data with a protein-protein interaction network.  We test the impact of in-silico deletions of different proteins on the flow of information in the network and use the results to infer potential drug targets.  We apply our method to AML data from TCGA and validate the predicted drug targets using known targets.  To benchmark our patient-specific approach, we compare the personalized setting predictions to those of the conventional setting.  Our predicted drug targets are highly enriched with known targets from DrugBank and COSMIC (p < 10(-5) outperforming the non-personalized predictions.  Finally, we focus on the largest AML patient subgroup (~30%) which is characterized by an FLT3 mutation, and utilize our prediction score to rank patient sensitivity to inhibition of each predicted target, reproducing previous findings of in-vitro experiments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRJlV_JdjB3j77NkxioyDXTfW6udTcc2eaZnn-jdJRWMLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28nhsVKktw%253D%253D&md5=b71b20ab2c69490a634306548e971b1e</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1142%2F9789814749411_0015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1142%252F9789814749411_0015%26sid%3Dliteratum%253Aachs%26aulast%3DShnaps%26aufirst%3DO.%26aulast%3DPerry%26aufirst%3DE.%26aulast%3DSilverbush%26aufirst%3DD.%26aulast%3DSharan%26aufirst%3DR.%26atitle%3DInference%2520of%2520Personalized%2520Drug%2520Targets%2520via%2520Network%2520Propagation%26jtitle%3DPacific%2520Symp.%2520Biocomput.%26date%3D2016%26spage%3D156%26epage%3D167%26doi%3D10.1142%2F9789814749411_0015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Himmelstein, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lizee, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hessler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brueggeman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hadley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khankhanian, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baranzini, S. E.</span></span> <span> </span><span class="NLM_article-title">Systematic Integration of Biomedical Knowledge Prioritizes Drugs for Repurposing</span>. <i>eLife</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e26726</span> <span class="refDoi"> DOI: 10.7554/eLife.26726</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.7554%2FeLife.26726" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=28936969" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlOitLzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2017&author=D.+S.+Himmelsteinauthor=A.+Lizeeauthor=C.+Hesslerauthor=L.+Brueggemanauthor=S.+L.+Chenauthor=D.+Hadleyauthor=A.+Greenauthor=P.+Khankhanianauthor=S.+E.+Baranzini&title=Systematic+Integration+of+Biomedical+Knowledge+Prioritizes+Drugs+for+Repurposing&doi=10.7554%2FeLife.26726"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic integration of biomedical knowledge prioritizes drugs for repurposing</span></div><div class="casAuthors">Himmelstein, Daniel Scott; Lizee, Antoine; Hessler, Christine; Brueggeman, Leo; Chen, Sabrina L.; Hadley, Dexter; Green, Ari; Khankhanian, Pouya; Baranzini, Sergio E.</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e26726/1-e26726/35</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">The ability to computationally predict whether a compd. treats a disease would improve the economy and success rate of drug approval.  This study describes Project Rephetio to systematically model drug efficacy based on 755 existing treatments.  First, we constructed Hetionet (neo4j.het.io), an integrative network encoding knowledge from millions of biomedical studies.  Hetionet v1.0 consists of 47,031 nodes of 11 types and 2,250,197 relationships of 24 types.  Data were integrated from 29 public resources to connect compds., diseases, genes, anatomies, pathways, biol. processes, mol. functions, cellular components, pharmacol. classes, side effects, and symptoms.  Next, we identified network patterns that distinguish treatments from non-treatments.  Then, we predicted the probability of treatment for 209,168 compd.-disease pairs (het.io/repurpose).  Our predictions validated on two external sets of treatment and provided pharmacol. insights on epilepsy, suggesting they will help prioritize drug repurposing candidates.  This study was entirely open and received realtime feedback from 40 community members.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXC1uVD5EnNrVg90H21EOLACvtfcHk0lh9rzuCUmwhjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlOitLzJ&md5=885b93e215fe5c564f973f7b25dd6d32</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.7554%2FeLife.26726&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.26726%26sid%3Dliteratum%253Aachs%26aulast%3DHimmelstein%26aufirst%3DD.%2BS.%26aulast%3DLizee%26aufirst%3DA.%26aulast%3DHessler%26aufirst%3DC.%26aulast%3DBrueggeman%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DS.%2BL.%26aulast%3DHadley%26aufirst%3DD.%26aulast%3DGreen%26aufirst%3DA.%26aulast%3DKhankhanian%26aufirst%3DP.%26aulast%3DBaranzini%26aufirst%3DS.%2BE.%26atitle%3DSystematic%2520Integration%2520of%2520Biomedical%2520Knowledge%2520Prioritizes%2520Drugs%2520for%2520Repurposing%26jtitle%3DeLife%26date%3D2017%26volume%3D6%26doi%3D10.7554%2FeLife.26726" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gaur, D.</span>; <span class="NLM_string-name">Shastri, A.</span>; <span class="NLM_string-name">Biswas, R.</span></span> <span> </span><span class="NLM_article-title">Metagraph: A New Model of Data Structure</span>.  <i>Proceedings</i>, 2008 International Conference on Computer Science and Information Technology, ICCSIT 2008, Singapore, 2008; <span class="NLM_publisher-name">IEEE</span>, <span class="NLM_year">2008</span>; pp  <span class="NLM_fpage">729</span>– <span class="NLM_lpage">733</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&pages=729-733&author=D.+Gaur&author=A.+Shastri&author=R.+Biswas&title=Proceedings"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DGaur%26aufirst%3DD.%26atitle%3DMetagraph%253A%2520A%2520New%2520Model%2520of%2520Data%2520Structure%26btitle%3DProceedings%26pub%3DIEEE%26date%3D2008%26spage%3D729%26epage%3D733" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chien, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuang, R.</span></span> <span> </span><span class="NLM_article-title">Network-Based Machine Learning and Graph Theory Algorithms for Precision Oncology</span>. <i>npj Precis. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">25</span>, <span class="refDoi"> DOI: 10.1038/s41698-017-0029-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1038%2Fs41698-017-0029-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=29872707" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A280%3ADC%252BC1MbktlGnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2017&pages=25&author=W.+Zhangauthor=J.+Chienauthor=J.+Yongauthor=R.+Kuang&title=Network-Based+Machine+Learning+and+Graph+Theory+Algorithms+for+Precision+Oncology&doi=10.1038%2Fs41698-017-0029-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Network-based machine learning and graph theory algorithms for precision oncology</span></div><div class="casAuthors">Zhang Wei; Kuang Rui; Chien Jeremy; Yong Jeongsik</div><div class="citationInfo"><span class="NLM_cas:title">NPJ precision oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">25</span>
        ISSN:<span class="NLM_cas:issn">2397-768X</span>.
    </div><div class="casAbstract">Network-based analytics plays an increasingly important role in precision oncology.  Growing evidence in recent studies suggests that cancer can be better understood through mutated or dysregulated pathways or networks rather than individual mutations and that the efficacy of repositioned drugs can be inferred from disease modules in molecular networks.  This article reviews network-based machine learning and graph theory algorithms for integrative analysis of personal genomic data and biomedical knowledge bases to identify tumor-specific molecular mechanisms, candidate targets and repositioned drugs for personalized treatment.  The review focuses on the algorithmic design and mathematical formulation of these methods to facilitate applications and implementations of network-based analysis in the practice of precision oncology.  We review the methods applied in three scenarios to integrate genomic data and network models in different analysis pipelines, and we examine three categories of network-based approaches for repositioning drugs in drug-disease-gene networks.  In addition, we perform a comprehensive subnetwork/pathway analysis of mutations in 31 cancer genome projects in the Cancer Genome Atlas and present a detailed case study on ovarian cancer.  Finally, we discuss interesting observations, potential pitfalls and future directions in network-based precision oncology.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT5YHmp9KDVO2gRQsDqrjW_fW6udTcc2eaFqj8blsAXxbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MbktlGnuw%253D%253D&md5=5f85ff628227c45c42dea2f702fac0be</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1038%2Fs41698-017-0029-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41698-017-0029-7%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DChien%26aufirst%3DJ.%26aulast%3DYong%26aufirst%3DJ.%26aulast%3DKuang%26aufirst%3DR.%26atitle%3DNetwork-Based%2520Machine%2520Learning%2520and%2520Graph%2520Theory%2520Algorithms%2520for%2520Precision%2520Oncology%26jtitle%3Dnpj%2520Precis.%2520Oncol.%26date%3D2017%26volume%3D1%26spage%3D25%26doi%3D10.1038%2Fs41698-017-0029-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref88"><div class="reference"><strong class="refLabel"><a href="#ref88" class="rightTabRefNumLink">88</a></strong><div class="NLM_citation" id="rightTab-cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span> <span> </span><span class="NLM_article-title">Prediction of Drug-Disease Associations for Drug Repositioning through Drug-MiRNA-Disease Heterogeneous Network</span>. <i>IEEE Access</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">45281</span>– <span class="NLM_lpage">45287</span>, <span class="refDoi"> DOI: 10.1109/ACCESS.2018.2860632</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1109%2FACCESS.2018.2860632" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2018&pages=45281-45287&author=H.+Chenauthor=Z.+Zhang&title=Prediction+of+Drug-Disease+Associations+for+Drug+Repositioning+through+Drug-MiRNA-Disease+Heterogeneous+Network&doi=10.1109%2FACCESS.2018.2860632"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1109%2FACCESS.2018.2860632&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1109%252FACCESS.2018.2860632%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DZ.%26atitle%3DPrediction%2520of%2520Drug-Disease%2520Associations%2520for%2520Drug%2520Repositioning%2520through%2520Drug-MiRNA-Disease%2520Heterogeneous%2520Network%26jtitle%3DIEEE%2520Access%26date%3D2018%26volume%3D6%26spage%3D45281%26epage%3D45287%26doi%3D10.1109%2FACCESS.2018.2860632" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref89"><div class="reference"><strong class="refLabel"><a href="#ref89" class="rightTabRefNumLink">89</a></strong><div class="NLM_citation" id="rightTab-cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zong, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ngo, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harismendy, O.</span></span> <span> </span><span class="NLM_article-title">Deep Mining Heterogeneous Networks of Biomedical Linked Data to Predict Novel Drug-Target Associations</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">2337</span>– <span class="NLM_lpage">2344</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btx160</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fbioinformatics%2Fbtx160" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=28430977" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A280%3ADC%252BC1cvpvFKlsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2017&pages=2337-2344&author=N.+Zongauthor=H.+Kimauthor=V.+Ngoauthor=O.+Harismendy&title=Deep+Mining+Heterogeneous+Networks+of+Biomedical+Linked+Data+to+Predict+Novel+Drug-Target+Associations&doi=10.1093%2Fbioinformatics%2Fbtx160"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Deep mining heterogeneous networks of biomedical linked data to predict novel drug-target associations</span></div><div class="casAuthors">Zong Nansu; Kim Hyeoneui; Harismendy Olivier; Ngo Victoria; Harismendy Olivier</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics (Oxford, England)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2337-2344</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Motivation:  A heterogeneous network topology possessing abundant interactions between biomedical entities has yet to be utilized in similarity-based methods for predicting drug-target associations based on the array of varying features of drugs and their targets.  Deep learning reveals features of vertices of a large network that can be adapted in accommodating the similarity-based solutions to provide a flexible method of drug-target prediction.  Results:  We propose a similarity-based drug-target prediction method that enhances existing association discovery methods by using a topology-based similarity measure.  DeepWalk, a deep learning method, is adopted in this study to calculate the similarities within Linked Tripartite Network (LTN), a heterogeneous network generated from biomedical linked datasets.  This proposed method shows promising results for drug-target association prediction: 98.96% AUC ROC score with a 10-fold cross-validation and 99.25% AUC ROC score with a Monte Carlo cross-validation with LTN.  By utilizing DeepWalk, we demonstrate that: (i) this method outperforms other existing topology-based similarity computation methods, (ii) the performance is better for tripartite than with bipartite networks and (iii) the measure of similarity using network topology outperforms the ones derived from chemical structure (drugs) or genomic sequence (targets).  Our proposed methodology proves to be capable of providing a promising solution for drug-target prediction based on topological similarity with a heterogeneous network, and may be readily re-purposed and adapted in the existing of similarity-based methodologies.  Availability and Implementation:  The proposed method has been developed in JAVA and it is available, along with the data at the following URL: https://github.com/zongnansu1982/drug-target-prediction .  Contact:  <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="f39d92899c9d94b386908097dd969786dd">[email protected]</a>  Supplementary information:  Supplementary data are available at Bioinformatics online.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSnJQmQ93XAz91QT_1O5WBYfW6udTcc2ebmQCKO7bR9xbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cvpvFKlsw%253D%253D&md5=92bb2d32e82bd9ac613e9966912997a0</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtx160&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtx160%26sid%3Dliteratum%253Aachs%26aulast%3DZong%26aufirst%3DN.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DNgo%26aufirst%3DV.%26aulast%3DHarismendy%26aufirst%3DO.%26atitle%3DDeep%2520Mining%2520Heterogeneous%2520Networks%2520of%2520Biomedical%2520Linked%2520Data%2520to%2520Predict%2520Novel%2520Drug-Target%2520Associations%26jtitle%3DBioinformatics%26date%3D2017%26volume%3D33%26spage%3D2337%26epage%3D2344%26doi%3D10.1093%2Fbioinformatics%2Fbtx160" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref90"><div class="reference"><strong class="refLabel"><a href="#ref90" class="rightTabRefNumLink">90</a></strong><div class="NLM_citation" id="rightTab-cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luo, H.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Li, M.</span>; <span class="NLM_string-name">Luo, J.</span>; <span class="NLM_string-name">Ni, P.</span>; <span class="NLM_string-name">Zhao, K.</span>; <span class="NLM_string-name">Wu, F.</span>; <span class="NLM_string-name">Pan, Y.</span></span> <span> </span><span class="NLM_article-title">Computational Drug Repositioning with Random Walk on a Heterogeneous Network</span>.  <i>IEEE/ACM Transactions on Computational Biology and Bioinformatics</i>; <span class="NLM_publisher-name">IEEE</span>, <span class="NLM_year">2018</span>; <span class="refDoi"> DOI: 10.1109/TCBB.2018.2832078</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1109%2FTCBB.2018.2832078" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=H.+Luo&author=J.+Wang&author=M.+Li&author=J.+Luo&author=P.+Ni&author=K.+Zhao&author=F.+Wu&author=Y.+Pan&title=IEEE%2FACM+Transactions+on+Computational+Biology+and+Bioinformatics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1109%2FTCBB.2018.2832078&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1109%252FTCBB.2018.2832078%26sid%3Dliteratum%253Aachs%26aulast%3DLuo%26aufirst%3DH.%26atitle%3DComputational%2520Drug%2520Repositioning%2520with%2520Random%2520Walk%2520on%2520a%2520Heterogeneous%2520Network%26btitle%3DIEEE%252FACM%2520Transactions%2520on%2520Computational%2520Biology%2520and%2520Bioinformatics%26pub%3DIEEE%26date%3D2018%26doi%3D10.1109%2FTCBB.2018.2832078" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref91"><div class="reference"><strong class="refLabel"><a href="#ref91" class="rightTabRefNumLink">91</a></strong><div class="NLM_citation" id="rightTab-cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kauffman, S. A.</span></span> <span> </span><span class="NLM_article-title">Metabolic Stability and Epigenesis in Randomly Constructed Genetic Nets</span>. <i>J. Theor. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1969</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">437</span>– <span class="NLM_lpage">467</span>, <span class="refDoi"> DOI: 10.1016/0022-5193(69)90015-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1016%2F0022-5193%2869%2990015-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=5803332" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A280%3ADyaF1M3msVOhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1969&pages=437-467&author=S.+A.+Kauffman&title=Metabolic+Stability+and+Epigenesis+in+Randomly+Constructed+Genetic+Nets&doi=10.1016%2F0022-5193%2869%2990015-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolic stability and epigenesis in randomly constructed genetic nets</span></div><div class="casAuthors">Kauffman S A</div><div class="citationInfo"><span class="NLM_cas:title">Journal of theoretical biology</span>
        (<span class="NLM_cas:date">1969</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">437-67</span>
        ISSN:<span class="NLM_cas:issn">0022-5193</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTHZToYWQRjGOvW160gIxP_fW6udTcc2eZp8zeU6ZyUlrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaF1M3msVOhsw%253D%253D&md5=98f4e4aea59ba4a321ffe3f6be718c4a</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1016%2F0022-5193%2869%2990015-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0022-5193%252869%252990015-0%26sid%3Dliteratum%253Aachs%26aulast%3DKauffman%26aufirst%3DS.%2BA.%26atitle%3DMetabolic%2520Stability%2520and%2520Epigenesis%2520in%2520Randomly%2520Constructed%2520Genetic%2520Nets%26jtitle%3DJ.%2520Theor.%2520Biol.%26date%3D1969%26volume%3D22%26spage%3D437%26epage%3D467%26doi%3D10.1016%2F0022-5193%2869%2990015-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref92"><div class="reference"><strong class="refLabel"><a href="#ref92" class="rightTabRefNumLink">92</a></strong><div class="NLM_citation" id="rightTab-cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kauffman, S.</span></span> <span> </span><span class="NLM_article-title">Homeostasis and Differentiation in Random Genetic Control Networks</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1969</span>,  <span class="NLM_volume"><i>224</i></span>,  <span class="NLM_fpage">177</span>– <span class="NLM_lpage">178</span>, <span class="refDoi"> DOI: 10.1038/224177a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1038%2F224177a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=5343519" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A280%3ADyaE3c%252FgsFyrtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=224&publication_year=1969&pages=177-178&author=S.+Kauffman&title=Homeostasis+and+Differentiation+in+Random+Genetic+Control+Networks&doi=10.1038%2F224177a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">Homeostasis and differentiation in random genetic control networks</span></div><div class="casAuthors">Kauffman S</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">1969</span>),
    <span class="NLM_cas:volume">224</span>
        (<span class="NLM_cas:issue">5215</span>),
    <span class="NLM_cas:pages">177-8</span>
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQed00svzJ7TBLDWw4HIq2sfW6udTcc2eZp8zeU6ZyUlrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaE3c%252FgsFyrtQ%253D%253D&md5=5af5af5ffa244a7b73d92b0f1e8c34bb</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1038%2F224177a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F224177a0%26sid%3Dliteratum%253Aachs%26aulast%3DKauffman%26aufirst%3DS.%26atitle%3DHomeostasis%2520and%2520Differentiation%2520in%2520Random%2520Genetic%2520Control%2520Networks%26jtitle%3DNature%26date%3D1969%26volume%3D224%26spage%3D177%26epage%3D178%26doi%3D10.1038%2F224177a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref93"><div class="reference"><strong class="refLabel"><a href="#ref93" class="rightTabRefNumLink">93</a></strong><div class="NLM_citation" id="rightTab-cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drossel, B.</span></span> <span> </span><span class="NLM_article-title">Random Boolean Networks</span>. In  <i>Reviews of Nonlinear Dynamics and Complexity</i>; <span class="NLM_publisher-name">Wiley-VCH</span>: <span class="NLM_publisher-loc">Weinheim, Germany</span>, <span class="NLM_year">2008</span>; pp  <span class="NLM_fpage">69</span>– <span class="NLM_lpage">110</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1002%2F9783527626359.ch3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2008&pages=69-110&author=B.+Drossel&title=Reviews+of+Nonlinear+Dynamics+and+Complexity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1002%2F9783527626359.ch3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9783527626359.ch3%26sid%3Dliteratum%253Aachs%26aulast%3DDrossel%26aufirst%3DB.%26atitle%3DRandom%2520Boolean%2520Networks%26btitle%3DReviews%2520of%2520Nonlinear%2520Dynamics%2520and%2520Complexity%26pub%3DWiley-VCH%26date%3D2008%26spage%3D69%26epage%3D110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref94"><div class="reference"><strong class="refLabel"><a href="#ref94" class="rightTabRefNumLink">94</a></strong><div class="NLM_citation" id="rightTab-cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bornholdt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kauffman, S.</span></span> <span> </span><span class="NLM_article-title">Ensembles, Dynamics, and Cell Types: Revisiting the Statistical Mechanics Perspective on Cellular Regulation</span>. <i>J. Theor. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>467</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">22</span>, <span class="refDoi"> DOI: 10.1016/j.jtbi.2019.01.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1016%2Fj.jtbi.2019.01.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=30711453" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A280%3ADC%252BB3cjot1Glsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=467&publication_year=2019&pages=15-22&author=S.+Bornholdtauthor=S.+Kauffman&title=Ensembles%2C+Dynamics%2C+and+Cell+Types%3A+Revisiting+the+Statistical+Mechanics+Perspective+on+Cellular+Regulation&doi=10.1016%2Fj.jtbi.2019.01.036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Ensembles, dynamics, and cell types: Revisiting the statistical mechanics perspective on cellular regulation</span></div><div class="casAuthors">Bornholdt Stefan; Kauffman Stuart</div><div class="citationInfo"><span class="NLM_cas:title">Journal of theoretical biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">467</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">15-22</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Genetic regulatory networks control ontogeny.  For fifty years Boolean networks have served as models of such systems, ranging from ensembles of random Boolean networks as models for generic properties of gene regulation to working dynamical models of a growing number of sub-networks of real cells.  At the same time, their statistical mechanics has been thoroughly studied.  Here we recapitulate their original motivation in the context of current theoretical and empirical research.  We discuss ensembles of random Boolean networks whose dynamical attractors model cell types.  A sub-ensemble is the critical ensemble.  There is now strong evidence that genetic regulatory networks are dynamically critical, and that evolution is exploring the critical sub-ensemble.  The generic properties of this sub-ensemble predict essential features of cell differentiation.  In particular, the number of attractors in such networks scales as the DNA content raised to the 0.63 power.  Data on the number of cell types as a function of the DNA content per cell shows a scaling relationship of 0.88.  Thus, the theory correctly predicts a power law relationship between the number of cell types and the DNA contents per cell, and a comparable slope.  We discuss these new scaling values and show prospects for new research lines for Boolean networks as a base model for systems biology.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcReJOPEu9JaVAWRJ3OJdH_qfW6udTcc2eZp8zeU6ZyUlrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cjot1Glsw%253D%253D&md5=7bacdf0ff77bd9a84fae76ec7f5a75b1</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1016%2Fj.jtbi.2019.01.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jtbi.2019.01.036%26sid%3Dliteratum%253Aachs%26aulast%3DBornholdt%26aufirst%3DS.%26aulast%3DKauffman%26aufirst%3DS.%26atitle%3DEnsembles%252C%2520Dynamics%252C%2520and%2520Cell%2520Types%253A%2520Revisiting%2520the%2520Statistical%2520Mechanics%2520Perspective%2520on%2520Cellular%2520Regulation%26jtitle%3DJ.%2520Theor.%2520Biol.%26date%3D2019%26volume%3D467%26spage%3D15%26epage%3D22%26doi%3D10.1016%2Fj.jtbi.2019.01.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref95"><div class="reference"><strong class="refLabel"><a href="#ref95" class="rightTabRefNumLink">95</a></strong><div class="NLM_citation" id="rightTab-cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eichler, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bar-Yam, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingber, D. E.</span></span> <span> </span><span class="NLM_article-title">Cell Fates as High-Dimensional Attractor States of a Complex Gene Regulatory Network</span>. <i>Phys. Rev. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">128701</span>, <span class="refDoi"> DOI: 10.1103/PhysRevLett.94.128701</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1103%2FPhysRevLett.94.128701" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=15903968" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BD2MXivFSrs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2005&pages=128701&author=S.+Huangauthor=G.+Eichlerauthor=Y.+Bar-Yamauthor=D.+E.+Ingber&title=Cell+Fates+as+High-Dimensional+Attractor+States+of+a+Complex+Gene+Regulatory+Network&doi=10.1103%2FPhysRevLett.94.128701"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Cell Fates as High-Dimensional Attractor States of a Complex Gene Regulatory Network</span></div><div class="casAuthors">Huang, Sui; Eichler, Gabriel; Bar-Yam, Yaneer; Ingber, Donald E.</div><div class="citationInfo"><span class="NLM_cas:title">Physical Review Letters</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">128701/1-128701/4</span>CODEN:
                <span class="NLM_cas:coden">PRLTAO</span>;
        ISSN:<span class="NLM_cas:issn">0031-9007</span>.
    
            (<span class="NLM_cas:orgname">American Physical Society</span>)
        </div><div class="casAbstract">Cells in multicellular organisms switch between distinct cell fates, such as proliferation or differentiation into specialized cell types.  Genome-wide gene regulatory networks govern this behavior.  Theor. studies of complex networks suggest that they can exhibit ordered (stable) dynamics, raising the possibility that cell fates may represent high-dimensional attractor states.  We used gene expression profiling to show that trajectories of neutrophil differentiation converge to a common state from different directions of a 2773-dimensional gene expression state space, providing the first exptl. evidence for a high-dimensional stable attractor that represents a distinct cellular phenotype.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3MRBJUfK8WrVg90H21EOLACvtfcHk0lgD3igWJubXrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXivFSrs7k%253D&md5=afa34e2bedf468935ac7042b5fa28774</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1103%2FPhysRevLett.94.128701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevLett.94.128701%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DS.%26aulast%3DEichler%26aufirst%3DG.%26aulast%3DBar-Yam%26aufirst%3DY.%26aulast%3DIngber%26aufirst%3DD.%2BE.%26atitle%3DCell%2520Fates%2520as%2520High-Dimensional%2520Attractor%2520States%2520of%2520a%2520Complex%2520Gene%2520Regulatory%2520Network%26jtitle%3DPhys.%2520Rev.%2520Lett.%26date%3D2005%26volume%3D94%26spage%3D128701%26doi%3D10.1103%2FPhysRevLett.94.128701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref96"><div class="reference"><strong class="refLabel"><a href="#ref96" class="rightTabRefNumLink">96</a></strong><div class="NLM_citation" id="rightTab-cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bloomingdale, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mager, D. E.</span></span> <span> </span><span class="NLM_article-title">Boolean Network Modeling in Systems Pharmacology</span>. <i>J. Pharmacokinet. Pharmacodyn.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">159</span>– <span class="NLM_lpage">180</span>, <span class="refDoi"> DOI: 10.1007/s10928-017-9567-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1007%2Fs10928-017-9567-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=29307099" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1cXlvVyhtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2018&pages=159-180&author=P.+Bloomingdaleauthor=V.+A.+Nguyenauthor=J.+Niuauthor=D.+E.+Mager&title=Boolean+Network+Modeling+in+Systems+Pharmacology&doi=10.1007%2Fs10928-017-9567-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">Boolean network modeling in systems pharmacology</span></div><div class="casAuthors">Bloomingdale, Peter; Nguyen, Van Anh; Niu, Jin; Mager, Donald E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacokinetics and Pharmacodynamics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">159-180</span>CODEN:
                <span class="NLM_cas:coden">JPPOAH</span>;
        ISSN:<span class="NLM_cas:issn">1567-567X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Quant. systems pharmacol. (QSP) is an emerging discipline that aims to discover how drugs modulate the dynamics of biol. components in mol. and cellular networks and the impact of those perturbations on human pathophysiol.  The integration of systems-based exptl. and computational approaches is required to facilitate the advancement of this field.  QSP models typically consist of a series of ordinary differential equations (ODE).  However, this math. framework requires extensive knowledge of parameters pertaining to biol. processes, which is often unavailable.  An alternative framework that does not require knowledge of system-specific parameters, such as Boolean network modeling, could serve as an initial foundation prior to the development of an ODE-based model.  Boolean network models have been shown to efficiently describe, in a qual. manner, the complex behavior of signal transduction and gene/protein regulatory processes.  In addn. to providing a starting point prior to quant. modeling, Boolean network models can also be utilized to discover novel therapeutic targets and combinatorial treatment strategies.  Identifying drug targets using a network-based approach could supplement current drug discovery methodologies and help to fill the innovation gap across the pharmaceutical industry.  In this review, we discuss the process of developing Boolean network models and the various analyses that can be performed to identify novel drug targets and combinatorial approaches.  An example for each of these analyses is provided using a previously developed Boolean network of signaling pathways in multiple myeloma.  Selected examples of Boolean network models of human (patho-)physiol. systems are also reviewed in brief.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLhhqodM6mfLVg90H21EOLACvtfcHk0lgD3igWJubXrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXlvVyhtg%253D%253D&md5=9fce87d27de012fa22ce15efeaa13861</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1007%2Fs10928-017-9567-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10928-017-9567-4%26sid%3Dliteratum%253Aachs%26aulast%3DBloomingdale%26aufirst%3DP.%26aulast%3DNguyen%26aufirst%3DV.%2BA.%26aulast%3DNiu%26aufirst%3DJ.%26aulast%3DMager%26aufirst%3DD.%2BE.%26atitle%3DBoolean%2520Network%2520Modeling%2520in%2520Systems%2520Pharmacology%26jtitle%3DJ.%2520Pharmacokinet.%2520Pharmacodyn.%26date%3D2018%26volume%3D45%26spage%3D159%26epage%3D180%26doi%3D10.1007%2Fs10928-017-9567-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref97"><div class="reference"><strong class="refLabel"><a href="#ref97" class="rightTabRefNumLink">97</a></strong><div class="NLM_citation" id="rightTab-cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Anda-Jáuregui, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espinal-Enríquez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandoval-Motta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernández-Lemus, E.</span></span> <span> </span><span class="NLM_article-title">A Boolean Network Approach to Estrogen Transcriptional Regulation</span>. <i>Complexity</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>2019</i></span>,  <span class="NLM_fpage">8740279</span>, <span class="refDoi"> DOI: 10.1155/2019/8740279</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1155%2F2019%2F8740279" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2019&publication_year=2019&pages=8740279&author=G.+De%0AAnda-J%C3%A1ureguiauthor=J.+Espinal-Enr%C3%ADquezauthor=S.+Sandoval-Mottaauthor=E.+Hern%C3%A1ndez-Lemus&title=A+Boolean+Network+Approach+to+Estrogen+Transcriptional+Regulation&doi=10.1155%2F2019%2F8740279"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1155%2F2019%2F8740279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2019%252F8740279%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BAnda-J%25C3%25A1uregui%26aufirst%3DG.%26aulast%3DEspinal-Enr%25C3%25ADquez%26aufirst%3DJ.%26aulast%3DSandoval-Motta%26aufirst%3DS.%26aulast%3DHern%25C3%25A1ndez-Lemus%26aufirst%3DE.%26atitle%3DA%2520Boolean%2520Network%2520Approach%2520to%2520Estrogen%2520Transcriptional%2520Regulation%26jtitle%3DComplexity%26date%3D2019%26volume%3D2019%26spage%3D8740279%26doi%3D10.1155%2F2019%2F8740279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref98"><div class="reference"><strong class="refLabel"><a href="#ref98" class="rightTabRefNumLink">98</a></strong><div class="NLM_citation" id="rightTab-cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, K. H.</span></span> <span> </span><span class="NLM_article-title">Attractor Landscape Analysis Reveals Feedback Loops in the P53 Network That Control the Cellular Response to DNA Damage</span>. <i>Sci. Signaling</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">ra83</span>, <span class="refDoi"> DOI: 10.1126/scisignal.2003363</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1126%2Fscisignal.2003363" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=23169817" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2012&pages=ra83&author=M.+Choiauthor=J.+Shiauthor=S.+H.+Jungauthor=X.+Chenauthor=K.+H.+Cho&title=Attractor+Landscape+Analysis+Reveals+Feedback+Loops+in+the+P53+Network+That+Control+the+Cellular+Response+to+DNA+Damage&doi=10.1126%2Fscisignal.2003363"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1126%2Fscisignal.2003363&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscisignal.2003363%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DJung%26aufirst%3DS.%2BH.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DCho%26aufirst%3DK.%2BH.%26atitle%3DAttractor%2520Landscape%2520Analysis%2520Reveals%2520Feedback%2520Loops%2520in%2520the%2520P53%2520Network%2520That%2520Control%2520the%2520Cellular%2520Response%2520to%2520DNA%2520Damage%26jtitle%3DSci.%2520Signaling%26date%3D2012%26volume%3D5%26spage%3Dra83%26doi%3D10.1126%2Fscisignal.2003363" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref99"><div class="reference"><strong class="refLabel"><a href="#ref99" class="rightTabRefNumLink">99</a></strong><div class="NLM_citation" id="rightTab-cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, K. H.</span></span> <span> </span><span class="NLM_article-title">Network Dynamics-Based Cancer Panel Stratification for Systemic Prediction of Anticancer Drug Response</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1940</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-02160-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1038%2Fs41467-017-02160-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=29208897" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A280%3ADC%252BC1M3otlWgsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1940&author=M.+Choiauthor=J.+Shiauthor=Y.+Zhuauthor=R.+Yangauthor=K.+H.+Cho&title=Network+Dynamics-Based+Cancer+Panel+Stratification+for+Systemic+Prediction+of+Anticancer+Drug+Response&doi=10.1038%2Fs41467-017-02160-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Network dynamics-based cancer panel stratification for systemic prediction of anticancer drug response</span></div><div class="casAuthors">Choi Minsoo; Cho Kwang-Hyun; Shi Jue; Zhu Yanting; Yang Ruizhen</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1940</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cancer is a complex disease involving multiple genomic alterations that disrupt the dynamic response of signaling networks.  The heterogeneous nature of cancer, which results in highly variable drug response, is a major obstacle to developing effective cancer therapy.  Previous studies of cancer therapeutic response mostly focus on static analysis of genome-wide alterations, thus they are unable to unravel the dynamic, network-specific origin of variation.  Here we present a network dynamics-based approach to integrate cancer genomics with dynamics of biological network for drug response prediction and design of drug combination.  We select the p53 network as an example and analyze its cancer-specific state transition dynamics under distinct anticancer drug treatments by attractor landscape analysis.  Our results not only enable stratification of cancer into distinct drug response groups, but also reveal network-specific drug targets that maximize p53 network-mediated cell death, providing a basis to design combinatorial therapeutic strategies for distinct cancer genomic subtypes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT6sxPcQa39gcScOqQTSkOffW6udTcc2ebCdsW0TkO_Grntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M3otlWgsQ%253D%253D&md5=5a210fe74d025fc4d143243637cab5ae</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-02160-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-02160-5%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DR.%26aulast%3DCho%26aufirst%3DK.%2BH.%26atitle%3DNetwork%2520Dynamics-Based%2520Cancer%2520Panel%2520Stratification%2520for%2520Systemic%2520Prediction%2520of%2520Anticancer%2520Drug%2520Response%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D1940%26doi%3D10.1038%2Fs41467-017-02160-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref100"><div class="reference"><strong class="refLabel"><a href="#ref100" class="rightTabRefNumLink">100</a></strong><div class="NLM_citation" id="rightTab-cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Capobianco, E.</span></span> <span> </span><span class="NLM_article-title">Next Generation Networks: Featuring the Potential Role of Emerging Applications in Translational Oncology</span>. <i>J. Clin. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">664</span>, <span class="refDoi"> DOI: 10.3390/jcm8050664</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.3390%2Fjcm8050664" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2019&pages=664&author=E.+Capobianco&title=Next+Generation+Networks%3A+Featuring+the+Potential+Role+of+Emerging+Applications+in+Translational+Oncology&doi=10.3390%2Fjcm8050664"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.3390%2Fjcm8050664&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fjcm8050664%26sid%3Dliteratum%253Aachs%26aulast%3DCapobianco%26aufirst%3DE.%26atitle%3DNext%2520Generation%2520Networks%253A%2520Featuring%2520the%2520Potential%2520Role%2520of%2520Emerging%2520Applications%2520in%2520Translational%2520Oncology%26jtitle%3DJ.%2520Clin.%2520Med.%26date%3D2019%26volume%3D8%26spage%3D664%26doi%3D10.3390%2Fjcm8050664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref101"><div class="reference"><strong class="refLabel"><a href="#ref101" class="rightTabRefNumLink">101</a></strong><div class="NLM_citation" id="rightTab-cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Husic, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pande, V. S.</span></span> <span> </span><span class="NLM_article-title">Markov State Models: From an Art to a Science</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">2386</span>– <span class="NLM_lpage">2396</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b12191</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b12191" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotV2huw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=2386-2396&author=B.+E.+Husicauthor=V.+S.+Pande&title=Markov+State+Models%3A+From+an+Art+to+a+Science&doi=10.1021%2Fjacs.7b12191"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Markov State Models: From an Art to a Science</span></div><div class="casAuthors">Husic, Brooke E.; Pande, Vijay S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2386-2396</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Markov state models (MSMs) are a powerful framework for analyzing dynamical systems, such as mol. dynamics (MD) simulations, that have gained widespread use over the past several decades.  This perspective offers an overview of the MSM field to date, presented for a general audience as a timeline of key developments in the field.  We sequentially address early studies that motivated the method, canonical papers that established the use of MSMs for MD anal., and subsequent advances in software and anal. protocols.  The derivation of a variational principle for MSMs in 2013 signified a turning point from expertise-driving MSM building to a systematic, objective protocol.  The variational approach, combined with best practices for model selection and open-source software, enabled a wide range of MSM anal. for applications such as protein folding and allostery, ligand binding, and protein-protein assocn.  To conclude, the current frontiers of methods development are highlighted, as well as exciting applications in exptl. design and drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo43eE2dHXoHrVg90H21EOLACvtfcHk0li5UAF9alagow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotV2huw%253D%253D&md5=f00943acedec985c4d21abd86ecfce4a</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b12191&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b12191%26sid%3Dliteratum%253Aachs%26aulast%3DHusic%26aufirst%3DB.%2BE.%26aulast%3DPande%26aufirst%3DV.%2BS.%26atitle%3DMarkov%2520State%2520Models%253A%2520From%2520an%2520Art%2520to%2520a%2520Science%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D2386%26epage%3D2396%26doi%3D10.1021%2Fjacs.7b12191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref102"><div class="reference"><strong class="refLabel"><a href="#ref102" class="rightTabRefNumLink">102</a></strong><div class="NLM_citation" id="rightTab-cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bernetti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masetti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Recanatini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amaro, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span> <span> </span><span class="NLM_article-title">An Integrated Markov State Model and Path Metadynamics Approach to Characterize Drug Binding Processes</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">5689</span>– <span class="NLM_lpage">5702</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.9b00450</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.9b00450" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1egurjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=5689-5702&author=M.+Bernettiauthor=M.+Masettiauthor=M.+Recanatiniauthor=R.+E.+Amaroauthor=A.+Cavalli&title=An+Integrated+Markov+State+Model+and+Path+Metadynamics+Approach+to+Characterize+Drug+Binding+Processes&doi=10.1021%2Facs.jctc.9b00450"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">An Integrated Markov State Model and Path Metadynamics Approach To Characterize Drug Binding Processes</span></div><div class="casAuthors">Bernetti, Mattia; Masetti, Matteo; Recanatini, Maurizio; Amaro, Rommie E.; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5689-5702</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Unveiling the mechanistic features of drug-target binding is of central interest in biophysics and drug discovery.  Herein, the authors address this challenge by combining two major computational approaches, namely Mol. Dynamics (MD) simulations and Markov State Models (MSM), with a Path Collective Variables (PCVs) description coupled with metadynamics.  The authors apply the methodol. to reconstruct the binding process of the antagonist alprenolol to the β2-adrenergic receptor, a well-established pharmaceutical target.  The devised protocol allowed us to est. the binding free energy and identify the min. free energy path leading to the protein-ligand complex.  In summary, the authors show that MSM and PCVs can be efficiently integrated to shed light upon mechanistic and energetic details underlying complex recognition processes in biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbe1nT6X31cLVg90H21EOLACvtfcHk0ljM1c9A3JA-jA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1egurjP&md5=f3f4bfadf4a14e2beb74764fa8dae955</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.9b00450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.9b00450%26sid%3Dliteratum%253Aachs%26aulast%3DBernetti%26aufirst%3DM.%26aulast%3DMasetti%26aufirst%3DM.%26aulast%3DRecanatini%26aufirst%3DM.%26aulast%3DAmaro%26aufirst%3DR.%2BE.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DAn%2520Integrated%2520Markov%2520State%2520Model%2520and%2520Path%2520Metadynamics%2520Approach%2520to%2520Characterize%2520Drug%2520Binding%2520Processes%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2019%26volume%3D15%26spage%3D5689%26epage%3D5702%26doi%3D10.1021%2Facs.jctc.9b00450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref103"><div class="reference"><strong class="refLabel"><a href="#ref103" class="rightTabRefNumLink">103</a></strong><div class="NLM_citation" id="rightTab-cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chu, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tse, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, E. L.</span></span> <span> </span><span class="NLM_article-title">Markov State Models of Gene Regulatory Networks</span>. <i>BMC Syst. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">14</span>, <span class="refDoi"> DOI: 10.1186/s12918-017-0394-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1186%2Fs12918-017-0394-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=28166778" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjt1SkurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2017&pages=14&author=B.+K.+Chuauthor=M.+J.+Tseauthor=R.+R.+Satoauthor=E.+L.+Read&title=Markov+State+Models+of+Gene+Regulatory+Networks&doi=10.1186%2Fs12918-017-0394-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Markov State Models of gene regulatory networks</span></div><div class="casAuthors">Chu, Brian K.; Tse, Margaret J.; Sato, Royce R.; Read, Elizabeth L.</div><div class="citationInfo"><span class="NLM_cas:title">BMC Systems Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">14/1-14/17</span>CODEN:
                <span class="NLM_cas:coden">BSBMCC</span>;
        ISSN:<span class="NLM_cas:issn">1752-0509</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Gene regulatory networks with dynamics characterized by multiple stable states underlie cell fate-decisions.  Quant. models that can link mol.-level knowledge of gene regulation to a global understanding of network dynamics have the potential to guide cell-reprogramming strategies.  Networks are often modeled by the stochastic Chem. Master Equation, but methods for systematic identification of key properties of the global dynamics are currently lacking.  Results: The method identifies the no., phenotypes, and lifetimes of long-lived states for a set of common gene regulatory network models.  Application of transition path theory to the constructed Markov State Model decomps. global dynamics into a set of dominant transition paths and assocd. relative probabilities for stochastic state-switching.  Conclusions: In this proof-of-concept study, we found that the Markov State Model provides a general framework for analyzing and visualizing stochastic multistability and state-transitions in gene networks.  Our results suggest that this framework-adopted from the field of atomistic Mol. Dynamics-can be a useful tool for quant. Systems Biol. at the network scale.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1h_HrYHhRhbVg90H21EOLACvtfcHk0ljM1c9A3JA-jA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjt1SkurY%253D&md5=58df2525842dd65cd630fa4704a4a16c</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1186%2Fs12918-017-0394-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12918-017-0394-4%26sid%3Dliteratum%253Aachs%26aulast%3DChu%26aufirst%3DB.%2BK.%26aulast%3DTse%26aufirst%3DM.%2BJ.%26aulast%3DSato%26aufirst%3DR.%2BR.%26aulast%3DRead%26aufirst%3DE.%2BL.%26atitle%3DMarkov%2520State%2520Models%2520of%2520Gene%2520Regulatory%2520Networks%26jtitle%3DBMC%2520Syst.%2520Biol.%26date%3D2017%26volume%3D11%26spage%3D14%26doi%3D10.1186%2Fs12918-017-0394-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref104"><div class="reference"><strong class="refLabel"><a href="#ref104" class="rightTabRefNumLink">104</a></strong><div class="NLM_citation" id="rightTab-cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hansch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujita, T.</span></span> <span> </span><span class="NLM_article-title">ρ-σ-π Analysis. A Method for the Correlation of Biological Activity and Chemical Structure</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1964</span>,  <span class="NLM_volume"><i>86</i></span>,  <span class="NLM_fpage">1616</span>– <span class="NLM_lpage">1626</span>, <span class="refDoi"> DOI: 10.1021/ja01062a035</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja01062a035" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADyaF2cXptF2gsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=86&publication_year=1964&pages=1616-1626&author=C.+Hanschauthor=T.+Fujita&title=%CF%81-%CF%83-%CF%80+Analysis.+A+Method+for+the+Correlation+of+Biological+Activity+and+Chemical+Structure&doi=10.1021%2Fja01062a035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">ρ-σ-πAnalysis; method for the correlation of biological activity and chemical structure</span></div><div class="casAuthors">Hansch, Corwin; Fujita, Toshio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1964</span>),
    <span class="NLM_cas:volume">86</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1616-26</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    </div><div class="casAbstract">Using the substituent constant, σ, and a substituent constant, π, defined as σ = log Px - log PH(PH is the partition coeff. of a parent compd. and Px that of a deriv.), regression analyses have been made of the effect of substituents on the biol. activity of benzoic acids on mosquito larvae, phenols on gram-pos. and gram-neg. bacteria, phenyl ethyl phosphate insecticides on houseflies, thyroxine derivs. on rodents, diethylaminoethyl benzoates on guinea pigs, and carcinogenic compds. on mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoMZAWYjkBhLVg90H21EOLACvtfcHk0ljM1c9A3JA-jA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF2cXptF2gsQ%253D%253D&md5=190de0722f5c3a1d98a2419df7d28c58</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1021%2Fja01062a035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja01062a035%26sid%3Dliteratum%253Aachs%26aulast%3DHansch%26aufirst%3DC.%26aulast%3DFujita%26aufirst%3DT.%26atitle%3D%25CF%2581-%25CF%2583-%25CF%2580%2520Analysis.%2520A%2520Method%2520for%2520the%2520Correlation%2520of%2520Biological%2520Activity%2520and%2520Chemical%2520Structure%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1964%26volume%3D86%26spage%3D1616%26epage%3D1626%26doi%3D10.1021%2Fja01062a035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref105"><div class="reference"><strong class="refLabel"><a href="#ref105" class="rightTabRefNumLink">105</a></strong><div class="NLM_citation" id="rightTab-cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whitmore, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wray, J.</span></span> <span> </span><span class="NLM_article-title">Networks for Better Drug Discovery: Making Optimal Use of Existing Chemical Space</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">2497</span>– <span class="NLM_lpage">2499</span>, <span class="refDoi"> DOI: 10.4155/fmc-2018-0265</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.4155%2Ffmc-2018-0265" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVKjtLnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=2497-2499&author=A.+V.+Whitmoreauthor=J.+Wray&title=Networks+for+Better+Drug+Discovery%3A+Making+Optimal+Use+of+Existing+Chemical+Space&doi=10.4155%2Ffmc-2018-0265"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">Networks for better drug discovery: making optimal use of existing chemical space</span></div><div class="casAuthors">Whitmore, Alan Victor; Wray, Jonny</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">2497-2499</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVJe5nC7HA0rVg90H21EOLACvtfcHk0ljxJiwlGCkh7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVKjtLnM&md5=991600275a08702cfd88cfdc17e38c01</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2018-0265&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2018-0265%26sid%3Dliteratum%253Aachs%26aulast%3DWhitmore%26aufirst%3DA.%2BV.%26aulast%3DWray%26aufirst%3DJ.%26atitle%3DNetworks%2520for%2520Better%2520Drug%2520Discovery%253A%2520Making%2520Optimal%2520Use%2520of%2520Existing%2520Chemical%2520Space%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2018%26volume%3D10%26spage%3D2497%26epage%3D2499%26doi%3D10.4155%2Ffmc-2018-0265" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref106"><div class="reference"><strong class="refLabel"><a href="#ref106" class="rightTabRefNumLink">106</a></strong><div class="NLM_citation" id="rightTab-cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oprea, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">May, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leitão, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tropsha, A.</span></span> <span> </span><span class="NLM_article-title">Computational Systems Chemical Biology</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>672</i></span>,  <span class="NLM_fpage">459</span>– <span class="NLM_lpage">488</span>, <span class="refDoi"> DOI: 10.1007/978-1-60761-839-3_18</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1007%2F978-1-60761-839-3_18" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=20838980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlejsr%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=672&publication_year=2011&pages=459-488&author=T.+I.+Opreaauthor=E.+E.+Mayauthor=A.+Leit%C3%A3oauthor=A.+Tropsha&title=Computational+Systems+Chemical+Biology&doi=10.1007%2F978-1-60761-839-3_18"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">Computational systems chemical biology</span></div><div class="casAuthors">Oprea, Tudor I.; May, Elebeoba E.; Leitao, Andrei; Tropsha, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">672</span>
        (<span class="NLM_cas:issue">Chemoinformatics and Computational Chemical Biology</span>),
    <span class="NLM_cas:pages">459-488</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  There is a crit. need for improving the level of chem. awareness in systems biol.  The data and information related to modulation of genes and proteins by small mols. continue to accumulate at the same time as simulation tools in systems biol. and whole body physiol. based pharmacokinetics (PBPK) continue to evolve.  We called this emerging area at the interface between chem. biol. and systems biol. systems chem. biol. (SCB).  The overarching goal of computational SCB is to develop tools for integrated chem.-biol. data acquisition, filtering and processing, by taking into account relevant information related to interactions between proteins and small mols., possible metabolic transformations of small mols., as well as assocd. information related to genes, networks, small mols., and, where applicable, mutants and variants of those proteins.  There is yet an unmet need to develop an integrated in silico pharmacol./systems biol. continuum that embeds drug-target-clin. outcome (DTCO) triplets, a capability that is vital to the future of chem. biol., pharmacol., and systems biol.  Through the development of the SCB approach, scientists will be able to start addressing, in an integrated simulation environment, questions that make the best use of our ever-growing chem. and biol. data repositories at the system-wide level.  This chapter reviews some of the major research concepts and describes key components that constitute the emerging area of computational systems chem. biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHfdr3pD_PjrVg90H21EOLACvtfcHk0ljxJiwlGCkh7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlejsr%252FP&md5=b31b3427ddc5ad4b7a6550787e7a717d</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1007%2F978-1-60761-839-3_18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-60761-839-3_18%26sid%3Dliteratum%253Aachs%26aulast%3DOprea%26aufirst%3DT.%2BI.%26aulast%3DMay%26aufirst%3DE.%2BE.%26aulast%3DLeit%25C3%25A3o%26aufirst%3DA.%26aulast%3DTropsha%26aufirst%3DA.%26atitle%3DComputational%2520Systems%2520Chemical%2520Biology%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2011%26volume%3D672%26spage%3D459%26epage%3D488%26doi%3D10.1007%2F978-1-60761-839-3_18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref107"><div class="reference"><strong class="refLabel"><a href="#ref107" class="rightTabRefNumLink">107</a></strong><div class="NLM_citation" id="rightTab-cit107"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vamathevan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czodrowski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dunham, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferran, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madabhushi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spitzer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span> <span> </span><span class="NLM_article-title">Applications of Machine Learning in Drug Discovery and Development</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">463</span>– <span class="NLM_lpage">477</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0024-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1038%2Fs41573-019-0024-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=30976107" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosF2rtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=463-477&author=J.+Vamathevanauthor=D.+Clarkauthor=P.+Czodrowskiauthor=I.+Dunhamauthor=E.+Ferranauthor=G.+Leeauthor=B.+Liauthor=A.+Madabhushiauthor=P.+Shahauthor=M.+Spitzerauthor=S.+Zhao&title=Applications+of+Machine+Learning+in+Drug+Discovery+and+Development&doi=10.1038%2Fs41573-019-0024-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">Applications of machine learning in drug discovery and development</span></div><div class="casAuthors">Vamathevan, Jessica; Clark, Dominic; Czodrowski, Paul; Dunham, Ian; Ferran, Edgardo; Lee, George; Li, Bin; Madabhushi, Anant; Shah, Parantu; Spitzer, Michaela; Zhao, Shanrong</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">463-477</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Drug discovery and development pipelines are long, complex and depend on numerous factors.  Machine learning (ML) approaches provide a set of tools that can improve discovery and decision making for well-specified questions with abundant, high-quality data.  Opportunities to apply ML occur in all stages of drug discovery.  Examples include target validation, identification of prognostic biomarkers and anal. of digital pathol. data in clin. trials.  Applications have ranged in context and methodol., with some approaches yielding accurate predictions and insights.  The challenges of applying ML lie primarily with the lack of interpretability and repeatability of ML-generated results, which may limit their application.  In all areas, systematic and comprehensive high-dimensional data still need to be generated.  With ongoing efforts to tackle these issues, as well as increasing awareness of the factors needed to validate ML approaches, the application of ML can promote data-driven decision making and has the potential to speed up the process and reduce failure rates in drug discovery and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreREm2gmVwV7Vg90H21EOLACvtfcHk0ljJZAU9Kbe5gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosF2rtrY%253D&md5=211782aeea3d8b9f50368f89177a70d2</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0024-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0024-5%26sid%3Dliteratum%253Aachs%26aulast%3DVamathevan%26aufirst%3DJ.%26aulast%3DClark%26aufirst%3DD.%26aulast%3DCzodrowski%26aufirst%3DP.%26aulast%3DDunham%26aufirst%3DI.%26aulast%3DFerran%26aufirst%3DE.%26aulast%3DLee%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DB.%26aulast%3DMadabhushi%26aufirst%3DA.%26aulast%3DShah%26aufirst%3DP.%26aulast%3DSpitzer%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DS.%26atitle%3DApplications%2520of%2520Machine%2520Learning%2520in%2520Drug%2520Discovery%2520and%2520Development%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D463%26epage%3D477%26doi%3D10.1038%2Fs41573-019-0024-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref108"><div class="reference"><strong class="refLabel"><a href="#ref108" class="rightTabRefNumLink">108</a></strong><div class="NLM_citation" id="rightTab-cit108"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Byrne, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span> <span> </span><span class="NLM_article-title">Concepts of Artificial Intelligence for Computer-Assisted Drug Discovery</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">10520</span>– <span class="NLM_lpage">10594</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.8b00728</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.8b00728" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlCgsbzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2019&pages=10520-10594&author=X.+Yangauthor=Y.+Wangauthor=R.+Byrneauthor=G.+Schneiderauthor=S.+Yang&title=Concepts+of+Artificial+Intelligence+for+Computer-Assisted+Drug+Discovery&doi=10.1021%2Facs.chemrev.8b00728"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Concepts of Artificial Intelligence for Computer-Assisted Drug Discovery</span></div><div class="casAuthors">Yang, Xin; Wang, Yifei; Byrne, Ryan; Schneider, Gisbert; Yang, Shengyong</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">10520-10594</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Artificial intelligence (AI), and, in particular, deep learning as a subcategory of AI, provides opportunities for the discovery and development of innovative drugs.  Various machine learning approaches have recently (re)emerged, some of which may be considered instances of domain-specific AI which have been successfully employed for drug discovery and design.  This review provides a comprehensive portrayal of these machine learning techniques and of their applications in medicinal chem.  After introducing the basic principles, alongside some application notes, of the various machine learning algorithms, the current state-of-the art of AI-assisted pharmaceutical discovery is discussed, including applications in structure- and ligand-based virtual screening, de novo drug design, physicochem. and pharmacokinetic property prediction, drug repurposing, and related aspects.  Finally, several challenges and limitations of the current methods are summarized, with a view to potential future directions for AI-assisted drug discovery and design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBUfBzNv587rVg90H21EOLACvtfcHk0ljJZAU9Kbe5gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlCgsbzN&md5=6e9816e30251cd4f15c831e2dd20f100</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.8b00728&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.8b00728%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DByrne%26aufirst%3DR.%26aulast%3DSchneider%26aufirst%3DG.%26aulast%3DYang%26aufirst%3DS.%26atitle%3DConcepts%2520of%2520Artificial%2520Intelligence%2520for%2520Computer-Assisted%2520Drug%2520Discovery%26jtitle%3DChem.%2520Rev.%26date%3D2019%26volume%3D119%26spage%3D10520%26epage%3D10594%26doi%3D10.1021%2Facs.chemrev.8b00728" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref109"><div class="reference"><strong class="refLabel"><a href="#ref109" class="rightTabRefNumLink">109</a></strong><div class="NLM_citation" id="rightTab-cit109"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamanishi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Araki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutteridge, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honda, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanehisa, M.</span></span> <span> </span><span class="NLM_article-title">Prediction of Drug-Target Interaction Networks from the Integration of Chemical and Genomic Spaces</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">i232</span>– <span class="NLM_lpage">240</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btn162</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fbioinformatics%2Fbtn162" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=18586719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvV2rurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2008&pages=i232-240&author=Y.+Yamanishiauthor=M.+Arakiauthor=A.+Gutteridgeauthor=W.+Hondaauthor=M.+Kanehisa&title=Prediction+of+Drug-Target+Interaction+Networks+from+the+Integration+of+Chemical+and+Genomic+Spaces&doi=10.1093%2Fbioinformatics%2Fbtn162"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of drug-target interaction networks from the integration of chemical and genomic spaces</span></div><div class="casAuthors">Yamanishi, Yoshihiro; Araki, Michihiro; Gutteridge, Alex; Honda, Wataru; Kanehisa, Minoru</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">i232-i240</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: The identification of interactions between drugs and target proteins is a key area in genomic drug discovery.  Therefore, there is a strong incentive to develop new methods capable of detecting these potential drug-target interactions efficiently.  Results: In this article, we characterize four classes of drug-target interaction networks in humans involving enzymes, ion channels, G-protein-coupled receptors (GPCRs) and nuclear receptors, and reveal significant correlations between drug structure similarity, target sequence similarity and the drug-target interaction network topol.  We then develop new statistical methods to predict unknown drug-target interaction networks from chem. structure and genomic sequence information simultaneously on a large scale.  The originality of the proposed method lies in the formalization of the drug-target interaction inference as a supervised learning problem for a bipartite graph, the lack of need for 3D structure information of the target proteins, and in the integration of chem. and genomic spaces into a unified space that we call pharmacol. space'.  In the results, we demonstrate the usefulness of our proposed method for the prediction of the four classes of drug-target interaction networks.  Our comprehensively predicted drug-target interaction networks enable us to suggest many potential drug-target interactions and to increase research productivity toward genomic drug discovery.  Availability: Softwares are available upon request.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpO8mG-TwM7v7Vg90H21EOLACvtfcHk0ljJZAU9Kbe5gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvV2rurg%253D&md5=3fcd40a826851fc6a5d3758f00552e60</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtn162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtn162%26sid%3Dliteratum%253Aachs%26aulast%3DYamanishi%26aufirst%3DY.%26aulast%3DAraki%26aufirst%3DM.%26aulast%3DGutteridge%26aufirst%3DA.%26aulast%3DHonda%26aufirst%3DW.%26aulast%3DKanehisa%26aufirst%3DM.%26atitle%3DPrediction%2520of%2520Drug-Target%2520Interaction%2520Networks%2520from%2520the%2520Integration%2520of%2520Chemical%2520and%2520Genomic%2520Spaces%26jtitle%3DBioinformatics%26date%3D2008%26volume%3D24%26spage%3Di232%26epage%3D240%26doi%3D10.1093%2Fbioinformatics%2Fbtn162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref110"><div class="reference"><strong class="refLabel"><a href="#ref110" class="rightTabRefNumLink">110</a></strong><div class="NLM_citation" id="rightTab-cit110"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mei, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwoh, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span> <span> </span><span class="NLM_article-title">Drug-Target Interaction Prediction by Learning from Local Information and Neighbors</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">238</span>– <span class="NLM_lpage">245</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/bts670</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1093%2Fbioinformatics%2Fbts670" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=23162055" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFygsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2013&pages=238-245&author=J.+P.+Meiauthor=C.+K.+Kwohauthor=P.+Yangauthor=X.+L.+Liauthor=J.+Zheng&title=Drug-Target+Interaction+Prediction+by+Learning+from+Local+Information+and+Neighbors&doi=10.1093%2Fbioinformatics%2Fbts670"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Drug-target interaction prediction by learning from local information and neighbors</span></div><div class="casAuthors">Mei, Jian-Ping; Kwoh, Chee-Keong; Yang, Peng; Li, Xiao-Li; Zheng, Jie</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">238-245</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: In silico methods provide efficient ways to predict possible interactions between drugs and targets.  Supervised learning approach, bipartite local model (BLM), has recently been shown to be effective in prediction of drug-target interactions.  However, for drug-candidate compds. or target-candidate proteins that currently have no known interactions available, its pure  local' model is not able to be learned and hence BLM may fail to make correct prediction when involving such kind of new candidates.  Results: We present a simple procedure called neighbor-based interaction-profile inferring (NII) and integrate it into the existing BLM method to handle the new candidate problem.  Specifically, the inferred interaction profile is treated as label information and is used for model learning of new candidates.  This functionality is particularly important in practice to find targets for new drug-candidate compds. and identify targeting drugs for new target-candidate proteins.  Consistent good performance of the new BLM-NII approach has been obsd. in the expt. for the prediction of interactions between drugs and four categories of target proteins.  Esp. for nuclear receptors, BLM-NII achieves the most significant improvement as this dataset contains many drugs/targets with no interactions in the cross-validation.  This demonstrates the effectiveness of the NII strategy and also shows the great potential of BLM-NII for prediction of compd.-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqv1_LYslIEi7Vg90H21EOLACvtfcHk0liz8e57H8jLpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFygsbg%253D&md5=bcc7440053bb5716fedf0a4fbbc6dc9c</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbts670&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbts670%26sid%3Dliteratum%253Aachs%26aulast%3DMei%26aufirst%3DJ.%2BP.%26aulast%3DKwoh%26aufirst%3DC.%2BK.%26aulast%3DYang%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DX.%2BL.%26aulast%3DZheng%26aufirst%3DJ.%26atitle%3DDrug-Target%2520Interaction%2520Prediction%2520by%2520Learning%2520from%2520Local%2520Information%2520and%2520Neighbors%26jtitle%3DBioinformatics%26date%3D2013%26volume%3D29%26spage%3D238%26epage%3D245%26doi%3D10.1093%2Fbioinformatics%2Fbts670" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref111"><div class="reference"><strong class="refLabel"><a href="#ref111" class="rightTabRefNumLink">111</a></strong><div class="NLM_citation" id="rightTab-cit111"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, L.</span></span> <span> </span><span class="NLM_article-title">Revealing Drug-Target Interactions with Computational Models and Algorithms</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1714</span>, <span class="refDoi"> DOI: 10.3390/molecules24091714</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.3390%2Fmolecules24091714" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpvF2mt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2019&pages=1714&author=L.+Zhouauthor=Z.+Liauthor=J.+Yangauthor=G.+Tianauthor=F.+Liuauthor=H.+Wenauthor=L.+Pengauthor=M.+Chenauthor=J.+Xiangauthor=L.+Peng&title=Revealing+Drug-Target+Interactions+with+Computational+Models+and+Algorithms&doi=10.3390%2Fmolecules24091714"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">Revealing drug-target interactions with computational models and algorithms</span></div><div class="casAuthors">Zhou, Liqian; Li, Zejun; Yang, Jialiang; Tian, Geng; Liu, Fuxing; Wen, Hong; Peng, Li; Chen, Min; Xiang, Ju; Peng, Lihong</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1714</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Background: Identifying possible drug-target interactions (DTIs) has become an important task in drug research and development.  Although high-throughput screening is becoming available, exptl. methods narrow down the validation space because of extremely high cost, low success rate, and time consumption.  Therefore, various computational models have been exploited to infer DTI candidates.  Methods: We introduced relevant databases and packages, mainly provided a comprehensive review of computational models for DTI identification, including network-based algorithms and machine learning-based methods.  Specially, machine learning-based methods mainly include bipartite local model, matrix factorization, regularized least squares, and deep learning.  Results: Although computational methods have obtained significant improvement in the process of DTI prediction, these models have their limitations.  We discussed potential avenues for boosting DTI prediction accuracy as well as further directions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpopTa4kc7VZrVg90H21EOLACvtfcHk0liz8e57H8jLpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpvF2mt7w%253D&md5=f2f063cbcd310a0535cb6f75a2b96de9</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.3390%2Fmolecules24091714&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules24091714%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DTian%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DWen%26aufirst%3DH.%26aulast%3DPeng%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DM.%26aulast%3DXiang%26aufirst%3DJ.%26aulast%3DPeng%26aufirst%3DL.%26atitle%3DRevealing%2520Drug-Target%2520Interactions%2520with%2520Computational%2520Models%2520and%2520Algorithms%26jtitle%3DMolecules%26date%3D2019%26volume%3D24%26spage%3D1714%26doi%3D10.3390%2Fmolecules24091714" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref112"><div class="reference"><strong class="refLabel"><a href="#ref112" class="rightTabRefNumLink">112</a></strong><div class="NLM_citation" id="rightTab-cit112"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hastie, T.</span>; <span class="NLM_string-name">Tibshirani, R.</span>; <span class="NLM_string-name">Friedman, J.</span></span> <i>The Elements of Statistical Learning: Data Mining, Inference, and Prediction</i>, <span class="NLM_edition">2</span>nd ed.; <span class="NLM_publisher-name">Springer</span>, <span class="NLM_year">2009</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.1007%2F978-0-387-84858-7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=T.+Hastie&author=R.+Tibshirani&author=J.+Friedman&title=The+Elements+of+Statistical+Learning%3A+Data+Mining%2C+Inference%2C+and+Prediction"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1007%2F978-0-387-84858-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-0-387-84858-7%26sid%3Dliteratum%253Aachs%26aulast%3DHastie%26aufirst%3DT.%26btitle%3DThe%2520Elements%2520of%2520Statistical%2520Learning%253A%2520Data%2520Mining%252C%2520Inference%252C%2520and%2520Prediction%26pub%3DSpringer%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref113"><div class="reference"><strong class="refLabel"><a href="#ref113" class="rightTabRefNumLink">113</a></strong><div class="NLM_citation" id="rightTab-cit113"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mih, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palsson, B. O.</span></span> <span> </span><span class="NLM_article-title">Expanding the Uses of Genome-scale Models with Protein Structures</span>. <i>Mol. Syst. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>, <span class="NLM_elocation-id">e8601</span> <span class="refDoi"> DOI: 10.15252/msb.20188601</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=10.15252%2Fmsb.20188601" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=31777175" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVGht7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&author=N.+Mihauthor=B.+O.+Palsson&title=Expanding+the+Uses+of+Genome-scale+Models+with+Protein+Structures&doi=10.15252%2Fmsb.20188601"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit113R"><div class="casContent"><span class="casTitleNuber">113</span><div class="casTitle"><span class="NLM_cas:atitle">Expanding the uses of genome-scale models with protein structures</span></div><div class="casAuthors">Mih, Nathan; Palsson, Bernhard O.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Systems Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e8601</span>CODEN:
                <span class="NLM_cas:coden">MSBOC3</span>;
        ISSN:<span class="NLM_cas:issn">1744-4292</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Biol. is reaching a convergence point of its historic reductionist and modern holistic approaches to understanding the living system.  Structural biol. has historically taken the reductionist approach to deeply probe the inner workings of complex mol. machines.  In contrast, systems biol. and genome-scale modeling have organically grown out of the wealth of data now being generated by diverse omics measurements.  In the late 2000s, a proposed interdisciplinary field of structural systems biol. pitched the merger of these two approaches, with widespread applications in pharmacol., disease modeling, protein engineering, and evolutionary studies.  In this commentary, we highlight the challenges of integrating these two fields, with a focus on genome-scale metabolic modeling, and the novel findings that are made possible from such a merger.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlMY-Y1wr5f7Vg90H21EOLACvtfcHk0li3oia4QE-dHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVGht7zP&md5=2115cd81c3f187d5c0bbf65e7c17e50d</span></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.15252%2Fmsb.20188601&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.15252%252Fmsb.20188601%26sid%3Dliteratum%253Aachs%26aulast%3DMih%26aufirst%3DN.%26aulast%3DPalsson%26aufirst%3DB.%2BO.%26atitle%3DExpanding%2520the%2520Uses%2520of%2520Genome-scale%2520Models%2520with%2520Protein%2520Structures%26jtitle%3DMol.%2520Syst.%2520Biol.%26date%3D2019%26volume%3D15%26doi%3D10.15252%2Fmsb.20188601" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref114"><div class="reference"><strong class="refLabel"><a href="#ref114" class="rightTabRefNumLink">114</a></strong><div class="NLM_citation" id="rightTab-cit114"><span><span class="NLM_contrib-group"><span class="NLM_string-name">von Bertalanffy, L.</span></span> <i>General System Theory. Foundations, Development, Applications</i>; <span class="NLM_publisher-name">George Braziller, Inc.</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">1969</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1969&author=L.+von+Bertalanffy&title=General+System+Theory.+Foundations%2C+Development%2C+Applications"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3Dvon%2BBertalanffy%26aufirst%3DL.%26btitle%3DGeneral%2520System%2520Theory.%2520Foundations%252C%2520Development%252C%2520Applications%26pub%3DGeorge%2520Braziller%252C%2520Inc%26date%3D1969" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i24"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01989">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_81500"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b01989?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01989</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Short presentation of the basic concepts of network theory (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01989/suppl_file/jm9b01989_si_001.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01989/suppl_file/jm9b01989_si_001.pdf">jm9b01989_si_001.pdf (864.87 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01989&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01989%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-16%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01989" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799e115594e2290","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
