
<html lang="en"     class="pb-page"  data-request-id="c37e83a9-9f73-4b73-bbd2-6451f97c7764"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2016.59.issue-22;article:article:10.1021/acs.jmedchem.6b01118;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Rational Design of Cyclic Peptide Inhibitors of U2AF Homology Motif (UHM) Domains To Modulate Pre-mRNA Splicing" /></meta><meta name="dc.Creator" content="Pravin Kumar Ankush  Jagtap" /></meta><meta name="dc.Creator" content="Divita  Garg" /></meta><meta name="dc.Creator" content="Tobias G.  Kapp" /></meta><meta name="dc.Creator" content="Cindy L.  Will" /></meta><meta name="dc.Creator" content="Oliver  Demmer" /></meta><meta name="dc.Creator" content="Reinhard  Lührmann" /></meta><meta name="dc.Creator" content="Horst  Kessler" /></meta><meta name="dc.Creator" content="Michael  Sattler" /></meta><meta name="dc.Description" content="U2AF homology motifs (UHMs) are atypical RNA recognition motif domains that mediate critical protein–protein interactions during the regulation of alternative pre-mRNA splicing and other processes...." /></meta><meta name="Description" content="U2AF homology motifs (UHMs) are atypical RNA recognition motif domains that mediate critical protein–protein interactions during the regulation of alternative pre-mRNA splicing and other processes...." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="November 4, 2016" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.6b01118" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2016 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01118" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.6b01118" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01118" /></link>
        
    
    

<title>Rational Design of Cyclic Peptide Inhibitors of U2AF Homology Motif (UHM) Domains To Modulate Pre-mRNA Splicing | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.6b01118" /></meta><meta property="og:title" content="Rational Design of Cyclic Peptide Inhibitors of U2AF Homology Motif (UHM) Domains To Modulate Pre-mRNA Splicing" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-22/acs.jmedchem.6b01118/20161118/images/large/jm-2016-01118z_0005.jpeg" /></meta><meta property="og:description" content="U2AF homology motifs (UHMs) are atypical RNA recognition motif domains that mediate critical protein–protein interactions during the regulation of alternative pre-mRNA splicing and other processes. The recognition of UHM domains by UHM ligand motif (ULM) peptide sequences plays important roles during early steps of spliceosome assembly. Splicing factor 45 kDa (SPF45) is an alternative splicing factor implicated in breast and lung cancers, and splicing regulation of apoptosis-linked pre-mRNAs by SPF45 was shown to depend on interactions between its UHM domain and ULM motifs in constitutive splicing factors. We have developed cyclic peptide inhibitors that target UHM domains. By screening a focused library of linear and cyclic peptides and performing structure–activity relationship analysis, we designed cyclic peptides with 4-fold improved binding affinity for the SPF45 UHM domain compared to native ULM ligands and 270-fold selectivity to discriminate UHM domains from alternative and constitutive splicing factors. These inhibitors are useful tools to modulate and dissect mechanisms of alternative splicing regulation." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.6b01118"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.6b01118">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.6b01118&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.6b01118&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.6b01118&amp;href=/doi/10.1021/acs.jmedchem.6b01118" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 22</span><span class="cit-fg-pageRange">, 10190-10197</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/22" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.6b01122" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.6b01181" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Rational Design of Cyclic Peptide Inhibitors of U2AF Homology Motif (UHM) Domains To Modulate Pre-mRNA Splicing</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Pravin+Kumar+Ankush++Jagtap">Pravin Kumar Ankush Jagtap</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol "><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Divita++Garg">Divita Garg</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol "><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Tobias+G.++Kapp">Tobias G. Kapp</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Cindy+L.++Will">Cindy L. Will</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Oliver++Demmer">Oliver Demmer</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Reinhard++L%C3%BChrmann">Reinhard Lührmann</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Horst++Kessler">Horst Kessler</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Michael++Sattler">Michael Sattler</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Institute of Structural Biology, Helmholtz Zentrum München, Ingolstaedter Landstrasse 1, 85764 Neuherberg, Germany</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Center for Integrated Protein Science Munich (CIPSM), Department Chemie, Technische Universität München, Lichtenbergstrasse 4, 85747 Garching, Germany</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Institute for Advanced Study (IAS), Technische Universität München, Lichtenbergstrasse 4, 85747 Garching, Germany</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">Max Planck Institute for Biophysical Chemistry, Department of Cellular Biochemistry, Am Fassberg 11, 37077 Göttingen, Germany</span></div><div class="corresp-info"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#8dfeecf9f9e1e8ffcde5e8e1e0e5e2e1f9f7a0e0f8e8e3eee5e8e3a3e9e8"><span class="__cf_email__" data-cfemail="4437253030282136042c2128292c2b28303e692931212a272c212a6a2021">[email protected]</span></a>. Phone: +49-89-289-13418.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01118&amp;href=/doi/10.1021%2Facs.jmedchem.6b01118" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 22</span><span class="cit-pageRange">, 10190–10197</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">October 18, 2016</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>9 August 2016</li><li><span class="item_label"><b>Published</b> online</span>4 November 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 23 November 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.6b01118" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01118</a></div><div class="article_header-article-copyright"><strong>Copyright © 2016 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D10190%26pageCount%3D8%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DPravin%2BKumar%2BAnkush%2BJagtap%252C%2BDivita%2BGarg%252C%2BTobias%2BG.%2BKapp%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D59%26issueNum%3D22%26contentID%3Dacs.jmedchem.6b01118%26title%3DRational%2BDesign%2Bof%2BCyclic%2BPeptide%2BInhibitors%2Bof%2BU2AF%2BHomology%2BMotif%2B%2528UHM%2529%2BDomains%2BTo%2BModulate%2BPre-mRNA%2BSplicing%26numPages%3D8%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D10197%26publicationDate%3DNovember%2B2016">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.6b01118"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">897</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">13</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.6b01118" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Rational Design of Cyclic Peptide Inhibitors of U2AF Homology Motif (UHM) Domains To Modulate Pre-mRNA Splicing&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Pravin&quot;,&quot;last_name&quot;:&quot;Kumar Ankush Jagtap&quot;},{&quot;first_name&quot;:&quot;Divita&quot;,&quot;last_name&quot;:&quot;Garg&quot;},{&quot;first_name&quot;:&quot;Tobias&quot;,&quot;last_name&quot;:&quot;G. Kapp&quot;},{&quot;first_name&quot;:&quot;Cindy&quot;,&quot;last_name&quot;:&quot;L. Will&quot;},{&quot;first_name&quot;:&quot;Oliver&quot;,&quot;last_name&quot;:&quot;Demmer&quot;},{&quot;first_name&quot;:&quot;Reinhard&quot;,&quot;last_name&quot;:&quot;Lührmann&quot;},{&quot;first_name&quot;:&quot;Horst&quot;,&quot;last_name&quot;:&quot;Kessler&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;Sattler&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;11&quot;,&quot;day&quot;:&quot;04&quot;,&quot;issue&quot;:&quot;22&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;10190-10197&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.6b01118&quot;},&quot;abstract&quot;:&quot;U2AF homology motifs (UHMs) are atypical RNA recognition motif domains that mediate critical protein–protein interactions during the regulation of alternative pre-mRNA splicing and other processes. The recognition of UHM domains by UHM ligand motif (ULM) peptide sequences plays important roles during early steps of spliceosome assembly. Splicing factor 45 kDa (SPF45) is an alternative splicing factor implicated in breast and lung cancers, and splicing regulation of apoptosis-linked pre-mRNAs by SPF45 was shown to depend on interactions between its UHM domain and ULM motifs in constitutive splicing factors. We have developed cyclic peptide inhibitors that target UHM domains. By screening a focused library of linear and cyclic peptides and performing structure–activity relationship analysis, we designed cyclic peptides with 4-fold improved binding affinity for the SPF45 UHM domain compared to native ULM ligands and 270-fold selectivity to discriminate UHM domains from alternative and constitutive splicing facto&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01118&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01118" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01118&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01118" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01118&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01118" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01118&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01118&amp;href=/doi/10.1021/acs.jmedchem.6b01118" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.6b01118" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.6b01118" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.6b01118%26sid%3Dliteratum%253Aachs%26pmid%3D27753493%26genre%3Darticle%26aulast%3DJagtap%26date%3D2016%26atitle%3DRational%2BDesign%2Bof%2BCyclic%2BPeptide%2BInhibitors%2Bof%2BU2AF%2BHomology%2BMotif%2B%2528UHM%2529%2BDomains%2BTo%2BModulate%2BPre-mRNA%2BSplicing%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D22%26spage%3D10190%26epage%3D10197%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=291240" title="Cyclization">Cyclization</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/22" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-22/jmcmar.2016.59.issue-22/20161123/jmcmar.2016.59.issue-22.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-22/acs.jmedchem.6b01118/20161118/images/medium/jm-2016-01118z_0005.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-22/acs.jmedchem.6b01118/20161118/images/large/jm-2016-01118z_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01118&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">U2AF homology motifs (UHMs) are atypical RNA recognition motif domains that mediate critical protein–protein interactions during the regulation of alternative pre-mRNA splicing and other processes. The recognition of UHM domains by UHM ligand motif (ULM) peptide sequences plays important roles during early steps of spliceosome assembly. Splicing factor 45 kDa (SPF45) is an alternative splicing factor implicated in breast and lung cancers, and splicing regulation of apoptosis-linked pre-mRNAs by SPF45 was shown to depend on interactions between its UHM domain and ULM motifs in constitutive splicing factors. We have developed cyclic peptide inhibitors that target UHM domains. By screening a focused library of linear and cyclic peptides and performing structure–activity relationship analysis, we designed cyclic peptides with 4-fold improved binding affinity for the SPF45 UHM domain compared to native ULM ligands and 270-fold selectivity to discriminate UHM domains from alternative and constitutive splicing factors. These inhibitors are useful tools to modulate and dissect mechanisms of alternative splicing regulation.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55481" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55481" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Alternative splicing (AS) is an essential cellular process and greatly expands the coding capacity of eukaryotic genomes by generating multiple protein isoforms from a single primary transcript.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a> The regulation of AS involves the recognition of <i>cis</i> regulatory elements, i.e., short RNA sequence motifs, by <i>trans</i>-acting factors, i.e., RNA binding proteins.<a onclick="showRef(event, 'ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5">(3-5)</a> Aberrant splicing has been implicated in human diseases,<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6, 7)</a> and the role of AS in cancer progression suggests that targeting splicing regulation may be effective for cancer therapy.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8, 9)</a></div><div class="NLM_p">Splicing factor SPF45 was identified as a component of the spliceosome<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> and has been shown to activate a cryptic 3′ splice site in β-thalassemia.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> It harbors a C-terminal U2AF homology motif (UHM) domain that mediates protein–protein interactions. The SPF45 UHM domain is required to regulate alternative splicing of the pre-mRNA encoding the apoptosis promoting factor FAS.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Given that the switch in expression of pro- and antiapoptotic isoforms of FAS is tightly regulated,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> an imbalance in the FAS isoforms by overexpression of SPF45 could provide a means for tumor cells to escape apoptosis. This links overexpression of SPF45 to breast, lung, colon, and ovarian tumors<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> and to the reported multidrug resistance to anticancer drugs.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14, 15)</a></div><div class="NLM_p">UHM domains are atypical RNA recognition domains containing a signature Arg-Xaa-Phe (RXF) amino acid motif (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01118/suppl_file/jm6b01118_si_001.pdf" class="ext-link">Figure S1</a>). They were identified as protein recognition motifs in both subunits of the U2AF heterodimer<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16, 17)</a> and have since then been found in many other proteins with diverse biological functions.<a onclick="showRef(event, 'ref12 ref18 ref19 ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref12 ref18 ref19 ref20 ref21 ref22">(12, 18-22)</a> UHM domains recognize UHM ligand motif (ULM) peptide sequences that harbor a conserved tryptophan residue flanked by a stretch of basic and negatively charged residues.<a onclick="showRef(event, 'ref12 ref18 ref21'); return false;" href="javascript:void(0);" class="ref ref12 ref18 ref21">(12, 18, 21)</a> UHM–ULM interactions play a crucial role in early spliceosome assembly (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A,B), and a given UHM domain can bind to various ULM ligands. For example, SPF45 UHM binds to SF3b155, SF1, and U2AF65 ULMs,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> and U2AF65 UHM binds to all five ULMs from SF3b155 with low micromolar affinity.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23, 24)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-22/acs.jmedchem.6b01118/20161118/images/medium/jm-2016-01118z_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-22/acs.jmedchem.6b01118/20161118/images/large/jm-2016-01118z_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Schematic overview of UHM–ULM interactions during spliceosome assembly. UHM domains and ULM peptide motifs are shown in green and red, respectively. (B) Alignment of ULM sequences.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-22/acs.jmedchem.6b01118/20161118/images/large/jm-2016-01118z_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01118&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Here, we have developed a cyclic peptide UHM inhibitor that targets the SPF45 UHM with high selectivity as a molecular probe to delineate the contributions of UHM domains in splicing regulation. The cyclic peptide inhibitor is optimized to target the SPF45 UHM domain and exhibits 4-fold higher affinity and 270-fold selectivity compared to the constitutive splicing factor U2AF65. We show that the inhibitor is an efficient tool for studying molecular mechanisms of splicing regulation during early stages of spliceosome assembly.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31601" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31601" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Design of Minimal ULM Cyclic Peptide</h3><div class="NLM_p">Analysis of the crystal structure of the SPF45 UHM–SF3b155 ULM5 complex<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> (PDB accession: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2PEH">2PEH</a>) reveals that the ULM peptide adopts a β-turn with a hydrogen bond between the backbone amide of Glu340 and carbonyl of Arg337 (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01118/suppl_file/jm6b01118_si_001.pdf" class="ext-link">Figure S2</a>). Ser336 stabilizes this β-turn by forming an intrapeptide hydrogen bond between its side chain hydroxyl and the backbone amide of Trp338, although it does not contribute directly to the interaction with the UHM domain. To understand the requirement of a predefined tertiary structure of the ULM peptide, we replaced Ser336 by alanine and analyzed the thermodynamic contributions to binding by isothermal titration calorimetry (ITC).</div><div class="NLM_p">Removal of serine shows an 8-fold decrease in binding affinity owing to a simultaneous loss of binding enthalpy and entropy compared to the native ULM sequence (RKSRWDETP vs RKARWDETP; <a href="/doi/suppl/10.1021/acs.jmedchem.6b01118/suppl_file/jm6b01118_si_001.pdf" class="ext-link">Table S1</a>). In order to further understand the role of Ser336, we compared the binding affinity of two minimal ULM peptides, SRWDET and RWDET (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01118/suppl_file/jm6b01118_si_001.pdf" class="ext-link">Table S1</a>). Inclusion of serine leads to a significant gain of favorable entropy; however, this comes at the cost of binding enthalpy. This reflects enthalpy–entropy compensation effects and highlights the need to maintain sufficient peptide flexibility in order to reduce enthalpy losses by restricting the conformation of the peptide. As Ser336 supports the β-turn conformation, we hypothesized that the ULM peptide β-turn can be further stabilized via side chain cyclization of Lys335 and Glu340, which are not in direct contact with the UHM domain. Different types of cyclizations were used: forming an amide bond between the backbone carboxyl group of Glu340 (peptide <b>1</b>) or the side chain carboxyl group of Glu340 (peptide 3) to the side chain of Lys335 or the side chain of Glu340 to ornithine instead of Lys335 (peptide <b>2</b>). The rationale of these cyclizations was to explore different linker lengths and possible cyclization options and to maintain sufficient flexibility in the cyclized peptide (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). Although decreasing the linker length of the cyclized part in peptide <b>1</b> significantly reduced the unfavorable binding entropy contribution compared to the noncyclized peptide <b>0</b>, it also reduced the binding enthalpy. Nevertheless, peptides <b>2</b> and <b>3</b> exhibit higher binding enthalpy compared to peptide <b>1</b>. This highlights the importance of peptide flexibility to enable optimal contacts with the UHM domain. As peptide <b>3</b> has the highest binding affinity with a dissociation constant <i>K</i><sub>D</sub> = 1.4 μM, it was chosen for further optimization.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Binding Affinity of Different Cyclic Peptides Synthesized To Explore Different Linker Lengths for Cyclization<a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">peptide</th><th class="colsep0 rowsep0" align="center">sequence<a class="ref internalNav" href="#t1fn1" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char="±"><i>K</i><sub>D</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char="±">Δ<i>H</i><a class="ref internalNav" href="#t1fn2" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char="±">–<i>T</i>Δ<i>S</i><a class="ref internalNav" href="#t1fn2" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>0</b></td><td class="colsep0 rowsep0" align="left">KSRWDE</td><td class="colsep0 rowsep0" align="char" char="±">15.7 ± 0.98</td><td class="colsep0 rowsep0" align="char" char="±">–10.4 ± 36</td><td class="colsep0 rowsep0" align="char" char="±">3.8 ± 0.17</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="left">[sc,bb(KSRWDE)]</td><td class="colsep0 rowsep0" align="char" char="±">122.3 ± 3.9</td><td class="colsep0 rowsep0" align="char" char="±">–7.8 ± 0.23</td><td class="colsep0 rowsep0" align="char" char="±">2.5 ± 0.21</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="left">[sc,sc(OrnSRWDE)]</td><td class="colsep0 rowsep0" align="char" char="±">2.0 ± 0.3</td><td class="colsep0 rowsep0" align="char" char="±">–16.5 ± 0.55</td><td class="colsep0 rowsep0" align="char" char="±">8.7 ± 0.55</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="left">[sc,sc(KSRWDE)]</td><td class="colsep0 rowsep0" align="char" char="±">1.4 ± 0.27</td><td class="colsep0 rowsep0" align="char" char="±">–18.3 ± 1.3</td><td class="colsep0 rowsep0" align="char" char="±">10.3 ± 1.4</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup>a</sup><p class="last">Binding affinity was determined by isothermal titration calorimetry.</p></div><div class="footnote" id="t1fn1"><sup>b</sup><p class="last">Errors represent standard deviation of fitting errors calculated by error propagation.</p></div><div class="footnote" id="t1fn2"><sup>c</sup><p class="last">In kcal/mol.</p></div></div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Crystal Structure of SPF45 UHM–Peptide <b>3</b> Complex</h3><div class="NLM_p">To investigate the structural basis of the peptide <b>3</b> interaction, we determined the crystal structure of the SPF45 UHM domain bound to peptide <b>3</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01118/suppl_file/jm6b01118_si_001.pdf" class="ext-link">Table S2</a>). The structure shows clear electron density for the cyclic peptide in the ULM binding pocket of the UHM (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01118/suppl_file/jm6b01118_si_001.pdf" class="ext-link">Figure S3</a>). Overall, there are only minor structural differences in the UHM domain when bound to the linear versus the cyclic peptide. The backbone amides and carbonyls of cyclic peptide <b>3</b> form an extensive network of hydrogen bonds with the backbone amides of the UHM domain. The Trp338 side chain inserts into the hydrophobic tryptophan binding pocket and forms parallel and T-stacking interactions with the side chains of Arg375 and Phe377 in the UHM domain, respectively. Thus, critical interactions with the UHM domain are preserved between the linear and cyclic peptide bound forms (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-22/acs.jmedchem.6b01118/20161118/images/medium/jm-2016-01118z_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-22/acs.jmedchem.6b01118/20161118/images/large/jm-2016-01118z_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Co-crystal structure of SPF45 UHM in complex with peptide <b>3</b> inhibitor. The peptide and the UHM domain are shown in salmon and green, respectively. Yellow dashed lines indicate hydrogen bonds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-22/acs.jmedchem.6b01118/20161118/images/large/jm-2016-01118z_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01118&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The conformation of the cyclic peptide shows some differences compared to the linear peptide when bound to the SPF45 UHM domain (backbone coordinate rmsd = 0.97 Å; <a href="/doi/suppl/10.1021/acs.jmedchem.6b01118/suppl_file/jm6b01118_si_001.pdf" class="ext-link">Figure S4</a>). As expected, most of the structural differences are limited to the Lys335 and Glu340 side chains, which mediate the cyclization. Interestingly, when bound to the SPF45 UHM domain (PDB accession: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2PEH">2PEH</a>), the side chain of Lys335 does not show visible electron density, whereas the terminal atoms of Lys335 and Glu340 are clearly visible with the cyclic peptide. This indicates a significant rigidification of the peptide induced by the cyclization.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Structure-Based Optimization of Peptide <b>3</b></h3><div class="NLM_p">For further optimization of the affinity of peptide <b>3</b>, a focused library of seven peptides with different amino acid additions at carboxyl termini of Glu340 was synthesized (position R1 in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A) to enable either π–π or cation−π interactions with Tyr376 in the UHM domain (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>C). Addition of aromatic amino acids at position R1 exhibits a strong effect on favorable binding entropy (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). This presumably is linked to the hydrophobic effect<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> that arises from aromatic stacking of Tyr376 and the aromatic side chain in the cyclic peptide. However, as observed with other peptide ligands, the affinity increase obtained from favorable binding entropy is often counteracted by a decrease in binding enthalpy. Adding lysine at position R1 (peptide <b>10</b>) maintains the binding enthalpy and yields a ∼4-fold increase in binding affinity (<i>K</i><sub>D</sub> = 180 nM) for the cyclic peptide compared to the native linear peptide (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Binding Affinity of Cyclic Peptides Synthesized during Optimization<a class="ref internalNav" href="#tbl2-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">peptide</th><th class="colsep0 rowsep0" align="center">sequence<a class="ref internalNav" href="#t2fn1" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center" char="±"><i>K</i><sub>D</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char="±">Δ<i>H</i><a class="ref internalNav" href="#t2fn2" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center" char="±">–<i>T</i>Δ<i>S</i><a class="ref internalNav" href="#t2fn2" aria-label="c">c</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="left">[sc,sc(KSRWDE)]-H</td><td class="colsep0 rowsep0" align="char" char="±">0.96 ± 0.08</td><td class="colsep0 rowsep0" align="char" char="±">–10.0 ± 0.19</td><td class="colsep0 rowsep0" align="char" char="±">1.8 ± 0.19</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left">[sc,sc(KSRWDE)]-Y</td><td class="colsep0 rowsep0" align="char" char="±">1.85 ± 0.21</td><td class="colsep0 rowsep0" align="char" char="±">–9.2 ± 0.31</td><td class="colsep0 rowsep0" align="char" char="±">1.4 ± 0.32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="left">[sc,sc(KSRWDE)]-W</td><td class="colsep0 rowsep0" align="char" char="±">0.80 ± 0.14</td><td class="colsep0 rowsep0" align="char" char="±">–11.0 ± 0.54</td><td class="colsep0 rowsep0" align="char" char="±">2.6 ± 0.55</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">[sc,sc(KSRWDE)]-R</td><td class="colsep0 rowsep0" align="char" char="±">0.51 ± 0.12</td><td class="colsep0 rowsep0" align="char" char="±">–13.7 ± 0.76</td><td class="colsep0 rowsep0" align="char" char="±">5.2 ± 0.77</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">[sc,sc((N<sub>ε</sub>Me)KSRWDE)]-K</td><td class="colsep0 rowsep0" align="char" char="±">1.26 ± 0.11</td><td class="colsep0 rowsep0" align="char" char="±">–9.2 ± 0.20</td><td class="colsep0 rowsep0" align="char" char="±">1.1 ± 0.21</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="left">[sc,sc(KSRWDE)]-homoR</td><td class="colsep0 rowsep0" align="char" char="±">0.75 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="±">–20.2 ± 0.19</td><td class="colsep0 rowsep0" align="char" char="±">11.9 ± 0.19</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="left">[sc,sc(KSRWDE)]-K</td><td class="colsep0 rowsep0" align="char" char="±">0.18 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">–16.4 ± 0.18</td><td class="colsep0 rowsep0" align="char" char="±">7.2 ± 0.19</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl2-fn1"><div class="footnote" id="tbl2-fn1"><sup>a</sup><p class="last">Binding affinity was determined by isothermal titration calorimetry.</p></div><div class="footnote" id="t2fn1"><sup>b</sup><p class="last">Errors represent standard deviation of the fitting errors calculated by error propagation.</p></div><div class="footnote" id="t2fn2"><sup>c</sup><p class="last">In kcal/mol.</p></div></div></div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-22/acs.jmedchem.6b01118/20161118/images/medium/jm-2016-01118z_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-22/acs.jmedchem.6b01118/20161118/images/large/jm-2016-01118z_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) Schematic overview of the synthesized focused library of the cyclic peptides. The different amino acids and functional groups added to peptide <b>3</b> at the R1 and R2 positions are listed. (B) <sup>1</sup>H,<sup>15</sup>N HSQC spectra superposition of the SPF45 UHM domain bound to peptides <b>3</b> (black) and <b>10</b> (red) is shown. In the SPF45 UHM domain, residues Gly379, Gly378, and Phe377 exhibit large chemical shift changes induced by peptide <b>10</b> binding compared to peptide <b>3</b>. These residues are in close proximity to Tyr376. (C) Backbone amides of Gly379, Gly378, and Phe377 residues are shown with red spheres on the SPF45 UHM–peptide <b>3</b> complex structure. A red oval shows the attachment position of amino acid R1 in peptide <b>3</b> along with the direction in which it is expected to extend. (D) Selectivity profile of the designed cyclic peptides for the SPF45 and U2AF65 UHM domains (selectivity = <i>K</i><sub>D</sub><sup>U2AF65</sup>/<i>K</i><sub>D</sub><sup>SPF45</sup>; <a href="/doi/suppl/10.1021/acs.jmedchem.6b01118/suppl_file/jm6b01118_si_001.pdf" class="ext-link">Table S3</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-22/acs.jmedchem.6b01118/20161118/images/large/jm-2016-01118z_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01118&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To test whether the lysine at position R1 in peptide <b>10</b> indeed contacts Tyr376, we monitored chemical shift perturbations (CSPs) in <sup>1</sup>H,<sup>15</sup>N NMR correlation experiments upon addition of peptides <b>3</b> and <b>10</b> to <sup>15</sup>N-labeled SPF45 UHM domain. Overlaying the two spectra at saturating concentrations of the cyclic peptides shows that many signals exhibit similar chemical shifts in the bound state. However, significant chemical shift differences are found for the backbone amides of residues Phe377, Gly378, and Gly379, which flank Tyr376 (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B,C). These differential CSPs may reflect ring current effects induced by a conformational rearrangement of the Tyr376 side chain caused by stacking of attached lysine at position R1 in peptide <b>10</b> and changes in hydrogen bond strengths of the amide groups of these residues.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Selectivity of Peptide <b>10</b></h3><div class="NLM_p last">The linear ULM peptide exhibits some selectivity (∼9-fold) toward the U2AF65 UHM (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>D). The cyclization of the peptide (peptide <b>3</b>) alone does not lead to an increase in selectivity, as the cyclic peptide does not establish novel interactions with the UHM domain but mainly stabilizes the β-turn conformation. However, the lysine side chain variant of the cyclic peptide (peptide <b>10</b>) affords 270-fold selectivity against the U2AF65 UHM domain. Analysis of a multiple sequence alignment of UHM domains (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01118/suppl_file/jm6b01118_si_001.pdf" class="ext-link">Figure S1</a>) shows that besides the variable X position (Tyr376 in the SPF45 UHM) in the RXF motif, the residues following this motif (Gly378, Gly379) are variable among different UHMs, which might be exploited to discriminate UHMs and increase selectivity of inhibitors. Notably, the amide signals of these two residues exhibit the largest chemical shift differences upon titration with cyclic peptide <b>10</b>, suggesting that peptide <b>10</b> indeed exploits interactions with this region. The Lys addition in peptide <b>10</b> targets this variable region and increases affinity of the cyclic peptides, thereby providing enhanced selectivity and affinity against SPF45 UHM. This strategy may be used for the design of more potent, selective cyclic peptides against specific UHM domains.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> In Vitro Splicing Assays</h3><div class="NLM_p">We next tested the functional activity of peptide <b>10</b> using in vitro splicing assays. Assembly of the early spliceosomal complex E, which precedes formation of the spliceosomal A complex, depends on UHM–ULM interactions involving U2AF35, U2AF65, SF1, and SF3b155.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23, 24)</a> SF3b155-ULM5 has been previously shown to bind to both SPF45-UHM<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> and U2AF65-UHM<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23, 24)</a> with <i>K</i><sub>D</sub> = 0.7 and 6.7 μM, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01118/suppl_file/jm6b01118_si_001.pdf" class="ext-link">Supporting Information Tables S1 and S3</a>). On the other hand, U2AF35-ULM binds to U2AF65-UHM with nanomolar affinity (<i>K</i><sub>D</sub> = 135 nM).<a onclick="showRef(event, 'ref12 ref16'); return false;" href="javascript:void(0);" class="ref ref12 ref16">(12, 16)</a> This suggests that the cyclic peptide inhibitor may interfere with UHM–ULM interactions involving the SF3b155 ULM and thereby modulate splicing. To test this hypothesis, we monitored splicing and splicing complex formation in vitro with an IgM pre-mRNA as the splicing substrate<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> in the presence of peptide <b>10</b>. The native ULM5 of SF3b155 and its W338A variant served as positive and negative controls, respectively (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>A,B). Splicing was nearly abolished at the highest concentration of peptide <b>10</b> tested. Similar results were observed with the MINX pre-mRNA, which possesses a stronger 3′ splice site compared to IgM pre-mRNA (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01118/suppl_file/jm6b01118_si_001.pdf" class="ext-link">Figure S5</a>). However, the extent of splicing inhibition for IgM and MINX pre-mRNA differed significantly, i.e., 9% spliced product was formed relative to the no peptide control vs 29% at the highest tested concentrations of peptide <b>10</b>. This is consistent with a differential requirement of UHM–ULM interactions for splicing of substrates with weak versus strong 3′ splice sites.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-22/acs.jmedchem.6b01118/20161118/images/medium/jm-2016-01118z_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-22/acs.jmedchem.6b01118/20161118/images/large/jm-2016-01118z_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) In vitro splicing inhibition assays with peptide <b>10</b>. SF3b155 ULM (RKSRWDETP) and its mutant (RKSRADETP) were used as positive and negative controls, respectively. (B) Quantification of the splicing assay showing % spliced product (normalized to the control without added peptide) formed vs the peptide concentration. Error bars represent standard deviation of the % spliced product measured in two independent experiments. (C) Spliceosome complex formation analyzed by agarose gel electrophoresis. Peptide <b>10</b> inhibits complex A formation during spliceosome assembly, where U2AF UHM–ULM interactions play a crucial role.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-22/acs.jmedchem.6b01118/20161118/images/large/jm-2016-01118z_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01118&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To confirm that peptide <b>10</b> indeed modulates UHM–ULM interactions during spliceosome assembly and rule out that the observed splicing inhibition is due to other factors, splicing complex assembly was monitored on an agarose gel (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>C). In the absence of the peptide inhibitor, splicing complexes A, B, and C are formed during the course of the reaction. In the presence of peptide <b>10</b>, assembly is stalled already at the A complex stage, where SF1 is replaced by SF3b155,<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27, 28)</a> indicating that the cyclic peptide indeed inhibits UHM–ULM interactions, consistent with previous results.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64551" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64551" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Aberrant splicing regulation is the cause of a growing number of pathological conditions including cancer and neurodegenerative diseases. Early spliceosome assembly offers an attractive target for inhibiting and modulating splicing as most of the splicing regulation takes place during this stage. Moreover, stalling of early spliceosome assembly provides novel opportunities for the structural analysis of these early complexes, which have been difficult to capture so far. Here, we have developed inhibitors of protein–protein interactions between spliceosomal UHM domains and ULM peptides, which play a crucial role during early spliceosome assembly.</div><div class="NLM_p">We have designed a cyclic peptide that selectively binds the UHM domain of the alternative splicing factor SPF45 and discriminates against the UHM domain of the essential splicing factor U2AF65.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> By targeting the less conserved region in the UHM domains flanking the RXF motif, our cyclic peptide inhibitor not only shows better selectivity but also exhibits increased affinity for the SPF45 UHM domain compared to the linear peptides. This approach could also be used to design cyclic ULM peptide inhibitors with variable selectivity by targeting less conserved residues on the UHM binding surface. Such selective UHM inhibitors can thus be used to delineate the functions of individual UHM domains without interfering with the roles of other domains in multidomain UHM proteins.</div><div class="NLM_p">Our in vitro splicing data confirm that the novel inhibitors are able to modulate UHM–ULM interactions and stall spliceosome assembly during or prior to complex A formation. Surprisingly, the affinity of peptide <b>10</b> as well as the linear peptide is 500–1000-fold lower in cell extracts compared to in vitro binding studies. This is likely due to nonspecific interaction of the peptide with other cellular proteins. Further experiments will aim to optimize the activity of these peptides in vitro and in vivo. Nevertheless, we show that peptide <b>10</b> inhibits splicing at an early stage prior to the formation of the spliceosomal A complex. This presumably involves an inhibition of the U2AF65 UHM domain, which plays a critical role at this stage. Thus, although peptide <b>10</b> has higher affinity and selectivity for the SPF45 UHM domain in vitro, it is nevertheless able to inhibit early spliceosome assembly. On the other hand, the functional data may also indicate a potential role for SPF45 in early spliceosome assembly. This is an interesting possible function of SPF45 that is suggested by our data and will be the starting point for future studies.</div><div class="NLM_p last">Our results represent a first proof of principle showing that spliceosome assembly can be stalled at initial stages by targeting UHM–ULM interactions, where most of the splicing regulation takes place. A number of splicing inhibitors have been reported in recent years.<a onclick="showRef(event, 'ref8 ref30'); return false;" href="javascript:void(0);" class="ref ref8 ref30">(8, 30)</a> However, the molecular targets and modes of action of these inhibitors are poorly understood. Notably, most of these inhibitors are natural products, which renders their chemical modification and optimization difficult. Also, most of these inhibitors interfere with the splicing reaction at later stages. In this respect, our UHM inhibitor is distinct in two aspects: (1) the structural and rational design approach is based on detailed knowledge of the target and the mode of action involving UHM inhibition, which is confirmed by our splicing assays, and (2) the inhibitor opens novel ways to modulate splicing and interfere with spliceosome assembly at early stages, where alternative splicing is typically regulated by alternative splicing factors. Importantly, the fact that our UHM inhibitor is able to stall spliceosome assembly at an early stage opens the possibility for biochemical and structural studies of very early splicing complexes.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30783" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30783" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Protein Expression and Purification</h3><div class="NLM_p last">SPF45 was expressed and purified as mentioned previously.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Briefly, protein was expressed in BL21 (DE3) cells as a His-Ztag fusion protein with overnight induction at 20 °C in either LB or minimal media supplemented with 0.5 g/L <sup>15</sup>NH<sub>4</sub>Cl. Protein expression was induced with 0.5 mM IPTG. Protein was purified by affinity chromatography using Ni-NTA resin, followed by overnight TEV cleavage of the His-Ztag, ion exchange, and size exclusion chromatography. Purified protein was then concentrated to 10 mg/mL and stored in aliquots at −80 °C until further use.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Crystallization of the SPF45 UHM–Peptide <b>3</b> Complex</h3><div class="NLM_p last">Crystals were obtained by mixing 2 μL (10 mg/mL in 20 mM Tris, pH 7, 150 mM NaCl, and 1 mM DTT) of protein (SPF45 UHM domain and peptide <b>3</b> mixed in 1:1.5 molar ratio) and 2 μL of reservoir solution (50 mM MES, pH 6, 70% MPD) by the hanging drop method. Thin plate crystals were obtained in 5–7 days, which were flash frozen in liquid nitrogen for data collection. Data sets were collected at the PXIII beamline at the Swiss Light Source and were integrated and scaled with the XDS package.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> The structure was solved via molecular replacement using the native structure of SPF45 as a search model (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2PE8">2PE8</a>) using Phaser<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> and refined in Refmac<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> from the CCP4 suite.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Isothermal Titration Calorimetry Experiments</h3><div class="NLM_p last">Isothermal titration calorimetry experiments were performed with an ITC200 Microcal system. The SPF45 UHM domain protein at concentrations ranging from 5 to 20 μM was used in the cell and was titrated with 10-fold concentrated peptides in the syringe with a 1.5 μL injection volume. The data was fitted to a one-site binding model with the Origin software provided with the instrument.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> NMR Titrations</h3><div class="NLM_p last"><sup>1</sup>H,<sup>15</sup>N HSQC NMR spectra were acquired at 298 K using an AVIII600 Bruker NMR spectrometer equipped with a cryogenic probe. 50 μM <sup>15</sup>N-labeled SPF45 UHM domain in 50 mM Kpi, 150 mM NaCl, 1 mM DTT, pH 6.8 and 10% D<sub>2</sub>O was titrated with 2× cyclic peptide. All spectra were processed using NMRPipe/Draw<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> and analyzed using CCPN analysis<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> software.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> In Vitro Splicing Assays</h3><div class="NLM_p last">Splicing reactions containing 45% HeLa nuclear extract were prepared essentially as described previously<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> (60 mM KCl, 3 mM MgCl<sub>2</sub>, 2 mM ATP, 20 mM creatine phosphate, and 10 nM uniformly <sup>32</sup>P-labeled, m7G-capped MINX<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> or IgM<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> pre-mRNA) and were incubated at 30 °C for the indicated times. Reactions were supplemented with RKSRWDETP, RKSRADETP, or peptide <b>10</b> to a final concentration of 25–500 μM and incubated for 20 min on ice prior to the addition of the pre-mRNA and subsequent incubation at 30 °C. RNA was recovered at the indicated time points and separated on a 14% denaturing polyacrylamide gel. Unspliced pre-mRNA and splicing intermediates and products were detected using a Typhoon PhosphoImager (GE Healthcare). Spliceosomal complex formation was analyzed by agarose gel electrophoresis in the presence of 0.65 μg/μL heparin,<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> and bands were visualized by autoradiography or with a Typhoon PhosphoImager. The pre-mRNA and splicing intermediates and products (spliced mRNA and excised lariat intron) were quantified using ImageQuantTL (GE Healthcare). The percent spliced product was calculated by dividing the amount of mRNA and spliced lariat intron by the amount of pre-mRNA/lariat-intermediate/splicing products and multiplying by 100. Percent spliced product values were then normalized to the control (i.e., no added peptide) spliced product value, which was set to 100%.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Synthesis of Linear Peptides</h3><div class="NLM_p last">Peptides RWDET, SRWDET, RKARWDETP, and KSRWDE were synthesized on a solid support (trityl chloride resin, Intavis, Germany) using a standard Fmoc procedure. Couplings were performed using 2 equiv of amino acid, 2 equiv of HATU, 2 equiv of HOAt, and 5 equiv of DIPEA. The peptide was cleaved from the resin using a mixture containing 95% TFA, 2.5% H<sub>2</sub>O, and 2.5% TIPS for 1 h. After evaporation of the solvent, the peptide was taken up in a mixture of ACN/H<sub>2</sub>O and purified using semipreparative HPLC. The peptide was obtained in good purity (>95%) as a white powder.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Synthesis of Peptide <b>1</b></h3><div class="NLM_p last">The orthogonally protected, linear peptide Boc-Lys(Fmoc)-Ser(<i>t</i>Bu)-Arg(Pbf)-Trp(Boc)-Asp(O<i>t</i>Bu)-Glu(O<i>t</i>Bu)-OH was synthesized on a solid support (trityl chloride resin, Intavis, Germany) using a standard Fmoc procedure. Couplings were performed using 2 equiv of amino acid, 2 equiv of HATU, 2 equiv of HOAt, and 5 equiv of DIPEA. After the final Fmoc-deprotection on the solid support, the peptide was cleaved from the resin using a solution of 20% hexafluoroisopropanol in DCM. After evaporation of the solvent, peptide cyclization was achieved by dissolution of the peptide in DMF (app. 10–2 M), followed by subsequent addition of solid NaHCO<sub>3</sub> (5 equiv) and DPPA (3 equiv).<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Compared to procedures using standard peptide coupling reagents for cyclization, this technique allows the reaction to proceed under mild conditions corresponding to the dilution principle in cyclization as the low solubility of the base allows cyclization only at the interface between the solvent and basic solid. This avoids cyclodimerization reactions and leads to a relatively low racemization rate despite the long reaction time of approximately 12 h. After evaporation of the solvent, the peptide was subsequently treated with a solution of 85% TFA, 10% DCM, 2.5% TIPS, and 2.5% H<sub>2</sub>O to deprotect the acid-labile side chain protecting groups. Afterward, the peptide was purified by semipreparative HPLC and lyophilized, yielding a colorless powder. The peptide was obtained in good purity (>95%).</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Synthesis of Peptides <b>2</b> and <b>3</b></h3><div class="NLM_p last">The orthogonally protected, linear peptides Boc-Lys(Fmoc)-Ser(<i>t</i>Bu)-Arg(Pbf)-Trp(Boc)-Asp(O<i>t</i>Bu)-Glu(OAllyl) and Boc-Orn(Fmoc)-Ser(<i>t</i>Bu)-Arg(Pbf)-Trp(Boc)-Asp(O<i>t</i>Bu)-Glu(OAllyl) were synthesized on a solid support (trityl chloride resin, Intavis, Germany) using a standard Fmoc procedure. Couplings were performed using 2 equiv of amino acid, 2 equiv of HATU, 2 equiv of HOAt, and 5 equiv of DIPEA. After final Fmoc and Allyl-deprotection on the solid support, the side chains of the peptide’s lysine (or ornithine, respectively) and glutamic acid were cyclized on the resin by incubating the orthogonally deprotected peptide for 1 h with a solution of 2 equiv of HATU, 2 equiv of HOAt, and 5 equiv of DIPEA in DMF. After completion of the reaction (monitoring by LC-MS), the peptide was cleaved from the resin using a 20% HFIP solution in DCM. After evaporation of the solvent, the peptide was treated with a solution of 85% TFA, 10% DCM, 2.5% TIPS, and 2.5% H<sub>2</sub>O to deprotect the acid-labile side chain protecting groups. Afterward, the peptide was purified by semipreparative HPLC and lyophilized, yielding a colorless powder. The peptide was obtained in good purity (>95%).</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Synthesis of the Precursor [sc,sc(Boc-KS(<i>t</i>Bu)R(Pbf)W(Boc)D(O<i>t</i>Bu)E)]-OH</h3><div class="NLM_p last">The orthogonally protected, linear peptide Boc-Lys(Fmoc)-Ser(<i>t</i>Bu)-Arg(Pbf)-Trp(Boc)-Asp(O<i>t</i>Bu)-Glu(OAllyl) was synthesized on a solid support (trityl chloride resin, Intavis, Germany) using a standard Fmoc procedure. Couplings were performed using 2 equiv of amino acid, 2 equiv of HATU, 2 equiv of HOAt, and 5 equiv of DIPEA. After final Fmoc and Allyl-deprotection on the solid support, the side chains of the peptide’s lysine and glutamic acid were cyclized on the resin by incubating the orthogonally deprotected peptide for 1 h with a solution of 2 equiv of HATU, 2 equiv of HOAt, and 5 equiv of DIPEA in DMF. After completion of the reaction (monitoring by LC-MS), the peptide was cleaved from the resin using a 20% HFIP solution in DCM. After evaporation of the solvent, the product was dissolved in H<sub>2</sub>O/ACN and lyophilized overnight. The peptide [sc,sc(Boc-KS(<i>t</i>Bu)R(Pbf)W(Boc)D(O<i>t</i>Bu)E)]-OH was obtained in good purity (>90%) as a white powder.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Synthesis of Peptides <b>4</b>–<b>10</b></h3><div class="NLM_p last">The corresponding amino acid (e.g., His for peptide <b>4</b>) was loaded onto trityl chloride resin (Intavis, Germany) and Fmoc-deprotected. Subsequently, the resin was incubated with a solution of the compound [sc,sc(Boc-KS(<i>t</i>Bu)R(Pbf)W(Boc)D(O<i>t</i>Bu)E)]-OH (1.5 equiv), HATU (1.5 equiv), HOAt (1.5 equiv), and DIPEA (3 equiv) in DMF until all resin-bound starting material was consumed (at least 3 h). The reaction was monitored using LC-MS. After completion of the reaction, the coupled peptide was cleaved from the resin (20% HFIP/DCM) and subsequently treated with a solution of 85% TFA, 10% DCM, 2.5% TIPS, and 2.5% H<sub>2</sub>O to deprotect the acid-labile side chain protecting groups. Afterward, the peptide was purified by semipreparative HPLC and lyophilized, yielding a colorless powder. The peptides were obtained in good purity (>95%).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01118">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58735" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58735" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.6b01118" class="ext-link">10.1021/acs.jmedchem.6b01118</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Dissociation constants, X-ray crystallography statistics, analytical data for synthesized peptides, sequence alignment, structures, electron density, and splicing data (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01118/suppl_file/jm6b01118_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01118/suppl_file/jm6b01118_si_001.pdf">jm6b01118_si_001.pdf (905.38 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates and structure factors for the SPF45 UHM domain in complex with cyclic peptide <b>3</b> have been deposited in the PDB with accession number <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LSO">5LSO</a>.</p><div class="testing" data-doi="10.1021/acs.jmedchem.6b01118" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31851" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31851" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael Sattler</span> - <span class="hlFld-Affiliation affiliation">Institute
of Structural Biology, Helmholtz Zentrum
München, Ingolstaedter
Landstrasse 1, 85764 Neuherberg, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Center
for Integrated Protein Science Munich (CIPSM), Department Chemie, Technische Universität München, Lichtenbergstrasse 4, 85747 Garching, Germany</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#94e7f5e0e0f8f1e6d4fcf1f8f9fcfbf8e0eeb9f9e1f1faf7fcf1fabaf0f1"><span class="__cf_email__" data-cfemail="7a091b0e0e161f083a121f16171215160e0057170f1f1419121f14541e1f">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Pravin Kumar Ankush Jagtap</span> - <span class="hlFld-Affiliation affiliation">Institute
of Structural Biology, Helmholtz Zentrum
München, Ingolstaedter
Landstrasse 1, 85764 Neuherberg, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Center
for Integrated Protein Science Munich (CIPSM), Department Chemie, Technische Universität München, Lichtenbergstrasse 4, 85747 Garching, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Divita Garg</span> - <span class="hlFld-Affiliation affiliation">Institute
of Structural Biology, Helmholtz Zentrum
München, Ingolstaedter
Landstrasse 1, 85764 Neuherberg, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Center
for Integrated Protein Science Munich (CIPSM), Department Chemie, Technische Universität München, Lichtenbergstrasse 4, 85747 Garching, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tobias G. Kapp</span> - <span class="hlFld-Affiliation affiliation">Center
for Integrated Protein Science Munich (CIPSM), Department Chemie, Technische Universität München, Lichtenbergstrasse 4, 85747 Garching, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
for Advanced Study (IAS), Technische Universität
München, Lichtenbergstrasse
4, 85747 Garching, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cindy L. Will</span> - <span class="hlFld-Affiliation affiliation">Max
Planck Institute for Biophysical Chemistry, Department of Cellular Biochemistry, Am Fassberg 11, 37077 Göttingen, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Oliver Demmer</span> - <span class="hlFld-Affiliation affiliation">Center
for Integrated Protein Science Munich (CIPSM), Department Chemie, Technische Universität München, Lichtenbergstrasse 4, 85747 Garching, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
for Advanced Study (IAS), Technische Universität
München, Lichtenbergstrasse
4, 85747 Garching, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Reinhard Lührmann</span> - <span class="hlFld-Affiliation affiliation">Max
Planck Institute for Biophysical Chemistry, Department of Cellular Biochemistry, Am Fassberg 11, 37077 Göttingen, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Horst Kessler</span> - <span class="hlFld-Affiliation affiliation">Center
for Integrated Protein Science Munich (CIPSM), Department Chemie, Technische Universität München, Lichtenbergstrasse 4, 85747 Garching, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
for Advanced Study (IAS), Technische Universität
München, Lichtenbergstrasse
4, 85747 Garching, Germany</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>D.G., O.D., M.S., and H.K. designed the early cyclic peptides, O.D. synthesized the peptides, and D.G. performed the ITC experiments. P.K.A.J., T.G.K., M.S., and H.K. designed the cyclic peptides in the later part of the project, T.G.K. synthesized the peptides, and P.K.A.J. performed the ITC experiments. P.K.A.J. crystallized and determined the structure of UHM–peptide 3 complex and performed NMR experiments. C.L.W. performed the in vitro splicing assays. P.K.A.J. and M.S. wrote major parts of the manuscript. All authors have given approval to the final version of the manuscript.</p><span class="author-information-subsection-header">Author Contributions</span><p>P.K.A.J. and D.G. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d49e1292-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i26">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35336" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35336" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Gabi Heyne for help with in vitro splicing assays. P.K.A.J acknowledges Boehringer Ingelheim Fonds for a doctoral fellowship. This work was supported by the DFG (grants SFB1035 and GRK1721 to M.S. and SFB806, project A1, to R.L.).</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i27" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i27"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i28" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i28"> Abbreviations Used</h2><tr><td class="NLM_term">UHM</td><td class="NLM_def"><p class="first last">U2AF homology motif</p></td></tr><tr><td class="NLM_term">ULM</td><td class="NLM_def"><p class="first last">U2AF ligand motif</p></td></tr><tr><td class="NLM_term"><i>K</i><sub>D</sub></td><td class="NLM_def"><p class="first last">equilibrium dissociation constant</p></td></tr><tr><td class="NLM_term">Δ<i>H</i></td><td class="NLM_def"><p class="first last">enthalpy</p></td></tr><tr><td class="NLM_term">Δ<i>S</i></td><td class="NLM_def"><p class="first last">entropy</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07208" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07208" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 40 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Nilsen, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graveley, B. R.</span><span> </span><span class="NLM_article-title">Expansion of the eukaryotic proteome by alternative splicing</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">463</span><span class="NLM_x">, </span> <span class="NLM_fpage">457</span><span class="NLM_x">–</span> <span class="NLM_lpage">463</span><span class="refDoi"> DOI: 10.1038/nature08909</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1038%2Fnature08909" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=20110989" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1SltbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=463&publication_year=2010&pages=457-463&author=T.+W.+Nilsenauthor=B.+R.+Graveley&title=Expansion+of+the+eukaryotic+proteome+by+alternative+splicing&doi=10.1038%2Fnature08909"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Expansion of the eukaryotic proteome by alternative splicing</span></div><div class="casAuthors">Nilsen, Timothy W.; Graveley, Brenton R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">463</span>
        (<span class="NLM_cas:issue">7280</span>),
    <span class="NLM_cas:pages">457-463</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The collection of components required to carry out the intricate processes involved in generating and maintaining a living, breathing and, sometimes, thinking organism is staggeringly complex.  Where do all of the parts come from.  Early ests. stated that about 100,000 genes would be required to make up a mammal; however, the actual no. is less than one-quarter of that, barely four times the no. of genes in budding yeast.  It is now clear that the 'missing' information is in large part provided by alternative splicing, the process by which multiple different functional mRNAs, and therefore proteins, can be synthesized from a single gene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeWu61GQZ5hbVg90H21EOLACvtfcHk0liA2w8o3tdasQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1SltbY%253D&md5=133169961152db13f6bcf6e3d4b5eb6a</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnature08909&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08909%26sid%3Dliteratum%253Aachs%26aulast%3DNilsen%26aufirst%3DT.%2BW.%26aulast%3DGraveley%26aufirst%3DB.%2BR.%26atitle%3DExpansion%2520of%2520the%2520eukaryotic%2520proteome%2520by%2520alternative%2520splicing%26jtitle%3DNature%26date%3D2010%26volume%3D463%26spage%3D457%26epage%3D463%26doi%3D10.1038%2Fnature08909" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Wahl, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Will, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luhrmann, R.</span><span> </span><span class="NLM_article-title">The spliceosome: design principles of a dynamic RNP machine</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">136</span><span class="NLM_x">, </span> <span class="NLM_fpage">701</span><span class="NLM_x">–</span> <span class="NLM_lpage">718</span><span class="refDoi"> DOI: 10.1016/j.cell.2009.02.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1016%2Fj.cell.2009.02.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=19239890" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADC%252BD1MXkvFGksbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2009&pages=701-718&author=M.+C.+Wahlauthor=C.+L.+Willauthor=R.+Luhrmann&title=The+spliceosome%3A+design+principles+of+a+dynamic+RNP+machine&doi=10.1016%2Fj.cell.2009.02.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The spliceosome: Design principles of a dynamic RNP machine</span></div><div class="casAuthors">Wahl, Markus C.; Will, Cindy L.; Luehrmann, Reinhard</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">701-718</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Ribonucleoproteins (RNPs) mediate key cellular functions such as gene expression and its regulation.  Whereas most RNP enzymes are stable in compn. and harbor preformed active sites, the spliceosome, which removes noncoding introns from precursor mRNAs (pre-mRNAs), follows fundamentally different strategies.  In order to provide both accuracy to the recognition of reactive splice sites in the pre-mRNA and flexibility to the choice of splice sites during alternative splicing, the spliceosome exhibits exceptional compositional and structural dynamics that are exploited during substrate-dependent complex assembly, catalytic activation, and active site remodeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphwXkeaEfBWLVg90H21EOLACvtfcHk0liA2w8o3tdasQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXkvFGksbk%253D&md5=589aa1782ba069c55bff0b6bb7762b84</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2009.02.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2009.02.009%26sid%3Dliteratum%253Aachs%26aulast%3DWahl%26aufirst%3DM.%2BC.%26aulast%3DWill%26aufirst%3DC.%2BL.%26aulast%3DLuhrmann%26aufirst%3DR.%26atitle%3DThe%2520spliceosome%253A%2520design%2520principles%2520of%2520a%2520dynamic%2520RNP%2520machine%26jtitle%3DCell%26date%3D2009%26volume%3D136%26spage%3D701%26epage%3D718%26doi%3D10.1016%2Fj.cell.2009.02.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Xiong, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alipanahi, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bretschneider, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merico, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuen, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hua, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gueroussov, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Najafabadi, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morris, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barash, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krainer, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jojic, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scherer, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blencowe, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frey, B. J.</span><span> </span><span class="NLM_article-title">RNA splicing. The human splicing code reveals new insights into the genetic determinants of disease</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">347</span><span class="NLM_x">, </span> <span class="NLM_fpage">1254806</span><span class="refDoi"> DOI: 10.1126/science.1254806</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1126%2Fscience.1254806" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=25525159" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A280%3ADC%252BC2MzosV2rug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=347&publication_year=2015&pages=1254806&author=H.+Y.+Xiongauthor=B.+Alipanahiauthor=L.+J.+Leeauthor=H.+Bretschneiderauthor=D.+Mericoauthor=R.+K.+Yuenauthor=Y.+Huaauthor=S.+Gueroussovauthor=H.+S.+Najafabadiauthor=T.+R.+Hughesauthor=Q.+Morrisauthor=Y.+Barashauthor=A.+R.+Krainerauthor=N.+Jojicauthor=S.+W.+Schererauthor=B.+J.+Blencoweauthor=B.+J.+Frey&title=RNA+splicing.+The+human+splicing+code+reveals+new+insights+into+the+genetic+determinants+of+disease&doi=10.1126%2Fscience.1254806"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">RNA splicing. The human splicing code reveals new insights into the genetic determinants of disease</span></div><div class="casAuthors">Xiong Hui Y; Alipanahi Babak; Lee Leo J; Najafabadi Hamed S; Bretschneider Hannes; Merico Daniele; Yuen Ryan K C; Hua Yimin; Krainer Adrian R; Gueroussov Serge; Hughes Timothy R; Morris Quaid; Barash Yoseph; Jojic Nebojsa; Scherer Stephen W; Blencowe Benjamin J; Frey Brendan J</div><div class="citationInfo"><span class="NLM_cas:title">Science (New York, N.Y.)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">347</span>
        (<span class="NLM_cas:issue">6218</span>),
    <span class="NLM_cas:pages">1254806</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">To facilitate precision medicine and whole-genome annotation, we developed a machine-learning technique that scores how strongly genetic variants affect RNA splicing, whose alteration contributes to many diseases.  Analysis of more than 650,000 intronic and exonic variants revealed widespread patterns of mutation-driven aberrant splicing.  Intronic disease mutations that are more than 30 nucleotides from any splice site alter splicing nine times as often as common variants, and missense exonic disease mutations that have the least impact on protein function are five times as likely as others to alter splicing.  We detected tens of thousands of disease-causing mutations, including those involved in cancers and spinal muscular atrophy.  Examination of intronic and exonic variants found using whole-genome sequencing of individuals with autism revealed misspliced genes with neurodevelopmental phenotypes.  Our approach provides evidence for causal variants and should enable new discoveries in precision medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQChTiwuM_BhekdrW1FkXG-fW6udTcc2eY3zyKg8m5VgLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MzosV2rug%253D%253D&md5=3edc5e7004277b12dc3fd9cd06430f08</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1126%2Fscience.1254806&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1254806%26sid%3Dliteratum%253Aachs%26aulast%3DXiong%26aufirst%3DH.%2BY.%26aulast%3DAlipanahi%26aufirst%3DB.%26aulast%3DLee%26aufirst%3DL.%2BJ.%26aulast%3DBretschneider%26aufirst%3DH.%26aulast%3DMerico%26aufirst%3DD.%26aulast%3DYuen%26aufirst%3DR.%2BK.%26aulast%3DHua%26aufirst%3DY.%26aulast%3DGueroussov%26aufirst%3DS.%26aulast%3DNajafabadi%26aufirst%3DH.%2BS.%26aulast%3DHughes%26aufirst%3DT.%2BR.%26aulast%3DMorris%26aufirst%3DQ.%26aulast%3DBarash%26aufirst%3DY.%26aulast%3DKrainer%26aufirst%3DA.%2BR.%26aulast%3DJojic%26aufirst%3DN.%26aulast%3DScherer%26aufirst%3DS.%2BW.%26aulast%3DBlencowe%26aufirst%3DB.%2BJ.%26aulast%3DFrey%26aufirst%3DB.%2BJ.%26atitle%3DRNA%2520splicing.%2520The%2520human%2520splicing%2520code%2520reveals%2520new%2520insights%2520into%2520the%2520genetic%2520determinants%2520of%2520disease%26jtitle%3DScience%26date%3D2015%26volume%3D347%26spage%3D1254806%26doi%3D10.1126%2Fscience.1254806" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Ray, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kazan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, K. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weirauch, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Najafabadi, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gueroussov, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Na, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irimia, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matzat, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dale, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yarosh, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nabet, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mecenas, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laishram, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qiao, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lipshitz, H. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piano, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corbett, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carstens, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frey, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Penalva, L. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, E. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fraser, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blencowe, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morris, Q. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, T. R.</span><span> </span><span class="NLM_article-title">A compendium of RNA-binding motifs for decoding gene regulation</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">499</span><span class="NLM_x">, </span> <span class="NLM_fpage">172</span><span class="NLM_x">–</span> <span class="NLM_lpage">177</span><span class="refDoi"> DOI: 10.1038/nature12311</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1038%2Fnature12311" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=23846655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVyksbjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=499&publication_year=2013&pages=172-177&author=D.+Rayauthor=H.+Kazanauthor=K.+B.+Cookauthor=M.+T.+Weirauchauthor=H.+S.+Najafabadiauthor=X.+Liauthor=S.+Gueroussovauthor=M.+Albuauthor=H.+Zhengauthor=A.+Yangauthor=H.+Naauthor=M.+Irimiaauthor=L.+H.+Matzatauthor=R.+K.+Daleauthor=S.+A.+Smithauthor=C.+A.+Yaroshauthor=S.+M.+Kellyauthor=B.+Nabetauthor=D.+Mecenasauthor=W.+Liauthor=R.+S.+Laishramauthor=M.+Qiaoauthor=H.+D.+Lipshitzauthor=F.+Pianoauthor=A.+H.+Corbettauthor=R.+P.+Carstensauthor=B.+J.+Freyauthor=R.+A.+Andersonauthor=K.+W.+Lynchauthor=L.+O.+Penalvaauthor=E.+P.+Leiauthor=A.+G.+Fraserauthor=B.+J.+Blencoweauthor=Q.+D.+Morrisauthor=T.+R.+Hughes&title=A+compendium+of+RNA-binding+motifs+for+decoding+gene+regulation&doi=10.1038%2Fnature12311"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">A compendium of RNA-binding motifs for decoding gene regulation</span></div><div class="casAuthors">Ray, Debashish; Kazan, Hilal; Cook, Kate B.; Weirauch, Matthew T.; Najafabadi, Hamed S.; Li, Xiao; Gueroussov, Serge; Albu, Mihai; Zheng, Hong; Yang, Ally; Na, Hong; Irimia, Manuel; Matzat, Leah H.; Dale, Ryan K.; Smith, Sarah A.; Yarosh, Christopher A.; Kelly, Seth M.; Nabet, Behnam; Mecenas, Desirea; Li, Weimin; Laishram, Rakesh S.; Qiao, Mei; Lipshitz, Howard D.; Piano, Fabio; Corbett, Anita H.; Carstens, Russ P.; Frey, Brendan J.; Anderson, Richard A.; Lynch, Kristen W.; Penalva, Luiz O. F.; Lei, Elissa P.; Fraser, Andrew G.; Blencowe, Benjamin J.; Morris, Quaid D.; Hughes, Timothy R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">499</span>
        (<span class="NLM_cas:issue">7457</span>),
    <span class="NLM_cas:pages">172-177</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">RNA-binding proteins are key regulators of gene expression, yet only a small fraction have been functionally characterized.  Here we report a systematic anal. of the RNA motifs recognized by RNA-binding proteins, encompassing 205 distinct genes from 24 diverse eukaryotes.  The sequence specificities of RNA-binding proteins display deep evolutionary conservation, and the recognition preferences for a large fraction of metazoan RNA-binding proteins can thus be inferred from their RNA-binding domain sequence.  The motifs that we identify in vitro correlate well with in vivo RNA-binding data.  Moreover, we can assoc. them with distinct functional roles in diverse types of post-transcriptional regulation, enabling new insights into the functions of RNA-binding proteins both in normal physiol. and in human disease.  These data provide an unprecedented overview of RNA-binding proteins and their targets, and constitute an invaluable resource for detg. post-transcriptional regulatory mechanisms in eukaryotes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-R4r7lsTSl7Vg90H21EOLACvtfcHk0ljXf9SvEFbD4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVyksbjF&md5=4729aa8faf50eae0e2e91d31112d6311</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnature12311&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12311%26sid%3Dliteratum%253Aachs%26aulast%3DRay%26aufirst%3DD.%26aulast%3DKazan%26aufirst%3DH.%26aulast%3DCook%26aufirst%3DK.%2BB.%26aulast%3DWeirauch%26aufirst%3DM.%2BT.%26aulast%3DNajafabadi%26aufirst%3DH.%2BS.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DGueroussov%26aufirst%3DS.%26aulast%3DAlbu%26aufirst%3DM.%26aulast%3DZheng%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DA.%26aulast%3DNa%26aufirst%3DH.%26aulast%3DIrimia%26aufirst%3DM.%26aulast%3DMatzat%26aufirst%3DL.%2BH.%26aulast%3DDale%26aufirst%3DR.%2BK.%26aulast%3DSmith%26aufirst%3DS.%2BA.%26aulast%3DYarosh%26aufirst%3DC.%2BA.%26aulast%3DKelly%26aufirst%3DS.%2BM.%26aulast%3DNabet%26aufirst%3DB.%26aulast%3DMecenas%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLaishram%26aufirst%3DR.%2BS.%26aulast%3DQiao%26aufirst%3DM.%26aulast%3DLipshitz%26aufirst%3DH.%2BD.%26aulast%3DPiano%26aufirst%3DF.%26aulast%3DCorbett%26aufirst%3DA.%2BH.%26aulast%3DCarstens%26aufirst%3DR.%2BP.%26aulast%3DFrey%26aufirst%3DB.%2BJ.%26aulast%3DAnderson%26aufirst%3DR.%2BA.%26aulast%3DLynch%26aufirst%3DK.%2BW.%26aulast%3DPenalva%26aufirst%3DL.%2BO.%26aulast%3DLei%26aufirst%3DE.%2BP.%26aulast%3DFraser%26aufirst%3DA.%2BG.%26aulast%3DBlencowe%26aufirst%3DB.%2BJ.%26aulast%3DMorris%26aufirst%3DQ.%2BD.%26aulast%3DHughes%26aufirst%3DT.%2BR.%26atitle%3DA%2520compendium%2520of%2520RNA-binding%2520motifs%2520for%2520decoding%2520gene%2520regulation%26jtitle%3DNature%26date%3D2013%26volume%3D499%26spage%3D172%26epage%3D177%26doi%3D10.1038%2Fnature12311" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Hennig, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gebauer, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sattler, M.</span><span> </span><span class="NLM_article-title">Breaking the protein-RNA recognition code</span> <span class="citation_source-journal">Cell Cycle</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">3619</span><span class="NLM_x">–</span> <span class="NLM_lpage">3620</span><span class="refDoi"> DOI: 10.4161/15384101.2014.986625</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.4161%2F15384101.2014.986625" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=3619-3620&author=J.+Hennigauthor=F.+Gebauerauthor=M.+Sattler&title=Breaking+the+protein-RNA+recognition+code&doi=10.4161%2F15384101.2014.986625"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.4161%2F15384101.2014.986625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252F15384101.2014.986625%26sid%3Dliteratum%253Aachs%26aulast%3DHennig%26aufirst%3DJ.%26aulast%3DGebauer%26aufirst%3DF.%26aulast%3DSattler%26aufirst%3DM.%26atitle%3DBreaking%2520the%2520protein-RNA%2520recognition%2520code%26jtitle%3DCell%2520Cycle%26date%3D2014%26volume%3D13%26spage%3D3619%26epage%3D3620%26doi%3D10.4161%2F15384101.2014.986625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Cooper, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dreyfuss, G.</span><span> </span><span class="NLM_article-title">RNA and disease</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">136</span><span class="NLM_x">, </span> <span class="NLM_fpage">777</span><span class="NLM_x">–</span> <span class="NLM_lpage">793</span><span class="refDoi"> DOI: 10.1016/j.cell.2009.02.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1016%2Fj.cell.2009.02.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=19239895" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADC%252BD1MXkvFGktrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2009&pages=777-793&author=T.+A.+Cooperauthor=L.+Wanauthor=G.+Dreyfuss&title=RNA+and+disease&doi=10.1016%2Fj.cell.2009.02.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">RNA and disease</span></div><div class="casAuthors">Cooper, Thomas A.; Wan, Lili; Dreyfuss, Gideon</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">777-793</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Cellular functions depend on numerous protein-coding and noncoding RNAs and the RNA-binding proteins assocd. with them, which form ribonucleoprotein complexes (RNPs).  Mutations that disrupt either the RNA or protein components of RNPs or the factors required for their assembly can be deleterious.  Alternative splicing provides cells with an exquisite capacity to fine-tune their transcriptome and proteome in response to cues.  Splicing depends on a complex code, numerous RNA-binding proteins, and an enormously intricate network of interactions among them, increasing the opportunity for exposure to mutations and misregulation that cause disease.  The discovery of disease-causing mutations in RNAs is yielding a wealth of new therapeutic targets, and the growing understanding of RNA biol. and chem. is providing new RNA-based tools for developing therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotOMwPdFvVpLVg90H21EOLACvtfcHk0ljXf9SvEFbD4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXkvFGktrw%253D&md5=f165df2b33f97df6e895045b81e853d4</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2009.02.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2009.02.011%26sid%3Dliteratum%253Aachs%26aulast%3DCooper%26aufirst%3DT.%2BA.%26aulast%3DWan%26aufirst%3DL.%26aulast%3DDreyfuss%26aufirst%3DG.%26atitle%3DRNA%2520and%2520disease%26jtitle%3DCell%26date%3D2009%26volume%3D136%26spage%3D777%26epage%3D793%26doi%3D10.1016%2Fj.cell.2009.02.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">David, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manley, J. L.</span><span> </span><span class="NLM_article-title">Alternative pre-mRNA splicing regulation in cancer: pathways and programs unhinged</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">2343</span><span class="NLM_x">–</span> <span class="NLM_lpage">2364</span><span class="refDoi"> DOI: 10.1101/gad.1973010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1101%2Fgad.1973010" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=2343-2364&author=C.+J.+Davidauthor=J.+L.+Manley&title=Alternative+pre-mRNA+splicing+regulation+in+cancer%3A+pathways+and+programs+unhinged&doi=10.1101%2Fgad.1973010"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1101%2Fgad.1973010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1973010%26sid%3Dliteratum%253Aachs%26aulast%3DDavid%26aufirst%3DC.%2BJ.%26aulast%3DManley%26aufirst%3DJ.%2BL.%26atitle%3DAlternative%2520pre-mRNA%2520splicing%2520regulation%2520in%2520cancer%253A%2520pathways%2520and%2520programs%2520unhinged%26jtitle%3DGenes%2520Dev.%26date%3D2010%26volume%3D24%26spage%3D2343%26epage%3D2364%26doi%3D10.1101%2Fgad.1973010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Bonnal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vigevani, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valcarcel, J.</span><span> </span><span class="NLM_article-title">The spliceosome as a target of novel antitumour drugs</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">847</span><span class="NLM_x">–</span> <span class="NLM_lpage">859</span><span class="refDoi"> DOI: 10.1038/nrd3823</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1038%2Fnrd3823" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=23123942" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1ajsbrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=847-859&author=S.+Bonnalauthor=L.+Vigevaniauthor=J.+Valcarcel&title=The+spliceosome+as+a+target+of+novel+antitumour+drugs&doi=10.1038%2Fnrd3823"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The spliceosome as a target of novel antitumour drugs</span></div><div class="casAuthors">Bonnal, Sophie; Vigevani, Luisa; Valcarcel, Juan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">847-859</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Several bacterial fermn. products and their synthetic derivs. display antitumor activities and bind tightly to components of the spliceosome, which is the complex mol. machinery involved in the removal of introns from mRNA precursors in eukaryotic cells.  The drugs alter gene expression, including alternative splicing, of genes that are important for cancer progression.  A flurry of recent reports has revealed that genes encoding splicing factors, including the drug target splicing factor 3B subunit 1 (SF3B1), are among the most highly mutated in various haematol. malignancies such as chronic lymphocytic leukemia and myelodysplastic syndromes.  These observations highlight the role of splicing factors in cancer and suggest that an understanding of the mol. effects of drugs targeting these proteins could open new perspectives for studies of the spliceosome and its role in cancer progression, and for the development of novel antitumor therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLmM2p9jQOMLVg90H21EOLACvtfcHk0ljyYdOUvhf0Kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1ajsbrJ&md5=c26edf63e7ca3e1ea2ffd93325f9ab1d</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrd3823&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3823%26sid%3Dliteratum%253Aachs%26aulast%3DBonnal%26aufirst%3DS.%26aulast%3DVigevani%26aufirst%3DL.%26aulast%3DValcarcel%26aufirst%3DJ.%26atitle%3DThe%2520spliceosome%2520as%2520a%2520target%2520of%2520novel%2520antitumour%2520drugs%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D847%26epage%3D859%26doi%3D10.1038%2Fnrd3823" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Daguenet, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dujardin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valcarcel, J.</span><span> </span><span class="NLM_article-title">The pathogenicity of splicing defects: mechanistic insights into pre-mRNA processing inform novel therapeutic approaches</span> <span class="citation_source-journal">EMBO Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1640</span><span class="NLM_x">–</span> <span class="NLM_lpage">1655</span><span class="refDoi"> DOI: 10.15252/embr.201541116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.15252%2Fembr.201541116" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=1640-1655&author=E.+Daguenetauthor=G.+Dujardinauthor=J.+Valcarcel&title=The+pathogenicity+of+splicing+defects%3A+mechanistic+insights+into+pre-mRNA+processing+inform+novel+therapeutic+approaches&doi=10.15252%2Fembr.201541116"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.15252%2Fembr.201541116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.15252%252Fembr.201541116%26sid%3Dliteratum%253Aachs%26aulast%3DDaguenet%26aufirst%3DE.%26aulast%3DDujardin%26aufirst%3DG.%26aulast%3DValcarcel%26aufirst%3DJ.%26atitle%3DThe%2520pathogenicity%2520of%2520splicing%2520defects%253A%2520mechanistic%2520insights%2520into%2520pre-mRNA%2520processing%2520inform%2520novel%2520therapeutic%2520approaches%26jtitle%3DEMBO%2520Rep.%26date%3D2015%26volume%3D16%26spage%3D1640%26epage%3D1655%26doi%3D10.15252%2Fembr.201541116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Neubauer, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rappsilber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calvio, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ajuh, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sleeman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamond, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mann, M.</span><span> </span><span class="NLM_article-title">Mass spectrometry and EST-database searching allows characterization of the multi-protein spliceosome complex</span> <span class="citation_source-journal">Nat. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">46</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1038/1700</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1038%2F1700" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=9731529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADyaK1cXmtVOmtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=1998&pages=46-50&author=G.+Neubauerauthor=A.+Kingauthor=J.+Rappsilberauthor=C.+Calvioauthor=M.+Watsonauthor=P.+Ajuhauthor=J.+Sleemanauthor=A.+Lamondauthor=M.+Mann&title=Mass+spectrometry+and+EST-database+searching+allows+characterization+of+the+multi-protein+spliceosome+complex&doi=10.1038%2F1700"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Mass spectrometry and EST-database searching allows characterization of the multi-protein spliceosome complex</span></div><div class="casAuthors">Neubauer, Gitte; King, Angus; Rappsilber, Juri; Calvio, Cinzia; Watson, Mark; Ajuh, Paul; Sleeman, Judith; Lamond, Angus; Mann, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">46-50</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature America</span>)
        </div><div class="casAbstract">Many important cell mechanisms are carried out and regulated by multi-protein complexes, for example, transcription and RNA processing machinery, receptor complexes and cytoskeletal structures.  Most of these complexes remain only partially characterized due to the difficulty of conventional protein anal. methods.  The rapid expansion of DNA sequence databases now provides whole or partial gene sequences of model organisms, and recent advances in protein microcharacterization via mass spectrometry allow the possibility of linking these DNA sequences to the proteins in functional complexes.  This approach has been demonstrated in organisms whose genomes have been sequenced, such as budding yeast.  Here we report the first characterization of an entire mammalian multi-protein complex using these methods.  The machinery that removes introns from mRNA precursors-the spliceosome-is a large multi-protein complex.  Approx. half of the components excised from a two-dimensional gel sepn. of the spliceosome were found in protein sequence databases.  Using nanoelectrospray mass spectrometry, the remainder were identified and cloned using public expressed sequence tag (EST) databases.  Existing EST databases are thus already sufficiently complete to allow rapid characterization of large mammalian protein complexes via mass spectrometry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrm9v6Hl41AxrVg90H21EOLACvtfcHk0likrWvDPbVwhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmtVOmtLw%253D&md5=d67e8a9aea0e391d5495c7113626c501</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2F1700&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F1700%26sid%3Dliteratum%253Aachs%26aulast%3DNeubauer%26aufirst%3DG.%26aulast%3DKing%26aufirst%3DA.%26aulast%3DRappsilber%26aufirst%3DJ.%26aulast%3DCalvio%26aufirst%3DC.%26aulast%3DWatson%26aufirst%3DM.%26aulast%3DAjuh%26aufirst%3DP.%26aulast%3DSleeman%26aufirst%3DJ.%26aulast%3DLamond%26aufirst%3DA.%26aulast%3DMann%26aufirst%3DM.%26atitle%3DMass%2520spectrometry%2520and%2520EST-database%2520searching%2520allows%2520characterization%2520of%2520the%2520multi-protein%2520spliceosome%2520complex%26jtitle%3DNat.%2520Genet.%26date%3D1998%26volume%3D20%26spage%3D46%26epage%3D50%26doi%3D10.1038%2F1700" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Lallena, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chalmers, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Llamazares, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamond, A. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valcarcel, J.</span><span> </span><span class="NLM_article-title">Splicing regulation at the second catalytic step by Sex-lethal involves 3′ splice site recognition by SPF45</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">285</span><span class="NLM_x">–</span> <span class="NLM_lpage">296</span><span class="refDoi"> DOI: 10.1016/S0092-8674(02)00730-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1016%2FS0092-8674%2802%2900730-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2002&pages=285-296&author=M.+J.+Lallenaauthor=K.+J.+Chalmersauthor=S.+Llamazaresauthor=A.+I.+Lamondauthor=J.+Valcarcel&title=Splicing+regulation+at+the+second+catalytic+step+by+Sex-lethal+involves+3%E2%80%B2+splice+site+recognition+by+SPF45&doi=10.1016%2FS0092-8674%2802%2900730-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2802%2900730-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252802%252900730-4%26sid%3Dliteratum%253Aachs%26aulast%3DLallena%26aufirst%3DM.%2BJ.%26aulast%3DChalmers%26aufirst%3DK.%2BJ.%26aulast%3DLlamazares%26aufirst%3DS.%26aulast%3DLamond%26aufirst%3DA.%2BI.%26aulast%3DValcarcel%26aufirst%3DJ.%26atitle%3DSplicing%2520regulation%2520at%2520the%2520second%2520catalytic%2520step%2520by%2520Sex-lethal%2520involves%25203%25E2%2580%25B2%2520splice%2520site%2520recognition%2520by%2520SPF45%26jtitle%3DCell%26date%3D2002%26volume%3D109%26spage%3D285%26epage%3D296%26doi%3D10.1016%2FS0092-8674%2802%2900730-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Corsini, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonna, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basquin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hothorn, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheffzek, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valcarcel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sattler, M.</span><span> </span><span class="NLM_article-title">U2AF-homology motif interactions are required for alternative splicing regulation by SPF45</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">620</span><span class="NLM_x">–</span> <span class="NLM_lpage">629</span><span class="refDoi"> DOI: 10.1038/nsmb1260</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1038%2Fnsmb1260" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=17589525" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnt1Kgsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2007&pages=620-629&author=L.+Corsiniauthor=S.+Bonnaauthor=J.+Basquinauthor=M.+Hothornauthor=K.+Scheffzekauthor=J.+Valcarcelauthor=M.+Sattler&title=U2AF-homology+motif+interactions+are+required+for+alternative+splicing+regulation+by+SPF45&doi=10.1038%2Fnsmb1260"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">U2AF-homology motif interactions are required for alternative splicing regulation by SPF45</span></div><div class="casAuthors">Corsini, Lorenzo; Bonna, Sophie; Basquin, Jerome; Hothorn, Michael; Scheffzek, Klaus; Valcarcel, Juan; Sattler, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">620-629</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The U2AF-homol. motif (UHM) mediates protein-protein interactions between factors involved in constitutive RNA splicing.  Here we report that the splicing factor SPF45 regulates alternative splicing of the apoptosis regulatory gene FAS (also called CD95).  The SPF45 UHM is necessary for this activity and binds UHM-ligand motifs (ULMs) present in the 3' splice site-recognizing factors U2AF65, SF1 and SF3b155.  We describe a 2.1-Å crystal structure of SPF45-UHM in complex with a ULM peptide from SF3b155.  Features distinct from those of previously described UHM-ULM structures allowed the design of mutations in the SPF45 UHM that selectively impair binding to individual ULMs.  Splicing assays using the ULM-selective SPF45 variants demonstrate that individual UHM-ULM interactions are required for FAS splicing regulation by SPF45 in vivo.  Our data suggest that networks of UHM-ULM interactions are involved in regulating alternative splicing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9hirtJhwE6bVg90H21EOLACvtfcHk0likrWvDPbVwhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnt1Kgsrk%253D&md5=f82647f494645f7e9bbb549491a2867a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1260&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1260%26sid%3Dliteratum%253Aachs%26aulast%3DCorsini%26aufirst%3DL.%26aulast%3DBonna%26aufirst%3DS.%26aulast%3DBasquin%26aufirst%3DJ.%26aulast%3DHothorn%26aufirst%3DM.%26aulast%3DScheffzek%26aufirst%3DK.%26aulast%3DValcarcel%26aufirst%3DJ.%26aulast%3DSattler%26aufirst%3DM.%26atitle%3DU2AF-homology%2520motif%2520interactions%2520are%2520required%2520for%2520alternative%2520splicing%2520regulation%2520by%2520SPF45%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2007%26volume%3D14%26spage%3D620%26epage%3D629%26doi%3D10.1038%2Fnsmb1260" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Izquierdo, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majos, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castelo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guigo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bilbao, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valcarcel, J.</span><span> </span><span class="NLM_article-title">Regulation of Fas alternative splicing by antagonistic effects of TIA-1 and PTB on exon definition</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">475</span><span class="NLM_x">–</span> <span class="NLM_lpage">484</span><span class="refDoi"> DOI: 10.1016/j.molcel.2005.06.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1016%2Fj.molcel.2005.06.015" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2005&pages=475-484&author=J.+M.+Izquierdoauthor=N.+Majosauthor=S.+Bonnalauthor=C.+Martinezauthor=R.+Casteloauthor=R.+Guigoauthor=D.+Bilbaoauthor=J.+Valcarcel&title=Regulation+of+Fas+alternative+splicing+by+antagonistic+effects+of+TIA-1+and+PTB+on+exon+definition&doi=10.1016%2Fj.molcel.2005.06.015"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2005.06.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2005.06.015%26sid%3Dliteratum%253Aachs%26aulast%3DIzquierdo%26aufirst%3DJ.%2BM.%26aulast%3DMajos%26aufirst%3DN.%26aulast%3DBonnal%26aufirst%3DS.%26aulast%3DMartinez%26aufirst%3DC.%26aulast%3DCastelo%26aufirst%3DR.%26aulast%3DGuigo%26aufirst%3DR.%26aulast%3DBilbao%26aufirst%3DD.%26aulast%3DValcarcel%26aufirst%3DJ.%26atitle%3DRegulation%2520of%2520Fas%2520alternative%2520splicing%2520by%2520antagonistic%2520effects%2520of%2520TIA-1%2520and%2520PTB%2520on%2520exon%2520definition%26jtitle%3DMol.%2520Cell%26date%3D2005%26volume%3D19%26spage%3D475%26epage%3D484%26doi%3D10.1016%2Fj.molcel.2005.06.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Sampath, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Long, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shepard, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xia, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Devanarayan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandusky, G. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, W. L.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dantzig, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williamson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rolfe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, R. E.</span><span> </span><span class="NLM_article-title">Human SPF45, a splicing factor, has limited expression in normal tissues, is overexpressed in many tumors, and can confer a multidrug-resistant phenotype to cells</span> <span class="citation_source-journal">Am. J. Pathol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">163</span><span class="NLM_x">, </span> <span class="NLM_fpage">1781</span><span class="NLM_x">–</span> <span class="NLM_lpage">1790</span><span class="refDoi"> DOI: 10.1016/S0002-9440(10)63538-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1016%2FS0002-9440%2810%2963538-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=14578179" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpsVWltL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=163&publication_year=2003&pages=1781-1790&author=J.+Sampathauthor=P.+R.+Longauthor=R.+L.+Shepardauthor=X.+Xiaauthor=V.+Devanarayanauthor=G.+E.+Sanduskyauthor=W.+L.+Perryauthor=A.+H.+Dantzigauthor=M.+Williamsonauthor=M.+Rolfeauthor=R.+E.+Moore&title=Human+SPF45%2C+a+splicing+factor%2C+has+limited+expression+in+normal+tissues%2C+is+overexpressed+in+many+tumors%2C+and+can+confer+a+multidrug-resistant+phenotype+to+cells&doi=10.1016%2FS0002-9440%2810%2963538-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Human SPF45, a splicing factor, has limited expression in normal tissues, is overexpressed in many tumors, and can confer a multidrug-resistant phenotype to cells</span></div><div class="casAuthors">Sampath, Janardhan; Long, Pandy R.; Shepard, Robert L.; Xia, Xiaoling; Devanarayan, Viswanath; Sandusky, George E.; Perry, William L., III; Dantzig, Anne H.; Williamson, Mark; Rolfe, Mark; Moore, Robert E.</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Pathology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">163</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1781-1790</span>CODEN:
                <span class="NLM_cas:coden">AJPAA4</span>;
        ISSN:<span class="NLM_cas:issn">0002-9440</span>.
    
            (<span class="NLM_cas:orgname">American Society for Investigative Pathology</span>)
        </div><div class="casAbstract">The authors' effort to identify novel drug-resistant genes in cyclophosphamide-resistant EMT6 mouse mammary tumors led the authors to the identification of SPF45.  Simultaneously, other groups identified SPF45 as a component of the spliceosome that is involved in alternative splicing.  The authors isolated the human homolog and examd. the normal human tissue expression, tumor expression, and the phenotype caused by overexpression of human SPF45.  The authors' analyses revealed that SPF45 is expressed in many, but not all, normal tissues tested with predominant expression in normal ductal epithelial cells of the breast, liver, pancreas, and prostate.  The authors' analyses using tissue microarrays and sausages of tumors indicated that SPF45 is highly expressed in numerous carcinomas including bladder, breast, colon, lung, ovarian, pancreatic, and prostate.  Interestingly, this study revealed that overexpression of SPF45 in HeLa, a cervical carcinoma cell line, resulted in drug resistance to doxorubicin and vincristine, two chemotherapeutic drugs commonly used in cancer.  To the authors' knowledge, this is the first study showing tumor overexpression of an alternate splicing factor resulting in drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQdUUfuDDLB7Vg90H21EOLACvtfcHk0liBJwDUD1WHPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpsVWltL4%253D&md5=5e82f79953ec02c8c31da98afed038eb</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2FS0002-9440%2810%2963538-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0002-9440%252810%252963538-9%26sid%3Dliteratum%253Aachs%26aulast%3DSampath%26aufirst%3DJ.%26aulast%3DLong%26aufirst%3DP.%2BR.%26aulast%3DShepard%26aufirst%3DR.%2BL.%26aulast%3DXia%26aufirst%3DX.%26aulast%3DDevanarayan%26aufirst%3DV.%26aulast%3DSandusky%26aufirst%3DG.%2BE.%26aulast%3DPerry%26aufirst%3DW.%2BL.%26aulast%3DDantzig%26aufirst%3DA.%2BH.%26aulast%3DWilliamson%26aufirst%3DM.%26aulast%3DRolfe%26aufirst%3DM.%26aulast%3DMoore%26aufirst%3DR.%2BE.%26atitle%3DHuman%2520SPF45%252C%2520a%2520splicing%2520factor%252C%2520has%2520limited%2520expression%2520in%2520normal%2520tissues%252C%2520is%2520overexpressed%2520in%2520many%2520tumors%252C%2520and%2520can%2520confer%2520a%2520multidrug-resistant%2520phenotype%2520to%2520cells%26jtitle%3DAm.%2520J.%2520Pathol.%26date%3D2003%26volume%3D163%26spage%3D1781%26epage%3D1790%26doi%3D10.1016%2FS0002-9440%2810%2963538-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Perry, W. L.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shepard, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sampath, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yaden, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chin, W. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iversen, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lesoon, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Brien, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peek, V. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rolfe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shyjan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tighe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williamson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krishnan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dantzig, A. H.</span><span> </span><span class="NLM_article-title">Human splicing factor SPF45 (RBM17) confers broad multidrug resistance to anticancer drugs when overexpressed--a phenotype partially reversed by selective estrogen receptor modulators</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">6593</span><span class="NLM_x">–</span> <span class="NLM_lpage">6600</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-03-3675</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1158%2F0008-5472.CAN-03-3675" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2005&pages=6593-6600&author=W.+L.+Perryauthor=R.+L.+Shepardauthor=J.+Sampathauthor=B.+Yadenauthor=W.+W.+Chinauthor=P.+W.+Iversenauthor=S.+Jinauthor=A.+Lesoonauthor=K.+A.+O%E2%80%99Brienauthor=V.+L.+Peekauthor=M.+Rolfeauthor=A.+Shyjanauthor=M.+Tigheauthor=M.+Williamsonauthor=V.+Krishnanauthor=R.+E.+Mooreauthor=A.+H.+Dantzig&title=Human+splicing+factor+SPF45+%28RBM17%29+confers+broad+multidrug+resistance+to+anticancer+drugs+when+overexpressed%2D%2Da+phenotype+partially+reversed+by+selective+estrogen+receptor+modulators&doi=10.1158%2F0008-5472.CAN-03-3675"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-03-3675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-03-3675%26sid%3Dliteratum%253Aachs%26aulast%3DPerry%26aufirst%3DW.%2BL.%26aulast%3DShepard%26aufirst%3DR.%2BL.%26aulast%3DSampath%26aufirst%3DJ.%26aulast%3DYaden%26aufirst%3DB.%26aulast%3DChin%26aufirst%3DW.%2BW.%26aulast%3DIversen%26aufirst%3DP.%2BW.%26aulast%3DJin%26aufirst%3DS.%26aulast%3DLesoon%26aufirst%3DA.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DK.%2BA.%26aulast%3DPeek%26aufirst%3DV.%2BL.%26aulast%3DRolfe%26aufirst%3DM.%26aulast%3DShyjan%26aufirst%3DA.%26aulast%3DTighe%26aufirst%3DM.%26aulast%3DWilliamson%26aufirst%3DM.%26aulast%3DKrishnan%26aufirst%3DV.%26aulast%3DMoore%26aufirst%3DR.%2BE.%26aulast%3DDantzig%26aufirst%3DA.%2BH.%26atitle%3DHuman%2520splicing%2520factor%2520SPF45%2520%2528RBM17%2529%2520confers%2520broad%2520multidrug%2520resistance%2520to%2520anticancer%2520drugs%2520when%2520overexpressed--a%2520phenotype%2520partially%2520reversed%2520by%2520selective%2520estrogen%2520receptor%2520modulators%26jtitle%3DCancer%2520Res.%26date%3D2005%26volume%3D65%26spage%3D6593%26epage%3D6600%26doi%3D10.1158%2F0008-5472.CAN-03-3675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Kielkopf, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodionova, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burley, S. K.</span><span> </span><span class="NLM_article-title">A novel peptide recognition mode revealed by the X-ray structure of a core U2AF35/U2AF65 heterodimer</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">595</span><span class="NLM_x">–</span> <span class="NLM_lpage">605</span><span class="refDoi"> DOI: 10.1016/S0092-8674(01)00480-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1016%2FS0092-8674%2801%2900480-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2001&pages=595-605&author=C.+L.+Kielkopfauthor=N.+A.+Rodionovaauthor=M.+R.+Greenauthor=S.+K.+Burley&title=A+novel+peptide+recognition+mode+revealed+by+the+X-ray+structure+of+a+core+U2AF35%2FU2AF65+heterodimer&doi=10.1016%2FS0092-8674%2801%2900480-9"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2801%2900480-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252801%252900480-9%26sid%3Dliteratum%253Aachs%26aulast%3DKielkopf%26aufirst%3DC.%2BL.%26aulast%3DRodionova%26aufirst%3DN.%2BA.%26aulast%3DGreen%26aufirst%3DM.%2BR.%26aulast%3DBurley%26aufirst%3DS.%2BK.%26atitle%3DA%2520novel%2520peptide%2520recognition%2520mode%2520revealed%2520by%2520the%2520X-ray%2520structure%2520of%2520a%2520core%2520U2AF35%252FU2AF65%2520heterodimer%26jtitle%3DCell%26date%3D2001%26volume%3D106%26spage%3D595%26epage%3D605%26doi%3D10.1016%2FS0092-8674%2801%2900480-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Selenko, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gregorovic, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sprangers, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stier, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rhani, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kramer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sattler, M.</span><span> </span><span class="NLM_article-title">Structural basis for the molecular recognition between human splicing factors U2AF65 and SF1/mBBP</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">965</span><span class="NLM_x">–</span> <span class="NLM_lpage">976</span><span class="refDoi"> DOI: 10.1016/S1097-2765(03)00115-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1016%2FS1097-2765%2803%2900115-1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2003&pages=965-976&author=P.+Selenkoauthor=G.+Gregorovicauthor=R.+Sprangersauthor=G.+Stierauthor=Z.+Rhaniauthor=A.+Kramerauthor=M.+Sattler&title=Structural+basis+for+the+molecular+recognition+between+human+splicing+factors+U2AF65+and+SF1%2FmBBP&doi=10.1016%2FS1097-2765%2803%2900115-1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2803%2900115-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252803%252900115-1%26sid%3Dliteratum%253Aachs%26aulast%3DSelenko%26aufirst%3DP.%26aulast%3DGregorovic%26aufirst%3DG.%26aulast%3DSprangers%26aufirst%3DR.%26aulast%3DStier%26aufirst%3DG.%26aulast%3DRhani%26aufirst%3DZ.%26aulast%3DKramer%26aufirst%3DA.%26aulast%3DSattler%26aufirst%3DM.%26atitle%3DStructural%2520basis%2520for%2520the%2520molecular%2520recognition%2520between%2520human%2520splicing%2520factors%2520U2AF65%2520and%2520SF1%252FmBBP%26jtitle%3DMol.%2520Cell%26date%3D2003%26volume%3D11%26spage%3D965%26epage%3D976%26doi%3D10.1016%2FS1097-2765%2803%2900115-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Kielkopf, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lucke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, M. R.</span><span> </span><span class="NLM_article-title">U2AF homology motifs: protein recognition in the RRM world</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">1513</span><span class="NLM_x">–</span> <span class="NLM_lpage">1526</span><span class="refDoi"> DOI: 10.1101/gad.1206204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1101%2Fgad.1206204" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2004&pages=1513-1526&author=C.+L.+Kielkopfauthor=S.+Luckeauthor=M.+R.+Green&title=U2AF+homology+motifs%3A+protein+recognition+in+the+RRM+world&doi=10.1101%2Fgad.1206204"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1101%2Fgad.1206204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1206204%26sid%3Dliteratum%253Aachs%26aulast%3DKielkopf%26aufirst%3DC.%2BL.%26aulast%3DLucke%26aufirst%3DS.%26aulast%3DGreen%26aufirst%3DM.%2BR.%26atitle%3DU2AF%2520homology%2520motifs%253A%2520protein%2520recognition%2520in%2520the%2520RRM%2520world%26jtitle%3DGenes%2520Dev.%26date%3D2004%26volume%3D18%26spage%3D1513%26epage%3D1526%26doi%3D10.1101%2Fgad.1206204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Manceau, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kielkopf, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sobel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maucuer, A.</span><span> </span><span class="NLM_article-title">Different requirements of the kinase and UHM domains of KIS for its nuclear localization and binding to splicing factors</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">381</span><span class="NLM_x">, </span> <span class="NLM_fpage">748</span><span class="NLM_x">–</span> <span class="NLM_lpage">762</span><span class="refDoi"> DOI: 10.1016/j.jmb.2008.06.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1016%2Fj.jmb.2008.06.026" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=381&publication_year=2008&pages=748-762&author=V.+Manceauauthor=C.+L.+Kielkopfauthor=A.+Sobelauthor=A.+Maucuer&title=Different+requirements+of+the+kinase+and+UHM+domains+of+KIS+for+its+nuclear+localization+and+binding+to+splicing+factors&doi=10.1016%2Fj.jmb.2008.06.026"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2008.06.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2008.06.026%26sid%3Dliteratum%253Aachs%26aulast%3DManceau%26aufirst%3DV.%26aulast%3DKielkopf%26aufirst%3DC.%2BL.%26aulast%3DSobel%26aufirst%3DA.%26aulast%3DMaucuer%26aufirst%3DA.%26atitle%3DDifferent%2520requirements%2520of%2520the%2520kinase%2520and%2520UHM%2520domains%2520of%2520KIS%2520for%2520its%2520nuclear%2520localization%2520and%2520binding%2520to%2520splicing%2520factors%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2008%26volume%3D381%26spage%3D748%26epage%3D762%26doi%3D10.1016%2Fj.jmb.2008.06.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Maucuer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ozon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manceau, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gavet, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lawler, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curmi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sobel, A.</span><span> </span><span class="NLM_article-title">KIS is a protein kinase with an RNA recognition motif</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">272</span><span class="NLM_x">, </span> <span class="NLM_fpage">23151</span><span class="NLM_x">–</span> <span class="NLM_lpage">23156</span><span class="refDoi"> DOI: 10.1074/jbc.272.37.23151</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1074%2Fjbc.272.37.23151" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=272&publication_year=1997&pages=23151-23156&author=A.+Maucuerauthor=S.+Ozonauthor=V.+Manceauauthor=O.+Gavetauthor=S.+Lawlerauthor=P.+Curmiauthor=A.+Sobel&title=KIS+is+a+protein+kinase+with+an+RNA+recognition+motif&doi=10.1074%2Fjbc.272.37.23151"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1074%2Fjbc.272.37.23151&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.272.37.23151%26sid%3Dliteratum%253Aachs%26aulast%3DMaucuer%26aufirst%3DA.%26aulast%3DOzon%26aufirst%3DS.%26aulast%3DManceau%26aufirst%3DV.%26aulast%3DGavet%26aufirst%3DO.%26aulast%3DLawler%26aufirst%3DS.%26aulast%3DCurmi%26aufirst%3DP.%26aulast%3DSobel%26aufirst%3DA.%26atitle%3DKIS%2520is%2520a%2520protein%2520kinase%2520with%2520an%2520RNA%2520recognition%2520motif%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1997%26volume%3D272%26spage%3D23151%26epage%3D23156%26doi%3D10.1074%2Fjbc.272.37.23151" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Corsini, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hothorn, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stier, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rybin, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheffzek, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibson, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sattler, M.</span><span> </span><span class="NLM_article-title">Dimerization and protein binding specificity of the U2AF homology motif of the splicing factor Puf60</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">630</span><span class="NLM_x">–</span> <span class="NLM_lpage">639</span><span class="refDoi"> DOI: 10.1074/jbc.M805395200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1074%2Fjbc.M805395200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=630-639&author=L.+Corsiniauthor=M.+Hothornauthor=G.+Stierauthor=V.+Rybinauthor=K.+Scheffzekauthor=T.+J.+Gibsonauthor=M.+Sattler&title=Dimerization+and+protein+binding+specificity+of+the+U2AF+homology+motif+of+the+splicing+factor+Puf60&doi=10.1074%2Fjbc.M805395200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M805395200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M805395200%26sid%3Dliteratum%253Aachs%26aulast%3DCorsini%26aufirst%3DL.%26aulast%3DHothorn%26aufirst%3DM.%26aulast%3DStier%26aufirst%3DG.%26aulast%3DRybin%26aufirst%3DV.%26aulast%3DScheffzek%26aufirst%3DK.%26aulast%3DGibson%26aufirst%3DT.%2BJ.%26aulast%3DSattler%26aufirst%3DM.%26atitle%3DDimerization%2520and%2520protein%2520binding%2520specificity%2520of%2520the%2520U2AF%2520homology%2520motif%2520of%2520the%2520splicing%2520factor%2520Puf60%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D630%26epage%3D639%26doi%3D10.1074%2Fjbc.M805395200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Loerch, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maucuer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manceau, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kielkopf, C. L.</span><span> </span><span class="NLM_article-title">Cancer-relevant splicing factor CAPERalpha engages the essential splicing factor SF3b155 in a specific ternary complex</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">289</span><span class="NLM_x">, </span> <span class="NLM_fpage">17325</span><span class="NLM_x">–</span> <span class="NLM_lpage">17337</span><span class="refDoi"> DOI: 10.1074/jbc.M114.558825</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1074%2Fjbc.M114.558825" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=17325-17337&author=S.+Loerchauthor=A.+Maucuerauthor=V.+Manceauauthor=M.+R.+Greenauthor=C.+L.+Kielkopf&title=Cancer-relevant+splicing+factor+CAPERalpha+engages+the+essential+splicing+factor+SF3b155+in+a+specific+ternary+complex&doi=10.1074%2Fjbc.M114.558825"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.558825&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.558825%26sid%3Dliteratum%253Aachs%26aulast%3DLoerch%26aufirst%3DS.%26aulast%3DMaucuer%26aufirst%3DA.%26aulast%3DManceau%26aufirst%3DV.%26aulast%3DGreen%26aufirst%3DM.%2BR.%26aulast%3DKielkopf%26aufirst%3DC.%2BL.%26atitle%3DCancer-relevant%2520splicing%2520factor%2520CAPERalpha%2520engages%2520the%2520essential%2520splicing%2520factor%2520SF3b155%2520in%2520a%2520specific%2520ternary%2520complex%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D17325%26epage%3D17337%26doi%3D10.1074%2Fjbc.M114.558825" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Spadaccini, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reidt, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dybkov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Will, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frank, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stier, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corsini, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wahl, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luhrmann, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sattler, M.</span><span> </span><span class="NLM_article-title">Biochemical and NMR analyses of an SF3b155-p14-U2AF-RNA interaction network involved in branch point definition during pre-mRNA splicing</span> <span class="citation_source-journal">RNA</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">410</span><span class="NLM_x">–</span> <span class="NLM_lpage">425</span><span class="refDoi"> DOI: 10.1261/rna.2271406</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1261%2Frna.2271406" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2006&pages=410-425&author=R.+Spadacciniauthor=U.+Reidtauthor=O.+Dybkovauthor=C.+Willauthor=R.+Frankauthor=G.+Stierauthor=L.+Corsiniauthor=M.+C.+Wahlauthor=R.+Luhrmannauthor=M.+Sattler&title=Biochemical+and+NMR+analyses+of+an+SF3b155-p14-U2AF-RNA+interaction+network+involved+in+branch+point+definition+during+pre-mRNA+splicing&doi=10.1261%2Frna.2271406"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1261%2Frna.2271406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1261%252Frna.2271406%26sid%3Dliteratum%253Aachs%26aulast%3DSpadaccini%26aufirst%3DR.%26aulast%3DReidt%26aufirst%3DU.%26aulast%3DDybkov%26aufirst%3DO.%26aulast%3DWill%26aufirst%3DC.%26aulast%3DFrank%26aufirst%3DR.%26aulast%3DStier%26aufirst%3DG.%26aulast%3DCorsini%26aufirst%3DL.%26aulast%3DWahl%26aufirst%3DM.%2BC.%26aulast%3DLuhrmann%26aufirst%3DR.%26aulast%3DSattler%26aufirst%3DM.%26atitle%3DBiochemical%2520and%2520NMR%2520analyses%2520of%2520an%2520SF3b155-p14-U2AF-RNA%2520interaction%2520network%2520involved%2520in%2520branch%2520point%2520definition%2520during%2520pre-mRNA%2520splicing%26jtitle%3DRNA%26date%3D2006%26volume%3D12%26spage%3D410%26epage%3D425%26doi%3D10.1261%2Frna.2271406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Thickman, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swenson, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kabogo, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gryczynski, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kielkopf, C. L.</span><span> </span><span class="NLM_article-title">Multiple U2AF65 binding sites within SF3b155: thermodynamic and spectroscopic characterization of protein-protein interactions among pre-mRNA splicing factors</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">356</span><span class="NLM_x">, </span> <span class="NLM_fpage">664</span><span class="NLM_x">–</span> <span class="NLM_lpage">683</span><span class="refDoi"> DOI: 10.1016/j.jmb.2005.11.067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1016%2Fj.jmb.2005.11.067" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=356&publication_year=2006&pages=664-683&author=K.+R.+Thickmanauthor=M.+C.+Swensonauthor=J.+M.+Kabogoauthor=Z.+Gryczynskiauthor=C.+L.+Kielkopf&title=Multiple+U2AF65+binding+sites+within+SF3b155%3A+thermodynamic+and+spectroscopic+characterization+of+protein-protein+interactions+among+pre-mRNA+splicing+factors&doi=10.1016%2Fj.jmb.2005.11.067"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2005.11.067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2005.11.067%26sid%3Dliteratum%253Aachs%26aulast%3DThickman%26aufirst%3DK.%2BR.%26aulast%3DSwenson%26aufirst%3DM.%2BC.%26aulast%3DKabogo%26aufirst%3DJ.%2BM.%26aulast%3DGryczynski%26aufirst%3DZ.%26aulast%3DKielkopf%26aufirst%3DC.%2BL.%26atitle%3DMultiple%2520U2AF65%2520binding%2520sites%2520within%2520SF3b155%253A%2520thermodynamic%2520and%2520spectroscopic%2520characterization%2520of%2520protein-protein%2520interactions%2520among%2520pre-mRNA%2520splicing%2520factors%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D356%26spage%3D664%26epage%3D683%26doi%3D10.1016%2Fj.jmb.2005.11.067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Chandler, D.</span><span> </span><span class="NLM_article-title">Interfaces and the driving force of hydrophobic assembly</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">437</span><span class="NLM_x">, </span> <span class="NLM_fpage">640</span><span class="NLM_x">–</span> <span class="NLM_lpage">647</span><span class="refDoi"> DOI: 10.1038/nature04162</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1038%2Fnature04162" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=16193038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVCjsLzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=640-647&author=D.+Chandler&title=Interfaces+and+the+driving+force+of+hydrophobic+assembly&doi=10.1038%2Fnature04162"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Interfaces and the driving force of hydrophobic assembly</span></div><div class="casAuthors">Chandler, David</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">437</span>
        (<span class="NLM_cas:issue">7059</span>),
    <span class="NLM_cas:pages">640-647</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The hydrophobic effect - the tendency for oil and water to segregate - is important in diverse phenomena, from the cleaning of laundry, to the creation of micro-emulsions to make new materials, to the assembly of proteins into functional complexes.  This effect is multifaceted depending on whether hydrophobic mols. are individually hydrated or driven to assemble into larger structures.  Despite the basic principles underlying the hydrophobic effect being qual. well understood, only recently have theor. developments begun to explain and quantify many features of this ubiquitous phenomenon.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVPZjVicyZaLVg90H21EOLACvtfcHk0lgm8Wa0_SLkPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVCjsLzM&md5=58e58a6f3dfaf11a9a158308bdcacbae</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnature04162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04162%26sid%3Dliteratum%253Aachs%26aulast%3DChandler%26aufirst%3DD.%26atitle%3DInterfaces%2520and%2520the%2520driving%2520force%2520of%2520hydrophobic%2520assembly%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D640%26epage%3D647%26doi%3D10.1038%2Fnature04162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Guth, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaur, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valcarcel, J.</span><span> </span><span class="NLM_article-title">Evidence for substrate-specific requirement of the splicing factor U2AF(35) and for its function after polypyrimidine tract recognition by U2AF(65)</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">8263</span><span class="NLM_x">–</span> <span class="NLM_lpage">8271</span><span class="refDoi"> DOI: 10.1128/MCB.19.12.8263</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1128%2FMCB.19.12.8263" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADyaK1MXns1ygtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1999&pages=8263-8271&author=S.+Guthauthor=C.+Martinezauthor=R.+K.+Gaurauthor=J.+Valcarcel&title=Evidence+for+substrate-specific+requirement+of+the+splicing+factor+U2AF%2835%29+and+for+its+function+after+polypyrimidine+tract+recognition+by+U2AF%2865%29&doi=10.1128%2FMCB.19.12.8263"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Evidence for substrate-specific requirement of the splicing factor U2AF35 and for its function after polypyrimidine tract recognition by U2AF65</span></div><div class="casAuthors">Guth, Sabine; Martinez, Concepcion; Gaur, Rajesh K.; Valcarcel, Juan</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">8263-8271</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">U2 snRNP auxiliary factor (U2AF) promotes U2 snRNP binding to pre-mRNAs and consists of two subunits of 65 and 35 kDa, U2AF65 and U2AF35.  U2AF65 binds to the polypyrimidine (Py) tract upstream from the 3' splice site and plays a key role in assisting U2 snRNP recruitment.  It has been proposed that U2AF35 facilitates U2AF65 binding through a network of protein-protein interactions with other splicing factors, but the requirement and function of U2AF35 remain controversial.  Here we show that recombinant U2AF65 is sufficient to activate the splicing of two constitutively spliced pre-mRNAs in exts. that were chromatog. depleted of U2AF.  In contrast, U2AF65, U2AF35, and the interaction between them are required for splicing of an Ig μ pre-RNA contg. an intron with a weak Py tract and a purine-rich exonic splicing enhancer.  Remarkably, splicing activation by U2AF35 occurs without changes in U2AF65 crosslinking to the Py tract.  These results reveal substrate-specific requirements for U2AF35 and a novel function for this factor in pre-mRNA splicing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpS5oTjnj5vaLVg90H21EOLACvtfcHk0lgm8Wa0_SLkPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXns1ygtL0%253D&md5=2d38e0308e8e6ad8757e950a09aa0211</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1128%2FMCB.19.12.8263&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.19.12.8263%26sid%3Dliteratum%253Aachs%26aulast%3DGuth%26aufirst%3DS.%26aulast%3DMartinez%26aufirst%3DC.%26aulast%3DGaur%26aufirst%3DR.%2BK.%26aulast%3DValcarcel%26aufirst%3DJ.%26atitle%3DEvidence%2520for%2520substrate-specific%2520requirement%2520of%2520the%2520splicing%2520factor%2520U2AF%252835%2529%2520and%2520for%2520its%2520function%2520after%2520polypyrimidine%2520tract%2520recognition%2520by%2520U2AF%252865%2529%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1999%26volume%3D19%26spage%3D8263%26epage%3D8271%26doi%3D10.1128%2FMCB.19.12.8263" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Gozani, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Potashkin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reed, R.</span><span> </span><span class="NLM_article-title">A potential role for U2AF-SAP 155 interactions in recruiting U2 snRNP to the branch site</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">4752</span><span class="NLM_x">–</span> <span class="NLM_lpage">4760</span><span class="refDoi"> DOI: 10.1128/MCB.18.8.4752</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1128%2FMCB.18.8.4752" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=9671485" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADyaK1cXkvVartLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1998&pages=4752-4760&author=O.+Gozaniauthor=J.+Potashkinauthor=R.+Reed&title=A+potential+role+for+U2AF-SAP+155+interactions+in+recruiting+U2+snRNP+to+the+branch+site&doi=10.1128%2FMCB.18.8.4752"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">A potential role for U2AF-SAP 155 interactions in recruiting U2 snRNP to the branch site</span></div><div class="casAuthors">Gozani, Or; Potashkin, Judith; Reed, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4752-4760</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Base pairing between U2 snRNA and the branchpoint sequence (BPS) is essential for pre-mRNA splicing.  Because the metazoan BPS is short and highly degenerate, this interaction alone is insufficient for specific binding of U2 snRNP.  The splicing factor U2AF binds to the pyrimidine tract at the 3' splice site in the earliest spliceosomal complex, E, and is essential for U2 snRNP binding in the spliceosomal complex A.  The authors show that the U2 snRNP protein SAP 155 UV cross-links to pre-mRNA on both sides of the BPS in the A complex.  SAP 155's downstream crosslinking site is immediately adjacent to the U2AF binding site, and the two proteins interact directly in protein-protein interaction assays.  Using UV crosslinking, together with functional analyses of pre-mRNAs contg. duplicated BPSs, the authors show a direct correlation between BPS selection and UV crosslinking of SAP 155 on both sides of the BPS.  Together, these data are consistent with a model in which U2AF binds to the pyrimidine tract in the E complex and then interacts with SAP 155 to recruit U2 snRNP to the BPS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWlODMFtuI5LVg90H21EOLACvtfcHk0lg6iKDHV-5Z3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXkvVartLo%253D&md5=8db87cfb4812b22757eba14138a7502e</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1128%2FMCB.18.8.4752&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.18.8.4752%26sid%3Dliteratum%253Aachs%26aulast%3DGozani%26aufirst%3DO.%26aulast%3DPotashkin%26aufirst%3DJ.%26aulast%3DReed%26aufirst%3DR.%26atitle%3DA%2520potential%2520role%2520for%2520U2AF-SAP%2520155%2520interactions%2520in%2520recruiting%2520U2%2520snRNP%2520to%2520the%2520branch%2520site%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1998%26volume%3D18%26spage%3D4752%26epage%3D4760%26doi%3D10.1128%2FMCB.18.8.4752" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Das, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reed, R.</span><span> </span><span class="NLM_article-title">Functional association of U2 snRNP with the ATP-independent spliceosomal complex E</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">779</span><span class="NLM_x">–</span> <span class="NLM_lpage">787</span><span class="refDoi"> DOI: 10.1016/S1097-2765(00)80318-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1016%2FS1097-2765%2800%2980318-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2000&pages=779-787&author=R.+Dasauthor=Z.+Zhouauthor=R.+Reed&title=Functional+association+of+U2+snRNP+with+the+ATP-independent+spliceosomal+complex+E&doi=10.1016%2FS1097-2765%2800%2980318-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2800%2980318-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252800%252980318-4%26sid%3Dliteratum%253Aachs%26aulast%3DDas%26aufirst%3DR.%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DReed%26aufirst%3DR.%26atitle%3DFunctional%2520association%2520of%2520U2%2520snRNP%2520with%2520the%2520ATP-independent%2520spliceosomal%2520complex%2520E%26jtitle%3DMol.%2520Cell%26date%3D2000%26volume%3D5%26spage%3D779%26epage%3D787%26doi%3D10.1016%2FS1097-2765%2800%2980318-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Banerjee, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rahn, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gawande, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guth, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valcarcel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, R.</span><span> </span><span class="NLM_article-title">The conserved RNA recognition motif 3 of U2 snRNA auxiliary factor (U2AF 65) is essential in vivo but dispensable for activity in vitro</span> <span class="citation_source-journal">RNA</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">253</span><span class="refDoi"> DOI: 10.1261/rna.5153204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1261%2Frna.5153204" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=240-253&author=H.+Banerjeeauthor=A.+Rahnauthor=B.+Gawandeauthor=S.+Guthauthor=J.+Valcarcelauthor=R.+Singh&title=The+conserved+RNA+recognition+motif+3+of+U2+snRNA+auxiliary+factor+%28U2AF+65%29+is+essential+in+vivo+but+dispensable+for+activity+in+vitro&doi=10.1261%2Frna.5153204"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1261%2Frna.5153204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1261%252Frna.5153204%26sid%3Dliteratum%253Aachs%26aulast%3DBanerjee%26aufirst%3DH.%26aulast%3DRahn%26aufirst%3DA.%26aulast%3DGawande%26aufirst%3DB.%26aulast%3DGuth%26aufirst%3DS.%26aulast%3DValcarcel%26aufirst%3DJ.%26aulast%3DSingh%26aufirst%3DR.%26atitle%3DThe%2520conserved%2520RNA%2520recognition%2520motif%25203%2520of%2520U2%2520snRNA%2520auxiliary%2520factor%2520%2528U2AF%252065%2529%2520is%2520essential%2520in%2520vivo%2520but%2520dispensable%2520for%2520activity%2520in%2520vitro%26jtitle%3DRNA%26date%3D2004%26volume%3D10%26spage%3D240%26epage%3D253%26doi%3D10.1261%2Frna.5153204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Zaharieva, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chipman, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soller, M.</span><span> </span><span class="NLM_article-title">Alternative splicing interference by xenobiotics</span> <span class="citation_source-journal">Toxicology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">296</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">12</span><span class="refDoi"> DOI: 10.1016/j.tox.2012.01.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1016%2Fj.tox.2012.01.014" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=296&publication_year=2012&pages=1-12&author=E.+Zaharievaauthor=J.+K.+Chipmanauthor=M.+Soller&title=Alternative+splicing+interference+by+xenobiotics&doi=10.1016%2Fj.tox.2012.01.014"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.tox.2012.01.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tox.2012.01.014%26sid%3Dliteratum%253Aachs%26aulast%3DZaharieva%26aufirst%3DE.%26aulast%3DChipman%26aufirst%3DJ.%2BK.%26aulast%3DSoller%26aufirst%3DM.%26atitle%3DAlternative%2520splicing%2520interference%2520by%2520xenobiotics%26jtitle%3DToxicology%26date%3D2012%26volume%3D296%26spage%3D1%26epage%3D12%26doi%3D10.1016%2Fj.tox.2012.01.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">Xds</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">132</span><span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=Xds&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0ljFzE43SwQNSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXds%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storoni, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title">Phaser crystallographic software</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">–</span> <span class="NLM_lpage">674</span><span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0ljFzE43SwQNSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">255</span><span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lgIDXJbg7vdRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballard, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keegan, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krissinel, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leslie, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNicholas, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pannu, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Potterton, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, K. S.</span><span> </span><span class="NLM_article-title">Overview of the CCP4 suite and current developments</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span><span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+suite+and+current+developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lgIDXJbg7vdRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Delaglio, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grzesiek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vuister, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pfeifer, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bax, A.</span><span> </span><span class="NLM_article-title">NMRPipe: a multidimensional spectral processing system based on UNIX pipes</span> <span class="citation_source-journal">J. Biomol. NMR</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">277</span><span class="NLM_x">–</span> <span class="NLM_lpage">293</span><span class="refDoi"> DOI: 10.1007/BF00197809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1007%2FBF00197809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=8520220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADyaK2MXhtVSmurfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1995&pages=277-293&author=F.+Delaglioauthor=S.+Grzesiekauthor=G.+W.+Vuisterauthor=G.+Zhuauthor=J.+Pfeiferauthor=A.+Bax&title=NMRPipe%3A+a+multidimensional+spectral+processing+system+based+on+UNIX+pipes&doi=10.1007%2FBF00197809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">NMRPipe: a multidimensional spectral processing system based on UNIX pipes</span></div><div class="casAuthors">Delaglio, Frank; Grzesiek, Stephan; Vuister, Geerten W.; Zhu, Guang; Pfeifer, John; Bax, Ad</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">277-93</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">ESCOM</span>)
        </div><div class="casAbstract">The NMRPipe system is a UNIX software environment of processing, graphics, and anal. tools designed to meet current routine and research-oriented multidimensional processing requirements, and to anticipate and accommodate future demands and developments.  The system is based on UNIX pipes, which allow programs running simultaneously to exchange streams of data under user control.  In an NMRPipe processing scheme, a stream of spectral data flows through a pipeline of processing programs, each of which performs one component of the overall scheme, such as Fourier transformation or linear prediction.  Complete multidimensional processing schemes are constructed as simple UNIX shell scripts.  The processing modules themselves maintain and exploit accurate records of data sizes, detection modes, and calibration information in all dimensions, so that schemes can be constructed without the need to explicitly define or anticipate data sizes or storage details of real and imaginary channels during processing.  The asynchronous pipeline scheme provides other substantial advantages, including high flexibility, favorable processing speeds, choice of both all-in-memory and disk-bound processing, easy adaptation to different data formats, simpler software development and maintenance, and the ability to distribute processing tasks on multi-CPU computers and computer networks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSIUdXrdcEgbVg90H21EOLACvtfcHk0lgIDXJbg7vdRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXhtVSmurfK&md5=a670fca5b164083e2178fafd2fb951ff</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1007%2FBF00197809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00197809%26sid%3Dliteratum%253Aachs%26aulast%3DDelaglio%26aufirst%3DF.%26aulast%3DGrzesiek%26aufirst%3DS.%26aulast%3DVuister%26aufirst%3DG.%2BW.%26aulast%3DZhu%26aufirst%3DG.%26aulast%3DPfeifer%26aufirst%3DJ.%26aulast%3DBax%26aufirst%3DA.%26atitle%3DNMRPipe%253A%2520a%2520multidimensional%2520spectral%2520processing%2520system%2520based%2520on%2520UNIX%2520pipes%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D1995%26volume%3D6%26spage%3D277%26epage%3D293%26doi%3D10.1007%2FBF00197809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Vranken, W. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boucher, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fogh, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pajon, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Llinas, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ulrich, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Markley, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ionides, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laue, E. D.</span><span> </span><span class="NLM_article-title">The CCPN data model for NMR spectroscopy: development of a software pipeline</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">687</span><span class="NLM_x">–</span> <span class="NLM_lpage">696</span><span class="refDoi"> DOI: 10.1002/prot.20449</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1002%2Fprot.20449" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=15815974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADC%252BD2MXksVKmt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2005&pages=687-696&author=W.+F.+Vrankenauthor=W.+Boucherauthor=T.+J.+Stevensauthor=R.+H.+Foghauthor=A.+Pajonauthor=M.+Llinasauthor=E.+L.+Ulrichauthor=J.+L.+Markleyauthor=J.+Ionidesauthor=E.+D.+Laue&title=The+CCPN+data+model+for+NMR+spectroscopy%3A+development+of+a+software+pipeline&doi=10.1002%2Fprot.20449"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The CCPN data model for NMR spectroscopy: development of a software pipeline</span></div><div class="casAuthors">Vranken, Wim F.; Boucher, Wayne; Stevens, Tim J.; Fogh, Rasmus H.; Pajon, Anne; Llinas, Miguel; Ulrich, Eldon L.; Markley, John L.; Ionides, John; Laue, Ernest D.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">687-696</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">To address data management and data exchange problems in the NMR community, the Collaborative Computing Project for the NMR community (CCPN) created a "Data Model" that describes all the different types of information needed in an NMR structural study, from mol. structure and NMR parameters to coordinates.  This paper describes the development of a set of software applications that use the Data Model and its assocd. libraries, thus validating the approach.  These applications are freely available and provide a pipeline for high-throughput anal. of NMR data.  Three programs work directly with the Data Model: CcpNmr Anal., an entirely new anal. and interactive display program, the CcpNmr FormatConverter, which allows transfer of data from programs commonly used in NMR to and from the Data Model, and the CLOUDS software for automated structure calcn. and assignment (Carnegie Mellon University), which was rewritten to interact directly with the Data Model.  The ARIA 2.0 software for structure calcn. (Institut Pasteur) and the QUEEN program for validation of restraints (University of Nijmegen) were extended to provide conversion of their data to the Data Model.  During these developments the Data Model has been thoroughly tested and used, demonstrating that applications can successfully exchange data via the Data Model.  The software architecture developed by CCPN is now ready for new developments, such as integration with addnl. software applications and extensions of the Data Model into other areas of research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpl6jPIIOYWbVg90H21EOLACvtfcHk0liJulRqjNyj_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXksVKmt7c%253D&md5=b07b9701f64f281d58d178f9c2033596</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fprot.20449&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.20449%26sid%3Dliteratum%253Aachs%26aulast%3DVranken%26aufirst%3DW.%2BF.%26aulast%3DBoucher%26aufirst%3DW.%26aulast%3DStevens%26aufirst%3DT.%2BJ.%26aulast%3DFogh%26aufirst%3DR.%2BH.%26aulast%3DPajon%26aufirst%3DA.%26aulast%3DLlinas%26aufirst%3DM.%26aulast%3DUlrich%26aufirst%3DE.%2BL.%26aulast%3DMarkley%26aufirst%3DJ.%2BL.%26aulast%3DIonides%26aufirst%3DJ.%26aulast%3DLaue%26aufirst%3DE.%2BD.%26atitle%3DThe%2520CCPN%2520data%2520model%2520for%2520NMR%2520spectroscopy%253A%2520development%2520of%2520a%2520software%2520pipeline%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2005%26volume%3D59%26spage%3D687%26epage%3D696%26doi%3D10.1002%2Fprot.20449" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Dignam, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lebovitz, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roeder, R. G.</span><span> </span><span class="NLM_article-title">Accurate transcription initiation by RNA polymerase II in a soluble extract from isolated mammalian nuclei</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1983</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1475</span><span class="NLM_x">–</span> <span class="NLM_lpage">1489</span><span class="refDoi"> DOI: 10.1093/nar/11.5.1475</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1093%2Fnar%2F11.5.1475" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=6828386" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADyaL3sXhslCit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1983&pages=1475-1489&author=J.+D.+Dignamauthor=R.+M.+Lebovitzauthor=R.+G.+Roeder&title=Accurate+transcription+initiation+by+RNA+polymerase+II+in+a+soluble+extract+from+isolated+mammalian+nuclei&doi=10.1093%2Fnar%2F11.5.1475"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate transcription initiation by RNA polymerase II in a soluble extract from isolated mammalian nuclei</span></div><div class="casAuthors">Dignam, John David; Lebovitz, Russell M.; Roeder, Robert G.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1475-89</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    </div><div class="casAbstract">A new procedure was developed for prepg. exts. from nuclei of human tissue culture cells (HeLa cells) that directs accurate transcription initiation in vitro from promoters of class II by RNA polymerase II  [9014-24-8].  Conditions of extn. and assay were optimized for max. activity by using the major late promoter of adenovirus 2.  The ext. also directs accurate transcription initiation from other adenovirus promoters and cellular promoters.  The ext. also directs accurate transcription initiation from class III promoters (tRNA and adenovirus 2 VA).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpacSBbgqUIyrVg90H21EOLACvtfcHk0liJulRqjNyj_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXhslCit74%253D&md5=9a1031cfc1503d0179d166d9efe68ac5</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F11.5.1475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F11.5.1475%26sid%3Dliteratum%253Aachs%26aulast%3DDignam%26aufirst%3DJ.%2BD.%26aulast%3DLebovitz%26aufirst%3DR.%2BM.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26atitle%3DAccurate%2520transcription%2520initiation%2520by%2520RNA%2520polymerase%2520II%2520in%2520a%2520soluble%2520extract%2520from%2520isolated%2520mammalian%2520nuclei%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1983%26volume%3D11%26spage%3D1475%26epage%3D1489%26doi%3D10.1093%2Fnar%2F11.5.1475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Zillmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zapp, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berget, S. M.</span><span> </span><span class="NLM_article-title">Gel electrophoretic isolation of splicing complexes containing U1 small nuclear ribonucleoprotein particles</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">814</span><span class="NLM_x">–</span> <span class="NLM_lpage">821</span><span class="refDoi"> DOI: 10.1128/MCB.8.2.814</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1128%2FMCB.8.2.814" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=2832738" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADyaL1cXhtV2isr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1988&pages=814-821&author=M.+Zillmannauthor=M.+L.+Zappauthor=S.+M.+Berget&title=Gel+electrophoretic+isolation+of+splicing+complexes+containing+U1+small+nuclear+ribonucleoprotein+particles&doi=10.1128%2FMCB.8.2.814"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Gel electrophoretic isolation of splicing complexes containing U1 small nuclear ribonucleoprotein particles</span></div><div class="casAuthors">Zillmann, Martin; Zapp, Maria L.; Berget, Susan M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">814-21</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    </div><div class="casAbstract">Assembly of splicing precursor RNAs into ribonucleoprotein particle (RNP) complexes during incubation in in vitro splicing exts. was monitored by a system of RNP gel electrophoresis.  The temporal pattern of assembly obsd. by the system was identical to that obtained by other gel and gradient methodologies.  In contrast to the results obtained by other systems; however, requirements of U1 small nuclear RNPs (snRNPs) and 5' splice junction sequences were obsd. for formation of specific complexes and retention of U1 snRNPs within gel-fractionated complexes.  Single-intron substrate RNAs rapidly assembled into slow-migration complexes.  The 1st specific complex (A) appeared within a minute of incubation and required ATP, 5' and 3' precursor RNA consensus sequences, and intact U1 and I1 RNAs for formation.  A 2nd complex (B) contg. precursor RNA appeared after 15 min of incubation.  Lariat-exon 2 and exon 1 intermediates first appeared in this complex, operationally defining it as the active spliceosome.  U4 RNA was required for appearance of complex B.  Released lariat 1st appeared in a complex of intermediate mobility (A') and subsequently in rapidly migrating diffuse complexes.  Ligated product RNA was obsd. only in fast-migrating complexes.  U1 snRNPs were detected as components of gel-isolated complexes.  Radiolabeled RNA within the A and B complexes were immunopptd. by U1-specific antibodies under gel-loading conditions and from gel-isolated complexes.  Therefore, the RNP antigen remained assocd. with assembled complexes during gel electrophoresis.  In addn., 5' splice junction sequences within gel-isolated A and B complexes were inaccessible to RNase H cleavage in the presence of a complementary oligonucleotide.  Therefore, nuclear factors that bind 5' splice junctions also remained assocd. with 5' splice junctions under these gel conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6jWKP8lpIy7Vg90H21EOLACvtfcHk0lhymQDzjieIMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXhtV2isr8%253D&md5=0f865109f6601b5b8694cff408827500</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1128%2FMCB.8.2.814&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.8.2.814%26sid%3Dliteratum%253Aachs%26aulast%3DZillmann%26aufirst%3DM.%26aulast%3DZapp%26aufirst%3DM.%2BL.%26aulast%3DBerget%26aufirst%3DS.%2BM.%26atitle%3DGel%2520electrophoretic%2520isolation%2520of%2520splicing%2520complexes%2520containing%2520U1%2520small%2520nuclear%2520ribonucleoprotein%2520particles%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1988%26volume%3D8%26spage%3D814%26epage%3D821%26doi%3D10.1128%2FMCB.8.2.814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Das, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reed, R.</span><span> </span><span class="NLM_article-title">Resolution of the mammalian E complex and the ATP-dependent spliceosomal complexes on native agarose mini-gels</span> <span class="citation_source-journal">RNA</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1504</span><span class="NLM_x">–</span> <span class="NLM_lpage">1508</span><span class="refDoi"> DOI: 10.1017/S1355838299991501</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1017%2FS1355838299991501" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1999&pages=1504-1508&author=R.+Dasauthor=R.+Reed&title=Resolution+of+the+mammalian+E+complex+and+the+ATP-dependent+spliceosomal+complexes+on+native+agarose+mini-gels&doi=10.1017%2FS1355838299991501"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1017%2FS1355838299991501&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS1355838299991501%26sid%3Dliteratum%253Aachs%26aulast%3DDas%26aufirst%3DR.%26aulast%3DReed%26aufirst%3DR.%26atitle%3DResolution%2520of%2520the%2520mammalian%2520E%2520complex%2520and%2520the%2520ATP-dependent%2520spliceosomal%2520complexes%2520on%2520native%2520agarose%2520mini-gels%26jtitle%3DRNA%26date%3D1999%26volume%3D5%26spage%3D1504%26epage%3D1508%26doi%3D10.1017%2FS1355838299991501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Heckmann, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kessler, H.</span><span> </span><span class="NLM_article-title">Design and chemical synthesis of integrin ligands</span> <span class="citation_source-journal">Methods Enzymol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">426</span><span class="NLM_x">, </span> <span class="NLM_fpage">463</span><span class="NLM_x">–</span> <span class="NLM_lpage">503</span><span class="refDoi"> DOI: 10.1016/S0076-6879(07)26020-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1016%2FS0076-6879%2807%2926020-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=17697896" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFOit7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=426&publication_year=2007&pages=463-503&author=D.+Heckmannauthor=H.+Kessler&title=Design+and+chemical+synthesis+of+integrin+ligands&doi=10.1016%2FS0076-6879%2807%2926020-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Design and chemical synthesis of integrin ligands</span></div><div class="casAuthors">Heckmann, Dominik; Kessler, Horst</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">426</span>
        (<span class="NLM_cas:issue">Integrins</span>),
    <span class="NLM_cas:pages">463-503</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The design and synthesis of peptidic and nonpeptidic integrin ligands derived from the most abundant natural tripeptide sequence, RGD, are described in this article.  Special emphasis is placed on the activity and selectivity of the ligands to integrin subtypes.  Two approaches are described - ligand- and structure-oriented design.  When no structure of the complex or the target is known, one may derive suitable starting points from natural peptide sequences, which often require conformational restriction for a further optimization.  A "spatial screening" procedure was used to identify highly active and selective ligands for the integrin subtypes αvβ3 and αIIbβ3.  Structure-based methods require knowledge of the binding domain of the target.  Hence, the first structure of the αvβ3 integrin with bound cilengitide was a landmark for the structure-based approach.  Meanwhile, a design using homol. models of other integrin subtypes has also been successfully applied.  To improve the ADME profile, nonpeptidic ligands have been developed using the information of the spatial distances and orientations of the most important pharmacophoric groups (esp. the carboxyl group and the basic moiety at the other end of the mol.).  Applications of the αvβ3 ligands as drugs in antiangiogenic tumor therapy for mol. imaging of metastases and for improvement of biocompatibility of grafts are briefly described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHsE0gLE-XLbVg90H21EOLACvtfcHk0lhymQDzjieIMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFOit7s%253D&md5=9adbfa1684b2f5f33997c5c686314a64</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2807%2926020-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252807%252926020-3%26sid%3Dliteratum%253Aachs%26aulast%3DHeckmann%26aufirst%3DD.%26aulast%3DKessler%26aufirst%3DH.%26atitle%3DDesign%2520and%2520chemical%2520synthesis%2520of%2520integrin%2520ligands%26jtitle%3DMethods%2520Enzymol.%26date%3D2007%26volume%3D426%26spage%3D463%26epage%3D503%26doi%3D10.1016%2FS0076-6879%2807%2926020-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 13 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Alpeshkumar
K. Malde, Timothy A. Hill, Abishek Iyer, <span class="NLM_string-name hlFld-ContribAuthor">David P. Fairlie</span>. </span><span class="cited-content_cbyCitation_article-title">Crystal Structures of Protein-Bound Cyclic Peptides. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2019,</strong> <em>119 </em>
                                    (17)
                                     , 9861-9914. <a href="https://doi.org/10.1021/acs.chemrev.8b00807" title="DOI URL">https://doi.org/10.1021/acs.chemrev.8b00807</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.8b00807&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.8b00807%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DCrystal%252BStructures%252Bof%252BProtein-Bound%252BCyclic%252BPeptides%26aulast%3DMalde%26aufirst%3DAlpeshkumar%2BK.%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D31122018%26date%3D02052019%26volume%3D119%26issue%3D17%26spage%3D9861%26epage%3D9914" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">I Gede Putu  Supadmanaba</span>, <span class="hlFld-ContribAuthor ">Giulia  Mantini</span>, <span class="hlFld-ContribAuthor ">Ornella  Randazzo</span>, <span class="hlFld-ContribAuthor ">Mjriam  Capula</span>, <span class="hlFld-ContribAuthor ">Ittai B.  Muller</span>, <span class="hlFld-ContribAuthor ">Stella  Cascioferro</span>, <span class="hlFld-ContribAuthor ">Patrizia  Diana</span>, <span class="hlFld-ContribAuthor ">Godefridus J.  Peters</span>, <span class="hlFld-ContribAuthor ">Elisa  Giovannetti</span>. </span><span class="cited-content_cbyCitation_article-title">Interrelationship between miRNA and splicing factors in pancreatic ductal adenocarcinoma. </span><span class="cited-content_cbyCitation_journal-name">Epigenetics</span><span> <strong>2021,</strong> <em>111 </em>, 1-24. <a href="https://doi.org/10.1080/15592294.2021.1916697" title="DOI URL">https://doi.org/10.1080/15592294.2021.1916697</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/15592294.2021.1916697&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F15592294.2021.1916697%26sid%3Dliteratum%253Aachs%26jtitle%3DEpigenetics%26atitle%3DInterrelationship%252Bbetween%252BmiRNA%252Band%252Bsplicing%252Bfactors%252Bin%252Bpancreatic%252Bductal%252Badenocarcinoma%26aulast%3DSupadmanaba%26aufirst%3DI%2BGede%2BPutu%26date%3D2021%26date%3D2021%26volume%3D111%26spage%3D1%26epage%3D24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maxime  Blijlevens</span>, <span class="hlFld-ContribAuthor ">Jing  Li</span>, <span class="hlFld-ContribAuthor ">Victor W.  van Beusechem</span>. </span><span class="cited-content_cbyCitation_article-title">Biology of the mRNA Splicing Machinery and Its Dysregulation in Cancer Providing Therapeutic Opportunities. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2021,</strong> <em>22 </em>
                                    (10)
                                     , 5110. <a href="https://doi.org/10.3390/ijms22105110" title="DOI URL">https://doi.org/10.3390/ijms22105110</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms22105110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms22105110%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DBiology%252Bof%252Bthe%252BmRNA%252BSplicing%252BMachinery%252Band%252BIts%252BDysregulation%252Bin%252BCancer%252BProviding%252BTherapeutic%252BOpportunities%26aulast%3DBlijlevens%26aufirst%3DMaxime%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D10%26spage%3D5110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Silpa  S</span>, <span class="hlFld-ContribAuthor ">Rupachandra  S</span>. </span><span class="cited-content_cbyCitation_article-title">Cyclic peptide production from lactic acid bacteria (LAB) and their diverse applications. </span><span class="cited-content_cbyCitation_journal-name">Critical Reviews in Food Science and Nutrition</span><span> <strong>2020,</strong> <em>67 </em>, 1-20. <a href="https://doi.org/10.1080/10408398.2020.1860900" title="DOI URL">https://doi.org/10.1080/10408398.2020.1860900</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/10408398.2020.1860900&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F10408398.2020.1860900%26sid%3Dliteratum%253Aachs%26jtitle%3DCritical%2520Reviews%2520in%2520Food%2520Science%2520and%2520Nutrition%26atitle%3DCyclic%252Bpeptide%252Bproduction%252Bfrom%252Blactic%252Bacid%252Bbacteria%252B%252528LAB%252529%252Band%252Btheir%252Bdiverse%252Bapplications%26aulast%3DS%26aufirst%3DSilpa%26date%3D2020%26date%3D2020%26volume%3D67%26spage%3D1%26epage%3D20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pravin Kumar Ankush  Jagtap</span>, <span class="hlFld-ContribAuthor ">Tomáš  Kubelka</span>, <span class="hlFld-ContribAuthor ">Komal  Soni</span>, <span class="hlFld-ContribAuthor ">Cindy L.  Will</span>, <span class="hlFld-ContribAuthor ">Divita  Garg</span>, <span class="hlFld-ContribAuthor ">Claudia  Sippel</span>, <span class="hlFld-ContribAuthor ">Tobias G.  Kapp</span>, <span class="hlFld-ContribAuthor ">Harish Kumar  Potukuchi</span>, <span class="hlFld-ContribAuthor ">Kenji  Schorpp</span>, <span class="hlFld-ContribAuthor ">Kamyar  Hadian</span>, <span class="hlFld-ContribAuthor ">Horst  Kessler</span>, <span class="hlFld-ContribAuthor ">Reinhard  Lührmann</span>, <span class="hlFld-ContribAuthor ">Felix  Hausch</span>, <span class="hlFld-ContribAuthor ">Thorsten  Bach</span>, <span class="hlFld-ContribAuthor ">Michael  Sattler</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of phenothiazine derivatives as UHM-binding inhibitors of early spliceosome assembly. </span><span class="cited-content_cbyCitation_journal-name">Nature Communications</span><span> <strong>2020,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41467-020-19514-1" title="DOI URL">https://doi.org/10.1038/s41467-020-19514-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41467-020-19514-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41467-020-19514-1%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Communications%26atitle%3DIdentification%252Bof%252Bphenothiazine%252Bderivatives%252Bas%252BUHM-binding%252Binhibitors%252Bof%252Bearly%252Bspliceosome%252Bassembly%26aulast%3DJagtap%26aufirst%3DPravin%2BKumar%2BAnkush%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sophie C.  Bonnal</span>, <span class="hlFld-ContribAuthor ">Irene  López-Oreja</span>, <span class="hlFld-ContribAuthor ">Juan  Valcárcel</span>. </span><span class="cited-content_cbyCitation_article-title">Roles and mechanisms of alternative splicing in cancer — implications for care. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Clinical Oncology</span><span> <strong>2020,</strong> <em>17 </em>
                                    (8)
                                     , 457-474. <a href="https://doi.org/10.1038/s41571-020-0350-x" title="DOI URL">https://doi.org/10.1038/s41571-020-0350-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41571-020-0350-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41571-020-0350-x%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Clinical%2520Oncology%26atitle%3DRoles%252Band%252Bmechanisms%252Bof%252Balternative%252Bsplicing%252Bin%252Bcancer%252B%2525E2%252580%252594%252Bimplications%252Bfor%252Bcare%26aulast%3DBonnal%26aufirst%3DSophie%2BC.%26date%3D2020%26date%3D2020%26volume%3D17%26issue%3D8%26spage%3D457%26epage%3D474" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anne C.  Conibear</span>, <span class="hlFld-ContribAuthor ">Alanca  Schmid</span>, <span class="hlFld-ContribAuthor ">Meder  Kamalov</span>, <span class="hlFld-ContribAuthor ">Christian F.W.  Becker</span>, <span class="hlFld-ContribAuthor ">Claudia  Bello</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Advances in Peptide-Based Approaches for Cancer Treatment. </span><span class="cited-content_cbyCitation_journal-name">Current Medicinal Chemistry</span><span> <strong>2020,</strong> <em>27 </em>
                                    (8)
                                     , 1174-1205. <a href="https://doi.org/10.2174/0929867325666171123204851" title="DOI URL">https://doi.org/10.2174/0929867325666171123204851</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/0929867325666171123204851&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F0929867325666171123204851%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Medicinal%2520Chemistry%26atitle%3DRecent%252BAdvances%252Bin%252BPeptide-Based%252BApproaches%252Bfor%252BCancer%252BTreatment%26aulast%3DConibear%26aufirst%3DAnne%2BC.%26date%3D2020%26volume%3D27%26issue%3D8%26spage%3D1174%26epage%3D1205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Haley I.  Merritt</span>, <span class="hlFld-ContribAuthor ">Nicholas  Sawyer</span>, <span class="hlFld-ContribAuthor ">Paramjit S.  Arora</span>. </span><span class="cited-content_cbyCitation_article-title">Bent into shape: Folded peptides to mimic protein structure and modulate protein function. </span><span class="cited-content_cbyCitation_journal-name">Peptide Science</span><span> <strong>2020,</strong> <em>112 </em>
                                    (1)
                                     <a href="https://doi.org/10.1002/pep2.24145" title="DOI URL">https://doi.org/10.1002/pep2.24145</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/pep2.24145&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fpep2.24145%26sid%3Dliteratum%253Aachs%26jtitle%3DPeptide%2520Science%26atitle%3DBent%252Binto%252Bshape%25253A%252BFolded%252Bpeptides%252Bto%252Bmimic%252Bprotein%252Bstructure%252Band%252Bmodulate%252Bprotein%252Bfunction%26aulast%3DMerritt%26aufirst%3DHaley%2BI.%26date%3D2020%26date%3D2020%26volume%3D112%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Joon-Seok  Choi</span>, <span class="hlFld-ContribAuthor ">Sang Hoon  Joo</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Trends in Cyclic Peptides as Therapeutic Agents and Biochemical Tools. </span><span class="cited-content_cbyCitation_journal-name">Biomolecules & Therapeutics</span><span> <strong>2020,</strong> <em>28 </em>
                                    (1)
                                     , 18-24. <a href="https://doi.org/10.4062/biomolther.2019.082" title="DOI URL">https://doi.org/10.4062/biomolther.2019.082</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4062/biomolther.2019.082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4062%2Fbiomolther.2019.082%26sid%3Dliteratum%253Aachs%26jtitle%3DBiomolecules%2520%2526%2520Therapeutics%26atitle%3DRecent%252BTrends%252Bin%252BCyclic%252BPeptides%252Bas%252BTherapeutic%252BAgents%252Band%252BBiochemical%252BTools%26aulast%3DChoi%26aufirst%3DJoon-Seok%26date%3D2020%26volume%3D28%26issue%3D1%26spage%3D18%26epage%3D24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anthony B  DeNicola</span>, <span class="hlFld-ContribAuthor ">Yi  Tang</span>. </span><span class="cited-content_cbyCitation_article-title">Therapeutic approaches to treat human spliceosomal diseases. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Biotechnology</span><span> <strong>2019,</strong> <em>60 </em>, 72-81. <a href="https://doi.org/10.1016/j.copbio.2019.01.003" title="DOI URL">https://doi.org/10.1016/j.copbio.2019.01.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.copbio.2019.01.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.copbio.2019.01.003%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Biotechnology%26atitle%3DTherapeutic%252Bapproaches%252Bto%252Btreat%252Bhuman%252Bspliceosomal%252Bdiseases%26aulast%3DDeNicola%26aufirst%3DAnthony%2BB%26date%3D2019%26volume%3D60%26spage%3D72%26epage%3D81" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marta  Pabis</span>, <span class="hlFld-ContribAuthor ">Lorenzo  Corsini</span>, <span class="hlFld-ContribAuthor ">Michelle  Vincendeau</span>, <span class="hlFld-ContribAuthor ">Konstantinos  Tripsianes</span>, <span class="hlFld-ContribAuthor ">Toby J  Gibson</span>, <span class="hlFld-ContribAuthor ">Ruth  Brack-Werner</span>, <span class="hlFld-ContribAuthor ">Michael  Sattler</span>. </span><span class="cited-content_cbyCitation_article-title">Modulation of HIV-1 gene expression by binding of a ULM motif in the Rev protein to UHM-containing splicing factors. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2019,</strong> <em>47 </em>
                                    (9)
                                     , 4859-4871. <a href="https://doi.org/10.1093/nar/gkz185" title="DOI URL">https://doi.org/10.1093/nar/gkz185</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gkz185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgkz185%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DModulation%252Bof%252BHIV-1%252Bgene%252Bexpression%252Bby%252Bbinding%252Bof%252Ba%252BULM%252Bmotif%252Bin%252Bthe%252BRev%252Bprotein%252Bto%252BUHM-containing%252Bsplicing%252Bfactors%26aulast%3DPabis%26aufirst%3DMarta%26date%3D2019%26date%3D2019%26volume%3D47%26issue%3D9%26spage%3D4859%26epage%3D4871" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jianglong  Lu</span>, <span class="hlFld-ContribAuthor ">Qun  Li</span>, <span class="hlFld-ContribAuthor ">Lin  Cai</span>, <span class="hlFld-ContribAuthor ">Zhangzhang  Zhu</span>, <span class="hlFld-ContribAuthor ">Jiaqing  Guan</span>, <span class="hlFld-ContribAuthor ">Chunyong  Wang</span>, <span class="hlFld-ContribAuthor ">Jia  Xia</span>, <span class="hlFld-ContribAuthor ">Lei  Xia</span>, <span class="hlFld-ContribAuthor ">Min  Wen</span>, <span class="hlFld-ContribAuthor ">Weiming  Zheng</span>, <span class="hlFld-ContribAuthor ">Zhipeng  Su</span>, <span class="hlFld-ContribAuthor ">Chengde  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">RBM17 controls apoptosis and proliferation to promote Glioma progression. </span><span class="cited-content_cbyCitation_journal-name">Biochemical and Biophysical Research Communications</span><span> <strong>2018,</strong> <em>505 </em>
                                    (1)
                                     , 20-28. <a href="https://doi.org/10.1016/j.bbrc.2018.09.056" title="DOI URL">https://doi.org/10.1016/j.bbrc.2018.09.056</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbrc.2018.09.056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbrc.2018.09.056%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520and%2520Biophysical%2520Research%2520Communications%26atitle%3DRBM17%252Bcontrols%252Bapoptosis%252Band%252Bproliferation%252Bto%252Bpromote%252BGlioma%252Bprogression%26aulast%3DLu%26aufirst%3DJianglong%26date%3D2018%26volume%3D505%26issue%3D1%26spage%3D20%26epage%3D28" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anant A  Agrawal</span>, <span class="hlFld-ContribAuthor ">Lihua  Yu</span>, <span class="hlFld-ContribAuthor ">Peter G  Smith</span>, <span class="hlFld-ContribAuthor ">Silvia  Buonamici</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting splicing abnormalities in cancer. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Genetics & Development</span><span> <strong>2018,</strong> <em>48 </em>, 67-74. <a href="https://doi.org/10.1016/j.gde.2017.10.010" title="DOI URL">https://doi.org/10.1016/j.gde.2017.10.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.gde.2017.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.gde.2017.10.010%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Genetics%2520%2526%2520Development%26atitle%3DTargeting%252Bsplicing%252Babnormalities%252Bin%252Bcancer%26aulast%3DAgrawal%26aufirst%3DAnant%2BA%26date%3D2018%26volume%3D48%26spage%3D67%26epage%3D74" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-22/acs.jmedchem.6b01118/20161118/images/medium/jm-2016-01118z_0005.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-22/acs.jmedchem.6b01118/20161118/images/large/jm-2016-01118z_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01118&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-22/acs.jmedchem.6b01118/20161118/images/medium/jm-2016-01118z_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-22/acs.jmedchem.6b01118/20161118/images/large/jm-2016-01118z_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Schematic overview of UHM–ULM interactions during spliceosome assembly. UHM domains and ULM peptide motifs are shown in green and red, respectively. (B) Alignment of ULM sequences.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-22/acs.jmedchem.6b01118/20161118/images/large/jm-2016-01118z_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01118&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-22/acs.jmedchem.6b01118/20161118/images/medium/jm-2016-01118z_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-22/acs.jmedchem.6b01118/20161118/images/large/jm-2016-01118z_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Co-crystal structure of SPF45 UHM in complex with peptide <b>3</b> inhibitor. The peptide and the UHM domain are shown in salmon and green, respectively. Yellow dashed lines indicate hydrogen bonds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-22/acs.jmedchem.6b01118/20161118/images/large/jm-2016-01118z_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01118&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-22/acs.jmedchem.6b01118/20161118/images/medium/jm-2016-01118z_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-22/acs.jmedchem.6b01118/20161118/images/large/jm-2016-01118z_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) Schematic overview of the synthesized focused library of the cyclic peptides. The different amino acids and functional groups added to peptide <b>3</b> at the R1 and R2 positions are listed. (B) <sup>1</sup>H,<sup>15</sup>N HSQC spectra superposition of the SPF45 UHM domain bound to peptides <b>3</b> (black) and <b>10</b> (red) is shown. In the SPF45 UHM domain, residues Gly379, Gly378, and Phe377 exhibit large chemical shift changes induced by peptide <b>10</b> binding compared to peptide <b>3</b>. These residues are in close proximity to Tyr376. (C) Backbone amides of Gly379, Gly378, and Phe377 residues are shown with red spheres on the SPF45 UHM–peptide <b>3</b> complex structure. A red oval shows the attachment position of amino acid R1 in peptide <b>3</b> along with the direction in which it is expected to extend. (D) Selectivity profile of the designed cyclic peptides for the SPF45 and U2AF65 UHM domains (selectivity = <i>K</i><sub>D</sub><sup>U2AF65</sup>/<i>K</i><sub>D</sub><sup>SPF45</sup>; <a href="/doi/suppl/10.1021/acs.jmedchem.6b01118/suppl_file/jm6b01118_si_001.pdf" class="ext-link">Table S3</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-22/acs.jmedchem.6b01118/20161118/images/large/jm-2016-01118z_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01118&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-22/acs.jmedchem.6b01118/20161118/images/medium/jm-2016-01118z_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-22/acs.jmedchem.6b01118/20161118/images/large/jm-2016-01118z_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) In vitro splicing inhibition assays with peptide <b>10</b>. SF3b155 ULM (RKSRWDETP) and its mutant (RKSRADETP) were used as positive and negative controls, respectively. (B) Quantification of the splicing assay showing % spliced product (normalized to the control without added peptide) formed vs the peptide concentration. Error bars represent standard deviation of the % spliced product measured in two independent experiments. (C) Spliceosome complex formation analyzed by agarose gel electrophoresis. Peptide <b>10</b> inhibits complex A formation during spliceosome assembly, where U2AF UHM–ULM interactions play a crucial role.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-22/acs.jmedchem.6b01118/20161118/images/large/jm-2016-01118z_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01118&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i29">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84285" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84285" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 40 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Nilsen, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graveley, B. R.</span><span> </span><span class="NLM_article-title">Expansion of the eukaryotic proteome by alternative splicing</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">463</span><span class="NLM_x">, </span> <span class="NLM_fpage">457</span><span class="NLM_x">–</span> <span class="NLM_lpage">463</span><span class="refDoi"> DOI: 10.1038/nature08909</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1038%2Fnature08909" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=20110989" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1SltbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=463&publication_year=2010&pages=457-463&author=T.+W.+Nilsenauthor=B.+R.+Graveley&title=Expansion+of+the+eukaryotic+proteome+by+alternative+splicing&doi=10.1038%2Fnature08909"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Expansion of the eukaryotic proteome by alternative splicing</span></div><div class="casAuthors">Nilsen, Timothy W.; Graveley, Brenton R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">463</span>
        (<span class="NLM_cas:issue">7280</span>),
    <span class="NLM_cas:pages">457-463</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The collection of components required to carry out the intricate processes involved in generating and maintaining a living, breathing and, sometimes, thinking organism is staggeringly complex.  Where do all of the parts come from.  Early ests. stated that about 100,000 genes would be required to make up a mammal; however, the actual no. is less than one-quarter of that, barely four times the no. of genes in budding yeast.  It is now clear that the 'missing' information is in large part provided by alternative splicing, the process by which multiple different functional mRNAs, and therefore proteins, can be synthesized from a single gene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeWu61GQZ5hbVg90H21EOLACvtfcHk0lhMfQhedbMDYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1SltbY%253D&md5=133169961152db13f6bcf6e3d4b5eb6a</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnature08909&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08909%26sid%3Dliteratum%253Aachs%26aulast%3DNilsen%26aufirst%3DT.%2BW.%26aulast%3DGraveley%26aufirst%3DB.%2BR.%26atitle%3DExpansion%2520of%2520the%2520eukaryotic%2520proteome%2520by%2520alternative%2520splicing%26jtitle%3DNature%26date%3D2010%26volume%3D463%26spage%3D457%26epage%3D463%26doi%3D10.1038%2Fnature08909" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Wahl, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Will, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luhrmann, R.</span><span> </span><span class="NLM_article-title">The spliceosome: design principles of a dynamic RNP machine</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">136</span><span class="NLM_x">, </span> <span class="NLM_fpage">701</span><span class="NLM_x">–</span> <span class="NLM_lpage">718</span><span class="refDoi"> DOI: 10.1016/j.cell.2009.02.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1016%2Fj.cell.2009.02.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=19239890" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADC%252BD1MXkvFGksbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2009&pages=701-718&author=M.+C.+Wahlauthor=C.+L.+Willauthor=R.+Luhrmann&title=The+spliceosome%3A+design+principles+of+a+dynamic+RNP+machine&doi=10.1016%2Fj.cell.2009.02.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The spliceosome: Design principles of a dynamic RNP machine</span></div><div class="casAuthors">Wahl, Markus C.; Will, Cindy L.; Luehrmann, Reinhard</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">701-718</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Ribonucleoproteins (RNPs) mediate key cellular functions such as gene expression and its regulation.  Whereas most RNP enzymes are stable in compn. and harbor preformed active sites, the spliceosome, which removes noncoding introns from precursor mRNAs (pre-mRNAs), follows fundamentally different strategies.  In order to provide both accuracy to the recognition of reactive splice sites in the pre-mRNA and flexibility to the choice of splice sites during alternative splicing, the spliceosome exhibits exceptional compositional and structural dynamics that are exploited during substrate-dependent complex assembly, catalytic activation, and active site remodeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphwXkeaEfBWLVg90H21EOLACvtfcHk0lhMfQhedbMDYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXkvFGksbk%253D&md5=589aa1782ba069c55bff0b6bb7762b84</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2009.02.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2009.02.009%26sid%3Dliteratum%253Aachs%26aulast%3DWahl%26aufirst%3DM.%2BC.%26aulast%3DWill%26aufirst%3DC.%2BL.%26aulast%3DLuhrmann%26aufirst%3DR.%26atitle%3DThe%2520spliceosome%253A%2520design%2520principles%2520of%2520a%2520dynamic%2520RNP%2520machine%26jtitle%3DCell%26date%3D2009%26volume%3D136%26spage%3D701%26epage%3D718%26doi%3D10.1016%2Fj.cell.2009.02.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Xiong, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alipanahi, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bretschneider, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merico, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuen, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hua, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gueroussov, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Najafabadi, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morris, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barash, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krainer, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jojic, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scherer, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blencowe, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frey, B. J.</span><span> </span><span class="NLM_article-title">RNA splicing. The human splicing code reveals new insights into the genetic determinants of disease</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">347</span><span class="NLM_x">, </span> <span class="NLM_fpage">1254806</span><span class="refDoi"> DOI: 10.1126/science.1254806</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1126%2Fscience.1254806" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=25525159" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A280%3ADC%252BC2MzosV2rug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=347&publication_year=2015&pages=1254806&author=H.+Y.+Xiongauthor=B.+Alipanahiauthor=L.+J.+Leeauthor=H.+Bretschneiderauthor=D.+Mericoauthor=R.+K.+Yuenauthor=Y.+Huaauthor=S.+Gueroussovauthor=H.+S.+Najafabadiauthor=T.+R.+Hughesauthor=Q.+Morrisauthor=Y.+Barashauthor=A.+R.+Krainerauthor=N.+Jojicauthor=S.+W.+Schererauthor=B.+J.+Blencoweauthor=B.+J.+Frey&title=RNA+splicing.+The+human+splicing+code+reveals+new+insights+into+the+genetic+determinants+of+disease&doi=10.1126%2Fscience.1254806"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">RNA splicing. The human splicing code reveals new insights into the genetic determinants of disease</span></div><div class="casAuthors">Xiong Hui Y; Alipanahi Babak; Lee Leo J; Najafabadi Hamed S; Bretschneider Hannes; Merico Daniele; Yuen Ryan K C; Hua Yimin; Krainer Adrian R; Gueroussov Serge; Hughes Timothy R; Morris Quaid; Barash Yoseph; Jojic Nebojsa; Scherer Stephen W; Blencowe Benjamin J; Frey Brendan J</div><div class="citationInfo"><span class="NLM_cas:title">Science (New York, N.Y.)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">347</span>
        (<span class="NLM_cas:issue">6218</span>),
    <span class="NLM_cas:pages">1254806</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">To facilitate precision medicine and whole-genome annotation, we developed a machine-learning technique that scores how strongly genetic variants affect RNA splicing, whose alteration contributes to many diseases.  Analysis of more than 650,000 intronic and exonic variants revealed widespread patterns of mutation-driven aberrant splicing.  Intronic disease mutations that are more than 30 nucleotides from any splice site alter splicing nine times as often as common variants, and missense exonic disease mutations that have the least impact on protein function are five times as likely as others to alter splicing.  We detected tens of thousands of disease-causing mutations, including those involved in cancers and spinal muscular atrophy.  Examination of intronic and exonic variants found using whole-genome sequencing of individuals with autism revealed misspliced genes with neurodevelopmental phenotypes.  Our approach provides evidence for causal variants and should enable new discoveries in precision medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQChTiwuM_BhekdrW1FkXG-fW6udTcc2eZRFFymALUqprntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MzosV2rug%253D%253D&md5=3edc5e7004277b12dc3fd9cd06430f08</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1126%2Fscience.1254806&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1254806%26sid%3Dliteratum%253Aachs%26aulast%3DXiong%26aufirst%3DH.%2BY.%26aulast%3DAlipanahi%26aufirst%3DB.%26aulast%3DLee%26aufirst%3DL.%2BJ.%26aulast%3DBretschneider%26aufirst%3DH.%26aulast%3DMerico%26aufirst%3DD.%26aulast%3DYuen%26aufirst%3DR.%2BK.%26aulast%3DHua%26aufirst%3DY.%26aulast%3DGueroussov%26aufirst%3DS.%26aulast%3DNajafabadi%26aufirst%3DH.%2BS.%26aulast%3DHughes%26aufirst%3DT.%2BR.%26aulast%3DMorris%26aufirst%3DQ.%26aulast%3DBarash%26aufirst%3DY.%26aulast%3DKrainer%26aufirst%3DA.%2BR.%26aulast%3DJojic%26aufirst%3DN.%26aulast%3DScherer%26aufirst%3DS.%2BW.%26aulast%3DBlencowe%26aufirst%3DB.%2BJ.%26aulast%3DFrey%26aufirst%3DB.%2BJ.%26atitle%3DRNA%2520splicing.%2520The%2520human%2520splicing%2520code%2520reveals%2520new%2520insights%2520into%2520the%2520genetic%2520determinants%2520of%2520disease%26jtitle%3DScience%26date%3D2015%26volume%3D347%26spage%3D1254806%26doi%3D10.1126%2Fscience.1254806" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Ray, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kazan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, K. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weirauch, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Najafabadi, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gueroussov, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Na, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Irimia, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matzat, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dale, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yarosh, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nabet, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mecenas, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laishram, R. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qiao, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lipshitz, H. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piano, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corbett, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carstens, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frey, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Penalva, L. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, E. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fraser, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blencowe, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morris, Q. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, T. R.</span><span> </span><span class="NLM_article-title">A compendium of RNA-binding motifs for decoding gene regulation</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">499</span><span class="NLM_x">, </span> <span class="NLM_fpage">172</span><span class="NLM_x">–</span> <span class="NLM_lpage">177</span><span class="refDoi"> DOI: 10.1038/nature12311</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1038%2Fnature12311" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=23846655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVyksbjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=499&publication_year=2013&pages=172-177&author=D.+Rayauthor=H.+Kazanauthor=K.+B.+Cookauthor=M.+T.+Weirauchauthor=H.+S.+Najafabadiauthor=X.+Liauthor=S.+Gueroussovauthor=M.+Albuauthor=H.+Zhengauthor=A.+Yangauthor=H.+Naauthor=M.+Irimiaauthor=L.+H.+Matzatauthor=R.+K.+Daleauthor=S.+A.+Smithauthor=C.+A.+Yaroshauthor=S.+M.+Kellyauthor=B.+Nabetauthor=D.+Mecenasauthor=W.+Liauthor=R.+S.+Laishramauthor=M.+Qiaoauthor=H.+D.+Lipshitzauthor=F.+Pianoauthor=A.+H.+Corbettauthor=R.+P.+Carstensauthor=B.+J.+Freyauthor=R.+A.+Andersonauthor=K.+W.+Lynchauthor=L.+O.+Penalvaauthor=E.+P.+Leiauthor=A.+G.+Fraserauthor=B.+J.+Blencoweauthor=Q.+D.+Morrisauthor=T.+R.+Hughes&title=A+compendium+of+RNA-binding+motifs+for+decoding+gene+regulation&doi=10.1038%2Fnature12311"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">A compendium of RNA-binding motifs for decoding gene regulation</span></div><div class="casAuthors">Ray, Debashish; Kazan, Hilal; Cook, Kate B.; Weirauch, Matthew T.; Najafabadi, Hamed S.; Li, Xiao; Gueroussov, Serge; Albu, Mihai; Zheng, Hong; Yang, Ally; Na, Hong; Irimia, Manuel; Matzat, Leah H.; Dale, Ryan K.; Smith, Sarah A.; Yarosh, Christopher A.; Kelly, Seth M.; Nabet, Behnam; Mecenas, Desirea; Li, Weimin; Laishram, Rakesh S.; Qiao, Mei; Lipshitz, Howard D.; Piano, Fabio; Corbett, Anita H.; Carstens, Russ P.; Frey, Brendan J.; Anderson, Richard A.; Lynch, Kristen W.; Penalva, Luiz O. F.; Lei, Elissa P.; Fraser, Andrew G.; Blencowe, Benjamin J.; Morris, Quaid D.; Hughes, Timothy R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">499</span>
        (<span class="NLM_cas:issue">7457</span>),
    <span class="NLM_cas:pages">172-177</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">RNA-binding proteins are key regulators of gene expression, yet only a small fraction have been functionally characterized.  Here we report a systematic anal. of the RNA motifs recognized by RNA-binding proteins, encompassing 205 distinct genes from 24 diverse eukaryotes.  The sequence specificities of RNA-binding proteins display deep evolutionary conservation, and the recognition preferences for a large fraction of metazoan RNA-binding proteins can thus be inferred from their RNA-binding domain sequence.  The motifs that we identify in vitro correlate well with in vivo RNA-binding data.  Moreover, we can assoc. them with distinct functional roles in diverse types of post-transcriptional regulation, enabling new insights into the functions of RNA-binding proteins both in normal physiol. and in human disease.  These data provide an unprecedented overview of RNA-binding proteins and their targets, and constitute an invaluable resource for detg. post-transcriptional regulatory mechanisms in eukaryotes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-R4r7lsTSl7Vg90H21EOLACvtfcHk0ljYUhGyl_oPxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVyksbjF&md5=4729aa8faf50eae0e2e91d31112d6311</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnature12311&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12311%26sid%3Dliteratum%253Aachs%26aulast%3DRay%26aufirst%3DD.%26aulast%3DKazan%26aufirst%3DH.%26aulast%3DCook%26aufirst%3DK.%2BB.%26aulast%3DWeirauch%26aufirst%3DM.%2BT.%26aulast%3DNajafabadi%26aufirst%3DH.%2BS.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DGueroussov%26aufirst%3DS.%26aulast%3DAlbu%26aufirst%3DM.%26aulast%3DZheng%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DA.%26aulast%3DNa%26aufirst%3DH.%26aulast%3DIrimia%26aufirst%3DM.%26aulast%3DMatzat%26aufirst%3DL.%2BH.%26aulast%3DDale%26aufirst%3DR.%2BK.%26aulast%3DSmith%26aufirst%3DS.%2BA.%26aulast%3DYarosh%26aufirst%3DC.%2BA.%26aulast%3DKelly%26aufirst%3DS.%2BM.%26aulast%3DNabet%26aufirst%3DB.%26aulast%3DMecenas%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLaishram%26aufirst%3DR.%2BS.%26aulast%3DQiao%26aufirst%3DM.%26aulast%3DLipshitz%26aufirst%3DH.%2BD.%26aulast%3DPiano%26aufirst%3DF.%26aulast%3DCorbett%26aufirst%3DA.%2BH.%26aulast%3DCarstens%26aufirst%3DR.%2BP.%26aulast%3DFrey%26aufirst%3DB.%2BJ.%26aulast%3DAnderson%26aufirst%3DR.%2BA.%26aulast%3DLynch%26aufirst%3DK.%2BW.%26aulast%3DPenalva%26aufirst%3DL.%2BO.%26aulast%3DLei%26aufirst%3DE.%2BP.%26aulast%3DFraser%26aufirst%3DA.%2BG.%26aulast%3DBlencowe%26aufirst%3DB.%2BJ.%26aulast%3DMorris%26aufirst%3DQ.%2BD.%26aulast%3DHughes%26aufirst%3DT.%2BR.%26atitle%3DA%2520compendium%2520of%2520RNA-binding%2520motifs%2520for%2520decoding%2520gene%2520regulation%26jtitle%3DNature%26date%3D2013%26volume%3D499%26spage%3D172%26epage%3D177%26doi%3D10.1038%2Fnature12311" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Hennig, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gebauer, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sattler, M.</span><span> </span><span class="NLM_article-title">Breaking the protein-RNA recognition code</span> <span class="citation_source-journal">Cell Cycle</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">3619</span><span class="NLM_x">–</span> <span class="NLM_lpage">3620</span><span class="refDoi"> DOI: 10.4161/15384101.2014.986625</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.4161%2F15384101.2014.986625" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=3619-3620&author=J.+Hennigauthor=F.+Gebauerauthor=M.+Sattler&title=Breaking+the+protein-RNA+recognition+code&doi=10.4161%2F15384101.2014.986625"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.4161%2F15384101.2014.986625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252F15384101.2014.986625%26sid%3Dliteratum%253Aachs%26aulast%3DHennig%26aufirst%3DJ.%26aulast%3DGebauer%26aufirst%3DF.%26aulast%3DSattler%26aufirst%3DM.%26atitle%3DBreaking%2520the%2520protein-RNA%2520recognition%2520code%26jtitle%3DCell%2520Cycle%26date%3D2014%26volume%3D13%26spage%3D3619%26epage%3D3620%26doi%3D10.4161%2F15384101.2014.986625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Cooper, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dreyfuss, G.</span><span> </span><span class="NLM_article-title">RNA and disease</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">136</span><span class="NLM_x">, </span> <span class="NLM_fpage">777</span><span class="NLM_x">–</span> <span class="NLM_lpage">793</span><span class="refDoi"> DOI: 10.1016/j.cell.2009.02.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1016%2Fj.cell.2009.02.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=19239895" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADC%252BD1MXkvFGktrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2009&pages=777-793&author=T.+A.+Cooperauthor=L.+Wanauthor=G.+Dreyfuss&title=RNA+and+disease&doi=10.1016%2Fj.cell.2009.02.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">RNA and disease</span></div><div class="casAuthors">Cooper, Thomas A.; Wan, Lili; Dreyfuss, Gideon</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">777-793</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Cellular functions depend on numerous protein-coding and noncoding RNAs and the RNA-binding proteins assocd. with them, which form ribonucleoprotein complexes (RNPs).  Mutations that disrupt either the RNA or protein components of RNPs or the factors required for their assembly can be deleterious.  Alternative splicing provides cells with an exquisite capacity to fine-tune their transcriptome and proteome in response to cues.  Splicing depends on a complex code, numerous RNA-binding proteins, and an enormously intricate network of interactions among them, increasing the opportunity for exposure to mutations and misregulation that cause disease.  The discovery of disease-causing mutations in RNAs is yielding a wealth of new therapeutic targets, and the growing understanding of RNA biol. and chem. is providing new RNA-based tools for developing therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotOMwPdFvVpLVg90H21EOLACvtfcHk0lj6P9EQIK0hSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXkvFGktrw%253D&md5=f165df2b33f97df6e895045b81e853d4</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2009.02.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2009.02.011%26sid%3Dliteratum%253Aachs%26aulast%3DCooper%26aufirst%3DT.%2BA.%26aulast%3DWan%26aufirst%3DL.%26aulast%3DDreyfuss%26aufirst%3DG.%26atitle%3DRNA%2520and%2520disease%26jtitle%3DCell%26date%3D2009%26volume%3D136%26spage%3D777%26epage%3D793%26doi%3D10.1016%2Fj.cell.2009.02.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">David, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manley, J. L.</span><span> </span><span class="NLM_article-title">Alternative pre-mRNA splicing regulation in cancer: pathways and programs unhinged</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">2343</span><span class="NLM_x">–</span> <span class="NLM_lpage">2364</span><span class="refDoi"> DOI: 10.1101/gad.1973010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1101%2Fgad.1973010" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2010&pages=2343-2364&author=C.+J.+Davidauthor=J.+L.+Manley&title=Alternative+pre-mRNA+splicing+regulation+in+cancer%3A+pathways+and+programs+unhinged&doi=10.1101%2Fgad.1973010"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1101%2Fgad.1973010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1973010%26sid%3Dliteratum%253Aachs%26aulast%3DDavid%26aufirst%3DC.%2BJ.%26aulast%3DManley%26aufirst%3DJ.%2BL.%26atitle%3DAlternative%2520pre-mRNA%2520splicing%2520regulation%2520in%2520cancer%253A%2520pathways%2520and%2520programs%2520unhinged%26jtitle%3DGenes%2520Dev.%26date%3D2010%26volume%3D24%26spage%3D2343%26epage%3D2364%26doi%3D10.1101%2Fgad.1973010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Bonnal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vigevani, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valcarcel, J.</span><span> </span><span class="NLM_article-title">The spliceosome as a target of novel antitumour drugs</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">847</span><span class="NLM_x">–</span> <span class="NLM_lpage">859</span><span class="refDoi"> DOI: 10.1038/nrd3823</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1038%2Fnrd3823" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=23123942" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1ajsbrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=847-859&author=S.+Bonnalauthor=L.+Vigevaniauthor=J.+Valcarcel&title=The+spliceosome+as+a+target+of+novel+antitumour+drugs&doi=10.1038%2Fnrd3823"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The spliceosome as a target of novel antitumour drugs</span></div><div class="casAuthors">Bonnal, Sophie; Vigevani, Luisa; Valcarcel, Juan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">847-859</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Several bacterial fermn. products and their synthetic derivs. display antitumor activities and bind tightly to components of the spliceosome, which is the complex mol. machinery involved in the removal of introns from mRNA precursors in eukaryotic cells.  The drugs alter gene expression, including alternative splicing, of genes that are important for cancer progression.  A flurry of recent reports has revealed that genes encoding splicing factors, including the drug target splicing factor 3B subunit 1 (SF3B1), are among the most highly mutated in various haematol. malignancies such as chronic lymphocytic leukemia and myelodysplastic syndromes.  These observations highlight the role of splicing factors in cancer and suggest that an understanding of the mol. effects of drugs targeting these proteins could open new perspectives for studies of the spliceosome and its role in cancer progression, and for the development of novel antitumor therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLmM2p9jQOMLVg90H21EOLACvtfcHk0lj6P9EQIK0hSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1ajsbrJ&md5=c26edf63e7ca3e1ea2ffd93325f9ab1d</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrd3823&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3823%26sid%3Dliteratum%253Aachs%26aulast%3DBonnal%26aufirst%3DS.%26aulast%3DVigevani%26aufirst%3DL.%26aulast%3DValcarcel%26aufirst%3DJ.%26atitle%3DThe%2520spliceosome%2520as%2520a%2520target%2520of%2520novel%2520antitumour%2520drugs%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D847%26epage%3D859%26doi%3D10.1038%2Fnrd3823" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Daguenet, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dujardin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valcarcel, J.</span><span> </span><span class="NLM_article-title">The pathogenicity of splicing defects: mechanistic insights into pre-mRNA processing inform novel therapeutic approaches</span> <span class="citation_source-journal">EMBO Rep.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">1640</span><span class="NLM_x">–</span> <span class="NLM_lpage">1655</span><span class="refDoi"> DOI: 10.15252/embr.201541116</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.15252%2Fembr.201541116" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=1640-1655&author=E.+Daguenetauthor=G.+Dujardinauthor=J.+Valcarcel&title=The+pathogenicity+of+splicing+defects%3A+mechanistic+insights+into+pre-mRNA+processing+inform+novel+therapeutic+approaches&doi=10.15252%2Fembr.201541116"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.15252%2Fembr.201541116&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.15252%252Fembr.201541116%26sid%3Dliteratum%253Aachs%26aulast%3DDaguenet%26aufirst%3DE.%26aulast%3DDujardin%26aufirst%3DG.%26aulast%3DValcarcel%26aufirst%3DJ.%26atitle%3DThe%2520pathogenicity%2520of%2520splicing%2520defects%253A%2520mechanistic%2520insights%2520into%2520pre-mRNA%2520processing%2520inform%2520novel%2520therapeutic%2520approaches%26jtitle%3DEMBO%2520Rep.%26date%3D2015%26volume%3D16%26spage%3D1640%26epage%3D1655%26doi%3D10.15252%2Fembr.201541116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Neubauer, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rappsilber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calvio, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ajuh, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sleeman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamond, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mann, M.</span><span> </span><span class="NLM_article-title">Mass spectrometry and EST-database searching allows characterization of the multi-protein spliceosome complex</span> <span class="citation_source-journal">Nat. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">46</span><span class="NLM_x">–</span> <span class="NLM_lpage">50</span><span class="refDoi"> DOI: 10.1038/1700</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1038%2F1700" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=9731529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADyaK1cXmtVOmtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=1998&pages=46-50&author=G.+Neubauerauthor=A.+Kingauthor=J.+Rappsilberauthor=C.+Calvioauthor=M.+Watsonauthor=P.+Ajuhauthor=J.+Sleemanauthor=A.+Lamondauthor=M.+Mann&title=Mass+spectrometry+and+EST-database+searching+allows+characterization+of+the+multi-protein+spliceosome+complex&doi=10.1038%2F1700"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Mass spectrometry and EST-database searching allows characterization of the multi-protein spliceosome complex</span></div><div class="casAuthors">Neubauer, Gitte; King, Angus; Rappsilber, Juri; Calvio, Cinzia; Watson, Mark; Ajuh, Paul; Sleeman, Judith; Lamond, Angus; Mann, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">46-50</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature America</span>)
        </div><div class="casAbstract">Many important cell mechanisms are carried out and regulated by multi-protein complexes, for example, transcription and RNA processing machinery, receptor complexes and cytoskeletal structures.  Most of these complexes remain only partially characterized due to the difficulty of conventional protein anal. methods.  The rapid expansion of DNA sequence databases now provides whole or partial gene sequences of model organisms, and recent advances in protein microcharacterization via mass spectrometry allow the possibility of linking these DNA sequences to the proteins in functional complexes.  This approach has been demonstrated in organisms whose genomes have been sequenced, such as budding yeast.  Here we report the first characterization of an entire mammalian multi-protein complex using these methods.  The machinery that removes introns from mRNA precursors-the spliceosome-is a large multi-protein complex.  Approx. half of the components excised from a two-dimensional gel sepn. of the spliceosome were found in protein sequence databases.  Using nanoelectrospray mass spectrometry, the remainder were identified and cloned using public expressed sequence tag (EST) databases.  Existing EST databases are thus already sufficiently complete to allow rapid characterization of large mammalian protein complexes via mass spectrometry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrm9v6Hl41AxrVg90H21EOLACvtfcHk0lgNLaTazdx_oQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmtVOmtLw%253D&md5=d67e8a9aea0e391d5495c7113626c501</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2F1700&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F1700%26sid%3Dliteratum%253Aachs%26aulast%3DNeubauer%26aufirst%3DG.%26aulast%3DKing%26aufirst%3DA.%26aulast%3DRappsilber%26aufirst%3DJ.%26aulast%3DCalvio%26aufirst%3DC.%26aulast%3DWatson%26aufirst%3DM.%26aulast%3DAjuh%26aufirst%3DP.%26aulast%3DSleeman%26aufirst%3DJ.%26aulast%3DLamond%26aufirst%3DA.%26aulast%3DMann%26aufirst%3DM.%26atitle%3DMass%2520spectrometry%2520and%2520EST-database%2520searching%2520allows%2520characterization%2520of%2520the%2520multi-protein%2520spliceosome%2520complex%26jtitle%3DNat.%2520Genet.%26date%3D1998%26volume%3D20%26spage%3D46%26epage%3D50%26doi%3D10.1038%2F1700" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Lallena, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chalmers, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Llamazares, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamond, A. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valcarcel, J.</span><span> </span><span class="NLM_article-title">Splicing regulation at the second catalytic step by Sex-lethal involves 3′ splice site recognition by SPF45</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">285</span><span class="NLM_x">–</span> <span class="NLM_lpage">296</span><span class="refDoi"> DOI: 10.1016/S0092-8674(02)00730-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1016%2FS0092-8674%2802%2900730-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2002&pages=285-296&author=M.+J.+Lallenaauthor=K.+J.+Chalmersauthor=S.+Llamazaresauthor=A.+I.+Lamondauthor=J.+Valcarcel&title=Splicing+regulation+at+the+second+catalytic+step+by+Sex-lethal+involves+3%E2%80%B2+splice+site+recognition+by+SPF45&doi=10.1016%2FS0092-8674%2802%2900730-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2802%2900730-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252802%252900730-4%26sid%3Dliteratum%253Aachs%26aulast%3DLallena%26aufirst%3DM.%2BJ.%26aulast%3DChalmers%26aufirst%3DK.%2BJ.%26aulast%3DLlamazares%26aufirst%3DS.%26aulast%3DLamond%26aufirst%3DA.%2BI.%26aulast%3DValcarcel%26aufirst%3DJ.%26atitle%3DSplicing%2520regulation%2520at%2520the%2520second%2520catalytic%2520step%2520by%2520Sex-lethal%2520involves%25203%25E2%2580%25B2%2520splice%2520site%2520recognition%2520by%2520SPF45%26jtitle%3DCell%26date%3D2002%26volume%3D109%26spage%3D285%26epage%3D296%26doi%3D10.1016%2FS0092-8674%2802%2900730-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Corsini, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonna, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basquin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hothorn, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheffzek, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valcarcel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sattler, M.</span><span> </span><span class="NLM_article-title">U2AF-homology motif interactions are required for alternative splicing regulation by SPF45</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">620</span><span class="NLM_x">–</span> <span class="NLM_lpage">629</span><span class="refDoi"> DOI: 10.1038/nsmb1260</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1038%2Fnsmb1260" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=17589525" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnt1Kgsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2007&pages=620-629&author=L.+Corsiniauthor=S.+Bonnaauthor=J.+Basquinauthor=M.+Hothornauthor=K.+Scheffzekauthor=J.+Valcarcelauthor=M.+Sattler&title=U2AF-homology+motif+interactions+are+required+for+alternative+splicing+regulation+by+SPF45&doi=10.1038%2Fnsmb1260"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">U2AF-homology motif interactions are required for alternative splicing regulation by SPF45</span></div><div class="casAuthors">Corsini, Lorenzo; Bonna, Sophie; Basquin, Jerome; Hothorn, Michael; Scheffzek, Klaus; Valcarcel, Juan; Sattler, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">620-629</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The U2AF-homol. motif (UHM) mediates protein-protein interactions between factors involved in constitutive RNA splicing.  Here we report that the splicing factor SPF45 regulates alternative splicing of the apoptosis regulatory gene FAS (also called CD95).  The SPF45 UHM is necessary for this activity and binds UHM-ligand motifs (ULMs) present in the 3' splice site-recognizing factors U2AF65, SF1 and SF3b155.  We describe a 2.1-Å crystal structure of SPF45-UHM in complex with a ULM peptide from SF3b155.  Features distinct from those of previously described UHM-ULM structures allowed the design of mutations in the SPF45 UHM that selectively impair binding to individual ULMs.  Splicing assays using the ULM-selective SPF45 variants demonstrate that individual UHM-ULM interactions are required for FAS splicing regulation by SPF45 in vivo.  Our data suggest that networks of UHM-ULM interactions are involved in regulating alternative splicing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9hirtJhwE6bVg90H21EOLACvtfcHk0lhAZG78V6GmWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnt1Kgsrk%253D&md5=f82647f494645f7e9bbb549491a2867a</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnsmb1260&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb1260%26sid%3Dliteratum%253Aachs%26aulast%3DCorsini%26aufirst%3DL.%26aulast%3DBonna%26aufirst%3DS.%26aulast%3DBasquin%26aufirst%3DJ.%26aulast%3DHothorn%26aufirst%3DM.%26aulast%3DScheffzek%26aufirst%3DK.%26aulast%3DValcarcel%26aufirst%3DJ.%26aulast%3DSattler%26aufirst%3DM.%26atitle%3DU2AF-homology%2520motif%2520interactions%2520are%2520required%2520for%2520alternative%2520splicing%2520regulation%2520by%2520SPF45%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2007%26volume%3D14%26spage%3D620%26epage%3D629%26doi%3D10.1038%2Fnsmb1260" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Izquierdo, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majos, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonnal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castelo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guigo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bilbao, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valcarcel, J.</span><span> </span><span class="NLM_article-title">Regulation of Fas alternative splicing by antagonistic effects of TIA-1 and PTB on exon definition</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">475</span><span class="NLM_x">–</span> <span class="NLM_lpage">484</span><span class="refDoi"> DOI: 10.1016/j.molcel.2005.06.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1016%2Fj.molcel.2005.06.015" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2005&pages=475-484&author=J.+M.+Izquierdoauthor=N.+Majosauthor=S.+Bonnalauthor=C.+Martinezauthor=R.+Casteloauthor=R.+Guigoauthor=D.+Bilbaoauthor=J.+Valcarcel&title=Regulation+of+Fas+alternative+splicing+by+antagonistic+effects+of+TIA-1+and+PTB+on+exon+definition&doi=10.1016%2Fj.molcel.2005.06.015"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2005.06.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2005.06.015%26sid%3Dliteratum%253Aachs%26aulast%3DIzquierdo%26aufirst%3DJ.%2BM.%26aulast%3DMajos%26aufirst%3DN.%26aulast%3DBonnal%26aufirst%3DS.%26aulast%3DMartinez%26aufirst%3DC.%26aulast%3DCastelo%26aufirst%3DR.%26aulast%3DGuigo%26aufirst%3DR.%26aulast%3DBilbao%26aufirst%3DD.%26aulast%3DValcarcel%26aufirst%3DJ.%26atitle%3DRegulation%2520of%2520Fas%2520alternative%2520splicing%2520by%2520antagonistic%2520effects%2520of%2520TIA-1%2520and%2520PTB%2520on%2520exon%2520definition%26jtitle%3DMol.%2520Cell%26date%3D2005%26volume%3D19%26spage%3D475%26epage%3D484%26doi%3D10.1016%2Fj.molcel.2005.06.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Sampath, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Long, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shepard, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xia, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Devanarayan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandusky, G. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, W. L.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dantzig, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williamson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rolfe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, R. E.</span><span> </span><span class="NLM_article-title">Human SPF45, a splicing factor, has limited expression in normal tissues, is overexpressed in many tumors, and can confer a multidrug-resistant phenotype to cells</span> <span class="citation_source-journal">Am. J. Pathol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">163</span><span class="NLM_x">, </span> <span class="NLM_fpage">1781</span><span class="NLM_x">–</span> <span class="NLM_lpage">1790</span><span class="refDoi"> DOI: 10.1016/S0002-9440(10)63538-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1016%2FS0002-9440%2810%2963538-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=14578179" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpsVWltL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=163&publication_year=2003&pages=1781-1790&author=J.+Sampathauthor=P.+R.+Longauthor=R.+L.+Shepardauthor=X.+Xiaauthor=V.+Devanarayanauthor=G.+E.+Sanduskyauthor=W.+L.+Perryauthor=A.+H.+Dantzigauthor=M.+Williamsonauthor=M.+Rolfeauthor=R.+E.+Moore&title=Human+SPF45%2C+a+splicing+factor%2C+has+limited+expression+in+normal+tissues%2C+is+overexpressed+in+many+tumors%2C+and+can+confer+a+multidrug-resistant+phenotype+to+cells&doi=10.1016%2FS0002-9440%2810%2963538-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Human SPF45, a splicing factor, has limited expression in normal tissues, is overexpressed in many tumors, and can confer a multidrug-resistant phenotype to cells</span></div><div class="casAuthors">Sampath, Janardhan; Long, Pandy R.; Shepard, Robert L.; Xia, Xiaoling; Devanarayan, Viswanath; Sandusky, George E.; Perry, William L., III; Dantzig, Anne H.; Williamson, Mark; Rolfe, Mark; Moore, Robert E.</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Pathology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">163</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1781-1790</span>CODEN:
                <span class="NLM_cas:coden">AJPAA4</span>;
        ISSN:<span class="NLM_cas:issn">0002-9440</span>.
    
            (<span class="NLM_cas:orgname">American Society for Investigative Pathology</span>)
        </div><div class="casAbstract">The authors' effort to identify novel drug-resistant genes in cyclophosphamide-resistant EMT6 mouse mammary tumors led the authors to the identification of SPF45.  Simultaneously, other groups identified SPF45 as a component of the spliceosome that is involved in alternative splicing.  The authors isolated the human homolog and examd. the normal human tissue expression, tumor expression, and the phenotype caused by overexpression of human SPF45.  The authors' analyses revealed that SPF45 is expressed in many, but not all, normal tissues tested with predominant expression in normal ductal epithelial cells of the breast, liver, pancreas, and prostate.  The authors' analyses using tissue microarrays and sausages of tumors indicated that SPF45 is highly expressed in numerous carcinomas including bladder, breast, colon, lung, ovarian, pancreatic, and prostate.  Interestingly, this study revealed that overexpression of SPF45 in HeLa, a cervical carcinoma cell line, resulted in drug resistance to doxorubicin and vincristine, two chemotherapeutic drugs commonly used in cancer.  To the authors' knowledge, this is the first study showing tumor overexpression of an alternate splicing factor resulting in drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQdUUfuDDLB7Vg90H21EOLACvtfcHk0liUc8QNadEO6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpsVWltL4%253D&md5=5e82f79953ec02c8c31da98afed038eb</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2FS0002-9440%2810%2963538-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0002-9440%252810%252963538-9%26sid%3Dliteratum%253Aachs%26aulast%3DSampath%26aufirst%3DJ.%26aulast%3DLong%26aufirst%3DP.%2BR.%26aulast%3DShepard%26aufirst%3DR.%2BL.%26aulast%3DXia%26aufirst%3DX.%26aulast%3DDevanarayan%26aufirst%3DV.%26aulast%3DSandusky%26aufirst%3DG.%2BE.%26aulast%3DPerry%26aufirst%3DW.%2BL.%26aulast%3DDantzig%26aufirst%3DA.%2BH.%26aulast%3DWilliamson%26aufirst%3DM.%26aulast%3DRolfe%26aufirst%3DM.%26aulast%3DMoore%26aufirst%3DR.%2BE.%26atitle%3DHuman%2520SPF45%252C%2520a%2520splicing%2520factor%252C%2520has%2520limited%2520expression%2520in%2520normal%2520tissues%252C%2520is%2520overexpressed%2520in%2520many%2520tumors%252C%2520and%2520can%2520confer%2520a%2520multidrug-resistant%2520phenotype%2520to%2520cells%26jtitle%3DAm.%2520J.%2520Pathol.%26date%3D2003%26volume%3D163%26spage%3D1781%26epage%3D1790%26doi%3D10.1016%2FS0002-9440%2810%2963538-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Perry, W. L.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shepard, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sampath, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yaden, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chin, W. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iversen, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lesoon, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Brien, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peek, V. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rolfe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shyjan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tighe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williamson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krishnan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dantzig, A. H.</span><span> </span><span class="NLM_article-title">Human splicing factor SPF45 (RBM17) confers broad multidrug resistance to anticancer drugs when overexpressed--a phenotype partially reversed by selective estrogen receptor modulators</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">6593</span><span class="NLM_x">–</span> <span class="NLM_lpage">6600</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-03-3675</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1158%2F0008-5472.CAN-03-3675" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2005&pages=6593-6600&author=W.+L.+Perryauthor=R.+L.+Shepardauthor=J.+Sampathauthor=B.+Yadenauthor=W.+W.+Chinauthor=P.+W.+Iversenauthor=S.+Jinauthor=A.+Lesoonauthor=K.+A.+O%E2%80%99Brienauthor=V.+L.+Peekauthor=M.+Rolfeauthor=A.+Shyjanauthor=M.+Tigheauthor=M.+Williamsonauthor=V.+Krishnanauthor=R.+E.+Mooreauthor=A.+H.+Dantzig&title=Human+splicing+factor+SPF45+%28RBM17%29+confers+broad+multidrug+resistance+to+anticancer+drugs+when+overexpressed%2D%2Da+phenotype+partially+reversed+by+selective+estrogen+receptor+modulators&doi=10.1158%2F0008-5472.CAN-03-3675"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-03-3675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-03-3675%26sid%3Dliteratum%253Aachs%26aulast%3DPerry%26aufirst%3DW.%2BL.%26aulast%3DShepard%26aufirst%3DR.%2BL.%26aulast%3DSampath%26aufirst%3DJ.%26aulast%3DYaden%26aufirst%3DB.%26aulast%3DChin%26aufirst%3DW.%2BW.%26aulast%3DIversen%26aufirst%3DP.%2BW.%26aulast%3DJin%26aufirst%3DS.%26aulast%3DLesoon%26aufirst%3DA.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DK.%2BA.%26aulast%3DPeek%26aufirst%3DV.%2BL.%26aulast%3DRolfe%26aufirst%3DM.%26aulast%3DShyjan%26aufirst%3DA.%26aulast%3DTighe%26aufirst%3DM.%26aulast%3DWilliamson%26aufirst%3DM.%26aulast%3DKrishnan%26aufirst%3DV.%26aulast%3DMoore%26aufirst%3DR.%2BE.%26aulast%3DDantzig%26aufirst%3DA.%2BH.%26atitle%3DHuman%2520splicing%2520factor%2520SPF45%2520%2528RBM17%2529%2520confers%2520broad%2520multidrug%2520resistance%2520to%2520anticancer%2520drugs%2520when%2520overexpressed--a%2520phenotype%2520partially%2520reversed%2520by%2520selective%2520estrogen%2520receptor%2520modulators%26jtitle%3DCancer%2520Res.%26date%3D2005%26volume%3D65%26spage%3D6593%26epage%3D6600%26doi%3D10.1158%2F0008-5472.CAN-03-3675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Kielkopf, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodionova, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burley, S. K.</span><span> </span><span class="NLM_article-title">A novel peptide recognition mode revealed by the X-ray structure of a core U2AF35/U2AF65 heterodimer</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">595</span><span class="NLM_x">–</span> <span class="NLM_lpage">605</span><span class="refDoi"> DOI: 10.1016/S0092-8674(01)00480-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1016%2FS0092-8674%2801%2900480-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2001&pages=595-605&author=C.+L.+Kielkopfauthor=N.+A.+Rodionovaauthor=M.+R.+Greenauthor=S.+K.+Burley&title=A+novel+peptide+recognition+mode+revealed+by+the+X-ray+structure+of+a+core+U2AF35%2FU2AF65+heterodimer&doi=10.1016%2FS0092-8674%2801%2900480-9"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2801%2900480-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252801%252900480-9%26sid%3Dliteratum%253Aachs%26aulast%3DKielkopf%26aufirst%3DC.%2BL.%26aulast%3DRodionova%26aufirst%3DN.%2BA.%26aulast%3DGreen%26aufirst%3DM.%2BR.%26aulast%3DBurley%26aufirst%3DS.%2BK.%26atitle%3DA%2520novel%2520peptide%2520recognition%2520mode%2520revealed%2520by%2520the%2520X-ray%2520structure%2520of%2520a%2520core%2520U2AF35%252FU2AF65%2520heterodimer%26jtitle%3DCell%26date%3D2001%26volume%3D106%26spage%3D595%26epage%3D605%26doi%3D10.1016%2FS0092-8674%2801%2900480-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Selenko, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gregorovic, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sprangers, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stier, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rhani, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kramer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sattler, M.</span><span> </span><span class="NLM_article-title">Structural basis for the molecular recognition between human splicing factors U2AF65 and SF1/mBBP</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">965</span><span class="NLM_x">–</span> <span class="NLM_lpage">976</span><span class="refDoi"> DOI: 10.1016/S1097-2765(03)00115-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1016%2FS1097-2765%2803%2900115-1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2003&pages=965-976&author=P.+Selenkoauthor=G.+Gregorovicauthor=R.+Sprangersauthor=G.+Stierauthor=Z.+Rhaniauthor=A.+Kramerauthor=M.+Sattler&title=Structural+basis+for+the+molecular+recognition+between+human+splicing+factors+U2AF65+and+SF1%2FmBBP&doi=10.1016%2FS1097-2765%2803%2900115-1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2803%2900115-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252803%252900115-1%26sid%3Dliteratum%253Aachs%26aulast%3DSelenko%26aufirst%3DP.%26aulast%3DGregorovic%26aufirst%3DG.%26aulast%3DSprangers%26aufirst%3DR.%26aulast%3DStier%26aufirst%3DG.%26aulast%3DRhani%26aufirst%3DZ.%26aulast%3DKramer%26aufirst%3DA.%26aulast%3DSattler%26aufirst%3DM.%26atitle%3DStructural%2520basis%2520for%2520the%2520molecular%2520recognition%2520between%2520human%2520splicing%2520factors%2520U2AF65%2520and%2520SF1%252FmBBP%26jtitle%3DMol.%2520Cell%26date%3D2003%26volume%3D11%26spage%3D965%26epage%3D976%26doi%3D10.1016%2FS1097-2765%2803%2900115-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Kielkopf, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lucke, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, M. R.</span><span> </span><span class="NLM_article-title">U2AF homology motifs: protein recognition in the RRM world</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">1513</span><span class="NLM_x">–</span> <span class="NLM_lpage">1526</span><span class="refDoi"> DOI: 10.1101/gad.1206204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1101%2Fgad.1206204" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2004&pages=1513-1526&author=C.+L.+Kielkopfauthor=S.+Luckeauthor=M.+R.+Green&title=U2AF+homology+motifs%3A+protein+recognition+in+the+RRM+world&doi=10.1101%2Fgad.1206204"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1101%2Fgad.1206204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1206204%26sid%3Dliteratum%253Aachs%26aulast%3DKielkopf%26aufirst%3DC.%2BL.%26aulast%3DLucke%26aufirst%3DS.%26aulast%3DGreen%26aufirst%3DM.%2BR.%26atitle%3DU2AF%2520homology%2520motifs%253A%2520protein%2520recognition%2520in%2520the%2520RRM%2520world%26jtitle%3DGenes%2520Dev.%26date%3D2004%26volume%3D18%26spage%3D1513%26epage%3D1526%26doi%3D10.1101%2Fgad.1206204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Manceau, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kielkopf, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sobel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maucuer, A.</span><span> </span><span class="NLM_article-title">Different requirements of the kinase and UHM domains of KIS for its nuclear localization and binding to splicing factors</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">381</span><span class="NLM_x">, </span> <span class="NLM_fpage">748</span><span class="NLM_x">–</span> <span class="NLM_lpage">762</span><span class="refDoi"> DOI: 10.1016/j.jmb.2008.06.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1016%2Fj.jmb.2008.06.026" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=381&publication_year=2008&pages=748-762&author=V.+Manceauauthor=C.+L.+Kielkopfauthor=A.+Sobelauthor=A.+Maucuer&title=Different+requirements+of+the+kinase+and+UHM+domains+of+KIS+for+its+nuclear+localization+and+binding+to+splicing+factors&doi=10.1016%2Fj.jmb.2008.06.026"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2008.06.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2008.06.026%26sid%3Dliteratum%253Aachs%26aulast%3DManceau%26aufirst%3DV.%26aulast%3DKielkopf%26aufirst%3DC.%2BL.%26aulast%3DSobel%26aufirst%3DA.%26aulast%3DMaucuer%26aufirst%3DA.%26atitle%3DDifferent%2520requirements%2520of%2520the%2520kinase%2520and%2520UHM%2520domains%2520of%2520KIS%2520for%2520its%2520nuclear%2520localization%2520and%2520binding%2520to%2520splicing%2520factors%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2008%26volume%3D381%26spage%3D748%26epage%3D762%26doi%3D10.1016%2Fj.jmb.2008.06.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Maucuer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ozon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manceau, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gavet, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lawler, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curmi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sobel, A.</span><span> </span><span class="NLM_article-title">KIS is a protein kinase with an RNA recognition motif</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">272</span><span class="NLM_x">, </span> <span class="NLM_fpage">23151</span><span class="NLM_x">–</span> <span class="NLM_lpage">23156</span><span class="refDoi"> DOI: 10.1074/jbc.272.37.23151</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1074%2Fjbc.272.37.23151" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=272&publication_year=1997&pages=23151-23156&author=A.+Maucuerauthor=S.+Ozonauthor=V.+Manceauauthor=O.+Gavetauthor=S.+Lawlerauthor=P.+Curmiauthor=A.+Sobel&title=KIS+is+a+protein+kinase+with+an+RNA+recognition+motif&doi=10.1074%2Fjbc.272.37.23151"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1074%2Fjbc.272.37.23151&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.272.37.23151%26sid%3Dliteratum%253Aachs%26aulast%3DMaucuer%26aufirst%3DA.%26aulast%3DOzon%26aufirst%3DS.%26aulast%3DManceau%26aufirst%3DV.%26aulast%3DGavet%26aufirst%3DO.%26aulast%3DLawler%26aufirst%3DS.%26aulast%3DCurmi%26aufirst%3DP.%26aulast%3DSobel%26aufirst%3DA.%26atitle%3DKIS%2520is%2520a%2520protein%2520kinase%2520with%2520an%2520RNA%2520recognition%2520motif%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1997%26volume%3D272%26spage%3D23151%26epage%3D23156%26doi%3D10.1074%2Fjbc.272.37.23151" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Corsini, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hothorn, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stier, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rybin, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheffzek, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gibson, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sattler, M.</span><span> </span><span class="NLM_article-title">Dimerization and protein binding specificity of the U2AF homology motif of the splicing factor Puf60</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">630</span><span class="NLM_x">–</span> <span class="NLM_lpage">639</span><span class="refDoi"> DOI: 10.1074/jbc.M805395200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1074%2Fjbc.M805395200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=630-639&author=L.+Corsiniauthor=M.+Hothornauthor=G.+Stierauthor=V.+Rybinauthor=K.+Scheffzekauthor=T.+J.+Gibsonauthor=M.+Sattler&title=Dimerization+and+protein+binding+specificity+of+the+U2AF+homology+motif+of+the+splicing+factor+Puf60&doi=10.1074%2Fjbc.M805395200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M805395200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M805395200%26sid%3Dliteratum%253Aachs%26aulast%3DCorsini%26aufirst%3DL.%26aulast%3DHothorn%26aufirst%3DM.%26aulast%3DStier%26aufirst%3DG.%26aulast%3DRybin%26aufirst%3DV.%26aulast%3DScheffzek%26aufirst%3DK.%26aulast%3DGibson%26aufirst%3DT.%2BJ.%26aulast%3DSattler%26aufirst%3DM.%26atitle%3DDimerization%2520and%2520protein%2520binding%2520specificity%2520of%2520the%2520U2AF%2520homology%2520motif%2520of%2520the%2520splicing%2520factor%2520Puf60%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D630%26epage%3D639%26doi%3D10.1074%2Fjbc.M805395200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Loerch, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maucuer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manceau, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kielkopf, C. L.</span><span> </span><span class="NLM_article-title">Cancer-relevant splicing factor CAPERalpha engages the essential splicing factor SF3b155 in a specific ternary complex</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">289</span><span class="NLM_x">, </span> <span class="NLM_fpage">17325</span><span class="NLM_x">–</span> <span class="NLM_lpage">17337</span><span class="refDoi"> DOI: 10.1074/jbc.M114.558825</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1074%2Fjbc.M114.558825" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=17325-17337&author=S.+Loerchauthor=A.+Maucuerauthor=V.+Manceauauthor=M.+R.+Greenauthor=C.+L.+Kielkopf&title=Cancer-relevant+splicing+factor+CAPERalpha+engages+the+essential+splicing+factor+SF3b155+in+a+specific+ternary+complex&doi=10.1074%2Fjbc.M114.558825"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.558825&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.558825%26sid%3Dliteratum%253Aachs%26aulast%3DLoerch%26aufirst%3DS.%26aulast%3DMaucuer%26aufirst%3DA.%26aulast%3DManceau%26aufirst%3DV.%26aulast%3DGreen%26aufirst%3DM.%2BR.%26aulast%3DKielkopf%26aufirst%3DC.%2BL.%26atitle%3DCancer-relevant%2520splicing%2520factor%2520CAPERalpha%2520engages%2520the%2520essential%2520splicing%2520factor%2520SF3b155%2520in%2520a%2520specific%2520ternary%2520complex%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D17325%26epage%3D17337%26doi%3D10.1074%2Fjbc.M114.558825" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Spadaccini, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reidt, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dybkov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Will, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frank, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stier, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corsini, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wahl, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luhrmann, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sattler, M.</span><span> </span><span class="NLM_article-title">Biochemical and NMR analyses of an SF3b155-p14-U2AF-RNA interaction network involved in branch point definition during pre-mRNA splicing</span> <span class="citation_source-journal">RNA</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">410</span><span class="NLM_x">–</span> <span class="NLM_lpage">425</span><span class="refDoi"> DOI: 10.1261/rna.2271406</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1261%2Frna.2271406" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2006&pages=410-425&author=R.+Spadacciniauthor=U.+Reidtauthor=O.+Dybkovauthor=C.+Willauthor=R.+Frankauthor=G.+Stierauthor=L.+Corsiniauthor=M.+C.+Wahlauthor=R.+Luhrmannauthor=M.+Sattler&title=Biochemical+and+NMR+analyses+of+an+SF3b155-p14-U2AF-RNA+interaction+network+involved+in+branch+point+definition+during+pre-mRNA+splicing&doi=10.1261%2Frna.2271406"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1261%2Frna.2271406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1261%252Frna.2271406%26sid%3Dliteratum%253Aachs%26aulast%3DSpadaccini%26aufirst%3DR.%26aulast%3DReidt%26aufirst%3DU.%26aulast%3DDybkov%26aufirst%3DO.%26aulast%3DWill%26aufirst%3DC.%26aulast%3DFrank%26aufirst%3DR.%26aulast%3DStier%26aufirst%3DG.%26aulast%3DCorsini%26aufirst%3DL.%26aulast%3DWahl%26aufirst%3DM.%2BC.%26aulast%3DLuhrmann%26aufirst%3DR.%26aulast%3DSattler%26aufirst%3DM.%26atitle%3DBiochemical%2520and%2520NMR%2520analyses%2520of%2520an%2520SF3b155-p14-U2AF-RNA%2520interaction%2520network%2520involved%2520in%2520branch%2520point%2520definition%2520during%2520pre-mRNA%2520splicing%26jtitle%3DRNA%26date%3D2006%26volume%3D12%26spage%3D410%26epage%3D425%26doi%3D10.1261%2Frna.2271406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Thickman, K. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swenson, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kabogo, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gryczynski, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kielkopf, C. L.</span><span> </span><span class="NLM_article-title">Multiple U2AF65 binding sites within SF3b155: thermodynamic and spectroscopic characterization of protein-protein interactions among pre-mRNA splicing factors</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">356</span><span class="NLM_x">, </span> <span class="NLM_fpage">664</span><span class="NLM_x">–</span> <span class="NLM_lpage">683</span><span class="refDoi"> DOI: 10.1016/j.jmb.2005.11.067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1016%2Fj.jmb.2005.11.067" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=356&publication_year=2006&pages=664-683&author=K.+R.+Thickmanauthor=M.+C.+Swensonauthor=J.+M.+Kabogoauthor=Z.+Gryczynskiauthor=C.+L.+Kielkopf&title=Multiple+U2AF65+binding+sites+within+SF3b155%3A+thermodynamic+and+spectroscopic+characterization+of+protein-protein+interactions+among+pre-mRNA+splicing+factors&doi=10.1016%2Fj.jmb.2005.11.067"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2005.11.067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2005.11.067%26sid%3Dliteratum%253Aachs%26aulast%3DThickman%26aufirst%3DK.%2BR.%26aulast%3DSwenson%26aufirst%3DM.%2BC.%26aulast%3DKabogo%26aufirst%3DJ.%2BM.%26aulast%3DGryczynski%26aufirst%3DZ.%26aulast%3DKielkopf%26aufirst%3DC.%2BL.%26atitle%3DMultiple%2520U2AF65%2520binding%2520sites%2520within%2520SF3b155%253A%2520thermodynamic%2520and%2520spectroscopic%2520characterization%2520of%2520protein-protein%2520interactions%2520among%2520pre-mRNA%2520splicing%2520factors%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2006%26volume%3D356%26spage%3D664%26epage%3D683%26doi%3D10.1016%2Fj.jmb.2005.11.067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Chandler, D.</span><span> </span><span class="NLM_article-title">Interfaces and the driving force of hydrophobic assembly</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">437</span><span class="NLM_x">, </span> <span class="NLM_fpage">640</span><span class="NLM_x">–</span> <span class="NLM_lpage">647</span><span class="refDoi"> DOI: 10.1038/nature04162</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1038%2Fnature04162" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=16193038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVCjsLzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=640-647&author=D.+Chandler&title=Interfaces+and+the+driving+force+of+hydrophobic+assembly&doi=10.1038%2Fnature04162"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Interfaces and the driving force of hydrophobic assembly</span></div><div class="casAuthors">Chandler, David</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">437</span>
        (<span class="NLM_cas:issue">7059</span>),
    <span class="NLM_cas:pages">640-647</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The hydrophobic effect - the tendency for oil and water to segregate - is important in diverse phenomena, from the cleaning of laundry, to the creation of micro-emulsions to make new materials, to the assembly of proteins into functional complexes.  This effect is multifaceted depending on whether hydrophobic mols. are individually hydrated or driven to assemble into larger structures.  Despite the basic principles underlying the hydrophobic effect being qual. well understood, only recently have theor. developments begun to explain and quantify many features of this ubiquitous phenomenon.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVPZjVicyZaLVg90H21EOLACvtfcHk0lhLxENnl6fu6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVCjsLzM&md5=58e58a6f3dfaf11a9a158308bdcacbae</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnature04162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature04162%26sid%3Dliteratum%253Aachs%26aulast%3DChandler%26aufirst%3DD.%26atitle%3DInterfaces%2520and%2520the%2520driving%2520force%2520of%2520hydrophobic%2520assembly%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D640%26epage%3D647%26doi%3D10.1038%2Fnature04162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Guth, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaur, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valcarcel, J.</span><span> </span><span class="NLM_article-title">Evidence for substrate-specific requirement of the splicing factor U2AF(35) and for its function after polypyrimidine tract recognition by U2AF(65)</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">8263</span><span class="NLM_x">–</span> <span class="NLM_lpage">8271</span><span class="refDoi"> DOI: 10.1128/MCB.19.12.8263</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1128%2FMCB.19.12.8263" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADyaK1MXns1ygtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1999&pages=8263-8271&author=S.+Guthauthor=C.+Martinezauthor=R.+K.+Gaurauthor=J.+Valcarcel&title=Evidence+for+substrate-specific+requirement+of+the+splicing+factor+U2AF%2835%29+and+for+its+function+after+polypyrimidine+tract+recognition+by+U2AF%2865%29&doi=10.1128%2FMCB.19.12.8263"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Evidence for substrate-specific requirement of the splicing factor U2AF35 and for its function after polypyrimidine tract recognition by U2AF65</span></div><div class="casAuthors">Guth, Sabine; Martinez, Concepcion; Gaur, Rajesh K.; Valcarcel, Juan</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">8263-8271</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">U2 snRNP auxiliary factor (U2AF) promotes U2 snRNP binding to pre-mRNAs and consists of two subunits of 65 and 35 kDa, U2AF65 and U2AF35.  U2AF65 binds to the polypyrimidine (Py) tract upstream from the 3' splice site and plays a key role in assisting U2 snRNP recruitment.  It has been proposed that U2AF35 facilitates U2AF65 binding through a network of protein-protein interactions with other splicing factors, but the requirement and function of U2AF35 remain controversial.  Here we show that recombinant U2AF65 is sufficient to activate the splicing of two constitutively spliced pre-mRNAs in exts. that were chromatog. depleted of U2AF.  In contrast, U2AF65, U2AF35, and the interaction between them are required for splicing of an Ig μ pre-RNA contg. an intron with a weak Py tract and a purine-rich exonic splicing enhancer.  Remarkably, splicing activation by U2AF35 occurs without changes in U2AF65 crosslinking to the Py tract.  These results reveal substrate-specific requirements for U2AF35 and a novel function for this factor in pre-mRNA splicing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpS5oTjnj5vaLVg90H21EOLACvtfcHk0libfwt5-YwImA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXns1ygtL0%253D&md5=2d38e0308e8e6ad8757e950a09aa0211</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1128%2FMCB.19.12.8263&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.19.12.8263%26sid%3Dliteratum%253Aachs%26aulast%3DGuth%26aufirst%3DS.%26aulast%3DMartinez%26aufirst%3DC.%26aulast%3DGaur%26aufirst%3DR.%2BK.%26aulast%3DValcarcel%26aufirst%3DJ.%26atitle%3DEvidence%2520for%2520substrate-specific%2520requirement%2520of%2520the%2520splicing%2520factor%2520U2AF%252835%2529%2520and%2520for%2520its%2520function%2520after%2520polypyrimidine%2520tract%2520recognition%2520by%2520U2AF%252865%2529%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1999%26volume%3D19%26spage%3D8263%26epage%3D8271%26doi%3D10.1128%2FMCB.19.12.8263" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Gozani, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Potashkin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reed, R.</span><span> </span><span class="NLM_article-title">A potential role for U2AF-SAP 155 interactions in recruiting U2 snRNP to the branch site</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">4752</span><span class="NLM_x">–</span> <span class="NLM_lpage">4760</span><span class="refDoi"> DOI: 10.1128/MCB.18.8.4752</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1128%2FMCB.18.8.4752" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=9671485" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADyaK1cXkvVartLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1998&pages=4752-4760&author=O.+Gozaniauthor=J.+Potashkinauthor=R.+Reed&title=A+potential+role+for+U2AF-SAP+155+interactions+in+recruiting+U2+snRNP+to+the+branch+site&doi=10.1128%2FMCB.18.8.4752"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">A potential role for U2AF-SAP 155 interactions in recruiting U2 snRNP to the branch site</span></div><div class="casAuthors">Gozani, Or; Potashkin, Judith; Reed, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4752-4760</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Base pairing between U2 snRNA and the branchpoint sequence (BPS) is essential for pre-mRNA splicing.  Because the metazoan BPS is short and highly degenerate, this interaction alone is insufficient for specific binding of U2 snRNP.  The splicing factor U2AF binds to the pyrimidine tract at the 3' splice site in the earliest spliceosomal complex, E, and is essential for U2 snRNP binding in the spliceosomal complex A.  The authors show that the U2 snRNP protein SAP 155 UV cross-links to pre-mRNA on both sides of the BPS in the A complex.  SAP 155's downstream crosslinking site is immediately adjacent to the U2AF binding site, and the two proteins interact directly in protein-protein interaction assays.  Using UV crosslinking, together with functional analyses of pre-mRNAs contg. duplicated BPSs, the authors show a direct correlation between BPS selection and UV crosslinking of SAP 155 on both sides of the BPS.  Together, these data are consistent with a model in which U2AF binds to the pyrimidine tract in the E complex and then interacts with SAP 155 to recruit U2 snRNP to the BPS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWlODMFtuI5LVg90H21EOLACvtfcHk0libfwt5-YwImA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXkvVartLo%253D&md5=8db87cfb4812b22757eba14138a7502e</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1128%2FMCB.18.8.4752&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.18.8.4752%26sid%3Dliteratum%253Aachs%26aulast%3DGozani%26aufirst%3DO.%26aulast%3DPotashkin%26aufirst%3DJ.%26aulast%3DReed%26aufirst%3DR.%26atitle%3DA%2520potential%2520role%2520for%2520U2AF-SAP%2520155%2520interactions%2520in%2520recruiting%2520U2%2520snRNP%2520to%2520the%2520branch%2520site%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1998%26volume%3D18%26spage%3D4752%26epage%3D4760%26doi%3D10.1128%2FMCB.18.8.4752" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Das, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reed, R.</span><span> </span><span class="NLM_article-title">Functional association of U2 snRNP with the ATP-independent spliceosomal complex E</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">779</span><span class="NLM_x">–</span> <span class="NLM_lpage">787</span><span class="refDoi"> DOI: 10.1016/S1097-2765(00)80318-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1016%2FS1097-2765%2800%2980318-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2000&pages=779-787&author=R.+Dasauthor=Z.+Zhouauthor=R.+Reed&title=Functional+association+of+U2+snRNP+with+the+ATP-independent+spliceosomal+complex+E&doi=10.1016%2FS1097-2765%2800%2980318-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2800%2980318-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252800%252980318-4%26sid%3Dliteratum%253Aachs%26aulast%3DDas%26aufirst%3DR.%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DReed%26aufirst%3DR.%26atitle%3DFunctional%2520association%2520of%2520U2%2520snRNP%2520with%2520the%2520ATP-independent%2520spliceosomal%2520complex%2520E%26jtitle%3DMol.%2520Cell%26date%3D2000%26volume%3D5%26spage%3D779%26epage%3D787%26doi%3D10.1016%2FS1097-2765%2800%2980318-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Banerjee, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rahn, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gawande, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guth, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valcarcel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, R.</span><span> </span><span class="NLM_article-title">The conserved RNA recognition motif 3 of U2 snRNA auxiliary factor (U2AF 65) is essential in vivo but dispensable for activity in vitro</span> <span class="citation_source-journal">RNA</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">253</span><span class="refDoi"> DOI: 10.1261/rna.5153204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1261%2Frna.5153204" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2004&pages=240-253&author=H.+Banerjeeauthor=A.+Rahnauthor=B.+Gawandeauthor=S.+Guthauthor=J.+Valcarcelauthor=R.+Singh&title=The+conserved+RNA+recognition+motif+3+of+U2+snRNA+auxiliary+factor+%28U2AF+65%29+is+essential+in+vivo+but+dispensable+for+activity+in+vitro&doi=10.1261%2Frna.5153204"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1261%2Frna.5153204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1261%252Frna.5153204%26sid%3Dliteratum%253Aachs%26aulast%3DBanerjee%26aufirst%3DH.%26aulast%3DRahn%26aufirst%3DA.%26aulast%3DGawande%26aufirst%3DB.%26aulast%3DGuth%26aufirst%3DS.%26aulast%3DValcarcel%26aufirst%3DJ.%26aulast%3DSingh%26aufirst%3DR.%26atitle%3DThe%2520conserved%2520RNA%2520recognition%2520motif%25203%2520of%2520U2%2520snRNA%2520auxiliary%2520factor%2520%2528U2AF%252065%2529%2520is%2520essential%2520in%2520vivo%2520but%2520dispensable%2520for%2520activity%2520in%2520vitro%26jtitle%3DRNA%26date%3D2004%26volume%3D10%26spage%3D240%26epage%3D253%26doi%3D10.1261%2Frna.5153204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Zaharieva, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chipman, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soller, M.</span><span> </span><span class="NLM_article-title">Alternative splicing interference by xenobiotics</span> <span class="citation_source-journal">Toxicology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">296</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">12</span><span class="refDoi"> DOI: 10.1016/j.tox.2012.01.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1016%2Fj.tox.2012.01.014" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=296&publication_year=2012&pages=1-12&author=E.+Zaharievaauthor=J.+K.+Chipmanauthor=M.+Soller&title=Alternative+splicing+interference+by+xenobiotics&doi=10.1016%2Fj.tox.2012.01.014"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.tox.2012.01.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tox.2012.01.014%26sid%3Dliteratum%253Aachs%26aulast%3DZaharieva%26aufirst%3DE.%26aulast%3DChipman%26aufirst%3DJ.%2BK.%26aulast%3DSoller%26aufirst%3DM.%26atitle%3DAlternative%2520splicing%2520interference%2520by%2520xenobiotics%26jtitle%3DToxicology%26date%3D2012%26volume%3D296%26spage%3D1%26epage%3D12%26doi%3D10.1016%2Fj.tox.2012.01.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">Xds</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">132</span><span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=Xds&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0ljtlkcVE_JU8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXds%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storoni, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title">Phaser crystallographic software</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">–</span> <span class="NLM_lpage">674</span><span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0ljtlkcVE_JU8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">255</span><span class="refDoi"> DOI: 10.1107/S0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2FS0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. σA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial α-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lj_xA_gPIxmBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2FS0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballard, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keegan, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krissinel, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leslie, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNicholas, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pannu, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Potterton, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, K. S.</span><span> </span><span class="NLM_article-title">Overview of the CCP4 suite and current developments</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span><span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+suite+and+current+developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lj_xA_gPIxmBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Delaglio, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grzesiek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vuister, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pfeifer, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bax, A.</span><span> </span><span class="NLM_article-title">NMRPipe: a multidimensional spectral processing system based on UNIX pipes</span> <span class="citation_source-journal">J. Biomol. NMR</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">277</span><span class="NLM_x">–</span> <span class="NLM_lpage">293</span><span class="refDoi"> DOI: 10.1007/BF00197809</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1007%2FBF00197809" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=8520220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADyaK2MXhtVSmurfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1995&pages=277-293&author=F.+Delaglioauthor=S.+Grzesiekauthor=G.+W.+Vuisterauthor=G.+Zhuauthor=J.+Pfeiferauthor=A.+Bax&title=NMRPipe%3A+a+multidimensional+spectral+processing+system+based+on+UNIX+pipes&doi=10.1007%2FBF00197809"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">NMRPipe: a multidimensional spectral processing system based on UNIX pipes</span></div><div class="casAuthors">Delaglio, Frank; Grzesiek, Stephan; Vuister, Geerten W.; Zhu, Guang; Pfeifer, John; Bax, Ad</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular NMR</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">277-93</span>CODEN:
                <span class="NLM_cas:coden">JBNME9</span>;
        ISSN:<span class="NLM_cas:issn">0925-2738</span>.
    
            (<span class="NLM_cas:orgname">ESCOM</span>)
        </div><div class="casAbstract">The NMRPipe system is a UNIX software environment of processing, graphics, and anal. tools designed to meet current routine and research-oriented multidimensional processing requirements, and to anticipate and accommodate future demands and developments.  The system is based on UNIX pipes, which allow programs running simultaneously to exchange streams of data under user control.  In an NMRPipe processing scheme, a stream of spectral data flows through a pipeline of processing programs, each of which performs one component of the overall scheme, such as Fourier transformation or linear prediction.  Complete multidimensional processing schemes are constructed as simple UNIX shell scripts.  The processing modules themselves maintain and exploit accurate records of data sizes, detection modes, and calibration information in all dimensions, so that schemes can be constructed without the need to explicitly define or anticipate data sizes or storage details of real and imaginary channels during processing.  The asynchronous pipeline scheme provides other substantial advantages, including high flexibility, favorable processing speeds, choice of both all-in-memory and disk-bound processing, easy adaptation to different data formats, simpler software development and maintenance, and the ability to distribute processing tasks on multi-CPU computers and computer networks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSIUdXrdcEgbVg90H21EOLACvtfcHk0ligrWhmdTKPYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXhtVSmurfK&md5=a670fca5b164083e2178fafd2fb951ff</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1007%2FBF00197809&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00197809%26sid%3Dliteratum%253Aachs%26aulast%3DDelaglio%26aufirst%3DF.%26aulast%3DGrzesiek%26aufirst%3DS.%26aulast%3DVuister%26aufirst%3DG.%2BW.%26aulast%3DZhu%26aufirst%3DG.%26aulast%3DPfeifer%26aufirst%3DJ.%26aulast%3DBax%26aufirst%3DA.%26atitle%3DNMRPipe%253A%2520a%2520multidimensional%2520spectral%2520processing%2520system%2520based%2520on%2520UNIX%2520pipes%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D1995%26volume%3D6%26spage%3D277%26epage%3D293%26doi%3D10.1007%2FBF00197809" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Vranken, W. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boucher, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevens, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fogh, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pajon, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Llinas, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ulrich, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Markley, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ionides, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laue, E. D.</span><span> </span><span class="NLM_article-title">The CCPN data model for NMR spectroscopy: development of a software pipeline</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">687</span><span class="NLM_x">–</span> <span class="NLM_lpage">696</span><span class="refDoi"> DOI: 10.1002/prot.20449</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1002%2Fprot.20449" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=15815974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADC%252BD2MXksVKmt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2005&pages=687-696&author=W.+F.+Vrankenauthor=W.+Boucherauthor=T.+J.+Stevensauthor=R.+H.+Foghauthor=A.+Pajonauthor=M.+Llinasauthor=E.+L.+Ulrichauthor=J.+L.+Markleyauthor=J.+Ionidesauthor=E.+D.+Laue&title=The+CCPN+data+model+for+NMR+spectroscopy%3A+development+of+a+software+pipeline&doi=10.1002%2Fprot.20449"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The CCPN data model for NMR spectroscopy: development of a software pipeline</span></div><div class="casAuthors">Vranken, Wim F.; Boucher, Wayne; Stevens, Tim J.; Fogh, Rasmus H.; Pajon, Anne; Llinas, Miguel; Ulrich, Eldon L.; Markley, John L.; Ionides, John; Laue, Ernest D.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">687-696</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">To address data management and data exchange problems in the NMR community, the Collaborative Computing Project for the NMR community (CCPN) created a "Data Model" that describes all the different types of information needed in an NMR structural study, from mol. structure and NMR parameters to coordinates.  This paper describes the development of a set of software applications that use the Data Model and its assocd. libraries, thus validating the approach.  These applications are freely available and provide a pipeline for high-throughput anal. of NMR data.  Three programs work directly with the Data Model: CcpNmr Anal., an entirely new anal. and interactive display program, the CcpNmr FormatConverter, which allows transfer of data from programs commonly used in NMR to and from the Data Model, and the CLOUDS software for automated structure calcn. and assignment (Carnegie Mellon University), which was rewritten to interact directly with the Data Model.  The ARIA 2.0 software for structure calcn. (Institut Pasteur) and the QUEEN program for validation of restraints (University of Nijmegen) were extended to provide conversion of their data to the Data Model.  During these developments the Data Model has been thoroughly tested and used, demonstrating that applications can successfully exchange data via the Data Model.  The software architecture developed by CCPN is now ready for new developments, such as integration with addnl. software applications and extensions of the Data Model into other areas of research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpl6jPIIOYWbVg90H21EOLACvtfcHk0ligrWhmdTKPYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXksVKmt7c%253D&md5=b07b9701f64f281d58d178f9c2033596</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fprot.20449&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.20449%26sid%3Dliteratum%253Aachs%26aulast%3DVranken%26aufirst%3DW.%2BF.%26aulast%3DBoucher%26aufirst%3DW.%26aulast%3DStevens%26aufirst%3DT.%2BJ.%26aulast%3DFogh%26aufirst%3DR.%2BH.%26aulast%3DPajon%26aufirst%3DA.%26aulast%3DLlinas%26aufirst%3DM.%26aulast%3DUlrich%26aufirst%3DE.%2BL.%26aulast%3DMarkley%26aufirst%3DJ.%2BL.%26aulast%3DIonides%26aufirst%3DJ.%26aulast%3DLaue%26aufirst%3DE.%2BD.%26atitle%3DThe%2520CCPN%2520data%2520model%2520for%2520NMR%2520spectroscopy%253A%2520development%2520of%2520a%2520software%2520pipeline%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2005%26volume%3D59%26spage%3D687%26epage%3D696%26doi%3D10.1002%2Fprot.20449" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Dignam, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lebovitz, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roeder, R. G.</span><span> </span><span class="NLM_article-title">Accurate transcription initiation by RNA polymerase II in a soluble extract from isolated mammalian nuclei</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1983</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1475</span><span class="NLM_x">–</span> <span class="NLM_lpage">1489</span><span class="refDoi"> DOI: 10.1093/nar/11.5.1475</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1093%2Fnar%2F11.5.1475" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=6828386" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADyaL3sXhslCit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1983&pages=1475-1489&author=J.+D.+Dignamauthor=R.+M.+Lebovitzauthor=R.+G.+Roeder&title=Accurate+transcription+initiation+by+RNA+polymerase+II+in+a+soluble+extract+from+isolated+mammalian+nuclei&doi=10.1093%2Fnar%2F11.5.1475"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate transcription initiation by RNA polymerase II in a soluble extract from isolated mammalian nuclei</span></div><div class="casAuthors">Dignam, John David; Lebovitz, Russell M.; Roeder, Robert G.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1475-89</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    </div><div class="casAbstract">A new procedure was developed for prepg. exts. from nuclei of human tissue culture cells (HeLa cells) that directs accurate transcription initiation in vitro from promoters of class II by RNA polymerase II  [9014-24-8].  Conditions of extn. and assay were optimized for max. activity by using the major late promoter of adenovirus 2.  The ext. also directs accurate transcription initiation from other adenovirus promoters and cellular promoters.  The ext. also directs accurate transcription initiation from class III promoters (tRNA and adenovirus 2 VA).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpacSBbgqUIyrVg90H21EOLACvtfcHk0lgF4ExKYy37jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXhslCit74%253D&md5=9a1031cfc1503d0179d166d9efe68ac5</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F11.5.1475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F11.5.1475%26sid%3Dliteratum%253Aachs%26aulast%3DDignam%26aufirst%3DJ.%2BD.%26aulast%3DLebovitz%26aufirst%3DR.%2BM.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26atitle%3DAccurate%2520transcription%2520initiation%2520by%2520RNA%2520polymerase%2520II%2520in%2520a%2520soluble%2520extract%2520from%2520isolated%2520mammalian%2520nuclei%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1983%26volume%3D11%26spage%3D1475%26epage%3D1489%26doi%3D10.1093%2Fnar%2F11.5.1475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Zillmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zapp, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berget, S. M.</span><span> </span><span class="NLM_article-title">Gel electrophoretic isolation of splicing complexes containing U1 small nuclear ribonucleoprotein particles</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">814</span><span class="NLM_x">–</span> <span class="NLM_lpage">821</span><span class="refDoi"> DOI: 10.1128/MCB.8.2.814</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1128%2FMCB.8.2.814" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=2832738" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADyaL1cXhtV2isr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1988&pages=814-821&author=M.+Zillmannauthor=M.+L.+Zappauthor=S.+M.+Berget&title=Gel+electrophoretic+isolation+of+splicing+complexes+containing+U1+small+nuclear+ribonucleoprotein+particles&doi=10.1128%2FMCB.8.2.814"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Gel electrophoretic isolation of splicing complexes containing U1 small nuclear ribonucleoprotein particles</span></div><div class="casAuthors">Zillmann, Martin; Zapp, Maria L.; Berget, Susan M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">814-21</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    </div><div class="casAbstract">Assembly of splicing precursor RNAs into ribonucleoprotein particle (RNP) complexes during incubation in in vitro splicing exts. was monitored by a system of RNP gel electrophoresis.  The temporal pattern of assembly obsd. by the system was identical to that obtained by other gel and gradient methodologies.  In contrast to the results obtained by other systems; however, requirements of U1 small nuclear RNPs (snRNPs) and 5' splice junction sequences were obsd. for formation of specific complexes and retention of U1 snRNPs within gel-fractionated complexes.  Single-intron substrate RNAs rapidly assembled into slow-migration complexes.  The 1st specific complex (A) appeared within a minute of incubation and required ATP, 5' and 3' precursor RNA consensus sequences, and intact U1 and I1 RNAs for formation.  A 2nd complex (B) contg. precursor RNA appeared after 15 min of incubation.  Lariat-exon 2 and exon 1 intermediates first appeared in this complex, operationally defining it as the active spliceosome.  U4 RNA was required for appearance of complex B.  Released lariat 1st appeared in a complex of intermediate mobility (A') and subsequently in rapidly migrating diffuse complexes.  Ligated product RNA was obsd. only in fast-migrating complexes.  U1 snRNPs were detected as components of gel-isolated complexes.  Radiolabeled RNA within the A and B complexes were immunopptd. by U1-specific antibodies under gel-loading conditions and from gel-isolated complexes.  Therefore, the RNP antigen remained assocd. with assembled complexes during gel electrophoresis.  In addn., 5' splice junction sequences within gel-isolated A and B complexes were inaccessible to RNase H cleavage in the presence of a complementary oligonucleotide.  Therefore, nuclear factors that bind 5' splice junctions also remained assocd. with 5' splice junctions under these gel conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6jWKP8lpIy7Vg90H21EOLACvtfcHk0lgF4ExKYy37jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXhtV2isr8%253D&md5=0f865109f6601b5b8694cff408827500</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1128%2FMCB.8.2.814&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.8.2.814%26sid%3Dliteratum%253Aachs%26aulast%3DZillmann%26aufirst%3DM.%26aulast%3DZapp%26aufirst%3DM.%2BL.%26aulast%3DBerget%26aufirst%3DS.%2BM.%26atitle%3DGel%2520electrophoretic%2520isolation%2520of%2520splicing%2520complexes%2520containing%2520U1%2520small%2520nuclear%2520ribonucleoprotein%2520particles%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1988%26volume%3D8%26spage%3D814%26epage%3D821%26doi%3D10.1128%2FMCB.8.2.814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Das, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reed, R.</span><span> </span><span class="NLM_article-title">Resolution of the mammalian E complex and the ATP-dependent spliceosomal complexes on native agarose mini-gels</span> <span class="citation_source-journal">RNA</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">1504</span><span class="NLM_x">–</span> <span class="NLM_lpage">1508</span><span class="refDoi"> DOI: 10.1017/S1355838299991501</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1017%2FS1355838299991501" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1999&pages=1504-1508&author=R.+Dasauthor=R.+Reed&title=Resolution+of+the+mammalian+E+complex+and+the+ATP-dependent+spliceosomal+complexes+on+native+agarose+mini-gels&doi=10.1017%2FS1355838299991501"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1017%2FS1355838299991501&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS1355838299991501%26sid%3Dliteratum%253Aachs%26aulast%3DDas%26aufirst%3DR.%26aulast%3DReed%26aufirst%3DR.%26atitle%3DResolution%2520of%2520the%2520mammalian%2520E%2520complex%2520and%2520the%2520ATP-dependent%2520spliceosomal%2520complexes%2520on%2520native%2520agarose%2520mini-gels%26jtitle%3DRNA%26date%3D1999%26volume%3D5%26spage%3D1504%26epage%3D1508%26doi%3D10.1017%2FS1355838299991501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Heckmann, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kessler, H.</span><span> </span><span class="NLM_article-title">Design and chemical synthesis of integrin ligands</span> <span class="citation_source-journal">Methods Enzymol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">426</span><span class="NLM_x">, </span> <span class="NLM_fpage">463</span><span class="NLM_x">–</span> <span class="NLM_lpage">503</span><span class="refDoi"> DOI: 10.1016/S0076-6879(07)26020-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=10.1016%2FS0076-6879%2807%2926020-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=17697896" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFOit7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=426&publication_year=2007&pages=463-503&author=D.+Heckmannauthor=H.+Kessler&title=Design+and+chemical+synthesis+of+integrin+ligands&doi=10.1016%2FS0076-6879%2807%2926020-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Design and chemical synthesis of integrin ligands</span></div><div class="casAuthors">Heckmann, Dominik; Kessler, Horst</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">426</span>
        (<span class="NLM_cas:issue">Integrins</span>),
    <span class="NLM_cas:pages">463-503</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The design and synthesis of peptidic and nonpeptidic integrin ligands derived from the most abundant natural tripeptide sequence, RGD, are described in this article.  Special emphasis is placed on the activity and selectivity of the ligands to integrin subtypes.  Two approaches are described - ligand- and structure-oriented design.  When no structure of the complex or the target is known, one may derive suitable starting points from natural peptide sequences, which often require conformational restriction for a further optimization.  A "spatial screening" procedure was used to identify highly active and selective ligands for the integrin subtypes αvβ3 and αIIbβ3.  Structure-based methods require knowledge of the binding domain of the target.  Hence, the first structure of the αvβ3 integrin with bound cilengitide was a landmark for the structure-based approach.  Meanwhile, a design using homol. models of other integrin subtypes has also been successfully applied.  To improve the ADME profile, nonpeptidic ligands have been developed using the information of the spatial distances and orientations of the most important pharmacophoric groups (esp. the carboxyl group and the basic moiety at the other end of the mol.).  Applications of the αvβ3 ligands as drugs in antiangiogenic tumor therapy for mol. imaging of metastases and for improvement of biocompatibility of grafts are briefly described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHsE0gLE-XLbVg90H21EOLACvtfcHk0lg56OdEHnbCFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFOit7s%253D&md5=9adbfa1684b2f5f33997c5c686314a64</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2807%2926020-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252807%252926020-3%26sid%3Dliteratum%253Aachs%26aulast%3DHeckmann%26aufirst%3DD.%26aulast%3DKessler%26aufirst%3DH.%26atitle%3DDesign%2520and%2520chemical%2520synthesis%2520of%2520integrin%2520ligands%26jtitle%3DMethods%2520Enzymol.%26date%3D2007%26volume%3D426%26spage%3D463%26epage%3D503%26doi%3D10.1016%2FS0076-6879%2807%2926020-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2PEH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2PEH','PDB','2PEH'); return false;">PDB: 2PEH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2PE8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2PE8','PDB','2PE8'); return false;">PDB: 2PE8</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LSO" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LSO','PDB','5LSO'); return false;">PDB: 5LSO</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i22"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01118">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_01138"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.6b01118">10.1021/acs.jmedchem.6b01118</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Dissociation constants, X-ray crystallography statistics, analytical data for synthesized peptides, sequence alignment, structures, electron density, and splicing data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01118/suppl_file/jm6b01118_si_001.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01118/suppl_file/jm6b01118_si_001.pdf">jm6b01118_si_001.pdf (905.38 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates and structure factors for the SPF45 UHM domain in complex with cyclic peptide <b>3</b> have been deposited in the PDB with accession number <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5LSO">5LSO</a>.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.6b01118&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-22%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.6b01118%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.6b01118" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679993b2ca933c10","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
