
<html lang="en"     class="pb-page"  data-request-id="d8b0e881-9cb2-4d16-a20e-95e30e1e7213"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;issue:issue:10.1021/jmcmar.2018.61.issue-3;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.7b01654;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Species-Selective Pyrimidineamine Inhibitors of Trypanosoma brucei S-Adenosylmethionine Decarboxylase" /></meta><meta name="dc.Creator" content="Oleg A.  Volkov" /></meta><meta name="dc.Creator" content="Anthony J.  Brockway" /></meta><meta name="dc.Creator" content="Stephen A.  Wring" /></meta><meta name="dc.Creator" content="Michael  Peel" /></meta><meta name="dc.Creator" content="Zhe  Chen" /></meta><meta name="dc.Creator" content="Margaret A.  Phillips" /></meta><meta name="dc.Creator" content="Jef K.  De Brabander" /></meta><meta name="dc.Description" content="New therapeutic options are needed for treatment of human African trypanosomiasis (HAT) caused by protozoan parasite Trypanosoma brucei. S-Adenosylmethionine decarboxylase (AdoMetDC) is an essentia..." /></meta><meta name="Description" content="New therapeutic options are needed for treatment of human African trypanosomiasis (HAT) caused by protozoan parasite Trypanosoma brucei. S-Adenosylmethionine decarboxylase (AdoMetDC) is an essentia..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="January 5, 2018" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b01654" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01654" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b01654" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b01654" /></link>
        
    
    

<title>Species-Selective Pyrimidineamine Inhibitors of Trypanosoma brucei S-Adenosylmethionine Decarboxylase | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b01654" /></meta><meta property="og:title" content="Species-Selective Pyrimidineamine Inhibitors of Trypanosoma brucei S-Adenosylmethionine Decarboxylase" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/large/jm-2017-01654m_0013.jpeg" /></meta><meta property="og:description" content="New therapeutic options are needed for treatment of human African trypanosomiasis (HAT) caused by protozoan parasite Trypanosoma brucei. S-Adenosylmethionine decarboxylase (AdoMetDC) is an essential enzyme in the polyamine pathway of T. brucei. Previous attempts to target this enzyme were thwarted by the lack of brain penetration of the most advanced series. Herein, we describe a T. brucei AdoMetDC inhibitor series based on a pyrimidineamine pharmacophore that we identified by target-based high-throughput screening. The pyrimidineamines showed selectivity for T. brucei AdoMetDC over the human enzyme, inhibited parasite growth in whole-cell assay, and had good predicted blood–brain barrier penetration. The medicinal chemistry program elucidated structure–activity relationships within the series. Features of the series that were required for binding were revealed by determining the X-ray crystal structure of TbAdoMetDC bound to one analog. The pyrimidineamine series provides a novel starting point for an anti-HAT lead optimization." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b01654"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b01654">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b01654&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b01654&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b01654&amp;href=/doi/10.1021/acs.jmedchem.7b01654" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 3</span><span class="cit-fg-pageRange">, 1182-1203</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/3" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b01470" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.7b01674" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Species-Selective Pyrimidineamine Inhibitors of <i>Trypanosoma brucei S</i>-Adenosylmethionine Decarboxylase</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Oleg+A.++Volkov">Oleg A. Volkov</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Anthony+J.++Brockway">Anthony J. Brockway</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol "><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Stephen+A.++Wring">Stephen A. Wring</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Michael++Peel">Michael Peel</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Zhe++Chen">Zhe Chen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Margaret+A.++Phillips">Margaret A. Phillips</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0001-5250-5578" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jef+K.++De+Brabander">Jef K. De Brabander</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0001-8816-5798" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text">Departments of <sup>†</sup>Pharmacology, <sup>‡</sup>Biochemistry, and <sup>§</sup>Biophysics, University of Texas Southwestern Medical Center, 5323 Harry Hines Boulevard, Dallas, Texas 75390-9038, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">∥</span> <span class="aff-text">Scynexis, Inc. (now Avista Pharma Solutions), 3501 Tricenter Boulevard, Suite C, Durham, North Carolina 27713, United States</span></div><div class="corresp-info"><strong>*</strong>M.A.P.: e-mail, <a href="/cdn-cgi/l/email-protection#206d415247415245540e7048494c4c495053607574734f5554485745535445524e0e454455"><span class="__cf_email__" data-cfemail="f7ba96859096859283d9a79f9e9b9b9e8784b7a2a3a49882839f80928483928599d9929382">[email protected]</span></a>; phone, (214) 645-6164.</div><div class="corresp-info"><strong>*</strong>J.K.D.B.: e-mail, <a href="/cdn-cgi/l/email-protection#84cee1e2aac0e1c6f6e5e6e5eae0e1f6c4d1d0d7ebf1f0ecf3e1f7f0e1f6eaaae1e0f1"><span class="__cf_email__" data-cfemail="e6ac8380c8a283a49487848788828394a6b3b2b58993928e91839592839488c8838293">[email protected]</span></a>; phone, (214) 648-7808.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01654&amp;href=/doi/10.1021%2Facs.jmedchem.7b01654" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 3</span><span class="cit-pageRange">, 1182–1203</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">December 22, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>9 November 2017</li><li><span class="item_label"><b>Published</b> online</span>5 January 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 8 February 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b01654" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01654</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1182%26pageCount%3D22%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DOleg%2BA.%2BVolkov%252C%2BAnthony%2BJ.%2BBrockway%252C%2BStephen%2BA.%2BWring%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D3%26contentID%3Dacs.jmedchem.7b01654%26title%3DSpecies-Selective%2BPyrimidineamine%2BInhibitors%2Bof%2BTrypanosoma%2Bbrucei%2BS-Adenosylmethionine%2BDecarboxylase%26numPages%3D22%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D1203%26publicationDate%3DFebruary%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b01654"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">968</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">7</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b01654" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Species-Selective Pyrimidineamine Inhibitors of Trypanosoma brucei S-Adenosylmethionine Decarboxylase&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Oleg&quot;,&quot;last_name&quot;:&quot;A. Volkov&quot;},{&quot;first_name&quot;:&quot;Anthony&quot;,&quot;last_name&quot;:&quot;J. Brockway&quot;},{&quot;first_name&quot;:&quot;Stephen&quot;,&quot;last_name&quot;:&quot;A. Wring&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;Peel&quot;},{&quot;first_name&quot;:&quot;Zhe&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Margaret&quot;,&quot;last_name&quot;:&quot;A. Phillips&quot;},{&quot;first_name&quot;:&quot;Jef&quot;,&quot;last_name&quot;:&quot;K. De Brabander&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;05&quot;,&quot;issue&quot;:&quot;3&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;1182-1203&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b01654&quot;},&quot;abstract&quot;:&quot;New therapeutic options are needed for treatment of human African trypanosomiasis (HAT) caused by protozoan parasite Trypanosoma brucei. S-Adenosylmethionine decarboxylase (AdoMetDC) is an essential enzyme in the polyamine pathway of T. brucei. Previous attempts to target this enzyme were thwarted by the lack of brain penetration of the most advanced series. Herein, we describe a T. brucei AdoMetDC inhibitor series based on a pyrimidineamine pharmacophore that we identified by target-based high-throughput screening. The pyrimidineamines showed selectivity for T. brucei AdoMetDC over the human enzyme, inhibited parasite growth in whole-cell assay, and had good predicted blood–brain barrier penetration. The medicinal chemistry program elucidated structure–activity relationships within the series. Features of the series that were required for binding were revealed by determining the X-ray crystal structure of TbAdoMetDC bound to one analog. The pyrimidineamine series provides a novel starting point for an anti-H&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01654&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01654" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01654&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01654" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b01654&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01654" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b01654&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b01654&amp;href=/doi/10.1021/acs.jmedchem.7b01654" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b01654" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b01654" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01654&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b01654%26sid%3Dliteratum%253Aachs%26pmid%3D29271204%26genre%3Darticle%26aulast%3DVolkov%26date%3D2018%26atitle%3DSpecies-Selective%2BPyrimidineamine%2BInhibitors%2Bof%2BTrypanosoma%2Bbrucei%2BS-Adenosylmethionine%2BDecarboxylase%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D3%26spage%3D1182%26epage%3D1203%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292200" title="Layers">Layers</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/3" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/jmcmar.2018.61.issue-3/20180208/jmcmar.2018.61.issue-3.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/medium/jm-2017-01654m_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/large/jm-2017-01654m_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01654&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">New therapeutic options are needed for treatment of human African trypanosomiasis (HAT) caused by protozoan parasite <i>Trypanosoma brucei</i>. <i>S</i>-Adenosylmethionine decarboxylase (AdoMetDC) is an essential enzyme in the polyamine pathway of <i>T. brucei</i>. Previous attempts to target this enzyme were thwarted by the lack of brain penetration of the most advanced series. Herein, we describe a <i>T. brucei</i> AdoMetDC inhibitor series based on a pyrimidineamine pharmacophore that we identified by target-based high-throughput screening. The pyrimidineamines showed selectivity for <i>T. brucei</i> AdoMetDC over the human enzyme, inhibited parasite growth in whole-cell assay, and had good predicted blood–brain barrier penetration. The medicinal chemistry program elucidated structure–activity relationships within the series. Features of the series that were required for binding were revealed by determining the X-ray crystal structure of <i>Tb</i>AdoMetDC bound to one analog. The pyrimidineamine series provides a novel starting point for an anti-HAT lead optimization.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82458" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82458" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Human African trypanosomiasis (HAT), also known as sleeping sickness, is caused by the single celled protozoal parasite <i>Trypanosoma brucei</i> and is classified by the WHO as a neglected tropical disease (NTD).<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> HAT is transmitted by the tsetse fly to humans and animals where it replicates extracellularly in the blood and lymph causing fever and influenza-like symptoms. Later in infection, the parasite crosses the blood–brain barrier (BBB) and invades the central nervous system (CNS) leading to a variety of neurological symptoms, including disruption of the sleep–wake cycle, coma, and eventual death in most patients.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> The number of reported cases has declined steadily since peaking in 1998 at nearly 40 000 cases a year.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3, 4)</a> Control and surveillance programs, which include both vector control and identification and treatment of infected individuals, have been credited with this decline.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> The WHO has targeted HAT for sustainable elimination by 2030;<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> however, the recent discovery that some infected individuals are asymptomatic carriers threatens the elimination program.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Additionally, it is now recognized that skin and fat serves as a significant reservoir for the parasites, and thus screening of blood underestimates the magnitude of the parasite burden in the endemic communities.<a onclick="showRef(event, 'ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9">(7-9)</a></div><div class="NLM_p">Currently approved HAT treatments have significant liabilities hindering efforts to control the disease.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Particularly problematic are treatments for the CNS stage of the disease. Nifurtimox–eflornithine combination therapy (NECT) has replaced the poorly tolerated melarsoprol as a front-line therapy against Gambian form of the infection.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> However, both the high cost of this therapy<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> and its limited efficacy against Rhodesian form of HAT have led to continued use of melarsoprol.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Thus, it is recognized that a safer and less expensive therapy is needed that would cure both early- and CNS-stage infections caused by either <i>T. brucei</i> subspecies, eliminating the need to stage the disease. There are currently only two drug candidates in clinical development: oxaborole SCYX-7158<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> and nitroheterocyclic fexinidazole.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Both have the potential to meet the outlined treatment goals, but it is too early to know if either will make it to registration.</div><div class="NLM_p">Of the approved drugs only eflornithine has a well understood mechanism of action. It targets ornithine decarboxylase, a key enzyme in polyamine biosynthesis, suggesting that other enzymes in the pathway would also provide potential targets for drug discovery.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14, 15)</a> A second key enzyme in the pathway, <i>S</i>-adenosylmethionine decarboxylase (AdoMetDC), has been shown to be essential by genetic studies,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> and small-molecule inhibitors of AdoMetDC have been reported that show in vivo activity against <i>T. brucei</i> in mouse models.<a onclick="showRef(event, 'ref14 ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref14 ref17 ref18 ref19">(14, 17-19)</a> However, <i>T. brucei</i> AdoMetDC inhibitors, such as CGP 40215, mechanism-based MDL 73811, and its derivative Genz-644131 (<b>1</b>), were deemed unsuitable for anti-HAT development as their physicochemical properties were not consistent with good blood–brain barrier (BBB) penetration, and thus they were not effective for CNS stage of the disease.<a onclick="showRef(event, 'ref14 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref14 ref17 ref18">(14, 17, 18)</a> AdoMetDCs from trypanosomatids have a novel subunit configuration that differentiates them from the human enzyme: they are allosterically activated by heterodimerization with an inactive paralogous pseudoenzyme that we termed Prozyme.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Structural analysis by X-ray crystallography demonstrated that heterodimerization leads to displacement of an autoinhibitory sequence and to a coupled structural reorganization that stabilizes the active conformation through insertion of the N-terminus into the heterodimer interface.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> These structural differences suggested that selective inhibition of the parasite enzyme over the host enzyme was plausible.</div><div class="NLM_p last">We recently described a mass spectrometry based high-throughput screening (HTS) campaign that identified 13 classes of novel small-molecule inhibitors of <i>T. brucei</i> AdoMetDC that also inhibited parasite growth.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Several of the identified series demonstrated high propensity to cross the BBB and were at least partially on-target in the parasite cells.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Herein, we characterize a pyrimidineamine chemotype that was identified as a hit in the HTS. Initially, the hit failed to demonstrate substantial AdoMetDC inhibition in validation studies and thus was not previously reported. We later discovered that the potency of the initial hit compound was pH-dependent, triggering reprioritization of the pyrimidineamines for a medicinal chemistry effort to establish the structure–activity relationships (SARs) within the series. A number of pyrimidineamine analogs with low-micromolar activity on both the enzyme and the parasites were identified. Compounds in the series also showed good selectivity versus the human enzyme and were predicted to cross the blood–brain barrier. Finally, a crystal structure of <i>Tb</i>AdoMetDC bound to one analog was determined, providing insight into the structural basis for the pH dependence of inhibitor binding to <i>T. brucei</i> AdoMetDC and paving the way for future lead development. This work has led to the validation of the pyrimidineamines as a hit series that could be taken forward into a hit-to-lead optimization program for the treatment of HAT.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85241" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85241" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2-1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Synthesis</h3><div class="NLM_p">Compound <b>7</b> was purchased from Maybridge (part of Thermo Fisher Scientific), and compounds <b>56</b>–<b>59</b>, <b>74</b>–<b>88</b>, and <b>90</b> were purchased from ChemBridge (San Diego, CA, USA). The synthesis of all other analogs is shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>.</div><figure id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/medium/jm-2017-01654m_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/large/jm-2017-01654m_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Pyrimidineamines<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/large/jm-2017-01654m_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01654&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p><span class="fn-label">Scheme a</span>Reagents and conditions: (a) <b>2a</b>, 3,5-Cl<sub>2</sub>PhNH<sub>2</sub>, DMSO, 100 °C, 18 h (<b>8</b>); (b) <b>2a</b>–<b>c</b>,<b>f</b>–<b>h</b> or <b>3g</b>, aniline (3,5-Cl<sub>2</sub>PhNH<sub>2</sub> or 3-Br-5-ClPhNH<sub>2</sub> or <b>6</b>), conc HCl, EtOH, microwave (140 °C), 1.5 h (<b>9</b>, <b>10</b>, <b>15</b>, <b>16</b>, <b>26</b>, <b>32</b>–<b>34</b>, <b>36</b>, <b>37</b>, <b>44</b>, <b>45</b>, <b>53</b>, <b>54</b>, <b>61</b>–<b>68</b>, <b>72</b>, <b>73</b>, <b>89</b>); (c) <b>2a</b>,<b>d</b>,<b>e</b>,<b>h</b> or <b>3d</b>–<b>f</b>,<b>h</b>, aniline (3,5-Cl<sub>2</sub>PhNH<sub>2</sub> or 3-Br-5-ClPhNH<sub>2</sub> or <b>5a</b> or <b>6</b>) conc HCl, EtOH, reflux, 18 h (<b>13</b>, <b>14</b>, <b>23</b>–<b>25</b>, <b>27</b>, <b>28</b>, <b>38</b>–<b>43</b>, <b>46</b>–<b>48</b>, <b>52</b>, <b>55</b>, <b>60</b>, <b>69</b>–<b>71</b>); (d) <b>7</b> or <b>13</b> or <b>32</b> or <b>38</b>, ArylB(OH)<sub>2</sub>, Pd(OAc)<sub>2</sub>, PPh<sub>3</sub>, Na<sub>2</sub>CO<sub>3</sub>, H<sub>2</sub>O/DME, 90 °C, 18 h (<b>11</b>, <b>35</b>, <b>50</b>, <b>91</b>–<b>109</b>, <b>116</b>–<b>118</b>); (e) <b>13,</b> DIPEA, morpholine, CH<sub>3</sub>CN, microwave (140 °C), 2 h (<b>12</b>); (f) 3,5-Cl<sub>2</sub>PhNH<sub>2</sub>, <b>3a</b>,<b>c</b>,<b>i</b>,<b>j</b>, conc HCl, BuOH, microwave (185 °C), 4 h (<b>18</b>–<b>20</b>, <b>22</b>); (g) 3,5-Cl<sub>2</sub>PhNH<sub>2</sub>, <b>3b</b>, conc HCl, MeOH/H<sub>2</sub>O (1/1), sealed tube, 100 °C, 18 h (<b>21</b>); (h) <b>4</b>, 3,5-Cl<sub>2</sub>PhNH<sub>2</sub>, cat. Pd<sub>2</sub>(dba)<sub>3</sub>, BINAP, NaO<sup><i>t</i></sup>Bu, toluene, 70 °C, 18 h; then 3 N aq NaOH, EtOH, reflux, 18 h (<b>17</b>); (i) <b>2h</b>, <b>5b</b>, Et<sub>3</sub>N, DMF, rt, 24 h (<b>29</b>); (j) <b>2a</b>, <b>5c</b>,<b>d</b>, DIPEA, CH<sub>3</sub>CN, microwave (140 °C), 2 h (<b>30</b>, <b>31</b>); (k) <b>48</b>, conc H<sub>2</sub>SO<sub>4</sub>, MeOH, 4 Å molecular sieves, reflux, 18 h (<b>49</b>); (l) <b>48</b>, 3,5-Cl<sub>2</sub>PhNH<sub>2</sub>, EDI, py/DCM/DMF (0.4/1/1), rt, 18 h (<b>51</b>); (m) <b>32</b>, heteroarylboronic acid MIDA ester, cat. Pd(OAc)<sub>2</sub>, PPh<sub>3</sub>, sodium triphosphate, dioxane/H<sub>2</sub>O) (4/1), 90 °C, 18 h (<b>110</b>–<b>115</b>).</p>.</p></figure><div class="NLM_p last">Compounds <b>9</b>, <b>10</b>, <b>13</b>–<b>16</b>, <b>18</b>–<b>28</b>, <b>32</b>–<b>34</b>, <b>36</b>–<b>48</b>, <b>52</b>–<b>55</b>, <b>60</b>–<b>73</b>, and <b>89</b> were readily available via reaction of the appropriate chloropyrimidine (<b>2a</b>–<b>h</b>, <b>3d</b>–<b>h</b>) or chloropyridine (<b>3a</b>–<b>c</b>, <b>3i</b>–<b>j</b>) with a variety of substituted anilines in the presence of acidic ethanol or butanol under conventional<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> or microwave heating.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Compound <b>8</b> was obtained similarly but under neutral conditions via heating of chloropyrimidine <b>2a</b> with 3,5-Cl<sub>2</sub>PhNH<sub>2</sub> in DMSO at 100 °C, whereas amine-linker analogs <b>30</b> and <b>31</b> resulted from microwave heating of chloropyrimidine <b>2h</b> with amines <b>5c</b>,<b>d</b> in acetonitrile in the presence of <sup><i>i</i></sup>Pr<sub>2</sub>NEt.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Stirring thiol <b>5b</b> with chloropyrimidine <b>2h</b> and triethylamine in DMF at room temperature yielded thioether-linked analog <b>29</b>.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Analog <b>12</b> was obtained from chloride displacement in <b>13</b> with morpholine in the presence of <sup><i>i</i></sup>Pr<sub>2</sub>NEt under microwave heating in acetonitrile.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> The bis-ester and amide analogs <b>49</b> and <b>50</b> resulted from Fisher esterification and EDI-mediated amide coupling of <b>48</b>, respectively. Pyridine analog <b>17</b> was obtained via palladium-catalyzed amination of pyridinyl bromide <b>4</b> with 3,5-Cl<sub>2</sub>PhNH<sub>2</sub> followed by base-mediated hydrolysis of the acetamide.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Finally, all biaryl-containing compounds (<b>11</b>, <b>35</b>, <b>50</b>, <b>91</b>–<b>118</b>) were prepared via Suzuki cross-coupling of the corresponding aryl chloride (<b>7</b>, <b>13</b>) or aryl bromide (<b>32</b>, <b>38</b>) analogs with arylboronic acids<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> or the corresponding MIDA boronates.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a></div></div><div id="sec2-2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Discovery of Pyrimidineamines as <i>T. brucei</i> AdoMetDC Inhibitors</h3><div class="NLM_p">The pyrimidineamine <b>7</b> was identified as a singleton in our previously reported HTS campaign to identify inhibitors of <i>Tb</i>AdoMetDC (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A).<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Compound <b>7</b> did not comprise substructural motifs characteristic of pan-assay interference compounds (PAINS),<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> according to our analysis performed as described in ref <a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">22</a>. The compound was repurchased and tested for inhibition of <i>T. brucei</i> cell growth in vitro demonstrating a half-maximal effective growth-inhibitory concentration (EC<sub>50</sub>) of 5.6 μM after 48 h treatment. However, retesting of the <i>Tb</i>AdoMetDC enzyme activity yielded a half-maximal inhibitory concentration (IC<sub>50</sub>) of 30 μM (assay pH 7.7). Thus, the compound was not presented in the original HTS publication, where the IC<sub>50</sub> threshold for an active hit was set at 25 μM.</div><figure id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/medium/jm-2017-01654m_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/large/jm-2017-01654m_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Pyrimidineamines are pH-dependent <i>Tb</i>AdoMetDC inhibitors. (A) Structure of the protonated form of pyrimidineamine <b>7</b> (N1 is predicted to have p<i>K</i><sub>a</sub> of 7.3) and structure of mechanism-based inhibitor Genz-644131 (<b>1</b>). (B) IC<sub>50</sub> of pyrimidineamines plotted as a function of the assay buffer pH and fitted with linear regression analysis in Prism (GraphPad). Predicted p<i>K</i><sub>a</sub>(N1) values are shown in parentheses. The slopes for pyrimidineamines with p<i>K</i><sub>a</sub>(N1) = 7.3 (solid lines) are compared with those for analogs with p<i>K</i><sub>a</sub>(N1) > 9 (dashed lines) and with that for a non-pyrimidineamine control (dotted line).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/large/jm-2017-01654m_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01654&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Later, characterization of the pH dependence of the <i>Tb</i>AdoMetDC inhibition profile showed that there was a 60-fold increase in inhibitory activity of <b>7</b> at pH 6.8 compared with pH 7.7 (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>B; IC<sub>50</sub> = 30, 3.7, and 0.50 μM at pH 7.7, 7.2, and 6.8, respectively). The p<i>K</i><sub>a</sub> value of the conjugated acid of the aromatic nitrogen in position 1 (further denoted as p<i>K</i><sub>a</sub>(N1)) was estimated using MarvinSketch (version 16.3.28.0, ChemAxon, Budapest, Hungary) to be 7.3 (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A). Thus, we hypothesized that protonation of N1 in <b>7</b> was important for binding to the <i>T. brucei</i> AdoMetDC. Pyrimidineamine analogs (described below) with predicted p<i>K</i><sub>a</sub>(N1) of 7.3 had similar slopes of IC<sub>50</sub>–pH dependence, while analogs with p<i>K</i><sub>a</sub>(N1) > 9 had much shallower slopes reflecting more comparable percentages of protonated species among the three pH points (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>B). IC<sub>50</sub> values for the structurally unrelated <b>1</b> tested as a control were not appreciably pH-dependent, as only a 2-fold difference was observed between potencies at pH 6.8 and pH 7.7 (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>B).</div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/medium/jm-2017-01654m_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/large/jm-2017-01654m_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Mechanism of inhibition studies for <b>7</b>. (A) Evaluation of reversibility. Recovery of enzyme activity after a 30 min incubation with inhibitor followed by a rapid 100-fold dilution of the inhibitor. The progression of the reaction was measured as product formation using RapidFire–MS analysis. (B) Determination of the inhibition modality. Double-reciprocal Lineweaver–Burk plot of substrate titrations in the presence of four inhibitor concentrations shows competitive inhibition.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/large/jm-2017-01654m_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01654&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Encouragingly, compound <b>7</b> was predicted to have a high likelihood for the ability to cross the BBB as assessed using an MDCKII-hMDR1 monolayer cell assay in the presence or absence of P-glycoprotein 1 (Pgp) inhibitor <i>N</i>-(4-(2-(6,7-dimethoxy-3,4-dihydroisoquinolin-2(1<i>H</i>)-yl)ethyl)phenyl)-5-methoxy-9-oxo-9,10-dihydroacridine-4-carboxamide (GF120918). The assay was performed as previously described.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> The apparent permeability of <b>7</b> was 135 nm/s ± inhibitor, suggesting that <b>7</b> is not a substrate of Pgp-mediated efflux. This permeability rate is close to an ideal threshold value of 150 nm/s cited as an indicator of a good likelihood that the compound will cross into the CNS.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a></div></div><div id="sec2-3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Mechanism of Inhibition Studies</h3><div class="NLM_p">Reversibility of inhibition by <b>7</b> was tested in a rapid dilution assay.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a><i>Tb</i>AdoMetDC was equilibrated with <b>7</b> (36 μM ≈ IC<sub>90</sub> at pH 7.2) for 30 min and then diluted 100-fold to a concentration that was no longer inhibitory (<IC<sub>10</sub>) into reaction buffer containing ∼<i>K</i><sub>m</sub> concertation (50 μM) of substrate, <i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine (AdoMet), to initiate the reaction. Aliquots were taken out at 5 min intervals, enzyme was quenched, and produced decarboxylated AdoMet (dcAdoMet) was quantified using RapidFire–mass spectrometry (MS) (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A). The data were compared with those obtained for the vehicle control. Enzyme activity was fully recovered after diluting out the inhibitor following the 30 min treatment, which is consistent with rapidly reversible inhibition.</div><div class="NLM_p last">To determine the modality of the reversible inhibition, we followed dcAdoMet produced at varied concentrations of AdoMet and <b>7</b>. The double-reciprocal Lineweaver–Burk plot of the substrate titrations at four inhibitor concentrations is consistent with the competitive inhibition (<i>K</i><sub>i</sub> = 2.6 (2.2–3.0) μM; 95% confidence interval in parentheses) (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>B).</div></div><div id="sec2-4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Hit Expansion and Development of Structure–Activity Relationships (SARs)</h3><div class="NLM_p last">On the basis of a tractable chemotype and anticipated BBB permeability, <b>7</b> was prioritized for a hit-to-lead expansion. The atomic structure of <i>Tb</i>AdoMetDC was not available at the start of this project. Thus, the medicinal chemistry effort to elucidate the SAR relied on iterative structure alterations informed by enzyme and <i>T. brucei</i> cell-based assays. AdoMetDC inhibitory activity was determined at three pH values (6.8, 7.2, and 7.7) to assess the importance of N1 protonation for binding; only pH 7.2 data are reported in the main text figures and tables. Data collected at the alternative pH values are provided in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01654/suppl_file/jm7b01654_si_002.pdf" class="ext-link">Supporting Information (Table S1)</a>. Additionally, the inhibition of <i>Hs</i>AdoMetDC at pH 7.2 was measured to assess parasite–host enzyme selectivity.</div></div><div id="sec2-5" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> SAR Studies: Pyrimidine Ring Substituents</h3><div class="NLM_p">We began the SAR studies by modifying the pyrimidine ring (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). Replacement of the methyl substituent at C6 with ethyl (<b>9</b>) did not disrupt activity in the enzyme or cell-based assays, while increasing bulk at C6 by addition of isopropyl (<b>10</b>) led to a ∼3-fold increase in the IC<sub>50</sub> for AdoMetDC inhibition, as did C5–C6 cyclization (<b>15</b>) or removal of the C6-Me (<b>8</b>). Introduction of aromatic (<b>11</b>), heterocyclic (<b>12</b>), or electronegative (<b>13</b>, <b>14</b>) functionality at C6 caused a sharp reduction in inhibitor potency. It is unclear whether this drop is due to steric hindrance, adverse effect on the N1 protonation at the assay pH, or a combination of both. Notably, there was a lack of a corresponding drop in the cell growth assay, suggesting off-target activity. C5 could be methylated (<b>16</b>) without loss of enzyme inhibitory activity.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SAR of Pyrimidine C5 and C6 Positions</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/medium/jm-2017-01654m_0007.gif" alt="" id="GRAPHIC-d7e1291-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">R<sup>1</sup></th><th class="colsep0 rowsep0" align="center">R<sup>2</sup></th><th class="colsep0 rowsep0" align="center" char=".">p<i>K</i><sub>a</sub>(N1)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center"><i>Tb</i> IC<sub>50</sub>,<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a><sup>,</sup><a class="ref internalNav" href="#t1fn5" aria-label="e">e</a> μM</th><th class="colsep0 rowsep0" align="center"><i>Hs</i> IC<sub>50</sub>,<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a><sup>,</sup><a class="ref internalNav" href="#t1fn5" aria-label="e">e</a> μM</th><th class="colsep0 rowsep0" align="center"><i>Tb</i>427 EC<sub>50</sub>,<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a> μM</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="char" char=".">7.3</td><td class="colsep0 rowsep0" align="left">3.7 (3.3–4.1)</td><td class="colsep0 rowsep0" align="left">27% at 180</td><td class="colsep0 rowsep0" align="left">5.6 (4.5–7.1)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char=".">6.6</td><td class="colsep0 rowsep0" align="left">13 (11–15)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">9.9 (7.7–13)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Et</td><td class="colsep0 rowsep0" align="char" char=".">7.1</td><td class="colsep0 rowsep0" align="left">3.0 (2.7–3.3)</td><td class="colsep0 rowsep0" align="left">36% at 180</td><td class="colsep0 rowsep0" align="left">4.7 (4.1–5.4)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left"><sup><i>i</i></sup>Pr</td><td class="colsep0 rowsep0" align="char" char=".">7.0</td><td class="colsep0 rowsep0" align="left">12 (10–14)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">4.7 (3.9–5.8)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="char" char=".">6.1</td><td class="colsep0 rowsep0" align="left">24% at 21</td><td class="colsep0 rowsep0" align="left">>21</td><td class="colsep0 rowsep0" align="left">7.2 (6–8.5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left"><i>N</i>-morpholino</td><td class="colsep0 rowsep0" align="char" char=".">6.0</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">13 (10–17)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="char" char=".">–2.2</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">8.9 (7.3–11)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CF<sub>3</sub></td><td class="colsep0 rowsep0" align="char" char=".">–3.3</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">10 (7.5–14)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" colspan="2" align="center">-(CH<sub>2</sub>)<sub>3</sub>-</td><td class="colsep0 rowsep0" align="char" char=".">6.8</td><td class="colsep0 rowsep0" align="left">12 (11–14)</td><td class="colsep0 rowsep0" align="left">>21</td><td class="colsep0 rowsep0" align="left">19 (15–24)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="char" char=".">7.0</td><td class="colsep0 rowsep0" align="left">4.8 (4.5–5.2)</td><td class="colsep0 rowsep0" align="left">25% at 180</td><td class="colsep0 rowsep0" align="left">17 (14–20)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">The p<i>K</i><sub>a</sub> value of the conjugated acid of the pyrimidine nitrogen N1 estimated using MarvinSketch.</p></div><div class="footnote" id="t1fn2"><sup>b</sup><p class="last">Mean IC<sub>50</sub> for <i>T. brucei</i> AdoMetDC/Prozyme at pH 7.2 determined using the RapidFire–MS-based enzyme activity assay, in triplicate, with 95% confidence interval (CI) shown in parentheses.</p></div><div class="footnote" id="t1fn3"><sup>c</sup><p class="last">Mean IC<sub>50</sub> for human AdoMetDC at pH 7.2 determined in the RapidFire–MS-based enzyme activity assay, in triplicate, with 95% CI shown in parentheses.</p></div><div class="footnote" id="t1fn4"><sup>d</sup><p class="last">Mean EC<sub>50</sub> as measured in the ATP-bioluminescence-based bloodstream form <i>T. brucei</i> Lister 427 cell viability assay, in triplicate, with 95% CI shown in parentheses.</p></div><div class="footnote" id="t1fn5"><sup>e</sup><p class="last">Percent inhibition at the maximal tested concentration (in μM) is shown when maximum mean inhibition was <50%.</p></div></div></div></div><div id="sec2-6" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> SAR Studies: Pyrimidine Ring Nitrogens and Amino Group</h3><div class="NLM_p">Next, compounds were prepared to probe the function of the pyrimidine ring nitrogen atoms and C2-amino group (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). Substitution of N1 with CH (<b>17</b> vs <b>8</b>) led to complete loss of enzyme inhibitory activity consistent with the hypothesized importance of protonated NH<sup>+</sup> base for binding. This activity loss of 1-deaza analogs could not be rescued by moving the C2-amino group to the C6 position (<b>18</b> and <b>19)</b>. Conversely, replacement of N3 with CH led to only a 2- to 4-fold decrease in inhibitory activity (<b>20</b> and <b>21</b> compared with the respective parent compounds <b>7</b> and <b>8</b>). Interestingly, the p<i>K</i><sub>a</sub>(N1) for compounds with CH in place of N3 was predicted to increase by 2–3 units compared with the parent compounds. Thus the concentration of the N1-protonated species was expected to be >99% at pH 7.2, compared with only 55% or 20%, respectively, for the parent compounds, which was predicted to lead to an increase in inhibitory activity. These data suggest that the positive gains of increased N1 protonation were offset by loss of other interactions with N3.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. SAR of Pyrimidine Ring Nitrogens</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/medium/jm-2017-01654m_0008.gif" alt="" id="GRAPHIC-d7e1704-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">X</th><th class="colsep0 rowsep0" align="center">Y</th><th class="colsep0 rowsep0" align="center">R<sup>1</sup></th><th class="colsep0 rowsep0" align="center">R<sup>2</sup></th><th class="colsep0 rowsep0" align="center">R<sup>3</sup></th><th class="colsep0 rowsep0" align="center">p<i>K</i><sub>a</sub>(N1)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center"><i>Tb</i> IC<sub>50</sub>,<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a><sup>,</sup><a class="ref internalNav" href="#t2fn5" aria-label="e">e</a> μM</th><th class="colsep0 rowsep0" align="center"><i>Hs</i> IC<sub>50</sub>,<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a><sup>,</sup><a class="ref internalNav" href="#t2fn5" aria-label="e">e</a> μM</th><th class="colsep0 rowsep0" align="center"><i>Tb</i>427 EC<sub>50</sub>,<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a> μM</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">7.3</td><td class="colsep0 rowsep0" align="left">3.7 (3.3–4.1)</td><td class="colsep0 rowsep0" align="left">27% at 180</td><td class="colsep0 rowsep0" align="left">5.6 (4.5–7.1)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">6.6</td><td class="colsep0 rowsep0" align="left">13 (11–15)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">9.9 (7.7–13)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">96 (66–140)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">38 (31–47)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">24% at 180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">13 (8.2–22)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">9.6</td><td class="colsep0 rowsep0" align="left">15 (13–18)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">2.2 (1.7–2.9)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">9.4</td><td class="colsep0 rowsep0" align="left">29 (27–31)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">11 (9.5–12)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">CNH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">8.4</td><td class="colsep0 rowsep0" align="left">21% at 180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">18 (14–24)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">5.2</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">27 (22–33)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">NHMe</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">7.1</td><td class="colsep0 rowsep0" align="left">>63</td><td class="colsep0 rowsep0" align="left">>63</td><td class="colsep0 rowsep0" align="left">2.7 (2.4–3.1)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">NMe<sub>2</sub></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">7.0</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">14 (8.1–24)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">6.4</td><td class="colsep0 rowsep0" align="left">48 (43–54)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">14 (12–17)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">Br</td><td class="colsep0 rowsep0" align="left">6.4</td><td class="colsep0 rowsep0" align="left">24% at 180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">20 (17–24)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">The p<i>K</i><sub>a</sub> value of the conjugated acid of the pyrimidine nitrogen N1 estimated using MarvinSketch. NA, not applicable.</p></div><div class="footnote" id="t2fn2"><sup>b</sup><p class="last">Mean IC<sub>50</sub> for <i>T. brucei</i> AdoMetDC/Prozyme at pH 7.2 determined using the RapidFire–MS-based enzyme activity assay, in triplicate, with 95% confidence interval (CI) shown in parentheses.</p></div><div class="footnote" id="t2fn3"><sup>c</sup><p class="last">Mean IC<sub>50</sub> for human AdoMetDC at pH 7.2 determined in the RapidFire–MS-based enzyme activity assay, in triplicate, with 95% CI shown in parentheses.</p></div><div class="footnote" id="t2fn4"><sup>d</sup><p class="last">Mean EC<sub>50</sub> as measured in the ATP-bioluminescence-based bloodstream form <i>T. brucei</i> Lister 427 cell viability assay, in triplicate, with 95% CI shown in parentheses.</p></div><div class="footnote" id="t2fn5"><sup>e</sup><p class="last">Percent inhibition at the maximal tested concentration (in μM) is shown when maximum mean inhibition was <50%.</p></div></div></div><div class="NLM_p">Deletion (<b>23</b>) or methylation (<b>24</b> and <b>25</b>) of the C2-amino group led to the loss of the enzyme inhibition. While the former decreased the p<i>K</i><sub>a</sub>(N1) to 5.2, the latter had little effect on the predicted p<i>K</i><sub>a</sub> for N1 protonation but introduced steric interactions preventing optimal engagement of Glu266 (vide infra, <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B). Additionally, moving the C2-amino group to C3 in the background of a Me → H substitution at C6 led to further loss of inhibitory activity (<b>22</b> vs <b>21</b>). Taken together, these data suggest the importance of the primary C2-amine and N1 for binding interactions, which was later supported by crystallographic data (vide infra) showing H-bonding and salt-bridge of a protonated pyrimidineamine to Glu266 (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B).</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/medium/jm-2017-01654m_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/large/jm-2017-01654m_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Crystal structure of <i>Tb</i>AdoMetDC/Prozyme in complex with <b>44</b>. (A) Surface representation of <i>Tb</i>AdoMetDC/Prozyme (blue/yellow) with <b>44</b> (orange sticks) occupying the catalytic site. Pyruvoyl residue (Pvl) is shown as spheres. (B) Surface representation of the catalytic site. Residues forming the catalytic-site cavity are shown as sticks. Select interatomic distances are shown as dashed lines with values in Å. The catalytic-site cavity is shown as a semitransparent gray surface. Two pockets that are partially occupied by bromines are labeled <i>a</i> and <i>b</i> in purple.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/large/jm-2017-01654m_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01654&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Substituting an amino group for methyl at C6 (<b>26</b>) or swapping the C2-amino and C6-methyl groups (<b>27</b>; compare with the parent compound <b>44</b> in <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>) resulted in lower p<i>K</i><sub>a</sub>(N1) and a marked drop in the enzymatic inhibition. This is reflected in a better activity for <b>26</b> at pH 6.8 (5.8 μM against <i>Tb</i>AdoMetDC; see <a href="/doi/suppl/10.1021/acs.jmedchem.7b01654/suppl_file/jm7b01654_si_002.pdf" class="ext-link">Table S1</a>). Poorer activity of <b>27</b> can be rationalized by the loss of the C2-amino group essential for interaction with Glu266 (vide infra, <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B).</div><div class="NLM_p last">Notably, most compounds that showed poor enzyme inhibitory activity, with the exception of <b>17</b>, retained antitrypanosomal activity, suggesting the presence of at least a partial off-target component in the mechanism of the cell growth inhibition.</div></div><div id="sec2-7" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> SAR Studies: Linker Variation</h3><div class="NLM_p">Having established the pyrimidine ring traits required for activity, we shifted our focus to understanding the role of the linker (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. SAR of the Linker</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/medium/jm-2017-01654m_0009.gif" alt="" id="GRAPHIC-d7e2402-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">-X-</th><th class="colsep0 rowsep0" align="center">p<i>K</i><sub>a</sub>(N1)<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center"><i>Tb</i> IC<sub>50</sub>,<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a> μM</th><th class="colsep0 rowsep0" align="center">Hs IC<sub>50</sub>,<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a><sup>,</sup><a class="ref internalNav" href="#t3fn5" aria-label="e">e</a> μM</th><th class="colsep0 rowsep0" align="center"><i>Tb</i>427 EC<sub>50</sub>,<a class="ref internalNav" href="#t3fn4" aria-label="d">d</a> μM</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">-NH-</td><td class="colsep0 rowsep0" align="left">7.3</td><td class="colsep0 rowsep0" align="left">3.7 (3.3–4.1)</td><td class="colsep0 rowsep0" align="left">27% at 180</td><td class="colsep0 rowsep0" align="left">5.6 (4.5–7.1)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="left">-NMe-</td><td class="colsep0 rowsep0" align="left">7.2</td><td class="colsep0 rowsep0" align="left">81 (74–89)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">38 (34–43)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="left">-S-</td><td class="colsep0 rowsep0" align="left">4.3</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30</b></td><td class="colsep0 rowsep0" align="left">-NH(CH<sub>2</sub>)-</td><td class="colsep0 rowsep0" align="left">8.1</td><td class="colsep0 rowsep0" align="left">139 (130–150)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">11 (9.9–13)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="left">-NH(CH<sub>2</sub>)<sub>2</sub>-</td><td class="colsep0 rowsep0" align="left">8.2</td><td class="colsep0 rowsep0" align="left">28 (25–32)</td><td class="colsep0 rowsep0" align="left">29% at 180</td><td class="colsep0 rowsep0" align="left">5.7 (1.6–21)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>a</sup><p class="last">The p<i>K</i><sub>a</sub> value of the conjugated acid of the pyrimidine nitrogen N1 estimated using MarvinSketch.</p></div><div class="footnote" id="t3fn2"><sup>b</sup><p class="last">Mean IC<sub>50</sub> for <i>T. brucei</i> AdoMetDC/Prozyme at pH 7.2 determined using the RapidFire–MS-based enzyme activity assay, in triplicate, with 95% confidence interval (CI) shown in parentheses.</p></div><div class="footnote" id="t3fn3"><sup>c</sup><p class="last">Mean IC<sub>50</sub> for human AdoMetDC at pH 7.2 determined in the RapidFire–MS-based enzyme activity assay, in triplicate, with 95% CI shown in parentheses.</p></div><div class="footnote" id="t3fn4"><sup>d</sup><p class="last">Mean EC<sub>50</sub> as measured in the ATP-bioluminescence-based bloodstream form <i>T. brucei</i> Lister 427 cell viability assay, in triplicate, with 95% CI shown in parentheses.</p></div><div class="footnote" id="t3fn5"><sup>e</sup><p class="last">Percent inhibition at the maximal tested concentration (in μM) is shown when maximum mean inhibition was <50%.</p></div></div></div><div class="NLM_p last">Methylation of the linking amine (<b>28</b>) resulted in a sharp decrease in inhibitory activity in both enzyme and cell growth assays. The replacement of the amine with a thiol linker (<b>29</b>) eliminated enzyme inhibitory activity completely, presumably due to the effect on N1 acidity (p<i>K</i><sub>a</sub>(N1) = 4.3). Extension of the linker decreased potency to varying degrees. Addition of a single methylene (<b>30</b>) led to a 40-fold decrease in enzyme inhibition, while extension by two methylene units (<b>31</b>) decreased enzyme inhibition only by 8-fold.</div></div><div id="sec2-8" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> SAR Studies: Aniline Ring Substitutions</h3><div class="NLM_p">We next focused our efforts on elucidating the role of the substituted aniline ring. Both the size and electronic nature of the substituents in the meta positions were probed first (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>). Replacement of one of the two chlorines in <b>7</b> with moderately sized or large functional groups (Br, CN, CF<sub>3</sub>, Ph in <b>32</b>–<b>35</b>) was well tolerated, and these analogs had <i>Tb</i>AdoMetDC IC<sub>50</sub> values similar to the parent dichloro compound <b>7</b>. In contrast, pentafluorosulfanyl substitution led to 4-fold increase in the IC<sub>50</sub> for <i>Tb</i>AdoMetDC inhibition (<b>36</b> and <b>37</b>). Since Br was a good substituent for Cl, we used 3-bromo analogs to probe the second meta positions of the aniline ring with alkyl and <i>O</i>-alkyl functionalities of varying length, including branched and unsaturated chains (<b>39</b>–<b>43</b>). All of these analogs showed <i>T. brucei</i> AdoMetDC inhibitory activity. The trifluoroethoxy of <b>40</b> conferred the highest potency of the tested substitutions, showing 3.8-fold lower IC<sub>50</sub> than the similarly sized ethoxy of <b>39</b>. This underlines the importance of electronics in the terminal position of the substituent. Linear allyloxy and butoxy of <b>41</b> and <b>42</b>, respectively, performed 2.4-fold better than isobutoxy of <b>43</b>, suggesting a preference for linear over branched substituents. Also, ethyl in <b>38</b> conferred activity similar to that of ethoxy in <b>39</b>, suggesting that the phenolic ether was dispensable.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. SAR of Meta Aniline Ring Substitutions<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/medium/jm-2017-01654m_0010.gif" alt="" id="GRAPHIC-d7e2729-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">R<sup>2</sup></th><th class="colsep0 rowsep0" align="center">R<sup>4</sup></th><th class="colsep0 rowsep0" align="center"><i>Tb</i> IC<sub>50</sub>,<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a><sup>,</sup><a class="ref internalNav" href="#t4fn5" aria-label="e">e</a> μM</th><th class="colsep0 rowsep0" align="center"><i>Hs</i> IC<sub>50</sub>,<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a><sup>,</sup><a class="ref internalNav" href="#t4fn5" aria-label="e">e</a> μM</th><th class="colsep0 rowsep0" align="center"><i>Tb</i>427 EC<sub>50</sub>,<a class="ref internalNav" href="#t4fn4" aria-label="d">d</a> μM</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">3.7 (3.3–4.1)</td><td class="colsep0 rowsep0" align="left">27% at 180</td><td class="colsep0 rowsep0" align="left">5.6 (4.5–7.1)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b></td><td class="colsep0 rowsep0" align="left">Br</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">2.2 (2.0–2.4)</td><td class="colsep0 rowsep0" align="left">33% at 180</td><td class="colsep0 rowsep0" align="left">4.7 (3.9–5.8)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33</b></td><td class="colsep0 rowsep0" align="left">CN</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">3.9 (3.2–4.8)</td><td class="colsep0 rowsep0" align="left">29% at 180</td><td class="colsep0 rowsep0" align="left">12 (10–14)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34</b></td><td class="colsep0 rowsep0" align="left">CF<sub>3</sub></td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">3.9 (3.4–4.4)</td><td class="colsep0 rowsep0" align="left">39% at 180</td><td class="colsep0 rowsep0" align="left">13 (11–15)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>35</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">3.3 (2.8–3.9)</td><td class="colsep0 rowsep0" align="left">30% at 63</td><td class="colsep0 rowsep0" align="left">4.5 (2.1–9.4)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>36</b></td><td class="colsep0 rowsep0" align="left">SF<sub>5</sub></td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">16 (14–18)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">13 (11–15)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>37</b></td><td class="colsep0 rowsep0" align="left">SF<sub>5</sub></td><td class="colsep0 rowsep0" align="left">Br</td><td class="colsep0 rowsep0" align="left">8.7 (7.4–10)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">0.45 (0.32–0.62)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>38</b></td><td class="colsep0 rowsep0" align="left">Et</td><td class="colsep0 rowsep0" align="left">Br</td><td class="colsep0 rowsep0" align="left">7.6 (5.7–10)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">5.5 (4.6–6.6)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>39</b></td><td class="colsep0 rowsep0" align="left">OEt</td><td class="colsep0 rowsep0" align="left">Br</td><td class="colsep0 rowsep0" align="left">6.0 (5.2–6.9)</td><td class="colsep0 rowsep0" align="left">36% at 180</td><td class="colsep0 rowsep0" align="left">4.4 (4.2–4.7)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>40</b></td><td class="colsep0 rowsep0" align="left">OCH<sub>2</sub>CF<sub>3</sub></td><td class="colsep0 rowsep0" align="left">Br</td><td class="colsep0 rowsep0" align="left">1.6 (1.1–2.4)</td><td class="colsep0 rowsep0" align="left">44</td><td class="colsep0 rowsep0" align="left">4.2 (3.6–4.8)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>41</b></td><td class="colsep0 rowsep0" align="left">OCH<sub>2</sub>CHCH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">Br</td><td class="colsep0 rowsep0" align="left">4.5 (4.1–4.9)</td><td class="colsep0 rowsep0" align="left">28% at 180</td><td class="colsep0 rowsep0" align="left">4.4 (3.7–5.3)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>42</b></td><td class="colsep0 rowsep0" align="left">OBu</td><td class="colsep0 rowsep0" align="left">Br</td><td class="colsep0 rowsep0" align="left">4.5 (4.0–5.1)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">1.9 (1.7–2.1)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>43</b></td><td class="colsep0 rowsep0" align="left">OCH<sub>2</sub><sup><i>i</i></sup>Pr</td><td class="colsep0 rowsep0" align="left">Br</td><td class="colsep0 rowsep0" align="left">11 (9.5–12)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">4.4 (3.5–5.5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>44</b></td><td class="colsep0 rowsep0" align="left">Br</td><td class="colsep0 rowsep0" align="left">Br</td><td class="colsep0 rowsep0" align="left">1.9 (1.7–2.1)</td><td class="colsep0 rowsep0" align="left">31% at 180</td><td class="colsep0 rowsep0" align="left">4.8 (4.1–5.6)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>45</b></td><td class="colsep0 rowsep0" align="left">SO<sub>2</sub>Me</td><td class="colsep0 rowsep0" align="left">SO<sub>2</sub>Me</td><td class="colsep0 rowsep0" align="left">1.5 (1.3–1.7)</td><td class="colsep0 rowsep0" align="left">39% at 180</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>46</b></td><td class="colsep0 rowsep0" align="left">CF<sub>3</sub></td><td class="colsep0 rowsep0" align="left">CF<sub>3</sub></td><td class="colsep0 rowsep0" align="left">11 (9.8–12)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">14 (11–17)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>47</b></td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">17 (14–20)</td><td class="colsep0 rowsep0" align="left">21% at 180</td><td class="colsep0 rowsep0" align="left">41 (33–51)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>48</b></td><td class="colsep0 rowsep0" align="left">CO<sub>2</sub>H</td><td class="colsep0 rowsep0" align="left">CO<sub>2</sub>H</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>49</b></td><td class="colsep0 rowsep0" align="left">CO<sub>2</sub>Me</td><td class="colsep0 rowsep0" align="left">CO<sub>2</sub>Me</td><td class="colsep0 rowsep0" align="left">40% at 180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">42 (22–78)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>50</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">1.5 (1.3–1.6)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>51</b></td><td class="colsep0 rowsep0" align="left"><i>N</i>-(3,5-Cl<sub>2</sub>Ph)-acetamide</td><td class="colsep0 rowsep0" align="left"><i>N</i>-(3,5-Cl<sub>2</sub>Ph)-acetamide</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">2.0 (1.2–3.7)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>52</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">141 (119–167)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">19 (17–22)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>53</b></td><td class="colsep0 rowsep0" align="left"><sup><i>t</i></sup>Bu</td><td class="colsep0 rowsep0" align="left"><sup><i>t</i></sup>Bu</td><td class="colsep0 rowsep0" align="left">>63</td><td class="colsep0 rowsep0" align="left">>63</td><td class="colsep0 rowsep0" align="left">4.1 (4.1–4.5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>54</b></td><td class="colsep0 rowsep0" align="left">SF<sub>5</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">23% at 180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">14 (11–17)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>55</b></td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">35% at 180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">42 (34–52)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>56</b></td><td class="colsep0 rowsep0" align="left">Br</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">41% at 180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">22 (19–26)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>57</b></td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>58</b></td><td class="colsep0 rowsep0" align="left">SMe</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">149 (122–181)</td><td class="colsep0 rowsep0" align="left">23% at 180</td><td class="colsep0 rowsep0" align="left">>25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>59</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>60</b></td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">180 (160–201)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">70 (61–80)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>61</b></td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">32 (13–82)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">17 (15–20)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>a</sup><p class="last">All compounds have p<i>K</i><sub>a</sub>(N1) of 7.3. The p<i>K</i><sub>a</sub> value of the conjugated acid of the pyrimidine nitrogen N1 estimated using MarvinSketch.</p></div><div class="footnote" id="t4fn2"><sup>b</sup><p class="last">Mean IC<sub>50</sub> for <i>T. brucei</i> AdoMetDC/Prozyme at pH 7.2 determined using the RapidFire–MS-based enzyme activity assay, in triplicate, with 95% confidence interval (CI) shown in parentheses.</p></div><div class="footnote" id="t4fn3"><sup>c</sup><p class="last">Mean IC<sub>50</sub> for human AdoMetDC at pH 7.2 determined in the RapidFire–MS-based enzyme activity assay, in triplicate, with 95% CI shown in parentheses.</p></div><div class="footnote" id="t4fn4"><sup>d</sup><p class="last">Mean EC<sub>50</sub> as measured in the ATP-bioluminescence-based bloodstream form <i>T. brucei</i> Lister 427 cell viability assay, in triplicate, with 95% CI shown in parentheses.</p></div><div class="footnote" id="t4fn5"><sup>e</sup><p class="last">Percent inhibition at the maximal tested concentration (in μM) is shown when maximum mean inhibition was <50%.</p></div></div></div><div class="NLM_p">When both chlorines were simultaneously replaced with either bromines (<b>44</b>) or methylsulfonyls (<b>45</b>), inhibitor potency on the enzyme was improved by 2-fold (although <b>45</b> had no antitrypanosomal activity, presumably due to impaired cell uptake). Conversely, other moderately sized nonpolar and polar (Me, <i>t</i>-Bu, CF<sub>3</sub>, OMe, CO<sub>2</sub>H, CO<sub>2</sub>Me) or larger aryl (Ph and <i>N</i>-(3,5-(di-Cl)-Ph)-acetamide) substituents in both meta positions led to reduced (<b>46</b> and <b>47</b>) or nearly absent (<b>48</b>–<b>53</b>) enzyme inhibitory activity. Removal of one or both meta substituents (<b>54</b>–<b>59</b>) or replacement with a smaller fluorine substituent (<b>60</b>, <b>61</b>) substantially reduced (<b>54</b>–<b>56</b>, <b>58</b>, <b>60</b>, <b>61</b>) or eliminated (<b>57</b>, <b>59</b>) enzyme inhibitory activity.</div><div class="NLM_p">The correlation between enzyme and cell growth inhibition was dependent on the aniline-ring substitution pattern. Both enzyme and cell-based activity correlated reasonably for analogs <b>32</b>–<b>44</b>, <b>46</b>–<b>49</b>, <b>52</b>, <b>57</b>–<b>61</b>, whereas enzyme-inactive compounds <b>50</b>, <b>51</b>, <b>53</b> retained substantial antitrypanosomal activity indicating off-target activity. The very weak AdoMeDC inhibitors <b>54</b>–<b>56</b> did retain some albeit reduced cellular activity indicating a partial off-target activity or increased cellular uptake/accumulation.</div><div class="NLM_p">Having established the effects of meta substitution, we looked at SAR related to substitution patterns that include ortho and para positions (<a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>).</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. SAR of Ortho and Para Aniline Ring Substitutions<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/medium/jm-2017-01654m_0011.gif" alt="" id="GRAPHIC-d7e3660-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">R<sup>1</sup></th><th class="colsep0 rowsep0" align="center">R<sup>2</sup></th><th class="colsep0 rowsep0" align="center">R<sup>3</sup></th><th class="colsep0 rowsep0" align="center">R<sup>4</sup></th><th class="colsep0 rowsep0" align="center"><i>Tb</i> IC<sub>50</sub>,<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a><sup>,</sup><a class="ref internalNav" href="#t5fn5" aria-label="e">e</a> μM</th><th class="colsep0 rowsep0" align="center"><i>Hs</i> IC<sub>50</sub>,<a class="ref internalNav" href="#t5fn3" aria-label="c">c</a><sup>,</sup><a class="ref internalNav" href="#t5fn5" aria-label="e">e</a> μM</th><th class="colsep0 rowsep0" align="center"><i>Tb</i>427 EC<sub>50</sub>,<a class="ref internalNav" href="#t5fn4" aria-label="d">d</a><sup>,</sup><a class="ref internalNav" href="#t5fn5" aria-label="e">e</a> μM</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">3.7 (3.3–4.1)</td><td class="colsep0 rowsep0" align="left">27% at 180</td><td class="colsep0 rowsep0" align="left">5.6 (4.5–7.1)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>62</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">1.5 (1.4–1.8)</td><td class="colsep0 rowsep0" align="left">160 (135–191)</td><td class="colsep0 rowsep0" align="left">4.8 (3.3–7)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>63</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Br</td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">Br</td><td class="colsep0 rowsep0" align="left">1.6 (1.3–1.8)</td><td class="colsep0 rowsep0" align="left">43% at 180</td><td class="colsep0 rowsep0" align="left">4.6 (3.6–5.9)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>64</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">11 (8.8–13)</td><td class="colsep0 rowsep0" align="left">>63</td><td class="colsep0 rowsep0" align="left">4.5 (3.4–5.9)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>65</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">20 (17–24)</td><td class="colsep0 rowsep0" align="left">>63</td><td class="colsep0 rowsep0" align="left">4.4 (4.2–4.8)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>66</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">OPh</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">38% at 180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">1.1 (0.97–1.3)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>67</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">33% at 180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">13 (10–16)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>68</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">OCHF<sub>2</sub></td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">>63</td><td class="colsep0 rowsep0" align="left">>63</td><td class="colsep0 rowsep0" align="left">4.5 (3.7–5.4)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>69</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">NMe<sub>2</sub></td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">5.8 (4.2–7.9)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>70</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left"><i>N</i>-morpholino</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">18 (14–24)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>71</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left"><i>N</i>-piperidino</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">2.1 (1.3–3.2)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>72</b></td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">37 (29–46)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">26% at 100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>73</b></td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">2.2 (2.0–2.6)</td><td class="colsep0 rowsep0" align="left">>63</td><td class="colsep0 rowsep0" align="left">10 (8.5–12)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>74</b></td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">115 (89–150)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">23% at 25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>75</b></td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">137 (95–198)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">27% at 25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>76</b></td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">28% at 25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>77</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>78</b></td><td class="colsep0 rowsep0" align="left">SMe</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>79</b></td><td class="colsep0 rowsep0" align="left">Br</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>80</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">29% at 25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>81</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">28% at 25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>82</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>83</b></td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Br</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">25% at 25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>84</b></td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CF<sub>3</sub></td><td class="colsep0 rowsep0" align="left">199 (164–240)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">28% at 25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>85</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">28% at 25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>86</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Et</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">19 (17–22)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>87</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left"><sup><i>i</i></sup>Pr</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">11 (8.5–15)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>88</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CF<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">15 (13–17)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>89</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">3,5-Cl<sub>2</sub>Ph</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">1.0 (0.88–1.2)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>90</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">33% at 180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">25% at 25</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup>a</sup><p class="last">All compounds have p<i>K</i><sub>a</sub>(N1) of 7.3 except <b>67</b> (p<i>K</i><sub>a</sub>(N1) = 7.0); <b>72</b> (p<i>K</i><sub>a</sub>(N1) = 7.1); and <b>73</b>, <b>74</b>, and <b>83</b> (p<i>K</i><sub>a</sub>(N1) = 7.2). The p<i>K</i><sub>a</sub> value of the conjugated acid of the pyrimidine nitrogen N1 was estimated using MarvinSketch.</p></div><div class="footnote" id="t5fn2"><sup>b</sup><p class="last">Mean IC<sub>50</sub> for <i>T. brucei</i> AdoMetDC/Prozyme at pH 7.2 determined using the RapidFire–MS-based enzyme activity assay, in triplicate, with 95% confidence interval (CI) shown in parentheses.</p></div><div class="footnote" id="t5fn3"><sup>c</sup><p class="last">Mean IC<sub>50</sub> for human AdoMetDC at pH 7.2 determined in the RapidFire–MS-based enzyme activity assay, in triplicate, with 95% CI shown in parentheses.</p></div><div class="footnote" id="t5fn4"><sup>d</sup><p class="last">Mean EC<sub>50</sub> as measured in the ATP-bioluminescence-based bloodstream form <i>T. brucei</i> Lister 427 cell viability assay, in triplicate, with 95% CI shown in parentheses.</p></div><div class="footnote" id="t5fn5"><sup>e</sup><p class="last">Percent inhibition at the maximal tested concentration (in μM) is shown when maximum mean inhibition was <50%.</p></div></div></div><div class="NLM_p">Introduction of a fluorine into the para position led to up to 2-fold enhancement in potency toward <i>Tb</i>AdoMetDC (<b>62</b>, <b>63</b> vs <b>7</b>, <b>44</b>), whereas chlorine and methyl substitution (<b>64</b>, <b>65</b>) reduced activity 3- to 6-fold vs comparator <b>7</b>. All other para substituents that were explored, irrespective of size or electronic nature, led to a complete loss of inhibitory activity toward the enzyme. Retrospectively, this SAR is in agreement with the cocrystal structure data, which indicates a size restriction in the AdoMetDC binding pocket along the para position vector (vide infra, <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B). The ortho position tolerated addition of a fluoro (<b>73</b>), but substitution with OH (<b>72</b>) resulted in 10-fold reduced inhibitory activity vs <b>7</b>. Addition of a Cl, Br, Me, or SMe substituent in the ortho position of the inactive unsubstituted phenyl analog <b>59</b> or very weakly active mono meta-chloro or bromo analog <b>55</b> or <b>56</b> was ineffective at introducing any activity for compounds <b>74</b>–<b>84</b> (some additional inactive compounds are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.7b01654/suppl_file/jm7b01654_si_002.pdf" class="ext-link">Table S2</a>).</div><div class="NLM_p">In this series, there was a reasonable correlation between the activity on the enzyme and cellular activity for compounds without para substituents or with para halogens (<b>62</b>–<b>64</b>, <b>72</b>–<b>85</b>, <b>90</b>). However, this correlation breaks down for every other para-substituted compounds tested (<b>65</b>–<b>71</b>, <b>86</b>–<b>89</b>). Notably, enzyme inactive para-substituted analogs <b>66</b>–<b>71</b> and <b>86</b>–<b>89</b> retained substantial antitrypanosomal activity, with EC<sub>50</sub> values between 1 and 20 μM.</div><div class="NLM_p">The data so far suggested that meta-positioning of substituents on the aniline ring was crucial for activity and minimizing off-target antitrypanosomal activity. Minimally, one of the two meta positions requires substitution with a chlorine or bromine, whereas the other meta position needs to be substituted but has more flexibility such that halogens, cyano, CF<sub>3</sub>, Et, alkyl ethers, and Ph can be accommodated. However, while these modifications were tolerated, none resulted in a compound with substantially increased potency over the parent compound. Since we discovered that phenyl in one of the two meta positions yielded an active compound (<b>35</b>; see <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>), we prepared a number of substituted phenyl, heteroaromatic, and fused-ring analogs, aiming to further tune the inhibitory activity and on-target antitrypanosomal activity (<a class="ref internalNav" href="#tbl6" aria-label="Table 6">Table 6</a>).</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. SAR of Meta-Aryl Aniline Ring Substitutions<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/medium/jm-2017-01654m_0012.gif" alt="" id="GRAPHIC-d7e4755-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">R<sup>4</sup></th><th class="colsep0 rowsep0" align="center"><i>Tb</i> IC<sub>50</sub>,<a class="ref internalNav" href="#t6fn2" aria-label="b">b</a><sup>,</sup><a class="ref internalNav" href="#t6fn5" aria-label="e">e</a> μM</th><th class="colsep0 rowsep0" align="center"><i>Hs</i> IC<sub>50</sub>,<a class="ref internalNav" href="#t6fn3" aria-label="c">c</a><sup>,</sup><a class="ref internalNav" href="#t6fn5" aria-label="e">e</a> μM</th><th class="colsep0 rowsep0" align="center"><i>Tb</i>427 EC<sub>50</sub>,<a class="ref internalNav" href="#t6fn4" aria-label="d">d</a> μM</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>35</b></td><td class="colsep0 rowsep0" align="left">Ph Cl</td><td class="colsep0 rowsep0" align="left">3.3 (2.8–3.9)</td><td class="colsep0 rowsep0" align="left">30% at 63</td><td class="colsep0 rowsep0" align="left">4.5 (2.1–9.4)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>91</b></td><td class="colsep0 rowsep0" align="left">4-FPh</td><td class="colsep0 rowsep0" align="left">1.1 (0.95–1.2)</td><td class="colsep0 rowsep0" align="left">>21</td><td class="colsep0 rowsep0" align="left">1.6 (0.94–2.8)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>92</b></td><td class="colsep0 rowsep0" align="left">4-ClPh</td><td class="colsep0 rowsep0" align="left">1.7 (1.6–2.0)</td><td class="colsep0 rowsep0" align="left">>63</td><td class="colsep0 rowsep0" align="left">2.2 (1.8–2.7)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>93</b></td><td class="colsep0 rowsep0" align="left">4-MeOPh</td><td class="colsep0 rowsep0" align="left">2.3 (2.1–2.6)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">4.5 (3.1–6.6)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>94</b></td><td class="colsep0 rowsep0" align="left">4-MePh</td><td class="colsep0 rowsep0" align="left">4.6 (4.0–5.3)</td><td class="colsep0 rowsep0" align="left">>63</td><td class="colsep0 rowsep0" align="left">4.3 (3.3–5.7)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>95</b></td><td class="colsep0 rowsep0" align="left">4-CF<sub>3</sub>Ph</td><td class="colsep0 rowsep0" align="left">4.7 (3.9–5.7)</td><td class="colsep0 rowsep0" align="left">>21</td><td class="colsep0 rowsep0" align="left">2.0 (1.4–3)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>96</b></td><td class="colsep0 rowsep0" align="left">4-CNPh</td><td class="colsep0 rowsep0" align="left">7.6 (3.2–18)</td><td class="colsep0 rowsep0" align="left">>7</td><td class="colsep0 rowsep0" align="left">1.0 (0.96–1.1)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>97</b></td><td class="colsep0 rowsep0" align="left">4-AcPh</td><td class="colsep0 rowsep0" align="left">28% at 21</td><td class="colsep0 rowsep0" align="left">>21</td><td class="colsep0 rowsep0" align="left">6.7 (4.1–11)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>98</b></td><td class="colsep0 rowsep0" align="left">4-HOPh</td><td class="colsep0 rowsep0" align="left">143 (115–178)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">9.7 (7.5–13)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>99</b></td><td class="colsep0 rowsep0" align="left">4-PhPh</td><td class="colsep0 rowsep0" align="left">167 (150–190)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">0.91 (0.82–1)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>100</b></td><td class="colsep0 rowsep0" align="left">4-NMe<sub>2</sub>Ph</td><td class="colsep0 rowsep0" align="left">34% at 180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">5.0 (4.2–6)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>101</b></td><td class="colsep0 rowsep0" align="left">2-ClPh</td><td class="colsep0 rowsep0" align="left">26 (22–33)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">8.5 (7.6–9.5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>102</b></td><td class="colsep0 rowsep0" align="left">3-ClPh</td><td class="colsep0 rowsep0" align="left">81 (57–114)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">1.9 (1.6–2.3)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>103</b></td><td class="colsep0 rowsep0" align="left">3,4-(Cl)<sub>2</sub>Ph</td><td class="colsep0 rowsep0" align="left">102 (82–125)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">2.9 (1.3–3.6)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>104</b></td><td class="colsep0 rowsep0" align="left">3-Cl-4-FPh</td><td class="colsep0 rowsep0" align="left">157 (95–260)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">3.2 (2.8–3.7)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>105</b></td><td class="colsep0 rowsep0" align="left">3-CF<sub>3</sub>-4-ClPh</td><td class="colsep0 rowsep0" align="left">53 (46–61)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">1.5 (1–2.2)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>106</b></td><td class="colsep0 rowsep0" align="left">3,5-(CF<sub>3</sub>)<sub>2</sub>Ph</td><td class="colsep0 rowsep0" align="left">28 (25–31)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">0.53 (0.47–0.59)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>107</b></td><td class="colsep0 rowsep0" align="left">2-naphthyl</td><td class="colsep0 rowsep0" align="left">57 (51–64)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">1.7 (1.4–2)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>108</b></td><td class="colsep0 rowsep0" align="left">9-phenanthrenyl</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">1.8 (1.5–2.1)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>109</b></td><td class="colsep0 rowsep0" align="left">5-benzo-1,3-dioxolyl</td><td class="colsep0 rowsep0" align="left">6.2 (4.6–8.3)</td><td class="colsep0 rowsep0" align="left">>21</td><td class="colsep0 rowsep0" align="left">2.3 (1.6–3.2)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>110</b></td><td class="colsep0 rowsep0" align="left">5-benzofurazanyl</td><td class="colsep0 rowsep0" align="left">27 (21–35)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">0.57 (0.13–2.5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>111</b></td><td class="colsep0 rowsep0" align="left">3-furanyl</td><td class="colsep0 rowsep0" align="left">7.4 (6.2–8.7)</td><td class="colsep0 rowsep0" align="left">30% at 180</td><td class="colsep0 rowsep0" align="left">7.1 (5.7–8.8)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>112</b></td><td class="colsep0 rowsep0" align="left">2-benzofuranyl</td><td class="colsep0 rowsep0" align="left">23% at 63</td><td class="colsep0 rowsep0" align="left">>63</td><td class="colsep0 rowsep0" align="left">1.4 (1.4–1.5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>113</b></td><td class="colsep0 rowsep0" align="left">3-thiopheneyl</td><td class="colsep0 rowsep0" align="left">26 (22–32)</td><td class="colsep0 rowsep0" align="left">30% at 180</td><td class="colsep0 rowsep0" align="left">29 (23–35)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>114</b></td><td class="colsep0 rowsep0" align="left">2-benzothiopheneyl</td><td class="colsep0 rowsep0" align="left">41 (36–46)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">1.6 (1.5–1.8)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>115</b></td><td class="colsep0 rowsep0" align="left">5-thiazolyl</td><td class="colsep0 rowsep0" align="left">42% at 21</td><td class="colsep0 rowsep0" align="left">>21</td><td class="colsep0 rowsep0" align="left">3.6 (3.1–4.1)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>116</b></td><td class="colsep0 rowsep0" align="left"><i>N</i>-Me-4-pyrazolyl</td><td class="colsep0 rowsep0" align="left">164 (132–203)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">6.7 (4.7–9.6)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>117</b></td><td class="colsep0 rowsep0" align="left">5-pyrimidinyl</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">58 (33–100)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>118</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">2.7 (2.2–3.4)</td><td class="colsep0 rowsep0" align="left">>180</td><td class="colsep0 rowsep0" align="left">1.3 (1.1–1.5)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup>a</sup><p class="last">All compounds have p<i>K</i><sub>a</sub>(N1) of 7.3. The p<i>K</i><sub>a</sub> value of the conjugated acid of the pyrimidine nitrogen N1 estimated using MarvinSketch.</p></div><div class="footnote" id="t6fn2"><sup>b</sup><p class="last">Mean IC<sub>50</sub> for <i>T. brucei</i> AdoMetDC/Prozyme at pH 7.2 determined using the RapidFire–MS-based enzyme activity assay, in triplicate, with 95% confidence interval (CI) shown in parentheses.</p></div><div class="footnote" id="t6fn3"><sup>c</sup><p class="last">Mean IC<sub>50</sub> for human AdoMetDC at pH 7.2 determined in the RapidFire–MS-based enzyme activity assay, in triplicate, with 95% CI shown in parentheses.</p></div><div class="footnote" id="t6fn4"><sup>d</sup><p class="last">Mean EC<sub>50</sub> as measured in the ATP-bioluminescence-based bloodstream form <i>T. brucei</i> Lister 427 cell viability assay, in triplicate, with 95% CI shown in parentheses.</p></div><div class="footnote" id="t6fn5"><sup>e</sup><p class="last">Percent inhibition at the maximal tested concentration (in μM) is shown when maximum mean inhibition was <50%.</p></div></div></div><div class="NLM_p">Compared to the parent <i>m</i>-phenyl substituted biaryl analog <b>35</b>, 4-FPh and 4-ClPh substituted biaryl analogs <b>91</b> and <b>92</b> were about 2- to 3-fold more potent in the enzyme and cell-based assays, whereas 4-OMe, 4-Me, and 4-CF<sub>3</sub> substitution delivered roughly equipotent compounds in both assays (<b>93</b>–<b>95</b>). A 2-fold loss of enzyme inhibition was observed for the 4-CNPh analog (<b>96</b>), but this was accompanied by an increase in cellular activity (∼5-fold). Moving to 4-Ac (<b>97</b>), 4-OH (<b>98</b>), and 4-NMe<sub>2</sub> (<b>100</b>) substituents, enzyme inhibition dropped off markedly but cell-based inhibition remained at single-digit micromolar EC<sub>50</sub> levels, indicating off-target activity. Similarly, moving chlorine to the 2- or 3-position (<b>101</b>, <b>102</b>), adding an extra Cl or CF<sub>3</sub> to the 3-position (<b>103</b>–<b>105</b>), or 3,5-(CF<sub>3</sub>)<sub>2</sub> substitution (<b>106</b>) also abolished enzyme inhibition while retaining the off-target cell-based activity.</div><div class="NLM_p last">Biaryls (4-PhPh, <b>99</b>), polycyclic aryls (naphthyl <b>107</b>, phenanthrenyl <b>108</b>), benzo-fused heteroaryls (<b>110</b>, <b>112</b>, <b>114</b>), or five-membered heteroaromatics (5-thiazolyl <b>115</b>, <i>N</i>-Me-4-pyrazolyl <b>116</b>) were largely incompatible with enzyme inhibition but retained potent off-target cell activity. Enzyme and cell activity tracked better for benzodioxolyl (<b>109</b>), furanyl (<b>111</b>), thiophenyl (<b>113</b>), and pyrimidinyl (<b>117</b>) substituents, with the former two (<b>109</b>, <b>111</b>) active in the low micromolar range and the latter two (<b>113</b>, <b>117</b>) largely devoid of activity in both assays. Finally, we prepared and tested compound <b>118</b> that combined 4-fluorophenyl and ethyl meta substitutions. Compared to the corresponding 3-(4-FPh)-5-Cl analog <b>91</b>, the similar activity of <b>118</b> indicates that <i>m</i>-Cl can be exchanged for <i>m</i>-Et without affecting enzyme or cell-based potency.</div></div><div id="sec2-9" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Crystal Structure of <i>Tb</i>AdoMetDC/Prozyme in Complex with a Pyrimidineamine Analog Reveals Binding Interactions</h3><div class="NLM_p">To assist in rational design of pyrimidineamine analogs with improved binding properties, we solved the crystal structure of the <i>Tb</i>AdoMetDC/prozyme complex cocrystallized with <b>44</b>. Compound <b>44</b> was selected for cocrystallization on the basis of its potency and solubility in aqueous solutions. Additionally, the presence of two bromines facilitated location of the bound compound in the complex, and the anomalous diffraction data collected at the Br K-edge were instrumental in determining the correct orientation of this quasi-symmetrical molecule in the binding pocket (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>). The structure was solved by molecular replacement using our previously determined <i>Tb</i>AdoMetDC/Prozyme heterodimeric structure (Protein Data Bank (PDB) identifier <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TVM">5TVM</a>)<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> as a search model and refined to 3.0 Å resolution with <i>R</i><sub>work</sub>/<i>R</i><sub>free</sub> = 20.0/25.3%. Overall, the structure was similar to the unliganded active heterodimer (rmsd of 0.28 Å between current structure and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TVM">5TVM</a> as determined by TM-align<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> for 636 equivalent C<sub>α</sub> atoms).</div><div class="NLM_p">A single molecule of <b>44</b> was found in the catalytic site of the <i>Tb</i>AdoMetDC in the heterodimer of <i>Tb</i>AdoMetDC/Prozyme, positioned 3.9 Å from the catalytic pyruvate (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A). This binding mode is consistent with biochemically demonstrated competitive mode of inhibition (see <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>B). The binding interactions observed between <b>44</b> and the enzyme also explain much of the above-described SAR for the series. The requirement for N1 to be protonated and for C2 to have a primary amine is explained by the finding that both groups are within hydrogen-bonding distance of the side-chain carboxyl oxygens of Glu266 in the binding pocket. It is thus reasonable to assume that a protonated pyrimidineamine participates in favorable bidentate H-bonding and salt-bridge interactions (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B).</div><div class="NLM_p">Additional binding energy is also likely contributed by π–π stacking interactions between the pyrimidine ring and Phe28 and between aniline ring and Tyr243. The proximity of the catalytic Cys100 to the aniline ring of <b>44</b> (3.2 Å between SH and C4′ of the ring) is suggestive of the aromatic–thiol interaction (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B), though the angle of the interaction is not ideal as determined by previously described interactions of this type.<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35, 36)</a> Finally, an H-bonding interaction<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> between the partially negative lateral end of one of the aniline ring <i>m</i>-bromines and Ser249 hydroxyl group is also potentially present and may contribute to the binding energy (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B).</div><div class="NLM_p last">Importantly, the structure of <i>Tb</i>AdoMetDC cocrystallized with <b>44</b> revealed that the 3′- and 5′-substituents on the aniline ring projected into two distinct pockets designated <i>a</i> and <i>b</i> (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B). Building on the structural data and the observations reported in <a class="ref internalNav" href="#tbl4" aria-label="Tables 4">Tables 4</a>–<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a>, we hypothesize that of the two pockets, only pocket <i>a</i> could accommodate a phenyl-sized substitution at the meta position of the aniline ring. Conversely, the shape and depth of pocket <i>b</i> suggested that it would be accessible to linear, flexible substituents (for example, alkyls and alkyl ethers <b>38</b>–<b>43</b> in <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>). The fact that <b>118</b>, functionalized with <i>p</i>-fluorophenyl at one meta position and ethyl at the other, inhibited the enzyme with good potency (IC<sub>50</sub> = 2.7 μM; see <a class="ref internalNav" href="#tbl6" aria-label="Table 6">Table 6</a>) suggests that both pockets can be accessed simultaneously in the context of a single compound. Additionally, our structure suggested that para substituents larger than F, Cl, or Me would be inaccessible as confirmed by the enzyme inhibition data in <a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a> (<b>66</b>–<b>71</b>).</div></div><div id="sec2-10" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Structural Basis for Parasite–Host Selectivity</h3><div class="NLM_p"><i>Hs</i>AdoMetDC (PDB identifier <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3DZ7">3DZ7</a>)<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> was aligned to the <i>Tb</i>AdoMetDC bound to <b>44</b> to provide insight into the structural basis for selectivity of this and other pyrimidineamines for <i>T. brucei</i> AdoMetDC compared with the human enzyme (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>). The catalytic sites are mostly conserved between the two species, and their geometries are similar. One residue at the lid of the pocket, however, differs between them. This residue (His5 in human AdoMetDC and Arg26 in <i>T. brucei</i> AdoMetDC) is positioned differently in the two structures. His5 in human AdoMetDC is within 3.2 Å of <b>44</b> in the overlaid enzyme and may function to restrict the ligand-binding site compared with the <i>T. brucei</i> structure (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>). The analogous position in <i>T. brucei</i> AdoMetDC (Arg26) was not well resolved in the electron density maps of the structure reported herein; however, it was observed in previous structures (PDB depositions <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TVF">5TVF</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TVM">5TVM</a>) where it was positioned away from the opening of the binding pocket (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>).</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/medium/jm-2017-01654m_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/large/jm-2017-01654m_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Comparison of human and <i>T. brucei</i> AdoMetDC active sites. Human AdoMetDC (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3DZ7">3DZ7</a>, brown) was aligned with <i>Tb</i>AdoMetDC structure (blue) cocrystallized with <b>44</b> (orange). The analogous residues are labeled in the corresponding colors (<i>T. brucei</i> residues on top). Alignment was done in PyMol, and the side chains within 4 Å of an inhibitor (with the exception of <i>Tb</i>Glu86/<i>Hs</i>Glu68) are shown here. A dashed line represents a distance, in Å, between His5 of the human protein and <b>44</b> in the aligned <i>T. brucei</i> structure. Residues Ser25 and Arg26, which were not modeled in <b>44</b>-liganded structure, are from the previously reported unliganded <i>Tb</i>AdoMetDC structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TVM">5TVM</a>, yellow). The active-site cavities for human (brown) and <b>44</b>-bound <i>T. brucei</i> (blue) AdoMetDCs are shown as semitransparent surfaces (pocket <i>b</i> is pointing away from the viewer).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/large/jm-2017-01654m_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01654&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Additionally, the substitution of Cys49 in the human structure for Ala67 in <i>T. brucei</i> AdoMetDC restricts the size of the human pocket <i>a</i> relative to <i>T. brucei</i> (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>). This size restriction would be expected to provide additional selectivity for analogs that have larger substituents in pocket <i>a</i>, such as the <i>p</i>-fluorophenyl of <b>91</b> and <b>118</b>.</div></div><div id="sec2-11" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> On-Target Mechanism-of-Action Studies</h3><div class="NLM_p">As was noted above, the correlation between the enzyme and cell-growth inhibition was weak. This suggested that trypanocidal effect was at least in part due to off-target mechanism of action. Thus, we sought to identify whether the pyrimidineamines were inhibiting AdoMetDC in <i>T. brucei</i> cells. We previously reported that the levels of Prozyme protein markedly increased when AdoMetDC activity in <i>T. brucei</i> cells was reduced either by genetic knockdown of AdoMetDC or by chemical inhibition.<a onclick="showRef(event, 'ref16 ref39'); return false;" href="javascript:void(0);" class="ref ref16 ref39">(16, 39)</a> We exploited this feedback loop to evaluate whether hit compounds identified in the HTS were affecting the parasite cell growth through intracellular AdoMetDC inhibition.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Here, we compared Prozyme protein levels in cells treated for 24 h with representative pyrimidineamine analogs that (1) inhibited both <i>T. brucei</i> cell growth and AdoMetDC (<b>7</b> and <b>8</b>), (2) inhibited only <i>T. brucei</i> growth (<b>23</b> and <b>13</b>), or (3) were inactive in both assays (<b>29</b>). Only compounds that were active on both the enzyme and cells induced Prozyme above the vehicle-only control levels (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>). These results show that the pyrimidineamines with <i>Tb</i>AdoMetDC inhibitory activity on the purified enzyme also block AdoMetDC activity in <i>T. brucei</i> cells. However, these results also confirm that there was a substantial off-target component to pyrimidineamine-induced cell growth inhibition, as <b>23</b> and <b>13</b>, which did not inhibit <i>Tb</i>AdoMetDC either in the purified-enzyme assay or in the cell (as evident from vehicle-only levels of the Prozyme protein), inhibited cell growth with low-micromolar potencies (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>).</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/medium/jm-2017-01654m_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/large/jm-2017-01654m_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Mechanism of action studies in <i>T. brucei</i> bloodstream form parasites. Western blot analysis of Prozyme levels in <i>T. brucei</i> cells treated with active or inactive pyrimidineamine analogs for 24 h. Three types of compounds were compared: (1) compounds that demonstrated low-micromolar inhibition of <i>Tb</i>AdoMetDC at pH 7.2 (IC<sub>50(pH7.2)</sub>) and of <i>T. brucei</i> cells in culture (EC<sub>50</sub>) (<b>7</b> and <b>8</b>); (2) compounds that did not inhibit <i>Tb</i>AdoMetDC but did inhibit <i>T. brucei</i> cell growth (<b>23</b> and <b>13</b>); (3) a compound that did not inhibit either <i>Tb</i>AdoMetDC or the cells (<b>29</b>). Cells were treated with 1.5 × EC<sub>50</sub> concentrations of inhibitors (or 30 μM <b>29</b>). <i>T. brucei</i> binding protein (BiP) was used as a loading control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/large/jm-2017-01654m_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01654&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i21">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76486" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76486" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The number of cases of HAT has decreased 10-fold since the beginning of the 21st century owing to proactive surveillance and control measures set in place in endemic countries.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Yet the disease has a history of resurgence following years of a downward trend.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Sustainability of HAT elimination relies in part on the availability of patient-friendly treatment options.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Neither melarsoprol nor eflornithine, alone or as a part of NECT, is ideal due to high toxicity of the former and high price and requirement for multiple intravenous infusions of the latter.<a onclick="showRef(event, 'ref2 ref10'); return false;" href="javascript:void(0);" class="ref ref2 ref10">(2, 10)</a> The potential for eflornithine resistance to emerge is another negative factor.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Thus, new safe antitrypanosomal drugs with CNS-stage efficacy are needed.</div><div class="NLM_p">We identified and characterized a pyrimidineamine series as a new class of reversible, competitive inhibitors of <i>T. brucei</i> AdoMetDC. This series has a number of positive attributes, including species-selective inhibition of the <i>T. brucei</i> AdoMetDC compared with the human enzyme and antitrypanosomal activity against <i>T. brucei</i> in whole-cell assays. Importantly, the series is predicted to have good brain penetration on the basis of the monolayer permeability assay performed on the parent compound. Thus, the pyrimidineamines have the potential to be used for treatment of both early-stage and late-stage HAT. Herein, we have established the preliminary SAR within the series and have solved the crystal structure of one analog bound to <i>T. brucei</i> AdoMetDC, providing insight for future lead optimization of the series.</div><div class="NLM_p">While the pyrimidineamines look to be a promising series for further optimization, three key issues were identified that would need to be addressed in future chemistry programs. First, while the pyrimidineamines did show activity against both the enzyme and the parasite, the correlation between these activities was poor. On-target activity was present as evident from the finding that Prozyme was upregulated in cells treated with compounds that inhibited AdoMetDC. In addition, good correlation between enzyme inhibition and antiparasite activity was observed for most active analogs with two meta substituents in the aniline ring, with at least one substituent being Cl or Br. If the second meta position is a phenyl-ring substituted in the 4-position with F, Cl, OMe, Me, or CF<sub>3</sub>, good activity in both assays was observed. However, there was also clear evidence for an off-target activity, as a substantial fraction of pyrimidineamines that had lost <i>Tb</i>AdoMetDC inhibitory activity elicited cell growth inhibition in the low-micromolar range. Off-target activity was especially noticeable for para-substituted aniline rings (<a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>) and bis-meta-substituted aniline rings with meta position occupied by 2-, 3-, 2,3-, or other specifically substituted aryls, biaryls, polycyclic aryls, and heteroaryls (<a class="ref internalNav" href="#tbl6" aria-label="Table 6">Table 6</a>). Second, despite synthesis of a number of analogs of the original parent compound, we were unable to substantially improve compound potency below the low-micromolar level. This issue may also relate to the off-target activity, and thus identification of more potent analogs may lead to reduced off-target activity as the on-target component improves. As the analogs described herein had been synthesized prior to obtaining the crystal structure with an inhibitor bound, there remains room for a structure-based lead optimization program to facilitate the identification of more potent, on-target compounds. Lastly, <b>7</b> showed poor metabolic stability; however, this liability was not the focus of the current chemistry program and remains to be addressed during further optimization of the series.</div><div class="NLM_p">A notable feature of pyrimidineamines is the finding that inhibitor binding to <i>Tb</i>AdoMetDC is pH-dependent in the relevant physiological range (pH 6.8–7.7). This dependence has its basis in the p<i>K</i><sub>a</sub> of N1 in the pyrimidine ring, which is predicted to be 7.3 in the parent compound <b>7</b>. Within the series, modifying the ring substituents to lower the p<i>K</i><sub>a</sub>(N1) (<6) led to reduced inhibitory activity at physiological pH, while optimizing the ring electronics to yield a higher p<i>K</i><sub>a</sub>(N1) (>9) reduced the pH dependence of binding. The crystal structure of <i>Tb</i>AdoMetDC bound to <b>44</b> revealed that N1 forms a salt bridge with Glu266, explaining the requirement for protonation of N1 for inhibitor binding. While we were able to identify compounds with higher p<i>K</i><sub>a</sub>, this was achieved through removal of N3 (<b>20</b> and <b>21</b>), which led to a loss of the possible interaction between N3 and main-chain carbonyl of Cys246 (3.5 Å away; see <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B). Thus, we did not accomplish the goal of improved binding affinity by this strategy. One potential direction for future optimization of this series will be to identify compounds that retain full charge at N1 at physiological pH, without compromising other binding interactions.</div><div class="NLM_p">In addition to the protonation state of N1, we identified a number of other features of the pharmacophore that were required for <i>Tb</i>AdoMetDC inhibition. A primary amine is required at C2, and it is involved in an H-bond with the Glu266 carboxyl group. The amine linker could not be modified as there was a loss of potency when the linker was methylated (<b>28</b>), extended by one or two methylene links (<b>30</b> and <b>31</b>), or replaced with sulfur (<b>29</b>). The amine linker does not make any clear interactions with the protein, so its importance may relate to geometry and steric considerations. For sulfur-linked analog <b>29</b>, the decreased p<i>K</i><sub>a</sub> at N1 (4.3) could explain the loss of enzyme inhibition. Finally, both pyrimidine and aniline rings are required, and these form stabilizing π–π stacking interactions with the two aromatic residues, Phe28 and Tyr243, on opposite sides of the catalytic site.</div><div class="NLM_p">The aniline ring is the most buried part of the ligand. SAR and the crystal structure indicate that substituents in both of the meta positions are required for good binding. We identified both steric and electronic requirements to the meta substitutions. At least one of the meta positions can include an alkyl, alkyloxy, halide, or 4-substituted phenyl ring, whereas the second position is best occupied by Cl or Br. Notably, both an alkyl (ethyl) and a 4-substituted phenyl could be accommodated at the same time as indicated by anti-<i>Tb</i>AdoMetDC activity of <b>118</b>. A hydrogen bond between Ser249 γ-hydroxyl and a partially negative acceptor in a meta position such as halide or alkoxy may contribute to the binding energy and explain the finding.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> The next steps, informed by the cocrystal structure data, will involve a much more in depth combinatorial exploration of both meta positions in such a manner as to ideally occupy both <i>a</i> and <i>b</i> pockets as well as maximize π–π stacking interactions.</div><div class="NLM_p">All active analogs were strongly selective for <i>T. brucei</i> AdoMetDC over the human enzyme. The selectivity ratio (<i>Hs</i>IC<sub>50</sub>/<i>Tb</i>IC<sub>50</sub>) was at least 27 (<b>40</b>) and in many cases exceeded 100 (e.g., <b>44</b> and <b>62</b>). Yet the structural basis for species selectivity was not immediately obvious given very similar geometries and residue composition of the catalytic sites in the two species. Specifically, the human site has Phe7 and Phe223 in the positions analogous to <i>T. brucei</i> Phe28 and Tyr243, which could provide π–π stacking interactions with a compound (see <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>). Also, Glu247 in the human enzyme closely aligns with <i>T. brucei</i> Glu266 and could potentially also form H-bonding and salt-bridge interactions with pyrimidineamine part of an inhibitor. One possible hypothesis suggested by our cocrystal structure is that His5 in the human AdoMetDC sterically limits access to the binding site, excluding certain classes of inhibitors, and thus may be responsible for the observed species selectivity of the pyrimidineamines. In <i>T. brucei</i>, the equivalent position is occupied by an Arg residue with a more labile side chain (see <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>). Notably, all 13 classes of validated active hits from our HTS campaign were also selective inhibitors of the <i>T. brucei</i> enzyme,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> suggesting this may be a general principle of parasite–host selectivity. In contrast, a number of compounds that have greater binding affinity for the human enzyme have also been described, such as methylglyoxal bis(guanylhydrazone), or MGBG,<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> and CGP 40215,<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> so evidently this mechanism does not limit general access to the active site. An additional possible origin for species selectivity can be inferred from the allosteric mechanism of <i>T. brucei</i> AdoMetDC activation: its N-terminal residues form an autoinhibitory lid blocking the active site in the monomer that is displaced from the active site upon binding to Prozyme.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Conversely, the N-terminus of the human enzyme is 21 residues shorter than that of the <i>T. brucei</i> enzyme, and it has not been observed to bind in the active site in any of the published X-ray crystal structures.<a onclick="showRef(event, 'ref38 ref43 ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref38 ref43 ref44 ref45">(38, 43-45)</a> However, these data do not rule out the possibility that these residues undergo dynamic transitions during catalysis or ligand binding. Thus, potentially selectivity could arise if the N-terminal residues of the human enzyme compete for ligand binding. In this scenario, <i>T. brucei</i> selective inhibitors would be unable to effectively compete with the N-terminal residue lid, whereas human selective inhibitors and substrate would be effective competitors. Finally, we note that pocket <i>a</i> in human AdoMetDC is smaller than in the <i>T. brucei</i> enzyme, and this difference will enhance selectivity for compounds that fully occupy this pocket (e.g., <b>91</b>), though this difference is unlikely to explain the selectivity of compounds with smaller substitutions in pocket <i>a</i> (e.g., <b>44</b>).</div><div class="NLM_p last">The work described herein has led to the discovery of pyrimidineamines with low-micromolar activity against both <i>T. brucei</i> AdoMetDC and the parasite. Through a combination of medicinal chemistry and protein crystallography, we have defined the key components of the pharmacophore that are required for <i>Tb</i>AdoMetDC inhibitory activity. If this series is to advance to lead optimization, future work will need to focus on identifying more potent compounds. Two pockets were discovered in the crystal structure that were not fully exploited in the current work. Pocket <i>a</i> is larger of the two and is lined with both polar (Glu85 and Pvl86) and nonpolar (Ile65, Ala67, Leu83, and Ile103) side chains (see <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B). The <i>p</i>-substituted phenyl analogs such as <b>91</b> provided for good size complementarity with the available pocket, but the crystal structure and activity data suggest that more polar substituents that could engage in H-bonding interactions with Glu85 and backbone carbonyls in the pocket might provide better binding energy (see <a class="ref internalNav" href="#tbl6" aria-label="Table 6">Table 6</a> and <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B). Alternatively, a nucleophilic group (e.g, aminooxy) might allow formation of a Schiff base with the prosthetic pyruvoyl group, transforming pyrimidineamines into mechanism-based inhibitors, as described for substrate analogs.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Pocket <i>b</i> also has substantial polar character with several opportunities for H-bonding interactions presented by side chains of Glu32, Ser273, Ser249, and His262 (see <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B). Current ring substituents did not fully exploit these opportunities, and thus further series development should focus on optimizing these potential polar interactions at the meta positions of the aniline ring. The search of the suitable substituents can be aided by computational protein–ligand docking through virtual screening of meta position analogs. Iterative cocrystallization with new analogs will aid development.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16252" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16252" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4-1-1" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Chemistry. General Procedures</h3><div class="NLM_p last">Unless otherwise specified, all commercially available reagents were used as received. All reactions using dried solvents were carried out under an atmosphere of argon in flame-dried glassware with magnetic stirring. Dry solvent was dispensed from a solvent purification system that passes solvent through two columns of dry neutral alumina. Silica gel chromatographic purifications were performed by flash chromatography with silica gel (Sigma, grade 62, 60–200 mesh) packed in glass columns; the eluting solvent was determined by thin layer chromatography (TLC). Analytical TLC was performed on glass plates coated with 0.25 mm silica gel using UV for visualization. Melting points are uncorrected. Routine <sup>1</sup>H and proton-decoupled <sup>13</sup>C NMR spectra were obtained on a Bruker 400 MHz NMR spectrometer. Chemical shifts (δ) are reported in parts per million (ppm) from low to high field relative to residual solvent. Multiplicities are given as s (singlet), d (doublet), t (triplet), q (quartet), dd (doublet of doublets), m (multiplet). All synthetic compounds exhibited >95% purity as determined by LC–MS analysis performed on an Agilent 1100 HPLC system using an Eclipse XDB-C18 column (4.6 mm × 150 mm, 5 μm; Agilent) that was coupled to an Agilent G1956A (or 6120) ESI mass spectrometer run in the positive mode with a scan range of 100–800 (or 1000) <i>m</i>/<i>z</i>. Liquid chromatography was carried out at a flow rate of 0.5 mL/min at 30 °C with a 5 μL injection volume, using the gradient elution with aqueous acetonitrile containing 0.1% formic acid: 20–50% over 10 min followed by 50–90% for the next 10 min. All tested purchased compounds were verified to be >95% pure using the method described above with the following modifications: Agilent Infinity II HPLC system coupled to an Agilent 6120 ESI mass spectrometer was used; the range was from 100 to 1000 <i>m</i>/<i>z</i>; flow rate was 0.8 mL/min; and gradient elution was done with aqueous acetonitrile containing 0.1% trifluoroacetic acid, 10–100% over 7 min followed by 100% for the next 8 min.</div></div><div id="sec4-1-2" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> General Procedure A for the Synthesis of Compounds <b>13</b>, <b>14</b>, <b>23</b>–<b>25</b>, <b>27</b>, <b>28</b>, <b>38</b>–<b>43</b>, <b>46</b>–<b>48</b>, <b>52</b>, <b>55</b>, <b>60</b>, <b>69</b>–<b>71</b></h3><div class="NLM_p last">To a stirred solution of chloropyrimidine <b>2a</b>,<b>d</b>,<b>e</b>,<b>h</b> or <b>3d</b>–<b>f</b>,<b>h</b> (1 equiv) and substituted aniline (1 equiv) in EtOH (4 mL/mmol) was added concentrated HCl (0.1 mL/mmol). The solution was then heated to reflux for 18 h. After cooling to rt, the volatiles were concentrated in vacuo. The residual oil was partitioned between EtOAc (4 mL/mmol) and 10% aqueous NaOH (4 mL/mmol). The layers were separated, and the aqueous layer was extracted with EtOAc (3×; 4 mL/mmol). The combined organic layers were dried (Na<sub>2</sub>SO<sub>4</sub>) and concentrated. The resulting residue was purified by flash chromatography.</div></div><div id="sec4-1-3" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> General Procedure B for the Synthesis of Compounds <b>18</b>–<b>20</b>, <b>22</b></h3><div class="NLM_p last">The appropriate chloropyridine <b>3a</b>,<b>c</b>,<b>i</b>,<b>j</b> (1 equiv) was dissolved in <i>n</i>-BuOH (1.23 mL/mmol chloropyridine) in a microwave vial before 3,5-dichloroaniline (1.5 equiv) was added followed by concentrated HCl (0.154 mL/mmol chloropyridine). The vial was sealed and the solution was heated in the microwave at 185 °C for 4 h before being cooled to rt and diluted with 30 mL of EtOAc (4.6 mL/mmol chloropyridine). 10% aqueous NaOH (4.6 mL/mmol chloropyridine) was added, the layers were separated, and the aqueous layer was extracted with EtOAc (3×; 2.3 mL/mmol chloropyridine). The organic layers were combined, dried (Na<sub>2</sub>SO<sub>4</sub>), and concentrated in vacuo. The resulting residue was purified by flash chromatography.</div></div><div id="sec4-1-4" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> General Procedure C for the Synthesis of Compounds <b>9</b>, <b>10</b>, <b>15</b>, <b>16</b>, <b>26</b>, <b>32</b>–<b>34</b>, <b>36</b>, <b>37</b>, <b>44</b>, <b>45</b>, <b>53</b>, <b>54</b>, <b>61</b>–<b>68</b>, <b>72</b>, <b>73</b>, <b>89</b></h3><div class="NLM_p last">The appropriate aniline (1 equiv) and chloropyrimidine <b>2a</b>–<b>c</b>,<b>f</b>–<b>h</b> or <b>3g</b> (1 equiv) were dissolved in EtOH (1.03 mL/mmol) containing concentrated HCl (0.129 mL/mmol) in a microwave vial. The vial was sealed and the solution was heated in the microwave at 140 °C for 1.5 h before being cooled to rt and diluted with 10 mL of EtOAc (5.15 mL/mmol). 10% aqueous NaOH (5.15 mL/mmol) was added, the layers were separated, and the aqueous layer was extracted with EtOAc (3×; 5.15 mL/mmol). The organic layers were combined, dried (Na<sub>2</sub>SO<sub>4</sub>), and concentrated in vacuo. The resulting residue was purified by flash chromatography.</div></div><div id="sec4-1-5" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> General Procedure D for the Synthesis of Compounds <b>11</b>, <b>50</b>, <b>91</b>–<b>109</b>, <b>116</b>–<b>118</b></h3><div class="NLM_p last">To a solution of sodium carbonate (3 equiv) in water (1 mL/mmol carbonate) and DME (3.33 mL/mmol carbonate) were added aryl chloride <b>7</b> or <b>13</b> or aryl bromide <b>32</b> or <b>38</b> (1 equiv), phenylboronic acid (1.1 equiv) (note that an amount of 2.2 equiv is used for bis-arylation), and PPh<sub>3</sub> (0.244 equiv). The solution was evacuated and backfilled with Ar 5× before Pd(OAc)<sub>2</sub> (0.122 equiv) was added. The solution was again evacuated and backfilled with Ar 5× before being heated to 90 °C for 18 h. The solution was cooled to rt and filtered through Celite. The filtrate was diluted with EtOAc (29 mL/mmol aryl halide), and the solution was washed with 10% aqueous NaOH followed by brine. The organic layer was dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated in vacuo to afford a residue that was purified by flash chromatography.</div></div><div id="sec4-1-6" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> General Procedure E for the Synthesis of Compounds <b>110</b>–<b>115</b></h3><div class="NLM_p last">To a solution of sodium triphosphate (7.48 equiv) in water/dioxane (1:4; 13 mL/mmol <b>32</b>) were added bromopyrimidine <b>32</b> (1 equiv) and PPh<sub>3</sub> (0.3 equiv). The solution was evacuated and backfilled with Ar 5× before the appropriate heteroarylboronic acid MIDA ester (1.19 equiv) was added. The solution was again evacuated and backfilled with Ar 5×, and Pd(OAc)<sub>2</sub> (0.145 equiv) was added. The solution was again evacuated and backfilled with Ar 5× before being heated to 90 °C for 18 h. The solution was cooled to rt and filtered through Celite. The filtrate was diluted with EtOAc (48 mL/mmol <b>32</b>), and the solution was washed with 10% aqueous NaOH followed by brine. The organic layer was dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated in vacuo to afford a residue that was purified by flash chromatography.</div></div><div id="sec4-2" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> <i>N</i><sup>2</sup>-(3,5-Dichlorophenyl)pyridine-2,4-diamine (<b>18</b>)</h3><div class="NLM_p last">According to general procedure B, 2-chloropyridin-4-amine (1.0 g, 7.76 mmol) and 3,5-dichloroaniline (2.50 g, 15.5 mmol) yielded compound <b>18</b> as an off-white solid (0.75 g, 39%); mp >250 °C. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.95 (s, 1H), 7.73 (d, <i>J</i> = 5.5 Hz, 1H), 7.06 (d, <i>J</i> = 5.5 Hz, 1H), 6.97 (d, <i>J</i> = 1.8 Hz, 2H), 6.93 (t, <i>J</i> = 1.8 Hz, 1H). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ 144.6, 138.6, 136.3, 135.5, 135.2, 121.6, 120.3, 116.0, 111.9. LC/MS (ESI) calcd for C<sub>11</sub>H<sub>10</sub>Cl<sub>2</sub>N<sub>3</sub> (M + H)<sup>+</sup> 254.0, found 254.0.</div></div><div id="sec4-3" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> <i>N</i><sup>2</sup>-(3,5-Dichlorophenyl)-6-methylpyridine-2,4-diamine (<b>19</b>)</h3><div class="NLM_p last">*According to general procedure B, 2-chloro-6-methylpyridin-4-amine (0.18 g, 1.28 mmol) and 3,5-dichloroaniline (0.41 g, 2.56 mmol) yielded compound <b>19</b> as a purple oil (0.07 g, 20%). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.26 (t, <i>J</i> = 1.8 Hz, 1H), 7.22 (d, <i>J</i> = 1.8 Hz, 2H), 6.21 (d, <i>J</i> = 2.1, Hz, 1H), 6.16 (d, <i>J</i> = 2.1 Hz, 1H), 2.33 (s, 3H). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ 156.9, 156.1, 155.1, 144.9, 134.6, 118.8, 115.3, 103.1, 92.0, 22.4. LC/MS (ESI) calcd for C<sub>12</sub>H<sub>11</sub>Cl<sub>2</sub>N<sub>3</sub> (M + H)<sup>+</sup> 268.0, found 268.0.</div></div><div id="sec4-4" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> <i>N</i><sup>4</sup>-(3,5-Dichlorophenyl)-6-methylpyridine-2,4-diamine (<b>20</b>)</h3><div class="NLM_p last">According to general procedure B, 4-chloro-6-methylpyridin-2-amine (0.25 g, 1.75 mmol) and 3,5-dichloroaniline (0.426 g, 2.62 mmol) yielded compound <b>20</b> as an off-white foam (0.09 g, 19%). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.26 (t, <i>J</i> = 1.8 Hz, 1H), 7.22 (d, <i>J</i> = 1.8 Hz, 2H), 6.21 (d, <i>J</i> = 2.7 Hz, 1H), 6.16 (d, <i>J</i> = 2.7 Hz, 1H), 2.33 (s, 3H). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>CN) δ 154.9, 148.3, 141.1, 136.2, 135.3, 124.3, 120.6, 102.1, 88.2, 18.6. LC/MS (ESI) calcd for C<sub>11</sub>H<sub>12</sub>Cl<sub>2</sub>N<sub>3</sub> (M + H)<sup>+</sup> 268.1, found 268.1.</div></div><div id="sec4-5" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> <i>N</i><sup>4</sup>-(3,5-Dichlorophenyl)pyridine-2,4-diamine (<b>21</b>)</h3><div class="NLM_p last">4-Chloropyridin-2-amine (1.00 g, 7.76 mmol) and 3,5-dichloroaniline (1.23 g, 7.76 mmol) were dissolved in 22 mL of a 50:50 mixture of MeOH/H<sub>2</sub>O in a sealed tube. To this solution was added 1 mL of concentrated HCl, and the tube was sealed and heated to 100 °C for 18 h. After cooling to rt, the volatiles were concentrated in vacuo. The residual oil was partitioned between 30 mL of EtOAc and 30 mL of 10% NaOH. The layers were separated, and the aqueous layer was extracted with 3 × 30 mL of EtOAc. The combined organic layers were dried (Na<sub>2</sub>SO<sub>4</sub>) and concentrated. The resulting oil was purified by flash chromatography (0–10% MeOH/DCM) to give amine <b>21</b> as an off-white foam (0.41 g, 21%). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.66 (d, <i>J</i> = 5.9 Hz, 1H), 7.10 (s, 2H), 7.00 (s, 1H), 6.30–6.24 (m, 2H). <sup>13</sup>C NMR (101 MHz, methanol-<i>d</i><sub>4</sub>) δ 160.3, 151.2, 146.7, 143.7, 135.2, 121.1, 117.3, 102.6, 92.4. LC/MS (ESI) calcd for C<sub>11</sub>H<sub>10</sub>Cl<sub>2</sub>N<sub>3</sub> (M + H)<sup>+</sup> 254.0, found 254.0.</div></div><div id="sec4-6" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> <i>N</i><sup>4</sup>-(3,5-Dichlorophenyl)pyridine-3,4-diamine (<b>22</b>)</h3><div class="NLM_p last">According to general procedure B, 4-chloropyridin-3-amine (0.835 g, 6.50 mmol) and 3,5-dichloroaniline (1.58 g, 9.73 mmol) yielded compound <b>22</b> as a yellow solid (0.53 g, 32%). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 7.95 (s, 1H), 7.73 (d, <i>J</i> = 5.5 Hz, 1H), 7.06 (d, <i>J</i> = 5.5 Hz, 1H), 6.97 (d, <i>J</i> = 1.8 Hz, 2H), 6.93 (t, <i>J</i> = 1.8 Hz, 1H). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ 144.6, 138.6, 136.3, 135.5, 135.2, 121.6, 120.3, 116.0, 111.9.</div></div><div id="sec4-7" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> <i>N</i>-(3,5-Dichlorophenyl)-6-methylpyrimidin-4-amine (<b>23</b>)</h3><div class="NLM_p last">According to general procedure A, 4-chloro-6-methylpyrimidine (0.69 g, 5.37 mmol) and 3,5-dichloroaniline (0.87 g, 5.37 mmol) yielded compound <b>23</b> (0.86 g, 63%) as a white foam. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.55 (s, 1H), 7.74 (s, 2H), 7.03 (s, 1H), 6.62 (s, 1H), 2.37 (s, 3H). <sup>13</sup>C NMR (101 MHz, methanol-<i>d</i><sub>4</sub>) δ 164.7, 160.6, 157.0, 142.0, 134.6, 121.5, 117.5, 105.9, 21.9. LC/MS (ESI) calcd for C<sub>11</sub>H<sub>10</sub>Cl<sub>2</sub>N<sub>3</sub> (M + H)<sup>+</sup> 254.0, found 254.0.</div></div><div id="sec4-8" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> <i>N</i><sup>4</sup>-(3,5-Dichlorophenyl)-<i>N</i><sup>2</sup>,6-dimethylpyrimidine-2,4-diamine (<b>24</b>)</h3><div class="NLM_p last">According to general procedure A, 4-chloro-<i>N</i>,6-dimethylpyrimidin-2-amine (0.53 g, 3.37 mmol) and 3,5-dichloroaniline (0.55 g, 3.37 mmol) yielded compound <b>24</b> (0.56 g, 67%) as an off-white solid; mp 200–201 °C. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.81 (s, 2H), 6.95 (s, 1H), 5.87 (s, 1H), 2.94 (s, 3H), 2.18 (s, 3H). <sup>13</sup>C NMR (101 MHz, methanol-<i>d</i><sub>4</sub>) δ 164.8, 162.1, 161.4, 142.7, 134.4, 120.6, 117.1, 95.1, 27.2, 21.7. LC/MS (ESI) calcd for C<sub>12</sub>H<sub>13</sub>Cl<sub>2</sub>N<sub>4</sub> (M + H)<sup>+</sup> 283.1, found 283.1.</div></div><div id="sec4-9" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> <i>N</i><sup>4</sup>-(3,5-Dichlorophenyl)-<i>N</i><sup>2</sup>,<i>N</i><sup>2</sup>,6-trimethylpyrimidine-2,4-diamine (<b>25</b>)</h3><div class="NLM_p last">According to general procedure A, 4-chloro-<i>N</i>,<i>N</i>,6-trimethylpyrimidin-2-amine (0.86 g, 4.98 mmol) and 3,5-dichloroaniline (0.81 g, 4.98 mmol) yielded compound <b>25</b> (0.63 g, 48%) as an off-white foam. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.72 (s, 2H), 6.85 (s, 1H), 5.77 (s, 1H), 3.10 (s, 6H), 2.17 (s, 3H). <sup>13</sup>C NMR (101 MHz, methanol-<i>d</i><sub>4</sub>) δ 165.4, 161.8, 160.6, 142.9, 134.3, 120.1, 116.7, 94.6, 36.4, 22.4. LC/MS (ESI) calcd for C<sub>13</sub>H<sub>15</sub>Cl<sub>2</sub>N<sub>4</sub> (M + H)<sup>+</sup> 297.1, found 297.1.</div></div><div id="sec4-10" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> <i>N</i><sup>4</sup>-(3,5-Dichlorophenyl)pyrimidine-2,4,6-triamine (<b>26</b>)</h3><div class="NLM_p last">According to general procedure C, 6-chloropyrimidine-2,4-diamine (1.00 g, 6.91 mmol) and 3,5-dichloroaniline (1.46 g, 8.99 mmol) yielded compound <b>26</b> as an off-white solid (0.95 g, 51%); mp 119–120 °C. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.97 (s, 1H), 7.75 (d, <i>J</i> = 1.8 Hz, 2H), 6.95 (t, <i>J</i> = 1.8 Hz, 1H), 6.04 (s, 2H), 5.92 (s, 2H), 5.21 (s, 1H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.0, 163.1, 161.2, 144.7, 134.2, 119.3, 116.6, 78.1. LC/MS (ESI) calcd for C<sub>10</sub>H<sub>10</sub>Cl<sub>2</sub>N<sub>5</sub> (M + H)<sup>+</sup> 270.0, found 270.0.</div></div><div id="sec4-11" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> <i>N</i><sup>4</sup>-(3-Bromo-5-chlorophenyl)-2-methylpyrimidine-4,6-diamine (<b>27</b>)</h3><div class="NLM_p last">According to general procedure A, 6-chloro-2-methylpyrimidin-4-amine (0.50 g, 3.48 mmol) and 3-chloro-5-bromoaniline (0.94 g, 4.52 mmol) yielded compound <b>27</b> as a white solid (0.82 g, 75%); mp 178–179 °C. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.69 (t, <i>J</i> = 1.8 Hz, 1H), 7.61 (t, <i>J</i> = 1.8 Hz, 1H), 7.10 (t, <i>J</i> = 1.8 Hz, 1H), 5.72 (s, 1H), 2.34 (s, 3H). <sup>13</sup>C NMR (101 MHz, methanol-<i>d</i><sub>4</sub>) δ 166.1, 163.6, 160.5, 143.0, 134.7, 123.6, 122.1, 120.3, 117.8, 82.8, 23.7. LC/MS (ESI) calcd for C<sub>11</sub>H<sub>10</sub>BrClN<sub>4</sub> (M + H)<sup>+</sup> 313.0, found 313.0.</div></div><div id="sec4-12" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> <i>N</i><sup>4</sup>-(3,5-Dichlorophenyl)-<i>N</i><sup>4</sup>,6-dimethylpyrimidine-2,4-diamine (<b>28</b>)</h3><div class="NLM_p last">According to general procedure A, pyrimidineamine (0.56 g, 2.83 mmol) and 3,5-dichloro-<i>N</i>-methylaniline (0.41 g, 2.83 mmol) yielded compound <b>28</b> (0.47 g, 59%) as an off-white solid; mp 156–157 °C. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.39 (t, <i>J</i> = 1.9 Hz, 1H), 7.30 (d, <i>J</i> = 1.9 Hz, 2H), 5.73 (s, 1H), 3.38 (s, 3H), 2.13 (s, 3H). <sup>13</sup>C NMR (101 MHz methanol-<i>d</i><sub>4</sub>) δ 164.3, 163.5, 161.9, 147.0, 135.4, 126.3, 125.7, 93.7, 37.0, 21.6. LC/MS (ESI) calcd for C<sub>12</sub>H<sub>13</sub>Cl<sub>2</sub>N<sub>4</sub> (M + H)<sup>+</sup> 283.1, found 283.1.</div></div><div id="sec4-13" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> 4-((3,5-Dichlorophenyl)thio)-6-methylpyrimidin-2-amine (<b>29</b>)</h3><div class="NLM_p last">To a stirred solution of 4-chloro-6-methylpyrimidin-2-amine (0.45 g, 3.11 mmol) and 3,5-dichlorothiophenol (0.56 g, 3.11 mmol) in 6.2 mL of DMF was added Et<sub>3</sub>N (0.87 mL, 6.22 mmol). The solution was stirred at rt for 24 h The mixture was partitioned between 20 mL of EtOAc and 20 mL of H<sub>2</sub>O. The layers were separated, and the aqueous layer was extracted with 3 × 20 mL of EtOAc. The combined organic layers were dried (Na<sub>2</sub>SO<sub>4</sub>) and concentrated. The resulting dark oil was purified by flash chromatography (0–50% EtOAc/hexanes) to afford thioether <b>29</b> as a white foam (0.58 g, 65%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.44 (d, <i>J</i> = 1.9 Hz, 2H), 7.39 (t, <i>J</i> = 1.9 Hz, 1H), 6.04 (s, 1H), 5.59 (br s, 2H), 2.18 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 170.2, 167.63 162.2, 135.5, 133.2, 131.9, 129.7, 106.9, 23.8. LC/MS (ESI) calcd for C<sub>11</sub>H<sub>10</sub>Cl<sub>2</sub>N<sub>3</sub>S (M + H)<sup>+</sup> 286.0, found 286.0.</div></div><div id="sec4-14" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> <i>N</i><sup>4</sup>-(3,5-Dichlorobenzyl)-6-methylpyrimidine-2,4-diamine (<b>30</b>)</h3><div class="NLM_p last">4-Chloro-6-methylpyrimidin-2-amine (0.68 g, 4.73 mmol) and (3,5-dichlorophenyl)methanamine (1.00 g, 5.67 mmol) were charged into a microwave vial followed by DIPEA (1.65 mL, 9.46 mmol) and 8 mL of CH<sub>3</sub>CN. The vial was capped and heated to 140 °C for 2 h in the microwave. After cooling to rt, the volatiles were concentrated in vacuo. The residual oil was partitioned between 20 mL of EtOAc and 20 mL of 10% NaOH. The layers were separated, and the aqueous layer was extracted with 3 × 20 mL of EtOAc. The combined organic layers were dried (Na<sub>2</sub>SO<sub>4</sub>) and concentrated. The resulting dark oil was purified by flash chromatography (0–10% MeOH/DCM) to give amine <b>30</b> as a white solid (0.66 g, 49%); mp 129–130 °C. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.17 (app s, 3H), 5.69 (s, 1H), 4.44 (s, 2H), 2.06 (s, 3H). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ 164.3, 163.8, 162.7, 143.8, 134.6, 126.4, 125.4, 94.2, 47.3, 42.7, 21.0. LC/MS (ESI) calcd for C<sub>12</sub>H<sub>13</sub>Cl<sub>2</sub>N<sub>4</sub> (M + H)<sup>+</sup> 283.0, found 283.0.</div></div><div id="sec4-15" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> <i>N</i><sup>4</sup>-(3,5-Dichlorophenethyl)-6-methylpyrimidine-2,4-diamine (<b>31</b>)</h3><div class="NLM_p last">4-Chloro-6-methylpyrimidin-2-amine (0.22 g, 1.53 mmol) and 2-(3,5-dichlorophenyl)ethan-1-amine (0.35 g, 1.83 mmol) were charged into a microwave vial followed by DIPEA (0.5 mL, 3.06 mmol) and 4 mL of CH<sub>3</sub>CN. The vial was capped and heated to 140 °C for 2 h in the microwave. After cooling to rt, the volatiles were concentrated in vacuo. The residual oil was partitioned between 20 mL of EtOAc and 20 mL of 10% NaOH. The layers were separated, and the aqueous layer was extracted with 3 × 20 mL of EtOAc. The combined organic layers were dried (Na<sub>2</sub>SO<sub>4</sub>) and concentrated. The resulting dark oil was purified by flash chromatography (0–10% MeOH/DCM) to give amine <b>31</b> as a clear oil (0.29 g, 64%). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>CN) δ 7.30 (d, <i>J</i> = 1.9 Hz, 1H), 7.22 (t, <i>J</i> = 1.9 Hz, 2H), 6.09 (s, 1H), 5.71 (s, 1H), 4.08 (br s, 2H), 3.52 (q, <i>J</i> = 6.7 Hz, 2H), 2.84 (t, <i>J</i> = 6.7 Hz, 2H), 2.10 (s, 3H). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>CN) δ 163.9, 143.8, 134.3, 134.3, 127.6, 126.0, 125.9, 93.5, 41.2, 34.6, 34.4. LC/MS (ESI) calcd for C<sub>13</sub>H<sub>15</sub>Cl<sub>2</sub>N<sub>4</sub> (M + H)<sup>+</sup> 297.1, found 297.1.</div></div><div id="sec4-16" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> <i>N</i><sup>4</sup>-(3-Bromo-5-chlorophenyl)-6-methylpyrimidine-2,4-diamine (<b>32</b>)</h3><div class="NLM_p last">According to general procedure C, 4-chloro-6-methylpyrimidin-2-amine (1.32 g, 8.07 mmol) and 3-bromo-5-chloroaniline (2.00 g, 9.68 mmol) yielded compound <b>32</b> as a white solid (2.20 g, 87%); mp 155–156 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.28 (s, 1H), 7.89 (t, <i>J</i> = 1.9 Hz, 1H), 7.84 (t, <i>J</i> = 1.9 Hz, 1H), 7.13 (t, <i>J</i> = 1.9 Hz, 1H), 6.32 (s, 2H), 5.83 (s, 1H), 2.08 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.3, 163.2, 161.2, 144.0, 134.5 123.0, 122.4, 119.9, 117.5, 95.9, 23.9. LC/MS (ESI) calcd for C<sub>11</sub>H<sub>11</sub>BrClN<sub>4</sub> (M + H)<sup>+</sup> 313.0, found 313.0.</div></div><div id="sec4-17" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> 3-((2-Amino-6-methylpyrimidin-4-yl)amino)-5-chlorobenzonitrile (<b>33</b>)</h3><div class="NLM_p last">According to general procedure C, 4-chloro-6-methylpyrimidin-2-amine (0.47 g, 2.73 mmol) and 3-chloro-5-cyanoaniline (0.50 g, 3.27 mmol) yielded compound <b>33</b> as an off-white solid (0.52 g, 73%); mp 270–271 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.46 (s, 1H), 8.21 (t, <i>J</i> = 1.8 Hz, 1H), 8.01 (t, <i>J</i> = 1.8 Hz, 1H), 7.45 (t, <i>J</i> = 1.8 Hz, 1H), 6.41 (s, 2H), 5.85 (s, 1H), 2.09 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.6, 163.2, 161.1, 143.6, 134.4, 123.7, 122.7, 120.4, 118.3, 113.5, 95.9, 24.0. LC/MS (ESI) calcd for C<sub>11</sub>H<sub>11</sub>ClN<sub>5</sub> (M + H)<sup>+</sup> 260.1, found 260.1.</div></div><div id="sec4-18" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> <i>N</i><sup>4</sup>-(3-Chloro-5-(trifluoromethyl)phenyl)-6-methylpyrimidine-2,4-diamine (<b>34</b>)</h3><div class="NLM_p last">According to general procedure C, 4-chloro-6-methylpyrimidin-2-amine (0.38 g, 2.64 mmol) and 3-chloro-5-trifluoromethylaniline (0.62 g, 3.17 mmol) yielded compound <b>34</b> as a white solid (0.62 g, 81%); mp 159–160 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.47 (s, 1H), 8.30 (t, <i>J</i> = 1.9 Hz, 1H), 7.84 (t, <i>J</i> = 1.9 Hz, 1H), 7.25 (t, <i>J</i> = 1.9 Hz, 1H), 6.34 (br s, 2H), 5.85 (s, 1H), 2.09 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO- <i>d</i><sub>6</sub>) δ 166.5, 163.2, 161.2, 143.6, 134.6, 131.3 (q, <i>J</i><sub>CF</sub> = 32.0 Hz), 125.2, 121.7, 117.0, 113.9, 95.9, 23.9. LC/MS (ESI) calcd for C<sub>12</sub>H<sub>11</sub>ClF<sub>3</sub>N<sub>4</sub> (M + H)<sup>+</sup> 303.1, found 303.1.</div></div><div id="sec4-19" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> <i>N</i><sup>4</sup>-(5-Chloro-[1,1′-biphenyl]-3-yl)-6-methylpyrimidine-2,4-diamine (<b>35</b>)</h3><div class="NLM_p last">According to general procedure D, bromopyrimidineamine <b>32</b> (0.50 g, 1.59 mmol) and phenylboronic acid (0.21 g, 1.75 mmol) yielded compound <b>35</b> as an off-white foam (0.33 g, 67%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.53–7.50 (m, 2H), 7.45–7.40 (m, 3H), 7.39–7.34 (m, 2H), 7.28 (m, 1H), 6.86 (s, 1H), 5.99 (s, 1H), 4.98 (s, 2H), 2.22 (s, 3H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 167.6, 162.7, 161.6, 143.7, 140.4, 139.4, 135.0, 128.9, 128.18, 127.0, 122.6, 120.1, 118.3, 95.0, 24.0. LC/MS (ESI) calcd for C<sub>17</sub>H<sub>16</sub>ClN<sub>4</sub> (M + H)<sup>+</sup> 311.1, found 311.1.</div></div><div id="sec4-20" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> <i>N</i><sup>4</sup>-(3-Chloro-5-(pentafluoro-λ<sup>6</sup>-sulfaneyl)phenyl)-6-methylpyrimidine-2,4-diamine (<b>36</b>)</h3><div class="NLM_p last">According to general procedure C, 4-chloro-6-methylpyrimidin-2-amine (0.14 g, 0.97 mmol) and 3-chloro-5-(pentafluorosulfanyl)aniline (0.30 g, 1.18 mmol) yielded compound <b>36</b> as a white solid (0.22 g, 63%); mp 194–195 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.55 (s, 1H), 8.43 (s, 1H), 7.95 (s, 1H), 7.39 (s, 1H), 6.40 (s, 2H), 5.86 (s, 1H), 2.10 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.6, 163.1, 161.2, 153.7 (sext, <i>J</i><sub>CF</sub> = 17.1 Hz), 143.3, 134.1, 121.6, 117.4 114.5, 96.0, 23.8. LC/MS (ESI) calcd for C<sub>11</sub>H<sub>11</sub>ClF<sub>5</sub>N<sub>4</sub>S (M + H)<sup>+</sup> 361.1, found 361.1.</div></div><div id="sec4-21" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> <i>N</i><sup>4</sup>-(3-Bromo-5-(pentafluoro-λ<sup>6</sup>-sulfaneyl)phenyl)-6-methylpyrimidine-2,4-diamine (<b>37</b>)</h3><div class="NLM_p last">According to general procedure C, 4-chloro-6-methylpyrimidin-2-amine (0.09 g, 0.58 mmol) and 3-bromo-5-(pentafluorosulfanyl)aniline (0.21 g, 0.70 mmol) yielded compound <b>37</b> as a white solid (0.21 g, 77%); mp 175–176 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.54 (s, 1H), 8.48 (t, <i>J</i> = 1.8 Hz, 1H), 8.05 (t, <i>J</i> = 1.8 Hz, 1H), 7.54 (t, <i>J</i> = 1.8 Hz, 1H), 6.36 (s, 2H), 5.86 (s, 1H), 2.11 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.6, 163.1, 161.2, 143.4, 124.5, 122.1, 120.1, 120.0, 114.9, 96.0, 23.9. LC/MS (ESI) calcd for C<sub>11</sub>H<sub>11</sub>BrF<sub>5</sub>N<sub>4</sub>S (M + H)<sup>+</sup> 405.0, found 405.0.</div></div><div id="sec4-22" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> <i>N</i><sup>4</sup>-(3-Bromo-5-ethylphenyl)-6-methylpyrimidine-2,4-diamine (<b>38</b>)</h3><div class="NLM_p last">According to general procedure A, 4-chloro-6-methylpyrimidin-2-amine (0.28 g, 1.92 mmol) and 3-bromo-5-ethylaniline (0.50 g, 2.50 mmol) yielded compound <b>38</b> as an off-white foam (0.48 g, 81%). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.75 (t, <i>J</i> = 1.8 Hz, 1H), 7.30 (t, <i>J</i> = 1.8 Hz,1H), 6.96 (t, <i>J</i> = 1.8 Hz, 1H), 5.90 (s, 1H), 2.57 (q, <i>J</i> = 7.6 Hz, 2H), 1.19 (t, <i>J</i> = 7.6 Hz, 3H). <sup>13</sup>C NMR (101 MHz, methanol-<i>d</i><sub>4</sub>) δ 164.7, 162.5, 161.9, 146.7, 141.3, 124.4, 121.7, 120.0, 118.0, 95.6, 28.2, 21.8, 14.5. LC/MS (ESI) calcd for C<sub>13</sub>H<sub>16</sub>BrN<sub>4</sub> (M + H)<sup>+</sup> 307.1, found 307.1.</div></div><div id="sec4-23" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> <i>N</i><sup>4</sup>-(3-Bromo-5-ethoxyphenyl)-6-methylpyrimidine-2,4-diamine (<b>39</b>)</h3><div class="NLM_p last">According to general procedure A, 4-chloro-6-methylpyrimidin-2-amine (0.15 g, 1.07 mmol) and 3-bromo-5-ethoxyaniline (0.35 g, 1.29 mmol) yielded compound <b>39</b> as an orange oil (0.25 g, 71%). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.35 (t, <i>J</i> = 1.9 Hz, 1H), 7.27 (t, <i>J</i> = 1.9 Hz, 1H), 6.66 (t, <i>J</i> = 1.9 Hz, 1H), 5.90 (s, 1H), 3.98 (q, <i>J</i> = 7.0 Hz, 2H), 2.15 (s, 3H), 1.35 (t, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (101 MHz, methanol-<i>d</i><sub>4</sub>) δ 164.9, 162.6, 161.9, 160.0, 142.2, 122.0, 114.7, 111.0, 104.9, 95.1, 63.4, 21.8, 13.6. LC/MS (ESI) calcd for C<sub>13</sub>H<sub>16</sub>BrN<sub>4</sub>O (M + H)<sup>+</sup> 323.0, found 323.0.</div></div><div id="sec4-24" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> <i>N</i><sup>4</sup>-(3-Bromo-5-(2,2,2-trifluoroethoxy)phenyl)-6-methylpyrimidine-2,4-diamine (<b>40</b>)</h3><div class="NLM_p last">According to general procedure A, 4-chloro-6-methylpyrimidin-2-amine (0.12 g, 0.86 mmol) and 3-bromo-5-(2,2,2-trifluoroethoxy)aniline (0.35 g, 1.29 mmol) yielded compound <b>40</b> as a white solid (0.21 g, 65%); mp 94–95 °C. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.53–7.44 (m, 2H), 6.87 (s, 1H), 5.96 (s, 1H), 4.54 (q, <i>J</i><sub>HF</sub> = 8.6 Hz, 2H), 2.22 (s, 3H). <sup>13</sup>C NMR (101 MHz, methanol-<i>d</i><sub>4</sub>) δ 162.1, 159.9, 158.3, 141.6, 125.0, 122.2, 116.7, 112.1, 105.8, 96.4, 65.2 (q, <i>J</i><sub><i>CF</i></sub> = 35.4 Hz), 19.7. LC/MS (ESI) calcd for C<sub>13</sub>H<sub>13</sub>BrF<sub>3</sub>N<sub>4</sub>O (M + H)<sup>+</sup> 377.0, found 377.0.</div></div><div id="sec4-25" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> <i>N</i><sup>4</sup>-(3-(Allyloxy)-5-bromophenyl)-6-methylpyrimidine-2,4-diamine (<b>41</b>)</h3><div class="NLM_p last">According to general procedure A, 4-chloro-6-methylpyrimidin-2-amine (0.15 g, 1.01 mmol) and 3-(allyloxy)-5-bromoaniline (0.30 g, 1.32 mmol) yielded compound <b>41</b> as an off-white foam (0.25 g, 73%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 8.38 (s, 1H), 7.57–7.43 (m, 2H), 6.70 (s, 1H), 6.01 (m, 1H), 5.93–5.86 (m 3H), 5.40 (dd, <i>J</i> = 17.4, 1.8 Hz, 1H), 5.24 (dd, <i>J</i> = 10.6, 1.8 Hz, 1H), 4.57 (d, <i>J</i> = 5.5 Hz, 2H), 2.10 (s, 3H). <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 166.2, 163.2, 161.6, 159.8, 143.2, 133.4, 121.9, 116.8, 114.5, 110.9, 104.8, 95.5, 68.6, 22.9. LC/MS (ESI) calcd for C<sub>14</sub>H<sub>16</sub>BrN<sub>4</sub>O (M + H)<sup>+</sup> 335.0, found 335.0.</div></div><div id="sec4-26" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> <i>N</i><sup>4</sup>-(3-Bromo-5-butoxyphenyl)-6-methylpyrimidine-2,4-diamine (<b>42</b>)</h3><div class="NLM_p last">According to general procedure A, 4-chloro-6-methylpyrimidin-2-amine (0.14 g, 0.94 mmol) and 3-bromo-5-butoxyaniline (0.30 g, 1.23 mmol) yielded compound <b>42</b> as a tan oil (0.2 g, 61%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 8.35 (s, 1H), 7.53 (s, 1H), 7.37 (s, 1H), 6.68 (s, 1H), 5.94 (s, 1H), 5.85 (s, 2H), 3.99 (t, <i>J</i> = 5.6 Hz, 2H), 2.11 (s, 3H), 1.81–1.63 (m, 2H), 1.56–1.41 (m, 2H), 0.95 (t, <i>J</i> = 5.8 Hz, 3H). <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 166.2, 163.2, 161.7, 160.3, 143.2, 122.0, 114.3, 110.7, 104.5, 95.5, 67.7, 31.1, 22.9, 18.9, 13.2. LC/MS (ESI) calcd for C<sub>15</sub>H<sub>20</sub>BrN<sub>4</sub>O (M + H)<sup>+</sup> 351.1, found 351.1.</div></div><div id="sec4-27" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> <i>N</i><sup>4</sup>-(3-Bromo-5-isobutoxyphenyl)-6-methylpyrimidine-2,4-diamine (<b>43</b>)</h3><div class="NLM_p last">According to general procedure A, 4-chloro-6-methylpyrimidin-2-amine (0.14 g, 0.99 mmol) and 3-bromo-5-isobutoxyaniline (0.30 g, 1.22 mmol) yielded compound <b>43</b> as a tan oil (0.22 g, 65%). <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 8.38 (s, 1H), 7.56 (s, 1H), 7.33 (s, 1H), 6.69 (s, 1H), 6.09–5.84 (m, 3H), 3.74 (d, <i>J</i> = 6.8 Hz, 2H), 2.11 (s, 3H), 2.04 (m, 1H), 0.99 (d, <i>J</i> = 6.8 Hz, 6H). <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 166.2, 163.3, 161.7, 160.4, 143.1, 122.0, 114.4, 110.8, 104.6, 95.5, 74.3, 28.0, 22.9, 18.5. LC/MS (ESI) calcd for C<sub>15</sub>H<sub>20</sub>BrN<sub>4</sub>O (M + H)<sup>+</sup> 351.1, found 351.1.</div></div><div id="sec4-28" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> <i>N</i><sup>4</sup>-(3,5-Dibromophenyl)-6-methylpyrimidine-2,4-diamine (<b>44</b>)</h3><div class="NLM_p last">According to general procedure C, 4-chloro-6-methylpyrimidin-2-amine (1.08 g, 6.64 mmol) and 3,5-dibromoaniline (2.00 g, 7.97 mmol) yielded compound <b>44</b> as an off-white solid (1.92 g, 81%); mp 179–180 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.25 (s, 1H), 7.95 (d, <i>J</i> = 1.8 Hz, 2H), 7.21 (t, <i>J</i> = 1.8 Hz, 1H), 6.39 (s, 2H), 5.85 (s, 1H), 2.08 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.3, 163.2, 161.3, 144.1, 125.6, 122.6, 120.3, 96.0, 23.8. LC/MS (ESI) calcd for C<sub>11</sub>H<sub>11</sub>Br<sub>2</sub>N<sub>4</sub> (M + H)<sup>+</sup> 357.0, found 357.0.</div></div><div id="sec4-29" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> <i>N</i><sup>4</sup>-(3,5-Bis(methylsulfonyl)phenyl)-6-methylpyrimidine-2,4-diamine (<b>45</b>)</h3><div class="NLM_p last">According to general procedure C, 4-chloro-6-methylpyrimidin-2-amine (0.24 g, 1.67 mmol) and 3,5-bis(methylsulfonyl)aniline (0.50 g, 2.00 mmol) yielded compound <b>45</b> as a white solid (0.35 g, 59%); mp >290 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.80 (s, 1H), 8.50 (t, <i>J</i> = 1.5 Hz, 2H), 7.86 (d, <i>J</i> = 1.5 Hz, 1H), 6.34 (s, 2H), 5.91 (s, 1H), 3.33 (s, 6H), 2.12 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.8, 163.1, 161.1, 143.2, 142.8, 121.3, 117.0, 96.0, 43.7, 24.0. LC/MS (ESI) calcd for C<sub>13</sub>H<sub>17</sub>N<sub>4</sub>O<sub>4</sub>S<sub>2</sub> (M + H)<sup>+</sup> 357.1, found 357.1.</div></div><div id="sec4-30" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> <i>N</i><sup>4</sup>-(3,5-Bis(trifluoromethyl)phenyl)-6-methylpyrimidine-2,4-diamine (<b>46</b>)</h3><div class="NLM_p last">According to general procedure A, pyrimidineamine (1.22 g, 8.52 mmol) and 3,5-bis-trifluoromethylaniline (1.32 g, 8.52 mmol) yielded compound <b>46</b> (1.57 g, 55%) as white solid; mp 119–120 °C. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.31 (s, 2H), 7.56 (s, 1H), 6.05 (s, 1H), 2.27 (s, 3H). <sup>13</sup>C NMR (101 MHz, methanol-<i>d</i><sub>4</sub>) δ 162.1, 160.2, 159.7, 141.2, 132.21, 131.9, 131.7 (q, <i>J</i> = 33.2 Hz), 131.6, 131.2, 127.4, 124.7, 121.9, 119.5, 115.3, 96.7, 20.6. LC/MS (ESI) calcd for C<sub>13</sub>H<sub>11</sub>F<sub>6</sub>N<sub>4</sub> (M + H)<sup>+</sup> 337.1, found 337.1.</div></div><div id="sec4-31" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> <i>N</i><sup>4</sup>-(3,5-Dimethoxyphenyl)-6-methylpyrimidine-2,4-diamine (<b>47</b>)</h3><div class="NLM_p last">According to general procedure A, 4-chloro-6-methylpyrimidin-2-amine (0.72 g, 5.00 mmol) and 3,5-dimethoxyaniline (0.76 g, 5.00 mmol) yielded amine <b>47</b> as a white solid (1.08 g, 83%); mp 184–185 °C. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 6.81 (d, <i>J</i> = 2.3 Hz, 2H), 6.17 (t, <i>J</i> = 2.3 Hz, 1H), 5.94 (s, 1H), 3.75 (s, 6H), 2.15 (s, 3H). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ 164.6, 162.6, 162.2, 161.0, 141.4, 98.6, 95.4, 94.5, 54.3, 21.8. LC/MS (ESI) calcd for C<sub>11</sub>H<sub>10</sub>Cl<sub>2</sub>N<sub>3</sub> (M + H)<sup>+</sup> 254.0, found 254.0.</div></div><div id="sec4-32" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> 5-((2-Amino-6-methylpyrimidin-4-yl)amino)isophthalic Acid (<b>48</b>)</h3><div class="NLM_p last">According to general procedure A, 4-chloro-6-methylpyrimidin-2-amine (1.00 g, 6.69 mmol) and 5-aminoisophthalic acid (1.21 g, 6.69 mmol) yielded compound <b>48</b> as a white solid (1.25 g, 65%); mp >300 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.13 (br s, 2H), 11.18 (br s, 1H), 8.38 (s, 2H), 8.21–8.18 (m, 3H), 6.27 (s, 1H), 2.26 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO- <i>d</i><sub>6</sub>) δ 167.0, 162.3, 156.2, 153.5, 138.9, 132.4, 126.8, 126.2, 97.9, 18.7. LC/MS (ESI) calcd for C<sub>13</sub>H<sub>13</sub>N<sub>4</sub>O<sub>4</sub> (M + H)<sup>+</sup> 289.1, found 289.1.</div></div><div id="sec4-33" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> Dimethyl 5-((2-Amino-6-methylpyrimidin-4-yl)amino)isophthalate (<b>49</b>)</h3><div class="NLM_p last">To a stirred solution of 5-((2-amino-6-methylpyrimidin-4-yl)amino)isophthalic acid <b>48</b> (0.05 g, 0.17 mmol) in 25 mL of MeOH was added 0.8 mL of concentrated H<sub>2</sub>SO<sub>4</sub>. 4 Å molecular sieves were added, and the solution was heated to reflux for 18 h. After cooling to rt, the volatiles were concentrated in vacuo. The residual oil was partitioned between 20 mL of EtOAc and 20 mL of 10% NaOH. The layers were separated, and the aqueous layer was extracted with 3 × 20 mL of EtOAc. The combined organic layers were dried (Na<sub>2</sub>SO<sub>4</sub>) and concentrated. The resulting dark oil was purified by flash chromatography (0–10% MeOH/DCM) to give amine <b>49</b> as a white foam (0.05 g, 93%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.43 (s, 1H), 8.51 (d, <i>J</i> = 1.6 Hz, 2H), 8.00 (t, <i>J</i> = 1.6 Hz, 1H), 6.13 (s, 2H), 5.86 (s, 1H), 3.86 (s, 6H), 2.09 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.7, 166.3, 166.0, 163.2, 142.2, 131.0, 123.9, 122.3, 95.8, 52.9, 23.9. LC/MS (ESI) calcd for C<sub>15</sub>H<sub>17</sub>N<sub>4</sub>O<sub>4</sub> (M + H)<sup>+</sup> 317.1, found 317.1.</div></div><div id="sec4-34" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> <i>N</i><sup>4</sup>-([1,1′:3′,1″-Terphenyl]-5′-yl)-6-methylpyrimidine-2,4-diamine (<b>50</b>)</h3><div class="NLM_p last">According to general procedure D, dibromopyrimidineamine <b>44</b> (0.5 g, 1.40 mmol) and phenylboronic acid (0.41 g, 3.35 mmol) yielded compound <b>50</b> as a yellow foam (0.36 g, 72%). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.65–7.62 (m, 4H), 7.61–7.58 (m, 2H), 7.52 (d, <i>J</i> = 1.6 Hz, 2H), 7.50–7.45 (m, 4H), 7.43–7.38 (m, 2H), 6.10 (s, 1H), 5.29 (s, 2H), 2.24 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 167.4, 162.9, 162.4, 142.9, 140.6, 139.6, 128.9, 127.7, 127.2, 122.29, 120.2, 94.4, 24.0. LC/MS (ESI) calcd for C<sub>23</sub>H<sub>21</sub>N<sub>4</sub> (M + H)<sup>+</sup> 353.2, found 353.2.</div></div><div id="sec4-35" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> 5-((2-Amino-6-methylpyrimidin-4-yl)amino)-<i>N</i><sup>1</sup>,<i>N</i><sup>3</sup>-bis(3,5-dichlorophenyl)isophthalamide (<b>51</b>)</h3><div class="NLM_p last">To a solution of diacid <b>48</b> (0.32 g, 1.09 mmol) and 3,5-dichloroaniline (0.45 g, 2.74 mmol) in a solution of 3 mL of pyridine, 7 mL of DCM, and 7 mL of DMF was added EDI (0.63 g, 3.27 mmol). The solution was stirred at rt for 18 h before being diluted with 50 mL of water. The solution was then extracted with 3 × 50 mL of EtOAc. The combined organic layers were then washed with water followed by brine, dried (Na<sub>2</sub>SO<sub>4</sub>), and concentrated in vacuo to afford a brown oil. The oil was purified by flash chromatography (0–20% MeOH/DCM gradient) to afford diamide <b>51</b> as a white solid (0.43 g, 69%); mp 235–236 °C. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.71 (s, 2H), 9.48 (s, 1H), 8.41 (d, <i>J</i> = 1.4 Hz, 2H), 8.04 (t, <i>J</i> = 1.4 Hz, 1H), 7.93 (d, <i>J</i> = 1.9 Hz, 4H), 7.37 (t, <i>J</i> = 1.9 Hz, 2H), 6.23 (s, 2H), 5.96 (s, 1H), 2.14 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.3, 166.2, 163.2, 161.5, 141.9, 141.9, 135.7, 134.4, 123.4, 121.9, 119.7, 118.7, 95.6, 24.0. LC/MS (ESI) calcd for C<sub>25</sub>H<sub>19</sub>Cl<sub>4</sub>N<sub>6</sub>O<sub>2</sub> (M + H)<sup>+</sup> 575.0, found 575.0.</div></div><div id="sec4-36" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> <i>N</i><sup>4</sup>-(3,5-Dimethylphenyl)-6-methylpyrimidine-2,4-diamine (<b>52</b>)</h3><div class="NLM_p last">According to general procedure A, pyrimidineamine (0.76 g, 6.10 mmol) and 3,5-dimethylaniline (0.88 g, 6.10 mmol) yielded compound <b>52</b> (0.85 g, 61%) as white solid; mp 163–164 °C. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.10 (s, 2H), 6.67 (s, 1H), 5.89 (s, 1H), 2.25 (s, 6H), 2.12 (s, 3H). <sup>13</sup>C NMR (101 MHz methanol-<i>d</i><sub>4</sub>) δ 164.7, 162.7, 162.4, 139.3, 138.1, 124.5, 118.7, 94.8, 21.8, 20.1. LC/MS (ESI) calcd for C<sub>13</sub>H<sub>17</sub>N<sub>4</sub> (M + H)<sup>+</sup> 229.1, found 229.1.</div></div><div id="sec4-37" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> <i>N</i><sup>4</sup>-(3,5-Di-<i>tert</i>-butylphenyl)-6-methylpyrimidine-2,4-diamine (<b>53</b>)</h3><div class="NLM_p last">According to general procedure C, 4-chloro-6-methylpyrimidin-2-amine (0.32 g, 2.24 mmol) and 3,5-di-<i>tert</i>-butylaniline (0.55 g, 2.68 mmol) yielded compound <b>53</b> as a white solid (0.56 g, 80%); mp 194–195 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.58 (s, 1H), 7.39 (d, <i>J</i> = 1.8 Hz, 2H), 7.06 (t, <i>J</i> = 1.8 Hz, 1H), 6.71 (s, 2H), 5.97 (s, 1H), 3.13 (s, 2H), 2.13 (m, 2H), 1.25 (s, 18H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.1, 160.0, 159.6, 151.1, 139.0, 117.3, 115.9, 95.9, 35.0, 31.7. LC/MS (ESI) calcd for C<sub>19</sub>H<sub>29</sub>N<sub>4</sub> (M + H)<sup>+</sup> 313.2, found 313.2.</div></div><div id="sec4-38" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> 6-Methyl-<i>N</i><sup>4</sup>-(3-(pentafluoro-λ<sup>6</sup>-sulfaneyl)phenyl)pyrimidine-2,4-diamine (<b>54</b>)</h3><div class="NLM_p last">According to general procedure C, 4-chloro-6-methylpyrimidin-2-amine (0.27 g, 1.90 mmol) and 3-pentafluorosulfanylaniline (0.50 g, 2.28 mmol) yielded compound <b>54</b> as a white solid (0.24 g, 87%); mp 175–176 °C. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.05 (t, <i>J</i> = 2.0 Hz, 1H), 7.96 (m, 1H), 7.40–7.28 (m, 2H), 5.90 (s, 1H), 2.13 (s, 3H). <sup>13</sup>C NMR (101 MHz, methanol-<i>d</i><sub>4</sub>) δ 165.3, 162.6, 161.8, 153.7 (sext, <i>J</i><sub>CF</sub> = 16.6 Hz), 140.8, 128.6, 122.6, 118.6, 116.7, 95.8, 21.9. LC/MS (ESI) calcd for C<sub>11</sub>H<sub>12</sub>F<sub>5</sub>N<sub>4</sub>S (M + H)<sup>+</sup> 327.1, found 327.1.</div></div><div id="sec4-39" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> <i>N</i><sup>4</sup>-(3-Chlorophenyl)-6-methylpyrimidine-2,4-diamine (<b>55</b>)</h3><div class="NLM_p last">According to general procedure A, 4-chloro-6-methylpyrimidin-2-amine (0.73 g, 5.06 mmol) and 3-chloroaniline (0.534 g, 5.06 mmol) yielded compound <b>55</b> (0.57 g, 48%) as white solid; mp 142–143 °C. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.75 (s, 1H), 7.43 (d, <i>J</i> = 8.2 Hz, 1H), 7.16 (t, <i>J</i> = 8.2 Hz, 1H), 6.91 (d, <i>J</i> = 8.2 Hz, 1H), 5.88 (s, 1H), 2.13 (s, 3H). <sup>13</sup>C NMR (101 MHz methanol-<i>d</i><sub>4</sub>) δ 165.1, 162.7, 161.9, 141.4, 133.9, 129.5 121.8, 119.6, 118.0, 95.7, 22.0. LC/MS (ESI) calcd for C<sub>11</sub>H<sub>12</sub>ClN<sub>4</sub> (M + H)<sup>+</sup> 235.1, found 235.1.</div></div><div id="sec4-40" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> <i>N</i><sup>4</sup>-(3,5-Difluorophenyl)-6-methylpyrimidine-2,4-diamine (<b>60</b>)</h3><div class="NLM_p last">According to general procedure A, 4-chloro-6-methylpyrimidin-2-amine (1.09 g, 7.61 mmol) and 3,5-fluoroaniline (0.98 g, 7.61 mmol) yielded compound <b>60</b> (0.97 g, 54%) as a white solid; mp 179–180 °C. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.38–7.33 (m, 2H), 6.43 (tt, <i>J</i> = 9.2, 2.4 Hz, 1H), 5.88 (s, 1H), 2.15 (s, 3H). <sup>13</sup>C NMR (101 MHz, methanol-<i>d</i><sub>4</sub>) δ 165.2, 163.1 (dd, <i>J</i><sub>CF</sub> = 242.8, 15.2 Hz), 162.6, 161.7, 142.90 (t, <i>J</i><sub>CF</sub> = 13.8 Hz), 101.96–101.47 (m), 96.0, 95.9 (dd, <i>J</i><sub>CF</sub> = 52.7, 3.4 Hz), 21.9. LC/MS (ESI) calcd for C<sub>11</sub>H<sub>11</sub>F<sub>2</sub>N<sub>4</sub> (M + H)<sup>+</sup> 237.1, found 237.1.</div></div><div id="sec4-41" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> <i>N</i><sup>4</sup>-(3-Chloro-5-fluorophenyl)-6-methylpyrimidine-2,4-diamine (<b>61</b>)</h3><div class="NLM_p last">According to general procedure C, 3-chloro-5-fluoroaniline (0.28 g, 1.94 mmol) and 4-chloro-6-methylpyrimidin-2-amine (0.28 g, 1.94 mmol) yielded compound <b>61</b> as a white solid (0.378 g, 77%); mp 143–144 °C. <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 8.63 (br s, 1H), 7.80 (dt, <i>J</i> = 10.2, 2.1 Hz, 1H), 7.53 (q, <i>J</i> = 2.1 Hz, 1H), 6.74 (dt, <i>J</i> = 10.2, 2.1 Hz, 1H), 6.15 (br s, 2H), 5.94 (s, 1H), 2.12 (s, 3H). <sup>13</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 166.6, 162.9 (d, <i>J</i><sub>CF</sub> = 243.6 Hz), 161.4, 163.3, 143.7 (d, <i>J</i><sub>CF</sub> = 13.1 Hz), 134.2 (d, <i>J</i><sub>CF</sub> = 13.3 Hz), 114.5 (d, <i>J</i><sub>CF</sub> = 2.9 Hz), 108.0 (d, <i>J</i><sub>CF</sub> = 25.8 Hz), 104.5 (d, <i>J</i><sub><i>CF</i></sub> = 27.0 Hz) 95.7, 22.90. LC/MS (ESI) calcd for C<sub>10</sub>H<sub>12</sub>ClFN<sub>4</sub> (M + H)<sup>+</sup> 253.1, found 225.1.</div></div><div id="sec4-42" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> <i>N</i><sup>4</sup>-(3,5-Dichloro-4-fluorophenyl)-6-methylpyrimidine-2,4-diamine (<b>62</b>)</h3><div class="NLM_p last">According to general procedure C, 4-chloro-6-methylpyrimidin-2-amine (1.00 g, 6.69 mmol) and 3,5-dichloro-4-fluoroaniline (1.24 g, 6.69 mmol) yielded compound <b>62</b> as a white solid (1.55 g, 81%); mp 204–205 °C. <sup>1</sup>H NMR (400 MHz, acetone-<i>d</i><sub>6</sub>) δ 8.48 (br s, 1H), 7.93 (d, <i>J</i><sub>HF</sub> = 6.1 Hz, 2H), 5.95 (br s, 2H), 5.91 (s, 1H), 2.11 (s, 3H). <sup>3</sup>C NMR (101 MHz, acetone-<i>d</i><sub>6</sub>) δ 166.7, 163.1, 161.3, 148.4 (d, <i>J</i> = 242.2 Hz), 138.2 (d, <i>J</i> = 3.8 Hz), 121.1 (d, <i>J</i> = 17.9 Hz), 119.1, 95.5, 23.0. LC/MS (ESI) calcd for C<sub>11</sub>H<sub>9</sub>Cl<sub>2</sub>FN<sub>4</sub> (M + H)<sup>+</sup> 287.0, found 287.0.</div></div><div id="sec4-43" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> <i>N</i><sup>4</sup>-(3,5-Dibromo-4-fluorophenyl)-6-methylpyrimidine-2,4-diamine (<b>63</b>)</h3><div class="NLM_p last">According to general procedure C, 4-chloro-6-methylpyrimidin-2-amine (0.26 g, 1.84 mmol) and 3,5-dibromo-4-fluoroaniline (0.50 g, 2.39 mmol) yielded compound <b>63</b> as a white solid (0.59 g, 85%); mp 205–206 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.20 (s, 1H), 8.02 (d, <i>J</i><sub>HF</sub> = 5.7 Hz, 2H), 6.33 (s, 2H), 5.80 (s, 1H), 2.07 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.2, 163.1, 161.2, 149.7 (d, <i>J</i> = 237.2 Hz), 139.6 (d, <i>J</i> = 3.3 Hz), 122.4, 108.8 (d, <i>J</i> = 22.5 Hz), 95.6, 23.9. LC/MS (ESI) calcd for C<sub>11</sub>H<sub>10</sub>Br<sub>2</sub>FN<sub>4</sub> (M + H)<sup>+</sup> 374.9, found 374.9.</div></div><div id="sec4-44" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> 6-Methyl-<i>N</i><sup>4</sup>-(3,4,5-trichlorophenyl)pyrimidine-2,4-diamine (<b>64</b>)</h3><div class="NLM_p last">According to general procedure C, 4-chloro-6-methylpyrimidin-2-amine (0.54 g, 3.79 mmol) and 3,4,5-trichloroaniline (0.80 g, 3.79 mmol) yielded compound <b>64</b> as a white solid (0.94 g, 82%); mp 219–220 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.36 (s, 1H), 8.00 (s, 2H), 6.37 (s, 2H), 5.83 (s, 1H), 2.08 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.5, 163.1, 161.1, 141.6, 133.0, 120.8, 118.9, 95.9, 23.9. LC/MS (ESI) calcd for C<sub>11</sub>H<sub>10</sub>Cl<sub>3</sub>N<sub>4</sub> (M + H)<sup>+</sup> 303.0, found 303.3.</div></div><div id="sec4-45" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> <i>N</i><sup>4</sup>-(3,5-Dichloro-4-methylphenyl)-6-methylpyrimidine-2,4-diamine (<b>65</b>)</h3><div class="NLM_p last">According to general procedure C, 4-chloro-6-methylpyrimidin-2-amine (0.31 g, 2.18 mmol) and 3,5-dichloro-4-methylaniline (0.50 g, 2.84 mmol) yielded compound <b>65</b> as a white solid (0.51 g, 83%); mp 185–186 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.16 (s, 1H), 7.80 (s, 2H), 6.30 (s, 2H), 5.81 (s, 1H), 2.28 (s, 3H), 2.07 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.1, 163.2, 161.3, 140.7, 134.5, 125.3, 118.1, 95.7, 23.9, 16.8. LC/MS (ESI) calcd for C<sub>12</sub>H<sub>13</sub>Cl<sub>2</sub>N<sub>4</sub> (M + H)<sup>+</sup> 283.1, found 283.1.</div></div><div id="sec4-46" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> <i>N</i><sup>4</sup>-(3,5-Dichloro-4-phenoxyphenyl)-6-methylpyrimidine-2,4-diamine (<b>66</b>)</h3><div class="NLM_p last">According to general procedure C, 4-chloro-6-methylpyrimidin-2-amine (0.09 g, 0.67 mmol) and 3,5-dichloro-4-phenoxyaniline (0.50 g, 0.78 mmol) yielded compound <b>66</b> as an off-white solid (0.20 g, 71%); mp 100–101 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.35 (s, 1H), 7.96 (s, 2H), 7.48–7.23 (m, 2H), 7.07 (m, 1H), 6.78 (m, 2H), 6.35 (s, 2H), 5.85 (s, 1H), 2.09 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.1, 163.1, 161.2, 157.2, 140.1, 139.6, 130.3, 128.7, 122.9, 119.1, 114.9, 95.8, 23.9. LC/MS (ESI) calcd for C<sub>17</sub>H<sub>15</sub>Cl<sub>2</sub>N<sub>4</sub>O (M + H)<sup>+</sup> 361.1, found 361.1.</div></div><div id="sec4-47" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74"> 4-((2-Amino-6-methylpyrimidin-4-yl)amino)-2,6-dichlorophenol (<b>67</b>)</h3><div class="NLM_p last">According to general procedure C, 4-chloro-6-methylpyrimidin-2-amine (0.62 g, 4.32 mmol) and 4-amino-2,6-dichlorophenol (1.00 g, 5.62 mmol) yielded compound <b>67</b> as an off-white solid (1.00 g, 82%); mp 250 °C (decomposes). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.99 (s, 1H), 7.72 (s, 2H), 6.25 (s, 2H), 5.80 (s, 1H), 2.09 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.6, 163.2, 161.5, 143.9, 134.5, 122.8, 119.6, 95.2, 23.9. LC/MS (ESI) calcd for C<sub>11</sub>H<sub>11</sub>Cl<sub>2</sub>N<sub>4</sub>O (M + H)<sup>+</sup> 285.0, found 285.0.</div></div><div id="sec4-48" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75"> <i>N</i><sup>4</sup>-(3,5-Dichloro-4-(difluoromethoxy)phenyl)-6-methylpyrimidine-2,4-diamine (<b>68</b>)</h3><div class="NLM_p last">According to general procedure C, 4-chloro-6-methylpyrimidin-2-amine (0.13 g, 0.87 mmol) and 3,5-dichloro-4-difluoromethoxyaniline (0.24 g, 1.05 mmol) yielded compound <b>68</b> as an off-white solid (0.22 g, 62%); mp 169–170 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.35 (s, 1H), 7.93 (s, 2H), 7.03 (t, <i>J</i><sub>HF</sub> = 72.8 Hz, 1H), 6.36 (s, 2H), 5.83 (s, 1H), 2.08 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.4, 163.1, 161.1, 140.9, 136.8, 128.7, 118.8, 118.0 (t, <i>J</i> = 261.8 Hz), 95.82, 23.91. LC/MS (ESI) calcd for C<sub>12</sub>H<sub>11</sub>Cl<sub>2</sub>F<sub>2</sub>N<sub>4</sub>O (M + H)<sup>+</sup> 335.0, found 335.0.</div></div><div id="sec4-49" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> <i>N</i><sup>4</sup>-(3,5-Dichloro-4-(dimethylamino)phenyl)-6-methylpyrimidine-2,4-diamine (<b>69</b>)</h3><div class="NLM_p last">According to general procedure A, 4-chloro-6-methylpyrimidin-2-amine (0.36 g, 2.50 mmol) and 2,6-dichloro-<i>N</i><sup>1</sup>,<i>N</i><sup>1</sup>-dimethylbenzene-1,4-diamine (0.62 g, 3.00 mmol) yielded compound <b>69</b> as a clear oil (0.56 g, 72%). <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.60 (s, 2H), 5.85 (s, 1H), 2.77 (s, 6H), 2.12 (s, 3H). <sup>13</sup>C NMR (101 MHz, methanol-<i>d</i><sub>4</sub>) δ 165.1, 162.6, 161.6, 139.9, 138.3, 135.3 119.7, 95.8, 41.2, 22.0. LC/MS (ESI) calcd for C<sub>13</sub>H<sub>16</sub>Cl<sub>2</sub>N<sub>5</sub> (M + H)<sup>+</sup> 312.1, found 312.1.</div></div><div id="sec4-50" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77"> <i>N</i><sup>4</sup>-(3,5-Dichloro-4-morpholinophenyl)-6-methylpyrimidine-2,4-diamine (<b>70</b>)</h3><div class="NLM_p last">According to general procedure A, 4-chloro-6-methylpyrimidin-2-amine (0.11 g, 0.75 mmol) and 3,5-dichloro-4-morpholinoaniline (0.22 g, 0.89 mmol) yielded compound <b>70</b> as a white solid (0.21 g, 81%); mp 114–115 °C. 1H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.19 (s, 1H), 7.76 (s, 2H), 6.29 (s, 2H), 5.81 (s, 1H), 3.71–3.54 (m, 4H), 3.11–2.98 (m, 4H), 2.07 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.0, 163.1, 161.2, 140.0, 137.3, 134.9, 119.1, 95.7, 67.5, 49.9, 23.8. LC/MS (ESI) calcd for C<sub>15</sub>H<sub>18</sub>Cl<sub>2</sub>N<sub>5</sub>O (M + H)<sup>+</sup> 354.1, found 354.1.</div></div><div id="sec4-51" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> <i>N</i><sup>4</sup>-(3,5-Dichloro-4-(piperidin-1-yl)phenyl)-6-methylpyrimidine-2,4-diamine (<b>71</b>)</h3><div class="NLM_p last">According to general procedure A, 4-chloro-6-methylpyrimidin-2-amine (0.37 g, 2.54 mmol) and 3,5-dichloro-4-(piperidin-1-yl)aniline (0.75 g, 3.06 mmol) yielded compound <b>71</b> as a yellow foam (0.70 g, 79%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.14 (s, 1H), 7.73 (s, 2H), 6.24 (s, 2H), 5.80 (s, 1H), 3.05–2.95 (m, 4H), 2.05 (s, 3H), 1.62–1.44 (m, 6H). <sup>13</sup>C NMR (101 MHz DMSO-<i>d</i><sub>6</sub>) δ 166.1, 163.2, 161.3, 139.4, 139.0, 134.7, 119.1, 95.6, 51.1, 26.8, 24.3, 23.9. LC/MS (ESI) calcd for C<sub>16</sub>H<sub>20</sub>Cl<sub>2</sub>N<sub>5</sub>(M + H)<sup>+</sup> 352.1, found 352.1.</div></div><div id="sec4-52" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> 2-((2-Amino-6-methylpyrimidin-4-yl)amino)-4,6-dichlorophenol (<b>72</b>)</h3><div class="NLM_p last">According to general procedure C, 4-chloro-6-methylpyrimidin-2-amine (1.00 g, 6.96 mmol) and 2-amino-4,6-dichlorophenol (1.48 g, 8.35 mmol) yielded compound <b>72</b> as a tan solid (1.40 g, 71%); mp 216–217 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.53 (d, <i>J</i> = 2.6 Hz, 1H), 7.13 (d, <i>J</i> = 2.6 Hz, 1H), 6.02 (s, 1H), 2.08 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.7, 161.9, 161.1, 144.2, 131.8, 123.5, 123.4, 122.9, 120.6, 95.6, 23.7. LC/MS (ESI) calcd for C<sub>11</sub>H<sub>11</sub>Cl<sub>2</sub>N<sub>4</sub>O (M + H)<sup>+</sup> 285.0, found 285.0.</div></div><div id="sec4-53" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> <i>N</i><sup>4</sup>-(3,5-Dichloro-2,4-difluorophenyl)-6-methylpyrimidine-2,4-diamine (<b>73</b>)</h3><div class="NLM_p last">According to general procedure C, 4-chloro-6-methylpyrimidin-2-amine (0.30 g, 2.10 mmol) and 3,5-dichloro-2,4-difluoroaniline (0.50 g, 2.52 mmol) yielded compound <b>73</b> as a white solid (0.63 g, 81%); mp 203–204 °C. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.38 (t, <i>J</i><sub>HF</sub> = 7.9 Hz, 1H), 6.03 (s, 1H), 2.18 (s, 3H). <sup>13</sup>C NMR (101 MHz, methanol-<i>d</i><sub>4</sub>) δ 165.8, 162.6, 161.9, 122.2, 95.6, 21.9 (halogenated carbons not visible). LC/MS (ESI) calcd for C<sub>11</sub>H<sub>8</sub>Cl<sub>2</sub>F<sub>2</sub>N<sub>4</sub> (M + H)<sup>+</sup> 305.0, found 305.0.</div></div><div id="sec4-54" class="NLM_sec NLM_sec_level_2"><div id="ac_i81" class="anchor-spacer"></div><h3 class="article-section__title" id="_i81"> <i>N</i><sup>4</sup>-(3′,5′-Dichloro-[1,1′-biphenyl]-4-yl)-6-methylpyrimidine-2,4-diamine (<b>89</b>)</h3><div class="NLM_p last">According to general procedure C, 4-chloro-6-methylpyrimidin-2-amine (0.38 g, 2.62 mmol) and 3′,5′-dichloro-[1,1′-biphenyl]-4-amine (0.75 g, 3.15 mmol) yielded compound <b>89</b> as a white solid (0.69 g, 76%); mp 237–238 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.16 (s, 1H), 7.81 (d, <i>J</i> = 8.0 Hz, 2H), 7.65 (s, 2H), 7.61 (d, <i>J</i> = 8.0 Hz, 2H), 7.46 (s, 1H), 6.20 (br s, 2H), 5.90 (s, 1H), 2.08 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.7, 163.3, 161.5, 144.0, 142.1, 135.0, 129.9, 127.6, 126.3, 125.0, 119.7, 95.6, 23.9. LC/MS (ESI) calcd for C<sub>17</sub>H<sub>15</sub>Cl<sub>2</sub>N<sub>4</sub> (M + H)<sup>+</sup> 345.1, found 345.1.</div></div><div id="sec4-55" class="NLM_sec NLM_sec_level_2"><div id="ac_i82" class="anchor-spacer"></div><h3 class="article-section__title" id="_i82"> <i>N</i><sup>4</sup>-(5-Chloro-4′-fluoro-[1,1′-biphenyl]-3-yl)-6-methylpyrimidine-2,4-diamine (<b>91</b>)</h3><div class="NLM_p last">According to general procedure D, bromopyrimidineamine <b>32</b> (0.3 g, 0.95 mmol) yielded compound <b>91</b> as an off-white solid (0.20 g, 65%); mp 105–106 °C. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.81 (t, <i>J</i> = 1.9 Hz, 1H), 7.62 (t, <i>J</i> = 1.9 Hz, 1H), 7.60–7.55 (m, 2H), 7.17–7.11 (m, 3H), 5.92 (s, 1H), 2.16 (s, 3H). <sup>13</sup>C NMR (101 MHz, methanol-<i>d</i><sub>4</sub>) δ 165.1, 164.0, 162.8 (d, <i>J</i><sub><i>CF</i></sub> = 245.9 Hz), 161.9, 141.9 (d, <i>J</i><sub>CF</sub> = 16.2 Hz), 135.99, 135.95, 134.4, 128.5 (d, <i>J</i><sub>CF</sub> = 8.2 Hz), 120.0, 118.1, 116.2, 115.2 (d, <i>J</i><sub>CF</sub> = 21.9 Hz), 95.7, 21.8. LC/MS (ESI) calcd for C<sub>17</sub>H<sub>15</sub>ClFN<sub>4</sub> (M + H)<sup>+</sup> 329.1, found 329.1.</div></div><div id="sec4-56" class="NLM_sec NLM_sec_level_2"><div id="ac_i83" class="anchor-spacer"></div><h3 class="article-section__title" id="_i83"> <i>N</i><sup>4</sup>-(4′,5-Dichloro-[1,1′-biphenyl]-3-yl)-6-methylpyrimidine-2,4-diamine (<b>92</b>)</h3><div class="NLM_p last">According to general procedure D, bromopyrimidineamine <b>32</b> (0.50 g, 1.59 mmol) yielded compound <b>92</b> as a yellow foam (0.39 g, 72%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.26 (s, 1H), 7.93 (t, <i>J</i> = 1.8 Hz, 1H), 7.83 (t, <i>J</i> = 1.8 Hz, 1H), 7.66 (d, <i>J</i> = 8.5 Hz, 2H), 7.47 (d, <i>J</i> = 8.5 Hz, 2H), 7.20 (t, <i>J</i> = 1.8 Hz, 1H), 6.32 (s, 2H), 5.88 (s, 1H), 2.09 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.0, 163.2, 161.6, 143.2, 141.3, 138.2, 134.3, 133.3, 129.4, 129.0, 119.2, 117.9, 116.0, 95.8, 23.9. LC/MS (ESI) calcd for C<sub>17</sub>H<sub>15</sub>Cl<sub>2</sub>N<sub>4</sub> (M + H)<sup>+</sup> 345.1, found 345.1.</div></div><div id="sec4-57" class="NLM_sec NLM_sec_level_2"><div id="ac_i84" class="anchor-spacer"></div><h3 class="article-section__title" id="_i84"> <i>N</i><sup>4</sup>-(5-Chloro-4′-methoxy-[1,1′-biphenyl]-3-yl)-6-methylpyrimidine-2,4-diamine (<b>93</b>)</h3><div class="NLM_p last">According to general procedure D, bromopyrimidineamine <b>32</b> (0.46 g, 1.45 mmol) yielded compound <b>93</b> as a yellow foam (0.30 g, 61%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.20 (s, 1H), 7.90 (t, <i>J</i> = 1.8 Hz, 1H), 7.71 (t, <i>J</i> = 1.8 Hz, 1H), 7.58 (d, <i>J</i> = 8.8 Hz, 2H), 7.15 (t, <i>J</i> = 1.8 Hz, 1H), 7.00 (d, <i>J</i> = 8.8 Hz, 2H), 6.25 (s, 2H), 5.87 (s, 1H), 3.77 (s, 3H), 2.08 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.0, 163.2, 161.6, 159.7, 143.1, 142.3, 134.1, 131.6, 128.3, 118.8, 117.0, 115.5, 114.8, 95.7, 55.6, 23.9. LC/MS (ESI) calcd for C<sub>18</sub>H<sub>18</sub>ClN<sub>4</sub>O (M + H)<sup>+</sup> 341.1, found 341.1.</div></div><div id="sec4-58" class="NLM_sec NLM_sec_level_2"><div id="ac_i85" class="anchor-spacer"></div><h3 class="article-section__title" id="_i85"> <i>N</i><sup>4</sup>-(5-Chloro-4′-methyl-[1,1′-biphenyl]-3-yl)-6-methylpyrimidine-2,4-diamine (<b>94</b>)</h3><div class="NLM_p last">According to general procedure D, bromopyrimidineamine <b>32</b> (0.50 g, 1.59 mmol) yielded compound <b>94</b> as a yellow oil (0.40 g, 81%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.23 (s, 1H), 7.98 (t, <i>J</i> = 1.8 Hz, 1H), 7.73 (t, <i>J</i> = 1.8 Hz, 1H), 7.52 (d, <i>J</i> = 8.0 Hz, 2H), 7.23 (d, <i>J</i> = 8.0 Hz, 2H), 7.16 (t, <i>J</i> = 1.8 Hz, 1H), 6.30 (s, 2H), 5.89 (s, 1H), 2.30 (s, 3H), 2.09 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.0, 163.3, 161.6, 143.1, 142.6, 137.8, 136.5, 134.1, 130.0, 127.0, 119.1, 117.4, 115.8, 95.8, 23.9, 21.1. LC/MS (ESI) calcd for C<sub>18</sub>H<sub>18</sub>ClN<sub>4</sub> (M + H)<sup>+</sup> 325.1, found 325.1.</div></div><div id="sec4-59" class="NLM_sec NLM_sec_level_2"><div id="ac_i86" class="anchor-spacer"></div><h3 class="article-section__title" id="_i86"> <i>N</i><sup>4</sup>-(5-Chloro-4′-(trifluoromethyl)-[1,1′-biphenyl]-3-yl)-6-methylpyrimidine-2,4-diamine (<b>95</b>)</h3><div class="NLM_p last">According to general procedure D, bromopyrimidineamine <b>32</b> (0.5 g, 1.59 mmol) yielded compound <b>95</b> as a yellow foam (0.41 g, 67%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.31 (s, 1H), 7.96 (m, 2H), 7.85 (d, <i>J</i> = 8.3 Hz, 2H), 7.76 (d, <i>J</i> = 8.3 Hz, 2H), 7.25 (t, <i>J</i> = 1.7 Hz, 1H), 6.38 (s, 2H), 5.91 (s, 1H), 2.10 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.0, 163.3, 161.6, 143.4, 141.0, 134.3, 128.0, 128.8 (q, <i>J</i><sub>CF</sub> = 31.9 Hz), 126.2 (q, <i>J</i><sub>CF</sub> = 3.7 Hz), 126.0, 123.3, 119.5, 118.4, 116.3, 95.9, 23.8. LC/MS (ESI) calcd for C<sub>18</sub>H<sub>15</sub>ClF<sub>3</sub>N<sub>4</sub> (M + H)<sup>+</sup> 379.1, found 379.1.</div></div><div id="sec4-60" class="NLM_sec NLM_sec_level_2"><div id="ac_i87" class="anchor-spacer"></div><h3 class="article-section__title" id="_i87"> 3′-((2-Amino-6-methylpyrimidin-4-yl)amino)-5′-chloro-[1,1′-biphenyl]-4-carbonitrile (<b>96</b>)</h3><div class="NLM_p last">According to general procedure D, bromopyrimidineamine <b>32</b> (0.3 g, 0.95 mmol) yielded compound <b>96</b> as a tan solid (0.28 g, 88%); mp 197–198 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.28 (s, 1H), 7.95 (m, 1H), 7.92 (t, <i>J</i> = 1.8 Hz, 1H), 7.90–7.82 (m, 4H), 7.27 (t, <i>J</i> = 1.8 Hz, 1H), 6.32 (s, 2H), 5.87 (s, 1H), 2.09 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.1, 163.2, 161.4, 143.9, 143.3 140.7, 134.5, 133.3, 128.2, 119.5, 119.2, 118.5, 116.2, 111.0, 95.8, 23.9. LC/MS (ESI) calcd for C<sub>18</sub>H<sub>15</sub>ClN<sub>5</sub> (M + H)<sup>+</sup> 336.1, found 336.1.</div></div><div id="sec4-61" class="NLM_sec NLM_sec_level_2"><div id="ac_i88" class="anchor-spacer"></div><h3 class="article-section__title" id="_i88"> 1-(3′-((2-Amino-6-methylpyrimidin-4-yl)amino)-5′-chloro-[1,1′-biphenyl]-4-yl)ethan-1-one (<b>97</b>)</h3><div class="NLM_p last">According to general procedure D, bromopyrimidineamine <b>32</b> (0.3 g, 0.95 mmol) yielded compound <b>97</b> as a white solid (0.24 g, 72%); mp 225 °C (chars). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.29 (s, 1H), 8.02 (d, <i>J</i> = 8.3 Hz, 2H), 7.97 (s, 1H), 7.91 (s, 1H), 7.80 (d, <i>J</i> = 8.3 Hz, 2H), 7.29 (s, 1H), 6.28 (s, 2H), 5.87 (s, 1H), 2.59 (s, 3H), 2.09 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 197.9, 166.1, 163.2, 161.5, 143.6, 143.3, 141.3, 136.5, 134.3, 129.4, 127.45, 119.4, 118.3, 116.2, 95.8, 27.2, 23.9. LC/MS (ESI) calcd for C<sub>19</sub>H<sub>18</sub>ClN<sub>4</sub>O (M + H)<sup>+</sup> 353.1, found 353.1.</div></div><div id="sec4-62" class="NLM_sec NLM_sec_level_2"><div id="ac_i89" class="anchor-spacer"></div><h3 class="article-section__title" id="_i89"> 3′-((2-Amino-6-methylpyrimidin-4-yl)amino)-5′-chloro-[1,1′-biphenyl]-4-ol (<b>98</b>)</h3><div class="NLM_p last">According to general procedure D, bromopyrimidineamine <b>32</b> (0.31 g, 0.99 mmol) yielded compound <b>98</b> as a white solid (0.22 g, 69%); mp 142–143 °C. <sup>1</sup>H NMR (400 MHz DMSO-<i>d</i><sub>6</sub>) δ 9.18 (s, 1H), 7.86 (s, 1H), 7.66 (s, 1H), 7.46 (d, <i>J</i> = 8.0 Hz, 2H), 7.11 (s, 1H), 6.82 (d, <i>J</i> = 8.0 Hz, 2H), 6.24 (s, 2H), 5.86 (s, 1H), 2.08 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.9, 163.2, 161.6 158.0, 143.0, 142.7, 134.0, 130.0, 128.3, 118.6, 116.7, 116. 2, 115.3, 95.7, 23.9. LC/MS (ESI) calcd for C<sub>17</sub>H<sub>16</sub>ClN<sub>4</sub>O (M + H)<sup>+</sup> 327.1, found 327.1.</div></div><div id="sec4-63" class="NLM_sec NLM_sec_level_2"><div id="ac_i90" class="anchor-spacer"></div><h3 class="article-section__title" id="_i90"> <i>N</i><sup>4</sup>-(5-Chloro-[1,1′:4′,1″-terphenyl]-3-yl)-6-methylpyrimidine-2,4-diamine (<b>99</b>)</h3><div class="NLM_p last">According to general procedure D, bromopyrimidineamine <b>32</b> (0.3 g, 0.95 mmol) yielded compound <b>99</b> as a tan solid (0.25 g, 69%); mp 104–105 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.25 (s, 1H), 7.96 (t, <i>J</i> = 1.7 Hz, 1H), 7.84 (t, <i>J</i> = 1.7 Hz, 1H), 7.75 (s, 4H), 7.70 (d, <i>J</i> = 7.6 Hz, 2H), 7.46 (d, <i>J</i> = 7.6 Hz, 2H), 7.35 (m, 1H), 7.27 (t, <i>J</i> = 1.7 Hz, 1H), 6.27 (s, 2H), 5.88 (s, 1H), 2.09 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.0, 163.3, 161.6 143.2, 142.0, 140.1, 139.9, 138.3, 134.2, 129. 5, 128.1, 127.7, 127.7, 127.0, 119.2, 117.7, 115.9, 95.8, 24.0. LC/MS (ESI) calcd for C<sub>23</sub>H<sub>20</sub>ClN<sub>4</sub> (M + H)<sup>+</sup> 387.2, found 387.2.</div></div><div id="sec4-64" class="NLM_sec NLM_sec_level_2"><div id="ac_i91" class="anchor-spacer"></div><h3 class="article-section__title" id="_i91"> <i>N</i><sup>4</sup>-(5-Chloro-4′-(dimethylamino)-[1,1′-biphenyl]-3-yl)-6-methylpyrimidine-2,4-diamine (<b>100</b>)</h3><div class="NLM_p last">According to general procedure D, bromopyrimidineamine <b>32</b> (0.3 g, 0.95 mmol) yielded compound <b>100</b> as a yellow solid (0.28 g, 82%); mp 104–105 °C. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.70 (t, <i>J</i> = 1.8 Hz, 1H), 7.51 (t, <i>J</i> = 1.8 Hz, 1H), 7.40 (d, <i>J</i> = 8.8 Hz, 2H), 7.12 (t, <i>J</i> = 1.8 Hz, 1H), 6.75 (d, <i>J</i> = 8.8 Hz, 2H), 5.91 (s, 1H), 2.90 (s, 6H), 2.14 (s, 3H). <sup>13</sup>C NMR (101 MHz, methanol-<i>d</i><sub>4</sub>) δ 165.0, 162.7, 161.9, 150.5, 143.2, 141.5, 134.2, 127.3, 127.0, 119.4, 117.0, 115.4, 112.6, 95.7, 39.3, 21.9. LC/MS (ESI) calcd for C<sub>19</sub>H<sub>21</sub>ClN<sub>5</sub> (M + H)<sup>+</sup> 354.2, found 354.2.</div></div><div id="sec4-65" class="NLM_sec NLM_sec_level_2"><div id="ac_i92" class="anchor-spacer"></div><h3 class="article-section__title" id="_i92"> <i>N</i><sup>4</sup>-(2′,5-Dichloro-[1,1′-biphenyl]-3-yl)-6-methylpyrimidine-2,4-diamine (<b>101</b>)</h3><div class="NLM_p last">According to general procedure D, bromopyrimidineamine <b>32</b> (0.3 g, 0.95 mmol) yielded compound <b>101</b> as an off-white solid (0.22 g, 66%); mp 83–84 °C. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.91 (m, 1H), 7.48 (m, 1H), 7.44 (t, <i>J</i> = 1.7 Hz, 1H), 7.37–7.30 (m, 3H), 6.99 (t, <i>J</i> = 1.7 Hz, 1H), 5.95 (s, 1H), 2.16 (s, 3H). <sup>13</sup>C NMR (101 MHz, methanol-<i>d</i><sub>4</sub>) δ 165.1, 162.7, 161.9, 141.2, 141.2, 139.2, 133.5, 131.9, 130.9, 129.6, 128.9, 126.9, 122.5, 118.8, 118.5, 95.7, 21.8. LC/MS (ESI) calcd for C<sub>17</sub>H<sub>15</sub>Cl<sub>2</sub>N<sub>4</sub> (M + H)<sup>+</sup> 345.1, found 345.1.</div></div><div id="sec4-66" class="NLM_sec NLM_sec_level_2"><div id="ac_i93" class="anchor-spacer"></div><h3 class="article-section__title" id="_i93"> <i>N</i><sup>4</sup>-(3′,5-Dichloro-[1,1′-biphenyl]-3-yl)-6-methylpyrimidine-2,4-diamine (<b>102</b>)</h3><div class="NLM_p last">According to general procedure D, bromopyrimidineamine <b>32</b> (0.3 g, 0.95 mmol) yielded compound <b>102</b> as an off-white solid (0.23 g, 71%); mp 169–170 °C. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.91 (m, 1H), 7.57 (t, <i>J</i> = 1.9 Hz, 1H), 7.54 (t, <i>J</i> = 1.9 Hz, 1H), 7.47 (m, 1H), 7.38 (t, <i>J</i> = 7.8 Hz, 1H), 7.33 (m, 1H), 7.15 (t, <i>J</i> = 1.9 Hz, 1H), 5.92 (s, 1H), 2.16 (s, 3H). <sup>13</sup>C NMR (101 MHz, methanol-<i>d</i><sub>4</sub>) δ 165.1, 162.7, 161.8, 141.9, 141.7, 141.5, 134.5, 134.4, 130.0, 127.5, 126.5, 125.0, 120.1, 118.6, 116.2, 95.8, 21.9. LC/MS (ESI) calcd for C<sub>17</sub>H<sub>15</sub>Cl<sub>2</sub>N<sub>4</sub> (M + H)<sup>+</sup> 345.1, found 345.1.</div></div><div id="sec4-67" class="NLM_sec NLM_sec_level_2"><div id="ac_i94" class="anchor-spacer"></div><h3 class="article-section__title" id="_i94"> 6-Methyl-<i>N</i><sup>4</sup>-(3′,4′,5-trichloro-[1,1′-biphenyl]-3-yl)pyrimidine-2,4-diamine (<b>103</b>)</h3><div class="NLM_p last">According to general procedure D, bromopyrimidineamine <b>32</b> (0.29 g, 0.93 mmol) yielded compound <b>103</b> as a yellow foam (0.27 g, 76%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.55 (s, 1H), 7.49–7.42 (m, 2H), 7.34–7.27 (m, 2H), 7.20–7.09 (m, 2H), 5.94 (s, 1H), 5.18 (s, 2H), 2.21 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 167.6, 162.7, 161.4, 141.0, 140.7, 139.4, 135.3, 133.0, 132.3, 130.8, 128.8, 126.2, 122.0, 120.5, 117.8, 95.2, 23.9. LC/MS (ESI) calcd for C<sub>17</sub>H<sub>14</sub>Cl<sub>3</sub>N<sub>4</sub> (M + H)<sup>+</sup> 379.0, found 379.0.</div></div><div id="sec4-68" class="NLM_sec NLM_sec_level_2"><div id="ac_i95" class="anchor-spacer"></div><h3 class="article-section__title" id="_i95"> (3′,5-Dichloro-4′-fluoro-[1,1′-biphenyl]-3-yl)-6-methylpyrimidine-2,4-diamine (<b>104</b>)</h3><div class="NLM_p last">According to general procedure D, bromopyrimidineamine <b>32</b> (0.3 g, 0.95 mmol) yielded compound <b>104</b> as an off-white solid (0.22 g, 64%); mp 129–130 °C. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.90 (t, <i>J</i> = 1.8 Hz, 1H), 7.70 (m, 1H), 7.64 (t, <i>J</i> = 1.8 Hz, 1H), 7.56 (m, 1H), 7.31 (m, 1H), 7.19 (t, <i>J</i> = 1.8 Hz, 1H), 5.95 (s, 1H), 2.17 (s, 3H). LC/MS (ESI) calcd for C<sub>17</sub>H<sub>14</sub>Cl<sub>2</sub>FN<sub>4</sub> (M + H)<sup>+</sup> 363.1, found 363.1.</div></div><div id="sec4-69" class="NLM_sec NLM_sec_level_2"><div id="ac_i96" class="anchor-spacer"></div><h3 class="article-section__title" id="_i96"> <i>N</i><sup>4</sup>-(4′,5-Dichloro-3′-(trifluoromethyl)-[1,1′-biphenyl]-3-yl)-6-methylpyrimidine-2,4-diamine (<b>105</b>)</h3><div class="NLM_p last">According to general procedure D, bromopyrimidineamine <b>32</b> (0.3 g, 0.95 mmol) yielded compound <b>105</b> as an off-white foam (0.22 g, 67%). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.34 (s, 1H), 8.07 (m, 1H), 8.01 (m, 2H), 7.91–7.78 (m, 2H), 7.39 (s, 1H), 6.32 (s, 2H), 5.91 (s, 1H), 2.13 (s, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.1, 163.2, 161.5, 143.36, 139.9, 138.9, 134.4, 134.3 132.9, 132.7, 126.4, 119.5, 118.4, 117.0, 116.2, 95.8, 24.0 (CF<sub>3</sub> carbon not visible). LC/MS (ESI) calcd for C<sub>18</sub>H<sub>14</sub>Cl<sub>2</sub>F<sub>3</sub>N<sub>4</sub> (M + H)<sup>+</sup> 413.1, found 413.1.</div></div><div id="sec4-70" class="NLM_sec NLM_sec_level_2"><div id="ac_i97" class="anchor-spacer"></div><h3 class="article-section__title" id="_i97"> <i>N</i><sup>4</sup>-(5-Chloro-3′,5′-bis(trifluoromethyl)-[1,1′-biphenyl]-3-yl)-6-methylpyrimidine-2,4-diamine (<b>106</b>)</h3><div class="NLM_p last">According to general procedure D, bromopyrimidineamine <b>32</b> (0.3 g, 0.95 mmol) yielded compound <b>106</b> as a tan solid (0.26 g, 61%); mp 159–160 °C. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.13 (s, 2H), 8.08 (t, <i>J</i> = 1.8 Hz, 1H), 7.96 (s, 1H), 7.67 (t, <i>J</i> = 1.8 Hz, 1H), 7.29 (t, <i>J</i> = 1.8 Hz, 1H), 5.94 (s, 1H), 2.17 (s, 3H). <sup>13</sup>C NMR (101 MHz, methanol-<i>d</i><sub>4</sub>) δ 165.3, 162.7, 161.8, 142.3, 142.3, 139.7, 135.0, 131.9 (q, <i>J</i><sub>CF</sub> = 33.2 Hz), 127.1, 122.0, 121.0, 120.1, 119.4, 116.3, 95.9, 21.8. LC/MS (ESI) calcd for C<sub>19</sub>H<sub>14</sub>ClF<sub>6</sub>N<sub>4</sub> (M + H)<sup>+</sup> 447.1, found 447.1.</div></div><div id="sec4-71" class="NLM_sec NLM_sec_level_2"><div id="ac_i98" class="anchor-spacer"></div><h3 class="article-section__title" id="_i98"> <i>N</i><sup>4</sup>-(3-Chloro-5-(naphthalen-2-yl)phenyl)-6-methylpyrimidine-2,4-diamine (<b>107</b>)</h3><div class="NLM_p last">According to general procedure D, bromopyrimidineamine <b>32</b> (0.3 g, 0.95 mmol) yielded compound <b>107</b> as tan solid (0.26 g, 75%); mp 189–190 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.30 (s, 1H), 8.20 (m, 1H), 8.04 (t, <i>J</i> = 1.8 Hz, 1H), 8.02–7.96 (m, 2H), 7.94–7.88 (m, 2H), 7.80 (m, 1H), 7.56–7.47 (m, 2H), 7.37 (t, <i>J</i> = 1.8 Hz, 1H), 6.31 (s, 2H), 5.90 (s, 1H), 2.10 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.0, 163.3, 161.6, 143.3, 142.5, 136.7, 134.3, 133.7, 132.9, 129.0, 128.8, 127.9, 126.9, 126.8, 126.0, 125.4, 119.6, 117.8, 116.3, 95.8, 23.9. C/MS (ESI) calcd for C<sub>21</sub>H<sub>18</sub>ClN<sub>4</sub> (M + H)<sup>+</sup> 361.1, found 361.1.</div></div><div id="sec4-72" class="NLM_sec NLM_sec_level_2"><div id="ac_i99" class="anchor-spacer"></div><h3 class="article-section__title" id="_i99"> <i>N</i><sup>4</sup>-(3-Chloro-5-(phenanthren-9-yl)phenyl)-6-methylpyrimidine-2,4-diamine (<b>108</b>)</h3><div class="NLM_p last">According to general procedure D, bromopyrimidineamine <b>32</b> (0.3 g, 0.95 mmol) yielded compound <b>108</b> as a white solid (0.28 g, 71%); mp 110–111 °C. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.75 (d, <i>J</i> = 8.4 Hz, 1H), 8.69 (d, <i>J</i> = 8.4 Hz, 1H), 7.94 (s, 1H), 7.90–7.78 (m, 2H), 7.64–7.46 (m, 6H), 7.05 (s, 1H), 5.89 (s, 1H), 2.12 (s, 3H). <sup>13</sup>C NMR (101 MHz, methanol-<i>d</i><sub>4</sub>) δ 164.98, 162.6, 161.9, 142.6, 141.4, 137.2, 133.9, 131.3, 130.6, 130.4, 129.9, 128.4, 127.1, 126.6, 126.6, 126.3, 126.3, 126.1, 123.0, 122.7, 122.1, 119.2, 118.3, 95.8, 21.8. LC/MS (ESI) calcd for C<sub>25</sub>H<sub>20</sub>ClN<sub>4</sub> (M + H)<sup>+</sup> 411.1, found 411.1.</div></div><div id="sec4-73" class="NLM_sec NLM_sec_level_2"><div id="ac_i100" class="anchor-spacer"></div><h3 class="article-section__title" id="_i100"> <i>N</i><sup>4</sup>-(3-(Benzo[<i>d</i>][1,3]dioxol-5-yl)-5-chlorophenyl)-6-methylpyrimidine-2,4-diamine (<b>109</b>)</h3><div class="NLM_p last">According to general procedure D, bromopyrimidineamine <b>32</b> (0.3 g, 0.95 mmol) yielded compound <b>109</b> a tan solid (0.27 g, 82%); mp 225–226 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.18 (s, 1H), 7.93 (t, <i>J</i> = 2.0 Hz, 1H), 7.64 (t, <i>J</i> = 2.0 Hz, 1H), 7.20 (t, <i>J</i> = 2.0 Hz, 1H), 7.16–7.08 (m, 2H), 6.96 (m, 1H), 6.26 (s, 2H), 6.04 (s, 2H), 5.87 (s, 1H), 2.08 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.0, 163.3, 161.6, 148.4, 147.7, 143.0, 142.4, 134.0, 133.6, 121.0, 119.2, 117.3, 115.9, 109.1, 107.6, 101.7, 95.7, 23.9. LC/MS (ESI) calcd for C<sub>18</sub>H<sub>16</sub>ClN<sub>4</sub>O<sub>2</sub> (M + H)<sup>+</sup> 355.1, found 355.1.</div></div><div id="sec4-74" class="NLM_sec NLM_sec_level_2"><div id="ac_i101" class="anchor-spacer"></div><h3 class="article-section__title" id="_i101"> <i>N</i><sup>4</sup>-(3-(Benzo[<i>c</i>][1,2,5]oxadiazol-5-yl)-5-chlorophenyl)-6-methylpyrimidine-2,4-diamine (<b>110</b>)</h3><div class="NLM_p last">According to general procedure E, bromopyrimidineamine <b>32</b> (0.3 g, 0.95 mmol) yielded compound <b>110</b> as a yellow solid (0.24 g, 71%); mp 234–235 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.32 (s, 1H), 8.29 (d, <i>J</i> = 1.6 Hz, 1H), 8.12 (m, 1H), 8.05–7.96 (m, 2H), 7.93 (m, 1H), 7.42 (d, <i>J</i> = 1.6 Hz, 1H), 6.34 (s, 2H), 5.89 (s, 1H), 2.10 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.2, 163.3, 161.5, 149.9, 148.9, 143.3, 143.1, 140.3, 134.4, 133.8, 119.9, 119.0, 117.15, 116.7, 113.3, 95.8, 24.0. LC/MS (ESI) calcd for C<sub>17</sub>H<sub>14</sub>ClN<sub>6</sub>O (M + H)<sup>+</sup> 353.1, found 353.1.</div></div><div id="sec4-75" class="NLM_sec NLM_sec_level_2"><div id="ac_i102" class="anchor-spacer"></div><h3 class="article-section__title" id="_i102"> <i>N</i><sup>4</sup>-(3-Chloro-5-(furan-3-yl)phenyl)-6-methylpyrimidine-2,4-diamine (<b>111</b>)</h3><div class="NLM_p last">According to general procedure E, bromopyrimidineamine <b>32</b> (0.30 g, 0.95 mmol) yielded compound <b>111</b> as a white foam (0.21 g, 72%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.17 (s, 1H), 8.24 (s, 1H), 7.91 (m, 1H), 7.72 (t, <i>J</i> = 1.8 Hz, 1H), 7.66 (t, <i>J</i> = 1.8 Hz, 1H), 7.21 (t, <i>J</i> = 1.8 Hz, 1H), 6.95 (m, 1H), 6.32 (s, 2H), 5.86 (s, 1H), 2.08 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.9, 163.3, 161.6, 144.8, 143.2, 140.6, 134.5, 134.0, 125.4, 118.2, 117.1, 114.5, 109.2, 95.3, 23.9. LC/MS (ESI) calcd for C<sub>15</sub>H<sub>14</sub>ClN<sub>4</sub>O (M + H)<sup>+</sup> 301.1, found 301.1.</div></div><div id="sec4-76" class="NLM_sec NLM_sec_level_2"><div id="ac_i103" class="anchor-spacer"></div><h3 class="article-section__title" id="_i103"> <i>N</i><sup>4</sup>-(3-(Benzofuran-2-yl)-5-chlorophenyl)-6-methylpyrimidine-2,4-diamine (<b>112</b>)</h3><div class="NLM_p last">According to general procedure E, bromopyrimidineamine <b>32</b> (0.3 g, 0.95 mmol) yielded compound <b>112</b> as an off-white foam (0.23 g, 69%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.36 (s, 1H), 8.09 (t, <i>J</i> = 1.8 Hz, 1H), 8.01 (t, <i>J</i> = 1.8 Hz, 1H), 7.72–7.58 (m, 2H), 7.52 (s, 1H), 7.48 (t, <i>J</i> = 1.8 Hz, 1H), 7.32 (t, <i>J</i> = 7.5 Hz, 1H), 7.25 (t, <i>J</i> = 7.5 Hz, 1H), 6.34 (s, 2H), 5.89 (s, 1H), 2.10 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.1, 163.2, 161.5, 154.7, 154.2, 143.4, 134.4, 132.0, 129.1, 125.4, 123.8, 121.8, 118.6, 117.0, 113.4, 111.7, 103.7, 95.9, 23.4. LC/MS (ESI) calcd for C<sub>19</sub>H<sub>15</sub>ClN<sub>4</sub>O (M + H)<sup>+</sup> 351.1, found 351.1.</div></div><div id="sec4-77" class="NLM_sec NLM_sec_level_2"><div id="ac_i104" class="anchor-spacer"></div><h3 class="article-section__title" id="_i104"> <i>N</i><sup>4</sup>-(3-Chloro-5-(thiophen-3-yl)phenyl)-6-methylpyrimidine-2,4-diamine (<b>113</b>)</h3><div class="NLM_p last">According to general procedure E, bromopyrimidineamine <b>32</b> (0.3 g, 0.95 mmol) yielded compound <b>113</b> as a yellow foam (0.22 g, 74%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.21 (s, 1H), 7.97 (s, 1H), 7.92 (s, 1H), 7.78 (d, <i>J</i> = 2.2 Hz, 1H), 7.60 (m, 1H), 7.53 (s, 1H), 7.30 (d, <i>J</i> = 2.2 Hz, 1H), 6.34 (s, 2H), 5.89 (s, 1H), 2.10 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.0, 163.3, 161.6, 143.2, 140.6, 137.5, 134.1, 127.7 126.6, 122.6, 118.76, 117.3, 115.2, 95.8, 23.9. LC/MS (ESI) calcd for C<sub>15</sub>H<sub>14</sub>ClN<sub>4</sub>S (M + H)<sup>+</sup> 317.1, found 317.1.</div></div><div id="sec4-78" class="NLM_sec NLM_sec_level_2"><div id="ac_i105" class="anchor-spacer"></div><h3 class="article-section__title" id="_i105"> <i>N</i><sup>4</sup>-(3-(Benzo[<i>b</i>]thiophen-2-yl)-5-chlorophenyl)-6-methylpyrimidine-2,4-diamine (<b>114</b>)</h3><div class="NLM_p last">According to general procedure E, bromopyrimidineamine <b>32</b> (0.3 g, 0.95 mmol) yielded compound <b>114</b> as an off-white foam (0.21 g, 64%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.35 (s, 1H), 8.01–7.90 (m, 4H), 7.84 (m, 1H), 7.38–7.34 (m, 3H), 6.31 (s, 2H), 5.89 (s, 1H), 2.10 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.2, 163.2, 161.5, 143.4, 142.2, 140.7, 139.1, 135.8, 134.4, 125.4, 125.3, 124.5, 122.9, 121.6, 118.3, 118.2, 115.1, 95.9, 23.9. LC/MS (ESI) calcd for C<sub>19</sub>H<sub>16</sub>ClN<sub>4</sub>S (M + H)<sup>+</sup> 367.1, found 367.1.</div></div><div id="sec4-79" class="NLM_sec NLM_sec_level_2"><div id="ac_i106" class="anchor-spacer"></div><h3 class="article-section__title" id="_i106"> <i>N</i><sup>4</sup>-(3-Chloro-5-(thiazol-5-yl)phenyl)-6-methylpyrimidine-2,4-diamine (<b>115</b>)</h3><div class="NLM_p last">According to general procedure E bromopyrimidineamine <b>32</b> (0.30 g, 0.96 mmol) yielded compound <b>115</b> as a yellow solid (0.25 g, 81%); mp 220–221 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.29 (s, 1H), 9.09 (s, 1H), 8.36 (s, 1H), 7.95 (t, <i>J</i> = 1.8 Hz, 1H), 7.81 (t, <i>J</i> = 1.8 Hz, 1H), 7.30 (t, <i>J</i> = 1.8 Hz, 1H), 6.30 (s, 2H), 5.86 (s, 1H), 2.08 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.1, 163.2, 161.4, 154.6, 143.4, 140.7, 137.8, 134.4, 133.0, 118.6, 118.2, 115.6, 95.8, 23.9. LC/MS (ESI) calcd for C<sub>14</sub>H<sub>13</sub>ClN<sub>5</sub>S (M + H)<sup>+</sup> 318.1, found 318.1.</div></div><div id="sec4-80" class="NLM_sec NLM_sec_level_2"><div id="ac_i107" class="anchor-spacer"></div><h3 class="article-section__title" id="_i107"> <i>N</i><sup>4</sup>-(3-Chloro-5-(1-methyl-1<i>H</i>-pyrazol-4-yl)phenyl)-6-methylpyrimidine-2,4-diamine (<b>116</b>)</h3><div class="NLM_p last">According to general procedure D, bromopyrimidineamine <b>32</b> (0.3 g, 0.95 mmol) yielded compound <b>116</b> as a white solid (0.29 g, 88%); mp 204–205 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.13 (s, 1H), 8.15 (s, 1H), 7.98–7.79 (m, 2H), 7.54 (s, 1H), 7.15 (s, 1H), 6.29 (s, 2H), 5.85 (s, 1H), 3.84 (s, 3H), 2.08 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.9, 163.23, 161.6, 143.2, 136.7, 135.1, 133.9, 128.8, 121.3, 117. 6, 116.1, 113.9, 95.7, 39.1, 23.9. LC/MS (ESI) calcd for C<sub>15</sub>H<sub>16</sub>ClN<sub>6</sub> (M + H)<sup>+</sup> 315.1, found 315.1.</div></div><div id="sec4-81" class="NLM_sec NLM_sec_level_2"><div id="ac_i108" class="anchor-spacer"></div><h3 class="article-section__title" id="_i108"> <i>N</i><sup>4</sup>-(3-Chloro-5-(pyrimidin-5-yl)phenyl)-6-methylpyrimidine-2,4-diamine (<b>117</b>)</h3><div class="NLM_p last">According to general procedure D, bromopyrimidineamine <b>32</b> (0.3 g, 0.95 mmol) yielded compound <b>117</b> as a white solid (0.25 g, 84%); mp 184–185 °C. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.32 (s, 1H), 9.19 (s, 1H), 9.10 (s, 2H), 7.99 (s, 1H), 7.90 (s, 1H), 7.39 (s, 1H), 6.33 (s, 2H), 5.87 (s, 1H), 2.08 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.7, 161.7, 157.5, 155.0, 147.0, 146.0, 145.8, 142.9, 136.3, 119.8, 119.1, 116.1, 95.9, 22.5. LC/MS (ESI) calcd for C<sub>15</sub>H<sub>14</sub>ClN<sub>6</sub> (M + H)<sup>+</sup> 313.1, found 313.1.</div></div><div id="sec4-82" class="NLM_sec NLM_sec_level_2"><div id="ac_i109" class="anchor-spacer"></div><h3 class="article-section__title" id="_i109"> <i>N</i><sup>4</sup>-(5-Ethyl-4′-fluoro-[1,1′-biphenyl]-3-yl)-6-methylpyrimidine-2,4-diamine (<b>118</b>)</h3><div class="NLM_p last">According to general procedure D, bromopyrimidineamine <b>38</b> (0.21 g, 0.70 mmol) yielded compound <b>118</b> a tan oil (0.19 g, 83%). <sup>1</sup>H NMR (500 MHz, methanol-<i>d</i><sub>4</sub>) δ 7.65–7.59 (m, 3H), 7.36 (t, <i>J</i> = 1.7 Hz, 1H), 7.19–7.13 (m, 2H), 7.10 (t, <i>J</i> = 1.7 Hz, 1H), 5.97 (s, 1H), 2.70 (q, <i>J</i> = 7.6 Hz, 2H), 2.18 (s, 3H), 1.29 (t, <i>J</i> = 7.6 Hz, 3H). <sup>13</sup>C NMR (126 MHz, methanol-<i>d</i><sub>4</sub>) δ 164.8, 163.4, 162.3, 162.1 (d, <i>J</i><sub><i>CF</i></sub> = 162.1 Hz), 145.5, 140.7, 140.2, 137.5 (d, <i>J</i><sub><i>CF</i></sub> = 3.3 Hz), 128.4 (d, <i>J</i><sub><i>CF</i></sub> = 8.0 Hz), 120.9, 119.0, 116.7, 115.0 (d, <i>J</i><sub><i>CF</i></sub> = 21.5 Hz), 95.2, 28.6, 21.9, 14.8. LC/MS (ESI) calcd for C<sub>19</sub>H<sub>20</sub>FN<sub>4</sub> (M + H)<sup>+</sup> 323.2, found 323.2.</div></div><div id="sec4-83" class="NLM_sec NLM_sec_level_2"><div id="ac_i110" class="anchor-spacer"></div><h3 class="article-section__title" id="_i110"> Biological Assays</h3><div id="sec4-83-1" class="NLM_sec NLM_sec_level_3"><div id="ac_i111" class="anchor-spacer"></div><h4 class="article-section__title" id="_i111"> Materials</h4><div class="NLM_p last">Reagents were purchased from Sigma-Aldrich (St. Louis, MO, USA) unless otherwise specified. <i>S</i>-Adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine (AdoMet) sulfate <i>p</i>-toluenesulfonate salt was purchased from Affymetrix (Santa Clara, CA, USA), Gibco Iscove’s modified Dulbecco’s medium (IMDM) was from Thermo Fisher Scientific (Waltham, MA, USA). Compound <b>1</b> was kindly provided by Genzyme (now Sanofi-Genzyme, Waltham, MA, USA). Pyrimidineamine compound stocks were made in dimethyl sulfoxide (DMSO), with an initial stock concentration of 25 or 50 mM (or the highest achievable concentration for compounds with lower solubility). Stocks were stored at −20 °C and thawed once to generate fresh solutions in DMSO for enzyme and cell-based assays. DMSO based dilution series for the enzyme assays were generated using Echo 555 acoustic liquid dispenser (Labcyte, Sunnyvale, CA, USA); final DMSO concentrations in the assays were 2%. Cell growth inhibition was determined for a 3-fold serial dilution series generated in DMSO so that final DMSO concentrations in the assay were 0.1%.</div></div><div id="sec4-83-2" class="NLM_sec NLM_sec_level_3"><div id="ac_i112" class="anchor-spacer"></div><h4 class="article-section__title" id="_i112"> Protein Expression and Purification</h4><div class="NLM_p last">Heterologous expression and purification of recombinant <i>Tb</i>AdoMetDC/Prozyme heterodimer and <i>Hs</i>AdoMetDC used in the enzyme activity assays and for structure determination were carried out as previously described.<a onclick="showRef(event, 'ref21 ref47'); return false;" href="javascript:void(0);" class="ref ref21 ref47">(21, 47)</a> The purified <i>Tb</i>AdoMetDC/Prozyme complex was concentrated to 73 mg/mL and stored at −80 °C.</div></div><div id="sec4-83-3" class="NLM_sec NLM_sec_level_3"><div id="ac_i113" class="anchor-spacer"></div><h4 class="article-section__title" id="_i113"> Mass Spectrometry Based AdoMetDC Activity Assay</h4><div class="NLM_p last">The steady-state activity of <i>Tb</i>AdoMetDC/Prozyme and <i>Hs</i>AdoMetDC was measured using RapidFire–MS-based assay as previously described.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Inhibition data were acquired in a concentration–response format based on a 3-fold dilution series over the range from 0.01 μM to 180 μM final of inhibitor. Data were normalized to a fully inhibited control (0.5 μM <b>1</b>) and fitted to the normalized response versus log(inhibitor concentration) equation in Prism constraining bottom and Hill slope to 0 and −1, respectively, to determine the IC<sub>50</sub>.</div></div><div id="sec4-83-4" class="NLM_sec NLM_sec_level_3"><div id="ac_i114" class="anchor-spacer"></div><h4 class="article-section__title" id="_i114"> Parasite Culture, Cell Viability Assay, and Western Blot Analysis</h4><div class="NLM_p last"><i>T. brucei brucei</i> Lister 427 bloodstream-form parasites were cultured in HMI-19 medium supplemented with fetal bovine serum (Atlanta Biologicals, Flowery Branch, GA, USA) in 96-well plates for the cell viability assay or in culture flasks for the Prozyme Western blot analysis, as previously described.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Cell densities in the cell viability assay were analyzed using ATP-bioluminescence assay with CellTiter-Glo reagent (Promega, Madison, WI, USA) after 48 h culturing in the presence of serial dilutions of compound (ranging from 0.0038 to 25 μM or from 0.015 to 100 μM final in the assay) and at a fixed concentration of vehicle (0.1% DMSO) as previously described.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> The data were fitted to the normalized response versus log(inhibitor concentration) equation in Prism constraining bottom to 0. Hill slope was constrained to −8 (based on the best-fit value of a four-parameter fit in Prism) when the steepness of the curve resulted in a fit with an ambiguous EC<sub>50</sub> value (see <a href="/doi/suppl/10.1021/acs.jmedchem.7b01654/suppl_file/jm7b01654_si_002.pdf" class="ext-link">Table S1</a>). The Western blot analysis of Prozyme levels in the treated cell lysates was performed as previously described.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></div></div><div id="sec4-83-5" class="NLM_sec NLM_sec_level_3"><div id="ac_i115" class="anchor-spacer"></div><h4 class="article-section__title" id="_i115"> In Vitro Metabolism and Distribution Analysis</h4><div class="NLM_p last">Compound metabolic stability was assessed using male CD-1 mouse and mixed-gender human hepatic microsomes as previously described.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> BBB permeability assay was done using MDCKII-hMDR1 monolayer cells as previously described.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></div></div></div><div id="sec4-84" class="NLM_sec NLM_sec_level_2"><div id="ac_i116" class="anchor-spacer"></div><h3 class="article-section__title" id="_i116"> X-ray Crystallography</h3><div id="sec4-84-1" class="NLM_sec NLM_sec_level_3"><div id="ac_i117" class="anchor-spacer"></div><h4 class="article-section__title" id="_i117"> Protein Crystallization and Data Collection</h4><div class="NLM_p last"><i>Tb</i>AdoMetDC/Prozyme was diluted to final concentration of 4 mg/mL into a crystallization buffer (50 mM Bis-Tris propane, pH 7.2, 50 mM NaCl, 4 mM tris(2-carboxyethyl)phosphine (TCEP), 2 mM putrescine) containing 0.5 mM <b>44</b> added as 50 mM DMSO stock yielding a final DMSO concentration of 1%. Protein and the inhibitor were allowed to equilibrate at room temperature for 3 h, after which the sample was filtered at 23 °C using Ultrafree-MC 0.22 μm PVDF centrifugal filter units (Merck Millipore, Tullagreen, Ireland). Crystals were obtained using hanging-drop vapor diffusion method by combining 1 μL of the protein in crystallization buffer, 1 μL of reservoir solution, and 0.5 μL of unliganded protein microseeds obtained using seed–bead method (Hampton Research, Aliso Viejo, CA, USA) and diluted 50-fold as previously described.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Reservoir solution contained 17% PEG 6000 and 100 mM Bis-Tris propane, pH 7.0. Crystals formed on day 5 and were prepared for data collection by cryoprotection in a combination of the crystallization buffer and reservoir solution supplemented with 18% ethylene glycol. Crystals were then mounted and cryochilled in liquid nitrogen. Diffraction data were collected at the Br K-edge at the Structure Biology Center at the Advance Photon Source, beamline 19-ID at 100 K (see <a href="/doi/suppl/10.1021/acs.jmedchem.7b01654/suppl_file/jm7b01654_si_002.pdf" class="ext-link">Table S2</a>). Data reduction was carried out by HKL package.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a></div></div><div id="sec4-84-2" class="NLM_sec NLM_sec_level_3"><div id="ac_i118" class="anchor-spacer"></div><h4 class="article-section__title" id="_i118"> Structure Determination and Model Refinement</h4><div class="NLM_p last">Previously solved atomic coordinates of <i>Tb</i>AdoMetDC/Prozyme (PDB identifier <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TVM">5TVM</a>)<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> were used as a search model for molecular replacement in Phaser<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> as implemented in Phenix (version 1.10.1-2155).<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> The coordinates and geometry restraints for <b>44</b> were generated from a chemical formula in Elbow.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> The ligand was placed after initial positional refinement in Phenix<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> on the basis of <i>mF</i><sub>o</sub>–<i>DF</i><sub>c</sub> positive density in the active site and the anomalous diffraction map (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01654/suppl_file/jm7b01654_si_002.pdf" class="ext-link">Figure S1</a>). The structure was further improved through alternating manual model building in Coot<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> and positional, individual <i>B</i>-factor, occupancy, and anomalous groups refinement in Phenix. Visualization of the atomic coordinates and electron densities was done using PyMOL molecular graphics system (version 1.8, Schrödinger).</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i119"><a href="/doi/suppl/10.1021/acs.jmedchem.7b01654">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38280" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38280" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.7b01654" class="ext-link">10.1021/acs.jmedchem.7b01654</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Molecular formula strings and assay data (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01654/suppl_file/jm7b01654_si_001.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Table S1 with structures and biological assay data for all tested compounds, Table S2 with X-ray crystallography data collection and refinement statistics, Figure S1 with electron density maps supporting ligand placement in the X-ray crystal structure, and H and <sup>13</sup>C NMR spectra of synthesized analogs (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01654/suppl_file/jm7b01654_si_002.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Coordinates information for structure representation (<a href="/doi/suppl/10.1021/acs.jmedchem.7b01654/suppl_file/jm7b01654_si_003.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01654/suppl_file/jm7b01654_si_001.csv">jm7b01654_si_001.csv (15.63 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01654/suppl_file/jm7b01654_si_002.pdf">jm7b01654_si_002.pdf (5.55 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01654/suppl_file/jm7b01654_si_003.pdb">jm7b01654_si_003.pdb (1.55 MB)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The crystal structure of <i>Trypanosoma brucei</i> AdoMetDC/prozyme heterodimer in complex with pyrimidineamine inhibitor <b>44</b> was deposited at the RCSB Protein Data Bank under accession number <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BM7">6BM7</a> (authors O. A. Volkov, Z. Chen, and M. A. Phillips). Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.7b01654" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65673" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65673" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Margaret A. Phillips</span> - <span class="hlFld-Affiliation affiliation">Departments of †Pharmacology, ‡Biochemistry, and §Biophysics, University of Texas Southwestern
Medical Center, 5323 Harry Hines Boulevard, Dallas, Texas 75390-9038, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Scynexis,
Inc. (now Avista Pharma Solutions), 3501 Tricenter Boulevard, Suite C, Durham, North Carolina 27713, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5250-5578" title="Orcid link">http://orcid.org/0000-0001-5250-5578</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#733e120114120116075d231b1a1f1f1a0300332627201c06071b0416000716011d5d161706"><span class="__cf_email__" data-cfemail="1954786b7e786b7c6d37497170757570696a594c4d4a766c6d716e7c6a6d7c6b77377c7d6c">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jef K. De Brabander</span> - <span class="hlFld-Affiliation affiliation">Departments of †Pharmacology, ‡Biochemistry, and §Biophysics, University of Texas Southwestern
Medical Center, 5323 Harry Hines Boulevard, Dallas, Texas 75390-9038, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Scynexis,
Inc. (now Avista Pharma Solutions), 3501 Tricenter Boulevard, Suite C, Durham, North Carolina 27713, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8816-5798" title="Orcid link">http://orcid.org/0000-0001-8816-5798</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#ace6c9ca82e8c9eedecdcecdc2c8c9deecf9f8ffc3d9d8c4dbc9dfd8c9dec282c9c8d9"><span class="__cf_email__" data-cfemail="0b416e6d254f6e49796a696a656f6e794b5e5f58647e7f637c6e787f6e7965256e6f7e">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Oleg A. Volkov</span> - <span class="hlFld-Affiliation affiliation">Departments of †Pharmacology, ‡Biochemistry, and §Biophysics, University of Texas Southwestern
Medical Center, 5323 Harry Hines Boulevard, Dallas, Texas 75390-9038, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Scynexis,
Inc. (now Avista Pharma Solutions), 3501 Tricenter Boulevard, Suite C, Durham, North Carolina 27713, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anthony J. Brockway</span> - <span class="hlFld-Affiliation affiliation">Departments of †Pharmacology, ‡Biochemistry, and §Biophysics, University of Texas Southwestern
Medical Center, 5323 Harry Hines Boulevard, Dallas, Texas 75390-9038, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Scynexis,
Inc. (now Avista Pharma Solutions), 3501 Tricenter Boulevard, Suite C, Durham, North Carolina 27713, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephen A. Wring</span> - <span class="hlFld-Affiliation affiliation">Scynexis,
Inc. (now Avista Pharma Solutions), 3501 Tricenter Boulevard, Suite C, Durham, North Carolina 27713, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael Peel</span> - <span class="hlFld-Affiliation affiliation">Scynexis,
Inc. (now Avista Pharma Solutions), 3501 Tricenter Boulevard, Suite C, Durham, North Carolina 27713, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhe Chen</span> - <span class="hlFld-Affiliation affiliation">Departments of †Pharmacology, ‡Biochemistry, and §Biophysics, University of Texas Southwestern
Medical Center, 5323 Harry Hines Boulevard, Dallas, Texas 75390-9038, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Scynexis,
Inc. (now Avista Pharma Solutions), 3501 Tricenter Boulevard, Suite C, Durham, North Carolina 27713, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>O.A.V. and A.J.B. contributed equally to this work</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i122">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05377" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05377" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by National Institutes of Health Grants 2R37AI034432 (to M.A.P.) and R01AI090599 (to M.A.P. and J.K.D.B.). M.A.P. and J.K.D.B. acknowledge the support of the Welch Foundation (Grants I-1257 and I-1422). M.A.P. holds the Sam G. Winstead and F. Andrew Bell Distinguished Chair in Biochemistry, and J.K.D.B. holds the Julie and Louis Beecherl, Jr. Chair in Medical Science. The authors thank Sara Schock for help with in vitro permeability studies and Thomas E. Richardson, both of Scynexis, Inc. (now Avista Pharma Solutions) for helpful medicinal chemistry discussions. The authors also thank, from UT Southwestern, Diana R. Tomchick for advice on crystallization studies, Karen MacMillan for her help in preparing the manuscript, Melissa McCoy and Bruce A. Posner for the RapidFire–mass spectrometry analysis of the enzyme assay, Lin You and Chuo Chen for LC–MS analysis of compound purity, and Shihua Zhong for performing parasite viability assays. Structural data shown in this report are derived from work performed at Argonne National Laboratory, Structural Biology Center at the Advanced Photon Source. Argonne is operated by UChicago Argonne, LLC, for the U.S. Department of Energy, Office of Biological and Environmental Research under Contract DE-AC02-06CH11357.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i123" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i123"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i124" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i124"> Abbreviations Used</h2><tr><td class="NLM_term">HAT</td><td class="NLM_def"><p class="first last">human African trypanosomiasis</p></td></tr><tr><td class="NLM_term">AdoMet</td><td class="NLM_def"><p class="first last"><i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine</p></td></tr><tr><td class="NLM_term">AdoMetDC</td><td class="NLM_def"><p class="first last"><i>S</i>-adenosylmethionine decarboxylase</p></td></tr><tr><td class="NLM_term">dcAdoMet</td><td class="NLM_def"><p class="first last">decarboxylated <i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine</p></td></tr><tr><td class="NLM_term"><i>Tb</i></td><td class="NLM_def"><p class="first last"><i>T. brucei</i></p></td></tr><tr><td class="NLM_term"><i>Hs</i></td><td class="NLM_def"><p class="first last"><i>Homo sapiens</i></p></td></tr><tr><td class="NLM_term">BBB</td><td class="NLM_def"><p class="first last">blood–brain barrier</p></td></tr><tr><td class="NLM_term">CNS</td><td class="NLM_def"><p class="first last">central nervous system</p></td></tr><tr><td class="NLM_term">NECT</td><td class="NLM_def"><p class="first last">nifurtimox–eflornithine combination therapy</p></td></tr><tr><td class="NLM_term">HTS</td><td class="NLM_def"><p class="first last">high-throughput screening</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">MS</td><td class="NLM_def"><p class="first last">mass spectrometry</p></td></tr><tr><td class="NLM_term">Pgp</td><td class="NLM_def"><p class="first last">P-glycoprotein 1</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">Pvl</td><td class="NLM_def"><p class="first last">pyruvoyl prosthetic group</p></td></tr><tr><td class="NLM_term">PDB</td><td class="NLM_def"><p class="first last">Protein Data Bank.</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i125">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06641" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06641" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 52 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><a href="http://www.who.int/neglected_diseases/diseases/en/" class="extLink">http://www.who.int/neglected_diseases/diseases/en/</a> (accessed Jun 12,<span class="NLM_x"> </span><span class="NLM_year">2017</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=http%3A%2F%2Fwww.who.int%2Fneglected_diseases%2Fdiseases%2Fen%2F+%28accessed+Jun+12%2C+2017%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Kennedy, P. G. E.</span><span> </span><span class="NLM_article-title">Clinical features, diagnosis, and treatment of human African trypanosomiasis (sleeping sickness)</span> <span class="citation_source-journal">Lancet Neurol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">186</span><span class="NLM_x">–</span> <span class="NLM_lpage">194</span><span class="refDoi"> DOI: 10.1016/S1474-4422(12)70296-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1016%2FS1474-4422%2812%2970296-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=23260189" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A280%3ADC%252BC3s3ktVCgsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=186-194&author=P.+G.+E.+Kennedy&title=Clinical+features%2C+diagnosis%2C+and+treatment+of+human+African+trypanosomiasis+%28sleeping+sickness%29&doi=10.1016%2FS1474-4422%2812%2970296-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical features, diagnosis, and treatment of human African trypanosomiasis (sleeping sickness)</span></div><div class="casAuthors">Kennedy Peter Ge</div><div class="citationInfo"><span class="NLM_cas:title">The Lancet. Neurology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">186-94</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Human African trypanosomiasis, or sleeping sickness, is caused by infection with parasites of the genus Trypanosoma, transmitted by the tsetse fly.  The disease has two forms, Trypanosoma brucei (T b) rhodesiense and T b gambiense; and is almost always fatal if untreated.  Despite a recent reduction in the number of reported cases, patients with African trypanosomiasis continue to present major challenges to clinicians.  Because treatment for CNS-stage disease can be very toxic, diagnostic staging to distinguish early-stage from late-stage disease when the CNS in invaded is crucial but remains problematic.  Melarsoprol is the only available treatment for late-stage T b rhodesiense infection, but can be lethal to 5% of patients owing to post-treatment reactive encephalopathy.  Eflornithine combined with nifurtimox is the first-line treatment for late-stage T b gambiense.  New drugs are in the pipeline for treatment of CNS human African trypanosomiasis, giving rise to cautious optimism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSDi0PRo3CXkCEJCPKhJPuqfW6udTcc2ear0TKtxLNBRbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3s3ktVCgsQ%253D%253D&md5=c9ac342f345240d413300c17cd1ae80f</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2FS1474-4422%2812%2970296-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1474-4422%252812%252970296-X%26sid%3Dliteratum%253Aachs%26aulast%3DKennedy%26aufirst%3DP.%2BG.%2BE.%26atitle%3DClinical%2520features%252C%2520diagnosis%252C%2520and%2520treatment%2520of%2520human%2520African%2520trypanosomiasis%2520%2528sleeping%2520sickness%2529%26jtitle%3DLancet%2520Neurol.%26date%3D2013%26volume%3D12%26spage%3D186%26epage%3D194%26doi%3D10.1016%2FS1474-4422%2812%2970296-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Franco, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simarro, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diarra, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jannin, J. G.</span><span> </span><span class="NLM_article-title">Epidemiology of human African trypanosomiasis</span> <span class="citation_source-journal">Clin. Epidemiol</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">257</span><span class="NLM_x">–</span> <span class="NLM_lpage">275</span><span class="refDoi"> DOI: 10.2147/CLEP.S39728</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.2147%2FCLEP.S39728" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=25125985" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A280%3ADC%252BC2M%252FitFOhtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=257-275&author=J.+R.+Francoauthor=P.+P.+Simarroauthor=A.+Diarraauthor=J.+G.+Jannin&title=Epidemiology+of+human+African+trypanosomiasis&doi=10.2147%2FCLEP.S39728"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Epidemiology of human African trypanosomiasis</span></div><div class="casAuthors">Franco Jose R; Simarro Pere P; Jannin Jean G; Diarra Abdoulaye</div><div class="citationInfo"><span class="NLM_cas:title">Clinical epidemiology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">257-75</span>
        ISSN:<span class="NLM_cas:issn">1179-1349</span>.
    </div><div class="casAbstract">Human African trypanosomiasis (HAT), or sleeping sickness, is caused by Trypanosoma brucei gambiense, which is a chronic form of the disease present in western and central Africa, and by Trypanosoma brucei rhodesiense, which is an acute disease located in eastern and southern Africa.  The rhodesiense form is a zoonosis, with the occasional infection of humans, but in the gambiense form, the human being is regarded as the main reservoir that plays a key role in the transmission cycle of the disease.  The gambiense form currently assumes that 98% of the cases are declared; the Democratic Republic of the Congo is the most affected country, with more than 75% of the gambiense cases declared.  The epidemiology of the disease is mediated by the interaction of the parasite (trypanosome) with the vectors (tsetse flies), as well as with the human and animal hosts within a particular environment.  Related to these interactions, the disease is confined in spatially limited areas called "foci", which are located in Sub-Saharan Africa, mainly in remote rural areas.  The risk of contracting HAT is, therefore, determined by the possibility of contact of a human being with an infected tsetse fly.  Epidemics of HAT were described at the beginning of the 20th century; intensive activities have been set up to confront the disease, and it was under control in the 1960s, with fewer than 5,000 cases reported in the whole continent.  The disease resurged at the end of the 1990s, but renewed efforts from endemic countries, cooperation agencies, and nongovernmental organizations led by the World Health Organization succeeded to raise awareness and resources, while reinforcing national programs, reversing the trend of the cases reported, and bringing the disease under control again.  In this context, sustainable elimination of the gambiense HAT, defined as the interruption of the transmission of the disease, was considered as a feasible target for 2030.  Since rhodesiense HAT is a zoonosis, where the animal reservoir plays a key role, the interruption of the disease's transmission is not deemed feasible.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRLQPTkbz605s25P7XSLhESfW6udTcc2ebt3if0sgDXxrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M%252FitFOhtw%253D%253D&md5=484f0de3184f034ce92654ee5baf2bd2</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.2147%2FCLEP.S39728&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FCLEP.S39728%26sid%3Dliteratum%253Aachs%26aulast%3DFranco%26aufirst%3DJ.%2BR.%26aulast%3DSimarro%26aufirst%3DP.%2BP.%26aulast%3DDiarra%26aufirst%3DA.%26aulast%3DJannin%26aufirst%3DJ.%2BG.%26atitle%3DEpidemiology%2520of%2520human%2520African%2520trypanosomiasis%26jtitle%3DClin.%2520Epidemiol%26date%3D2014%26volume%3D6%26spage%3D257%26epage%3D275%26doi%3D10.2147%2FCLEP.S39728" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><a href="http://www.who.int/mediacentre/factsheets/fs259/en/" class="extLink">http://www.who.int/mediacentre/factsheets/fs259/en/</a> (accessed Jun 12,<span class="NLM_x"> </span><span class="NLM_year">2017</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=http%3A%2F%2Fwww.who.int%2Fmediacentre%2Ffactsheets%2Ffs259%2Fen%2F+%28accessed+Jun+12%2C+2017%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Franco, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simarro, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diarra, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruiz-Postigo, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jannin, J. G.</span><span> </span><span class="NLM_article-title">The journey towards elimination of gambiense human African trypanosomiasis: not far, nor easy</span> <span class="citation_source-journal">Parasitology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">141</span><span class="NLM_x">, </span> <span class="NLM_fpage">748</span><span class="NLM_x">–</span> <span class="NLM_lpage">760</span><span class="refDoi"> DOI: 10.1017/S0031182013002102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1017%2FS0031182013002102" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2014&pages=748-760&author=J.+R.+Francoauthor=P.+P.+Simarroauthor=A.+Diarraauthor=J.+A.+Ruiz-Postigoauthor=J.+G.+Jannin&title=The+journey+towards+elimination+of+gambiense+human+African+trypanosomiasis%3A+not+far%2C+nor+easy&doi=10.1017%2FS0031182013002102"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1017%2FS0031182013002102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS0031182013002102%26sid%3Dliteratum%253Aachs%26aulast%3DFranco%26aufirst%3DJ.%2BR.%26aulast%3DSimarro%26aufirst%3DP.%2BP.%26aulast%3DDiarra%26aufirst%3DA.%26aulast%3DRuiz-Postigo%26aufirst%3DJ.%2BA.%26aulast%3DJannin%26aufirst%3DJ.%2BG.%26atitle%3DThe%2520journey%2520towards%2520elimination%2520of%2520gambiense%2520human%2520African%2520trypanosomiasis%253A%2520not%2520far%252C%2520nor%2520easy%26jtitle%3DParasitology%26date%3D2014%26volume%3D141%26spage%3D748%26epage%3D760%26doi%3D10.1017%2FS0031182013002102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Berthier, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breniere, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bras-Goncalves, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lemesre, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jamonneau, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solano, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lejon, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thevenon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bucheton, B.</span><span> </span><span class="NLM_article-title">Tolerance to trypanosomatids: a threat, or a key for disease elimination?</span> <span class="citation_source-journal">Trends Parasitol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">157</span><span class="NLM_x">–</span> <span class="NLM_lpage">168</span><span class="refDoi"> DOI: 10.1016/j.pt.2015.11.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1016%2Fj.pt.2015.11.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=26643519" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A280%3ADC%252BC28votlOktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2016&pages=157-168&author=D.+Berthierauthor=S.+F.+Breniereauthor=R.+Bras-Goncalvesauthor=J.+L.+Lemesreauthor=V.+Jamonneauauthor=P.+Solanoauthor=V.+Lejonauthor=S.+Thevenonauthor=B.+Bucheton&title=Tolerance+to+trypanosomatids%3A+a+threat%2C+or+a+key+for+disease+elimination%3F&doi=10.1016%2Fj.pt.2015.11.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Tolerance to Trypanosomatids: A Threat, or a Key for Disease Elimination?</span></div><div class="casAuthors">Berthier David; Thevenon Sophie; Breniere Simone F; Bras-Goncalves Rachel; Lemesre Jean-Loup; Jamonneau Vincent; Solano Philippe; Lejon Veerle; Bucheton Bruno</div><div class="citationInfo"><span class="NLM_cas:title">Trends in parasitology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">157-68</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">So far, research on trypanosomatid infections has been driven by 'disease by disease' approaches, leading to different concepts and control strategies.  It is, however, increasingly clear that they share common features such as the ability to generate long-lasting asymptomatic infections in their mammalian hosts.  Trypanotolerance, long integrated in animal African trypanosomiasis control, historically refers to the ability of cattle breeds to limit Trypanosoma infection and pathology, but has only recently been recognized in humans.  Whilst trypanotolerance is absent from the vocabulary on leishmaniasis and Chagas disease, asymptomatic infections also occur.  We review the concept of trypanotolerance across the trypanosomatids and discuss the importance of asymptomatic carriage in the current context of elimination.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSwxKZVEIfebJdYoSR9eyvsfW6udTcc2ebt3if0sgDXxrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28votlOktQ%253D%253D&md5=7fcfa49a6aeb63f78dffb3150806be43</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.pt.2015.11.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pt.2015.11.001%26sid%3Dliteratum%253Aachs%26aulast%3DBerthier%26aufirst%3DD.%26aulast%3DBreniere%26aufirst%3DS.%2BF.%26aulast%3DBras-Goncalves%26aufirst%3DR.%26aulast%3DLemesre%26aufirst%3DJ.%2BL.%26aulast%3DJamonneau%26aufirst%3DV.%26aulast%3DSolano%26aufirst%3DP.%26aulast%3DLejon%26aufirst%3DV.%26aulast%3DThevenon%26aufirst%3DS.%26aulast%3DBucheton%26aufirst%3DB.%26atitle%3DTolerance%2520to%2520trypanosomatids%253A%2520a%2520threat%252C%2520or%2520a%2520key%2520for%2520disease%2520elimination%253F%26jtitle%3DTrends%2520Parasitol.%26date%3D2016%26volume%3D32%26spage%3D157%26epage%3D168%26doi%3D10.1016%2Fj.pt.2015.11.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Capewell, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cren-Travaille, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchesi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnston, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clucas, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benson, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorman, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calvo-Alvarez, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crouzols, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jouvion, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jamonneau, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weir, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevenson, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’neill, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooper, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swar, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bucheton, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ngoyi, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garside, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rotureau, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macleod, A.</span><span> </span><span class="NLM_article-title">The skin is a significant but overlooked anatomical reservoir for vector-borne African trypanosomes</span> <span class="citation_source-journal">eLife</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">e17716</span><span class="refDoi"> DOI: 10.7554/eLife.17716</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.7554%2FeLife.17716" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=27653219" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BC1cXovV2jsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&pages=e17716&author=P.+Capewellauthor=C.+Cren-Travailleauthor=F.+Marchesiauthor=P.+Johnstonauthor=C.+Clucasauthor=R.+A.+Bensonauthor=T.+A.+Gormanauthor=E.+Calvo-Alvarezauthor=A.+Crouzolsauthor=G.+Jouvionauthor=V.+Jamonneauauthor=W.+Weirauthor=M.+L.+Stevensonauthor=K.+O%E2%80%99neillauthor=A.+Cooperauthor=N.+K.+Swarauthor=B.+Buchetonauthor=D.+M.+Ngoyiauthor=P.+Garsideauthor=B.+Rotureauauthor=A.+Macleod&title=The+skin+is+a+significant+but+overlooked+anatomical+reservoir+for+vector-borne+African+trypanosomes&doi=10.7554%2FeLife.17716"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The skin is a significant but overlooked anatomical reservoir for vector-borne African trypanosomes</span></div><div class="casAuthors">Capewell, Paul; Cren-Travaille, Christelle; Marchesi, Francesco; Johnston, Pamela; Clucas, Caroline; Benson, Robert A.; Gorman, Taylor-Anne; Calvo-Alvarez, Estefania; Crouzols, Aline; Jouvion, Gregory; Jammoneau, Vincent; Weir, William; Stevenson, M. Lynn; O'Neill, Kerry; Cooper, Anneli; Swar, Nono-Raymond Kuispond; Bucheton, Bruno; Ngoyi, Dieudonne Mumba; Garside, Paul; Rotureau, Brice; MacLeod, Annette</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e17716/1-e17716/17</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">The role of mammalian skin in harbouring and transmitting arthropod-borne protozoan parasites has been overlooked for decades as these pathogens have been regarded primarily as blood-dwelling organisms.  Intriguingly, infections with low or undetected blood parasites are common, particularly in the case of Human African Trypanosomiasis caused by Trypanosoma brucei gambiense.  We hypothesise, therefore, the skin represents an anat. reservoir of infection.  Here we definitively show that substantial quantities of trypanosomes exist within the skin following exptl. infection, which can be transmitted to the tsetse vector, even in the absence of detectable parasitemia.  Importantly, we demonstrate the presence of extravascular parasites in human skin biopsies from undiagnosed individuals.  The identification of this novel reservoir requires a re-evaluation of current diagnostic methods and control policies.  More broadly, our results indicate that transmission is a key evolutionary force driving parasite extravasation that could further result in tissue invasion-dependent pathol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbxYT8sS1X77Vg90H21EOLACvtfcHk0lhXRHaJSn7ziQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXovV2jsLY%253D&md5=abb13f75ab2392c328f7e93bc925926e</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.7554%2FeLife.17716&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.17716%26sid%3Dliteratum%253Aachs%26aulast%3DCapewell%26aufirst%3DP.%26aulast%3DCren-Travaille%26aufirst%3DC.%26aulast%3DMarchesi%26aufirst%3DF.%26aulast%3DJohnston%26aufirst%3DP.%26aulast%3DClucas%26aufirst%3DC.%26aulast%3DBenson%26aufirst%3DR.%2BA.%26aulast%3DGorman%26aufirst%3DT.%2BA.%26aulast%3DCalvo-Alvarez%26aufirst%3DE.%26aulast%3DCrouzols%26aufirst%3DA.%26aulast%3DJouvion%26aufirst%3DG.%26aulast%3DJamonneau%26aufirst%3DV.%26aulast%3DWeir%26aufirst%3DW.%26aulast%3DStevenson%26aufirst%3DM.%2BL.%26aulast%3DO%25E2%2580%2599neill%26aufirst%3DK.%26aulast%3DCooper%26aufirst%3DA.%26aulast%3DSwar%26aufirst%3DN.%2BK.%26aulast%3DBucheton%26aufirst%3DB.%26aulast%3DNgoyi%26aufirst%3DD.%2BM.%26aulast%3DGarside%26aufirst%3DP.%26aulast%3DRotureau%26aufirst%3DB.%26aulast%3DMacleod%26aufirst%3DA.%26atitle%3DThe%2520skin%2520is%2520a%2520significant%2520but%2520overlooked%2520anatomical%2520reservoir%2520for%2520vector-borne%2520African%2520trypanosomes%26jtitle%3DeLife%26date%3D2016%26volume%3D5%26spage%3De17716%26doi%3D10.7554%2FeLife.17716" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Trindade, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rijo-Ferreira, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carvalho, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pinto-Neves, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guegan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aresta-Branco, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bento, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pinto, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Den Abbeele, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ribeiro, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dias, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Figueiredo, L. M.</span><span> </span><span class="NLM_article-title">Trypanosoma brucei parasites occupy and functionally adapt to the adipose tissue in mice</span> <span class="citation_source-journal">Cell Host Microbe</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">837</span><span class="NLM_x">–</span> <span class="NLM_lpage">848</span><span class="refDoi"> DOI: 10.1016/j.chom.2016.05.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1016%2Fj.chom.2016.05.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=27237364" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BC28Xosl2jtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2016&pages=837-848&author=S.+Trindadeauthor=F.+Rijo-Ferreiraauthor=T.+Carvalhoauthor=D.+Pinto-Nevesauthor=F.+Gueganauthor=F.+Aresta-Brancoauthor=F.+Bentoauthor=S.+A.+Youngauthor=A.+Pintoauthor=J.+Van+Den+Abbeeleauthor=R.+M.+Ribeiroauthor=S.+Diasauthor=T.+K.+Smithauthor=L.+M.+Figueiredo&title=Trypanosoma+brucei+parasites+occupy+and+functionally+adapt+to+the+adipose+tissue+in+mice&doi=10.1016%2Fj.chom.2016.05.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Trypanosoma brucei Parasites Occupy and Functionally Adapt to the Adipose Tissue in Mice</span></div><div class="casAuthors">Trindade, Sandra; Rijo-Ferreira, Filipa; Carvalho, Tania; Pinto-Neves, Daniel; Guegan, Fabien; Aresta-Branco, Francisco; Bento, Fabio; Young, Simon A.; Pinto, Andreia; Van Den Abbeele, Jan; Ribeiro, Ruy M.; Dias, Sergio; Smith, Terry K.; Figueiredo, Luisa M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Host & Microbe</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">837-848</span>CODEN:
                <span class="NLM_cas:coden">CHMECB</span>;
        ISSN:<span class="NLM_cas:issn">1931-3128</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Trypanosoma brucei is an extracellular parasite that causes sleeping sickness.  In mammalian hosts, trypanosomes are thought to exist in two major niches: early in infection, they populate the blood; later, they breach the blood-brain barrier.  Working with a well-established mouse model, we discovered that adipose tissue constitutes a third major reservoir for T. brucei.  Parasites from adipose tissue, here termed adipose tissue forms (ATFs), can replicate and were capable of infecting a naive animal.  ATFs were transcriptionally distinct from bloodstream forms, and the genes upregulated included putative fatty acid β-oxidn. enzymes.  Consistent with this, ATFs were able to utilize exogenous myristate and form β-oxidn. intermediates, suggesting that ATF parasites can use fatty acids as an external carbon source.  These findings identify the adipose tissue as a niche for T. brucei during its mammalian life cycle and could potentially explain the wt. loss assocd. with sleeping sickness.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCGv-3jWWQrrVg90H21EOLACvtfcHk0lhXRHaJSn7ziQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xosl2jtbw%253D&md5=5cc016076dcee97f48dc3eb0b268c036</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.chom.2016.05.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chom.2016.05.002%26sid%3Dliteratum%253Aachs%26aulast%3DTrindade%26aufirst%3DS.%26aulast%3DRijo-Ferreira%26aufirst%3DF.%26aulast%3DCarvalho%26aufirst%3DT.%26aulast%3DPinto-Neves%26aufirst%3DD.%26aulast%3DGuegan%26aufirst%3DF.%26aulast%3DAresta-Branco%26aufirst%3DF.%26aulast%3DBento%26aufirst%3DF.%26aulast%3DYoung%26aufirst%3DS.%2BA.%26aulast%3DPinto%26aufirst%3DA.%26aulast%3DVan%2BDen%2BAbbeele%26aufirst%3DJ.%26aulast%3DRibeiro%26aufirst%3DR.%2BM.%26aulast%3DDias%26aufirst%3DS.%26aulast%3DSmith%26aufirst%3DT.%2BK.%26aulast%3DFigueiredo%26aufirst%3DL.%2BM.%26atitle%3DTrypanosoma%2520brucei%2520parasites%2520occupy%2520and%2520functionally%2520adapt%2520to%2520the%2520adipose%2520tissue%2520in%2520mice%26jtitle%3DCell%2520Host%2520Microbe%26date%3D2016%26volume%3D19%26spage%3D837%26epage%3D848%26doi%3D10.1016%2Fj.chom.2016.05.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Tanowitz, H. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scherer, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mota, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Figueiredo, L. M.</span><span> </span><span class="NLM_article-title">Adipose tissue: a safe haven for parasites?</span> <span class="citation_source-journal">Trends Parasitol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">276</span><span class="NLM_x">–</span> <span class="NLM_lpage">284</span><span class="refDoi"> DOI: 10.1016/j.pt.2016.11.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1016%2Fj.pt.2016.11.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=28007406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A280%3ADC%252BC1c%252FnslCmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2017&pages=276-284&author=H.+B.+Tanowitzauthor=P.+E.+Schererauthor=M.+M.+Motaauthor=L.+M.+Figueiredo&title=Adipose+tissue%3A+a+safe+haven+for+parasites%3F&doi=10.1016%2Fj.pt.2016.11.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Adipose Tissue: A Safe Haven for Parasites?</span></div><div class="casAuthors">Tanowitz Herbert B; Scherer Philipp E; Mota Maria M; Figueiredo Luisa M</div><div class="citationInfo"><span class="NLM_cas:title">Trends in parasitology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">276-284</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Adipose tissue (AT) is no longer regarded as an inert lipid storage, but as an important central regulator in energy homeostasis and immunity.  Three parasite species are uniquely associated with AT during part of their life cycle: Trypanosoma cruzi, the causative agent of Chagas disease; Trypanosoma brucei, the causative agent of African sleeping sickness; and Plasmodium spp., the causative agents of malaria.  In AT, T. cruzi resides inside adipocytes, T. brucei is found in the interstitial spaces between adipocytes, while Plasmodium spp. infect red blood cells, which may adhere to the blood vessels supplying AT.  Here, we discuss how each parasite species adapts to this tissue environment and what the implications are for pathogenesis, clinical manifestations, and therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRFtbJdIvuLe0NoizSOkG_FfW6udTcc2ebNrTmnIqtZ4rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c%252FnslCmsA%253D%253D&md5=c1b93d6948329aad712d682b2f970906</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.pt.2016.11.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pt.2016.11.008%26sid%3Dliteratum%253Aachs%26aulast%3DTanowitz%26aufirst%3DH.%2BB.%26aulast%3DScherer%26aufirst%3DP.%2BE.%26aulast%3DMota%26aufirst%3DM.%2BM.%26aulast%3DFigueiredo%26aufirst%3DL.%2BM.%26atitle%3DAdipose%2520tissue%253A%2520a%2520safe%2520haven%2520for%2520parasites%253F%26jtitle%3DTrends%2520Parasitol.%26date%3D2017%26volume%3D33%26spage%3D276%26epage%3D284%26doi%3D10.1016%2Fj.pt.2016.11.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Simarro, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Franco, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diarra, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Postigo, J. a. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jannin, J.</span><span> </span><span class="NLM_article-title">Update on field use of the available drugs for the chemotherapy of human African trypanosomiasis</span> <span class="citation_source-journal">Parasitology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">139</span><span class="NLM_x">, </span> <span class="NLM_fpage">842</span><span class="NLM_x">–</span> <span class="NLM_lpage">846</span><span class="refDoi"> DOI: 10.1017/S0031182012000169</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1017%2FS0031182012000169" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2012&pages=842-846&author=P.+P.+Simarroauthor=J.+Francoauthor=A.+Diarraauthor=J.+a.+R.+Postigoauthor=J.+Jannin&title=Update+on+field+use+of+the+available+drugs+for+the+chemotherapy+of+human+African+trypanosomiasis&doi=10.1017%2FS0031182012000169"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1017%2FS0031182012000169&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS0031182012000169%26sid%3Dliteratum%253Aachs%26aulast%3DSimarro%26aufirst%3DP.%2BP.%26aulast%3DFranco%26aufirst%3DJ.%26aulast%3DDiarra%26aufirst%3DA.%26aulast%3DPostigo%26aufirst%3DJ.%2Ba.%2BR.%26aulast%3DJannin%26aufirst%3DJ.%26atitle%3DUpdate%2520on%2520field%2520use%2520of%2520the%2520available%2520drugs%2520for%2520the%2520chemotherapy%2520of%2520human%2520African%2520trypanosomiasis%26jtitle%3DParasitology%26date%3D2012%26volume%3D139%26spage%3D842%26epage%3D846%26doi%3D10.1017%2FS0031182012000169" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Iten, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mett, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Enyaru, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brun, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaminsky, R.</span><span> </span><span class="NLM_article-title">Alterations in ornithine decarboxylase characteristics account for tolerance of Trypanosoma brucei rhodesiense to D,L-alpha-difluoromethylornithine</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">1922</span><span class="NLM_x">–</span> <span class="NLM_lpage">1925</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1997&pages=1922-1925&author=M.+Itenauthor=H.+Mettauthor=A.+Evansauthor=J.+C.+Enyaruauthor=R.+Brunauthor=R.+Kaminsky&title=Alterations+in+ornithine+decarboxylase+characteristics+account+for+tolerance+of+Trypanosoma+brucei+rhodesiense+to+D%2CL-alpha-difluoromethylornithine"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DIten%26aufirst%3DM.%26aulast%3DMett%26aufirst%3DH.%26aulast%3DEvans%26aufirst%3DA.%26aulast%3DEnyaru%26aufirst%3DJ.%2BC.%26aulast%3DBrun%26aufirst%3DR.%26aulast%3DKaminsky%26aufirst%3DR.%26atitle%3DAlterations%2520in%2520ornithine%2520decarboxylase%2520characteristics%2520account%2520for%2520tolerance%2520of%2520Trypanosoma%2520brucei%2520rhodesiense%2520to%2520D%252CL-alpha-difluoromethylornithine%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1997%26volume%3D41%26spage%3D1922%26epage%3D1925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><a href="https://www.dndi.org/diseases-projects/portfolio/scyx-7158/" class="extLink">https://www.dndi.org/diseases-projects/portfolio/scyx-7158/</a> (accessed Jun 13,<span class="NLM_x"> </span><span class="NLM_year">2017</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=https%3A%2F%2Fwww.dndi.org%2Fdiseases-projects%2Fportfolio%2Fscyx-7158%2F+%28accessed+Jun+13%2C+2017%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><a href="https://www.dndi.org/diseases-projects/portfolio/fexinidazole/" class="extLink">https://www.dndi.org/diseases-projects/portfolio/fexinidazole/</a> (accessed Jun 13,<span class="NLM_x"> </span><span class="NLM_year">2017</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=https%3A%2F%2Fwww.dndi.org%2Fdiseases-projects%2Fportfolio%2Ffexinidazole%2F+%28accessed+Jun+13%2C+2017%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Jacobs, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nare, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, M. A.</span><span> </span><span class="NLM_article-title">State of the art in African trypanosome drug discovery</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1255</span><span class="NLM_x">–</span> <span class="NLM_lpage">1274</span><span class="refDoi"> DOI: 10.2174/156802611795429167</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.2174%2F156802611795429167" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=21401507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnt1CnsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=1255-1274&author=R.+T.+Jacobsauthor=B.+Nareauthor=M.+A.+Phillips&title=State+of+the+art+in+African+trypanosome+drug+discovery&doi=10.2174%2F156802611795429167"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">State of the art in African trypanosome drug discovery</span></div><div class="casAuthors">Jacobs, Robert T.; Nare, Bakela; Phillips, Margaret A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1255-1274</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  African sleeping sickness is endemic in sub-Saharan Africa where the WHO ests. that 60 million people are at risk for the disease.  Human African trypanosomiasis (HAT) is 100% fatal if untreated and the current drug therapies have significant limitations due to toxicity and difficult treatment regimes.  No new chem. agents were approved since eflornithine in 1990.  The pentamidine analog DB289, which was in late stage clin. trials for the treatment of early stage HAT recently failed due to toxicity issues.  A new protocol for the treatment of late-stage T. brucei gambiense that uses combination nifurtomox/eflornithine (NECT) was recently shown to have better safety and efficacy than eflornithine alone, while being easier to administer.  This breakthrough represents the only new therapy for HAT since the approval of eflornithine.  A no. of research programs are on going to exploit the unusual biochem. pathways in the parasite to identify new targets for target based drug discovery programs.  HTS efforts are also underway to discover new chem. entities through whole organism screening approaches.  A no. of inhibitors with anti-trypanosomal activity have been identified by both approaches, but none of the programs are yet at the stage of identifying a preclin. candidate.  This dire situation underscores the need for continued effort to identify new chem. agents for the treatment of HAT.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhIrKnBIULOLVg90H21EOLACvtfcHk0lhgYKx9gM4IJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnt1CnsLc%253D&md5=15f997caa236c0537e20241dce8ccc5f</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.2174%2F156802611795429167&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802611795429167%26sid%3Dliteratum%253Aachs%26aulast%3DJacobs%26aufirst%3DR.%2BT.%26aulast%3DNare%26aufirst%3DB.%26aulast%3DPhillips%26aufirst%3DM.%2BA.%26atitle%3DState%2520of%2520the%2520art%2520in%2520African%2520trypanosome%2520drug%2520discovery%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2011%26volume%3D11%26spage%3D1255%26epage%3D1274%26doi%3D10.2174%2F156802611795429167" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Willert, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, M. A.</span><span> </span><span class="NLM_article-title">Regulation and function of polyamines in African trypanosomes</span> <span class="citation_source-journal">Trends Parasitol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">66</span><span class="NLM_x">–</span> <span class="NLM_lpage">72</span><span class="refDoi"> DOI: 10.1016/j.pt.2011.11.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1016%2Fj.pt.2011.11.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=22192816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVCjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=66-72&author=E.+Willertauthor=M.+A.+Phillips&title=Regulation+and+function+of+polyamines+in+African+trypanosomes&doi=10.1016%2Fj.pt.2011.11.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation and function of polyamines in African trypanosomes</span></div><div class="casAuthors">Willert, Erin; Phillips, Margaret A.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Parasitology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">66-72</span>CODEN:
                <span class="NLM_cas:coden">TPRACT</span>;
        ISSN:<span class="NLM_cas:issn">1471-4922</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The polyamine biosynthetic pathway is an important drug target for the treatment of human African trypanosomiasis (HAT), raising interest in understanding polyamine function and their mechanism of regulation.  Polyamine levels are tightly controlled in mammalian cells, but similar regulatory mechanisms appear absent in trypanosomes.  Instead trypanosomatid S-adenosylmethionine decarboxylase (AdoMetDC), which catalyzes a key step in the biosynthesis of the polyamine spermidine, is activated by dimerization with an inducible protein termed prozyme.  Prozyme is an inactive paralog of the active AdoMetDC enzyme that evolved by gene duplication and is found only in the trypanosomatids.  In Trypanosoma brucei, AdoMetDC activity appears to be controlled by regulation of prozyme protein levels, potentially at the translational level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo26D7Br10oEbVg90H21EOLACvtfcHk0lhgYKx9gM4IJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVCjtbo%253D&md5=5535fe1c6340ac6c101261bc95bb1595</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.pt.2011.11.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pt.2011.11.001%26sid%3Dliteratum%253Aachs%26aulast%3DWillert%26aufirst%3DE.%26aulast%3DPhillips%26aufirst%3DM.%2BA.%26atitle%3DRegulation%2520and%2520function%2520of%2520polyamines%2520in%2520African%2520trypanosomes%26jtitle%3DTrends%2520Parasitol.%26date%3D2012%26volume%3D28%26spage%3D66%26epage%3D72%26doi%3D10.1016%2Fj.pt.2011.11.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Willert, E. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, M. A.</span><span> </span><span class="NLM_article-title">Regulated expression of an essential allosteric activator of polyamine biosynthesis in African trypanosomes</span> <span class="citation_source-journal">PLoS Pathog.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1000183</span><span class="refDoi"> DOI: 10.1371/journal.ppat.1000183</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1371%2Fjournal.ppat.1000183" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=e1000183&author=E.+K.+Willertauthor=M.+A.+Phillips&title=Regulated+expression+of+an+essential+allosteric+activator+of+polyamine+biosynthesis+in+African+trypanosomes&doi=10.1371%2Fjournal.ppat.1000183"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1000183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1000183%26sid%3Dliteratum%253Aachs%26aulast%3DWillert%26aufirst%3DE.%2BK.%26aulast%3DPhillips%26aufirst%3DM.%2BA.%26atitle%3DRegulated%2520expression%2520of%2520an%2520essential%2520allosteric%2520activator%2520of%2520polyamine%2520biosynthesis%2520in%2520African%2520trypanosomes%26jtitle%3DPLoS%2520Pathog.%26date%3D2008%26volume%3D4%26spage%3De1000183%26doi%3D10.1371%2Fjournal.ppat.1000183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Bacchi, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brun, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Croft, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alicea, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buhler, Y.</span><span> </span><span class="NLM_article-title">In vivo trypanocidal activities of new S-adenosylmethionine decarboxylase inhibitors</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">1448</span><span class="NLM_x">–</span> <span class="NLM_lpage">1453</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=1996&pages=1448-1453&author=C.+J.+Bacchiauthor=R.+Brunauthor=S.+L.+Croftauthor=K.+Aliceaauthor=Y.+Buhler&title=In+vivo+trypanocidal+activities+of+new+S-adenosylmethionine+decarboxylase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBacchi%26aufirst%3DC.%2BJ.%26aulast%3DBrun%26aufirst%3DR.%26aulast%3DCroft%26aufirst%3DS.%2BL.%26aulast%3DAlicea%26aufirst%3DK.%26aulast%3DBuhler%26aufirst%3DY.%26atitle%3DIn%2520vivo%2520trypanocidal%2520activities%2520of%2520new%2520S-adenosylmethionine%2520decarboxylase%2520inhibitors%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1996%26volume%3D40%26spage%3D1448%26epage%3D1453" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Bacchi, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barker, R. H.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodriguez, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirth, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rattendi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yarlett, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hendrick, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sybertz, E.</span><span> </span><span class="NLM_article-title">Trypanocidal activity of 8-methyl-5′-{[(Z)-4-aminobut-2-enyl]-(methylamino)}adenosine (Genz-644131), an adenosylmethionine decarboxylase inhibitor</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">3269</span><span class="NLM_x">–</span> <span class="NLM_lpage">3272</span><span class="refDoi"> DOI: 10.1128/AAC.00076-09</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1128%2FAAC.00076-09" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2009&pages=3269-3272&author=C.+J.+Bacchiauthor=R.+H.+Barkerauthor=A.+Rodriguezauthor=B.+Hirthauthor=D.+Rattendiauthor=N.+Yarlettauthor=C.+L.+Hendrickauthor=E.+Sybertz&title=Trypanocidal+activity+of+8-methyl-5%E2%80%B2-%7B%5B%28Z%29-4-aminobut-2-enyl%5D-%28methylamino%29%7Dadenosine+%28Genz-644131%29%2C+an+adenosylmethionine+decarboxylase+inhibitor&doi=10.1128%2FAAC.00076-09"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1128%2FAAC.00076-09&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00076-09%26sid%3Dliteratum%253Aachs%26aulast%3DBacchi%26aufirst%3DC.%2BJ.%26aulast%3DBarker%26aufirst%3DR.%2BH.%26aulast%3DRodriguez%26aufirst%3DA.%26aulast%3DHirth%26aufirst%3DB.%26aulast%3DRattendi%26aufirst%3DD.%26aulast%3DYarlett%26aufirst%3DN.%26aulast%3DHendrick%26aufirst%3DC.%2BL.%26aulast%3DSybertz%26aufirst%3DE.%26atitle%3DTrypanocidal%2520activity%2520of%25208-methyl-5%25E2%2580%25B2-%257B%255B%2528Z%2529-4-aminobut-2-enyl%255D-%2528methylamino%2529%257Dadenosine%2520%2528Genz-644131%2529%252C%2520an%2520adenosylmethionine%2520decarboxylase%2520inhibitor%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2009%26volume%3D53%26spage%3D3269%26epage%3D3272%26doi%3D10.1128%2FAAC.00076-09" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Bitonti, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Byers, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bush, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casara, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bacchi, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarkson, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mccann, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sjoerdsma, A.</span><span> </span><span class="NLM_article-title">Cure of Trypanosoma brucei brucei and Trypanosoma brucei rhodesiense infections in mice with an irreversible inhibitor of S-adenosylmethionine decarboxylase</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">1485</span><span class="NLM_x">–</span> <span class="NLM_lpage">1490</span><span class="refDoi"> DOI: 10.1128/AAC.34.8.1485</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1128%2FAAC.34.8.1485" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1990&pages=1485-1490&author=A.+J.+Bitontiauthor=T.+L.+Byersauthor=T.+L.+Bushauthor=P.+J.+Casaraauthor=C.+J.+Bacchiauthor=A.+B.+Clarksonauthor=P.+P.+Mccannauthor=A.+Sjoerdsma&title=Cure+of+Trypanosoma+brucei+brucei+and+Trypanosoma+brucei+rhodesiense+infections+in+mice+with+an+irreversible+inhibitor+of+S-adenosylmethionine+decarboxylase&doi=10.1128%2FAAC.34.8.1485"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1128%2FAAC.34.8.1485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.34.8.1485%26sid%3Dliteratum%253Aachs%26aulast%3DBitonti%26aufirst%3DA.%2BJ.%26aulast%3DByers%26aufirst%3DT.%2BL.%26aulast%3DBush%26aufirst%3DT.%2BL.%26aulast%3DCasara%26aufirst%3DP.%2BJ.%26aulast%3DBacchi%26aufirst%3DC.%2BJ.%26aulast%3DClarkson%26aufirst%3DA.%2BB.%26aulast%3DMccann%26aufirst%3DP.%2BP.%26aulast%3DSjoerdsma%26aufirst%3DA.%26atitle%3DCure%2520of%2520Trypanosoma%2520brucei%2520brucei%2520and%2520Trypanosoma%2520brucei%2520rhodesiense%2520infections%2520in%2520mice%2520with%2520an%2520irreversible%2520inhibitor%2520of%2520S-adenosylmethionine%2520decarboxylase%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1990%26volume%3D34%26spage%3D1485%26epage%3D1490%26doi%3D10.1128%2FAAC.34.8.1485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Willert, E. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fitzpatrick, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, M. A.</span><span> </span><span class="NLM_article-title">Allosteric regulation of an essential trypanosome polyamine biosynthetic enzyme by a catalytically dead homolog</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x">, </span> <span class="NLM_fpage">8275</span><span class="NLM_x">–</span> <span class="NLM_lpage">8280</span><span class="refDoi"> DOI: 10.1073/pnas.0701111104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1073%2Fpnas.0701111104" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=8275-8280&author=E.+K.+Willertauthor=R.+Fitzpatrickauthor=M.+A.+Phillips&title=Allosteric+regulation+of+an+essential+trypanosome+polyamine+biosynthetic+enzyme+by+a+catalytically+dead+homolog&doi=10.1073%2Fpnas.0701111104"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0701111104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0701111104%26sid%3Dliteratum%253Aachs%26aulast%3DWillert%26aufirst%3DE.%2BK.%26aulast%3DFitzpatrick%26aufirst%3DR.%26aulast%3DPhillips%26aufirst%3DM.%2BA.%26atitle%3DAllosteric%2520regulation%2520of%2520an%2520essential%2520trypanosome%2520polyamine%2520biosynthetic%2520enzyme%2520by%2520a%2520catalytically%2520dead%2520homolog%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D8275%26epage%3D8280%26doi%3D10.1073%2Fpnas.0701111104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Volkov, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kinch, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ariagno, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grishin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomchick, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, M. A.</span><span> </span><span class="NLM_article-title">Relief of autoinhibition by conformational switch explains enzyme activation by a catalytically dead paralog</span> <span class="citation_source-journal">eLife</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">e20198</span><span class="refDoi"> DOI: 10.7554/eLife.20198</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.7554%2FeLife.20198" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&pages=e20198&author=O.+A.+Volkovauthor=L.+Kinchauthor=C.+Ariagnoauthor=X.+Dengauthor=S.+Zhongauthor=N.+Grishinauthor=D.+R.+Tomchickauthor=Z.+Chenauthor=M.+A.+Phillips&title=Relief+of+autoinhibition+by+conformational+switch+explains+enzyme+activation+by+a+catalytically+dead+paralog&doi=10.7554%2FeLife.20198"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.7554%2FeLife.20198&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.20198%26sid%3Dliteratum%253Aachs%26aulast%3DVolkov%26aufirst%3DO.%2BA.%26aulast%3DKinch%26aufirst%3DL.%26aulast%3DAriagno%26aufirst%3DC.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DZhong%26aufirst%3DS.%26aulast%3DGrishin%26aufirst%3DN.%26aulast%3DTomchick%26aufirst%3DD.%2BR.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DPhillips%26aufirst%3DM.%2BA.%26atitle%3DRelief%2520of%2520autoinhibition%2520by%2520conformational%2520switch%2520explains%2520enzyme%2520activation%2520by%2520a%2520catalytically%2520dead%2520paralog%26jtitle%3DeLife%26date%3D2016%26volume%3D5%26spage%3De20198%26doi%3D10.7554%2FeLife.20198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Volkov, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosner, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brockway, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kramer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Booker, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ketcherside, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Longgood, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mccoy, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wring, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klinger, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Posner, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Brabander, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, M. A.</span><span> </span><span class="NLM_article-title">Identification of Trypanosoma brucei AdoMetDC inhibitors using a high-throughput mass spectrometry-based assay</span> <span class="citation_source-journal">ACS Infect. Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">512</span><span class="NLM_x">–</span> <span class="NLM_lpage">526</span><span class="refDoi"> DOI: 10.1021/acsinfecdis.7b00022</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.7b00022" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BC2sXltVSqtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2017&pages=512-526&author=O.+A.+Volkovauthor=C.+C.+Cosnerauthor=A.+J.+Brockwayauthor=M.+Kramerauthor=M.+Bookerauthor=S.+Zhongauthor=A.+Ketchersideauthor=S.+Weiauthor=J.+Longgoodauthor=M.+Mccoyauthor=T.+E.+Richardsonauthor=S.+A.+Wringauthor=M.+Peelauthor=J.+D.+Klingerauthor=B.+A.+Posnerauthor=J.+K.+De+Brabanderauthor=M.+A.+Phillips&title=Identification+of+Trypanosoma+brucei+AdoMetDC+inhibitors+using+a+high-throughput+mass+spectrometry-based+assay&doi=10.1021%2Facsinfecdis.7b00022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Trypanosoma brucei AdoMetDC Inhibitors Using a High-Throughput Mass Spectrometry-Based Assay</span></div><div class="casAuthors">Volkov, Oleg A.; Cosner, Casey C.; Brockway, Anthony J.; Kramer, Martin; Booker, Michael; Zhong, Shihua; Ketcherside, Ariel; Wei, Shuguang; Longgood, Jamie; McCoy, Melissa; Richardson, Thomas E.; Wring, Stephen A.; Peel, Michael; Klinger, Jeffrey D.; Posner, Bruce A.; De Brabander, Jef K.; Phillips, Margaret A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">512-526</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human African trypanosomiasis (HAT) is a fatal infectious disease caused by the eukaryotic pathogen Trypanosoma brucei (Tb).  Available treatments are difficult to administer and have significant safety issues.  S-Adenosylmethionine decarboxylase (AdoMetDC) is an essential enzyme in the parasite polyamine biosynthetic pathway.  Previous attempts to develop TbAdoMetDC inhibitors into anti-HAT therapies failed due to poor brain exposure.  Here, we describe a large screening campaign of two small-mol. libraries (∼400,000 compds.) employing a new high-throughput (∼7 s per sample) mass spectrometry-based assay for AdoMetDC activity.  As a result of primary screening, followed by hit confirmation and validation, we identified 13 new classes of reversible TbAdoMetDC inhibitors with low-micromolar potency (IC50) against both TbAdoMetDC and T. brucei parasite cells.  The majority of these compds. were >10-fold selective against the human enzyme.  Importantly, compds. from four classes demonstrated high propensity to cross the blood-brain barrier in a cell monolayer assay.  Biochem. anal. demonstrated that compds. from eight classes inhibited intracellular TbAdoMetDC in the parasite, although evidence for a secondary off-target component was also present.  The discovery of several new TbAdoMetDC inhibitor chemotypes provides new hits for lead optimization programs aimed to deliver a novel treatment for HAT.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1WsiJeQdpIrVg90H21EOLACvtfcHk0liqnsc25sfbAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXltVSqtbs%253D&md5=99fbff6e441a8e43959700dc66a3c360</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.7b00022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.7b00022%26sid%3Dliteratum%253Aachs%26aulast%3DVolkov%26aufirst%3DO.%2BA.%26aulast%3DCosner%26aufirst%3DC.%2BC.%26aulast%3DBrockway%26aufirst%3DA.%2BJ.%26aulast%3DKramer%26aufirst%3DM.%26aulast%3DBooker%26aufirst%3DM.%26aulast%3DZhong%26aufirst%3DS.%26aulast%3DKetcherside%26aufirst%3DA.%26aulast%3DWei%26aufirst%3DS.%26aulast%3DLonggood%26aufirst%3DJ.%26aulast%3DMccoy%26aufirst%3DM.%26aulast%3DRichardson%26aufirst%3DT.%2BE.%26aulast%3DWring%26aufirst%3DS.%2BA.%26aulast%3DPeel%26aufirst%3DM.%26aulast%3DKlinger%26aufirst%3DJ.%2BD.%26aulast%3DPosner%26aufirst%3DB.%2BA.%26aulast%3DDe%2BBrabander%26aufirst%3DJ.%2BK.%26aulast%3DPhillips%26aufirst%3DM.%2BA.%26atitle%3DIdentification%2520of%2520Trypanosoma%2520brucei%2520AdoMetDC%2520inhibitors%2520using%2520a%2520high-throughput%2520mass%2520spectrometry-based%2520assay%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2017%26volume%3D3%26spage%3D512%26epage%3D526%26doi%3D10.1021%2Facsinfecdis.7b00022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schnekenburger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwergel, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosconati, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gros, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tardugno, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labella, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Florean, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minden, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hashimoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirsch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Novellino, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arimondo, P. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miele, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferretti, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gulino, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diederich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span> </span><span class="NLM_article-title">Selective non-nucleoside inhibitors of human DNA methyltransferases active in cancer including in cancer stem cells</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">701</span><span class="NLM_x">–</span> <span class="NLM_lpage">713</span><span class="refDoi"> DOI: 10.1021/jm4012627</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4012627" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisFOisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=701-713&author=S.+Valenteauthor=Y.+Liuauthor=M.+Schnekenburgerauthor=C.+Zwergelauthor=S.+Cosconatiauthor=C.+Grosauthor=M.+Tardugnoauthor=D.+Labellaauthor=C.+Floreanauthor=S.+Mindenauthor=H.+Hashimotoauthor=Y.+Changauthor=X.+Zhangauthor=G.+Kirschauthor=E.+Novellinoauthor=P.+B.+Arimondoauthor=E.+Mieleauthor=E.+Ferrettiauthor=A.+Gulinoauthor=M.+Diederichauthor=X.+Chengauthor=A.+Mai&title=Selective+non-nucleoside+inhibitors+of+human+DNA+methyltransferases+active+in+cancer+including+in+cancer+stem+cells&doi=10.1021%2Fjm4012627"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Non-nucleoside Inhibitors of Human DNA Methyltransferases Active in Cancer Including in Cancer Stem Cells</span></div><div class="casAuthors">Valente, Sergio; Yiwei, Liu; Schnekenburger, Michael; Zwergel, Clemens; Cosconati, Sandro; Gros, Christina; Tardugno, Maria; Labella, Donatella; Florean, Cristina; Minden, Steven; Hashimoto, Hideharu; Chang, Yanqi; Zhang, Xing; Kirsch, Gilbert; Novellino, Ettore; Arimondo, Paola B.; Miele, Evelina; Ferretti, Elisabetta; Gulino, Alberto; Diederich, Marc; Cheng, Xiaodong; Mai, Antonello</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">701-713</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DNA methyltransferases (DNMTs) are important enzymes involved in epigenetic control of gene expression and represent valuable targets in cancer chemotherapy.  A no. of nucleoside DNMT inhibitors (DNMTi) have been studied in cancer, including in cancer stem cells, and two of them (azacytidine and decitabine) have been approved for treatment of myelodysplastic syndromes.  However, only a few non-nucleoside DNMTi have been identified so far, and even fewer have been validated in cancer.  Through a process of hit-to-lead optimization, we report here the discovery of compd. 5 as a potent non-nucleoside DNMTi that is also selective toward other AdoMet-dependent protein methyltransferases.  Compd. 5 was potent at single-digit micromolar concns. against a panel of cancer cells and was less toxic in peripheral blood mononuclear cells than two other compds. tested.  In mouse medulloblastoma stem cells, 5 inhibited cell growth, whereas related compd. 2 showed high cell differentiation.  To the best of our knowledge, 2 and 5 are the first non-nucleoside DNMTi tested in a cancer stem cell line.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxOcRR02xFkLVg90H21EOLACvtfcHk0liqnsc25sfbAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisFOisw%253D%253D&md5=0749e33ad2324aa591342d7c7f23a955</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm4012627&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4012627%26sid%3Dliteratum%253Aachs%26aulast%3DValente%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DSchnekenburger%26aufirst%3DM.%26aulast%3DZwergel%26aufirst%3DC.%26aulast%3DCosconati%26aufirst%3DS.%26aulast%3DGros%26aufirst%3DC.%26aulast%3DTardugno%26aufirst%3DM.%26aulast%3DLabella%26aufirst%3DD.%26aulast%3DFlorean%26aufirst%3DC.%26aulast%3DMinden%26aufirst%3DS.%26aulast%3DHashimoto%26aufirst%3DH.%26aulast%3DChang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DKirsch%26aufirst%3DG.%26aulast%3DNovellino%26aufirst%3DE.%26aulast%3DArimondo%26aufirst%3DP.%2BB.%26aulast%3DMiele%26aufirst%3DE.%26aulast%3DFerretti%26aufirst%3DE.%26aulast%3DGulino%26aufirst%3DA.%26aulast%3DDiederich%26aufirst%3DM.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DMai%26aufirst%3DA.%26atitle%3DSelective%2520non-nucleoside%2520inhibitors%2520of%2520human%2520DNA%2520methyltransferases%2520active%2520in%2520cancer%2520including%2520in%2520cancer%2520stem%2520cells%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D701%26epage%3D713%26doi%3D10.1021%2Fjm4012627" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Borman, R. A.; Coleman, R. A.; Clark, K. L.; Oxford, A. W.; Hynd, G.; Archer, J. A.; Aley, A.; Harris, N. V.</span> 5-HT2B Receptor Antagonists. Patent WO 2005/012263 A1,<span class="NLM_x"> </span><span class="NLM_year">2005</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Borman%2C+R.+A.%3B+Coleman%2C+R.+A.%3B+Clark%2C+K.+L.%3B+Oxford%2C+A.+W.%3B+Hynd%2C+G.%3B+Archer%2C+J.+A.%3B+Aley%2C+A.%3B+Harris%2C+N.+V.+5-HT2B+Receptor+Antagonists.+Patent+WO+2005%2F012263+A1%2C+2005."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBorman%26aufirst%3DR.%2BA.%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Sander, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kottke, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanrikulu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Proschak, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weizel, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seifert, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stark, H.</span><span> </span><span class="NLM_article-title">2,4-Diaminopyrimidines as histamine H4 receptor ligands–Scaffold optimization and pharmacological characterization</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">7186</span><span class="NLM_x">–</span> <span class="NLM_lpage">7196</span><span class="refDoi"> DOI: 10.1016/j.bmc.2009.08.059</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1016%2Fj.bmc.2009.08.059" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2009&pages=7186-7196&author=K.+Sanderauthor=T.+Kottkeauthor=Y.+Tanrikuluauthor=E.+Proschakauthor=L.+Weizelauthor=E.+H.+Schneiderauthor=R.+Seifertauthor=G.+Schneiderauthor=H.+Stark&title=2%2C4-Diaminopyrimidines+as+histamine+H4+receptor+ligands%E2%80%93Scaffold+optimization+and+pharmacological+characterization&doi=10.1016%2Fj.bmc.2009.08.059"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2009.08.059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2009.08.059%26sid%3Dliteratum%253Aachs%26aulast%3DSander%26aufirst%3DK.%26aulast%3DKottke%26aufirst%3DT.%26aulast%3DTanrikulu%26aufirst%3DY.%26aulast%3DProschak%26aufirst%3DE.%26aulast%3DWeizel%26aufirst%3DL.%26aulast%3DSchneider%26aufirst%3DE.%2BH.%26aulast%3DSeifert%26aufirst%3DR.%26aulast%3DSchneider%26aufirst%3DG.%26aulast%3DStark%26aufirst%3DH.%26atitle%3D2%252C4-Diaminopyrimidines%2520as%2520histamine%2520H4%2520receptor%2520ligands%25E2%2580%2593Scaffold%2520optimization%2520and%2520pharmacological%2520characterization%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2009%26volume%3D17%26spage%3D7186%26epage%3D7196%26doi%3D10.1016%2Fj.bmc.2009.08.059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Sekiya, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takashima, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueda, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamiya, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuasa, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujimura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ubasawa, M.</span><span> </span><span class="NLM_article-title">2-Amino-6-arylthio-9-[2-(phosphonomethoxy)ethyl]purine bis(2,2,2-trifluoroethyl) esters as novel HBV-specific antiviral reagents</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">3138</span><span class="NLM_x">–</span> <span class="NLM_lpage">3142</span><span class="refDoi"> DOI: 10.1021/jm020036x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm020036x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=3138-3142&author=K.+Sekiyaauthor=H.+Takashimaauthor=N.+Uedaauthor=N.+Kamiyaauthor=S.+Yuasaauthor=Y.+Fujimuraauthor=M.+Ubasawa&title=2-Amino-6-arylthio-9-%5B2-%28phosphonomethoxy%29ethyl%5Dpurine+bis%282%2C2%2C2-trifluoroethyl%29+esters+as+novel+HBV-specific+antiviral+reagents&doi=10.1021%2Fjm020036x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm020036x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020036x%26sid%3Dliteratum%253Aachs%26aulast%3DSekiya%26aufirst%3DK.%26aulast%3DTakashima%26aufirst%3DH.%26aulast%3DUeda%26aufirst%3DN.%26aulast%3DKamiya%26aufirst%3DN.%26aulast%3DYuasa%26aufirst%3DS.%26aulast%3DFujimura%26aufirst%3DY.%26aulast%3DUbasawa%26aufirst%3DM.%26atitle%3D2-Amino-6-arylthio-9-%255B2-%2528phosphonomethoxy%2529ethyl%255Dpurine%2520bis%25282%252C2%252C2-trifluoroethyl%2529%2520esters%2520as%2520novel%2520HBV-specific%2520antiviral%2520reagents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D3138%26epage%3D3142%26doi%3D10.1021%2Fjm020036x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Medina, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Becker, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blackledge, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duquenne, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heerding, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romeril, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scherzer, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bobko, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chadderton, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dumble, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardiner, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabindran, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sudakin, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brady, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campobasso, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Axten, J. M.</span><span> </span><span class="NLM_article-title">Structure-based design of potent and selective 3-phosphoinositide-dependent kinase-1 (PDK1) inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1871</span><span class="NLM_x">–</span> <span class="NLM_lpage">1895</span><span class="refDoi"> DOI: 10.1021/jm101527u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm101527u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisVWksrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1871-1895&author=J.+R.+Medinaauthor=C.+J.+Beckerauthor=C.+W.+Blackledgeauthor=C.+Duquenneauthor=Y.+Fengauthor=S.+W.+Grantauthor=D.+Heerdingauthor=W.+H.+Liauthor=W.+H.+Millerauthor=S.+P.+Romerilauthor=D.+Scherzerauthor=A.+Shuauthor=M.+A.+Bobkoauthor=A.+R.+Chaddertonauthor=M.+Dumbleauthor=C.+M.+Gardinerauthor=S.+Gilbertauthor=Q.+Liuauthor=S.+K.+Rabindranauthor=V.+Sudakinauthor=H.+Xiangauthor=P.+G.+Bradyauthor=N.+Campobassoauthor=P.+Wardauthor=J.+M.+Axten&title=Structure-based+design+of+potent+and+selective+3-phosphoinositide-dependent+kinase-1+%28PDK1%29+inhibitors&doi=10.1021%2Fjm101527u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Potent and Selective 3-Phosphoinositide-Dependent Kinase-1 (PDK1) Inhibitors</span></div><div class="casAuthors">Medina, Jesus R.; Becker, Christopher J.; Blackledge, Charles W.; Duquenne, Celine; Feng, Yanhong; Grant, Seth W.; Heerding, Dirk; Li, William H.; Miller, William H.; Romeril, Stuart P.; Scherzer, Daryl; Shu, Arthur; Bobko, Mark A.; Chadderton, Antony R.; Dumble, Melissa; Gardiner, Christine M.; Gilbert, Seth; Liu, Qi; Rabindran, Sridhar K.; Sudakin, Valery; Xiang, Hong; Brady, Pat G.; Campobasso, Nino; Ward, Paris; Axten, Jeffrey M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1871-1895</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Phosphoinositide-dependent protein kinase-1 (PDK1) is a master regulator of the AGC family of kinases and an integral component of the PI3K/AKT/mTOR pathway.  As this pathway is among the most commonly deregulated across all cancers, a selective inhibitor of PDK1 might have utility as an anticancer agent.  Herein, the lead optimization of compd. I (R1 = R2 = H) toward highly potent and selective PDK1 inhibitors via a structure-based design strategy is described.  A series of (amino)(aminopyrimidinyl)indazoles I (R1 = H, Me, Et; R2 = Me, i-Pr, PhNH, 1-piperidinyl, etc.) was synthesized, and the inhibiting activity of the products towards PDK1 was studied.  The most potent and selective inhibitors, e.g. I [R1 = Me; R2 = (3S,6R)-3-(cyclohexylaminocarbonyl)-6-methyl-1-piperidinyl], demonstrated submicromolar activity as measured by inhibition of phosphorylation of PDK1 substrates as well as antiproliferative activity against a subset of AML cell lines.  In addn., redn. of phosphorylation of PDK1 substrates was demonstrated in vivo in mice bearing OCl-AML2 xenografts.  These observations demonstrate the utility of these mols. as tools to further delineate the biol. of PDK1 and the potential pharmacol. uses of a PDK1 inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrd83HnbHh5j7Vg90H21EOLACvtfcHk0lg288-Np6DVOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisVWksrk%253D&md5=2911a5b7dbfa944a7c523da0be1026ac</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm101527u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm101527u%26sid%3Dliteratum%253Aachs%26aulast%3DMedina%26aufirst%3DJ.%2BR.%26aulast%3DBecker%26aufirst%3DC.%2BJ.%26aulast%3DBlackledge%26aufirst%3DC.%2BW.%26aulast%3DDuquenne%26aufirst%3DC.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DGrant%26aufirst%3DS.%2BW.%26aulast%3DHeerding%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DW.%2BH.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26aulast%3DRomeril%26aufirst%3DS.%2BP.%26aulast%3DScherzer%26aufirst%3DD.%26aulast%3DShu%26aufirst%3DA.%26aulast%3DBobko%26aufirst%3DM.%2BA.%26aulast%3DChadderton%26aufirst%3DA.%2BR.%26aulast%3DDumble%26aufirst%3DM.%26aulast%3DGardiner%26aufirst%3DC.%2BM.%26aulast%3DGilbert%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DRabindran%26aufirst%3DS.%2BK.%26aulast%3DSudakin%26aufirst%3DV.%26aulast%3DXiang%26aufirst%3DH.%26aulast%3DBrady%26aufirst%3DP.%2BG.%26aulast%3DCampobasso%26aufirst%3DN.%26aulast%3DWard%26aufirst%3DP.%26aulast%3DAxten%26aufirst%3DJ.%2BM.%26atitle%3DStructure-based%2520design%2520of%2520potent%2520and%2520selective%25203-phosphoinositide-dependent%2520kinase-1%2520%2528PDK1%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1871%26epage%3D1895%26doi%3D10.1021%2Fjm101527u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Yi, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Long, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez-Cestari, T. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henderson, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pavlovicz, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Werbovetz, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mckay, D. B.</span><span> </span><span class="NLM_article-title">Discovery of benzamide analogs as negative allosteric modulators of human neuronal nicotinic receptors: pharmacophore modeling and structure-activity relationship studies</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">4730</span><span class="NLM_x">–</span> <span class="NLM_lpage">4743</span><span class="refDoi"> DOI: 10.1016/j.bmc.2013.03.082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1016%2Fj.bmc.2013.03.082" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=4730-4743&author=B.+Yiauthor=S.+Longauthor=T.+F.+Gonzalez-Cestariauthor=B.+J.+Hendersonauthor=R.+E.+Pavloviczauthor=K.+Werbovetzauthor=C.+Liauthor=D.+B.+Mckay&title=Discovery+of+benzamide+analogs+as+negative+allosteric+modulators+of+human+neuronal+nicotinic+receptors%3A+pharmacophore+modeling+and+structure-activity+relationship+studies&doi=10.1016%2Fj.bmc.2013.03.082"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.03.082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.03.082%26sid%3Dliteratum%253Aachs%26aulast%3DYi%26aufirst%3DB.%26aulast%3DLong%26aufirst%3DS.%26aulast%3DGonzalez-Cestari%26aufirst%3DT.%2BF.%26aulast%3DHenderson%26aufirst%3DB.%2BJ.%26aulast%3DPavlovicz%26aufirst%3DR.%2BE.%26aulast%3DWerbovetz%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DMckay%26aufirst%3DD.%2BB.%26atitle%3DDiscovery%2520of%2520benzamide%2520analogs%2520as%2520negative%2520allosteric%2520modulators%2520of%2520human%2520neuronal%2520nicotinic%2520receptors%253A%2520pharmacophore%2520modeling%2520and%2520structure-activity%2520relationship%2520studies%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D4730%26epage%3D4743%26doi%3D10.1016%2Fj.bmc.2013.03.082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Wijtmans, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scholten, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roumen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Canals, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Custers, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glas, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vreeker, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Kanter, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Graaf, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smit, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Esch, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leurs, R.</span><span> </span><span class="NLM_article-title">Chemical subtleties in small-molecule modulation of peptide receptor function: the case of CXCR3 biaryl-type ligands</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">10572</span><span class="NLM_x">–</span> <span class="NLM_lpage">10583</span><span class="refDoi"> DOI: 10.1021/jm301240t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301240t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=10572-10583&author=M.+Wijtmansauthor=D.+J.+Scholtenauthor=L.+Roumenauthor=M.+Canalsauthor=H.+Custersauthor=M.+Glasauthor=M.+C.+Vreekerauthor=F.+J.+De+Kanterauthor=C.+De+Graafauthor=M.+J.+Smitauthor=I.+J.+De+Eschauthor=R.+Leurs&title=Chemical+subtleties+in+small-molecule+modulation+of+peptide+receptor+function%3A+the+case+of+CXCR3+biaryl-type+ligands&doi=10.1021%2Fjm301240t"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm301240t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301240t%26sid%3Dliteratum%253Aachs%26aulast%3DWijtmans%26aufirst%3DM.%26aulast%3DScholten%26aufirst%3DD.%2BJ.%26aulast%3DRoumen%26aufirst%3DL.%26aulast%3DCanals%26aufirst%3DM.%26aulast%3DCusters%26aufirst%3DH.%26aulast%3DGlas%26aufirst%3DM.%26aulast%3DVreeker%26aufirst%3DM.%2BC.%26aulast%3DDe%2BKanter%26aufirst%3DF.%2BJ.%26aulast%3DDe%2BGraaf%26aufirst%3DC.%26aulast%3DSmit%26aufirst%3DM.%2BJ.%26aulast%3DDe%2BEsch%26aufirst%3DI.%2BJ.%26aulast%3DLeurs%26aufirst%3DR.%26atitle%3DChemical%2520subtleties%2520in%2520small-molecule%2520modulation%2520of%2520peptide%2520receptor%2520function%253A%2520the%2520case%2520of%2520CXCR3%2520biaryl-type%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D10572%26epage%3D10583%26doi%3D10.1021%2Fjm301240t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Knapp, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gillis, E. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, M. D.</span><span> </span><span class="NLM_article-title">A general solution for unstable boronic acids: slow-release cross-coupling from air-stable MIDA boronates</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">131</span><span class="NLM_x">, </span> <span class="NLM_fpage">6961</span><span class="NLM_x">–</span> <span class="NLM_lpage">6963</span><span class="refDoi"> DOI: 10.1021/ja901416p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja901416p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlt1Cls7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=6961-6963&author=D.+M.+Knappauthor=E.+P.+Gillisauthor=M.+D.+Burke&title=A+general+solution+for+unstable+boronic+acids%3A+slow-release+cross-coupling+from+air-stable+MIDA+boronates&doi=10.1021%2Fja901416p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A General Solution for Unstable Boronic Acids: Slow-Release Cross-Coupling from Air-Stable MIDA Boronates</span></div><div class="casAuthors">Knapp, David M.; Gillis, Eric P.; Burke, Martin D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6961-6963</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Many boronic acids, including 2-heterocyclic, vinyl, and cyclopropyl derivs., are inherently unstable, which can limit their bench-top storage and/or efficient cross-coupling.  The authors herein report the 1st general soln. to this problem: in situ slow release of unstable boronic acids from the corresponding air-stable N-methyliminodiacetic acid (MIDA) boronates.  This remarkably general approach has transformed all three classes of these unstable boronic acids into shelf-stable and highly effective building blocks for cross-coupling with a wide range of aryl and heteroaryl chlorides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnIQpAbX2N_7Vg90H21EOLACvtfcHk0lj9ihAUfovsfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlt1Cls7s%253D&md5=0865bc71ca79319e2116c23932c7ed55</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fja901416p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja901416p%26sid%3Dliteratum%253Aachs%26aulast%3DKnapp%26aufirst%3DD.%2BM.%26aulast%3DGillis%26aufirst%3DE.%2BP.%26aulast%3DBurke%26aufirst%3DM.%2BD.%26atitle%3DA%2520general%2520solution%2520for%2520unstable%2520boronic%2520acids%253A%2520slow-release%2520cross-coupling%2520from%2520air-stable%2520MIDA%2520boronates%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D6961%26epage%3D6963%26doi%3D10.1021%2Fja901416p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Baell, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holloway, G. A.</span><span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2719</span><span class="NLM_x">–</span> <span class="NLM_lpage">2740</span><span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lj9ihAUfovsfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Mahar Doan, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humphreys, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webster, L. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wring, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shampine, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serabjit-Singh, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adkison, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polli, J. W.</span><span> </span><span class="NLM_article-title">Passive permeability and P-glycoprotein-mediated efflux differentiate central nervous system (CNS) and non-CNS marketed drugs</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">303</span><span class="NLM_x">, </span> <span class="NLM_fpage">1029</span><span class="NLM_x">–</span> <span class="NLM_lpage">1037</span><span class="refDoi"> DOI: 10.1124/jpet.102.039255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1124%2Fjpet.102.039255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=12438524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A280%3ADC%252BD38nptVKmsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=303&publication_year=2002&pages=1029-1037&author=K.+M.+Mahar+Doanauthor=J.+E.+Humphreysauthor=L.+O.+Websterauthor=S.+A.+Wringauthor=L.+J.+Shampineauthor=C.+J.+Serabjit-Singhauthor=K.+K.+Adkisonauthor=J.+W.+Polli&title=Passive+permeability+and+P-glycoprotein-mediated+efflux+differentiate+central+nervous+system+%28CNS%29+and+non-CNS+marketed+drugs&doi=10.1124%2Fjpet.102.039255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Passive permeability and P-glycoprotein-mediated efflux differentiate central nervous system (CNS) and non-CNS marketed drugs</span></div><div class="casAuthors">Mahar Doan Kelly M; Humphreys Joan E; Webster Lindsey O; Wring Stephen A; Shampine Larry J; Serabjit-Singh Cosette J; Adkison Kimberly K; Polli Joseph W</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of pharmacology and experimental therapeutics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">303</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1029-37</span>
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    </div><div class="casAbstract">Membrane permeability and P-glycoprotein (Pgp) can be limiting factors for blood-brain barrier penetration.  The objectives of this study were to determine whether there are differences in the in vitro permeability, Pgp substrate profiles, and physicochemical properties of drugs for central nervous system (CNS) and non-CNS indications, and whether these differences are useful criteria in selecting compounds for drug development.  Apparent permeability (P(app)) and Pgp substrate profiles for 93 CNS (n = 48) and non-CNS (n = 45) drugs were determined by monolayer efflux.  Calcein-AM inhibition assays were used to supplement the efflux results.  The CNS set (2 of 48, 4.2%) had a 7-fold lower incidence of passive permeability values <150 nm/s compared with the non-CNS set (13 of 45, 28.9%).  The majority of drugs (72.0%, 67 of 93) were not Pgp substrates; however, 49.5% (46 of 93) were positive in the calcein-AM assay when tested at 100 microM.  The CNS drug set (n = 7 of 48, 14.6%) had a 3-fold lower incidence of Pgp-mediated efflux than the non-CNS drug set (n = 19 of 45, 42.2%).  Analysis of 18 physicochemical properties revealed that the CNS drug set had fewer hydrogen bond donors, fewer positive charges, greater lipophilicity, lower polar surface area, and reduced flexibility compared with the non-CNS group (p < 0.05), properties that enhance membrane permeability.  This study on a large, diverse set of marketed compounds clearly demonstrates that permeability, Pgp-mediated efflux, and certain physicochemical properties are factors that differentiate CNS and non-CNS drugs.  For CNS delivery, a drug should ideally have an in vitro passive permeability >150 nm/s and not be a good (B --> A/A --> B ratio <2.5) Pgp substrate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT_HiV1Xqt18Vf1XRtfYM0dfW6udTcc2ebmnBydQRpfabntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD38nptVKmsw%253D%253D&md5=b841902aa8502de47c1e1603c80e09fb</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1124%2Fjpet.102.039255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.102.039255%26sid%3Dliteratum%253Aachs%26aulast%3DMahar%2BDoan%26aufirst%3DK.%2BM.%26aulast%3DHumphreys%26aufirst%3DJ.%2BE.%26aulast%3DWebster%26aufirst%3DL.%2BO.%26aulast%3DWring%26aufirst%3DS.%2BA.%26aulast%3DShampine%26aufirst%3DL.%2BJ.%26aulast%3DSerabjit-Singh%26aufirst%3DC.%2BJ.%26aulast%3DAdkison%26aufirst%3DK.%2BK.%26aulast%3DPolli%26aufirst%3DJ.%2BW.%26atitle%3DPassive%2520permeability%2520and%2520P-glycoprotein-mediated%2520efflux%2520differentiate%2520central%2520nervous%2520system%2520%2528CNS%2529%2520and%2520non-CNS%2520marketed%2520drugs%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2002%26volume%3D303%26spage%3D1029%26epage%3D1037%26doi%3D10.1124%2Fjpet.102.039255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Copeland, R. A.</span> <span class="citation_source-book">Evaluation of Enzyme Inhibitors in Drug Discovery: A Guide for Medicinal Chemists and Pharmacologists</span>; <span class="NLM_publisher-name">Wiley-Interscience</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>,<span class="NLM_x"> </span><span class="NLM_year">2005</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&author=R.+A.+Copeland&title=Evaluation+of+Enzyme+Inhibitors+in+Drug+Discovery%3A+A+Guide+for+Medicinal+Chemists+and+Pharmacologists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26btitle%3DEvaluation%2520of%2520Enzyme%2520Inhibitors%2520in%2520Drug%2520Discovery%253A%2520A%2520Guide%2520for%2520Medicinal%2520Chemists%2520and%2520Pharmacologists%26pub%3DWiley-Interscience%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skolnick, J.</span><span> </span><span class="NLM_article-title">TM-align: a protein structure alignment algorithm based on the TM-score</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">2302</span><span class="NLM_x">–</span> <span class="NLM_lpage">2309</span><span class="refDoi"> DOI: 10.1093/nar/gki524</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1093%2Fnar%2Fgki524" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=15849316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjsl2gsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=2302-2309&author=Y.+Zhangauthor=J.+Skolnick&title=TM-align%3A+a+protein+structure+alignment+algorithm+based+on+the+TM-score&doi=10.1093%2Fnar%2Fgki524"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">TM-align: a protein structure alignment algorithm based on the TM-score</span></div><div class="casAuthors">Zhang, Yang; Skolnick, Jeffrey</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2302-2309</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">We have developed TM-align, a new algorithm to identify the best structural alignment between protein pairs that combines the TM-score rotation matrix and Dynamic Programming (DP).  The algorithm is ∼4 times faster than CE and 20 times faster than DALI and SAL.  On av., the resulting structure alignments have higher accuracy and coverage than those provided by these most often-used methods.  TM-align is applied to an all-against-all structure comparison of 10 515 representative protein chains from the Protein Data Bank (PDB) with a sequence identity cutoff <95%: 1996 distinct folds are found when a TM-score threshold of 0.5 is used.  We also use TM-align to match the models predicted by TASSER for solved non-homologous proteins in PDB.  For both folded and misfolded models, TM-align can almost always find close structural analogs, with an av. root mean square deviation, RMSD, of 3 Å and 87% alignment coverage.  Nevertheless, there exists a significant correlation between the correctness of the predicted structure and the structural similarity of the model to the other proteins in the PDB.  This correlation could be used to assist in model selection in blind protein structure predictions.  The TM-align program is freely downloadable at http://bioinformatics.buffalo.edu/TM-align.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonPrM9Mg2FZ7Vg90H21EOLACvtfcHk0lgdrI4-vGabLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjsl2gsLY%253D&md5=15689409a6d9281184a7f943f5cc45d3</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgki524&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgki524%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DSkolnick%26aufirst%3DJ.%26atitle%3DTM-align%253A%2520a%2520protein%2520structure%2520alignment%2520algorithm%2520based%2520on%2520the%2520TM-score%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2005%26volume%3D33%26spage%3D2302%26epage%3D2309%26doi%3D10.1093%2Fnar%2Fgki524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Duan, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, V. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weaver, D. F.</span><span> </span><span class="NLM_article-title">Characterization of aromatic-thiol π-type hydrogen bonding and phenylalanine-cysteine side chain interactions throughab initiocalculations and protein database analyses</span> <span class="citation_source-journal">Mol. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">1689</span><span class="NLM_x">–</span> <span class="NLM_lpage">1699</span><span class="refDoi"> DOI: 10.1080/00268970110063917</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1080%2F00268970110063917" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2001&pages=1689-1699&author=G.+Duanauthor=V.+H.+Smithauthor=D.+F.+Weaver&title=Characterization+of+aromatic-thiol+%CF%80-type+hydrogen+bonding+and+phenylalanine-cysteine+side+chain+interactions+throughab+initiocalculations+and+protein+database+analyses&doi=10.1080%2F00268970110063917"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1080%2F00268970110063917&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00268970110063917%26sid%3Dliteratum%253Aachs%26aulast%3DDuan%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DV.%2BH.%26aulast%3DWeaver%26aufirst%3DD.%2BF.%26atitle%3DCharacterization%2520of%2520aromatic-thiol%2520%25CF%2580-type%2520hydrogen%2520bonding%2520and%2520phenylalanine-cysteine%2520side%2520chain%2520interactions%2520throughab%2520initiocalculations%2520and%2520protein%2520database%2520analyses%26jtitle%3DMol.%2520Phys.%26date%3D2001%26volume%3D99%26spage%3D1689%26epage%3D1699%26doi%3D10.1080%2F00268970110063917" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Meyer, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castellano, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diederich, F.</span><span> </span><span class="NLM_article-title">Interactions with aromatic rings in chemical and biological recognition</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">1210</span><span class="NLM_x">–</span> <span class="NLM_lpage">1250</span><span class="refDoi"> DOI: 10.1002/anie.200390319</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1002%2Fanie.200390319" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=12645054" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BD3sXivVSrtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2003&pages=1210-1250&author=E.+A.+Meyerauthor=R.+K.+Castellanoauthor=F.+Diederich&title=Interactions+with+aromatic+rings+in+chemical+and+biological+recognition&doi=10.1002%2Fanie.200390319"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Interactions with aromatic rings in chemical and biological recognition</span></div><div class="casAuthors">Meyer, Emmanuel A.; Castellano, Ronald K.; Diederich, Francois</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1210-1250</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Intermol. interactions involving arom. rings are key processes in both chem. and biol. recognition.  Their understanding is essential for rational drug design and lead optimization in medicinal chem.  Different approaches-biol. studies, mol. recognition studies with artificial receptors, crystallog. database mining, gas-phase studies, and theor. calcns. - are pursued to generate a profound understanding of the structural and energetic parameters of individual recognition modes involving arom. rings.  This review attempts to combine and summarize the knowledge gained from these investigations.  The review focuses mainly on examples with biol. relevance since one of its aims it to enhance the knowledge of mol. recognition forces that is essential for drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHUbAhGHcGtLVg90H21EOLACvtfcHk0lgdrI4-vGabLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXivVSrtbw%253D&md5=e91e6a8d115d56229e1660697a286b04</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fanie.200390319&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200390319%26sid%3Dliteratum%253Aachs%26aulast%3DMeyer%26aufirst%3DE.%2BA.%26aulast%3DCastellano%26aufirst%3DR.%2BK.%26aulast%3DDiederich%26aufirst%3DF.%26atitle%3DInteractions%2520with%2520aromatic%2520rings%2520in%2520chemical%2520and%2520biological%2520recognition%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2003%26volume%3D42%26spage%3D1210%26epage%3D1250%26doi%3D10.1002%2Fanie.200390319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Cavallo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metrangolo, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milani, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pilati, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Priimagi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Resnati, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terraneo, G.</span><span> </span><span class="NLM_article-title">The halogen bond</span> <span class="citation_source-journal">Chem. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">116</span><span class="NLM_x">, </span> <span class="NLM_fpage">2478</span><span class="NLM_x">–</span> <span class="NLM_lpage">2601</span><span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00484</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00484" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVyhs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2016&pages=2478-2601&author=G.+Cavalloauthor=P.+Metrangoloauthor=R.+Milaniauthor=T.+Pilatiauthor=A.+Priimagiauthor=G.+Resnatiauthor=G.+Terraneo&title=The+halogen+bond&doi=10.1021%2Facs.chemrev.5b00484"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">The Halogen Bond</span></div><div class="casAuthors">Cavallo, Gabriella; Metrangolo, Pierangelo; Milani, Roberto; Pilati, Tullio; Priimagi, Arri; Resnati, Giuseppe; Terraneo, Giancarlo</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2478-2601</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The halogen bond occurs when there is evidence of a net attractive interaction between an electrophilic region assocd. with a halogen atom in a mol. entity and a nucleophilic region in another, or the same, mol. entity.  In this fairly extensive review, after a brief history of the interaction, we will provide the reader with a snapshot of where the research on the halogen bond is now, and, perhaps, where it is going.  The specific advantages brought up by a design based on the use of the halogen bond will be demonstrated in quite different fields spanning from material sciences to biomol. recognition and drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreUcfocIywvLVg90H21EOLACvtfcHk0lgdrI4-vGabLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVyhs74%253D&md5=307f97f5056bdd5300ea0b0eac9fb558</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00484&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00484%26sid%3Dliteratum%253Aachs%26aulast%3DCavallo%26aufirst%3DG.%26aulast%3DMetrangolo%26aufirst%3DP.%26aulast%3DMilani%26aufirst%3DR.%26aulast%3DPilati%26aufirst%3DT.%26aulast%3DPriimagi%26aufirst%3DA.%26aulast%3DResnati%26aufirst%3DG.%26aulast%3DTerraneo%26aufirst%3DG.%26atitle%3DThe%2520halogen%2520bond%26jtitle%3DChem.%2520Rev.%26date%3D2016%26volume%3D116%26spage%3D2478%26epage%3D2601%26doi%3D10.1021%2Facs.chemrev.5b00484" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Mccloskey, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bale, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Secrist, J. A.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tiwari, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moss, T. H.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valiyaveettil, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brooks, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guida, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pegg, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ealick, S. E.</span><span> </span><span class="NLM_article-title">New insights into the design of inhibitors of human S-adenosylmethionine decarboxylase: studies of adenine C8 substitution in structural analogues of S-adenosylmethionine</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1388</span><span class="NLM_x">–</span> <span class="NLM_lpage">1407</span><span class="refDoi"> DOI: 10.1021/jm801126a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801126a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=1388-1407&author=D.+E.+Mccloskeyauthor=S.+Baleauthor=J.+A.+Secristauthor=A.+Tiwariauthor=T.+H.+Mossauthor=J.+Valiyaveettilauthor=W.+H.+Brooksauthor=W.+C.+Guidaauthor=A.+E.+Peggauthor=S.+E.+Ealick&title=New+insights+into+the+design+of+inhibitors+of+human+S-adenosylmethionine+decarboxylase%3A+studies+of+adenine+C8+substitution+in+structural+analogues+of+S-adenosylmethionine&doi=10.1021%2Fjm801126a"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm801126a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801126a%26sid%3Dliteratum%253Aachs%26aulast%3DMccloskey%26aufirst%3DD.%2BE.%26aulast%3DBale%26aufirst%3DS.%26aulast%3DSecrist%26aufirst%3DJ.%2BA.%26aulast%3DTiwari%26aufirst%3DA.%26aulast%3DMoss%26aufirst%3DT.%2BH.%26aulast%3DValiyaveettil%26aufirst%3DJ.%26aulast%3DBrooks%26aufirst%3DW.%2BH.%26aulast%3DGuida%26aufirst%3DW.%2BC.%26aulast%3DPegg%26aufirst%3DA.%2BE.%26aulast%3DEalick%26aufirst%3DS.%2BE.%26atitle%3DNew%2520insights%2520into%2520the%2520design%2520of%2520inhibitors%2520of%2520human%2520S-adenosylmethionine%2520decarboxylase%253A%2520studies%2520of%2520adenine%2520C8%2520substitution%2520in%2520structural%2520analogues%2520of%2520S-adenosylmethionine%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D1388%26epage%3D1407%26doi%3D10.1021%2Fjm801126a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Xiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Volkov, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tu, B. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, M. A.</span><span> </span><span class="NLM_article-title">Product feedback regulation implicated in translational control of the Trypanosoma brucei S-adenosylmethionine decarboxylase regulatory subunit Prozyme</span> <span class="citation_source-journal">Mol. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">88</span><span class="NLM_x">, </span> <span class="NLM_fpage">846</span><span class="NLM_x">–</span> <span class="NLM_lpage">861</span><span class="refDoi"> DOI: 10.1111/mmi.12226</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1111%2Fmmi.12226" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=23634831" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BC3sXot1artLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2013&pages=846-861&author=Y.+Xiaoauthor=S.+Nguyenauthor=S.+H.+Kimauthor=O.+A.+Volkovauthor=B.+P.+Tuauthor=M.+A.+Phillips&title=Product+feedback+regulation+implicated+in+translational+control+of+the+Trypanosoma+brucei+S-adenosylmethionine+decarboxylase+regulatory+subunit+Prozyme&doi=10.1111%2Fmmi.12226"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Product feedback regulation implicated in translational control of the Trypanosoma brucei S-adenosylmethionine decarboxylase regulatory subunit prozyme</span></div><div class="casAuthors">Xiao, Yanjing; Nguyen, Suong; Kim, Sok Ho; Volkov, Oleg A.; Tu, Benjamin P.; Phillips, Margaret A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Microbiology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">846-861</span>CODEN:
                <span class="NLM_cas:coden">MOMIEE</span>;
        ISSN:<span class="NLM_cas:issn">0950-382X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Summary : Human African sleeping sickness (HAT) is caused by the parasitic protozoan Trypanosoma brucei.  Polyamine biosynthesis is an important drug target in the treatment of HAT.  Previously we showed that trypanosomatid S-adenosylmethionine decarboxylase (AdoMetDC), a key enzyme for biosynthesis of the polyamine spermidine, is activated by heterodimer formation with an inactive paralog termed prozyme.  Furthermore, prozyme protein levels were regulated in response to reduced AdoMetDC activity.  Herein we show that T. brucei encodes three prozyme transcripts.  The 3'UTRs of these transcripts were mapped and chloramphenicol acetyltransferase (CAT) reporter constructs were used to identify a 1.2 kb region that contained a 3'UTR prozyme regulatory element sufficient to upregulate CAT protein levels (but not RNA) upon AdoMetDC inhibition, supporting the hypothesis that prozyme expression is regulated translationally.  To gain insight into trans-acting factors, genetic rescue of AdoMetDC RNAi knock-down lines with human AdoMetDC was performed leading to rescue of the cell growth block, and restoration of prozyme protein to wild-type levels.  Metabolite anal. showed that prozyme protein levels were inversely proportional to intracellular levels of decarboxylated AdoMet (dcAdoMet).  These data suggest that prozyme translation may be regulated by dcAdoMet, a metabolite not previously identified to play a regulatory role.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyHtqmD3SgBLVg90H21EOLACvtfcHk0ljvmHWh6H7ypg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXot1artLg%253D&md5=b88b6fc64d32cf78ccf509d0bd4a46a1</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1111%2Fmmi.12226&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fmmi.12226%26sid%3Dliteratum%253Aachs%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DNguyen%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DS.%2BH.%26aulast%3DVolkov%26aufirst%3DO.%2BA.%26aulast%3DTu%26aufirst%3DB.%2BP.%26aulast%3DPhillips%26aufirst%3DM.%2BA.%26atitle%3DProduct%2520feedback%2520regulation%2520implicated%2520in%2520translational%2520control%2520of%2520the%2520Trypanosoma%2520brucei%2520S-adenosylmethionine%2520decarboxylase%2520regulatory%2520subunit%2520Prozyme%26jtitle%3DMol.%2520Microbiol.%26date%3D2013%26volume%3D88%26spage%3D846%26epage%3D861%26doi%3D10.1111%2Fmmi.12226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Vincent, I. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Creek, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamleh, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woods, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burchmore, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barrett, M. P.</span><span> </span><span class="NLM_article-title">A molecular mechanism for eflornithine resistance in African trypanosomes</span> <span class="citation_source-journal">PLoS Pathog.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1001204</span><span class="refDoi"> DOI: 10.1371/journal.ppat.1001204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1371%2Fjournal.ppat.1001204" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=e1001204&author=I.+M.+Vincentauthor=D.+Creekauthor=D.+G.+Watsonauthor=M.+A.+Kamlehauthor=D.+J.+Woodsauthor=P.+E.+Wongauthor=R.+J.+Burchmoreauthor=M.+P.+Barrett&title=A+molecular+mechanism+for+eflornithine+resistance+in+African+trypanosomes&doi=10.1371%2Fjournal.ppat.1001204"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1001204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1001204%26sid%3Dliteratum%253Aachs%26aulast%3DVincent%26aufirst%3DI.%2BM.%26aulast%3DCreek%26aufirst%3DD.%26aulast%3DWatson%26aufirst%3DD.%2BG.%26aulast%3DKamleh%26aufirst%3DM.%2BA.%26aulast%3DWoods%26aufirst%3DD.%2BJ.%26aulast%3DWong%26aufirst%3DP.%2BE.%26aulast%3DBurchmore%26aufirst%3DR.%2BJ.%26aulast%3DBarrett%26aufirst%3DM.%2BP.%26atitle%3DA%2520molecular%2520mechanism%2520for%2520eflornithine%2520resistance%2520in%2520African%2520trypanosomes%26jtitle%3DPLoS%2520Pathog.%26date%3D2010%26volume%3D6%26spage%3De1001204%26doi%3D10.1371%2Fjournal.ppat.1001204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Bitonti, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dumont, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mccann, P. P.</span><span> </span><span class="NLM_article-title">Characterization of Trypanosoma brucei brucei S-adenosyl-L-methionine decarboxylase and its inhibition by Berenil, pentamidine and methylglyoxal bis(guanylhydrazone)</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1986</span><span class="NLM_x">, </span> <span class="NLM_volume">237</span><span class="NLM_x">, </span> <span class="NLM_fpage">685</span><span class="NLM_x">–</span> <span class="NLM_lpage">689</span><span class="refDoi"> DOI: 10.1042/bj2370685</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1042%2Fbj2370685" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=237&publication_year=1986&pages=685-689&author=A.+J.+Bitontiauthor=J.+A.+Dumontauthor=P.+P.+Mccann&title=Characterization+of+Trypanosoma+brucei+brucei+S-adenosyl-L-methionine+decarboxylase+and+its+inhibition+by+Berenil%2C+pentamidine+and+methylglyoxal+bis%28guanylhydrazone%29&doi=10.1042%2Fbj2370685"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1042%2Fbj2370685&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbj2370685%26sid%3Dliteratum%253Aachs%26aulast%3DBitonti%26aufirst%3DA.%2BJ.%26aulast%3DDumont%26aufirst%3DJ.%2BA.%26aulast%3DMccann%26aufirst%3DP.%2BP.%26atitle%3DCharacterization%2520of%2520Trypanosoma%2520brucei%2520brucei%2520S-adenosyl-L-methionine%2520decarboxylase%2520and%2520its%2520inhibition%2520by%2520Berenil%252C%2520pentamidine%2520and%2520methylglyoxal%2520bis%2528guanylhydrazone%2529%26jtitle%3DBiochem.%2520J.%26date%3D1986%26volume%3D237%26spage%3D685%26epage%3D689%26doi%3D10.1042%2Fbj2370685" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Brun, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buhler, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandmeier, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaminsky, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bacchi, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rattendi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lane, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Croft, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snowdon, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yardley, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caravatti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frei, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanek, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mett, H.</span><span> </span><span class="NLM_article-title">In vitro trypanocidal activities of new S-adenosylmethionine decarboxylase inhibitors</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">1442</span><span class="NLM_x">–</span> <span class="NLM_lpage">1447</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=1996&pages=1442-1447&author=R.+Brunauthor=Y.+Buhlerauthor=U.+Sandmeierauthor=R.+Kaminskyauthor=C.+J.+Bacchiauthor=D.+Rattendiauthor=S.+Laneauthor=S.+L.+Croftauthor=D.+Snowdonauthor=V.+Yardleyauthor=G.+Caravattiauthor=J.+Freiauthor=J.+Stanekauthor=H.+Mett&title=In+vitro+trypanocidal+activities+of+new+S-adenosylmethionine+decarboxylase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBrun%26aufirst%3DR.%26aulast%3DBuhler%26aufirst%3DY.%26aulast%3DSandmeier%26aufirst%3DU.%26aulast%3DKaminsky%26aufirst%3DR.%26aulast%3DBacchi%26aufirst%3DC.%2BJ.%26aulast%3DRattendi%26aufirst%3DD.%26aulast%3DLane%26aufirst%3DS.%26aulast%3DCroft%26aufirst%3DS.%2BL.%26aulast%3DSnowdon%26aufirst%3DD.%26aulast%3DYardley%26aufirst%3DV.%26aulast%3DCaravatti%26aufirst%3DG.%26aulast%3DFrei%26aufirst%3DJ.%26aulast%3DStanek%26aufirst%3DJ.%26aulast%3DMett%26aufirst%3DH.%26atitle%3DIn%2520vitro%2520trypanocidal%2520activities%2520of%2520new%2520S-adenosylmethionine%2520decarboxylase%2520inhibitors%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1996%26volume%3D40%26spage%3D1442%26epage%3D1447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Tolbert, W. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cottet, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ekstrom, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pegg, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ealick, S. E.</span><span> </span><span class="NLM_article-title">Mechanism of human S-adenosylmethionine decarboxylase proenzyme processing as revealed by the structure of the S68A mutant</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">2386</span><span class="NLM_x">–</span> <span class="NLM_lpage">2395</span><span class="refDoi"> DOI: 10.1021/bi0268854</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi0268854" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2003&pages=2386-2395&author=W.+D.+Tolbertauthor=Y.+Zhangauthor=S.+E.+Cottetauthor=E.+M.+Bennettauthor=J.+L.+Ekstromauthor=A.+E.+Peggauthor=S.+E.+Ealick&title=Mechanism+of+human+S-adenosylmethionine+decarboxylase+proenzyme+processing+as+revealed+by+the+structure+of+the+S68A+mutant&doi=10.1021%2Fbi0268854"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fbi0268854&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi0268854%26sid%3Dliteratum%253Aachs%26aulast%3DTolbert%26aufirst%3DW.%2BD.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DCottet%26aufirst%3DS.%2BE.%26aulast%3DBennett%26aufirst%3DE.%2BM.%26aulast%3DEkstrom%26aufirst%3DJ.%2BL.%26aulast%3DPegg%26aufirst%3DA.%2BE.%26aulast%3DEalick%26aufirst%3DS.%2BE.%26atitle%3DMechanism%2520of%2520human%2520S-adenosylmethionine%2520decarboxylase%2520proenzyme%2520processing%2520as%2520revealed%2520by%2520the%2520structure%2520of%2520the%2520S68A%2520mutant%26jtitle%3DBiochemistry%26date%3D2003%26volume%3D42%26spage%3D2386%26epage%3D2395%26doi%3D10.1021%2Fbi0268854" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Bale, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brooks, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanes, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahesan, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guida, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ealick, S. E.</span><span> </span><span class="NLM_article-title">Role of the sulfonium center in determining the ligand specificity of human S-adenosylmethionine decarboxylase</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">6423</span><span class="NLM_x">–</span> <span class="NLM_lpage">6430</span><span class="refDoi"> DOI: 10.1021/bi900590m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi900590m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=6423-6430&author=S.+Baleauthor=W.+Brooksauthor=J.+W.+Hanesauthor=A.+M.+Mahesanauthor=W.+C.+Guidaauthor=S.+E.+Ealick&title=Role+of+the+sulfonium+center+in+determining+the+ligand+specificity+of+human+S-adenosylmethionine+decarboxylase&doi=10.1021%2Fbi900590m"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fbi900590m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi900590m%26sid%3Dliteratum%253Aachs%26aulast%3DBale%26aufirst%3DS.%26aulast%3DBrooks%26aufirst%3DW.%26aulast%3DHanes%26aufirst%3DJ.%2BW.%26aulast%3DMahesan%26aufirst%3DA.%2BM.%26aulast%3DGuida%26aufirst%3DW.%2BC.%26aulast%3DEalick%26aufirst%3DS.%2BE.%26atitle%3DRole%2520of%2520the%2520sulfonium%2520center%2520in%2520determining%2520the%2520ligand%2520specificity%2520of%2520human%2520S-adenosylmethionine%2520decarboxylase%26jtitle%3DBiochemistry%26date%3D2009%26volume%3D48%26spage%3D6423%26epage%3D6430%26doi%3D10.1021%2Fbi900590m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Bale, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopez, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Makhatadze, G. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pegg, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ealick, S. E.</span><span> </span><span class="NLM_article-title">Structural basis for putrescine activation of human S-adenosylmethionine decarboxylase</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">13404</span><span class="NLM_x">–</span> <span class="NLM_lpage">13417</span><span class="refDoi"> DOI: 10.1021/bi801732m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi801732m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=13404-13417&author=S.+Baleauthor=M.+M.+Lopezauthor=G.+I.+Makhatadzeauthor=Q.+Fangauthor=A.+E.+Peggauthor=S.+E.+Ealick&title=Structural+basis+for+putrescine+activation+of+human+S-adenosylmethionine+decarboxylase&doi=10.1021%2Fbi801732m"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fbi801732m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi801732m%26sid%3Dliteratum%253Aachs%26aulast%3DBale%26aufirst%3DS.%26aulast%3DLopez%26aufirst%3DM.%2BM.%26aulast%3DMakhatadze%26aufirst%3DG.%2BI.%26aulast%3DFang%26aufirst%3DQ.%26aulast%3DPegg%26aufirst%3DA.%2BE.%26aulast%3DEalick%26aufirst%3DS.%2BE.%26atitle%3DStructural%2520basis%2520for%2520putrescine%2520activation%2520of%2520human%2520S-adenosylmethionine%2520decarboxylase%26jtitle%3DBiochemistry%26date%3D2008%26volume%3D47%26spage%3D13404%26epage%3D13417%26doi%3D10.1021%2Fbi801732m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Shantz, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanley, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Secrist, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pegg, A. E.</span><span> </span><span class="NLM_article-title">Purification of human S-adenosylmethionine decarboxylase expressed in Escherichia coli and use of this protein to investigate the mechanism of inhibition by the irreversible inhibitors, 5′-deoxy-5′-[(3-hydrazinopropyl)methylamino]adenosine and 5′-{[(Z)-4-amino-2-butenyl]methylamino}-5′-deoxyadenosine</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">6848</span><span class="NLM_x">–</span> <span class="NLM_lpage">6855</span><span class="refDoi"> DOI: 10.1021/bi00144a027</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00144a027" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1992&pages=6848-6855&author=L.+M.+Shantzauthor=B.+A.+Stanleyauthor=J.+A.+Secristauthor=A.+E.+Pegg&title=Purification+of+human+S-adenosylmethionine+decarboxylase+expressed+in+Escherichia+coli+and+use+of+this+protein+to+investigate+the+mechanism+of+inhibition+by+the+irreversible+inhibitors%2C+5%E2%80%B2-deoxy-5%E2%80%B2-%5B%283-hydrazinopropyl%29methylamino%5Dadenosine+and+5%E2%80%B2-%7B%5B%28Z%29-4-amino-2-butenyl%5Dmethylamino%7D-5%E2%80%B2-deoxyadenosine&doi=10.1021%2Fbi00144a027"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fbi00144a027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00144a027%26sid%3Dliteratum%253Aachs%26aulast%3DShantz%26aufirst%3DL.%2BM.%26aulast%3DStanley%26aufirst%3DB.%2BA.%26aulast%3DSecrist%26aufirst%3DJ.%2BA.%26aulast%3DPegg%26aufirst%3DA.%2BE.%26atitle%3DPurification%2520of%2520human%2520S-adenosylmethionine%2520decarboxylase%2520expressed%2520in%2520Escherichia%2520coli%2520and%2520use%2520of%2520this%2520protein%2520to%2520investigate%2520the%2520mechanism%2520of%2520inhibition%2520by%2520the%2520irreversible%2520inhibitors%252C%25205%25E2%2580%25B2-deoxy-5%25E2%2580%25B2-%255B%25283-hydrazinopropyl%2529methylamino%255Dadenosine%2520and%25205%25E2%2580%25B2-%257B%255B%2528Z%2529-4-amino-2-butenyl%255Dmethylamino%257D-5%25E2%2580%25B2-deoxyadenosine%26jtitle%3DBiochemistry%26date%3D1992%26volume%3D31%26spage%3D6848%26epage%3D6855%26doi%3D10.1021%2Fbi00144a027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Beswick, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willert, E. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, M. A.</span><span> </span><span class="NLM_article-title">Mechanisms of allosteric regulation of Trypanosoma cruzi S-adenosylmethionine decarboxylase</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">7797</span><span class="NLM_x">–</span> <span class="NLM_lpage">7807</span><span class="refDoi"> DOI: 10.1021/bi0603975</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi0603975" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BD28Xlt1ehsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2006&pages=7797-7807&author=T.+C.+Beswickauthor=E.+K.+Willertauthor=M.+A.+Phillips&title=Mechanisms+of+allosteric+regulation+of+Trypanosoma+cruzi+S-adenosylmethionine+decarboxylase&doi=10.1021%2Fbi0603975"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of Allosteric Regulation of Trypanosoma cruzi S-Adenosylmethionine Decarboxylase</span></div><div class="casAuthors">Beswick, Tracy Clyne; Willert, Erin K.; Phillips, Margaret A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">7797-7807</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">S-Adenosylmethionine decarboxylase (AdoMetDC) is a pyruvoyl-dependent enzyme that catalyzes an essential step in polyamine biosynthesis.  The polyamines are required for cell growth, and the biosynthetic enzymes are targets for antiproliferative drugs.  The function of AdoMetDC is regulated by the polyamine-precursor putrescine in a species-specific manner.  AdoMetDC from the protozoal parasite Trypanosoma cruzi requires putrescine for maximal enzyme activity, but not for processing to generate the pyruvoyl cofactor.  The putrescine-binding site is distant from the active site, suggesting a mechanism of allosteric regulation.  To probe the structural basis by which putrescine stimulates T. cruzi AdoMetDC, we generated mutations in both the putrescine-binding site and the enzyme active site.  The catalytic efficiency of the mutant enzymes, and the binding of the diamidine inhibitors, CGP 48664A and CGP 40215, were analyzed.  Putrescine stimulates the kcat/Km for wild-type T. cruzi AdoMetDC by 27-fold, and it stimulates the binding of both inhibitors (IC50s decrease 10-20-fold with putrescine).  Unexpectedly, CGP 48664A activated the T. cruzi enzyme at low concns. (0.1-10 μM), while at higher concns. (>100 μM), or in the presence of putrescine, inhibition was obsd.  Anal. of the mutant data suggests that this inhibitor binds both the putrescine-binding site and the active site, providing evidence that the putrescine-binding site of the T. cruzi enzyme has broad ligand specificity.  Mutagenesis of the active site identified residues that are important for putrescine stimulation of activity (F7 and T245), while none of the active site mutations altered the apparent putrescine-binding const.  Mutations of residues in the putrescine-binding site that resulted in reduced (S111R) and enhanced (F285H) catalytic efficiency were both identified.  These data provide evidence for coupling between residues in the putrescine-binding site and the active site, consistent with a mechanism of allosteric regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKTqMLZ3tiIbVg90H21EOLACvtfcHk0lgEAzHYobaUTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xlt1ehsrg%253D&md5=76be996bb2845d90383ce22e10a82bec</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fbi0603975&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi0603975%26sid%3Dliteratum%253Aachs%26aulast%3DBeswick%26aufirst%3DT.%2BC.%26aulast%3DWillert%26aufirst%3DE.%2BK.%26aulast%3DPhillips%26aufirst%3DM.%2BA.%26atitle%3DMechanisms%2520of%2520allosteric%2520regulation%2520of%2520Trypanosoma%2520cruzi%2520S-adenosylmethionine%2520decarboxylase%26jtitle%3DBiochemistry%26date%3D2006%26volume%3D45%26spage%3D7797%26epage%3D7807%26doi%3D10.1021%2Fbi0603975" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Minor, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cymborowski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Otwinowski, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chruszcz, M.</span><span> </span><span class="NLM_article-title">HKL-3000: the integration of data reduction and structure solution - from diffraction images to an initial model in minutes</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">859</span><span class="NLM_x">–</span> <span class="NLM_lpage">866</span><span class="refDoi"> DOI: 10.1107/S0907444906019949</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1107%2FS0907444906019949" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=16855301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BD28XntVWksL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=859-866&author=W.+Minorauthor=M.+Cymborowskiauthor=Z.+Otwinowskiauthor=M.+Chruszcz&title=HKL-3000%3A+the+integration+of+data+reduction+and+structure+solution+-+from+diffraction+images+to+an+initial+model+in+minutes&doi=10.1107%2FS0907444906019949"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">HKL-3000: the integration of data reduction and structure solution - from diffraction images to an initial model in minutes</span></div><div class="casAuthors">Minor, Wladek; Cymborowski, Marcin; Otwinowski, Zbyszek; Chruszcz, Maksymilian</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">859-866</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A new approach that integrates data collection, data redn., phasing and model building significantly accelerates the process of structure detn. and on av. minimizes the no. of data sets and synchrotron time required for structure soln.  Initial testing of the HKL-3000 system (the beta version was named HKL-2000_ph) with more than 140 novel structure detns. has proven its high value for MAD/SAD expts.  The heuristics for choosing the best computational strategy at different data resoln. limits of phasing signal and crystal diffraction are being optimized.  The typical end result is an interpretable electron-d. map with a partially built structure and, in some cases, an almost complete refined model.  The current development is oriented towards very fast structure soln. in order to provide feedback during the diffraction expt.  Work is also proceeding towards improving the quality of phasing calcn. and model building.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0c0DYtjFP7rVg90H21EOLACvtfcHk0lhqtFrrgMREMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XntVWksL8%253D&md5=2cd2fe288f954837abb4acae4c008a2f</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS0907444906019949&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906019949%26sid%3Dliteratum%253Aachs%26aulast%3DMinor%26aufirst%3DW.%26aulast%3DCymborowski%26aufirst%3DM.%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DChruszcz%26aufirst%3DM.%26atitle%3DHKL-3000%253A%2520the%2520integration%2520of%2520data%2520reduction%2520and%2520structure%2520solution%2520-%2520from%2520diffraction%2520images%2520to%2520an%2520initial%2520model%2520in%2520minutes%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D859%26epage%3D866%26doi%3D10.1107%2FS0907444906019949" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Mccoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storoni, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title">Phaser crystallographic software</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">–</span> <span class="NLM_lpage">674</span><span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+Mccoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lhqtFrrgMREMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMccoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunkoczi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mccoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeffner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span><span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+Mccoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lhqtFrrgMREMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMccoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span> </span><span class="NLM_article-title">Electronic Ligand Builder and Optimization Workbench (eLBOW): a tool for ligand coordinate and restraint generation</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">1074</span><span class="NLM_x">–</span> <span class="NLM_lpage">1080</span><span class="refDoi"> DOI: 10.1107/S0907444909029436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1107%2FS0907444909029436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=19770504" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2009&pages=1074-1080&author=N.+W.+Moriartyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adams&title=Electronic+Ligand+Builder+and+Optimization+Workbench+%28eLBOW%29%3A+a+tool+for+ligand+coordinate+and+restraint+generation&doi=10.1107%2FS0907444909029436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">electronic Ligand Builder and Optimization Workbench (eLBOW): a tool for ligand coordinate and restraint generation</span></div><div class="casAuthors">Moriarty, Nigel W.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1074-1080</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The electronic Ligand Builder and Optimization Workbench (eLBOW) is a program module of the PHENIX suite of computational crystallog. software.  It is designed to be a flexible procedure that uses simple and fast quantum-chem. techniques to provide chem. accurate information for novel and known ligands alike.  A variety of input formats and options allow the attainment of a no. of diverse goals including geometry optimization and generation of restraints.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojnN4AVY_DwrVg90H21EOLACvtfcHk0lju57DlbNEFoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnN&md5=3f9a9c6c31c9a4b014a9f151fb6aa8d9</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1107%2FS0907444909029436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909029436%26sid%3Dliteratum%253Aachs%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DElectronic%2520Ligand%2520Builder%2520and%2520Optimization%2520Workbench%2520%2528eLBOW%2529%253A%2520a%2520tool%2520for%2520ligand%2520coordinate%2520and%2520restraint%2520generation%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2009%26volume%3D65%26spage%3D1074%26epage%3D1080%26doi%3D10.1107%2FS0907444909029436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and development of Coot</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">486</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span><span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lju57DlbNEFoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 7 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="NLM_string-name hlFld-ContribAuthor">Matthew D. Lloyd</span>. </span><span class="cited-content_cbyCitation_article-title">High-Throughput Screening for the Discovery of Enzyme Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (19)
                                     , 10742-10772. <a href="https://doi.org/10.1021/acs.jmedchem.0c00523" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00523</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00523%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHigh-Throughput%252BScreening%252Bfor%252Bthe%252BDiscovery%252Bof%252BEnzyme%252BInhibitors%26aulast%3DLloyd%26aufirst%3DMatthew%2BD.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D30032020%26date%3D10062020%26date%3D20052020%26volume%3D63%26issue%3D19%26spage%3D10742%26epage%3D10772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Maude Giroud, Uwe Dietzel, Lilli Anselm, David Banner, Andreas Kuglstatter, Jörg Benz, Jean-Baptiste Blanc, Delphine Gaufreteau, Haixia Liu, Xianfeng Lin, August Stich, Bernd Kuhn, Franz Schuler, Marcel Kaiser, Reto Brun, Tanja Schirmeister, Caroline Kisker, François Diederich, <span class="NLM_string-name hlFld-ContribAuthor">Wolfgang Haap</span>. </span><span class="cited-content_cbyCitation_article-title">Repurposing a Library of Human Cathepsin L Ligands: Identification of Macrocyclic Lactams as Potent Rhodesain and Trypanosoma brucei Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (8)
                                     , 3350-3369. <a href="https://doi.org/10.1021/acs.jmedchem.7b01869" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01869</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b01869%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRepurposing%252Ba%252BLibrary%252Bof%252BHuman%252BCathepsin%252BL%252BLigands%25253A%252BIdentification%252Bof%252BMacrocyclic%252BLactams%252Bas%252BPotent%252BRhodesain%252Band%252BTrypanosoma%252Bbrucei%252BInhibitors%26aulast%3DGiroud%26aufirst%3DMaude%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D28122017%26date%3D13042018%26date%3D29032018%26volume%3D61%26issue%3D8%26spage%3D3350%26epage%3D3369" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Maude Giroud, Bernd Kuhn, Sarah Saint-Auret, Christoph Kuratli, Rainer E. Martin, Franz Schuler, François Diederich, Marcel Kaiser, Reto Brun, Tanja Schirmeister, <span class="NLM_string-name hlFld-ContribAuthor">Wolfgang Haap</span>. </span><span class="cited-content_cbyCitation_article-title">2H-1,2,3-Triazole-Based Dipeptidyl Nitriles: Potent, Selective, and Trypanocidal Rhodesain Inhibitors by Structure-Based Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (8)
                                     , 3370-3388. <a href="https://doi.org/10.1021/acs.jmedchem.7b01870" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b01870</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b01870&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b01870%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3D2H-1%25252C2%25252C3-Triazole-Based%252BDipeptidyl%252BNitriles%25253A%252BPotent%25252C%252BSelective%25252C%252Band%252BTrypanocidal%252BRhodesain%252BInhibitors%252Bby%252BStructure-Based%252BDesign%26aulast%3DGiroud%26aufirst%3DMaude%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D28122017%26date%3D06042018%26date%3D29032018%26volume%3D61%26issue%3D8%26spage%3D3370%26epage%3D3388" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">William J.  Robinson</span>, <span class="hlFld-ContribAuthor ">Annie E.  Taylor</span>, <span class="hlFld-ContribAuthor ">Solange  Lauga-Cami</span>, <span class="hlFld-ContribAuthor ">George W.  Weaver</span>, <span class="hlFld-ContribAuthor ">Randolph R.J.  Arroo</span>, <span class="hlFld-ContribAuthor ">Marcel  Kaiser</span>, <span class="hlFld-ContribAuthor ">Sheraz  Gul</span>, <span class="hlFld-ContribAuthor ">Maria  Kuzikov</span>, <span class="hlFld-ContribAuthor ">Bernhard  Ellinger</span>, <span class="hlFld-ContribAuthor ">Kuldip  Singh</span>, <span class="hlFld-ContribAuthor ">Tanja  Schirmeister</span>, <span class="hlFld-ContribAuthor ">Adolfo  Botana</span>, <span class="hlFld-ContribAuthor ">Chatchakorn  Eurtivong</span>, <span class="hlFld-ContribAuthor ">Avninder S.  Bhambra</span>. </span><span class="cited-content_cbyCitation_article-title">The discovery of novel antitrypanosomal 4-phenyl-6-(pyridin-3-yl)pyrimidines. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>209 </em>, 112871. <a href="https://doi.org/10.1016/j.ejmech.2020.112871" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112871</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112871&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112871%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DThe%252Bdiscovery%252Bof%252Bnovel%252Bantitrypanosomal%252B4-phenyl-6-%252528pyridin-3-yl%252529pyrimidines%26aulast%3DRobinson%26aufirst%3DWilliam%2BJ.%26date%3D2021%26volume%3D209%26spage%3D112871" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vasiliki  Pardali</span>, <span class="hlFld-ContribAuthor ">Erofili  Giannakopoulou</span>, <span class="hlFld-ContribAuthor ">Dimitrios-Ilias  Balourdas</span>, <span class="hlFld-ContribAuthor ">Vassilios  Myrianthopoulos</span>, <span class="hlFld-ContribAuthor ">Martin C.  Taylor</span>, <span class="hlFld-ContribAuthor ">Marina  Šekutor</span>, <span class="hlFld-ContribAuthor ">Kata  Mlinarić-Majerski</span>, <span class="hlFld-ContribAuthor ">John M.  Kelly</span>, <span class="hlFld-ContribAuthor ">Grigoris  Zoidis</span>. </span><span class="cited-content_cbyCitation_article-title">Lipophilic Guanylhydrazone Analogues as Promising Trypanocidal Agents: An Extended SAR Study. </span><span class="cited-content_cbyCitation_journal-name">Current Pharmaceutical Design</span><span> <strong>2020,</strong> <em>26 </em>
                                    (8)
                                     , 838-866. <a href="https://doi.org/10.2174/1381612826666200210150127" title="DOI URL">https://doi.org/10.2174/1381612826666200210150127</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1381612826666200210150127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1381612826666200210150127%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Pharmaceutical%2520Design%26atitle%3DLipophilic%252BGuanylhydrazone%252BAnalogues%252Bas%252BPromising%252BTrypanocidal%252BAgents%25253A%252BAn%252BExtended%252BSAR%252BStudy%26aulast%3DPardali%26aufirst%3DVasiliki%26date%3D2020%26volume%3D26%26issue%3D8%26spage%3D838%26epage%3D866" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Margaret A.  Phillips</span>. </span><span class="cited-content_cbyCitation_article-title">Polyamines in protozoan pathogens. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biological Chemistry</span><span> <strong>2018,</strong> <em>293 </em>
                                    (48)
                                     , 18746-18756. <a href="https://doi.org/10.1074/jbc.TM118.003342" title="DOI URL">https://doi.org/10.1074/jbc.TM118.003342</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1074/jbc.TM118.003342&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1074%2Fjbc.TM118.003342%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biological%2520Chemistry%26atitle%3DPolyamines%252Bin%252Bprotozoan%252Bpathogens%26aulast%3DPhillips%26aufirst%3DMargaret%2BA.%26date%3D2018%26volume%3D293%26issue%3D48%26spage%3D18746%26epage%3D18756" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Manish M.  Patel</span>, <span class="hlFld-ContribAuthor ">Oleg A.  Volkov</span>, <span class="hlFld-ContribAuthor ">Christopher  Leija</span>, <span class="hlFld-ContribAuthor ">Andrew  Lemoff</span>, <span class="hlFld-ContribAuthor ">Margaret A.  Phillips</span>, . </span><span class="cited-content_cbyCitation_article-title">A dual regulatory circuit consisting of S-adenosylmethionine decarboxylase protein and its reaction product controls expression of the paralogous activator prozyme in Trypanosoma brucei. </span><span class="cited-content_cbyCitation_journal-name">PLOS Pathogens</span><span> <strong>2018,</strong> <em>14 </em>
                                    (10)
                                     , e1007404. <a href="https://doi.org/10.1371/journal.ppat.1007404" title="DOI URL">https://doi.org/10.1371/journal.ppat.1007404</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.ppat.1007404&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.ppat.1007404%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520Pathogens%26atitle%3DA%252Bdual%252Bregulatory%252Bcircuit%252Bconsisting%252Bof%252BS-adenosylmethionine%252Bdecarboxylase%252Bprotein%252Band%252Bits%252Breaction%252Bproduct%252Bcontrols%252Bexpression%252Bof%252Bthe%252Bparalogous%252Bactivator%252Bprozyme%252Bin%252BTrypanosoma%252Bbrucei%26aulast%3DPatel%26aufirst%3DManish%2BM.%26date%3D2018%26date%3D2018%26volume%3D14%26issue%3D10%26spage%3De1007404" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/medium/jm-2017-01654m_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/large/jm-2017-01654m_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01654&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/medium/jm-2017-01654m_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/large/jm-2017-01654m_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Pyrimidineamines<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/large/jm-2017-01654m_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01654&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p><span class="fn-label">Scheme a</span>Reagents and conditions: (a) <b>2a</b>, 3,5-Cl<sub>2</sub>PhNH<sub>2</sub>, DMSO, 100 °C, 18 h (<b>8</b>); (b) <b>2a</b>–<b>c</b>,<b>f</b>–<b>h</b> or <b>3g</b>, aniline (3,5-Cl<sub>2</sub>PhNH<sub>2</sub> or 3-Br-5-ClPhNH<sub>2</sub> or <b>6</b>), conc HCl, EtOH, microwave (140 °C), 1.5 h (<b>9</b>, <b>10</b>, <b>15</b>, <b>16</b>, <b>26</b>, <b>32</b>–<b>34</b>, <b>36</b>, <b>37</b>, <b>44</b>, <b>45</b>, <b>53</b>, <b>54</b>, <b>61</b>–<b>68</b>, <b>72</b>, <b>73</b>, <b>89</b>); (c) <b>2a</b>,<b>d</b>,<b>e</b>,<b>h</b> or <b>3d</b>–<b>f</b>,<b>h</b>, aniline (3,5-Cl<sub>2</sub>PhNH<sub>2</sub> or 3-Br-5-ClPhNH<sub>2</sub> or <b>5a</b> or <b>6</b>) conc HCl, EtOH, reflux, 18 h (<b>13</b>, <b>14</b>, <b>23</b>–<b>25</b>, <b>27</b>, <b>28</b>, <b>38</b>–<b>43</b>, <b>46</b>–<b>48</b>, <b>52</b>, <b>55</b>, <b>60</b>, <b>69</b>–<b>71</b>); (d) <b>7</b> or <b>13</b> or <b>32</b> or <b>38</b>, ArylB(OH)<sub>2</sub>, Pd(OAc)<sub>2</sub>, PPh<sub>3</sub>, Na<sub>2</sub>CO<sub>3</sub>, H<sub>2</sub>O/DME, 90 °C, 18 h (<b>11</b>, <b>35</b>, <b>50</b>, <b>91</b>–<b>109</b>, <b>116</b>–<b>118</b>); (e) <b>13,</b> DIPEA, morpholine, CH<sub>3</sub>CN, microwave (140 °C), 2 h (<b>12</b>); (f) 3,5-Cl<sub>2</sub>PhNH<sub>2</sub>, <b>3a</b>,<b>c</b>,<b>i</b>,<b>j</b>, conc HCl, BuOH, microwave (185 °C), 4 h (<b>18</b>–<b>20</b>, <b>22</b>); (g) 3,5-Cl<sub>2</sub>PhNH<sub>2</sub>, <b>3b</b>, conc HCl, MeOH/H<sub>2</sub>O (1/1), sealed tube, 100 °C, 18 h (<b>21</b>); (h) <b>4</b>, 3,5-Cl<sub>2</sub>PhNH<sub>2</sub>, cat. Pd<sub>2</sub>(dba)<sub>3</sub>, BINAP, NaO<sup><i>t</i></sup>Bu, toluene, 70 °C, 18 h; then 3 N aq NaOH, EtOH, reflux, 18 h (<b>17</b>); (i) <b>2h</b>, <b>5b</b>, Et<sub>3</sub>N, DMF, rt, 24 h (<b>29</b>); (j) <b>2a</b>, <b>5c</b>,<b>d</b>, DIPEA, CH<sub>3</sub>CN, microwave (140 °C), 2 h (<b>30</b>, <b>31</b>); (k) <b>48</b>, conc H<sub>2</sub>SO<sub>4</sub>, MeOH, 4 Å molecular sieves, reflux, 18 h (<b>49</b>); (l) <b>48</b>, 3,5-Cl<sub>2</sub>PhNH<sub>2</sub>, EDI, py/DCM/DMF (0.4/1/1), rt, 18 h (<b>51</b>); (m) <b>32</b>, heteroarylboronic acid MIDA ester, cat. Pd(OAc)<sub>2</sub>, PPh<sub>3</sub>, sodium triphosphate, dioxane/H<sub>2</sub>O) (4/1), 90 °C, 18 h (<b>110</b>–<b>115</b>).</p>.</p></figure><figure data-id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/medium/jm-2017-01654m_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/large/jm-2017-01654m_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Pyrimidineamines are pH-dependent <i>Tb</i>AdoMetDC inhibitors. (A) Structure of the protonated form of pyrimidineamine <b>7</b> (N1 is predicted to have p<i>K</i><sub>a</sub> of 7.3) and structure of mechanism-based inhibitor Genz-644131 (<b>1</b>). (B) IC<sub>50</sub> of pyrimidineamines plotted as a function of the assay buffer pH and fitted with linear regression analysis in Prism (GraphPad). Predicted p<i>K</i><sub>a</sub>(N1) values are shown in parentheses. The slopes for pyrimidineamines with p<i>K</i><sub>a</sub>(N1) = 7.3 (solid lines) are compared with those for analogs with p<i>K</i><sub>a</sub>(N1) > 9 (dashed lines) and with that for a non-pyrimidineamine control (dotted line).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/large/jm-2017-01654m_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01654&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/medium/jm-2017-01654m_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/large/jm-2017-01654m_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Mechanism of inhibition studies for <b>7</b>. (A) Evaluation of reversibility. Recovery of enzyme activity after a 30 min incubation with inhibitor followed by a rapid 100-fold dilution of the inhibitor. The progression of the reaction was measured as product formation using RapidFire–MS analysis. (B) Determination of the inhibition modality. Double-reciprocal Lineweaver–Burk plot of substrate titrations in the presence of four inhibitor concentrations shows competitive inhibition.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/large/jm-2017-01654m_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01654&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/medium/jm-2017-01654m_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/large/jm-2017-01654m_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Crystal structure of <i>Tb</i>AdoMetDC/Prozyme in complex with <b>44</b>. (A) Surface representation of <i>Tb</i>AdoMetDC/Prozyme (blue/yellow) with <b>44</b> (orange sticks) occupying the catalytic site. Pyruvoyl residue (Pvl) is shown as spheres. (B) Surface representation of the catalytic site. Residues forming the catalytic-site cavity are shown as sticks. Select interatomic distances are shown as dashed lines with values in Å. The catalytic-site cavity is shown as a semitransparent gray surface. Two pockets that are partially occupied by bromines are labeled <i>a</i> and <i>b</i> in purple.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/large/jm-2017-01654m_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01654&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/medium/jm-2017-01654m_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/large/jm-2017-01654m_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Comparison of human and <i>T. brucei</i> AdoMetDC active sites. Human AdoMetDC (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3DZ7">3DZ7</a>, brown) was aligned with <i>Tb</i>AdoMetDC structure (blue) cocrystallized with <b>44</b> (orange). The analogous residues are labeled in the corresponding colors (<i>T. brucei</i> residues on top). Alignment was done in PyMol, and the side chains within 4 Å of an inhibitor (with the exception of <i>Tb</i>Glu86/<i>Hs</i>Glu68) are shown here. A dashed line represents a distance, in Å, between His5 of the human protein and <b>44</b> in the aligned <i>T. brucei</i> structure. Residues Ser25 and Arg26, which were not modeled in <b>44</b>-liganded structure, are from the previously reported unliganded <i>Tb</i>AdoMetDC structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TVM">5TVM</a>, yellow). The active-site cavities for human (brown) and <b>44</b>-bound <i>T. brucei</i> (blue) AdoMetDCs are shown as semitransparent surfaces (pocket <i>b</i> is pointing away from the viewer).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/large/jm-2017-01654m_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01654&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/medium/jm-2017-01654m_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/large/jm-2017-01654m_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Mechanism of action studies in <i>T. brucei</i> bloodstream form parasites. Western blot analysis of Prozyme levels in <i>T. brucei</i> cells treated with active or inactive pyrimidineamine analogs for 24 h. Three types of compounds were compared: (1) compounds that demonstrated low-micromolar inhibition of <i>Tb</i>AdoMetDC at pH 7.2 (IC<sub>50(pH7.2)</sub>) and of <i>T. brucei</i> cells in culture (EC<sub>50</sub>) (<b>7</b> and <b>8</b>); (2) compounds that did not inhibit <i>Tb</i>AdoMetDC but did inhibit <i>T. brucei</i> cell growth (<b>23</b> and <b>13</b>); (3) a compound that did not inhibit either <i>Tb</i>AdoMetDC or the cells (<b>29</b>). Cells were treated with 1.5 × EC<sub>50</sub> concentrations of inhibitors (or 30 μM <b>29</b>). <i>T. brucei</i> binding protein (BiP) was used as a loading control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-3/acs.jmedchem.7b01654/20180201/images/large/jm-2017-01654m_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b01654&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i125">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32252" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32252" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 52 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><a href="http://www.who.int/neglected_diseases/diseases/en/" class="extLink">http://www.who.int/neglected_diseases/diseases/en/</a> (accessed Jun 12,<span class="NLM_x"> </span><span class="NLM_year">2017</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=http%3A%2F%2Fwww.who.int%2Fneglected_diseases%2Fdiseases%2Fen%2F+%28accessed+Jun+12%2C+2017%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Kennedy, P. G. E.</span><span> </span><span class="NLM_article-title">Clinical features, diagnosis, and treatment of human African trypanosomiasis (sleeping sickness)</span> <span class="citation_source-journal">Lancet Neurol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">186</span><span class="NLM_x">–</span> <span class="NLM_lpage">194</span><span class="refDoi"> DOI: 10.1016/S1474-4422(12)70296-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1016%2FS1474-4422%2812%2970296-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=23260189" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A280%3ADC%252BC3s3ktVCgsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=186-194&author=P.+G.+E.+Kennedy&title=Clinical+features%2C+diagnosis%2C+and+treatment+of+human+African+trypanosomiasis+%28sleeping+sickness%29&doi=10.1016%2FS1474-4422%2812%2970296-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical features, diagnosis, and treatment of human African trypanosomiasis (sleeping sickness)</span></div><div class="casAuthors">Kennedy Peter Ge</div><div class="citationInfo"><span class="NLM_cas:title">The Lancet. Neurology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">186-94</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Human African trypanosomiasis, or sleeping sickness, is caused by infection with parasites of the genus Trypanosoma, transmitted by the tsetse fly.  The disease has two forms, Trypanosoma brucei (T b) rhodesiense and T b gambiense; and is almost always fatal if untreated.  Despite a recent reduction in the number of reported cases, patients with African trypanosomiasis continue to present major challenges to clinicians.  Because treatment for CNS-stage disease can be very toxic, diagnostic staging to distinguish early-stage from late-stage disease when the CNS in invaded is crucial but remains problematic.  Melarsoprol is the only available treatment for late-stage T b rhodesiense infection, but can be lethal to 5% of patients owing to post-treatment reactive encephalopathy.  Eflornithine combined with nifurtimox is the first-line treatment for late-stage T b gambiense.  New drugs are in the pipeline for treatment of CNS human African trypanosomiasis, giving rise to cautious optimism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSDi0PRo3CXkCEJCPKhJPuqfW6udTcc2eaYnWDMAQyRZrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3s3ktVCgsQ%253D%253D&md5=c9ac342f345240d413300c17cd1ae80f</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2FS1474-4422%2812%2970296-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1474-4422%252812%252970296-X%26sid%3Dliteratum%253Aachs%26aulast%3DKennedy%26aufirst%3DP.%2BG.%2BE.%26atitle%3DClinical%2520features%252C%2520diagnosis%252C%2520and%2520treatment%2520of%2520human%2520African%2520trypanosomiasis%2520%2528sleeping%2520sickness%2529%26jtitle%3DLancet%2520Neurol.%26date%3D2013%26volume%3D12%26spage%3D186%26epage%3D194%26doi%3D10.1016%2FS1474-4422%2812%2970296-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Franco, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simarro, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diarra, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jannin, J. G.</span><span> </span><span class="NLM_article-title">Epidemiology of human African trypanosomiasis</span> <span class="citation_source-journal">Clin. Epidemiol</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">257</span><span class="NLM_x">–</span> <span class="NLM_lpage">275</span><span class="refDoi"> DOI: 10.2147/CLEP.S39728</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.2147%2FCLEP.S39728" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=25125985" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A280%3ADC%252BC2M%252FitFOhtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=257-275&author=J.+R.+Francoauthor=P.+P.+Simarroauthor=A.+Diarraauthor=J.+G.+Jannin&title=Epidemiology+of+human+African+trypanosomiasis&doi=10.2147%2FCLEP.S39728"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Epidemiology of human African trypanosomiasis</span></div><div class="casAuthors">Franco Jose R; Simarro Pere P; Jannin Jean G; Diarra Abdoulaye</div><div class="citationInfo"><span class="NLM_cas:title">Clinical epidemiology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">257-75</span>
        ISSN:<span class="NLM_cas:issn">1179-1349</span>.
    </div><div class="casAbstract">Human African trypanosomiasis (HAT), or sleeping sickness, is caused by Trypanosoma brucei gambiense, which is a chronic form of the disease present in western and central Africa, and by Trypanosoma brucei rhodesiense, which is an acute disease located in eastern and southern Africa.  The rhodesiense form is a zoonosis, with the occasional infection of humans, but in the gambiense form, the human being is regarded as the main reservoir that plays a key role in the transmission cycle of the disease.  The gambiense form currently assumes that 98% of the cases are declared; the Democratic Republic of the Congo is the most affected country, with more than 75% of the gambiense cases declared.  The epidemiology of the disease is mediated by the interaction of the parasite (trypanosome) with the vectors (tsetse flies), as well as with the human and animal hosts within a particular environment.  Related to these interactions, the disease is confined in spatially limited areas called "foci", which are located in Sub-Saharan Africa, mainly in remote rural areas.  The risk of contracting HAT is, therefore, determined by the possibility of contact of a human being with an infected tsetse fly.  Epidemics of HAT were described at the beginning of the 20th century; intensive activities have been set up to confront the disease, and it was under control in the 1960s, with fewer than 5,000 cases reported in the whole continent.  The disease resurged at the end of the 1990s, but renewed efforts from endemic countries, cooperation agencies, and nongovernmental organizations led by the World Health Organization succeeded to raise awareness and resources, while reinforcing national programs, reversing the trend of the cases reported, and bringing the disease under control again.  In this context, sustainable elimination of the gambiense HAT, defined as the interruption of the transmission of the disease, was considered as a feasible target for 2030.  Since rhodesiense HAT is a zoonosis, where the animal reservoir plays a key role, the interruption of the disease's transmission is not deemed feasible.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRLQPTkbz605s25P7XSLhESfW6udTcc2ea08XM-wuI317ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M%252FitFOhtw%253D%253D&md5=484f0de3184f034ce92654ee5baf2bd2</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.2147%2FCLEP.S39728&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FCLEP.S39728%26sid%3Dliteratum%253Aachs%26aulast%3DFranco%26aufirst%3DJ.%2BR.%26aulast%3DSimarro%26aufirst%3DP.%2BP.%26aulast%3DDiarra%26aufirst%3DA.%26aulast%3DJannin%26aufirst%3DJ.%2BG.%26atitle%3DEpidemiology%2520of%2520human%2520African%2520trypanosomiasis%26jtitle%3DClin.%2520Epidemiol%26date%3D2014%26volume%3D6%26spage%3D257%26epage%3D275%26doi%3D10.2147%2FCLEP.S39728" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><a href="http://www.who.int/mediacentre/factsheets/fs259/en/" class="extLink">http://www.who.int/mediacentre/factsheets/fs259/en/</a> (accessed Jun 12,<span class="NLM_x"> </span><span class="NLM_year">2017</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=http%3A%2F%2Fwww.who.int%2Fmediacentre%2Ffactsheets%2Ffs259%2Fen%2F+%28accessed+Jun+12%2C+2017%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Franco, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simarro, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diarra, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruiz-Postigo, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jannin, J. G.</span><span> </span><span class="NLM_article-title">The journey towards elimination of gambiense human African trypanosomiasis: not far, nor easy</span> <span class="citation_source-journal">Parasitology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">141</span><span class="NLM_x">, </span> <span class="NLM_fpage">748</span><span class="NLM_x">–</span> <span class="NLM_lpage">760</span><span class="refDoi"> DOI: 10.1017/S0031182013002102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1017%2FS0031182013002102" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=141&publication_year=2014&pages=748-760&author=J.+R.+Francoauthor=P.+P.+Simarroauthor=A.+Diarraauthor=J.+A.+Ruiz-Postigoauthor=J.+G.+Jannin&title=The+journey+towards+elimination+of+gambiense+human+African+trypanosomiasis%3A+not+far%2C+nor+easy&doi=10.1017%2FS0031182013002102"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1017%2FS0031182013002102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS0031182013002102%26sid%3Dliteratum%253Aachs%26aulast%3DFranco%26aufirst%3DJ.%2BR.%26aulast%3DSimarro%26aufirst%3DP.%2BP.%26aulast%3DDiarra%26aufirst%3DA.%26aulast%3DRuiz-Postigo%26aufirst%3DJ.%2BA.%26aulast%3DJannin%26aufirst%3DJ.%2BG.%26atitle%3DThe%2520journey%2520towards%2520elimination%2520of%2520gambiense%2520human%2520African%2520trypanosomiasis%253A%2520not%2520far%252C%2520nor%2520easy%26jtitle%3DParasitology%26date%3D2014%26volume%3D141%26spage%3D748%26epage%3D760%26doi%3D10.1017%2FS0031182013002102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Berthier, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breniere, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bras-Goncalves, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lemesre, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jamonneau, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solano, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lejon, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thevenon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bucheton, B.</span><span> </span><span class="NLM_article-title">Tolerance to trypanosomatids: a threat, or a key for disease elimination?</span> <span class="citation_source-journal">Trends Parasitol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">157</span><span class="NLM_x">–</span> <span class="NLM_lpage">168</span><span class="refDoi"> DOI: 10.1016/j.pt.2015.11.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1016%2Fj.pt.2015.11.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=26643519" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A280%3ADC%252BC28votlOktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2016&pages=157-168&author=D.+Berthierauthor=S.+F.+Breniereauthor=R.+Bras-Goncalvesauthor=J.+L.+Lemesreauthor=V.+Jamonneauauthor=P.+Solanoauthor=V.+Lejonauthor=S.+Thevenonauthor=B.+Bucheton&title=Tolerance+to+trypanosomatids%3A+a+threat%2C+or+a+key+for+disease+elimination%3F&doi=10.1016%2Fj.pt.2015.11.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Tolerance to Trypanosomatids: A Threat, or a Key for Disease Elimination?</span></div><div class="casAuthors">Berthier David; Thevenon Sophie; Breniere Simone F; Bras-Goncalves Rachel; Lemesre Jean-Loup; Jamonneau Vincent; Solano Philippe; Lejon Veerle; Bucheton Bruno</div><div class="citationInfo"><span class="NLM_cas:title">Trends in parasitology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">157-68</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">So far, research on trypanosomatid infections has been driven by 'disease by disease' approaches, leading to different concepts and control strategies.  It is, however, increasingly clear that they share common features such as the ability to generate long-lasting asymptomatic infections in their mammalian hosts.  Trypanotolerance, long integrated in animal African trypanosomiasis control, historically refers to the ability of cattle breeds to limit Trypanosoma infection and pathology, but has only recently been recognized in humans.  Whilst trypanotolerance is absent from the vocabulary on leishmaniasis and Chagas disease, asymptomatic infections also occur.  We review the concept of trypanotolerance across the trypanosomatids and discuss the importance of asymptomatic carriage in the current context of elimination.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSwxKZVEIfebJdYoSR9eyvsfW6udTcc2eY_jBs5ioorRLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28votlOktQ%253D%253D&md5=7fcfa49a6aeb63f78dffb3150806be43</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.pt.2015.11.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pt.2015.11.001%26sid%3Dliteratum%253Aachs%26aulast%3DBerthier%26aufirst%3DD.%26aulast%3DBreniere%26aufirst%3DS.%2BF.%26aulast%3DBras-Goncalves%26aufirst%3DR.%26aulast%3DLemesre%26aufirst%3DJ.%2BL.%26aulast%3DJamonneau%26aufirst%3DV.%26aulast%3DSolano%26aufirst%3DP.%26aulast%3DLejon%26aufirst%3DV.%26aulast%3DThevenon%26aufirst%3DS.%26aulast%3DBucheton%26aufirst%3DB.%26atitle%3DTolerance%2520to%2520trypanosomatids%253A%2520a%2520threat%252C%2520or%2520a%2520key%2520for%2520disease%2520elimination%253F%26jtitle%3DTrends%2520Parasitol.%26date%3D2016%26volume%3D32%26spage%3D157%26epage%3D168%26doi%3D10.1016%2Fj.pt.2015.11.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Capewell, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cren-Travaille, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchesi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnston, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clucas, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benson, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorman, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calvo-Alvarez, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crouzols, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jouvion, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jamonneau, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weir, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevenson, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’neill, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cooper, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swar, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bucheton, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ngoyi, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garside, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rotureau, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macleod, A.</span><span> </span><span class="NLM_article-title">The skin is a significant but overlooked anatomical reservoir for vector-borne African trypanosomes</span> <span class="citation_source-journal">eLife</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">e17716</span><span class="refDoi"> DOI: 10.7554/eLife.17716</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.7554%2FeLife.17716" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=27653219" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BC1cXovV2jsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&pages=e17716&author=P.+Capewellauthor=C.+Cren-Travailleauthor=F.+Marchesiauthor=P.+Johnstonauthor=C.+Clucasauthor=R.+A.+Bensonauthor=T.+A.+Gormanauthor=E.+Calvo-Alvarezauthor=A.+Crouzolsauthor=G.+Jouvionauthor=V.+Jamonneauauthor=W.+Weirauthor=M.+L.+Stevensonauthor=K.+O%E2%80%99neillauthor=A.+Cooperauthor=N.+K.+Swarauthor=B.+Buchetonauthor=D.+M.+Ngoyiauthor=P.+Garsideauthor=B.+Rotureauauthor=A.+Macleod&title=The+skin+is+a+significant+but+overlooked+anatomical+reservoir+for+vector-borne+African+trypanosomes&doi=10.7554%2FeLife.17716"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The skin is a significant but overlooked anatomical reservoir for vector-borne African trypanosomes</span></div><div class="casAuthors">Capewell, Paul; Cren-Travaille, Christelle; Marchesi, Francesco; Johnston, Pamela; Clucas, Caroline; Benson, Robert A.; Gorman, Taylor-Anne; Calvo-Alvarez, Estefania; Crouzols, Aline; Jouvion, Gregory; Jammoneau, Vincent; Weir, William; Stevenson, M. Lynn; O'Neill, Kerry; Cooper, Anneli; Swar, Nono-Raymond Kuispond; Bucheton, Bruno; Ngoyi, Dieudonne Mumba; Garside, Paul; Rotureau, Brice; MacLeod, Annette</div><div class="citationInfo"><span class="NLM_cas:title">eLife</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">e17716/1-e17716/17</span>CODEN:
                <span class="NLM_cas:coden">ELIFA8</span>;
        ISSN:<span class="NLM_cas:issn">2050-084X</span>.
    
            (<span class="NLM_cas:orgname">eLife Sciences Publications Ltd.</span>)
        </div><div class="casAbstract">The role of mammalian skin in harbouring and transmitting arthropod-borne protozoan parasites has been overlooked for decades as these pathogens have been regarded primarily as blood-dwelling organisms.  Intriguingly, infections with low or undetected blood parasites are common, particularly in the case of Human African Trypanosomiasis caused by Trypanosoma brucei gambiense.  We hypothesise, therefore, the skin represents an anat. reservoir of infection.  Here we definitively show that substantial quantities of trypanosomes exist within the skin following exptl. infection, which can be transmitted to the tsetse vector, even in the absence of detectable parasitemia.  Importantly, we demonstrate the presence of extravascular parasites in human skin biopsies from undiagnosed individuals.  The identification of this novel reservoir requires a re-evaluation of current diagnostic methods and control policies.  More broadly, our results indicate that transmission is a key evolutionary force driving parasite extravasation that could further result in tissue invasion-dependent pathol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbxYT8sS1X77Vg90H21EOLACvtfcHk0liQkaXZVkqyxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXovV2jsLY%253D&md5=abb13f75ab2392c328f7e93bc925926e</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.7554%2FeLife.17716&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.17716%26sid%3Dliteratum%253Aachs%26aulast%3DCapewell%26aufirst%3DP.%26aulast%3DCren-Travaille%26aufirst%3DC.%26aulast%3DMarchesi%26aufirst%3DF.%26aulast%3DJohnston%26aufirst%3DP.%26aulast%3DClucas%26aufirst%3DC.%26aulast%3DBenson%26aufirst%3DR.%2BA.%26aulast%3DGorman%26aufirst%3DT.%2BA.%26aulast%3DCalvo-Alvarez%26aufirst%3DE.%26aulast%3DCrouzols%26aufirst%3DA.%26aulast%3DJouvion%26aufirst%3DG.%26aulast%3DJamonneau%26aufirst%3DV.%26aulast%3DWeir%26aufirst%3DW.%26aulast%3DStevenson%26aufirst%3DM.%2BL.%26aulast%3DO%25E2%2580%2599neill%26aufirst%3DK.%26aulast%3DCooper%26aufirst%3DA.%26aulast%3DSwar%26aufirst%3DN.%2BK.%26aulast%3DBucheton%26aufirst%3DB.%26aulast%3DNgoyi%26aufirst%3DD.%2BM.%26aulast%3DGarside%26aufirst%3DP.%26aulast%3DRotureau%26aufirst%3DB.%26aulast%3DMacleod%26aufirst%3DA.%26atitle%3DThe%2520skin%2520is%2520a%2520significant%2520but%2520overlooked%2520anatomical%2520reservoir%2520for%2520vector-borne%2520African%2520trypanosomes%26jtitle%3DeLife%26date%3D2016%26volume%3D5%26spage%3De17716%26doi%3D10.7554%2FeLife.17716" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Trindade, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rijo-Ferreira, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carvalho, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pinto-Neves, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guegan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aresta-Branco, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bento, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pinto, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Den Abbeele, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ribeiro, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dias, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Figueiredo, L. M.</span><span> </span><span class="NLM_article-title">Trypanosoma brucei parasites occupy and functionally adapt to the adipose tissue in mice</span> <span class="citation_source-journal">Cell Host Microbe</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">837</span><span class="NLM_x">–</span> <span class="NLM_lpage">848</span><span class="refDoi"> DOI: 10.1016/j.chom.2016.05.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1016%2Fj.chom.2016.05.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=27237364" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BC28Xosl2jtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2016&pages=837-848&author=S.+Trindadeauthor=F.+Rijo-Ferreiraauthor=T.+Carvalhoauthor=D.+Pinto-Nevesauthor=F.+Gueganauthor=F.+Aresta-Brancoauthor=F.+Bentoauthor=S.+A.+Youngauthor=A.+Pintoauthor=J.+Van+Den+Abbeeleauthor=R.+M.+Ribeiroauthor=S.+Diasauthor=T.+K.+Smithauthor=L.+M.+Figueiredo&title=Trypanosoma+brucei+parasites+occupy+and+functionally+adapt+to+the+adipose+tissue+in+mice&doi=10.1016%2Fj.chom.2016.05.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Trypanosoma brucei Parasites Occupy and Functionally Adapt to the Adipose Tissue in Mice</span></div><div class="casAuthors">Trindade, Sandra; Rijo-Ferreira, Filipa; Carvalho, Tania; Pinto-Neves, Daniel; Guegan, Fabien; Aresta-Branco, Francisco; Bento, Fabio; Young, Simon A.; Pinto, Andreia; Van Den Abbeele, Jan; Ribeiro, Ruy M.; Dias, Sergio; Smith, Terry K.; Figueiredo, Luisa M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Host & Microbe</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">837-848</span>CODEN:
                <span class="NLM_cas:coden">CHMECB</span>;
        ISSN:<span class="NLM_cas:issn">1931-3128</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Trypanosoma brucei is an extracellular parasite that causes sleeping sickness.  In mammalian hosts, trypanosomes are thought to exist in two major niches: early in infection, they populate the blood; later, they breach the blood-brain barrier.  Working with a well-established mouse model, we discovered that adipose tissue constitutes a third major reservoir for T. brucei.  Parasites from adipose tissue, here termed adipose tissue forms (ATFs), can replicate and were capable of infecting a naive animal.  ATFs were transcriptionally distinct from bloodstream forms, and the genes upregulated included putative fatty acid β-oxidn. enzymes.  Consistent with this, ATFs were able to utilize exogenous myristate and form β-oxidn. intermediates, suggesting that ATF parasites can use fatty acids as an external carbon source.  These findings identify the adipose tissue as a niche for T. brucei during its mammalian life cycle and could potentially explain the wt. loss assocd. with sleeping sickness.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCGv-3jWWQrrVg90H21EOLACvtfcHk0liQkaXZVkqyxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xosl2jtbw%253D&md5=5cc016076dcee97f48dc3eb0b268c036</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.chom.2016.05.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chom.2016.05.002%26sid%3Dliteratum%253Aachs%26aulast%3DTrindade%26aufirst%3DS.%26aulast%3DRijo-Ferreira%26aufirst%3DF.%26aulast%3DCarvalho%26aufirst%3DT.%26aulast%3DPinto-Neves%26aufirst%3DD.%26aulast%3DGuegan%26aufirst%3DF.%26aulast%3DAresta-Branco%26aufirst%3DF.%26aulast%3DBento%26aufirst%3DF.%26aulast%3DYoung%26aufirst%3DS.%2BA.%26aulast%3DPinto%26aufirst%3DA.%26aulast%3DVan%2BDen%2BAbbeele%26aufirst%3DJ.%26aulast%3DRibeiro%26aufirst%3DR.%2BM.%26aulast%3DDias%26aufirst%3DS.%26aulast%3DSmith%26aufirst%3DT.%2BK.%26aulast%3DFigueiredo%26aufirst%3DL.%2BM.%26atitle%3DTrypanosoma%2520brucei%2520parasites%2520occupy%2520and%2520functionally%2520adapt%2520to%2520the%2520adipose%2520tissue%2520in%2520mice%26jtitle%3DCell%2520Host%2520Microbe%26date%3D2016%26volume%3D19%26spage%3D837%26epage%3D848%26doi%3D10.1016%2Fj.chom.2016.05.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Tanowitz, H. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scherer, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mota, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Figueiredo, L. M.</span><span> </span><span class="NLM_article-title">Adipose tissue: a safe haven for parasites?</span> <span class="citation_source-journal">Trends Parasitol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">276</span><span class="NLM_x">–</span> <span class="NLM_lpage">284</span><span class="refDoi"> DOI: 10.1016/j.pt.2016.11.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1016%2Fj.pt.2016.11.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=28007406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A280%3ADC%252BC1c%252FnslCmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2017&pages=276-284&author=H.+B.+Tanowitzauthor=P.+E.+Schererauthor=M.+M.+Motaauthor=L.+M.+Figueiredo&title=Adipose+tissue%3A+a+safe+haven+for+parasites%3F&doi=10.1016%2Fj.pt.2016.11.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Adipose Tissue: A Safe Haven for Parasites?</span></div><div class="casAuthors">Tanowitz Herbert B; Scherer Philipp E; Mota Maria M; Figueiredo Luisa M</div><div class="citationInfo"><span class="NLM_cas:title">Trends in parasitology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">276-284</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Adipose tissue (AT) is no longer regarded as an inert lipid storage, but as an important central regulator in energy homeostasis and immunity.  Three parasite species are uniquely associated with AT during part of their life cycle: Trypanosoma cruzi, the causative agent of Chagas disease; Trypanosoma brucei, the causative agent of African sleeping sickness; and Plasmodium spp., the causative agents of malaria.  In AT, T. cruzi resides inside adipocytes, T. brucei is found in the interstitial spaces between adipocytes, while Plasmodium spp. infect red blood cells, which may adhere to the blood vessels supplying AT.  Here, we discuss how each parasite species adapts to this tissue environment and what the implications are for pathogenesis, clinical manifestations, and therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRFtbJdIvuLe0NoizSOkG_FfW6udTcc2eYed8w4QvEeZrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1c%252FnslCmsA%253D%253D&md5=c1b93d6948329aad712d682b2f970906</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.pt.2016.11.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pt.2016.11.008%26sid%3Dliteratum%253Aachs%26aulast%3DTanowitz%26aufirst%3DH.%2BB.%26aulast%3DScherer%26aufirst%3DP.%2BE.%26aulast%3DMota%26aufirst%3DM.%2BM.%26aulast%3DFigueiredo%26aufirst%3DL.%2BM.%26atitle%3DAdipose%2520tissue%253A%2520a%2520safe%2520haven%2520for%2520parasites%253F%26jtitle%3DTrends%2520Parasitol.%26date%3D2017%26volume%3D33%26spage%3D276%26epage%3D284%26doi%3D10.1016%2Fj.pt.2016.11.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Simarro, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Franco, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diarra, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Postigo, J. a. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jannin, J.</span><span> </span><span class="NLM_article-title">Update on field use of the available drugs for the chemotherapy of human African trypanosomiasis</span> <span class="citation_source-journal">Parasitology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">139</span><span class="NLM_x">, </span> <span class="NLM_fpage">842</span><span class="NLM_x">–</span> <span class="NLM_lpage">846</span><span class="refDoi"> DOI: 10.1017/S0031182012000169</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1017%2FS0031182012000169" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2012&pages=842-846&author=P.+P.+Simarroauthor=J.+Francoauthor=A.+Diarraauthor=J.+a.+R.+Postigoauthor=J.+Jannin&title=Update+on+field+use+of+the+available+drugs+for+the+chemotherapy+of+human+African+trypanosomiasis&doi=10.1017%2FS0031182012000169"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1017%2FS0031182012000169&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS0031182012000169%26sid%3Dliteratum%253Aachs%26aulast%3DSimarro%26aufirst%3DP.%2BP.%26aulast%3DFranco%26aufirst%3DJ.%26aulast%3DDiarra%26aufirst%3DA.%26aulast%3DPostigo%26aufirst%3DJ.%2Ba.%2BR.%26aulast%3DJannin%26aufirst%3DJ.%26atitle%3DUpdate%2520on%2520field%2520use%2520of%2520the%2520available%2520drugs%2520for%2520the%2520chemotherapy%2520of%2520human%2520African%2520trypanosomiasis%26jtitle%3DParasitology%26date%3D2012%26volume%3D139%26spage%3D842%26epage%3D846%26doi%3D10.1017%2FS0031182012000169" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Iten, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mett, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Enyaru, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brun, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaminsky, R.</span><span> </span><span class="NLM_article-title">Alterations in ornithine decarboxylase characteristics account for tolerance of Trypanosoma brucei rhodesiense to D,L-alpha-difluoromethylornithine</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">1922</span><span class="NLM_x">–</span> <span class="NLM_lpage">1925</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1997&pages=1922-1925&author=M.+Itenauthor=H.+Mettauthor=A.+Evansauthor=J.+C.+Enyaruauthor=R.+Brunauthor=R.+Kaminsky&title=Alterations+in+ornithine+decarboxylase+characteristics+account+for+tolerance+of+Trypanosoma+brucei+rhodesiense+to+D%2CL-alpha-difluoromethylornithine"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DIten%26aufirst%3DM.%26aulast%3DMett%26aufirst%3DH.%26aulast%3DEvans%26aufirst%3DA.%26aulast%3DEnyaru%26aufirst%3DJ.%2BC.%26aulast%3DBrun%26aufirst%3DR.%26aulast%3DKaminsky%26aufirst%3DR.%26atitle%3DAlterations%2520in%2520ornithine%2520decarboxylase%2520characteristics%2520account%2520for%2520tolerance%2520of%2520Trypanosoma%2520brucei%2520rhodesiense%2520to%2520D%252CL-alpha-difluoromethylornithine%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1997%26volume%3D41%26spage%3D1922%26epage%3D1925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><a href="https://www.dndi.org/diseases-projects/portfolio/scyx-7158/" class="extLink">https://www.dndi.org/diseases-projects/portfolio/scyx-7158/</a> (accessed Jun 13,<span class="NLM_x"> </span><span class="NLM_year">2017</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=https%3A%2F%2Fwww.dndi.org%2Fdiseases-projects%2Fportfolio%2Fscyx-7158%2F+%28accessed+Jun+13%2C+2017%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><a href="https://www.dndi.org/diseases-projects/portfolio/fexinidazole/" class="extLink">https://www.dndi.org/diseases-projects/portfolio/fexinidazole/</a> (accessed Jun 13,<span class="NLM_x"> </span><span class="NLM_year">2017</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=https%3A%2F%2Fwww.dndi.org%2Fdiseases-projects%2Fportfolio%2Ffexinidazole%2F+%28accessed+Jun+13%2C+2017%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Jacobs, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nare, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, M. A.</span><span> </span><span class="NLM_article-title">State of the art in African trypanosome drug discovery</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1255</span><span class="NLM_x">–</span> <span class="NLM_lpage">1274</span><span class="refDoi"> DOI: 10.2174/156802611795429167</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.2174%2F156802611795429167" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=21401507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnt1CnsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=1255-1274&author=R.+T.+Jacobsauthor=B.+Nareauthor=M.+A.+Phillips&title=State+of+the+art+in+African+trypanosome+drug+discovery&doi=10.2174%2F156802611795429167"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">State of the art in African trypanosome drug discovery</span></div><div class="casAuthors">Jacobs, Robert T.; Nare, Bakela; Phillips, Margaret A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1255-1274</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  African sleeping sickness is endemic in sub-Saharan Africa where the WHO ests. that 60 million people are at risk for the disease.  Human African trypanosomiasis (HAT) is 100% fatal if untreated and the current drug therapies have significant limitations due to toxicity and difficult treatment regimes.  No new chem. agents were approved since eflornithine in 1990.  The pentamidine analog DB289, which was in late stage clin. trials for the treatment of early stage HAT recently failed due to toxicity issues.  A new protocol for the treatment of late-stage T. brucei gambiense that uses combination nifurtomox/eflornithine (NECT) was recently shown to have better safety and efficacy than eflornithine alone, while being easier to administer.  This breakthrough represents the only new therapy for HAT since the approval of eflornithine.  A no. of research programs are on going to exploit the unusual biochem. pathways in the parasite to identify new targets for target based drug discovery programs.  HTS efforts are also underway to discover new chem. entities through whole organism screening approaches.  A no. of inhibitors with anti-trypanosomal activity have been identified by both approaches, but none of the programs are yet at the stage of identifying a preclin. candidate.  This dire situation underscores the need for continued effort to identify new chem. agents for the treatment of HAT.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhIrKnBIULOLVg90H21EOLACvtfcHk0liIMLyh3ZLirg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnt1CnsLc%253D&md5=15f997caa236c0537e20241dce8ccc5f</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.2174%2F156802611795429167&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802611795429167%26sid%3Dliteratum%253Aachs%26aulast%3DJacobs%26aufirst%3DR.%2BT.%26aulast%3DNare%26aufirst%3DB.%26aulast%3DPhillips%26aufirst%3DM.%2BA.%26atitle%3DState%2520of%2520the%2520art%2520in%2520African%2520trypanosome%2520drug%2520discovery%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2011%26volume%3D11%26spage%3D1255%26epage%3D1274%26doi%3D10.2174%2F156802611795429167" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Willert, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, M. A.</span><span> </span><span class="NLM_article-title">Regulation and function of polyamines in African trypanosomes</span> <span class="citation_source-journal">Trends Parasitol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">66</span><span class="NLM_x">–</span> <span class="NLM_lpage">72</span><span class="refDoi"> DOI: 10.1016/j.pt.2011.11.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1016%2Fj.pt.2011.11.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=22192816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVCjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2012&pages=66-72&author=E.+Willertauthor=M.+A.+Phillips&title=Regulation+and+function+of+polyamines+in+African+trypanosomes&doi=10.1016%2Fj.pt.2011.11.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation and function of polyamines in African trypanosomes</span></div><div class="casAuthors">Willert, Erin; Phillips, Margaret A.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Parasitology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">66-72</span>CODEN:
                <span class="NLM_cas:coden">TPRACT</span>;
        ISSN:<span class="NLM_cas:issn">1471-4922</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The polyamine biosynthetic pathway is an important drug target for the treatment of human African trypanosomiasis (HAT), raising interest in understanding polyamine function and their mechanism of regulation.  Polyamine levels are tightly controlled in mammalian cells, but similar regulatory mechanisms appear absent in trypanosomes.  Instead trypanosomatid S-adenosylmethionine decarboxylase (AdoMetDC), which catalyzes a key step in the biosynthesis of the polyamine spermidine, is activated by dimerization with an inducible protein termed prozyme.  Prozyme is an inactive paralog of the active AdoMetDC enzyme that evolved by gene duplication and is found only in the trypanosomatids.  In Trypanosoma brucei, AdoMetDC activity appears to be controlled by regulation of prozyme protein levels, potentially at the translational level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo26D7Br10oEbVg90H21EOLACvtfcHk0ljez5PwO4raUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVCjtbo%253D&md5=5535fe1c6340ac6c101261bc95bb1595</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.pt.2011.11.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pt.2011.11.001%26sid%3Dliteratum%253Aachs%26aulast%3DWillert%26aufirst%3DE.%26aulast%3DPhillips%26aufirst%3DM.%2BA.%26atitle%3DRegulation%2520and%2520function%2520of%2520polyamines%2520in%2520African%2520trypanosomes%26jtitle%3DTrends%2520Parasitol.%26date%3D2012%26volume%3D28%26spage%3D66%26epage%3D72%26doi%3D10.1016%2Fj.pt.2011.11.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Willert, E. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, M. A.</span><span> </span><span class="NLM_article-title">Regulated expression of an essential allosteric activator of polyamine biosynthesis in African trypanosomes</span> <span class="citation_source-journal">PLoS Pathog.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1000183</span><span class="refDoi"> DOI: 10.1371/journal.ppat.1000183</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1371%2Fjournal.ppat.1000183" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=e1000183&author=E.+K.+Willertauthor=M.+A.+Phillips&title=Regulated+expression+of+an+essential+allosteric+activator+of+polyamine+biosynthesis+in+African+trypanosomes&doi=10.1371%2Fjournal.ppat.1000183"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1000183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1000183%26sid%3Dliteratum%253Aachs%26aulast%3DWillert%26aufirst%3DE.%2BK.%26aulast%3DPhillips%26aufirst%3DM.%2BA.%26atitle%3DRegulated%2520expression%2520of%2520an%2520essential%2520allosteric%2520activator%2520of%2520polyamine%2520biosynthesis%2520in%2520African%2520trypanosomes%26jtitle%3DPLoS%2520Pathog.%26date%3D2008%26volume%3D4%26spage%3De1000183%26doi%3D10.1371%2Fjournal.ppat.1000183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Bacchi, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brun, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Croft, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alicea, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buhler, Y.</span><span> </span><span class="NLM_article-title">In vivo trypanocidal activities of new S-adenosylmethionine decarboxylase inhibitors</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">1448</span><span class="NLM_x">–</span> <span class="NLM_lpage">1453</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=1996&pages=1448-1453&author=C.+J.+Bacchiauthor=R.+Brunauthor=S.+L.+Croftauthor=K.+Aliceaauthor=Y.+Buhler&title=In+vivo+trypanocidal+activities+of+new+S-adenosylmethionine+decarboxylase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBacchi%26aufirst%3DC.%2BJ.%26aulast%3DBrun%26aufirst%3DR.%26aulast%3DCroft%26aufirst%3DS.%2BL.%26aulast%3DAlicea%26aufirst%3DK.%26aulast%3DBuhler%26aufirst%3DY.%26atitle%3DIn%2520vivo%2520trypanocidal%2520activities%2520of%2520new%2520S-adenosylmethionine%2520decarboxylase%2520inhibitors%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1996%26volume%3D40%26spage%3D1448%26epage%3D1453" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Bacchi, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barker, R. H.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodriguez, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirth, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rattendi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yarlett, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hendrick, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sybertz, E.</span><span> </span><span class="NLM_article-title">Trypanocidal activity of 8-methyl-5′-{[(Z)-4-aminobut-2-enyl]-(methylamino)}adenosine (Genz-644131), an adenosylmethionine decarboxylase inhibitor</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">3269</span><span class="NLM_x">–</span> <span class="NLM_lpage">3272</span><span class="refDoi"> DOI: 10.1128/AAC.00076-09</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1128%2FAAC.00076-09" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2009&pages=3269-3272&author=C.+J.+Bacchiauthor=R.+H.+Barkerauthor=A.+Rodriguezauthor=B.+Hirthauthor=D.+Rattendiauthor=N.+Yarlettauthor=C.+L.+Hendrickauthor=E.+Sybertz&title=Trypanocidal+activity+of+8-methyl-5%E2%80%B2-%7B%5B%28Z%29-4-aminobut-2-enyl%5D-%28methylamino%29%7Dadenosine+%28Genz-644131%29%2C+an+adenosylmethionine+decarboxylase+inhibitor&doi=10.1128%2FAAC.00076-09"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1128%2FAAC.00076-09&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.00076-09%26sid%3Dliteratum%253Aachs%26aulast%3DBacchi%26aufirst%3DC.%2BJ.%26aulast%3DBarker%26aufirst%3DR.%2BH.%26aulast%3DRodriguez%26aufirst%3DA.%26aulast%3DHirth%26aufirst%3DB.%26aulast%3DRattendi%26aufirst%3DD.%26aulast%3DYarlett%26aufirst%3DN.%26aulast%3DHendrick%26aufirst%3DC.%2BL.%26aulast%3DSybertz%26aufirst%3DE.%26atitle%3DTrypanocidal%2520activity%2520of%25208-methyl-5%25E2%2580%25B2-%257B%255B%2528Z%2529-4-aminobut-2-enyl%255D-%2528methylamino%2529%257Dadenosine%2520%2528Genz-644131%2529%252C%2520an%2520adenosylmethionine%2520decarboxylase%2520inhibitor%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2009%26volume%3D53%26spage%3D3269%26epage%3D3272%26doi%3D10.1128%2FAAC.00076-09" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Bitonti, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Byers, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bush, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casara, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bacchi, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarkson, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mccann, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sjoerdsma, A.</span><span> </span><span class="NLM_article-title">Cure of Trypanosoma brucei brucei and Trypanosoma brucei rhodesiense infections in mice with an irreversible inhibitor of S-adenosylmethionine decarboxylase</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">1485</span><span class="NLM_x">–</span> <span class="NLM_lpage">1490</span><span class="refDoi"> DOI: 10.1128/AAC.34.8.1485</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1128%2FAAC.34.8.1485" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1990&pages=1485-1490&author=A.+J.+Bitontiauthor=T.+L.+Byersauthor=T.+L.+Bushauthor=P.+J.+Casaraauthor=C.+J.+Bacchiauthor=A.+B.+Clarksonauthor=P.+P.+Mccannauthor=A.+Sjoerdsma&title=Cure+of+Trypanosoma+brucei+brucei+and+Trypanosoma+brucei+rhodesiense+infections+in+mice+with+an+irreversible+inhibitor+of+S-adenosylmethionine+decarboxylase&doi=10.1128%2FAAC.34.8.1485"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1128%2FAAC.34.8.1485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.34.8.1485%26sid%3Dliteratum%253Aachs%26aulast%3DBitonti%26aufirst%3DA.%2BJ.%26aulast%3DByers%26aufirst%3DT.%2BL.%26aulast%3DBush%26aufirst%3DT.%2BL.%26aulast%3DCasara%26aufirst%3DP.%2BJ.%26aulast%3DBacchi%26aufirst%3DC.%2BJ.%26aulast%3DClarkson%26aufirst%3DA.%2BB.%26aulast%3DMccann%26aufirst%3DP.%2BP.%26aulast%3DSjoerdsma%26aufirst%3DA.%26atitle%3DCure%2520of%2520Trypanosoma%2520brucei%2520brucei%2520and%2520Trypanosoma%2520brucei%2520rhodesiense%2520infections%2520in%2520mice%2520with%2520an%2520irreversible%2520inhibitor%2520of%2520S-adenosylmethionine%2520decarboxylase%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1990%26volume%3D34%26spage%3D1485%26epage%3D1490%26doi%3D10.1128%2FAAC.34.8.1485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Willert, E. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fitzpatrick, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, M. A.</span><span> </span><span class="NLM_article-title">Allosteric regulation of an essential trypanosome polyamine biosynthetic enzyme by a catalytically dead homolog</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x">, </span> <span class="NLM_fpage">8275</span><span class="NLM_x">–</span> <span class="NLM_lpage">8280</span><span class="refDoi"> DOI: 10.1073/pnas.0701111104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1073%2Fpnas.0701111104" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=8275-8280&author=E.+K.+Willertauthor=R.+Fitzpatrickauthor=M.+A.+Phillips&title=Allosteric+regulation+of+an+essential+trypanosome+polyamine+biosynthetic+enzyme+by+a+catalytically+dead+homolog&doi=10.1073%2Fpnas.0701111104"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0701111104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0701111104%26sid%3Dliteratum%253Aachs%26aulast%3DWillert%26aufirst%3DE.%2BK.%26aulast%3DFitzpatrick%26aufirst%3DR.%26aulast%3DPhillips%26aufirst%3DM.%2BA.%26atitle%3DAllosteric%2520regulation%2520of%2520an%2520essential%2520trypanosome%2520polyamine%2520biosynthetic%2520enzyme%2520by%2520a%2520catalytically%2520dead%2520homolog%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D8275%26epage%3D8280%26doi%3D10.1073%2Fpnas.0701111104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Volkov, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kinch, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ariagno, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grishin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomchick, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, M. A.</span><span> </span><span class="NLM_article-title">Relief of autoinhibition by conformational switch explains enzyme activation by a catalytically dead paralog</span> <span class="citation_source-journal">eLife</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">e20198</span><span class="refDoi"> DOI: 10.7554/eLife.20198</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.7554%2FeLife.20198" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&pages=e20198&author=O.+A.+Volkovauthor=L.+Kinchauthor=C.+Ariagnoauthor=X.+Dengauthor=S.+Zhongauthor=N.+Grishinauthor=D.+R.+Tomchickauthor=Z.+Chenauthor=M.+A.+Phillips&title=Relief+of+autoinhibition+by+conformational+switch+explains+enzyme+activation+by+a+catalytically+dead+paralog&doi=10.7554%2FeLife.20198"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.7554%2FeLife.20198&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7554%252FeLife.20198%26sid%3Dliteratum%253Aachs%26aulast%3DVolkov%26aufirst%3DO.%2BA.%26aulast%3DKinch%26aufirst%3DL.%26aulast%3DAriagno%26aufirst%3DC.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DZhong%26aufirst%3DS.%26aulast%3DGrishin%26aufirst%3DN.%26aulast%3DTomchick%26aufirst%3DD.%2BR.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DPhillips%26aufirst%3DM.%2BA.%26atitle%3DRelief%2520of%2520autoinhibition%2520by%2520conformational%2520switch%2520explains%2520enzyme%2520activation%2520by%2520a%2520catalytically%2520dead%2520paralog%26jtitle%3DeLife%26date%3D2016%26volume%3D5%26spage%3De20198%26doi%3D10.7554%2FeLife.20198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Volkov, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosner, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brockway, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kramer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Booker, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ketcherside, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Longgood, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mccoy, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wring, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klinger, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Posner, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Brabander, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, M. A.</span><span> </span><span class="NLM_article-title">Identification of Trypanosoma brucei AdoMetDC inhibitors using a high-throughput mass spectrometry-based assay</span> <span class="citation_source-journal">ACS Infect. Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">512</span><span class="NLM_x">–</span> <span class="NLM_lpage">526</span><span class="refDoi"> DOI: 10.1021/acsinfecdis.7b00022</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.7b00022" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BC2sXltVSqtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2017&pages=512-526&author=O.+A.+Volkovauthor=C.+C.+Cosnerauthor=A.+J.+Brockwayauthor=M.+Kramerauthor=M.+Bookerauthor=S.+Zhongauthor=A.+Ketchersideauthor=S.+Weiauthor=J.+Longgoodauthor=M.+Mccoyauthor=T.+E.+Richardsonauthor=S.+A.+Wringauthor=M.+Peelauthor=J.+D.+Klingerauthor=B.+A.+Posnerauthor=J.+K.+De+Brabanderauthor=M.+A.+Phillips&title=Identification+of+Trypanosoma+brucei+AdoMetDC+inhibitors+using+a+high-throughput+mass+spectrometry-based+assay&doi=10.1021%2Facsinfecdis.7b00022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Trypanosoma brucei AdoMetDC Inhibitors Using a High-Throughput Mass Spectrometry-Based Assay</span></div><div class="casAuthors">Volkov, Oleg A.; Cosner, Casey C.; Brockway, Anthony J.; Kramer, Martin; Booker, Michael; Zhong, Shihua; Ketcherside, Ariel; Wei, Shuguang; Longgood, Jamie; McCoy, Melissa; Richardson, Thomas E.; Wring, Stephen A.; Peel, Michael; Klinger, Jeffrey D.; Posner, Bruce A.; De Brabander, Jef K.; Phillips, Margaret A.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">512-526</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human African trypanosomiasis (HAT) is a fatal infectious disease caused by the eukaryotic pathogen Trypanosoma brucei (Tb).  Available treatments are difficult to administer and have significant safety issues.  S-Adenosylmethionine decarboxylase (AdoMetDC) is an essential enzyme in the parasite polyamine biosynthetic pathway.  Previous attempts to develop TbAdoMetDC inhibitors into anti-HAT therapies failed due to poor brain exposure.  Here, we describe a large screening campaign of two small-mol. libraries (∼400,000 compds.) employing a new high-throughput (∼7 s per sample) mass spectrometry-based assay for AdoMetDC activity.  As a result of primary screening, followed by hit confirmation and validation, we identified 13 new classes of reversible TbAdoMetDC inhibitors with low-micromolar potency (IC50) against both TbAdoMetDC and T. brucei parasite cells.  The majority of these compds. were >10-fold selective against the human enzyme.  Importantly, compds. from four classes demonstrated high propensity to cross the blood-brain barrier in a cell monolayer assay.  Biochem. anal. demonstrated that compds. from eight classes inhibited intracellular TbAdoMetDC in the parasite, although evidence for a secondary off-target component was also present.  The discovery of several new TbAdoMetDC inhibitor chemotypes provides new hits for lead optimization programs aimed to deliver a novel treatment for HAT.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1WsiJeQdpIrVg90H21EOLACvtfcHk0lhpuy_OIgGapg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXltVSqtbs%253D&md5=99fbff6e441a8e43959700dc66a3c360</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.7b00022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.7b00022%26sid%3Dliteratum%253Aachs%26aulast%3DVolkov%26aufirst%3DO.%2BA.%26aulast%3DCosner%26aufirst%3DC.%2BC.%26aulast%3DBrockway%26aufirst%3DA.%2BJ.%26aulast%3DKramer%26aufirst%3DM.%26aulast%3DBooker%26aufirst%3DM.%26aulast%3DZhong%26aufirst%3DS.%26aulast%3DKetcherside%26aufirst%3DA.%26aulast%3DWei%26aufirst%3DS.%26aulast%3DLonggood%26aufirst%3DJ.%26aulast%3DMccoy%26aufirst%3DM.%26aulast%3DRichardson%26aufirst%3DT.%2BE.%26aulast%3DWring%26aufirst%3DS.%2BA.%26aulast%3DPeel%26aufirst%3DM.%26aulast%3DKlinger%26aufirst%3DJ.%2BD.%26aulast%3DPosner%26aufirst%3DB.%2BA.%26aulast%3DDe%2BBrabander%26aufirst%3DJ.%2BK.%26aulast%3DPhillips%26aufirst%3DM.%2BA.%26atitle%3DIdentification%2520of%2520Trypanosoma%2520brucei%2520AdoMetDC%2520inhibitors%2520using%2520a%2520high-throughput%2520mass%2520spectrometry-based%2520assay%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2017%26volume%3D3%26spage%3D512%26epage%3D526%26doi%3D10.1021%2Facsinfecdis.7b00022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schnekenburger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwergel, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosconati, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gros, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tardugno, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labella, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Florean, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minden, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hashimoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirsch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Novellino, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arimondo, P. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miele, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferretti, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gulino, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diederich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span> </span><span class="NLM_article-title">Selective non-nucleoside inhibitors of human DNA methyltransferases active in cancer including in cancer stem cells</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">701</span><span class="NLM_x">–</span> <span class="NLM_lpage">713</span><span class="refDoi"> DOI: 10.1021/jm4012627</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4012627" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisFOisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=701-713&author=S.+Valenteauthor=Y.+Liuauthor=M.+Schnekenburgerauthor=C.+Zwergelauthor=S.+Cosconatiauthor=C.+Grosauthor=M.+Tardugnoauthor=D.+Labellaauthor=C.+Floreanauthor=S.+Mindenauthor=H.+Hashimotoauthor=Y.+Changauthor=X.+Zhangauthor=G.+Kirschauthor=E.+Novellinoauthor=P.+B.+Arimondoauthor=E.+Mieleauthor=E.+Ferrettiauthor=A.+Gulinoauthor=M.+Diederichauthor=X.+Chengauthor=A.+Mai&title=Selective+non-nucleoside+inhibitors+of+human+DNA+methyltransferases+active+in+cancer+including+in+cancer+stem+cells&doi=10.1021%2Fjm4012627"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Non-nucleoside Inhibitors of Human DNA Methyltransferases Active in Cancer Including in Cancer Stem Cells</span></div><div class="casAuthors">Valente, Sergio; Yiwei, Liu; Schnekenburger, Michael; Zwergel, Clemens; Cosconati, Sandro; Gros, Christina; Tardugno, Maria; Labella, Donatella; Florean, Cristina; Minden, Steven; Hashimoto, Hideharu; Chang, Yanqi; Zhang, Xing; Kirsch, Gilbert; Novellino, Ettore; Arimondo, Paola B.; Miele, Evelina; Ferretti, Elisabetta; Gulino, Alberto; Diederich, Marc; Cheng, Xiaodong; Mai, Antonello</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">701-713</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DNA methyltransferases (DNMTs) are important enzymes involved in epigenetic control of gene expression and represent valuable targets in cancer chemotherapy.  A no. of nucleoside DNMT inhibitors (DNMTi) have been studied in cancer, including in cancer stem cells, and two of them (azacytidine and decitabine) have been approved for treatment of myelodysplastic syndromes.  However, only a few non-nucleoside DNMTi have been identified so far, and even fewer have been validated in cancer.  Through a process of hit-to-lead optimization, we report here the discovery of compd. 5 as a potent non-nucleoside DNMTi that is also selective toward other AdoMet-dependent protein methyltransferases.  Compd. 5 was potent at single-digit micromolar concns. against a panel of cancer cells and was less toxic in peripheral blood mononuclear cells than two other compds. tested.  In mouse medulloblastoma stem cells, 5 inhibited cell growth, whereas related compd. 2 showed high cell differentiation.  To the best of our knowledge, 2 and 5 are the first non-nucleoside DNMTi tested in a cancer stem cell line.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxOcRR02xFkLVg90H21EOLACvtfcHk0lgxZFRRAjFnbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisFOisw%253D%253D&md5=0749e33ad2324aa591342d7c7f23a955</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fjm4012627&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4012627%26sid%3Dliteratum%253Aachs%26aulast%3DValente%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DSchnekenburger%26aufirst%3DM.%26aulast%3DZwergel%26aufirst%3DC.%26aulast%3DCosconati%26aufirst%3DS.%26aulast%3DGros%26aufirst%3DC.%26aulast%3DTardugno%26aufirst%3DM.%26aulast%3DLabella%26aufirst%3DD.%26aulast%3DFlorean%26aufirst%3DC.%26aulast%3DMinden%26aufirst%3DS.%26aulast%3DHashimoto%26aufirst%3DH.%26aulast%3DChang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DKirsch%26aufirst%3DG.%26aulast%3DNovellino%26aufirst%3DE.%26aulast%3DArimondo%26aufirst%3DP.%2BB.%26aulast%3DMiele%26aufirst%3DE.%26aulast%3DFerretti%26aufirst%3DE.%26aulast%3DGulino%26aufirst%3DA.%26aulast%3DDiederich%26aufirst%3DM.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DMai%26aufirst%3DA.%26atitle%3DSelective%2520non-nucleoside%2520inhibitors%2520of%2520human%2520DNA%2520methyltransferases%2520active%2520in%2520cancer%2520including%2520in%2520cancer%2520stem%2520cells%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D701%26epage%3D713%26doi%3D10.1021%2Fjm4012627" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Borman, R. A.; Coleman, R. A.; Clark, K. L.; Oxford, A. W.; Hynd, G.; Archer, J. A.; Aley, A.; Harris, N. V.</span> 5-HT2B Receptor Antagonists. Patent WO 2005/012263 A1,<span class="NLM_x"> </span><span class="NLM_year">2005</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Borman%2C+R.+A.%3B+Coleman%2C+R.+A.%3B+Clark%2C+K.+L.%3B+Oxford%2C+A.+W.%3B+Hynd%2C+G.%3B+Archer%2C+J.+A.%3B+Aley%2C+A.%3B+Harris%2C+N.+V.+5-HT2B+Receptor+Antagonists.+Patent+WO+2005%2F012263+A1%2C+2005."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBorman%26aufirst%3DR.%2BA.%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Sander, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kottke, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanrikulu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Proschak, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weizel, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seifert, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stark, H.</span><span> </span><span class="NLM_article-title">2,4-Diaminopyrimidines as histamine H4 receptor ligands–Scaffold optimization and pharmacological characterization</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">7186</span><span class="NLM_x">–</span> <span class="NLM_lpage">7196</span><span class="refDoi"> DOI: 10.1016/j.bmc.2009.08.059</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1016%2Fj.bmc.2009.08.059" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2009&pages=7186-7196&author=K.+Sanderauthor=T.+Kottkeauthor=Y.+Tanrikuluauthor=E.+Proschakauthor=L.+Weizelauthor=E.+H.+Schneiderauthor=R.+Seifertauthor=G.+Schneiderauthor=H.+Stark&title=2%2C4-Diaminopyrimidines+as+histamine+H4+receptor+ligands%E2%80%93Scaffold+optimization+and+pharmacological+characterization&doi=10.1016%2Fj.bmc.2009.08.059"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2009.08.059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2009.08.059%26sid%3Dliteratum%253Aachs%26aulast%3DSander%26aufirst%3DK.%26aulast%3DKottke%26aufirst%3DT.%26aulast%3DTanrikulu%26aufirst%3DY.%26aulast%3DProschak%26aufirst%3DE.%26aulast%3DWeizel%26aufirst%3DL.%26aulast%3DSchneider%26aufirst%3DE.%2BH.%26aulast%3DSeifert%26aufirst%3DR.%26aulast%3DSchneider%26aufirst%3DG.%26aulast%3DStark%26aufirst%3DH.%26atitle%3D2%252C4-Diaminopyrimidines%2520as%2520histamine%2520H4%2520receptor%2520ligands%25E2%2580%2593Scaffold%2520optimization%2520and%2520pharmacological%2520characterization%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2009%26volume%3D17%26spage%3D7186%26epage%3D7196%26doi%3D10.1016%2Fj.bmc.2009.08.059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Sekiya, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takashima, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ueda, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamiya, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuasa, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujimura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ubasawa, M.</span><span> </span><span class="NLM_article-title">2-Amino-6-arylthio-9-[2-(phosphonomethoxy)ethyl]purine bis(2,2,2-trifluoroethyl) esters as novel HBV-specific antiviral reagents</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">3138</span><span class="NLM_x">–</span> <span class="NLM_lpage">3142</span><span class="refDoi"> DOI: 10.1021/jm020036x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm020036x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=3138-3142&author=K.+Sekiyaauthor=H.+Takashimaauthor=N.+Uedaauthor=N.+Kamiyaauthor=S.+Yuasaauthor=Y.+Fujimuraauthor=M.+Ubasawa&title=2-Amino-6-arylthio-9-%5B2-%28phosphonomethoxy%29ethyl%5Dpurine+bis%282%2C2%2C2-trifluoroethyl%29+esters+as+novel+HBV-specific+antiviral+reagents&doi=10.1021%2Fjm020036x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm020036x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020036x%26sid%3Dliteratum%253Aachs%26aulast%3DSekiya%26aufirst%3DK.%26aulast%3DTakashima%26aufirst%3DH.%26aulast%3DUeda%26aufirst%3DN.%26aulast%3DKamiya%26aufirst%3DN.%26aulast%3DYuasa%26aufirst%3DS.%26aulast%3DFujimura%26aufirst%3DY.%26aulast%3DUbasawa%26aufirst%3DM.%26atitle%3D2-Amino-6-arylthio-9-%255B2-%2528phosphonomethoxy%2529ethyl%255Dpurine%2520bis%25282%252C2%252C2-trifluoroethyl%2529%2520esters%2520as%2520novel%2520HBV-specific%2520antiviral%2520reagents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D3138%26epage%3D3142%26doi%3D10.1021%2Fjm020036x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Medina, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Becker, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blackledge, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duquenne, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heerding, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romeril, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scherzer, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bobko, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chadderton, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dumble, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardiner, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabindran, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sudakin, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brady, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campobasso, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Axten, J. M.</span><span> </span><span class="NLM_article-title">Structure-based design of potent and selective 3-phosphoinositide-dependent kinase-1 (PDK1) inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1871</span><span class="NLM_x">–</span> <span class="NLM_lpage">1895</span><span class="refDoi"> DOI: 10.1021/jm101527u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm101527u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisVWksrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1871-1895&author=J.+R.+Medinaauthor=C.+J.+Beckerauthor=C.+W.+Blackledgeauthor=C.+Duquenneauthor=Y.+Fengauthor=S.+W.+Grantauthor=D.+Heerdingauthor=W.+H.+Liauthor=W.+H.+Millerauthor=S.+P.+Romerilauthor=D.+Scherzerauthor=A.+Shuauthor=M.+A.+Bobkoauthor=A.+R.+Chaddertonauthor=M.+Dumbleauthor=C.+M.+Gardinerauthor=S.+Gilbertauthor=Q.+Liuauthor=S.+K.+Rabindranauthor=V.+Sudakinauthor=H.+Xiangauthor=P.+G.+Bradyauthor=N.+Campobassoauthor=P.+Wardauthor=J.+M.+Axten&title=Structure-based+design+of+potent+and+selective+3-phosphoinositide-dependent+kinase-1+%28PDK1%29+inhibitors&doi=10.1021%2Fjm101527u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of Potent and Selective 3-Phosphoinositide-Dependent Kinase-1 (PDK1) Inhibitors</span></div><div class="casAuthors">Medina, Jesus R.; Becker, Christopher J.; Blackledge, Charles W.; Duquenne, Celine; Feng, Yanhong; Grant, Seth W.; Heerding, Dirk; Li, William H.; Miller, William H.; Romeril, Stuart P.; Scherzer, Daryl; Shu, Arthur; Bobko, Mark A.; Chadderton, Antony R.; Dumble, Melissa; Gardiner, Christine M.; Gilbert, Seth; Liu, Qi; Rabindran, Sridhar K.; Sudakin, Valery; Xiang, Hong; Brady, Pat G.; Campobasso, Nino; Ward, Paris; Axten, Jeffrey M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1871-1895</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Phosphoinositide-dependent protein kinase-1 (PDK1) is a master regulator of the AGC family of kinases and an integral component of the PI3K/AKT/mTOR pathway.  As this pathway is among the most commonly deregulated across all cancers, a selective inhibitor of PDK1 might have utility as an anticancer agent.  Herein, the lead optimization of compd. I (R1 = R2 = H) toward highly potent and selective PDK1 inhibitors via a structure-based design strategy is described.  A series of (amino)(aminopyrimidinyl)indazoles I (R1 = H, Me, Et; R2 = Me, i-Pr, PhNH, 1-piperidinyl, etc.) was synthesized, and the inhibiting activity of the products towards PDK1 was studied.  The most potent and selective inhibitors, e.g. I [R1 = Me; R2 = (3S,6R)-3-(cyclohexylaminocarbonyl)-6-methyl-1-piperidinyl], demonstrated submicromolar activity as measured by inhibition of phosphorylation of PDK1 substrates as well as antiproliferative activity against a subset of AML cell lines.  In addn., redn. of phosphorylation of PDK1 substrates was demonstrated in vivo in mice bearing OCl-AML2 xenografts.  These observations demonstrate the utility of these mols. as tools to further delineate the biol. of PDK1 and the potential pharmacol. uses of a PDK1 inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrd83HnbHh5j7Vg90H21EOLACvtfcHk0lhtxcZGobm2Lw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisVWksrk%253D&md5=2911a5b7dbfa944a7c523da0be1026ac</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm101527u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm101527u%26sid%3Dliteratum%253Aachs%26aulast%3DMedina%26aufirst%3DJ.%2BR.%26aulast%3DBecker%26aufirst%3DC.%2BJ.%26aulast%3DBlackledge%26aufirst%3DC.%2BW.%26aulast%3DDuquenne%26aufirst%3DC.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DGrant%26aufirst%3DS.%2BW.%26aulast%3DHeerding%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DW.%2BH.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26aulast%3DRomeril%26aufirst%3DS.%2BP.%26aulast%3DScherzer%26aufirst%3DD.%26aulast%3DShu%26aufirst%3DA.%26aulast%3DBobko%26aufirst%3DM.%2BA.%26aulast%3DChadderton%26aufirst%3DA.%2BR.%26aulast%3DDumble%26aufirst%3DM.%26aulast%3DGardiner%26aufirst%3DC.%2BM.%26aulast%3DGilbert%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DRabindran%26aufirst%3DS.%2BK.%26aulast%3DSudakin%26aufirst%3DV.%26aulast%3DXiang%26aufirst%3DH.%26aulast%3DBrady%26aufirst%3DP.%2BG.%26aulast%3DCampobasso%26aufirst%3DN.%26aulast%3DWard%26aufirst%3DP.%26aulast%3DAxten%26aufirst%3DJ.%2BM.%26atitle%3DStructure-based%2520design%2520of%2520potent%2520and%2520selective%25203-phosphoinositide-dependent%2520kinase-1%2520%2528PDK1%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1871%26epage%3D1895%26doi%3D10.1021%2Fjm101527u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Yi, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Long, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez-Cestari, T. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henderson, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pavlovicz, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Werbovetz, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mckay, D. B.</span><span> </span><span class="NLM_article-title">Discovery of benzamide analogs as negative allosteric modulators of human neuronal nicotinic receptors: pharmacophore modeling and structure-activity relationship studies</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">4730</span><span class="NLM_x">–</span> <span class="NLM_lpage">4743</span><span class="refDoi"> DOI: 10.1016/j.bmc.2013.03.082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1016%2Fj.bmc.2013.03.082" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=4730-4743&author=B.+Yiauthor=S.+Longauthor=T.+F.+Gonzalez-Cestariauthor=B.+J.+Hendersonauthor=R.+E.+Pavloviczauthor=K.+Werbovetzauthor=C.+Liauthor=D.+B.+Mckay&title=Discovery+of+benzamide+analogs+as+negative+allosteric+modulators+of+human+neuronal+nicotinic+receptors%3A+pharmacophore+modeling+and+structure-activity+relationship+studies&doi=10.1016%2Fj.bmc.2013.03.082"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.03.082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.03.082%26sid%3Dliteratum%253Aachs%26aulast%3DYi%26aufirst%3DB.%26aulast%3DLong%26aufirst%3DS.%26aulast%3DGonzalez-Cestari%26aufirst%3DT.%2BF.%26aulast%3DHenderson%26aufirst%3DB.%2BJ.%26aulast%3DPavlovicz%26aufirst%3DR.%2BE.%26aulast%3DWerbovetz%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DMckay%26aufirst%3DD.%2BB.%26atitle%3DDiscovery%2520of%2520benzamide%2520analogs%2520as%2520negative%2520allosteric%2520modulators%2520of%2520human%2520neuronal%2520nicotinic%2520receptors%253A%2520pharmacophore%2520modeling%2520and%2520structure-activity%2520relationship%2520studies%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D4730%26epage%3D4743%26doi%3D10.1016%2Fj.bmc.2013.03.082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Wijtmans, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scholten, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roumen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Canals, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Custers, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glas, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vreeker, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Kanter, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Graaf, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smit, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Esch, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leurs, R.</span><span> </span><span class="NLM_article-title">Chemical subtleties in small-molecule modulation of peptide receptor function: the case of CXCR3 biaryl-type ligands</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">10572</span><span class="NLM_x">–</span> <span class="NLM_lpage">10583</span><span class="refDoi"> DOI: 10.1021/jm301240t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301240t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=10572-10583&author=M.+Wijtmansauthor=D.+J.+Scholtenauthor=L.+Roumenauthor=M.+Canalsauthor=H.+Custersauthor=M.+Glasauthor=M.+C.+Vreekerauthor=F.+J.+De+Kanterauthor=C.+De+Graafauthor=M.+J.+Smitauthor=I.+J.+De+Eschauthor=R.+Leurs&title=Chemical+subtleties+in+small-molecule+modulation+of+peptide+receptor+function%3A+the+case+of+CXCR3+biaryl-type+ligands&doi=10.1021%2Fjm301240t"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm301240t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301240t%26sid%3Dliteratum%253Aachs%26aulast%3DWijtmans%26aufirst%3DM.%26aulast%3DScholten%26aufirst%3DD.%2BJ.%26aulast%3DRoumen%26aufirst%3DL.%26aulast%3DCanals%26aufirst%3DM.%26aulast%3DCusters%26aufirst%3DH.%26aulast%3DGlas%26aufirst%3DM.%26aulast%3DVreeker%26aufirst%3DM.%2BC.%26aulast%3DDe%2BKanter%26aufirst%3DF.%2BJ.%26aulast%3DDe%2BGraaf%26aufirst%3DC.%26aulast%3DSmit%26aufirst%3DM.%2BJ.%26aulast%3DDe%2BEsch%26aufirst%3DI.%2BJ.%26aulast%3DLeurs%26aufirst%3DR.%26atitle%3DChemical%2520subtleties%2520in%2520small-molecule%2520modulation%2520of%2520peptide%2520receptor%2520function%253A%2520the%2520case%2520of%2520CXCR3%2520biaryl-type%2520ligands%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D10572%26epage%3D10583%26doi%3D10.1021%2Fjm301240t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Knapp, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gillis, E. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, M. D.</span><span> </span><span class="NLM_article-title">A general solution for unstable boronic acids: slow-release cross-coupling from air-stable MIDA boronates</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">131</span><span class="NLM_x">, </span> <span class="NLM_fpage">6961</span><span class="NLM_x">–</span> <span class="NLM_lpage">6963</span><span class="refDoi"> DOI: 10.1021/ja901416p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja901416p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlt1Cls7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=6961-6963&author=D.+M.+Knappauthor=E.+P.+Gillisauthor=M.+D.+Burke&title=A+general+solution+for+unstable+boronic+acids%3A+slow-release+cross-coupling+from+air-stable+MIDA+boronates&doi=10.1021%2Fja901416p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A General Solution for Unstable Boronic Acids: Slow-Release Cross-Coupling from Air-Stable MIDA Boronates</span></div><div class="casAuthors">Knapp, David M.; Gillis, Eric P.; Burke, Martin D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6961-6963</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Many boronic acids, including 2-heterocyclic, vinyl, and cyclopropyl derivs., are inherently unstable, which can limit their bench-top storage and/or efficient cross-coupling.  The authors herein report the 1st general soln. to this problem: in situ slow release of unstable boronic acids from the corresponding air-stable N-methyliminodiacetic acid (MIDA) boronates.  This remarkably general approach has transformed all three classes of these unstable boronic acids into shelf-stable and highly effective building blocks for cross-coupling with a wide range of aryl and heteroaryl chlorides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnIQpAbX2N_7Vg90H21EOLACvtfcHk0ljroHD3E7T6tQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlt1Cls7s%253D&md5=0865bc71ca79319e2116c23932c7ed55</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fja901416p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja901416p%26sid%3Dliteratum%253Aachs%26aulast%3DKnapp%26aufirst%3DD.%2BM.%26aulast%3DGillis%26aufirst%3DE.%2BP.%26aulast%3DBurke%26aufirst%3DM.%2BD.%26atitle%3DA%2520general%2520solution%2520for%2520unstable%2520boronic%2520acids%253A%2520slow-release%2520cross-coupling%2520from%2520air-stable%2520MIDA%2520boronates%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D6961%26epage%3D6963%26doi%3D10.1021%2Fja901416p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Baell, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holloway, G. A.</span><span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2719</span><span class="NLM_x">–</span> <span class="NLM_lpage">2740</span><span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lgAkR6JkuGUrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Mahar Doan, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humphreys, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webster, L. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wring, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shampine, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serabjit-Singh, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adkison, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polli, J. W.</span><span> </span><span class="NLM_article-title">Passive permeability and P-glycoprotein-mediated efflux differentiate central nervous system (CNS) and non-CNS marketed drugs</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">303</span><span class="NLM_x">, </span> <span class="NLM_fpage">1029</span><span class="NLM_x">–</span> <span class="NLM_lpage">1037</span><span class="refDoi"> DOI: 10.1124/jpet.102.039255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1124%2Fjpet.102.039255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=12438524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A280%3ADC%252BD38nptVKmsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=303&publication_year=2002&pages=1029-1037&author=K.+M.+Mahar+Doanauthor=J.+E.+Humphreysauthor=L.+O.+Websterauthor=S.+A.+Wringauthor=L.+J.+Shampineauthor=C.+J.+Serabjit-Singhauthor=K.+K.+Adkisonauthor=J.+W.+Polli&title=Passive+permeability+and+P-glycoprotein-mediated+efflux+differentiate+central+nervous+system+%28CNS%29+and+non-CNS+marketed+drugs&doi=10.1124%2Fjpet.102.039255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Passive permeability and P-glycoprotein-mediated efflux differentiate central nervous system (CNS) and non-CNS marketed drugs</span></div><div class="casAuthors">Mahar Doan Kelly M; Humphreys Joan E; Webster Lindsey O; Wring Stephen A; Shampine Larry J; Serabjit-Singh Cosette J; Adkison Kimberly K; Polli Joseph W</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of pharmacology and experimental therapeutics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">303</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1029-37</span>
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    </div><div class="casAbstract">Membrane permeability and P-glycoprotein (Pgp) can be limiting factors for blood-brain barrier penetration.  The objectives of this study were to determine whether there are differences in the in vitro permeability, Pgp substrate profiles, and physicochemical properties of drugs for central nervous system (CNS) and non-CNS indications, and whether these differences are useful criteria in selecting compounds for drug development.  Apparent permeability (P(app)) and Pgp substrate profiles for 93 CNS (n = 48) and non-CNS (n = 45) drugs were determined by monolayer efflux.  Calcein-AM inhibition assays were used to supplement the efflux results.  The CNS set (2 of 48, 4.2%) had a 7-fold lower incidence of passive permeability values <150 nm/s compared with the non-CNS set (13 of 45, 28.9%).  The majority of drugs (72.0%, 67 of 93) were not Pgp substrates; however, 49.5% (46 of 93) were positive in the calcein-AM assay when tested at 100 microM.  The CNS drug set (n = 7 of 48, 14.6%) had a 3-fold lower incidence of Pgp-mediated efflux than the non-CNS drug set (n = 19 of 45, 42.2%).  Analysis of 18 physicochemical properties revealed that the CNS drug set had fewer hydrogen bond donors, fewer positive charges, greater lipophilicity, lower polar surface area, and reduced flexibility compared with the non-CNS group (p < 0.05), properties that enhance membrane permeability.  This study on a large, diverse set of marketed compounds clearly demonstrates that permeability, Pgp-mediated efflux, and certain physicochemical properties are factors that differentiate CNS and non-CNS drugs.  For CNS delivery, a drug should ideally have an in vitro passive permeability >150 nm/s and not be a good (B --> A/A --> B ratio <2.5) Pgp substrate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT_HiV1Xqt18Vf1XRtfYM0dfW6udTcc2eZnd6mYCwCGnLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD38nptVKmsw%253D%253D&md5=b841902aa8502de47c1e1603c80e09fb</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1124%2Fjpet.102.039255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.102.039255%26sid%3Dliteratum%253Aachs%26aulast%3DMahar%2BDoan%26aufirst%3DK.%2BM.%26aulast%3DHumphreys%26aufirst%3DJ.%2BE.%26aulast%3DWebster%26aufirst%3DL.%2BO.%26aulast%3DWring%26aufirst%3DS.%2BA.%26aulast%3DShampine%26aufirst%3DL.%2BJ.%26aulast%3DSerabjit-Singh%26aufirst%3DC.%2BJ.%26aulast%3DAdkison%26aufirst%3DK.%2BK.%26aulast%3DPolli%26aufirst%3DJ.%2BW.%26atitle%3DPassive%2520permeability%2520and%2520P-glycoprotein-mediated%2520efflux%2520differentiate%2520central%2520nervous%2520system%2520%2528CNS%2529%2520and%2520non-CNS%2520marketed%2520drugs%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2002%26volume%3D303%26spage%3D1029%26epage%3D1037%26doi%3D10.1124%2Fjpet.102.039255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Copeland, R. A.</span> <span class="citation_source-book">Evaluation of Enzyme Inhibitors in Drug Discovery: A Guide for Medicinal Chemists and Pharmacologists</span>; <span class="NLM_publisher-name">Wiley-Interscience</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>,<span class="NLM_x"> </span><span class="NLM_year">2005</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&author=R.+A.+Copeland&title=Evaluation+of+Enzyme+Inhibitors+in+Drug+Discovery%3A+A+Guide+for+Medicinal+Chemists+and+Pharmacologists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26btitle%3DEvaluation%2520of%2520Enzyme%2520Inhibitors%2520in%2520Drug%2520Discovery%253A%2520A%2520Guide%2520for%2520Medicinal%2520Chemists%2520and%2520Pharmacologists%26pub%3DWiley-Interscience%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skolnick, J.</span><span> </span><span class="NLM_article-title">TM-align: a protein structure alignment algorithm based on the TM-score</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">2302</span><span class="NLM_x">–</span> <span class="NLM_lpage">2309</span><span class="refDoi"> DOI: 10.1093/nar/gki524</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1093%2Fnar%2Fgki524" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=15849316" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjsl2gsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=2302-2309&author=Y.+Zhangauthor=J.+Skolnick&title=TM-align%3A+a+protein+structure+alignment+algorithm+based+on+the+TM-score&doi=10.1093%2Fnar%2Fgki524"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">TM-align: a protein structure alignment algorithm based on the TM-score</span></div><div class="casAuthors">Zhang, Yang; Skolnick, Jeffrey</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2302-2309</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">We have developed TM-align, a new algorithm to identify the best structural alignment between protein pairs that combines the TM-score rotation matrix and Dynamic Programming (DP).  The algorithm is ∼4 times faster than CE and 20 times faster than DALI and SAL.  On av., the resulting structure alignments have higher accuracy and coverage than those provided by these most often-used methods.  TM-align is applied to an all-against-all structure comparison of 10 515 representative protein chains from the Protein Data Bank (PDB) with a sequence identity cutoff <95%: 1996 distinct folds are found when a TM-score threshold of 0.5 is used.  We also use TM-align to match the models predicted by TASSER for solved non-homologous proteins in PDB.  For both folded and misfolded models, TM-align can almost always find close structural analogs, with an av. root mean square deviation, RMSD, of 3 Å and 87% alignment coverage.  Nevertheless, there exists a significant correlation between the correctness of the predicted structure and the structural similarity of the model to the other proteins in the PDB.  This correlation could be used to assist in model selection in blind protein structure predictions.  The TM-align program is freely downloadable at http://bioinformatics.buffalo.edu/TM-align.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonPrM9Mg2FZ7Vg90H21EOLACvtfcHk0ljL01tr7EHspA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjsl2gsLY%253D&md5=15689409a6d9281184a7f943f5cc45d3</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgki524&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgki524%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DSkolnick%26aufirst%3DJ.%26atitle%3DTM-align%253A%2520a%2520protein%2520structure%2520alignment%2520algorithm%2520based%2520on%2520the%2520TM-score%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2005%26volume%3D33%26spage%3D2302%26epage%3D2309%26doi%3D10.1093%2Fnar%2Fgki524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Duan, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, V. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weaver, D. F.</span><span> </span><span class="NLM_article-title">Characterization of aromatic-thiol π-type hydrogen bonding and phenylalanine-cysteine side chain interactions throughab initiocalculations and protein database analyses</span> <span class="citation_source-journal">Mol. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">1689</span><span class="NLM_x">–</span> <span class="NLM_lpage">1699</span><span class="refDoi"> DOI: 10.1080/00268970110063917</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1080%2F00268970110063917" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2001&pages=1689-1699&author=G.+Duanauthor=V.+H.+Smithauthor=D.+F.+Weaver&title=Characterization+of+aromatic-thiol+%CF%80-type+hydrogen+bonding+and+phenylalanine-cysteine+side+chain+interactions+throughab+initiocalculations+and+protein+database+analyses&doi=10.1080%2F00268970110063917"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1080%2F00268970110063917&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00268970110063917%26sid%3Dliteratum%253Aachs%26aulast%3DDuan%26aufirst%3DG.%26aulast%3DSmith%26aufirst%3DV.%2BH.%26aulast%3DWeaver%26aufirst%3DD.%2BF.%26atitle%3DCharacterization%2520of%2520aromatic-thiol%2520%25CF%2580-type%2520hydrogen%2520bonding%2520and%2520phenylalanine-cysteine%2520side%2520chain%2520interactions%2520throughab%2520initiocalculations%2520and%2520protein%2520database%2520analyses%26jtitle%3DMol.%2520Phys.%26date%3D2001%26volume%3D99%26spage%3D1689%26epage%3D1699%26doi%3D10.1080%2F00268970110063917" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Meyer, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castellano, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diederich, F.</span><span> </span><span class="NLM_article-title">Interactions with aromatic rings in chemical and biological recognition</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">1210</span><span class="NLM_x">–</span> <span class="NLM_lpage">1250</span><span class="refDoi"> DOI: 10.1002/anie.200390319</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1002%2Fanie.200390319" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=12645054" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BD3sXivVSrtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2003&pages=1210-1250&author=E.+A.+Meyerauthor=R.+K.+Castellanoauthor=F.+Diederich&title=Interactions+with+aromatic+rings+in+chemical+and+biological+recognition&doi=10.1002%2Fanie.200390319"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Interactions with aromatic rings in chemical and biological recognition</span></div><div class="casAuthors">Meyer, Emmanuel A.; Castellano, Ronald K.; Diederich, Francois</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1210-1250</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Intermol. interactions involving arom. rings are key processes in both chem. and biol. recognition.  Their understanding is essential for rational drug design and lead optimization in medicinal chem.  Different approaches-biol. studies, mol. recognition studies with artificial receptors, crystallog. database mining, gas-phase studies, and theor. calcns. - are pursued to generate a profound understanding of the structural and energetic parameters of individual recognition modes involving arom. rings.  This review attempts to combine and summarize the knowledge gained from these investigations.  The review focuses mainly on examples with biol. relevance since one of its aims it to enhance the knowledge of mol. recognition forces that is essential for drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHUbAhGHcGtLVg90H21EOLACvtfcHk0ljN8J7ltPRXXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXivVSrtbw%253D&md5=e91e6a8d115d56229e1660697a286b04</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fanie.200390319&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200390319%26sid%3Dliteratum%253Aachs%26aulast%3DMeyer%26aufirst%3DE.%2BA.%26aulast%3DCastellano%26aufirst%3DR.%2BK.%26aulast%3DDiederich%26aufirst%3DF.%26atitle%3DInteractions%2520with%2520aromatic%2520rings%2520in%2520chemical%2520and%2520biological%2520recognition%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2003%26volume%3D42%26spage%3D1210%26epage%3D1250%26doi%3D10.1002%2Fanie.200390319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Cavallo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metrangolo, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milani, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pilati, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Priimagi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Resnati, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terraneo, G.</span><span> </span><span class="NLM_article-title">The halogen bond</span> <span class="citation_source-journal">Chem. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">116</span><span class="NLM_x">, </span> <span class="NLM_fpage">2478</span><span class="NLM_x">–</span> <span class="NLM_lpage">2601</span><span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00484</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00484" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVyhs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2016&pages=2478-2601&author=G.+Cavalloauthor=P.+Metrangoloauthor=R.+Milaniauthor=T.+Pilatiauthor=A.+Priimagiauthor=G.+Resnatiauthor=G.+Terraneo&title=The+halogen+bond&doi=10.1021%2Facs.chemrev.5b00484"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">The Halogen Bond</span></div><div class="casAuthors">Cavallo, Gabriella; Metrangolo, Pierangelo; Milani, Roberto; Pilati, Tullio; Priimagi, Arri; Resnati, Giuseppe; Terraneo, Giancarlo</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2478-2601</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The halogen bond occurs when there is evidence of a net attractive interaction between an electrophilic region assocd. with a halogen atom in a mol. entity and a nucleophilic region in another, or the same, mol. entity.  In this fairly extensive review, after a brief history of the interaction, we will provide the reader with a snapshot of where the research on the halogen bond is now, and, perhaps, where it is going.  The specific advantages brought up by a design based on the use of the halogen bond will be demonstrated in quite different fields spanning from material sciences to biomol. recognition and drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreUcfocIywvLVg90H21EOLACvtfcHk0ljN8J7ltPRXXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVyhs74%253D&md5=307f97f5056bdd5300ea0b0eac9fb558</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00484&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00484%26sid%3Dliteratum%253Aachs%26aulast%3DCavallo%26aufirst%3DG.%26aulast%3DMetrangolo%26aufirst%3DP.%26aulast%3DMilani%26aufirst%3DR.%26aulast%3DPilati%26aufirst%3DT.%26aulast%3DPriimagi%26aufirst%3DA.%26aulast%3DResnati%26aufirst%3DG.%26aulast%3DTerraneo%26aufirst%3DG.%26atitle%3DThe%2520halogen%2520bond%26jtitle%3DChem.%2520Rev.%26date%3D2016%26volume%3D116%26spage%3D2478%26epage%3D2601%26doi%3D10.1021%2Facs.chemrev.5b00484" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Mccloskey, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bale, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Secrist, J. A.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tiwari, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moss, T. H.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valiyaveettil, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brooks, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guida, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pegg, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ealick, S. E.</span><span> </span><span class="NLM_article-title">New insights into the design of inhibitors of human S-adenosylmethionine decarboxylase: studies of adenine C8 substitution in structural analogues of S-adenosylmethionine</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1388</span><span class="NLM_x">–</span> <span class="NLM_lpage">1407</span><span class="refDoi"> DOI: 10.1021/jm801126a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm801126a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2009&pages=1388-1407&author=D.+E.+Mccloskeyauthor=S.+Baleauthor=J.+A.+Secristauthor=A.+Tiwariauthor=T.+H.+Mossauthor=J.+Valiyaveettilauthor=W.+H.+Brooksauthor=W.+C.+Guidaauthor=A.+E.+Peggauthor=S.+E.+Ealick&title=New+insights+into+the+design+of+inhibitors+of+human+S-adenosylmethionine+decarboxylase%3A+studies+of+adenine+C8+substitution+in+structural+analogues+of+S-adenosylmethionine&doi=10.1021%2Fjm801126a"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fjm801126a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm801126a%26sid%3Dliteratum%253Aachs%26aulast%3DMccloskey%26aufirst%3DD.%2BE.%26aulast%3DBale%26aufirst%3DS.%26aulast%3DSecrist%26aufirst%3DJ.%2BA.%26aulast%3DTiwari%26aufirst%3DA.%26aulast%3DMoss%26aufirst%3DT.%2BH.%26aulast%3DValiyaveettil%26aufirst%3DJ.%26aulast%3DBrooks%26aufirst%3DW.%2BH.%26aulast%3DGuida%26aufirst%3DW.%2BC.%26aulast%3DPegg%26aufirst%3DA.%2BE.%26aulast%3DEalick%26aufirst%3DS.%2BE.%26atitle%3DNew%2520insights%2520into%2520the%2520design%2520of%2520inhibitors%2520of%2520human%2520S-adenosylmethionine%2520decarboxylase%253A%2520studies%2520of%2520adenine%2520C8%2520substitution%2520in%2520structural%2520analogues%2520of%2520S-adenosylmethionine%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2009%26volume%3D52%26spage%3D1388%26epage%3D1407%26doi%3D10.1021%2Fjm801126a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Xiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Volkov, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tu, B. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, M. A.</span><span> </span><span class="NLM_article-title">Product feedback regulation implicated in translational control of the Trypanosoma brucei S-adenosylmethionine decarboxylase regulatory subunit Prozyme</span> <span class="citation_source-journal">Mol. Microbiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">88</span><span class="NLM_x">, </span> <span class="NLM_fpage">846</span><span class="NLM_x">–</span> <span class="NLM_lpage">861</span><span class="refDoi"> DOI: 10.1111/mmi.12226</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1111%2Fmmi.12226" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=23634831" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BC3sXot1artLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2013&pages=846-861&author=Y.+Xiaoauthor=S.+Nguyenauthor=S.+H.+Kimauthor=O.+A.+Volkovauthor=B.+P.+Tuauthor=M.+A.+Phillips&title=Product+feedback+regulation+implicated+in+translational+control+of+the+Trypanosoma+brucei+S-adenosylmethionine+decarboxylase+regulatory+subunit+Prozyme&doi=10.1111%2Fmmi.12226"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Product feedback regulation implicated in translational control of the Trypanosoma brucei S-adenosylmethionine decarboxylase regulatory subunit prozyme</span></div><div class="casAuthors">Xiao, Yanjing; Nguyen, Suong; Kim, Sok Ho; Volkov, Oleg A.; Tu, Benjamin P.; Phillips, Margaret A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Microbiology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">846-861</span>CODEN:
                <span class="NLM_cas:coden">MOMIEE</span>;
        ISSN:<span class="NLM_cas:issn">0950-382X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Summary : Human African sleeping sickness (HAT) is caused by the parasitic protozoan Trypanosoma brucei.  Polyamine biosynthesis is an important drug target in the treatment of HAT.  Previously we showed that trypanosomatid S-adenosylmethionine decarboxylase (AdoMetDC), a key enzyme for biosynthesis of the polyamine spermidine, is activated by heterodimer formation with an inactive paralog termed prozyme.  Furthermore, prozyme protein levels were regulated in response to reduced AdoMetDC activity.  Herein we show that T. brucei encodes three prozyme transcripts.  The 3'UTRs of these transcripts were mapped and chloramphenicol acetyltransferase (CAT) reporter constructs were used to identify a 1.2 kb region that contained a 3'UTR prozyme regulatory element sufficient to upregulate CAT protein levels (but not RNA) upon AdoMetDC inhibition, supporting the hypothesis that prozyme expression is regulated translationally.  To gain insight into trans-acting factors, genetic rescue of AdoMetDC RNAi knock-down lines with human AdoMetDC was performed leading to rescue of the cell growth block, and restoration of prozyme protein to wild-type levels.  Metabolite anal. showed that prozyme protein levels were inversely proportional to intracellular levels of decarboxylated AdoMet (dcAdoMet).  These data suggest that prozyme translation may be regulated by dcAdoMet, a metabolite not previously identified to play a regulatory role.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoyHtqmD3SgBLVg90H21EOLACvtfcHk0liyJopPMxT4Og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXot1artLg%253D&md5=b88b6fc64d32cf78ccf509d0bd4a46a1</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1111%2Fmmi.12226&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fmmi.12226%26sid%3Dliteratum%253Aachs%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DNguyen%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DS.%2BH.%26aulast%3DVolkov%26aufirst%3DO.%2BA.%26aulast%3DTu%26aufirst%3DB.%2BP.%26aulast%3DPhillips%26aufirst%3DM.%2BA.%26atitle%3DProduct%2520feedback%2520regulation%2520implicated%2520in%2520translational%2520control%2520of%2520the%2520Trypanosoma%2520brucei%2520S-adenosylmethionine%2520decarboxylase%2520regulatory%2520subunit%2520Prozyme%26jtitle%3DMol.%2520Microbiol.%26date%3D2013%26volume%3D88%26spage%3D846%26epage%3D861%26doi%3D10.1111%2Fmmi.12226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Vincent, I. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Creek, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamleh, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woods, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burchmore, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barrett, M. P.</span><span> </span><span class="NLM_article-title">A molecular mechanism for eflornithine resistance in African trypanosomes</span> <span class="citation_source-journal">PLoS Pathog.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1001204</span><span class="refDoi"> DOI: 10.1371/journal.ppat.1001204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1371%2Fjournal.ppat.1001204" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=e1001204&author=I.+M.+Vincentauthor=D.+Creekauthor=D.+G.+Watsonauthor=M.+A.+Kamlehauthor=D.+J.+Woodsauthor=P.+E.+Wongauthor=R.+J.+Burchmoreauthor=M.+P.+Barrett&title=A+molecular+mechanism+for+eflornithine+resistance+in+African+trypanosomes&doi=10.1371%2Fjournal.ppat.1001204"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1001204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1001204%26sid%3Dliteratum%253Aachs%26aulast%3DVincent%26aufirst%3DI.%2BM.%26aulast%3DCreek%26aufirst%3DD.%26aulast%3DWatson%26aufirst%3DD.%2BG.%26aulast%3DKamleh%26aufirst%3DM.%2BA.%26aulast%3DWoods%26aufirst%3DD.%2BJ.%26aulast%3DWong%26aufirst%3DP.%2BE.%26aulast%3DBurchmore%26aufirst%3DR.%2BJ.%26aulast%3DBarrett%26aufirst%3DM.%2BP.%26atitle%3DA%2520molecular%2520mechanism%2520for%2520eflornithine%2520resistance%2520in%2520African%2520trypanosomes%26jtitle%3DPLoS%2520Pathog.%26date%3D2010%26volume%3D6%26spage%3De1001204%26doi%3D10.1371%2Fjournal.ppat.1001204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Bitonti, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dumont, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mccann, P. P.</span><span> </span><span class="NLM_article-title">Characterization of Trypanosoma brucei brucei S-adenosyl-L-methionine decarboxylase and its inhibition by Berenil, pentamidine and methylglyoxal bis(guanylhydrazone)</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1986</span><span class="NLM_x">, </span> <span class="NLM_volume">237</span><span class="NLM_x">, </span> <span class="NLM_fpage">685</span><span class="NLM_x">–</span> <span class="NLM_lpage">689</span><span class="refDoi"> DOI: 10.1042/bj2370685</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1042%2Fbj2370685" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=237&publication_year=1986&pages=685-689&author=A.+J.+Bitontiauthor=J.+A.+Dumontauthor=P.+P.+Mccann&title=Characterization+of+Trypanosoma+brucei+brucei+S-adenosyl-L-methionine+decarboxylase+and+its+inhibition+by+Berenil%2C+pentamidine+and+methylglyoxal+bis%28guanylhydrazone%29&doi=10.1042%2Fbj2370685"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1042%2Fbj2370685&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbj2370685%26sid%3Dliteratum%253Aachs%26aulast%3DBitonti%26aufirst%3DA.%2BJ.%26aulast%3DDumont%26aufirst%3DJ.%2BA.%26aulast%3DMccann%26aufirst%3DP.%2BP.%26atitle%3DCharacterization%2520of%2520Trypanosoma%2520brucei%2520brucei%2520S-adenosyl-L-methionine%2520decarboxylase%2520and%2520its%2520inhibition%2520by%2520Berenil%252C%2520pentamidine%2520and%2520methylglyoxal%2520bis%2528guanylhydrazone%2529%26jtitle%3DBiochem.%2520J.%26date%3D1986%26volume%3D237%26spage%3D685%26epage%3D689%26doi%3D10.1042%2Fbj2370685" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Brun, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buhler, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandmeier, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaminsky, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bacchi, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rattendi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lane, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Croft, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snowdon, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yardley, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caravatti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frei, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanek, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mett, H.</span><span> </span><span class="NLM_article-title">In vitro trypanocidal activities of new S-adenosylmethionine decarboxylase inhibitors</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">1442</span><span class="NLM_x">–</span> <span class="NLM_lpage">1447</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=1996&pages=1442-1447&author=R.+Brunauthor=Y.+Buhlerauthor=U.+Sandmeierauthor=R.+Kaminskyauthor=C.+J.+Bacchiauthor=D.+Rattendiauthor=S.+Laneauthor=S.+L.+Croftauthor=D.+Snowdonauthor=V.+Yardleyauthor=G.+Caravattiauthor=J.+Freiauthor=J.+Stanekauthor=H.+Mett&title=In+vitro+trypanocidal+activities+of+new+S-adenosylmethionine+decarboxylase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBrun%26aufirst%3DR.%26aulast%3DBuhler%26aufirst%3DY.%26aulast%3DSandmeier%26aufirst%3DU.%26aulast%3DKaminsky%26aufirst%3DR.%26aulast%3DBacchi%26aufirst%3DC.%2BJ.%26aulast%3DRattendi%26aufirst%3DD.%26aulast%3DLane%26aufirst%3DS.%26aulast%3DCroft%26aufirst%3DS.%2BL.%26aulast%3DSnowdon%26aufirst%3DD.%26aulast%3DYardley%26aufirst%3DV.%26aulast%3DCaravatti%26aufirst%3DG.%26aulast%3DFrei%26aufirst%3DJ.%26aulast%3DStanek%26aufirst%3DJ.%26aulast%3DMett%26aufirst%3DH.%26atitle%3DIn%2520vitro%2520trypanocidal%2520activities%2520of%2520new%2520S-adenosylmethionine%2520decarboxylase%2520inhibitors%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1996%26volume%3D40%26spage%3D1442%26epage%3D1447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Tolbert, W. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cottet, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ekstrom, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pegg, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ealick, S. E.</span><span> </span><span class="NLM_article-title">Mechanism of human S-adenosylmethionine decarboxylase proenzyme processing as revealed by the structure of the S68A mutant</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">2386</span><span class="NLM_x">–</span> <span class="NLM_lpage">2395</span><span class="refDoi"> DOI: 10.1021/bi0268854</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi0268854" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2003&pages=2386-2395&author=W.+D.+Tolbertauthor=Y.+Zhangauthor=S.+E.+Cottetauthor=E.+M.+Bennettauthor=J.+L.+Ekstromauthor=A.+E.+Peggauthor=S.+E.+Ealick&title=Mechanism+of+human+S-adenosylmethionine+decarboxylase+proenzyme+processing+as+revealed+by+the+structure+of+the+S68A+mutant&doi=10.1021%2Fbi0268854"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fbi0268854&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi0268854%26sid%3Dliteratum%253Aachs%26aulast%3DTolbert%26aufirst%3DW.%2BD.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DCottet%26aufirst%3DS.%2BE.%26aulast%3DBennett%26aufirst%3DE.%2BM.%26aulast%3DEkstrom%26aufirst%3DJ.%2BL.%26aulast%3DPegg%26aufirst%3DA.%2BE.%26aulast%3DEalick%26aufirst%3DS.%2BE.%26atitle%3DMechanism%2520of%2520human%2520S-adenosylmethionine%2520decarboxylase%2520proenzyme%2520processing%2520as%2520revealed%2520by%2520the%2520structure%2520of%2520the%2520S68A%2520mutant%26jtitle%3DBiochemistry%26date%3D2003%26volume%3D42%26spage%3D2386%26epage%3D2395%26doi%3D10.1021%2Fbi0268854" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Bale, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brooks, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanes, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahesan, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guida, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ealick, S. E.</span><span> </span><span class="NLM_article-title">Role of the sulfonium center in determining the ligand specificity of human S-adenosylmethionine decarboxylase</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">6423</span><span class="NLM_x">–</span> <span class="NLM_lpage">6430</span><span class="refDoi"> DOI: 10.1021/bi900590m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi900590m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=6423-6430&author=S.+Baleauthor=W.+Brooksauthor=J.+W.+Hanesauthor=A.+M.+Mahesanauthor=W.+C.+Guidaauthor=S.+E.+Ealick&title=Role+of+the+sulfonium+center+in+determining+the+ligand+specificity+of+human+S-adenosylmethionine+decarboxylase&doi=10.1021%2Fbi900590m"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fbi900590m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi900590m%26sid%3Dliteratum%253Aachs%26aulast%3DBale%26aufirst%3DS.%26aulast%3DBrooks%26aufirst%3DW.%26aulast%3DHanes%26aufirst%3DJ.%2BW.%26aulast%3DMahesan%26aufirst%3DA.%2BM.%26aulast%3DGuida%26aufirst%3DW.%2BC.%26aulast%3DEalick%26aufirst%3DS.%2BE.%26atitle%3DRole%2520of%2520the%2520sulfonium%2520center%2520in%2520determining%2520the%2520ligand%2520specificity%2520of%2520human%2520S-adenosylmethionine%2520decarboxylase%26jtitle%3DBiochemistry%26date%3D2009%26volume%3D48%26spage%3D6423%26epage%3D6430%26doi%3D10.1021%2Fbi900590m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Bale, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopez, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Makhatadze, G. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pegg, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ealick, S. E.</span><span> </span><span class="NLM_article-title">Structural basis for putrescine activation of human S-adenosylmethionine decarboxylase</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">13404</span><span class="NLM_x">–</span> <span class="NLM_lpage">13417</span><span class="refDoi"> DOI: 10.1021/bi801732m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi801732m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=13404-13417&author=S.+Baleauthor=M.+M.+Lopezauthor=G.+I.+Makhatadzeauthor=Q.+Fangauthor=A.+E.+Peggauthor=S.+E.+Ealick&title=Structural+basis+for+putrescine+activation+of+human+S-adenosylmethionine+decarboxylase&doi=10.1021%2Fbi801732m"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fbi801732m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi801732m%26sid%3Dliteratum%253Aachs%26aulast%3DBale%26aufirst%3DS.%26aulast%3DLopez%26aufirst%3DM.%2BM.%26aulast%3DMakhatadze%26aufirst%3DG.%2BI.%26aulast%3DFang%26aufirst%3DQ.%26aulast%3DPegg%26aufirst%3DA.%2BE.%26aulast%3DEalick%26aufirst%3DS.%2BE.%26atitle%3DStructural%2520basis%2520for%2520putrescine%2520activation%2520of%2520human%2520S-adenosylmethionine%2520decarboxylase%26jtitle%3DBiochemistry%26date%3D2008%26volume%3D47%26spage%3D13404%26epage%3D13417%26doi%3D10.1021%2Fbi801732m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Shantz, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanley, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Secrist, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pegg, A. E.</span><span> </span><span class="NLM_article-title">Purification of human S-adenosylmethionine decarboxylase expressed in Escherichia coli and use of this protein to investigate the mechanism of inhibition by the irreversible inhibitors, 5′-deoxy-5′-[(3-hydrazinopropyl)methylamino]adenosine and 5′-{[(Z)-4-amino-2-butenyl]methylamino}-5′-deoxyadenosine</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">6848</span><span class="NLM_x">–</span> <span class="NLM_lpage">6855</span><span class="refDoi"> DOI: 10.1021/bi00144a027</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00144a027" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1992&pages=6848-6855&author=L.+M.+Shantzauthor=B.+A.+Stanleyauthor=J.+A.+Secristauthor=A.+E.+Pegg&title=Purification+of+human+S-adenosylmethionine+decarboxylase+expressed+in+Escherichia+coli+and+use+of+this+protein+to+investigate+the+mechanism+of+inhibition+by+the+irreversible+inhibitors%2C+5%E2%80%B2-deoxy-5%E2%80%B2-%5B%283-hydrazinopropyl%29methylamino%5Dadenosine+and+5%E2%80%B2-%7B%5B%28Z%29-4-amino-2-butenyl%5Dmethylamino%7D-5%E2%80%B2-deoxyadenosine&doi=10.1021%2Fbi00144a027"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fbi00144a027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00144a027%26sid%3Dliteratum%253Aachs%26aulast%3DShantz%26aufirst%3DL.%2BM.%26aulast%3DStanley%26aufirst%3DB.%2BA.%26aulast%3DSecrist%26aufirst%3DJ.%2BA.%26aulast%3DPegg%26aufirst%3DA.%2BE.%26atitle%3DPurification%2520of%2520human%2520S-adenosylmethionine%2520decarboxylase%2520expressed%2520in%2520Escherichia%2520coli%2520and%2520use%2520of%2520this%2520protein%2520to%2520investigate%2520the%2520mechanism%2520of%2520inhibition%2520by%2520the%2520irreversible%2520inhibitors%252C%25205%25E2%2580%25B2-deoxy-5%25E2%2580%25B2-%255B%25283-hydrazinopropyl%2529methylamino%255Dadenosine%2520and%25205%25E2%2580%25B2-%257B%255B%2528Z%2529-4-amino-2-butenyl%255Dmethylamino%257D-5%25E2%2580%25B2-deoxyadenosine%26jtitle%3DBiochemistry%26date%3D1992%26volume%3D31%26spage%3D6848%26epage%3D6855%26doi%3D10.1021%2Fbi00144a027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Beswick, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willert, E. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, M. A.</span><span> </span><span class="NLM_article-title">Mechanisms of allosteric regulation of Trypanosoma cruzi S-adenosylmethionine decarboxylase</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">7797</span><span class="NLM_x">–</span> <span class="NLM_lpage">7807</span><span class="refDoi"> DOI: 10.1021/bi0603975</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi0603975" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BD28Xlt1ehsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2006&pages=7797-7807&author=T.+C.+Beswickauthor=E.+K.+Willertauthor=M.+A.+Phillips&title=Mechanisms+of+allosteric+regulation+of+Trypanosoma+cruzi+S-adenosylmethionine+decarboxylase&doi=10.1021%2Fbi0603975"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of Allosteric Regulation of Trypanosoma cruzi S-Adenosylmethionine Decarboxylase</span></div><div class="casAuthors">Beswick, Tracy Clyne; Willert, Erin K.; Phillips, Margaret A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">7797-7807</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">S-Adenosylmethionine decarboxylase (AdoMetDC) is a pyruvoyl-dependent enzyme that catalyzes an essential step in polyamine biosynthesis.  The polyamines are required for cell growth, and the biosynthetic enzymes are targets for antiproliferative drugs.  The function of AdoMetDC is regulated by the polyamine-precursor putrescine in a species-specific manner.  AdoMetDC from the protozoal parasite Trypanosoma cruzi requires putrescine for maximal enzyme activity, but not for processing to generate the pyruvoyl cofactor.  The putrescine-binding site is distant from the active site, suggesting a mechanism of allosteric regulation.  To probe the structural basis by which putrescine stimulates T. cruzi AdoMetDC, we generated mutations in both the putrescine-binding site and the enzyme active site.  The catalytic efficiency of the mutant enzymes, and the binding of the diamidine inhibitors, CGP 48664A and CGP 40215, were analyzed.  Putrescine stimulates the kcat/Km for wild-type T. cruzi AdoMetDC by 27-fold, and it stimulates the binding of both inhibitors (IC50s decrease 10-20-fold with putrescine).  Unexpectedly, CGP 48664A activated the T. cruzi enzyme at low concns. (0.1-10 μM), while at higher concns. (>100 μM), or in the presence of putrescine, inhibition was obsd.  Anal. of the mutant data suggests that this inhibitor binds both the putrescine-binding site and the active site, providing evidence that the putrescine-binding site of the T. cruzi enzyme has broad ligand specificity.  Mutagenesis of the active site identified residues that are important for putrescine stimulation of activity (F7 and T245), while none of the active site mutations altered the apparent putrescine-binding const.  Mutations of residues in the putrescine-binding site that resulted in reduced (S111R) and enhanced (F285H) catalytic efficiency were both identified.  These data provide evidence for coupling between residues in the putrescine-binding site and the active site, consistent with a mechanism of allosteric regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKTqMLZ3tiIbVg90H21EOLACvtfcHk0lj3PW5KEdCrbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xlt1ehsrg%253D&md5=76be996bb2845d90383ce22e10a82bec</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fbi0603975&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi0603975%26sid%3Dliteratum%253Aachs%26aulast%3DBeswick%26aufirst%3DT.%2BC.%26aulast%3DWillert%26aufirst%3DE.%2BK.%26aulast%3DPhillips%26aufirst%3DM.%2BA.%26atitle%3DMechanisms%2520of%2520allosteric%2520regulation%2520of%2520Trypanosoma%2520cruzi%2520S-adenosylmethionine%2520decarboxylase%26jtitle%3DBiochemistry%26date%3D2006%26volume%3D45%26spage%3D7797%26epage%3D7807%26doi%3D10.1021%2Fbi0603975" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Minor, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cymborowski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Otwinowski, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chruszcz, M.</span><span> </span><span class="NLM_article-title">HKL-3000: the integration of data reduction and structure solution - from diffraction images to an initial model in minutes</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">859</span><span class="NLM_x">–</span> <span class="NLM_lpage">866</span><span class="refDoi"> DOI: 10.1107/S0907444906019949</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1107%2FS0907444906019949" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=16855301" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BD28XntVWksL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2006&pages=859-866&author=W.+Minorauthor=M.+Cymborowskiauthor=Z.+Otwinowskiauthor=M.+Chruszcz&title=HKL-3000%3A+the+integration+of+data+reduction+and+structure+solution+-+from+diffraction+images+to+an+initial+model+in+minutes&doi=10.1107%2FS0907444906019949"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">HKL-3000: the integration of data reduction and structure solution - from diffraction images to an initial model in minutes</span></div><div class="casAuthors">Minor, Wladek; Cymborowski, Marcin; Otwinowski, Zbyszek; Chruszcz, Maksymilian</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">D62</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">859-866</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A new approach that integrates data collection, data redn., phasing and model building significantly accelerates the process of structure detn. and on av. minimizes the no. of data sets and synchrotron time required for structure soln.  Initial testing of the HKL-3000 system (the beta version was named HKL-2000_ph) with more than 140 novel structure detns. has proven its high value for MAD/SAD expts.  The heuristics for choosing the best computational strategy at different data resoln. limits of phasing signal and crystal diffraction are being optimized.  The typical end result is an interpretable electron-d. map with a partially built structure and, in some cases, an almost complete refined model.  The current development is oriented towards very fast structure soln. in order to provide feedback during the diffraction expt.  Work is also proceeding towards improving the quality of phasing calcn. and model building.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0c0DYtjFP7rVg90H21EOLACvtfcHk0lisSQkErYpp9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XntVWksL8%253D&md5=2cd2fe288f954837abb4acae4c008a2f</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS0907444906019949&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906019949%26sid%3Dliteratum%253Aachs%26aulast%3DMinor%26aufirst%3DW.%26aulast%3DCymborowski%26aufirst%3DM.%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DChruszcz%26aufirst%3DM.%26atitle%3DHKL-3000%253A%2520the%2520integration%2520of%2520data%2520reduction%2520and%2520structure%2520solution%2520-%2520from%2520diffraction%2520images%2520to%2520an%2520initial%2520model%2520in%2520minutes%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2006%26volume%3D62%26spage%3D859%26epage%3D866%26doi%3D10.1107%2FS0907444906019949" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Mccoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storoni, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title">Phaser crystallographic software</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">–</span> <span class="NLM_lpage">674</span><span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+Mccoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lisSQkErYpp9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMccoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunkoczi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mccoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeffner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span><span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+Mccoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0ljB7eFlLwrXPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMccoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span> </span><span class="NLM_article-title">Electronic Ligand Builder and Optimization Workbench (eLBOW): a tool for ligand coordinate and restraint generation</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">1074</span><span class="NLM_x">–</span> <span class="NLM_lpage">1080</span><span class="refDoi"> DOI: 10.1107/S0907444909029436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1107%2FS0907444909029436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=19770504" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2009&pages=1074-1080&author=N.+W.+Moriartyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adams&title=Electronic+Ligand+Builder+and+Optimization+Workbench+%28eLBOW%29%3A+a+tool+for+ligand+coordinate+and+restraint+generation&doi=10.1107%2FS0907444909029436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">electronic Ligand Builder and Optimization Workbench (eLBOW): a tool for ligand coordinate and restraint generation</span></div><div class="casAuthors">Moriarty, Nigel W.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1074-1080</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The electronic Ligand Builder and Optimization Workbench (eLBOW) is a program module of the PHENIX suite of computational crystallog. software.  It is designed to be a flexible procedure that uses simple and fast quantum-chem. techniques to provide chem. accurate information for novel and known ligands alike.  A variety of input formats and options allow the attainment of a no. of diverse goals including geometry optimization and generation of restraints.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojnN4AVY_DwrVg90H21EOLACvtfcHk0ljkfOvjC4iqYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnN&md5=3f9a9c6c31c9a4b014a9f151fb6aa8d9</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1107%2FS0907444909029436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909029436%26sid%3Dliteratum%253Aachs%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DElectronic%2520Ligand%2520Builder%2520and%2520Optimization%2520Workbench%2520%2528eLBOW%2529%253A%2520a%2520tool%2520for%2520ligand%2520coordinate%2520and%2520restraint%2520generation%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2009%26volume%3D65%26spage%3D1074%26epage%3D1080%26doi%3D10.1107%2FS0907444909029436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and development of Coot</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">486</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span><span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0ljkfOvjC4iqYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TVM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TVM','PDB','5TVM'); return false;">PDB: 5TVM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3DZ7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3DZ7','PDB','3DZ7'); return false;">PDB: 3DZ7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TVF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5TVF','PDB','5TVF'); return false;">PDB: 5TVF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BM7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BM7','PDB','6BM7'); return false;">PDB: 6BM7</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i119"><a href="/doi/suppl/10.1021/acs.jmedchem.7b01654">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_73730"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.7b01654">10.1021/acs.jmedchem.7b01654</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Molecular formula strings and assay data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01654/suppl_file/jm7b01654_si_001.csv">CSV</a>)</p></li><li><p class="inline">Table S1 with structures and biological assay data for all tested compounds, Table S2 with X-ray crystallography data collection and refinement statistics, Figure S1 with electron density maps supporting ligand placement in the X-ray crystal structure, and H and <sup>13</sup>C NMR spectra of synthesized analogs (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01654/suppl_file/jm7b01654_si_002.pdf">PDF</a>)</p></li><li><p class="inline">Coordinates information for structure representation (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b01654/suppl_file/jm7b01654_si_003.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01654/suppl_file/jm7b01654_si_001.csv">jm7b01654_si_001.csv (15.63 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01654/suppl_file/jm7b01654_si_002.pdf">jm7b01654_si_002.pdf (5.55 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b01654/suppl_file/jm7b01654_si_003.pdb">jm7b01654_si_003.pdb (1.55 MB)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The crystal structure of <i>Trypanosoma brucei</i> AdoMetDC/prozyme heterodimer in complex with pyrimidineamine inhibitor <b>44</b> was deposited at the RCSB Protein Data Bank under accession number <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6BM7">6BM7</a> (authors O. A. Volkov, Z. Chen, and M. A. Phillips). Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b01654&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-3%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b01654%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b01654" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799da5ae927191d","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
