
<html lang="en"     class="pb-page"  data-request-id="709bd591-5faa-43c4-add7-6678d115f6e7"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.0c00660;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2020.63.issue-17;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Screening of a Custom-Designed Acid Fragment Library Identifies 1-Phenylpyrroles and 1-Phenylpyrrolidines as Inhibitors of Notum Carboxylesterase Activity" /></meta><meta name="dc.Creator" content="William  Mahy" /></meta><meta name="dc.Creator" content="Mikesh  Patel" /></meta><meta name="dc.Creator" content="David  Steadman" /></meta><meta name="dc.Creator" content="Hannah L.  Woodward" /></meta><meta name="dc.Creator" content="Benjamin N.  Atkinson" /></meta><meta name="dc.Creator" content="Fredrik  Svensson" /></meta><meta name="dc.Creator" content="Nicky J.  Willis" /></meta><meta name="dc.Creator" content="Alister  Flint" /></meta><meta name="dc.Creator" content="Dimitra  Papatheodorou" /></meta><meta name="dc.Creator" content="Yuguang  Zhao" /></meta><meta name="dc.Creator" content="Luca  Vecchia" /></meta><meta name="dc.Creator" content="Reinis R.  Ruza" /></meta><meta name="dc.Creator" content="James  Hillier" /></meta><meta name="dc.Creator" content="Sarah  Frew" /></meta><meta name="dc.Creator" content="Amy  Monaghan" /></meta><meta name="dc.Creator" content="Artur  Costa" /></meta><meta name="dc.Creator" content="Magda  Bictash" /></meta><meta name="dc.Creator" content="Magnus W.  Walter" /></meta><meta name="dc.Creator" content="E. Yvonne  Jones" /></meta><meta name="dc.Creator" content="Paul V.  Fish" /></meta><meta name="dc.Description" content="The Wnt family of proteins are secreted signaling proteins that play key roles in regulating cellular functions. Recently, carboxylesterase Notum was shown to act as a negative regulator of Wnt sig..." /></meta><meta name="Description" content="The Wnt family of proteins are secreted signaling proteins that play key roles in regulating cellular functions. Recently, carboxylesterase Notum was shown to act as a negative regulator of Wnt sig..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 29, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00660" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright Â© 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00660" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00660" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00660" /></link>
        
    
    

<title>Screening of a Custom-Designed Acid Fragment Library Identifies 1-Phenylpyrroles and 1-Phenylpyrrolidines as Inhibitors of Notum Carboxylesterase Activity | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00660" /></meta><meta property="og:title" content="Screening of a Custom-Designed Acid Fragment Library Identifies 1-Phenylpyrroles and 1-Phenylpyrrolidines as Inhibitors of Notum Carboxylesterase Activity" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/large/jm0c00660_0014.jpeg" /></meta><meta property="og:description" content="The Wnt family of proteins are secreted signaling proteins that play key roles in regulating cellular functions. Recently, carboxylesterase Notum was shown to act as a negative regulator of Wnt signaling by mediating the removal of an essential palmitoleate. Here we disclose two new chemical scaffolds that inhibit Notum enzymatic activity. Our approach was to create a fragment library of 250 acids for screening against Notum in a biochemical assay followed by structure determination by X-ray crystallography. Twenty fragments were identified as hits for Notum inhibition, and 14 of these fragments were shown to bind in the palmitoleate pocket of Notum. Optimization of 1-phenylpyrrole 20, guided by structure-based drug design, identified 20z as the most potent compound from this series. Similarly, the optimization of 1-phenylpyrrolidine 8 gave acid 26. This work demonstrates that inhibition of Notum activity can be achieved by small, drug-like molecules possessing favorable in vitro ADME profiles." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00660"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00660">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00660&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00660&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00660&amp;href=/doi/10.1021/acs.jmedchem.0c00660" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 17</span><span class="cit-fg-pageRange">, 9464-9483</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/17" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00656" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00683" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Screening of a Custom-Designed Acid Fragment Library Identifies 1-Phenylpyrroles and 1-Phenylpyrrolidines as Inhibitors of Notum Carboxylesterase Activity</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">William Mahy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">William Mahy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Alzheimerâs Research UK UCL Drug Discovery Institute, University College London, Cruciform Building, Gower Street, London WC1E 6BT, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=William++Mahy">William Mahy</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Mikesh Patel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mikesh Patel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University College London, 20 Gordon Street, London WC1H 0AJ, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mikesh++Patel">Mikesh Patel</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">David Steadman</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David Steadman</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Alzheimerâs Research UK UCL Drug Discovery Institute, University College London, Cruciform Building, Gower Street, London WC1E 6BT, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David++Steadman">David Steadman</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4271-5525" title="Orcid link">http://orcid.org/0000-0003-4271-5525</a></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Hannah L. Woodward</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hannah L. Woodward</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Alzheimerâs Research UK UCL Drug Discovery Institute, University College London, Cruciform Building, Gower Street, London WC1E 6BT, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hannah+L.++Woodward">Hannah L. Woodward</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8429-2546" title="Orcid link">http://orcid.org/0000-0001-8429-2546</a></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Benjamin N. Atkinson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Benjamin N. Atkinson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Alzheimerâs Research UK UCL Drug Discovery Institute, University College London, Cruciform Building, Gower Street, London WC1E 6BT, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Benjamin+N.++Atkinson">Benjamin N. Atkinson</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5511-9859" title="Orcid link">http://orcid.org/0000-0001-5511-9859</a></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Fredrik Svensson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fredrik Svensson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Alzheimerâs Research UK UCL Drug Discovery Institute, University College London, Cruciform Building, Gower Street, London WC1E 6BT, United Kingdom</div><div class="loa-info-affiliations-info">The Francis Crick Institute, 1 Midland Road, Kings Cross, London NW1 1AT, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fredrik++Svensson">Fredrik Svensson</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5556-8133" title="Orcid link">http://orcid.org/0000-0002-5556-8133</a></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Nicky J. Willis</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nicky J. Willis</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Alzheimerâs Research UK UCL Drug Discovery Institute, University College London, Cruciform Building, Gower Street, London WC1E 6BT, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nicky+J.++Willis">Nicky J. Willis</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Alister Flint</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alister Flint</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, University College London, 20 Gordon Street, London WC1H 0AJ, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alister++Flint">Alister Flint</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Dimitra Papatheodorou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dimitra Papatheodorou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">UCL School of Pharmacy, University College London, 29-39 Brunswick Square, Bloomsbury, London WC1N 1AX, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dimitra++Papatheodorou">Dimitra Papatheodorou</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Yuguang Zhao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yuguang Zhao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Structural Biology, Wellcome Centre for Human Genetics, University of Oxford, The Henry Wellcome Building for Genomic Medicine, Roosevelt Drive, Oxford OX3 7BN, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yuguang++Zhao">Yuguang Zhao</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8916-8552" title="Orcid link">http://orcid.org/0000-0001-8916-8552</a></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Luca Vecchia</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Luca Vecchia</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Structural Biology, Wellcome Centre for Human Genetics, University of Oxford, The Henry Wellcome Building for Genomic Medicine, Roosevelt Drive, Oxford OX3 7BN, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Luca++Vecchia">Luca Vecchia</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Reinis R. Ruza</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Reinis R. Ruza</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Structural Biology, Wellcome Centre for Human Genetics, University of Oxford, The Henry Wellcome Building for Genomic Medicine, Roosevelt Drive, Oxford OX3 7BN, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Reinis+R.++Ruza">Reinis R. Ruza</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">James Hillier</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">James Hillier</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Structural Biology, Wellcome Centre for Human Genetics, University of Oxford, The Henry Wellcome Building for Genomic Medicine, Roosevelt Drive, Oxford OX3 7BN, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=James++Hillier">James Hillier</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Sarah Frew</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sarah Frew</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Alzheimerâs Research UK UCL Drug Discovery Institute, University College London, Cruciform Building, Gower Street, London WC1E 6BT, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sarah++Frew">Sarah Frew</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Amy Monaghan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Amy Monaghan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Alzheimerâs Research UK UCL Drug Discovery Institute, University College London, Cruciform Building, Gower Street, London WC1E 6BT, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Amy++Monaghan">Amy Monaghan</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Artur Costa</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Artur Costa</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Alzheimerâs Research UK UCL Drug Discovery Institute, University College London, Cruciform Building, Gower Street, London WC1E 6BT, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Artur++Costa">Artur Costa</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Magda Bictash</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Magda Bictash</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Alzheimerâs Research UK UCL Drug Discovery Institute, University College London, Cruciform Building, Gower Street, London WC1E 6BT, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Magda++Bictash">Magda Bictash</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Magnus W. Walter</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Magnus W. Walter</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Eli Lilly, Erl Wood Manor, Windlesham, Surrey GU20 6PH, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Magnus+W.++Walter">Magnus W. Walter</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">E. Yvonne Jones</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">E. Yvonne Jones</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Division of Structural Biology, Wellcome Centre for Human Genetics, University of Oxford, The Henry Wellcome Building for Genomic Medicine, Roosevelt Drive, Oxford OX3 7BN, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=E.+Yvonne++Jones">E. Yvonne Jones</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3834-1893" title="Orcid link">http://orcid.org/0000-0002-3834-1893</a></div></div></span></li><span class="comma-separator">,Â andÂ </span><li><span><span class="hlFld-ContribAuthor">Paul V. Fish</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paul V. Fish</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Alzheimerâs Research UK UCL Drug Discovery Institute, University College London, Cruciform Building, Gower Street, London WC1E 6BT, United Kingdom</div><div class="loa-info-affiliations-info">The Francis Crick Institute, 1 Midland Road, Kings Cross, London NW1 1AT, United Kingdom</div></div><span class="conrtib-corresp"><strong>*</strong>Phone: +44-(0)20-7679-6971. Email: <a href="/cdn-cgi/l/email-protection#ea9ac48c839982aa9f8986c48b89c49f81"><span class="__cf_email__" data-cfemail="bece90d8d7cdd6fecbddd290dfdd90cbd5">[emailÂ protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paul+V.++Fish">Paul V. Fish</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2117-2173" title="Orcid link">http://orcid.org/0000-0002-2117-2173</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00660&amp;href=/doi/10.1021%2Facs.jmedchem.0c00660" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 17</span><span class="cit-pageRange">, 9464â9483</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 29, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>21 April 2020</li><li><span class="item_label"><b>Published</b> online</span>29 July 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 10 September 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00660" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00660</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D9464%26pageCount%3D20%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DWilliam%2BMahy%252C%2BMikesh%2BPatel%252C%2BDavid%2BSteadman%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D17%26contentID%3Dacs.jmedchem.0c00660%26title%3DScreening%2Bof%2Ba%2BCustom-Designed%2BAcid%2BFragment%2BLibrary%2BIdentifies%2B1-Phenylpyrroles%2Band%2B1-Phenylpyrrolidines%2Bas%2BInhibitors%2Bof%2BNotum%2BCarboxylesterase%2BActivity%26numPages%3D20%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D9483%26publicationDate%3DSeptember%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00660"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">938</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">5</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00660" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Screening of a Custom-Designed Acid Fragment Library Identifies 1-Phenylpyrroles and 1-Phenylpyrrolidines as Inhibitors of Notum Carboxylesterase Activity&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;William&quot;,&quot;last_name&quot;:&quot;Mahy&quot;},{&quot;first_name&quot;:&quot;Mikesh&quot;,&quot;last_name&quot;:&quot;Patel&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;Steadman&quot;},{&quot;first_name&quot;:&quot;Hannah&quot;,&quot;last_name&quot;:&quot;L. Woodward&quot;},{&quot;first_name&quot;:&quot;Benjamin&quot;,&quot;last_name&quot;:&quot;N. Atkinson&quot;},{&quot;first_name&quot;:&quot;Fredrik&quot;,&quot;last_name&quot;:&quot;Svensson&quot;},{&quot;first_name&quot;:&quot;Nicky&quot;,&quot;last_name&quot;:&quot;J. Willis&quot;},{&quot;first_name&quot;:&quot;Alister&quot;,&quot;last_name&quot;:&quot;Flint&quot;},{&quot;first_name&quot;:&quot;Dimitra&quot;,&quot;last_name&quot;:&quot;Papatheodorou&quot;},{&quot;first_name&quot;:&quot;Yuguang&quot;,&quot;last_name&quot;:&quot;Zhao&quot;},{&quot;first_name&quot;:&quot;Luca&quot;,&quot;last_name&quot;:&quot;Vecchia&quot;},{&quot;first_name&quot;:&quot;Reinis&quot;,&quot;last_name&quot;:&quot;R. Ruza&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;Hillier&quot;},{&quot;first_name&quot;:&quot;Sarah&quot;,&quot;last_name&quot;:&quot;Frew&quot;},{&quot;first_name&quot;:&quot;Amy&quot;,&quot;last_name&quot;:&quot;Monaghan&quot;},{&quot;first_name&quot;:&quot;Artur&quot;,&quot;last_name&quot;:&quot;Costa&quot;},{&quot;first_name&quot;:&quot;Magda&quot;,&quot;last_name&quot;:&quot;Bictash&quot;},{&quot;first_name&quot;:&quot;Magnus&quot;,&quot;last_name&quot;:&quot;W. Walter&quot;},{&quot;first_name&quot;:&quot;E.&quot;,&quot;last_name&quot;:&quot;Yvonne Jones&quot;},{&quot;first_name&quot;:&quot;Paul&quot;,&quot;last_name&quot;:&quot;V. Fish&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;29&quot;,&quot;issue&quot;:&quot;17&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;9464-9483&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00660&quot;},&quot;abstract&quot;:&quot;The Wnt family of proteins are secreted signaling proteins that play key roles in regulating cellular functions. Recently, carboxylesterase Notum was shown to act as a negative regulator of Wnt signaling by mediating the removal of an essential palmitoleate. Here we disclose two new chemical scaffolds that inhibit Notum enzymatic activity. Our approach was to create a fragment library of 250 acids for screening against Notum in a biochemical assay followed by structure determination by X-ray crystallography. Twenty fragments were identified as hits for Notum inhibition, and 14 of these fragments were shown to bind in the palmitoleate pocket of Notum. Optimization of 1-phenylpyrrole 20, guided by structure-based drug design, identified 20z as the most potent compound from this series. Similarly, the optimization of 1-phenylpyrrolidine 8 gave acid 26. This work demonstrates that inhibition of Notum activity can be achieved by small, drug-like molecules possessing favorable in vitro ADME profiles.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00660&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00660" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00660&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00660" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00660&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00660" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00660&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00660&amp;href=/doi/10.1021/acs.jmedchem.0c00660" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00660" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00660" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00660&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00660%26sid%3Dliteratum%253Aachs%26pmid%3D32787107%26genre%3Darticle%26aulast%3DMahy%26date%3D2020%26atitle%3DScreening%2Bof%2Ba%2BCustom-Designed%2BAcid%2BFragment%2BLibrary%2BIdentifies%2B1-Phenylpyrroles%2Band%2B1-Phenylpyrrolidines%2Bas%2BInhibitors%2Bof%2BNotum%2BCarboxylesterase%2BActivity%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D17%26spage%3D9464%26epage%3D9483%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290676" title="Substituents">Substituents</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292178" title="Organic compounds">Organic compounds</a>,</li><li><a href="/action/doSearch?ConceptID=292329" title="Pyrroles">Pyrroles</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/17" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/jmcmar.2020.63.issue-17/20200910/jmcmar.2020.63.issue-17.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/medium/jm0c00660_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/large/jm0c00660_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00660&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The Wnt family of proteins are secreted signaling proteins that play key roles in regulating cellular functions. Recently, carboxylesterase Notum was shown to act as a negative regulator of Wnt signaling by mediating the removal of an essential palmitoleate. Here we disclose two new chemical scaffolds that inhibit Notum enzymatic activity. Our approach was to create a fragment library of 250 acids for screening against Notum in a biochemical assay followed by structure determination by X-ray crystallography. Twenty fragments were identified as hits for Notum inhibition, and 14 of these fragments were shown to bind in the palmitoleate pocket of Notum. Optimization of 1-phenylpyrrole <b>20</b>, guided by structure-based drug design, identified <b>20z</b> as the most potent compound from this series. Similarly, the optimization of 1-phenylpyrrolidine <b>8</b> gave acid <b>26</b>. This work demonstrates that inhibition of Notum activity can be achieved by small, drug-like molecules possessing favorable <i>in vitro</i> ADME profiles.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68642" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68642" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The Wnt family of proteins are secreted signaling proteins that play key roles in regulating cellular functions in embryonic development and adult stem cell biology.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The Wnt pathways control cell proliferation, cell fate determination, cell polarity, cell migration, and apoptosis. Wnt signaling is activated by the binding of a Wnt protein to a member of the Frizzled (FZD) family of cell surface receptors and to coreceptors such as the LDL-receptor-related protein (LRP) family. Upon binding, an intracellular signaling cascade is triggered by the pathway activating protein Disheveled (Dsh), which acts as a switch and branching point into multiple downstream Wnt pathways (canonical, noncanonical planar cell polarity, and noncanonical Wnt/calcium). Signaling by Wnt proteins is finely balanced and tightly regulated by a sophisticated network of modulators and feedback processes including secreted inhibitory proteins and post-translational modifications (PTM).<a onclick="showRef(event, 'ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4">(2â4)</a> Conversely, misregulation of Wnt signaling disrupts normal development and cell homeostasis and has been associated with cancer,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Alzheimerâs disease (AD),<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> metabolic disorders, and other diseases.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a></div><div class="NLM_p">Recently, carboxylesterase Notum was shown to act as a negative regulator of the Wnt signaling pathway by mediating the removal of an essential palmitoleate group from a conserved serine residue of Wnt proteins.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> This palimitoleate group is an important PTM required for efficient binding of the Wnt proteins to the FZD receptors. Hence, modulation of Notum activity represents a new approach to treat disease, where dysregulation of Wnt signaling is an underlying cause, and Notum has been implicated as the responsible agent.</div><div class="NLM_p">There is an emerging understanding of the potential role Notum plays in disease. Notum is involved in the progression of colorectal cancer (CRC) as Notum expression was increased in human metastatic CRC cells and proliferation was suppressed by inhibiting expression of Notum.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Osteoblast-derived Notum reduces cortical bone mass in mice, whereas Notum inhibition increases endocortical bone formation and bone strength and represents a potential treatment for preventing nonvertebral fractures.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a> Recent work has shown that the <i>Notum</i> gene is expressed and upregulated in endothelial cells in the hippocampus of APP/PS1 mice and AD patients compared to the control, although the role of Notum in the mammalian central nervous system (CNS) has yet to be established.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a></div><div class="NLM_p">The Notum crystal structures reveal a well-defined, large (ca. 380 Ã<sup>3</sup>), hydrophobic active-site pocket adjacent to the catalytic triad (Ser232, His389, Asp340) that accommodates the palmitoleate group of Wnt (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4UZQ">4UZQ</a>).<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> The <i>cis</i>-C9â10 double bond of the C16 palmitoleate tail is positioned at the base of the pocket between Ile291, Phe319, and Phe320, which helps explain substrate preference. The binding pocket can accommodate extended carbon tails up to C8/C10; however, longer fatty acid chains need to have a bend in their structure at C9 in order to fit. In addition, the Gly127âTrp128 amide participates in the formation of the oxyanion hole along with the canonical Ser232âAla233 and Gly126âGly127 amides, thereby providing extra stabilization to the tetrahedral transition state during ester hydrolysis. The pocket entrance, lined by Ser232 and His389, is relatively narrow but shows substantial flexibility. These structural features of Notum combine to render the palmitoleate pocket to be regarded as highly druggable.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a></div><div class="NLM_p">The number of reports of inhibitors of Notum carboxylesterase activity has gradually increased in recent years (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Thieno[3,2-<i>d</i>]pyrimidine <b>LP-922056</b> (<b>1</b>) is a potent, orally active inhibitor of Notum that has been found to increase cortical bone thickness in mouse models of bone growth.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Scaffold-hopping from acid <b>1</b>, supported by X-ray structure determination, identified amide <b>2</b> as a potent inhibitor of Notum.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Pharmacokinetic studies in a mouse with <b>2</b> showed good plasma exposure and reasonable brain penetration; in contrast, <b>1</b> had negligible brain exposure.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a><i>N</i>-Hydroxyhydantoin carbamate <b>ABC99</b> is a potent and selective covalent irreversible inhibitor of Notum.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a><b>ABC99</b> preserves Wnt-mediated cell signaling in the presence of Notum and has been used to show the role of Notum in the regeneration of aged intestinal epithelium.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> A crystallographic fragment screen using the XChem platform at Diamond Light Source identified a number of fragment hits that bound in the palmitoleate pocket of Notum.<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20,21)</a> Phenoxyacetamide <b>3</b> was identified through the optimization of one of these hits and showed an unexpected flipped binding mode when compared to the structures of close analogues bound to Notum.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> A second fragment hit from this screen was <i>N</i>-[2-(5-fluoro-1<i>H</i>-indol-3-yl)ethyl]acetamide (<b>4</b>), which is structurally related to the brain hormone melatonin. A detailed structural analysis of the binding of <b>4</b> (and close analogues) to Notum has been disclosed, which may guide the development of new potent inhibitors.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Several of these inhibitors (<b>1</b>, <b>3</b>, <b>4</b> plus others) have been soaked into crystals of Notum and the structures solved to determine the inhibitor binding modes. Hence, under suitable conditions, Notum is very amenable to structure-based drug design (SBDD) to help guide progress toward the discovery of potent inhibitors.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/medium/jm0c00660_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/large/jm0c00660_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures and PDB codes of Notum inhibitors. <sup>a</sup>Notum IC<sub>50</sub> data presented for comparison in a common assay format. <sup>b</sup>Determined by competitive gel-based activity-based protein profiling (ABPP); see ref <a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a>. <sup>c</sup>As a covalent inhibitor of Notum, the IC<sub>50</sub> value will be time-dependent. <sup>d</sup>See ref <a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/large/jm0c00660_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00660&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Our objective was to identify alternative chemical scaffolds to the prior art that efficiently inhibit Notum enzymatic activity with the potential to deliver structurally orthogonal chemical tools for use in drug target validation experiments. We adopted a strategy to simultaneously explore multiple chemotypes in order to increase our chances of delivering a âfit-for-purposeâ chemical tool. Furthermore, diverse chemotypes, which share the same primary pharmacology, are likely to offer the benefit of a different secondary pharmacology fingerprint profile and help identify any confounding off-target effects in disease models.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28817" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28817" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Our approach was to create a custom-designed fragment library of carboxylic acids for screening against Notum (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Each compound of the library would have three distinct structural features: (1) a carboxylic acid to interact with the catalytic triad and to anchor the inhibitor through a polar interaction as observed previously;<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> (2) a suitable lipophilic group to be presented into the palmitoleate binding pocket of Notum; and (3) a linker of various lengths to join the acid with the lipophilic group (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/medium/jm0c00660_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/large/jm0c00660_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Design and screening of the fragment library. (A) Compound structural features. (B) Plot of lipophilicity (clog<i>P</i>) versus molecular size (MW) for the acid fragment library. (C) Flow diagram showing key activities and results.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/large/jm0c00660_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00660&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">A single commercial provider was selected for ease and consistency of compound supply and to simplify procurement. The Enamine Carboxylic Acids Fragment Library was provisionally selected because of its size (4350 compounds) and features of ânovelty and core diversityâ; these features were assessed through evaluation of the libraryâs content.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> The full library was filtered using criteria for molecular properties and structural parameters as described in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, and this virtual collection (514 compounds) was further manually curated by inspection to ensure a balanced representation of physicochemical property space (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B) along with structural diversity, while minimizing repetition of a particular core. This process of design and selection gave an acid fragment library of 250 compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00660/suppl_file/jm0c00660_si_001.pdf" class="ext-link">Table S1</a>), which was screened for inhibition of Notum activity (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Design and Selection Criteria for the Carboxylic Acid Fragment Library</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">parameter (measure)</th><th class="colsep0 rowsep0" align="center">criteria</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">molecular size (MW)</td><td class="colsep0 rowsep0" align="left">170Â â¤Â MWÂ â¤Â 240</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">lipophilicity (clog<i>P</i>)</td><td class="colsep0 rowsep0" align="left">â2 â¤ clog<i>P</i> â¤ 2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">polarity (TPSA)</td><td class="colsep0 rowsep0" align="left">â¤90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">H-bonding capacity (HBD, HBA)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">1Â â¤Â HBDÂ â¤Â 2; HBAÂ â¤Â 3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">aromatic rings</td><td class="colsep0 rowsep0" align="left">â¤2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">minimize no. of stereocenters</td><td class="colsep0 rowsep0" align="left">â¤1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">maximize no. of sp<sup>3</sup> carbons</td><td class="colsep0 rowsep0" align="left">â¥1 (where possible)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Excluding the COOH group.</p></div></div></div><div class="NLM_p">Inhibition of Notum carboxylesterase activity of this library was measured in a biochemical assay where test compounds (dispensed to give 10-point concentration-response-curves up to 100 Î¼M) were incubated with Notum(81â451 Cys330Ser) and trisodium 8-octanoyloxypyrene-1,3,6-trisulfonate (OPTS) as the substrate for 40 min, and fluorescence was recorded. Twenty compounds showed inhibition of Notum activity with initial IC<sub>50</sub> < 25 Î¼M (<i>n</i> = 2), although most of these hits showed a modest decrease in activity upon routine retesting (<i>n</i> = 5â10). All 20 of these putative hits were submitted to a Notum crystallographic screen by soaking crystals of <i>C</i>-terminal his-tagged Notum (81â451 Cys330Ser) with the compounds screened at 10 mM, and 14 fragments were observed to bind in the palmitoleate pocket.</div><div class="NLM_p">From the Notum biochemical and crystallographic screen, two preferred hit series were selected for further investigation: pyrrole-3-carboxylic acids <b>5</b>-<b>7</b> and pyrrolidine-3-carboxylic acids <b>8</b>â<b>10</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). These six hits shared a closely related structural motif and had excellent hit-like properties (LE = 0.45â0.57) with no obvious reactive groups. In addition, compounds <b>5</b>â<b>8</b> and <b>10</b> had been successfully crystallized with Notum to support an SBDD program (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/medium/jm0c00660_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/large/jm0c00660_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Pyrrole-3-carboxylic acids <b>5</b>â<b>7</b> and pyrrolidine-3-carboxylic acids <b>8</b>â<b>10</b> as fragment screening hits. Notum IC<sub>50</sub> (Î¼M) (<i>n</i> = 5â10).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/large/jm0c00660_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00660&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/medium/jm0c00660_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/large/jm0c00660_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Notum structures with fragment screening hits <b>5</b>â<b>8</b> and <b>10</b>. Crystal structures of (A) <b>5</b> (orange), (B) <b>6</b> (teal), (C) <b>7</b> (gray), (D) <b>8</b> (magenta), and (E) <b>10</b> (yellow). (F) Overlay of all ligands. Binding site residues shown within 4 Ã of their respective ligands. Key hydrogen bond interactions are shown as dashed lines. Water molecules have been removed for clarity. The surface of the Notum palmitoleoate binding pocket outlined (gray) from the <i>O</i>-palmitoleoyl serineâNotum (S232A) structure (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4UZQ">4UZQ</a>). Atomic coordinates have been deposited in the Protein Data Bank (PDB). PDB ID codes: <b>5</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YUY">6YUY</a>; <b>6</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YV4">6YV4</a>; <b>7</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YUW">6YUW</a>; <b>8</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YV2">6YV2</a>; <b>10</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YV0">6YV0</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/large/jm0c00660_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00660&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The original six hits <b>5</b>â<b>10</b> provided some insight into preliminary structureâactivity relationships (SARs), along with two structurally related inactive compounds, which were also present in the library (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00660/suppl_file/jm0c00660_si_001.pdf" class="ext-link">Figure S1</a>). The three pyrroles <b>5</b>â<b>7</b> all had 2- and 5-flanking groups either side of the <i>N</i>-atom supporting the inclusion of Me and/or CF<sub>3</sub> groups at these positions. There was some variation in the N1 substituent with both cPr and CH<sub>2</sub>cPr represented and showing activity. The three pyrrolidines <b>8</b>â<b>10</b> had a single point change in the 2-position of the N1-phenyl ring, showing activity could be improved with suitable groups (F, Cl > H). Despite these six hits providing limited SAR, it was encouraging that they represented a series of related hits (rather than singletons) with the potential for further optimization.</div><div class="NLM_p">The crystal structures reveal that the inhibitors preferably occupy the core of the pocket, interacting with the side chains of Phe268 and Trp128, with some ligands (<b>7</b>, <b>8</b>, and <b>10</b>) also extending to the oxyanion hole (the backbone of Trp128, Gly127, and Ala233) in the proximity of the catalytic triad (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). Due to the fragmentâs modest size, they fail to occupy the pocket fully. The suboptimal occupation of the pocket aligns with the observed modest potencies.</div><div class="NLM_p">Our general design strategy for new compounds was initially directed at exploring three principle areas of these hit structures: (1) the pyrrole-3-carboxylic acid ring <b>11</b>â<b>24</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>); (2) the 1-phenylpyrrole ring that binds in the palmitoleate pocket <b>20</b> (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>); and (3) the 1-phenylpyrrolidine rings <b>25</b>â<b>27</b> (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Further bespoke modifications of preferred leads were then performed to improve activity and tune physicochemical properties <b>28</b>â<b>30</b> (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00660/suppl_file/jm0c00660_si_001.pdf" class="ext-link">Figure S2</a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Notum Inhibition of Pyrrole-3-carboxylic Acids <b>11</b>â<b>24</b>; SAR of the Pyrrole Ring<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/medium/jm0c00660_0008.gif" alt="" id="gr7" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="rowsep1 colsep0" colspan="4" align="center">R</th><th class="colsep0 rowsep0" align="center">Â </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">R<sup>1</sup></th><th class="colsep0 rowsep0" align="center">R<sup>2</sup></th><th class="colsep0 rowsep0" align="center">R<sup>4</sup></th><th class="colsep0 rowsep0" align="center">R<sup>5</sup></th><th class="colsep0 rowsep0" align="center">Notum IC<sub>50</sub> (Î¼M)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CF<sub>3</sub></td><td class="colsep0 rowsep0">6.7Â Â±Â 3.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0">â«100<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0">â«100<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0">ca. 100<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0">ca. 100<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CF<sub>3</sub></td><td class="colsep0 rowsep0">â«100<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0">ca. 100<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0">â«100<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0">>100<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0">3.8Â Â±Â 1.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0">1.7Â Â±Â 0.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CF<sub>3</sub></td><td class="colsep0 rowsep0">22Â Â±Â 3.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="left">CH<sub>2</sub>Ph</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0">3.1Â Â±Â 0.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="left">CH<sub>2</sub>Ph</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0">14Â Â±Â 3.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="left">CH<sub>2</sub>Ph</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CF<sub>3</sub></td><td class="colsep0 rowsep0">>100<a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">All values are mean Â± sd of <i>n</i> = 2â4 experiments quoted to 2 sf. Differences of <2-fold should not be considered significant. For details of the assay protocol, see ref <a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a>.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last"><20% I @ 100 Î¼M.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">ca. 50% I @ 100 Î¼M.</p></div><div class="footnote" id="t2fn4"><sup><sup>d</sup></sup><p class="last">20â40% I @ 100 Î¼M.</p></div></div></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Notum Inhibition of 2,5-Dimethyl-1-phenylpyrrole-3-carboxylic Acids <b>20</b>; SAR of the Phenyl Ring<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/medium/jm0c00660_0009.gif" alt="" id="gr8" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="Â±" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="Â±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">R<sup>2</sup>âR<sup>6</sup></th><th class="colsep0 rowsep0" align="center" char="Â±">Notum IC<sub>50</sub> (Î¼M)</th><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">R<sup>2</sup>âR<sup>6</sup></th><th class="colsep0 rowsep0" align="center" char="Â±">Notum IC<sub>50</sub> (Î¼M)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char="Â±">1.7Â Â±Â 0.6</td><td class="colsep0 rowsep0" align="left"><b>20r</b></td><td class="colsep0 rowsep0" align="left">2-Cl,Â 3-Cl</td><td class="colsep0 rowsep0" align="char" char="Â±">0.22Â Â±Â 0.09</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20b</b></td><td class="colsep0 rowsep0" align="left">2-F</td><td class="colsep0 rowsep0" align="char" char="Â±">3.3Â Â±Â 0.7</td><td class="colsep0 rowsep0" align="left"><b>20s</b></td><td class="colsep0 rowsep0" align="left">2-Cl,Â 4-Cl</td><td class="colsep0 rowsep0" align="char" char="Â±">0.43Â Â±Â 0.17</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20c</b></td><td class="colsep0 rowsep0" align="left">3-F</td><td class="colsep0 rowsep0" align="char" char="Â±">1.5Â Â±Â 0.2</td><td class="colsep0 rowsep0" align="left"><b>20t</b></td><td class="colsep0 rowsep0" align="left">2-Cl,Â 5-Cl</td><td class="colsep0 rowsep0" align="char" char="Â±">1.7Â Â±Â 0.15</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20d</b></td><td class="colsep0 rowsep0" align="left">4-F</td><td class="colsep0 rowsep0" align="char" char="Â±">1.9Â Â±Â 0.6</td><td class="colsep0 rowsep0" align="left"><b>20u</b></td><td class="colsep0 rowsep0" align="left">2-Cl,Â 6-Cl</td><td class="colsep0 rowsep0" align="char" char="Â±">2.3Â Â±Â 0.77</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20e</b></td><td class="colsep0 rowsep0" align="left">2-Me</td><td class="colsep0 rowsep0" align="char" char="Â±">2.2Â Â±Â 0.90</td><td class="colsep0 rowsep0" align="left"><b>20v</b></td><td class="colsep0 rowsep0" align="left">3-Cl,Â 4-Cl</td><td class="colsep0 rowsep0" align="char" char="Â±">0.27Â Â±Â 0.10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20f</b></td><td class="colsep0 rowsep0" align="left">3-Me</td><td class="colsep0 rowsep0" align="char" char="Â±">1.5Â Â±Â 0.51</td><td class="colsep0 rowsep0" align="left"><b>20w</b></td><td class="colsep0 rowsep0" align="left">3-Cl,Â 5-Cl</td><td class="colsep0 rowsep0" align="char" char="Â±">0.85Â Â±Â 0.30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20g</b></td><td class="colsep0 rowsep0" align="left">4-Me</td><td class="colsep0 rowsep0" align="char" char="Â±">2.0Â Â±Â 0.10</td><td class="colsep0 rowsep0" align="left"><b>20x</b></td><td class="colsep0 rowsep0" align="left">2-F,Â 3-Cl</td><td class="colsep0 rowsep0" align="char" char="Â±">0.38Â Â±Â 0.03</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20h</b></td><td class="colsep0 rowsep0" align="left">2-Cl</td><td class="colsep0 rowsep0" align="char" char="Â±">2.3Â Â±Â 0.74</td><td class="colsep0 rowsep0" align="left"><b>20y</b></td><td class="colsep0 rowsep0" align="left">2-F,Â 3-cPr</td><td class="colsep0 rowsep0" align="char" char="Â±">0.20Â Â±Â 0.09</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20i</b></td><td class="colsep0 rowsep0" align="left">3-Cl</td><td class="colsep0 rowsep0" align="char" char="Â±">0.43Â Â±Â 0.04</td><td class="colsep0 rowsep0" align="left"><b>20z</b></td><td class="colsep0 rowsep0" align="left">2-F,Â 3-iPr</td><td class="colsep0 rowsep0" align="char" char="Â±">0.15Â Â±Â 0.02</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20j</b></td><td class="colsep0 rowsep0" align="left">4-Cl</td><td class="colsep0 rowsep0" align="char" char="Â±">0.86Â Â±Â 0.10</td><td class="colsep0 rowsep0" align="left"><b>20aa</b></td><td class="colsep0 rowsep0" align="left">2-F,Â 3-CN</td><td class="colsep0 rowsep0" align="char" char="Â±">0.63Â Â±Â 0.23</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20k</b></td><td class="colsep0 rowsep0" align="left">3-CF<sub>3</sub></td><td class="colsep0 rowsep0" align="char" char="Â±">1.5Â Â±Â 0.20</td><td class="colsep0 rowsep0" align="left"><b>20bb</b></td><td class="colsep0 rowsep0" align="left">2-OMe,Â 3-Cl</td><td class="colsep0 rowsep0" align="char" char="Â±">0.80Â Â±Â 0.25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20l</b></td><td class="colsep0 rowsep0" align="left">3-cPr</td><td class="colsep0 rowsep0" align="char" char="Â±">0.58Â Â±Â 0.19</td><td class="colsep0 rowsep0" align="left"><b>20cc</b></td><td class="colsep0 rowsep0" align="left">2-CN,Â 3-Cl</td><td class="colsep0 rowsep0" align="char" char="Â±">2.0Â Â±Â 0.65</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20m</b></td><td class="colsep0 rowsep0" align="left">3-iPr</td><td class="colsep0 rowsep0" align="char" char="Â±">0.36Â Â±Â 0.20</td><td class="colsep0 rowsep0" align="left"><b>20dd</b></td><td class="colsep0 rowsep0" align="left">3-Cl,Â 4-CF<sub>3</sub></td><td class="colsep0 rowsep0" align="char" char="Â±">1.5Â Â±Â 0.16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20n</b></td><td class="colsep0 rowsep0" align="left">3-tBu</td><td class="colsep0 rowsep0" align="char" char="Â±">0.46Â Â±Â 0.09</td><td class="colsep0 rowsep0" align="left"><b>20ee</b></td><td class="colsep0 rowsep0" align="left">3-CF<sub>3</sub>,Â 4-Cl</td><td class="colsep0 rowsep0" align="char" char="Â±">2.1Â Â±Â 0.12</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20o</b></td><td class="colsep0 rowsep0" align="left">3-OMe</td><td class="colsep0 rowsep0" align="char" char="Â±">1.5Â Â±Â 0.3</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="char" char="Â±">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20p</b></td><td class="colsep0 rowsep0" align="left">3-OEt</td><td class="colsep0 rowsep0" align="char" char="Â±">0.64Â Â±Â 0.25</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="char" char="Â±">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20q</b></td><td class="colsep0 rowsep0" align="left">3-CN</td><td class="colsep0 rowsep0" align="char" char="Â±">3.1Â Â±Â 1.1</td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="char" char="Â±">Â </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">See footnotes in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>.</p></div></div></div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Notum Inhibition of 1-Phenylpyrrolidine-3-carboxylic Acids <b>25</b>â<b>27</b>; SAR of the Phenyl Ring<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/medium/jm0c00660_0010.gif" alt="" id="gr10" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="Â±" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="Â±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">R<sup>2</sup>âR<sup>6</sup></th><th class="colsep0 rowsep0" align="center" char="Â±">NotumÂ IC<sub>50</sub>Â (Î¼M)</th><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">R<sup>2</sup>âR<sup>6</sup></th><th class="colsep0 rowsep0" align="center" char="Â±">NotumÂ IC<sub>50</sub>Â (Î¼M)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char="Â±">48Â Â±Â 31</td><td class="colsep0 rowsep0" align="left"><b>25j</b></td><td class="colsep0 rowsep0" align="left">2-Cl, 3-Cl</td><td class="colsep0 rowsep0" align="char" char="Â±">8.0Â Â±Â 1.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="left">2-F</td><td class="colsep0 rowsep0" align="char" char="Â±">11Â Â±Â 9</td><td class="colsep0 rowsep0" align="left"><b>25k</b></td><td class="colsep0 rowsep0" align="left">2-Cl, 4-Cl</td><td class="colsep0 rowsep0" align="char" char="Â±">10Â Â±Â 2.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="left">2-Cl</td><td class="colsep0 rowsep0" align="char" char="Â±">19Â Â±Â 14</td><td class="colsep0 rowsep0" align="left"><b>25l</b></td><td class="colsep0 rowsep0" align="left">2-Cl, 5-Cl</td><td class="colsep0 rowsep0" align="char" char="Â±">7.9Â Â±Â 1.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25a</b></td><td class="colsep0 rowsep0" align="left">3-Cl</td><td class="colsep0 rowsep0" align="char" char="Â±">14Â Â±Â 0.9</td><td class="colsep0 rowsep0" align="left"><b>25m</b></td><td class="colsep0 rowsep0" align="left">2-Cl, 6-Cl</td><td class="colsep0 rowsep0" align="char" char="Â±">26Â Â±Â 8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25b</b></td><td class="colsep0 rowsep0" align="left">4-Cl</td><td class="colsep0 rowsep0" align="char" char="Â±">19Â Â±Â 4.4</td><td class="colsep0 rowsep0" align="left"><b>25n</b></td><td class="colsep0 rowsep0" align="left">3-Cl, 4-Cl</td><td class="colsep0 rowsep0" align="char" char="Â±">1.2Â Â±Â 0.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25c</b></td><td class="colsep0 rowsep0" align="left">3-Me</td><td class="colsep0 rowsep0" align="char" char="Â±">49Â Â±Â 11</td><td class="colsep0 rowsep0" align="left"><b>25o</b></td><td class="colsep0 rowsep0" align="left">3-Cl, 5-Cl</td><td class="colsep0 rowsep0" align="char" char="Â±">11Â Â±Â 1.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25d</b></td><td class="colsep0 rowsep0" align="left">3-CF<sub>3</sub></td><td class="colsep0 rowsep0" align="char" char="Â±">11Â Â±Â 0.6</td><td class="colsep0 rowsep0" align="left"><b>25p</b></td><td class="colsep0 rowsep0" align="left">3-Cl, 4-CF<sub>3</sub></td><td class="colsep0 rowsep0" align="char" char="Â±">8.5Â Â±Â 0.77</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25e</b></td><td class="colsep0 rowsep0" align="left">4-CF<sub>3</sub></td><td class="colsep0 rowsep0" align="char" char="Â±">50Â Â±Â 3.8</td><td class="colsep0 rowsep0" align="left"><b>25q</b></td><td class="colsep0 rowsep0" align="left">3-CF<sub>3</sub>,Â 4-Cl</td><td class="colsep0 rowsep0" align="char" char="Â±">0.20Â Â±Â 0.02</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25f</b></td><td class="colsep0 rowsep0" align="left">3-iPr</td><td class="colsep0 rowsep0" align="char" char="Â±">57Â Â±Â 19</td><td class="colsep0 rowsep0" align="left"><b>25r</b></td><td class="colsep0 rowsep0" align="left">3-CF<sub>3</sub>, 4-F</td><td class="colsep0 rowsep0" align="char" char="Â±">2.5Â Â±Â 0.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25g</b></td><td class="colsep0 rowsep0" align="left">3-OCF<sub>3</sub></td><td class="colsep0 rowsep0" align="char" char="Â±">63Â Â±Â 13</td><td class="colsep0 rowsep0" align="left"><b>25s</b></td><td class="colsep0 rowsep0" align="left">2-F, 3-CF<sub>3</sub></td><td class="colsep0 rowsep0" align="char" char="Â±">6.8Â Â±Â 0.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25h</b></td><td class="colsep0 rowsep0" align="left">3-CN</td><td class="colsep0 rowsep0" align="char" char="Â±">42Â Â±Â 12</td><td class="colsep0 rowsep0" align="left"><b>25t</b></td><td class="colsep0 rowsep0" align="left">2-Cl,Â 3-Cl,Â 4-Cl</td><td class="colsep0 rowsep0" align="char" char="Â±">0.60Â Â±Â 0.23</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25i</b></td><td class="colsep0 rowsep0" align="left">4-F</td><td class="colsep0 rowsep0" align="char" char="Â±">15Â Â±Â 1.2</td><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="left">3-CF<sub>3</sub>, 4-Cl</td><td class="colsep0 rowsep0" align="char" char="Â±">0.11Â Â±Â 0.01</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="char" char="Â±">Â </td><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="left">3-CF<sub>3</sub>, 4-Cl</td><td class="colsep0 rowsep0" align="char" char="Â±">0.28Â Â±Â 0.02</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">See footnotes in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>.</p></div></div></div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Notum Inhibition of 1-Phenylpyrrolidine-3-carboxamides <b>28</b>, Oxadiazole <b>29</b>, and Oxadiazolone <b>30</b><a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/medium/jm0c00660_0011.gif" alt="" id="gr11" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">See footnotes in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>.</p></div></div><div></div></div><div class="NLM_p">Minimizing lipophilicity is a well-established approach to improve ADME properties, and target compounds were designed to have molecular and physicochemical properties consistent with drug-like space.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> The palmitoleate binding pocket of Notum is a very lipophilic environment, and we were keen to avoid gains in activity simply through increased compound lipophilicity. A helpful design metric was lipophilic ligand efficiency (LLE),<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> used to track improvements in Notum inhibition activity against lipophilicity of the compounds as determined by calculated clog<i>P</i> and measured CHI log<i>D</i><sub>7.4</sub> values.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></div><div class="NLM_p">The synthetic routes to new inhibitors of Notum in the pyrrole-3-carboxylic acid (<b>11</b>â<b>24</b>) and pyrrolidine-3-carboxylic acid series (<b>25</b>â<b>30</b>) are presented in <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a>, respectively.</div><figure id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/medium/jm0c00660_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/large/jm0c00660_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Preparation of Pyrrole-3-carboxylic Acids <b>11</b>â<b>24</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/large/jm0c00660_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00660&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Representative reagents and conditions: (a) NaH (1.2 equiv), THF, 0 Â°C then R<sup>5</sup>C(O)CH<sub>2</sub>Cl <b>32</b> (1.0 equiv), NaI (0.1 equiv) 0 Â°C to RT, 24 h; (b) amine <b>34</b> (R<sup>1</sup>NH<sub>2</sub>) (1.0 equiv), <i>p</i>-TsOHÂ·H<sub>2</sub>O (0.1 equiv), EtOH, 80 Â°C; (c) LiOHÂ·H<sub>2</sub>O (15 equiv), 1,4-dioxaneâH<sub>2</sub>O, 100 Â°C.</p></p></figure><figure id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/medium/jm0c00660_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/large/jm0c00660_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Preparation of Pyrrolidine-3-carboxylic Acids <b>25</b>â<b>27</b>, Amides <b>28</b>, Oxadiazole <b>29</b>, and Oxadiazolone <b>30</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/large/jm0c00660_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00660&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Representative reagents and conditions: (a) <b>36</b> (1.2 equiv), ArI <b>37</b> (1.0 equiv), Pd<sub>2</sub>dba<sub>3</sub> (2.5 mol %), Xantphos (5 mol %), Cs<sub>2</sub>CO<sub>3</sub> (2.7 equiv), 1,4-dioxane, 100 Â°C, 16 h; (b) aq NaOH (1 M, 2.0 equiv), MeOH, RT, 2 h; (c) HATU (1.1 equiv), DMF, RT, 15 min; then iPr<sub>2</sub>NEt (2.2 equiv), amine <b>39</b> (HNRRâ²) (1.5 equiv); (d) NH<sub>2</sub>NH<sub>2</sub>Â·H<sub>2</sub>O (4 equiv), EtOHâPhMe, 80 Â°C, 16 h; (e) HC(OEt)<sub>3</sub> (4.5 equiv), <i>p</i>-TsOHÂ·H<sub>2</sub>O (0.1 equiv), 60 Â°C, 5 min then RT, 1 h; (f) triphosgene (0.4 equiv), iPr<sub>2</sub>NEt (2.0 equiv), CH<sub>2</sub>Cl<sub>2</sub>, 0 Â°C to RT, 15 min.</p></p></figure><div class="NLM_p">Pyrrole-3-carboxylic acid target compounds <b>11</b>â<b>24</b> were either purchased or prepared using a classical PaalâKnorr pyrrole synthesis (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). Alkylation of Î²-keto ester <b>31</b> with chloromethyl ketone <b>32</b> (R<sup>5</sup>C(O)CH<sub>2</sub>Cl) gave 1,4-diketo ester <b>33</b>, and cyclocondensation of <b>33</b> with amine <b>34</b> (R<sup>1</sup>NH<sub>2</sub>) gave pyrrole ester <b>35</b>. Hydrolysis with LiOH afforded the pyrrole acid. Initially, ethyl esters (<b>31</b>: R = Et) were used, but the corresponding pyrrole esters (<b>35</b>: R = Et) were found to be resilient to hydrolysis even under forcing conditions. Switching to the <i>t</i>-butyl esters (<b>31</b>: R = <i>t</i>-Bu) improved the synthetic route as the pyrrole esters (<b>35</b>: R = <i>t</i>-Bu) were found to hydrolyze under the conditions of the PaalâKnorr reaction, giving the pyrrole-3-carboxylic acids directly .</div><div class="NLM_p">Pyrrolidine-3-carboxylic acid target compounds <b>25</b>â<b>27</b> were prepared using a simple two-step sequence (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). 3-Pyrrolidine ester <b>36</b> was cross-coupled with the aryl iodide <b>37</b> under Pd-mediated catalysis to give the corresponding 1-arylpyrrolidine <b>38</b> and then hydrolysis of the ester gave acid <b>25</b>.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> The single enantiomers (<i>S</i>)-<b>26</b> and (<i>R</i>)-<b>27</b> were prepared from the corresponding chiral esters, methyl (<i>S</i>)- and (<i>R</i>)-pyrrolidine-3-carboxylate (<b>36</b>), respectively, as described above, to provide samples of high chemical and optical purity.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Amides <b>28</b> were prepared by activation of the acid <b>25</b> with HATU and subsequent reaction with the amine <b>39</b> (HNRRâ²). Heterocycles <b>29</b> and <b>30</b> were prepared from ester <b>38</b> by reaction with hydrazine to give the acyl hydrazide, and then treatment with triethyl orthoformate gave 1,3,4-oxadiazole <b>29</b>, whereas treatment with triphosgene gave 1,3,4-oxadiazol-2(3<i>H</i>)-one <b>30</b>.</div><div class="NLM_p">The design of new inhibitors with improved Notum activity initially focused on the SARs of the pyrrole-3-carboxylic acids by exploring the substitution on the pyrrole rings <b>11</b>â<b>24</b> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). This set allowed for the direct comparison of a number of matched pairs to identify beneficial changes. Deconstructing <b>5</b>â<b>7</b> to their components in order to identify the minimum pharmacophore was achieved with a new set of fragments <b>11</b>â<b>18</b>. However, all these smaller fragments had a significant decrease in Notum activity, suggesting a further reduction in size was not feasible (e.g., <b>5</b> vs <b>11</b> vs <b>12</b> vs <b>15</b>). In contrast, the N1 substituent (R<sup>1</sup>) proved to be quite influential with the Ph group preferred: Ph (<b>20</b>) > cPr (<b>6</b>) > CH<sub>2</sub>cPr (<b>7</b>), CH<sub>2</sub>Ph (<b>23</b>) â« Me (<b>18</b>), H (<b>17</b>). There was a subtle balance of interplay between the R<sup>1</sup> and R<sup>5</sup> groups as shown by the significant decrease in activity when poorly partnered (e.g., R<sup>1</sup> = H prefers R<sup>5</sup> = CF<sub>3</sub> (<b>5</b>) â« Me (<b>17</b>), whereas R<sup>1</sup> = Ph prefers R<sup>5</sup> = Me (<b>20</b>) â¥ H (<b>19</b>) > CF<sub>3</sub> (<b>21</b>) and R<sup>1</sup> = CH<sub>2</sub>Ph prefers R<sup>5</sup> = H (<b>22</b>) > Me (<b>23</b>) â« CF<sub>3</sub> (<b>24</b>)). From this limited set, 1-phenylpyrrole <b>20</b> emerged as having an attractive profile suitable for further optimization.</div><div class="NLM_p">A wider range of <i>N</i>-aryls around <b>20</b> were then prepared to determine preferred substituents and substitution patterns while retaining the 2,5-dimethyl groups on the pyrrole (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). The standard tactic of walking a F (<b>20b</b>â<b>d</b>), Me (<b>20e</b>â<b>g</b>), and Cl (<b>20h</b>â<b>j</b>) around the phenyl ring showed a clear preference for the 3-Cl group <b>20i</b> when compared to the unsubstituted Ph ring (<b>20</b>). Notum crystals were soaked with <b>20i</b>, and the structure was solved to show the ligand sitting deep in the lipophilic pocket with no direct interaction being made by the acid (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). Additional single substituents were then introduced to the 3-position (<b>20k</b>â<b>q</b>) with a general trend emerging, showing a preference for small lipophilic groups and Notum inhibition tracking with lipophilicity: iPr (<b>20m</b>) â¥ Cl (<b>20i</b>), tBu (<b>20n</b>), cPr (<b>20l</b>), OEt (<b>20p</b>) > CF<sub>3</sub> (<b>20k</b>), OMe (<b>20o</b>), Me (<b>20f</b>), F (<b>20c</b>), H (<b>20</b>) â¥ CN (<b>20q</b>).</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/medium/jm0c00660_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/large/jm0c00660_0005.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Notum structures with pyrrole and pyrrolidine inhibitors. Crystal structures of (A) <b>20i</b> (emerald) and (B) <b>26</b> (green). (C) Overlay of <b>20i</b> (emerald) and <b>26</b> (green) shows different binding modes. (D) Overlay of <b>26</b> (green) with <b>1</b> (purple; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T2K">6T2K</a>) shows acid groups aligned. Binding site residues shown within 4 Ã of their respective ligands. Key hydrogen bond interactions are shown as dashed lines. Water molecules have been removed for clarity. The surface of the Notum palmitoleoate binding pocket outlined (gray) from the <i>O</i>-palmitoleoyl serineâNotum (S232A) structure (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4UZQ">4UZQ</a>). Atomic coordinates have been deposited in the Protein Data Bank (PDB). PDB ID codes: <b>20i</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YXI">6YXI</a>; <b>26</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YSK">6YSK</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/large/jm0c00660_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00660&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The combination of two substituents on the phenyl ring was initially explored by using two Cl atoms (<b>20r</b>â<b>w</b>) as a single 3- and 4-Cl atom had proved to be favorable (<b>20i</b>, <b>20j</b>) (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>) and the required dichloroanilines <b>34</b> were readily available. The 2,3- (<b>20r</b>) and 3,4-dichloro (<b>20v</b>) isomers were the most potent from this set and further improved Notum inhibition activity to ca. 200â300 nM. Additional SARs were then developed using a wider range of substituents at the 2,3- and 3,4-positions (<b>20x</b>â<b>ee</b>) by building on the earlier SAR, while aiming to moderate overall lipophilicity of the compounds. A combination of a 2-F with preferred groups at the 3-position (iPr, cPr) gave some of the most potent compounds prepared so far: 2-F,3-iPr (<b>20z</b>) and 2-F,3-cPr (<b>20y</b>).</div><div class="NLM_p">Through an exploration of the SARs within the pyrrole series, gains in Notum inhibition activity had been achieved when compared to the original fragment hits (e.g., for <b>20z</b> vs <b>6</b>, 40-fold increase). However, the molecular properties of the most active inhibitors showed these compounds to be quite lipophilic (e.g., <b>20z</b>: IC<sub>50</sub> 0.15 Î¼M, clog<i>P</i> 5.5). Despite our efforts to moderate compound properties, Notum inhibition activity tracked with lipophilicity in this series resulted in the most potent compounds having the highest clog<i>P</i> values.</div><div class="NLM_p">Our focus then switched to the pyrrolidine-3-carboxylic acids hit series <b>8</b> as this template offered the advantage of significantly lower lipophilicity when compared to the matched pyrrole, albeit with weaker Notum inhibition (e.g., <b>8</b> vs <b>20</b>; IC<sub>50</sub> 48 vs 1.7 Î¼M; Îclog<i>P</i> = â2.4). The application of preferred aryl groups from the pyrrole series to the 1-phenylpyrrolidines <b>25</b> showed the SARs were not directly transferable; there was a significant decrease in Notum inhibition activity and a change in the rank order (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). However, the 2-Cl (<b>10</b>), 3-Cl (<b>25a</b>), and 4-Cl (<b>25b</b>) were again among the preferred single groups (2-F, 2-Cl, 3-CF<sub>3</sub>, 3-Cl, 4-F, 4-Cl; IC<sub>50</sub> < 20 Î¼M) and used to explore the effect of two substituents on Notum activity (<b>25j</b>â<b>o</b>) with the 3,4-dichloro <b>25n</b> as the most potent isomer within this set.</div><div class="NLM_p">Extending this approach by exploring closely related halogenated analogues (<b>25p</b>â<b>s</b>) that incorporated a 3-CF<sub>3</sub> (cf. <b>25d</b>) identified 3-CF<sub>3</sub>,4-Cl (<b>25q</b>) as a breakthrough compound due to its Notum inhibition activity and lower lipophilicity (IC<sub>50</sub> 0.20 Î¼M, clog<i>P</i> 3.3). It is interesting to compare that the 3-CF<sub>3</sub>,4-Cl substituents were significantly weaker in the pyrrole series (<b>20ee</b>, IC<sub>50</sub> 2.1 Î¼M). Hence, despite the apparent structural similarity between the pyrrole <b>20</b> and pyrrolidine <b>25</b> templates, the SARs proved to be divergent. This was rationalized by the binding modes observed in the crystal structures of compounds <b>20i</b> and <b>8</b>, where the pyrrole series sits deeper in the pocket compared to the pyrrolidine. The 2- and 5-methyl substituents on the pyrrole template also change the preferred conformation of the N1âAr group. The 2,3,4-trichloro analogue <b>25s</b> (IC<sub>50</sub> 0.60 Î¼M, clog<i>P</i> 3.7) was also a submicromolar inhibitor of Notum, albeit with higher lipophilicity, and was not pursued further at this time.</div><div class="NLM_p">The two enantiomers of <b>25q</b> were then prepared to give (<i>S</i>)-<b>26 a</b>nd (<i>R</i>)-<b>27</b>.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Both enantiomers had good Notum activity with a slight preference for <b>26</b> (IC<sub>50</sub> 0.11 Î¼M, clog<i>P</i> 3.3). Additional structural information was then sought to guide the next phase of design. Notum crystals were soaked with the most potent pyrrolidine <b>26</b>, and the structure was solved to show the compound binding in a similar mode as observed for hit <b>8</b> but with the 3-COOH more optimally positioned for the interaction with the oxyanion hole (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B,C). This is possibly due to the 3- and 4-aryl substituents allowing the inhibitor to both fully occupy the lipophilic pocket and make these interactions.</div><div class="NLM_p">The X-ray structure of <b>26</b> in the active site of Notum was overlaid with the Notum structure of <b>1</b> (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T2K">6T2K</a>) and showed very good alignment of the acid groups supporting the use of carboxamides at this position in the template (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D).<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Initial SAR studies with amides <b>28</b> (derived from <b>25q</b>) suggested that Notum activity was particularly sensitive to the amide substituent as most groups were detrimental (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). Only the 3,3-difluoroazetine <b>28e</b> and pyrrolidine <b>28f</b> amides retained activity similar to the corresponding acid <b>25q</b>.</div><div class="NLM_p">An extension of this design strategy was to explore small heterocycles as bioisosteres for the carboxamide (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). Oxadiazoles were initially selected due to their successful application in drug discovery programs and accessibility from ester <b>38</b>.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> However, 1,3,4-oxadiazole <b>29</b> gave a significant drop in potency compared to acid <b>25q</b> and amides <b>28</b>. In contrast, switching to the corresponding weakly acidic 1,3,4-oxadiazol-2(3<i>H</i>)-one gave <b>30</b> (IC<sub>50</sub> 4.2 Î¼M, clog<i>P</i> 3.2) (p<i>K</i><sub>a</sub> ca. 6.3),<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> which restored moderate inhibition of Notum activity. Oxadiazolone <b>30</b> was of particular interest as a lead to explore the potential of weak acids to penetrate the brain as molecules with these properties are under-represented in CNS drug discovery.<a onclick="showRef(event, 'ref30 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref30 ref31 ref32">(30â32)</a></div><div class="NLM_p">A set of preferred inhibitors (<b>20z</b>, <b>26</b>, <b>28e</b>, and <b>30</b>) were then assessed in <i>in vitro</i> ADME assays to compare their aqueous solubility, microsomal stability (MLM), and cell permeability (MDCK-MDR1) (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). These inhibitors were selected based on their Notum activity but also their chemical structural diversity and complementary physicochemical properties (clog<i>P</i>, p<i>K</i><sub>a</sub>). On balance, pyrrolidine-3-acid <b>26</b> demonstrated a superior profile consistent with its properties: good solubility; good metabolic stability in MLM that predicts for low clearance; and good permeability without efflux by the P-glycoprotein (P-gp) transporter as measured by transit performance in the MDCK-MDR1 cell line. Oxadiazolone <b>30</b> also had a satisfactory combination of ADME results, whereas pyrrole-3-acid <b>20z</b> suffered from moderate MLM stability, probably due to its higher lipophilicity. It is of note that there was evidence of non-CYP-mediated metabolism in MLM for amide <b>28e</b> (i.e., loss of a parent in an NADPH-independent manner).<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a></div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Summary of Physicochemical and Molecular Properties, Notum Inhibition, and ADME Data for <b>20z</b>, <b>26</b>, <b>28e</b>, and <b>30</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/medium/jm0c00660_0012.gif" alt="" id="gr13" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="colsep0 rowsep0" align="center"><b>20z</b></th><th class="colsep0 rowsep0" align="center"><b>26</b></th><th class="colsep0 rowsep0" align="center"><b>28e</b></th><th class="colsep0 rowsep0" align="center"><b>30</b></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Â </th><th class="colsep0 rowsep0" align="center">Â </th><th class="colsep0 rowsep0" align="center">Â </th><th class="colsep0 rowsep0" align="center">Â </th><th class="colsep0 rowsep0" align="center">Â </th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="2" align="left"><i>Physicochemical and molecular properties</i></td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">mol wt</td><td class="colsep0 rowsep0" align="left">275</td><td class="colsep0 rowsep0" align="left">293</td><td class="colsep0 rowsep0" align="left">368</td><td class="colsep0 rowsep0" align="left">333</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">clog<i>P</i></td><td class="colsep0 rowsep0" align="left">5.5</td><td class="colsep0 rowsep0" align="left">3.3</td><td class="colsep0 rowsep0" align="left">4.1</td><td class="colsep0 rowsep0" align="left">3.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CHI log<i>D</i><sub>7.4</sub></td><td class="colsep0 rowsep0" align="left">3.1</td><td class="colsep0 rowsep0" align="left">1.5</td><td class="colsep0 rowsep0" align="left">4.0</td><td class="colsep0 rowsep0" align="left">3.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TPSA (Ã<sup>2</sup>)</td><td class="colsep0 rowsep0" align="left">40.5</td><td class="colsep0 rowsep0" align="left">40.5</td><td class="colsep0 rowsep0" align="left">23.6</td><td class="colsep0 rowsep0" align="left">53.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ligand efficiency (LE)</td><td class="colsep0 rowsep0" align="left">0.48</td><td class="colsep0 rowsep0" align="left">0.51</td><td class="colsep0 rowsep0" align="left">0.39</td><td class="colsep0 rowsep0" align="left">0.34</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">lipophilic ligand efficiency (LLE)<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">3.7</td><td class="colsep0 rowsep0" align="left">5.4</td><td class="colsep0 rowsep0" align="left">2.5</td><td class="colsep0 rowsep0" align="left">1.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>Notum inhibition</i><a class="ref internalNav" href="#t6fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">OPTS, IC<sub>50</sub> (Î¼M)</td><td class="colsep0 rowsep0" align="left">0.15Â Â±Â 0.02</td><td class="colsep0 rowsep0" align="left">0.11Â Â±Â 0.01</td><td class="colsep0 rowsep0" align="left">0.26Â Â±Â 0.03</td><td class="colsep0 rowsep0" align="left">4.2Â Â±Â 0.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">(<i>n</i>Â =Â 4)</td><td class="colsep0 rowsep0" align="left">(<i>n</i>Â =Â 6)</td><td class="colsep0 rowsep0" align="left">(<i>n</i>Â =Â 4)</td><td class="colsep0 rowsep0" align="left">(<i>n</i>Â =Â 4)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>ADME profile</i><a class="ref internalNav" href="#t6fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td><td class="colsep0 rowsep0" align="left">Â </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">aq solubility (Î¼g/mL)/(Î¼M)</td><td class="colsep0 rowsep0" align="left">14/51</td><td class="colsep0 rowsep0" align="left">>145/>595</td><td class="colsep0 rowsep0" align="left">7/19</td><td class="colsep0 rowsep0" align="left">23/69</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MLM, Cl<sub>i</sub> (Î¼L/min/mg protein)</td><td class="colsep0 rowsep0" align="left">52</td><td class="colsep0 rowsep0" align="left">2.5</td><td class="colsep0 rowsep0" align="left">21<a class="ref internalNav" href="#t6fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="left">10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MDCK-MDR1, AB/BA <i>P</i><sub>app</sub> (Ã10<sup>â6</sup> cm/s)</td><td class="colsep0 rowsep0" align="left">46/43</td><td class="colsep0 rowsep0" align="left">39/43</td><td class="colsep0 rowsep0" align="left">8/10</td><td class="colsep0 rowsep0" align="left">13/19</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MDCK-MDR1, efflux ratio (ER)</td><td class="colsep0 rowsep0" align="left">0.93</td><td class="colsep0 rowsep0" align="left">1.1</td><td class="colsep0 rowsep0" align="left">1.2</td><td class="colsep0 rowsep0" align="left">1.5</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">LLE based on the calculation with measured CHI log<i>D</i><sub>7.4</sub> values.</p></div><div class="footnote" id="t6fn2"><sup><sup>b</sup></sup><p class="last">See footnotes in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>.</p></div><div class="footnote" id="t6fn3"><sup><sup>c</sup></sup><p class="last">Studies were performed by GVK Biosciences (Hyderabad, India).</p></div><div class="footnote" id="t6fn4"><sup><sup>d</sup></sup><p class="last">Evidence of non-CYP-mediated metabolism in MLM.</p></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58203" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58203" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">We have disclosed two new chemical scaffolds that efficiently inhibit Notum enzymatic activity. Our approach was to create a custom-designed fragment library of carboxylic acids for screening against Notum in a biochemical assay followed by structure determination by X-ray crystallography. Twenty fragments were identified as hits for Notum inhibition (IC<sub>50</sub> < 25 Î¼M) with two preferred hit series selected for further investigation; these preferred fragments were shown by crystallography to bind in the palmitoleate pocket of Notum. Optimization of 1-phenylpyrrole <b>20</b> by SAR studies guided by SBDD identified <b>20z</b> (IC<sub>50</sub> 0.15 Î¼M) as the most potent compound from this series. Similarly, optimization of 1-phenylpyrrolidine <b>8</b> gave acid <b>26</b> (IC<sub>50</sub> 0.11 Î¼M), amide <b>28e</b> (IC<sub>50</sub> 0.26 Î¼M), and oxadiazolone <b>30</b> (IC<sub>50</sub> 4.2 Î¼M).</div><div class="NLM_p">This work demonstrates that inhibition of Notum carboxylesterase activity can be achieved by small, drug-like molecules supported by attractive <i>in vitro</i> ADME profiles, some of which possess structural features and physicochemical properties consistent with CNS design space,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> including favorable CNS MPO scores (e.g., <b>30</b>, CNS MPO 5.6/6.0).<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/medium/jm0c00660_0013.gif" alt="" id="gr14" /></img></div><div class="NLM_p last">However, despite the significant progress in developing fragment hit <b>8</b> into lead <b>26</b> (>400-fold increase in activity), these compounds proved to be inferior to leads discovered by complementary screening strategies. In brief, screening of an alternative fragment library<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20,21)</a> identified (1-(4-chlorophenyl)-1<i>H</i>-1,2,3-triazol-4-yl)methanol (<b>40</b>),<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> which was optimized to give a potent, selective and CNS penetrant inhibitor of Notum activity suitable for use in both cellular and <i>in vivo</i> models of disease.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87531" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87531" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Chemistry</h3><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> General Information</h4><div class="NLM_p">Unless preparative details are provided, all reagents were purchased from commercial suppliers and used without further purification. Solvents were of ACS reagent grade or higher and purchased from commercial suppliers without further purification. Anhydrous solvents were purchased as such from Acros Organics or Sigma-Aldrich. Thin-layer chromatography (TLC) was carried out on aluminum-backed silica plates. The plates were visualized under UV (254 nm) light, followed by staining with phosphomolybdic acid dip or potassium permanganate and gentle heating.</div><div class="NLM_p">Compound purification by column chromatography was performed using a Biotage Isolera using prepacked Biotage SNAP KP-Sil silica cartridges or Biotage SNAP Ultra C18 reverse-phase cartridges. Chemical and chiral SFC analysis was performed by Reach Separation Ltd. (Nottingham, UK) using the conditions described below. Organic solvent layers were routinely dried with anhydrous MgSO<sub>4</sub> and concentrated using a BÃ¼chi rotary evaporator.</div><div class="NLM_p">Melting points (mp) were determined using Stuart SMP20 melting point equipment using closed-end glass capillary tubes and are uncorrected.</div><div class="NLM_p">Infrared (IR) spectra were recorded on a Shimadzu IRTracer-100 FT-IR spectrometer, using a Universal ATR accessory for sampling, with relevant absorbance quoted as Î½<sub>max</sub> in cm<sup>â1</sup>.</div><div class="NLM_p"><sup>1</sup>H NMR and <sup>13</sup>C NMR spectra were run in deuterated (â¥99.5%) solvents on either a Bruker Avance 300 (300 MHz), Bruker Avance 400 (400 MHz), Bruker Avance 600 (600 MHz), or Bruker Avance 700 (700 MHz). Chemical shifts (Î´) are reported as parts per million (ppm), coupling constants (<i>J</i>) are reported in hertz (Hz), and signal multiplicities are reported as singlet (s), doublet (d), triplet (t), quartet (q), pentet (p), quintet (qu), sextet (sext), doublet of doublets (dd), doublet of triplets (dt), triplet of triplets (tt), multiplet (m), or broad singlet (br s).</div><div class="NLM_p">LCMS analysis was performed on a Waters Acquity H-Class UPLC system with either an acidic (HSS C18 Column, H<sub>2</sub>O/MeCN, 0.1% TFA) or basic (BEH C18 Column, H<sub>2</sub>O/MeCN, 10 mM NH<sub>4</sub>OH) mobile phase.</div><div class="NLM_p">HRMS data acquisition was on a Waters Micromass LCT Premier electrospray time-of-flight (ESI-TOF) mass spectrometer. The observed mass and isotope pattern matched the corresponding theoretical values as calculated from the expected elemental formula.</div><div class="NLM_p">Optical rotations [Î±]<sub>D</sub> were measured on a Bellingham + Stanley ADP450 polarimeter.</div><div class="NLM_p last">The purity of screening compounds <b>11</b>â<b>30</b> was evaluated by NMR spectroscopy and LCMS analysis. All compounds had purity â¥95%.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> Preparation of 1,4-Diketo Esters <b>33</b></h4><div id="sec4_1_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i17" class="anchor-spacer"></div><h5 class="article-section__title" id="_i17"> (General Method 1)</h5><div class="NLM_p">The compound was prepared using General Method 1 adapted from Kazembe et al.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a></div><div class="NLM_p last">Î²-Keto ester <b>31</b> (8.0 mmol, 1.0 equiv) was added to a stirred solution of NaH (385 mg, 60% in mineral oil, 9.6 mmol, 1.2 equiv) in THF (10 mL) at 0 Â°C, and the solution was stirred at 0 Â°C for 30 min. A solution of 2-chloroacetone (0.66 mL, 8.0 mmol, 1.0 equiv) in THF (4 mL) was added dropwise and then NaI (120 mg, 0.8 mmol, 0.1 equiv). The reaction mixture was warmed to RT and stirred for 24 h. The mixture was quenched with dilute HCl (10 mL, 1 M) and extracted with EtOAc (3 Ã 30 mL). The combined organic phases were washed with NaHCO<sub>3</sub> (10 mL, 1 M) and brine (10 mL), dried (MgSO<sub>4</sub>), and concentrated <i>in vacuo</i>. The product was then purified by flash column chromatography on silica gel with gradient elution of EtOAc in cyclohexane.</div></div><div id="sec4_1_2_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i18" class="anchor-spacer"></div><h5 class="article-section__title" id="_i18"> Ethyl 2-Acetyl-4-oxo-pentanoate</h5><div class="NLM_p">The compound was prepared by General Method 1 with ethyl acetoacetate (1.94 mL, 15.4 mmol). Purification by column chromatography (2â20% EtOAc/cyclohexane) gave the product as a yellow oil (859 mg, 4.61 mmol, 30%).</div><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 4.11 (q, <i>J</i> = 7.1 Hz, 2H), 3.93 (dd, <i>J</i> = 8.2, 5.7 Hz, 1H), 3.05 (dd, <i>J</i> = 18.5, 8.2 Hz, 1H), 2.87 (dd, <i>J</i> = 18.5, 5.7 Hz, 1H), 2.27 (s, 3H), 2.11 (s, 3H), 1.19 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): Î´ 205.70, 202.27, 168.87, 61.83, 53.86, 41.64, 30.17, 29.79, 14.12.</div></div><div id="sec4_1_2_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i19" class="anchor-spacer"></div><h5 class="article-section__title" id="_i19"> <i>t</i>-Butyl 2-Acetyl-4-oxo-pentanoate</h5><div class="NLM_p">The compound was prepared by General Method 1 with <i>tert</i>-butyl acetoacetate (2.1 mL, 12.6 mmol). Purification by column chromatography (5â40% EtOAc/cyclohexane) gave the product as a yellow oil (759 mg, 3.54 mmol, 28%).</div><div class="NLM_p last">IR Î½<sub>max</sub> (film): 1735, 1710, 1367, 1357, 1255, 1138 cm<sup>â1</sup>. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): Î´ 3.82 (dd, <i>J</i> = 8.1, 5.8 Hz, 1H), 2.97 (dd, <i>J</i> = 18.4, 8.1 Hz, 1H), 2.80 (dd, <i>J</i> = 18.4, 5.8 Hz, 1H), 2.23 (s, 3H), 2.07 (s, 3H), 1.40â1.33 (s, 9H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): Î´ 205.71, 202.54, 167.83, 82.31, 54.88, 41.53, 29.95, 29.68, 27.85. LCMS (basic method): <i>t</i><sub>R</sub> 1.65 min, <i>m</i>/<i>z</i> 214.3 [M â H]<sup>â</sup>.</div></div><div id="sec4_1_2_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i20" class="anchor-spacer"></div><h5 class="article-section__title" id="_i20"> PaalâKnorr Pyrrole Synthesis <b>35</b>: R = Et (General Method 2)</h5><div class="NLM_p">The compound was prepared using General Method 2 adapted from Kazembe et al.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a></div><div class="NLM_p last">Aniline <b>34</b> (1.2 equiv) and <i>p</i>-toluenesulfonic acid monohydrate (23 mg, 0.12 mmol, 0.1 equiv) were added to a solution of ethyl 2-acetyl-4-oxopentanoate (200 mg, 1.08 mmol, 1 equiv) in EtOH (5 mL), and the reaction mixture was heated at 80 Â°C for 24 h. The reaction was quenched with water (10 mL) and concentrated <i>in vacuo.</i> The residue was dissolved in dilute HCl (10 mL, 2 M) and extracted with EtOAc (3 Ã 15 mL). The combined organics were washed with NaOH (3 Ã 15 mL, 1 M), dried (MgSO<sub>4</sub>), and concentrated <i>in vacuo</i> to give the 3-pyrrole ethyl ester.</div></div><div id="sec4_1_2_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i21" class="anchor-spacer"></div><h5 class="article-section__title" id="_i21"> Hydrolysis of 3-Pyrrole Ethyl Ester (General Method 3)</h5><div class="NLM_p last">Solid lithium hydroxide monohydrate (670 mg, 16 mmol, 15 equiv) was added to a solution of the 3-pyrrole ester (ca. 1.1 mmol, 1.0 equiv) in 1,4-dioxane (4.0 mL) and water (1.2 mL), and the reaction mixture was heated to 100 Â°C for 24 h. The mixture was cooled to RT and quenched with water (5.0 mL). The mixture was acidified with concentrated HCl (1 mL) to pH 1 and extracted with EtOAc (3 Ã 15 mL). The combined organic phases were concentrated <i>in vacuo</i>, and the crude product was purified by reverse-phase flash silica chromatography (0â100% MeCN/H<sub>2</sub>O; 0.1% formic acid modifier) to give the 3-pyrrole acid.</div></div><div id="sec4_1_2_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i22" class="anchor-spacer"></div><h5 class="article-section__title" id="_i22"> PaalâKnorr Pyrrole Synthesis <b>35</b>: R = <i>t</i>-Bu (General Method 4)</h5></div><div id="sec4_1_2_4_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i23" class="anchor-spacer"></div><h5 class="article-section__title" id="_i23"> DMSO as a Solvent (General Method 4.1)</h5><div class="NLM_p last"><i>t</i>-Butyl 2-acetyl-4-oxopentanoate (100 mg, 0.47 mmol) was dissolved in DMSO (2.5 mL) in a thick-walled reaction vial. The aniline <b>34</b> (0.47 mmol) and <i>p</i>-toluenesulfonic acid monohydrate (20 mg, 0.11 mmol) were added, and the vial sealed with a Teflon-lined crimp cap. The vial was heated at 120 Â°C for 30 min under microwave irradiation to give the pyrrole-3-carboxylic acid directly. The crude product was purified by reverse-phase flash silica chromatography (20â100% MeCN/H<sub>2</sub>O; 0.1% formic acid modifier).</div></div><div id="sec4_1_2_4_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i24" class="anchor-spacer"></div><h5 class="article-section__title" id="_i24"> Ethanol as a Solvent (General Method 4.2)</h5><div class="NLM_p last"><i>t</i>-Butyl 2-acetyl-4-oxopentanoate (100 mg, 0.47 mmol) was dissolved in EtOH (5.0 mL) in a thick-walled reaction vial. The aniline <b>34</b> (0.47 mmol) and <i>p</i>-toluenesulfonic acid monohydrate (20 mg, 0.11 mmol) were added, and the vial was sealed with a Teflon-lined crimp cap. The vial was heated at 80 Â°C thermally for 24 h to give the pyrrole-3-carboxylic acid directly. DMSO was added, and the EtOH was evaporated under a flow of compressed air. The crude product was purified by reverse-phase flash silica chromatography (20â100% MeCN/H<sub>2</sub>O; 0.1% formic acid modifier).</div></div><div id="sec4_1_2_4_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i25" class="anchor-spacer"></div><h5 class="article-section__title" id="_i25"> Xylenes as a Solvent (General Method 4.3)</h5><div class="NLM_p last"><i>t</i>-Butyl 2-acetyl-4-oxopentanoate (100 mg, 0.47 mmol) was dissolved in xylenes (5.0 mL) in a thick-walled reaction vial. The aniline <b>34</b> (0.47 mmol) and <i>p</i>-toluenesulfonic acid monohydrate (20 mg, 0.11 mmol) were added, and the vial was sealed with a Teflon-lined crimp cap. The vial was heated at 110 Â°C thermally for 24 h to give the pyrrole-3-carboxylic acid directly . DMSO was added, and the xylenes were evaporated <i>in vacuo</i>. The crude product was purified by reverse-phase flash silica chromatography (20â100% MeCN/H<sub>2</sub>O; 0.1% formic acid modifier).</div></div><div id="sec4_1_2_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i26" class="anchor-spacer"></div><h5 class="article-section__title" id="_i26"> <i>N</i>-Benzylation of 3-Pyrrole Esters (General Method 5)</h5><div class="NLM_p">The compound was prepared using General Method 5 adapted from Beard et al.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></div><div class="NLM_p last">3-Pyrrole ester (0.65 mmol, 1.0 equiv) was added to a solution of NaH (ca. 31 mg, 60% in mineral oil, 0.78 mmol, 1.2 equiv) in THF (2.5 mL), the mixture was stirred at 0 Â°C under nitrogen, and the solution was stirred at 0 Â°C for 15 min. Benzyl bromide (93 Î¼L, 0.78 mmol, 1.2 equiv) was then added, and the reaction was allowed to warm to RT and stirred for 24 h. The reaction was quenched with water (10 mL), extracted with EtOAc (3 Ã 15 mL), washed with brine (2 Ã 15 mL), dried (MgSO<sub>4</sub>), and concentrated <i>in vacuo</i>. The product was purified by column chromatography (0â15% EtOAc/cyclohexane) to give 1-benzyl-3-pyrrole ester.</div></div><div id="sec4_1_2_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i27" class="anchor-spacer"></div><h5 class="article-section__title" id="_i27"> Cross-Coupling of 3-Pyrrolidine Ester <b>36</b> with Aryl Iodides <b>37</b> (General Method 6)</h5></div><div id="sec4_1_2_6_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i28" class="anchor-spacer"></div><h5 class="article-section__title" id="_i28"> BuchwaldâHartwig Amination (General Method 6.1)</h5><div class="NLM_p last">To a thick-walled reaction vial were added methyl pyrrolidine-3-carboxylate (<b>36</b>) (320 mg, 2.0 mmol), tris(dibenzylideneacetone)dipalladium (0) (37 mg, 0.04 mmol), cesium(II) carbonate (1.4 g, 4.4 mmol), and Xantphos (47 mg, 0.08 mmol). The vial was sealed with a Teflon-lined crimp cap and charged with a solution of the aryl iodide <b>37</b> (1.6 mmol) in 1,4-dioxane (7.8 mL). The stirred reaction mixture was degassed with Ar and then heated to 100 Â°C for 4 h. The reaction mixture was cooled to RT and diluted with CH<sub>2</sub>Cl<sub>2</sub>. The resulting suspension was filtered through a plug of sand and cotton wool, and the filtrate was concentrated under a continuous flow of air to give the ester <b>38</b>. This material was used directly in the next step without further purification.</div></div><div id="sec4_1_2_6_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i29" class="anchor-spacer"></div><h5 class="article-section__title" id="_i29"> Alternative BuchwaldâHartwig Amination (General Method 6.2)</h5><div class="NLM_p last">To a thick-walled reaction vial were added 2-dicyclohexylphosphino-2,4,6-triisopropylbiphenyl (18.4 mg, 0.03 mmol), water (0.76 uL, 0.04 mmol), and palladium acetate (2.4 mg, 0.01 mmol). The vial was purged with N<sub>2</sub>, and the reagents were dissolved in 1,4-dioxane (5 mL). This mixture was stirred at 80 Â°C under N<sub>2</sub> until the characteristic change in color was seen (yellow to green). The catalyst mixture was then transferred to a second thick-walled reaction vial containing the aryl iodide <b>37</b> (1.06 mmol), cesium(II) carbonate (933.0 mg, 2.86 mmol, and methyl pyrrolidine-3-carboxylate hydrochloride (<b>36</b>Â·<b>HCl</b>) (211 mg, 1.3 mmol) in 1,4-dioxane (5 mL). The reaction was then stirred at 110 Â°C for 16 h under N<sub>2</sub> until all of the starting material was shown to be consumed by TLC. Reaction workup and purification were the same as in General Method 6.1.</div></div><div id="sec4_1_2_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i30" class="anchor-spacer"></div><h5 class="article-section__title" id="_i30"> Hydrolysis of 3-Pyrrolidine Methyl Ester <b>38</b> (General Method 7)</h5><div class="NLM_p last">To a solution of methyl 1-arylpyrrolidine-3-carboxylate <b>38</b> (0.32 mmol) in MeOH (3 mL) was added aqueous NaOH (0.65 mL, 1 N, 0.65 mmol). The reaction mixture was stirred at RT for 2 h and then quenched with dilute aqueous HCl (1 M, 2 equiv). The sample was concentrated under reduced pressure, redissolved in a minimum of DMSO, and then purified by reverse-phase chromatography (0â95% MeCN/H<sub>2</sub>O; 0.1% formic acid modifier).</div></div><div id="sec4_1_2_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i31" class="anchor-spacer"></div><h5 class="article-section__title" id="_i31"> Preparation of 3-Pyrrolidine Carboxamides <b>28</b> (General Method 8)</h5><div class="NLM_p last">To a thick-walled reaction vial was added methyl 1-[4-chloro-3-(trifluoromethyl)phenyl]pyrrolidine-3-carboxylic acid (<b>25q</b>) (50 mg, 0.17 mmol). Anhydrous DMF (1.0 mL) and hexafluorophosphate azabenzotriazole tetramethyl uronium (HATU) (72 mg, 0.19 mmol) were added, and the reaction mixture was stirred at RT for 10 min. DIPEA (0.07 mL, 0.40 mmol) was added followed by the amine <b>39</b> (0.26 mmol), and the reaction was stirred at RT for 1 h. The reaction mixture was directly purified by reverse-phase chromatography (0â100% MeCN/H<sub>2</sub>O, 0.1% NH<sub>4</sub>OH modifier).</div></div><div id="sec4_1_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32"> 2-Methyl-5-(trifluoromethyl)-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>5</b>)</h5><div class="NLM_p last">The compound was purchased from Enamine, EN300-38888.</div></div><div id="sec4_1_3_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33"> 1-Cyclopropyl-2,5-dimethyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>6</b>)</h5><div class="NLM_p last">The compound was purchased from Enamine, EN300-13229.</div></div><div id="sec4_1_3_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i34" class="anchor-spacer"></div><h5 class="article-section__title" id="_i34"> 1-(Cyclopropylmethyl)-2,5-dimethyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>7</b>)</h5><div class="NLM_p last">The compound was purchased from Enamine, EN300-69261.</div></div><div id="sec4_1_3_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> 1-Phenylpyrrolidine-3-carboxylic Acid (<b>8</b>)</h5><div class="NLM_p last">The compound was purchased from Enamine, EN300-226265.</div></div><div id="sec4_1_3_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> 1-(2-Fluorophenyl)pyrrolidine-3-carboxylic Acid (<b>9</b>)</h5><div class="NLM_p last">The compound was purchased from Enamine, EN300-217084.</div></div><div id="sec4_1_3_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> 1-(2-Chlorophenyl)pyrrolidine-3-carboxylic Acid (<b>10</b>)</h5><div class="NLM_p last">The compound was purchased from Enamine, EN300-218252.</div></div><div id="sec4_1_3_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38"> 1<i>H</i>-Pyrrole-3-carboxylic Acid (<b>11</b>)</h5><div class="NLM_p last">The compound was purchased from Fluorochem, 075258.</div></div><div id="sec4_1_3_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> 2-Methyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>12</b>)</h5><div class="NLM_p last">The compound was purchased from Enamine, EN300-98523.</div></div><div id="sec4_1_3_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40"> 4-Methyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>13</b>)</h5><div class="NLM_p">The compound was available from multiple suppliers [CAS reg. no. 64276-66-0]. Our sample was prepared by General Method 3 from methyl 4-methyl-1<i>H</i>-pyrrole-3-carboxylate and isolated as a white solid (8 mg).</div><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 8.23 (br s, 1H), 7.45 (m, 1H), 6.55 (m, 1H), 2.30 (s, 3H). LCMS (acidic method): <i>t</i><sub>R</sub> 0.54 min, <i>m</i>/<i>z</i> 126.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41"> 5-Methyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>14</b>)</h5><div class="NLM_p last">The compound was purchased from Fluorochem, 091685.</div></div><div id="sec4_1_3_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42"> 5-(Trifluoromethyl)-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>15</b>)</h5><div class="NLM_p last">The compound was purchased from Enamine, EN300-177045.</div></div><div id="sec4_1_3_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> 2,4-Dimethyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>16</b>)</h5><div class="NLM_p">The compound was available from multiple suppliers [CAS reg. no. 17106-13-7]. Our sample was prepared by General Method 3 from ethyl 2,4-dimethyl-1<i>H</i>-pyrrole-3-carboxylate and isolated as a yellow gum (7.4 mg)</div><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 9.39 (br s, 1H), 7.91 (br s, 1H), 6.37 (m, 1H), 2.51 (s, 3H), 2.25 (d, <i>J</i> = 1.1 Hz, 3H). LCMS (acidic method): <i>t</i><sub>R</sub> 0.81 min, <i>m</i>/<i>z</i> 140.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44"> 2,5-Dimethyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>17</b>)</h5><div class="NLM_p last">The compound was purchased from Alfa Aesar, B20635.</div></div><div id="sec4_1_3_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45"> 1,2,5-Trimethyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>18</b>)</h5><div class="NLM_p last">The compound was purchased from Maybridge, CC08601.</div></div><div id="sec4_1_3_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> 2-Methyl-1-phenyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>19</b>)</h5><div class="NLM_p">The compound was prepared by General Method 2 and General Method 3 and isolated as a pale pink solid (33.4 mg, 54%).</div><div class="NLM_p last">Mp: 185 Â°C. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): Î´ 7.52 (t, <i>J</i> = 8.0 Hz, 2H), 7.45 (t, <i>J</i> = 8.0 Hz, 1H), 7.34 (d, <i>J</i> = 8.0 Hz, 2H), 6.74 (d, <i>J</i> = 3.0 Hz, 1H), 6.61 (d, <i>J</i> = 3.0 Hz, 1H), 2.40 (s, 3H). LCMS (acidic method): <i>t</i><sub>R</sub> 1.55 min, <i>m</i>/<i>z</i> 202.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i47" class="anchor-spacer"></div><h5 class="article-section__title" id="_i47"> 2,5-Dimethyl-1-phenyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>20</b>)</h5><div class="NLM_p">The compound was prepared by General Method 2 and General Method 3 and isolated as a pink solid (7.7 mg, 32%).</div><div class="NLM_p last">Mp: 204 Â°C. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 7.53â7.44 (m, 3H), 7.22â7.17 (m, 2H), 6.43 (d, <i>J</i> = 1.0 Hz, 1H), 2.31 (s, 3H), 1.98 (d, <i>J</i> = 0.5 Hz, 3H). LCMS (acidic method): <i>t</i><sub>R</sub> 0.65 min, <i>m</i>/<i>z</i> 216.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48"> 2,5-Dimethyl-1-(2-fluorophenyl)-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>20b</b>)</h5><div class="NLM_p">The compound was prepared by General Method 4.1 with 2-fluoroaniline and isolated as a white solid (70 mg, 64%).</div><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, CDCl<sub>3</sub>): Î´ 7.49â7.45 (m, 1H), 7.31â7.21 (m, 3H), 6.45 (d, <i>J</i> = 0.9 Hz, 1H), 2.31 (s, 3H), 1.98 (s, 3H). <sup>13</sup>C NMR (176 MHz, CDCl<sub>3</sub>): Î´ 170.59, 158.32 (d, <i>J</i> = 251.8 Hz), 138.17, 130.87 (d, <i>J</i> = 7.7 Hz), 130.49, 129.44, 125.42 (d, <i>J</i> = 13.2 Hz), 124.92 (d, <i>J</i> = 4.0 Hz), 117.01 (d, <i>J</i> = 19.9 Hz), 111.30, 108.48, 12.35, 12.25. LCMS (acidic method): <i>t</i><sub>R</sub> 1.63 min, <i>m</i>/<i>z</i> 234.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49"> 2,5-Dimethyl-1-(3-fluorophenyl)-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>20c</b>)</h5><div class="NLM_p">The compound was prepared by General Method 4.1 with 3-fluoroaniline and isolated as a white solid (80 mg, 74%).</div><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, CDCl<sub>3</sub>): Î´ 7.48 (m, 1H), 7.22â7.16 (m, 1H), 7.03â6.99 (m, 1H), 6.95 (dt, <i>J</i> = 9.0, 2.1 Hz, 1H), 6.43 (s, 1H), 2.32 (s, 3H), 2.00 (s, 3H). <sup>13</sup>C NMR (176 MHz, CDCl3): Î´ 170.68, 162.96 (d, <i>J</i> = 249.3 Hz), 139.18 (d, <i>J</i> = 9.6 Hz), 137.62, 130.78 (d, <i>J</i> = 9.1 Hz), 129.11, 124.26 (d, <i>J</i> = 3.2 Hz), 116.05 (d, <i>J</i> = 19.9 Hz), 115.87, 111.09, 108.51, 12.77, 12.63. LCMS (acidic method): <i>t</i><sub>R</sub> 1.70 min, <i>m</i>/<i>z</i> 234.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_19" class="NLM_sec NLM_sec_level_4"><div id="ac_i50" class="anchor-spacer"></div><h5 class="article-section__title" id="_i50"> 2,5-Dimethyl-1-(4-fluorophenyl)-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>20d</b>)</h5><div class="NLM_p">The compound was prepared by General Method 4.1 with 4-fluoroaniline and isolated as an off-white solid (60 mg, 55%).</div><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 7.18 (m, 4H), 6.42 (d, <i>J</i> = 0.8 Hz, 1H), 2.29 (s, 3H), 1.97 (s, 3H). <sup>13</sup>C NMR (151 MHz, CDCl3): Î´ 171.11, 162.51 (d, <i>J</i> = 249.0 Hz), 137.88, 133.69 (d, <i>J</i> = 3.0 Hz), 130.02 (d, <i>J</i> = 8.6 Hz), 129.32, 116.61 (d, <i>J</i> = 22.9 Hz), 110.90, 108.26, 12.83, 12.68. LCMS (acidic method): <i>t</i><sub>R</sub> 1.71 min, <i>m</i>/<i>z</i> 234.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_20" class="NLM_sec NLM_sec_level_4"><div id="ac_i51" class="anchor-spacer"></div><h5 class="article-section__title" id="_i51"> 2,5-Dimethyl-1-(2-methylphenyl)-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>20e</b>)</h5><div class="NLM_p">The compound was prepared by General Method 4.2 with <i>o</i>-toluidine and isolated as a light green solid (51 mg, 48%).</div><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, CDCl<sub>3</sub>): Î´ 7.35 (m, 3H), 7.12 (d, <i>J</i> = 7.5 Hz, 1H), 6.45 (d, <i>J</i> = 0.8 Hz, 1H), 2.22 (s, 3H), 1.95 (s, 3H), 1.89 (s, 3H). <sup>13</sup>C NMR (176 MHz, CDCl<sub>3</sub>): Î´ 170.87, 137.36, 136.84, 136.72, 131.15, 129.24, 128.69, 128.63, 127.13, 110.65, 108.08, 17.19, 12.42, 12.29. LCMS (acidic method): <i>t</i><sub>R</sub> 1.75 min, <i>m</i>/<i>z</i> 230.2 [M + H]<sup>+</sup></div></div><div id="sec4_1_3_21" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52"> 2,5-Dimethyl-1-(3-methylphenyl)-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>20f</b>)</h5><div class="NLM_p">The compound was prepared by General Method 4.3 with <i>m</i>-toluidine and isolated as light gray solid (97 mg, 57%).</div><div class="NLM_p last">Mp: 195â197 Â°C. IR Î½<sub>max</sub> (film) 1653, 1606, 1265, 1244, 771, 729 cm<sup>â1</sup>. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): Î´ 7.37 (t, <i>J</i> = 7.7 Hz, 1H), 7.27 (s, 1H), 6.99 (d, <i>J</i> = 8.4 Hz, 2H), 6.40 (d, <i>J</i> = 1.0 Hz, 1H), 2.42 (s, 3H), 2.30 (s, 3H), 1.97 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): Î´ 169.44, 139.62, 137.75, 129.48, 129.26, 128.80, 125.25, 110.28, 108.01, 56.10, 32.10, 21.38, 12.78, 12.64. LCMS (acidic method): <i>t</i><sub>R</sub> 2.04 min, <i>m</i>/<i>z</i> 230.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_22" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53"> 2,5-Dimethyl-1-(4-methylphenyl)-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>20g</b>)</h5><div class="NLM_p">The compound was prepared by General Method 2 and General Method 3 with <i>p</i>-toluidine and isolated as a white solid (123 mg, 37%).</div><div class="NLM_p last">Mp: 246 Â°C. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): Î´ 7.29 (d, <i>J</i> = 8.0 Hz, 2H), 7.07 (d, <i>J</i> = 8.0 Hz, 2H), 6.41 (s, 1H), 2.43 (s, 3H), 2.30 (s, 3H), 1.97 (s, 3H). LCMS (acidic method): <i>t</i><sub>R</sub> 1.68 min, <i>m</i>/<i>z</i> 230.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_23" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54"> 1-(2-Chlorophenyl)-2,5-dimethyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>20h</b>)</h5><div class="NLM_p">The compound was prepared by General Method 4.2 with 2-chloroaniline and isolated as a white solid (60 mg, 52%).</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 11.73 (s, 1H), 7.77â7.70 (m, 1H), 7.63â7.45 (m, 3H), 6.25 (d, <i>J</i> = 1.0 Hz, 1H), 2.11 (s, 3H), 1.85 (s, 3H). <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 166.23, 134.96, 134.72, 132.20, 131.07, 130.84, 130.33, 128.66, 127.82, 111.81, 107.79, 11.93, 11.66. LCMS (acidic method): <i>t</i><sub>R</sub> 1.73 min, <i>m</i>/<i>z</i> 250.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_24" class="NLM_sec NLM_sec_level_4"><div id="ac_i55" class="anchor-spacer"></div><h5 class="article-section__title" id="_i55"> 1-(3-Chlorophenyl)-2,5-dimethyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>20i</b>)</h5><div class="NLM_p">The compound was prepared by General Method 2 and General Method 3 with 3-chloroaniline and isolated as an amber oil (206 mg, 76%).</div><div class="NLM_p last">IR Î½<sub>max</sub> (film): 3346, 3069â3087, 2834â2990, 2582, 1646, 1378â1594, 1330, 1250, 776 cm<sup>â1</sup>. <sup>1</sup>H NMR (700 MHz, CDCl<sub>3</sub>): Î´ 7.45 (m, 2H), 7.22 (s, 1H), 7.10 (d, <i>J</i> = 8.0 Hz, 1H), 6.40 (s, 1H), 2.31 (s, 3H), 1.99 (s, 3H). <sup>13</sup>C NMR (176 MHz, CDCl<sub>3</sub>): Î´ 169.22, 138.86, 137.70, 135.22, 130.57, 129.20, 128.63, 126.64, 110.67, 108.49, 67.22, 12.79, 12.63. HRMS: calcd for C<sub>13</sub>H<sub>12</sub>ClNO<sub>2</sub>, 250.0635 [M + H]<sup>+</sup>; found, 250.0637.</div></div><div id="sec4_1_3_25" class="NLM_sec NLM_sec_level_4"><div id="ac_i56" class="anchor-spacer"></div><h5 class="article-section__title" id="_i56"> 1-(4-Chlorophenyl)-2,5-dimethyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>20j</b>)</h5><div class="NLM_p">The compound was prepared by General Method 2 and General Method 3 with 4-chloroaniline and isolated as a white solid (34 mg, 55%).</div><div class="NLM_p last">Mp: 236 Â°C. <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 7.48 (d, <i>J</i> = 9.0 Hz, 2H), 7.14 (d, <i>J</i> = 9.0 Hz, 2H), 6.42 (s, 1H), 2.31 (s, 3H), 1.98 (s, 3H). LCMS (basic method): <i>t</i><sub>R</sub> 1.26 min, <i>m</i>/<i>z</i> 248.1 [M â H]<sup>â</sup>.</div></div><div id="sec4_1_3_26" class="NLM_sec NLM_sec_level_4"><div id="ac_i57" class="anchor-spacer"></div><h5 class="article-section__title" id="_i57"> 2,5-Dimethyl-1-(3-(trifluoromethyl)phenyl)-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>20k</b>)</h5><div class="NLM_p">The compound was prepared by General Method 2 and General Method 3 with 3-(trifluoromethyl)aniline and isolated as a white solid (71 mg, 70%).</div><div class="NLM_p last">Mp: 243 Â°C. IR Î½<sub>max</sub> (powder) 3460, 2582â3080, 2857â3080, 1739, 1435â1650, 1320, 1227, 1030â1163 cm<sup>â1</sup>. <sup>1</sup>H NMR (700 MHz, methanol-<i>d</i><sub>4</sub>): Î´ 7.80 (d, <i>J</i> = 8.0 Hz, 1H), 7.75 (t, <i>J</i> = 8.0 Hz, 1H), 7.58 (s, 1H), 7.53 (d, <i>J</i> = 8.0 Hz, 1H), 6.32 (s, 1H), 2.25 (s, 3H), 1.96 (s, 3H). <sup>13</sup>C NMR (175 MHz, methanol-<i>d</i><sub>4</sub>): Î´ 150.9, 140.3, 135.3, 133.4, 132.8 (q, <i>J</i> = 32 Hz), 131.5, 128.8, 126.3 (q, <i>J</i> = 3.8 Hz), 126.2 (q, <i>J</i> = 3.8 Hz), 125.0 (q, <i>J</i> = 270 Hz), 109.8, 106.9, 12.6, 12.5. HRMS: calcd for C<sub>14</sub>H<sub>12</sub>F<sub>3</sub>NO<sub>2</sub>, 284.0898 [M + H]<sup>+</sup>; found, 284.0893.</div></div><div id="sec4_1_3_27" class="NLM_sec NLM_sec_level_4"><div id="ac_i58" class="anchor-spacer"></div><h5 class="article-section__title" id="_i58"> 1-(3-Cyclopropylphenyl)-2,5-dimethyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>20l</b>)</h5><div class="NLM_p">The compound was prepared by General Method 4.2 with 3-cyclopropylaniline and isolated as a solid (79 mg, 33%).</div><div class="NLM_p last">Mp: 120â122 Â°C. IR Î½<sub>max</sub> (film): 1651, 1533, 1490, 1427, 1259, 781, 474 cm<sup>â1</sup>. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): Î´ 7.36 (t, <i>J</i> = 7.8 Hz, 1H), 7.15 (d, <i>J</i> = 7.8 Hz, 1H), 6.95 (ddd, <i>J</i> = 7.7, 2.0, 1.0 Hz, 1H), 6.86 (t, <i>J</i> = 1.9 Hz, 1H), 6.40 (d, <i>J</i> = 1.0 Hz, 1H), 2.30 (s, 3H), 1.97 (s, 3H), 1.94 (ddd, <i>J</i> = 13.4, 8.4, 4.9 Hz, 1H), 1.04 (dd, <i>J</i> = 8.4, 1.7 Hz, 2H), 0.72 (dd, <i>J</i> = 4.9, 1.6 Hz, 2H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) 174.4, 149.4, 146.1, 137.7, 129.2, 129.2, 125.9, 125.3, 125.0, 108.0, 100.5, 15.3, 12.7, 12.6, 9.9. LCMS (acidic method): <i>t</i><sub>R</sub> 1.73 min, <i>m</i>/<i>z</i> 256.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_28" class="NLM_sec NLM_sec_level_4"><div id="ac_i59" class="anchor-spacer"></div><h5 class="article-section__title" id="_i59"> 2,5-Dimethyl-1-(3-isopropylphenyl)-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>20m</b>)</h5><div class="NLM_p">The compound was prepared by General Method 4.2 with 3-isopropylaniline and solated as an off-white solid (51 mg, 43%).</div><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, CDCl<sub>3</sub>): Î´ 7.40 (t, <i>J</i> = 7.7 Hz, 1H), 7.31 (d, <i>J</i> = 7.7 Hz, 1H), 7.04 (s, 1H), 7.00 (d, <i>J</i> = 7.7 Hz, 1H), 6.42 (s, 1H), 2.97 (hept, <i>J</i> = 6.9 Hz, 1H), 2.31 (s, 3H), 1.98 (s, 3H), 1.28 (d, <i>J</i> = 6.9 Hz, 6H). <sup>13</sup>C NMR (176 MHz, CDCl<sub>3</sub>): Î´ 170.1, 150.7, 137.7, 137.6, 129.3, 129.2, 126.8, 126.2, 125.4, 110.4, 108.0, 34.0, 24.0, 12.8, 12.7. LCMS (acidic method): <i>t</i><sub>R</sub> 1.85 min, <i>m</i>/<i>z</i> 258.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_29" class="NLM_sec NLM_sec_level_4"><div id="ac_i60" class="anchor-spacer"></div><h5 class="article-section__title" id="_i60"> 1-(3-(<i>tert-</i>Butyl)phenyl)-2,5-dimethyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>20n</b>)</h5><div class="NLM_p">The compound was prepared by General Method 4.1 with 3-<i>tert</i>-butylaniline and isolated as a white solid (84 mg, 66%).</div><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, CDCl<sub>3</sub>): Î´ 7.47 (ddd, <i>J</i> = 7.6, 1.8, 1.0 Hz, 1H), 7.41 (t, <i>J</i> = 7.8 Hz, 1H), 7.19 (t, <i>J</i> = 1.8 Hz, 1H), 7.00 (ddd, <i>J</i> = 7.6, 1.8, 1.0 Hz, 1H), 6.43 (d, <i>J</i> = 1.0 Hz, 1H), 2.32 (s, 3H), 1.99 (s, 3H), 1.34 (s, 9H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 170.9, 153.0, 137.8, 137.4, 129.2, 129.0, 125.5, 125.4, 125.1, 110.5, 108.0, 34.9, 31.4, 12.9, 12.7. LCMS (acidic method): <i>t</i><sub>R</sub> 1.88 min, <i>m</i>/<i>z</i> 272.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_30" class="NLM_sec NLM_sec_level_4"><div id="ac_i61" class="anchor-spacer"></div><h5 class="article-section__title" id="_i61"> 2,5-Dimethyl-1-(3-methoxyphenyl)-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>20o</b>)</h5><div class="NLM_p">The compound was prepared by General Method 4.1 with <i>m</i>-anisidine and isolated as a brown solid (56 mg, 49%).</div><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, CDCl<sub>3</sub>): Î´ 7.39 (t, <i>J</i> = 8.0 Hz, 1H), 7.00 (d, <i>J</i> = 8.0 Hz, 1H), 6.78 (d, <i>J</i> = 8.0 Hz, 1H), 6.72 (d, <i>J</i> = 1.9 Hz, 1H), 6.42 (s, 1H), 3.84 (s, 3H), 2.32 (s, 3H), 2.00 (s, 3H). <sup>13</sup>C NMR (176 MHz, CDCl<sub>3</sub>): Î´ 170.7, 160.4, 138.8, 137.7, 130.2, 129.2, 120.5, 114.4, 114.0, 110.6, 108.1, 55.6, 12.7, 12.6. LCMS (acidic method): <i>t</i><sub>R</sub> 1.65 min, <i>m</i>/<i>z</i> 246.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_31" class="NLM_sec NLM_sec_level_4"><div id="ac_i62" class="anchor-spacer"></div><h5 class="article-section__title" id="_i62"> 2,5-Dimethyl-1-(3-ethoxyphenyl)-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>20p</b>)</h5><div class="NLM_p">The compound was prepared by General Method 4.1 with 3-ethoxyaniline and isolated as an off-white solid (87 mg, 72%).</div><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, CDCl<sub>3</sub>): Î´ 7.38 (t, <i>J</i> = 8.1 Hz, 1H), 6.98 (dd, <i>J</i> = 8.1, 2.4 Hz, 1H), 6.76 (d, <i>J</i> = 8.1 Hz, 1H), 6.71 (t, <i>J</i> = 2.1 Hz, 1H), 6.41 (s, 1H), 4.05 (q, <i>J</i> = 7.0 Hz, 2H), 2.32 (s, 3H), 2.00 (s, 3H), 1.44 (t, <i>J</i> = 7.0 Hz, 3H). <sup>13</sup>C NMR (176 MHz, CDCl<sub>3</sub>): Î´ 170.7, 159.8, 138.7, 137.7, 130.1, 129.2, 120.3, 115.0, 114.4, 110.6, 108.0, 63.9, 14.8, 12.7, 12.6. LCMS (acidic method): <i>t</i><sub>R</sub> 1.73 min, <i>m</i>/<i>z</i> 260.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_32" class="NLM_sec NLM_sec_level_4"><div id="ac_i63" class="anchor-spacer"></div><h5 class="article-section__title" id="_i63"> 1-(3-cyanophenyl)-2,5-dimethyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>20q</b>)</h5><div class="NLM_p">The compound was prepared by General Method 4.2 with 3-cyanoaniline and isolated as an off-white solid (200 mg, 72%).</div><div class="NLM_p last">Mp: 201â203 Â°C. IR Î½<sub>max</sub> (film): 1645, 1533, 1261, 1238, 775, 727 cm<sup>â1</sup>. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): Î´ 7.80â7.77 (m, 1H), 7.66 (t, <i>J</i> = 7.9 Hz, 1H), 7.54 (t, <i>J</i> = 1.6 Hz, 1H), 7.48 (ddd, <i>J</i> = 8.0, 2.0, 1.1 Hz, 1H), 6.45 (d, <i>J</i> = 1.0 Hz, 1H), 2.31 (s, 3H), 1.99 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): Î´ 169.6, 138.7, 137.3, 132.9, 132.4, 131.8, 130.7, 128.9, 117.5, 114.1, 111.5, 109.0, 12.8, 12.6. LCMS (acidic method): <i>t</i><sub>R</sub> 1.51 min, <i>m</i>/<i>z</i> 241.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_33" class="NLM_sec NLM_sec_level_4"><div id="ac_i64" class="anchor-spacer"></div><h5 class="article-section__title" id="_i64"> 1-(2,3-Dichlorophenyl)-2,5-dimethyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>20r</b>)</h5><div class="NLM_p">The compound was prepared by General Method 4.2 with 2,3-dichloroaniline and solated as an off-white solid (150 mg, 71%).</div><div class="NLM_p last">Mp: 246â248 Â°C. IR Î½<sub>max</sub> (film): 1647, 1423, 1265, 1238, 783, 729 cm<sup>â1</sup>. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): Î´ 7.62 (dd, <i>J</i> = 8.2, 1.5 Hz, 1H), 7.36 (t, <i>J</i> = 8.0 Hz, 1H), 7.21 (dd, <i>J</i> = 7.8, 1.5 Hz, 1H), 6.45 (d, <i>J</i> = 1.0 Hz, 1H), 2.25 (s, 3H), 1.93 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): Î´ 170.7, 144.2, 134.5, 132.9, 131.3, 128.8, 128.6, 127.8, 116.5, 111.0, 108.4, 12.2, 12.1. LCMS (acidic method): <i>t</i><sub>R</sub> 1.70 min, <i>m</i>/<i>z</i> 284.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_34" class="NLM_sec NLM_sec_level_4"><div id="ac_i65" class="anchor-spacer"></div><h5 class="article-section__title" id="_i65"> 1-(2,4-Dichlorophenyl)-2,5-dimethyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>20s</b>)</h5><div class="NLM_p">The compound was prepared by General Method 4.1 with 2,4-dichloroaniline and isolated as a white solid (70 mg, 53%).</div><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, CDCl<sub>3</sub>): Î´ 7.60 (d, <i>J</i> = 2.3 Hz, 1H), 7.41 (dd, <i>J</i> = 8.4, 2.3 Hz, 1H), 7.22 (d, <i>J</i> = 8.4 Hz, 1H), 6.45 (d, <i>J</i> = 0.9 Hz, 1H), 2.25 (s, 3H), 1.93 (s, 3H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): Î´ 170.7, 137.7, 135.9, 134.7, 134.2, 131.1, 130.5, 129.0, 128.3, 111.3, 108.5, 12.3, 12.2. LCMS (acidic method): <i>t</i><sub>R</sub> 1.79 min, <i>m</i>/<i>z</i> 284.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_35" class="NLM_sec NLM_sec_level_4"><div id="ac_i66" class="anchor-spacer"></div><h5 class="article-section__title" id="_i66"> 1-(2,5-Dichlorophenyl)-2,5-dimethyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>20t</b>)</h5><div class="NLM_p">The compound was prepared by General Method 4.2 with 2,5-dichloroaniline and isolated as a white solid (185 mg, 88%).</div><div class="NLM_p last">IR Î½<sub>max</sub> (film): 3356, 1689, 1473, 1249, 634, 455 cm<sup>â1</sup>. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): Î´ 7.52 (d, <i>J</i> = 8.6 Hz, 1H), 7.44 (dd, <i>J</i> = 8.6, 2.5 Hz, 1H), 7.31 (d, <i>J</i> = 2.4 Hz, 1H), 6.44 (d, <i>J</i> = 1.0 Hz, 1H), 2.27 (s, 3H), 1.95 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): Î´ 169.4, 137.5, 136.6, 133.4, 132.3, 131.3, 130.7, 130.5, 128.8, 111.2, 108.5, 12.2, 12.1. LCMS (acidic method): <i>t</i><sub>R</sub> 1.35 min, <i>m</i>/<i>z</i> 284.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_36" class="NLM_sec NLM_sec_level_4"><div id="ac_i67" class="anchor-spacer"></div><h5 class="article-section__title" id="_i67"> 1-(2,6-Dichlorophenyl)-2,5-dimethyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>20u</b>)</h5><div class="NLM_p">The compound was prepared by General Method 4.2 with 2,6-dichloroaniline and isolated as a white solid (10 mg, 8%).</div><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): Î´ 7.54â7.47 (m, 2H), 7.39 (dd, <i>J</i> = 8.8, 7.4 Hz, 1H), 6.49 (d, <i>J</i> = 1.0 Hz, 1H), 2.25 (s, 3H), 1.93 (s, 3H). <sup>13</sup>C NMR (176 MHz, CDCl<sub>3</sub>): Î´ 169.6, 137.2, 135.7, 133.7, 130.8, 128.8, 128.3, 111.4, 108.6, 11.9, 11.7. LCMS (acidic method): <i>t</i><sub>R</sub> 1.72 min, <i>m</i>/<i>z</i> 284.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_37" class="NLM_sec NLM_sec_level_4"><div id="ac_i68" class="anchor-spacer"></div><h5 class="article-section__title" id="_i68"> 1-(3,4-Dichlorophenyl)-2,5-dimethyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>20v</b>)</h5><div class="NLM_p">The compound was prepared by General Method 4.2 with 3,4-dichloraniline and solated as an off-white solid (53 mg, 40%).</div><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, CDCl<sub>3</sub>): Î´ 7.59 (d, <i>J</i> = 8.4 Hz, 1H), 7.34 (d, <i>J</i> = 2.4 Hz, 1H), 7.08 (dd, <i>J</i> = 8.4, 2.4 Hz, 1H), 6.42 (d, <i>J</i> = 0.8 Hz, 1H), 2.32 (s, 3H), 2.00 (s, 3H). <sup>13</sup>C NMR (176 MHz, CDCl<sub>3</sub>): Î´ 170.1, 137.5, 137.0, 133.6, 133.4, 131.3, 130.3, 129.0, 127.7, 111.3, 108.7, 12.8, 12.6. LCMS (acidic method): <i>t</i><sub>R</sub> 1.82 min, <i>m</i>/<i>z</i> 284.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_38" class="NLM_sec NLM_sec_level_4"><div id="ac_i69" class="anchor-spacer"></div><h5 class="article-section__title" id="_i69"> 1-(3,5-Dichlorophenyl)-2,5-dimethyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>20w</b>)</h5><div class="NLM_p">The compound was prepared by General Method 4.2 with 3,5-dichloroaniline and isolated as a light pink solid (139 mg, 52%).</div><div class="NLM_p last">Mp: 215â217 Â°C. IR Î½<sub>max</sub> (film): 1654, 1566, 1537, 1255, 1238, 418 cm<sup>â1</sup>. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): Î´ 7.49 (t, <i>J</i> = 1.9 Hz, 1H), 7.14 (d, <i>J</i> = 1.9 Hz, 2H), 6.41 (d, <i>J</i> = 1.0 Hz, 1H), 2.33 (s, 3H), 2.00 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): Î´ 170.9, 139.6, 137.3, 135.8, 129.3, 128.9, 127.1, 116.8, 111.3, 108.8, 12.7, 12.6. LCMS (acidic method): <i>t</i><sub>R</sub> 1.78 min, <i>m</i>/<i>z</i> 284.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_39" class="NLM_sec NLM_sec_level_4"><div id="ac_i70" class="anchor-spacer"></div><h5 class="article-section__title" id="_i70"> 1-(3-Chloro-2-fluorophenyl)-2,5-dimethyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>20x</b>)</h5><div class="NLM_p">The compound was prepared by General Method 4.2 with 3-chloro-2-fluoroaniline and isolated as an off-white solid (97 mg, 39%).</div><div class="NLM_p last">Mp: 230â232 Â°C. IR Î½<sub>max</sub> (film): 1653, 1460, 1273, 1234, 1087, 781 cm<sup>â1</sup>. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): Î´ 7.54 (ddd, <i>J</i> = 8.2, 6.6, 1.7 Hz, 1H), 7.23 (td, <i>J</i> = 8.1, 1.4 Hz, 2H), 7.15 (ddd, <i>J</i> = 8.1, 6.6, 1.7 Hz, 1H), 6.45 (d, <i>J</i> = 1.0 Hz, 1H), 2.31 (s, 3H), 1.98 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): Î´ 170.1, 154.4 (d, <i>J</i> = 253.9 Hz), 137.9, 131.5, 129.3, 128.8, 126.8 (d, <i>J</i> = 13.3 Hz), 124.8 (d, <i>J</i> = 5.3 Hz), 122.8 (d, <i>J</i> = 16.8 Hz), 111.6, 108.8, 12.3, 12.2. LCMS (acidic method): <i>t</i><sub>R</sub> 2.01 min, <i>m</i>/<i>z</i> 268.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_40" class="NLM_sec NLM_sec_level_4"><div id="ac_i71" class="anchor-spacer"></div><h5 class="article-section__title" id="_i71"> 1-(3-Cyclopropyl-2-fluorophenyl)-2,5-dimethyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>20y</b>)</h5><div class="NLM_p">The compound was prepared by General Method 4.2 with 3-cyclopropyl-2-fluoroaniline and isolated as a white solid (70 mg, 55%).</div><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, CDCl<sub>3</sub>): Î´ 7.14 (t, <i>J</i> = 7.8 Hz, 1H), 6.99 (dd, <i>J</i> = 15.0, 7.8 Hz, 2H), 6.45 (s, 1H), 2.31 (s, 3H), 2.20â2.10 (m, 1H), 1.98 (s, 3H), 1.07 (m, 2H), 0.84â0.76 (m, 2H). <sup>13</sup>C NMR (176 MHz, CDCl<sub>3</sub>): Î´ 170.3, 157.2 (d, <i>J</i> = 250.1 Hz), 138.2, 132.9 (d, <i>J</i> = 13.5 Hz), 129.4, 126.9, 126.5 (d, <i>J</i> = 4.2 Hz), 125.0 (d, <i>J</i> = 14.1 Hz), 124.2 (d, <i>J</i> = 4.6 Hz), 111.0, 108.3, 12.3, 12.3, 8.8 (d, <i>J</i> = 5.5 Hz), 8.72 (d, <i>J</i> = 3.8 Hz). LCMS (acidic method): <i>t</i><sub>R</sub> 1.78 min, <i>m</i>/<i>z</i> 274.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_41" class="NLM_sec NLM_sec_level_4"><div id="ac_i72" class="anchor-spacer"></div><h5 class="article-section__title" id="_i72"> 1-(2-Fluoro-3-isopropylphenyl)-2,5-dimethyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>20z</b>)</h5><div class="NLM_p">The compound was prepared by General Method 4.1 with 2-fluoro-3-isopropylaniline and isolated as an off-white solid (80 mg, 62%).</div><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, CDCl<sub>3</sub>): Î´ 7.37 (t, <i>J</i> = 6.7 Hz, 1H), 7.21 (m, 1H), 7.05 (t, <i>J</i> = 6.7 Hz, 1H), 6.45 (s, 1H), 3.29 (m, 1H), 2.30 (s, 3H), 1.98 (s, 3H), 1.30 (dd, <i>J</i> = 6.9, 3.0 Hz, 6H). <sup>13</sup>C NMR (176 MHz, CDCl<sub>3</sub>): Î´ 170.6 (d, <i>J</i> = 4.9 Hz), 156.0 (d, <i>J</i> = 250.2 Hz), 138.1, 137.3 (d, <i>J</i> = 13.9 Hz), 129.4, 128.0 (d, <i>J</i> = 5.2 Hz), 127.4, 125.2 (d, <i>J</i> = 14.5 Hz), 124.4 (d, <i>J</i> = 4.6 Hz), 111.1, 108.3, 27.4 (d, <i>J</i> = 2.0 Hz), 22.7 (d, <i>J</i> = 6.9 Hz), 12.3, 12.2. LCMS (acidic method): <i>t</i><sub>R</sub> 1.83 min, <i>m</i>/<i>z</i> 276.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_42" class="NLM_sec NLM_sec_level_4"><div id="ac_i73" class="anchor-spacer"></div><h5 class="article-section__title" id="_i73"> 1-(3-Cyano-2-fluorophenyl)-2,5-dimethyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>20aa</b>)</h5><div class="NLM_p">The compound was prepared by General Method 4.2 with 3-cyano-2-fluoroaniline and isolated as an off-white solid (58 mg, 48%).</div><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, CDCl<sub>3</sub>): Î´ 7.78 (ddd, <i>J</i> = 7.9, 5.6, 1.7 Hz, 1H), 7.53 (td, <i>J</i> = 7.9, 1.7 Hz, 1H), 7.44 (t, <i>J</i> = 7.9 Hz, 1H), 6.48 (d, <i>J</i> = 1.0 Hz, 1H), 2.31 (s, 3H), 1.99 (s, 3H). <sup>13</sup>C NMR (176 MHz, CDCl<sub>3</sub>): Î´ 170.1, 159.0 (d, <i>J</i> = 264.4 Hz), 137.7, 135.6, 134.2, 129.2, 126.8 (d, <i>J</i> = 11.9 Hz), 125.7 (d, <i>J</i> = 4.9 Hz), 112.9, 112.2, 109.3, 103.6 (d, <i>J</i> = 14.6 Hz), 12.3, 12.2. LCMS (acidic method): <i>t</i><sub>R</sub> 1.62 min, <i>m</i>/<i>z</i> 259.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_43" class="NLM_sec NLM_sec_level_4"><div id="ac_i74" class="anchor-spacer"></div><h5 class="article-section__title" id="_i74"> 1-(3-Chloro-2-methoxyphenyl)-2,5-dimethyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>20bb</b>)</h5><div class="NLM_p">The compound was prepared by General Method 4.2 with 3-chloro-2-methoxyaniline and isolated as an off-white solid (124 mg, 62%).</div><div class="NLM_p last">Mp: 208â210 Â°C. IR Î½<sub>max</sub> (film): 1653, 1533, 1271, 1240, 783, 433 cm<sup>â1</sup>. <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): Î´ 7.51 (dd, <i>J</i> = 8.1, 1.7 Hz, 1H), 7.16 (t, <i>J</i> = 8.0 Hz, 1H), 7.08 (dd, <i>J</i> = 7.9, 1.7 Hz, 1H), 6.45 (d, <i>J</i> = 1.0 Hz, 1H), 3.47 (s, 3H), 2.32 (s, 3H), 1.99 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): Î´ 169.9, 152.9, 138.0, 131.8, 131.2, 129.5, 129.1, 128.7, 124.4, 111.1, 108.5, 60.4, 12.5, 12.4. LCMS (acidic method): <i>t</i><sub>R</sub> 1.69 min, <i>m</i>/<i>z</i> 280.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_44" class="NLM_sec NLM_sec_level_4"><div id="ac_i75" class="anchor-spacer"></div><h5 class="article-section__title" id="_i75"> 1-(3-Chloro-2-cyanophenyl)-2,5-dimethyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>20cc</b>)</h5><div class="NLM_p">The compound was prepared by General Method 4.2 with 3-chloro-2-cyanoaniline and isolated as an off-white solid (59 mg, 30%).</div><div class="NLM_p last">Mp: 242â244 Â°C. IR Î½<sub>max</sub> (film): 1656, 1533, 1421, 1263, 1238, 783 cm<sup>â1</sup>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 11.90 (br s, 1H), 7.98â7.91 (m, 2H), 7.65 (dd, <i>J</i> = 7.2, 1.8 Hz, 1H), 6.30 (app d, <i>J</i> = 1.0 Hz, 1H), 2.20 (s, 3H), 1.93 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 171.9, 151.2, 141.6, 136.0, 131.1, 129.5 (2C), 128.5, 113.6, 113.4, 111.3, 108.9, 12.2, 11.9. LCMS (acidic method): <i>t</i><sub>R</sub> 1.57 min, <i>m</i>/<i>z</i> 275.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_45" class="NLM_sec NLM_sec_level_4"><div id="ac_i76" class="anchor-spacer"></div><h5 class="article-section__title" id="_i76"> 1-(3-Chloro-4-(trifluoromethyl)phenyl)-2,5-dimethyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>20dd</b>)</h5><div class="NLM_p">The compound was prepared by General Method 2 and General Method 3 with 3-chloro-4-(trifluoromethyl)aniline and isolated as a white solid (102 mg, 59%).</div><div class="NLM_p last">Mp: 186 Â°C. IR Î½<sub>max</sub> (powder) 3459, 2584â3068, 1739, 1441â1645, 1313, 1228, 1029, 776 cm<sup>â1</sup>. <sup>1</sup>H NMR (700 MHz, methanol-<i>d</i><sub>4</sub>): Î´ 7.96 (d, <i>J</i> = 8.3 Hz, 1H), 7.64 (s, 1H), 7.42 (d, <i>J</i> = 8.3 Hz, 1H), 6.34 (s, 1H), 2.29 (s, 3H), 2.00 (s, 3H). <sup>13</sup>C NMR (176 Hz, methanol-<i>d</i><sub>4</sub>): Î´ 170.2, 143.5, 136.4 (m), 134.1, 132.6, 129.8 (q, <i>J</i> = 5 Hz), 129.4, 129.2 (q, <i>J</i> = 31 Hz), 128.6, 123.9 (q, <i>J</i> = 272 Hz), 110.7, 109.9, 12.5, 12.4. HRMS: calcd for C<sub>14</sub>H<sub>11</sub>ClF<sub>3</sub>NO<sub>2</sub>, 316.0352 [M âH]<sup>â</sup>; found, 316.0352.</div></div><div id="sec4_1_3_46" class="NLM_sec NLM_sec_level_4"><div id="ac_i77" class="anchor-spacer"></div><h5 class="article-section__title" id="_i77"> 1-(4-Chloro-3-(trifluoromethyl)phenyl)-2,5-dimethyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>20ee</b>)</h5><div class="NLM_p">The compound was prepared by General Method 2 and General Method 3 with 4-chloro-3-(trifluoromethyl)aniline and isolated as a white solid (21 mg, 49%).</div><div class="NLM_p last">Mp: 178 Â°C. IR: Î½<sub>max</sub> (powder) 3460, 3080, 2945â3004, 1739, 1324â1533, 1217, 1109, 1035, 794 cm<sup>â1</sup>. <sup>1</sup>H NMR (700 MHz, Methanol-<i>d</i><sub>4</sub>): Î´ 7.77 (d, <i>J</i> = 8.3 Hz, 1H), 7.66 (s, 1H), 7.51 (d, <i>J</i> = 8.3 Hz, 1H), 6.29 (s, 1H), 2.25 (s, 3H), 1.96 (s, 3H). <sup>13</sup>C NMR (176 MHz, CDCl<sub>3</sub>): Î´ 169.3, 137.4, 136.5, 133.1 (m), 132.9, 132.7, 129.9 (q, <i>J</i> = 32 Hz), 129.0, 127.5 (q, <i>J</i> = 5 Hz), 122.2 (q, <i>J</i> = 272 Hz), 111.4, 109.0, 12.8, 12.6. HRMS: calcd for C<sub>14</sub>H<sub>11</sub>ClF<sub>3</sub>NO<sub>2</sub>, 318.0509 [M + H]<sup>+</sup>; found, 318.0508.</div></div><div id="sec4_1_3_47" class="NLM_sec NLM_sec_level_4"><div id="ac_i78" class="anchor-spacer"></div><h5 class="article-section__title" id="_i78"> 2-Methyl-1-phenyl-5-(trifluoromethyl)-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>21</b>)</h5><div class="NLM_p">The compound was prepared by General Method 2 and General Method 3 and isolated as a pink solid (35 mg, quant.).</div><div class="NLM_p last">Mp: 188 Â°C. IR: Î½<sub>max</sub> (powder) 3384, 3164, 2852â2920, 2538â2663, 1660, 1444â1568, 1305, 1249, 1118 cm<sup>â1</sup>. <sup>1</sup>H NMR (700 MHz, CDCl<sub>3</sub>): Î´ 7.53â7.48 (m, 3H), 7.29 (d, <i>J</i> = 7.5 Hz, 2H), 7.09 (s, 1H), 2.45 (s, 3H). <sup>13</sup>C (175 MHz, CDCl<sub>3</sub>): Î´ 168.3, 140.6, 138.0, 129.8 (2C), 129.2, 126.6 (2C), 123.7 (q, <i>J</i> = 7 Hz), 124.5 (q, <i>J</i> = 268 Hz), 115.3 (q, <i>J</i> = 37 Hz), 109.4, 12.9. HRMS: calcd for C<sub>13</sub>H<sub>10</sub>F<sub>3</sub>NO<sub>2</sub>, 270.0742 [M + H]<sup>+</sup>; found, 270.0743.</div></div><div id="sec4_1_3_48" class="NLM_sec NLM_sec_level_4"><div id="ac_i79" class="anchor-spacer"></div><h5 class="article-section__title" id="_i79"> 1-Benzyl-2-methyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>22</b>)</h5><div class="NLM_p">The compound was prepared by General Method 5 and General Method 3 and isolated as an amber oil (52 mg, 49%).</div><div class="NLM_p last"><sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): Î´ 7.34â7.31 (m, 2H), 7.28 (m, 1H), 7.01 (app d, <i>J</i> = 7.9 Hz, 2H), 6.66 (d, <i>J</i> = 3.1 Hz, 1H), 6.57 (d, <i>J</i> = 3.1 Hz, 1H), 5.06 (s, 2H), 2.47 (s, 3H). LCMS (basic method): <i>t</i><sub>R</sub> 0.63 min, <i>m</i>/<i>z</i> 216.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_49" class="NLM_sec NLM_sec_level_4"><div id="ac_i80" class="anchor-spacer"></div><h5 class="article-section__title" id="_i80"> 1-Benzyl-2,5-dimethyl-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>23</b>)</h5><div class="NLM_p">The compound was prepared by General Method 5 and General Method 3 and isolated as a white solid (91 mg, 61%).</div><div class="NLM_p last">Mp: 199 Â°C. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>): Î´ 7.33â7.29 (m, 2H), 7.24 (m, 1H), 6.92â6.89 (m, 2H), 6.28 (app d, <i>J</i> = 0.8 Hz, 1H), 5.13 (s, 2H), 2.41 (s, 3H), 2.11 (s, 3H). LCMS (acidic method): <i>t</i><sub>R</sub> 1.60 min, <i>m</i>/<i>z</i> 230.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_50" class="NLM_sec NLM_sec_level_4"><div id="ac_i81" class="anchor-spacer"></div><h5 class="article-section__title" id="_i81"> 1-Benzyl-2-methyl- 5-(trifluoromethyl)-1<i>H</i>-pyrrole-3-carboxylic Acid (<b>24</b>)</h5><div class="NLM_p">The compound was prepared by General Method 5 and General Method 3 and isolated as a white solid (67 mg, 96%).</div><div class="NLM_p last">Mp: 202 Â°C. IR Î½<sub>max</sub> (film): 3386, 3016â3030, 2854â2970, 2582â2806, 1739, 1313â1669, 1216, 1129, 746 cm<sup>â1</sup>. <sup>1</sup>H NMR (700 MHz, methanol-<i>d</i><sub>4</sub>): Î´ 7.35 (t, <i>J</i> = 7.4 Hz, 2H), 7.28 (t, <i>J</i> = 7.4 Hz, 1H), 7.18 (s, 1H), 7.07 (d, <i>J</i> = 7.4 Hz, 2H), 5.18 (s, 2H), 2.40 (s, 3H). <sup>13</sup>C NMR (175 MHz, methanol-<i>d</i><sub>4</sub>): Î´ 167.3, 139.7, 138.0, 129.9 (2C), 128.8, 127.6 (2C), 124.6 (q, <i>J</i> = 265 Hz), 124.2 (q, <i>J</i> = 6.5 Hz), 114.9 (q, <i>J</i> = 37 Hz), 111.5, 51.5, 11.4. HRMS: calcd for C<sub>14</sub>H<sub>12</sub>F<sub>3</sub>NO<sub>2</sub>, 282.0742 [M â H]<sup>â</sup>; found, 282.0737.</div></div><div id="sec4_1_3_51" class="NLM_sec NLM_sec_level_4"><div id="ac_i82" class="anchor-spacer"></div><h5 class="article-section__title" id="_i82"> 1-(3-Chlorophenyl)pyrrolidine-3-carboxylic Acid (<b>25a</b>)</h5><div class="NLM_p">The compound was prepared by General Method 6.1 and General Method 7 with 1-chloro-3-iodobenzene and isolated as a white gum (71 mg, 36%).</div><div class="NLM_p last">IR Î½<sub>max</sub> (film): 3100, 2923, 2849, 1700, 1592, 755 cm<sup>â1</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 7.13 (t, <i>J</i> = 8.1 Hz, 1H), 6.57 (app dd, <i>J</i> = 7.7, 1.7, Hz, 1H), 6.48 (app t, <i>J</i> = 2.0 Hz, 1H), 6.45 (dd, <i>J</i> = 8.1, 2.0 Hz, 1H), 3.27â3.18 (m, 4H), 3.02 (quin, <i>J</i> = 7.0 Hz, 1H), 2.12 (q, <i>J</i> = 7.0 Hz, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 175.3, 148,7, 133.8, 130.4, 114.5, 110.8, 110.3, 50.4, 47.0, 40.0, 28.7. HRMS: calcd for C<sub>11</sub>H<sub>12</sub>ClNO<sub>2</sub>, 226.0629 [M + H]<sup>+</sup>; found, 226.0629.</div></div><div id="sec4_1_3_52" class="NLM_sec NLM_sec_level_4"><div id="ac_i83" class="anchor-spacer"></div><h5 class="article-section__title" id="_i83"> 1-(4-Chlorophenyl)pyrrolidine-3-carboxylic Acid (<b>25b</b>)</h5><div class="NLM_p">The compound was prepared by General Method 6.1 and General Method 7 with 1-chloro-4-iodobenzene and isolated as a brown gum (44 mg, 92%).</div><div class="NLM_p last">IR Î½<sub>max</sub> (film): 3100, 2923, 2849, 1700, 1592, 755 cm<sup>â1</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 7.15â7.13 (m, 2H), 6.40â6.46 (m, 2H), 3.36 (dd, <i>J</i> = 9.3, 6.3 Hz, 1H), 3.31 (dd, <i>J</i> = 9.3, 8.0 Hz, 1H), 3.23â3.19 (m, 1H), 3.17â3.13 (m, 1H), 2.92 (quin, <i>J</i> = 7.2 Hz, 1H), 2.15â2.05 (m, 2H). HRMS: calcd for C<sub>11</sub>H<sub>12</sub>ClNO<sub>2</sub>, 226.0629 [M + H]<sup>+</sup>; found, 226.0629.</div></div><div id="sec4_1_3_53" class="NLM_sec NLM_sec_level_4"><div id="ac_i84" class="anchor-spacer"></div><h5 class="article-section__title" id="_i84"> 1-(3-Methylphenyl)pyrrolidine-3-carboxylic Acid (<b>25c</b>)</h5><div class="NLM_p">The compound was prepared by General Method 6.1 and General Method 7 with 1-iodo-3-methylbenzene and isolated as a yellow solid (68 mg, 29%).</div><div class="NLM_p last">Mp: 105 Â°C. IR Î½<sub>max</sub> (film): 3036, 2920, 2835, 1690, 1600 cm<sup>â1</sup>. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 12.45 (s, 1H), 7.02 (t, <i>J</i> = 7.7 Hz, 1H), 6.42 (d, <i>J</i> = 7.4 Hz, 1H), 6.35 (app s, 1H), 6.33 (dd, <i>J</i> = 8.0, 2.2 Hz, 1H), 3.40 (dd, <i>J</i> = 9.3, 8.0 Hz, 1H), 3.36 (dd, <i>J</i> = 9.3, 6.2 Hz, 1H), 3.25 (m, 1H), 3.21 (m, 1H), 3.15 (m, 1H), 2.22 (s, 3H), 2.17 (m, 1H), 2.12 (m, 1H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 175.0, 147.6, 137.9, 128.8, 116.5, 112.5, 109.2, 49.9, 46.9, 42.4, 28.12, 21.5. HRMS: calcd for C<sub>12</sub>H<sub>15</sub>NO<sub>2</sub>, 206.1176 [M + H]<sup>+</sup>; found, 206.1176.</div></div><div id="sec4_1_3_54" class="NLM_sec NLM_sec_level_4"><div id="ac_i85" class="anchor-spacer"></div><h5 class="article-section__title" id="_i85"> 1-(3-(Trifluoro)phenyl)pyrrolidine-3-carboxylic Acid (<b>25d</b>)</h5><div class="NLM_p">The compound was prepared by General Method 6.1 and General Method 7 using 1-iodo-3-(trifluoromethyl)benzene and isolated as a yellow solid (113 mg, 53%).</div><div class="NLM_p last">Mp: 101 Â°C. IR Î½<sub>max</sub> (film): 2849, 1703, 1608, 782. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 12.51 (s, 1H), 7.35 (t, <i>J</i> = 7.9 Hz, 1H), 6.88 (d, <i>J</i> = 7.5 Hz, 1H), 6.80 (dd, <i>J</i> = 8.3, 2.1, Hz, 1H), 6.71 (app s, 1H), 3.48 (dd, <i>J</i> = 9.5, 8.0 Hz, 1H), 3.43 (dd, <i>J</i> = 9.5, 6.0 Hz, 1H), 3.35â3.27 (m, 2H), 3.20 (m, 1H), 2.22 (m, 1H), 2.16 (m, 1H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 174.7, 147.5, 129.9, 129.9 (q, <i>J</i> = 31 Hz), 124.6 (q, <i>J</i> = 272 Hz), 115.41, 111.4 (q, <i>J</i> = 4 Hz), 107.3 (q, <i>J</i> = 4 Hz), 49.8, 46.9, 42.4, 28.2. HRMS: calcd for C<sub>12</sub>H<sub>12</sub>F<sub>3</sub>NO<sub>2</sub>, 260.0893 [M + H]<sup>+</sup>; found, 260.0904.</div></div><div id="sec4_1_3_55" class="NLM_sec NLM_sec_level_4"><div id="ac_i86" class="anchor-spacer"></div><h5 class="article-section__title" id="_i86"> 1-(4-(Trifluoro)phenyl)pyrrolidine-3-carboxylic Acid (<b>25e</b>)</h5><div class="NLM_p">The compound was prepared by General Method 6.1 and General Method 7 with 1-iodo-4-(trifluoromethyl)benzene and isolated as an off-white solid (73 mg, 33%).</div><div class="NLM_p last">Mp: 105 Â°C. IR Î½<sub>max</sub> (film): 3047, 2911, 2849, 1710, 1612, 815 cm<sup>â1</sup>. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 12.52 (s, 1H), 7.44 (d, <i>J</i> = 8.8 Hz, 2H), 6.63 (d, <i>J</i> = 8.8 Hz, 2H), 3.50 (dd, <i>J</i> = 9.7, 7.8 Hz, 1H), 3.45 (dd, <i>J</i> = 9.7, 6.3 Hz, 1H), 3.36â3.29 (m, 2H) 3.20 (m, 1H), 2.22 (m, 1H), 2.16 (m, 1H). <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 174.6, 149.6, 126.2 (q, <i>J</i> = 4 Hz, 2C), 125.5 (q, <i>J</i> = 270 Hz), 115.1 (q, <i>J</i> = 33 Hz), 111.3 (2C), 49.6, 46.8, 42.4, 28.2. HRMS: calcd for C<sub>12</sub>H<sub>12</sub>F<sub>3</sub>NO<sub>2</sub>, 260.0893 [M + H]<sup>+</sup>; found, 260.0899.</div></div><div id="sec4_1_3_56" class="NLM_sec NLM_sec_level_4"><div id="ac_i87" class="anchor-spacer"></div><h5 class="article-section__title" id="_i87"> 1-(3-Isopropylphenyl)pyrrolidine-3-carboxylic Acid (<b>25f</b>)</h5><div class="NLM_p">The compound was prepared by General Method 6.1 and General Method 7 with 1-iodo-3-isopropylbenzene and isolated as a yellow oil (52.1 mg, 16%).</div><div class="NLM_p last">IR Î½<sub>max</sub> (film): 3040, 2957, 2866, 1703, 1601 cm<sup>â1</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 12.49 (s, 1H), 7.06 (t, <i>J</i> = 7.7 Hz, 1H), 6.50 (d, <i>J</i> = 7.7 Hz, 1H), 6.39 (t, <i>J</i> = 1.9 Hz, 1H), 6.35 (ddd, <i>J</i> = 8.1, 2.4, 0.6 Hz, 1H), 3.43 (dd, <i>J</i> = 9.2, 8.1 Hz, 1H), 3.38 (dd, <i>J</i> = 9.2, 6.0 Hz, 1H), 3.30â3.21 (m, 2H), 3.16 (m, 1H), 2.79 (sept, <i>J</i> = 6.9 Hz, 1H), 2.22â2.10 (m, 2H), 1.18 (d, <i>J</i> = 6.9 Hz, 6H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 175.0, 149.2, 147.6, 128.9, 113.6, 109.9, 109.5, 49.9, 46.9, 42.4, 33.9, 28.2, 24.1 (2C). HRMS: calcd for C<sub>14</sub>H<sub>19</sub>NO<sub>2</sub>, 234.1489 [M + H]<sup>+</sup>; found, 234.1487.</div></div><div id="sec4_1_3_57" class="NLM_sec NLM_sec_level_4"><div id="ac_i88" class="anchor-spacer"></div><h5 class="article-section__title" id="_i88"> 1-(3-(Trifluoromethoxy)phenyl)pyrrolidine-3-carboxylic Acid (<b>25g</b>)</h5><div class="NLM_p">The compound was prepared by General Method 6.1 and General Method 7 with 1-iodo-3-(trifluoromethoxy)benzene and isolated as an amber oil (90.8 mg, 40%).</div><div class="NLM_p last">IR Î½<sub>max</sub> (film): 3082, 2911, 2860, 1699, 1615, 822 cm<sup>â1</sup>. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 12.50 (s, 1H), 7.23 (t, <i>J</i> = 8.2 Hz, 1H), 6.53 (dd, <i>J</i> = 8.3, 2.3 Hz, 1H), 6.51 (d, <i>J</i> = 8.3 Hz, 1H), 6.39 (app s, 1H), 3.44 (dd, <i>J</i> = 9.6, 7.8 Hz, 1H), 3.39 (dd, <i>J</i> = 9.6, 6.0 Hz, 1H), 3.29â3.23 (m, 2H), 3.18 (m, 1H), 2.20 (m, 1H), 2.14 (m, 1H). <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 174.7, 149.6 (q, <i>J</i> = 1.6 Hz), 148.8, 130.4, 120.2 (q, <i>J</i> = 256 Hz), 110.7, 106,8, 103.8, 49.8, 46.9, 42.4, 28.2. HRMS: calcd for C<sub>12</sub>H<sub>12</sub>F<sub>3</sub>NO<sub>3</sub>, 276.0842 [M + H]<sup>+</sup>; found, 276.0841.</div></div><div id="sec4_1_3_58" class="NLM_sec NLM_sec_level_4"><div id="ac_i89" class="anchor-spacer"></div><h5 class="article-section__title" id="_i89"> 1-(3-Cyanophenyl)pyrrolidine-3-carboxylic Acid (<b>25h</b>)</h5><div class="NLM_p">The compound was prepared by General Method 6.1 and General Method 7 with 3-iodobenzonitrile and isolated as an off-white solid (72 mg, 27%).</div><div class="NLM_p last">Mp: 139 Â°C. IR Î½<sub>max</sub> (film): 3038, 2915, 2850, 1702, 1554 cm<sup>â1</sup>. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 12.53 (s, 1H), 7.32 (app t, <i>J</i> = 8.0 Hz, 1H), 6.97 (app d, <i>J</i> = 7.5 Hz, 1H), 6.88 (t, <i>J</i> = 2.2 Hz, 1H), 6.84 (dd, <i>J</i> = 8.3, 2.2 Hz, 1H), 3.46 (dd, <i>J</i> = 9.6, 7.9 Hz, 1H), 3.41 (td, <i>J</i> = 9.6, 6.0 Hz, 1H), 3.34â3.26 (m, 2H), 3.19 (m, 1H), 2.21 (m, 1H), 2.14 (m, 1H). <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 174.7, 147.4, 130,1, 119.6, 118.5, 116.4, 114.1, 111.8, 49.7, 46.8, 42.4, 28.2. LCMS (acidic method): Rt 1.53 min, <i>m</i>/<i>z</i> 217.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_59" class="NLM_sec NLM_sec_level_4"><div id="ac_i90" class="anchor-spacer"></div><h5 class="article-section__title" id="_i90"> 1-(4-Fluorophenyl)pyrrolidine-3-carboxylic Acid (<b>25i</b>)</h5><div class="NLM_p">The compound was prepared by General Method 6.1 and General Method 7 with 1-fluoro-4-iodobenzene and isolated as a brown solid (51.3 mg, 32%).</div><div class="NLM_p last">Mp: 105 Â°C. IR Î½<sub>max</sub> (film): 3031, 2910, 2852, 1697, 1518, 809 cm<sup>â1</sup>. <sup>1</sup>H NMR Î´ (600 MHz, DMSO-<i>d</i><sub>6</sub>) 12.50 (s, 1H), 7.02â6.98 (m, 2H), 6.54â6.50 (m, 2H), 3.40 (dd, <i>J</i> = 9.3, 8.1 Hz, 1H), 3.36 (dd, <i>J</i> = 9.3, 6.1 Hz, 1H), 3.27â3.15 (m, 3H), 2.19 (m, 1H), 2.13 (m, 1H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 175.0, 154.2 (d, <i>J</i> = 231 Hz), 144.5, 115.4 (d, <i>J</i> = 22 Hz, 2C), 112.5 (d, <i>J</i> = 7.2 Hz, 2C), 50.3, 47.4, 42.5, 28.3 ppm. HRMS: calcd for C<sub>11</sub>H<sub>12</sub>FNO<sub>2</sub>, 210.0925 [M + H]<sup>+</sup>; found, 210.0927.</div></div><div id="sec4_1_3_60" class="NLM_sec NLM_sec_level_4"><div id="ac_i91" class="anchor-spacer"></div><h5 class="article-section__title" id="_i91"> 1-(2,3-Dichlorophenyl)pyrrolidine-3-carboxylic Acid (<b>25j</b>)</h5><div class="NLM_p">The compound was prepared by General Method 6.1 and General Method 7 with 1,2-dichloro-3-iodobenzene and isolated as an orange solid (84 mg, 50%).</div><div class="NLM_p last">IR Î½<sub>max</sub> (film): 3074, 2946, 2855, 1699, 1578, 775 cm<sup>â1</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 12.50 (s, 1H), 7.20 (t, <i>J</i> = 8.2 Hz, 1H), 7.10 (dd, <i>J</i> = 7.9, 1.4, Hz, 1H), 6.96 (dd, <i>J</i> = 8.2, 1.4, Hz, 1H), 3.56 (dd, <i>J</i> = 9.6, 6.4 Hz, 1H), 3.48 (dd, <i>J</i> = 9.6, 7.9 Hz, 1H), 3.37 (m, 1H), 3.31 (m, 1H), 3.11 (quin, <i>J</i> = 7.3 Hz, 1H), 2.15 (m, 1H), 2.08 (m, 1H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) 174.8, 148.4, 133.0, 128.1, 121.3, 120.6, 116.4, 53.2, 50.2, 42.3, 28.1. HRMS: calcd for C<sub>11</sub>H<sub>11</sub>Cl<sub>2</sub>NO<sub>2</sub>, 260.0240 [M + H]<sup>+</sup>; found, 260.0240.</div></div><div id="sec4_1_3_61" class="NLM_sec NLM_sec_level_4"><div id="ac_i92" class="anchor-spacer"></div><h5 class="article-section__title" id="_i92"> 1-(2,4-Dichlorophenyl)pyrrolidine-3-carboxylic Acid (<b>25k</b>)</h5><div class="NLM_p">The compound was prepared by General Method 6.1 and General Method 7 with 2,4-dichloro-1-iodobenzene and isolated as a brown oil (69 mg, 38%).</div><div class="NLM_p last">IR Î½<sub>max</sub> (film): 3030, 2951, 2840, 1703, 1475, 800 cm<sup>â1</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 12.50 (s, 1H), 7.42 (d, <i>J</i> = 2.5 Hz, 1H), 7.24 (dd, <i>J</i> = 8.8, 2.5, Hz, 1H), 6.95 (d, <i>J</i> = 8.8 Hz, 1H), 3.54 (dd, <i>J</i> = 9.5, 6.6 Hz, 1H), 3.47 (dd, <i>J</i> = 9.5, 7.8 Hz, 1H), 3.34 (m, 1H), 3.29 (m, 1H), 3.11 (quin, <i>J</i> = 7.4 Hz, 1H), 2.14 (m, 1H), 2.07 (m, 1H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 174.8, 145.3, 130.0, 127.5, 123.3, 122.7, 118.7, 53.0, 50.0, 42.3, 28.0. HRMS: calcd for C<sub>11</sub>H<sub>11</sub>NO<sub>2</sub>Cl<sub>2</sub>, 260.0240 [M + H]<sup>+</sup>; found, 260.0241.</div></div><div id="sec4_1_3_62" class="NLM_sec NLM_sec_level_4"><div id="ac_i93" class="anchor-spacer"></div><h5 class="article-section__title" id="_i93"> 1-(2,5-Dichlorophenyl)pyrrolidine-3-carboxylic Acid (<b>25l</b>)</h5><div class="NLM_p">The compound was prepared by General Method 6.1 and General Method 7 using 1,4-dichloro-2-iodobenzene and isolated as an orange oil (88 mg, 39%).</div><div class="NLM_p last">IR Î½<sub>max</sub> (film): 3038, 2953, 2848, 1704, 1584, 785 cm<sup>â1</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 12.51 (s, 1H), 7.31 (d, <i>J</i> = 8.4 Hz, 1H), 6.91 (d, <i>J</i> = 2.4 Hz, 1H), 6.85 (dd, <i>J</i> = 8.4, 2.4 Hz, 1H), 3.59 (dd, <i>J</i> = 9.6, 6.5 Hz, 1H), 3.52 (dd, <i>J</i> = 9.6, 7.7 Hz, 1H), 3.40â3.31 (m, 2H), 3.11 (app quin, <i>J</i> = 7.3, 1H), 2.14 (m, 1H), 2.07 (m, 1H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 174.7, 147.3, 132.2 (2C), 120.2, 119.7, 116.8, 52.9, 49.9, 42.4, 28.1. HRMS: calcd for C<sub>11</sub>H<sub>11</sub>Cl<sub>2</sub>NO<sub>2</sub>,260.0240 [M + H]<sup>+</sup>; found, 260.0240.</div></div><div id="sec4_1_3_63" class="NLM_sec NLM_sec_level_4"><div id="ac_i94" class="anchor-spacer"></div><h5 class="article-section__title" id="_i94"> 1-(2,6-Dichlorophenyl)pyrrolidine-3-carboxylic Acid (<b>25m</b>)</h5><div class="NLM_p">The compound was prepared by General Method 6.1 and General Method 7 using 1,3-dichloro-2-iodobenzene and isolated as a brown oil (82 mg, 42%).</div><div class="NLM_p last">IR Î½<sub>max</sub> (film): 3028, 2915, 2850, 1702, 1554, 775 cm<sup>â1</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 12.41 (s, 1H), 7.45 (d, <i>J</i> = 8.1 Hz, 2H), 7.22â7.17 (m, 1H), 3.46 (t, <i>J</i> = 8.2 Hz, 1H), 3.41 (dd, <i>J</i> = 8.3, 7.0 Hz, 1H), 3.34â3.30 (m, 1H), 3.27 (q, <i>J</i> = 7.3 Hz, 1H), 3.20â3.10 (m, 1H), 2.20â2.13 (m, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 174.9, 141.6, 135.6, 129.4, 127.6, 52.1, 49.3, 43.5, 29.0. HRMS: calcd for C<sub>11</sub>H<sub>11</sub>NO<sub>2</sub>Cl<sub>2</sub>, 260.0240 [M + H]<sup>+</sup>; found, 260.0241.</div></div><div id="sec4_1_3_64" class="NLM_sec NLM_sec_level_4"><div id="ac_i95" class="anchor-spacer"></div><h5 class="article-section__title" id="_i95"> 1-(3,4-Dichlorophenyl)pyrrolidine-3-carboxylic Acid (<b>25n</b>)</h5><div class="NLM_p">The compound was prepared by General Method 6.1 and General Method 7 with 1,2-dichloro-4-iodobenzene and isolated as an off-white solid (70.8 mg, 36%).</div><div class="NLM_p last">Mp: 138 Â°C. IR Î½<sub>max</sub> (film): 3300, 2954, 2846, 1700, 1596, 804 cm<sup>â1</sup>. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 12.51 (s, 1H), 7.33 (d, <i>J</i> = 8.9 Hz, 1H), 6.70 (d, <i>J</i> = 2.8 Hz, 1H), 6.52 (dd, <i>J</i> = 8.9, 2.8 Hz, 1H), 3.47â3.41 (m, 1H), 3.41â3.36 (m, 1H), 3.30â3.23 (m, 2H), 3.21â3.15 (m, 1H), 2.24â2.10 (m, 2H). <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 174.6, 147.1, 131.4, 130.4, 116.4, 112.6, 112.1, 49.8, 47.0, 42.4, 28.2. HRMS: calcd for C<sub>11</sub>H<sub>11</sub>Cl<sub>2</sub>NO<sub>2</sub>, 260.0240 [M + H]<sup>+</sup>; found, 260.0240.</div></div><div id="sec4_1_3_65" class="NLM_sec NLM_sec_level_4"><div id="ac_i96" class="anchor-spacer"></div><h5 class="article-section__title" id="_i96"> 1-(3,5-Dichlorophenyl)pyrrolidine-3-carboxylic Acid (<b>25o</b>)</h5><div class="NLM_p">The compound was prepared by General Method 6.1 and General Method 7 with 1,3-dichloro-5-iodobenzene and isolated as a brown solid (63.9 mg, 32%).</div><div class="NLM_p last">Mp: 135 Â°C. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 12.56 (s, 1H), 6.68 (apparent t, <i>J</i> = 1.8 Hz, 1H), 6.52 (d, <i>J</i> = 1.8 Hz, 2H), 3.43 (ddd, <i>J</i> = 15.9, 9.9, 7.0 Hz, 2H), 3.32â3.24 (m, 2H), 3.22â3.14 (m, 1H), 2.25â2.10 (m, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 174.6, 149.0, 134.5, 114.1, 110.0, 49. 8, 46.9, 42.3, 28.2. HRMS: calcd for C<sub>11</sub>H<sub>11</sub>Cl<sub>2</sub>NO<sub>2</sub>, 260.0240 [M + H]<sup>+</sup>; found, 260.0240.</div></div><div id="sec4_1_3_66" class="NLM_sec NLM_sec_level_4"><div id="ac_i97" class="anchor-spacer"></div><h5 class="article-section__title" id="_i97"> 1-(3-Chloro-4-(trifluoromethyl)phenyl)pyrrolidine-3-carboxylic Acid (<b>25p</b>)</h5><div class="NLM_p">The compound was prepared by General Method 6.1 and General Method 7 with 3-chloro-4-(trifluoromethyl)-1-iodobenzene except purification by reverse-phase chromatography (0â95% MeCN/H<sub>2</sub>O; 0.1% NH<sub>4</sub>OH modifier) and isolated as a white solid (7 mg, 7%).</div><div class="NLM_p last"><sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 7.47 (d, <i>J</i> = 8.9 Hz, 1H), 6.60 (d, <i>J</i> = 2.1 Hz, 1H), 6.48 (dd, <i>J</i> = 8.9, 2.1 Hz, 1H), 3.47 (dd, <i>J</i> = 9.4, 6.1 Hz, 1H), 3.30 (m, 2H), 3.26 (dd, <i>J</i> = 14.7, 8.2 Hz, 1H), 3.21 (dd, <i>J</i> = 16.1, 7.3 Hz, 1H), 2.75â2.67 (m, 1H), 2.13 (td, <i>J</i> = 14.1, 7.1 Hz, 1H), 1.99 (td, <i>J</i> = 12.9, 7.1 Hz, 1H). <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 175.4, 150.6, 131.7, 128.3 (q, <i>J</i> = 5.0 Hz), 124.2 (q, <i>J</i> = 269.8 Hz), 112.5, 111.1 (d, <i>J</i> = 31.2 Hz), 109.2, 51.6, 47.3, 46.0, 29.5. LCMS (acidic method): <i>t</i><sub>R</sub> 1.89 min, <i>m</i>/<i>z</i> 294.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_67" class="NLM_sec NLM_sec_level_4"><div id="ac_i98" class="anchor-spacer"></div><h5 class="article-section__title" id="_i98"> 1-(4-Chloro-3-(trifluoromethyl)phenyl)pyrrolidine-3-carboxylic Acid (<b>25q</b>)</h5><div class="NLM_p">The compound was prepared by General Method 6.1 and  General Method 6.2, and then 7 with 2-chloro-4-iodo-1-(trifluoromethyl)benzene and isolated as an off-white solid (84 mg, 47%).</div><div class="NLM_p last">Mp: 158 Â°C. IR Î½<sub>max</sub> (film): 3047, 2915, 2861, 1707, 1608, 1130, 812 cm<sup>â1</sup>. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 12.53 (s, 1H), 7.41 (d, <i>J</i> = 8.8 Hz, 1H), 6.81 (d, <i>J</i> = 2.9 Hz, 1H), 6.78 (dd, <i>J</i> = 8.8, 2.8 Hz, 1H), 3.51â3.47 (m, 1H), 3.46â3.42 (m, 1H), 3.36â3.27 (m, 2H), 3.23â3.17 (m, 1H), 2.26â2.20 (m, 1H), 2.19â2.13 (m, 1H). <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 174.6, 146.0, 131.9, 126.9 (q, <i>J</i> = 30.0 Hz), 123.1 (q, <i>J</i> = 273.1 Hz), 116.2 (s), 115.3, 109.7 (q, <i>J</i> = 5.5 Hz), 49.8, 47.0, 42.4, 28.2. HRMS: calcd for C<sub>12</sub>H<sub>11</sub>ClF<sub>3</sub>NO<sub>2</sub>, 294.0503 [M + H]<sup>+</sup>; found, 294.0503.</div></div><div id="sec4_1_3_68" class="NLM_sec NLM_sec_level_4"><div id="ac_i99" class="anchor-spacer"></div><h5 class="article-section__title" id="_i99"> 1-(4-Fluoro-3-(trifluoromethyl)phenyl)pyrrolidine-3-carboxylic Acid (<b>25r</b>)</h5><div class="NLM_p">The compound was prepared by General Method 6.1 and General Method 7 with 1-fluoro-4-iodo-2-(trifluoromethyl)benzene and isolated as an oil (88 mg, 49%).</div><div class="NLM_p last">IR Î½<sub>max</sub> (film): 3014, 2965, 2853, 1706, 1514, 812 cm<sup>â1</sup>. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 12.50 (s, 1H), 7.28 (t, <i>J</i> = 9.8 Hz, 1H), 6.85â6.79 (m, 1H), 6.69 (dd, <i>J</i> = 5.7, 3.1 Hz, 1H), 3.48â3.45 (m, 1H), 3.43â3.39 (m, 1H), 3.34â3.25 (m, 2H), 3.23â3.17 (m, 1H), 2.27â2.19 (m, 1H), 2.18â2.12 (m, 1H). <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 174.7, 150.9 (d, <i>J</i> = 2.0 Hz), 149.5 (d, <i>J</i> = 2.1 Hz), 144.2, 123.0 (q, <i>J</i> = 272.0 Hz), 117.6 (d, <i>J</i> = 21.2 Hz), 117.0â116.2 (m), 108.0 (q, <i>J</i> = 4.8 Hz), 50.1, 47.2, 42.4, 28.3. HRMS: calcd for C<sub>12</sub>H<sub>11</sub>F<sub>4</sub>NO<sub>2</sub>, 278.0799 [M + H]<sup>+</sup>; found, 278.0798.</div></div><div id="sec4_1_3_69" class="NLM_sec NLM_sec_level_4"><div id="ac_i100" class="anchor-spacer"></div><h5 class="article-section__title" id="_i100"> 1-(2-Fluoro-3-(trifluoromethyl)phenyl)pyrrolidine-3-carboxylic Acid (<b>25s</b>)</h5><div class="NLM_p">The compound was prepared by General Method 6.1 and General Method 7 with 2-fluoro-1-iodo-3-(trifluoromethyl)benzene and isolated as an oil (90 mg, 59%).</div><div class="NLM_p last">IR Î½<sub>max</sub> (film): 3051, 2915, 2853, 1704, 1498, 780 cm<sup>â1</sup>. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 12.51 (s, 1H), 7.17 (t, <i>J</i> = 8.0 Hz, 1H), 7.03 (t, <i>J</i> = 8.3 Hz, 1H), 6.96 (t, <i>J</i> = 6.5 Hz, 1H), 3.57 (dd, <i>J</i> = 7.0, 2.7 Hz, 2H), 3.48â3.37 (m, 2H), 3.15 (p, <i>J</i> = 7.1 Hz, 1H), 2.20â2.14 (m, 1H), 2.14â2.08 (m, 1H). <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 174.7, 147.9 (dd, <i>J</i> = 249.3, 2.0 Hz), 137.3 (d, <i>J</i> = 7.8 Hz), 124.0 (d, <i>J</i> = 3.5 Hz), 123.1 (q, <i>J</i> = 272.3 Hz), 120.2 (d, <i>J</i> = 5.8 Hz), 117.5 (qd, <i>J</i> = 31.3, 11.1 Hz), 113.6 (q, <i>J</i> = 4.7 Hz), 51.9 (d, <i>J</i> = 5.7 Hz), 49.0 (d, <i>J</i> = 4.8 Hz), 42.2 (d, <i>J</i> = 1.4 Hz), 28.0 (d, <i>J</i> = 1.1 Hz). LCMS (acidic method): <i>t</i><sub>R</sub> 1.79 min, <i>m</i>/<i>z</i> 278.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_70" class="NLM_sec NLM_sec_level_4"><div id="ac_i101" class="anchor-spacer"></div><h5 class="article-section__title" id="_i101"> 1-(2,3,4-Trichlorophenyl)pyrrolidine-3-carboxylic Acid (<b>25t</b>)</h5><div class="NLM_p">The compound was prepared by General Method 6.1 and General Method 7 with 1,2,3-trichloro-4-iodobenzene and isolated as an orange solid (56 mg, 53%).</div><div class="NLM_p last">Mp: 111 Â°C. IR Î½<sub>max</sub> (film): 3058, 2920, 2852, 1580, 1702, 789 cm<sup>â1</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 12.52 (s, 1H), 7.44 (d, <i>J</i> = 9.1 Hz, 1H), 6.99 (d, <i>J</i> = 9.1 Hz, 1H), 3.61â3.55 (m, 1H), 3.52â3.45 (m, 1H), 3.41â3.30 (m, 2H), 3.17â3.05 (m, 1H), 2.19â2.11 (m, 1H), 2.11â2.04 (m, 1H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 174.7, 147.3, 131.2, 128.4, 122.4, 121.8, 116.8, 53.2, 50.2, 42.3, 28.1. HRMS: calcd for C<sub>11</sub>H<sub>10</sub>Cl<sub>3</sub>NO<sub>2</sub>, 293.9850 [M + H]<sup>+</sup>; found, 293.9851.</div></div><div id="sec4_1_3_71" class="NLM_sec NLM_sec_level_4"><div id="ac_i102" class="anchor-spacer"></div><h5 class="article-section__title" id="_i102"> (<i>S</i>)-1-(4-Chloro-3-(trifluoromethyl)phenyl)pyrrolidine-3-carboxylic Acid (<b>26</b>)</h5><div class="NLM_p">The compound was prepared by General Method 6.1 and General Method 7 and isolated as an off-white solid (258 mg, 54%) (SFC chemical purity 99.78%).</div><div class="NLM_p">Mp: 155 Â°C. IR Î½<sub>max</sub> (film): 3059, 2915, 2861, 1708, 1608, 812 cm<sup>â1</sup>. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 12.53 (s, 1H), 7.41 (d, <i>J</i> = 8.8 Hz, 1H), 6.81 (d, <i>J</i> = 2.9 Hz, 1H), 6.78 (dd, <i>J</i> = 8.8, 2.9 Hz, 1H), 3.49 (dd, <i>J</i> = 9.6, 8.0 Hz, 1H), 3.44 (dd, <i>J</i> = 9.6, 6.1 Hz, 1H), 3.36â3.27 (m, 2H), 3.24â3.17 (m, 1H), 2.23 (dtd, <i>J</i> = 13.0, 7.4, 5.7 Hz, 1H), 2.19â2.12 (m, 1H). <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 174.6, 146.0, 132.0, 126.9 (q, <i>J</i> = 30.0 Hz), 123.2 (q, <i>J</i> = 273.1 Hz), 116.3, 115.3, 109.7 (q, <i>J</i> = 5.5 Hz), 49.9, 47.0, 42.4, 28.3. LCMS (acidic method): <i>t</i><sub>R</sub> 1.90 min, <i>m</i>/<i>z</i> 294.1 [M + H]<sup>+</sup>.</div><div class="NLM_p last">[Î±]<sub class="stack">D</sub><sup class="stack">20</sup> +0.29 (<i>c</i> 1, CHCl<sub>3</sub>). Chiral SFC analysis: <i>R</i>/<i>S</i> 1.5:98.5; 97.0 enantiomeric excess.</div></div><div id="sec4_1_3_72" class="NLM_sec NLM_sec_level_4"><div id="ac_i103" class="anchor-spacer"></div><h5 class="article-section__title" id="_i103"> (<i>R</i>)-1-(4-Chloro-3-(trifluoromethyl)phenyl)pyrrolidine-3-carboxylic Acid (<b>27</b>)</h5><div class="NLM_p">The compound was prepared by General Method 6.1 and General Method 7 and isolated as a white solid (291 mg, 61%) (SFC chemical purity 99.80%).</div><div class="NLM_p">Mp: 157 Â°C. IR, <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>), and <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>) were consistent with <b>26</b>. LCMS (acidic method): <i>t</i><sub>R</sub> 1.86 min, <i>m</i>/<i>z</i> 294.1 [M + H]<sup>+</sup>.</div><div class="NLM_p last">Chiral SFC analysis: <i>R</i>/<i>S</i> 98.2:1.8; 96.4 enantiomeric excess.</div></div><div id="sec4_1_3_73" class="NLM_sec NLM_sec_level_4"><div id="ac_i104" class="anchor-spacer"></div><h5 class="article-section__title" id="_i104"> 1-(4-Chloro-3-(trifluoromethyl)phenyl)<i>-N</i>-methylpyrrolidine-3-carboxamide (<b>28a</b>)</h5><div class="NLM_p">The compound was prepared by General Method 8 with methylamine and isolated as a colorless gum (37 mg, 71% yield).</div><div class="NLM_p last">IR Î½<sub>max</sub> (film): 3310, 2917, 2824, 1638, 1566, 797 cm<sup>â1</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 7.99 (q, <i>J</i> = 4.5 Hz, 1H), 7.41 (d, <i>J</i> = 8.8 Hz, 1H), 6.79 (d, <i>J</i> = 2.8 Hz, 1H), 6.76 (dd, <i>J</i> = 8.8, 2.9 Hz, 1H), 3.47â3.43 (m, 1H), 3.41â3.25 (m, 3H), 3.05 (p, <i>J</i> = 7.6 Hz, 1H), 2.60 (d, <i>J</i> = 4.5 Hz, 3H), 2.21â2.11 (m, 1H), 2.11â1.97 (m, 1H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 172.5, 146.0, 131.9, 126.8 (q, <i>J</i> = 29.9 Hz), 123.2 (q, <i>J</i> = 273.1 Hz), 116.2, 115.1â115.0 (m), 109.6 (q, <i>J</i> = 5.6 Hz), 50.5, 47.3, 43.2, 29.0, 25.7. HRMS: calcd for C<sub>13</sub>H<sub>14</sub>ClF<sub>3</sub>N<sub>2</sub>, 307.0820 [M + H]<sup>+</sup>; found, 307.0820.</div></div><div id="sec4_1_3_74" class="NLM_sec NLM_sec_level_4"><div id="ac_i105" class="anchor-spacer"></div><h5 class="article-section__title" id="_i105"> 1-(4-Chloro-3-(trifluoromethyl)phenyl)<i>-N</i>-(2-methoxyethyl)pyrrolidine-3-carboxamide (<b>28b</b>)</h5><div class="NLM_p">The compound was prepared by General Method 8 with 2-methoxyethylamine and isolated as a white solid (49 mg, 82%).</div><div class="NLM_p last">Mp: 112 Â°C. IR Î½<sub>max</sub> (film): 3297, 2915, 2827, 1642, 1117, 801 cm<sup>â1</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 8.14 (t, <i>J</i> = 5.4 Hz, 1H), 7.41 (d, <i>J</i> = 8.8 Hz, 1H), 6.79 (d, <i>J</i> = 2.8 Hz, 1H), 6.76 (dd, <i>J</i> = 8.8, 2.8 Hz, 1H), 3.45 (t, <i>J</i> = 8.7 Hz, 1H), 3.40â3.21 (m, 10H), 3.14â3.06 (m, 1H), 2.20â2.12 (m, 1H), 2.11â1.99 (m, 1H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 172.3, 146.1, 131.9, 126.8 (q, <i>J</i> = 29.8 Hz), 123.2 (q, <i>J</i> = 273.1 Hz), 116.2, 115.1, 109.6 (q, <i>J</i> = 5.5 Hz), 70.6, 58.0, 50.5, 47.3, 43.1, 38.5, 29.1. HRMS: calcd for C<sub>15</sub>H<sub>18</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>2</sub>, 351.1082 [M + H]<sup>+</sup>; found, 351.1082.</div></div><div id="sec4_1_3_75" class="NLM_sec NLM_sec_level_4"><div id="ac_i106" class="anchor-spacer"></div><h5 class="article-section__title" id="_i106"> 1-(4-Chloro-3-(trifluoromethyl)phenyl)<i>-N</i>-phenylpyrrolidine-3-carboxamide (<b>28c</b>)</h5><div class="NLM_p">The compound was prepared by General Method 8 with aniline and isolated as an off-white solid (54 mg, 86%).</div><div class="NLM_p last">Mp: 115 Â°C. IR Î½<sub>max</sub> (film): 3296, 2864, 1658, 1434, 1114, 753 cm<sup>â1</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 10.13 (s, 1H), 7.61 (d, <i>J</i> = 7.6 Hz, 2H), 7.43 (d, <i>J</i> = 8.8 Hz, 1H), 7.30 (t, <i>J</i> = 7.9 Hz, 2H), 7.04 (q, <i>J</i> = 7.6 Hz, 1H), 6.83 (d, <i>J</i> = 2.8 Hz, 1H), 6.80 (dd, <i>J</i> = 8.9, 2.8 Hz, 1H), 3.63â3.53 (m, 1H), 3.51â3.39 (m, 2H), 3.39â3.26 (m, 2H), 2.36â2.24 (m, 1H), 2.25â2.12 (m, 1H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 171.2, 146.1, 139.1, 132.0, 131.8â131.6 (m), 128.8, 128.2 (q, <i>J</i> = 11.8 Hz), 126.9 (q, <i>J</i> = 30.0 Hz), 123.3, 123.2 (q, <i>J</i> = 273.2 Hz), 119.2, 116.2, 115.2, 109.7 (q, <i>J</i> = 5.6 Hz), 50.5, 47.3, 44.2, 29.2. HRMS: calcd for C<sub>18</sub>H<sub>17</sub>ClF<sub>3</sub>N<sub>2</sub>O, 369.0981 [M + H]<sup>+</sup>; found, 369.0976.</div></div><div id="sec4_1_3_76" class="NLM_sec NLM_sec_level_4"><div id="ac_i107" class="anchor-spacer"></div><h5 class="article-section__title" id="_i107"> 1-(4-Chloro-3-(trifluoromethyl)phenyl)<i>-N</i>,<i>N</i>-dimethylpyrrolidine-3-carboxamide (<b>28d</b>)</h5><div class="NLM_p">The compound was prepared by General Method 8 with dimethylamine and isolated as an orange gum (29 mg, 53%).</div><div class="NLM_p last">IR Î½<sub>max</sub> (film): 2932, 2855, 1640, 1115 cm<sup>â1</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 7.42 (d, <i>J</i> = 8.8 Hz, 1H), 6.81 (d, <i>J</i> = 2.9 Hz, 1H), 6.78 (dd, <i>J</i> = 8.8, 2.9 Hz, 1H), 3.57â3.48 (m, 2H), 3.40â3.27 (m, 3H), 3.07 (s, 3H), 2.84 (s, 3H), 2.25â2.14 (m, 1H), 2.12â1.99 (m, 1H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 171.9, 146.1, 131.9, 126.8 (q, <i>J</i> = 30.0 Hz), 123.2 (q, <i>J</i> = 273.2 Hz), 116.2, 115.1, 109.7 (q, <i>J</i> = 4.8 Hz), 50.4, 47.3, 36.8, 35.2, 28.6. LCMS (basic method): <i>t</i><sub>R</sub> 1.98 min, <i>m</i>/<i>z</i> 321.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_77" class="NLM_sec NLM_sec_level_4"><div id="ac_i108" class="anchor-spacer"></div><h5 class="article-section__title" id="_i108"> (1-(4-Chloro-3-(trifluoromethyl)phenyl)pyrrolidin-3-yl)(3,3-difluoroazetidin-1-yl)methanone (<b>28e</b>)</h5><div class="NLM_p">The compound was prepared by General Method 8 with 3,3-difluoroazetidine and isolated as a white solid (62 mg, 99% yield).</div><div class="NLM_p last">Mp: 104 Â°C. IR Î½<sub>max</sub> (film): 2929, 2849, 1654, 1110. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 7.42 (d, <i>J</i> = 8.8 Hz, 1H), 6.82 (d, <i>J</i> = 2.9 Hz, 1H), 6.79 (dd, <i>J</i> = 8.8, 2.9 Hz, 1H), 4.85â4.62 (m, 2H), 4.31 (t, <i>J</i> = 12.6 Hz, 2H), 3.52 (dd, <i>J</i> = 9.2, 8.3 Hz, 1H), 3.41â3.36 (m, 1H), 3.35â3.21 (m, 3H), 2.26â2.16 (m, 1H), 2.16â2.00 (m, 1H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 172.6, 146.0, 132.0, 126.9 (q, <i>J</i> = 30.1 Hz), 123.2 (q, <i>J</i> = 273.1 Hz), 116.3, 116.2 (t, <i>J</i> = 272.1 Hz), 115.3, 109.8 (q, <i>J</i> = 5.5 Hz), 61.3 (t, <i>J</i> = 28.0 Hz), 59.5 (t, <i>J</i> = 27.8 Hz), 49.9, 47.2, 28.1. HRMS: calcd for C<sub>15</sub>H<sub>14</sub>ClF<sub>5</sub>N<sub>2</sub>O, 369.0788 [M + H]<sup>+</sup>; found, 369.0787.</div></div><div id="sec4_1_3_78" class="NLM_sec NLM_sec_level_4"><div id="ac_i109" class="anchor-spacer"></div><h5 class="article-section__title" id="_i109"> (1-(4-Chloro-3-(trifluoromethyl)phenyl)pyrrolidin-3-yl)(pyrrolidin-1-yl)methanone (<b>28f</b>)</h5><div class="NLM_p">The compound was prepared by General 8 with pyrrolidine and isolated as a yellow gum (51 mg, 87%).</div><div class="NLM_p last">IR Î½<sub>max</sub> (film): 2977, 2873, 1632, 1129, 847 cm<sup>â1</sup>. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 7.41 (d, <i>J</i> = 8.8 Hz, 1H), 6.81 (d, <i>J</i> = 2.9 Hz, 1H), 6.78 (dd, <i>J</i> = 8.8, 2.9 Hz, 1H), 3.58â3.48 (m, 3H), 3.42â3.33 (m, 3H), 3.33â3.26 (m, 3H), 2.25â2.17 (m, 1H), 2.16â2.04 (m, 1H), 1.90 (p, <i>J</i> = 6.7 Hz, 2H), 1.78 (p, <i>J</i> = 6.9 Hz, 2H). <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 170.3, 146.1, 131.9, 126.8 (q, <i>J</i> = 30.0 Hz), 123.2 (q, <i>J</i> = 273.1 Hz), 116.2, 115.1, 110.5â109.3 (m), 50.2, 47.3, 45.9, 45.6, 41.4, 28.3, 25.6, 23.8. LCMS (basic method): <i>t</i><sub>R</sub> 1.95 min, <i>m</i>/<i>z</i> 347.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_79" class="NLM_sec NLM_sec_level_4"><div id="ac_i110" class="anchor-spacer"></div><h5 class="article-section__title" id="_i110"> (1-(4-Chloro-3-(trifluoromethyl)phenyl)pyrrolidin-3-yl)(morpholin-4-yl)methanone (<b>28g</b>)</h5><div class="NLM_p">The compound was prepared by General Method 8 with morpholine and isolated as a clear gum (54 mg, 87%).</div><div class="NLM_p last">IR Î½<sub>max</sub> (film): 2965, 2856, 1639, 1112 cm<sup>â1</sup>. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 7.42 (d, <i>J</i> = 8.8 Hz, 1H), 6.81 (d, <i>J</i> = 2.9 Hz, 1H), 6.78 (dd, <i>J</i> = 8.8, 2.9 Hz, 1H), 3.64â3.43 (m, 8H), 3.41â3.33 (m, 2H), 3.41â3.33 (m, 2H), 3.30 (dd, <i>J</i> = 9.1, 7.4 Hz, 1H), 2.18 (dtd, <i>J</i> = 12.3, 7.2, 5.1 Hz, 1H), 2.15â2.06 (m, 1H). <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 170.7, 146.1, 131.9, 126.8 (q, <i>J</i> = 30.0 Hz), 123.1 (q, <i>J</i> = 273.1 Hz), 116.2, 115.1, 109.7 (q, <i>J</i> = 5.4 Hz), 66.2 (d, <i>J</i> = 26.1 Hz), 50.3, 47.2, 45.5, 41.9, 28.5. HRMS: calcd for C<sub>16</sub>H<sub>18</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>2</sub>, 363.1082 [M + H]<sup>+</sup>; found, 363.1081.</div></div><div id="sec4_1_3_80" class="NLM_sec NLM_sec_level_4"><div id="ac_i111" class="anchor-spacer"></div><h5 class="article-section__title" id="_i111"> (1-(4-Chloro-3-(trifluoromethyl)phenyl)pyrrolidin-3-yl)(tetrahydro-1<i>H</i>-furo[3,4-<i>c</i>]pyrrol-5(3<i>H</i>)-yl)methanone (<b>28h</b>)</h5><div class="NLM_p">The compound was prepared by General Method 8 with hexahydro-1<i>H</i>-furo[3,4-<i>c</i>]pyrrole and isolated as a clear gum (63.0 mg, 95%).</div><div class="NLM_p last">IR Î½<sub>max</sub> (film): 2962, 2863, 1635, 1114 cm<sup>â1</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 7.42 (d, <i>J</i> = 8.8 Hz, 1H), 6.81 (d, <i>J</i> = 2.6 Hz, 1H), 6.80â6.76 (m, 1H), 3.84â3.71 (m, 3H), 3.61â3.45 (m, 5H), 3.40â3.33 (m, 3H), 3.33â3.25 (m, 2H), 3.02â2.94 (m, 1H), 2.90â2.83 (m, 1H), 2.24â2.14 (m, 1H), 2.13â2.03 (m, 1H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 170.3 (d, <i>J</i> = 3.4 Hz), 146.1, 131.9, 126.8 (q, <i>J</i> = 30.0 Hz), 123.2 (q, <i>J</i> = 273.0 Hz), 116.3, 115.1, 109.7 (q), 73.0 (d, <i>J</i> = 7.1 Hz), 72.7 (d, <i>J</i> = 10.4 Hz), 50.5 (d, <i>J</i> = 3.5 Hz), 50.2, 50.2 (d, <i>J</i> = 8.2 Hz), 47.3 (d, <i>J</i> = 2.1 Hz), 43.5, 41.6, 41.3, 28.4 (d, <i>J</i> = 13.1 Hz). LCMS (basic method): <i>t</i><sub>R</sub> 1.90 min, <i>m</i>/<i>z</i> 389.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3_81" class="NLM_sec NLM_sec_level_4"><div id="ac_i112" class="anchor-spacer"></div><h5 class="article-section__title" id="_i112"> (1-(4-Chloro-3-(trifluoromethyl)phenyl)pyrrolidine-3-carbonyl)-3-methylimidazolidin-4-one (<b>28i</b>)</h5><div class="NLM_p">The compound was prepared by General Method 8 with 3-methylimidazolidin-4-one and isolated as a clear gum (47 mg, 74%).</div><div class="NLM_p last">IR Î½<sub>max</sub> (film): 2956, 2868, 1702, 1634, 1113 cm<sup>â1</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ rotamers 7.48â7.40 (m, 1H), 6.88â6.81 (m, 1H), 6.81â6.73 (m, 1H), 5.04 (dd, <i>J</i> = 16.7, 5.4 Hz, 0.5H), 4.73 (s, 1.5H), 4.22 (q, <i>J</i> = 14.9 Hz, 1.5H), 3.84 (s, 0.5H), 3.59â3.54 (m, 1H), 3.45â3.23 (m, 4H), 2.86â2.75 (m, 3H), 2.31â2.19 (m, 1H), 2.18â2.09 (m, 1H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ rotamers 170.5, 169.9, 167.2, 167.0, 146.0, 132.0, 131.9, 126.9 (q, <i>J</i> = 30.0 Hz), 123.2 (q, <i>J</i> = 273.1 Hz), 116.3, 109.8 (q, <i>J</i> = 5.8 Hz), 63.7, 50.1, 49.9, 47.8, 47.4, 47.3, 47.2, 41.0, 28.2, 27.9, 27.11, 27.08. HRMS: calcd for C<sub>16</sub>H<sub>17</sub>ClF<sub>3</sub>N<sub>3</sub>O<sub>2</sub>,376.1040 [M + H]<sup>+</sup>; found, 376.1029.</div></div><div id="sec4_1_3_82" class="NLM_sec NLM_sec_level_4"><div id="ac_i113" class="anchor-spacer"></div><h5 class="article-section__title" id="_i113"> (1-(4-Chloro-3-(trifluoromethyl)phenyl)pyrrolidine-3-carbonyl)-1-methylpiperazin-2-one (<b>28j</b>)</h5><div class="NLM_p">The compound was prepared by General Method 8 with 1-methylpiperazin-2-one and isolated as a white solid (64 mg, 97%).</div><div class="NLM_p last">IR Î½<sub>max</sub> (film): 2954, 2847, 1636, 1132 cm<sup>â1</sup>. <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 7.42 (dd, <i>J</i> = 8.8, 3.8 Hz, 1H), 6.86â6.80 (m, 1H), 6.80â6.76 (m, 1H), 4.30â4.13 (m, 1H), 4.01 (s, 1H), 3.93â3.77 (m, 1H), 3.77â3.66 (m, 1H), 3.66â3.45 (m, 2H), 3.45â3.25 (m, 5H), 2.87 (s, 2H), 2.28â2.04 (m, 3H). <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ rotamers 170.8, 170.6, 164.6, 164.2, 146.0, 131.9, 126.8 (q, <i>J</i> = 29.9 Hz), 123.1 (q, <i>J</i> = 273.1 Hz), 116.2, 115.2, 115.1, 109.7 (q, <i>J</i> = 5.5 Hz), 50.3, 50.2, 48.4, 47.9, 47.2, 47.1, 45.9, 41.8, 38.5, 33.7, 33.5, 28.5, 28.4. LCMS (basic method): <i>t</i><sub>R</sub> 1.80 min, <i>m</i>/<i>z</i> 390.2 [M + H]<sup>+</sup>. HRMS: calcd for C<sub>17</sub>H<sub>19</sub>ClF<sub>3</sub>N<sub>3</sub>O<sub>2</sub>, 390.1191 [M + H]<sup>+</sup>; found, 390.1190.</div></div><div id="sec4_1_3_83" class="NLM_sec NLM_sec_level_4"><div id="ac_i114" class="anchor-spacer"></div><h5 class="article-section__title" id="_i114"> 2-(1-(4-Chloro-3-(trifluoromethyl)phenyl)pyrrolidin-3-yl)-1,3,4-oxadiazole (<b>29</b>)</h5><div class="NLM_p">Hydrazine monohydrate (0.05 mL, 1.0 mmol) was added to a thick-walled reaction vial (20 mL) containing a solution of methyl 1-[4-chloro-3-(trifluoromethyl)phenyl]pyrrolidine-3-carboxylate (100 mg, 0.30 mmol) in EtOH (2.0 mL) and PhMe (1.0 mL). The vial was then sealed and heated to 80 Â°C for 16 h. The reaction mixture was concentrated under reduced pressure, azeotroped with EtOH/PhMe (1:1, 4 Ã 20 mL), and 1-(4-chloro-3-(trifluoromethyl)phenyl)pyrrolidine-3-carbohydrazide was isolated as an off-white solid, which was carried forward without further purification.</div><div class="NLM_p">Triethyl orthoformate (0.52 mL, 3.4 mmol) was added to a thick-walled reaction vial (10 mL) containing 1-[4-chloro-3-(trifluoromethyl)phenyl]pyrrolidine-3-carbohydrazide (0.76 mmol) and <i>p</i>-toluenesulfonic acid monohydrate (0.08 mmol) under N<sub>2</sub> and the mixture warmed to 60 Â°C for 1 h. The cooled mixture was diluted with saturated aq NaHCO<sub>3</sub> (5 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (Ã3), and the combined organic extracts were dried (MgSO<sub>4</sub>) and concentrated under reduced pressure. The residue was purified by column chromatography (0â5% MeOH/CH<sub>2</sub>Cl<sub>2</sub>) to give <b>29</b> as a yellow oil (125 mg, 52%).</div><div class="NLM_p last">IR Î½<sub>max</sub> (film): 2920, 2855, 1609, 1114, 808 cm<sup>â1</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 9.21 (s, 1H), 7.44 (d, <i>J</i> = 8.8 Hz, 1H), 6.89â6.81 (m, 2H), 4.00â3.90 (m, 1H), 3.76 (dd, <i>J</i> = 9.7, 7.6 Hz, 1H), 3.60 (dd, <i>J</i> = 9.8, 6.0 Hz, 1H), 3.48â3.38 (m, 2H), 2.51â2.42 (m, 1H), 2.36â2.25 (m, 1H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 166.7, 154.7, 145.9, 132.0, 126.9 (q, <i>J</i> = 30.1 Hz), 123.2 (q, <i>J</i> = 273.1 Hz), 116.5, 115.6, 109.9 (q, <i>J</i> = 5.6 Hz), 50.7, 46.8, 34.4, 29.1. HRMS: calcd for C<sub>13</sub>H<sub>11</sub>ClF<sub>3</sub>N<sub>3</sub>O, 318.0616 [M + H]<sup>+</sup>; found, 318.0610.</div></div><div id="sec4_1_3_84" class="NLM_sec NLM_sec_level_4"><div id="ac_i115" class="anchor-spacer"></div><h5 class="article-section__title" id="_i115"> 5-(1-(4-Chloro-3-(trifluoromethyl)phenyl)pyrrolidin-3-yl)-1,3,4-oxadiazol-2(3<i>H</i>)-one (<b>30</b>)</h5><div class="NLM_p">To a solution of 1-(4-chloro-3-(trifluoromethyl)phenyl)pyrrolidine-3-carbohydrazide (0.30 mmol) and DIPEA (0.11 mL, 0.65 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (5 mL) at 0 Â°C was added dropwise a solution of triphosgene (39 mg, 0.10 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (5 mL). The reaction mixture was warmed to RT and stirred for 10 min. The reaction was diluted with sat. aq NaHCO<sub>3</sub> solution (2 mL), and the organic phase was separated, dried (MgSO<sub>4</sub>), and concentrated under reduced pressure. The residue was dissolved in a minimum volume of DMSO and purified by column chromatography (0â100% MeCN/H<sub>2</sub>O, 0.1% NH<sub>4</sub>OH modifier) to give <b>30</b> as a white solid (104 mg, 34%).</div><div class="NLM_p last">Mp: 98 Â°C. IR Î½<sub>max</sub> (film): 3166, 2900, 2850, 1771, 1608, 1115, 808 cm<sup>â1</sup>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 12.20 (s, 1H), 7.44 (d, <i>J</i> = 8.8 Hz, 1H), 6.85 (d, <i>J</i> = 2.8 Hz, 1H), 6.82 (dd, <i>J</i> = 8.8, 2.8 Hz, 1H), 3.65â3.57 (m, 2H), 3.57â3.48 (m, 1H), 3.45â3.35 (m, 2H), 2.36â2.28 (m, 1H), 2.26â2.16 (m, 1H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): Î´ 157.2, 155.0, 145.9, 132.0, 126.9 (q, <i>J</i> = 30.0 Hz), 123.2 (q, <i>J</i> = 273.2 Hz), 116.4, 115.6, 109.9 (q, <i>J</i> = 5.5 Hz), 49.7, 46.7, 35.3, 28.0. HRMS: calcd for C<sub>13</sub>H<sub>11</sub>ClF<sub>3</sub>N<sub>3</sub>O<sub>2</sub>, 334.0570 [M + H]<sup>+</sup>; found, 334.0564.</div></div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i116" class="anchor-spacer"></div><h3 class="article-section__title" id="_i116"> Notum Protein Expression and Purification</h3><div class="NLM_p last">Methods have been described in detail.<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20,21)</a> In brief, the human Notum (UniProtKB ID: <a href="https://Q6P988" class="ext-link">Q6P988</a>) enzyme core sequence comprising amino acids S81âT451 with a C330S mutation was cloned into a stable cell line vector pNeo_sec. A stable HEK293S GNTI-cell line was obtained and used for protein production as described previously. The cells were expanded and grown in roller bottles (Greiner). The conditioned media were dialyzed and passed through a 5 mL HisTrap Excel column (GE Healthcare), followed by a 20 mM imidazole PBS wash. Notum protein was eluted with 300 mM imidazole PBS. To remove flexible glycans, the protein was deglycosylated with endo-Î²-<i>N</i>-acetylglucosaminidase F1 (37 Â°C, 1 h) and further purified by size-exclusion chromatography (Superdex 200 16/60 column, GE Healthcare) in 10 mM Hepes, pH 7.4, 150 mM NaCl buffer.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i117" class="anchor-spacer"></div><h3 class="article-section__title" id="_i117"> Notum OPTS Biochemical Assay</h3><div class="NLM_p">Methods have been described in detail.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> A Labcyte Echo 550 acoustic liquid handler was used to dispense 500 nL of the test compound into 384-well plates (Greiner catalog no. 781076). Twenty-five Î¼L of 2 Î¼M trisodium 8-octanoyloxypyrene-1,3,6-trisulfonate (OPTS, Sigma no. 74875) solution in 50 mM Tris, 5 mM CaCl<sub>2</sub>, 0.5 mM MgCl<sub>2</sub>, pH 7.4 assay buffer was added to each well, followed by 25 Î¼L of 2.38 nM Notum carboxylesterase enzyme solution. Plates were incubated for 40 min at room temperature, and the end point fluorescence was measured on a PheraSTAR FSX microplate reader with an excitation wavelength of 485 nm and an emission wavelength of 520 nm. Plates were quality controlled based on a signal window greater than 3-fold between DMSO and the positive control compound and a Z prime greater than 0.5.</div><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i118" class="anchor-spacer"></div><h4 class="article-section__title" id="_i118"> DoseâResponse Curve Analysis</h4><div class="NLM_p last">Plate layout and measured fluorescence values were input into the Dotmatics Studies data management tool. After implementing Quality Control with a cutoff of Z prime values higher than 0.5, compound IC<sub>50</sub> values were calculated from curves using a 4PL fit and compared between technical replicates. Final potency values implied an <i>n</i> â¥ 2 with IC<sub>50</sub> within 0.5â2Ã between replicates.</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i119" class="anchor-spacer"></div><h4 class="article-section__title" id="_i119"> Data Analysis</h4><div class="NLM_p last">IC<sub>50</sub> values presented in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Tables <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>â<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a> are arithmetic mean Â± SD (<i>n</i> = 2â10).</div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i120" class="anchor-spacer"></div><h3 class="article-section__title" id="_i120"> <i>In Vitro</i> ADME Screens</h3><div class="NLM_p">Selected compounds were screened for aqueous solubility, transit performance in MDCK-MDR1 cell lines for permeability, and metabolic stability in mouse liver microsomes as a measure of clearance. Assay protocols and additional data are presented in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00660/suppl_file/jm0c00660_si_001.pdf" class="ext-link">Figure S4</a>.</div><div class="NLM_p last">ADME studies reported in this work were performed by GVK Biosciences (Hyderabad, India). See <a href="https://www.gvkbio.com/discovery-services/biology-services/dmpk-services/" class="extLink">https://www.gvkbio.com/discovery-services/biology-services/dmpk-services/</a>.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i121"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00660" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68776" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68776" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00660?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00660</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Acid fragment library of 250 compounds (catalogue nos. and SMILES); X-ray structure determination including PDB codes, data collection, and refinement; chemical structures for inactive pyrrole and pyrrolidine-3-carboxylic acids present in the acid fragment library; inhibition of Notum activity of additional pyrroles and pyrrolidines; spectroscopic and analytical data for (<i>S</i>)-<b>26</b>; ADME protocols and data; Notum protein production, crystallization, compound soaking, data collection, and structure determination (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00660/suppl_file/jm0c00660_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00660/suppl_file/jm0c00660_si_002.xlsx" class="ext-link">XLSX</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00660/suppl_file/jm0c00660_si_001.pdf">jm0c00660_si_001.pdf (1.15 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00660/suppl_file/jm0c00660_si_002.xlsx">jm0c00660_si_002.xlsx (14.35 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates for X-ray structures of Notum crystallized with <b>5</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YUY">6YUY</a>), <b>6</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YV4">6YV4</a>), <b>7</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YUW">6YUW</a>), <b>8</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YV2">6YV2</a>), <b>10</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YV0">6YV0</a>), <b>20i</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YXI">6YXI</a>), and <b>26</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YSK">6YSK</a>) have been deposited in the Protein Data Bank.</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c00660" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44867" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44867" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paul V. Fish</span> - <span class="hlFld-Affiliation affiliation">Alzheimerâs
Research UK UCL Drug Discovery Institute, University College London, Cruciform Building, Gower Street, London WC1E 6BT, United Kingdom</span>;Â 
    <span class="hlFld-Affiliation affiliation">The
Francis Crick Institute, 1 Midland Road, Kings Cross, London NW1 1AT, United Kingdom</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2117-2173" title="Orcid link">http://orcid.org/0000-0002-2117-2173</a>;Â 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#3f4f1159564c577f4a5c53115e5c114a54"><span class="__cf_email__" data-cfemail="12623c747b617a5267717e3c73713c6779">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">William Mahy</span> - <span class="hlFld-Affiliation affiliation">Alzheimerâs
Research UK UCL Drug Discovery Institute, University College London, Cruciform Building, Gower Street, London WC1E 6BT, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mikesh Patel</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University College London, 20 Gordon Street, London WC1H 0AJ, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David Steadman</span> - <span class="hlFld-Affiliation affiliation">Alzheimerâs
Research UK UCL Drug Discovery Institute, University College London, Cruciform Building, Gower Street, London WC1E 6BT, United Kingdom</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4271-5525" title="Orcid link">http://orcid.org/0000-0003-4271-5525</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hannah L. Woodward</span> - <span class="hlFld-Affiliation affiliation">Alzheimerâs
Research UK UCL Drug Discovery Institute, University College London, Cruciform Building, Gower Street, London WC1E 6BT, United Kingdom</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8429-2546" title="Orcid link">http://orcid.org/0000-0001-8429-2546</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Benjamin N. Atkinson</span> - <span class="hlFld-Affiliation affiliation">Alzheimerâs
Research UK UCL Drug Discovery Institute, University College London, Cruciform Building, Gower Street, London WC1E 6BT, United Kingdom</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5511-9859" title="Orcid link">http://orcid.org/0000-0001-5511-9859</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fredrik Svensson</span> - <span class="hlFld-Affiliation affiliation">Alzheimerâs
Research UK UCL Drug Discovery Institute, University College London, Cruciform Building, Gower Street, London WC1E 6BT, United Kingdom</span>;Â 
    <span class="hlFld-Affiliation affiliation">The
Francis Crick Institute, 1 Midland Road, Kings Cross, London NW1 1AT, United Kingdom</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5556-8133" title="Orcid link">http://orcid.org/0000-0002-5556-8133</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicky J. Willis</span> - <span class="hlFld-Affiliation affiliation">Alzheimerâs
Research UK UCL Drug Discovery Institute, University College London, Cruciform Building, Gower Street, London WC1E 6BT, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alister Flint</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry, University College London, 20 Gordon Street, London WC1H 0AJ, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dimitra Papatheodorou</span> - <span class="hlFld-Affiliation affiliation">UCL
School of Pharmacy, University College London, 29-39 Brunswick Square, Bloomsbury, London WC1N 1AX, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yuguang Zhao</span> - <span class="hlFld-Affiliation affiliation">Division
of Structural Biology, Wellcome Centre for Human Genetics, University of Oxford, The Henry Wellcome Building for Genomic Medicine,
Roosevelt Drive, Oxford OX3 7BN, United Kingdom</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8916-8552" title="Orcid link">http://orcid.org/0000-0001-8916-8552</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Luca Vecchia</span> - <span class="hlFld-Affiliation affiliation">Division
of Structural Biology, Wellcome Centre for Human Genetics, University of Oxford, The Henry Wellcome Building for Genomic Medicine,
Roosevelt Drive, Oxford OX3 7BN, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Reinis R. Ruza</span> - <span class="hlFld-Affiliation affiliation">Division
of Structural Biology, Wellcome Centre for Human Genetics, University of Oxford, The Henry Wellcome Building for Genomic Medicine,
Roosevelt Drive, Oxford OX3 7BN, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James Hillier</span> - <span class="hlFld-Affiliation affiliation">Division
of Structural Biology, Wellcome Centre for Human Genetics, University of Oxford, The Henry Wellcome Building for Genomic Medicine,
Roosevelt Drive, Oxford OX3 7BN, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sarah Frew</span> - <span class="hlFld-Affiliation affiliation">Alzheimerâs
Research UK UCL Drug Discovery Institute, University College London, Cruciform Building, Gower Street, London WC1E 6BT, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Amy Monaghan</span> - <span class="hlFld-Affiliation affiliation">Alzheimerâs
Research UK UCL Drug Discovery Institute, University College London, Cruciform Building, Gower Street, London WC1E 6BT, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Artur Costa</span> - <span class="hlFld-Affiliation affiliation">Alzheimerâs
Research UK UCL Drug Discovery Institute, University College London, Cruciform Building, Gower Street, London WC1E 6BT, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Magda Bictash</span> - <span class="hlFld-Affiliation affiliation">Alzheimerâs
Research UK UCL Drug Discovery Institute, University College London, Cruciform Building, Gower Street, London WC1E 6BT, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Magnus W. Walter</span> - <span class="hlFld-Affiliation affiliation">Eli Lilly, Erl Wood Manor, Windlesham, Surrey GU20 6PH, United Kingdom</span>;Â 
    <span>Present Address:
                        M.E.W.: AbbVie Germany, Knollstr., 67061 Ludwigshafen, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">E. Yvonne Jones</span> - <span class="hlFld-Affiliation affiliation">Division
of Structural Biology, Wellcome Centre for Human Genetics, University of Oxford, The Henry Wellcome Building for Genomic Medicine,
Roosevelt Drive, Oxford OX3 7BN, United Kingdom</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3834-1893" title="Orcid link">http://orcid.org/0000-0002-3834-1893</a></span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i124">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70374" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70374" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank our colleagues Jim Anderson, Sarah Jolly, Patricia Salinas, J.-P. Vincent, and Paul Whiting for their support and advice. The Cell Services and Structural Biology Science Technology Platforms (STPs) at the Francis Crick Institute are gratefully acknowledged for their provision and purification of recombinant Notum. We thank the staff at Diamond Light Source for assistance with X-ray data collection and the UCL Department of Chemistry for spectroscopic and analytical services. Elliott Bayle is gratefully acknowledged for the determination of CHI log<i>D</i><sub>7.4</sub> values. ADME studies reported in this work were performed by GVK Biosciences (Hyderabad, India). Chemical and chiral SFC analysis were performed by Ed Hogarth and Daniel von Emloh at Reach Separations Ltd (Nottingham, UK). This work was supported by Alzheimerâs Research UK (ARUK), the Francis Crick Institute, and Eli Lilly. The ARUK UCL Drug Discovery Institute is core funded by Alzheimerâs Research UK (520909). The Francis Crick Institute receives its core funding from Cancer Research UK (FC001002), the UK Medical Research Council (FC001002), and the Wellcome Trust (FC001002). W.M. received funding from the UCL Innovative Therapeutics Postdoctoral Training Programme, which was sponsored, in part by Eli Lilly, and in part by EPSRC UCL IAA, and led by Judy Hayler (UCL Translational Research Office.) Structural analysis was performed by Y.Z., L.V., R.R.R., and E.Y.J. supported by Cancer Research UK (Program Grant C375/A17721) and the Wellcome Trust funds the Wellcome Centre for Human Genetics, University of Oxford (Centre Grant 203141/Z/16/Z).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">AD</td><td class="NLM_def"><p class="first last">Alzheimerâs disease</p></td></tr><tr><td class="NLM_term">ADME</td><td class="NLM_def"><p class="first last">absorption distribution metabolism elimination</p></td></tr><tr><td class="NLM_term">CHI</td><td class="NLM_def"><p class="first last">chromatographic hydrophobicity index</p></td></tr><tr><td class="NLM_term">CNS</td><td class="NLM_def"><p class="first last">central nervous system</p></td></tr><tr><td class="NLM_term">CRC</td><td class="NLM_def"><p class="first last">colorectal cancer</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N-</i>diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-dimethylformamide</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">ER</td><td class="NLM_def"><p class="first last">efflux ratio</p></td></tr><tr><td class="NLM_term">FZD</td><td class="NLM_def"><p class="first last">Frizzled receptors</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">(1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxide hexafluorophosphate</p></td></tr><tr><td class="NLM_term">HPLC</td><td class="NLM_def"><p class="first last">high-performance liquid chromatography</p></td></tr><tr><td class="NLM_term">LE</td><td class="NLM_def"><p class="first last">ligand efficiency</p></td></tr><tr><td class="NLM_term">LLE</td><td class="NLM_def"><p class="first last">lipophilic ligand efficiency</p></td></tr><tr><td class="NLM_term">MLM</td><td class="NLM_def"><p class="first last">mouse liver microsomes</p></td></tr><tr><td class="NLM_term">MPO</td><td class="NLM_def"><p class="first last">multiparameter optimization</p></td></tr><tr><td class="NLM_term">OPTS</td><td class="NLM_def"><p class="first last">trisodium 8-octanoyloxypyrene-1,3,6-trisulfonate</p></td></tr><tr><td class="NLM_term">P-gp</td><td class="NLM_def"><p class="first last">P-glycoprotein</p></td></tr><tr><td class="NLM_term">PMT</td><td class="NLM_def"><p class="first last">post-translational modifications</p></td></tr><tr><td class="NLM_term">RT</td><td class="NLM_def"><p class="first last">room temperature</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structureâactivity relationship</p></td></tr><tr><td class="NLM_term">SBDD</td><td class="NLM_def"><p class="first last">structure-based drug design</p></td></tr><tr><td class="NLM_term">SFC</td><td class="NLM_def"><p class="first last">supercritical fluid chromatography</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">TPSA</td><td class="NLM_def"><p class="first last">topological polar surface area</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i126">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_03321" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_03321" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 38 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nusse, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clevers, H.</span></span> <span> </span><span class="NLM_article-title">Wnt/Î²-Catenin signaling, disease, and emerging therapeutic modalities</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>169</i></span>,  <span class="NLM_fpage">985</span>â <span class="NLM_lpage">999</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2017.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1016%2Fj.cell.2017.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=28575679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpt1agsro%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=169&publication_year=2017&pages=985-999&author=R.+Nusseauthor=H.+Clevers&title=Wnt%2F%CE%B2-Catenin+signaling%2C+disease%2C+and+emerging+therapeutic+modalities&doi=10.1016%2Fj.cell.2017.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Wnt/Î²-catenin signaling, disease, and emerging therapeutic modalities</span></div><div class="casAuthors">Nusse, Roel; Clevers, Hans</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">169</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">985-999</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The WNT signal transduction cascade is a main regulator of development throughout the animal kingdom.  Wnts are also key drivers of most types of tissue stem cells in adult mammals.  Unsurprisingly, mutated Wnt pathway components are causative to multiple growth-related pathologies and to cancer.  Here, we describe the core Wnt/Î²-catenin signaling pathway, how it controls stem cells, and contributes to disease.  Finally, we discuss strategies for Wnt-based therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnL11nCbSs8LVg90H21EOLACvtfcHk0lgGkUgM0cxxnA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpt1agsro%253D&md5=869cc558296677481527ac138f68fbb0</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DNusse%26aufirst%3DR.%26aulast%3DClevers%26aufirst%3DH.%26atitle%3DWnt%252F%25CE%25B2-Catenin%2520signaling%252C%2520disease%252C%2520and%2520emerging%2520therapeutic%2520modalities%26jtitle%3DCell%26date%3D2017%26volume%3D169%26spage%3D985%26epage%3D999%26doi%3D10.1016%2Fj.cell.2017.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niehrs, C.</span></span> <span> </span><span class="NLM_article-title">Function and biological roles of the Dickkopf family of Wnt modulators</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">7469</span>â <span class="NLM_lpage">7481</span>, <span class="refDoi">Â DOI: 10.1038/sj.onc.1210054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1038%2Fsj.onc.1210054" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=17143291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1Kgt7%252FN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=7469-7481&author=C.+Niehrs&title=Function+and+biological+roles+of+the+Dickkopf+family+of+Wnt+modulators&doi=10.1038%2Fsj.onc.1210054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Function and biological roles of the Dickkopf family of Wnt modulators</span></div><div class="casAuthors">Niehrs, C.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">57</span>),
    <span class="NLM_cas:pages">7469-7481</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Dickkopf (Dkk) genes comprise an evolutionary conserved small gene family of 4 members (Dkk1-4) and a unique Dkk3-related gene, Dkkl1 (soggy).  They encode secreted proteins that typically antagonize Wnt/Î²-catenin signaling, by inhibiting the Wnt coreceptors Lrp5 and 6.  Addnl., Dkks are high affinity ligands for the transmembrane proteins Kremen1 and 2, which also modulate Wnt signaling.  Dkks play an important role in vertebrate development, where they locally inhibit Wnt regulated processes such as antero-posterior axial patterning, limb development, somitogenesis, and eye formation.  In the adult, Dkks are implicated in bone formation and bone disease, cancer, and Alzheimer's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-xPIRQqosmbVg90H21EOLACvtfcHk0ljKHsXBJtoHPg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1Kgt7%252FN&md5=f871bcae6fe6f367d650f423723eef84</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1210054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1210054%26sid%3Dliteratum%253Aachs%26aulast%3DNiehrs%26aufirst%3DC.%26atitle%3DFunction%2520and%2520biological%2520roles%2520of%2520the%2520Dickkopf%2520family%2520of%2520Wnt%2520modulators%26jtitle%3DOncogene%26date%3D2006%26volume%3D25%26spage%3D7469%26epage%3D7481%26doi%3D10.1038%2Fsj.onc.1210054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niehrs, C.</span></span> <span> </span><span class="NLM_article-title">The complex world of WNT receptor signaling</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">767</span>â <span class="NLM_lpage">779</span>, <span class="refDoi">Â DOI: 10.1038/nrm3470</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1038%2Fnrm3470" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=23151663" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Klsr%252FM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=767-779&author=C.+Niehrs&title=The+complex+world+of+WNT+receptor+signaling&doi=10.1038%2Fnrm3470"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The complex world of WNT receptor signalling</span></div><div class="casAuthors">Niehrs, Christof</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">767-779</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  30 Years after the identification of WNTs, their signal transduction has become increasingly complex, with the discovery of more than 15 receptors and co-receptors in seven protein families.  The recent discovery of three receptor classes for the R-spondin family of WNT agonists further adds to this complexity.  What emerges is an intricate network of receptors that form higher-order ligand-receptor complexes routing downstream signaling.  These are regulated both extracellularly by agonists such as R-spondin and intracellularly by post-translational modifications such as phosphorylation, proteolytic processing and endocytosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpb-xsoqljhjLVg90H21EOLACvtfcHk0ljKHsXBJtoHPg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Klsr%252FM&md5=b0f3fe641f7a5f77774ab54d8cc5aeb9</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrm3470&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3470%26sid%3Dliteratum%253Aachs%26aulast%3DNiehrs%26aufirst%3DC.%26atitle%3DThe%2520complex%2520world%2520of%2520WNT%2520receptor%2520signaling%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2012%26volume%3D13%26spage%3D767%26epage%3D779%26doi%3D10.1038%2Fnrm3470" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malinauskas, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. Y.</span></span> <span> </span><span class="NLM_article-title">Extracellular modulators of Wnt signaling</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">77</span>â <span class="NLM_lpage">84</span>, <span class="refDoi">Â DOI: 10.1016/j.sbi.2014.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1016%2Fj.sbi.2014.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=25460271" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslKjtrrJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2014&pages=77-84&author=T.+Malinauskasauthor=E.+Y.+Jones&title=Extracellular+modulators+of+Wnt+signaling&doi=10.1016%2Fj.sbi.2014.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Extracellular modulators of Wnt signalling</span></div><div class="casAuthors">Malinauskas, Tomas; Jones, E. Yvonne</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">77-84</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Wnt morphogens are secreted signalling proteins that play leading roles in embryogenesis and tissue homeostasis throughout life.  Wnt signalling is controlled by multiple mechanisms, including posttranslational modification of Wnts, antagonist binding (to Wnts or their receptors), and regulation of the availability of Wnt receptors.  Recent crystallog., structure-guided biophys. and cell-based studies have advanced our understanding of how Wnt signalling is regulated at the cell surface.  Structures include Wnt in complex with the cysteine-rich domain (CRD) of Frizzled, extracellular fragments of Wnt co-receptor LRP6, LRP6-binding antagonists Dickkopf and Sclerostin, antagonists 5T4/WAIF1 and Wnt inhibitory factor 1 (WIF-1), as well as Frizzled-ubiquitin ligases ZNRF3/RNF43 (in isolation and in complexes with Wnt signalling promoters R-spondins and LGR5).  We review recent discoveries and remaining questions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuYvmbUHeJhLVg90H21EOLACvtfcHk0lgXe0yhGL4MaQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslKjtrrJ&md5=68ae12b2925971654ea428a3ae4179ca</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2014.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2014.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DMalinauskas%26aufirst%3DT.%26aulast%3DJones%26aufirst%3DE.%2BY.%26atitle%3DExtracellular%2520modulators%2520of%2520Wnt%2520signaling%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2014%26volume%3D29%26spage%3D77%26epage%3D84%26doi%3D10.1016%2Fj.sbi.2014.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rindtorff, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutros, M.</span></span> <span> </span><span class="NLM_article-title">Wnt signaling in cancer</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">1461</span>â <span class="NLM_lpage">1473</span>, <span class="refDoi">Â DOI: 10.1038/onc.2016.304</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1038%2Fonc.2016.304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=27617575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsV2gsL3O" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2017&pages=1461-1473&author=T.+Zhanauthor=N.+Rindtorffauthor=M.+Boutros&title=Wnt+signaling+in+cancer&doi=10.1038%2Fonc.2016.304"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Wnt signaling in cancer</span></div><div class="casAuthors">Zhan, T.; Rindtorff, N.; Boutros, M.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1461-1473</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Wnt signaling is one of the key cascades regulating development and stemness, and has also been tightly assocd. with cancer.  The role of Wnt signaling in carcinogenesis has most prominently been described for colorectal cancer, but aberrant Wnt signaling is obsd. in many more cancer entities.  Here, we review current insights into novel components of Wnt pathways and describe their impact on cancer development.  Furthermore, we highlight expanding functions of Wnt signaling for both solid and liq. tumors.  We also describe current findings how Wnt signaling affects maintenance of cancer stem cells, metastasis and immune control.  Finally, we provide an overview of current strategies to antagonize Wnt signaling in cancer and challenges that are assocd. with such approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5DnHKAd0M3rVg90H21EOLACvtfcHk0lgXe0yhGL4MaQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsV2gsL3O&md5=abef967a4e295e8d997ea2f241b035bc</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fonc.2016.304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2016.304%26sid%3Dliteratum%253Aachs%26aulast%3DZhan%26aufirst%3DT.%26aulast%3DRindtorff%26aufirst%3DN.%26aulast%3DBoutros%26aufirst%3DM.%26atitle%3DWnt%2520signaling%2520in%2520cancer%26jtitle%3DOncogene%26date%3D2017%26volume%3D36%26spage%3D1461%26epage%3D1473%26doi%3D10.1038%2Fonc.2016.304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Palomer, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buechler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salinas, P. C.</span></span> <span> </span><span class="NLM_article-title">Wnt signalling deregulation in the aging and Alzheimerâs brain</span>. <i>Front. Cell. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">227</span>, <span class="refDoi">Â DOI: 10.3389/fncel.2019.00227</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.3389%2Ffncel.2019.00227" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=31191253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BB3cXktV2ksLg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2019&pages=227&author=E.+Palomerauthor=J.+Buechlerauthor=P.+C.+Salinas&title=Wnt+signalling+deregulation+in+the+aging+and+Alzheimer%E2%80%99s+brain&doi=10.3389%2Ffncel.2019.00227"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Wnt signaling deregulation in the aging and Alzheimer's brain</span></div><div class="casAuthors">Palomer, Ernest; Buechler, Johanna; Salinas, Patricia C.</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Cellular Neuroscience</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">227</span>CODEN:
                <span class="NLM_cas:coden">FCNRAH</span>;
        ISSN:<span class="NLM_cas:issn">1662-5102</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">Growing evidence suggests that synaptic signaling is compromised in the aging brain and in Alzheimer's disease (AD), contributing to synaptic decline.  Wnt signaling is a prominent pathway at the synapse and is required for synaptic plasticity and maintenance in the adult brain.  In this review, we summarize the current knowledge on deregulation of Wnt signaling in the context of aging and AD.  Emerging studies suggest that enhancing Wnt signaling could boost synaptic function during aging, and ameliorate synaptic pathol. in AD.  Although further research is needed to det. the precise contribution of deficient Wnt signaling to AD pathogenesis, targeting Wnt signaling components may provide novel therapeutic avenues for synapse protection or restoration in the brain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBS_KuzUNAgbVg90H21EOLACvtfcHk0lgXe0yhGL4MaQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXktV2ksLg%253D&md5=e807df970865d7e712425076ef013ac2</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.3389%2Ffncel.2019.00227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffncel.2019.00227%26sid%3Dliteratum%253Aachs%26aulast%3DPalomer%26aufirst%3DE.%26aulast%3DBuechler%26aufirst%3DJ.%26aulast%3DSalinas%26aufirst%3DP.%2BC.%26atitle%3DWnt%2520signalling%2520deregulation%2520in%2520the%2520aging%2520and%2520Alzheimer%25E2%2580%2599s%2520brain%26jtitle%3DFront.%2520Cell.%2520Neurosci.%26date%3D2019%26volume%3D13%26spage%3D227%26doi%3D10.3389%2Ffncel.2019.00227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ng, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnag, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suresh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, I. C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Q. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolwinski, N. S.</span></span> <span> </span><span class="NLM_article-title">WNT signaling in disease</span>. <i>Cells</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">826</span>, <span class="refDoi">Â DOI: 10.3390/cells8080826</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.3390%2Fcells8080826" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlsVGis7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2019&pages=826&author=L.+F.+Ngauthor=P.+Kaurauthor=N.+Bunnagauthor=J.+Sureshauthor=I.+C.+H.+Sungauthor=Q.+H.+Tanauthor=J.+Gruberauthor=N.+S.+Tolwinski&title=WNT+signaling+in+disease&doi=10.3390%2Fcells8080826"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">WNT signaling in disease</span></div><div class="casAuthors">Ng, Li Fang; Kaur, Prameet; Bunnag, Nawat; Suresh, Jahnavi; Sung, Isabelle Chiao Han; Tan, Qian Hui; Gruber, Jan; Tolwinski, Nicholas S.</div><div class="citationInfo"><span class="NLM_cas:title">Cells</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">826</span>CODEN:
                <span class="NLM_cas:coden">CELLC6</span>;
        ISSN:<span class="NLM_cas:issn">2073-4409</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Developmental signaling pathways control a vast array of biol. processes during embryogenesis and in adult life.  The WNT pathway was discovered simultaneously in cancer and development.  Recent advances have expanded the role of WNT to a wide range of pathologies in humans.  Here, we discuss the WNT pathway and its role in human disease and some of the advances in WNT-related treatments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDDLPhmGEBILVg90H21EOLACvtfcHk0liCu1KFj1P-LQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlsVGis7w%253D&md5=f88171d5418fd12052faa352a7196326</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.3390%2Fcells8080826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcells8080826%26sid%3Dliteratum%253Aachs%26aulast%3DNg%26aufirst%3DL.%2BF.%26aulast%3DKaur%26aufirst%3DP.%26aulast%3DBunnag%26aufirst%3DN.%26aulast%3DSuresh%26aufirst%3DJ.%26aulast%3DSung%26aufirst%3DI.%2BC.%2BH.%26aulast%3DTan%26aufirst%3DQ.%2BH.%26aulast%3DGruber%26aufirst%3DJ.%26aulast%3DTolwinski%26aufirst%3DN.%2BS.%26atitle%3DWNT%2520signaling%2520in%2520disease%26jtitle%3DCells%26date%3D2019%26volume%3D8%26spage%3D826%26doi%3D10.3390%2Fcells8080826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kakugawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langton, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zebisch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feizi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bineva, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâReilly, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snijders, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, J. P.</span></span> <span> </span><span class="NLM_article-title">NOTUM deacylates Wnt proteins to suppress signaling activity</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>519</i></span>,  <span class="NLM_fpage">187</span>â <span class="NLM_lpage">192</span>, <span class="refDoi">Â DOI: 10.1038/nature14259</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1038%2Fnature14259" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=25731175" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC2MXktVCnsb4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=519&publication_year=2015&pages=187-192&author=S.+Kakugawaauthor=P.+F.+Langtonauthor=M.+Zebischauthor=S.+Howellauthor=T.+H.+Changauthor=Y.+Liuauthor=T.+Feiziauthor=G.+Binevaauthor=N.+O%E2%80%99Reillyauthor=A.+P.+Snijdersauthor=E.+Y.+Jonesauthor=J.+P.+Vincent&title=NOTUM+deacylates+Wnt+proteins+to+suppress+signaling+activity&doi=10.1038%2Fnature14259"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Notum deacylates Wnt proteins to suppress signalling activity</span></div><div class="casAuthors">Kakugawa, Satoshi; Langton, Paul F.; Zebisch, Matthias; Howell, Steven A.; Chang, Tao-Hsin; Liu, Yan; Feizi, Ten; Bineva, Ganka; O'Reilly, Nicola; Snijders, Ambrosius P.; Jones, E. Yvonne; Vincent, Jean-Paul</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">519</span>
        (<span class="NLM_cas:issue">7542</span>),
    <span class="NLM_cas:pages">187-192</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Signalling by Wnt proteins is finely balanced to ensure normal development and tissue homeostasis while avoiding diseases such as cancer.  This is achieved in part by Notum, a highly conserved secreted feedback antagonist.  Notum has been thought to act as a phospholipase, shedding glypicans and assocd. Wnt proteins from the cell surface.  However, this view fails to explain specificity, as glypicans bind many extracellular ligands.  Here we provide genetic evidence in Drosophila that Notum requires glypicans to suppress Wnt signalling, but does not cleave their glycophosphatidylinositol anchor.  Structural analyses reveal glycosaminoglycan binding sites on Notum, which probably help Notum to co-localize with Wnt proteins.  They also identify, at the active site of human and Drosophila Notum, a large hydrophobic pocket that accommodates palmitoleate.  Kinetic and mass spectrometric analyses of human proteins show that Notum is a carboxylesterase that removes an essential palmitoleate moiety from Wnt proteins and thus constitutes the first known extracellular protein deacylase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqffMMn2oxsbLVg90H21EOLACvtfcHk0liCu1KFj1P-LQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXktVCnsb4%253D&md5=0668dc3cdc07e4bd527551424c4b91d4</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnature14259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14259%26sid%3Dliteratum%253Aachs%26aulast%3DKakugawa%26aufirst%3DS.%26aulast%3DLangton%26aufirst%3DP.%2BF.%26aulast%3DZebisch%26aufirst%3DM.%26aulast%3DHowell%26aufirst%3DS.%26aulast%3DChang%26aufirst%3DT.%2BH.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DFeizi%26aufirst%3DT.%26aulast%3DBineva%26aufirst%3DG.%26aulast%3DO%25E2%2580%2599Reilly%26aufirst%3DN.%26aulast%3DSnijders%26aufirst%3DA.%2BP.%26aulast%3DJones%26aufirst%3DE.%2BY.%26aulast%3DVincent%26aufirst%3DJ.%2BP.%26atitle%3DNOTUM%2520deacylates%2520Wnt%2520proteins%2520to%2520suppress%2520signaling%2520activity%26jtitle%3DNature%26date%3D2015%26volume%3D519%26spage%3D187%26epage%3D192%26doi%3D10.1038%2Fnature14259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheong, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amado, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reis, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacDonald, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zebisch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abreu, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, X.</span></span> <span> </span><span class="NLM_article-title">NOTUM is required for neural and head induction via Wnt deacylation, oxidation, and inactivation</span>. <i>Dev. Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">719</span>â <span class="NLM_lpage">730</span>, <span class="refDoi">Â DOI: 10.1016/j.devcel.2015.02.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1016%2Fj.devcel.2015.02.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=25771893" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC2MXksVaisLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2015&pages=719-730&author=X.+Zhangauthor=S.+M.+Cheongauthor=N.+G.+Amadoauthor=A.+H.+Reisauthor=B.+T.+MacDonaldauthor=M.+Zebischauthor=E.+Y.+Jonesauthor=J.+G.+Abreuauthor=X.+He&title=NOTUM+is+required+for+neural+and+head+induction+via+Wnt+deacylation%2C+oxidation%2C+and+inactivation&doi=10.1016%2Fj.devcel.2015.02.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Notum Is Required for Neural and Head Induction via Wnt Deacylation, Oxidation, and Inactivation</span></div><div class="casAuthors">Zhang, Xinjun; Cheong, Seong-Moon; Amado, Nathalia G.; Reis, Alice H.; MacDonald, Bryan T.; Zebisch, Matthias; Jones, E. Yvonne; Abreu, Jose Garcia; He, Xi</div><div class="citationInfo"><span class="NLM_cas:title">Developmental Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">719-730</span>CODEN:
                <span class="NLM_cas:coden">DCEEBE</span>;
        ISSN:<span class="NLM_cas:issn">1534-5807</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Secreted Wnt morphogens are essential for embryogenesis and homeostasis and require a lipid/palmitoleoylate modification for receptor binding and activity.  Notum is a secreted Wnt antagonist that belongs to the Î±/Î² hydrolase superfamily, but its mechanism of action and roles in vertebrate embryogenesis are not fully understood.  Here, we report that Notum hydrolyzes the Wnt palmitoleoylate adduct extracellularly, resulting in inactivated Wnt proteins that form oxidized oligomers incapable of receptor binding.  Thus, Notum is a Wnt deacylase, and palmitoleoylation is obligatory for the Wnt structure that maintains its active monomeric conformation.  Notum is expressed in naive ectoderm and neural plate in Xenopus and is required for neural and head induction.  These findings suggest that Notum is a prerequisite for the "default" neural fate and that distinct mechanisms of Wnt inactivation by the Tiki protease in the Organizer and the Notum deacylase in presumptive neuroectoderm orchestrate vertebrate brain development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBxen8RtuF37Vg90H21EOLACvtfcHk0lj26rMN2GFaew"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXksVaisLk%253D&md5=2f5bd637c5b10522452933e9592a888b</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.devcel.2015.02.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.devcel.2015.02.014%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DCheong%26aufirst%3DS.%2BM.%26aulast%3DAmado%26aufirst%3DN.%2BG.%26aulast%3DReis%26aufirst%3DA.%2BH.%26aulast%3DMacDonald%26aufirst%3DB.%2BT.%26aulast%3DZebisch%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DE.%2BY.%26aulast%3DAbreu%26aufirst%3DJ.%2BG.%26aulast%3DHe%26aufirst%3DX.%26atitle%3DNOTUM%2520is%2520required%2520for%2520neural%2520and%2520head%2520induction%2520via%2520Wnt%2520deacylation%252C%2520oxidation%252C%2520and%2520inactivation%26jtitle%3DDev.%2520Cell%26date%3D2015%26volume%3D32%26spage%3D719%26epage%3D730%26doi%3D10.1016%2Fj.devcel.2015.02.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suh, P.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryu, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. G.</span></span> <span> </span><span class="NLM_article-title">NOTUM is involved in the progression of colorectal cancer</span>. <i>Cancer Genomics Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">485</span>â <span class="NLM_lpage">497</span>, <span class="refDoi">Â DOI: 10.21873/cgp.20107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.21873%2Fcgp.20107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=30343282" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlWlsr%252FM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=485-497&author=J.+H.+Yoonauthor=D.+Kimauthor=J.+Kimauthor=H.+Leeauthor=J.+Ghimauthor=B.+J.+Kangauthor=P.+Songauthor=P.-G.+Suhauthor=S.+H.+Ryuauthor=T.+G.+Lee&title=NOTUM+is+involved+in+the+progression+of+colorectal+cancer&doi=10.21873%2Fcgp.20107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">NOTUM Is involved in the progression of colorectal cancer</span></div><div class="casAuthors">Yoon, Jong Hyuk; Kim, Dayea; Kim, Jaeyoon; Lee, Hyeongjoo; Ghim, Jaewang; Kang, Byung Jun; Song, Parkyong; Suh, Pann-Ghill; Ryu, Sung Ho; Lee, Taehoon G.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Genomics & Proteomics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">485-497</span>CODEN:
                <span class="NLM_cas:coden">CGPAC7</span>;
        ISSN:<span class="NLM_cas:issn">1790-6245</span>.
    
            (<span class="NLM_cas:orgname">International Institute of Anticancer Research</span>)
        </div><div class="casAbstract">Background: There are limitations to current colorectal cancer (CRC)-specific diagnostic methods and therapies.  Tumorigenesis proceeds because of interaction between cancer cells and various surrounding cells; discovering new mol. mediators through studies of the CRC secretome is a promising approach for the development of CRC diagnostics and therapies.  Materials and Methods: A comparative secretomic anal. was performed using primary and metastatic human isogenic CRC cells.  Proliferation was detd. by MTT and thymidine incorporation assay, migration was detd. by wound-healing assay (ELISA).  The level of palmitoleoyl-protein carboxylesterase (NOTUM) in plasma from patients with CRC was detd. by ELISA.  Results: NOTUM expression was increased in metastatic cells.  Proliferation was suppressed by inhibiting expression of NOTUM.  Knockdown of NOTUM genes inhibited proliferation as well as migration, with possible involvement of p38 and c-JUN N-terminal kinase in this process.  The result was verified in patients with CRC.  Conclusion: NOTUM may be a new candidate for diagnostics and therapy of CRC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUMxJxU9DSy7Vg90H21EOLACvtfcHk0lj26rMN2GFaew"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlWlsr%252FM&md5=9970defbfb6dc7da63d869e2ba28909f</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.21873%2Fcgp.20107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.21873%252Fcgp.20107%26sid%3Dliteratum%253Aachs%26aulast%3DYoon%26aufirst%3DJ.%2BH.%26aulast%3DKim%26aufirst%3DD.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DGhim%26aufirst%3DJ.%26aulast%3DKang%26aufirst%3DB.%2BJ.%26aulast%3DSong%26aufirst%3DP.%26aulast%3DSuh%26aufirst%3DP.-G.%26aulast%3DRyu%26aufirst%3DS.%2BH.%26aulast%3DLee%26aufirst%3DT.%2BG.%26atitle%3DNOTUM%2520is%2520involved%2520in%2520the%2520progression%2520of%2520colorectal%2520cancer%26jtitle%3DCancer%2520Genomics%2520Proteomics%26date%3D2018%26volume%3D15%26spage%3D485%26epage%3D497%26doi%3D10.21873%2Fcgp.20107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moverare-Skrtic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilsson, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henning, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funck-Brentano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nethander, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivadeneira, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunes, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koskela, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuukkanen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuckermann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perret, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, P. P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerner, U. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohlsson, C.</span></span> <span> </span><span class="NLM_article-title">Osteoblast-derived NOTUM reduces cortical bone mass in mice and the NOTUM locus is associated with bone mineral density in humans</span>. <i>FASEB J.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">11163</span>â <span class="NLM_lpage">11179</span>, <span class="refDoi">Â DOI: 10.1096/fj.201900707R</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1096%2Ffj.201900707R" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=31307226" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVOltLrE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2019&pages=11163-11179&author=S.+Moverare-Skrticauthor=K.+H.+Nilssonauthor=P.+Henningauthor=T.+Funck-Brentanoauthor=M.+Nethanderauthor=F.+Rivadeneiraauthor=G.+C.+Nunesauthor=A.+Koskelaauthor=J.+Tuukkanenauthor=J.+Tuckermannauthor=C.+Perretauthor=P.+P.+C.+Souzaauthor=U.+H.+Lernerauthor=C.+Ohlsson&title=Osteoblast-derived+NOTUM+reduces+cortical+bone+mass+in+mice+and+the+NOTUM+locus+is+associated+with+bone+mineral+density+in+humans&doi=10.1096%2Ffj.201900707R"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Osteoblast-derived NOTUM reduces cortical bone mass in mice and the NOTUM locus is associated with bone mineral density in humans</span></div><div class="casAuthors">Moverare-Skrtic, Sofia; Nilsson, Karin H.; Henning, Petra; Funck-Brentano, Thomas; Nethander, Maria; Rivadeneira, Fernando; Nunes, Glaucia Coletto; Koskela, Antti; Tuukkanen, Juha; Tuckermann, Jan; Perret, Christine; Souza, Pedro Paulo Chaves; Lerner, Ulf H.; Ohlsson, Claes</div><div class="citationInfo"><span class="NLM_cas:title">FASEB Journal</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">11163-11179</span>CODEN:
                <span class="NLM_cas:coden">FAJOEC</span>;
        ISSN:<span class="NLM_cas:issn">0892-6638</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">Osteoporosis is a common skeletal disease, affecting millions of individuals worldwide.  Currently used osteoporosis treatments substantially reduce vertebral fracture risk, whereas nonvertebral fracture risk, mainly caused by reduced cortical bone mass, has only moderately been improved by the osteoporosis drugs used, defining an unmet medical need.  Because several wingless-type MMTV integration site family members (WNTs) and modulators of WNT activity are major regulators of bone mass, we hypothesized that NOTUM, a secreted WNT lipase, might modulate bone mass via an inhibition of WNT activity.  To characterize the possible role of endogenous NOTUM as a physiol. modulator of bone mass, we developed global, cell-specific, and inducible Notum-inactivated mouse models.  Notum expression was high in the cortical bone in mice, and conditional Notum inactivation revealed that osteoblast lineage cells are the principal source of NOTUM in the cortical bone.  Large-scale human genetic analyses identified genetic variants mapping to the NOTUM locus that are strongly assocd. with bone mineral d. (BMD) as estd. with quant. ultrasound in the heel.  Thus, osteoblast-derived NOTUM is an essential local physiol. regulator of cortical bone mass via effects on periosteal bone formation in adult mice, and genetic variants in the NOTUM locus are assocd. with BMD variation in adult humans.  Therapies targeting osteoblast-derived NOTUM may prevent nonvertebral fractures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAEpf9m9-nLLVg90H21EOLACvtfcHk0lhO6zgZf21asA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVOltLrE&md5=b61892ced94f651095f0b167afb52de0</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1096%2Ffj.201900707R&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffj.201900707R%26sid%3Dliteratum%253Aachs%26aulast%3DMoverare-Skrtic%26aufirst%3DS.%26aulast%3DNilsson%26aufirst%3DK.%2BH.%26aulast%3DHenning%26aufirst%3DP.%26aulast%3DFunck-Brentano%26aufirst%3DT.%26aulast%3DNethander%26aufirst%3DM.%26aulast%3DRivadeneira%26aufirst%3DF.%26aulast%3DNunes%26aufirst%3DG.%2BC.%26aulast%3DKoskela%26aufirst%3DA.%26aulast%3DTuukkanen%26aufirst%3DJ.%26aulast%3DTuckermann%26aufirst%3DJ.%26aulast%3DPerret%26aufirst%3DC.%26aulast%3DSouza%26aufirst%3DP.%2BP.%2BC.%26aulast%3DLerner%26aufirst%3DU.%2BH.%26aulast%3DOhlsson%26aufirst%3DC.%26atitle%3DOsteoblast-derived%2520NOTUM%2520reduces%2520cortical%2520bone%2520mass%2520in%2520mice%2520and%2520the%2520NOTUM%2520locus%2520is%2520associated%2520with%2520bone%2520mineral%2520density%2520in%2520humans%26jtitle%3DFASEB%2520J.%26date%3D2019%26volume%3D33%26spage%3D11163%26epage%3D11179%26doi%3D10.1096%2Ffj.201900707R" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brommage, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mseeh, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potter, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shadoan, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeter-Jones, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bright, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardenhagen, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doree, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moverare-Skrtic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilsson, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henning, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerner, U. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohlsson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sands, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarver, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zambrowicz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span> <span> </span><span class="NLM_article-title">NOTUM inhibition increases endocortical bone formation and bone strength</span>. <i>Bone Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">2</span>, <span class="refDoi">Â DOI: 10.1038/s41413-018-0038-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1038%2Fs41413-018-0038-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=30622831" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmvVCmtbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2019&pages=2&author=R.+Brommageauthor=J.+Liuauthor=P.+Vogelauthor=F.+Mseehauthor=A.+Y.+Thompsonauthor=D.+G.+Potterauthor=M.+K.+Shadoanauthor=G.+M.+Hansenauthor=S.+Jeter-Jonesauthor=J.+Cuiauthor=D.+Brightauthor=J.+P.+Bardenhagenauthor=D.+D.+Doreeauthor=S.+Moverare-Skrticauthor=K.+H.+Nilssonauthor=P.+Henningauthor=U.+H.+Lernerauthor=C.+Ohlssonauthor=A.+T.+Sandsauthor=J.+E.+Tarverauthor=D.+R.+Powellauthor=B.+Zambrowiczauthor=Q.+Liu&title=NOTUM+inhibition+increases+endocortical+bone+formation+and+bone+strength&doi=10.1038%2Fs41413-018-0038-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">NOTUM inhibition increases endocortical bone formation and bone strength</span></div><div class="casAuthors">Brommage, Robert; Liu, Jeff; Vogel, Peter; Mseeh, Faika; Thompson, Andrea Y.; Potter, David G.; Shadoan, Melanie K.; Hansen, Gwenn M.; Jeter-Jones, Sabrina; Cui, Jie; Bright, Dawn; Bardenhagen, Jennifer P.; Doree, Deon D.; Moverare-Skrtic, Sofia; Nilsson, Karin H.; Henning, Petra; Lerner, Ulf H.; Ohlsson, Claes; Sands, Arthur T.; Tarver, James E.; Powell, David R.; Zambrowicz, Brian; Liu, Qingyun</div><div class="citationInfo"><span class="NLM_cas:title">Bone Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2</span>CODEN:
                <span class="NLM_cas:coden">BROEBB</span>;
        ISSN:<span class="NLM_cas:issn">2095-6231</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The disability, mortality and costs caused by non-vertebral osteoporotic fractures are enormous.  Existing osteoporosis therapies are highly effective at reducing vertebral but not non-vertebral fractures.  Cortical bone is a major determinant of non-vertebral bone strength.  To identify novel osteoporosis drug targets, we phenotyped cortical bone of 3 366 viable mouse strains with global knockouts of druggable genes.  Cortical bone thickness was substantially elevated in Notum-/- mice.  NOTUM is a secreted WNT lipase and we obsd. high NOTUM expression in cortical bone and osteoblasts but not osteoclasts.  Three orally active small mols. and a neutralizing antibody inhibiting NOTUM lipase activity were developed.  They increased cortical bone thickness and strength at multiple skeletal sites in both gonadal intact and ovariectomized rodents by stimulating endocortical bone formation.  Thus, inhibition of NOTUM activity is a potential novel anabolic therapy for strengthening cortical bone and preventing non-vertebral fractures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3-CO74hYhmLVg90H21EOLACvtfcHk0lj-Ug6vSKkzuw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmvVCmtbs%253D&md5=d63656138816ed9b86b804f0b5f9369c</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fs41413-018-0038-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41413-018-0038-3%26sid%3Dliteratum%253Aachs%26aulast%3DBrommage%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DVogel%26aufirst%3DP.%26aulast%3DMseeh%26aufirst%3DF.%26aulast%3DThompson%26aufirst%3DA.%2BY.%26aulast%3DPotter%26aufirst%3DD.%2BG.%26aulast%3DShadoan%26aufirst%3DM.%2BK.%26aulast%3DHansen%26aufirst%3DG.%2BM.%26aulast%3DJeter-Jones%26aufirst%3DS.%26aulast%3DCui%26aufirst%3DJ.%26aulast%3DBright%26aufirst%3DD.%26aulast%3DBardenhagen%26aufirst%3DJ.%2BP.%26aulast%3DDoree%26aufirst%3DD.%2BD.%26aulast%3DMoverare-Skrtic%26aufirst%3DS.%26aulast%3DNilsson%26aufirst%3DK.%2BH.%26aulast%3DHenning%26aufirst%3DP.%26aulast%3DLerner%26aufirst%3DU.%2BH.%26aulast%3DOhlsson%26aufirst%3DC.%26aulast%3DSands%26aufirst%3DA.%2BT.%26aulast%3DTarver%26aufirst%3DJ.%2BE.%26aulast%3DPowell%26aufirst%3DD.%2BR.%26aulast%3DZambrowicz%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DQ.%26atitle%3DNOTUM%2520inhibition%2520increases%2520endocortical%2520bone%2520formation%2520and%2520bone%2520strength%26jtitle%3DBone%2520Res.%26date%3D2019%26volume%3D7%26spage%3D2%26doi%3D10.1038%2Fs41413-018-0038-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jolly, S.</span>; <span class="NLM_string-name">Schuhmacher, L.</span>; <span class="NLM_string-name">Palomer, E.</span>; <span class="NLM_string-name">Frew, S.</span>; <span class="NLM_string-name">Mahy, W.</span>; <span class="NLM_string-name">Monaghan, A.</span>; <span class="NLM_string-name">Jones, E. Y.</span>; <span class="NLM_string-name">Bilsland, J.</span>; <span class="NLM_string-name">Fish, P. V.</span>; <span class="NLM_string-name">Vincent, J. P.</span>; <span class="NLM_string-name">Bictash, M.</span>; <span class="NLM_string-name">Salinas, P.</span>; <span class="NLM_string-name">Whiting, P.</span></span> <i>Notum, a Negative Regulator of Wnt Signalling, as a Therapeutic Target for Alzheimerâs Disease? From Target Validation to the Development of Small Molecule Inhibitors</i>; <span class="NLM_publisher-name">2019 UCL Neuroscience Symposium</span>: <span class="NLM_publisher-loc">London, UK</span>, June 21, <span class="NLM_year">2019</span>. See: <a href="https://www.ucl.ac.uk/research/domains/sites/research_domains/files/abstract_booklet_ucl_neuroscience_symposium_2019.pdf" class="extLink">https://www.ucl.ac.uk/research/domains/sites/research_domains/files/abstract_booklet_ucl_neuroscience_symposium_2019.pdf</a>. (accessed Aug 9, 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=S.+Jolly&author=L.+Schuhmacher&author=E.+Palomer&author=S.+Frew&author=W.+Mahy&author=A.+Monaghan&author=E.+Y.+Jones&author=J.+Bilsland&author=P.+V.+Fish&author=J.+P.+Vincent&author=M.+Bictash&author=P.+Salinas&author=P.+Whiting&title=Notum%2C+a+Negative+Regulator+of+Wnt+Signalling%2C+as+a+Therapeutic+Target+for+Alzheimer%E2%80%99s+Disease%3F+From+Target+Validation+to+the+Development+of+Small+Molecule+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DJolly%26aufirst%3DS.%26btitle%3DNotum%252C%2520a%2520Negative%2520Regulator%2520of%2520Wnt%2520Signalling%252C%2520as%2520a%2520Therapeutic%2520Target%2520for%2520Alzheimer%25E2%2580%2599s%2520Disease%253F%2520From%2520Target%2520Validation%2520to%2520the%2520Development%2520of%2520Small%2520Molecule%2520Inhibitors%26pub%3D2019%2520UCL%2520Neuroscience%2520Symposium%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="note"><p class="first last">As assessed by SchrÃ¶dinger SiteMap, which is used to detect and score binding sites. The score is constructed and calibrated so that the average SiteScore for 157 investigated submicromolar sites is 1.0; a score > 1 suggests a site of particular promise for druggability. Application of SiteMap to the palmitoleate binding site of Notum (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4UZQ">4UZQ</a>) generated a SiteScore of 1.2. SchrÃ¶dinger Release 2019â3: SiteMap, SchrÃ¶dinger, LLC, New York, NY, 2019. See:</p></div><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span> <span> </span><span class="NLM_article-title">Identifying and characterizing binding sites and assessing druggability</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">377</span>â <span class="NLM_lpage">389</span>, <span class="refDoi">Â DOI: 10.1021/ci800324m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800324m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BD1MXns1OhsA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=377-389&author=T.+A.+Halgren&title=Identifying+and+characterizing+binding+sites+and+assessing+druggability&doi=10.1021%2Fci800324m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying and Characterizing Binding Sites and Assessing Druggability</span></div><div class="casAuthors">Halgren, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">377-389</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Identification and characterization of binding sites is key in the process of structure-based drug design.  In some cases there may not be any information about the binding site for a target of interest.  In other cases, a putative binding site has been identified by computational or exptl. means, but the druggability of the target is not known.  Even when a site for a given target is known, it may be desirable to find addnl. sites whose targeting could produce a desired biol. response.  A new program, called SiteMap, is presented for identifying and analyzing binding sites and for predicting target druggability.  In a large-scale validation, SiteMap correctly identifies the known binding site as the top-ranked site in 86% of the cases, with best results (>98%) coming for sites that bind ligands with subnanomolar affinity.  In addn., a modified version of the score employed for binding-site identification allows SiteMap to accurately classify the druggability of proteins as measured by their ability to bind passively absorbed small mols. tightly.  In characterizing binding sites, SiteMap provides quant. and graphical information that can help guide efforts to critically assess virtual hits in a lead-discovery application or to modify ligand structure to enhance potency or improve phys. properties in a lead-optimization context.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfAt0-gvcXDLVg90H21EOLACvtfcHk0lj0r6ecsEsajQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXns1OhsA%253D%253D&md5=f03c7b27e30ae03d10e4127ed1f1ed3c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fci800324m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800324m%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26atitle%3DIdentifying%2520and%2520characterizing%2520binding%2520sites%2520and%2520assessing%2520druggability%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D377%26epage%3D389%26doi%3D10.1021%2Fci800324m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tarver, J. E.,  Jr</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pabba, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbosa, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardyan, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brommage, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A. G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carson, K. G.</span></span> <span> </span><span class="NLM_article-title">Stimulation of cortical bone formation with thienopyrimidine based inhibitors of NOTUM pectinacetylesterase</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1525</span>â <span class="NLM_lpage">1528</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2016.02.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1016%2Fj.bmcl.2016.02.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=26897593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC28Xislyrsrg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=1525-1528&author=J.+E.+Tarverauthor=P.+K.+Pabbaauthor=J.+Barbosaauthor=Q.+Hanauthor=M.+W.+Gardyanauthor=R.+Brommageauthor=A.+Y.+Thompsonauthor=J.+M.+Schmidtauthor=A.+G.+E.+Wilsonauthor=W.+Heauthor=V.+K.+Lombardoauthor=K.+G.+Carson&title=Stimulation+of+cortical+bone+formation+with+thienopyrimidine+based+inhibitors+of+NOTUM+pectinacetylesterase&doi=10.1016%2Fj.bmcl.2016.02.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Stimulation of cortical bone formation with thienopyrimidine based inhibitors of Notum Pectinacetylesterase</span></div><div class="casAuthors">Tarver, James E., Jr.; Pabba, Praveen K.; Barbosa, Joseph; Han, Qiang; Gardyan, Michael W.; Brommage, Robert; Thompson, Andrea Y.; Schmidt, James M.; Wilson, Alan G. E.; He, Wei; Lombardo, Victoria K.; Carson, Kenneth G.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1525-1528</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A group of small mol. thienopyrimidine inhibitors of Notum Pectinacetylesterase are described.  The authors explored both 2-((5,6-thieno[2,3-d]pyrimidin-4-yl)thio)acetic acids and 2-((6,7-thieno[3,2-d]pyrimidin-4-yl)thio)acetic acids.  In both series, highly potent, orally active Notum Pectinacetylesterase inhibitors were identified.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQqBjxmVwBErVg90H21EOLACvtfcHk0lj0r6ecsEsajQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xislyrsrg%253D&md5=7040e360736e8b94a687d67fe92a1d22</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.02.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.02.021%26sid%3Dliteratum%253Aachs%26aulast%3DTarver%26aufirst%3DJ.%2BE.%26aulast%3DPabba%26aufirst%3DP.%2BK.%26aulast%3DBarbosa%26aufirst%3DJ.%26aulast%3DHan%26aufirst%3DQ.%26aulast%3DGardyan%26aufirst%3DM.%2BW.%26aulast%3DBrommage%26aufirst%3DR.%26aulast%3DThompson%26aufirst%3DA.%2BY.%26aulast%3DSchmidt%26aufirst%3DJ.%2BM.%26aulast%3DWilson%26aufirst%3DA.%2BG.%2BE.%26aulast%3DHe%26aufirst%3DW.%26aulast%3DLombardo%26aufirst%3DV.%2BK.%26aulast%3DCarson%26aufirst%3DK.%2BG.%26atitle%3DStimulation%2520of%2520cortical%2520bone%2520formation%2520with%2520thienopyrimidine%2520based%2520inhibitors%2520of%2520NOTUM%2520pectinacetylesterase%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D1525%26epage%3D1528%26doi%3D10.1016%2Fj.bmcl.2016.02.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steadman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahy, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sipthorp, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayle, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papageorgiou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeganathan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frew, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monaghan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bictash, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span></span> <span> </span><span class="NLM_article-title">Scaffold-hopping identifies furano[2,3-<i>d</i>]pyrimidine amides as potent Notum inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">126751</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2019.126751</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1016%2Fj.bmcl.2019.126751" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=31862412" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVGjtrrI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=126751&author=B.+N.+Atkinsonauthor=D.+Steadmanauthor=W.+Mahyauthor=Y.+Zhaoauthor=J.+Sipthorpauthor=E.+D.+Bayleauthor=F.+Svenssonauthor=G.+Papageorgiouauthor=F.+Jeganathanauthor=S.+Frewauthor=A.+Monaghanauthor=M.+Bictashauthor=E.+Y.+Jonesauthor=P.+V.+Fish&title=Scaffold-hopping+identifies+furano%5B2%2C3-d%5Dpyrimidine+amides+as+potent+Notum+inhibitors&doi=10.1016%2Fj.bmcl.2019.126751"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Scaffold-hopping identifies furano[2,3-d]pyrimidine amides as potent Notum inhibitors</span></div><div class="casAuthors">Atkinson, Benjamin N.; Steadman, David; Mahy, William; Zhao, Yuguang; Sipthorp, James; Bayle, Elliott D.; Svensson, Fredrik; Papageorgiou, George; Jeganathan, Fiona; Frew, Sarah; Monaghan, Amy; Bictash, Magda; Jones, E. Yvonne; Fish, Paul V.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">126751</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The carboxylesterase Notum is a key neg. regulator of the Wnt signaling pathway by mediating the depalmitoleoylation of Wnt proteins.  Our objective was to discover potent small mol. inhibitors of Notum suitable for exploring the regulation of Wnt signaling in the central nervous system.  Scaffold-hopping from thienopyrimidine acids 1 and 2, supported by X-ray structure detn., identified 3-methylimidazolin-4-one amides 20-24 as potent inhibitors of Notum with activity across three orthogonal assay formats (biochem., extra-cellular, occupancy).  A preferred example 24 demonstrated good stability in mouse microsomes and plasma, and cell permeability in the MDCK-MDR1 assay albeit with modest P-gp mediated efflux.  Pharmacokinetic studies with 24 were performed in vivo in mouse with single oral administration of 24 showing good plasma exposure and reasonable CNS penetration.  We propose that 24 is a new chem. tool suitable for cellular studies to explore the fundamental biol. of Notum.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMbQIEOi9W3bVg90H21EOLACvtfcHk0lgY70RWKPAXZw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVGjtrrI&md5=9cbdde4a81de2782dfdd8fa8a1be523f</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.126751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.126751%26sid%3Dliteratum%253Aachs%26aulast%3DAtkinson%26aufirst%3DB.%2BN.%26aulast%3DSteadman%26aufirst%3DD.%26aulast%3DMahy%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DSipthorp%26aufirst%3DJ.%26aulast%3DBayle%26aufirst%3DE.%2BD.%26aulast%3DSvensson%26aufirst%3DF.%26aulast%3DPapageorgiou%26aufirst%3DG.%26aulast%3DJeganathan%26aufirst%3DF.%26aulast%3DFrew%26aufirst%3DS.%26aulast%3DMonaghan%26aufirst%3DA.%26aulast%3DBictash%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DE.%2BY.%26aulast%3DFish%26aufirst%3DP.%2BV.%26atitle%3DScaffold-hopping%2520identifies%2520furano%255B2%252C3-d%255Dpyrimidine%2520amides%2520as%2520potent%2520Notum%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D30%26spage%3D126751%26doi%3D10.1016%2Fj.bmcl.2019.126751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Willis, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayle, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papageorgiou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steadman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahy, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span></span> <span> </span><span class="NLM_article-title">An improved, scalable synthesis of Notum inhibitor LP-922056 using 1-chloro-1,2-benziodoxol-3-one as a superior electrophilic chlorinating agent</span>. <i>Beilstein J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">2790</span>â <span class="NLM_lpage">2797</span>, <span class="refDoi">Â DOI: 10.3762/bjoc.15.271</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.3762%2Fbjoc.15.271" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=31807213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlOqsrjJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=2790-2797&author=N.+J.+Willisauthor=E.+D.+Bayleauthor=G.+Papageorgiouauthor=D.+Steadmanauthor=B.+N.+Atkinsonauthor=W.+Mahyauthor=P.+V.+Fish&title=An+improved%2C+scalable+synthesis+of+Notum+inhibitor+LP-922056+using+1-chloro-1%2C2-benziodoxol-3-one+as+a+superior+electrophilic+chlorinating+agent&doi=10.3762%2Fbjoc.15.271"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">An improved, scalable synthesis of Notum inhibitor LP-922056 using 1-chloro-1,2-benziodoxol-3-one as a superior electrophilic chlorinating agent</span></div><div class="casAuthors">Willis, Nicky J.; Bayle, Elliott D.; Papageorgiou, George; Steadman, David; Atkinson, Benjamin N.; Mahy, William; Fish, Paul V.</div><div class="citationInfo"><span class="NLM_cas:title">Beilstein Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2790-2797</span>CODEN:
                <span class="NLM_cas:coden">BJOCBH</span>;
        ISSN:<span class="NLM_cas:issn">1860-5397</span>.
    
            (<span class="NLM_cas:orgname">Beilstein-Institut zur Foerderung der Chemischen Wissenschaften</span>)
        </div><div class="casAbstract">We are investigating the role of Notum in modulating Wnt signalling in the central nervous system and wished to establish if 1 would serve as a peripherally restricted control.  Key modifications include: (1) the introduction of the C7-cyclopropyl group was most effectively achieved with a Suzuki-Miyaura cross-coupling reaction with MIDA-boronate 11 (5 â 6), and (2) C6 chlorination was performed with 1-chloro-1,2-benziodoxol-3-one (12) (6 â 7) as a mild and selective electrophilic chlorination agent.  Pharmacokinetic studies in mouse showed CNS penetration of 1 is very low with a brain/plasma concn. ratio of just 0.01.  A small library of amides 17 were prepd. from acid 1 to explore if 1 could be modified to deliver a CNS penetrant tool by capping off the acid as an amide.  Although significant Notum inhibition activity could be achieved, none of these amides demonstrated the required combination of metabolic stability along with cell permeability without evidence of P-gp mediated efflux.  Mouse pharmacokinetic studies demonstrate that 1 is unsuitable for use in models of disease where brain penetration is an essential requirement of the compd. but would be an ideal peripherally restricted control.  These data will contribute to the understanding of drug levels of 1 to overlay with appropriate in vivo efficacy endpoints, i.e., the PK-PD relationship.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpe2Pvyx9sImbVg90H21EOLACvtfcHk0lgY70RWKPAXZw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlOqsrjJ&md5=ddd50870154275e08f397e28301e2e6e</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.3762%2Fbjoc.15.271&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3762%252Fbjoc.15.271%26sid%3Dliteratum%253Aachs%26aulast%3DWillis%26aufirst%3DN.%2BJ.%26aulast%3DBayle%26aufirst%3DE.%2BD.%26aulast%3DPapageorgiou%26aufirst%3DG.%26aulast%3DSteadman%26aufirst%3DD.%26aulast%3DAtkinson%26aufirst%3DB.%2BN.%26aulast%3DMahy%26aufirst%3DW.%26aulast%3DFish%26aufirst%3DP.%2BV.%26atitle%3DAn%2520improved%252C%2520scalable%2520synthesis%2520of%2520Notum%2520inhibitor%2520LP-922056%2520using%25201-chloro-1%252C2-benziodoxol-3-one%2520as%2520a%2520superior%2520electrophilic%2520chlorinating%2520agent%26jtitle%3DBeilstein%2520J.%2520Org.%2520Chem.%26date%3D2019%26volume%3D15%26spage%3D2790%26epage%3D2797%26doi%3D10.3762%2Fbjoc.15.271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Suciu, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cognetta, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potter, Z. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Selective irreversible inhibitors of the Wnt-deacylating enzyme NOTUM developed by activity-based protein profiling</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">563</span>â <span class="NLM_lpage">568</span>, <span class="refDoi">Â DOI: 10.1021/acsmedchemlett.8b00191</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00191" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVSktr3I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=563-568&author=R.+M.+Suciuauthor=A.+B.+Cognettaauthor=Z.+E.+Potterauthor=B.+F.+Cravatt&title=Selective+irreversible+inhibitors+of+the+Wnt-deacylating+enzyme+NOTUM+developed+by+activity-based+protein+profiling&doi=10.1021%2Facsmedchemlett.8b00191"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Irreversible Inhibitors of the Wnt-Deacylating Enzyme NOTUM Developed by Activity-Based Protein Profiling</span></div><div class="casAuthors">Suciu, Radu M.; Cognetta, Armand B.; Potter, Zachary E.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">563-568</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Wnt proteins are secreted morphogens that play crit. roles in embryonic development and tissue remodeling in adult organisms.  Aberrant Wnt signaling contributes to diseases such as cancer.  Wnts are modified by an unusual O-fatty acylation event (O-linked palmitoleoylation of a conserved serine) that is required for binding to Frizzled receptors.  O-Palmitoleoylation of Wnts is introduced by the porcupine (PORCN) acyltransferase and removed by the serine hydrolase NOTUM.  PORCN inhibitors are under development for oncol., while NOTUM inhibitors have potential for treating degenerative diseases.  Here we describe the use of activity-based protein profiling (ABPP) to discover and advance a class of N-hydroxyhydantoin (NHH) carbamates that potently and selectively inhibit NOTUM.  An optimized NHH carbamate inhibitor - ABC99 - preserves Wnt-mediated cell signaling in the presence of NOTUM and was also converted into an ABPP probe for visualizing NOTUM in native biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp28DF3H0VimrVg90H21EOLACvtfcHk0ljOnueSNuioqA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVSktr3I&md5=dd39eb16fc4bd557f2e3db10ca82fcef</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00191&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00191%26sid%3Dliteratum%253Aachs%26aulast%3DSuciu%26aufirst%3DR.%2BM.%26aulast%3DCognetta%26aufirst%3DA.%2BB.%26aulast%3DPotter%26aufirst%3DZ.%2BE.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DSelective%2520irreversible%2520inhibitors%2520of%2520the%2520Wnt-deacylating%2520enzyme%2520NOTUM%2520developed%2520by%2520activity-based%2520protein%2520profiling%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D9%26spage%3D563%26epage%3D568%26doi%3D10.1021%2Facsmedchemlett.8b00191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pentinmikko, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iqbal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mana, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cognetta, A. B.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suciu, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roper, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LuopajÃ¤rvi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markelin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gopalakrishnan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smolander, O.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naranjo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saarinen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juuti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">PietilÃ¤inen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auvinen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">RistimÃ¤ki, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tammela, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacks, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabatini, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yilmaz, O. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katajisto, P.</span></span> <span> </span><span class="NLM_article-title">Notum produced by Paneth cells attenuates regeneration of aged intestinal epithelium</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>571</i></span>,  <span class="NLM_fpage">398</span>â <span class="NLM_lpage">402</span>, <span class="refDoi">Â DOI: 10.1038/s41586-019-1383-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1038%2Fs41586-019-1383-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=31292548" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtl2qsLfE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=571&publication_year=2019&pages=398-402&author=N.+Pentinmikkoauthor=S.+Iqbalauthor=M.+Manaauthor=S.+Anderssonauthor=A.+B.+Cognettaauthor=R.+M.+Suciuauthor=J.+Roperauthor=K.+Luopaj%C3%A4rviauthor=E.+Markelinauthor=S.+Gopalakrishnanauthor=O.-P.+Smolanderauthor=S.+Naranjoauthor=T.+Saarinenauthor=A.+Juutiauthor=K.+Pietil%C3%A4inenauthor=P.+Auvinenauthor=A.+Ristim%C3%A4kiauthor=N.+Guptaauthor=T.+Tammelaauthor=T.+Jacksauthor=D.+M.+Sabatiniauthor=B.+F.+Cravattauthor=O.+H.+Yilmazauthor=P.+Katajisto&title=Notum+produced+by+Paneth+cells+attenuates+regeneration+of+aged+intestinal+epithelium&doi=10.1038%2Fs41586-019-1383-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Notum produced by Paneth cells attenuates regeneration of aged intestinal epithelium</span></div><div class="casAuthors">Pentinmikko, Nalle; Iqbal, Sharif; Mana, Miyeko; Andersson, Simon; Cognetta, Armand B., III; Suciu, Radu M.; Roper, Jatin; Luopajarvi, Kalle; Markelin, Eino; Gopalakrishnan, Swetha; Smolander, Olli-Pekka; Naranjo, Santiago; Saarinen, Tuure; Juuti, Anne; Pietilainen, Kirsi; Auvinen, Petri; Ristimaki, Ari; Gupta, Nitin; Tammela, Tuomas; Jacks, Tyler; Sabatini, David M.; Cravatt, Benjamin F.; Yilmaz, Omer H.; Katajisto, Pekka</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">571</span>
        (<span class="NLM_cas:issue">7765</span>),
    <span class="NLM_cas:pages">398-402</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A decline in stem cell function impairs tissue regeneration during ageing, but the role of the stem-cell-supporting niche in ageing is not well understood.  The small intestine is maintained by actively cycling intestinal stem cells that are regulated by the Paneth cell niche1,2.  Here we show that the regenerative potential of human and mouse intestinal epithelium diminishes with age owing to defects in both stem cells and their niche.  The functional decline was caused by a decrease in stemness-maintaining Wnt signalling due to prodn. of Notum, an extracellular Wnt inhibitor, in aged Paneth cells.  Mechanistically, high activity of mammalian target of rapamycin complex 1 (mTORC1) in aged Paneth cells inhibits activity of peroxisome proliferator activated receptor Î± (PPAR-Î±)3, and lowered PPAR-Î± activity increased Notum expression.  Genetic targeting of Notum or Wnt supplementation restored function of aged intestinal organoids.  Moreover, pharmacol. inhibition of Notum in mice enhanced the regenerative capacity of aged stem cells and promoted recovery from chemotherapy-induced damage.  Our results reveal a role of the stem cell niche in ageing and demonstrate that targeting of Notum can promote regeneration of aged tissues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYdrtPIC9dYbVg90H21EOLACvtfcHk0ljOnueSNuioqA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtl2qsLfE&md5=9d93cd7bdc33cacf617a025a74aecf48</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fs41586-019-1383-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-019-1383-0%26sid%3Dliteratum%253Aachs%26aulast%3DPentinmikko%26aufirst%3DN.%26aulast%3DIqbal%26aufirst%3DS.%26aulast%3DMana%26aufirst%3DM.%26aulast%3DAndersson%26aufirst%3DS.%26aulast%3DCognetta%26aufirst%3DA.%2BB.%26aulast%3DSuciu%26aufirst%3DR.%2BM.%26aulast%3DRoper%26aufirst%3DJ.%26aulast%3DLuopaj%25C3%25A4rvi%26aufirst%3DK.%26aulast%3DMarkelin%26aufirst%3DE.%26aulast%3DGopalakrishnan%26aufirst%3DS.%26aulast%3DSmolander%26aufirst%3DO.-P.%26aulast%3DNaranjo%26aufirst%3DS.%26aulast%3DSaarinen%26aufirst%3DT.%26aulast%3DJuuti%26aufirst%3DA.%26aulast%3DPietil%25C3%25A4inen%26aufirst%3DK.%26aulast%3DAuvinen%26aufirst%3DP.%26aulast%3DRistim%25C3%25A4ki%26aufirst%3DA.%26aulast%3DGupta%26aufirst%3DN.%26aulast%3DTammela%26aufirst%3DT.%26aulast%3DJacks%26aufirst%3DT.%26aulast%3DSabatini%26aufirst%3DD.%2BM.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26aulast%3DYilmaz%26aufirst%3DO.%2BH.%26aulast%3DKatajisto%26aufirst%3DP.%26atitle%3DNotum%2520produced%2520by%2520Paneth%2520cells%2520attenuates%2520regeneration%2520of%2520aged%2520intestinal%2520epithelium%26jtitle%3DNature%26date%3D2019%26volume%3D571%26spage%3D398%26epage%3D402%26doi%3D10.1038%2Fs41586-019-1383-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steadman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sipthorp, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vecchia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruza, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeganathan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lines, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frew, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monaghan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kjaer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bictash, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span></span> <span> </span><span class="NLM_article-title">Discovery of 2-phenoxyacetamides as inhibitors of the Wnt-depalmitoleating enzyme NOTUM from an X-ray fragment screen</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1361</span>â <span class="NLM_lpage">1369</span>, <span class="refDoi">Â DOI: 10.1039/C9MD00096H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1039%2FC9MD00096H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=31534655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosVCgsb4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1361-1369&author=B.+N.+Atkinsonauthor=D.+Steadmanauthor=Y.+Zhaoauthor=J.+Sipthorpauthor=L.+Vecchiaauthor=R.+R.+Ruzaauthor=F.+Jeganathanauthor=G.+Linesauthor=S.+Frewauthor=A.+Monaghanauthor=S.+Kjaerauthor=M.+Bictashauthor=E.+Y.+Jonesauthor=P.+V.+Fish&title=Discovery+of+2-phenoxyacetamides+as+inhibitors+of+the+Wnt-depalmitoleating+enzyme+NOTUM+from+an+X-ray+fragment+screen&doi=10.1039%2FC9MD00096H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 2-phenoxyacetamides as inhibitors of the Wnt-depalmitoleating enzyme NOTUM from an X-ray fragment screen</span></div><div class="casAuthors">Atkinson, Benjamin N.; Steadman, David; Zhao, Yuguang; Sipthorp, James; Vecchia, Luca; Ruza, Reinis R.; Jeganathan, Fiona; Lines, Georgie; Frew, Sarah; Monaghan, Amy; Kjaer, Svend; Bictash, Magda; Jones, E. Yvonne; Fish, Paul V.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1361-1369</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">NOTUM is a carboxylesterase that has been shown to act by mediating the O-depalmitoleoylation of Wnt proteins resulting in suppression of Wnt signaling.  Here, we describe the development of NOTUM inhibitors that restore Wnt signaling for use in in vitro disease models where NOTUM over activity is an underlying cause.  A crystallog. fragment screen with NOTUM identified 2-phenoxyacetamide 3 as binding in the palmitoleate pocket with modest inhibition activity (IC50 33 Î¼M).  Optimization of hit 3 by SAR studies guided by SBDD identified indazole 38 (IC50 0.032 Î¼M) and isoquinoline 45 (IC50 0.085 Î¼M) as potent inhibitors of NOTUM.  The binding of 45 to NOTUM was rationalized through an X-ray co-crystal structure detn. which showed a flipped binding orientation compared to 3.  However, it was not possible to combine NOTUM inhibition activity with metabolic stability as the majority of the compds. tested were rapidly metabolized in an NADPH-independent manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOeg6yfRBz57Vg90H21EOLACvtfcHk0ljx9vWbk6pTZg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosVCgsb4%253D&md5=041061f0da5d9ff778d26adc5a14ab0e</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1039%2FC9MD00096H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9MD00096H%26sid%3Dliteratum%253Aachs%26aulast%3DAtkinson%26aufirst%3DB.%2BN.%26aulast%3DSteadman%26aufirst%3DD.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DSipthorp%26aufirst%3DJ.%26aulast%3DVecchia%26aufirst%3DL.%26aulast%3DRuza%26aufirst%3DR.%2BR.%26aulast%3DJeganathan%26aufirst%3DF.%26aulast%3DLines%26aufirst%3DG.%26aulast%3DFrew%26aufirst%3DS.%26aulast%3DMonaghan%26aufirst%3DA.%26aulast%3DKjaer%26aufirst%3DS.%26aulast%3DBictash%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DE.%2BY.%26aulast%3DFish%26aufirst%3DP.%2BV.%26atitle%3DDiscovery%2520of%25202-phenoxyacetamides%2520as%2520inhibitors%2520of%2520the%2520Wnt-depalmitoleating%2520enzyme%2520NOTUM%2520from%2520an%2520X-ray%2520fragment%2520screen%26jtitle%3DMedChemComm%26date%3D2019%26volume%3D10%26spage%3D1361%26epage%3D1369%26doi%3D10.1039%2FC9MD00096H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hillier, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. Y.</span></span> <span> </span><span class="NLM_article-title">Structural characterisation of melatonin as an inhibitor of the Wnt deacylase Notum</span>. <i>J. Pineal Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>68</i></span>, <span class="NLM_elocation-id">e12630</span> <span class="refDoi">Â DOI: 10.1111/jpi.12630</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1111%2Fjpi.12630" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=31876313" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1SitL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2020&author=Y.+Zhaoauthor=J.+Renauthor=J.+Hillierauthor=M.+Jonesauthor=W.+Luauthor=E.+Y.+Jones&title=Structural+characterisation+of+melatonin+as+an+inhibitor+of+the+Wnt+deacylase+Notum&doi=10.1111%2Fjpi.12630"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Structural characterization of melatonin as an inhibitor of the Wnt deacylase Notum</span></div><div class="casAuthors">Zhao, Yuguang; Ren, Jingshan; Hillier, James; Jones, Margaret; Lu, Weixian; Jones, Edith Yvonne</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pineal Research</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e12630</span>CODEN:
                <span class="NLM_cas:coden">JPRSE9</span>;
        ISSN:<span class="NLM_cas:issn">0742-3098</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The hormone melatonin, secreted from the pineal gland, mediates multiple physiol. effects including modulation of Wnt/Î²-catenin signalling.  The Wnt palmitoleate lipid modification is essential for its signalling activity, while the carboxylesterase Notum can remove the lipid from Wnt and inactivate it.  Notum enzyme inhibition can therefore upregulate Wnt signalling.  While searching for Notum inhibitors by crystallog. fragment screening, a hit compd. N-[2-(5-fluoro-1H-indol-3-yl)ethyl]acetamide that is structurally similar to melatonin came to our attention.  We then soaked melatonin and its precursor N-acetylserotonin into Notum crystals and obtained high-resoln. structures (=1.5 Ã) of their complexes.  In each of the structures, two compd. mols. bind with Notum: one at the enzyme's catalytic pocket, overlapping the space occupied by the acyl tail of the Wnt palmitoleate lipid, and the other at the edge of the pocket opposite the substrate entrance.  Although the inhibitory activity of melatonin shown by in vitro enzyme assays is low (IC50 75Î¼mol/L), the structural information reported here provides a basis for the design of potent and brain accessible drugs for neurodegenerative diseases such as Alzheimer's disease, in which upregulation of Wnt signalling may be beneficial.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq41bwOb_70PLVg90H21EOLACvtfcHk0ljx9vWbk6pTZg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1SitL8%253D&md5=4b1ebc6562fd38de68bfaffa2cfd8f1f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1111%2Fjpi.12630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fjpi.12630%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DHillier%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DM.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DJones%26aufirst%3DE.%2BY.%26atitle%3DStructural%2520characterisation%2520of%2520melatonin%2520as%2520an%2520inhibitor%2520of%2520the%2520Wnt%2520deacylase%2520Notum%26jtitle%3DJ.%2520Pineal%2520Res.%26date%3D2020%26volume%3D68%26doi%3D10.1111%2Fjpi.12630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span>See: <a href="https://enamine.net/fragments/fragment-collection/carboxylic-acid-fragments" class="extLink">https://enamine.net/fragments/fragment-collection/carboxylic-acid-fragments</a>. Compounds were purchased from this collection on Dec 15, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=See%3A+https%3A%2F%2Fenamine.net%2Ffragments%2Ffragment-collection%2Fcarboxylic-acid-fragments.+Compounds+were+purchased+from+this+collection+on+Dec+15%2C+2017."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Springthorpe, B.</span></span> <span> </span><span class="NLM_article-title">The influence of drug-like concepts on decision-making in medicinal chemistry</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">881</span>â <span class="NLM_lpage">890</span>, <span class="refDoi">Â DOI: 10.1038/nrd2445</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1038%2Fnrd2445" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=17971784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1els7rL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=881-890&author=P.+D.+Leesonauthor=B.+Springthorpe&title=The+influence+of+drug-like+concepts+on+decision-making+in+medicinal+chemistry&doi=10.1038%2Fnrd2445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The influence of drug-like concepts on decision-making in medicinal chemistry</span></div><div class="casAuthors">Leeson, Paul D.; Springthorpe, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">881-890</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Despite the wide acceptance of drug-like principles such as the 'rule of five', this anal. of mols. currently being synthesized in leading pharmaceutical companies reveals that their phys. properties differ significantly from those of recently discovered oral drugs.  The marked increase in lipophilicity in particular could increase the likelihood of attrition in drug development.  The application of guidelines linked to the concept of drug-likeness, such as the 'rule of five', has gained wide acceptance as an approach to reduce attrition in drug discovery and development.  However, despite this acceptance, anal. of recent trends reveals that the phys. properties of mols. that are currently being synthesized in leading drug discovery companies differ significantly from those of recently discovered oral drugs and compds. in clin. development.  The consequences of the marked increase in lipophilicity - the most important drug-like phys. property - include a greater likelihood of lack of selectivity and attrition in drug development.  Tackling the threat of compd.-related toxicol. attrition needs to move to the mainstream of medicinal chem. decision-making.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTUeonLD6iebVg90H21EOLACvtfcHk0liYvhoTX9kj4A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1els7rL&md5=5665a9e2dc266a1ee096e20625757ef7</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnrd2445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2445%26sid%3Dliteratum%253Aachs%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DSpringthorpe%26aufirst%3DB.%26atitle%3DThe%2520influence%2520of%2520drug-like%2520concepts%2520on%2520decision-making%2520in%2520medicinal%2520chemistry%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D6%26spage%3D881%26epage%3D890%26doi%3D10.1038%2Fnrd2445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallego, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, M. P.</span></span> <span> </span><span class="NLM_article-title">Lipophilic efficiency as an important metric in drug design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6401</span>â <span class="NLM_lpage">6420</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b00077</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00077" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmsVahsbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6401-6420&author=T.+W.+Johnsonauthor=R.+A.+Gallegoauthor=M.+P.+Edwards&title=Lipophilic+efficiency+as+an+important+metric+in+drug+design&doi=10.1021%2Facs.jmedchem.8b00077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Lipophilic Efficiency as an Important Metric in Drug Design</span></div><div class="casAuthors">Johnson, Ted W.; Gallego, Rebecca A.; Edwards, Martin P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6401-6420</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Lipophilic efficiency (LipE) is an important metric that has been increasingly applied in drug discovery medicinal chem. lead optimization programs.  In this perspective, using literature drug discovery examples, we discuss the concept of rigorously applying LipE to guide medicinal chem. lead optimization toward drug candidates with potential for superior in vivo efficacy and safety, esp. when guided by physiochem. property-based optimization (PPBO).  Also highlighted are examples of small structural modifications such as addn. of single atoms, small functional groups, and cyclizations that produce large increases in LipE.  Understanding the factors that may contribute to LipE changes through anal. of ligand-protein crystal structures and using structure-based drug design (SBDD) to increase LipE by design is also discussed.  Herein we advocate for use of LipE anal. coupled with PPBO and SBDD as an efficient mechanism for drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXQ8DOcOj6GrVg90H21EOLACvtfcHk0liYvhoTX9kj4A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmsVahsbg%253D&md5=29f38880914fa75c33be3a640ace4309</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00077%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DT.%2BW.%26aulast%3DGallego%26aufirst%3DR.%2BA.%26aulast%3DEdwards%26aufirst%3DM.%2BP.%26atitle%3DLipophilic%2520efficiency%2520as%2520an%2520important%2520metric%2520in%2520drug%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6401%26epage%3D6420%26doi%3D10.1021%2Facs.jmedchem.8b00077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span> clog<i>P</i> values were calculated using ChemDraw version
16.0.1.4 (77) software,
and chromatographic hydrophobicity index (CHI) log<i>D</i><sub>7.4</sub> values (<i>n</i> = 2) were measured by the
following method:<span class="NLM_contrib-group"><span class="NLM_string-name">ValkÃ³, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bevan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, D.</span></span> <span> </span><span class="NLM_article-title">Chromatographic hydrophobicity index by Fast-Gradient RP-HPLC: a high-throughput alternative to log P/log D</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">2022</span>â <span class="NLM_lpage">2029</span>, <span class="refDoi">Â DOI: 10.1021/ac961242d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac961242d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADyaK2sXivVertLs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=1997&pages=2022-2029&author=K.+Valk%C3%B3author=C.+Bevanauthor=D.+Reynolds&title=Chromatographic+hydrophobicity+index+by+Fast-Gradient+RP-HPLC%3A+a+high-throughput+alternative+to+log+P%2Flog+D&doi=10.1021%2Fac961242d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatographic Hydrophobicity Index by Fast-Gradient RP-HPLC: A High-Throughput Alternative to Log P/log D</span></div><div class="casAuthors">Valko, Klara; Bevan, Chris; Reynolds, Derek</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2022-2029</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new chromatog. hydrophobicity index (CHI) is described which can be used as part of a protocol for high-throughput (50-100 compds./day) physicochem. property profiling for rational drug design.  The index is derived from retention times (tR) obsd. in a fast gradient reversed-phase HPLC method.  The isocratic retention factors (log k') were measured for a series of 76 structurally unrelated compds. by using various concns. of acetonitrile in the mobile phase.  By plotting the log k' as a function of the acetonitrile concn., the slope (S) and the intercept (log k'w) values were calcd.  The previously validated index of hydrophobicity Ï0 was calcd. as -log k'w/S.  A good linear correlation was obtained between the gradient retention time values, tR and the isocratically detd. Ï0 values for the 76 compds.  The consts. of this linear correlation can be used to calc. CHI.  For most compds., CHI is between 0 and 100 and in this range it approximates to the percentage (by vol.) of acetonitrile required to achieve an equal distribution of compd. between the mobile and the stationary phases.  CHI values can be measured using acidic, neutral, or slightly basic eluents.  Values corresponding to the neutral form of mols. could be measured for 52 of the compds. and showed good correlation (r = 0.851) to the calcd. octanol/water partition coeff. (c log P) values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozvQI1B1cnpbVg90H21EOLACvtfcHk0lioNLuvw8CmHQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivVertLs%253D&md5=fcbb641105c12dc490d4a20db8fc5032</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fac961242d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac961242d%26sid%3Dliteratum%253Aachs%26aulast%3DValk%25C3%25B3%26aufirst%3DK.%26aulast%3DBevan%26aufirst%3DC.%26aulast%3DReynolds%26aufirst%3DD.%26atitle%3DChromatographic%2520hydrophobicity%2520index%2520by%2520Fast-Gradient%2520RP-HPLC%253A%2520a%2520high-throughput%2520alternative%2520to%2520log%2520P%252Flog%2520D%26jtitle%3DAnal.%2520Chem.%26date%3D1997%26volume%3D69%26spage%3D2022%26epage%3D2029%26doi%3D10.1021%2Fac961242d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Forero-CortÃ©s, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haydl, A. M.</span></span> <span> </span><span class="NLM_article-title">The 25th anniversary of the BuchwaldâHartwig amination: development, applications, and outlook</span>. <i>Org. Process Res. Dev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1478</span>â <span class="NLM_lpage">1483</span>, <span class="refDoi">Â DOI: 10.1021/acs.oprd.9b00161</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.oprd.9b00161" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1yhs77J" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2019&pages=1478-1483&author=P.+A.+Forero-Cort%C3%A9sauthor=A.+M.+Haydl&title=The+25th+anniversary+of+the+Buchwald%E2%80%93Hartwig+amination%3A+development%2C+applications%2C+and+outlook&doi=10.1021%2Facs.oprd.9b00161"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">The 25th Anniversary of the Buchwald-Hartwig Amination: Development, Applications, and Outlook</span></div><div class="casAuthors">Forero-Cortes, Paola A.; Haydl, Alexander M.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Process Research & Development</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1478-1483</span>CODEN:
                <span class="NLM_cas:coden">OPRDFK</span>;
        ISSN:<span class="NLM_cas:issn">1083-6160</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The palladium-catalyzed cross-coupling of amines and aryl (pseudo)halides, now commonly referred to as the Buchwald-Hartwig amination was first reported 25 years ago.  Since the simultaneous breakthrough reports of Buchwald and Hartwig dating 1995, this reaction has transformed the way synthetic chemists think about synthesizing arom. amines.  In this highlight article, a short showcasing discussion about the genesis of this reaction is provided, as well as selected examples showing the impact of this transformation in synthetic chem., both in academic and industrial settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDqjUPVJ2XvrVg90H21EOLACvtfcHk0lioNLuvw8CmHQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1yhs77J&md5=11ee132c194b07792110102e2ed63bf7</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.oprd.9b00161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.oprd.9b00161%26sid%3Dliteratum%253Aachs%26aulast%3DForero-Cort%25C3%25A9s%26aufirst%3DP.%2BA.%26aulast%3DHaydl%26aufirst%3DA.%2BM.%26atitle%3DThe%252025th%2520anniversary%2520of%2520the%2520Buchwald%25E2%2580%2593Hartwig%2520amination%253A%2520development%252C%2520applications%252C%2520and%2520outlook%26jtitle%3DOrg.%2520Process%2520Res.%2520Dev.%26date%3D2019%26volume%3D23%26spage%3D1478%26epage%3D1483%26doi%3D10.1021%2Facs.oprd.9b00161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="note"><p class="first last">Confirmation of stereochemical assignment and purity of enantiomers <b>26</b> and <b>27</b> was determined by chiral SFC supported by X-ray structure determination. See <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for details.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="note"><p class="first last">For recent reviews, see:</p></div><div class="NLM_citation" id="cit28a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akhtar, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alam, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akhter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaquiquzzaman, M.</span></span> <span> </span><span class="NLM_article-title">A review exploring therapeutic worth of 1,3,4-oxadiazole tailored compounds</span>. <i>Mini-Rev. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">477</span>â <span class="NLM_lpage">509</span>, <span class="refDoi">Â DOI: 10.2174/1389557518666181015152433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.2174%2F1389557518666181015152433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=30324877" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC1MXltVSlu74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2019&pages=477-509&author=G.+Vermaauthor=M.+F.+Khanauthor=W.+Akhtarauthor=M.+M.+Alamauthor=M.+Akhterauthor=M.+Shaquiquzzaman&title=A+review+exploring+therapeutic+worth+of+1%2C3%2C4-oxadiazole+tailored+compounds&doi=10.2174%2F1389557518666181015152433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28aR"><div class="casContent"><span class="casTitleNuber">28a</span><div class="casTitle"><span class="NLM_cas:atitle">A Review Exploring Therapeutic Worth of 1,3,4-Oxadiazole Tailored Compounds</span></div><div class="casAuthors">Verma, Garima; Khan, Mohemmed F.; Akhtar, Wasim; Alam, Mohammad Mumtaz; Akhter, Mymoona; Shaquiquzzaman, Mohammad</div><div class="citationInfo"><span class="NLM_cas:title">Mini-Reviews in Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">477-509</span>CODEN:
                <span class="NLM_cas:coden">MMCIAE</span>;
        ISSN:<span class="NLM_cas:issn">1389-5575</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  1,3,4-Oxadiazole, a five-membered arom. ring can be seen in a no. of synthetic mols.  The peculiar structural feature of 1,3,4-oxadiazole ring with pyridine type of nitrogen atom is beneficial for 1,3,4-oxadiazole derivs. to have effective binding with different enzymes and receptors in biol. systems through numerous weak interactions, thereby eliciting an array of bioactivities.  Research in the area of development of 1,3,4-oxadiazole-based derivs. has become an interesting topic for the scientists.  A no. of 1,3,4-oxadiazole based compds. with high therapeutic potency are being extensively used for the treatment of different ailments, contributing to enormous development value.  This work provides a systematic and comprehensive review highlighting current developments of 1,3,4-oxadiazole based compds. in the entire range of medicinal chem. such as anticancer, antifungal, antibacterial, antitubercular, anti-inflammatory, antineuropathic, antihypertensive, antihistaminic, antiparasitic, antiobesity, antiviral, and other medicinal agents.  It is believed that this review will be of great help for new thoughts in the pursuit for rational designs for the development of more active and less toxic 1,3,4-oxadiazole based medicinal agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDw_H8wFO9QbVg90H21EOLACvtfcHk0liTowNqPRQ1aw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXltVSlu74%253D&md5=0315e4b67aaf2e7a95e4052e13058511</span></div><a href="/servlet/linkout?suffix=cit28a&amp;dbid=16384&amp;doi=10.2174%2F1389557518666181015152433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1389557518666181015152433%26sid%3Dliteratum%253Aachs%26aulast%3DVerma%26aufirst%3DG.%26aulast%3DKhan%26aufirst%3DM.%2BF.%26aulast%3DAkhtar%26aufirst%3DW.%26aulast%3DAlam%26aufirst%3DM.%2BM.%26aulast%3DAkhter%26aufirst%3DM.%26aulast%3DShaquiquzzaman%26aufirst%3DM.%26atitle%3DA%2520review%2520exploring%2520therapeutic%2520worth%2520of%25201%252C3%252C4-oxadiazole%2520tailored%2520compounds%26jtitle%3DMini-Rev.%2520Med.%2520Chem.%26date%3D2019%26volume%3D19%26spage%3D477%26epage%3D509%26doi%3D10.2174%2F1389557518666181015152433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit28b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Makawana, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H. L.</span></span> <span> </span><span class="NLM_article-title">1,3,4-oxadiazole derivatives as potential biological agents</span>. <i>Mini-Rev. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1725</span>â <span class="NLM_lpage">1743</span>, <span class="refDoi">Â DOI: 10.2174/13895575113139990071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.2174%2F13895575113139990071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=23815585" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFyrsbbF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=1725-1743&author=J.+Sunauthor=J.+A.+Makawanaauthor=H.+L.+Zhu&title=1%2C3%2C4-oxadiazole+derivatives+as+potential+biological+agents&doi=10.2174%2F13895575113139990071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28bR"><div class="casContent"><span class="casTitleNuber">28b</span><div class="casTitle"><span class="NLM_cas:atitle">1,3,4-Oxadiazole Derivatives as Potential Biological Agents</span></div><div class="casAuthors">Sun, Juan; Makawana, Jigar A.; Zhu, Hai-Liang</div><div class="citationInfo"><span class="NLM_cas:title">Mini-Reviews in Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1725-1743</span>CODEN:
                <span class="NLM_cas:coden">MMCIAE</span>;
        ISSN:<span class="NLM_cas:issn">1389-5575</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The synthesis of novel compd. libraries along with screening is a rapid and effective approach for the discovery of potential chem. agents, and it becomes an important method in pharmaceutical chem. research. 1,3,4-oxadiazole derivs. as the typical heterocyclic compds., exhibit a broad spectrum of biol. activities and vital leading compds. for the development of chem. drugs.  Herein, we focus on the synthesis and screening of novel 1,3,4-oxadiazoles derivs. with antimicrobial, antitumor or antiviral activities during the past decade.  In this review, we discussed the synthetic development of 1,3,4-oxadiazoles derivs., and also the relevant bioactivity and their prospects as the potential chem. drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGwV3yfnROebVg90H21EOLACvtfcHk0liTowNqPRQ1aw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFyrsbbF&md5=0bf041b3cc9c11cde0d27523a8b4aafa</span></div><a href="/servlet/linkout?suffix=cit28b&amp;dbid=16384&amp;doi=10.2174%2F13895575113139990071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F13895575113139990071%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DMakawana%26aufirst%3DJ.%2BA.%26aulast%3DZhu%26aufirst%3DH.%2BL.%26atitle%3D1%252C3%252C4-oxadiazole%2520derivatives%2520as%2520potential%2520biological%2520agents%26jtitle%3DMini-Rev.%2520Med.%2520Chem.%26date%3D2013%26volume%3D13%26spage%3D1725%26epage%3D1743%26doi%3D10.2174%2F13895575113139990071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span>1,3,4-Oxadiazole-2(3<i>H</i>)-one predicted p<i>K</i><sub>a</sub>: 6.3 Â±
0.7 (most
acidic), calculated using ACD/Labs software v11.02.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=1%2C3%2C4-Oxadiazole-2%283H%29-one+predicted+pKa%3A+6.3+%C2%B1%0A0.7+%28most%0Aacidic%29%2C+calculated+using+ACD%2FLabs+software+v11.02."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="note"><p class="first last">Acidic p<i>K</i><sub>a</sub> values for CNS drugs rarely fall below 6.0; see:</p></div><div class="NLM_citation" id="cit30a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Manallack, D. T.</span></span> <span> </span><span class="NLM_article-title">The p<i>K</i><sub>a</sub> Distribution of drugs: application to drug discovery</span>. <i>Perspect. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">38</span>, <span class="refDoi">Â DOI: 10.1177/1177391X0700100003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1177%2F1177391X0700100003" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2007&pages=38&author=D.+T.+Manallack&title=The+pKa+Distribution+of+drugs%3A+application+to+drug+discovery&doi=10.1177%2F1177391X0700100003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30a&amp;dbid=16384&amp;doi=10.1177%2F1177391X0700100003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1177391X0700100003%26sid%3Dliteratum%253Aachs%26aulast%3DManallack%26aufirst%3DD.%2BT.%26atitle%3DThe%2520pKa%2520Distribution%2520of%2520drugs%253A%2520application%2520to%2520drug%2520discovery%26jtitle%3DPerspect.%2520Med.%2520Chem.%26date%3D2007%26volume%3D1%26spage%3D38%26doi%3D10.1177%2F1177391X0700100003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="cit30b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Manallack, D. T.</span></span> <span> </span><span class="NLM_article-title">The acid-base profile of a contemporary set of drugs: implications for drug discovery</span>. <i>SAR QSAR Environ. Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">611</span>â <span class="NLM_lpage">655</span>, <span class="refDoi">Â DOI: 10.1080/10629360903438313</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1080%2F10629360903438313" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=20024802" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFyitbrO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2009&pages=611-655&author=D.+T.+Manallack&title=The+acid-base+profile+of+a+contemporary+set+of+drugs%3A+implications+for+drug+discovery&doi=10.1080%2F10629360903438313"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30bR"><div class="casContent"><span class="casTitleNuber">30b</span><div class="casTitle"><span class="NLM_cas:atitle">The acid-base profile of a contemporary set of drugs: implications for drug discovery</span></div><div class="casAuthors">Manallack, D. T.</div><div class="citationInfo"><span class="NLM_cas:title">SAR and QSAR in Environmental Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">7-8</span>),
    <span class="NLM_cas:pages">611-655</span>CODEN:
                <span class="NLM_cas:coden">SQERED</span>;
        ISSN:<span class="NLM_cas:issn">1062-936X</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Acid-base ionization const. (pKa) values have considerable influence on the physicochem. and pharmacokinetic properties of therapeutic substances.  A set of 907 drugs was examd. to det. the proportion of drugs that contain an ionizable group and the distribution of their pKa values.  Using this contemporary set of compds. it was found that 64% of these compds. contained an ionizable group.  Within this group of ionizable compds., 34% contained a single basic group while only 20% contained a single acidic functional group.  The single acid and single base contg. substances were investigated further to examine the distributions of their pKa values.  These data are discussed and analyzed with a focus on the entire set as well as central nervous system, non-central nervous system and oral drugs.  The findings from this research will prompt pharmaceutical companies to assess the constitution of their screening libraries, such that focus is placed on the proportion of ionizable substances, the ratio of acids to bases and the distribution of pKa values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJXs5vGE4IUrVg90H21EOLACvtfcHk0liO-5D55D2jsw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFyitbrO&md5=05a7b0dee5ab8200340ac9a769f0b9a2</span></div><a href="/servlet/linkout?suffix=cit30b&amp;dbid=16384&amp;doi=10.1080%2F10629360903438313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F10629360903438313%26sid%3Dliteratum%253Aachs%26aulast%3DManallack%26aufirst%3DD.%2BT.%26atitle%3DThe%2520acid-base%2520profile%2520of%2520a%2520contemporary%2520set%2520of%2520drugs%253A%2520implications%2520for%2520drug%2520discovery%26jtitle%3DSAR%2520QSAR%2520Environ.%2520Res.%26date%3D2009%26volume%3D20%26spage%3D611%26epage%3D655%26doi%3D10.1080%2F10629360903438313" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span> There is a paucity
of acids with p<i>K</i><sub>a</sub> values between 6 and
7, probably due to the dominance of carboxylic acids and phenols,
and the p<i>K</i><sub>a</sub> values usually encountered
for these groups; see:<span class="NLM_contrib-group"><span class="NLM_string-name">Manallack, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prankerd, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuriev, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oprea, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chalmers, D. K.</span></span> <span> </span><span class="NLM_article-title">The significance of acid/base properties in drug discovery</span>. <i>Chem. Soc. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">485</span>â <span class="NLM_lpage">496</span>, <span class="refDoi">Â DOI: 10.1039/C2CS35348B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1039%2FC2CS35348B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=23099561" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVKmtb7N" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2013&pages=485-496&author=D.+T.+Manallackauthor=R.+J.+Prankerdauthor=E.+Yurievauthor=T.+I.+Opreaauthor=D.+K.+Chalmers&title=The+significance+of+acid%2Fbase+properties+in+drug+discovery&doi=10.1039%2FC2CS35348B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">The significance of acid/base properties in drug discovery</span></div><div class="casAuthors">Manallack, David T.; Prankerd, Richard J.; Yuriev, Elizabeth; Oprea, Tudor I.; Chalmers, David K.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">485-496</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  While drug discovery scientists take heed of various guidelines concerning drug-like character, the influence of acid/base properties often remains under-scrutinized.  Ionisation consts. (pKa values) are fundamental to the variability of the biopharmaceutical characteristics of drugs and to underlying parameters such as logD and soly. pKa values affect physicochem. properties such as aq. soly., which in turn influences drug formulation approaches.  More importantly, absorption, distribution, metab., excretion and toxicity (ADMET) are profoundly affected by the charge state of compds. under varying pH conditions.  Consideration of pKa values in conjunction with other mol. properties is of great significance and has the potential to be used to further improve the efficiency of drug discovery.  Given the recent low annual output of new drugs from pharmaceutical companies, this review will provide a timely reminder of an important mol. property that influences clin. success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKAwdQSoeZxLVg90H21EOLACvtfcHk0liO-5D55D2jsw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVKmtb7N&md5=48d72c5bdae09cc3969cffaea36cf30f</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1039%2FC2CS35348B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC2CS35348B%26sid%3Dliteratum%253Aachs%26aulast%3DManallack%26aufirst%3DD.%2BT.%26aulast%3DPrankerd%26aufirst%3DR.%2BJ.%26aulast%3DYuriev%26aufirst%3DE.%26aulast%3DOprea%26aufirst%3DT.%2BI.%26aulast%3DChalmers%26aufirst%3DD.%2BK.%26atitle%3DThe%2520significance%2520of%2520acid%252Fbase%2520properties%2520in%2520drug%2520discovery%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2013%26volume%3D42%26spage%3D485%26epage%3D496%26doi%3D10.1039%2FC2CS35348B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mahy, W.</span>; <span class="NLM_string-name">Willis, N. J.</span>; <span class="NLM_string-name">Zhao, Y.</span>; <span class="NLM_string-name">Woodward, H.</span>; <span class="NLM_string-name">Svensson, F.</span>; <span class="NLM_string-name">Sipthorp, J.</span>; <span class="NLM_string-name">Vecchia, L.</span>; <span class="NLM_string-name">Ruza, R. R.</span>; <span class="NLM_string-name">Kjaer, S.</span>; <span class="NLM_string-name">Frew, S.</span>; <span class="NLM_string-name">Monaghan, A.</span>; <span class="NLM_string-name">Bictash, M.</span>; <span class="NLM_string-name">Salinas, P.</span>; <span class="NLM_string-name">Whiting, P.</span>; <span class="NLM_string-name">Vincent, J.-P.</span>; <span class="NLM_string-name">Jones, E. Y.</span>; <span class="NLM_string-name">Fish, P. V.</span></span> <span> </span><span class="NLM_article-title">5-Phenyl-1,3,4-oxadiazole-2(3<i>H</i>)-ones are Potent Inhibitors of Notum Carboxylesterase Activity Identified by the Optimization of a Crystallographic Fragment Screening Hit</span>.  <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_fpage">01391</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&pages=01391&author=W.+Mahy&author=N.+J.+Willis&author=Y.+Zhao&author=H.+Woodward&author=F.+Svensson&author=J.+Sipthorp&author=L.+Vecchia&author=R.+R.+Ruza&author=S.+Kjaer&author=S.+Frew&author=A.+Monaghan&author=M.+Bictash&author=P.+Salinas&author=P.+Whiting&author=J.-P.+Vincent&author=E.+Y.+Jones&author=P.+V.+Fish&title=5-Phenyl-1%2C3%2C4-oxadiazole-2%283H%29-ones+are+Potent+Inhibitors+of+Notum+Carboxylesterase+Activity+Identified+by+the+Optimization+of+a+Crystallographic+Fragment+Screening+Hit"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMahy%26aufirst%3DW.%26atitle%3D5-Phenyl-1%252C3%252C4-oxadiazole-2%25283H%2529-ones%2520are%2520Potent%2520Inhibitors%2520of%2520Notum%2520Carboxylesterase%2520Activity%2520Identified%2520by%2520the%2520Optimization%2520of%2520a%2520Crystallographic%2520Fragment%2520Screening%2520Hit%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26spage%3D01391" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rankovic, Z.</span></span> <span> </span><span class="NLM_article-title">CNS drug design: balancing physicochemical properties for optimal brain exposure</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">2584</span>â <span class="NLM_lpage">2608</span>, <span class="refDoi">Â DOI: 10.1021/jm501535r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501535r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVyhsrrP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=2584-2608&author=Z.+Rankovic&title=CNS+drug+design%3A+balancing+physicochemical+properties+for+optimal+brain+exposure&doi=10.1021%2Fjm501535r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">CNS Drug Design: Balancing Physicochemical Properties for Optimal Brain Exposure</span></div><div class="casAuthors">Rankovic, Zoran</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2584-2608</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The human brain is a uniquely complex organ, which has evolved a sophisticated protection system to prevent injury from external insults and toxins.  Designing mols. that can overcome this protection system and achieve optimal concn. at the desired therapeutic target in the brain is a specific and major challenge for medicinal chemists working in CNS drug discovery.  Analogous to the now widely accepted rule of 5 in the design of oral drugs, the physicochem. properties required for optimal brain exposure have been extensively studied in an attempt to similarly define the attributes of successful CNS drugs and drug candidates.  This body of work is systematically reviewed here, with a particular emphasis on the interplay between the most crit. physicochem. and pharmacokinetic parameters of CNS drugs as well as their impact on medicinal chem. strategies toward mols. with optimal brain exposure.  A summary of modern CNS pharmacokinetic concepts and methods is also provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8V0N1j49uuLVg90H21EOLACvtfcHk0lhQzTx-oaYHQQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVyhsrrP&md5=f01e3f2dc4e2eff7b0618124277a5cf8</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm501535r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501535r%26sid%3Dliteratum%253Aachs%26aulast%3DRankovic%26aufirst%3DZ.%26atitle%3DCNS%2520drug%2520design%253A%2520balancing%2520physicochemical%2520properties%2520for%2520optimal%2520brain%2520exposure%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D2584%26epage%3D2608%26doi%3D10.1021%2Fjm501535r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wager, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhoest, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villalobos, A.</span></span> <span> </span><span class="NLM_article-title">Moving beyond rules: the development of a central nervous system multiparameter optimization (CNS MPO) approach to enable alignment of druglike properties</span>. <i>ACS Chem. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">435</span>â <span class="NLM_lpage">449</span>, <span class="refDoi">Â DOI: 10.1021/cn100008c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cn100008c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvVejtr8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=435-449&author=T.+T.+Wagerauthor=X.+Houauthor=P.+R.+Verhoestauthor=A.+Villalobos&title=Moving+beyond+rules%3A+the+development+of+a+central+nervous+system+multiparameter+optimization+%28CNS+MPO%29+approach+to+enable+alignment+of+druglike+properties&doi=10.1021%2Fcn100008c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Moving beyond Rules: The Development of a Central Nervous System Multiparameter Optimization (CNS MPO) Approach To Enable Alignment of Druglike Properties</span></div><div class="casAuthors">Wager, Travis T.; Hou, Xinjun; Verhoest, Patrick R.; Villalobos, Anabella</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Neuroscience</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">435-449</span>CODEN:
                <span class="NLM_cas:coden">ACNCDM</span>;
        ISSN:<span class="NLM_cas:issn">1948-7193</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The interplay among commonly used physicochem. properties in drug design was examd. and utilized to create a prospective design tool focused on the alignment of key druglike attributes.  Using a set of six physicochem. parameters ((a) lipophilicity, calcd. partition coeff. (ClogP); (b) calcd. distribution coeff. at pH = 7.4 (ClogD); (c) mol. wt. (MW); (d) topol. polar surface area (TPSA); (e) no. of hydrogen bond donors (HBD); (f) most basic center (pKa)), a druglikeness central nervous system multiparameter optimization (CNS MPO) algorithm was built and applied to a set of marketed CNS drugs (N = 119) and Pfizer CNS candidates (N = 108), as well as to a large diversity set of Pfizer proprietary compds. (N = 11 303).  The novel CNS MPO algorithm showed that 74% of marketed CNS drugs displayed a high CNS MPO score (MPO desirability score â¥ 4, using a scale of 0-6), in comparison to 60% of the Pfizer CNS candidates.  This anal. suggests that this algorithm could potentially be used to identify compds. with a higher probability of successfully testing hypotheses in the clinic.  In addn., a relationship between an increasing CNS MPO score and alignment of key in vitro attributes of drug discovery (favorable permeability, P-glycoprotein (P-gp) efflux, metabolic stability, and safety) was seen in the marketed CNS drug set, the Pfizer candidate set, and the Pfizer proprietary diversity set.  The CNS MPO scoring function offers advantages over hard cutoffs or utilization of single parameters to optimize structure-activity relationships (SAR) by expanding medicinal chem. design space through a holistic assessment approach.  Based on six physicochem. properties commonly used by medicinal chemists, the CNS MPO function may be used prospectively at the design stage to accelerate the identification of compds. with increased probability of success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqB3Mvkgm_X5bVg90H21EOLACvtfcHk0lhQzTx-oaYHQQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvVejtr8%253D&md5=e94998f65a420be6ee4dec4a987cb983</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fcn100008c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcn100008c%26sid%3Dliteratum%253Aachs%26aulast%3DWager%26aufirst%3DT.%2BT.%26aulast%3DHou%26aufirst%3DX.%26aulast%3DVerhoest%26aufirst%3DP.%2BR.%26aulast%3DVillalobos%26aufirst%3DA.%26atitle%3DMoving%2520beyond%2520rules%253A%2520the%2520development%2520of%2520a%2520central%2520nervous%2520system%2520multiparameter%2520optimization%2520%2528CNS%2520MPO%2529%2520approach%2520to%2520enable%2520alignment%2520of%2520druglike%2520properties%26jtitle%3DACS%2520Chem.%2520Neurosci.%26date%3D2010%26volume%3D1%26spage%3D435%26epage%3D449%26doi%3D10.1021%2Fcn100008c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span></span> <i>Development of Potent, Selective, CNS Penetrant Small Molecule Inhibitors of Notum to Potentiate Wnt Signalling</i>; <span class="NLM_publisher-name">EFMC: XXVth International Symposium on Medicinal Chemistry</span>: <span class="NLM_publisher-loc">Ljubljana, Slovenia</span>, Sep 2â6, <span class="NLM_year">2018</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=P.+V.+Fish&title=Development+of+Potent%2C+Selective%2C+CNS+Penetrant+Small+Molecule+Inhibitors+of+Notum+to+Potentiate+Wnt+Signalling"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DFish%26aufirst%3DP.%2BV.%26btitle%3DDevelopment%2520of%2520Potent%252C%2520Selective%252C%2520CNS%2520Penetrant%2520Small%2520Molecule%2520Inhibitors%2520of%2520Notum%2520to%2520Potentiate%2520Wnt%2520Signalling%26pub%3DEFMC%253A%2520XXVth%2520International%2520Symposium%2520on%2520Medicinal%2520Chemistry%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span>; <span class="NLM_string-name">Mahy, W.</span>; <span class="NLM_string-name">Willis, N. J.</span>; <span class="NLM_string-name">Woodward, H.</span>; <span class="NLM_string-name">Atkinson, B. N.</span>; <span class="NLM_string-name">Bayle, E. D.</span>; <span class="NLM_string-name">Sipthorp, J.</span>; <span class="NLM_string-name">Jones, E. Y.</span>; <span class="NLM_string-name">Zhao, Y.</span>; <span class="NLM_string-name">Vecchia, L.</span>; <span class="NLM_string-name">Ruza, R. R.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of Secreted Carboxylesterase Notum and their use in Increasing Wnt Protein Signalling and Treatment of Central Nervous System and Other Disorders</span>. WO Patent Application <span class="NLM_patent">2020043866</span>, March 5, <span class="NLM_year">2020</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=P.+V.+Fish&author=W.+Mahy&author=N.+J.+Willis&author=H.+Woodward&author=B.+N.+Atkinson&author=E.+D.+Bayle&author=J.+Sipthorp&author=E.+Y.+Jones&author=Y.+Zhao&author=L.+Vecchia&author=R.+R.+Ruza&title=Inhibitors+of+Secreted+Carboxylesterase+Notum+and+their+use+in+Increasing+Wnt+Protein+Signalling+and+Treatment+of+Central+Nervous+System+and+Other+Disorders"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DFish%26aufirst%3DP.%2BV.%26atitle%3DInhibitors%2520of%2520Secreted%2520Carboxylesterase%2520Notum%2520and%2520their%2520use%2520in%2520Increasing%2520Wnt%2520Protein%2520Signalling%2520and%2520Treatment%2520of%2520Central%2520Nervous%2520System%2520and%2520Other%2520Disorders%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kazembe, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, D. A.</span></span> <span> </span><span class="NLM_article-title">Synthesis of pyrrolo[1,2-a]indoles: Vilsmeier formylation of some 3-methylindol-2-yl ketones and 3(3-methylindol-2-yl)propenoic esters</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">1980</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">2125</span>â <span class="NLM_lpage">2132</span>, <span class="refDoi">Â DOI: 10.1016/0040-4020(80)80102-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1016%2F0040-4020%2880%2980102-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADyaL3MXotlyksQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1980&pages=2125-2132&author=C.+G.+Kazembeauthor=D.+A.+Taylor&title=Synthesis+of+pyrrolo%5B1%2C2-a%5Dindoles%3A+Vilsmeier+formylation+of+some+3-methylindol-2-yl+ketones+and+3%283-methylindol-2-yl%29propenoic+esters&doi=10.1016%2F0040-4020%2880%2980102-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of pyrrolo[1,2-a]indoles: Vilsmeier formylation of some 3-methylindol-2-yl ketones and 3(3-methylindol-2-yl)propenoic ester</span></div><div class="casAuthors">Kazembe, Takawira C. G.; Taylor, Deryck A.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">1980</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2125-32</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    </div><div class="casAbstract">The prepn. of several pyrrolo[1,2-a]indoles (e.g. I) by formylation/cyclization of indoles is reported.  E.g., methylindole ester II was prepd. and reacted with POCl3/DMF at 60Â° for 1 h to give 67% I.  The prepn. of some monomethylated 1-phenylpyrrole-2,4- and -3,4-dicarboxylates is also reported.  E.g., BrCH2COCO2Et reacted with PhNHCMe:CHCO2Et for 20 min in refluxing EtOH to give 72% diester III.  The sapon. of these diesters is remarkably selective, and in the case of 2,4-dicarboxylates contradicts accepted generalizations concerning the lability of pyrrole Î±,Î²-diesters to selective hydrolysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpr1bzL4PfdN7Vg90H21EOLACvtfcHk0lgZ_JDW48yDEg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3MXotlyksQ%253D%253D&md5=ac25d0e3f5bac6994005dd0c538e790e</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2F0040-4020%2880%2980102-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0040-4020%252880%252980102-5%26sid%3Dliteratum%253Aachs%26aulast%3DKazembe%26aufirst%3DC.%2BG.%26aulast%3DTaylor%26aufirst%3DD.%2BA.%26atitle%3DSynthesis%2520of%2520pyrrolo%255B1%252C2-a%255Dindoles%253A%2520Vilsmeier%2520formylation%2520of%2520some%25203-methylindol-2-yl%2520ketones%2520and%25203%25283-methylindol-2-yl%2529propenoic%2520esters%26jtitle%3DTetrahedron%26date%3D1980%26volume%3D36%26spage%3D2125%26epage%3D2132%26doi%3D10.1016%2F0040-4020%2880%2980102-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beard, R. L.</span>, <span class="NLM_string-name">Donello, J. E.</span>, <span class="NLM_string-name">Yuan, H.</span>, <span class="NLM_string-name">Colon, D. F.</span>, <span class="NLM_string-name">Duong, T.</span>, <span class="NLM_string-name">Liu, X.</span>, <span class="NLM_string-name">Hu, Y.</span></span> <span> </span><span class="NLM_article-title">Indole-3-carboxylic Acid Amide, Ester, Thioamide, and Thiol Ester Compounds Bearing Aryl or Heteroaryl Groups Having Sphingosine-1-phosphate (s1p) Receptor Antagonist Biological Activity</span>. US Patent <span class="NLM_patent">US2007/0191313-A1</span>, Aug 16, <span class="NLM_year">2007</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&author=R.+L.+Beard&author=J.+E.+Donello&author=H.+Yuan&author=D.+F.+Colon&author=T.+Duong&author=X.+Liu&author=Y.+Hu&title=Indole-3-carboxylic+Acid+Amide%2C+Ester%2C+Thioamide%2C+and+Thiol+Ester+Compounds+Bearing+Aryl+or+Heteroaryl+Groups+Having+Sphingosine-1-phosphate+%28s1p%29+Receptor+Antagonist+Biological+Activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBeard%26aufirst%3DR.%2BL.%26atitle%3DIndole-3-carboxylic%2520Acid%2520Amide%252C%2520Ester%252C%2520Thioamide%252C%2520and%2520Thiol%2520Ester%2520Compounds%2520Bearing%2520Aryl%2520or%2520Heteroaryl%2520Groups%2520Having%2520Sphingosine-1-phosphate%2520%2528s1p%2529%2520Receptor%2520Antagonist%2520Biological%2520Activity%26date%3D2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':[],'ref28':['cit28a','cit28b'],'ref29':['cit29'],'ref30':['cit30a','cit30b'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 5 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Yuguang Zhao, Fredrik Svensson, David Steadman, Sarah Frew, Amy Monaghan, Magda Bictash, Tiago Moreira, Rod Chalk, Weixian Lu, Paul V. Fish, <span class="NLM_string-name hlFld-ContribAuthor">E. Yvonne Jones</span>. </span><span class="cited-content_cbyCitation_article-title">Structural Insights into Notum Covalent Inhibition. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00701%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructural%252BInsights%252Binto%252BNotum%252BCovalent%252BInhibition%26aulast%3DZhao%26aufirst%3DYuguang%26date%3D2021%26date%3D2021%26date%3D17042021%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Elliott D. Bayle, Fredrik Svensson, Benjamin N. Atkinson, David Steadman, Nicky J. Willis, Hannah L. Woodward, Paul Whiting, Jean-Paul Vincent, <span class="NLM_string-name hlFld-ContribAuthor">Paul V. Fish</span>. </span><span class="cited-content_cbyCitation_article-title">Carboxylesterase Notum Is a Druggable Target to Modulate Wnt Signaling. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (8)
                                     , 4289-4311. <a href="https://doi.org/10.1021/acs.jmedchem.0c01974" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01974</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01974%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DCarboxylesterase%252BNotum%252BIs%252Ba%252BDruggable%252BTarget%252Bto%252BModulate%252BWnt%252BSignaling%26aulast%3DBayle%26aufirst%3DElliott%2BD.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D14112020%26date%3D30032021%26volume%3D64%26issue%3D8%26spage%3D4289%26epage%3D4311" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">William Mahy, Nicky J. Willis, Yuguang Zhao, Hannah L. Woodward, Fredrik Svensson, James Sipthorp, Luca Vecchia, Reinis R. Ruza, James Hillier, Svend KjÃ¦r, Sarah Frew, Amy Monaghan, Magda Bictash, Patricia C. Salinas, Paul Whiting, Jean-Paul Vincent, E. Yvonne Jones, <span class="NLM_string-name hlFld-ContribAuthor">Paul V. Fish</span>. </span><span class="cited-content_cbyCitation_article-title">5-Phenyl-1,3,4-oxadiazol-2(3H)-ones Are Potent Inhibitors of Notum Carboxylesterase Activity Identified by the Optimization of a Crystallographic Fragment Screening Hit. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (21)
                                     , 12942-12956. <a href="https://doi.org/10.1021/acs.jmedchem.0c01391" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01391</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01391&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01391%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3D5-Phenyl-1%25252C3%25252C4-oxadiazol-2%2525283H%252529-ones%252BAre%252BPotent%252BInhibitors%252Bof%252BNotum%252BCarboxylesterase%252BActivity%252BIdentified%252Bby%252Bthe%252BOptimization%252Bof%252Ba%252BCrystallographic%252BFragment%252BScreening%252BHit%26aulast%3DMahy%26aufirst%3DWilliam%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D08082020%26date%3D30102020%26volume%3D63%26issue%3D21%26spage%3D12942%26epage%3D12956" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuguang  Zhao</span>, <span class="hlFld-ContribAuthor ">Laura-Nadine  Schuhmacher</span>, <span class="hlFld-ContribAuthor ">Morgan  Roberts</span>, <span class="hlFld-ContribAuthor ">Satoshi  Kakugawa</span>, <span class="hlFld-ContribAuthor ">Ganka  Bineva-Todd</span>, <span class="hlFld-ContribAuthor ">Steve  Howell</span>, <span class="hlFld-ContribAuthor ">Nicola  O'Reilly</span>, <span class="hlFld-ContribAuthor ">Christine  Perret</span>, <span class="hlFld-ContribAuthor ">Ambrosius P.  Snijders</span>, <span class="hlFld-ContribAuthor ">Jean-Paul  Vincent</span>, <span class="hlFld-ContribAuthor ">E. Yvonne  Jones</span>. </span><span class="cited-content_cbyCitation_article-title">Notum deacylates octanoylated ghrelin. </span><span class="cited-content_cbyCitation_journal-name">Molecular Metabolism</span><span> <strong>2021,</strong> <em>49 </em>, 101201. <a href="https://doi.org/10.1016/j.molmet.2021.101201" title="DOI URL">https://doi.org/10.1016/j.molmet.2021.101201</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.molmet.2021.101201&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.molmet.2021.101201%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Metabolism%26atitle%3DNotum%252Bdeacylates%252Boctanoylated%252Bghrelin%26aulast%3DZhao%26aufirst%3DYuguang%26date%3D2021%26volume%3D49%26spage%3D101201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuguang  Zhao</span>, <span class="hlFld-ContribAuthor ">Sarah  Jolly</span>, <span class="hlFld-ContribAuthor ">Stefano  Benvegnu</span>, <span class="hlFld-ContribAuthor ">E Yvonne  Jones</span>, <span class="hlFld-ContribAuthor ">Paul V  Fish</span>. </span><span class="cited-content_cbyCitation_article-title">Small-molecule inhibitors of carboxylesterase Notum. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2021,</strong> <em>13 </em>
                                    (11)
                                     , 1001-1015. <a href="https://doi.org/10.4155/fmc-2021-0036" title="DOI URL">https://doi.org/10.4155/fmc-2021-0036</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2021-0036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2021-0036%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DSmall-molecule%252Binhibitors%252Bof%252Bcarboxylesterase%252BNotum%26aulast%3DZhao%26aufirst%3DYuguang%26date%3D2021%26volume%3D13%26issue%3D11%26spage%3D1001%26epage%3D1015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/medium/jm0c00660_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/large/jm0c00660_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00660&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/medium/jm0c00660_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/large/jm0c00660_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures and PDB codes of Notum inhibitors. <sup>a</sup>Notum IC<sub>50</sub> data presented for comparison in a common assay format. <sup>b</sup>Determined by competitive gel-based activity-based protein profiling (ABPP); see ref <a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a>. <sup>c</sup>As a covalent inhibitor of Notum, the IC<sub>50</sub> value will be time-dependent. <sup>d</sup>See ref <a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/large/jm0c00660_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00660&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/medium/jm0c00660_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/large/jm0c00660_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Design and screening of the fragment library. (A) Compound structural features. (B) Plot of lipophilicity (clog<i>P</i>) versus molecular size (MW) for the acid fragment library. (C) Flow diagram showing key activities and results.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/large/jm0c00660_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00660&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/medium/jm0c00660_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/large/jm0c00660_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Pyrrole-3-carboxylic acids <b>5</b>â<b>7</b> and pyrrolidine-3-carboxylic acids <b>8</b>â<b>10</b> as fragment screening hits. Notum IC<sub>50</sub> (Î¼M) (<i>n</i> = 5â10).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/large/jm0c00660_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00660&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/medium/jm0c00660_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/large/jm0c00660_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Notum structures with fragment screening hits <b>5</b>â<b>8</b> and <b>10</b>. Crystal structures of (A) <b>5</b> (orange), (B) <b>6</b> (teal), (C) <b>7</b> (gray), (D) <b>8</b> (magenta), and (E) <b>10</b> (yellow). (F) Overlay of all ligands. Binding site residues shown within 4 Ã of their respective ligands. Key hydrogen bond interactions are shown as dashed lines. Water molecules have been removed for clarity. The surface of the Notum palmitoleoate binding pocket outlined (gray) from the <i>O</i>-palmitoleoyl serineâNotum (S232A) structure (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4UZQ">4UZQ</a>). Atomic coordinates have been deposited in the Protein Data Bank (PDB). PDB ID codes: <b>5</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YUY">6YUY</a>; <b>6</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YV4">6YV4</a>; <b>7</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YUW">6YUW</a>; <b>8</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YV2">6YV2</a>; <b>10</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YV0">6YV0</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/large/jm0c00660_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00660&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/medium/jm0c00660_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/large/jm0c00660_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Preparation of Pyrrole-3-carboxylic Acids <b>11</b>â<b>24</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/large/jm0c00660_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00660&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Representative reagents and conditions: (a) NaH (1.2 equiv), THF, 0 Â°C then R<sup>5</sup>C(O)CH<sub>2</sub>Cl <b>32</b> (1.0 equiv), NaI (0.1 equiv) 0 Â°C to RT, 24 h; (b) amine <b>34</b> (R<sup>1</sup>NH<sub>2</sub>) (1.0 equiv), <i>p</i>-TsOHÂ·H<sub>2</sub>O (0.1 equiv), EtOH, 80 Â°C; (c) LiOHÂ·H<sub>2</sub>O (15 equiv), 1,4-dioxaneâH<sub>2</sub>O, 100 Â°C.</p></p></figure><figure data-id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/medium/jm0c00660_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/large/jm0c00660_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Preparation of Pyrrolidine-3-carboxylic Acids <b>25</b>â<b>27</b>, Amides <b>28</b>, Oxadiazole <b>29</b>, and Oxadiazolone <b>30</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/large/jm0c00660_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00660&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Representative reagents and conditions: (a) <b>36</b> (1.2 equiv), ArI <b>37</b> (1.0 equiv), Pd<sub>2</sub>dba<sub>3</sub> (2.5 mol %), Xantphos (5 mol %), Cs<sub>2</sub>CO<sub>3</sub> (2.7 equiv), 1,4-dioxane, 100 Â°C, 16 h; (b) aq NaOH (1 M, 2.0 equiv), MeOH, RT, 2 h; (c) HATU (1.1 equiv), DMF, RT, 15 min; then iPr<sub>2</sub>NEt (2.2 equiv), amine <b>39</b> (HNRRâ²) (1.5 equiv); (d) NH<sub>2</sub>NH<sub>2</sub>Â·H<sub>2</sub>O (4 equiv), EtOHâPhMe, 80 Â°C, 16 h; (e) HC(OEt)<sub>3</sub> (4.5 equiv), <i>p</i>-TsOHÂ·H<sub>2</sub>O (0.1 equiv), 60 Â°C, 5 min then RT, 1 h; (f) triphosgene (0.4 equiv), iPr<sub>2</sub>NEt (2.0 equiv), CH<sub>2</sub>Cl<sub>2</sub>, 0 Â°C to RT, 15 min.</p></p></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/medium/jm0c00660_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/large/jm0c00660_0005.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Notum structures with pyrrole and pyrrolidine inhibitors. Crystal structures of (A) <b>20i</b> (emerald) and (B) <b>26</b> (green). (C) Overlay of <b>20i</b> (emerald) and <b>26</b> (green) shows different binding modes. (D) Overlay of <b>26</b> (green) with <b>1</b> (purple; PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T2K">6T2K</a>) shows acid groups aligned. Binding site residues shown within 4 Ã of their respective ligands. Key hydrogen bond interactions are shown as dashed lines. Water molecules have been removed for clarity. The surface of the Notum palmitoleoate binding pocket outlined (gray) from the <i>O</i>-palmitoleoyl serineâNotum (S232A) structure (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4UZQ">4UZQ</a>). Atomic coordinates have been deposited in the Protein Data Bank (PDB). PDB ID codes: <b>20i</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YXI">6YXI</a>; <b>26</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YSK">6YSK</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-17/acs.jmedchem.0c00660/20200903/images/large/jm0c00660_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00660&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i126">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30634" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30634" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 38 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nusse, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clevers, H.</span></span> <span> </span><span class="NLM_article-title">Wnt/Î²-Catenin signaling, disease, and emerging therapeutic modalities</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>169</i></span>,  <span class="NLM_fpage">985</span>â <span class="NLM_lpage">999</span>, <span class="refDoi">Â DOI: 10.1016/j.cell.2017.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1016%2Fj.cell.2017.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=28575679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpt1agsro%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=169&publication_year=2017&pages=985-999&author=R.+Nusseauthor=H.+Clevers&title=Wnt%2F%CE%B2-Catenin+signaling%2C+disease%2C+and+emerging+therapeutic+modalities&doi=10.1016%2Fj.cell.2017.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Wnt/Î²-catenin signaling, disease, and emerging therapeutic modalities</span></div><div class="casAuthors">Nusse, Roel; Clevers, Hans</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">169</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">985-999</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The WNT signal transduction cascade is a main regulator of development throughout the animal kingdom.  Wnts are also key drivers of most types of tissue stem cells in adult mammals.  Unsurprisingly, mutated Wnt pathway components are causative to multiple growth-related pathologies and to cancer.  Here, we describe the core Wnt/Î²-catenin signaling pathway, how it controls stem cells, and contributes to disease.  Finally, we discuss strategies for Wnt-based therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnL11nCbSs8LVg90H21EOLACvtfcHk0litlbwz5O2c4g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpt1agsro%253D&md5=869cc558296677481527ac138f68fbb0</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DNusse%26aufirst%3DR.%26aulast%3DClevers%26aufirst%3DH.%26atitle%3DWnt%252F%25CE%25B2-Catenin%2520signaling%252C%2520disease%252C%2520and%2520emerging%2520therapeutic%2520modalities%26jtitle%3DCell%26date%3D2017%26volume%3D169%26spage%3D985%26epage%3D999%26doi%3D10.1016%2Fj.cell.2017.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niehrs, C.</span></span> <span> </span><span class="NLM_article-title">Function and biological roles of the Dickkopf family of Wnt modulators</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">7469</span>â <span class="NLM_lpage">7481</span>, <span class="refDoi">Â DOI: 10.1038/sj.onc.1210054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1038%2Fsj.onc.1210054" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=17143291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1Kgt7%252FN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=7469-7481&author=C.+Niehrs&title=Function+and+biological+roles+of+the+Dickkopf+family+of+Wnt+modulators&doi=10.1038%2Fsj.onc.1210054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Function and biological roles of the Dickkopf family of Wnt modulators</span></div><div class="casAuthors">Niehrs, C.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">57</span>),
    <span class="NLM_cas:pages">7469-7481</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Dickkopf (Dkk) genes comprise an evolutionary conserved small gene family of 4 members (Dkk1-4) and a unique Dkk3-related gene, Dkkl1 (soggy).  They encode secreted proteins that typically antagonize Wnt/Î²-catenin signaling, by inhibiting the Wnt coreceptors Lrp5 and 6.  Addnl., Dkks are high affinity ligands for the transmembrane proteins Kremen1 and 2, which also modulate Wnt signaling.  Dkks play an important role in vertebrate development, where they locally inhibit Wnt regulated processes such as antero-posterior axial patterning, limb development, somitogenesis, and eye formation.  In the adult, Dkks are implicated in bone formation and bone disease, cancer, and Alzheimer's disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-xPIRQqosmbVg90H21EOLACvtfcHk0lhPlPQI_dW0eQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1Kgt7%252FN&md5=f871bcae6fe6f367d650f423723eef84</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1210054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1210054%26sid%3Dliteratum%253Aachs%26aulast%3DNiehrs%26aufirst%3DC.%26atitle%3DFunction%2520and%2520biological%2520roles%2520of%2520the%2520Dickkopf%2520family%2520of%2520Wnt%2520modulators%26jtitle%3DOncogene%26date%3D2006%26volume%3D25%26spage%3D7469%26epage%3D7481%26doi%3D10.1038%2Fsj.onc.1210054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niehrs, C.</span></span> <span> </span><span class="NLM_article-title">The complex world of WNT receptor signaling</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">767</span>â <span class="NLM_lpage">779</span>, <span class="refDoi">Â DOI: 10.1038/nrm3470</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1038%2Fnrm3470" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=23151663" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Klsr%252FM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=767-779&author=C.+Niehrs&title=The+complex+world+of+WNT+receptor+signaling&doi=10.1038%2Fnrm3470"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The complex world of WNT receptor signalling</span></div><div class="casAuthors">Niehrs, Christof</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">767-779</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  30 Years after the identification of WNTs, their signal transduction has become increasingly complex, with the discovery of more than 15 receptors and co-receptors in seven protein families.  The recent discovery of three receptor classes for the R-spondin family of WNT agonists further adds to this complexity.  What emerges is an intricate network of receptors that form higher-order ligand-receptor complexes routing downstream signaling.  These are regulated both extracellularly by agonists such as R-spondin and intracellularly by post-translational modifications such as phosphorylation, proteolytic processing and endocytosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpb-xsoqljhjLVg90H21EOLACvtfcHk0lhPlPQI_dW0eQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Klsr%252FM&md5=b0f3fe641f7a5f77774ab54d8cc5aeb9</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrm3470&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3470%26sid%3Dliteratum%253Aachs%26aulast%3DNiehrs%26aufirst%3DC.%26atitle%3DThe%2520complex%2520world%2520of%2520WNT%2520receptor%2520signaling%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2012%26volume%3D13%26spage%3D767%26epage%3D779%26doi%3D10.1038%2Fnrm3470" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malinauskas, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. Y.</span></span> <span> </span><span class="NLM_article-title">Extracellular modulators of Wnt signaling</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">77</span>â <span class="NLM_lpage">84</span>, <span class="refDoi">Â DOI: 10.1016/j.sbi.2014.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1016%2Fj.sbi.2014.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=25460271" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslKjtrrJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2014&pages=77-84&author=T.+Malinauskasauthor=E.+Y.+Jones&title=Extracellular+modulators+of+Wnt+signaling&doi=10.1016%2Fj.sbi.2014.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Extracellular modulators of Wnt signalling</span></div><div class="casAuthors">Malinauskas, Tomas; Jones, E. Yvonne</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">77-84</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Wnt morphogens are secreted signalling proteins that play leading roles in embryogenesis and tissue homeostasis throughout life.  Wnt signalling is controlled by multiple mechanisms, including posttranslational modification of Wnts, antagonist binding (to Wnts or their receptors), and regulation of the availability of Wnt receptors.  Recent crystallog., structure-guided biophys. and cell-based studies have advanced our understanding of how Wnt signalling is regulated at the cell surface.  Structures include Wnt in complex with the cysteine-rich domain (CRD) of Frizzled, extracellular fragments of Wnt co-receptor LRP6, LRP6-binding antagonists Dickkopf and Sclerostin, antagonists 5T4/WAIF1 and Wnt inhibitory factor 1 (WIF-1), as well as Frizzled-ubiquitin ligases ZNRF3/RNF43 (in isolation and in complexes with Wnt signalling promoters R-spondins and LGR5).  We review recent discoveries and remaining questions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuYvmbUHeJhLVg90H21EOLACvtfcHk0lhPlPQI_dW0eQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslKjtrrJ&md5=68ae12b2925971654ea428a3ae4179ca</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2014.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2014.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DMalinauskas%26aufirst%3DT.%26aulast%3DJones%26aufirst%3DE.%2BY.%26atitle%3DExtracellular%2520modulators%2520of%2520Wnt%2520signaling%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2014%26volume%3D29%26spage%3D77%26epage%3D84%26doi%3D10.1016%2Fj.sbi.2014.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rindtorff, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutros, M.</span></span> <span> </span><span class="NLM_article-title">Wnt signaling in cancer</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">1461</span>â <span class="NLM_lpage">1473</span>, <span class="refDoi">Â DOI: 10.1038/onc.2016.304</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1038%2Fonc.2016.304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=27617575" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsV2gsL3O" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2017&pages=1461-1473&author=T.+Zhanauthor=N.+Rindtorffauthor=M.+Boutros&title=Wnt+signaling+in+cancer&doi=10.1038%2Fonc.2016.304"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Wnt signaling in cancer</span></div><div class="casAuthors">Zhan, T.; Rindtorff, N.; Boutros, M.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1461-1473</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Wnt signaling is one of the key cascades regulating development and stemness, and has also been tightly assocd. with cancer.  The role of Wnt signaling in carcinogenesis has most prominently been described for colorectal cancer, but aberrant Wnt signaling is obsd. in many more cancer entities.  Here, we review current insights into novel components of Wnt pathways and describe their impact on cancer development.  Furthermore, we highlight expanding functions of Wnt signaling for both solid and liq. tumors.  We also describe current findings how Wnt signaling affects maintenance of cancer stem cells, metastasis and immune control.  Finally, we provide an overview of current strategies to antagonize Wnt signaling in cancer and challenges that are assocd. with such approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5DnHKAd0M3rVg90H21EOLACvtfcHk0lhPlPQI_dW0eQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsV2gsL3O&md5=abef967a4e295e8d997ea2f241b035bc</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fonc.2016.304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2016.304%26sid%3Dliteratum%253Aachs%26aulast%3DZhan%26aufirst%3DT.%26aulast%3DRindtorff%26aufirst%3DN.%26aulast%3DBoutros%26aufirst%3DM.%26atitle%3DWnt%2520signaling%2520in%2520cancer%26jtitle%3DOncogene%26date%3D2017%26volume%3D36%26spage%3D1461%26epage%3D1473%26doi%3D10.1038%2Fonc.2016.304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Palomer, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buechler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salinas, P. C.</span></span> <span> </span><span class="NLM_article-title">Wnt signalling deregulation in the aging and Alzheimerâs brain</span>. <i>Front. Cell. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">227</span>, <span class="refDoi">Â DOI: 10.3389/fncel.2019.00227</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.3389%2Ffncel.2019.00227" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=31191253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BB3cXktV2ksLg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2019&pages=227&author=E.+Palomerauthor=J.+Buechlerauthor=P.+C.+Salinas&title=Wnt+signalling+deregulation+in+the+aging+and+Alzheimer%E2%80%99s+brain&doi=10.3389%2Ffncel.2019.00227"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Wnt signaling deregulation in the aging and Alzheimer's brain</span></div><div class="casAuthors">Palomer, Ernest; Buechler, Johanna; Salinas, Patricia C.</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Cellular Neuroscience</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">227</span>CODEN:
                <span class="NLM_cas:coden">FCNRAH</span>;
        ISSN:<span class="NLM_cas:issn">1662-5102</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">Growing evidence suggests that synaptic signaling is compromised in the aging brain and in Alzheimer's disease (AD), contributing to synaptic decline.  Wnt signaling is a prominent pathway at the synapse and is required for synaptic plasticity and maintenance in the adult brain.  In this review, we summarize the current knowledge on deregulation of Wnt signaling in the context of aging and AD.  Emerging studies suggest that enhancing Wnt signaling could boost synaptic function during aging, and ameliorate synaptic pathol. in AD.  Although further research is needed to det. the precise contribution of deficient Wnt signaling to AD pathogenesis, targeting Wnt signaling components may provide novel therapeutic avenues for synapse protection or restoration in the brain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBS_KuzUNAgbVg90H21EOLACvtfcHk0lgymKlsjs26LQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXktV2ksLg%253D&md5=e807df970865d7e712425076ef013ac2</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.3389%2Ffncel.2019.00227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffncel.2019.00227%26sid%3Dliteratum%253Aachs%26aulast%3DPalomer%26aufirst%3DE.%26aulast%3DBuechler%26aufirst%3DJ.%26aulast%3DSalinas%26aufirst%3DP.%2BC.%26atitle%3DWnt%2520signalling%2520deregulation%2520in%2520the%2520aging%2520and%2520Alzheimer%25E2%2580%2599s%2520brain%26jtitle%3DFront.%2520Cell.%2520Neurosci.%26date%3D2019%26volume%3D13%26spage%3D227%26doi%3D10.3389%2Ffncel.2019.00227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ng, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaur, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnag, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suresh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sung, I. C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Q. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolwinski, N. S.</span></span> <span> </span><span class="NLM_article-title">WNT signaling in disease</span>. <i>Cells</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">826</span>, <span class="refDoi">Â DOI: 10.3390/cells8080826</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.3390%2Fcells8080826" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlsVGis7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2019&pages=826&author=L.+F.+Ngauthor=P.+Kaurauthor=N.+Bunnagauthor=J.+Sureshauthor=I.+C.+H.+Sungauthor=Q.+H.+Tanauthor=J.+Gruberauthor=N.+S.+Tolwinski&title=WNT+signaling+in+disease&doi=10.3390%2Fcells8080826"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">WNT signaling in disease</span></div><div class="casAuthors">Ng, Li Fang; Kaur, Prameet; Bunnag, Nawat; Suresh, Jahnavi; Sung, Isabelle Chiao Han; Tan, Qian Hui; Gruber, Jan; Tolwinski, Nicholas S.</div><div class="citationInfo"><span class="NLM_cas:title">Cells</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">826</span>CODEN:
                <span class="NLM_cas:coden">CELLC6</span>;
        ISSN:<span class="NLM_cas:issn">2073-4409</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Developmental signaling pathways control a vast array of biol. processes during embryogenesis and in adult life.  The WNT pathway was discovered simultaneously in cancer and development.  Recent advances have expanded the role of WNT to a wide range of pathologies in humans.  Here, we discuss the WNT pathway and its role in human disease and some of the advances in WNT-related treatments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDDLPhmGEBILVg90H21EOLACvtfcHk0lgymKlsjs26LQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlsVGis7w%253D&md5=f88171d5418fd12052faa352a7196326</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.3390%2Fcells8080826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcells8080826%26sid%3Dliteratum%253Aachs%26aulast%3DNg%26aufirst%3DL.%2BF.%26aulast%3DKaur%26aufirst%3DP.%26aulast%3DBunnag%26aufirst%3DN.%26aulast%3DSuresh%26aufirst%3DJ.%26aulast%3DSung%26aufirst%3DI.%2BC.%2BH.%26aulast%3DTan%26aufirst%3DQ.%2BH.%26aulast%3DGruber%26aufirst%3DJ.%26aulast%3DTolwinski%26aufirst%3DN.%2BS.%26atitle%3DWNT%2520signaling%2520in%2520disease%26jtitle%3DCells%26date%3D2019%26volume%3D8%26spage%3D826%26doi%3D10.3390%2Fcells8080826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kakugawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langton, P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zebisch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feizi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bineva, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâReilly, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snijders, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, J. P.</span></span> <span> </span><span class="NLM_article-title">NOTUM deacylates Wnt proteins to suppress signaling activity</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>519</i></span>,  <span class="NLM_fpage">187</span>â <span class="NLM_lpage">192</span>, <span class="refDoi">Â DOI: 10.1038/nature14259</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1038%2Fnature14259" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=25731175" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC2MXktVCnsb4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=519&publication_year=2015&pages=187-192&author=S.+Kakugawaauthor=P.+F.+Langtonauthor=M.+Zebischauthor=S.+Howellauthor=T.+H.+Changauthor=Y.+Liuauthor=T.+Feiziauthor=G.+Binevaauthor=N.+O%E2%80%99Reillyauthor=A.+P.+Snijdersauthor=E.+Y.+Jonesauthor=J.+P.+Vincent&title=NOTUM+deacylates+Wnt+proteins+to+suppress+signaling+activity&doi=10.1038%2Fnature14259"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Notum deacylates Wnt proteins to suppress signalling activity</span></div><div class="casAuthors">Kakugawa, Satoshi; Langton, Paul F.; Zebisch, Matthias; Howell, Steven A.; Chang, Tao-Hsin; Liu, Yan; Feizi, Ten; Bineva, Ganka; O'Reilly, Nicola; Snijders, Ambrosius P.; Jones, E. Yvonne; Vincent, Jean-Paul</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">519</span>
        (<span class="NLM_cas:issue">7542</span>),
    <span class="NLM_cas:pages">187-192</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Signalling by Wnt proteins is finely balanced to ensure normal development and tissue homeostasis while avoiding diseases such as cancer.  This is achieved in part by Notum, a highly conserved secreted feedback antagonist.  Notum has been thought to act as a phospholipase, shedding glypicans and assocd. Wnt proteins from the cell surface.  However, this view fails to explain specificity, as glypicans bind many extracellular ligands.  Here we provide genetic evidence in Drosophila that Notum requires glypicans to suppress Wnt signalling, but does not cleave their glycophosphatidylinositol anchor.  Structural analyses reveal glycosaminoglycan binding sites on Notum, which probably help Notum to co-localize with Wnt proteins.  They also identify, at the active site of human and Drosophila Notum, a large hydrophobic pocket that accommodates palmitoleate.  Kinetic and mass spectrometric analyses of human proteins show that Notum is a carboxylesterase that removes an essential palmitoleate moiety from Wnt proteins and thus constitutes the first known extracellular protein deacylase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqffMMn2oxsbLVg90H21EOLACvtfcHk0lgymKlsjs26LQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXktVCnsb4%253D&md5=0668dc3cdc07e4bd527551424c4b91d4</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnature14259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14259%26sid%3Dliteratum%253Aachs%26aulast%3DKakugawa%26aufirst%3DS.%26aulast%3DLangton%26aufirst%3DP.%2BF.%26aulast%3DZebisch%26aufirst%3DM.%26aulast%3DHowell%26aufirst%3DS.%26aulast%3DChang%26aufirst%3DT.%2BH.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DFeizi%26aufirst%3DT.%26aulast%3DBineva%26aufirst%3DG.%26aulast%3DO%25E2%2580%2599Reilly%26aufirst%3DN.%26aulast%3DSnijders%26aufirst%3DA.%2BP.%26aulast%3DJones%26aufirst%3DE.%2BY.%26aulast%3DVincent%26aufirst%3DJ.%2BP.%26atitle%3DNOTUM%2520deacylates%2520Wnt%2520proteins%2520to%2520suppress%2520signaling%2520activity%26jtitle%3DNature%26date%3D2015%26volume%3D519%26spage%3D187%26epage%3D192%26doi%3D10.1038%2Fnature14259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheong, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amado, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reis, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacDonald, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zebisch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abreu, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, X.</span></span> <span> </span><span class="NLM_article-title">NOTUM is required for neural and head induction via Wnt deacylation, oxidation, and inactivation</span>. <i>Dev. Cell</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">719</span>â <span class="NLM_lpage">730</span>, <span class="refDoi">Â DOI: 10.1016/j.devcel.2015.02.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1016%2Fj.devcel.2015.02.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=25771893" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC2MXksVaisLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2015&pages=719-730&author=X.+Zhangauthor=S.+M.+Cheongauthor=N.+G.+Amadoauthor=A.+H.+Reisauthor=B.+T.+MacDonaldauthor=M.+Zebischauthor=E.+Y.+Jonesauthor=J.+G.+Abreuauthor=X.+He&title=NOTUM+is+required+for+neural+and+head+induction+via+Wnt+deacylation%2C+oxidation%2C+and+inactivation&doi=10.1016%2Fj.devcel.2015.02.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Notum Is Required for Neural and Head Induction via Wnt Deacylation, Oxidation, and Inactivation</span></div><div class="casAuthors">Zhang, Xinjun; Cheong, Seong-Moon; Amado, Nathalia G.; Reis, Alice H.; MacDonald, Bryan T.; Zebisch, Matthias; Jones, E. Yvonne; Abreu, Jose Garcia; He, Xi</div><div class="citationInfo"><span class="NLM_cas:title">Developmental Cell</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">719-730</span>CODEN:
                <span class="NLM_cas:coden">DCEEBE</span>;
        ISSN:<span class="NLM_cas:issn">1534-5807</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Secreted Wnt morphogens are essential for embryogenesis and homeostasis and require a lipid/palmitoleoylate modification for receptor binding and activity.  Notum is a secreted Wnt antagonist that belongs to the Î±/Î² hydrolase superfamily, but its mechanism of action and roles in vertebrate embryogenesis are not fully understood.  Here, we report that Notum hydrolyzes the Wnt palmitoleoylate adduct extracellularly, resulting in inactivated Wnt proteins that form oxidized oligomers incapable of receptor binding.  Thus, Notum is a Wnt deacylase, and palmitoleoylation is obligatory for the Wnt structure that maintains its active monomeric conformation.  Notum is expressed in naive ectoderm and neural plate in Xenopus and is required for neural and head induction.  These findings suggest that Notum is a prerequisite for the "default" neural fate and that distinct mechanisms of Wnt inactivation by the Tiki protease in the Organizer and the Notum deacylase in presumptive neuroectoderm orchestrate vertebrate brain development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBxen8RtuF37Vg90H21EOLACvtfcHk0li9xy2BDsZgdA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXksVaisLk%253D&md5=2f5bd637c5b10522452933e9592a888b</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.devcel.2015.02.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.devcel.2015.02.014%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DCheong%26aufirst%3DS.%2BM.%26aulast%3DAmado%26aufirst%3DN.%2BG.%26aulast%3DReis%26aufirst%3DA.%2BH.%26aulast%3DMacDonald%26aufirst%3DB.%2BT.%26aulast%3DZebisch%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DE.%2BY.%26aulast%3DAbreu%26aufirst%3DJ.%2BG.%26aulast%3DHe%26aufirst%3DX.%26atitle%3DNOTUM%2520is%2520required%2520for%2520neural%2520and%2520head%2520induction%2520via%2520Wnt%2520deacylation%252C%2520oxidation%252C%2520and%2520inactivation%26jtitle%3DDev.%2520Cell%26date%3D2015%26volume%3D32%26spage%3D719%26epage%3D730%26doi%3D10.1016%2Fj.devcel.2015.02.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suh, P.-G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryu, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. G.</span></span> <span> </span><span class="NLM_article-title">NOTUM is involved in the progression of colorectal cancer</span>. <i>Cancer Genomics Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">485</span>â <span class="NLM_lpage">497</span>, <span class="refDoi">Â DOI: 10.21873/cgp.20107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.21873%2Fcgp.20107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=30343282" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlWlsr%252FM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=485-497&author=J.+H.+Yoonauthor=D.+Kimauthor=J.+Kimauthor=H.+Leeauthor=J.+Ghimauthor=B.+J.+Kangauthor=P.+Songauthor=P.-G.+Suhauthor=S.+H.+Ryuauthor=T.+G.+Lee&title=NOTUM+is+involved+in+the+progression+of+colorectal+cancer&doi=10.21873%2Fcgp.20107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">NOTUM Is involved in the progression of colorectal cancer</span></div><div class="casAuthors">Yoon, Jong Hyuk; Kim, Dayea; Kim, Jaeyoon; Lee, Hyeongjoo; Ghim, Jaewang; Kang, Byung Jun; Song, Parkyong; Suh, Pann-Ghill; Ryu, Sung Ho; Lee, Taehoon G.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Genomics & Proteomics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">485-497</span>CODEN:
                <span class="NLM_cas:coden">CGPAC7</span>;
        ISSN:<span class="NLM_cas:issn">1790-6245</span>.
    
            (<span class="NLM_cas:orgname">International Institute of Anticancer Research</span>)
        </div><div class="casAbstract">Background: There are limitations to current colorectal cancer (CRC)-specific diagnostic methods and therapies.  Tumorigenesis proceeds because of interaction between cancer cells and various surrounding cells; discovering new mol. mediators through studies of the CRC secretome is a promising approach for the development of CRC diagnostics and therapies.  Materials and Methods: A comparative secretomic anal. was performed using primary and metastatic human isogenic CRC cells.  Proliferation was detd. by MTT and thymidine incorporation assay, migration was detd. by wound-healing assay (ELISA).  The level of palmitoleoyl-protein carboxylesterase (NOTUM) in plasma from patients with CRC was detd. by ELISA.  Results: NOTUM expression was increased in metastatic cells.  Proliferation was suppressed by inhibiting expression of NOTUM.  Knockdown of NOTUM genes inhibited proliferation as well as migration, with possible involvement of p38 and c-JUN N-terminal kinase in this process.  The result was verified in patients with CRC.  Conclusion: NOTUM may be a new candidate for diagnostics and therapy of CRC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUMxJxU9DSy7Vg90H21EOLACvtfcHk0li9xy2BDsZgdA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlWlsr%252FM&md5=9970defbfb6dc7da63d869e2ba28909f</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.21873%2Fcgp.20107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.21873%252Fcgp.20107%26sid%3Dliteratum%253Aachs%26aulast%3DYoon%26aufirst%3DJ.%2BH.%26aulast%3DKim%26aufirst%3DD.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DGhim%26aufirst%3DJ.%26aulast%3DKang%26aufirst%3DB.%2BJ.%26aulast%3DSong%26aufirst%3DP.%26aulast%3DSuh%26aufirst%3DP.-G.%26aulast%3DRyu%26aufirst%3DS.%2BH.%26aulast%3DLee%26aufirst%3DT.%2BG.%26atitle%3DNOTUM%2520is%2520involved%2520in%2520the%2520progression%2520of%2520colorectal%2520cancer%26jtitle%3DCancer%2520Genomics%2520Proteomics%26date%3D2018%26volume%3D15%26spage%3D485%26epage%3D497%26doi%3D10.21873%2Fcgp.20107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moverare-Skrtic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilsson, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henning, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funck-Brentano, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nethander, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivadeneira, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunes, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koskela, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuukkanen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuckermann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perret, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, P. P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerner, U. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohlsson, C.</span></span> <span> </span><span class="NLM_article-title">Osteoblast-derived NOTUM reduces cortical bone mass in mice and the NOTUM locus is associated with bone mineral density in humans</span>. <i>FASEB J.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">11163</span>â <span class="NLM_lpage">11179</span>, <span class="refDoi">Â DOI: 10.1096/fj.201900707R</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1096%2Ffj.201900707R" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=31307226" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVOltLrE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2019&pages=11163-11179&author=S.+Moverare-Skrticauthor=K.+H.+Nilssonauthor=P.+Henningauthor=T.+Funck-Brentanoauthor=M.+Nethanderauthor=F.+Rivadeneiraauthor=G.+C.+Nunesauthor=A.+Koskelaauthor=J.+Tuukkanenauthor=J.+Tuckermannauthor=C.+Perretauthor=P.+P.+C.+Souzaauthor=U.+H.+Lernerauthor=C.+Ohlsson&title=Osteoblast-derived+NOTUM+reduces+cortical+bone+mass+in+mice+and+the+NOTUM+locus+is+associated+with+bone+mineral+density+in+humans&doi=10.1096%2Ffj.201900707R"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Osteoblast-derived NOTUM reduces cortical bone mass in mice and the NOTUM locus is associated with bone mineral density in humans</span></div><div class="casAuthors">Moverare-Skrtic, Sofia; Nilsson, Karin H.; Henning, Petra; Funck-Brentano, Thomas; Nethander, Maria; Rivadeneira, Fernando; Nunes, Glaucia Coletto; Koskela, Antti; Tuukkanen, Juha; Tuckermann, Jan; Perret, Christine; Souza, Pedro Paulo Chaves; Lerner, Ulf H.; Ohlsson, Claes</div><div class="citationInfo"><span class="NLM_cas:title">FASEB Journal</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">11163-11179</span>CODEN:
                <span class="NLM_cas:coden">FAJOEC</span>;
        ISSN:<span class="NLM_cas:issn">0892-6638</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">Osteoporosis is a common skeletal disease, affecting millions of individuals worldwide.  Currently used osteoporosis treatments substantially reduce vertebral fracture risk, whereas nonvertebral fracture risk, mainly caused by reduced cortical bone mass, has only moderately been improved by the osteoporosis drugs used, defining an unmet medical need.  Because several wingless-type MMTV integration site family members (WNTs) and modulators of WNT activity are major regulators of bone mass, we hypothesized that NOTUM, a secreted WNT lipase, might modulate bone mass via an inhibition of WNT activity.  To characterize the possible role of endogenous NOTUM as a physiol. modulator of bone mass, we developed global, cell-specific, and inducible Notum-inactivated mouse models.  Notum expression was high in the cortical bone in mice, and conditional Notum inactivation revealed that osteoblast lineage cells are the principal source of NOTUM in the cortical bone.  Large-scale human genetic analyses identified genetic variants mapping to the NOTUM locus that are strongly assocd. with bone mineral d. (BMD) as estd. with quant. ultrasound in the heel.  Thus, osteoblast-derived NOTUM is an essential local physiol. regulator of cortical bone mass via effects on periosteal bone formation in adult mice, and genetic variants in the NOTUM locus are assocd. with BMD variation in adult humans.  Therapies targeting osteoblast-derived NOTUM may prevent nonvertebral fractures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAEpf9m9-nLLVg90H21EOLACvtfcHk0li9xy2BDsZgdA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVOltLrE&md5=b61892ced94f651095f0b167afb52de0</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1096%2Ffj.201900707R&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffj.201900707R%26sid%3Dliteratum%253Aachs%26aulast%3DMoverare-Skrtic%26aufirst%3DS.%26aulast%3DNilsson%26aufirst%3DK.%2BH.%26aulast%3DHenning%26aufirst%3DP.%26aulast%3DFunck-Brentano%26aufirst%3DT.%26aulast%3DNethander%26aufirst%3DM.%26aulast%3DRivadeneira%26aufirst%3DF.%26aulast%3DNunes%26aufirst%3DG.%2BC.%26aulast%3DKoskela%26aufirst%3DA.%26aulast%3DTuukkanen%26aufirst%3DJ.%26aulast%3DTuckermann%26aufirst%3DJ.%26aulast%3DPerret%26aufirst%3DC.%26aulast%3DSouza%26aufirst%3DP.%2BP.%2BC.%26aulast%3DLerner%26aufirst%3DU.%2BH.%26aulast%3DOhlsson%26aufirst%3DC.%26atitle%3DOsteoblast-derived%2520NOTUM%2520reduces%2520cortical%2520bone%2520mass%2520in%2520mice%2520and%2520the%2520NOTUM%2520locus%2520is%2520associated%2520with%2520bone%2520mineral%2520density%2520in%2520humans%26jtitle%3DFASEB%2520J.%26date%3D2019%26volume%3D33%26spage%3D11163%26epage%3D11179%26doi%3D10.1096%2Ffj.201900707R" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brommage, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mseeh, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potter, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shadoan, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeter-Jones, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bright, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardenhagen, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doree, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moverare-Skrtic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilsson, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henning, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerner, U. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohlsson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sands, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tarver, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zambrowicz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span> <span> </span><span class="NLM_article-title">NOTUM inhibition increases endocortical bone formation and bone strength</span>. <i>Bone Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">2</span>, <span class="refDoi">Â DOI: 10.1038/s41413-018-0038-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1038%2Fs41413-018-0038-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=30622831" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC1MXmvVCmtbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2019&pages=2&author=R.+Brommageauthor=J.+Liuauthor=P.+Vogelauthor=F.+Mseehauthor=A.+Y.+Thompsonauthor=D.+G.+Potterauthor=M.+K.+Shadoanauthor=G.+M.+Hansenauthor=S.+Jeter-Jonesauthor=J.+Cuiauthor=D.+Brightauthor=J.+P.+Bardenhagenauthor=D.+D.+Doreeauthor=S.+Moverare-Skrticauthor=K.+H.+Nilssonauthor=P.+Henningauthor=U.+H.+Lernerauthor=C.+Ohlssonauthor=A.+T.+Sandsauthor=J.+E.+Tarverauthor=D.+R.+Powellauthor=B.+Zambrowiczauthor=Q.+Liu&title=NOTUM+inhibition+increases+endocortical+bone+formation+and+bone+strength&doi=10.1038%2Fs41413-018-0038-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">NOTUM inhibition increases endocortical bone formation and bone strength</span></div><div class="casAuthors">Brommage, Robert; Liu, Jeff; Vogel, Peter; Mseeh, Faika; Thompson, Andrea Y.; Potter, David G.; Shadoan, Melanie K.; Hansen, Gwenn M.; Jeter-Jones, Sabrina; Cui, Jie; Bright, Dawn; Bardenhagen, Jennifer P.; Doree, Deon D.; Moverare-Skrtic, Sofia; Nilsson, Karin H.; Henning, Petra; Lerner, Ulf H.; Ohlsson, Claes; Sands, Arthur T.; Tarver, James E.; Powell, David R.; Zambrowicz, Brian; Liu, Qingyun</div><div class="citationInfo"><span class="NLM_cas:title">Bone Research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2</span>CODEN:
                <span class="NLM_cas:coden">BROEBB</span>;
        ISSN:<span class="NLM_cas:issn">2095-6231</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The disability, mortality and costs caused by non-vertebral osteoporotic fractures are enormous.  Existing osteoporosis therapies are highly effective at reducing vertebral but not non-vertebral fractures.  Cortical bone is a major determinant of non-vertebral bone strength.  To identify novel osteoporosis drug targets, we phenotyped cortical bone of 3 366 viable mouse strains with global knockouts of druggable genes.  Cortical bone thickness was substantially elevated in Notum-/- mice.  NOTUM is a secreted WNT lipase and we obsd. high NOTUM expression in cortical bone and osteoblasts but not osteoclasts.  Three orally active small mols. and a neutralizing antibody inhibiting NOTUM lipase activity were developed.  They increased cortical bone thickness and strength at multiple skeletal sites in both gonadal intact and ovariectomized rodents by stimulating endocortical bone formation.  Thus, inhibition of NOTUM activity is a potential novel anabolic therapy for strengthening cortical bone and preventing non-vertebral fractures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3-CO74hYhmLVg90H21EOLACvtfcHk0liicdR_9riFpA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXmvVCmtbs%253D&md5=d63656138816ed9b86b804f0b5f9369c</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fs41413-018-0038-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41413-018-0038-3%26sid%3Dliteratum%253Aachs%26aulast%3DBrommage%26aufirst%3DR.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DVogel%26aufirst%3DP.%26aulast%3DMseeh%26aufirst%3DF.%26aulast%3DThompson%26aufirst%3DA.%2BY.%26aulast%3DPotter%26aufirst%3DD.%2BG.%26aulast%3DShadoan%26aufirst%3DM.%2BK.%26aulast%3DHansen%26aufirst%3DG.%2BM.%26aulast%3DJeter-Jones%26aufirst%3DS.%26aulast%3DCui%26aufirst%3DJ.%26aulast%3DBright%26aufirst%3DD.%26aulast%3DBardenhagen%26aufirst%3DJ.%2BP.%26aulast%3DDoree%26aufirst%3DD.%2BD.%26aulast%3DMoverare-Skrtic%26aufirst%3DS.%26aulast%3DNilsson%26aufirst%3DK.%2BH.%26aulast%3DHenning%26aufirst%3DP.%26aulast%3DLerner%26aufirst%3DU.%2BH.%26aulast%3DOhlsson%26aufirst%3DC.%26aulast%3DSands%26aufirst%3DA.%2BT.%26aulast%3DTarver%26aufirst%3DJ.%2BE.%26aulast%3DPowell%26aufirst%3DD.%2BR.%26aulast%3DZambrowicz%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DQ.%26atitle%3DNOTUM%2520inhibition%2520increases%2520endocortical%2520bone%2520formation%2520and%2520bone%2520strength%26jtitle%3DBone%2520Res.%26date%3D2019%26volume%3D7%26spage%3D2%26doi%3D10.1038%2Fs41413-018-0038-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jolly, S.</span>; <span class="NLM_string-name">Schuhmacher, L.</span>; <span class="NLM_string-name">Palomer, E.</span>; <span class="NLM_string-name">Frew, S.</span>; <span class="NLM_string-name">Mahy, W.</span>; <span class="NLM_string-name">Monaghan, A.</span>; <span class="NLM_string-name">Jones, E. Y.</span>; <span class="NLM_string-name">Bilsland, J.</span>; <span class="NLM_string-name">Fish, P. V.</span>; <span class="NLM_string-name">Vincent, J. P.</span>; <span class="NLM_string-name">Bictash, M.</span>; <span class="NLM_string-name">Salinas, P.</span>; <span class="NLM_string-name">Whiting, P.</span></span> <i>Notum, a Negative Regulator of Wnt Signalling, as a Therapeutic Target for Alzheimerâs Disease? From Target Validation to the Development of Small Molecule Inhibitors</i>; <span class="NLM_publisher-name">2019 UCL Neuroscience Symposium</span>: <span class="NLM_publisher-loc">London, UK</span>, June 21, <span class="NLM_year">2019</span>. See: <a href="https://www.ucl.ac.uk/research/domains/sites/research_domains/files/abstract_booklet_ucl_neuroscience_symposium_2019.pdf" class="extLink">https://www.ucl.ac.uk/research/domains/sites/research_domains/files/abstract_booklet_ucl_neuroscience_symposium_2019.pdf</a>. (accessed Aug 9, 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&author=S.+Jolly&author=L.+Schuhmacher&author=E.+Palomer&author=S.+Frew&author=W.+Mahy&author=A.+Monaghan&author=E.+Y.+Jones&author=J.+Bilsland&author=P.+V.+Fish&author=J.+P.+Vincent&author=M.+Bictash&author=P.+Salinas&author=P.+Whiting&title=Notum%2C+a+Negative+Regulator+of+Wnt+Signalling%2C+as+a+Therapeutic+Target+for+Alzheimer%E2%80%99s+Disease%3F+From+Target+Validation+to+the+Development+of+Small+Molecule+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DJolly%26aufirst%3DS.%26btitle%3DNotum%252C%2520a%2520Negative%2520Regulator%2520of%2520Wnt%2520Signalling%252C%2520as%2520a%2520Therapeutic%2520Target%2520for%2520Alzheimer%25E2%2580%2599s%2520Disease%253F%2520From%2520Target%2520Validation%2520to%2520the%2520Development%2520of%2520Small%2520Molecule%2520Inhibitors%26pub%3D2019%2520UCL%2520Neuroscience%2520Symposium%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="note"><p class="first last">As assessed by SchrÃ¶dinger SiteMap, which is used to detect and score binding sites. The score is constructed and calibrated so that the average SiteScore for 157 investigated submicromolar sites is 1.0; a score > 1 suggests a site of particular promise for druggability. Application of SiteMap to the palmitoleate binding site of Notum (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4UZQ">4UZQ</a>) generated a SiteScore of 1.2. SchrÃ¶dinger Release 2019â3: SiteMap, SchrÃ¶dinger, LLC, New York, NY, 2019. See:</p></div><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span> <span> </span><span class="NLM_article-title">Identifying and characterizing binding sites and assessing druggability</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">377</span>â <span class="NLM_lpage">389</span>, <span class="refDoi">Â DOI: 10.1021/ci800324m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800324m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BD1MXns1OhsA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=377-389&author=T.+A.+Halgren&title=Identifying+and+characterizing+binding+sites+and+assessing+druggability&doi=10.1021%2Fci800324m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying and Characterizing Binding Sites and Assessing Druggability</span></div><div class="casAuthors">Halgren, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">377-389</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Identification and characterization of binding sites is key in the process of structure-based drug design.  In some cases there may not be any information about the binding site for a target of interest.  In other cases, a putative binding site has been identified by computational or exptl. means, but the druggability of the target is not known.  Even when a site for a given target is known, it may be desirable to find addnl. sites whose targeting could produce a desired biol. response.  A new program, called SiteMap, is presented for identifying and analyzing binding sites and for predicting target druggability.  In a large-scale validation, SiteMap correctly identifies the known binding site as the top-ranked site in 86% of the cases, with best results (>98%) coming for sites that bind ligands with subnanomolar affinity.  In addn., a modified version of the score employed for binding-site identification allows SiteMap to accurately classify the druggability of proteins as measured by their ability to bind passively absorbed small mols. tightly.  In characterizing binding sites, SiteMap provides quant. and graphical information that can help guide efforts to critically assess virtual hits in a lead-discovery application or to modify ligand structure to enhance potency or improve phys. properties in a lead-optimization context.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfAt0-gvcXDLVg90H21EOLACvtfcHk0liicdR_9riFpA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXns1OhsA%253D%253D&md5=f03c7b27e30ae03d10e4127ed1f1ed3c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fci800324m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800324m%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26atitle%3DIdentifying%2520and%2520characterizing%2520binding%2520sites%2520and%2520assessing%2520druggability%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D377%26epage%3D389%26doi%3D10.1021%2Fci800324m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tarver, J. E.,  Jr</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pabba, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbosa, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardyan, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brommage, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, A. G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carson, K. G.</span></span> <span> </span><span class="NLM_article-title">Stimulation of cortical bone formation with thienopyrimidine based inhibitors of NOTUM pectinacetylesterase</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1525</span>â <span class="NLM_lpage">1528</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2016.02.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1016%2Fj.bmcl.2016.02.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=26897593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC28Xislyrsrg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=1525-1528&author=J.+E.+Tarverauthor=P.+K.+Pabbaauthor=J.+Barbosaauthor=Q.+Hanauthor=M.+W.+Gardyanauthor=R.+Brommageauthor=A.+Y.+Thompsonauthor=J.+M.+Schmidtauthor=A.+G.+E.+Wilsonauthor=W.+Heauthor=V.+K.+Lombardoauthor=K.+G.+Carson&title=Stimulation+of+cortical+bone+formation+with+thienopyrimidine+based+inhibitors+of+NOTUM+pectinacetylesterase&doi=10.1016%2Fj.bmcl.2016.02.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Stimulation of cortical bone formation with thienopyrimidine based inhibitors of Notum Pectinacetylesterase</span></div><div class="casAuthors">Tarver, James E., Jr.; Pabba, Praveen K.; Barbosa, Joseph; Han, Qiang; Gardyan, Michael W.; Brommage, Robert; Thompson, Andrea Y.; Schmidt, James M.; Wilson, Alan G. E.; He, Wei; Lombardo, Victoria K.; Carson, Kenneth G.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1525-1528</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A group of small mol. thienopyrimidine inhibitors of Notum Pectinacetylesterase are described.  The authors explored both 2-((5,6-thieno[2,3-d]pyrimidin-4-yl)thio)acetic acids and 2-((6,7-thieno[3,2-d]pyrimidin-4-yl)thio)acetic acids.  In both series, highly potent, orally active Notum Pectinacetylesterase inhibitors were identified.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQqBjxmVwBErVg90H21EOLACvtfcHk0liicdR_9riFpA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xislyrsrg%253D&md5=7040e360736e8b94a687d67fe92a1d22</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.02.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.02.021%26sid%3Dliteratum%253Aachs%26aulast%3DTarver%26aufirst%3DJ.%2BE.%26aulast%3DPabba%26aufirst%3DP.%2BK.%26aulast%3DBarbosa%26aufirst%3DJ.%26aulast%3DHan%26aufirst%3DQ.%26aulast%3DGardyan%26aufirst%3DM.%2BW.%26aulast%3DBrommage%26aufirst%3DR.%26aulast%3DThompson%26aufirst%3DA.%2BY.%26aulast%3DSchmidt%26aufirst%3DJ.%2BM.%26aulast%3DWilson%26aufirst%3DA.%2BG.%2BE.%26aulast%3DHe%26aufirst%3DW.%26aulast%3DLombardo%26aufirst%3DV.%2BK.%26aulast%3DCarson%26aufirst%3DK.%2BG.%26atitle%3DStimulation%2520of%2520cortical%2520bone%2520formation%2520with%2520thienopyrimidine%2520based%2520inhibitors%2520of%2520NOTUM%2520pectinacetylesterase%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D1525%26epage%3D1528%26doi%3D10.1016%2Fj.bmcl.2016.02.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steadman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahy, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sipthorp, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayle, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svensson, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papageorgiou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeganathan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frew, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monaghan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bictash, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span></span> <span> </span><span class="NLM_article-title">Scaffold-hopping identifies furano[2,3-<i>d</i>]pyrimidine amides as potent Notum inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">126751</span>, <span class="refDoi">Â DOI: 10.1016/j.bmcl.2019.126751</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1016%2Fj.bmcl.2019.126751" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=31862412" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisVGjtrrI" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2020&pages=126751&author=B.+N.+Atkinsonauthor=D.+Steadmanauthor=W.+Mahyauthor=Y.+Zhaoauthor=J.+Sipthorpauthor=E.+D.+Bayleauthor=F.+Svenssonauthor=G.+Papageorgiouauthor=F.+Jeganathanauthor=S.+Frewauthor=A.+Monaghanauthor=M.+Bictashauthor=E.+Y.+Jonesauthor=P.+V.+Fish&title=Scaffold-hopping+identifies+furano%5B2%2C3-d%5Dpyrimidine+amides+as+potent+Notum+inhibitors&doi=10.1016%2Fj.bmcl.2019.126751"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Scaffold-hopping identifies furano[2,3-d]pyrimidine amides as potent Notum inhibitors</span></div><div class="casAuthors">Atkinson, Benjamin N.; Steadman, David; Mahy, William; Zhao, Yuguang; Sipthorp, James; Bayle, Elliott D.; Svensson, Fredrik; Papageorgiou, George; Jeganathan, Fiona; Frew, Sarah; Monaghan, Amy; Bictash, Magda; Jones, E. Yvonne; Fish, Paul V.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">126751</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The carboxylesterase Notum is a key neg. regulator of the Wnt signaling pathway by mediating the depalmitoleoylation of Wnt proteins.  Our objective was to discover potent small mol. inhibitors of Notum suitable for exploring the regulation of Wnt signaling in the central nervous system.  Scaffold-hopping from thienopyrimidine acids 1 and 2, supported by X-ray structure detn., identified 3-methylimidazolin-4-one amides 20-24 as potent inhibitors of Notum with activity across three orthogonal assay formats (biochem., extra-cellular, occupancy).  A preferred example 24 demonstrated good stability in mouse microsomes and plasma, and cell permeability in the MDCK-MDR1 assay albeit with modest P-gp mediated efflux.  Pharmacokinetic studies with 24 were performed in vivo in mouse with single oral administration of 24 showing good plasma exposure and reasonable CNS penetration.  We propose that 24 is a new chem. tool suitable for cellular studies to explore the fundamental biol. of Notum.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMbQIEOi9W3bVg90H21EOLACvtfcHk0lj6hrx-fFLFqA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisVGjtrrI&md5=9cbdde4a81de2782dfdd8fa8a1be523f</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.126751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.126751%26sid%3Dliteratum%253Aachs%26aulast%3DAtkinson%26aufirst%3DB.%2BN.%26aulast%3DSteadman%26aufirst%3DD.%26aulast%3DMahy%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DSipthorp%26aufirst%3DJ.%26aulast%3DBayle%26aufirst%3DE.%2BD.%26aulast%3DSvensson%26aufirst%3DF.%26aulast%3DPapageorgiou%26aufirst%3DG.%26aulast%3DJeganathan%26aufirst%3DF.%26aulast%3DFrew%26aufirst%3DS.%26aulast%3DMonaghan%26aufirst%3DA.%26aulast%3DBictash%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DE.%2BY.%26aulast%3DFish%26aufirst%3DP.%2BV.%26atitle%3DScaffold-hopping%2520identifies%2520furano%255B2%252C3-d%255Dpyrimidine%2520amides%2520as%2520potent%2520Notum%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D30%26spage%3D126751%26doi%3D10.1016%2Fj.bmcl.2019.126751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Willis, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayle, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papageorgiou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steadman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahy, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span></span> <span> </span><span class="NLM_article-title">An improved, scalable synthesis of Notum inhibitor LP-922056 using 1-chloro-1,2-benziodoxol-3-one as a superior electrophilic chlorinating agent</span>. <i>Beilstein J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">2790</span>â <span class="NLM_lpage">2797</span>, <span class="refDoi">Â DOI: 10.3762/bjoc.15.271</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.3762%2Fbjoc.15.271" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=31807213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlOqsrjJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=2790-2797&author=N.+J.+Willisauthor=E.+D.+Bayleauthor=G.+Papageorgiouauthor=D.+Steadmanauthor=B.+N.+Atkinsonauthor=W.+Mahyauthor=P.+V.+Fish&title=An+improved%2C+scalable+synthesis+of+Notum+inhibitor+LP-922056+using+1-chloro-1%2C2-benziodoxol-3-one+as+a+superior+electrophilic+chlorinating+agent&doi=10.3762%2Fbjoc.15.271"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">An improved, scalable synthesis of Notum inhibitor LP-922056 using 1-chloro-1,2-benziodoxol-3-one as a superior electrophilic chlorinating agent</span></div><div class="casAuthors">Willis, Nicky J.; Bayle, Elliott D.; Papageorgiou, George; Steadman, David; Atkinson, Benjamin N.; Mahy, William; Fish, Paul V.</div><div class="citationInfo"><span class="NLM_cas:title">Beilstein Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2790-2797</span>CODEN:
                <span class="NLM_cas:coden">BJOCBH</span>;
        ISSN:<span class="NLM_cas:issn">1860-5397</span>.
    
            (<span class="NLM_cas:orgname">Beilstein-Institut zur Foerderung der Chemischen Wissenschaften</span>)
        </div><div class="casAbstract">We are investigating the role of Notum in modulating Wnt signalling in the central nervous system and wished to establish if 1 would serve as a peripherally restricted control.  Key modifications include: (1) the introduction of the C7-cyclopropyl group was most effectively achieved with a Suzuki-Miyaura cross-coupling reaction with MIDA-boronate 11 (5 â 6), and (2) C6 chlorination was performed with 1-chloro-1,2-benziodoxol-3-one (12) (6 â 7) as a mild and selective electrophilic chlorination agent.  Pharmacokinetic studies in mouse showed CNS penetration of 1 is very low with a brain/plasma concn. ratio of just 0.01.  A small library of amides 17 were prepd. from acid 1 to explore if 1 could be modified to deliver a CNS penetrant tool by capping off the acid as an amide.  Although significant Notum inhibition activity could be achieved, none of these amides demonstrated the required combination of metabolic stability along with cell permeability without evidence of P-gp mediated efflux.  Mouse pharmacokinetic studies demonstrate that 1 is unsuitable for use in models of disease where brain penetration is an essential requirement of the compd. but would be an ideal peripherally restricted control.  These data will contribute to the understanding of drug levels of 1 to overlay with appropriate in vivo efficacy endpoints, i.e., the PK-PD relationship.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpe2Pvyx9sImbVg90H21EOLACvtfcHk0lj6hrx-fFLFqA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlOqsrjJ&md5=ddd50870154275e08f397e28301e2e6e</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.3762%2Fbjoc.15.271&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3762%252Fbjoc.15.271%26sid%3Dliteratum%253Aachs%26aulast%3DWillis%26aufirst%3DN.%2BJ.%26aulast%3DBayle%26aufirst%3DE.%2BD.%26aulast%3DPapageorgiou%26aufirst%3DG.%26aulast%3DSteadman%26aufirst%3DD.%26aulast%3DAtkinson%26aufirst%3DB.%2BN.%26aulast%3DMahy%26aufirst%3DW.%26aulast%3DFish%26aufirst%3DP.%2BV.%26atitle%3DAn%2520improved%252C%2520scalable%2520synthesis%2520of%2520Notum%2520inhibitor%2520LP-922056%2520using%25201-chloro-1%252C2-benziodoxol-3-one%2520as%2520a%2520superior%2520electrophilic%2520chlorinating%2520agent%26jtitle%3DBeilstein%2520J.%2520Org.%2520Chem.%26date%3D2019%26volume%3D15%26spage%3D2790%26epage%3D2797%26doi%3D10.3762%2Fbjoc.15.271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Suciu, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cognetta, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potter, Z. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Selective irreversible inhibitors of the Wnt-deacylating enzyme NOTUM developed by activity-based protein profiling</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">563</span>â <span class="NLM_lpage">568</span>, <span class="refDoi">Â DOI: 10.1021/acsmedchemlett.8b00191</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.8b00191" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVSktr3I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=563-568&author=R.+M.+Suciuauthor=A.+B.+Cognettaauthor=Z.+E.+Potterauthor=B.+F.+Cravatt&title=Selective+irreversible+inhibitors+of+the+Wnt-deacylating+enzyme+NOTUM+developed+by+activity-based+protein+profiling&doi=10.1021%2Facsmedchemlett.8b00191"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Irreversible Inhibitors of the Wnt-Deacylating Enzyme NOTUM Developed by Activity-Based Protein Profiling</span></div><div class="casAuthors">Suciu, Radu M.; Cognetta, Armand B.; Potter, Zachary E.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">563-568</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Wnt proteins are secreted morphogens that play crit. roles in embryonic development and tissue remodeling in adult organisms.  Aberrant Wnt signaling contributes to diseases such as cancer.  Wnts are modified by an unusual O-fatty acylation event (O-linked palmitoleoylation of a conserved serine) that is required for binding to Frizzled receptors.  O-Palmitoleoylation of Wnts is introduced by the porcupine (PORCN) acyltransferase and removed by the serine hydrolase NOTUM.  PORCN inhibitors are under development for oncol., while NOTUM inhibitors have potential for treating degenerative diseases.  Here we describe the use of activity-based protein profiling (ABPP) to discover and advance a class of N-hydroxyhydantoin (NHH) carbamates that potently and selectively inhibit NOTUM.  An optimized NHH carbamate inhibitor - ABC99 - preserves Wnt-mediated cell signaling in the presence of NOTUM and was also converted into an ABPP probe for visualizing NOTUM in native biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp28DF3H0VimrVg90H21EOLACvtfcHk0lj6hrx-fFLFqA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVSktr3I&md5=dd39eb16fc4bd557f2e3db10ca82fcef</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.8b00191&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.8b00191%26sid%3Dliteratum%253Aachs%26aulast%3DSuciu%26aufirst%3DR.%2BM.%26aulast%3DCognetta%26aufirst%3DA.%2BB.%26aulast%3DPotter%26aufirst%3DZ.%2BE.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DSelective%2520irreversible%2520inhibitors%2520of%2520the%2520Wnt-deacylating%2520enzyme%2520NOTUM%2520developed%2520by%2520activity-based%2520protein%2520profiling%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D9%26spage%3D563%26epage%3D568%26doi%3D10.1021%2Facsmedchemlett.8b00191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pentinmikko, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iqbal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mana, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andersson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cognetta, A. B.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suciu, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roper, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LuopajÃ¤rvi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markelin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gopalakrishnan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smolander, O.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naranjo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saarinen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juuti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">PietilÃ¤inen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auvinen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">RistimÃ¤ki, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tammela, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacks, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabatini, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yilmaz, O. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katajisto, P.</span></span> <span> </span><span class="NLM_article-title">Notum produced by Paneth cells attenuates regeneration of aged intestinal epithelium</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>571</i></span>,  <span class="NLM_fpage">398</span>â <span class="NLM_lpage">402</span>, <span class="refDoi">Â DOI: 10.1038/s41586-019-1383-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1038%2Fs41586-019-1383-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=31292548" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtl2qsLfE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=571&publication_year=2019&pages=398-402&author=N.+Pentinmikkoauthor=S.+Iqbalauthor=M.+Manaauthor=S.+Anderssonauthor=A.+B.+Cognettaauthor=R.+M.+Suciuauthor=J.+Roperauthor=K.+Luopaj%C3%A4rviauthor=E.+Markelinauthor=S.+Gopalakrishnanauthor=O.-P.+Smolanderauthor=S.+Naranjoauthor=T.+Saarinenauthor=A.+Juutiauthor=K.+Pietil%C3%A4inenauthor=P.+Auvinenauthor=A.+Ristim%C3%A4kiauthor=N.+Guptaauthor=T.+Tammelaauthor=T.+Jacksauthor=D.+M.+Sabatiniauthor=B.+F.+Cravattauthor=O.+H.+Yilmazauthor=P.+Katajisto&title=Notum+produced+by+Paneth+cells+attenuates+regeneration+of+aged+intestinal+epithelium&doi=10.1038%2Fs41586-019-1383-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Notum produced by Paneth cells attenuates regeneration of aged intestinal epithelium</span></div><div class="casAuthors">Pentinmikko, Nalle; Iqbal, Sharif; Mana, Miyeko; Andersson, Simon; Cognetta, Armand B., III; Suciu, Radu M.; Roper, Jatin; Luopajarvi, Kalle; Markelin, Eino; Gopalakrishnan, Swetha; Smolander, Olli-Pekka; Naranjo, Santiago; Saarinen, Tuure; Juuti, Anne; Pietilainen, Kirsi; Auvinen, Petri; Ristimaki, Ari; Gupta, Nitin; Tammela, Tuomas; Jacks, Tyler; Sabatini, David M.; Cravatt, Benjamin F.; Yilmaz, Omer H.; Katajisto, Pekka</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">571</span>
        (<span class="NLM_cas:issue">7765</span>),
    <span class="NLM_cas:pages">398-402</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A decline in stem cell function impairs tissue regeneration during ageing, but the role of the stem-cell-supporting niche in ageing is not well understood.  The small intestine is maintained by actively cycling intestinal stem cells that are regulated by the Paneth cell niche1,2.  Here we show that the regenerative potential of human and mouse intestinal epithelium diminishes with age owing to defects in both stem cells and their niche.  The functional decline was caused by a decrease in stemness-maintaining Wnt signalling due to prodn. of Notum, an extracellular Wnt inhibitor, in aged Paneth cells.  Mechanistically, high activity of mammalian target of rapamycin complex 1 (mTORC1) in aged Paneth cells inhibits activity of peroxisome proliferator activated receptor Î± (PPAR-Î±)3, and lowered PPAR-Î± activity increased Notum expression.  Genetic targeting of Notum or Wnt supplementation restored function of aged intestinal organoids.  Moreover, pharmacol. inhibition of Notum in mice enhanced the regenerative capacity of aged stem cells and promoted recovery from chemotherapy-induced damage.  Our results reveal a role of the stem cell niche in ageing and demonstrate that targeting of Notum can promote regeneration of aged tissues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYdrtPIC9dYbVg90H21EOLACvtfcHk0lisW57Gfc9MgA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtl2qsLfE&md5=9d93cd7bdc33cacf617a025a74aecf48</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fs41586-019-1383-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-019-1383-0%26sid%3Dliteratum%253Aachs%26aulast%3DPentinmikko%26aufirst%3DN.%26aulast%3DIqbal%26aufirst%3DS.%26aulast%3DMana%26aufirst%3DM.%26aulast%3DAndersson%26aufirst%3DS.%26aulast%3DCognetta%26aufirst%3DA.%2BB.%26aulast%3DSuciu%26aufirst%3DR.%2BM.%26aulast%3DRoper%26aufirst%3DJ.%26aulast%3DLuopaj%25C3%25A4rvi%26aufirst%3DK.%26aulast%3DMarkelin%26aufirst%3DE.%26aulast%3DGopalakrishnan%26aufirst%3DS.%26aulast%3DSmolander%26aufirst%3DO.-P.%26aulast%3DNaranjo%26aufirst%3DS.%26aulast%3DSaarinen%26aufirst%3DT.%26aulast%3DJuuti%26aufirst%3DA.%26aulast%3DPietil%25C3%25A4inen%26aufirst%3DK.%26aulast%3DAuvinen%26aufirst%3DP.%26aulast%3DRistim%25C3%25A4ki%26aufirst%3DA.%26aulast%3DGupta%26aufirst%3DN.%26aulast%3DTammela%26aufirst%3DT.%26aulast%3DJacks%26aufirst%3DT.%26aulast%3DSabatini%26aufirst%3DD.%2BM.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26aulast%3DYilmaz%26aufirst%3DO.%2BH.%26aulast%3DKatajisto%26aufirst%3DP.%26atitle%3DNotum%2520produced%2520by%2520Paneth%2520cells%2520attenuates%2520regeneration%2520of%2520aged%2520intestinal%2520epithelium%26jtitle%3DNature%26date%3D2019%26volume%3D571%26spage%3D398%26epage%3D402%26doi%3D10.1038%2Fs41586-019-1383-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steadman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sipthorp, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vecchia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruza, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeganathan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lines, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frew, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monaghan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kjaer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bictash, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span></span> <span> </span><span class="NLM_article-title">Discovery of 2-phenoxyacetamides as inhibitors of the Wnt-depalmitoleating enzyme NOTUM from an X-ray fragment screen</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1361</span>â <span class="NLM_lpage">1369</span>, <span class="refDoi">Â DOI: 10.1039/C9MD00096H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1039%2FC9MD00096H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=31534655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosVCgsb4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1361-1369&author=B.+N.+Atkinsonauthor=D.+Steadmanauthor=Y.+Zhaoauthor=J.+Sipthorpauthor=L.+Vecchiaauthor=R.+R.+Ruzaauthor=F.+Jeganathanauthor=G.+Linesauthor=S.+Frewauthor=A.+Monaghanauthor=S.+Kjaerauthor=M.+Bictashauthor=E.+Y.+Jonesauthor=P.+V.+Fish&title=Discovery+of+2-phenoxyacetamides+as+inhibitors+of+the+Wnt-depalmitoleating+enzyme+NOTUM+from+an+X-ray+fragment+screen&doi=10.1039%2FC9MD00096H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 2-phenoxyacetamides as inhibitors of the Wnt-depalmitoleating enzyme NOTUM from an X-ray fragment screen</span></div><div class="casAuthors">Atkinson, Benjamin N.; Steadman, David; Zhao, Yuguang; Sipthorp, James; Vecchia, Luca; Ruza, Reinis R.; Jeganathan, Fiona; Lines, Georgie; Frew, Sarah; Monaghan, Amy; Kjaer, Svend; Bictash, Magda; Jones, E. Yvonne; Fish, Paul V.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1361-1369</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">NOTUM is a carboxylesterase that has been shown to act by mediating the O-depalmitoleoylation of Wnt proteins resulting in suppression of Wnt signaling.  Here, we describe the development of NOTUM inhibitors that restore Wnt signaling for use in in vitro disease models where NOTUM over activity is an underlying cause.  A crystallog. fragment screen with NOTUM identified 2-phenoxyacetamide 3 as binding in the palmitoleate pocket with modest inhibition activity (IC50 33 Î¼M).  Optimization of hit 3 by SAR studies guided by SBDD identified indazole 38 (IC50 0.032 Î¼M) and isoquinoline 45 (IC50 0.085 Î¼M) as potent inhibitors of NOTUM.  The binding of 45 to NOTUM was rationalized through an X-ray co-crystal structure detn. which showed a flipped binding orientation compared to 3.  However, it was not possible to combine NOTUM inhibition activity with metabolic stability as the majority of the compds. tested were rapidly metabolized in an NADPH-independent manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOeg6yfRBz57Vg90H21EOLACvtfcHk0lisW57Gfc9MgA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosVCgsb4%253D&md5=041061f0da5d9ff778d26adc5a14ab0e</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1039%2FC9MD00096H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9MD00096H%26sid%3Dliteratum%253Aachs%26aulast%3DAtkinson%26aufirst%3DB.%2BN.%26aulast%3DSteadman%26aufirst%3DD.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DSipthorp%26aufirst%3DJ.%26aulast%3DVecchia%26aufirst%3DL.%26aulast%3DRuza%26aufirst%3DR.%2BR.%26aulast%3DJeganathan%26aufirst%3DF.%26aulast%3DLines%26aufirst%3DG.%26aulast%3DFrew%26aufirst%3DS.%26aulast%3DMonaghan%26aufirst%3DA.%26aulast%3DKjaer%26aufirst%3DS.%26aulast%3DBictash%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DE.%2BY.%26aulast%3DFish%26aufirst%3DP.%2BV.%26atitle%3DDiscovery%2520of%25202-phenoxyacetamides%2520as%2520inhibitors%2520of%2520the%2520Wnt-depalmitoleating%2520enzyme%2520NOTUM%2520from%2520an%2520X-ray%2520fragment%2520screen%26jtitle%3DMedChemComm%26date%3D2019%26volume%3D10%26spage%3D1361%26epage%3D1369%26doi%3D10.1039%2FC9MD00096H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hillier, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. Y.</span></span> <span> </span><span class="NLM_article-title">Structural characterisation of melatonin as an inhibitor of the Wnt deacylase Notum</span>. <i>J. Pineal Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>68</i></span>, <span class="NLM_elocation-id">e12630</span> <span class="refDoi">Â DOI: 10.1111/jpi.12630</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1111%2Fjpi.12630" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=31876313" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1SitL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2020&author=Y.+Zhaoauthor=J.+Renauthor=J.+Hillierauthor=M.+Jonesauthor=W.+Luauthor=E.+Y.+Jones&title=Structural+characterisation+of+melatonin+as+an+inhibitor+of+the+Wnt+deacylase+Notum&doi=10.1111%2Fjpi.12630"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Structural characterization of melatonin as an inhibitor of the Wnt deacylase Notum</span></div><div class="casAuthors">Zhao, Yuguang; Ren, Jingshan; Hillier, James; Jones, Margaret; Lu, Weixian; Jones, Edith Yvonne</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pineal Research</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e12630</span>CODEN:
                <span class="NLM_cas:coden">JPRSE9</span>;
        ISSN:<span class="NLM_cas:issn">0742-3098</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The hormone melatonin, secreted from the pineal gland, mediates multiple physiol. effects including modulation of Wnt/Î²-catenin signalling.  The Wnt palmitoleate lipid modification is essential for its signalling activity, while the carboxylesterase Notum can remove the lipid from Wnt and inactivate it.  Notum enzyme inhibition can therefore upregulate Wnt signalling.  While searching for Notum inhibitors by crystallog. fragment screening, a hit compd. N-[2-(5-fluoro-1H-indol-3-yl)ethyl]acetamide that is structurally similar to melatonin came to our attention.  We then soaked melatonin and its precursor N-acetylserotonin into Notum crystals and obtained high-resoln. structures (=1.5 Ã) of their complexes.  In each of the structures, two compd. mols. bind with Notum: one at the enzyme's catalytic pocket, overlapping the space occupied by the acyl tail of the Wnt palmitoleate lipid, and the other at the edge of the pocket opposite the substrate entrance.  Although the inhibitory activity of melatonin shown by in vitro enzyme assays is low (IC50 75Î¼mol/L), the structural information reported here provides a basis for the design of potent and brain accessible drugs for neurodegenerative diseases such as Alzheimer's disease, in which upregulation of Wnt signalling may be beneficial.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq41bwOb_70PLVg90H21EOLACvtfcHk0lhlbXQFtJHHeQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1SitL8%253D&md5=4b1ebc6562fd38de68bfaffa2cfd8f1f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1111%2Fjpi.12630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fjpi.12630%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DHillier%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DM.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DJones%26aufirst%3DE.%2BY.%26atitle%3DStructural%2520characterisation%2520of%2520melatonin%2520as%2520an%2520inhibitor%2520of%2520the%2520Wnt%2520deacylase%2520Notum%26jtitle%3DJ.%2520Pineal%2520Res.%26date%3D2020%26volume%3D68%26doi%3D10.1111%2Fjpi.12630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span>See: <a href="https://enamine.net/fragments/fragment-collection/carboxylic-acid-fragments" class="extLink">https://enamine.net/fragments/fragment-collection/carboxylic-acid-fragments</a>. Compounds were purchased from this collection on Dec 15, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=See%3A+https%3A%2F%2Fenamine.net%2Ffragments%2Ffragment-collection%2Fcarboxylic-acid-fragments.+Compounds+were+purchased+from+this+collection+on+Dec+15%2C+2017."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Springthorpe, B.</span></span> <span> </span><span class="NLM_article-title">The influence of drug-like concepts on decision-making in medicinal chemistry</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">881</span>â <span class="NLM_lpage">890</span>, <span class="refDoi">Â DOI: 10.1038/nrd2445</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1038%2Fnrd2445" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=17971784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1els7rL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=881-890&author=P.+D.+Leesonauthor=B.+Springthorpe&title=The+influence+of+drug-like+concepts+on+decision-making+in+medicinal+chemistry&doi=10.1038%2Fnrd2445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The influence of drug-like concepts on decision-making in medicinal chemistry</span></div><div class="casAuthors">Leeson, Paul D.; Springthorpe, Brian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">881-890</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Despite the wide acceptance of drug-like principles such as the 'rule of five', this anal. of mols. currently being synthesized in leading pharmaceutical companies reveals that their phys. properties differ significantly from those of recently discovered oral drugs.  The marked increase in lipophilicity in particular could increase the likelihood of attrition in drug development.  The application of guidelines linked to the concept of drug-likeness, such as the 'rule of five', has gained wide acceptance as an approach to reduce attrition in drug discovery and development.  However, despite this acceptance, anal. of recent trends reveals that the phys. properties of mols. that are currently being synthesized in leading drug discovery companies differ significantly from those of recently discovered oral drugs and compds. in clin. development.  The consequences of the marked increase in lipophilicity - the most important drug-like phys. property - include a greater likelihood of lack of selectivity and attrition in drug development.  Tackling the threat of compd.-related toxicol. attrition needs to move to the mainstream of medicinal chem. decision-making.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTUeonLD6iebVg90H21EOLACvtfcHk0lhlbXQFtJHHeQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1els7rL&md5=5665a9e2dc266a1ee096e20625757ef7</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnrd2445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2445%26sid%3Dliteratum%253Aachs%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DSpringthorpe%26aufirst%3DB.%26atitle%3DThe%2520influence%2520of%2520drug-like%2520concepts%2520on%2520decision-making%2520in%2520medicinal%2520chemistry%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D6%26spage%3D881%26epage%3D890%26doi%3D10.1038%2Fnrd2445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallego, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, M. P.</span></span> <span> </span><span class="NLM_article-title">Lipophilic efficiency as an important metric in drug design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6401</span>â <span class="NLM_lpage">6420</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.8b00077</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00077" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC1cXmsVahsbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6401-6420&author=T.+W.+Johnsonauthor=R.+A.+Gallegoauthor=M.+P.+Edwards&title=Lipophilic+efficiency+as+an+important+metric+in+drug+design&doi=10.1021%2Facs.jmedchem.8b00077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Lipophilic Efficiency as an Important Metric in Drug Design</span></div><div class="casAuthors">Johnson, Ted W.; Gallego, Rebecca A.; Edwards, Martin P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6401-6420</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Lipophilic efficiency (LipE) is an important metric that has been increasingly applied in drug discovery medicinal chem. lead optimization programs.  In this perspective, using literature drug discovery examples, we discuss the concept of rigorously applying LipE to guide medicinal chem. lead optimization toward drug candidates with potential for superior in vivo efficacy and safety, esp. when guided by physiochem. property-based optimization (PPBO).  Also highlighted are examples of small structural modifications such as addn. of single atoms, small functional groups, and cyclizations that produce large increases in LipE.  Understanding the factors that may contribute to LipE changes through anal. of ligand-protein crystal structures and using structure-based drug design (SBDD) to increase LipE by design is also discussed.  Herein we advocate for use of LipE anal. coupled with PPBO and SBDD as an efficient mechanism for drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXQ8DOcOj6GrVg90H21EOLACvtfcHk0lidO4x8eFO12w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXmsVahsbg%253D&md5=29f38880914fa75c33be3a640ace4309</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00077%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DT.%2BW.%26aulast%3DGallego%26aufirst%3DR.%2BA.%26aulast%3DEdwards%26aufirst%3DM.%2BP.%26atitle%3DLipophilic%2520efficiency%2520as%2520an%2520important%2520metric%2520in%2520drug%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6401%26epage%3D6420%26doi%3D10.1021%2Facs.jmedchem.8b00077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span> clog<i>P</i> values were calculated using ChemDraw version
16.0.1.4 (77) software,
and chromatographic hydrophobicity index (CHI) log<i>D</i><sub>7.4</sub> values (<i>n</i> = 2) were measured by the
following method:<span class="NLM_contrib-group"><span class="NLM_string-name">ValkÃ³, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bevan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, D.</span></span> <span> </span><span class="NLM_article-title">Chromatographic hydrophobicity index by Fast-Gradient RP-HPLC: a high-throughput alternative to log P/log D</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">2022</span>â <span class="NLM_lpage">2029</span>, <span class="refDoi">Â DOI: 10.1021/ac961242d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac961242d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADyaK2sXivVertLs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=1997&pages=2022-2029&author=K.+Valk%C3%B3author=C.+Bevanauthor=D.+Reynolds&title=Chromatographic+hydrophobicity+index+by+Fast-Gradient+RP-HPLC%3A+a+high-throughput+alternative+to+log+P%2Flog+D&doi=10.1021%2Fac961242d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatographic Hydrophobicity Index by Fast-Gradient RP-HPLC: A High-Throughput Alternative to Log P/log D</span></div><div class="casAuthors">Valko, Klara; Bevan, Chris; Reynolds, Derek</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2022-2029</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A new chromatog. hydrophobicity index (CHI) is described which can be used as part of a protocol for high-throughput (50-100 compds./day) physicochem. property profiling for rational drug design.  The index is derived from retention times (tR) obsd. in a fast gradient reversed-phase HPLC method.  The isocratic retention factors (log k') were measured for a series of 76 structurally unrelated compds. by using various concns. of acetonitrile in the mobile phase.  By plotting the log k' as a function of the acetonitrile concn., the slope (S) and the intercept (log k'w) values were calcd.  The previously validated index of hydrophobicity Ï0 was calcd. as -log k'w/S.  A good linear correlation was obtained between the gradient retention time values, tR and the isocratically detd. Ï0 values for the 76 compds.  The consts. of this linear correlation can be used to calc. CHI.  For most compds., CHI is between 0 and 100 and in this range it approximates to the percentage (by vol.) of acetonitrile required to achieve an equal distribution of compd. between the mobile and the stationary phases.  CHI values can be measured using acidic, neutral, or slightly basic eluents.  Values corresponding to the neutral form of mols. could be measured for 52 of the compds. and showed good correlation (r = 0.851) to the calcd. octanol/water partition coeff. (c log P) values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozvQI1B1cnpbVg90H21EOLACvtfcHk0lidO4x8eFO12w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXivVertLs%253D&md5=fcbb641105c12dc490d4a20db8fc5032</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fac961242d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac961242d%26sid%3Dliteratum%253Aachs%26aulast%3DValk%25C3%25B3%26aufirst%3DK.%26aulast%3DBevan%26aufirst%3DC.%26aulast%3DReynolds%26aufirst%3DD.%26atitle%3DChromatographic%2520hydrophobicity%2520index%2520by%2520Fast-Gradient%2520RP-HPLC%253A%2520a%2520high-throughput%2520alternative%2520to%2520log%2520P%252Flog%2520D%26jtitle%3DAnal.%2520Chem.%26date%3D1997%26volume%3D69%26spage%3D2022%26epage%3D2029%26doi%3D10.1021%2Fac961242d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Forero-CortÃ©s, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haydl, A. M.</span></span> <span> </span><span class="NLM_article-title">The 25th anniversary of the BuchwaldâHartwig amination: development, applications, and outlook</span>. <i>Org. Process Res. Dev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1478</span>â <span class="NLM_lpage">1483</span>, <span class="refDoi">Â DOI: 10.1021/acs.oprd.9b00161</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.oprd.9b00161" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1yhs77J" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2019&pages=1478-1483&author=P.+A.+Forero-Cort%C3%A9sauthor=A.+M.+Haydl&title=The+25th+anniversary+of+the+Buchwald%E2%80%93Hartwig+amination%3A+development%2C+applications%2C+and+outlook&doi=10.1021%2Facs.oprd.9b00161"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">The 25th Anniversary of the Buchwald-Hartwig Amination: Development, Applications, and Outlook</span></div><div class="casAuthors">Forero-Cortes, Paola A.; Haydl, Alexander M.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Process Research & Development</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1478-1483</span>CODEN:
                <span class="NLM_cas:coden">OPRDFK</span>;
        ISSN:<span class="NLM_cas:issn">1083-6160</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The palladium-catalyzed cross-coupling of amines and aryl (pseudo)halides, now commonly referred to as the Buchwald-Hartwig amination was first reported 25 years ago.  Since the simultaneous breakthrough reports of Buchwald and Hartwig dating 1995, this reaction has transformed the way synthetic chemists think about synthesizing arom. amines.  In this highlight article, a short showcasing discussion about the genesis of this reaction is provided, as well as selected examples showing the impact of this transformation in synthetic chem., both in academic and industrial settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDqjUPVJ2XvrVg90H21EOLACvtfcHk0lidO4x8eFO12w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1yhs77J&md5=11ee132c194b07792110102e2ed63bf7</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Facs.oprd.9b00161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.oprd.9b00161%26sid%3Dliteratum%253Aachs%26aulast%3DForero-Cort%25C3%25A9s%26aufirst%3DP.%2BA.%26aulast%3DHaydl%26aufirst%3DA.%2BM.%26atitle%3DThe%252025th%2520anniversary%2520of%2520the%2520Buchwald%25E2%2580%2593Hartwig%2520amination%253A%2520development%252C%2520applications%252C%2520and%2520outlook%26jtitle%3DOrg.%2520Process%2520Res.%2520Dev.%26date%3D2019%26volume%3D23%26spage%3D1478%26epage%3D1483%26doi%3D10.1021%2Facs.oprd.9b00161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="note"><p class="first last">Confirmation of stereochemical assignment and purity of enantiomers <b>26</b> and <b>27</b> was determined by chiral SFC supported by X-ray structure determination. See <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> for details.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="note"><p class="first last">For recent reviews, see:</p></div><div class="NLM_citation" id="rightTab-cit28a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akhtar, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alam, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akhter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaquiquzzaman, M.</span></span> <span> </span><span class="NLM_article-title">A review exploring therapeutic worth of 1,3,4-oxadiazole tailored compounds</span>. <i>Mini-Rev. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">477</span>â <span class="NLM_lpage">509</span>, <span class="refDoi">Â DOI: 10.2174/1389557518666181015152433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.2174%2F1389557518666181015152433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=30324877" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC1MXltVSlu74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2019&pages=477-509&author=G.+Vermaauthor=M.+F.+Khanauthor=W.+Akhtarauthor=M.+M.+Alamauthor=M.+Akhterauthor=M.+Shaquiquzzaman&title=A+review+exploring+therapeutic+worth+of+1%2C3%2C4-oxadiazole+tailored+compounds&doi=10.2174%2F1389557518666181015152433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28aR"><div class="casContent"><span class="casTitleNuber">28a</span><div class="casTitle"><span class="NLM_cas:atitle">A Review Exploring Therapeutic Worth of 1,3,4-Oxadiazole Tailored Compounds</span></div><div class="casAuthors">Verma, Garima; Khan, Mohemmed F.; Akhtar, Wasim; Alam, Mohammad Mumtaz; Akhter, Mymoona; Shaquiquzzaman, Mohammad</div><div class="citationInfo"><span class="NLM_cas:title">Mini-Reviews in Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">477-509</span>CODEN:
                <span class="NLM_cas:coden">MMCIAE</span>;
        ISSN:<span class="NLM_cas:issn">1389-5575</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  1,3,4-Oxadiazole, a five-membered arom. ring can be seen in a no. of synthetic mols.  The peculiar structural feature of 1,3,4-oxadiazole ring with pyridine type of nitrogen atom is beneficial for 1,3,4-oxadiazole derivs. to have effective binding with different enzymes and receptors in biol. systems through numerous weak interactions, thereby eliciting an array of bioactivities.  Research in the area of development of 1,3,4-oxadiazole-based derivs. has become an interesting topic for the scientists.  A no. of 1,3,4-oxadiazole based compds. with high therapeutic potency are being extensively used for the treatment of different ailments, contributing to enormous development value.  This work provides a systematic and comprehensive review highlighting current developments of 1,3,4-oxadiazole based compds. in the entire range of medicinal chem. such as anticancer, antifungal, antibacterial, antitubercular, anti-inflammatory, antineuropathic, antihypertensive, antihistaminic, antiparasitic, antiobesity, antiviral, and other medicinal agents.  It is believed that this review will be of great help for new thoughts in the pursuit for rational designs for the development of more active and less toxic 1,3,4-oxadiazole based medicinal agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDw_H8wFO9QbVg90H21EOLACvtfcHk0liuQn43jBp3vw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXltVSlu74%253D&md5=0315e4b67aaf2e7a95e4052e13058511</span></div><a href="/servlet/linkout?suffix=cit28a&amp;dbid=16384&amp;doi=10.2174%2F1389557518666181015152433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1389557518666181015152433%26sid%3Dliteratum%253Aachs%26aulast%3DVerma%26aufirst%3DG.%26aulast%3DKhan%26aufirst%3DM.%2BF.%26aulast%3DAkhtar%26aufirst%3DW.%26aulast%3DAlam%26aufirst%3DM.%2BM.%26aulast%3DAkhter%26aufirst%3DM.%26aulast%3DShaquiquzzaman%26aufirst%3DM.%26atitle%3DA%2520review%2520exploring%2520therapeutic%2520worth%2520of%25201%252C3%252C4-oxadiazole%2520tailored%2520compounds%26jtitle%3DMini-Rev.%2520Med.%2520Chem.%26date%3D2019%26volume%3D19%26spage%3D477%26epage%3D509%26doi%3D10.2174%2F1389557518666181015152433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit28b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Makawana, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H. L.</span></span> <span> </span><span class="NLM_article-title">1,3,4-oxadiazole derivatives as potential biological agents</span>. <i>Mini-Rev. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1725</span>â <span class="NLM_lpage">1743</span>, <span class="refDoi">Â DOI: 10.2174/13895575113139990071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.2174%2F13895575113139990071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=23815585" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFyrsbbF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=1725-1743&author=J.+Sunauthor=J.+A.+Makawanaauthor=H.+L.+Zhu&title=1%2C3%2C4-oxadiazole+derivatives+as+potential+biological+agents&doi=10.2174%2F13895575113139990071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28bR"><div class="casContent"><span class="casTitleNuber">28b</span><div class="casTitle"><span class="NLM_cas:atitle">1,3,4-Oxadiazole Derivatives as Potential Biological Agents</span></div><div class="casAuthors">Sun, Juan; Makawana, Jigar A.; Zhu, Hai-Liang</div><div class="citationInfo"><span class="NLM_cas:title">Mini-Reviews in Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1725-1743</span>CODEN:
                <span class="NLM_cas:coden">MMCIAE</span>;
        ISSN:<span class="NLM_cas:issn">1389-5575</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The synthesis of novel compd. libraries along with screening is a rapid and effective approach for the discovery of potential chem. agents, and it becomes an important method in pharmaceutical chem. research. 1,3,4-oxadiazole derivs. as the typical heterocyclic compds., exhibit a broad spectrum of biol. activities and vital leading compds. for the development of chem. drugs.  Herein, we focus on the synthesis and screening of novel 1,3,4-oxadiazoles derivs. with antimicrobial, antitumor or antiviral activities during the past decade.  In this review, we discussed the synthetic development of 1,3,4-oxadiazoles derivs., and also the relevant bioactivity and their prospects as the potential chem. drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGwV3yfnROebVg90H21EOLACvtfcHk0liuQn43jBp3vw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFyrsbbF&md5=0bf041b3cc9c11cde0d27523a8b4aafa</span></div><a href="/servlet/linkout?suffix=cit28b&amp;dbid=16384&amp;doi=10.2174%2F13895575113139990071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F13895575113139990071%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DMakawana%26aufirst%3DJ.%2BA.%26aulast%3DZhu%26aufirst%3DH.%2BL.%26atitle%3D1%252C3%252C4-oxadiazole%2520derivatives%2520as%2520potential%2520biological%2520agents%26jtitle%3DMini-Rev.%2520Med.%2520Chem.%26date%3D2013%26volume%3D13%26spage%3D1725%26epage%3D1743%26doi%3D10.2174%2F13895575113139990071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span>1,3,4-Oxadiazole-2(3<i>H</i>)-one predicted p<i>K</i><sub>a</sub>: 6.3 Â±
0.7 (most
acidic), calculated using ACD/Labs software v11.02.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=1%2C3%2C4-Oxadiazole-2%283H%29-one+predicted+pKa%3A+6.3+%C2%B1%0A0.7+%28most%0Aacidic%29%2C+calculated+using+ACD%2FLabs+software+v11.02."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="note"><p class="first last">Acidic p<i>K</i><sub>a</sub> values for CNS drugs rarely fall below 6.0; see:</p></div><div class="NLM_citation" id="rightTab-cit30a"><span><span class="NLM_label">(a) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Manallack, D. T.</span></span> <span> </span><span class="NLM_article-title">The p<i>K</i><sub>a</sub> Distribution of drugs: application to drug discovery</span>. <i>Perspect. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">38</span>, <span class="refDoi">Â DOI: 10.1177/1177391X0700100003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1177%2F1177391X0700100003" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2007&pages=38&author=D.+T.+Manallack&title=The+pKa+Distribution+of+drugs%3A+application+to+drug+discovery&doi=10.1177%2F1177391X0700100003"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30a&amp;dbid=16384&amp;doi=10.1177%2F1177391X0700100003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1177391X0700100003%26sid%3Dliteratum%253Aachs%26aulast%3DManallack%26aufirst%3DD.%2BT.%26atitle%3DThe%2520pKa%2520Distribution%2520of%2520drugs%253A%2520application%2520to%2520drug%2520discovery%26jtitle%3DPerspect.%2520Med.%2520Chem.%26date%3D2007%26volume%3D1%26spage%3D38%26doi%3D10.1177%2F1177391X0700100003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div>. </div><div class="NLM_citation" id="rightTab-cit30b"><span><span class="NLM_label">(b) </span> <span class="NLM_contrib-group"><span class="NLM_string-name">Manallack, D. T.</span></span> <span> </span><span class="NLM_article-title">The acid-base profile of a contemporary set of drugs: implications for drug discovery</span>. <i>SAR QSAR Environ. Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">611</span>â <span class="NLM_lpage">655</span>, <span class="refDoi">Â DOI: 10.1080/10629360903438313</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1080%2F10629360903438313" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=20024802" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFyitbrO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2009&pages=611-655&author=D.+T.+Manallack&title=The+acid-base+profile+of+a+contemporary+set+of+drugs%3A+implications+for+drug+discovery&doi=10.1080%2F10629360903438313"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30bR"><div class="casContent"><span class="casTitleNuber">30b</span><div class="casTitle"><span class="NLM_cas:atitle">The acid-base profile of a contemporary set of drugs: implications for drug discovery</span></div><div class="casAuthors">Manallack, D. T.</div><div class="citationInfo"><span class="NLM_cas:title">SAR and QSAR in Environmental Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">7-8</span>),
    <span class="NLM_cas:pages">611-655</span>CODEN:
                <span class="NLM_cas:coden">SQERED</span>;
        ISSN:<span class="NLM_cas:issn">1062-936X</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Acid-base ionization const. (pKa) values have considerable influence on the physicochem. and pharmacokinetic properties of therapeutic substances.  A set of 907 drugs was examd. to det. the proportion of drugs that contain an ionizable group and the distribution of their pKa values.  Using this contemporary set of compds. it was found that 64% of these compds. contained an ionizable group.  Within this group of ionizable compds., 34% contained a single basic group while only 20% contained a single acidic functional group.  The single acid and single base contg. substances were investigated further to examine the distributions of their pKa values.  These data are discussed and analyzed with a focus on the entire set as well as central nervous system, non-central nervous system and oral drugs.  The findings from this research will prompt pharmaceutical companies to assess the constitution of their screening libraries, such that focus is placed on the proportion of ionizable substances, the ratio of acids to bases and the distribution of pKa values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJXs5vGE4IUrVg90H21EOLACvtfcHk0liuQn43jBp3vw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFyitbrO&md5=05a7b0dee5ab8200340ac9a769f0b9a2</span></div><a href="/servlet/linkout?suffix=cit30b&amp;dbid=16384&amp;doi=10.1080%2F10629360903438313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F10629360903438313%26sid%3Dliteratum%253Aachs%26aulast%3DManallack%26aufirst%3DD.%2BT.%26atitle%3DThe%2520acid-base%2520profile%2520of%2520a%2520contemporary%2520set%2520of%2520drugs%253A%2520implications%2520for%2520drug%2520discovery%26jtitle%3DSAR%2520QSAR%2520Environ.%2520Res.%26date%3D2009%26volume%3D20%26spage%3D611%26epage%3D655%26doi%3D10.1080%2F10629360903438313" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span> There is a paucity
of acids with p<i>K</i><sub>a</sub> values between 6 and
7, probably due to the dominance of carboxylic acids and phenols,
and the p<i>K</i><sub>a</sub> values usually encountered
for these groups; see:<span class="NLM_contrib-group"><span class="NLM_string-name">Manallack, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prankerd, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuriev, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oprea, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chalmers, D. K.</span></span> <span> </span><span class="NLM_article-title">The significance of acid/base properties in drug discovery</span>. <i>Chem. Soc. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">485</span>â <span class="NLM_lpage">496</span>, <span class="refDoi">Â DOI: 10.1039/C2CS35348B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1039%2FC2CS35348B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=23099561" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVKmtb7N" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2013&pages=485-496&author=D.+T.+Manallackauthor=R.+J.+Prankerdauthor=E.+Yurievauthor=T.+I.+Opreaauthor=D.+K.+Chalmers&title=The+significance+of+acid%2Fbase+properties+in+drug+discovery&doi=10.1039%2FC2CS35348B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">The significance of acid/base properties in drug discovery</span></div><div class="casAuthors">Manallack, David T.; Prankerd, Richard J.; Yuriev, Elizabeth; Oprea, Tudor I.; Chalmers, David K.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">485-496</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  While drug discovery scientists take heed of various guidelines concerning drug-like character, the influence of acid/base properties often remains under-scrutinized.  Ionisation consts. (pKa values) are fundamental to the variability of the biopharmaceutical characteristics of drugs and to underlying parameters such as logD and soly. pKa values affect physicochem. properties such as aq. soly., which in turn influences drug formulation approaches.  More importantly, absorption, distribution, metab., excretion and toxicity (ADMET) are profoundly affected by the charge state of compds. under varying pH conditions.  Consideration of pKa values in conjunction with other mol. properties is of great significance and has the potential to be used to further improve the efficiency of drug discovery.  Given the recent low annual output of new drugs from pharmaceutical companies, this review will provide a timely reminder of an important mol. property that influences clin. success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKAwdQSoeZxLVg90H21EOLACvtfcHk0liuQn43jBp3vw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVKmtb7N&md5=48d72c5bdae09cc3969cffaea36cf30f</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1039%2FC2CS35348B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC2CS35348B%26sid%3Dliteratum%253Aachs%26aulast%3DManallack%26aufirst%3DD.%2BT.%26aulast%3DPrankerd%26aufirst%3DR.%2BJ.%26aulast%3DYuriev%26aufirst%3DE.%26aulast%3DOprea%26aufirst%3DT.%2BI.%26aulast%3DChalmers%26aufirst%3DD.%2BK.%26atitle%3DThe%2520significance%2520of%2520acid%252Fbase%2520properties%2520in%2520drug%2520discovery%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2013%26volume%3D42%26spage%3D485%26epage%3D496%26doi%3D10.1039%2FC2CS35348B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mahy, W.</span>; <span class="NLM_string-name">Willis, N. J.</span>; <span class="NLM_string-name">Zhao, Y.</span>; <span class="NLM_string-name">Woodward, H.</span>; <span class="NLM_string-name">Svensson, F.</span>; <span class="NLM_string-name">Sipthorp, J.</span>; <span class="NLM_string-name">Vecchia, L.</span>; <span class="NLM_string-name">Ruza, R. R.</span>; <span class="NLM_string-name">Kjaer, S.</span>; <span class="NLM_string-name">Frew, S.</span>; <span class="NLM_string-name">Monaghan, A.</span>; <span class="NLM_string-name">Bictash, M.</span>; <span class="NLM_string-name">Salinas, P.</span>; <span class="NLM_string-name">Whiting, P.</span>; <span class="NLM_string-name">Vincent, J.-P.</span>; <span class="NLM_string-name">Jones, E. Y.</span>; <span class="NLM_string-name">Fish, P. V.</span></span> <span> </span><span class="NLM_article-title">5-Phenyl-1,3,4-oxadiazole-2(3<i>H</i>)-ones are Potent Inhibitors of Notum Carboxylesterase Activity Identified by the Optimization of a Crystallographic Fragment Screening Hit</span>.  <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_fpage">01391</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&pages=01391&author=W.+Mahy&author=N.+J.+Willis&author=Y.+Zhao&author=H.+Woodward&author=F.+Svensson&author=J.+Sipthorp&author=L.+Vecchia&author=R.+R.+Ruza&author=S.+Kjaer&author=S.+Frew&author=A.+Monaghan&author=M.+Bictash&author=P.+Salinas&author=P.+Whiting&author=J.-P.+Vincent&author=E.+Y.+Jones&author=P.+V.+Fish&title=5-Phenyl-1%2C3%2C4-oxadiazole-2%283H%29-ones+are+Potent+Inhibitors+of+Notum+Carboxylesterase+Activity+Identified+by+the+Optimization+of+a+Crystallographic+Fragment+Screening+Hit"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMahy%26aufirst%3DW.%26atitle%3D5-Phenyl-1%252C3%252C4-oxadiazole-2%25283H%2529-ones%2520are%2520Potent%2520Inhibitors%2520of%2520Notum%2520Carboxylesterase%2520Activity%2520Identified%2520by%2520the%2520Optimization%2520of%2520a%2520Crystallographic%2520Fragment%2520Screening%2520Hit%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26spage%3D01391" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rankovic, Z.</span></span> <span> </span><span class="NLM_article-title">CNS drug design: balancing physicochemical properties for optimal brain exposure</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">2584</span>â <span class="NLM_lpage">2608</span>, <span class="refDoi">Â DOI: 10.1021/jm501535r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501535r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVyhsrrP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=2584-2608&author=Z.+Rankovic&title=CNS+drug+design%3A+balancing+physicochemical+properties+for+optimal+brain+exposure&doi=10.1021%2Fjm501535r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">CNS Drug Design: Balancing Physicochemical Properties for Optimal Brain Exposure</span></div><div class="casAuthors">Rankovic, Zoran</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2584-2608</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The human brain is a uniquely complex organ, which has evolved a sophisticated protection system to prevent injury from external insults and toxins.  Designing mols. that can overcome this protection system and achieve optimal concn. at the desired therapeutic target in the brain is a specific and major challenge for medicinal chemists working in CNS drug discovery.  Analogous to the now widely accepted rule of 5 in the design of oral drugs, the physicochem. properties required for optimal brain exposure have been extensively studied in an attempt to similarly define the attributes of successful CNS drugs and drug candidates.  This body of work is systematically reviewed here, with a particular emphasis on the interplay between the most crit. physicochem. and pharmacokinetic parameters of CNS drugs as well as their impact on medicinal chem. strategies toward mols. with optimal brain exposure.  A summary of modern CNS pharmacokinetic concepts and methods is also provided.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8V0N1j49uuLVg90H21EOLACvtfcHk0lhLXoej27O7lw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVyhsrrP&md5=f01e3f2dc4e2eff7b0618124277a5cf8</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm501535r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501535r%26sid%3Dliteratum%253Aachs%26aulast%3DRankovic%26aufirst%3DZ.%26atitle%3DCNS%2520drug%2520design%253A%2520balancing%2520physicochemical%2520properties%2520for%2520optimal%2520brain%2520exposure%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D2584%26epage%3D2608%26doi%3D10.1021%2Fjm501535r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wager, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhoest, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villalobos, A.</span></span> <span> </span><span class="NLM_article-title">Moving beyond rules: the development of a central nervous system multiparameter optimization (CNS MPO) approach to enable alignment of druglike properties</span>. <i>ACS Chem. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">435</span>â <span class="NLM_lpage">449</span>, <span class="refDoi">Â DOI: 10.1021/cn100008c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cn100008c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvVejtr8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=435-449&author=T.+T.+Wagerauthor=X.+Houauthor=P.+R.+Verhoestauthor=A.+Villalobos&title=Moving+beyond+rules%3A+the+development+of+a+central+nervous+system+multiparameter+optimization+%28CNS+MPO%29+approach+to+enable+alignment+of+druglike+properties&doi=10.1021%2Fcn100008c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Moving beyond Rules: The Development of a Central Nervous System Multiparameter Optimization (CNS MPO) Approach To Enable Alignment of Druglike Properties</span></div><div class="casAuthors">Wager, Travis T.; Hou, Xinjun; Verhoest, Patrick R.; Villalobos, Anabella</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Neuroscience</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">435-449</span>CODEN:
                <span class="NLM_cas:coden">ACNCDM</span>;
        ISSN:<span class="NLM_cas:issn">1948-7193</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The interplay among commonly used physicochem. properties in drug design was examd. and utilized to create a prospective design tool focused on the alignment of key druglike attributes.  Using a set of six physicochem. parameters ((a) lipophilicity, calcd. partition coeff. (ClogP); (b) calcd. distribution coeff. at pH = 7.4 (ClogD); (c) mol. wt. (MW); (d) topol. polar surface area (TPSA); (e) no. of hydrogen bond donors (HBD); (f) most basic center (pKa)), a druglikeness central nervous system multiparameter optimization (CNS MPO) algorithm was built and applied to a set of marketed CNS drugs (N = 119) and Pfizer CNS candidates (N = 108), as well as to a large diversity set of Pfizer proprietary compds. (N = 11 303).  The novel CNS MPO algorithm showed that 74% of marketed CNS drugs displayed a high CNS MPO score (MPO desirability score â¥ 4, using a scale of 0-6), in comparison to 60% of the Pfizer CNS candidates.  This anal. suggests that this algorithm could potentially be used to identify compds. with a higher probability of successfully testing hypotheses in the clinic.  In addn., a relationship between an increasing CNS MPO score and alignment of key in vitro attributes of drug discovery (favorable permeability, P-glycoprotein (P-gp) efflux, metabolic stability, and safety) was seen in the marketed CNS drug set, the Pfizer candidate set, and the Pfizer proprietary diversity set.  The CNS MPO scoring function offers advantages over hard cutoffs or utilization of single parameters to optimize structure-activity relationships (SAR) by expanding medicinal chem. design space through a holistic assessment approach.  Based on six physicochem. properties commonly used by medicinal chemists, the CNS MPO function may be used prospectively at the design stage to accelerate the identification of compds. with increased probability of success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqB3Mvkgm_X5bVg90H21EOLACvtfcHk0lhLXoej27O7lw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvVejtr8%253D&md5=e94998f65a420be6ee4dec4a987cb983</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fcn100008c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcn100008c%26sid%3Dliteratum%253Aachs%26aulast%3DWager%26aufirst%3DT.%2BT.%26aulast%3DHou%26aufirst%3DX.%26aulast%3DVerhoest%26aufirst%3DP.%2BR.%26aulast%3DVillalobos%26aufirst%3DA.%26atitle%3DMoving%2520beyond%2520rules%253A%2520the%2520development%2520of%2520a%2520central%2520nervous%2520system%2520multiparameter%2520optimization%2520%2528CNS%2520MPO%2529%2520approach%2520to%2520enable%2520alignment%2520of%2520druglike%2520properties%26jtitle%3DACS%2520Chem.%2520Neurosci.%26date%3D2010%26volume%3D1%26spage%3D435%26epage%3D449%26doi%3D10.1021%2Fcn100008c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span></span> <i>Development of Potent, Selective, CNS Penetrant Small Molecule Inhibitors of Notum to Potentiate Wnt Signalling</i>; <span class="NLM_publisher-name">EFMC: XXVth International Symposium on Medicinal Chemistry</span>: <span class="NLM_publisher-loc">Ljubljana, Slovenia</span>, Sep 2â6, <span class="NLM_year">2018</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=P.+V.+Fish&title=Development+of+Potent%2C+Selective%2C+CNS+Penetrant+Small+Molecule+Inhibitors+of+Notum+to+Potentiate+Wnt+Signalling"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DFish%26aufirst%3DP.%2BV.%26btitle%3DDevelopment%2520of%2520Potent%252C%2520Selective%252C%2520CNS%2520Penetrant%2520Small%2520Molecule%2520Inhibitors%2520of%2520Notum%2520to%2520Potentiate%2520Wnt%2520Signalling%26pub%3DEFMC%253A%2520XXVth%2520International%2520Symposium%2520on%2520Medicinal%2520Chemistry%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fish, P. V.</span>; <span class="NLM_string-name">Mahy, W.</span>; <span class="NLM_string-name">Willis, N. J.</span>; <span class="NLM_string-name">Woodward, H.</span>; <span class="NLM_string-name">Atkinson, B. N.</span>; <span class="NLM_string-name">Bayle, E. D.</span>; <span class="NLM_string-name">Sipthorp, J.</span>; <span class="NLM_string-name">Jones, E. Y.</span>; <span class="NLM_string-name">Zhao, Y.</span>; <span class="NLM_string-name">Vecchia, L.</span>; <span class="NLM_string-name">Ruza, R. R.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of Secreted Carboxylesterase Notum and their use in Increasing Wnt Protein Signalling and Treatment of Central Nervous System and Other Disorders</span>. WO Patent Application <span class="NLM_patent">2020043866</span>, March 5, <span class="NLM_year">2020</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=P.+V.+Fish&author=W.+Mahy&author=N.+J.+Willis&author=H.+Woodward&author=B.+N.+Atkinson&author=E.+D.+Bayle&author=J.+Sipthorp&author=E.+Y.+Jones&author=Y.+Zhao&author=L.+Vecchia&author=R.+R.+Ruza&title=Inhibitors+of+Secreted+Carboxylesterase+Notum+and+their+use+in+Increasing+Wnt+Protein+Signalling+and+Treatment+of+Central+Nervous+System+and+Other+Disorders"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DFish%26aufirst%3DP.%2BV.%26atitle%3DInhibitors%2520of%2520Secreted%2520Carboxylesterase%2520Notum%2520and%2520their%2520use%2520in%2520Increasing%2520Wnt%2520Protein%2520Signalling%2520and%2520Treatment%2520of%2520Central%2520Nervous%2520System%2520and%2520Other%2520Disorders%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kazembe, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, D. A.</span></span> <span> </span><span class="NLM_article-title">Synthesis of pyrrolo[1,2-a]indoles: Vilsmeier formylation of some 3-methylindol-2-yl ketones and 3(3-methylindol-2-yl)propenoic esters</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">1980</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">2125</span>â <span class="NLM_lpage">2132</span>, <span class="refDoi">Â DOI: 10.1016/0040-4020(80)80102-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=10.1016%2F0040-4020%2880%2980102-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;key=1%3ACAS%3A528%3ADyaL3MXotlyksQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1980&pages=2125-2132&author=C.+G.+Kazembeauthor=D.+A.+Taylor&title=Synthesis+of+pyrrolo%5B1%2C2-a%5Dindoles%3A+Vilsmeier+formylation+of+some+3-methylindol-2-yl+ketones+and+3%283-methylindol-2-yl%29propenoic+esters&doi=10.1016%2F0040-4020%2880%2980102-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of pyrrolo[1,2-a]indoles: Vilsmeier formylation of some 3-methylindol-2-yl ketones and 3(3-methylindol-2-yl)propenoic ester</span></div><div class="casAuthors">Kazembe, Takawira C. G.; Taylor, Deryck A.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">1980</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2125-32</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    </div><div class="casAbstract">The prepn. of several pyrrolo[1,2-a]indoles (e.g. I) by formylation/cyclization of indoles is reported.  E.g., methylindole ester II was prepd. and reacted with POCl3/DMF at 60Â° for 1 h to give 67% I.  The prepn. of some monomethylated 1-phenylpyrrole-2,4- and -3,4-dicarboxylates is also reported.  E.g., BrCH2COCO2Et reacted with PhNHCMe:CHCO2Et for 20 min in refluxing EtOH to give 72% diester III.  The sapon. of these diesters is remarkably selective, and in the case of 2,4-dicarboxylates contradicts accepted generalizations concerning the lability of pyrrole Î±,Î²-diesters to selective hydrolysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpr1bzL4PfdN7Vg90H21EOLACvtfcHk0lhE680-y0C2Mg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3MXotlyksQ%253D%253D&md5=ac25d0e3f5bac6994005dd0c538e790e</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2F0040-4020%2880%2980102-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0040-4020%252880%252980102-5%26sid%3Dliteratum%253Aachs%26aulast%3DKazembe%26aufirst%3DC.%2BG.%26aulast%3DTaylor%26aufirst%3DD.%2BA.%26atitle%3DSynthesis%2520of%2520pyrrolo%255B1%252C2-a%255Dindoles%253A%2520Vilsmeier%2520formylation%2520of%2520some%25203-methylindol-2-yl%2520ketones%2520and%25203%25283-methylindol-2-yl%2529propenoic%2520esters%26jtitle%3DTetrahedron%26date%3D1980%26volume%3D36%26spage%3D2125%26epage%3D2132%26doi%3D10.1016%2F0040-4020%2880%2980102-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beard, R. L.</span>, <span class="NLM_string-name">Donello, J. E.</span>, <span class="NLM_string-name">Yuan, H.</span>, <span class="NLM_string-name">Colon, D. F.</span>, <span class="NLM_string-name">Duong, T.</span>, <span class="NLM_string-name">Liu, X.</span>, <span class="NLM_string-name">Hu, Y.</span></span> <span> </span><span class="NLM_article-title">Indole-3-carboxylic Acid Amide, Ester, Thioamide, and Thiol Ester Compounds Bearing Aryl or Heteroaryl Groups Having Sphingosine-1-phosphate (s1p) Receptor Antagonist Biological Activity</span>. US Patent <span class="NLM_patent">US2007/0191313-A1</span>, Aug 16, <span class="NLM_year">2007</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&author=R.+L.+Beard&author=J.+E.+Donello&author=H.+Yuan&author=D.+F.+Colon&author=T.+Duong&author=X.+Liu&author=Y.+Hu&title=Indole-3-carboxylic+Acid+Amide%2C+Ester%2C+Thioamide%2C+and+Thiol+Ester+Compounds+Bearing+Aryl+or+Heteroaryl+Groups+Having+Sphingosine-1-phosphate+%28s1p%29+Receptor+Antagonist+Biological+Activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBeard%26aufirst%3DR.%2BL.%26atitle%3DIndole-3-carboxylic%2520Acid%2520Amide%252C%2520Ester%252C%2520Thioamide%252C%2520and%2520Thiol%2520Ester%2520Compounds%2520Bearing%2520Aryl%2520or%2520Heteroaryl%2520Groups%2520Having%2520Sphingosine-1-phosphate%2520%2528s1p%2529%2520Receptor%2520Antagonist%2520Biological%2520Activity%26date%3D2007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4UZQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4UZQ','PDB','4UZQ'); return false;">PDB: 4UZQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YUY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YUY','PDB','6YUY'); return false;">PDB: 6YUY</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YV4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YV4','PDB','6YV4'); return false;">PDB: 6YV4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YUW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YUW','PDB','6YUW'); return false;">PDB: 6YUW</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YV2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YV2','PDB','6YV2'); return false;">PDB: 6YV2</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YV0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YV0','PDB','6YV0'); return false;">PDB: 6YV0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T2K" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6T2K','PDB','6T2K'); return false;">PDB: 6T2K</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YXI" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YXI','PDB','6YXI'); return false;">PDB: 6YXI</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YSK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YSK','PDB','6YSK'); return false;">PDB: 6YSK</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i121"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00660">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_55131"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00660?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00660</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Acid fragment library of 250 compounds (catalogue nos. and SMILES); X-ray structure determination including PDB codes, data collection, and refinement; chemical structures for inactive pyrrole and pyrrolidine-3-carboxylic acids present in the acid fragment library; inhibition of Notum activity of additional pyrroles and pyrrolidines; spectroscopic and analytical data for (<i>S</i>)-<b>26</b>; ADME protocols and data; Notum protein production, crystallization, compound soaking, data collection, and structure determination (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00660/suppl_file/jm0c00660_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00660/suppl_file/jm0c00660_si_002.xlsx">XLSX</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00660/suppl_file/jm0c00660_si_001.pdf">jm0c00660_si_001.pdf (1.15 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00660/suppl_file/jm0c00660_si_002.xlsx">jm0c00660_si_002.xlsx (14.35 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates for X-ray structures of Notum crystallized with <b>5</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YUY">6YUY</a>), <b>6</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YV4">6YV4</a>), <b>7</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YUW">6YUW</a>), <b>8</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YV2">6YV2</a>), <b>10</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YV0">6YV0</a>), <b>20i</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YXI">6YXI</a>), and <b>26</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6YSK">6YSK</a>) have been deposited in the Protein Data Bank.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00660&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00660%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-17%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00660" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67994b02285c21e5","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
