
<html lang="en"     class="pb-page"  data-request-id="c37db695-1902-46ac-8281-e3a92926ec07"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2009.52.issue-12;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/jm9000659"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Design and Implementation of an Ribonucleic Acid (RNA) Directed Fragment Library∥" /></meta><meta name="dc.Creator" content="Khaled  Bodoor" /></meta><meta name="dc.Creator" content="Vamsi  Boyapati" /></meta><meta name="dc.Creator" content="Vikram  Gopu" /></meta><meta name="dc.Creator" content="Marietta  Boisdore" /></meta><meta name="dc.Creator" content="Kiran  Allam" /></meta><meta name="dc.Creator" content="Janae  Miller" /></meta><meta name="dc.Creator" content="W. Dale  Treleaven" /></meta><meta name="dc.Creator" content="Thomas  Weldeghiorghis" /></meta><meta name="dc.Creator" content="Fareed  Aboul-ela" /></meta><meta name="dc.Description" content="The design of RNA binding ligands is complicated by issues of specificity, target flexibility, and the tractability of known RNA inhibitors toward chemical derivitization. To address these difficul..." /></meta><meta name="Description" content="The design of RNA binding ligands is complicated by issues of specificity, target flexibility, and the tractability of known RNA inhibitors toward chemical derivitization. To address these difficul..." /></meta><meta name="dc.Publisher" content=" American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="May 15, 2009" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm9000659" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2009 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm9000659" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm9000659" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm9000659" /></link>
        
    
    

<title>Design and Implementation of an Ribonucleic Acid (RNA) Directed Fragment Library | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm9000659" /></meta><meta property="og:title" content="Design and Implementation of an Ribonucleic Acid (RNA) Directed Fragment Library∥" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/large/jm-2009-000659_0005.jpeg" /></meta><meta property="og:description" content=" The design of RNA binding ligands is complicated by issues of specificity, target flexibility, and the tractability of known RNA inhibitors toward chemical derivitization. To address these difficulties, an RNA-directed fragment compound library is presented. We began with an analysis of 120 small molecules with reported RNA-binding activity. Calculated physical and chemical properties for the RNA ligands are comparable to those of ligands for established protein drug targets. To ensure that our library contained RNA-binding functionalities that might not be detected by the above comparisons, 114 fragment compounds were purchased on the basis of similarity to substructures of RNA ligands. Five “hits” were identified for the decoding site from the bacterial ribosome by NMR. These included fragments derived from A-site binding ligands but also compounds not previously identified as A-site binders. Hits generated in this manner can be used to probe the interaction surface of RNA and its conformational plasticity, facilitating structure-based optimization." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm9000659"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm9000659">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm9000659&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm9000659&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm9000659&amp;href=/doi/10.1021/jm9000659" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2009</span><span class="cit-fg-volume">, 52</span><span class="cit-fg-issue">, 12</span><span class="cit-fg-pageRange">, 3753-3761</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/52/12" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm9001296" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm900036m" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Design and Implementation of an Ribonucleic Acid (RNA) Directed Fragment Library<a class="ref internalNav" href="#fnded" aria-label="∥">∥</a></span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Khaled++Bodoor">Khaled Bodoor</a></span><span class="author-xref-symbol "><sup>†</sup></span><span class="author-xref-symbol "><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Vamsi++Boyapati">Vamsi Boyapati</a></span><span class="author-xref-symbol "><sup>‡</sup></span><span class="author-xref-symbol "><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Vikram++Gopu">Vikram Gopu</a></span><span class="author-xref-symbol "><sup>‡</sup></span><span class="author-xref-symbol "><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Marietta++Boisdore">Marietta Boisdore</a></span><span class="author-xref-symbol "><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kiran++Allam">Kiran Allam</a></span><span class="author-xref-symbol "><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Janae++Miller">Janae Miller</a></span><span class="author-xref-symbol "><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=W.+Dale++Treleaven">W. Dale Treleaven</a></span><span class="author-xref-symbol "><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Thomas++Weldeghiorghis">Thomas Weldeghiorghis</a></span><span class="author-xref-symbol "><sup>§</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Fareed++Aboul-ela">Fareed Aboul-ela</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol "><sup>‡</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info"><span class="aff-text">Departments of Biological Sciences and Chemistry, Louisiana State University, Baton Rouge, Louisiana 70803, and Department of Physics, University of Jordan, Amman 11942, Jordan</span></div><div class="corresp-fn" id="fnded">∥We dedicate this manuscript to the memory of our friend and colleague, Dr. Kiran Allam.</div><div class="corresp-info"><strong>*</strong> To whom correspondence should be addressed. Phone: 225-578-2791. Fax: 225-578-7258. E-mail: <a href="/cdn-cgi/l/email-protection#345255565b4158745847411a515041"><span class="__cf_email__" data-cfemail="aec8cfccc1dbc2eec2dddb80cbcadb">[email protected]</span></a></div><div class="corresp-fn" id="afn1">†University of Jordan.</div><div class="corresp-fn" id="afn2">#These authors contributed equally to this work.</div><div class="corresp-fn" id="afn3">‡Department of Biological Sciences, Louisiana State University.</div><div class="corresp-fn" id="afn4">§Department of Chemistry, Louisiana State University.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm9000659&amp;href=/doi/10.1021%2Fjm9000659" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2009</span></span><span class="cit-volume">, 52</span><span class="cit-issue">, 12</span><span class="cit-pageRange">, 3753–3761</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 15, 2009</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>16 January 2009</li><li><span class="item_label"><b>Published</b> online</span>15 May 2009</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 25 June 2009</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm9000659" title="DOI URL">https://doi.org/10.1021/jm9000659</a></div><div class="article_header-article-copyright"><strong>Copyright © 2009 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D3753%26pageCount%3D9%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DKhaled%2BBodoor%252C%2BVamsi%2BBoyapati%252C%2BVikram%2BGopu%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D52%26issueNum%3D12%26contentID%3Djm9000659%26title%3DDesign%2Band%2BImplementation%2Bof%2Ban%2BRibonucleic%2BAcid%2B%2528RNA%2529%2BDirected%2BFragment%2BLibrary%26numPages%3D9%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D3761%26publicationDate%3DJune%2B2009">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm9000659"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2319</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">27</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm9000659" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Design and Implementation of an Ribonucleic Acid (RNA) Directed Fragment Library&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Khaled&quot;,&quot;last_name&quot;:&quot;Bodoor&quot;},{&quot;first_name&quot;:&quot;Vamsi&quot;,&quot;last_name&quot;:&quot;Boyapati&quot;},{&quot;first_name&quot;:&quot;Vikram&quot;,&quot;last_name&quot;:&quot;Gopu&quot;},{&quot;first_name&quot;:&quot;Marietta&quot;,&quot;last_name&quot;:&quot;Boisdore&quot;},{&quot;first_name&quot;:&quot;Kiran&quot;,&quot;last_name&quot;:&quot;Allam&quot;},{&quot;first_name&quot;:&quot;Janae&quot;,&quot;last_name&quot;:&quot;Miller&quot;},{&quot;first_name&quot;:&quot;W.&quot;,&quot;last_name&quot;:&quot;Dale Treleaven&quot;},{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;Weldeghiorghis&quot;},{&quot;first_name&quot;:&quot;Fareed&quot;,&quot;last_name&quot;:&quot;Aboul-ela&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2009&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;15&quot;,&quot;issue&quot;:&quot;12&quot;,&quot;volume&quot;:&quot;52&quot;,&quot;pages&quot;:&quot;3753-3761&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm9000659&quot;},&quot;abstract&quot;:&quot;The design of RNA binding ligands is complicated by issues of specificity, target flexibility, and the tractability of known RNA inhibitors toward chemical derivitization. To address these difficulties, an RNA-directed fragment compound library is presented. We began with an analysis of 120 small molecules with reported RNA-binding activity. Calculated physical and chemical properties for the RNA ligands are comparable to those of ligands for established protein drug targets. To ensure that our library contained RNA-binding functionalities that might not be detected by the above comparisons, 114 fragment compounds were purchased on the basis of similarity to substructures of RNA ligands. Five “hits” were identified for the decoding site from the bacterial ribosome by NMR. These included fragments derived from A-site binding ligands but also compounds not previously identified as A-site binders. Hits generated in this manner can be used to probe the interaction surface of RNA and its conformational plasticity,&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm9000659&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm9000659" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm9000659&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm9000659" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm9000659&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm9000659" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm9000659&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm9000659&amp;href=/doi/10.1021/jm9000659" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm9000659" /></input><a href="/doi/pdf/10.1021/jm9000659" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm9000659&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm9000659%26sid%3Dliteratum%253Aachs%26pmid%3D19445516%26genre%3Darticle%26aulast%3DBodoor%26date%3D2009%26atitle%3DDesign%2Band%2BImplementation%2Bof%2Ban%2BRibonucleic%2BAcid%2B%2528RNA%2529%2BDirected%2BFragment%2BLibrary%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D52%26issue%3D12%26spage%3D3753%26epage%3D3761%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=292202" title="Drug discovery">Drug discovery</a>,</li><li><a href="/action/doSearch?ConceptID=290731" title="Biological databases">Biological databases</a>,</li><li><a href="/action/doSearch?ConceptID=291967" title="Cluster chemistry">Cluster chemistry</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/52/12" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_null" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/medium/jm-2009-000659_0005.gif" alt="" id="_i1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/large/jm-2009-000659_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm9000659&amp;id=f_null"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The design of RNA binding ligands is complicated by issues of specificity, target flexibility, and the tractability of known RNA inhibitors toward chemical derivitization. To address these difficulties, an RNA-directed fragment compound library is presented. We began with an analysis of 120 small molecules with reported RNA-binding activity. Calculated physical and chemical properties for the RNA ligands are comparable to those of ligands for established protein drug targets. To ensure that our library contained RNA-binding functionalities that might not be detected by the above comparisons, 114 fragment compounds were purchased on the basis of similarity to substructures of RNA ligands. Five “hits” were identified for the decoding site from the bacterial ribosome by NMR. These included fragments derived from A-site binding ligands but also compounds not previously identified as A-site binders. Hits generated in this manner can be used to probe the interaction surface of RNA and its conformational plasticity, facilitating structure-based optimization.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63500" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63500" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Strong evidence exists for the importance of RNA as a drug target.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a> X-ray structures of clinically used antibiotics bound to ribosomal subunits demonstrated that these drugs contact primarily RNA.<a onclick="showRef(event, 'ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5">(3-5)</a> Structured elements in untranslated regions of viral mRNAs<a class="ref internalNav" href="#fnabbrev" aria-label="a">a</a><sup>,</sup><a onclick="showRef(event, 'ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8">(6-8)</a><div class="NLM_fn" id="fnabbrev">a<p class="last">Abbreviations: NMR, nuclear magnetic resonance, QC, quality control; water-LOGSY, water-ligand observed via gradient spectroscopy; NCI, National Cancer Institute; UPMCLD, University of Pittsburgh Center for Chemical Libraries and Library Development; 1D, one-dimensional; MHz, megahertz; TPSA, topological polar surface area, MOE, Molecular Operating Environment (software); STD, saturation transfer difference spectroscopy; rRNA, ribosomal RNA; mRNA, messenger RNA.</p></div> are needed for viral growth. Genomes of higher organisms contain a surprising amount of noncoding RNA, which is not translated into proteins.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> The roles so far discovered for these RNAs raise the possibility that small molecule drugs targeted at RNA could be used to treat cancer, neurological, and genetic diseases.<a onclick="showRef(event, 'ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref10 ref11 ref12">(10-12)</a> Small molecule RNA-binding ligands may be useful as “knockdown” probes for RNA function. Ligands that can modify or cleave a target RNA may be even more potent as therapeutics and functional probes.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13, 14)</a></div><div class="NLM_p">Efforts to target RNA molecules from HIV<a onclick="showRef(event, 'ref15 ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref17 ref18">(15-18)</a> and the bacterial ribosome<a onclick="showRef(event, 'ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21">(19-21)</a> produced a number of novel chemical entities with potent binding. However, existing drugs that target RNA have been identified through serendipity.</div><div class="NLM_p">The polyanionic and highly solvated character of RNA presents unique challenges for medicinal chemistry.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> RNA conformational flexibility complicates structure-based drug design,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> and, particularly, the application of in silico docking to RNA.<a onclick="showRef(event, 'ref22 ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref22 ref23 ref24 ref25">(22-25)</a></div><div class="NLM_p">In recent years, fragment-based strategies combined structural biology with careful library design to develop viable clinical drug candidates targeted against proteins.<a onclick="showRef(event, 'ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28">(26-28)</a> These strategies begin with careful design of a screening “fragment” library consisting of “leadlike” molecules.<a onclick="showRef(event, 'ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref29 ref30 ref31">(29-31)</a> Typically, fragment compounds are of low molecular weight (<300), contain two or more polar groups, and a moderate predicted log <i>P</i> (where <i>P</i> is the partition coefficient for the compound in a water/octanol mixture). Fragments are screened for weak binding to the molecular target. In contrast to standard in silico docking protocols, a rigid, preformed target conformation is not assumed. As a framework for computational structure-based design, a set of fragment hit/target complex structures is superior to a single apo-structure.</div><div class="NLM_p">The fragment approach has potential to address the distinct challenges posed by RNA. It can probe conformational flexibility. Cheminformatics tools used to design libraries can be applied to the special medicinal chemistry issues presented by RNA. A handful of studies discovered RNA-binding ligands by screening generic fragment libraries against RNA targets.<a onclick="showRef(event, 'ref32 ref33 ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref32 ref33 ref34 ref35">(32-35)</a> However, none of the reported RNA fragment screens utilized a fragment library specifically designed to enhance the probability of binding to RNA.</div><div class="NLM_p last">Here, we report the design of an RNA-directed fragment library. We assembled a database of 120 small molecule ligands that, according to literature, bind directly to an RNA. We show that known RNA binding ligands, in their calculated physicochemical properties, span a range comparable to those of ligands for kinases and proteases. We therefore chose 114 RNA-focused fragments via direct incorporation of compounds with similarity to substructures of RNA ligands. We screened the library for binding to a 27 nucleotide oligonucleotide incorporating the binding site for aminoglycoside antibiotics<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> and obtained a set of hits, including at least two compounds that had not been previously reported as ribosomal decoding site binders. Our RNA-directed fragment library provides a powerful tool for identifying novel chemical starting points for design of RNA-binding ligands.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Experimental Details</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82447" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82447" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> RNA-Binding Ligand Database</h3><div class="NLM_p last">The RNA-binding ligand database was assembled by searching published literature for reports of small molecules binding to any RNA molecule. A small molecule was defined as a nonpeptidic or peptido-mimetic ligand, although natural products such as thiazole antibiotics, which contain peptidic linkages, were included. Ligands were only included when the reported <i>K</i><sub>d</sub> is less than 50 μM. Where a series of chemically similar compounds was reported to bind to an RNA, the ligand with the strongest reported affinity was chosen as a representative, together with any other member of the series that contained a chemically distinctive substituent, provided that the latter fit the <i>K</i><sub>d</sub> criteria. The full list of compounds together with references is included in the <a class="ref internalNav" href="#si1" aria-label="Supporting Information">Supporting Information</a>.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Descriptor Comparison of Ligand Sets</h3><div class="NLM_p">Seven ligand databases were analyzed: the RNA-binding ligand database, the NCI/DTP Open Chemical Repository (NCI) diversity set prepared for use with AutoDock, the 327 compound University of Pittsburgh Center for Chemical Libraries and Library Development (UPCMLD) diversity set, the 211 nonredundant entries from the first 300 entries of the Ashgate Drug Index (purchased from CamSoft), a set of 626 protein-binding ligands, a set of 162 kinase binding ligands and 353 protease binding ligands, the latter three identified from the PDB as described (see below). Each ligand set was imported into MOE (Molecular Operating Environment, CCG) as a set of SD files. The NCI and UPCMLD libraries were downloaded as SD files, while the RNA-binding and Ashgate sets were manually drawn using ChemDraw and exported as SD files. Protein binding ligands were downloaded from the PDB Web site as sdf files.</div><div class="NLM_p last">In a few cases, outliers were observed in descriptor values that distorted overall distribution statistics for specific descriptors. The descriptors in question were recalculated with the outlying ligand excluded. This exclusion was applied only when (1) the mean descriptor value for the ligand set was changed by greater than a standard deviation as a result of the value calculated for a single ligand and (2) similar ligands had very different descriptor values, suggesting that the program may have encountered a problem in executing the calculation in question. It was never necessary to exclude more than one ligand from any given database for any given calculation, except for the protein-binding ligand set, for which five SD files that generated nonphysical chemical connectivities had to be excluded.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Fragmenting of Data-Basing Ligands</h3><div class="NLM_p">Each reported RNA-binding ligand was subdivided using ChemDraw. Fragmentation points were chosen at “linker” positions next to rings. The primary criterion for fragmentation corresponded roughly to prioritization rules 2 and 3 for scaffold formation as proposed by Schuffenhauer and colleagues.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> However, as our purpose was to preserve all potential RNA-binding entities from all fragments, rather than preserving only a scaffold, other prioritization rules were not relevant. Where multiple bonds separated two ring moieties, the linker segment itself was treated as a fragment. In general, we aimed to obtain products that fit the molecular weight criteria for leadlike fragments (MW < 250). In a few cases (e.g., tetracycline) polycyclic rings did not present obvious fragmentation points for products this small. In those instances, the polycyclic ring was broken into components containing, for example, two to three rings each, but the original compound was also included as a fragment. Very large macrocycles, such as those occurring in some macrolides, were retained intact for the clustering analysis.</div><div class="NLM_p last">PDB coordinate sets for 25 of the database ligands bound to their respective RNA targets were examined to determine which ligand fragments were in contact with RNA. Every fragment of every RNA binding ligand examined was found to have contacts within 3 Å heavy atom to heavy atom distance to the RNA. Because of this finding, we proceeded with the assumption that virtually all fragments of RNA binding ligands are likely to make contact with RNA, and no ligand fragments were discarded on the basis of structural contact criteria.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Clustering of Fragments of RNA-Binding Ligands</h3><div class="NLM_p">The descriptors for clustering and related details are available in the <a class="ref internalNav" href="#si1" aria-label="Supporting Information">Supporting Information</a>.</div><div class="NLM_p last">A representative fragment was chosen from each descriptive cluster. At that stage, because of the overlap between components of the descriptive and fingerprint clusters, some fingerprint clusters were already represented. One fragment was then chosen to represent each of the original 53 fingerprint clusters that was not already represented. Thirteen more fragments were chosen to represent fingerprint subclusters that were at that stage still unrepresented.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Selection, Plating, and Quality Control (QC) of Library Compounds</h3><div class="NLM_p last">These are described in <a class="ref internalNav" href="#si1" aria-label="Supporting Information">Supporting Information</a>.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Screening of Fragments for Binding to the Bacterial Ribosomal A Site RNA</h3><div class="NLM_p last">Instrumentation used and buffer conditions were as described for QC in <a class="ref internalNav" href="#si1" aria-label="Supporting Information">Supporting Information</a>. One-hundred-two compounds were screened in 29 mixtures of 3−4 per tube, containing the same buffer as the QC spectra but with 10 μM A-site RNA. Running the experiment with excess ligand has the advantage of amplifying the bound signal.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> A competitor with known binding to the A-site RNA, gentamycin, was added to a concentration of 30 μM, and experiments were repeated with identical parameters for each mixture. Spectra from a hit compound are identified from the following characteristics: a positive signal in the water-LOGSY spectrum (320 scans, 1.2 ms mixing period) and a reduction in signal in the <i>T</i><sub>2</sub> filter experiment (128 scans, 300 ms <i>T</i><sub>2</sub> filter period) relative to the 1 d experiment with excitation sculpting. We expected that hits that bind specifically to the aminoglycoside binding site show a reduction in positive signal in the water-LOGSY and some recovery of signal in the <i>T</i><sub>2</sub> filter experiment when gentamycin is added. However, gentamycin is a relatively weak affinity ligand compared to competitor ligands commonly utilized in fragment screening against kinases. Ligands that showed small or ambiguous indications of competition with gentamycin were further tested as described below.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Follow-Up of Fragment Hits</h3><div class="NLM_p last">Hit candidates as identified from the library screen were tested directly for binding to the A-site RNA. Buffer conditions were 10 mM NaCl, 10 mM Na phosphate pH 6.7, 0.1 mM EDTA in 700 μL of 90% H<sub>2</sub>O, 10% D<sub>2</sub>O. Compounds were included singly as shown at 200−300 μM. Water-LOGSY, <i>T</i><sub>2</sub>-filter, and 1D Watergate and excitation sculpting experiments were acquired on a 700 MHz Varian NMR spectrometer with an HCN cold probe. A long acquisition (number of transients = 1024) 1D Watergate spectrum with a large sweep width (24 ppm) was performed to obtain a spectrum of the RNA imino resonances.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21355" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21355" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> RNA-Binding Ligand Database</h3><div class="NLM_p last">The first step in designing an RNA-focused fragment library is to determine principles that can enhance the probability for selecting RNA-binding compounds. We assembled a list of compounds that have been reported to bind to any RNA with a <i>K</i><sub>d</sub> in the range of 50 μM or stronger. A few ligands were included for which the reported <i>K</i><sub>d</sub> was not available, if a PDB coordinate set existed showing the ligand bound to an RNA. We incorporated compounds from a variety of sources, including antibiotics, riboswitch effectors, and products of in vitro drug design studies. We intended to limit redundancy in the database while including as much of proven RNA-binding chemical space as possible. Compounds that belonged to similar chemical classes were included if they incorporated distinct chemical moieties that were not present elsewhere in the list. Altogether, 121 compounds were included. A list of the RNA-binding ligands, together with documentation regarding the reported binding target and activity of each, is included in <a class="ref internalNav" href="#si1" aria-label="Supporting Information">Supporting Information</a> (Table S1).</div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Comparison of RNA-Binding Ligands to Drug, in Silico Screening, and Protein-Binding Ligands</h3><div class="NLM_p">Screening library design typically includes filters that ensure that library compounds fit criteria for “druglikeness” and/or “leadlikeness”. At this “proof-of principle” stage in our investigations we do not wish to be limited by these restrictions, and we consider that any designed RNA-binding ligand provides valuable insights into RNA recognition. Nonetheless, we wish to determine whether there is a correlation between physicochemical properties and RNA-binding and whether such properties may conflict with those required for drug design. We used MOE to calculate the distribution of values for a set of conformation-independent physicochemical descriptors for the RNA-binding ligand set. For comparison, we analyzed a similar sized data set derived from the Ashgate Drug Index (CamSoft). These compounds are in commercial use as therapeutic drugs. In addition, we ran a similar analysis on two publicly available “diversity” (from the National Cancer Institute (NCI) and the University of Pittsburgh Center for Chemical Methodologies and Library Development (UPMCSD)) sets commonly used for virtual screening/in silico docking exercises. We also calculated descriptor values for two ligand sets obtained by searching the protein databank (PDB) for small molecules identified as binding to kinases and proteases. Both these classes of proteins include well established drug targets. Finally, we formulated a ligand set of manageable size that could be taken to represent ligands binding to any protein in a “generic” sense (see methods section).</div><div class="NLM_p">Comparative analysis of these ligand sets is shown in Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a> and, in more detail, in the <a class="ref internalNav" href="#si1" aria-label="Supporting Information">Supporting Information</a>. For all 35 physicochemical descriptors computed, the average value for the RNA-binding set was within a standard deviation of that for each of the other five ligand sets. In a few cases, discussed in more detail in the <a class="ref internalNav" href="#si1" aria-label="Supporting Information">Supporting Information</a>, there were small differences in the distribution of values for the four databases.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/medium/jm-2009-000659_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/large/jm-2009-000659_0006.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Average values and standard deviations for physicochemical descriptors for nine ligand sets. For each ligand set, two bars are shown: the mean value is shown on the left and standard deviation on the right. RNA: 125 ligands reported to bind RNA with <i>K</i><sub>d</sub> < 50 μM. NCI: 1944 NCI diversity set compounds. Pittcon: 327 UPMCSD virtual screening set. Ashgate: 211 Ashgate Drug Index compounds. RNA_sub: 67 compound subset of RNA ligands with reported <i>K</i><sub>d</sub> < 3 μM. Fragments: 109 compounds that underwent QC for RNA-directed fragment library. Kinase: 162 kinase-binding ligands identified from PDB. Protease: 282 protease-binding ligands identified from PDB. Protein: 628 protein binding ligands identified from PDB. The following plots are shown (from left to right): (a) molecular weight, (b) Chi0v (zero order atomic valence connectivity index)<sup>53</sup>, (c) topological polar surface area<sup>39</sup>, (d) hydrogen bond acceptors as defined by Lipinski<sup>54</sup>, (e) simulated log of the partition coefficient in a water/octanol mixture, (f) positively charged fraction of the summed partial charge.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/large/jm-2009-000659_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm9000659&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Average values for most 2D descriptors correlate with the molecular weight. Average molecular weight is largest for RNA and protease binding ligands (near 500), whereas average molecular weight is slightly over 300 for the Ashgate and NCI sets. Average values for total surface charge, fractional positive charge, and the topological polar surface area (TPSA)<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> are higher for the kinase binding ligands than expected on the basis of molecular weight.</div><div class="NLM_p">We had set our threshold for RNA binding activity at a relatively weak level (<i>K</i><sub>d</sub> < 50 μM) in order to ensure a thorough exploration of RNA-binding chemical space for our fragment library design below. In order to probe the possibility that by doing so we may be somehow diluting the distinctiveness of our RNA-binding ligands, we repeated the analysis with a subset of 67 ligands with reported <i>K</i><sub>d</sub> of less than 3 μM (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). Again, the average value for each descriptor was within a standard deviation of the average for the comparison ligand sets. The average molecular weight is slightly higher for the set of submicromolar RNA ligands than it is for the complete RNA ligand set, as are the average values for the correlated 2D descriptors. This effect, together with a small increase in TPSA, suggests that the large, polar, and chemically complex ligands pick up affinity through a larger surface area of interaction.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Clustering Statistics</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead valign="middle"><tr valign="top"><th align="center" valign="middle">parameter</th><th align="center" valign="middle">value</th></tr></thead><tbody><tr valign="top"><td align="left">no. of descriptive clusters</td><td align="left">55</td></tr><tr valign="top"><td align="left">no. of fingerprint clusters</td><td align="left">53</td></tr><tr valign="top"><td align="left">subclusters of fingerprint cluster 3</td><td align="left">71</td></tr><tr valign="top"><td align="left">sublusters of fingerprint cluster 74</td><td align="left">19</td></tr><tr valign="top"><td align="left">total RNA-binding ligand fragments</td><td align="left">260</td></tr><tr valign="top"><td align="left">total clusters + subclusters</td><td align="left">198</td></tr><tr valign="top"><td align="left">compounds purchased</td><td align="left">114</td></tr><tr valign="top"><td align="left">fragments passing QC</td><td align="left">100</td></tr></tbody></table></div></div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Choice of Fragment Library Ligands</h3><div class="NLM_p">Our analysis convinced us that development of RNA binding ligands is not intrinsically incompatible with drug development, but it gave us no obvious “rules” for choosing compounds with propensity to bind RNA. This result does not preclude the likelihood that a finite set of chemical moieties constitute a “privileged”<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> RNA binding set. The special properties of these functionalities may be too subtle or complex to detect using standard descriptors.</div><div class="NLM_p">We proceeded with a strategy (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>) to build our library as directly as possible from fragments of compounds with proven RNA-binding propensity. Data base ligand structures were “cleaved” in silico (see <a class="ref internalNav" href="#sec2" aria-label="Experimental Details">Experimental Details</a>). Altogether approximately 250 fragments were identified. Many of these fragments are chemically similar. Therefore, we clustered the fragments, first, based upon chemical descriptors and, second, based upon molecular fingerprints using MOE (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>). Altogether this process produced 55 descriptive clusters and 53 fingerprint clusters. We wished to represent each cluster in our library, but we noted that a disproportionate fraction of the original fragments were represented in two fingerprint clusters (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>, Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>, Table S3). We judged that these clusters may represent a region of chemical space with particularly favorable RNA-binding propensity. We therefore further clustered these ligand subsets with the aim of achieving a “denser” representation of the corresponding chemical space regions.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/medium/jm-2009-000659_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/large/jm-2009-000659_0007.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) Schematic illustration of the need for selecting RNA-directed compounds for screening for binding to RNA. While we can assume that there is some overlap between the regions of “chemical space” with the propensity to bind to RNA or to proteins, it is likely that the probabilities for binding to the two classes of target will differ in any given region. Generic and commercial screening libraries, designed to cover the overlap between protein-binding and “druglike” chemical spaces, yield low hit rates for RNA targets. Note simplifications: for example, regions of RNA-binding space (and other subregions) are not necessarily contiguous. (b) Schematic of overall strategy for selection of an RNA-directed fragment library and for screening the library against RNA. Fragment library compounds are derived from the database of RNA-binding ligands. NMR is used to screen the library for compounds that bind with weak affinity to the RNA target. The complexes of these compounds with the RNA go into structural studies and provide a starting point for structure-based design of more potent ligands.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/large/jm-2009-000659_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm9000659&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/medium/jm-2009-000659_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/large/jm-2009-000659_0001.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (a) Distribution of number of fragments per cluster derived from fragmenting RNA-binding ligand structures in silico and clustering the resulting fragments according to physicochemical descriptors. (b) Distribution of number of fragments per cluster derived from fragmenting RNA-binding ligand structures in silico and clustering the resulting fragments according to chemical fingerprints. Over half of the fragments were clustered together in either cluster 3 or 71. These two clusters were further subclustered (see text).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/large/jm-2009-000659_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm9000659&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We searched for at least one purchasable compound that could represent each cluster, and some subclusters, which had been identified through the above process. Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a> shows an example of a descriptive cluster together with the original ligand fragments composing the cluster and the purchasable fragment(s) chosen to represent the cluster. A more exhaustive catalogue of clusters and purchased fragments for the library is included in the <a class="ref internalNav" href="#si1" aria-label="Supporting Information">Supporting Information</a> (Table S3).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/medium/jm-2009-000659_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/large/jm-2009-000659_0002.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (a) Fragments of RNA binding ligands grouped together as “descriptive cluster 40”. (b) Library compounds that represent chemical space surrounding that covered by cluster 40. The fragment on the left has been chosen to represent the given cluster. The other two fragments, representing other descriptive or fingerprint clusters, nonetheless have similarity to a consensus structure represented by cluster 40.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/large/jm-2009-000659_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm9000659&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Altogether, 102 compounds were initially selected, purchased, survived quality control, and were plated as described in <a class="ref internalNav" href="#sec2" aria-label="Experimental Details">Experimental Details</a>.</div></div><div id="sec3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Cheminformatics Analysis and Quality Control of the RNA-Directed Fragment Library</h3><div class="NLM_p">We subjected the compounds that we had chosen for the RNA directed fragment library to the same descriptor analysis that was described above for the ligand, drug, and diversity sets. Being of smaller molecular weight (on average) than the other sets, the fragment compounds show a more “leadlike”<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> (though still not rigorously leadlike) distribution of descriptor values than any of the ligand sets. This result illustrates one of the important advantages of the fragment strategy. Our fragments are able to span a chemical space derived from bulkier, chemically more complex compounds.</div><div class="NLM_p last">Before being added to RNA for screening, each fragment compound was subjected to a quality control process (described below). Briefly, the integrity and solubility of each compound were tested by inspection of a one-dimensional NMR spectrum. Each compound was also subjected to the full suite of NMR experiments to detect ligand binding. The latter spectra are stored in a database and provide “control” spectra for comparison to screening data.</div></div><div id="sec3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Screen of Fragment Library for Binding to the Ribosomal Decoding Site (“A-Site”) RNA</h3><div class="NLM_p">We used NMR to screen the fragment library for weak affinity binding to a 27-residue oligonucleotide containing the internal loop sequence from the decoding site of the <i>E. coli</i> 16S rRNA (the “A-site” RNA). The sequence of the oligonucleotide was identical to that used in previous NMR studies of the A-site structure and its interaction with antibiotics.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> We chose this target for our first fragment library screen because the same ribosomal site was the target for several of the known RNA-binding ligands that provided the source of our fragment structures.</div><div class="NLM_p">Altogether, 102 compounds were screened for A-site binding in 29 mixtures of three to four compounds, each at 200 μM concentration, through the acquisition of four 1D NMR experiments on each mixture. Each sample contained 10 μM RNA. The four experiments were chosen to include one that detects signal from bound ligands (water-LOGSY), one that filters out signals from bound ligands (<i>T</i><sub>2</sub>-filter), and two routine spectra acquired with alternative solvent suppression schemes. The screening experiments work on the principle of detecting ligand signals, through the transfer of magnetization properties from bound to free signal via chemical exchange. Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>a shows an example of the screening results for one mixture.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/medium/jm-2009-000659_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/large/jm-2009-000659_0003.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (a) Screening data. The sample contained a mixture including compounds <b>2</b> and <b>4</b> and two nonbinding compounds, together with 27 nucleotide A-site RNA. From top to bottom: water-LOGSY, without and with gentamycin competitor, 1D spectra with excitation sculpting (<i>T</i><sub>2</sub>-filter period of 0 s) and with (0.3 s) <i>T</i><sub>2</sub> filter, without gentamycin, and repeat of the excitation sculpting 1D with and without <i>T</i><sub>2</sub>-filter with gentamycin present. Competition of <b>4</b> for the gentamycin site is evident from the recovery of signal in the <i>T</i><sub>2</sub>-filter experiment, although little competition is observed in the water-LOGSY. Binding of compound <b>2</b> is indicated by signals near 6.9 and 7.3−8.1 pm in the water-LOGSY. A nonbinding compound, a furan, gives negative signals between 6.5 and 6.9 ppm. (b) Follow-up experiment confirming binding of <b>4</b> to A-site RNA. Water-LOGSY, <i>T</i><sub>2</sub>-filter, and 1D data are shown with (blue) and without (black) RNA. (c) RNA imino resonances from samples containing 10:1 excess of the respective compounds. Compounds <b>1</b>, <b>4</b>, and <b>5</b> all interact with the RNA, as indicated by shifts in imino resonances, notably in the signal for the G1491 residue (near 12 ppm), indicating that all bind near the active site. The spectrum in the second panel from the top, shows no change from the “free RNA” control, indicating that diethyl nicotinamide does not bind A-site RNA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/large/jm-2009-000659_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm9000659&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Five hit compounds were verified through follow-up experiments performed at higher sensitivity on a 700 MHz spectrometer with a cold probe and a long acquisition time in order to detect imino signals from the RNA. The latter are diagnostic of RNA base pairing. Chemical shift change and/or line broadening in this region indicates ligand contact with the RNA. Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a> shows examples of follow-up screening experiments (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>b) and imino spectra (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>c) for hit compounds mixed with A site RNA. The set of hits includes compounds that have been, or that are closely related to compounds that have been, previously reported as A-site binders, and at least two fragments that, to our knowledge, have not been reported as binding to the RNA in question.</div><div class="NLM_p">The initial screen was followed by a “competition screen” with the known A-site binding ligand, gentamycin, present. The competition experiment indicates which, if any compounds, bind specifically to the known “active site”. Some of the hit compounds showed reduced evidence of binding (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a> and Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>). However, some hits, such as 2-aminoqunoline (<b>2</b>), which has been reported as a component of an A-site active site binder, did not show reduced evidence of binding upon addition of competitor. This result may indicate that the competitor ligand does not bind tightly enough to fully displace the fragment hit. Alternatively, the fragment may be capable of binding in the same region, or other parts of the RNA, simultaneously with the competitor. In the case of compound <b>2</b>, the result is complicated by competition with acridine (<b>4)</b>, which happened to be present in the same initial mixture.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. A-Site Fragment Hits</div></div></div><div class="scrollable-table-wrap"><table class="table single-image" border="0"><colgroup><col align="left" /></col></colgroup><tbody><tr valign="top"><td align="left"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/medium/jm-2009-000659_0004.gif" alt="" id="gr6" /></img></td></tr></tbody></table></div></div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21137" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21137" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Properties of RNA-Binding Compounds</h3><div class="NLM_p">The average values for molecular weight, the correlated 2D descriptors, TPSA, sum of partial charges, and fractional positive charge for the RNA ligand set are relatively high. These averages reflect the component of the ligand set that includes bulky, chemically complex natural product antibiotics, including aminoglycosides and macrolides. Though many of these ligands are clinically useful drugs, they do not fit the classical paradigm of druglikeness. TPSA, which is correlated to the number of hydrogen bond donors and acceptors, is exceptionally large for aminoglycosides and macrolides. It is sometimes considered to be a predictor of oral absorption.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></div><div class="NLM_p last">Our cheminformatics analysis of RNA-binding ligands did not reveal distinct properties that can be characterized using standard physical and chemical descriptors. Nonetheless, the hypothesis that the properties of RNA-binding ligands are dependent upon the properties of RNA, and that therefore their distribution in chemical space will differ from that of the set of protein-binding ligands, is not disproved. Clearly there is overlap between these regions of chemical space, as many small molecules are known to bind both. Yet medicinal chemists are well aware of chemical functionalities that are commonly found in nucleic acid binding ligands and often select to avoid them. The chemically complex ribosome-binding antibiotics in particular not only violate standard druglikeness rules but also present a huge synthetic challenge. Screening of “generic” fragment libraries against RNA targets is therefore likely to miss some of the most important RNA-binding moieties. Our library is designed to “rescue” these functionalities from their presumably undesirable natural product context and to present them to RNA targets in a more “leadlike” form. One important qualification to this statement is that at this stage our library is not restricted to compounds that fit classical criteria for leadlikeness. For example, sugar moieties that are not generally considered as good lead candidates are included because of their proven role in RNA recognition. Our fragments are more leadlike than the original ligand sets.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Test Screen of the RNA-Directed Fragment Library</h3><div class="NLM_p last">We chose the A-site RNA as our first target to enhance the chances of obtaining hits from “control” fragments. For example, 2-deoxystreptamine<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a><b>1</b> and <b>2</b><a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> had already been reported in the literature as A-site binding ligands. The latter was discovered from a generic fragment library screen and incorporated into a stronger affinity derivative, which was then included in our original database. Thus, as we anticipated, we have recapitulated interactions discovered in the literature. On the other hand, we also obtained and confirmed A-site RNA binding by <b>4</b> and lysinamide <b>5</b>. Though the A-site RNA has been subject to intense ligand binding studies and drug design efforts, including a screen against 10 000 compounds in the above-cited study,<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> we could not find a report of binding by these two common compounds. That the strategy depicted in Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>, with the RNA-directed fragment library, was able to identify novel ligands for such a well-characterized RNA bodes well for the prospects of identifying novel chemical starting points for other RNA targets.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Screening the RNA Fragment Library against Other RNA Targets</h3><div class="NLM_p">During the course of this study, we learned lessons regarding the process of screening fragments for weak binding to RNA using NMR. Some binding fragments do not produce positive signals in the water-LOGSY (see Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>). In these cases, the original screen gave an ambiguous result. Only in the follow-up screen, in which the spectrum of the excess ligand mixed with RNA is compared to that without RNA, does the binding become clear. In that case, the compound mixture in the water-LOGSY in the absence of RNA produces negative signals that are reduced when RNA is added. Signals from some ligands become broad in the presence of RNA, leading to poor sensitivity. Ligands containing exchangeable protons give stronger water-LOGSY signals than those that do not contain protons that are capable of exchange with water. The hit rate reported in this study should be seen as a lower limit of that achievable with the library, since potential hits may have been overlooked in the initial screen.</div><div class="NLM_p">An alternative NMR screening method used by the Abbot group<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> used a 1D imino spectrum, with and without ligand present, as a primary screen for ligand binding. As Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>c shows, this can be an effective strategy, but it requires larger amounts of RNA and/or instrumentation of greater sensitivity than the ligand-detecting experiments. A good compromise may be to add an initial ligand-detected screen with compound mixture alone before adding RNA and to follow-up putative hits via the 1D imino measurement. A recent study of in silico hits from a “virtual screen” utilized a saturation transfer difference (STD) experiment to identify RNA binding ligands.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> In our hands the STD experiment is less sensitive for RNA than water-mediated transferred NOE experiments, as has been reported by others.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a></div><div class="NLM_p">Hit rates may be increased by screening against larger RNA targets, such as riboswitches. Larger RNAs form tertiary folds that in turn produce suitable binding pockets for the small ligand. Ligand-detected NMR methods rely on differences in size between free species and bound complex and therefore become more sensitive for large molecular weight targets.</div><div class="NLM_p last">Lessons have also been learned in choice of fragment compounds. We did not impose a lower molecular weight limit. We noted that pyrimidines of molecular weight less than 100, often used as a lower limit in the selection of fragments, have been previously been identified as ribosomal A-site binders. However, in our NMR screen we were unable to reproduce a clear indication of binding to our A-site RNA for these ligands.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Optimization of Fragment Hits</h3><div class="NLM_p last">A second stage of ligand design can link hit fragments or synthesize/select larger compounds that contain the initial hits as substructures. A number of quantitative assays, such as fluorescence-based assays,<a onclick="showRef(event, 'ref15 ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref15 ref43 ref44">(15, 43, 44)</a> mass spectrometry,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> or surface plasmon resonance (SPR),<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> are available to screen the second round of compounds for RNA binding. Ligands that have been identified through assays of ribozyme cleavage activity<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> have not been included in our database, since these measurements do not directly provide a <i>K</i><sub>d</sub> for binding. Nonetheless, these assays may detect relatively weak binding. Moreover, ligands identified in this way clearly bind to RNA and provide additional material for expanding the library.</div></div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66702" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66702" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">This study represents a first step toward the development of an RNA-directed cheminformatics. We hope that such a development will be realized in the coming years, exploiting the ligand databases that we and others are assembling. At this time, we have not uncovered a set of clearly definable criteria that can be applied across the range of RNA targets for enhancing the possibility of RNA binding. Nor have we discovered a set of negative criteria that would preclude RNA binding. Therefore, we have chosen our fragments to mimic chemical functionalities already proven to bind to an RNA in one context, in order to test for binding in other contexts.</div><div class="NLM_p">Screening fragments of the RNA-binding ligands provide an important advantage over screening the original ligands directly, since the fragments will contain smaller polar and hydrophobic surface areas than the original compounds. In addition, starting the drug design process from a fragment hit allows for the design of ligands with specificity for a given RNA target. This approach contrasts with those studies that have worked predominantly with relatively bulky RNA-binding drugs, such as aminoglycosides, to detect binding to other RNA targets.<a onclick="showRef(event, 'ref47 ref48 ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref47 ref48 ref49 ref50">(47-50)</a> The latter approach tends to select for formation of complexes driven by relatively nonspecific electrostatic interactions involving large, polar surface contacts.</div><div class="NLM_p">Another advantage of the fragments approach is that no prior assumption is made regarding the target structure. By the probing of target flexibility through multiple conformations observed with multiple hit fragments, fragments will complement in silico screening methods recently designed to incorporate RNA induced fit.<a onclick="showRef(event, 'ref25 ref42'); return false;" href="javascript:void(0);" class="ref ref25 ref42">(25, 42)</a></div><div class="NLM_p">A drawback to our library is that it is limited to known regions of RNA-binding chemical space. However, though this report concentrates on the first stages of fragment-based ligand discovery, it is important to remember that later stages will “grow” the compound, based on substructure searches and virtual screening from larger purchasable catalogues. These derivatives will sample functionalities not previously known to bind to RNA, when attached to fragment hits that will help to direct the compound toward the RNA binding pocket. A recent study applied template-based de novo design toward the design of TAR RNA binding ligands.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> The de novo approach and other such methods can be used to derive novel chemical entities from fragment hits. Another study has applied microarray technology to simultaneously explore ligand and RNA chemical space to identify binding partners.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> By use of fragment hits as starting points for ligand libraries, such methods may identify more druglike ligands.</div><div class="NLM_p last">The RNA fragment library will be pruned and appended, benefiting from the experience of each screen against each new target. Additions to the library based upon pharmacophores derived from the RNA binding ligand database and derived fragment clusters, as well as from analysis of the surfaces of RNA-binding motifs, will lead to the inclusion of new chemical entities to the screening library. Later iterations will also split the library into more and less drug leadlike sections, for the sake of screens that are directed toward purely therapeutic goals or for use as functional probes.</div></div><div class="NLM_back"><div class="article_supporting-info" id="si1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27"><a href="/doi/suppl/10.1021/jm9000659">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84262" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84262" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Extended discussion of cheminformatics; extended description of experimental procedures for descriptor comparisons, selection, plating, and quality control of fragments, fragment clustering, RNA binding ligands database (Table S1); average and standard deviations of MOE calculated descriptors of eight ligand sets (Table S2); RNA directed fragment library compounds (Table S3); list of the 20 RNA-binding ligands with the largest calculated TPSA value; distribution of calculated TPSA for eight ligand sets (Figure S1). This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm9000659/suppl_file/jm9000659_si_001.pdf">jm9000659_si_001.pdf (1.28 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm9000659" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51716" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51716" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fareed Aboul-ela</span> - <span class="hlFld-Affiliation affiliation">Departments of Biological Sciences and Chemistry, Louisiana State University, Baton Rouge, Louisiana 70803, and Department of Physics, University of Jordan, Amman 11942, Jordan</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#aec8cfccc1dbc2eec2dddb80cbcadb"><span class="__cf_email__" data-cfemail="a4c2c5c6cbd1c8e4c8d7d18ac1c0d1">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Khaled Bodoor</span> - <span class="hlFld-Affiliation affiliation">Departments of Biological Sciences and Chemistry, Louisiana State University, Baton Rouge, Louisiana 70803, and Department of Physics, University of Jordan, Amman 11942, Jordan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Vamsi Boyapati</span> - <span class="hlFld-Affiliation affiliation">Departments of Biological Sciences and Chemistry, Louisiana State University, Baton Rouge, Louisiana 70803, and Department of Physics, University of Jordan, Amman 11942, Jordan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Vikram Gopu</span> - <span class="hlFld-Affiliation affiliation">Departments of Biological Sciences and Chemistry, Louisiana State University, Baton Rouge, Louisiana 70803, and Department of Physics, University of Jordan, Amman 11942, Jordan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marietta Boisdore</span> - <span class="hlFld-Affiliation affiliation">Departments of Biological Sciences and Chemistry, Louisiana State University, Baton Rouge, Louisiana 70803, and Department of Physics, University of Jordan, Amman 11942, Jordan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kiran Allam</span> - <span class="hlFld-Affiliation affiliation">Departments of Biological Sciences and Chemistry, Louisiana State University, Baton Rouge, Louisiana 70803, and Department of Physics, University of Jordan, Amman 11942, Jordan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Janae Miller</span> - <span class="hlFld-Affiliation affiliation">Departments of Biological Sciences and Chemistry, Louisiana State University, Baton Rouge, Louisiana 70803, and Department of Physics, University of Jordan, Amman 11942, Jordan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">W. Dale Treleaven</span> - <span class="hlFld-Affiliation affiliation">Departments of Biological Sciences and Chemistry, Louisiana State University, Baton Rouge, Louisiana 70803, and Department of Physics, University of Jordan, Amman 11942, Jordan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas Weldeghiorghis</span> - <span class="hlFld-Affiliation affiliation">Departments of Biological Sciences and Chemistry, Louisiana State University, Baton Rouge, Louisiana 70803, and Department of Physics, University of Jordan, Amman 11942, Jordan</span></div></li></ul></li><li></li><li></li><li></li></ul></div><div class="ack"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04330" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04330" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Dr. Ron Crouch (Varian, Inc.) for help with implementing automation on the Varian Innova 500 spectrometer. We thank Deepthi Ipparthi for helping to assemble the RNA ligand database. M.B. was supported by Bridge to the Doctorate Program. We thank Dr. Graca Vicente (on behalf of K.A.) for helpful discussions, and Drs. Azeem Hasan and Dan Pu for ESI analysis of hit compounds. This work was supported by startup funds from the Department of Biological Sciences of Louisiana State University, a grant from the Binational Fulbright Commission in Jordan (K.B.), and a Faculty Research Grant (Louisiana State University Office of Research).</p></div><div class="postAsapNotesSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="postAsapNotesSection">Note Added After ASAP Publication</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20266" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20266" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This paper was published on the web May 15, 2009 with a typographical error in the title. The revised version was published on June 3, 2009.</p></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78354" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78354" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 54 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Lange, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Locher, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyss, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Then, R. L.</span><span> </span><span class="NLM_article-title">The targets of currently used antibacterial agents: lessons for drug discovery</span> <span class="citation_source-journal">Curr. Pharm. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">3140</span><span class="NLM_x">–</span> <span class="NLM_lpage">3154</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm9000659&amp;key=10.2174%2F138161207782110408" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm9000659&amp;key=17979755" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlarurnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2007&pages=3140-3154&author=R.+P.+Langeauthor=H.+H.+Locherauthor=P.+C.+Wyssauthor=R.+L.+Then&title=The+targets+of+currently+used+antibacterial+agents%3A+lessons+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The targets of currently used antibacterial agents: lessons for drug discovery</span></div><div class="casAuthors">Lange, Roland P.; Locher, Hans H.; Wyss, Pierre C.; Then, Rudolf L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">3140-3154</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Based on the mode of action of antibacterial drugs currently used, targets can be defined as distinct cellular constituents such as enzymes, enzyme substrates, RNA, DNA, and membranes which exhibit very specific binding sites at the surface of these components or at the interface of macromol. complexes assembled in the cell.  Intriguingly, growth inhibition or even complete loss of bacterial viability is often the result of a cascade of events elicited upon treatment with an antibacterial agent.  In addn., their mode of action frequently involves more than one single target.  A comprehensive description of the targets exploited so far by commercialized antibacterial agents, including anti-mycobacterial agents, is given.  The no. of targets exploited so far by com. antibacterial agents is estd. to be about 40.  The most important biosynthetic pathways and cellular structures affected by antibacterial drugs are the cell wall biosynthesis, protein biosynthesis, DNA per se, replication, RNA per se, transcription and the folate biosynthetic pathway.  The disillusionment with the genomics driven antibacterial drug discovery is a result of the restrictive definition of targets as products of essential and conserved genes.  Emphasis is made to not only focus on proteins as potential drug targets, but increase efforts and devise screening technologies to discover new agents interacting with different RNA species, DNA, new protein families or macromol. complexes of these constituents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo__dPkTDx3OLVg90H21EOLACvtfcHk0ljfxHrMZt1Prg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlarurnE&md5=07c183e1c60e0701f0758b0d7a024f10</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.2174%2F138161207782110408&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161207782110408%26sid%3Dliteratum%253Aachs%26aulast%3DLange%26aufirst%3DR.%2BP.%26aulast%3DLocher%26aufirst%3DH.%2BH.%26aulast%3DWyss%26aufirst%3DP.%2BC.%26aulast%3DThen%26aufirst%3DR.%2BL.%26atitle%3DThe%2520targets%2520of%2520currently%2520used%2520antibacterial%2520agents%253A%2520lessons%2520for%2520drug%2520discovery%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2007%26volume%3D13%26spage%3D3140%26epage%3D3154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Foloppe, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matassova, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aboul-ela, F.</span><span> </span><span class="NLM_article-title">Towards the discovery of drug-like RNA ligands</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x"> (</span><span class="NLM_issue">21/22</span><span class="NLM_x">) </span> <span class="NLM_fpage">1019</span><span class="NLM_x">–</span> <span class="NLM_lpage">1027</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=1019-1027&issue=21%2F22&author=N.+Foloppeauthor=N.+Matassovaauthor=F.+Aboul-ela&title=Towards+the+discovery+of+drug-like+RNA+ligands"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFoloppe%26aufirst%3DN.%26aulast%3DMatassova%26aufirst%3DN.%26aulast%3DAboul-ela%26aufirst%3DF.%26atitle%3DTowards%2520the%2520discovery%2520of%2520drug-like%2520RNA%2520ligands%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2006%26volume%3D11%26issue%3D21%252F22%26spage%3D1019%26epage%3D1027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Yonath, A.</span><span> </span><span class="NLM_article-title">Antibiotics targeting ribosomes: resistance, selectivity, synergism, and cellular regulation</span> <span class="citation_source-journal">Annu. Rev. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">74</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">649</span><span class="NLM_x">–</span> <span class="NLM_lpage">679</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm9000659&amp;key=10.1146%2Fannurev.biochem.74.082803.133130" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm9000659&amp;key=16180279" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpsVensLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2005&pages=649-679&issue=1&author=A.+Yonath&title=Antibiotics+targeting+ribosomes%3A+resistance%2C+selectivity%2C+synergism%2C+and+cellular+regulation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Antibiotics targeting ribosomes: Resistance, selectivity, synergism, and cellular regulation</span></div><div class="casAuthors">Yonath, Ada</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">649-679</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Antibiotics target ribosomes at distinct locations within functionally relevant sites.  They exert their inhibitory action by diverse modes, including competing with substrate binding, interfering with ribosomal dynamics, minimizing ribosomal mobility, facilitating miscoding, hampering the progression of the mRNA chain, and blocking the nascent protein exit tunnel.  Although the ribosomes are highly conserved organelles, they possess subtle sequence and/or conformational variations.  These enable drug selectivity, thus facilitating clin. usage.  The structural implications of these differences were deciphered by comparisons of high-resoln. structures of complexes of antibiotics with ribosomal particles from eubacteria resembling pathogens and from an archaeon that shares properties with eukaryotes.  The various antibiotic-binding modes detected in these structures demonstrate that members of antibiotic families possessing common chem. elements with minute differences might bind to ribosomal pockets in significantly different modes, governed by their chem. properties.  Similarly, the nature of seemingly identical mechanisms of drug resistance is dominated, directly or via cellular effects, by the antibiotics' chem. properties.  The obsd. variability in antibiotic binding and inhibitory modes justifies expectations for structurally based improved properties of existing compds. as well as for the discovery of novel drug classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZrnIOd6ZPE7Vg90H21EOLACvtfcHk0ljfxHrMZt1Prg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpsVensLk%253D&md5=8e91bdf13bb63cdadae3bd26376f8246</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.74.082803.133130&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.74.082803.133130%26sid%3Dliteratum%253Aachs%26aulast%3DYonath%26aufirst%3DA.%26atitle%3DAntibiotics%2520targeting%2520ribosomes%253A%2520resistance%252C%2520selectivity%252C%2520synergism%252C%2520and%2520cellular%2520regulation%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2005%26volume%3D74%26issue%3D1%26spage%3D649%26epage%3D679" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Steitz, T. A.</span><span> </span><span class="NLM_article-title">On the structural basis of peptide-bond formation and antibiotic resistance from atomic structures of the large ribosomal subunit</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">579</span><span class="NLM_x">, </span> <span class="NLM_fpage">955</span><span class="NLM_x">–</span> <span class="NLM_lpage">958</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=579&publication_year=2005&pages=955-958&author=T.+A.+Steitz&title=On+the+structural+basis+of+peptide-bond+formation+and+antibiotic+resistance+from+atomic+structures+of+the+large+ribosomal+subunit"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSteitz%26aufirst%3DT.%2BA.%26atitle%3DOn%2520the%2520structural%2520basis%2520of%2520peptide-bond%2520formation%2520and%2520antibiotic%2520resistance%2520from%2520atomic%2520structures%2520of%2520the%2520large%2520ribosomal%2520subunit%26jtitle%3DFEBS%2520Lett.%26date%3D2005%26volume%3D579%26spage%3D955%26epage%3D958" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Ogle, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramakrishnan, V.</span><span> </span><span class="NLM_article-title">Structural insights into translational fidelity</span> <span class="citation_source-journal">Annu. Rev. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span><span class="NLM_issue">74</span><span class="NLM_x">) </span> <span class="NLM_fpage">129</span><span class="NLM_x">–</span> <span class="NLM_lpage">177</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm9000659&amp;key=10.1146%2Fannurev.biochem.74.061903.155440" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm9000659&amp;key=15952884" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpsVens78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&pages=129-177&issue=74&author=J.+M.+Ogleauthor=V.+Ramakrishnan&title=Structural+insights+into+translational+fidelity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into translational fidelity</span></div><div class="casAuthors">Ogle, James M.; Ramakrishnan, V.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">129-177</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  The underlying basis for the accuracy of protein synthesis has been the subject of >4 decades of investigation.  Recent biochem. and structural data make it possible to understand at least in outline the structural basis for tRNA selection, in which codon recognition by cognate tRNA results in the hydrolysis of GTP by EF-Tu over 75 Å away.  The ribosome recognizes the geometry of codon-anticodon base pairing at the 1st 2 positions but monitors the 3rd, or wobble position, less stringently.  Part of the addnl. binding energy of cognate tRNA is used to induce conformational changes in the ribosome that stabilize a transition state for GTP hydrolysis by EF-Tu and subsequently result in accelerated accommodation of tRNA into the peptidyltransferase center.  The transition state for GTP hydrolysis is characterized, among other things, by a distorted tRNA.  This picture explains a large body of data on the effect of antibiotics and mutations on translational fidelity.  However, many fundamental questions remain, such as the mechanism of activation of GTP hydrolysis by EF-Tu, and the relation between decoding and frameshifting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsYMR5zqFMsrVg90H21EOLACvtfcHk0lhxjVyyNLOVeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpsVens78%253D&md5=a88c18a7164caf7fc3bdb711a855f5a1</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.74.061903.155440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.74.061903.155440%26sid%3Dliteratum%253Aachs%26aulast%3DOgle%26aufirst%3DJ.%2BM.%26aulast%3DRamakrishnan%26aufirst%3DV.%26atitle%3DStructural%2520insights%2520into%2520translational%2520fidelity%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2005%26issue%3D74%26spage%3D129%26epage%3D177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Karn, J.; Gait, M. J.; Churcher, M. J.; Mann, D. A.; Mikaelian, I. P. C.</span><span> </span><span class="NLM_article-title">Control of Human Immunodeficiency Virus Gene Expression by the RNA Binding Proteins Tat and Rev</span>. In  <span class="citation_source-book">RNA−Protein Interactions</span>; <span class="NLM_contrib-group">Nagai, K.; Mattai, I. W.</span>, Eds.; <span class="NLM_publisher-name">IRL Press</span>: <span class="NLM_publisher-loc">Oxford, U.K.</span>,<span class="NLM_x"> </span><span class="NLM_year">1995</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1995&author=J.+Karn&author=M.+J.+Gait&author=M.+J.+Churcher&author=D.+A.+Mann&author=I.+P.+C.+Mikaelianauthor=K.+Nagai&author=I.+W.+Mattai&title=RNA%E2%88%92Protein+Interactions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DKarn%26aufirst%3DJ.%26atitle%3DControl%2520of%2520Human%2520Immunodeficiency%2520Virus%2520Gene%2520Expression%2520by%2520the%2520RNA%2520Binding%2520Proteins%2520Tat%2520and%2520Rev%26btitle%3DRNA%25E2%2588%2592Protein%2520Interactions%26aulast%3DNagai%26aufirst%3DK.%26pub%3DIRL%2520Press%26date%3D1995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Lemon, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honda, M.</span><span> </span><span class="NLM_article-title">Internal ribosome entry sites within the RNA genomes of hepatitis C virus and other flaviviruses</span> <span class="citation_source-journal">Semin. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">274</span><span class="NLM_x">–</span> <span class="NLM_lpage">288</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm9000659&amp;key=10.1006%2Fsmvy.1997.0129" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADyaK1cXitFamtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1997&pages=274-288&author=S.+M.+Lemonauthor=M.+Honda&title=Internal+ribosome+entry+sites+within+the+RNA+genomes+of+hepatitis+C+virus+and+other+flaviviruses"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Internal ribosome entry sites within the RNA genomes of hepatitis C virus and other flaviviruses</span></div><div class="casAuthors">Lemon, Stanley M.; Honda, Masao</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Virology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">274-288</span>CODEN:
                <span class="NLM_cas:coden">SEVIEL</span>;
        ISSN:<span class="NLM_cas:issn">1044-5773</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">A review with 53 refs.  The 5' nontranslated RNAs of hepatitis C virus (HCV) and several other members of the Flaviviridae contain highly structured segments which form internal ribosome entry sites (IRESs).  These cis-active RNA elements direct the cap-independent initiation of translation of the viral polyprotein in assocn. with trans-acting cellular and possibly viral proteins, and thus they play a key role in the replication of the virus.  The structure of the HCV IRES does not resemble that of any picornaviral IRES, and its function is uniquely dependent upon RNA sequence extending 3' of the site of translation initiation as well as structure surrounding the initiator AUG.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpc8r_9CAqZDLVg90H21EOLACvtfcHk0lhxjVyyNLOVeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXitFamtr0%253D&md5=09797ed8f2ac38176377d151086690f1</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1006%2Fsmvy.1997.0129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fsmvy.1997.0129%26sid%3Dliteratum%253Aachs%26aulast%3DLemon%26aufirst%3DS.%2BM.%26aulast%3DHonda%26aufirst%3DM.%26atitle%3DInternal%2520ribosome%2520entry%2520sites%2520within%2520the%2520RNA%2520genomes%2520of%2520hepatitis%2520C%2520virus%2520and%2520other%2520flaviviruses%26jtitle%3DSemin.%2520Virol.%26date%3D1997%26volume%3D8%26spage%3D274%26epage%3D288" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Blackwell, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brinton, M. A.</span><span> </span><span class="NLM_article-title">BHK cell proteins that bind to the 3′ stem-loop structure of the West Nile virus genome RNA</span> <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">5650</span><span class="NLM_x">–</span> <span class="NLM_lpage">5658</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm9000659&amp;key=7637011" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADyaK2MXns1Kiu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=1995&pages=5650-5658&issue=9&author=J.+L.+Blackwellauthor=M.+A.+Brinton&title=BHK+cell+proteins+that+bind+to+the+3%E2%80%B2+stem-loop+structure+of+the+West+Nile+virus+genome+RNA"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">BHK cell proteins that bind to the 3' stem-loop structure of the West Nile virus genome RNA</span></div><div class="casAuthors">Blackwell, Jerry L.; Brinton, Margo A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">5650-8</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The first 83 3' nucleotides of the genome RNA of the flavivirus West Nile encephalitis virus (WNV) form a stable stem-loop (SL) structure which is followed in the genome by a smaller SL.  These 3' structures are highly conserved among divergent flaviviruses, suggesting that they may function as cis-acting signals for RNA replication and as such might specifically bind to cellular or viral proteins.  Cellular proteins from uninfected and WNV-infected BHK-21 S100 cytoplasmic exts. formed 3 distinct complexes with the WNV plus-strand 3' SL [(+)3'SL] RNA in a gel mobility shift assay.  Subsequent competitor gel shift analyses showed that 2 of these RNA-protein complexes, complexes 1 and 2, contained cell proteins that specifically bound to the WNV (+)3'SL RNA.  UV-induced crosslinking and Northwestern blotting analyses detected WNV (+)3'SL RNA-binding proteins of 56, 84, and 105 kDa.  When the S100 cytoplasmic exts. were partially purified by ion-exchange chromatog., a complex that comigrated with complex 1 was detected in fraction 19, whereas a complex that comigrated with complex 2 was detected in fraction 17.  UV-induced crosslinking expts. indicated that an 84-kDa cell protein in fraction 17 and a 105-kDa protein in fraction 19 bound specifically to the WNV (+)3'SL RNA.  In addn. to binding to the (+)3'SL RNA, the 105-kDa protein bound to the SL structure located at the 3' end of the WNV minus-strand RNA.  Initial mapping studies indicated that the 84- and 105-kDa proteins bind to different regions of the (+)3'SL RNA.  The 3'-terminal SL RNA of another flavivirus, dengue virus type 3, specifically competed with the WNV (+)3'SL RNA in gel shift assays, suggesting that the host proteins identified in this study are flavivirus specific.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq19kNf60fuurVg90H21EOLACvtfcHk0lhxjVyyNLOVeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXns1Kiu7o%253D&md5=7e4641eddbf9052d1d4799d9350dfcf9</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBlackwell%26aufirst%3DJ.%2BL.%26aulast%3DBrinton%26aufirst%3DM.%2BA.%26atitle%3DBHK%2520cell%2520proteins%2520that%2520bind%2520to%2520the%25203%25E2%2580%25B2%2520stem-loop%2520structure%2520of%2520the%2520West%2520Nile%2520virus%2520genome%2520RNA%26jtitle%3DJ.%2520Virol.%26date%3D1995%26volume%3D69%26issue%3D9%26spage%3D5650%26epage%3D5658" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Amaral, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dinger, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mercer, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattick, J. S.</span><span> </span><span class="NLM_article-title">The eukaryotic genome as an RNA machine</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">319</span><span class="NLM_x"> (</span><span class="NLM_issue">5871</span><span class="NLM_x">) </span> <span class="NLM_fpage">1787</span><span class="NLM_x">–</span> <span class="NLM_lpage">1789</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=319&publication_year=2008&pages=1787-1789&issue=5871&author=P.+P.+Amaralauthor=M.+E.+Dingerauthor=T.+R.+Mercerauthor=J.+S.+Mattick&title=The+eukaryotic+genome+as+an+RNA+machine"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAmaral%26aufirst%3DP.%2BP.%26aulast%3DDinger%26aufirst%3DM.%2BE.%26aulast%3DMercer%26aufirst%3DT.%2BR.%26aulast%3DMattick%26aufirst%3DJ.%2BS.%26atitle%3DThe%2520eukaryotic%2520genome%2520as%2520an%2520RNA%2520machine%26jtitle%3DScience%26date%3D2008%26volume%3D319%26issue%3D5871%26spage%3D1787%26epage%3D1789" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Kumari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bugaut, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huppert, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">An RNA G-quadruplex in the 5′ UTR of the NRAS proto-oncogene modulates translation</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">218</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Fjm9000659&amp;key=10.1038%2Fnchembio864" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Fjm9000659&amp;key=17322877" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjtFKnsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=218-221&issue=4&author=S.+Kumariauthor=A.+Bugautauthor=J.+L.+Huppertauthor=S.+Balasubramanian&title=An+RNA+G-quadruplex+in+the+5%E2%80%B2+UTR+of+the+NRAS+proto-oncogene+modulates+translation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">An RNA G-quadruplex in the 5' UTR of the NRAS proto-oncogene modulates translation</span></div><div class="casAuthors">Kumari, Sunita; Bugaut, Anthony; Huppert, Julian L.; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">218-221</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Guanine-rich nucleic acid sequences can adopt noncanonical four-stranded secondary structures called guanine (G)-quadruplexes.  Bioinformatics anal. suggests that G-quadruplex motifs are prevalent in genomes, which raises the need to elucidate their function.  There is now evidence for the existence of DNA G-quadruplexes at telomeres with assocd. biol. function.  A recent hypothesis supports the notion that gene promoter elements contain DNA G-quadruplex motifs that control gene expression at the transcriptional level.  We discovered a highly conserved, thermodynamically stable RNA G-quadruplex in the 5' untranslated region (UTR) of the gene transcript of the human NRAS proto-oncogene.  Using a cell-free translation system coupled to a reporter gene assay, we have demonstrated that this NRAS RNA G-quadruplex modulates translation.  This is the first example of translational repression by an RNA G-quadruplex.  Bioinformatics anal. has revealed 2,922 other 5' UTR RNA G-quadruplex elements in the human genome.  We propose that RNA G-quadruplexes in the 5' UTR modulate gene expression at the translational level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUZXUhKbJ4cLVg90H21EOLACvtfcHk0liBESAPQbHJAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjtFKnsrw%253D&md5=c4dda371d46b95d25b285d1adeb599be</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnchembio864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio864%26sid%3Dliteratum%253Aachs%26aulast%3DKumari%26aufirst%3DS.%26aulast%3DBugaut%26aufirst%3DA.%26aulast%3DHuppert%26aufirst%3DJ.%2BL.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DAn%2520RNA%2520G-quadruplex%2520in%2520the%25205%25E2%2580%25B2%2520UTR%2520of%2520the%2520NRAS%2520proto-oncogene%2520modulates%2520translation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2007%26volume%3D3%26issue%3D4%26spage%3D218%26epage%3D221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Hagerman, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rivera, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hagerman, P. J.</span><span> </span><span class="NLM_article-title">The fragile X family of disorders: a model for autism and targeted treatments</span> <span class="citation_source-journal">Curr. Pediatr. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">40</span><span class="NLM_x">–</span> <span class="NLM_lpage">52</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm9000659&amp;key=10.2174%2F157339608783565770" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlsleltbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=40-52&author=R.+J.+Hagermanauthor=S.+M.+Riveraauthor=P.+J.+Hagerman&title=The+fragile+X+family+of+disorders%3A+a+model+for+autism+and+targeted+treatments"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">The fragile X family of disorders: a model for autism and targeted treatments</span></div><div class="casAuthors">Hagerman, Randi J.; Rivera, Susan M.; Hagerman, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Pediatric Reviews</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">40-52</span>CODEN:
                <span class="NLM_cas:coden">CPRUC4</span>;
        ISSN:<span class="NLM_cas:issn">1875-6336</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  CGG-repeat expansion mutations of the fragile X mental retardation 1 (FMR1) gene are the leading known cause of autism and autism spectrum disorders (ASD).  Full mutation expansions (>200 CGG repeats) of the gene are generally silenced, resulting in absence of the FMR1 protein and fragile X syndrome.  By contrast, smaller expansions in the premutation range (55-200 CGG repeats) result in excess gene activity and RNA toxicity, which is responsible for the neurodegenerative disorder, fragile X-assocd. tremor/ataxia syndrome (FXTAS), and likely addnl. cases of developmental delay and autism.  Thus, the FMR1 gene is causative of a common (autism) phenotype via two entirely different pathogenic mechanisms, RNA toxicity and gene silencing.  The study of this gene and its pathogenic mechanisms therefore represents a paradigm for understanding gene-brain relationships and the means by which diverse genetic mechanisms can give rise to a common behavioral phenotype.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdvix_7PzRS7Vg90H21EOLACvtfcHk0liBESAPQbHJAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlsleltbs%253D&md5=5722cff1ef704b32d41dfe0588120acc</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.2174%2F157339608783565770&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F157339608783565770%26sid%3Dliteratum%253Aachs%26aulast%3DHagerman%26aufirst%3DR.%2BJ.%26aulast%3DRivera%26aufirst%3DS.%2BM.%26aulast%3DHagerman%26aufirst%3DP.%2BJ.%26atitle%3DThe%2520fragile%2520X%2520family%2520of%2520disorders%253A%2520a%2520model%2520for%2520autism%2520and%2520targeted%2520treatments%26jtitle%3DCurr.%2520Pediatr.%2520Rev.%26date%3D2008%26volume%3D4%26spage%3D40%26epage%3D52" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Shay, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, W. E.</span><span> </span><span class="NLM_article-title">Telomerase therapeutics for cancer: challenges and new directions</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">577</span><span class="NLM_x">–</span> <span class="NLM_lpage">584</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=577-584&issue=7&author=J.+W.+Shayauthor=W.+E.+Wright&title=Telomerase+therapeutics+for+cancer%3A+challenges+and+new+directions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DShay%26aufirst%3DJ.%2BW.%26aulast%3DWright%26aufirst%3DW.%2BE.%26atitle%3DTelomerase%2520therapeutics%2520for%2520cancer%253A%2520challenges%2520and%2520new%2520directions%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26issue%3D7%26spage%3D577%26epage%3D584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Burgstaller, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Famulok, M.</span><span> </span><span class="NLM_article-title">Flavin-dependent photocleavage of RNA at G*U base pairs</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">1137</span><span class="NLM_x">–</span> <span class="NLM_lpage">1138</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja962918p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=1997&pages=1137-1138&issue=5&author=P.+Burgstallerauthor=M.+Famulok&title=Flavin-dependent+photocleavage+of+RNA+at+G*U+base+pairs"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fja962918p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja962918p%26sid%3Dliteratum%253Aachs%26aulast%3DBurgstaller%26aufirst%3DP.%26aulast%3DFamulok%26aufirst%3DM.%26atitle%3DFlavin-dependent%2520photocleavage%2520of%2520RNA%2520at%2520G%252AU%2520base%2520pairs%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1997%26volume%3D119%26issue%3D5%26spage%3D1137%26epage%3D1138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Burgstaller, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hermann, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huber, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westhof, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Famulok, M.</span><span> </span><span class="NLM_article-title">Isoalloxazine derivatives promote photocleavage of natural RNAs at G.U base pairs embedded within helices</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x"> (</span><span class="NLM_issue">20</span><span class="NLM_x">) </span> <span class="NLM_fpage">4018</span><span class="NLM_x">–</span> <span class="NLM_lpage">4027</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1997&pages=4018-4027&issue=20&author=P.+Burgstallerauthor=T.+Hermannauthor=C.+Huberauthor=E.+Westhofauthor=M.+Famulok&title=Isoalloxazine+derivatives+promote+photocleavage+of+natural+RNAs+at+G.U+base+pairs+embedded+within+helices"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBurgstaller%26aufirst%3DP.%26aulast%3DHermann%26aufirst%3DT.%26aulast%3DHuber%26aufirst%3DC.%26aulast%3DWesthof%26aufirst%3DE.%26aulast%3DFamulok%26aufirst%3DM.%26atitle%3DIsoalloxazine%2520derivatives%2520promote%2520photocleavage%2520of%2520natural%2520RNAs%2520at%2520G.U%2520base%2520pairs%2520embedded%2520within%2520helices%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1997%26volume%3D25%26issue%3D20%26spage%3D4018%26epage%3D4027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Murchie, A. I. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isel, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afshar, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drysdale, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bower, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Potter, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Starkey, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swarbrick, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mirza, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prescott, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vaglio, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aboul-ela, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karn, J.</span><span> </span><span class="NLM_article-title">Structure-based drug design targeting an inactive RNA conformation: exploiting the flexibility of HIV-1 TAR RNA</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">336</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">625</span><span class="NLM_x">–</span> <span class="NLM_lpage">638</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=336&publication_year=2004&pages=625-638&issue=3&author=A.+I.+H.+Murchieauthor=B.+Davisauthor=C.+Iselauthor=M.+Afsharauthor=M.+J.+Drysdaleauthor=J.+Bowerauthor=A.+J.+Potterauthor=I.+D.+Starkeyauthor=T.+M.+Swarbrickauthor=S.+Mirzaauthor=C.+D.+Prescottauthor=P.+Vaglioauthor=F.+Aboul-elaauthor=J.+Karn&title=Structure-based+drug+design+targeting+an+inactive+RNA+conformation%3A+exploiting+the+flexibility+of+HIV-1+TAR+RNA"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMurchie%26aufirst%3DA.%2BI.%2BH.%26aulast%3DDavis%26aufirst%3DB.%26aulast%3DIsel%26aufirst%3DC.%26aulast%3DAfshar%26aufirst%3DM.%26aulast%3DDrysdale%26aufirst%3DM.%2BJ.%26aulast%3DBower%26aufirst%3DJ.%26aulast%3DPotter%26aufirst%3DA.%2BJ.%26aulast%3DStarkey%26aufirst%3DI.%2BD.%26aulast%3DSwarbrick%26aufirst%3DT.%2BM.%26aulast%3DMirza%26aufirst%3DS.%26aulast%3DPrescott%26aufirst%3DC.%2BD.%26aulast%3DVaglio%26aufirst%3DP.%26aulast%3DAboul-ela%26aufirst%3DF.%26aulast%3DKarn%26aufirst%3DJ.%26atitle%3DStructure-based%2520drug%2520design%2520targeting%2520an%2520inactive%2520RNA%2520conformation%253A%2520exploiting%2520the%2520flexibility%2520of%2520HIV-1%2520TAR%2520RNA%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2004%26volume%3D336%26issue%3D3%26spage%3D625%26epage%3D638" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Davis, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afshar, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murchie, A. I. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karn, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lentzen, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drysdale, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bower, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Potter, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Starkey, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swarbrick, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aboul-ela, F.</span><span> </span><span class="NLM_article-title">Rational design of inhibtors of HIV-1 TAR RNA through the stabilisation of electrostatic “hot spots”</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">336</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">343</span><span class="NLM_x">–</span> <span class="NLM_lpage">356</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=336&publication_year=2004&pages=343-356&issue=2&author=B.+Davisauthor=M.+Afsharauthor=G.+Varaniauthor=A.+I.+H.+Murchieauthor=J.+Karnauthor=G.+Lentzenauthor=M.+Drysdaleauthor=J.+Bowerauthor=A.+J.+Potterauthor=I.+D.+Starkeyauthor=T.+Swarbrickauthor=F.+Aboul-ela&title=Rational+design+of+inhibtors+of+HIV-1+TAR+RNA+through+the+stabilisation+of+electrostatic+%E2%80%9Chot+spots%E2%80%9D"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DB.%26aulast%3DAfshar%26aufirst%3DM.%26aulast%3DVarani%26aufirst%3DG.%26aulast%3DMurchie%26aufirst%3DA.%2BI.%2BH.%26aulast%3DKarn%26aufirst%3DJ.%26aulast%3DLentzen%26aufirst%3DG.%26aulast%3DDrysdale%26aufirst%3DM.%26aulast%3DBower%26aufirst%3DJ.%26aulast%3DPotter%26aufirst%3DA.%2BJ.%26aulast%3DStarkey%26aufirst%3DI.%2BD.%26aulast%3DSwarbrick%26aufirst%3DT.%26aulast%3DAboul-ela%26aufirst%3DF.%26atitle%3DRational%2520design%2520of%2520inhibtors%2520of%2520HIV-1%2520TAR%2520RNA%2520through%2520the%2520stabilisation%2520of%2520electrostatic%2520%25E2%2580%259Chot%2520spots%25E2%2580%259D%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2004%26volume%3D336%26issue%3D2%26spage%3D343%26epage%3D356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Du, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lind, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, T. L.</span><span> </span><span class="NLM_article-title">Structure of TAR RNA complexed with a Tat-TAR interaction nanomolar inhibitor that was identified by computational screening</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">707</span><span class="NLM_x">–</span> <span class="NLM_lpage">712</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Fjm9000659&amp;key=10.1016%2FS1074-5521%2802%2900151-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Fjm9000659&amp;key=12079782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADC%252BD38XksFOisLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2002&pages=707-712&author=Z.+Duauthor=K.+E.+Lindauthor=T.+L.+James&title=Structure+of+TAR+RNA+complexed+with+a+Tat-TAR+interaction+nanomolar+inhibitor+that+was+identified+by+computational+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of TAR RNA Complexed with a Tat-TAR: Interaction Nanomolar Inhibitor that Was Identified by Computational Screening</span></div><div class="casAuthors">Du, Zhihua; Lind, Kenneth E.; James, Thomas L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">707-712</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">HIV-1 TAR RNA functions critically in viral replication by binding the transactivating regulatory protein Tat.  We recently identified several compds. that exptl. inhibit the Tat-TAR interaction completely at a 100 nM concn.  We used computational screening of the 181,000-compd. Available Chems. Directory against the three-dimensional structure of TAR.  Here we report the NMR-derived structure of TAR complexed with acetylpromazine.  This structure represents a new class of compds. with good bioavailability and low toxicity that bind with high affinity to TAR.  NMR data unambiguously show that acetylpromazine binds only to the unique 5' bulge site to which the Tat protein binds.  Specificity and affinity of binding are conferred primarily by a network of base stacking and hydrophobic interactions.  Acetylpromazine alters the structure of free TAR less than Tat peptides and neomycin do.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpc1TG4IQ3rzLVg90H21EOLACvtfcHk0likRTrxvmUhCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XksFOisLs%253D&md5=97f7881f6b358888c9144b69c8149163</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2FS1074-5521%2802%2900151-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1074-5521%252802%252900151-5%26sid%3Dliteratum%253Aachs%26aulast%3DDu%26aufirst%3DZ.%26aulast%3DLind%26aufirst%3DK.%2BE.%26aulast%3DJames%26aufirst%3DT.%2BL.%26atitle%3DStructure%2520of%2520TAR%2520RNA%2520complexed%2520with%2520a%2520Tat-TAR%2520interaction%2520nanomolar%2520inhibitor%2520that%2520was%2520identified%2520by%2520computational%2520screening%26jtitle%3DChem.%2520Biol.%26date%3D2002%26volume%3D9%26spage%3D707%26epage%3D712" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Hamy, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brondani, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Florsheimer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stark, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blommers, M. J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klimkait, T.</span><span> </span><span class="NLM_article-title">A new class of HIV-1 Tat antagonist acting through Tat-TAR inhibition</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">5086</span><span class="NLM_x">–</span> <span class="NLM_lpage">5095</span></span><div class="citationLinks">[<a href="/doi/10.1021/bi972947s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1998&pages=5086-5095&author=F.+Hamyauthor=V.+Brondaniauthor=A.+Florsheimerauthor=W.+Starkauthor=M.+J.+J.+Blommersauthor=T.+Klimkait&title=A+new+class+of+HIV-1+Tat+antagonist+acting+through+Tat-TAR+inhibition"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fbi972947s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi972947s%26sid%3Dliteratum%253Aachs%26aulast%3DHamy%26aufirst%3DF.%26aulast%3DBrondani%26aufirst%3DV.%26aulast%3DFlorsheimer%26aufirst%3DA.%26aulast%3DStark%26aufirst%3DW.%26aulast%3DBlommers%26aufirst%3DM.%2BJ.%2BJ.%26aulast%3DKlimkait%26aufirst%3DT.%26atitle%3DA%2520new%2520class%2520of%2520HIV-1%2520Tat%2520antagonist%2520acting%2520through%2520Tat-TAR%2520inhibition%26jtitle%3DBiochemistry%26date%3D1998%26volume%3D37%26spage%3D5086%26epage%3D5095" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Zhou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gregor, V. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ayida, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winters, G. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haley, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bailey, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Froelich, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fish, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webber, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hermann, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wall, D.</span><span> </span><span class="NLM_article-title">Synthesis and SAR of 3,5-diamino-piperidine derivatives: novel antibacterial translation inhibitors as aminoglycoside mimetics</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">1206</span><span class="NLM_x">–</span> <span class="NLM_lpage">1210</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=1206-1210&issue=5&author=Y.+Zhouauthor=V.+E.+Gregorauthor=B.+K.+Ayidaauthor=G.+C.+Wintersauthor=Z.+Sunauthor=D.+Murphyauthor=G.+Haleyauthor=D.+Baileyauthor=J.+M.+Froelichauthor=S.+Fishauthor=S.+E.+Webberauthor=T.+Hermannauthor=D.+Wall&title=Synthesis+and+SAR+of+3%2C5-diamino-piperidine+derivatives%3A+novel+antibacterial+translation+inhibitors+as+aminoglycoside+mimetics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DGregor%26aufirst%3DV.%2BE.%26aulast%3DAyida%26aufirst%3DB.%2BK.%26aulast%3DWinters%26aufirst%3DG.%2BC.%26aulast%3DSun%26aufirst%3DZ.%26aulast%3DMurphy%26aufirst%3DD.%26aulast%3DHaley%26aufirst%3DG.%26aulast%3DBailey%26aufirst%3DD.%26aulast%3DFroelich%26aufirst%3DJ.%2BM.%26aulast%3DFish%26aufirst%3DS.%26aulast%3DWebber%26aufirst%3DS.%2BE.%26aulast%3DHermann%26aufirst%3DT.%26aulast%3DWall%26aufirst%3DD.%26atitle%3DSynthesis%2520and%2520SAR%2520of%25203%252C5-diamino-piperidine%2520derivatives%253A%2520novel%2520antibacterial%2520translation%2520inhibitors%2520as%2520aminoglycoside%2520mimetics%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2007%26volume%3D17%26issue%3D5%26spage%3D1206%26epage%3D1210" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Franceschi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duffy, E. M.</span><span> </span><span class="NLM_article-title">Structure-based drug design meets the ribosome</span> <span class="citation_source-journal">Biochem. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">71</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">1016</span><span class="NLM_x">–</span> <span class="NLM_lpage">1025</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2006&pages=1016-1025&issue=7&author=F.+Franceschiauthor=E.+M.+Duffy&title=Structure-based+drug+design+meets+the+ribosome"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFranceschi%26aufirst%3DF.%26aulast%3DDuffy%26aufirst%3DE.%2BM.%26atitle%3DStructure-based%2520drug%2520design%2520meets%2520the%2520ribosome%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2006%26volume%3D71%26issue%3D7%26spage%3D1016%26epage%3D1025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Kung, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casper, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson-Lingardo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Risen, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vickers, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ranken, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blyn, L. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyatt, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ecker, D. J.</span><span> </span><span class="NLM_article-title">Structure−activity relationships of novel 2-substituted quinazoline antibacterial agents</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x"> (</span><span class="NLM_issue">22</span><span class="NLM_x">) </span> <span class="NLM_fpage">4705</span><span class="NLM_x">–</span> <span class="NLM_lpage">4713</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm9903500" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=4705-4713&issue=22&author=P.+P.+Kungauthor=M.+D.+Casperauthor=K.+L.+Cookauthor=L.+Wilson-Lingardoauthor=L.+M.+Risenauthor=T.+A.+Vickersauthor=R.+Rankenauthor=L.+B.+Blynauthor=J.+R.+Wyattauthor=P.+D.+Cookauthor=D.+J.+Ecker&title=Structure%E2%88%92activity+relationships+of+novel+2-substituted+quinazoline+antibacterial+agents"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm9903500&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9903500%26sid%3Dliteratum%253Aachs%26aulast%3DKung%26aufirst%3DP.%2BP.%26aulast%3DCasper%26aufirst%3DM.%2BD.%26aulast%3DCook%26aufirst%3DK.%2BL.%26aulast%3DWilson-Lingardo%26aufirst%3DL.%26aulast%3DRisen%26aufirst%3DL.%2BM.%26aulast%3DVickers%26aufirst%3DT.%2BA.%26aulast%3DRanken%26aufirst%3DR.%26aulast%3DBlyn%26aufirst%3DL.%2BB.%26aulast%3DWyatt%26aufirst%3DJ.%2BR.%26aulast%3DCook%26aufirst%3DP.%2BD.%26aulast%3DEcker%26aufirst%3DD.%2BJ.%26atitle%3DStructure%25E2%2588%2592activity%2520relationships%2520of%2520novel%25202-substituted%2520quinazoline%2520antibacterial%2520agents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26issue%3D22%26spage%3D4705%26epage%3D4713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Morley, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afshar, M.</span><span> </span><span class="NLM_article-title">Validation of an empirical RNA−ligand scoring function for fast flexible docking using RiboDock</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">189</span><span class="NLM_x">–</span> <span class="NLM_lpage">208</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2003&pages=189-208&issue=3&author=S.+D.+Morleyauthor=M.+Afshar&title=Validation+of+an+empirical+RNA%E2%88%92ligand+scoring+function+for+fast+flexible+docking+using+RiboDock"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMorley%26aufirst%3DS.%2BD.%26aulast%3DAfshar%26aufirst%3DM.%26atitle%3DValidation%2520of%2520an%2520empirical%2520RNA%25E2%2588%2592ligand%2520scoring%2520function%2520for%2520fast%2520flexible%2520docking%2520using%2520RiboDock%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2003%26volume%3D18%26issue%3D3%26spage%3D189%26epage%3D208" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Lind, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujinaga, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterlin, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, T. L.</span><span> </span><span class="NLM_article-title">Structure-based computational database screening, in vitro assay, and NMR assessment of compounds that target RNA</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">185</span><span class="NLM_x">–</span> <span class="NLM_lpage">193</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm9000659&amp;key=10.1016%2FS1074-5521%2802%2900106-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Fjm9000659&amp;key=11880033" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADC%252BD38Xhs1yltLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2002&pages=185-193&author=K.+E.+Lindauthor=Z.+Duauthor=K.+Fujinagaauthor=B.+M.+Peterlinauthor=T.+L.+James&title=Structure-based+computational+database+screening%2C+in+vitro+assay%2C+and+NMR+assessment+of+compounds+that+target+RNA"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Computational Database Screening, In Vitro Assay, and NMR Assessment of Compounds that Target TAR RNA</span></div><div class="casAuthors">Lind, Kenneth E.; Du, Zhihua; Fujinaga, Koh; Peterlin, B. Matija; James, Thomas L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">185-193</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">There has been little prior effort to discover new drugs on the basis of a unique RNA structure.  Binding of the viral transactivator Tat to the 5' bulge of the transactivation response (TAR) element is necessary for HIV-1 replication, so TAR RNA is a superb target.  A computational approach was developed to screen a large chem. library for binding to a three-dimensional RNA structure.  Scoring function development, flexible ligand docking, and limited target flexibility were essential.  From the ranked list of compds. predicted to bind TAR, 43 were assayed for inhibition of the Tat-TAR interaction via electrophoretic mobility shift assays.  Eleven compds. (between 0.1 and 1 μM) inhibited the Tat-TAR interaction, and some inhibited Tat transactivation in cells.  NMR spectra verified specific binding to the 5' bulge and no interaction with other regions of TAR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptBQXlpQMPSrVg90H21EOLACvtfcHk0lgw8WiZRpbKmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xhs1yltLg%253D&md5=6aefd491cbfcd6fcd98706015df02632</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2FS1074-5521%2802%2900106-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1074-5521%252802%252900106-0%26sid%3Dliteratum%253Aachs%26aulast%3DLind%26aufirst%3DK.%2BE.%26aulast%3DDu%26aufirst%3DZ.%26aulast%3DFujinaga%26aufirst%3DK.%26aulast%3DPeterlin%26aufirst%3DB.%2BM.%26aulast%3DJames%26aufirst%3DT.%2BL.%26atitle%3DStructure-based%2520computational%2520database%2520screening%252C%2520in%2520vitro%2520assay%252C%2520and%2520NMR%2520assessment%2520of%2520compounds%2520that%2520target%2520RNA%26jtitle%3DChem.%2520Biol.%26date%3D2002%26volume%3D9%26spage%3D185%26epage%3D193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Detering, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varani, G.</span><span> </span><span class="NLM_article-title">Validation of automated docking programs for docking and database screening against RNA drug targets</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x"> (</span><span class="NLM_issue">17</span><span class="NLM_x">) </span> <span class="NLM_fpage">4188</span><span class="NLM_x">–</span> <span class="NLM_lpage">4201</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm030650o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=4188-4201&issue=17&author=C.+Deteringauthor=G.+Varani&title=Validation+of+automated+docking+programs+for+docking+and+database+screening+against+RNA+drug+targets"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm030650o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030650o%26sid%3Dliteratum%253Aachs%26aulast%3DDetering%26aufirst%3DC.%26aulast%3DVarani%26aufirst%3DG.%26atitle%3DValidation%2520of%2520automated%2520docking%2520programs%2520for%2520docking%2520and%2520database%2520screening%2520against%2520RNA%2520drug%2520targets%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26issue%3D17%26spage%3D4188%26epage%3D4201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Guilbert, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, T. L.</span><span> </span><span class="NLM_article-title">Docking to RNA via root-mean-square-deviation-driven energy minimization with flexible ligands and flexible targets</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">1257</span><span class="NLM_x">–</span> <span class="NLM_lpage">1268</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci8000327" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmsVOhurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2008&pages=1257-1268&issue=6&author=C.+Guilbertauthor=T.+L.+James&title=Docking+to+RNA+via+root-mean-square-deviation-driven+energy+minimization+with+flexible+ligands+and+flexible+targets"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Docking to RNA via Root-Mean-Square-Deviation-Driven Energy Minimization with Flexible Ligands and Flexible Targets</span></div><div class="casAuthors">Guilbert, Christophe; James, Thomas L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1257-1268</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structure-based drug design is now well-established for proteins as a key first step in the lengthy process of developing new drugs.  In many ways, RNA may be a better target to treat disease than a protein because it is upstream in the translation pathway, so inhibiting a single mRNA mol. could prevent the prodn. of thousands of protein gene products.  Virtual screening is often the starting point for structure-based drug design.  However, computational docking of a small mol. to RNA seems to be more challenging than that to protein due to the higher intrinsic flexibility and highly charged structure of RNA.  Previous attempts at docking to RNA showed the need for a new approach.  We present here a novel algorithm using mol. simulation techniques to account for both nucleic acid and ligand flexibility.  In this approach, with both the ligand and the receptor permitted some flexibility, they can bind one another via an induced fit, as the flexible ligand probes the surface of the receptor.  A possible ligand can explore a low-energy path at the surface of the receptor by carrying out energy minimization with root-mean-square-distance constraints.  Our procedure was tested on 57 RNA complexes (33 crystal and 24 NMR structures); this is the largest data set to date to reproduce exptl. RNA binding poses.  With our procedure, the lowest-energy conformations reproduced the exptl. binding poses within an at. root-mean-square deviation of 2.5 Å for 74% of tested complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPPar0XY4iPrVg90H21EOLACvtfcHk0liHn9RXxfyvzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmsVOhurc%253D&md5=451852050ffc81c000ec95f5e496adcd</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fci8000327&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci8000327%26sid%3Dliteratum%253Aachs%26aulast%3DGuilbert%26aufirst%3DC.%26aulast%3DJames%26aufirst%3DT.%2BL.%26atitle%3DDocking%2520to%2520RNA%2520via%2520root-mean-square-deviation-driven%2520energy%2520minimization%2520with%2520flexible%2520ligands%2520and%2520flexible%2520targets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2008%26volume%3D48%26issue%3D6%26spage%3D1257%26epage%3D1268" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Hubbard, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drysdale, M. J.</span><span> </span><span class="NLM_article-title">The SeeDs approach: integrating fragments into drug discovery</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x"> (</span><span class="NLM_issue">16</span><span class="NLM_x">) </span> <span class="NLM_fpage">1568</span><span class="NLM_x">–</span> <span class="NLM_lpage">1581</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm9000659&amp;key=10.2174%2F156802607782341109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Fjm9000659&amp;key=17979768" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVOrur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=1568-1581&issue=16&author=R.+E.+Hubbardauthor=B.+Davisauthor=I.+Chenauthor=M.+J.+Drysdale&title=The+SeeDs+approach%3A+integrating+fragments+into+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">The SeeDs approach: integrating fragments into drug discovery</span></div><div class="casAuthors">Hubbard, Roderick E.; Davis, Ben; Chen, Ijen; Drysdale, Martin J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1568-1581</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Finding novel compds. as starting points for optimization is a major challenge in drug discovery research.  Fragment-based methods have emerged in the past ten years as an effective way to sample chem. diversity with a limited no. of low mol. wt. compds.  The structures of the fragments(s) binding to the protein can then be used to design new compds. with increased affinity, specificity and novelty.  This article describes the Vernalis approach to fragment based drug discovery, called SeeDs (Structural exploitation of exptl. Drug startpoints).  The approach includes the design of a fragment library, identification of fragments that bind competitively to a target by ligand-based NMR techniques and protein crystal structures to characterize binding.  Fragments that bind are then evolved to hits, either by growing the fragment or by combining structural features from a no. of compds.  The process is illustrated with examples from recent medicinal chem. programs to discover compds. against the oncol. targets Hsp90 and PDK1.  In addn., we summarize our experience with using mol. docking calcns. to predict fragment binding and anecdotes on the selectivity and binding modes for fragments seen against a range of targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGox6z6oFlEi67Vg90H21EOLACvtfcHk0liHn9RXxfyvzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVOrur7J&md5=6bfa0109e33c2c23cc4c5a5bcd1ed12f</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.2174%2F156802607782341109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802607782341109%26sid%3Dliteratum%253Aachs%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DDavis%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DI.%26aulast%3DDrysdale%26aufirst%3DM.%2BJ.%26atitle%3DThe%2520SeeDs%2520approach%253A%2520integrating%2520fragments%2520into%2520drug%2520discovery%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2007%26volume%3D7%26issue%3D16%26spage%3D1568%26epage%3D1581" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greer, J.</span><span> </span><span class="NLM_article-title">A decade of fragment-based drug design: strategic advances and lessons learned</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">211</span><span class="NLM_x">–</span> <span class="NLM_lpage">219</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2007&pages=211-219&issue=3&author=P.+J.+Hajdukauthor=J.+Greer&title=A+decade+of+fragment-based+drug+design%3A+strategic+advances+and+lessons+learned"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DGreer%26aufirst%3DJ.%26atitle%3DA%2520decade%2520of%2520fragment-based%2520drug%2520design%253A%2520strategic%2520advances%2520and%2520lessons%2520learned%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D5%26issue%3D3%26spage%3D211%26epage%3D219" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Jhoti, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cleasby, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdonk, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, G.</span><span> </span><span class="NLM_article-title">Fragment-based screening using X-ray crystallography and NMR spectroscopy</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">485</span><span class="NLM_x">–</span> <span class="NLM_lpage">493</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Fjm9000659&amp;key=10.1016%2Fj.cbpa.2007.07.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Fjm9000659&amp;key=17851109" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtF2rsb3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2007&pages=485-493&issue=5&author=H.+Jhotiauthor=A.+Cleasbyauthor=M.+Verdonkauthor=G.+Williams&title=Fragment-based+screening+using+X-ray+crystallography+and+NMR+spectroscopy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based screening using X-ray crystallography and NMR spectroscopy</span></div><div class="casAuthors">Jhoti, Harren; Cleasby, Anne; Verdonk, Marcel; Williams, Glyn</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">485-493</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Approaches which start from a study of the interaction of very simple mols. (fragments) with the protein target are proving to be valuable addns. to drug design.  Fragment-based screening allows the complementarity between a protein active site and drug-like mols. to be rapidly and effectively explored, using structural methods.  Recent improvements in the intensities of lab. x-ray sources permits the collection of greater amts. of high-quality diffraction data and were matched by developments in automation, crystn. and data anal.  Developments in NMR screening, including the use of cryogenically cooled NMR probes and 19F-contg. reporter mols. have expanded the scope of this technique, while increasing the availability of binding site and quant. affinity data for the fragments.  Application of these methods has led to a greater knowledge of the chem. variety, structural features and energetics of protein-fragment interactions.  While fragment-based screening has already been shown to reduce the timescales of the drug discovery process, a more detailed characterization of fragment screening hits can reveal unexpected similarities between fragment chemotypes and protein active sites leading to improved understanding of the pharmacophores and the re-use of this information against other protein targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFTfwnJaPUpLVg90H21EOLACvtfcHk0liHn9RXxfyvzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtF2rsb3E&md5=5a4e431308fe3a8d435cea55960dc663</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2007.07.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2007.07.010%26sid%3Dliteratum%253Aachs%26aulast%3DJhoti%26aufirst%3DH.%26aulast%3DCleasby%26aufirst%3DA.%26aulast%3DVerdonk%26aufirst%3DM.%26aulast%3DWilliams%26aufirst%3DG.%26atitle%3DFragment-based%2520screening%2520using%2520X-ray%2520crystallography%2520and%2520NMR%2520spectroscopy%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2007%26volume%3D11%26issue%3D5%26spage%3D485%26epage%3D493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Fejzo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lepre, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bemis, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ajay; Murcko, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, J. M.</span><span> </span><span class="NLM_article-title">The SHAPES strategy: an NMR-based approach for lead generation in drug discovery</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">755</span><span class="NLM_x">–</span> <span class="NLM_lpage">769</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Fjm9000659&amp;key=10.1016%2FS1074-5521%2800%2980022-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Fjm9000659&amp;key=10508679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADyaK1MXmvVOns7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1999&pages=755-769&issue=10&author=J.+Fejzoauthor=C.+A.+Lepreauthor=J.+W.+Pengauthor=G.+W.+Bemisauthor=M.+A.+Ajay%3B+Murckoauthor=J.+M.+Moore&title=The+SHAPES+strategy%3A+an+NMR-based+approach+for+lead+generation+in+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The SHAPES strategy: an NMR-based approach for lead generation in drug discovery</span></div><div class="casAuthors">Fejzo, Jasna; Lepre, Christopher A.; Peng, Jeffrey W.; Bemis, Guy W.; Ajay; Murcko, Mark A.; Moore, Jonathan M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">755-769</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Current Biology Publications</span>)
        </div><div class="casAbstract">Background: Recently, it has been shown that NMR may be used to identify ligands that bind to low mol. wt. protein drug targets.  Recognizing the utility of NMR as a very sensitive method for detecting binding, we have focused on developing alternative approaches that are applicable to larger mol. wt. drug targets and do not require isotopic labeling.  Results: A new method for lead generation (SHAPES) is described that uses NMR to detect binding of a limited but diverse library of small mols. to a potential drug target.  The compd. scaffolds are derived from shapes most commonly found in known therapeutic agents.  NMR detection of low (μM-mM) affinity binding is achieved using either differential line broadening or transferred NOE (nuclear Overhauser effect) NMR techniques.  Conclusions: The SHAPES method for lead generation by NMR is useful for identifying potential lead classes of drugs early in a drug design program, and is easily integrated with other discovery tools such as virtual screening, high-throughput screening and combinatorial chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqu88axGMNTgbVg90H21EOLACvtfcHk0li5ruX3cNy_Hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmvVOns7w%253D&md5=5f0d56e9b78579b0458ed49cf98d7c0b</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2FS1074-5521%2800%2980022-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1074-5521%252800%252980022-8%26sid%3Dliteratum%253Aachs%26aulast%3DFejzo%26aufirst%3DJ.%26aulast%3DLepre%26aufirst%3DC.%2BA.%26aulast%3DPeng%26aufirst%3DJ.%2BW.%26aulast%3DBemis%26aufirst%3DG.%2BW.%26aulast%3DAjay%253B%2BMurcko%26aufirst%3DM.%2BA.%26aulast%3DMoore%26aufirst%3DJ.%2BM.%26atitle%3DThe%2520SHAPES%2520strategy%253A%2520an%2520NMR-based%2520approach%2520for%2520lead%2520generation%2520in%2520drug%2520discovery%26jtitle%3DChem.%2520Biol.%26date%3D1999%26volume%3D6%26issue%3D10%26spage%3D755%26epage%3D769" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bures, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Praestgaard, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span> </span><span class="NLM_article-title">Privileged molecules for protein binding identified from NMR-based screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x"> (</span><span class="NLM_issue">18</span><span class="NLM_x">) </span> <span class="NLM_fpage">3443</span><span class="NLM_x">–</span> <span class="NLM_lpage">3447</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm000164q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=3443-3447&issue=18&author=P.+J.+Hajdukauthor=M.+Buresauthor=J.+Praestgaardauthor=S.+W.+Fesik&title=Privileged+molecules+for+protein+binding+identified+from+NMR-based+screening"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm000164q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm000164q%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DBures%26aufirst%3DM.%26aulast%3DPraestgaard%26aufirst%3DJ.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DPrivileged%2520molecules%2520for%2520protein%2520binding%2520identified%2520from%2520NMR-based%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26issue%3D18%26spage%3D3443%26epage%3D3447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Baurin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aboul-ela, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barril, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drysdale, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dymock, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finch, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fromont, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmonite, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard, R. E.</span><span> </span><span class="NLM_article-title">Design and characterization of molecular fragments for use in NMR screening against protein tagets</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">2157</span><span class="NLM_x">–</span> <span class="NLM_lpage">2166</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci049806z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADC%252BD2cXotlaru78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2004&pages=2157-2166&author=N.+Baurinauthor=F.+Aboul-elaauthor=X.+Barrilauthor=B.+Davisauthor=M.+Drysdaleauthor=B.+Dymockauthor=H.+Finchauthor=C.+Fromontauthor=C.+Richardsonauthor=H.+Simmoniteauthor=R.+E.+Hubbard&title=Design+and+characterization+of+molecular+fragments+for+use+in+NMR+screening+against+protein+tagets"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Characterization of Libraries of Molecular Fragments for Use in NMR Screening against Protein Targets</span></div><div class="casAuthors">Baurin, Nicolas; Aboul-Ela, Fareed; Barril, Xavier; Davis, Ben; Drysdale, Martin; Dymock, Brian; Finch, Harry; Fromont, Christophe; Richardson, Christine; Simmonite, Heather; Hubbard, Roderick E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2157-2166</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have designed four generations of a low mol. wt. fragment library for use in NMR-based screening against protein targets.  The library initially contained 723 fragments which were selected manually from the Available Chems. Directory.  A series of in silico filters and property calcns. were developed to automate the selection process, allowing a larger database of 1.79M available compds. to be searched for a further 357 compds. that were added to the library.  A kinase binding pharmacophore was then derived to select 174 kinase-focused fragments.  Finally, an addnl. 61 fragments were selected to increase the no. of different pharmacophores represented within the library.  All of the fragments added to the library passed quality checks to ensure they were suitable for the screening protocol, with appropriate soly., purity, chem. stability, and unambiguous NMR spectrum.  The successive generations of libraries have been characterized through anal. of structural properties (mol. wt., lipophilicity, polar surface area, no. of rotatable bonds, and hydrogen-bonding potential) and by analyzing their pharmacophoric complexity.  These calcns. have been used to compare the fragment libraries with a drug-like ref. set of compds. and a set of mols. that bind to protein active sites.  In addn., an anal. of the overall results of screening the library against the ATP binding site of two protein targets (HSP90 and CDK2) reveals different patterns of fragment binding, demonstrating that the approach can find selective compds. that discriminate between related binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo652Eyyn5Lc7Vg90H21EOLACvtfcHk0li5ruX3cNy_Hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXotlaru78%253D&md5=7376be270294dae6997609448513c5d7</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fci049806z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049806z%26sid%3Dliteratum%253Aachs%26aulast%3DBaurin%26aufirst%3DN.%26aulast%3DAboul-ela%26aufirst%3DF.%26aulast%3DBarril%26aufirst%3DX.%26aulast%3DDavis%26aufirst%3DB.%26aulast%3DDrysdale%26aufirst%3DM.%26aulast%3DDymock%26aufirst%3DB.%26aulast%3DFinch%26aufirst%3DH.%26aulast%3DFromont%26aufirst%3DC.%26aulast%3DRichardson%26aufirst%3DC.%26aulast%3DSimmonite%26aufirst%3DH.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DDesign%2520and%2520characterization%2520of%2520molecular%2520fragments%2520for%2520use%2520in%2520NMR%2520screening%2520against%2520protein%2520tagets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2004%26volume%3D44%26spage%3D2157%26epage%3D2166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Johnson, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feher, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williamson, J. R.</span><span> </span><span class="NLM_article-title">Application of NMR SHAPES screening to an RNA target</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x"> (</span><span class="NLM_issue">51</span><span class="NLM_x">) </span> <span class="NLM_fpage">15724</span><span class="NLM_x">–</span> <span class="NLM_lpage">15725</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja037499s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2003&pages=15724-15725&issue=51&author=E.+C.+Johnsonauthor=V.+A.+Feherauthor=J.+W.+Pengauthor=J.+M.+Mooreauthor=J.+R.+Williamson&title=Application+of+NMR+SHAPES+screening+to+an+RNA+target"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fja037499s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja037499s%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DE.%2BC.%26aulast%3DFeher%26aufirst%3DV.%2BA.%26aulast%3DPeng%26aufirst%3DJ.%2BW.%26aulast%3DMoore%26aufirst%3DJ.%2BM.%26aulast%3DWilliamson%26aufirst%3DJ.%2BR.%26atitle%3DApplication%2520of%2520NMR%2520SHAPES%2520screening%2520to%2520an%2520RNA%2520target%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2003%26volume%3D125%26issue%3D51%26spage%3D15724%26epage%3D15725" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Swayze, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jefferson, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sannes-Lowery, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blyn, L. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Risen, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arakawa, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Osgood, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hofstadler, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffey, R. H.</span><span> </span><span class="NLM_article-title">SAR by MS: a ligand based technique for drug lead discovery against structured RNA targets</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x"> (</span><span class="NLM_issue">18</span><span class="NLM_x">) </span> <span class="NLM_fpage">3816</span><span class="NLM_x">–</span> <span class="NLM_lpage">3819</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0255466" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=3816-3819&issue=18&author=E.+E.+Swayzeauthor=E.+A.+Jeffersonauthor=K.+A.+Sannes-Loweryauthor=L.+B.+Blynauthor=L.+M.+Risenauthor=S.+Arakawaauthor=S.+A.+Osgoodauthor=S.+A.+Hofstadlerauthor=R.+H.+Griffey&title=SAR+by+MS%3A+a+ligand+based+technique+for+drug+lead+discovery+against+structured+RNA+targets"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm0255466&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0255466%26sid%3Dliteratum%253Aachs%26aulast%3DSwayze%26aufirst%3DE.%2BE.%26aulast%3DJefferson%26aufirst%3DE.%2BA.%26aulast%3DSannes-Lowery%26aufirst%3DK.%2BA.%26aulast%3DBlyn%26aufirst%3DL.%2BB.%26aulast%3DRisen%26aufirst%3DL.%2BM.%26aulast%3DArakawa%26aufirst%3DS.%26aulast%3DOsgood%26aufirst%3DS.%2BA.%26aulast%3DHofstadler%26aufirst%3DS.%2BA.%26aulast%3DGriffey%26aufirst%3DR.%2BH.%26atitle%3DSAR%2520by%2520MS%253A%2520a%2520ligand%2520based%2520technique%2520for%2520drug%2520lead%2520discovery%2520against%2520structured%2520RNA%2520targets%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26issue%3D18%26spage%3D3816%26epage%3D3819" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Seth, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyaji, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jefferson, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sannes-Lowry, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Osgood, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Propp, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ranken, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Massire, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sampath, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ecker, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swayze, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffey, R. H.</span><span> </span><span class="NLM_article-title">SAR by MS: Discovery of a new class of RNA-binding small molecules for the hepatitis C virus: internal ribosome entry site IIA domain</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">7099</span><span class="NLM_x">–</span> <span class="NLM_lpage">7102</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm050815o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=7099-7102&author=P.+P.+Sethauthor=A.+Miyajiauthor=E.+A.+Jeffersonauthor=K.+A.+Sannes-Lowryauthor=S.+A.+Osgoodauthor=S.+S.+Proppauthor=R.+Rankenauthor=C.+Massireauthor=R.+Sampathauthor=D.+J.+Eckerauthor=E.+E.+Swayzeauthor=R.+H.+Griffey&title=SAR+by+MS%3A+Discovery+of+a+new+class+of+RNA-binding+small+molecules+for+the+hepatitis+C+virus%3A+internal+ribosome+entry+site+IIA+domain"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm050815o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050815o%26sid%3Dliteratum%253Aachs%26aulast%3DSeth%26aufirst%3DP.%2BP.%26aulast%3DMiyaji%26aufirst%3DA.%26aulast%3DJefferson%26aufirst%3DE.%2BA.%26aulast%3DSannes-Lowry%26aufirst%3DK.%2BA.%26aulast%3DOsgood%26aufirst%3DS.%2BA.%26aulast%3DPropp%26aufirst%3DS.%2BS.%26aulast%3DRanken%26aufirst%3DR.%26aulast%3DMassire%26aufirst%3DC.%26aulast%3DSampath%26aufirst%3DR.%26aulast%3DEcker%26aufirst%3DD.%2BJ.%26aulast%3DSwayze%26aufirst%3DE.%2BE.%26aulast%3DGriffey%26aufirst%3DR.%2BH.%26atitle%3DSAR%2520by%2520MS%253A%2520Discovery%2520of%2520a%2520new%2520class%2520of%2520RNA-binding%2520small%2520molecules%2520for%2520the%2520hepatitis%2520C%2520virus%253A%2520internal%2520ribosome%2520entry%2520site%2520IIA%2520domain%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2005%26volume%3D48%26spage%3D7099%26epage%3D7102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Yu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oost, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schkeryantz, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janowick, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span> </span><span class="NLM_article-title">Discovery of aminoglycoside mimetics by NMR-based screening of <i>Escherichia coli</i> A-site RNA</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x"> (</span><span class="NLM_issue">15</span><span class="NLM_x">) </span> <span class="NLM_fpage">4444</span><span class="NLM_x">–</span> <span class="NLM_lpage">4450</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja021354o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2003&pages=4444-4450&issue=15&author=L.+Yuauthor=T.+K.+Oostauthor=J.+M.+Schkeryantzauthor=J.+Yangauthor=D.+Janowickauthor=S.+W.+Fesik&title=Discovery+of+aminoglycoside+mimetics+by+NMR-based+screening+of+Escherichia+coli+A-site+RNA"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fja021354o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja021354o%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DL.%26aulast%3DOost%26aufirst%3DT.%2BK.%26aulast%3DSchkeryantz%26aufirst%3DJ.%2BM.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DJanowick%26aufirst%3DD.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDiscovery%2520of%2520aminoglycoside%2520mimetics%2520by%2520NMR-based%2520screening%2520of%2520Escherichia%2520coli%2520A-site%2520RNA%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2003%26volume%3D125%26issue%3D15%26spage%3D4444%26epage%3D4450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Fourmy, D. R., M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blanchard, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Puglis, J. D.</span><span> </span><span class="NLM_article-title">Structure of the A site of <i>Escherichia coli</i> 16S ribosomal RNA complexed with an aminoglycoside antibiotic</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">274</span><span class="NLM_x">, </span> <span class="NLM_fpage">1367</span><span class="NLM_x">–</span> <span class="NLM_lpage">1371</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Fjm9000659&amp;key=10.1126%2Fscience.274.5291.1367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Fjm9000659&amp;key=8910275" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADyaK28XntFKlsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1996&pages=1367-1371&author=D.+R.%2C+M.+I.+Fourmyauthor=S.+C.+Blanchardauthor=J.+D.+Puglis&title=Structure+of+the+A+site+of+Escherichia+coli+16S+ribosomal+RNA+complexed+with+an+aminoglycoside+antibiotic"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the A site of Escherichia coli 16S ribosomal RNA complexed with an aminoglycoside antibiotic</span></div><div class="casAuthors">Fourmy, Dominique; Recht, Michael I.; Blanchard, Scott C.; Puglisi, Joseph D.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">274</span>
        (<span class="NLM_cas:issue">5291</span>),
    <span class="NLM_cas:pages">1367-1371</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Aminoglycoside antibiotics that bind to 30S ribosomal A-site RNA cause misreading of the genetic code and inhibit translocation.  The aminoglycoside antibiotic paromomycin (I) binds specifically to an RNA oligonucleotide that contains the 30S subunit A site, and the soln. structure of the RNA-I complex was detd. by NMR spectroscopy.  The antibiotic binds in the major groove of the model A-site RNA within a pocket created by an A-A base pair and a single bulged adenine.  Specific interactions occur between aminoglycoside chem. groups important for antibiotic activity and conserved nucleotides in the RNA.  The structure explains binding of diverse aminoglycosides to the ribosome, their specific activity against prokaryotic organisms, and various resistance mechanisms, and provides insight into ribosome function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorBok0q_DmNbVg90H21EOLACvtfcHk0li3Q_6aE7kEag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XntFKlsrw%253D&md5=61f144a580809d46c416c2e2ab67fcc1</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1126%2Fscience.274.5291.1367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.274.5291.1367%26sid%3Dliteratum%253Aachs%26aulast%3DFourmy%26aufirst%3DD.%2BR.%252C%2BM.%2BI.%26aulast%3DBlanchard%26aufirst%3DS.%2BC.%26aulast%3DPuglis%26aufirst%3DJ.%2BD.%26atitle%3DStructure%2520of%2520the%2520A%2520site%2520of%2520Escherichia%2520coli%252016S%2520ribosomal%2520RNA%2520complexed%2520with%2520an%2520aminoglycoside%2520antibiotic%26jtitle%3DScience%26date%3D1996%26volume%3D274%26spage%3D1367%26epage%3D1371" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Schuffenhauer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ertl, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roggo, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wetzel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koch, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waldmann, H.</span><span> </span><span class="NLM_article-title">The scaffold tree. Visualization of the scaffold universe by hierarchical scaffold classification</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">47</span><span class="NLM_x">–</span> <span class="NLM_lpage">58</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci600338x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlCit7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=47-58&issue=1&author=A.+Schuffenhauerauthor=P.+Ertlauthor=S.+Roggoauthor=S.+Wetzelauthor=M.+A.+Kochauthor=H.+Waldmann&title=The+scaffold+tree.+Visualization+of+the+scaffold+universe+by+hierarchical+scaffold+classification"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">The Scaffold Tree - Visualization of the Scaffold Universe by Hierarchical Scaffold Classification</span></div><div class="casAuthors">Schuffenhauer, Ansgar; Ertl, Peter; Roggo, Silvio; Wetzel, Stefan; Koch, Marcus A.; Waldmann, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">47-58</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A hierarchical classification of chem. scaffolds (mol. framework, which is obtained by pruning all terminal side chains) has been introduced.  The mol. frameworks form the leaf nodes in the hierarchy trees.  By an iterative removal of rings, scaffolds forming the higher levels in the hierarchy tree are obtained.  Prioritization rules ensure that less characteristic, peripheral rings are removed first.  All scaffolds in the hierarchy tree are well-defined chem. entities making the classification chem. intuitive.  The classification is deterministic, data-set-independent, and scales linearly with the no. of compds. included in the data set.  The application of the classification is demonstrated on two data sets extd. from the PubChem database, namely, pyruvate kinase binders and a collection of pesticides.  The examples shown demonstrate that the classification procedure handles robustly synthetic structures and natural products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8XfR2xvGasbVg90H21EOLACvtfcHk0li3Q_6aE7kEag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlCit7jO&md5=89cec5a01a37802ab0826ecababfaf0a</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fci600338x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600338x%26sid%3Dliteratum%253Aachs%26aulast%3DSchuffenhauer%26aufirst%3DA.%26aulast%3DErtl%26aufirst%3DP.%26aulast%3DRoggo%26aufirst%3DS.%26aulast%3DWetzel%26aufirst%3DS.%26aulast%3DKoch%26aufirst%3DM.%2BA.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DThe%2520scaffold%2520tree.%2520Visualization%2520of%2520the%2520scaffold%2520universe%2520by%2520hierarchical%2520scaffold%2520classification%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26issue%3D1%26spage%3D47%26epage%3D58" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Stockman, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalvit, C.</span><span> </span><span class="NLM_article-title">NMR screening techniques in drug discovery and drug design</span> <span class="citation_source-journal">Prog. Nucl. Magn. Reson. Spectrosc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x"> (</span><span class="NLM_issue">3−4</span><span class="NLM_x">) </span> <span class="NLM_fpage">187</span><span class="NLM_x">–</span> <span class="NLM_lpage">231</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2002&pages=187-231&issue=3%E2%88%924&author=B.+J.+Stockmanauthor=C.+Dalvit&title=NMR+screening+techniques+in+drug+discovery+and+drug+design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DStockman%26aufirst%3DB.%2BJ.%26aulast%3DDalvit%26aufirst%3DC.%26atitle%3DNMR%2520screening%2520techniques%2520in%2520drug%2520discovery%2520and%2520drug%2520design%26jtitle%3DProg.%2520Nucl.%2520Magn.%2520Reson.%2520Spectrosc.%26date%3D2002%26volume%3D41%26issue%3D3%25E2%2588%25924%26spage%3D187%26epage%3D231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Ertl, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rohde, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Selzer, P.</span><span> </span><span class="NLM_article-title">Fast calculation of molecular polar surface area as a sum of fragment-based contributions and its application to the prediction of drug transport properties</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x"> (</span><span class="NLM_issue">20</span><span class="NLM_x">) </span> <span class="NLM_fpage">3714</span><span class="NLM_x">–</span> <span class="NLM_lpage">3717</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm000942e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmsFeitb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=3714-3717&issue=20&author=P.+Ertlauthor=B.+Rohdeauthor=P.+Selzer&title=Fast+calculation+of+molecular+polar+surface+area+as+a+sum+of+fragment-based+contributions+and+its+application+to+the+prediction+of+drug+transport+properties"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Fast Calculation of Molecular Polar Surface Area as a Sum of Fragment-Based Contributions and Its Application to the Prediction of Drug Transport Properties</span></div><div class="casAuthors">Ertl, Peter; Rohde, Bernhard; Selzer, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3714-3717</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. polar surface area (PSA), i.e., surface belonging to polar atoms, is a descriptor that was shown to correlate well with passive mol. transport through membranes and, therefore, allows prediction of transport properties of drugs.  The calcn. of PSA, however, is rather time-consuming because of the necessity to generate a reasonable 3D mol. geometry and the calcn. of the surface itself.  A new approach for the calcn. of the PSA is presented here, based on the summation of tabulated surface contributions of polar fragments.  The method, termed topol. PSA (TPSA), provides results which are practically identical with the 3D PSA (the correlation coeff. between 3D PSA and fragment-based TPSA for 34 810 mols. from the World Drug Index is 0.99), while the computation speed is 2-3 orders of magnitude faster.  The new methodol. may, therefore, be used for fast bioavailability screening of virtual libraries having millions of mols.  This article describes the new methodol. and shows the results of validation studies based on sets of published absorption data, including intestinal absorption, Caco-2 monolayer penetration, and blood-brain barrier penetration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhP6Qncx_h6LVg90H21EOLACvtfcHk0ljfz64VNUb_lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmsFeitb4%253D&md5=23ae4e7708bd8ddeb4a24ad160a29bd6</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm000942e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm000942e%26sid%3Dliteratum%253Aachs%26aulast%3DErtl%26aufirst%3DP.%26aulast%3DRohde%26aufirst%3DB.%26aulast%3DSelzer%26aufirst%3DP.%26atitle%3DFast%2520calculation%2520of%2520molecular%2520polar%2520surface%2520area%2520as%2520a%2520sum%2520of%2520fragment-based%2520contributions%2520and%2520its%2520application%2520to%2520the%2520prediction%2520of%2520drug%2520transport%2520properties%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26issue%3D20%26spage%3D3714%26epage%3D3717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Hou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, X.</span><span> </span><span class="NLM_article-title">ADME evaluation in drug discovery. 7. Prediction of oral absorption by correlation and classification</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">208</span><span class="NLM_x">–</span> <span class="NLM_lpage">218</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci600343x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtF2it7nK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=208-218&issue=1&author=T.+Houauthor=J.+Wangauthor=W.+Zhangauthor=X.+Xu&title=ADME+evaluation+in+drug+discovery.+7.+Prediction+of+oral+absorption+by+correlation+and+classification"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">ADME evaluation in drug discovery. 7. Prediction of oral absorption by correlation and classification</span></div><div class="casAuthors">Hou, Tingjun; Wang, Junmei; Zhang, Wei; Xu, Xiaojie</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">208-218</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  A critically evaluated database of human intestinal absorption for 648 chem. compds. is reported in this study, among which 579 are believed to be transported by passive diffusion.  The correlation anal. between the intestinal absorption and several important mol. properties demonstrated that no single mol. property could be used as a good discriminator to efficiently distinguish the poorly absorbed compds. from those that are well absorbed.  The theor. correlation models for a training set of 455 compds. were proposed by the genetic function approxn. technique.  The best prediction model contains 4 mol. descriptors: topol. polar surface area, the predicted distribution coeff. at pH = 6.5, the no. of violations of the Lipinski's rule-of-five, and the square of the no. of hydrogen-bond donors.  The model was able to predict the fractional absorption with an r = 0.84 and a prediction error (abs. mean error) of 11.2% for the training set.  Moreover, it achieves an r = 0.90 and a prediction error of 7.8% for a 98-compd. test set.  The recursive partitioning technique was applied to find the simple hierarchical rules to classify the compds. into poor (%FA ≤ 30%) and good (%FA > 30%) intestinal absorption classes.  The high quality of the classification model was validated by the satisfactory predictions on the training set (correctly identifying 95.9% of the compds. in the poor-absorption class and 96.1% of the compds. in the good-absorption class) and on the test set (correctly identifying 100% of the compds. in the poor-absorption class and 96.8% of the compds. in the good-absorption class).  We expect that, in the future, the rules for the prediction of carrier-mediated transporting and 1st pass metab. can be integrated into the current hierarchical rules, and the classification model may become more powerful in the prediction of intestinal absorption or even human bioavailability.  The databases of human intestinal absorption reported here are available for download from the supporting Web site: http://modem.ucsd.edu/adme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqY2usDYjS_aLVg90H21EOLACvtfcHk0ljfz64VNUb_lg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtF2it7nK&md5=4ed85580d8ba7cbbeb7431f7f0c083f6</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fci600343x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600343x%26sid%3Dliteratum%253Aachs%26aulast%3DHou%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DXu%26aufirst%3DX.%26atitle%3DADME%2520evaluation%2520in%2520drug%2520discovery.%25207.%2520Prediction%2520of%2520oral%2520absorption%2520by%2520correlation%2520and%2520classification%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26issue%3D1%26spage%3D208%26epage%3D218" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Griffey, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sannes-Lowery, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drader, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mohan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swayze, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hofstadler, S. A.</span><span> </span><span class="NLM_article-title">Characterization of low-affinity complexes between RNA and small molecules using electrospray ionization mass spectrometry</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">122</span><span class="NLM_x"> (</span><span class="NLM_issue">41</span><span class="NLM_x">) </span> <span class="NLM_fpage">9933</span><span class="NLM_x">–</span> <span class="NLM_lpage">9938</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja0017108" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2000&pages=9933-9938&issue=41&author=R.+H.+Griffeyauthor=K.+A.+Sannes-Loweryauthor=J.+J.+Draderauthor=V.+Mohanauthor=E.+E.+Swayzeauthor=S.+A.+Hofstadler&title=Characterization+of+low-affinity+complexes+between+RNA+and+small+molecules+using+electrospray+ionization+mass+spectrometry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fja0017108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0017108%26sid%3Dliteratum%253Aachs%26aulast%3DGriffey%26aufirst%3DR.%2BH.%26aulast%3DSannes-Lowery%26aufirst%3DK.%2BA.%26aulast%3DDrader%26aufirst%3DJ.%2BJ.%26aulast%3DMohan%26aufirst%3DV.%26aulast%3DSwayze%26aufirst%3DE.%2BE.%26aulast%3DHofstadler%26aufirst%3DS.%2BA.%26atitle%3DCharacterization%2520of%2520low-affinity%2520complexes%2520between%2520RNA%2520and%2520small%2520molecules%2520using%2520electrospray%2520ionization%2520mass%2520spectrometry%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2000%26volume%3D122%26issue%3D41%26spage%3D9933%26epage%3D9938" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Pinto, I. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guilbert, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ulyanov, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stearns, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, T. L.</span><span> </span><span class="NLM_article-title">Discovery of ligands for a novel target, the human telomerase RNA, based on flexible-target virtual screening and NMR</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">7205</span><span class="NLM_x">–</span> <span class="NLM_lpage">7215</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=7205-7215&author=I.+G.+Pintoauthor=C.+Guilbertauthor=N.+B.+Ulyanovauthor=J.+Stearnsauthor=T.+L.+James&title=Discovery+of+ligands+for+a+novel+target%2C+the+human+telomerase+RNA%2C+based+on+flexible-target+virtual+screening+and+NMR"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPinto%26aufirst%3DI.%2BG.%26aulast%3DGuilbert%26aufirst%3DC.%26aulast%3DUlyanov%26aufirst%3DN.%2BB.%26aulast%3DStearns%26aufirst%3DJ.%26aulast%3DJames%26aufirst%3DT.%2BL.%26atitle%3DDiscovery%2520of%2520ligands%2520for%2520a%2520novel%2520target%252C%2520the%2520human%2520telomerase%2520RNA%252C%2520based%2520on%2520flexible-target%2520virtual%2520screening%2520and%2520NMR%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D7205%26epage%3D7215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Jenne, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartig, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piganeau, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tauer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Samarsky, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Famulok, M.</span><span> </span><span class="NLM_article-title">Rapid identification and characterization of hammerhead-ribozyme inhibitors using fluorescence-based technology</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">56</span><span class="NLM_x">–</span> <span class="NLM_lpage">61</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2001&pages=56-61&issue=1&author=A.+Jenneauthor=J.+S.+Hartigauthor=N.+Piganeauauthor=A.+Tauerauthor=D.+A.+Samarskyauthor=M.+R.+Greenauthor=J.+Daviesauthor=M.+Famulok&title=Rapid+identification+and+characterization+of+hammerhead-ribozyme+inhibitors+using+fluorescence-based+technology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJenne%26aufirst%3DA.%26aulast%3DHartig%26aufirst%3DJ.%2BS.%26aulast%3DPiganeau%26aufirst%3DN.%26aulast%3DTauer%26aufirst%3DA.%26aulast%3DSamarsky%26aufirst%3DD.%2BA.%26aulast%3DGreen%26aufirst%3DM.%2BR.%26aulast%3DDavies%26aufirst%3DJ.%26aulast%3DFamulok%26aufirst%3DM.%26atitle%3DRapid%2520identification%2520and%2520characterization%2520of%2520hammerhead-ribozyme%2520inhibitors%2520using%2520fluorescence-based%2520technology%26jtitle%3DNat.%2520Biotechnol.%26date%3D2001%26volume%3D19%26issue%3D1%26spage%3D56%26epage%3D61" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Blount, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Puskarz, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Penchovsky, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breaker, R.</span><span> </span><span class="NLM_article-title">Development and application of a high-throughput assay for glmS riboswitch activators</span> <span class="citation_source-journal">RNA Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">20</span><span class="NLM_x">–</span> <span class="NLM_lpage">24</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2006&pages=20-24&issue=2&author=K.+Blountauthor=I.+Puskarzauthor=R.+Penchovskyauthor=R.+Breaker&title=Development+and+application+of+a+high-throughput+assay+for+glmS+riboswitch+activators"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBlount%26aufirst%3DK.%26aulast%3DPuskarz%26aufirst%3DI.%26aulast%3DPenchovsky%26aufirst%3DR.%26aulast%3DBreaker%26aufirst%3DR.%26atitle%3DDevelopment%2520and%2520application%2520of%2520a%2520high-throughput%2520assay%2520for%2520glmS%2520riboswitch%2520activators%26jtitle%3DRNA%2520Biol.%26date%3D2006%26volume%3D3%26issue%3D2%26spage%3D20%26epage%3D24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Wong, C.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hendrix, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Priestley, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenberg, W. A.</span><span> </span><span class="NLM_article-title">Specificity of aminoglycoside antibiotics for the A-site of the decoding region of ribosomal RNA</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">397</span><span class="NLM_x">–</span> <span class="NLM_lpage">406</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Fjm9000659&amp;key=10.1016%2FS1074-5521%2898%2990073-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Fjm9000659&amp;key=9662506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADyaK1cXltVeqsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1998&pages=397-406&author=C.-H.+Wongauthor=M.+Hendrixauthor=E.+S.+Priestleyauthor=W.+A.+Greenberg&title=Specificity+of+aminoglycoside+antibiotics+for+the+A-site+of+the+decoding+region+of+ribosomal+RNA"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Specificity of aminoglycoside antibiotics for the A-site of the decoding region of ribosomal RNA</span></div><div class="casAuthors">Wong, Chi-Huey; Hendrix, Martin; Priestley, E. Scott; Greenberg, William A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">397-406</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Current Biology Ltd.</span>)
        </div><div class="casAbstract">Aminoglycoside antibiotics bind to the A-site of the decoding region of 16S RNA in the bacterial ribosome, an interaction that is probably responsible for their activity.  A detailed study of the specificity of aminoglycoside binding to A-site RNA would improve our understanding of their mechanism of antibiotic activity.  We have studied the binding specificity of several aminoglycosides with model RNA sequences derived from the 16S ribosomal A-site using surface plasmon resonance.  The 4,5-linked (neomycin) class of aminoglycosides showed specificity for wild-type A-site sequences, but the 4,6-linked class (kanamycins and gentamicins), generally showed poor specificity for the same sequences.  Methylation of a cytidine in the target RNA, as found in the Escherichia coli ribosome, had negligible effects on aminoglycoside binding.  Although both 4,5- and 4,6-linked aminoglycosides target the same ribosomal site, they appear to bind and effect antibiotic activity in different manners.  The aminoglycosides might recognize different RNA conformations or the interaction might involve different RNA tertiary structures that are not equally sampled in our ribosome-free model.  These results imply that models of rRNA must be carefully designed if the data are expected to accurately reflect biol. activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTLZJpmzaNnrVg90H21EOLACvtfcHk0ljvxTTJWGpvkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXltVeqsbk%253D&md5=1d0c410d2a011ff6f771c1fb18859038</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2FS1074-5521%2898%2990073-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1074-5521%252898%252990073-4%26sid%3Dliteratum%253Aachs%26aulast%3DWong%26aufirst%3DC.-H.%26aulast%3DHendrix%26aufirst%3DM.%26aulast%3DPriestley%26aufirst%3DE.%2BS.%26aulast%3DGreenberg%26aufirst%3DW.%2BA.%26atitle%3DSpecificity%2520of%2520aminoglycoside%2520antibiotics%2520for%2520the%2520A-site%2520of%2520the%2520decoding%2520region%2520of%2520ribosomal%2520RNA%26jtitle%3DChem.%2520Biol.%26date%3D1998%26volume%3D5%26spage%3D397%26epage%3D406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Mayer, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Famulok, M.</span><span> </span><span class="NLM_article-title">High-throughput-compatible assay for glmS riboswitch metabolite dependence</span> <span class="citation_source-journal">ChemBioChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">602</span><span class="NLM_x">–</span> <span class="NLM_lpage">604</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&pages=602-604&issue=4&author=G.+Mayerauthor=M.+Famulok&title=High-throughput-compatible+assay+for+glmS+riboswitch+metabolite+dependence"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMayer%26aufirst%3DG.%26aulast%3DFamulok%26aufirst%3DM.%26atitle%3DHigh-throughput-compatible%2520assay%2520for%2520glmS%2520riboswitch%2520metabolite%2520dependence%26jtitle%3DChemBioChem%26date%3D2006%26volume%3D7%26issue%3D4%26spage%3D602%26epage%3D604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Faber, C. S., H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schweimer, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosch, P.</span><span> </span><span class="NLM_article-title">Structural rearrangements of HIV-Tat-responsive RNA upon binding of neomycin B</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">275</span><span class="NLM_x"> (</span><span class="NLM_issue">27</span><span class="NLM_x">) </span> <span class="NLM_fpage">20660</span><span class="NLM_x">–</span> <span class="NLM_lpage">20666</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=275&publication_year=2000&pages=20660-20666&issue=27&author=C.+S.%2C+H.+Faberauthor=K.+Schweimerauthor=P.+Rosch&title=Structural+rearrangements+of+HIV-Tat-responsive+RNA+upon+binding+of+neomycin+B"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFaber%26aufirst%3DC.%2BS.%252C%2BH.%26aulast%3DSchweimer%26aufirst%3DK.%26aulast%3DRosch%26aufirst%3DP.%26atitle%3DStructural%2520rearrangements%2520of%2520HIV-Tat-responsive%2520RNA%2520upon%2520binding%2520of%2520neomycin%2520B%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2000%26volume%3D275%26issue%3D27%26spage%3D20660%26epage%3D20666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Ennifar, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paillart, J.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bodlenner, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walter, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weibel, J.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aubertin, A.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pale, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dumas, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marquet, R.</span><span> </span><span class="NLM_article-title">Targeting the dimerization initiation site of HIV-1 RNA with aminoglycosides: from crystal to cell</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">2328</span><span class="NLM_x">–</span> <span class="NLM_lpage">2339</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=2328-2339&issue=8&author=E.+Ennifarauthor=J.-C.+Paillartauthor=A.+Bodlennerauthor=P.+Walterauthor=J.-M.+Weibelauthor=A.-M.+Aubertinauthor=P.+Paleauthor=P.+Dumasauthor=R.+Marquet&title=Targeting+the+dimerization+initiation+site+of+HIV-1+RNA+with+aminoglycosides%3A+from+crystal+to+cell"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEnnifar%26aufirst%3DE.%26aulast%3DPaillart%26aufirst%3DJ.-C.%26aulast%3DBodlenner%26aufirst%3DA.%26aulast%3DWalter%26aufirst%3DP.%26aulast%3DWeibel%26aufirst%3DJ.-M.%26aulast%3DAubertin%26aufirst%3DA.-M.%26aulast%3DPale%26aufirst%3DP.%26aulast%3DDumas%26aufirst%3DP.%26aulast%3DMarquet%26aufirst%3DR.%26atitle%3DTargeting%2520the%2520dimerization%2520initiation%2520site%2520of%2520HIV-1%2520RNA%2520with%2520aminoglycosides%253A%2520from%2520crystal%2520to%2520cell%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26issue%3D8%26spage%3D2328%26epage%3D2339" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Staple, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Venditti, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niccolai, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elson-Schwab, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tor, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butcher, S. E.</span><span> </span><span class="NLM_article-title">Guanidinoneomycin B recognition of an HIV-1 RNA helix</span> <span class="citation_source-journal">ChemBioChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">93</span><span class="NLM_x">–</span> <span class="NLM_lpage">102</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=93-102&issue=1&author=D.+W.+Stapleauthor=V.+Vendittiauthor=N.+Niccolaiauthor=L.+Elson-Schwabauthor=Y.+Torauthor=S.+E.+Butcher&title=Guanidinoneomycin+B+recognition+of+an+HIV-1+RNA+helix"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DStaple%26aufirst%3DD.%2BW.%26aulast%3DVenditti%26aufirst%3DV.%26aulast%3DNiccolai%26aufirst%3DN.%26aulast%3DElson-Schwab%26aufirst%3DL.%26aulast%3DTor%26aufirst%3DY.%26aulast%3DButcher%26aufirst%3DS.%2BE.%26atitle%3DGuanidinoneomycin%2520B%2520recognition%2520of%2520an%2520HIV-1%2520RNA%2520helix%26jtitle%3DChemBioChem%26date%3D2008%26volume%3D9%26issue%3D1%26spage%3D93%26epage%3D102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Anupam, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denapoli, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muchenditsi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hines, J. V.</span><span> </span><span class="NLM_article-title">Identification of neomycin B-binding site in T box antiterminator model RNA</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">4466</span><span class="NLM_x">–</span> <span class="NLM_lpage">4470</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2008&pages=4466-4470&issue=8&author=R.+Anupamauthor=L.+Denapoliauthor=A.+Muchenditsiauthor=J.+V.+Hines&title=Identification+of+neomycin+B-binding+site+in+T+box+antiterminator+model+RNA"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAnupam%26aufirst%3DR.%26aulast%3DDenapoli%26aufirst%3DL.%26aulast%3DMuchenditsi%26aufirst%3DA.%26aulast%3DHines%26aufirst%3DJ.%2BV.%26atitle%3DIdentification%2520of%2520neomycin%2520B-binding%2520site%2520in%2520T%2520box%2520antiterminator%2520model%2520RNA%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2008%26volume%3D16%26issue%3D8%26spage%3D4466%26epage%3D4470" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Schüller, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suhartono, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fechner, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanrikulu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breitung, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheffer, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Göbel, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, G.</span><span> </span><span class="NLM_article-title">The concept of template-based de novo design from drug-derived molecular fragments and its application to TAR RNA</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">59</span><span class="NLM_x">–</span> <span class="NLM_lpage">68</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=59-68&author=A.+Sch%C3%BCllerauthor=M.+Suhartonoauthor=U.+Fechnerauthor=Y.+Tanrikuluauthor=S.+Breitungauthor=U.+Schefferauthor=M.+W.+G%C3%B6belauthor=G.+Schneider&title=The+concept+of+template-based+de+novo+design+from+drug-derived+molecular+fragments+and+its+application+to+TAR+RNA"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSch%25C3%25BCller%26aufirst%3DA.%26aulast%3DSuhartono%26aufirst%3DM.%26aulast%3DFechner%26aufirst%3DU.%26aulast%3DTanrikulu%26aufirst%3DY.%26aulast%3DBreitung%26aufirst%3DS.%26aulast%3DScheffer%26aufirst%3DU.%26aulast%3DG%25C3%25B6bel%26aufirst%3DM.%2BW.%26aulast%3DSchneider%26aufirst%3DG.%26atitle%3DThe%2520concept%2520of%2520template-based%2520de%2520novo%2520design%2520from%2520drug-derived%2520molecular%2520fragments%2520and%2520its%2520application%2520to%2520TAR%2520RNA%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2008%26volume%3D22%26spage%3D59%26epage%3D68" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Disney, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labuda, L. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paul, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poplawski, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pushechnikov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tran, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velagapudi, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Childs-Disney, J. L.</span><span> </span><span class="NLM_article-title">Two-dimensional combinatorial screening identifies specific aminoglycoside-RNA internal loop partners</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">130</span><span class="NLM_x"> (</span><span class="NLM_issue">33</span><span class="NLM_x">) </span> <span class="NLM_fpage">11185</span><span class="NLM_x">–</span> <span class="NLM_lpage">11194</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja803234t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=11185-11194&issue=33&author=M.+D.+Disneyauthor=L.+P.+Labudaauthor=D.+J.+Paulauthor=S.+G.+Poplawskiauthor=A.+Pushechnikovauthor=T.+Tranauthor=S.+P.+Velagapudiauthor=M.+Wuauthor=J.+L.+Childs-Disney&title=Two-dimensional+combinatorial+screening+identifies+specific+aminoglycoside-RNA+internal+loop+partners"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fja803234t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja803234t%26sid%3Dliteratum%253Aachs%26aulast%3DDisney%26aufirst%3DM.%2BD.%26aulast%3DLabuda%26aufirst%3DL.%2BP.%26aulast%3DPaul%26aufirst%3DD.%2BJ.%26aulast%3DPoplawski%26aufirst%3DS.%2BG.%26aulast%3DPushechnikov%26aufirst%3DA.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DWu%26aufirst%3DM.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26atitle%3DTwo-dimensional%2520combinatorial%2520screening%2520identifies%2520specific%2520aminoglycoside-RNA%2520internal%2520loop%2520partners%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26issue%3D33%26spage%3D11185%26epage%3D11194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Hall, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kier, L. B.</span><span> </span><span class="NLM_article-title">The molecular connectivity chi indexes and kappa shape indexes in structure-property modeling</span> <span class="citation_source-journal">Reviews in Computational Chemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">367</span><span class="NLM_x">–</span> <span class="NLM_lpage">422</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=1991&pages=367-422&author=L.+H.+Hallauthor=L.+B.+Kier&title=The+molecular+connectivity+chi+indexes+and+kappa+shape+indexes+in+structure-property+modeling"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHall%26aufirst%3DL.%2BH.%26aulast%3DKier%26aufirst%3DL.%2BB.%26atitle%3DThe%2520molecular%2520connectivity%2520chi%2520indexes%2520and%2520kappa%2520shape%2520indexes%2520in%2520structure-property%2520modeling%26jtitle%3DReviews%2520in%2520Computational%2520Chemistry%26date%3D1991%26volume%3D2%26spage%3D367%26epage%3D422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Lipinski, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lombardo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dominy, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feeney, P. J.</span><span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span> <span class="citation_source-journal">Advanced Drug Delivery Reviews</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x"> (</span><span class="NLM_issue">1−3</span><span class="NLM_x">) </span> <span class="NLM_fpage">3</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=1997&pages=3-26&issue=1%E2%88%923&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdvanced%2520Drug%2520Delivery%2520Reviews%26date%3D1997%26volume%3D46%26issue%3D1%25E2%2588%25923%26spage%3D3%26epage%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 27 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lirui  Guan</span> and <span class="hlFld-ContribAuthor ">Matthew D.  Disney</span>  . </span><span class="cited-content_cbyCitation_article-title">Recent Advances in Developing Small Molecules Targeting RNA. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2012,</strong> <em>7 </em>
                                    (1)
                                     , 73-86. <a href="https://doi.org/10.1021/cb200447r" title="DOI URL">https://doi.org/10.1021/cb200447r</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/cb200447r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fcb200447r%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DRecent%252BAdvances%252Bin%252BDeveloping%252BSmall%252BMolecules%252BTargeting%252BRNA%26aulast%3DGuan%26aufirst%3DLirui%26date%3D2012%26date%3D2012%26date%3D2011%26date%3D2011%26date%3D31102011%26date%3D20122011%26date%3D12012012%26date%3D20012012%26date%3D20122011%26volume%3D7%26issue%3D1%26spage%3D73%26epage%3D86" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ryan J.  Marcheschi</span>, <span class="hlFld-ContribAuthor ">Marco  Tonelli</span>, <span class="hlFld-ContribAuthor ">Arvind  Kumar</span>, and <span class="hlFld-ContribAuthor ">Samuel E.  Butcher</span>  . </span><span class="cited-content_cbyCitation_article-title">Structure of the HIV-1 Frameshift Site RNA Bound to a Small Molecule Inhibitor of Viral Replication. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2011,</strong> <em>6 </em>
                                    (8)
                                     , 857-864. <a href="https://doi.org/10.1021/cb200082d" title="DOI URL">https://doi.org/10.1021/cb200082d</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/cb200082d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fcb200082d%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DStructure%252Bof%252Bthe%252BHIV-1%252BFrameshift%252BSite%252BRNA%252BBound%252Bto%252Ba%252BSmall%252BMolecule%252BInhibitor%252Bof%252BViral%252BReplication%26aulast%3DMarcheschi%26aufirst%3DRyan%2BJ.%26date%3D2011%26date%3D2011%26date%3D2011%26date%3D2011%26date%3D11032011%26date%3D07062011%26date%3D15062011%26date%3D19082011%26date%3D07062011%26volume%3D6%26issue%3D8%26spage%3D857%26epage%3D864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yaozong  Li</span>, <span class="hlFld-ContribAuthor ">Jie  Shen</span>, <span class="hlFld-ContribAuthor ">Xianqiang  Sun</span>, <span class="hlFld-ContribAuthor ">Weihua  Li</span>, <span class="hlFld-ContribAuthor ">Guixia  Liu</span> and <span class="hlFld-ContribAuthor ">Yun  Tang</span>. </span><span class="cited-content_cbyCitation_article-title">Accuracy Assessment of Protein-Based Docking Programs against RNA Targets. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2010,</strong> <em>50 </em>
                                    (6)
                                     , 1134-1146. <a href="https://doi.org/10.1021/ci9004157" title="DOI URL">https://doi.org/10.1021/ci9004157</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci9004157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci9004157%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DAccuracy%252BAssessment%252Bof%252BProtein-Based%252BDocking%252BPrograms%252Bagainst%252BRNA%252BTargets%26aulast%3DLi%26aufirst%3DYaozong%26date%3D2010%26date%3D2010%26date%3D2009%26date%3D26%252010%25202009%26date%3D19052010%26date%3D28062010%26volume%3D50%26issue%3D6%26spage%3D1134%26epage%3D1146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Liuhong  Chen</span>, <span class="hlFld-ContribAuthor ">Elena  Cressina</span>, <span class="hlFld-ContribAuthor ">Finian J.  Leeper</span>, <span class="hlFld-ContribAuthor ">Alison G.  Smith</span> and <span class="hlFld-ContribAuthor ">Chris  Abell</span>  . </span><span class="cited-content_cbyCitation_article-title">A Fragment-Based Approach to Identifying Ligands for Riboswitches. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2010,</strong> <em>5 </em>
                                    (4)
                                     , 355-358. <a href="https://doi.org/10.1021/cb9003139" title="DOI URL">https://doi.org/10.1021/cb9003139</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/cb9003139&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fcb9003139%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DA%252BFragment-Based%252BApproach%252Bto%252BIdentifying%252BLigands%252Bfor%252BRiboswitches%26aulast%3DChen%26aufirst%3DLiuhong%26date%3D2010%26date%3D2010%26date%3D2010%26date%3D2009%26date%3D15%252012%25202009%26date%3D16%252002%25202010%26date%3D25022010%26date%3D16042010%26date%3D16022010%26volume%3D5%26issue%3D4%26spage%3D355%26epage%3D358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sanjay  Dutta</span>, <span class="hlFld-ContribAuthor ">Cody J.  Higginson</span>, <span class="hlFld-ContribAuthor ">Bao T.  Ho</span>, <span class="hlFld-ContribAuthor ">Kevin D.  Rynearson</span>, <span class="hlFld-ContribAuthor ">Sergey M.  Dibrov</span> and <span class="hlFld-ContribAuthor ">Thomas  Hermann</span>. </span><span class="cited-content_cbyCitation_article-title">1,3-Diazepanes of Natural Product-Like Complexity from Cyanamide-Induced Rearrangement of Epoxy-δ-lactams. </span><span class="cited-content_cbyCitation_journal-name">Organic Letters</span><span> <strong>2010,</strong> <em>12 </em>
                                    (2)
                                     , 360-363. <a href="https://doi.org/10.1021/ol9026914" title="DOI URL">https://doi.org/10.1021/ol9026914</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ol9026914&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fol9026914%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Letters%26atitle%3D1%25252C3-Diazepanes%252Bof%252BNatural%252BProduct-Like%252BComplexity%252Bfrom%252BCyanamide-Induced%252BRearrangement%252Bof%252BEpoxy-%2525CE%2525B4-lactams%26aulast%3DDutta%26aufirst%3DSanjay%26date%3D2010%26date%3D2009%26date%3D2009%26date%3D22%252011%25202009%26date%3D22122009%26date%3D15012010%26volume%3D12%26issue%3D2%26spage%3D360%26epage%3D363" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aline  Umuhire Juru</span>, <span class="hlFld-ContribAuthor ">Amanda E.  Hargrove</span>. </span><span class="cited-content_cbyCitation_article-title">Frameworks for targeting RNA with small molecules. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biological Chemistry</span><span> <strong>2021,</strong> <em>296 </em>, 100191. <a href="https://doi.org/10.1074/jbc.REV120.015203" title="DOI URL">https://doi.org/10.1074/jbc.REV120.015203</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1074/jbc.REV120.015203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1074%2Fjbc.REV120.015203%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biological%2520Chemistry%26atitle%3DFrameworks%252Bfor%252Btargeting%252BRNA%252Bwith%252Bsmall%252Bmolecules%26aulast%3DUmuhire%2BJuru%26aufirst%3DAline%26date%3D2021%26volume%3D296%26spage%3D100191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">A.  Di Giorgio</span>, <span class="hlFld-ContribAuthor ">M.  Duca</span>. </span><span class="cited-content_cbyCitation_article-title">Synthetic small-molecule RNA ligands: future prospects as therapeutic agents. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2019,</strong> <em>10 </em>
                                    (8)
                                     , 1242-1255. <a href="https://doi.org/10.1039/C9MD00195F" title="DOI URL">https://doi.org/10.1039/C9MD00195F</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9MD00195F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9MD00195F%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DSynthetic%252Bsmall-molecule%252BRNA%252Bligands%25253A%252Bfuture%252Bprospects%252Bas%252Btherapeutic%252Bagents%26aulast%3DDi%2BGiorgio%26aufirst%3DA.%26date%3D2019%26date%3D2019%26volume%3D10%26issue%3D8%26spage%3D1242%26epage%3D1255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Joanna  Sztuba-Solinska</span>, <span class="hlFld-ContribAuthor ">Gabriela  Chavez-Calvillo</span>, <span class="hlFld-ContribAuthor ">Sabrina Elizabeth  Cline</span>. </span><span class="cited-content_cbyCitation_article-title">Unveiling the druggable RNA targets and small molecule therapeutics. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2019,</strong> <em>27 </em>
                                    (10)
                                     , 2149-2165. <a href="https://doi.org/10.1016/j.bmc.2019.03.057" title="DOI URL">https://doi.org/10.1016/j.bmc.2019.03.057</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2019.03.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2019.03.057%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DUnveiling%252Bthe%252Bdruggable%252BRNA%252Btargets%252Band%252Bsmall%252Bmolecule%252Btherapeutics%26aulast%3DSztuba-Solinska%26aufirst%3DJoanna%26date%3D2019%26volume%3D27%26issue%3D10%26spage%3D2149%26epage%3D2165" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Amparo  Garcia-Lopez</span>, <span class="hlFld-ContribAuthor ">Francesca  Tessaro</span>, <span class="hlFld-ContribAuthor ">Hendrik R. A.  Jonker</span>, <span class="hlFld-ContribAuthor ">Anna  Wacker</span>, <span class="hlFld-ContribAuthor ">Christian  Richter</span>, <span class="hlFld-ContribAuthor ">Arnaud  Comte</span>, <span class="hlFld-ContribAuthor ">Nikolaos  Berntenis</span>, <span class="hlFld-ContribAuthor ">Roland  Schmucki</span>, <span class="hlFld-ContribAuthor ">Klas  Hatje</span>, <span class="hlFld-ContribAuthor ">Olivier  Petermann</span>, <span class="hlFld-ContribAuthor ">Gianpaolo  Chiriano</span>, <span class="hlFld-ContribAuthor ">Remo  Perozzo</span>, <span class="hlFld-ContribAuthor ">Daniel  Sciarra</span>, <span class="hlFld-ContribAuthor ">Piotr  Konieczny</span>, <span class="hlFld-ContribAuthor ">Ignacio  Faustino</span>, <span class="hlFld-ContribAuthor ">Guy  Fournet</span>, <span class="hlFld-ContribAuthor ">Modesto  Orozco</span>, <span class="hlFld-ContribAuthor ">Ruben  Artero</span>, <span class="hlFld-ContribAuthor ">Friedrich  Metzger</span>, <span class="hlFld-ContribAuthor ">Martin  Ebeling</span>, <span class="hlFld-ContribAuthor ">Peter  Goekjian</span>, <span class="hlFld-ContribAuthor ">Benoît  Joseph</span>, <span class="hlFld-ContribAuthor ">Harald  Schwalbe</span>, <span class="hlFld-ContribAuthor ">Leonardo  Scapozza</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting RNA structure in SMN2 reverses spinal muscular atrophy molecular phenotypes. </span><span class="cited-content_cbyCitation_journal-name">Nature Communications</span><span> <strong>2018,</strong> <em>9 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41467-018-04110-1" title="DOI URL">https://doi.org/10.1038/s41467-018-04110-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41467-018-04110-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41467-018-04110-1%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Communications%26atitle%3DTargeting%252BRNA%252Bstructure%252Bin%252BSMN2%252Breverses%252Bspinal%252Bmuscular%252Batrophy%252Bmolecular%252Bphenotypes%26aulast%3DGarcia-Lopez%26aufirst%3DAmparo%26date%3D2018%26date%3D2018%26volume%3D9%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Brittany S.  Morgan</span>, <span class="hlFld-ContribAuthor ">Jordan E.  Forte</span>, <span class="hlFld-ContribAuthor ">Rebecca N.  Culver</span>, <span class="hlFld-ContribAuthor ">Yuqi  Zhang</span>, <span class="hlFld-ContribAuthor ">Amanda E.  Hargrove</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Key Physicochemical, Structural, and Spatial Properties of RNA-Targeted Bioactive Ligands. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2017,</strong> <em>129 </em>
                                    (43)
                                     , 13683-13687. <a href="https://doi.org/10.1002/ange.201707641" title="DOI URL">https://doi.org/10.1002/ange.201707641</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.201707641&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.201707641%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DDiscovery%252Bof%252BKey%252BPhysicochemical%25252C%252BStructural%25252C%252Band%252BSpatial%252BProperties%252Bof%252BRNA-Targeted%252BBioactive%252BLigands%26aulast%3DMorgan%26aufirst%3DBrittany%2BS.%26date%3D2017%26date%3D2017%26volume%3D129%26issue%3D43%26spage%3D13683%26epage%3D13687" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Brittany S.  Morgan</span>, <span class="hlFld-ContribAuthor ">Jordan E.  Forte</span>, <span class="hlFld-ContribAuthor ">Rebecca N.  Culver</span>, <span class="hlFld-ContribAuthor ">Yuqi  Zhang</span>, <span class="hlFld-ContribAuthor ">Amanda E.  Hargrove</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Key Physicochemical, Structural, and Spatial Properties of RNA-Targeted Bioactive Ligands. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2017,</strong> <em>56 </em>
                                    (43)
                                     , 13498-13502. <a href="https://doi.org/10.1002/anie.201707641" title="DOI URL">https://doi.org/10.1002/anie.201707641</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.201707641&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.201707641%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DDiscovery%252Bof%252BKey%252BPhysicochemical%25252C%252BStructural%25252C%252Band%252BSpatial%252BProperties%252Bof%252BRNA-Targeted%252BBioactive%252BLigands%26aulast%3DMorgan%26aufirst%3DBrittany%2BS.%26date%3D2017%26date%3D2017%26volume%3D56%26issue%3D43%26spage%3D13498%26epage%3D13502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Thomas  Wehler</span>, <span class="hlFld-ContribAuthor ">Ruth  Brenk</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Discovery of Small Molecules Binding to RNA. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 47-77. <a href="https://doi.org/10.1007/7355_2016_29" title="DOI URL">https://doi.org/10.1007/7355_2016_29</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2016_29&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2016_29%26sid%3Dliteratum%253Aachs%26atitle%3DStructure-Based%252BDiscovery%252Bof%252BSmall%252BMolecules%252BBinding%252Bto%252BRNA%26aulast%3DWehler%26aufirst%3DThomas%26date%3D2017%26date%3D2017%26spage%3D47%26epage%3D77%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DRNA%252BTherapeutics%26aulast%3DGarner%26aufirst%3DAmanda%2BL.%26date%3D2018%26volume%3D27" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">A.  Brunschweiger</span>, <span class="hlFld-ContribAuthor ">L. F. R.  Gebert</span>, <span class="hlFld-ContribAuthor ">M.  Lucic</span>, <span class="hlFld-ContribAuthor ">U.  Pradère</span>, <span class="hlFld-ContribAuthor ">H.  Jahns</span>, <span class="hlFld-ContribAuthor ">C.  Berk</span>, <span class="hlFld-ContribAuthor ">J.  Hunziker</span>, <span class="hlFld-ContribAuthor ">J.  Hall</span>. </span><span class="cited-content_cbyCitation_article-title">Site-specific conjugation of drug-like fragments to an antimiR scaffold as a strategy to target miRNAs inside RISC. </span><span class="cited-content_cbyCitation_journal-name">Chemical Communications</span><span> <strong>2016,</strong> <em>52 </em>
                                    (1)
                                     , 156-159. <a href="https://doi.org/10.1039/C5CC07478A" title="DOI URL">https://doi.org/10.1039/C5CC07478A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C5CC07478A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC5CC07478A%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Communications%26atitle%3DSite-specific%252Bconjugation%252Bof%252Bdrug-like%252Bfragments%252Bto%252Ban%252BantimiR%252Bscaffold%252Bas%252Ba%252Bstrategy%252Bto%252Btarget%252BmiRNAs%252Binside%252BRISC%26aulast%3DBrunschweiger%26aufirst%3DA.%26date%3D2016%26date%3D2016%26volume%3D52%26issue%3D1%26spage%3D156%26epage%3D159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Joseph D.  Bauman</span>, <span class="hlFld-ContribAuthor ">Disha  Patel</span>, <span class="hlFld-ContribAuthor ">Eddy  Arnold</span>. </span><span class="cited-content_cbyCitation_article-title">Adventures in Small Molecule Fragment Screening by X-ray Crystallography. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2015,</strong>,, 197-208. <a href="https://doi.org/10.1007/978-94-017-9719-1_15" title="DOI URL">https://doi.org/10.1007/978-94-017-9719-1_15</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-94-017-9719-1_15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-94-017-9719-1_15%26sid%3Dliteratum%253Aachs%26atitle%3DAdventures%252Bin%252BSmall%252BMolecule%252BFragment%252BScreening%252Bby%252BX-ray%252BCrystallography%26aulast%3DBauman%26aufirst%3DJoseph%2BD.%26date%3D2015%26date%3D2015%26spage%3D197%26epage%3D208%26pub%3DSpringer%2520Netherlands%26atitle%3DMultifaceted%252BRoles%252Bof%252BCrystallography%252Bin%252BModern%252BDrug%252BDiscovery%26aulast%3DScapin%26aufirst%3DGiovanna%26date%3D2015%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mirco  Zeiger</span>, <span class="hlFld-ContribAuthor ">Sebastian  Stark</span>, <span class="hlFld-ContribAuthor ">Elisabeth  Kalden</span>, <span class="hlFld-ContribAuthor ">Bettina  Ackermann</span>, <span class="hlFld-ContribAuthor ">Jan  Ferner</span>, <span class="hlFld-ContribAuthor ">Ute  Scheffer</span>, <span class="hlFld-ContribAuthor ">Fatemeh  Shoja-Bazargani</span>, <span class="hlFld-ContribAuthor ">Veysel  Erdel</span>, <span class="hlFld-ContribAuthor ">Harald  Schwalbe</span>, <span class="hlFld-ContribAuthor ">Michael W.  Göbel</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment based search for small molecule inhibitors of HIV-1 Tat-TAR. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2014,</strong> <em>24 </em>
                                    (24)
                                     , 5576-5580. <a href="https://doi.org/10.1016/j.bmcl.2014.11.004" title="DOI URL">https://doi.org/10.1016/j.bmcl.2014.11.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2014.11.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2014.11.004%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DFragment%252Bbased%252Bsearch%252Bfor%252Bsmall%252Bmolecule%252Binhibitors%252Bof%252BHIV-1%252BTat-TAR%26aulast%3DZeiger%26aufirst%3DMirco%26date%3D2014%26volume%3D24%26issue%3D24%26spage%3D5576%26epage%3D5580" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shou  Zhou</span>, <span class="hlFld-ContribAuthor ">Kevin D.  Rynearson</span>, <span class="hlFld-ContribAuthor ">Kejia  Ding</span>, <span class="hlFld-ContribAuthor ">Nicholas D.  Brunn</span>, <span class="hlFld-ContribAuthor ">Thomas  Hermann</span>. </span><span class="cited-content_cbyCitation_article-title">Screening for inhibitors of the hepatitis C virus internal ribosome entry site RNA. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2013,</strong> <em>21 </em>
                                    (20)
                                     , 6139-6144. <a href="https://doi.org/10.1016/j.bmc.2013.03.054" title="DOI URL">https://doi.org/10.1016/j.bmc.2013.03.054</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2013.03.054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2013.03.054%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DScreening%252Bfor%252Binhibitors%252Bof%252Bthe%252Bhepatitis%252BC%252Bvirus%252Binternal%252Bribosome%252Bentry%252Bsite%252BRNA%26aulast%3DZhou%26aufirst%3DShou%26date%3D2013%26volume%3D21%26issue%3D20%26spage%3D6139%26epage%3D6144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Duncan E.  Scott</span>, <span class="hlFld-ContribAuthor ">Matthias T.  Ehebauer</span>, <span class="hlFld-ContribAuthor ">Tara  Pukala</span>, <span class="hlFld-ContribAuthor ">May  Marsh</span>, <span class="hlFld-ContribAuthor ">Tom L.  Blundell</span>, <span class="hlFld-ContribAuthor ">Ashok R.  Venkitaraman</span>, <span class="hlFld-ContribAuthor ">Chris  Abell</span>, <span class="hlFld-ContribAuthor ">Marko  Hyvönen</span>. </span><span class="cited-content_cbyCitation_article-title">Using a Fragment-Based Approach To Target Protein-Protein Interactions. </span><span class="cited-content_cbyCitation_journal-name">ChemBioChem</span><span> <strong>2013,</strong> <em>14 </em>
                                    (3)
                                     , 332-342. <a href="https://doi.org/10.1002/cbic.201200521" title="DOI URL">https://doi.org/10.1002/cbic.201200521</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbic.201200521&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbic.201200521%26sid%3Dliteratum%253Aachs%26jtitle%3DChemBioChem%26atitle%3DUsing%252Ba%252BFragment-Based%252BApproach%252BTo%252BTarget%252BProtein-Protein%252BInteractions%26aulast%3DScott%26aufirst%3DDuncan%2BE.%26date%3D2013%26date%3D2013%26volume%3D14%26issue%3D3%26spage%3D332%26epage%3D342" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Roba  Moumné</span>, <span class="hlFld-ContribAuthor ">Marjorie  Catala</span>, <span class="hlFld-ContribAuthor ">Valéry  Larue</span>, <span class="hlFld-ContribAuthor ">Laurent  Micouin</span>, <span class="hlFld-ContribAuthor ">Carine  Tisné</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-based design of small RNA binders: Promising developments and contribution of NMR. </span><span class="cited-content_cbyCitation_journal-name">Biochimie</span><span> <strong>2012,</strong> <em>94 </em>
                                    (7)
                                     , 1607-1619. <a href="https://doi.org/10.1016/j.biochi.2012.02.002" title="DOI URL">https://doi.org/10.1016/j.biochi.2012.02.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.biochi.2012.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.biochi.2012.02.002%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochimie%26atitle%3DFragment-based%252Bdesign%252Bof%252Bsmall%252BRNA%252Bbinders%25253A%252BPromising%252Bdevelopments%252Band%252Bcontribution%252Bof%252BNMR%26aulast%3DMoumn%25C3%25A9%26aufirst%3DRoba%26date%3D2012%26volume%3D94%26issue%3D7%26spage%3D1607%26epage%3D1619" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stephen  Neidle</span>. </span><span class="cited-content_cbyCitation_article-title">Design Principles for Quadruplex-binding Small Molecules. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2012,</strong>,, 151-174. <a href="https://doi.org/10.1016/B978-0-12-375138-6.00009-1" title="DOI URL">https://doi.org/10.1016/B978-0-12-375138-6.00009-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-375138-6.00009-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-375138-6.00009-1%26sid%3Dliteratum%253Aachs%26atitle%3DDesign%252BPrinciples%252Bfor%252BQuadruplex-binding%252BSmall%252BMolecules%26aulast%3DNeidle%26aufirst%3DStephen%26date%3D2012%26spage%3D151%26epage%3D174%26pub%3DElsevier%26atitle%3DTherapeutic%252BApplications%252Bof%252BQuadruplex%252BNucleic%252BAcids%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Amy  Davidson</span>, <span class="hlFld-ContribAuthor ">Darren W.  Begley</span>, <span class="hlFld-ContribAuthor ">Carmen  Lau</span>, <span class="hlFld-ContribAuthor ">Gabriele  Varani</span>. </span><span class="cited-content_cbyCitation_article-title">A Small-Molecule Probe Induces a Conformation in HIV TAR RNA Capable of Binding Drug-Like Fragments. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Biology</span><span> <strong>2011,</strong> <em>410 </em>
                                    (5)
                                     , 984-996. <a href="https://doi.org/10.1016/j.jmb.2011.03.039" title="DOI URL">https://doi.org/10.1016/j.jmb.2011.03.039</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmb.2011.03.039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmb.2011.03.039%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Biology%26atitle%3DA%252BSmall-Molecule%252BProbe%252BInduces%252Ba%252BConformation%252Bin%252BHIV%252BTAR%252BRNA%252BCapable%252Bof%252BBinding%252BDrug-Like%252BFragments%26aulast%3DDavidson%26aufirst%3DAmy%26date%3D2011%26volume%3D410%26issue%3D5%26spage%3D984%26epage%3D996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Peter  Daldrop</span>, <span class="hlFld-ContribAuthor ">Francis E.  Reyes</span>, <span class="hlFld-ContribAuthor ">David A.  Robinson</span>, <span class="hlFld-ContribAuthor ">Colin M.  Hammond</span>, <span class="hlFld-ContribAuthor ">David M.  Lilley</span>, <span class="hlFld-ContribAuthor ">Robert T.  Batey</span>, <span class="hlFld-ContribAuthor ">Ruth  Brenk</span>. </span><span class="cited-content_cbyCitation_article-title">Novel Ligands for a Purine Riboswitch Discovered by RNA-Ligand Docking. </span><span class="cited-content_cbyCitation_journal-name">Chemistry & Biology</span><span> <strong>2011,</strong> <em>18 </em>
                                    (3)
                                     , 324-335. <a href="https://doi.org/10.1016/j.chembiol.2010.12.020" title="DOI URL">https://doi.org/10.1016/j.chembiol.2010.12.020</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2010.12.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2010.12.020%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520%2526%2520Biology%26atitle%3DNovel%252BLigands%252Bfor%252Ba%252BPurine%252BRiboswitch%252BDiscovered%252Bby%252BRNA-Ligand%252BDocking%26aulast%3DDaldrop%26aufirst%3DPeter%26date%3D2011%26volume%3D18%26issue%3D3%26spage%3D324%26epage%3D335" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Katherine E.  Berry</span>, <span class="hlFld-ContribAuthor ">Betty  Peng</span>, <span class="hlFld-ContribAuthor ">David  Koditek</span>, <span class="hlFld-ContribAuthor ">Douglas  Beeman</span>, <span class="hlFld-ContribAuthor ">Nikos  Pagratis</span>, <span class="hlFld-ContribAuthor ">Jason K.  Perry</span>, <span class="hlFld-ContribAuthor ">Jay  Parrish</span>, <span class="hlFld-ContribAuthor ">Weidong  Zhong</span>, <span class="hlFld-ContribAuthor ">Jennifer A.  Doudna</span>, <span class="hlFld-ContribAuthor ">I-hung  Shih</span>. </span><span class="cited-content_cbyCitation_article-title">Optimized High-Throughput Screen for Hepatitis C Virus Translation Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular Screening</span><span> <strong>2011,</strong> <em>16 </em>
                                    (2)
                                     , 211-220. <a href="https://doi.org/10.1177/1087057110391665" title="DOI URL">https://doi.org/10.1177/1087057110391665</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/1087057110391665&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F1087057110391665%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520Screening%26atitle%3DOptimized%252BHigh-Throughput%252BScreen%252Bfor%252BHepatitis%252BC%252BVirus%252BTranslation%252BInhibitors%26aulast%3DBerry%26aufirst%3DKatherine%2BE.%26date%3D2011%26date%3D2011%26volume%3D16%26issue%3D2%26spage%3D211%26epage%3D220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elena  Cressina</span>, <span class="hlFld-ContribAuthor ">Liuhong  Chen</span>, <span class="hlFld-ContribAuthor ">Chris  Abell</span>, <span class="hlFld-ContribAuthor ">Finian J.  Leeper</span>, <span class="hlFld-ContribAuthor ">Alison G.  Smith</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment screening against the thiamine pyrophosphate riboswitchthiM. </span><span class="cited-content_cbyCitation_journal-name">Chem. Sci.</span><span> <strong>2011,</strong> <em>2 </em>
                                    (1)
                                     , 157-165. <a href="https://doi.org/10.1039/C0SC00406E" title="DOI URL">https://doi.org/10.1039/C0SC00406E</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C0SC00406E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC0SC00406E%26sid%3Dliteratum%253Aachs%26jtitle%3DChem.%2520Sci.%26atitle%3DFragment%252Bscreening%252Bagainst%252Bthe%252Bthiamine%252Bpyrophosphate%252BriboswitchthiM%26aulast%3DCressina%26aufirst%3DElena%26date%3D2011%26date%3D2011%26volume%3D2%26issue%3D1%26spage%3D157%26epage%3D165" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David H  Drewry</span>, <span class="hlFld-ContribAuthor ">Ricardo  Macarron</span>. </span><span class="cited-content_cbyCitation_article-title">Enhancements of screening collections to address areas of unmet medical need: an industry perspective. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2010,</strong> <em>14 </em>
                                    (3)
                                     , 289-298. <a href="https://doi.org/10.1016/j.cbpa.2010.03.024" title="DOI URL">https://doi.org/10.1016/j.cbpa.2010.03.024</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2010.03.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2010.03.024%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DEnhancements%252Bof%252Bscreening%252Bcollections%252Bto%252Baddress%252Bareas%252Bof%252Bunmet%252Bmedical%252Bneed%25253A%252Ban%252Bindustry%252Bperspective%26aulast%3DDrewry%26aufirst%3DDavid%2BH%26date%3D2010%26volume%3D14%26issue%3D3%26spage%3D289%26epage%3D298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Edward C.  Sherer</span>. </span><span class="cited-content_cbyCitation_article-title">Antibiotics Targeting the Ribosome: Structure-Based Design and the Nobel Prize. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2010,</strong>,, 139-166. <a href="https://doi.org/10.1016/S1574-1400(10)06009-3" title="DOI URL">https://doi.org/10.1016/S1574-1400(10)06009-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/S1574-1400(10)06009-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FS1574-1400%2810%2906009-3%26sid%3Dliteratum%253Aachs%26atitle%3DAntibiotics%252BTargeting%252Bthe%252BRibosome%25253A%252BStructure-Based%252BDesign%252Band%252Bthe%252BNobel%252BPrize%26aulast%3DSherer%26aufirst%3DEdward%2BC.%26date%3D2010%26spage%3D139%26epage%3D166%26pub%3DElsevier%26date%3D2010%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fareed  Aboul-ela</span>. </span><span class="cited-content_cbyCitation_article-title">Strategies for the design of RNA-binding small molecules. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2010,</strong> <em>2 </em>
                                    (1)
                                     , 93-119. <a href="https://doi.org/10.4155/fmc.09.149" title="DOI URL">https://doi.org/10.4155/fmc.09.149</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc.09.149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc.09.149%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DStrategies%252Bfor%252Bthe%252Bdesign%252Bof%252BRNA-binding%252Bsmall%252Bmolecules%26aulast%3DAboul-ela%26aufirst%3DFareed%26date%3D2010%26volume%3D2%26issue%3D1%26spage%3D93%26epage%3D119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">N.K.  Terrett</span>. </span><span class="cited-content_cbyCitation_article-title">Combinatorial Chemistry Online. </span><span class="cited-content_cbyCitation_journal-name">Combinatorial Chemistry - an Online Journal</span><span> <strong>2009,</strong> <em>11 </em>
                                    (8)
                                     , 29-32. <a href="https://doi.org/10.1016/j.comche.2009.07.001" title="DOI URL">https://doi.org/10.1016/j.comche.2009.07.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.comche.2009.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.comche.2009.07.001%26sid%3Dliteratum%253Aachs%26jtitle%3DCombinatorial%2520Chemistry%2520-%2520an%2520Online%2520Journal%26atitle%3DCombinatorial%252BChemistry%252BOnline%26aulast%3DTerrett%26aufirst%3DN.K.%26date%3D2009%26volume%3D11%26issue%3D8%26spage%3D29%26epage%3D32" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_null" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/medium/jm-2009-000659_0005.gif" alt="" id="_i1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/large/jm-2009-000659_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm9000659&amp;id=f_null"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/medium/jm-2009-000659_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/large/jm-2009-000659_0006.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Average values and standard deviations for physicochemical descriptors for nine ligand sets. For each ligand set, two bars are shown: the mean value is shown on the left and standard deviation on the right. RNA: 125 ligands reported to bind RNA with <i>K</i><sub>d</sub> < 50 μM. NCI: 1944 NCI diversity set compounds. Pittcon: 327 UPMCSD virtual screening set. Ashgate: 211 Ashgate Drug Index compounds. RNA_sub: 67 compound subset of RNA ligands with reported <i>K</i><sub>d</sub> < 3 μM. Fragments: 109 compounds that underwent QC for RNA-directed fragment library. Kinase: 162 kinase-binding ligands identified from PDB. Protease: 282 protease-binding ligands identified from PDB. Protein: 628 protein binding ligands identified from PDB. The following plots are shown (from left to right): (a) molecular weight, (b) Chi0v (zero order atomic valence connectivity index)<sup>53</sup>, (c) topological polar surface area<sup>39</sup>, (d) hydrogen bond acceptors as defined by Lipinski<sup>54</sup>, (e) simulated log of the partition coefficient in a water/octanol mixture, (f) positively charged fraction of the summed partial charge.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/large/jm-2009-000659_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm9000659&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/medium/jm-2009-000659_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/large/jm-2009-000659_0007.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) Schematic illustration of the need for selecting RNA-directed compounds for screening for binding to RNA. While we can assume that there is some overlap between the regions of “chemical space” with the propensity to bind to RNA or to proteins, it is likely that the probabilities for binding to the two classes of target will differ in any given region. Generic and commercial screening libraries, designed to cover the overlap between protein-binding and “druglike” chemical spaces, yield low hit rates for RNA targets. Note simplifications: for example, regions of RNA-binding space (and other subregions) are not necessarily contiguous. (b) Schematic of overall strategy for selection of an RNA-directed fragment library and for screening the library against RNA. Fragment library compounds are derived from the database of RNA-binding ligands. NMR is used to screen the library for compounds that bind with weak affinity to the RNA target. The complexes of these compounds with the RNA go into structural studies and provide a starting point for structure-based design of more potent ligands.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/large/jm-2009-000659_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm9000659&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/medium/jm-2009-000659_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/large/jm-2009-000659_0001.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (a) Distribution of number of fragments per cluster derived from fragmenting RNA-binding ligand structures in silico and clustering the resulting fragments according to physicochemical descriptors. (b) Distribution of number of fragments per cluster derived from fragmenting RNA-binding ligand structures in silico and clustering the resulting fragments according to chemical fingerprints. Over half of the fragments were clustered together in either cluster 3 or 71. These two clusters were further subclustered (see text).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/large/jm-2009-000659_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm9000659&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/medium/jm-2009-000659_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/large/jm-2009-000659_0002.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (a) Fragments of RNA binding ligands grouped together as “descriptive cluster 40”. (b) Library compounds that represent chemical space surrounding that covered by cluster 40. The fragment on the left has been chosen to represent the given cluster. The other two fragments, representing other descriptive or fingerprint clusters, nonetheless have similarity to a consensus structure represented by cluster 40.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/large/jm-2009-000659_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm9000659&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/medium/jm-2009-000659_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/large/jm-2009-000659_0003.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (a) Screening data. The sample contained a mixture including compounds <b>2</b> and <b>4</b> and two nonbinding compounds, together with 27 nucleotide A-site RNA. From top to bottom: water-LOGSY, without and with gentamycin competitor, 1D spectra with excitation sculpting (<i>T</i><sub>2</sub>-filter period of 0 s) and with (0.3 s) <i>T</i><sub>2</sub> filter, without gentamycin, and repeat of the excitation sculpting 1D with and without <i>T</i><sub>2</sub>-filter with gentamycin present. Competition of <b>4</b> for the gentamycin site is evident from the recovery of signal in the <i>T</i><sub>2</sub>-filter experiment, although little competition is observed in the water-LOGSY. Binding of compound <b>2</b> is indicated by signals near 6.9 and 7.3−8.1 pm in the water-LOGSY. A nonbinding compound, a furan, gives negative signals between 6.5 and 6.9 ppm. (b) Follow-up experiment confirming binding of <b>4</b> to A-site RNA. Water-LOGSY, <i>T</i><sub>2</sub>-filter, and 1D data are shown with (blue) and without (black) RNA. (c) RNA imino resonances from samples containing 10:1 excess of the respective compounds. Compounds <b>1</b>, <b>4</b>, and <b>5</b> all interact with the RNA, as indicated by shifts in imino resonances, notably in the signal for the G1491 residue (near 12 ppm), indicating that all bind near the active site. The spectrum in the second panel from the top, shows no change from the “free RNA” control, indicating that diethyl nicotinamide does not bind A-site RNA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-12/jm9000659/production/images/large/jm-2009-000659_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm9000659&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i28">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85258" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85258" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 54 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Lange, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Locher, H. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyss, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Then, R. L.</span><span> </span><span class="NLM_article-title">The targets of currently used antibacterial agents: lessons for drug discovery</span> <span class="citation_source-journal">Curr. Pharm. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">3140</span><span class="NLM_x">–</span> <span class="NLM_lpage">3154</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm9000659&amp;key=10.2174%2F138161207782110408" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm9000659&amp;key=17979755" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtlarurnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2007&pages=3140-3154&author=R.+P.+Langeauthor=H.+H.+Locherauthor=P.+C.+Wyssauthor=R.+L.+Then&title=The+targets+of+currently+used+antibacterial+agents%3A+lessons+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The targets of currently used antibacterial agents: lessons for drug discovery</span></div><div class="casAuthors">Lange, Roland P.; Locher, Hans H.; Wyss, Pierre C.; Then, Rudolf L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">3140-3154</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Based on the mode of action of antibacterial drugs currently used, targets can be defined as distinct cellular constituents such as enzymes, enzyme substrates, RNA, DNA, and membranes which exhibit very specific binding sites at the surface of these components or at the interface of macromol. complexes assembled in the cell.  Intriguingly, growth inhibition or even complete loss of bacterial viability is often the result of a cascade of events elicited upon treatment with an antibacterial agent.  In addn., their mode of action frequently involves more than one single target.  A comprehensive description of the targets exploited so far by commercialized antibacterial agents, including anti-mycobacterial agents, is given.  The no. of targets exploited so far by com. antibacterial agents is estd. to be about 40.  The most important biosynthetic pathways and cellular structures affected by antibacterial drugs are the cell wall biosynthesis, protein biosynthesis, DNA per se, replication, RNA per se, transcription and the folate biosynthetic pathway.  The disillusionment with the genomics driven antibacterial drug discovery is a result of the restrictive definition of targets as products of essential and conserved genes.  Emphasis is made to not only focus on proteins as potential drug targets, but increase efforts and devise screening technologies to discover new agents interacting with different RNA species, DNA, new protein families or macromol. complexes of these constituents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo__dPkTDx3OLVg90H21EOLACvtfcHk0lgg4McLy7Z6Fw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtlarurnE&md5=07c183e1c60e0701f0758b0d7a024f10</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.2174%2F138161207782110408&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161207782110408%26sid%3Dliteratum%253Aachs%26aulast%3DLange%26aufirst%3DR.%2BP.%26aulast%3DLocher%26aufirst%3DH.%2BH.%26aulast%3DWyss%26aufirst%3DP.%2BC.%26aulast%3DThen%26aufirst%3DR.%2BL.%26atitle%3DThe%2520targets%2520of%2520currently%2520used%2520antibacterial%2520agents%253A%2520lessons%2520for%2520drug%2520discovery%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2007%26volume%3D13%26spage%3D3140%26epage%3D3154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Foloppe, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matassova, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aboul-ela, F.</span><span> </span><span class="NLM_article-title">Towards the discovery of drug-like RNA ligands</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x"> (</span><span class="NLM_issue">21/22</span><span class="NLM_x">) </span> <span class="NLM_fpage">1019</span><span class="NLM_x">–</span> <span class="NLM_lpage">1027</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=1019-1027&issue=21%2F22&author=N.+Foloppeauthor=N.+Matassovaauthor=F.+Aboul-ela&title=Towards+the+discovery+of+drug-like+RNA+ligands"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFoloppe%26aufirst%3DN.%26aulast%3DMatassova%26aufirst%3DN.%26aulast%3DAboul-ela%26aufirst%3DF.%26atitle%3DTowards%2520the%2520discovery%2520of%2520drug-like%2520RNA%2520ligands%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2006%26volume%3D11%26issue%3D21%252F22%26spage%3D1019%26epage%3D1027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Yonath, A.</span><span> </span><span class="NLM_article-title">Antibiotics targeting ribosomes: resistance, selectivity, synergism, and cellular regulation</span> <span class="citation_source-journal">Annu. Rev. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">74</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">649</span><span class="NLM_x">–</span> <span class="NLM_lpage">679</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm9000659&amp;key=10.1146%2Fannurev.biochem.74.082803.133130" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm9000659&amp;key=16180279" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpsVensLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2005&pages=649-679&issue=1&author=A.+Yonath&title=Antibiotics+targeting+ribosomes%3A+resistance%2C+selectivity%2C+synergism%2C+and+cellular+regulation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Antibiotics targeting ribosomes: Resistance, selectivity, synergism, and cellular regulation</span></div><div class="casAuthors">Yonath, Ada</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">649-679</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Antibiotics target ribosomes at distinct locations within functionally relevant sites.  They exert their inhibitory action by diverse modes, including competing with substrate binding, interfering with ribosomal dynamics, minimizing ribosomal mobility, facilitating miscoding, hampering the progression of the mRNA chain, and blocking the nascent protein exit tunnel.  Although the ribosomes are highly conserved organelles, they possess subtle sequence and/or conformational variations.  These enable drug selectivity, thus facilitating clin. usage.  The structural implications of these differences were deciphered by comparisons of high-resoln. structures of complexes of antibiotics with ribosomal particles from eubacteria resembling pathogens and from an archaeon that shares properties with eukaryotes.  The various antibiotic-binding modes detected in these structures demonstrate that members of antibiotic families possessing common chem. elements with minute differences might bind to ribosomal pockets in significantly different modes, governed by their chem. properties.  Similarly, the nature of seemingly identical mechanisms of drug resistance is dominated, directly or via cellular effects, by the antibiotics' chem. properties.  The obsd. variability in antibiotic binding and inhibitory modes justifies expectations for structurally based improved properties of existing compds. as well as for the discovery of novel drug classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZrnIOd6ZPE7Vg90H21EOLACvtfcHk0lgogtqjSil0Fw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpsVensLk%253D&md5=8e91bdf13bb63cdadae3bd26376f8246</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.74.082803.133130&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.74.082803.133130%26sid%3Dliteratum%253Aachs%26aulast%3DYonath%26aufirst%3DA.%26atitle%3DAntibiotics%2520targeting%2520ribosomes%253A%2520resistance%252C%2520selectivity%252C%2520synergism%252C%2520and%2520cellular%2520regulation%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2005%26volume%3D74%26issue%3D1%26spage%3D649%26epage%3D679" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Steitz, T. A.</span><span> </span><span class="NLM_article-title">On the structural basis of peptide-bond formation and antibiotic resistance from atomic structures of the large ribosomal subunit</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">579</span><span class="NLM_x">, </span> <span class="NLM_fpage">955</span><span class="NLM_x">–</span> <span class="NLM_lpage">958</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=579&publication_year=2005&pages=955-958&author=T.+A.+Steitz&title=On+the+structural+basis+of+peptide-bond+formation+and+antibiotic+resistance+from+atomic+structures+of+the+large+ribosomal+subunit"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSteitz%26aufirst%3DT.%2BA.%26atitle%3DOn%2520the%2520structural%2520basis%2520of%2520peptide-bond%2520formation%2520and%2520antibiotic%2520resistance%2520from%2520atomic%2520structures%2520of%2520the%2520large%2520ribosomal%2520subunit%26jtitle%3DFEBS%2520Lett.%26date%3D2005%26volume%3D579%26spage%3D955%26epage%3D958" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Ogle, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramakrishnan, V.</span><span> </span><span class="NLM_article-title">Structural insights into translational fidelity</span> <span class="citation_source-journal">Annu. Rev. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span><span class="NLM_issue">74</span><span class="NLM_x">) </span> <span class="NLM_fpage">129</span><span class="NLM_x">–</span> <span class="NLM_lpage">177</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm9000659&amp;key=10.1146%2Fannurev.biochem.74.061903.155440" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm9000659&amp;key=15952884" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpsVens78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&pages=129-177&issue=74&author=J.+M.+Ogleauthor=V.+Ramakrishnan&title=Structural+insights+into+translational+fidelity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into translational fidelity</span></div><div class="casAuthors">Ogle, James M.; Ramakrishnan, V.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">74</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">129-177</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  The underlying basis for the accuracy of protein synthesis has been the subject of >4 decades of investigation.  Recent biochem. and structural data make it possible to understand at least in outline the structural basis for tRNA selection, in which codon recognition by cognate tRNA results in the hydrolysis of GTP by EF-Tu over 75 Å away.  The ribosome recognizes the geometry of codon-anticodon base pairing at the 1st 2 positions but monitors the 3rd, or wobble position, less stringently.  Part of the addnl. binding energy of cognate tRNA is used to induce conformational changes in the ribosome that stabilize a transition state for GTP hydrolysis by EF-Tu and subsequently result in accelerated accommodation of tRNA into the peptidyltransferase center.  The transition state for GTP hydrolysis is characterized, among other things, by a distorted tRNA.  This picture explains a large body of data on the effect of antibiotics and mutations on translational fidelity.  However, many fundamental questions remain, such as the mechanism of activation of GTP hydrolysis by EF-Tu, and the relation between decoding and frameshifting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsYMR5zqFMsrVg90H21EOLACvtfcHk0lgogtqjSil0Fw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpsVens78%253D&md5=a88c18a7164caf7fc3bdb711a855f5a1</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.74.061903.155440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.74.061903.155440%26sid%3Dliteratum%253Aachs%26aulast%3DOgle%26aufirst%3DJ.%2BM.%26aulast%3DRamakrishnan%26aufirst%3DV.%26atitle%3DStructural%2520insights%2520into%2520translational%2520fidelity%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2005%26issue%3D74%26spage%3D129%26epage%3D177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Karn, J.; Gait, M. J.; Churcher, M. J.; Mann, D. A.; Mikaelian, I. P. C.</span><span> </span><span class="NLM_article-title">Control of Human Immunodeficiency Virus Gene Expression by the RNA Binding Proteins Tat and Rev</span>. In  <span class="citation_source-book">RNA−Protein Interactions</span>; <span class="NLM_contrib-group">Nagai, K.; Mattai, I. W.</span>, Eds.; <span class="NLM_publisher-name">IRL Press</span>: <span class="NLM_publisher-loc">Oxford, U.K.</span>,<span class="NLM_x"> </span><span class="NLM_year">1995</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1995&author=J.+Karn&author=M.+J.+Gait&author=M.+J.+Churcher&author=D.+A.+Mann&author=I.+P.+C.+Mikaelianauthor=K.+Nagai&author=I.+W.+Mattai&title=RNA%E2%88%92Protein+Interactions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DKarn%26aufirst%3DJ.%26atitle%3DControl%2520of%2520Human%2520Immunodeficiency%2520Virus%2520Gene%2520Expression%2520by%2520the%2520RNA%2520Binding%2520Proteins%2520Tat%2520and%2520Rev%26btitle%3DRNA%25E2%2588%2592Protein%2520Interactions%26aulast%3DNagai%26aufirst%3DK.%26pub%3DIRL%2520Press%26date%3D1995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Lemon, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honda, M.</span><span> </span><span class="NLM_article-title">Internal ribosome entry sites within the RNA genomes of hepatitis C virus and other flaviviruses</span> <span class="citation_source-journal">Semin. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">274</span><span class="NLM_x">–</span> <span class="NLM_lpage">288</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm9000659&amp;key=10.1006%2Fsmvy.1997.0129" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADyaK1cXitFamtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1997&pages=274-288&author=S.+M.+Lemonauthor=M.+Honda&title=Internal+ribosome+entry+sites+within+the+RNA+genomes+of+hepatitis+C+virus+and+other+flaviviruses"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Internal ribosome entry sites within the RNA genomes of hepatitis C virus and other flaviviruses</span></div><div class="casAuthors">Lemon, Stanley M.; Honda, Masao</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Virology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">274-288</span>CODEN:
                <span class="NLM_cas:coden">SEVIEL</span>;
        ISSN:<span class="NLM_cas:issn">1044-5773</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">A review with 53 refs.  The 5' nontranslated RNAs of hepatitis C virus (HCV) and several other members of the Flaviviridae contain highly structured segments which form internal ribosome entry sites (IRESs).  These cis-active RNA elements direct the cap-independent initiation of translation of the viral polyprotein in assocn. with trans-acting cellular and possibly viral proteins, and thus they play a key role in the replication of the virus.  The structure of the HCV IRES does not resemble that of any picornaviral IRES, and its function is uniquely dependent upon RNA sequence extending 3' of the site of translation initiation as well as structure surrounding the initiator AUG.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpc8r_9CAqZDLVg90H21EOLACvtfcHk0lgogtqjSil0Fw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXitFamtr0%253D&md5=09797ed8f2ac38176377d151086690f1</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1006%2Fsmvy.1997.0129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fsmvy.1997.0129%26sid%3Dliteratum%253Aachs%26aulast%3DLemon%26aufirst%3DS.%2BM.%26aulast%3DHonda%26aufirst%3DM.%26atitle%3DInternal%2520ribosome%2520entry%2520sites%2520within%2520the%2520RNA%2520genomes%2520of%2520hepatitis%2520C%2520virus%2520and%2520other%2520flaviviruses%26jtitle%3DSemin.%2520Virol.%26date%3D1997%26volume%3D8%26spage%3D274%26epage%3D288" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Blackwell, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brinton, M. A.</span><span> </span><span class="NLM_article-title">BHK cell proteins that bind to the 3′ stem-loop structure of the West Nile virus genome RNA</span> <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x"> (</span><span class="NLM_issue">9</span><span class="NLM_x">) </span> <span class="NLM_fpage">5650</span><span class="NLM_x">–</span> <span class="NLM_lpage">5658</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Fjm9000659&amp;key=7637011" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADyaK2MXns1Kiu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=1995&pages=5650-5658&issue=9&author=J.+L.+Blackwellauthor=M.+A.+Brinton&title=BHK+cell+proteins+that+bind+to+the+3%E2%80%B2+stem-loop+structure+of+the+West+Nile+virus+genome+RNA"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">BHK cell proteins that bind to the 3' stem-loop structure of the West Nile virus genome RNA</span></div><div class="casAuthors">Blackwell, Jerry L.; Brinton, Margo A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">5650-8</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The first 83 3' nucleotides of the genome RNA of the flavivirus West Nile encephalitis virus (WNV) form a stable stem-loop (SL) structure which is followed in the genome by a smaller SL.  These 3' structures are highly conserved among divergent flaviviruses, suggesting that they may function as cis-acting signals for RNA replication and as such might specifically bind to cellular or viral proteins.  Cellular proteins from uninfected and WNV-infected BHK-21 S100 cytoplasmic exts. formed 3 distinct complexes with the WNV plus-strand 3' SL [(+)3'SL] RNA in a gel mobility shift assay.  Subsequent competitor gel shift analyses showed that 2 of these RNA-protein complexes, complexes 1 and 2, contained cell proteins that specifically bound to the WNV (+)3'SL RNA.  UV-induced crosslinking and Northwestern blotting analyses detected WNV (+)3'SL RNA-binding proteins of 56, 84, and 105 kDa.  When the S100 cytoplasmic exts. were partially purified by ion-exchange chromatog., a complex that comigrated with complex 1 was detected in fraction 19, whereas a complex that comigrated with complex 2 was detected in fraction 17.  UV-induced crosslinking expts. indicated that an 84-kDa cell protein in fraction 17 and a 105-kDa protein in fraction 19 bound specifically to the WNV (+)3'SL RNA.  In addn. to binding to the (+)3'SL RNA, the 105-kDa protein bound to the SL structure located at the 3' end of the WNV minus-strand RNA.  Initial mapping studies indicated that the 84- and 105-kDa proteins bind to different regions of the (+)3'SL RNA.  The 3'-terminal SL RNA of another flavivirus, dengue virus type 3, specifically competed with the WNV (+)3'SL RNA in gel shift assays, suggesting that the host proteins identified in this study are flavivirus specific.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq19kNf60fuurVg90H21EOLACvtfcHk0li3pBM9k20jjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXns1Kiu7o%253D&md5=7e4641eddbf9052d1d4799d9350dfcf9</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBlackwell%26aufirst%3DJ.%2BL.%26aulast%3DBrinton%26aufirst%3DM.%2BA.%26atitle%3DBHK%2520cell%2520proteins%2520that%2520bind%2520to%2520the%25203%25E2%2580%25B2%2520stem-loop%2520structure%2520of%2520the%2520West%2520Nile%2520virus%2520genome%2520RNA%26jtitle%3DJ.%2520Virol.%26date%3D1995%26volume%3D69%26issue%3D9%26spage%3D5650%26epage%3D5658" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Amaral, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dinger, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mercer, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattick, J. S.</span><span> </span><span class="NLM_article-title">The eukaryotic genome as an RNA machine</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">319</span><span class="NLM_x"> (</span><span class="NLM_issue">5871</span><span class="NLM_x">) </span> <span class="NLM_fpage">1787</span><span class="NLM_x">–</span> <span class="NLM_lpage">1789</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=319&publication_year=2008&pages=1787-1789&issue=5871&author=P.+P.+Amaralauthor=M.+E.+Dingerauthor=T.+R.+Mercerauthor=J.+S.+Mattick&title=The+eukaryotic+genome+as+an+RNA+machine"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAmaral%26aufirst%3DP.%2BP.%26aulast%3DDinger%26aufirst%3DM.%2BE.%26aulast%3DMercer%26aufirst%3DT.%2BR.%26aulast%3DMattick%26aufirst%3DJ.%2BS.%26atitle%3DThe%2520eukaryotic%2520genome%2520as%2520an%2520RNA%2520machine%26jtitle%3DScience%26date%3D2008%26volume%3D319%26issue%3D5871%26spage%3D1787%26epage%3D1789" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Kumari, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bugaut, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huppert, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">An RNA G-quadruplex in the 5′ UTR of the NRAS proto-oncogene modulates translation</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">218</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Fjm9000659&amp;key=10.1038%2Fnchembio864" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Fjm9000659&amp;key=17322877" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjtFKnsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2007&pages=218-221&issue=4&author=S.+Kumariauthor=A.+Bugautauthor=J.+L.+Huppertauthor=S.+Balasubramanian&title=An+RNA+G-quadruplex+in+the+5%E2%80%B2+UTR+of+the+NRAS+proto-oncogene+modulates+translation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">An RNA G-quadruplex in the 5' UTR of the NRAS proto-oncogene modulates translation</span></div><div class="casAuthors">Kumari, Sunita; Bugaut, Anthony; Huppert, Julian L.; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">218-221</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Guanine-rich nucleic acid sequences can adopt noncanonical four-stranded secondary structures called guanine (G)-quadruplexes.  Bioinformatics anal. suggests that G-quadruplex motifs are prevalent in genomes, which raises the need to elucidate their function.  There is now evidence for the existence of DNA G-quadruplexes at telomeres with assocd. biol. function.  A recent hypothesis supports the notion that gene promoter elements contain DNA G-quadruplex motifs that control gene expression at the transcriptional level.  We discovered a highly conserved, thermodynamically stable RNA G-quadruplex in the 5' untranslated region (UTR) of the gene transcript of the human NRAS proto-oncogene.  Using a cell-free translation system coupled to a reporter gene assay, we have demonstrated that this NRAS RNA G-quadruplex modulates translation.  This is the first example of translational repression by an RNA G-quadruplex.  Bioinformatics anal. has revealed 2,922 other 5' UTR RNA G-quadruplex elements in the human genome.  We propose that RNA G-quadruplexes in the 5' UTR modulate gene expression at the translational level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUZXUhKbJ4cLVg90H21EOLACvtfcHk0li3pBM9k20jjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjtFKnsrw%253D&md5=c4dda371d46b95d25b285d1adeb599be</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnchembio864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio864%26sid%3Dliteratum%253Aachs%26aulast%3DKumari%26aufirst%3DS.%26aulast%3DBugaut%26aufirst%3DA.%26aulast%3DHuppert%26aufirst%3DJ.%2BL.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DAn%2520RNA%2520G-quadruplex%2520in%2520the%25205%25E2%2580%25B2%2520UTR%2520of%2520the%2520NRAS%2520proto-oncogene%2520modulates%2520translation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2007%26volume%3D3%26issue%3D4%26spage%3D218%26epage%3D221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Hagerman, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rivera, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hagerman, P. J.</span><span> </span><span class="NLM_article-title">The fragile X family of disorders: a model for autism and targeted treatments</span> <span class="citation_source-journal">Curr. Pediatr. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">40</span><span class="NLM_x">–</span> <span class="NLM_lpage">52</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm9000659&amp;key=10.2174%2F157339608783565770" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlsleltbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=40-52&author=R.+J.+Hagermanauthor=S.+M.+Riveraauthor=P.+J.+Hagerman&title=The+fragile+X+family+of+disorders%3A+a+model+for+autism+and+targeted+treatments"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">The fragile X family of disorders: a model for autism and targeted treatments</span></div><div class="casAuthors">Hagerman, Randi J.; Rivera, Susan M.; Hagerman, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Pediatric Reviews</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">40-52</span>CODEN:
                <span class="NLM_cas:coden">CPRUC4</span>;
        ISSN:<span class="NLM_cas:issn">1875-6336</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  CGG-repeat expansion mutations of the fragile X mental retardation 1 (FMR1) gene are the leading known cause of autism and autism spectrum disorders (ASD).  Full mutation expansions (>200 CGG repeats) of the gene are generally silenced, resulting in absence of the FMR1 protein and fragile X syndrome.  By contrast, smaller expansions in the premutation range (55-200 CGG repeats) result in excess gene activity and RNA toxicity, which is responsible for the neurodegenerative disorder, fragile X-assocd. tremor/ataxia syndrome (FXTAS), and likely addnl. cases of developmental delay and autism.  Thus, the FMR1 gene is causative of a common (autism) phenotype via two entirely different pathogenic mechanisms, RNA toxicity and gene silencing.  The study of this gene and its pathogenic mechanisms therefore represents a paradigm for understanding gene-brain relationships and the means by which diverse genetic mechanisms can give rise to a common behavioral phenotype.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdvix_7PzRS7Vg90H21EOLACvtfcHk0li3pBM9k20jjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlsleltbs%253D&md5=5722cff1ef704b32d41dfe0588120acc</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.2174%2F157339608783565770&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F157339608783565770%26sid%3Dliteratum%253Aachs%26aulast%3DHagerman%26aufirst%3DR.%2BJ.%26aulast%3DRivera%26aufirst%3DS.%2BM.%26aulast%3DHagerman%26aufirst%3DP.%2BJ.%26atitle%3DThe%2520fragile%2520X%2520family%2520of%2520disorders%253A%2520a%2520model%2520for%2520autism%2520and%2520targeted%2520treatments%26jtitle%3DCurr.%2520Pediatr.%2520Rev.%26date%3D2008%26volume%3D4%26spage%3D40%26epage%3D52" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Shay, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, W. E.</span><span> </span><span class="NLM_article-title">Telomerase therapeutics for cancer: challenges and new directions</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">577</span><span class="NLM_x">–</span> <span class="NLM_lpage">584</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=577-584&issue=7&author=J.+W.+Shayauthor=W.+E.+Wright&title=Telomerase+therapeutics+for+cancer%3A+challenges+and+new+directions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DShay%26aufirst%3DJ.%2BW.%26aulast%3DWright%26aufirst%3DW.%2BE.%26atitle%3DTelomerase%2520therapeutics%2520for%2520cancer%253A%2520challenges%2520and%2520new%2520directions%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26issue%3D7%26spage%3D577%26epage%3D584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Burgstaller, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Famulok, M.</span><span> </span><span class="NLM_article-title">Flavin-dependent photocleavage of RNA at G*U base pairs</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">1137</span><span class="NLM_x">–</span> <span class="NLM_lpage">1138</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja962918p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=1997&pages=1137-1138&issue=5&author=P.+Burgstallerauthor=M.+Famulok&title=Flavin-dependent+photocleavage+of+RNA+at+G*U+base+pairs"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fja962918p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja962918p%26sid%3Dliteratum%253Aachs%26aulast%3DBurgstaller%26aufirst%3DP.%26aulast%3DFamulok%26aufirst%3DM.%26atitle%3DFlavin-dependent%2520photocleavage%2520of%2520RNA%2520at%2520G%252AU%2520base%2520pairs%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1997%26volume%3D119%26issue%3D5%26spage%3D1137%26epage%3D1138" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Burgstaller, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hermann, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huber, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westhof, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Famulok, M.</span><span> </span><span class="NLM_article-title">Isoalloxazine derivatives promote photocleavage of natural RNAs at G.U base pairs embedded within helices</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x"> (</span><span class="NLM_issue">20</span><span class="NLM_x">) </span> <span class="NLM_fpage">4018</span><span class="NLM_x">–</span> <span class="NLM_lpage">4027</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1997&pages=4018-4027&issue=20&author=P.+Burgstallerauthor=T.+Hermannauthor=C.+Huberauthor=E.+Westhofauthor=M.+Famulok&title=Isoalloxazine+derivatives+promote+photocleavage+of+natural+RNAs+at+G.U+base+pairs+embedded+within+helices"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBurgstaller%26aufirst%3DP.%26aulast%3DHermann%26aufirst%3DT.%26aulast%3DHuber%26aufirst%3DC.%26aulast%3DWesthof%26aufirst%3DE.%26aulast%3DFamulok%26aufirst%3DM.%26atitle%3DIsoalloxazine%2520derivatives%2520promote%2520photocleavage%2520of%2520natural%2520RNAs%2520at%2520G.U%2520base%2520pairs%2520embedded%2520within%2520helices%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1997%26volume%3D25%26issue%3D20%26spage%3D4018%26epage%3D4027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Murchie, A. I. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isel, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afshar, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drysdale, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bower, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Potter, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Starkey, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swarbrick, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mirza, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prescott, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vaglio, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aboul-ela, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karn, J.</span><span> </span><span class="NLM_article-title">Structure-based drug design targeting an inactive RNA conformation: exploiting the flexibility of HIV-1 TAR RNA</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">336</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">625</span><span class="NLM_x">–</span> <span class="NLM_lpage">638</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=336&publication_year=2004&pages=625-638&issue=3&author=A.+I.+H.+Murchieauthor=B.+Davisauthor=C.+Iselauthor=M.+Afsharauthor=M.+J.+Drysdaleauthor=J.+Bowerauthor=A.+J.+Potterauthor=I.+D.+Starkeyauthor=T.+M.+Swarbrickauthor=S.+Mirzaauthor=C.+D.+Prescottauthor=P.+Vaglioauthor=F.+Aboul-elaauthor=J.+Karn&title=Structure-based+drug+design+targeting+an+inactive+RNA+conformation%3A+exploiting+the+flexibility+of+HIV-1+TAR+RNA"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMurchie%26aufirst%3DA.%2BI.%2BH.%26aulast%3DDavis%26aufirst%3DB.%26aulast%3DIsel%26aufirst%3DC.%26aulast%3DAfshar%26aufirst%3DM.%26aulast%3DDrysdale%26aufirst%3DM.%2BJ.%26aulast%3DBower%26aufirst%3DJ.%26aulast%3DPotter%26aufirst%3DA.%2BJ.%26aulast%3DStarkey%26aufirst%3DI.%2BD.%26aulast%3DSwarbrick%26aufirst%3DT.%2BM.%26aulast%3DMirza%26aufirst%3DS.%26aulast%3DPrescott%26aufirst%3DC.%2BD.%26aulast%3DVaglio%26aufirst%3DP.%26aulast%3DAboul-ela%26aufirst%3DF.%26aulast%3DKarn%26aufirst%3DJ.%26atitle%3DStructure-based%2520drug%2520design%2520targeting%2520an%2520inactive%2520RNA%2520conformation%253A%2520exploiting%2520the%2520flexibility%2520of%2520HIV-1%2520TAR%2520RNA%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2004%26volume%3D336%26issue%3D3%26spage%3D625%26epage%3D638" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Davis, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afshar, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murchie, A. I. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karn, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lentzen, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drysdale, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bower, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Potter, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Starkey, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swarbrick, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aboul-ela, F.</span><span> </span><span class="NLM_article-title">Rational design of inhibtors of HIV-1 TAR RNA through the stabilisation of electrostatic “hot spots”</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">336</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">343</span><span class="NLM_x">–</span> <span class="NLM_lpage">356</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=336&publication_year=2004&pages=343-356&issue=2&author=B.+Davisauthor=M.+Afsharauthor=G.+Varaniauthor=A.+I.+H.+Murchieauthor=J.+Karnauthor=G.+Lentzenauthor=M.+Drysdaleauthor=J.+Bowerauthor=A.+J.+Potterauthor=I.+D.+Starkeyauthor=T.+Swarbrickauthor=F.+Aboul-ela&title=Rational+design+of+inhibtors+of+HIV-1+TAR+RNA+through+the+stabilisation+of+electrostatic+%E2%80%9Chot+spots%E2%80%9D"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DB.%26aulast%3DAfshar%26aufirst%3DM.%26aulast%3DVarani%26aufirst%3DG.%26aulast%3DMurchie%26aufirst%3DA.%2BI.%2BH.%26aulast%3DKarn%26aufirst%3DJ.%26aulast%3DLentzen%26aufirst%3DG.%26aulast%3DDrysdale%26aufirst%3DM.%26aulast%3DBower%26aufirst%3DJ.%26aulast%3DPotter%26aufirst%3DA.%2BJ.%26aulast%3DStarkey%26aufirst%3DI.%2BD.%26aulast%3DSwarbrick%26aufirst%3DT.%26aulast%3DAboul-ela%26aufirst%3DF.%26atitle%3DRational%2520design%2520of%2520inhibtors%2520of%2520HIV-1%2520TAR%2520RNA%2520through%2520the%2520stabilisation%2520of%2520electrostatic%2520%25E2%2580%259Chot%2520spots%25E2%2580%259D%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2004%26volume%3D336%26issue%3D2%26spage%3D343%26epage%3D356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Du, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lind, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, T. L.</span><span> </span><span class="NLM_article-title">Structure of TAR RNA complexed with a Tat-TAR interaction nanomolar inhibitor that was identified by computational screening</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">707</span><span class="NLM_x">–</span> <span class="NLM_lpage">712</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Fjm9000659&amp;key=10.1016%2FS1074-5521%2802%2900151-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Fjm9000659&amp;key=12079782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADC%252BD38XksFOisLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2002&pages=707-712&author=Z.+Duauthor=K.+E.+Lindauthor=T.+L.+James&title=Structure+of+TAR+RNA+complexed+with+a+Tat-TAR+interaction+nanomolar+inhibitor+that+was+identified+by+computational+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of TAR RNA Complexed with a Tat-TAR: Interaction Nanomolar Inhibitor that Was Identified by Computational Screening</span></div><div class="casAuthors">Du, Zhihua; Lind, Kenneth E.; James, Thomas L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">707-712</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">HIV-1 TAR RNA functions critically in viral replication by binding the transactivating regulatory protein Tat.  We recently identified several compds. that exptl. inhibit the Tat-TAR interaction completely at a 100 nM concn.  We used computational screening of the 181,000-compd. Available Chems. Directory against the three-dimensional structure of TAR.  Here we report the NMR-derived structure of TAR complexed with acetylpromazine.  This structure represents a new class of compds. with good bioavailability and low toxicity that bind with high affinity to TAR.  NMR data unambiguously show that acetylpromazine binds only to the unique 5' bulge site to which the Tat protein binds.  Specificity and affinity of binding are conferred primarily by a network of base stacking and hydrophobic interactions.  Acetylpromazine alters the structure of free TAR less than Tat peptides and neomycin do.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpc1TG4IQ3rzLVg90H21EOLACvtfcHk0lgyWdPlAQWnnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XksFOisLs%253D&md5=97f7881f6b358888c9144b69c8149163</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2FS1074-5521%2802%2900151-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1074-5521%252802%252900151-5%26sid%3Dliteratum%253Aachs%26aulast%3DDu%26aufirst%3DZ.%26aulast%3DLind%26aufirst%3DK.%2BE.%26aulast%3DJames%26aufirst%3DT.%2BL.%26atitle%3DStructure%2520of%2520TAR%2520RNA%2520complexed%2520with%2520a%2520Tat-TAR%2520interaction%2520nanomolar%2520inhibitor%2520that%2520was%2520identified%2520by%2520computational%2520screening%26jtitle%3DChem.%2520Biol.%26date%3D2002%26volume%3D9%26spage%3D707%26epage%3D712" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Hamy, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brondani, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Florsheimer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stark, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blommers, M. J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klimkait, T.</span><span> </span><span class="NLM_article-title">A new class of HIV-1 Tat antagonist acting through Tat-TAR inhibition</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">5086</span><span class="NLM_x">–</span> <span class="NLM_lpage">5095</span></span><div class="citationLinks">[<a href="/doi/10.1021/bi972947s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1998&pages=5086-5095&author=F.+Hamyauthor=V.+Brondaniauthor=A.+Florsheimerauthor=W.+Starkauthor=M.+J.+J.+Blommersauthor=T.+Klimkait&title=A+new+class+of+HIV-1+Tat+antagonist+acting+through+Tat-TAR+inhibition"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fbi972947s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi972947s%26sid%3Dliteratum%253Aachs%26aulast%3DHamy%26aufirst%3DF.%26aulast%3DBrondani%26aufirst%3DV.%26aulast%3DFlorsheimer%26aufirst%3DA.%26aulast%3DStark%26aufirst%3DW.%26aulast%3DBlommers%26aufirst%3DM.%2BJ.%2BJ.%26aulast%3DKlimkait%26aufirst%3DT.%26atitle%3DA%2520new%2520class%2520of%2520HIV-1%2520Tat%2520antagonist%2520acting%2520through%2520Tat-TAR%2520inhibition%26jtitle%3DBiochemistry%26date%3D1998%26volume%3D37%26spage%3D5086%26epage%3D5095" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Zhou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gregor, V. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ayida, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winters, G. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haley, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bailey, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Froelich, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fish, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webber, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hermann, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wall, D.</span><span> </span><span class="NLM_article-title">Synthesis and SAR of 3,5-diamino-piperidine derivatives: novel antibacterial translation inhibitors as aminoglycoside mimetics</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">1206</span><span class="NLM_x">–</span> <span class="NLM_lpage">1210</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=1206-1210&issue=5&author=Y.+Zhouauthor=V.+E.+Gregorauthor=B.+K.+Ayidaauthor=G.+C.+Wintersauthor=Z.+Sunauthor=D.+Murphyauthor=G.+Haleyauthor=D.+Baileyauthor=J.+M.+Froelichauthor=S.+Fishauthor=S.+E.+Webberauthor=T.+Hermannauthor=D.+Wall&title=Synthesis+and+SAR+of+3%2C5-diamino-piperidine+derivatives%3A+novel+antibacterial+translation+inhibitors+as+aminoglycoside+mimetics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DGregor%26aufirst%3DV.%2BE.%26aulast%3DAyida%26aufirst%3DB.%2BK.%26aulast%3DWinters%26aufirst%3DG.%2BC.%26aulast%3DSun%26aufirst%3DZ.%26aulast%3DMurphy%26aufirst%3DD.%26aulast%3DHaley%26aufirst%3DG.%26aulast%3DBailey%26aufirst%3DD.%26aulast%3DFroelich%26aufirst%3DJ.%2BM.%26aulast%3DFish%26aufirst%3DS.%26aulast%3DWebber%26aufirst%3DS.%2BE.%26aulast%3DHermann%26aufirst%3DT.%26aulast%3DWall%26aufirst%3DD.%26atitle%3DSynthesis%2520and%2520SAR%2520of%25203%252C5-diamino-piperidine%2520derivatives%253A%2520novel%2520antibacterial%2520translation%2520inhibitors%2520as%2520aminoglycoside%2520mimetics%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2007%26volume%3D17%26issue%3D5%26spage%3D1206%26epage%3D1210" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Franceschi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duffy, E. M.</span><span> </span><span class="NLM_article-title">Structure-based drug design meets the ribosome</span> <span class="citation_source-journal">Biochem. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">71</span><span class="NLM_x"> (</span><span class="NLM_issue">7</span><span class="NLM_x">) </span> <span class="NLM_fpage">1016</span><span class="NLM_x">–</span> <span class="NLM_lpage">1025</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2006&pages=1016-1025&issue=7&author=F.+Franceschiauthor=E.+M.+Duffy&title=Structure-based+drug+design+meets+the+ribosome"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFranceschi%26aufirst%3DF.%26aulast%3DDuffy%26aufirst%3DE.%2BM.%26atitle%3DStructure-based%2520drug%2520design%2520meets%2520the%2520ribosome%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2006%26volume%3D71%26issue%3D7%26spage%3D1016%26epage%3D1025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Kung, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casper, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson-Lingardo, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Risen, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vickers, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ranken, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blyn, L. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyatt, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ecker, D. J.</span><span> </span><span class="NLM_article-title">Structure−activity relationships of novel 2-substituted quinazoline antibacterial agents</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x"> (</span><span class="NLM_issue">22</span><span class="NLM_x">) </span> <span class="NLM_fpage">4705</span><span class="NLM_x">–</span> <span class="NLM_lpage">4713</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm9903500" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=4705-4713&issue=22&author=P.+P.+Kungauthor=M.+D.+Casperauthor=K.+L.+Cookauthor=L.+Wilson-Lingardoauthor=L.+M.+Risenauthor=T.+A.+Vickersauthor=R.+Rankenauthor=L.+B.+Blynauthor=J.+R.+Wyattauthor=P.+D.+Cookauthor=D.+J.+Ecker&title=Structure%E2%88%92activity+relationships+of+novel+2-substituted+quinazoline+antibacterial+agents"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm9903500&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm9903500%26sid%3Dliteratum%253Aachs%26aulast%3DKung%26aufirst%3DP.%2BP.%26aulast%3DCasper%26aufirst%3DM.%2BD.%26aulast%3DCook%26aufirst%3DK.%2BL.%26aulast%3DWilson-Lingardo%26aufirst%3DL.%26aulast%3DRisen%26aufirst%3DL.%2BM.%26aulast%3DVickers%26aufirst%3DT.%2BA.%26aulast%3DRanken%26aufirst%3DR.%26aulast%3DBlyn%26aufirst%3DL.%2BB.%26aulast%3DWyatt%26aufirst%3DJ.%2BR.%26aulast%3DCook%26aufirst%3DP.%2BD.%26aulast%3DEcker%26aufirst%3DD.%2BJ.%26atitle%3DStructure%25E2%2588%2592activity%2520relationships%2520of%2520novel%25202-substituted%2520quinazoline%2520antibacterial%2520agents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26issue%3D22%26spage%3D4705%26epage%3D4713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Morley, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afshar, M.</span><span> </span><span class="NLM_article-title">Validation of an empirical RNA−ligand scoring function for fast flexible docking using RiboDock</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">189</span><span class="NLM_x">–</span> <span class="NLM_lpage">208</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2003&pages=189-208&issue=3&author=S.+D.+Morleyauthor=M.+Afshar&title=Validation+of+an+empirical+RNA%E2%88%92ligand+scoring+function+for+fast+flexible+docking+using+RiboDock"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMorley%26aufirst%3DS.%2BD.%26aulast%3DAfshar%26aufirst%3DM.%26atitle%3DValidation%2520of%2520an%2520empirical%2520RNA%25E2%2588%2592ligand%2520scoring%2520function%2520for%2520fast%2520flexible%2520docking%2520using%2520RiboDock%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2003%26volume%3D18%26issue%3D3%26spage%3D189%26epage%3D208" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Lind, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujinaga, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterlin, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, T. L.</span><span> </span><span class="NLM_article-title">Structure-based computational database screening, in vitro assay, and NMR assessment of compounds that target RNA</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">185</span><span class="NLM_x">–</span> <span class="NLM_lpage">193</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm9000659&amp;key=10.1016%2FS1074-5521%2802%2900106-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Fjm9000659&amp;key=11880033" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADC%252BD38Xhs1yltLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2002&pages=185-193&author=K.+E.+Lindauthor=Z.+Duauthor=K.+Fujinagaauthor=B.+M.+Peterlinauthor=T.+L.+James&title=Structure-based+computational+database+screening%2C+in+vitro+assay%2C+and+NMR+assessment+of+compounds+that+target+RNA"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Computational Database Screening, In Vitro Assay, and NMR Assessment of Compounds that Target TAR RNA</span></div><div class="casAuthors">Lind, Kenneth E.; Du, Zhihua; Fujinaga, Koh; Peterlin, B. Matija; James, Thomas L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">185-193</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">There has been little prior effort to discover new drugs on the basis of a unique RNA structure.  Binding of the viral transactivator Tat to the 5' bulge of the transactivation response (TAR) element is necessary for HIV-1 replication, so TAR RNA is a superb target.  A computational approach was developed to screen a large chem. library for binding to a three-dimensional RNA structure.  Scoring function development, flexible ligand docking, and limited target flexibility were essential.  From the ranked list of compds. predicted to bind TAR, 43 were assayed for inhibition of the Tat-TAR interaction via electrophoretic mobility shift assays.  Eleven compds. (between 0.1 and 1 μM) inhibited the Tat-TAR interaction, and some inhibited Tat transactivation in cells.  NMR spectra verified specific binding to the 5' bulge and no interaction with other regions of TAR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptBQXlpQMPSrVg90H21EOLACvtfcHk0limmEqXnx6AaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xhs1yltLg%253D&md5=6aefd491cbfcd6fcd98706015df02632</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2FS1074-5521%2802%2900106-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1074-5521%252802%252900106-0%26sid%3Dliteratum%253Aachs%26aulast%3DLind%26aufirst%3DK.%2BE.%26aulast%3DDu%26aufirst%3DZ.%26aulast%3DFujinaga%26aufirst%3DK.%26aulast%3DPeterlin%26aufirst%3DB.%2BM.%26aulast%3DJames%26aufirst%3DT.%2BL.%26atitle%3DStructure-based%2520computational%2520database%2520screening%252C%2520in%2520vitro%2520assay%252C%2520and%2520NMR%2520assessment%2520of%2520compounds%2520that%2520target%2520RNA%26jtitle%3DChem.%2520Biol.%26date%3D2002%26volume%3D9%26spage%3D185%26epage%3D193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Detering, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varani, G.</span><span> </span><span class="NLM_article-title">Validation of automated docking programs for docking and database screening against RNA drug targets</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x"> (</span><span class="NLM_issue">17</span><span class="NLM_x">) </span> <span class="NLM_fpage">4188</span><span class="NLM_x">–</span> <span class="NLM_lpage">4201</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm030650o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=4188-4201&issue=17&author=C.+Deteringauthor=G.+Varani&title=Validation+of+automated+docking+programs+for+docking+and+database+screening+against+RNA+drug+targets"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm030650o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030650o%26sid%3Dliteratum%253Aachs%26aulast%3DDetering%26aufirst%3DC.%26aulast%3DVarani%26aufirst%3DG.%26atitle%3DValidation%2520of%2520automated%2520docking%2520programs%2520for%2520docking%2520and%2520database%2520screening%2520against%2520RNA%2520drug%2520targets%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26issue%3D17%26spage%3D4188%26epage%3D4201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Guilbert, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, T. L.</span><span> </span><span class="NLM_article-title">Docking to RNA via root-mean-square-deviation-driven energy minimization with flexible ligands and flexible targets</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x"> (</span><span class="NLM_issue">6</span><span class="NLM_x">) </span> <span class="NLM_fpage">1257</span><span class="NLM_x">–</span> <span class="NLM_lpage">1268</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci8000327" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmsVOhurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2008&pages=1257-1268&issue=6&author=C.+Guilbertauthor=T.+L.+James&title=Docking+to+RNA+via+root-mean-square-deviation-driven+energy+minimization+with+flexible+ligands+and+flexible+targets"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Docking to RNA via Root-Mean-Square-Deviation-Driven Energy Minimization with Flexible Ligands and Flexible Targets</span></div><div class="casAuthors">Guilbert, Christophe; James, Thomas L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1257-1268</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structure-based drug design is now well-established for proteins as a key first step in the lengthy process of developing new drugs.  In many ways, RNA may be a better target to treat disease than a protein because it is upstream in the translation pathway, so inhibiting a single mRNA mol. could prevent the prodn. of thousands of protein gene products.  Virtual screening is often the starting point for structure-based drug design.  However, computational docking of a small mol. to RNA seems to be more challenging than that to protein due to the higher intrinsic flexibility and highly charged structure of RNA.  Previous attempts at docking to RNA showed the need for a new approach.  We present here a novel algorithm using mol. simulation techniques to account for both nucleic acid and ligand flexibility.  In this approach, with both the ligand and the receptor permitted some flexibility, they can bind one another via an induced fit, as the flexible ligand probes the surface of the receptor.  A possible ligand can explore a low-energy path at the surface of the receptor by carrying out energy minimization with root-mean-square-distance constraints.  Our procedure was tested on 57 RNA complexes (33 crystal and 24 NMR structures); this is the largest data set to date to reproduce exptl. RNA binding poses.  With our procedure, the lowest-energy conformations reproduced the exptl. binding poses within an at. root-mean-square deviation of 2.5 Å for 74% of tested complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPPar0XY4iPrVg90H21EOLACvtfcHk0lihoEHcN_cOEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmsVOhurc%253D&md5=451852050ffc81c000ec95f5e496adcd</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fci8000327&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci8000327%26sid%3Dliteratum%253Aachs%26aulast%3DGuilbert%26aufirst%3DC.%26aulast%3DJames%26aufirst%3DT.%2BL.%26atitle%3DDocking%2520to%2520RNA%2520via%2520root-mean-square-deviation-driven%2520energy%2520minimization%2520with%2520flexible%2520ligands%2520and%2520flexible%2520targets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2008%26volume%3D48%26issue%3D6%26spage%3D1257%26epage%3D1268" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Hubbard, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drysdale, M. J.</span><span> </span><span class="NLM_article-title">The SeeDs approach: integrating fragments into drug discovery</span> <span class="citation_source-journal">Curr. Top. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x"> (</span><span class="NLM_issue">16</span><span class="NLM_x">) </span> <span class="NLM_fpage">1568</span><span class="NLM_x">–</span> <span class="NLM_lpage">1581</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm9000659&amp;key=10.2174%2F156802607782341109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Fjm9000659&amp;key=17979768" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVOrur7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=1568-1581&issue=16&author=R.+E.+Hubbardauthor=B.+Davisauthor=I.+Chenauthor=M.+J.+Drysdale&title=The+SeeDs+approach%3A+integrating+fragments+into+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">The SeeDs approach: integrating fragments into drug discovery</span></div><div class="casAuthors">Hubbard, Roderick E.; Davis, Ben; Chen, Ijen; Drysdale, Martin J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1568-1581</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Finding novel compds. as starting points for optimization is a major challenge in drug discovery research.  Fragment-based methods have emerged in the past ten years as an effective way to sample chem. diversity with a limited no. of low mol. wt. compds.  The structures of the fragments(s) binding to the protein can then be used to design new compds. with increased affinity, specificity and novelty.  This article describes the Vernalis approach to fragment based drug discovery, called SeeDs (Structural exploitation of exptl. Drug startpoints).  The approach includes the design of a fragment library, identification of fragments that bind competitively to a target by ligand-based NMR techniques and protein crystal structures to characterize binding.  Fragments that bind are then evolved to hits, either by growing the fragment or by combining structural features from a no. of compds.  The process is illustrated with examples from recent medicinal chem. programs to discover compds. against the oncol. targets Hsp90 and PDK1.  In addn., we summarize our experience with using mol. docking calcns. to predict fragment binding and anecdotes on the selectivity and binding modes for fragments seen against a range of targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGox6z6oFlEi67Vg90H21EOLACvtfcHk0lihoEHcN_cOEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVOrur7J&md5=6bfa0109e33c2c23cc4c5a5bcd1ed12f</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.2174%2F156802607782341109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802607782341109%26sid%3Dliteratum%253Aachs%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26aulast%3DDavis%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DI.%26aulast%3DDrysdale%26aufirst%3DM.%2BJ.%26atitle%3DThe%2520SeeDs%2520approach%253A%2520integrating%2520fragments%2520into%2520drug%2520discovery%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2007%26volume%3D7%26issue%3D16%26spage%3D1568%26epage%3D1581" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greer, J.</span><span> </span><span class="NLM_article-title">A decade of fragment-based drug design: strategic advances and lessons learned</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x"> (</span><span class="NLM_issue">3</span><span class="NLM_x">) </span> <span class="NLM_fpage">211</span><span class="NLM_x">–</span> <span class="NLM_lpage">219</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2007&pages=211-219&issue=3&author=P.+J.+Hajdukauthor=J.+Greer&title=A+decade+of+fragment-based+drug+design%3A+strategic+advances+and+lessons+learned"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DGreer%26aufirst%3DJ.%26atitle%3DA%2520decade%2520of%2520fragment-based%2520drug%2520design%253A%2520strategic%2520advances%2520and%2520lessons%2520learned%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D5%26issue%3D3%26spage%3D211%26epage%3D219" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Jhoti, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cleasby, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdonk, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, G.</span><span> </span><span class="NLM_article-title">Fragment-based screening using X-ray crystallography and NMR spectroscopy</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x"> (</span><span class="NLM_issue">5</span><span class="NLM_x">) </span> <span class="NLM_fpage">485</span><span class="NLM_x">–</span> <span class="NLM_lpage">493</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Fjm9000659&amp;key=10.1016%2Fj.cbpa.2007.07.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Fjm9000659&amp;key=17851109" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtF2rsb3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2007&pages=485-493&issue=5&author=H.+Jhotiauthor=A.+Cleasbyauthor=M.+Verdonkauthor=G.+Williams&title=Fragment-based+screening+using+X-ray+crystallography+and+NMR+spectroscopy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based screening using X-ray crystallography and NMR spectroscopy</span></div><div class="casAuthors">Jhoti, Harren; Cleasby, Anne; Verdonk, Marcel; Williams, Glyn</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">485-493</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Approaches which start from a study of the interaction of very simple mols. (fragments) with the protein target are proving to be valuable addns. to drug design.  Fragment-based screening allows the complementarity between a protein active site and drug-like mols. to be rapidly and effectively explored, using structural methods.  Recent improvements in the intensities of lab. x-ray sources permits the collection of greater amts. of high-quality diffraction data and were matched by developments in automation, crystn. and data anal.  Developments in NMR screening, including the use of cryogenically cooled NMR probes and 19F-contg. reporter mols. have expanded the scope of this technique, while increasing the availability of binding site and quant. affinity data for the fragments.  Application of these methods has led to a greater knowledge of the chem. variety, structural features and energetics of protein-fragment interactions.  While fragment-based screening has already been shown to reduce the timescales of the drug discovery process, a more detailed characterization of fragment screening hits can reveal unexpected similarities between fragment chemotypes and protein active sites leading to improved understanding of the pharmacophores and the re-use of this information against other protein targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFTfwnJaPUpLVg90H21EOLACvtfcHk0lihoEHcN_cOEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtF2rsb3E&md5=5a4e431308fe3a8d435cea55960dc663</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2007.07.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2007.07.010%26sid%3Dliteratum%253Aachs%26aulast%3DJhoti%26aufirst%3DH.%26aulast%3DCleasby%26aufirst%3DA.%26aulast%3DVerdonk%26aufirst%3DM.%26aulast%3DWilliams%26aufirst%3DG.%26atitle%3DFragment-based%2520screening%2520using%2520X-ray%2520crystallography%2520and%2520NMR%2520spectroscopy%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2007%26volume%3D11%26issue%3D5%26spage%3D485%26epage%3D493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Fejzo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lepre, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bemis, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ajay; Murcko, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, J. M.</span><span> </span><span class="NLM_article-title">The SHAPES strategy: an NMR-based approach for lead generation in drug discovery</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x"> (</span><span class="NLM_issue">10</span><span class="NLM_x">) </span> <span class="NLM_fpage">755</span><span class="NLM_x">–</span> <span class="NLM_lpage">769</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Fjm9000659&amp;key=10.1016%2FS1074-5521%2800%2980022-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Fjm9000659&amp;key=10508679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADyaK1MXmvVOns7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1999&pages=755-769&issue=10&author=J.+Fejzoauthor=C.+A.+Lepreauthor=J.+W.+Pengauthor=G.+W.+Bemisauthor=M.+A.+Ajay%3B+Murckoauthor=J.+M.+Moore&title=The+SHAPES+strategy%3A+an+NMR-based+approach+for+lead+generation+in+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The SHAPES strategy: an NMR-based approach for lead generation in drug discovery</span></div><div class="casAuthors">Fejzo, Jasna; Lepre, Christopher A.; Peng, Jeffrey W.; Bemis, Guy W.; Ajay; Murcko, Mark A.; Moore, Jonathan M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">755-769</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Current Biology Publications</span>)
        </div><div class="casAbstract">Background: Recently, it has been shown that NMR may be used to identify ligands that bind to low mol. wt. protein drug targets.  Recognizing the utility of NMR as a very sensitive method for detecting binding, we have focused on developing alternative approaches that are applicable to larger mol. wt. drug targets and do not require isotopic labeling.  Results: A new method for lead generation (SHAPES) is described that uses NMR to detect binding of a limited but diverse library of small mols. to a potential drug target.  The compd. scaffolds are derived from shapes most commonly found in known therapeutic agents.  NMR detection of low (μM-mM) affinity binding is achieved using either differential line broadening or transferred NOE (nuclear Overhauser effect) NMR techniques.  Conclusions: The SHAPES method for lead generation by NMR is useful for identifying potential lead classes of drugs early in a drug design program, and is easily integrated with other discovery tools such as virtual screening, high-throughput screening and combinatorial chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqu88axGMNTgbVg90H21EOLACvtfcHk0lhGF9lPnFr3YA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmvVOns7w%253D&md5=5f0d56e9b78579b0458ed49cf98d7c0b</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2FS1074-5521%2800%2980022-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1074-5521%252800%252980022-8%26sid%3Dliteratum%253Aachs%26aulast%3DFejzo%26aufirst%3DJ.%26aulast%3DLepre%26aufirst%3DC.%2BA.%26aulast%3DPeng%26aufirst%3DJ.%2BW.%26aulast%3DBemis%26aufirst%3DG.%2BW.%26aulast%3DAjay%253B%2BMurcko%26aufirst%3DM.%2BA.%26aulast%3DMoore%26aufirst%3DJ.%2BM.%26atitle%3DThe%2520SHAPES%2520strategy%253A%2520an%2520NMR-based%2520approach%2520for%2520lead%2520generation%2520in%2520drug%2520discovery%26jtitle%3DChem.%2520Biol.%26date%3D1999%26volume%3D6%26issue%3D10%26spage%3D755%26epage%3D769" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Hajduk, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bures, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Praestgaard, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span> </span><span class="NLM_article-title">Privileged molecules for protein binding identified from NMR-based screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x"> (</span><span class="NLM_issue">18</span><span class="NLM_x">) </span> <span class="NLM_fpage">3443</span><span class="NLM_x">–</span> <span class="NLM_lpage">3447</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm000164q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=3443-3447&issue=18&author=P.+J.+Hajdukauthor=M.+Buresauthor=J.+Praestgaardauthor=S.+W.+Fesik&title=Privileged+molecules+for+protein+binding+identified+from+NMR-based+screening"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm000164q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm000164q%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DBures%26aufirst%3DM.%26aulast%3DPraestgaard%26aufirst%3DJ.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DPrivileged%2520molecules%2520for%2520protein%2520binding%2520identified%2520from%2520NMR-based%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26issue%3D18%26spage%3D3443%26epage%3D3447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Baurin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aboul-ela, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barril, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drysdale, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dymock, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finch, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fromont, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmonite, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard, R. E.</span><span> </span><span class="NLM_article-title">Design and characterization of molecular fragments for use in NMR screening against protein tagets</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">2157</span><span class="NLM_x">–</span> <span class="NLM_lpage">2166</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci049806z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADC%252BD2cXotlaru78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2004&pages=2157-2166&author=N.+Baurinauthor=F.+Aboul-elaauthor=X.+Barrilauthor=B.+Davisauthor=M.+Drysdaleauthor=B.+Dymockauthor=H.+Finchauthor=C.+Fromontauthor=C.+Richardsonauthor=H.+Simmoniteauthor=R.+E.+Hubbard&title=Design+and+characterization+of+molecular+fragments+for+use+in+NMR+screening+against+protein+tagets"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Characterization of Libraries of Molecular Fragments for Use in NMR Screening against Protein Targets</span></div><div class="casAuthors">Baurin, Nicolas; Aboul-Ela, Fareed; Barril, Xavier; Davis, Ben; Drysdale, Martin; Dymock, Brian; Finch, Harry; Fromont, Christophe; Richardson, Christine; Simmonite, Heather; Hubbard, Roderick E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2157-2166</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have designed four generations of a low mol. wt. fragment library for use in NMR-based screening against protein targets.  The library initially contained 723 fragments which were selected manually from the Available Chems. Directory.  A series of in silico filters and property calcns. were developed to automate the selection process, allowing a larger database of 1.79M available compds. to be searched for a further 357 compds. that were added to the library.  A kinase binding pharmacophore was then derived to select 174 kinase-focused fragments.  Finally, an addnl. 61 fragments were selected to increase the no. of different pharmacophores represented within the library.  All of the fragments added to the library passed quality checks to ensure they were suitable for the screening protocol, with appropriate soly., purity, chem. stability, and unambiguous NMR spectrum.  The successive generations of libraries have been characterized through anal. of structural properties (mol. wt., lipophilicity, polar surface area, no. of rotatable bonds, and hydrogen-bonding potential) and by analyzing their pharmacophoric complexity.  These calcns. have been used to compare the fragment libraries with a drug-like ref. set of compds. and a set of mols. that bind to protein active sites.  In addn., an anal. of the overall results of screening the library against the ATP binding site of two protein targets (HSP90 and CDK2) reveals different patterns of fragment binding, demonstrating that the approach can find selective compds. that discriminate between related binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo652Eyyn5Lc7Vg90H21EOLACvtfcHk0lhGF9lPnFr3YA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXotlaru78%253D&md5=7376be270294dae6997609448513c5d7</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fci049806z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049806z%26sid%3Dliteratum%253Aachs%26aulast%3DBaurin%26aufirst%3DN.%26aulast%3DAboul-ela%26aufirst%3DF.%26aulast%3DBarril%26aufirst%3DX.%26aulast%3DDavis%26aufirst%3DB.%26aulast%3DDrysdale%26aufirst%3DM.%26aulast%3DDymock%26aufirst%3DB.%26aulast%3DFinch%26aufirst%3DH.%26aulast%3DFromont%26aufirst%3DC.%26aulast%3DRichardson%26aufirst%3DC.%26aulast%3DSimmonite%26aufirst%3DH.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DDesign%2520and%2520characterization%2520of%2520molecular%2520fragments%2520for%2520use%2520in%2520NMR%2520screening%2520against%2520protein%2520tagets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2004%26volume%3D44%26spage%3D2157%26epage%3D2166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Johnson, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feher, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williamson, J. R.</span><span> </span><span class="NLM_article-title">Application of NMR SHAPES screening to an RNA target</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x"> (</span><span class="NLM_issue">51</span><span class="NLM_x">) </span> <span class="NLM_fpage">15724</span><span class="NLM_x">–</span> <span class="NLM_lpage">15725</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja037499s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2003&pages=15724-15725&issue=51&author=E.+C.+Johnsonauthor=V.+A.+Feherauthor=J.+W.+Pengauthor=J.+M.+Mooreauthor=J.+R.+Williamson&title=Application+of+NMR+SHAPES+screening+to+an+RNA+target"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fja037499s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja037499s%26sid%3Dliteratum%253Aachs%26aulast%3DJohnson%26aufirst%3DE.%2BC.%26aulast%3DFeher%26aufirst%3DV.%2BA.%26aulast%3DPeng%26aufirst%3DJ.%2BW.%26aulast%3DMoore%26aufirst%3DJ.%2BM.%26aulast%3DWilliamson%26aufirst%3DJ.%2BR.%26atitle%3DApplication%2520of%2520NMR%2520SHAPES%2520screening%2520to%2520an%2520RNA%2520target%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2003%26volume%3D125%26issue%3D51%26spage%3D15724%26epage%3D15725" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Swayze, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jefferson, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sannes-Lowery, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blyn, L. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Risen, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arakawa, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Osgood, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hofstadler, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffey, R. H.</span><span> </span><span class="NLM_article-title">SAR by MS: a ligand based technique for drug lead discovery against structured RNA targets</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x"> (</span><span class="NLM_issue">18</span><span class="NLM_x">) </span> <span class="NLM_fpage">3816</span><span class="NLM_x">–</span> <span class="NLM_lpage">3819</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0255466" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=3816-3819&issue=18&author=E.+E.+Swayzeauthor=E.+A.+Jeffersonauthor=K.+A.+Sannes-Loweryauthor=L.+B.+Blynauthor=L.+M.+Risenauthor=S.+Arakawaauthor=S.+A.+Osgoodauthor=S.+A.+Hofstadlerauthor=R.+H.+Griffey&title=SAR+by+MS%3A+a+ligand+based+technique+for+drug+lead+discovery+against+structured+RNA+targets"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm0255466&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0255466%26sid%3Dliteratum%253Aachs%26aulast%3DSwayze%26aufirst%3DE.%2BE.%26aulast%3DJefferson%26aufirst%3DE.%2BA.%26aulast%3DSannes-Lowery%26aufirst%3DK.%2BA.%26aulast%3DBlyn%26aufirst%3DL.%2BB.%26aulast%3DRisen%26aufirst%3DL.%2BM.%26aulast%3DArakawa%26aufirst%3DS.%26aulast%3DOsgood%26aufirst%3DS.%2BA.%26aulast%3DHofstadler%26aufirst%3DS.%2BA.%26aulast%3DGriffey%26aufirst%3DR.%2BH.%26atitle%3DSAR%2520by%2520MS%253A%2520a%2520ligand%2520based%2520technique%2520for%2520drug%2520lead%2520discovery%2520against%2520structured%2520RNA%2520targets%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26issue%3D18%26spage%3D3816%26epage%3D3819" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Seth, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyaji, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jefferson, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sannes-Lowry, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Osgood, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Propp, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ranken, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Massire, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sampath, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ecker, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swayze, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffey, R. H.</span><span> </span><span class="NLM_article-title">SAR by MS: Discovery of a new class of RNA-binding small molecules for the hepatitis C virus: internal ribosome entry site IIA domain</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">7099</span><span class="NLM_x">–</span> <span class="NLM_lpage">7102</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm050815o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=7099-7102&author=P.+P.+Sethauthor=A.+Miyajiauthor=E.+A.+Jeffersonauthor=K.+A.+Sannes-Lowryauthor=S.+A.+Osgoodauthor=S.+S.+Proppauthor=R.+Rankenauthor=C.+Massireauthor=R.+Sampathauthor=D.+J.+Eckerauthor=E.+E.+Swayzeauthor=R.+H.+Griffey&title=SAR+by+MS%3A+Discovery+of+a+new+class+of+RNA-binding+small+molecules+for+the+hepatitis+C+virus%3A+internal+ribosome+entry+site+IIA+domain"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm050815o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050815o%26sid%3Dliteratum%253Aachs%26aulast%3DSeth%26aufirst%3DP.%2BP.%26aulast%3DMiyaji%26aufirst%3DA.%26aulast%3DJefferson%26aufirst%3DE.%2BA.%26aulast%3DSannes-Lowry%26aufirst%3DK.%2BA.%26aulast%3DOsgood%26aufirst%3DS.%2BA.%26aulast%3DPropp%26aufirst%3DS.%2BS.%26aulast%3DRanken%26aufirst%3DR.%26aulast%3DMassire%26aufirst%3DC.%26aulast%3DSampath%26aufirst%3DR.%26aulast%3DEcker%26aufirst%3DD.%2BJ.%26aulast%3DSwayze%26aufirst%3DE.%2BE.%26aulast%3DGriffey%26aufirst%3DR.%2BH.%26atitle%3DSAR%2520by%2520MS%253A%2520Discovery%2520of%2520a%2520new%2520class%2520of%2520RNA-binding%2520small%2520molecules%2520for%2520the%2520hepatitis%2520C%2520virus%253A%2520internal%2520ribosome%2520entry%2520site%2520IIA%2520domain%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2005%26volume%3D48%26spage%3D7099%26epage%3D7102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Yu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oost, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schkeryantz, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janowick, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesik, S. W.</span><span> </span><span class="NLM_article-title">Discovery of aminoglycoside mimetics by NMR-based screening of <i>Escherichia coli</i> A-site RNA</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x"> (</span><span class="NLM_issue">15</span><span class="NLM_x">) </span> <span class="NLM_fpage">4444</span><span class="NLM_x">–</span> <span class="NLM_lpage">4450</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja021354o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2003&pages=4444-4450&issue=15&author=L.+Yuauthor=T.+K.+Oostauthor=J.+M.+Schkeryantzauthor=J.+Yangauthor=D.+Janowickauthor=S.+W.+Fesik&title=Discovery+of+aminoglycoside+mimetics+by+NMR-based+screening+of+Escherichia+coli+A-site+RNA"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fja021354o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja021354o%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DL.%26aulast%3DOost%26aufirst%3DT.%2BK.%26aulast%3DSchkeryantz%26aufirst%3DJ.%2BM.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DJanowick%26aufirst%3DD.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26atitle%3DDiscovery%2520of%2520aminoglycoside%2520mimetics%2520by%2520NMR-based%2520screening%2520of%2520Escherichia%2520coli%2520A-site%2520RNA%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2003%26volume%3D125%26issue%3D15%26spage%3D4444%26epage%3D4450" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Fourmy, D. R., M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blanchard, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Puglis, J. D.</span><span> </span><span class="NLM_article-title">Structure of the A site of <i>Escherichia coli</i> 16S ribosomal RNA complexed with an aminoglycoside antibiotic</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">274</span><span class="NLM_x">, </span> <span class="NLM_fpage">1367</span><span class="NLM_x">–</span> <span class="NLM_lpage">1371</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Fjm9000659&amp;key=10.1126%2Fscience.274.5291.1367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Fjm9000659&amp;key=8910275" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADyaK28XntFKlsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=274&publication_year=1996&pages=1367-1371&author=D.+R.%2C+M.+I.+Fourmyauthor=S.+C.+Blanchardauthor=J.+D.+Puglis&title=Structure+of+the+A+site+of+Escherichia+coli+16S+ribosomal+RNA+complexed+with+an+aminoglycoside+antibiotic"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the A site of Escherichia coli 16S ribosomal RNA complexed with an aminoglycoside antibiotic</span></div><div class="casAuthors">Fourmy, Dominique; Recht, Michael I.; Blanchard, Scott C.; Puglisi, Joseph D.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">274</span>
        (<span class="NLM_cas:issue">5291</span>),
    <span class="NLM_cas:pages">1367-1371</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Aminoglycoside antibiotics that bind to 30S ribosomal A-site RNA cause misreading of the genetic code and inhibit translocation.  The aminoglycoside antibiotic paromomycin (I) binds specifically to an RNA oligonucleotide that contains the 30S subunit A site, and the soln. structure of the RNA-I complex was detd. by NMR spectroscopy.  The antibiotic binds in the major groove of the model A-site RNA within a pocket created by an A-A base pair and a single bulged adenine.  Specific interactions occur between aminoglycoside chem. groups important for antibiotic activity and conserved nucleotides in the RNA.  The structure explains binding of diverse aminoglycosides to the ribosome, their specific activity against prokaryotic organisms, and various resistance mechanisms, and provides insight into ribosome function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorBok0q_DmNbVg90H21EOLACvtfcHk0lgbup6ibcmXLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XntFKlsrw%253D&md5=61f144a580809d46c416c2e2ab67fcc1</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1126%2Fscience.274.5291.1367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.274.5291.1367%26sid%3Dliteratum%253Aachs%26aulast%3DFourmy%26aufirst%3DD.%2BR.%252C%2BM.%2BI.%26aulast%3DBlanchard%26aufirst%3DS.%2BC.%26aulast%3DPuglis%26aufirst%3DJ.%2BD.%26atitle%3DStructure%2520of%2520the%2520A%2520site%2520of%2520Escherichia%2520coli%252016S%2520ribosomal%2520RNA%2520complexed%2520with%2520an%2520aminoglycoside%2520antibiotic%26jtitle%3DScience%26date%3D1996%26volume%3D274%26spage%3D1367%26epage%3D1371" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Schuffenhauer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ertl, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roggo, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wetzel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koch, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waldmann, H.</span><span> </span><span class="NLM_article-title">The scaffold tree. Visualization of the scaffold universe by hierarchical scaffold classification</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">47</span><span class="NLM_x">–</span> <span class="NLM_lpage">58</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci600338x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlCit7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=47-58&issue=1&author=A.+Schuffenhauerauthor=P.+Ertlauthor=S.+Roggoauthor=S.+Wetzelauthor=M.+A.+Kochauthor=H.+Waldmann&title=The+scaffold+tree.+Visualization+of+the+scaffold+universe+by+hierarchical+scaffold+classification"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">The Scaffold Tree - Visualization of the Scaffold Universe by Hierarchical Scaffold Classification</span></div><div class="casAuthors">Schuffenhauer, Ansgar; Ertl, Peter; Roggo, Silvio; Wetzel, Stefan; Koch, Marcus A.; Waldmann, Herbert</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">47-58</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A hierarchical classification of chem. scaffolds (mol. framework, which is obtained by pruning all terminal side chains) has been introduced.  The mol. frameworks form the leaf nodes in the hierarchy trees.  By an iterative removal of rings, scaffolds forming the higher levels in the hierarchy tree are obtained.  Prioritization rules ensure that less characteristic, peripheral rings are removed first.  All scaffolds in the hierarchy tree are well-defined chem. entities making the classification chem. intuitive.  The classification is deterministic, data-set-independent, and scales linearly with the no. of compds. included in the data set.  The application of the classification is demonstrated on two data sets extd. from the PubChem database, namely, pyruvate kinase binders and a collection of pesticides.  The examples shown demonstrate that the classification procedure handles robustly synthetic structures and natural products.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8XfR2xvGasbVg90H21EOLACvtfcHk0ljAXhfWK-TOVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlCit7jO&md5=89cec5a01a37802ab0826ecababfaf0a</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fci600338x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600338x%26sid%3Dliteratum%253Aachs%26aulast%3DSchuffenhauer%26aufirst%3DA.%26aulast%3DErtl%26aufirst%3DP.%26aulast%3DRoggo%26aufirst%3DS.%26aulast%3DWetzel%26aufirst%3DS.%26aulast%3DKoch%26aufirst%3DM.%2BA.%26aulast%3DWaldmann%26aufirst%3DH.%26atitle%3DThe%2520scaffold%2520tree.%2520Visualization%2520of%2520the%2520scaffold%2520universe%2520by%2520hierarchical%2520scaffold%2520classification%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26issue%3D1%26spage%3D47%26epage%3D58" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Stockman, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalvit, C.</span><span> </span><span class="NLM_article-title">NMR screening techniques in drug discovery and drug design</span> <span class="citation_source-journal">Prog. Nucl. Magn. Reson. Spectrosc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x"> (</span><span class="NLM_issue">3−4</span><span class="NLM_x">) </span> <span class="NLM_fpage">187</span><span class="NLM_x">–</span> <span class="NLM_lpage">231</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2002&pages=187-231&issue=3%E2%88%924&author=B.+J.+Stockmanauthor=C.+Dalvit&title=NMR+screening+techniques+in+drug+discovery+and+drug+design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DStockman%26aufirst%3DB.%2BJ.%26aulast%3DDalvit%26aufirst%3DC.%26atitle%3DNMR%2520screening%2520techniques%2520in%2520drug%2520discovery%2520and%2520drug%2520design%26jtitle%3DProg.%2520Nucl.%2520Magn.%2520Reson.%2520Spectrosc.%26date%3D2002%26volume%3D41%26issue%3D3%25E2%2588%25924%26spage%3D187%26epage%3D231" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Ertl, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rohde, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Selzer, P.</span><span> </span><span class="NLM_article-title">Fast calculation of molecular polar surface area as a sum of fragment-based contributions and its application to the prediction of drug transport properties</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x"> (</span><span class="NLM_issue">20</span><span class="NLM_x">) </span> <span class="NLM_fpage">3714</span><span class="NLM_x">–</span> <span class="NLM_lpage">3717</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm000942e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmsFeitb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=3714-3717&issue=20&author=P.+Ertlauthor=B.+Rohdeauthor=P.+Selzer&title=Fast+calculation+of+molecular+polar+surface+area+as+a+sum+of+fragment-based+contributions+and+its+application+to+the+prediction+of+drug+transport+properties"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Fast Calculation of Molecular Polar Surface Area as a Sum of Fragment-Based Contributions and Its Application to the Prediction of Drug Transport Properties</span></div><div class="casAuthors">Ertl, Peter; Rohde, Bernhard; Selzer, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3714-3717</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. polar surface area (PSA), i.e., surface belonging to polar atoms, is a descriptor that was shown to correlate well with passive mol. transport through membranes and, therefore, allows prediction of transport properties of drugs.  The calcn. of PSA, however, is rather time-consuming because of the necessity to generate a reasonable 3D mol. geometry and the calcn. of the surface itself.  A new approach for the calcn. of the PSA is presented here, based on the summation of tabulated surface contributions of polar fragments.  The method, termed topol. PSA (TPSA), provides results which are practically identical with the 3D PSA (the correlation coeff. between 3D PSA and fragment-based TPSA for 34 810 mols. from the World Drug Index is 0.99), while the computation speed is 2-3 orders of magnitude faster.  The new methodol. may, therefore, be used for fast bioavailability screening of virtual libraries having millions of mols.  This article describes the new methodol. and shows the results of validation studies based on sets of published absorption data, including intestinal absorption, Caco-2 monolayer penetration, and blood-brain barrier penetration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhP6Qncx_h6LVg90H21EOLACvtfcHk0ljAXhfWK-TOVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmsFeitb4%253D&md5=23ae4e7708bd8ddeb4a24ad160a29bd6</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fjm000942e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm000942e%26sid%3Dliteratum%253Aachs%26aulast%3DErtl%26aufirst%3DP.%26aulast%3DRohde%26aufirst%3DB.%26aulast%3DSelzer%26aufirst%3DP.%26atitle%3DFast%2520calculation%2520of%2520molecular%2520polar%2520surface%2520area%2520as%2520a%2520sum%2520of%2520fragment-based%2520contributions%2520and%2520its%2520application%2520to%2520the%2520prediction%2520of%2520drug%2520transport%2520properties%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26issue%3D20%26spage%3D3714%26epage%3D3717" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Hou, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, X.</span><span> </span><span class="NLM_article-title">ADME evaluation in drug discovery. 7. Prediction of oral absorption by correlation and classification</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">208</span><span class="NLM_x">–</span> <span class="NLM_lpage">218</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci600343x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtF2it7nK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=208-218&issue=1&author=T.+Houauthor=J.+Wangauthor=W.+Zhangauthor=X.+Xu&title=ADME+evaluation+in+drug+discovery.+7.+Prediction+of+oral+absorption+by+correlation+and+classification"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">ADME evaluation in drug discovery. 7. Prediction of oral absorption by correlation and classification</span></div><div class="casAuthors">Hou, Tingjun; Wang, Junmei; Zhang, Wei; Xu, Xiaojie</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">208-218</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  A critically evaluated database of human intestinal absorption for 648 chem. compds. is reported in this study, among which 579 are believed to be transported by passive diffusion.  The correlation anal. between the intestinal absorption and several important mol. properties demonstrated that no single mol. property could be used as a good discriminator to efficiently distinguish the poorly absorbed compds. from those that are well absorbed.  The theor. correlation models for a training set of 455 compds. were proposed by the genetic function approxn. technique.  The best prediction model contains 4 mol. descriptors: topol. polar surface area, the predicted distribution coeff. at pH = 6.5, the no. of violations of the Lipinski's rule-of-five, and the square of the no. of hydrogen-bond donors.  The model was able to predict the fractional absorption with an r = 0.84 and a prediction error (abs. mean error) of 11.2% for the training set.  Moreover, it achieves an r = 0.90 and a prediction error of 7.8% for a 98-compd. test set.  The recursive partitioning technique was applied to find the simple hierarchical rules to classify the compds. into poor (%FA ≤ 30%) and good (%FA > 30%) intestinal absorption classes.  The high quality of the classification model was validated by the satisfactory predictions on the training set (correctly identifying 95.9% of the compds. in the poor-absorption class and 96.1% of the compds. in the good-absorption class) and on the test set (correctly identifying 100% of the compds. in the poor-absorption class and 96.8% of the compds. in the good-absorption class).  We expect that, in the future, the rules for the prediction of carrier-mediated transporting and 1st pass metab. can be integrated into the current hierarchical rules, and the classification model may become more powerful in the prediction of intestinal absorption or even human bioavailability.  The databases of human intestinal absorption reported here are available for download from the supporting Web site: http://modem.ucsd.edu/adme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqY2usDYjS_aLVg90H21EOLACvtfcHk0ljAXhfWK-TOVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtF2it7nK&md5=4ed85580d8ba7cbbeb7431f7f0c083f6</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fci600343x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600343x%26sid%3Dliteratum%253Aachs%26aulast%3DHou%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DXu%26aufirst%3DX.%26atitle%3DADME%2520evaluation%2520in%2520drug%2520discovery.%25207.%2520Prediction%2520of%2520oral%2520absorption%2520by%2520correlation%2520and%2520classification%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26issue%3D1%26spage%3D208%26epage%3D218" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Griffey, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sannes-Lowery, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drader, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mohan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swayze, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hofstadler, S. A.</span><span> </span><span class="NLM_article-title">Characterization of low-affinity complexes between RNA and small molecules using electrospray ionization mass spectrometry</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">122</span><span class="NLM_x"> (</span><span class="NLM_issue">41</span><span class="NLM_x">) </span> <span class="NLM_fpage">9933</span><span class="NLM_x">–</span> <span class="NLM_lpage">9938</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja0017108" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2000&pages=9933-9938&issue=41&author=R.+H.+Griffeyauthor=K.+A.+Sannes-Loweryauthor=J.+J.+Draderauthor=V.+Mohanauthor=E.+E.+Swayzeauthor=S.+A.+Hofstadler&title=Characterization+of+low-affinity+complexes+between+RNA+and+small+molecules+using+electrospray+ionization+mass+spectrometry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fja0017108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0017108%26sid%3Dliteratum%253Aachs%26aulast%3DGriffey%26aufirst%3DR.%2BH.%26aulast%3DSannes-Lowery%26aufirst%3DK.%2BA.%26aulast%3DDrader%26aufirst%3DJ.%2BJ.%26aulast%3DMohan%26aufirst%3DV.%26aulast%3DSwayze%26aufirst%3DE.%2BE.%26aulast%3DHofstadler%26aufirst%3DS.%2BA.%26atitle%3DCharacterization%2520of%2520low-affinity%2520complexes%2520between%2520RNA%2520and%2520small%2520molecules%2520using%2520electrospray%2520ionization%2520mass%2520spectrometry%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2000%26volume%3D122%26issue%3D41%26spage%3D9933%26epage%3D9938" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Pinto, I. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guilbert, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ulyanov, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stearns, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, T. L.</span><span> </span><span class="NLM_article-title">Discovery of ligands for a novel target, the human telomerase RNA, based on flexible-target virtual screening and NMR</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">7205</span><span class="NLM_x">–</span> <span class="NLM_lpage">7215</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=7205-7215&author=I.+G.+Pintoauthor=C.+Guilbertauthor=N.+B.+Ulyanovauthor=J.+Stearnsauthor=T.+L.+James&title=Discovery+of+ligands+for+a+novel+target%2C+the+human+telomerase+RNA%2C+based+on+flexible-target+virtual+screening+and+NMR"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPinto%26aufirst%3DI.%2BG.%26aulast%3DGuilbert%26aufirst%3DC.%26aulast%3DUlyanov%26aufirst%3DN.%2BB.%26aulast%3DStearns%26aufirst%3DJ.%26aulast%3DJames%26aufirst%3DT.%2BL.%26atitle%3DDiscovery%2520of%2520ligands%2520for%2520a%2520novel%2520target%252C%2520the%2520human%2520telomerase%2520RNA%252C%2520based%2520on%2520flexible-target%2520virtual%2520screening%2520and%2520NMR%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D7205%26epage%3D7215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Jenne, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartig, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piganeau, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tauer, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Samarsky, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Famulok, M.</span><span> </span><span class="NLM_article-title">Rapid identification and characterization of hammerhead-ribozyme inhibitors using fluorescence-based technology</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">56</span><span class="NLM_x">–</span> <span class="NLM_lpage">61</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2001&pages=56-61&issue=1&author=A.+Jenneauthor=J.+S.+Hartigauthor=N.+Piganeauauthor=A.+Tauerauthor=D.+A.+Samarskyauthor=M.+R.+Greenauthor=J.+Daviesauthor=M.+Famulok&title=Rapid+identification+and+characterization+of+hammerhead-ribozyme+inhibitors+using+fluorescence-based+technology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DJenne%26aufirst%3DA.%26aulast%3DHartig%26aufirst%3DJ.%2BS.%26aulast%3DPiganeau%26aufirst%3DN.%26aulast%3DTauer%26aufirst%3DA.%26aulast%3DSamarsky%26aufirst%3DD.%2BA.%26aulast%3DGreen%26aufirst%3DM.%2BR.%26aulast%3DDavies%26aufirst%3DJ.%26aulast%3DFamulok%26aufirst%3DM.%26atitle%3DRapid%2520identification%2520and%2520characterization%2520of%2520hammerhead-ribozyme%2520inhibitors%2520using%2520fluorescence-based%2520technology%26jtitle%3DNat.%2520Biotechnol.%26date%3D2001%26volume%3D19%26issue%3D1%26spage%3D56%26epage%3D61" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Blount, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Puskarz, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Penchovsky, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breaker, R.</span><span> </span><span class="NLM_article-title">Development and application of a high-throughput assay for glmS riboswitch activators</span> <span class="citation_source-journal">RNA Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x"> (</span><span class="NLM_issue">2</span><span class="NLM_x">) </span> <span class="NLM_fpage">20</span><span class="NLM_x">–</span> <span class="NLM_lpage">24</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2006&pages=20-24&issue=2&author=K.+Blountauthor=I.+Puskarzauthor=R.+Penchovskyauthor=R.+Breaker&title=Development+and+application+of+a+high-throughput+assay+for+glmS+riboswitch+activators"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBlount%26aufirst%3DK.%26aulast%3DPuskarz%26aufirst%3DI.%26aulast%3DPenchovsky%26aufirst%3DR.%26aulast%3DBreaker%26aufirst%3DR.%26atitle%3DDevelopment%2520and%2520application%2520of%2520a%2520high-throughput%2520assay%2520for%2520glmS%2520riboswitch%2520activators%26jtitle%3DRNA%2520Biol.%26date%3D2006%26volume%3D3%26issue%3D2%26spage%3D20%26epage%3D24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Wong, C.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hendrix, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Priestley, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenberg, W. A.</span><span> </span><span class="NLM_article-title">Specificity of aminoglycoside antibiotics for the A-site of the decoding region of ribosomal RNA</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">397</span><span class="NLM_x">–</span> <span class="NLM_lpage">406</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Fjm9000659&amp;key=10.1016%2FS1074-5521%2898%2990073-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Fjm9000659&amp;key=9662506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Fjm9000659&amp;key=1%3ACAS%3A528%3ADyaK1cXltVeqsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1998&pages=397-406&author=C.-H.+Wongauthor=M.+Hendrixauthor=E.+S.+Priestleyauthor=W.+A.+Greenberg&title=Specificity+of+aminoglycoside+antibiotics+for+the+A-site+of+the+decoding+region+of+ribosomal+RNA"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Specificity of aminoglycoside antibiotics for the A-site of the decoding region of ribosomal RNA</span></div><div class="casAuthors">Wong, Chi-Huey; Hendrix, Martin; Priestley, E. Scott; Greenberg, William A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">397-406</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Current Biology Ltd.</span>)
        </div><div class="casAbstract">Aminoglycoside antibiotics bind to the A-site of the decoding region of 16S RNA in the bacterial ribosome, an interaction that is probably responsible for their activity.  A detailed study of the specificity of aminoglycoside binding to A-site RNA would improve our understanding of their mechanism of antibiotic activity.  We have studied the binding specificity of several aminoglycosides with model RNA sequences derived from the 16S ribosomal A-site using surface plasmon resonance.  The 4,5-linked (neomycin) class of aminoglycosides showed specificity for wild-type A-site sequences, but the 4,6-linked class (kanamycins and gentamicins), generally showed poor specificity for the same sequences.  Methylation of a cytidine in the target RNA, as found in the Escherichia coli ribosome, had negligible effects on aminoglycoside binding.  Although both 4,5- and 4,6-linked aminoglycosides target the same ribosomal site, they appear to bind and effect antibiotic activity in different manners.  The aminoglycosides might recognize different RNA conformations or the interaction might involve different RNA tertiary structures that are not equally sampled in our ribosome-free model.  These results imply that models of rRNA must be carefully designed if the data are expected to accurately reflect biol. activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTLZJpmzaNnrVg90H21EOLACvtfcHk0lio2rrl-xosqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXltVeqsbk%253D&md5=1d0c410d2a011ff6f771c1fb18859038</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2FS1074-5521%2898%2990073-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1074-5521%252898%252990073-4%26sid%3Dliteratum%253Aachs%26aulast%3DWong%26aufirst%3DC.-H.%26aulast%3DHendrix%26aufirst%3DM.%26aulast%3DPriestley%26aufirst%3DE.%2BS.%26aulast%3DGreenberg%26aufirst%3DW.%2BA.%26atitle%3DSpecificity%2520of%2520aminoglycoside%2520antibiotics%2520for%2520the%2520A-site%2520of%2520the%2520decoding%2520region%2520of%2520ribosomal%2520RNA%26jtitle%3DChem.%2520Biol.%26date%3D1998%26volume%3D5%26spage%3D397%26epage%3D406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Mayer, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Famulok, M.</span><span> </span><span class="NLM_article-title">High-throughput-compatible assay for glmS riboswitch metabolite dependence</span> <span class="citation_source-journal">ChemBioChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x"> (</span><span class="NLM_issue">4</span><span class="NLM_x">) </span> <span class="NLM_fpage">602</span><span class="NLM_x">–</span> <span class="NLM_lpage">604</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&pages=602-604&issue=4&author=G.+Mayerauthor=M.+Famulok&title=High-throughput-compatible+assay+for+glmS+riboswitch+metabolite+dependence"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMayer%26aufirst%3DG.%26aulast%3DFamulok%26aufirst%3DM.%26atitle%3DHigh-throughput-compatible%2520assay%2520for%2520glmS%2520riboswitch%2520metabolite%2520dependence%26jtitle%3DChemBioChem%26date%3D2006%26volume%3D7%26issue%3D4%26spage%3D602%26epage%3D604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Faber, C. S., H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schweimer, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosch, P.</span><span> </span><span class="NLM_article-title">Structural rearrangements of HIV-Tat-responsive RNA upon binding of neomycin B</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">275</span><span class="NLM_x"> (</span><span class="NLM_issue">27</span><span class="NLM_x">) </span> <span class="NLM_fpage">20660</span><span class="NLM_x">–</span> <span class="NLM_lpage">20666</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=275&publication_year=2000&pages=20660-20666&issue=27&author=C.+S.%2C+H.+Faberauthor=K.+Schweimerauthor=P.+Rosch&title=Structural+rearrangements+of+HIV-Tat-responsive+RNA+upon+binding+of+neomycin+B"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFaber%26aufirst%3DC.%2BS.%252C%2BH.%26aulast%3DSchweimer%26aufirst%3DK.%26aulast%3DRosch%26aufirst%3DP.%26atitle%3DStructural%2520rearrangements%2520of%2520HIV-Tat-responsive%2520RNA%2520upon%2520binding%2520of%2520neomycin%2520B%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2000%26volume%3D275%26issue%3D27%26spage%3D20660%26epage%3D20666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Ennifar, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paillart, J.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bodlenner, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walter, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weibel, J.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aubertin, A.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pale, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dumas, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marquet, R.</span><span> </span><span class="NLM_article-title">Targeting the dimerization initiation site of HIV-1 RNA with aminoglycosides: from crystal to cell</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">2328</span><span class="NLM_x">–</span> <span class="NLM_lpage">2339</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=2328-2339&issue=8&author=E.+Ennifarauthor=J.-C.+Paillartauthor=A.+Bodlennerauthor=P.+Walterauthor=J.-M.+Weibelauthor=A.-M.+Aubertinauthor=P.+Paleauthor=P.+Dumasauthor=R.+Marquet&title=Targeting+the+dimerization+initiation+site+of+HIV-1+RNA+with+aminoglycosides%3A+from+crystal+to+cell"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEnnifar%26aufirst%3DE.%26aulast%3DPaillart%26aufirst%3DJ.-C.%26aulast%3DBodlenner%26aufirst%3DA.%26aulast%3DWalter%26aufirst%3DP.%26aulast%3DWeibel%26aufirst%3DJ.-M.%26aulast%3DAubertin%26aufirst%3DA.-M.%26aulast%3DPale%26aufirst%3DP.%26aulast%3DDumas%26aufirst%3DP.%26aulast%3DMarquet%26aufirst%3DR.%26atitle%3DTargeting%2520the%2520dimerization%2520initiation%2520site%2520of%2520HIV-1%2520RNA%2520with%2520aminoglycosides%253A%2520from%2520crystal%2520to%2520cell%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26issue%3D8%26spage%3D2328%26epage%3D2339" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Staple, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Venditti, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niccolai, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elson-Schwab, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tor, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butcher, S. E.</span><span> </span><span class="NLM_article-title">Guanidinoneomycin B recognition of an HIV-1 RNA helix</span> <span class="citation_source-journal">ChemBioChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">93</span><span class="NLM_x">–</span> <span class="NLM_lpage">102</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=93-102&issue=1&author=D.+W.+Stapleauthor=V.+Vendittiauthor=N.+Niccolaiauthor=L.+Elson-Schwabauthor=Y.+Torauthor=S.+E.+Butcher&title=Guanidinoneomycin+B+recognition+of+an+HIV-1+RNA+helix"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DStaple%26aufirst%3DD.%2BW.%26aulast%3DVenditti%26aufirst%3DV.%26aulast%3DNiccolai%26aufirst%3DN.%26aulast%3DElson-Schwab%26aufirst%3DL.%26aulast%3DTor%26aufirst%3DY.%26aulast%3DButcher%26aufirst%3DS.%2BE.%26atitle%3DGuanidinoneomycin%2520B%2520recognition%2520of%2520an%2520HIV-1%2520RNA%2520helix%26jtitle%3DChemBioChem%26date%3D2008%26volume%3D9%26issue%3D1%26spage%3D93%26epage%3D102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Anupam, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denapoli, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muchenditsi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hines, J. V.</span><span> </span><span class="NLM_article-title">Identification of neomycin B-binding site in T box antiterminator model RNA</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x"> (</span><span class="NLM_issue">8</span><span class="NLM_x">) </span> <span class="NLM_fpage">4466</span><span class="NLM_x">–</span> <span class="NLM_lpage">4470</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2008&pages=4466-4470&issue=8&author=R.+Anupamauthor=L.+Denapoliauthor=A.+Muchenditsiauthor=J.+V.+Hines&title=Identification+of+neomycin+B-binding+site+in+T+box+antiterminator+model+RNA"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAnupam%26aufirst%3DR.%26aulast%3DDenapoli%26aufirst%3DL.%26aulast%3DMuchenditsi%26aufirst%3DA.%26aulast%3DHines%26aufirst%3DJ.%2BV.%26atitle%3DIdentification%2520of%2520neomycin%2520B-binding%2520site%2520in%2520T%2520box%2520antiterminator%2520model%2520RNA%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2008%26volume%3D16%26issue%3D8%26spage%3D4466%26epage%3D4470" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Schüller, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suhartono, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fechner, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanrikulu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breitung, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheffer, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Göbel, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, G.</span><span> </span><span class="NLM_article-title">The concept of template-based de novo design from drug-derived molecular fragments and its application to TAR RNA</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">59</span><span class="NLM_x">–</span> <span class="NLM_lpage">68</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2008&pages=59-68&author=A.+Sch%C3%BCllerauthor=M.+Suhartonoauthor=U.+Fechnerauthor=Y.+Tanrikuluauthor=S.+Breitungauthor=U.+Schefferauthor=M.+W.+G%C3%B6belauthor=G.+Schneider&title=The+concept+of+template-based+de+novo+design+from+drug-derived+molecular+fragments+and+its+application+to+TAR+RNA"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSch%25C3%25BCller%26aufirst%3DA.%26aulast%3DSuhartono%26aufirst%3DM.%26aulast%3DFechner%26aufirst%3DU.%26aulast%3DTanrikulu%26aufirst%3DY.%26aulast%3DBreitung%26aufirst%3DS.%26aulast%3DScheffer%26aufirst%3DU.%26aulast%3DG%25C3%25B6bel%26aufirst%3DM.%2BW.%26aulast%3DSchneider%26aufirst%3DG.%26atitle%3DThe%2520concept%2520of%2520template-based%2520de%2520novo%2520design%2520from%2520drug-derived%2520molecular%2520fragments%2520and%2520its%2520application%2520to%2520TAR%2520RNA%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2008%26volume%3D22%26spage%3D59%26epage%3D68" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Disney, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labuda, L. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paul, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poplawski, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pushechnikov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tran, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velagapudi, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Childs-Disney, J. L.</span><span> </span><span class="NLM_article-title">Two-dimensional combinatorial screening identifies specific aminoglycoside-RNA internal loop partners</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">130</span><span class="NLM_x"> (</span><span class="NLM_issue">33</span><span class="NLM_x">) </span> <span class="NLM_fpage">11185</span><span class="NLM_x">–</span> <span class="NLM_lpage">11194</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja803234t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=11185-11194&issue=33&author=M.+D.+Disneyauthor=L.+P.+Labudaauthor=D.+J.+Paulauthor=S.+G.+Poplawskiauthor=A.+Pushechnikovauthor=T.+Tranauthor=S.+P.+Velagapudiauthor=M.+Wuauthor=J.+L.+Childs-Disney&title=Two-dimensional+combinatorial+screening+identifies+specific+aminoglycoside-RNA+internal+loop+partners"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fja803234t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja803234t%26sid%3Dliteratum%253Aachs%26aulast%3DDisney%26aufirst%3DM.%2BD.%26aulast%3DLabuda%26aufirst%3DL.%2BP.%26aulast%3DPaul%26aufirst%3DD.%2BJ.%26aulast%3DPoplawski%26aufirst%3DS.%2BG.%26aulast%3DPushechnikov%26aufirst%3DA.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DWu%26aufirst%3DM.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26atitle%3DTwo-dimensional%2520combinatorial%2520screening%2520identifies%2520specific%2520aminoglycoside-RNA%2520internal%2520loop%2520partners%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26issue%3D33%26spage%3D11185%26epage%3D11194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Hall, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kier, L. B.</span><span> </span><span class="NLM_article-title">The molecular connectivity chi indexes and kappa shape indexes in structure-property modeling</span> <span class="citation_source-journal">Reviews in Computational Chemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">367</span><span class="NLM_x">–</span> <span class="NLM_lpage">422</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=1991&pages=367-422&author=L.+H.+Hallauthor=L.+B.+Kier&title=The+molecular+connectivity+chi+indexes+and+kappa+shape+indexes+in+structure-property+modeling"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHall%26aufirst%3DL.%2BH.%26aulast%3DKier%26aufirst%3DL.%2BB.%26atitle%3DThe%2520molecular%2520connectivity%2520chi%2520indexes%2520and%2520kappa%2520shape%2520indexes%2520in%2520structure-property%2520modeling%26jtitle%3DReviews%2520in%2520Computational%2520Chemistry%26date%3D1991%26volume%3D2%26spage%3D367%26epage%3D422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Lipinski, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lombardo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dominy, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feeney, P. J.</span><span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span> <span class="citation_source-journal">Advanced Drug Delivery Reviews</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x"> (</span><span class="NLM_issue">1−3</span><span class="NLM_x">) </span> <span class="NLM_fpage">3</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=1997&pages=3-26&issue=1%E2%88%923&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdvanced%2520Drug%2520Delivery%2520Reviews%26date%3D1997%26volume%3D46%26issue%3D1%25E2%2588%25923%26spage%3D3%26epage%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-si1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i27"><a href="/doi/suppl/10.1021/jm9000659">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_41118"></div></div></div></div></div><hr /></hr><p class="last">Extended discussion of cheminformatics; extended description of experimental procedures for descriptor comparisons, selection, plating, and quality control of fragments, fragment clustering, RNA binding ligands database (Table S1); average and standard deviations of MOE calculated descriptors of eight ligand sets (Table S2); RNA directed fragment library compounds (Table S3); list of the 20 RNA-binding ligands with the largest calculated TPSA value; distribution of calculated TPSA for eight ligand sets (Figure S1). This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm9000659/suppl_file/jm9000659_si_001.pdf">jm9000659_si_001.pdf (1.28 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm9000659&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2009.52.issue-12%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Fjm9000659" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm9000659" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799ee9e3d603ca6","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
